<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/2901" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/2901.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/2901.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/9/2901.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/2901.atom" />
<meta name="HW.pisa" content="ggg;8/9/2901" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes" />
<meta name="DC.Identifier" content="10.1534/g3.118.200522" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Deng, Kuo. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6227180" />
<meta name="DC.Description" content="The spindle assembly checkpoint (SAC) is key to faithful segregation of chromosomes. One requirement that satisfies SAC is appropriate tension between sister chromatids at the metaphase-anaphase juncture. Proper tension generated by poleward pulling of mitotic spindles signals biorientation of the underlying chromosome. In the budding yeast, the tension status is monitored by the conserved Shugoshin protein, [Sgo1p][1], and the tension sensing motif (TSM) of histone H3. ChIP-seq reveals a unique TSM-dependent, tripartite domain of [Sgo1p][1] in each mitotic chromosome. This domain consists of one centromeric and two flanking peaks 3 – 4 kb away, present exclusively in mitosis. Strikingly, this trident motif coincides with cohesin localization, but only at the centromere and the two immediate adjacent loci, despite that cohesin is enriched at numerous regions throughout mitotic chromosomes. Chromosome conformation capture assays reveal apparent looping at the centromeric and pericentric regions. The TSM-[Sgo1p][1]-cohesin triad is therefore at the center stage of higher-ordered chromatin architecture for error-free segregation.

 [1]: http://www.yeastgenome.org/locus/S000005599/overview" />
<meta name="DC.Contributor" content="Xiexiong Deng" />
<meta name="DC.Contributor" content="Min-Hao Kuo" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The spindle assembly checkpoint (SAC) is key to faithful segregation of chromosomes. One requirement that satisfies SAC is appropriate tension between sister chromatids at the metaphase-anaphase juncture. Proper tension generated by poleward pulling of mitotic spindles signals biorientation of the underlying chromosome. In the budding yeast, the tension status is monitored by the conserved Shugoshin protein, Sgo1p, and the tension sensing motif (TSM) of histone H3. ChIP-seq reveals a unique TSM-dependent, tripartite domain of Sgo1p in each mitotic chromosome. This domain consists of one centromeric and two flanking peaks 3 – 4 kb away, present exclusively in mitosis. Strikingly, this trident motif coincides with cohesin localization, but only at the centromere and the two immediate adjacent loci, despite that cohesin is enriched at numerous regions throughout mitotic chromosomes. Chromosome conformation capture assays reveal apparent looping at the centromeric and pericentric regions. The TSM-Sgo1p-cohesin triad is therefore at the center stage of higher-ordered chromatin architecture for error-free segregation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/2901" />
<meta name="citation_id" content="8/9/2901" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/2901" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/2901.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/2901.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/2901.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200522" />
<meta name="citation_pmid" content="30002083" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2901" />
<meta name="citation_lastpage" content="2911" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xiexiong Deng" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0025-5383" />
<meta name="citation_author" content="Min-Hao Kuo" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824" />
<meta name="citation_author_email" content="kuom@msu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8096-5344" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. Akada;citation_author=I. Hirosawa;citation_author=M. Kawahata;citation_author=H. Hoshida;citation_author=Y. Nishizawa;citation_title=Sets of integrating plasmids and gene disruption cassettes containing improved counter-selection markers designed for repeated use in budding yeast.;citation_pages=393-402;citation_volume=19;citation_year=2002;citation_issue=5;citation_pmid=11921088;citation_doi=10.1002/yea.841" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Protoc.;citation_author=J. M. Belton;citation_author=J. Dekker;citation_title=Chromosome Conformation Capture (3C) in Budding Yeast.;citation_pages=580-586;citation_volume=2015;citation_year=2015;citation_doi=10.1101/pdb.prot085175" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. M. Brady;citation_author=K. G. Hardwick;citation_title=Complex formation between Mad1p, Bub1p and Bub3p is crucial for spindle checkpoint function.;citation_pages=675-678;citation_volume=10;citation_year=2000;citation_issue=11;citation_pmid=10837255;citation_doi=10.1016/S0960-9822(00)00515-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. J. Buehl;citation_author=X. Deng;citation_author=J. Luo;citation_author=V. Buranasudja;citation_author=T. Hazbun;citation_title=A Failsafe for Sensing Chromatid Tension in Mitosis with the Histone H3 Tail in Saccharomyces cerevisiae.;citation_volume=208;citation_year=2018;citation_pmid=29242290;citation_doi=10.1534/genetics.117.300606" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. W. Cleveland;citation_author=Y. Mao;citation_author=K. F. Sullivan;citation_title=Centromeres and kinetochores: from epigenetics to mitotic checkpoint signaling.;citation_pages=407-421;citation_volume=112;citation_year=2003;citation_issue=4;citation_pmid=12600307;citation_doi=10.1016/S0092-8674(03)00115-6" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. Clift;citation_author=F. Bizzari;citation_author=A. L. Marston;citation_title=Shugoshin prevents cohesin cleavage by PP2ACdc55-dependent inhibition of separase;citation_pages=766-780;citation_volume=23;citation_year=2009;citation_issue=6;citation_pmid=19299562;citation_doi=10.1101/gad.507509" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Dekker;citation_author=K. Rippe;citation_author=M. Dekker;citation_author=N. Kleckner;citation_title=Capturing Chromosome Conformation;citation_pages=1306-1311;citation_volume=295;citation_year=2002;citation_issue=5558;citation_pmid=11847345;citation_doi=10.1126/science.1067799" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=H. D. Eshleman;citation_author=D. O. Morgan;citation_title=Sgo1 recruits PP2A to chromosomes to ensure sister chromatid bi-orientation during mitosis.;citation_volume=127;citation_year=2014;citation_pmid=25236599;citation_doi=10.1242/jcs.161273" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Fernius;citation_author=K. G. Hardwick;citation_title=Bub1 kinase targets Sgo1 to ensure efficient chromosome biorientation in budding yeast mitosis.;citation_pages=e213-e213;citation_volume=3;citation_year=2007;citation_issue=11;citation_pmid=18081426;citation_doi=10.1371/journal.pgen.0030213" />
<meta name="citation_reference" content="citation_journal_title=BMC Res. Notes;citation_author=E. Ford;citation_author=C. Nikopoulou;citation_author=A. Kokkalis;citation_author=D. Thanos;citation_title=A method for generating highly multiplexed ChIP-seq libraries.;citation_pages=312;citation_volume=7;citation_year=2014;citation_pmid=24885602;citation_doi=10.1186/1756-0500-7-312" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. Gietz;citation_author=A. St Jean;citation_author=R. A. Woods;citation_author=R. H. Schiestl;citation_title=Improved method for high efficiency transformation of intact yeast cells;citation_pages=1425-1425;citation_volume=20;citation_year=1992;citation_issue=6;citation_pmid=1561104;citation_doi=10.1093/nar/20.6.1425" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. F. Glynn;citation_author=P. C. Megee;citation_author=H. G. Yu;citation_author=C. Mistrot;citation_author=E. Unal;citation_title=Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae.;citation_pages=E259-E259;citation_volume=2;citation_year=2004;citation_issue=9;citation_pmid=15309048;citation_doi=10.1371/journal.pbio.0020259" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. He;citation_author=S. Asthana;citation_author=P. K. Sorger;citation_title=Transient sister chromatid separation and elastic deformation of chromosomes during mitosis in budding yeast.;citation_pages=763-775;citation_volume=101;citation_year=2000;citation_issue=7;citation_pmid=10892747;citation_doi=10.1016/S0092-8674(00)80888-0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. He;citation_author=D. R. Rines;citation_author=C. W. Espelin;citation_author=P. K. Sorger;citation_title=Molecular analysis of kinetochore-microtubule attachment in budding yeast.;citation_pages=195-206;citation_volume=106;citation_year=2001;citation_issue=2;citation_pmid=11511347;citation_doi=10.1016/S0092-8674(01)00438-X" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=D. F. Hudson;citation_author=K. M. Marshall;citation_author=W. C. Earnshaw;citation_title=Condensin: Architect of mitotic chromosomes.;citation_pages=131-144;citation_volume=17;citation_year=2009;citation_issue=2;citation_pmid=19308696;citation_doi=10.1007/s10577-008-9009-7" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. B. Indjeian;citation_author=B. M. Stern;citation_author=A. W. Murray;citation_title=The Centromeric Protein Sgo1 Is Required to Sense Lack of Tension on Mitotic Chromosomes;citation_pages=130-133;citation_volume=307;citation_year=2005;citation_issue=5706;citation_pmid=15637284;citation_doi=10.1126/science.1101366" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=R. S. Isaac;citation_author=S. Sanulli;citation_author=R. Tibble;citation_author=M. Hornsby;citation_author=M. Ravalin;citation_title=Biochemical Basis for Distinct Roles of the Heterochromatin Proteins Swi6 and Chp2.;citation_pages=3666-3677;citation_volume=429;citation_year=2017;citation_doi=10.1016/j.jmb.2017.09.012" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=Q. Jin;citation_author=E. Trelles-Sticken;citation_author=H. Scherthan;citation_author=J. Loidl;citation_title=Yeast nuclei display prominent centromere clustering that is reduced in nondividing cells and in meiotic prophase.;citation_volume=141;citation_year=1998;citation_pmid=9531545;citation_doi=10.1083/jcb.141.1.21" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Kawashima;citation_author=Y. Yamagishi;citation_author=T. Honda;citation_author=K. Ishiguro;citation_author=Y. Watanabe;citation_title=Phosphorylation of H2A by Bub1 prevents chromosomal instability through localizing shugoshin.;citation_volume=327;citation_year=2010;citation_pmid=19965387;citation_doi=10.1126/science.1180189" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=B. M. Kiburz;citation_author=A. Amon;citation_author=A. L. Marston;citation_title=Shugoshin promotes sister kinetochore biorientation in Saccharomyces cerevisiae.;citation_volume=19;citation_year=2008;citation_pmid=18094053;citation_doi=10.1091/mbc.E07-06-0584" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=B. M. Kiburz;citation_author=D. B. Reynolds;citation_author=P. C. Megee;citation_author=A. L. Marston;citation_author=B. H. Lee;citation_title=The core centromere and Sgo1 establish a 50-kb cohesin-protected domain around centromeres during meiosis I;citation_pages=3017-3030;citation_volume=19;citation_year=2005;citation_issue=24;citation_pmid=16357219;citation_doi=10.1101/gad.1373005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. S. Kitajima;citation_author=S. A. Kawashima;citation_author=Y. Watanabe;citation_title=The conserved kinetochore protein shugoshin protects centromeric cohesion during meiosis.;citation_pages=510-517;citation_volume=427;citation_year=2004;citation_issue=6974;citation_pmid=14730319;citation_doi=10.1038/nature02312" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=M. H. Kuo;citation_author=C. D. Allis;citation_title=In vivo cross-linking and immunoprecipitation for studying dynamic Protein:DNA associations in a chromatin environment.;citation_pages=425-433;citation_volume=19;citation_year=1999;citation_issue=3;citation_pmid=10579938;citation_doi=10.1006/meth.1999.0879" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=J. Lawrimore;citation_author=P. A. Vasquez;citation_author=M. R. Falvo;citation_author=R. M. Taylor;citation_author=L. Vicci;citation_title=DNA loops generate intracentromere tension in mitosis.;citation_volume=210;citation_year=2015;citation_pmid=26283798;citation_doi=10.1083/jcb.201502046" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Lengronne;citation_author=Y. Katou;citation_author=S. Mori;citation_author=S. Yokobayashi;citation_author=G. P. Kelly;citation_title=Cohesin relocation from sites of chromosomal loading to places of convergent transcription.;citation_pages=573-578;citation_volume=430;citation_year=2004;citation_issue=6999;citation_pmid=15229615;citation_doi=10.1038/nature02742" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=H. Liu;citation_author=L. Jia;citation_author=H. Yu;citation_title=Phospho-H2A and cohesin specify distinct tension-regulated Sgo1 pools at kinetochores and inner centromeres.;citation_pages=1927-1933;citation_volume=23;citation_year=2013;citation_pmid=24055156;citation_doi=10.1016/j.cub.2013.07.078" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=H. Liu;citation_author=Q. Qu;citation_author=R. Warrington;citation_author=A. Rice;citation_author=N. Cheng;citation_title=Mitotic Transcription Installs Sgo1 at Centromeres to Coordinate Chromosome Segregation.;citation_pages=426-436;citation_volume=59;citation_year=2015;citation_pmid=26190260;citation_doi=10.1016/j.molcel.2015.06.018" />
<meta name="citation_reference" content="citation_journal_title=Nat. Cell Biol.;citation_author=H. Liu;citation_author=S. Rankin;citation_author=H. Yu;citation_title=Phosphorylation-enabled binding of SGO1–PP2A to cohesin protects sororin and centromeric cohesion during mitosis.;citation_pages=40-49;citation_volume=15;citation_year=2013;citation_pmid=23242214;citation_doi=10.1038/ncb2637" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Luo;citation_author=X. Deng;citation_author=C. Buehl;citation_author=X. Xu;citation_author=M. H. Kuo;citation_title=Identification of Tension Sensing Motif of Histone H3 in Saccharomyces cerevisiae and Its Regulation by Histone Modifying Enzymes.;citation_volume=204;citation_year=2016;citation_pmid=27672091;citation_doi=10.1534/genetics.116.192443" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=J. Luo;citation_author=X. Xu;citation_author=H. Hall;citation_author=E. M. Hyland;citation_author=J. D. Boeke;citation_title=Histone h3 exerts a key function in mitotic checkpoint control.;citation_volume=30;citation_year=2010;citation_pmid=19917722;citation_doi=10.1128/MCB.00980-09" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=A. L. Marston;citation_title=Shugoshins: tension-sensitive pericentromeric adaptors safeguarding chromosome segregation.;citation_volume=35;citation_year=2015;citation_pmid=25452306;citation_doi=10.1128/MCB.01176-14" />
<meta name="citation_reference" content="citation_journal_title=FEBS Lett.;citation_author=G. D. Mehta;citation_author=R. Kumar;citation_author=S. Srivastava;citation_author=S. K. Ghosh;citation_title=Cohesin: functions beyond sister chromatid cohesion.;citation_pages=2299-2312;citation_volume=587;citation_year=2013;citation_pmid=23831059;citation_doi=10.1016/j.febslet.2013.06.035" />
<meta name="citation_reference" content="Mishra, P. K., K. S. Thapa, P. Chen, S. Wang, T. R. Hazbun et al., 2017 Budding yeast CENP-A(Cse4) interacts with the N-terminus of Sgo1 and regulates its association with centromeric chromatin. Cell Cycle: 0. 10.1080/15384101.2017.1380129" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=K. Nasmyth;citation_title=Cohesin: a catenase with separate entry and exit gates?;citation_pages=1170-1177;citation_volume=13;citation_year=2011;citation_issue=10;citation_pmid=21968990;citation_doi=10.1038/ncb2349" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=O. O. Nerusheva;citation_author=S. Galander;citation_author=J. Fernius;citation_author=D. Kelly;citation_author=A. L. Marston;citation_title=Tension-dependent removal of pericentromeric shugoshin is an indicator of sister chromosome biorientation;citation_pages=1291-1309;citation_volume=28;citation_year=2014;citation_issue=12;citation_pmid=24939933;citation_doi=10.1101/gad.240291.114" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=M. E. Petracek;citation_author=M. S. Longtine;citation_title=PCR-based engineering of yeast genome.;citation_pages=445-469;citation_volume=350;citation_year=2002;citation_pmid=12073329;citation_doi=10.1016/S0076-6879(02)50978-2" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=B. A. Pinsky;citation_author=S. Biggins;citation_title=The spindle checkpoint: tension versus attachment.;citation_pages=486-493;citation_volume=15;citation_year=2005;citation_issue=9;citation_pmid=16084093;citation_doi=10.1016/j.tcb.2005.07.005" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. M. Ricke;citation_author=J. H. van Ree;citation_author=J. M. van Deursen;citation_title=Whole chromosome instability and cancer: a complex relationship.;citation_pages=457-466;citation_volume=24;citation_year=2008;citation_issue=9;citation_pmid=18675487;citation_doi=10.1016/j.tig.2008.07.002" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=E. D. Salmon;citation_author=K. Bloom;citation_title=Tension sensors reveal how the kinetochore shares its load.;citation_pages=1600216;citation_volume=39;citation_year=2017;citation_doi=10.1002/bies.201600216" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=F. Sherman;citation_title=Getting started with yeast.;citation_pages=3-21;citation_volume=194;citation_year=1991;citation_pmid=2005794;citation_doi=10.1016/S1046-2023(05)80158-6" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=A. D. Stephens;citation_author=J. Haase;citation_author=L. Vicci;citation_author=R. M. Taylor;citation_author=K. Bloom;citation_title=Cohesin, condensin, and the intramolecular centromere loop together generate the mitotic chromatin spring.;citation_volume=193;citation_year=2011;citation_pmid=21708976;citation_doi=10.1083/jcb.201103138" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=B. J. Venters;citation_author=S. Wachi;citation_author=T. N. Mavrich;citation_author=B. E. Andersen;citation_author=P. Jena;citation_title=A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces.;citation_pages=480-492;citation_volume=41;citation_year=2011;citation_issue=4;citation_pmid=21329885;citation_doi=10.1016/j.molcel.2011.01.015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. Vernarecci;citation_author=P. Ornaghi;citation_author=A. Bagu;citation_author=E. Cundari;citation_author=P. Ballario;citation_title=Gcn5p plays an important role in centromere kinetochore function in budding yeast.;citation_volume=28;citation_year=2008;citation_pmid=18039853;citation_doi=10.1128/MCB.01366-07" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=K. F. Verzijlbergen;citation_author=O. O. Nerusheva;citation_author=D. Kelly;citation_author=A. Kerr;citation_author=D. Clift;citation_title=Shugoshin biases chromosomes for biorientation through condensin recruitment to the pericentromere.;citation_pages=e01374;citation_volume=3;citation_year=2014;citation_pmid=24497542;citation_doi=10.7554/eLife.01374" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=C. L. White;citation_author=R. K. Suto;citation_author=K. Luger;citation_title=Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions;citation_pages=5207-5218;citation_volume=20;citation_year=2001;citation_issue=18;citation_pmid=11566884;citation_doi=10.1093/emboj/20.18.5207" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Yamagishi;citation_author=T. Sakuno;citation_author=M. Shimura;citation_author=Y. Watanabe;citation_title=Heterochromatin links to centromeric protection by recruiting shugoshin.;citation_pages=251-255;citation_volume=455;citation_year=2008;citation_issue=7210;citation_pmid=18716626;citation_doi=10.1038/nature07217" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. Yeh;citation_author=J. Haase;citation_author=L. V. Paliulis;citation_author=A. Joglekar;citation_author=L. Bond;citation_title=Pericentric chromatin is organized into an intramolecular loop in mitosis.;citation_pages=81-90;citation_volume=18;citation_year=2008;citation_issue=2;citation_pmid=18211850;citation_doi=10.1016/j.cub.2007.12.019" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="The spindle assembly checkpoint (SAC) is key to faithful segregation of chromosomes. One requirement that satisfies SAC is appropriate tension between sister chromatids at the metaphase-anaphase juncture. Proper tension generated by poleward pulling of mitotic spindles signals biorientation of the underlying chromosome. In the budding yeast, the tension status is monitored by the conserved Shugoshin protein, [Sgo1p][1], and the tension sensing motif (TSM) of histone H3. ChIP-seq reveals a unique TSM-dependent, tripartite domain of [Sgo1p][1] in each mitotic chromosome. This domain consists of one centromeric and two flanking peaks 3 – 4 kb away, present exclusively in mitosis. Strikingly, this trident motif coincides with cohesin localization, but only at the centromere and the two immediate adjacent loci, despite that cohesin is enriched at numerous regions throughout mitotic chromosomes. Chromosome conformation capture assays reveal apparent looping at the centromeric and pericentric regions. The TSM-[Sgo1p][1]-cohesin triad is therefore at the center stage of higher-ordered chromatin architecture for error-free segregation.

 [1]: http://www.yeastgenome.org/locus/S000005599/overview" />
<meta name="og-title" property="og:title" content="Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/2901" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The spindle assembly checkpoint (SAC) is key to faithful segregation of chromosomes. One requirement that satisfies SAC is appropriate tension between sister chromatids at the metaphase-anaphase juncture. Proper tension generated by poleward pulling of mitotic spindles signals biorientation of the underlying chromosome. In the budding yeast, the tension status is monitored by the conserved Shugoshin protein, [Sgo1p][1], and the tension sensing motif (TSM) of histone H3. ChIP-seq reveals a unique TSM-dependent, tripartite domain of [Sgo1p][1] in each mitotic chromosome. This domain consists of one centromeric and two flanking peaks 3 – 4 kb away, present exclusively in mitosis. Strikingly, this trident motif coincides with cohesin localization, but only at the centromere and the two immediate adjacent loci, despite that cohesin is enriched at numerous regions throughout mitotic chromosomes. Chromosome conformation capture assays reveal apparent looping at the centromeric and pericentric regions. The TSM-[Sgo1p][1]-cohesin triad is therefore at the center stage of higher-ordered chromatin architecture for error-free segregation.

 [1]: http://www.yeastgenome.org/locus/S000005599/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
    <title>Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__Ta-UXINC0YONAj3hv7M-Srfc-_B4Wn0KATTOQGieTps__agTSMicuoC50kRvu-VcTNoPubLJgflHIUuLq_Jt5dMY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"M9TusAlcSWKMoLDdpTHbERkM-MSP8x2d0r5v8Zy0Wi8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490737","apath":"\/ggg\/8\/9\/2901.atom","pisa":"ggg;8\/9\/2901","ac":{"\/ggg\/8\/9\/2901.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/2901.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2901"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2901"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/2901":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jW9ldS6+1n8K4ogjTh8lAM1SAxR3WLsTwpmyzHjWFOyYYiwOnjQvaFjUBi6\\\\\\\/GQbsgqgZvCHSGcpU9Ga02LaN27Xqyheh32YjdbVQyJ1vCFjKCACc\\\\\\\/eLuY2TzZex1w\\\\\\\/9vq4Y\\\\\\\/i1wXNd1MoqeU5n85QUr96CliQ+u4\\\\\\\/o7CudyXwRD+bqz7wWZBMpRd6DH8ERlEoHpHZlhbnDD1Ty3R18ACaifSS8B4HbKVVw3v86IfT0YxvgilKyds2u8hXXNB1mMVyHyriLh4zK7r1EJ+iZtYy3giPVg2YNasffx5q9sOY\\\\\\\/8Iuvw31xc2W\\\\\\\/3kasUQEF\\\\\\\/mye1+ENHlodpYms43ur3a1qV3w4TKK6XwWchvleke5rKKkEsbZt54jq1PHZOqwPgbqWutVvd7b8yH1fa\\\\\\\/E01sL0\\\\\\\/Sw7jf35zgVB2eamdqfWl6sII2qDi66xAsCOWc+ar\\\\\\\/Ts8h83JdOjzEnRCHAmqPSJTHyvLTjXkLKmiL3XEudF0XRfi1qPmj076C6yF78Vh41UWkNUgSgxFNxMuqIheYJWfuPSUeNniqQ63LXjLZfDaBeFdyj9VtO9+n9hJHDhRHOKLCwC1Iva72i38oQbS+AwNt3AAK6ItBjc11iuLLtUj58oRuKzJKlcAYpG3EVPqBUydESbjrpCoq7J0iYy8KJ6cM1CsNyOBhR1HzpiTfQYyguardUiRpU+VB0VMW1kIj8zyZ7Zmnkv8dYr8LahdBb8DV5lzFsqH60YA50v7xtownBIxXk3fF1g5QLJEC6s2dL2cn7uUaImhJBSpzBf\\\\\\\/Dnx3o7ZaSjVJBwonOxOYiDFGGBxp7dSzQa\\\\\\\/j9ayVa\\\\\\\/FJeFK2S9pR3\\\\\\\/7J91bHR0wRSs4aZ5rarxS0pdypoKUzog0flznCiNcgIVoEVvZOzDir2q4CnByv2ZEB+mJyb0C8RmY5w01xaokJGPi60BbY2rJWZva3NyeaqLZZFfrykCOJ+5uTvPuaV1nsfCb6mS6YdrRQBUSS40n2txLeHm2B0es7HePvmFkRNImnyXSg+FmU\\\\\\\/JgOmGyKrugbmGtm1qtycELZXBCbbpI4u8M5U5M0MQa2TNnr2nWj+gYERNgCRD\\\\\\\/xSC7le\\\\\\\/ZGbFh6hQGIwFcjsjGLqnpVHmk8UXoXVaTZoqT4SfSh7iqiKYnJ64PJ6S5X7ZEPNAdeJVHYnUdnyjq7L1NsmJVSbGnx29GGKuPw8GWx7gHfGupCnmaHR5oqA57UL+Eg7l3k68N6TdUWOInmRv6z93kr8HbOnsjP7j6cyUc3Kuzca\\\\\\\/JfEuB8N33YOVg5g\\\u0022,\\\u0022iv\\\u0022:\\\u00226Jy79coQL4bexbT4xKwMpA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022073c9d8b60195d1f4d186cbf5c1c4951ae9124749d262f51d474a5a91c835fc0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022M9DRAcOn\\\\\\\/tUPft\\\\\\\/AB8PH5tPvaCMiLNrmBXyP4wzItcj059EzFDQlKJR9eVsST1f\\\\\\\/zVXjgX62lDkUzSZXD7W\\\\\\\/D++dMYM35vLUNn8sKuc2TL\\\\\\\/Jp0fp8GvLjAaidicvahDIhpqUOeoCq7OOOqKe6tSwMiZmq73CZqOo3M0M4TE+p2zcYwCOSJTHoqmK21AG++NHR842hASNHxUm8xMmlvjAZ5\\\\\\\/gwUS7MhaanEXuAYe8Qd7DzWdrlzeiCCSDOf6S2w8HDjvzC9IIbZQADUe5FhB0RkUvxWqUhZGoLZ2gLmVjPhRTRSiWkS7enZsKtVwhIVUprLdmJzahYsQEF36q96PNTetIFuA+ZJ9Rl+oVQIiba\\\\\\\/8i3ycDK9QB0\\\\\\\/XgQIntsL61XlKy8Fp0mMnp5M8PyVXmZ4CXqopshM5T5r7pXHy5Al1TqaU9UK2DoZnpAt4suWZBOaNwRi9A1lXUxDsSU5mZNUi3k\\\\\\\/LbAbiCJukH+WjqHK8A3NO9Qiw7sBEl61nuI4taomddCK8xzs8fBbkElUdiUbi7\\\\\\\/J85X118KSN+R4NcIJB64I3eIQmJHoQBkCRnVpH6ZROkejgxo1RjrlH9G2hvseZMRb2bV4yGw3JT\\\\\\\/bMOVUuc9JIeMPtMM9W59lhfIRMeIJDrHrK+vAhICzHB8QrVvylWKRZ5hr45rCKQgXFwF8ivGsl6xn+GKyI5c\\\\\\\/GLtwVYsiVlKtQ9lofRTgjSXFjJ6kyvabja\\\\\\\/DiaZM6BWjP1Af0m2ywOpqu8DG439vxb9ru6opSpTHY4UTIv08JFxe5GtOJHWzbLIfumbTMKMGzvBhRYMT+GhY59Jp2jMq9Ww+Pt62t7Ahigy5vFhIl8j9csuWGeBYnW3ajP4YJd6jbRh3AuNtxPCHXaGXEn3WCOaZo1hx8PsL\\\\\\\/q4qGMlxYlYLtDB4korwz19oejuvfKXKHU++u4ARy2zfYVguyuYIktEtWlG8K1yg2uj6T7NzwR49\\\\\\\/6f\\\\\\\/8tZv96\\\\\\\/6CJMrOuNGi2Sa2J33GoZybMR4ImO2FYwFN9RQarVxoR9BhghMB8oYKghv3dAFyfNNOsNUfyyQwnN3aUFIw3otCa8trMV6MEfVvsnDVLMRHrmWl9Eo9PIHuI4x2cJRvWJ1aRSjruCh5Azc13cwrz\\\\\\\/MCzkmoHC+ZibhYrB3lTocrsjX36KhRPJoLO0zQtph5BUo9B1NX3tq+jxv3Bc02Av6BX18YbwWDmn0ZcB15DFnzwb5WarYDozzCEcMqotTEYpJy+0z8Zb7cBzppmG8ox7k\\\\\\\/pYDjEE2mXklTg6ZRUJu2zYjmOAThnB58FbVJCbReb4Wwyfr6LZW0T7gW5bQyUFCVoQvu4XRzf7ivJMtyCKag57uiyKWCN7o\\\\\\\/lTbZb5wMhZyu89K5sltPupgyYFhKxjmAjXL5KpGbMDvbSNZqUO\\\\\\\/WOethGL3Ua3DLmkDe6isASYmYxX2JU9RgzYBUtgK9dg5oKrgCdptj40UmlvLpZsR9Dc8SY1aCSwnaL\\\\\\\/Kk8Ti7INGWhAn9ylzU8vmSw2d8OO4bn7kbntTV\\\\\\\/Bqi3yHcbwU0zGHSwA0Fn0hO2LVYGF436aZyxm6XsSbrWJDQ1BYgbnyv1C0YQo7U8WZOBBsCzUew\\\\\\\/St90BUXFvjdVfFpX1Lw\\\\\\\/vocFIRRKK9\\\\\\\/LVlljreMhOXQ9ErLLUNbew4jJyUTNJIMUOPKEYLNVRrOXzrL9BHTyyYiwOLI5xx\\\\\\\/8rTCbCezJNxES3GaoAGOLpyo4Zds5dEE\\\\\\\/7zdKQlwbE63lCUNnip\\\\\\\/q7lS2NJYvpSbeeHbQbqWqHEpJGIi+XWbDYsae8bFercWzMX6ocyai8hgURgArOnxLYJorwpShGt8gl6gFiV5DfUQ39p6aKOGidAmniBKiObEX7lKH9X405biPRKk+f3EEHZN4LM67bqc\\\\\\\/dZRnxGq9y3\\\\\\\/NXSMOUzm9JwIXaFvkr8MIErAkwXTYozBIl6WT5un3nUUVpUp5X3q1N1YnhMmWW17RnlrhxvMgKWCKcq7xGTNm2wpA\\\\\\\/eRXrHa8DekFGetqocT+7ROOcFypuX64aDgJY+7i5AW0EPACr\\\\\\\/EEpHwbcYWk5zuiZhIB4OV2Q7InAn8uncWiwHkokBtgH38mvtjB1kWh+jUJrB2Aiu4lq1OUjdrBCsENS0Ol4OOH5xSQaA+5NLXGEv8LnF8xEpNhX3eMJEhFYUawQ+Tonz495CTz4dmAOYSANVsUea52bxjpfD6LVoZdpLnqaqbjBll72Ibp3DNxmAbyW\\\\\\\/cTMIPyd1M8zzxsF45AmGsjrdJe653cH\\\\\\\/RViWFtSO\\\\\\\/KdEA1jd1mjOsF9ncBim6kp6vMPTVPOTQ94Mc+0psuNaDH8QHc6MVbcRMlvBklWStzCwypXKHcFFbfOuj0UqAB1IgWwsqpjC3hULt9AXq3+0jG+wbDbjLEW0YFaHOK0O696HSz2VtsxnElNuxBawn4f4F3jcIwMfEievEkQ0qdvo6DVxnimULsDamgYOaHRlwwOeBEMgPtpJjAJafd019+7KuXwSCXIPbwgm\\\\\\\/fnKBWzWdsGzRlBVCMFrrL+iZbSSwiRdEWuUyuHg\\\\\\\/3vLHsrtMLMRSHIm\\\\\\\/Tf3uTOzPADIyxQXmPygqeXafL1i5mubEVxG84fTC3JK98\\\\\\\/8sOi31Bh1pY\\\\\\\/MPv5QSloCiUqAgJX2FKAjSVd4AdA+5lMdbCHDS6u88f1vpH3wGnlQMnoC6C0mhl5VDfSeBw0qRpBYugvDk1eKoeaUvEsGawmUgnmq8V5TxB16Rex+0vsqWCp5aqlsXyOOWWXl043XttPiygG1Z1NcEl668Oc\\\\\\\/n93XlHAGxXZS20PqhN05gjGeP7PMYOrFonZ+hksaiNHZur\\\\\\\/z\\\\\\\/0kzvYl2RSZXuRLa8JG4R6Al4Pz8S2018csD0XGS5dljaSkIJ6jSlWISPOgYQGZsi8Xnt0rjDDu0VKBhS5iKSk7RR3U4OuQE8Es6cLeH2GgD6eeRuk7GO6pIsGKoFB1KWWVuWovFEKECBrQfsYxypmLJXASPP5Pu8EwpPhVOBEbxLw\\\\\\\/+nvrcczKdX63ZUGKK0SF1tMqtMIpAoQ78ao5ihooIMbw1BtThuzNGNl06NjCYAO6OX9RohuyJMtts6NdDg9+GrxMgWVcxV\\\\\\\/H82vZiDYfi6PdYqwQD+7xkPuYFtI6yCZO\\\\\\\/ExOPvLc+zIgP2nUu2gYSGicb5UQg2GfmOJEOIObs6E5dZUkhWPBvzpk4F04Ul\\\\\\\/ezL\\\\\\\/EJaYkd4wtKaQtrsxAqYJNWsGUoWz4Mc8vZBtmcXlYn\\\\\\\/lIY4bS5U7+2SqSeloxBzuQ+X7HQIshP0Co2EzPIdi9H4g4iY2QLmGdg\\\\\\\/UIq484TycWuGKOsms1rdS1qemYcVy6Fu6+V+E\\\\\\\/ZYjBnRBEENNPBqlRwdnqBzzsTgR9g46My8ru3QSPBhtX5Zm1uuaewNJYOTAryJKsUVw1OtBMoWoOZ9I\\\\\\\/0W2ia440TbAURdlDRqyiJsse\\\\\\\/8EsBZwCtOD2\\\\\\\/VEi6ZXpwlVgRduSFTD6cOXh9LgseVIU06uhD6OCvtS4BapXrPUWc6jCRl2lhiEHEj71BNrFEMHKbPVrT0YmbMcNzjOARa4iOvBnlp3FCft2IlzHZ8KTQz3Z9aHHFgwwISIaTLax8ONmZdcMwXkUS+rvGzEzu6A4k1k33vVsDeoLxfU9Xxwy\\\\\\\/R4FnmX0OxCoF4005paH2CYFdyFNFc9+91U5sepxEjgDldGr9\\\\\\\/25I1Sy\\\\\\\/0\\\\\\\/ExefAd4sHGfN4Dgh24+iv9dSv3nXFlgs35p+uWNkhTzQzzoTBUsDRCy9aJXmGfbFwKrZoN\\\\\\\/pxwTdgGbbTXtp\\\\\\\/eLupIkm3Z2kojLVGUFXdfE8Zs0weMzxvQnEB6wIu0rLOF62P6Lhff\\\\\\\/98eQzgyLDm1dRDSNNVFW77bDyRR4xAX90PQx4LZMkZAleQmO2AoVbUtzulolpCIw2ylMORz6oZFPw9\\\\\\\/rjcE3Yn6JHq2nPFmdgjYwB2HXnL8Q0WheEp4pykTonwJtiOIWIJyXRnzxMgxJff3HpqYd7gtg7FIw9Iqc8qLpRy4DMSc06TUDs+BdVDirqIQstX1iLbrK6hcMe2ARdgcfLJWTjda8YWGUcGvB2sLdnD0zRwCe+M0sSLtVL0dto1SRQI6vvTgTzmewcCRpdD52EpJAdSopUiVqST11kP\\\\\\\/Z4hME3CFciav2fIzfAFoMNPdPWMa7Nlb1MUZ\\\\\\\/WcST4TTuRWJYxpqzD7RxZ0p86XVtL34OIMA+\\\\\\\/RqARtX63DSsRfKu0gVrK3fTyblnvO2cH4V9GnGtmz34moDsJFZf8j2jvU2xDY1DzK+lyTVvON94FXDFsJQ7NauYXGCpS4yOttmg4cEou2CsErZtsipOPMDCI+HyVdBciaARBAMfQXXSYshQh6kMNxHxC7RqQRdhvVmGb6fdqyMWAeJ\\\\\\\/tE\\\\\\\/\\\\\\\/TyuFb9wKkm16EZ8pKNuJIzb6Od4DKUM9Uj4316iaZVpgGRBwIntUMtiqhed2d5cvLZKOOWMIXAblBhXPHAbzWsmpVcCwvjdCebhUShDwTNclotRWkxXUn5WeWz40KzCFOMPaW6b0oxDrjtfgPS6sRFLJmFrsy7Z9VxE3qG62fdvnTpMM1W0K788f\\\\\\\/iyobEEj8WP25ebd8050Xjluai0NnodOfFMTfAqW3ZgCT8E4a7h\\\\\\\/k6gcsLOgz\\\\\\\/cxYNDvHP45MEdGuAtn+dVE7K9fD9ENnrIg210FtCC7XEBOWQ63fVH4NJa++r29NpKKt0GPmruv8eWysHHl1XtFmIVFBsmmqBa70eSj6Hn55vIl03oOxwl7C5+QVAUv2bjkSI+Y4O59vDLqI\\\\\\\/Zdje92BRftp1FHEZR1\\\\\\\/Jp+5FL+dUqWK\\\\\\\/Mp2M3VBmPPfchc7vKEn\\\\\\\/osW1hKmSMO\\\\\\\/eoSjQJOs+9Fz8Vo\\\\\\\/N\\\\\\\/cMYuxJ+810EooX571vm53DXZEEAtYUy+fF9WkXFqRdxb9VSCYpkARlIQzkHX7xxt03kRZi4Fl9bvVZ5IZHqSA5HOuuF3Uo7yIVB8YDr4zIq\\\\\\\/VohAHdtsLH46BbK2untggTWyHqAvGtncRF7Oetcw\\\\\\\/S7ljf4eU4LmnDRgmA3SFfycC1Frdr7zatkUCAMGTGnrbwgBEMOKj\\\\\\\/OEDw4iKGUdXCNL25y98MuHewu0WwcX4flrrDMkVP504mSxb0pZLJcfhkGI0jQRa4SQe\\\\\\\/nQ8Q1EdxDE0lACTRYBCS7p9kV4QKA\\\\\\\/CRpnaFeyXxyCqMbk5RBnhujTX+w4tUgrYwWZ3r619XZUzO7KLESapV7mABPBgbgaRpUWvwRubiu1Tpw+u7ccmJdiE1n3UL5iN+kuC5MGvGKGE5zAJkJ8nVVBDGiGt9hUmyEb31guBWWDJF6QOCSeY\\\\\\\/rxDpVRMXirv2ne1eEY9QbapF6sJNQcXtuJgPY+TFO0iNPHQMM4SVP0fO9xujxNiuA+to9BnOWEJY4Fsc8sUl\\\\\\\/dVUxV09gppZ+VXQUfaD5DeuqdckxKWOku6xtd2yw7Ve2MzbM6b1JV4de1vKsFIjzlT17OAxxqYEmK5plB0HGlITApvZD\\\\\\\/LY5d4oxkg2zC9fuDmWr5oS2zzUu1bR7zFgme4Dq5W0Ip46JaCmZKucS\\\\\\\/Wy+xE7cGzO0tM6f1RYcXjto8YxHPRXQnjI8jPjqWGO6uSAJ9iL3Jt2IAX0M2n0E+gY7fRon52z2x8L0KUWc9GbI1HvtTotaWRQXbI3WD6VFV7pdzTJ24hIvwIARnlA3w9kiSAi2j3\\\\\\\/bqx\\\\\\\/E3UucY7owA8SgqpYOJkSMcjtM+V\\\\\\\/yaQGIgF06DmzEsuBiuK+auwNOpMsvehToWJyW2nGKt7TOBCdUZw3t\\\\\\\/ibXeNUHK3Qx4s3jL1O94AnrxbAny4mqESoQ6K1nkRDDirrGLzK+odhWlns7uLx+YBqTvjP+1lzMgjBaURv\\\\\\\/+rE5KIW6vyRBdIhGvvgCZiBx8YG1v+gEjcTFF1C\\\\\\\/axhyUPQNtAhNaG59LPRaZ0x+DIN5czo\\\\\\\/6oPylGI7RQXHNSc1CGGrsy95zhi8I7qMMXI05zCErg1TgR0h9E54ybprbTZBYqsT8aZoLsM5VzgkJgRN9ErXKRIflwjl0QnonWihNny4MSnihR5b0pngbjNk65a7C9UpfL2RXjOU5z9G\\\\\\\/FYSbeWscqHEkzqVc2GFQ8ry5G4a9TR1NCXhQqW9eNiOEv6VUS8PBIbTIIea9SICv+Du38abvFzzfMGVFU0rwDB0vO2uazi8FsfGX13CWQ\\\\\\\/p1MntXWFAXZTpO1dRV4M5N\\\\\\\/sQWl3HkdoW0sEgELzYOZI\\\\\\\/Eiv2qzkl4ZiDUnppFSGhQmCsIWQUoFtYY5dzXO+sFfyJLHk9f0Qmf8eiwqoCh9WwyI8fPDoEhrrw37MIOGLxh4xLDrBtcOVj6ey7EX8C2h2FWMS9JKCOWZ2H2i1ICQ6BURFrCCsIr+ecAMT7Vx7WXIxswyI3w\\\\\\\/rbDvb+MJTf4bBTpeGnjmuEmg5hG4+T4iRmNO\\\\\\\/T4vcSy1GsywKCrlXph60XExT+BgsWVQPP\\\\\\\/ynBmHPQqv8va3mMoTFvlxtgs7UAwWQQi978dq24zaEclbm4Z1L1nOFXuQ0SEP84z89o4qNW3TxzL5ZmJ\\\\\\\/uG6LgFz43ceThEqEcNjEJWXA1\\\\\\\/MWHiLdxMt\\\\\\\/8NMYQXGX4UCi0C\\\\\\\/AkTU+KcHsyClY9gU5wFCyRdAegvydzE2QUwpAIl02LELBr66brfTl\\\\\\\/vJ15QOIfgekZfwb6XYyNtKPi9hCCZUHlQRXBuGTc8b7VSOw2+1InZg3GcN3Wp0xjoQYu07+aB\\\\\\\/q3ba+5AZm\\\\\\\/6W4I+J+TEiwmV9GDhRg29\\\\\\\/Oy0bITBSBDyY+S+rRaX9BZrj55Wur2frk\\\\\\\/9pfn0z0DMyUQwuWBxZcg0JrtoGjZlqlEu4TfnESBTjtYG8gD6rUop+jZzfPx0ehTvAiP0ME9yja2slSlsv8mOoy9H3sh5JSwbVyx7piwqwWW1vgB02Gfz72zzVgIyh5Kr6a+H1MtmU7gqeXvF3T9M8eK9FP8cnpIxe3LU9c8tM\\\\\\\/zTFBksM0vn1NGmqZu3CjzaThHWJaW9oohjAZzsylAgbfBl+84efLJ4oFTxnhcN1Ptp4XppPRtYSsTQBCUzFtvud5\\\\\\\/awMRNUM78e\\\\\\\/lCHismr6QZ3AMIQATNTc3nRKIAf0J8U9aUkOfrYfii9oSRxKrLviMGsq1jCFrsiRk8gFefHYaRpCgYphY2BjSpGfiR240Me4GHw0JLnCc4\\\\\\\/Pm91WUACw77jU4lNKD9JLDrSswUpbnLB2B7zGHc7WOFNNbSdCJFv48ocNjVfCHP6zS9lZKaoCq3DxNiMnmPVH35eZnpGGJ+m\\\\\\\/M1vWgoMgcheJm1\\\\\\\/GRHagFiSzxa9593rjeLnpbSBvM9GPOxjKJgIIl1ppsYP7IEA\\\\\\\/7QJsk+fGJXfkqJYLb91dc3rAcBQYz6kM6wINmrWkkFyNW73U6Z2AY\\\\\\\/aANsxfuI1\\\\\\\/HWZ1YawYxlcFb\\\\\\\/bvuLYHoVQTDZHSDEZCjM5SZqxscrCTWLSdw0gmRTQ7IOw5gXiVy0swNrfk8G52hHlnfPF\\\\\\\/zOJPzqKezI3dWrwazPL0e4HpKm4qPlZJp+CioNhBWeDOCQgmhoC887h246BGdQpnJ+ZhWYpnx+97khI59pcuRnDMKvEN335s2dXmlp+3\\\\\\\/bBIUkSHlTapJf1WNH+\\\\\\\/DA3b37gyLeWTsBq2\\\\\\\/X+Niz7KJx5hdUMFVdkbdBvIBXWHAhJ9ptmpUn5NJoBHO6o4vAqcIS65wSV2E3gmt0dVYWxKX62BwsT1xK0aMwxR3zNFMXGkT7b2mRwxEaFRkYlkNsUtUTjKi2YT9gV+FmCNGH9OfB8jJv+7EzPWgUGY2FO+hy2cnqrnvlBYWLXYaa5eXR8V1u9Hb1QCCDrfof\\\\\\\/uYmwtAXtn7+vNEOps6Ntp3tFyvkyRj+e+6P\\\\\\\/BKoNVWS148K17LyPARIRMsU12lRB7oRTkfVRwFvp8BehyiH2kWg5tpxNO7cIyw6GQpRh6Zfsvg8VFIm4y+bV6797F8Tf7wxzUZtJr+FgbdnrwEdFdriSnhUEqIlrbMyJyPrEkyWSt426Iw63WanrXx7FvhyLWVTDyQaL7ddpXAXmGTh55mjZ\\\\\\\/TJkNJSf1HCAGD5WFNZ3i7pgN7xLxnfxLrSStnOAiLMEPKIT2lQuAbawW3cp8OvKQk7ZqILyU5s+OXNI+OekF49HKIre7mhZh5pND6AW8P0JKrEx6o0SOg7ewVDm+woOwghxC2aF7RikkGuZbk1ktd\\\\\\\/9eLDeobSx+sDw51sNEeNp5m5NtXCEccW8r4\\\\\\\/RVRM\\\\\\\/xse4Spt6v9R1Pt4RrLsJS1vub5+0i2308mAahUCjHeUtUwlrLE6DkDWm5uo+pS2X+fT5k5AjQng3SHXisVWvQBnQ6uujBb8VJF\\\\\\\/YebjzUs0\\\\\\\/YBnI9Wyduz17wsv\\\\\\\/pJUetn3BwwQXR1XJDmfhGwu+fqkg\\\\\\\/EBuGnUCJURptyW+gn02RtwHkUeO4fBbG2SG60vf9RZFjm4cApXQNPP9C3+Pgmy4msaZmMXBXPgf1ohWOPMi3Hqel5+M0KfhvKKtsuB2Ez1wVZTnO2MVpgC6M67MtayV5n\\\\\\\/bi3cSFZRI0BKOkNZ\\\\\\\/k5mUe6ieXouk7iXktRdRlYA77In5u6wTiezVQkt7xKQ8aez+yhIH59B5JX5LcvLxj8I6g\\\\\\\/h\\\\\\\/tBVL1PifCgr\\\\\\\/\\\\\\\/1cWFUwpXXv+zy\\\\\\\/qXS9+33twqrvaL84M4wXZUXLGBw0u5OFIRYlkleIs1dpvJM2JiUjwdUGBzW48BfP7NpjaaYmEi+Kdzf9K9ZShxHsYrVKcZAP4C0WtFavM7hKAcKh0tI9vNmV1K8BC+bAEdNmdl42ju2uPJP0hpkUY9PeeV8gNbW4v7hSQKszAcmJbZFrbTvmEIqLPKcD4jydqbfpdgCC3Iji3TXInz0sXG\\\\\\\/mueV05Y9fWkSR8mF2Li9UJD0ven6LYZ0sYyGuK8nLKXmfRJ6kYzOuuDsDACRjlTyRlmKJmQ4n8Stz3rIhvvpCgBYeDLcjOvH6cq6uyPUr+o4UpMW7aqlFbQQ3F8CzoCL+8hAmDgx\\\\\\\/EDwkrL\\\\\\\/C94FNBNrNDIo8K2I\\\\\\\/9zy2LVPiqohxoAieui76PMddFyEcK++1\\\\\\\/fg6Vo21m0jAqD1ybOaktAMFtCNNZykyjzrdpRmsIp+nDc7mlGFNQ+i2dVhBdnXatgMfPZchcA3l5\\\\\\\/de9XTf+q8qbRz1S85KwV81FwjO9Jurm5YW5pzcPoLfVWsu5g6QvknqZqvsmo7MUXpXsgoqaRDDRYF4b8dsVydHyc18LnOdZ1uQX56FKpaVnrP1WRI7Rgd0z2igNtGUh8qKEh1qOohuJHTP3W70cNFBrawKyTgr4UxMICYrb2WJ7b+OzaD2pmBsWmf680+61wAGl1GmT9HnN5ToVZ5rBo9qeUx\\\\\\\/PL7t5soeOze9ZEYiLQIVUizmcs7r8uCL8q6T97BcuOsCqD7u7zxSPvAp95oGYil8EFI9C0G0q4\\\\\\\/v+de9U2VWNm\\\\\\\/UxYo8v3VWAgRIFWR+6m2fhS1kljFQPbEsyCgAlDEuswH5Pjoc6wQ\\\\\\\/GGScvgOrWB7P4B7VPTfUe\\\\\\\/1vRQKVPnHXstVyBVFJHkhG91aSfsGOYacQQsR98trsLvFFOq\\\\\\\/+ghfbf\\\\\\\/y\\\\\\\/tVNMOBD4DCCv5NzpeoOL5iVpAqhA8yC7vy8rXTbY9RFSjSLsrYX103MkGHj52h7BkptNj+r+fQr1Svl6bQ2zYtZJ3XZVPRmisOpA5zQhlu\\\\\\\/vjrWFKFMdCCvGMfGi5DZNdbngoiWZsV6LF2CV3jESEDBwJ9laqdIk7z9W0ejO3oTPO6oqEybsy9XtYGdSadGTIM8LCAMgzla9I3aaMq3i1MWzg3+Rbh3+nP7FpU82zMypWXU+Mtc5bGXlDtLhUT4o5qtb6Kak\\\\\\\/E4wyA3h3w6YaZZGMGqSLGqsNsBxf73z5mHaIEPKS52pXkGhdRXmztF53LHr0qUkRcjwiRmQvbhu4SSxMTLQ3VuOR8DnM4R+71PBr7XlvJgHZiSxHrnnjbnGfc1TM4Kr49+ttVlf\\\\\\\/cQKa3i\\\\\\\/rFPYAHe3t0jgA1HR7W3BxCGJSDMPP5LE0Ui5IZQ0Ib+BqoPt1D1oQRcQKURRYo4Y8NhOyNG9z5WZtJ+zTlepcP+TJmZInUpvc0xGcndummslhJiD3nFl8hfYTM0KFuXG\\\\\\\/64wlqss1GF+HAm+sL9DPRowhHBBEEHTkED3HIIjkz9hp0DyoLc+BTkXffpDUl+ml7Fe+1ENZev9wIM03wxqdE2BP1Cd+yjCDvFO3IwStqedLLbxgKYa4wjtYVQCflunnE5Hb6Azae82yIkgOl7Crpjd+PiKAZhnDa8krQ3TgSuqaJMAdM8bnP1jlR6QsHyTgJTWLlKSuEBfKvTOxGFnuBEhd2ySVgQOm3GHtYj4M5eQeOX088N1rbiTLGzYDTem\\\\\\\/F+OD\\\\\\\/RdtLfXv\\\\\\\/I17FXGjIYx3qmtRXI8sI1PZ6AlrFix0T59mWqYW9IaxgNgQujuwoOMxn+tQx69siw6qSCVs+OwztrjG8z+rFEqQj6DV6Di7UzD8TbRWzpYZaTkq1Y3cjoFCPXPU8GfIigeTEhV2PMvKmaPA4pD1kbkXH6dNHNqqNMhW6yPRX7Y+CJZ2VbD+SKq4JAU9oNUVS816FHKB5wJOUjMU7H8zb7fS2QlZdMQ\\\\\\\/UCawfbqdTDgS4axODiGs9V22d5\\\\\\\/XJY2qChJK0tSs3xRCslWLAlBjWrTUbLCPZsLKfTjcaXNiEVcBNq6Qmvs\\\\\\\/RJjTsHdtFlE5\\\\\\\/zZTqcsAeKYlDq\\\\\\\/W\\\\\\\/Luo9u0HG+5Q4SRpBDTaXQ2syhMFh0pq+U93I0gO7eO4spLfoKCCKTjrdg3VyX4c9ktJO3x8RiBMuM0\\\\\\\/TG+x7wh7GDxv\\\\\\\/W9+t5weLbkEaFk3xNkLya\\\\\\\/Q2dj6CjLUNiMl0TFeaBlI6xyGtDXQQ8\\\\\\\/c98xl9g0LEvzanh+4aXU1Df8AvuIzhyzzLAQ3q5H5s7AQ29DiUQ9IgeuqqlpyRZXabtnwZ11Bxd59gAQGZrq8Cg8T9DaimROrVtBQ0pL0E67o\\\\\\\/GjuFoX8DFuyv1oPTdbBawCpyMkdBxILZM8akZ\\\\\\\/u1ig83lTRF8jViA93dfje8AOdPqI6c+g9nokz1UV\\\\\\\/Lg9DWmQ6T1gxcbOOlQmir8UHDd7G9QOJs4c5bSKFb+ri8HvGcydQevQRAzaiEn7GL4jikG3gvKrugwqMIpnvZyHRM3ni68e64NlHpFXfaJHLT2JsDmk33MnCvZ0W92lZfRWk0QSJ80mZK518FBcTgKjV\\\\\\\/lFMizy0UnYZhxqUQ4S5wKhfSGqOB+tsI7dLKcIE4iUGZt9uIsoYqpGNuQJW21CKjpEGN3wIr8hsqWH6XmiZcA5D9IHlJy+gCWCiKlk7ksR5bCOhvCAoVmYQznoroOZiAwk+Kx9QzCgQAbsDBxzj+kzahm5i2suVr+ROxaX4DMg7BhO3G1xmIXj\\\\\\\/a4pZPxLc6XMzLfzAW20TBHhMY0quq8ny\\\\\\\/4S43Nt2x6ypyYDA4sva6mwPUDRvLhgGtBgIvc2mByaOaZW4iqCdsxJ+qxrYAmvsO6+mEYrbEs7zdxa2eICsbY02M8UQifVmAGjw7u9rhOyOwXAXX03DdBO6Vfh102hGjxl1vmI4MBKkJ\\\\\\\/Ir2EkaD7jT4AlIT8kPXa75uFvWbObI8afkXw6n+K+1XP\\\\\\\/fN\\\\\\\/K9vFBt3SkIAaVJIGeys\\\\\\\/+f\\\\\\\/pCaO01TVX\\\\\\\/OP+HtJMRvidiRxXj72Wg8QEuPDjXbTh89wZYbuCV1P468tb1DTLOtPF3eOVp5Db4Mcn18JhxP5BrksrV4seVSZ+jKZadXYtZrtmFWdZAqPoJL\\\\\\\/1QQnpFWgtVFlXA4EUwujiJAGR6HF+UBWDXxvH7UnUm2tgXRuGXOCiyAOF6ZoP+obvadJQAVsWdZXbmZTHhcMqyjLLY682ClsqJ4yJYHsOiV4jZxAhzJlnhRV+\\\\\\\/K0ZuD\\\\\\\/8fGKfCYWMyQuH24v5L52JVM0yXCxBo24htyeaHA\\\\\\\/IuSy6wYYHma9dKYKcBZdokLlaLQUTL+GXAkQjLNxkLGeRSCOGrm9uVeK3WoSYybwCqs9tAchTsKBsMN8uzeEMKGzvZmAzWTsoUt6TMmLlrl78xcR1lpCewDHB2Welc+sJWhsDehYkplQLbdltEn0WgMdfZJOxAdibtFts1Fa49TLwbc\\\\\\\/J3GXL5qBOmiwJg8ttSptiXPeywZfhtI3lrEc8UKQQJ6OSRkzzilJNBmfvXIhVSVAwB\\\\\\\/x7D57aQ8O1Oxgv5y\\\\\\\/SI+yc4usXvYUExghC7PzWcGQvIFXgE8c5l6tXIO6p07mVUn5iU4eNsvfyLksuoPDwEutWehRvewHAtYNG7\\\\\\\/BTGAB0kaUzGjG\\\\\\\/75XeC6QnCD8nHVUst5eDw4Yw0X33C8XyVh7EXj87FY0qRUjrRBxhQxqItpp16ZUcLHFYufWDHRfiKL3PRYoTqpvkyFMIgQ3Xq3SpQnalst69LqggjD8lb0s4sCbNJxsKmKl8MDstZd3Gh5pspCKxHfWSk8JquriAiYTMnh95pv1nXYAJ2x\\\\\\\/Hw\\\\\\\/gKEt5MepV\\\\\\\/Qv264N0yZqn6juOO2ypqAAtOWZSpdnOmGLpef2vueBTOvycQbZ1cLU4rjou2hZ5niG+g++IrQpCLJ\\\\\\\/FxLa\\\\\\\/ZCyCEbhTjCXnh+GmCNqrb5B\\\\\\\/aVHYtRbP7xem9G3g9VkdsmSR73g7t45vosz4Ma5ItAnSGUwj9ChTjTZwYff5O2AgUkIWaXiPnun1wBO4JQWiIWuXztbm1w2\\\\\\\/5\\\\\\\/lOi0+0bsKco6tzXVsPleHVJjdKWfqko92DfmxDGmFFtdg8FYFibdV1QOeYYL4DZPOrgWiM5ypaC8cOs7FB5VVy6ZkwvuiwjWkcbFQu1U8tqyRxCKHYrtKtAcNOQyeftGEGMX1yVf\\\\\\\/wujhQXRLJsetvTtHgcTKQsPFF6Bnj896fciBP7Yv9VhrsArIwsvDoqMf4FuoCcy0\\\\\\\/qG8ZhMeb9wlxUb2NTn5Ed1y8UjIjm4JHgxYiRpGroS8luQFIVAoxO8mBnRn5O5\\\\\\\/TcDCxMacK+7f6OmpzbVQBj5FK141bo9PzAwwXyy9oIV\\\\\\\/9dKd0\\\\\\\/XIWbb7VJXXyKInlr8B+EBzUqNs8No50VfOQL1tTkLA8QGc1OnNwnqzu9iy0vFDRCq6Gmn6y+iYgx0j888PeC69YFUOlHg1PwBu21mlZkMGbAgUNQryB1pyi\\\\\\\/+5gKziwrC3VPXgVE6YaiL5j\\\\\\\/XiyQMmuWHcTw3p0Kkptp9XpkmN\\\\\\\/LLv7aRDVvk2IAEfkkkd23yfBdvf6NRCE+Zdza+Us8tvb7Z08eMRylLJK6M2K0g+eUyS9Uahj5JVbKexCFZeKxfFguI3Te5+tytwPTLuvkTZ6Qq0vXEJup7r4xO2zuZ6WEKUU6S91pqxpZmhPBlXcAUp+VXauFqNMjrSpLVNTD3J+Wo4FbjlzbhxGm\\\\\\\/7JL\\\\\\\/IESECQMGHbBC+E9+r1iEqxIteVGdj4vsvwA5ytFZGgz+rDQrm0GC3NgGbvm4hfJwQoCBCeq4KKIsfKRGKGT7z1qZ+d33JG0s3mBPAfcv4fpB+JscquZAmmY6xiizRRxrMxPzNOWjCWEbb9YehyAgPBp9KuEoK8CrKMGHXTwUh2LAMuOG+f71CcTJoH6NOcd3XCbKQl25t0cNU4O\\\\\\\/9\\\\\\\/49ulw+rmtuWIEKGaX87WMkaqW9t\\\\\\\/nqUg7RE0APM0r7Ms91Zcy978JTLMF5NF7v5D8kyW3qvNIPWLh7R0k7DnE6Q24KCVQsHCWhtf04+GtKZiMKhxOYcOkOiCrPU\\\\\\\/BPPNuXCPU7LaFhLTXBq5Y\\\\\\\/1xIkCLKuXT42YurGBiWLQ32+Tr7kCRYZeM1Ay3im\\\\\\\/gsjgg9+V4OWtaC1\\\\\\\/SuEfWKD1z595iEu06LfG4kai+up4iUW\\\\\\\/cYuQ5bKzOgMaGVh\\\\\\\/iW2+dSMWsQ4q3Qd2CZG272X7I2HoJ38raBRKw6skV+hdX2DLZi9fv\\\\\\\/yRJng2xjEHTG2cumaSsb2V2p6IxqY0QILQVqbcGFFuqaWvaxIp5VP0xa45U9VcdK0HjV5PNQg0zQZ7demx6hTn3bXv3m8X1rD1Vyx7\\\\\\\/ei4WhktrOer8qbLnhFdsSHbB1Hs4eYCyP6wD5W1qdW7vHuvffaNy8+AC2\\\\\\\/GAf6h0e2Zf4QM2HExjsjcM+XdR5m1gb5xTsC\\\\\\\/BalzURt\\\\\\\/6XvR5dirghx019AVQH9ylcbMbyDFp01CN2By7ipjGzm1m8T8f1XYEZdiSzvErmEY7DNDQFJcLwCneZymxliH8bqtIKGFzgqdDiSpETpucsdXea7QSsYMUwh7cbNzjCyz54XAPBetISclE6N6Az13\\\\\\\/mOqOQfn3f6fLn1r7z1WJvuCrHpeIxsTVy35LNCwfSu+PeSdftSDRzKXkya0hjd0yawnKdE6\\\\\\\/zVPZLdmpaTrbcy4zu69Kw+SORQcLPSN0urHFehMIo4BoYa8\\\\\\\/YaDDg9pJxXHIkLTJiHumdH7\\\\\\\/JWW+GHvP5nGc50VhFSHOSIl6dy0JmnATrqliPNvPS3ymLfdAQRT9PmoHTQou23LYN1MGm8a6dtyoqcHoCg3BUh4yaCHMD5G\\\\\\\/\\\\\\\/kc662miZyUZ\\\\\\\/ANjE0oDao3tT4vIeuvjiuDgj75djSbpahMLg4REqShrgP6iFaaBMCZXHP31iBh0VhtYUm\\\\\\\/uMlsiRoyjqawIZIB9emjLWXg\\\\\\\/b\\\\\\\/ULuEJYszBOlxn5XXhq59vugQreZX7dYPbpF32ynGq0TZLvKA++QDm5dJrELqTECYMgnW91r+jrW+Awkuoi57UNGm4aaHuPm62et6gNATp2\\\\\\\/fI9VcxNMY\\\\\\\/tRRvMqnHnFuIWzg0PjbrRv29846QY5zNeB0qPQlvBY3miSQCTh8TJIyYj++vmrHc18KXOEjHqs3oEQWlKxSwfZEo7\\\\\\\/X\\\\\\\/MUwzNMFzDKvnSSrZfN802W8mhKS5MIKepvHsP+1izO3tPof5aU9qQF0dIf\\\\\\\/7DFy\\\\\\\/wB8Nw0XI8oZDM8MuAvzKlj6n2s9zLFI+Zy2yL3dXCd5KhXALOK0U0vC989DQ2MKqCpanJidZpCXKvQu+1KeO2PY\\\\\\\/trg59bS4FgomgxvmrdPc17wqNmTsd6e+PxdktrixPZyFCEHA5ssL7bRVLPZainI8RWjXAT+Ab\\\\\\\/C34oIvWg2\\\\\\\/5Z+7+fWRw7gCDimKyRn6+9LExp\\\\\\\/taEegVZQAt2f9R0D3JAJc+rRNXUOIRVs\\\\\\\/p8h5PBFqJI9mWO6S8IaFWhkVPyK699UbY37SLJ60cQzZpjEZkSGIUpjVbu0U9Xo0uaxPDUYNWuhL1HAFE3jkxTvAaAJOvjrmX1AufftodsMUNY\\\\\\\/eQmyfbA7TvH0PQGhhWwsJrHjcrA4FauWG7ZyrolOdUYg4dQJtv5vPxjRL\\\\\\\/LZZK9mgW18SIewzbtQAGvIa6fXxmbyEwoVL8ldEIqZC6RrxMxPOZfKcKw8+ymD7anvjHEVhKQS56X4bnyWraM+6N8kaHjd3Xib2YgebUwAmhKrcIpwTGH5TqOpchqSMnvthCxZkt8efkyB+k0TNXNE2PTCxguKsVBbuxiF7FnDPdGZXaBzyyvMETHjFihasjR333auEu+4zsJCe5S1+VPbYG4cemeYx4LBhtdsFIphOWlhDjWMBHV65S1XbIwnZxdWrYIENsO1Z\\\\\\\/nIJvmJ2MCUyhfvNW\\\\\\\/143ypglJf5\\\\\\\/29xsW8fA85xOLJY6uHE5wy9PqlqC8mL6pkBQtPsb42HuoMOAEzgLkPWDrl7gfg+TbRXW\\\\\\\/dmNu9L0cUnYgXc9FKXHZeu1EWkYht82919Rn1u4\\\\\\\/2XI5AdEpl0JwCQGLZEUK8VNtpN8U9uE232KZv8TZZhybmU\\\\\\\/Lq7MLJ\\\\\\\/X72xneWOc3iyEuw4wJuwgDKC8G31EBMoA0UV7o\\\\\\\/FY1ocMNRgO7ZaIO4loMa5lkRbjkWxdohYtJ9lrh5g95dYkVhU7sdGisdBeLsf1iSrR775lgae1GvPHbmCp5z678nITkONXaJdCLV6naVR6zCLW2QU+Q2ALG8TpZNpsa5jgN9k2\\\\\\\/CstesS2blVdfIR0ucJWtD1JNFu7we+YHAMcm1zI\\\\\\\/VMhhra77mIE4u2\\\\\\\/\\\\\\\/mOb8bw3OSXTJkJSfn14sQzSwuErHGSuClH1FNya6v0UGPqLSrniSNhK98Ps6BScbQn6Pe9nGLnEBhXdBnvY2YvOB+fCPpB6JFy0Fb0sa8pPoTBs\\\\\\\/dyanyvpJGI+C4KRNci7TG9kF5VmcKlxHgMkTPeTzYwhPvfCnNjitfCt6GuJL2DxDIrBuzZMNjjZopySfatLWLj482nu3svuZ2idh6WNWHgT48UjMeS84UWh1DXJc1VRFUn\\\\\\\/FaC2dXJmcrY+S4TMw81gs9H\\\\\\\/+oahGhDN3lLIeNVROJl0ZPz2ZJ1zzbaJfXhhHPgw\\\\\\\/57EPXm73GBTungJDN\\\\\\\/QyQR7HyJBG+9TZ6SmJUow+n6fTXrqaI9xhyA0S9be3Qx6WUkmnysUbZMr3+j1AtAG1BioQFPmlGQArUrOw3esC3bZZu1aW+OSUOwHWOkk6IONquKfLTWMLnMUwd7uZX0Vd1glULQBT66xMDNh2WfMQmnQyzirhcLLF6KSEmSYqU0i3v8GLj2isV7dXtXwx0oVQ\\\\\\\/RJiZf3JsTpe8X42gd7F2LqyMH9NfQtl2uQ8K7UqxTZehaRAU\\\\\\\/xaL0YYmiMsjwy98jvA0ycK7haXK\\\\\\\/gRDyXY9xcANFO0zxN4lLo0cHgabD9z9l4teW\\\\\\\/bVxfgv6\\\\\\\/dd9buzznYxcPvGJK4zpINgrfUeLjWm6VTwY6F2AKjlsupqqyq+Z9a+3+lTBx1wjN5m7VnfCiIpP9A0aBUCkm+pgZV9vZfIm2UIqb\\\\\\\/xaHGH4O9ERmZL6a9wz0aethGSKxtL4PJwCCQjm4qM4fkEheewpizr1PJjk4toAUkNzE8RtgDc0h5DrsbMmmfHyKtXkbA\\\\\\\/UM1Y5+T24Jx+YxT7wzLTTsFRSlv9FWnpi5trFlaLmpylFG6nq3v2MTChaOyasbDKQFk5vWGxrdOpSt3UxxNRngTUxy37pWQlcO1aO0J9nGSdm\\\\\\\/F8qXxxPpl9GeH4czZTbKfgNU7tV6OpoPp8q9EfLdCsQEO9WvIWHu9eOlqO2TaYH8qe+RpYrKAOHRJPOmwSU3E1ynxaqhwio3mGnX7wvyTis+26BqiATbqYBEXeO6rx1q0tTChfr+NmW7bEqJ0BIsBMUnVruM9GpuQRb73cHw23V+8l740jGhQj1LjuwN7eOVepuk+iyUUJ0vG46IUC+87b3gMeiZQPIOEtJ\\\\\\\/zX2Zv5f8K0uxb4vKkfi0fnbwU3fa+y+6yFttLIaOaYfpwLQGGIJ987ddcFHmEz8J6R1EjosH+HEFRxrM88Qir1zkogRkFoMwcU6ZmFpU81rgXvHv33akYVfrvxhJLMorksVfFmozrFqr6OtKtrRycRHZ4Z4B4P3nXFwMvuJQDnpqKgiCQ6XhLxohdRCU903BRnCpy3pMWeu19d9z45a2VxblgrheEMOLjJ5K+Q1CJquWibHFBkDO\\\\\\\/wURbcKxPOWE19A49akOUjOwZ0aCXbJqRTKrcDdt2CUNjWVrtCS\\\\\\\/aO6j+H3hrMtOpcvtzqlmXkA4JGSEPOQ3UvpbBZUu1hVVM\\\\\\\/T6LCFV0gA7\\\\\\\/ZgyJWGmbbDZIQQ4MHPnPxHZ4rbhgldAeojdWKypTdLF33YhFVhEKiBZvGIlE\\\\\\\/nmnTU+Y3fVzDxtafQmMcPJKPfE8uDzILDwunXEJFcQRnoGC76pnqkVkffeK88JNyOvTnZUqbWJr0MIL2EGrCEIhe7w5kQvVyLAaeMFq73C9qmLGj4Fm9dbymqg7fiXTmuC2iPKJ\\\\\\\/Wqtt+sWjZrHlCd612CndYcpr46shc3OJ012xcbnkhFwEIG\\\\\\\/9+ukWSoEf38IPaFkRaqkunri8eyJjVfoYS4yGMosTwtODB23YKkiMAF\\\\\\\/oVq1wr4PjDgsyB7wvS4o+FU+2GUCY0mEVF2xLdyfLyUHmjqug8zr9Vb352FpWcGK7Qr0t9pYGZU4eB+Q0Ph6a70uPiVlHWTqoMaJv3kcDly7vy7JJJEdBntw\\\\\\\/WOqTW7BCK6fQ3qAFPIviYaNNaiaifE1UUoG6v3psqCsTyxJPlYa2i94W6whX1C0lkJoSzsdrthykYb5Bn81FjmcF+J1s4Ft0s27X+TkIQwhWSgDQdATTJo\\\\\\\/5+YSlC73iijjCow8MpA\\\\\\\/u3RWGua5U0\\\\\\\/mjBkP588kjMJB\\\\\\\/5OaxkY5oo81+0MA\\\\\\\/r8x3ZvX0E+HVq71nuXvJTiDUknUtlH54tzt7N8rZSMLq3F9gTGaq\\\\\\\/X20k57H1Uy\\\\\\\/nVV6gd0Wz5K8ZY0E3WZtxhi9argsRNwixsoGfO\\\\\\\/VUlD81GjxTdtf9yl0iygDwUjRpbkNSKBuAj85dNncmQLyUAMYyMjmHRKwG0p4enVmSQ6QiLLw\\\\\\\/gDBzCEgyOKWtbEL5L\\\\\\\/JoFgAU8rTChPnHJsLzuptPdakk1FSg7Axyz1qGujJQ\\\\\\\/62pUf5nSL\\\\\\\/Nk2uSt98y\\\\\\\/x7d7YnBecR5dvVKWKg61k3oYyLeIMpLsRQj8w27v2Q7ilgfFDBB7W2ihX\\\\\\\/wJL6na5q0kYyZENhUE\\\\\\\/5mJiq+gl6L\\\\\\\/9gH6zMccVvwxH2QSDdZ0qeZ3FVlmvbcF1meoWraqdxEmhLCa9PrYJXg9GJawq3KAoqdhCLBh\\\\\\\/HHAm6Z9R33qnNzHIz005wwjuFKB5K5yrr1yAOrPij98CEeX3OvDsgInXDG8vTdc9qV+4KkCdaCU6Sj\\\\\\\/k4sVlbZpYMIWbku4JjRVld3\\\\\\\/5T5QhfptnXX1F4F1maH9XeYVzWzm9+Za\\\\\\\/jTqEcFdRsvURH+I4WGn8dkc4yd7ibIP+o9dES7kwglqoy0NdwjgvkK9rQY3Rt8arqomjNhD7aeR1MtHkXCFPUed+WPvebjON2ix7HaNJEsMm4aw+W49VXQ05Pl29lahXxFkn8q7T7A\\\\\\\/aBnza5V9FQFviKAfL+N\\\\\\\/2aoA\\\\\\\/IThTZkLOPws28TEhhXyrNzTd7ZD\\\\\\\/Nu9TiLsscGLUkH6ViaUXovAMqwRRjtdnBJdB3XzfC2TY6wMQBkb74tviW94Z++h36lu\\\\\\\/A+84Q4MM1bIBIy1yAIjMdvw\\\\\\\/pw4t5qMMJTC2h7wH7dBaGO41gdX6BDPdZFlH6aMuwNp55YsAofoCVy7YA9oCewZZZ9cURFxAUFCfdZE4gB1KO9TgLusWsRHXXTnwzAYCYm6Im6q36Pwd3Sa+WbgvQ+goi0ppMoEuQSqdP1f5JhLMpZat58CGBanFN8qFYhGYvOGMxfqoLJlPMmH1RXMSpldOfiRGOc\\\\\\\/bvxSPkutDjSfecqjUlnJ1v1EUyetX\\\\\\\/cwBEutYvap0NNmFkkQw1FmAKQRlapfzPZeRVXkB8zMXVtYrTM9n+1WuQVQUVn8hY37Ru+88veIeuHWnU4NoOO3WxDY0bt96nbl6tp4tnjgbml\\\\\\\/zy5grUJ299s2d61zKCJRP4s+M5YACPGLhgOeYIoLj1mlYhcarxrFehQ0EyYJnA9hRNSk7Ae\\\\\\\/NEcemAqwnW38efNsXxAd0w8qL6ZOPwEtHR6dMFr6OunvQv1MTNPxw5fETXuEOn8HusWH50dZl22RrE0XFBWRqdMZQOC8V42afx3NaRWk4zSS5BRC8ZtnsnQe1d6wF+pyTnG6duDNDlUu4nGdQghcdWgVAkI1pL73Fx862+tOYMVH43PN4EgUt34V\\\\\\\/YI0yFjSOOZG1CmXEmwNXnirniZbAKxCqvoW\\\\\\\/cr1TJ8NF0s22EZ36aYmPioctZATpSu39vMPje\\\\\\\/NdBDRipJ+rUgF1x9QQnUo4YWn1rH1fGbNy3HhYuakl56owhpEwFtgK1RPNwtwqyzla40b9zWnwvQ60WmoZBLtv83ktODn0aAi5THN4j0LuwAettB6a8elr4n8OTWhjGDjpPp6Es\\\\\\\/wOwzs9SJpILFSV56xdNR6RX5zesmGJnHR4vTfA0Xtt6WjA5w5yqimXgEQdBJcdto5IAKCLQaVdmUVEBaSa+6V1H1YTpDPNpEmfocdWvFtudNaQBebQbaAC67xq+LKajN7o3rpzkzxE8aQSQvtzIslKxSnw6AbZV6VBVA5EUvamxjzVr4Rp2+8ToUUHFAtx9IzBeYJBf6NxAHHBhQEioYNtACqQDaYdHLQEaalzfYEjMQ465aX2OFMU39PmK2R8RHFNUqWPKzJ5wGyphVx0PkAqbIWQZavafpACC9WxUBf93UdhJMvqAPb14TFUfap0Zpu+2kqFmNNMXkH24Eb6a\\\\\\\/fZQ0hKtKxbf6TLp+3\\\\\\\/xA6kjKhNDjuyqlUcTGuNCotv5LIH7uFpgYN72iir6MKWFBJiv7223AjzNxmCzC94BmPxGxyvPogbz\\\\\\\/C1vEeqNhaNDZBLojVaIYwuaE0qjMAV0EdmCtpQL0A4J6dffEFTrBqqAzhQpTtXvnOxY+eZ6Rz25Q7DxcpKsRuI9fYN3b1nqEZwQcmSSPK2JJZzIKUzK682og3XnM+pJpMzDyimgmpen9zYijBGFsL1H++dHM2ZLJ4VDlryE4Ru\\\\\\\/ceQ7rReXd9iN+6CEdQMG\\\\\\\/u6u9Xlr7puOOvyHh6ISslt10lmBGb1\\\\\\\/Ato0XlMJjCEx0TN6To6l3gBGtjUlSiV\\\\\\\/0hve7bK1NX3l6Thb9kqSoi7Fn\\\\\\\/qc8soPxo6IwkbVKncVf7wxzF4ZeE1tnNE5wzHaa0sMZsoWnTcYVXZjDP\\\\\\\/LPlehkhKI2xVmhmHmME1CldG7MG+ghRx\\\\\\\/eJlXo789J9Jd2xY4WhQDAfngjh2D9LcQmzgVcDJcI3\\\\\\\/WUgIiG56ucRFm1Upm7MgvlGaDNTPvz4+OPcHhUjlrwqzXfqhmJaa+4HhFDFOlf\\\\\\\/mgCIqHSESNAZyDEtu0IPWL+taB07jmSH+Cne1EHV37qHvv7ZDwVGBk3fVnii+kGMAGcQHN3G8xD7fU5rdMmzcgxFptTix0E\\\\\\\/dDZQci5C2HtW4QeEs7TD6aiCogIpGGHjM4+7080iKh\\\\\\\/aAzaN1mCt1HqwTYL7NArfG7ezTKPCxxB3gQ2eIxEeSAbv+AEQZM4yjJHYPeFBlN4njEIETa7osBV\\\\\\\/giJS9B6IVzdctPZxABCnLDTBqXQXS+EM+nItV6BHgLftA3IHy9C7Y8Srka9gfDo1fEtn3KoXFrOOatku7JNsgBI\\\\\\\/mzCy4IWCbQK5eH26RMso1fYSYPczq5\\\\\\\/3xHXZQYKcXv9970UXUDkPDGGqHxL8FUZfoRbG1JRiGM190tGWeSXsLXwhS2f0lMA6vKc0hNvDtnAE\\\\\\\/mWHc6ICDAo+raYZKwLSeTJEUYYoXl+a5uD1hZfaKPLJclsHQANKF4ejCMmh0r5YLOdCgIiHnNUkN5GJOxgH1yHY2bmAqNhz3BhIMOKoKDljzGhBwuwEyLioss8FtXjIVHcHwWtVFUnQhE5jG30lKfw1ZBHCF4+zokQPk5CxlyURUAy5VSbGs21Y2ixAff6J32b9l++XLHfhxvA7dLiT95WbaQD45xBpRVQqqEyWLU9KmC8qtWixoiN4d+JznCSEXaf2u\\\\\\\/xPDvgr1VCKokt+EZ9kEkD4m0VBwuQGCKFKPG0y8YugfgoJrYKQJTx8kfH84cF+w9vN26t2MefNoiC2mShgmsObhOGxA2wLZITY2IseTbG6TbovP5AfSHeSulErlD+eHwRy2ozhzlbvO0Dz9O\\\\\\\/3T\\\\\\\/wkEe7TTC9MPb4BcKbqT6VbagHPZE8D7C8V6G70Vn4\\\\\\\/l2tkX1IDYO2TwhBjJVvBjQhPXKK9ygycYZAjq5hyW3ZRE2RsFErGUvfap7e3ZJoue9M44LxK2E+ewKerlVGaCHJKSq\\\\\\\/EyhRZNpGdhzZmIY0KfnmRyaCoZ4NNbR5SPQiO3LTaQFUO9YO6hvYSN2kRFZtHhYbu3lUwww6BMif8uoa12AIiV\\\\\\\/ZID6vNCRHYN8bIw01GsDZzqcx4fjeU82dcdSJVr4M0Pz0re5u8d9H2KQZJUzFx2I5C8EtFJeTUq1\\\\\\\/FSgiUVOEpJ3U2Y2ls\\\\\\\/9k0qWsvq\\\\\\\/ZoLRfRPrSB5MCTDPaa3QVJK3489k+PV7WgdPqNnunQVY6dlU+iy\\\\\\\/3mUlXZLRBunMZxw7uXuGonUIh68VBzj1M8Qzl1vzSZS4E9l23nqtCprm6XlxskalDcDUFRaPFAzxdVz7Fwd0OidErM0rDPo5wmH5LWUAWFSpAdb2spLcjTOwg8Y7Zw3CRoz95qrl8I6w\\\\\\\/vkzZv0xq6K\\\\\\\/+kP4ggOKP7V6WAGjmAw08LMnjKeTX7QtgWYUM8GZD56kMyqap1zg+5PNoBOHNEroXkbvlAuT5eYd0JhhGpE60gK87Np5+cBZLsLFdYPxoFUc6dis5zu8c+M7+\\\\\\\/\\\\\\\/xjOrW+4LCxHr3Kc+I5WdclqfNX4bQ8DlZO8ymJ9qXYMdarhgP+yCDyCLdPNpPXoLBS0UQOkp7FDoEml3GigwSZbN2RJGaDHtGi2kg4JhxcOohWWLfPWGRp3yWHo73zohyCI7G5Fnx5LRDfYHLRnqRVvWU0NJFdpOwF1kaxv2KkxEUgMDAJbW\\\\\\\/Z8J4J0XFFnNou\\\\\\\/fFiB6WFBQD1om9dLOGWXoBPNDFhe9PQmZFXA3wdfAbuFbFCmhd\\\\\\\/7X+l6td12YTlC2fMIswr4a62VlWk2Vn0L3tzE74QX0zK1NRrAFPpyO9e6VpcSCLY4EVeNVD1mNH+Ob381LaIfnSL0eQ874umSclylN9Z3Dh3gvcsWDVJQBIbU4+RlXvD7PTauY5QhoSQMNQ5WxKZy8g\\\\\\\/bMh0\\\\\\\/j8xz\\\\\\\/WfaII2KcPrI+gTba9gzg+cqUk7lVCk+0EEr38reAqXtbMdsg0dvAdkJ9Uau+wBnFSGFps0jDscQYwb5p92xrxIM64GrQuVdXAfZcrv6\\\\\\\/6JlIxndtklcmcqLL8JAEzvnA+1oFkDIboREDOr4k9btrCPNlmfN4qa2hrA9iIK+odyYXW8I8nuO1due7N9VoPIs\\\\\\\/ksHnULLtMIHHDYOzIHN2SOIBrnQdly5N82UIkXyLMNtmOpZlGRwpAENmad54tY3+gJccclgq+avyPnMz6XmqwUYtoEiNKiFyhO5sdmhHDPmgKE2sYcO\\\\\\\/iFKtaiSgWYHAKwmorq8SbBG62o7OfNunTUH6yA4n8K1+aqY5ajSfyzyhIfkZkiJDJLcdZVRnWWX25ASGu83amalN64SjkWthdsnC\\\\\\\/saFYtB72DWoZnHKYm002m1NRGy0qBZYuJ6Dgxax3zZM2RSZYu0zyal8dL5TLQhB1BRRq+tT84nw+00N6k7Nxg\\\\\\\/WxOFMl4je9byTQ0khx5ExGNhKs1fa9kEq9yNt5HBYAwJmWLOY58JSE4JcC9iybqFEAkpBs7v1lJZXZTKziZF\\\\\\\/g+33WtQxBH6m4sUz8MFusO6wZnBPg674Q2TPMXhQgTojVV4infUB4SPuffNEJuIFqbZCe6zxlpVVULK\\\\\\\/5O9sZ8UeMlUo7uWVSj\\\\\\\/qOelTY0GRDFo5tE8qSsu9vPRtAIH\\\\\\\/q\\\\\\\/tZFFJAHDe7E++xScfydK+5Ljn1IT2eLPRfdhSGweosrY22LVTGSKCn0t7ECqpT8VV83Yzw0ow+iA0uaR4ITE53uBZu3T1dCiezCZlT6ssfOBaHPEnWfzD+4FZr0jmlAXWb6Mz4GUZIf\\\\\\\/T9cy5pRcHoQPapkFrRk4xC2OdAvuMl2kdSr9adjma9pnSo4y5kCCTARzMAnkt1bBkE379oMZBrhZjNLtwUmUrALFvl91aQ\\\\\\\/CAYJJDOWpNNb2RqfAAQ\\\\\\\/M8H\\\\\\\/pCyMUMIDaJGQEhaPB8H9Zv9uFHpL+ZkQCnR8DZ038tnWGIJ55079WFR78U8eEu+wcMysR28DyHJxKxnmvnGqaE4LRgUbl7x0V1gsjjfNLWhZaIclEn2jhGFQy\\\\\\\/EbLQ2QYIsWbBeZOMr6+mqNGnw022jBYJSGGRr7\\\\\\\/NuxQBT4SVIlxEm1mZv3aR1IhM7aHkjc6g0XddqdPXUBsm8uGga\\\\\\\/TunzdqHR6eYQo\\\\\\\/1WlrYGHFhGBqMOBeij5sjapxPPXKKaKn2iHlcRU\\\\\\\/PvkU+WSTW8\\\\\\\/SJI7LBqOuFdFerVzx0O8v2z7dCm11cBH8qs8MTVgqyWuY4O750ElC5tREAUAOR+MMqiuzMyYSwGyN6530ubIwoKUlG8KGfsyPTf8ksNSPf4dVb8hsuo0YqX1vrBP5Q\\\\\\\/ogCxFG8MBHczupSB6Xdj42eVhIGWqYoRm5KcfvA\\\\\\\/5\\\\\\\/Lig+cCxVbEGACdKU3AzH0As21vJpr8alScQdNUgz97Ojt9rC\\\\\\\/2BiFfC8psxaQ\\\\\\\/xdVyq\\\\\\\/yz7YJQOpcyZP+q+R7xhc8FfX5rOfLoTNGAFCZfVPRoceBT2UfizpIRg8q29ukMJ36T5a2xT\\\\\\\/CaqD8n42gLBMVcOwqbYWEW7gJGxluJmzIVsXxuKGSll1XduXIlhA74KSmLr+c3xRKJptMXG2jAJDHxTFVDfGEOjhb7x66EA0ibtijjs90VqZXLRUM0X3KdnU9rrxOcK5RAmBZ6iskY4hO3\\\\\\\/3\\\\\\\/72a4u\\\\\\\/hkLfUn7i2MyFpH0iIcnQUA8fUxx+610uUM\\\\\\\/qVVC3Gh1HHdAgxNOVgh7RF6UkUebMIuSi\\\\\\\/Q0edZR2NyOmYla5ynO6TalIlGSyhCLZulTiRFheOIZpX4a16xwTiBrJuPzmMDkclcOu7QgHdqxqOJ05W+YgUZ6+n\\\\\\\/qbPsD0fVPcvMeYM4kHvw9B633q28i+uOYz6FxV8+ICSiNlqDh6u1eHeQe3p+zcJ9YmhJHJEYeG4cBE3QzvtxHiA6CadjZGiAWsSlpyWNyaALcP1atx57E1KmhmU2O297X4BsoVW65hNjLPiV+4eOeXzKxQoBITbjSywSr\\\\\\\/OQNlO\\\\\\\/r4b7\\\\\\\/ZeMmVhSQo9KPYFeZDne8YW9uJM7CIcFouNCIkVr8Pfjv9z4ZEYc9jUyzuGskwQ9pnrw7aD9Wl48nmCIm64qM18v2ktDf3ewyJ7ETDt9WG8GK3MPEhxxvlergJ61WTso1TgXtGFYZ\\\\\\\/x\\\\\\\/vs+O6smUK7VC46JX0Pd4jUtU8Eas51CGSgwlpymql6IcAv0qSbXBVd15OLNFR3LE+ayE\\\\\\\/56bSyKoJYoqUkVFP9ejsJ3ephpv7bdnnKpjP1+KUra5dQsGKSdcDVkfo3o7vqKPXqFIZdV8nFnb3XtJiaZoGH5M3GkuPAiVwA97DmSxwvRu0OTgWQgf2X2i53P1NHZ6WEienFjFmPcocYhQp6hdS2buYZzu6\\\\\\\/D0tYbUmZ\\\\\\\/Djr12iNAIcoyWhC2jyNoZ2fuLI+v2u\\\\\\\/cNTFRPeyW0tpLyrz+F0bJXfz2a4HWvhMHpXbGKpd6r+XJvsu3FJOfWfcZ7a0TXusabmXWfFcj6HblH6oVOzFvsOFCNsw8gA8mRCs\\\\\\\/B56fHg\\\\\\\/zYggkia\\\\\\\/phoyDpB60zPqp9uONYLteDyyDw3aXw\\\\\\\/6wAUwzUZFdJa4zCOGQ3L5xeNYyZxd985r2686OKqjp4Cb6cojwayfIguIXvc+zehzNJOMATjaWqzDk6u5LJUWr4ty65R5LFe4iYIcR5d8Pt+whuWNBktiXpJNbiHnvpq\\\\\\\/kRIQX8xguQe3d5pZ9sBrqz6o75OWoNDoOOBzoAqctXkIHqyLbP01rc7fWnpRKuHNsWgcC3iPlHVU3ZIaeW6\\\\\\\/HXa+RS5+dTLC9fNg4GxAoNVVY0bklqn2R5LQ4WkosDLJHBFkbvtXEaaqT7dGqnUNN2Ba194Y775dR+6Y7NQan25N8mM\\\\\\\/eo5mpssUFSS7GVcjKCDDo09nhjcvQQ2g4Cls3mE\\\\\\\/7CWk2y7Jhq0uCoBmqIzM7A1f9ri9ZnmCnmB1gFOTOBDsGEebjpz26mGCpVAsFWb1WkLG1OzK2S36qLeJhEd2zR6X94\\\\\\\/SImbwVmDu0D5HuKNu8VT2sl3lcq92uzLGtscGR8thSQz0Sr5mUIRidDL6BaocZP2KusGZbjcaymlnrKgTq6iRWPxpGpm7F6Se7X7evj1G5fo9tM8cbWozmdfvxDhxJ+7woIPv1TuyfP9T88xt7R8M8zDL4edqzdvvKXES1xYtyqbrwW282SICkolz\\\\\\\/p914inBSuefhnyB384UB\\\\\\\/1c4VqAJ4iloljsfQG1Q3CAp2cziXrAsmpmpj8xxUkqqD9DLqNuqRCmK5UasXyfYalI9b\\\\\\\/N5HlCepNlQT0LGjvRuJCnGdcrpxa7O9Vk9fCj4BtXQfpMBdBR2ZMreK0JA0HAGhe6sWSKNkcXrra3SwbbYLVhWTaEkZ5FLK6DikGW2jLlS7GVevBsTi5sIDSntUcKRkD4KMALwf7hwzLb50\\\\\\\/skjqWHZ+uiV5+ml8n+cxehsqGBBv0rW1vch\\\\\\\/AyufVnZujFw\\\\\\\/NLfCBVzPbfFytifUUAQ90zND\\\\\\\/S6etLIrzI2C10CmA\\\\\\\/gjAN4df9rx\\\\\\\/pbmC1i2wjfGMUGmFaMH\\\\\\\/45gGml0re944YFb3ijhZaIDq9ohiTlOqELyW0Bfnqp3P2FZ7GkS0pGlzboRDgdZ8iAkABpsgZcvDJwQH5GzFGDIRz8UQNbesHST\\\\\\\/ko\\\\\\\/R3hIg8PeJWZmKOC03XfD06iUs\\\\\\\/Xa\\\\\\\/g8vbw1\\\\\\\/tIth3DUtJcVzbSUsksYcoH5Vttls9umXIUYPP6C8szAgIBtCcuLrh1WY9JyjHO7wZhznM1UANLepsay3gNI5znMI3V3GpCJr2t\\\\\\\/Ls8iYRkRqd9np5IxEW4KusnqQNAp9sae6nArq\\\\\\\/B5GnT0S\\\\\\\/VLooRZekS02lm8647VBSQ9kKBY3+ZFSIVPyzWyUX6uCjnHpVV8UKAey3kvmfF36dLsb9L4wLfcMXKnBcdEcE2Xq\\\\\\\/bsBF1D7GZz1PNuc4\\\\\\\/yQ2XEpRiXHRaZ69p\\\\\\\/25x+8LLUmUZBtYTrr9l2uhp7or37FroXTHxmtHP9K49PVBm2tI95hF4iqXp8AdQ4fqysdzLGqhXT5p+pfbphUbWfFNgVl8YkPwdbY9YqypFrIcYMt6KVTnzQ7igKAxf7dmRBUI+bwqVYdAW9hYp3Zh5f+qiQ92le+0OfNIEXnScEqkioS4HT3masG+mgTjsw9v11hRjFrL2QILMsv9Irx5EWWHOhDloZSnRXmhVotBu+XeV26AHYHp4ID3yppaYCsxVvY5arzckOzX8CfQBqOY4AqXRPy5vxALnCgI5BUxjkLJUZKdlHnU+Vf0j5822VjykrQTnEOKtdoVaCykezE6dHzy3osEwmOXDcn96BxlDdC0MyjjvtbBkd1UGyKZGi72YFEYkool7c2NU1vNohkPjdoVezcZ5WygJ307ROrSvH7THpUXJsuO+ULwl2WHJXEkbZFBknrByynUcRepV8QXl2KIDOt5tU6SN7WKbdAsrPjdjb8vUWSdrfiCEMdvpDYlGnoAgZW\\\\\\\/TV\\\\\\\/hP\\\\\\\/RYGKi4THrPoMTTKqaJgU1k1bRFDyh6QtkHg0T0IUUBYgJINrF9XfULRgz++Wlive6fz8OmqEAQW723AXQ0NtP5vO78+2gn7tl9Y3LbxT7Uefm5089mNzla07Bn5lg6b5C1vAp6+WlCx7N1s+Rs3ctn1dMDggvYdDjmQhKnDKy+Am5S2TktMB6kGao\\\\\\\/33u\\\\\\\/aWej+UgIYsdaMwSUi5Ji8DiJAETmY0oMqQWDtT\\\\\\\/IIbe6Ifoyl6dLhPMSwMfWVVN8kbQASHGdaNKvyN56wMrw39ApcZCenoLiPuBXI\\\\\\\/P8JRAaa4M24yNvHSMTI\\\\\\\/zdFkF9GCioFmrESvlOQ6I\\\\\\\/VkK2rYx8hCuJJTVLogb4Bd1BI9QGJWou6Fb8lTkXqGtIOkKlPRwIW7QEG\\\\\\\/6ZaRkEnTkpa8udoweI7\\\\\\\/uwvBVGc+q8v9kEcC2BeR+X1LdDCBLF9V0g+\\\\\\\/QSmrD4mz7KxXa7yrV67NQ1TYyCY3kh098z\\\\\\\/PL7lvHDfxSS8Rci5k+cyRtqfZQionnW1GhecJ76pmMcMMSgiLo9IKdVNfnxVU50xQd+DI4i5EZGdmxdfcv9EacJG1JU+1Sw208LSKQQh4IjrtCuCWmA3uh4RLDIje2U1df0TmG77ZIRKsRinPhoxIlBV87jOx\\\\\\\/ItcRTedQm6hLqd5YPkTcVXEZg9iPlduacas1lR6TElulmW9ZNX4jc0RT8t7vNQ+EcQSWcIH4o8bq3MIGoX\\\\\\\/jWMC31uLVfTA9nQNfx4jM6GnoYbRDficpEEhwslFvtKoU2691E50\\\\\\\/AniYylSU2Zid7v3BLP5DDNDQlHboRplWmtGWzU1eWd8S9eEFZ2gT+12Jmi0ntsVZ7+ISA4ltyZbTlALo9pRhUKYAcKX000UmXJfBTzdsgjimixFNzIyqN\\\\\\\/0BnORYjoFE1iFK8j7y+1PAvX3lJF1+G3Khw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022I8f2eaAqOQ4LqO9h6htbOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220c61fcccaeeb18141a01085c9178cd26e5127e3d33b0c14304c9ec34afe7aad8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022i5hYsy8oUpr4ipLKHBUnktGIs\\\\\\\/nCxdhZg3jUg0W6GYE4mbWzuIsV5qKxg0TDZp2feHYTmmg1qMbWJK29pHF3NIE19V7TuPJ6Q\\\\\\\/GUFy27SZudNeeKnPrcxTeh4mx\\\\\\\/fo31GJq4DyUIJXpr+A0ymkzSPUabDO8CFIKNvPP5LuP9C12uCaMZz88BTwf7xn75PALG1i3GI24idXD4dsP8Wb6oUfxgcvA60F91NCCisWFkN0iV\\\\\\\/OxKbKLcuBeH\\\\\\\/V08t7jTwq\\\\\\\/5FVJ+PPUWPfhQ7Ej+goPeAE3G+VX4msUY5hRFoCzg\\\\\\\/NS+bjdziouQRPgxuqYyNJYY0\\\\\\\/Kppx0OhpjcvfuHRACw3GZ8AdsFPgXSKBE9tZ2j49xj7qiz0U75IEZHQh3VlsW3US6lvDwhKh\\\\\\\/vc6Na\\\\\\\/WX4YuqzG6o0JE5xIUZLZIICTvlVGzHKTt10CA7nVmu\\\\\\\/3fj6DpdkBVN4rGlGKmsHQZvlVLt1b6dV0Pr3GcIxGYCTb6twqx24nXXG0PKEFSQunEOYur47cwjcB\\\\\\\/s4HoyKsMvC\\\\\\\/IQWl6wU8kb98OieAvkBN6eUpjK2NuqmtN7AV3vGoPqIQcOmHj23KyORTnRz6CLbSDUG3Vib+AZaBMZmDrh97wA02so\\\\\\\/rI5yQ+bmT9kY4C0RRjCL9CSoYAXmFk+f1YQBLm94VyRedmdZm4mMazbyTOUerZuW4VwE2U3Xyr3ug0+1wdbyOOH8LMQ2bE2py8Fo7n8K+RVumLcIt7FSePIrgdbpdAyup5n75APQ3p5A1tie+dEw1gLF3EkE8eu8cxN9yDq6sGB7kz1uq\\\\\\\/radUbNPlJQBFKbDs2AAdLCR9QnkG68aP99kVDswXwgQsQyF0jAp50h5+MruLuWHoZ122pXd3kN4Fl3DMm4oc\\\\\\\/oYkJwkz6D6so6DQ2YKLoQeYb6\\\\\\\/W3+Lm+BUUSYNl125OVbM5LNsotI26C4UHE8aImRrHqfBHda2\\\\\\\/UqnY3N0p9F\\\\\\\/o69LLZBvYoGRl7etxWH5K2Kc\\\\\\\/DLMvZmRmrKAh0cGBIfn3AoqN38RL6D+XOy0NGHKh29pNU\\\\\\\/R\\\\\\\/47a03R3v5XylvIOvNztGughDwmEbPwQZhStGQTRXHY7mqv4lijs6hO5\\\\\\\/Pf40BPA4yEzTasH33PfbGe18+q1J8tIDfckBTFL81r99Upfc8HbNMPEUC3rLOtYsfkHCwZKaPciAwsKwIrsQvtghnrLaNV40LOl7MfMwXYGQmS38l5yI3tNtuWrLw+i72faOQbTiPRszwsVhXjeuxFdTBYB1qyOzbBdSTc18L8lQDsyTQxARG8GAKIhvWzcGpE\\\\\\\/dSnmlNQnXW04OifIekw2deQbqSQ9LukalX8p28j1KGIllO7V\\\\\\\/tOJ1C9IOHz09zVAtpfSlZ0d7b7jp0ARaH0UB4qSp3bT0zvsU95\\\\\\\/aNk4MM1sdUW4dx9NkFm0Nf2SzFy11kygPf2ATB4dgWhCHJQKTK7xmRu\\\\\\\/FW2MFotvSP5nU\\\\\\\/y34Byz2Z9OK86RHlheXadcoscf+wqCrnZ9Nt1rjhP14cYg71VzsL2QRr+u9eRyk6uakp79SI\\\\\\\/Eg+\\\\\\\/W7S7vlI8cgX8xCkJPOkrbAwyCt96yEKvSf6oWUjQzo4D6UwqdyzK6OQtcAMv5ilX\\\\\\\/5xE1BhLiLY5KfcobPaGLgHDek8as7x132Mr0tSHPPX7c1SfdC\\\\\\\/A37p4nrHFgVvEKzQZ\\\\\\\/uyo+QGxTAth+eUqcuaZWdW\\\\\\\/lgCnDd4jjbLw1bwXet6BrqPF3nvt4QuAUOKiP9khnynT6tMM8OJNkm+0EhAnUyaVtL\\\\\\\/77d6RBidpyjZFNjPfDW1Vk880YDpKQHx6TVQZPtwXDNU73279njSVOAMWGed8KkAf2fb1LbUV88cNy4qD9xP3IpgNVdk\\\\\\\/qViUYItq6JxAOwsYL3yNnhlBVLF0TFRyK8SXGmTLqJWVAKn5N+b+5lBFLlOSYKl8+ioRJfvXs8C\\\\\\\/6CPmseVSmveBNIDjHrruTmIL68fJPgjQuDuPVr9q7E7pEPiJ6w+QXY2tskEkAFTMmA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MMzcbktlOx\\\\\\\/Oj9wsdNtPeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220eeb69b4fd9c51ceb7feda37d59b39015b6444c555abe045048cbbb918d4a339\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490737 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2901" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AlSTtYjum9VyWHu5ielafjGej1E2CIx7O7NQDkOG0uA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2901" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LB8UGwHsf6g6xa5dzdIl_mvCLs6fxVi78CJdfn6V-L0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2889" title="Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2913" title="Disruption of INOS, a Gene Encoding myo-Inositol Phosphate Synthase, Causes Male Sterility in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490737" id="node490737" data-pisa="ggg;8/9/2901" data-pisa-master="ggg;g3.118.200522" data-apath="/ggg/8/9/2901.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0025-5383" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiexiong Deng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8096-5344" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Min-Hao Kuo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2901-2911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200522 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490737" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiexiong Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiexiong%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiexiong%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0025-5383" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiexiong Deng</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Min-Hao Kuo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min-Hao%2BKuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMin-Hao%2BKuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8096-5344" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Min-Hao Kuo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kuom@msu.edu" class="" data-icon-position="" data-hide-link-title="0">kuom@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/2901" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490737" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490737/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2901.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490737" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490737/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2901.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490737" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490737/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2901.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490737" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490737/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/2901.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490737" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490737/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The spindle assembly checkpoint (SAC) is key to faithful segregation of chromosomes. One requirement that satisfies SAC is appropriate tension between sister chromatids at the metaphase-anaphase juncture. Proper tension generated by poleward pulling of mitotic spindles signals biorientation of the underlying chromosome. In the budding yeast, the tension status is monitored by the conserved Shugoshin protein, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, and the tension sensing motif (TSM) of histone H3. ChIP-seq reveals a unique TSM-dependent, tripartite domain of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in each mitotic chromosome. This domain consists of one centromeric and two flanking peaks 3 – 4 kb away, present exclusively in mitosis. Strikingly, this trident motif coincides with cohesin localization, but only at the centromere and the two immediate adjacent loci, despite that cohesin is enriched at numerous regions throughout mitotic chromosomes. Chromosome conformation capture assays reveal apparent looping at the centromeric and pericentric regions. The TSM-<a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>-cohesin triad is therefore at the center stage of higher-ordered chromatin architecture for error-free segregation.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li><li class="kwd"><a href="/keyword/spindle-assembly-checkpoint" class="hw-term hw-article-keyword hw-article-keyword-spindle-assembly-checkpoint" rel="nofollow">spindle assembly checkpoint</a></li><li class="kwd"><a href="/keyword/shugoshin" class="hw-term hw-article-keyword hw-article-keyword-shugoshin" rel="nofollow">Shugoshin</a></li><li class="kwd"><a href="/keyword/histone-h3" class="hw-term hw-article-keyword hw-article-keyword-histone-h3" rel="nofollow">histone H3</a></li><li class="kwd"><a href="/keyword/tension-sensing" class="hw-term hw-article-keyword hw-article-keyword-tension-sensing" rel="nofollow">tension sensing</a></li></ul><p id="p-3">Equal partition of the duplicated chromosomes is crucial for genome integrity and species perpetuation. Aneuploidy resulting from erroneous segregation causes developmental defects and tumorigenesis (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">R<span class="sc">icke</span><em>et al.</em> 2008</a>). The spindle assembly checkpoint (SAC) is a failsafe for faithful segregation. The SAC registers the kinetochore-microtubule attachment and the tension between sister chromatids (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">P<span class="sc">insky and </span>B<span class="sc">iggins</span> 2005</a>). The tension generated by poleward pulling of the spindles signals bipolar attachment, after which cells irreversibly initiates events leading to the onset of anaphase.</p><p id="p-4">In <em>Saccharomyces cerevisiae</em>, each kinetochore attaches to a single microtubule spindle emanating from the spindle pole bodies (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">C<span class="sc">leveland</span><em>et al.</em> 2003</a>). To the two sister kinetochores, three types of attachment may occur: monotelic, syntelic and amphitelic (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">P<span class="sc">insky and </span>B<span class="sc">iggins</span> 2005</a>). While the amphitelic attachment signals biorientation, monotelic and syntelic attachment errors have to be corrected before anaphase onset. Monotelic attachment refers to the situation when only one of the two sister kinetochores is attached to the microtubule. The presence of an unoccupied kinetochore triggers the formation of the Mitotic Checkpoint Complex (MCC) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">B<span class="sc">rady and </span>H<span class="sc">ardwick</span> 2000</a>) that halts cell cycle progression by trapping <a href="http://www.yeastgenome.org/locus/S000003084/overview">Cdc20p</a>, the E3 ligase subunit of Anaphase Promoting Complex (APC). In syntelic attachment, both sister kinetochores are occupied by spindles, but these two spindles originate from the same spindle pole body. Even though the attachment requirement is met, there may be no tension between syntelic sister chromatids as they are pulled toward the same pole. Left uncorrected, monotelic and syntelic attachment results in aneuploidy.</p><p id="p-5">In what form tension is perceived by the mitotic machinery remains elusive. In prometaphase, transient sister chromatid separation without cohesin proteolysis is caused by kinetochore-microtubule attachment (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">H<span class="sc">e</span><em>et al.</em> 2000</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">H<span class="sc">e</span><em>et al.</em> 2001</a>). Conformational changes of centromeric chromatin (DNA, nucleosomal arrays, and selective proteins) thus are suggested to be the “tensiometer” or “spring” that reflects the tension status (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">S<span class="sc">almon and </span>B<span class="sc">loom</span> 2017</a>). Among these candidates, Shugoshin proteins are of particular interest. Shugoshin is a family of conserved proteins playing critical roles in ensuring appropriate chromatid cohesion during cell division (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">M<span class="sc">arston</span> 2015</a>). The budding yeast Shugoshin, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, was first identified as a protector of meiotic cohesin against precocious cleavage (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">K<span class="sc">itajima</span><em>et al.</em> 2004</a>), and later found to be also crucial for cells to activate the SAC in coping with tensionless conditions in mitosis (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">I<span class="sc">ndjeian</span><em>et al.</em> 2005</a>). Expressed in S and M phases of the cell cycle (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">I<span class="sc">ndjeian</span><em>et al.</em> 2005</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">E<span class="sc">shleman and </span>M<span class="sc">organ</span> 2014</a>), <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is localized to centromeres and pericentromere (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">K<span class="sc">iburz</span><em>et al.</em> 2005</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">F<span class="sc">ernius and </span>H<span class="sc">ardwick</span> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">K<span class="sc">iburz</span><em>et al.</em> 2008</a>) without stashing a significant extrachromosomal pool (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">B<span class="sc">uehl</span><em>et al.</em> 2018</a>). Shugoshin is recruited to centromeres by binding to histone H2A phosphorylated by the <a href="http://www.yeastgenome.org/locus/S000003420/overview">Bub1</a> kinase (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">K<span class="sc">awashima</span><em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">L<span class="sc">iu</span><em>et al.</em> 2013a</a>). The centromeric recruitment of budding yeast <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> may also involve the interaction with the centromere-specific histone H3 variant <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">M<span class="sc">ishra</span><em>et al.</em> 2017</a>). In human mitotic cells, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> recruited to the outer kinetochore nucleosomes is then driven by RNA polymerase II to the inner centromere where it is retained by cohesin (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">L<span class="sc">iu</span><em>et al.</em> 2015</a>). Besides cohesin, the fission yeast meiosis-specific Shugoshin <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> interacts with the heterochromatin protein 1 (HP1) homolog <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6</a> that docks on the heterochromatic mark H3K9me3 in pericentromere (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Y<span class="sc">amagishi</span><em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">I<span class="sc">saac</span><em>et al.</em> 2017</a>). Unlike other eukaryotes where heterochromatic marks decorate pericentromere to create a footing for Shugoshin, budding yeast lacks such heterochromatic features in the region immediately next to centromeres (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">C<span class="sc">leveland</span><em>et al.</em> 2003</a>). The geographic pericentromere recruitment of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in budding yeast, instead, is accomplished by the association with the tension sensing motif (TSM) of histone H3 in pericentric regions (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>). TSM (<sup>42</sup>KPGT) is a conserved β-turn that connects the flexible N’ tail to the rigid histonefold domain of H3 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">W<span class="sc">hite</span><em>et al.</em> 2001</a>). Mutations at K43, G44, or T45 diminish the pericentric localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and obliterate the cellular response to defects in tension. Restoring pericentric association of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> by overexpression, via <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>-bromodomain fusion (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>), or by mutating the inhibitory residues K14 or K23 of the H3 tail (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">B<span class="sc">uehl</span><em>et al.</em> 2018</a>) rescues the mitotic defects of these TSM mutations, thus manifesting the pivotal role of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> retention at the pericentromere. <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is removed from chromatin after tension is built up in the metaphase (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">N<span class="sc">erusheva</span><em>et al.</em> 2014</a>). The inverse correlation between <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> retention and amphitelic attachment suggests that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is an integral part of the gauge by which cells use to monitor the tension status.</p><p id="p-6">In addition to the TSM, another factor important for targeting <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> to the pericentromere is the cohesin complex. Mutations that impair cohesin loading ablate pericentric localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, while leaving the centromeric <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> largely unaffected (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">K<span class="sc">iburz</span><em>et al.</em> 2005</a>). A similar contribution of cohesin to <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> localization has been observed in human systems as well (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">L<span class="sc">iu</span><em>et al.</em> 2015</a>). Cohesin performs its tension sensing-related function by facilitating the formation of the “C” loop of chromatin near the centromeres in mitosis (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Y<span class="sc">eh</span><em>et al.</em> 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">S<span class="sc">tephens</span><em>et al.</em> 2011</a>). Direct interaction between cohesin and the human <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> has been reported (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">L<span class="sc">iu</span><em>et al.</em> 2013b</a>). The triad of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a>, H3 TSM, and cohesin thus likely constitute the core of the tension sensing device. The present work presents evidence for a cohesin- and TSM-dependent tripartite chromatin localization domain of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> that also involves high-ordered chromatin architecture.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and plasmid constructs</h3><p id="p-7">The yeast strains, plasmids, and primers used in this work are listed in Supplemental Tables 1 to 3.</p><p id="p-8">To study the genome wide localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, the 6HA epitope-tagged <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> strains, yJL345 (H3WT) and yJL346 (H3G44S) were constructed as previous described (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>). The <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> overexpression strains, yJL322 (H3WT) and yJL324 (H3G44S) were generated by transforming pJL51 (a <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> plasmid with p<em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em>-3HA-<em><a href="http://www.yeastgenome.org/locus/S000005599/overview">SGO1</a></em>-t<em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em>) into yMK1361 and yJL170, whose endogenous <em><a href="http://www.yeastgenome.org/locus/S000005599/overview">SGO1</a></em> gene was deleted using <em><a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a></em> marker. To ChIP <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a>, a 13Myc tag was introduced to the C terminus of <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">MCD1</a></em> locus in yJL347 using pFA6a-13Myc-His3MX6 plasmid as described (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">P<span class="sc">etracek and </span>L<span class="sc">ongtine</span> 2002</a>). The resultant strain yXD225 was transformed with either pMK439H3WT or pMK439H3G44S (a <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> plasmid bearing all four core histone genes) and followed by 5-FOA selection to select against pMK440 (a <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> plasmid bearing all four core histone genes) containing cells, generating yXD233 (H3WT) and yXD234 (H3G44S). <em><a href="http://www.yeastgenome.org/locus/S000001277/overview">BAR1</a></em> was deleted in yXD233 and yXD234 to yield yXD237 and yXD238 respectively, using homologous recombination approach with <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> marker. Another version of <em><a href="http://www.yeastgenome.org/locus/S000001277/overview">bar1</a></em> deletion was made in yXD233 to yield yXD282, using <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> recycling approach as described previously (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">A<span class="sc">kada</span><em>et al.</em> 2002</a>). An adapted <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> recycling method was used to replace the CAR sequence between <em><a href="http://www.yeastgenome.org/locus/S000002411/overview">RAD57</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002412/overview">MAF1</a></em> with <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a></em> promoter. There were 4 steps PCR to attain the recombinant fragment. Step 1, primers oXD236 and oXD237 were used to amplify 3′ end of <em><a href="http://www.yeastgenome.org/locus/S000002411/overview">RAD57</a></em> from genomic DNA. Step 2, amplified <em>pGAL1</em> from plasmid pFA6a-<em><a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a>-pGAL1-3HA</em> with primers oXD252 and oXD253. Step 3, PCR the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> from plasmid pMK440 using primers oXD254, oXD255 and oXD240. Step 4, combined PCR products from the previous three steps and used primers oXD236 and oXD240 to amplify the final fragment. The resultant DNA was transformed into yXD282 to attain Ura<sup>+</sup> transformant, which was then subjected to 5-FOA selection to generate yXD286.</p></div><div id="sec-3" class="subsection"><h3>Yeast methods</h3><p id="p-9">Yeast growth media, conditions, and transformation were based on standard procedures (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">S<span class="sc">herman</span> 1991</a>). When appropriate, 5% casamino acids (CAA) were used to substitute for synthetic amino acid mixtures as selective medium for uracil, tryptophan, or adenine prototroph. Yeast transformation was done with the lithium acetate method (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">G<span class="sc">ietz</span><em>et al.</em> 1992</a>).</p></div><div id="sec-4" class="subsection"><h3>ChIP-qPCR and ChIP-seq</h3><p id="p-10">ChIP was conducted as previously described (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">K<span class="sc">uo and </span>A<span class="sc">llis</span> 1999</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>). To quantify the ChIP results, ChIP DNAs were analyzed with quantitative PCR using primers from Table S3. The libraries of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> ChIP-seq were prepared as described previously (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">F<span class="sc">ord</span><em>et al.</em> 2014</a>). 10 ng of ChIP DNA was used for each library preparation. Size selection of libraries was 300-500 bp. Libraries passed quality control were then subjected to Illumina HiSeq 2500 to get 50 bp single-end reads. Reads were mapped to <em>S. cerevisiae</em> genome (Saccer 3.0) by Bowtie2 (version 2.2.6) using -m 1 setting for unique matching reads. BEDgraph files of each ChIP-seq experiments were generated by HOMER (version 4.7.2) and were visualized by Intergrative Genomics Viewer (Broad Institute). Read analysis across centromeres was done by using code of Cen-boxplot_100kb.pl adapted from <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Verzijlbergen <em>et al.</em> (2014)</a>. All ChIP-seq data in this study are available at the Gene Expression Omnibus with accession number GSE110953.</p></div><div id="sec-5" class="subsection"><h3>Chromosome Conformation Capture, 3C</h3><p id="p-11">3C was performed in 100 OD<sub>600</sub> cells of G1 or G2M arrest cells as previously described (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">B<span class="sc">elton and </span>D<span class="sc">ekker</span> 2015</a>). Instead of using mortar and pestle to lyses cells, 50 U/mL lyticase was used to digest the cell wall for 25 min at room temperature. Primers are designed around 50 bp upstream of the targeted <em>Eco</em>R I sites. The digestion efficiency of each libraries was evaluated by qPCR. Samples with at least 70% digestion were carried on for following assay. PCR products were resolved by 9% PAGE and stained by ethidium bromide. The intensity of band was analyzed by NIH ImageJ.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-12">Strains and plasmids are available upon request. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6227180">https://doi.org/10.25387/g3.6227180</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Sgo1p displays unique tripartite localization in each mitotic chromosome</h3><p id="p-13"><a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is critical for the tension sensing branch of the SAC function in mitosis (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">M<span class="sc">arston</span> 2015</a>). We and others have previously used chromatin immunoprecipitation (ChIP) to demonstrate that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is enriched at centromeres and several kb on either side of the centromere in mitosis (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">K<span class="sc">iburz</span><em>et al.</em> 2005</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">F<span class="sc">ernius and </span>H<span class="sc">ardwick</span> 2007</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">N<span class="sc">erusheva</span><em>et al.</em> 2014</a>). However, the range of the <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> enrichment on each mitotic chromosome has not been carefully delineated. To better understand <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> retention pertaining to its checkpoint function, we used ChIP-seq to map the <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> distribution on mitotic chromosomes at a higher resolution. Cells bearing a C-terminally HA-tagged <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> expressed from its native locus were arrested by benomyl for ChIP-seq. At a lower resolution scale, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is detectable in one area per mitotic chromosome (Figure S1A), consistent with the anticipation of centromeric and pericentric enrichment (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">K<span class="sc">iburz</span><em>et al.</em> 2005</a>). However, more rigorous inspection revealed that each chromosomal domain of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is actually composed of discrete peaks of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> that form a trident-like structure, not a continuous motif covering several kb of a centromeric and pericentric area (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Each of the trident motif consists of a middle centromere (CEN) and typically one pericentromere (PC) peak on each side of the CEN enrichment. Some chromosomes such as I, VIII, and XI, show weaker PC peaks and therefore a less conspicuous tripartite pattern. However, overexpressing <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> from the <em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em> promoter on a multi-copy plasmid increased the overall IP efficiency (reflected by broader scales) and heightened these peaks (green curves, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). These observations suggest that the mechanism for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> retention is conserved, whereas the relative strengths may differ among chromosomes. By aligning all sixteen chromosomes at the centromeres, the average counts plot for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> enrichment as a function of distance to CEN shows that the average distance between the PC and CEN peaks is approximately 4 kb (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, magenta line). Additional outward peaks may be seen in some chromosomes, but the overall peak height drops quickly.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sgo1p is recruited to centromeres and pericentromere to form a tripartite localization domain on each mitotic chromosome. The 100-kb region centering on the centromere of all 16 chromosome is aligned. Sgo1p expressed from its native locus (magenta), or from a multi-copy episomal plasmid (green) are compared with the Mcd1p distribution (dataset from Verzijlbergen et al., 2014). The two peaks labeled &#x201C;a&#x201D; and &#x201C;b&#x201D; close to CEN2 and CEN5 correspond to ARS209 and URA3 respectively. These loci were from two plasmids in the strains used for experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sgo1p is recruited to centromeres and pericentromere to form a tripartite localization domain on each mitotic chromosome. The 100-kb region centering on the centromere of all 16 chromosome is aligned. Sgo1p expressed from its native locus (magenta), or from a multi-copy episomal plasmid (green) are compared with the Mcd1p distribution (dataset from Verzijlbergen &lt;em&gt;et al.&lt;/em&gt;, 2014). The two peaks labeled &#x201C;a&#x201D; and &#x201C;b&#x201D; close to &lt;em&gt;CEN2&lt;/em&gt; and &lt;em&gt;CEN5&lt;/em&gt; correspond to &lt;em&gt;ARS209&lt;/em&gt; and &lt;em&gt;URA3&lt;/em&gt; respectively. These loci were from two plasmids in the strains used for experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F1.medium.gif" width="355" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/9/2901/F1.medium.gif" width="355" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490844" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Sgo1p is recruited to centromeres and pericentromere to form a tripartite localization domain on each mitotic chromosome. The 100-kb region centering on the centromere of all 16 chromosome is aligned. Sgo1p expressed from its native locus (magenta), or from a multi-copy episomal plasmid (green) are compared with the Mcd1p distribution (dataset from <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Verzijlbergen <em>et al.</em>, 2014</a>). The two peaks labeled “a” and “b” close to <em>CEN2</em> and <em>CEN5</em> correspond to <em>ARS209</em> and <em>URA3</em> respectively. These loci were from two plasmids in the strains used for experiments.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sgo1p enrichment overlaps with cohesin domains at the centromeres and pericentromere. Average counts (per million reads) plot comparing the distribution of Sgo1p expressed in different backgrounds (panel A) or between Sgo1p and cohesin (panel B). Sgo1-6HA ChIP-seq and 3HA-Sgo1 ChIP-seq data were from three and two biological replicas respectively. The Mcd1p ChIP-seq data were from Verzijlbergen et al., 2014." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sgo1p enrichment overlaps with cohesin domains at the centromeres and pericentromere. Average counts (per million reads) plot comparing the distribution of Sgo1p expressed in different backgrounds (panel A) or between Sgo1p and cohesin (panel B). Sgo1-6HA ChIP-seq and 3HA-Sgo1 ChIP-seq data were from three and two biological replicas respectively. The Mcd1p ChIP-seq data were from Verzijlbergen &lt;em&gt;et al.&lt;/em&gt;, 2014.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F2.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/9/2901/F2.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490864" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">Sgo1p enrichment overlaps with cohesin domains at the centromeres and pericentromere. Average counts (per million reads) plot comparing the distribution of Sgo1p expressed in different backgrounds (panel A) or between Sgo1p and cohesin (panel B). Sgo1-6HA ChIP-seq and 3HA-Sgo1 ChIP-seq data were from three and two biological replicas respectively. The Mcd1p ChIP-seq data were from Verzijlbergen <em>et al.</em>, 2014.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Chromosomal retention of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> depends critically on the tension sensing motif (TSM) of histone H3 (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>), and the cohesin complex (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">V<span class="sc">erzijlbergen</span><em>et al.</em> 2014</a>). H3 is a ubiquitous component of chromatin, yet it controls the pericentric localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> (<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>), despite that no discernible epigenetic marks have been found specifically in budding yeast pericentromere that are relevant to mitotic regulation. Mutations introduced to the tension sensing motif (<sup>42</sup>KGPT<sup>45</sup>) cause defects in detecting and/or responding to tension defects (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>). These mutations diminish the affinity for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, a molecular defect that can be suppressed by overproduction of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> (<a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>). Indeed, ChIP-seq data show that the overall chromatin association of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is significantly reduced in a tension sensing motif mutant, G44S (Figure S1A, orange curve). Overexpressing <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> restored the tripartite chromatin association (green curves, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, and brown curve, Figure S1A). In addition to re-establishing the original enrichment pattern, a small number of new peaks distal to the CEN/PC peaks were seen. Intriguingly, these still are discrete peaks with clear valleys in between (see, for example, chromosome XVI, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The emergence of these new enrichments is consistent with our original model that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is recruited to the centromeres and then spills over to the nearby chromatin region (<a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>). However, the non-continuous nature of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> distribution suggests the involvement of at least one other factor (see below).</p><p id="p-17">While histone H3 and its tension sensing motif are ubiquitously distributed throughout the genome, another <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> recruitment factor, the cohesin complex, localizes at specific loci of chromosomes. Besides centromeres and pericentric regions, the majority of cohesin-associated regions are the intergenic area between two convergent transcription units throughout the genome (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">G<span class="sc">lynn</span><em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">L<span class="sc">engronne</span><em>et al.</em> 2004</a>). By comparing with the chromosomal distribution of <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> (the kleisin subunit of cohesin) (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">V<span class="sc">erzijlbergen</span><em>et al.</em> 2014</a>), we observed that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> co-localizes with cohesin at and immediately adjacent to centromeres (compare magenta and blue peaks, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The plot of average count reads (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) clearly shows the highly significant co-localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>- and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> at the centromeric and pericentric region. It is also noteworthy that most additional <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> peaks resulting from overexpression are at the loci where cohesin is also enriched (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). These results strongly suggest that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> targets existing cohesin enrichment sites for interaction with the tension sensing motif of histone H3.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The histone H3 tension sensing motif is essential for pericentric Sgo1p localization but not Mcd1p. A. Distribution of Sgo1p (magenta) and Mcd1p (blue) across chromosome XVI as revealed by ChIP-seq. The centromeric region is blown up to show the detail distribution of these two proteins. PCR amplicons are enumerated and shown in light pink bars below the Mcd1 peaks. The open reading frames and their transcription directions are shown at the bottom. B and C. Quantitative real-time PCR analysis of separate ChIP experiments. Sgo1p-HA and Mcd1p-Myc (both expressed from their native loci) were ChIP&#x2019;ed from cells bearing the wildtype or a mutant TSM (G44S). The three enrichment sites are marked with shaded boxes. ChIP-qPCR data were from three biological replicas. We repetitively observed that the Mcd1p ChIP signals to be significantly higher than those of Sgo1p (also see Figure 5). This differentiation may result from the choice of the epitope tags (13-Myc vs. 6- or 3-HA), or the nature of chromosome association, or both." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The histone H3 tension sensing motif is essential for pericentric Sgo1p localization but not Mcd1p. A. Distribution of Sgo1p (magenta) and Mcd1p (blue) across chromosome XVI as revealed by ChIP-seq. The centromeric region is blown up to show the detail distribution of these two proteins. PCR amplicons are enumerated and shown in light pink bars below the Mcd1 peaks. The open reading frames and their transcription directions are shown at the bottom. B and C. Quantitative real-time PCR analysis of separate ChIP experiments. Sgo1p-HA and Mcd1p-Myc (both expressed from their native loci) were ChIP&#x2019;ed from cells bearing the wildtype or a mutant TSM (G44S). The three enrichment sites are marked with shaded boxes. ChIP-qPCR data were from three biological replicas. We repetitively observed that the Mcd1p ChIP signals to be significantly higher than those of Sgo1p (also see Figure 5). This differentiation may result from the choice of the epitope tags (13-Myc &lt;em&gt;vs.&lt;/em&gt; 6- or 3-HA), or the nature of chromosome association, or both.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F3.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/9/2901/F3.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490815" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-18" class="first-child">The histone H3 tension sensing motif is essential for pericentric Sgo1p localization but not Mcd1p. A. Distribution of Sgo1p (magenta) and Mcd1p (blue) across chromosome XVI as revealed by ChIP-seq. The centromeric region is blown up to show the detail distribution of these two proteins. PCR amplicons are enumerated and shown in light pink bars below the Mcd1 peaks. The open reading frames and their transcription directions are shown at the bottom. B and C. Quantitative real-time PCR analysis of separate ChIP experiments. Sgo1p-HA and Mcd1p-Myc (both expressed from their native loci) were ChIP’ed from cells bearing the wildtype or a mutant TSM (G44S). The three enrichment sites are marked with shaded boxes. ChIP-qPCR data were from three biological replicas. We repetitively observed that the Mcd1p ChIP signals to be significantly higher than those of Sgo1p (also see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This differentiation may result from the choice of the epitope tags (13-Myc <em>vs.</em> 6- or 3-HA), or the nature of chromosome association, or both.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">In addition to comparing our <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> ChIP-seq data with a published <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> dataset (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">V<span class="sc">erzijlbergen</span><em>et al.</em> 2014</a>), we conducted another set of ChIP assays and used quantitative PCR to examine the localization of <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> and <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in the same genetic background. To this end, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>-HA and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a>-Myc expressed from their native loci were subjected to ChIP. DNA products were then examined by quantitative PCR for 21 amplicons that spanned 11 kb of the centromeric region on chromosome XVI, including the three CEN and PC peaks (shaded boxes, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> top panel). Discrete peaks and valleys are readily visible and show a high degree of overlapping between <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> with the ChIP-qPCR data. Additional qPCR analysis of chromosome I amplicons equivalent to those of chromosome XVI also verifies the ChIP-seq observations (Figure S2). In addition, parallel ChIP reactions were conducted in the G44S <em>tsm</em><sup>-</sup> background. While the <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> signals diminish significantly in this region (orange bars, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>), the <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a>-Myc enrichment is not significantly affected, which demonstrates that TSM is required for the retention of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, not <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a>, at pericentromere.</p><p id="p-20">The exceptional selectivity of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> for a subset of cohesin localization motifs prompted us to compare its genome-wide distribution to that of <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> in mitotic chromosomes. <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> is a critical transcription regulatory histone acetyltransferase. In mitosis, <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> negatively regulates the tension sensing motif (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>), and is important for maintaining the normal centromere chromatin structure (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">V<span class="sc">ernarecci</span><em>et al.</em> 2008</a>). Consistently, <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> is present at mitotic centromeres (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>). To see whether <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> exhibits a mitotic chromosome localization pattern similar to that of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, ChIP-seq was conducted on a Myc-tagged <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a>. The results show that, while <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> is found enriched at all centromeres, its pericentric presence is practically negligible (shaded boxes showing CEN/PC peaks of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Importantly, throughout the genome, there is very little overlapping between <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> enrichment. This is not unexpected for <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> is recruited to the 5′ region of many genes for transcriptional regulation, but <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> and the rest of the cohesin complex are enriched at the intergenic region of convergent genes. There appears to be an enrichment of <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> at RNA polymerase III-controlled targets, such as tRNA genes. These ChIP-seq results are consistent with the canonical roles of <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> in transcription (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">V<span class="sc">enters</span><em>et al.</em> 2011</a>), although we do not exclude the possibility that at least part of the mitotic distribution pattern of <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> might be for chromatin metabolism during mitosis. Together, ChIP-seq data presented above reveal unique association between <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> at and near the centromeres. However, this connection does not apply to the recruitment of <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a>, indicating a specific functional interplay between <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and the cohesin complex.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gcn5p is enriched in centromeres but shows no overlap with cohesin elsewhere. Genome-wide distribution of Gcn5p is compared with that of Sgop1 (magenta) and Mcd1p (blue). The trident Sgo1p localization domain in each chromosome is marked with the shaded boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Gcn5p is enriched in centromeres but shows no overlap with cohesin elsewhere. Genome-wide distribution of Gcn5p is compared with that of Sgop1 (magenta) and Mcd1p (blue). The trident Sgo1p localization domain in each chromosome is marked with the shaded boxes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F4.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/9/2901/F4.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490826" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-21" class="first-child">Gcn5p is enriched in centromeres but shows no overlap with cohesin elsewhere. Genome-wide distribution of Gcn5p is compared with that of Sgop1 (magenta) and Mcd1p (blue). The trident Sgo1p localization domain in each chromosome is marked with the shaded boxes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The cohesin complex is required for chromatin association of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in both budding yeast and human (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Kiburz <em>et al.</em>, 2005</a>, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Liu <em>et al.</em>, 2015</a>). To further confirm that the highly specific centromeric and pericentric localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> requires the local cohesin populations, we took two approaches. First, we deleted <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">IML3</a></em> that encodes a subunit of the Ctf19 kinetochore subcomplex. ChIP and quantitative PCR analysis shows that this manipulation disrupts only the pericentric, but not chromosome arm recruitment of cohesin (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Kiburz <em>et al.</em> 2005</a>) (Figure S3A). As predicted, the pericentric <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> enrichment in chromosome XVI is completely lost in <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a>∆</em> cells (Figure S3B). In the second approach, we targeted a specific cohesin associated region (CAR) on chromosome IV for inducible disruption. Active transcription can dislodge cohesin enrichment (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Glynn <em>et al.</em>, 2004</a>). Accordingly, we replaced the pericentric CAR between <a href="http://www.yeastgenome.org/locus/S000002411/overview">YDR004W</a> and <a href="http://www.yeastgenome.org/locus/S000002412/overview">YDR005C</a> to a galactose-inducible promoter <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a></em> (<em>pGAL1</em>, Figure S4). Changing from a non-inducing (raffinose) to an inducing (galactose) condition caused transcription-driven removal of both cohesin and <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> (Figure S4).</p><p id="p-23">From data presented in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> and Supplemental Figures 3 and 4, we conclude that the tripartite localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in each chromosome depends on an intact tension sensing motif and likely is established at pre-existing or concomitantly with cohesin localization domains.</p></div><div id="sec-9" class="subsection"><h3>Pericentric Sgo1p domain formation does not appear to involve intervening valley regions</h3><p id="p-24"><a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> docks on centromeres via direct association with <a href="http://www.yeastgenome.org/locus/S000003420/overview">Bub1p</a>-phosphorylated Ser121 of histone H2A (phos.H2A) within the single centromeric nucleosome (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">K<span class="sc">awashima</span><em>et al.</em> 2010</a>). <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> also binds the N’ tail of the centromere-specific histone H3 variant, <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">M<span class="sc">ishra</span><em>et al.</em> 2017</a>). It is likely that phos.H2A and <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> provide the docking site for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> that nucleates outward spread toward the pericentric regions. The establishment of PC enrichment of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> may be accomplished by one of two mechanisms. In the rippling mode, a wave of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> spreads along the nucleosomal path before it stops and accumulates at the first cohesin block. Alternatively, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> “leaps” directly from centromeres to the PC region where it is retained by the tension sensing motif. In both modes, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is underrepresented at the region between the CEN and PC peaks, resulting in the “valleys” seen in the two-dimensional presentation of the ChIP-seq results. These two modes of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> recruitment can be differentiated by examining the dynamics of CEN and PC peaks emergence when cells progress through mitosis. An intermediate stage where a significant elevation of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> signals at the valley region before they move outward to generate the final PC peaks would support the rippling mode. To test these two models, we tagged <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> in the same strain to avoid any variation between cells with different genotypes. Cells expressing <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a>-6HA and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1</a>-13Myc were arrested in G1 phase by α factor. They were then released into the division cycle before collection at 30, 37.5, 45, 52.5, 60, 75, and 90 min after the release. Budding index revealed the timing of the progression through mitosis during the course of experiments (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). ChIP results (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a> and Figure S5) show that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> was first detectable at <em>CEN16</em> 37.5 min after release from G1 arrest, when cells were at the juncture of G1 and S phases. This coincides with the time when <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> expression starts (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">I<span class="sc">ndjeian</span><em>et al.</em> 2005</a>). While <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> centromeric abundance continued to rise, the adjacent PC peaks started to surface in the next 7.5 min (amplicons 3, 16, and 21). These signals culminated at T<sub>60’</sub> (green bars, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>) and progressively diminished at T<sub>75’</sub> T<sub>90’</sub>. Between T<sub>60’</sub> and T<sub>75’</sub>, approximately 20% of cells entered the anaphase (green sector, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a>), indicating that biorientation had been established in this population of cells. The concomitant reduction of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> signals is in excellent agreement with the tension-dependent removal of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> from the chromatin (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">N<span class="sc">erusheva</span><em>et al.</em> 2014</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamic recruitment of Sgo1p and cohesin at centromere and pericentromere through cell cycle. A. Budding index of cells collected from the indicated time points. B and C. Sgo1p-HA and Mcd1p-Myc co-expressed in the same cells were examined by ChIP-qPCR. PCR amplicons correspond to CEN16 and nearby regions. See Figure 3 for positions of these amplicons. In addition to the conspicuous difference in the IP efficiencies between Mcd1p-Myc and Sgo1p-HA (also see Figure 3), we noticed that the ChIP efficiency (%IP) of both Sgo1-6HA and Mcd1-13Myc in cells synchronously progressing through cell cycle was consistently 5- to 10-fold higher than in those benomyl-arrested cells. Results shown in this figure are a representative of two biological replicas. In both cases, the Mcd1p-13Myc exhibited significantly higher IP efficiency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dynamic recruitment of Sgo1p and cohesin at centromere and pericentromere through cell cycle. A. Budding index of cells collected from the indicated time points. B and C. Sgo1p-HA and Mcd1p-Myc co-expressed in the same cells were examined by ChIP-qPCR. PCR amplicons correspond to &lt;em&gt;CEN16&lt;/em&gt; and nearby regions. See Figure 3 for positions of these amplicons. In addition to the conspicuous difference in the IP efficiencies between Mcd1p-Myc and Sgo1p-HA (also see Figure 3), we noticed that the ChIP efficiency (%IP) of both Sgo1-6HA and Mcd1-13Myc in cells synchronously progressing through cell cycle was consistently 5- to 10-fold higher than in those benomyl-arrested cells. Results shown in this figure are a representative of two biological replicas. In both cases, the Mcd1p-13Myc exhibited significantly higher IP efficiency.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F5.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/9/2901/F5.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490767" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Dynamic recruitment of Sgo1p and cohesin at centromere and pericentromere through cell cycle. A. Budding index of cells collected from the indicated time points. B and C. Sgo1p-HA and Mcd1p-Myc co-expressed in the same cells were examined by ChIP-qPCR. PCR amplicons correspond to <em>CEN16</em> and nearby regions. See <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> for positions of these amplicons. In addition to the conspicuous difference in the IP efficiencies between Mcd1p-Myc and Sgo1p-HA (also see <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>), we noticed that the ChIP efficiency (%IP) of both Sgo1-6HA and Mcd1-13Myc in cells synchronously progressing through cell cycle was consistently 5- to 10-fold higher than in those benomyl-arrested cells. Results shown in this figure are a representative of two biological replicas. In both cases, the Mcd1p-13Myc exhibited significantly higher IP efficiency.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The kinetics of <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> association with CEN and PC exhibited several important distinctions. First, while <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> signals jumped at T30’, the three subsequent time points (T37.5′, T45’ and T52.5′) saw a reduction of the overall <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> signals, which then climbed up again, and peaked at T75’ before abrupt disappearance by T90’, when the majority of cells passed the metaphase-to-anaphase transition (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a>). The dynamic changes before T60’ probably resulted from transcriptional activities in S and G2 phases. The abrupt increase of <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> signal at T60’ agreed well with the budding index that 80% of the cells were in the metaphase when cohesion of sister chromatids was most critical. Lastly, the highest levels of the <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> abundance were found to be at T75’ before its quick disappearance by T90’, both were 15’ later than <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>. The different kinetics of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> dissolution concurs with the anticipated sequence of biorientation, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> removal, and <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> cleavage that marks anaphase onset.</p><p id="p-27">Similar to results shown in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> tagged with 13 tandem copies of Myc tag exhibited significantly higher IP efficiency than <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>-6HA. The IP efficiency of <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a>-13Myc became even higher in synchronous cultures, suggesting that the cohesin complex plays a major structural role in shaping the pericentric portion of mitotic chromosomes.</p><p id="p-28">One critical observation from results in <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a> is that during the formation of the <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> CEN and PC tripartite motif, the two valleys flanking the CEN peak never rose to the levels of PC at any given time. While we cannot formally rule out the possibility that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is pushed along the nucleosomal array between CEN and PC peaks at a rate that is significantly faster than the 7.5-minute interval for ChIP assays, given the stochastic nature of cellular physiology even in a synchronized population (see budding index, <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5A</a>), the lack of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> signal at these valley regions favors the notion that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> spreads from CEN by a “hopping” mechanism to PC, or is recruited simultaneously to CEN and PC to generate the tripartite motif. This conclusion is also consistent with the existence of higher-ordered chromatin architecture near the centromeres as shown below.</p></div><div id="sec-10" class="subsection"><h3>Chromosome conformation capture reveals correlation Between Sgo1p enrichment and chromatin architecture</h3><p id="p-29">If <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> targets its pericentric destination immediately after or concomitantly with the centromeric recruitment, it is likely that the PC regions are rendered accessible to <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> whereas the intervening regions are somehow hidden from <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>. Because the interaction between <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and TSM does not require any posttranslational modification (<a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>; <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>), a non-epigenetic feature may distinguish the PC <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> targets from other areas nearby. We felt that chromatin architecture would be a good candidate that dictates the (in)accessibility of the CEN/PC region to <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>. Compaction of chromatin in mitosis involves condensin and cohesin complexes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">H<span class="sc">udson</span><em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">M<span class="sc">ehta</span><em>et al.</em> 2013</a>). Both complexes are also shown to be critical for organizing pericentromere in prometaphase (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Y<span class="sc">eh</span><em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">N<span class="sc">asmyth</span> 2011</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">S<span class="sc">tephens</span><em>et al.</em> 2011</a>). Cohesin facilitates the formation of intrachromosomal centromeric loops for mitotic segregation and resides near the summits of these loops. On the other hand, the condensin complex holds and organizes the bottom of these loops along the spindle axis (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">S<span class="sc">tephens</span><em>et al.</em> 2011</a>). Taking together these models and our results shown above, we suspect that higher-ordered chromosomal architecture, <em>e.g.</em>, chromosome looping, might be part of the mechanism underlining the highly selective pericentric localization for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>.</p><p id="p-30">If <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> recruitment is linked to chromosome looping in mitosis, we predicted that PC and CEN peaks of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> were spatially near each other owing to the action of such complexes as cohesin and condensin. This hypothesis was tested by chromosome conformation capture (3C) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">D<span class="sc">ekker</span><em>et al.</em> 2002</a>). Yeast nuclei were harvested from G1 and G2/M arrest and were subjected to <em>Eco</em>R I digestion with or without formaldehyde fixation, followed by ligation under a condition that favored intramolecular ligation. The resultant DNA libraries were analyzed by PCR using one of two centromere-proximal anchor primers, oXD159 for <em>CEN1</em> and oXD162 for <em>CEN16</em>. In each quantitative PCR reaction, these anchor primers were paired with a distal primer that is 3 – 50 kb away (black arrows, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). All primers hybridized to the same strand of DNA, hence should not produce any PCR product without the 3C treatment. On the other hand, ligation at the anticipated <em>Eco</em>R I sites after formaldehyde fixation would generate templates amplifiable by the anchor and the locus-specific primers. Comparing the intensity of PCR products amplified from samples with or without formaldehyde treatment yielded “crosslinking frequency” that is indicative of the propensity for the two primer target regions to be spatially brought together by chromatin-associating factors.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sgo1p tripartite localization domain is associated with high-ordered chromatin architecture in mitosis. Chromosome conformation capture (3C) assay was used to examine chromatin looping near CEN1 and CEN16. Cells arrested in G1 or G2/M phase were fixed with formaldehyde, and the isolated nuclei treated with Eco RI before DNA ligation. An identical amount of final ligated DNA library was amplified by PCR using one of two common anchor primers (oXD159 and oXD162 for chromosomes I and XVI, respectively; red arrows) against different locus-specific primers (black arrows; named for their distance to the centromere, L = left; R = right) 3 &#x2013; 50 kb away. All primers face toward the same direction. PCR products were resolved by gel electrophoresis and quantified with the NIH Image J software. Shown are the signals relative to the same amplicons without formaldehyde crosslinking. Error bars are standard deviations from three biological replicas." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sgo1p tripartite localization domain is associated with high-ordered chromatin architecture in mitosis. Chromosome conformation capture (3C) assay was used to examine chromatin looping near &lt;em&gt;CEN1&lt;/em&gt; and &lt;em&gt;CEN16&lt;/em&gt;. Cells arrested in G1 or G2/M phase were fixed with formaldehyde, and the isolated nuclei treated with &lt;em&gt;Eco&lt;/em&gt; RI before DNA ligation. An identical amount of final ligated DNA library was amplified by PCR using one of two common anchor primers (oXD159 and oXD162 for chromosomes I and XVI, respectively; red arrows) against different locus-specific primers (black arrows; named for their distance to the centromere, L = left; R = right) 3 &#x2013; 50 kb away. All primers face toward the same direction. PCR products were resolved by gel electrophoresis and quantified with the NIH Image J software. Shown are the signals relative to the same amplicons without formaldehyde crosslinking. Error bars are standard deviations from three biological replicas.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F6.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/9/2901/F6.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490878" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">Sgo1p tripartite localization domain is associated with high-ordered chromatin architecture in mitosis. Chromosome conformation capture (3C) assay was used to examine chromatin looping near <em>CEN1</em> and <em>CEN16</em>. Cells arrested in G1 or G2/M phase were fixed with formaldehyde, and the isolated nuclei treated with <em>Eco</em> RI before DNA ligation. An identical amount of final ligated DNA library was amplified by PCR using one of two common anchor primers (oXD159 and oXD162 for chromosomes I and XVI, respectively; red arrows) against different locus-specific primers (black arrows; named for their distance to the centromere, L = left; R = right) 3 – 50 kb away. All primers face toward the same direction. PCR products were resolved by gel electrophoresis and quantified with the NIH Image J software. Shown are the signals relative to the same amplicons without formaldehyde crosslinking. Error bars are standard deviations from three biological replicas.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The 3C assays indeed show that, after crosslinking, the centromeric primers oXD159 and oXD162 could amplify with primers hybridizing to <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> peaks that were 3 to 15 kb away (<em>e.g.</em>, oXD159 + CEN1L 5kb or CEN1R 5kb, and oXD162 + CEN16L 8kb or CEN16R 3kb; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). Some of the amplification products spanned a region with a conspicuous <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> signal without <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> (<em>e.g.</em>, oXD159 + CEN1L 20kb, oXD162 + CEN16L 15kb), consistent with the idea that chromosomal loops generated by the cohesin complex is upstream to and a prerequisite for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> localization (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">S<span class="sc">tephens</span><em>et al.</em> 2011</a>; <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">V<span class="sc">erzijlbergen</span><em>et al.</em> 2014</a>). The crosslinking frequency from G2/M nuclei was in general higher than G1 (orange <em>vs.</em> blue bars), which indicates that the nuclear architecture climaxes during mitosis, but may be partially preserved after exiting from M phase. This notion is consistent with the weak but readily recognizable <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> peaks in cells arrested at G1 (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>).</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-33">This work captures high-resolution genome-wide localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in mitotic <em>S. cerevisiae</em> cells. On each chromosome, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> displays a tripartite localization domain consisting of a middle centromeric and typically two flanking pericentric peaks. Some chromosomes have a few extra peaks (<em>e.g.</em>, Chr.V) while the PC peaks of several others (<em>e.g.</em>, Chr. XII and XIII) are weaker. Presently, it is unclear what contributes to such differentiation. Because <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> co-localizes with the cohesin complex, and that cohesin localization is tied to transcription (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lengronne <em>et al.</em>, 2004</a>), we suspect that differential <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> intensities may be related to the transcription status of the local genes. Regardless of the reason, it is very clear that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is restricted to regions enriched for cohesin. Despite that cohesin is recruited to numerous loci across the genome, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> only rendezvouses with the centromeric and the adjacent pericentric cohesin. This confined localization of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> requires an intact tension sensing motif of histone H3. Ectopic transcription that disrupts pericentric cohesin localization also dislodges <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> <em>in situ</em>. Overexpression causes <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> to expand its presence, but the new <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> peaks have high propensity to co-localize with cohesin. This unique trident shape of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> domain on each chromosome appears to be associated with chromatin looping in mitosis, thus linking higher-ordered chromatin architecture to positioning <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> for the crucial tension sensing function of segregation.</p><p id="p-34">Studies of yeast and human cells have demonstrated the importance of cohesin in <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> recruitment to pericentromere (<a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">K<span class="sc">iburz</span><em>et al.</em> 2005</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">L<span class="sc">iu</span><em>et al.</em> 2013a</a>). However, cohesin alone is not sufficient for the pericentric retention of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>. The tension sensing motif of H3 is also required for keeping <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in this region to ensure error-free segregation. While a Gly-to-Ser mutation in the TSM has no effect on cohesin localization, both pericentric and centromeric (though to a lesser extent) enrichment of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is compromised ((<a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>) and <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3C</a>). The establishment of the centromeric and pericentric domain of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> likely follows a spillover model in that <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is first recruited to the centromeres via direct association with <a href="http://www.yeastgenome.org/locus/S000001532/overview">Cse4p</a> (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">M<span class="sc">ishra</span><em>et al.</em> 2017</a>) and histone H2A phosphorylated at Ser121 by kinase <a href="http://www.yeastgenome.org/locus/S000003420/overview">Bub1p</a> (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">F<span class="sc">ernius and </span>H<span class="sc">ardwick</span> 2007</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">K<span class="sc">awashima</span><em>et al.</em> 2010</a>). Congregation of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> molecules at centromeres permits its spread to the adjacent pericentric nucleosomes where cohesin has already been loaded. This spread may result from the turnover of a transient complex involving <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and centromeric proteins. Alternatively, the homodimerization activity of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, evidenced by yeast two-hybrid tests (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">M<span class="sc">ishra</span><em>et al.</em> 2017</a>), may facilitate the growth of the <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> domain from centromeres to pericentric regions where the cohesin complex resides. By binding to nucleosomes, cohesin may also help to make the tension sensing motif more accessible for <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> before biorientation is established (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">F<span class="sc">ernius and </span>H<span class="sc">ardwick</span> 2007</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">K<span class="sc">awashima</span><em>et al.</em> 2010</a>; <a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>; <a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>). Due possibly to the total pool size of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, it only spreads to the first and nearest cohesin cluster. Overexpression of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> can further its spread primarily to adjacent pre-existing cohesin conglomerates (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-35">The distinct kinetics of engaging <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and cohesin (<a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a>) at <em>CEN16</em> (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>) and <em>CEN1</em> (Figure S5) is consistent with the notion that cohesin organizes chromatin into a platform for mitotic machinery to execute error-free segregation. <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> appears earlier than <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> but fluctuates in abundance before metaphase. In the meantime, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> continues to accumulate at CEN and PC peaks until it reaches the maximum. When cells enter anaphase, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> dissipates. It is critical that before <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1p</a> levels climb to the highest, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> already starts disappearing from CEN and PC regions (compare T<sub>60’</sub> and T<sub>75’</sub>, <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a> and Figure S5). This time difference echoes the report of tension-dependent removal of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> from chromatin at the juncture of metaphase and anaphase (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">N<span class="sc">erusheva</span><em>et al.</em> 2014</a>), and is consistent with the model that the removal of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> from chromatin is registered by cells as achieving biorientation.</p><p id="p-36">The centromeric and pericentric clusters of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> appear almost simultaneously, leaving the intervening areas relatively free of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> throughout the lifespan of these peaks. The ChIP-qPCR data in <a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5</a> were obtained from synchronous cells collected every 7.5 min in the M phase. Despite the short intervals for sampling, it is formally possible that an exceedingly fast mechanism pushes <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> along the nucleosomes from CEN to PC. A high-precision, single-cell or fast-kinetics approach may provide a definitive answer. Results from the current resolution favor the hopping model for the establishment of PC peaks of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>.</p><p id="p-37">Considering that the histone H3 tension sensing motif decorates the whole genome and functions without a post-translational modification, the non-continuous nature of the confined <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> peaks on each chromosome strongly suggests physical hindrance in these <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>-free intervening sections. Our recent findings that <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> acts as a negative regulator for tension sensing motif and <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> functional interaction (<a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">B<span class="sc">uehl</span><em>et al.</em> 2018</a>) alludes to an intriguing possibility that <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a>, acetylated H3, or a downstream effector may prevent <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> from binding to the chromosome arms. ChIP-seq data show a lack of correlation between <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a> and these <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>-free valleys in mitosis (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>), arguing against a direct, physical role of <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a>. Rather, we favor the possibility that a structural feature dictates the accessibility of pericentric chromatin to <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>. Indeed, the chromosome conformation capture results (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>) show that the DNA around the centromere loops into a higher-ordered structure that includes centromere and the adjacent <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> and cohesin clusters, a scenario reminiscent of the C-loop model put forth by Bloom and colleagues (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Y<span class="sc">eh</span><em>et al.</em> 2008</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">S<span class="sc">almon and </span>B<span class="sc">loom</span> 2017</a>). The C-loop conformation posits that pericentric chromatin harbors alternating cohesin and condensin complex clusters. Condensin and the associated chromatin in pericentromere are restricted to the microtubule axis between spindle pole bodies, whereas cohesin and the cognate CARs are radially positioned, forming the wall of a barrel. In this model, multiple layers of chromatin loops distribute axially, with the top and bottom of this barrel being the clustered centromeres from all 16 chromosomes. Poleward pulling from biorientation stretches the length of this barrel and narrows its diameter.</p><p id="p-38">How does <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> fit into the tension sensing function? Taking together the ChIP-seq and 3C results, we suggest that cohesin is responsible for creating and joining multiple loops in pericentromere. With centromeres clustering in the center (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">J<span class="sc">in</span><em>et al.</em> 1998</a>), these cohesin-capped loops (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>) can be viewed as a series of concentric circles (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> is recruited to the centromere cluster, from which it encroaches radially to the first pericentric cohesin circle (red circles, <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). Biorientation instigates both intra- and inter-chromosomal tension (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">S<span class="sc">almon and </span>B<span class="sc">loom</span> 2017</a>). The increased space between individual nucleosomes causes a conformational change of the tension sensing motif (<a id="xref-ref-30-13" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>; <a id="xref-ref-29-9" class="xref-bibr" href="#ref-29">L<span class="sc">uo</span><em>et al.</em> 2016</a>) or even nucleosome dissociation from pericentromere (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">L<span class="sc">awrimore</span><em>et al.</em> 2015</a>). In either case, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> loses its footings and dissipates from chromatin (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a>, green circles). Tension-induced clearance of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> in pericentromere signals biorientation to the spindle assembly checkpoint (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">N<span class="sc">erusheva</span><em>et al.</em> 2014</a>). Anaphase thus ensues. This model provides a mechanistic explanation for the mitotic delay caused by <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> overexpression (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">C<span class="sc">lift</span><em>et al.</em> 2009</a>). Biochemical fractionation experiments demonstrated that yeast cells do not seem to have a soluble pool of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a>, but rather keep all <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> molecules in the CEN/PC region (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">B<span class="sc">uehl</span><em>et al.</em> 2018</a>). If true, the overall size of the <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> motif on chromosomes (red circles, <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>) would be dictated by the number of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> molecules. Overexpression raises <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> levels and expands the range of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> occupancy to the next cohesin circle farther from the centromere cluster. Consequently, more extended axial separation of kinetochores is required in order to evict the outermost <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> molecules. Assuming that the quantitative removal of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> from centromeric and pericentric regions signals biorientation, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> overdose would require more time to clear <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> before anaphase onset, resulting in mitotic delay. On the contrary, deleting <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> or preventing the formation of the pericentric <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1p</a> domain by mutating the tension sensing motif would be interpreted erroneously as biorientation by cells, thus triggering precocious anaphase onset and aneuploidy (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">I<span class="sc">ndjeian</span><em>et al.</em> 2005</a>; <a id="xref-ref-30-14" class="xref-bibr" href="#ref-30">L<span class="sc">uo</span><em>et al.</em> 2010</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model for the formation and dynamics of Sgo1p chromatin domain. A. Sgo1p is first recruited to the centromeres via association with phosphorylated histone H2A (Pi). Centromere-bound Sgo1p then spreads to the nearby cohesin-occupied region. B. At the whole genome level, congregation of centromeres aligns the adjacent chromatin loops to form concentric rings (gradient yellow circle) that become the two terminals of the chromatin column. Prior to biorientation, Sgo1p (gradient red circle) resides on the centromere cluster and the first ring of chromatin loops. Poleward pulling from bipolar attachment stretches the centromeric and pericentric chromatin, resulting in a conformational change (gradient green circle) and evicting Sgo1p." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031986543" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model for the formation and dynamics of Sgo1p chromatin domain. A. Sgo1p is first recruited to the centromeres via association with phosphorylated histone H2A (Pi). Centromere-bound Sgo1p then spreads to the nearby cohesin-occupied region. B. At the whole genome level, congregation of centromeres aligns the adjacent chromatin loops to form concentric rings (gradient yellow circle) that become the two terminals of the chromatin column. Prior to biorientation, Sgo1p (gradient red circle) resides on the centromere cluster and the first ring of chromatin loops. Poleward pulling from bipolar attachment stretches the centromeric and pericentric chromatin, resulting in a conformational change (gradient green circle) and evicting Sgo1p.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2901/F7.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/9/2901/F7.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2901/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490912" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-39" class="first-child">Model for the formation and dynamics of Sgo1p chromatin domain. A. Sgo1p is first recruited to the centromeres via association with phosphorylated histone H2A (Pi). Centromere-bound Sgo1p then spreads to the nearby cohesin-occupied region. B. At the whole genome level, congregation of centromeres aligns the adjacent chromatin loops to form concentric rings (gradient yellow circle) that become the two terminals of the chromatin column. Prior to biorientation, Sgo1p (gradient red circle) resides on the centromere cluster and the first ring of chromatin loops. Poleward pulling from bipolar attachment stretches the centromeric and pericentric chromatin, resulting in a conformational change (gradient green circle) and evicting Sgo1p.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We are grateful for the technical assistance from Monique Floer, Alison Gjidoda, Mohita Tagore, Michael McAndrew, Kurtus Kok, and Sandhya Payankaulam. We thank Christopher Buehl for his critical reading of this manuscript and frequent discussion of the project. This work was supported the National Science Foundation (MCB1050132) and partly by National Institutes of Health (AG051820) to M.-H. Kuo. XD was also supported by a Thesis Completion Scholarship by the Michigan State University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-41">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6227180">https://doi.org/10.25387/g3.6227180</a>.</p></li><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: J. Berman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-04-26"><span class="received-label">Received </span>April 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-02"><span class="accepted-label">Accepted </span>July 2, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Deng, Kuo</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.1" data-doi="10.1002/yea.841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akada</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirosawa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawahata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoshida</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishizawa</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Sets of integrating plasmids and gene disruption cassettes containing improved counter-selection markers designed for repeated use in budding yeast.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.841</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DAkada%26rft.auinit1%253DR.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D393%26rft.epage%253D402%26rft.atitle%253DSets%2Bof%2Bintegrating%2Bplasmids%2Band%2Bgene%2Bdisruption%2Bcassettes%2Bcontaining%2Bimproved%2Bcounter-selection%2Bmarkers%2Bdesigned%2Bfor%2Brepeated%2Buse%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.841%26rft_id%253Dinfo%253Apmid%252F11921088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11921088&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174648000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.2" data-doi="10.1101/pdb.prot085175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromosome Conformation Capture (3C) in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Protoc.</abbr> <span class="cit-vol">2015</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/pdb.prot085175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BProtoc.%26rft.volume%253D2015%26rft.spage%253D580%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.prot085175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/pdb.prot085175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.3" data-doi="10.1016/S0960-9822(00)00515-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardwick</span>  <span class="cit-name-given-names">K. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Complex formation between Mad1p, Bub1p and Bub3p is crucial for spindle checkpoint function.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0960-9822(00)00515-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBrady%26rft.auinit1%253DD.%2BM.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D675%26rft.epage%253D678%26rft.atitle%253DComplex%2Bformation%2Bbetween%2BMad1p%252C%2BBub1p%2Band%2BBub3p%2Bis%2Bcrucial%2Bfor%2Bspindle%2Bcheckpoint%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252800%252900515-7%26rft_id%253Dinfo%253Apmid%252F10837255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(00)00515-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10837255&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088978600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.4" data-doi="10.1534/genetics.117.300606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buehl</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buranasudja</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazbun</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A Failsafe for Sensing Chromatid Tension in Mitosis with the Histone H3 Tail in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300606%26rft_id%253Dinfo%253Apmid%252F29242290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA4LzIvNTY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.5" data-doi="10.1016/S0092-8674(03)00115-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Centromeres and kinetochores: from epigenetics to mitotic checkpoint signaling.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(03)00115-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCleveland%26rft.auinit1%253DD.%2BW.%26rft.volume%253D112%26rft.issue%253D4%26rft.spage%253D407%26rft.epage%253D421%26rft.atitle%253DCentromeres%2Band%2Bkinetochores%253A%2Bfrom%2Bepigenetics%2Bto%2Bmitotic%2Bcheckpoint%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252900115-6%26rft_id%253Dinfo%253Apmid%252F12600307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)00115-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12600307&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181252600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.6" data-doi="10.1101/gad.507509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clift</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bizzari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Shugoshin prevents cohesin cleavage by PP2A(Cdc55)-dependent inhibition of separase.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">766</span>–<span class="cit-lpage">780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.507509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DClift%26rft.auinit1%253DD.%26rft.volume%253D23%26rft.issue%253D6%26rft.spage%253D766%26rft.epage%253D780%26rft.atitle%253DShugoshin%2Bprevents%2Bcohesin%2Bcleavage%2Bby%2BPP2ACdc55-dependent%2Binhibition%2Bof%2Bseparase%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.507509%26rft_id%253Dinfo%253Apmid%252F19299562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjMvNi83NjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.7" data-doi="10.1126/science.1067799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rippe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Capturing chromosome conformation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">295</span>: <span class="cit-fpage">1306</span>–<span class="cit-lpage">1311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1067799</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDekker%26rft.auinit1%253DJ.%26rft.volume%253D295%26rft.issue%253D5558%26rft.spage%253D1306%26rft.epage%253D1311%26rft.atitle%253DCapturing%2BChromosome%2BConformation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1067799%26rft_id%253Dinfo%253Apmid%252F11847345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTUvNTU1OC8xMzA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.8" data-doi="10.1242/jcs.161273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eshleman</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">D. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sgo1 recruits PP2A to chromosomes to ensure sister chromatid bi-orientation during mitosis.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">4974</span>–<span class="cit-lpage">4983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jcs.161273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.161273%26rft_id%253Dinfo%253Apmid%252F25236599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyNy8yMi80OTc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.9" data-doi="10.1371/journal.pgen.0030213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardwick</span>  <span class="cit-name-given-names">K. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bub1 kinase targets Sgo1 to ensure efficient chromosome biorientation in budding yeast mitosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e213</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0030213</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFernius%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De213%26rft.epage%253De213%26rft.atitle%253DBub1%2Bkinase%2Btargets%2BSgo1%2Bto%2Bensure%2Befficient%2Bchromosome%2Bbiorientation%2Bin%2Bbudding%2Byeast%2Bmitosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030213%26rft_id%253Dinfo%253Apmid%252F18081426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18081426&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.10" data-doi="10.1186/1756-0500-7-312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikopoulou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kokkalis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thanos</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A method for generating highly multiplexed ChIP-seq libraries.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1756-0500-7-312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes.%2BNotes%26rft.volume%253D7%26rft.spage%253D312%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-7-312%26rft_id%253Dinfo%253Apmid%252F24885602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-7-312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24885602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.11" data-doi="10.1093/nar/20.6.1425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Improved method for high efficiency transformation of intact yeast cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/20.6.1425</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGietz%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D6%26rft.spage%253D1425%26rft.epage%253D1425%26rft.atitle%253DImproved%2Bmethod%2Bfor%2Bhigh%2Befficiency%2Btransformation%2Bof%2Bintact%2Byeast%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.6.1425%26rft_id%253Dinfo%253Apmid%252F1561104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/20.6.1425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1561104&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HM42000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.12" data-doi="10.1371/journal.pbio.0020259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glynn</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistrot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">E259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0020259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGlynn%26rft.auinit1%253DE.%2BF.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253DE259%26rft.epage%253DE259%26rft.atitle%253DGenome-wide%2Bmapping%2Bof%2Bthe%2Bcohesin%2Bcomplex%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020259%26rft_id%253Dinfo%253Apmid%252F15309048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15309048&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.13" data-doi="10.1016/S0092-8674(00)80888-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asthana</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorger</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Transient sister chromatid separation and elastic deformation of chromosomes during mitosis in budding yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">775</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80888-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHe%26rft.auinit1%253DX.%26rft.volume%253D101%26rft.issue%253D7%26rft.spage%253D763%26rft.epage%253D775%26rft.atitle%253DTransient%2Bsister%2Bchromatid%2Bseparation%2Band%2Belastic%2Bdeformation%2Bof%2Bchromosomes%2Bduring%2Bmitosis%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980888-0%26rft_id%253Dinfo%253Apmid%252F10892747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80888-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10892747&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087821400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.14" data-doi="10.1016/S0092-8674(01)00438-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rines</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espelin</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorger</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular analysis of kinetochore-microtubule attachment in budding yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(01)00438-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHe%26rft.auinit1%253DX.%26rft.volume%253D106%26rft.issue%253D2%26rft.spage%253D195%26rft.epage%253D206%26rft.atitle%253DMolecular%2Banalysis%2Bof%2Bkinetochore-microtubule%2Battachment%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900438-X%26rft_id%253Dinfo%253Apmid%252F11511347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00438-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11511347&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170193100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.15" data-doi="10.1007/s10577-008-9009-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Earnshaw</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Condensin: Architect of mitotic chromosomes.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-008-9009-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DHudson%26rft.auinit1%253DD.%2BF.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D131%26rft.epage%253D144%26rft.atitle%253DCondensin%253A%2BArchitect%2Bof%2Bmitotic%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-008-9009-7%26rft_id%253Dinfo%253Apmid%252F19308696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-008-9009-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19308696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264515300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.16" data-doi="10.1126/science.1101366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Indjeian</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The centromeric protein Sgo1 is required to sense lack of tension on mitotic chromosomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1101366</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DIndjeian%26rft.auinit1%253DV.%2BB.%26rft.volume%253D307%26rft.issue%253D5706%26rft.spage%253D130%26rft.epage%253D133%26rft.atitle%253DThe%2BCentromeric%2BProtein%2BSgo1%2BIs%2BRequired%2Bto%2BSense%2BLack%2Bof%2BTension%2Bon%2BMitotic%2BChromosomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101366%26rft_id%253Dinfo%253Apmid%252F15637284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcwNi8xMzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.17" data-doi="10.1016/j.jmb.2017.09.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isaac</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanulli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibble</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornsby</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravalin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Biochemical Basis for Distinct Roles of the Heterochromatin Proteins Swi6 and Chp2.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">429</span>: <span class="cit-fpage">3666</span>–<span class="cit-lpage">3677</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2017.09.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D429%26rft.spage%253D3666%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2017.09.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2017.09.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.18" data-doi="10.1083/jcb.141.1.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trelles-Sticken</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherthan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Yeast nuclei display prominent centromere clustering that is reduced in nondividing cells and in meiotic prophase.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.141.1.21</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.141.1.21%26rft_id%253Dinfo%253Apmid%252F9531545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE0MS8xLzIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.19" data-doi="10.1126/science.1180189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamagishi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishiguro</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phosphorylation of H2A by Bub1 prevents chromosomal instability through localizing shugoshin.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1180189</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180189%26rft_id%253Dinfo%253Apmid%252F19965387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Mi8xNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.20" data-doi="10.1091/mbc.E07-06-0584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiburz</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Shugoshin promotes sister kinetochore biorientation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e07-06-0584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-06-0584%26rft_id%253Dinfo%253Apmid%252F18094053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE5LzMvMTE5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.21" data-doi="10.1101/gad.1373005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiburz</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The core centromere and Sgo1 establish a 50-kb cohesin-protected domain around centromeres during meiosis I.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3017</span>–<span class="cit-lpage">3030</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1373005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKiburz%26rft.auinit1%253DB.%2BM.%26rft.volume%253D19%26rft.issue%253D24%26rft.spage%253D3017%26rft.epage%253D3030%26rft.atitle%253DThe%2Bcore%2Bcentromere%2Band%2BSgo1%2Bestablish%2Ba%2B50-kb%2Bcohesin-protected%2Bdomain%2Baround%2Bcentromeres%2Bduring%2Bmeiosis%2BI%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1373005%26rft_id%253Dinfo%253Apmid%252F16357219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzI0LzMwMTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.22" data-doi="10.1038/nature02312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitajima</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The conserved kinetochore protein shugoshin protects centromeric cohesion during meiosis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">427</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">517</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKitajima%26rft.auinit1%253DT.%2BS.%26rft.volume%253D427%26rft.issue%253D6974%26rft.spage%253D510%26rft.epage%253D517%26rft.atitle%253DThe%2Bconserved%2Bkinetochore%2Bprotein%2Bshugoshin%2Bprotects%2Bcentromeric%2Bcohesion%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02312%26rft_id%253Dinfo%253Apmid%252F14730319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14730319&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188721800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.23" data-doi="10.1006/meth.1999.0879"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allis</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">In vivo cross-linking and immunoprecipitation for studying dynamic Protein:DNA associations in a chromatin environment.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/meth.1999.0879</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DKuo%26rft.auinit1%253DM.%2BH.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D425%26rft.epage%253D433%26rft.atitle%253DIn%2Bvivo%2Bcross-linking%2Band%2Bimmunoprecipitation%2Bfor%2Bstudying%2Bdynamic%2BProtein%253ADNA%2Bassociations%2Bin%2Ba%2Bchromatin%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.1999.0879%26rft_id%253Dinfo%253Apmid%252F10579938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.1999.0879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10579938&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084168300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.24" data-doi="10.1083/jcb.201502046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrimore</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasquez</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falvo</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">R. M.</span>  <span class="cit-name-suffix">2nd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vicci</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA loops generate intracentromere tension in mitosis.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.201502046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201502046%26rft_id%253Dinfo%253Apmid%252F26283798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIxMC80LzU1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.25" data-doi="10.1038/nature02742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lengronne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yokobayashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Cohesin relocation from sites of chromosomal loading to places of convergent transcription.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLengronne%26rft.auinit1%253DA.%26rft.volume%253D430%26rft.issue%253D6999%26rft.spage%253D573%26rft.epage%253D578%26rft.atitle%253DCohesin%2Brelocation%2Bfrom%2Bsites%2Bof%2Bchromosomal%2Bloading%2Bto%2Bplaces%2Bof%2Bconvergent%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02742%26rft_id%253Dinfo%253Apmid%252F15229615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229615&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222946100051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.26" data-doi="10.1016/j.cub.2013.07.078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Phospho-H2A and cohesin specify distinct tension-regulated Sgo1 pools at kinetochores and inner centromeres.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1927</span>–<span class="cit-lpage">1933</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2013.07.078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253D1927%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.07.078%26rft_id%253Dinfo%253Apmid%252F24055156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.07.078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24055156&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.27" data-doi="10.1016/j.molcel.2015.06.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warrington</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mitotic Transcription Installs Sgo1 at Centromeres to Coordinate Chromosome Segregation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">426</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2015.06.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D59%26rft.spage%253D426%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.06.018%26rft_id%253Dinfo%253Apmid%252F26190260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.06.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26190260&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.28" data-doi="10.1038/ncb2637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rankin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Phosphorylation-enabled binding of SGO1–PP2A to cohesin protects sororin and centromeric cohesion during mitosis.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncb2637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCell%2BBiol.%26rft.volume%253D15%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2637%26rft_id%253Dinfo%253Apmid%252F23242214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23242214&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312841300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.29" data-doi="10.1534/genetics.116.192443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buehl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of Tension Sensing Motif of Histone H3 in Saccharomyces cerevisiae and Its Regulation by Histone Modifying Enzymes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1043</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.192443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.192443%26rft_id%253Dinfo%253Apmid%252F27672091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzEwMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.30" data-doi="10.1128/MCB.00980-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyland</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Histone h3 exerts a key function in mitotic checkpoint control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.00980-09</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00980-09%26rft_id%253Dinfo%253Apmid%252F19917722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjMwLzIvNTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.31" data-doi="10.1128/MCB.01176-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Shugoshins: tension-sensitive pericentromeric adaptors safeguarding chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">634</span>–<span class="cit-lpage">648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.01176-14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01176-14%26rft_id%253Dinfo%253Apmid%252F25452306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjM1LzQvNjM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.32" data-doi="10.1016/j.febslet.2013.06.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cohesin: functions beyond sister chromatid cohesion.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">587</span>: <span class="cit-fpage">2299</span>–<span class="cit-lpage">2312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.febslet.2013.06.035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLett.%26rft.volume%253D587%26rft.spage%253D2299%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2013.06.035%26rft_id%253Dinfo%253Apmid%252F23831059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2013.06.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23831059&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322606000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-8.9.2901.33" data-doi="10.1080/15384101.2017.1380129"><div class="cit-metadata"><cite>Mishra, P. K., K. S. Thapa, P. Chen, S. Wang, T. R. Hazbun <em>et al.</em>, 2017 <span class="cit-comment">Budding yeast CENP-A(Cse4) interacts with the N-terminus of Sgo1 and regulates its association with centromeric chromatin. Cell Cycle: 0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15384101.2017.1380129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D17%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15384101.2017.1380129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15384101.2017.1380129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.34" data-doi="10.1038/ncb2349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cohesin: a catenase with separate entry and exit gates?</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1170</span>–<span class="cit-lpage">1177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncb2349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DNasmyth%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D1170%26rft.epage%253D1177%26rft.atitle%253DCohesin%253A%2Ba%2Bcatenase%2Bwith%2Bseparate%2Bentry%2Band%2Bexit%2Bgates%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2349%26rft_id%253Dinfo%253Apmid%252F21968990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21968990&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295617900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.35" data-doi="10.1101/gad.240291.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nerusheva</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galander</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tension-dependent removal of pericentromeric shugoshin is an indicator of sister chromosome biorientation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.240291.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DNerusheva%26rft.auinit1%253DO.%2BO.%26rft.volume%253D28%26rft.issue%253D12%26rft.spage%253D1291%26rft.epage%253D1309%26rft.atitle%253DTension-dependent%2Bremoval%2Bof%2Bpericentromeric%2Bshugoshin%2Bis%2Ban%2Bindicator%2Bof%2Bsister%2Bchromosome%2Bbiorientation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.240291.114%26rft_id%253Dinfo%253Apmid%252F24939933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI4LzEyLzEyOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.36" data-doi="10.1016/S0076-6879(02)50978-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petracek</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">PCR-based engineering of yeast genome.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">469</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0076-6879(02)50978-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DPetracek%26rft.auinit1%253DM.%2BE.%26rft.volume%253D350%26rft.spage%253D445%26rft.epage%253D469%26rft.atitle%253DPCR-based%2Bengineering%2Bof%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252802%252950978-2%26rft_id%253Dinfo%253Apmid%252F12073329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(02)50978-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12073329&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176466300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.37" data-doi="10.1016/j.tcb.2005.07.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinsky</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The spindle checkpoint: tension <em>vs.</em> attachment.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">493</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tcb.2005.07.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DPinsky%26rft.auinit1%253DB.%2BA.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D486%26rft.epage%253D493%26rft.atitle%253DThe%2Bspindle%2Bcheckpoint%253A%2Btension%2Bversus%2Battachment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2005.07.005%26rft_id%253Dinfo%253Apmid%252F16084093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2005.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16084093&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232333700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.38" data-doi="10.1016/j.tig.2008.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricke</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ree</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Deursen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Whole chromosome instability and cancer: a complex relationship.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2008.07.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRicke%26rft.auinit1%253DR.%2BM.%26rft.volume%253D24%26rft.issue%253D9%26rft.spage%253D457%26rft.epage%253D466%26rft.atitle%253DWhole%2Bchromosome%2Binstability%2Band%2Bcancer%253A%2Ba%2Bcomplex%2Brelationship.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.07.002%26rft_id%253Dinfo%253Apmid%252F18675487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18675487&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259460900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.39" data-doi="10.1002/bies.201600216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Tension sensors reveal how the kinetochore shares its load.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">39</span> 7, 1600216.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201600216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D39%26rft.spage%253D1600216%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201600216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201600216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.40" data-doi="10.1016/S1046-2023(05)80158-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Getting started with yeast.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0076-6879(91)94004-V</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DSherman%26rft.auinit1%253DF.%26rft.volume%253D194%26rft.spage%253D3%26rft.epage%253D21%26rft.atitle%253DGetting%2Bstarted%2Bwith%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252805%252980158-6%26rft_id%253Dinfo%253Apmid%252F2005794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(05)80158-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005794&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FN84200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.41" data-doi="10.1083/jcb.201103138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vicci</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">R. M.</span>  <span class="cit-name-suffix">2nd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cohesin, condensin, and the intramolecular centromere loop together generate the mitotic chromatin spring.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1167</span>–<span class="cit-lpage">1180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.201103138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201103138%26rft_id%253Dinfo%253Apmid%252F21708976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOTMvNy8xMTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.42" data-doi="10.1016/j.molcel.2011.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venters</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavrich</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jena</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">492</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2011.01.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DVenters%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D480%26rft.epage%253D492%26rft.atitle%253DA%2Bcomprehensive%2Bgenomic%2Bbinding%2Bmap%2Bof%2Bgene%2Band%2Bchromatin%2Bregulatory%2Bproteins%2Bin%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.01.015%26rft_id%253Dinfo%253Apmid%252F21329885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329885&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287773300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.43" data-doi="10.1128/MCB.01366-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vernarecci</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornaghi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bagu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cundari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballario</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gcn5p plays an important role in centromere kinetochore function in budding yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">996</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.01366-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01366-07%26rft_id%253Dinfo%253Apmid%252F18039853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI4LzMvOTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.44" data-doi="10.7554/eLife.01374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verzijlbergen</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nerusheva</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clift</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Shugoshin biases chromosomes for biorientation through condensin recruitment to the pericentromere.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e01374</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.01374</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D3%26rft.spage%253De01374%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01374%26rft_id%253Dinfo%253Apmid%252F24497542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.01374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24497542&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.45" data-doi="10.1093/emboj/20.18.5207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suto</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">5207</span>–<span class="cit-lpage">5218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/emboj/20.18.5207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DWhite%26rft.auinit1%253DC.%2BL.%26rft.volume%253D20%26rft.issue%253D18%26rft.spage%253D5207%26rft.epage%253D5218%26rft.atitle%253DStructure%2Bof%2Bthe%2Byeast%2Bnucleosome%2Bcore%2Bparticle%2Breveals%2Bfundamental%2Bchanges%2Bin%2Binternucleosome%2Binteractions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.18.5207%26rft_id%253Dinfo%253Apmid%252F11566884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTgvNTIwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.46" data-doi="10.1038/nature07217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamagishi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakuno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heterochromatin links to centromeric protection by recruiting shugoshin.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07217</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYamagishi%26rft.auinit1%253DY.%26rft.volume%253D455%26rft.issue%253D7210%26rft.spage%253D251%26rft.epage%253D255%26rft.atitle%253DHeterochromatin%2Blinks%2Bto%2Bcentromeric%2Bprotection%2Bby%2Brecruiting%2Bshugoshin.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07217%26rft_id%253Dinfo%253Apmid%252F18716626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716626&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259090800053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2901.47" data-doi="10.1016/j.cub.2007.12.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paliulis</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joglekar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pericentric chromatin is organized into an intramolecular loop in mitosis.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2007.12.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DYeh%26rft.auinit1%253DE.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D81%26rft.epage%253D90%26rft.atitle%253DPericentric%2Bchromatin%2Bis%2Borganized%2Binto%2Ban%2Bintramolecular%2Bloop%2Bin%2Bmitosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.12.019%26rft_id%253Dinfo%253Apmid%252F18211850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.12.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18211850&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2901.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252693900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/2901.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2889" title="Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2913" title="Disruption of INOS, a Gene Encoding myo-Inositol Phosphate Synthase, Causes Male Sterility in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2901" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-x2u-7cw-KpVyAF0VCXW3s6iPdLJSn60UtdXKf5iRMdc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/2901" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/2901" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/2901" class="active" data-icon-position="" data-hide-link-title="0">Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490737" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZUULLH--o05Rpgd5ose4ZAYJoJkIIR-cYIY3TcPs_EY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2901.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/2901" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lFcVFnUGxw9_ENIN68qILGkrDZ2M4OMbDYHK3aYCS-o" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/2901" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2901.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490737" id="node490737--2" data-pisa="ggg;8/9/2901" data-pisa-master="ggg;g3.118.200522" data-apath="/ggg/8/9/2901.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0025-5383" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiexiong Deng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8096-5344" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Min-Hao Kuo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2901-2911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200522 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490737--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiexiong Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiexiong%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiexiong%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0025-5383" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiexiong Deng</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Min-Hao Kuo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min-Hao%2BKuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMin-Hao%2BKuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8096-5344" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Min-Hao Kuo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kuom@msu.edu" class="" data-icon-position="" data-hide-link-title="0">kuom@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/2901" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/2901&amp;title=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/2901&amp;title=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/2901&amp;title=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/2901&amp;text=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/2901&amp;title=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/2901&amp;t=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/2901&amp;title=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/2901&amp;title=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490737/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490737" id="node490737--3" data-pisa="ggg;8/9/2901" data-pisa-master="ggg;g3.118.200522" data-apath="/ggg/8/9/2901.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0025-5383" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiexiong Deng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8096-5344" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Min-Hao Kuo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2901-2911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200522 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490737--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xiexiong Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiexiong%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiexiong%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0025-5383" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiexiong Deng</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Min-Hao Kuo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biochemistry and Molecular Biology, Michigan State University. East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min-Hao%2BKuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMin-Hao%2BKuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8096-5344" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Min-Hao Kuo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kuom@msu.edu" class="" data-icon-position="" data-hide-link-title="0">kuom@msu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2901&amp;rft.epage=2911&amp;rft.atitle=Tripartite%20Chromatin%20Localization%20of%20Budding%20Yeast%20Shugoshin%20Involves%20Higher-Ordered%20Architecture%20of%20Mitotic%20Chromosomes&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Deng%2C+Xiexiong&amp;rft.au=Kuo%2C+Min-Hao"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490737/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490737/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490737/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490737/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490737/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490737/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490737/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490737/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490737/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490737/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490737/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490737/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/2901" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/2901.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/2901.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/9/2901.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/9/2901.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
