<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/2923" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/2923.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/2923.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/9/2923.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/2923.atom" />
<meta name="HW.pisa" content="ggg;8/9/2923" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data" />
<meta name="DC.Identifier" content="10.1534/g3.118.200373" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Newman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6205793" />
<meta name="DC.Description" content="Alternative splicing leverages genomic content by allowing the synthesis of multiple transcripts and, by implication, protein isoforms, from a single gene. However, estimating the abundance of transcripts produced in a given tissue from short sequencing reads is difficult and can result in both the construction of transcripts that do not exist, and the failure to identify true transcripts. An alternative approach is to catalog the events that make up isoforms (splice junctions and exons). We present here the Event Analysis (EA) approach, where we project transcripts onto the genome and identify overlapping/unique regions and junctions. In addition, all possible logical junctions are assembled into a catalog. Transcripts are filtered before quantitation based on simple measures: the proportion of the events detected, and the coverage. We find that mapping to a junction catalog is more efficient at detecting novel junctions than mapping in a splice aware manner. We identify 99.8% of true transcripts while iReckon identifies 82% of the true transcripts and creates more transcripts not included in the simulation than were initially used in the simulation. Using PacBio Iso-seq data from a mouse neural progenitor cell model, EA detects 60% of the novel junctions that are combinations of existing exons while only 43% are detected by STAR. EA further detects ∼5,000 annotated junctions missed by STAR. Filtering transcripts based on the proportion of the transcript detected and the number of reads on average supporting that transcript captures 95% of the PacBio transcriptome. Filtering the reference transcriptome before quantitation, results in is a more stable estimate of isoform abundance, with improved correlation between replicates. This was particularly evident when EA is applied to an RNA-seq study of type 1 diabetes (T1D), where the coefficient of variation among subjects (n = 81) in the transcript abundance estimates was substantially reduced compared to the estimation using the full reference. EA focuses on individual transcriptional events. These events can be quantitate and analyzed directly or used to identify the probable set of expressed transcripts. Simple rules based on detected events and coverage used in filtering result in a dramatic improvement in isoform estimation without the use of ancillary data ( e.g. , ChIP, long reads) that may not be available for many studies." />
<meta name="DC.Contributor" content="Jeremy R. B. Newman" />
<meta name="DC.Contributor" content="Patrick Concannon" />
<meta name="DC.Contributor" content="Manuel Tardaguila" />
<meta name="DC.Contributor" content="Ana Conesa" />
<meta name="DC.Contributor" content="Lauren M. McIntyre" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Alternative splicing leverages genomic content by allowing the synthesis of multiple transcripts and, by implication, protein isoforms, from a single gene. However, estimating the abundance of transcripts produced in a given tissue from short sequencing reads is difficult and can result in both the construction of transcripts that do not exist, and the failure to identify true transcripts. An alternative approach is to catalog the events that make up isoforms (splice junctions and exons). We present here the Event Analysis (EA) approach, where we project transcripts onto the genome and identify overlapping/unique regions and junctions. In addition, all possible logical junctions are assembled into a catalog. Transcripts are filtered before quantitation based on simple measures: the proportion of the events detected, and the coverage. We find that mapping to a junction catalog is more efficient at detecting novel junctions than mapping in a splice aware manner. We identify 99.8% of true transcripts while iReckon identifies 82% of the true transcripts and creates more transcripts not included in the simulation than were initially used in the simulation. Using PacBio Iso-seq data from a mouse neural progenitor cell model, EA detects 60% of the novel junctions that are combinations of existing exons while only 43% are detected by STAR. EA further detects ∼5,000 annotated junctions missed by STAR. Filtering transcripts based on the proportion of the transcript detected and the number of reads on average supporting that transcript captures 95% of the PacBio transcriptome. Filtering the reference transcriptome before quantitation, results in is a more stable estimate of isoform abundance, with improved correlation between replicates. This was particularly evident when EA is applied to an RNA-seq study of type 1 diabetes (T1D), where the coefficient of variation among subjects (n = 81) in the transcript abundance estimates was substantially reduced compared to the estimation using the full reference. EA focuses on individual transcriptional events. These events can be quantitate and analyzed directly or used to identify the probable set of expressed transcripts. Simple rules based on detected events and coverage used in filtering result in a dramatic improvement in isoform estimation without the use of ancillary data (&lt;i&gt;e.g.&lt;/i&gt;, ChIP, long reads) that may not be available for many studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/2923" />
<meta name="citation_id" content="8/9/2923" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/2923" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/2923.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/2923.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/2923.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200373" />
<meta name="citation_pmid" content="30021829" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2923" />
<meta name="citation_lastpage" content="2940" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jeremy R. B. Newman" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8486-2801" />
<meta name="citation_author" content="Patrick Concannon" />
<meta name="citation_author_institution" content="Genetics Institute and Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida" />
<meta name="citation_author" content="Manuel Tardaguila" />
<meta name="citation_author_institution" content="Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences, University of Florida, Gainesville, Florida, 32610; Wellcome Trust Sanger Institute, Hinxton, United Kingdom" />
<meta name="citation_author" content="Ana Conesa" />
<meta name="citation_author_institution" content="Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences; Genetics Institute, University of Florida, Gainesville, Florida, 32610; Genomics of Gene Expression Lab, Prince Felipe Research Center, Valencia, Spain" />
<meta name="citation_author_email" content="aconesa@ufl.edu" />
<meta name="citation_author_email" content="mcintyre@ufl.edu" />
<meta name="citation_author" content="Lauren M. McIntyre" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida" />
<meta name="citation_author_email" content="aconesa@ufl.edu" />
<meta name="citation_author_email" content="mcintyre@ufl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0077-3359" />
<meta name="citation_reference" content="citation_journal_title=Mol. Vis.;citation_author=D. Akin;citation_author=J. R. B. Newman;citation_author=L. M. McIntyre;citation_author=S. P. Sugrue;citation_title=RNA-seq analysis of impact of PNN on gene expression and alternative splicing in corneal epithelial cells.;citation_pages=40-60;citation_volume=22;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Anders;citation_author=A. Reyes;citation_author=W. Huber;citation_title=Detecting differential usage of exons from RNA-seq data.;citation_volume=22;citation_year=2012;citation_pmid=22722343;citation_doi=10.1101/gr.133744.111" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. Angelini;citation_author=D. De Canditiis;citation_author=I. De Feis;citation_title=Computational approaches for isoform detection and estimation: good and bad news.;citation_pages=135;citation_volume=15;citation_year=2014;citation_pmid=24885830;citation_doi=10.1186/1471-2105-15-135" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. F. Au;citation_author=V. Sebastiano;citation_author=P. T. Afshar;citation_author=J. D. Durruthy;citation_author=L. Lee;citation_title=Characterization of the human ESC transcriptome by hybrid sequencing.;citation_volume=110;citation_year=2013;citation_pmid=24282307;citation_doi=10.1073/pnas.1320101110" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. C. Barrett;citation_author=D. G. Clayton;citation_author=P. Concannon;citation_author=B. Akolkar;citation_author=J. D. Cooper;citation_title=Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes.;citation_pages=703-707;citation_volume=41;citation_year=2009;citation_issue=6;citation_pmid=19430480;citation_doi=10.1038/ng.381" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=E. Bernard;citation_author=L. Jacob;citation_author=J. Mairal;citation_author=E. Viara;citation_author=J. P. Vert;citation_title=A convex formulation for joint RNA isoform detection and quantification from multiple RNA-seq samples.;citation_pages=262;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12859-015-0695-9" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=J. M. Bland;citation_author=D. G. Altman;citation_title=Statistical methods for assessing agreement between two methods of clinical measurement.;citation_pages=307-310;citation_volume=1;citation_year=1986;citation_issue=8476;citation_pmid=2868172;citation_doi=10.1016/S0140-6736(86)90837-8" />
<meta name="citation_reference" content="citation_journal_title=Clinical genetics;citation_journal_abbrev=Clin Genet;citation_author=P. Booms;citation_author=J. Cisler;citation_author=K. R. Mathews;citation_author=M. Godfrey;citation_author=F. Tiecke;citation_title=Novel exon skipping mutation in the fibrillin-1 gene: two 'hot spots' for the neonatal Marfan syndrome.;citation_pages=110-117;citation_volume=55;citation_year=1999;citation_issue=2;citation_pmid=10189088;citation_doi=10.1034/j.1399-0004.1999.550207.x" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Canzar;citation_author=S. Andreotti;citation_author=D. Weese;citation_author=K. Reinert;citation_author=G. W. Klau;citation_title=CIDANE: comprehensive isoform discovery and abundance estimation.;citation_pages=16;citation_volume=17;citation_year=2016;citation_doi=10.1186/s13059-015-0865-0" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=N. Cloonan;citation_author=A. R. R. Forrest;citation_author=G. Kolle;citation_author=B. B. A. Gardiner;citation_author=G. J. Faulkner;citation_title=Stem cell transcriptome profiling via massive-scale mRNA sequencing.;citation_pages=613-619;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516046;citation_doi=10.1038/nmeth.1223" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. E. Dalton;citation_author=J. M. Fear;citation_author=S. Knott;citation_author=B. S. Baker;citation_author=L. M. McIntyre;citation_title=Male-specific Fruitless isoforms have different regulatory roles conferred by distinct zinc finger DNA binding domains.;citation_pages=659;citation_volume=14;citation_year=2013;citation_pmid=24074028;citation_doi=10.1186/1471-2164-14-659" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genomics;citation_author=L. Z. Ding;citation_author=E. Rath;citation_author=Y. S. Bai;citation_title=Comparison of alternative splicing junction detection tools using RNA-seq data.;citation_pages=268-277;citation_volume=18;citation_year=2017;citation_doi=10.2174/1389202918666170215125048" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Djebali;citation_author=C. A. Davis;citation_author=A. Merkel;citation_author=A. Dobin;citation_author=T. Lassmann;citation_title=Landscape of transcription in human cells.;citation_pages=101-108;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955620;citation_doi=10.1038/nature11233" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Eswaran;citation_author=A. Horvath;citation_author=S. Godbole;citation_author=S. D. Reddy;citation_author=P. Mudvari;citation_title=RNA sequencing of cancer reveals novel splicing alterations.;citation_pages=1689;citation_volume=3;citation_year=2013;citation_pmid=23604310;citation_doi=10.1038/srep01689" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Fear;citation_author=L. G. Leon-Novelo;citation_author=A. M. Morse;citation_author=A. R. Gerken;citation_author=K. Van Lehmann;citation_title=Buffering of genetic regulatory networks in Drosophila melanogaster.;citation_volume=203;citation_year=2016;citation_pmid=27194752;citation_doi=10.1534/genetics.116.188797" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Florea;citation_author=G. Hartzell;citation_author=Z. Zhang;citation_author=G. M. Rubin;citation_author=W. Miller;citation_title=A Computer Program for Aligning a cDNA Sequence with a Genomic DNA Sequence;citation_pages=967-974;citation_volume=8;citation_year=1998;citation_issue=9;citation_pmid=9750195;citation_doi=10.1101/gr.8.9.967" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Foissac;citation_author=M. Sammeth;citation_title=ASTALAVISTA: dynamic and flexible analysis of alternative splicing events in custom gene datasets.;citation_volume=35;citation_year=2007;citation_pmid=17485470;citation_doi=10.1093/nar/gkm311" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. C. Frazee;citation_author=A. E. Jaffe;citation_author=B. Langmead;citation_author=J. T. Leek;citation_title=Polyester: simulating RNA-seq datasets with differential transcript expression.;citation_volume=31;citation_year=2015;citation_pmid=25926345;citation_doi=10.1093/bioinformatics/btv272" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=N. A. Gabreski;citation_author=J. K. Vaghasia;citation_author=S. S. Novakova;citation_author=N. Q. McDonald;citation_author=B. A. Pierchala;citation_title=Exon Skipping in the RET Gene Encodes Novel Isoforms That Differentially Regulate RET Protein Signal Transduction.;citation_volume=291;citation_year=2016;citation_pmid=27226544;citation_doi=10.1074/jbc.M115.709675" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Glaus;citation_author=A. Honkela;citation_author=M. Rattray;citation_title=Identifying differentially expressed transcripts from RNA-seq data with biological variation.;citation_volume=28;citation_year=2012;citation_pmid=22563066;citation_doi=10.1093/bioinformatics/bts260" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. D. Goldstein;citation_author=Y. Cao;citation_author=G. Pau;citation_author=M. Lawrence;citation_author=T. D. Wu;citation_title=Prediction and quantification of splice events from RNA-seq data.;citation_pages=e0156132;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0156132" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. M. Graze;citation_author=L. M. McIntyre;citation_author=A. M. Morse;citation_author=B. M. Boyd;citation_author=S. V. Nuzhdin;citation_title=What the X has to do with it: differences in regulatory variability between the sexes in Drosophila simulans.;citation_volume=6;citation_year=2014;citation_pmid=24696400;citation_doi=10.1093/gbe/evu060" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. M. Graze;citation_author=L. L. Novelo;citation_author=V. Amin;citation_author=J. M. Fear;citation_author=G. Casella;citation_title=Allelic imbalance in Drosophila hybrid heads: exons, isoforms, and evolution.;citation_volume=29;citation_year=2012;citation_pmid=22319150;citation_doi=10.1093/molbev/msr318" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. E. Hayer;citation_author=A. Pizarro;citation_author=N. F. Lahens;citation_author=J. B. Hogenesch;citation_author=G. R. Grant;citation_title=Benchmark analysis of algorithms for determining and quantifying full-length mRNA splice forms from RNA-seq data.;citation_volume=31;citation_year=2015;citation_pmid=26338770;citation_doi=10.1093/bioinformatics/btv488" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. A. Hide;citation_author=V. N. Babenko;citation_author=P. A. van Heusden;citation_author=C. Seoighe;citation_author=J. F. Kelso;citation_title=The contribution of exon-skipping events on chromosome 22 to protein coding diversity.;citation_volume=11;citation_year=2001;citation_pmid=11691849" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=Y. Hosokawa;citation_author=Y. Maeda;citation_author=E. Takahashi;citation_author=M. Suzuki;citation_author=M. Seto;citation_title=Human aiolos, an ikaros-related zinc finger DNA binding protein: cDNA cloning, tissue expression pattern, and chromosomal mapping.;citation_pages=326-329;citation_volume=61;citation_year=1999;citation_issue=3;citation_pmid=10552935;citation_doi=10.1006/geno.1999.5949" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=M. Irimia;citation_author=R. J. Weatheritt;citation_author=J. D. Ellis;citation_author=N. N. Parikshak;citation_author=T. Gonatopoulos-Pournatzis;citation_title=A Highly Conserved Program of Neuronal Microexons Is Misregulated in Autistic Brains.;citation_volume=159;citation_year=2014;citation_pmid=25525873;citation_doi=10.1016/j.cell.2014.11.035" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Jiang;citation_author=W. H. Wong;citation_title=Statistical inferences for isoform expression in RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19244387;citation_doi=10.1093/bioinformatics/btp113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Y. Kang;citation_author=L. J. Martin;citation_author=S. Mangul;citation_author=W. Isvilanonda;citation_author=J. Zou;citation_title=Discovering single nucleotide polymorphisms regulating human gene expression using allele specific expression from RNA-seq data.;citation_volume=204;citation_year=2016;citation_pmid=27765809;citation_doi=10.1534/genetics.115.177246" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Kanitz;citation_author=F. Gypas;citation_author=A. J. Gruber;citation_author=A. R. Gruber;citation_author=G. Martin;citation_title=Comparative assessment of methods for the computational inference of transcript isoform abundance from RNA-seq data.;citation_pages=150;citation_volume=16;citation_year=2015;citation_pmid=26201343;citation_doi=10.1186/s13059-015-0702-5" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=Y. Katz;citation_author=E. T. Wang;citation_author=E. M. Airoldi;citation_author=C. B. Burge;citation_title=Analysis and design of RNA sequencing experiments for identifying isoform regulation.;citation_pages=1009-1015;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21057496;citation_doi=10.1038/nmeth.1528" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=J. Kim;citation_author=H. S. Lee;citation_author=Y. H. In;citation_author=H. G. Moon;citation_author=M. Lee;citation_title=Highlights from Recent Cancer Literature;citation_pages=1-2;citation_volume=73;citation_year=2013;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Plant Epigenetics and Epigenomics: Methods and Protocols;citation_author=P. K. Korir;citation_author=C. Seoighe;citation_title=Inference of allele-specific expression from RNA-seq data.;citation_pages=49-69;citation_volume=1112;citation_year=2014;citation_doi=10.1007/978-1-62703-773-0_4" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=S. Lee;citation_author=C. H. Seo;citation_author=B. H. Alver;citation_author=S. Lee;citation_author=P. J. Park;citation_title=EMSAR: estimation of transcript abundance from RNA-seq data by mappability-based segmentation and reclustering.;citation_pages=278;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12859-015-0704-z" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. G. Leon-Novelo;citation_author=L. M. McIntyre;citation_author=J. M. Fear;citation_author=R. M. Graze;citation_title=A flexible Bayesian method for detecting allelic imbalance in RNA-seq data.;citation_pages=920;citation_volume=15;citation_year=2014;citation_pmid=25339465;citation_doi=10.1186/1471-2164-15-920" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=B. Li;citation_author=C. N. Dewey;citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.;citation_pages=323-323;citation_volume=12;citation_year=2011;citation_pmid=21816040;citation_doi=10.1186/1471-2105-12-323" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Li;citation_author=V. Ruotti;citation_author=R. M. Stewart;citation_author=J. A. Thomson;citation_author=C. N. Dewey;citation_title=RNA-Seq gene expression estimation with read mapping uncertainty.;citation_volume=26;citation_year=2010;citation_pmid=20022975;citation_doi=10.1093/bioinformatics/btp692" />
<meta name="citation_reference" content="Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXic preprint arXiv:1303.3997." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Li;citation_author=T. Jiang;citation_title=Transcriptome assembly and isoform expression level estimation from biased RNA-Seq reads.;citation_volume=28;citation_year=2012;citation_pmid=23060617;citation_doi=10.1093/bioinformatics/bts559" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=H. T. Lim;citation_author=T. Zhong;citation_author=I. C. Cho;citation_author=B. Y. Seo;citation_author=J. H. Kim;citation_title=Novel alternative splicing by exon skipping in KIT associated with whole-body roan in an intercrossed population of Landrace and Korean Native pigs.;citation_pages=451-455;citation_volume=42;citation_year=2011;citation_issue=4;citation_pmid=21749430;citation_doi=10.1111/j.1365-2052.2011.02225.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. T. Liu;citation_author=T. Yu;citation_author=T. Jiang;citation_author=G. J. Li;citation_title=TransComb: genome-guided transcriptome assembly via combing junctions in splicing graphs.;citation_pages=213;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Liu;citation_author=R. Sanalkumar;citation_author=E. H. Bresnick;citation_author=S. Keles;citation_author=C. N. Dewey;citation_title=Integrative analysis with ChIP-seq advances the limits of transcript quantification from RNA-seq.;citation_volume=26;citation_year=2016;citation_pmid=27405803;citation_doi=10.1101/gr.199174.115" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. J. Main;citation_author=R. D. Bickel;citation_author=L. M. McIntyre;citation_author=R. M. Graze;citation_author=P. P. Calabrese;citation_title=Allele-specific expression assays using Solexa.;citation_pages=422-422;citation_volume=10;citation_year=2009;citation_pmid=19740431;citation_doi=10.1186/1471-2164-10-422" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. M. McIntyre;citation_author=K. K. Lopiano;citation_author=A. M. Morse;citation_author=V. Amin;citation_author=A. L. Oberg;citation_title=RNA-seq: technical variability and sampling.;citation_pages=293-293;citation_volume=12;citation_year=2011;citation_pmid=21645359;citation_doi=10.1186/1471-2164-12-293" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. M. Mezlini;citation_author=E. J. M. Smith;citation_author=M. Fiume;citation_author=O. Buske;citation_author=G. L. Savich;citation_title=iReckon: Simultaneous isoform discovery and abundance estimation from RNA-seq data.;citation_volume=23;citation_year=2013;citation_pmid=23204306;citation_doi=10.1101/gr.142232.112" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. B. Montgomery;citation_author=M. Sammeth;citation_author=M. Gutierrez-Arcelus;citation_author=R. P. Lach;citation_author=C. Ingle;citation_title=Transcriptome genetics using second generation sequencing in a Caucasian population.;citation_pages=773-777;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20220756;citation_doi=10.1038/nature08903" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=B. Morgan;citation_author=L. Sun;citation_author=N. Avitahl;citation_author=K. Andrikopoulos;citation_author=T. Ikeda;citation_title=Aiolos, a lymphoid restricted transcription factor that interacts with Ikaros to regulate lymphocyte differentiation.;citation_pages=2004-2013;citation_volume=16;citation_year=1997;citation_issue=8;citation_pmid=9155026;citation_doi=10.1093/emboj/16.8.2004" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_title=Curr. Protoc. Mol. Biol.;citation_author=U. Nagalakshmi;citation_author=K. Waern;citation_author=M. Snyder;citation_title=RNA-Seq: a method for comprehensive transcriptome analysis.;citation_volume=Chapter 4;citation_year=2010;citation_doi=10.1002/0471142727.mb0411s89" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=N. Nariai;citation_author=K. Kojima;citation_author=T. Mimori;citation_author=Y. Sato;citation_author=Y. Kawai;citation_title=TIGAR2: sensitive and accurate estimation of transcript isoform expression with longer RNA-Seq reads.;citation_pages=S5;citation_volume=15;citation_year=2014;citation_pmid=24564547;citation_doi=10.1186/1471-2164-15-S5-S5" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Nellore;citation_author=A. E. Jaffe;citation_author=J.-P. Fortin;citation_author=J. Alquicira-Hernandez;citation_author=L. Collado-Torres;citation_title=Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive.;citation_pages=266;citation_volume=17;citation_year=2016;citation_doi=10.1186/s13059-016-1118-6" />
<meta name="citation_reference" content="citation_author=N.R. Newell;citation_author=F.N. New;citation_author=J.E. Dalton;citation_author=L.M. McIntyre;citation_author=M.N. Arbeitman;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. R. B. Newman;citation_author=A. Conesa;citation_author=M. Mika;citation_author=F. N. New;citation_author=S. Onengut-Gumuscu;citation_title=Disease-specific biases in alternative splicing and tissue-specific dysregulation revealed by multitissue profiling of lymphocyte gene expression in type 1 diabetes.;citation_volume=27;citation_year=2017;citation_pmid=29025893;citation_doi=10.1101/gr.217984.116" />
<meta name="citation_reference" content="citation_journal_abbrev=Algorithms Mol Biol;citation_author=M. Nicolae;citation_author=S. Mangul;citation_author=I. I. Mandoiu;citation_author=A. Zelikovsky;citation_title=Estimation of alternative splicing isoform frequencies from RNA-Seq data.;citation_pages=9-9;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21504602;citation_doi=10.1186/1748-7188-6-9" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Ning;citation_author=X. Cheng;citation_author=P. Luo;citation_author=F. Liang;citation_author=Z. Wang;citation_title=Hybrid sequencing and map finding (HySeMaFi): optional strategies for extensively deciphering gene splicing and expression in organisms without reference genome.;citation_pages=43793;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep43793" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Pastinen;citation_title=Genome-wide allele-specific analysis: insights into regulatory variation.;citation_pages=533-538;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20567245;citation_doi=10.1038/nrg2815" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=R. Patro;citation_author=S. M. Mount;citation_author=C. Kingsford;citation_title=Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms.;citation_pages=462-464;citation_volume=32;citation_year=2014;citation_pmid=24752080;citation_doi=10.1038/nbt.2862" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=J. Biomed. Inform.;citation_author=I. Rezaeian;citation_author=A. Tavakoli;citation_author=D. Cavallo-Medved;citation_author=L. A. Porter;citation_author=L. Rueda;citation_title=A novel model used to detect differential splice junctions as biomarkers in prostate cancer from RNA-Seq data.;citation_pages=422-430;citation_volume=60;citation_year=2016;citation_doi=10.1016/j.jbi.2016.03.010" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. Roberts;citation_author=L. Pachter;citation_title=Streaming fragment assignment for real-time analysis of sequencing experiments.;citation_pages=71-73;citation_volume=10;citation_year=2013;citation_pmid=23160280;citation_doi=10.1038/nmeth.2251" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Roberts;citation_author=C. Trapnell;citation_author=J. Donaghey;citation_author=J. Rinn;citation_author=L. Pachter;citation_title=Improving RNA-Seq expression estimates by correcting for fragment bias.;citation_pages=R22-R22;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21410973;citation_doi=10.1186/gb-2011-12-3-r22" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=D. Sharon;citation_author=H. Tilgner;citation_author=F. Grubert;citation_author=M. Snyder;citation_title=A single-molecule long-read survey of the human transcriptome.;citation_pages=1009-1014;citation_volume=31;citation_year=2013;citation_pmid=24108091;citation_doi=10.1038/nbt.2705" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=C. Soneson;citation_author=K. Matthes;citation_author=M. Nowicka;citation_author=C. Law;citation_author=M. Robinson;citation_title=Isoform prefiltering improves performance of count-based methods for analysis of differential transcript usage.;citation_pages=12;citation_volume=17;citation_year=2016;citation_doi=10.1186/s13059-015-0862-3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Song;citation_author=S. Sabunciyan;citation_author=L. Florea;citation_title=CLASS2: accurate and efficient splice variant annotation from RNA-seq reads.;citation_volume=44;citation_year=2016;citation_pmid=26975657;citation_doi=10.1093/nar/gkw158" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=T. Steijger;citation_author=J. F. Abril;citation_author=P. G. Engstrom;citation_author=F. Kokocinski;citation_author=T. J. Hubbard;citation_title=Assessment of transcript reconstruction methods for RNA-seq.;citation_pages=1177-1184;citation_volume=10;citation_year=2013;citation_pmid=24185837;citation_doi=10.1038/nmeth.2714" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=D. Sturgill;citation_author=J. H. Malone;citation_author=X. Sun;citation_author=H. E. Smith;citation_author=L. Rabinow;citation_title=Design of RNA splicing analysis null models for post hoc filtering of Drosophila head RNA-Seq data with the splicing analysis kit (Spanki).;citation_pages=320;citation_volume=14;citation_year=2013;citation_pmid=24209455;citation_doi=10.1186/1471-2105-14-320" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=W. Sun;citation_title=A Statistical Framework for eQTL Mapping Using RNA-seq Data.;citation_pages=1-11;citation_volume=68;citation_year=2012;citation_pmid=21838806;citation_doi=10.1111/j.1541-0420.2011.01654.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Tardaguila;citation_author=L. de la Fuente;citation_author=C. Marti;citation_author=C. Pereira;citation_author=F. J. Pardo-Palacios;citation_title=SQANTI: extensive characterization of long-read transcript sequences for quality control in full-length transcriptome identification and quantification.;citation_volume=28;citation_year=2018;citation_pmid=29440222;citation_doi=10.1101/gr.222976.117" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Tombácz;citation_author=Z. Csabai;citation_author=P. Oláh;citation_author=Z. Balázs;citation_author=I. Likó;citation_title=Full-length isoform sequencing reveals novel transcripts and substantial transcriptional overlaps in a herpesvirus.;citation_pages=e0162868;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0162868" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=C. Trapnell;citation_author=A. Roberts;citation_author=L. Goff;citation_author=G. Pertea;citation_author=D. Kim;citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.;citation_pages=562-578;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22383036;citation_doi=10.1038/nprot.2012.016" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. J. Treangen;citation_author=S. L. Salzberg;citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions.;citation_pages=36-46;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22124482;citation_doi=10.1038/nrg3117" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=E. Turro;citation_author=S. Y. Su;citation_author=A. Goncalves;citation_author=L. J. M. Coin;citation_author=S. Richardson;citation_title=Haplotype and isoform specific expression estimation using multi-mapping RNA-seq reads.;citation_pages=R13-R13;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21310039;citation_doi=10.1186/gb-2011-12-2-r13" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. RNA;citation_author=D. Ustianenko;citation_author=S. M. Weyn-Vanhentenryck;citation_author=C. L. Zhang;citation_title=Microexons: discovery, regulation, and function.;citation_volume=8;citation_year=2017;citation_doi=10.1002/wrna.1418" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=B. Wang;citation_author=E. Tseng;citation_author=M. Regulski;citation_author=T. A. Clark;citation_author=T. Hon;citation_title=Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing.;citation_pages=11708;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms11708" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. Wang;citation_author=M. Gerstein;citation_author=M. Snyder;citation_title=RNA-Seq: a revolutionary tool for transcriptomics.;citation_pages=57-63;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19015660;citation_doi=10.1038/nrg2484" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. R. Williams;citation_author=A. Baccarella;citation_author=J. Z. Parrish;citation_author=C. C. Kim;citation_title=Empirical assessment of analysis workflows for differential expression analysis of human samples using RNA-Seq.;citation_pages=38;citation_volume=18;citation_year=2017;citation_pmid=28095772;citation_doi=10.1186/s12859-016-1457-z" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Wu;citation_author=O. Anczukow;citation_author=A. R. Krainer;citation_author=M. Q. Zhang;citation_author=C. L. Zhang;citation_title=OLego: fast and sensitive mapping of spliced mRNA-Seq reads using small seeds.;citation_volume=41;citation_year=2013;citation_pmid=23571760;citation_doi=10.1093/nar/gkt216" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. J. Zhang;citation_author=E. W. Lameijer;citation_author=P. A. ’t Hoen;citation_author=Z. Ning;citation_author=P. E. Slagboom;citation_title=PASSion: a pattern growth algorithm-based pipeline for splice junction detection in paired-end RNA-Seq data.;citation_volume=28;citation_year=2012;citation_pmid=22219203;citation_doi=10.1093/bioinformatics/btr712" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. Zhou;citation_author=F. Zhang;citation_author=J. Y. Chen;citation_title=PEPPI: a peptidomic database of human protein isoforms for proteomics experiments.;citation_pages=S7;citation_volume=11;citation_year=2010;citation_doi=http://dx.doi.org/10.1186/1471-2105-11-S11-S7" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="Alternative splicing leverages genomic content by allowing the synthesis of multiple transcripts and, by implication, protein isoforms, from a single gene. However, estimating the abundance of transcripts produced in a given tissue from short sequencing reads is difficult and can result in both the construction of transcripts that do not exist, and the failure to identify true transcripts. An alternative approach is to catalog the events that make up isoforms (splice junctions and exons). We present here the Event Analysis (EA) approach, where we project transcripts onto the genome and identify overlapping/unique regions and junctions. In addition, all possible logical junctions are assembled into a catalog. Transcripts are filtered before quantitation based on simple measures: the proportion of the events detected, and the coverage. We find that mapping to a junction catalog is more efficient at detecting novel junctions than mapping in a splice aware manner. We identify 99.8% of true transcripts while iReckon identifies 82% of the true transcripts and creates more transcripts not included in the simulation than were initially used in the simulation. Using PacBio Iso-seq data from a mouse neural progenitor cell model, EA detects 60% of the novel junctions that are combinations of existing exons while only 43% are detected by STAR. EA further detects ∼5,000 annotated junctions missed by STAR. Filtering transcripts based on the proportion of the transcript detected and the number of reads on average supporting that transcript captures 95% of the PacBio transcriptome. Filtering the reference transcriptome before quantitation, results in is a more stable estimate of isoform abundance, with improved correlation between replicates. This was particularly evident when EA is applied to an RNA-seq study of type 1 diabetes (T1D), where the coefficient of variation among subjects (n = 81) in the transcript abundance estimates was substantially reduced compared to the estimation using the full reference. EA focuses on individual transcriptional events. These events can be quantitate and analyzed directly or used to identify the probable set of expressed transcripts. Simple rules based on detected events and coverage used in filtering result in a dramatic improvement in isoform estimation without the use of ancillary data ( e.g. , ChIP, long reads) that may not be available for many studies." />
<meta name="og-title" property="og:title" content="Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/2923" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Alternative splicing leverages genomic content by allowing the synthesis of multiple transcripts and, by implication, protein isoforms, from a single gene. However, estimating the abundance of transcripts produced in a given tissue from short sequencing reads is difficult and can result in both the construction of transcripts that do not exist, and the failure to identify true transcripts. An alternative approach is to catalog the events that make up isoforms (splice junctions and exons). We present here the Event Analysis (EA) approach, where we project transcripts onto the genome and identify overlapping/unique regions and junctions. In addition, all possible logical junctions are assembled into a catalog. Transcripts are filtered before quantitation based on simple measures: the proportion of the events detected, and the coverage. We find that mapping to a junction catalog is more efficient at detecting novel junctions than mapping in a splice aware manner. We identify 99.8% of true transcripts while iReckon identifies 82% of the true transcripts and creates more transcripts not included in the simulation than were initially used in the simulation. Using PacBio Iso-seq data from a mouse neural progenitor cell model, EA detects 60% of the novel junctions that are combinations of existing exons while only 43% are detected by STAR. EA further detects ∼5,000 annotated junctions missed by STAR. Filtering transcripts based on the proportion of the transcript detected and the number of reads on average supporting that transcript captures 95% of the PacBio transcriptome. Filtering the reference transcriptome before quantitation, results in is a more stable estimate of isoform abundance, with improved correlation between replicates. This was particularly evident when EA is applied to an RNA-seq study of type 1 diabetes (T1D), where the coefficient of variation among subjects (n = 81) in the transcript abundance estimates was substantially reduced compared to the estimation using the full reference. EA focuses on individual transcriptional events. These events can be quantitate and analyzed directly or used to identify the probable set of expressed transcripts. Simple rules based on detected events and coverage used in filtering result in a dramatic improvement in isoform estimation without the use of ancillary data ( e.g. , ChIP, long reads) that may not be available for many studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
    <title>Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__Ta-UXINC0YONAj3hv7M-Srfc-_B4Wn0KATTOQGieTps__agTSMicuoC50kRvu-VcTNoPubLJgflHIUuLq_Jt5dMY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4QmuEIjWC1n7XBRDXpo1O7fm3eof1qJ-DBt15uT8SQg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490734","apath":"\/ggg\/8\/9\/2923.atom","pisa":"ggg;8\/9\/2923","ac":{"\/ggg\/8\/9\/2923.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/2923.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2923"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2923"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/2923":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mePXFdVwcIVz3rNb7GgAyyKqixHAjkmaQZAJ67GaTl1\\\\\\\/OImwR9+IRw0vM9B4+wJ7f\\\\\\\/JkwhItynHXfzXaCa0Zm3B6GozUiZHp16dFpqMCwyboV2bUxqwbsVHcSEaqutDnA5T+jTL\\\\\\\/lTlkWaTKUFVtQT22QDzuDD3irvBmNX5PmNVP7tqldlxPr+pIsv9cWwkVxjNFbQZZ6lP5urxF1xuQe+Y8zwX8w34O8lbeGRyd2TMzFk+YXvYIjh9pIbXUbDatv3EVZ18I4XhxEd0BH8i43v0Tdy8hhp\\\\\\\/z2VcAB\\\\\\\/FxokvdKukajs9Nz7ZmZJYrj8zWPjgjfWL2sPhqGo8e3ec8nUVOMibVDNrW\\\\\\\/hbQeShH\\\\\\\/Yjeepie9fxaxrHccro2hAqovkQvNzAqi7B9lkHhey2v7eyGgee1IQRqalIokuaLMJUOirIGP6KLwkFQczgxvhbP6Cm9ewBLGNnGlwvJqUyYB+q49v6d1Q+gfzWlvPYxHFIUn1o6baydoVVDyPnpD6E9o67RvRNv3HBtGWZvWmUd+D1NI1ny3lC1elqdidLMhxIix+\\\\\\\/6PstN8jOQAtU7r9BwZlEEZuPNCql9vlLNARgiFOndeLAEvmwr\\\\\\\/yVJ91bLoWiOTwWOKxWfsnnNaAs3Nzq8yjQ9VQ8LaWtnOwfiw8XY4l4zbDQRsuuZxwKLux1JcXgkHFCBZUZXk4WZswG98L33EZJXj5cIh587C8baCRdxNqORwpVm7JHhPb\\\\\\\/Yhzfk\\\\\\\/ggZhLwbUNI+mdZYheIuCqTb5dcPzOFAV2mHyyHurUJETOXYWM9NcDyb19NhTtxbb11G9dZnre+9IwtEGK9iRnXd\\\\\\\/jyz\\\\\\\/uved9w41+2oJsIC+jZErBlHLQQxRs6O3y3jWOEvnYnxNOQAECXRFsDeOzMSn1NkjzKwjuLtRle1ad6sed7nwuFm1QXD8d1w3CDE\\\\\\\/PcIqwbMG00IAIFNSQRhwNE8trSJjoz7fHikXNSS5mEjnbMC7hzU9IRe5ETbBsLmJQSf3KKTva1QKxMUVod92H66or\\\\\\\/UdreRUzAhYds6\\\\\\\/RYUKwiAwbo6YEocsOPGtS0oI6LXCjtfHbufXiuB\\\\\\\/1nYFsMXWpK4LQb0\\\\\\\/ECzpc31PPREHhyhdtLJRThEeubQtatNbD4hXkvqUzdRp6k2mVfodOS\\\\\\\/JxXj9l3f6+LizFqmGlxAet+Nvtr7ZwZvKAVHEqp4dAo54k2qPyXEJFYocMhkjNSt8eRQfOv0mA7uEvUOc6j4LF4ihaOREGTE1Japwv3QUzlnVHqruKkS+0BKMvqeyHj1\\\u0022,\\\u0022iv\\\u0022:\\\u0022jAzy0Ocdq5K0hhRoofG56Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d50d4034c1774018e0d2e620c57dfbee63d44fad9e6c49918d776edd66de441\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CH1OAX7P9TmWBcWGeX0U+TsY0DMamQPgP5DjB1UOKrF7daktithj17esMgs\\\\\\\/MHyUwcjx4\\\\\\\/6+yKPo\\\\\\\/4Gvdv\\\\\\\/sCOLwDsdPobb1qvWxXknSDGCoszVbMBs3cB1ZHjvsimKvMqcqMyBgY05iEpyjJOHofZ2NQcNS4r8jteE9chJg+bZ9vl8FHp3Q7AN4b5J5WfdvU9usbMJkrFtz+ZdBKS9DvhvWVL0NFna\\\\\\\/jkWBvwTdU3P73o9uofS7Eh+hDubU+AvtK7zN4RiY\\\\\\\/xD75985b\\\\\\\/5EWxLMz4TNLT55App3m1GvGsPmLORrZt3es6N6ovNBg0FAHP1jPMyqyQFGxinyQwFaQQ3hchjQ\\\\\\\/hNVpDalXcAdTnWNj59w\\\\\\\/vZ9+I3WrQLpCEe+G3a+Fwde846oZbdX4vINA1gruUjVp7TjVmYkCqIXQ7XKi3mYFkU0t98+GnMX0sA7UAc2eWvWZkKbKeQjl919kg5exdCCtfifiumIypD4sufcc3sE1P1YkQ0Lebl70sdddihoghX8UaKFOEH82zr58M5UGxMpiB8uhNUsF38cagR8Njx8DSoAaogTeB6UOZjXN\\\\\\\/BhrcOzGAfjvCU5A0vh09pTpgoBY0vZuE5hxe7jRU97NBqFGizldeR+Sh2Qyh5bxW2+Cx29HOv0LOEfQqLDZqrBb37h0UUPuatOSIy9rUe2t+\\\\\\\/1UkHKf0zXtGXnQ8zvIUvVD546kQ1OaOPe9T38VcmZuWaQPQutwOrhZWtq2KNH+EtHICGuqGZMNDnpPeeEWzDyg+pULioCy6RymF8ZUV84BdyDBUPeV9J4XUfK0fe0xZck9dSrm0vvhT06JFkpz7eGuPwwtcPCgF4N3z2nJmAW4xTWU6pYC\\\\\\\/OzPLNhZrkFeI5uRxZYqk3NcXUDvBWFr8qPSePpb+\\\\\\\/89QzBIPwCCSrQpACjDpkY\\\\\\\/Xyr+Q4r\\\\\\\/bDPQSyXRTMZfBdQYhG+UrqvDf8n3erNyMGtKajdtwNkbcGS6TdwnFNmowwh14wwOv1c3Fe1NCrSCBvGlTGEu75fMoHgG2fruUfqwyI6bxn0Nle0qdJB9t6+e4LZZbdnS9j7lvxP5KHS724rqVlLc+xzV70DJ5\\\\\\\/4fTdlma6q8DWiBKOfItaE92jW1LbUz2ExBeiEJjQrfS3Pp\\\\\\\/fHVXZNS+UEPgz8YdacEKRaotDJk8rw7Lx8bmViFlTnBgbzDMYw9Ae2XCZDKAIPaZtdU5Ryz1ExitZksqxgIiwr7Z6xMZmIK5hckepQq+D9I69gZPkLWLOcW1mvhnEvp4ZRfNbzn2BM7aqGHa+fw+Lq9Uja1jNpLGysfRKgSqMUtQfzwXnAAIyWveW5wNMjtWlj\\\\\\\/ueBmdfH7SRT6\\\\\\\/NIFqAOFqLK2QC2EFQGAu6ylpvb0GDNrDPWS4xUAUy55HQmSd8OJt\\\\\\\/LzFvlMXdQXJHd\\\\\\\/Azk989B2g9Foo5nlq\\\\\\\/wuf+SWK5mkmCvD\\\\\\\/RisRaEk1bfPj0pAjyYi1nF3j2eAsKhvqp2SqPDAnn\\\\\\\/Upzu1jO4T6P\\\\\\\/4TXWj\\\\\\\/h7hRHZaQ70+Lchk4deH2eqodG6teOK1yQxC5g94Ca\\\\\\\/pWZWHlU45voUBktykhIt9Ad79dpAGKJVaYA97TYxD\\\\\\\/D465sZgZaQVa3q2Td9BmmqGIRbeUj3zPpWgjm45i8j8smQ4ZjSwSD5LvLEQ49vBILav+rFXAmc0G+ckhRvBMMJjtvSQIrBePuH9UboDoeCLmQuQPuY1VoVwdCRSM8tZS9yoN8xhmlVYn9wy48Z3VtYrYgwQB25xzjaqW847nNFV6f86wmS\\\\\\\/VXuGXbU08CJ\\\\\\\/d20CkzAtjdbgbowWuQtwbiU6BnvuOs9fW9LW7gtmo+EsS38ih7R9Im\\\\\\\/282h3eKziZgmZ3bWmxN0RgGNabvYi6IiO1T1P\\\\\\\/YuCXRCcu4+fwNzvGyAraE\\\\\\\/kb1S4r8O4X9WEpKL8Os8a0pSFcYo27XEd5gezOgC2Ea+87T6\\\\\\\/5ck3XlvDdtpXsobsiAoIetZcXGTZPky8OdKRKcefi26y3x91F8Yf+RbJe1xX5wBcvEE4+ZWpZgPMDCtzfgM0pV0UdGATpLuie2kvWHM2vCxNMMIkldwzn99892lamUm1ObcsuAkNeZ5kihUH6oshoMdaSaCQK8Lt4WRueuEQfjHXyVxY7RS55zQV13uS3TTj2HQ+PNDMjJpgpbp\\\\\\\/\\\\\\\/MZdIkcqgLRsX4PC5XHKA9D1qye1pM9syIMy+W0Na6OGR4MD56Aq2MMRszrrCqkpIlmUXN0Qr6ARZzVA3yjqOUJ5PRzgL+sgyrbbHk7Monu8p4\\\\\\\/AtX8zUQuvND+58Ps9wNVPVM5Tf1ehQ8RTDkJ8ea7Z3rm\\\\\\\/lb+\\\\\\\/mQe0quQsdaruWX10SG3maxqIoj9hNBZwVJ4lCu1WRgW0M33kSjMuhr9Q7vCUzmnbfyYhX8n9nkwjQPyXKo8wOVKQyioA81+oWaCo5f5zj6wLQxuUBo+ds01faE7JGLrLX9QR1gYNzqv40c7Ty8POYvKXgGpDvBgWK6N2R2jkHFdS3VngTZY2+rb3fjx\\\\\\\/1Ook+N9SPa5hlisdijxtti1KpydkqdIicaz+q0CjK1H\\\\\\\/wBNDwhNu8r8umqu4RHyDsbhx+ZyzyQ6w2q6xSJ050HRqHx3kLE0LpcBwnkZuXjIl2FVsXQaxR5p1hucVR+aAgXoF7bxNc8JuYzjvOBwlWPAd90ymq8vTgvNCyP+B1+r7BGkvJsikX3AcTs9emWWxIM91eT19unNNmTg+jBdW8IuHvE61Wp5oeQBidTeUotemUrKxwgNPzC1VvUso5YFjWFk+G+HXhT02c\\\\\\\/zgsLMZlHlZovYwnutxdKgn71ETNoI7cAWoxVcDBAHTmsvWHclhUfJXVi+QhMGQlAzUrkA8Xw1arrrOlF\\\\\\\/Tvqg7oJy2\\\\\\\/YFnkHppRUNcR3o8dDD91Vf1gegUzOTV0vcoim+CgRsZ3Bg8UmtDtBmNo6nVDykB\\\\\\\/InEqjA\\\\\\\/wdvoIYYymBnokgs5dH9DRdSHD0WcHVgOHCIPDZ6s0S5AyoQM2Gn9l8tUICysyDxNUxgJH1ADGmGNZw+Zm29BVNaLC1z79c2U3nW0p00VIUwHjAXeTU3OH6yHtW1GuXJpgYRbvSuaLSbe9zJscSwx9mceflvrhuu\\\\\\\/bErGd+ZmkP5YbLWZMhDArKrv5mJRhcV80Iu578yWcjvLeUqYLANzClsPOgPrRZK7kygoo5ZTZEaUqb13IuIuJhRTezQzA72mA+6FLrFyPeIrsA0pCex4RDeY5yJJbTZl7SuMViO+cRQe6Ade9t9yFDqA+lo\\\\\\\/sg9InVrbj6CDvgRANP1U+qqVA+JNHyEe0DC7Bh8bp6Piy8TuvrvnggDV\\\\\\\/qSqDH7EjQk+9qh5zPqDJjm3Nr+MrXSSe8S4j81wbfl6F2qIursCnTF6hqnpi4Uous31hOgX94B++oA6NclCAoa2SAaekOIaGhm8uB7jL73YWStJkfUv5y5Vgcu4xQB1fsJbMynmW4PejjZC2g0\\\\\\\/tSDztqh7HPWWvKZktnv6MKSFbkuUGY4CQyU+kQ90\\\\\\\/ZF3f9F3VIlR8OmO9c0fWUsyb6ceO73vQzIBSBdNKevpXtBG7HyAIlcy746fZZMKPdoMINSoUp5VNNXROqG+zrV7+n2u9g9fZn9DZ5TeIcHum5HfqIaDdEEmx3rp3mHFvX0pkUE7byiSUv8BNO9j9+DC3FWGLBTVA2wBatdIXkustrpCfHui93KmxmpfjOq6AVC7XgE1l6Qt9ucwOc6o6e93xsaY0hI4exyDQCRdKbnUsj+qANrkOUutYE4l+lLZFGnbQB0NiqizY0fklQtGIXDY9JwarLZrebMf3aUzSmrPtBuzNw2BZpHiUO7lmgVWjmN62GgvDncqFheYnKATS102V1\\\\\\\/dE9E+G2Aua3YNkgUHUnL2xHYkzu89ySe6PXnnDM83m1EVFWBpgpr+r4vjfvFmE19Xc\\\\\\\/5MoBdE40e73LVRyWRJehiVsXFDnJmrQPLFwFx1E3Rax+0cZanxQRSUstJ0pHoi+4svCpNLYT7OV8LSe7NySazeVGG21gsT6AajcuJ2cW5nTuYvUMxhBLugysvvAjlqPF9YJdPanuLirkshbbWoqzWGPIVHZe8lP+1ttmAmqrBaMwEE7TVQ7Xu8S7\\\\\\\/XlwdsZOgVkyx0R15FkGLaIFfMBakhm4CEA+cWG4BaQj7sP7DO\\\\\\\/XEzTnWKsTcqtR8GG0y4MylrrshGiV8FRQDS0VIeLl4qpfQSy+G+4JWk3S6BAtC9omj8QsoDiwaR3EgIt5K8XYcqTIpJ\\\\\\\/OOUC6b\\\\\\\/fzpXAcCZzgiiYwNQErOeLegKy\\\\\\\/dQFD+jtpK2VtGF32cNLFobQNfzfLut8rHxBGVHkFB0m8vjyWR3ORWIf7u9YejN8GULwJYFWSnmbMO\\\\\\\/m14RQfS1J8aqxNHOji82HbPpGsIWIt64PCrCppDBNWQaCuGhMBc5Rkdn\\\\\\\/L5SOknh31zbJ48Eq\\\\\\\/dww5gC9TeU4cBSNAkqyfaW+CElpBOvEpeJ5NYSQlqWUawCiyHpuqUNDOveOV81Zmjwp2cdc6uuL\\\\\\\/dStXeYhWfUE8wphoGyiqgSkchJFleEw2bx0yUJhh5zH+twc9whjZ6R0ATg1xShg5s3bhFdjTAkzOQQZ55SL2OibZYS9O0DVlJB0KSTxm25y0105WFhaIszHbHPbohZQkyMUJuhIur14H+y7UTGZlPFTsiVNqo5P7S8haSrxK3iKADG9Ot2AUdzLql12FooN75LZLFxPefFu5uVSaBi+s2nojzt5zqBdOUMBIJFju\\\\\\\/DrTM0K7s7nNRW0xdRuRjSMJ4rohyyPcBZ5D2jLfcQxRP8hepnJmtfmVtRbD9Yb2x6NOdj7VJClPaPtUr7LApUpaaCWEMMOySmnwRCgVB0ZKCDBfhMKRyp8RqH73R2xjY2xfQm9ece4CPHOkX0vS6YZtxETlNzf3\\\\\\\/YI\\\\\\\/MrHI63hUV\\\\\\\/rTFeWZfIkdOG0uIuPlfe4WeGA+LngLRx1285ZZFUOu9sXljZWga4+jmNGDTRr80d9nm1jOlXNwEi39hwrCiqqffS3X4hAGjuEiqKSLoarKClZtOm1FIrt0vzs9UheyaFr5jYV\\\\\\\/JwUeu45\\\\\\\/KSJNqtH9ftP8gaaog2fQzb8AtrUvTAGIq9UAypBuy8rO9y8SCQzMus4YKiq13yc+Mvdmop6PXLy\\\\\\\/uh5K+aFW\\\\\\\/wrYp4kUTYJRPUaZ2O+mXWTshEVQJWPG+lDbMi0c3NFSNETeLjTlZhvWbHudKJ1bGdRQRkWw8Cu0iLkyTCOQ5O6O8lB6ZuipIWzqST6Y84sEKiBPj2FYqSfaJ4SCT2ik3ZLTb\\\\\\\/UIc0fvelybEe3nMZv9LuFHk9AHGpvR+YFiUg9QbCyVsPMxOgnR9voioFOmnRcFl6kR2hPU4pSMG\\\\\\\/aiUDaLLHTUq5JDAS0ZxtkfOMozkINir0O9OT+oV85IA9bt24FfzO2RxBOuaS5ap5mfV1ongIQdpiDlvlJ6nfH2eMAPas8fRnJztiSAwnck+DMUra4DGfiIVfez8acKw2KCr1JMrzK4uXmrw\\\\\\\/Qtuy13xXy\\\\\\\/2x0GvgQp7CNv1XarAbQ71F\\\\\\\/quSgXJP3SghiTu+2Hhw4DaQgCZUddMyxyaDlL+iGdmL4c9YlIb5eCQ6ZZ3kyEwWORedEZnDNblek6KRk5AeoiRXpqbha7OBxDH1w7RRHDAK\\\\\\\/mSnaS5pq\\\\\\\/isivSvDcQQgVhomd\\\\\\\/\\\\\\\/YwlvzEP\\\\\\\/Z4DE+lBkNwo\\\\\\\/3K6drozTWCj24JKxN9k5eaYZENZxRJf9Vn6fXbv0SKQFUxueJqDSnLwtMh1ul4O5S3jCT3qU322q\\\\\\\/wtXiTDomDr7HKBRTwZOl8she9T6pQKx\\\\\\\/Z5XcAg09O90oJ+raXUOv1Ndywl3qJTRgWAlTAxXUu0kkAqLUnqWH3lY11zJ0GiwvJnfcbM3QV3dXOH8AuhSi11dvOwnTlLzBCA3ICnmYx5gL3G4CqVeF1ZIJ6dEqxLvztOEJIFokaLxMDkXQ9Ps2fy3Mylz3Rg+OG7bd14zU6TJhyVB165HHaES7COssrAKJlyOqDEDLi7RiK+afIxrkHeWdx7B9hhQpXTZIhM12ygWpGZCqUUSnuSXbgZPL+B6n8F97D89TkZe\\\\\\\/Y2huteNzuK2mDNtaS9OiORDHZA0Jpm\\\\\\\/BMTmEL9IFBB3SxSQQ1DnMPAc11X2Hb0N0XyhOw4DeWONt4JW0+tkXe51nkrzgjDRCa+1FfA+o+0zDwcDAphbQqcHbWbjweH8a1fwwkiGAvznH6CPYsHBFPZlfbeMGlSZZzUc5bEjXxhzNvCg9fW\\\\\\\/OEPBrJPfvWAgB9ubgojJFyBYC61LJL5W1JyWF7b6QVT+oZVjVO73a6YxXUBGurGogjVtR3O81AlprvfkWjg6BdPmS9XWrtCRISWkQbONekvgBg6HWlnsOha4HiTSKcqgfqak617H8sfK286nPW3toovMS33VhAc+jlzHTDLMVdKngAZiq8ZHKFOEpiEdieJlt38aSPQ\\\\\\\/ty7ti8uCPN4i+hHW6Pdys25yf87fSYAEqCRDuZTBr7W1ES+yliDYgjFuH+Oq1AymkDaMHYEeHzba5HalmYqsFQ85l036viUU+EAKYWN5xRD1ztz\\\\\\\/3SdbjvqBttQNb3UII77D0o5ED9lmY+EnNPaltQT22P19yPMjv6z8DTbhnpfKq6dGvOvFs63tqcwbi+g8xRcq3LorBADDOfT\\\\\\\/FvV9SV84AlpHToap9bjot7kMMci8pYLe6xA5DzZVyslQEoMsBAGbJyjSdQxC7g7Kfy3B69mAjOiXmDN8WgDkXYGlJJVrCTuQ3RvkXv2bKrIqf7UfdhQ2Vpx\\\\\\\/8ErBJhqUbsuJBpp0VouMgGkamaA9eDWedN+zF\\\\\\\/Tzu\\\\\\\/iDBfDKyJcoFkk\\\\\\\/A1dl5KbQdP7+0Li2GGHhb8A+zadVmRqW2mITjdJwWewa5U2MmjLAfIqR4MQncU9i1XqmT1xiEGiekGRm0AO38sdkPPCyDI3khs8ZM8MlS+7+Y\\\\\\\/wXwROWJrUNCvYOoFd5ogl32a5mFZzvH7+tc1oiEuARq83W7Qu229nwT67blS9Y0juQPV\\\\\\\/w8RpZhsdUdg9PjURslSgt+XS0Moel7sJ8pIW2TtEasEYQvh7yDR+AJPFRYKH497n9SCxqcNzLDopTLN0GVOstyqWuE+iBV9ufm8ZYhjYoioQV8J3Kk9fsacCs7QtIys4tP5neCGsqLzu35gW5W\\\\\\\/VkIdJU8UHgfvw1Hu0ltcqYBWH8vqOz9igasRmAZ18atPevn1LXOhdNHHlvTcYqLic9hQ9esdpB8kZPz3eo6a9frlN0qRH2B2hMYs8YCOfB6oid50Ar51\\\\\\\/FbFLeqWGren2fXC9WVF6ovnHkRN19trVCal9eZmu5nWDI2VtCHj4tGDaxhbNByIFPERuQYbiBKd2tw20EV\\\\\\\/8kFpdmoFclkkRUpWRAT+TNMsK5EfJAKyxFfB2ATgzlugxUyadFCPncu5GDdBQgunDCnkXAr9Ssnwz4gbN1mpAJ4Qkz3JC9A6U\\\\\\\/XynA3\\\\\\\/9v95daFUm7V3GZh6K9mLqBWGnuYLVPxUaia+XiphwSFxtaAQLzBOJUwryEd57cB5dO+iqo7or6f0Bi5rEqsuuYvkMnXwka\\\\\\\/qArhzc+qCnmV3yNa12ztK0sSFICPE8nqOUa035H8MFGVGfC6qNuU5d6ho2pXkh3h57xIKh72Yop4f5alYqrCxa6RXs+q\\\\\\\/Jhf5FVK79uPqmfQpo8QBNJ5O0MqeYO+QY9thkbi6HVtoiVSkC1RE6lWgP7aMzjXCVm4kzFSwwHBSVjZxFmn07ohTCZvHhqGZEFYabtqyP0HfYetL32Fq0pSzWxFYZUlzwqB1sMw63z4K1Hg8eDJ3si4TyJ+jW8TclPo0o1eMA+mgRQUmU\\\\\\\/30XQTg9+4WOdF6RS2OeIbXNxDT6LE0JsotFGfM0bfaL2XmZD76kuZHkji6f0aRsXJ2fn9\\\\\\\/Pts17zhSAadIrumq28mJXbToeNSDlOw\\\\\\\/6ym3ln1v+BF\\\\\\\/BZ6FKe4EUNUEloEJZib0ys8VrKOEyrW2aG8McQCIX7GUV6q4elDIowQ5h3H1z0b3zt6\\\\\\\/wVLKMvmvq4a8gxTyG\\\\\\\/OxO5Rwn4+DHFIvJUY52bCDj+Ho3P\\\\\\\/raAj464O1+3D8gMcxS47RgiTflc4eKvlha8CrXNOP1A1ZE+k1iju8OLPQ4Ou6wW83NXSlv7wnjtyU+gCAWA4G6+dc3f8Jey2CbvcfRbJ8WunF+ryr8tCbpZXfu6WI4PWzmxQPItJu8PfmZNvbGfNEP4V7A2QZ5MBaXctplSDGtzbWpKHDn3jkAe87dMOLG94TFo09w8jY60Z8IHJ+RJ347IrYTm1mLwle+WOZJxFBuUoIUlqKHO4qZ1UyK0gmIxyUSnSOcUrLeubmf\\\\\\\/LhCg5jvTBYKm2kgffdfa08HNBf9A65HQ+mcMpGHrWLFkTvlwzP4l4lPKgPoB6woux\\\\\\\/gPNZhvAGFCM7wjanSpswiojz5yxihy9buFfZY\\\\\\\/scgr7nfmcBBu+NJ0plSQaAP0LRHDQR7ffJr6V+UPINz\\\\\\\/d+i3Zlux2\\\\\\\/raJCSYtQtXFnan2YA15ng1LTWGZ7SDey6m\\\\\\\/EkFvbSShX2UURiPR+0\\\\\\\/HjjklGuokr0uHy8loo4r2\\\\\\\/JVFpvtbP6jrycTm8ZJxf2PEehHfLYyMtlhIofddNraLPo55mW70O0NPnEolcm2P7Nj619LnPtk\\\\\\\/YwW01OAx1X40EvgzVMbZKFqEzpUxIx3UkGw2PStircxUTTSiu0olw8zgUNrXcpRD9SqRGk1L99BwGZ+Xy1wlRSTEKRIndxJ4cgMRg71aQG\\\\\\\/b6wPXDCQg\\\\\\\/AYdYB9kuvZYPm8G\\\\\\\/j\\\\\\\/ttlySoPzERUUMrGP4iSCzPzjGrszR+stopgRy9ZnFNepBqPyXWW\\\\\\\/YOMKuMNJvmWinEf\\\\\\\/CYfDhhzfeyz+2HHFHOKp\\\\\\\/mywGxFzjspPX9luoN\\\\\\\/rIC2xjSBuKB98JPmDnjrr8kk4YLt4OqvyvFejOoDrLzlhLx8lffSXNUHr1xuH+RIUKvJPSzdC8n1PIEewWLhgI7CsgsPpDwc47EUVAKCPocVi9G1E7W6vuW+eUxzuukIICAbujrcvGrrEXfAAAQ7gb9jLGKJIVRe0eRyrRVfiQFqglt+X2k99qwuS3tyBBhT90MsTQVm0Qld5WIFRahnJru9WPdcuOOXdcb2gZ3KunRsiZ8zhHJL+FgUSxQZnGAfkayrp8xONS6WHrtcvhbvvTttdakRwI+SEhz3MwgvHolQI8TfN16DB5jtPbjS2pe60w3lRtCUUczqefhAoeivPomYiogQWOSLZf5AUsZcbuVYjBVkI9AlB4mcCSDJeSn7lPG35dtwHXQXl6k5rAxHkJqWUmCuDrntqzCPXIg9WyMGebrJAwH8Zhgsk\\\\\\\/u2sq6Reg5uI\\\\\\\/eRXR2cy2HMZfwthd+6NplMqY4uTAa4kpNhBfUlph91K8YVeZR1E9RF2BgwC5d1wxCFwq8Vb3HodQ5\\\\\\\/dH7cZ+SrPsEFB2h8aHIt152VSozs\\\\\\\/LtHQ1AMEQBpGgVQGTXq9XluQZxUXXIH8ak4yopq8MQ+3cWKroT1+aa8U1H61K7Us94DA0UjDv7MAqBHOSsDuiwfW3ANjOV8RdIw2PqYr8nZ3KYjxogn2WlzpXw0pmDvKFW0pJTweUAqLCO+72UYfkapTM6Leqf9io1grmWCzfsaz4QZSUe41Ip3rdEBbnTdaGH4kL8tmv3ve8rpNmBVeaoVOQX8Bvf+u2I4ypqSSXrqAhBL+CvvAG74KJNDzvJh\\\\\\\/aMCL65Je4jYX9k0aIXe07cFdGxBkpVS5o0XM5fiQACFuTjsVII9Mj13qor6ytbOCY3\\\\\\\/TAl47u0Lwh6LKmaJPhZ19sINi07saDecm0zlbVY2OL3LZoiakKX2OpYaOsQTwv97B7B7XdTlC1HtY5ZMZ\\\\\\\/trcg86milzpjnvIS303oeUrzyCA\\\\\\\/WG2u9iPhbh5IznAyZXZ4ZbROO55BKIEaZoKCVWW8HdpeyJKRIDU\\\\\\\/rzotihowkB9bErJWW19eC2ttgoBhHcKmAVtPdyhCwopOCEsIgxUNlfcS2XQaBX2+0QSDDP0b7C99++eyjsxsgwdE48z2jl9N92JSfwotNLVzpIwhir\\\\\\\/8E\\\\\\\/MI8T0vLM2ErdXuvhp3mxwTVmcOq5p0Zbi9uorDhetkZNZkmMhV1DxQk\\\\\\\/qrEiM75sJkvkeCZPIFURb9Gqvm+VhYw65suHNpkP1auyatQfGd\\\\\\\/rHsnA8nnjcEKSnEdEUtkIcFvO+gZtbrLZU8VGiAXs2fEi1kSSft30qOBgs5SmrpDktTf9CPjZEfhZSIo0R6qMXGQzxInEWWhQAJP5qd9mZ+AgovgL\\\\\\\/fL13U7mIKUUoPAwLDINEhUpnXhNJVlmMREioVyhEmsjQBseRIfvH5DYa\\\\\\\/wPnYnD1\\\\\\\/P7Fy1XSv2WFZbGzJU5hZa0ZtcWtL5wDYqmTzR3iD45FdSvA+eVM5Ggk0jozHqvClaTkRYTs7+X8oWs1Sfr43HjM4GhgGqTUjDp4wjlWKbAsVp82cYqeaDmDr1N5T4ydEWiYam3L7\\\\\\\/kC7\\\\\\\/6Ih3yMlGYqD7xSIvTx1j0UVT72eaFNNVg9qnzF5vdQu3m5ULJvitLnyKAt5dY9luxWHLlKgOn3nHNb3wD7GmAjEiMYms7YMUCQEOaKeVZPG2HRYImgUNW7RY7IQXSFHBQHc4HLs63iLAe4FRgFpIN5dS7JnjMbUXpGdQS4FbG8iCMOYXVRB0ELXwGQTWDJJgma9A+JVTzUHQ9nsJmYVUoTD8HBzfat7QTlqbyZcj6bpn4soxXf7YhU+mu\\\\\\\/pT2bbh85eP+26T55lH\\\\\\\/Lefnx4vsr8EVNDg2nWAuA90aExx0VyxQjaWPykxQBr0LaZ+Ovw6gV5utQ1Q2lXdoST35ymd6zaRBNV6nLIcBXNBxEXTkp+7M2N7WNpy9PYFXPaCSUF4fHkgGaEBK8yZtDrIpi8IAqDlgRA6Q0iul9T5KPnmG5l8wjo7Vs3TxbCoXOSFqbRFIbn\\\\\\\/wZcupYCZQI79cGyRO+cpSVgdIhbDyw7q0kSxrvdf3P4huLVnOyR\\\\\\\/eHTk30fDW1gpQKGWUKkAlFs\\\\\\\/r3CZbSLz3VVA6FLjeckSkU+M23Wp47kV+xe57FyHiqLaM6z70NC+DatCZy5ZhOA8vaCbfychH\\\\\\\/wMDOnnvBd7LLCXxZANSjS5Hohu9JqCSXAM08j8+rOZhjBKw2pn5MRS+WR5IULKpSG\\\\\\\/Z\\\\\\\/wd9J5u+ykQAi2ijRfdk3nBrxVroG4H2QIVpnhP+dGlNwX+qpsTDKJmfCZ4dvo+5m8OW2ecNMJJGzBZRbQLTTyIeB79bo4TW9Oi5dMzA5u+l1KX+k1lQ5\\\\\\\/1C\\\\\\\/Kq1ZEGV65iahbEKdBs1NwAkLAHxDSuImKRYYRtCejTqioA24zm8qiWbwahcPwXOrlKX19syTow+wLqsvrfqLbwtYO6m12ZpCljkDIHDSHHn7atrgAz9nvVB4myAMd7r86pNY+14hlbzSJsdfG8f\\\\\\\/lOAHgshH9PwXU6xCl8mVDFYcpqQAb4x9f56xDnvYCewcUiWhb2nkymYtBpairk6xoWMgltiBkoj9kbP1cLGSIqv09lmSsDtNikd\\\\\\\/Bi5z\\\\\\\/vNvIxi5vaWN+uiCymyP5OZBM83QA1y\\\\\\\/WqgU3o61V+cuC3l0\\\\\\\/iyk5b\\\\\\\/c9Mi1It0vt2OL6UWxa2w3QD4imGNGgJKerMR\\\\\\\/+j9b9Sdj2vUBGTZGpMA4bB3KT6tOSGO4axdC9eeoHrEZlG9QGJipC616WUONjd5TeiAGaGr5NEpZUDppPq4GQ91nuUpWquvPaT+MwFb9F\\\\\\\/LAmt2Lllz5TeHxfPMC8tkXvAtfMf63ZLLEKQPgG81LMCC8E+7vEfquwSei0rCWE9wKrf4VGxzCseX2a4OgyWNN4fUF8k6f3wgKUdu1e74AxgpgHY3Pi578l6tU9d+64farVuOYD5hz+YNRGPhhyALbp1ryre\\\\\\\/PPKlx3AueMTOd8LCsuiLbDQe4BIjtczuCU9aCBDGlmAL6P8qo9IY5JB+mlFB24Kqmgm3Dzefz\\\\\\\/vvJBSn39DnI\\\\\\\/tWx+w\\\\\\\/t0M9wkTQ4OzKeowlM6+qVHicqD67SfiFCzERNJdFQr1PUvCRzqM5oMWzgnA\\\\\\\/+oXeFLmZKe5AAKjHe\\\\\\\/AveN7yFemVCtRLRbvrf6+Yxq8nOd071W67Rd9e5a3rI9UnMmBLqKBcQglwMiUaO44lK4AlSqf+rZLFF95sYMPIOhhL49mluMvYu+Kgocn65bUG4a4DLYCvp9xRR0H3Ug\\\\\\\/718PRyrs4j4fwN31nN6drkfq5uv7lQRqfcb0iJscI3t0KLJ2m3fi8R9Gp\\\\\\\/oDdvj7oDpF+1WDbp2fwMLZhQZlnqH5XdlgqmTTOoaBfiaQSlG7ks0zI9G+m5XFCqjg0qFkBS7X9SqrEKEaVas+Dw1QK9ehkHtHgvMZXNWc6H39lmdQd0xiHnE4VPapixsfsEBa\\\\\\\/L\\\\\\\/TGGrvLa+aFE33YKBYbDeLItNalqoSXeWXRLyvkcwmrPCi3aHHwY4b5XXKW5qwnykG7bLS5UZVgDsQmn9tWEwS0jqJbf\\\\\\\/ne2uOYT9NI38pBnufdqIX2XXxmngGqnNno3QexHLUugGWt20yFdiV42eFXoH5IsJKR3pgz2DGBdnCZB30XD53UbfEgZ997Xb9+EjQPMyNN74CO54hU2xc0\\\\\\\/+HKDumw3Io4zwxLc38Le12WIsg\\\\\\\/OlQUbk8Fq54JpktK6HCFzCKPSSB\\\\\\\/8SMePVUB5rOB95KdWOOXUbnZA\\\\\\\/2fT+ml8gdZ4NmGBqpw7MZYP8dbUrmtDtHRbIsQmSf3UA5I2y6zoJ61bPT6dlhBrvdzlagbIhcMWeMezw6Uu3mDCmD51uciPfARSR\\\\\\\/5pfBDMbbcewE4g4LdaTTYKXmn+9JnWsTICUCVlmy0ULYSAhy2B2usPly62plMXZekYWeb3VjCeXCeHO0v+Yx4EjEgW\\\\\\\/HMz8zAeXIhZizB0gLSoMOcFqSDGXu9BZOJ0yBIYLaDYBsXHZOqI2F835WLn4gYdUfTfTAV7EEc+B8zZUmoTjfj56anlBHDK1P5GEgUcrQLCagHByIIvwzEW45hXeHIFkQSDD1sdzin6HdPKGYhBNVVLv2Q5pWc9tUGk3N8qMg+XOo42wGNQRugT4EFJ5xuabEE22bNOFNbF7ZrR5x2zA\\\\\\\/olgAvr1kUa1lF2+0yMmp\\\\\\\/vvqVPelHFkxlHU35CtQA+zkdd\\\\\\\/T4hxR4aD8+bp1Liv0\\\\\\\/c5k087R\\\\\\\/trd4LVeDC\\\\\\\/MhRWJh9S8KAilUgXxfTCgl0AsjXlyxn5sLVQue9XhJFkAMb7Xhj0AlLUQuMcE8VYB0T9R8+Y4xmbZh5i8uuXxK5rbajaKzuUACaaUgRJlIVWYcmmNGfa74NlC15o6SmNT2JbJyEl0+HU57IrYmDsCRQBf9vac559qIhOm94V0\\\\\\\/p1ctQmS9UC5g9DUMir7Wje3HocqHmrhxnKHH5R5FY4+BLfcPqi7FaeNpO\\\\\\\/\\\\\\\/cFuLQldL8rqihmay\\\\\\\/0\\\\\\\/VekTWQrHc\\\\\\\/0owCYO347aUNfFJ3qusnulFBrxlUyErnCK1B\\\\\\\/K5ohQM5DeFiPyA3tHaYAFpi99K48AjiUO9TOTgEdUqIqoMxxQc5futiLWrETksYY7RqMl8XXfakkAOVXTNR1yBPTN7h4Jr+ESe1wS0ABgXkv+ImIERr3wTDJDCM7AfI0Px7DyiEOypA1+LHj\\\\\\\/IFhV650LfxZr0Xgsuyrb2JSJoZzq2CJw7aiWwkzvfzBYsrTEIkQIOidtcC\\\\\\\/CKkd9+EHfPvYPyNqSuQATWkMrKl4wc2cyBfxm2M7skAKxpfrK8hnoqTfM9qWXN+U9IvvJZpWW0a84PO2voW+BDL813mWRsBBU+m\\\\\\\/Jy3Rr8RbkRXcGxXNnr7WNrhrPOuE6LeJdDvwmdiQPezTEiw2dtirjgLb2KuGHf4JFDaMqPOE16TYA7LuT7CsRZiUfBs9j8WggZpx0dbr8PLFI0wFYrbYOCxW3JXzaDuyEbA5yNU6B1zmbhBSg6YOnYoWWvuWqRtpXapBfFBi0qOZ79AgIxV8Nv3lYJIxtL3JjaSu81fNts7GqZ4dJH5uvuXekbbRboab+F\\\\\\\/zMB6JSYQJN+ZdpIcSHg2GCStGn2TILcJ\\\\\\\/ghvgF+NBP59UA1CBfUFuCjm+SpmwJPcqylpx51hRZKhXwnTLCqzakak9uQdo8HcDDF9OZL5xHvuroue4ybdUyyWQkfohGhK11ns+oj1rswuHOy967jaeWSkZ\\\\\\\/gLA3lMN4cHlmTkY7w1nsG3Q4nJbiiLrjaoBbE2W4Zz03ghCNd7LoGvlVmBKSKoAW5FsdGMQUdJMQRbm5\\\\\\\/7LSUUu7SyGOKP+P8nRc8EaGwi4JDKpqr9kNvoRRbBukkwmert0ZmQJY3CBkl\\\\\\\/Il3W0pPLECHy2XB3z3FZhmPkj3ZUlJiItBfDYVauc0UxbnT8dl0ECnZQubnFmlr689d+IKtKQfJ28MJuWIxIAUL\\\\\\\/ltE9dhbCHcvGi9ixCS9yxVnltnPKCGXTIjgKEFOHTsdL4kaKp8K0coFhy9jcN4KNATdXAJCgDceYny1\\\\\\\/bqkQUvLReRTmYCjUIFDWj+Rt64FFONq6rttyxEHcupfr5INbg\\\\\\\/cpwUzaUfj+3XK+dZ83NTimkVypWWzlUuiDLG59RgCEOaPlXZ8\\\\\\\/rdVjkd\\\\\\\/A2dEYu0obb0z5Kko7DeJxUYhayU1aXUtBR5kjpLNTdDkqDHvCmU\\\\\\\/2MgeptnNxLgpJazVbD6cdWQCsJwKWKqWAaLR764SLWQ2np3EYbjYomwjtCunT8jyuT8osUG8UMl4rp9G4iJiiTBu8os3kAoHZZUtERf644qJJff3FA9Tol4+WPxMpnNrzjjRASXFg9RDX8IJ8jSh2C8QFzoA7treSWmb5n8I1vjfyg3rxMLOKQbNxe2452P\\\\\\\/4\\\\\\\/sU9Q3EclRYa0nqej68cCVGzNBLWAzoyFa9Q6Iybsl7nmnP1j49iqtJvNzdI\\\\\\\/Qd\\\\\\\/V2DWMqqtaLX5JVrSQR7TSnGreLh9\\\\\\\/b5WpVt1mjQBf\\\\\\\/Yo8pqYoFNh1EQn5q3tPa0qqhiP9WJuj2krhgtsofb+mc6SIKVRva9HoyY36fH5AaN0P5nDi6gP\\\\\\\/2EzueyMVecRFkCIiVVOh3IEXkmXm4S65v8Ibu4xrYBwjvqm2NxiVlnLCwoDy2i4T1ncB8mBykuSTyA4WTkt9Et+ojTb3wss5sf5Yi3\\\\\\\/FOfUJEMK+fzrHrDciU9ADg0SbLudlJpzoYORxaEvN+wWgKUhbBlePyk5F9nLyfi9SoTY2KEngu+3oPHL3tIwVP6Zi1aQzG2IdUDD1je9a1BW5bbItziFEQW9lTyAIjcDC1wcMMOle+LAjZBgHlz6ri\\\\\\\/eqzmDvTRfXo1PFPvSgYed9wBXKIdOq62N3iD4Jng0uuBt8M3zNag03aT9FcY4oDPw8KqpnNI5ZUBBbw6RuwpErv5Kmm77in1PjCREM6f5E4LyNRGxR7irz85a+E5wgvHl8ceIVH+1CLWGzOyyTJUaNPniu844tBTPqG39NlZM81AlDuNipqrYzn0ruYkS1rArTabm05d+OfuLWNQ+N40ZfYfZ18jCP+4DH5kU6cm0K0bRm43MZBQURvGA2FNfIdGZyBaBVzN5qyHXn\\\\\\\/P\\\\\\\/GcnTLliQEZkj5wawFK6pe6syO5fuDY00GQlsbd192b\\\\\\\/otpjFSFTjm\\\\\\\/U0I2XPzSSSBgz5SAx5xOsulRKmiallshkxHHicX51OGZg30xcoKYEZS1fbJnL\\\\\\\/tzjSwKBn1yskAcUxhrB\\\\\\\/aVMYD7U9OgIo4X2Lmprq+loE6drfgSPB1nGFUdeo4fPD59cEPNBalXEy6NsnikV6P2jAWXHrC+BdQ7iX\\\\\\\/oB4a7cLcPkz84Ag5xsXLIklqCpZVxW+MwxY3t2VZuxXBz3T5unCYBL77Gob1ik+IuXU1C5GQO1PZEzN8fMuk+3tGMZIBwI+OKIJlcErbseI5NvTaFt8wK7eQzGWAmooyk3uG5DQ4ybU\\\\\\\/uw\\\\\\\/WIwmW30SaNFz2b42s5rZzdIKu3spx8d\\\\\\\/sZ2QmyAxUXp5952qiOxJDIcc8QYcMn2N+d8JPnSQ5hE064gay\\\\\\\/iPNlFw8sUOZorzmCeTRl6FfyYk3qpO6CaBsAFjsxjfANHdviT7guzuW\\\\\\\/5iGIeb5TEKk5cWc5EIrnxeYwxg4Ectz\\\\\\\/aG0sHPIaaw8M28QpUkkd2e8L1ZShayJ7peUoTEDDRr32120lSMGt5Gavq8SIRstUwXpcRJ4UfXyONWnHRQk+aBhfZaUfv8Eda1Lnsl8nlJay9UOx2VgYBVgo3pd8Y4LtWfepBGsdN2fQnXvnG0kdEosSGRfCBB5RhD5kGTKMMJonkG7sOG2fV82pneXeF1nVOs66m7ORJ8T3WsUMbfx6TNXXoSBfV0M2bNztmEYB7cuDoRU6gLX3dexSfc\\\\\\\/UpXqmq8kwg4mSGXpIiLuMHrAqmjXMGESR5v+A0NKHcVFyONvJDuP4iwJT5LWucTLWMHbDYTLZhCWQQ\\\\\\\/MJQaHhQce3p0O0YZX6O3OXQLeZa3dEppRON7FVoprqSN0ezjflHvZvf043SJClZpf5of0GfE8R4PmVVcl8ZNGB\\\\\\\/8SKvfNmAxTl3MG6Rcwe3izToxD50M9reSGkiJNvU+4O6QVPKcWFweHD+nQqEsBLjnyQoEWdfDFpgFqI5d6mtjPffb8QC1qLNAgblJNsFAId3ZhzEusOsBUyj1X4eV7cw26GAMHuWtgg\\\\\\\/3eORJN15c0dCZ6mH1BHFyGDA+Tvpt+I\\\\\\\/HoOJKCuAwUB5D\\\\\\\/DLmr+4SXQseLOQNhtUH+PKrSNjyiIx4WwDI80jiZbZHDAfq6WIccM3yaIZrLdB7GzrUpPJpvUq\\\\\\\/bfJ+CTJcLEyGrIMFS9u7U22tyZJoZRhm71BQDPwCtr\\\\\\\/0RCBBMAJp+jCcyYCZSVQ3RSEy7U\\\\\\\/FvjPU4FHR4njSUH3FLfhWhp7VdEYq\\\\\\\/VXcuzxD3uWsNM2noOIhIQQFAGAlhX2G3lDrpgmFUpSu3goDokED8G4KeVWH\\\\\\\/HWPwan3EOjX8nUI9KtErwDIXe2J6WmY+2oEXLHAfMRtJgT3XQr3yyYICG7bn\\\\\\\/pDDR+t8prpcWK6PkCzXYn\\\\\\\/B2XCa4Xp4IGC\\\\\\\/ixRKmkmirNEeaBIoF2Peuf5lXQnfpCYD26iAvwxA8HwNs5XfwTh9xAhy85yXo8OkoBiYJvW+\\\\\\\/hD\\\\\\\/V3Pu7ZWG\\\\\\\/DjBquDdOjozhOWUM+5OgvHranxK4aYgZHrTFxR3zo1hpkZA0+628QBBXfC277yxiFU2UsqQXEE270vF5qDpMUzF0Ydu+5dGl1L6aWnSqXfP0od4BmgQe8Uxuo0vR2mpc7kwUfpSq9dVWy76C2NC8cnfWDJjVjrX2pyCXurkOXveaw1BWACF2k7QpMUdPHUF4cp\\\\\\\/cUD7mAFgfRD62CXjArWYUTnLrZvm+qZ0BQ+n08TxzbPpMd0dCNSWd8DKicx0xJS5qwqGSeNE3KcI0roAZ8JZzeDo\\\\\\\/vSTFFizv5s8Ky2nVE6MJeOy5fDSTBN8JMNF+b0JnyPeq58H2P4jGxWonYyknVQC\\\\\\\/Qasi5lM7j53ZNki4tt\\\\\\\/ws7W0Ua9FNIjUe0aEgq2jyCW8wT\\\\\\\/e\\\\\\\/QiNX9UwYKKPRno1qSLbZOVWnLfGFMOfsh2pfKpVsAhQCKNwQ9YbNq5DvsxMJrqxayT6k+aUkUUUy5KEPYOKooKGMVGmtShtFTxZDErYfYQxlVtel+4cCC9jcBcYN3LVtqWXYAoXV43IwzghXTGgKxT+y6cU8tdKG\\\\\\\/GMtTcZglUQe+IgUahDxesyBlYrdinbNzM8boOaZm8F2QPrlCCaX7Gbxy3Vr4iil4mayWfqp42kSP7AY4WM5ztNWKUkllzzNPq\\\\\\\/kkIz4jKzDE7i\\\\\\\/i1t2UNOrm8qZlt7bIbdeXRVHloaM5JUqGO81S0s7r0Q6A3N4YD6G3GM6A5bStmX14193GxlezIIbq19zFvM37dH6\\\\\\\/6IstXxCU5s1oRQXHnleHt71n5vwWF1uBbTssrOnw1pZuvPSO3AbVAZ5Jf5odkQBQQt4uNsUsjbfp5g0w2C8bX5cUDwPPl83VPZQIng6QddsGbnIpUZrURYAmTIUmaMTKusVLiPrrWzmNObVmvSh6nG2HS5WoU9sgBpsFptKv9cKkyYo7SeiQx\\\\\\\/v9ojPrayaTHFsfL3aRfFq3YK888NMBwdRzXuHCQU4Jt7lQ1b+fUU3UGIe8oUhXNuB8Y42vcdPUBdlrlxe0hv4cEr6LO5vmeOo7kWgd98bjj66BPpe3cHF+hF+XlpY3J\\\\\\\/X58hYwQYooK93Mr6Tv4YhhMiLgs2Ia6WBshKGwuNzSMjaqLosGSVdHUyvSLqoDD2j7ZvCxCC+IbOEbalY9A86E7RZcvI2Y4ADytNa5FW0rouke1qU+v6Vj+OOBlO49kxjCdBQWO0kBV8d0PK\\\\\\\/aixFOitmYIXfNte77iPqrENbt4qtfeg+kw\\\\\\\/Myz6rwmfryp1QZ9X2W76OXEppe2viSb7cNbHsSFuZOir4gMr0TWU\\\\\\\/FmVXVl4T0LGq89wK7N3VxdDKS\\\\\\\/z5WBysq6GLSrSK8a3n7RDSQCHf7ZrXUciC60nPl1vtaP19Z+6eUpnlcOEUJQrKtLEIMY13FXfxyW6WFTiXJ0yEYwGVgvLzGo7Lv2ySIVUOe+l\\\\\\\/9H8ec+Lyg4NwxxH\\\\\\\/jmQ3P\\\\\\\/LhCLoC+PNb0mnCP7FfK2otFYIxPJmlIMtNm3doxBdrHgU8PK4bY+efhCbhSheL7kjE7t8EYEtTLqOyr5NNo6ITY3nm5RLf0JinME+uw3uGLNZ\\\\\\\/lfctBQiliAHBx9ecYDC+dBLOdAensWc+iNIA\\\\\\\/YpEu19XhzGXR1wqNrydVbuks2HvxW0ta+XJFiCYHmuRlztuHwnirW8XWDFZxImEJbXL+BQZOnNxFh\\\\\\\/p4A0Me5VaoguidrVntZ62kxXhFDEmP7lGVd4pQKRqEoXNuoinFSFKp5sbZ7Mu\\\\\\\/NHSyPWsjj+IECF+5dZHwxfIJ7BoDGvRUCBXbtPXvtjlW+lWsdjdnDOh7bK3xhZgspVCvoHAY+TfC8UGlvUDfpfWNuAUjDGEQT1a25jfDANcOL3KUl169yhTyPFLiigMD7CzUzRmmGQH1Z0SVxarAhPz7ZIgGHLTsITmxkTjY0tq7yeJ4+l\\\\\\\/h2HXqhRiR0cM+8mJwK\\\\\\\/wn715SM9eKhwid4Y5vHVrURVIgK\\\\\\\/s\\\\\\\/v9UuCFFdDsVdg6aJi4A+0k0+nVzl5rV4xLaW6ZwpUqMeKlTALWRl\\\\\\\/n\\\\\\\/fJ7RwHmWuvVVYMuwH6ajgfzTOpjaOLpnqKkLmsPd8gszRjoMbViRBAOwd4Y0ig2L90EJo5sQEBGyfXB3DxiB7+kdMv4jMzlUY+QLmh2MpQbTwSjY9iPW8s1c4NXKr8LvrMTr0EduAuPVrjCQam36mPcTlm+hf7E9hR+Phz85Ve2AHi6tXlDDTmKZuBH2fRl4m6ZKAhRZDAzLNExPUOkGzBNEyi\\\\\\\/WbLtRDdwNI+vQHUJuhMjoYaFPMk3bCVWPOqMyIYhBJWqQWHEcSmMDe8vMdlQFVFlI5A6T+cuFyWKEmfag35zT8BwfHMKkEYjAUyCSX4q\\\\\\\/29Dj6htDlMnG2npk6ieOmmp2xVs3PDdlTLudmfI8KB4tO16Eh9rR6k87lTtQLyyQdHmVPxXkZ28VvhOMJ1c1AbvEIG7b9MnmuySF18RURbYq+pFrNWTZoze0+d+Ve5SsIF6Utzr7DGj\\\\\\\/QjJsxi70zXiM2QhJy2GmgU3pICl97V9gSRqhWnTK2RhmSPlv8AoOSBecM3\\\\\\\/V8e1zn64P2IlIA99NXSK+AxTNKzvy+lgmo0fpARppz8IsYNf6cZMuKWeju6Z5w9nZDYOGx6s1y1ACzmqA2fRVYCTKG1z1Hr765hHukf0Tx1kneqoAdNYdYk4PFSvRalIbc22Wj7PErzdK1PZdMIZv6eo3h+oGgLUDG7wOBxoOS7js4jewl7l33s9kYkLqB7kDO4ASeBE1iW5mSBAfSTqfJ6cFXoitA4jw82huM3nWheU+405VCHphMcTrAA5PaIzHdBA33ORO5ywm2L53WiJB\\\\\\\/afh5tf5ZCefIqwYfXy5eUa3qFcUey5PSTUZJV5o7VZ5p0z9EVRbKzZsQA8AGww1KolKjn3ry7hJxAC0F09QURSEaYv\\\\\\\/SWYmMn2+xKPxDYag\\\\\\\/vcLzK\\\\\\\/UQsUusI54maX\\\\\\\/Nm8ax1KwrO2wJuTJOqND0hB5o5sujZOM60sDgiQWRgqZ9f7OSER+0Z4+ouiOvZ6DVkvmKq9unArdcJ0I3jqbBU6q1echLQatjVmSZLDW48\\\\\\\/YQKcN03XSceIqWVcfiAH5kVrVSwr7dOAK2oaEfSD7dFKydF29\\\\\\\/uSwokKQbyG+MQcggq1DX4Hp6vw182E0tlUE7htE9m0l8TYzAG02d1KEfbKCQCB5uhdmdWl3LMQeNrWRYbAYO6fCJ0JrbXcA99nxatw1Kzix+euFchAOS7IXyeMg0X30wKeRGs2ogV5nSJl6yU56y3Apuxl51nvAFwKD7EbHRDiKl8qYFSQYJgJ+0ZyU97cke0dkWJ\\\\\\\/rkKfL9YdNQQGgUduUvJ1Hm04lhZT+FWxxH4gC8yMi205muCYYT2J4gHgi3pQKozV8eA1jJStbU3ITE2dKIJBcj9YqwG151UJMhDFwfLMupOlUoc\\\\\\\/uLzjLsAcbhw6gOFxSp0AFLuZxrerx8jRY8E3i+R83IL7icgOuyFjqKlYBKVk84sIeoLF3HigytqGDIvuh4dNnASxhN51q3cbXN2nq5Fv9Yk0moPVN0Sw8M1JOoROaAHlWrEvGUIRozH3Ng1w0Wxu5HmzQhD6mB+i8a9unArKZD+8gveBEyG6jhsae3IqoJva65R+RblAUpPoeSRvcGvxELBQFBJuNnYyQ\\\\\\\/NBjX\\\\\\\/ciMKwJuaBnl5fJZUVRCIqdwirXemV4bxx\\\\\\\/fVZn\\\\\\\/miXGBBYcBI2hTKx1GY8illWjHKXmDxCa4Zxmwbz+lNYvNOa4tvZL00LllLXEJI6P01jROBiSyhAdsv0GsNtS5xCa8B+ObDlL6nBaUMGEqmipx3Dsvq\\\\\\\/mYdLITLNGpu\\\\\\\/6iUVtmFhWmqeMjUIW4SKOW6z2TXXzSN7+PN+Hi\\\\\\\/4qCvdd7bGV6Q7YZuLce7O4KQMHYRHGodwMRwaa6uicDwbInXU3iYqJTCPEvJk3Qsbv+sD5tJkwPPOKD2JDfjgiWKiy6qjVD7yAIfkkYXs39p4nc7sQAKGsnoYIuhRerdEuPh5xbVsrfM+i6p7LQ0pRtrYymOTqUHAycM5Jqga\\\\\\\/Nk27AaFkBIVphC\\\\\\\/3GwEe3jX2PhrezpgU8+31tQWGvhwFYzx1lUPf5ICJgwTfH1VrnjO0vg0h6iupmUEwlNIGwRqdvL52Qayq+32jXt5XbXgxGZxOIvoxjP3mbzNuNDymT5uOYH\\\\\\\/pvOsNg1JGzB\\\\\\\/no3JrqcEQT\\\\\\\/ItwFXWMfUcvCz11oZgKjdQZH\\\\\\\/WYDDWk7Xg5srqnhbnvVD2TTSqbhZhptQwT9c3suH+a3FRQ59TaZVRANfECumGGxlCcv0iUX7cHEC7ZBdzi33s0a1\\\\\\\/lUiPY7WQgzXkUVPxdYEPuc+N9mTQLbW2Nnh\\\\\\\/ZNOc28WnCRCMcrDG4U1VX0PuwUl49afWjgHJJM3Pf10GlfzDf3JQkGVsZ8EQ7v2Wr+Ylo3WjMGRFO1XNMuOT7mRgMgMSKvB4AnNZoFGO5FRQe3BVonKmTaWABaDeuHqBJW\\\\\\\/xh429PtfyMIRfLROCzniFoQdNoYVJ2OWlEXgdaCAbyqTm4s6bouszq0woXffLQVgqI5URwyDizCAgwTo8Pge\\\\\\\/dYOT8TSf6COou82ISazE5Kh09MJBc+YRfc0YkrzGmWsC9Ow8JdGgudQ9r7dkRXwkT2d1J7YTC40Y3X74G9LlXb9kMAZyReNFJRUZJqR0SO0jxx6nYconjAd2eQTjocagUNeVLAuySAWz+tbRhrIu8hjv4Hwbsn5fUhlCsNE\\\\\\\/Jo7F\\\\\\\/4cMdXQU8YZHIImuqr70BWmEn5Ho52rgFvkT4LU2bUrGT83B9rHjjE+ncLAsXs2eR\\\\\\\/dYSQ65Fn5JmWYa9mjhr1CS3fMuDstYK2qtDv8kWgQgYYrOGwFhArCn0hXF\\\\\\\/I41ai2N+nowyW2LYnUm8f878GDXp4Bb0jSSIlWgnJCs3qUCMTbVl77JgKx4+t6JQO0rOvqeylWXJJifPL5KfxN7xuMrBCPxTf7O8OyitPIMX1tjQOi0cgW2aGllPNCrfsfU2EHXOIhptA83GkXGpaD3vV9yMYlQPWcZb7RdDmp0K1IqEGZJ3zArxRNOnl3KAZQHvIE7XeX4LnuFBeByHb42mmfjq7jpD4GqnipMFeEiVZkv31DSGpPFwrui8p7zD00oyS6NkQCks7uZuaKSPmkObCaLtY9A5ranfn+3irn+baNlzpDASr0c9jtamX26Nip6HdhXTj9D4B\\\\\\\/MTBuwPLWjY01e29aZ0C1vG6UJuq3MVqeK96yuafkt3\\\\\\\/X08l9yYYmbmKu7EOM50DXX9xnTQ5kb6Z4KvU4c6H3lHMizWTppaRYMn269lOKuiALZqLOytyGSKjHQtFKgXETAWhZQp1AVLVQ\\\\\\\/ZHZuIstCdW37idxpM9yq\\\\\\\/Jfvaf1yFgDWHGg6HSCYVuVqFlFJ1a+q76i1fUYwtWF1BdN9yVPP8zqf9RK09TyfxPYPJIm4D4bamHz5WmUY2rdtrDpgIqK++U4P94F3M8aC+KbMKRoWfajtGxC4+r4mqc1U\\\\\\\/5J0ZAQTUAoFcMjA85ByuWvxTStS23C6NdWCJ91pX5aQrEAp21ANiJlkTtYjSekwaNHonqTUJY726En\\\\\\\/3BPDETcDbyxnGpP8Pq\\\\\\\/VXD4xYICeA5W+VjnGd8aG+bFnob\\\\\\\/vPa12Ye0M3e1Gp43b2o4FwrQwlyQIOHieEMAf0LCkcddN8tUpLL6pGab\\\\\\\/lb41IFlwebIrEDPAW8mn2l7HAA1ZI8I1J6EwGv6KuVkl6r9J0BMkZqyQ3kMgTOIxsQiM9N7J4ZYLAKRgqfLgfZz1y4ksMJEgBf4ohnxXdZ7LnXc1C2g68L4XPyRL4HVvrvAJwCxyJuZh5AupdhBdU+RxOnO7wRTeOS5GSzvR\\\\\\\/tRIVItj4X40IXcFgo8HryxLstiUVMxtrJ\\\\\\\/H2x657EFpohLoci0+x6nntoqmi0pEeZsl9foccO3bGktR2lhMkzioJy7\\\\\\\/fXtNBINFBx9hGlv45VCG9O\\\\\\\/YIidk4cxF978c5lgI2pIjkLzVUlWsaQ3E9cBkx4nLald5ePW9wSftgP\\\\\\\/vAl0iNEXqvvR2dnJzDzizj5HTUHwYu6jRKesNOxc0u2LwrW76naIwtYwFiwt4WI\\\\\\\/zidQUAvNNpXE6FsaC0v8XKg19KFK\\\\\\\/vrmeAFCEW0GV2NYfX+Un3ige\\\\\\\/9YsiJqjjzx1ZsFvzi8in4kM7Xdvq2iRfQbk+9dvST0eSvaafPVM4kAggimotdx6AEw+TKdsUaCj\\\\\\\/6zDfrZEer0gj+osYDFKd3Rb8xWEV0QW712ZinApO7pX0choS+9e3jxn2czcRfEID1ni1v3o3U9SDTF3Zzq3O3C4wiQOAjyJu\\\\\\\/Z2DuagZCVUKuRet3U\\\\\\\/2Uxu6oarKE2tLIAwKBWSxdDNvkbnsQoezHhg\\\\\\\/EgXn9geZM3ZrEYQChQN3H36Nx8AHLAGoIwVhHBQejSkqSmTm6kgh9GJ7XPJY48gp37h\\\\\\\/xRj31riR4K0P1WpQzygXZonN5ljyc5lbDjVtDaPENun2XtlaZoBaKohpoYK\\\\\\\/8XQGzvvo2WCS9tLmzz0na728vmxeCCXe8xLviWXxLxBpoCRnBAM2veefdwjqzcwigLYXlAGj6whDC9OuamuyzadMm7xxCJAQEKs+h4bUdd8VphvW0Mkc6uvopkJUtlMBSkHo9T+PwQWx+DPSLoSJVIRWVEktYp9j52mX4r59phKtefuRx9MddoAadqRTALyX5vsGfSFCbDDLkcWYDE+wRxlI8emouYop+JeNY6dqtHmZdIYlm1\\\\\\\/bZOM4jE7osulRqWuWlJylr823voaj2QJzaeXo64GoWQudBhGm7cM3cPjRZPgmisNZs0GTNfu1qduXAA1sJAZBUvd5K71QdN6kC7B1HSdRVMrRfTkLZtSmj7U9tEBZWjN3Xx11zFbZWFAuCnXW0o0SFhGy4E6+OrJ6Xa6hzqDETVJbIjqCiOz0EbJdNJW7dwkVUzoLknmjhh9ucJgMRKGOY9yje2ypDY4sw3tHqcCMgXQ8e3yOEnz\\\\\\\/sE+byIl7MqXttKKRCHOZvin83PHlKqrKYNlYuG7iGqRN6cRrbe\\\\\\\/Zi8\\\\\\\/xTs9++OQZETFOvraiEH9xnEi7Jwgn8tHJMKDGPYp41uJf8RctMVrU9gGOMaXKaEVH1Aexlq3\\\\\\\/LEqGdxFkMH1lu1acNzoAN1vDHdPsYYE9V5+Xyvpqrgp4agXiBwWAAIN+4BKXTn2EqRB27m30AGFL7vBKHr2qX9rmsEII6Epij3ilThQbAYfuziRvTmlvSeSW+f\\\\\\\/nP4GkG2vRHB8QfsIc+ufZyOyxBl7qziSIfvllEOdXJhXH38fhj59KjQahDXnTiSMZq\\\\\\\/XSZZSLhCDP4+jJsZYowbDkKHiq+\\\\\\\/\\\\\\\/OU+27M4L1tjioe+hIPjtyewDBGpgAo8\\\\\\\/b2QO2dKWPifNeq4vT\\\\\\\/TrFopR7FmWVjz2WOZsKPh\\\\\\\/\\\\\\\/w6bmUOMV8y\\\\\\\/CUUvXMV7z6Yx\\\\\\\/Wo3maC8lI+NOcox\\\\\\\/zXXB487Js9NbQMH96s2N1VpPHHILFhAHz9YTeBMSuj8iPjW75YRuTVc3Cag4E0e\\\\\\\/+Syul54VTijbMYoydQveeYBy5N2jvhbBgRUX\\\\\\\/YrrJAtJq7yLY0HTdNj4sYTxNPkY2PugfKWE591kk1UZBqEfc1PzE+Xwh+vjeoEsIwN7yCjFWziIrXv61JxGYrlRpgLHGj\\\\\\\/GGfbRR5d+E4Iq7zzZd0aO3CEerF9FDX0g0W7S13BtfT1sr9mY80n+XtkTewWKbNb9z1bq6Wedq23r0+yotumf6PWJW+YIatV9PI+ITtKX\\\\\\\/5mFjwphXUbdVK7SuUCYS8qMkt+RbLDJOAkM64KIeLhM2NRaRgw5I7gtz\\\\\\\/BCa14J5ThS\\\\\\\/42Q\\\\\\\/1ZSoxDxZl8HcqJdD+0GpaRVe8h4wjVA6OGetgMWbJWPxw23m88dOS9xiIxbcvmy7ppXl3HRGpnEyusjM5NudNQrqiAUJ1AB7B\\\\\\\/NvYzfvlOKdnjGhTkbV7XZEBP3W2M9D8FDW754wCV2lId2dQehDaT60nWZyo+HaHScN1P9vbQoA3SmHQ9k+QJIj5mWsBjC1Pb8cT8PBzMXtOkXNeVMRvQMkx0EDmM7cDf7dkDsG9orVAHbCxH8ZUBUm83UM7Fe0F92g5vpruELL+5HsVgiAeZufMIyjYn5B+LsXOJ\\\\\\\/VaUpMPPv7vLtgZcO\\\\\\\/vPH\\\\\\\/gJjng86\\\\\\\/WnLCV1LDtBQQicnd2XRdov6XRAw8+P7Cc2nB16XKmIa96ABNDg2Ct7es8fUhqrCxj06fIBzAam2zyYBayb70jWVtCtOcnHKdf+wokKNRjMKBIZ0Yd\\\\\\\/vvbdh8ToAooyAfrwl9CHjuGDtsQ6bx1FBVWj4yhD56fwtGYUJKRrdVkAKA3VjQgayLzynWrilShAF1eHnhZmoFPxnG\\\\\\\/NX68uS8kqs3XsKWZCUk002P2Qsj+4J7fZGg+O0jesaY+4g0zxmN\\\\\\\/8XioKEeexTV1MP1kgv87ZzFVI\\\\\\\/3XTCOyYRY204n9SOOb35zZ23Ca\\\\\\\/ujplatjrxHlcqGhxoBHmHPowwgZ6F5V3DFcToCO+Z0Y\\\\\\\/alFEH1szz3xqss1JdkQ1gx0NC2APzaa\\\\\\\/7lTW7645J3KUQN3dlIFCmGlL9saCa0bnCMWG1e+R6\\\\\\\/eqlQj2kkMKzsZ1fjE8vlZ7Pjg96DQz4WbDSuDyqAl+NekXcG6YrhX9JzBAxRXuYt54jDd+Il44bNl3eo1bb3xcfafvqBLeDDiI0TRBJEDZHBzqVaDPcKd1OTYBHAF+PVF96PxyU0CzQSz9sTIkFxSUPC5Lbj43a0qFMMvmS\\\\\\\/zOCVFvndO0ho4+J082kTdcrBoiRwtFhrMnNS1JfSDuuTy5hkaKv0A\\\\\\\/p8bSQ9S2ugym1Hgd03WFFV\\\\\\\/vgvzf7chQcpdxDZhXqsfLSd1e\\\\\\\/OXPV9nyVQ5ClZUN7o83WOMkgX8tXkTrqoFiUXmjjbRa8XAJX6iUJMH5qmgX9ZU9IGka2jwH65moBhsPtb6cCFh2\\\\\\\/5uzA2svPA82vkLpuMBotheCdpo20FmC55uHpMao6ZaUjexqA+uftgdjCwSU4t7hldjlGCXHyyy\\\\\\\/z8ekcTBf65RU4l9HaXNSmhSlWf06duh1YBg6tXe\\\\\\\/52kEVGt1R2qNs2NVbGkQ57Mcbw0Z5js4hmu\\\\\\\/43quwLMoi067c2jzN9cytE3m8tjdzebcWr7jsKy8o5hZOyl8magRt\\\\\\\/4ToJGzpIrfAo8eaJ5kWBKnKqWRTPzHkQGMC+Sv2HWbPixUauhE9fl4dvKzvSR5PoA83NhBTL9Jbl0dB4hwsO8JpSm9KUtKD4J0FR3IUnKDVzYBg3UnJsuZMqiq9DUsjvJ49e+CArsCSFvAp4Kpu8T55JsFTHaqfo4DvfSo5Cc5vTfcvK1Od\\\\\\\/Vt045Esol9RbdKLONpL6gvEqj3+H7sSNDskpvV4T+w1jA++GY4oliACOidy7a4z0BuDcimpMSCKP7PjMOZQOVWZXVTqQF2VOgoU7NhHfAp\\\\\\\/NhQi77AjnsLSEzF0eKrh5CQPsDeGFDrnMQSBnToZsRvh\\\\\\\/LOtbSv9745nI2iXHlIL3gdc2fP+p7cTv14Cn5\\\\\\\/SA7L6Yn9FG5k9kOI9cXVz4aoZqEqTEbnwcEPFGaHSsPb+GB8Sg3vdVWrNFp3PM\\\\\\\/sjj14wJn4e9FvgxtFokQajHqk5KK3NtQhEwq8ZAFDLplb9OPfW9iBs\\\\\\\/VW526W87u3428KE7NWqn0BWPTrSJRE57rlHzdv\\\\\\\/ZgnK0NmwdO\\\\\\\/EdnDJkra8r4BPMKk3ivx\\\\\\\/8Ppi1Mv7hcUN+IYd9KQKrKW\\\\\\\/2Wsm5htr6T06GbgWtdsEogrlr309wM6vz\\\\\\\/bpA3CHVNjMMsv6cK6bd5oFoHVxWv73ndQ5HuZblNhp6wnyO3rqYdOoT0m4U7ve9kLPIvQ5aefbNPd9peFnOtZcpEvJoidp2\\\\\\\/QkscvdyQh\\\\\\\/Of54tsXU+P688HkEm85Nz1VGWzKo28aMmPdUGV0da6etkoTMPgCRUth7cy5NGRqo\\\\\\\/6U21Ixoi1T3cBhoIr\\\\\\\/xAMBSCuLQIceEjmtkzfA+CLnOI+dFaABs+NUhb0LbrOS23z+GOLem+M7LJUSdi95rF1nqieoQ1QGkVjSrqMHEH2YDs6fCl+9c7V2mnfvNmwCyh7ENGhB9tE+aQxBN7CdWC96m2zXw\\\\\\\/J7CJtVI6NW6\\\\\\\/uSDrotTwHy5DGbLWIuuGCi6sBNsJ7gi\\\\\\\/+CNq7FwfcKFMQ8uMBffIsWQTvRobTA7WvRPVBDeYqNRw2CNNU6jmprYTx2IVusORLZ6W3en1uJP8p4jCfeO2cIlrV+I9HVghIcMzBYUUC1FAW8dX6FMFmBi56lLJoLTGBuDTD+s9Dfes4KcXAEetrBjq7rQGowXAY\\\\\\\/U45Jmp4ZoIHf\\\\\\\/9aJArJj6j5k5fHY\\\\\\\/eIhi\\\\\\\/qciqxTEFCN9WcwqCRiM3OeyD5vNttxb\\\\\\\/8lzvsn+xc8IPeFQGvht+GsPCfPzeeSqi12aRUBhzMC2ZJthkkb\\\\\\\/OnBZWErgCGbm92YrzZs87YD1s+g3dLJJZfEW\\\\\\\/NjrKQZwMvcArlL0mCSXYS1SietklDCz6chyIgrl3cE13+OP3z2rfr4eMB80E17G68o0\\\\\\\/OdbQrwigSb1H9w3aHTINFnAncd1TaiT9Rm5YqXNN8MH\\\\\\\/+zwiexMnoFGPgcsApXegs\\\\\\\/wqa+9zA+5L3DYGvXRwJZGAO68q1PzCFfZDUECDfWrCsIEyOkagsOdu1osjQOjk3I3QgzdrGNiaG6GtP+XKZOQZq\\\\\\\/Dtnw6jPMg2hI0WVDPhHwpbJ98NRp8QEeoRwPsnXN4Vrym+wM1e8\\\\\\\/hNoDTBktw03r3uZIPyc8L6LCdahyzR0TWGMSXzLDYvzGCPx7Th0UG2CTIwBvnoMK2ZtmDhqmURsqj6lUOnRRvSMMUm4HE29a4irlIl1lSuFIlmI1dL6p6zsdQvmAry0SacqU6FVBNebJKa55HIDYBw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kts4j28Kwe3XP5xBsOsVYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ffb916c7305f8eb0f8d2d9f4c298225fe1c78fd26ab7a07a2e9e4dd09facee99\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T7V+nZSiTyMP18Zm8\\\\\\\/jdXleTi2Ptf8KrFvI1HMcd3cF\\\\\\\/71ZwLZJ8icdnZFc9kOOLSJsMiAGb9uLPBCnyDyraoQYJo\\\\\\\/\\\\\\\/Qa\\\\\\\/R47b8wk3YwvHkdBkWAjCCpLnMtyFRznHituiUZGcY2+EHJddsztX4L37wSo3Jbt\\\\\\\/Mzf8ZBj33gl\\\\\\\/m1qYg+MzroIEfe5SZSVHKBkGvWAkmSRvtj336WkVxJHDPaY3\\\\\\\/RmJj5IdXy8f3TE0zW4n8Wu7+rWikiScYYzvOjsYlxKZVgWQIt9x4zD\\\\\\\/1G\\\\\\\/unQhVtTjIQKF\\\\\\\/ejgNdrvioQfJlrl8GcKJk3N+e\\\\\\\/GFft5ImbO72opI2Ox17E33Ok6UOEsTdrNRQmPE5VLa1mDgOs\\\\\\\/XUPJB0SEKr+y3qFDwTjWcvywVwsCnw0LA7kpwszBmQQExdHhGEoE\\\\\\\/7lS4Uh903vFpovkMQPTTelnycUqWSfszcx\\\\\\\/bk4ETP8yPitsBMk0Yide\\\\\\\/RJ1FF3di+8GbPSee+pzifDVXTnQbY1diOsgyzSM06dLPQZlI5Jklo0lciWKeqz+NBAR5KGeTmMGDKbLaHqbDnyLk76rIAysMh9OdN6C5RRKD1CbKRxeIKgebObmnrumxGu3CF\\\\\\\/hxufrK4f1MyRpnQkvmJt31ARId1mOA9WTnfoj8GVqGfTE4Tfm7m+ulghKibW3FxtmvXnALpqku5lhUwa157AkCCmSZxNWdCsL18pRwk0JRBqPx23GNDGwjBwxqtYgU6UCfye1LaTBXqsZYI+BIkQAAr75KqTVNNdjhTyu3ItIkOHt\\\\\\\/Jk\\\\\\\/5E5m4zK5WvvlMtx1P5D5l+B4cZT35U1D0RB+u1qzdaa3RKZ5JXRmMCI+S2MNvoB4cyQU9P0XFmx2HiUqLFRRQ4tn6nwVifxWrpCox\\\\\\\/\\\\\\\/iU9PAvKw16DzvI5+DY80zIdBSFp6c\\\\\\\/i5+Qc8kL6\\\\\\\/Woe9Asj8IRFmB7eO6dzcV2Ifm+DFJteMIgkS5yC4Zqq+bgl7cuXyLPVfx\\\\\\\/+brYtf9+BYX3NbNvEI1KAWyhyCdwao9xG8TJRA37F51mATPfJ5RS036nbGNAu\\\\\\\/Ebf0w3JFEoY7Y2uDO1EBaFa9ocAXKYKrA+2+W\\\\\\\/sx8oQ4Dsf7kPbTEQxICeRqsOhbrKldM5ehcJd6g0zj0jo81KSBYkUPvHfm+XygDWjlhOVSnOq+M7qgiP1Ejiq3cj5XOHM0NiLWHdGAFyQgkGgbWhPfD5bmXMwdTyrzXUwPCPNMW8X12jNJ\\\\\\\/TXHfHSKmhIG60s7xb5w55u6M9BvjgdyeARw8UJcznO7tf5FhQNQYzB\\\\\\\/jGI9YWcutXFL5Sg2Jx7B0cqgimOkvn0Bj5f4fn5nSaO4hNg0jSteQMch2tkxHuM7MvbvBrQWvzYocqNXYnopsDoPCUU8VoRbacGIDgrim4KxpbWgCxdP26VfkKsx8yZeC3Bq3sDPpglIW\\\\\\\/sd03pb7YWICgj6C3mqjTCS5Ppo6Rlcx7zfsWgzZQ4cMxsuvbI6B4rtYVgIcogyleRG+Pw+5Io7PjjLYK1zu6NZ8eKKpl5nCIRR3iY2Hz+yQlLqNaxIeSO2KtJYlv+oRZowULg6Xui2T48T+WZyAFTckJ73BtAso4WJ6Xur0KwS79FW7aSj0uS6Ho+CLT34wZUtUp+jnQfigAg\\\\\\\/bXF1fr0M+S3ibvPAbFeFlzUr+Z5NQM1tJ58UtKaCdiB8wkiatPe9SWFkVY9xGIltRkSEMgE0F27IaTLLoc8jdlUmyl\\\\\\\/gdSg7mfcywUnkeyhOZzRu3QXtVdxHUeJZjW7pdX6bs46J7La2oBDvdma8VbieO+grd6RbFipUfZ3KBcdVnzfOTgk9LaFQOGtXkp\\\\\\\/L71Jj6D7HXOWCEIoHRDKjaFJ5kzXiaVU\\\\\\\/Vih\\\\\\\/BAgM9Wke7WOgVFoJQ5Q9cBf5BXhRbLBGfQL3p26jP3rjJrYruRHPKf4ukl1caLPFD1+OnGTf0ywzOCIp9pU8cZWQpjVZYP0b1lE3J1zomp22Vb0Wprp3d+5GJd5RPZxVybO7sKcxuQIkmxy39A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cHlurSnOww+DwJ9M\\\\\\\/P1rIQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227db20ae8d632c1781bc8958360e206ff183c7736d91ed9d7ac61adbd75224936\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490734 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2923" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GU48X8Ve9qlP-_lMNCLxY-b350XJ_00VYPXaCOzOxio" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2923" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RVctN_vzRS_wrZsvLVoLQ9vXUI42AXcJIcycmtSB_rA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2913" title="Disruption of INOS, a Gene Encoding myo-Inositol Phosphate Synthase, Causes Male Sterility in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2941" title="Acute Effects of Drugs on Caenorhabditis elegans Movement Reveal Complex Responses and Plasticity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490734" id="node490734" data-pisa="ggg;8/9/2923" data-pisa-master="ggg;g3.118.200373" data-apath="/ggg/8/9/2923.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8486-2801" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeremy R. B. Newman</span>, <span class="highwire-citation-author" data-delta="1">Patrick Concannon</span>, <span class="highwire-citation-author" data-delta="2">Manuel Tardaguila</span>, <span class="highwire-citation-author" data-delta="3">Ana Conesa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lauren M. McIntyre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2923-2940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200373 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490734" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeremy R. B. Newman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BR.%2BB.%2BNewman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Newman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeremy%2BR.%2BB.%2BNewman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8486-2801" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeremy R. B. Newman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick Concannon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Institute and Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BConcannon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Concannon%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BConcannon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Manuel Tardaguila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences, University of Florida, Gainesville, Florida, 32610; Wellcome Trust Sanger Institute, Hinxton, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manuel%2BTardaguila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tardaguila%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManuel%2BTardaguila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ana Conesa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences; Genetics Institute, University of Florida, Gainesville, Florida, 32610; Genomics of Gene Expression Lab, Prince Felipe Research Center, Valencia, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BConesa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conesa%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BConesa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aconesa@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">aconesa@ufl.edu</a>
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lauren M. McIntyre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aconesa@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">aconesa@ufl.edu</a>
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/2923" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490734" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2923.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490734" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2923.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490734" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2923.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490734" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/2923.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490734" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490734/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Alternative splicing leverages genomic content by allowing the synthesis of multiple transcripts and, by implication, protein isoforms, from a single gene. However, estimating the abundance of transcripts produced in a given tissue from short sequencing reads is difficult and can result in both the construction of transcripts that do not exist, and the failure to identify true transcripts. An alternative approach is to catalog the events that make up isoforms (splice junctions and exons). We present here the Event Analysis (EA) approach, where we project transcripts onto the genome and identify overlapping/unique regions and junctions. In addition, all possible logical junctions are assembled into a catalog. Transcripts are filtered before quantitation based on simple measures: the proportion of the events detected, and the coverage. We find that mapping to a junction catalog is more efficient at detecting novel junctions than mapping in a splice aware manner. We identify 99.8% of true transcripts while iReckon identifies 82% of the true transcripts and creates more transcripts not included in the simulation than were initially used in the simulation. Using PacBio Iso-seq data from a mouse neural progenitor cell model, EA detects 60% of the novel junctions that are combinations of existing exons while only 43% are detected by STAR. EA further detects ∼5,000 annotated junctions missed by STAR. Filtering transcripts based on the proportion of the transcript detected and the number of reads on average supporting that transcript captures 95% of the PacBio transcriptome. Filtering the reference transcriptome before quantitation, results in is a more stable estimate of isoform abundance, with improved correlation between replicates. This was particularly evident when EA is applied to an RNA-seq study of type 1 diabetes (T1D), where the coefficient of variation among subjects (n = 81) in the transcript abundance estimates was substantially reduced compared to the estimation using the full reference. EA focuses on individual transcriptional events. These events can be quantitate and analyzed directly or used to identify the probable set of expressed transcripts. Simple rules based on detected events and coverage used in filtering result in a dramatic improvement in isoform estimation without the use of ancillary data (<em>e.g.</em>, ChIP, long reads) that may not be available for many studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rna-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-rna-sequencing" rel="nofollow">RNA sequencing</a></li><li class="kwd"><a href="/keyword/gene-expression" class="hw-term hw-article-keyword hw-article-keyword-gene-expression" rel="nofollow">gene expression</a></li><li class="kwd"><a href="/keyword/transcriptomics" class="hw-term hw-article-keyword hw-article-keyword-transcriptomics" rel="nofollow">transcriptomics</a></li><li class="kwd"><a href="/keyword/alternative-splicing" class="hw-term hw-article-keyword hw-article-keyword-alternative-splicing" rel="nofollow">alternative splicing</a></li><li class="kwd"><a href="/keyword/isoform-estimation" class="hw-term hw-article-keyword hw-article-keyword-isoform-estimation" rel="nofollow">isoform estimation</a></li></ul><div class="section other1" id="sec-1"><h2 class="">Background</h2><p id="p-3">RNA-seq is a powerful tool for profiling gene expression, and has been employed to quantitate expression, and infer allele specific expression and alternative splicing (<em>e.g.</em>, (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mortazavi <em>et al.</em> 2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Main <em>et al.</em> 2009</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wang <em>et al.</em> 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Montgomery <em>et al.</em> 2010</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Nagalakshmi <em>et al.</em> 2010</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pastinen 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Graze <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dalton <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Korir and Seoighe 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Leon-Novelo <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akin <em>et al.</em> 2016</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fear <em>et al.</em> 2016</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goldstein <em>et al.</em> 2016</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kang <em>et al.</em> 2016</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Nellore <em>et al.</em> 2016</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Newell <em>et al.</em> 2016</a>)). The importance of alternative splicing has led to the development of numerous algorithms to estimate isoform abundance from RNA-seq data, including Cufflinks (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Trapnell <em>et al.</em> 2012</a>), RSEM (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li and Dewey 2011</a>), and eXpress (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Roberts <em>et al.</em> 2011</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Roberts and Pachter 2013</a>), and more recently iReckon (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mezlini <em>et al.</em> 2013</a>) and CIDANE (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Canzar <em>et al.</em> 2016</a>), and others (<em>e.g.</em>, (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jiang and Wong 2009</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Nicolae <em>et al.</em> 2011</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Turro <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Li and Jiang 2012</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sun 2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sturgill <em>et al.</em> 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Glaus <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Patro <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Nariai <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lee <em>et al.</em> 2015</a>)). The accurate identification of an individual transcript requires the presence of at least one exon or splicing event unique to that transcript (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cloonan <em>et al.</em> 2008</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 2016b</a>). However, there are transcript isoforms that contain no events unique to that isoform. Even when a unique event is detected in one isoform, reads mapping to non-unique portions of the transcript cannot be assigned with certainty. Recent evaluations conclude that while some algorithms, such as RSEM, perform better than others in simulations or particular example data, there are, unsurprisingly, errors in all current methods (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Angelini <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kanitz <em>et al.</em> 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ding <em>et al.</em> 2017</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Williams <em>et al.</em> 2017</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). Alternatives to isoform estimation include focusing on differential abundance of junctions (Zhang <em>et al.</em> 2012; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rezaeian <em>et al.</em> 2016</a>) and alternative exon inclusion (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Katz <em>et al.</em> 2010</a>). These event-based approaches have the benefit of not propagating the uncertainty of an isoform estimate in inferences about splicing. Tests of differential splicing are then exon based or exon/junction based (<em>e.g.</em>, (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders <em>et al.</em> 2012</a>)). However, there are drawbacks with these approaches, as currently implemented, that include an increased multiple testing burden, difficulties in making inferences about the impact of splicing for a particular gene, and challenges in identifying patterns in results.</p><p id="p-4">Here, we generalize the event- or feature-based analysis approach to assessing alternative splicing, resulting in a number of improvements in sensitivity and specificity, and in improved replicate-to-replicate concordance of transcript estimates. We take advantage of prior observations from long read PacBio data indicating that, while there are many novel isoforms detected (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sharon <em>et al.</em> 2013</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tombácz <em>et al.</em> 2016</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wang <em>et al.</em> 2016</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>), most are new combinations of known components (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Au <em>et al.</em> 2013</a>; <a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). Nellore <em>et al.</em> (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Nellore <em>et al.</em> 2016</a>) studied more than 20,000 human RNA-seq samples derived from multiple cell types and found that that only 3.5% of junctions are not derivable in from existing genome annotations in some form, and that 81.4% are from already annotated transcripts. We also note that in the literature, there are examples of identified novel transcripts of several genes that are comprised of new combinations of known splice sites, frequently those that are the result of a variant inducing an exon-skipping event (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Booms <em>et al.</em> 1999</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hide <em>et al.</em> 2001</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhou <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lim <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eswaran <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kim <em>et al.</em> 2013b</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gabreski <em>et al.</em> 2016</a>). This suggests that an augmented reference consisting of probable new events based on the set of existing transcripts can be developed. Here we present Event Analysis (EA), a strategy for pre-filtering sample-specific transcriptomes to improve transcript quantification. Basically, we expand the reference catalog by including all plausible events based on known exons and annotate all events. Then, for a particular RNA-seq sample, we quantitate events and use this information to calculate summary measures for transcripts which are used to filter the transcriptome of that sample. When the resulting filtered set of likely expressed transcripts is used as input for transcript quantification by RSEM (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Li and Dewey 2011</a>) or eXpress (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Roberts and Pachter 2013</a>) the correlation between replicates is improved compared to use of the unfiltered reference. Mapping to an expanded catalog identifies more true junctions compared to mapping using splice aware techniques (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dobin <em>et al.</em> 2013</a>) both in simulation and in data from mouse neural progenitor cells. Filtering based on events results in more accurate estimates of the transcriptome than a state of the art combination <em>de novo/</em>annotation method iReckon (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Mezlini <em>et al.</em> 2013</a>). In a large scale RNA-seq experiment in different cell types of ∼100 individuals with type 1 diabetes (T1D) we show that our approach substantially decreases the coefficient of variation for estimates of isoform abundance across samples.</p></div><div class="section methods" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Simulated data</h3><p id="p-5">Polyester (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Frazee <em>et al.</em> 2015</a>) was used to simulate reads for the mouse genome based on two scenarios. For the first simulation (Simulation 1), a set of highly expressed genes with few isoforms per gene was designed. This represents a best case scenario. Here, 10,000 RefSeq transcripts were randomly selected. The only restriction that was placed on transcript selection was that it could not come from a gene with an exonic sequence shared across more than one gene, thus avoiding regions of genic ambiguity. The results was a set of 10,000 transcripts representing 7,876 genes. Paired end reads were simulated for 6 replicates at 100 × coverage (approximately 53.4 million paired end reads per sample). Read size was set to 2×56 bp (matching the mouse neural data below). A second simulation (Simulation 2) was performed using all 467 annotated RefSeq transcripts from 59 genes, and represents a small-scale scenario where all transcripts of a gene are expressed (approximately 1.4 million paired end reads per sample). Genes were selected to represent a variety of difficult to resolve multi-transcript situations – the worst case scenario (see File S1). These simulated datasets are available at <a href="https://github.com/McIntyre-Lab/events">https://github.com/McIntyre-Lab/events</a>.</p></div><div id="sec-4" class="subsection"><h3>Mouse RNA-seq data</h3><p id="p-6">RNA-seq data from mouse neural progenitor cells (NPCs) and oligodendrocyte precursor cells (OPCs) used in this analysis were generated as described in Tardaguila <em>et al.</em> (<a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>) and are available from the NCBI Sequence Read Archive (study accession number SRP101446). Briefly, NPCs were isolated from the subventricular zone of killed neonatal c57/BL6 mice. NPCs were cultured as neurospheres in EGF/bFGF-supplemented media, and OPCs were derived by differentiating NPCs with All Trans Retinoic Acid. Total RNA was isolated using the Nucleospin RNA kit (Macherey-Nagel). Two biological replicates were performed for each cell type. RINs (RNA Integrity Numbers) were between 10 and 9.7 for all samples. Full-length cDNA was synthesized using the SMARTer PCR cDNA Synthesis kit (Clontech, version 040114) following PacBio recommendations. The reaction input was 1 µg of total RNA. Two first-strand cDNA synthesis reactions were performed per sample. First strand cDNA was then divided into nine PCR reactions ultimately yielding approximately 14–16 µg full-length cDNA per sample (<a id="xref-ref-72-5" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). The same cDNA preparation from each sample was used to prepare both Illumina and PacBio sequencing libraries. Illumina sequencing was performed at the Interdisciplinary Center for Biotechnology Research (University of Florida) with the Illumina Nextseq instrument using Nextera tagmentation, resulting in approximately 60 million 56 bp single end reads per sample (<a id="xref-ref-72-6" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>).</p></div><div id="sec-5" class="subsection"><h3>Mouse long read validation data</h3><p id="p-7">PacBio sequencing was performed as per the Iso-Seq protocol using P4-C2 chemistry at the Interdisciplinary Center for Biotechnology Research (University of Florida). The same cDNA used for the RNA-seq experiments was fractionated into three fractions (1-2kb, 2-3kb and 3-6kb) using BluePippin and sequenced on the RSII instrument using a total of eight SMRT cells per sample (two SMRT cells for the 1-2kb fraction, three SMRT cells for the 2-3kb and 3-6kb fractions), generating approximately 0.6M reads of insert (<a id="xref-ref-72-7" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>).</p></div><div id="sec-6" class="subsection"><h3>T1D RNA-seq data</h3><p id="p-8">RNA-seq data were obtained from three major classes of lymphocyte (CD4+ T cells, CD8+ T cells and CD19+ B cells) from 82 T1D cases from the Type 1 Diabetes Genetics Consortium, as previously described (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Newman <em>et al.</em> 2017</a>), and is available from NCBI database of Genotype of Phenotypes (dbGaP), accession number phs001426.v1.p1. Briefly, peripheral blood mononuclear cells were fractionated by positive selection on antibody-coated magnetic beads into the three lymphocyte populations (CD4<sup>+</sup> T cells, CD8<sup>+</sup> T cells, CD19<sup>+</sup> B cells). Purities of these populations (&gt;90%) were confirmed by flow cytometry. RNA was purified and libraries were prepared and sequenced (approximately 50 million 2×50bp reads per sample) in three pools using the Illumina HiSeq 2000 platform at the HudsonAlpha Genome Services Laboratory (Huntsville, Alabama). Individual cell samples with low coverage were excluded from all analyses (2 CD4+ samples, 2 CD8+ samples, 1 CD19+ sample). In total, there were 81 subjects with sequencing data from at least one cell type were included, of which 79 subjects had usable sequencing data from all three cell types (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Newman <em>et al.</em> 2017</a>).</p></div><div id="sec-7" class="subsection"><h3>Event Analysis</h3><div id="sec-8" class="subsection"><h4>Creation of the Junction catalog:</h4><p id="p-9">The first step of the EA approach creates a junction catalog starting with existing junctions, and augment this catalog by creating all possible logical junctions based on the set of known exons and map reads directly to junctions. Starting with a GFF3 file (<em>e.g.</em>, Ensembl, RefSeq, UCSC, AceView, or an individually curated custom reference) all junctions from logical combinations of exon pairs within the gene going from 5′-to-3′ were generated (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). By extending the 3′ sequence of a donor exon, or the 5′ region of an acceptor exon into the neighboring intron, putative novel exon donor/acceptors can be identified (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, teal reads; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, blue arrows above exons; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>; Table S1, Additional File 1). To ensure that only currently annotated intronic sequences are evaluated for putative novel donor/acceptor sites, the junction from the 5′-most donor exon and the junction from 3′-most acceptor exon were used as the starting points. The complete catalog contains all previously annotated junctions; unannotated junctions that are novel combinations of existing donor/acceptor sites; and potential novel new donor/acceptor sites. The detection of previously unannotated junctions provides evidence that a gene may be producing a novel isoform (<em>e.g.</em>, see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>, brown unfilled reads). We adopt the naming convention used by AStalavista (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Foissac and Sammeth 2007</a>) and others, where each junction is identified by the combination of chromosome (or scaffold, contig, etc.), the last position of the donor exon, first position of the acceptor exon, and strand (<em>i.e.</em>, [chromosome]:[donor position]:[acceptor position]:[strand]).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Example of alignment of reads to three transcripts of an example gene, and are displayed relative to their positions in the gene. Exons are labeled sequentially by their 5&#x2032; start and 3&#x2032; end positions relative to the orientation of their respective gene, and exons with the same 5&#x2032; start and 3&#x2032; end positions are considered to be the same exon. Reads mapping to exonic regions or exon-intron borders are indicated by single bars, and reads mapping to junctions are denoted by pairs of bars connected by a line. Reads can map to sequences unique to a single transcript (blue, yellow, red), multiple transcripts (gray), theoretical junctions not present in any transcript (brown with white fill) or exon-intron boundaries (teal). (B) Exons, exonic regions and exon fragments. This example shows exons that differ in donor and acceptor sites. The exonic region is defined as the genome co-ordinates where overlapping exons from different transcripts are found. Within the exonic region, sequences can be classified based upon the exons that are annotated in the region. These exon fragments may be shared across all exons (black), unique to one exon (blue, yellow, read) or common to a subset of exons (orange, green, purple). Within an exonic region, 3&#x2032; (donor) sites and 5&#x2032; (acceptor) sites are classified as either the reference site (blue hatching) or an alternative site (red hatching). For each exonic region, a single border junction is selected from sequences corresponding to unambiguous exon-intron borders. The exonic regions that comprise of only a single exon (i.e., exons D, E, H and I) are termed single-exons. By definition single-exons have no fragments, and can be unique (exon G), common (exon H) or constitutive (exon D and E) depending on their isoform membership. Examples of exons, exonic regions and exon fragments are demonstrated with two real genes from the mm10 RefSeq annotations: (C) Psma3 (Entrez ID: 19167), consisting of 11 exonic regions and three transcripts that share most of the same events; and (D) Rpl3 (Entrez ID: 27367), consisting of 10 exonic regions and two transcripts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Example of alignment of reads to three transcripts of an example gene, and are displayed relative to their positions in the gene. Exons are labeled sequentially by their 5&#x2032; start and 3&#x2032; end positions relative to the orientation of their respective gene, and exons with the same 5&#x2032; start and 3&#x2032; end positions are considered to be the same exon. Reads mapping to exonic regions or exon-intron borders are indicated by single bars, and reads mapping to junctions are denoted by pairs of bars connected by a line. Reads can map to sequences unique to a single transcript (blue, yellow, red), multiple transcripts (gray), theoretical junctions not present in any transcript (brown with white fill) or exon-intron boundaries (teal). (B) Exons, exonic regions and exon fragments. This example shows exons that differ in donor and acceptor sites. The exonic region is defined as the genome co-ordinates where overlapping exons from different transcripts are found. Within the exonic region, sequences can be classified based upon the exons that are annotated in the region. These exon fragments may be shared across all exons (black), unique to one exon (blue, yellow, read) or common to a subset of exons (orange, green, purple). Within an exonic region, 3&#x2032; (donor) sites and 5&#x2032; (acceptor) sites are classified as either the reference site (blue hatching) or an alternative site (red hatching). For each exonic region, a single border junction is selected from sequences corresponding to unambiguous exon-intron borders. The exonic regions that comprise of only a single exon (&lt;em&gt;i.e.&lt;/em&gt;, exons D, E, H and I) are termed single-exons. By definition single-exons have no fragments, and can be unique (exon G), common (exon H) or constitutive (exon D and E) depending on their isoform membership. Examples of exons, exonic regions and exon fragments are demonstrated with two real genes from the mm10 RefSeq annotations: (C) &lt;em&gt;Psma3&lt;/em&gt; (Entrez ID: 19167), consisting of 11 exonic regions and three transcripts that share most of the same events; and (D) &lt;em&gt;Rpl3&lt;/em&gt; (Entrez ID: 27367), consisting of 10 exonic regions and two transcripts.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F1.medium.gif" width="324" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/9/2923/F1.medium.gif" width="324" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490796" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">(A) Example of alignment of reads to three transcripts of an example gene, and are displayed relative to their positions in the gene. Exons are labeled sequentially by their 5′ start and 3′ end positions relative to the orientation of their respective gene, and exons with the same 5′ start and 3′ end positions are considered to be the same exon. Reads mapping to exonic regions or exon-intron borders are indicated by single bars, and reads mapping to junctions are denoted by pairs of bars connected by a line. Reads can map to sequences unique to a single transcript (blue, yellow, red), multiple transcripts (gray), theoretical junctions not present in any transcript (brown with white fill) or exon-intron boundaries (teal). (B) Exons, exonic regions and exon fragments. This example shows exons that differ in donor and acceptor sites. The exonic region is defined as the genome co-ordinates where overlapping exons from different transcripts are found. Within the exonic region, sequences can be classified based upon the exons that are annotated in the region. These exon fragments may be shared across all exons (black), unique to one exon (blue, yellow, read) or common to a subset of exons (orange, green, purple). Within an exonic region, 3′ (donor) sites and 5′ (acceptor) sites are classified as either the reference site (blue hatching) or an alternative site (red hatching). For each exonic region, a single border junction is selected from sequences corresponding to unambiguous exon-intron borders. The exonic regions that comprise of only a single exon (<em>i.e.</em>, exons D, E, H and I) are termed single-exons. By definition single-exons have no fragments, and can be unique (exon G), common (exon H) or constitutive (exon D and E) depending on their isoform membership. Examples of exons, exonic regions and exon fragments are demonstrated with two real genes from the mm10 RefSeq annotations: (C) <em>Psma3</em> (Entrez ID: 19167), consisting of 11 exonic regions and three transcripts that share most of the same events; and (D) <em>Rpl3</em> (Entrez ID: 27367), consisting of 10 exonic regions and two transcripts.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490903/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490903/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">All splice junctions derived from transcripts the gene model in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> example</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490829/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490829/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Examples of other possible, logical splice junctions derived from the gene model in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> example</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Exonic regions:</h4><p id="p-17">The second step of event analysis is to identify all the exonic regions. In higher eukaryotes, exons may overlap with one another (<em>e.g.</em>, due to alternative donors, alternative acceptors, alternative transcription initiation/termination sites, etc.). The exonic region may be quantified without regard to differences in donor/acceptor sites (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dalton <em>et al.</em> 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Graze <em>et al.</em> 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fear <em>et al.</em> 2016</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Newell <em>et al.</em> 2016</a>) to avoid double counting reads. In regions where differences are small (less than 10 bp) there is no meaningful loss of information in this approach. However, where exonic regions are comprised of overlapping exons that differ measurably, the 5′ and 3′ positions of exons within the exonic region are used to separate the region into exon fragments. EA annotates each exon fragment within an exonic region to indicate whether it is exclusive to a single exon or is shared among sets of exons (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>) and is further annotated to transcripts.</p></div><div id="sec-10" class="subsection"><h4>Annotating events:</h4><p id="p-18">EA annotates junctions based on whether they exclude exons (exon-skipping splice junctions) or if they use alternative 5′ and/or 3′ splice sites (alternative donor junctions, alternative acceptor junctions). Exon-skipping junctions are defined as splice junctions that exclude (“skip”) one or more exons that are situated in the reference annotation between the donor and acceptor site of the junction (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>, <em>e.g.</em>, junction between exons E and I; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Where there are multiple possible donor and/or acceptor sites within a group of overlapping exons, all donors/acceptors are classified as “alternative”. This follows a similar convention to other splicing definitions, such as those used by AStalavista (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Foissac and Sammeth 2007</a>). Junction definitions are not mutually exclusive: a junction can utilize both alternative donor and alternative acceptor sites and also exclude one or more intermediary exons (see examples in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The unannotated, putative novel donor/novel acceptor sites may reflect unprocessed transcript and intron retention events. To accommodate this uncertainty EA annotates these as border junctions. The likelihood of truly novel donor/acceptor sites is evaluated in a separate step based on the experimental data that includes mapping information from the adjoining exons/intron region. (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Possible classifications for border junctions. Detected border junctions are only classified if their adjacent exon is also detected. These border junctions can be categorized as a possible novel donor/acceptor, and possible intron retention. If the coverage at the border junction was similar in coverage to its adjacent exon (defined as the border junction having a mean APN of at least 90% of the APN of the exon), then the border junction is classified as a possible novel donor. If the coverage (as APN) of the border junction was at least 10% of the adjacent exon and the adjacent intron was also detected, then the border junction is classified as possible intron retention. These values are conservative and can be changed. Where a border junction could be classified as both a possible novel donor and possible intron retention, the event is classified as ambiguous intron retention. All remaining border junctions are classified as possible unprocessed transcript." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Possible classifications for border junctions. Detected border junctions are only classified if their adjacent exon is also detected. These border junctions can be categorized as a possible novel donor/acceptor, and possible intron retention. If the coverage at the border junction was similar in coverage to its adjacent exon (defined as the border junction having a mean APN of at least 90% of the APN of the exon), then the border junction is classified as a possible novel donor. If the coverage (as APN) of the border junction was at least 10% of the adjacent exon and the adjacent intron was also detected, then the border junction is classified as possible intron retention. These values are conservative and can be changed. Where a border junction could be classified as both a possible novel donor and possible intron retention, the event is classified as ambiguous intron retention. All remaining border junctions are classified as possible unprocessed transcript.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F2.medium.gif" width="292" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/9/2923/F2.medium.gif" width="292" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Possible classifications for border junctions. Detected border junctions are only classified if their adjacent exon is also detected. These border junctions can be categorized as a possible novel donor/acceptor, and possible intron retention. If the coverage at the border junction was similar in coverage to its adjacent exon (defined as the border junction having a mean APN of at least 90% of the APN of the exon), then the border junction is classified as a possible novel donor. If the coverage (as APN) of the border junction was at least 10% of the adjacent exon and the adjacent intron was also detected, then the border junction is classified as possible intron retention. These values are conservative and can be changed. Where a border junction could be classified as both a possible novel donor and possible intron retention, the event is classified as ambiguous intron retention. All remaining border junctions are classified as possible unprocessed transcript.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Events – junctions, exonic regions, and exon fragments – that are only annotated in a single isoform in the reference database are classified as “unique”. Events that are common to a multiple but not all isoforms are classified as “common”, while events that are annotated in all isoforms are classified as “constitutive”. Events that are not annotated to any isoform are classified as “unannotated”. Non-unique sequences can also result from the same event being shared in a set of transcripts as well as genes within the same gene family that are highly homologous. Uniqueness is expected to increase with longer reads. When events are not unique among different genes, it is classified as “multi-gene”. The complexity of resolving multi-gene events is well known (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Djebali <em>et al.</em> 2012</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Treangen and Salzberg 2012</a>) and multi-gene events are not analyzed further. As more experiments are strand specific, the incidence of multi-gene junctions and multi-gene exonic regions will be reduced.</p></div><div id="sec-11" class="subsection"><h4>Mapping to the junction catalog:</h4><p id="p-21">To apply EA to data, reads are mapped to the junction catalog (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For the RNA-seq data used here, duplicate reads are removed and no other processing/trimming is performed. Distinct (non-duplicate) RNA sequence reads are aligned as single-ended reads to the set of cataloged junction sequences using the Bowtie1 algorithm (version 0.12.9) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Langmead <em>et al.</em> 2009</a>), allowing for only a single alignment per read (parameter “-m 1”) and for up to three mismatched nucleotides (“-v 3”).The “–tryhard” parameter, and the “best” alignment in terms of stratum were reported using the options “–best–strata–chunkmbs 1024”.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Event Analysis workflow. Event Analysis consists of three main components: (1) annotation generation; (2) alignment and coverage; (3) event detection and transcript identification. First, annotations for exon-level annotations (exonic regions, exon fragments, introns) and junction annotations are generated. A GFF3 file as input to generate annotations for exonic regions, exon fragments, distinct introns and a catalog of all possible, logical junctions within a gene, and a genome FASTA file is used to extract junction sequences to use for quantification. Second, RNA-seq reads are first aligned to junction sequences and coverage is calculated. Reads that do not align to junctions are then aligned to the genome to calculate the coverage of exonic regions, exon fragments and introns. Third, the output set of counts from alignments are then analyzed to determine what transcriptional events (exonic regions, exon fragment, introns, and junctions) are detected and what transcripts likely present/absent in a given condition." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The Event Analysis workflow. Event Analysis consists of three main components: (1) annotation generation; (2) alignment and coverage; (3) event detection and transcript identification. First, annotations for exon-level annotations (exonic regions, exon fragments, introns) and junction annotations are generated. A GFF3 file as input to generate annotations for exonic regions, exon fragments, distinct introns and a catalog of all possible, logical junctions within a gene, and a genome FASTA file is used to extract junction sequences to use for quantification. Second, RNA-seq reads are first aligned to junction sequences and coverage is calculated. Reads that do not align to junctions are then aligned to the genome to calculate the coverage of exonic regions, exon fragments and introns. Third, the output set of counts from alignments are then analyzed to determine what transcriptional events (exonic regions, exon fragment, introns, and junctions) are detected and what transcripts likely present/absent in a given condition.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F3.medium.gif" width="257" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/9/2923/F3.medium.gif" width="257" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">The Event Analysis workflow. Event Analysis consists of three main components: (1) annotation generation; (2) alignment and coverage; (3) event detection and transcript identification. First, annotations for exon-level annotations (exonic regions, exon fragments, introns) and junction annotations are generated. A GFF3 file as input to generate annotations for exonic regions, exon fragments, distinct introns and a catalog of all possible, logical junctions within a gene, and a genome FASTA file is used to extract junction sequences to use for quantification. Second, RNA-seq reads are first aligned to junction sequences and coverage is calculated. Reads that do not align to junctions are then aligned to the genome to calculate the coverage of exonic regions, exon fragments and introns. Third, the output set of counts from alignments are then analyzed to determine what transcriptional events (exonic regions, exon fragment, introns, and junctions) are detected and what transcripts likely present/absent in a given condition.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Mapping to the genome:</h4><p id="p-23">The next step of EA is to map reads not mapped to the junction catalog to the genome. Reads unmapped to junctions were mapped to the genome (GRCm38/mm10 version for the mouse data, GRCh37/hg19 version (release 73) for the T1D data) using BWA-MEM (version 0.7.12) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li 2013</a>). Reads mapping to exon fragments were identified using the BED file for the genomic regions defining these events. Coverage is summarized as the average number of reads per nucleotide (APN = number of reads aligning to region / region length). For the simulated data, detection was defined as an APN &gt; 0 in at least three of the six samples. For the mouse neural data, detection was defined as an APN &gt; 0 for both replicates. For the T1D data, detection was defined as an APN &gt; 5 in at least 50% of subjects per cell type (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Newman <em>et al.</em> 2017</a>).</p></div><div id="sec-13" class="subsection"><h4>Filtering transcripts:</h4><p id="p-24">The EA process continues with a summarization of the events as they map to transcripts. Each transcript consists of a series of events (junctions and fragments) (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>). To identify transcripts that may be expressed and filter those that are likely not expressed, EA uses simple measures that capture the elements determining the likelihood a transcript is expressed: coverage, the proportion of events detected, and the proportion of unique events detected. EA outputs a tab-delimited file containing these simple summary statistics. Users can use this to decide what combination of detection, coverage and uniqueness is to be used to filter transcripts. For example, transcripts with no reads mapping to them are clearly not expressed and can be filtered.</p><p id="p-25">For each transcript, coverage is calculated by combining all of the events for that transcript. Since the events are of varying lengths, EA reports the average number of reads per nucleotide (reads in region/ length of the region: APN). APN can be interpreted directly: an APN of 2 means that an average of 2 reads cover that region. The proportion of events detected is calculated by counting the number of these events with coverage (detected; APN &gt; 0) and dividing this by the total number of events in a transcript. Similarly, for transcripts with unique events the proportion of unique events detected is calculated. The inclusion of non-expressed transcripts in the reference can impact the expression estimates of transcripts that are expressed (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 2016b</a>; <a id="xref-ref-72-8" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). Using the measures proposed, it is possible to filter transcripts. For example, a transcript with no unique events but only 10% of its events detected is a candidate for filtering. The measures, coverage, proportion of events detected and proportion of unique events detected are all given in the output of the code provided and so the users can set their own filtering criteria. For the purposes of this manuscript, we focus on several specific references: the complete RefSeq transcriptome (<em>i.e.</em>, no filtering), a reduced reference of transcripts with all their associated events detected at APN &gt; 0, and a reduced reference of transcripts with at least 75% of their associated events detected at APN ≥ 5.</p></div></div><div id="sec-14" class="subsection"><h3>Splice aware mapping: STAR</h3><p id="p-26">Reads were aligned to the mm10 genome using STAR (version 2.5.4b) (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Dobin <em>et al.</em> 2013</a>), using the alignment mode “EndToEnd” (no soft-clipping), minimum allowable read junction overhang was set to 16 bp, a maximum allowable junction overhang of 40 bp, up to three mismatches and no multimapping alignments were allowed. Junctions were converted to genome in the format compatible with AStalavista naming (chromosome, last position of donor exon, first position of acceptor exon, strand) (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Foissac and Sammeth 2007</a>).</p></div><div id="sec-15" class="subsection"><h3>iReckon</h3><p id="p-27">Genome alignments and junctions identified using STAR (see above) were used as input. Annotations consisting of a 12-column BED file of transcripts (including genomic start positions and lengths of each exon) was used to guide transcript assembly. Alignments to transcript sequences were carried out using BWA-MEM (version 0.7.15) (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Li 2013</a>) as per the iReckon user guide, and all default parameters were selected.</p></div><div id="sec-16" class="subsection"><h3>Estimating transcript abundance using RSEM and eXpress</h3><p id="p-28">We estimated transcript abundance using RSEM (version 1.2.28) (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Li and Dewey 2011</a>) and eXpress (version 1.5.1) (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Roberts and Pachter 2013</a>). For RSEM, references were prepared using the tool ‘rsem-prepare-reference’, the set of transcript sequence for a given transcriptome (complete RefSeq, PacBio, reduced references, etc.) and a tab-delimited gene-to-transcript index file as input, as described in the RSEM user guide. The default aligner (Bowtie, version 0.12.9) was used for mapping. For the mouse neural data, reads were aligned single-end, with a mean fragment length of 80 bp and standard deviation of 50 bp. 95% confidence intervals and posterior mean estimates of transcript abundance were also requested. The read start position distribution was estimated from the data. Default settings for RSEM for all other parameters were used. For the T1D data, reads were aligned as paired-end and all other parameters were the same as the mouse neural data.</p><p id="p-29">For eXpress, reads were first aligned to transcript sequences using BWA-MEM version 0.7.15 (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Li 2013</a>) with default parameters. Output SAM files were then analyzed using eXpress to estimate abundances of transcripts in each reference transcriptome. Mean fragment length of 80bp and with a standard deviation of 50 bp was set for eXpress, with all other parameters left at their default settings. We estimate transcript abundances for the complete RefSeq transcriptome and two the EA reduced references and the iReckon transcriptome. All iReckon transcripts assembled in each NPC replicate were combined and transcripts present in both samples (based on identifiers and genomic coordinates) were combined into a single transcript (iReckon “union” transcriptome).</p></div><div id="sec-17" class="subsection"><h3>Evaluation of the RNA-seq data</h3><p id="p-30">To estimate concordance of transcript estimates between replicates, we first calculated the simple agreement between replicates as the proportion of transcripts detected in both replicates. Transcripts were then binned based on their TPM estimates (transcripts per kilobase million) estimates: “no expression” (TPM = 0 in both replicates), “very low expression” (minimum log-TPM: 0 - 0.5), “low expression” (minimum log-TPM: 0.5 - 2), “moderate expression” (minimum log-TPM: 2 – 4) and “high expression” (minimum log-TPM: 4 or greater). This approximately corresponds to no expression, 25<sup>th</sup> percentile of expression (“very low expression”), 25<sup>th</sup> to 75<sup>th</sup> percentile of expression (“low expression”), 75<sup>th</sup> to 90<sup>th</sup> percentile of expression (“moderate expression”), and 90<sup>th</sup> to 100<sup>th</sup> percentile of expression (“high expression”). The coefficient of variation (CV) for each TPM bin was then calculated. Replicates are further compared by generating Bland-Altman plots (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bland and Altman 1986</a>).</p></div><div id="sec-18" class="subsection"><h3>Validation of Event Analysis, STAR and iReckon via simulation</h3><p id="p-31">The two simulated datasets generated for this study (Simulations 1 and Simulation 2) were used to evaluate and compare the performances of EA and STAR in terms of junction identification, and EA and iReckon in terms of estimated transcriptomes. To compare EA and STAR, junctions were identified and quantified. As STAR only reports the number of reads mapping to a particular junction, the APN for each junction sequence in each sample was estimated by dividing the number of mapped reads reported to the expected junction size (<em>i.e.</em>, twice maximum allowable junction overhang). EA cataloged junctions and STAR junctions were matched on genomic coordinates (chromosome, last position of donor exon, first position of acceptor exon, strand). The number of junctions detected (APN &gt; 0) and at different levels of support (APN ≥ 2, 5, 10) by each method, by both methods (intersection of EA and STAR), and by either method (union of EA and STAR) was compared with the number of expected junctions (<em>e.g.</em>, the number of annotated junctions from transcripts in the simulation).</p><p id="p-32">The reference reduction using EA was compared with the transcript reassembly of iReckon. For EA, three reduced reference transcriptomes were created: (i) 100% events detected at APN &gt; 0; (ii) at least 75% of events detected at APN &gt; 0; and, (iii) at least 75% of events detected at APN ≥ 5). From the iReckon results, two transcriptomes were used: one consisting of transcripts observed all simulated samples, and one consisting of transcripts observed in at least one simulated sample. Transcripts from these simulations were classified into the following categories: (1) transcripts that were used for read simulation and were correctly identified by EA/iReckon; (2) “related RefSeq transcripts”, which are transcripts not selected for simulating reads but are from the same gene as a transcript used for read simulation; (3) “related non-RefSeq transcripts”, which are transcripts assembled by iReckon but not in the RefSeq annotations, and are from the same gene as a transcript used for read simulation; (4) “unrelated transcripts”, which are those transcripts that are not from genes used to simulate reads; and (5) “missing transcripts”, which are those that were used for read simulation but were not identified by EA/iReckon.</p></div><div id="sec-19" class="subsection"><h3>Validation of Event Analysis With PacBio sequenced transcriptome</h3><p id="p-33">The set of transcripts identified using PacBio sequencing (<a id="xref-ref-72-9" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>) was used for validation. Sequences for each detected exon fragment and junction were compared to the PacBio transcript sequences. Transcripts were compared to PacBio using MegaBLAST (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Camacho <em>et al.</em> 2009</a>) and events/transcripts were considered validated if the alignment covers at least 90% and had no gaps or mismatches.</p></div><div id="sec-20" class="subsection"><h3>Validation of iReckon-assembled transcripts With PacBio sequenced transcriptomes</h3><p id="p-34">To determine how well iReckon reassembles transcripts, the set of transcripts identified using PacBio sequencing (<a id="xref-ref-72-10" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>) was used for validation. Sequences for each iReckon transcript for each NPC replicate were extracted using BEDtools (version 2.17.0; (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Quinlan and Hall 2010</a>)) from the output GTF files and compared to the complete set of 16,104 PacBio transcript sequences using the MegaBLAST algorithm (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Camacho <em>et al.</em> 2009</a>). BLAST alignments were classified based on the length of the BLAST hit (100% of the iReckon transcript sequence matches a PacBio transcript with more than 10bp different in length between iReckon and PacBio transcripts, at least 90%, 75% or 50% of the iReckon transcript sequence matches a PacBio transcript), and whether there were no mismatches or no more than 5 mismatches nucleotides. iReckon transcripts with gaps or multiple fragmented hits to the same PacBio transcript were excluded from analysis.</p><p id="p-35">For identifying potential RefSeq matches to the iReckon-assembled transcripts, a similar BLAST alignment of iReckon transcripts to the complete RefSeq transcriptome was performed using the MegaBLAST algorithm. We only considered BLAST hits with at least 90% of the sequences matching, allowing for up to five mismatched nucleotides.</p></div><div id="sec-21" class="subsection"><h3>Software availability</h3><p id="p-36">The bioinformatics workflow can be found in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. All code is in python and scripts and documentation can be found at <a href="https://github.com/McIntyre-Lab/events">https://github.com/McIntyre-Lab/events</a>.</p></div><div id="sec-22" class="subsection"><h3>Data availability</h3><p id="p-37">The mouse neural dataset analyzed in this study is available from the NCBI Sequence Read Archive under accession number SRP101446. The T1D dataset used in this study is available from the dbGaP repository under accession number phs001426.v1.p1. The simulated datasets used to evaluate the performance of EA, iReckon and STAR as well as all code pertaining to the analyses presented in this manuscript can be found at <a href="https://github.com/McIntyre-Lab/events">https://github.com/McIntyre-Lab/events</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6205793">https://doi.org/10.25387/g3.6205793</a>.</p></div></div><div class="section results" id="sec-23"><h2 class="">Results</h2><div id="sec-24" class="subsection"><h3>Simulation results</h3><div id="sec-25" class="subsection"><h4>Junctions:</h4><p id="p-38">EA uses mapping to a catalog of junctions for the detection of quantification of junction sequences. An alternative to EA is to map to the genome in a splice aware fashion. We compared the junction detection rate of EA with that of the splice-aware aligner STAR (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Dobin <em>et al.</em> 2013</a>), using alignment parameters were comparable to those used for mapping to the junction catalog to enable direct comparisons.</p><p id="p-39">In Simulation 1, we found that EA and STAR identify almost the same set of junctions, with EA detecting a few more true junctions than STAR (File S1, Table S1.3). STAR detected ∼100 true junctions that EA did not, and these are almost all complex multi-junction alignments involving microexons (File S1, Table S1.3). Without annotation of microexons (defined as exons of 51 nucleotides or fewer in length) in the reference these events will be missed by any reference-based approach, including EA. However, papers carefully looking at microexons point to the difficulty in capturing these events (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Florea <em>et al.</em> 1998</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wu <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Irimia <em>et al.</em> 2014</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Ustianenko <em>et al.</em> 2017</a>).</p><p id="p-40">In Simulation 2, we found that the junctions EA and STAR identify are also largely concordant (File S1, Table S1.5). EA detects 17 annotated junctions that STAR did not. Of the nine unannotated junctions included in this simulation, EA identified all of them, while STAR missed three (File S1, Table S1.6).</p></div><div id="sec-26" class="subsection"><h4>Transcripts:</h4><p id="p-41">For Simulation 1, using the definitions APN &gt; 0 and 100% of the events detected, EA detects 93% of all transcripts simulated and an additional 25,368 transcripts cannot be eliminated from consideration due to similarity with simulated transcripts (File S1, Table S1.4). At APN ≥ 5 combined with the requirement that 75% of events be detected; 90% of the true transcripts are detected 24,201 additional transcripts cannot be eliminated from consideration. iReckon detects fewer (80%) of all simulated transcripts in at least one sample but only 66% in all samples and generates an almost equal number of unrelated transcripts. (File S1, Table S1.4).</p><p id="p-42">For Simulation 2, EA correctly retained more transcripts than iReckon correctly reassembles (File S1, Table S1.7). Using an event detection criteria of APN &gt; 0 and requiring 100% of events detected, EA identified 90% of the 467 simulated RefSeq transcripts, as well one additional, unrelated transcript. When the event detection criteria was set to APN ≥ 5 and transcripts with at least 75% of events detected were retained, EA identified all but one of the 467 RefSeq transcripts and no unrelated transcripts (File S1, Table S1.7). iReckon correctly assembled 391 of the 467 simulated transcripts, and assembled an additional 565 unrelated transcripts (File S1, Table S1.7). When only transcripts in common to all samples were included, iReckon only retained 180 of the 467 transcripts (File S1, Table S1.7).</p></div></div><div id="sec-27" class="subsection"><h3>Detection of expression in mouse neural progenitors</h3><p id="p-43">In cultured mouse NPCs, a total of 105,891 single-exons and 39,739 exon fragments of length at least 10 bp, and 125,716 junctions were detected in 20,875 genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">4B</a>). This included 89,812 exon-exon junctions that are annotated to at least one transcript (71% of detected junctions), 8,412 junctions not annotated to any known mouse RefSeq transcript (7% of detected junctions), and 27,492 border junctions (22% of detected junctions; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">4C</a>). The majority of transcript events are constitutively included in all known transcripts for a gene or set of genes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Events detected in cultured mouse NPCs. (A) Exons and exon fragments detected in mouse NPCs. (B) Splicing events by event type detected in mouse NPCs. (C) Detected exon-exon junctions by event annotation. Blue bars indicate exon-exon junctions annotated to known transcripts, gray bars indicate exon-exon junctions that are not annotated to known transcripts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Events detected in cultured mouse NPCs. (A) Exons and exon fragments detected in mouse NPCs. (B) Splicing events by event type detected in mouse NPCs. (C) Detected exon-exon junctions by event annotation. Blue bars indicate exon-exon junctions annotated to known transcripts, gray bars indicate exon-exon junctions that are not annotated to known transcripts.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F4.medium.gif" width="409" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/9/2923/F4.medium.gif" width="409" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490776" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Events detected in cultured mouse NPCs. (A) Exons and exon fragments detected in mouse NPCs. (B) Splicing events by event type detected in mouse NPCs. (C) Detected exon-exon junctions by event annotation. Blue bars indicate exon-exon junctions annotated to known transcripts, gray bars indicate exon-exon junctions that are not annotated to known transcripts.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcript-specificity of events. Transcriptional events (single-exon exons, exon fragments, junctions) are divided on the basis of transcript-specificity (unique, common, constitutive) Exons of mono-transcript genes are a special case of single-exon exons as these are both unique and constitutive. These are indicated as a subset within unique single-exon exons, and are included within the total set of unique events. The majority of detected events correspond to those constitutively included in all known transcripts for a gene or set of genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Transcript-specificity of events. Transcriptional events (single-exon exons, exon fragments, junctions) are divided on the basis of transcript-specificity (unique, common, constitutive) Exons of mono-transcript genes are a special case of single-exon exons as these are both unique and constitutive. These are indicated as a subset within unique single-exon exons, and are included within the total set of unique events. The majority of detected events correspond to those constitutively included in all known transcripts for a gene or set of genes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F5.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/9/2923/F5.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490742" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Transcript-specificity of events. Transcriptional events (single-exon exons, exon fragments, junctions) are divided on the basis of transcript-specificity (unique, common, constitutive) Exons of mono-transcript genes are a special case of single-exon exons as these are both unique and constitutive. These are indicated as a subset within unique single-exon exons, and are included within the total set of unique events. The majority of detected events correspond to those constitutively included in all known transcripts for a gene or set of genes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Validation of novel events</h3><p id="p-46">The set of detected unannotated splicing events (unannotated junctions and putative IR events) identified in EA are potentially from novel transcripts. We identified 583 putative novel junction events with both flanking exons detected from 520 genes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>); most of these genes only have single unannotated junction (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). We compared this set of putatively novel junctions against PacBio transcripts to determine the proportion of putative novel events with supporting evidence in the PacBio transcriptome. We found that of 69% were captured in the PacBio transcriptome. Similar to annotated events, novel events captured in PacBio transcripts have a slightly higher mean APN values compared to putatively novel events not present in the PacBio transcriptome (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>; Figure S1, Additional File 1). The 31% of the unannotated junctions not found in the PacBio data may be from transcripts with lower levels of expression, or they may be false positives.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unannotated splicing events detected in mouse NPCs. (A) Genes with detected unannotated events. (B) Distribution of the number of detected unannotated splicing events per gene. Density plot of the log-APN distribution between detected unannotated junctions (C) and border junctions (D), with and without BLAST hits to the PacBio transcriptome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Unannotated splicing events detected in mouse NPCs. (A) Genes with detected unannotated events. (B) Distribution of the number of detected unannotated splicing events per gene. Density plot of the log-APN distribution between detected unannotated junctions (C) and border junctions (D), with and without BLAST hits to the PacBio transcriptome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F6.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/9/2923/F6.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490789" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-47" class="first-child">Unannotated splicing events detected in mouse NPCs. (A) Genes with detected unannotated events. (B) Distribution of the number of detected unannotated splicing events per gene. Density plot of the log-APN distribution between detected unannotated junctions (C) and border junctions (D), with and without BLAST hits to the PacBio transcriptome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Border junctions may represent retained introns, novel donors, novel acceptors or incomplete messenger RNA processing. From the set of 154,934 introns, their associated border events and 5′ donor exonic regions, only those with the junction and exonic region detected (APN ≥ 5) were analyzed further (505 border junctions from 391 genes). This set of 505 were classified into their likely sources as described in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, and the results for confirmation with PacBio are on average 32% (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490772/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490772/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Border junctions and the overlap with PacBio</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>Using Event Analysis as a prior for estimation of transcript quantity</h3><p id="p-49">One of the applications of EA is to filter transcripts not likely expressed (<em>e.g.</em>, transcripts with few or none of their associated events detected), and thus define a reduced reference. This reduced transcriptome can then be used in transcript quantification methods. The expectation is that reducing the complexity of a reference transcriptome yields better estimates of transcript abundance (<a id="xref-ref-72-11" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). The agreement between replicates was calculated for the RSEM quantification based on the complete RefSeq transcriptome (∼160,000 transcripts in mouse) as a baseline and overall 17% of transcripts disagreed between replicates in terms of detection (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The CV of lower expressed transcripts was high, particularly for very-lowly expressed transcripts (average CV = 96.290), whereas highly expressed transcripts had a much lower CV (average CV = 19.234; <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> and Figure S2, Additional File 1). Concordance at low levels of expression is known to be problematic (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">McIntyre <em>et al.</em> 2011</a>; <a id="xref-ref-72-12" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>); low expression events (defined reads per kilobase million (RPKM) &lt; 5) typically have a much higher variance (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">McIntyre <em>et al.</em> 2011</a>), and this is observed with the both the variances of very-low and low expression transcripts (RPKM of 5 is approximately equivalent to a log-TPM of 2). There were 20,875 genes with at least one exonic region detected corresponding to 75,926 annotated transcripts. 2,391 transcripts from the set of expressed genes had none of their exons or junctions detected and were removed for a total of 73,535 possible expressed transcripts (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The agreement among replicates based on this transcriptome was similar to that of the complete RefSeq transcriptome (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>), and the average CV only marginally improved for the lower expressed transcripts (Supplementary Figures 2 and 3, Additional File 1), indicating that RSEM already effectively removes clearly unexpressed transcripts and there is no incremental benefit to only removing these transcripts.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490846/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490846/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Agreement statistics of detection of transcripts per transcriptome</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the total events detected per transcript. (A) Density plot of the distribution of proportion of total events detected. (B) Proportion of transcripts by the proportion of total events detected. (C) Distribution of number of transcripts detected per gene. (D) The proportions of transcripts with at least one unique event detected, transcripts with unique events but none detected, and transcripts with no assigned unique events. (E) Distribution of the expressed transcripts with at least one unique event detected by the proportion of unique events detected. (F) Scatterplot of number of unique events and proportion of unique events detected per transcript. Most of the transcripts with detected unique events typically had fewer than 3 unique events detected, and about 61% of these 22,319 transcripts had only a single unique event detected." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of the total events detected per transcript. (A) Density plot of the distribution of proportion of total events detected. (B) Proportion of transcripts by the proportion of total events detected. (C) Distribution of number of transcripts detected per gene. (D) The proportions of transcripts with at least one unique event detected, transcripts with unique events but none detected, and transcripts with no assigned unique events. (E) Distribution of the expressed transcripts with at least one unique event detected by the proportion of unique events detected. (F) Scatterplot of number of unique events and proportion of unique events detected per transcript. Most of the transcripts with detected unique events typically had fewer than 3 unique events detected, and about 61% of these 22,319 transcripts had only a single unique event detected.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F7.medium.gif" width="440" height="373"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/9/2923/F7.medium.gif" width="440" height="373"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490888" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-50" class="first-child">Distribution of the total events detected per transcript. (A) Density plot of the distribution of proportion of total events detected. (B) Proportion of transcripts by the proportion of total events detected. (C) Distribution of number of transcripts detected per gene. (D) The proportions of transcripts with at least one unique event detected, transcripts with unique events but none detected, and transcripts with no assigned unique events. (E) Distribution of the expressed transcripts with at least one unique event detected by the proportion of unique events detected. (F) Scatterplot of number of unique events and proportion of unique events detected per transcript. Most of the transcripts with detected unique events typically had fewer than 3 unique events detected, and about 61% of these 22,319 transcripts had only a single unique event detected.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">The proportion of total events, and the proportion of unique events that were detected was calculated for each RefSeq transcript. The transcripts fell into two distinct modes, those with at least half of their associated events detected and those with a much lower proportion detected, with approximately 20% of the potential transcripts with less than 25% of their events detected (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a> and <a id="xref-fig-7-3" class="xref-fig" href="#F7">7B</a>). The number of transcripts per gene was lower when requiring all events to be detected (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>). There were 34,900 expressed transcripts without any assigned unique events (<em>i.e.</em>, non-resolvable transcripts (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7D</a> and <a id="xref-fig-7-6" class="xref-fig" href="#F7">7E</a>). Because these transcripts are not resolvable, their detection is by definition ambiguous. Among the transcripts with detected unique events, there was little evidence that the number of unique events assigned to a transcript affected the proportion that were detected per transcript, but that the proportion of unique events detected was a function of the amount of total events detected (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7F</a>).</p><p id="p-52">We reasoned that transcripts consistently detected across their length were more likely to be present than those inconsistently detected. We then examined the agreement between replicates in transcript detection and variation in transcript estimates after eliminating transcripts with different proportions of their events detected and at different levels of average coverage. There was a distinct improvement in replicate agreement and estimate precision when limiting the set of possible transcripts to those with more consistent within transcript detection (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>; Supplementary Figures 2 and 3, Additional File 1). Restricting the list of possible transcripts before quantification to those for which there were at least 5 reads on average for 75% or more of their events resulted in an average CV of 36.051 for lowly expressed transcripts, and an agreement of 91% between replicates for detection of transcripts. Restricting the list of possible transcripts before quantification to those for which there were at least 5 reads on average with 100% of their events detected resulted in an average CV of 28.249 for lowly expressed transcripts, and an agreement of almost 99% between replicates for detection of transcripts.</p></div><div id="sec-30" class="subsection"><h3>Validation of Event Analysis transcriptomes With the PacBio transcriptome</h3><p id="p-53">From the set of 20,785 genes not filtered from the RNA-seq data, we examined the how many of the detected annotated events – 39,739 exon fragments, 105,891 single-exons and 89,632 annotated junctions – were present in the set of PacBio transcripts. PacBio transcripts were assigned reference transcript identifiers (RefSeq or Ensembl) if: (1) the PacBio transcript matched at all splice junctions of a reference transcript (“full splice match” or FSM (<a id="xref-ref-72-13" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>)), or (2) splice junctions of the PacBio transcript matched consecutive but not all splice junctions of a reference transcript (“incomplete splice match”; ISM (<a id="xref-ref-72-14" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>)). Monoexonic PacBio transcripts were considered a “full splice match” if they corresponded to a monoexonic reference transcript, or “incomplete splice match” if they were corresponded to multiexonic reference. Of the total 235,262 events detected, 66,646 events from 4,426 genes (3,796 fragments, 19,388 single-exons and 43,462 junctions) matched at least one PacBio transcript (28% of events, 21% genes). We hypothesize that the remaining RNA-seq mapped reads to annotated events are representative of transcripts that are not captured by PacBio due to lack of sequence depth, and thus will have lower short-read coverage than annotated events that mapped to PacBio transcripts. To test this we compared the APN in events captured by PacBio to events not captured by PacBio. (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Event Analysis with PacBio sequenced transcripts. (E) Distribution of log-APN of annotated events with and without PacBio hits. Blue line represents the of log-APN distribution of annotated events with PacBio hits, red line represents the of log-APN distribution of annotated events with no PacBio hits. Overlap between the Event Analysis trancriptome (yellow) against the PacBio transcriptome (blue = complete splice match; red = incomplete splice match), for EA transcripts with 100% of events detected (B), at least 75% of events detected (C), at least 50% of events detected (D), and at least 25% of events detected (E)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of Event Analysis with PacBio sequenced transcripts. (E) Distribution of log-APN of annotated events with and without PacBio hits. Blue line represents the of log-APN distribution of annotated events with PacBio hits, red line represents the of log-APN distribution of annotated events with no PacBio hits. Overlap between the Event Analysis trancriptome (yellow) against the PacBio transcriptome (blue = complete splice match; red = incomplete splice match), for EA transcripts with 100% of events detected (B), at least 75% of events detected (C), at least 50% of events detected (D), and at least 25% of events detected (E).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F8.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/9/2923/F8.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-54" class="first-child">Comparison of Event Analysis with PacBio sequenced transcripts. (E) Distribution of log-APN of annotated events with and without PacBio hits. Blue line represents the of log-APN distribution of annotated events with PacBio hits, red line represents the of log-APN distribution of annotated events with no PacBio hits. Overlap between the Event Analysis trancriptome (yellow) against the PacBio transcriptome (blue = complete splice match; red = incomplete splice match), for EA transcripts with 100% of events detected (B), at least 75% of events detected (C), at least 50% of events detected (D), and at least 25% of events detected (E).</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">Of the ∼16,000 PacBio transcripts (<a id="xref-ref-72-15" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>), there are 9,653 that match to RefSeq transcripts, and of these 5,880 have all events unambiguously annotated to a single gene. Only 24 of the PacBio transcripts (&lt;1%) had fewer than 25% of their total associated events detected. Of the remaining set of 5,856 PacBio transcripts, all were included in the set of probable transcripts identified by EA (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). EA transcripts were categorized based on whether they had unique events, the proportion of total events detected, and the average coverage. The number of transcripts in each category for the 73,535 EA transcripts and 5,856 PacBio Transcripts are listed in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. Eliminating EA transcripts with fewer than 75% of their annotated events detected (38,913 transcripts with &lt;75% of total features detected, APN &gt; 0) reduces the complexity of the EA transcriptome by 53% (35,342 transcripts remaining) while missing only 3% of the PacBio transcriptome (194 of 5,880 PacBio transcripts). If a higher APN threshold is used to determine event detected (<em>e.g.</em>, APN ≥ 5) with a lower frequency of event detection (50%) results in less than 1% of the true PacBio transcriptome is missed and the EA transcriptome is much less complex (20,336 transcripts; <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). A similar number of transcripts, with a similar set of PacBio transcripts are identified with two thresholds (100% events detected (APN &gt; 0), 14,734 transcripts; ≥75% events detected (APN ≥ 5), 13,740 transcripts). These two sets have 9,310 transcripts are in common between the two possible transcriptomes. This overlap captures 79% of the PacBio transcripts that match to either of these possible transcriptomes. This demonstrates that the choice of parameters impacts the proportion of PacBio-validated transcripts that can be captured in the possible, reduced transcriptomes and that selecting a single set of parameters will included more PacBio transcripts than the intersection of two parameter sets.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490755/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490755/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Distribution of transcripts by proportion of events detected Number in parenthesis is the number of transcripts with a matching PacBio transcript</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h3>Event Analysis improves transcript abundance estimates</h3><p id="p-56">The mouse neural data used thus far demonstrates that using a reduced transcriptome as a prior improves the concordance between technical replicates and also captures most of the same reference transcripts as PacBio. An RNA-seq dataset of 3 lymphocyte populations (CD4<sup>+</sup>, CD8<sup>+</sup> and CD19<sup>+</sup> lymphocytes) from 81 T1D cases was used to determine whether a reduced transcriptome also reduces the variability of transcript estimates in a larger set of RNA-seq data.</p><p id="p-57">In T1D data, a total of 16,156 single-exons and 67,888 exon fragments of length at least 10 bp, and 38,367 junctions were detected from 8,700 genes in at least one cell type. This included 34,421 exon-exon junctions that are annotated to at least one transcript (90% of detected junctions), 402 junctions not annotated to any known human AceView transcript (1% of detected junctions), and 3,544 border junctions (9% of detected junctions). We used the criteria APN ≥ 5 and 75% of more events detected to a filter the reference with a resulting reduced transcriptome consisting of 26,437 transcripts (Figure S4, Additional File 1).</p><p id="p-58">We then examined whether using this reduced transcriptome reduced variability in abundance estimates compared to the complete transcriptome (∼596,000 AceView transcripts in human). As there are 81 subjects who have been examined for 3 lymphocyte cell types, we calculated the coefficient of variation (CV) for each transcript/cell type after quantitation using RSEM for the complete AceView transcriptome (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>, blue line) and for the reduced reference (26,437 transcripts). The distribution of the CV across transcripts was compared between the full reference and the filtered reference (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>). For each transcript the CV between the two was compared and if the CV was larger in the full reference it was scored (+) and of the CV was smaller it was scored (-). The null hypothesis that the distribution is random was tested using a sign test (<em>P</em> &lt; 0.0001). The overall distribution showed a dramatic decrease in the CV in the reduced reference and per transcript the CV was lower in the filtered reference. Together this indicates a dramatic increase in the concordance of estimates among samples. Similar results were obtained with eXpress (File S2).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the coefficient of variation of transcript estimates in T1D lymphocyte data. The distribution of CV between samples from the quantification for all AceView transcripts (blue line) and the reduced set of transcripts (red line) for (A) CD4+, (B) CD8+ and (C) CD19+ lymphocytes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413422693" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of the coefficient of variation of transcript estimates in T1D lymphocyte data. The distribution of CV between samples from the quantification for all AceView transcripts (blue line) and the reduced set of transcripts (red line) for (A) CD4&lt;sup&gt;+&lt;/sup&gt;, (B) CD8&lt;sup&gt;+&lt;/sup&gt; and (C) CD19&lt;sup&gt;+&lt;/sup&gt; lymphocytes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2923/F9.medium.gif" width="207" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/9/2923/F9.medium.gif" width="207" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2923/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490822" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-59" class="first-child">Distribution of the coefficient of variation of transcript estimates in T1D lymphocyte data. The distribution of CV between samples from the quantification for all AceView transcripts (blue line) and the reduced set of transcripts (red line) for (A) CD4<sup>+</sup>, (B) CD8<sup>+</sup> and (C) CD19<sup>+</sup> lymphocytes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">As an example, for the T1D data, in the <em>IKZF3</em> gene, a lymphoid transcription factor and T1D candidate gene (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Morgan <em>et al.</em> 1997</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hosokawa <em>et al.</em> 1999</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> 2009</a>), EA reduces the number of transcripts from 18 to four, and quantitation demonstrates there is a shift in expression such that the isoform IKZF3.iAug10 is expressed more frequently in CD4+ and CD8+ T cells than in CD19+ B cells (Supplementary Figures 5 and 6, Additional File 1). This isoform corresponds to the isoform variant 5 of <em>IKZF3</em>, which lacks two exons and several zinc finger binding domains (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Morgan <em>et al.</em> 1997</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hosokawa <em>et al.</em> 1999</a>). This enables us to follow up on the most-likely, relevant <em>IKZF3</em> transcripts in these cell types without the burden of transcripts that provide little additional information about the transcription of this gene.</p></div><div id="sec-32" class="subsection"><h3>Comparison of mouse NPC data With STAR</h3><p id="p-61">STAR identifies ∼24% of the junctions (883 out of 3,691) not present in the current annotation, and ∼95% of the junctions present in the annotation and verified by PacBio reads while EA detects 100% of the junctions present in the annotations. When adding the criteria of at least two reads to support the identification of a junction, STAR identifies only 3% of junctions while EA identifies 76%. STAR identifies 43% (229 out of 536) of the junctions that are novel combinations of existing donor/acceptors and EA detects ∼60% (321/536) of these junctions. Thus, mapping directly to the junction catalog that includes all possible logical junctions improves support of annotated junctions (Tables S3.2 to S3.5, File S3).</p></div><div id="sec-33" class="subsection"><h3>Comparison of mouse NPC data With iReckon</h3><p id="p-62">We also compared the set of possible transcripts identified by EA and the transcripts assembled by iReckon against the 5,880 PacBio transcripts with RefSeq identifiers. iReckon assembles 10% of the validation set of transcripts. EA is able to identify as many as 97% of the 5,880 PacBio transcripts, when considering transcripts with at least 75% of their associated events detected (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>; File S4). Even when only transcripts with 100% of their events detected are considered, 77% of the PacBio transcriptome is identified with EA, a substantial improvement over that of iReckon. In addition, the majority of iReckon transcripts are classified as novel transcripts (File S4), and only approximately 13% of the iReckon transcripts have good sequence similarity to the PacBio transcriptome. EA is also unable to eliminate ∼10,000-30,000 additional RefSeq transcripts, owing to the high degree of sequence similarity and event sharing with other transcripts within in the same genes.</p></div></div><div class="section discussion" id="sec-34"><h2 class="">Discussion</h2><p id="p-63">At the core of EA is the preparation of a catalog of junctions. Some analysis approaches (<em>e.g.</em>, (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Katz <em>et al.</em> 2010</a>)) restrict the definition of junctions to those present in existing annotations. Others attempt to estimate junctions <em>de novo</em> from short reads (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Trapnell <em>et al.</em> 2009</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Dobin <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kim <em>et al.</em> 2013a</a>). The literature indicates that the vast majority of novel isoforms are assembled from currently annotated exons (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Booms <em>et al.</em> 1999</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hide <em>et al.</em> 2001</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Zhou <em>et al.</em> 2010</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lim <em>et al.</em> 2011</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Eswaran <em>et al.</em> 2013</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kim <em>et al.</em> 2013b</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gabreski <em>et al.</em> 2016</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Nellore <em>et al.</em> 2016</a>; <a id="xref-ref-72-16" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). By designing a junction catalog that includes junctions created from all possible logical combinations of existing exons, and mapping to that catalog more true novel junctions are mapped than by mapping to the genome with a splice aware aligner. While completely novel junctions (<em>i.e.</em>, junctions with donor and/or acceptor sites not present in the reference annotation) will be missed, we find that in the study here of mouse NPC cells novel junctions and transcripts identified by PacBio are not frequently identified by <em>de novo</em> tools STAR or iReckon (Supplementary Files 3 and 4).</p><p id="p-64">EA views each transcript as an annotated set of discrete events. The detection and coverage of each event can be used directly in an analysis or to filter transcripts. Code for this is provided. EA does not rely on complex probabilistic models of transcription nor specific sequencing technology. Using this approach, in mouse neural progenitor cell RNA-seq data EA identified the overwhelming majority of annotated PacBio transcripts (5,686 of 5,880 transcripts, 97%) using the criteria 75% or more events detected (APN &gt; 0). A more stringent definition (least 75% of their events detected at APN ≥ 5) identifies 4,402 annotated PacBio transcripts.</p><p id="p-65">There is a limit to the ability of any RNA-seq based approach to identify transcripts. As EA relies on existing genome annotations, it is not able to identify completely novel exons or splice sites. This is a common limitation for annotation-based quantification approaches. EA does not <em>de novo</em> assemble transcripts, as accurate transcript reassembly and resolution from short-read data are known to result in a higher number of false transcripts (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Steijger <em>et al.</em> 2013</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Angelini <em>et al.</em> 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernard <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hayer <em>et al.</em> 2015</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Liu <em>et al.</em> 2016a</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Song <em>et al.</em> 2016</a>) and shown here. Long reads are able to clearly identify novel isoforms, provided adequate quality control mechanisms are applied (<a id="xref-ref-72-17" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). Another limitation is the resolvability of individual transcripts. Transcript resolvability is not a limitation of EA but of the uniqueness of transcripts. When we examined presence/absence of unique events among the PacBio transcripts we found that 30% of the PacBio transcripts have no unique events distinguishing them, meaning that these will never be identifiable from short read data alone. Issues with transcript quantification from short read RNA-seq data alone have been acknowledged and discussed (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Angelini <em>et al.</em> 2014</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kanitz <em>et al.</em> 2015</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Ding <em>et al.</em> 2017</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Williams <em>et al.</em> 2017</a>; <a id="xref-ref-72-18" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>).</p><p id="p-66">The newly released pRSEM cleverly tackles the issue of transcript resolvability by leveraging ChIP-seq data. In addition, pRSEM uses a binomial distribution to divide transcripts into two groups, probably expressed and unexpressed, before quantification. Dramatic improvements in transcript quantification were demonstrated using information from ChIP-seq and PolII to estimate the initial transcript set (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 2016b</a>). Additional methods aim to reduce the number of transcripts to quantify by leveraging long-read sequencing like PacBio to serve as a reference set of transcripts (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Au <em>et al.</em> 2013</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ning <em>et al.</em> 2017</a>), which improve the accuracy of genes model (<a id="xref-ref-72-19" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>) but at the cost of excluding other, likely-expressed transcripts. However, much like pRSEM, such approaches rely on additional data to determine the likely transcriptome. Other methods that require only short read data to identify reduced reference set have also been successful (<em>e.g.</em>, (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Mezlini <em>et al.</em> 2013</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Soneson <em>et al.</em> 2016</a>)). One of these, iReckon, which leverages existing annotations to guide the reassembly of transcripts from short reads and quantification of assembled sequences (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Mezlini <em>et al.</em> 2013</a>). iReckon also creates novel junctions and exons based on read mapping. The overall approach is to create all possible transcripts from the data, then eliminate those with low/no coverage. However, we find iReckon tends to reconstruct many novel sample-specific transcripts not supported by long read data and also reported in the literature (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Angelini <em>et al.</em> 2014</a>).</p><p id="p-67">Another approach, taken by Soneson <em>et al.</em> (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Soneson <em>et al.</em> 2016</a>), has the transcript abundance algorithm guide what transcripts are likely present. First, all possible transcripts are quantified and the proportion that each transcripts contributes toward the overall expression of it corresponding gene. Transcripts with low abundance and that contribute little toward to its gene’s expression are removed, and the remaining transcript sequences are re-quantified. While this approach is similar to EA (<em>i.e.</em>, elimination of unlikely transcripts based on coverage), it may eliminate transcripts that are actually present but whose reads have been initially misassigned to other transcripts. For example, from the mouse neural data used in this manuscript, elimination of the transcript NM_025668 as suggested by this algorithm also eliminates the only PacBio-sequenced transcript for the <em>Spcs2</em> gene, and assigns all reads to a different transcript, XM_006508117 (<a id="xref-ref-72-20" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). This read misassignment is likely due to variability of the length of the 3′ UTR: while the junctions of the PacBio transcript completely match those NM_025668, the 3′UTR of the PacBio transcript better matches that of XM_006508117 (<a id="xref-ref-72-21" class="xref-bibr" href="#ref-72">Tardaguila <em>et al.</em> 2018</a>). This occurs because RSEM infers the likely transcript based on coverage in the 3′ UTR. This also highlights the need for a good-quality reference transcriptome that best reflects the actual expression of the experimental system in use.</p><p id="p-68">Providing a reduced set of reference transcript sequences based on the expression of individual transcript events dramatically improves the performance of both RSEM (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Li and Dewey 2011</a>) and eXpress (<a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Roberts and Pachter 2013</a>). The agreement between replicates jumps from 80 to above 90% and is as high as 99%, with a dramatic reduction in estimates of variability. The advantages to EA are: (1) only RNA-seq data are required; (2) no model for transcript generation is needed; (3) no model for bias in sequencing technology is needed; and (4) the degree of over/underspecification of the likely transcriptome can be easily adjusted based on the goals of the particular study by applying different thresholds (see Figure S4, Additional File 1). The improved transcript estimates, in turn, will improve the accuracy of differential expression analyses.</p><p id="p-69">Junctions and exonic regions can be used directly to test for differential expression. In a comparison of cell types among type 1 diabetes patients, we found that 20% of splicing events were differentially detected, 34% of genes were alternatively spliced among cell types (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Newman <em>et al.</em> 2017</a>) and the genes implicated in immune function and autoimmunity are enriched for splicing differences (<a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Newman <em>et al.</em> 2017</a>).</p></div><div class="section conclusions" id="sec-35"><h2 class="">Conclusions</h2><p id="p-70">EA is straightforward to apply. There are only two relatively simple new steps that are needed- the creation of an expanded junction catalog, and the scoring of transcripts based on simple measures of detection and abundance. Here we demonstrate that the junction catalog successfully identifies novel junctions and that using transcriptional events to reduce the complexity of a probable transcriptome dramatically improves transcript estimates from RNA-seq data. EA provides rich annotation for events (junctions and exons) enhancing our understanding of the effect of splicing in human disease (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akin <em>et al.</em> 2016</a>; <a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Newman <em>et al.</em> 2017</a>). Examining individual junctions, exons and exon fragments reveals detail in differential detection/expression patterns previously obscured.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-71">Human subjects data provided by the Type 1 Diabetes Genetics Consortium, a collaborative clinical study sponsored by NIDDK, National Institute of Allergy and Infectious Diseases, National Human Genome Research Institute, National Institute of Child Health and Human Development, and Juvenile Diabetes Research Foundation International.</p><p id="p-72">This work was supported by NIGMS R01-102227 (LMM), the University of Florida Informatics Institute Seed Funds and the Spanish Ministry of Economy and Competitiveness grant BIO2015-71658-R (AC), and by DK106718 and DK116954 grant (PC). The work was also supported by University of Florida Research Computing and HiPerGator2. The contents of this article are solely the responsibility of the authors and do not necessarily represent the official views of the NIH or the University of Florida.</p><p id="p-73">JRBN and LMM conceived and designed the project. JRBN performed all of the analysis. AC and MT provided the mouse neural data, and assisted in the interpretation. PC provided the T1D data and assisted in interpretation of results. All authors contributed to the writing and revision of the manuscript, and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-74">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6205793">https://doi.org/10.25387/g3.6205793</a>.</p></li><li class="fn" id="fn-8"><p id="p-75"><em>Communicating editor: B. Oliver</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-01"><span class="received-label">Received </span>May 1, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-04"><span class="accepted-label">Accepted </span>July 4, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Newman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugrue</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RNA-seq analysis of impact of PNN on gene expression and alternative splicing in corneal epithelial cells.</span> <abbr class="cit-jnl-abbrev">Mol. Vis.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BVis.%26rft.volume%253D22%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.2" data-doi="10.1101/gr.133744.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting differential usage of exons from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2008</span>–<span class="cit-lpage">2017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.133744.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.133744.111%26rft_id%253Dinfo%253Apmid%252F22722343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8yMDA4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.3" data-doi="10.1186/1471-2105-15-135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angelini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Canditiis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Feis</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Computational approaches for isoform detection and estimation: good and bad news.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-15-135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-135%26rft_id%253Dinfo%253Apmid%252F24885830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24885830&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.4" data-doi="10.1073/pnas.1320101110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sebastiano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Afshar</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durruthy</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterization of the human ESC transcriptome by hybrid sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E4821</span>–<span class="cit-lpage">E4830</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1320101110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1320101110%26rft_id%253Dinfo%253Apmid%252F24282307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzUwL0U0ODIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.5" data-doi="10.1038/ng.381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Concannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akolkar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide association study and meta-analysis find that over 40 loci affect risk of type 1 diabetes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.381</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D41%26rft.issue%253D6%26rft.spage%253D703%26rft.epage%253D707%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Band%2Bmeta-analysis%2Bfind%2Bthat%2Bover%2B40%2Bloci%2Baffect%2Brisk%2Bof%2Btype%2B1%2Bdiabetes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.381%26rft_id%253Dinfo%253Apmid%252F19430480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19430480&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266411700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.6" data-doi="10.1186/s12859-015-0695-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacob</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mairal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viara</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vert</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A convex formulation for joint RNA isoform detection and quantification from multiple RNA-seq samples.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-015-0695-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0695-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0695-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.7" data-doi="10.1016/S0140-6736(86)90837-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Statistical methods for assessing agreement between two methods of clinical measurement.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0140-6736(86)90837-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DBland%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D8476%26rft.spage%253D307%26rft.epage%253D310%26rft.atitle%253DStatistical%2Bmethods%2Bfor%2Bassessing%2Bagreement%2Bbetween%2Btwo%2Bmethods%2Bof%2Bclinical%2Bmeasurement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252886%252990837-8%26rft_id%253Dinfo%253Apmid%252F2868172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(86)90837-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2868172&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986AYW4000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.8" data-doi="10.1034/j.1399-0004.1999.550207.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Booms</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cisler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godfrey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiecke</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Novel exon skipping mutation in the fibrillin-1 gene: Two ‘hot spots’ for the neonatal Marfan syndrome.</span> <abbr class="cit-jnl-abbrev">Clin. Genet.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1034/j.1399-0004.1999.550207.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bgenetics%26rft.stitle%253DClin%2BGenet%26rft.aulast%253DBooms%26rft.auinit1%253DP.%26rft.volume%253D55%26rft.issue%253D2%26rft.spage%253D110%26rft.epage%253D117%26rft.atitle%253DNovel%2Bexon%2Bskipping%2Bmutation%2Bin%2Bthe%2Bfibrillin-1%2Bgene%253A%2Btwo%2B%2527hot%2Bspots%2527%2Bfor%2Bthe%2Bneonatal%2BMarfan%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1399-0004.1999.550207.x%26rft_id%253Dinfo%253Apmid%252F10189088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1399-0004.1999.550207.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10189088&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079072900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.9" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.10" data-doi="10.1186/s13059-015-0865-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canzar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreotti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weese</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klau</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CIDANE: comprehensive isoform discovery and abundance estimation.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0865-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0865-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0865-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.11" data-doi="10.1038/nmeth.1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cloonan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">A. R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolle</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">B. B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faulkner</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Stem cell transcriptome profiling via massive-scale mRNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1223</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DCloonan%26rft.auinit1%253DN.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D613%26rft.epage%253D619%26rft.atitle%253DStem%2Bcell%2Btranscriptome%2Bprofiling%2Bvia%2Bmassive-scale%2BmRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1223%26rft_id%253Dinfo%253Apmid%252F18516046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516046&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.12" data-doi="10.1186/1471-2164-14-659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Male-specific Fruitless isoforms have different regulatory roles conferred by distinct zinc finger DNA binding domains.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-659</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-659%26rft_id%253Dinfo%253Apmid%252F24074028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24074028&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.13" data-doi="10.2174/1389202918666170215125048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">L. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rath</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparison of alternative splicing junction detection tools using RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2174/1389202918666170215125048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenomics%26rft.volume%253D18%26rft.spage%253D268%26rft_id%253Dinfo%253Adoi%252F10.2174%252F1389202918666170215125048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/1389202918666170215125048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.14" data-doi="10.1038/nature11233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Landscape of transcription in human cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDjebali%26rft.auinit1%253DS.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D101%26rft.epage%253D108%26rft.atitle%253DLandscape%2Bof%2Btranscription%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11233%26rft_id%253Dinfo%253Apmid%252F22955620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955620&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.15" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.16" data-doi="10.1038/srep01689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eswaran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godbole</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mudvari</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA sequencing of cancer reveals novel splicing alterations.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1689</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep01689</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D1689%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep01689%26rft_id%253Dinfo%253Apmid%252F23604310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep01689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23604310&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.17" data-doi="10.1534/genetics.116.188797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon-Novelo</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerken</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Lehmann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Buffering of genetic regulatory networks in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.188797</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.188797%26rft_id%253Dinfo%253Apmid%252F27194752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy8zLzExNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.18" data-doi="10.1101/gr.8.9.967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Florea</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartzell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A computer program for aligning a cDNA sequence with a genomic DNA sequence.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.8.9.967</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DFlorea%26rft.auinit1%253DL.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D967%26rft.epage%253D974%26rft.atitle%253DA%2BComputer%2BProgram%2Bfor%2BAligning%2Ba%2BcDNA%2BSequence%2Bwith%2Ba%2BGenomic%2BDNA%2BSequence%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.9.967%26rft_id%253Dinfo%253Apmid%252F9750195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvOS85NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.19" data-doi="10.1093/nar/gkm311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foissac</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">ASTALAVISTA: dynamic and flexible analysis of alternative splicing events in custom gene datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W297</span>–<span class="cit-lpage">W299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm311%26rft_id%253Dinfo%253Apmid%252F17485470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485470&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255311500055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.20" data-doi="10.1093/bioinformatics/btv272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frazee</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaffe</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leek</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Polyester: simulating RNA-seq datasets with differential transcript expression.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2778</span>–<span class="cit-lpage">2784</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv272%26rft_id%253Dinfo%253Apmid%252F25926345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25926345&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.21" data-doi="10.1074/jbc.M115.709675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabreski</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaghasia</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novakova</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">N. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierchala</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Exon Skipping in the RET Gene Encodes Novel Isoforms That Differentially Regulate RET Protein Signal Transduction.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">16249</span>–<span class="cit-lpage">16262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M115.709675</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.709675%26rft_id%253Dinfo%253Apmid%252F27226544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTEvMzEvMTYyNDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.22" data-doi="10.1093/bioinformatics/bts260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaus</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honkela</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattray</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identifying differentially expressed transcripts from RNA-seq data with biological variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1721</span>–<span class="cit-lpage">1728</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts260%26rft_id%253Dinfo%253Apmid%252F22563066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22563066&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305825600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.23" data-doi="10.1371/journal.pone.0156132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Prediction and quantification of splice events from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0156132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0156132</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0156132%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.24" data-doi="10.1093/gbe/evu060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">What the X has to do with it: differences in regulatory variability between the sexes in Drosophila simulans.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">818</span>–<span class="cit-lpage">829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evu060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu060%26rft_id%253Dinfo%253Apmid%252F24696400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24696400&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.25" data-doi="10.1093/molbev/msr318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novelo</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allelic imbalance in Drosophila hybrid heads: exons, isoforms, and evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1521</span>–<span class="cit-lpage">1532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr318%26rft_id%253Dinfo%253Apmid%252F22319150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22319150&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304200600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.26" data-doi="10.1093/bioinformatics/btv488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayer</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pizarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahens</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogenesch</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Benchmark analysis of algorithms for determining and quantifying full-length mRNA splice forms from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3938</span>–<span class="cit-lpage">3945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv488%26rft_id%253Dinfo%253Apmid%252F26338770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26338770&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hide</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babenko</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heusden</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seoighe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelso</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The contribution of exon-skipping events on chromosome 22 to protein coding diversity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1848</span>–<span class="cit-lpage">1853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Apmid%252F11691849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS8xODQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.28" data-doi="10.1006/geno.1999.5949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosokawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seto</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Human Aiolos, an Ikaros-related zinc finger DNA binding protein: cDNA cloning, tissue expression pattern, and chromosomal mapping.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/geno.1999.5949</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHosokawa%26rft.auinit1%253DY.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D326%26rft.epage%253D329%26rft.atitle%253DHuman%2Baiolos%252C%2Ban%2Bikaros-related%2Bzinc%2Bfinger%2BDNA%2Bbinding%2Bprotein%253A%2BcDNA%2Bcloning%252C%2Btissue%2Bexpression%2Bpattern%252C%2Band%2Bchromosomal%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.1999.5949%26rft_id%253Dinfo%253Apmid%252F10552935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.1999.5949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10552935&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083824400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.29" data-doi="10.1016/j.cell.2014.11.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irimia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weatheritt</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parikshak</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonatopoulos-Pournatzis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Highly Conserved Program of Neuronal Microexons Is Misregulated in Autistic Brains.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1511</span>–<span class="cit-lpage">1523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.11.035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.11.035%26rft_id%253Dinfo%253Apmid%252F25525873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.11.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25525873&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.30" data-doi="10.1093/bioinformatics/btp113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Statistical inferences for isoform expression in RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1026</span>–<span class="cit-lpage">1032</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp113%26rft_id%253Dinfo%253Apmid%252F19244387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19244387&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265094400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.31" data-doi="10.1534/genetics.115.177246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangul</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isvilanonda</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Discovering single nucleotide polymorphisms regulating human gene expression using allele specific expression from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177246%26rft_id%253Dinfo%253Apmid%252F27765809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzEwNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.32" data-doi="10.1186/s13059-015-0702-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanitz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gypas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative assessment of methods for the computational inference of transcript isoform abundance from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0702-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0702-5%26rft_id%253Dinfo%253Apmid%252F26201343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0702-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26201343&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.33" data-doi="10.1038/nmeth.1528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airoldi</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis and design of RNA sequencing experiments for identifying isoform regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1015</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1528</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DKatz%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1009%26rft.epage%253D1015%26rft.atitle%253DAnalysis%2Band%2Bdesign%2Bof%2BRNA%2Bsequencing%2Bexperiments%2Bfor%2Bidentifying%2Bisoform%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1528%26rft_id%253Dinfo%253Apmid%252F21057496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1528&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21057496&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.34" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-4-r36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">In</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Extensive novel hybrid isoforms revealed by RNA sequencing of 120 primary breast cancer samples.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D2%26rft.atitle%253DHighlights%2Bfrom%2BRecent%2BCancer%2BLiterature%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjY6IjczLzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.36" data-doi="10.1007/978-1-62703-773-0_4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korir</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seoighe</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inference of allele-specific expression from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Plant Epigenetics and Epigenomics: Methods and Protocols</abbr> <span class="cit-vol">1112</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-773-0_4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BEpigenetics%2Band%2BEpigenomics%253A%2BMethods%2Band%2BProtocols%26rft.volume%253D1112%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-773-0_4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-773-0_4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.37" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2009-10-3-r25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.38" data-doi="10.1186/s12859-015-0704-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alver</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">EMSAR: estimation of transcript abundance from RNA-seq data by mappability-based segmentation and reclustering.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-015-0704-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0704-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0704-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.39" data-doi="10.1186/1471-2164-15-920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leon-Novelo</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A flexible Bayesian method for detecting allelic imbalance in RNA-seq data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">920</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D920%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-920%26rft_id%253Dinfo%253Apmid%252F25339465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25339465&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.40" data-doi="10.1186/1471-2105-12-323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewey</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-323</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.spage%253D323%26rft.epage%253D323%26rft.atitle%253DRSEM%253A%2Baccurate%2Btranscript%2Bquantification%2Bfrom%2BRNA-Seq%2Bdata%2Bwith%2Bor%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-323%26rft_id%253Dinfo%253Apmid%252F21816040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21816040&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.41" data-doi="10.1093/bioinformatics/btp692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruotti</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewey</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RNA-Seq gene expression estimation with read mapping uncertainty.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">500</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp692%26rft_id%253Dinfo%253Apmid%252F20022975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20022975&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274484300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-8.9.2923.42"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXic preprint arXiv:1303.3997.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.43" data-doi="10.1093/bioinformatics/bts559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome assembly and isoform expression level estimation from biased RNA-Seq reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2914</span>–<span class="cit-lpage">2921</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts559%26rft_id%253Dinfo%253Apmid%252F23060617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060617&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311303500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.44" data-doi="10.1111/j.1365-2052.2011.02225.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">I. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">B. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Novel alternative splicing by exon skipping in KIT associated with whole-body roan in an intercrossed population of Landrace and Korean Native pigs.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2052.2011.02225.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DLim%26rft.auinit1%253DH.%2BT.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D451%26rft.epage%253D455%26rft.atitle%253DNovel%2Balternative%2Bsplicing%2Bby%2Bexon%2Bskipping%2Bin%2BKIT%2Bassociated%2Bwith%2Bwhole-body%2Broan%2Bin%2Ban%2Bintercrossed%2Bpopulation%2Bof%2BLandrace%2Band%2BKorean%2BNative%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2011.02225.x%26rft_id%253Dinfo%253Apmid%252F21749430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2011.02225.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21749430&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">TransComb: genome-guided transcriptome assembly via combing junctions in splicing graphs.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.46" data-doi="10.1101/gr.199174.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanalkumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bresnick</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewey</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Integrative analysis with ChIP-seq advances the limits of transcript quantification from RNA-seq.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.199174.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.199174.115%26rft_id%253Dinfo%253Apmid%252F27405803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI2LzgvMTEyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.47" data-doi="10.1186/1471-2164-10-422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Main</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Allele-specific expression assays using Solexa.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">422</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-10-422</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMain%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D10%26rft.spage%253D422%26rft.epage%253D422%26rft.atitle%253DAllele-specific%2Bexpression%2Bassays%2Busing%2BSolexa.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-422%26rft_id%253Dinfo%253Apmid%252F19740431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19740431&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.48" data-doi="10.1186/1471-2164-12-293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopiano</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oberg</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA-seq: technical variability and sampling.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMcIntyre%26rft.auinit1%253DL.%2BM.%26rft.volume%253D12%26rft.spage%253D293%26rft.epage%253D293%26rft.atitle%253DRNA-seq%253A%2Btechnical%2Bvariability%2Band%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-293%26rft_id%253Dinfo%253Apmid%252F21645359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21645359&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.49" data-doi="10.1101/gr.142232.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mezlini</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiume</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buske</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savich</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">iReckon: Simultaneous isoform discovery and abundance estimation from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.142232.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.142232.112%26rft_id%253Dinfo%253Apmid%252F23204306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzMvNTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.50" data-doi="10.1038/nature08903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez-Arcelus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lach</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcriptome genetics using second generation sequencing in a Caucasian population.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">777</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08903</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMontgomery%26rft.auinit1%253DS.%2BB.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D773%26rft.epage%253D777%26rft.atitle%253DTranscriptome%2Bgenetics%2Busing%2Bsecond%2Bgeneration%2Bsequencing%2Bin%2Ba%2BCaucasian%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08903%26rft_id%253Dinfo%253Apmid%252F20220756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220756&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.51" data-doi="10.1093/emboj/16.8.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avitahl</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrikopoulos</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Aiolos, a lymphoid restricted transcription factor that interacts with Ikaros to regulate lymphocyte differentiation.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2004</span>–<span class="cit-lpage">2013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/emboj/16.8.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DMorgan%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D2004%26rft.epage%253D2013%26rft.atitle%253DAiolos%252C%2Ba%2Blymphoid%2Brestricted%2Btranscription%2Bfactor%2Bthat%2Binteracts%2Bwith%2BIkaros%2Bto%2Bregulate%2Blymphocyte%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.8.2004%26rft_id%253Dinfo%253Apmid%252F9155026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxNi84LzIwMDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjkyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.52" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.2923.53" data-doi="10.1002/0471142727.mb0411s89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RNA-Seq: a method for comprehensive transcriptome analysis.</span> <span class="cit-source">Curr. Protoc. Mol. Biol.</span> <span class="cit-vol">Chapter 4</span>: <span class="cit-fpage">Unit 4.11.1</span>.–<span class="cit-lpage">Unit 4.11.13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471142727.mb0411s89</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMol.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471142727.mb0411s89%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471142727.mb0411s89&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.54" data-doi="10.1186/1471-2164-15-S5-S5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nariai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mimori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TIGAR2: sensitive and accurate estimation of transcript isoform expression with longer RNA-Seq reads.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">S5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-S10-S5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253DS5%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-S5-S5%26rft_id%253Dinfo%253Apmid%252F24564547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-S5-S5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24564547&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.55" data-doi="10.1186/s13059-016-1118-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nellore</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaffe</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortin</span>  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alquicira-Hernandez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collado-Torres</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1118-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D266%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1118-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1118-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">N.R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">New</span>  <span class="cit-name-given-names">F.N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalton</span>  <span class="cit-name-given-names">J.E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arbeitman</span>  <span class="cit-name-given-names">M.N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Neurons that underlie Drosophila melanogaster reproductive behaviors: detection of a large male-bias in gene expression in fruitless-expressing neurons. G3 (Bethesda) 6: 2455–2465.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.57" data-doi="10.1101/gr.217984.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mika</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">New</span>  <span class="cit-name-given-names">F. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onengut-Gumuscu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Disease-specific biases in alternative splicing and tissue-specific dysregulation revealed by multitissue profiling of lymphocyte gene expression in type 1 diabetes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1807</span>–<span class="cit-lpage">1815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.217984.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.217984.116%26rft_id%253Dinfo%253Apmid%252F29025893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMS8xODA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.2923.58" data-doi="10.1186/1748-7188-6-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicolae</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangul</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandoiu</span>  <span class="cit-name-given-names">I. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zelikovsky</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimation of alternative splicing isoform frequencies from RNA-Seq data.</span> <span class="cit-source">Algorithms Mol. Biol.</span> <span class="cit-vol">6</span>: <span class="cit-fpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1748-7188-6-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DNicolae%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DEstimation%2Bof%2Balternative%2Bsplicing%2Bisoform%2Bfrequencies%2Bfrom%2BRNA-Seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-9%26rft_id%253Dinfo%253Apmid%252F21504602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1748-7188-6-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21504602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.59" data-doi="10.1038/srep43793"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Hybrid sequencing and map finding (HySeMaFi): optional strategies for extensively deciphering gene splicing and expression in organisms without reference genome.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">43793</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep43793</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D43793%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep43793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep43793&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.60" data-doi="10.1038/nrg2815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pastinen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide allele-specific analysis: insights into regulatory variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2815</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPastinen%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D533%26rft.epage%253D538%26rft.atitle%253DGenome-wide%2Ballele-specific%2Banalysis%253A%2Binsights%2Binto%2Bregulatory%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2815%26rft_id%253Dinfo%253Apmid%252F20567245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20567245&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279988800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.61" data-doi="10.1038/nbt.2862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D462%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2862%26rft_id%253Dinfo%253Apmid%252F24752080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24752080&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.62" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.63" data-doi="10.1016/j.jbi.2016.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rezaeian</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavakoli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavallo-Medved</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rueda</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A novel model used to detect differential splice junctions as biomarkers in prostate cancer from RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">J. Biomed. Inform.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">430</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jbi.2016.03.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiomed.%2BInform.%26rft.volume%253D60%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbi.2016.03.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbi.2016.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.64" data-doi="10.1038/nmeth.2251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Streaming fragment assignment for real-time analysis of sequencing experiments.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2251%26rft_id%253Dinfo%253Apmid%252F23160280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160280&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312810100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.65" data-doi="10.1186/gb-2011-12-3-r22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaghey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Improving RNA-Seq expression estimates by correcting for fragment bias.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-3-r22</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRoberts%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253DR22%26rft.epage%253DR22%26rft.atitle%253DImproving%2BRNA-Seq%2Bexpression%2Bestimates%2Bby%2Bcorrecting%2Bfor%2Bfragment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-3-r22%26rft_id%253Dinfo%253Apmid%252F21410973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-3-r22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21410973&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.66" data-doi="10.1038/nbt.2705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilgner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A single-molecule long-read survey of the human transcriptome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D1009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2705%26rft_id%253Dinfo%253Apmid%252F24108091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24108091&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.67" data-doi="10.1186/s13059-015-0862-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soneson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowicka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Isoform prefiltering improves performance of count-based methods for analysis of differential transcript usage.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0862-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0862-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0862-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.68" data-doi="10.1093/nar/gkw158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabunciyan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florea</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CLASS2: accurate and efficient splice variant annotation from RNA-seq reads.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw158%26rft_id%253Dinfo%253Apmid%252F26975657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26975657&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.69" data-doi="10.1038/nmeth.2714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steijger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abril</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engstrom</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kokocinski</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assessment of transcript reconstruction methods for RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2714</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2714%26rft_id%253Dinfo%253Apmid%252F24185837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24185837&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327698100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.70" data-doi="10.1186/1471-2105-14-320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturgill</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinow</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Design of RNA splicing analysis null models for post hoc filtering of Drosophila head RNA-Seq data with the splicing analysis kit (Spanki).</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-14-320</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D320%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-320%26rft_id%253Dinfo%253Apmid%252F24209455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24209455&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.71" data-doi="10.1111/j.1541-0420.2011.01654.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Statistical Framework for eQTL Mapping Using RNA-seq Data.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1541-0420.2011.01654.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D68%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2011.01654.x%26rft_id%253Dinfo%253Apmid%252F21838806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2011.01654.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21838806&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301924400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.72" data-doi="10.1101/gr.222976.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tardaguila</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Fuente</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Palacios</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">SQANTI: extensive characterization of long-read transcript sequences for quality control in full-length transcriptome identification and quantification.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">411</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.222976.117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.222976.117%26rft_id%253Dinfo%253Apmid%252F29440222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI4LzMvMzk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.73" data-doi="10.1371/journal.pone.0162868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tombácz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csabai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oláh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balázs</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Likó</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Full-length isoform sequencing reveals novel transcripts and substantial transcriptional overlaps in a herpesvirus.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0162868</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0162868</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0162868%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0162868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0162868&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.74" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19289445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.75" data-doi="10.1038/nprot.2012.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2012.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D562%26rft.epage%253D578%26rft.atitle%253DDifferential%2Bgene%2Band%2Btranscript%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bexperiments%2Bwith%2BTopHat%2Band%2BCufflinks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.016%26rft_id%253Dinfo%253Apmid%252F22383036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22383036&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.76" data-doi="10.1038/nrg3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repetitive DNA and next-generation sequencing: computational challenges and solutions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTreangen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D46%26rft.atitle%253DRepetitive%2BDNA%2Band%2Bnext-generation%2Bsequencing%253A%2Bcomputational%2Bchallenges%2Band%2Bsolutions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3117%26rft_id%253Dinfo%253Apmid%252F22124482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22124482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.77" data-doi="10.1186/gb-2011-12-2-r13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goncalves</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype and isoform specific expression estimation using multi-mapping RNA-seq reads.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-2-r13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DTurro%26rft.auinit1%253DE.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253DR13%26rft.epage%253DR13%26rft.atitle%253DHaplotype%2Band%2Bisoform%2Bspecific%2Bexpression%2Bestimation%2Busing%2Bmulti-mapping%2BRNA-seq%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-2-r13%26rft_id%253Dinfo%253Apmid%252F21310039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-2-r13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310039&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.78" data-doi="10.1002/wrna.1418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ustianenko</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weyn-Vanhentenryck</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Microexons: discovery, regulation, and function.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. RNA</abbr> <span class="cit-vol">8</span>: e1418.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wrna.1418</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BRNA%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.1418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.1418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.79" data-doi="10.1038/ncomms11708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regulski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">11708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms11708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D11708%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.80" data-doi="10.1038/nrg2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.81" data-doi="10.1186/s12859-016-1457-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baccarella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Empirical assessment of analysis workflows for differential expression analysis of human samples using RNA-Seq.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1457-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1457-z%26rft_id%253Dinfo%253Apmid%252F28095772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1457-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28095772&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.82" data-doi="10.1093/nar/gkt216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anczukow</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krainer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">OLego: fast and sensitive mapping of spliced mRNA-Seq reads using small seeds.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">5149</span>–<span class="cit-lpage">5163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt216%26rft_id%253Dinfo%253Apmid%252F23571760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23571760&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319806600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.9.2923.83" data-doi="10.1093/bioinformatics/btr712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lameijer</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">’t Hoen</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slagboom</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PASSion: a pattern growth algorithm-based pipeline for splice junction detection in paired-end RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr712</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr712%26rft_id%253Dinfo%253Apmid%252F22219203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22219203&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2923.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300490500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2923.84" data-doi="http://dx.doi.org/10.1186/1471-2105-11-S11-S7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PEPPI: a peptidomic database of human protein isoforms for proteomics experiments.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">S7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D11%26rft.spage%253DS7%26rft_id%253Dinfo%253Adoi%252Fhttp%253A%252F%252Fdx.doi.org%252F10.1186%252F1471-2105-11-S11-S7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=http://dx.doi.org/10.1186/1471-2105-11-S11-S7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/2923.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2913" title="Disruption of INOS, a Gene Encoding myo-Inositol Phosphate Synthase, Causes Male Sterility in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2941" title="Acute Effects of Drugs on Caenorhabditis elegans Movement Reveal Complex Responses and Plasticity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2923" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aFPmq0pJZFpI1_fNqac4My6SseiY6-cMvyOENQq8MeA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/2923" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/2923" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/2923" class="active" data-icon-position="" data-hide-link-title="0">Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490734" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ghgSvs70qpGZxnrbaF_3mfGpe7zaP0IMptHrOCno26U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2923.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/2923" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-tH2xYo1usD0uxN4tjWTe3BHoQw-2fVTRhiXyP3OMz3g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/2923" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2923.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490734" id="node490734--2" data-pisa="ggg;8/9/2923" data-pisa-master="ggg;g3.118.200373" data-apath="/ggg/8/9/2923.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8486-2801" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeremy R. B. Newman</span>, <span class="highwire-citation-author" data-delta="1">Patrick Concannon</span>, <span class="highwire-citation-author" data-delta="2">Manuel Tardaguila</span>, <span class="highwire-citation-author" data-delta="3">Ana Conesa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lauren M. McIntyre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2923-2940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200373 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490734--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeremy R. B. Newman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BR.%2BB.%2BNewman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Newman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeremy%2BR.%2BB.%2BNewman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8486-2801" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeremy R. B. Newman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick Concannon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Institute and Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BConcannon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Concannon%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BConcannon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Manuel Tardaguila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences, University of Florida, Gainesville, Florida, 32610; Wellcome Trust Sanger Institute, Hinxton, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manuel%2BTardaguila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tardaguila%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManuel%2BTardaguila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ana Conesa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences; Genetics Institute, University of Florida, Gainesville, Florida, 32610; Genomics of Gene Expression Lab, Prince Felipe Research Center, Valencia, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BConesa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conesa%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BConesa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aconesa@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">aconesa@ufl.edu</a>
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lauren M. McIntyre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aconesa@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">aconesa@ufl.edu</a>
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/2923" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/2923&amp;title=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/2923&amp;title=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/2923&amp;title=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/2923&amp;text=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/2923&amp;title=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/2923&amp;t=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/2923&amp;title=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/2923&amp;title=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490734/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490734" id="node490734--3" data-pisa="ggg;8/9/2923" data-pisa-master="ggg;g3.118.200373" data-apath="/ggg/8/9/2923.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Event Analysis: Using Transcript Events To Improve Estimates of Abundance in RNA-seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8486-2801" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeremy R. B. Newman</span>, <span class="highwire-citation-author" data-delta="1">Patrick Concannon</span>, <span class="highwire-citation-author" data-delta="2">Manuel Tardaguila</span>, <span class="highwire-citation-author" data-delta="3">Ana Conesa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lauren M. McIntyre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2923-2940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200373 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490734--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeremy R. B. Newman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BR.%2BB.%2BNewman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Newman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeremy%2BR.%2BB.%2BNewman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8486-2801" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeremy R. B. Newman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick Concannon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Institute and Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BConcannon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Concannon%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BConcannon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Manuel Tardaguila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences, University of Florida, Gainesville, Florida, 32610; Wellcome Trust Sanger Institute, Hinxton, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manuel%2BTardaguila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tardaguila%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManuel%2BTardaguila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ana Conesa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology and Cell Science, Institute for Food and Agricultural Sciences; Genetics Institute, University of Florida, Gainesville, Florida, 32610; Genomics of Gene Expression Lab, Prince Felipe Research Center, Valencia, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BConesa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conesa%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BConesa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aconesa@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">aconesa@ufl.edu</a>
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Molecular Genetics and Microbiology and Genetics Institute, University of Florida, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lauren M. McIntyre</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aconesa@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">aconesa@ufl.edu</a>
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2923&amp;rft.epage=2940&amp;rft.atitle=Event%20Analysis%3A%20Using%20Transcript%20Events%20To%20Improve%20Estimates%20of%20Abundance%20in%20RNA-seq%20Data&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Newman%2C+Jeremy+R.+B.&amp;rft.au=Concannon%2C+Patrick&amp;rft.au=Tardaguila%2C+Manuel&amp;rft.au=Conesa%2C+Ana&amp;rft.au=McIntyre%2C+Lauren+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490734/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490734/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490734/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490734/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490734/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490734/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490734/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490734/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490734/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490734/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490734/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490734/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/2923" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Background</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-34" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#sec-35" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/2923.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/2923.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/9/2923.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/9/2923.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
