<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/2/263" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/2/263.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/2/263.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/2/263.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/2/263/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/2/263.atom" />
<meta name="HW.pisa" content="ggg;6/2/263" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Mouse Universal Genotyping Array: From Substrains to Subspecies" />
<meta name="DC.Identifier" content="10.1534/g3.115.022087" />
<meta name="DC.Date" content="2016-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Morgan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping microarrays are an important resource for genetic mapping, population genetics, and monitoring of the genetic integrity of laboratory stocks. We have developed the third generation of the Mouse Universal Genotyping Array (MUGA) series, GigaMUGA, a 143,259-probe Illumina Infinium II array for the house mouse ( Mus musculus ). The bulk of the content of GigaMUGA is optimized for genetic mapping in the Collaborative Cross and Diversity Outbred populations, and for substrain-level identification of laboratory mice. In addition to 141,090 single nucleotide polymorphism probes, GigaMUGA contains 2006 probes for copy number concentrated in structurally polymorphic regions of the mouse genome. The performance of the array is characterized in a set of 500 high-quality reference samples spanning laboratory inbred strains, recombinant inbred lines, outbred stocks, and wild-caught mice. GigaMUGA is highly informative across a wide range of genetically diverse samples, from laboratory substrains to other Mus species. In addition to describing the content and performance of the array, we provide detailed probe-level annotation and recommendations for quality control." />
<meta name="DC.Contributor" content="Andrew P. Morgan" />
<meta name="DC.Contributor" content="Chen-Ping Fu" />
<meta name="DC.Contributor" content="Chia-Yu Kao" />
<meta name="DC.Contributor" content="Catherine E. Welsh" />
<meta name="DC.Contributor" content="John P. Didion" />
<meta name="DC.Contributor" content="Liran Yadgary" />
<meta name="DC.Contributor" content="Leeanna Hyacinth" />
<meta name="DC.Contributor" content="Martin T. Ferris" />
<meta name="DC.Contributor" content="Timothy A. Bell" />
<meta name="DC.Contributor" content="Darla R. Miller" />
<meta name="DC.Contributor" content="Paola Giusti-Rodriguez" />
<meta name="DC.Contributor" content="Randal J. Nonneman" />
<meta name="DC.Contributor" content="Kevin D. Cook" />
<meta name="DC.Contributor" content="Jason K. Whitmire" />
<meta name="DC.Contributor" content="Lisa E. Gralinski" />
<meta name="DC.Contributor" content="Mark Keller" />
<meta name="DC.Contributor" content="Alan D. Attie" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="DC.Contributor" content="Petko Petkov" />
<meta name="DC.Contributor" content="Patrick F. Sullivan" />
<meta name="DC.Contributor" content="Jennifer R. Brennan" />
<meta name="DC.Contributor" content="Leonard McMillan" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="article:published_time" content="2016-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Mouse Universal Genotyping Array: From Substrains to Subspecies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping microarrays are an important resource for genetic mapping, population genetics, and monitoring of the genetic integrity of laboratory stocks. We have developed the third generation of the Mouse Universal Genotyping Array (MUGA) series, GigaMUGA, a 143,259-probe Illumina Infinium II array for the house mouse (&lt;i&gt;Mus musculus&lt;/i&gt;). The bulk of the content of GigaMUGA is optimized for genetic mapping in the Collaborative Cross and Diversity Outbred populations, and for substrain-level identification of laboratory mice. In addition to 141,090 single nucleotide polymorphism probes, GigaMUGA contains 2006 probes for copy number concentrated in structurally polymorphic regions of the mouse genome. The performance of the array is characterized in a set of 500 high-quality reference samples spanning laboratory inbred strains, recombinant inbred lines, outbred stocks, and wild-caught mice. GigaMUGA is highly informative across a wide range of genetically diverse samples, from laboratory substrains to other &lt;i&gt;Mus&lt;/i&gt; species. In addition to describing the content and performance of the array, we provide detailed probe-level annotation and recommendations for quality control.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/02/01" />
<meta name="citation_mjid" content="ggg;6/2/263" />
<meta name="citation_id" content="6/2/263" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/2/263" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/2/263.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/2/263.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/2/263.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.022087" />
<meta name="citation_pmid" content="26684931" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="263" />
<meta name="citation_lastpage" content="279" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew P. Morgan" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1942-4543" />
<meta name="citation_author" content="Chen-Ping Fu" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Chia-Yu Kao" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Catherine E. Welsh" />
<meta name="citation_author_institution" content="Department of Mathematics and Computer Science, Rhodes College, Memphis, Tennessee 38112" />
<meta name="citation_author" content="John P. Didion" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Liran Yadgary" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Leeanna Hyacinth" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Martin T. Ferris" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Timothy A. Bell" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Darla R. Miller" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Paola Giusti-Rodriguez" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Randal J. Nonneman" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Kevin D. Cook" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Jason K. Whitmire" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Lisa E. Gralinski" />
<meta name="citation_author_institution" content="Department of Epidemiology, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Mark Keller" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Alan D. Attie" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Petko Petkov" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Patrick F. Sullivan" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Psychiatry, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Jennifer R. Brennan" />
<meta name="citation_author_institution" content="Mutant Mouse Resource and Research Center, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Leonard McMillan" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="fernando@med.unc.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging Collaborative Cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. L. Baker;citation_author=S. Kajita;citation_author=M. Walker;citation_author=R. L. Saxl;citation_author=N. Raghupathy;citation_title=PRDM9 drives evolutionary erosion of hotspots in Mus musculus through haplotype-specific initiation of meiotic recombination.;citation_pages=e1004916;citation_volume=11;citation_year=2015;citation_pmid=25568937;citation_doi=10.1371/journal.pgen.1004916" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. A. Beck;citation_author=S. Lloyd;citation_author=M. Hafezparast;citation_author=M. Lennon-Pierce;citation_author=J. T. Eppig;citation_title=Genealogies of mouse inbred strains.;citation_pages=23-25;citation_volume=24;citation_year=2000;citation_issue=1;citation_pmid=10615122;citation_doi=10.1038/71641" />
<meta name="citation_reference" content="citation_journal_title=Mol. Immunol.;citation_author=M. H. Beltrame;citation_author=A. B. Boldt;citation_author=S. J. Catarino;citation_author=H. C. Mendes;citation_author=S. E. Boschmann;citation_title=MBL-associated serine proteases (MASPs) and infectious diseases.;citation_pages=85-100;citation_volume=67;citation_year=2015;citation_pmid=25862418;citation_doi=10.1016/j.molimm.2015.03.245" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. E. Bishop;citation_author=P. Boursot;citation_author=B. Baron;citation_author=F. Bonhomme;citation_author=D. Hatat;citation_title=Most classical Mus musculus domesticus laboratory mouse strains carry a Mus musculus musculus Y chromosome.;citation_pages=70-72;citation_volume=315;citation_year=1985;citation_issue=6014;citation_pmid=2986012;citation_doi=10.1038/315070a0" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Cahan;citation_author=Y. Li;citation_author=M. Izumi;citation_author=T. A. Graubert;citation_title=The impact of copy number variation on local gene expression in mouse hematopoietic stem and progenitor cells.;citation_pages=430-437;citation_volume=41;citation_year=2009;citation_issue=4;citation_pmid=19270704;citation_doi=10.1038/ng.350" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. D. Calaway;citation_author=A. B. Lenarcic;citation_author=J. P. Didion;citation_author=J. R. Wang;citation_author=J. B. Searle;citation_title=Genetic architecture of skewed X inactivation in the laboratory mouse.;citation_pages=e1003853;citation_volume=9;citation_year=2013;citation_pmid=24098153;citation_doi=10.1371/journal.pgen.1003853" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=P. Carbonetto;citation_author=R. Cheng;citation_author=J. P. Gyekis;citation_author=C. C. Parker;citation_author=D. A. Blizard;citation_title=Discovery and refinement of muscle weight QTLs in B6 x D2 advanced intercross mice.;citation_volume=46;citation_year=2014;citation_pmid=24963006;citation_doi=10.1152/physiolgenomics.00055.2014" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. G. Clark;citation_author=M. J. Hubisz;citation_author=C. D. Bustamante;citation_author=S. H. Williamson;citation_author=R. Nielsen;citation_title=Ascertainment bias in studies of human genome-wide polymorphism;citation_pages=1496-1502;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251459;citation_doi=10.1101/gr.4107905" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. J. Crowley;citation_author=V. Zhabotynsky;citation_author=W. Sun;citation_author=S. Huang;citation_author=I. K. Pakatci;citation_title=Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.;citation_pages=353-360;citation_volume=47;citation_year=2015;citation_pmid=25730764;citation_doi=10.1038/ng.3222" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Cutler;citation_author=L. A. Marshall;citation_author=N. Chin;citation_author=H. Baribault;citation_author=P. D. Kassner;citation_title=Significant gene content variation characterizes the genomes of inbred mouse strains.;citation_volume=17;citation_year=2007;citation_pmid=17989247;citation_doi=10.1101/gr.6754607" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Didion;citation_author=H. Yang;citation_author=K. Sheppard;citation_author=C.-P. Fu;citation_author=L. McMillan;citation_title=Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.;citation_pages=34-34;citation_volume=13;citation_year=2012;citation_pmid=22260749;citation_doi=10.1186/1471-2164-13-34" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=Z. Naghashfar;citation_author=D. W. Threadgill;citation_author=H. C. Morse;citation_title=SNP array profiling of mouse cell lines identifies their strains of origin and reveals cross-contamination and widespread aneuploidy.;citation_pages=847;citation_volume=15;citation_year=2014;citation_pmid=25277546;citation_doi=10.1186/1471-2164-15-847" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. P. Didion;citation_author=A. P. Morgan;citation_author=A. M.-F. Clayshulte;citation_author=R. C. McMullan;citation_author=L. Yadgary;citation_title=A multi-megabase copy number gain causes maternal transmission ratio distortion on mouse chromosome 2.;citation_pages=e1004850;citation_volume=11;citation_year=2015;citation_pmid=25679959;citation_doi=10.1371/journal.pgen.1004850" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. M. Egan;citation_author=S. Sridhar;citation_author=M. Wigler;citation_author=I. M. Hall;citation_title=Recurrent DNA copy number variation in the laboratory mouse.;citation_pages=1384-1389;citation_volume=39;citation_year=2007;citation_issue=11;citation_pmid=17965714;citation_doi=10.1038/ng.2007.19" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. A. Frazer;citation_author=E. Eskin;citation_author=H. M. Kang;citation_author=M. A. Bogue;citation_author=D. A. Hinds;citation_title=A sequence-based variation map of 8.27 million SNPs in inbred mouse strains.;citation_pages=1050-1053;citation_volume=448;citation_year=2007;citation_issue=7157;citation_pmid=17660834;citation_doi=10.1038/nature06067" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_issue=9;citation_pmid=25237114;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. N. Henrichsen;citation_author=N. Vinckenbosch;citation_author=S. Zöllner;citation_author=E. Chaignat;citation_author=S. Pradervand;citation_title=Segmental copy number variation shapes tissue transcriptomes.;citation_pages=424-429;citation_volume=41;citation_year=2009;citation_issue=4;citation_pmid=19270705;citation_doi=10.1038/ng.345" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Lindblad-Toh;citation_author=E. Winchester;citation_author=M. J. Daly;citation_author=D. G. Wang;citation_author=J. N. Hirschhorn;citation_title=Large-scale discovery and genotyping of single-nucleotide polymorphisms in the mouse.;citation_pages=381-386;citation_volume=24;citation_year=2000;citation_issue=4;citation_pmid=10742102;citation_doi=10.1038/74215" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Y. Liu;citation_author=A. P. Morgan;citation_author=E. J. Chesler;citation_author=W. Wang;citation_author=G. A. Churchill;citation_title=High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.;citation_volume=197;citation_year=2014;citation_pmid=24578350;citation_doi=10.1534/genetics.114.161653" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_pages=281-286" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=A. P. Morgan;citation_author=C. E. Welsh;citation_title=Informatics resources for the collaborative cross and related mouse populations.;citation_pages=521-539;citation_volume=26;citation_year=2015;citation_pmid=26135136;citation_doi=10.1007/s00335-015-9581-z" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Peiffer;citation_author=J. M. Le;citation_author=F. J. Steemers;citation_author=W. Chang;citation_author=T. Jenniges;citation_title=High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping.;citation_volume=16;citation_year=2006;citation_pmid=16899659;citation_doi=10.1101/gr.5402306" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. R. Quinlan;citation_author=R. A. Clark;citation_author=S. Sokolova;citation_author=M. L. Leibowitz;citation_author=Y. Zhang;citation_title=Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.;citation_volume=20;citation_year=2010;citation_pmid=20308636;citation_doi=10.1101/gr.102970.109" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=A. R. Rogala;citation_author=A. P. Morgan;citation_author=A. M. Christensen;citation_author=T. J. Gooch;citation_author=T. A. Bell;citation_title=The Collaborative Cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.;citation_pages=95-108;citation_volume=25;citation_year=2014;citation_pmid=24487921;citation_doi=10.1007/s00335-013-9499-2" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. She;citation_author=Z. Cheng;citation_author=S. Zöllner;citation_author=D. M. Church;citation_author=E. E. Eichler;citation_title=Mouse segmental duplication and copy number variation.;citation_pages=909-914;citation_volume=40;citation_year=2008;citation_issue=7;citation_pmid=18500340;citation_doi=10.1038/ng.172" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. Shifman;citation_author=J. T. Bell;citation_author=R. R. Copley;citation_author=M. S. Taylor;citation_author=R. W. Williams;citation_title=A high-resolution single nucleotide polymorphism genetic map of the mouse genome.;citation_pages=e395-e395;citation_volume=4;citation_year=2006;citation_issue=12;citation_pmid=17105354;citation_doi=10.1371/journal.pbio.0040395" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. Staaf;citation_author=J. Vallon-Christersson;citation_author=D. Lindgren;citation_author=G. Juliusson;citation_author=R. Rosenquist;citation_title=Normalization of Illumina Infinium whole-genome SNP data improves copy number estimates and allelic intensity ratios.;citation_pages=409-409;citation_volume=9;citation_year=2008;citation_pmid=18831757;citation_doi=10.1186/1471-2105-9-409" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=F. J. Steemers;citation_author=W. Chang;citation_author=G. Lee;citation_author=D. L. Barker;citation_author=R. Shen;citation_title=Whole-genome genotyping with the single-base extension assay.;citation_pages=31-33;citation_volume=3;citation_year=2006;citation_issue=1;citation_pmid=16369550;citation_doi=10.1038/nmeth842" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-Resolution Genetic Mapping Using the Mouse Diversity Outbred Population;citation_pages=437-447;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=J. G. Swallow;citation_author=P. A. Carter;citation_author=T. Garland;citation_title=Artificial selection for increased wheel-running behavior in house mice.;citation_pages=227-237;citation_volume=28;citation_year=1998;citation_issue=3;citation_pmid=9670598;citation_doi=10.1023/A:1021479331779" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A haplotype map of the human genome.;citation_pages=1299-1320;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16255080;citation_doi=10.1038/nature04226" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Wang;citation_author=M. Li;citation_author=D. Hadley;citation_author=R. Liu;citation_author=J. Glessner;citation_title=PennCNV: An integrated hidden markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data.;citation_volume=17;citation_year=2007;citation_pmid=17921354;citation_doi=10.1101/gr.6861907" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. E. Welsh;citation_author=D. R. Miller;citation_author=K. F. Manly;citation_author=J. Wang;citation_author=L. McMillan;citation_title=Status and access to the Collaborative Cross population.;citation_pages=706-712;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22847377;citation_doi=10.1007/s00335-012-9410-6" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Wong;citation_author=S. Bumpstead;citation_author=L. Van Der Weyden;citation_author=L. G. Reinholdt;citation_author=L. G. Wilming;citation_title=Sequencing and characterisation of the FVB/NJ mouse genome.;citation_pages=R72-R72;citation_volume=13;citation_year=2012;citation_issue=8;citation_pmid=22916792;citation_doi=10.1186/gb-2012-13-8-r72" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=T. A. Bell;citation_author=G. A. Churchill;citation_author=F. Pardo-Manuel de Villena;citation_title=On the subspecific origin of the laboratory mouse.;citation_pages=1100-1107;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17660819;citation_doi=10.1038/ng2087" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. N. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. a. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Mouse Universal Genotyping Array: From Substrains to Subspecies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping microarrays are an important resource for genetic mapping, population genetics, and monitoring of the genetic integrity of laboratory stocks. We have developed the third generation of the Mouse Universal Genotyping Array (MUGA) series, GigaMUGA, a 143,259-probe Illumina Infinium II array for the house mouse ( Mus musculus ). The bulk of the content of GigaMUGA is optimized for genetic mapping in the Collaborative Cross and Diversity Outbred populations, and for substrain-level identification of laboratory mice. In addition to 141,090 single nucleotide polymorphism probes, GigaMUGA contains 2006 probes for copy number concentrated in structurally polymorphic regions of the mouse genome. The performance of the array is characterized in a set of 500 high-quality reference samples spanning laboratory inbred strains, recombinant inbred lines, outbred stocks, and wild-caught mice. GigaMUGA is highly informative across a wide range of genetically diverse samples, from laboratory substrains to other Mus species. In addition to describing the content and performance of the array, we provide detailed probe-level annotation and recommendations for quality control." />
<meta name="og-title" property="og:title" content="The Mouse Universal Genotyping Array: From Substrains to Subspecies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/2/263" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping microarrays are an important resource for genetic mapping, population genetics, and monitoring of the genetic integrity of laboratory stocks. We have developed the third generation of the Mouse Universal Genotyping Array (MUGA) series, GigaMUGA, a 143,259-probe Illumina Infinium II array for the house mouse ( Mus musculus ). The bulk of the content of GigaMUGA is optimized for genetic mapping in the Collaborative Cross and Diversity Outbred populations, and for substrain-level identification of laboratory mice. In addition to 141,090 single nucleotide polymorphism probes, GigaMUGA contains 2006 probes for copy number concentrated in structurally polymorphic regions of the mouse genome. The performance of the array is characterized in a set of 500 high-quality reference samples spanning laboratory inbred strains, recombinant inbred lines, outbred stocks, and wild-caught mice. GigaMUGA is highly informative across a wide range of genetically diverse samples, from laboratory substrains to other Mus species. In addition to describing the content and performance of the array, we provide detailed probe-level annotation and recommendations for quality control." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
    <title>The Mouse Universal Genotyping Array: From Substrains to Subspecies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VaSkH6sjnDG1CVw7vQTdzfoWOowrYTnLEphUlGiTV5c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460066","apath":"\/ggg\/6\/2\/263.atom","pisa":"ggg;6\/2\/263","ac":{"\/ggg\/6\/2\/263.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/2\/263.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/2\/263"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/2\/263"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/2\/263":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I0Po\\\\\\\/w2sRbnG8JoSrxQbDKVjvHSPOt5KrVVcVQ0KmbTREQKnxh6Gq6XdGFSM36I2AFmC\\\\\\\/U\\\\\\\/Y1Uo5Nfj\\\\\\\/CFmDb+QZvhwdyu5XIsKEqPDlr6XLlC6iZbAJ+BtrLCjctViygm73mNglaWonDiehGuKpRrbHJ6Bf1vEMYrgLA4Oy7RNYCb6SE\\\\\\\/O95RSldBo+nsIJ8rL2wVmZmIreQgJF0Ha7ibB4XlzhiNFph\\\\\\\/K20szRoOeByh08R8NPobbhLUj+MxLyaT2Fgtu\\\\\\\/ao9aqmTXDFqUyPWMlB0sz+jDdb4DcxBfzQHvO9nnCq+YeuLeqVsbS2IHZnaX9+oUvlusUk42sgVF44X6mk03v76jfoxiKarl2O4ddtPF+xGvlV4VmTraGxZdR5WbpwbfXqk7OJb6Vul1luExMoN4MyRNLZpUqhmirr2jfeXP+EEIAD3bwR3hjv5dYm0WbNOSgA0lkD8FaXjEL\\\\\\\/FvpBAyYkSjhEnAm2sjwdqskxjiC2qln6MMb8lpgx+yJAJKJBw833MfE7iE4F7u+GLk8Ucr9dfQtsgcEXGxqmcxEHfq3DcsMZ+hFJJTWxjEj8lohovrPaqDPMRzKxsOjTpesuTfuqJFFAFyTsyyy\\\\\\\/dFi6RajdrCPclCe6cy+8bfoowDOHRnV04iJ9chh8x2bbBUwMg5iU5G+\\\\\\\/Adpb+itb7udfdDTIYydhoxWckCnQfZ3pyB6kQPBs4ys2nMun+7aanDffv4sEYQgasxQ0pnwPc85qKudARijdOxGurbOJ\\\\\\\/VzzCGsW02\\\\\\\/1hYRA8KvOylT98rcc\\\\\\\/aoTUNzVte+4HMDbszHbSgPQUASa6XMzeIrG0ROJmOQnlPcp2CVmCMZjvM7aPfClvkZcOzP+CpKJlC8nBd09vnmThQtGLfZwPH77bBw9jvv6waygeaztepwesE44R\\\\\\\/26Shl+XwcFkJjUeFoRCvVjePIEgkXJllun5nMcxXvcxW1Ud5hdMcdQHB6El2CgpfRXcERqHtsAXD+aa5jzL1jRdLlAyNicArKBQMUEM6YFnWou1SOBv3z+njLW6K2bH9Yn2p+XRD1LU3uVgXdQ3VweHdMVbwWgnglH3iphpJsl2gQxafYtL+BhD2eH40THOO0iz9GUyUKCcqMedtThdN4NYyHdkde99K6XeLpPXEokYwt1SoRKaeoWG+etmok1zGkQlOJnhK7i+GON+wxfbjQmeLK3N5rpA+axGJqVVWh2Q4jWBAK7KCN+UNJZwHMInjSCuCJYtuLjECfmDKO5J\\\\\\\/JzhqygdEk4UquCBCAqbP\\\u0022,\\\u0022iv\\\u0022:\\\u00228t4E7A\\\\\\\/88P+wwrtB33ugbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eb7a41321818807b7f9264943d36f7d8ef5f03d4deef5f3fd77e1de2ef484e94\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bcZQsEMPK2XUP3r5IPMXAooHQANI6zBf\\\\\\\/mFNwH+ChSgu+kRZjhlDftz1mgijjVnXDcopLN56FfFr8MH\\\\\\\/Z9GncgvHZSQZNJQ9CiyucM2OxCDkkLu\\\\\\\/zEtyyWJT7yiMHZrCovVwOvlcwgRp0sPwCDQGSk22gUNjuA8uGIFLOR1hDOal69j+Q7xgiRfcplXbDRoaP3OCFq4F6aGJVPErOK+j8Su1BJbetshWHe1en7vpGaXg3GxR\\\\\\\/fCMsrILMiE7te3AAMXwForeonC4saKP+UlaQEldCSYEQxUxtC2VXOhaq9dYiGNnWvUgMkWYbMO+JZIw7029zoCRMQ36xIrtJ9KUDb9zEBKRPlBSNsqtm39emBFQC9025Giy\\\\\\\/zeSz0tuJk+7ZLXBSlOsnIoRrJn\\\\\\\/uOB+2wrajeO04eKUw9oCTvnyfA4t14h+KXm8owNwgn23fuHnstX6UMs8vWt073KdLpLhXgYBJh5seobOmivPJaJHUpEXldnTS9tm\\\\\\\/L6kpFt+BAh0P8RxKqQws+L4H1xy5D0AjxCgX53Ps0CqFDh1LkZXA3diVVIuVTKCJ0VMltdjgCNU4qUaMhWtppRdmYXInBugipRw9m+jAatRIdZ8oNoYGo0ucYs+4gmDbO0BsjQb6Oej2TAap9A1Vkug6TGNey+1W5RZFO6yifyxvQQrLuiVHKlMTKCaaoWPEbc7qK647eR9\\\\\\\/qDe+V1L8ItGUggsT2NjO2NFKRuR9anxyTLyDg\\\\\\\/Dp+GMB7wFqhmlFW3u9\\\\\\\/IAWYDhuTj5ofrGFR+4U1ree+yaQtqjmDfCbC2Y7rifkXA2BnDA3biTiPGFDWJPPuzpRvnzD8NZ9bah0CfhZrtYnpiE\\\\\\\/AhgtKf0SVhx72MAhXaiejzY8c6SeGcgp71msTuyRf7AwMdGtgmmuFLiYnoIi02hsk3y4fB\\\\\\\/Gh7n4yyO81KOxg7pe+EM1e+\\\\\\\/rxIJ+esp9f8v76bm+lqTsK3pkc\\\\\\\/JZpWbkI1RP8Hq+MvbUFuZN9gjVyHafGmlL8lc1IIqpCD3YHTHJZ3QMIKYuZt33hGEXj\\\\\\\/W9ebXxRuhHEm282ebW6\\\\\\\/UBVnLvgs9CxNDHPUVvLLeBwnu8TEs+PTW7M5qiR+1uLsthatwwZ2bRSmInw77RLjfhce1MzUMV\\\\\\\/HE8kVoBUTXM2s2+XmCj\\\\\\\/hReoDtJmoe5yIof9pvP3bPgAOFtTvw\\\\\\\/YT7f8tFDA6Pw+xE12ueCcCaxyqvdThPhEZ+Fh5+5Eoba8ZABvGt3DvR7mrnwp26lhhr3m6yB1cgDfntydAuajUxTER3u71zCLQ2tTwfFZngG+Ib62fduoZD8s3t0\\\\\\\/P7M+tDOug\\\\\\\/fGyjneXSsM\\\\\\\/4SL3woLkx1Wa55GqX05xncf6hgdDferD3hkMRISDNAhe12X2n9BIqWLbPel6+RSgGABxIT\\\\\\\/jsQzmKbN\\\\\\\/k6n7qvQKx\\\\\\\/wMyq9Y5\\\\\\\/n8LYAi3YWgT7E3rz0tAm1rb6sRmfyhUliSL6YfbqtOWosdP9+wVSKD2n5S1BHIulyAkEI0fJfDQVJTNcl7iUQvB2gKQca6le01CzWsduXbdoySq8KTCmoVayG5z95dQ+355jmfNDw0R9ekYabjs5rdcjC52+5sc9Q84+rCpQ\\\\\\\/8nq\\\\\\\/Yk5h4vOx6TbA0ZySAUtyykNn\\\\\\\/Jq1nJVvMjErBjmLmEnxQYSWgty2wtJpTOqafTLmw+3E6lOSj+eGZHk1qMtK+uQwNCfjvqEKIFRrCocrm\\\\\\\/3jPwmlRCZN614bGtzBb3pG0tYW35WaB\\\\\\\/BvXDN9OTbynxBC3PWkwX+mKuBSglhs2QR\\\\\\\/RVfwWYulNlCfBDZVtL7nqGC3MXLWXpZTyEtOVAk8rOP9UvKE9qCzQ1etojKfQn1hgR8L+\\\\\\\/8I3WoWPxbxeThPLUDCRamjC4WTSBfWVdPkKaXnUdYOiWbBD5e2fG1alVVHZGAanwiWPBPoq238namIslLH3wTSNYQv4vTIkFaKs2sp0EXXSzFL\\\\\\\/xpi5KJbgDzI5EOltIIDAS1eQjnefvDRs8EM9BuBOIayflXhOMJPqaVgoM9\\\\\\\/sFks4ly3OhYg8lx2ca8RqeGbN6Atj+g0YMQtphnnBg2ilqK86m5YtJxZgEHKRgam8mC5neKry9RjUtGrpochUh5WmPVJqeprSOJU\\\\\\\/5KIQYmvUzvkf3B3Lf1gUGFwX+\\\\\\\/jxZSyRyOXjLTNTHRCMO7sNqA2xKcqAWrBVWj\\\\\\\/aeXGiS+GagndNR7tCX9vbtV8O73OItSRt+kKTZhxLBi3Ro8ci3Eqf+owxkwOoCB7Tx6CJcroLlIl0pMAjHGIRLhonz4OlSxATzjRJujgsxJMJOqQkxjzIhLO5FPMEWGGbOzYFdzfbcnvj1e\\\\\\\/8nlzgLO345fj06Kcrorb12e2IUuWNQE1RYOTpnJstbekv8Jyf4l9dGnSPhRV58Pvq00HcvAKZ+t18zUwdKFu3qO939BOAPpKeRfZqdTgkmFBf3s1Nab8VvgpY95zYc\\\\\\\/L0uQA7cj4Zn8ryqIj4zF2fgTjE91SLShZlPnW5hQRec7VooU9hDMK+l\\\\\\\/GRmIyXTKg1awJHh\\\\\\\/oA0EYA\\\\\\\/xjLcaP4FbMSXXzmxA8GnDRTZI4T5nDG07jLxph9rUA58rsFlbJ0LuvHi95mZSEdJTm\\\\\\\/2TrU0avQLCGmNItJ+2lC1TqW2EFV1NyZXlKL9tW1ucnuem\\\\\\\/fz3Glui6VXTzSH0jWBGxsjIUCmgyKNHzWV9TDLiuHpgmCBRSGgBORpzeYFkMfrEYJQCVOao1xHSsP1c6DGkXO9uZ8C1aA20iLRcawm+ZOPgpPYw7i6nMOdtVbOjRwhsgd1BWkAt3av4mTxs69fnYSZHc1rbm6RITWcoDjD2HVucjUwOh7hpEZsYvsexbj1YKPbJUQfdh804Hiu7uKJYSOFvBQNnHzDRJ0pxmvHFczfIocBbrkfVKx7AkAB0FQzoF9n5nFFPTlm\\\\\\\/ayloIagLXeQrco9k1ZxJZX7FnwCU5wuB1riJXDa\\\\\\\/X0A5WyNHMUeDw9n+Z1Yzu+fuQ4KYGhk9Ie8cwODF6VPf+TgcrhALt3gTSHHXi2Y+Mh+Y2naLavfo1M4bU1vY4LOoHOtGHrKSte11LDyBghoNTpGrBjdDgYw1sJIf2ZP6TR6Tg59cfNWga8mIpZln4uOwrIMtvpyRuoGhLhz4xyag4HMsoP3Gws3HcsuRgGXWwDAMQDszbZsqopASm0aj8tP5piztWEf1TlMEJJdMOY3jsGoCu+blPYoVDxga8c02LnuDfn7y5wcBov+0Du9D0WQl\\\\\\\/upAWq3X4XMtQqmNRZqjudbRmNwua8z4ub6EBBHqlXUPaVALnHRoc2\\\\\\\/yygBQEsgGfnR4Os2gle7jrCoyOuDPePXHFgbxN5mLN9LYgM+OL\\\\\\\/07X\\\\\\\/okqLAopK0+tPRQ9+6GnkMWkYGVus3gerLZWb5sYexV4X3NtVOv7ShR7GqjU4SJHdoEUcAllYAjoZ\\\\\\\/65o+z3qMuYA\\\\\\\/vIUH7tnvnw3WiBA0YQ4k5111KarOSKdBcPHCfawG+aF5AWNEcxHbVIR0VaI0kXc0HdAj3Ex16GvCBDpCYo\\\\\\\/JzNFOmQPYxIzB9uSbJxnfRtFbuoSQhkYBOV0vbST89yhc1V6dhHbI3yKWK8MuUPqIvpenwKTBtQ8Y27hELzKX8dlp2BHxVxlXAZThYiO81SgTfMYSNJlyyllsctffr1jiejWyz8Am+OucxAe5tPcWy1mUbcZnF93FRwulASUub+aNOD4tFgLRESZSe1jVl3TCGd81tm5sbYubM732cTngM2G3EwLoKMjqdoGjCGfwmHezmE+FvPS1tEWLH5Os5zFvkSuRbNLgTNF4PtpXSMEz\\\\\\\/NZUb220OntYqHvns70uJAhB3m\\\\\\\/c6reK0sQHXdDBImwM+FSM++ofD9jobzwPNO7LurOF5ojW4374tWBGZz\\\\\\\/H4LO3pZ3X0Y0\\\\\\\/U0tL66hcamsXfQyb7HIgDfd1p6D5WGalPpd1A2XqUzyfXHdZhq2YaRVP3I0RLjB7NAUoFaUl25\\\\\\\/rHQmhNq6DrpevJIVJVCqH0v\\\\\\\/4HnsbYFNXNgV0A2TA1sg5Gr\\\\\\\/wqhQMZF0A0DuZQ7LW8fpy8SLuBqt5mR9zKTDPmAVR3ThoBpT7veyj8opFDklmJZ4ar2vvFXUkwOH30r8rtBXTeon7HqJDPANA4OE8IScb41bWCesOE+rC4QWKk3cu204x4klfVZWl2TeH2y98cE4REeU+hN6LhL+6mRII0m\\\\\\\/XqX4wubeaDv4EUOh5x6MDa+q0DGKk2UsKOmDXm0sukKIs5Rm0pcU\\\\\\\/GVJwOOT4u1F+tAoGiznxfFSJaF1y94LEdpniIeWzsytvMKUDSJt9qNfp0wc9iFtPQ\\\\\\\/F+JYaWwd0sUywuaHg4k4wdBiG2t6pRMedyjJORIgQ2NB1sf\\\\\\\/GHCFJ0Df05nQjr+gh3CU0kfwNSpuvtyMhize01Gf5NUvvII7g2O7eAYidbDmJ8pW01Ui89oEzXv2EEp4qtGzUFcLMwNDcFbSvq74TCX+NqqfQQ69iQA5VSO7ZZ3Pw\\\\\\\/2O2vJS4a5voschffVVW+pXikk1\\\\\\\/ub0jYk9JAKKAp1CcGGVQDhDq+6dnIUmGBLHVYpNvJKtdBsnOXYO27aJxaluOlEMO1Hy9tZ\\\\\\\/zP+X7qsfktZYngw6LzHiZ0xsQarpQCwWG7dEHL9SEhDRSwUCejeXcqfbpag3WmDVOqMeEiT4VSG2D8ijBLlonC7uOmfHG0\\\\\\\/vNJ+4R15SJvXOZ4qX8nAMBvNnDJBXwO3V6hOemSVYbf0zqtDy8dso5a2WXZBiLwY98ol8g3p9YE\\\\\\\/s+GdL0l5U6ZeGkzj3aN2u\\\\\\\/+HCC0O06AQI+4qzBNY+MJkxTAikPsTscFOItrumt7P86ssjtRxkYKt5fnoPPcqw4F3OcccPag2\\\\\\\/1CA6vHtVumPvbQN8F7jwHdfwOWD0OO26szfpbj5\\\\\\\/y9yL1UkIlGUxVPIOqJE+pD\\\\\\\/yu\\\\\\\/mIBVrk218YAV8bBgweyHcLEWCfnRSyD1YkCgLTDCftjldQzhy8J7gRm7LjclEXt+hcZvm9PLxvOgqOqQYZ\\\\\\\/Q4t8\\\\\\\/PRsrNPG112+ct4TohLnjww18ZvY9JIKutpAN5ecJI0PwGpg\\\\\\\/RKT9VVbtRJd+pfhTVYTNYTQLnbfvFjInfta2vk7cWepBzD4T833dmmrisyPeRHXz0\\\\\\\/ZudK0L6Iz7l7a4ihMuN3XRBBdfYmVDZIMA0KG2Roq39FPrTjD6bJ\\\\\\\/RY1vWdFZXnWUlkvrJB0kueDtKsiLZlDXnRA\\\\\\\/CsfrJTjOO2GfA5ck4btLNEaCYOmXmRaRHRdl4H+uEG6iFmMRfGAbnwsjAiDcZmzDtuClFDNKCLQRdFr8qKQT3dohRJy0UpLnu+NMnlzquTi\\\\\\\/1rK4TPtzFPujm3AddSzPlu8sx7Jv1zXv5787jUHLtmeDw3yQ\\\\\\\/Lfjmc+421q2A52S4Gl3reMN9u+q0H9ca4HzM1CC64z1Zv\\\\\\\/f6ltl8Fxj0JKI3X2SHTkG7TN21MdsZVH4waAmShdVRpIw5IT37\\\\\\\/YO2tClc3t6KgGk9Sl1Gdp7Kb2DNjihys9KLh3SbLScB2jn9pewFfkPSM5THYucsxLosixW5vbokLxcZ+PN9ZyzuImPt0ujLU7C1KxvoA3vvInYbRIhEQckBbGYSSwItRd901GWXv5HhkjV6MDfkMswWBwSP97p+Xut9SRCgFTDeoZiFE2KZFML6MtCXTPyislizDbrS7CWe4lIgqV6Bat1fmG\\\\\\\/8UI+2p8yHR2JCie6F7AJ98gyfMzTkgwzXeIiBg2dQPbhR79e4rejGwY7hGvUPPQl8Fpq2ej\\\\\\\/JZn4SZGnyY630h\\\\\\\/e2Ix+rvhljYCfzxl3yTgdXrNxLFaruch\\\\\\\/xcqQcskEsH1Gh1ZzQkg2kQtPlKLHIkCYcuns\\\\\\\/+Jj5QnKe34j1JOxuL\\\\\\\/idttqoSo1qC3ZBfvoP3tlHwf3skYJuuxU8\\\\\\\/mgEEuRyYkyqAcDOeIYNyLEXNQhpgNjoxCQzcGlhK0HBxQ1CvluZ+QyjcSym5NH4dRyxoQdbC9ncJnt574ABa4+xo+Mk13miDnH7+Sbh8Gqi77uSFZaGfYOYKQIGdrMKbEP\\\\\\\/JI8gi1BTBEhgCdtiDPZjbVVq9h+UxX+ofOGZNYtmEQBrUwnPB8IsXVrwNTdoFWFiv0UgWGLOSWBYRGbSBgnkzbOLQmtpLJsoRlRbq3Ra2JN2nI4THT1jotL2SYSjlX+3b+6zDzsEFqcygROira\\\\\\\/5auld1J6CMMZYVh8PPJX2VPmGVpAq88sUV9A1SSxnjnrXHholqAFO\\\\\\\/rA4I4\\\\\\\/NZ8CZUveTDL2RTOcLBt0yxJM+KUtQnj6gQGtGFypGFx1yA2kp3\\\\\\\/YLFZJU9qnLgNDu0BG9+DxxxfZwOB9OQOlOO3nA9VGL6wBxqbfG7cfQgkkyUAzMehKOvzSBWkYt5sIBS\\\\\\\/c3422R9G4j4PuQ66NxtJNRkAk4Sx3Iofe5EjKCIlbq+uDXHgfiBfIegI7tUZ9nWj\\\\\\\/kedPJxb25AspMmmkNxxH0fd2a5xERXnAPmmRkuAjDcrVOfDqwscYWKQNM8nBFRH08Hf+Ap1jYUruPCwG95mruhdq51NlSQsHIO7itFY4PssBTB8GKpLoEUh3mXg160M\\\\\\\/Pk9FGQ\\\\\\\/EwmAFlfdEpK9\\\\\\\/cWv4bF9+D\\\\\\\/6y7bJeCHQX4U9cJzAQcSp5CgrjEP5men5bc7HuFyeYz+XKtmbjTjSpEyrs1K2HHmh4OhU7Kq6eiR0oSE2v2AWcsD\\\\\\\/xfOeEwggylIXV\\\\\\\/yyzpe6Iodce1tAmD9lZVXF1jgwAB2L0bdzjXPk1YrcDzPmGJGsyMaFf17dLnnVfKvjTIXOoNFmOs+PqZppDgG4aUhEyLU4+YZLf1\\\\\\\/paG1ur5DQX5rlFrDK+fwAOMfD08QMVUGQTH513DnkU9Q7ggWevmBH4XSZdbJ3HZTigJT85tggwUesYizUGmbq8iPSEHxjS4REcotlPsoMQT68hyMfhnfJ8BmNnISGENWe\\\\\\\/j2QHJdlv+t6CsqOU1Hbar5k5QiBWHcOuc9J87D\\\\\\\/myrWna6MpyCxqxAv8AXDTP6DeDe7MqP2qdSYBDSCMGOA4xx1j5OVRdfPOCHh3ebk3dpzOH8lc7KNqV\\\\\\\/NKgMqlqjzJ4TGLTY7CvCRYhzbWgX6SDyX3IUDMyomtuYounRmecp4kpytO0+7nSuDLr4Tuq3j2KK3WqLHl9FmETM0\\\\\\\/b4MxYULkASwySheF1V4GklqVE1qJsE\\\\\\\/CfZhfrocqDv+Zb9bKr6tSA3rGldOyxOwEjoG4OesJAI09H5AlE9yPcOuIF43j2m+tTXrFk2tQly\\\\\\\/SJtB0sBoRqv\\\\\\\/ZZPvqiaw4nb0z09bOdVkqakOVux4Myeti45L6htT73eMKkdQFiut7vX35aL2xyoXboahCfjtAyWrNxErdcM6xfsju6IayMzWo5LScArm1ww75JcU4iGvPKQcWgDuviflk5lvI8g4rtNfp26QCPe\\\\\\\/oJ1RzhfXDMDyRbTdMu\\\\\\\/WrKr7wxFACLGfFkxoIjQSUsSP8aILYYR+qgAJPIlcbjedm7RZdXdApedPAQpEk0sPrcbyKUUJOwVkkB470LL5Up\\\\\\\/u6jlQCEzaOBuWkke4ITTfoGwNRHX3ranSHd\\\\\\\/Kvtv5sPNqQgQbQaSdo\\\\\\\/g7cYIGtWRnNfovElOaPp8cSm8cUIhg+AM71AeKAE3zA\\\\\\\/pqoYORFrWWg\\\\\\\/iEjh8WazIE1MMHcOqRoEiTlprse0Fi4Dd+nERF9vvumlQ2NX6KKIM3tid6EaMAaP9tlTae5eSte7puwTRCJA1JyL8vTiZ6iZDL0x8FVNGTIVmACscWvbhm1hI9vXgYY2xM18KsCn6IXG4tnYzqhU3054IDLSLShNUd5rf9LMCBFSf8aY2zPWEDFObCmxRSroGkfhlxjY6Yp0HiH3tLqe\\\\\\\/bDdyXzxNJFWO0CGCfHRjvfwg4SUm2J4YChy6a5rP8tTCSN67GOACZPdUSSyPX2clWxg7Ag7PvSEYa5XV+NGXfkMY1lyvhQBiKWIZuEhnspvg5nxIJxWH3RCFoh\\\\\\\/DDPoyPziGB3wV8mxsZn3rq\\\\\\\/IhV6nDCe4Uix7ggu1P5b071eWU3yhjJXjXSk4MO1aekqU1hGxHzGaSU54D\\\\\\\/H25581YROdrA6KrdfUwLy5NnwNF3qb1DRxu2Wa8Vt43Ro0zzKpmXWY4fLxdiepyAJKXjbVdFR1pXddm6twebfKQMEojGiMJHu4dhOaCjlOaJVFS+6IpuBABFp5X3Qlc8FwUAmMNUE9fEGy1KCIWKy71MrKWvx8i7rLc+jPNnf0cw7rCn44z5DRsAmCWvHQe8U2Nw5J\\\\\\\/fzj6t8ltGb3A4yz7AnW1CI32tRedA+T56Si+\\\\\\\/YZmloY3VcdPooOOzlm3B+XJ6\\\\\\\/KmuG33n48PSBLH2aL7UK1ObRlyEQxmdmRWyMsAOzWblWim09Rf5PxcPNSLq1hQl+Y7iV5z4HuWjouS\\\\\\\/Z3leQmSLSJfTY\\\\\\\/JLkfrI6c2QtsdkUIH4XMbO3Ur573ktSQZ+n7mDKWgvSbf+yiBnEs5Zhz0sGAKLhlUVtlU+DYggwOLsYk9iIT6+1MMTYtehaIpiRaaZ\\\\\\\/z5l+4aweOHatNB+3WisAVZIAaYc7+NKEnKp\\\\\\\/5uB1S\\\\\\\/a1N7RhfgO7Y0+Cw1ImL542N076iEPG01SU6jBKUQbC1ZLERFvlmUTNSeYeRo25VbXRuXW8VgT0hQ4E8dAgoAFNP8DnfsYPUs5LHZSqNENNPSky17+4sNbqdcgPttFCFw\\\\\\\/Tqkzi4vxV9xJmRN1wfsKKN9DJWg+xW7D2uvf4kWOwzihBPuj73fvPzU1j+wZW4R8PCt7m9pTl\\\\\\\/DhGFAeEO6PNt133txxHTYZKmlVEbuuetM\\\\\\\/7X9FS6qf7nfegAaVgUIgFNIgAjze\\\\\\\/z9iPCNyqOJNc2fLaG\\\\\\\/jOdksuDX4dtPKniEoyERL6eFocorGsl0kAHfXiIZfCDaNMzGjo3lkdBX\\\\\\\/BZokCX4mdKR61DkLqukPpMohGBvGlKyPoXzkUzrvikXsdV+kAB9i\\\\\\\/14HyqZl8jtZN2R0clHGFZaBWSYTObL\\\\\\\/wKuXVXtMqOfsT81wG7tA9QlV8XQZNpFq8AwKbvL74cSTmE+eDdIolgNgOdrJaGQzc6KRaeelqKhBlzDJHLe9eekA7\\\\\\\/K+UzN67OiIyChzCxQVXUu34GSbPaebWOtJUBosEv7RpYw6tx7Xt5\\\\\\\/tw7B5uDf2V9ZXhfKo3hkhvr4J9IPXYpiyBK9usfNa3IK+S4caDmt11qqACctA\\\\\\\/7jwITJ1LiCLWuqwa4sueFMwMSKG2ZPNQ8LKE8upiT+SSyArXjI1mrh77NrN\\\\\\\/mPfvuSWJBbkw3B2xbudQkukq\\\\\\\/Zou0G8SEgzNqIrylSWYCg1IykEHOOQfzR4qmUDnKH4HE+yfk8kqk9st94U2oZURsD42\\\\\\\/LvHPHzQI+ebuIY142pY3SZvO6RLnvPFxl+d4BccVS83luYugfsiX+J1JqAOH0yCdFk20H8s9Lit87ohauvYade2rMqX5dkAGf8Z8Vu3zr9pK7+Fg4+iz0tr0jvcSiGb\\\\\\\/tiD8mRWLOxzCBWiQUwO97J2R3nbWAmGIfwZRF2KS6o0uDzVpkeD+UHnC0aaXxKegEayU1JFToFuQkD4wzL4L95k0tTvDt+iCeLNWZSrDHY7ndTej2b3QLcYZQU4wr0ZK66sjLFTlGYz5LBoPKstEl8hF19JPry4+K52zlMLS3FL\\\\\\\/o8t4FrM6f4BHjtHZjzmCy\\\\\\\/kVnq8\\\\\\\/EerFP72oMlcrPU9snPGcNLw+sUe85D+8\\\\\\\/YkFQmxomdugpYYqsj\\\\\\\/uLjkMKNCkaRAzX1HPc2o+\\\\\\\/2hkhjGmEYS8WI2SRXZAy88u0L7wkxpPrk9GjMU2rk5Bi4UoalGy7lsh17HsiNp\\\\\\\/bH8w6nJmpz0Z4vpoGQWhcEaggmtv3clfFs1TGV2w9nOJPeI2xzsakHpXJGW3kQCGC6InRl6YdjtA7yN7FmB8MB7aPYrS7QsMF5aeD7dtQcfw7DHzP+X68K6P9ZVN2n8FP3WrDJ3+6QsiCu8H9K0RETcPj2sTFuhF7D0kC7cIQSzxDgtiod0Iaq5NGkI+jocl+9WEwsDPHq0Y8pZOckSeZjENYMaBG61CFH1jkkrrW0Y+AFJ17SQfiztI3oYRt7pNPOTV1c\\\\\\\/JfFRAjSU0jEQgDe9k6x1wi\\\\\\\/dHn4paKJLEvwIaoWBEdcUMjGGXEErM+BIEZ4TZeMUEyXUkXjq7gPIF0q34ebeG0+P5dcHbqxHstr7kEveH5NUlszqDWi\\\\\\\/g4n2pbQ+Ra7hu9GlQIQ3D5KP4jsj91hSNiDeFYNvBIb4T1O2PFQZnkEPAdTpoFuv7rpplPR\\\\\\\/qiz8d7qA6V0vfI2CA2W7EzT8S5yWNW5JaxoLOEWUggtjTesUhu\\\\\\\/mZeq0J7bUWEk\\\\\\\/bHamFELZ9T5M32o5\\\\\\\/2yuXyhxpij4YRwQBK6+nwbs6bxqYbmR1+OEjNK1ieU2iU8zN3OoO\\\\\\\/R\\\\\\\/jGjykk1gx3IHSDmAE0RFUfRiB6z2BPt7ArIM9c5d3\\\\\\\/+FhVjnMfn3Opo2nC2OIXMlmXlhbOotm08S0NQyNZ2zKLv3oPu7BZ7s\\\\\\\/OhlDjRuM7mdk+2PIeUYnlxeJv466HO+jy0mJyhyIN1Ya6XFByREseMCMwk7aq8YoOVxo4vY+ZaBZyyEB53DfwjIwPSbZsWeIPKqydF39q\\\\\\\/fBxJp041szrmdi9SG2bQ0EuRc6cpNvoi2fvcfU3R0vYbOVsgLo4PSEuXY3DeuQHS79iICV872RMiEmyJEGQOhvBjcKaE36044P\\\\\\\/donm2tSTup8\\\\\\\/aEvG7iKmdIKZR2gFHD8sVx\\\\\\\/2zC8dSSTUDkEbo3cnVpO9jzuIYCjpCwOChwPSzKrarYeDtMhPdMmhpCAnR5rFIEMU7DyDRI\\\\\\\/cG4YoYrQlvgfwXcTj8Bgw4VWNMvnXKpHO\\\\\\\/tlUKhwqJqAjU3PUdb\\\\\\\/B6i\\\\\\\/WVuYJTxuLaRXhTzoTMLnc1rjCpvC7vsz4HXxfJyvAx67woyoiDpSyLNuM8x1Ea8Ar8fXl1jolz4Jx5yZqwfZ1Le9\\\\\\\/ZAJq\\\\\\\/3dBZlaaeU2HUoLezsFZhqgwvAMIJ2rF7QNN8Zpj72txazFsBYrrFwgjK67vKelxo1h9GgSzVfdsJ30l7yW0tkpsk8l+aHBB5DtfytaaUsqIml1hhEq0C7g3mb4\\\\\\\/JaZvnrrOYQCg8j89bUsYVN0+ORsh75B8pL2q9WwYRcO6IreEPR9da5bFID4lJGWutgsQeVRuR63tuuOPx8hmfhCrjQKwf\\\\\\\/jpn0J9Xn1\\\\\\\/KrN8tJXE4jGkw6toqBJNB6oUw6KskCFKqK7JXZYaCzak2yE4gBOy+o9GqIGAVfiCt9FD1X8fgU\\\\\\\/jYaI8RayfVD5+GxeUnU+p4AAh9vli0uYw4V5mrmp4otbhaOriI0ceWUpZx1OQUg9fLHEKdYOh2UjMcFiy6Yd0z5ENlwQXOQ56\\\\\\\/MQDaPXOklobZ\\\\\\\/hWPidAubOAbcJmd\\\\\\\/IabaE4MtlkWyq2nEFXv3JYaOs28kM7Fa57\\\\\\\/Xpe0EHWT2SilQ2cZlQbAb70NCzcl\\\\\\\/d6q4IcqXyDxekLORkdpOMlilYb6mopmsEAwT9vzQxj003shG4pC85p7BPm477ZfMwfZQYD5hLRnkA\\\\\\\/38CmO8v32Dg6n\\\\\\\/z7fYnwu00GM962XFDd1XadrcpmciSYsHUUWaQkS8PUuPNvVJWIUMssvyrDHdIudZbxJgfkK\\\\\\\/rRYKDRuS+bv4XUAYAuvV9vQK+IfRS+l14FbX4LtjuTu6EvvHl5J2663tSXGujWa\\\\\\\/3zpCnUMk7m\\\\\\\/7vAqtI5KgMYMHdGd1e7kytJoBURRoQq3ceEmGxt2HKXmHcyrgxEJOneR8ksaEDpTmzgoxPfRI4eJP6PKHhb4oaXBTkEluLXssHk0Ycwhrqa5yf7Ip5Ysc2myS3GCPbJXD6qT5fxIj+Via5U0VWZ1JKt0aur17u47WvoArP61+hUD24btmyxKW+NPhDiC\\\\\\\/\\\\\\\/ZAdedJ3w+nt9ZuzCQKkkqfxS\\\\\\\/hHtG5p6W9Aw4nxKs8O7LdeqIiYnm3Eag6Rguzp0fXFB\\\\\\\/Wct05NnZkYZauh7uUdaiVIVC0JMclqjo12AQBmuRD2DnCkvPHTW4jLZOX6vm7Gw2U5iuvvt6mQNvE8exUxfKdwPIDh2J+Mba+WPiPJOtEYGd1VDh0dzX340blX+e7TTYbL04R+rOWuQu4RQ8iE0ILXKC\\\\\\\/fseLDMW+csGT4ldRW1nihcJq7zLtGtFzmBDnj0w6GFu1IFqDDkrkZaQzx4Evml3vPOReWXXcikswqeZ2yDMI29ODfTmheizvokbdnONVLOj7a\\\\\\\/LYKbjSI9beq8KxkN4E8CDEck7RrnbfS27PFtTJnVtBnGYuImgkx+fK1PBrXzd13bi7I3NcphpJu17gvYj+yA69n9fhs9PcjKKaTanD9XCR+sAxIzc8WDjbkB\\\\\\\/WCK5AKPvOeQKm3x+tp4MhFLQw0xulwUbu+IpRYfkGfXezxRk78Hu95cVLlHZCsX3OQmjxy0vxm8bA8H\\\\\\\/9gcr\\\\\\\/EIASpR6wMix33qd3JLsH3t5VoivKYyPm3Wf2cNm+LFY+Y0w7solXnCEqDZUBMC3FxDM5LMl+k2PwxY0jqOogfNZxcvpmamU3bFOZI8+RKEUCBH\\\\\\\/5RThmjJF27ZhmfjBrS13BAU6C0\\\\\\\/r852RJFxUH6Kk4DwRZERZr\\\\\\\/rGTmdzWVaiakAmkBq2gaXXJ1HJpRPXalt4YtEX+9oZL8kAfEBe3O+9QA3+gHdy5rBePlhMRy1NUwgEdVTIS5J\\\\\\\/wLzOk14fZ9jY+Zcp6J8MW5LJcPlFNP7YycjnrvQitOEeoZr\\\\\\\/KY5Cov7uLWcvY1SvJVIzGmT9sDGzvBzfSQTxqQV4Ce2rUSRzZeYQbKbuDIKQxkWHOEX3R76FjWEgniXAx6PpY+xlsGcTJvZ3I\\\\\\\/7gFSUvG8h2pAQF107715+ievHR+5VSBhCuGFD8WBL5wfMIVXKx7OXRLPjcO+FL7cDw9P0pyZWuBRcDFQmbTwF7agU4ZFzv5lL2F9KeiaObNt+EfjR1vR0PFfwhvkTsmM8w\\\\\\\/4Ya6I0Pyqcw0sNYRTEQ7GEdTJwbBouRZQh\\\\\\\/4X2zLCLwaJT6vH5RstgaJ3in4l\\\\\\\/XoOtbggBlgw7vc7RxZhiqQAQZddqxtYiD89tPPr9DS7cCtGck\\\\\\\/8\\\\\\\/P6pV00J0BeY0aZlMTPtE3nuJ\\\\\\\/44z4XSc9fe1nTumSnQxdmktHnec\\\\\\\/Ai66FDvrzso98uve5LYRxocBSXT+BhQ2JuHzKmiMJrb7aLIJbmzIv161o33EJQy9TS3B1qIggg4bQfgrP2fgJ4QeasGOH20EG9WUiHK06PRxTEiZkEjGpsJBwO5YvxxYIQJmf0A52ErIoPhU16tZlz\\\\\\\/GC\\\\\\\/Cxd7TPbK+3AsvSxdsFoxWZD6RMVkkRS5S3wAj4KG\\\\\\\/lOkctSp5oUhkD6EaNf31Pr9Re6M\\\\\\\/ZjTwZYAKmLLtw6JmHe95KJSOb3xrTiKOOhztdIbsl9gWMiWyf+RjmR4oFGJLbIivP68sxIEKmmfDzNBEWGEPZmMajOAyWUCUUSpqRGpVYjvqyPf6Sahh0YUJt0KNNiICdoeQahgmzfp0oK+48y9StFunriegS73CWiNDV6CjXoy1aq7lNBf2y9638NN\\\\\\\/fwWN1QtEEEIMEQt7W4BIT139R8jFr51j1JIxa+icr0gYcp+mwglXxwJ39eIYJ3amBXkUwK5TuWhA2gmahidz36ISOa7GGlgu27EBzw15RDLMAKmTbJAt23eG3oLaC+q8\\\\\\\/BGatCusa179aIb36VFVl8o\\\\\\\/LCdoccrkqdHGgotzPJv4tp9BdASme\\\\\\\/dqAiZHhJL\\\\\\\/6JuxG5PhQ2iTjBdSQ72OotCQVdCUO53KLma2zKhQWjuAEvZfGcgmgWSXQjsBnabWQ3dMF\\\\\\\/W6MHAKbgQywiqddMCpKMEAeS2QR+s1QMRcQpeppsRuoJT3KrnvgpkK+HPZSPLAvafy5AabTpgS3cJRbsnmFEL7Wa69LkrvBQat1z9izd1UbdhgnIPPH1winYI8ffliqqegcKYCd1X4\\\\\\\/33wwQgV5xclaXkxvTCWAnkKfjiNHuSKIWzZS7SlzFDsP0uJPs1Iyvfophq9n087VlqG8HXBiTM7FhWHp3RgdSnNweurS1rYvsecGauXciIbx1Kdt5Qy\\\\\\\/4ZNhcuNI19dwZ6U3QsGWisII3NFPBLMV7fcJFUWz4bit1nlpdPBSAAvVJGc1+DahEpCBakwvgw4IZ\\\\\\\/XH82o2UPaNVElN+r6U7CKiy1urfMqURaUbOIUGSo19DPEHvgvzJlIeAlqIAkvQnmOFbMd46GQiPWnRcEDPb6U4dffeS6ny8bdVcYj+iika8Eq2aFCacAxUbugzvHVr4QWoAfLVa0GiHWlzd3C7rbWcb6XsnIPZOjUb2dP5K1SS0ZOx5FD41nnUiL4QejT58rU8yweqeLI0i1J+8wUGPBqqlwy0fNGPdxPa+619iwMtkOUku46RHspfxiI8gcmJ9cAaVrah7O\\\\\\\/hAGZLiHihii6HeTqE\\\\\\\/ygagOZtBdB6zgKyENxZrw148ALfa1WKURyxzYqYkun0N0Ow3tdW4onJZrSpTzOVPrvq5xqeZoZobjfV02wbgiE\\\\\\\/szf\\\\\\\/kjeOQfEJqYDS+Zp1n5iRMfGuzt6qZ4pbIyrJVWCVxWnLdfpNEVoZjujS0oU36P6G8pEVxr75VMFE4DPyInPItI841ONhw0mGlSRW2EcetvW2VCe1Rta5rcHoWiH3lSrYz+yFnWGb1ht3jTF1CJEb1K514eudCyvppfE+qMmiVckbsb556jWBqmwLsgK7wGzXWMH270Sp+AtjKXhSwpDqpTaeUTZMlb6jecjsZexM2oG0eza2eaUBEfsv2vLdADjq1A\\\\\\\/Z8EzaCpWCbXPSvpmVMUD\\\\\\\/bT6hnV6HuDDo+ZoLa30ogGkG9tGguaN\\\\\\\/TRG58TSoJb9q5XSWs2aP3pAUnms2Ld7E1onC3RweyiXWJzg0m2wzcUL8wj0OD\\\\\\\/lyRPWNDrrxuxZSe4AbQW+fiHdvzjlOSKcmF7zsyGbaghIVV3kf8jMfurn3F\\\\\\\/\\\\\\\/ItlpgxVWTbQQ8teo5fYNwTZF3YYEkEcd0NGHcuK4YcBy7gzEzfkscxX\\\\\\\/vxKZ2x8dmF4+bGNqaHRwQ+E8qWfsdsyWeft0LMxC751AL3RWgeKoaVagQpsz18VBAHvQSO3bjEp7ee20DdIfyZG453LWnpfT8bRHLDN9Le5p5hHsLKE2qF31FS+nuj0dEBpWvGJcs0xgt8tZVN10fjCXiBU1hhMrUhiOBcO2c75LZdh4Qz50aT4M37l7znLcj849hksArCXva4hCnI\\\\\\\/vPHVcxad2BU8odp32uSta2Dr2BrlnlIiGU7b6rNP1k8lpC33o7XQTtie+Nwg4ODXK+GlJgKXQgeODdUuTOybnbGX74\\\\\\\/IYbDwSmIpMbzxegcpTGW2zc7PxAFl8jEhEkCPAAw\\\\\\\/ST9eng2EkJsOiNxxJ7WxxsTV2QItbjsLuUCaG5HEH5cKsx0HqMfFPXsq6vKNDIUni673NGXlJc5pNwROsVoJAnhGGD\\\\\\\/C5yVMkoyHYMC9XwrtpncY1r7kamSRZaamFy2V+epnotigcNUjjKCGbAZ\\\\\\\/2yN2x5hFOFeY7MawoLM4WJ53TZJ7QEpQ0auYIos53h8bdNRLWWNJXRRx3mF5v15OBm0nZzbPD45DiC9ziDniA3jdAfYI66MxpL3DhS3Ym9Zo1771KG3r1FK0NUIGi2HyegEHum+5D0sK2cRHvRzIoVIIJp4A8tbmi0WLPqXRadwDi9BFJWK7gaD\\\\\\\/aIbgK2mT1OVB9eEIYg7OkPJEA9\\\\\\\/Q6ZCaQOevtviy2qBG4b1NEK8aoFNGHBDj99GI1MXwNg3sToAv4x9XCsolKmvMFRLkp4oCg1bwLSvh++kxxvEIUL5C8PKeg0B5LC30DGnzpGDzsA+RzdNqBi61Nrw6VPbekUweGoyAK+ozRzY8dzG8E5KFJFjJNwXwQVWvUbAT7T77cJnJ2Z\\\\\\\/7i6CLDAZnvcau7MbiAEA7u1aRbhiCaKpoT3jhW8K+LOj6GnfyWDu7kax\\\\\\\/KeYJMWynYlt+8VwGO7+LsFEfbDHLowGNbkk2Lg\\\\\\\/6LhAo8g2EY5wnF\\\\\\\/WNDedqDeX1cnPHJ0X58fbSvTruNnFLb44hiq5vT4GYoJBv0hAmWx5+1GfOSxacsXPVj6fTmIWj204St3q0RghfJzPzkuYtrALN0ocgvVNWmxgjUMUQ2n6RrmD\\\\\\\/vUUstcCCmX0K98wuJkrmTadEJUtHbLY1u\\\\\\\/a8fPRIRYQFt4bAGhHvMjrlGo6c9TxEol9TpapUr4c918ZmAKitQpf27YyMXBWnjq2s2gc+3KVDWEMzpKf3LV4eN4GKsgsLLX7WUFT2MeNjg0G9Uf9cZTWB2cGKvA+H2IGITmacVtaqppqaHfDHrpu2pMekGL\\\\\\\/BX2nktqlDlxwiwzCq53vro2LQAU2Rhfq0S667YQva5cAXd3YTXRbH1HJEKgho7WjMz3KrILPOXyRqpazNaAeskDIALxgHTVn4l7NMrZybY4qcQmld07ZXDvIldaD4aaBMDFP5HI9xps+uwI0c6Np+VHygu2ghD3Nbrdzz4KK\\\\\\\/7\\\\\\\/+t23NpZn971t6Cxa0jL7vXhcgr6b3hM2\\\\\\\/fogj8LIF2k9aAcEhLeY8PKcRKYL3khWu2DbvFLJe5pJ18w26q1TXFWDEIbYBOCLq+Dwa9jLd9TO9eWAz\\\\\\\/K3USMDA9tlqzwx4TuLJ64fTRsO4bR9ArQ2n2Dmnxb7XOhXMwyDe7Ht6+GX1VnYM1H70SFmx1u5YxZ3loBEvqKPSjQ1Uo2TcytqCFcBBQZ5482wrMy2dVDFKrMk9Z0xJJss8E4jz7yib5i4J7lM3vPEgpeqpTCjZZWrMILHGtoMbM4HyKhmlbehE5pzQfwMxZ2mcFXbvvAdsm+G8sEp+CYkqt2iY4gSBpaeYrKwO62Aet3LLB0U7Muxg\\\\\\\/n6DZrO2vPGjq1tBLLExhKCL2nZJcvEQjGclLHH0jmSCoFXIR8UjvphI1ScPTRu+I4irA1lyLkoh7VRuy5q8v7MBnqtVW2cx0cf17ufraIMpuSFs9T28mv2DQfEBfE65yn0xfuNcEnPzAV7oeazmhQTcAeriWcR4Izrf5+g09EBoK9IZhm0yOGL+b+VvZXJahoFmmmPJ+O2kXdb1IDfD0Rc+XK39sdNWsMQ471+jYjJdveA8\\\\\\\/3hIy\\\\\\\/mwRXaGvKBOOiSSan8Ri3v9bP72xXbX6kGGtTLFjo8eRODasfYKS76Wfr0c4HFOgJC0tbTtgafo1crmz7uvnUlkSFVHjQFhk8glR9o32NfI\\\\\\\/zOw8+ZHiyJQsr+FVZxwiTo7Me70s36XR3TKnMwDW1xnGRYz4NrkARAWFkv+Sa39IdGvQ6TUnebLQOG7OS30CLfJs\\\\\\\/jeKAczYGxgS8dzljh4Cpc\\\\\\\/onn3thSzT33L5Cqsnnv2TVnSHBS21+6M7D1h9qlZ3BFi5RHq0bexVrKTP3G62\\\\\\\/o1L9AEEEYlOJB4j8DicV74r4b8IJ8L4AoI+c6YldjgzNzaHC3hLCuTuSqlw2bA8yDP7JBX3QrY4s0qhoZNwLsNIZ1UDyi0pGtESkO5PmibZiwJaRgwS29cJTMqEij9CQ5fXNf8CzuuO3UDM\\\\\\\/J9R\\\\\\\/N+UztFT4HjnTujUZDCtTSbF5WGuWwRYHJQn2L0jML3\\\\\\\/LvlBnHNLctLNi9+uFhDvVfg2WwPbffOhnwwgmx4V6cS8k7OhifQIHu8IqHcvZGdQQWOabHw9vcR+t1LY3WUoNmWkE0+zlEGizTESIFNY90jVRU3jaj99S1zxC\\\\\\\/85uFcUfmMC+LdYom+li+NX345irlGcaaattcWAs1gD+pr+ShiqlV+M2eHg267xNiF8fxW39SeAywVQ\\\\\\\/LscMeD5es4HTUXQ2UwetZGqM1JHuJTsATAgMQBuvlQmA2r+2wj3O9XR4FtUsN7AZin5BMX0QwTgOhcRss3gSSXbMROgcNvLjrYmc34RiSaIfxfjRTxTi+\\\\\\\/Rddupd3dGcoOo00QwMQPvt7yw1oHXuSZoDJ\\\\\\\/0ExxklKJ0mIfUEdQTCMshDV+3jTTkjtojuTUXkPdqRL+ggIjn7Wc5VuqA2o2EhogOgUpqMKZHEbSfgnwBHOwctU8s5M0P9jRRnKSkqU4tuHnvOuT4n\\\\\\\/7nl1crodttKlzD5yM7sEoryN+mOIog42UGTKORr69ASDk4TKyqAVAm2G3eWWRMwdTk6mOgufvU3v\\\\\\\/sNe\\\\\\\/5h452NTAdkhj+dr053u0MJ6ayhmbFav18yUirFXxiFF8l4cufceDGwggpdQpu2Q7OiwYoBvXB0a2l7svSNOb4HrBNbG7moTmeLzECCCqQiT8ZeN4qC4+v8PggRJIuciYLa0BiauuMsE\\\\\\\/6RM+MV2q+338HBzHEgJeaQ8yIaihltHYTuMA8qemqlCY9FqNv+ozSF9ACKvAFsNhHoOzr6M6buyza9A31vH5MYv9sSlTQHONTTPcQtq8VonvG2w1vJ85iDZie2OtTbJKzcIAPWRmk+TnPh0HfnONy+z\\\\\\\/k3xxjLkCuKbrBnzFzuP3oSTwneGY7uXDGo9NOLZYTt\\\\\\\/VDygUYfgAbwLS6v514QVfIlCz1mu1lR47GJS26+tUwBwRa39Tyv4NR5gXA5mrZp4aGp4C36VDyyD8FLtPmUT9gNlxU5Ro7Uz8VaJ6o\\\\\\\/KUYe5mx5apPS4RekJMXIJXG5toywbJ4GDDr2Vka7j4MnC5L4+bMwY9\\\\\\\/Ci1VvQnW+cs\\\\\\\/HTMZimoIKfcPRzR+NS8tfrNrSCq7rEAtxMkR4YEf\\\\\\\/9DWiIJjVMe0ZOxYQ4FQEq7+aOzDbQ58DVEb8XVorG6YhvP4Cla39BEck732QCHxpojUWfJo0qWz9pvLiM7Fisykn\\\\\\\/SgeXmz43BcAb67OVUoMTNiBzn3c89lfs6hVf29wvYczibpa3HRf5GI4JKvAwWfs4ge41gTX79Pe41r6oh6RsMnA41hstBB5Uu8V\\\\\\\/EqGuIiOGCw0p7i+OOHJN+DfrPOuAEDWwc98zTudbt81JVQZhDVhfCbsv2l1C5y9PgdmxiwzOZvgq8mIsgiSLk117Cnr3bg1MhRyFIJMtCKT39sESQBQkmOLGOYIHZTkWmRJNIWYKUo3YsbN27CWi0znpXoLCYdGyhixgeu28M8bt9smAUql9m5MAuGMstGOYg9xuHFChSNT6\\\\\\\/7VhJqrm4RcgWIR8QCM5uEVjt8P\\\\\\\/2P6hqq4qGGJ26VxgHhEYkBRoXXO1qolpqiO7nyDpguz5j9QvOjD+ZQ84zXdC6MXej4JYNgZymPWhdgMrNOVeuNgnzYdPaAiZM3oY0PpLNelRsceD7Q3sUhJQaqT81M6HOQJoAGZrbjpK4n\\\\\\\/DJKLmQzqNBIKF8Ex791U\\\\\\\/m2f4NtK2nhZfiYWefjl1u3tDtWW56YihZm7NIjoEMgl5OgPXVgyaAEGCYcfv4jCJgKfaeLpSbv0OgpVY7DmqZT93F+wIhGHfPSmkyc+WE2ERhMH3Ku6+eDls3yRXPuIXdAQ6nvLPXkCQMlUY2RF+kFja2\\\\\\\/hAxhSVSra5JmvaTO8MZo2CR+W9t4OpXaZb9mbZmrmREPfQZ9Lo72ehoFfbwhP4uUVqrQ8uDlaMXnf8Oew7qCOXaJcOaBo+EIL5PhFvAtJKTJeV9GCuYzzzSj+YWTI2B4vjZb4JeLiU+rL5s+kRadbj\\\\\\\/3KTs\\\\\\\/uGGaZ09TQIVcJmwIpM7F+UZCnFRh\\\\\\\/4+JPzp86RGsP99eViIz3L+ReDhx8LSaCZU293gEI+Hm4oXjc8Meezz63OHCAz0OVC7mINuJrcCzG8O7Mm7Sub81wNYy7drrYQ8Kdgyt4KO\\\\\\\/4iJFEJ+1P10mv1leunzS+9aNsTDQ1ybTa2i9ILJHzQEDeo7Ex0D4Z1d4doZLcyVL6AELdX\\\\\\\/aj4XonVKv2ZNKk8XpKEVl2CSi2gt1sd09nbUmSvAOffiPoStijRcuZzUYQhTqa0PS6uF1h2QnJdlD7EHXF7HtI0unLdjnrRFDKq1MMeI0x0o4ABxh41j\\\\\\\/TSQ4fvVVzRLqN3OGxBU8On0hUtcYqBNYC7amzGaGIM7aUIhtYB8yeSUXpFE\\\\\\\/oixfVaa2gEdvNVONnuqQ3\\\\\\\/1BZNjhmvk+xJto3w7TFgw7xWVBepf0TGXgtdYS8PagkOZpZFUyn6SgImr4BrEWT8qqx9jzZWVIxZ6ZtnAv2a3mX2DpbrtpX4sNe2BaCKQrttH5wW0utAymzvivSY\\\\\\\/Fm8YrkYb8zwHaMSmiyVdCRGmWkEIX8oK6tYc7c9WYAwLSqH8oiMpomc8iw829MFZ18ohzkQJdxAe83lxuRX4nq+M2j4CqEkdxiW63urGGyeKs1t6mB0csDb1zrfNiFLekIrxrKek6VdShYQdoNEIkUQBpW7T9egyaiXOQ4iSe6ACcEs+Kk1XZBTLeBq2vtP2qsQT0674nkiZspKs3FuGWnW7IPzvBgi\\\\\\\/JAmSlFcDNDTf9yX2ES7j1D6iMmaqXmv5hJ\\\\\\\/vqMfOkuygGu9oCFJQpA\\\\\\\/nIVzZvfC54B\\\\\\\/p6yg4MDdoGw7cn9m2gwXii1SViOu79nR3tqMWEcNi62TaUa6Ii567AbGTyQW+ICHaZJdhGYrWPgC+4OwtPm\\\\\\\/cuYDb8CtfxGOvoazo2rFpWcAxlv+sThEMCEIVfe\\\\\\\/U6XVMWucQs4yujact3c6f96vwdK0O6iRu4I4tDuAccjYrV3T9dEPi0FqOsT2WdcA8PywKFHoFEDpS6HJtVO18ignolkEXcUMnY7pfKopAoQBh\\\\\\\/I+XjyHjtbaQQtEdtv8i+9WzbV4awovaaqOAPDankdMTGD4kwKe6cDrz\\\\\\\/yz+qHuuKfSXcXISx0kgduEVGP6G3qyFnPdxq9hbgw\\\\\\\/yaOuh87g5UcPSGQs7OqeWeVa70Eoh4I3yBp6o\\\\\\\/SEW3jOaiw4fLk1nMBjFXXDa9qTwPc+QU7FGKlrTsFeJB+dwMXJ0whIoUDVJDSTk5mw317xVy4o9jnpRcE1tXWcQtXLe96Vga8tuMfgPOA13xRazDbZpkoOtdhO03IL8BeA+Kaqy1DONz4hxC0mjAE8ZrLv9JfslzDtD0TUCDftVy2cUk19BzBPIFETo2E\\\\\\\/jXFKSy5uYSLIFnmKA\\\\\\\/uLTy+U+wbCkhMb4hYlbpkm00aPhXrbtyhcgSEIfTBENRWpn\\\\\\\/NMOfQIJq8Y0Ttk577BErdyv0v7Vx4NAaNQJl4cfJidgm7RomC5fpI1F2clZk3ye3yYJ3qevympeMVoFOQN4UotugOVqrNANm9eNGrsE5420o45RZ2WguUBbUN5TsR+VIo0SVkPUgHWDZn7N29Iu85NKl4+pT7+4g+ONOQ8dGMAuaT+6Gg1hWl\\\\\\\/Njq9GFIWy9lIJmZK+BZ7u8Jd5Eo6mJ3+aaill3fAA6O1\\\\\\\/rLGWc4xZs8NYbINEpwLcApmGq+GK45yXfoL54zwgcFetFCI7FWO7D0bWlTmmQLx\\\\\\\/jhhUssyvPQ\\\\\\\/w5MIb4z7\\\\\\\/VV0v1OLTlOqO82uNaqn+BsZvtwQGSYzkE0YzeNU0PenouwzcAhE2nWxDk6LzATz29Th5WgJ4pi7oCSN5pK5bY\\\\\\\/X20t8PFnMgJ4Rc9xYtRSQLIEvA186\\\\\\\/dJi3aHBr0geGlOJL6aRZmEp5pXg1FgCJRTW+6WyNruUWvaQhhG3Cryi8XMXF5p+kM07gsh0f1CHOvv2IG4ANaWZVWYZ4oTsaEPPU306v9y43aLNjS6S4v1XTXkbPdr0bOo0p6034+9mCri5n6GXJNFN9Z7194JyBYtjMYn8RTTZnrghxzPR2E0JZvTTyY0g1WiBQ\\\\\\\/HfGBb6UCeVsb+lE5yVNZDkS2MwlH0d6kxhWHNCbET6n1nnn4dFCzpbYcZ4fjTg9NgmYZIgdQcEJa3r6xOEFzeXrrRCnVFgKDqmbhRJS10R\\\\\\\/JVAbTKgb7QBlqpSgsPfdJLOgpC3Ed2Rs+eO\\\\\\\/kS1OeUSH75uEtmWhxfKJD3XFKiKSEFgfcuy6ZYcgRULwaOd6JAhkV9DaEHhVR\\\\\\\/92XjrGxe0RUZMpxolUIRwKPiwGvosixAUQhMYOCRzE0df7RpedMvDLDzXe7etT76RUVZjpzNsOix5XGiq63me3o6z90\\\\\\\/i+PjMJ9RzEIUEq5\\\\\\\/xvhYLdVMy5Q+\\\\\\\/dUu7aOkuTwdAkl6hoesPHvqEgENV87O3t02JVMR6ao2be4yVgikf2OvZQpmdjzzTnvTXdNrnMQX9xVNLz2pLU201GWmOuiT\\\\\\\/5ZFeesjJDKyXBsj7riqmuZecI6TliclvagM0AgXLzRUyEmyNaDrB8DoCy4TAZI44UFpnUucjtdVuXcT\\\\\\\/ZyEHI\\\\\\\/QyGy80RY7eGEGbCeKCaeJ4Ebfh1GNk58sxVz2LhUO8SDAMiGVEyEfnhR8u5OTEDNTImnPgZ+5gQ9Q4RasBQXePMwc72sxydGez4VS1O4lHTD6Mt4EQOzwhW5dr9IJ6SdQ04Oxyvqh2qmKCsmzT10HtWK8giS67IwebtcQjDM9PcG89RU\\\\\\\/+3sFxUEOTcugb99DZNVvkR9wfO0zLPS52mNsOTADxAmjNod7o1zrt4b9qss9Nutc3X19Gjyg9TOa3FdDg7Qfgr3x2Gnbw1GPgdXNX7fhqMH54cvihsokSoA2puuJvuMjjfI2gq8lAmo3yrBzmFYa0rX84iL\\\\\\\/iHWPhT70JO9ubaHanzn94wDw0CpLggOQPda5Xt55FVcVYQZDrEAeoQ3Uc9q2QW+vvvaNhmndB+fdQOTgb44zFjl80NnVhierNe9abDpVaGZhBMWKGicLhzhcYGFdr\\\\\\\/Za+Pa4wbuOPNuqARVrbc8ojPzNzbqjIcRYRsgN3IW0\\\\\\\/aEb6MIiHrb3d9NIRULHRXnbavmF5FCLpdKKuhmWBV8LFmMt\\\\\\\/MLI4jPhOd4lPNliS8XvRMweu\\\\\\\/L7YXiyBNZ5uF6oKVSukI8ydPfPoGCzuacd2izK9pu78wVMkK2nXfewqBQD5AEMdH91mQRYcN1XGG3cUgdwOkyGEJ24QiMhUbabkb0ErPB93HNXdMu4TncQ0mfpOnwYwmVCVtWxQXfIj1RD6f5jPfAnzFtcd71AUkDZrQWMevLFtWn4GxcnTXMVYJZXZIL\\\\\\\/SwwvkFhvou317ja8fpyDK0wVHioTMDHa\\\\\\\/6q4IX\\\\\\\/lWM0pnrzxeinDZscK7LaoMzbKO0nbmrCUouwRlHsNWuf98GYOx2OhGWfM59\\\\\\\/\\\\\\\/FwNTdnhO5Nw6W8t\\\\\\\/+t3LVK7+aoy85UWSkZ00vr6cwe\\\\\\\/ZvtKYl+Uw1cKV1X45Wq3NCKlhhcIM5nil5h31ifnGXcU2haIoG2nqqfEmSU7mnGBA4t7ohbdMimONHg2xtoUK1\\\\\\\/K67ESs3emJxCLtVMTmv7ALVrcAuwGAmXe88L25y9EitJAZFhxG66eNsNQh4qIQnSaH9Hph1IPSaFwmdagb37h\\\\\\\/F4+lYG5MBmzZ3SL48NJMcBwcZrjgAIZ1bjTy+TnAvO3d6AUU7Iw8Gk2LV8hVrBH58JF30Adp3aGD9pi0f4RyeDFWJUcNSfgNI39eSQ5O5ekM6wqEgfRaE12Ii1Mo0d8FM5R4QgkqiNi4GghVSB\\\\\\\/4wTJGWfxOG8PGD3bf\\\\\\\/Au+ROAd7FT6RBz6sgYDpaS3trtKqRlftT242Nc0qsYCB3lr\\\\\\\/PS4PwVe8knf\\\\\\\/1mcKjP9Y43pcLGmKyeDzu4Oc9GjV\\\\\\\/nu4vpqx36YmCQVnfTqNDQu01Nzsmb0gk2GrBkgppEpAqvLAlVOasm04xloGW889jBzevLGWjG3J+AIwd9M2wXdy1A52vsvdfhiSXdgtPBOez58bjnC9V2y\\\\\\\/vKYv2tQFHXbHZVkRc8261eCmgKaI5GV3QgoEBe6XOEtSZ1iI+\\\\\\\/L55vhganiyyt7SCtjpPP5xBDAMrdUN89LQhT5jOlovaVvnLAOa07XcQmwgLgFuA+DlGbsWr+QITFwDRh6yc1VlDH6Ag5JXR7RRVYX8FZnbMgyBIcz0V9202hqzzU1ZkRgAjFbvvNK7sT9fgVnlt9nPrlD32UM5qIWRH7f+K2iK\\\\\\\/trxDz18EQNEh8SVypa\\\\\\\/xwrmQzS4SwlfiAnPg\\\\\\\/UAmzvGho13zC4Us19mXh7tI9vy1Euljrg9Kz4\\\\\\\/pBco4EXAqow\\\\\\\/fsr7QbLHO5FMucfu4tN5RaLSbrCzxuWXEYCMmDORH5dhI7+ALKgupHYljY5w91Vp2b9I1gEPiCTIS2UTSpw100FyZKnI5vhWECbO1HJ8DCbjH\\\\\\\/GO64unmvF\\\\\\\/UN8PTl9eVNbNm+MKa8xVL8JsLU0NA98SYomHQgDQqB4f\\\\\\\/Y3IueD+VV+IOQ3e4UD3Lxz5yy7GAcn\\\\\\\/hfeQhiWe5UI841kQiw+tS0yUaK27WathRXBICl9hYYFS646L\\\\\\\/D6A\\\\\\\/LTaKOPEUe10UafT9rS04hXtok\\\\\\\/nFcVwx4afS1eW4STM1rEBBp3p\\\\\\\/+UHhpaHilTX68lcNtxB9sUVHFmSUrAh5JxqtXwpJW5wap5BiqFUP6Vfv0UI9uB4aT3ryDJNGt3tF8Rn79kwxdy1eXaf4xDL3D5hP1PPsp4H7fime1zlSl2Kifb19qG5G4xteMdfuBnEVi31Y9Yqu9JLI1ewDhUgkQ9284yKzoliYaiNxrAsXe5QC8CgGplUlfpJjdyRUFUH1lZQPt5BDMFWjLw2h1n1DGbcryRElZMw9YXmQVW8wFIO9r4UBt87S\\\\\\\/0zODeuzwUUh3KgIBCeinlDTWmr7Byp2xeVVdMS11\\\\\\\/9WW8fuy8R4LRlkrWxn9sUn8fvjKRJlNuW7dn\\\\\\\/NxVXZ19\\\\\\\/F5PeVc6AE1rYz\\\\\\\/F4bP0DZMrEAC12D2fUOvYoUQLiTw7aq9m16kAfG+rlcV1Z32M1+yQjFQMBgyTZZMxc6LihKScVbxlH17GhA25QmmnGfL\\\\\\\/p5wYkJAJ2JDROyy0oL7mUyNxYsKHVEX9G8ujda9xNjB+RLULNeoNsja83HKVxxP0erXpweYCgD8cX5krlIfO57TpyT7UJr44m\\\\\\\/VlRzMyYGa8moyjz4vVeo1CaZQTfVK79NNAlZaeFWI\\\\\\\/xBWKF2VjiCUemTiBEa2G\\\\\\\/T2Ab4kOADTlF6ld1AnXQ5IMWVW31JaVzXx5tvaB1bAVtp346dAbHMpw8TFczYdus740wFGdXSGCuyGy3sYnJwgnHqlxKBXzkt9wtTrM4YtIoND5YelKqj8\\\\\\\/UD9pCc1LbmK9mXHat18+gIcZnqd5jLnKjxvCF07rJMNNyej1KAcfp1eu05C0lU\\\\\\\/KcpJAugBUTKI3h3KnEOPl1yv9mWISacjrSF0OccDBTi1XjmwfuV5pWCNmg2mXQm9\\\\\\\/nGiw6fSdmNEhiF5fGi8a+ro29\\\\\\\/2\\\\\\\/kpbQ+Dvf8itDshPypKehvpPLUUh87fBNgdnijYBNvLifL2xjSBkfvinsLFZonfnvMWzqfdh0DAtrjEuMabx6bilvvBOehNiHB4gvq84Pw9jzJRHC5AGkQ2HRMt4PF3Vk\\\\\\\/WiyH2V7+erny4iER2eCEIXsOGx\\\\\\\/KQPKzsJkTCJhL9AVvqYaD3eRrGanLaUm3MwCkIhIyFro55EyQF9Qk1cjuvN84uPvYoU205SJcZ21SolYqWesmORwG8PlKsU+42HAzZqp\\\\\\\/GZ+aJT+hA8q3zGIwXaiGYAE1MNZVKhpDA5bO2U0LsfFTs9tcy7iSrv1rjhhFowTpcrdmOR19Su1O3R55V1xzMYC4WU34OJhHPssWz+jVaiHSkmy1OSc09oavDdjYnkhe6dV1I6kOSKO3Gyg5E8BBG9RhufgsYPx4GaBfaK3KITn0zsD5oeMl4DkCBhZAy08tDHXUsJl\\\\\\\/PV9QpwxsVp4royIDnfu3iOneUPaPl+U6SPW7JPF9HerVQEFlv2JSwtQ7G8IhP+88MVUfcfqF1BsyG0r\\\\\\\/LyptnQsPSIqhvY6PPZuksSWIS5Ie9DhT2e9e31haWMyjXzTnbqr\\\\\\\/0ivmN4BQUaDyDY7cAsmelV2w8RPH5f\\\\\\\/eiVQcUctKgXeN9cOmyl4s\\\\\\\/YZhoZfwhub88xFKdBXmbgDCzANs+ZVBu0CTtPmYz7y9gOy3utpCXHtFn1Vt9sMeYsavmCmSnl3n5Ifr\\\\\\\/Tn7uPpVApxpIxga7mNtNCY7Ltlst6gCRF3fFYMfp+CmDjwpTR2WMmSsIy7Ai2nnpOO\\\\\\\/DSB1gXHkz0Asz0KAUqgGDBq22AeVT21knOn6ytCWtbL2bNLrFNhQ65OaGpEIZ\\\\\\\/DbP4Gb4pRtjpy3\\\\\\\/G\\\\\\\/57r2UlZFBR1r4J2v9Q7j\\\\\\\/tEt7LqCxh0o\\\\\\\/NBGU0SnLsFFUqqpNzabEJQXRbB+5yiJo\\\\\\\/Y6ENA5TJtiQPl5hOTH2S46bl7SMSZ5KpQZIQqbyIwAEHNhEjku3Nn48UHtHnrgSb\\\\\\\/4NfuBryI7Ugj5JHpA3Pvuz5iHN9dAOelpEW5W+H4FSrJLe0aLiVDl83tERy2efxnyR82Lxdt1D8K08DwzdrYdtE6ITEdtoHud\\\\\\\/Aj+bYU2EW5hB+Fj6c9D+3d9Yk1owaTB77sbobigec8SphtifNCXRzurIP7Xp3VxOoM+Vx9r\\\\\\\/tyholeCJJMcqhyrnby3h5fXFGpYKrd990ik5ubGRXxRBqOU\\\\\\\/9JUg3y1ZguYpNW33NX3klGkbuPGOqqfBKvavC\\\\\\\/x1EG4e0mr9juCVg8g6lqIKyjyprS9ez9o0aHdDsqkIMO+p5tly+rBPZy6xS+Pyxkf5027zOp62+ua5PS3GQRRxOP2QHfwqvWNJn3qhw5XmDkl3rG4p+I\\\\\\\/fifS4Dk\\\\\\\/fnPvn97ovDvjyqBgB0ZCZYuLdSStw6tSt\\\\\\\/fUidJMuT1v0QhHuVeeyoLe0GbAn\\\\\\\/RrhE5OJO4oKYLPjcOlKwBzIchGXQZRUuWnoveEtmpCV7d8J0CL8+KUd9WsQCnjmgS2mLCkVyvtYze4E9h00loo+RYxYfcDpZN+wRB\\\\\\\/XURqbMXSqrYxvqKe9Vfnuf4jWTcTA6mRxTtyXdf9Ol1KfkFgrlhnlNjHcyUgozrsz6ctOlBNvoUSMxtCpfzi1ajO4tdKoGOV6p6kpzncV9S9cXc9IBEoYEY6ATRo7xKP2iE\\\\\\\/UqCpPwDNpgvpqQMl9KFSBF\\\\\\\/Fw1UmRV+ygfwlqu21FszGCELpjNUAxSFIizs23reA\\\\\\\/j3EAEb+v7c1KXkg\\\\\\\/MV8estzTp76katuSqfxdW+NMp5R4tT2d0sE3LyqX1OXrXzVUtKSST2Xf6f28W5RbjuhFFo86EYeSQPVyWluAv9HekLdb4wP20aQuYfEVSB5HMQebDCU5\\\\\\\/gQyYhwV3oq2z74VvK4Cd1PnDSIqQFK7nJ\\\\\\\/gspeIDX5H0BEYwI1vltsntJED4H6EaSN9bESSHH2Ccr4KrWXVYjN\\\\\\\/I2wcwg6xjosn0Nu+eOIMMJ2cclCIzQseZf9QLGq4N3Qo4iJKiaG5SXoDIy6p4IkRNGQU8QV3d4TySv0YWpsfcTSHfJpXAthU1xvqUOPouRhTtvkFbU+XC7S8NumtSTI1mRvKES00PqvCKa\\\\\\\/NjNtix6nyQEFsFMY8m8kP8sLKfc2a5TOrv1vF6tp0VZQMeBW5nX6lNHTNw4GqWISU5TDsrJfy38GTX869uKdRpg935LDy9Iy28sMUyoXVNaNjMao4j9RY9jjmDQq+BmSeMoYCKsgh4g5djiZo50klrfNHONdiyg3smCUxUUUBH2zxBFhsNOH1+JECJCMWBinLfmfGsMEMs0f1I3LVemfcgXO14BTVeJc84ODnRhTokwZITprR2z9V3qjUywNcNYF9blaTKUtiqoTy+JYcJajbvZ\\\\\\\/BK8sMOqv7l6XNTVasJu+WWz8+zOjmVbRWlf5hq8J87bhIgTin6abHekFC6EOGTztVcthR3d5pUBFARiB\\\\\\\/PNeJL2cDmHdKh1Iceu5Zhu2rhRvkODG2M21NeCQPb8MtydvpUrxcPDjv2MwzVyE7UR4zMl0gfCJAlFcO\\\\\\\/UnTJYHyRaoeHQDQ3eAsw6myjqjQkBYfXnuDemznX2kpBOFCcg4JgLcgweQm8snjPR+K2c89y2XewwdSO0dJLKU1VCtzPq7GK\\\\\\\/m0CotJgbDTVU4BpDU0uV9f\\\\\\\/trSwh0x7hn3eO\\\\\\\/YlfI\\\\\\\/NZYg800D4j9MW4WtqS8H+BRUj8oSyki7A+e+peVXcXueDUgL1kiYa3IaXOJgLl0qi+Oy9Ckg2pB\\\\\\\/gdqw2hKwUdYQNLpBgv\\\\\\\/VWzKsCSVAFqm3M4XX2YBMfmncNxhhqmW9EeFdQDsWmpROrHH09anKfk1XA3vjYrtkiQBNdwSdA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HfhbxX+HlPs+i2gD+zgG1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002281723aba5b6536afaef472e10556884afff13b2d9b9eef31cca9a9e189d210bf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AHWguQMyI9WMHqMtxYCNgwnfTjM7RtRgzZB+jhKWQcB2yisk3kVI\\\\\\\/PfTt4\\\\\\\/pqlOsoZ6N+mB5hi2dpxVBaX5\\\\\\\/gYE3ycDfugU4jbjszsxnxdF48vdyOXNSdU2iVXy1XtYwOpm0Vr7fHUdv2cLkJ5uJhHq5jC3wh7KUfpLjxNvi3Gw43oaAzDhUoUfeJVZq8wSBy1mH6l27m4\\\\\\\/9iKHtpLtdqDxt3zm3JLzk2hzJDpiyGClhz3rV7Gh6xEv4YWUKOtp4P\\\\\\\/+WCY7HWhAQTfOK0wThHLgH4Uke5ysyXs4nXNuKxHonugNBQv4ukvzy8ujAa0\\\\\\\/2UDqpDEskNsJfHDeuHYhG5Ax9U1w1yYMZtkvzzrWrSL8XoUlZZb9lMmFLMUjPXUUCr1AcakS7N3DXfQxDDJUMNbir2uH8lb2CgA6O6B1+2lWMWDOclLS1zSzKSxgjcQmD1eYcpe6ttHnvLwuh9XirjCSkVCvEghx4p2hDKL8fia758LEi8DrNaLF8UZTNHPUHuEmYyDS68+lHtmUgzcgteuJIjo13UFm3nXR4W5Hm8cOnPORzKzpw0npbowYimSXV3J1A1leF3R6+NlIH48XWAVXgtIf8Yxv8ANGvlFBBr2USciRWTZd\\\\\\\/Dp\\\\\\\/8UE05R6bCDqEKLbW+cXRPmleown7wXIIGbdCrOSe9SygPX3COapc1Z2tjRZKykZgw3LhSsCKlvgFfTSFXyo7MBe0f8x8pNGecrHZKjnchfPUE1iaJxyDPrH3x3+d72BzLRgPukfhfOk+at1tBeLK50O\\\\\\\/prHh3wK6D\\\\\\\/2iRwhw1bNw6AHkK6HFUQsqRkP9frtZzEG14yO\\\\\\\/ngehfH4mAIRBTcgig4+CUrzJDef\\\\\\\/CYypJM1tUqkXdNvqXNM8D\\\\\\\/ssMo8llGCAW7eo0tvDo8jqG7zpYr0VbNeUiWCyfR+YqqyZbDCRI6PqDh2gUJJ3K2ZXeraiAo9mtrHRpOmlhl6SG7Dw8pkq9ZY4Tdf17DKBEjcqXLmJZ3+OS9zG6vda+iJsCNg2dqtczAd710Re6O1ah5NMBM2yQsLP75Bgv8zNJinXTmuPDCTiFeeZFoEbjQcX8JkAgGYWcQS+UgzeV6Eipr0\\\\\\\/NIX5V0iC0YQFMnnMs1s2tjveHOWUP+We+CBCrcMZErCWVTvzoQ7KvhbnZ01glafO9Qav1ZpulgCUtVAI3GntTFODUS2XyxHMkv+7a9uHrZo6CgS7X67JKw2\\\\\\\/75lPnjb7Gmt4EDbWKTFyDv5yko67MNKD45jCwoO6mDm2vZ1YjWq6nuVRDf9Svln6oYbvXuGS5vMB8CB+CrPbRbVFLfEFgZ\\\\\\\/nKHLPeN5zVUsI9oVsnQWqo3QqQfEhCbbEf8hIx5XrB6vGAan42R8CnpFUMAA5aCIAKVyV6F12Pxh0kNFZZH8fFHXW\\\\\\\/Qek+WiITY8SesWw6yRth\\\\\\\/8Q0E\\\\\\\/t0FA4gjSB4iy7hAwynNIMfubnWx\\\\\\\/0iIGVi2Jlg0AigSeoLs3O\\\\\\\/VIJ66sAx5BLlZ8T6qsGtUn7iDTcs0Mt50wrgNWvB4HIa+TDKA9XgbJ3rjzG9xzu6UN0l2Z4yij0LG8ZMutfvfo9cxYUhfqyPAtLCBTV2vXKtvAMB46cfwhv38h2FQWmiZTnogtlxPZa\\\\\\\/dEaHpVqfoLYfc9rrwaeACrsvgv\\\\\\\/+iIsLZ5j00+QgegWCoj2S0F0A6Gg8XrbKMljF25RUPYS7+uUrSWHiAKUiRzJxHWI36b5OxF8djzw2Nyq\\\\\\\/ge3XTlmmRIqqHL6Rif08gZVIZvEE\\\\\\\/IKkONxmfp75QIv5d0Hz\\\\\\\/uvSIcVVFJbpmaAcfnTHLK+0uDzN6C5\\\\\\\/he2rvyCY\\\\\\\/rXUmnLcPfap6BGGJ\\\\\\\/0IHtkjXtEfOdodG6GU\\\\\\\/soBtAk1fvqzSiK7guZR\\\\\\\/OeDtcBpGWonO6mMKK3Po6SxgJ\\\\\\\/+aarfO8fIkm4g\\\\\\\/\\\\\\\/fgSl19wOh56kUpzx9rjnzbw9HHhDkdkYpIorIQ58tmAz0QB3yF9iITqzNG8U+mMApIDenjW96Y3o3Ag3QwNwf291JOdA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZaqwJ+xbMmYHi3A0qMnHwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c372554c3e8ba2a6d8feddf4b5035a9d4c24031059d9d88bd970f2e29185ff83\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460066 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/263" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-t74Be-CkeVog346CLiZ4zjLsyZ1HOWLnSpsJsH1Y_XQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iTN6UDh0gN7KkA6mt1vYqymJ12jHt_7JqmbQRvFONBU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/255" title="A RAB3GAP1 SINE Insertion in Alaskan Huskies with Polyneuropathy, Ocular Abnormalities, and Neuronal Vacuolation (POANV) Resembling Human Warburg Micro Syndrome 1 (WARBM1)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/281" title="argyle: An R Package for Analysis of Illumina Genotyping Arrays" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460066" id="node460066" data-pisa="ggg;6/2/263" data-pisa-master="ggg;g3.115.022087" data-apath="/ggg/6/2/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Mouse Universal Genotyping Array: From Substrains to Subspecies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew P. Morgan</span>, <span class="highwire-citation-author" data-delta="1">Chen-Ping Fu</span>, <span class="highwire-citation-author" data-delta="2">Chia-Yu Kao</span>, <span class="highwire-citation-author" data-delta="3">Catherine E. Welsh</span>, <span class="highwire-citation-author" data-delta="4">John P. Didion</span>, <span class="highwire-citation-author" data-delta="5">Liran Yadgary</span>, <span class="highwire-citation-author" data-delta="6">Leeanna Hyacinth</span>, <span class="highwire-citation-author" data-delta="7">Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="8">Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="9">Darla R. Miller</span>, <span class="highwire-citation-author" data-delta="10">Paola Giusti-Rodriguez</span>, <span class="highwire-citation-author" data-delta="11">Randal J. Nonneman</span>, <span class="highwire-citation-author" data-delta="12">Kevin D. Cook</span>, <span class="highwire-citation-author" data-delta="13">Jason K. Whitmire</span>, <span class="highwire-citation-author" data-delta="14">Lisa E. Gralinski</span>, <span class="highwire-citation-author" data-delta="15">Mark Keller</span>, <span class="highwire-citation-author" data-delta="16">Alan D. Attie</span>, <span class="highwire-citation-author" data-delta="17">Gary A. Churchill</span>, <span class="highwire-citation-author" data-delta="18">Petko Petkov</span>, <span class="highwire-citation-author" data-delta="19">Patrick F. Sullivan</span>, <span class="highwire-citation-author" data-delta="20">Jennifer R. Brennan</span>, <span class="highwire-citation-author" data-delta="21">Leonard McMillan</span> and <span class="highwire-citation-author" data-delta="22">Fernando Pardo-Manuel de Villena</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-279;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022087 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460066" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew P. Morgan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chen-Ping Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chen-Ping%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChen-Ping%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chia-Yu Kao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chia-Yu%2BKao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChia-Yu%2BKao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine E. Welsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematics and Computer Science, Rhodes College, Memphis, Tennessee 38112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BE.%2BWelsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welsh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BE.%2BWelsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John P. Didion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BDidion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Didion%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BP.%2BDidion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Liran Yadgary </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liran%2BYadgary%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yadgary%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiran%2BYadgary%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Leeanna Hyacinth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leeanna%2BHyacinth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyacinth%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeeanna%2BHyacinth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Paola Giusti-Rodriguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paola%2BGiusti-Rodriguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giusti-Rodriguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaola%2BGiusti-Rodriguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Randal J. Nonneman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randal%2BJ.%2BNonneman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nonneman%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandal%2BJ.%2BNonneman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kevin D. Cook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BD.%2BCook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cook%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BD.%2BCook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jason K. Whitmire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BK.%2BWhitmire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmire%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BK.%2BWhitmire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Lisa E. Gralinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BE.%2BGralinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gralinski%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BE.%2BGralinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Mark Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Alan D. Attie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BD.%2BAttie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Attie%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BD.%2BAttie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Petko Petkov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petko%2BPetkov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petkov%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetko%2BPetkov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Patrick F. Sullivan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Psychiatry, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BF.%2BSullivan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sullivan%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BF.%2BSullivan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Jennifer R. Brennan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Mutant Mouse Resource and Research Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BR.%2BBrennan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brennan%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BR.%2BBrennan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/2/263" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460066" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460066/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/263.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460066" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460066/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/263.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460066" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460066/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/263.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460066" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460066/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/2/263.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460066" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460066/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genotyping microarrays are an important resource for genetic mapping, population genetics, and monitoring of the genetic integrity of laboratory stocks. We have developed the third generation of the Mouse Universal Genotyping Array (MUGA) series, GigaMUGA, a 143,259-probe Illumina Infinium II array for the house mouse (<em>Mus musculus</em>). The bulk of the content of GigaMUGA is optimized for genetic mapping in the Collaborative Cross and Diversity Outbred populations, and for substrain-level identification of laboratory mice. In addition to 141,090 single nucleotide polymorphism probes, GigaMUGA contains 2006 probes for copy number concentrated in structurally polymorphic regions of the mouse genome. The performance of the array is characterized in a set of 500 high-quality reference samples spanning laboratory inbred strains, recombinant inbred lines, outbred stocks, and wild-caught mice. GigaMUGA is highly informative across a wide range of genetically diverse samples, from laboratory substrains to other <em>Mus</em> species. In addition to describing the content and performance of the array, we provide detailed probe-level annotation and recommendations for quality control.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/microarrays" class="hw-term hw-article-keyword hw-article-keyword-microarrays" rel="nofollow">microarrays</a></li><li class="kwd"><a href="/keyword/genetic-mapping" class="hw-term hw-article-keyword hw-article-keyword-genetic-mapping" rel="nofollow">genetic mapping</a></li><li class="kwd"><a href="/keyword/inbred-strains" class="hw-term hw-article-keyword hw-article-keyword-inbred-strains" rel="nofollow">inbred strains</a></li></ul><p id="p-4">High-throughput genotyping of single nucleotide polymorphisms (SNPs) using oligonucleotide microarrays is now standard practice in genetics. SNPs have largely supplanted microsatellite loci as the markers of choice for genome-wide genotyping: the low information content of individual (biallelic) SNP markers relative to (multiallelic) microsatellites is overcome by the ability to simultaneously type many thousands of SNPs (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">The International HapMap Consortium 2005</a>). Current technologies provide rapid, robust, and accurate genotyping of hundreds of thousands of markers at a cost of less than $0.001 per genotype.</p><p id="p-5">Unlike sequencing approaches, which ascertain and genotype polymorphic sites in the study population in a single pass, arrays interrogate a fixed number of known sites. This presents an optimization problem: given a set of known SNPs, what subset provides maximal information content for the populations and experiments of interest? Marker selection also raises the possibility of ascertainment bias (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clark <em>et al.</em> 2005</a>). In this manuscript, we describe the Mouse Universal Genotyping Array (MUGA), a general-purpose genotyping array for the laboratory mouse (<em>Mus musculus</em>), and discuss the strategies used for SNP selection with respect to global and local information content.</p><p id="p-6">The first mouse genotyping arrays were based on polymorphism data from a limited number of laboratory strains (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lindblad-Toh <em>et al.</em> 2000</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shifman <em>et al.</em> 2006</a>). Their content was biased heavily toward alleles segregating in the subspecies <em>Mus musculus domesticus</em>, the predominant ancestral component of classical laboratory mice (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Yang <em>et al.</em> 2007</a>). Next, the Mouse Diversity Array (MDA) was designed to interrogate variation across a broader swath of the mouse phylogeny (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yang <em>et al.</em> 2009</a>), taking advantage of new sources of polymorphism data (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Frazer <em>et al.</em> 2007</a>). The MDA enabled characterization of the ancestry of laboratory strains and wild mice (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2011</a>), construction of high-resolution recombination maps (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2014</a>), and haplotype inference in recombinant inbred panels including the Collaborative Cross (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aylor <em>et al.</em> 2011</a>). However, the MDA is relatively expensive for routine use and its sample-preparation procedure is labor-intensive.</p><p id="p-7">The MUGA was designed to fill a need for a low-cost (approximately $100 per sample) genotyping platform to support the development of the Collaborative Cross (CC) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Welsh <em>et al.</em> 2012</a>), and Diversity Outbred (DO) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Svenson <em>et al.</em> 2012</a>) populations . MUGA was developed on the Illumina Infinium platform (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Steemers <em>et al.</em> 2006</a>), in cooperation with Neogen Inc. (Lincoln, NE). The 7851 SNP markers on the first-generation MUGA were spaced uniformly every <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-1.gif"/></noscript></span></span> kb across the mouse reference genome and were selected to uniquely identify the eight founder haplotypes of the CC and DO—A/J, C57BL/6J, 129S1/SvImJ, NOD/ShiLtJ, NZO/HlLtJ, CAST/EiJ, PWK/PhJ and WSB/EiJ—in any window of <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-2.gif"/></noscript></span></span> consecutive markers. Although MUGA was reliable and inexpensive, it lacked the marker density to capture the increasing number of recombination events in later generations of the DO (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 2012</a>). It provided less phylogenetic coverage, and limited discrimination between closely related laboratory strains in comparison to the MDA, and had narrower dynamic range, making it less useful for copy-number analyses. The second-generation MegaMUGA, available in 2012, was designed to address some of these limitations. It provided 10-fold greater marker density than the first-generation MUGA (77,808 markers), again mostly optimized for information content in the CC and DO (about 65,000 markers), but with an additional 14,000 probes targeting variants segregating in wild-caught mice and wild-derived strains. The remaining fraction of the array (about 1000 markers) included markers segregating between C57BL/6J and C57BL/6NJ, and probes targeted to transgenes and other engineered constructs (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Morgan and Welsh 2015</a>). In contrast to MUGA, the content of MegaMUGA was optimized for discriminating between CC founder haplotypes in both homozygous and heterozygous states.</p><p id="p-8">The MUGA and MegaMUGA arrays have been used for monitoring of inbreeding in the CC (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Collaborative Cross Consortium 2012</a>), and for quantitative-trait mapping in outbred stocks (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Svenson <em>et al.</em> 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gatti <em>et al.</em> 2014</a>) and experimental crosses (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rogala <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carbonetto <em>et al.</em> 2014</a>). They have also been deployed to detect contamination and aneuploidy in cell lines (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Didion <em>et al.</em> 2014</a>), and to characterize structural variants in inbred lines (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Calaway <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crowley <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Didion <em>et al.</em> 2015</a>).</p><p id="p-9">GigaMUGA, the third generation in the MUGA family, improves on MegaMUGA by providing a further increase in marker density (to 143,259 markers) and substantially expanded content. The design goals of GigaMUGA were fourfold: (1) to increase resolution for detecting recombination events in the CC and DO; (2) to increase power to discriminate between closely related laboratory strains; (3) to increase information content for wild-caught mice and wild-derived lines; and (4) to assay copy number in genomic regions prone to structural variation. Approximately half of the array is comprised of validated CC-/DO-targeted markers carried over from MegaMUGA. An additional set of 46,000 markers flank recombination hotspots predicted to be active in the CC and DO (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baker <em>et al.</em> 2015</a>). About 15,000 probes target SNPs ascertained in widely used laboratory mice, including the 129, BALB, C3H, C57BL/6, and DBA strain complexes, and the ICR outbred stock. Another 7700 probes were designed against SNPs segregating in wild mice of <em>M. m. domesticus, M. m. musculus</em> and <em>M. m. castaneus</em> ancestry. Finally, 2000 probes were spaced across segmental duplications to detect copy-number variation (CNV) in these mutation-prone regions of the genome (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Egan <em>et al.</em> 2007</a>).</p><p id="p-10">In this paper, we describe the selection of markers for the GigaMUGA platform and characterize their performance in a set of 500 reference samples spanning classical laboratory strains, wild-derived strains, wild-caught mice, and sister species from the <em>Mus</em> genus. We highlight the utility of GigaMUGA for substrain-level identification of laboratory mice.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Microarray platform</h3><p id="p-11">GigaMUGA was designed on the Illumina Infinium HD platform (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Steemers <em>et al.</em> 2006</a>). Invariable oligonucleotide probes 50 bp in length are conjugated to silica beads that are then addressed to wells on a chip. Sample DNA is hybridized to the oligonucleotide probes and a single-base-pair templated-extension reaction is performed with fluorescently labeled nucleotides. Nucleotides are labeled such that one bead is required to genotype most SNPs, and two beads for [A/T] and [C/G] SNPs. The relative signal intensity from alternate fluorophores at the target nucleotide is processed into a discrete genotype call (AA, AB, BB) using the Illumina BeadStudio software. Although the two-color Infinium readout is optimized for genotyping biallelic SNPs, both total and relative signal intensity are also informative for copy-number changes.</p></div><div id="sec-3" class="subsection"><h3>Probe design</h3><p id="p-12">The vast majority of probes (141,090; <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-3.gif"/></noscript></span></span>) on GigaMUGA target biallelic SNPs. The remaining 2169 probes fall in two classes. The first class consists of presence-absence probes for engineered constructs or known structural variants (<em>e.g.</em>, <em>Mx1</em>, <em>R2d2</em>). The second class consists of copy-number probes. In order to maximize usage of space the array, target SNPs were biased toward (single-bead) transitions (final transition:transversion ratio = 3.83).</p><div id="sec-4" class="subsection"><h4>Informative SNPs in the CC and DO populations:</h4><p id="p-13">The bulk of the content of GigaMUGA was designed to interrogate SNPs segregating in the eight CC and DO founder strains ascertained by the Sanger Mouse Genomes Project (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Keane <em>et al.</em> 2011</a>), and the MDA. The subset of SNPs targeted by GigaMUGA was selected to maximize discrimination between the eight homozygous CC founder haplotypes as well as their <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-4.gif"/></noscript></span></span> possible heterozygous combinations (ignoring phase.) First, candidate target SNPs were identified as SNPs assayable with a single bead, located at least 50 bp from any adjacent SNP or indel, and whose 50-bp flanking sequences are unique in the reference genome. Each chromosome was then divided into <em>n</em> target intervals of uniform size on the genetic map (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2014</a>) such that each interval contained at least one candidate target SNP.</p><p id="p-14">One target SNP was chosen per target interval using a dynamic-programming-like algorithm as follows. Define a path (<em>q</em>) as a sequence of one target SNP per target interval along a chromosome. Possible paths were scored via a score function <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-5.gif"/></noscript></span></span> by counting the total number (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-6.gif"/></noscript></span></span>) of genotype states that can be distinguished in five-SNP sliding windows along the path; denote the score on path <em>q</em> for the first <em>i</em> intervals <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-7.gif"/></noscript></span></span>. Although the number of possible paths is exponential in the number of target intervals, the score follows the recurrence relation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-8.gif"/></noscript></span></span>where <em>s</em> is a candidate target SNP; <em>V</em> is the set of candidate target SNPs in interval <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-9.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-10.gif"/></noscript></span></span> is the last four SNPs along the current path; and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-11.gif"/></noscript></span></span> is the scoring function for a single five-SNP window.</p><p id="p-15">Scores for possible paths along each chromosome were calculated, pruning the set of paths to keep only the highest-scoring <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-12.gif"/></noscript></span></span> paths at each step. The (approximately) optimal set of SNPs for each chromosome was then chosen by tracing back along the path with maximum <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-13.gif"/></noscript></span></span>. A total of 54,250 probes was selected in this manner, all carried over from the MegaMUGA array. The majority (53,529) were selected from the Sanger Mouse Genomes Project SNP calls; 666 were carried over from the MDA.</p><p id="p-16">An additional 46,020 probes were designed to target SNPs flanking 25,000 predicted recombination hotspots associated with <em>Prdm9</em> alleles segregating in the CC and DO (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baker <em>et al.</em> 2015</a>). These SNPs were selected to be locally informative in four-SNP windows overlapping the central 100 bp of each <em>Prdm9</em> binding site, so instead of using the recursion introduced above, we selected SNPs maximizing the local scoring function <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-14.gif"/></noscript></span></span> around each hotspot rather than along entire chromosomes.</p><p id="p-17">Finally, to fill any remaining gaps, probes were designed against a further 1,943 Sanger SNPs predicted to be segregating in the CC and DO.</p></div><div id="sec-5" class="subsection"><h4>Informative SNPs in common laboratory mouse strains:</h4><p id="p-18">To boost informativeness of the array for laboratory stocks not represented in the Sanger Mouse Genomes Project, we included SNPs from two sources: MDA, and resequencing of selection lines derived from a common outbred stock. First, 13,036 additional MDA probes informative among laboratory mice were carried over to GigaMUGA.</p><p id="p-19">Second, SNPs were ascertained from whole-genome sequencing (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-15.gif"/></noscript></span></span>×) of five selection lines (the “high-runner” or HR lines) derived from the ICR:Hsd outbred stock (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Swallow <em>et al.</em> 1998</a>). This stock has a similar genetic background to a group of commonly used laboratory strains (so-called “Swiss mice”) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beck <em>et al.</em> 2000</a>). Briefly, reads from one individual from each of the five selection lines were aligned to the mouse reference genome (mm9/GRCm37 build) using bowtie2 v2.2.3 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Langmead and Salzberg 2012</a>) with default options. Suspected PCR duplicates were removed using Picard v1.88 (<a href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a>). SNPs were called using samtools mpileup v0.1.19-44428cd (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>) and filtered against the Sanger Mouse Genomes Project variant catalog. We targeted the resulting novel SNPs for inclusion on GigaMUGA if they met several additional criteria: not present on the MegaMUGA array, polymorphic in the five HR samples, and located in regions of low marker density on MegaMUGA array, but high SNP density in the five HR samples. A total of 3693 SNPs from the HR lines was included on the final array.</p></div><div id="sec-6" class="subsection"><h4>Informative SNPs between closely related strains:</h4><p id="p-20">To increase the value of GigaMUGA as a tool for discriminating between closely related inbred strains, we used data from MegaMUGA, MDA, and the Sanger Mouse Genomes Project to identify variants segregating between substrains. We included all 139 MegaMUGA probes discriminating between substrains of C57BL/6, and designed probes for an additional 251 variants between C57BL/6J and C57BL/6NJ ascertained by the Sanger Mouse Genomes Project. MDA data were used to select 540 variants useful for discriminating between several other substrain pairs: 129S1/SvImJ <em>vs.</em> 129S6/SvEvTac (221), A/J <em>vs.</em> A/WySnJ (148), AEJ/GnLeJ <em>vs.</em> AEJ/GnRk (31), BALB/cJ <em>vs.</em> BALB/cByJ (105), C3H/HeJ <em>vs.</em> C3HeB/FeJ (96), DBA/1J <em>vs.</em> DBA/1LacJ (20), DBA/2J <em>vs.</em> DBA/2DeJ (161), SEC/1GnLeJ <em>vs.</em> SEC/1ReJ (13), and SJL/Bm <em>vs.</em> SJL/J (8). These markers were selected to cover the genome uniformly. In some genomic regions for some strain pairs, many additional markers will be informative. Variation in these regions is not due to mutation and drift since the establishment of the lines, but was either segregating in the ancestors of the inbred line, or is due to contamination from other laboratory stocks.</p></div><div id="sec-7" class="subsection"><h4>Informative SNPs in wild mice:</h4><p id="p-21">To facilitate studies of wild mice, we included SNPs informative for subspecies of origin. Our goal was to achieve a density of at least one “diagnostic marker” per 300 kb for each subspecies, and to place at least one diagnostic marker for each subspecies within each recombination of the intervals identified in <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> (2014)</a>. We identified diagnostic markers based on a cohort of wild mice genotyped on the MDA (J. P. Didion, unpublished data) using the method of <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> (2011)</a>. We used a hidden Markov model (HMM) to assign each region of the genome within each individual to one of the three <em>M. musculus</em> subspecies using a panel of reference samples of known pure ancestry. We then computed the allele frequency at each MDA marker within each subspecies. Every marker with an allele exclusive to a single subspecies (allowing up to two mismatches) was considered diagnostic for that subspecies.</p><p id="p-22">We next identified regions of the genome in which marker density was lower than <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-16.gif"/></noscript></span></span> kb. Within each region, and within each subspecies having less than the required marker density, we performed an iterative search for diagnostic markers using a progressively decreasing minor-allele frequency (MAF) threshold (from <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-17.gif"/></noscript></span></span> in steps of 0.05). At each step, we identified all markers with a MAF greater than the threshold, and with as uniform spacing as possible. We next identified recombination intervals that still lacked at least one diagnostic marker for each subspecies, and attempted to select a diagnostic marker at random, if one was available. A total of 12,489 MDA probes was selected for GigaMUGA using this scheme.</p><p id="p-23">In addition, we designed probes for 7748 SNPs ascertained by whole-genome sequencing of two wild-caught <em>M. m. domesticus</em> mice (one from eastern Spain, and one from northern Italy), and two wild-derived inbred strains of <em>M. m. domesticus</em> ancestry (ZALENDE/EiJ and LEWES/EiJ). Our goal was to identify SNPs in these mice that had not been discovered in the 18 strains sequenced as part of the Sanger Mouse Genomes Project. Briefly, reads were aligned to the mm9 (GRCm37) reference genome using bwa 0.6.2-r126 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Durbin 2009</a>), and local realignment around indels was performed with the Genome Analysis Toolkit (GATK) IndelRealigner v2.4-7-g5e89f01 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McKenna <em>et al.</em> 2010</a>). SNPs were called using samtools mpileup v0.1.19-44428cd, and putative variants were filtered (by position only) against dbSNP and the Mouse Genomes Project variant catalog. We then attempted to place three novel SNPs within each 1-Mb window along the genome (two transitions and one transversion), selected at random from all the novel SNPs in that region. We attempted to space them evenly by placing one transition each in the first and second 500-kb windows of each 1-Mb region, and the transversion within the middle 333 kb. We favored SNPs with higher MAF within the four wild mice. We avoided placing SNPs closer than 100 kb apart unless that was the only option for the 1-Mb window.</p></div><div id="sec-8" class="subsection"><h4>Copy-number probes:</h4><p id="p-24">Copy-number variants in laboratory mouse strains are clustered near tracts of large (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-18.gif"/></noscript></span></span> kb) tandem segmental duplications (SDs) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">She <em>et al.</em> 2008</a>). Several groups, including ours, have recognized that SDs are a source of recurrent <em>de novo</em> structural variation in mouse (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Egan <em>et al.</em> 2007</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2014</a>). Most SD-rich regions of the mouse genome are also “cold regions” for meiotic recombination, and we have hypothesized that these patterns are causally related (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2014</a>).</p><p id="p-25">Although not optimized for detecting copy-number changes in the same manner as tiling arrays (aCGH), hybridization intensity on SNP arrays can capture the signal of aberrant copy number. Increased (or decreased) copy number of a genomic region should result in higher (lower) hybridization intensity at SNPs within the region. Although signal from a single SNP probe is noisy [and may be confounded by off-target variation in or near the probe sequence (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Didion <em>et al.</em> 2012</a>)], the aggregate signal across many consecutive probes is informative (see, for example, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Crowley <em>et al.</em> 2015</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Didion <em>et al.</em> 2015</a>).</p><p id="p-26">We designed a subset of 2006 probes to detect CNVs in 22 SD-rich cold regions described in <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> (2014)</a>. First the genomic sequence (from the mm10/GRCm38 reference assembly) for each of 59 target regions was extracted and aligned to itself using lastz (<a href="http://www.bx.psu.edu/~rsharris/lastz/">http://www.bx.psu.edu/~rsharris/lastz/</a>). Segmentally duplicated intervals were identified as intervals of self-similarity (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-19.gif"/></noscript></span></span>) longer than 10 kb. Every such interval is, by definition, present more than once; we retained the interval with the smallest genomic coordinate as the unique representative of that sequence. Because the Illumina postprocessing software is optimized for probes with signal from two alleles (an <em>x</em>- and <em>y</em>-coordinate), we next identified paralogous SNPs (positions that vary between copies of a duplicated sequence on the same chromosome) within the duplicated intervals. Using samtools mpileup on BAM files from the Sanger Mouse Genomes Project, we identified paralogous SNPs as any positions with evidence for both pseudoheterozygosity (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-20.gif"/></noscript></span></span> reads containing each of two or more bases), and excess coverage (read depth <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-21.gif"/></noscript></span></span>). Probe sequences were designed as 50-mers extending upstream from (or downstream from the reverse complement of) each paralogous SNP located <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-22.gif"/></noscript></span></span> bp away from another paralogous SNP. Of 2338 such candidate probes, 2006 were successfully fabricated on the array.</p></div><div id="sec-9" class="subsection"><h4>Probes for complement cascade genes:</h4><p id="p-27">Putative functional SNPs in 25 genes in the complement cascade (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>) were targeted as follows. First we identified biallelic variants in transcribed regions of the 25 target genes that are segregating in the eight founder strains of the CC using data from the Sanger Mouse Genomes Project. Variants within 50 bp of another variant were filtered. Probes were designed against the resulting 803 variants; of these, 105 were included on the final array.</p></div><div id="sec-10" class="subsection"><h4>Probes for genetically engineered constructs:</h4><p id="p-28">To increase the utility of GigaMUGA for verifying the integrity of genetically engineered mice, a set of 87 probes was carried over from the MegaMUGA array. These were designed to assay the presence or absence of a variety of transgenes and other exogenous constructs including the Cre and iCre recombinases; reporters such as LacZ and GFP; the CMV, SV40, and rabbit β-globin promoter sequences; and resistance cassettes to tetracycline, chloramphenicol, neomycin, puromycin, hygromycin, and ampicillin. Probe sequences were designed against 51 bp of known construct sequence; alternate alleles were arbitrarily selected and are not informative. Of this group of probes, 79 were successfully fabricated on the final array.</p></div><div id="sec-11" class="subsection"><h4>Genomic annotation:</h4><p id="p-29">Genomic positions were assigned for all markers on the array by mapping the final manufactured probe sequences, excluding the terminal polymorphic position, to the mouse reference genome (mm10/GRCm38 build) with bwa mem v0.7.12 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li 2013</a>) using default parameters. The annotated position for a marker is the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-23.gif"/></noscript></span></span>(coordinate of the <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-24.gif"/></noscript></span></span> aligned end of the probe sequence), on the aligned strand. For probes that align equally well to multiple positions, a position was chosen at random. Markers whose probe sequence did not align to the reference genome were assigned a missing value for chromosome and a position of 0. Markers coincident with known SNPs from the Sanger Mouse Genomes Project were identified using bedtools intersect v2.22.1 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Quinlan and Hall 2010</a>) and annotated with an rsID if available.</p></div></div><div id="sec-12" class="subsection"><h3>Reference samples</h3><p id="p-30">A diverse panel of 522 samples was chosen for calibrating and evaluating the performance of the array. These included 49 classical laboratory strains, 12 wild-derived strains, 53 <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-25.gif"/></noscript></span></span> hybrids between inbred strains, 62 <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-26.gif"/></noscript></span></span> hybrids between lines from the CC, 100 individuals from the DO, 29 wild-caught <em>M. musculus</em> specimens, and 20 specimens from other <em>Mus</em> species. Because the array was designed to be maximally informative in the CC and DO, we included in our reference panel eight technical replicates (corresponding to at least three biological replicates) for each of the eight founder strains of the CC. All reference samples are listed in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>.</p><p id="p-31">The method of DNA preparation is indicated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>. DNA stocks for most classical inbred strains were purchased from the Jackson Laboratory (“Jax”). High-molecular-weight DNA (“HMW”) from most <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-27.gif"/></noscript></span></span> hybrids and wild-caught specimens was extracted from tissues using a standard phenol-chloroform method (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sambrook and Russell 2006</a>). DNA from most other samples was prepared from tail clips or spleens using the Qiagen DNeasy Blood &amp; Tissue Kit (catalog no. 69506; Qiagen, Hilden, Germany) (“Qiagen”). DNAs donated by other laboratories are listed as “external.”</p><p id="p-32">Samples indicated as “SGCF” in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a> were processed by the UNC Systems Genetics Core Facility. The UNC SGCF service includes DNA extraction from tissue samples; preparation of DNAs for shipment to Neogen Inc.; data processing and storage; and consultation on interpretation of genotype data.</p></div><div id="sec-13" class="subsection"><h3>Array hybridization and genotype calling</h3><p id="p-33">Approximately <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-28.gif"/></noscript></span></span> genomic DNA per sample was shipped to Neogen Inc. (Lincoln, NE) for array hybridization. Genotypes were called jointly for all reference samples using the GenCall algorithm implemented in the Illumina BeadStudio software.</p></div><div id="sec-14" class="subsection"><h3>Quality control</h3><p id="p-34">Arrays were subject to three quality checks before further analysis: (1) distribution of total hybridization intensity; (2) total number of missing and heterozygous calls; and (3) concordance between known sex of each sample and calls on the sex chromosomes.</p><div id="sec-15" class="subsection"><h4>(1) Hybridization intensity:</h4><p id="p-35">Let <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-30.gif"/></noscript></span></span> be the raw hybridization intensity values for the reference and alternate alleles, respectively, within a hybridization batch. Illumina’s normalization procedure transforms <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-32.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-33.gif"/></noscript></span></span> and the two homozygous clusters lay along the axes of a two-dimensional coordinate plane (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Peiffer <em>et al.</em> 2006</a>). Our group has anecdotally observed that, within an array, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-34.gif"/></noscript></span></span> is a slightly better measure of total intensity than <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-35.gif"/></noscript></span></span>. (<em>R</em> overestimates intensity in highly heterozygous samples because, by the triangle inequality, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-36.gif"/></noscript></span></span>.) The distribution of within-array mean and standard deviation of <em>d</em> across 522 arrays is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quality checks for GigaMUGA arrays. Distribution of per-array mean (A) and standard deviation (B) of total hybridization intensity d (see Materials and Methods). (C) Examples of the distribution of d within single arrays. From left to right: a high-quality array (an inbred C3H/HeJ mouse), with approximately symmetric distribution and mean near 1.0; a failed array (a Diversity Outbred mouse), with right-skewed distribution; and a high-quality array for a genetically divergent individual (species Mus spretus), whose distribution is a mixture of a symmetric component and a spike near zero. (D) Cumulative distribution of Kolmogorov-Smirnov statistic (K) for departure from the expected  distribution of d. (E) Count of missing calls vs. heterozygous calls for reference samples, by sample group (see Table S1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Quality checks for GigaMUGA arrays. Distribution of per-array mean (A) and standard deviation (B) of total hybridization intensity &lt;em&gt;d&lt;/em&gt; (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (C) Examples of the distribution of &lt;em&gt;d&lt;/em&gt; within single arrays. From left to right: a high-quality array (an inbred C3H/HeJ mouse), with approximately symmetric distribution and mean near 1.0; a failed array (a Diversity Outbred mouse), with right-skewed distribution; and a high-quality array for a genetically divergent individual (species &lt;em&gt;Mus spretus&lt;/em&gt;), whose distribution is a mixture of a symmetric component and a spike near zero. (D) Cumulative distribution of Kolmogorov-Smirnov statistic (&lt;em&gt;K&lt;/em&gt;) for departure from the expected &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; distribution of &lt;em&gt;d&lt;/em&gt;. (E) Count of missing calls &lt;em&gt;vs.&lt;/em&gt; heterozygous calls for reference samples, by sample group (see Table S1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F1.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/2/263/F1.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467675" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-36" class="first-child">Quality checks for GigaMUGA arrays. Distribution of per-array mean (A) and standard deviation (B) of total hybridization intensity <em>d</em> (see <em>Materials and Methods</em>). (C) Examples of the distribution of <em>d</em> within single arrays. From left to right: a high-quality array (an inbred C3H/HeJ mouse), with approximately symmetric distribution and mean near 1.0; a failed array (a Diversity Outbred mouse), with right-skewed distribution; and a high-quality array for a genetically divergent individual (species <em>Mus spretus</em>), whose distribution is a mixture of a symmetric component and a spike near zero. (D) Cumulative distribution of Kolmogorov-Smirnov statistic (<em>K</em>) for departure from the expected <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F1/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F1/embed/mml-math-37.gif"/></noscript></span></span> distribution of <em>d</em>. (E) Count of missing calls <em>vs.</em> heterozygous calls for reference samples, by sample group (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The distribution of <em>d</em> within an array is an important indicator of genotyping quality. We recognize three general patterns (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a>). For successful arrays (left panel), <em>d</em> has an approximately symmetric distribution, with mean 0.97 and standard deviation 0.42. A distribution of <em>d</em> skewed toward low values (middle panel) is associated with a high proportion of missing genotype calls, and indicates a failed array (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Didion <em>et al.</em> 2014</a>). Finally, the distribution of <em>d</em> for samples that are diverged from the mouse reference genome (right panel) is a mixture of a symmetric distribution, with mean near 1, and a spike near 0. This spike represents a population of probes whose hybridization is disrupted by off-target variants within the probe sequence (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Didion <em>et al.</em> 2012</a>).</p><p id="p-38">Based on these observations, we computed the Kolmogorov-Smirnov statistic <em>K</em> for difference in the distribution of <em>d</em> from <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-38.gif"/></noscript></span></span> for each sample and flagged 18 samples at an empirically defined threshold of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-39.gif"/></noscript></span></span> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>).</p></div><div id="sec-16" class="subsection"><h4>(2) Call rate:</h4><p id="p-39">We inspected the rate of missing and heterozygous calls within groups of reference samples to establish group-specific thresholds (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/FigureS1.pdf">Figure S1</a>). A set of 12 <em>Mus musculus</em> samples with <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-40.gif"/></noscript></span></span> missing calls, and samples of other <em>Mus</em> species with <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-41.gif"/></noscript></span></span> missing calls, were flagged. An additional four samples from classical inbred strains with <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-42.gif"/></noscript></span></span> heterozygous calls were flagged.</p></div><div id="sec-17" class="subsection"><h4>(3) Concordance for sex chromosomes:</h4><p id="p-40">Female samples should have zero nonmissing calls at truly Y-linked markers, while males should be hemizygous. We counted the number of nonmissing, nonheterozygous calls at markers nominally mapped to the Y chromosome among samples known to be female (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-43.gif"/></noscript></span></span>, median <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-44.gif"/></noscript></span></span> MAD; maximum 33), or male (<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-45.gif"/></noscript></span></span>; minimum 42). Four samples fell in the ambiguous range (more than 33 but less than 42 good calls at Y-chromosome markers), but all were from other <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-46.gif"/></noscript></span></span> species. We computed the mean value of <em>d</em> (total intensity) at probes on the X chromosome within each sample as an additional check for sex-chromosome concordance. Female samples should have higher hybridization intensity on the X since they have two copies. On the basis of X-chromosome intensity, the four ambiguous samples were confirmed to be male. A visual summary of the sex-chromosome analyses is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/FigureS2.pdf">Figure S2</a>.</p><p id="p-41">In total, 22 samples failed one or more quality filters (marked as “FAIL” in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>), leaving a final set of 500 reference samples (marked as “PASS”), which was used in subsequent analyses.</p></div></div><div id="sec-18" class="subsection"><h3>Normalization</h3><p id="p-42">We transformed <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-47.gif"/></noscript></span></span> to sum intensity <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-48.gif"/></noscript></span></span>, and angle <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-49.gif"/></noscript></span></span>. (As noted above, <em>d</em> is a slightly better estimate of total intensity that <em>R</em>, but we use <em>R</em> for consistency with published methods.) We then computed the log2(intensity ratio) (LRR) and B-allele frequency (BAF) transformations defined in <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Peiffer <em>et al.</em> (2006)</a> using a modified form of the Illumina-specific thresholded quantile normalization (tQN) approach proposed by <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Staaf <em>et al.</em> (2008)</a>. These normalization procedures require precomputed centroids for each of the three canonical genotype clusters (AA, AB, BB) at each marker. We estimated these centroids as the trimmed mean (omitting the most extreme <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-50.gif"/></noscript></span></span> of values) of <em>R</em> and θ among samples called AA, AB or BB at each marker.</p></div><div id="sec-19" class="subsection"><h3>Identification of multiallelic probes</h3><p id="p-43">The number of clusters (in the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-51.gif"/></noscript></span></span>-plane) for each probe was determined using a nonparametric method that leverages parent–offspring trios (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kao <em>et al.</em> 2014</a>). Briefly, the algorithm proceeds in two steps: first, samples from the eight founder strains of the CC are used to identify clusters representing homozygous states. These clusters are iteratively merged using a <em>k</em>-nearest-neighbor approach. Second, samples from each of the <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-52.gif"/></noscript></span></span> possible <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-53.gif"/></noscript></span></span> genotypes are assigned either to a new cluster or to an existing cluster, depending on the cluster assignment of their respective parents. The <em>k</em>-nearest-neighbor merging procedure is repeated to yield a final set of clusters for each marker.</p></div><div id="sec-20" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-44">We assessed the phylogenetic information content of GigaMUGA on the male-specific portion of the Y chromosome and the mitochondrial genome. These sequences are commonly used for phylogenetic analyses because they are both hemizygous and nonrecombining, and because each provides complementary insight into ancestry and demographic history. A set of 67 male samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>) was selected to span the three principal subspecies of <em>M. musculus</em>, including wild, wild-derived, and classical laboratory mice, plus the outgroup species <em>Mus spretus</em>. Genotype calls at 83 Y-chromosome markers, and 32 mitochondrial markers, were recoded to capture information from probes with aberrant hybridization patterns due to off-target variation in or near the probe sequence [“variable-intensity oligonucleotides”, or VINOs; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Didion <em>et al.</em> (2012)</a>]. At each marker, heterozygous calls and no-calls were assigned random nonallelic nucleotides: for instance, at a [T/G] SNP, a heterozygous call might be assigned A, and no-call might be assigned C. A parsimony tree was inferred separately for the resulting Y-chromosome and mitochondrial genotype matrices with RAxML v8.1.9 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Stamatakis 2014</a>). Although the topology of these trees is likely to be meaningful, branch lengths are distorted by ascertainment bias in the SNP panel. The trees in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> are plotted with uniform branch lengths.</p></div><div id="sec-21" class="subsection"><h3>Inspection of B6.PL-<em>Thy1<sup>a</sup></em>/CyJ congenic line</h3><p id="p-45">The genetic background of the B6.PL-<em>Thy1<sup>a</sup></em>/CyJ line (JAX stock number 000406) was investigated using a single male sample. This line carries a <em>Thy1</em> allele from PL/J in a C57BL/6 background. We used a HMM to reconstruct that sample’s genome as a mosaic of contributions from C57BL/6J, C57BL/6NJ, C57BL/6CR (Charles River), C57BL/6Tc (Taconic), C57BL/10ScN, and NON/ShiLtJ. The PL/J strain was not included in our set of reference samples, so we chose NON/ShiLtJ as a surrogate because it shares most of the interval around <em>Thy1</em> identical-by-descent with PL/J (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2011</a>). We found that, although the HMM could easily identify contributions from non-J substrains of C57BL/6, it could not robustly discriminate between the several non-J substrains (owing to the paucity of informative markers in these comparisons, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Intervals consistent with C57BL/6 ancestry for which C57BL/6J can be ruled out as the donor were therefore simply labeled “non-C57BL/6J.”</p></div><div id="sec-22" class="subsection"><h3>Performance of probes for genetically engineered constructs</h3><p id="p-46">To test the performance of the assays tracking the presence of genetically engineered constructs, we used 587 mouse samples that have been genotyped on the MegaMUGA platform, representing both samples known or presumed to carry at least one of the constructs and samples known to be devoid of them. Cluster plots of the raw <em>x</em>- and <em>y</em>-intensities for all 83 constructed-targeted probes on MegaMUGA were manually inspected. A subset of 38 was designated as informative on the basis of clustering patterns: samples known or presumed to carry the targeted construct had relatively high raw intensity on the expected axis (the allele corresponding to the true sequence of the construct), while negative control samples had low raw intensity. The 38 markers were grouped according to the targeted construct. Within each target, raw intensity (again, along only the informative axis) was summed across probes and a two-component (absence <em>vs.</em> presence) Gaussian mixture model was fit to the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-54.gif"/></noscript></span></span> sum intensities using the R package mclust (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fraley <em>et al.</em> 2012</a>). A table of probe IDs, targets and informative alleles is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS3.xlsx">Table S3</a>.</p></div><div id="sec-23" class="subsection"><h3>Data availability</h3><p id="p-47">Genotype calls and hybridization intensity data (both raw and processed) for 522 reference samples are available for download from <a href="http://csbio.unc.edu/MUGA">http://csbio.unc.edu/MUGA</a>. Routines for quality checks and intensity normalization are implemented in the R package argyle, described elsewhere (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Morgan 2016</a>, this issue), and available for download from GitHub (<a href="https://github.com/andrewparkermorgan/argyle">https://github.com/andrewparkermorgan/argyle</a>).</p><p id="p-48">The GigaMUGA genotyping service is provided exclusively by Neogen Inc., Lincoln, NE. Users may provide samples as tissues or DNA aliquots. Data are returned in Illumina BeadStudio format via a secure file transfer. The University of North Carolina Systems Genetics Core Facility offers sample preparation, shipment to Neogen, and postprocessing of data to both internal and external users.</p><p id="p-49">Annotation files for the MUGA family of arrays are available from <a href="http://csbio.unc.edu/MUGA">http://csbio.unc.edu/MUGA</a>.</p></div></div><div class="section results-discussion" id="sec-24"><h2 class="">Results and Discussion</h2><p id="p-50">The final GigaMUGA array comprises 143,259 probes distributed across all 19 mouse autosomes, the X- and Y-chromosomes, and the mitochondrial genome. Of these, 67,645 (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-55.gif"/></noscript></span></span>) were carried over from MegaMUGA. The vast majority of probes (141,090; <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-56.gif"/></noscript></span></span>) are designed to interrogate biallelic SNPs, with the remainder designed to assay copy number (2006; <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-57.gif"/></noscript></span></span>), multiallelic loci (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-58.gif"/></noscript></span></span>), or the presence of engineered constructs (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-59.gif"/></noscript></span></span>). We classified probes into nine types (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) based on the types of variants they target, and how they were ascertained.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467449/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467449/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Probe types on GigaMUGA</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">The genomic distribution of SNP and copy-number probes is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. SNP probes are tiled along the autosomes every <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-60.gif"/></noscript></span></span> kb (median <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-61.gif"/></noscript></span></span> 1 median absolute deviation) or every <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-62.gif"/></noscript></span></span> cM, and every <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-63.gif"/></noscript></span></span> kb (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-64.gif"/></noscript></span></span> cM) on the X-chromosome. The nonrecombining Y-chromosome and mitochondrial genome are tagged with 83 and 32 probes, respectively. Because recombination is enriched in subtelomeric regions in mouse (<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2014</a>), the density of probes is higher at the distal ends of the autosomes than at the proximal ends. A final annotated array manifest is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS2.zip">Table S2</a>. The performance of GigaMUGA was assessed in a panel of 522 reference samples, of which 500 passed quality controls. All reference samples are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic distribution of GigaMUGA probes. (A) Marker density across the autosomes and X-chromosome, plotted in 500-kb bins. Fill color indicates probe type. Individual markers are shown for the Y-chromosome and mitochondrial genome in the insets at right. (B) Relative representation of markers in all but the largest two classes (&#x201C;haplotype discrimination&#x201D; and &#x201C;recombination hotspot&#x201D;). (C) Distribution of marker density, in markers per Mb, for probes targeting biallelic SNPs (see Materials and Methods ). Dot indicates median, dark bar  percentile, light bar  percentile. (D) Distribution of physical distance between adjacent marker pairs. (E) Distribution of genetic distance between adjacent marker pairs, calculated by linear interpolation on the genetic map of Liu et al. (2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Genomic distribution of GigaMUGA probes. (A) Marker density across the autosomes and X-chromosome, plotted in 500-kb bins. Fill color indicates probe type. Individual markers are shown for the Y-chromosome and mitochondrial genome in the insets at right. (B) Relative representation of markers in all but the largest two classes (&#x201C;haplotype discrimination&#x201D; and &#x201C;recombination hotspot&#x201D;). (C) Distribution of marker density, in markers per Mb, for probes targeting biallelic SNPs (see &lt;em&gt;Materials and Methods&lt;/em&gt; ). Dot indicates median, dark bar &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-65&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; percentile, light bar &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-66&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; percentile. (D) Distribution of physical distance between adjacent marker pairs. (E) Distribution of genetic distance between adjacent marker pairs, calculated by linear interpolation on the genetic map of Liu &lt;em&gt;et al.&lt;/em&gt; (2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F2.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/2/263/F2.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467470" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-52" class="first-child">Genomic distribution of GigaMUGA probes. (A) Marker density across the autosomes and X-chromosome, plotted in 500-kb bins. Fill color indicates probe type. Individual markers are shown for the Y-chromosome and mitochondrial genome in the insets at right. (B) Relative representation of markers in all but the largest two classes (“haplotype discrimination” and “recombination hotspot”). (C) Distribution of marker density, in markers per Mb, for probes targeting biallelic SNPs (see <em>Materials and Methods</em> ). Dot indicates median, dark bar <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F2/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F2/embed/mml-math-65.gif"/></noscript></span></span> percentile, light bar <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F2/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F2/embed/mml-math-66.gif"/></noscript></span></span> percentile. (D) Distribution of physical distance between adjacent marker pairs. (E) Distribution of genetic distance between adjacent marker pairs, calculated by linear interpolation on the genetic map of <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> (2014)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-25" class="subsection"><h3>Assignment of probes to quality tiers</h3><p>Probes were assigned to four (mutually exclusive) tiers of decreasing quality based on their performance as biallelic SNP markers in the set of reference samples, using the following criteria. We denote genotype calls as “AA”, homozygous for the reference (C57BL/6J) allele; “BB”, homozygous for the alternate allele; “AB”, heterozygous; and “N”, no-call (missing).</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-54">Tier 1:<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-67.gif"/></noscript></span></span> sample called each of AA, BB and AB, with no-call rate <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-68.gif"/></noscript></span></span></p></li><li id="list-item-2"><p id="p-55">Tier 2: all probes not in Tier 1, with <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-69.gif"/></noscript></span></span> sample called each of AA and BB, with no-call rate <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-70.gif"/></noscript></span></span></p></li><li id="list-item-3"><p id="p-56">Tier 3: all probes not in Tiers 1 or 2, with no-call rate <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-71.gif"/></noscript></span></span></p></li><li id="list-item-4"><p id="p-57">Tier 4: all remaining probes</p></li></ul><p>These definitions are motivated by the observation that the Illumina intensity-normalization and genotype-calling algorithms perform best when all three genotype states (AA, BB, AB) are present for each probe. However, assignment of probes to quality tiers is dependent on the composition of the set of reference samples: markers with low expected minor-allele frequency are unlikely to be represented in both homozygous states. Because both the content of the array, and the composition of the reference sample set are biased toward genetic backgrounds represented in common laboratory strains and the CC, quality tiers are particularly relevant to users of those and other common laboratory mouse strains. Users applying the array in other populations, such as wild-caught mice, should verify that probes perform as expected in their populations of interest. (We note that probes in lower quality tiers still provide information if treated as multiallelic markers and/or copy-number probes.) Assignments are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467604/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467604/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Allocation of probes to quality tiers</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Genotype call rate and concordance between replicates</h3><p id="p-58">Among probes in tiers 1–3, the rate of nonmissing genotype calls is <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-72.gif"/></noscript></span></span> (mean <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-73.gif"/></noscript></span></span> standard deviation). The rate of concordance between 33 biological replicates of inbred strains is <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-74.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-75.gif"/></noscript></span></span> in 79 biological replicates of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-76.gif"/></noscript></span></span> hybrids (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Concordance between the observed autosomal genotypes in <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-77.gif"/></noscript></span></span> hybrids and the predicted genotypes based on parental strains is somewhat lower at <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-78.gif"/></noscript></span></span>. This decrease is due almost entirely to <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-79.gif"/></noscript></span></span>s for which one parent is a wild-derived strain (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), and is therefore likely attributable to off-target variation in or near probe sequences in those strains (VINOs). VINOs are especially difficult to genotype in the heterozygous state (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Didion <em>et al.</em> 2012</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance in genotype calls. (A) Concordance between biological replicates of the eight founder strains of the Collaborative Cross (on the diagonal), and  hybrids between them (off the diagonal.) Maternal strain is indicated on the vertical axis, and paternal strain on the horizontal axis. Strain names are abbreviated as: A, A/J; B, C57BL/6J; C, 129S1/SvImJ; D, NOD/ShiLtJ; E, NZO/HlLtJ; F, CAST/EiJ; G, PWK/PhJ; H, WSB/EiJ. Blank cells indicate missing  combinations. Note that the EF and EG crosses do not produce viable offspring. (B) Concordance between observed genotypes in  hybrids and predicted genotype based on genotypes of the parental strains. Note the difference in color scale between panels. Gray cells indicate homozygous genotypes which are omitted from this analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Concordance in genotype calls. (A) Concordance between biological replicates of the eight founder strains of the Collaborative Cross (on the diagonal), and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-80&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; hybrids between them (off the diagonal.) Maternal strain is indicated on the vertical axis, and paternal strain on the horizontal axis. Strain names are abbreviated as: A, A/J; B, C57BL/6J; C, 129S1/SvImJ; D, NOD/ShiLtJ; E, NZO/HlLtJ; F, CAST/EiJ; G, PWK/PhJ; H, WSB/EiJ. Blank cells indicate missing &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-81&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; combinations. Note that the E&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-82&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;F and E&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-83&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;G crosses do not produce viable offspring. (B) Concordance between observed genotypes in &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-84&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; hybrids and predicted genotype based on genotypes of the parental strains. Note the difference in color scale between panels. Gray cells indicate homozygous genotypes which are omitted from this analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F3.medium.gif" width="314" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/2/263/F3.medium.gif" width="314" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467599" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-59" class="first-child">Concordance in genotype calls. (A) Concordance between biological replicates of the eight founder strains of the Collaborative Cross (on the diagonal), and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-80.gif"/></noscript></span></span> hybrids between them (off the diagonal.) Maternal strain is indicated on the vertical axis, and paternal strain on the horizontal axis. Strain names are abbreviated as: A, A/J; B, C57BL/6J; C, 129S1/SvImJ; D, NOD/ShiLtJ; E, NZO/HlLtJ; F, CAST/EiJ; G, PWK/PhJ; H, WSB/EiJ. Blank cells indicate missing <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-81.gif"/></noscript></span></span> combinations. Note that the E<span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-82.gif"/></noscript></span></span>F and E<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-83.gif"/></noscript></span></span>G crosses do not produce viable offspring. (B) Concordance between observed genotypes in <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F3/embed/mml-math-84.gif"/></noscript></span></span> hybrids and predicted genotype based on genotypes of the parental strains. Note the difference in color scale between panels. Gray cells indicate homozygous genotypes which are omitted from this analysis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-27" class="subsection"><h3>Multiallelic probes</h3><p id="p-60">Although most probes on the array were designed to behave as biallelic SNPs, we and others have observed that off-target variation in or near the probe sequence creates aberrant hybridization patterns that function as additional alleles or additional partially-informative markers (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Didion <em>et al.</em> 2012</a>). Distinguishing VINOs from sporadic no-calls requires a panel of training samples that includes replicates of all homozygous and heterozygous genotypes at each marker. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows an example of a standard biallelic probe with three clusters, and a mutiallelic probe with six clusters (representing three homozygous states and the corresponding three heterozygous combinations). We used a panel of 170 reference samples covering all 36 possible genotypes in the CC and DO to determine the number of clusters for each probe on GigaMUGA (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Although probes with three clusters in the CC—that is, probes that behave as biallelic SNPs—are the largest class among probe types designed to assay SNPs, additional alleles can be distinguished for 36,615 (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-85.gif"/></noscript></span></span>). In the remainder of this report, we treat SNP probes as biallelic. Although this does not bias our results or interpretations of the overall utility of GigaMUGA, it does entail some loss of information (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fu <em>et al.</em> 2012</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hybridization patterns at a multiallelic probe. (A) A probe with six distinct genotype clusters. Each point represents one sample; colored points are the Collaborative Cross founder strains and gray points are s between them. Although the standard calling algorithm would assign WSB/EiJ, PWK/PhJ and CAST/EiJ, and their heterozygous combinations to the same genotype, it is clear that WSB/EiJ and PWK/PhJ (cluster a) have a different allele than CAST/EiJ (cluster b). The remaining five strains form a single homozygous cluster (cluster c). Cluster d corresponds to the heterozygotes between clusters a and c, and cluster f the heterozygotes between clusters a and b. (B) For comparison, a probe that behaves as a standard biallelic SNP marker with three clusters, two homozygous (clusters a, c), and one heterozygous (cluster b)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Hybridization patterns at a multiallelic probe. (A) A probe with six distinct genotype clusters. Each point represents one sample; colored points are the Collaborative Cross founder strains and gray points are &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-86&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;s between them. Although the standard calling algorithm would assign WSB/EiJ, PWK/PhJ and CAST/EiJ, and their heterozygous combinations to the same genotype, it is clear that WSB/EiJ and PWK/PhJ (cluster a) have a different allele than CAST/EiJ (cluster b). The remaining five strains form a single homozygous cluster (cluster c). Cluster d corresponds to the heterozygotes between clusters a and c, and cluster f the heterozygotes between clusters a and b. (B) For comparison, a probe that behaves as a standard biallelic SNP marker with three clusters, two homozygous (clusters a, c), and one heterozygous (cluster b).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F4.medium.gif" width="440" height="162"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/2/263/F4.medium.gif" width="440" height="162"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467543" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-61" class="first-child">Hybridization patterns at a multiallelic probe. (A) A probe with six distinct genotype clusters. Each point represents one sample; colored points are the Collaborative Cross founder strains and gray points are <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F4/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F4/embed/mml-math-86.gif"/></noscript></span></span>s between them. Although the standard calling algorithm would assign WSB/EiJ, PWK/PhJ and CAST/EiJ, and their heterozygous combinations to the same genotype, it is clear that WSB/EiJ and PWK/PhJ (cluster a) have a different allele than CAST/EiJ (cluster b). The remaining five strains form a single homozygous cluster (cluster c). Cluster d corresponds to the heterozygotes between clusters a and c, and cluster f the heterozygotes between clusters a and b. (B) For comparison, a probe that behaves as a standard biallelic SNP marker with three clusters, two homozygous (clusters a, c), and one heterozygous (cluster b).</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467580/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467580/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of alleles per probe, by probe type</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Information content in laboratory populations</h3><p id="p-62">A key measure of the utility of a genotyping array for laboratory mice is the number of informative markers between commonly used inbred strains. We calculated the number of informative markers—markers in tiers 1 and 2 called for opposite homozygous genotypes in the members of a pair—between all pairs of 47 inbred strains (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). As expected, GigaMUGA is highly informative for the CC and DO, with a median of 50,285 markers expected to be segregating between any pair of CC founder strains. Although fewer markers (median 32,539) are informative between pairs of classical inbred strains, owing both to their shared ancestry and to our decisions about which SNPs to target, this number is still sufficient to achieve a density of <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-87.gif"/></noscript></span></span> SNP per 100 kb.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise informative markers between laboratory mouse strains. (A) Heatmap of the number of informative markers (SNP probes only; no special probes) between pairs of inbred strains. (B) Distribution of the number of informative markers among pairs chosen from different subsets of laboratory strains. (C) Markers informative in pairs of substrains. Each track shows the genomic position of markers informative between the two closely related inbred strains indicated at left. Points are colored as blue or gray on alternating chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pairwise informative markers between laboratory mouse strains. (A) Heatmap of the number of informative markers (SNP probes only; no special probes) between pairs of inbred strains. (B) Distribution of the number of informative markers among pairs chosen from different subsets of laboratory strains. (C) Markers informative in pairs of substrains. Each track shows the genomic position of markers informative between the two closely related inbred strains indicated at left. Points are colored as blue or gray on alternating chromosomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F5.medium.gif" width="281" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/2/263/F5.medium.gif" width="281" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467693" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-63" class="first-child">Pairwise informative markers between laboratory mouse strains. (A) Heatmap of the number of informative markers (SNP probes only; no special probes) between pairs of inbred strains. (B) Distribution of the number of informative markers among pairs chosen from different subsets of laboratory strains. (C) Markers informative in pairs of substrains. Each track shows the genomic position of markers informative between the two closely related inbred strains indicated at left. Points are colored as blue or gray on alternating chromosomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">An additional feature of GigaMUGA is its inclusion of probes for discriminating between substrains within several groups including the 129, BALB, C3H, C57BL/6, and DBA clusters. The genomic distribution of probes informative between selected substrains is shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, and corresponding counts in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>. For most substrain pairs, GigaMUGA provides markers on all autosomes, the X-chromosome, and the mitochondrial genome, at sufficient density to saturate the genome in a standard <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-88.gif"/></noscript></span></span> cross.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467670/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467670/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Number of informative markers between closely related strains</span><div class="sb-div caption-clear"></div></div></div><p id="p-65">The availability of informative markers between laboratory strains makes GigaMUGA a valuable tool for determining the components of the genetic background of laboratory stocks with substrain-level precision. Applications include verification of genetic background in knockout lines; precise characterization of congenic lines; and forensic examination of stocks or cell lines of unknown origin. As an example, we genotyped an individual from the B6.PL-<em>Thy1<sup>a</sup></em>/CyJ strain (JAX stock number 000406). This congenic strain carries a <em>Thy1</em> allele from PL/J (at chr9: 44 Mb) backcrossed into a C57BL/6 background. We confirmed the presence of a large PL/J segment on chromosome 9 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Our analysis further identified contamination most likely from C57BL/10J on proximal chromosome 11, and suggests that one or more other substrains of C57BL/6 in addition to C57BL/6J contributed to the genetic background.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Verification of genetic backgrounds in a congenic line. Strain contributions to the congenic strain B6.PL-Thy1a/CyJ were reconstructed from genotype calls using a hidden Markov model. A large PL/J region (green) containing the Thy1a allele was identified on chromosome 9, as expected. Some contribution from C57BL/10J (gold) and C57BL/6 substrains besides C57BL/6J (gray) was also discovered." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Verification of genetic backgrounds in a congenic line. Strain contributions to the congenic strain B6.PL-&lt;em&gt;Thy1&lt;sup&gt;a&lt;/sup&gt;&lt;/em&gt;/CyJ were reconstructed from genotype calls using a hidden Markov model. A large PL/J region (green) containing the &lt;em&gt;Thy1&lt;sup&gt;a&lt;/sup&gt;&lt;/em&gt; allele was identified on chromosome 9, as expected. Some contribution from C57BL/10J (gold) and C57BL/6 substrains besides C57BL/6J (gray) was also discovered.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F6.medium.gif" width="440" height="362"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/2/263/F6.medium.gif" width="440" height="362"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-66" class="first-child">Verification of genetic backgrounds in a congenic line. Strain contributions to the congenic strain B6.PL-<em>Thy1<sup>a</sup></em>/CyJ were reconstructed from genotype calls using a hidden Markov model. A large PL/J region (green) containing the <em>Thy1<sup>a</sup></em> allele was identified on chromosome 9, as expected. Some contribution from C57BL/10J (gold) and C57BL/6 substrains besides C57BL/6J (gray) was also discovered.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>Utility for population genetics and phylogeny</h3><p id="p-67">We define a “diagnostic marker” as a marker at which genotype is informative for ancestry at the level of subspecies. Following the approach described in <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> (2011)</a> we used 30 wild-caught or wild-derived samples (19 <em>M. m. domesticus</em>, six <em>M. m. musculus</em> and five <em>M. m. castaneus</em>) with known pure ancestry and broad geographic distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1/TableS1.xlsx">Table S1</a>) to identify 33,357 markers on the autosomes, X-chromosome, and mitochondrial genome at which the minor allele is present in only one subspecies. (We note that this definition is sensitive to the choice of reference samples, and to introgression: if any of the training samples carry an introgression tract, no diagnostic markers will be identified for the donor subspecies within that tract. We intend to revisit this problem with a more robust approach after more wild-caught training samples have been genotyped.) Because marker ascertainment was strongly biased toward SNPs segregating in <em>M. m. domesticus</em>, most diagnostic markers are diagnostic for <em>M. m. domesticus</em> (18,184), with fewer for <em>M. m. musculus</em> (7484) and <em>M. m. castaneus</em> (7689). <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a> demonstrates the ability of diagnostic SNPs on GigaMUGA to recover local ancestry in a region of chromosome 16 in which the CAST/EiJ strain was previously shown to have intersubspecific admixture (<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2011</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic information content of the GigaMUGA array. (A) Diagnostic alleles (blue, M. m. domesticus; red, M. m. musculus; green, M. m. castaneus) are shown as hash marks in a region of chromosome 16 where CAST/EiJ (top) has mixed ancestry. Pure representatives of each subspecies are shown for comparison. Smoothed ancestry blocks inferred using genotypes from the Mouse Diversity Array (Yang et al. 2011) are underlaid. (B) First two PCs from principal components analysis (PCA) of 20 wild M. musculus specimens at a randomly chosen subset of 3000 diagnostic markers (1000 per subspecies) on the autosomes. Individuals are colored according to their subspecies of origin, using the color scheme of panel A. (C) Phylogenetic tree constructed from 83 markers in the male-specific region of the Y chromosome in 67 male samples. Samples are colored according to their nominal subspecies or species of origin: blue, red and green as in panel A; maroon, M. m. molossinus (a M. m. musculus and M. m. castaneus hybrid); and gray, Mus spretus. See Materials and Methods for details of tree construction. Filled dots, wild-caught samples; open dots, inbred strains. (D) Phylogenetic tree for the same 67 male samples as in (C) but constructed from 32 mitochondrial markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic information content of the GigaMUGA array. (A) Diagnostic alleles (blue, &lt;em&gt;M. m. domesticus&lt;/em&gt;; red, &lt;em&gt;M. m. musculus&lt;/em&gt;; green, &lt;em&gt;M. m. castaneus&lt;/em&gt;) are shown as hash marks in a region of chromosome 16 where CAST/EiJ (top) has mixed ancestry. Pure representatives of each subspecies are shown for comparison. Smoothed ancestry blocks inferred using genotypes from the Mouse Diversity Array (Yang &lt;em&gt;et al.&lt;/em&gt; 2011) are underlaid. (B) First two PCs from principal components analysis (PCA) of 20 wild &lt;em&gt;M. musculus&lt;/em&gt; specimens at a randomly chosen subset of 3000 diagnostic markers (1000 per subspecies) on the autosomes. Individuals are colored according to their subspecies of origin, using the color scheme of panel A. (C) Phylogenetic tree constructed from 83 markers in the male-specific region of the Y chromosome in 67 male samples. Samples are colored according to their nominal subspecies or species of origin: blue, red and green as in panel A; maroon, &lt;em&gt;M. m. molossinus&lt;/em&gt; (a &lt;em&gt;M. m. musculus&lt;/em&gt; and &lt;em&gt;M. m. castaneus&lt;/em&gt; hybrid); and gray, &lt;em&gt;Mus spretus&lt;/em&gt;. See &lt;em&gt;Materials and Methods&lt;/em&gt; for details of tree construction. Filled dots, wild-caught samples; open dots, inbred strains. (D) Phylogenetic tree for the same 67 male samples as in (C) but constructed from 32 mitochondrial markers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F7.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/2/263/F7.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467462" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-68" class="first-child">Phylogenetic information content of the GigaMUGA array. (A) Diagnostic alleles (blue, <em>M. m. domesticus</em>; red, <em>M. m. musculus</em>; green, <em>M. m. castaneus</em>) are shown as hash marks in a region of chromosome 16 where CAST/EiJ (top) has mixed ancestry. Pure representatives of each subspecies are shown for comparison. Smoothed ancestry blocks inferred using genotypes from the Mouse Diversity Array (<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2011</a>) are underlaid. (B) First two PCs from principal components analysis (PCA) of 20 wild <em>M. musculus</em> specimens at a randomly chosen subset of 3000 diagnostic markers (1000 per subspecies) on the autosomes. Individuals are colored according to their subspecies of origin, using the color scheme of panel A. (C) Phylogenetic tree constructed from 83 markers in the male-specific region of the Y chromosome in 67 male samples. Samples are colored according to their nominal subspecies or species of origin: blue, red and green as in panel A; maroon, <em>M. m. molossinus</em> (a <em>M. m. musculus</em> and <em>M. m. castaneus</em> hybrid); and gray, <em>Mus spretus</em>. See <em>Materials and Methods</em> for details of tree construction. Filled dots, wild-caught samples; open dots, inbred strains. (D) Phylogenetic tree for the same 67 male samples as in (C) but constructed from 32 mitochondrial markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-69">To demonstrate the performance of GigaMUGA for phylogenetic studies in <em>M. musculus</em> and related species, we constructed trees using genotypes at 83 Y-chromosome probes and 32 mitochondrial probes. To mitigate ascertainment bias, we recoded genotypes as discrete characters based on clustering patterns (see <em>Materials and methods</em>) rather than using genotype calls directly. The resulting trees are shown in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7, C–D</a>. The Y-chromosome tree recovers known features of the patrilineal phylogeny of laboratory mice, including the presence of a <em>M. m. musculus</em> Y chromosome in most classical laboratory strains (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bishop <em>et al.</em> 1985</a>), and in CAST/EiJ (<a id="xref-ref-53-7" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2011</a>). The Y chromosome from wild pure <em>M. m. domesticus</em> constitutes a separate clade. The mitochondrial tree is concordant with the prior knowledge of the matrilineal phylogeny of house mice, separating the subspecies into monophyletic clades. It reveals evidence of intersubspecific hybridization in a wild sample trapped near the <em>musculus-domesticus</em> hybrid zone in Denmark (labeled “mus (DK)”): although most of its genome is of <em>M. m. musculus</em> origin, it has an <em>M. m. domesticus</em> mitochondrial genome.</p></div><div id="sec-30" class="subsection"><h3>Copy-number analyses</h3><p id="p-70">Hybridization-intensity signals from Illumina arrays have two components informative for copy number: total intensity (<em>R</em>) and relative intensity from the alternative <em>vs.</em> the reference allele (<em>θ</em>). These can be normalized within and between arrays (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Peiffer <em>et al.</em> 2006</a>) to give the “log2-intensity ratio” (LRR) and “B-allele frequency” (BAF) respectively. Copy-number variants cause deviations of LRR away from zero and (at heterozygous sites) of BAF away from 0.5.</p><p id="p-71">In addition to 141,090 SNP probes, GigaMUGA has 2006 copy-number probes, which are concentrated in segmentally duplicated regions of the mouse genome associated with recurrent structural mutations (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Egan <em>et al.</em> 2007</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">She <em>et al.</em> 2008</a>). To demonstrate the performance of GigaMUGA’s copy-number probes, we compared LRR and read depth from whole-genome sequencing in an interval on chromosome 6 (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>) containing a known CNV (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Keane <em>et al.</em> 2011</a>). C57BL/6NJ, a close substrain of the C57BL/6J reference, has normal LRR and read depth. The BALB/cJ strain has reduced LRR across the targeted region, consistent with a deletion, while NOD/ShiLtJ has increased LRR, consistent with a duplication (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>). The wild-derived strains LEWES/EiJ and WSB/EiJ appear to have normal diploid copy number. Inspection of read-depth profiles (panel B) confirms a large deletion in BALB/cJ and a large duplication in NOD/ShiLtJ, with a more complex pattern of small gains and losses in 129S1/SvImJ, LEWES/EiJ and WSB/EiJ. The affected region is a patchwork of SDs, and contains genes from the <em>Klr</em> superfamily of immunoglobulin-like dendritic cell receptors.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of large copy-number variants. (A) Normalized hybridization intensity for several inbred strains at standard SNP probes (gray), or copy-number probes (colors) across the region chr6:  Mb. Strains with the reference copy number are shown in blue; those with putative deletions in red; and those with putative duplication in green. Dashed line indicates the reference value of zero. (B) Normalized read depth from whole-genome sequencing, calculated in 1-kb bins, for the same strains and region as in (A). Dashed line indicates the reference value of one. (C) Segmental duplications (black, from UCSC genomicSuperDups table) and genes from the Clec, Klr and Tas2r families (colors, from Ensembl). (D) Identical to (C), reproduced for reference against (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Detection of large copy-number variants. (A) Normalized hybridization intensity for several inbred strains at standard SNP probes (gray), or copy-number probes (colors) across the region chr6: &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-133&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Mb. Strains with the reference copy number are shown in blue; those with putative deletions in red; and those with putative duplication in green. Dashed line indicates the reference value of zero. (B) Normalized read depth from whole-genome sequencing, calculated in 1-kb bins, for the same strains and region as in (A). Dashed line indicates the reference value of one. (C) Segmental duplications (black, from UCSC genomicSuperDups table) and genes from the &lt;em&gt;Clec&lt;/em&gt;, &lt;em&gt;Klr&lt;/em&gt; and &lt;em&gt;Tas2r&lt;/em&gt; families (colors, from Ensembl). (D) Identical to (C), reproduced for reference against (B).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F8.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/2/263/F8.medium.gif" width="345" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467554" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-72" class="first-child">Detection of large copy-number variants. (A) Normalized hybridization intensity for several inbred strains at standard SNP probes (gray), or copy-number probes (colors) across the region chr6: <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F8/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F8/embed/mml-math-133.gif"/></noscript></span></span> Mb. Strains with the reference copy number are shown in blue; those with putative deletions in red; and those with putative duplication in green. Dashed line indicates the reference value of zero. (B) Normalized read depth from whole-genome sequencing, calculated in 1-kb bins, for the same strains and region as in (A). Dashed line indicates the reference value of one. (C) Segmental duplications (black, from UCSC genomicSuperDups table) and genes from the <em>Clec</em>, <em>Klr</em> and <em>Tas2r</em> families (colors, from Ensembl). (D) Identical to (C), reproduced for reference against (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-73">Although optimization of CNV calling is beyond the scope of this manuscript, we note that existing software packages such as PennCNV (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2007</a>) can make use of signal from both SNP probes and invariant copy-number probes on GigaMUGA.</p></div><div id="sec-31" class="subsection"><h3>Targeted content: the complement cascade</h3><p id="p-74">The complement cascade bridges the innate and adaptive immune responses. Its constituent genes are well-defined, and functional polymorphisms within them underlie differential susceptibility to a variety of infectious and autoimmune diseases [see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beltrame <em>et al.</em> (2015)</a> for a recent review]. Most of the genes within the complement cascade arose via ancestral gene duplications, and many are copy-number variable in mouse and human (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nonaka and Miyazawa 2001</a>). We therefore designed 105 probes to directly genotype variants with functional significance within this important pathway, as well as further characterize copy number variation within the complement cascade across a range of mouse strains. They assay putative functional SNPs identified by the Sanger Mouse Genomes Project as segregating in the CC founder strains in 25 genes in the complement cascade (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467529/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467529/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Probes targeting functional variants in genes in the complement cascade</span><div class="sb-div caption-clear"></div></div></div><p id="p-77">Interpretation of discrete genotypes calls at these probes is complicated by paralogy between genes in the complement pathway, and further by CNV: 25 of 105 complement probes (<span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-135.gif"/></noscript></span></span>) have a no-call rate <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-136.gif"/></noscript></span></span> compared to <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-137.gif"/></noscript></span></span> array-wide. Most probes in these regions behave as multiallelic markers rather than biallelic SNPs (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probes targeted to complement-pathway genes detect copy-number variation. (A) Cluster plots for a probe in the C1s1 gene. Inbred individuals from the CC founder strains are colored according to the scheme used throughout this paper; s between them are colored gray. (B) Normalized hybridization intensity for one individual from each of the eight founder strains of the Collaborative Cross. Dashed line indicates the reference value of zero. (C) Normalized read depth from whole-genome sequencing, calculated in 500-bp bins, for the region highlighted in (A). Dashed line indicates the reference value of one. (D) Genes in the complement pathway (black) and other genes (gray), from Ensembl. (E) Segmental duplications (black, from UCSC genomicSuperDups table)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Probes targeted to complement-pathway genes detect copy-number variation. (A) Cluster plots for a probe in the &lt;em&gt;C1s1&lt;/em&gt; gene. Inbred individuals from the CC founder strains are colored according to the scheme used throughout this paper; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;s between them are colored gray. (B) Normalized hybridization intensity for one individual from each of the eight founder strains of the Collaborative Cross. Dashed line indicates the reference value of zero. (C) Normalized read depth from whole-genome sequencing, calculated in 500-bp bins, for the region highlighted in (A). Dashed line indicates the reference value of one. (D) Genes in the complement pathway (black) and other genes (gray), from Ensembl. (E) Segmental duplications (black, from UCSC genomicSuperDups table).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F9.medium.gif" width="286" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/6/2/263/F9.medium.gif" width="286" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467620" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-78" class="first-child">Probes targeted to complement-pathway genes detect copy-number variation. (A) Cluster plots for a probe in the <em>C1s1</em> gene. Inbred individuals from the CC founder strains are colored according to the scheme used throughout this paper; <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F9/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/F9/embed/mml-math-138.gif"/></noscript></span></span>s between them are colored gray. (B) Normalized hybridization intensity for one individual from each of the eight founder strains of the Collaborative Cross. Dashed line indicates the reference value of zero. (C) Normalized read depth from whole-genome sequencing, calculated in 500-bp bins, for the region highlighted in (A). Dashed line indicates the reference value of one. (D) Genes in the complement pathway (black) and other genes (gray), from Ensembl. (E) Segmental duplications (black, from UCSC genomicSuperDups table).</p><div class="sb-div caption-clear"></div></div></div><p id="p-79">As an example, we focus on the genes encoding the C1 complex on chromosome 6. The C1 complex has two components, C1R and C1S, which arose by an ancient duplication near the base of the vertebrate lineage. Further duplications in the mouse lineage gave rise to <em>C1ra</em>, <em>C1rb</em>, <em>C1s1</em>, and <em>C1s2</em> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nonaka and Miyazawa 2001</a>). Hybridization patterns within <em>C1s1</em> (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9, A and B</a>) are characteristic of duplicated sequence. Apparently heterozygous calls in inbred strains—such as for NZO/HlLtJ at marker complement 120—are frequently diagnostic for cross-hybridization between paralogous sequences. In this case, both LRR and read depth from whole-genome sequence data indicate the presence of a large copy-number gain encompassing the entire <em>C1</em> region in NZO/HlLtJ (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9, C–E</a>). Its boundaries coincide with a segmental duplication in the reference genome. Integration of allele calls and intensity patterns at complement probes will be useful for directly characterizing alleles in the complement pathway.</p></div><div id="sec-32" class="subsection"><h3>Targeted content: probes for engineered constructs</h3><p id="p-80">Probes targeted to engineered constructs were validated using raw intensity data from 587 samples genotyped on the MegaMUGA platform. A panel of 38 probes for 21 constructs provided robust discrimination between known negative and known or presumed positive samples (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>). These probes are informative only for presence or absence, and do not discriminate between heterozygous or homozygous states. Furthermore, because only one allele at each probe exists (the other is arbitrarily chosen), the intensity normalization performed by Illumina BeadStudio introduces artifacts. We recommend using the raw fluorescence values for determining the presence or absence of engineered constructs.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/263/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of probe sets targeted to engineered constructs. (A) Distribution of sum-intensity among samples with (&#x201C;present&#x201D;, red), and without (&#x201C;absent&#x201D;, blue) each of 21 target constructs. Note that the scale for the y-axis is logarithmic. (B) Cluster plot for two probes targeted to an ampicillin resistance cassette (AmpR), showing normalized intensities. Points are colored according to the Illumina genotype call (N or H = missing; A or B = nonmissing), and shapes correspond to the classification inferred based on sum-intensity across all probes for this target. Note the curvilinear artifact for probe AmpR001. (C) Same as (B), but using raw intensities. Separation between positive and negative groups is slightly more obvious, and the curvilinear artifact is not present." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133555017" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance of probe sets targeted to engineered constructs. (A) Distribution of sum-intensity among samples with (&#x201C;present&#x201D;, red), and without (&#x201C;absent&#x201D;, blue) each of 21 target constructs. Note that the scale for the &lt;em&gt;y&lt;/em&gt;-axis is logarithmic. (B) Cluster plot for two probes targeted to an ampicillin resistance cassette (AmpR), showing normalized intensities. Points are colored according to the Illumina genotype call (N or H = missing; A or B = nonmissing), and shapes correspond to the classification inferred based on sum-intensity across all probes for this target. Note the curvilinear artifact for probe AmpR001. (C) Same as (B), but using raw intensities. Separation between positive and negative groups is slightly more obvious, and the curvilinear artifact is not present.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/263/F10.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/6/2/263/F10.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/263/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/263/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467516" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-81" class="first-child">Performance of probe sets targeted to engineered constructs. (A) Distribution of sum-intensity among samples with (“present”, red), and without (“absent”, blue) each of 21 target constructs. Note that the scale for the <em>y</em>-axis is logarithmic. (B) Cluster plot for two probes targeted to an ampicillin resistance cassette (AmpR), showing normalized intensities. Points are colored according to the Illumina genotype call (N or H = missing; A or B = nonmissing), and shapes correspond to the classification inferred based on sum-intensity across all probes for this target. Note the curvilinear artifact for probe AmpR001. (C) Same as (B), but using raw intensities. Separation between positive and negative groups is slightly more obvious, and the curvilinear artifact is not present.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-33" class="subsection"><h3>Concluding remarks</h3><p id="p-82">The Mouse Universal Genotyping Array (MUGA) series was designed to provide a low-cost, general-purpose solution for genotyping laboratory and wild mice. GigaMUGA array is the third generation of the MUGA platform. At 143,259 probes, it offers almost double the marker density of its predecessor, MegaMUGA, while retaining MegaMUGA’s top <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/263/embed/mml-math-139.gif"/></noscript></span></span> best-performing markers. GigaMUGA’s content is optimized for discrimination between common laboratory strains, both classical and wild-derived, including substrains of very recent common origin. The array is also informative for ancestry and population structure in wild-caught and wild-derived mice. A new panel of copy-number probes tags regions of structural polymorphism to enable simultaneous CNV discovery and genotyping of SNPs.</p><p id="p-83">Although the costs of sequencing continue to fall, analysis of sequencing datasets—especially from low-coverage or reduced-representation protocols (<em>e.g.</em>, RAD-seq)—remains challenging for nonexpert users. Furthermore, hybridization intensity even at biallelic SNPs can be used to detect copy-number variants. The robustness, simplicity and curated content of microarrays continues to make them a valuable tool in model organisms.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-84">The authors thank Amelia Clayshulte and Rachel McMullan for assistance in organizing the panel of reference samples. We thank April Binder, Francois Bonhomme, Richard Chandler, Frank Conlon, Nigel Crawford, Jim Crowley, Ted Garland, Virginia Godfrey, Kent Hunter, Molly Plehaty, Marshall Runge, David Threadgill, and George Weinstock for providing DNA samples. This work was supported in part by U42OD010924 (J.B., F.P.M.V.); P50HG006582 (P.F.S., F.P.M.V.); U19AI100625 (F.P.M.V., L.M., M.T.F., L.E.G.); R01HD065024 (F.P.M.V.); R01DK101573 (A.D.A., M.P.K.); Vaadia-BARD Postdoctoral Fellowship Award FI-12 478-13 (L.Y.); T32GM067553 (J.P.D., A.P.M.); F30MH103925 (A.P.M.). The Systems Genetics Core Facility and Mutant Mouse Resource and Research Center at the University of North Carolina provided administrative support and computing resources. Both MegaMUGA and GigaMUGA were developed under a service contract to F.P.M.V. and L.M. from Neogen Inc., Lincoln, NE. The authors have no conflict of interest to declare. None of the authors have a financial relationship with Neogen Inc. apart from the service contract listed above.</p><p id="p-85"><em>Note added in proof:</em> See Morgan 2015 (pp. 281–286) in this issue, for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-86">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022087/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-87"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-11"><span class="received-label">Received </span>September 11, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-11-19"><span class="accepted-label">Accepted </span>November 19, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Morgan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.1" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.2" data-doi="10.1371/journal.pgen.1004916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxl</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghupathy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PRDM9 drives evolutionary erosion of hotspots in <em>Mus musculus</em> through haplotype-specific initiation of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004916%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004916%26rft_id%253Dinfo%253Apmid%252F25568937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25568937&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.3" data-doi="10.1038/71641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafezparast</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon-Pierce</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBeck%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D25%26rft.atitle%253DGenealogies%2Bof%2Bmouse%2Binbred%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F71641%26rft_id%253Dinfo%253Apmid%252F10615122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/71641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10615122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084609200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.4" data-doi="10.1016/j.molimm.2015.03.245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beltrame</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boldt</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catarino</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boschmann</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MBL-associated serine proteases (MASPs) and infectious diseases.</span> <abbr class="cit-jnl-abbrev">Mol. Immunol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BImmunol.%26rft.volume%253D67%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molimm.2015.03.245%26rft_id%253Dinfo%253Apmid%252F25862418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molimm.2015.03.245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25862418&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.5" data-doi="10.1038/315070a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boursot</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baron</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatat</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Most classical <em>Mus musculus domesticus</em> laboratory mouse strains carry a <em>Mus musculus</em> Y chromosome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">315</span>: <span class="cit-fpage">70</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBishop%26rft.auinit1%253DC.%2BE.%26rft.volume%253D315%26rft.issue%253D6014%26rft.spage%253D70%26rft.epage%253D72%26rft.atitle%253DMost%2Bclassical%2BMus%2Bmusculus%2Bdomesticus%2Blaboratory%2Bmouse%2Bstrains%2Bcarry%2Ba%2BMus%2Bmusculus%2Bmusculus%2BY%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F315070a0%26rft_id%253Dinfo%253Apmid%252F2986012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/315070a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2986012&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.2.263.6" data-doi="10.1038/ng.350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cahan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izumi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graubert</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The impact of copy number variation on local gene expression in mouse hematopoietic stem and progenitor cells.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCahan%26rft.auinit1%253DP.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D430%26rft.epage%253D437%26rft.atitle%253DThe%2Bimpact%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bon%2Blocal%2Bgene%2Bexpression%2Bin%2Bmouse%2Bhematopoietic%2Bstem%2Band%2Bprogenitor%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.350%26rft_id%253Dinfo%253Apmid%252F19270704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19270704&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264659000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.7" data-doi="10.1371/journal.pgen.1003853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calaway</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic architecture of skewed X inactivation in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003853%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003853%26rft_id%253Dinfo%253Apmid%252F24098153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24098153&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.8" data-doi="10.1152/physiolgenomics.00055.2014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyekis</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blizard</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Discovery and refinement of muscle weight QTLs in B6 x D2 advanced intercross mice.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00055.2014%26rft_id%253Dinfo%253Apmid%252F24963006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo5OiI0Ni8xNi81NzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.9" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The diversity outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.10" data-doi="10.1101/gr.4107905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ascertainment bias in studies of human genome-wide polymorphism.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1496</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1496%26rft.epage%253D1502%26rft.atitle%253DAscertainment%2Bbias%2Bin%2Bstudies%2Bof%2Bhuman%2Bgenome-wide%2Bpolymorphism%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4107905%26rft_id%253Dinfo%253Apmid%252F16251459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNDk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.11" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.12" data-doi="10.1038/ng.3222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhabotynsky</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pakatci</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3222%26rft_id%253Dinfo%253Apmid%252F25730764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25730764&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.2.263.13" data-doi="10.1101/gr.6754607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baribault</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassner</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Significant gene content variation characterizes the genomes of inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1743</span>–<span class="cit-lpage">1754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6754607%26rft_id%253Dinfo%253Apmid%252F17989247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xNzQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.14" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.15" data-doi="10.1186/1471-2164-15-847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naghashfar</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SNP array profiling of mouse cell lines identifies their strains of origin and reveals cross-contamination and widespread aneuploidy.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D847%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-847%26rft_id%253Dinfo%253Apmid%252F25277546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25277546&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.16" data-doi="10.1371/journal.pgen.1004850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayshulte</span>  <span class="cit-name-given-names">A. M.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadgary</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A multi-megabase copy number gain causes maternal transmission ratio distortion on mouse chromosome 2.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004850%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004850%26rft_id%253Dinfo%253Apmid%252F25679959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25679959&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.17" data-doi="10.1038/ng.2007.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Egan</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sridhar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wigler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recurrent DNA copy number variation in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSalmon%2BHillbertz%26rft.auinit1%253DN.%2BH.%26rft.volume%253D39%26rft.issue%253D11%26rft.spage%253D1384%26rft.epage%253D1389%26rft.atitle%253DRecurrent%2BDNA%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.19%26rft_id%253Dinfo%253Apmid%252F17965714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17965714&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-6.2.263.18"><div class="cit-metadata unstructured">Fraley, C., A. E. Raftery, T. B. Murphy, and L. Scrucca, 2012 <em>mclust Version 4 for R: normal mixture modeling for model-based clustering</em>, <em>classification</em>, <em>and density estimation</em>. Technical Report No. 597, Department of Statistics, University of Washington.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.19" data-doi="10.1038/nature06067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogue</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A sequence-based variation map of 8.27 million SNPs in inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>: <span class="cit-fpage">1050</span>–<span class="cit-lpage">1053</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D448%26rft.issue%253D7157%26rft.spage%253D1050%26rft.epage%253D1053%26rft.atitle%253DA%2Bsequence-based%2Bvariation%2Bmap%2Bof%2B8.27%2Bmillion%2BSNPs%2Bin%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06067%26rft_id%253Dinfo%253Apmid%252F17660834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17660834&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249097600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.2.263.20"><div class="cit-metadata unstructured">Fu, C.-P., C. E. Welsh, F. P.-M. de Villena, and L. McMillan, 2012 Inferring ancestry in admixed populations using microarray probe intensities. In <em>Proceedings of the ACM Conference on Bioinformatics</em>, <em>Computational Biology and Biomedicine - BCB ′12</em>, Association for Computing Machinery (ACM), New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.21" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for Diversity Outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DGatti%26rft.auinit1%253DD.%2BM.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1623%26rft.epage%253D1633%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2BMapping%2BMethods%2Bfor%2BDiversity%2BOutbred%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft_id%253Dinfo%253Apmid%252F25237114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.2.263.22" data-doi="10.1038/ng.345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henrichsen</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinckenbosch</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaignat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pradervand</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Segmental copy number variation shapes tissue transcriptomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHenrichsen%26rft.auinit1%253DC.%2BN.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D424%26rft.epage%253D429%26rft.atitle%253DSegmental%2Bcopy%2Bnumber%2Bvariation%2Bshapes%2Btissue%2Btranscriptomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.345%26rft_id%253Dinfo%253Apmid%252F19270705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19270705&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264659000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.2.263.23"><div class="cit-metadata unstructured">Kao, C.-Y., C.-P. Fu, and L. McMillan, 2014 InstantGenotype: A non-parametric model for genotype inference using microarray probe intensities. In <em>Proceedings of the 5th ACM Conference on Bioinformatics</em>, <em>Computational Biology</em>, <em>and Health Informatics - BCB ′14</em>, Association for Computing Machinery (ACM), New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.24" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <em>et al</em>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.25" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-6.2.263.26"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v1 [q-bio.GN].</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.27" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.29" data-doi="10.1038/74215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winchester</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Large-scale discovery and genotyping of single-nucleotide polymorphisms in the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLindblad-Toh%26rft.auinit1%253DK.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D381%26rft.epage%253D386%26rft.atitle%253DLarge-scale%2Bdiscovery%2Band%2Bgenotyping%2Bof%2Bsingle-nucleotide%2Bpolymorphisms%2Bin%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F74215%26rft_id%253Dinfo%253Apmid%252F10742102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/74215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742102&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086192800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.30" data-doi="10.1534/genetics.114.161653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161653%26rft_id%253Dinfo%253Apmid%252F24578350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk3LzEvOTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.31" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-6.2.263.32"><div class="cit-metadata"><cite>Morgan, A. P., 2015 argyle: an R package for analysis of Illumina genotyping arrays. G3 (Bethesda) 6: <span class="cit-fpage">281</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.33" data-doi="10.1007/s00335-015-9581-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Informatics resources for the collaborative cross and related mouse populations.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D521%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-015-9581-z%26rft_id%253Dinfo%253Apmid%252F26135136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-015-9581-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26135136&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-other" id="cit-6.2.263.34"><div class="cit-metadata unstructured">Nonaka, M. and S. Miyazawa, 2001 Evolution of the initiating enzymes of the complement system. Genome Biol. 3: reviews1001.1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.35" data-doi="10.1101/gr.5402306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steemers</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenniges</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1136</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5402306%26rft_id%253Dinfo%253Apmid%252F16899659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE2LzkvMTEzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.36" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.2.263.37" data-doi="10.1101/gr.102970.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102970.109%26rft_id%253Dinfo%253Apmid%252F20308636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.38" data-doi="10.1007/s00335-013-9499-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogala</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gooch</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Collaborative Cross as a resource for modeling human disease: CC011/Unc, a new mouse model for spontaneous colitis.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9499-2%26rft_id%253Dinfo%253Apmid%252F24487921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9499-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487921&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-6.2.263.39"><div class="cit-metadata unstructured">Sambrook, J., and D. W. Russell, 2006 <em>Molecular Cloning: A Laboratory Manual</em>, Vol. 3, Ed. 3rd. Cold Spring Harbor Laboratory Press. Cold Spring Harbor, NY.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.40" data-doi="10.1038/ng.172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mouse segmental duplication and copy number variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DShe%26rft.auinit1%253DX.%26rft.volume%253D40%26rft.issue%253D7%26rft.spage%253D909%26rft.epage%253D914%26rft.atitle%253DMouse%2Bsegmental%2Bduplication%2Band%2Bcopy%2Bnumber%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.172%26rft_id%253Dinfo%253Apmid%252F18500340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18500340&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.41" data-doi="10.1371/journal.pbio.0040395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copley</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution single nucleotide polymorphism genetic map of the mouse genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DShifman%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De395%26rft.epage%253De395%26rft.atitle%253DA%2Bhigh-resolution%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenetic%2Bmap%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040395%26rft_id%253Dinfo%253Apmid%252F17105354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105354&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.42" data-doi="10.1186/1471-2105-9-409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staaf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallon-Christersson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juliusson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenquist</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Normalization of Illumina Infinium whole-genome SNP data improves copy number estimates and allelic intensity ratios.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStaaf%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.spage%253D409%26rft.epage%253D409%26rft.atitle%253DNormalization%2Bof%2BIllumina%2BInfinium%2Bwhole-genome%2BSNP%2Bdata%2Bimproves%2Bcopy%2Bnumber%2Bestimates%2Band%2Ballelic%2Bintensity%2Bratios.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-409%26rft_id%253Dinfo%253Apmid%252F18831757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18831757&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.43" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.44" data-doi="10.1038/nmeth842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steemers</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Whole-genome genotyping with the single-base extension assay.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSteemers%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D33%26rft.atitle%253DWhole-genome%2Bgenotyping%2Bwith%2Bthe%2Bsingle-base%2Bextension%2Bassay.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth842%26rft_id%253Dinfo%253Apmid%252F16369550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16369550&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234528000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.45" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the mouse Diversity Outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSvenson%26rft.auinit1%253DK.%2BL.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D447%26rft.atitle%253DHigh-Resolution%2BGenetic%2BMapping%2BUsing%2Bthe%2BMouse%2BDiversity%2BOutbred%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.46" data-doi="10.1023/A:1021479331779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swallow</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garland</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Artificial selection for increased wheel-running behavior in house mice.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DSwallow%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D227%26rft.epage%253D237%26rft.atitle%253DArtificial%2Bselection%2Bfor%2Bincreased%2Bwheel-running%2Bbehavior%2Bin%2Bhouse%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1021479331779%26rft_id%253Dinfo%253Apmid%252F9670598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1021479331779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9670598&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074583600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.47" data-doi="10.1038/nature04226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A haplotype map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1299%26rft.epage%253D1320%26rft.atitle%253DA%2Bhaplotype%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04226%26rft_id%253Dinfo%253Apmid%252F16255080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16255080&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.48" data-doi="10.1101/gr.6861907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glessner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PennCNV: An integrated hidden markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6861907%26rft_id%253Dinfo%253Apmid%252F17921354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMS8xNjY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.49" data-doi="10.1007/s00335-012-9410-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Status and access to the collaborative cross population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWelsh%26rft.auinit1%253DC.%2BE.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D706%26rft.epage%253D712%26rft.atitle%253DStatus%2Band%2Baccess%2Bto%2Bthe%2BCollaborative%2BCross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9410-6%26rft_id%253Dinfo%253Apmid%252F22847377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847377&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.2.263.50" data-doi="10.1186/gb-2012-13-8-r72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bumpstead</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Weyden</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinholdt</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilming</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequencing and characterization of the FVB/NJ mouse genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DWong%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253DR72%26rft.epage%253DR72%26rft.atitle%253DSequencing%2Band%2Bcharacterisation%2Bof%2Bthe%2BFVB%252FNJ%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-8-r72%26rft_id%253Dinfo%253Apmid%252F22916792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-8-r72&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916792&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.51" data-doi="10.1038/ng2087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">On the subspecific origin of the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1100%26rft.epage%253D1107%26rft.atitle%253DOn%2Bthe%2Bsubspecific%2Borigin%2Bof%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2087%26rft_id%253Dinfo%253Apmid%252F17660819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17660819&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249122400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.52" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. a.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.263.53" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/2/263.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/255" title="A RAB3GAP1 SINE Insertion in Alaskan Huskies with Polyneuropathy, Ocular Abnormalities, and Neuronal Vacuolation (POANV) Resembling Human Warburg Micro Syndrome 1 (WARBM1)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/281" title="argyle: An R Package for Analysis of Illumina Genotyping Arrays" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/2"><span>Volume 6 Issue 2, February 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/2.cover-source.jpg?itok=7FK5uFFF" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-syMHC4Y5nzKmHDPyyrowDdgYbWYGWO0-NrJe69yYsbw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/2/263" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/2/263" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/2/263" class="active" data-icon-position="" data-hide-link-title="0">The Mouse Universal Genotyping Array: From Substrains to Subspecies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460066" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NAFvy4fN4ZXD0tYm1QGE82H0NCdLI1V8LDdjsV2m5zg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/263.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/2/263" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ef2MPeWbAkoLzgjP9kIcq7zqbYx0Omc5i7D7Vz8q8cQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/2/263" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/263.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460066" id="node460066--2" data-pisa="ggg;6/2/263" data-pisa-master="ggg;g3.115.022087" data-apath="/ggg/6/2/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Mouse Universal Genotyping Array: From Substrains to Subspecies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew P. Morgan</span>, <span class="highwire-citation-author" data-delta="1">Chen-Ping Fu</span>, <span class="highwire-citation-author" data-delta="2">Chia-Yu Kao</span>, <span class="highwire-citation-author" data-delta="3">Catherine E. Welsh</span>, <span class="highwire-citation-author" data-delta="4">John P. Didion</span>, <span class="highwire-citation-author" data-delta="5">Liran Yadgary</span>, <span class="highwire-citation-author" data-delta="6">Leeanna Hyacinth</span>, <span class="highwire-citation-author" data-delta="7">Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="8">Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="9">Darla R. Miller</span>, <span class="highwire-citation-author" data-delta="10">Paola Giusti-Rodriguez</span>, <span class="highwire-citation-author" data-delta="11">Randal J. Nonneman</span>, <span class="highwire-citation-author" data-delta="12">Kevin D. Cook</span>, <span class="highwire-citation-author" data-delta="13">Jason K. Whitmire</span>, <span class="highwire-citation-author" data-delta="14">Lisa E. Gralinski</span>, <span class="highwire-citation-author" data-delta="15">Mark Keller</span>, <span class="highwire-citation-author" data-delta="16">Alan D. Attie</span>, <span class="highwire-citation-author" data-delta="17">Gary A. Churchill</span>, <span class="highwire-citation-author" data-delta="18">Petko Petkov</span>, <span class="highwire-citation-author" data-delta="19">Patrick F. Sullivan</span>, <span class="highwire-citation-author" data-delta="20">Jennifer R. Brennan</span>, <span class="highwire-citation-author" data-delta="21">Leonard McMillan</span> and <span class="highwire-citation-author" data-delta="22">Fernando Pardo-Manuel de Villena</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-279;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022087 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460066--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew P. Morgan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chen-Ping Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chen-Ping%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChen-Ping%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chia-Yu Kao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chia-Yu%2BKao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChia-Yu%2BKao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine E. Welsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematics and Computer Science, Rhodes College, Memphis, Tennessee 38112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BE.%2BWelsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welsh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BE.%2BWelsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John P. Didion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BDidion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Didion%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BP.%2BDidion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Liran Yadgary </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liran%2BYadgary%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yadgary%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiran%2BYadgary%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Leeanna Hyacinth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leeanna%2BHyacinth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyacinth%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeeanna%2BHyacinth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Paola Giusti-Rodriguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paola%2BGiusti-Rodriguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giusti-Rodriguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaola%2BGiusti-Rodriguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Randal J. Nonneman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randal%2BJ.%2BNonneman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nonneman%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandal%2BJ.%2BNonneman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kevin D. Cook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BD.%2BCook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cook%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BD.%2BCook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jason K. Whitmire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BK.%2BWhitmire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmire%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BK.%2BWhitmire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Lisa E. Gralinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BE.%2BGralinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gralinski%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BE.%2BGralinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Mark Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Alan D. Attie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BD.%2BAttie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Attie%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BD.%2BAttie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Petko Petkov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petko%2BPetkov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petkov%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetko%2BPetkov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Patrick F. Sullivan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Psychiatry, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BF.%2BSullivan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sullivan%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BF.%2BSullivan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Jennifer R. Brennan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Mutant Mouse Resource and Research Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BR.%2BBrennan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brennan%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BR.%2BBrennan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/2/263" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/2/263&amp;title=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/2/263&amp;title=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/2/263&amp;title=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/2/263&amp;text=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/2/263&amp;title=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/2/263&amp;t=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/2/263&amp;title=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/2/263&amp;title=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460066/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460066" id="node460066--3" data-pisa="ggg;6/2/263" data-pisa-master="ggg;g3.115.022087" data-apath="/ggg/6/2/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Mouse Universal Genotyping Array: From Substrains to Subspecies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew P. Morgan</span>, <span class="highwire-citation-author" data-delta="1">Chen-Ping Fu</span>, <span class="highwire-citation-author" data-delta="2">Chia-Yu Kao</span>, <span class="highwire-citation-author" data-delta="3">Catherine E. Welsh</span>, <span class="highwire-citation-author" data-delta="4">John P. Didion</span>, <span class="highwire-citation-author" data-delta="5">Liran Yadgary</span>, <span class="highwire-citation-author" data-delta="6">Leeanna Hyacinth</span>, <span class="highwire-citation-author" data-delta="7">Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="8">Timothy A. Bell</span>, <span class="highwire-citation-author" data-delta="9">Darla R. Miller</span>, <span class="highwire-citation-author" data-delta="10">Paola Giusti-Rodriguez</span>, <span class="highwire-citation-author" data-delta="11">Randal J. Nonneman</span>, <span class="highwire-citation-author" data-delta="12">Kevin D. Cook</span>, <span class="highwire-citation-author" data-delta="13">Jason K. Whitmire</span>, <span class="highwire-citation-author" data-delta="14">Lisa E. Gralinski</span>, <span class="highwire-citation-author" data-delta="15">Mark Keller</span>, <span class="highwire-citation-author" data-delta="16">Alan D. Attie</span>, <span class="highwire-citation-author" data-delta="17">Gary A. Churchill</span>, <span class="highwire-citation-author" data-delta="18">Petko Petkov</span>, <span class="highwire-citation-author" data-delta="19">Patrick F. Sullivan</span>, <span class="highwire-citation-author" data-delta="20">Jennifer R. Brennan</span>, <span class="highwire-citation-author" data-delta="21">Leonard McMillan</span> and <span class="highwire-citation-author" data-delta="22">Fernando Pardo-Manuel de Villena</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-279;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022087 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460066--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew P. Morgan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chen-Ping Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chen-Ping%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChen-Ping%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chia-Yu Kao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chia-Yu%2BKao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChia-Yu%2BKao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine E. Welsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematics and Computer Science, Rhodes College, Memphis, Tennessee 38112</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BE.%2BWelsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welsh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BE.%2BWelsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John P. Didion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BDidion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Didion%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BP.%2BDidion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Liran Yadgary </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liran%2BYadgary%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yadgary%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiran%2BYadgary%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Leeanna Hyacinth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leeanna%2BHyacinth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyacinth%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeeanna%2BHyacinth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Timothy A. Bell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BBell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bell%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BA.%2BBell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Paola Giusti-Rodriguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paola%2BGiusti-Rodriguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giusti-Rodriguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaola%2BGiusti-Rodriguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Randal J. Nonneman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Randal%2BJ.%2BNonneman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nonneman%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARandal%2BJ.%2BNonneman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kevin D. Cook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BD.%2BCook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cook%20KD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BD.%2BCook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jason K. Whitmire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BK.%2BWhitmire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmire%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BK.%2BWhitmire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Lisa E. Gralinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BE.%2BGralinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gralinski%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BE.%2BGralinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Mark Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Alan D. Attie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BD.%2BAttie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Attie%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BD.%2BAttie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Petko Petkov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Jackson Laboratory, Bar Harbor, Maine 04609</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petko%2BPetkov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petkov%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetko%2BPetkov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Patrick F. Sullivan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Psychiatry, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BF.%2BSullivan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sullivan%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BF.%2BSullivan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Jennifer R. Brennan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Mutant Mouse Resource and Research Center, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BR.%2BBrennan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brennan%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BR.%2BBrennan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Leonard McMillan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Science, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonard%2BMcMillan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMillan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonard%2BMcMillan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lineberger Comprehensive Cancer Center and Carolina Center for Genome Sciences, University of North Carolina, Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fernando@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">fernando@med.unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=263&amp;rft.epage=279&amp;rft.atitle=The%20Mouse%20Universal%20Genotyping%20Array%3A%20From%20Substrains%20to%20Subspecies&amp;rft.volume=6&amp;rft.issue=2&amp;rft.date=2016-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Morgan%2C+Andrew+P.&amp;rft.au=Fu%2C+Chen-Ping&amp;rft.au=Kao%2C+Chia-Yu&amp;rft.au=Welsh%2C+Catherine+E.&amp;rft.au=Didion%2C+John+P.&amp;rft.au=Yadgary%2C+Liran&amp;rft.au=Hyacinth%2C+Leeanna&amp;rft.au=Ferris%2C+Martin+T.&amp;rft.au=Bell%2C+Timothy+A.&amp;rft.au=Miller%2C+Darla+R.&amp;rft.au=Giusti-Rodriguez%2C+Paola&amp;rft.au=Nonneman%2C+Randal+J.&amp;rft.au=Cook%2C+Kevin+D.&amp;rft.au=Whitmire%2C+Jason+K.&amp;rft.au=Gralinski%2C+Lisa+E.&amp;rft.au=Keller%2C+Mark&amp;rft.au=Attie%2C+Alan+D.&amp;rft.au=Churchill%2C+Gary+A.&amp;rft.au=Petkov%2C+Petko&amp;rft.au=Sullivan%2C+Patrick+F.&amp;rft.au=Brennan%2C+Jennifer+R.&amp;rft.au=McMillan%2C+Leonard&amp;rft.au=Pardo-Manuel+de+Villena%2C+Fernando"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460066/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460066/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460066/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460066/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460066/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460066/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460066/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460066/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460066/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460066/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460066/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460066/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/2/263" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/2/263.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/2/263.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/2/263.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/2/263.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
