<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/2/365" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/2/365.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/2/365.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/2/365.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/2/365/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/2/365.atom" />
<meta name="HW.pisa" content="ggg;6/2/365" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Dynamics of Dark-Fly Genome Under Environmental Selections" />
<meta name="DC.Identifier" content="10.1534/g3.115.023549" />
<meta name="DC.Date" content="2016-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Izutsu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Environmental adaptation is one of the most fundamental features of organisms. Modern genome science has identified some genes associated with adaptive traits of organisms, and has provided insights into environmental adaptation and evolution. However, how genes contribute to adaptive traits and how traits are selected under an environment in the course of evolution remain mostly unclear. To approach these issues, we utilize “Dark-fly”, a Drosophila melanogaster line maintained in constant dark conditions for more than 60 years. Our previous analysis identified 220,000 single nucleotide polymorphisms (SNPs) in the Dark-fly genome, but did not clarify which SNPs of Dark-fly are truly adaptive for living in the dark. We found here that Dark-fly dominated over the wild-type fly in a mixed population under dark conditions, and based on this domination we designed an experiment for genome reselection to identify adaptive genes of Dark-fly. For this experiment, large mixed populations of Dark-fly and the wild-type fly were maintained in light conditions or in dark conditions, and the frequencies of Dark-fly SNPs were compared between these populations across the whole genome. We thereby detected condition-dependent selections toward approximately 6% of the genome. In addition, we observed the time-course trajectory of SNP frequency in the mixed populations through generations 0, 22, and 49, which resulted in notable categorization of the selected SNPs into three types with different combinations of positive and negative selections. Our data provided a list of about 100 strong candidate genes associated with the adaptive traits of Dark-fly." />
<meta name="DC.Contributor" content="Minako Izutsu" />
<meta name="DC.Contributor" content="Atsushi Toyoda" />
<meta name="DC.Contributor" content="Asao Fujiyama" />
<meta name="DC.Contributor" content="Kiyokazu Agata" />
<meta name="DC.Contributor" content="Naoyuki Fuse" />
<meta name="article:published_time" content="2016-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Dynamics of Dark-Fly Genome Under Environmental Selections" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Environmental adaptation is one of the most fundamental features of organisms. Modern genome science has identified some genes associated with adaptive traits of organisms, and has provided insights into environmental adaptation and evolution. However, how genes contribute to adaptive traits and how traits are selected under an environment in the course of evolution remain mostly unclear. To approach these issues, we utilize “Dark-fly”, a &lt;i&gt;Drosophila melanogaster&lt;/i&gt; line maintained in constant dark conditions for more than 60 years. Our previous analysis identified 220,000 single nucleotide polymorphisms (SNPs) in the Dark-fly genome, but did not clarify which SNPs of Dark-fly are truly adaptive for living in the dark. We found here that Dark-fly dominated over the wild-type fly in a mixed population under dark conditions, and based on this domination we designed an experiment for genome reselection to identify adaptive genes of Dark-fly. For this experiment, large mixed populations of Dark-fly and the wild-type fly were maintained in light conditions or in dark conditions, and the frequencies of Dark-fly SNPs were compared between these populations across the whole genome. We thereby detected condition-dependent selections toward approximately 6% of the genome. In addition, we observed the time-course trajectory of SNP frequency in the mixed populations through generations 0, 22, and 49, which resulted in notable categorization of the selected SNPs into three types with different combinations of positive and negative selections. Our data provided a list of about 100 strong candidate genes associated with the adaptive traits of Dark-fly.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/02/01" />
<meta name="citation_mjid" content="ggg;6/2/365" />
<meta name="citation_id" content="6/2/365" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/2/365" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/2/365.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/2/365.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/2/365.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023549" />
<meta name="citation_pmid" content="26637434" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="365" />
<meta name="citation_lastpage" content="376" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Minako Izutsu" />
<meta name="citation_author_institution" content="Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan" />
<meta name="citation_author" content="Atsushi Toyoda" />
<meta name="citation_author_institution" content="Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan" />
<meta name="citation_author" content="Asao Fujiyama" />
<meta name="citation_author_institution" content="Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan" />
<meta name="citation_author_institution" content="Department of Genetics, Graduate University for Advanced Studies, Mishima 411-8540, Japan" />
<meta name="citation_author" content="Kiyokazu Agata" />
<meta name="citation_author_institution" content="Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan" />
<meta name="citation_author" content="Naoyuki Fuse" />
<meta name="citation_author_institution" content="Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan" />
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Graduate School of Biostudies, Kyoto University, Kyoto 606-8501, Japan" />
<meta name="citation_author_email" content="nfuse@gcoe.biol.sci.kyoto-u.ac.jp" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. W. Albert;citation_author=S. Treusch;citation_author=A. H. Shockley;citation_author=J. S. Bloom;citation_author=L. Kruglyak;citation_title=Genetics of single-cell protein abundance variation in large yeast populations.;citation_pages=494-497;citation_volume=506;citation_year=2014;citation_pmid=24402228;citation_doi=10.1038/nature12904" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Araya;citation_author=C. Payen;citation_author=M. Dunham;citation_author=S. Fields;citation_title=Whole-genome sequencing of a laboratory-evolved yeast strain.;citation_pages=88-88;citation_volume=11;citation_year=2010;citation_pmid=20128923;citation_doi=10.1186/1471-2164-11-88" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. D. H. Barrett;citation_author=H. E. Hoekstra;citation_title=Molecular spandrels: tests of adaptation at the genetic level.;citation_pages=767-780;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=22005986;citation_doi=10.1038/nrg3015" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. E. Barrick;citation_author=R. E. Lenski;citation_title=Genome dynamics during experimental evolution.;citation_pages=827-839;citation_volume=14;citation_year=2013;citation_pmid=24166031;citation_doi=10.1038/nrg3564" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Barrick;citation_author=D. S. Yu;citation_author=S. H. Yoon;citation_author=H. Jeong;citation_author=T. K. Oh;citation_title=Genome evolution and adaptation in a long-term experiment with Escherichia coli.;citation_pages=1243-1247;citation_volume=461;citation_year=2009;citation_issue=7268;citation_pmid=19838166;citation_doi=10.1038/nature08480" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=N. H. Barton;citation_author=P. D. Keightley;citation_title=Understanding quantitative genetic variation.;citation_pages=11-21;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823787;citation_doi=10.1038/nrg700" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. Borowsky;citation_author=H. Wilkens;citation_title=Mapping a Cave Fish Genome: Polygenic Systems and Regressive Evolution;citation_pages=19-21;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011170;citation_doi=10.1093/jhered/93.1.19" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. K. Burke;citation_author=J. P. Dunham;citation_author=P. Shahrestani;citation_author=K. R. Thornton;citation_author=M. R. Rose;citation_title=Genome-wide analysis of a long-term evolution experiment with Drosophila.;citation_pages=587-590;citation_volume=467;citation_year=2010;citation_issue=7315;citation_pmid=20844486;citation_doi=10.1038/nature09352" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. R. Burton;citation_author=D. G. Clayton;citation_author=L. R. Cardon;citation_author=N. Craddock;citation_author=P. Deloukas;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_journal_title=Cell and tissue research;citation_journal_abbrev=Cell Tissue Res;citation_author=M.-Y. Choi;citation_author=A. Rafaeli;citation_author=R. A. Jurenka;citation_title=Pyrokinin/PBAN-like peptides in the central nervous system of Drosophila melanogaster.;citation_pages=459-465;citation_volume=306;citation_year=2001;citation_issue=3;citation_pmid=11735047;citation_doi=10.1007/s00441-001-0467-x" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. le Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=A. Claridge-Chang;citation_author=H. Wijnen;citation_author=F. Naef;citation_author=C. Boothroyd;citation_author=N. Rajewsky;citation_title=Circadian regulation of gene expression systems in the Drosophila head.;citation_pages=657-671;citation_volume=32;citation_year=2001;citation_issue=4;citation_pmid=11719206;citation_doi=10.1016/S0896-6273(01)00515-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=P. F. Colosimo;citation_author=C. L. Peichel;citation_author=K. Nereng;citation_author=B. K. Blackman;citation_author=M. D. Shapiro;citation_title=The genetic architecture of parallel armor plate reduction in threespine sticklebacks.;citation_pages=635-641;citation_volume=2;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. S. Edgar;citation_author=E. W. Green;citation_author=Y. Zhao;citation_author=G. van Ooijen;citation_author=M. Olmedo;citation_title=Peroxiredoxins are conserved markers of circadian rhythms.;citation_pages=459-464;citation_volume=485;citation_year=2012;citation_issue=7399;citation_pmid=22622569;citation_doi=10.1038/nature11088" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Edwards;citation_author=D. Gifford;citation_title=High-resolution genetic mapping with pooled sequencing.;citation_pages=S8;citation_volume=13;citation_year=2012;citation_pmid=23282012;citation_doi=10.1186/1471-2105-13-S14-S8" />
<meta name="citation_reference" content="citation_title=Evolution in the Dark: Adaptation of Drosophila in the Laboratory;citation_author=N. Fuse;citation_author=T. Kitamura;citation_author=T. Haramura;citation_author=K. Arikawa;citation_author=M. Imafuku;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Germer;citation_author=M. J. Holland;citation_author=R. Higuchi;citation_title=High-Throughput SNP Allele-Frequency Determination in Pooled DNA Samples by Kinetic PCR;citation_pages=258-266;citation_volume=10;citation_year=2000;citation_issue=2;citation_pmid=10673283;citation_doi=10.1101/gr.10.2.258" />
<meta name="citation_reference" content="citation_journal_title=Zoology;citation_author=B. R. Grant;citation_title=Evolution in Darwin’s finches: a review of a study on Isla Daphne Major in the Galapagos Archipelago.;citation_pages=255-259;citation_volume=106;citation_year=2003;citation_pmid=16351911;citation_doi=10.1078/0944-2006-00123" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Hey;citation_author=R. M. Kliman;citation_title=Interactions Between Natural Selection, Recombination and Gene Density in the Genes of Drosophila;citation_pages=595-608;citation_volume=160;citation_year=2002;citation_issue=2;citation_pmid=11861564" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. A. Hooven;citation_author=K. A. Sherman;citation_author=S. Butcher;citation_author=J. M. Giebultowicz;citation_title=Does the clock make the poison? Circadian variation in response to pesticides.;citation_pages=e6469-e6469;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19649249;citation_doi=10.1371/journal.pone.0006469" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=M. Imafuku;citation_author=T. Haramura;citation_title=Activity rhythm of Drosophila kept in complete darkness for 1300 generations.;citation_pages=195-198;citation_volume=28;citation_year=2011;citation_issue=3;citation_pmid=21385060;citation_doi=10.2108/zsj.28.195" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Izutsu;citation_author=J. Zhou;citation_author=Y. Sugiyama;citation_author=O. Nishimura;citation_author=T. Aizu;citation_title=Genome features of &quot;Dark-fly&quot;, a Drosophila line reared long-term in a dark environment.;citation_pages=e33288-e33288;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22432011;citation_doi=10.1371/journal.pone.0033288" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. B. Kim;citation_author=X. Fang;citation_author=A. A. Fushan;citation_author=Z. Huang;citation_author=A. V. Lobanov;citation_title=Genome sequencing reveals insights into physiology and longevity of the naked mole rat.;citation_pages=223-227;citation_volume=479;citation_year=2011;citation_issue=7372;citation_pmid=21993625;citation_doi=10.1038/nature10533" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. C. Koboldt;citation_author=K. Chen;citation_author=T. Wylie;citation_author=D. E. Larson;citation_author=M. D. McLellan;citation_title=VarScan: variant detection in massively parallel sequencing of individual and pooled samples.;citation_volume=25;citation_year=2009;citation_pmid=19542151;citation_doi=10.1093/bioinformatics/btp373" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Kofler;citation_author=R. V. Pandey;citation_author=C. Schlötterer;citation_title=PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).;citation_volume=27;citation_year=2011;citation_pmid=22025480;citation_doi=10.1093/bioinformatics/btr589" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. A. Lamia;citation_author=U. M. Sachdeva;citation_author=L. DiTacchio;citation_author=E. C. Williams;citation_author=J. G. Alvarez;citation_title=AMPK Regulates the Circadian Clock by Cryptochrome Phosphorylation and Degradation;citation_pages=437-440;citation_volume=326;citation_year=2009;citation_issue=5951;citation_pmid=19833968;citation_doi=10.1126/science.1172156" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lamichhaney;citation_author=J. Berglund;citation_author=M. S. Almen;citation_author=K. Maqbool;citation_author=M. Grabherr;citation_title=Evolution of Darwin/’s finches and their beaks revealed by genome sequencing.;citation_pages=371-375;citation_volume=518;citation_year=2015;citation_pmid=25686609;citation_doi=10.1038/nature14181" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. I. McCarthy;citation_author=G. R. Abecasis;citation_author=L. R. Cardon;citation_author=D. B. Goldstein;citation_author=J. Little;citation_title=Genome-wide association studies for complex traits: consensus, uncertainty and challenges.;citation_pages=356-369;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18398418;citation_doi=10.1038/nrg2344" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=R. McQuillan;citation_author=A.-L. Leutenegger;citation_author=R. Abdel-Rahman;citation_author=C. S. Franklin;citation_author=M. Pericic;citation_title=Runs of homozygosity in European populations.;citation_pages=359-372;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760389;citation_doi=10.1016/j.ajhg.2008.08.007" />
<meta name="citation_reference" content="citation_journal_title=Zoolog. Sci.;citation_author=S. Mori;citation_title=Changes of characters of Drosophila melanogaster brought about during the life in constant darkness and considerations on the processes through which theses changes were induced.;citation_pages=945-957;citation_volume=3;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Jpn. J. Genet.;citation_author=S. Mori;citation_author=S. Yanagishima;citation_title=Variations of Drosophila in relation to its environment VII Does Drosophila change its characters during dark life? (1).;citation_pages=151-161;citation_volume=34;citation_year=1959;citation_doi=10.1266/jjg.34.151" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=N. J. Nadeau;citation_author=C. D. Jiggins;citation_title=A golden age for evolutionary genetics? Genomic studies of adaptation in natural populations.;citation_pages=484-492;citation_volume=26;citation_year=2010;citation_issue=11;citation_pmid=20851493;citation_doi=10.1016/j.tig.2010.08.004" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Orozco-terWengel;citation_author=M. Kapun;citation_author=V. Nolte;citation_author=R. Kofler;citation_author=T. Flatt;citation_title=Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.;citation_pages=4931-4941;citation_volume=21;citation_year=2012;citation_issue=20;citation_pmid=22726122;citation_doi=10.1111/j.1365-294X.2012.05673.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Pal;citation_author=B. Papp;citation_author=M. J. Lercher;citation_title=An integrated view of protein evolution.;citation_pages=337-348;citation_volume=7;citation_year=2006;citation_issue=5;citation_pmid=16619049;citation_doi=10.1038/nrg1838" />
<meta name="citation_reference" content="citation_journal_title=The Biological Bulletin;citation_journal_abbrev=Biol. Bull.;citation_author=F. Payne;citation_title=DROSOPHILA AMPELOPHILA LOEW BRED IN THE DARK FOR SIXTY-NINE GENERATIONS;citation_pages=297-301;citation_volume=21;citation_year=1911;citation_issue=5;citation_doi=10.2307/1536021" />
<meta name="citation_reference" content="citation_journal_title=J. Gen. Physiol.;citation_author=R. Pearl;citation_title=A synthetic food medium for the cultivation of Drosophila: preliminary note.;citation_volume=9;citation_year=1926;citation_pmid=19872271;citation_doi=10.1085/jgp.9.4.513" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=C. Rosenkilde;citation_author=G. Cazzamali;citation_author=M. Williamson;citation_author=F. Hauser;citation_author=L. Søndergaard;citation_title=Molecular cloning, functional expression, and gene silencing of two Drosophila receptors for the Drosophila neuropeptide pyrokinin-2.;citation_pages=485-494;citation_volume=309;citation_year=2003;citation_issue=2;citation_pmid=12951076;citation_doi=10.1016/j.bbrc.2003.08.022" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. L. Rutherford;citation_author=S. Lindquist;citation_title=Hsp90 as a capacitor for morphological evolution.;citation_pages=336-342;citation_volume=396;citation_year=1998;citation_issue=6709;citation_pmid=9845070;citation_doi=10.1038/24550" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. Saxena;citation_author=B. F. Voight;citation_author=V. Lyssenko;citation_author=N. P. Burtt;citation_author=P. I. de Bakker;citation_title=Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels.;citation_volume=316;citation_year=2007;citation_pmid=17463246;citation_doi=10.1126/science.1142358" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=C. Schlotterer;citation_author=R. Tobler;citation_author=R. Kofler;citation_author=V. Nolte;citation_title=Sequencing pools of individuals [mdash] mining genome-wide polymorphism data without big funding.;citation_pages=749-763;citation_volume=15;citation_year=2014;citation_pmid=25246196" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=T. S. Simonson;citation_author=Y. Yang;citation_author=C. D. Huff;citation_author=H. Yun;citation_author=G. Qin;citation_title=genetic evidence for high-altitude adaptation in Tibet.;citation_volume=329;citation_year=2010;citation_pmid=20466884;citation_doi=10.1126/science.1189406" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=O. Tenaillon;citation_author=A. Rodríguez-Verdugo;citation_author=R. L. Gaut;citation_author=P. McDonald;citation_author=A. F. Bennett;citation_title=The Molecular Diversity of Adaptive Convergence;citation_pages=457-461;citation_volume=335;citation_year=2012;citation_issue=6067;citation_pmid=22282810;citation_doi=10.1126/science.1212986" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=C. H. Waddington;citation_title=Genetic assimilation of an acquired character.;citation_pages=118-126;citation_volume=7;citation_year=1953;citation_doi=10.2307/2405747" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Walkinshaw;citation_author=Y. Gai;citation_author=C. Farkas;citation_author=D. Richter;citation_author=E. Nicholas;citation_title=Identification of genes that promote or inhibit olfactory memory formation in Drosophila.;citation_volume=199;citation_year=2015;citation_pmid=25644700;citation_doi=10.1534/genetics.114.173575" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Zhou;citation_author=N. Udpa;citation_author=M. Gersten;citation_author=D. W. Visk;citation_author=A. Bashir;citation_title=Experimental selection of hypoxia-tolerant Drosophila melanogaster.;citation_volume=108;citation_year=2011;citation_pmid=21262834;citation_doi=10.1073/pnas.1010643108" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Dynamics of Dark-Fly Genome Under Environmental Selections" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="twitter:description" content="Environmental adaptation is one of the most fundamental features of organisms. Modern genome science has identified some genes associated with adaptive traits of organisms, and has provided insights into environmental adaptation and evolution. However, how genes contribute to adaptive traits and how traits are selected under an environment in the course of evolution remain mostly unclear. To approach these issues, we utilize “Dark-fly”, a Drosophila melanogaster line maintained in constant dark conditions for more than 60 years. Our previous analysis identified 220,000 single nucleotide polymorphisms (SNPs) in the Dark-fly genome, but did not clarify which SNPs of Dark-fly are truly adaptive for living in the dark. We found here that Dark-fly dominated over the wild-type fly in a mixed population under dark conditions, and based on this domination we designed an experiment for genome reselection to identify adaptive genes of Dark-fly. For this experiment, large mixed populations of Dark-fly and the wild-type fly were maintained in light conditions or in dark conditions, and the frequencies of Dark-fly SNPs were compared between these populations across the whole genome. We thereby detected condition-dependent selections toward approximately 6% of the genome. In addition, we observed the time-course trajectory of SNP frequency in the mixed populations through generations 0, 22, and 49, which resulted in notable categorization of the selected SNPs into three types with different combinations of positive and negative selections. Our data provided a list of about 100 strong candidate genes associated with the adaptive traits of Dark-fly." />
<meta name="og-title" property="og:title" content="Dynamics of Dark-Fly Genome Under Environmental Selections" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/2/365" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Environmental adaptation is one of the most fundamental features of organisms. Modern genome science has identified some genes associated with adaptive traits of organisms, and has provided insights into environmental adaptation and evolution. However, how genes contribute to adaptive traits and how traits are selected under an environment in the course of evolution remain mostly unclear. To approach these issues, we utilize “Dark-fly”, a Drosophila melanogaster line maintained in constant dark conditions for more than 60 years. Our previous analysis identified 220,000 single nucleotide polymorphisms (SNPs) in the Dark-fly genome, but did not clarify which SNPs of Dark-fly are truly adaptive for living in the dark. We found here that Dark-fly dominated over the wild-type fly in a mixed population under dark conditions, and based on this domination we designed an experiment for genome reselection to identify adaptive genes of Dark-fly. For this experiment, large mixed populations of Dark-fly and the wild-type fly were maintained in light conditions or in dark conditions, and the frequencies of Dark-fly SNPs were compared between these populations across the whole genome. We thereby detected condition-dependent selections toward approximately 6% of the genome. In addition, we observed the time-course trajectory of SNP frequency in the mixed populations through generations 0, 22, and 49, which resulted in notable categorization of the selected SNPs into three types with different combinations of positive and negative selections. Our data provided a list of about 100 strong candidate genes associated with the adaptive traits of Dark-fly." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
    <title>Dynamics of Dark-Fly Genome Under Environmental Selections | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"MSLrT2yuDc9VGYiRnLm1x_lCT71UXNWcqmJ6CqC71q4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460074","apath":"\/ggg\/6\/2\/365.atom","pisa":"ggg;6\/2\/365","ac":{"\/ggg\/6\/2\/365.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/2\/365.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/2\/365"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/2\/365"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/2\/365?iss=2":true,"\/content\/6\/2\/365":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TuMWzxCnIFGbX+xwSw8djNI\\\\\\\/QGBh1DyaHklIAhDeO0GxXuonG+74x5ZrLi6SBy4XTGJjjfVBCQlPElIJpNuYwmGkCBQPeTYbJPb1akQNmQnkK2H9Urn3963lG+\\\\\\\/4gniViWzsmddLwEJVnm\\\\\\\/e1Fzjo6VGsktuocX4YDCirY4hnQ3mvltHEAaWXfKb6diSb8xhLTma\\\\\\\/95UVka1NaKwXVvmDKTVgghCTgUWGCgh5CDkPEMWkYDI6X37BnLa4XMPq+ERupiRRSzOVHlS9ZlVyBqR6y7nyYaCI6LS729eCkjZJykY3lvsujRzRyp7ce1EvGLAbkGZg7jovt+6ftyj9Jj9GYB0+aPTYgq4Qyt0Nzbbsxg5UGGhX4jRu2LhDU9xXqQalyjJ6JpFe7osWFmbTltgqdwALlOLOGE1WjCMXdaA\\\\\\\/DGQgvsM+SxdDHlPoTFb4OQR4gb3XYdiykqX\\\\\\\/cvpi\\\\\\\/vSQvBQTpUzTQXLFgPzUskdss9NlYMg3t\\\\\\\/FpXxgEl\\\\\\\/Y2XLW8Ga36zbvCX1tPdGfkANH5j0XT6DFl\\\\\\\/ZEJtUi5F5BlFTRtzusk5+RFqHsH1b6SnFtKKUYQ87HoD6E2ioPegalgQGjzuBLvJ\\\\\\\/Y7zNZc+THDxuRkHuNyOY6QxGsL6bnUCQQRn8yhjxp1cDJdPBDnDkHyY1wLUiLJZepvBYavoOVKxXEfioicQc8qdyrkXGTxay3d6YXbXZV06+YxlPKkIfJJlVesy3uiLdVCpFWkJagVA0IfReIumUJvlhoLtUevVDRNf4YW0YBn7orng7YOIZGvhnRk1unjkdzazYySzC3VjpkFEcjk1f\\\\\\\/7fvcQlFqzqZ8\\\\\\\/ANbzhQDC0tW0lAhwbmOb\\\\\\\/94qObfWDIIPl1TpvGEFI6I13ApQZsydXT7pHRXLZLJ38e2VUnazuvM6XQj4Rrjd6LdKBqkIpm0ptOlsqL7RfKAK3zRILKTG+lFvY3QarAZFFirqq+TkZcDLDp3STx6ilkEsX6UEP4R7YJXzdI\\\\\\\/UeuYB47gOao4dViGiSAJoULiWv9ITbM0F\\\\\\\/RG9p5yfz7S63rNAhhi2YwRt0G+5ghc7c5nBP2GlLF3P3C3vFBcHEG+ZWy2jB02fXF+MMgrQ0vIa87My1K5IRQbrkr0j0RwjfbWRQQDKOqmG+B7mANaHw1J07vYsogSv7AKYCHxhhAEfmcT9kZyEe9er4kBFLuverQa6Lw22uxpnTlsj6SmkQ\\\\\\\/O\\\\\\\/choJu2AWnqFS2wbiUjd69g3tVDA5crxi9zv1G6T6HLnNg9xsGfrCYb7kh12shj7\\\u0022,\\\u0022iv\\\u0022:\\\u0022ukwH5rMmE4YIwLQTMZ35bw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022acd46a28e856c91938bc0fcc6e39d091fd6998f7c470cebe9463aa7ad97c542c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r7bCTJuWKKNCU\\\\\\\/5YWQM7aJX57QFVazi5bvOAtW82F3DfNOv+HN5ucnzbiJEIv2YuhUHfrF9hCffRNFkTu5AE6NOrVZcdmHp7fy3orAn9USeVZt2w50UMt89gRykiuwHyJ+LfimT1kCJFyoh5FJc2CC\\\\\\\/eJO+SoEkYyTo5\\\\\\\/CG\\\\\\\/qLp\\\\\\\/cOiTYcT6mUeEFv7xeSmwDlePMdniSv5\\\\\\\/E+7kGPPI90RznV7F\\\\\\\/5i\\\\\\\/i7m33W+gKsq6zFxZUPwcdA3JmLTYNynbxMGYdXHGm4To2rIBLQz8rv\\\\\\\/5YgY9c5mLceen4JuqToDizi0hasG\\\\\\\/c71q1ia0SC78llbRXCAoQwactA5UY4mQGax3yYZutX36xAVBPjJIITw3q4Fjz1JuMQhsgyE3rQJFx7dEWfUsdp0xT494+7wxRaoXzmOFUFH17EbcC4LdN1fT5FL1cdJk6ckQ65uuY\\\\\\\/FoYVFWSuoziBnYFnz5kMJV7yzWVg54gff87394nXHlUH3UAMSd0ANI95FJftHKzcQ9c+FR7HSIl9Ax8hM62TAPBstQT37Ux4PcjThBu0zIiM2AeFpy0t+0hv2qdxScqEE18J164dljhG+FbAvvXlaJmvfkyQ\\\\\\\/kglU+eECrFKUiyPAUFF0AvcqZvVhm6RolU1600ChvAhQI\\\\\\\/RY1pzuqiMpudSQy4kP0TQa0m3tTeVaScD\\\\\\\/TFUVA+cS5SuH\\\\\\\/n+V3bgCuS3Lro1OPMpzBEBd+q0rqkbHthH0wqkkXuMpZgmC00RUcKv3s2oT9Xzq5Urix1sSF6VovLETWLba9YLL4Ul6m1CU8mJAK7P\\\\\\\/6QYPvH\\\\\\\/6M5v7gA+6hl8n5i0MdNG\\\\\\\/tys4sqpzhVM+4AC9Mx57yErtDkwp9C6WjQ7U6ynbF2wQ7zSW9gld+3yxLQfHAi6F\\\\\\\/u1hJG0oYwQvFR2+da\\\\\\\/tcm2RSCpcEXWxO2GWSIQsT+5U8FxyyqOy5eKkNSRn1+1ZUlfyutRNNM61R3XD31ysoAEYup5Fb55Dw1w+PH1VX41Cw6XH\\\\\\\/y96LEAtxV25EerzJrgGY9kzaZzbhUGCriXtKaZj6Mwf13APi5sTfN9ai8aEoVo7\\\\\\\/bDg20qcgYFZGztovS\\\\\\\/hmuB1jHlMEWZO+bFpf0saeAvI9XkRyzL\\\\\\\/3JuT60k3\\\\\\\/6wMbzBfgAD8ZXz9+jw2O0VaKe+HluesoAt8y9QYKJ+5hy0vgrl5eeNXGM6tseZx7+mbv8C94u5wDgjJHYXY2R9lbldhQHeF\\\\\\\/iW5GP9fQxKsxCARj0onL1JcwXy9GrsJa7q6XQREOKXeHxSK1qgkYdVZ58pqpN4Wx+urCmNs\\\\\\\/g4XmCCTY+WNDORAQ\\\\\\\/Y\\\\\\\/AX2fEo\\\\\\\/V06DT2GiX53e0L0zoE2DpFgguPRT3leSFe\\\\\\\/mVnUoorH\\\\\\\/YOB4lPh7Bhr2nKGBHv7I11vXe84j+bKuHs+sJJS4Rw7lA1GHwaj+WTgS+CDMKd9fN8offBNL\\\\\\\/WdzkwfarUiYb00LCITBuEwbQESrKNqMCutSRcmCL\\\\\\\/5tQoxoJ4cR65AhozvTK50+hDRuVHVabX3MGvKN5VggbbDcBoaW8\\\\\\\/kdH2GKqGGv2cF6o9hBmOdF+yJUeFWJEZ\\\\\\\/Gc5suDy6oxByyBw\\\\\\\/zxx1GbAwwuaZbfwtLL89wg5aaUU46+o7NtrXbYC8+sZa99m1Mn0fhCAX1zJnN4aGa79C3N9gaiyZegHRVF+6liu5AffHutmmlq\\\\\\\/zIiJHIHlNXagSOMPXrLY0VimBZD\\\\\\\/trjGsNgwS\\\\\\\/i4TeqpX9SHBfey6\\\\\\\/7KeA1vK8WmaXxTu7jeiBfkxL3TBntvzoGAVrdNRPAtr1sZ4FInMtgzfvxOsW6XIRjjyJ4uZ0\\\\\\\/dgczmZ+kevwR1xiXEt7RHU+frWj8elV6cFOJqYQGyTDp87jkpI6KQLrHxJ9+jg1HGqVcryHwnYi4iGC7N\\\\\\\/cYSK8xuVPTKvifQS3SR5Z5DO4SZlTZi7lZfKHtA0rayhRY6bXyvkVYe7UE\\\\\\\/R3stVAuWFL3OvNRV3CZPk098IxdIcUnMy7uCCm5jtYj6PrzkbV5QLdLz47xcyFlFtnNbi9CvmfHMtOKSEhaZuJcvh8PCevfiPYvBtBoCY0iYsst02nXqGs6N2RdZtpXlT3amOPtKwGjPplnRNp54J4UcnXI09tw+ctW1X0ajIhoP1hDiSaVjhlqAz2PMoDWoW1YlBpSTpqnmOvL8\\\\\\\/E+BpzmGZCA3hYBBESGjQdxe+TUQyyW7aLJUHrKxUOXC4ECORoZcTSXfi6sdEcYQjLcO1U+Y4tMnjeMEcRsTWUrIJzsdP9zofG8K8d5U3FRLMpCgZrYtKXGZ6knYK1Zi0XE8vFt1WtPS98wo07MuKiXRhH9BiIMfV6Pop8Y5\\\\\\\/qv05vmeYkBVzj95YljelvzMLCajYPojYx5uhs64NBHSj9+IQVrOtGTstwl3PLwo83ssR4g00\\\\\\\/5ag3MJi\\\\\\\/SsmyzlhI6kCHvQYT4I846VYS82JgBcmfPDxdqIdWGU+uatvdwmqbq8liCKKJVSlF5B0olU447SwB12Q8zs3Qwcenhjxd5ciKeXj2taLuOqM\\\\\\\/IjIxuKo5RIbHhcQ8FSX7COBgUmxe52lB92AJCgfi\\\\\\\/QavVW9i0HNQeAvBYh9\\\\\\\/sm10ckcX6SyMFT\\\\\\\/x1ZhSN+obZCbkgEUg9Q3sTeuHLU8BbIVAdbzr69sFCk0O51P\\\\\\\/pAvS8UENSL7u+M85e01hAxQcfijCUOxWPxRm\\\\\\\/689hIPIgI++Ilr2JuBsNPQJOaGq8igeBnHXim20dRfICl\\\\\\\/GE2MYIJjX\\\\\\\/aVKbttI5Vzs1PjzNPDqMVFratFdFOdtfAJa3sAMrWlLIWEoPtbDZgpoE0QA0S97VhDRU8U57oUBxanKm9b5JMYtOk5yCuAJqMQEYTA4GLbbZfp2O49ZtjSH5kK\\\\\\\/K5Nbvs4ztmbxhtkPVhibqC9ixq98N1LKtQZLFpFOonHSIkDeL9itftXZkG6W927v99LQke7IdnpRJgakihtBCKwfOkElVcj1OHE3VS3dvGX02C5m\\\\\\\/7UYxjcvjCUFH6Oe44RMhUnqmg\\\\\\\/w5vSz4FIVKbB61Y8CV4EZE89pOoFwTfOSWrEh3qmsUvL5l8bQt\\\\\\\/GYqGi00sqBmLVp4EHziBhP2mHp3TJbwEnTOFWvaqRkSPpm93wQ69GU+JfrsfLTSQyNqje05i8zpPrMR+jtibAhKlsBKoCZkb8wm2GYQDWmaGZde+Yw80GAjhfrC9d2vVVDkQfgXSy1ul2cfUyggqyPW7cTDe3e+ddNK\\\\\\\/WzqZCJ4aJrtSSoCIdnDF3nC916JmC+L6NeGFN2dgjsv+A6r9EaQmFnrW1ppfazYPDG9csMB4+x44XIjQuTnVZuyjRNJ+EbDG2BGk9p3h5OoMXOHZ2qoqUa851a3ZITo03SKmoAtT1dxfSktUPCHOO4AnfVJyijksnYfgcpzgb9KOrGGkMlz2HFxQKi\\\\\\\/VkkC1c4wWhK0YxC2grFlb84gGMZOCXxxlEBlzQPCDOfBBHIcyLw30IY7IAP7NvAmWYJDyHl9YxVbFhP0eK4ul0JXh0XgOmjaypL2E730FcfSOklXWZ9ZtzMk+sd9\\\\\\\/vWlmX5pMTTLD3i3RVyO19nkN2curZBKkbecga2yeeMGnuPowuDFQIu9F9ue9pAX+eu6TQGdNkAditYXLppy6UoneDNFp3Cq2qcBCfWuNEiu43PlGZZTipQFxZro9FQ5CHt6sb52in8C2kClMoFhwkQal71cE217ZnxIN1NeIRzSNRoGaRg+7Kh2eip4YqLC+8K72ny96O80A7GhZvX9ZKzSWd7zF50itsKcaBW\\\\\\\/ZXKmlmJc8fU+wT0ZfAGplorr5yrB\\\\\\\/JMXYTnKtccyGyYy+osXi24BWPKs1KX0hrP\\\\\\\/nL9vwgX\\\\\\\/OqDe+D4GSnHr4a3rt2NXW2hxUp7Pqi7KlVbTmy7Azyrb3OSet8Hx9utsGFMA5iu6c3KnZC6Dj3T6syBIA\\\\\\\/+ydfWpKBNQmUuEC+UWil1fsGCd19riCVePdRyKxkw\\\\\\\/VvyEYaWXpTHI+FsKcXu3hHUfmyXT7\\\\\\\/TnvijrjYoptqVyj2B5QC61EkWxm9JNOIPgj4cG1xD2o9Diy1BH3sH6x28wlZO8c0bU\\\\\\\/knOCj0t4Y5\\\\\\\/xiHSgX8OEjkIUJ9+Ck0+c2+XpYf1h1JPgcclTdjeHwEAP9AC12\\\\\\\/P2MIanKf7\\\\\\\/Wfb\\\\\\\/ahcZ032idxYo06SPYO1PdthzOoyKnp4suoIfbCrTW94EWxP2P9ZcLak50VD3hSS7ZqZKAs6JAWz7khb4AZUjZrs0wCAm8+mdeoMZ6IOQbf2IzIzQSpxwjUdLyOA8aGu0B8ytj\\\\\\\/T2IBoOyUpAh7ky1t8NSKrJek1YdX0JPuCnkkk2WmTV\\\\\\\/eP3J66UJfQoTokAlGbKFVUfh1vqssgZEzAbwMZ\\\\\\\/ryTxU4V\\\\\\\/rTBtCLzI3MCL2c3EV27Hp\\\\\\\/zKypUIpTDbCYl+48ypk9PnmOqs48N2KycAWOqYzI31Ubpi2x5j1HDA8DC2rItceOdQgdbvNB2agpKJgVy7HsRPoIG2zqORZAviaVXvDWCSsYNfHtFuGegZ5wY78MY91z9yOk4qu3x4hxpkG8HP8fuLii9NrLldLV6fCW1p\\\\\\\/JUizqlVNjIdo38DwkByp0e9XK82xNvywWnwu3iQheFUP4ggEkRmD4IHKxc9eNHfKVJfpsCaYFZzfTfQSnLFUMK2y\\\\\\\/VCVvbYn0NmeByvcI+Q09dGqwGFoHD2RARG1BCb6kWR7S73lUXOimNjUmFGt2YpBYIY4mQrtN6osggP89j6Buulo+HFTufZxDFzK2gpflfxLmnju+VX01qU3mYz653n8\\\\\\\/8\\\\\\\/sIMIVTo6pqVSRKT6BfAE3f7NoDj+prQmV9TZOQqzZ71vlR20FbS1O4W9eU98sf1cBAH724glmHAbIchTYzPZyd4xGEnBE6JlVzAeSdyRqBFvOKWUSTCLW1RbnoFWmfKi7qGvJHepTGckBXlPo5D7qM9kIbwX9P7XmHmaCe1PLWZP7AZs6rP1riCKqoxhIc9dL+1f7h9GA1Taq1ouBm4\\\\\\\/bTbtFzT4S9KlgaoO08bkR56wicJhpjUZHMR6OoUUauYWts18NEdxFMjinRD2yaE7CtiSRG4pAXVjSd8QRCslMPAMh0d8KPGPVnkdA3YwwW\\\\\\\/R7SHmTP6AqGgt9tZpaN1KcuL2A2z9zKksmQEE4wSJUBJ7ZnlK\\\\\\\/8jk2ttZLiUeBWzTO7wG\\\\\\\/dxW\\\\\\\/MEtI3KBAUvKLwXhxJLf2yhfNZw9GnksDsg4vmK6qcT\\\\\\\/dkMGPdkwJlTcGEjm8IOYCyXni4Iddo5Dc9\\\\\\\/mGlfPGgnWPWPpU8Nili269RLC5Z+nZCujCYyQBkliaLeJCjJaNN22xhIqnAkycLMAeZTUZMuRBEKsySERvVWrDPVD7UD5ENKV1kZzsBfp08Vad8rOYUt0lWN8xgANYDq+zLK\\\\\\\/UBZ1CxVqQWNn\\\\\\\/x9Bp3vWzphJjgDwGhJEuXp8a3hbBviK6TIYpo5BY5ht27UO61FVeQ6bFU67SBqh1iwfCiAH52YUhE5YEIG5SQahC0bf7\\\\\\\/8TNFr3\\\\\\\/Ih6lEY9f31Ev1RzqHT0LkvwHPeBE1P7RVKDC1Rn5RnYYx4J0ZeCgvEtLLUPNiztAsQldY1aQhQvjfCf3l3UoBudfY+T0msdvGnKFqMxk9FcdaUOIGNgTAZ44GJcyo9MlsezHmC29TiQY0RTKqtca9OEodrAywn2y3wTZ2fcXh9T43TR8fRY4QK2OOrkVfs9sGXEDCnd98axeIl8elKZe5z9Q4Aw4rFRWDgsZFKWobsLIo9oMFgO4eo0KuMSK2pAl03TLAxXU+Vsju8+FJInlsv1krGfkxDANMRKz0PZTTrc7xW+Q9MVFQ2Uf3aL8F1qCc\\\\\\\/UXTYVuNVBJo\\\\\\\/Eb7uTMLWtsK5i3\\\\\\\/C3lHfSKt63\\\\\\\/1T94dvcaDhuw4eZnwtkX9uQJN\\\\\\\/2O8QThJkKp01I+WHBQ+2QDKVSF+CgpR9vM660rg8lG+e1KcYAzkinXHnbQR3JFXeT81TkxEJ5pGpF4rUylsqYDz7dCes\\\\\\\/+UtktNbtCiWt7tTntg3NzDY8iEmMeviocMNpgtB4ylXRzs6vJcf6Ut78ZpaeIZtvDJ2wXi54eXVAXP36\\\\\\\/kkU7exbMEGGbncXIpmlF9CBfGHBzEH\\\\\\\/06gvTIaIhDQo7WFOqdp8cco64D6GvpKRzS2BdcBJygrCPnO95kANKwcUD4zMHTLJQJ6vfZnaPaHjdeLhq5N5Tk2n71V8kQHA0m2t+bWoNdKCV3AJ9uNQvYqaBqBEp7mGsYKZN+FkP2PDVB4RCOCsV41aX5PUPGEo5MlYl6x828FtEHf4QeY\\\\\\\/ssnP7Sji6AI6dalQQaeYxBf0N4\\\\\\\/Oz0l1J3BRJ33UmUzdTtVN1NcTedGH3VdhN6ZKw94mE2Zydi1sykBEizYeaw1yHa5Ux8lQKZCkToLpMomMzv0ye9I21OkycflIQOfcLCZhi4sHKIBc5u6ZZG4AxQRneu+qEac6Yoy\\\\\\\/pB\\\\\\\/917RLiH26f3pmMwaXBBGO0OEohuMqrhPawL1w6ROFW4i\\\\\\\/IqDA3weKS57Nd6UpJxncvwWgISW2g6rHHGv2ERNGPw0GbtXIRV69Q5\\\\\\\/NAqzWLSq6cotwyMWj6se05WJ\\\\\\\/ZvVzh2Uw6FOHj8pq96ViHvCbDE0wGYagkk5glaVv+w2ZLw7g0GoCpPHjGP+eQgSWLHv5ec071UkkHNtJpsGRhIitfV7fACJEc3g5xQwBqGZUjM7J5R3v\\\\\\\/P\\\\\\\/f2\\\\\\\/K2Wc7dKF\\\\\\\/SOqXnV65pc6g07uwNVTt0t\\\\\\\/nqZBwTdX8doPrIvZJxUP2rpOvBxs6Fazxgbto82uv8gcrSJFutDyAR7WXSCbrlMU+KKovjsuULDSz6znwYI+U5XgH8tmf6ZfJC+Q0wpeLnfi\\\\\\\/a\\\\\\\/Itrn54V0Juv9oCaSdwauoRNv4jXRT93MMnjMVs2W2hX3yJRyyGiPlwqBrinEVSzZ6WzDKanQuUQv0an6OMIIPmPTbYXh0gkHLnGc1hD9s+J\\\\\\\/rTsDNsH\\\\\\\/AudsnO1gSXFu5rmt1+h0BBei5DYPBgf9PbBEkpUFhfoRuSok+we1ZOUO2\\\\\\\/KzddeJoUE0jW68iAI0NKVcTnxZAv5AQo3c0m6A0lJh8i7GP216f50by32DkrQqmL7I6\\\\\\\/VSbmkK26581uhGToGSpfGkD4UfnphumA61Q5UmJ2lOsxQuBoklNiRv7QozkBNwuQ9ljwzfF1AQfCxdu2mtElmWw5ogIdY7hq2IrOPndJJzRbIjmTp+xGaTlTfO48LW+MpGP+c6HPyoz268BFQgkhxDad6dh6AIkc8OgwcJOS1hLV9GwZ9HWH1ZImbPFXCR1LTgKu9BcQT7nTwKsChyxo\\\\\\\/aubiTjZKNnaAK\\\\\\\/x38LADynUGZcoGbxCAK2k2mhmJ58iXdxB5EaRRobZnGB8j0FBHsUdJJ892dXswXp6PFbjd\\\\\\\/NLR51Z9AHBH0LhcfnO+e1THp1go4Sx+hldeyitp5FdcY6jKoH\\\\\\\/47kze2sMi1l6gNue40Ze00JC2q3o+PIf+XLJuiAavb8kWAX1ilpHRT9+dUyWIqGzRNfA4GGM17eTE83T\\\\\\\/GRm47pNUeBLVnoM5piAuUg1znzmCYhIQ+QclT7zi6GhZkBhzO4ye9AGY+olEzMBEpE+UeV5sTCxOWc5l\\\\\\\/X0oKKF9efzYmsPUSrDqUQ+rvaygZyH714GK2637fyS\\\\\\\/2uQCfrl0N2ArfKPwUCKERcGa4ChuDvC37fNLRKB3SX270kwK3j3F0KZ6B\\\\\\\/Oa5+0qQYD6dgCiA7rzCOQCDPnHCagx\\\\\\\/ro8lLI0ZpqQZ4HWw5aqAcVezGbCCP5uqY\\\\\\\/ZYrpDF2OYdqq+MLO013JMQqpMJYu11tU9SdwTw\\\\\\\/RvcBdwwt9irye0ZwRDBjCA0LDhqeJSfVMU8tlruRGuKPtLMv80Dlo5YgqQgYPd9o\\\\\\\/7gw5cCX+WxaHdkeUFIntACbgQvdcqsCPvfxcl9sd6NvdooPBVUOGArF2mFP7iaJlgAz+zRURsMzPBux6x4MCH0gP+xWlxpukLsGsSPbbkTfNbOCty+pVBU4r5IIvvX1yUgjNfXJMJYR7Rh08Pgtf3\\\\\\\/49iDgrvrzcaa7YWd23GLEzuvjlh8\\\\\\\/rfW3DtfcaNVVTj7Qv4adZVmyHbWTMKf8jOLUKTrNwJ4t8MuLAiKeR9fV+QlBjP43mgSxOCSCV5xo33kuKtRTFTJYtGkBXBza6sPyMnV8UVSrgRNkREmaGQuhidB3s\\\\\\\/ScvgjBAkCgaU42LTedvi2YaDnwM6vrZWoTwhGnNM4f6EQTTBOO7SYTC2cOMZeuEdCNMwzo0rBe09FO\\\\\\\/C3sDo2Iy6D17DxlkDzavB3WSqbyf6ClyDaheDQb3v8dOwyNq7phQDAbVcQ+wMfgvTR6ChpHz1RQBuMuFDFtYcPkomiVZR4XOnxjsOYb8eJpOeZcfqflglMWf4o7ADzbO2\\\\\\\/wHmPN\\\\\\\/E3nlVPxcEQBoKAaKGkt9hIkM9tbuNroVQils9rJDZ\\\\\\\/QcbU3hg7r4RmaE3+Jb2+ZNLt6YffqPKEw34RZe+uZP6n4QGdvLhSjlLx7qbhscFsWMe037bF61\\\\\\\/H94kAeHUVt2xSXeY57B18JF2+k5w+ZXEYmAhhnnQ9GWPFqFpwuI7Q9XlSWXzpZTCtyQTg9kcGTggAcb5OF09pLskaQw67ClEBfKs7HKKipUzCvhbToukSKuDfbV\\\\\\\/hJlYnDwcZjkAKY2gMnfsXoAYqBUJjSwbqaRN7Zd7MiAFQHM4diTG7XvxmQ5FFX3Ve5znE9x9mzxe6UTYV03mfRNA1A1Ko3pmg6nLETsT1Uu6JICnpEKhnsuMzAxvq5cvfozNqiMJfzM0wd6251J8etNq2OsaxgVIIy02g2BLE70dTiR2EDEY1HTcbqLWaZqojEo3QizGTEfP7VKNGSW2JefcRRljpjpGLnb5JLkDHpMZ21QbHFgJnF0MfwfoinuBw08wg3cUZxd53H9cpLMIqdN4s\\\\\\\/aityflOyvFwxQajFOoyEl6YWzDPFwzHUgicEj+JZh74hdfiiEI6+RpdcIPgJ1rvc6PSMJ\\\\\\\/L1BbCOkJzDz8VKfjVFseYOJLqzzKHsQ6A7pChycj0hMhF4EKCTzZAYbg7C13gK4Up9qd0WUj8rNLJ4ELjrViQu7HsJB6qNaiqKuu1BqSLpijIDbV3WaH42P2U6PaRZP9NQ6l+WFjBsVZK+cv8FrWhFJzrGPwyKy0ogUbRq13Nv3bfqZu4cV3Z8mVf\\\\\\\/5H1xONV582VB8QEfeKS7sO1B2UBH835nsffz2O2MwHPvK5AXjRH3JJSlLWdx2RTJ\\\\\\\/qhXgE4nZ0aVP\\\\\\\/+2Cv6GjEDw2PFmvqtKDyzPWmBcWsOeBc9ANIlo8EM1Bou0JKL3\\\\\\\/oDyBn+NrFZyqoOMFIt9TjYadWiONR4yFZ6I2THQSkGBMyCC6CkabKoWGRq5X+wC36gQzfMsjuJ8py0kdFF0q+0HQYWdMf7BxXhQWrkcwIIcA3q2XLipAz0vY+tL1bvNbTUAGyyAI8\\\\\\\/V4XylLReSYMNAyLvO9p5tqO3NCpEsR1hoVT1HpuqKn0g6a7I+ewlrdyhkj8X4UY41kMVe0XybW+av7ps4bDEyKl5sJDJ2sTwdwwihatZjsbADZ3zzNPiKJOz8cwqMTcZnqvzxH3R1xuN4l+YcBYwGMurJD4oGqs4Z304kETlTobuurg9GpJ1eukTVW8LPN1+TJzKUqQe2+4w4908UNauEKBVnOEAXQ\\\\\\\/hBEPKZ5Ny8ntt6OpRPzrERb1H7KWKML1Hst\\\\\\\/0r3Hm2QqCQsAiZSL1bdEGtMms+CEbIAzX6f30xs6\\\\\\\/6hE+6VPbgbC9KS+8uJ09DIKKTxysH2JiyzK\\\\\\\/kQlTLAaqnKPO7ppKi6zQLHp12XXnoyMJNr2CtfPXMeFtnRHIFtZLYY8qMEEF9iyaVSyPlKW25mYw7I7jBIhWfQOQX1dkrrY425cq0sHMY+qliOLPq+KP1XElJmATonP1uJunPSqPctvXJZgz7NaewZn1Xc4huJFcj7PKLy4RNUV4W+o\\\\\\\/m1SiIQoZfsl2HmuLxIOB\\\\\\\/qkQ7s+TavTFPEieOwpr\\\\\\\/8nathxD0WnjwSY3qvF21f0Qm7C29xZ405+K\\\\\\\/vo2s2+nravlzjluD43Vv8SYYNAO3XUUUaglkGkxK3OLKGazwF6flhioss41V\\\\\\\/EWR8pKO6P+t7+7XIC59wPibdZ\\\\\\\/q9bl1du3Tr5zP5uawFuLF3lLENConFBRlDpAvAXNN\\\\\\\/UBA4OqvqoxxyZ+DyTEWxoDWeATJ7F2+wlZ9wTICOjCrP9ZmsSVqC5+Y6R0aq+qHzujQTs\\\\\\\/WaaphC6PAtozAtsPZVMg4oxJqbJ5i6P+fKI3jxBhPt8lmI5hX+F5\\\\\\\/ZxBxrbVi4+mtgfDenbhES99\\\\\\\/Vij6R+QK69AShohKcyrO5YucucDvNRfg8ELKuHtFuPVnQ4xqo8sKb9VLzggHP4IOAjoT8ZSIku1fv4Zx9O7a1ub4MjpyEO2vDeWkTwdCX7Pb6Qk7ozIclylApWicRgquItunFf71yK70ZmUbcOOv3J1mhOYR9N5v2J6w9umjfz6Vi2F5e0bF2VEUw8NuUMPa6blLtjB94xcNbdnwU6hUgs+P8CMkZN5i8uo3d+DXqQ7C0yfiAAqd4KkQ3ncO6f\\\\\\\/QuSd8qAFwmzPafRe45nrnQYNWaxUIjukEz0WQu\\\\\\\/ajYN5bwNTcsX0pTyOFYMpHaZq7epuGF5nszzgsvAhWcKbFxm7Zv3a3+gxxcBJ3gQNjJZD2\\\\\\\/WBrauBZsGpzzdwolzsxskwT7e9vfPjTCxyRK9tLcDe0UcvXsX6alLculm+5VkU39Es1vvgOi4Jb9z8aEsBL7db77LD8gROCFFK27s3knXblfmUKtZ2uFWR4sREaVq95AAFtmRVjmRHdElljDDm0nMHEgLxvU8JznnYilq35d8ySHT50iwl1J3DeN\\\\\\\/lq9zmGxQukEJd68FbCS7SYd3utLrzBCJRTSTKG8JBib+TdB3jMnrenawXQrmofcKP3DrNEjClZqK6WnbN+orLh\\\\\\\/ZlHRCB\\\\\\\/egGr1jTqj1pr6E8S5V+TUpJUAbCIHYEb0SK8VPqq2Kd7ufT842JcOBJk+Cz6+rM\\\\\\\/FuNG8VAEH9ID9UwigPjcPHEr7fp4cGCje0XyI1qwCt9NBLLbLcRdEe2rS\\\\\\\/UFA5ayq3u7oFwgJP5OhkaCfosNmLNCM49EeHAH+hM0mgwcSxLLk1Gq0AGwP9XiHLrthMAQsJ2EsCA4cJH5qqYy4+d267\\\\\\\/ym1D\\\\\\\/oZRjf18fyrXravz9s0sKWFm6NCDjxFIZ2c5KSNMbBlFOIbrBWc9hBHFKpog\\\\\\\/P2SFefyxkdiVmqCgMZQdeu3NEV0MaiIiKn2GiAjIUBkTb2W2jyJOu5ufOPwGV11g3RTw\\\\\\\/PZ2Iq8vK3UI5iI34h\\\\\\\/LD1\\\\\\\/Bp4sGs+nX+otWHDCVg3SFq7KJ+w10qRqAix1+BjsgNy146zVxviQITvklWOMzifWIah15Jf3W\\\\\\\/QkYWa5f2cBg+Xz86ruHIlOg0kqH38vEmk2myUUNeBu6mYV6Ng4rpnEAUYymlPSKCYDstT+mNoW8n0UHDse5FJlHUsn1GCh05QPIc0NF\\\\\\\/NNYcVkBGoTtVXmQL5KfvMH7APo\\\\\\\/fVdugb77Lul9KpYMRMJnr6RCrKioAXpBmbO+picqgp1To2gSrLmEzY6hkubaLIRqAYg4OfHSMLN9Qcnrur9O2IEfcR9nWzngdYpvU6Sjmijf+1dR8qPUxKgfOUwM1jht1E6DGBqAbnjglsJxL4Ca3XUK2WbMtoNvqixKm1i4ja2gmGGaOp7t5i\\\\\\\/DniuqC7WneeolGOARj4lTPFPaWRnECUqMUhHf8OGih4d7z14QWfVUwZT6jC3rhPgOkisIj86NiZcctoSaKWIa\\\\\\\/q3m1a3y09U\\\\\\\/IjG+0SSeyuscVOdA7grYzZ\\\\\\\/nFrCYy6iu4\\\\\\\/TOFTREYJRgw226DBepGSji+tb86zQYPy83JUWk8lqOy4w2RQmX7q8TZ12eYiwh8WpAlak5Z77E6L2VbbAb+1OVKZIG\\\\\\\/QTMiTART7N5QgBtpUIDv6GgPH\\\\\\\/nHgIAtRf1qBLWYTf876JlSpK2xJ43E+1L+VzoHuwDcrI6Ky0goRelI4jLtIZMp3lawtadQXtqrhX3a23fNZpfOExRGL5mvioMj\\\\\\\/M2k1VZe0iQeSYlymw\\\\\\\/Pk0N8AzZ6qxjJAevWL3+\\\\\\\/LcQE6H2CYq+6dVF89\\\\\\\/OA8yRECDE2w5+LJQB3\\\\\\\/dLBogk2ZJYuM3ym+Wjht55TRcCGRpXkDGmm1Baj0D7NpKicS7xqIu6rnUDKQBd3ppXjcW52CjfXi5WnXorfCw\\\\\\\/zPCEu3aFYNLBrABQQsZA0EtYEfynXFv1m15Mh9NgHRWQlTh4D93kj3xRzuAkxmK8M9FJVsDpULyWvJTTY2+jYlpq\\\\\\\/EDBpNPCSMidiFM9q2v3dExY5S0pBiDoLm6jcspJspeUxOklP\\\\\\\/aLhIfaV\\\\\\\/h5kM0CMrMl8uevb3fOYr8QlcoP\\\\\\\/bZv4oFIKkTCsCQCu\\\\\\\/ejrSTKAuKm454JZJdRclGYyZO9vAMu9OHAVEKocrUNPINd4i2lnx9v1aUfeCsrou\\\\\\\/rYinIlKAoK26U1VWb5qgDbcmYb\\\\\\\/HQBpEvBKPOQ7YsXdrjrzVYwXj6LL45rNVe8h4wz10lOSCxaHNr\\\\\\\/YLdW6Qg9fqy7G2jHy8iRZNmF9K+dIxOj3t4QAI01z0Ve3EyzOkTv4r\\\\\\\/6YqPIjZQG6bxcVoLayIlNJ7yDLlYgWIKQMRtDgTHtC5AjkjZL+hssyLgjXJ7gNJF2SWXZ6Wzrx2DGl9Bx2czGzXkmWzNtpKuxcBuY1pdVdDiHEtzXAiwh5cAXx1avb+RKbOrrIyL0ZrBVIFHO3ARaq91YZnqeiroSpCMXJWF5ywedOq71gB3IaXpsOVFu1A6sE8o5UN\\\\\\\/zzTju8xrelzbVsMJtdiNy0hescqpGU0y7CHrCwRSAMhewgesFmrm1y\\\\\\\/vf1K7f6Bf2kOccnkLQKHQ2JTtOVpOaGRk8cY4I5iriYiubUkmxrTAogXYXzXP\\\\\\\/M+N1AQI6qdthx0LCxMQPSM1g212CbaYQISk5AX4xuH61z6z2Ucgi2c16oK23CTmWbwSDrRqUEWTWUH1oBZZ+jsBwXgFzzRtsB0imVTLa165h9dbwCYRW\\\\\\\/GDVlI3NiI+bT9dm8YINEIUnoUJQd00VnxKqIFrMyLDe+S9pOp8uGE2uijORquMwWSoEtQ\\\\\\\/y\\\\\\\/rrHXm6nXA8ugpn\\\\\\\/BsEO7LjWmRwwgiPtMjP6gfOdAeohyIc2rFZ5eTkzntxn1Z4LSjQVcZLOJH+w6MHA\\\\\\\/PZd13EI6ktCMKSUjNvWqbXkVAMfDVGVjdeSOEO+6gxpKnTEc8XgoBD8me2gqU531O8eHnBOXDpyTBe2URl1QRBAGIiOK8vdGy9E+DK+SXcLfuL5jivLEEfhqI2QPza73BZMnlefVnAAP\\\\\\\/DEjGQjN1Iq8H0rNZQKl9LGYK8dhdpTBwvLiLL6egQl9G\\\\\\\/TnLBBVcvglCGwRJK91eGy1ymH6kl80qBiYmNE6eadnP+t0D1UpjvisB0+KOzotUs0Zb9NHppNHjbk+05Sz4Ypc\\\\\\\/RqHK8Vr\\\\\\\/0mfj60QqLVteFEaUfW0WR5R+VQfPhDJBT9eakh9MaZSt4i6qV2bDMmIx4MzasWeAl\\\\\\\/PVoWjuGGhxa\\\\\\\/1P9G8NgYPilc0z0vo6Hw8ASi9Q4YXiMVvP\\\\\\\/3zuyfQylZHyYaflDETRR9Nn2yCbJPUWqIVEyLAQcPUJAL3c+WPNtZ6xEEtpVCC0Wys4kwFtlgFDCYqHFHg5g15TkSJVMVSpAwtYE7CL7iNE\\\\\\\/MOHmyyelZfUkHV76nhQLFGN6P8omxZmc+lUOiTgxTm4Ml\\\\\\\/u7vaLKeG0ptL1YyV90C5a2WEJd9206q+O3HuThy0uucoqkqC5QFmmkgrHQLd7p+z7mejS7xuzwCsJ6PXyvRHaCzwhUU15Lj\\\\\\\/kH1GHV8rf4fIjD8Sh1Z7DLxOeYd5S4H\\\\\\\/pTwbo6VAG6Zpn9b\\\\\\\/zzHRUjkchoN\\\\\\\/wBPCtaOiqK9G9iyM4bM+7+tsRLuJ+uHsPLqsuZs6kj3AugAouOuVZXWCOpNVzUvWr9FowCv8vflrYlORjtXNDjlDrVCqix1IiG\\\\\\\/shb9+sPgCD532d+WUQN2CdwVhFfceUbJ+SJ5r+2hqpYIlPjaxB9Zf05QG8+eiGT4AVZXakZiIr9qqPnuWm2EJtzqnlcVNq8fL7pY3pg0iyKOEsgTexM8M0R+3ZSLvWPGc1G03xAvqEFM2IrlKMoyTD5Eg21MRtJRbhfMgq3pkuA+E9qnti7RQ9af0z8zmMFnYU\\\\\\\/NNBTYeJFkbe\\\\\\\/wCFN26Ta9VgOXxm2nTQFT48+Hu0gvX47AzIPxgamyteaS+AcvGyQxyAF921oaUun4xdNwkr5CS1TrJEIFyKga7dFGIq3jVVe4qhp4LBXdp75V1C2oPl7ixkozoFlxzT52bHpFHII0eVAiL+9kZTE762iUKqBIqfm9NuQw7kANNGgqIf17hmJQEoYwS1MJP8xFOyMwpA5aKrEX4ckaDngtzPrlmyFfrTp\\\\\\\/EV25rO2gk6nlHiLKkHD9G7BCJU2XiMhD26IlmSJj98a+pW20ur5PRNiIAOa4JYpH9zJ2TT7Yqd9QBcSB7eLPT85oSD1worvvFqZYp\\\\\\\/EBQQwQEplfmrDGH6P5sunkRtUfZLFFOTAsq3lNXSTIQvttbwB9ZFhIcrHQcTHIyAkaPEbKPPKwD8VyT0X2Jio5KTRzDMdlm0YQLiT0Y34LGkb93AD5ZqQFIdEqRG6A9OuAnGLcUeMI\\\\\\\/mzxGWpC4xFSRaKGtNJHvowsiw9KcCT7tGYnMgCj5QYxbBxh6dUM\\\\\\\/uGhGayXmCQI+vMf1qX6NIBjLPblPNP9DSN6N9bCVrs\\\\\\\/ZvuOtYqfFoQHUKrKQigh+Po\\\\\\\/lXtmzv33gf8R8FvrRXf5oj7hVXcWYZSHmHwAq5EFiK+X7AxxNy+Lx7wpqMsick\\\\\\\/Yj3xTD4C43S\\\\\\\/YCAVYX9lRsi8E\\\\\\\/oC7tOev7ZcYXCNfFFA7H276Z7qFD2YbnTtMR10B+X5JgozUDN0QuEQFPKQE4t1S7GfTgkggmpYdziEtxNDziyNb2L9JWKrwXTDBKhPXrJYDUzpLFzREEqssQ161g5Sz9FMpjGCUzCBOLzrgxZ8ss+ZXzepl9s15NfYDj25K\\\\\\\/jOJu2jyvWlaEAuWiul5N0eMGI0lI4XKtxH3Tqx5FhCCSI9WCrBbHYWclTsQMLGt91h0iZL7O2KxX8E4R\\\\\\\/EZTx4y+v7sDD5nWm9sMQteArBodJFDAlXprkfLu0jWjpKNUneVbl72\\\\\\\/Zli2x7Wu1eWr+EpmH2SQPa5v8a\\\\\\\/W4\\\\\\\/AqEA1ET9+8gTEOyalCTGP2IXQGCNy5bB25cjxawdiRuoITWGwwx0q7SuF7FByjYfv3cTHu2Mcqks9YjVq8YjNf\\\\\\\/LVWH4g+IJiiXZ6HtFjtlEfC4BkiarZcjDMzoBNyqNicHGseRLCPEfDayz8++uxAlfJjntGv9mn3XQHaTFo\\\\\\\/aM3IIgNLoMI4A+Arhat2JBioLuCjrQeVwRcmOO87k1L0DD9cWTVMTwMvJYMh3kBbdvEgYCYSSlbwSk0lkrPIvHwYm7WlOIyGLMr94OVbTLpiYXdxIxonJxQ3nXF4begbur3Jahjah5cdLdpfgsxOjtyLxad5H8y2eHZJL6UfBMHenNP9tBroxmeK\\\\\\\/mzkpFHzCB3FeZP43eGRWQBd+YlkvtzRh9tJzIOO4909EaXIedxNQwZLfdq\\\\\\\/p3CGp8U\\\\\\\/F9zlDbyjNG1JvARAXlYUgZuMRuTlcIn9tX+Zi6BGt79wLWy+pIzxsJX2NzZdIp\\\\\\\/X\\\\\\\/4uWDzusX9EcWgK6VOVDYtRnTKuZ+dgkp9BL2kYLwh8QYuMdmu6HhYqfScQsHSdgwbdyCcp5Y4MKyYP8d3ZAB7IJHFFtUteTYSWoD4pE0abgQMoO3i1wQx1avXGcWqXuHVqAxxhYL6+A3\\\\\\\/eB3O2lxOONYl1ABRZj7L7CsBPJHbpw8Xmadq3Ef2nGTyVItsXH0S8qmzuFg50oh7ukuQxB9XMLUbVQSjTKs+zSxlrfaUNQgwAP+hTLZAHga+bXHT8hlY56S0ROq4FQrpv985CkYrG9FxVqp0lhR9Skl+irK3klhJq2BALRQNY9Chq2c5ZrZ2IFoLwLHJeUONeqHrB79kYz5AJ30w\\\\\\\/q5PHNGNkOGGpQEk\\\\\\\/N4OkZa4v259Ol\\\\\\\/M+EQADLw6NKcUssDYO\\\\\\\/fU0VBY7W623JWOJcoYDVpqYr3IH0iT\\\\\\\/w0VO0adOWMbR4rNWOSc24SzxpNGfH1\\\\\\\/V7UOlLZCyaLrMeeXtG\\\\\\\/vBWdNtfRUlrhHcvXr3siPRuRLPXudn8\\\\\\\/ybrJmbHPzMSe8y26QnZT6jZPseZHEH0eFtc0vLnohpftbOKR8vxMw7lbQGNPWmfc08Kmb0dDjtgX8+C4WdFt7cX9x6M8E1hhzdTwxOig5jQc8dSeiBn92TBoFUPAaPIFbyoUKQ7OggL+76dPp\\\\\\\/nkmZn9JoVzfEHMC5xyoBbIlyFyodsrU+UV4p8mKZbNc2KdtvEHu6q2599\\\\\\\/6+rHjWRUXyKd3azalM2kNDwmvRS4ruMaodqtmjFc6jVpc2CH9juTXVyLXZfJfK2I0qB5QvlGMJAXpeqAeKvtQbU0qxFhMSdRuA0SiYTnedqKaoWhG7GDzyMXwurIWaLk0wvoSg9cC1FCuIDsdZxo2t6l9BKm\\\\\\\/fgRx9I3eS6HNhKvUSohPacL30xMksiJSFyAUfRTiCtp8Bq87xL+53siRr0Osc5fOvbcKsIM+72WDV78PjPqfsl1nT89wGbLnJBW7n6ButxlI3RfJQniJDkjnSUSB\\\\\\\/EwFVTTe+HYUqbe66zdLBL+2Pu5wWM0WXSgb\\\\\\\/A3cOoJ1KHWW28sr+FD8Z7FkdcOtFv+18vKf\\\\\\\/3ZC5ak\\\\\\\/Li9FPiWPnUtCZIDUHgL52iSvxHgnIkBlBMYk9O4IJWO6c9spIR0NZqsw1Kk2157wKGlufJPyriuUaf45LJw\\\\\\\/WyjoH8FHP\\\\\\\/snmKGt7Uw2ONEVE2RQIAkbA6WU\\\\\\\/kSw2XSGjJapTpiKy0o4zAUyHpgxHYFi1jhBPXvRJhXsY9AfMJea2Zwqji2xlRQB++x+bbHQzryK2ugeD+1Nx1\\\\\\\/YiD0fdfjGO7drON2WtjXRSV+cOuiXE5VMRfR0rbl+\\\\\\\/2voBg7faMJPuoARFezRTvLt5xkbNCXHD8UmDK6ztD3S\\\\\\\/WQwjaXxcBSnJbOyLIkBlmscevjGZO45esUBhDo9u26Y7V9StYZIsLaK+fcXDy42K9lC7yphay+qyjdRHQ6qlg+gUtuP3Mh0GGPLByFbwNdYNm7j73zcYee6pCFyRgzEZK2w3DPtmCmadyC3Y4L\\\\\\\/WDn90nO\\\\\\\/S3XQHJgTZXpinyBHTn+faPklefWj314f3CuabKL8cEkbmYwrFk8g0fBloDSyuWjcaJapAp7yv3LNKrx+0LL2PMiHKKmxU\\\\\\\/IIEGkZqJLVvS12V2gM\\\\\\\/K2lhiWeIF9RPCTg4MJP+t4hONJN86HyiD9uxJVYY7O1s3taQbtc81JTHVdN18x8zoxTVEHN4SDHOU7q78hHvu8YZTGi31teEnjYV9x8qehN7BIu1GGEUc2MeYm33yZewdpEcMGbVmCYwIoMc9szAx1+FcrG3uKLV1Z425Y1KcPul8YfuoHvpuqLiPs4bc016fiZ3sU4iyEtZwPuL29WVhD\\\\\\\/7zu7h+\\\\\\\/nhgmCCgE1xps0L1\\\\\\\/3ezwmCW+KiYCqFWxQYUtBUn9lHbpSaXocwqcUWxJKmkFQu6iKEIwJ3g8Hm637AX4urL8Qlzx6TWyiKejxNCAvNhJg1GLhrWMKNGomg6ezkAkyfBPS1cY2r7zrKguj6nDu\\\\\\\/N0eh9ZrdSvjp+9xditZAnO89bWCn1IRGEgx9035WVcCxz5gJwFZyp6u3af06leDsBpECmWtn+wOPIIOuerhQbSmeYhBMUpGbS8bkV1AF6aL406Xik0uIcJQvuIhvqio0wk2n5\\\\\\\/VEj9r2oJaHxULWmff2sidP0EohHO4HN7zCYO4S6beOT8PJq73bCRCWyduvuggiXxrdjSZLagt6rTY2Dwiwbme880YTfOCEt98VUNWvxBWHZ0OV32DfiVSt\\\\\\\/xWioJItRJEwKm0xQn0\\\\\\\/nsECADiawNDWdKGF+pHBD3NAKFbLNP91JWbIdO3JDJ+o35D6Pdp6YBT93umMBaJfAwqELhabGZ4dbbBYNH2e\\\\\\\/7sOjQCKSMN1r18RK0zSNhc4XuskTntSUH4O5AYQhTbHAMLkk4+imyK\\\\\\\/16FrY6qnTVzG91JNMPCfereI6rB9PxaEoL5tkiHu8ZLHxuwjriMxOMmtyfswqlhfkiNCiEogUlNzhIZpzr2bIv6Mg3\\\\\\\/q4tcpucrxswWlHCNwbV\\\\\\\/HZV5iVdJrR4Q2kUCSsadEofy1h\\\\\\\/Uk6ODDQruncrJY0Ent2PuYSPRE3MiE7OaPM7d7o2RHujfrvgWWHo3QiB9dMU+H9pJhZr06y4+fsF9olsizntcNrUPKWajewNxta7me\\\\\\\/rEtRq8m5HZkogGN5VsZFRLSxal+UnGizIR\\\\\\\/bMnR8Jyj16SNMPHnypM4TqtqR4Y2qZCY5U0JnG+OdiU7vGuTmgULFXn\\\\\\\/\\\\\\\/hwey21a9S5yQ4tY7uF\\\\\\\/bCDMguMSkdyqrzMd6qIynuMDo7Xv5QYCJQ6ZWT9mmm2qiBE\\\\\\\/cMjfD6ylaVnRoPylnT1r0cGLrmb1kXA+FJgrqB53WDcCyYz39Xi\\\\\\\/8tW7NIf2tOlt8KMRfkztm9OCgBK33sHs2PNkpbf5QYREUcCfCL9mht+fwEw6Ew2KntRHR8ilrUB1\\\\\\\/+nUQbDt5MXySVPlZVr6j+uvml9ynyEKhbnmpY8ILuJ19Bq8T4h2D99AxbJt1rUiOsKVr55JE3e\\\\\\\/R\\\\\\\/mA\\\\\\\/i8Oh9g5KkqpCSaQHFL7yLLypt5KDh\\\\\\\/gxEg\\\\\\\/eUNlX+YojgxwQW0AGKRV2VulEyLKLQ7qiYz9wIPb+5DbqA1EVnKGJDhfdTwB9kYPyp40sDlhFoynqrDPNA232vzly3is6rEFtYHImKDSW53AeJvrqrsFgdMEJc0Bwm+pPSwChmSzRGSmyccidVMLAwOHcHG9XRXnSLjnOeMnPXWn6xrpO+DPWNNVLMSVxuF0XTicTa4H0ypIw4W6kEOdAa671C1giDLhyZFfqGQjbbcQ8w3Ywtt5C28Gm5n2YV0cvT81e\\\\\\\/\\\\\\\/BltIoU7HYecX5J9kcpETSzltorIMpKdRrTyW9buu4JdnTaEr9CqArr+bKkvi18asS2MgLGdUF74g8QkSSRh2QLM6V1r\\\\\\\/7y5N7mllLm9+mKUOka4ba1O0PWwG412Nz11HGVSkh0PZaclhG4Yy1g2p7fUCmSkzJ6bi9kwbguFzvvY89GHiFiMjha6dRrWe2z0sM7zZQ\\\\\\\/y7gXCNF3EGHbFXa8JFlVwqb6B+JEbUsAT+3kpFm87DB83ABHRAeCA3cWnru2EWI5UvOSZVaTfpKaNf6+FssyhiuS\\\\\\\/pRwqQ+iRUT1XSpPGIk0L1bYFvhR7A0NEmAOmw\\\\\\\/NkL+y8P6n+dP79e5R1KjckAhEtXEu7rSB46CnxrXF8EX6hKgRdHAGpes+ihyhbarodAU88SQ8GNIzsN6XieKO5wczpkRvrF0nCOUHoNeHxNlrZQHw+wu6Ao8sArQV4m5rnnTNP4XRugVy0z1R3s8cM4RFVz6XdrUccVBE4DQpZmu7Q0UKxTDY9HM5G9m+pk\\\\\\\/HKu5tLBD6oFKQBFWppqxVUfW+xVaupj5MUkxNBSvN044Y+mQZOmawR6BMDy2tyQnVLxIDraX2akkQ1j5uJ8l\\\\\\\/eEHnv0MiSmXlHaYsnUyV096pQUIxEeht9MekXa74wezVc21Ln0XVQJfiHzLfRIK\\\\\\\/Etzc9WQ62dkmpHvcUfR5hPHWXX+ppeA3B6rm+j6OVhFIgamv1AigF9eJsYjMdrOm3Jz\\\\\\\/94ckMwrWZGDDOlEmBrCsesfxUw60LRAOU62Xtl\\\\\\\/MLm1Q5vZT++MGt9CpE1dz7N2kO9een+jFoBZ0RCpTkxn9+BfsV0uPtbb09vhzXyrGGcOR9v6OKR+ekQXWIfRc\\\\\\\/kJwgNapPX8RrP3yJzPWfUD51Z3NBb2OwekJQ9\\\\\\\/wvEpOl1fcdCERZtYDzbT8PY7quHioQrazyw2UB778Dk7hhkJYgzwSpPUL+Jfvl1kwFresiU7vXrU3SpIxD+PN4z8XeLWKJAcWf83uGKrWo1W3\\\\\\\/nhkQcI8i05C4DDuENmlyw4zNNKPD0wf9I97AkV8Bjf3dB+STsp4k66jz6nkxTtEDwlKVn8FsB8U7CmTgaCap\\\\\\\/RXv+8Rthllt3uC8Ghi0RqJfz3gi7Oq9nqvVkkWcFaAN2BvsFpeYV8HtUlNBUxSDjnZ3+GnGry95XIUL4jLOU7QaBOrQrtOWxY1TrdmZ6qsRS0EoHsDbC8OeRsYCEhDMfH0CtvqY6gtwrlT0WQEwYL\\\\\\\/5vg33TQ7cBGq2vyRp8Xa\\\\\\\/v8hulRQZqvwdj4vpsYCt6djh5QKUdNVFFmVWRxKklboFsLSNDTDIWTIhkLNvYMnFrTy26zg43+gD2nW4H5s8tEJLWmyERDzuo3wrEz6duxv5DqReNI1f+Tebtt8fLII+KXunCs8blw7Ri6KTC3vKdpGPwZm\\\\\\\/i49HQN2dJP2Dqt9bssSlHYyoQ5nNK8ii7imDiFi5zqE8yfg8C5S+pSVYHEFWd\\\\\\\/bGTB1oT+OJJ6RrzFh3gIIY4SHlD9xVCtTxMNCOTHSQzvotlHncmS8APuUo4qSU+43cabg0cPblJF4arhXzyQmc4KG9crPhykgByFwrG6WH0EfIkCGS\\\\\\\/Is8rAxUQW\\\\\\\/YjBn9ZzPMtU5Ic3UX\\\\\\\/KykpbbojuK2d+2\\\\\\\/ijm2EVBctKjqknm+2OvVeCVJilJhWLqZJDxzeqN5K9TYettkBDiU0a9\\\\\\\/wxL57q0ELuzkzysNEXbN7p9bDQI1n9R+kk\\\\\\\/V2QFmcgK3QskIM+iq63Raz\\\\\\\/FfKwizaB7wwUaRiRHT6KS2RVjAtajHGzFk\\\\\\\/06cYrORgGvdRp7udwyg9g\\\\\\\/0nJWG2rSug7dWdvVLa1reZ5JNQDBQfR0AOJFofAOuKC1oOFIvyOM\\\\\\\/dmlQdRGDD350jrlErEE0PNel2xS2TTVMM38XtZ7di2BjBLiyFFto7XRzQNmNkBG\\\\\\\/ac5\\\\\\\/EkTX7T\\\\\\\/gAMlQNcmTxET0G6Xf2T1ELwQZBW6QxAfl0iC1rRAmLHjaKGd2bjNhAIJ2L9zExkBMFR+s0IfFsUK9elS2YdzunWyQQSYLlBOJljtcWIJ0iBN6uf75JEslR0eONQ0d7hhJiWUTjyF74nHHRNSAaOVLDi+85czESg5yPjljj5G5\\\\\\\/JtNzYfQxYBHkjd2VyEwfqUu2W7uO\\\\\\\/jCeU9tJPLJYPtoXwIUZggQ0ZxY05azX3nIdVegG1oT4pwFbaZbqJZxJPISulFLODxoga3U8Cury33yqvG2E72vm91jZSbSejUQNDnFiV4JNuDkNVfYCm17XD3mmlF9bGeB6TT+yrFfnAwm4jmsxRdj7YHrKLoL\\\\\\\/wDJFVIG6qa5BM+gvtDuFJxaRN7\\\\\\\/997iGtR03mAD3AryblL6uHwH+x\\\\\\\/htnw5U3QS7jdRBVpEKQ2WGXOIceVy6IJOa3jgR2sfPmOFQUJwhxJE8+9eN8HWqrqG+j4fAdfGXdLZArv0d7gAZNn4JEADW2ANqY86kqxjJg3MOyVYIJrD6mH6yG8mY4HVbtuRT13MDoXKX87b3ciDNrG0\\\\\\\/LGuoc7HKTgZ0o9b0kVXjHFMFB0qIituSzBiw1ILO775Up9KYVUKoBrPHYw+QzlL4sFpM8MH3iRHOXKBiNWm4pl2y7bsU7S6JzJ+S9wPBNneXjiwmvcbM4RzTYcK9mkKBC\\\\\\\/SpmZeK0yZt12FNQxhsvFE389jp4vBUBV3Vx4qxTM8f1blLEaLDk1uOQvyV53poXS4INW2ZCnZk7zyKe9xMLksbV\\\\\\\/VMo6X57dJuuEZ7uzTpAmMGlgiy2nuti7sxd5frt4nLSfal4G2n7B5qdrLWMK1gd09USqctAUlkHJvnqpDevQ1R+3XV+CO3efqsIjzHQNr+fnEUU+QV9wiYtbwnk3l1MCuNohAQlfav5VTpA\\\\\\\/F\\\\\\\/GdyTQGkfizwpDTJsYoN9HRtUDsrkRkqnLiJxu3\\\\\\\/O965FtdB3jV+XI\\\\\\\/bAFDg85fxdfVKe80ZRfdSkGnyDOCDIMJTOTBElbZtnUfiiHp4w3SyhFUQP8epUV9H\\\\\\\/BFGru7WlvpjboFbusc1mnHiKOT3z\\\\\\\/G4KmuYtck5NbuG+9\\\\\\\/zfQXFF+eMPl6OSHJXf3YtRFWYCG\\\\\\\/7eYNiQzkMAp9rYz\\\\\\\/zZGUMMXFMUEh1ugOPXTt1awbKe7UZ13Bjr9SUscLcoRI5YEbkEyTVLYw81md6HafbTxFHVgWvHyATyJ7XbvHKYSPLfqscnQ4BsYSKrV3Fy3lNtJ19V4Rg3pjSua7heLM3+e3ovMPi\\\\\\\/OcBOJFgXZ3EWRiHWLeVtubBovtRcLj9X9Cvai6aU8hZQeyXELhmnzN1h\\\\\\\/HbDfiDejxZ1cLUEM7VTpCrnms7DRRMUC1NdDyxtHyonNQ4wIuxynTFxh2HeBJSV0mu9Fh+V1sNzzoLQvUt7oplRE8LjQkYf755xNQcIoqDvdPiqD19vrV2PzRIGIhTrz+j5w\\\\\\\/VqDpXEwo0X7gTyyS5w8mleeUGJxWcpJKCoULVJYhL7tlDwk2lSCve0y\\\\\\\/fMJ2fhpLEX4pjre4MlFKOhRALG+yckZ8T57Xlo+n957POoZ6e5fue1p0WGZ5gXrxcE0gDUddSU9XXRXmOQXGB1jjAu6UUpZ+i1q7bwvyjlmDwTaUpaHD6mgAPcpEAuBG9YzvWcbPTEpIHbb0tDmVAstspezc21Qhe\\\\\\\/23XNk4RJ4iv4Pf1fh6rFyOerIP3YNGfQ9Ev3kqutbi1IO0T9b9WXt8RMtQbP6Nza1ighj+UwhNMqjtsXGsoDmfi1bi6BsvsBSVwfqx6JmV7pnG\\\\\\\/jVW0TcWk5kDmFLbxYGZUUyq9uvuUo8cUg0qmAu2Qum\\\\\\\/8B\\\\\\\/EdiaTJBVUZNMWf\\\\\\\/T7ONnH++zHgQN21lsN+rDpi5dOLcC2fzZmmI\\\\\\\/NeX7JLy31s0NW8WzgwlG\\\\\\\/iYgjNxZVLjclOqXDBl3BNyFOCi9MPb8XiHXkmrsPIx6EJrt++NHPLHr57AxSbcEsAIjy3gKZlHzAdgj8bUlDB8TvsEtP3l+ISFsMTSqQrU6im0nu8lvRazeNCRbu+zyDQrxvlKVMUA9ITnBWJ4XP3VNkDzFftWMOtNwi9FAPiSmyPMG7TP82V7QCi7rY9Mo7nVi41zsKUMUBYjWYghTQpdCOrBQlH6skINIu7Sg4AiTcSL6z1+V7FY7vLBM83syUnbft8z6tfvE9M8dyZgLkFVVNq3MKqFlMqahUX4qGafHoTw91z8wZbpUqO53lmfcoIzw5Y7NMLN9MRuGkt3p63KXXEqtMU6+7BP5GwHAUNbFYH3KdPyzWTYprj4tJXVPQ6+BhtBu4GVAwENibHwXOcybCEWiz601Jck8nPjAuMX7X+deiEDC8FM3fnHPcVyc\\\\\\\/1EX4+2lFbT+Gu4ifhGocH8Byz3RHYIc3z1Fb+DJ1PgHaVEyiOv4mLbgTweIBZrrKqI8\\\\\\\/Jaw1S5+06Ak1iUCziMWXdhh6L2uoFu2zJxuo7jG2KivY6BtFA2ocLIyH7x55oEY4+zypvq1b5jZ0gJrCTU1J0SxwitMWNQGZ+FfVQTjfp0RqriEuOxlPkOZgEs2FAZF\\\\\\\/GPdbPt7cMFbzMoPtvszzS1lt2xq9VxySL4PqNe7BrdZHDgjJVgkkf2PWluy2XmYxsFSmBlFN6ZgBx3VpEkhrqhg429eKUEiMd\\\\\\\/+dv+iebwCWXiU7XPPZpyo6VSmDG\\\\\\\/nrEOUULyDL1pDYdcikz6zZHNr\\\\\\\/DlesWyoixjtHRY35QPc85T0PZGYF5PU2aEch6FXx7\\\\\\\/cTu12oexC1q3ifUubuDKrW7j4jas3d++TqX4LHur2+cU9AMFl7OcJSHgBNtZRD+eU4HvbC662jwjcAIcX4jBQ0mw+dg1TBerhM345BLYCc2e9S4EGJLPbVDwCGfc5sQ+OsuSxVCvgxkkVy9cSQICqt5cMRugDr8fDbOPrfnc0aABAFV8d3LuBftbbNoKan3vW3I\\\\\\\/qw1maDt11ZbuZUPxCCHHqGVBUW3j\\\\\\\/wwEKs1Mlny0IiVEtHiOyTyEox86JRNx88MTieWjuSYeydhzN0oY4ktH4\\\\\\\/GgaoSswPiYtL\\\\\\\/7IlWUQ4llcX5XUXWzB\\\\\\\/CfQv75kxLz6g9qvkowOsmApFgdXyD+3mFTBJbjlvYtfoTeuQ3dkJhfo+nzmnM9Y8x5VpB1IFzl8l9EtbcBwBWhIZTuyQV2lMTgsYX1CaqecidZ6nIicipqpVozcicmMAzXOCZ\\\\\\\/N9fnLwfXwUyVIxLVBjcMN\\\\\\\/nsfSzw4XMYzrMxU0FWTZ\\\\\\\/a5uzuMJt7odCcZg9dUpUJOipXw8R6aQlfPD6cyLZ05dbjkoFsUvo6preHV0F3XYo+Rh08fNTFMhtAUGDnRFbb5vHimIB7gaqWOzFdvBwq42cpEB4BJ\\\\\\\/5FEpwvdna+mWZLD5qpU8c2WQhwbIfRtdVJOp49jN3z7JNxwF6UIFqC6MrhDr5CBV059ldc6urhMPTu5\\\\\\\/RlLk3XCXpwykw3Lsdr\\\\\\\/U0c10XvcqOGYzbb+WpzfedVs3olWVdV91Hp0aaYSFo5rQWu0PHlBJN33vvTm26RfW0bAIWJiHn0hTR8L2SUImSBkuMca3GmUg9cjbq2c+r48i8MGXSpPFpT4u9IZUcWP8kwJPc1Y93a4vOgqpI3hIn7o0ekjAsrIUh0kMqlfkUhyjNKXho6huHy+yzCAJjmZcvjVsTp+2BJBYH+j+E1Q4vIuAbLX3iejgcuHl2Zu05p1n\\\\\\\/nmfnOMjU5I3kg4Kvk6GrF41PU3CDTMD8mXmWzt+vfRw3mUj9FDcNpdFrY3ftk6R+SH9GzwfsWWPrPq5Zh1qf6MeudvdCIvLQu01wTICy7u5P5yZJWdrNXJpWtT3UgQeUqxlLF\\\\\\\/oAk17rKbicbdiEuUXAdBxfbYptP064RTmhWskfCd6vHHVxo4c0Dt6jcd6kDNrSUIGvCL6Zt2WVDcaupZCpTohk\\\\\\\/sJAklb841xhPeTv2FTN5PlaKUskTSqn9E6Uo0rVYkBvSpc3UG4aqSX4xDN2\\\\\\\/x4iznEXSVpj\\\\\\\/0K1AWfyYYfp5r0GG\\\\\\\/8yG0+QbpoSAl1lzlDh5zmynTTnDfE90gdXFgz6NeFehX505Sm4k61IRe6Qd2jCOsW+I1eVGWdLmoWpmZQT\\\\\\\/FWRNXmvdbRcpLJRl19hmT2edc2g7YkiCDPnmr5dd80Per+ByWYO\\\\\\\/uWyytRi+mTAsQCTnws3MOZqbrHYW3hiGVLxEQww1A9KciQm\\\\\\\/LtYezFpY8+a0jScigBqe7VQEP5zJF3DMFXzcA2ToJ4RC6shfs\\\\\\\/CIaUmq5QSQsrgKNYG7RxdxoXEl0hs1UCJoD2ju148\\\\\\\/4qesVt0bytSf63fbB60S3cA6hY8Psihy6uHmv7TlpuGf7ygXP3PD3584JCgbeEUT\\\\\\\/jxshL3d+Jbmf9\\\\\\\/GamwHbpOHWQpVYvfnzKTeu2NTTggveXZolJsLFd+7PlMVXE9HTd\\\\\\\/nHhwTfqfAlcSY2A7DuSt0zdzzIVN2AW71Gf\\\\\\\/9yjBR6BOQtt9a6ieisFZeBFZ3ePuYJQNX0mdqC8BrWn44TppicKEeyIWju06+113kpINdeylIXJ9zvL+SMe0s\\\\\\\/rPOBcKn2Jy85X2xzQCRBjbP6gjS50UgeeolKPcq165IR+F\\\\\\\/n0zGMycMQVsJPKg0MuSFwScMdVt\\\\\\\/yzV087uKZoTKbZJ\\\\\\\/NkZuoakFpm6PhwuymrjPj\\\\\\\/4HeBnJNn4Mx9gKLKM8f11p6eSpikTFIqoVWnuxvQVvp58hzo0xuhAfiHT9K+Jb2u8yCNvQNhCMG9aJc+bSTU2pWJi4A02blNBrYAe4XHZLzgP4cGHKu8LMLok0ad0Js0ezLB03FGoOwPq5bJ0\\\\\\\/fJ6vQ91ktLbtXlt92cdDVqtbaImQEH6o7MCcrHiXMv+ygINApLozxQnlrD2pBkjTfOQvDcYcJaXvRGYiwqXIBn0tAVeI7w6xE8PxT\\\\\\\/A7iOq0R5KlTvukXIzpIYZUqjj3BSU6HbBQzKUuLtur\\\\\\\/POD+UdKCWVASCBSh4sy2bU+lBhjxYE38UGL+u1J0TuUaPX5gy7QoMYc+StYNfXnLheXKhTCbVjR1jHrhM6eYfbjfq8sBKpA\\\\\\\/x4t4B4XdT0kFelCzbUQ7aKYw87d03o5Jwgr8CNReA3Ss3Y0ZTKCa+MLUlcfv\\\\\\\/Eugk891KrTmiemzBflriwGv7MMZUIZTfpdKT+h8x4NczwTjOXC+kiOj7cwDHjY6LaMq3uvaL+rPvwxu5DixN1PyO5J0HTjylPi\\\\\\\/s4zGpqwK\\\\\\\/ZuoMBQsDJoDcV+fb9b42Fd+sTjZUNXbw69Z0dv6lmikz8eNQrsMXG4XP10wkxkWcI1XxTlfSZ2Gu6+OIat4xxol1JExo7kNOmnl66rex1eJIwbkgvpIkHmjLie7zuW0FCZA7z6+qPJG0RL4aBVUoFjkD+uQ8xZPuJ7odrRpCQPSdiZ6wdSg0ulYCUx+gsOP6wR9F6xhBw9tuHByCnv90wZ+YIQ4fTjSI0g8hELt3HBtPsE+dUyhbJ9BdoSVAbdaCGe+HtqYSMQDwVNS+1MJKt5uaPKmxIHeR88znQEOsyYZy\\\\\\\/Qlc5L8jCohbyLvhSzniA1h5QoZ3ILOHPFRTelyzV3Oi2EerhkXkTS4mi3dl9Iox96K3JzF1E6MZrzKdozULpL9NqS8Rjm12qX+c951lcEU8zLPNXlYfcnHzrl5KSocOQgZScS4dlbVC+fIGKjdHHZVwLFUZn2h63+0ZEtfs67vZke1rQN4Bw3xDyMlsQfKdQhdTeoccQFPNKlkTGeX1hUJTnLrr1vEDcSrQW\\\\\\\/kj0LQwjPe0ZRmZr55l6c+vTlLSmwCAVMhVJaQWIuLKjIEh56HZ\\\\\\\/boMhVPchHg3wxc\\\\\\\/Z94xAa8\\\\\\\/EubIiHZzJZGpYZh1jkcEbQeYc8NKwSR1+Xwzwa30GzG1glo7YZpPeq5Nwwc+FixW1A3OO1cT1Ew2\\\\\\\/TinrFO5i99WKOU+5nXeSAhpfXrwnPKUSZ2G6TsVUUg7NlNUFJYeIy4YCJ\\\\\\\/\\\\\\\/4J7p7TlXDKF7nW2aCPPYtNRXySqCKW0ratTJpwC94cY14mv6tlt5NzESzrZK+yV8LGpGLnUn1r7qA0sdeBLG7bdH\\\\\\\/ZGS0xN0LPN2iSpb358uKq0d8+Mb9jSP1ljqpVjAiWcU3MANAqsMxkQ7PXI+KlgLDg+5YlubzhfPWEkWFE+J7Y5l3qrkkoo3DOeJMfn05R0IWDCPhQ4b5ZHVRfmUdZfSLkT7spviT0gesF3eBeEqLIVG2eKeu97eUEmazAzBmNbPsM1CpTv3ek3wEs2iMexvz+pnbgwkzzhONlsrqnj68z8zQHbJTYApSP3Ub9X1+nP99aP3RVm2BuRQ+TG3ONhOd+WKuBqyH2ef+oe5zf1qu66MYLZ3wX7XGUWf+9SnZ9f00lTHGf\\\\\\\/szkoP5RO+XmP41\\\\\\\/vGrkn3DtehsdXGg+Urq\\\\\\\/tjmR3csvVcCUEoc4XP52ND170mc7oNghi+N9F\\\\\\\/WQDZkMuG6uO\\\\\\\/5FUUVIHtWWVzj0ZNU1v8wafBEsMImlUnECe0BlmH1BnBObwFElCHLPyeVvkK4XITGeFJ1q3L8wanBIghYbgkzRURX08+LENkXh6qniR+AgpzuB3mb5gD6PDVTDgnV+mkLkpEU3N785BdsaFhKT1PILHthNH2vtv2tdpnq2CyJumcpoADqsi3FrXnG8GqfJkDWOL8haoJFUTSi8OY9KJcTdJvSF5+pPYulkU95hjHFnusUO\\\\\\\/tZ8VStsa+jIU3HzHONUnZ8Q3ERJ5ZrlSJFhKAWr+C9QBfQNhmkoWbANxvNCKgwhDk+i+0pShiPwXKwg+pdwuhxTQ3GT\\\\\\\/0n+uzRbHmcznIgNVl87CxjHjSptmPU+XyILfH1qTCVmmIfft1W5FsCw7NK1Ki79iT4B7JxWJD1nVAyE+XKdsh\\\\\\\/\\\\\\\/Io2+em+N+k9XyClxpNFf89CJFFtchTDRCOYHsFoGc2TW0MIlYMB0OqZKZj+2JgRZk5VCQ0M9HKPOPs4tr71tMfUs+8J5IELI8SEQGSKpRn8vqNXt2482qClJ8xQYB8UIFr18JR382wqkYCm5bkiHju6\\\\\\\/B0KfjoNYqXnlH1Vl1pu76UaTyNZA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Mkg2U4owOO5vJYzktMAouQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002237cbc0a0bd1432f139e5b0807b211f4b01335f6ad64bbef2dc7809eea9f8735a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TU1z8RWS+cL6So+5iIIT39HvmN8I7U4Evq6a8mr5OaN5I2ir7yEDvxl3LkVig6I8JW5I\\\\\\\/RSdaFznLFDB6OnJmUMiXdHNG7hlIgZhsdi5xILA5+eRwqrDHYtjEtKiUJ5Cind9HxbiEu1svOZeZMuXDbL+DRDGnSfAeTU4bq0WBSI\\\\\\\/WEO9+HLilPyJ5B9RbmFgFqYEoOHPkcy868mYIHOWAeF\\\\\\\/YK0RIwqJRnhRqXxSyms\\\\\\\/UjefTIy2pghGsF\\\\\\\/ZhcWAryol1h6IakEy7k4JnI8hwviM6gUkoXuDhiJwYPm62Fv53fZ8YAJdn9zTBo2z9arp09xnrBftq+gVkCAAhaDl6PgcstaOmml0STtxN1Pf3UtVds+W08EV\\\\\\\/BUlWPabYZFB4tUVRCD2ub3VA6fuu9HZ7hKjuhdiauywoDtPtD8zlA6DBGHBQxYXNlkRqCya7VMkZ12WQtfMH8s9Qvz9jo4huNuUpKof4cngk82eZXAoXJhykcXUznbxoTm8RtYilEBb+OkYENyY2KnCWohY36I93xhb3\\\\\\\/QOoi+KZhwKSj0utvMvlUi4GnvDq7x7FSZHjIbRPlDdXKunaz3qE0jaF94TuHpo8X0Im6rSm\\\\\\\/4NpQlIYmmOySfDzH8BqSk9g++Wf0Rb6dtNTAbMc69m10Id0bZme76XWE\\\\\\\/jKsaARj3xnyd646ykHrCTOVzHVCNZXawtNh1AKAW98tHBaH0MjyT7GfA72tOio+3\\\\\\\/DTiRShoOIyH5cWYYIWcONClmBJU4Xu7JxqOjK6bJtob1Mt+\\\\\\\/52jHyFFYZypSBXvWRbEisvcNDtbTev1lnE6qoWr3Io3WMj0gxrlIFdaPSBO6QTLFWNchT3pAVhe8EEK7vZTz4hj5Lj8RCeQ3e0OrHXcD9RmREi4CB\\\\\\\/\\\\\\\/rn5R3Hgpx6mj4rOSUFtOTyO62oiFYJzzccvWC59Dqgv44ljr4E9KaZalOLGvl2MFO2ySTdVQGU7UY32NSvwRwvXfWOoG9QOqhZVqp+WbSslQVQ04Uaq5oxyWFmtbLCnpjuwPTQ6X4red6P2alGOUPNzeYgVk6TcYFdEQKIFj07v5lYGZkLA81JhNl2st3WdPG118VVKwQDEdkIuDEZJmV3pU7c8F0WIs9szxTacR+o6rv7qOYw9iDoG4YkxCeTNIKnMLO\\\\\\\/cxlO+m92610hxq57tmUr84FGjAZ7WG\\\\\\\/Cg8Vx5u5Nv72bCmfLCRW2xO18uzZ6sseQt18bJcPEL0G+6LmLAGE\\\\\\\/DiPMs+gHwbX3e9mRdPs5sWlE6\\\\\\\/anqNVmO2w0gVQXyZHHk6O2zstR2fo895aF5Sc980HFlId7WBrdUhk01omDBf4FbgKbE75XihhNtzmHCsF8ybAgGkFmZ3Sth5zM+HLTSM8is2aehAC8EnKgijrtddx3s1NUqZ9NBZstkUfC\\\\\\\/+ePTpZQRaD\\\\\\\/HLM7sPlR1V17T+tpwnc+NMXNqBNYp02WgGrzjZ7KuMuefPFfy\\\\\\\/AYrfVTuAa08ig5aI\\\\\\\/vFJpUC2ODvkORh1qcTUe2KESlY1NKkzCQG1o\\\\\\\/VM9OJyUBQlJkYWPQgq4PX5FNrVy+9TQ6qQr5p9tw+gRwYSHa4syfz5uVLY3cYyxMouZzcfFGEB5lsK4cCu8uYFNUNqOCht0f8u+1sYyzw8UsrAuEJEPK5DAf408bdS9TKTc9jGWgYgwEiwQ08qafc8uB6zqFo\\\\\\\/gQGNKUKUrcv4GxNx8LbpBpwu7gGi9H2UHzkVJbXuYcMFCV+mALkMedT7mhDW5otY+ou1mhGtIAXxhF4tafc8AG7EbeP4jhAdr\\\\\\\/3K6w+hwekBgD4HpaMnANQAdV9ndZT\\\\\\\/ZkSmKuQh5+WnCfMK38EGDjNhCMQ4f7Dux6yz7091fCLdmBb+HS5gjyEuuRgPaAe6CK6C0Opuo8aem7\\\\\\\/19IgDjyuGvTamziy0Ev6eOgpJ1sX+deIYGW\\\\\\\/UEFqplfopB40q2emvFJD\\\\\\\/xREUXFSg2utmJKpv6vVDBQ8vkxPRmLyIZVLhV2S4If0hz34Uy09f5uLc0SQWadmntrggsx29J\\\\\\\/uimhnqfJbe++uKbamBuFFIV68go\\\u0022,\\\u0022iv\\\u0022:\\\u00229Ol6rVU6EGpw\\\\\\\/liic7PyOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002233dea0a3d214051426b2aa8f7286088d795e2ad199f26e666d868463dc5c1127\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460074 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/365" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mnLelZ1GnL8Y6PVK_wqJSmnSDd2hv0n-LI7BGzkr0R8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uKds4vZE9RjkXa6yUghyTxftCU8tdnCcUVRZK5U72Y0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/357" title="Effects of DNA Methylation and Chromatin State on Rates of Molecular Evolution in Insects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/377" title="Genetic Mapping of Resistance to Meloidogyne arenaria in Arachis stenosperma: A New Source of Nematode Resistance for Peanut" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460074" id="node460074" data-pisa="ggg;6/2/365" data-pisa-master="ggg;g3.115.023549" data-apath="/ggg/6/2/365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dynamics of Dark-Fly Genome Under Environmental Selections</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Minako Izutsu</span>, <span class="highwire-citation-author" data-delta="1">Atsushi Toyoda</span>, <span class="highwire-citation-author" data-delta="2">Asao Fujiyama</span>, <span class="highwire-citation-author" data-delta="3">Kiyokazu Agata</span> and <span class="highwire-citation-author" data-delta="4">Naoyuki Fuse</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">365-376;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023549 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460074" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Minako Izutsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minako%2BIzutsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Izutsu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinako%2BIzutsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Atsushi Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Asao Fujiyama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Graduate University for Advanced Studies, Mishima 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asao%2BFujiyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujiyama%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsao%2BFujiyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kiyokazu Agata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyokazu%2BAgata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agata%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiyokazu%2BAgata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Naoyuki Fuse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Cell and Developmental Biology, Graduate School of Biostudies, Kyoto University, Kyoto 606-8501, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoyuki%2BFuse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuse%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANaoyuki%2BFuse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nfuse@gcoe.biol.sci.kyoto-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">nfuse@gcoe.biol.sci.kyoto-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/2/365" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460074" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460074/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/365.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460074" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460074/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/365.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460074" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460074/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/365.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460074" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460074/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/2/365.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460074" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460074/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Environmental adaptation is one of the most fundamental features of organisms. Modern genome science has identified some genes associated with adaptive traits of organisms, and has provided insights into environmental adaptation and evolution. However, how genes contribute to adaptive traits and how traits are selected under an environment in the course of evolution remain mostly unclear. To approach these issues, we utilize “Dark-fly”, a <em>Drosophila melanogaster</em> line maintained in constant dark conditions for more than 60 years. Our previous analysis identified 220,000 single nucleotide polymorphisms (SNPs) in the Dark-fly genome, but did not clarify which SNPs of Dark-fly are truly adaptive for living in the dark. We found here that Dark-fly dominated over the wild-type fly in a mixed population under dark conditions, and based on this domination we designed an experiment for genome reselection to identify adaptive genes of Dark-fly. For this experiment, large mixed populations of Dark-fly and the wild-type fly were maintained in light conditions or in dark conditions, and the frequencies of Dark-fly SNPs were compared between these populations across the whole genome. We thereby detected condition-dependent selections toward approximately 6% of the genome. In addition, we observed the time-course trajectory of SNP frequency in the mixed populations through generations 0, 22, and 49, which resulted in notable categorization of the selected SNPs into three types with different combinations of positive and negative selections. Our data provided a list of about 100 strong candidate genes associated with the adaptive traits of Dark-fly.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li><li class="kwd"><a href="/keyword/environmental-adaptation" class="hw-term hw-article-keyword hw-article-keyword-environmental-adaptation" rel="nofollow">environmental adaptation</a></li><li class="kwd"><a href="/keyword/reselection-experiment" class="hw-term hw-article-keyword hw-article-keyword-reselection-experiment" rel="nofollow">reselection experiment</a></li><li class="kwd"><a href="/keyword/genome-wide-analysis" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-analysis" rel="nofollow">genome-wide analysis</a></li><li class="kwd"><a href="/keyword/experimental-evolution" class="hw-term hw-article-keyword hw-article-keyword-experimental-evolution" rel="nofollow">experimental evolution</a></li></ul><p id="p-3">The adaptive responses of organisms are thought to be a driving force for evolution. The characteristic traits of organisms, such as physiology, morphology, and behavior, impact organisms’ fitness in any particular environment, and are selected during the process of evolution. To understand the mechanisms of environmental adaptation, researchers have generally examined the traits of organisms that have evolved in nature. One of the best-known examples is the shape of the beaks of Darwin’s finches, which are related to the feeding habitats of each species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grant 2003</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lamichhaney <em>et al.</em> 2015</a>). However, it is laborious to clarify the precise roles of a particular trait in environmental adaptation, because adaptation is ultimately the output resulting from multiple traits, and because the natural environment includes fluctuating features.</p><p id="p-4">The genetic basis of environmental adaptation is even more complicated. Recent progress in next generation sequencing (NGS) technology enables us to examine the whole genome as a means of studying environmental adaptation. Genome-wide association studies (GWAS) have revealed genome differences between two populations without any <em>a priori</em> information about traits, and have linked genes with traits that vary in populations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burton <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McCarthy <em>et al.</em> 2008</a>). For example, GWAS successfully identified genes associated with hypoxia tolerance in Tibetan people and thermal regulation in naked mole rat (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Simonson <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim <em>et al.</em> 2011</a>). Quantitative trait loci (QTL) analysis is a traditional approach to identify genes that contribute to the evolved traits of organisms, for example, the genes involved in the pelvic loss of stickleback and in the albino skin of cave fish (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Borowsky and Wilkens 2002</a>). Although these approaches have provided great insights for studying environmental adaptation, it is still unclear how the combination of multiple genes contributes to the fitness in an environment and how the adaptive genes are selected during the course of evolution (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barton and Keightley 2002</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett and Hoekstra 2011</a>).</p><p id="p-5">Experimental evolution studies are a powerful approach to observe genome selection during evolutionary processes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrick <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tenaillon <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Araya <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrick and Lenski 2013</a>). In this kind of study, organisms evolve in a defined environment, and the genome alterations and selections are analyzed across a time course. Early experimental evolution studies using <em>Drosophila</em> revealed several important concepts of evolutionary biology, such as genetic assimilation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Waddington 1953</a>) and evolutionary capacitors (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rutherford and Lindquist 1998</a>), and now experimental evolution studies are achieving great progress by means of genome analysis technology. Although some studies have characterized the relationship between genes and evolved traits (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhou <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Orozco-Terwengel <em>et al.</em> 2012</a>), genome research on experimentally evolved sexual organisms is still limited.</p><p id="p-6">A <em>Drosophila melanogaster</em> line has been maintained in constant dark conditions for more than 60 years (1500 generations), since 1954, by a laboratory at Kyoto University (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mori 1986</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fuse <em>et al.</em> 2014</a>). We designated this fly line “Dark-fly”, and utilize it to investigate molecular mechanisms underlying environmental adaptation. Dark-fly is an invaluable material of a long-term experiment, and the term is even longer than that of a similar historic study performed by Fernandus Payne (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Payne 1911</a>). Although Dark-fly has no apparent morphological features related to dark-adaptation, previous studies revealed that Dark-fly possesses strong phototactic behavior and somehow retains circadian locomotor rhythm (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mori 1983</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Imafuku and Haramura 2011</a>). We also reported high fecundity of Dark-fly in constant dark conditions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>), implying that Dark-fly possesses some traits advantageous for living in the dark. To explore the genetic basis of Dark-fly’s traits, we previously performed whole-genome sequencing, and identified approximately 220,000 single nucleotide polymorphisms (SNPs) and 4000 insertions or deletions (InDels) in the Dark-fly genome (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>). However, at that time we did not identify which SNPs and InDels contribute to environmental adaptation of Dark-fly. A serious problem of the long-term Dark-fly project in this regard is that the control sister flies were accidentally lost over the course of the 60 years of maintaining the Dark-fly line (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fuse <em>et al.</em> 2014</a>). Therefore, it is impossible to precisely examine the genome evolution in the Dark-fly history and to accurately compare genomes and traits between Dark-fly and its control sisters.</p><p id="p-7">To circumvent this problem, we designed an experiment to accomplish reselection of the dark-adapted genome. That is, we reared mixed populations of Dark-fly and a wild-type fly in constant dark (DD) and normal light–dark cycling (LD) conditions. We compared the population genome using an analogy of QTL analysis, and detected condition-dependent selection of the fly genome during the time course. The results obtained demonstrated the usefulness of genome reselection experiments for studying environmental adaptation, and provided us with a list of potential candidate genes involved in Dark-fly’s adaptive traits.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Flies</h3><p id="p-8"><em>D. melanogaster</em> Dark-fly Oregon-R-S line (Dark-fly) has been reared in constant dark conditions at 25° with a minimal nutrient medium, Pearl’s medium (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pearl 1926</a>) (termed M condition hereafter), since 1954 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mori and Yanagishima 1959</a>). We have also reared Dark-fly in constant dark conditions with a standard cornmeal medium (termed F condition) since 2008 (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>). Just before starting the experiments reported here, Dark-fly was reared under 12-hour light–dark cycling conditions with standard cornmeal medium (LD condition) for 3–18 generations to examine the genetically fixed traits. We designate the Dark-fly that was reared in the M condition for 1351 generations, in the F condition for 56 generations, and in the LD condition for three generations as “D1351+56-3”. D1351+56-3 was used for the fitness assay against Oregon-R-S. The Dark-fly used for the fitness assay against Urbana-S was D1351+81-18 (so designated using the same nomenclature system), and that used for the mixed population experiment was D1388-3.</p><p id="p-9">The Oregon-R-S, Canton-S-iso3, and Urbana-S strains used for the experiments were obtained from the Bloomington Stock Center (stocks #4269, #6366, and #4272). The transgenic fly lines expressing Green Fluorescent Protein (GFP) and <em>Discosoma</em> red fluorescent protein (DsRed) [P(GFP)Zasp66(ZCL0663) and P(DsRed)ap(PyR10)] were obtained from the Kyoto Stock Center (stocks #110740 and #109136). To make competitor lines for the fitness assay, both transgenic lines were backcrossed with the Oregon-R-S or Urbana-S line more than 10 times. We called these flies GFP-Oregon-R-S, DsRed-Oregon-R-S, GFP-Urbana-S, and DsRed-Urbana-S.</p></div><div id="sec-3" class="subsection"><h3>Fitness assay</h3><p id="p-10">For the fitness assay, we put four kinds of flies (five individuals each of females carrying GFP, males carrying DsRed, tester females, and tester males) into the same culture vial (about 30 ml volume), and kept them in DD or in 12-hour LD conditions for 3 d. Then, the parental flies were discarded and the number of progeny was counted after the adult offspring emerged. We observed the adult offspring under a fluorescence microscope (Olympus SZX16) and classified them according to their expression of GFP and DsRed (see <em>Results</em>). We evaluated the fitness of tester parents based on the number of these classified progeny. Statistical tests for the proportions of offspring were performed using the Mann–Whitney <em>U</em>-test (wilcox.exact function of the R program: <a href="http://www.r-project.org/">http://www.r-project.org/</a>) and the Welch <em>t</em>-test (t.test function with var.equal = F option). We also calculated the comparative fitness (CF) of the parental tester strain in the offspring population using the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/365/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/365/embed/mml-math-1.gif"/></noscript></span></span>where <em>W</em> is the number of White progeny (from tester females and tester males), <em>G</em> is the number of Green progeny (from GFP females and tester males), <em>R</em> is the number of Red progeny (from tester females and DsRed males), and <em>Y</em> is the number of Yellow progeny (from GFP females and DsRed males). Thus, CF = 50% means that the tester and competitor flies had equal fitness.</p><p id="p-11">For the assay against competitor with Urbana-S background, we used a modified fly food (46 g corn flour, 23 g corn grits, 40 g dry yeast, 100 g D-glucose, 7.2 g agar, 0.5 g butyl benzoate, 5 ml propionic acid / 1 L final volume in water).</p><p id="p-12">Canton-S-iso3 was used as a tester strain to examine the effects of distinct genetic backgrounds between tester and competitor strains.</p></div><div id="sec-4" class="subsection"><h3>Mixed population experiment</h3><p id="p-13">First, we prepared 16 vials each containing a total of 20 flies (five female and five male hybrids between Oregon-R-S female and Dark-fly male, plus five female and five male hybrids between Dark-fly female and Oregon-R-S male) as one replicate population. Three such replicate populations each were placed in LD and DD conditions. The parental flies were removed from the vials after 3 d and stored at −80° for use in the subsequent analyses. After the adult progeny emerged, we collected all flies from the 16 vials together into one vial, mixed them carefully, and divided them again into 16 fresh vials every generation with ice anesthesia, so that the flies reared in the 16 vials were kept as one population. One vial usually contained about 60 offspring flies, and thus one replicate population in 16 vials contained about 1000 flies. The mixed populations were maintained by transfer to fresh vials every 11–16 d. After generation 50, we started to use the modified fly food described above.</p><p id="p-14">To estimate the size of mixed populations, we measured the weight of frozen flies. Each value of weight was converted to number of flies using the mean weight of six populations at generation 0 (320 flies = 0.42 g). The statistical test of population size was performed using the one-way ANOVA of Microsoft Excel.</p></div><div id="sec-5" class="subsection"><h3>Genome sequencing for mixed populations</h3><p id="p-15">To extract genomic DNA from the mixed populations, about 1000 frozen flies were crushed with liquid nitrogen and the fly powder was thoroughly mixed. The genomic DNA was extracted in quadruplicate from four small portions of the fly powder according to our previously reported standard method (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>) and the DNA from these four extracts were combined. Paired-end sequencing libraries were constructed according to Illumina’s standard protocols. Sequencing was performed using the Illumina HiSequation 2000 system. Raw sequence data were deposited in DNA Data Bank of Japan under accession number DRA004032 (DRR046855-DRR046867).</p><p id="p-16">Raw data of read sequences were aligned on the reference genome (Flybase FB2009_09 October, Dmel Release 5.22) using the aln and sampe functions (with default parameters) of BWA software (bwa-0.5.9rcl) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li and Durbin 2009</a>), and the files obtained were converted to the variant sequence data (pileup or mpileup files) using samtools-0.1.16 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2009</a>). To calculate SNP frequency, we used the pileup function (with –B options) of samtools and the pileup2snp (minimum allele count ≥ 0, minimum coverage ≥ 5, minimum frequency ≥ 0) function of VarScan (ver. 2.3.5) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Koboldt <em>et al.</em> 2009</a>). To extract Dark-fly-specific SNPs, we used the compare functions of VarScan with a slight modification for comparing the specific allele.</p></div><div id="sec-6" class="subsection"><h3>SNP analyses</h3><p id="p-17">Nonmetric multidimensional scaling (MDS) analysis was performed using the isoMDS function in the MASS package of R. Other analyses of overall SNP frequency were performed using standard functions of the R program.</p><p id="p-18">For Fisher’s exact test of SNP frequency, we first merged bam files of replicate populations with the merge function and mpileup function (with –B –d 9000) of samtools-0.1.19, and performed Fisher’s exact test (max-coverage 9000) using popoolation2 software (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kofler <em>et al.</em> 2011</a>). We used p-values to identify SNPs showing a significant difference in frequency (Bonferroni-corrected p-value &lt; 0.01, and top 5% in ranking) between LD- and DD-reared populations. Quantile–quantile (QQ) plotting of p-values was done using an R script (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Saxena <em>et al.</em> 2007</a>). The chi-square test was performed using chisq.test function of the R program. We grouped SNPs showing the top 5% p-values of Fisher’s exact test into three types: type 1, 2, and 3 (0.8–1.0, 0.35–0.8, and 0–0.35 frequency in DD-reared populations, respectively).</p></div><div id="sec-7" class="subsection"><h3>Identifying candidate regions</h3><p id="p-19">We assumed that two significant SNPs that were located within 100 kb of each other were linked, and grouped them into the same region. We then listed the candidate regions that possibly included dark-adaptive genes. We also compared these candidate regions to the previously identified runs of homozygosity (ROH) regions (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>). Effects of SNPs and InDels were examined using snpEff software (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cingolani 2012</a>).</p><p id="p-20">To narrow down the candidate regions, we used the MULTIPOOL program, ver. 0.10.2 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Edwards and Gifford 2012</a>), which estimates a QTL locus from the allele count data using a dynamic Bayesian network model. This program is known to be sensitive for fixed SNPs (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2014</a>), and therefore we filtered the data with SNP frequency &lt; 0.01 or &gt; 0.99 before performing the analyses. We set the parameters of the program as -n 3000 -r 1000 -c 12000 -m contrast, because we processed the data of about 3000 flies and roughly estimated the recombination rate after 49 consecutive generations (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hey and Kliman 2002</a>). We initially obtained logarithm of odds (LOD) scores of 1-kb windows for each chromosome arm. After detection of the LOD score peaks (LOD score &gt; 75 and interval &gt; 1 Mb), we recalculated the LOD scores from the data of 1 Mb regions centered at the peak, and obtained the 90% credible interval spans for each peak, except for peak one. For analysis of peak one, we used the data of a 3-Mb region, because high LOD scores in this region were extended longer than that in other peaks.</p></div><div id="sec-8" class="subsection"><h3>qPCR-based measurements of SNP frequency</h3><p id="p-21">To confirm the reliability of SNP frequency calculated from Illumina NGS data, we also measured SNP frequency by the quantitative PCR (qPCR)-based method (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Germer <em>et al.</em> 2000</a>). We analyzed the SNP frequency of 116 samples in total, consisting of nine SNPs each of 12 or 13 population genomes. To detect a SNP, we used two pairs of primers: one consisting of the reference sequence and another one carrying the Dark-fly SNP at its 3′ end, each paired with a common reverse primer. Although the difference between the primers was only one nucleotide, PCR amplification was affected by whether the 3′ end of the primer matched or not. From the difference in the number of amplification cycles between these two PCR reactions, we could estimate the SNP frequency of the template DNA. Before examining the population genome, we tested the usefulness of primer pairs using genomic DNAs of Dark-fly and Oregon-R-S (Dark-fly SNP frequency: 1.0 and 0.0, respectively). If the difference in the number PCR amplification cycles was large enough (more than 4.25 cycles), the primer pairs were used for the analysis. The primers used are listed in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS1.pdf">Table S1</a>. Each PCR reaction mixture (total 10 μl) consisted of genomic DNA (1 ng/μl) and primers (0.3 μM each) in 1 × QuantiTect SYBR Green PCR master mix (Qiagen), except for the SNP at 2R:12478534. For this SNP, the PCR reaction mixture consisted of genomic DNA (1 ng/μl), primers (0.3 μM each), dNTP Mix (0.2 mM), Rox dye (Toyobo), EvaGreen dye (Biotium), and DNA polymerase Stoffel Fragment (Applied Biosystems) (1 unit/μl) in the Stoffel Buffer with 2.5 mM MgCl<sub>2</sub>. qPCR was performed using an ABI 7900HT. The data were normalized by the value of a hybrid line between Dark-fly and Oregon-R-S (Dark-fly SNP frequency: 0.5).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-22">Raw sequence data are available at <a href="http://trace.ddbj.nig.ac.jp/DRASearch/">http://trace.ddbj.nig.ac.jp/DRASearch/</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Fitness of Dark-fly under dark environment</h3><p id="p-23">To examine whether Dark-fly has high fitness in dark conditions, we developed an assay system to measure the relative fitness under mating competition. For this assay, we utilized transgenic fly lines expressing fluorescent proteins, GFP and DsRed, as competitor lines, and put them together with tester flies into the same culture vial (competitor females carrying GFP, competitor males carrying DsRed, tester females, and tester males; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). These parental flies could mate freely in any combination, and therefore the relative fitness of the tester compared to the competitor could be estimated from the relative proportion of offspring. After adult offspring emerged, they were classified according to their expression of GFP and DsRed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Briefly, the flies expressing only GFP (termed “Green progeny” hereafter) would be from parents of a GFP female and a tester male, the flies expressing only DsRed (termed “Red progeny”) would be from a tester female and a DsRed male, the flies expressing both markers (termed “Yellow progeny”) would be from a GFP female and a DsRed male, and the flies expressing neither marker (termed “White progeny”) would be from a tester female and a tester male. We performed the competition assay under different light conditions: DD or normal LD conditions, and examined the condition-dependent shift of the proportion of each type of offspring.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/365/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Competition assay for measuring relative fitness. (A) Four kinds of flies (competitor females carrying GFP, competitor males carrying DsRed, tester females, and tester males) were reared together and mating could occur in any combination of them. From the numbers of progeny, the relative fitness of tester parents was calculated. (B) An example of observations of progeny. The views under bright-field and fluorescent lights were merged in the image. According to the fluorescent markers, progeny were categorized into four groups. (C) The mean proportion of progeny in each test. Left: assays (n = 10) against the Oregon-R-S competitors (GFP-Oregon-R-S females and DsRed-Oregon-R-S males). Right: assays (n = 5) against the Urbana-S competitors (GFP-Urbana-S females and DsRed-Urbana-S males). The colors correspond to the combinations of parental flies shown in (A) and the terms used to designate the progeny group (Yellow, Green, Red, and White). *p-value &lt; 0.05; **p-value &lt; 0.01, Mann&#x2013;Whitney U-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2120685473" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Competition assay for measuring relative fitness. (A) Four kinds of flies (competitor females carrying GFP, competitor males carrying DsRed, tester females, and tester males) were reared together and mating could occur in any combination of them. From the numbers of progeny, the relative fitness of tester parents was calculated. (B) An example of observations of progeny. The views under bright-field and fluorescent lights were merged in the image. According to the fluorescent markers, progeny were categorized into four groups. (C) The mean proportion of progeny in each test. Left: assays (n = 10) against the Oregon-R-S competitors (GFP-Oregon-R-S females and DsRed-Oregon-R-S males). Right: assays (n = 5) against the Urbana-S competitors (GFP-Urbana-S females and DsRed-Urbana-S males). The colors correspond to the combinations of parental flies shown in (A) and the terms used to designate the progeny group (Yellow, Green, Red, and White). *p-value &lt; 0.05; **p-value &lt; 0.01, Mann&#x2013;Whitney &lt;em&gt;U&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/365/F1.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/2/365/F1.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/365/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/365/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467424" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Competition assay for measuring relative fitness. (A) Four kinds of flies (competitor females carrying GFP, competitor males carrying DsRed, tester females, and tester males) were reared together and mating could occur in any combination of them. From the numbers of progeny, the relative fitness of tester parents was calculated. (B) An example of observations of progeny. The views under bright-field and fluorescent lights were merged in the image. According to the fluorescent markers, progeny were categorized into four groups. (C) The mean proportion of progeny in each test. Left: assays (n = 10) against the Oregon-R-S competitors (GFP-Oregon-R-S females and DsRed-Oregon-R-S males). Right: assays (n = 5) against the Urbana-S competitors (GFP-Urbana-S females and DsRed-Urbana-S males). The colors correspond to the combinations of parental flies shown in (A) and the terms used to designate the progeny group (Yellow, Green, Red, and White). *p-value &lt; 0.05; **p-value &lt; 0.01, Mann–Whitney <em>U</em>-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We prepared two pairs of competitor strains by backcrossing with wild-type strains, Oregon-R-S or Urbana-S (see <em>Materials and Methods</em>), and then tested five combinations of competitor and tester (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). Each combination produced a particular number and proportion of offspring (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS2.pdf">Table S2</a>). Even when the same strains were used as competitor and tester (for example, Oregon-R-S competitor <em>vs.</em> Oregon-R-S tester), the proportions of Yellow/Green/Red/White offspring were not equal (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). These unequal proportions were probably due to the remaining genetic background of competitor lines and/or the effects of the GFP and DsRed transgenes. Importantly, we did not find any significant difference in these proportions between the LD and DD conditions in most cases. The exceptions were the cases of the Dark-fly tester. In the assay for Dark-fly tester and Oregon-R-S competitor, the proportion of the progeny changed according to the light conditions (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). The proportion of Red progeny was significantly decreased in the DD compared to the LD condition (Mann-Whitney <em>U</em>-test, p-value = 0.0062, n = 10, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS3.pdf">Table S3</a>). Conversely, the proportions of the White and Green progenies tended to be increased, though the differences were not statistically significant. These alterations of offspring proportion might have been due to different reproductive ability of Dark-fly in different light conditions (see <em>Discussion</em>). When we examined Dark-fly tester and Urbana-S competitor, the proportion of White progeny was significantly increased in the DD condition compared to the LD condition (p-value = 0.0159, n = 5, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS3.pdf">Table S3</a> and <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>), whereas the proportion of the Green progeny was decreased instead (p-value = 0.0318). These statistical evaluations were reconfirmed by the Welch <em>t</em>-test after the angular transformation of proportions (data not shown). These results indicate that regardless of which competitor line was used, Dark-fly exhibited a unique difference of its ability to reproduce in LD <em>vs.</em> DD conditions.</p><p id="p-26">From the above data, we also calculated the CF of tester <em>vs.</em> competitor (CF of 50% indicates equal fitness of tester and competitor; see <em>Materials and Methods</em>). The CF of Dark-fly <em>vs.</em> Oregon-R-S competitor showed a slight increase, from 54.1% in the LD to 55.6% in the DD condition (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS2.pdf">Table S2</a>). Against Urbana-S competitor, the CF of Dark-fly was significantly increased, from 54.8% in the LD condition to 66.8% in DD (Mann–Whitney <em>U</em>-test, p-value = 0.0318, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS2.pdf">Table S2</a>). We did not detect such a condition-dependent advantage for other testers we examined. These results suggest that Dark-fly produces more progeny in the DD than in the LD condition and as a result dominates over the wild-type fly in the dark, and support the idea that Dark-fly is adapted to a dark environment.</p></div><div id="sec-12" class="subsection"><h3>Mixed population experiment for identifying adaptive genes</h3><p id="p-27">To identify genes associated with Dark-fly’s adaptive traits, we designed a genome reselection experiment under differing environments. That is, we maintained large mixed populations of Dark-fly and Oregon-R-S (1:1 mixture starting from hybrid flies, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) under different light conditions: LD and DD. During successive generations, the Dark-fly genome and Oregon-R-S genome would be mixed in the population, and selection would act on each genomic locus. We hypothesized that the genes involved in dark-adaptation would dominate in the DD-reared population, but not in the LD-reared population. To test this, we reared three replicate populations each in the LD and DD conditions, and stored (frozen) adult flies at every generation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/365/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mixed population experiment. (A) A schematic drawing of the mixed population experiment. The experiment started from hybrids of Dark-fly and Oregon-R-S. Those mixed populations were reared in LD and DD conditions during consecutive generations. (B) The history of the mixed population. The sizes of the mixed populations were estimated by measuring the weight of flies at every generation, except for generations 10 and 22. Three replicate populations reared in the LD condition are shown by dark-red, red, and light-red lines, and those reared in the DD condition are shown by dark-blue, blue, and light-blue lines, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2120685473" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mixed population experiment. (A) A schematic drawing of the mixed population experiment. The experiment started from hybrids of Dark-fly and Oregon-R-S. Those mixed populations were reared in LD and DD conditions during consecutive generations. (B) The history of the mixed population. The sizes of the mixed populations were estimated by measuring the weight of flies at every generation, except for generations 10 and 22. Three replicate populations reared in the LD condition are shown by dark-red, red, and light-red lines, and those reared in the DD condition are shown by dark-blue, blue, and light-blue lines, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/365/F2.medium.gif" width="319" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/2/365/F2.medium.gif" width="319" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/365/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/365/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467691" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Mixed population experiment. (A) A schematic drawing of the mixed population experiment. The experiment started from hybrids of Dark-fly and Oregon-R-S. Those mixed populations were reared in LD and DD conditions during consecutive generations. (B) The history of the mixed population. The sizes of the mixed populations were estimated by measuring the weight of flies at every generation, except for generations 10 and 22. Three replicate populations reared in the LD condition are shown by dark-red, red, and light-red lines, and those reared in the DD condition are shown by dark-blue, blue, and light-blue lines, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">We examined the transition of population size during the first 60 generations by measuring the weight of the fly population (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The estimated average number of flies in a replicate population was 985, and the minimum and maximum numbers were 296 and 2069, respectively. There was no significant size difference among all populations (one-way ANOVA, p-value = 0.171, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS4.pdf">Table S4</a>). Thus, the mixed populations were kept without showing any effect of the LD or DD conditions on population size.</p><p id="p-30">We extracted the genomic DNA of the mixed population at generations 0, 22, and 49 (one replicate at generation 0, three replicates each in LD and DD conditions at generations 22 and 49, 13 populations in total) and performed whole-genome sequencing to measure the frequency of SNPs in each population’s genome. The average sequence coverage for the genome was 179 × in 13 populations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), so that we could detect a difference of SNP frequency for one out of 179 reads (0.006 frequency). First, we analyzed the SNP frequency at generation 0. All of the flies at generation 0 were hybrids of Dark-fly and Oregon-R-S, so we expected that Dark-fly SNPs would show a frequency of around 0.5 in the population. Among 217,329 SNPs previously identified in the Dark-fly genome (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>), we found 143,685 SNPs (66.11%) that had a frequency of between 0.4 and 0.6 in the population at generation 0 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS2.pdf">Figure S2</a>). The remaining 73,644 SNPs did not show a frequency of around 0.5, and most of them showed a frequency of over 0.9, implying that these SNPs would be common between Dark-fly and Oregon-R-S. In previous analyses, we might have overlooked some SNPs, probably due to low depth of coverage (mean depth: 13.7). Therefore, we took the 143,685 SNPs with a frequency of 0.4 to 0.6 at generation 0 as Dark-fly SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS2.pdf">Figure S2</a>, the black-outlined area) and used them in the following analyses.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467547/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467547/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of genome sequencing of the mixed population</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">We then calculated the frequency of each Dark-fly SNP in the replicate populations at generations 22 and 49. To evaluate the reliability of these frequencies calculated from the NGS data, we measured the frequency of some SNPs by a qPCR-based method (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Germer <em>et al.</em> 2000</a>). Briefly, we used two pairs of primers: one containing the reference sequence and the other carrying the Dark-fly SNP at its 3′ end, and estimated the SNP frequency in the population from the difference of the amplification cycles of qPCR using these two primer pairs. We found that SNP frequency measured by qPCR (total 116 samples) correlated well with the frequency calculated from NGS data (coefficient of determination R<sup>2</sup> = 0.909, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS3.pdf">Figure S3</a>), confirming the accuracy of our SNP frequency data.</p></div><div id="sec-13" class="subsection"><h3>Analyses of population genome</h3><p id="p-33">We characterized the features of the overall SNP frequency in the populations, and observed that the SNP frequency became progressively more diverse during successive generations (comparing generations 0–22 and 22–49), as expected (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The average overall SNP frequency was not markedly different between populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS5.pdf">Table S5</a>), but was slightly higher in the DD-reared population than in the LD-reared population at generation 49 (0.588 <em>vs.</em> 0.560 for the average of three replicates). Plotting the SNP frequency along the chromosomal position (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS4.pdf">Figure S4</a>) revealed that the frequency of SNPs increased or decreased in position-dependent manners, but was roughly conserved among populations. The plots of SNP frequency displayed overall a “belt-like” pattern, with the only exception being around the centromere of chromosome 2: the discontinuity of SNP frequency in this region might be due to structural variants of this chromosome (see <em>Discussion</em>). These observations of SNP frequency suggest that some of Dark-fly’s SNPs would be advantageous, and some would be disadvantageous, so that their frequencies would increase or decrease, respectively, during successive generations.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/365/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency of Dark-fly&#x2019;s SNPs in the LD- and DD-reared populations. (A) Violin plot of the frequency of Dark-fly&#x2019;s SNPs in the populations reared in LD and DD conditions at generations 0, 22, and 49. White points and back thick bars represent median values and interquartile ranges of data, respectively. (B) MDS analysis of the overall SNP frequency. Profiles of SNP frequency in each replicate population were plotted in two dimensions using MDS analysis (stress = 15.9). Dimension one divided populations at different time points, and dimension two divided populations in LD and DD conditions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2120685473" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequency of Dark-fly&#x2019;s SNPs in the LD- and DD-reared populations. (A) Violin plot of the frequency of Dark-fly&#x2019;s SNPs in the populations reared in LD and DD conditions at generations 0, 22, and 49. White points and back thick bars represent median values and interquartile ranges of data, respectively. (B) MDS analysis of the overall SNP frequency. Profiles of SNP frequency in each replicate population were plotted in two dimensions using MDS analysis (stress = 15.9). Dimension one divided populations at different time points, and dimension two divided populations in LD and DD conditions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/365/F3.medium.gif" width="255" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/2/365/F3.medium.gif" width="255" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/365/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/365/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467433" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Frequency of Dark-fly’s SNPs in the LD- and DD-reared populations. (A) Violin plot of the frequency of Dark-fly’s SNPs in the populations reared in LD and DD conditions at generations 0, 22, and 49. White points and back thick bars represent median values and interquartile ranges of data, respectively. (B) MDS analysis of the overall SNP frequency. Profiles of SNP frequency in each replicate population were plotted in two dimensions using MDS analysis (stress = 15.9). Dimension one divided populations at different time points, and dimension two divided populations in LD and DD conditions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">To characterize the overall SNP frequency of successive populations reared under LD and DD conditions, we performed nonmetric MDS analysis of SNP frequencies. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> shows that the SNP frequencies showed a similar profile among replicate populations, and that the profile shifted from generation 22 to 49. Importantly, dimension two of the plot showed a clear difference between the LD- and DD-reared populations. These results indicate that the overall SNP frequency was similar among replicate populations reared in the same condition, but was progressively altered between LD- and DD-reared populations during successive generations. We suggest that overall, the changes of SNP frequency were rarely caused by random genetic drift, but rather were mainly caused by selection forces and that the selections under LD and DD conditions were recorded in the population genome.</p></div><div id="sec-14" class="subsection"><h3>Comparison of SNP frequency between LD and DD conditions in successive generations</h3><p id="p-36">We next sought to identify SNPs showing different frequency in LD- and DD-reared populations. The mean frequency of each SNP in replicate populations was obtained for comparison in subsequent analyses. Then, allele frequency changes (AFCs) between conditions (frequency in DD − frequency in LD) were calculated for each SNP at generations 22 and 49. The distribution of AFCs gradually became broader during successive generations, <em>i.e.</em>, the difference of the frequency of SNPs increased with increasing generations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS5.pdf">Figure S5</a>). AFCs at generation 49 were slightly shifted in the plus-direction, indicating that the number of the Dark-fly’s SNPs showing higher frequency in the DD than in the LD condition was larger than that of the Dark-fly’s SNPs showing the opposite trend. This biased AFC distribution suggests that the Dark-fly genome carries a large fraction of SNPs positively selected in the DD condition or negatively selected in the LD condition.</p><p id="p-37">Next, we examined how the SNP frequencies were affected by light condition. For this, we plotted the frequency of each SNP in the LD condition (<em>x</em>-axis) and DD condition (<em>y</em>-axis) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>). Many SNPs showed the same frequency in the LD and DD conditions, and thus were plotted near the <em>y</em> = <em>x</em> line. However, some SNPs showed a different frequency in the LD and DD conditions, and thus were scattered far from this line. Compared to the results of generation 22, more SNPs showed differential frequency at generation 49. Fisher’s exact test was then performed to identify the SNPs showing significantly different frequency between the LD- and DD-reared populations, and identified 12,141 such SNPs at generation 22 and 31,704 such SNPs at generation 49 (Bonferroni-corrected p-value &lt; 0.01). Among them, the number of SNPs showing higher frequency in DD than in LD was significantly larger than the number showing lower frequency (chi-square test, p-value = 1.9E-5 at generation 22 and p-value &lt; 2.2E-16 at generation 49), confirming that a large fraction of Dark-fly’s SNPs increased in frequency in DD conditions. We focused on SNPs that showed higher frequency in the DD than in the LD condition, and whose p-values were ranked in the top 5% (colored dots in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A and B</a>). At generation 49, this criterion of the p-value (p-value &lt; 1e-17.3) was adequate for detecting outliers from SNPs with a linear distribution of expected values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS6.pdf">Figure S6</a>), though this criteria when applied at generation 22 (p-value &lt; 1e-8.67) was not adequate for detecting such outliers, indicating that more SNPs had been selected in the population at generation 49 than at generation 22. At generation 49, 6015 such SNPs were grouped into three types according to their relative frequency in LD <em>vs.</em> DD conditions: type 1, 2, and 3 showed high, middle, and low relative frequency, respectively, in the DD compared to LD condition on average (red, green, and blue in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>, see <em>Materials and Methods</em>). These three different types might have resulted from different selection patterns of the respective SNPs by DD <em>vs.</em> LD conditions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/365/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of SNP frequency in LD- and DD-reared populations. (A, B) Scatter plots comparing SNP frequency in LD and DD conditions at generation 22 (A) and 49 (B). The SNPs showing a significant difference of frequency in Fisher&#x2019;s exact test (top 5% of p-values) and higher frequency in the DD than in the LD condition were colored black at generation 22 and red, green, and blue (type 1, type 2, and type 3, respectively) at generation 49. Other SNPs were colored gray. (C, D) P-values (Fisher&#x2019;s exact test) for each SNP were plotted as reverse logarithm values along chromosomal position at generation 22 (C) and 49 (D). Colors corresponded to those shown in (A, B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2120685473" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of SNP frequency in LD- and DD-reared populations. (A, B) Scatter plots comparing SNP frequency in LD and DD conditions at generation 22 (A) and 49 (B). The SNPs showing a significant difference of frequency in Fisher&#x2019;s exact test (top 5% of p-values) and higher frequency in the DD than in the LD condition were colored black at generation 22 and red, green, and blue (type 1, type 2, and type 3, respectively) at generation 49. Other SNPs were colored gray. (C, D) P-values (Fisher&#x2019;s exact test) for each SNP were plotted as reverse logarithm values along chromosomal position at generation 22 (C) and 49 (D). Colors corresponded to those shown in (A, B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/365/F4.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/2/365/F4.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/365/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/365/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467472" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Comparison of SNP frequency in LD- and DD-reared populations. (A, B) Scatter plots comparing SNP frequency in LD and DD conditions at generation 22 (A) and 49 (B). The SNPs showing a significant difference of frequency in Fisher’s exact test (top 5% of p-values) and higher frequency in the DD than in the LD condition were colored black at generation 22 and red, green, and blue (type 1, type 2, and type 3, respectively) at generation 49. Other SNPs were colored gray. (C, D) P-values (Fisher’s exact test) for each SNP were plotted as reverse logarithm values along chromosomal position at generation 22 (C) and 49 (D). Colors corresponded to those shown in (A, B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">To further characterize the patterns of selection of these SNPs, we integrated the data of generations 0, 22, and 49, and examined the trajectory of SNP frequency through successive generations (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Type 1 SNPs (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>, red, n = 1108) increased in frequency with increasing generations in both LD and DD conditions, but the increment was higher in the DD condition (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). Fisher’s exact tests for the temporal changes of frequency indicated that a larger number of SNPs showed significant increments of frequency in the DD than in the LD condition (data not shown). These findings imply that type 1 SNPs were selected positively in both conditions, but the positive selection was stronger in the DD condition. Likewise, type 2 SNPs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>, green, n = 4000) were selected positively in the DD but negatively in the LD condition (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). Finally, type 3 SNPs (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>, blue, n = 907) were selected negatively in both conditions, but the negative selection was stronger in the LD condition (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). These results suggested that there were at least three types of environmental selection acting on the Dark-fly genome.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/365/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trajectory of frequency of SNPs during successive generations. Temporal changes of SNP frequency were analyzed for each SNP type (A: type 1, B: type 2, C: type 3). Blue and red lines represent data of each SNP in DD- and LD-reared populations, respectively. Black lines represent mean SNP frequency of each type in DD- and LD-reared populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2120685473" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Trajectory of frequency of SNPs during successive generations. Temporal changes of SNP frequency were analyzed for each SNP type (A: type 1, B: type 2, C: type 3). Blue and red lines represent data of each SNP in DD- and LD-reared populations, respectively. Black lines represent mean SNP frequency of each type in DD- and LD-reared populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/365/F5.medium.gif" width="242" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/2/365/F5.medium.gif" width="242" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/365/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/365/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467616" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Trajectory of frequency of SNPs during successive generations. Temporal changes of SNP frequency were analyzed for each SNP type (A: type 1, B: type 2, C: type 3). Blue and red lines represent data of each SNP in DD- and LD-reared populations, respectively. Black lines represent mean SNP frequency of each type in DD- and LD-reared populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We then examined the chromosomal positions of these selected SNPs, and found that the SNPs were accumulated in several distinct chromosomal regions (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4D</a>). The accumulation of many SNPs in a particular locus implies that one or a few SNPs in each locus would be truly under selection force, and other nearby SNPs in that locus would change their frequencies as a result of the effects of linkage (see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS4.pdf">Figure S4</a>). However, we found that SNPs categorized into the same selection type were mapped to several separate loci on different chromosomes. That is, several loci shared the same trajectory of selection, and thus we speculate that several responsible SNPs (genes) at separate loci might function together for an adaptive trait of Dark-fly (see <em>Discussion</em>).</p></div><div id="sec-15" class="subsection"><h3>Candidates for adaptive genes</h3><p id="p-42">To search for a link between selected SNPs and Dark-fly’s traits, we further characterized the chromosomal loci carrying selected SNPs. We identified 28 such regions, which had accumulated selected SNPs and are separated from each other by at least 100 kb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS6.pdf">Table S6</a>). In our previous genome analyses, we identified ROH regions in the Dark-fly genome (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>). An ROH region is a genomic region with homozygosity extending for a long distance, and is known as a signature of positive selection during a population’s history (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McQuillan <em>et al.</em> 2008</a>). We found that four out of the 28 regions selected in mixed populations overlapped with ROH regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS6.pdf">Table S6</a>). This overlapping could not be explained merely by coincidence, because ROH regions cover only a much smaller fraction (3.5%) of the whole genome. Rather, the overlapping regions had been selected both in the history of Dark-fly and in the current experiment of selection in mixed populations.</p><p id="p-43">Among the genes located in the 28 selected regions, we identified 1283 genes carrying SNPs or InDels in the Dark-fly genome (see <em>Materials and Methods</em>). The nonsynonymous SNPs and InDels located in the gene coding regions might alter the function and activity of gene products. Alternatively, the SNPs and InDels located in intergenic regions might affect the expression of genes. We consider these 1283 genes to be primary candidates selected in the mixed populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FileS1.zip">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FileS2.zip">File S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FileS5.zip">File S5</a>).</p><p id="p-44">The 28 selected regions we identified constitute a total of 10,856,126 bases (6.4% of the genome), and include more than 1000 genes, so it would be laborious to test the role of each of these candidate genes for adaptive traits of Dark-fly. Thus, to narrow down the candidates, we utilized the MULTIPOOL program (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Edwards and Gifford 2012</a>), which predicts QTL in bulked segregant analysis. Comparing the LD- and DD-reared populations, the LOD scores at 1-kb windows were computed along the chromosome from the data of generations 22 and 49. We set an optional threshold of LOD score as 75, for extracting a clear difference between generation 22 and 49 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS7.pdf">Figure S7</a>). Using this threshold, we identified nine peaks of LOD scores in the data of generation 49 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>). The LOD peaks detected were well correlated with the regions with high values of absolute AFCs, meaning that there was a large difference in these regions between the LD- and DD-reared populations (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, A and B</a>). Among the nine LOD peaks, seven peaks showed a higher SNP frequency in the DD than in the LD condition, and were narrowed to a span (26–131 kb) with a 90% credible interval (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS8.pdf">Figure S8</a>). We listed 84 genes carrying SNPs and InDels within these LOD peaks (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For example, peak 9 (spanning for 26 kb) was located within a large gene, Ptp99A, encoding a protein tyrosine phosphatase (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6, C and D</a>). These 84 genes are candidates for genes associated with Dark-fly’s traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FileS3.zip">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FileS4.zip">File S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FileS5.pdf">File S5</a>, see <em>Discussion</em>), and we need to examine the roles of these genes in future work.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/365/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosomal regions selected in the mixed populations. (A) AFC of each SNP (frequency in DD minus frequency in LD) at generation 49 were plotted along chromosomal position. A blue line represents AFC = 0, meaning equal frequency in LD- and DD-reared populations. (B) LOD scores of 1-kb windows at generation 49 were plotted along chromosomal position. The blue line represents the threshold (75) for detecting LOD peaks. Numbers in the graph indicate LOD peak number. (C) A magnified view of LOD peak nine. Red points indicate SNP frequency in LD and blue points indicate that in DD. The green line and gray bar represent LOD score and 90% credible interval span, respectively. (D) A view of UCSC Genomic Browser around the 90% credible interval span of LOD peak nine. Position of Dark-fly&#x2019;s SNPs and InDels are represented by bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2120685473" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosomal regions selected in the mixed populations. (A) AFC of each SNP (frequency in DD minus frequency in LD) at generation 49 were plotted along chromosomal position. A blue line represents AFC = 0, meaning equal frequency in LD- and DD-reared populations. (B) LOD scores of 1-kb windows at generation 49 were plotted along chromosomal position. The blue line represents the threshold (75) for detecting LOD peaks. Numbers in the graph indicate LOD peak number. (C) A magnified view of LOD peak nine. Red points indicate SNP frequency in LD and blue points indicate that in DD. The green line and gray bar represent LOD score and 90% credible interval span, respectively. (D) A view of UCSC Genomic Browser around the 90% credible interval span of LOD peak nine. Position of Dark-fly&#x2019;s SNPs and InDels are represented by bars.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/365/F6.medium.gif" width="360" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/2/365/F6.medium.gif" width="360" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/365/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/365/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467590" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">Chromosomal regions selected in the mixed populations. (A) AFC of each SNP (frequency in DD minus frequency in LD) at generation 49 were plotted along chromosomal position. A blue line represents AFC = 0, meaning equal frequency in LD- and DD-reared populations. (B) LOD scores of 1-kb windows at generation 49 were plotted along chromosomal position. The blue line represents the threshold (75) for detecting LOD peaks. Numbers in the graph indicate LOD peak number. (C) A magnified view of LOD peak nine. Red points indicate SNP frequency in LD and blue points indicate that in DD. The green line and gray bar represent LOD score and 90% credible interval span, respectively. (D) A view of UCSC Genomic Browser around the 90% credible interval span of LOD peak nine. Position of Dark-fly’s SNPs and InDels are represented by bars.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467535/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467535/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Candidate regions identified at LOD peaks</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>The adaptive traits of Dark-fly in a dark environment</h3><p id="p-47">Here we found that the relative fitness of Dark-fly under reproductive competition against other strains of flies was higher in the DD compared to the LD condition. Various wild-type lines we examined did not show such an advantage in the DD condition, indicating that Dark-fly is a unique line with regard to its display of reproductive success in a dark environment. Reproductive success could be increased via various traits, such as mating behaviors, egg-laying ability, egg-to-adult viability, and so on. From the results of our reproductive competition experiments between Dark-fly and the Oregon-R-S strain, we found that the proportion of Red progeny (resulting from Dark-fly females and DsRed-Oregon-R-S males) was decreased in the DD condition, whereas the proportion of White progeny (resulting from Dark-fly females and Dark-fly males) tended to be increased in this condition. There are various possible explanations for this result. One possibility is that Dark-fly females might prefer Dark-fly males as partners, and avoid DsRed-Oregon-R-S males, in the dark. Regarding such a preference, Dark-fly females might detect some specific cues, such as odors or sounds, to discriminate partners in the absence of visual cues (also see below).</p><p id="p-48">When we performed the competition assay against Urbana-S competitors, we observed a different pattern of performance of Dark-fly. That is, the proportion of White progeny was increased in the DD condition, whereas that of Green progeny (from GFP-Urbana-S females and Dark-fly males) was decreased. In this case, Dark-fly males might have a preference for Dark-fly females compared to Urbana-S females in the dark. Thus, Dark-fly exhibited different advantages when reared in the dark depending on the competitor line. Although we could not define particular traits contributing to Dark-fly’s fitness in the dark, we confirmed here that Dark-fly possesses some advantageous traits for reproduction in a dark environment, and these advantages suggest that Dark-fly could dominate over the wild-type fly in the dark.</p></div><div id="sec-18" class="subsection"><h3>Mixed population experiment</h3><p id="p-49">We designed a mixed population experiment to identify genes involved in the adaptation of Dark-fly. In many previous studies about environmental adaptation, QTL analyses revealed the genes associated with traits of organisms (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nadeau and Jiggins 2010</a>). Most of those studies focused on unique traits, such as morphological features, presumably related to the adaptation. Although Dark-fly does not possess any obvious morphological traits, it exhibits some advantages regarding reproduction in the dark. Therefore, we carried out experiments aiming at fitness-based selection of genes in a mixed population of Dark-fly and other fly strains, and as a result succeeded in observing environment-dependent selections of some genes (although we might have overlooked some weak selections using this approach).</p><p id="p-50">GWAS is one of the approaches for analyzing population genomes with large genetic diversity in their background. Its statistical power is very high for detecting the consistency of genome alterations across large populations. Since our mixed population experiment started from two fly strains, the genome diversity in the population was quite low compared to that in GWAS. However, our experiment has the advantage that it uses a defined environment for the selection and makes it possible to observe the temporal pattern of the selection during successive generations. The mixed population experiment is thus a powerful strategy for identifying genome variations related to the fitness in a defined environment, and could be useful for application to other organisms.</p></div><div id="sec-19" class="subsection"><h3>Genome analysis of mixed populations</h3><p id="p-51">We analyzed the SNP frequency in the pooled genome of the populations. Analysis of pooled genomes has now become widely used because of its cost-effective high performance (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schlotterer <em>et al.</em> 2014</a>). Although the pooled genome loses the information of the haplotype structure of the population, it deals with a large population size and the whole genome at once. We extracted genomic DNA from 1000 individual flies, and obtained sequence data covering 179×-depth of the genome on average. We analyzed more than 140,000 SNPs, and thus obtained fine-scale information along the whole genome (at approximately 1-kb intervals on average). From these data, we characterized the genomes of the large populations.</p><p id="p-52">For this population genome analysis, we used confirmed Dark-fly SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS2.pdf">Figure S2</a>, the black-outlined area), which were identified as Dark-fly’s SNPs in our previous study (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Izutsu <em>et al.</em> 2012</a>) and showed a frequency of 0.4–0.6 in the mixed population at 0 generation. It should be noted, however, that we detected 161,893 SNPs which showed a frequency of 0.4–0.6 at 0 generation but were identified in neither Dark-fly nor Oregon-R-S in our previous study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS2.pdf">Figure S2</a>). We might have overlooked these SNPs in the previous analysis, probably because of low coverage of sequencing. In addition to such hidden SNPs, we did not evaluate heterozygous SNPs of the original Dark-fly in this study. Although we detected a small number of heterozygous SNPs in the previous analyses of the Dark-fly genome, we might have overlooked more such SNPs. In this study, we used only confirmed homozygous Dark-fly SNPs to examine selections, although as a result we might have missed some SNPs actually selected. In order to minimize such an oversight, we will need to obtain more precise and comprehensive genome data for parental Dark-fly. Thus, improved accuracy of the sequence of the parental genome would be an important factor for ensuring the analytical scale and precision in the pooled genome analysis.</p><p id="p-53">The SNP frequency in the population would be affected by selective pressure and by recombination in the genome, and consequently the frequency of SNPs would change in a coordinated manner along chromosomal positions. However, we observed marked discontinuity of the SNP frequency in the region near the centromere of chromosome 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/FigureS4.pdf">Figure S4</a>). This was probably not due to accidental extraction of some SNPs in a population, because replicate populations showed almost the same zigzag pattern of frequency changes along this chromosomal region. Rather, we speculate that there might be structural variants (duplication, deletion, inversion, insertion, <em>etc</em>.) of the Dark-fly or Oregon-RS genome in this region that might cause discontinuous changes of SNP frequency. Our analysis did not evaluate SNP selection precisely in this region, and we had to eliminate this region for identifying adaptive SNPs. Such structural variants of chromosomal regions would be a confounding issue for analyzing population genomes, and detailed information about the parental genome would be required for resolving the issues presented by such variants.</p></div><div id="sec-20" class="subsection"><h3>SNP frequency trajectory during selection</h3><p id="p-54">We detected SNPs showing different frequencies between the LD and DD-reared populations. We unexpectedly detected some Dark-fly SNPs showing higher frequency in the LD compared to the DD condition (see <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6A</a>, minus values of AFC). Dark-fly might possibly carry such SNPs advantageous in LD as a result of random genetic drift during its history. Alternatively, it is possible that these SNPs are neutral but that their frequencies in the mixed populations were changed by random genetic drift. However, we emphasize that a much larger fraction of Dark-fly’s SNPs increased in frequency in the DD condition (higher in DD: 6015 SNPs; higher in LD: 1129 SNPs). This implies that SNP selection was not totally random, but rather mostly biased to dark-selection. We therefore suggest that unique features of Dark-fly supporting condition-dependent fitness largely result in the difference of SNP frequency in the mixed population in LD and DD conditions.</p><p id="p-55">From analyses of the population genome at generation 49, we identified Dark-fly SNPs showing higher frequency in the DD than in the LD condition, and categorized them into three types according to overall frequency: type 1, 2, and 3 showed high, middle, and low frequency, respectively. We also observed the time-course trajectory of the SNP frequency through successive generations for each type, and found apparently different selections acting on each type. From the overall trajectory of their increasing frequency across successive generations, we suggest that the type 1 SNPs were advantageous and positively selected in the DD condition. From the overall trajectory of decline, type 3 SNPs appeared to be disadvantageous and negatively selected in the LD condition. Type 2 SNPs appeared to be advantageous in DD and simultaneously disadvantageous in the LD condition, so they were selected positively in DD and negatively in LD. Thus, Dark-fly carries at least three types of SNPs undergoing different modes of selection in the dark.</p><p id="p-56">It is possible that SNPs showing a similar trajectory might be associated with the same trait of Dark-fly. In order to address the possibility, we will need to identify the genes responsible for a Dark-fly trait and to examine the epistatic interactions of several genes in the future. From the present analyses of SNP frequency trajectory, we suggest that Dark-fly possesses at least three traits differentially selected in dark conditions, and that a number of genes contribute to each trait.</p></div><div id="sec-21" class="subsection"><h3>Candidates for Dark-fly adaptive genes</h3><p id="p-57">The selected SNPs were accumulated in several chromosomal loci. It is unlikely that all of those SNPs contribute to an advantage or disadvantage in DD or LD conditions. Instead, one or a few SNPs in each locus were probably selected positively or negatively, and the frequencies of the other SNPs were changed as a result of the effects of linkage. It is also possible that InDels or other structural variants in the locus might contribute to the Dark-fly’s traits. In this case, the SNPs would simply be markers for detecting the locus.</p><p id="p-58">In general, there are two possible effects of mutations (SNPs, InDels, or structural variants) that affect unique traits of organisms. First, these mutations might be located within the coding region of a gene and might generate nonsynonymous or frameshift mutations in the gene. In this case, the mutations would alter the activity of gene products. Second, the mutations might be located in an intergenic region, and might alter the expression of a gene(s). Both cases indeed have been documented in previous studies of evolved traits of organisms (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pal <em>et al.</em> 2006</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nadeau and Jiggins 2010</a>). One way to distinguish between these possibilities is transcriptome analysis. We have performed transcriptome analysis for the wild-type fly and Dark-fly reared in LD and DD conditions and have detected some differentially expressed genes (N. Fuse, M. Izutsu, and K. Agata, unpublished data); we will report the results in the near future.</p><p id="p-59">As primary candidates for Dark-fly’s adaptive genes, here we identified 1283 genes that are located in selected chromosomal regions and that carry SNPs or InDels in the Dark-fly genome (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1/TableS6.pdf">Table S6</a>). We used the MULTIPOOL program, a program that has successfully identified QTL loci in previous studies, and that enabled us to narrow down the candidates to seven selected regions, and to identify 84 genes carrying SNPs or InDels in these regions of the Dark-fly genome. We suggest that these genes are potentially strong candidates for those involved in Dark-fly’s adaptation.</p><p id="p-60">Concerning the reproductive success of Dark-fly in a dark environment, as shown in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>, we considered several hypotheses about possible roles of candidate genes. Among the candidates (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), Or22a, Or22b, and CheA87a encode olfactory and chemosensory receptors. Although the chemical entities binding to these receptors have not been identified yet, it is possible that these receptors might perceive pheromonal signals. We speculate that the mutations of pheromone receptors might enhance sexual activity in Dark-fly. Other notable candidate genes are CG8784 and CG8795 (also called PK2-R1 and -R2), encoding receptors for a neuropeptide, Pyrokinin-2 (also called Pheromone Biosynthesis Activating Neuropeptide; PBAN), and it is known that this humoral signaling activates pheromone synthesis in several <em>Diptera</em> species (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Choi <em>et al.</em> 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rosenkilde <em>et al.</em> 2003</a>). We also note that Ptp99A, encoding a protein tyrosine phosphatase, is known to be involved in olfactory memory formation (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Walkinshaw <em>et al.</em> 2015</a>). Thus, these candidate genes might be related to unusual pheromone synthesis and/or perception in Dark-fly.</p><p id="p-61">We also considered another hypothesis, namely, that Dark-fly might possess abnormal circadian rhythm that results in higher fitness in the dark. While Dark-fly retains circadian locomotor rhythms (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Imafuku and Haramura 2011</a>) as noted in the <em>Introduction</em>, its rhythms of mating behavior and gametogenesis physiology have not been investigated. Some genes in the candidate list are indeed expressed in circadian oscillatory manners (Ace and Dip-B; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hooven <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Claridge-Chang <em>et al.</em> 2001</a>). The mouse homolog of SNF1A (also called AMPKα) is known to regulate the circadian clock through cryptochrome phosphorylation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lamia <em>et al.</em> 2009</a>). Another notable candidate, Jafrac1, is an evolutionarily conserved marker for circadian rhythms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Edgar <em>et al.</em> 2012</a>). In the future, we will further analyze the functions of these candidate genes identified here to elucidate the molecular mechanisms underlying the environmental adaptation of Dark-fly.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We thank Drs. M. Imafuku (Seika University) and T. Kitamura (Tokyo University) for providing us with Dark-fly. We thank Mr. Y. Sugiyama (Kyoto University) for helping with the initial experiments of the fitness assay, Mr. K. Tsujimoto (Kyoto University) for giving us the illustrations of the fly, Mr. O. Nishimura (RIKEN, CDB), Drs. T. Takano (Kyoto Institute of Technology), S. Hayashi (RIKEN CDB), and T. Uemura (Kyoto University) for giving us valuable advice, and Dr. E. Nakajima (Kyoto University) for critical reading of the manuscript. We also thank the members of the Agata laboratory and the Uemura laboratory for discussions and encouragement. This work was supported by GCOE program A06 of Kyoto University, NIG Collaborative Research Program A-67, a Japan Society for the Promotion of Science (JSPS) grant (26430189) and a Research fellowship of the JSPS for young scientists (to M.I.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-63">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023549/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-64">Sequence data from this article have been deposited with the EMBL/GenBank Data Libraries under accession no. DRA004032 (DRR046855-DRR046867).</p></li><li class="fn" id="fn-5"><p id="p-65"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-08"><span class="received-label">Received </span>October 8, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-11-30"><span class="accepted-label">Accepted </span>November 30, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Izutsu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.1" data-doi="10.1038/nature12904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treusch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of single-cell protein abundance variation in large yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">506</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D506%26rft.spage%253D494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12904%26rft_id%253Dinfo%253Apmid%252F24402228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24402228&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332165100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.2" data-doi="10.1186/1471-2164-11-88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araya</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing of a laboratory-evolved yeast strain.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAraya%26rft.auinit1%253DC.%2BL.%26rft.volume%253D11%26rft.spage%253D88%26rft.epage%253D88%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Ba%2Blaboratory-evolved%2Byeast%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-88%26rft_id%253Dinfo%253Apmid%252F20128923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20128923&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.3" data-doi="10.1038/nrg3015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">R. D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular spandrels: tests of adaptation at the genetic level.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D767%26rft.epage%253D780%26rft.atitle%253DMolecular%2Bspandrels%253A%2Btests%2Bof%2Badaptation%2Bat%2Bthe%2Bgenetic%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3015%26rft_id%253Dinfo%253Apmid%252F22005986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22005986&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.4" data-doi="10.1038/nrg3564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome dynamics during experimental evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3564%26rft_id%253Dinfo%253Apmid%252F24166031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166031&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.5" data-doi="10.1038/nature08480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome evolution and adaptation in a long-term experiment with Escherichia coli.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrick%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D461%26rft.issue%253D7268%26rft.spage%253D1243%26rft.epage%253D1247%26rft.atitle%253DGenome%2Bevolution%2Band%2Badaptation%2Bin%2Ba%2Blong-term%2Bexperiment%2Bwith%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08480%26rft_id%253Dinfo%253Apmid%252F19838166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19838166&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271190800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.6" data-doi="10.1038/nrg700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Understanding quantitative genetic variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D21%26rft.atitle%253DUnderstanding%2Bquantitative%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg700%26rft_id%253Dinfo%253Apmid%252F11823787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823787&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.7" data-doi="10.1093/jhered/93.1.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkens</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mapping a cave fish genome: polygenic systems and regressive evolution.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBorowsky%26rft.auinit1%253DR.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D21%26rft.atitle%253DMapping%2Ba%2BCave%2BFish%2BGenome%253A%2BPolygenic%2BSystems%2Band%2BRegressive%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.19%26rft_id%253Dinfo%253Apmid%252F12011170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.8" data-doi="10.1038/nature09352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahrestani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of a long-term evolution experiment with Drosophila.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurke%26rft.auinit1%253DM.%2BK.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Ba%2Blong-term%2Bevolution%2Bexperiment%2Bwith%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09352%26rft_id%253Dinfo%253Apmid%252F20844486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844486&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.9" data-doi="10.1038/nature05911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craddock</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deloukas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.10" data-doi="10.1007/s00441-001-0467-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">M.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafaeli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurenka</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Pyrokinin/PBAN-like peptides in the central nervous system of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Cell Tissue Res.</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Band%2Btissue%2Bresearch%26rft.stitle%253DCell%2BTissue%2BRes%26rft.aulast%253DChoi%26rft.auinit1%253DM.%2BY.%26rft.volume%253D306%26rft.issue%253D3%26rft.spage%253D459%26rft.epage%253D465%26rft.atitle%253DPyrokinin%252FPBAN-like%2Bpeptides%2Bin%2Bthe%2Bcentral%2Bnervous%2Bsystem%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00441-001-0467-x%26rft_id%253Dinfo%253Apmid%252F11735047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00441-001-0467-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11735047&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172560900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.11" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">le Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.12" data-doi="10.1016/S0896-6273(01)00515-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claridge-Chang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijnen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naef</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boothroyd</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajewsky</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Circadian regulation of gene expression systems in the Drosophila head.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DClaridge-Chang%26rft.auinit1%253DA.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D657%26rft.epage%253D671%26rft.atitle%253DCircadian%2Bregulation%2Bof%2Bgene%2Bexpression%2Bsystems%2Bin%2Bthe%2BDrosophila%2Bhead.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0896-6273%252801%252900515-3%26rft_id%253Dinfo%253Apmid%252F11719206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0896-6273(01)00515-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11719206&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172363900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nereng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genetic architecture of parallel armor plate reduction in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D2%26rft.spage%253D635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000221599500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.14" data-doi="10.1038/nature11088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmedo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Peroxiredoxins are conserved markers of circadian rhythms.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BS.%26rft.volume%253D485%26rft.issue%253D7399%26rft.spage%253D459%26rft.epage%253D464%26rft.atitle%253DPeroxiredoxins%2Bare%2Bconserved%2Bmarkers%2Bof%2Bcircadian%2Brhythms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11088%26rft_id%253Dinfo%253Apmid%252F22622569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22622569&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304344500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.15" data-doi="10.1186/1471-2105-13-S14-S8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping with pooled sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">S8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-S14-S8%26rft_id%253Dinfo%253Apmid%252F23282012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-S14-S8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23282012&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-6.2.365.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitamura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haramura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arikawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imafuku</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Evolution in the Dark: Adaptation of Drosophila in the Laboratory</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Tokyo</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.17" data-doi="10.1101/gr.10.2.258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Germer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">High-throughput SNP allele-frequency determination in pooled DNA samples by kinetic PCR.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DGermer%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D258%26rft.epage%253D266%26rft.atitle%253DHigh-Throughput%2BSNP%2BAllele-Frequency%2BDetermination%2Bin%2BPooled%2BDNA%2BSamples%2Bby%2BKinetic%2BPCR%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.2.258%26rft_id%253Dinfo%253Apmid%252F10673283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzIvMjU4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.18" data-doi="10.1078/0944-2006-00123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evolution in Darwin’s finches: a review of a study on Isla Daphne Major in the Galapagos Archipelago.</span> <abbr class="cit-jnl-abbrev">Zoology</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoology%26rft.volume%253D106%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.1078%252F0944-2006-00123%26rft_id%253Dinfo%253Apmid%252F16351911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/0944-2006-00123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16351911&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Interactions between natural selection, recombination and gene density in the genes of Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHey%26rft.auinit1%253DJ.%26rft.volume%253D160%26rft.issue%253D2%26rft.spage%253D595%26rft.epage%253D608%26rft.atitle%253DInteractions%2BBetween%2BNatural%2BSelection%252C%2BRecombination%2Band%2BGene%2BDensity%2Bin%2Bthe%2BGenes%2Bof%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F11861564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzIvNTk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.20" data-doi="10.1371/journal.pone.0006469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hooven</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butcher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giebultowicz</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Does the clock make the poison? Circadian variation in response to pesticides.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHooven%26rft.auinit1%253DL.%2BA.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De6469%26rft.epage%253De6469%26rft.atitle%253DDoes%2Bthe%2Bclock%2Bmake%2Bthe%2Bpoison%253F%2BCircadian%2Bvariation%2Bin%2Bresponse%2Bto%2Bpesticides.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006469%26rft_id%253Dinfo%253Apmid%252F19649249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19649249&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.21" data-doi="10.2108/zsj.28.195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imafuku</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haramura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Activity rhythm of Drosophila kept in complete darkness for 1300 generations.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DImafuku%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D198%26rft.atitle%253DActivity%2Brhythm%2Bof%2BDrosophila%2Bkept%2Bin%2Bcomplete%2Bdarkness%2Bfor%2B1300%2Bgenerations.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.28.195%26rft_id%253Dinfo%253Apmid%252F21385060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.28.195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21385060&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.22" data-doi="10.1371/journal.pone.0033288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Izutsu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugiyama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aizu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome features of “Dark-fly”, a Drosophila line reared long-term in a dark environment.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e33288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DIzutsu%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33288%26rft.epage%253De33288%26rft.atitle%253DGenome%2Bfeatures%2Bof%2B%2522Dark-fly%2522%252C%2Ba%2BDrosophila%2Bline%2Breared%2Blong-term%2Bin%2Ba%2Bdark%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0033288%26rft_id%253Dinfo%253Apmid%252F22432011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0033288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22432011&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.23" data-doi="10.1038/nature10533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fushan</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobanov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome sequencing reveals insights into physiology and longevity of the naked mole rat.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">479</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKim%26rft.auinit1%253DE.%2BB.%26rft.volume%253D479%26rft.issue%253D7372%26rft.spage%253D223%26rft.epage%253D227%26rft.atitle%253DGenome%2Bsequencing%2Breveals%2Binsights%2Binto%2Bphysiology%2Band%2Blongevity%2Bof%2Bthe%2Bnaked%2Bmole%2Brat.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10533%26rft_id%253Dinfo%253Apmid%252F21993625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21993625&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298030800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.24" data-doi="10.1093/bioinformatics/btp373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wylie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">VarScan: variant detection in massively parallel sequencing of individual and pooled samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp373%26rft_id%253Dinfo%253Apmid%252F19542151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTcvMjI4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.25" data-doi="10.1093/bioinformatics/btr589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3435</span>–<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr589%26rft_id%253Dinfo%253Apmid%252F22025480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjQvMzQzNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.26" data-doi="10.1126/science.1172156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamia</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachdeva</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiTacchio</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">AMPK regulates the circadian clock by cryptochrome phosphorylation and degradation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLamia%26rft.auinit1%253DK.%2BA.%26rft.volume%253D326%26rft.issue%253D5951%26rft.spage%253D437%26rft.epage%253D440%26rft.atitle%253DAMPK%2BRegulates%2Bthe%2BCircadian%2BClock%2Bby%2BCryptochrome%2BPhosphorylation%2Band%2BDegradation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172156%26rft_id%253Dinfo%253Apmid%252F19833968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjYvNTk1MS80MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.27" data-doi="10.1038/nature14181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamichhaney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berglund</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almen</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolution of Darwin/’s finches and their beaks revealed by genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14181%26rft_id%253Dinfo%253Apmid%252F25686609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25686609&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.28" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.29" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.30" data-doi="10.1038/nrg2344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide association studies for complex traits: consensus, uncertainty and challenges.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMcCarthy%26rft.auinit1%253DM.%2BI.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D369%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcomplex%2Btraits%253A%2Bconsensus%252C%2Buncertainty%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2344%26rft_id%253Dinfo%253Apmid%252F18398418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18398418&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255057300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.31" data-doi="10.1016/j.ajhg.2008.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McQuillan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leutenegger</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdel-Rahman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pericic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Runs of homozygosity in European populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMcQuillan%26rft.auinit1%253DR.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D359%26rft.epage%253D372%26rft.atitle%253DRuns%2Bof%2Bhomozygosity%2Bin%2BEuropean%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.007%26rft_id%253Dinfo%253Apmid%252F18760389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760389&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-6.2.365.32"><div class="cit-metadata unstructured">Mori, S., 1983 Variations of Drosophila in relation to its environment VIII Change of behavior of Drosophila melanogaster as seen during 581 generations kept successively in total darkness (2). <em>Zoological Magazine</em> 92: 138–148.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Changes of characters of Drosophila melanogaster brought about during the life in constant darkness and considerations on the processes through which theses changes were induced.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoolog.%2BSci.%26rft.volume%253D3%26rft.spage%253D945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.34" data-doi="10.1266/jjg.34.151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanagishima</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Variations of Drosophila in relation to its environment VII Does Drosophila change its characters during dark life? (1).</span> <abbr class="cit-jnl-abbrev">Jpn. J. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJpn.%2BJ.%2BGenet.%26rft.volume%253D34%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fjjg.34.151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/jjg.34.151&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.35" data-doi="10.1016/j.tig.2010.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A golden age for evolutionary genetics? Genomic studies of adaptation in natural populations.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNadeau%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D26%26rft.issue%253D11%26rft.spage%253D484%26rft.epage%253D492%26rft.atitle%253DA%2Bgolden%2Bage%2Bfor%2Bevolutionary%2Bgenetics%253F%2BGenomic%2Bstudies%2Bof%2Badaptation%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.08.004%26rft_id%253Dinfo%253Apmid%252F20851493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20851493&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283678500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.36" data-doi="10.1111/j.1365-294X.2012.05673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flatt</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4931</span>–<span class="cit-lpage">4941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOrozco-Terwengel%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D20%26rft.spage%253D4931%26rft.epage%253D4941%26rft.atitle%253DAdaptation%2Bof%2BDrosophila%2Bto%2Ba%2Bnovel%2Blaboratory%2Benvironment%2Breveals%2Btemporally%2Bheterogeneous%2Btrajectories%2Bof%2Bselected%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05673.x%26rft_id%253Dinfo%253Apmid%252F22726122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309888200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.37" data-doi="10.1038/nrg1838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An integrated view of protein evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPal%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D337%26rft.epage%253D348%26rft.atitle%253DAn%2Bintegrated%2Bview%2Bof%2Bprotein%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1838%26rft_id%253Dinfo%253Apmid%252F16619049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16619049&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236885500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.38" data-doi="10.2307/1536021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1911</span> <span class="cit-article-title">Drosophila ampelophila Loew bred in the dark for sixty-nine generations.</span> <abbr class="cit-jnl-abbrev">Biol. Bull.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBiological%2BBulletin%26rft.stitle%253DBiol.%2BBull.%26rft.aulast%253DPAYNE%26rft.auinit1%253DF.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D297%26rft.epage%253D301%26rft.atitle%253DDROSOPHILA%2BAMPELOPHILA%2BLOEW%2BBRED%2BIN%2BTHE%2BDARK%2BFOR%2BSIXTY-NINE%2BGENERATIONS%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1536021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJiaW9sYnVsbCI7czo1OiJyZXNpZCI7czo4OiIyMS81LzI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.39" data-doi="10.1085/jgp.9.4.513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1926</span> <span class="cit-article-title">A synthetic food medium for the cultivation of Drosophila: preliminary note.</span> <abbr class="cit-jnl-abbrev">J. Gen. Physiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1085%252Fjgp.9.4.513%26rft_id%253Dinfo%253Apmid%252F19872271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamdwIjtzOjU6InJlc2lkIjtzOjc6IjkvNC81MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.40" data-doi="10.1016/j.bbrc.2003.08.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenkilde</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cazzamali</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Søndergaard</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Molecular cloning, functional expression, and gene silencing of two Drosophila receptors for the Drosophila neuropeptide pyrokinin-2.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">309</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DRosenkilde%26rft.auinit1%253DC.%26rft.volume%253D309%26rft.issue%253D2%26rft.spage%253D485%26rft.epage%253D494%26rft.atitle%253DMolecular%2Bcloning%252C%2Bfunctional%2Bexpression%252C%2Band%2Bgene%2Bsilencing%2Bof%2Btwo%2BDrosophila%2Breceptors%2Bfor%2Bthe%2BDrosophila%2Bneuropeptide%2Bpyrokinin-2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2003.08.022%26rft_id%253Dinfo%253Apmid%252F12951076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2003.08.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12951076&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185264300033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.41" data-doi="10.1038/24550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutherford</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Hsp90 as a capacitor for morphological evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">396</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRutherford%26rft.auinit1%253DS.%2BL.%26rft.volume%253D396%26rft.issue%253D6709%26rft.spage%253D336%26rft.epage%253D342%26rft.atitle%253DHsp90%2Bas%2Ba%2Bcapacitor%2Bfor%2Bmorphological%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F24550%26rft_id%253Dinfo%253Apmid%252F9845070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/24550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9845070&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077204000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.42" data-doi="10.1126/science.1142358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyssenko</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burtt</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bakker</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142358%26rft_id%253Dinfo%253Apmid%252F17463246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgyOS8xMzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlotterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing pools of individuals [mdash] mining genome-wide polymorphism data without big funding.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D749%26rft_id%253Dinfo%253Apmid%252F25246196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25246196&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.44" data-doi="10.1126/science.1189406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonson</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">genetic evidence for high-altitude adaptation in Tibet.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189406%26rft_id%253Dinfo%253Apmid%252F20466884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjkvNTk4Ny83MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.45" data-doi="10.1126/science.1212986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Verdugo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The molecular diversity of adaptive convergence.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTenaillon%26rft.auinit1%253DO.%26rft.volume%253D335%26rft.issue%253D6067%26rft.spage%253D457%26rft.epage%253D461%26rft.atitle%253DThe%2BMolecular%2BDiversity%2Bof%2BAdaptive%2BConvergence%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1212986%26rft_id%253Dinfo%253Apmid%252F22282810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2Ny80NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.46" data-doi="10.2307/2405747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waddington</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Genetic assimilation of an acquired character.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D7%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2405747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2405747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1953XW58600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.47" data-doi="10.1534/genetics.114.173575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walkinshaw</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farkas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicholas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of genes that promote or inhibit olfactory memory formation in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173575%26rft_id%253Dinfo%253Apmid%252F25644700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OS80LzExNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.365.48" data-doi="10.1073/pnas.1010643108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udpa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gersten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visk</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bashir</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Experimental selection of hypoxia-tolerant Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">2349</span>–<span class="cit-lpage">2354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010643108%26rft_id%253Dinfo%253Apmid%252F21262834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzYvMjM0OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8zNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/2/365.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/357" title="Effects of DNA Methylation and Chromatin State on Rates of Molecular Evolution in Insects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/377" title="Genetic Mapping of Resistance to Meloidogyne arenaria in Arachis stenosperma: A New Source of Nematode Resistance for Peanut" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/2"><span>Volume 6 Issue 2, February 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/2.cover-source.jpg?itok=7FK5uFFF" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/365?iss=2" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-be5aDKDidZJl9fISSKAAimQQ77PobpVBrVZtkVUIECk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/2/365" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/2/365" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/2/365" class="active" data-icon-position="" data-hide-link-title="0">Dynamics of Dark-Fly Genome Under Environmental Selections</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460074" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-OP_-57V5FONV7ogfFho-9MaQHIc6XKuHg3re7MDTOXE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/365.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/2/365" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-XZEw49sCdCrd0HzLDqDrAzxsVhQ5yRb0JP7NBpTxuog" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/2/365" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/365.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460074" id="node460074--2" data-pisa="ggg;6/2/365" data-pisa-master="ggg;g3.115.023549" data-apath="/ggg/6/2/365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dynamics of Dark-Fly Genome Under Environmental Selections</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Minako Izutsu</span>, <span class="highwire-citation-author" data-delta="1">Atsushi Toyoda</span>, <span class="highwire-citation-author" data-delta="2">Asao Fujiyama</span>, <span class="highwire-citation-author" data-delta="3">Kiyokazu Agata</span> and <span class="highwire-citation-author" data-delta="4">Naoyuki Fuse</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">365-376;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023549 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460074--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Minako Izutsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minako%2BIzutsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Izutsu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinako%2BIzutsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Atsushi Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Asao Fujiyama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Graduate University for Advanced Studies, Mishima 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asao%2BFujiyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujiyama%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsao%2BFujiyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kiyokazu Agata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyokazu%2BAgata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agata%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiyokazu%2BAgata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Naoyuki Fuse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Cell and Developmental Biology, Graduate School of Biostudies, Kyoto University, Kyoto 606-8501, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoyuki%2BFuse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuse%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANaoyuki%2BFuse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nfuse@gcoe.biol.sci.kyoto-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">nfuse@gcoe.biol.sci.kyoto-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/2/365" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/2/365&amp;title=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/2/365&amp;title=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/2/365&amp;title=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/2/365&amp;text=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/2/365&amp;title=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/2/365&amp;t=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/2/365&amp;title=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/2/365&amp;title=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460074/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460074" id="node460074--3" data-pisa="ggg;6/2/365" data-pisa-master="ggg;g3.115.023549" data-apath="/ggg/6/2/365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dynamics of Dark-Fly Genome Under Environmental Selections</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Minako Izutsu</span>, <span class="highwire-citation-author" data-delta="1">Atsushi Toyoda</span>, <span class="highwire-citation-author" data-delta="2">Asao Fujiyama</span>, <span class="highwire-citation-author" data-delta="3">Kiyokazu Agata</span> and <span class="highwire-citation-author" data-delta="4">Naoyuki Fuse</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">365-376;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023549 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460074--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Minako Izutsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minako%2BIzutsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Izutsu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinako%2BIzutsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Atsushi Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Asao Fujiyama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Comparative Genomics Laboratory, National Institute of Genetics, Mishima 411-8540, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics, Graduate University for Advanced Studies, Mishima 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asao%2BFujiyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujiyama%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsao%2BFujiyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kiyokazu Agata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyokazu%2BAgata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agata%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiyokazu%2BAgata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Naoyuki Fuse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto 606-8502, Japan</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Cell and Developmental Biology, Graduate School of Biostudies, Kyoto University, Kyoto 606-8501, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoyuki%2BFuse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuse%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANaoyuki%2BFuse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nfuse@gcoe.biol.sci.kyoto-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">nfuse@gcoe.biol.sci.kyoto-u.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=365&amp;rft.epage=376&amp;rft.atitle=Dynamics%20of%20Dark-Fly%20Genome%20Under%20Environmental%20Selections&amp;rft.volume=6&amp;rft.issue=2&amp;rft.date=2016-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Izutsu%2C+Minako&amp;rft.au=Toyoda%2C+Atsushi&amp;rft.au=Fujiyama%2C+Asao&amp;rft.au=Agata%2C+Kiyokazu&amp;rft.au=Fuse%2C+Naoyuki"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460074/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460074/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460074/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460074/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460074/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460074/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460074/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460074/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460074/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460074/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460074/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460074/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/2/365" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/2/365.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/2/365.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/2/365.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/2/365.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
