<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/2/435" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/2/435.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/2/435.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/2/435.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/2/435.atom" />
<meta name="HW.pisa" content="ggg;6/2/435" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Tying Down Loose Ends in the Chlamydomonas Genome: Functional Significance of Abundant Upstream Open Reading Frames" />
<meta name="DC.Identifier" content="10.1534/g3.115.023119" />
<meta name="DC.Date" content="2016-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Cross. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Chlamydomonas genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. The annotated genome is very rich in open reading frames upstream of the annotated coding sequence (‘uORFs’): almost three quarters of the assigned transcripts have at least one uORF, and frequently more than one. This is problematic with respect to the standard ‘scanning’ model for eukaryotic translation initiation. These uORFs can be grouped into three classes: class 1, initiating in-frame with the coding sequence (CDS) (thus providing a potential in-frame N-terminal extension); class 2, initiating in the 5′ untranslated sequences (5UT) and terminating out-of-frame in the CDS; and class 3, initiating and terminating within the 5UT. Multiple bioinformatics criteria (including analysis of Kozak consensus sequence agreement and BLASTP comparisons to the closely related Volvox genome, and statistical comparison to cds and to random sequence controls) indicate that of ∼4000 class 1 uORFs, approximately half are likely in vivo translation initiation sites. The proposed resulting N-terminal extensions in many cases will sharply alter the predicted biochemical properties of the encoded proteins. These results suggest significant modifications in ∼2000 of the ∼20,000 transcript models with respect to translation initiation and encoded peptides. In contrast, class 2 uORFs may be subject to purifying selection, and the existent ones (surviving selection) are likely inefficiently translated. Class 3 uORFs are found in more than half of transcripts, frequently multiple times per transcript; however, they are remarkably similar to random sequence expectations with respect to size, number, and composition, and therefore may in most cases be selectively neutral." />
<meta name="DC.Contributor" content="Frederick R. Cross" />
<meta name="article:published_time" content="2016-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Tying Down Loose Ends in the Chlamydomonas Genome: Functional Significance of Abundant Upstream Open Reading Frames" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Chlamydomonas&lt;/i&gt; genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. The annotated genome is very rich in open reading frames upstream of the annotated coding sequence (‘uORFs’): almost three quarters of the assigned transcripts have at least one uORF, and frequently more than one. This is problematic with respect to the standard ‘scanning’ model for eukaryotic translation initiation. These uORFs can be grouped into three classes: class 1, initiating in-frame with the coding sequence (CDS) (thus providing a potential in-frame N-terminal extension); class 2, initiating in the 5′ untranslated sequences (5UT) and terminating out-of-frame in the CDS; and class 3, initiating and terminating within the 5UT. Multiple bioinformatics criteria (including analysis of Kozak consensus sequence agreement and BLASTP comparisons to the closely related &lt;i&gt;Volvox&lt;/i&gt; genome, and statistical comparison to cds and to random sequence controls) indicate that of ∼4000 class 1 uORFs, approximately half are likely &lt;i&gt;in vivo&lt;/i&gt; translation initiation sites. The proposed resulting N-terminal extensions in many cases will sharply alter the predicted biochemical properties of the encoded proteins. These results suggest significant modifications in ∼2000 of the ∼20,000 transcript models with respect to translation initiation and encoded peptides. In contrast, class 2 uORFs may be subject to purifying selection, and the existent ones (surviving selection) are likely inefficiently translated. Class 3 uORFs are found in more than half of transcripts, frequently multiple times per transcript; however, they are remarkably similar to random sequence expectations with respect to size, number, and composition, and therefore may in most cases be selectively neutral.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/02/01" />
<meta name="citation_mjid" content="ggg;6/2/435" />
<meta name="citation_id" content="6/2/435" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/2/435" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/2/435.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/2/435.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/2/435.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023119" />
<meta name="citation_pmid" content="26701783" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="435" />
<meta name="citation_lastpage" content="446" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Frederick R. Cross" />
<meta name="citation_author_institution" content="The Rockefeller University, New York, New York 10065" />
<meta name="citation_author_email" content="fcross@mail.rockefeller.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=I. K. Blaby;citation_author=C. E. Blaby-Haas;citation_author=N. Tourasse;citation_author=E. F. Hom;citation_author=D. Lopez;citation_title=The Chlamydomonas genome project: a decade on.;citation_pages=672-680;citation_volume=19;citation_year=2014;citation_issue=10;citation_pmid=24950814" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. E. Crooks;citation_author=G. Hon;citation_author=J. M. Chandonia;citation_author=S. E. Brenner;citation_title=WebLogo: A Sequence Logo Generator;citation_pages=1188-1190;citation_volume=14;citation_year=2004;citation_issue=6;citation_pmid=15173120;citation_doi=10.1101/gr.849004" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. D. Herron;citation_author=J. D. Hackett;citation_author=F. O. Aylward;citation_author=R. E. Michod;citation_title=Triassic origin and early radiation of multicellular volvocine algae.;citation_volume=106;citation_year=2009;citation_issue=9;citation_pmid=19223580;citation_doi=10.1073/pnas.0811205106;citation_georef=2010002746" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=A. G. Hinnebusch;citation_title=Molecular Mechanism of Scanning and Start Codon Selection in Eukaryotes;citation_pages=434-467;citation_volume=75;citation_year=2011;citation_issue=3;citation_pmid=21885680;citation_doi=10.1128/MMBR.00008-11" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Kozak;citation_title=How do eucaryotic ribosomes select initiation regions in messenger RNA?;citation_pages=1109-1123;citation_volume=15;citation_year=1978;citation_issue=4;citation_pmid=215319;citation_doi=10.1016/0092-8674(78)90039-9" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=M. Kozak;citation_title=The scanning model for translation: an update;citation_pages=229-241;citation_volume=108;citation_year=1989;citation_issue=2;citation_pmid=2645293;citation_doi=10.1083/jcb.108.2.229" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. S. Merchant;citation_author=S. E. Prochnik;citation_author=O. Vallon;citation_author=E. H. Harris;citation_author=S. J. Karpowicz;citation_title=The Chlamydomonas Genome Reveals the Evolution of Key Animal and Plant Functions;citation_pages=245-250;citation_volume=318;citation_year=2007;citation_issue=5848;citation_pmid=17932292;citation_doi=10.1126/science.1143609" />
<meta name="citation_reference" content="citation_title=Evidence that an unconventional actin can provide essential F-actin function and that a surveillance system monitors F-actin integrity in chlamydomonas;citation_author=M. Onishi;citation_author=J. R. Pringle;citation_author=F. R. Cross;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. E. Prochnik;citation_author=J. Umen;citation_author=A. M. Nedelcu;citation_author=A. Hallmann;citation_author=S. M. Miller;citation_title=Genomic Analysis of Organismal Complexity in the Multicellular Green Alga Volvox carteri;citation_pages=223-226;citation_volume=329;citation_year=2010;citation_issue=5988;citation_pmid=20616280;citation_doi=10.1126/science.1188800" />
<meta name="citation_reference" content="citation_title=Introduction to Linear Algebra;citation_author=G. Strang;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Zachariae;citation_author=M. Schwab;citation_author=K. Nasmyth;citation_author=W. Seufert;citation_title=Control of Cyclin Ubiquitination by CDK-Regulated Binding of Hct1 to the Anaphase Promoting Complex;citation_pages=1721-1724;citation_volume=282;citation_year=1998;citation_issue=5394;citation_pmid=9831566;citation_doi=10.1126/science.282.5394.1721" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=H. Zur;citation_author=T. Tuller;citation_title=New universal rules of eukaryotic translation initiation fidelity.;citation_pages=e1003136;citation_volume=9;citation_year=2013;citation_issue=7;citation_pmid=23874179;citation_doi=10.1371/journal.pcbi.1003136" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Tying Down Loose Ends in the Chlamydomonas Genome: Functional Significance of Abundant Upstream Open Reading Frames" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="twitter:description" content="The Chlamydomonas genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. The annotated genome is very rich in open reading frames upstream of the annotated coding sequence (‘uORFs’): almost three quarters of the assigned transcripts have at least one uORF, and frequently more than one. This is problematic with respect to the standard ‘scanning’ model for eukaryotic translation initiation. These uORFs can be grouped into three classes: class 1, initiating in-frame with the coding sequence (CDS) (thus providing a potential in-frame N-terminal extension); class 2, initiating in the 5′ untranslated sequences (5UT) and terminating out-of-frame in the CDS; and class 3, initiating and terminating within the 5UT. Multiple bioinformatics criteria (including analysis of Kozak consensus sequence agreement and BLASTP comparisons to the closely related Volvox genome, and statistical comparison to cds and to random sequence controls) indicate that of ∼4000 class 1 uORFs, approximately half are likely in vivo translation initiation sites. The proposed resulting N-terminal extensions in many cases will sharply alter the predicted biochemical properties of the encoded proteins. These results suggest significant modifications in ∼2000 of the ∼20,000 transcript models with respect to translation initiation and encoded peptides. In contrast, class 2 uORFs may be subject to purifying selection, and the existent ones (surviving selection) are likely inefficiently translated. Class 3 uORFs are found in more than half of transcripts, frequently multiple times per transcript; however, they are remarkably similar to random sequence expectations with respect to size, number, and composition, and therefore may in most cases be selectively neutral." />
<meta name="og-title" property="og:title" content="Tying Down Loose Ends in the Chlamydomonas Genome: Functional Significance of Abundant Upstream Open Reading Frames" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/2/435" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Chlamydomonas genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. The annotated genome is very rich in open reading frames upstream of the annotated coding sequence (‘uORFs’): almost three quarters of the assigned transcripts have at least one uORF, and frequently more than one. This is problematic with respect to the standard ‘scanning’ model for eukaryotic translation initiation. These uORFs can be grouped into three classes: class 1, initiating in-frame with the coding sequence (CDS) (thus providing a potential in-frame N-terminal extension); class 2, initiating in the 5′ untranslated sequences (5UT) and terminating out-of-frame in the CDS; and class 3, initiating and terminating within the 5UT. Multiple bioinformatics criteria (including analysis of Kozak consensus sequence agreement and BLASTP comparisons to the closely related Volvox genome, and statistical comparison to cds and to random sequence controls) indicate that of ∼4000 class 1 uORFs, approximately half are likely in vivo translation initiation sites. The proposed resulting N-terminal extensions in many cases will sharply alter the predicted biochemical properties of the encoded proteins. These results suggest significant modifications in ∼2000 of the ∼20,000 transcript models with respect to translation initiation and encoded peptides. In contrast, class 2 uORFs may be subject to purifying selection, and the existent ones (surviving selection) are likely inefficiently translated. Class 3 uORFs are found in more than half of transcripts, frequently multiple times per transcript; however, they are remarkably similar to random sequence expectations with respect to size, number, and composition, and therefore may in most cases be selectively neutral." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
    <title>Tying Down Loose Ends in the Chlamydomonas Genome: Functional Significance of Abundant Upstream Open Reading Frames | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SzvYwyuwNOx3NgMFwbev-iua8gAUYqEPvQKbebjKigU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460068","apath":"\/ggg\/6\/2\/435.atom","pisa":"ggg;6\/2\/435","ac":{"\/ggg\/6\/2\/435.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/2\/435.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/2\/435"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/2\/435"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/2\/435.full":true,"\/content\/6\/2\/435":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221Cn7E6yvSLqzP4quML3jXO8vXg9BN9hHafBQimk\\\\\\\/p7hWl5+zmGRqDiFGdr2ZYNxtppDXWdsap+I7TQLgiWFQQXDKR1y+Z9ZeLSSosczps2G3gkXZyB0cN2IEJA8vjUt2JWiYCRyOjVFaK2CPI\\\\\\\/shprhgYJUeMssPlI61DEmZnYw6rtDCvNl\\\\\\\/9RsK\\\\\\\/uwiWidUbZu05UPoYm8bnX39P19f9iuphFlFEcGON\\\\\\\/C0Fp+TYgoTs3HiW2Xl0XydH\\\\\\\/kVKEbeB91Lgk1Uo9DgIHXrxLE0l\\\\\\\/V0s2hy2Vw2v15H3u+MB3TlMYAeMBA2Y4TO7MFweV8cOTq6uAhL4irogfZOXcwNe2pok90zK+9Tq2RG9F1gGWHoRwibaPgmcDLEOPsy2iBvKt8dmQ6MZiaPKEqpuoN7EPU9CKYKl2u\\\\\\\/SE8HyNx5II\\\\\\\/X7l7nr2CRcTpxQ0oX3VmzGgecM5X0hbsGIxIWUTFlbQNTt6vj1LlKbgrqsljj7fv7MAobymkI9hCeShePJQsgMqwK+8kF3cjrARqSyC0kjqUUbhU3WSxZjeGLaQsPYT9SN5PHG15ayAqtKhjk1IVosjqDQFydW2HB3bN97k\\\\\\\/B9KCDvfNzAJrIgAEfvLpQjDRcvGHGTSvGGSyxWXos\\\\\\\/JNHegivKS0bow1+RH6xcSISnnnuLUAXSW1e\\\\\\\/FRJaYFGcoqtSZwn3Ko6dEB\\\\\\\/F3EZek6yXVwPy20rGYhgD\\\\\\\/drQtV2g+nrnRgggsUOtv98GJdlBw9zR3uw+aWEJo81vMd2zZLNnVWQJtEnj9r1GfNRWU0K5RwdxnqYs7ym9dB2JUEhCZKFCPUHohiLKcqBejdZV1KKVJtbpd2NcEHYETJzc\\\\\\\/ybFWME6xwMzrF+CkOT00\\\\\\\/FJ0O+fZS0uT6ebDLeoKNyCS13IcnhDR5cyBXi73dzwQMZBCxQ5uT+jiWzEtI\\\\\\\/geUy+hEyppe6cXrPusa7RktOHHrt4SnWq89BGrZoVFSUErpTmSGON\\\\\\\/PS4B3SSU\\\\\\\/jCn8t6g8VkmThnFUnufBL3NQ\\\\\\\/rKpSXLrWQhOIelSzwMgVm7\\\\\\\/HrFGsgRXQ1JgI\\\\\\\/yvako2ZKUiJo6oWojZ\\\\\\\/cWi8i1lIpqawERxOqYJt2SaSIp4acD\\\\\\\/MSs3bp8lUzMjyUzRwAPy0tkW\\\\\\\/DqMs3oSrWH0KLWJa11ajnxzM56gonJ1Pk6btyvumvxjJUULXORYp5zQYJ4gtDnC2ChvJBgh4F92mHqXYLG4v1opSJn2\\\\\\\/QHoyDqp\\\\\\\/s\\\\\\\/wSe\\\\\\\/AW9aqXL3orB5FT9e1miJTBpjwDmQdK\\\\\\\/fvQUksYBkz1\\\u0022,\\\u0022iv\\\u0022:\\\u0022DGM7Ds6Wcv7H3rsWkcHetQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022916db392c41a1919bc9ec7cd916042552d2c9bf5e0e017750400635855875a3e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g9ulJtDR\\\\\\\/dIStKnY3c2x8unJeBMqow1JNRMEhQjujDDtoACZzt06gv91YMFMrXUvJOrlWPYY0OO72FRuJXX7tzu0XOvTS68yztkPTtnKt5Q4aG2RTZ0qZHgLLzpJbVpAANFCIPrZhpot1UBvaIPSWTVKL6sZ7vXC7\\\\\\\/1C1l+OYu\\\\\\\/hUN2sSj4zLgsFcqi5oQpH7tGvE7T9rnSEvHHS1lIWcp0iaN6HKxcjeEYVLaDb1uaHAwBOotd1h\\\\\\\/TEsDqzHNe7c5V6gn0JTcDHEqemkPpSPp4rvJXVV3Oy6IDuTVWazQ6+NhLc6V5gOKaib\\\\\\\/zDw3G\\\\\\\/O7saBGnHHk0PxGHjDtHXYRe\\\\\\\/rBqtAHh8f13JrHteeDMKhaegWjm1lPq8MYBmupqeX\\\\\\\/Cx+tLq3+2Mj4kcN136cZYbaRKxfR3wH1swtKGAyYgeQS3CpBWBp55x3\\\\\\\/hFVrr7x20VYqS3RaCSS6QdSZkGJ96X7ok2el6cXDSldiEG1M77MXQxgqYx9WYMfYTDNTzek1\\\\\\\/I7jkO+fyJ9Ok6JqZQ5gwLNLSv86Wv7AByH2GOWRbNWMBPISWQ+eNgZGS6SOPmmR1UYL9MrQOjfRhNgQsm9Y9nNHyF70O5q6u8880BNWE+2Wc\\\\\\\/do3vz5YHswJcxoH7M8Bzl8tliIGZ5Go55PFmGRJvGMBCtto+YqOwbQDYLYHz35+sw+3kCd6Hnvq7dbt2zAz0sv7T0\\\\\\\/w7drQPOYUerYhXdaPPa9hn59C0fdsgRJt1ujuTsoKrrvcIj+9tqKIh8sdgkoEX+pUt1wwl6zomND6lWc3M181lma3MrbiEGPjt8\\\\\\\/eStUriNE2d4SqbyhvGXB6ZAxx2uiRZTdudMhQMVP8dtAT92HYkyC2uYDiEUxV1HCH6AT2d7KkMG1ZqlHOYMkvTFSEUZe8WN4IONmH6KYsmXnkOiDkV1YjAtJLXKZQy4OPthKtUq6KvI9zfhXst9z2MfiLKzRs+UeorWsivKhMRFdgipFCdr7iXGIFna4uMME6IkzCPYHIrpMNKa2blqyuOJVNUfDlxG+hnvoMKu4kUKggO+pIkLtdCILEhAcQNLvFWXMYIzcoG3jy2vQmQrPzy9Equ7YtwMJkYqnn0Nt81xIAChf6YGmg8fMEMxJHCtLbZljoVzX+jfgrfE72RQbegxIDgF7jDS1ANT9X+idDwjtI4+U+6MNidgYP3LurMRgxmn6t+KMMAIaSH0SIvRe0F\\\\\\\/4MJLOSGDscpHMLf8yZsmlFciiG7CR1ogDtAy2\\\\\\\/0gkMPbCz\\\\\\\/X5Zpx6RSY5uFipPmShXIgguvkgmSvWgNvpOIBYG0N+0H7NPN4iZtQ\\\\\\\/HK0LuPguNYKM7V+n7M9XsoeGNXVf+\\\\\\\/Ps33k8LVVFW0JtVooh6wuyvAkn+yKEya1DUIebPhI4R3UxNBI25WbhfudN0FuTE5afgwvU\\\\\\\/LPUkJrpXDlKG4LSbFJlk0ASnDQUeVSyxtB355Gx0LexZQoIEZSXJkQBga4+vGWXDlstze1Cci4BsJdsWyYKNCVV4209q9IpVOgTMvaPAllzyisvetYFdXBUzTMNS\\\\\\\/ZtIikRSSXz56t6cdL80LU\\\\\\\/UTsuRCZCeNa9WnmeuqyLiG8QFhSULXlOeVRH9Plmtu6Rl3\\\\\\\/BacH1vdkjsHWHpmnIuulN2vY0KiTeyZZfb9u0k2GL+3hPAGL5258Pe\\\\\\\/9Ig8q9LQVgYj0vDhD1yMSUYy5d14EZGnSp9S2NVzVAYf5ia+8lyi52osR07zup18ga6cakuIhR2P8HlmWB8J03N6nVXO4FoxSaRE\\\\\\\/kkudV87gBUGg4PYEZeXx3zb2QaK2XmwvpL4mMEgWSwfwwer+4lfDLteyzY+duqBv1ulMNAVsYi+0Vg4eC3NEsS4GY+RBMpwOXksNuRO3D+e7a1sM\\\\\\\/jcRjyIoKCfi97ImfGecZxi1gzh3y8KPDaoQqualEPYulDM7p+t\\\\\\\/x6bu6A4KOxiw81VqW6DDCBrbOkIKUAKMXeJpyTMct9ioQVS+Ax0Zsr0kEiGyWt3geys1lReX1vurN6YpYI17FtrfvRgDtAAo9wtVohZfLFC+mwNV3fDldfojozq\\\\\\\/bZhJNizaFbj+UTmf3WQWGnyKHdmLdraVSBeRrfH8uU3MEI12qpnJe2t7X3799qvXsG6Fm5HCzbBE3ZO9FGXcnD7D9Ted\\\\\\\/yAl1ZLzhpNDyd\\\\\\\/4rR97cAbTAhwh7CIVwmjmM6oNEhDGrQo2p9L6zd3pg3NqXNhcSMW7SkyG1eaZLCr4GNrdSyd9nSbUjAx6EUmpt8zXlYxd7tLAVy50PtonHqf\\\\\\\/IYuypeOsRzne2SgK6NSgyscmiYAA\\\\\\\/wlboQDyTT\\\\\\\/CmxvGatCzsj9ke3uj6IF3yX5tGF5AVPS43jjIayVtBjg72DTeT5lpYfLW+kz9lq+H35CgKRor17IfW1WK9NnUItv5GXDiJvwzJOZWhiokycc0o51rRvxRcMesDmgzf1U8STK7ppdvKy3BN2sKy\\\\\\\/AW\\\\\\\/JI0PBx2uV6atdmeUhQ0sXt04y9SJVNvdpjp+D23BQlYW47KIgmu23Ml3lOEd7Qz6hfScIrUtVZ86brxSQRIVhQN8QqZyeoMzZiXGKrMAJ8Vu9BnI+YBvQ6AwbDT5W8IKjHCMkGM5e4eO6r7zu05d8fMfHSkkUlbtyXNA+KFcSRyNCqEpyGcrpjSPX7\\\\\\\/dWIZpKWoE57CAfbSAH6oFTMhdaVrsiExB3A9s1Tyio\\\\\\\/ct+v9JzCfohOxKzDPqnA2S4xIHZunQpDN9gtj1Rzncu0jFYQ1ST75xVM5Bl0kLd365ZieZqrLNMHYuJ9dLF3ok9mMnPyrjIyTsBG00xjbgkb4Xe7Mt\\\\\\\/R0gJnnldhRZ9NzKUQFV+R9KxwJp+R95s\\\\\\\/RGsdJZJbjwMqZQCThkUZDNlxwMV3Rv9JwNChjDiLFewJaEqR7FHJ5rcqg1ORziNykNqC7ymMEUs4D9MQVGx2AkF9RnFUM3fhWc7RdlE\\\\\\\/Zp4ZvPuoTQotrMyuMZ3kw5HyD5TTEwSNjk98jEmZYgxb\\\\\\\/8DoI2Y8QQCzIn3VZzQEL528hzN7HP2uohtZRdomuaHVVEFUfbKKauaQXpgt1tuRlv\\\\\\\/WawFo\\\\\\\/NY86CfWSH+qylXw1W0wTSnjcupkvw\\\\\\\/1blwJSV\\\\\\\/LSOB7PajBywjBco68JbHO3Mwm2Npi7bHO5cgocZX6ipkX0\\\\\\\/NMerIDfFoQWYrr8g4ztbkcC3lrOM0J\\\\\\\/rdmolnF0vChvmrIxKNzUk9CVcf8SKpyq13eDfc9Sx2XmD7lFvyukQPaqI\\\\\\\/w7ncuW0P4wsy9+vEF+O6wkW+5icW8vmkKIyGxC7L\\\\\\\/KmPgfjrJoJ1lduRxkNDcEKCAPoOWVxxenwB8FzHaBE\\\\\\\/lWp3Aot2lHh6g4FuOcm4jKpBcBhc+ztgc95JI5BmFBM5PWN66WTey0VyP+55e7mOFD8KMyLvo3t9\\\\\\\/UiYuYT2WW1bXF8iLkZg9zDfFL15YUEkZ1I3MOHwQsW2qsZl7FkbDzhKBn95pORuvnZX+lrnI4FHHQN24kPFmfDGLosFemzcZMqm\\\\\\\/07xOXFzvtYlS2G2F90UU2RfKAzXs4+iEfyzMXnwCy3kEbVP4J2vksp6GXz8V97+tXIJrOtSkNpKpTVProEsFrJLnLBti7+qboRzVph8QXPNvYy\\\\\\\/VYoGqNQiCVa0lZy7mzupxhR+EroyRkhlNfpy27PTmyhtAZMBoxi3U1h0Jw1cqtuKPnD+GyorwcCJhXSx3g73py5AHv2LUWKu+5Z2tb7dUkE+ctx9Cb64qOyJfGuuLxN02kKYztKdBx\\\\\\\/L7tQoiH17S9sz0AYU8A2r+Xi8a2ThILMncyBDMd9C7H6WJb4NkB3ujAe56PVvw3bBMnA7TDKfgVkvYN26GCm+28aY\\\\\\\/8\\\\\\\/EDGak0XxAaxHBejVSDw83eZnwYi7gP0GHeLmMb7ugmZorn3VOmoINruTVUqcSG6\\\\\\\/1Zk7Se69Nqgh8MZcD5lkbWdNI8y6WHXZ+MrfMWe1wOVcwYRtaIQoFD4efPh0XrOuxlezNZb\\\\\\\/AyVL+9Vdah+Fyl2K0+rs88EPCddlQMmM6F6gQb\\\\\\\/c1Y1wnyMdw+Va+nGU4BY6eX1WCriN4Nk49u0DnWN1VZAXWevUtsuei3WSKaNFzyoT8nfVN6hNQoAkpQWTvcBylJLqMS7I0f5ro2LNgJqtPAaSwcO1dzrwfarPszC81ijsOMAWyH0o4+fkTUzDmnl7bRqhuj9LISNIvN3\\\\\\\/jEtWpYp39BlEArR6cr5XdGEeba\\\\\\\/xiuYNPHvxs8wraTevaYA0XKwRXNiFUit8\\\\\\\/5ZjBFLJGjNNzTC3SdjA8nm6LYbwogWrtvqGMKgDmSruX0NFI3M3gzFa7f2v0qlttsWcD\\\\\\\/c3XVjUKzg9k63+f+cDysxyFMzf9Av0kKgTcwSgD5d2+VkIb+xSUHBPmY+Zx\\\\\\\/\\\\\\\/4xpE6gymlBg43VTvaXh3jbZlvUzO\\\\\\\/jbyQlK7fdmA3WNvErIjHh5ZZA8otBc4SLsiNnMj7uhiALUEqBRnjdEl1LOm5j1HBAnkLcAB\\\\\\\/1AwHzFnLLwfQdnTm+w8hYkpHG+fQf+\\\\\\\/O1VXQSL57FzE+zZd1MOMdyfl7D5pOutJVI+BcLeCdNyJkFKUz8xEYvKpWkmdttaK1nTcen\\\\\\\/hCw6ZKnvaIw+pSlBlmvnA\\\\\\\/OmNVVHnI3yvQ+c06aaYjZqd4MVDvLi1mwDmteL6UhaW9je9AaIqqsq8FesSxzOgtpt7ooxus5VIpb2j5o\\\\\\\/zVV7wZ9+I56gsKJjWtddRVTtqTUEq9PCN4lxAUixx0h+xqeb7MfzoDjddBC+ND6oltK8eD+4ltjRCwku2or9vo6b8zVmGZnspdee65NEzAPNse7lao5XgC1hCXpJ5rFWCEJQyIw0w6EoUgSSXojvMIl1rXAjI7hnlbqeKzLUpdt2VKGHTKOdF6XTxPwlhTF+oDXb6k9tCRCfcMVeNWdDi\\\\\\\/pdacftFIHxliOGpScnhy5l0hIOB2NGCHS4b58sKH0T0IUguGYFwL2CWCSnMjs2bFT58ybx7oQJEVJGZJREFSP6lnYn7qbobcEowFx\\\\\\\/w2HpDNtE1UVrMHjVErLMMTulSCfF1dWNOF6ymWK74naXF6jWkQ2lGeJ9nzOCW6s6rSa0MwpcXnP4LqD7de7WCAV1qG0VNdWXNN1j7l0pUMdgzoBBlBCbAvZ9nmNazgukOl3hr6YDsFtQGjDx\\\\\\\/xrsBHKA+BVbXlDrhrSo3tn\\\\\\\/VJgQg0mVSTSTOFOrBcZCfTIX\\\\\\\/XrpB8Ro1yTKG4ClKI+PD1MFa6OSu2HxWGvIuIsDY22hhJrOadu9FbAXu6zohRcJmdG42f3laEqSeiGRkFOtIQva4ttpbGpo3qvPxwjJVtiLXH4+s6WrYGyURv7O637PIFuTKClk2T3nK81HPya1hcYxN1E3JeEviKq76+mhjaABSRWWGhIPHshL8nU\\\\\\\/vUUjdkPuW5kAikshSOJRgbPPRRDAFBXsDJfYusf06uobHlY3MELoov8gEbFzlAtdEzZECW0QFQq5euiBXu7b9Rq+rYYGxRY55n7kG0FlD6uNCh8hpPss1K\\\\\\\/Spn7bdyZyp\\\\\\\/y6sDab4T2buAWL+kgII0vBzICBs\\\\\\\/1JcxWNmhfPWr5rPmtYaOW6ojolHRJIkcpHfrprUaOD7Y2vF\\\\\\\/UoqQnuMhdPzdjW\\\\\\\/z+So6KCkbjG8UrL2GEQT+i+xor+moqHCkrf+QCLMRxD45tkqK+PcaJq8rs1vNHKMRO7IT7dsbtVSP+AcX7XydT1zM9iKEzafwbxeYYmUj2gVVFTJojKL50I+SpSlHG2SNuA9zXK+6jt6irJvRQawarYT8R25T0F5JLfzg8b22fpOs63ubJVDpOvA9lGXUkz0\\\\\\\/262Ot6wtzeljG\\\\\\\/gEG38UrJSL1QC2FG4aa7WPnZHoMSG1MaqJrHjFTOySKUwgvexWiVhs1M86eLte86d36TEKj47WyV+Wn4uHnUYicLpuHuoahvfCkL+mW7D7wpQojbnZw\\\\\\\/Dz6to1Q+QxLygU7iHE9J3XYzwPfKZt2q\\\\\\\/IxnHLvu45JmdIvLQqo90C+pBTwwFlKnKsdDYzTQtVFGKwe0D27WKWDb74Scn8avTPqxARGLKOk+rwUG0JshB2TTItRRyRjhCBkBCQ3nZtGOJlxZKEXU0tuMceAPNEdM79vaY0gPqXll\\\\\\\/SQPu8wHsMNxiuDC0ObKKunQaFXgmsTXB45eq9wLQ25rsQ0jn81bL5mURA\\\\\\\/ikEMSr5jGZG6PsXu0ec9FDV5MWk2cgs14oLUxgm4j7ItvokjS\\\\\\\/ZlZ8SnqvAoafONJ+bRRqDNz4oTNrPel7nzWE+IAss8M\\\\\\\/1zLlh6tHtV317XsikWZc9efVOnN4H\\\\\\\/GVAW7aAm3D7E9owHmv22bNrFH9a2O+A9rtetGgB7SHAYBDs0Z7VLydf9Y9ONsgDnqkS3+1wYxt0BsJWxFfa6HcI6qIFjUu2QTcC7lGWaGb\\\\\\\/TTUNqyrWDyzw8dMvU8Iqh4fUBF7o1L+UneM2uZRYJYTwrpDtUI4MMXBh2O\\\\\\\/CTE1FIjaw7dJczi+2EdmDVJXglR6QK0K+fzB0mu6G92YYLD1l8WqA2r0x8O5vNQIqP6oxFkJnEktYfJic0JAUhm7DKNxnXYytrmfS6fzLGwKeYujdR6UjA\\\\\\\/3NLnFDR4FSvaJOk9MyE9YLBQrYHdTwEQjgDjmEPL7Wow+IZWBSYpfeUBsve5BgJRx1ELyzvx3ukRhibANdfAU4TIPEIK80ufMvB\\\\\\\/3d\\\\\\\/EPW6yyOW1D9JegYI3nJx5dW7MEwkS6JxppW9i8+PnEjgz8bDrCPsnxO6yyNsm41xg2\\\\\\\/2OZsXA0XVFTPF4JU1izLQ6hC4KCcdEIy1Z3hGo4o132V1kVqOQ3+EkRsQRflpjfr3PzgcKrBuxQQ+L8yd1CJvslQ0JViIc8NZJfqNm0+J3qWRGm4luIxXUcfk4l2TWqOoTRpq3cBXfxEjdP\\\\\\\/AJB1Cg1wvha7xBM22cGD2GWW+HYtqDI1ZScusyer+r3ulbch6ljG8tXqsaBcbV9CAZ\\\\\\\/mpX9gmnvAVI73j2ILJZUCTxzlmiSTMvfX6zNBlMIK3CQC0borbx4Pk2LpKowhyBbABX5nxufsDgsA9Dvi0wrS0bonb56nGy+24+MWdG9Js8tlxLFPcQcsslxSBtqEZh7JeZkJp13WWyTQZ69IOQ7o6WdVGBXYK07rAqBPWLP0MQwicviNbF6772xfO7nRZSOMx7dR6tyvqsxwq7C\\\\\\\/XIfO265l7E6ShGA2ttMjZqcrFTfTN+2kSMcNfjZDWVjp11IDQs7tXO30dJWvwU2HRVi\\\\\\\/69pFegeIC11jKBPkB5IlkXkWliiQ2tXNPf\\\\\\\/u0zKFV3Jvj9qJ9gWFu1fTsXGNAT\\\\\\\/I9g9R68fyhPdFNQxpwpAyK82eJMJqBZ\\\\\\\/5OjrtJfxQOY7yTdRhl7LfC\\\\\\\/GIwu4PYpPtO0QUE58ntMK7eiJBf14gCKxD0N1pKP+tIhxUAIY\\\\\\\/Y3CIii3ywL6L3YNe8pirYsS4Kdyfh29iW5mk2YGFMz05esezsVRJi7RfTgP5Ee5owo7TXw0WvDeMUWjuZ9LOrqp5Zi563Xz4MdTDxVHiVQaA6WccD44s5JamyW8\\\\\\\/n5FfkeQe8yh589iOZx3T8exWVeXfmD3gc96grX0kDBZG9ZRRlQ6QOD5JCmSE+C36HQ1Uq\\\\\\\/BBM23iypLt4VsB5R+vS9HM74GBiUkrcsZkZjIC0YGo5fCywISN1Stla6n\\\\\\\/9wd5RieePT0LL4aNDcqr6kH8logXFDLVSWS9Nmu2VnAXEbJAyFoPfY2rZ4V88ZV0r0\\\\\\\/zUkc\\\\\\\/RyjOw7DlBCIP3GhUyuu3Dbdd1jhdkAP2ABrp20xZRGnkYilkS7Eod1v1qXltyn\\\\\\\/2NcCM6wAaNBPgUYfNRsXMPgWeiqNU2lRingeOjTs04I+UBGji1zIiGvqcf8BSaWxKwFZYEAy3R8VfATG\\\\\\\/xTqX6SWRhq+bl66\\\\\\\/r\\\\\\\/8DwropJjYojZwF3vpkUS6GmxU1jRupg8iAlTqZLWuy4de2KTcKfmVAwLWztP17CM1UhNraCfBC7oQWq1VPWgKAJ3hpUJHZsWqcG9Hli7c8dNifwfR96gFnVaBb+Q\\\\\\\/TebaTOXJ2LpUp2cVBt4d8JiN\\\\\\\/gQubQWW\\\\\\\/nLUvaRmuMB3Oe1xMqG1dbny9kNRVU6jjSWyWMdVzwH9CF0rZKXq\\\\\\\/0UapgRaaoJ41Tt3yI2IhzFgHFcCIP4EWGA8\\\\\\\/0+1Q+9+2Jit\\\\\\\/JzibN+FA2h5kg8mk8n1rCqZzrEm2Tsg3g+5Qc2iUilV9wpfnb4XQ6\\\\\\\/CzRdVPEkxFPMVU0ZRRAgzh4Xs4+mBMPVniruiKyi+x2PEXXwuE+4X48XTCUfLKt57eh0W7XNjmRsR55tkhYPtM6OHRkk845KZysCkudDT8jbdunEElBJpu6Je\\\\\\\/kJJ2avjEgbNGeOCrqLCefMVCaiB8vcf317i6dbI6KHby+AT\\\\\\\/X0dV9XvidgjiukQRJ7OqV6GljSJWCLSIhMCuaWfzYmt9pjFo+v+uzc83G4uM4tZBQGCJpFm01i0DMA+LaBNFNwUf2+OrT1REpwUH8fUjsprStCRE0Mtzm5vObDmY5Ev2UQx0jRpi1Cf0nAKoHpRaA5zRVluRffq8DPLUGVtQrIl\\\\\\\/vgBvX+D17SwddpCx8HvEZomkDBS27K0siM4+IVebSEROV2Poj4hrvADM\\\\\\\/6OxxqwRcE8iB67a86Cpjjm4xyUlOGKK7l71unHa9TFvfNM+C1KVVH9bwZhdJjITEQMLVXsKHQh44n44fKNp2F0jFJRbEeUFcKVnIL66v5AvfH6BRXv816hqGvojmzjiTl\\\\\\\/XpgQeGwKLCWCJXtDjqiNM3Morkk8Y+X0fEkmwNTjHNmy7jlVygNggerYp6xEkmwDlOOIEw9zOFdSCRhWlMhKnnwJEoLWYfAUEvLd2zh3x2tSDFATOmLFejcbCbZIqUZ70oDBU7DslsLmP5ANq59mED3C16WXwKwsQH9DsWk\\\\\\\/kcEK3eHzCcyNK2nQANFKBMh\\\\\\\/2A8jx4rsNknLXnKm0S6cSRDAXyGteE8YAqbJlaQ4LtnsgSOmqv3O13zcnPXpDhWvQvaFLgYKelOb8PTxwS2fESPQMnGkbRNUKNwohnwH2A+OUcmfbaEh8yn8kFnaV1RFTpKKG+cwUB\\\\\\\/NUVCn528RxueTOlpaV7wjta9GNfLUnJ3t6Q\\\\\\\/KfIe7yLEpBUXZjqmwJf5rsserUF3mQH39Cp\\\\\\\/qYrIAt3pv4qId8pBx3IWL8xHH3brvjsvzJcYn5zmqfYb5+kmHR3CvgGiig\\\\\\\/OZHfSrqBbSxyIvdLe2GK1sWJRYXTxCSv62I94i7NMoi\\\\\\\/xcSbaB9cZblx2mW+bca7fNTpeWJ7M0wG2\\\\\\\/4p2fEeeL2UAIA+aGJFEcFVeTsZjXOVDu7dcM8Xv\\\\\\\/jObUXixl+WvkBXuA+UVBhRD73wliFZE9Ha7D0OPlXMBY9GL6bfkwo6Vpn68J6apO1oZmehq+nae5cREyYWTmamStQ8HcmR49kDvp9CrZlxTlX1sPB9w1\\\\\\\/FtEsWW6XKEvMdLGllfEvFtcwj+9ignYKGoj3pxAcFMwRFdDsKziOzMj1ewJMgsjm+sEGguWkOR7Dhe1FFLWEpX\\\\\\\/SFHI36NIUmExYfNOlRebKvOkmaV9BlqkdhwKsG+2UaoII1Rw95n9iwB0UHRb0FS2PHL7D+grZOKAf7v7PhhNYq9oDKPWO4Lt8iwdPch57tzqbiY9tpFYW0k2Uc43QZ2LBm9e+El+v92QnUhaSbsFaKlnFKO7+yjObYFocQy4AxerVfe9KHS9eEWcCpM3bNvA9QIlWCet9M+wyRAy38izBnMYy+ChcI2n4bAQ5LIF4u+vIB+DkZr6+q3CVAAG9tuMA50rzEfoTcgByWdfCLhN5X0RcTRW6oFY1m8DUmvwhoBVy5JG50T\\\\\\\/PcIbCUs7Kiuw7\\\\\\\/VA4vLic4Ga3TbXR6bD5sV95DN3n6UxhzejkoQCRDy2g4xQUvsi77eOhZq9T\\\\\\\/\\\\\\\/yL\\\\\\\/rtVnZGE+lvwayjw01VDjGYheoMZxq4trsnYXaytqXJDbR6TTo+B3PEHde\\\\\\\/PtdtK2\\\\\\\/QrcXM0slA53mSUYy5i4mWgLuwCUhybYc9+8KVuBJ0UQYIrejEedtfV7Ju5SACQV1NlLNx6y0LW05TfHo1ygMzanJtPXmb1E+NKOgArLZmzMDw4mDBhxZoN2Ek7IeAkI0uZTaROu6uLfzeaeTNHhGxxMolcrZ8nsvSWWnxjitbgrbPPT5Ub1GBlcuSv893GrYSfvpblebzwO+s1zwEO4FyvYF9HCGR1BX66J4TEGVl\\\\\\\/Tt7\\\\\\\/h1cV2F6uFK6SSO23uq10F07ebtr4eawcIL6e\\\\\\\/QaLfjjC+LcbaYLjNXRo+7zCc\\\\\\\/0uwG2zFNTujNX6nzkEnhJWzbgLM8G\\\\\\\/wcvioWiGRA4tldjZCD5eW7H+KcFunMcLJH0Dti\\\\\\\/5nNBn8RYzjGRD6WUKkhRJS\\\\\\\/gthpwmlzIDnPvlTmE3MvicAWHWb\\\\\\\/\\\\\\\/WYcCkqE+R2nJEUlrHyZpbCuxjvYvLry6a+t\\\\\\\/jG0JiHLYDg+QvhYDIO0ZbPqYUlazGFY5EIJM3E1EG6wRr2zhQTi7\\\\\\\/9+ZTnmsVnAQaOGry4+054PkF91hkZ6ckoH1AmYujWyf78w6yx\\\\\\\/gH+rSgOC9funQ+pQi44fcBwHzFpBRyVt2CtB4ijByhU\\\\\\\/D3FDzQmEBtc53w33NHk6kF7PFZIjP7iowc9qv0PA1oiAEwtYWQJbJIPjfW2pDvGHa0W0Cy+lbXGm9tfA+4zV3yZoWGoALvR9IQgtA1+4gVkIXe35rqwU1\\\\\\\/IG+gaVs6w+u1uFqj5wZs16\\\\\\\/QKlQ1ZIpkb8A1DZsx1OQcU\\\\\\\/CmHTrGw6epCobrXRJYjP9BxkqN+RiO+xgKwohYCllwPUVfb2HIEu5PKcJnRG23WyYi4ycaXQtxwZj2\\\\\\\/JN6HbXPWB3pOzZOmi9GmQAuwMX\\\\\\\/50Bqd2wuW2a+dbhFzvYsQ6u8EJoaFTXgwFTdawwM04aRgFjxTklupkt9XaSbvfWZcRoiJn+ZBnzYZOIf3OxB2abVL75NI1tuEfputS8gerJkxIxHkOkYJr3mvV2\\\\\\\/cMRgV4wTSIVKbGU3nDY0sNCdwgdT+JlvElxkq3v6qfDIa6aLLfQHSp8zD1cwGlgP9wtdjHQaNHy7xSNtXUHupGezypsWruQsbjMjiYjtW5y3j27eUTsH3GPSWYwPGmxMmEbhut8XwSGL4D76hDUQwR\\\\\\\/Ori\\\\\\\/FdNv7HytE4A20VvQDGwwO7MNj9kIFofbQhpOKZ3EuHE5kMyiC\\\\\\\/IKY5PstXCltjvZ6iSrC1m6VHlZcWNmoLqxrcwZaUHZBrfiqQZAhXhhhr\\\\\\\/0q4DangOxRSrOVo5BET2x1UdMdaHW2ZbCaeSjREkzQ4uv7N19VQ12v+urihQxsd6jChYWDas6+OswtEzAdcdy62TPU10m3JfmrsN8uwUdeNarllQKSNc+0nnjgcQ0163MzfF\\\\\\\/4vQxWkPuXNACYkKdIWwrp8H1uqtF\\\\\\\/pUINmz42MzIYJICLBPQUVW5rJIyo1c5+8uEHtwZSyR2OwNDz5qSTG2HyUDRKClrQkMNwhJA4Q7sOdIDhyGirSf\\\\\\\/zjbN2ytsrL+eh2XiT+PVHj5ydHC2l7zM27v+6DdhH0r4D3zgtDtjWV7Vchhwu4lWcnx38IW8b\\\\\\\/mVRrcrSrXLqnH9Eit2noSJ+JBYuioy52kmaqsZDM2PwOBooBMFbg\\\\\\\/2IzP6hVEDuHj5iDargF0BGknENEQ2VdpG4gfMskUAjevPSmfktrsNG1sdSFcDgi\\\\\\\/gUnCJxClh+ZnOiKERMSUsOPZVRItmUQEqnq\\\\\\\/+\\\\\\\/bAT0+HLyibI1BtX47UokTsN\\\\\\\/mP0mkfE+II7lTQbqurxX+HMMuR9NHdUka000\\\\\\\/iKudWnkp9dDajETzaZCjpFK0Lh9EPy8rps2K+R9U07SN08BoLL+mgBJwvVzZS9VDY1u99O+2\\\\\\\/fNJEkparVmpE5sdIVe6td8N4V+k5qp1KH\\\\\\\/saRjtXGw1csR2hDXTsD7\\\\\\\/wcJ6MQiXCs2OZ54SrO47AsPBysrfDPFmfL60cLbTpCizWli+DAuJsL4J7GH7uGEClbpql4kHDwzJNxFSyZIJsa908WKPoUI1Ln1QvyuX4vwZ5duY52zjJ1z7PbZTvDDIetTjvt4AaT9LM5Uf\\\\\\\/0MU55g+UuAqxxbnbIT7XYfLfGpxwOxX\\\\\\\/6m3g\\\\\\\/6MnHg4eavssohhYCm2vt7gajvYlQGEwzlkzVN5dgBCiTcWfed+RY4ZPsbauiyJOUuOvMlEVd3jBiGwAJlkrOhu0JE9VHJrTm7OxiCAONMq\\\\\\\/yvr6ez0PhKlm4Y1NJYpOree7n6RlDxUFWjizW\\\\\\\/TN1Px8ylWomL1AgwrborjRmiojO0HUntQPDwUv2CAatxKLFebaVmeUOWIvR5cLB\\\\\\\/7xrrn2GQrTOOLpnNu\\\\\\\/\\\\\\\/drOXckq9nBvZa3ZIutd\\\\\\\/LDekbJoa4ai8QgwMUe934bAQzTnhbEq3d0nTzybmQBsLGQcoZDlo\\\\\\\/\\\\\\\/5hIMHjMM3cdDgtm6H6rlEeg0An35skXVHsNF+j6Jjovyr90D9b43SeoCrmltv\\\\\\\/+cEP5e57TIaAfwegSI2QKwiEw5v9rOtpqkJOZuTNi3EZ4Tax4s2rNxh5dgEm5QazYaSFU2XehtYXx1tvVNUThS88Ka0MUiDeVFl\\\\\\\/lLkS1roUh6ZqtHvdKWgQ3aCQSlX1C0JZoSwqYJTh\\\\\\\/Og5fPJ1nFj0E3zj2i1PEiEGPNp0p70BlM3wzfzluTAzkbb\\\\\\\/c38HAIhveM6NHkiJ7z80DMMUNDZ8BUAM8OPrG1kJiemzHaY9uY67hn7dAgL8sjR8uV7wO9LmOJux0GAPu9EQbF0LW5lLbEXA3jbSTVEAjSnp6j92HSDTaQYAXChyUk0qvhbRq6JkVnvbg+eyh0BRVRMQsWvNPTVGSOHLXyvK6ZN1ag0HfKlTdh6ELJ6o2UH0rvP7vS+qZZpJC7yoqFP6ta04uIkVaxMj14gg64W54FLiNUoaJiUyh+Ij2K7ZVg7zfVnlnLT0Qqigd2ZIFrc1dVFQTqrwP+ij\\\\\\\/Cr+23Kxj4szrmYduTF+3lCKoKtk1WdV+nEZzMCuKQSgDu\\\\\\\/wC37to4zaK5CCTAAcpfeng2eB5L76WFJq9N3NIzKN+7LSK8RCzMNwcd1iIr4KuYY1vsl7ZZwH4Rnfkur+jtUzYGARY1R+AjorQydXc8wukIKSO0TSD9Kx55vcJvjIqhFcFTTS6UDzJcHFpr8C4Kti6BXGhbgqj16PEd+gTKZmVypiF+xkk76r9\\\\\\\/eFP1dhuXjByRNmu4aFRvU8U9njn56a\\\\\\\/2SrdS0Ft5SyrP9CQqHEXM27ePfJ8MVkSwoCwC4vXtN\\\\\\\/nKOSViKEM+dQ5\\\\\\\/Yt5RZpV\\\\\\\/JWebgqwXD9gXj91J9mq9HH6BSooJmzGDGf9Mn9JONrRXRxXptvr\\\\\\\/+dTHsfPBzdQuTR5U1mGiSbJSFuDPn7Cd\\\\\\\/kIdNel6v89nh8kVhzrYEoC8EbAqQHKvOQwzjou1jPTC08OFZ0zAZIZS4ujRL00QnBFhEpDVtwzy0VRhF6tTvWRjm49aR2BhStYwHunsDNYTjDAzrWu5UP\\\\\\\/2BGrhpFXMl8MMYbZpf7pCyyNJ7CYgqtxlcvT2KWvzh7\\\\\\\/ZXDgLs+ZAkcORcnTxPW08496S1xpUXF5Ay0ME+E0x59LPlAjW9NgI6isGLZmrSdqpSZAnZ6p9zOZny48K6oyonLVMai3sjPSKIAo3DY\\\\\\\/TGkPT2L6uQnucWsI\\\\\\\/TQyMT7UuzgJ1S8qIDGyH4r8tOMxOLigTlISItbKM50jDojkkxKDJ52OEq4mBDTE+7S1gQ\\\\\\\/4ecioCH9E1WGxbQ85sDhchjvTRP4NVw3d\\\\\\\/O6Rz30xAUhzEq9LR+oQz4NGtIoCOh5wJ3Horxu0W\\\\\\\/wDYdmffwjSrhBN9RzG6FxHr2xZH7ok+zq9dbVX3AxM\\\\\\\/GVEO6Ji+69DENhy7HbyjN5DBNjbDFu7ehwJKMhSDsAeHIj69QYjNWnrn36cNPuKkEXT0Ul2J59Twl92ZJ1zs587EcEkrxPSFaZeUWebxh1zMGqdo4+OOK9NKmyqtPa+CVhJcAn\\\\\\\/HdZ+pe9B8e9LuU\\\\\\\/tfKuHZGsAll6731Ffo52v2hXuOwBSyFjCF8Drwz9HXBFRj0WsOSh3z6ITwJLOI8XAD445yBJUqD5fPtAsdwqnN\\\\\\\/v7CDM9Nthm9XVr+S5NHOfHW\\\\\\\/mgUUvauq3NQd+GmC3K4sOKJKBRGIDaHR\\\\\\\/4vmEzw89a\\\\\\\/I8UHZsxlDK+lKD7nCoZmxfYuP7Ww5bnj+NXBLKIWQdxNkot7eIfPgsf3JZPJHz9dz45Fzh66PYOW6p9Qgv66SZWc0dj5vs0kbZVqvoxO0vIsguH6dxJCwS5kutMWFPRxtK2r4raEmXJGd\\\\\\\/4QWcnQVjP69eXfxSaucAh5y9oTph0vPjzR96xYORgU89wdgZ+CMlKoCAJ7GktFH6p59oUQFdLajEyY6yraQW2GIJJT0Y0aYqck9l6E\\\\\\\/91iPL0di7PONCZ8HSFuGW7giXmspR8NcQEFqiixMocHdWV5IjFpB19Z6QMlbx16WJWHpTjRhMSjiTaATgk8+w6QpY21m92LdD2difrw1ghn1BclAKU8NmUfFAtm40+FnMSc1QXxrC4Un9j15cTeBxdWzAo8U3qF0qZqhK341\\\\\\\/h4LN7XQ2uTj3vlsFllO05dj6ArnPLWlk9EEmV3y9JHCh8Zl1DpfDv8HDMk7jizubz19dIUh8YYrP0XDlC71b+1I79aF3eUORbKDZd4p3KHjlEt3qX1dhT\\\\\\\/49K3C1QqAYxB8JPi0NSa8CiZfzgSvjRCumesRa3nrQIdszLXWgh3I2DdL3pU1J3dOovHiCQsEkWIjtqGvT4QvbZgP\\\\\\\/+J4sFN26CEDdtrR3G136WSvYEs5+EILs18Lqu6TG39yHTOM8Ah8\\\\\\\/uAa90IH+7EpuEcpJgWhMpAqCZgUAayGeU4+kl9Rcd2QeRO\\\\\\\/jM1KsVbarTOKrk7v4dJqEgxS+jw9C9vEEtn466iyWYv48MJ4eL9ps\\\\\\\/YETj+VgWcOHYdPDRsTotRbH8AsgBhgjBSh1lCt1ln7jgkOkjpmTdDZUSBkJ17+lxA6CdRA63OXDd+cwwdf9rqzcN3lcAYUiD9EF3xLFeYB7R4vteOAmBFeB\\\\\\\/Bde9gW1auYToIJMhRK7nGS5QnIsylOyvbriieqydKJ997YFWGdAKUBHM3bW\\\\\\\/+ykqHs5JK38nrd0UtWlXNMfrwotZlCXDl1+rFT65r6b11St2o62PF57C9HqgmC3RWTwqZwQ6hycQzRZAJ6MhXJlzmR3ypNpmTu7WkLUVzLuHpr5JnewI6VDuEFNATEcyVmfePGZ6XUPsW2PPp6qWjZ3kmSolgDuoZSOtUbnFr+gQy7VtWVPjjiu3BQHJ8WRF4BCRmz2qc8wnqumcLHVcwZgDE94JwbqRWnI5H+eCY9F+MXn\\\\\\\/1idPeP\\\\\\\/oTuUxPOnGXwYroNKrfP2tilXGuS0PXqbtm6cqEixtH9nDZ6MIMi5qml68o+dWQL7\\\\\\\/q3ARWKZFf2yWt3ffdPBhN4ZnyNS\\\\\\\/sMipleT6z8uQSXtmcz4v0TDdp2OzMvN73PIVEFji9p5lwKa2luru8StvxQkx6NdlnfPRyX5x3za3CQQQ6GRv+A9ewuZpNFR07IhLVIGLvaiP4PGx\\\\\\\/rGYdzRrE3P1ZiHTHoQT85cHeBWBoc5pDa+Ss94zwLQN32oeT59SWkE1xPCYCrSPhtkrf7zsVcF4Kp2nfe0xFMGe+5eUvK8jzx\\\\\\\/4dKqoJ\\\\\\\/pHE+F6UkcMEDuWlefIu\\\\\\\/pyV2WHe2UKch20fZihQ\\\\\\\/ndnDd2nQT7cvwqulWWo66V2YZnIVNkJ7L\\\\\\\/KZh9Je46aiptI6C7tUuDlEOcRwKxwB9Jw7I656OpIvSr4W5HBXwCScpPdnlqfIrqQeHr1Mcv3wGwr4YBYQqzdmipBLEArEbCMYqdkm+inr2tzLrf3sOWAEvS35BkL84ntlFA2u+JybAmiHmyMeYy4yotyL5xLBAd8fTk8okcF9sLcD1vnGrSv9qeb3IP+hH\\\\\\\/ceN9G5Em2kjKuKXzM9umiS\\\\\\\/wGOnkth3s1vBV57QlkWNqk2ON8\\\\\\\/9QxlC3pzYpUU\\\\\\\/wcmuSv2TR6cBR2qpK9I557oVYWBgHb8TQwvMyanhCkbfnE\\\\\\\/e6FqD7XST7W1w8dS4KBrqBXc94jTAT2fabqrR6LIW9MIM2jzR7gT+Ru+qhRt\\\\\\\/cEbateOmXF5AT3dSYEAwrcJ665OCUvZzHPDi9hfS4d+v7iIa8nXBYc\\\\\\\/IsCrsZq0tYgQC39JHggoM\\\\\\\/KMb56WWu5HSV2YB7HrVplIamSkV+GlSfG9pO7MmLPIT5Js8dpN7AnWrKqBx3jNSNzApH9xiAMMXWUwHpLGWo1u0uuaU5IVWbTg7B0uIRlIkOzC9HWZLe8bOrSpz7kjo5srLX0RucsYUhGVC4wF6YTrAmTwqDAv1MVOoXrZC3L\\\\\\\/9BL8aSUc\\\\\\\/qkOQwU+9DwVOEE0uCa7hdOC8+sCxwWn71\\\\\\\/YpgYHM9VYSpkeKvH8s7maSwAlbe+cCNmYZp7MHSPqnVbl45XKZ0Ga8x+QZxOxEozcDVD1RPMGJsrpA3HFSgAt6jklzqzFp9x8S6mS+QcL6V5fhSRRPp2zfDTgpDHv7qqGjWHh4LhbRevYY1Ct1Q6kq\\\\\\\/FiQvr925JO7WzkrmIqGx1JLNBbNi2mQqLPDq9AMGsv9mb3nrvzZ6semCNwV8x1Fin1zBj0y4JD03LJ4ZYDB8S+NDP+37D\\\\\\\/sXbvepHSM+pIeLrhJiNf+\\\\\\\/NZ5iTvLIVr9xkuZC2SLs8Uq5XYmlyxhm+e\\\\\\\/TFFtfBmsmMCpS0toGe60txdgmzLDgIt++u8SZd2Qg0zZthdBnwjxqFwJgnxhm7EDfnQxKpRp8MI9Gb0WGxSlvijmFpCxLtuvE6qX2wJOPgOmOxW2+yIZJA0QaVn5Vpd5uLcJm64rHCVOJcHTvF7nZNGDZe3XKjz4S5AFFCDXcYTkKBqutxxgC6ZxnHtysbtbMXlQAa072mtfx5kBTrTGQeecjm1194lYdvfpbthLdhduxP4C2jkDHyBTi4cuhE9n7vQxqZcEAR5uoA\\\\\\\/EMrBbGXvx88L3GDYsRBRdUWq8soXIKztz6cT10pqBZRBBUjEVzc8QjEOzcN6ggcfICZBt3d\\\\\\\/REmg7Br8EgpPULxMi7UzgHrkWa\\\\\\\/KGU1h1tN8d4F1riTAbP3ZP1JdSXuRq\\\\\\\/WNEi96g90e9gHye+S\\\\\\\/MwsJctwqsTssSOlV4PV4q64jtq30EdKdbOcFj89rAzQGkK5F\\\\\\\/CF+D9XyX+WuV7UaCVHGusDQQ0SWyKDgwA5PiDnAKYtuUBbp79gRnb7V5HJRjgiCi9gm7UvmPAQLv+r3gKWfHtLCrXRK+xnaKrZGUKGRVLh8BLSP5JKPTSlgHWKwmkjBPaDHl4adFtxUstsTW+5Aeg1KdJYlDGsf+zu4LWxnriACqGVE4yTdad0Py7fGCn7j0ecHi\\\\\\\/yulAvFhoh6RPB+vwJWDdl6nccYd3FiIoHkiAv9mOa8TXGfTx54WW4dQ30\\\\\\\/y38QjhQ0arOy670zv3jV\\\\\\\/1RDjkdQAO4dVUSa4547mYYW+i7TbKfeC9YE3fogwWfw2CBXdiNN0QbPx2wge6Ze8W82LC3gaOXg0eD6r4Ijwvr98B82baCCD\\\\\\\/NLFFuG20Auj6VhXMBcFx46pEjzC+z+6MG5L+AbEawbBtIEH5Dp\\\\\\\/I03SjiHNX5COZ1fSiQB9bsnWq6WdWEwUo6zRRqIgLtL8TKMx6lfNt72CoErz7YYyagkvD0f820WpN4wX8OXMQ4dHTwACAB3cGYkFFA+tRWVBlFDs9N9xdnwrGdlJ12Y6VRe\\\\\\\/3R6QVWrGtD85KJ1Gq\\\\\\\/gXouXvfaSKD1vGPjBWH8WbeVea\\\\\\\/XNmHlJuPuU44jyPCSLPmaU43k6Fauntm6o3PUQfaKo0xQw9Gz18LVrajzKljHLKfdILlfFkLnujv1bBxfBQufKATgu9OFvtocAkk1geHmTtBma7yOhTEiMzzCbwVPwZvtel1OlkPj5vVAGLr+0QkulxYAsvJ3MgRnFihKWH\\\\\\\/nTalDLE5iOscTLbTg8nSuyuWNDISV8ceh7IZGpxfPeYbJ2dT6syJ5YNLoeNvt6hDJx7yxPetRRkxZqUXVj3aMigL9fPPVBkBflsuNBhy3Iss5VLsfOYvKKghyuqGHKra+98ZvVbJMiSPSZOQzYkeu6D9xxptH1NR8f1rgmhP5uZS5RfhLE8YohcZBiS9aS6vwwaRs5hiSRE5mEDKZ2YUFMA+WdWzMVrYSbN7BwFAgUZ3Jzc+MFcUOeFDNlnTq6h2rJUn7EObxltQsxreHUUXZHWyKsF9qDofvTluhkmZ5Cr5cITCXCtQYi9BVcMjwRR9djmy32iCXRAJ58twqAIpxWteiwSWiU9Af1OFrW7e5fe8F0mEAPaOvBUriB2mUik9TtK8I7awZUSF68AnG4rOzv9WgOtKFzDaN9vlXwcUx+Kxa4seuiXJTxSmwoJX+ixdxhz400eoCNUH6+EvaSPoas0fsyDOsvNVd0wxHH7tZK0Lzw+ZbAFmIivg01pYjccTO7qS+UHZW0qSmDsMg2IUSDc8rK0j601pouNtJWoudAWA4MycA6gb+vS44eUf5JYSQxKUNs2FT1xCQgWTgQfRgrm9aM4UX1fiZ1z9nlf9a6Tz0oIIcdtHIVUBlgg8Fr6q0Mc71DqYihKnMBoIbbTsXaw8C4h3wh278UbGUCeR6Gw4xbozsHPjtkljJ1WlChw+d6XL3cDy7V\\\\\\\/ZY7pXwfQsTRp+lFInduTPZW4MnmO2i64u2p2RdDfNyMMSQoDNbjk2p79cN\\\\\\\/TmTNfYMWWTHAJbPiQErR\\\\\\\/YZTYR8o+l55+aEj1eyK7QMZ9q8yZ98DTmZjYMfiHRfIcP3pRYce7G7TanncZkZLrz7oIYs9e1mShBjSXj2ApE3wT9vcu5J6diohjr8q6nrKAfbq5BAK4QolwMWIJsQNdg46BRdM+6LOZX\\\\\\\/BKkUknzYeo2\\\\\\\/b2j6PhMk0WKSsNnbI8ZZ4f3rr6r\\\\\\\/XVc9OrTgIqB87nyitsOD3TJSpnS13N5kNd7jsw2hmU3Nxaipd9wDKL03Vov9ZGnxTqjrzeAOph5EjrTqZWti4yYSfWYK+\\\\\\\/PO+4UymuofY4zJDpMPpGrTMEHYIKytnJDQ90wYtx\\\\\\\/vZ5rXNaEuHToZ+odD2Dq+NXV4MgAjrq59yNWFNZvkcCjN4RC0eTauxcR8kAVqAIhlglwJE6l8172ZegCTUAKzU65hr3M8uWr0rf1Pp+anN2RCREHMtB7rgksjylPPe\\\\\\\/tuJ5Uyq7+eBckIdiZFUK0lhFrega7F3lk6BaS4mPJqjsraQpxs6R44WAX0koIMGObBxe0BRoZgsU7oq1DK+QDCduCsL1aE5Eke4ukE0ypevZzBeGe581L7Qnit1agYXsdPIlkj7fbY0\\\\\\\/1Ma93j9P6mr5Sy1zszBAJXDgfrlZ8Q4tlYyCNkSiZhH3V91911RcjoDINg16UgtVMHh0sfUivaoDGJFf6hSkjudoA1C0bN5itCueL+25mXB0Z6SJ+CIRsvFsZnSxdKpJYfJ6U8Q2KfXzQK2p41zBIAE9q86lianvOslsiAYy+OSz4jcgPgdzvZ3Ms+7SToYIgTabfQZhda+69D7lCp\\\\\\\/1tTBKXZcRLKmwn6FGxmgnTAvvWwu0RzFwdUUa1lidcIOsjByxX7GFhw76Cyqo1SOvLrag1UZarJisktWiEYo8FeWLVeo8qN3IExa4KbS1sQXa9JHXw3\\\\\\\/fwlxu2EjJJkY8jD02AF0MxofwAmtuHSgV081vtfLVPeZr7lmsd7XnRYmCIgAut3oggT+YFwdugP4Len+TWu6lEQ9c8yTsO3rJ8mxjl2tlJQ5HJsFChyTtBrDhYTMViAyluLGQI2zwUyhhELVnIZ3Hdl88aHT61FdHjefudu86EMF43C8lkWAA\\\\\\\/2gPX5VgJGF\\\\\\\/EG1inFXx89mtG512vMJNh3wAhNYqds+auuHJAmfI+KZCEmXpgVPPpN0NbjehROmBKI\\\\\\\/ATIldhZMYFGK\\\\\\\/5hmHn+wumTFXeAjNCUWVyYA6N5iWsfrG0DaBS3lipI\\\\\\\/NbPV5kR4aFhr5VX5aTnRKmr+89oT86AZHJJJrUsPub7KDofDGzjClWCz1Lfm\\\\\\\/2p1v8eJMWeW\\\\\\\/3o82vNua75EAeQOPQDj+s8UHwPOK4nVVmPAAT66J6\\\\\\\/0Yu8urUGAXs9znYJdjlwI7T0\\\\\\\/Fux+tIwr+ZgLBjG6D1EEuLEGQEd2sM01+GJZMzbvHWozlqrgE7jQQv4z+nKOv4OxJDD76MODvKl0Oyco1WAFeDrHuCHC6DHbQDpFHpIxbGp2SgPhhjo8WU1wAYkZvuupVCIqADIqlFW84+gEwQN0V3yTjm9Pk\\\\\\\/VvX3J5n9Pg0BVTy8hM+BFYkbtYJWRkXpz7ufrJcIHSUSCBFTULvyRHOXQXQoj9ejHy0QKC3+nc8gSqeFLdTZB00YXMzrLN3qGd+YuWjexwJ4bHtflQc\\\\\\\/G39KBggLFuwYAJRjQXmfJpDSDxxhaTyumqEZJfUxwSTSF2E67X8WMwS26gKWQGVvUzuAkx3uhXvJ\\\\\\\/AYwzMwKG3pZ3Ujt1QYPV2zDwzg7UBUwwSCJe1H3UkpfrKSMYzEvAC2YeuHszV8PNgmx1K\\\\\\\/guTjea1L\\\\\\\/BD2b3NL+BQBGmZrKXa2mBhNm1nQc7ECr0ONaA0R5amT\\\\\\\/eQzJohumKB96R0Rn9e4XyVgqEtkm9s58PWzrLp63hZ9JZEwV5cWVWsj+TJGK5jkcctPqnK885oYYSEVFXBotrHy6mz7ZXGV+7A02yOoxIDcGVyWK\\\\\\\/x+qrm\\\\\\\/zHKjtD3wAHMFZ5fFVy9mEQuIv4\\\\\\\/zpvj+Nqni3RloiJG4QvzqkIMFAu5q6O9fASceD+nRmb2NJC\\\\\\\/p7694cUMz4fWdHhTgHdpH7jZSHpc792uoqmm1ijZ0Q4Zft+UU8t4aT4IK4Odt8VpKDSzDK86+SmvcVoHi7c9uZKJQEBOuf0BJ\\\\\\\/i4sm8vt6v6dVqNTphQxNL+r03hDZUtk3KuOoQnHnP342V+OrqlpdlgeHj0L1zkbPMWN5QxBmqcfU7wNsGeSwgBz3kpxYRu11Sh36AgIqxvwt97Nk5j8Ou6bkSCuvM5yDAOKgoEWWrWHbXiK3lDgFXYQbXsUDHiLX0q5\\\\\\\/3zRBpYlJ\\\\\\\/YD8Hce2yGO8EBFPX9QDmHy2mat9DwtcOgpgvCTt9K8jkAYbf6E9vn8SyEqpyEayolFuxAVjTxhr1RHxhc9+gyt\\\\\\\/WZNhvmRB0fUvcfYWuepBPUgOrGF4po4\\\\\\\/yCHGKW7hq785gqjJLHxUY4q7kp2ePFnI+oG+THFgSrQFmrG9eStxYIQwBGjNMYb4UsPX2ccdKXSzBeFgWD785RMbqLNl2DO6EBo6h1riEJtFTAYwuS6FSpcSsUqzdO5\\\\\\\/G8iF2fVUP+wa0bxf2WqKF3aTuARav7YKBNgzFfzbES75ND5scNtSCowdA3ybNGLK6ztL2NoizBKOcV6KyUzLNgboKY8KXYxkNTTFqokU2Or4+x0OmgQaG2875XNLQCURmybuEO0Njk1D7GGQKZrmLQ09haDDx\\\\\\\/3XmMV9pwSQD7y5h\\\\\\\/KQKGoGxKMJn1DWC\\\\\\\/+ELL4PgQvZoDYUQ32SlKuKmlFIaRWiExJba2rfkxC1yiA+en4tJSy4WNDLy8HJ2ox4QGCfj1nbxI6ii9IMrQuM2RpzrgF57an+iDF9e+xSrfvfmnNWr3I6MyQTyKc0vvRmsBYbpngyxGQ\\\\\\\/11u6BzJNflkdng6cSIL9unVJMnnhGI0pud5nR5QfHXzHTYw8YWAKfeJnMJSXdqgEV7hquAYphCoMbmgL9dvhxfxNpjAh8huWo6Ju7k5U6KvI6XnU6VQhEq\\\\\\\/BZM4TKP3lWjGQntsLsLJXceasBJrgsgEcJPw6h4MR0VNN795rfCJ76V1vc3ciwxUTCOu2cTxMad\\\\\\\/ZnivU74+rlciy5fjO0w3gpbBeGeL72AHMxLk9wJPZcpASGbCoLbv5qZDkq8MGCYtObrFXp7y4ma4ml1XReQpmHdD+\\\\\\\/dWs0ysNgESjYr9TOX0JiLt3+653EzKW5u3MmtV286faLPqNiMqfEB0hVPvUpbv6c+P7rKgaawKz4IxpNj31zLcut7K8A+AHQFVB1U8MRnuSua4VJTXyC9Z8bz8Uk2yaL\\\\\\\/JJ5nKOYbpsfnIZ5+l5STC70lJu0FqBgBbVDDyYkQ\\\\\\\/Qibr2A7LJXXeaOLDxp1DHZVJi9QoqPo2qQdMXQXdTSuKHb+DRtAlrSHsOwoILcWzr\\\\\\\/rWoSc+VZtnt3ftkt31uAnrf17j2m+Rd\\\\\\\/GN4U8PozOKmVMLDg9GVMm5oUjjDyv3sXb3mC81h6ZutMv1eS2rDvDOfX5Hf2cVXb4Tbww1ujDdTohdkJ2sCINAbiKqM63zAcVQIm2YjKdECYcOB5kcUVqSgGBMeG6MOn06cFny3op7WL3i0FpNyLYXFUyKGd71LiNAiDPtkUGQHyeLkbEcbuca+6geI+\\\\\\\/ZQjIbW1oqBZqAhGDN0peO5GGbDHPSVVkMfFoAXS6tAK6ENM9c5t+34nY0BpXcWdrEbPVg5ue7OR1n1AfUzGtcoxzn+0ps3acceziNeKXcLx52ph\\\\\\\/wCh4kBMnX67jPqKYEhyT8rt9OGyDjFfSpMhdQ1jswUFOHwRswxcQzHbkV4eF2tCD87Y+8Px3Vy4+sU3du22VCjvdaSzVC9J+lGV2qmkKHxBIxOFwZG1g2qdePbr9qp0NTcdgT+ywa+f53Nz0jQdJx7vhAyL3a20ukcwPIDY18FxRQ22OTwQdC6PyoGymgNraNTzPZycuPnXCLbNjxFHE+HaZcyZh\\\\\\\/DMf5r9wjNftj1KdjJJi5aIgVaBna009onqFX2dfvNJW7FqVDgpuabZByjl4Jv6k4cSs+HvZ8VXxAZC0uQv1zfryEDa3tER\\\\\\\/68kz7iYPqKlHSj5EobvUmAtkss\\\\\\\/HQwAj2MCWLbrm4LXyyKH7NVG\\\\\\\/q8i1AkWmYX19n+HtTGtoFKhjqNf1hUQEWMLeElaeC5h3GSpQv7EuPoYptQ5sK6LQZrqSqoJIQWCMkyOqqCF2cqCiPqmYt5gIGDmj1uV7BcJvPB8w9lbRU\\\\\\\/2gPjsNObBzFtJOM\\\\\\\/rXapIjDbyrKTqSATtW+k+LhUBQIGigbvXy\\\\\\\/zL2tWANgjw0eAEo7XkIEJJ42ULDdnm\\\\\\\/\\\\\\\/FOXyu\\\\\\\/d3VLN\\\\\\\/7O3VAsVvv+Fk1yxoJ\\\\\\\/viIg\\\\\\\/TPkH4ppp8gzlzsObqlTuptf7E3+0NCya\\\\\\\/eWkM4GrCjMsHQpmDIng8aJ+Qb3gzHEQH6COCmoV3MMJ6V3bLIyWx4L89IShflTZUAZSTu1uGLRDuT\\\\\\\/v2ZfCiTDVdchWgwel+yFK24Z5P4Kp9kFyzU1KA8Lb1QyTqtOPaqiUpN49D7PrEssKBi3iPHL+xFxZbUTDMWSPZWVT9C33bR1ErCQnE3\\\\\\\/\\\\\\\/ecMKNlDhckdIVPhhIMQzm1M1fbyDzmokzBR8UB2RNGXH3WGRnE7AS\\\\\\\/eQ7s8z80tAJludcld5SXD3u4HCXhT1ZiWYExpE1EqZkHfxX2ddeqMNcMtX3cup59Z+S++u+D1xxMnP48tcU7xghrfrgVG8yC3AALtaeQDzmllpDwbqI\\\\\\\/VAFMVgtRJin9RJO0MU0cC2Y7xKb6NNID1It06imkZ7iHkCTTmGGP6lXkLKOjwOhOtB\\\\\\\/evreZz0ioLm8Lcn6O+3TkGknQXwmU2EMMzjOeAWzVOrgBggalPDF27orrTomdlorAA8OOJ8j7MhM4uAetQupqWh6uVc0RTSjbA40\\\\\\\/LkrNKmEhbmSL7MTA\\\\\\\/xntezwiO1n9sDbFTbsZT58tkc4fDG7aLUa0fbVy0ypzrIhI6q8lCBRVsPVWrtLKlQw+5zhOSRZVJQPfNFVIMk2oH1bwBhZtDLvMsoEYnwyKW2YaSuKQyyXX1M6uDAhGhB03j0cl1cRPPSFZ8jPTU9Z3NZ6IBqBFzLzxB5qMe9a+CeBS7JbepEbCMWhjmEHMZ\\\\\\\/JoXWVLuOoQl6zmB\\\\\\\/apqK8Rfiaoc76PsH4nT7e385C3XDWrXh8KIFn6CNn4JhSOiLjibcs1ajuVXCG+a6mwmAwHzqh7h5imO6NkEfO0SP\\\\\\\/DOhG2Yy7Zzyjj8sjS14KvOg\\\\\\\/sAJigqdTf\\\\\\\/VqqzCsYWFwNBLia3lr4ZYoQ2OfxAmZtdUZIHOLrftRfKFE4ucaH4yITbl0WjMjjB3ICZJfkM\\\\\\\/sB1XGsklBNS9+jZb1+WBH8QQwdhcURHWCiW2sEQpUrOhAWNj0XdMoJ4lOA8ErCoJAZAzhRZq8u9bHk1aqY56qSlqcDJQ9\\\\\\\/Tz29UDaeUN5JXMhUyHJKeuB8poQf1c8BL1dOZxNwcc+NxKcYGog\\\\\\\/0JGECD+cUOUDg\\\\\\\/lYklffg+KTyq+Ps7P3A3SZHd6BrIghA5jchsQnf+Bx72PrUGFd6dPhH1PcuRSGsIjkR49pwqtBK+YPAy1Iv+J5JPVElHySeUpW5JooByWjezFWecGWq6xzb0Sn6dsPX5+IS6LEnjHQIj48WcVJTcErmDN8liLLfm3zWkieByhp+kIi5FzU9iPPJi2hmvE0P3D8w2BEW9ZZp2DxPsNMlWcwb1xKkZQgJNwRMfB0IBcLzGsZ2oR1QKGoenBJyjzJ4+MI\\\\\\\/zomIZggCqcAwOIeT\\\\\\\/cQmoszenhv27hUPU2HWGWjaTC\\\\\\\/RRQuPuTq\\\\\\\/nM7OZC38VNb4hbHa0SXGrEsmoxIIpevtO7TWt2lizlQWUDaBvk8qyTme52gRbX+1KL6qRrYB++Sck7fr0s2K6m8tW5gOgjhkrJuXkbSpr8Ycdv42s3XDEAeaP7PsHMnwQ7k7gyBv8cuwpDrXwnl0q3R1yUK+V65H6PZKdoMgT5juYLHVJ5QQdY93E2AK+TeRbaHaew2L5aKMybubE0f\\\\\\\/qqah9\\\\\\\/RufqpKs0QnXi365opvyGk\\\\\\\/1x\\\\\\\/4QsmMVwkr3UmDwsMFss2nzRctYuVoKBt0br4KwMWjv8XJTUwdBr\\\\\\\/r851FfmQ5gEgye38Xl4kp0E18zBLfL6Anirc3P2B81TM0vNr+tn0AJ4Qp2HM0cN4s8BLpQp51dn5V\\\\\\\/Dp5gbRU056Wz3qpYm3SE5mPDt\\\\\\\/VQI6kHqFycopQRx1zL4ipRVUBxYbN9xkxX8NGYl\\\\\\\/lYSA+Jf5g\\\\\\\/hH38iuWIWYyuKZg7hfOFp7MIey3kqybhqkujbjm3GmD5HN3DtXjxN6Nvrzt3KRslc+s\\\\\\\/D4t8yoV7DEMwrvnK\\\\\\\/iAxJsN+letbP5vHso3LI3jnOOQ3f1+Nnp69MLAOfk1hSjPdoKYAvhgVuYRzkpsis7KjwQXryd95jYHTixwsRIq\\\\\\\/5WVE8s+p\\\\\\\/2JmQiI9Cj+ercbqZPVcELIyoFkR0tPde13egCFJaSbSzQ6YNYjPyIEJDnDvfNJ9omPU\\\\\\\/YyNEquxGlMFAG\\\\\\\/\\\\\\\/SM0NuwDLpMzhXP3dIOdA1gqIc\\\\\\\/wcnMMoexCCnl+XrodN2KysZKiQjSbaJXRNcc+h9n\\\\\\\/o2i7cB2LGEpM8jCw\\\\\\\/1JgaWy4uUXKNbgIvOLALP8si1PWaTnAvkyDJm7evlvcunR+9jCMUQix6x9rfnTNBGlooRZ0hHDzarWyLDhjKTw4QYOw\\\\\\\/uiV8KF2Bz6CNW3yASucP7QLc0B1vFJsFGifkOn7O3hqh9NtOaCiKuzf01zh+6a+6Q0+KXT6Wpn2g13+ssiA3VL00wurA+SpDmVwV17Lyy1y2PgF3dq2yzMjHT6ZAAyB8CKn0CdHtbO4J2rWcaygK7ga9vPmzQ4JOvOSBE76bEWsWzm+\\\\\\\/jCX\\\\\\\/rFVvv6WhFcwUi9GqR8pMKpAE7L4k5mpJQlgnTP9HXK3bRF3ynHShzUVmbf11zqexGB2RWxGP75zjuAlp+W9mhyBZrq0hEsqixjRJoXJpXdseBCrf7xR+uroxB+YmqtwOCDjRU9GIpMoiUuSMmSsnTc1IdUlQL4KI3CpN8DksCltNWz14l2ybwTSBgn3z6+Y4vs6cJtnZ3VR2Uamhb6un3ATjT8U53KknLO6wDiVoi3rTSHMCbLbGlRjJ8LEEO9Ot066Hl\\\\\\\/Fm\\\\\\\/WbK+1vbZYALdCo68Ol5\\\\\\\/WlTNh+5dLIc0cBPPoxkpml3x7XvxQgtIYuXRx\\\\\\\/zMH6FEbjFhJx22R9rIXZ31O6vrZh5YGFtZbw6Y4cq\\\\\\\/jUfBE8WPGaotwVWv+ULqxhNgd+PRmJnebGDhXrWvo7guKUztKGGa2ROf4UzaohGq+SeBu0vCZPOaQU91Yvx669AuK5BdyBEkaOR0qAzniAkMuikedL3zc45Tn8q2yILk0MPGuYJlgl4GgThApF6lLP6Xwpe6IkR1B5m8jgDIoU603kXRg5b\\\\\\\/9EGjpj3cU5MfApExZ1Ntku2tuETT3zdTGys1aOfEGs8wcs+5Cd\\\\\\\/kO8\\\\\\\/Zb2RxpKi4AebFAmcCh1svHXTDtvw0LwdRDKAGR50lF6SYbS60zYqs4jhpkWp0\\\\\\\/HTchF5pgSY\\\\\\\/G+T6q8PFxfDDintpJ\\\\\\\/yiGi6IuxqoYg3h7XAxONNZ7P9ot6qRs1fb2oc02DU\\\\\\\/KNBuh\\\\\\\/x80l3cUpx6\\\\\\\/s1toXz+2GGLSbHXCWyjOgTTzXe4OisD6F87UFLBnVGAUWVJAna5DEk0HyN0kXNHCwQefBZpG5cp35iIucCpUGyFc524SksOXX3MAs2d\\\\\\\/TXV5HYSgaHJaAzCWNRH9DLcZAOtxkn2d9uv351HjXeBaqXxoQbVvrJ0moDmErreZxrwAVUQNF5\\\\\\\/Q5w4P+5+HwQj\\\\\\\/3a5T\\\\\\\/UMnMuq2GFdCHTNcBH+lV6Q1q8f5w0S+VL1KvCLpMrSIzj\\\\\\\/iIQQ5xDjq2fYZoRoxSwCvvDT6c3nNR+NvcPftAwKDp99aJ95I+229iD8zpk2ofseZ5aMQ7M1QJslgtUUbxIaAaMSxmjiXFdbyi1ywbh+X2Y0wP8EQdkQ+\\\\\\\/fpfTTDXlTDRFrmr9L0Rwf2KWs5syhvrNx8RV9FkGCT3dgVKz0GsUngsrSJmWQDJTMbRfnZjKf6ut1c8NxSefsEN6jBlWv0BH\\\\\\\/VHj8xrtCmWN6iIWkzgdxORU6tM8WLZ+GZ6OuM7jWqS5GpApXNFdH7WdBUcqrAwmS7V4QhoF7CGIzkli35e6hNKG3ljPQQCk3AohWpgRjN1zhOp6w2\\\\\\\/cXcC3dfKtF8vilQN+AC+DXIwpsiZYtfoYNYL4NBs0hLBha4jRcfyCHG3QhqdeJYrA\\\\\\\/dPl1oXRW+NPhMjWJI14PaJtM9+\\\\\\\/Y3gdGqZTOJn+uDVvdh2cjumR4jMwfdnUUz\\\\\\\/nhsC9TahxiDyFb3\\\\\\\/mKmRHFPQxOuZyENnZ\\\\\\\/acORSEsGAP1P10cXw5dDyo9tIDspxsmbYcK7BG8uMy44jM+DsGBXvh7qM928v2bcsnJGL7nVB5JuGh3AjZoQgcdDN8r6V0VzdC8KSJpgN6nAWPYYStiJf6TebdsY8LO8YUhMulNwaL+0tNBD59WQVq\\\\\\\/u3IM+T2NIStPzaIpFvU7VSxpvC+qy6AEgHzPMG4uP5pEJWbp6lsgfK5GH12nDJ9Rt6wW3+zl4F4wlCedZklReg0WJvP\\\\\\\/z\\\\\\\/MrujityKEysCSoPn2nfaVj+9BC0+8Za7uGVqc3CBhvq8YFrKIMMAdVSg+4vDXSqsOHeCH2mz5zAHF7xvNSBaezoSHwsSmlqVvFdq37hMrBP8OECfGpG6mbcn3SWINyUnMvhWbWvE26HF77iWTCPbHOKHMl61H63Fdy\\\\\\\/MKW3aCVjWJ+p+7o3w6AQH2IqIK2dKS0E\\\\\\\/sQ+x32zXk2wc2QYW1jkceCs+E\\\\\\\/lC+YWbzgE28YeCgwP\\\\\\\/4dXBzx09vfE\\\\\\\/ikdu8cJ6fKI6Z7bXxQGvdbAzbw6LOsq4vFhX9bKG28zz5ng7HkPOLukaVfnA+Yam58rLpZHYQGmZYYJsyM\\\\\\\/GeVGYr8qUUSXtqwe61j2aicZKAVaweZZi0Dd73lQykJeamhgjA2b33CK+\\\\\\\/41jPN5x2M4d2f6GKlg6ngnJ7nzQ9Sle6LlDM9fEgR7FJSIebk\\\\\\\/7o6Pj23IzrcGa1VOrM1UzCT+9KkFMb7nzZ\\\\\\\/e\\\\\\\/\\\\\\\/kEgQXapHTV3MpOG2ufsUrrc704i397+bEw3NWe7t03fuKodFBvld3bxl86xW6Hl10vJp4Nwkc+DS\\\\\\\/lhOs4WGbCq3\\\\\\\/j1jmLY0OvWHxYwV7d15d2jvlYVcc0qtwAPma3VsukY10X393P88VPaUiq7s0uUgpO+hXBygKF3mmKv+pPFdRY72jGUwP9JUQlNhmMfz0v6rIk5xhq\\\\\\\/Dqq8sGvZfpNbmj48AbyBiwpmTOvhpPiOY4D3ju1elsa0sS\\\\\\\/lwN11GFKOsxt2A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MTRVgXwd7aFdOR3oE4VFCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227635a8a341819158f518216af60b701db5d080e68e21d478ff4ef4944774329f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CNZ52NUOc0pO64HKBLInLNUFHQUNpdDvrfKnIVOZSYpdG3l+3\\\\\\\/L0lm9BtW3QtYrjvTfoOaaTm6fHMBV142et1ubRjhXw2Xorj211eLSbxhVRMyzRVbW8M44TU7Jh0XaLMORYVkP9e\\\\\\\/A\\\\\\\/P2WKpFyUYjfo+pPYNQEaFEiwo+y1G1DtU385LLidYQqX1aVF1YgUZ1k+Gqm2RwTJ7+5N1Tou1oKcqIbqedJim6Ugzw57P1qApVxj4u9NAP9hVPC2H0QY5\\\\\\\/dCrvBq\\\\\\\/YMabb0aVLi8vSfXmlx2\\\\\\\/gqY+Fw31qFTfWUjglzddGRfCI7CTBYVHoaOfSYrtbULra7SGazUhiCmdH9Lyaz7qPfzHyUgOlSzcsZb+RjbHtaVfyqcD4ad1EOna7xXT0Sx3Uzq2qHXK\\\\\\\/B5reZ3\\\\\\\/zaUQalXInWtsC7eGIEZjbaU1a6CR3hy8H2sXsnz1Wqm3pbCq5revOS2QjyGrRDqPVQaoXFnuI3jh0DF1B4wzlyfjatowEY773EMKwr1pAoOGacqIwZ45wLGZWB00GLQL4rvc6xhEt3xIAKM3Kke\\\\\\\/kHEjmHs6QITrA5lR5Vo2rw4YOlx4DJ9zXtgEiI+A571FDrYatfvXuJvZRaggdgbzLaKkEEXF97sGSPwQUKAH1FkzAyqOEp6dJv88FxVAmxwM07oq5Can7jV1UT5zpjxSlTnpBd8RJnae9KhKPiD+5o0MWcA9OCdgH7PaAfddagR1XfsmVC92nUeeT5jQgUJFNojvXz0hTVpWNorMfzKEcA+HGn3PN\\\\\\\/UfGjZsOCrvsljKWXCmRr93XVrQ4La7Qs30Xgh6aWnsj5fvZJhIJYj9C8xVPJYi3TCNXUGdiliKVgRNHRGE3\\\\\\\/zRG8enC+rVrHIWkydB\\\\\\\/7x165kMHPBRAzu6uZQg3RTwQeUaIA7kI7G0IQIMorv4m5E\\\\\\\/GNOD6YWpEWmoR8F8Jh+Ktqd2UFH6cLDYXqiuUYr4vnMHfguNejoTI5HWZ\\\\\\\/nD0UJ9xS68KU0I0ve5yXkL5okLLZIzwoFZhZsnd9Xl2t1iLBjE33CqRp5YXr8zQtRSGw1+wms02J4Ypdz+mgQ70vo0+dDQY\\\\\\\/zhXlkfUn144cTTHij5VxBhhaXQZZSqfLnAEIUAAtgSNJxlvbK0CJ2LPabElte\\\\\\\/mmXtjwJ84otFujWRT3aH6tp+ZbF5\\\\\\\/vz8\\\\\\\/L8UZikrepDRykYCyMkJiePFSW8NrKaWI\\\\\\\/uBwF9hrv\\\\\\\/8gdDOZLTCZC9vMRK9Lw9883VxfIj8tMm6i2vbtZ7Uz8nXZtILScBSAq3+40vdHvH+mbfjsRICzuQje5LbdgQzf6NaLAzjIpIGc2YsM4lfMqO8ESw9JBTgssquHEbK8GFpAKcjbfmgoItKcAznpp+gifzSi\\\\\\\/1S\\\\\\\/5rniV5c1PqMr3n2KIxaLUBoqcURuHPoaoONjGrEpX4pbHuSHXbrsnE9iJRSVG3lmw5D93RCKJ40sW46pzaqe+9xlV+LkR\\\\\\\/nOZLgOL1pOW5eZD6WETOLVTo1fQjUP2VEJVrUgKBrx81w0btflCDzgsXFmP6HE5M9oZhxfJt0IrnEnnLZrBNiqnGdH5ouZ3rJeRKuUdF220WD\\\\\\\/3fJUXwBMTmfr1nDjBCmNC0AJLrXDWwyIkn447jesZqebwU8PLojOxUSkNyY367ucpd5HE\\\\\\\/3UfESXAyZn6z\\\\\\\/phEYi3Prx2rQQiVfRM2tcIIxjLUXRM6jEJEacGZZBqnLa+s3h0UhZ6wO84omqPURgdRRNl2rPPssQdGJNe9\\\\\\\/frErbbajZnQLdZ3REXsK9qAIigvwq3+E\\\\\\\/EWmuRcz8YemrSVq2PfC2ss0DYAzm3dvc5Sm3BeKKJO9yAV5zxkhI+mAR1dyrnnwMjOKYnEBpeEX4ec12GI3NjSbT1zXZa2D4yI1TMQENRg8FupcF9xKflHOMHHmFfJEqsQH5wrgEpW2QCnrPahVNyd5a4IFGK9VCgKE4yiJBfLY4Gz+4GxGEr1zS4gHi+Gpq9KojOJRJjrjNzQHey42xtmOOd9keEHY9EMDehBUKQQ11zo6uXhEK5SaRF4d2L1o+PY+Dxe\\\\\\\/cs2ZNYuOgtAVKOFo\\\\\\\/VPKCVScGNkrm0VWS07xsljqGNx\\\\\\\/u7BD49QPXxN41Mdawx1wRZb\\\u0022,\\\u0022iv\\\u0022:\\\u0022S\\\\\\\/uv2aeSWGFaXwS4B6Zhlg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220dbdb966e8c6c4c9bc541458aac53cf5d3f8d2902faaab3d57e23c0a31c935d3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460068 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/435" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C5E1lbTYT43bA-yoSqu4K8N0R2rT1TGW6J00_hKdc8o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/435" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Gy-cWSX0xRbfPEvnIzlJi1UPLBYHsARxWrH1aqXnfa0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/423" title="MicroRNA Maturation and MicroRNA Target Gene Expression Regulation Are Severely Disrupted in Soybean dicer-like1 Double Mutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/447" title="FASTmC: A Suite of Predictive Models for Nonreference-Based Estimations of DNA Methylation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460068" id="node460068" data-pisa="ggg;6/2/435" data-pisa-master="ggg;g3.115.023119" data-apath="/ggg/6/2/435.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tying Down Loose Ends in the <em>Chlamydomonas</em> Genome: Functional Significance of Abundant Upstream Open Reading Frames</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frederick R. Cross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">435-446;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023119 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460068" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fcross@mail.rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">fcross@mail.rockefeller.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/2/435" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460068" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460068/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/435.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460068" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460068/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/435.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460068" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460068/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/435.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460068" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460068/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/2/435.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460068" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460068/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The <em>Chlamydomonas</em> genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. The annotated genome is very rich in open reading frames upstream of the annotated coding sequence (‘uORFs’): almost three quarters of the assigned transcripts have at least one uORF, and frequently more than one. This is problematic with respect to the standard ‘scanning’ model for eukaryotic translation initiation. These uORFs can be grouped into three classes: class 1, initiating in-frame with the coding sequence (CDS) (thus providing a potential in-frame N-terminal extension); class 2, initiating in the 5′ untranslated sequences (5UT) and terminating out-of-frame in the CDS; and class 3, initiating and terminating within the 5UT. Multiple bioinformatics criteria (including analysis of Kozak consensus sequence agreement and BLASTP comparisons to the closely related <em>Volvox</em> genome, and statistical comparison to cds and to random sequence controls) indicate that of ∼4000 class 1 uORFs, approximately half are likely <em>in vivo</em> translation initiation sites. The proposed resulting N-terminal extensions in many cases will sharply alter the predicted biochemical properties of the encoded proteins. These results suggest significant modifications in ∼2000 of the ∼20,000 transcript models with respect to translation initiation and encoded peptides. In contrast, class 2 uORFs may be subject to purifying selection, and the existent ones (surviving selection) are likely inefficiently translated. Class 3 uORFs are found in more than half of transcripts, frequently multiple times per transcript; however, they are remarkably similar to random sequence expectations with respect to size, number, and composition, and therefore may in most cases be selectively neutral.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/chlamydomonas-genome" class="hw-term hw-article-keyword hw-article-keyword-chlamydomonas-genome" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Chlamydomonas</em> genome</a></li><li class="kwd"><a href="/keyword/annotation" class="hw-term hw-article-keyword hw-article-keyword-annotation" rel="nofollow">annotation</a></li><li class="kwd"><a href="/keyword/upstream-orfs" class="hw-term hw-article-keyword hw-article-keyword-upstream-orfs" rel="nofollow">upstream ORFs</a></li></ul><p id="p-3">The assembled <em>Chlamydomonas</em> reference genome is 120 Mb long, 65% GC, and very repeat-rich (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Merchant <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>). The assembly contains 17 chromosomes (∼1–10 Mb) and a further 37 repeat-rich ‘scaffolds’ (0.1–0.8 Mb). The genome has been annotated with 19,526 transcript models including transcription starts and stops, intron/exon boundaries, and coding sequence (CDS) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>), and the resulting annotated assembly is available on a public-access website (<a href="http://phytozome.jgi.doe.gov/pz/portal.html">http://phytozome.jgi.doe.gov/pz/portal.html</a>) maintained by JGI (hence ‘Phytozome’). For a subset of genes, many of these features have been verified by comparison to EST databases, as indicated on the Phytozome website. There still appears to be a need for bioinformatic methods to ‘proofread’ proposed selection of translation initiation codons in the transcript models because, as will be detailed below, potential initiation codons upstream of the reference initiator are very numerous in the annotation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-4"><em>Chlamydomonas</em> sequence files were downloaded from the Phytozome website, as was the .gff3 annotation file that specifies location and strand of transcript exons and CDS. <em>Volvox</em> predicted proteome sequences were also from the Phytozome website. BLASTP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) was by a local installation of the NCBI BLAST suite. Other calculations were coded in MATLAB. Scripts and functions that will operate on fasta and gff files downloaded from Phytozome are provided in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023119/-/DC1/FileS2.zip">File S2</a>. The codes produce most calculations and figures, as well as the file Creinhardtii_281_v5.5_transcript_summary.mat that contains preassembled data and calculations of various kinds about all annotated <em>Chlamydomonas</em> transcripts. Code is also provided to update this .mat file in the event of a future new gff or fasta release.</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Translation start sites and 5′ untranslated open reading frames</h3><p id="p-5">Annotating gene content from assembled genomic sequences poses many challenges (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>). Consensus sequences for transcriptional initiation are uncertain, so beginnings of transcripts are uncertain. Polyadenylation/termination and RNA splicing have moderately high information-content consensus sequences, but these sequences clearly do not account for all RNA processing ‘choices’, and splicing intrinsically adds a huge number of degrees of freedom for computationally assembling translational open reading frames and associated 5′ and 3′ untranslated regions of transcripts. This process was aptly called ‘Gene modeling, or finding needles in a haystack’ (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>).</p><p id="p-6">The genome sequence and the borders of annotated 5UT, cds, intron, and 3′ untranslated regions are available on Phytozome, allowing reassembly of the complete set of 19,526 transcript models on 17 chromosomes (and an additional 37 unassembled scaffolds) to provide sequences of all 5UT regions and associated cds. (Note: the 19,526 transcripts are derived from ∼17,000 ‘gene’ models; the extra transcripts are due to proposed alternative initiation, splicing, and or termination events. I elected to treat the transcript models as independent, since it is possible that different transcripts from some gene model might differ with respect to 5UT or other relevant features. This provides the possibility of a minor level of duplication of results for some findings; an informal evaluation suggests that this duplication is approximately randomly dispersed among functional categories.)</p><p id="p-7">There are no obvious bioinformatic methods to reliably determine transcriptional start sites; direct biochemical measurements (primer extension and sequencing on primary transcripts; PolII occupancy) are necessary. EST sequence comparisons provide approximate confirmation of transcription start sites in a substantial proportion of <em>Chlamydomonas</em> transcripts (Phytozome website). In the absence of other information I provisionally accept the annotated start sites as correct. These start sites, combined with annotated splicing and proposed translational start sites, result in annotated 5′ untranslated sequences (‘5UT’).</p><p id="p-8">The standard model for eukaryotic translation is the ‘scanning’ model (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Kozak 1978</a>): the 40S ribosomal subunit binds at the 5′ mRNA m7GPPP cap, then scans in the 3′ direction until the first AUG, which is the translation start codon. Location of this codon triggers joining of the 60S subunit and initiation of translation (reviewed by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Hinnebusch 2011</a>). Exceptions to this rule (skipped 5′ AUGs) may frequently be ascribed to lack of the ‘Kozak’ consensus (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Kozak 1989</a>) in inefficient initiators, which are skipped by the scanning ribosome. In some cases, a short upstream ORF (uORF) may be translated and terminated without full ribosome disengagement; provided the distance to the next AUG is not long, reinitiation can occur at a downstream AUG without rebinding to the cap. This provides the potential for regulatory mechanisms, the best-studied being yeast GCN4, where starvation effectively increases the distance the ribosome can continue scanning to reach an internal AUG (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Hinnebusch 2011</a>). Internal ribosome entry sites (IRES) are found in some viral RNAs encoding multiple polypeptides, which allow cap-independent ribosome binding and initiation; such sequences are rare or nonexistent outside of viral systems.</p><p id="p-9">Transcripts in the annotated <em>Chlamydomonas</em> genome have indicated transcription start sites and 5′ untranslated sequences (‘5UT’). These annotated landmarks and the reference sequence lead to the result that ∼13,000 of the 19,526 transcript models contain one or more 5′ untranslated ATGs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). (Note: although ‘ATG’ formally refers to a DNA sequence that never sees a ribosome, since this paper is entirely based on DNA sequence I will ignore this fact in discussing ATGs ‘initiating translation.’)</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467582/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467582/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Annotated 5′ untranslated sequences for all annotated transcripts were extracted from the reference <em>Chlamydomonas</em> genome and analyzed for potential uORF content</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">These uORFs fall into three classes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Class 1 was in-frame with the annotated translation start site (henceforth, the ‘reference’ start), with no intervening stop codon. Thus, if translation initiated at the upstream ATG, an N-terminal extension (the uORF) would be appended to the expected reference peptide. Class 2 ATGs are out-of-frame with the reference start site, with no intervening stop codon. Initiation at class 2 ATGs thus would produce a peptide (the uORF plus frameshifted translation from the annotated CDS) lacking any protein sequence relationship to the predicted peptide product of the Phytozome transcript (the ‘reference peptide’). Class 3 ATGs initiate potential 5′ uORFs that terminate within the annotated 5UT region. A given transcript model can have examples of all three classes of uORFs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Note that according to the default scanning ‘1st AUG’ model, class 2 and 3 uORFs should completely prevent translation of the annotated Phytozome CDS in ∼12,000 of the 19,526 transcripts, despite the fact that in many cases this CDS displays high evolutionary conservation (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Merchant <em>et al.</em> 2007</a>); the same rule would result in an obligatory N-terminal extension to numerous predicted peptides encoded by class 1 transcripts. Thus these highly abundant uORFs present a <em>prima facie</em> problem with respect to translational control and the predicted proteome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three classes of upstream open reading frames. The 5&#x2032; untranslated sequence (5UT) can contain no ATG in any frame (top), resulting in no upstream open reading frame (&#x2018;uORF&#x2019;). It can contain one or more ATGs in frame with the main coding sequence (&#x2018;Reference CDS&#x2019;), without an intervening stop codon (Class 1). Class 2 is the same as Class 1 but in a different reading frame; in general the Class 2 uORF will terminate shortly after entering the main CDS out-of-frame. Class 3 initiates (in any frame) and terminates within the 5UT. Note that I consider a maximum of one Class 1 and one Class 2 uORF per transcript, although these uORFs can contain internal ATGs that could in principle initiate a &#x2018;different&#x2019; Class 1 or Class 2 uORF. A given transcript can (and frequently does) contain multiple Class 3 uORFs, in the same or different frames. I consider all of these as separate individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Three classes of upstream open reading frames. The 5&#x2032; untranslated sequence (5UT) can contain no ATG in any frame (top), resulting in no upstream open reading frame (&#x2018;uORF&#x2019;). It can contain one or more ATGs in frame with the main coding sequence (&#x2018;Reference CDS&#x2019;), without an intervening stop codon (Class 1). Class 2 is the same as Class 1 but in a different reading frame; in general the Class 2 uORF will terminate shortly after entering the main CDS out-of-frame. Class 3 initiates (in any frame) and terminates within the 5UT. Note that I consider a maximum of one Class 1 and one Class 2 uORF per transcript, although these uORFs can contain internal ATGs that could in principle initiate a &#x2018;different&#x2019; Class 1 or Class 2 uORF. A given transcript can (and frequently does) contain multiple Class 3 uORFs, in the same or different frames. I consider all of these as separate individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F1.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/2/435/F1.medium.gif" width="440" height="275"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467456" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Three classes of upstream open reading frames. The 5′ untranslated sequence (5UT) can contain no ATG in any frame (top), resulting in no upstream open reading frame (‘uORF’). It can contain one or more ATGs in frame with the main coding sequence (‘Reference CDS’), without an intervening stop codon (Class 1). Class 2 is the same as Class 1 but in a different reading frame; in general the Class 2 uORF will terminate shortly after entering the main CDS out-of-frame. Class 3 initiates (in any frame) and terminates within the 5UT. Note that I consider a maximum of one Class 1 and one Class 2 uORF per transcript, although these uORFs can contain internal ATGs that could in principle initiate a ‘different’ Class 1 or Class 2 uORF. A given transcript can (and frequently does) contain multiple Class 3 uORFs, in the same or different frames. I consider all of these as separate individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">In a broad range of organisms, ATG frequency is significantly reduced in 5′UΤ compared to CDS (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Zur and Tuller 2013</a>). In the <em>Chlamydomonas</em> annotation, the frequency of ATGs in-frame with CDS is 40% lower in 5UT than in CDS (excluding the reference initiator itself), while the frequencies of out-of-frame ATGs is 73% higher (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These departures are largely due to deviations from random expectations specifically in the CDS (where the random model is based on overall dinucleotide frequencies). Overall, though, ATG frequency in 5UT and CDS are nearly identical (0.029 <em>vs.</em> 0.028), in contrast to results in other organisms (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Zur and Tuller 2013</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467690/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467690/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Frequency of ATG in combined coding sequence (CDS), 5UT sequence (5UT), and expected random frequency with length-matched sequence sets with the same dinucleotide frequency as in genomic CDS or 5UT</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Class 1 uORFs are longer than expected for random sequence</h3><p id="p-15">If uORFs have biological relevance, this could be reflected in statistical sequence differentiation from randomized controls. A ‘scrambled 5UT-ome’ was constructed by individually randomizing sequence of each 5UT sequence, thus preserving nucleotide composition but not sequence. This control set of uORFs will reflect sequence-independent consequences of the nucleotide composition and length distribution of the reference 5UTs. Because dinucleotide frequencies can vary between different genomes or different regions of the same genome, independent of single nucleotide frequencies, a second control set was constructed by determining dinucleotide frequencies in the 5UT-ome overall, and making random sequences matched for lengths and overall dinucleotide frequencies. Additional controls were derived by ‘mutagenizing’ the annotated 5UTs by randomly replacing on average one tenth, one fifth, or one half of the nucleotides in each 5UT with another nucleotide chosen based on the overall 5UT nucleotide frequency. These controls (especially the 1/10 or 1/5 mutagenesis) will lack only strongly sequence-dependent features when compared to the reference 5UTs.</p><p id="p-16">Results of these comparisons with respect to number and length of uORFs were very clear, and strikingly different for the three classes. For classes 2 and 3, uORF lengths were essentially identical for the reference and the random or mutated controls (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In contrast, class 1 uORFs from the genome were significantly longer than in any of the controls. Mutation of one in ten nucleotides was almost as effective at eliminating sequence dependence as complete randomization, suggesting a high degree of sequence dependence in the real sequence. The numbers of class 1 and class 3 uORFs were approximately similar in reference and controls (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>); however, class 2 uORFs doubled in abundance in the randomized control compared to the reference. Note that the random expectation is that class 2 should be twice as abundant as class 1, since there are two ‘wrong’ frames and one ‘right’ frame.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Statistics of uORF length and number in the genome, and in partially or fully randomized controls. (A) Class 1 but not Class 2 or Class 3 uORFs are longer than the random expectation. Cumulative length distribution of three classes of uORFs for the genome (green) and for randomized or mutagenized controls (three replicates of each). &#x2018;Scramble&#x2019;: each 5UT sequence was randomized (yellow); &#x2018;Rand(dinuc)&#x2019;: sequences of the same length as the real 5-UT sequences were constructed with identical dinucleotide frequencies to the overall &#x2018;5UT-ome&#x2019; (red); Mut 0.1/0.2/0.5: the set of 5UT sequences was &#x2018;mutagenized&#x2019; by replacing one in 10, one in five, or one in two nucleotides in each 5UT with random selections from the overall nucleotide frequency distribution of the complete collection of 5UT sequences. (Note: the randomized distribution for all classes is essentially identical to the class 3 length distribution for the actual genomic Class 3 sequences.) The indicated box in each graph is blown up at right to show high reproducibility of randomized results for the three replicates. (B) Total numbers of uORFs with and without randomization. The small red bar represents a hypothetical standard deviation based on the assumption that numbers in each category are Poisson-distributed (square root of the number observed). Stars represent P-values for a t-test comparing each randomization to the genome, using these standard deviations: * P &lt; 0.05; ** P &lt; 0.01; *** P &lt; 0.001). Randomizing by scrambling (shown) or by dinucleotide frequencies gave very similar results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Statistics of uORF length and number in the genome, and in partially or fully randomized controls. (A) Class 1 but not Class 2 or Class 3 uORFs are longer than the random expectation. Cumulative length distribution of three classes of uORFs for the genome (green) and for randomized or mutagenized controls (three replicates of each). &#x2018;Scramble&#x2019;: each 5UT sequence was randomized (yellow); &#x2018;Rand(dinuc)&#x2019;: sequences of the same length as the real 5-UT sequences were constructed with identical dinucleotide frequencies to the overall &#x2018;5UT-ome&#x2019; (red); Mut 0.1/0.2/0.5: the set of 5UT sequences was &#x2018;mutagenized&#x2019; by replacing one in 10, one in five, or one in two nucleotides in each 5UT with random selections from the overall nucleotide frequency distribution of the complete collection of 5UT sequences. (Note: the randomized distribution for all classes is essentially identical to the class 3 length distribution for the actual genomic Class 3 sequences.) The indicated box in each graph is blown up at right to show high reproducibility of randomized results for the three replicates. (B) Total numbers of uORFs with and without randomization. The small red bar represents a hypothetical standard deviation based on the assumption that numbers in each category are Poisson-distributed (square root of the number observed). Stars represent &lt;em&gt;P&lt;/em&gt;-values for a &lt;em&gt;t&lt;/em&gt;-test comparing each randomization to the genome, using these standard deviations: * &lt;em&gt;P&lt;/em&gt; &lt; 0.05; ** &lt;em&gt;P&lt;/em&gt; &lt; 0.01; *** &lt;em&gt;P&lt;/em&gt; &lt; 0.001). Randomizing by scrambling (shown) or by dinucleotide frequencies gave very similar results.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F2.medium.gif" width="254" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/2/435/F2.medium.gif" width="254" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467540" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Statistics of uORF length and number in the genome, and in partially or fully randomized controls. (A) Class 1 but not Class 2 or Class 3 uORFs are longer than the random expectation. Cumulative length distribution of three classes of uORFs for the genome (green) and for randomized or mutagenized controls (three replicates of each). ‘Scramble’: each 5UT sequence was randomized (yellow); ‘Rand(dinuc)’: sequences of the same length as the real 5-UT sequences were constructed with identical dinucleotide frequencies to the overall ‘5UT-ome’ (red); Mut 0.1/0.2/0.5: the set of 5UT sequences was ‘mutagenized’ by replacing one in 10, one in five, or one in two nucleotides in each 5UT with random selections from the overall nucleotide frequency distribution of the complete collection of 5UT sequences. (Note: the randomized distribution for all classes is essentially identical to the class 3 length distribution for the actual genomic Class 3 sequences.) The indicated box in each graph is blown up at right to show high reproducibility of randomized results for the three replicates. (B) Total numbers of uORFs with and without randomization. The small red bar represents a hypothetical standard deviation based on the assumption that numbers in each category are Poisson-distributed (square root of the number observed). Stars represent <em>P</em>-values for a <em>t</em>-test comparing each randomization to the genome, using these standard deviations: * <em>P</em> &lt; 0.05; ** <em>P</em> &lt; 0.01; *** <em>P</em> &lt; 0.001). Randomizing by scrambling (shown) or by dinucleotide frequencies gave very similar results.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">The effectiveness of ‘mutagenesis’, even at a 10% substitution rate, at reducing lengths of class 1 uORFs was primarily due to gain of internal termination codons; this mechanism preferentially removed longer class 1 uORFs from the compilation (by conversion to class 3). Class 1 uORFs were also lost or shortened due to loss of the initiator ATG. New class 1 uORFs appeared upon mutagenesis, but these were generally shorter, accounting for decrease in length with no change in overall number.</p><p id="p-19">These results suggest sequence-dependent constraints that (1) preserve class 1 uORFs at significantly longer than expected by chance and (2) suppress the numbers of class 2 uORFs to about half the level expected by chance. Class 3 uORF numbers and lengths are strikingly well predicted by nothing more than 5UT nucleotide composition (mono- or dinucleotide-based) and length distribution, and thus exhibit no sequence dependence detectable by this bulk approach. The close statistical correspondence of the genomic class 3 uORFs and those constructed from randomized sequence suggests that most class 3 uORFs are effectively neutral sequence [though statistically significant increases in class 3 uORF numbers in randomized controls (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) suggest purifying selection over at least a subset]. The suppression of class 2 uORF number relative to random controls suggests that class 2 uORFs, in contrast, are subject to significant purifying selection. This could be understood based on the scanning model for translation initiation since initiation from a class 2 AUG would block even post-termination reinitiation at the reference AUG. This is because scanning is probably generally (though perhaps not exclusively) unidirectional (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Hinnebusch 2011</a>), and class 2 termination occurs 3′ to the reference initiation site (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Thus class 2 AUGs could be particularly damaging to expression of the main CDS.</p></div><div id="sec-5" class="subsection"><h3>A Chlamydomonas Kozak consensus</h3><p id="p-20">These observations raise issues with respect to the standard translation model. Since the reference initiation ATG generally starts translation of a long peptide, which is frequently conserved across species, it is highly unlikely that Class 2 and Class 3 ATGs are exclusive sites of initiation. This suggests a high level of selectivity, since nearly two thirds of transcripts are assigned class 2 and/or class 3 uORFs—so if the reference ATG is in fact the one used <em>in vivo</em>, multiple 5′ ATGs must fail either initiation or ribosome disengagement in a majority of transcripts—thus, the scanning model would be the exception rather than the rule.</p><p id="p-21">There are two obvious escapes from this problem. The simplest is if the annotated transcription start site is misplaced at a position 5′ of the real start. For a substantial subset of Phytozome gene models, there is evidence from EST sequence that the transcript does indeed cover some or all of the proposed 5UT, so this is unlikely to be the entire explanation.</p><p id="p-22">Another escape would be strong sequence constraints suppressing initiation at uORFs. This is a known mechanism in other eukaryotes, where absence of a ‘Kozak’ consensus sequence can allow skipping of a 5′ AUG (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Kozak 1989</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Hinnebusch 2011</a>). Using the Weblogo calculation (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Crooks <em>et al.</em> 2004</a>) with the complete set of <em>Chlamydomonas</em> reference initiator ATGs yielded a consensus with striking similarity to the Kozak consensus found for human mRNAs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Kozak-like consensus sequence around Chlamydomonas reference initiator ATGs. All 19,228 sequences were fed to the online WebLogo tool (http://weblogo.threeplusone.com) (Crooks et al., 2004). A comparable plot for human mRNAs was downloaded from Wikipedia (https://en.wikipedia.org/wiki/ File:Human_Kozak_context._Version_2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Kozak-like consensus sequence around &lt;em&gt;Chlamydomonas&lt;/em&gt; reference initiator ATGs. All 19,228 sequences were fed to the online WebLogo tool (http://weblogo.threeplusone.com) (Crooks &lt;em&gt;et al.&lt;/em&gt;, 2004). A comparable plot for human mRNAs was downloaded from Wikipedia (https://en.wikipedia.org/wiki/ File:Human_Kozak_context._Version_2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F3.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/2/435/F3.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467614" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Kozak-like consensus sequence around <em>Chlamydomonas</em> reference initiator ATGs. All 19,228 sequences were fed to the online WebLogo tool (<a href="http://weblogo.threeplusone.com">http://weblogo.threeplusone.com</a>) (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Crooks <em>et al.</em>, 2004</a>). A comparable plot for human mRNAs was downloaded from Wikipedia (<a href="https://en.wikipedia.org/wiki/">https://en.wikipedia.org/wiki/</a> File:Human_Kozak_context._Version_2).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">This consensus allowed construction of an ATG context ‘Kozak’ score based on agreement with the consensus and the information content of the position. The distribution of Kozak scores for reference ATGs to scores for 100,000 random sequences composed with nucleotide composition of the overall 5UT-ome of <em>Chlamydomonas</em>. There was a clear separation between unimodal score peaks (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The result was the same with randomization by mono- or dinucleotide frequencies.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Class 1 uORFs, but not Class 2 or Class 3 uORFs, show significant agreement to the Kozak consensus. (A) Reference, random, and uORF agreement with the consensus. A &#x2018;Kozak score&#x2019; was defined as the sum of bits corresponding to the observed nucleotides at each position surrounding the ATG (Figure 3). Top left: comparison of scores of reference initiator ATGs (green) to scores of 100,000 random sequences composed with the nucleotide frequency of 5UT (red). Remaining panels: uORFs (blue) compared to reference and random distributions. (B) Projection of uORFs onto space spanned by reference initiators and random sequence. Define Matrix A = [distribution of reference;distribution of random]; vector C = distribution of uORF; then [c1 c2] = (ATA)-1 ATC gives the least-squares best-fit solution for c1 &#xD7; reference distribution + c2 * random distribution &#x2248; C (Strang 2009). The best fit is the dotted black line; weights and proportion of variance explained (PVE) are indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Class 1 uORFs, but not Class 2 or Class 3 uORFs, show significant agreement to the Kozak consensus. (A) Reference, random, and uORF agreement with the consensus. A &#x2018;Kozak score&#x2019; was defined as the sum of bits corresponding to the observed nucleotides at each position surrounding the ATG (Figure 3). Top left: comparison of scores of reference initiator ATGs (green) to scores of 100,000 random sequences composed with the nucleotide frequency of 5UT (red). Remaining panels: uORFs (blue) compared to reference and random distributions. (B) Projection of uORFs onto space spanned by reference initiators and random sequence. Define Matrix A = [distribution of reference;distribution of random]; vector C = distribution of uORF; then [c1 c2] = (A&lt;sup&gt;T&lt;/sup&gt;A)&lt;sup&gt;-1&lt;/sup&gt; A&lt;sup&gt;T&lt;/sup&gt;C gives the least-squares best-fit solution for c1 &#xD7; reference distribution + c2 * random distribution &#x2248; C (Strang 2009). The best fit is the dotted black line; weights and proportion of variance explained (PVE) are indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F4.medium.gif" width="285" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/2/435/F4.medium.gif" width="285" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467527" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Class 1 uORFs, but not Class 2 or Class 3 uORFs, show significant agreement to the Kozak consensus. (A) Reference, random, and uORF agreement with the consensus. A ‘Kozak score’ was defined as the sum of bits corresponding to the observed nucleotides at each position surrounding the ATG (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Top left: comparison of scores of reference initiator ATGs (green) to scores of 100,000 random sequences composed with the nucleotide frequency of 5UT (red). Remaining panels: uORFs (blue) compared to reference and random distributions. (B) Projection of uORFs onto space spanned by reference initiators and random sequence. Define Matrix A = [distribution of reference;distribution of random]; vector C = distribution of uORF; then [c1 c2] = (A<sup>T</sup>A)<sup>-1</sup> A<sup>T</sup>C gives the least-squares best-fit solution for c1 × reference distribution + c2 * random distribution ≈ C (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Strang 2009</a>). The best fit is the dotted black line; weights and proportion of variance explained (PVE) are indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The distribution of scores for reference initiators was also separated from randomized sequence at the nucleotide distribution of CDS, and from the distribution of scores for internal ATGs (3′ to the reference). Interestingly, the ATG immediately 3′ to the reference ATG (but not further-internal ATGs) had a distribution of scores slightly but significantly lower than the random expectation, possibly suggesting the importance of discriminating the first from the second potential initiator (figures generated by supplemental MATLAB code).</p></div><div id="sec-6" class="subsection"><h3>Distribution of Kozak consensus scores suggests that some Class 1 uORFs may be translated</h3><p id="p-27">Class 2 and Class 3 ATGs had a distribution of scores nearly identical to the random sequence control (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Thus, if the Kozak consensus enhances initiation efficiency, many class 2 and 3 ATGs may be skipped in favor of the downstream reference ATGs.</p><p id="p-28">Class 1 ATGs had a very different distribution of Kozak consensus scores, which resembled a bimodal mixture of score distributions similar to the randomized control, and similar to the reference ATGs. A simple linear algebra calculation (projection of the data onto a 2-dimensional space spanned by the randomized control and the reference ATG distributions) yielded the optimal mixture: a 51:49 combination of these distributions yielded a very good fit to the class 3 distribution (99% of variance explained by this linear fit) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-29">This observation suggests the hypothesis that class 1 uORFs are heterogeneous. Half of them might be inefficiently translated, thus resembling the Class 2 and Class 3 uORFs. Half, on the other hand, might be translated either as alternative or as the exclusive <em>in vivo</em> initiations. Such initiation would result in a peptide with an N-terminal uORF fused to the reference peptide.</p></div><div id="sec-7" class="subsection"><h3>Test of translation-dependent evolutionary selection on class 1 uORFs</h3><p id="p-30">If class 1 uORFs are translated and the resulting N-terminal extensions are under evolutionary constraint, then the N-terminal sequence could extend the alignment of the predicted peptide, when compared to other organisms. <em>Volvox</em> is a multicellular species with a recent common ancestor with <em>Chlamydomonas</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Herron <em>et al.</em> 2009</a>). Many <em>Chlamydomonas</em> peptides have highly similar orthologs in <em>Volvox</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Prochnik <em>et al.</em> 2010</a>). However, neutral nucleotide sequence divergence between <em>Volvox</em> and <em>Chlamydomonas</em> is &gt;50%, based on substitution rates at neutral positions in highly conserved proteins (unambiguously alignable without gaps: actin, tubulin, CDKB). Thus sequence not under selection for its protein coding potential should rapidly lose any recognizable BLASTP (protein) similarity, due to divergence and especially to fragmentation from gain of termination codons and loss of potential initiator ATGs. (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a> showed that a 50% divergence rate was equivalent to full randomization for completely eliminating enhanced lengths of class 1 uORFs.) In contrast, if a sequence is translated and the peptide product under selection, then BLASTP similarity will be retained.</p><p id="p-31">BLASTP scores of the <em>Volvox</em> proteome were determined against <em>Chlamydomonas</em> reference peptides, reference peptides with class 1 uORF N-terminal extensions, and controls of reference peptides with class 1 uORF extensions that were scrambled at the level of predicted peptide (two scrambling replicates). This scrambled control takes into account possible BLASTP score improvement due to simple-sequence features (<em>e.g.</em>, poly-Pro aligning similarly with and without scrambling with Pro-rich N-termini in subject proteins). The test statistic was the maximum BLASTP score due to the uORF extension compared to scrambled controls. (This score is in units of bits, which are logarithmic; thus the arithmetical difference in scores is an appropriate indicator of differential effect of the uORF). In many cases the real uORFs, but not the scrambled controls, increased the score by up to many hundreds of bits. Quantitative comparison of cumulative results suggests that at least 25–30% of class 1 uORFs are under selection for translated sequence content (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This is likely a lower bound for the proportion of class 1 uORFs that are translated <em>in vivo</em>: first, because many proteins lack BLASTP-detectable similarity at their N-termini; second, because the comparable stretch in the <em>Volvox</em> annotation might have also been incorrectly assigned to an untranslated uORF.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Many class 1 uORFs encode evolutionarily conserved N-terminal extensions. (A) Schematic of the test. BLASTP alignments to the Volvox proteome were carried out for each class 1 transcript using four different versions: the reference transcript; the reference N-terminally extended by the class 1 uORF; and the reference transcript N-terminally extended by scrambled versions of the class 1 uORF peptide. Sequence-dependent BLASTP score improvement (S1 &gt; S2, S2 &#x2245; S3) was taken to argue for evolutionary conservation of class 1 CDS. (B) The differences between maximal BLASTP scores of the reference peptide and the N-terminally extended versions (class 1 uORF peptide, or the scrambled uORF peptide) for all class 1 transcripts are plotted as cumulative distributions. The divergence of the uORF peptide from the scrambled versions at a score of &gt;4 and about 70%&#x2013;75% of transcripts (divergence point marked with green arrow) indicates sequence-dependent score increase (that is, score increase specific to the uORF and not to scrambled versions) in 25%&#x2013;30% of class 1 transcripts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Many class 1 uORFs encode evolutionarily conserved N-terminal extensions. (A) Schematic of the test. BLASTP alignments to the &lt;em&gt;Volvox&lt;/em&gt; proteome were carried out for each class 1 transcript using four different versions: the reference transcript; the reference N-terminally extended by the class 1 uORF; and the reference transcript N-terminally extended by scrambled versions of the class 1 uORF peptide. Sequence-dependent BLASTP score improvement (S1 &gt; S2, S2 &#x2245; S3) was taken to argue for evolutionary conservation of class 1 CDS. (B) The differences between maximal BLASTP scores of the reference peptide and the N-terminally extended versions (class 1 uORF peptide, or the scrambled uORF peptide) for all class 1 transcripts are plotted as cumulative distributions. The divergence of the uORF peptide from the scrambled versions at a score of &gt;4 and about 70%&#x2013;75% of transcripts (divergence point marked with green arrow) indicates sequence-dependent score increase (that is, score increase specific to the uORF and not to scrambled versions) in 25%&#x2013;30% of class 1 transcripts.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F5.medium.gif" width="276" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/2/435/F5.medium.gif" width="276" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467484" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Many class 1 uORFs encode evolutionarily conserved N-terminal extensions. (A) Schematic of the test. BLASTP alignments to the <em>Volvox</em> proteome were carried out for each class 1 transcript using four different versions: the reference transcript; the reference N-terminally extended by the class 1 uORF; and the reference transcript N-terminally extended by scrambled versions of the class 1 uORF peptide. Sequence-dependent BLASTP score improvement (S1 &gt; S2, S2 ≅ S3) was taken to argue for evolutionary conservation of class 1 CDS. (B) The differences between maximal BLASTP scores of the reference peptide and the N-terminally extended versions (class 1 uORF peptide, or the scrambled uORF peptide) for all class 1 transcripts are plotted as cumulative distributions. The divergence of the uORF peptide from the scrambled versions at a score of &gt;4 and about 70%–75% of transcripts (divergence point marked with green arrow) indicates sequence-dependent score increase (that is, score increase specific to the uORF and not to scrambled versions) in 25%–30% of class 1 transcripts.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">A few examples of the aligned sequences resulting from these BLASTP comparisons are presented in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. It is clear in these cases that the uORF encodes evolutionarily conserved sequence, relevant to the function of the peptide. The data comprise a continuous series from such obvious cases to addition of only a few amino acids, with marginal or no effect on BLASTP scores.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of BLASTP score improvement by class 1 uORF N-terminal extensions. The N-terminal sequence of the best Volvox BLASTP hit to the reference CDS is shown, starting from the indicated &#x2018;Reference initiator,&#x2019; along with the extended alignment from the class 1 uORF starting at the &#x2018;uORF initiator.&#x2019; Regions of alignment are sketched in green and BLASTP scores indicated at left. (A) Ankyrin repeat-containing protein. Two ankyrin repeats (bold in alignment below) are found in the N-terminal extension, and two more in the reference CDS. (B) Cdh1. Cdh1 is known to be regulated by cyclin-dependent-kinase phosphorylation (Zachariae et al., 1998) (minimal consensus S/T-P; extended consensus S/T-P-x-R/K). Three out of seven such sites are in the class 1 uORF N-terminal extension. (C) Dynamin-homologous protein, with characteristic GTP-binding domain of dynamins encoded in the class 1 uORF extension." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of BLASTP score improvement by class 1 uORF N-terminal extensions. The N-terminal sequence of the best &lt;em&gt;Volvox&lt;/em&gt; BLASTP hit to the reference CDS is shown, starting from the indicated &#x2018;Reference initiator,&#x2019; along with the extended alignment from the class 1 uORF starting at the &#x2018;uORF initiator.&#x2019; Regions of alignment are sketched in green and BLASTP scores indicated at left. (A) Ankyrin repeat-containing protein. Two ankyrin repeats (bold in alignment below) are found in the N-terminal extension, and two more in the reference CDS. (B) Cdh1. Cdh1 is known to be regulated by cyclin-dependent-kinase phosphorylation (Zachariae &lt;em&gt;et al.&lt;/em&gt;, 1998) (minimal consensus S/T-P; extended consensus S/T-P-x-R/K). Three out of seven such sites are in the class 1 uORF N-terminal extension. (C) Dynamin-homologous protein, with characteristic GTP-binding domain of dynamins encoded in the class 1 uORF extension.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F6.medium.gif" width="162" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/2/435/F6.medium.gif" width="162" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467561" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Examples of BLASTP score improvement by class 1 uORF N-terminal extensions. The N-terminal sequence of the best <em>Volvox</em> BLASTP hit to the reference CDS is shown, starting from the indicated ‘Reference initiator,’ along with the extended alignment from the class 1 uORF starting at the ‘uORF initiator.’ Regions of alignment are sketched in green and BLASTP scores indicated at left. (A) Ankyrin repeat-containing protein. Two ankyrin repeats (bold in alignment below) are found in the N-terminal extension, and two more in the reference CDS. (B) Cdh1. Cdh1 is known to be regulated by cyclin-dependent-kinase phosphorylation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Zachariae <em>et al.</em>, 1998</a>) (minimal consensus S/T-P; extended consensus S/T-P-x-R/K). Three out of seven such sites are in the class 1 uORF N-terminal extension. (C) Dynamin-homologous protein, with characteristic GTP-binding domain of dynamins encoded in the class 1 uORF extension.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">A complete tabulation of the results of the BLASTP analysis is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023119/-/DC1/TableS1.xls">Table S1</a>.</p></div><div id="sec-8" class="subsection"><h3>Correlation of high Kozak consensus score and probable in vivo translation for Class 1 uORFs</h3><p id="p-36">An apparent bimodal distribution of Kozak consensus scores among class 1 uORFs led to the suggestion above that the higher-scoring class 1 uORFs might be preferentially translated. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a> shows Kozak score distribution of the class 1 ATG and the reference ATG from the upper ∼25% of BLASTP improvement, compared to the remainder. The high-scoring cases had Kozak scores indistinguishable from overall reference ATG initiators, and interestingly had significantly better scores than the reference initiators from the same set of transcripts. The lower 75% were essentially indistinguishable from the bulk (compare <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a> to <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>, top right).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Class 1 uORFs with high BLASTP scores have high &#x2018;Kozak&#x2019; scores and long CDSs. (A) Above: Kozak scores for the class 1 uORF ATG and the reference ATG, for all transcripts in the high BLASTP-improvement score class (see Figure 5). Below: Kozak scores for the same ATGs in the low BLASTP improvement score class. Insets: probability by t-test in the high BLASTP class that reference ATGs have average scores greater than or equal to uORF ATGs (above), probability in the low BLASTP class that reference ATGs have average scores less than or equal to uORF ATGs (below). (B) High BLASTP-score class 1 uORFs are longer than average class 1 uORFs. (C) The length distribution of the total pool of class 1 uORFs can be accounted for as a 0.59:0.41 sum of the high BLASTP-score subset distribution and the Class 2 uORF distribution, by least-squares fitting as in Figure 4B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Class 1 uORFs with high BLASTP scores have high &#x2018;Kozak&#x2019; scores and long CDSs. (A) Above: Kozak scores for the class 1 uORF ATG and the reference ATG, for all transcripts in the high BLASTP-improvement score class (see Figure 5). Below: Kozak scores for the same ATGs in the low BLASTP improvement score class. Insets: probability by &lt;em&gt;t&lt;/em&gt;-test in the high BLASTP class that reference ATGs have average scores greater than or equal to uORF ATGs (above), probability in the low BLASTP class that reference ATGs have average scores less than or equal to uORF ATGs (below). (B) High BLASTP-score class 1 uORFs are longer than average class 1 uORFs. (C) The length distribution of the total pool of class 1 uORFs can be accounted for as a 0.59:0.41 sum of the high BLASTP-score subset distribution and the Class 2 uORF distribution, by least-squares fitting as in Figure 4B.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F7.medium.gif" width="162" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/2/435/F7.medium.gif" width="162" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-37" class="first-child">Class 1 uORFs with high BLASTP scores have high ‘Kozak’ scores and long CDSs. (A) Above: Kozak scores for the class 1 uORF ATG and the reference ATG, for all transcripts in the high BLASTP-improvement score class (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Below: Kozak scores for the same ATGs in the low BLASTP improvement score class. Insets: probability by <em>t</em>-test in the high BLASTP class that reference ATGs have average scores greater than or equal to uORF ATGs (above), probability in the low BLASTP class that reference ATGs have average scores less than or equal to uORF ATGs (below). (B) High BLASTP-score class 1 uORFs are longer than average class 1 uORFs. (C) The length distribution of the total pool of class 1 uORFs can be accounted for as a 0.59:0.41 sum of the high BLASTP-score subset distribution and the Class 2 uORF distribution, by least-squares fitting as in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">This finding supports the ideas that the Kozak consensus is relevant to translation efficiency, and that many class 1 uORFs are translated <em>in vivo</em>, either as alternative or as the sole initiation codons.</p><p id="p-39">Class 1 uORFs are significantly longer on average than either class 2 uORFs, or class 1 uORFs from ‘mutagenized’ or randomized sequence (see above). This length effect was markedly enhanced for the class 1 uORFs in the upper 25% of BLASTP improvement (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). If class 1 uORFs are not translated, there is no obvious reason for any length difference compared to class 2 uORFs, since they differ only in translational frame relative to the reference (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The total class 1 CDS length distribution could be closely approximated as a linear combination of the class 2 distribution and the high BLASTP-scoring class 1 distribution with ∼40:60 mixture (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>). This split is similar to the 49:51 split in the fit to Kozak consensus scores; both results suggest that around half of class 1 uORFs are translated efficiently. (Note that if the class 1 uORFs are a heterogeneous 50:50 mixture of neutral and functional sequences, then considering only the neutral class, the lengths and lack of sensitivity to mutagenesis and randomization becomes very similar to class 2; their numbers become just about half that of the class 2 uORFs, as expected given three reading frames.)</p></div><div id="sec-9" class="subsection"><h3>Weighted predictions for translation of class 1 uORFs</h3><p id="p-40">The results above suggest that approximately 50% of class 1 uORFs are alternative or exclusive sites of <em>in vivo</em> initiation. Where a strong improvement in BLASTP score could be detected by including the uORF N-terminal extension, these transcripts can be identified directly (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Such cases are a minority, though, and it would be desirable to have a quantitative estimate of the probability of translation initiation for all ∼4000 class 1 uORFs.</p><p id="p-41">Class 1 uORFs that are more likely to be translated have two sequence features (independent of <em>Volvox</em> alignments): the uORF CDS is longer, and the Kozak consensus is stronger. Neither feature is quantitatively strong enough to form a digital classifier. The two-dimensional differentiation between uORF classes 1, 2, and 3, and the ‘high BLASTP’ subclass of uORF class 1 was a stronger separator (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). The surface for class 1 could be almost exactly modeled as a 50:50 split between the class 1-high BLASTP subclass (representing efficiently translated uORFs) and class 2 (representing presumably poorly translated uORFs) (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). This allowed construction of a simple Bayesian test for the likelihood that a class 1 uORF is translated based on its CDS length and its Kozak score, using as training sets the high BLASTP class 1 subset (positive examples) and class 2 uORFs (negative examples). The strength of discrimination is illustrated in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8C</a>; the receiver–operator characteristic (ROC) curve shows the false positive/true positive relationship for various probability cutoffs. While this test is still certainly not definitive in any individual case, it does provide an empirically based estimate of the probability of translation in the complete set of class 1 uORFs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023119/-/DC1/TableS1.xls">Table S1</a>). In advance of more definitive empirical evidence, these probabilistic classifications may be helpful as an adjunct to the (necessarily) digital summary in Phytozome that some sequence ‘is’ or ‘is not’ part of the CDS.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two-dimensional accounting for uORF variability. (A) Surface of proportion of uORFs over a two-dimensional grid of Kozak score (exponentiated) and log CDS length. These transformations were chosen because the Kozak score is based on an essentially logarithmic information scale (Figure 3), whereas CDS lengths can be conceptualized as due to exponential decay from finite probability of hitting a stop codon (class 3) or the beginning of the CDS (classes 1 and 2). Classes 2 and 3 have broad peaks with low Kozak scores (left axis) and shorter CDSs (right axis). The high BLASTP-score subclass of class 1 uORFs (lower right) has a sharp peak at high Kozak score and longer CDSs. The complete pool of class 1 uORFs appears heterogeneous, with a peak similar to the high BLASTP subset and a shoulder similar to the distributions of classes 2 and 3. (B) Least-squares accounting for the two-dimensional class 1 uORF distribution as a sum of 0.50 high BLASTP class 1 uORF subset and 0.48 class 2 uORF (matrix calculation as in Figure 4B); 99% of variation is explained. (C) A Bayesian test for translation of class 1 uORFs. Call T the event of translation; K a given Kozak score; L a given length. We want to know P(T | K&amp;L). Bayes theorem says this is equal to: P(K&amp;L | T) &#xD7; P(T) / [P(K&amp;L | T) &#xD7; P(T) + P(K&amp;L | &#x223C;T) &#xD7; P(&#x223C;T)]. Assuming &#x223C;50:50 split of translated and untranslated class 1 uORFs (Figure 4C, Figure 7C, and Figure 8B) [i.e., P(T) = P(&#x223C;T)], this simplifies to P(T | K&amp;L) = P(K&amp;L | T) / [P(K&amp;L | T) + P(K&amp;L | &#x223C;T)]. K&amp;L specifies a point on the grid; P(K&amp;L | T) is proportional to the height of the high-BLASTP surface over this point, while P(K&amp;L | &#x223C;T) is proportional to the height of the class 2 surface over this point. So this ratio calculated over the whole grid provides a probability estimate. This surface is graphed on the left. On the right is the ROC curve given this surface, for varying cutoffs of nominal Bayesian probability, with the assumption that all translated class 1 uORFs will have sequence properties similar to the high BLASTP score positive training set, while all untranslated class 1 uORFs will be similar to (the presumably untranslated) class 2 uORFs negative training set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two-dimensional accounting for uORF variability. (A) Surface of proportion of uORFs over a two-dimensional grid of Kozak score (exponentiated) and log CDS length. These transformations were chosen because the Kozak score is based on an essentially logarithmic information scale (Figure 3), whereas CDS lengths can be conceptualized as due to exponential decay from finite probability of hitting a stop codon (class 3) or the beginning of the CDS (classes 1 and 2). Classes 2 and 3 have broad peaks with low Kozak scores (left axis) and shorter CDSs (right axis). The high BLASTP-score subclass of class 1 uORFs (lower right) has a sharp peak at high Kozak score and longer CDSs. The complete pool of class 1 uORFs appears heterogeneous, with a peak similar to the high BLASTP subset and a shoulder similar to the distributions of classes 2 and 3. (B) Least-squares accounting for the two-dimensional class 1 uORF distribution as a sum of 0.50 high BLASTP class 1 uORF subset and 0.48 class 2 uORF (matrix calculation as in Figure 4B); 99% of variation is explained. (C) A Bayesian test for translation of class 1 uORFs. Call T the event of translation; K a given Kozak score; L a given length. We want to know P(T | K&amp;L). Bayes theorem says this is equal to: P(K&amp;L | T) &#xD7; P(T) / [P(K&amp;L | T) &#xD7; P(T) + P(K&amp;L | &#x223C;T) &#xD7; P(&#x223C;T)]. Assuming &#x223C;50:50 split of translated and untranslated class 1 uORFs (Figure 4C, Figure 7C, and Figure 8B) [&lt;em&gt;i.e.&lt;/em&gt;, P(T) = P(&#x223C;T)], this simplifies to P(T | K&amp;L) = P(K&amp;L | T) / [P(K&amp;L | T) + P(K&amp;L | &#x223C;T)]. K&amp;L specifies a point on the grid; P(K&amp;L | T) is proportional to the height of the high-BLASTP surface over this point, while P(K&amp;L | &#x223C;T) is proportional to the height of the class 2 surface over this point. So this ratio calculated over the whole grid provides a probability estimate. This surface is graphed on the left. On the right is the ROC curve given this surface, for varying cutoffs of nominal Bayesian probability, with the assumption that all translated class 1 uORFs will have sequence properties similar to the high BLASTP score positive training set, while all untranslated class 1 uORFs will be similar to (the presumably untranslated) class 2 uORFs negative training set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-8.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-8.medium.gif" width="440" height="316"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467678" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two-dimensional accounting for uORF variability. (A) Surface of proportion of uORFs over a two-dimensional grid of Kozak score (exponentiated) and log CDS length. These transformations were chosen because the Kozak score is based on an essentially logarithmic information scale (Figure 3), whereas CDS lengths can be conceptualized as due to exponential decay from finite probability of hitting a stop codon (class 3) or the beginning of the CDS (classes 1 and 2). Classes 2 and 3 have broad peaks with low Kozak scores (left axis) and shorter CDSs (right axis). The high BLASTP-score subclass of class 1 uORFs (lower right) has a sharp peak at high Kozak score and longer CDSs. The complete pool of class 1 uORFs appears heterogeneous, with a peak similar to the high BLASTP subset and a shoulder similar to the distributions of classes 2 and 3. (B) Least-squares accounting for the two-dimensional class 1 uORF distribution as a sum of 0.50 high BLASTP class 1 uORF subset and 0.48 class 2 uORF (matrix calculation as in Figure 4B); 99% of variation is explained. (C) A Bayesian test for translation of class 1 uORFs. Call T the event of translation; K a given Kozak score; L a given length. We want to know P(T | K&amp;L). Bayes theorem says this is equal to: P(K&amp;L | T) &#xD7; P(T) / [P(K&amp;L | T) &#xD7; P(T) + P(K&amp;L | &#x223C;T) &#xD7; P(&#x223C;T)]. Assuming &#x223C;50:50 split of translated and untranslated class 1 uORFs (Figure 4C, Figure 7C, and Figure 8B) [i.e., P(T) = P(&#x223C;T)], this simplifies to P(T | K&amp;L) = P(K&amp;L | T) / [P(K&amp;L | T) + P(K&amp;L | &#x223C;T)]. K&amp;L specifies a point on the grid; P(K&amp;L | T) is proportional to the height of the high-BLASTP surface over this point, while P(K&amp;L | &#x223C;T) is proportional to the height of the class 2 surface over this point. So this ratio calculated over the whole grid provides a probability estimate. This surface is graphed on the left. On the right is the ROC curve given this surface, for varying cutoffs of nominal Bayesian probability, with the assumption that all translated class 1 uORFs will have sequence properties similar to the high BLASTP score positive training set, while all untranslated class 1 uORFs will be similar to (the presumably untranslated) class 2 uORFs negative training set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two-dimensional accounting for uORF variability. (A) Surface of proportion of uORFs over a two-dimensional grid of Kozak score (exponentiated) and log CDS length. These transformations were chosen because the Kozak score is based on an essentially logarithmic information scale (Figure 3), whereas CDS lengths can be conceptualized as due to exponential decay from finite probability of hitting a stop codon (class 3) or the beginning of the CDS (classes 1 and 2). Classes 2 and 3 have broad peaks with low Kozak scores (left axis) and shorter CDSs (right axis). The high BLASTP-score subclass of class 1 uORFs (lower right) has a sharp peak at high Kozak score and longer CDSs. The complete pool of class 1 uORFs appears heterogeneous, with a peak similar to the high BLASTP subset and a shoulder similar to the distributions of classes 2 and 3. (B) Least-squares accounting for the two-dimensional class 1 uORF distribution as a sum of 0.50 high BLASTP class 1 uORF subset and 0.48 class 2 uORF (matrix calculation as in Figure 4B); 99% of variation is explained. (C) A Bayesian test for translation of class 1 uORFs. Call T the event of translation; K a given Kozak score; L a given length. We want to know P(T | K&amp;L). Bayes theorem says this is equal to: P(K&amp;L | T) &#xD7; P(T) / [P(K&amp;L | T) &#xD7; P(T) + P(K&amp;L | &#x223C;T) &#xD7; P(&#x223C;T)]. Assuming &#x223C;50:50 split of translated and untranslated class 1 uORFs (Figure 4C, Figure 7C, and Figure 8B) [&lt;em&gt;i.e.&lt;/em&gt;, P(T) = P(&#x223C;T)], this simplifies to P(T | K&amp;L) = P(K&amp;L | T) / [P(K&amp;L | T) + P(K&amp;L | &#x223C;T)]. K&amp;L specifies a point on the grid; P(K&amp;L | T) is proportional to the height of the high-BLASTP surface over this point, while P(K&amp;L | &#x223C;T) is proportional to the height of the class 2 surface over this point. So this ratio calculated over the whole grid provides a probability estimate. This surface is graphed on the left. On the right is the ROC curve given this surface, for varying cutoffs of nominal Bayesian probability, with the assumption that all translated class 1 uORFs will have sequence properties similar to the high BLASTP score positive training set, while all untranslated class 1 uORFs will be similar to (the presumably untranslated) class 2 uORFs negative training set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-9.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-9.medium.gif" width="410" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F8/graphic-9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467678" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-42" class="first-child">Two-dimensional accounting for uORF variability. (A) Surface of proportion of uORFs over a two-dimensional grid of Kozak score (exponentiated) and log CDS length. These transformations were chosen because the Kozak score is based on an essentially logarithmic information scale (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), whereas CDS lengths can be conceptualized as due to exponential decay from finite probability of hitting a stop codon (class 3) or the beginning of the CDS (classes 1 and 2). Classes 2 and 3 have broad peaks with low Kozak scores (left axis) and shorter CDSs (right axis). The high BLASTP-score subclass of class 1 uORFs (lower right) has a sharp peak at high Kozak score and longer CDSs. The complete pool of class 1 uORFs appears heterogeneous, with a peak similar to the high BLASTP subset and a shoulder similar to the distributions of classes 2 and 3. (B) Least-squares accounting for the two-dimensional class 1 uORF distribution as a sum of 0.50 high BLASTP class 1 uORF subset and 0.48 class 2 uORF (matrix calculation as in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>); 99% of variation is explained. (C) A Bayesian test for translation of class 1 uORFs. Call T the event of translation; K a given Kozak score; L a given length. We want to know P(T | K&amp;L). Bayes theorem says this is equal to: P(K&amp;L | T) × P(T) / [P(K&amp;L | T) × P(T) + P(K&amp;L | ∼T) × P(∼T)]. Assuming ∼50:50 split of translated and untranslated class 1 uORFs (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4C</a>, <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7C</a>, and <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8B</a>) [<em>i.e.</em>, P(T) = P(∼T)], this simplifies to P(T | K&amp;L) = P(K&amp;L | T) / [P(K&amp;L | T) + P(K&amp;L | ∼T)]. K&amp;L specifies a point on the grid; P(K&amp;L | T) is proportional to the height of the high-BLASTP surface over this point, while P(K&amp;L | ∼T) is proportional to the height of the class 2 surface over this point. So this ratio calculated over the whole grid provides a probability estimate. This surface is graphed on the left. On the right is the ROC curve given this surface, for varying cutoffs of nominal Bayesian probability, with the assumption that all translated class 1 uORFs will have sequence properties similar to the high BLASTP score positive training set, while all untranslated class 1 uORFs will be similar to (the presumably untranslated) class 2 uORFs negative training set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The conclusion that a substantial proportion of class 1 uORFs is translated is based on multiple comparisons and tests that are entirely independent: length distributions of class 1 CDS compared to classes 2 and 3, and to randomized controls; Kozak score distributions of the same set, also compared to reference ATGs; BLASTP improvement in comparison to the <em>Volvox</em> proteome. The estimated 50:50 split is also based on projection of class 1 uORF data onto multiple distinct subspaces: spanned by Kozak scores of reference ATGs and random sequence; and spanned by CDS length of the high BLASTP class 1 subset and class 2 uORFs, alone or in two-dimensional combination with Kozak scores.</p></div><div id="sec-10" class="subsection"><h3>A large majority of reference ATGs are likely contained in CDS</h3><p id="p-44">The analysis so far was focused on the question of whether the reference ATG or a more 5′ ATG was a more likely site of translation initiation. A converse question can be asked: could some reference ATGs in fact themselves begin class 1 uORFs, with the authentic <em>in vivo</em> start codon being one annotated as internal?</p><p id="p-45">Another BLASTP comparison to <em>Volvox</em> provides a test. (In the following, for simplicity, I will call the reference ATG the ‘first’, and the succeeding ATG in the 3′ direction the ‘second’). In this comparison, I first determined the subset of transcripts for which there was a detectable <em>Volvox</em> BLASTP hit for which the maximum score was dependent on sequences immediately 3′ to the second ATG. There were 8320 such transcripts. In 78% of this set of transcripts, the segment between the first and the second ATG contributed further to the BLASTP score (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>), which would be unexpected if the first ATG was not in the translated product. This could be taken to imply that as many as 22% of ‘reference’ ATGs are not, in reality, part of CDS. Arguing against this idea, though, Kozak scores were on average higher for the first than for the second ATG (<em>P</em> &lt; &lt;&lt;0.001 by <em>t</em>-test for both comparisons) independent of BLASTP results (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>). Therefore, at least 78%, and likely a higher proportion, of the reference ATGs are actually within CDS.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Test for reference ATG being in CDS. (A) Call the reference ATG the &#x2018;first&#x2019; ATG, and the succeeding in-frame CDS-internal ATG the &#x2018;second&#x2019; ATG. In many coding sequences there will be BLASTP similarity to a Volvox peptide, aligning from 25 amino acids C-terminal to the second ATG to the end of the CDS, with score S0 (green bar indicates region of alignment). In a subset of these cases this alignment and BLASTP score will increase if the query for alignment begins exactly at the second ATG (rather than 25 amino acids to the C-terminus). This precondition (score S1 &gt; S0) for the test is met in 8230 transcripts. In such cases, if the first ATG is the genuine initiator, it is quite likely that the score will increase to S2 &gt; S1 if the BLASTP query additionally includes the segment encoded between the first and second ATG. In contrast, if the first ATG is not translated, then the score will almost surely not increase (S2 = S1), since conservation to Volvox should require selection based on evolutionarily conserved translation to peptide. In 78% of transcripts fitting the precondition S1 &gt; S0, we observed S2 &gt; S1, setting an upper bound of 22% for the frequency of reference initiators not actually translated. (B) Comparison of Kozak scores for the first (reference) ATG to those for the second ATG, in the two cases indicated in part A. In both cases the reference ATG population had significantly higher Kozak scores than second ATG, suggesting that the reference initiator is a more probable in vivo site of translation initiation even in the cases below failing the BLASTP score test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Test for reference ATG being in CDS. (A) Call the reference ATG the &#x2018;first&#x2019; ATG, and the succeeding in-frame CDS-internal ATG the &#x2018;second&#x2019; ATG. In many coding sequences there will be BLASTP similarity to a &lt;em&gt;Volvox&lt;/em&gt; peptide, aligning from 25 amino acids C-terminal to the second ATG to the end of the CDS, with score S0 (green bar indicates region of alignment). In a subset of these cases this alignment and BLASTP score will increase if the query for alignment begins exactly at the second ATG (rather than 25 amino acids to the C-terminus). This precondition (score S1 &gt; S0) for the test is met in 8230 transcripts. In such cases, if the first ATG is the genuine initiator, it is quite likely that the score will increase to S2 &gt; S1 if the BLASTP query additionally includes the segment encoded between the first and second ATG. In contrast, if the first ATG is not translated, then the score will almost surely not increase (S2 = S1), since conservation to &lt;em&gt;Volvox&lt;/em&gt; should require selection based on evolutionarily conserved translation to peptide. In 78% of transcripts fitting the precondition S1 &gt; S0, we observed S2 &gt; S1, setting an upper bound of 22% for the frequency of reference initiators not actually translated. (B) Comparison of Kozak scores for the first (reference) ATG to those for the second ATG, in the two cases indicated in part A. In both cases the reference ATG population had significantly higher Kozak scores than second ATG, suggesting that the reference initiator is a more probable &lt;em&gt;in vivo&lt;/em&gt; site of translation initiation even in the cases below failing the BLASTP score test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F9.medium.gif" width="248" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/6/2/435/F9.medium.gif" width="248" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-46" class="first-child">Test for reference ATG being in CDS. (A) Call the reference ATG the ‘first’ ATG, and the succeeding in-frame CDS-internal ATG the ‘second’ ATG. In many coding sequences there will be BLASTP similarity to a <em>Volvox</em> peptide, aligning from 25 amino acids C-terminal to the second ATG to the end of the CDS, with score S0 (green bar indicates region of alignment). In a subset of these cases this alignment and BLASTP score will increase if the query for alignment begins exactly at the second ATG (rather than 25 amino acids to the C-terminus). This precondition (score S1 &gt; S0) for the test is met in 8230 transcripts. In such cases, if the first ATG is the genuine initiator, it is quite likely that the score will increase to S2 &gt; S1 if the BLASTP query additionally includes the segment encoded between the first and second ATG. In contrast, if the first ATG is not translated, then the score will almost surely not increase (S2 = S1), since conservation to <em>Volvox</em> should require selection based on evolutionarily conserved translation to peptide. In 78% of transcripts fitting the precondition S1 &gt; S0, we observed S2 &gt; S1, setting an upper bound of 22% for the frequency of reference initiators not actually translated. (B) Comparison of Kozak scores for the first (reference) ATG to those for the second ATG, in the two cases indicated in part A. In both cases the reference ATG population had significantly higher Kozak scores than second ATG, suggesting that the reference initiator is a more probable <em>in vivo</em> site of translation initiation even in the cases below failing the BLASTP score test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Downstream open reading frames</h3><p id="p-47">The high prevalence of class 3 uORFs in the annotation suggests the possibility of translation reinitiation, so that if a class 3 uORF is translated this does not block translation of the main reference CDS. If such reinitiation is common, it raises the possibility that open reading frames in the 3′ untranslated region (downstream ORFs or ‘dORFs’) might similarly be translated after termination of the reference CDS. There are massively abundant open reading frames in the annotated 3′ untranslated regions – ∼300,000 in all (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>, top left), mostly less than 100 nt in length. A randomized control has about half as many dORFs. The distributions of dORF lengths and Kozak scores are identical between the real and randomized data. These observations do not strongly suggest that dORFs are translated in great abundance, although the increased number compared to the randomized control suggests some structure to this population.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/435/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Downstream open reading frames. Open reading frames within annotated 3&#x2032; UTR sequences (downstream open reading frames or dORFs) were compared to dORFs from a randomized control with the same overall nucleotide composition and same length distribution. Top left: cumulative numbers; bottom left: cumulative proportions. Right: distribution of Kozak scores for real and random dORF ATGs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718661574" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Downstream open reading frames. Open reading frames within annotated 3&#x2032; UTR sequences (downstream open reading frames or dORFs) were compared to dORFs from a randomized control with the same overall nucleotide composition and same length distribution. Top left: cumulative numbers; bottom left: cumulative proportions. Right: distribution of Kozak scores for real and random dORF ATGs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/435/F10.medium.gif" width="440" height="274"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/6/2/435/F10.medium.gif" width="440" height="274"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/435/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/435/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467558" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-48" class="first-child">Downstream open reading frames. Open reading frames within annotated 3′ UTR sequences (downstream open reading frames or dORFs) were compared to dORFs from a randomized control with the same overall nucleotide composition and same length distribution. Top left: cumulative numbers; bottom left: cumulative proportions. Right: distribution of Kozak scores for real and random dORF ATGs.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>The translational fates of abundant uORFs in the annotated Chlamydomonas genome</h3><p id="p-49">The annotated <em>Chlamydomonas</em> genome contains a very high number of uORFs in annotated 5UT. Since a large majority of these uORFs are upstream of a reference CDS that is evolutionarily conserved, it is very likely that the uORFs do not fully block translation of this reference.</p><p id="p-50">The following possibilities may explain the uORFs. (1) The transcription start site is incorrectly annotated in a majority of <em>Chlamydomonas</em> transcripts, and transcripts in fact generally initiate hundreds of nucleotides 3′ of the annotated position. This is logically possible; however, it is contradicted by EST evidence in the case of a substantial number of transcripts (Phytozome website). (2) The uORFs are not sites of translation initiation, due to sequence constraints (such as the Kozak consensus) resulting in their inefficient use. Our data support this possibility for class 2 and class 3 uORFs, and for about half of the class 1 uORFs. (3) The uORFs are sites of translation initiation, but do not interfere with translation of the reference CDS. This is our interpretation for about half of the class 1 uORFs, which likely encode translated N-terminal extensions to the reference CDS. It is an interesting possibility that class 3 uORFs are translated. Such translation can be compatible with translation of downstream AUGs (as in GCN4; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Hinnebusch 2011</a>); the GCN4 case also shows clearly that this situation can allow for regulated translation initiation. (Class 2 uORFs are likely not suitable for this mechanism, assuming that scanning is largely 3′-unidirectional, as by the time of translation termination the ribosome would be 3′ of the reference initiator.) Such mechanisms, if they exist, would likely not impose significant length or sequence constraints on class 3 uORFs, since the relevant feature might simply be having some number or density of ‘decoy’ uORFs—the sequence content might be irrelevant.</p></div><div id="sec-14" class="subsection"><h3>Functional consequences of N-terminal extensions</h3><p id="p-51">The class 1 uORFs vary tremendously in length, and it is likely that cases where only a few residues are missed at the N-terminus in the reference annotation will frequently have rather minor functional consequences. Also, protein N- and C-termini are probably more likely to be poorly folded than other regions (perhaps explaining why terminal epitope tagging so frequently is permissive for protein function). There are many cases, though, where functional consequences from exclusion of the class 1 uORF N-terminal extension are likely quite substantial (a few clear examples, selected from many, are given in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>).</p><p id="p-52">Protein N-termini can contain relatively unstructured ‘addressing’ sequences for post-translational modification and/or subcellular localization, such as signal sequences for secretion or organellar transport; these would likely be undetected by BLASTP analysis. A different kind of consequence comes in searching for causative mutations following random mutagenesis: even a small, potentially unstructured N-terminal extension due to a uORF can be the site of a chain-terminating null mutation, which can go completely unrecognized if the uORF is not annotated as a possible contributor to CDS. In fact, we found just such a case in a screen for latrunculin B-sensitive mutants: a one-nucleotide deletion in an annotated 5UT region resulted in a strong mutation in a specific molecularly identified complementation group (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">M. Onishi <em>et al</em>. 2015</a>). This finding was a major motivation for carrying out the present study; the deletion is, in fact, in a class 1 uORF. This ORF has a calculated probability of being translated of 0.90 by the Bayesian test described above (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8C</a>). Thus, this mutation is very likely an early chain terminator, consistent with other genetic results indicating that it produces a null allele.</p></div><div id="sec-15" class="subsection"><h3>Genome annotation is probabilistic</h3><p id="p-53">It is a subtle problem that a sophisticated data presentation such as the <em>Chlamydomonas</em> annotated genome (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>) necessarily requires selection of a single one of a large number of alternatives; the most probable on some accounting is presumably selected, and serves as the sole presented version. However, the reasoning behind producing the object will generally be probabilistic and based substantially on statistical evidence. This was indeed the case for assignment of reference initiator ATGs in the <em>Chlamydomonas</em> annotation (M. Stanke, personal communication). A more accurate (though pedantic) description than ‘finding needles in a haystack’ (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>) might be ‘finding the most probable needles in a haystack that surely contains at least some.’ Unfortunately, gene models are amenable to neither the calculation nor graphical presentation of error bars, and certainly, most end users desire to work with ‘the answer,’ not with some probabilistically graded series of alternatives.</p><p id="p-54">While this detailed examination of uORFs did uncover some issues and problems, the results suggest that for the most part, the reference initiating ATGs are the authentic <em>in vivo</em> initiators. It is encouraging that a largely computational assignment of the complex biochemical events of mRNA production and translation can be correct in a large majority of cases. The explicitly probabilistic evaluation of likely sites of <em>in vivo</em> translational initiation presented here may be useful in consideration of the <em>Chlamydomonas</em> proteome, at least until there are sufficient data from proteomics and other approaches to settle the matter definitively.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">I thank Masayuki Onishi for discussions of both uORFs and dORFs, and for contributions to finding the first examples of class 1 uORFs that motivated the broader study. I thank Sabeeha Merchant, Simon Prochnik, and Mario Stanke for discussions and for clarifying aspects of gene annotation used in developing the <em>Chlamydomonas</em> reference. I thank Frej Tulin for extensive discussions and for carrying out BLAST searches. Funding support was from National Institutes of Health 5RO1-GM078153.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-56">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023119/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023119/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: B. A. Cohen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-28"><span class="received-label">Received </span>September 28, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-12-08"><span class="accepted-label">Accepted </span>December 8, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Cross</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F435.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blaby</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaby-Haas</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tourasse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hom</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Chlamydomonas genome project: a decade on.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">672</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D672%26rft_id%253Dinfo%253Apmid%252F24950814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24950814&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F435.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.3" data-doi="10.1101/gr.849004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandonia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">WebLogo: a sequence logo generator.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1188</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi80MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.4" data-doi="10.1073/pnas.0811205106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herron</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylward</span>  <span class="cit-name-given-names">F. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michod</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Triassic origin and early radiation of multicellular volvocine algae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">3254</span>–<span class="cit-lpage">3258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0811205106%26rft_id%253Dinfo%253Apmid%252F19223580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzkvMzI1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi80MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.5" data-doi="10.1128/MMBR.00008-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinnebusch</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular mechanism of scanning and start codon selection in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">75</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">434</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DHinnebusch%26rft.auinit1%253DA.%2BG.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D434%26rft.epage%253D467%26rft.atitle%253DMolecular%2BMechanism%2Bof%2BScanning%2Band%2BStart%2BCodon%2BSelection%2Bin%2BEukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00008-11%26rft_id%253Dinfo%253Apmid%252F21885680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3NS8zLzQzNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi80MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.6" data-doi="10.1016/0092-8674(78)90039-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozak</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">How do eucaryotic ribosomes select initiation regions in messenger RNA?</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1109</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKozak%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D1109%26rft.epage%253D1123%26rft.atitle%253DHow%2Bdo%2Beucaryotic%2Bribosomes%2Bselect%2Binitiation%2Bregions%2Bin%2Bmessenger%2BRNA%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252878%252990039-9%26rft_id%253Dinfo%253Apmid%252F215319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(78)90039-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=215319&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F435.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978GC55400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.7" data-doi="10.1083/jcb.108.2.229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozak</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The scanning model for translation: an update.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">108</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">229</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DKozak%26rft.auinit1%253DM.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D241%26rft.atitle%253DThe%2Bscanning%2Bmodel%2Bfor%2Btranslation%253A%2Ban%2Bupdate%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.108.2.229%26rft_id%253Dinfo%253Apmid%252F2645293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjEwOC8yLzIyOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi80MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.8" data-doi="10.1126/science.1143609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpowicz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Chlamydomonas genome reveals the evolution of key animal and plant functions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>(<span class="cit-issue">5848</span>): <span class="cit-fpage">245</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMerchant%26rft.auinit1%253DS.%2BS.%26rft.volume%253D318%26rft.issue%253D5848%26rft.spage%253D245%26rft.epage%253D250%26rft.atitle%253DThe%2BChlamydomonas%2BGenome%2BReveals%2Bthe%2BEvolution%2Bof%2BKey%2BAnimal%2Band%2BPlant%2BFunctions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143609%26rft_id%253Dinfo%253Apmid%252F17932292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OC8yNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvNDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evidence that an unconventional actin can provide essential F-actin function and that a surveillance system monitors F-actin integrity in chlamydomonas</span>. <span class="cit-comment">Genetics DOI: 10.1534/genetics.115.184663.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.10" data-doi="10.1126/science.1188800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nedelcu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic analysis of organismal complexity in the multicellular green alga Volvo x carteri.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>(<span class="cit-issue">5988</span>): <span class="cit-fpage">223</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DProchnik%26rft.auinit1%253DS.%2BE.%26rft.volume%253D329%26rft.issue%253D5988%26rft.spage%253D223%26rft.epage%253D226%26rft.atitle%253DGenomic%2BAnalysis%2Bof%2BOrganismal%2BComplexity%2Bin%2Bthe%2BMulticellular%2BGreen%2BAlga%2BVolvox%2Bcarteri%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188800%26rft_id%253Dinfo%253Apmid%252F20616280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk4OC8yMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzIvNDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-6.2.435.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strang</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Introduction to Linear Algebra</span>. <span class="cit-publ-name">Wellesley-Cambridge Press</span>. <span class="cit-publ-loc">Wellesley, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.12" data-doi="10.1126/science.282.5394.1721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zachariae</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seufert</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Control of cyclin ubiquitination by CDK-regulated binding of Hct1 to the anaphase promoting complex.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>(<span class="cit-issue">5394</span>): <span class="cit-fpage">1721</span>–<span class="cit-lpage">1724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZachariae%26rft.auinit1%253DW.%26rft.volume%253D282%26rft.issue%253D5394%26rft.spage%253D1721%26rft.epage%253D1724%26rft.atitle%253DControl%2Bof%2BCyclin%2BUbiquitination%2Bby%2BCDK-Regulated%2BBinding%2Bof%2BHct1%2Bto%2Bthe%2BAnaphase%2BPromoting%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5394.1721%26rft_id%253Dinfo%253Apmid%252F9831566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5NC8xNzIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzQzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.435.13" data-doi="10.1371/journal.pcbi.1003136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zur</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">New universal rules of eukaryotic translation initiation fidelity.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1003136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003136%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003136%26rft_id%253Dinfo%253Apmid%252F23874179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874179&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F435.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/2/435.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/423" title="MicroRNA Maturation and MicroRNA Target Gene Expression Regulation Are Severely Disrupted in Soybean dicer-like1 Double Mutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/447" title="FASTmC: A Suite of Predictive Models for Nonreference-Based Estimations of DNA Methylation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/2"><span>Volume 6 Issue 2, February 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/2.cover-source.jpg?itok=7FK5uFFF" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/435.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kAzkRPUsVmP3BrHKVDbg4jZ_TGZD18QriDXfy3wE6hk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/2/435" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/2/435" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/2/435" class="active" data-icon-position="" data-hide-link-title="0">Tying Down Loose Ends in the Chlamydomonas Genome: Functional Significance of Abundant Upstream Open Reading Frames</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460068" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eXG_g63JcI-yoC2oPketgpuT9I8jKd6HFHxiQs1reRk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/435.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/2/435" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-mQPhduXJNKry-TlOnlgTiqvMHjRd8LG8rgvYRmXKzfY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/2/435" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/435.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460068" id="node460068--2" data-pisa="ggg;6/2/435" data-pisa-master="ggg;g3.115.023119" data-apath="/ggg/6/2/435.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tying Down Loose Ends in the <em>Chlamydomonas</em> Genome: Functional Significance of Abundant Upstream Open Reading Frames</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frederick R. Cross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">435-446;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023119 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460068--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fcross@mail.rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">fcross@mail.rockefeller.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/2/435" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/2/435&amp;title=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/2/435&amp;title=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/2/435&amp;title=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/2/435&amp;text=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/2/435&amp;title=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/2/435&amp;t=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/2/435&amp;title=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/2/435&amp;title=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460068/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460068" id="node460068--3" data-pisa="ggg;6/2/435" data-pisa-master="ggg;g3.115.023119" data-apath="/ggg/6/2/435.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tying Down Loose Ends in the <em>Chlamydomonas</em> Genome: Functional Significance of Abundant Upstream Open Reading Frames</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frederick R. Cross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">435-446;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023119 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460068--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fcross@mail.rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">fcross@mail.rockefeller.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=435&amp;rft.epage=446&amp;rft.atitle=Tying%20Down%20Loose%20Ends%20in%20the%20Chlamydomonas%20Genome%3A%20Functional%20Significance%20of%20Abundant%20Upstream%20Open%20Reading%20Frames&amp;rft.volume=6&amp;rft.issue=2&amp;rft.date=2016-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cross%2C+Frederick+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460068/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460068/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460068/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460068/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460068/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460068/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460068/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460068/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460068/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460068/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460068/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460068/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/2/435" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/2/435.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/2/435.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/2/435.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/2/435.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
