<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/2/281" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/2/281.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/2/281.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/2/281.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/2/281/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/2/281.atom" />
<meta name="HW.pisa" content="ggg;6/2/281" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="argyle: An R Package for Analysis of Illumina Genotyping Arrays" />
<meta name="DC.Identifier" content="10.1534/g3.115.023739" />
<meta name="DC.Date" content="2016-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Morgan. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping microarrays are an important and widely-used tool in genetics. I present argyle, an R package for analysis of genotyping array data tailored to Illumina arrays. The goal of the argyle package is to provide simple, expressive tools for nonexpert users to perform quality checks and exploratory analyses of genotyping data. To these ends, the package consists of a suite of quality-control functions, normalization procedures, and utilities for visually and statistically summarizing such data. Format-conversion tools allow interoperability with popular software packages for analysis of genetic data including PLINK, R/qtl and DOQTL. Detailed vignettes demonstrating common use cases are included as supporting information. argyle bridges the gap between the low-level tasks of quality control and high-level tasks of genetic analysis. It is freely available at &lt;https://github.com/andrewparkermorgan/argyle&gt; and has been submitted to Bioconductor." />
<meta name="DC.Contributor" content="Andrew P. Morgan" />
<meta name="article:published_time" content="2016-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="argyle: An R Package for Analysis of Illumina Genotyping Arrays" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping microarrays are an important and widely-used tool in genetics. I present argyle, an R package for analysis of genotyping array data tailored to Illumina arrays. The goal of the argyle package is to provide simple, expressive tools for nonexpert users to perform quality checks and exploratory analyses of genotyping data. To these ends, the package consists of a suite of quality-control functions, normalization procedures, and utilities for visually and statistically summarizing such data. Format-conversion tools allow interoperability with popular software packages for analysis of genetic data including PLINK, R/qtl and DOQTL. Detailed vignettes demonstrating common use cases are included as supporting information. argyle bridges the gap between the low-level tasks of quality control and high-level tasks of genetic analysis. It is freely available at https://github.com/andrewparkermorgan/argyle and has been submitted to Bioconductor.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/02/01" />
<meta name="citation_mjid" content="ggg;6/2/281" />
<meta name="citation_id" content="6/2/281" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/2/281" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/2/281.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/2/281.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/2/281.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023739" />
<meta name="citation_pmid" content="26684930" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="281" />
<meta name="citation_lastpage" content="286" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew P. Morgan" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7264" />
<meta name="citation_author_email" content="apm@email.unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1942-4543" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=B. Bolstad;citation_author=R. Irizarry;citation_author=M. Astrand;citation_author=T. Speed;citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias;citation_pages=185-193;citation_volume=19;citation_year=2003;citation_issue=2;citation_pmid=12538238;citation_doi=10.1093/bioinformatics/19.2.185" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=Z. Naghashfar;citation_author=D. W. Threadgill;citation_author=H. C. Morse;citation_title=SNP array profiling of mouse cell lines identifies their strains of origin and reveals cross-contamination and widespread aneuploidy.;citation_pages=847;citation_volume=15;citation_year=2014;citation_pmid=25277546;citation_doi=10.1186/1471-2164-15-847" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Du;citation_author=W. A. Kibbe;citation_author=S. M. Lin;citation_title=lumi: a pipeline for processing illumina microarray.;citation_volume=24;citation_year=2008;citation_pmid=18467348;citation_doi=10.1093/bioinformatics/btn224" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. J. Dunning;citation_author=M. L. Smith;citation_author=M. E. Ritchie;citation_author=S. Tavare;citation_title=beadarray: R classes and methods for illumina bead-based data.;citation_volume=23;citation_year=2007;citation_pmid=17586828;citation_doi=10.1093/bioinformatics/btm311" />
<meta name="citation_reference" content="citation_title=Seamless R and C++ Integration with Rcpp;citation_author=D. Eddelbuettel;citation_year=2013" />
<meta name="citation_reference" content="citation_author=C.-P. Fu;citation_author=C. E. Welsh;citation_author=F. P.-M. de Villena;citation_author=L. McMillan;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative Trait Locus Mapping Methods for Diversity Outbred Mice;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_issue=9;citation_pmid=25237114;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. M. Hulse-Kemp;citation_author=J. Lemm;citation_author=J. Plieske;citation_author=H. Ashrafi;citation_author=R. Buyyarapu;citation_title=Development of a 63k SNP array for cotton and high-density mapping of intraspecific and interspecific populations of gossypium spp.;citation_volume=5;citation_year=2015;citation_pmid=25908569;citation_doi=10.1534/g3.115.018416" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. E. Johnston;citation_author=M. Lindqvist;citation_author=E. Niemelä;citation_author=P. Orell;citation_author=J. Erkinaro;citation_title=Fish scales and SNP chips: SNP genotyping and allele frequency estimation in individual and pooled DNA from historical samples of atlantic salmon (salmo salar).;citation_pages=439;citation_volume=14;citation_year=2013;citation_pmid=23819691;citation_doi=10.1186/1471-2164-14-439" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. W. Kijas;citation_author=D. Townley;citation_author=B. P. Dalrymple;citation_author=M. P. Heaton;citation_author=J. F. Maddox;citation_title=A genome wide survey of SNP variation reveals the genetic structure of sheep breeds.;citation_pages=e4668-e4668;citation_volume=4;citation_year=2009;citation_issue=3;citation_pmid=19270757;citation_doi=10.1371/journal.pone.0004668" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A.P. Morgan;citation_author=C.-P. Fu;citation_author=C.-Y. Kao;citation_author=C. E. Welsh;citation_author=J.P. Didion;citation_title=The mouse universal genotyping array: from substrains to subspecies.;citation_pages=263-279;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Peiffer;citation_title=High-resolution genomic profiling of chromosomal aberrations using infinium whole-genome genotyping.;citation_volume=16;citation_year=2006;citation_pmid=16899659;citation_doi=10.1101/gr.5402306" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. E. Ritchie;citation_author=B. S. Carvalho;citation_author=K. N. Hetrick;citation_author=S. Tavare;citation_author=R. A. Irizarry;citation_title=R/bioconductor software for Illumina’s Infinium whole-genome genotyping BeadChips.;citation_volume=25;citation_year=2009;citation_pmid=19661241;citation_doi=10.1093/bioinformatics/btp470" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. Staaf;citation_author=J. Vallon-Christersson;citation_author=D. Lindgren;citation_author=G. Juliusson;citation_author=R. Rosenquist;citation_title=Normalization of Illumina Infinium whole-genome SNP data improves copy number estimates and allelic intensity ratios.;citation_pages=409-409;citation_volume=9;citation_year=2008;citation_pmid=18831757;citation_doi=10.1186/1471-2105-9-409" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=F. J. Steemers;citation_author=W. Chang;citation_author=G. Lee;citation_author=D. L. Barker;citation_author=R. Shen;citation_title=Whole-genome genotyping with the single-base extension assay.;citation_pages=31-33;citation_volume=3;citation_year=2006;citation_issue=1;citation_pmid=16369550;citation_doi=10.1038/nmeth842" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=W. Sun;citation_author=F. A. Wright;citation_author=Z. Tang;citation_author=S. H. Nordgard;citation_author=P. V. Loo;citation_title=Integrated study of copy number states and genotype calls using high-density SNP arrays.;citation_volume=37;citation_year=2009;citation_pmid=19581427;citation_doi=10.1093/nar/gkp493" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Wang;citation_author=M. Li;citation_author=D. Hadley;citation_author=R. Liu;citation_author=J. Glessner;citation_title=PennCNV: an integrated hidden markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data.;citation_volume=17;citation_year=2007;citation_pmid=17921354;citation_doi=10.1101/gr.6861907" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=C. E. Willet;citation_author=B. Haase;citation_title=An updated felCat5 SNP manifest for the illumina feline 63k SNP genotyping array.;citation_pages=614-615;citation_volume=45;citation_year=2014;citation_pmid=24804964;citation_doi=10.1111/age.12169" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="argyle: An R Package for Analysis of Illumina Genotyping Arrays" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping microarrays are an important and widely-used tool in genetics. I present argyle, an R package for analysis of genotyping array data tailored to Illumina arrays. The goal of the argyle package is to provide simple, expressive tools for nonexpert users to perform quality checks and exploratory analyses of genotyping data. To these ends, the package consists of a suite of quality-control functions, normalization procedures, and utilities for visually and statistically summarizing such data. Format-conversion tools allow interoperability with popular software packages for analysis of genetic data including PLINK, R/qtl and DOQTL. Detailed vignettes demonstrating common use cases are included as supporting information. argyle bridges the gap between the low-level tasks of quality control and high-level tasks of genetic analysis. It is freely available at &lt;https://github.com/andrewparkermorgan/argyle&gt; and has been submitted to Bioconductor." />
<meta name="og-title" property="og:title" content="argyle: An R Package for Analysis of Illumina Genotyping Arrays" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/2/281" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping microarrays are an important and widely-used tool in genetics. I present argyle, an R package for analysis of genotyping array data tailored to Illumina arrays. The goal of the argyle package is to provide simple, expressive tools for nonexpert users to perform quality checks and exploratory analyses of genotyping data. To these ends, the package consists of a suite of quality-control functions, normalization procedures, and utilities for visually and statistically summarizing such data. Format-conversion tools allow interoperability with popular software packages for analysis of genetic data including PLINK, R/qtl and DOQTL. Detailed vignettes demonstrating common use cases are included as supporting information. argyle bridges the gap between the low-level tasks of quality control and high-level tasks of genetic analysis. It is freely available at &lt;https://github.com/andrewparkermorgan/argyle&gt; and has been submitted to Bioconductor." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2.cover-source.jpg" />
    <title>argyle: An R Package for Analysis of Illumina Genotyping Arrays | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Ytt9x_b4TUwNAYWzJUbvEESJE59T7qJe7_fI3frt_9s","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460056","apath":"\/ggg\/6\/2\/281.atom","pisa":"ggg;6\/2\/281","ac":{"\/ggg\/6\/2\/281.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/2\/281.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/2\/281"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/2\/281"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/2\/281":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/aK4yI7b4OMoKhO5mkIl+5mPMqTL+iiFfXGsEqJF+08nrbodI2nZTQXuk14nI60WrWJT5QzJIRD4\\\\\\\/rkZvGjtb4smW6fG24FBLkOxCYn\\\\\\\/677womwuRR81auM0LnDOVLBw53owq7WQJpstxR8590iSbJAu1rHiEQSyha7czKfSQpH8AC4Jm\\\\\\\/yKyswV0zW8S3OP7HJZ76c1pVQQgSZpqlwLpzKKQJmJpr5Pr2jNH6UQCNdqcmmkd\\\\\\\/VDK4nRf+ilbeS\\\\\\\/ruqZ5Cgisnm3N6+lpjjewLvKwyfgRvdn+LYPSiPUxg7hjONX604tp1OB\\\\\\\/7CkQ+tZso7HrE054QSjjKGTDsWgskov5e3ow9yi8goITACk70jeIhgGEIFpozNI5AJrsD8EyepIj+6AkY7sz\\\\\\\/NjgB5urGtVIKGagkjp0Vasan0aEx+4gTG5nXFq78KxclUNMOT4qn8byvPZyfME\\\\\\\/sdL0dQN1U7GirYcX742N9XYu0ug9Wzep0EBltdZM8kMiZED5yhL5LS4J+cCgM0ZcNo6qk05uXnymWhHuj78fwFgkoxn1TQzm+zfE97xW66kTryRXFG6IgegWpsFwgG21zuRuY2sg45U8yc\\\\\\\/I6mgfsCScSfce8SpW0kj9Yvc\\\\\\\/q1ghBKXuwqsphgQxH2n4ofWxbjCf4+etEuKAmCcM9lH2oTO\\\\\\\/qqHlHmluLkCAYCnN2b\\\\\\\/jW2Zd03q+7kjf505v32nGpY2M0RaetjyrIMSOW3GUh7B4NP8sqj+f7S8\\\\\\\/qmYDDvWq+LyZEfZitFvhCo8tZN\\\\\\\/7mFzkZzY7pUMhkwbMXsAykiLs\\\\\\\/Yj54Wm84PwJbyYh6SnKJ9kpwi9+Cb+fLErERC+Gwcq6Qy9MIZLnQmo5TSpvKP3Sl72YeHmTjaFOoAPy9euCoTxysLDo7elPjdaWlFhc3o949XC0ZPRXIoSrRxY9nqa0KhtpXu9HP2TRKgL7JznAMlbNA60V8pOp+xFMm+0j2MdE8zW4XO2zEeCw0z7j9sd+dmDO88XADG2slHH0kpT75dt4cOMFlIunSksrIkOyb3OK3lDMCcXMuL7s1bcmDAH0bZdSV8xQSIprKDNpFu2Oz0SVavWN3Y\\\\\\\/aSjVD+Kh2dh1fFQ1\\\\\\\/cbwjjKzp1Vc1IdyFfUSygM+JpxobylKt94Sd8hsfmeRoVPopNDwkQOGmg3Mqj+KH2vAnFNsyl+jBFEmUHkhEbHROBJeRhp4uJgBgqXJ0xZz9v9wdbhpkQ0\\\\\\\/waL6OpT8obiDXO8iTbAgQccYEYfdiqRLs7jwBXNsbBkd6Lyq7zyn\\\u0022,\\\u0022iv\\\u0022:\\\u0022do895VN1nYnqRTL\\\\\\\/ht462Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a080fa76c32493e7e276afdb5b1ea4b378ba5c2ecbd7b481d1f7a2b89626559c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LI3DnhBtxwFYiUrJ5HnNs2MqdXkshHU9YudD66mU6I5lBaP7+Csm1d3AH6LzVGRr70gQzVbneOki5NzGzMKOBkZsGadIOZcBu4yUNt8u5WBDjb+33BEhwNQxt8TbDcIO7\\\\\\\/eSwvcXUHzQajjYh70\\\\\\\/bgLG8YtQ3I06xuY5BF8CCn7isqeFM7CdjvztyEQYu41lTSicAAKPop9p+EsVkjVpVwOZmXAyx728eowgJqTt8WQrcRJEQX1sLOINU4QE3pfOMlv0yspdSqb\\\\\\\/WHwO0omMpDsOaGSeSmsZB4X8Ji10P48d2h\\\\\\\/IZ+x5TlkbwxNt4nnjV57VnL7BtzPuU3M19WpvMS69VtjUdpd11mL28HAWuK0fg8syqUGLWB4JvX0ZdBELeV3zvpcolffuuHvGZF5eXfr43z+DmWl2rmCGxcPos2tPd2wv+tKVQPB7rAm2\\\\\\\/oaPsVPK0n4pOsOwy5QIv94Jwlop\\\\\\\/2b3ge062Y34wafsy2L3W6hgJBjNc2v5i4qdoh22fssXqC0EqqXXOsTaTbZyup9XcvZGbIZDNDVk\\\\\\\/xrgpK4bxpKojMbcZ\\\\\\\/Ly\\\\\\\/Mv5Q7u+HlAByfhulTJhmgsXMrccDquXWMxGNHIRh7HXlf0y6joc6GkwGZBO+PRwiJSDASCX7G0biahXaYYhySkcBgK4AXm4mLLncbu+39icYQoO6xYNwJJcU2P1pEzdyLX1s6vcLO6rbkiOL0hsW6qctNouXal4Zex9Nmptb9QKMXVUWzOGnejnvcC2RK\\\\\\\/IKWIhMzw\\\\\\\/Uk7YX3cxurbQ1Wh5oofOwQtwgLherQDCmW7L7Crj8DbayUf\\\\\\\/8Uk6cAaQlEi6XxnkHqcVUhct1Vn++EwYiiCO+P1Cj1OQB18SPrS29lAfqnoWSq9mg2tzO5Aw19vdbFGdB3qhI8vEmpJqUmAmc0VNlPUt\\\\\\\/g4C2ebsKBS6uHW8E2Sj0cUd2PXTOfzQJ24Ija5C7pWwufZuj9QuoOI+OFWUKaN2dNAHh1lYm7QbnbNVWUDLkckAu6kVxQoobpwjWqXxgiyQOHt5N1OZQlJg0Yg1sre1080n+3c4AEHNzDLRZe+VW8KJzYxM+2cmcs1vuUtQL1Yloikk8x5X91+QNJXpguo1VCqP5EQyxorecS+z8QbeTF54PyLUXm0P4f+n3TmwSPBkQs\\\\\\\/EtmD1W8A4AeC1PHHR2VRr2dylHv+ZRnKC7HyXpZYEC4uWa\\\\\\\/O0kU267M7ZvlHTuN1ODdNDizfkwOciYtMUUBNy\\\\\\\/vvayYSkjelo7LGDUUyTRAahJzj3cfy3c9ewk\\\\\\\/GXIzYGaL4\\\\\\\/nJp3PknKyxbNKE+ltxjRyr+2\\\\\\\/8k53F5m4YZf0QGi4\\\\\\\/zg5vKlyyXyZnplX3Zih+4dMacWCsOpsa4OHOfRWAestr7hRA8n0iePs0ibBZuvpL0CsqHEIg83wOewTrVn8AL1kGD6hQudb9MkpLX1QJYmMZ8qOkFU4aO\\\\\\\/4nOmT\\\\\\\/SnZF1Z\\\\\\\/Y1bjJgk1UCduv6C1VSShT4o\\\\\\\/XZfQir3ggKNPh4YUXkFKhj9M8ztei4jz8rLNqQa3Q9Op9E36TxVBGw2dc\\\\\\\/SRG7lpnzltjdoR3wd2oV9KTeqCjF2JvIW8XYr+m0cecoISR8jXAo0p4eWLa6kDI9oPlo\\\\\\\/Przh8DdqRw8PlCQfcyrfAQagJ\\\\\\\/zUOmEJJ49qec2O6Sk7+8Q2+PhvvQTyg\\\\\\\/+Ppg6s57Z9wm3m\\\\\\\/23D62zYv1y\\\\\\\/26qacgNB9lpbaDPalviYQwUEKnqWnzHlButV5xi8ozmtA1cJ5ORN0znX7ZHJuNi2ghOAlJ\\\\\\\/C2K46bW5WZxYdli4Ed5GUVJm00cmMjsTWpLFlWaU3dlqReNc2+orcVO+k4hm6ch77g2sbw2Ph+NXqSFsVHW3qMmy\\\\\\\/5EVYTT8k3\\\\\\\/5eDA6jwkkzxPNlm2chArYlFjyia1wn6TNFhHbz+PStC\\\\\\\/Gb9tdHW06zJGnQ9cnPXgN68jx3dewM3\\\\\\\/I+UqeFMOI9DtEwHgy8Jjesb3MK8snTF+MxvC98zPR4sRygYz+EGt2KqG2yPzxC5yETtcWF8DH\\\\\\\/1tqIsFqRlIzH9d7Kk2Nytn3fwUEbpkrjUFDHw9AgYYTEVM42+RijB6LC6Kh9qbb+CEHJkPG2voP+CzY9gJekzmToN20XRRnwCO1Kc2CakTY9tLAOUNQeAouK\\\\\\\/yTfaNRjgR2T\\\\\\\/69ixA4mU4RegYp3wGNkCLUzOa6b4bXGLEpniOhZRnowSTgkfJsqcTHHFZuDKyoQVgrQ4GTtZX127yV3MzpWUrDMFtNd4AvIXcWedNVzPOmBEN17+k6gMzFIJXZG0mirc0Hv016ohMLlc17SUrpJThLTv3fYPuCgKnfNBz1SaQYxyXj2ewY4jJ\\\\\\\/Bxs3Eky8DSiXwqlkq4D2HRQdu9tqHUueF7yxM9sSr0eEMnQYsI20qP0zkZgiRpXpgLUVHERAK23FkXLIYyrpvBfVvpS\\\\\\\/UlLh5TJjomaGmdxNyJgJFI2eq2ecRgjA9DBpYdJ4VTf9HUCX0+cmW+UdeAiiD+Z\\\\\\\/QNFERa555VOip0FZ3V3YJY4lYWe+NhtUCglcR2iHqibFyFb2Aqm0pKzsML6jr3gjLooVNHA6HyexBkVhsp9T2TGLiOrLFm5KE70mlOyhDPiOHjob0lo7tXV62YGAMz\\\\\\\/KMx6SlYcmNmNGCqzEjHBB+52+ZzGaXEQvBjdMnHXelzMrUYuSeVJJo8l8anfkcvYU4JWX1EEnucY7608y0MilvNwVpWeKypjxmX3g+K9YcGitZWydkQxU28qe8tMLdwOrMlxVdIx4v39vCORhaPRML\\\\\\\/kGgDcHJBF9NvUWuWS4VsIqsvmWTT+mU1hZHO+Fiv9BmuzgS1UccQaU5vLI+wlwZKK1jwfgBYkD\\\\\\\/IhF27BmCW4YPRKVOPYrYPLMKB66DZlCqSzgSTHv3W6nr5PDmz+nMvVApcmwARMDWZlRco09mXJzSOdiuiH1u88YzeTjzuWtr3AE9FUcRi2Mpv3nGFway4VXYNzd6bA\\\\\\\/HaviUhSEDinl6ijxiL6DJxKgh0h03tCdCYj7GD+\\\\\\\/cAp9H70dNRJuGgbVrcPb2jITpEGAcgtuJUPQ+PlACZBuZBj+KAYhyQf2or6mlWVHKBfEwnMz2eNfoY3SDW\\\\\\\/65cvXNkcWkwGn2Yhx2KnwQll2PcPYv8DPKLgN94D4Ak+Z+9VN+snU\\\\\\\/cMQshqWBY7H9TW2RHdPUb0yb+5GMffKIJGI7Y5OdmPxxMaSAN1BAbbEgEVQAVCH5QGuV0+hJmVcUx3azxMYHc9V+QuLFDKA+0EiAl4uL+EYFJMLfNzdxfQd4MKK+tQ5w4AE3Y13mg75pu+dekuentNvKpy8EGtEm1HM9zHtuK5ArIk2Lpe8e5L3XBTvtxIx+BiSSnWCNX8oFQxo7WwY5\\\\\\\/4b6aBWaJ5rSpkqg2+M3aORpYnu8dh0QWVa4F61cN2cQLJdb2U+Eu6LNzXVI5M4ZrakVUcW2c2zpTKbb50tn1UjDSEipAG5\\\\\\\/V2GkScqWaf5X92SVkzXiKsok5\\\\\\\/5TFZDm2GwpVIn6DZS6ghKqq85iVKtRF2g0WHcnVYizhqLXzWL13H\\\\\\\/ui07LfBNZyILOOazPuC6A8KMHtTTzmqUBp1q1p+1\\\\\\\/l8komeIw3Kxkc2UdKcOVKa3wh+scmRlwz70ITO2+glMLuSwZGLBMDe2muzaMF84ZE8RqOAPvva1PiM3IeFopo4s9Q11ySLwK956uPRgf309pgUUwoPsWSGOCx+vv2qSa0C1V8VnheTNWSTdqULqdaO41ufxpGEuXH8Ixwbmii8fPYa3o6ldtv5NkF9Xw7MjYxFAPyDxMYfRCWZvWX1mV0keUgCIrwNb860n19Gm3jWAjLqe2S7iVI+d7O2TYBgrECdNqh+X2fECpPdGXaWSL7qTJjgSRi9UHv6AYJDgNuKVwhsgZvb5JUCLmNcBRAzWIu61d36oKufk8OtUzEd04QlVXpklBo0DVRskXklccWRD3Z4sfxfCiPc\\\\\\\/D7mo97lah4O6ihJfIylD0WJ92AnM2EA7CUJ0EUCVNg1iL5ME1zR7M7kG602lpofoLCw+coRmgzxN0fcqzabcIAjXt9YmN8uTZuaWLZXOlJO+J+Od2gdd+frR6VzAiY6+EiS3ssoT3Wl48I85wLfISg4qTZ\\\\\\\/IvZ3EH2VWCFdi8MYxl3dyFtzfqXNAzJs9mAq5OQcPWkQmGhpeaIStcX\\\\\\\/vo2yYO7EJFCta2dlDPoZ8A8sX0DPCa5f1+LoUqqbf5HDUO2fC08IX2OWaqAUavuoeBmPsszadPpxgggpB\\\\\\\/76NBYBWHBx9\\\\\\\/wXpWTzpdDFoRey+d0mAHzMy4p3dQWos0Kn011dm3p9n\\\\\\\/D+76jbaaO9jvsRqDorRE4gdTpxDO7DLO6sXM+vH5bsJMKnqZsjNTlsmUaIeCvYL9YZu0nkV0aiPRs5fwF06\\\\\\\/xVzSWJc\\\\\\\/ARVBXyWqnTqni8DS96RpqILnP\\\\\\\/tTYtE+EmFui7pB+wyJ8psgLRWBt7tj8iZr8R1Oqe85rkM5EIoaAbRcLhj9FgdjLGPIwChMe7nqFbYwzwi\\\\\\\/QfO2d6F3BFBz8rHnIoIo6nw2JhHfQnuStNQZEuNMzswsVsTIVQgWUUjnx+gX0xMfy\\\\\\\/Sw4BrJ3cZgJhUTjaVusxRZivzqpMUCMzy1+J9l8Kky2Ep4yQf+A5ag21VI4JeUdDm7UujcMzpQUwzEC5S\\\\\\\/Yuuhnu6lzPTcXk28yvUNda+YGruYKKrkMmo41O16yNTrjr1CzOMrrWrkamW2OCMgC9gUpzvKprD8kq5uENS5rOmv+OytTv8qnqR2Fs1FzFkYGZ5ltkT7YyIAhunWv9uCXDuuInEAJyJ3G0kOyEK4WJGcQQwGTD13w73j813J+rHR+9nwfvXo5V2j3GmPiFJZe2lNDSPzlTGu9cTaBOd6ETvMemXy2fwiZRgna9anMgHVwoct2fVRG5saxEaLBVFxVHL5TlxD3r0riQDL6WmEi5zvV9An8cXvKYVaMn66MusqYRw3K2kw3d0qDvA0t3SUw4RgrKmEtL\\\\\\\/SZt8Buny7nBVfmhLE8GTlWNm9qUqDrhBctbsmeFuVvUatnBxpglCYgP1COeq9nnrcO9IV2FI9GZbR41469FpXjxM6NLysQefDS35FJPTjVbCf7F68xqRFg7EMn\\\\\\\/zuiBM1PeajgJ5olol9SpbtNDIhweMr8MrNSr5CmJWxF3+vcmLt8dFevOW7F4cxAwWplBZJ2RD87\\\\\\\/m\\\\\\\/4bOgIBXlWF3a7Tgzv+gIpxeIiAq07lmyI7hXil\\\\\\\/n\\\\\\\/dFXAcHmvpzsqaHKDlYdHIy8HRUjTeHG9ddzN\\\\\\\/87GuubQPdHMzy0jXYoz0n+9bjFQpWoAYeAFsAe9n+7q30rsT7LE5ORtD5gZsyOgYRsoE\\\\\\\/Ww4V8x5OEF29UQOshKhgXJzuc0Dx9fXFjjJi4tu7OWDB5MM9ikO6mD4MfAot1fb0WlNS87bHbhMnN929ACTxoF\\\\\\\/tycuySQfv07mlQv7vk62thulDRp674pSJF7yAAPAJp098leTKYgI+RlQ8aiN\\\\\\\/zuGrpSC8\\\\\\\/LJ4Mj8PPGcUG9xYpNR6y4AN2AxbggzbzDoG\\\\\\\/+kTL8nP20EYGMqHDBFWNIBr91Bi17EQh54Vph9owSIhyQzNAMz9Sc\\\\\\\/zV7uV4bH4H1HsD7iibzG2pE6ahYNHowjxUOi+S699c+xsiFonNByBkTXC1jsOTyYneRF7piKLzhjTcb6W58G8vxHGexla\\\\\\\/32a+kUUFnQ2L574wAGFV3NvYpTyFq8lDmYw9ahqsUQD\\\\\\\/r+ZtMljwsDQc0AXpvSHbDe06b2IR91A7ZgoJTBGTyF4+kzuXazZnpHQbyp4fntRDnIPE8CW4YNNf1Ra28InoDCZOSWFXZOtDDISqOSYiMnP1WJilMKXURK9Y1kurm6d8NPEP\\\\\\\/Cc7SBDfdtGDxJIui4\\\\\\\/NHt2Uo2Zp5N+CkFZ77ZjO1UdtL0QtC95muIQjp7pAqGJO8ciKmJ6hYcCVPjS2FCxt6khEwha1vxa\\\\\\\/x4diZ9DuhLU953cCzGuXWw6wt7TYU7APwkx3az2UVmNlBa87c\\\\\\\/\\\\\\\/zgnfoN3XA5g5naZQnCcwnxkZ0NW+BkPyquWQq1NXC+5MUCOAFHQWNTm+qgMwdbYLzBFbgh\\\\\\\/jMW3ON1jjDXP3dqe3pEZPVmqDaaqCFQdKHC8YTlM54B5WSjUAs4jAPBh+i9LsGXdoe0G3IzuuOzmiOTp4cSWDsLY1rfoV3YjZxosrLQ7qkWefIA57pxAvuVney13shDnL8V+5FdplSkuNdNiyTKr\\\\\\\/kpiezh4BzTq1iHOEkPLd9Nz5OflfhQhKQrO8BQkVNSM7LVPqAq3ypR\\\\\\\/Yfm3sq1UUJ3NMPMT0ikxDirPc4mIJ51scWkRORY3Xygaaj3M0pRG5TyRczTXFlrJwthZjoUdIe4EG3ClQRepgsFwxXYzQFx6kT\\\\\\\/ahE8k5XBgBe7RIqjhyHgq7MTC9q7x5rCBGutt\\\\\\\/ksIetkq8lgPf0jVudx4RI5Ek+5FKzXY0r+n\\\\\\\/yu4zP5164cROudwJyaOfiCCmz7gI3xoXqQs74SgqbMmVbDU+CJjAPJr5zoNdL\\\\\\\/cixHGMUoI6ju2VZk5SAXeyJhmKh6ZU5JJJv\\\\\\\/LpygiNxUz5pnmHhAS3Hf\\\\\\\/e1fgFMtE6WOLGM\\\\\\\/ZNSOL\\\\\\\/hH8mo3d4HGgiqlD5UnfYDFWktLoonJubFjNWC+jw\\\\\\\/46ZqIRhESeHCupqCkB4f+\\\\\\\/e10MOXwUpjaaLl7IdyAP58SvRVij90S4Qyjx0yt3P0SAR+kqJ84S63lPl9x8zjrX9\\\\\\\/FwKtAKeY67e8Ald3h76UUtsCjIj\\\\\\\/nr6pmxJkrOWYXztraOFRGVPBb2DbUmxp+F0RHkQCC2NazE\\\\\\\/fTAz396f7pSa2d5EKnux3F6e+4XcA+ZfgStTpG3J7\\\\\\\/vpPIIX0dQLXt1gDZXQCz2Wm04rcA8ryjVyL1jqS4MtRDCw76lLnttjCVn6LcPdkPGs3eVGopjlWk9V2EVEJ99ZhvnJ7BsIdBoUF5+Ap43jCtuSvk+U49BCK2geQhf4a3laZZlzvJTF9WEISSfSIezvstXaYLnhwoPo6KV\\\\\\\/PU\\\\\\\/73vG4jdgz8QRXVvMTSBGpur5CKrlFORxaSizG0og1z1SVSGOOAfe4mmyOJoL7XcVAkvzR\\\\\\\/V7dVK\\\\\\\/g24+A71Tf7gmQkndCNRg+dvpxPYRn2MqYnynCNM+hl+h+Eo\\\\\\\/zr5PLFxhyB2WXylQisvp4s9AsudoTvZzMMFHZ88+hCt8udpS3knlNvguar3aD+Tk6GO73fMfvB6Js3bGIx\\\\\\\/fX13V9x753fmYHRdUcnc7RA81lD\\\\\\\/wIG7j54CPjY+Zu0fg3Gl6lCCA0H\\\\\\\/JNlJIMkUcYwv\\\\\\\/MekmOWnQZ59GsL3ULCbske7fZrJovs9uBKrJ6nITx3Dz2ipWHmZpHStQOx3hEV2QA149uhcpiNuyBlA0o5wx0lz+K3XTdfpdlujTC\\\\\\\/TcT5LAFCL9kuKrseQPnC1GUM7jezJqYLwbePGOY3cT7i9CWIwiQ7BqYwEsgwYmVJ0NrGTPsvc+sZxsIibnqTB\\\\\\\/ULbSDqt83LEnNOVUoBitF9tRCFUoPwDAJ75oid5ffxVeiREv1N+Uv4uFLSkgahxMcfumL\\\\\\\/A4knSqpFKw8ZRBumnqcM9Ifr21\\\\\\\/orn0kwUkdF72+HwPJ+0e7r66rnxFfGMXXD5bGop6kqNDplA341Ebju\\\\\\\/gwErNjb0RcIUPMkzF+qcveQGmuSs4aDXPsSCH4SpVKrenr+azX5yt8Kqjj8v42XdI5kX+PdMea23so3IORdgmNv7uMCURqc8Mtr2s8iNDhC5QRsZXQugoEoYuQ7ZfNWE3VIsDfreVlGA7EsCgT5iI9z51lZMj19911JXmWyeVsvtBfOZN4ttDdAm\\\\\\\/NtjYbsPYq\\\\\\\/qYIG8HLX+0kHohUfVhsLrsd8wb1sT3mbbVd9Hbq5GNvWHOw8lkTLg5Y2qzbJtWlz5sU3q1sWCvOreeW8BauvKh3BK8qkZPz6JArYqDVKnNCf0Gdzr7SyLcPHjHQpxOjKtOdukAAjz3uND3q0\\\\\\\/sgRkxKOQjfhiGh5yLWQ1kdwuzGX8tA37OBvxJXPRGluczi3MEs09DpoKpCjaDF+rAocIyzjlcwdSG6dNJbdwcdBdlGEB+eSfk95ilr6kJurxwvnxhAZjDnP8qT57E7zp+s7BnMBz01DrdTEwCj\\\\\\\/x2vHbxyC\\\\\\\/Rxr5PbegD7aO0vzjn59uIxo+xU7kFpj6XhPzYmZQNCm2wJGY7GI5bYWYxzDFnbN7FDuwhUynzJF6GOziSTQkm1XAahLo1+x7P9NmFEiwcWrT9giVbwos5vvrMyhk1MJEQ\\\\\\\/Ey56OeHj4+L9bB+6ePF1LlJhHRfPZMhGPKeP3pCu5SuFtcR5zy8DmVjNWm9ML++iDThFxRCMIQXrRv7mCPO3g7vF2XwaXc8oY2dapYaWa8ocYLD2MUOzRQtN7D8GFfU\\\\\\\/CSvedzdgDqHWDzkNaoCJE3FujTXh2MTHmD5sZzDAcqBFwqDJs6IgDuZYfNlqNA8WNXWcMza8IvaPJ0\\\\\\\/wwkpQzl4+19Ft7UC5T4thiYhEarov4zcDhlBcrhWoMSilwfm7oMfW16fgzkLCDBk\\\\\\\/RKBulpEjfxgWmhL3MHatuT7i6N+qd9HZZNnqJ71gMMovUPaWKlToXEeRMDoXIaObSdxtT6xCkXQcLY\\\\\\\/KoUIj2KBn8g1qAn2eKEx2togWFOaPgUBGqgv3ufqWTtq4K9yU9l+VQR9gkLc2ZF1q7+oJg0MbHFLAfNG9N3tfLjjjachWhFrbkiYLgWFcP\\\\\\\/nvi+WIIyc4lYKXMuWmHk4RM6Y0QwtMLa7xEuLIL+fhS0jr9jBpow0mVetLy17HztJ\\\\\\\/TLyBePiAEWN6j2GelOfWue89UeE3rObpEflziUzgvaCfSlEiiprlNqXXzGfNhOUG6jlMkI14nhhsl1J\\\\\\\/5+vELarbvGEdACq+hRtdNhQGKI66XAcKsQJnlNJG8+1pjuKaBQfoHlyg\\\\\\\/fkiOdPVDWzA8HrxpK+J7ztw+GZDha+6cWC8yievqmdCeZxBIEaHPLX0pP7vlQMll+VM1qPOu6AC9tIQztOOp1Zf7DoXUncVP1UKbcCDm3p2uQPFdSO5MNNAI3TMJfx7bahGpWT9ZLEzUaSgCDoMeV1YFMcceCAiTZhGj1mnesfWJmmsKf8\\\\\\\/9H0DZe0dJZYeGVrRoPXvPPQBn7nPlZ3oTahVmSVCXWjAfNJBsi0epJcZ+NEnQ8RLNIj4+0p1tCM33MhDWrVOAybKy8Qdz0DteWVD5HUuJKVquCgjbibKapnkavu\\\\\\\/NUUqFg56Ph7SJkEw8OclvqHG2+A7yxymzmrQLpsNbdUZoiHWid1OXopNAhmiBjhz21nthZ9QGXODXSXBVBXPzqU3Yi2CwPAbI\\\\\\\/2wDEehg4a6+VEbjncFoDMIewG\\\\\\\/tCi\\\\\\\/KE9AzIZzotinH0fnwOKyOxqS8ZqPyr9JVZE4x9vsy4NGDso\\\\\\\/HQemgdpwbejN4FLAxclWnFthLN0pk3ELxIxQJshjewmk9xXWRkHQDYIzlAxcHUZOQOAzxuBf8SasH9qhANMVCwcSHtznnDu0pHcWdmsFMj5YZstmvcIpL03IKiRoJ+eeCqD6KhOhF\\\\\\\/S2TO60QDzVkvfIWkEhwDVOEwSk1SeW98Rbxj3UKz29vhK2sd+cnkrMmdhPIbh0OyM0yduHVgEMuC9GjBJzneBl0CQCQphzj9TspXljcM9+PTrbLAuzUhg5PpItTnuBS17Yz7DCBFgJWMSuMG3s49N2LHAfPnQkiR+WoiGwnboBJV+tT+UP6XceV58mEis8RQjvFN8qshay673ZUgK9o7ENS5NZdP9JrrnCQI3UzA+zmdp2HcDJ6tvcU\\\\\\\/gTPC2f6Ugv6z+uGkgTpZyXA2liPBo8JaT0LlB6v6PxdSxYQNvg8RK+SWA6B+8KZzmmQYKrnlFeQ6Gj3bs455sE3UM\\\\\\\/azuip\\\\\\\/Wj+d4xBJtNWLH8DfpHwKgpH4FdVnnL9Vg9GYyYAOhrlhiikYCJTaSfeDdBGzFCOCKOJSy7YLi8PPusxT+3DOV9ZOJ2\\\\\\\/b0jHJeEKYJZ4Iy32Hf0W4iJhjdK7RFb2xiqh5ccq4FJt7ulNTUu+pSebkDq8zSItwMKur9Rs+cSKMXXceWRN9cJelL6t\\\\\\\/Y8xpIJfpsf4xDPdfXlUpGZlq+cW+x5xNvjXJtggn46ivmvRi6UvMASBWJvQwI85mehuoEemrmqLWjsZXqQFFvIjruqtGB5EIHLx+7Eb8TlPLuUq5eo\\\\\\\/SDrZh8s7zj+TXdNPdJrpF2GTRwIQif+0bXsVbttofqobzEsm21cTaJL1zKB6PrgwErcZIzYoEYVrD0lEotvAZR3OXHdaLSunRyoKPCWQ75A8Wt2w\\\\\\\/aDjj95GZoSmYgJu6lJAEhs038y8rOoiHEKM4UkMcJZv2REFXj62JRaq1+QpdMxHDoVmxURLs\\\\\\\/+fVsUbJlsDRL6tVps\\\\\\\/dOLFl7tute6iZ1xrPny0ODDr2HCNaf7D7XKHa\\\\\\\/5\\\\\\\/4Ndl2sQn+dwwWcBSayI8aZIoTGdG\\\\\\\/Q98+K\\\\\\\/Yml+kPKMMAKRMGyMkeC4QJMfQt5Laj3kN5BkTB0At+sbMgDwhygN2\\\\\\\/wDVb8LzfCFz3UmZvscyM5s\\\\\\\/OTj3dAsPsjDxh425FtPDE3Jp0CEMSPZVRNiTgTAwKVPyggpmntJLTGPBJBb9I6CPfSjUojKylVbkvVWoqR1i3xAj\\\\\\\/VTLD+01qasHoXhfKu7v7HztacsdhnzYekafceWaUJYB7UyjYCjFjMm+k4BCGwZWyu9RQb32\\\\\\\/iMagXyHW\\\\\\\/KNOKCF95nSWZabvbYQjiwW5QPe7GoyKRPa7MD0Wwms4TThWEO88UHB8ugXExbuX9aRlOFWs4mBXwK7es044xbxg5kRwuDXxv0g6NUwRfpjgJxdzST1ZQ7yFZQFJ\\\\\\\/\\\\\\\/6id3VVtZq8rPZYWniIAgzK4lrneJdI3AAy0aWAbI5kpwCakssnaQu61b0J25wUA7MRr6GXpIBM7Jjyx0nPoiJBAmROy26fj2Qy\\\\\\\/m5DqAYDrQkFHDgeojCgRNLoFa+u+HH1MF6c+tUHOAaNOclmYnaWzq67LX8GXJaCy7i+6V6XWIYuUeeoSQ+EQ\\\\\\\/eSRnPu5myE9CtBDe\\\\\\\/6fs1nJ4PanOeKVnLGjLSLa7iEVA5DSd3BmTU9LvNp3X1Etor\\\\\\\/VdwMaTNZaGHiG8FJx9j8O8PNmPyYuP5j7O7BQYGpUbzqDTLnQxkjynY4EjJJiqfbeTXJs0b4RScDIGudOlpLkHNFJKsb6j3+Za6Yoeki52urj87wBrd9rusAK+zfsV6EmSmDDJAllbwJdHSR8NZjE1tMmid3D4SolemV9hu8Dc68HvvHmSmH6GPztLv9130H5W1JDefrBcg4EkcdhHwI9Fj7dAxa0A9i5dJ41ZiGEyGHF0pyL0u1Tgvm2x8+MS+jZ+xI6\\\\\\\/4lKU4tegKB\\\\\\\/iLXNakwxzct+mBBzDmNT98RKxKZIkU+qiPH2VyEWyGT+F8wYvpitk7We5ZWqx8freh3yJ\\\\\\\/Zqo65YuDFFi9NqMDVlme36GYHx1Tb\\\\\\\/fAt1APfofxf1cSHfvippKP09s+AOpqj8+coW31oVuJRaT052euYBdtexT8p8rqaLzAQYLMahJEnXHzDrr6oVnFVxoHzNpLVveElKWcmJZDmRneJYxIwNcBvxPz\\\\\\\/M\\\\\\\/DWwALbgHvhFHdQdfV9g37kNZWeuMV8bf1oSJ1HCumYm0GZvDlmCrOGnKGkNS2r3w847m26tZpzMedYvXQWJk\\\\\\\/cG5Ogz9VNX1A7YYMNuvzsE0ELDpR+Y6Os52hGINJ1mqrayZWOSQmZ1pyDFNdJlCr+z2lrvFp9VrO0wxFRFkup0K8F\\\\\\\/Thu2Fw5bks1Li8gfdReMdw\\\\\\\/8LyUVyb+KvhLRDLqLPLO5cxYTjNYwEVmreHsaQ7YJnwCehIr8eK1dfYjd1MTAXHMmwLnNAfOlS2IhbJG5wUzZoGcXwlhqIhg+JOO0S6kUmdRbeAFA1dysZfIaXNIXvnoYl6TUXZuIsB1NFDIziN5OvXycx\\\\\\\/dW0ZxzQyr\\\\\\\/oyD+x1HcneKSptXgcTRbzBUxUFm\\\\\\\/Cq+0hUUW5d1bUQPu4GNJAOZPVxxkAzDHUXKWkefvC380WF+vZKJD8i8cPude7awJlScKpKJ8v2iYqwrIwmYKLKWQ+sZIKbFdY847KHJ929xNnxQSM7V6TvdKaiHsVMKIq82Vp6DLgxohSAsl6lFhFpiKpNId6zhSd73ObwPboJnxaBFOw15ayXWNvuMRfKmrMp4M4uiDk2OOZP92bXqB62pgB6G5I1Bu1hNPsm\\\\\\\/bjE6vR55u+Q3dVUFKtJrgVDGBqQsuFBkLNsqFsno7fSNSMOobBA69Fqherf6lfgAV6gxjXKpmUGpvYZy+yEU+\\\\\\\/qQAeVDr5EZWAu0N\\\\\\\/jaIERSJdLJsMVPqbdLsIJskAgsfAEv2FhqzPL8ODm5tCmJWe5oJLcUrRM0byalhMOozAYvxed3TbXm0e\\\\\\\/U+lwuuSah8c10vEGUJXXrRbF6c+9ALlynjGiVaiVRw6+ptNEndf379CK3FFTb\\\\\\\/hI5pW5rwQzMU1aOJwbN21vVy\\\\\\\/XCVaqO1UENJ\\\\\\\/hTaexT8X5lJemptE0OSxaEoIIkgsbMqgs\\\\\\\/NXvydDyC\\\\\\\/PVg4Dxs5IBKLttlJp7b7XKta5Vl48k9PghUhdZRua4EJ\\\\\\\/pywSwjvvMltol4MXR0GwnFIVE49MngD4+xSC5mqATUcWC6t5Vr5y\\\\\\\/GbkG0pXHweVb49obt9qJSIEPMtbTLUqZYIMQsIK8aBHxo6H7JCUc1n2SwMKFNDPCHMSnqFYScq0\\\\\\\/XqByRpDKwup9cUzatAW5\\\\\\\/Df8aLPot8BaojQcUW0ZCD\\\\\\\/QPNJ0nx3Hfzh3bytY\\\\\\\/jzJvjiE7f3NeG+srRYWg7jz6QqLW58VFJLxrVei7q1m\\\\\\\/JfUxKe0BeJvgOVcVEd2O8Qsz8Nyn4kztHxnVnrOwWLP8KzAWyLVhRlgvK2eA4jkzNnurUdGSu0Qix7Q9rAmHcxDHlq4\\\\\\\/ccLS9Yq5LG7L+DG+\\\\\\\/8nR1451KsI3LYkmXQL6bXjmH00xgvXBL4oLxPK5ggTvH73zgkizBblaITN2knN9sWwqWgFJY8TdBToCcXl7pusXRCO4XMj93OvVA32549X6rfqh4CJSbwG9mcQWBxX+AOAdbBs2XXvKWlDEBx1IwbHZizNtC7FGsjEtAVeRlngeXA23Bi+k6mqat3jVw1aAEOUnlx8vt9GcIK4N7fK52+DdAmVZefGZsrg4mZ+RRNhJrLOjAF0uUIwyY1HhdmD34y3OCecAXuuSiriqoXCIBbjWAgluJkBo9DRuTemAjkk\\\\\\\/IqDiQjm+R6ByqUsSVdatvyxqSu3NAlNw5u2tIhZyiGOBp5naucrecuEtwnIXfo97XVNVARQD3SWsJrD9sYWbZvgAQuFDZv4KaI9H6tPIMarW+X\\\\\\\/H8NLyLZDXtoedt3UkTzpIa0ow2kq6+qg\\\\\\\/2Y+X0CQB9Su77Yv46JEvmnsrdU2YbWidKV2ssF2wZ4\\\\\\\/wEVYVVM9dncnEdEe\\\\\\\/0tRxOVwViv7WgBZu2rKV28lZwkDxb3zyDbdsRBOcVqheCbHflq0qb+c2M3BQc\\\\\\\/ZIJx70Epl51ysK16q3QV9HNJUHPvJDOrPVnxt\\\\\\\/bMtMMvgrADa9ZO\\\\\\\/lmo2q2NKnG6LeRc7Sudr49bLbVrABYNp0q7ONShHI4VT4MBmFP8wFQpQAICqFAEf3pEugrIZCYMo+24yEXgExqmZh3FLAYKQKbJ9EFxiyXfHZayn7KYpJhHsMOm7z9veKhUQgL6oxuKZuzDt+Nwmy5WrbWebAlZZvnf5yTSR04WgaWqqNWdK3gv5Tz\\\\\\\/HzjOvKSvV8sZL55R\\\\\\\/Z33akn0z1W8s4mKOcc+TkMJrr8m6RJ1XqejbkDy8Aijpau55k8LY\\\\\\\/VGZfb9QA6+zMChQCDdzn1nPlOosQCf+DkBFMZxNuju6c0msNkjbMLiWXi5b3Ghu7UyYClT5Pn\\\\\\\/tBX+FrjXH6zahSZOHaXtEqomnD1+qX8+2fiOmnn\\\\\\\/O9nrIMNjx5HZLtJFtlEeI8xf3ngXMt0vypDa5PolPsUJYE3mxg1R3kTZyCCOlVp5gYwDPEChigGdU6I+x485XuINuYw02V3oHPMOT5GmXpnvU0OHI0nZUFOGaWUQqH4wjVF3stElDqRt1jgfRqUPVUQQB4YZd7r26MmsG4phzhYNX4BLhrm723wYc\\\\\\\/D3efOjKPCFscu5AVEmJTrh6CEeXCslBLKnYM5XCmZLhxs6h4l2C9XiQTcvr+QMtzrHyufzegouKVCTP8D4Yr4miupKdsA\\\\\\\/TbQu+hy1iKoaRiVWjLIXXLoa0N\\\\\\\/cOeNpP7lbV0J2bI8GlgAGhT\\\\\\\/DNCPLBJ25FOf+rXTfweaUZtqni4jLy2ps6NkbthfA\\\\\\\/1M4ELnCsLp+u97TmyzzFxgGkkXBGuJ3CTBimlvZewc3jML2ow+u+f2sW3jvJ8HgsyU5G1yzMLVpLVuRN61jQQme2LJ3cMRT8IVln\\\\\\\/NFxdHUzdtihWaOR3uw4n7NuMkqtfwycy2VR5\\\\\\\/d+r5Nczwc2BHRenN7vufv42oLqet55sDRRjitKAg3mrEVxh+U1k71+sXOog5x5pseLg2YgbrlfNI06Rvea5y78LH\\\\\\\/cl44bv+CriUhcAwbVs2m9INSumDoBwv7S0ciLkUUxqi3Tf2BbAItPaemkAsAHhNOQdY7Zat4U\\\\\\\/8058kdcnJTpJ9oywpA5ibD\\\\\\\/hn8hIIO4Ws5F2L8Tb+\\\\\\\/LDoEhvEwRnmLQ3QjgNmo3nSLz7P8mu53W588evvSpOoLkWILL1U+2gjO83M31nru08oddWWeGx19SHS7oJExMnc+I\\\\\\\/x3ClAyQP6dzW4VsNoU7n8AQT5qBeIblblxATpb5\\\\\\\/LgDKifTFGV9Kl5Qzlo9pUMf\\\\\\\/FXGRNCCVITrWWYQR1lO9LyJUJ8Z\\\\\\\/HQt4fgG6vHslUoOeSZM89J3aP2MBG1\\\\\\\/hC5jnPZZz8vzI5wk39q3\\\\\\\/WDM9R4r\\\\\\\/bD1rjRMH5JGmFHRe9M2IyiHbaQ5vwjPd0OFWf\\\\\\\/VnasfOZGVUdBUHbAII2zkSaba1i0eAIVbAfb1pOMvRrQ76EGuJu22m1YlwR\\\\\\\/Z2ORj+VCzUT7gYwcCISZS07C\\\\\\\/MMh+T5F77JuFmAi5OqvI4qhPXgyMDaj4bb5ZuDMkOaQlxsG5W5TUtAvOhZ9XPgtsNPg9bteJGQT8QAumSixI1ZYKF4QybQQyQEG63NXRdOtESW0rN907SOGE4HB+VOw3TDIaF\\\\\\\/sBtzCYn5qmPipd4B5\\\\\\\/kiMp4bsj3YrjF5xapmLmsyArCj+TII3QKVpsz1KsWdffj1+MVY2bd\\\\\\\/Ra8FkgYR\\\\\\\/D7Dhh7VtI3mpAPndtEDCEqzpTdb9SRqGm+0sR42857zjKb16BCGBHuK8tB9vdytgP6E+i7LNO\\\\\\\/H1llKiRgYb5VOXT7pB+ORmZzB6EYyORIJ5DnevP0zXZ7Pi\\\\\\\/kkgwHFo8LwuQGxhUsKf0U70j0yTSsIqvD7U\\\\\\\/CibMJ8PSH47JkQbGFr8AsdbKbBMsaEAmt2mXoo9iiU3A51ecVlqmP8RC3DX\\\\\\\/TpjK9ScBkOUNe7jlSoijXWdYtqamrId\\\\\\\/TfjiNATjLGjxdDJzd6ddRZZy82Kystv9VI6Qxd6QWcRj+LeDm7U\\\\\\\/z6JLbWD1IEry9nOXImMy94GMHPphc++i2bMjnoGhRH2wDnbZOwzbB4g75kw6NhtthaoTKs7SsZEuygpjMfAJtQE6ARUqZ2vbQW5f1FIfwxDzA4zeMZi6XPL0LiOomV\\\\\\\/P+\\\\\\\/B+TOwFEggJBeU4\\\\\\\/Wd3zwM2IHGNmk\\\\\\\/ugkm1plElsmuRsNQpGZf03fbof8VWv81jsetA7ArzcSIwlv\\\\\\\/L6BLMRH7EG06srTolc8W5v8YWq\\\\\\\/d3ORr+aj+\\\\\\\/b3nCir5tY5L2izR\\\\\\\/aaSsLMCnjyYYLH4myaIg8TkhTLQSCbS+Aa6+EflypYhEw1kmUlC3PNzquhsWFHOo0NZreqVd2Onl0ImJGvEfOQ6fkCJPnQ64Q2xuGZER8r3MoVO4MBcJcf+ZMxdz7y6pUhY4ivaxzUBMwqqlgJ+Q4bFH1SZEvdJkTHvZifN1UocmIcx9rFQwd9tjKC6lVo5d+5PRy9aBs87bxhtkRMB04Gpl3mfsgVl1YW9GDFA6AsKCi00urIkGc6RKq0sgeCtnUMPpQCrP6iinTGsO7mFbkg+WEvgWEHeeZmvCMtqWAKHvtl+OG7xSYRYof+7Qz4tYZ1ZRbURC\\\\\\\/ecdF5+kwPbiiTZeua14YftSleIPnTYUcMsN9c8ok6tsoeLh9XyUvseZpX3xXyFLbIGWe7eHm\\\\\\\/Z2wrifI3aRtt7T69SnKUKQriL\\\\\\\/Zvo5l5DxtyCAc6EjQPhR0MdxRUcurtCdq0PK3uAelWZ+asPiMLfuIDKHX1cdlq8VB7LB1U\\\\\\\/9malQnecJoEftHD7jM8bigm5VkEwnktYCq91VHyVRIiL2+SagZ3A26e5Wcc9PUoAYrETr2\\\\\\\/4CyVOIjP5Y4dUu6pVlIp9OO0QmtowsyzhS5Uk3eQbxSYG9oY79uAG5joaMkoQG7OEjvVmPl84aj0piliWsY1vfI7nAobhOZUNbmphgW38tAvYKKVgYA2kiU0Dr1A7PTJHo9yC8Ay4GcBcR24zs1MRr2Vq9yfXTOoNCHe2gEQ9MYBurTO+14Lf7Yr0YgIPvBobni+dmrtc3XapFvFNFRWsCs3O\\\\\\\/KDAtq7sgyRMHNq1sAhcQydQr4N5MAemdCfIOyq28XGj1TOQ2pRmmd79uvK4QTxBRc6b9iHx89fHHeWRb0M7J1nvhZa76bwnFSqKBWydeegnqEJfdpqwNkQohyDoQ7ipZ7AszQDo4b0GNTnSfBro+QDFiTq7PXXaY4pYfccrid5uz+FJbbfnJ9m+yif6BISrbDXPX\\\\\\\/mX2ON05jkjyqv25Azrc+VNDLYcSWpL3M\\\\\\\/gDvlEmcBCYqRZJUQTvoIgrrKtvN9mAE29+naewtBTbpk1SCbaS+fClQstS0JbWVPRYosBHhdIOZIw\\\\\\\/aDySCvfNVoKoL4vhDeRhBA7HYeFaCBcUYO4vEkHGo3\\\\\\\/MOuby3+kIw2LxaPk1Ax3iMn6gD2bGegUEmYyNoBOwgc+AysfF\\\\\\\/nowgFikr900UUcDnknbEghlPOCIVQckfaMMx5+ouiQRWnA77Pjz0hV4EQyCKH0i0VU2kj6NMaYhnbA2zyMu3HBauJhyuP\\\\\\\/Bue148O\\\\\\\/GvqKEn1sJFyVUL95PH9RJjR2bNRoasRpZM+gsJHeAmcadSRtQEPVKILx3MKoTDaLU0U\\\\\\\/H\\\\\\\/zGE40QhflPFt92FNowrq+DCXsb2zq2+ooZPKtAM6AYoF\\\\\\\/Oh\\\\\\\/nfXwrzK3iD6+rPBU\\\\\\\/QYVH5eqQbg6WZfOAXaaJnDq5bQwALQg1\\\\\\\/eazTUT4xvjSbNsh4fBSA3ynoC20\\\\\\\/EDb5822uSOum7Z4qfsQ9KlB\\\\\\\/Zk\\\\\\\/ypQciIqffMjXj3oOmXJ7N1JXUJ8Wxoy1eHutrkQqiFg\\\\\\\/IDvKbU4IcC\\\\\\\/Y3JXK6zWJsum\\\\\\\/x+QRfpLCeq+Zev0wTv77BBnQUP9P3zNHQ4yTo4BgFuT9i7BrIqpicdIuEN7fSsCRe0zWKd1xQAuYjnWtPPYiVujObBtJdxVq4J6hNaY7+68wDzIk1eAVg4R4ASIYgSeKDybH8AUyrITvWZbdqRQhJjSaoPavVqaWBLfuRTWTvQl2JkRZrfa9KIAAXcrFIDCx8iss3zvMtCyuXPiVM5DGoZSHA3AWgzzhUAHZh1WP1q5D9dU9waLCaGIG7Vswxl002MlZkDv2Xc8t5q85KMGBaaFmYHGBZBDwYvVHmlWOPx2fU7HlBJ+AedauHFeWErftphC2zEf6\\\\\\\/+GPu4hUBCs18auStfXqJiLS1Y7++xvAVvYo729lzS8jB36Pn4JJH\\\\\\\/gil7cczJUNvVqKCwApY\\\\\\\/uj4Pv9igYf7zg5\\\\\\\/J\\\\\\\/UFgOH11ie4sfqEUyN7KHciscNHMbZEAmvR\\\\\\\/duN+hOCWOT9QjTNOIadaJe3VDwOs9KJ0ReNpFYCS7SfochKtSciS8UI+PRyrtgc4Ui\\\\\\\/G5QuFwqQOtITctIrlLa\\\\\\\/RAzgufPcxSuPj\\\\\\\/PX02F\\\\\\\/BoMKwQRwS88RztZvHIiO+1C6RKZfHnfVdJoRtNhAC1xaljkf3f7h5GDUuhlld9x1zilkTLULrUDDBvEdBSBqVbX\\\\\\\/moWe3XpeKq482kWHvAwaSyGX4yqe+WIPqbwb4ZIsKZ1jFui5gUp8ucRcabnoWSstO8hlCynU3MV9B6A5tii0ZZnCn8UhTIUONZeGYugx1XgbFdr5Gfac5Qa5QeRQ\\\\\\\/4EY3+eftoVR0D+YwCyzp6IVLxsLLPnFYhKY3C1BRMbUmf+prD\\\\\\\/OS6w7eoijynYMS8zzCnhokbZTnpGBP8x690D5bn91054N1h5B1BHYfcCyCqLP3pLZdw9Q33bjTwcy9pl1qpT79ygwQiKTM3SjAu9jqDx6HYOiuJji9tel4NQjtCRgSTJRYbJ8IEbncuf5Ls\\\\\\\/diByTe2L5c\\\\\\\/R5BjqmNFICQwuV9hInPfywFK5OYNIw6GN6p3licanf594sWyNcOkjVNea7E3bJbq05ZlcCy3Q4RZx4rYsRCURyrVpm3HGn5AR0dfbtkqK0H6SY7cLJX7KrnX6i2Lg1h3Mdd1sAbnX2D1XCSysW07+os\\\\\\\/+zxw4roB6UI865sMgzLSMgdtUK65eMSdNYRWHnTchjB8qkCTcr+tSLdSyEs5dFO3fErvLwrlr1GV5dSPl1W0K3kaKkOAYwQ4SI0Ng9wjmcmMHjcFrOiB0lLRWZLIiQvp9KMa9UmRYOFn4Bt54kGVYuNzgfFghNmTjzAqNctR50Y1Kg4DK\\\\\\\/plnJ1DUuKoxYBhTjdsQRpL0zPSkcDjwF6zKnYK8HskjWpeDH+fX8LUtEbQe+tYrLhfZ6Kr2SM3UqzxWN1oDeXX1rNIGHDL70B4BZTJ1q+UXxnajiH3dkwVGxbc2vojFu5Rudj8hNXJlySkHNu0TTaEpOI1ldC+mWJPRYS4uCKkOCnKGNk\\\\\\\/LPDGzA5Wb\\\\\\\/e5REkD+w2LyIDzQpdLHXZ3k1ZTufrXQ61cYi2CXcy9hcOwKkOcDmAL9EEf83N6B61l3vwVZ8Bry9HWHQTO+IvxOnTvHg6BxX4agcCHsgWnbGnmu2Ob+8QnnoNXbulHBGRczcQGxkpIuJ0kcKti6rF3gcLMeNSRAKv9W52u6E+q0ulvHluj4zuUOyrJ2qW3A706rvlX0LCn0ZqYQgasXyyY3ggW4xgQBS5J9huD32eiQxEr2zquCllu2JOvZ3ffD7o0QqULfuOHgtgc0wruX1X7hLQi+12PuVnFM7VXdVduaTQ\\\\\\\/j3fICIb2eJJ2Com8iGVLQqP2v3ozioYwMnHScNYDSWu1kH3XE0XQ8ieb0tG3ejw2FLNFNr\\\\\\\/ygDX+NGfiCRowE+2yFFv8ul0eoOVsIjchAtMB3YS9+Sh1tzJtVDFXY\\\\\\\/1BAcMzQiylWg9Z17UXpTT2tgLhPALGkcMzr0m9fzNfPoTevGDzBGHvQawYmq7hDyFhIzID3NOHDfNEFO\\\\\\\/ZeRa2leP+s3Eiev\\\\\\\/7wRGrrD9REHjD3HkEcRcK6dMBlv3hEpeuwzxTK\\\\\\\/dJrW5JRKC4ahhFRsxsTcsrDTRD7iATXqvzrNCnzc9FlHZUdRPttNFyXKwnveIIr9vpSP6R8urR26yxxYw3czm3l6J96iwwh\\\\\\\/frXfxeRrloURUR0uIeeChdZBQi399OX7HsR3tOE88YAcQCeKzMMqQynREy7TjaSRGmOnuZC34d5lzVQVooLEEYC7oxqI3+4LJGUGbjIp6YXBUhiHRAcMXijEuuHrspOgINiKOd\\\\\\\/uNu60Yvj6i7AP4ewIGIB9ZtuOhzeErbHMlR9MX1YLJ0oNkRodcNdOcv0dtdvwBhu9XbgwjB79UolzuIZnoh\\\\\\\/NlxltRSHDZkRtbp8cvw\\\\\\\/CMnpqg7G\\\\\\\/DB0eTjMUrdPvk\\\\\\\/jQQdHuZ4G\\\\\\\/0LY5NggNhzzJxfM6RQXj70W7D84KdsIfMEvJvJVOhVphga1mV0WG5QPcGWXMhMtpRnSpsn8RbKbnA\\\\\\\/II42TRolTtnQ+ezmNVanlZ344kG3kws\\\\\\\/\\\\\\\/mgi48FkYXTCCHtHijD7eWM3X04qDQswmDdTp8pU5VMwDkC7H3UJpGezOBcssIQq7qYS6Wsgs2feaQqlm85\\\\\\\/ikBm40R6m1dKm\\\\\\\/aMlI7wgDa6qIhXlwW+up86PNeZ7b8Xd\\\\\\\/Ce6RHfQqno0I45lXtNoDHttYL\\\\\\\/lY4nIDKepgOkgMSUsKQghvuPW0tqdX4S1SvYPfcWhtCSfOJ0VIMotGhreVmQyy65mY2FPTulGsN42DnZP91WvzVJnVu3vVxL\\\\\\\/4k\\\\\\\/h2tKtF6TpL3XaHegEkVX20EGTTa3h3hXB\\\\\\\/uCofiIHz875bJ3yh3b8XRHNBTI8qSH6mn08byL\\\\\\\/iHgMF2GhRLEkn+nyCOY32yG2V\\\\\\\/\\\\\\\/ptPy\\\\\\\/NqIgchDuo7RlUu\\\\\\\/mojGtMsXAtTzqtcqrUuudmUOrTEUY0a\\\\\\\/\\\\\\\/kvl3fJLcC3bYToY4fzuvQGT1ERlMtWTBikLXlLymCwla04XW35ZOZAe324+Sigodg6epHAR1lxGur4HnOz58PMhdzEPDnGKEb1ntwX\\\\\\\/lctQ5wDurDfdHDMeADHZx4d\\\\\\\/IFPd963RdV\\\\\\\/GXEEuT3jN0\\\\\\\/2GWQtHqEZwIEI9dBBurgIPC\\\\\\\/jpKVTs3BedXgVedME2bz+tyF6f2BHD2t5w8wO7asCIw6vnOI1m3lbIC++CcH9OZDAWgUDLgbO6eymEOCxIno3WspYM1IoZ7WH\\\\\\\/0ZSoeo9o7ijVleijF6ZMOYy9\\\\\\\/U2Nlb0KMJLa8Qn06o\\\\\\\/jZZKQ3xAqJ+2fROqBVUtX5VvDoK83c9unt5EFfbhzdcNnXdup2GOHCzMl2TR3dB0rEqJMULO0dbTleSmWIZKVxEZIWt9ttfrTanRbcp6\\\\\\\/uW2GEh2hL3LANbGMa0ff5UIydmv3Ms60uk7UtZq11mg0xSi+q1BboQXjDdjqxYZAubQjgPEBBy9AdzQCGdA+MX+kjHokj5ZHh+DE1JBsC2ILog8TQNNJ2poIgw1tGxUstK6vhCQ0DEvIurF3nbByPItHQdtfkmTx5mEplyGkVBs+cx1Nh1sMFguXzY++iOkfO\\\\\\\/qRxlzrm18gqSgPfyzIaQ3jcPzB9SWogYH2tvHheg9ZWAohIWUcc4u160+9irDME51LEjHYuaMiid0b1gSEWF\\\\\\\/+eifrqGcSF0\\\\\\\/XY8rT1xZ+SzWkBpjxKsB8qu4djnEp5l1xP36RwRLLy+c\\\\\\\/uv7N\\\\\\\/FPeLKYabFQA\\\\\\\/Oko\\\\\\\/shqgboHzHMUwUclLSD4U5R90O6JfBc95XCpjUqI92CE2ybjagIsnub3tclLa+JofIo6OybTJ07Br1kPabWweQ7ZX3GBcTvZB6+JuYCraOcJ8zOCc0MJX0O+E+oWxdQTWBfov72JhnEBo12YTFO9Nzox1RmmBnOD1FewI513i5g5c4Hs+1BOLcaJVbUv8b9W9dhe8bLw6LkuqYE+aVTcieUVNFRwel906Q+NyOxl6VET3K5JylbdG26E+HJXTsZjTn7vDIz3cEPnT67\\\\\\\/DHuvkDqIkPYwzPLzidFIcA++2Wuhm4wPGQWBmKi7NNRH92JUdFs2Fx5Hf7ugyy+hUiNnTgIeTuwYgypUAj3hJgroK7+2jsIAiCA6Bx3ZXKXEnd1Awy2jlw+bMDMOgHEXkx0PD41eAbdTIOwDqGjQt64U68dmgYjRBvtc5FybcjeL9ppXdILnTBitYQnbPof0V7kj1CHqd0p02hY0Fjw2tkE4cfLV9o8ZBVWj3StbOOW+fZZARU0mEahfI5eR98E9Le4wQPWFQrn+7v0keJwRje57ZcccHTrySVG6Dd9C8O99ENoN+AH7GRnqUFWWpZjv3gaZGfi6ZzZeiD+Lokz\\\\\\\/TjRUMYh9D9XwcBTafrY6nKGvRPIWD8KgvvEjlfhTzP2adSmq7DcvlxFdOkKwlNnx09tMwWHyWRBJx0DIBNENVdgcgor8Fg1oxi5G65i0F3uHL0BGGHLRPvCAlSdsKZI5TYOnERxRKN\\\\\\\/ZD8ugfk0gI+FFX0WSPMAVd8o2sG8GuPxo+Kl9ZYininAw\\\\\\\/TwjyRrhCudNpbu8PNYr\\\\\\\/bz5bimBktIoQyWZwPLjOes1SqMQKEEqTkCEmsdvw5bJaCvOaVEXCFEdEt6fd3LDLgypreyO2dvbawM7DEkITskjrBL\\\\\\\/htLF6bB+Nfl4te\\\\\\\/RsZjR4OxzU0MAAocdvLJRLHbQkXYg4EJbgIz3FIR4LRgp2z\\\\\\\/IOmBs5be\\\\\\\/\\\\\\\/VMOx2qMteBR0uBG71vBo1lkmmgl4MjXpw7HD+Q5rwQ3MP8XU6GC2LLnUslJlKh9cf\\\\\\\/SRIR7AUPsmddKR2D3gJxP9L9sfkbLEZNCGHdHrUwDfWKkZi9cGAzE2P9G5ES2dV98aSA2qq21wkZtVeinmQLSBjJH09UBoVrMs6aqX5jOnwZ\\\\\\\/G7wtEz6TJOU2iBhzD5+LL7ocCGggC5JyMiFLyu9hsq1qrPAgoSW5PMmlf6As7SSPqFNlcHFnUaD18GbPlz2KJovHuRNsIIoG\\\\\\\/N1\\\\\\\/MpnpRpRxxRosOkrsyYWoKKeQHP7tEPpLv1cC4baHO0JSAuYetZMa238fFJDsCnBkJRF0g7PupBcPNLKtfHI+w1SmLb5VtQoKoMQ6P2pizKoMgiO8zTgnN9dCFC5FVy3H1PS0erVhsxqb28oGHShrFrMOGtn6bKwHMR1y6H1Nq4FzzaLuWbFjD0LcmSyk28WD2\\\\\\\/+qC8CnbFwm8l3sVUhkofvHPIuHd8aLQrPH2RTGeYoVsbTh24EHlLXnaj3A69YzRiCKWJs2tHd4DXo0lWX3mNE3fCsAfDzMr7wTjX45J9zbnrvN8BX9KLQkqQ3QKu0CEv99Q6p5Lvq3fz8WLjcvFXvqiL1ETjJpcTumI9Y8E6gIBRoUjyQ0O6KTROFsY1n61GtZ6e43B6v\\\\\\\/PiZm1NsugpON\\\\\\\/dHZ15HJach8UF8+Stm0ZLHl5x8ZJOSbseWC3DcNn8WZBtPE4cEZc+9z52m5ifzC0WZA37zwRD9tk1TRYLmGid0TFIj8e30YdEuOsqQwJpM\\\\\\\/AHsCeWkecyTrcKttJXaVzaq7QS0+rXjdCvzDvuX8RIJmlb6JUxnSaiCrAjrUjYpJlkQC1Li2eYxzDDJQcgLUGJdFZyvJMu1EMuXIJVV9JQ1mVKHd3DYvcjPxBgr+B23qG2Z6E1pcOTcZ7w45UFrehmglG7nWrP\\\\\\\/dnngqrbqh9PHtcL3ncIa4NG3Gc+69O53YeZNNjcxXWMMW3XZe+YBtLefwWaNgEWL+5sOZSx7bRQBc6ieqoK+sq08qJMcPJ+1YumLlhZxLldi4+u28jU\\\\\\\/kmjPJtsXIvRszdA9\\\\\\\/uIDGhiW1bBHr7TQQNFb1ci4AFihabr19X9yvu6qPd3yAQUZBsFR9bt2TYZrEz5+DNl8eLWYQpVlqujUf3961RMD9qkMRc\\\\\\\/Rqd0h\\\\\\\/zM+0MVvDtE7i7QbPI73mh5Syp5EIwTvgsBKRM0ylL281K4RJb1G6HWhcckMxcIDuYmVMCElgRHxHUs3qgx0KjBrbe7sYXEiBSiVJo2dd1MTXtsLalSbvrYu3hvAqHTLF70NTdmDpR3pRmk3VWPLyw3rsTKlHlEt0YzT9Bd\\\\\\\/d92dKZI6XTxzuy2zgXMdYWJk\\\\\\\/UYkQSvN8h5j3+bteAgVS+zT7O39rJlrJgxjHvUdfFnN0zGPWYWlU6QzmIRgLV+n+R0WSHlkQ\\\\\\\/3TQbgwGvB0ct\\\\\\\/o9wHbwaDb6joGaEfgpLQW6k7XkJVgVP0+d1kVVcp9sLAmNPT0DO437KCah31FD33uUefVBlmvoiquwUoxWfmEvmaLJpJ97t4ibKfk5lYMHlBx+jEppaBHEE3ozHDDaNpo528+s+llJpxAGGb\\\\\\\/24QDAxlELIduQjA4dCF3G4\\\\\\\/\\\\\\\/xNla+qBa2Qatld8QlrTR31FvPDplp5sbXS3K3fsNZGXiED1owqAgLweOl9rednfExCLgR0DBzpqEMGUgQanWhP\\\\\\\/Cr1ibDxUmxxuynWLlT92mrmpCDbMRCszGAmxiXytIhLKqh9rjz0sVx8\\\\\\\/GxQ8t8Ky0r4A6YqnUWFuXNYiK2nmz46os6GdBmMKJzYyW1E2btiLBPFwI25I1gkM0VVab+I1P5JTgYNzComOsXFoeNAQM+FRX1m+Zztd++DjX9cr+zSU9I\\\\\\\/qXi+FyO0Z6QnQWR6QKtauc8uou5thS3sZNgWj3re0Ieeq8FvO4tc8DVAXWk9MgfsONqFUIGFVA51utRvSq3IiIQdMMaYl\\\\\\\/dRZjWh5ndr91WegtLB4uxtr8LzWFIBe83vN934Seagi\\\\\\\/NU81XObsiW+aHJ8bI\\\\\\\/wYcT8U9pa6WvSMQ8frNtvh9\\\\\\\/cL1qtyz042zU43ZKj75WGOg9ByVhfPQdSi3\\\\\\\/vwUOKzcqDpmkeBeANJKe\\\\\\\/gaPqZa87Y9c1fgDxQx3n8iOW+Z1jKFuC9JuJAZW4o7s894HAi7jD7z8Ux7PkcgQItoQd9UeythRtXpaQBH9Yeou8CNKXlLYHMuTNY\\\\\\\/0VXclNhEW58XT7n76uaCCPv12LWZfZRm5iIflvww650y9AGaQMFHZz3EqWRZMhJeMayT1cGpnAbinWBmvJd7lfiYrB78CnjPeIxNBtW564Sz1Qp7L\\\\\\\/ytSSHhI7LnTU49m0\\\\\\\/JSE6Rhcqf0Qd6VkA\\\\\\\/Oqbjyrn+WrGVS1sRyJmpqmlWeDJd+zPQ6RDt40t1JKV8\\\\\\\/ZifDQMysUCMnMllLOH4RX\\\\\\\/\\\\\\\/i7Sw20j43vGmbYZkyaDH6\\\\\\\/HADNVYI490mW19Ws0Ed20Wn0Kei1nV6Vs\\\\\\\/1M4i9QVmDorB4caoxXTXYRfgK8nJq4pK9XeoUPli\\\\\\\/OCS40Y5XrHSmfrxYx52yM6m342yORAnj8jJl5wtzCTkbLeElCG7NzU8Jo7+hSQQny\\\\\\\/hKtA3MvO6qTOXxpIv7ydnHEaaTN5iUPDr3JpvLwU6rfg+AbuUYgThRs4m2lpzr2UdpHZQGou92o4Iw2tchii88moEi+veWONc+wWHF2x6iTD6fs2kp\\\\\\\/0XKQ5ZdkJJpvPLtZ+6dPudTjzplMCHxCFWpOWCy90Y5W82XnTTMck4oxbIhw8uWKzj1hvPQrdjxKOAOD3VE5CQ0E9LFnpq7h23mYnns8UpELZF1ftKSuGp++PM6MUTViaWqw3ewL6G8WBzr20FttCI89aqhNxv86B54R1l3b0idpHpfJBj4quZ5WIGl5A8OpuHEYuWVgQewiG8OPRCw5On04y0t5Thjnh2C311su3vjCNqYTaOrAiNULCmRVzqicZOzOxBpe1dNDgr5sB7R9erurwqWHSS8IeqytzsSrBv3I421wnmG+9Ro6CM02vNiYgjYcHNcUx5ea2tp\\\\\\\/SFqgsWqBzw4waFJ96gSGEDABMQgiTd0b352IPfiVtoHDM2iDyyWmhj+vQal5wcQZubDwFoY4qn\\\\\\\/SI4mZyrAwvQa6bBkA5oOdhhVQexe81+55Jf2Z2g8SjaP36MqU5gvmK+IU7zkMMzQxDWNOtVAJ3dyxfyFswsfj6zmMl2oEKHeIXVcQLxHjUy9axhQX\\\\\\\/wFmgjup0o1QrapUSPkScsgxTV46Sbc8\\\\\\\/HExMpaNjVr0eNkIIsqIs7DhpcztrSyU8x4NaGfxxPoS\\\\\\\/rbXM8SdkvXj8lUGaXtXbXmnuBulww4EjjxnQS6K2Ka6XsTyp7aaofUdpk\\\\\\\/J4mcScKZnT9plCpWEYplSkWoeFFKg3yI4gOMGSXDuqZQSTJSeRqhItGDwE3Dc3Nk6yDIiQyYz9IdZybMIqO5TmZjiJi6S6NR901bA5vfng+VbZic++eAcTnx5Vc8bo8cDohZ0GP5dRaO\\\\\\\/lEkOsk9BT\\\\\\\/LOAjX\\\\\\\/EcK1mNt+GtGaNH0BpepBsYsiHz1ZD1PsWRdV+6my7pUS7oZR79R5TPebfSGfWerXRN9wqo2ZWnGmHDbjaDHMAoHtxR+WJ+2C5Sw63vltcmSXr5YpzpYO8qKG+dPTD94RQaOq17XwqpTlo2nGMXq4wkot4noCJCSus\\\\\\\/rp8icGXsx0oJlmux7zOCXfHRfRVUlTa6+lewuz5VhCEyqUXhaLoqGhAChWwBOWTLCeQDzQaBCUa0iemsV8FyKY6ZBlzs4+bBbPquWPURthYV3A09alrNbuOoSnPxdOkzZcWtJFh+Xu5wBdUWRPEbV7JbRjET1MN4pqbdQh4P\\\\\\\/vw0rtwTYXRYCYlp\\\\\\\/KNjQw54RWA7Aop\\\\\\\/RnaQL\\\\\\\/hAkq4mAe\\\\\\\/\\\\\\\/OXXVpXxtT\\\\\\\/vGvKLjeZnZwTzY9psHg6QEsf9uNJQWEINsbWM4ymNmQ\\\\\\\/My\\\\\\\/Kgdru8I\\\\\\\/TTwwsRNGg68B1l6KVsj+2iJ0okW\\\\\\\/BDnzSXlfE\\\\\\\/pyKklv08rp8WuSbp4708tdmkcVuTUXEEUzlPtBTyRrqLZ\\\\\\\/0p0QtWUCXSQ\\\\\\\/SuXIbZWRopNaw1xca8oJnsmTEDRfQ2IleA5VH0lhCMWJlyZrXLut\\\\\\\/LbquSwiW6UKUl0qZmEvf739jTa9QTrrvZJyX0xjS2xjIA5IC7GVi5Y5ty6ENkrfVS\\\\\\\/EBi1ZnXab86Pi5IqVvp5IcyMV1j1MIuMpmlOchV28ueSeSCZr\\\\\\\/H18bVpBDyqcOD3AoxhNS8KL5fe1tcg3meuKOll1Ev\\\\\\\/GP5bWqVMRFkSCo713arnCskLqxPVRcrsqo2V7MbmgpeJopD1rbabO30pYg2ALlJmeIDgm0uCf8pWM6XJxFOgTPzVebcKdJ93hjLQt80\\\\\\\/sOAcmfm\\\\\\\/BQBfTqaGx8lFHSQWB27hUBwehW9T4X3gUkRpqom9jnhkPCp2gQB3alryd2SLd8ugFeMTF61myJuyCmH6OH9ftnHEgR3QJpababn0PAlMs+pXJZIeunPL7FVxaNzs27NokKcfZZRxYDysMYqgavhVXJ4LAtIy0SxEmi\\\\\\\/oFLIKEyanzv6sRoEELoxqeFmrWkSW5ELdKHTQMPCYkzT682Af2PHJxkgCDjAmULyjq2CEhu0BYZqIHIk9hGy5vQ\\\\\\\/vm5KMMi4xiaAMkJaZ6XkxyghDQitwkm3Rg9E8CvvOwv04G9rLjT54pDlcQstVG3neHBYTViv5IKG0itVcN0DpRrH70R2an7H+MgRQGU98D7a8uWlNSx4Fld8hoSiDB+mZ3tkvCYdF2mc25ci6WNyks7xP5u05Cs8gYWqX\\\\\\\/vEeYSsAcwDY6ra9v5kwEDQtgq4aevLQCpWKMD6IzNe4a\\\\\\\/Al9CatC845G3dDzLVU+eOR\\\\\\\/x7tP+yNez+HrQh47vhRnmtIwTIHqNLqPRPkvmJ98yU\\\\\\\/uMiKN6sTsQ4AaK\\\\\\\/V7WYSCjG+FkL1uKcMrnodiQg6NITg7tIQ1G6M7yKMqVLcJxkIF6HPnCjFsFEhLJ8lB5AqUUI4laEOJza4KiztRfwGqTe7M7qbOsjA0KoVCbq9b8yY\\\\\\\/IexGouRPqefl0Kg1T3m6sgM71yOuu6+Ir9rEpsAFZENv2QgkNf\\\\\\\/VBPtXcQzlIQUJzIc0t5a+fPRu9efErnU\\\\\\\/Rl\\\\\\\/k2ke8emVfobmlL0T8WARsN2fin0RQ2LnbKKztAoWxJjIILGamHGGSqBryLTC8QlJJT11aevQlug1qNpCDx86EY89Dt3XufDjAaV40BB3wKOB17vNuKqRpLrA\\\\\\\/iYi7GqaGSqn56nEYktx2tJew9jYt4W7DGP0M9JCrhfaksIFLvi9DDSh1+Y3AjjoNoV\\\\\\\/sKpfmZbz+0rt24ly10pgftsbduFaDltpgpuIPDPu0OnILDmFxMxa2m2D8HBkf7a1cBOK78E5jfF6lKUmbFUrnCr78N90svEn\\\\\\\/KTMzOOe0Q1KrkoINnTobXwQ4RHuB+meVcmHkpFLRfeXLVvCAIcSkJxk+al09LShS8XUdKbSfVSogK+ueFL7HPBatb7MiOOTasfkM7gfMp7MbJx5tl19jGVDAszDiXmTCvcBR+ILKEvxigzB\\\\\\\/0FpgD2iSBWTUd8ypBVne5Q7Wp4bUHsRW2wQnA+cLLdxbgcZryqWxE2Sn2XZBx4t\\\\\\\/ka2gUAkK0t+XYakp2tziII6dubTfr3xMKnPPCgOZ0H4zSyn8g2OFcBM3c1TucdHuQ7fbKv6PkhoK63tFVr1Q5rJObAOzAZIFYwF86c8Ir3snjbbFCL2lBl2jEfBayKOSeC9oqmTJO46L+tiOhvoKY4PDtiAdBI7aizttKPYL2vIXR2IqOGqlQNrArcPPQnU8Sb+6BIw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/2dZhQ\\\\\\\/A6AaDVJUNYY7KVA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227c0f9600637012c8c558c4bf581a5fe87c4f9c3400a9f1cf94356b13ce7d1dd4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MkngY997vBkJc3jCeikIKIzEvSjMT1qaQd86yKQcWeBJBCYR8Sj3eQqMTQwCErmFOZnvx3lekYM+2C\\\\\\\/a+EDwlRtVs2xkiizTpzwLFOy6MMJiT\\\\\\\/npvUxTuP1gFGcAUvEYsjNxdANhGEcymcaFj1tFWCB3XNYQhx1GkQ\\\\\\\/E\\\\\\\/kGMLFl1rVMULQ+HSjNZTboedN1cUxpQwf8Ezl1xx1gXnzrkYGMyTVQowl2W4idZuKgV9r5bx51MeMcSd0HamITcXeIbuJfomQKimEaMRyJK+Fs0E+mi7+lm9ZCj96DwE9r+XQh7GRGOOGMjrNKM3KJrBOZ+J3YItLndG6wP3QvSdDOKw5r6N3fFhRXGR4RC4B42l6zmQCqdQvBtR1ccnERhrO\\\\\\\/M\\\\\\\/oyHooRFfMsi3\\\\\\\/bWesdC3pRD3zpldnj5CJ\\\\\\\/CaNO7ueAabHP9eKLTdUFWtnxEmYnnJ5nKlgoXlqiAqTeJmoUAHpDESOeMNDWI4C9R9MrLJiKkPnCksmNiJRksup59yC5kHOPCx1+An5uETplmyBJr77MzTi222YbOKo0n3yjNonuL5OGwAIgtjMLmtR3s+k6P2eAVbjE6wssHJ5oi6qshhfGo33CIsPf5dkh3hveX9iwROw48lcBQQunlu6YVia7glexbidvls7NTHcuDkJcpmCV4K\\\\\\\/b8c\\\\\\\/QUpBnmGlsgH+v2NYld2IOBvUCOCQlfUa5haio747OTgdE+Kw1CQcFG1QJJIUasJTRdUemx6DAgOMFBSV5jR9GNebMPbRpBI8KojW5HDmeaJ11RvE3ef1wsD\\\\\\\/t7YMxlOQSXRjqN+DZ5gBz5o3eKhnoy+sRKW8K0hKuRTuTUUswfK5s1ZXlCiZbdmg8Q8gTPhavPNspzLBgTGReIalcmeFJ53CmYfMBp+5aQF72SkGEVQSSl6CZf4h6ntRdAkpHUG\\\\\\\/oW514yf7uaP6UJFQ+6VonOkCLWJiIUtvcKPll5y7DYESbIp8diepGAz8s2woNbeCL7T849EKqJ7hdLgzdrdU+IDI3RX5GWYO5pG1dUS43XT7fNPSsqLINarCHPAGwTllVM17TgzS3RSB1EY\\\\\\\/G\\\\\\\/iIV9jHR8v6firRA5J2DJOQakvokjcw2Roo5i2qLLCLRG1Ml71StA0MTG4veu4DI4w2TFQSEHFV1B1kYwlT0zS57Xw\\\\\\\/JjnRlTKJJn28sZVyAXDyQrm\\\\\\\/IZYZXyaPaifRdzGEU0Of4jI71V8tfV2b4fJ5a2Eu8ptmqlMj4etNbQvhVN09Xbx9t+42Dun+IPHoUNOgD3gM3J9WD+qhvN9jlNErIwqMgGREXaRCKkDXOJ0b1wflWiuBd3+QSrCnlnu\\\\\\\/kLHpEbCbAB7s9WpWNt8\\\\\\\/eNXqkjTo\\\\\\\/m0puV4I\\\\\\\/f9OShqoRrWV5YilaPyn2n277UYvD6jxXC6t3qcC4fCqDmtnP53wEgMKtv5XTXOtlR4NrxCPB6FuVTSGIU7C2IYUTucPZ5Kr11e1QgvdVQ9rInDXjQBwPDqGSH+2\\\\\\\/pQH+CjPuffT23fHWA2hlcrBOH26mXAr09YgisF2UQsKFXCkFALDd1zXmsPataGDa1FvNXpV8E0I9YIN9jdD+S9z5uDROMlj7z0nt8oyXi0HQRqgEcCjP27W5d1CuWSz8RitYIbxEJ8pJHc\\\\\\\/zXrhdzVAj+bKbb3Cu2C0a4WoY+1ZuPeuGRAtJBawE4aJ2eUt94KdmXFEENtKyaV5G8d5NRZsXj3ZguIpoh5TK3XPoK63q1bFDLmrJZr34hVqHB9BXWcoEy\\\\\\\/Yr340wY16gT7z3Oi29NUmYLbvoLQklfJh8W84+tPCsj+UO1vIGHljYLJy3Yldm7gGH2n\\\\\\\/YNLoT\\\\\\\/dLZl5+q56WfcEneNfGbU7Jepb6FWf\\\\\\\/BhBnptX0CHllFloLzKjw78A\\\\\\\/\\\\\\\/kxXDprrcSWQqi4+TFvtS9ByXZ2tP+CiSPoGqza8YTUKe2UwCTcpXus1SBJWP1A2Qil19GPV5FSQ7c7Nk2QI50tBWQr81nCts\\\\\\\/RFDDqj2a35hZyq8E0w+fc5RS\\\\\\\/Zz5jg==\\\u0022,\\\u0022iv\\\u0022:\\\u00229SHezPakZJYUTRj+aAORtA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c4e3307f38646ed940acbd4972e4c08188cbda3ced30ee0e3dd1d711e6cd3546\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460056 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/281" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K03jc8rhaYlQcLxv227rEai4d_tGC0mvRxsEmomPOQ8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/281" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iNTv1Iou4_QmrGl6CA-VoudTiDZmxtxHan62_Zh6bpA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/263" title="The Mouse Universal Genotyping Array: From Substrains to Subspecies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/287" title="SAGA DUB-Ubp8 Deubiquitylates Centromeric Histone Variant Cse4" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460056" id="node460056" data-pisa="ggg;6/2/281" data-pisa-master="ggg;g3.115.023739" data-apath="/ggg/6/2/281.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">argyle: An R Package for Analysis of Illumina Genotyping Arrays</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew P. Morgan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">281-286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023739 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460056" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew P. Morgan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:apm@email.unc.edu" class="" data-icon-position="" data-hide-link-title="0">apm@email.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/2/281" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460056" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/281.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460056" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/281.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460056" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/2/281.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460056" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/2/281.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460056" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460056/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genotyping microarrays are an important and widely-used tool in genetics. I present argyle, an R package for analysis of genotyping array data tailored to Illumina arrays. The goal of the argyle package is to provide simple, expressive tools for nonexpert users to perform quality checks and exploratory analyses of genotyping data. To these ends, the package consists of a suite of quality-control functions, normalization procedures, and utilities for visually and statistically summarizing such data. Format-conversion tools allow interoperability with popular software packages for analysis of genetic data including PLINK, R/qtl and DOQTL. Detailed vignettes demonstrating common use cases are included as supporting information. argyle bridges the gap between the low-level tasks of quality control and high-level tasks of genetic analysis. It is freely available at <a href="https://github.com/andrewparkermorgan/argyle">https://github.com/andrewparkermorgan/argyle</a> and has been submitted to Bioconductor.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/snp-microarrays" class="hw-term hw-article-keyword hw-article-keyword-snp-microarrays" rel="nofollow">SNP microarrays</a></li><li class="kwd"><a href="/keyword/genotyping" class="hw-term hw-article-keyword hw-article-keyword-genotyping" rel="nofollow">genotyping</a></li><li class="kwd"><a href="/keyword/software" class="hw-term hw-article-keyword hw-article-keyword-software" rel="nofollow">software</a></li></ul><p id="p-3">High-throughput genotyping of tens of thousands of single nucleotide polymorphisms (SNPs) using microarrays is common practice in both laboratory and population genetics. Genotypes at a dense panel of biallelic markers with a low rate of missing data are a valuable resource for breeding, marker-assisted selection, genetic mapping, and analyses of population structure. The Illumina Infinium system is one popular and cost-effective (approximately $100/sample) platform. Custom Illumina arrays are available for many organisms of research, agricultural, or ecological interest including mouse (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Morgan <em>et al.</em> 2015</a>, this issue), dog, cat (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Willet and Haase 2014</a>), chicken, cow, pig, horse, sheep (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kijas <em>et al.</em> 2009</a>), salmon (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Johnston <em>et al.</em> 2013</a>), and cotton (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hulse-Kemp <em>et al.</em> 2015</a>).</p><p id="p-4">Infinium arrays consist of many thousands of short invariant oligonucleotide probes conjugated to silica beads. Sample DNA is hybridized to the probes and a single-base, hybridization-dependent extension reaction is performed at the target SNP. Alternate alleles (herein denoted A and B) are labeled with different fluorophores (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Steemers <em>et al.</em> 2006</a>). Raw fluorescence intensity from the two color channels is processed into a discrete genotype call at each SNP, and both the total intensity from both channels, and the relative intensity in one channel <em>vs.</em> the other, are informative for copy number.</p><p id="p-5">Many tools, both open-source and proprietary, already exist for postprocessing of raw hybridization intensity data. R packages include beadarray (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dunning <em>et al.</em> 2007</a>), lumi (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Du <em>et al.</em> 2008</a>), and crlmm (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ritchie <em>et al.</em> 2009</a>) among others. Illumina’s proprietary BeadStudio software is widely used by commercial laboratories and core facilities. BeadStudio applies a six-step “affine normalization” (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Peiffer 2006</a>), which pools data across many probes and many arrays. Intensities from the two color channels (herein denoted <em>x</em> and <em>y</em>) are transformed to lie in the standard coordinate plane, with homozygous genotypes near the <em>x</em> and <em>y</em> axes, heterozygous genotypes approximately on the <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-1.gif"/></noscript></span></span> diagonal, and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-2.gif"/></noscript></span></span>. Biallelic genotypes are then called by clustering in this space.</p><p id="p-6">Fewer tools exist for downstream quality control, exploratory analysis and interpretation of genotype calls jointly with underlying hybridization intensity data. To fill this gap, I present argyle, a package for the R statistical computing environment. The purpose of argyle is to provide simple and flexible tools for programmatic access to data from SNP arrays, with an emphasis on visualization. Although some functionality is tailored to Illumina arrays, many of the features are general enough to accommodate any dataset that can be expressed as a matrix of genotypes at biallelic markers. The main text of this paper outlines the key features of argyle; detailed code vignettes are provided as supplementary material.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-7">The design of argyle is inspired by the PLINK software [<a href="https://www.cog-genomics.org/plink2/">https://www.cog-genomics.org/plink2/</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Purcell <em>et al.</em> (2007)</a>]. A PLINK fileset has three parts: a genotype matrix, a marker map, and a “pedigree” (sample and family metadata) file. Likewise, the central data structure in argyle (the genotypes object) stores a matrix of genotype calls, and hybridization-intensity data when available, in parallel with a marker map and sample metadata. A genotypes object is therefore a self-contained and largely self-describing representation of a genotyping dataset. Installation of the package is described in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1/FileS1.pdf">File S1</a>, and the genotypes object is described in further detail in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1/FileS2.pdf">File S2</a>.</p><p id="p-8">This package explicitly favors <em>simplicity</em> and <em>readability</em> of code over raw efficiency. It is appropriate for the “medium-sized” data—tens of thousands of markers and hundreds of individuals—regularly encountered in experimental contexts. Users with larger datasets such as those routinely collected in human genetics—millions of markers and thousands of individuals—that do not fit comfortably in memory should explore more sophisticated R packages (such as the GenABEL suite: <a href="http://www.genabel.org/">http://www.genabel.org/</a>).</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-9">Source code for argyle and example datasets used to generate the figures in this manuscript are available on GitHub: <a href="https://github.com/andrewparkermorgan/argyle">https://github.com/andrewparkermorgan/argyle</a>.</p></div></div><div class="section other1" id="sec-3"><h2 class="">Quality Control</h2><p id="p-10">Removal of poorly-performing markers and poor-quality samples is an important precursor to genetic analysis. Failed arrays are characterized by aberrant intensity distributions, excess of missing and heterozygous calls, or both. A summary plot (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) facilitates the identification of low-quality samples. Concordance between biological sex and sex inferred from calls on the sex chromosomes is also useful for identifying contaminated or swapped samples. Failed arrays can be flagged and removed using global or subgroup-specific thresholds. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1/FileS3.pdf">File S3</a> for a worked example.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quality-control summary plot. Distribution of genotype calls is shown in upper panel, and a contour plot of intensity distributions across samples is shown in lower panel. Samples failing quality thresholds are marked with an open dot in the upper panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Quality-control summary plot. Distribution of genotype calls is shown in upper panel, and a contour plot of intensity distributions across samples is shown in lower panel. Samples failing quality thresholds are marked with an open dot in the upper panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F1.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/2/281/F1.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467557" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Quality-control summary plot. Distribution of genotype calls is shown in upper panel, and a contour plot of intensity distributions across samples is shown in lower panel. Samples failing quality thresholds are marked with an open dot in the upper panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">In addition to global summaries, argyle provides easy access to hybridization intensity data from individual probes. Inspection of “cluster plots” for individual probes is useful for confirming the accuracy of genotype calls and diagnosing poorly-performing markers (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). A dotplot (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) permits direct inspection of genotype calls at multiple markers over small genomic regions.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cluster plots for individual markers. Each point represents a single sample; points are colored according to genotype call, expressed as number of copies of the nonreference allele. The marker on the left performs as expected: the three canonical clusters are present in the expected locations. The marker on the right may be genotyped incorrectly: the homozygous reference cluster (red) is missing, and the nominally heterozygous samples (blue) fall into two clusters. This marker merits further inspection. For example, one nominally heterozygous cluster may correspond to homozygosity for the reference allele or, the marker may be detecting paralogous variation at off-target loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cluster plots for individual markers. Each point represents a single sample; points are colored according to genotype call, expressed as number of copies of the nonreference allele. The marker on the left performs as expected: the three canonical clusters are present in the expected locations. The marker on the right may be genotyped incorrectly: the homozygous reference cluster (red) is missing, and the nominally heterozygous samples (blue) fall into two clusters. This marker merits further inspection. For example, one nominally heterozygous cluster may correspond to homozygosity for the reference allele or, the marker may be detecting paralogous variation at off-target loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F2.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/2/281/F2.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467597" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-13" class="first-child">Cluster plots for individual markers. Each point represents a single sample; points are colored according to genotype call, expressed as number of copies of the nonreference allele. The marker on the left performs as expected: the three canonical clusters are present in the expected locations. The marker on the right may be genotyped incorrectly: the homozygous reference cluster (red) is missing, and the nominally heterozygous samples (blue) fall into two clusters. This marker merits further inspection. For example, one nominally heterozygous cluster may correspond to homozygosity for the reference allele or, the marker may be detecting paralogous variation at off-target loci.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dotplot representation of genotypes among nine wild-caught mice on proximal chromosome 19 (from Yang et al. 2011). Genotype calls are coded as counts of the reference allele, and points are colored according to genotype call. Blank spaces indicate missing calls. Markers are plotted with constant spacing in the main panel; gray lines indicate physical position along the chromosome in megabases (Mbp)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dotplot representation of genotypes among nine wild-caught mice on proximal chromosome 19 (from Yang &lt;em&gt;et al.&lt;/em&gt; 2011). Genotype calls are coded as counts of the reference allele, and points are colored according to genotype call. Blank spaces indicate missing calls. Markers are plotted with constant spacing in the main panel; gray lines indicate physical position along the chromosome in megabases (Mbp).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F3.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/2/281/F3.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467412" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-14" class="first-child">Dotplot representation of genotypes among nine wild-caught mice on proximal chromosome 19 (from <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Yang <em>et al.</em> 2011</a>). Genotype calls are coded as counts of the reference allele, and points are colored according to genotype call. Blank spaces indicate missing calls. Markers are plotted with constant spacing in the main panel; gray lines indicate physical position along the chromosome in megabases (Mbp).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section other2" id="sec-4"><h2 class="">Array Normalization</h2><p id="p-15">Illumina BeadStudio uses an “affine normalization” algorithm to perform within- and between-array adjustments to <em>x</em>- and <em>y</em>-hybridization intensities before calling genotypes. However, further normalization is helpful for analyses of sample contamination and copy number. Two standard metrics are the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-3.gif"/></noscript></span></span> ratio (LRR), which captures total hybridization intensity (<em>R</em>) relative to a reference level (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-4.gif"/></noscript></span></span>); and B-allele frequency (BAF), which captures the relative signal from the A and B alleles (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Peiffer 2006</a>). For an uncontaminated euploid sample, the expected value of LRR is 0, and the expected value of BAF is 0.5 at heterozygous markers.</p><p id="p-16">The argyle package implements the thresholded quantile normalization (tQN) approach described in <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Staaf <em>et al.</em> (2008)</a> and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Didion <em>et al.</em> (2014)</a>. Briefly, tQN performs within-array quantile normalization of the <em>x</em> channel against the <em>y</em> channel to account for dye biases specific to the Infinium chemistry, but places an upper bound on the difference between normalized and unnormalized intensity values. LRR and BAF are then computed using known cluster positions computed from a set of reference samples. The tQN procedure may optionally be preceded by preliminary between-array quantile normalization using routines implemented in the preprocessCore package (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bolstad <em>et al.</em> 2003</a>). A joint plot of BAF and LRR (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) is valuable for assessing heterozygosity, ploidy, sample purity, and sex-chromosome karyotype.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Joint plot of B-allele frequency (BAF, upper panel) and log2 intensity ratio (LRR, lower panel) for an outbred male mouse. The autosomes are almost entirely heterozygous, while the X chromosome is hemizygous: no points appear near BAF =&#xA0;0.5 on the X chromosome and its LRR is decreased relative to the autosomes. Red traces are a local smoothing of underlying points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Joint plot of B-allele frequency (BAF, upper panel) and log2 intensity ratio (LRR, lower panel) for an outbred male mouse. The autosomes are almost entirely heterozygous, while the X chromosome is hemizygous: no points appear near BAF =&#xA0;0.5 on the X chromosome and its LRR is decreased relative to the autosomes. Red traces are a local smoothing of underlying points.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F4.medium.gif" width="440" height="189"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/2/281/F4.medium.gif" width="440" height="189"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467657" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-17" class="first-child">Joint plot of B-allele frequency (BAF, upper panel) and log2 intensity ratio (LRR, lower panel) for an outbred male mouse. The autosomes are almost entirely heterozygous, while the X chromosome is hemizygous: no points appear near BAF = 0.5 on the X chromosome and its LRR is decreased relative to the autosomes. Red traces are a local smoothing of underlying points.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Copy-number inference from Illumina arrays is a well-studied problem for which good solutions already exist—for instance, the standalone software PennCNV (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Wang <em>et al.</em> 2007</a>), or the R package genoCN (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Sun <em>et al.</em> 2009</a>). Most of these packages take BAF and LRR values as input and so are easily integrated downstream of argyle.</p><p id="p-19">Systematic batch effects on intensity distributions are possible when analyzing samples processed that were not processed concurrently. The reliability of discrete genotype calls may be unchanged between batches, but downstream analyses that make use of hybridization intensities [<em>e.g.</em>, copy-number analyses, or hidden Markov models (HMM) for haplotype inference in multiparental populations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gatti <em>et al.</em> 2014</a>)] may benefit from a further batch correction. One possibility, given <em>k</em> nonoverlapping batches, is quantile normalization of batches <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-5.gif"/></noscript></span></span> against the <em>k</em>th batch. Although between-batch normalization is not yet implemented in argyle, it is slated for inclusion in future releases.</p></div><div class="section other3" id="sec-5"><h2 class="">Genetics Tools</h2><p id="p-20">Utilities are provided for efficient calculation of allele frequencies, heterozygosity and missingness by sample and by marker. When genotypes of both parents and offspring are available, pedigree relationships can be confirmed via checks for Mendelian inconsistencies. Separate datasets can be concatenated or merged using functions that ensure consistency of allele encoding and detect strand swaps [<em>e.g.</em>, an (A/G) <em>vs.</em> a (T/C) SNP].</p><p id="p-21">To facilitate analysis of genotypes from experimental crosses, argyle provides functions for recoding alleles with respect to parental lines. A general-purpose HMM allows for reconstruction of haplotype mosaics, given a panel of reference samples and a genetic map—although users are cautioned that more sophisticated implementations are available for some special cases (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman <em>et al.</em> 2003</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gatti <em>et al.</em> 2014</a>). Mature tools for genetic mapping in the R environment already exist (<em>e.g.</em>, R/qtl; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Broman <em>et al.</em> 2003</a>). Genotypes processed with argyle can be readily converted to R/qtl format to create a unified pipeline for quantitative-trait locus (QTL) mapping. One- and two-locus “mosaic plots” allow joint visualization of allele frequencies and phenotype at candidate QTL (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). A worked example is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1/FileS4.pdf">File S4</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) One-way mosaic plot. Width of each bar is proportional to the frequency of the corresponding genotype; fill colors indicate phenotype, here case or control status. (B) Two-way mosaic plot. Area of each block is proportional to two-locus genotype frequency, and fill colors indicate phenotype mean for each two-locus genotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) One-way mosaic plot. Width of each bar is proportional to the frequency of the corresponding genotype; fill colors indicate phenotype, here case or control status. (B) Two-way mosaic plot. Area of each block is proportional to two-locus genotype frequency, and fill colors indicate phenotype mean for each two-locus genotype.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F5.medium.gif" width="295" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/2/281/F5.medium.gif" width="295" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467548" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-22" class="first-child">(A) One-way mosaic plot. Width of each bar is proportional to the frequency of the corresponding genotype; fill colors indicate phenotype, here case or control status. (B) Two-way mosaic plot. Area of each block is proportional to two-locus genotype frequency, and fill colors indicate phenotype mean for each two-locus genotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Genome-wide patterns of relatedness can be explored using built-in functions for efficient kinship estimation (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) and principal components analysis (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1/FileS5.pdf">File S5</a> for more detailed demonstration of functions useful for population-genetic analysis.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap representation of matrix of pairwise genetic distances between 28 wild-caught mice from three different subspecies using data from the Mouse Diversity Array (Yang et al. 2011). Genetic distance is defined here as the proportion of alleles shared identical by state between two individuals. The matrix is hierarchically clustered to that more closely-related samples are adjacent to each other. The heatmap is useful for visualizing population structure; here it reveals obvious genetic differentiation between mouse subspecies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heatmap representation of matrix of pairwise genetic distances between 28 wild-caught mice from three different subspecies using data from the Mouse Diversity Array (Yang &lt;em&gt;et al.&lt;/em&gt; 2011). Genetic distance is defined here as the proportion of alleles shared identical by state between two individuals. The matrix is hierarchically clustered to that more closely-related samples are adjacent to each other. The heatmap is useful for visualizing population structure; here it reveals obvious genetic differentiation between mouse subspecies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F6.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/2/281/F6.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467419" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-24" class="first-child">Heatmap representation of matrix of pairwise genetic distances between 28 wild-caught mice from three different subspecies using data from the Mouse Diversity Array (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Yang <em>et al.</em> 2011</a>). Genetic distance is defined here as the proportion of alleles shared identical by state between two individuals. The matrix is hierarchically clustered to that more closely-related samples are adjacent to each other. The heatmap is useful for visualizing population structure; here it reveals obvious genetic differentiation between mouse subspecies.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projection of the same 28 samples from Figure 6 onto the top two principal components (PCs) of the genotypes matrix. The block structure of the kinship matrix corresponds to the three clusters revealed by principal component analysis (PCA), which in turn correspond to three distinct subspecies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Projection of the same 28 samples from Figure 6 onto the top two principal components (PCs) of the genotypes matrix. The block structure of the kinship matrix corresponds to the three clusters revealed by principal component analysis (PCA), which in turn correspond to three distinct subspecies.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F7.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/2/281/F7.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-25" class="first-child">Projection of the same 28 samples from <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> onto the top two principal components (PCs) of the genotypes matrix. The block structure of the kinship matrix corresponds to the three clusters revealed by principal component analysis (PCA), which in turn correspond to three distinct subspecies.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section other4" id="sec-6"><h2 class="">Data Export</h2><p id="p-26">The argyle package provides functions to convert genotypes objects to other formats either within the R session (for R/qtl and DOQTL) or on disk. Currently argyle supports export to either PLINK binary format (*.fam/*.bim/*.bed) or Stanford HGDP format. PLINK provides command-line utilities to convert its file format to many others, including VCF, LINKAGE (*.map/*.ped), Haploview, STRUCTURE, and fastPHASE. In addition, since genotypes objects are regular R matrices, users can adapt them to bespoke input formats required by other tools for genetic analysis.</p></div><div class="section other5" id="sec-7"><h2 class="">Performance</h2><p id="p-27">argyle and its dependencies are compatible with R (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-6.gif"/></noscript></span></span>) on Windows or Mac OS X. The performance of argyle benefits from optimized code in several existing R packages including data.table and Rcpp (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Eddelbuettel 2013</a>). Reading a dataset of realistic size – 96 samples <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-7.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-8.gif"/></noscript></span></span> markers (164 Mb ZIP-compressed on disk)—from Illumina BeadStudio output into an R session takes about 30 sec. The full dataset, including hybridization intensities and sample and marker metadata, occupies 202.4 Mb; without hybridization intensities, the size drops to 77.9 Mb. Memory usage scales approximately linearly with either the number of samples or the number of markers (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). The most computationally-intensive component of argyle is the tQN procedure, and is implemented in C++. Its running time is compared to the quantile normalization routine from the preprocessCore package in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>. These resource requirements are well within the range of a typical laptop or desktop computer.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/2/281/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Memory requirements for genotypes objects (estimated via R&#x2019;s object.size() with varying numbers of markers and samples, with and without hybridization intensities. (B) Running time per sample of the thresholded quantile normalization (tQN) procedure, with either initial quantile normalization or postpolishing, compared to quantile normalization alone." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1250799433" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Memory requirements for genotypes objects (estimated via R&#x2019;s object.size() with varying numbers of markers and samples, with and without hybridization intensities. (B) Running time per sample of the thresholded quantile normalization (tQN) procedure, with either initial quantile normalization or postpolishing, compared to quantile normalization alone.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/2/281/F8.medium.gif" width="283" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/2/281/F8.medium.gif" width="283" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/2/281/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/2/281/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467528" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-28" class="first-child">(A) Memory requirements for genotypes objects (estimated via R’s object.size() with varying numbers of markers and samples, with and without hybridization intensities. (B) Running time per sample of the thresholded quantile normalization (tQN) procedure, with either initial quantile normalization or postpolishing, compared to quantile normalization alone.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">R’s internal limit of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-9.gif"/></noscript></span></span> entries for any matrix or vector places an upper bound on the dimensions of a genotypes object. For arrays with between <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-11.gif"/></noscript></span></span> markers, this translates to a limit of between <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/2/281/embed/mml-math-13.gif"/></noscript></span></span> samples.</p><p id="p-30">Tests were performed in R 3.1.2 (64-bit) on a MacBook Air, with a single 1.7 Ghz Intel Core i7 processor, and 8 Gb RAM.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">The author thanks John Didion for advice on design and quality-control methods; John Didion, Dan Gatti, Marty Ferris, and Sofia Grize for valuable feedback on early versions of this software; and Robert Corty, Will Valdar, and Fernando Pardo-Manuel de Villena for helpful comments on this manuscript. This work was supported in part by NIH grants F30MH103925 (APM), U19AI100625 (FPMV), U42OD010924 (Terry Magnuson), and the UNC Bioinformatics and Computational Biology Training Grant (T32GM067553).</p><p id="p-32"><em>Note added in proof:</em> See <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Morgan <em>et al</em>. 2015</a> (pp. 263–279) in this issue, for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-33">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023739/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-34"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-16"><span class="received-label">Received </span>October 16, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-12-17"><span class="accepted-label">Accepted </span>December 17, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Morgan</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.1" data-doi="10.1093/bioinformatics/19.2.185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolstad</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astrand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBolstad%26rft.auinit1%253DB.M.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D185%26rft.epage%253D193%26rft.atitle%253DA%2Bcomparison%2Bof%2Bnormalization%2Bmethods%2Bfor%2Bhigh%2Bdensity%2Boligonucleotide%2Barray%2Bdata%2Bbased%2Bon%2Bvariance%2Band%2Bbias%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F19.2.185%26rft_id%253Dinfo%253Apmid%252F12538238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS8yLzE4NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.2" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.3" data-doi="10.1186/1471-2164-15-847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naghashfar</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SNP array profiling of mouse cell lines identifies their strains of origin and reveals cross-contamination and widespread aneuploidy.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D847%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-847%26rft_id%253Dinfo%253Apmid%252F25277546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25277546&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.4" data-doi="10.1093/bioinformatics/btn224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kibbe</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">lumi: a pipeline for processing illumina microarray.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1547</span>–<span class="cit-lpage">1548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn224%26rft_id%253Dinfo%253Apmid%252F18467348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTMvMTU0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.5" data-doi="10.1093/bioinformatics/btm311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunning</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">beadarray: R classes and methods for illumina bead-based data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2183</span>–<span class="cit-lpage">2184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm311%26rft_id%253Dinfo%253Apmid%252F17586828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTYvMjE4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-6.2.281.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Seamless R and C++ Integration with Rcpp</span>, <span class="cit-publ-name">Springer, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.2.281.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Inferring ancestry in admixed populations using microarray probe intensities, pp. 105–112 in <em>Proceedings of the ACM Conference on Bioinformatics</em>, <em>Computational Biology and Biomedicine - BCB ′12</em>, <span class="cit-publ-name">Association for Computing Machinery (ACM</span>). <span class="cit-comment">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.8" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DGatti%26rft.auinit1%253DD.%2BM.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1623%26rft.epage%253D1633%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2BMapping%2BMethods%2Bfor%2BDiversity%2BOutbred%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft_id%253Dinfo%253Apmid%252F25237114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.9" data-doi="10.1534/g3.115.018416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buyyarapu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development of a 63k SNP array for cotton and high-density mapping of intraspecific and interspecific populations of gossypium spp.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1187</span>–<span class="cit-lpage">1209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.018416%26rft_id%253Dinfo%253Apmid%252F25908569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.10" data-doi="10.1186/1471-2164-14-439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindqvist</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niemelä</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erkinaro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fish scales and SNP chips: SNP genotyping and allele frequency estimation in individual and pooled DNA from historical samples of atlantic salmon (salmo salar).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-439%26rft_id%253Dinfo%253Apmid%252F23819691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23819691&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.11" data-doi="10.1371/journal.pone.0004668"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalrymple</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heaton</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maddox</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome wide survey of SNP variation reveals the genetic structure of sheep breeds.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e4668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKijas%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De4668%26rft.epage%253De4668%26rft.atitle%253DA%2Bgenome%2Bwide%2Bsurvey%2Bof%2BSNP%2Bvariation%2Breveals%2Bthe%2Bgenetic%2Bstructure%2Bof%2Bsheep%2Bbreeds.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004668%26rft_id%253Dinfo%253Apmid%252F19270757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004668&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19270757&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A.P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J.P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The mouse universal genotyping array: from substrains to subspecies.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.13" data-doi="10.1101/gr.5402306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-resolution genomic profiling of chromosomal aberrations using infinium whole-genome genotyping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1136</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5402306%26rft_id%253Dinfo%253Apmid%252F16899659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE2LzkvMTEzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.14" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.15" data-doi="10.1093/bioinformatics/btp470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetrick</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">R/bioconductor software for Illumina’s Infinium whole-genome genotyping BeadChips.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2621</span>–<span class="cit-lpage">2623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp470%26rft_id%253Dinfo%253Apmid%252F19661241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTkvMjYyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMi8yODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.16" data-doi="10.1186/1471-2105-9-409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staaf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallon-Christersson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juliusson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenquist</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Normalization of Illumina Infinium whole-genome SNP data improves copy number estimates and allelic intensity ratios.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStaaf%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.spage%253D409%26rft.epage%253D409%26rft.atitle%253DNormalization%2Bof%2BIllumina%2BInfinium%2Bwhole-genome%2BSNP%2Bdata%2Bimproves%2Bcopy%2Bnumber%2Bestimates%2Band%2Ballelic%2Bintensity%2Bratios.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-409%26rft_id%253Dinfo%253Apmid%252F18831757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18831757&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.17" data-doi="10.1038/nmeth842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steemers</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Whole-genome genotyping with the single-base extension assay.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSteemers%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D33%26rft.atitle%253DWhole-genome%2Bgenotyping%2Bwith%2Bthe%2Bsingle-base%2Bextension%2Bassay.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth842%26rft_id%253Dinfo%253Apmid%252F16369550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16369550&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234528000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.18" data-doi="10.1093/nar/gkp493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordgard</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Integrated study of copy number states and genotype calls using high-density SNP arrays.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">5365</span>–<span class="cit-lpage">5377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp493%26rft_id%253Dinfo%253Apmid%252F19581427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8xNi81MzY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.19" data-doi="10.1101/gr.6861907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glessner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PennCNV: an integrated hidden markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6861907%26rft_id%253Dinfo%253Apmid%252F17921354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMS8xNjY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8yLzI4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.20" data-doi="10.1111/age.12169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willet</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An updated felCat5 SNP manifest for the illumina feline 63k SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D45%26rft.spage%253D614%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12169%26rft_id%253Dinfo%253Apmid%252F24804964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24804964&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.2.281.21" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F2%2F281.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/2/281.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/2/263" title="The Mouse Universal Genotyping Array: From Substrains to Subspecies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/2/287" title="SAGA DUB-Ubp8 Deubiquitylates Centromeric Histone Variant Cse4" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/2"><span>Volume 6 Issue 2, February 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/2.cover-source.jpg?itok=7FK5uFFF" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/2/281" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZFK0KyozI7-RzBpnISNDmmhYYfNH6xD6xdHl_G-ux1g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/2/281" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/2/281" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/2/281" class="active" data-icon-position="" data-hide-link-title="0">argyle: An R Package for Analysis of Illumina Genotyping Arrays</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460056" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hW20LAUTjkvA9_sLjURGVm907nRPDIs1oRIG7hhrLC0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/281.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/2/281" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2DLeOa4KfocEqjgtSZ76YRbHBfOa4ZdaXbO43bH1ByE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/2/281" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/2/281.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460056" id="node460056--2" data-pisa="ggg;6/2/281" data-pisa-master="ggg;g3.115.023739" data-apath="/ggg/6/2/281.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">argyle: An R Package for Analysis of Illumina Genotyping Arrays</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew P. Morgan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">281-286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023739 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460056--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew P. Morgan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:apm@email.unc.edu" class="" data-icon-position="" data-hide-link-title="0">apm@email.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/2/281" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/2/281&amp;title=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/2/281&amp;title=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/2/281&amp;title=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/2/281&amp;text=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/2/281&amp;title=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/2/281&amp;t=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/2/281&amp;title=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/2/281&amp;title=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460056/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460056" id="node460056--3" data-pisa="ggg;6/2/281" data-pisa-master="ggg;g3.115.023739" data-apath="/ggg/6/2/281.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">argyle: An R Package for Analysis of Illumina Genotyping Arrays</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew P. Morgan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">281-286;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023739 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460056--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew P. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7264</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BP.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1942-4543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew P. Morgan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:apm@email.unc.edu" class="" data-icon-position="" data-hide-link-title="0">apm@email.unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=281&amp;rft.epage=286&amp;rft.atitle=argyle%3A%20An%20R%20Package%20for%20Analysis%20of%20Illumina%20Genotyping%20Arrays&amp;rft.volume=6&amp;rft.issue=2&amp;rft.date=2016-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Morgan%2C+Andrew+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460056/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460056/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460056/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460056/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460056/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460056/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460056/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460056/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460056/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460056/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460056/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460056/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/2/281" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Quality Control</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Array Normalization</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Genetics Tools</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data Export</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Performance</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/2/281.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/2/281.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/2/281.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/2/281.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
