<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2735" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2735.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2735.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2735.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/8/2735/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2735.atom" />
<meta name="HW.pisa" content="ggg;8/8/2735" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program" />
<meta name="DC.Identifier" content="10.1534/g3.118.200415" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Belamkar et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic prediction (GP) is now routinely performed in crop plants to predict unobserved phenotypes. The use of predicted phenotypes to make selections is an active area of research. Here, we evaluate GP for predicting grain yield and compare genomic and phenotypic selection by tracking lines advanced. We examined four independent nurseries of F3:6 and F3:7 lines trialed at 6 to 10 locations each year. Yield was analyzed using mixed models that accounted for experimental design and spatial variations. Genotype-by-sequencing provided nearly 27,000 high-quality SNPs. Average genomic predictive ability, estimated for each year by randomly masking lines as missing in steps of 10% from 10 to 90%, and using the remaining lines from the same year as well as lines from other years in a training set, ranged from 0.23 to 0.55. The predictive ability estimated for a new year using the other years ranged from 0.17 to 0.28. Further, we tracked lines advanced based on phenotype from each of the four F3:6 nurseries. Lines with both above average genomic estimated breeding value (GEBV) and phenotypic value (BLUP) were retained for more years compared to lines with either above average GEBV or BLUP alone. The number of lines selected for advancement was substantially greater when predictions were made with 50% of the lines from the testing year added to the training set. Hence, evaluation of only 50% of the lines yearly seems possible. This study provides insights to assess and integrate genomic selection in breeding programs of autogamous crops." />
<meta name="DC.Contributor" content="Vikas Belamkar" />
<meta name="DC.Contributor" content="Mary J. Guttieri" />
<meta name="DC.Contributor" content="Waseem Hussain" />
<meta name="DC.Contributor" content="Diego Jarquín" />
<meta name="DC.Contributor" content="Ibrahim El-basyoni" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="DC.Contributor" content="P. Stephen Baenziger" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic prediction (GP) is now routinely performed in crop plants to predict unobserved phenotypes. The use of predicted phenotypes to make selections is an active area of research. Here, we evaluate GP for predicting grain yield and compare genomic and phenotypic selection by tracking lines advanced. We examined four independent nurseries of F&lt;sub&gt;3:6&lt;/sub&gt; and F&lt;sub&gt;3:7&lt;/sub&gt; lines trialed at 6 to 10 locations each year. Yield was analyzed using mixed models that accounted for experimental design and spatial variations. Genotype-by-sequencing provided nearly 27,000 high-quality SNPs. Average genomic predictive ability, estimated for each year by randomly masking lines as missing in steps of 10% from 10 to 90%, and using the remaining lines from the same year as well as lines from other years in a training set, ranged from 0.23 to 0.55. The predictive ability estimated for a new year using the other years ranged from 0.17 to 0.28. Further, we tracked lines advanced based on phenotype from each of the four F&lt;sub&gt;3:6&lt;/sub&gt; nurseries. Lines with both above average genomic estimated breeding value (GEBV) and phenotypic value (BLUP) were retained for more years compared to lines with either above average GEBV or BLUP alone. The number of lines selected for advancement was substantially greater when predictions were made with 50% of the lines from the testing year added to the training set. Hence, evaluation of only 50% of the lines yearly seems possible. This study provides insights to assess and integrate genomic selection in breeding programs of autogamous crops.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2735" />
<meta name="citation_id" content="8/8/2735" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2735" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2735.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2735.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2735.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200415" />
<meta name="citation_pmid" content="29945967" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2735" />
<meta name="citation_lastpage" content="2747" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vikas Belamkar" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0988-9710" />
<meta name="citation_author" content="Mary J. Guttieri" />
<meta name="citation_author_institution" content="USDA, Agricultural Research Service, Center for Grain and Animal Health Research, Hard Winter Wheat Genetics Research Unit, Manhattan, KS 66502" />
<meta name="citation_author" content="Waseem Hussain" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author" content="Diego Jarquín" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author" content="Ibrahim El-basyoni" />
<meta name="citation_author_institution" content="Crop Science Department, Faculty of Agriculture, Damanhour University, Egypt" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author" content="P. Stephen Baenziger" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author_email" content="pbaenziger1@unl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9109-6954" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J. I. Sanchez;citation_author=J. L. Jannink;citation_title=Optimization of genomic selection training populations with a genetic algorithm.;citation_pages=38;citation_volume=47;citation_year=2015;citation_pmid=25943105;citation_doi=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. M. Allen;citation_author=G. L. A. Barker;citation_author=P. Wilkinson;citation_author=A. Burridge;citation_author=M. Winfield;citation_title=Discovery and development of exome‐based, co‐dominant single nucleotide polymorphism markers in hexaploid wheat (Triticum aestivum L.).;citation_pages=279-295;citation_volume=11;citation_year=2013;citation_pmid=23279710;citation_doi=10.1111/pbi.12009" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. P. Arruda;citation_author=A. E. Lipka;citation_author=P. J. Brown;citation_author=A. M. Krill;citation_author=C. Thurber;citation_title=Comparing genomic selection and marker-assisted selection for Fusarium head blight resistance in wheat (Triticum aestivum L.).;citation_pages=84;citation_volume=36;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=N. A. Tinker;citation_title=Genomic, Marker-Assisted, and Pedigree-BLUP Selection Methods for β-Glucan Concentration in Elite Oat.;citation_pages=1894-1906;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.09.0526" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. J. Auinger;citation_author=M. Schonleben;citation_author=C. Lehermeier;citation_author=M. Schmidt;citation_author=V. Korzun;citation_title=Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (Secale cereale L.).;citation_pages=2043-2053;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2756-5" />
<meta name="citation_reference" content="Baenziger, P. S., D. Rose, D. Santra, M. Guttieri, and L. Xu, 2014 Improving wheat varieties for Nebraska: 2014 state breeding and quality evaluation report - retrieved on March 29, 2017 from https://agronomy.unl.edu/documents/Whtann14Final-3-17-15.pdf. University of Nebraska-Lincoln, Lincoln, Nebraska" />
<meta name="citation_reference" content="Baenziger, P. S., D. Rose, D. Santra, M. Guttieri, and L. Xu, 2015 Improving wheat varieties for Nebraska: 2015 state breeding and quality evaluation report - retrieved on March 29, 2017 from http://agronomy.unl.edu/Baenziger/Whtann15V9FIN.pdf. University of Nebraska-Lincoln, Lincoln, Nebraska" />
<meta name="citation_reference" content="Baenziger, P. S., D. Rose, D. Santra, and L. Xu, 2012 Improving wheat varieties for Nebraska: 2012 state breeding and quality evaluation report - retrieved on March 29, 2017 from http://agronomy.unl.edu/documents/Whtann12V8R2.pdf. University of Nebraska-Lincoln, Lincoln, Nebraska" />
<meta name="citation_reference" content="Baenziger, P. S., D. Rose, D. Santra, and L. Xu, 2013 Improving wheat varieties for Nebraska: 2013 state breeding and quality evaluation report - retrieved on March 29, 2017 from http://agronomy.unl.edu/documents/WheatAnnualReport2013.pdf. University of Nebraska-Lincoln, Lincoln, Nebraska" />
<meta name="citation_reference" content="citation_journal_title=Sustainability;citation_author=P. S. Baenziger;citation_author=I. Salah;citation_author=R. S. Little;citation_author=D. K. Santra;citation_author=T. Regassa;citation_title=Structuring an Efficient Organic Wheat Breeding Program.;citation_pages=1190-1205;citation_volume=3;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=P. S. Baenziger;citation_author=D. R. Shelton;citation_author=M. J. Shipman;citation_author=R. A. Graybosch;citation_title=Breeding for end-use quality: Reflections on the Nebraska experience.;citation_pages=95-100;citation_volume=119;citation_year=2001;citation_doi=10.1023/A:1017583514424" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=F. M. Bassi;citation_author=A. R. Bentley;citation_author=G. Charmet;citation_author=R. Ortiz;citation_author=J. Crossa;citation_title=Breeding schemes for the implementation of genomic selection in wheat (Triticum spp.).;citation_pages=23-36;citation_volume=242;citation_year=2016;citation_doi=10.1016/j.plantsci.2015.08.021" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=S. D. Battenfield;citation_author=C. Guzman;citation_author=R. C. Gaynor;citation_author=R. P. Singh;citation_author=R. J. Pena;citation_title=Genomic Selection for Processing and End-Use Quality Traits in the CIMMYT Spring Bread Wheat Breeding Program.;citation_pages=2;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.01.0005" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=V. Belamkar;citation_author=A. D. Farmer;citation_author=N. T. Weeks;citation_author=S. R. Kalberer;citation_author=W. J. Blackmon;citation_title=Genomics-assisted characterization of a breeding collection of Apios americana, an edible tuberous legume.;citation_pages=34908;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep34908" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A.-M. Bernal-Vasquez;citation_author=J. Möhring;citation_author=M. Schmidt;citation_author=M. Schönleben;citation_author=C.-C. Schön;citation_title=The importance of phenotypic data analysis for genomic prediction - a case study comparing different spatial models in rye.;citation_pages=646;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-646" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Bernardo;citation_title=Bandwagons I, too, have known.;citation_pages=2323-2332;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2772-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic Gains in Grain Yield Through Genomic Selection in Eight Bi-parental Maize Populations under Drought Stress.;citation_pages=154-163;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=F. Cericola;citation_author=A. Jahoor;citation_author=J. Orabi;citation_author=J. R. Andersen;citation_author=L. L. Janss;citation_title=Optimizing Training Population Size and Genotyping Strategy for Genomic Prediction Using Association Study Results and Pedigree Information. A Case of Study in Advanced Wheat Breeding Lines.;citation_pages=e0169606;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0169606" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. A. Chapman;citation_author=M. Mascher;citation_author=A. Buluç;citation_author=K. Barry;citation_author=E. Georganas;citation_title=A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.;citation_pages=26;citation_volume=16;citation_year=2015;citation_pmid=25637298;citation_doi=10.1186/s13059-015-0582-8" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. Combs;citation_author=R. Bernardo;citation_title=Genomewide Selection to Introgress Semidwarf Maize Germplasm into U.S. Corn Belt Inbreds.;citation_pages=1427-1436;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.11.0666" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=B. Cullis;citation_author=B. Gogel;citation_author=A. Verbyla;citation_author=R. Thompson;citation_title=Spatial Analysis of Multi-Environment Early Generation Variety Trials.;citation_pages=1-18;citation_volume=54;citation_year=1998;citation_doi=10.2307/2533991" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach.;citation_pages=e3395-e3395;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852893;citation_doi=10.1371/journal.pone.0003395" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. C. Dawson;citation_author=J. B. Endelman;citation_author=N. Heslot;citation_author=J. Crossa;citation_author=J. Poland;citation_title=The use of unbalanced historical data for genomic selection in an international wheat breeding program.;citation_pages=12-22;citation_volume=154;citation_year=2013;citation_doi=10.1016/j.fcr.2013.07.020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. J. Faville;citation_author=S. Ganesh;citation_author=M. Cao;citation_author=M. Z. Z. Jahufer;citation_author=T. P. Bilton;citation_title=Predictive ability of genomic selection models in a multi-population perennial ryegrass training set using genotyping-by-sequencing.;citation_pages=703-720;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-3030-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=A. R. Gilmour;citation_author=B. R. Cullis;citation_author=A. P. Verbyla;citation_title=Accounting for Natural and Extraneous Variation in the Analysis of Field Experiments.;citation_pages=269-293;citation_volume=2;citation_year=1997;citation_doi=10.2307/1400446" />
<meta name="citation_reference" content="Gilmour, A. R., B. J. Gogel, B. R. Cullis, and R. Thompson, 2009 ASReml User Guide Release 3.0 VSN International Ltd, Hemel Hempstead, HP1 1ES, UK www.vsni.co.uk" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: A High Capacity Genotyping by Sequencing Analysis Pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. He;citation_author=J. C. Reif;citation_author=V. Korzun;citation_author=R. Bothe;citation_author=E. Ebmeyer;citation_title=Genome-wide mapping and prediction suggests presence of local epistasis in a vast elite winter wheat populations adapted to Central Europe.;citation_pages=635-647;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2840-x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. He;citation_author=A. W. Schulthess;citation_author=V. Mirdita;citation_author=Y. Zhao;citation_author=V. Korzun;citation_title=Genomic selection in a commercial winter wheat population.;citation_pages=641-651;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2655-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection in Plant Breeding: A Comparison of Models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_author=R. Howard;citation_author=AL. Carriquiry;citation_author=WD. Beavis;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome;citation_pages=1251788-1251788;citation_volume=345;citation_year=2014;citation_issue=6194;citation_pmid=25035500;citation_doi=10.1126/science.1251788" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Jarquín;citation_author=K. Kocak;citation_author=L. Posadas;citation_author=K. Hyma;citation_author=J. Jedlicka;citation_title=Genotyping by sequencing for genomic prediction in a soybean breeding population.;citation_pages=740;citation_volume=15;citation_year=2014;citation_pmid=25174348;citation_doi=10.1186/1471-2164-15-740" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. C. Gaynor;citation_author=J. Poland;citation_author=A. Fritz;citation_title=Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. Kooke;citation_author=W. Kruijer;citation_author=R. Bours;citation_author=F. Becker;citation_author=A. Kuhn;citation_title=Genome-Wide Association Mapping and Genomic Prediction Elucidate the Genetic Architecture of Morphological Traits in Arabidopsis.;citation_volume=170;citation_year=2016;citation_pmid=26869705;citation_doi=10.1104/pp.15.00997" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=B. Lado;citation_author=I. Matus;citation_author=A. Rodriguez;citation_author=L. Inostroza;citation_author=J. Poland;citation_title=Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data.;citation_volume=3;citation_year=2013;citation_pmid=24082033;citation_doi=10.1534/g3.113.007807" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Massman;citation_author=H.-J. G. Jung;citation_author=R. Bernardo;citation_title=Genomewide Selection vs. Marker-assisted Recurrent Selection to Improve Grain Yield and Stover-quality Traits for Cellulosic Ethanol in Maize.;citation_pages=58-66;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.02.0112" />
<meta name="citation_reference" content="citation_journal_title=TAG Theoretical and Applied Genetics Theoretische Und Angewandte Genetik;citation_author=S. Michel;citation_author=C. Ametz;citation_author=H. Gungor;citation_author=B. Akgöl;citation_author=D. Epure;citation_title=Genomic assisted selection for enhancing line breeding: merging genomic and phenotypic selection in winter wheat breeding programs with preliminary yield trials.;citation_pages=363-376;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2818-8" />
<meta name="citation_reference" content="citation_author=JL. Neyhart;citation_author=Tiede. T;citation_author=Lorenz. AJ;citation_author=Smith. KP;citation_title= Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection. G3: Genes|Genomes|Genetics;citation_volume=7;citation_year=2017;citation_pmid=28315831;citation_doi=10.1534/g3.117.040550" />
<meta name="citation_reference" content="NOAA, 2016 National Centers for Environmental Information, State of the Climate: Global Analysis for Annual 2015, published online January 2016, retrieved on March 29, 2017 from https://www.ncdc.noaa.gov/sotc/global/201513." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Perez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez-Rodríguez;citation_author=Crossa. J;citation_author=Rutkoski. J;citation_author=Poland. J;citation_author=Singh. R;citation_title=Single-Step Genomic and Pedigree Genotype × Environment Interaction Models for Predicting Wheat Lines in International Environments.;citation_pages=2;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.09.0089" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=H. P. Piepho;citation_author=J. Möhring;citation_author=T. Schulz‐Streeck;citation_author=O. Ogutu Joseph;citation_title=A stage-wise approach for the analysis of multi-environment trials.;citation_pages=844-860;citation_volume=54;citation_year=2012;citation_issue=6;citation_pmid=23007738;citation_doi=10.1002/bimj.201100219" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=H. P. Piepho;citation_author=E. R. Williams;citation_title=Linear variance models for plant breeding trials.;citation_pages=1-8;citation_volume=129;citation_year=2010;citation_doi=10.1111/j.1439-0523.2009.01654.x" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=J. Poland;citation_title=Breeding-assisted genomics.;citation_pages=119-124;citation_volume=24;citation_year=2015;citation_doi=10.1016/j.pbi.2015.02.009" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic Selection in Wheat Breeding using Genotyping-by-Sequencing.;citation_pages=103;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-Sequencing for Plant Breeding and Genetics.;citation_pages=92;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="R Core Team, 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL https://www.R-project.org/" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=H. S. Randhawa;citation_author=M. Asif;citation_author=C. Pozniak;citation_author=J. M. Clarke;citation_author=R. J. Graf;citation_title=Application of molecular markers to wheat breeding in Canada.;citation_pages=458-471;citation_volume=132;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_title=Genetic Gain from Phenotypic and Genomic Selection for Quantitative Resistance to Stem Rust of Wheat.;citation_pages=2;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=JE. Rutkoski;citation_author=JA. Poland;citation_author=RP. Singh;citation_author=Huerta-Espino. J;citation_author=Bhavani. S;citation_author=Barbier. H;citation_author=Rouse. MN;citation_author=Jannink. J-L;citation_author=Sorrells. ME;citation_title=Genomic Selection for Quantitative Adult Plant Stem Rust Resistance in Wheat.;citation_volume=7;citation_year=2014;citation_doi=10.3835/plantgenome2014.02.0006" />
<meta name="citation_reference" content="citation_journal_title=GSE;citation_author=M. Saatchi;citation_author=M. C. McClure;citation_author=S. D. McKay;citation_author=M. M. Rolf;citation_author=J. Kim;citation_title=Accuracies of genomic breeding values in American Angus beef cattle using K-means clustering for cross-validation. Genetics, Selection, Evolution;citation_pages=40;citation_volume=43;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. H. Sallam;citation_author=J. B. Endelman;citation_author=J. L. Jannink;citation_author=K. P. Smith;citation_title=Assessing Genomic Selection Prediction Accuracy in a Dynamic Barley Breeding Population.;citation_pages=1;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. E. Spindel;citation_author=S. R. McCouch;citation_title=When more is better: how data sharing would accelerate genomic selection of crop plants.;citation_pages=814-826;citation_volume=212;citation_year=2016;citation_doi=10.1111/nph.14174" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=N. Tayeh;citation_author=A. Klein;citation_author=M.-C. Le Paslier;citation_author=F. Jacquin;citation_author=H. Houtin;citation_title=Genomic Prediction in Pea: Effect of Marker Density and Training Population Size and Composition on Prediction Accuracy.;citation_pages=941;citation_volume=6;citation_year=2015;citation_pmid=26635819;citation_doi=10.3389/fpls.2015.00941" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Torkamaneh;citation_author=F. Belzile;citation_title=Scanning and Filling: Ultra-Dense SNP Genotyping Combining Genotyping-By-Sequencing, SNP Array and Whole-Genome Resequencing Data.;citation_pages=e0131533;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0131533" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Velu;citation_author=J. Crossa;citation_author=R. P. Singh;citation_author=Y. Hao;citation_author=S. Dreisigacker;citation_title=Genomic prediction for grain zinc and iron concentrations in spring wheat.;citation_pages=1595-1605;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122&amp;ndash;016&amp;ndash;2726-y" />
<meta name="citation_reference" content="citation_journal_title=Aust. N. Z. J. Stat.;citation_author=J. S. Welham;citation_author=B. J. Gogel;citation_author=A. B. Smith;citation_author=R. Thompson;citation_author=B. R. Cullis;citation_title=A comparison of analysis methods for late‐stage variety evaluation trials.;citation_pages=125-149;citation_volume=52;citation_year=2010;citation_doi=10.1111/j.1467-842X.2010.00570.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Z. Weng;citation_author=A. Wolc;citation_author=X. Shen;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_title=Effects of number of training generations on genomic prediction for various traits in a layer chicken population.;citation_pages=22;citation_volume=48;citation_year=2016;citation_pmid=26992471;citation_doi=10.1186/s12711-016-0198-9" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis.;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Commun. Stat. Simul. Comput.;citation_author=R. Wolfinger;citation_title=Covariance structure selection in general mixed models.;citation_pages=1079-1106;citation_volume=22;citation_year=1993;citation_doi=10.1080/03610919308813143" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=16150;citation_volume=2;citation_year=2016;citation_doi=10.1038/nplants.2016.150" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=X. Zhang;citation_author=P. Perez-Rodriguez;citation_author=K. Semagn;citation_author=Y. Beyene;citation_author=R. Babu;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.;citation_pages=291-299;citation_volume=114;citation_year=2015;citation_pmid=25407079;citation_doi=10.1038/hdy.2014.99" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="Genomic prediction (GP) is now routinely performed in crop plants to predict unobserved phenotypes. The use of predicted phenotypes to make selections is an active area of research. Here, we evaluate GP for predicting grain yield and compare genomic and phenotypic selection by tracking lines advanced. We examined four independent nurseries of F3:6 and F3:7 lines trialed at 6 to 10 locations each year. Yield was analyzed using mixed models that accounted for experimental design and spatial variations. Genotype-by-sequencing provided nearly 27,000 high-quality SNPs. Average genomic predictive ability, estimated for each year by randomly masking lines as missing in steps of 10% from 10 to 90%, and using the remaining lines from the same year as well as lines from other years in a training set, ranged from 0.23 to 0.55. The predictive ability estimated for a new year using the other years ranged from 0.17 to 0.28. Further, we tracked lines advanced based on phenotype from each of the four F3:6 nurseries. Lines with both above average genomic estimated breeding value (GEBV) and phenotypic value (BLUP) were retained for more years compared to lines with either above average GEBV or BLUP alone. The number of lines selected for advancement was substantially greater when predictions were made with 50% of the lines from the testing year added to the training set. Hence, evaluation of only 50% of the lines yearly seems possible. This study provides insights to assess and integrate genomic selection in breeding programs of autogamous crops." />
<meta name="og-title" property="og:title" content="Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2735" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic prediction (GP) is now routinely performed in crop plants to predict unobserved phenotypes. The use of predicted phenotypes to make selections is an active area of research. Here, we evaluate GP for predicting grain yield and compare genomic and phenotypic selection by tracking lines advanced. We examined four independent nurseries of F3:6 and F3:7 lines trialed at 6 to 10 locations each year. Yield was analyzed using mixed models that accounted for experimental design and spatial variations. Genotype-by-sequencing provided nearly 27,000 high-quality SNPs. Average genomic predictive ability, estimated for each year by randomly masking lines as missing in steps of 10% from 10 to 90%, and using the remaining lines from the same year as well as lines from other years in a training set, ranged from 0.23 to 0.55. The predictive ability estimated for a new year using the other years ranged from 0.17 to 0.28. Further, we tracked lines advanced based on phenotype from each of the four F3:6 nurseries. Lines with both above average genomic estimated breeding value (GEBV) and phenotypic value (BLUP) were retained for more years compared to lines with either above average GEBV or BLUP alone. The number of lines selected for advancement was substantially greater when predictions were made with 50% of the lines from the testing year added to the training set. Hence, evaluation of only 50% of the lines yearly seems possible. This study provides insights to assess and integrate genomic selection in breeding programs of autogamous crops." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ulQAK49DQHkHf92groZAahY5NwaDMoB1Di76SDwOtyk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490462","apath":"\/ggg\/8\/8\/2735.atom","pisa":"ggg;8\/8\/2735","ac":{"\/ggg\/8\/8\/2735.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2735.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2735"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2735"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2735.abstract":true,"\/content\/8\/8\/2735":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/BAbGGo\\\\\\\/iw2Cpgs\\\\\\\/qalW3XE0lwaob\\\\\\\/Ntuzsi5oonBNsvZEopA2nGaXe6AfKUhJAoqk0RuANEXlZBc1cOxt+UnuUZSoBNcnXLI1vrYKW+3c4PSW2iUwK9uAlkvp47tovxeENZa+HqBSNDZG6rT6Ohtl09ZrS59\\\\\\\/V69yJl\\\\\\\/BxDCiAFKt9TvammHFo1By3pkKidE4ZQvtcABYklFFvbsGkZbM3jpclXs46BSmsrnDPrHLuTIGT5aNgh5GHuUrupZgHUb3r\\\\\\\/lrVrE6UjKcAuBjJilwwkTUVfvh8pgncTppY04zMJHWgCdmpMoA3dOsBWSeOyXbHVFwyiVNc2Ch6TTcbWDK+6bqTJ4gUYLdveSoJGX\\\\\\\/KNq\\\\\\\/6qHIecdXu1c8QZ0woch9aL3A2WYFRpktstM4iRkitSNhMmMvSvUPLhYP+fRoyM44uuOMCayyH1DEsLlUrKwkjSRwxW+7E7b1sOPwve4KwDate0DGefB49AvRgfev1QKt4LW\\\\\\\/ujX9a6A01RhLin65wbnY4xJM2jCNJfE1jiFQqLCXNzv5hQ51bsDDsxBQQNP9\\\\\\\/Wmbf4qk\\\\\\\/SprHtUir8upFLc6LnYdYi4WKfLVKXSyDSihQ4Hg4RnG3lZJT7ZcxRXEL3xqLTpooJEbsQ9sCoZg2nUdwbAbA\\\\\\\/oJ4sabuRNqIpGOY4QUATZq8qsQp3lCDnw9vsmwJBXNc8g5lJ5KX0j31kKg4vjg7BYOsmjRiwFX8Mobgol5NyVYNVnpYZjZ8kyAcgQHzKt2ojqn7jcf0ShFX1k6BSZ7f+RUVsojW6G41weuN5dgpdWI\\\\\\\/jl\\\\\\\/Ng4WNstWXxlazJ3eBDrl3ZyNAC+ltbUf3EIMyHnfE2oQO5FWA3dinNlVdGijGzqXChVIuW+skHSXiH6ZTl+wN8M90CCqgLzLSSYibUeUUMHpn8v038Cy14IfF9e1T6YyuIz6yFIqLUIOUXWWu10\\\\\\\/U\\\\\\\/hq1fpuxIN22wfhzT\\\\\\\/wWIhMfH2KvioGOA1tAbxet\\\\\\\/iWvO69IZoVMtrU+37oc4pTEVHOPgQ1\\\\\\\/05O30xyc2VZ\\\\\\\/zIZqUzXIPPLkQ4k8DQNdqkWPodGUy\\\\\\\/hILgz9NRSbS8e\\\\\\\/KJoLZvTdckL1kkHbLgoCHCMC26vn1Z1VrsZfaMhiGv02yOxSnMGEf8dNVdLlTGm0UJc\\\\\\\/b3BQiw1rYJDDmkWxweRnXZqxQJPiinpo3Qn77lHQiyhGO52InOwZIjtPLeq\\\\\\\/KlaciqM8+NVnymg4vQ11jPAYc6e8Ys3TabhI3GYtx\\\\\\\/\\\\\\\/F9N0h3qgDTDQFk+mvxyJT777fe\\\u0022,\\\u0022iv\\\u0022:\\\u0022AR1H8n272endonHznGqcjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002298fd16e45ac922a9af0c9878b0b96a390c4aa019a764a172cb3075fd23d5390f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vywODEnWN0juE0DHAK6w+JzU2u8gz7l1G9otCTyODydlkpuzoPReif2Muouc8sYaeQZ9r3cEaVpCW5FKaETvI9hYRXE7LG+JVyFBvUKVtbC6FKjtsNsChK1a850VefD44ygAxXiudTtrN18dYt1rF\\\\\\\/mMFO5TYfpPHkr1P9iiiz7mp11RofeNxRW4VVaWX9WdcGUfhYLGlwf4CCj8ODNHtP2EEeCBNcYHBvDoQPMogniFr1NXhb0qKx\\\\\\\/9JXLhNRs2Cywbr7I3yfmuWxE2z85XW0\\\\\\\/xUi069NOKKCgn4pGhtJWFhSMqbPDpGQO8+6CgGWgY4ikQev0r16NbfefJgSYexu1JR+Ex9Wo+7msv0zLkF61XZjIfYmt3rA23AyuKtL3UBbYQoVCRc07Hd0kH3dezFmiIYOq0R2C5zp4XG6zK5OimQxXKDN7+ubrYu9187tazSfhkvKqYdHVZdYKPTZob9g\\\\\\\/MUAQpmJDTZY7ODm8S1zPF2YtpOWev4tz17kXKZZrly6L0NrlIZBbm9r91lCYP+LVTg0uEKTkEZi3utBUbhFkGk\\\\\\\/L7VfoLtp\\\\\\\/5rYyQtTfmpRsDshAhpCUTK3q5ju522WAw9H5h6yZov5FypmiltrvwL0pOBd333mcbv2C5eoR4a615LkonrPzQCf2WX1hVskkq6sJGUtTh2cpbsCId3jTb2knI0UAPvNw3ubbQfkLSRTyxjPxgQUVEj47dEGgmE96RhdJySuf11LvjnnVMbJicDc7Q7PT+QsyHM1s\\\\\\\/L\\\\\\\/vZZH8xjacNvEhl7CUu6h7gK\\\\\\\/C\\\\\\\/ReINsjwEspLtcvC+OFGEuNEmrW9oeiw\\\\\\\/rH21oWaHSodmB8kOB6AwFSE6Rt3Tf41SKxUMwT\\\\\\\/L2K\\\\\\\/ZtF9H0LZALzs52i3mvBHsEVKjyM9Vs52O+6yPbQmCxgUJ\\\\\\\/Irakbqw5xKX92LsdkHBK3LDkaIk2HzHqAqQMRcysop+h\\\\\\\/rvh4WgJyipXe+r\\\\\\\/HrZ\\\\\\\/Ph3nFrR8aO8PiqVFoQQU3boHbtd4bqqjabqKr\\\\\\\/XU6IUPJWLTE10NIPOzYKdjZ4cyC7AB4yk4dL1TlV2YqZWhBBv9AinBRUw7W04FYS4rXHcPy\\\\\\\/bOoTNWx0PfkSskzH8C9QO0UigjBWFQvhMO4M7c+CXOl2tSAf9EWlIoTYkpbU7IjRzkHyaIbcPWsf49RUAptxdirCyomuIUmfh2eKWTVnydelKjgr7cM+elqPa6rvcHc1lY\\\\\\\/zdWWFeAwZ4Eldpytu5izfuYXstBY28es42lSznaymZs7yizu6uaPKwUjkWdcBuUnp5Xk5niWD5YU65pd+\\\\\\\/QZkmvc4cTCmnsQmXNER3\\\\\\\/cUvTKU\\\\\\\/Q+IRrI1FYNpY8i5IQyqhIvaF7lpNHqs1+fjgHnsBDgocDKoQVXFFArSWuZ4NVHiArAbysgyZJqPTBOSVnuHGCGaBi1JJscKOk6WibjoIlGGHWKc\\\\\\\/ym6cEleXJuv4\\\\\\\/Rk0nZIdzVWXABNUSF5ozgpRfuaYta3E\\\\\\\/5jQbQmBAlM\\\\\\\/GJEi1JcUjzrjykI3RxyVlBIt0VDRLM2xOIqGh8nabyHW2dBELUTpjkzIJiKOTTk186aZ3KwZl2MaWGY1TY5VxkZ933R4zKFFzm5yx0U83GDUCilo0XEXyomvP347j9zSSN2U8iqt+jkikT0X6BBE+EWRbdvTeh5z19iGmRZ72qNfzqRtUrmnLidLqcdSMx\\\\\\\/G0sQZnCWCoH7RG0kNRDNcFEl+TDY6HcRd3\\\\\\\/7rg0bjAj93a2OAbzieJWJ\\\\\\\/WNiyxTPUPypOcleN8YJyPIb6Pst3Q\\\\\\\/Xtpk8O\\\\\\\/Vn3lSu3hyN0LWqvfaYOPZt4PjTyItlh+nq4cs7yQHKDuCyuwROgxIwkUbMam+0CdWhpTxamTXyUvhhjT7MjDa2gG1rfkySz7mRmSb\\\\\\\/ARg7LyUVGZqjsIsBqDubEqUR9gd19l5ukZpipTmt8IKGlAx5bXqI\\\\\\\/iW2fsT8bsWcC3REUdAYzO39R4BFbqGCS4+HTOfZbmbltt97O9ZDpoZAt0uP0WOAifIlsyWqEZ6nOR05jzUSe3puFscbTU1pvODU\\\\\\\/y1hd6EGM91pf\\\\\\\/u2ZfELDMKs2H2sVBHQMzgeE4EIzKOmb4EIG9CD5m+4YyX7Zse6Eofh3LrS4DztI966qrV2GZoF\\\\\\\/a9mxJTAYHDpBrlt1vHnjNedK8QD1RitTmttT0tYHVeHgFNYxX97epmxPnj6yDsV68cF1wxhlhs0VqokPpkGheZdu2F+Vwitqhm7earRzQlQeAHdny+WpM0YKqoHb5lBE5RBMqdM80ArgbrdEhrSXc+gTN7FWF69h2ERTGcVomFwaRXMUicWe+00H6rn3faVC9QCPyNuR5+7Yux3Dul1KvB2eLjEgtdb3M+IkGIUsiqPbjQiWG5K3XB44NDberuyvvJc4\\\\\\\/\\\\\\\/EhNdQsk2GTvZ4qYViZ3tq5EoasnGovcXSBrSzGNgu5KwK+\\\\\\\/Q+fcNAn9ifV5U+7Bji80\\\\\\\/CCQecbvFhiSnrL+gRiZgkIKSAht41vgsBTwbMLkudzAU00hkchC1SzqRQoq3L9DzyrtoRLRSH1x9WCG+Lcn\\\\\\\/5zmFGNaP7oh27vzVTr5TEoQlGlwJnWP52R7EWCViQ3UMv99eh2wkQLJ+k\\\\\\\/dURDmw69AkaOeFdJCuzy39y51gCAvbJa8BLyRSWK2k0855EV7nUy8dTUtmA5yhSz\\\\\\\/rO\\\\\\\/ikbkGapesRLKUSLVLkfdnNxDrif0EHsbuRHEDVv6Zeea36zcOGghg40n3oLzc1e2TJTJI0ScX7n3ydVf+EJn86HKBUS83\\\\\\\/7BcfCdZZSDGto8sDr2VtvzyuOIltapeiCH7fk60JySv\\\\\\\/xSSRBEMtqNlIWcYG886zakAIbJ6s\\\\\\\/WxztwakO2axYRJQMqwICuLTSEtwCdmO35JJcejcc\\\\\\\/arqyu4GY5XB+caqi55CN4r+bz67lvBTdk9VsMZ\\\\\\\/a6AZW0HZwJ+W4ZhX6snyAj5XyHkPlSKG4hiphEmHJMJnm13q8VhbHaH3iBKLHcH3At65K\\\\\\\/kidR9SEsFRtwpvate9TtHMvqi\\\\\\\/sMLAdMWG7XJ\\\\\\\/gJ4e6w2MOXQAyptIL8IMKi6sQ5eqIHkcR8UkbmzxOFdPmG2oni8DXAXAeTkDytMujz8yHk\\\\\\\/q+VGQdqtlbYn66Ox2QpyRn2xh7bT6i53Pgc2OgvVgcSxUm9L7FvhE9\\\\\\\/Ap9Qnbvbwr2T8LkUf7yyD7hPOt9J5R615dWgLj3bsOhT9DrRH81e1QYkE8rjmNV9LNsFM325lmzr4jmMqu77RopNIGxWbqRlgSK0ztecL5rmScYhXeC1FYFkV1ldRdVoMkVcmIL9B7yhHuqOJFiaw39ELK8M+c9FxIxXxU1vSsDIsL\\\\\\\/KyK\\\\\\\/fRh9VaE0xM+qBaRYN41VvJ6vYcW+GHuURY7VSlMHuwga8PP33oUFPvilCaTl0NUXu87nrFNkcbW8jxAkH28UP+jgjYhMoGO3S2K5ZG04C\\\\\\\/RkLhto0\\\\\\\/2gsvnl0pepSHgsUIis30cUoMZW7IdgZqIHW7M1t2Gui06ijJHEva9iowKC39Z1TEaz5KcQDs4RzfF3FpAmfViFkN3zaDKBzHf7gvoeEFtNT8vUtMmRCnIgtTeJIXnP6GWj1Vtt18wZ2wQbPDyP57YANhw2ZOj0F5x4vWLAFaNv0wrfx7eYDXYAQQJOOUrd7GJt4cnZqHFevYrefbyZErBGn7kc3aIZMwx1G0DIL6kzdl9BZA7Nlgeaf2SJR8AUYsrY1dDUyWZleV0+7E1n1FF8imTSfJHfGzhnh6ASCbAkOJ4bnyQBU4rFrHN2tNDFQ5baIfrb7iXwClDIq6ZLi+QWGUkeyMTA9L+ofUsfMFP8PXPK1J6bNjACeV9tG8ZTTpEAnBQHtuimaQHWVcb0tlFBcpwGJxD9P23fHdu6QMwGPmRXLkld5mqD1CaKJE2n5zJhFAv87Y7hNCfS9RpE9h50l0bZ0L+f97pJZaMteL5blsUc6yi3FXxVEzi31Sotqfo0u3cwncZRIoVWUHIzX\\\\\\\/Wqkptj3MumKrgf9tqo7HpKIBOLnVpRpPcmSPJ2HJ3CDEJZKaVUFNfsi0rjRfbPB3rZipxKrOBnMdvOXpuOHp7agSOkqlP+cNIJHqJK4O2EgJWdRoh\\\\\\\/XkD6EH6eZ3J6jdfN2Rp7evDTaIfgjgYeU4Z0FRihAqhg+NGi1aNj+ec64uWKxsvxaTZwJ5l\\\\\\\/Ym1ejoYFyqKcfbCXp\\\\\\\/MHAuBUxKjr3D9betc68RCXT+0oQBXiF5vTMjLvjnLZIiGGTVzm4bMmQ1OdFZDEWXXcYBIuvB6EQl1jVVSNSMygMA5twgGRdIQnD2M5e3tQeSYn3kRQk9Xzm+kWvBOQHLwZDXQ\\\\\\\/hAIHRM27LUQ3ZCnSXrxB6uFOnyz6hZmhOx7ywdjdx61+BnmYSEzRI3GmnakalgHR+HHLMmofi6fgpe2iVuYHFES3xVaK5A5Pud3sO8Zw\\\\\\\/J05OFv7XdkNc\\\\\\\/+5TFLQr2uyBV+Wu3jQl05oVJT\\\\\\\/zH4Cbq47YW2ZeuZm79lbwh2CvhPNnnPJlVojpgqflT5afjyXT+rXXNbtCdnuzEUrz8mMIW6WCpDs\\\\\\\/qfVDG3K8i8KaGBmGCBONyFRFhYciayflhNY3IKsbQHySRaxLvwOWvfjTYm0p6E4al6sibLawu4ph+55SOAlzlDKl3bQo\\\\\\\/gSjhRNnJKLCxEUXfRvWdLUJDLi8gXAGv5RLSQj9c9N3ykRNV0G0v9j2n9Rx+2YQHf\\\\\\\/mt1dHtXqDoVHHW6KzhtxSqa46nY4iU2bQzFdEA\\\\\\\/eiZNgTgDcsttjtoXj9nGvugF9otBbaCf5004v8wUIltjGe0jzXdGnd3TghBKW6cgxtwxES29kN1kG4Lwe06riyz5E1Vzgz1ZNB7vUZBAKqYy7kQo3LqU535R1mC50NDg54lliitg9FOrLvGL4ZF71enYImF9ejleKdVUvtK4ChSk7VZQk8pyWXel7GLc77apTWN4Tt7B28qLdlAbNRRcRG7yWw3PLG80GQJlWldjTMgSCAUh9wEPOuEQeT2UXPkNwvbYGLbVhAyoFTko2ySI9OLa32u7loFd4aEaCdcKc6DjpPLHsaBbEd5ap+KdtCbGLZOd982OS4ZAxyzUe2PVlmxKb8O6Uf9qlcVZEz4SvWv8xvjuwuOXRMYqWAbhfzEVX7Dq7s947t52moppdhy1ubT17jAR8c5H43SHyx4BU\\\\\\\/FZirVsq5sLAVvJf1XdznRfaOSWP8vrfyjHkhqSk2uuhi6Lr6Pkej36eala8n+NHcdtjSazmj6qqBF\\\\\\\/VLn599Q2L\\\\\\\/crs+yZRW\\\\\\\/WJdawn6ZR0hF+VptePtsxOWRuVRAHFMbutaTwKXRsWd39nh1NE1ZUUfb4TNa1A8xrgmMUlTCWpH\\\\\\\/+mmNa7L1mocTXu0EBKuYR+ei8Dqk52zD4aKU4ALo0KPKqmmvrR8v2+mkY0Jako57biLsl4TIzScvieKGNvIeYYBgdp8MXIx4JWI\\\\\\\/5iCnedNwLHrfRVrK2bNzbjPOwoZe6yfAl9OWIr+0yE9lZqHcA5ULNHHtKya1wUoFFVGmTZgE2Iut2oySx59q9A9rl2rAvBLboBejs4G0wKtAxfILEISMnCb5g5FyvuNFd6t0V5TO7c2TA28+lfJOq3Z\\\\\\\/BH3qVmbHS4nM3zHcBp8a7bg1Coqu7oc\\\\\\\/09t1hHkBrPkusIhVhHhqk8mD7P6Mbqtsy1iW9zDQvVyTWkzyZFKqHemx6VCKqAGa8kHWYJPwo6+88u\\\\\\\/Ef5CrgjcNVGegPJt1dJVWtWzMAOyXfFPakN\\\\\\\/vQK9JEvGy21wRhVg+ijzgZIlH2f+BTQGYWqYa0izYUnIG\\\\\\\/9eHvGg8JWIUbT0n55rexnrC\\\\\\\/PnGlNJ5\\\\\\\/S0S7tFjVIxyOO1Zsw4sqpvpLCOaay5lpihsj5x\\\\\\\/OwJU4cwJq44pjkKrNiVuWMY436Ewc3qRWhd1pLuZaoku8JmA46FkL+fXDMqYOR6e6H\\\\\\\/YWvnX0744AnXXdik4H+VAYP47T2KOf0Wbqae+ip1Y4lwqMwByvMc6PSU67kEiAapcdVB\\\\\\\/TY2x4v2YQweyJV40q5mFrS+6IMO2\\\\\\\/6em2L8mQZZkz9XD31scBbUNYWTTtSbRpjPvqqGjNzOVPFKgFSB\\\\\\\/tmEuhD1fUVVyDHTm1qzF8anTvIkAY2fka4j\\\\\\\/kGsjR439NCZHmYlTbFGcPHCSHK1Ryby2st3XUCjX1UTyRYN9ngDZLhVIJ3IINC4Za3b788YO252bpkrmVMogV5rUCl5PQiUHQ805eQGlAhZP9OTgA3xDfZ15KBPQLxjjOawqqjzizwh2z7Wv+pziYFfdfdLLLBd7hAb\\\\\\\/Tu0sLZNE8YMk4vviINj7sVqyayWyeB\\\\\\\/QIhjOLw7WO55BzSFdY+vo2ewBMrGJD1V4Nh8XNENI1FK5\\\\\\\/O63dvwW+gEZmf82UNtqcl2f2jc71yB4CKhH7\\\\\\\/9iJrqrAyoCLpS56o65BWNnm\\\\\\\/NWAsIkUt7zc4UMOIoyEE+qvhRS+FJIB2dToyiXzUeatyG1SP79alPS8v9KpuxIEUlUqnE1y5NK++0T3O7d8P1pnSKVPq+HTlVn1d4ZTZ4TnDTS7o0qE1M6oRXbKO9neBkHTHgkCkNnrokAKcPz3Hf6ga8RPANQQ48mvwa4m29EInlhIsQ4uSt3mg0pomKfJZg1M7Di5mVbc+agU3eAnIoRhEGFERsSzo\\\\\\\/JmeYdDbxasXDEusZ6uSmw4HFhq3pJ8uIUIwZwWJz6UrvxhJkCZykK6eJjBlu8PEOiGdJ+Aa1AATNVdgCI9OrDoUFcJFDL1uZVYdrdXQvhlf00MV8OEYeSERwEVsEe0bBOI9aK379QLh3jV+jwjFAQOHvbkYqCQQg93ciTdaw6gZCp+jv3EA+rfSHKIz9pTXabVMdtIqkMuYN3j3E1kH0S71XLG8kOexUATz2hXbZzgr8OvVV6XnrnvheyImlf5SZ2pj2n9jzgKvGxD56qvUgVYus8BKuFbuUGBsUrjhvF5C1Bnp8vuu0ytVEVd4eUy4Bg1iEYD4mtwL4UiRKjt6BowBFaEQvZN58BiQ8k0t8QH9id80XxNMmDcalHalRxeT\\\\\\\/IdkCsHNGTOOZ9GVAMQ5MZyk2qKEKBnd2B9qOisTH2spGlc\\\\\\\/A1+ZuqiAY5qbU3OFRV\\\\\\\/2DfgrFiExEoD4gtlZkru+wNGhxDKKj4iirJN\\\\\\\/dxxVgItkBPSVox2DtrIi6HgXey2dY94iRwuY9fyV8oUS0gJMuJfWC+S2gjqDoA7FgtVXLlfeKT3K32riiI0xNkQlcSyv9kpn2TXMYE+Rr36XNOpSaaqkBtKYg5qRQA6lq7hLUpYKc1AETtCXBSxHA4949W60dm3DtIZyS6PkAaUUm\\\\\\\/HJKfdVwkeU6JENsNTZVg5uMUz6Tf6dAt2HMvtB2U188Las5\\\\\\\/F333vS\\\\\\\/2kaQp5qtuL0jYskO226CVzyFHYydzIN4EG\\\\\\\/bTjwoDLeY40onfK\\\\\\\/iBlYBcE3DbM6Um0LDoKXQBeyOr63r3pipPD4QxAqUcfc5p+kcmIjCM\\\\\\\/M9WV7MlLv8RdvjgPbfIW\\\\\\\/tnA7mleNHmt3J1Ab4HdmdNE2Q9J5drkm3SXNjXrw6lH\\\\\\\/7zOTWXGYOTvL+aTfwTcHPE78wzBr7chevkBE50RsspJDOUsugC3zV27apPtp+QkQuv+rGEpl0mFfSZ9JtSmpyQmc59BdvcLphD7ilreWFSiukMuPL8DMlskz7loYPeD4wyiX+jNKA93L3PMMrI5o0yt+XfgwrPbKoEMwAmW3EV97krZtF+SVsEGXBK2+YQbE2QCBHPE2F2EIDFYbFaIjfZEPhsbDef9UA5eC0jV41ayEcZZazncnUtZC9rFrMcGC9vUGd9oZ+oTf6jjNRuPNDP1h8\\\\\\\/Pm3UZHVFKigJDORDz9b6TVOMbqQ5IqEx5mtirA6ccuuKyspjaJlLaAUL3EirmuvIH188mFwou9gCxWbFIwAK7qYVxN9c6qE6o\\\\\\\/4\\\\\\\/Vpj1iL07c22+IU6Z2Crdz94XLOK0sRyksi\\\\\\\/\\\\\\\/JRAe0b+vR+CU6fED9qNc+e7Ezc5dEs3MSv93iXN36Rg8XwUJql4FiF8KMFCo3K4SlD4OF8WU0HFAsCM9qMK0jL4sRu7cqpYMtIWRhOnelHt\\\\\\\/AtvNIvapgISHW6GurA9I+UNHWejKpcxrODnG5mw8rXqf8a7m0BgPTrQuRdTSeg+g1lEVzPcYkPIih3n94+pKufwKeenM1LujclJwsXWuHh8onUkE3YekL2Rjsj3Ubz9K\\\\\\\/zj1QPKe6p+GrEHX\\\\\\\/KWrrbHj6xlYlbckLbkcS6ABuZHPbw5g0PRZGPf3TuPHkgHh830JvTxHZxyHGzOo5i8j2kndBfeLd\\\\\\\/sELJLcDDEgU\\\\\\\/V1AmGADG21hgePpKA9rcLfYs3AQNCvdpBfNWSsjy69\\\\\\\/DsK3d7\\\\\\\/b7wyx1caM4DobrMZPMSyoFI\\\\\\\/pYv6QH0OZWqERbLXd4Y8ry\\\\\\\/YFiWKiSpTaR\\\\\\\/91akwA9xocthP9o9zOw4C9qaSg0nllMGXkm3dvRDU0PyH+zIrsLQzXerLmuxFwSHnwXa\\\\\\\/EP4\\\\\\\/O1vcm8O6ElULh19AJNXwrgVDw51cgonV79l4IJBAlq4NhaQ0os8etsb+y4fj68zqmRg5TGb4sX747dE0GX34+QtHgTGLgW9mTtsIWzRFtSAyD0bZyrjXXizxr6IFRCeY7ZlW8iNIS9OKj+VbLTotWoUgVsQLwKLuovaKf5r\\\\\\\/uYyF97ZF6CwO7hASIR\\\\\\\/0e5yKMGu6F63dShM5woLti2WsO5HHZw9bqZPlp1aeH\\\\\\\/rAi1UzZ+fkgNN\\\\\\\/lQEmeIe1LfvOeBr0tecZSDHY93P7WzmU+ShH0ungb7\\\\\\\/r9oAcb6+l5691e9cx+1MiBy7RGtYx5Ez94wGHrovp3z4Enrw5J1kD8psP2aFXAYSu2f+zGzLCex1yIWFJ7pqkc3SLjLzEeBwAmDO+Tr99PlMMsEKHskz0Keevrwi0SARy4y0UGRHCl1uX6GI4DzdLMp0MMBOnmuMlpgUv75Qh9PnQonwSjdVA4xodSMrkU4c0NeG3RYep1pDQG3V2qaMkPE4lk9f92Q+F069rsHAdr4w\\\\\\\/SvA8lqQEdYc4wSlQERlC1nqTwBuftwum13S\\\\\\\/YQQcWglNmKa6o6s+TmT8tQCR2YVrh3Y0oJ1ri6UNjkyCPCr7lFxtY+88aE+jylQo0Zy3ByOscr1kmogOI4nNChtqvRvLNrDhdoBozM09zDmBduBXBNmOPQAidLbMpuNLpmcxQdOQMrKm\\\\\\\/76LhzmTTze0QF3AqofKefEqpJqxOUxFUZW4lc73k7uPnP6XAZoe4y427lTHrz+Kt53QglhMJVhpqXvrmQqC5\\\\\\\/bg\\\\\\\/cbglOmal678xPTqaoG3kjUHdNocIww1tM8bWO0AumfnL35M4soZU2webYSm5NuyR936AGRFJCqa0qYUpwcrPBA\\\\\\\/dK3gWgV9cewitpQnNQ7ngGJ\\\\\\\/CaXGflzf53kIwRqWpRS+YKm6ep\\\\\\\/\\\\\\\/g7Z0WNRctLtXyxXREPm1zr5KoBKDmQE16OweJTSnHQKovWMZLr5hiQqdWxyr9qkF4G260nN1BL3mQbZU2hVNhyH6XjDGmSS9qcubT2giOfsd35Q9Y8Ke6qpezBZ058fNd+bSGfA2Go1hYc7KEH\\\\\\\/4xDTCbuBpkvPOLacVkDBBiWw0bZcAIgbuq0ayhMJjnxHTuGQQs1yPFvXKdenEhiXYjBc6bpiDvUqza6WwENwHhh++MIDRW27k8gbht2fb1fzEllTkYBV1YrkLFbaY31t2D01Xm4G50PTK7mBnwrsVh498YSKKv+yOPrJKosZKTuQvUmA6URE5CJSmsfVs6530bbwnarRVY9\\\\\\\/8Mye6J2JgTexsNiaaqTsCRoI0BOGtRmh9u2jTPUPR1yDys7RwyYsXFAtJm+bGpfENfj4qkQXN7isVZpfF1K\\\\\\\/EDXoktRQUhCycsV7FT1dC+brPLF9ZCVJFtamgnOvHCCBi6VdnzqR3QEUNOS\\\\\\\/ECk+PKI2o\\\\\\\/DdyYaWQWq5bGJKQ\\\\\\\/y5oLYWg2lTHJMfPJH5oUd1YoFPLLv3UFB5lSFFyZWxzeJ3iND6lPihei8UFfrI8xSAijd31gaeTHym9jSF8h8wBYne\\\\\\\/GMRYeMbzeiYje3h1kAfPEBxGHxSZ5mQgB9nzhk87GgaFU0ZO8NgThws\\\\\\\/nQsf\\\\\\\/Mk1xVR5Mdne8EDCndSZvSkZ67TsD4hQZKd3klvrwkrWLGPlUbeIc93sK3TKDhv2\\\\\\\/Eh3OqrYLVxfBU\\\\\\\/9G8l8f3kue4VWpOBqrH6P+F+G4SzkVqUtB\\\\\\\/EYb4YXEo1LxmnWo\\\\\\\/OrjVIZi6a40H1eXbFjSftSM9\\\\\\\/54NOytViITFGBIOTfSu1m99\\\\\\\/iwhSGVVDwpo+m33Via9M1BHLte\\\\\\\/2oUCrquDby2sogpTndYFpMHRKwMK7yC+Df9bs6ST\\\\\\\/P3+56GN8VCku8aSE6yF3OaujpF605ODTRGMcKTPVebB0CgKL1LJpw+WNAdddjq0jwdFCWnTl8YngKiPmxcFhRvLwQ9n50RAkRuZ\\\\\\\/gLtuZGRbdy4jYV01vvFwBrYJKXxJoe8\\\\\\\/n4Rum3a4qmHta+Lh4t5VEFQrrw9VwGNSuuCyyihxODRfzcByRas8lvE1pE35nQIb9Z8AUQImsYWKgrFYj64Wp8i4L3JbwupPdSWThWndsqHuCeAGNDrz2bTU1SEny4R5fMMX1ntZpCzpf0i2D1tu9F3CHoEeMjP\\\\\\\/3Y42wTqNk0se0aLK4HqUgIv7r7bwo0Wp6VmdQc3q93UYJC6dPbgScW5wIwlK4gf1nOAOrJcepZDDy8k7u2WAJWEwL9wol7b70gdVEgPVwy4wwThJ\\\\\\\/bgw4RrvHf+Z7\\\\\\\/B8LXVoygAn7WKlbfwJPYXM1pONYKJyaXdBG9p2AhNq0JWm3y+s9fKkYxRB327a0\\\\\\\/9JiTDY73pF5rFNI6K0TRAxIJiT3\\\\\\\/Tedynt4w94l0GL4rMyRaZ9a7+p00SbFPl5qInkh\\\\\\\/wMCMLYZ5e3Q5kp6pPxzOI98bbxNIb12ITogtKOESmHQDlc6PQ3rC1vUxiFSaPAjyec2gmnubIbvoquvJp172PfYRpz4EVF4\\\\\\\/leOQ5xulcLA57WAKGXttYk2AhX7CNH5YYUot4DleoE7MJ5oHRGTogzpJL6qLNuW2odR\\\\\\\/gIM0DD\\\\\\\/Ky\\\\\\\/16XeMZpkquCp1KtYC0yatvv9tV1+WZNX1zlDSUm1rjaa75j2dWBn10BImpBzKLbTTtzI+oAsEuX1pShj+whnWl2NE5BwA3aTPge8qmYfWsCu+Yg5eF24bqjPClqINOCRsm0XsOA7GnmY7Fsm3Q1npSTU8tUiZDt+1mV73kQr1iOAvc99OMRjnOgZ8l6UaRQ5GKwyyn8drV6pRb+hxqc280KXoLLKPDTuWz7tnnIFSYxUO+fkMOPkHbLwUYXQgmmkXpDdkLey5Y7KEwKMrw+xGUwroGzZHJAudC+05Sp7iGn3A9TcUQaWaCuVKVPb10LAeqHQ5Kr\\\\\\\/c9wvwZZDoucpPkIQtBNKq+9MOBxvbNcAE9rjXg39KYtWFKACIhvqCmuY3o4OG9mH4s56Xq+cmdBRDVxEB9kSiOpo8dyvHObDb+MBCBBCgh\\\\\\\/4sv0Y8zw\\\\\\\/r7xsBaAbMY61dRASwhtd\\\\\\\/sKGniMIXR1IP4FbQdXluuzeJQ\\\\\\\/zC5oQp3sxL4qRZMwuofSTGXUod47JI3tOslFsXd6VgpKxl3WIYA12ZV\\\\\\\/1OCv+hVDuxedvLuYq9k5sJPUfYomFTTgnTbHTKuFXq8su1n9PBTkvIGuYFJFmb29tiAVhLXZWMYFKfj0pYJiTt92EJ16oj2dhJMnaYF0hlM81a27ar+yxCKTnR9tEk+yIGQz9XvtBIGeXl85or\\\\\\\/dJNxKswRFvRuAILrD9TYHDH6rexE+Rv8JN76tj59I6obPHLQBQ+wUQyhjT8whYx8KhqnBWwPtNhRBWR09XBt3tck18IZrk2vzRK\\\\\\\/A3GCfBkEi6oIygJEm5NbAit73BDglbRqozTFBGb8VHPhPOGhc\\\\\\\/q3d5Hk88Gcyrph4ag4A511Sy\\\\\\\/SC5oA3bUshEFAxODBi7ROBmfPvQTGwlpzhr73Z\\\\\\\/Ni2+pkIVNWQPtrthjXoD17Zq4pDXM7cvCr4V5WkkctbvXFb52pX27fyYp6RcYFHndQ1pJhs51IiOf4d68DPmfx0gx8ry2ietQlRvy9khMSr42P7np96YPSAvRtjyNqbF7PMhhdKg5XFWFH8Fp02eHB0biLY0c\\\\\\\/N4jLCySnHQrKjcdwrAwPK8KR1wWARttcY6E8eZG3dV8d+SrJl0jJZJxDj8B4ENAnMMTiMl8GY3KjsHV6kBg4Piaqs5nqpL42xG2iQv2tN00GEVGLAMZYWpGYQ6gStdOGKJAGyz2hfvSvR9K96+UUvhjQ6rQOZBIaU+pXsDm0ZCzYvsvbYQIratPd70e\\\\\\\/JPGJFu\\\\\\\/qpduXl3ELSFIGtv8Wdc0x+kxzGhMUxd2PUfjldJG+\\\\\\\/CAFnlif0x8hzY8FBZaoSlAsJPOFXoHqVWU\\\\\\\/5a9MZLk0NO+9mZ382qaAmR+Ll00CMwL2fAoIB3L3pYezACKEgqJuoe3znpL60FU+ZMudv9W4VSdats0hGvD4CzUl+2oonyK960HCHRaNtHfaa0fTXzlOSaKh8lLjJbKw+x9qhJtF+I+XsBAW+VA38N9ksUf0JOEs2i8m1pGyARi62NRSNf9f4YNI\\\\\\\/D8stO4rNfuLHrEr745D6Qo8Fl+FApoZL5m5xEke5Pi5PnylzjIC8OoakjgykB38\\\\\\\/FDPwxOlbZOw2EOdOn8z0oz8ib+OEUYsRu1KClPeMEKDe8IbYqTntKWBmZYR4HR9g9Mqh7JBYYYskDq\\\\\\\/nMmWsEaSKDiA6GItImHNw9JWst15HjsTjYyo0n3gAh0T5qXmW565sQHmbWQWMsYOzCpxbvlEqnwU1xKgcJkClXAcLXudu7EQ66zQcKIoLvHwAXNHk4QEmA4PkFyCQPuAcxmFf+nWGSsqltquscGhxr0uW7CloS5BRVuc0ipnSHIPA7moolYZ0rsH7B6GcVrU4sKIiSOUvYYu2SXER3ECLAlpqHMoncA4yPGKgbfNgYTRllKixQAgC2E0+eVGTgh84KEkwgq1dYKYv6nwvB5x5maNyuThVeo8qytCtNyIU0S1v8J1rfqJKkAWrXRCIEsp4NNsgtJweBARm8buXjYwLE\\\\\\\/DiBH9NB2W7ievoINxQKKJ3ItTqrtnVYKgxZF45BY2ieWbWfACszKjgBtE8fDCshZ9Q0iMnPO2LD6Y5otT9nJch48jwAsAWXBz4siwkhuLk0zVvpP4nseksJ90PIYu6+BLjg4z2HxKpvV7a+ZU2bL3cD6FttpjpU\\\\\\\/tDAGkjQyt++E4sEY\\\\\\\/zJrE2T9PGlZlefMUr3DgkbXdK3r64Abb5+RmlV4tHBSHCfiOpbCN+4rK2o9K4ZUIU5Ne+AHJS5+n73aPYL7f5uLQX9C6t5jlLxoQsxN+E1WFNmBwFCuCE1UtuypM\\\\\\\/XzOCMZfdTIs2Q5hnZUzHnE7UQg1CWSKL7FMm\\\\\\\/nlmvUu1Q7XLNy5Ubr+oUP+QIZkgd4jHIKvloR0v7u6vFVNj4jnhNqdy80Ld5Smnt0ARvyQXbbnA1FDYDWOr\\\\\\\/AnSSg5MCydUx07biMoXxvh1oTu23\\\\\\\/INJupzdxAyiTcRRQZ2+wMKyDChP3nnJwSGetU2qtlJjIWdIfHeyytvmYZMr5qpYr7BHTLJCSCTIaYAS+exDp8GsXl+CsjxJZJ5aEDPEYh1YEHQK1ooYlDW+xsmkx5gcBrVOBP44zcEmcb1KyUypeksIM\\\\\\\/aKdkND2Bk\\\\\\\/e3MaJmr+ZeycqZR5UPa3QyokJNQz8yWeMZKojjwWmhdWPqNbHuTyzX1DV\\\\\\\/2tUsVJOWx4CUzb5Rhlv+dY\\\\\\\/dS8HsfDWqTwWhJjqDOUMOTZv0WquwN8z2ra9wG97xC91UbPMskIjs1dq3posN+riPAiwvprEG9v1Auqp27NjEerc9SOwsdy1svtj8aGDc0zl0tVD1fDL6Zyc8tS32aaY1XGtXWQPZ\\\\\\\/WTB5f2\\\\\\\/qtPBfES5m7lbGR6k4qCaRarLY3kBF3acU9nKm64NOBTqHtDLipUl0aVpyCqJA+uYlsDZ0DFU9bDdBWR5E0r0Ou5VvkSD\\\\\\\/Rj65Fy8nGQipPerh+T4mQ8EFQrtYsctRYr38\\\\\\\/UdcU15eKbcyH\\\\\\\/R\\\\\\\/JGtW86VzXW644h4+wPC9gAQBda0dMy5hZM7AgLJiPv3YjhN2jESLfO0VDZUgvbYuAzSHFnSWo414UboMHmEah9p7hUO\\\\\\\/jTNVdPjnwdpqINvSP3KQB0nwLk2LFGAJ16JSSiVKYJCPmy2xzBaGoMhd2HhaBBls4iTRMguL6V1Rd4U6EShTxasRLcy2AsMxjv9bLjpDHX+mIFY+yjW40l0++c+sWFRhJTcDkhDEh7J7Uzt6GXuyaYzO2mPYxUbnb42R4QKqJuv3RVimWovkjWFoau9\\\\\\\/PgcqMDZ4j4HlEIcvEs8+YZHl4oJ+JKM6qvGXV8lw5MxrOq7fXn6yxUQ0z79ISZ5dl9sBy3BrcKoHAm98uZZZEG7To\\\\\\\/xwEcd2\\\\\\\/YjsPkbnnMzCS3z26EbkC2U1AxKPieCM22kcASWmpFqkBPbNrcmHi9IiFBenroeOeq5vI7BebbCKKxBEbixHebqn7U+X9elnZHr00Kk30CxNoICK\\\\\\\/pYnd1QsNcUcgFBHpovFubsFZEGSUtKL97mGzWBM6jU4ruEIvGCIllqBylKDgWFrmoF1aPxgxeq5qONJGjIsGQuepa+oT2MHKcyVNUJYvgw7WibKItTmi5i9043tvYeyKzcuSn0z37qhGmIdbJcRFwhI92Of+fCgr77jdCEK8JR7ONeac400GPV\\\\\\\/bGiEv+cZZm7SJXnq3tRw5+e3YW+J00Mbi2GUkaHXmdD8tIHWOXGKA\\\\\\\/3EBwRcRUAx1d1IJAFB2jbOwMdlEwIsUkyLWFnsrL31MgxHNmh\\\\\\\/g5N4HIg859xo\\\\\\\/LvDY2Cub1cLwtAk0cxyOelfBRHKZdzkYzj0Jh9sNFb\\\\\\\/dudjyn5CfVU2CnVRm5W3rFJszinxRGbbuiSP092m\\\\\\\/XYEj8Hsxl8ShvbDUqFqO752Th\\\\\\\/HvB7B\\\\\\\/oOrh\\\\\\\/Z\\\\\\\/i7E7FN+3bZz08RVruzdKRnEZ1ehnkSc6pc\\\\\\\/LUfv2vwJFN21VlYFVhmVJMHf5DpPF06nSGji7rrk3LdAb3LakfUSpyzKf3vj4DRF4aITP3rE+5qZ2EcILWTDbIk3pu\\\\\\\/hRIkNZaususpd0Onk57v29hnQLPZTUjpH8f4f\\\\\\\/kbK6f2AmpAaRcv+MS43d8Fwdeh964\\\\\\\/DbFYHhqExbobiD8kg\\\\\\\/ItkOrGDQwLhSuGpIxSH3sH+xrP0Hjn0OasbkWMAnZ5dTkOEUCyq6sft2p1jtxgLOpM\\\\\\\/7CJQsc7eRimVymmB0nXHaRFURPWymzeppEd6fuVZjP8H25bW4N84JcDAvyapmKG7U0P0A7kP6kxoq88Kw3mu\\\\\\\/iDgPMhkkPgdtfo6cle7tx6vNCjqKalZFT+our6dcj4hUKrSVvZm+g2h0AdiYzfl0ApkpC0lhgtymKKV1rGUFgbqd9PQtSo24PXuGu1Dwh9\\\\\\\/JHjASqFO+zw7nQT5MkYzJD4XqBK7CAGF9BTo4SMfpRJ9ti9yYyGsCE3GX4GqF77UVSpeo4C2kL1GelmaSgZws7GanbpDXnNSNdBV6CZvpJwvrTmc6sH7lbsCCX+E+n8nEBlTRgS25V\\\\\\\/vGpEARtzup6\\\\\\\/tHkw6wCd8pzkJ09SLChAbw60oyTfkjDWX9AuztmF7nHuO+NhLWYO47scTUAH08wCACdK\\\\\\\/pbWMudDl8fH0il3snGlPh1QPa4+9\\\\\\\/XbVibGH5VH83jlQ4FtpZ49i6QNCeOODTgmoABAwIYjf66zB2oIQS67bRKzD6Z5kXzYhD3h+AgzaAVfWQzCXX\\\\\\\/LKb\\\\\\\/AKWeWhU5EA108bpBS6RkCLbP2s2TeCaRYDS3DnhtCnuJtPHCIF7PsgWIv02NAC9JC77sez8nhC+QOQkH1UEeB6UO\\\\\\\/FyzwR2vYPkRkELRC3qggfugNdwLryRqJezjnS4rJkwsGu\\\\\\\/wp4t51Q9egjYt0goHK2giybG+Fuf6WET+k6425Tpuzq6BJH6NDLbOe3sgK7p8jl9LeTlkgB2DVx0nbKhvFv8vnZNO7zT+Jnmj1Vinig3vXV8oJHWMxF2+0nWrZryAn7WdFRGrJmzlMwFX582RG13xbXEME6MWx1Bmu1m6j0h1u8FMo4H3DksN69YGDt6d9biHEktlV2rxHG+1lzP\\\\\\\/gpUi9vPMYtap\\\\\\\/ypmknmmBqey9CW6LVHHqOfmfFKm8FwQh\\\\\\\/4Szkos+q1qZ7nr7xSKVtVrhHiv8WcJmuYgAYdxMKoAPi5FhEOIiWcIWBvdCO7GKxEtlu52kTVxbv86xdkscGopWCsCJNvAY3RvJR9Vfi8ZgCFqyKBpkrC\\\\\\\/gTDZ6SSVryNbceBD3BbsgU6acf1jwGUAoMqH3YGvCCqZsr9TaoK74SKzX6X9wWF\\\\\\\/23RsbaG6VbiywYVTuZ\\\\\\\/T1mIN9gWwqzqUfnBmopFvIwghM05cuDHOCjbCSmX7JJGArQU76f+s\\\\\\\/grYawtoyz7OCTLr\\\\\\\/puFCqXzRTAg04cRl8Iwa9WdDQjT\\\\\\\/Ejb9t+Sn3\\\\\\\/jtfbt1W3VCaN7dlvfwrl0SipAa7HFVUuhHJnOo\\\\\\\/yKZK+5G1HebkfB50e3Nn9Vx8\\\\\\\/lGFjqLHRK1cLLJRCDe05Fp\\\\\\\/UWHgOUAgGuMcx8CUeIZDhlqrC4D\\\\\\\/WzRXUpefL0y5xj8UgyYYjSYvaIuNSc\\\\\\\/bM6tpHimfi965niSOg8JfHh1UQ\\\\\\\/xHGrybQEQcD+f+iLoO2xb2lRXcLhq2dvpT2gjWt5REJRqBxhWl71Scr1+zVHNBrrKLVYUdAw1xnR+zczaVHBXbnQAEMetQojLpiKQIKkY9hXuUUcdDdeQo\\\\\\\/Y3BWFQiFP\\\\\\\/WJfyBlti\\\\\\\/uGcenotMIIkbs1Gh+LLFWbqaEcJnzcBZqG2rOAzZJSUcTWc8JmrsVYoKnuvsWCHAWTFoJa04IwMeEmwAraK4s6ToD3VzKBvCoU1cI8CPX6tXTI+w8VUGyJf0NBneSkgQd3Uf6Sq4V2FjtjkEvuIQHkD6uYvobCxbnfLD\\\\\\\/AsTsRMk8WstRrN\\\\\\\/yr7BVYv0z5TPe5GdNjx41EHal1UKf5\\\\\\\/pL+nqCTpt1ug\\\\\\\/yOcUW5oRGwiFPE+MQcJpPRqxIMnKs0sAa\\\\\\\/YTAyiw8D97iIwUNu4LeniCWHRkAjvIewL+JtgHMgkn1yZb1ZGY8G5u6737\\\\\\\/lXfY7wioc\\\\\\\/r\\\\\\\/8q4EifyTssoUzNsb4h3tJ\\\\\\\/qemyMG2tRPjjg8VCZTBLdWB2IDWQ1dbPubiMlhF89NZs1BkS2xma6+4NNWOgCVM5gWHZSm5n+7h8N9h61KbhT9jVCvQRHlVi0TlMflusVDZXyLZM4UkZ\\\\\\\/\\\\\\\/jHZ0HMtxHbsJeIGcDI4Q5GZb+aXit4cVNo+lVQ3MuviTPx+HwhILwXpN4Zt9wdt2pF\\\\\\\/LUaH\\\\\\\/s4+ybjPJYhdlXVxB1t5mQbegHLPJ0A2WY+FMW7BMlRW6eGAKarSJhZdnFKBOY6SVyObHo9TasAlWQtZxnsv5zig+s065EWZiLhbD6\\\\\\\/MkPiO71xXfsAKWqnilFru1EYGqk+iHNsEXPrkWIJO\\\\\\\/OHFWwvo\\\\\\\/Gx+lZmOTUAlGnNu8q1QfiIvBDT2Qlo1Ny5xG1c\\\\\\\/B1INM2WzGTTkqtv43F1AAmQcHZ1NnU+hljHmyZmgw3Sn6r78gPO1V6+PWjtt+HzQgsQQrutimZYP+PpBpiG7hkGDa0TZPxXx6cXEElrZbSYc+3FYfZdl7ALzUOtCe4Oz9Q+FHHP0CqE\\\\\\\/PHvncYuXf75xu8zENZQTF6BE\\\\\\\/lY5XowAIlzPldNPOtitXTYplf2gkRo\\\\\\\/ufkg2ameMV6dIv6dGpfnuy4fjR6V2fL68tmAPJ7nahDJPCoys1iqEuZGgjvTgRRACjb9GAzs\\\\\\\/LcXwUbl69e7MYUy7f\\\\\\\/6XO5BGNM2NJO9g5DgLwkc1myTPwWGbzQAg9El+EoQc\\\\\\\/eMcPQJ\\\\\\\/EUq01uyFUTgckngZ8Ev0fMreEaZxHj\\\\\\\/Ql0McUPt0VL4GLltD9DWBBIoVEKayB+8+fq3AXFsTYyyFQk0LTBfZwxRGELolKKlS33Ku4T\\\\\\\/LTX2SgCmv1iy62sZ2L1BKbLydKySCRq7BLHw+wsRPIyPTfIWqdfq0VQCk4iZb2MFzY39H0009FyqJKNkNXo6VDibyH7cyXsW4OG9zc1HbiZ8pKJScz9w01qYqP2EAPpwsqCKNZFv6zkgbDqAXypTj\\\\\\\/YmoA+VfUIshQBunB7ogCssdidOOsU7m+2hL+m9bkpInj6oIxhjAnjkhGCR4pqelOYmcXet4dlg8D\\\\\\\/KyYMUqSeJB2tll3lb2c2pE9Oqzjev02liBQSv3p7cs6Jj0RHDvuxTahtumFnZcXNLyk\\\\\\\/FeqmdlG2Hbl6pxJ78SGUHv5Q8SpI5FTtdwCLgXYn9EV5mi+0tmzgxurlfNqrE60lPh1x2Kd4Ea2ONXRR+PVhVKjIXH+\\\\\\\/wtQL0OLDea3Ko9uJHnIuPLtoI7R6nLSp2zz2hKSvnAV8EV8LA+NWbTaCmLpCR3AHl36cVY6wuavOlHMpdIYnqcErcgXDdd1SvQGyfNgfMJ4Cxr1XYzWimD\\\\\\\/\\\\\\\/FprAPGGroa8Jo63yzykyMJ0SNCtvWRD8Z40wSppA5NIvt77WLrqxiISdAbDJbW2F3SbMfFW7tSlM3xxSHov14tykCFz6CNRnqmnVUUFVvi9UF9WnQ2WMaBO1wU4ebWwGwR+RfUaW1e5p6t0L\\\\\\\/BvoNAv\\\\\\\/Zb7A6p7\\\\\\\/87wkLM2myiWu8Gt\\\\\\\/BxKodJzR+LqsB+gIF4BbvafkQ8ubgRox8iBIBHRLZYS5t8piAftZm+x4vi\\\\\\\/gBC15wGsRiTcSrY2G\\\\\\\/c2rsJxgYYnyAiD6X52kYxUlDS4Cih0AvmiqfHzpXKfyKdEbkmBZh40V4a65\\\\\\\/2ySMB\\\\\\\/5E2j7Z9uYdL8oShDIvxST9f9T0MJSqb+mV6MbiBPzkje6gnX++FDvZiMxesh2i6Jx5lmKQcZh6oY3abuqpGDgSJ29ZX+HcORHjrt6qflFfhhb3tqVTyL+VXMFxSrM553g3S4Mgs5JVGWSzOU2WYg2gheyr4D\\\\\\\/X39msezv86Ny4RARAZRlEQC1GSKWJkik3n8Hu\\\\\\\/6cqLZHs8a4qON19CPWkEtgWE2nSga9XISluB7iJacQTe\\\\\\\/7ZgEGjE8B4lGh1IzJjLZ4eJLzC2g+jYoqw7FX+K+NhM2TGPp0FN\\\\\\\/xgyeJGmN5Vq1tHiVWYWpAwAkp2xaGTEd0\\\\\\\/6umusElWwjWhizsWGz76LPLYHyForN88ROH3Q9CGF7ywGdQf1CV83hrJlzLh8NljDDZ\\\\\\\/XdlehS0ChtaLbjWI56wNCgyCJpTpj37A7TI5pZcP892GgLhqXz7Nq\\\\\\\/IhH11295JEtfl1r8ECVkrettB9I1TY1b7IyK4WFf1Pi2RajYRY3W2O+lqGEzc7ReqkSI4lLElhx9YBFYlsz\\\\\\\/dmSteqJwR+GgZ8T\\\\\\\/mCPL8ssKlgXmpJhpQWl0VsMZH4J66M8TR\\\\\\\/u4dHFYfrga8vPZqZclPctHGUywnLq1FyjFTVwGQ956eHj46UwIxvwVGoj\\\\\\\/\\\\\\\/p11yoYiIpgxOVLaNL+B0GGYS\\\\\\\/TrYuL7ohHJZ8bl9VDWZSSuV7RFxSxXzPmfth8uH\\\\\\\/\\\\\\\/lT8CZPr++3va8iRKrwmiP8rXQskjpdpCbvpGRohXZgEt6DmJXxYujWL8GnUg5Q9QhORrlhVQoD6eNjdyhEtmhMrQCfUPdapoR3DxGcWkTjDaR1hodSnxdtnkp97gr5uXjTpMUNYwwpTCMChyk9i6bjCgyd2NnztoPnrjTfYIK2ZsInPQMQ4sYYyb3Pl1k3JkQKBhM7H3zJh7eewGYwmgKnZzkfWs23WQzidyU1\\\\\\\/\\\\\\\/zabq1XDLDOWdBWtLBTD7jSAXInz3HzCgk\\\\\\\/I2adsLVndf0zZFL6S8JiQfgxAfyOviArwbHkcGBcmy5aw\\\\\\\/QmPrW1nMCjDc0WBtWvfAdht4ajKUurk8Gr3smV9fq0mXRv+44BVHxsmhCGzjz2LgUQkUcUUxpnz1ceFatK\\\\\\\/WyUAcDSbcgOhGK1w7cg4qDtuEe8UV2iaIlD6aH0wtbHaTpGGZjf4c4cg9ZTZsAOl+cVYq7\\\\\\\/GrksORnnKGjK2TGK4cEF85pAiRCM3jlX6k16t74dJNY9NipKdoEgRxCfNDnTSTGRFJ5s7Ch9WC9VW7vjN73ww51t\\\\\\\/RxM\\\\\\\/FjRiAbbZO3+kO5XWucH6ecxw3gDDgnsdiwR9VLHdc3ub2LrSaNbSYVlq6V4B4U5tfPaPiNVsVpkw63h1oqr6CptgqI6\\\\\\\/WcGv1s2RyvDd0idSnuk2bdVJU8JAyPddAhnzY6CCskNb\\\\\\\/84VXn49m8SoQwvgtyY1\\\\\\\/zHjFbE2SN9L30eVdFGHeVYxtNbTwRfEycQw5hM62SICWZPCkL5FyrJzA53H6DYa6ZeqUb9NHlFjHgkafFPYWaE3OVLn2bN5thsFnoAuXHWjJM7lPIVLImpFkfnNpk8QtQ7p6dMJEba\\\\\\\/kxzbxLGFCtXju8Ho3lZxJJzmGbYVHJxKRrmfrMoYyM3cpUG4mB4CR5SexZL6ldOVFJrf2JI+qbRSbxBzRyE5RrdsPRpgdG0+7X3n0h0zpMqDqFWT++kfepRx1dVxoAZGDaph4aj8Yg2+F3aALUV1J5sApzpskdxSVW0xbJ2reqdUe\\\\\\\/UQyTp+GskkxgnAg8chivibiWy7UsCKnFvX1i83FyRHzmXtNz7Fq3exMtyA3MefhNARjfZBPl7I0IH6ObL2a\\\\\\\/VsqAi7CfYXCvOaMS4a47yk2+otNpWKbX5SjkkWNt1SGkItTJWDh\\\\\\\/C3UJ8QRMb0OZEjB0VWOqOWHYhYMU\\\\\\\/zpY+hT\\\\\\\/NJyf7RtLEajbDeRlAF4hO+WupOlR3UGbImnJNQWDzUEfJm2CgSjeqAUCE4NFGl3t6iNpmCBp9hzzT21At8PH8k\\\\\\\/RRTu29+51pB72BbEObYnGQRTxYbYoRSdHMW87zqKiQC27KgIMbKrxNt5boQklmhMW1RyLBPwz6am\\\\\\\/FpgdbuVps65rT0oRr8gcwhmk4dzAXoinY3Ds2fkrYJEO1WAOfq\\\\\\\/J1+uxy29GHxtWZqIIASkwNE44\\\\\\\/y5oS0AdILehBGIMkGpI\\\\\\\/bSroFiT6Ez0rueAWQ7A7IqvRGrWVHFgwEVUzKFWj1Ri87C5rD+lygW4VoEIBzNCGhVVuGyFPRsa9V78ARx0uolShxjZ15Qs7Q25VJqWncw7GEjkedbywZeg0e0cpET\\\\\\\/7ZE2Tf3GryM19Sg3vu02fGLkCOpSNI1r3LA8Ye06jvKe3+mpLtvMlCuQz5GB9TkGckIJid\\\\\\\/CNTNtLv+cftAnpwch+5qezOJASPhGrlhkSMksHIG0y7G9CmSurz\\\\\\\/MUIxNBOQJKpFKIlpgKAyIib1GgtrUxcYQ9f4K1dq\\\\\\\/x2ueP1gG2TnuQdkgFei2C+Hhl+80TtOBA1gxt5BJ7pIkhAq5\\\\\\\/9nR4ApXC1OsH1H5niLmVdO1VDpSf+AXSGDop3QBkzOblhjMs2j5fW7GA4tV3OxhuB1NSiBsDiqZZ3D0HSgPz+09GvRwXyJgvOnA8iCtWS\\\\\\\/loWGZ8CopqvgOkz6OIEE6RrcyseyhMTeUOKnWp4rJes9Jw6yTUUNF6PXq66jul\\\\\\\/PvzdVcaYDhyTuPd97T3I6uGcW53kt1i2cjIGUtb4uBSLf2HOQNlkcQ\\\\\\\/11NSI7kzQuZPVnbU3a4KGPAROqxy+d6owqzRvSEBYyzo23gsEmupFz0X9aOrDtSgN\\\\\\\/PSv2NlajVIFQIrULQbeqJ\\\\\\\/\\\\\\\/29OZ2hYD3h4dTTWJpU2d6s6G+35to0tEOAY8skRAZuAiYGakP+H89US9QU\\\\\\\/bBszmkDu79auOpOs7797TIGyWhoytU\\\\\\\/S1mXfGBdbOgdQYTSEvAdTdz3einm8gTwTxM\\\\\\\/cALkc+CvuwCNj9QViKKC3db+\\\\\\\/KA2eYoeAPntJCklmrJhF1cOtGbIpswxoo+HeMkg7\\\\\\\/OaZrTRGGtjEi9vnH5VCMPY7+dhhasPB1A3wkbUeeUA+PBKMqm0\\\\\\\/K80DDS\\\\\\\/tk\\\\\\\/5l8qdIWnkk6uggJM1ycuoNfLUVvOQbWlHiDDIWckbP5jOug8qZE6FoY\\\\\\\/AFcYnmOaDvf\\\\\\\/u0eQUeTFHtb68IcWK3V6n9Zmlkfje7oYf6ktR2Kohn9OA7mdhAr9WA0v5NHDwM7P1kob3DAeS+SI79\\\\\\\/cAN8ePpIrUgN+3G5kV3Ig5PFcBUuQuG5X0e6DbFbaEIS1YBMMVbvoHqdq8oAB5mu4nhINZAVtS1h5FQ2T4\\\\\\\/U0SdtnFiH2HcjA7MEXJgm2Bw9KimgYH8HKwK2x0PKaeZ+YmI\\\\\\\/esuxdq\\\\\\\/EheMpZjdIAGkvz+w0I\\\\\\\/j3vDSmLwiUx3suawJGX8bWIyOo7AALjoHj8agaGPCqIKb1GG6\\\\\\\/pMEm9uJJQ8wn1\\\\\\\/sRKhIaDo\\\\\\\/ieW6z9AV69IaKszrbAbu015Qr4Gn6fLh4fWfc0KDV6LoMurF50qEmgEN8UTRaC++Df9ZpZZvvHF9MYOEm5LVAR30UF+4EdqEEcv1XXPBwryxkCXxJppglSZp1yZZ6UAO913MQjNXGcO2PSJ58JFRammMol2WHXQamQPWV6W2ydrKPgfOvObvQDeI6oJfkVo4kzRHGVkhHZ3QUl3\\\\\\\/OyuracFWBulAaYnjjLgHJJW6mOETjl3EKDKxZN26vBreJPLyaPKgovbQsIKks29BP8T2Rh8iZr3ZWknXVL5+q5gGtJLpOh8p5VjVPp5r+jZiBHUwbkqlLiYVK\\\\\\\/Zv7\\\\\\\/DFbfxckp5VlBNriiKjh6\\\\\\\/iEZJpDjSNQFips5vOlGKzMIYK0LBun6KWph9TdABdJxumZp9tsWhrD1i5RfMm7Rvkd73Uu+2PbjXKdt5Pd+07juJBZhtoRYAON\\\\\\\/+3x6yGXEgEvbOmfyuzJbDSaiDCz3bSEgZ1YgXuNTmjMKIxSoqg16vBfYrKQPYqdcPyCvw8O7UoeJL+4wiuRTXFHn+SDBxSPW2BgRYO18IlVlyX7o7EV6be4mtWqyP2cW5dIMnpbyKrNv2Ogr8tQzNZ\\\\\\\/d2n2mAg9KO4tuBhbbmHMsW\\\\\\\/4Mj\\\\\\\/ucx6TX5d1aEPCYNJC4wnRdDzbbCSnYUOrxBBBqLc+4FBc6QbWL1m7LURTaSluguuOWsP3j4vzhzODkddchqwDnme6sYg5ye6gGveQ+YzZf7uq\\\\\\\/osxHWvaSR0jJKCQ\\\\\\\/99BMShrhxMCnBYZQ7lgTrKSP7mWojjVdEH2fHzGdyZTu1\\\\\\\/R9VGlH+\\\\\\\/\\\\\\\/T8LRBJrPDheL6a0TMNw\\\\\\\/ghS1aF4K4BHJ2T7Ey1mUMr8TiUxBMisaPE7YM+QhGHIIwZ4RD4zaf+t7SfzrExx4gx+l8rLJnBz7HE\\\\\\\/T0T6RbelgNuV8o9ULwWccmnLhX5pZPO01NronQY6tBydEGZXEsp11QTXeW+QXA3FWB8ruKpxgdlG+g1epkBk0EiE7zVVDdPPqk+7YMzwrv7Qne22HBjmPARbLxHmkQFFJkryvoE8iHb2OdvgqtjK7fmXPfw0fPN5CSY4Jl4SMLgWqLcjm8wLZ6Y6RivgmZtGHSfV0QOKXlyTKZI+PiB\\\\\\\/hYr+3ztPMtZghvUIymvCKo9MmQ6YQ2MtgYsemVhKtk+fKUp0Oc5RemTpOBY9wZf48TQGaeiCqwj5xhbE5IxlK9MP0SGX\\\\\\\/4DP+su2kb0fzDQkqOcq3EHxfDKwzs64nTFrM3O+gNlqGEH7c7+u1\\\\\\\/ASTaZ8swtsmZhvUb7UK\\\\\\\/jSPs93JBxxOa5qf6hAiSiXj9HE8irH4Q8sp0+Xj2+0mk8WGq9BOnfie+eks917Iab+eda4KVVV4xI9a+iy07jOEPP0LJwwYSiHMk0MLZ4+DzP4BIWv1Cjj79SzVciW3UZu8sW7J9Yfc0GI\\\\\\\/4VTJacGQMVAdW899+P9Kuw4ud22Vam5hj5LOgba5gth8nPsA6+wuvPG\\\\\\\/WQG08fnGEzLAWZpH014mYr9eBT7I9CGE3JAE9Opt9\\\\\\\/55fvKYi1c5ax2FGv2xMQHANNo+uejHQXR2e\\\\\\\/jQSYKoyliDKU01T3ALBu7L56aooj+s5U7CNvquSaqNHacBGBZWUtV4lQZTRIQlkyvizyyqeHdRtXw8kWAVWSitExr779G3Une44+FA7FqR5zTtj+9AWGsfXYHBhRwT3TvptgLurKksIpUb8qpm4bSBAEP\\\\\\\/oZ9AW0ta+taD8qu5HAwp4kX07uK5aAlA+oCvFSsRq2HBxbkkNN5yL\\\\\\\/I7xnOoDH1sGx5S\\\\\\\/Prd6\\\\\\\/f9dC7\\\\\\\/0K3tUFf6VxcRdKhXmG1jI9kKhuzi9FQbQo3SYyWyNpWxxMZcedvKoYMjFQw2aIqBHe+HXRjaq3wzCpaaDaOmp2\\\\\\\/5FHU9byEGvz3VJ+H+CEL2bZ1Flzd3SUW055LylDCy7bYDjJIihkYG98WrieQqjTujKn37DnU54ZivagCMUJQERW5giLD9ZDLOg3fq5378Km4h3GrEtshgADbzWS1fP1SO4wSZ91oHUSkInQIMlth1iCPA0GFa0wY4EZ\\\\\\\/WSWjVmPFKLzvm0p3Y0OC8HOXdBTJEG7D\\\\\\\/o3t6ttc0So0KpaLei33OfUWe6z+XW49qmbzPUCM2Vt\\\\\\\/ipxrpQkGnA\\\\\\\/OpUULT9virioTBtewVNCiqEWlikMmo9jpav\\\\\\\/tyE7pf2tlLS0xve6TzAjwplN0ZB\\\\\\\/6QxKpVVvAyrOCROZgem9Nmqc3+wc1qEOqr2t2A5gMnSS1zm6SR6X00JcVqIs1t2mJXiyK5jrqmm2xVcWTObsrqiIEux+cJ3c3gkoPF9hEg7gvYav5vSbu5UqSeZjd6Mhf33TDc38BnexVJKscaPFvTqcg4tqbO7CjBwyEDOQVT+9Txg2mQEDtzUwa77VkWEcNJoUTEVVGlIkipnFH\\\\\\\/zVy85fqQqU9NZuWPfZlKZVnzWt9nJeXB15Anv6NQOnKTvrF6ILv\\\\\\\/YH7cvJYeMssciQ+61K4lqJqyixgEARXmr\\\\\\\/Y2xWyVZqn3wCfJSc8Cpbf3PK2mIDMoL4zPkyQNbPcOo5bM6QQgOCSPBV5242n448vAJWqLF5u0Xq3+onMShDF576RKehOIdna42+fHlv1ayhGfgj9CENr+mjrMor1byGWSDw5ke3lM5xTaq8G3fu4W0RonYmITi\\\\\\\/BWBMSnkforfQxePBo2CZNaByWgedwJSlDHKM6peONTp0m8JgToZCKLPNCV0i8oY7NeleLpXeGXdxe38HqTSSPQtkbdEzj2Ab0299grGHKYxkEm1HuJoptsGnJD2fZDA\\\\\\\/QTOEPo\\\\\\\/+D\\\\\\\/TOn2SScpZpITE2yW9i4r9EjbEMMyABSPIXp7a98XpdW9I1XTvOhInfJrMwiv+0962vYNg7L9IOhtYXFzNAnyecPc2XtCeA3WwXcGdhPk9uT1IIkaOOuFa+lxzzCMIdCr\\\\\\\/HhJpAWLDqJ8500LCAeNsN8AoxBEW780VzoSMmwghUQjmbHfo6NepQkOHinIhHRzjdSpQg+erjN2skXWsTYsfkD8h5UzdN1LyyOVa+0Q+3DrwtCYvier2ZtdcdOS0S3rO\\\\\\\/sbWCW+2II2CZQLMPt+BR2+A0l9VWYnCOLfclpfGgT12nQhy0G\\\\\\\/gVA\\\\\\\/BbphZb88mtBPB5nfKRV11tuKkc6JOnd6gQ7\\\\\\\/0smNTsmuqogokP2QVgExNdsnKvSVdeottTherq9fZ\\\\\\\/vB9h22YP+n18b22ovCBL2IKRTD9ST2tlmZuwvyx6896hfhFtXa2Qfue044IKJ98mBHMrMF+NvfuXFBKiqTGn9k+IOt0+FAe8cibL9ktzYtvoslM2qeQoRm2X6rmv2fIsEMIR4KLx9SuxhOE3KABHFbjIH9xVMt\\\\\\\/EsZunuMxvJlPVe4NzU\\\\\\\/wCctYOQC0c8yVPe\\\\\\\/Y6KaiiAPwvKf4prizxCNPwPtkE+\\\\\\\/s9HNN1mDYx7R0FzdMYHzr8h\\\\\\\/ekrycZ+n5vJyIwd+lSoI\\\\\\\/qgUskHIp+leD5PV6UagbcgLU5mldISxyYD\\\\\\\/Tj7ijBj7BmQcbpE0Pa\\\\\\\/9tcghR+AVhBF5n+X3XQtvXMqeBA7FSk+ySALXen1HtGRj4EciISNNYpgZbTQMa\\\\\\\/1nUUv9zk7L+5y0XCcYYLu0ii+7MdZ3J1DTJKqY0fOpCS6KnvYX4boIcR3bR4VzC9gpq7OFxQcdpwrOGV0DrDq\\\\\\\/wkJPrz+c7eGF+wKFPYn70XW+BtOHh9oOMxk\\\\\\\/abfrUam6+xALxHmBLpstzBswEluyZ9Hsoldr\\\\\\\/nFal2kv0fTOnwBdwsue\\\\\\\/0YFGi35xyuhlw1imvUKfW4fSWuwnJc4LQrhyieYPPTl4gCR3qz2Ha9BwdDN0jyOudaqngcExr00lLorwSCrrjp0AeQIMi98d0Rs8vx2NIv9LefMdSIAgdpMFFiMepsxkZMQhv+EbVJZV3ubUD6PAhvMPtnoFYlxj0rtOrv\\\\\\\/8GIUF\\\\\\\/K50sn5K++zo9ZV5xHKyMIzvE5f+HrfZorv5YILTrruWYHCbwKdBeZrc3UiTxZD5f\\\\\\\/0QKQ3YeRNu0wZMT+ydT83jnReS0om+\\\\\\\/PEDnuilFdnnKspCN1WjlKcrRFsgJ\\\\\\\/f+u+\\\\\\\/QScn5jvR963B+9+snC52pnmJmtMH4ERLXyR+vci2YYXUSStM20bMX4PxJ0\\\\\\\/QdnEAa9UyN0WB8sZICqwYYc7KyxitYlOBefavcyhKob95j\\\\\\\/jMGoKGbDmiq9Lb9b6UZkjd6fba6toF3ePd0st33EGpwhHJTTVKl+ksP8tahDb2syoyOktj0ni6GwL4vFc2wIPdb68HHz7I4AqJwH06nzSwN2numVMHrkW3sa7UmFRKVqn2MY3N0D1TBmM+sMIwzRkmwiQcMYqMwAzZNHLIFRBw8FjeJKED\\\\\\\/7xaDC05+MGuTwjOYvFHpvPgvkDHpa+F6W5o\\\\\\\/mccWcKFgM1ANkCiP6R8b9oDsnffr+Gos6+gC9ZKYQxtQ3d6QmnEIgciOcmAJGx0gcRmaETO4sjSOqunyFER+37mnru4nyg1OsMexYV2Jvinj9XM54hxc\\\\\\\/KiCtw6t3qw4rXhOzHXBYLqwXj0cUVuHyrZtRKSmC2AHjrvoC\\\\\\\/ueWWEVdNCN\\\\\\\/820SURqSRk7Nzg4ABQGd4FOD4kWacSddWJ75mb0FWgyWtNFfxpBQbnVs9P0DANq4jktPIhnHnl9lebfubjv8zbnjJeCj0b19NUHbSaEv3mUf4QaSPnmQSbrf5zHUvUyPWf5JsN1cDV4qKXPhtcWrkjd58+EwN\\\\\\\/Zqji\\\\\\\/CmYdwVio2NVbtX\\\\\\\/uERJjAAKuvgCAq+VcG8HwtcY0jfWKSz4PkOTvixBssqCr1dAuO\\\\\\\/dCRrLvpd5Pv9Kat\\\\\\\/WxsHIbuTPoygRRW4dYSBoWRb4pc0WqBtPFF97aJPIvtH6BeHAm0M6yIbGpXGOUSSoO1MzZqyWpTS97ztNn\\\\\\\/QjinHjJYQUSsRwvC3aDqRl8RqPHSdcN6qMEQyV+nvPg586h6B35AaX9TTMaoaBJAp6I2mDpKR\\\\\\\/z2JHBr\\\\\\\/kyFeA1sx\\\\\\\/otjwiE7AY3JyAeQLuyXszGzZRdcXDtSquJBO7PU\\\\\\\/5ATW90++rYJqugesWC\\\\\\\/nbusSgC6O7DHZvYCo2tmlVeXJs6NIHwVW9bN7anNHi1NdjgHvObCYx8La0tR2z18F2304rxUG72dX6oNoErU\\\\\\\/Xqngt1U1Olj7IltDHSw5+q2SjfjspoFPgCQk7INyapIJ\\\\\\\/iBi+Dk6UEpuDi9DC9IFCYOHJSmpm118ZrwmZMOJA2\\\\\\\/StVXDPmmGW0pKF+DDaR3y9G\\\\\\\/VAdyDXSe\\\\\\\/OXIz7kfK8FJ9zHDqHeU43ohtxUx2aVaeMIaJLsDKzUqBL1Ej+IXOLuW+6Y7m2v0jgBcWJziAnwjT6\\\\\\\/5NDZ1826dxTSLJ5RnO7ADODUI2L6tq3ceVuaiz5k33ETf4PlEsLTKymHnA5U7NRa+irpkLABW5y1mE4gFNKalzyvBB3\\\\\\\/GtzjUHi7lnwlEv1PQLR2VK6390s6GXsZaxNGV41kBg0Mo+6C7OkuHSJRuH5QhXRo\\\\\\\/DTQRggGZF9d\\\\\\\/NlwRIBzatzfYlliJ18NAMWlsG+jHdcVGrTlPtDe82PoAuuluWmr5ni4LYI52Qb0RJKB50l1AGc000\\\\\\\/sWrnf0WGkNGWKzkOXstJCFPdjuXec+eTOe5rowxsaSluwov9Ek2N96ZBRBoJQpbbacqY1b6nJ0M7Xkb1lDfBeePVU42EbTGtTiu38h3k7w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rKb\\\\\\\/EQNMKuUFUuXgvcyI4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022060b07dc3221346fd21151420e1f92f3548b3e30e185bb25ce9c061c312f7a4c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VDTNY2iahmr1o33dZIejcy8Hd69CLzijrC1phljpv\\\\\\\/BGdnzQmCZNxj6pchypXjCJQAB1ZhpHKXaVWZePxm\\\\\\\/Ok3ctPhe+i28Qq0ju\\\\\\\/JJtLEVKj2i6i\\\\\\\/dFBzNkB0APGmav8uOdbZq4Xx5nUlKL5F\\\\\\\/UC1vus1Gr0L2gXAGtdPzibGYzihDok3Ncg5OTF1t9qoAfFziqwQCcAlyOpFQ1VIcPlg+oGQc2Nn3DyzHH1ZgeReRQu69694xkwjcB27uBTTrC5E6mDj9LrhK89Xbf\\\\\\\/2hqfPikpKnF125PSi5YHP8WblwIvv72swID4uIwONFIonlRSQTMJP7tEQqEhwyKzZ+7kECPcee5aAwvyvIQYV\\\\\\\/2nl8k42pmXC5SrU6JbBj4r2U\\\\\\\/MItA0DKpiVZlVQ2ARGY1WU4BBbiuEjZEMJS6+Se4XE2OIfEsjlI\\\\\\\/qLCeIu+U8l6uzTCMgvR7l1CMYOLdTxni05\\\\\\\/x7AIqS00iXMynI2GrHjbPtPZ8m1CmnHDMTtcTHAZTyaARpI8b6eoos2cFTeNVX8YkJdwJ4eH5wIMSHJNWz6i8Xh\\\\\\\/vElukPEGZDIBlaMUPm9ZZZ4vMAQoZYjwCynqkawKSSbASXur6FjelJ7vQvM\\\\\\\/yc0X8mrZgtuCj2FZNwSj8es2z5eH0kWieM4zY2vqgEm6Kk+PtUD69kHWmcjLCP1ZorWSK\\\\\\\/+kOhVPUZP5dmKTVSS1Cc0r2OWvmlcHg1DraOA\\\\\\\/tARKoJxi+pxzN+rW\\\\\\\/MXIpEboF2dGPDcoHrnLC2SuRd2zr9euLtbnT5kXLWG7tEn3EXcBrOGDNdVtkDy7ayEJ\\\\\\\/3Eojbz+yTyLQG8M8MJSn\\\\\\\/140duf1wLfKrZ5SB7zue1XEE+6zf4GinLlhWEO\\\\\\\/VH3G22apLGwgGXDSLPaTYXqghBwTfvBCipYPCPEfHTjoasyLqXRICnlTIQ9db7K0m+wMVRtsFbgMt\\\\\\\/1sdHmlNbaae5lUAV2rsCRp8lUjk5desv5BP+wAjOP4F3OzP8YuNoYxD203JMw5mOZwVcFujIlTws4F18XNZu9sYoy1YJd172DHKeXQ1Ycbd+KT5ARmOp2mjxCgPzXylvG7L3Y+CRt2idj\\\\\\\/KwEb40SqbigLzQcmhZfnWJapTiv2aqb315ZGOVda2sB0+noglcHZKDJOaBEWUQB5mnd9CEvexf6ctS\\\\\\\/\\\\\\\/dWzmHXmREPn9lVqujRCq4HoK\\\\\\\/8meQwRR0swuydbgtmyvvgA9CgjgQFbp4+uVzSs3nvXKxLstj0PFvhx4+GvlxGXDdUhbTPLsQg06rnOFK8IBu1CAu+NCHmeRspZd6P1zV8OKegppoTC4S7PEtVpfNlIBKmwunliWzCMVNnMgB7oGcso1aOtq+hvd9m+dNFrbHw10MZqqbvrq4PczzS1wzYJQ6r9BMYz0CZhJzP9H8DAvd7M8PEhg1Wbcb8gF0nnqPPDJUpDUX2gpjVTEHvlVjfwF8CkJt6dK0SJ4+H1t3p0YWXlglFuWDhAsM0xuwpHeg5PEYXhwrsKpTKxFa1DrBy8cng9R8Ok9iqV9vC2BDQgKPBkLtMVANNWo33fIJgGanCUuqCPIZFHneW0Dn44tz4BdXF3uKifz4uH3FoXK9EOQbzNTX+UZdCmEQP8t4j6DkEw0wNWMdr0SCBffsF+FmvBZZmYQeLO8TI5m7bbamLHIXuKDQ64nZ0Y8NdM83xEQGa\\\\\\\/lGZZu+jvF7FRl6eopOSTP8F0+be7cjgNPvmACeDKL+FvaAi\\\\\\\/NyVeCS9bUH7+AS8xq6AVm5Q4P\\\\\\\/1N2O14PrL2AUKxP9MAT3oM2cZMyQziBaDcOQBMLiA3sDIlCSebbfuAsaNHILJa2F0NpCodLXXUEPGlgNeTQmgdwAEeo\\\\\\\/jw2goPptfBjRIaq1vuMF1IxuFndejFnk2l2sLJ5r41WH3tAHcsSSoD64ziqkyTzWiU07xcRgHj7LPCG\\\\\\\/IFGVSA92FUgwTgFENkNjYKgnt664U+rQ44U+MB2Ffbg8\\\\\\\/UBzwl\\\\\\\/DqGXTOtQ786qFbZl4+EALQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lDUO+myE406gXtcqNfsQ6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cdca0f1c4528235c306a98dcaa5cd83bb70706000e93fd3f1d53e9b875d7c89c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490462 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2735" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MA8djHQEZbt-eE2p_0H3vOvSSznCxQ7__EfDOeDSq4c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2735" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bZlGlJ21JvfKUZNr2ZJxdVRG87OJ6_NW2UMZBjslodA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2723" title="Fission Yeast CENP-C (Cnp3) Plays a Role in Restricting the Site of CENP-A Accumulation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2749" title="ovoD Co-selection: A Method for Enriching CRISPR/Cas9-Edited Alleles in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490462" id="node490462" data-pisa="ggg;8/8/2735" data-pisa-master="ggg;g3.118.200415" data-apath="/ggg/8/8/2735.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0988-9710" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vikas Belamkar</span>, <span class="highwire-citation-author" data-delta="1">Mary J. Guttieri</span>, <span class="highwire-citation-author" data-delta="2">Waseem Hussain</span>, <span class="highwire-citation-author" data-delta="3">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="4">Ibrahim El-basyoni</span>, <span class="highwire-citation-author" data-delta="5">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="6">Aaron J. Lorenz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9109-6954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>P. Stephen Baenziger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2735-2747;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200415 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490462" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vikas Belamkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikas%2BBelamkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belamkar%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikas%2BBelamkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0988-9710" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikas Belamkar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mary J. Guttieri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA, Agricultural Research Service, Center for Grain and Animal Health Research, Hard Winter Wheat Genetics Research Unit, Manhattan, KS 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BJ.%2BGuttieri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guttieri%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BJ.%2BGuttieri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Waseem Hussain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Waseem%2BHussain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hussain%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWaseem%2BHussain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ibrahim El-basyoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Crop Science Department, Faculty of Agriculture, Damanhour University, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ibrahim%2BEl-basyoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-basyoni%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIbrahim%2BEl-basyoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">P. Stephen Baenziger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BStephen%2BBaenziger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baenziger%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP.%2BStephen%2BBaenziger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9109-6954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for P. Stephen Baenziger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pbaenziger1@unl.edu" class="" data-icon-position="" data-hide-link-title="0">pbaenziger1@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2735" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490462" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2735.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490462" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2735.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490462" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2735.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490462" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2735.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490462" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic prediction (GP) is now routinely performed in crop plants to predict unobserved phenotypes. The use of predicted phenotypes to make selections is an active area of research. Here, we evaluate GP for predicting grain yield and compare genomic and phenotypic selection by tracking lines advanced. We examined four independent nurseries of F<sub>3:6</sub> and F<sub>3:7</sub> lines trialed at 6 to 10 locations each year. Yield was analyzed using mixed models that accounted for experimental design and spatial variations. Genotype-by-sequencing provided nearly 27,000 high-quality SNPs. Average genomic predictive ability, estimated for each year by randomly masking lines as missing in steps of 10% from 10 to 90%, and using the remaining lines from the same year as well as lines from other years in a training set, ranged from 0.23 to 0.55. The predictive ability estimated for a new year using the other years ranged from 0.17 to 0.28. Further, we tracked lines advanced based on phenotype from each of the four F<sub>3:6</sub> nurseries. Lines with both above average genomic estimated breeding value (GEBV) and phenotypic value (BLUP) were retained for more years compared to lines with either above average GEBV or BLUP alone. The number of lines selected for advancement was substantially greater when predictions were made with 50% of the lines from the testing year added to the training set. Hence, evaluation of only 50% of the lines yearly seems possible. This study provides insights to assess and integrate genomic selection in breeding programs of autogamous crops.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/triticum-aestivum" class="hw-term hw-article-keyword hw-article-keyword-triticum-aestivum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Triticum aestivum</em></a></li><li class="kwd"><a href="/keyword/spatial-variation" class="hw-term hw-article-keyword hw-article-keyword-spatial-variation" rel="nofollow">spatial variation</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/genomic-best-linear-unbiased-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-best-linear-unbiased-prediction" rel="nofollow">genomic best linear unbiased prediction</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li></ul><p id="p-3">Wheat (<em>Triticum aestivum</em>) is a cereal crop with the highest hectarage in the world. In contrast to maize (<em>Zea mays</em>) and soybean (<em>Glycine max</em>), wheat varieties developed by the public sector account for a majority (∼75–90%) of wheat acreage in the United States. Public sector breeding programs previously relied primarily on phenotypic data or a combination of phenotype and molecular markers associated with traits controlled by one or a few genes to make selections (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Bernardo 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Randhawa <em>et al.</em> 2013</a>). With the rapid decline in sequencing costs and the ease of generating genome-wide markers with genotyping methods such as genotyping-by-sequencing (GBS; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Poland and Rife 2012</a>), genomic selection (GS) has become a powerful tool to enhance selection for quantitative traits and consequently increase genetic gain over time (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Desta and Ortiz 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jarquín <em>et al.</em> 2017</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pérez-Rodríguez <em>et al.</em> 2017</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Poland 2015</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Poland and Rife 2012</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Spindel and McCouch 2016</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yu <em>et al.</em> 2016</a>). There is now increased interest in integrating GS into public sector breeding programs.</p><p id="p-4">Recent literature on genomic prediction (GP) has focused on estimating predictive ability (PA) of various traits using different GP models and cross-validation schemes. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arruda <em>et al.</em> (2016)</a> and <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kooke <em>et al.</em> (2016)</a> compared the genome-wide association studies and GP using cross-validations to determine the PA for <em>Fusarium</em> head blight resistance in wheat and various morphological traits in Arabidopsis (<em>Arabadopsis thaliana</em>). Similarly, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Velu <em>et al.</em> (2016)</a> investigated the PA through cross-validations for grain zinc and iron concentrations in spring wheat. These studies conclude that GS holds promise for improving the respective traits. The testing of various GP models to obtain increased PA is an active area of research. Linear models such as genomic best linear unbiased prediction (GBLUP) utilize all of the markers genotyped for construction of the genomic relationship matrix (GRM), whereas Bayesian variable selection models assume that a reduced number of markers explain the genetic variance of a trait. Nonlinear models such as reproducing kernel Hilbert space and neural networks are also used for GP. Genomic prediction models and their assumptions and features have been previously reviewed in other articles (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Desta and Ortiz 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Howard <em>et al.</em> 2014</a>).</p><p id="p-5">Although testing PA is critical information for GS, a large gap exists between findings in these studies and their application in breeding programs (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Bernardo 2016</a>). A limited number of articles have described the utilization of GS in real case scenarios such as for germplasm or cultivar development (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Asoro <em>et al.</em> 2013</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Bernardo 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Beyene <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Combs and Bernardo 2013</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rutkoski <em>et al.</em> 2015</a>). Tracking of lines advanced in a plant breeding program and comparing their observed and predicted phenotypes can provide insights to researchers for using GS in the breeding program. In this study, we test the performance of GP to predict grain yield and evaluate how GS can be used for making selections in preliminary yield trials, one of the critical stages in the University of Nebraska winter wheat breeding program (Figure S1 in File S1).</p><p id="p-6">The University of Nebraska winter wheat breeding program makes over 1,000 unique crosses annually and the progenies from these crosses are tested systematically over the subsequent 11 years to identify a cultivar for release (Figure S1 in File S1; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Baenziger <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Baenziger <em>et al.</em> 2001</a>). The F<sub>2</sub> and F<sub>3</sub> nurseries are grown in bulk populations and nearly 45,000 heads selected from the F<sub>3</sub> bulks are planted as headrows (F<sub>3:4</sub>). The selections made in F<sub>2</sub>, F<sub>3</sub>, and F<sub>3:4</sub> nurseries are primarily based on disease resistance, winterhardiness, and plant type. Experimental lines are evaluated for yield along with other traits such as agronomic performance and end-use quality starting from F<sub>3:5</sub> onwards. The observation yield trial (OYT; F<sub>3:5</sub> nursery) is grown in plots at one location for grain yield and in rows at a second location for winterhardiness and disease resistance due to the limited amount of seed. The preliminary yield trial (PYT; F<sub>3:6</sub> nursery), the first multi-location yield trial, is then grown in one or two replicates as augmented trials with replicated check cultivars. The advanced yield trial (AYT; F<sub>3:7</sub> nursery) and elite yield trials (EYTs, F<sub>3:8</sub> to F<sub>3:12</sub> nurseries) are grown in a replicated alpha lattice design in multiple locations, and lines are tested for more than one year. Subsequently, a few elite lines are advanced to regional nurseries such as Southern Regional Performance Nursery (SRPN), state cultivar testing, and foundation seed increase. The SRPN are replicated trials comprising 50 entries contributed by various breeding programs in the Great Plains and is grown at more than 30 locations, with sites in Texas, Oklahoma, Kansas, Colorado, Nebraska, and South Dakota. The SRPN trials are coordinated by the USDA-ARS at Lincoln, NE. Phenotypic selection for yield is most accurate in the AYT and EYTs due to the extensive replications and testing of the lines at multiple environments for multiple years. However, in the early generation (mainly F<sub>3:5</sub>) and PYT, phenotypic selection for yield is less effective either due to the absence of replication or testing done primarily in few environments, which do not represent the diverse range of environments possible when wheat is grown in Nebraska. The inaccuracy of phenotypic selection for yield in trials can be overcome or at least reduced by using GS, which has the potential to increase selection accuracy over phenotypic selection.</p><p id="p-7">In the present study, we focus on the PYT, from which lines are advanced to enter the resource demanding AYT and EYTs, grown with replication at multiple locations (Figure S1 in the File S1). Hence, making accurate selections in the PYT is important to advance lines that have the greatest expectation to subsequently perform well. In addition, selection of top-performing lines from the PYT allows for recycling of elite lines sooner to the subsequent breeding cycle as parents before testing them in the AYT and EYTs. Increasing selection accuracy and reducing the breeding cycle time will increase genetic gain over time (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bassi <em>et al.</em> 2016</a>). The specific objectives of this study were to: (1) determine how GP can predict grain yield of new lines (non-phenotyped) in the PYT and assist with selections; (2) compare GS and phenotypic selection for making advancement from PYT to AYT; and (3) compare genomic estimated breeding values (GEBVs) and phenotypic estimates of lines advanced from the PYT to the AYT and EYTs to determine the use of GS for making selections in the breeding program. It should be noted that although selections made from PYT (F<sub>3:6</sub>) onwards take into account agronomic performance, end-use quality, and disease resistance among other factors in the breeding program, yield is the most important trait considered for selections (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Baenziger <em>et al.</em> 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Baenziger <em>et al.</em> 2001</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-8">Four independent PYT nurseries (F<sub>3:6</sub>) of the University of Nebraska winter wheat breeding program grown in 2012-2015 were utilized for GP (File S2). Each year ∼270 lines are tested in the PYT nursery. Two hundred and eighty experimental lines were grown in 2012 and 2013 and 270 experimental lines in 2014 and 2015 thus a total of 1,100 unique lines were used in this study. There was no overlap between experimental lines tested across years. The experimental design was an augmented design with 10 incomplete blocks. Each incomplete block in 2012 and 2013 had 28 lines and two different check cultivars and in 2014 and 2015 there were 27 lines and three different check cultivars. Thus, 270 or 280 lines along with 20 or 30 check plots (each check cultivar was replicated 10 times) were grown in a single trial. Trials were grown as a single replicate, or sometimes two replicates, at 8 to 10 locations annually (File S2). Testing locations include Mead, Lincoln, Clay Center, North Platte, McCook, Grant, Sidney, and Alliance in Nebraska, and Hutchinson and Mount Hope in Kansas (File S2). Yield measured on the lines in the PYT nurseries (33 environments) was used for GS.</p><p id="p-9">We utilized four AYT nurseries (F<sub>3:7</sub>) grown in 2013-2016 to test the effectiveness of GS for making selections using GEBVs for yield from the PYT nurseries grown in 2012-2015. The AYT nursery was grown in an alpha lattice experimental design with 57 lines and three different check cultivars each year and had either two or three replicates at each location (File S2). The 57 lines in AYT are selected from PYT primarily based on yield, and also end-use quality and disease resistance, and represent ∼21% of top performing lines in the PYT. There was no overlap between the lines tested across years and a total of 228 unique lines were tested across four years. Each year the nursery was tested at six to nine locations with a total of 29 environments across four years. The locations were the same Nebraska locations as described earlier for the PYT nursery (File S2).</p><p id="p-10">Experimental lines in both the PYT and AYT nurseries were grown in plots of four rows of 3.0 m length and 0.30 m spacing between the rows. Each plot was planted at a seeding rate of 54 kg ha<sup>-1</sup>. Yield was measured from a combine harvest of all four rows of each plot.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-11">Grain yield at each location within a year was analyzed separately to account for spatial variation in the field and obtain the best estimates of the phenotype. Each PYT nursery was analyzed using mixed models that accounted for either experimental design features such as incomplete blocks, rows, and columns or spatial variation in the field (Table S1 in File S1; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gilmour <em>et al.</em> 1997</a>). Model performance was assessed using Akaike Information Criterion (AIC; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Piepho and Williams 2010</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wolfinger 1993</a>) with all terms except checks fit as random effects (Table S1 in File S1). Histogram, quantile-quantile plot, and residual plots were used to inspect normality of residuals, heteroscedasticity, and outliers in the dataset. If a mixed model accounting for spatial variation had better performance (smaller AIC values, and normal distribution and reduced heteroscedasticity of residuals) compared to models that accounted for just the experimental design features, we tested additional models that utilized the best spatial variation adjustment plus the experimental design features (Table S1 in File S1; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Gilmour <em>et al.</em> 1997</a>). Finally, the mixed model that performed best was used to generate best linear unbiased predictors (BLUPs) for each line at each location. Subsequently, the lines grown at different locations within a year were treated as replicates and BLUPs derived for each line at each location within a year were averaged into a single BLUP value for each line. This is a reasonable approach as all lines in PYT are tested in all locations and the dataset is well balanced. Besides this approach, there are many alternative ways to analyze multi-environment trials. For instance, models that account for covariance structure between environments, heterogeneity of residuals, and deregressing BLUPs following a one-step analysis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Cullis <em>et al.</em> 1998</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Garrick <em>et al.</em> 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Piepho <em>et al.</em> 2012</a>; Welham Sue <em>et al.</em> 2010).</p><p id="p-12">The AYT nursery was also analyzed separately at each location within a year and one linear model was used to generate the BLUP value for each line at each location. The model used to analyze yield data of the AYT nursery is as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y is the vector of phenotypes, μ is the mean, r is the replicate, b(<em>r</em>) is the incomplete block nested within a replicate, g is the line effect, and e is the error term. Replicate, block and lines were treated as random effects. Analyzing the PYT and AYT nursery separately at each location was necessary to test and account for the spatial variation at each location or inspect the quality of the yield data from each of the location before combining them into a single BLUP value for GP analysis.</p><p id="p-13">Broad-sense heritability (H) was estimated for the PYT nurseries at each of the locations grown in two replicates (File S2). The PYT nurseries that were not replicated, but had two to three different check cultivars grown in each of the augmented trials, separated the genetic and residual variances and were used to calculate the H. The variance components were generated using the mixed model that performed the best for each location. The H was also estimated at each of the locations for the AYT nurseries (File S2). The variance components were generated using the equation (1) and the H was estimated using the following equation:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-4.gif"/></noscript></span></span> are the variance of the lines and the residuals and <em>r</em> is the number of replicates within the location. Heritability was estimated across locations in both the PYT and the AYT nurseries (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">He <em>et al.</em> 2016</a>). Variance components were generated using the linear model:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where y is the vector of BLUPs estimated at each of the locations within a year, μ is the mean, L is the location effect, g is the line effect, and e is the error term. The location was set as a fixed effect because the locations were preselected in the breeding program and the line effect was set as a random effect. The H was estimated using the following equation:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-8.gif"/></noscript></span></span> are the variance components of the line and the residuals and L is the number of locations tested within a year (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">He <em>et al.</em> 2016</a>).</p><p id="p-14">All models were fit using ASreml v3.0 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gilmour <em>et al.</em> 2009</a>) in the R programming environment (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">R Core Team 2016</a>). The R script and an example dataset are provided (Files S3, S4).</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and genotyping-by-sequencing</h3><p id="p-15">Five seeds of each line in the PYT nursery were grown annually in the University of Nebraska-Lincoln (UNL) greenhouse and young leaves were bulked and harvested from 2-week old seedlings in 96-well plates. Two randomly chosen wells were kept blank in each plate for quality control purposes. The leaves were dried and desiccated by placing the plates in boxes filled with silica beads. DNA was extracted using BioSprint 96 DNA Plant Kit from QIAGEN per the manufacturer’s protocol. The genotyping-by-sequencing (GBS) was performed at the Wheat Genetics and Germplasm Improvement Laboratory (WGGIL) at Kansas State University. Samples were quantified using PicoGreen and GBS libraries constructed in 188-plex using the P384A adaptor set (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Poland <em>et al.</em> 2012b</a>). DNA was co-digested with two restriction enzymes, a rare-cutter <em>PstI</em> (CTGCAG), and a frequent-cutter <em>MspI</em> (CCGG), and barcoded adapters were ligated to individual samples. The GBS library was prepared by pooling samples from two 96 well plates and amplified in a thermocycler. Each library was then sequenced in a single flow cell lane of Illumina HiSeq. Detailed protocols and updates on the GBS protocol can be found on the WGGIL website (<a href="http://www.wheatgenetics.org">http://www.wheatgenetics.org</a>).</p></div><div id="sec-5" class="subsection"><h3>SNP calling, quality control, anchoring to the genome, and imputations</h3><p id="p-16">Quality inspection of sequence files and SNP calling was performed on the high-performance computing clusters available at the Holland Computing Center (<a href="http://hcc.unl.edu">http://hcc.unl.edu</a>) at UNL. The sequence quality was inspected using FASTQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). SNP calling was done using the TASSEL GBS pipeline v4.0 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Glaubitz <em>et al.</em> 2014</a>) and merging of multiple samples of the same line was done in version 3. SNP calling was implemented using default parameters except for the criterion of a number of times a GBS tag to be present to be included for SNP calling was changed from the default value of 1 to 5 to increase the stringency in SNP calling. A pseudo-reference genome built using the International Wheat Genome Sequencing Consortium Chromosome Survey Sequences (CSS) v2.0 of the hexaploid bread wheat variety <em>Chinese Spring</em> was utilized for SNP calling (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">International Wheat Genome Sequencing Consortium (IWGSC) 2014</a>). The GBS data of 1,100 lines of the four PYT nurseries grown in 2012-2015 were analyzed along with an additional 2,202 lines to get a better coverage of the genome, increasing read depth, and for inspecting SNP calling accuracy (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2015</a>). These additional samples included one bi-parental mapping population, an F<sub>3:5</sub> nursery of the breeding program grown in 2015, and check cultivars. A total of 3,302 unique samples were used for SNP calling. We compared SNP calls of 20 lines genotyped by GBS at least twice (biological replicates) over the years for assessing the SNP calling accuracy (File S5). SNP markers with missing information in more than 80% (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Torkamaneh and Belzile 2015</a>) of the samples were excluded using VCFtools (–max-missing 0.20; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Danecek <em>et al.</em> 2011</a>) and the distribution of missing values across SNPs and lines was investigated (Figure S2 in File S1). Imputations were performed on the remaining set of markers in Beagle v4.0 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Browning and Browning 2007</a>). Further, SNP markers with minor allele frequency less than 0.05 (–maf 0.05) and allelic R<sup>2</sup> estimated using Beagle less than 0.5 (SNPs that could be difficult to impute accurately) were excluded (Figure S2 in File S1; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Browning and Browning 2007</a>). The genomic location information of the SNPs was derived using the positions of the CSS contigs that were anchored and ordered by population sequencing and a high-density linkage map (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Chapman <em>et al.</em> 2015</a>).</p></div><div id="sec-6" class="subsection"><h3>Genomic prediction and cross-validation design</h3><p id="p-17">Phenotypic response of the line (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-9.gif"/></noscript></span></span>) can be described as the sum of an overall mean (μ), plus the random effect of the line (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-10.gif"/></noscript></span></span>), plus an error term (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-11.gif"/></noscript></span></span>) as follows:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where line effects are IID (independent and identically distributed) drawn from a normal distribution of the form <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-13.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-14.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-15.gif"/></noscript></span></span>. In equation (5) the effects of the different levels of random line effect are independent and the information is not borrowed across lines. When genome-wide markers are available, the genomic values of all the lines are normally distributed with <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-16.gif"/></noscript></span></span>, where G is the GRM calculated using genome-wide SNP markers and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-17.gif"/></noscript></span></span> is the genomic variance. Collecting the aforementioned assumptions the final GBLUP model results as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-19.gif"/></noscript></span></span> is the vector of phenotype (for example, BLUPs of lines grown in 4 years), and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-21.gif"/></noscript></span></span>. In equation (6) the effects of the different levels of random line effect are correlated as per the off-diagonal values of GRM and there is potentially borrowing of information across lines, which allows for predicting the phenotype of lines that are missing.</p><p id="p-18">The GRM was estimated as in <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Perez and de los Campos (2014)</a>. Briefly, the SNP markers obtained after quality control were converted from nucleotide format to numerical format using the <em>NumericalGenotypePlugin</em> implemented in the TASSEL software (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Bradbury <em>et al.</em> 2007</a>). Genotype matrix (X) was multiplied by 2 to change the genotype scale to 2, 0, and 1 for homozygous major, homozygous minor, and heterozygous before estimating GRM. The X matrix was centered and scaled by subtracting each value in the column from column means and by dividing with their standard deviations using the <em>scale</em> function in R. Subsequently, GRM was estimated as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-22.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2735/embed/mml-math-23.gif"/></noscript></span></span> is the number of SNP markers.</p><p id="p-19">The BGLR function in the Bayesian Generalized Linear Regression (BGLR; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Perez and de los Campos 2014</a>) package in R was used to fit the GBLUP model and genomic estimated breeding values were obtained. Each run was performed with 12,000 iterations and 2,000 burn-in. The BGLR function with default values was used for assigning prior distribution and hyper-parameter settings (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Perez and de los Campos 2014</a>).</p><p id="p-20">The cross-validation scheme was designed to investigate how GP can predict grain yield of PYT and address the practical scenarios that may occur in a breeding program (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For example, an occurrence of extreme weather events such as hailstorms during a breeding season may result in loss of a subset of the trial. The rest of the lines within the same location are not damaged, and the data from the undamaged lines will be available to combine with the training set and predict the phenotype of damaged lines. Another example where the cross-validation scheme will have practical utility is to use GS as a tool for allocation of breeding program resources. If we determine the number of lines that can be predicted with reasonable accuracy, it may be possible to grow only a subset of lines in the trial and predict the rest and save costs by reduced phenotyping. In our cross-validation scheme, the training set comprised F<sub>3:6</sub> lines from three years and randomly chosen lines in steps of 10% from the fourth year and test set included the rest of the lines from the fourth year (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). For instance, the training set may include all lines tested in 2013, 2014, and 2015 PYT nurseries, and 90% of the lines from 2012 PYT nursery randomly selected, and the test set includes the remaining 10% of the lines grown in 2012. This process was repeated 10 times by randomly sampling 90% of the lines in each run. We refer to this cross-validation scenario where 10% of the lines are excluded as NA10. Similarly, in 2012, NA50 training set will comprise all lines tested in 2013, 2014, and 2015, and 50% of the lines randomly chosen from 2012, and the test set will include the remaining 50% of the lines tested in 2012. We repeated this process 10 times by randomly selecting 50% of the lines from 2012 in each run. This cross-validation strategy was tested for each of the four years (2012-2015) with NA10 to the NA90 scenario and by randomly selecting lines 10 times (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490623/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490623/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of prediction scenarios and composition of training and test set using 2012 as an example</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">We also predicted the yield of all the lines in the PYT nursery in a new year using lines tested in other years (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). For example, we predicted the yield of lines grown in 2012 PYT nursery using lines tested in 2013, 2014, and 2015 PYT nurseries. We refer to it as NA100. Although we are predicting older lines (for example, 2012 PYT) using newer lines (2013, 2014, and 2015 PYT) the results will be insightful for predicting a new nursery in a new year. This is true because development of a wheat cultivar usually takes ∼12 years (Figure S1 in File S1) and elite lines are recycled as parents only six to seven years after the first cross is made.</p><p id="p-22">The PA (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Faville <em>et al.</em> 2018</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sallam <em>et al.</em> 2015</a>) for each run was assessed using the Pearson’s correlation between the predicted genotypic effects (GEBVs) and the adjusted observed phenotypes (BLUPs). We also calculated an average across the 10 runs to estimate average PA for each validation scenario.</p></div><div id="sec-7" class="subsection"><h3>Effectiveness of phenotypic and genomic selection to make selections</h3><p id="p-23">The 57 lines of the ∼270 lines advanced from the PYT nursery (2012-2015) to the AYT nursery (2013-2016) were used to evaluate GS and phenotype to make selections (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The GEBVs for the 57 lines were subdivided from the GEBVs estimated for all ∼270 lines in the PYT nursery (NA100 scenario of predictions). The GS ability was evaluated by estimating the correlation between GEBV of the PYT nursery and their BLUPs in the AYT nursery. Similarly, the phenotypic selection ability was estimated by calculating the correlation between BLUPs of the PYT nursery and their BLUPs in the AYT nursery. Comparing the correlation coefficients of GS and phenotypic selection abilities would indicate the effectiveness of using phenotypic selection and GS for making selections from PYT to AYT in each of the four years. Since the PYT conducted in validation year is included in the training set, we have observed the environment where the predictions will be tested in the training set. Therefore, the predictions are effectively made for new lines and are evaluated in an observed environment. This scenario is valuable as the new growing year may experience similar environmental conditions as one of the previously observed years.</p><p id="p-24">We also evaluated GS and phenotypic selection by skipping the PYT nursery tested in the AYT year in the training set (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). For example, the GEBV for lines grown in 2012 PYT was estimated using lines grown in 2014 and 2015 PYT (skipping 2013 PYT) in the training set (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). To match the training set comprising two PYTs to predict 2012, 2013 or 2014, the GEBVs for lines grown in 2015 are estimated using lines grown in 2012 and 2013 PYT as a training set. The downside of estimating GEBV by skipping the PYT lines from the AYT year in the training set is it reduces the size of the training set by 270 to 280 lines compared to the first approach (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). A smaller population size of the training set may affect the accuracy of the GEBVs. Overall, the results from these two scenarios will allow comparison of GS and phenotypic selection for making selections.</p><p id="p-25">Lastly, genomic and phenotypic selections are evaluated using the 57 lines advanced from the PYT to the AYT nursery and not all of the ∼270 lines in the PYT. These 57 lines generally represent high-yielding lines and belong to one tail of the distribution containing all lines in the PYT nursery (Figure S3 in File S1). Nevertheless, using PYT and AYT provides an opportunity to compare GS and phenotypic selection using existing datasets.</p></div><div id="sec-8" class="subsection"><h3>Prospects for implementing genomic selection in the preliminary yield trial</h3><p id="p-26">To evaluate the prospects of implementing GS in the PYT, we tracked the lines advanced until 2016 primarily based on phenotype from the PYT nurseries grown in 2012-2015 and compared it with BLUP and GEBV values estimated on the lines in the PYT nursery. For instance, lines advanced from the 2012 PYT nursery until 2016 were selected and retained in the breeding program for five years. Similarly, lines are retained for four years starting from 2013 PYT nursery, three years from 2014, and two years from 2015. The GEBVs of the lines grown in the PYT each year were estimated using the lines from the rest of the years as training dataset (NA00 scenario), which is equivalent to predicting new lines in a new year. Further, studying the frequency of lines with above average GEBV or BLUP or both, selected from the PYT nursery together with the information on the number of years a line was retained in the breeding program will allow for evaluation of the prospects of implementing GS in the PYTs in the breeding program. We repeated this entire process by making predictions on 50% of the lines randomly selected in each year and using the remaining 50% of the lines along with the lines from other years as training dataset (NA50 scenario). The NA50 scenario is expected to have higher PA compared to NA100 scenario. Thus, this comparison using the GEBVs from NA50 scenario will test the prospects of implementing GS when PA is higher.</p><p id="p-27">Phenotypic selection in the PYT nursery is primarily based on yield. Typically, lines with higher yield are selected first and then subsequently a few of these lines are dropped based on other traits such as resistance to stem rust (causal organism <em>Puccinia graminis</em> Pers.:Pers. f. sp. <em>tritici</em> Eriks. E. Henn.), agronomic performance, and end-use quality (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Baenziger <em>et al.</em> 2011</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Baenziger <em>et al.</em> 2001</a>). A lower yielding line in the PYT generally does not get advanced to AYT irrespective of the values of other traits. Therefore, tracking lines advanced from the PYT based on phenotype and comparing it with BLUP and GEBV values based on yield is a relatively fair comparison to determine the prospects of implementing GS in the PYTs.</p></div><div id="sec-9" class="subsection"><h3>Population structure and kinship</h3><p id="p-28">Population structure among the 1,110 F<sub>3:6</sub> lines was investigated using principal component analysis (PCA). The PCA was performed in TASSEL (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Bradbury <em>et al.</em> 2007</a>) and a plot of PC1 <em>vs.</em> PC2 was made using the ggplot2 package (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wickham 2009</a>) in R. The kinship between test and training set was analyzed by calculating the maximum realized kinship coefficient (MRKC; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Auinger <em>et al.</em> 2016</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Saatchi <em>et al.</em> 2011</a>) using the GRM (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Perez and de los Campos 2014</a>). The MRKC was calculated as max (<em>U<sub>ij</sub></em>) and <em>U<sub>ij</sub></em> is the realized kinship coefficient between line <em>i</em> in the test set and line <em>j</em> in the training set. The MRKC value represents the kinship between a specific line in the test set and all lines in the training set. Further, a mean of maximum kinship coefficient values across all lines in the test set was calculated.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-29">File S1 contains supplementary Tables S1-S2 and Figures S1-S7. File S2 has details of the preliminary yield trials (F<sub>3:6</sub>) and advanced yield trials (F<sub>3:7</sub>) grown in 2012-2015 and 2013-2016. The experimental design, number of locations, number of replicates per location, heritability for yield within and across locations is provided. File S3 contains the R script developed for analyzing preliminary yield trials (F<sub>3:6</sub>) grown in an augmented design. Mixed models incorporated both experimental design and spatial variation. File S4 provides an example phenotypic dataset for utilizing the R script provided in File S3. File S5 shows the SNP calling accuracy of 20 lines genotyped by genotyping-by-sequencing at least twice (biological replicates) over the years. The accuracy is presented as sequential pairwise comparisons of 20 unique samples. File S6 contains the yield data (best linear unbiased predictors) of the preliminary yield trials (F<sub>3:6</sub>) grown in 2012-2015 used for genomic selection. File S7 contains the SNP marker data of 26,925 SNPs on the 1,110 F<sub>3:6</sub> lines grown in 2012-2015. The homozygous major, homozygous minor, and heterozygous genotypes are coded as 1, 0, and 0.5 in the genotype matrix. File S8 shows the Akaike Information Criterion (AIC) values and the residual plots for each of the mixed models tested for the yield data collected on the preliminary yield trial (F<sub>3:6</sub>) grown in McCook, NE in 2014. The rest of the analyses described in this study (except for the phenotypic data analysis conducted using ASreml v3.0) were performed using publicly available software and R packages. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6249410">https://doi.org/10.25387/g3.6249410</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Accounting for spatial variation in field trials and broad-sense heritability</h3><p id="p-30">Grain yield data of the PYT nursery were analyzed using linear mixed models incorporating spatial variation. The motivation for testing and correcting for spatial variation to improve genomic predictions was derived from recent articles (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Bernal-Vasquez <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lado <em>et al.</em> 2013</a>) that showed accounting for spatial variation in the field result in either similar or higher predictive abilities. Of the 33 environments (location by year combination) tested, five environments (trials conducted by WestBred, LLC or Bayer CropScience) did not have field-layouts available and thus the model accounted for just the experimental design (incomplete block) when generating the BLUPs for the lines (Table S1 in File S1). For 27 of the remaining 28 environments, the model accounting for spatial variation showed better performance compared to models accounting for just the experimental design based on lower AIC values, normality, and reduced heteroscedasticity of residuals (Table S1 in File S1). As an example of the model selection results, a table with the AIC values and residual plots for various models is provided for the PYT nursery grown at McCook, NE in 2014 (File S8). Averaging the BLUPs of lines grown at various locations within a year significantly improved the broad-sense heritability (H). For instance, H across the PYT nurseries within a year increased from ∼0.10 (across two distant locations) to ∼0.74 (across all of the 7 to 10 locations; File S2). Higher H will result in increased predictive abilities in the GP analysis.</p><p id="p-31">In the PYT nursery, the H for grain yield within a location with no replicate varied from 0.25 to 0.95, whereas at locations with two replicates H ranged from 0.62 to 0.79 (File S2). The H across locations within a year ranged from 0.63 (2013 year) to 0.81 (2012 year; File S2). In the AYT nursery, the H was generally higher than the PYT nursery with values up to 0.94 and the H across locations within a year ranged from 0.58 (2014 year) to 0.79 (2015 year; File S2). The AYT nursery’s smaller sample size (57 lines) and increased replication (two or three replicates) improved heritability relative to the PYT nursery with ∼270 lines and one or two replicates. The 2013 PYT (except at Lincoln and Clay Center) and 2014 AYT (except at Mead, Lincoln, and Clay Center) had lower heritability trials at other locations. Sequential addition of locations (starting with two geographically distant locations and then subsequently adding sites generally from the eastern to the western side of the state) within a year in the AYT nursery led to a similar increase in H as was found in the PYT nursery. The H for the AYT nursery ranged from ∼0.20 (across two distant locations) to ∼0.79 (across all of the 6 to 7 locations; File S2).</p></div><div id="sec-13" class="subsection"><h3>SNP calling, quality filtering, population structure, and kinship</h3><p id="p-32">We performed SNP calling using the GBS data of 1,100 lines of the four PYT nurseries grown in 2012-2015 along with an additional 2,202 lines and identified 206,622 SNPs. The SNP calling accuracy tested using sequential pairwise comparisons of 20 samples genotyped using GBS at least twice was 95.7% (File S5). The 206,622 SNPs were filtered to exclude markers that had missing information in more than 80% of the samples, and this resulted in 79,118 SNPs. The distribution of missing sites across SNPs was left-skewed, and across lines, it was relatively normally distributed (Figure S2 in File S1). These SNPs were then processed using BEAGLE and the missing genotypes were imputed. The use of GBS results in low coverage and high missing information, which precludes the accurate estimation of imputation accuracy. Therefore, allelic R<sup>2</sup> generated in BEAGLE was used as a filtering criterion to exclude the likely low quality imputed SNPs (Figure S2 in File S1; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Browning and Browning 2007</a>). The distribution of missing information across these selected (allelic R<sup>2</sup> &gt; 0.5) SNPs prior to imputation looks random compared to the left-skewed distribution of all SNPs used for imputation (Figure S2 in File S1). Subsequently, the imputed marker data set was subdivided to include just the 1,100 lines of the 2012-2015 PYT nurseries and filtered using two criteria: (1) MAF &gt; 0.05; (2) MAF &gt; 0.05 and allelic R<sup>2</sup> &gt; 0.5. Filtering using only MAF criterion resulted in 41,913 SNPs and both MAF and allelic R<sup>2</sup> provided 26,925 SNPs. The Pearson’s correlation coefficient estimated between kinship coefficients estimated using 41,913 and 26,925 SNP markers was 0.99. This is not unexpected and previous studies have shown the method of imputation has little effect on the kinship and predictive abilities (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Poland <em>et al.</em> 2012a</a>). Hence, we decided to proceed with the higher-quality 26,925 SNPs for the subsequent analysis (File S7).</p><p id="p-33">The 26,925 SNPs were distributed on all 21 chromosomes. The number of markers on a chromosome ranged from 114 (4D) to 3,323 (3B) with an average of 1,282 per chromosome (Figure S4a in File S1). Nearly 19,707 of the 26,925 SNPs were placed in CSS contigs anchored and ordered on chromosomes using population sequencing (Figure S4b in File S1). The distribution of CSS contigs containing the 19,707 SNPs across each chromosome showed good coverage of the wheat genome except for a few D genome chromosomes, previously reported to be less polymorphic (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lado <em>et al.</em> 2013</a>). The population structure evaluated using PCA by plotting PC1 <em>vs.</em> PC2 did not indicate the presence of prominent clusters suggesting the absence of strong population structure in the PYT nurseries grown in 2012-2015 (Figure S5 in File S1). The PC1 and PC2 explained 4.3% and 3.2% of the variation in the dataset (Figure S5 in File S1).</p><p id="p-34">The average MRKC value for the PYT grown in 2012-2015 with other PYTs ranged from 0.32 to 0.37 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). The PYT 2013 and 2014 contained a few lines that had a substantially higher kinship with PYT lines grown in other years (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a>). The results were similar after subsetting MRKC values for the lines advanced from PYT to AYT and the average MRKC for the lines advanced each year and tested in AYT in 2013-2016 ranged from 0.34 to 0.41 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The maximum realized kinship coefficient (MRKC) between lines grown in a testing year and lines in rest of the years (training set). The MRKC was calculated as max (Uij) and Uij is the realized kinship coefficient between line i in the test set and line j in the training set. The MRKC value represents the kinship between a specific line in the test set and all lines in the training set. The kinship of each of the preliminary yield trials (PYT 2012-2015) with rest of the PYTs (training set) and kinship between lines advanced from the PYT to advanced yield trial (AYT) each year (example, AYT 2013) and lines in the training set (PYT 2013, 2014, and 2015) are shown in a and b. The average maximum realized kinship coefficient estimated across all lines grown in PYT and AYT is written on top of each of the violin plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1864905185" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The maximum realized kinship coefficient (MRKC) between lines grown in a testing year and lines in rest of the years (training set). The MRKC was calculated as max (&lt;em&gt;U&lt;sub&gt;ij&lt;/sub&gt;&lt;/em&gt;) and &lt;em&gt;U&lt;sub&gt;ij&lt;/sub&gt;&lt;/em&gt; is the realized kinship coefficient between line &lt;em&gt;i&lt;/em&gt; in the test set and line &lt;em&gt;j&lt;/em&gt; in the training set. The MRKC value represents the kinship between a specific line in the test set and all lines in the training set. The kinship of each of the preliminary yield trials (PYT 2012-2015) with rest of the PYTs (training set) and kinship between lines advanced from the PYT to advanced yield trial (AYT) each year (example, AYT 2013) and lines in the training set (PYT 2013, 2014, and 2015) are shown in a and b. The average maximum realized kinship coefficient estimated across all lines grown in PYT and AYT is written on top of each of the violin plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2735/F1.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2735/F1.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490637" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-35" class="first-child">The maximum realized kinship coefficient (MRKC) between lines grown in a testing year and lines in rest of the years (training set). The MRKC was calculated as max (<em>U<sub>ij</sub></em>) and <em>U<sub>ij</sub></em> is the realized kinship coefficient between line <em>i</em> in the test set and line <em>j</em> in the training set. The MRKC value represents the kinship between a specific line in the test set and all lines in the training set. The kinship of each of the preliminary yield trials (PYT 2012-2015) with rest of the PYTs (training set) and kinship between lines advanced from the PYT to advanced yield trial (AYT) each year (example, AYT 2013) and lines in the training set (PYT 2013, 2014, and 2015) are shown in a and b. The average maximum realized kinship coefficient estimated across all lines grown in PYT and AYT is written on top of each of the violin plots.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genomic predictions made on F<sub>3:6</sub> nurseries grown in 2012-2015</h3><p id="p-36">We predicted the performance of 10% (NA10) to 90% (NA90) of the lines grown annually in steps of 10% and repeated the random sampling of lines for the test set 10 times and estimated the average predictive abilities. In three of the four years (2012, 2014, and 2015), the average predictive abilities for NA10 ranged from 0.42 to 0.52, NA50 from 0.37 to 0.44, and NA90 from 0.23 to 0.30 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In 2013, the predictive abilities were slightly lower than in the other years and average predictive abilities for NA10, NA50, and NA90 scenarios were 0.37, 0.27, and 0.24 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>). Average predictive abilities steadily decreased from NA10 to NA90 in each year. This may be due to the varying degree of kinship between training and test set resulting from increase in the number of lines in the test set from ∼27 lines in NA10 to ∼243 lines in NA90 or a decrease in the number of lines in the training set from ∼1,073 in NA10 to ∼857 in NA90 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>; <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). In addition, the genotype by environment interactions may impact the PA (Table S2 in File S1). We also investigated the capability to predict all lines (∼270) in a year using data from other years, the NA100 scenario. The predictive abilities for predicting an entire trial each year (2012-2015) ranged from 0.17 to 0.26 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). The range of PA in the NA100 scenarios suggests the influence of genotype by environment interaction on PA (Table S2 in File S1).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive ability (PA) estimated for grain yield using various cross-validation scenarios (NA10 to NA90) and for an entire new preliminary yield trial nursery (NA100) in each of the four years, 2012 to 2015, a to d. Each of cross-validation scenarios was performed 10 times and the mean PA is written on top of each of the box plot and is also highlighted with an asterisk in the boxplots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1864905185" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predictive ability (PA) estimated for grain yield using various cross-validation scenarios (NA10 to NA90) and for an entire new preliminary yield trial nursery (NA100) in each of the four years, 2012 to 2015, a to d. Each of cross-validation scenarios was performed 10 times and the mean PA is written on top of each of the box plot and is also highlighted with an asterisk in the boxplots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2735/F2.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2735/F2.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490594" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Predictive ability (PA) estimated for grain yield using various cross-validation scenarios (NA10 to NA90) and for an entire new preliminary yield trial nursery (NA100) in each of the four years, 2012 to 2015, a to d. Each of cross-validation scenarios was performed 10 times and the mean PA is written on top of each of the box plot and is also highlighted with an asterisk in the boxplots.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Genomic selection <em>vs.</em> phenotypic selection</h3><p id="p-38">The AYT nurseries grown in 2013-2016, containing 57 lines advanced primarily based on the phenotype from the PYT nurseries grown in 2012-2015, were utilized to compare phenotypic selection and GS. In 2012 and 2015, the correlation coefficient between GEBV (estimated using NA100 scenario) of the PYT nursery and BLUP of the AYT nursery was higher than the correlation coefficient between BLUP of the PYT and AYT nursery, 0.48 <em>vs.</em> 0.36 in 2012-2013 and 0.36 <em>vs.</em> 0.11 in 2015-2016 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). When the AYT year was excluded from the training set, the correlation coefficient between GEBV of the PYT nursery and BLUP of the AYT nursery was similar or substantially higher than the correlation coefficient between BLUP of the PYT and AYT nursery, 0.36 <em>vs.</em> 0.33 in 2012-2013 and 0.45 <em>vs.</em> 0.11 in 2015-2016 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This suggests GS would either perform equally well or outperform phenotypic selection during 2012-2013 and 2015-2016. However, in 2014-2015, the correlation coefficient between GEBV of the PYT nursery and BLUP of the AYT nursery was nearly half (or lower than) the correlation coefficient between BLUP of the PYT and AYT nursery, 0.13 <em>vs.</em> 0.30, and -0.05 <em>vs.</em> 0.30 when the AYT year was excluded from the training set (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). And in 2013-2014, the correlation coefficient between GEBV of the PYT nursery and BLUP of the AYT nursery was significantly lower than the correlation coefficient between BLUP of the PYT and AYT nursery, -0.02 (-0.08 skipping the AYT year) <em>vs.</em> 0.37 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). This indicates phenotypic selection would outperform GS in 2013-2014 and 2014-2015.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of phenotypic and genomic selection. The comparison is made using 57 lines advanced from the preliminary yield trial (F3:6) grown in 2012-2015 to the advanced yield trial (F3:7) grown in 2013-2016. The suffix &#x201C;2&#x201D; separated by an underscore represents scenarios where the genomic estimated breeding values of the preliminary yield trials (F3:6) are estimated by excluding the following (F3:7) year from the training set. Correlation coefficients noted on the bar plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1864905185" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of phenotypic and genomic selection. The comparison is made using 57 lines advanced from the preliminary yield trial (F&lt;sub&gt;3:6&lt;/sub&gt;) grown in 2012-2015 to the advanced yield trial (F&lt;sub&gt;3:7&lt;/sub&gt;) grown in 2013-2016. The suffix &#x201C;2&#x201D; separated by an underscore represents scenarios where the genomic estimated breeding values of the preliminary yield trials (F&lt;sub&gt;3:6&lt;/sub&gt;) are estimated by excluding the following (F&lt;sub&gt;3:7&lt;/sub&gt;) year from the training set. Correlation coefficients noted on the bar plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2735/F3.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2735/F3.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490513" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Comparison of phenotypic and genomic selection. The comparison is made using 57 lines advanced from the preliminary yield trial (F<sub>3:6</sub>) grown in 2012-2015 to the advanced yield trial (F<sub>3:7</sub>) grown in 2013-2016. The suffix “2” separated by an underscore represents scenarios where the genomic estimated breeding values of the preliminary yield trials (F<sub>3:6</sub>) are estimated by excluding the following (F<sub>3:7</sub>) year from the training set. Correlation coefficients noted on the bar plots.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Tracking selections and comparison of observed phenotypes and genomic estimated breeding values</h3><p id="p-40">To examine the prospects of using GS in the PYT for making selections in the breeding program, we compared the advancements made primarily based on phenotype for five years 2012-2016 with the BLUP and GEBV values estimated on the lines in the PYT nurseries grown in 2012-2015 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; Figure S6 in File S1). Lines with both BLUP and GEBV values above the respective means were retained for more years in the breeding program. If the lines had just the BLUP or GEBV alone above the mean but not both, then these lines were dropped in subsequent years from the breeding program (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; Figure S6 in File S1). For example, in the 2012 PYT nursery, 36 of the 71 (∼50%) lines retained for two years had the BLUP and the GEBV values above the means. Subsequently, 25 of the 34 (∼74%), 7 of the 8 (87%), and 2 of the 2 (100%) lines retained for 3, 4, and 5 years had the BLUP and the GEBV values above the means (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4a</a>). Clearly, lines with both above average GEBV and BLUP values from each year are retained for more years. The predictive abilities for the four years are in the range of 0.17 to 0.26 (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>; Figure S6 in File S1). Although these prediction abilities seem low, it is interesting to note that using both GEBV and BLUP for grain yield can increase the accuracy of selections from the PYT nursery in the breeding program.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of adjusted observed grain yield values (best linear unbiased predictors, BLUPs) and genomic estimated breeding values (GEBVs) for lines tested in 2012 (a) and 2013 (b) preliminary yield trials (PYTs). The lines advanced in each year from the 2012 and 2013 PYT nursery until 2016 are highlighted with different colors and shapes. The two vertical lines represent the mean of the BLUPs of the PYT nursery and the mean of the BLUPs of the 57 PYT lines selected for advancement to the advanced yield trial nursery. The two horizontal lines indicate the mean (solid) and the 75th percentile of the genomic estimated breeding values (dashed)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1864905185" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of adjusted observed grain yield values (best linear unbiased predictors, BLUPs) and genomic estimated breeding values (GEBVs) for lines tested in 2012 (a) and 2013 (b) preliminary yield trials (PYTs). The lines advanced in each year from the 2012 and 2013 PYT nursery until 2016 are highlighted with different colors and shapes. The two vertical lines represent the mean of the BLUPs of the PYT nursery and the mean of the BLUPs of the 57 PYT lines selected for advancement to the advanced yield trial nursery. The two horizontal lines indicate the mean (solid) and the 75&lt;sup&gt;th&lt;/sup&gt; percentile of the genomic estimated breeding values (dashed).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2735/F4.medium.gif" width="344" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2735/F4.medium.gif" width="344" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Comparison of adjusted observed grain yield values (best linear unbiased predictors, BLUPs) and genomic estimated breeding values (GEBVs) for lines tested in 2012 (a) and 2013 (b) preliminary yield trials (PYTs). The lines advanced in each year from the 2012 and 2013 PYT nursery until 2016 are highlighted with different colors and shapes. The two vertical lines represent the mean of the BLUPs of the PYT nursery and the mean of the BLUPs of the 57 PYT lines selected for advancement to the advanced yield trial nursery. The two horizontal lines indicate the mean (solid) and the 75<sup>th</sup> percentile of the genomic estimated breeding values (dashed).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">In 2017, one (NE12561) of the two lines from the 2012 PYT nursery and five (NE13434, NE13515, NE13604, NW13493, NW13570) of the 13 lines from 2013 PYT are still retained and are tested in the elite yield trails (F<sub>3:8</sub> nursery; Nebraska Interstate Nursery). Three lines from 2013 PYT (NE13515, NW13493, NW13570) are also tested in the Southern Regional Performance Nursery. A few of these lines are included in the state variety testing program and they have performed well (data not shown).</p><p id="p-43">Next, we repeated the same analysis with the NA50 scenario of predictions in which 50% of the lines evaluated annually were added to the training dataset and predictions were made on the remaining 50% of the lines (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>; Figure S7 in File S1). This analysis was performed to test the use of GEBV and BLUP to make selections when the predictive abilities are higher than in the NA100 scenario. More lines with both above average GEBV and BLUPs are retained for more years in the breeding program compared to the NA100 scenario (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). Also, lines with either their GEBV or BLUP alone higher than the respective means but not both were significantly reduced (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>; Figure S7 in File S1). The predictive abilities for the four years for the NA50 scenario were higher than the NA100 scenario and were in the range of 0.26 to 0.37 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>; Figure S7 in File S1).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of adjusted observed grain yield (best linear unbiased predictors, BLUPs) and genomic breeding values estimated using all lines in other years and 50% of the lines randomly selected from 2012 (a) and 2013 (b) from preliminary yield trials (PYTs). The lines advanced in each year from the 2012 and 2013 PYT nursery until 2016 are highlighted with different colors and shapes. The two vertical lines represent the mean of the BLUPs of the PYT nursery and the mean of the BLUPs of the 57 PYT lines selected for advancement to the advanced yield trial nursery. The two horizontal lines indicate the mean (solid) and the 75th percentile of the genomic estimated breeding values (dashed)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1864905185" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of adjusted observed grain yield (best linear unbiased predictors, BLUPs) and genomic breeding values estimated using all lines in other years and 50% of the lines randomly selected from 2012 (a) and 2013 (b) from preliminary yield trials (PYTs). The lines advanced in each year from the 2012 and 2013 PYT nursery until 2016 are highlighted with different colors and shapes. The two vertical lines represent the mean of the BLUPs of the PYT nursery and the mean of the BLUPs of the 57 PYT lines selected for advancement to the advanced yield trial nursery. The two horizontal lines indicate the mean (solid) and the 75&lt;sup&gt;th&lt;/sup&gt; percentile of the genomic estimated breeding values (dashed).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2735/F5.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/8/2735/F5.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2735/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490483" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Comparison of adjusted observed grain yield (best linear unbiased predictors, BLUPs) and genomic breeding values estimated using all lines in other years and 50% of the lines randomly selected from 2012 (a) and 2013 (b) from preliminary yield trials (PYTs). The lines advanced in each year from the 2012 and 2013 PYT nursery until 2016 are highlighted with different colors and shapes. The two vertical lines represent the mean of the BLUPs of the PYT nursery and the mean of the BLUPs of the 57 PYT lines selected for advancement to the advanced yield trial nursery. The two horizontal lines indicate the mean (solid) and the 75<sup>th</sup> percentile of the genomic estimated breeding values (dashed).</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Experimental line NE13625 is an exception to the observation that the lines retained for more years in the breeding program have both BLUP and GEBV values about their respective means. NE13625 was the fourth highest yielding line in the 2013 PYT but had extremely low GEBV. It was advanced until 2016 and is marked with a “+” sign in the lower right quadrant in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4b</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5b</a>. The line was dropped in 2016 and did not make it to the 2017 nursery. This experimental line is a relatively tall wheat, and there is a preference to retain a few tall lines that may be high yielding relative to the existing commercially available tall lines or those in the breeding program.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Genotyping-by-sequencing derived genome-wide markers for genomic selection</h3><p id="p-46">Genotyping-by-sequencing provided thousands of high-quality SNPs after quality filtering for GP analysis. In this study, we utilized 188-plexing and obtained nearly 27,000 high-quality SNPs distributed across the genome. The number of SNPs obtained in this study using GBS after quality filtering is comparable (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Dawson <em>et al.</em> 2013</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Poland <em>et al.</em> 2012a</a>) or higher than the previously published articles on genomic predictions in wheat (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Battenfield <em>et al.</em> 2016</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rutkoski <em>et al.</em> 2014</a>). SNP calling accuracy was high (∼95%) when tested using biological replicates with small samples of F<sub>3:6</sub> lines and multiple check cultivars. Also, the percentage of SNP calling accuracy observed in this study using GBS is similar to other platforms such as RNA-Seq (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Belamkar <em>et al.</em> 2016</a>) and targeted resequencing of the wheat exome (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2013</a>). The GBS is a cost-efficient, reliable, and powerful technique for rapidly genotyping breeding germplasm with genome-wide markers. In addition, a cost-effective strategy for a self-pollinated crop such as wheat would be to genotype lines from a breeding program nursery with a manageable number of lines and then use the same markers for lines that are advanced. For example, we are presently genotyping F<sub>3:5</sub> nursery in the University of Nebraska breeding program that contains nearly ∼2,000 lines annually and using the same marker data for lines advanced to F<sub>3:6</sub> and F<sub>3:7</sub> nurseries. The error introduced due to the minor change in average heterozygosity with each generation is acceptable compared to the cost of re-genotyping lines advanced to the subsequent nurseries. This approach has also been demonstrated recently in a different wheat breeding program (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Michel <em>et al.</em> 2017</a>).</p></div><div id="sec-19" class="subsection"><h3>Genomic predictions made on preliminary yield trial nursery grown in 2012-2015</h3><p id="p-47">Genomic predictions made in each year (2012-2015) indicate predictive abilities are high for NA10, modest for NA50, and low for NA100. Also, the variability in the predictive abilities is higher from NA10 to NA40 compared to NA50 to NA90. Higher predictive abilities obtained on a smaller number of lines (NA10) and relatively lower predictive abilities on a larger number of lines (NA90) in the test set is probably due to the sample size differences and kinship between lines in the training and test sets. Predictive ability is a function of heritability of the trait, the size of the training population, and an effective number of chromosome segments affecting the trait (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Daetwyler <em>et al.</em> 2008</a>). Training and test population sizes affecting predictive abilities have also been demonstrated in previous studies on various species (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Asoro <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cericola <em>et al.</em> 2017</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tayeh <em>et al.</em> 2015</a>). The high variation in the predictive abilities with fewer lines (NA10-NA40) in the test set is probably due to a varying degree of kinship between lines in the test and training set (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Weng <em>et al.</em> 2016</a>). Recently, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">He <em>et al.</em> (2017)</a> showed that kinship is more important than genetic architecture of trait for determining the estimates of GP on grain yield in wheat. The PA of new lines in a new year using lines from other years (NA100 scenario) was low (∼0.22). Besides kinship, genotype by environment interaction can be responsible for these low values. Although same lines were not tested across years in the PYT, checks were grown in multiple years. Ranking of these checks indicated the influence of genotype by environment interactions on grain yield in different years (Table S2 in File S1). For instance, ‘Goodstreak’ was lower yielding than ‘Camelot’ in all years except in 2015 when it yielded significantly higher. Similarly, Camelot and ‘Freeman’ yielded nearly the same in 2014 whereas in 2015 Freeman yielded higher than Camelot. In summary, these results with the current training dataset generated (containing four PYT nurseries) in the University of Nebraska wheat breeding program indicate that we can predict the yield of nearly 50% of the lines in the PYT nursery with reasonable (0.37 to 0.44) predictive abilities. We believe it may be possible to reduce costs by phenotyping only a subset of lines, especially for hard to measure traits (with similar genetic architecture and heritability as yield), and also recover information for a subset of lines lost during extreme weather events such as a hailstorm or other abiotic and biotic stresses. However, predicting the yield of all lines in the PYT nursery in a new year (forward breeding) and skipping the PYT is not possible with the current selection intensity of advancing 57 of the 270 lines to the AYT.</p></div><div id="sec-20" class="subsection"><h3>Evaluation of genomic selection and phenotypic selection</h3><p id="p-48">The comparison of GS and phenotypic selection for making advancements from the PYT to the AYT nursery indicated GS did better than the phenotypic selection in 2012 and 2015, and in 2013 and 2014 phenotypic selection outperformed GS. When the AYT year was excluded from the training set (for example, predicting PYT 2012 using PYT 2014 and 2015 and skipping the PYT in the AYT year 2013 in the training set), GS was similar to phenotypic selection in 2012 (which makes GS superior because if both GS and phenotypic selection have the same efficiency, we can skip phenotyping), GS did better than phenotypic selection in 2015, and phenotypic selection outperformed GS in 2013 and 2014. The lower PA of GS compared to phenotypic selection after excluding the AYT year in the training set is likely either due to the smaller training set or the fact that the AYT year was not observed in the training set. Overall, GS did better than phenotypic selection in 2012 and 2015, and the phenotype selection was better than GS in 2013 and 2014.</p><p id="p-49">In 2012, Nebraska experienced severe drought (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Baenziger <em>et al.</em> 2012</a>), and in 2015 it received unusually high amounts of rain especially in May, which coincides with the initiation of the flowering of the NE lines (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Baenziger <em>et al.</em> 2015</a>). The 2015 crop year was the third wettest year since the national records began in 1895, and May 2015 was the wettest month of any month on record (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">NOAA 2016</a>). The high amounts of rain resulted in unusually severe disease pressure in 2015, primarily due to stripe rust (causal organism <em>Puccinia striiformis</em> f. sp. <em>tritici</em>) and <em>Fusarium</em> head blight (causal organism <em>Fusarium graminearum Schwabe</em> [teleomorph <em>Gibberella zeae</em> (Schwein.) Petch]) (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Baenziger <em>et al.</em> 2015</a>). The 2014 crop year did not witness any extreme weather events such as drought, flooding, or hailstorms and was a relatively normal year (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baenziger <em>et al.</em> 2014</a>). In both the extreme weather years (2012 and 2015), GS was comparable or better than phenotypic selections, whereas in a normal year (2014) the GS PA was nearly 50% lower than the phenotypic selection. It is interesting to note that GS did better than phenotypic selection while the predictive abilities for the nurseries grown in 2012 and 2015 were 0.22 and 0.17. This suggests predictive abilities alone are most likely not a true indicator of the success of genomic predictions, and the low values obtained may be due to the quality of the underlying phenotype data during extreme weather years. Based on these results in the PYT nursery, GS alone is effective for making selections when extreme weather events occur during the growing season.</p><p id="p-50">The results obtained in 2013 were quite different from all other years. The GS abilities were quite low compared to phenotypic selection. Also, the predictive abilities obtained for NA10 to NA90 were relatively lower. This is despite the fact that the kinship of lines grown in PYT 2013 with lines grown in PYTs in other years was similar to the kinship among other PYTs. In addition to kinship, we specifically looked at the clustering of the lines advanced from PYT 2013 to AYT 2014 with the rest of the PYT lines (Figure S5 in File S1). A plot of PC1 <em>vs.</em> PC2 and a parallel coordinate plot with five PCs did not suggest that these lines were different from the rest (Figure S5 in File S1). This result is expected because each experimental line in the University of Nebraska wheat breeding program is derived from a cross made with one or more lines adapted to the Nebraska region, and thus strong population structure is not expected among the breeding program lines. The possible reasons that may have affected the PA values are: (1) the phenotype of lines grown in 2013 was influenced by the quality of seeds harvested in 2012 coupled with the soil and field conditions affected following severe drought stress in 2012 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Baenziger <em>et al.</em> 2013</a>); (2) the broad-sense heritability values of PYT 2013 and AYT 2014 are lowest compared to PYTs and AYTs in other years (3) the selections performed in 2012 OYT nursery to advance lines to PYT nursery grown in 2013 would have been difficult to make accurately due to the drought stress; and (4) the GBS data of ∼92 lines grown in 2013 was of relatively lower quality (but still acceptable for including it in the analysis based on per base sequence average quality scores &gt;20 along the length of the GBS tags). The most common reasons for lower quality scores is a degradation of quality over the duration of long sequencing runs or a problem with the sequencing run such as bubbles passing through a flowcell (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>).</p></div><div id="sec-21" class="subsection"><h3>Genomic selection is promising for increasing selection accuracy</h3><p id="p-51">In the PYT, lines with both above average BLUP and GEBV values were retained for more years compared to lines with either just GEBV or BLUP alone. A limitation of using phenotype data alone for making selections is that they are based on the performance of lines only in the growing year. But, the expectation is that the selected line performs well in multiple years with potentially varying environmental conditions. The GEBV estimates based on lines grown in multiple years and multiple environmental conditions leveraged information across years and environments and overcomes the limitation of phenotype-based single-year selection. Thus, using both BLUP and GEBV independently and selecting lines that are having both their GEBV and BLUP above the respective means and excluding lines with just the GEBV or BLUP alone above the mean provided an increased opportunity to select lines likely to perform well across environments and years compared to lines selected based on the phenotype alone in one year.</p><p id="p-52">At present, with the current training dataset, the PA is approximately 0.22 for predicting all lines in the PYT nursery in a new year. Hence, with the current selection intensity (selection of 57 of the 270 lines; Figure S3 in File S1) for the subsequent nursery (AYT nursery), we can make highly accurate selections by using both phenotype and GEBV values. Another possibility would be to lower the selection intensity by selecting all lines above the average GEBV for the subsequent nursery. This would allow skipping the PYT, but the downside is the cost of many more lines advanced to the elite trials replicated and grown in many locations, which makes this approach practically challenging and potentially cost ineffective in a breeding program.</p><p id="p-53">The percentage of lines with above average GEBV and BLUP retained for more years in the breeding program increased when 50% of the lines tested in the prediction year are added to the training dataset and predictions are made on the remaining 50% of the lines. The predictive abilities for this NA50 scenario are higher than the NA100. In 2014, the predictive ability was greater than 0.37, and very few lines were in the quadrant with above average GEBV and lower than average BLUP. Our data suggest that growing only a subset of PYT and combining the information from previous years will effectively utilize GEBV for making selections for yield. Also, these results indicate when the predictive ability surpasses the average predictive abilities observed in the NA50 scenario (∼0.40), we may be able to use just GEBV alone for making selections with the current selection intensity for the subsequent nursery and skip the PYT. We have more than ∼1,000 g of seed for most of the lines in the OYT (F<sub>3:5</sub>), and thus enough seed is available to skip the PYT and test lines in the AYT. In addition, we now genotype the lines in OYT and the marker data will be available for lines selected for PYT prior to testing them in the field.</p><p id="p-54">In summary, our recommendation while the predictive abilities for forward breeding are lower (∼0.20), is to either use both the GEBV and BLUP values to make better selections or grow only a selected subset of lines (especially lines that are difficult to predict) advanced to the PYT nursery and use GS to predict the rest and make selections in the PYT nursery. The subset of lines whose phenotype is difficult to predict can be determined by using the reliability criterion recently described by <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Yu <em>et al.</em> (2016)</a> and the training set can be updated annually as described by <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Neyhart <em>et al.</em> (2017)</a>.</p></div><div id="sec-22" class="subsection"><h3>Current use of genomic selection in the University of Nebraska wheat breeding program</h3><p id="p-55">The results obtained in this study are encouraging, and this has motivated implementation of GS in the breeding program for grain yield starting in 2016. Here, we describe three examples of the current use of GS. First, in 2016, we used both GEBV and BLUP to make more accurate selections from the PYT nursery. Preliminary observations indicate in 2017 more lines with both above average GEBV and BLUP were advanced to 2018 and lines with either just above average GEBV or BLUP alone were dropped. Second, lines with both above average GEBVs and BLUPs were shortlisted as top performing lines in the PYT nursery and used as parental lines for the subsequent year’s crossing block. Recycling elite lines sooner saved one to two years in the breeding cycle. Third, hailstorms damaged more than half the OYT nursery (F<sub>3:5</sub>) containing nearly 2,000 lines mostly grown at one location in 2016. We used a training dataset comprising PYT nurseries plus 50% of the lines from the OYT nursery that were less hail damaged to predict the phenotype of the remaining 50% of the lines in the OYT nursery damaged due to hail and recovered lines that had the potential for high-yielding based on GS and advanced these to the PYT nursery for harvest in 2017.</p><p id="p-56">Overall, integration of GS for grain yield in the University of Nebraska wheat breeding program is positioned to successfully improve the efficiency of the breeding program. We currently are testing PAs of other traits and investigating approaches to improve the use of GS in the breeding program.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">The authors are thankful to Greg Dorn and Mitch Montgomery for assistance with conducting and harvesting the field trials in Nebraska; to WestBred, LLC and Bayer CropScience for conducting the preliminary yield trial in Mount Hope, KS and Hutchinson, KS; to Shuangye Wu for technical support of genotyping-by-sequencing; to Dr. Gina Brown-Guedira for building the pseudo-reference genome assembly used for SNP calls; to Amanda Easterly, Nicholas Garst, and Dr. Nonoy Bandillo for valuable suggestions and critical discussions; to all graduate and undergraduate students during 2012-2016 in the Baenziger laboratory for their invaluable support in the field; and to the Holland Computing Center at the University of Nebraska-Lincoln for providing the high-performance computing resources. Partial funding for P.S. Baenziger is from Hatch project NEB-22-328, AFRI/2011-68002-30029, the National Institute of Food and Agriculture as part of the International Wheat Yield Partnership, U.S. Department of Agriculture, under award number 2017-67007-2593, and USDA under Agreement No. 59-0790-4-092 which is a cooperative project with the U.S. Wheat and Barley Scab Initiative, and this project was supported by the National Research Initiative Competitive Grants 2011-68002-30029 and 2017-67007-25939 from the USDA National Institute of Food and Agriculture. Any opinions, findings, conclusions, or recommendations expressed in this publication are those of the authors and do not necessarily reflect the view of the USDA. Cooperative investigations of the Nebraska Agric. Res. Div., Univ. of Nebraska, and USDA-ARS. On behalf of all authors, the corresponding author states that there is no conflict of interest.</p><p id="p-58">Author contributions: VB and PSB conceived and designed the study. WH and MJG performed the DNA isolations. PSB, MJG, and IEB conducted field trials and collected grain yield data. VB and MJG developed the phenotypic data analysis strategy, and VB and WH performed phenotypic data analysis. JP performed genotyping-by-sequencing (GBS). VB and MJG analyzed GBS data and made SNP calls. DJ and AJL mentored VB to perform the genomic predictions. VB wrote the first draft of the manuscript. PSB, MJG, DJ, and AJL provided critical feedback on the initial draft of the manuscript. All authors revised and approved the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6249410">https://doi.org/10.25387/g3.6249410</a>.</p></li><li class="fn" id="fn-2"><p id="p-60"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-10"><span class="received-label">Received </span>May 10, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-19"><span class="accepted-label">Accepted </span>June 19, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Belamkar <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.1" data-doi="10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimization of genomic selection training populations with a genetic algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0116-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711%2526ndash%253B015%2526ndash%253B0116%2526ndash%253B6%26rft_id%253Dinfo%253Apmid%252F25943105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25943105&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.2" data-doi="10.1111/pbi.12009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burridge</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winfield</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Discovery and development of exome‐based, co‐dominant single nucleotide polymorphism markers in hexaploid wheat (Triticum aestivum L.).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12009%26rft_id%253Dinfo%253Apmid%252F23279710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23279710&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316825400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arruda</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krill</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurber</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparing genomic selection and marker-assisted selection for Fusarium head blight resistance in wheat (Triticum aestivum L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D84%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.4" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.02.0007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.5" data-doi="10.2135/cropsci2012.09.0526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic, Marker-Assisted, and Pedigree-BLUP Selection Methods for β-Glucan Concentration in Elite Oat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1894</span>–<span class="cit-lpage">1906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.09.0526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1894%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.09.0526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.09.0526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.6" data-doi="10.1007/s00122-016-2756-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schonleben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (Secale cereale L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2043</span>–<span class="cit-lpage">2053</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2756-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2043%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2756-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2756-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.7"><div class="cit-metadata unstructured">Baenziger, P. S., D. Rose, D. Santra, M. Guttieri, and L. Xu, 2014 Improving wheat varieties for Nebraska: 2014 state breeding and quality evaluation report - retrieved on March 29, 2017 from <a href="https://agronomy.unl.edu/documents/Whtann14Final-3-17-15.pdf">https://agronomy.unl.edu/documents/Whtann14Final-3-17-15.pdf</a>. University of Nebraska-Lincoln, Lincoln, Nebraska</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.8"><div class="cit-metadata unstructured">Baenziger, P. S., D. Rose, D. Santra, M. Guttieri, and L. Xu, 2015 Improving wheat varieties for Nebraska: 2015 state breeding and quality evaluation report - retrieved on March 29, 2017 from <a href="http://agronomy.unl.edu/Baenziger/Whtann15V9FIN.pdf">http://agronomy.unl.edu/Baenziger/Whtann15V9FIN.pdf</a>. University of Nebraska-Lincoln, Lincoln, Nebraska</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.9"><div class="cit-metadata unstructured">Baenziger, P. S., D. Rose, D. Santra, and L. Xu, 2012 Improving wheat varieties for Nebraska: 2012 state breeding and quality evaluation report - retrieved on March 29, 2017 from <a href="http://agronomy.unl.edu/documents/Whtann12V8R2.pdf">http://agronomy.unl.edu/documents/Whtann12V8R2.pdf</a>. University of Nebraska-Lincoln, Lincoln, Nebraska</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.10"><div class="cit-metadata unstructured">Baenziger, P. S., D. Rose, D. Santra, and L. Xu, 2013 Improving wheat varieties for Nebraska: 2013 state breeding and quality evaluation report - retrieved on March 29, 2017 from <a href="http://agronomy.unl.edu/documents/WheatAnnualReport2013.pdf">http://agronomy.unl.edu/documents/WheatAnnualReport2013.pdf</a>. University of Nebraska-Lincoln, Lincoln, Nebraska</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baenziger</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salah</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santra</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regassa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Structuring an Efficient Organic Wheat Breeding Program.</span> <abbr class="cit-jnl-abbrev">Sustainability</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1190</span>–<span class="cit-lpage">1205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSustainability%26rft.volume%253D3%26rft.spage%253D1190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.12" data-doi="10.1023/A:1017583514424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baenziger</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelton</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shipman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graybosch</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Breeding for end-use quality: Reflections on the Nebraska experience.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1017583514424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D119%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017583514424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017583514424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169211500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.13" data-doi="10.1016/j.plantsci.2015.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Breeding schemes for the implementation of genomic selection in wheat (Triticum spp.).</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2015.08.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2015.08.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2015.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.14" data-doi="10.3835/plantgenome2016.01.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Battenfield</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pena</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic Selection for Processing and End-Use Quality Traits in the CIMMYT Spring Bread Wheat Breeding Program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.01.0005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.01.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.01.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.15" data-doi="10.1038/srep34908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belamkar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalberer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackmon</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomics-assisted characterization of a breeding collection of Apios americana, an edible tuberous legume.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">34908</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep34908</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D34908%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep34908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep34908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.16" data-doi="10.1186/1471-2164-15-646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernal-Vasquez</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönleben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The importance of phenotypic data analysis for genomic prediction - a case study comparing different spatial models in rye.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.17" data-doi="10.1007/s00122-016-2772-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bandwagons I, too, have known.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2323</span>–<span class="cit-lpage">2332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2772-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2323%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2772-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2772-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.18" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarekegne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic Gains in Grain Yield Through Genomic Selection in Eight Bi-parental Maize Populations under Drought Stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.07.0460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.19" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.20" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and Accurate Haplotype Phasing and Missing-Data Inference for Whole-Genome Association Studies By Use of Localized Haplotype Clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.21" data-doi="10.1371/journal.pone.0169606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cericola</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahoor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orabi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Optimizing Training Population Size and Genotyping Strategy for Genomic Prediction Using Association Study Results and Pedigree Information. A Case of Study in Advanced Wheat Breeding Lines.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0169606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0169606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0169606%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.22" data-doi="10.1186/s13059-015-0582-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buluç</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georganas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0582-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0582-8%26rft_id%253Dinfo%253Apmid%252F25637298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0582-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25637298&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.23" data-doi="10.2135/cropsci2012.11.0666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide Selection to Introgress Semidwarf Maize Germplasm into U.S. Corn Belt Inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.11.0666</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1427%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.11.0666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.11.0666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.24" data-doi="10.2307/2533991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite> <span class="cit-pub-date">1998</span> <span class="cit-article-title">Spatial Analysis of Multi-Environment Early Generation Variety Trials.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2533991</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D54%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.25" data-doi="10.1371/journal.pone.0003395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of Predicting the Genetic Risk of Disease Using a Genome-Wide Approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3395%26rft.epage%253De3395%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bthe%2Bgenetic%2Brisk%2Bof%2Bdisease%2Busing%2Ba%2Bgenome-wide%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003395%26rft_id%253Dinfo%253Apmid%252F18852893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852893&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.26" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.27" data-doi="10.1016/j.fcr.2013.07.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The use of unbalanced historical data for genomic selection in an international wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2013.07.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D154%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2013.07.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2013.07.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.28" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.29" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2014.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.30" data-doi="10.1007/s00122-017-3030-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faville</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganesh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahufer</span>  <span class="cit-name-given-names">M. Z. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilton</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Predictive ability of genomic selection models in a multi-population perennial ryegrass training set using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">720</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3030-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3030-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3030-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.31" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.32" data-doi="10.2307/1400446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Accounting for Natural and Extraneous Variation in the Analysis of Field Experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/1400446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D2%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1400446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1400446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.33"><div class="cit-metadata unstructured">Gilmour, A. R., B. J. Gogel, B. R. Cullis, and R. Thompson, 2009 ASReml User Guide Release 3.0 VSN International Ltd, Hemel Hempstead, HP1 1ES, UK <a href="http://www.vsni.co.uk">www.vsni.co.uk</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.34" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: A High Capacity Genotyping by Sequencing Analysis Pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.35" data-doi="10.1007/s00122-016-2840-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide mapping and prediction suggests presence of local epistasis in a vast elite winter wheat populations adapted to Central Europe.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2840-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D635%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2840-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2840-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.36" data-doi="10.1007/s00122-015-2655-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirdita</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection in a commercial winter wheat population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">651</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2655-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2655-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2655-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.37" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: A Comparison of Models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">R</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carriquiry</span>  <span class="cit-name-given-names">AL</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">WD</span></span></li></ol><cite> (<span class="cit-pub-date">2014</span>) Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures. G3: Genes|Genomes|Genetics 4:1027</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.39" data-doi="10.1126/science.1251788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Wheat Genome Sequencing Consortium (IWGSC)</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1251788</span>. <span class="cit-comment">DOI: 10.1126/science.1251788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1251788</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BInternational%2BWheat%2BGenome%2BSequencing%2BConsorti%26rft.auinit1%253D%2B%26rft.volume%253D345%26rft.issue%253D6194%26rft.spage%253D1251788%26rft.epage%253D1251788%26rft.atitle%253DA%2Bchromosome-based%2Bdraft%2Bsequence%2Bof%2Bthe%2Bhexaploid%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2529%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251788%26rft_id%253Dinfo%253Apmid%252F25035500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDUvNjE5NC8xMjUxNzg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.40" data-doi="10.1186/1471-2164-15-740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posadas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jedlicka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping by sequencing for genomic prediction in a soybean breeding population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D740%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-740%26rft_id%253Dinfo%253Apmid%252F25174348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-740&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25174348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.41" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemes da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.42" data-doi="10.1104/pp.15.00997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kooke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruijer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bours</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-Wide Association Mapping and Genomic Prediction Elucidate the Genetic Architecture of Morphological Traits in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">2187</span>–<span class="cit-lpage">2203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.00997</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00997%26rft_id%253Dinfo%253Apmid%252F26869705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTcwLzQvMjE4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.43" data-doi="10.1534/g3.113.007807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inostroza</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.007807</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007807%26rft_id%253Dinfo%253Apmid%252F24082033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTIvMjEwNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.44" data-doi="10.2135/cropsci2012.02.0112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H.-J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide Selection <em>vs.</em> Marker-assisted Recurrent Selection to Improve Grain Yield and Stover-quality Traits for Cellulosic Ethanol in Maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.02.0112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.02.0112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.02.0112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.45" data-doi="10.1007/s00122-016-2818-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ametz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gungor</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akgöl</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epure</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic assisted selection for enhancing line breeding: merging genomic and phenotypic selection in winter wheat breeding programs with preliminary yield trials.</span> <abbr class="cit-jnl-abbrev">TAG Theoretical and Applied Genetics Theoretische Und Angewandte Genetik</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2818-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheoretical%2Band%2BApplied%2BGenetics%2BTheoretische%2BUnd%2BAngewandte%2BGenetik%26rft.volume%253D130%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2818-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2818-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.46" data-doi="10.1534/g3.117.040550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neyhart</span>  <span class="cit-name-given-names">JL</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">T</span>  <span class="cit-name-given-names">Tiede</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">AJ</span>  <span class="cit-name-given-names">Lorenz</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">KP</span>  <span class="cit-name-given-names">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-article-title"> Evaluating Methods of Updating Training Data in Long-Term Genomewide Selection. G3: Genes|Genomes|Genetics</span> <span class="cit-vol">7</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.040550</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.040550%26rft_id%253Dinfo%253Apmid%252F28315831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNS8xNDk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.47"><div class="cit-metadata unstructured">NOAA, 2016 National Centers for Environmental Information, State of the Climate: Global Analysis for Annual 2015, published online January 2016, retrieved on March 29, 2017 from <a href="https://www.ncdc.noaa.gov/sotc/global/201513">https://www.ncdc.noaa.gov/sotc/global/201513</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.48" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.49" data-doi="10.3835/plantgenome2016.09.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">J</span>  <span class="cit-name-given-names">Crossa</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">J</span>  <span class="cit-name-given-names">Rutkoski</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">J</span>  <span class="cit-name-given-names">Poland</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">R</span>  <span class="cit-name-given-names">Singh</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Single-Step Genomic and Pedigree Genotype × Environment Interaction Models for Predicting Wheat Lines in International Environments.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2</span>. DOI: <a href="http://10.3835/plantgenome2016.09.0089">10.3835/plantgenome2016.09.0089</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.09.0089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.09.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.09.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.50" data-doi="10.1002/bimj.201100219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz‐Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu Joseph</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A stage‐wise approach for the analysis of multi‐environment trials.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">860</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bimj.201100219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D844%26rft.epage%253D860%26rft.atitle%253DA%2Bstage-wise%2Bapproach%2Bfor%2Bthe%2Banalysis%2Bof%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201100219%26rft_id%253Dinfo%253Apmid%252F23007738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201100219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23007738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.51" data-doi="10.1111/j.1439-0523.2009.01654.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Linear variance models for plant breeding trials.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D129%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2009.01654.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2009.01654.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.52" data-doi="10.1016/j.pbi.2015.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Breeding-assisted genomics.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2015.02.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D24%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2015.02.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2015.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.53" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic Selection in Wheat Breeding using Genotyping-by-Sequencing.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.06.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.54" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.55" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-Sequencing for Plant Breeding and Genetics.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.05.0005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-web" id="cit-8.8.2735.56"><div class="cit-metadata unstructured">R Core Team, 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Randhawa</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asif</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pozniak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graf</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Application of molecular markers to wheat breeding in Canada.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000325412100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic Gain from Phenotypic and Genomic Selection for Quantitative Resistance to Stem Rust of Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.59" data-doi="10.3835/plantgenome2014.02.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">JE</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">JA</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">RP</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">J</span>  <span class="cit-name-given-names">Huerta-Espino</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">S</span>  <span class="cit-name-given-names">Bhavani</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">H</span>  <span class="cit-name-given-names">Barbier</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MN</span>  <span class="cit-name-given-names">Rouse</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">J-L</span>  <span class="cit-name-given-names">Jannink</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ME</span>  <span class="cit-name-given-names">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic Selection for Quantitative Adult Plant Stem Rust Resistance in Wheat.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">7</span><span class="cit-comment">:0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.02.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.02.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.02.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saatchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClure</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolf</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracies of genomic breeding values in American Angus beef cattle using K-means clustering for cross-validation. Genetics, Selection, Evolution</span>. <abbr class="cit-jnl-abbrev">GSE</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGSE%26rft.volume%253D43%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.61" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing Genomic Selection Prediction Accuracy in a Dynamic Barley Breeding Population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.62" data-doi="10.1111/nph.14174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">When more is better: how data sharing would accelerate genomic selection of crop plants.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">814</span>–<span class="cit-lpage">826</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D212%26rft.spage%253D814%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.63" data-doi="10.3389/fpls.2015.00941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tayeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Paslier</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houtin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic Prediction in Pea: Effect of Marker Density and Training Population Size and Composition on Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.00941</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D941%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.00941%26rft_id%253Dinfo%253Apmid%252F26635819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.00941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26635819&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.64" data-doi="10.1371/journal.pone.0131533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torkamaneh</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Scanning and Filling: Ultra-Dense SNP Genotyping Combining Genotyping-By-Sequencing, SNP Array and Whole-Genome Resequencing Data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0131533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0131533</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0131533%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.65" data-doi="10.1007/s00122&amp;ndash;016&amp;ndash;2726-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction for grain zinc and iron concentrations in spring wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2726-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B016%2526ndash%253B2726-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;016&amp;ndash;2726-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2735.66" data-doi="10.1111/j.1467-842X.2010.00570.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welham</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A comparison of analysis methods for late‐stage variety evaluation trials.</span> <abbr class="cit-jnl-abbrev">Aust. N. Z. J. Stat.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-842X.2010.00570.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BN.%2BZ.%2BJ.%2BStat.%26rft.volume%253D52%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-842X.2010.00570.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-842X.2010.00570.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.67" data-doi="10.1186/s12711-016-0198-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effects of number of training generations on genomic prediction for various traits in a layer chicken population.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0198-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0198-9%26rft_id%253Dinfo%253Apmid%252F26992471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0198-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26992471&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2735.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ggplot2: Elegant Graphics for Data Analysis</em>.</span> <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.69" data-doi="10.1080/03610919308813143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfinger</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Covariance structure selection in general mixed models.</span> <abbr class="cit-jnl-abbrev">Commun. Stat. Simul. Comput.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/03610919308813143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BStat.%2BSimul.%2BComput.%26rft.volume%253D22%26rft.spage%253D1079%26rft_id%253Dinfo%253Adoi%252F10.1080%252F03610919308813143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/03610919308813143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.70" data-doi="10.1038/nplants.2016.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2735.71" data-doi="10.1038/hdy.2014.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.99</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D114%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.99%26rft_id%253Dinfo%253Apmid%252F25407079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25407079&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2735.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2723" title="Fission Yeast CENP-C (Cnp3) Plays a Role in Restricting the Site of CENP-A Accumulation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2749" title="ovoD Co-selection: A Method for Enriching CRISPR/Cas9-Edited Alleles in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2735.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4CloSXv8N8Q-Oa2AB6E5HjaMDqlirLMsd4xCr7g4n1Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2735" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2735" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2735" class="active" data-icon-position="" data-hide-link-title="0">Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490462" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_N1VrIzWq7wx31K6Y1ur67C2MsYxRoQyGxjZMFcPfJI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2735.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2735" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-x9gawhmDpQyibbDihlAzt7_UBS8sOunVSipreYpcAvE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2735" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2735.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490462" id="node490462--2" data-pisa="ggg;8/8/2735" data-pisa-master="ggg;g3.118.200415" data-apath="/ggg/8/8/2735.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0988-9710" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vikas Belamkar</span>, <span class="highwire-citation-author" data-delta="1">Mary J. Guttieri</span>, <span class="highwire-citation-author" data-delta="2">Waseem Hussain</span>, <span class="highwire-citation-author" data-delta="3">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="4">Ibrahim El-basyoni</span>, <span class="highwire-citation-author" data-delta="5">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="6">Aaron J. Lorenz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9109-6954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>P. Stephen Baenziger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2735-2747;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200415 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490462--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vikas Belamkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikas%2BBelamkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belamkar%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikas%2BBelamkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0988-9710" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikas Belamkar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mary J. Guttieri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA, Agricultural Research Service, Center for Grain and Animal Health Research, Hard Winter Wheat Genetics Research Unit, Manhattan, KS 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BJ.%2BGuttieri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guttieri%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BJ.%2BGuttieri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Waseem Hussain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Waseem%2BHussain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hussain%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWaseem%2BHussain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ibrahim El-basyoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Crop Science Department, Faculty of Agriculture, Damanhour University, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ibrahim%2BEl-basyoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-basyoni%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIbrahim%2BEl-basyoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">P. Stephen Baenziger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BStephen%2BBaenziger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baenziger%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP.%2BStephen%2BBaenziger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9109-6954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for P. Stephen Baenziger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pbaenziger1@unl.edu" class="" data-icon-position="" data-hide-link-title="0">pbaenziger1@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2735" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2735&amp;title=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2735&amp;title=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2735&amp;title=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2735&amp;text=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2735&amp;title=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2735&amp;t=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2735&amp;title=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2735&amp;title=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490462/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490462" id="node490462--3" data-pisa="ggg;8/8/2735" data-pisa-master="ggg;g3.118.200415" data-apath="/ggg/8/8/2735.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0988-9710" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vikas Belamkar</span>, <span class="highwire-citation-author" data-delta="1">Mary J. Guttieri</span>, <span class="highwire-citation-author" data-delta="2">Waseem Hussain</span>, <span class="highwire-citation-author" data-delta="3">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="4">Ibrahim El-basyoni</span>, <span class="highwire-citation-author" data-delta="5">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="6">Aaron J. Lorenz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9109-6954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>P. Stephen Baenziger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2735-2747;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200415 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490462--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vikas Belamkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikas%2BBelamkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belamkar%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikas%2BBelamkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0988-9710" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikas Belamkar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mary J. Guttieri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA, Agricultural Research Service, Center for Grain and Animal Health Research, Hard Winter Wheat Genetics Research Unit, Manhattan, KS 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BJ.%2BGuttieri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guttieri%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BJ.%2BGuttieri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Waseem Hussain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Waseem%2BHussain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hussain%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWaseem%2BHussain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ibrahim El-basyoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Crop Science Department, Faculty of Agriculture, Damanhour University, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ibrahim%2BEl-basyoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-basyoni%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIbrahim%2BEl-basyoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Wheat Genetics Resource Center, Department of Plant Pathology, Kansas State University, Manhattan, KS 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">P. Stephen Baenziger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BStephen%2BBaenziger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baenziger%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AP.%2BStephen%2BBaenziger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9109-6954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for P. Stephen Baenziger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pbaenziger1@unl.edu" class="" data-icon-position="" data-hide-link-title="0">pbaenziger1@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2735&amp;rft.epage=2747&amp;rft.atitle=Genomic%20Selection%20in%20Preliminary%20Yield%20Trials%20in%20a%20Winter%20Wheat%20Breeding%20Program&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Belamkar%2C+Vikas&amp;rft.au=Guttieri%2C+Mary+J.&amp;rft.au=Hussain%2C+Waseem&amp;rft.au=Jarqu%C3%ADn%2C+Diego&amp;rft.au=El-basyoni%2C+Ibrahim&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Lorenz%2C+Aaron+J.&amp;rft.au=Baenziger%2C+P.+Stephen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490462/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490462/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490462/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490462/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490462/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490462/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490462/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490462/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490462/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490462/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490462/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490462/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2735" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2735.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2735.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2735.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2735.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
