<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2757" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2757.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2757.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2757.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2757.atom" />
<meta name="HW.pisa" content="ggg;8/8/2757" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes" />
<meta name="DC.Identifier" content="10.1534/g3.118.200481" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Hua et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6724913" />
<meta name="DC.Description" content="Nitrogen (N) is a non-mineral macronutrient essential for plant growth and development. Oilseed rape (AnAnCnCn, 2 n = 4 x = 38) has a high requirement for N nutrients whereas showing the lowest N use efficiency (NUE) among crops. The mechanisms underlying NUE regulation in Brassica napus remain unclear because of genome complexity. In this study, we performed high-depth and -coverage whole-genome re-sequencing (WGS) of an N-efficient (higher NUE) genotype “XY15” and an N-inefficient (lower NUE) genotype “814” of rapeseed. More than 687 million 150-bp paired-end reads were generated, which provided about 93% coverage and 50 × depth of the rapeseed genome. Applying stringent parameters, we identified a total of 1,449,157 single-nucleotide polymorphisms (SNPs), 335,228 InDels, 175,602 structure variations (SVs) and 86,280 copy number variations (CNVs) between the N-efficient and -inefficient genotypes. The largest proportion of various DNA polymorphisms occurred in the inter-genic regions. Unlike CNVs, the SNP/InDel and SV polymorphisms showed variation bias of the An and Cn subgenomes, respectively. Gene ontology analysis showed the genetic variants were mapped onto the genes involving N compound transport and ATPase complex metabolism, but not including N assimilation-related genes. On basis of identification of N-starvation responsive genes through high-throughput expression profiling, we also mapped these variants onto some key NUE-regulating genes, and validated their significantly differential expression between the N-efficient and -inefficient genotypes through qRT-PCR assays. Our data provide genome-wide high resolution DNA variants underlying NUE divergence in allotetraploid rapeseed genotypes, which would expedite the effective identification and functional validation of key NUE-regulating genes through genomics-assisted improvement of crop nutrient efficiency." />
<meta name="DC.Contributor" content="Ying-peng Hua" />
<meta name="DC.Contributor" content="Ting Zhou" />
<meta name="DC.Contributor" content="Qiong Liao" />
<meta name="DC.Contributor" content="Hai-xing Song" />
<meta name="DC.Contributor" content="Chun-yun Guan" />
<meta name="DC.Contributor" content="Zhen-hua Zhang" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Nitrogen (N) is a non-mineral macronutrient essential for plant growth and development. Oilseed rape (A&lt;sub&gt;n&lt;/sub&gt;A&lt;sub&gt;n&lt;/sub&gt;C&lt;sub&gt;n&lt;/sub&gt;C&lt;sub&gt;n&lt;/sub&gt;, 2&lt;i&gt;n&lt;/i&gt; = 4&lt;i&gt;x&lt;/i&gt; = 38) has a high requirement for N nutrients whereas showing the lowest N use efficiency (NUE) among crops. The mechanisms underlying NUE regulation in &lt;i&gt;Brassica napus&lt;/i&gt; remain unclear because of genome complexity. In this study, we performed high-depth and -coverage whole-genome re-sequencing (WGS) of an N-efficient (higher NUE) genotype “XY15” and an N-inefficient (lower NUE) genotype “814” of rapeseed. More than 687 million 150-bp paired-end reads were generated, which provided about 93% coverage and 50&lt;b&gt;×&lt;/b&gt; depth of the rapeseed genome. Applying stringent parameters, we identified a total of 1,449,157 single-nucleotide polymorphisms (SNPs), 335,228 InDels, 175,602 structure variations (SVs) and 86,280 copy number variations (CNVs) between the N-efficient and -inefficient genotypes. The largest proportion of various DNA polymorphisms occurred in the inter-genic regions. Unlike CNVs, the SNP/InDel and SV polymorphisms showed variation bias of the A&lt;sub&gt;n&lt;/sub&gt; and C&lt;sub&gt;n&lt;/sub&gt; subgenomes, respectively. Gene ontology analysis showed the genetic variants were mapped onto the genes involving N compound transport and ATPase complex metabolism, but not including N assimilation-related genes. On basis of identification of N-starvation responsive genes through high-throughput expression profiling, we also mapped these variants onto some key NUE-regulating genes, and validated their significantly differential expression between the N-efficient and -inefficient genotypes through qRT-PCR assays. Our data provide genome-wide high resolution DNA variants underlying NUE divergence in allotetraploid rapeseed genotypes, which would expedite the effective identification and functional validation of key NUE-regulating genes through genomics-assisted improvement of crop nutrient efficiency.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2757" />
<meta name="citation_id" content="8/8/2757" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2757" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2757.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2757.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2757.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200481" />
<meta name="citation_pmid" content="29967053" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2757" />
<meta name="citation_lastpage" content="2771" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ying-peng Hua" />
<meta name="citation_author_institution" content="Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China" />
<meta name="citation_author" content="Ting Zhou" />
<meta name="citation_author_institution" content="Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China" />
<meta name="citation_author" content="Qiong Liao" />
<meta name="citation_author_institution" content="Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China" />
<meta name="citation_author" content="Hai-xing Song" />
<meta name="citation_author_institution" content="Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China" />
<meta name="citation_author" content="Chun-yun Guan" />
<meta name="citation_author_institution" content="National Center of Oilseed Crops Improvement, Hunan Branch, Changsha, China" />
<meta name="citation_author" content="Zhen-hua Zhang" />
<meta name="citation_author_institution" content="Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China" />
<meta name="citation_author_email" content="zhzh1468@163.com" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=I. Bancroft;citation_author=C. Morgan;citation_author=F. Fraser;citation_author=J. Higgins;citation_author=R. Wells;citation_title=Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing.;citation_pages=762-766;citation_volume=29;citation_year=2011;citation_issue=8;citation_pmid=21804563;citation_doi=10.1038/nbt.1926" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. Batley;citation_author=G. Barker;citation_author=H. O’Sullivan;citation_author=K. Edwards;citation_author=D. Edwards;citation_title=Mining for Single Nucleotide Polymorphisms and Insertions/Deletions in Maize Expressed Sequence Tag Data;citation_pages=84-91;citation_volume=132;citation_year=2003;citation_issue=1;citation_pmid=12746514;citation_doi=10.1104/pp.102.019422" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=P. E. Bayer;citation_author=B. Hurgobin;citation_author=A. A. Golicz;citation_author=C. K. Chan;citation_author=Y. Yuan;citation_title=Assembly and comparison of two closely related Brassica napus genomes.;citation_pages=1602-1610;citation_volume=15;citation_year=2017;citation_doi=10.1111/pbi.12742" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Sci.;citation_author=R. Blackshaw;citation_author=E. Johnson;citation_author=Y. T. Gan;citation_author=W. May;citation_author=D. McAndrew;citation_title=Alternative oilseed crops for biodiesel feedstock on the Canadian prairies.;citation_pages=889-896;citation_volume=91;citation_year=2011;citation_doi=10.4141/cjps2011-002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Boeva;citation_author=T. Popova;citation_author=K. Bleakley;citation_author=P. Chiche;citation_author=J. Cappo;citation_title=Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data.;citation_volume=28;citation_year=2012;citation_pmid=22155870;citation_doi=10.1093/bioinformatics/btr670" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Chalhoub;citation_author=F. Denoeud;citation_author=S. Liu;citation_author=I. A. Parkin;citation_author=H. Tang;citation_title=Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome;citation_pages=950-953;citation_volume=345;citation_year=2014;citation_issue=6199;citation_pmid=25146293;citation_doi=10.1126/science.1253435" />
<meta name="citation_reference" content="citation_journal_title=Journal of Phycology;citation_journal_abbrev=Journal of Phycology;citation_author=S. G. A. Draisma;citation_author=W. F. Prud’homme van Reine;citation_author=W. T. Stam;citation_author=J. L. Olsen;citation_title=A reassessment of phylogenetic relationships within the Phaeophyceae based on RUBISCO large subunit and ribosomal DNA sequences;citation_pages=586-603;citation_volume=37;citation_year=2001;citation_issue=4;citation_doi=10.1046/j.1529-8817.2001.037004586.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=D. Edwards;citation_author=R. J. Henry;citation_author=K. J. Edwards;citation_title=Preface: advances in DNA sequencing accelerating plant biotechnology.;citation_pages=621-622;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22765873;citation_doi=10.1111/j.1467-7652.2012.00724.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. Fang;citation_author=H. Gong;citation_author=Y. Hu;citation_author=C. Liu;citation_author=B. Zhou;citation_title=Genomic insights into divergence and dual domestication of cultivated allotetraploid cottons.;citation_pages=33;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1167-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. S. Fletcher;citation_author=D. Herrmann;citation_author=J. L. Mullen;citation_author=Q. F. Li;citation_author=D. R. Schrider;citation_title=Identification of polymorphisms associated with drought adaptation QTL in Brassica napus by resequencing. Genes, Genomes;citation_pages=793-803;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.115.021279" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Sci.;citation_author=C. A. Grant;citation_author=L. D. Bailey;citation_title=Fertility management in canola production.;citation_pages=651-670;citation_volume=73;citation_year=1993;citation_doi=10.4141/cjps93-087" />
<meta name="citation_reference" content="citation_journal_title=Acta Physiol. Plant.;citation_author=Y. L. Han;citation_author=Q. Liao;citation_author=Y. Yu;citation_author=H. X. Song;citation_author=Q. Liu;citation_title=Nitrate reutilization mechanisms in the tonoplast of two Brassica napus genotypes with different nitrogen use efficiency.;citation_pages=42;citation_volume=37;citation_year=2015;citation_doi=10.1007/s11738-014-1744-0" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Growth Regul.;citation_author=Y. L. Han;citation_author=Q. Liu;citation_author=J. D. Gu;citation_author=J. M. Gong;citation_author=C. Y. Guan;citation_title=V-ATPase and V-PPase at the tonoplast affect NO3- content in Brassica napus by controlling distribution of NO3- between the cytoplasm and vacuole.;citation_pages=22-34;citation_volume=34;citation_year=2015;citation_doi=10.1007/s00344-014-9439-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. L. Han;citation_author=H. X. Song;citation_author=Q. Liao;citation_author=Y. Yu;citation_author=S. F. Jian;citation_title=Nitrogen use efficiency is mediated by vacuolar nitrate sequestration capacity in roots of Brassica napus.;citation_volume=170;citation_year=2016;citation_pmid=26757990;citation_doi=10.1104/pp.15.01377" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=B. Hu;citation_author=W. Wang;citation_author=S. Ou;citation_author=J. Tang;citation_author=H. Li;citation_title=Variation in NRT1.1B contributes to nitrate-use divergence between rice subspecies.;citation_pages=834-838;citation_volume=47;citation_year=2015;citation_pmid=26053497;citation_doi=10.1038/ng.3337" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=Y. P. Hua;citation_author=D. D. Zhang;citation_author=T. Zhou;citation_author=M. L. He;citation_author=G. D. Ding;citation_title=Transcriptomics-assisted quantitative trait locus fine mapping for the rapid identification of a nodulin 26-like intrinsic protein gene regulating boron efficiency in allotetraploid rapeseed.;citation_pages=1601-1618;citation_volume=39;citation_year=2016;citation_doi=10.1111/pce.12731" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Y. P. Hua;citation_author=T. Zhou;citation_author=G. D. Ding;citation_author=Q. Y. Yang;citation_author=L. Shi;citation_title=Physiological, genomic and transcriptional diversity in responses to boron deficiency in rapeseed genotypes.;citation_volume=67;citation_year=2016;citation_pmid=27639094;citation_doi=10.1093/jxb/erw342" />
<meta name="citation_reference" content="citation_journal_title=Plant Soil;citation_author=Y. P. Hua;citation_author=Y. N. Feng;citation_author=T. Zhou;citation_author=F. S. Xu;citation_title=Genome-scale mRNA transcriptomic insights into the responses of oilseed rape (Brassica napus L.) to varying boron availabilities.;citation_pages=205-225;citation_volume=416;citation_year=2017;citation_doi=10.1007/s11104-017-3204-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=M. Jain;citation_author=K. C. Moharana;citation_author=R. Shankar;citation_author=R. Kumari;citation_author=R. Garg;citation_title=Genome-wide discovery of DNA polymorphisms in rice cultivars with contrasting drought and salinity stress response and their functional relevance.;citation_pages=253-264;citation_volume=12;citation_year=2014;citation_pmid=24460890;citation_doi=10.1111/pbi.12133" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=T. Kiba;citation_author=A. B. Feria-Bourrellier;citation_author=F. Lafouge;citation_author=L. Lezhneva;citation_author=S. Boutet-Mercey;citation_title=The Arabidopsis nitrate transporter NRT2.4 plays a double role in roots and shoots of nitrogen-starved plants.;citation_volume=24;citation_year=2012;citation_pmid=22227893;citation_doi=10.1105/tpc.111.092221" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M. Kohl;citation_author=S. Wiese;citation_author=B. Warscheid;citation_title=Cytoscape: software for visualization and analysis of biological networks.;citation_pages=291-303;citation_volume=696;citation_year=2011;citation_pmid=21063955;citation_doi=10.1007/978-1-60761-987-1_18" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=M. Konishi;citation_author=S. Yanagisawa;citation_title=Emergence of a new step towards understanding the molecular mechanisms underlying nitrate-regulated gene expression.;citation_volume=65;citation_year=2014;citation_pmid=25005135;citation_doi=10.1093/jxb/eru267" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=K. Lai;citation_author=C. Duran;citation_author=P. J. Berkman;citation_author=M. T. Lorenc;citation_author=J. Stiller;citation_title=Single nucleotide polymorphism discovery from wheat next-generation sequence data.;citation_pages=743-749;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22748104;citation_doi=10.1111/j.1467-7652.2012.00718.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=K. J. Lee;citation_author=D. S. Kim;citation_author=J. B. Kim;citation_author=S. H. Jo;citation_author=S. Y. Kang;citation_title=Identification of candidate genes for an early-maturing soybean mutant by genome resequencing analysis.;citation_pages=1561-1571;citation_volume=291;citation_year=2016;citation_pmid=27033554;citation_doi=10.1007/s00438-016-1183-2" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=S. Léran;citation_author=K. Varala;citation_author=J. C. Boyer;citation_author=M. Chiurazzi;citation_author=N. Crawford;citation_title=A unified nomenclature of NITRATE TRANSPORTER 1/PEPTIDE TRANSPORTER family members in plants.;citation_pages=5-9;citation_volume=19;citation_year=2014;citation_pmid=24055139;citation_doi=10.1016/j.tplants.2013.08.008" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=L. Lezhneva;citation_author=T. Kiba;citation_author=A. B. Feria-Bourrellier;citation_author=F. Lafouge;citation_author=S. Boutet-Mercey;citation_title=The Arabidopsis nitrate transporter NRT2.5 plays a role in nitrate acquisition and remobilization in nitrogen-starved plants.;citation_pages=230-241;citation_volume=80;citation_year=2014;citation_pmid=25065551;citation_doi=10.1111/tpj.12626" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Y. Li;citation_author=Y. L. Fu;citation_author=S. M. Pike;citation_author=J. Bao;citation_author=W. Tian;citation_title=The Arabidopsis nitrate transporter NRT1.8 functions in nitrate removal from the xylem sap and mediates cadmium tolerance.;citation_volume=22;citation_year=2010;citation_pmid=20501909;citation_doi=10.1105/tpc.110.075242" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. H. Lin;citation_author=H. F. Kuo;citation_author=G. Canivenc;citation_author=C.S. Lin;citation_author=M. Lepetit;citation_title=Mutation of the Arabidopsis NRT1.5 nitrate transporter causes defective root-to-shoot nitrate transport.;citation_volume=20;citation_year=2008;citation_pmid=18780802;citation_doi=10.1105/tpc.108.060244" />
<meta name="citation_reference" content="citation_journal_title=Nature Commun.;citation_author=S. Liu;citation_author=Y. Liu;citation_author=X. Yang;citation_author=C. Tong;citation_author=D. Edwards;citation_title=The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes.;citation_pages=3930;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=W. Liu;citation_author=Q. Sun;citation_author=K. Wang;citation_author=Q. Du;citation_author=W. X. Li;citation_title=Nitrogen Limitation Adaptation (NLA) is involved in source-to-sink remobilization of nitrate by mediating the degradation of NRT1.7 in Arabidopsis.;citation_pages=734-744;citation_volume=214;citation_year=2017;citation_doi=10.1111/nph.14396" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=K. J. Livak;citation_author=T. D. Schmittgen;citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.;citation_pages=402-408;citation_volume=25;citation_year=2001;citation_issue=4;citation_pmid=11846609;citation_doi=10.1006/meth.2001.1262" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=A. Maillard;citation_author=P. Etienne;citation_author=S. Diquélou;citation_author=J. Trouverie;citation_author=V. Billard;citation_title=Nutrient deficiencies modify the ionomic composition of plant tissues: a focus on cross-talk between molybdenum and other nutrients in Brassica napus.;citation_volume=67;citation_year=2016;citation_pmid=27625417;citation_doi=10.1093/jxb/erw322" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Masterson;citation_title=Stomatal Size in Fossil Plants: Evidence for Polyploidy in Majority of Angiosperms;citation_pages=421-424;citation_volume=264;citation_year=1994;citation_issue=5157;citation_pmid=17836906;citation_doi=10.1126/science.264.5157.421;citation_georef=1994028739" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=P. Mehra;citation_author=B. K. Pandey;citation_author=J. Giri;citation_title=Genome-wide DNA polymorphisms in low phosphate tolerant and sensitive rice genotypes.;citation_pages=13090;citation_volume=5;citation_year=2015;citation_doi=10.1038/srep13090" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. Mi;citation_author=B. Lazareva-Ulitsky;citation_author=R. Loo;citation_author=A. Kejariwal;citation_author=J. Vandergriff;citation_title=The PANTHER database of protein families, subfamilies, functions and pathways;citation_pages=D284-D288;citation_volume=33;citation_year=2005;citation_issue=suppl_1;citation_pmid=15608197;citation_doi=10.1093/nar/gki418" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. G. Murray;citation_author=W. F. Thompson;citation_title=Rapid isolation of high molecular weight plant DNA;citation_pages=4321-4326;citation_volume=8;citation_year=1980;citation_issue=19;citation_pmid=7433111;citation_doi=10.1093/nar/8.19.4321" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=Predicting the effects of amino acid substitutions on protein function.;citation_pages=61-80;citation_volume=7;citation_year=2006;citation_pmid=16824020;citation_doi=10.1146/annurev.genom.7.080505.115630" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. K. Parida;citation_author=M. Mukerji;citation_author=A. K. Singh;citation_author=N. K. Singh;citation_author=T. Mohapatra;citation_title=SNPs in stress-responsive rice genes: validation, genotyping, functional relevance and population structure.;citation_pages=426-426;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22921105;citation_doi=10.1186/1471-2164-13-426" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. T. Pham;citation_author=J. D. Lee;citation_author=J. G. Shannon;citation_author=K. D. Bilyeu;citation_title=A novel FAD2-1 A allele in a soybean plant introduction offers an alternate means to produce soybean seed oil with 85% oleic acid content.;citation_pages=793-802;citation_volume=123;citation_year=2011;citation_issue=5;citation_pmid=21681491;citation_doi=10.1007/s00122-011-1627-3" />
<meta name="citation_reference" content="citation_journal_title=Agric. Ecosyst. Environ.;citation_author=G. W. Rathke;citation_author=T. Behrens;citation_author=W. Diepenbrock;citation_title=Integrated nitrogen management strategies to improve seed yield, oil content and nitrogen efficiency of winter oilseed rape (Brassica napus L.): a review.;citation_pages=80-108;citation_volume=117;citation_year=2006;citation_doi=10.1016/j.agee.2006.04.006" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=G. W. Rathke;citation_author=O. Christen;citation_author=W. Diepenbrock;citation_title=Effects of nitrogen source and rate on productivity and quality of winter oilseed rape (Brassica napus L.) grown in different crop rotations.;citation_pages=103-113;citation_volume=94;citation_year=2005;citation_doi=10.1016/j.fcr.2004.11.010" />
<meta name="citation_reference" content="Schulte auf’m Erley, G., K. A. Wijaya, A. Ulas, H. Becker, F. Wiesler et al., 2007 Leaf senescence and N uptake parameters as selection traits for nitrogen efficiency of oilseed rape cultivars. Physiol. Plant 130: 519–531." />
<meta name="citation_reference" content="citation_journal_title=Bot. Stud. (Taipei, Taiwan);citation_author=C. Sorin;citation_author=L. Leport;citation_author=M. Cambert;citation_author=A. Bouchereau;citation_author=F. Mariette;citation_title=Nitrogen deficiency impacts on leaf cell and tissue structure with consequences for senescence associated processes in Brassica napus.;citation_pages=11;citation_volume=57;citation_year=2016;citation_doi=10.1186/s40529-016-0125-y" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. Stein;citation_author=O. Coriton;citation_author=M. Rousseau-Gueutin;citation_author=B. Samans;citation_author=S. V. Schiessl;citation_title=Mapping of homoeologous chromosome exchanges influencing quantitative trait variation in Brassica napus.;citation_pages=1478-1489;citation_volume=15;citation_year=2017;citation_doi=10.1111/pbi.12732" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=G. K. Subbaiyan;citation_author=D. L. E. Waters;citation_author=S. K. Katiyar;citation_author=A. R. Sadananda;citation_author=S. Vaddadi;citation_title=Genome-wide DNA polymorphisms in elite indica rice inbreds discovered by whole-genome sequencing.;citation_pages=623-634;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22222031;citation_doi=10.1111/j.1467-7652.2011.00676.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=F. Sun;citation_author=G. Fan;citation_author=Q. Hu;citation_author=Y. Zhou;citation_author=M. Guan;citation_title=The high-quality genome of Brassica napus cultivar ‘ZS11’ reveals the introgression history in semi-winter morphotype.;citation_pages=452-468;citation_volume=92;citation_year=2017;citation_doi=10.1111/tpj.13669" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. Wakeley;citation_title=The excess of transitions among nucleotide substitutions: new methods of estimating transition bias underscore its significance.;citation_pages=158-162;citation_volume=11;citation_year=1996;citation_issue=4;citation_pmid=21237791;citation_doi=10.1016/0169-5347(96)10009-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=B. Wang;citation_author=Z. K. Wu;citation_author=Z. Li;citation_author=Q. Zhang;citation_author=J. Hu;citation_title=Dissection of the genetic architecture of three seed-quality traits and consequences for breeding in Brassica napus.;citation_pages=1336-1348;citation_volume=16;citation_year=2018;citation_doi=10.1111/pbi.12873" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=G. L. Wang;citation_author=G. D. Ding;citation_author=L. Li;citation_author=H. M. Cai;citation_author=X. S. Ye;citation_title=Identification and characterization of improved nitrogen efficiency in interspecific hybridized new-type Brassica napus.;citation_volume=114;citation_year=2014;citation_pmid=24989788;citation_doi=10.1093/aob/mcu135" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. Wang;citation_author=X. L. Dun;citation_author=J. Q. Shi;citation_author=X. Wang;citation_author=G. Liu;citation_title=Genetic dissection of root morphological traits related to nitrogen use efficiency in Brassica napus L. under two contrasting nitrogen conditions.;citation_pages=1709;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01709" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=W. Wang;citation_author=B. Hu;citation_author=D. Yuan;citation_author=Y. Liu;citation_author=R. Che;citation_title=Expression of the nitrate transporter gene OsNRT1.1A/OsNPF6.3 confers high yield and early maturation in rice.;citation_volume=30;citation_year=2018;citation_pmid=29475937;citation_doi=10.1105/tpc.17.00809" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Wang;citation_author=H. Wang;citation_author=J. Wang;citation_author=R. Sun;citation_author=J. Wu;citation_title=The genome of the mesopolyploid crop species Brassica rapa.;citation_pages=1035-1039;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21873998;citation_doi=10.1038/ng.919" />
<meta name="citation_reference" content="citation_journal_title=Database (Oxford);citation_author=X. Wang;citation_author=J. Wu;citation_author=J. Liang;citation_author=F. Cheng;citation_author=X. Wang;citation_title=Brassica database (BRAD) version 2.0: integrating and mining Brassicaceae species genomic resources.;citation_volume=2015;citation_year=2015;citation_pmid=26589635;citation_doi=10.1093/database/bav093" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=Y. Y. Wang;citation_author=P. K. Hsu;citation_author=Y. F. Tsay;citation_title=Uptake, allocation and signaling of nitrate.;citation_pages=458-467;citation_volume=17;citation_year=2012;citation_issue=8;citation_pmid=22658680;citation_doi=10.1016/j.tplants.2012.04.006" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=H. L. Yang;citation_author=J. Liu;citation_author=S. M. Huang;citation_author=T. T. Guo;citation_author=L. B. Deng;citation_title=Selection and evaluation of novel reference genes for quantitative reverse transcription PCR (qRT-PCR) based on genome and transcriptome data in Brassica napus L.;citation_pages=113-122;citation_volume=538;citation_year=2014;citation_pmid=24406618;citation_doi=10.1016/j.gene.2013.12.057" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Yu;citation_author=C. H. Wang;citation_author=Q. Xu;citation_author=Y. Feng;citation_author=X. P. Yuan;citation_title=Detection of copy number variations in rice using array-based comparative genomic hybridization.;citation_pages=372-372;citation_volume=12;citation_year=2011;citation_pmid=21771342;citation_doi=10.1186/1471-2164-12-372" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="Nitrogen (N) is a non-mineral macronutrient essential for plant growth and development. Oilseed rape (AnAnCnCn, 2 n = 4 x = 38) has a high requirement for N nutrients whereas showing the lowest N use efficiency (NUE) among crops. The mechanisms underlying NUE regulation in Brassica napus remain unclear because of genome complexity. In this study, we performed high-depth and -coverage whole-genome re-sequencing (WGS) of an N-efficient (higher NUE) genotype “XY15” and an N-inefficient (lower NUE) genotype “814” of rapeseed. More than 687 million 150-bp paired-end reads were generated, which provided about 93% coverage and 50 × depth of the rapeseed genome. Applying stringent parameters, we identified a total of 1,449,157 single-nucleotide polymorphisms (SNPs), 335,228 InDels, 175,602 structure variations (SVs) and 86,280 copy number variations (CNVs) between the N-efficient and -inefficient genotypes. The largest proportion of various DNA polymorphisms occurred in the inter-genic regions. Unlike CNVs, the SNP/InDel and SV polymorphisms showed variation bias of the An and Cn subgenomes, respectively. Gene ontology analysis showed the genetic variants were mapped onto the genes involving N compound transport and ATPase complex metabolism, but not including N assimilation-related genes. On basis of identification of N-starvation responsive genes through high-throughput expression profiling, we also mapped these variants onto some key NUE-regulating genes, and validated their significantly differential expression between the N-efficient and -inefficient genotypes through qRT-PCR assays. Our data provide genome-wide high resolution DNA variants underlying NUE divergence in allotetraploid rapeseed genotypes, which would expedite the effective identification and functional validation of key NUE-regulating genes through genomics-assisted improvement of crop nutrient efficiency." />
<meta name="og-title" property="og:title" content="Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2757" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Nitrogen (N) is a non-mineral macronutrient essential for plant growth and development. Oilseed rape (AnAnCnCn, 2 n = 4 x = 38) has a high requirement for N nutrients whereas showing the lowest N use efficiency (NUE) among crops. The mechanisms underlying NUE regulation in Brassica napus remain unclear because of genome complexity. In this study, we performed high-depth and -coverage whole-genome re-sequencing (WGS) of an N-efficient (higher NUE) genotype “XY15” and an N-inefficient (lower NUE) genotype “814” of rapeseed. More than 687 million 150-bp paired-end reads were generated, which provided about 93% coverage and 50 × depth of the rapeseed genome. Applying stringent parameters, we identified a total of 1,449,157 single-nucleotide polymorphisms (SNPs), 335,228 InDels, 175,602 structure variations (SVs) and 86,280 copy number variations (CNVs) between the N-efficient and -inefficient genotypes. The largest proportion of various DNA polymorphisms occurred in the inter-genic regions. Unlike CNVs, the SNP/InDel and SV polymorphisms showed variation bias of the An and Cn subgenomes, respectively. Gene ontology analysis showed the genetic variants were mapped onto the genes involving N compound transport and ATPase complex metabolism, but not including N assimilation-related genes. On basis of identification of N-starvation responsive genes through high-throughput expression profiling, we also mapped these variants onto some key NUE-regulating genes, and validated their significantly differential expression between the N-efficient and -inefficient genotypes through qRT-PCR assays. Our data provide genome-wide high resolution DNA variants underlying NUE divergence in allotetraploid rapeseed genotypes, which would expedite the effective identification and functional validation of key NUE-regulating genes through genomics-assisted improvement of crop nutrient efficiency." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wIUG1DyLtqcf-yng6yzQsBd7GvjbYm6Uxn9lDP_vJEg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490444","apath":"\/ggg\/8\/8\/2757.atom","pisa":"ggg;8\/8\/2757","ac":{"\/ggg\/8\/8\/2757.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2757.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2757"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2757"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2757":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wSVR4p0BUPnNg\\\\\\\/jt7Hdlx4e+Y9Fq6bSFcvdPg0+Mp15fJTEJ0GFE8Lg99BlxIZYOZs9vBmtoJGW9ZRtAQP+iJ2XzoSwy70Dkb+uAH4LsXJBu7HCTEQbzm2kXWjzjvSAZ+vFz4yYxsUgvx0Q\\\\\\\/Xq78FfVThKwotrO6UV8sz49+Zc01L0wcVwA+\\\\\\\/6\\\\\\\/aGMu54n8zKZmtNiFagclQlzhssPaGc0GJ7l0HCCDG2oZgkUxlN6Fkx3bLMag2fbqk+RAfdkIcnLaYFKEbYwCpCIOwal2JLL\\\\\\\/8MInOlcEgpzry4T6RYc2N1VJMdvwcI8PXc0GL+Oi6kWhrKA\\\\\\\/wnuVEVxeD2xglEEo\\\\\\\/y3vH1vI8P1bMLidvd91bVSgn1XU2c0tf9lU1enqBtmtEwYGb4CKHd4zNMPn+WipAvCN+aKg2+6VDB7QZ\\\\\\\/1OX47PROtGgw2A\\\\\\\/nCugjI2uJbO9e4hstTEZ13bPk4\\\\\\\/nH7ST0CbjwPMpajePLh\\\\\\\/ZWTzJpw2QOnYpB9IsSL9z8LPnqqTzR7TYvIVAjJxcJdzV8aZwi5RcS3f6qz6n4VibfxHlwUcJEzf9T\\\\\\\/5SmpF4E1VQW9qK7wWUpWpoEp0japnAnpDONnu\\\\\\\/OK1w0lAWhYKqe+t1JCGkoYL1TxPjuApp9EPcaUBJJZZWOYaFTwhoQP508GTA2sV7Q39OGUtqOeUD+YKmgWPwhcLUDoqkNYk+E013d0n0gqRqsTA4NtoJW73tPp0n\\\\\\\/dVO4sXhQ843D7t\\\\\\\/OMzXQdJAvbJ\\\\\\\/pMK9zzVrIg8+56OV1Jbg8\\\\\\\/RJLdTZ1ZMmKorFnowZWL4mpqHdDBgwojbBvGtxmdmKdhEuxIyMAPAlf0BbxyOChVvOO0yYM4hfMJ0vHfb8uoOkN2sCLrxkx3BsQYDljYQKIszk\\\\\\\/7d6GcPe7qvMJa8AeBQSxs0ZTDeopLMaEiLTvvV3nUJyPsF4soX8LbPCSwK0uN8echuw11oefkHGDUvUveQugil6ShuuZfs82BG0VdfeACXrZbBowM5WKAz5a\\\\\\\/AnoAjpX+H40P6EFuSjfkP2AROM5\\\\\\\/xhLvvuBm4+px\\\\\\\/u0wIPrLjF2Sr8BcTjyHUJxDHiURvfEkddz7ZQzmOvudDeKdXeFwnUSeU0Znyctt2PH6sjjQNo3NiqrL\\\\\\\/vVgVx4EUggwSse5Ro9QFuXDHSzDQbQlvB2c0+15WEkyf2RJOj5Reuv+elTRIibhwtmwi4Q8jpfqEiSbf0CmP12VUB5dctx58\\\\\\\/L\\\\\\\/JSKvEm8r9FcEEnYrlBZM3J86MVDImUIeXLmDLyvCBSKjf9mYv0\\\u0022,\\\u0022iv\\\u0022:\\\u0022Djy2fN5pheh6yZGFsWANbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022deea834b3334282235e0a24638065f5b882bd3585d11de05d10abc9c79af558a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220Z4LBwZHz\\\\\\\/6IFnrSL8g052VQviGMk\\\\\\\/VYUEK93JLoRsUlaoxj32l5WUI+Mm6JQkO1b3KOm0d0gr1a6kbglwVgfSmJo5ppP0m9JBupaI2swuhRaT19w6xcoo9e+unnn8bBcWjzjpl1tqZRiXELWWBhWHQQ\\\\\\\/outLd0f7nKPajq2P7r9KeikfCH2p+83YC29Gs5K4VRjBKj8yFhV1sKQ4khyR72YcKFaMwAH79ZqxZ04mUcAotSmQ8GI\\\\\\\/dpmbEnFGJ8LdvDDd8rLR73nK8j30E2oh126b+qD8oIqSKrY+O10B4E0QbXbyqv+NFWyvOZ9w40Tq5F242YQWxL+XzhXNnZHybqIqmoezV0Yd78IsonxJBQ1iA7QtbtYQEQaKlW0GNlfYmSnsYOiAHI3mgq6QlOZ4a3p289PkDj6dAzf2gNBebi2XdPvcovDkJMxp4P9eQsNy7yiWfFrd2covcxBILIQUtkKgQYkBw8\\\\\\\/8SXMZ6EzdmZCY6Toj62Nk7oPuRrTpuzDEcVNQ2nZOr4tnIV9tJu4PUOG5KLvBO1OfVKz4xckSR9xa7HA5+k9+\\\\\\\/wwfpvHXkzEqIf+08\\\\\\\/izgaYdglAdzfgtyXEjPdSVHGbaJNb+dlo\\\\\\\/xpctW9eUEe3Vc0xiCVO8X9r+vrVBAn72kCtbAfCL+j5EFGNk5gnq1QBhxMbkzv7Cm2x9UW0CsOFrUqXNCxBCBo5K71nJlNcBOHIEAorhDPGB+VLWRUpX7REn1BIUaKsys1Mkrqv7oYa34FUSbRfNlbo\\\\\\\/G0rC+JzcH78k3bqTrDa502O3huDbWyfmFg5\\\\\\\/X9qsllPp+wDFPDmzzRd2WYupFLIBEnHVPEIPUPI2H0DjRCYJpAwpOPplaEJa0K8UrANDXA+7LOyWNhFURuKgKTupyf74Z8mQeOYeSahFzeTSwvKl3Xe1Nh790itoE5CCNuLISAutkBGVtyGsfnemrtKvTVbFE\\\\\\\/pgAHzS9oVMmNLsFKZFRl\\\\\\\/Ilp7j2lH4eHTuLcgH7KJrB+iu8vcFjyQ6uxYgZ4ygRixCzPVtgcOPeagPDEe2Mq\\\\\\\/MMmzCDdQC3mkNRAUFKv+cU4gQD41z0KD81cIrKATiWTzGRwCx0xUNlBTsDqtDk8RZ9lJpg751JDya6wIqjKgOToaK6jQg0CvuiQ6qh1YYJzne6RAz2WkkN9+aK+5ZCEASunkoozaRYtpSeWQYNOLPjhx7ZJcgWeMjdx5BVFya0bLI9pyDgb9DM1k2wtGadXle47mUwF+hJ7\\\\\\\/DkKOX5Gv2kODAC\\\\\\\/tYYbEMF3A9SslcU6hga1W1SOuY3MdMe\\\\\\\/fIQA8e\\\\\\\/1ty2HtUp\\\\\\\/EoWk7uZ+vQZ\\\\\\\/J7fw+6s96mzBWj6\\\\\\\/LzEgepN2jGY8355dctOcX\\\\\\\/EtHDMXLfcgCRiKXXWsecyFMYzI45tcycX8tZfMWCoN+q7X9mhZ14bVW66\\\\\\\/Iooxh3VioIGKuztlBQb6V71Mi4yE\\\\\\\/Og3548oDWeROgMceaViRdBkInMnQpSBJUKQs5XZoUDulCZl0G6fKxf2T5vKasQVD1r6D0tgjUYSXRE8UAVbUG8PtmRDeWvqV3sT8zdT7s4dVEpC1voNpbc+q4mmNaihkVFJxT4KQvqAZdE5U9fwfM+0qu\\\\\\\/ROXPbcqe1QSDvNcWgrAE09HI45baiFcAyyeZzD5OtvGJMixUmA4x\\\\\\\/2F4QxdXh7m8dJueRF\\\\\\\/ACPfsEVoHzUEPKEcPpZBMOs4ESh\\\\\\\/bgyO2Bmxde0xYTbxQQ0mO8Yl6d5wx1mBKA6tqcPDHFZvOvuo1ia5+Lr56S4EM4FhWe+Hg6AYASDS5DEY57Og2J+1wDejZrb1zXCfr+9aTWClgxSniuHhzsrJwiVkjqes\\\\\\\/gHjAT3ioJL9rXQ\\\\\\\/I4gToljK5HtiVC0Rp84rDxz31+veiDm1W3KoD1tybtAeaRi9v\\\\\\\/cYTQObkt6Wa50Y3\\\\\\\/lE7I9UfWEBCss0FmX\\\\\\\/5XUCAivUpau6Adq0\\\\\\\/rCjGtcX6VKfsjItKDjMM7jxjONf+6NZR6MFwODDOhJND9Adl49djMSu7CGBwcEwhy9EhnePjfGBX5Ad\\\\\\\/fPruO+1N182p\\\\\\\/KfzE2XcG2FbxoV5D3CA7sfA3E3wfdeSgn5bI5YtJK93KrUjkettkHxPw14Z4nvju40mQ\\\\\\\/iwoQW378laHqQuOGd4chY2pf9Kr2XbszEneGn4AWPGClw4WnwaVgkudqE2DfxRPwKGRh5KNpR+2k9KTxlhqF47PnMqXZcScU8Tg1OFvsoOzPt5Z+mBksB+qUErr3gWWYSRGEJMCHHHSSoyIpIQH58JxyMnjPU1wfEjWJ9tImA1uZmcgXQzUNVuzH80IjGcdQFKcjYzcKyD1tpce17ws+pzUDRVcrBjaBc\\\\\\\/\\\\\\\/VBT4\\\\\\\/GTCkmaeg66NRxzaZnUtU1sVSzTbh4GcwSsY9Oo9wEOGKmM7zEpkzGlcoEgIlEIqrOKq8YYCsLe+JtfK2H5iYVrmYxykN\\\\\\\/GWbY7r15LoEhqyzxpiaZtad8Um2wYn7UKZtrDX91xtzNHdHNEUuIE9RmEUlzVHGxI2XwMjUyjydqZs6ZMbKbKcoRaK+LGF+D4rcXz\\\\\\\/gqJV1RmtzkjnivfkazmAIzdCALnw819KyiK9\\\\\\\/MDtfcJoPooBWIf5UrEmmv\\\\\\\/1HyBu1+k+jbm\\\\\\\/HNZ6Lh5YSqR5Gyw08QrMHSohMlPKP\\\\\\\/NVH8LONkxQtYlqjQFekf60GezZ2ArJ5laW3PPsPf\\\\\\\/2q52oLHOZP\\\\\\\/yDg3jgJH92cXZfsF25j0a97didw2idCCTRc\\\\\\\/boSrvY\\\\\\\/QcOfuPAGOPMjfPJFgWJtvuSe4Hh\\\\\\\/SsGVQ0kRJiExXqxzOpxwomvRw\\\\\\\/HGNXxuj2VatROYxq0hVDkgqzXF2lU18Kmspx9aYiKbdB1Pa7XTqH\\\\\\\/aP97\\\\\\\/81qCse3cdeH\\\\\\\/GIKHHWzhTJgQ4dQ+0jVtTEbt1++L4CBrmQSjEo5wpGmTzy1d0OgqjM0UE7jTIgRY0tElsC3DjpCvCujR9qnk+n8MhEf4gGBcMRxc4FFrRoo\\\\\\\/+YNTbJNpvZxk91T6dbpNrvKCVMigGq3ekh\\\\\\\/ExGYEonCm7edydhOeGh3y6g8J2uJvau+VlF5fVgZWUlo4\\\\\\\/4CTuWWAbd525RHo9y5VeT\\\\\\\/F2qhW62GpBQ0g\\\\\\\/XuJdQLYZiBpfit8DVeQaFTMrysuqXMFE8tpOzMywPVLPxWt04\\\\\\\/KbjWHUZKejVBCPx545pcX22aIZWk2kDDp3yFMO2ZMgLdu8jigeP7ZOEPhK12JPtxP9fF3EJGoqhS5dHLqFazM0vCsHuyF6I3lCMGu1liHESjHHHi\\\\\\\/6KP0FIDXRANhvCDk387BUuLTtm\\\\\\\/S1sfenTFSdSCK3rQOauREFldHximZNC6NXDGqqhVdp+f9hbgklP2YkLkMmzm2fDeMnWcQo8vh736mpkZcBR9vwecsFkl0sJYUeGGWEP9Wp8JT5CbbhjtlI6R5w5PrGlRtcfhdHDyLl5tYi1oqNOirwTI95yQZBAYjc4uvXr\\\\\\\/HBkzcr+aeAddRfCc2HsLIAKEA3Y4hdxHnfV9PT7f74cJrTQgHVWxZeeA+axsKn3H1S51SNb2eojuYCp6mxIGVZO\\\\\\\/cErUDJ9cmale\\\\\\\/xERezI8Xf5OE2Dgp9Grc5UdpwwhvpnKSwBHWMzaFTOM\\\\\\\/YYFpqjQnyterKvug50xW7fQ1k95fzilm45QHbfkT2nRP5MVVIi8Lk1bfqDkgWiP3bjJ17D+BclDfUENjh6ZQAbui3NFUTx\\\\\\\/o7YHjSFBLFPTG9HGhaEA9xSabJV5XbFVy2cSBbbq9dcDW\\\\\\\/SO5RXRvcOvBWgHTM\\\\\\\/oKh1dOEBK5nDnLb3zko0ZMUSPnVAvUAhqcZD+1WFW+UhUynh60sfnPoaYnJjYm0Tk3P3ul0f+lQ9bgRF3Gw5pjkdi7HkrHdOWW4NJLPywWGU+xqekX7zmCzp\\\\\\\/uLbmvPdln8TZwC\\\\\\\/nK\\\\\\\/PNGt\\\\\\\/75H+nkMRcmPtmPsBI37Jf4OGwEJgGGM7FyX+K5Od49xeTAa0wHEIrlsW\\\\\\\/4+Aq5cYh8Dsh9EW6EwZD61cVGhtsnd29pJ\\\\\\\/r1RlWEK3+55YiCMOZhDp4\\\\\\\/ZI5BAcFO\\\\\\\/pGsLu9zmavpmB6aFD\\\\\\\/ikTQ1xiNde2oqqFvjVyjVOLqcjIqVdki1ep1G5kyvSJDoSFYD9ryA8NsE\\\\\\\/zRTxJJJTKa2SkJov2qcWJOMcK\\\\\\\/ik5JIEvmipz6CZUL9XtEI1dnehjtkZc+Zo76QjCwqrD6BuMCYwo+mktR+mZf4Ae7GL\\\\\\\/FBccj4O21tAB98BXcAhjqf4+042ga3k2o911Wn2FQqZlXKxhg0E\\\\\\\/ha3m7Wa8bBESKOa+6KfGmJX0\\\\\\\/jpMwkzs7VyVhYucbtKwtIw00WFNqw1sslUhqyXd6M6SeOwVOXNzECXqQnWT2eNE1Npk+ow92hO\\\\\\\/EQNCTM1O\\\\\\\/\\\\\\\/ghc8JV1Q8aCOpmgvrBapnNIf6MCA6xwFpDb2BQgLZrup8xC5Jusox9CsoPb9v7kTpsTCDmFqESTkXXOXki1IsuZuDZoHfig6dgJguHfm9dvfWL6WgqFVqTKdMFzesOxEf59I7H6wjWHABfcwyhDx6uYYyAl7j0onUMvmzeeIXYk\\\\\\\/ascVNo0A+N3bIMTg1pgn392bzktFCzEedQG4iTD9gb4KmwrykJy+xxhl0etfmRxxIUCsfngew7IGXnLC9vNasxZiaL0gMwC2a4o1ZQdBrnS7SuvmWM96NjSaDh86csMYgBRrDfCjQUg5yjcfw\\\\\\\/7YFOrR6FFS6PCU4FANyk8eG\\\\\\\/\\\\\\\/QLiKMIBFqYoOXF3RyDTeQVmgbnR4uMW1fnjb7sThQgsiS9zdiT+7s7ucauihGgdZi+TbMEif+rLrKAoLAJXeoN6gxQXKclpH\\\\\\\/JgC5QNZkbn1L89wsB+g\\\\\\\/3Li+tTvfRsZnPPnbDkJC223FggxxXHiLSjHv5DcORobjb3qqXndwpGBEXtpzUiufmPhERHZEXFXB+q+ITnmXIEZs8cchNbLJaWbWcobH9YPvrgBc2XyWUqBiOgMwcBqZfzF8a191ghL5G6+dSC44ABY7LvTP+Y6V\\\\\\\/GST4GzI+CcDqvxhDTHSPkE87dLAaPIbWx151Ra\\\\\\\/Fr9IH+ldKGlJ0Iu6X4lCn6zmWMGETv7RGt+nrpf+5BLIMIuY\\\\\\\/392YGIVcWkt0Izr1kSB05MoFLgtCyrcLAlbxaJALAeWDmYhAr6H2+As0EggpLmmUnlwhb1WZRY770x2cLZQceyiQB4j2RBlQyaxJGJF7EzkI0cQMfiZ\\\\\\\/aw7gdseTc5bFjXKfIRwEIKEo1nlAdFPhc9HplZc72hw4XBQsvD9QTrECHxvMeDqJnE\\\\\\\/c2Plc8QbkKQmzbnSoUD9KgKVYJ69+RT5lIkHkfDMP3j6zH+MhfMGw7\\\\\\\/q8tIfdI6H93h1O0NHfDLNYEDdm6EY27hNNjcmHaXn17Hp9aXXqy0Y9L5LMMgKuYeswlyb12+tn3+nWd7ljIGjFau2ShMAOi+AdOkpS\\\\\\\/\\\\\\\/Ts9JspoqNgzxSQ1ECEtZJo1skXKPa4MJ4dGnbdGLuH0IT57xWthjXFHOSRJ\\\\\\\/Um9FsRmR2RLYx1pakQpvDe+VpEuNG7YCeSac\\\\\\\/cn6+lSsSe1UbuatZIUOU7aDBQk1MBAeZ9i4OUfwhCTu6SY4JMvCfVqqQrIdYPJO4h2671\\\\\\\/TheEufffoMWepuOPW974pGtQr+4xD+Sm9bK8hleXoAAqL5DaYJj2dNYl+HqTwFvNgmEx0Mi1xKWWWeZ2zlM71Cvy5T9SrDY0QNX2G3e4OeRezOmzky\\\\\\\/2e83psf09LXEMWV12BgKhPcNfVmBHCaHQ\\\\\\\/5Dhc1tGHrgb1INLMK57G1b3ZsmLpvN9vPdu6vOiT7YiF5snun2QAZcUfrauV2xmiAWY9tNDD6Vy5auOuIGTvA8ens\\\\\\\/E4bJjobItsTYVR\\\\\\\/jfaTgCgYN+WXuGXBhgEKmoaJl+6UYMro1zh9o+jRrS0yAMDiYRPcPex6KBw6sEIqOCRoo\\\\\\\/kMlb9koFGp2iE4nk9ROBmJFVDXjX80o2FWmnNEbhWoXa88ZWHXxYDUI4\\\\\\\/zvVTx9lkalpidmRZmxrXEqxC6jVt80db72e\\\\\\\/7jcFe8INVWYAbam2HmgMw+7HAEDzokKRGZ6cqeOz5krCMzYSf8h1lsv4\\\\\\\/jqXN8hk5PXINAjWMbXakPMXQJMidkbVunc8uP1E3fKtM\\\\\\\/LTYV16JfPKJSxLDiC59l14Km6tGN50ZYrB2YbL+Bs50EMNXlq2Zho0D10tqQToGV0GohCdcFxrM6SeptpfmrUkSVzuIDCztY75NFnUHgKUryUtdPIgfl+fka5FHQvZ9QvXjug+reXjVjaXdRAdNi0ICSo5nsPVs5dq+QcbQSyhGzrgU1TQoRi7\\\\\\\/qrvRltn8Dw\\\\\\\/v1fYSMyBBJRoyqD4BZzQz1ZFoByVVUjk0P+2SFMesDq75+ORlxC6NMe8d3XUbibdq3aiGYRFnQUIOaJ1WKH9FHGnBz0fzM+3qe41RBlzuADRtNxDvYSiOklC074DVbEks8OfseW21kphCmJlgynGbYEqCJePdDYVWL4iTFNugtAxVEaCFLZHuf6NINgMsOuRnpI2VtlLJImMYlxvrWtU3iM4TZwnWOfvqLdF9rNXtPw32Z+tMGYY4Pr1ej5XJS+KIBumbQHVx1mv12x6mv7KoTit8HTDaiDgUwesycqHs1Mnfy6I6MWjBzak4YjdNZflGrM29QobQSME2OAtDzCoyTMuNMYbxT+L3qY1q3tN+9RqIPl2YtOeqcEh\\\\\\\/CTZ+kmaF5m+LEk1eKxGsa3Z8A39oKCbF5aL2smFOJiZTmUwI3RBh9i0OU4kJ96AjV7tBhIErEdZ78p7Pag3NewOTwD3ERmg9jRHXZy++4Lj0nM8PmDcvDBjzdpS1BDSRGwJJ77GKgzRg6nQqfqWwTaUrmDU30kW7vpvxxPfeGEQbGj\\\\\\\/DuSCBRguY5WEDIhaNdAylw12IeRLpkLvcht9Ctxjtfzfpb\\\\\\\/KpHZoYxS6pKwwAE7mwPKngfSlxz+USnLnG7DmFCn1PMypiwzffXJZFSmXCkkbgqjIg\\\\\\\/EgkqK923MuinVDfIPOgx2wl+OnjsSWaked40f\\\\\\\/JwqjRYz0x+HN\\\\\\\/Cy3DSieQsdcc9Wu0x1n+V86V2IKQPO694mKF3SvOSoODjGg12eTAXa9b+s9dapzB9CtiHD1NBP\\\\\\\/70UqhWUm+tejaD+N1uAyUDiFt3LdUFlwaVpnzqQqd3AoXgRKEZzUR7ki6HpAehjYo8tEgMbwsB0+laOQUegF+7EqVEyflPy+lVy4dVLGIGsDtLQYORarYu7oHwVyq\\\\\\\/wCL\\\\\\\/iCIhvjOgIF6vsV27s2gQsT4W3AED9ItL7\\\\\\\/QPZr2U8HYh8Mzt1GdP8U0fA75VfbGlTaUW4nyEAKMWiDLjlBg4TuPspdzHatqxAErtPeaADTHxuF9OvhfGqtwAUbvT6xipxgGqEA4xBo3KQpWGeCj\\\\\\\/nX2nvlKcfbNfvARiHqAJdMmvBOapJ8TNoPZ+c9XEAGE2U+L18TPq9knHfKuIwuehpJY7uUO9Ua8QJNVr1JY3f\\\\\\\/TFyKZzQOuD6W+t6Bpg2eCwm7gFrFZw8DJRA0wUXw71MvFKDcC7133+YpCaUJg7Yf2GpPGcWOIIYLfR0wT0XnBKzRuvR0nymjIZM1JU05eV+jxwNOacfSODanK4E7tb4RqwAOwxg2fK03BX1FZmKzGxIq72Hz\\\\\\\/Xt6GWWYMCQY7N2JGi0njX\\\\\\\/ZMVhTDbRtTD4t+Jw+Rqj8FQHcW\\\\\\\/xzhPXGyaxajlFa6jbKqWQIG+K4s3Olnny\\\\\\\/0a+bjBz2Wz\\\\\\\/BZ+NTDEPqQ44oRuzO8lsIMB9PnlZPqjLDU4g\\\\\\\/txC0xd9eb5K\\\\\\\/1GfruPLpNmXF7AKuURzin7NrwYk3RMZ+WXNajeSA4hr+oodswYOUkAeI3eXyjHywdA+SFB1bBRriEM\\\\\\\/TMKGMNpgmD9rdLKrHT68q8sg7MbAL9Ytsl9d3HNsI4mb475UnQYBxibyg1SbiUqbYq9vhJdGdGydQAfiS7hfxdhzCJezrdSpBWCobptsqhjxu+VjwSBcD40c4rAlXqmyIzNUsiRrl3GYq+1ljvaTpclXXi9WOaQTgKRlFhS9WL9INw\\\\\\\/ajXq4m62yQJ76Y4Y10bUNbeHiJ\\\\\\\/7Le5hR6WQaj7OLM1rKmkkZkz8L7h5VyHpZlcYXmSx7wgr3pOjz1p8T8LnQeQcBECvTWChK0RXP\\\\\\\/28d0ciTI5ebWTiIS7T5mpH3BiTDlQXZjuzuYrm8ILeeE8PQJ3i\\\\\\\/Zz\\\\\\\/OVOYAZhzambAsa1XSnuAWIC56kJX1ja6h\\\\\\\/jWuAEC\\\\\\\/jSIzP95X3tWbRleSIz\\\\\\\/j8rKU848q1m6CH+\\\\\\\/TOILB3\\\\\\\/7MvWmGJQaI28IszAN3fYC0Krt2\\\\\\\/K6buwOEG2r2Heulpb9rxmoyT5rQ4Sbixs6YTleGH7yfP5jTEpbvGJJsn\\\\\\\/b2lCxjL+Ki54BeeoLsTeVWqZnRtQ+wsatyVehPau0liOVTIqu0loPXcnWy29LlyZuBRnsJrt8FJYmOMUIHNdEMWMK2JMfScEEDrnLX9bctUMuCWQJ1sVaKyUe+sIc3DC2T5ewEWf07Ws8DtCYgiDbv\\\\\\\/D\\\\\\\/J45GtnukLuStqCjlI2H95VQZcuxidSk1yGvMzAL8xOp1vl1ZkVQdyct35IuJsr3E55y9WQhFsVK0cwRV6pFJ3qpEs51TFAk+KKoPvMsVouB0I1iH9fMm\\\\\\\/Nx+EshofmUUeWARkQSJ2IsG0rM4SL46rR3TM\\\\\\\/nbK8hG4MF5tA7aUybykYaqs\\\\\\\/Aw0LGxuQxW79+KAnqh8P4YhnVKjUpfn\\\\\\\/rcFTBb5phx\\\\\\\/+mwLPi3ARLkaitlL3BjshbFWfTDQQXeu2RFC4nzcXyLPI1nafowjHgYuW8TlC+nXCggMt0vKphuqcPDrzP2LBWEa\\\\\\\/e55CXMo8PppZ3WF+xe12q3JU3e1V8T6bHf4HvT5SH1UIRyuw59N8mEOPdBKDkIBX7MDwzN182X1I7VrVursPv4mQ1VE3COBOvWV4xoKy6g4w6jJDlBExIP13ni+GbN\\\\\\\/3CBA+f\\\\\\\/h+ke5rJQit+x0htjHPm1kYSyGzG09afYfFJWbuDj8CBVj3G047Rh2ITxfrobDl0DKAvn4F+QphEoZRnzzKadY6OsD7\\\\\\\/dE2t4pR3rtabTjTqxX5hB1lA0olBxonm3z85+E+nQWUYD0kKuflxJNAiv1b6mwbPOSuhzvVVpqNuWUTlezQKZopkFuWetGK7hy3\\\\\\\/U5H7UMLTAoDRcWj0osjylnvFbovl8cCjF3teizx+4lJsCi8Ox7scCvplNC5CYzQA\\\\\\\/6seriGStXUi\\\\\\\/Sz8sFXxGp\\\\\\\/UERwT2wnEA79qRStv5LZ9nunGMSTyxunSAK9HAgawWSJF2B\\\\\\\/B5NgR1HBPsiKrSZV2D1\\\\\\\/nTz8e5I95H+sv+cMnYB18so59rHTwPaCech5CXEkdFM80EFag8gjWHhPW8uKEimBetgLu2roHqgmnmRufb\\\\\\\/xKiHRjEFJa4CI+ToM6\\\\\\\/W4SlBlpxMnG6O7eQ77kr1lwhPKx8imnMAhpTx9PgBTVY0KT9TmKTjElHRJA8t6QZeD+nPLEyUP5NNiN4Fli0y16NMelmhVN210zYIAxguj0DZf1i0Gsztkvteuj\\\\\\\/QlL1BMAItobGDPPXWr0KTZBbBdDIXr6v9+1W8sJG7SGiNh6bMvYX9meVOZJ6LDnHZwuPV3UdTHyihJyQsmkuM7oJtjxIC5TNuc222etp13+zxgasyhM3YTcc8nPIL+FEkriNQ2O+VhFS3cjW\\\\\\\/aeZP7zQsN0XeSwNHo5+4jywPhlOXUROC+A8g4IkExo+lIzAgfQOwqazOzuaGhqHhVxCPlTlGFLgFmslSp4HUcqco0Vn7ZR2y4DiyyCkuxrwahcOPWL4hspZQzBcn\\\\\\\/7ONbQx9zGJ0ROaFQuM9KgWlJ1lx289FOKm+rvjfaQSd6haX3ccYXN3DDvhNV7cp\\\\\\\/mgd2FZs0FkzvPK1j9lGjDaqt2W9V8wC6n1N7NHUcAhhFuedtRcwessqGJz3A6Hid9nXtcxMFEcr0Bs1YxuhDlsLikSit0aEpIdO416Wm4sYqBMIRcZQPaux\\\\\\\/Bqaa4NeXpaZbQecgDIy1mCWDZOIYZ8gnxERoYra5zYbYCTxDeIvsnnBxE0EG3me13\\\\\\\/fgwqARh3oKZFEjvmW1XycioMlypT7qkt\\\\\\\/aD8du0B+UeYMBQA88cb3WesiCG0zPNQwmwNUHNCr3l2c4beL90YU+71Ym5doNB62xlH+kNFqrbWzluSe8sY7GT8l8NVqTjMDl5x6QrzvgyFZ7B6Mx64T3UuCaS6wKVxY5D14o103nyzOOA19ao8\\\\\\\/Lm9J2yQM\\\\\\\/6FXya8PhuZu+KHZy9CCdcegxwkDwzHmXLMWdxeMe87QtPUKcuHIocKK6unAxZdJwHvOPnhfeNYLkRR8fb0d6A1bSp\\\\\\\/JLwbBZvV+GHOGKn\\\\\\\/Jzhmk+KniiIJxgGyucdoK\\\\\\\/SUAVCA2v0zcWEL3ukEvv2lDaJKLv\\\\\\\/TQHRp0FeXX0nciPdgqoMHWdovUdcU0kd6bnEFOguSWgSoYttIS0PpWDtxwYHF7VZdtHTvLTS\\\\\\\/tPtCL0mqv\\\\\\\/m1qYiT7yzo0HMqGNXmTm0hH18g3DldXnjuAQSya+D4MKAPY4zWAXIHGZFirjMQkJwqYq5xJmcIc8QX6BD+qn+U5Wxy4gBcEQY6\\\\\\\/tcWzqHETTmNo1i3l\\\\\\\/Yu74ZLOLrQVDMn+aVbSm0GNlWzuj\\\\\\\/O9WDpKunqsFEhzYvmDxD4VFFThny\\\\\\\/gVYDjvVAJ4Juun0wzjsswKkIy2yOlpDWB9fx5Rg28il4jLu30N69mNZ0DYFpPf4sGy\\\\\\\/kLwRW2v22TezibvFu4cLMMChLkp\\\\\\\/HWdOApjd9+3DLFWHZAoZPhJsV7NPQaxnBUt02sBx+bFUR5zYpt0WPHy8SdyshPYqSvTOlD0QHZbswG3dmARua8mvql1oZcXSwN5SnXxWc8SoMqe7E9GReIpAm1LS+cxYvmhFFt27qJHfpwoeR7lwNWty5pmDnVNdZFMt3UETi4HxiY3S\\\\\\\/wukQ2jdJSRhugGsQmU4sO38xYu0Off4FroAVssLXTkk7TIiDCBCNqyyi5PpT2nRNnxOhzc8Bv7K6TdrQmTsbx9snutM2n5JoPnpRyi2ui\\\\\\\/o1+BKaNU6rUud5R\\\\\\\/Yn4oTe+tl8XI6f0g9TuD\\\\\\\/bR0gLwYL\\\\\\\/\\\\\\\/Qjb9qmSTKVCEaWjFJD+EiD+8GcUw+Hdi7EyDHNH1LXbK\\\\\\\/aZXqZAJhQXNJ37HrSefhicYahwSuJxzQtjrNgZMoxzGUXqubxB2TtVqrfXbxbo7kNEHNDlA7mTajT\\\\\\\/8xH5JXgE+MDMeSNOhJBnua7jQARaZyjxPa\\\\\\\/\\\\\\\/\\\\\\\/b4Sc2PYj6vX4683P08gUmjLOiP5AqCLekNh1qR+7MB07Hiskp4kkjeQPFsZlD7WfHUrT32Lq0fVM477aprU7Ya\\\\\\\/TBocIly8CP43nqSY647wdNluyiBmV\\\\\\\/i4nLhssR5mXlFrlJXtXueFWlyOGDEKQlhQRT69bibPMIASyNPdYer\\\\\\\/hG7ptbmwm2tU52c5rSZ1qeUe1HW94cfBToARh0XNvaSZMla2ejGckTDb69Tk6uXqLNN\\\\\\\/HyMc9eizZ29zyAdjqe7RfNnlXZdJXjWL\\\\\\\/3p8g36Lvv08isHXrV5BYvfIIW8ralkXw75L4UWIUdkdG9Y5XLUnMSKW4668HF7hJg40CrrX9uEs+DiWAQjeuB7bRiJCQKAabqWmDwtwnVAIqo5uNM4IOkbfHDEH\\\\\\\/0i2nH5zAI1HbmZn3bF+e3c09sE7noevobvamQcVv+fuIDmVputlgso9uXECI+gzF0E\\\\\\\/2n3tIywW56fnRn0KZ\\\\\\\/M0HmbPoozqHYea9aDCDifW2CV9pQ6+lZWIZjVcIMCYyYklHqB3UghZHuRsPeWsiqFmVzgo82tV49CI+XhcxzJ3RTP1p6to1bP1yRoi7GE+y6Kxx1mXhfSuEyOSoiMcCCvvDqdt27Zf4CPWN\\\\\\\/VhV\\\\\\\/gAiLbwn0E13n+tX\\\\\\\/WzAcGWmupcWsz2sFOd9fIIHYw1nna\\\\\\\/sXTuKyBLlg5UYgzgCo56iDDUhNGB5xDI6goHk0auXPGsbO\\\\\\\/+9BJO+RTu8DZy1YE1+HcSMJox7rZ6xsDETgk9QVlBVtKsXIADpNOanLjY8SNCB6N8e+eS0RsKlG54IndU65k03AGDsdlY5xFU1Ljw4QvOMuqmUJdsomSNEXnmtaDcZRaQLtsrL4b\\\\\\\/6L+G6Vn34M3Ri5+3o8\\\\\\\/tPWDOSXe7dtHMvr7SNFJxtQKx0s0+z1HeEV5bdcZJMppcf1ezyleR2g6Sa4bR\\\\\\\/mwzNjaesAPzE4E55krFQHEYw9sJpbueODgv51YLngzDglBxSraJUEDOQUzuEhAp5VA5+KkmYFmqM2s8AxtMSq9dyx1SDXlV8itbD9T98bdEVr3nTaFacheqR3x9GHdadGgFtlK35AHtapr4xpogIM8rOyCSXhEoArjRhmRS\\\\\\\/\\\\\\\/qaNbz4YAhS2+TiY+A5xFmAAdK2rV20TB2u++t+ObQtI3gHKLEVUsJAshIytx1JrMSqNsvSs27r61hVqGsUQARgVZ\\\\\\\/RDNc0S0G977XMflCGhNAfXzxjo8ADjQxPvb1ZEeVYjEHklsjZeVlXbPZiq04rP60FxpoWuGk45eNROexJdspdWUyeuoXx9tNm3vSKV+vX9hIJ5R20EaCafEdMY+lMR+TuRXZWgp7WzFCTUYu7SnYB\\\\\\\/ihmTk3tjH8WqkOET8xYjsv+kCCUdArM7U9b7tnnnKKxzPU8C3JXguT94Winbqs4FEqF+z6I\\\\\\\/xY+6dd2rAx7lNU0jzFLfEaPoBjalY9lMepVEolyIeoyUh\\\\\\\/HIlIhoylsaBjUEuOXMLPywPS19WCd30Wxe9S\\\\\\\/MMgPHl1wUS7dtjFq533J5J9t0AefAoExfguOrQTFS5+VCUGNhEmmL\\\\\\\/IsOZWwhzR+0g9w+BqVHcgcX+n0sLZOuZQ450rOEugoVPSA4McqkPPfdZ4yvcEKgEVxHLxbksk8+GIutiDG3kif0p9hUxadlIDZnxkEgG1Yy7eI+g8yZvDPh3CQsUemRUJmpaFqLkunFbKLBihV30+zRhS2CIgZCsLEMsl5TBuQtLWzeYPt03Q\\\\\\\/i3UTH6ged2KygUEh34AmxI73zxibPwxB3VzA92Hi4PI7iqhaIoH8hduiU46ET0EqYMKPAHSkysBEfXV8naFXzv5jRt1mcxLwLQI1PFA2x6Rz23oAQ16E3tLyX7UZJw2TzyPhCocjgk8tOTDxqKAlS7yswssCBK2N1rCdMzrcb+ulKdaVgMTGOtYAzh\\\\\\\/7mlcZmFkwdF4dkkapfft6dJvnWwXRSa+E1vMAlfJDOmxFpS3KHIWzFjPCukdnJ8rO4zJmlUDj0WI0Fl4DXHp0+\\\\\\\/p51tflY1r5omXO1E0C+6hjkoaTYjGMCgFkJpXevVjx2LCySPkhpaf6wc+8KVKB16ArKYoMJrC9\\\\\\\/QSULQW0O0AwOCiX\\\\\\\/DiZQppcY+DWSRjNBAYv2uo0AJA9lasb6jQolAToczCw8mhrjmbyHVBTdr0RJOMmr\\\\\\\/W0EeuRNr69QdX0yfbDzavLM5Egtjfft\\\\\\\/10j1uo9Q0N9Vhz6upWFsmWAKoUdT6r2sae8ZViezgBQNakmFWqHjxyTkRKRPKADy96my3IDNz3whXLu9LLPMmn8Y6Py5LJxNG+f8A71cLEJM76qPhyWHOPfkTOwcjlCxAVZpXVKSP6KWW30uuf\\\\\\\/p1mwmhIz2i6sU8A4xBPi5\\\\\\\/Np9g6qq5dOHvyHosb3waleINBjypIjEdXPnxKmAiYPwlIGxcQFVCtwYPLMqXe4b6ou\\\\\\\/Eo6EzUfwiZo2OJBM\\\\\\\/z2knoIFFX1jmdS4Rvd2c4kgptOE5yQZciwuFWdKd24mmm1RBqfjNWUhMKqXacw0BhJXY83uUJv3UFrtM3e2UpGhr\\\\\\\/vOk9k0QuBxhDBOpudQboB45F9\\\\\\\/LRmSCioK8XGbbYrBABy3KFMsF48L15ZQNy3GEqLumbndWIOtgoRMpiVj8edcW11WcMVbX1QyWXoXYhOY2xhilGs+FGo1SOjGgd8SQV6UnjheEWXvBPeX6K1gsbsHwM6XuNt8LH4So5hO7ObJIWVnVDVP8diDJTvYX3b9CEPrDZBGYuEzfdEnjQPtI+3vLr8rSuPsJY\\\\\\\/8XPDbBRki8ifNsjm0Oi0BsmuIDO8hzz1DzOLjLdB9LELoKDU8GLUtqlJGHS0536klbUfkpRSWYP8BwAwbBZ\\\\\\\/5TlxOdHlaNbkWZc99Q5V9ONQUsVDRoYD6iUlsjIa4Mjn\\\\\\\/+4seD\\\\\\\/z+7EFz9l0wcD0ikHcQSSOJui3hgP08kQxlOva3yi8mCN3XC4EdDZuMeyrkbq8bsVQFWSdp9uy0yzcM0pQ53bnDqYUsDExTNzNrPCnz6EGWauwbZ5oeOjtJLmG2KoGIz9PvPBFAc8C9O5r2\\\\\\\/VVqQ8XNaAZDFzVgJaHefzIvKIb43jvmGItHTx\\\\\\\/YD\\\\\\\/sGTqelCeVz4xNtY04c9B+7PYoCGjqpK8TK5rNRc+S8tZeB5Cq6ZL7kleYeXw8REpV9JRCq7StwWj6fkw7ylNjTLhouES10\\\\\\\/hW38plTwus0xNYCrEbaZcf+SGuVnEPVNecTSi0YqM8hyiTf\\\\\\\/XhLqfof4qqzEEoMLuVn0QEH8e+15no1AzM6iehMEg8xLHaujtaMZcgLQAs+b+vKL0bmhRA8td7ea7LO+SxqnVkfHs5Tkia6yItFrX7GjSrCtgIXRHXk\\\\\\\/rGeI4M84Bp6WvCE195m1lFKsFniMMXAhA8LAQPzg4JTkbH9MeGvCKeZTWzM80YW3wBU7A9H5I8IkJvMNdSeEprtqCcD8Ml8WwKacFoTlIRyALCIV2Wudv8bBpppiMa524hSP0t11TOckJitoh2Q864s8qjlG6qePlIXT39KZ\\\\\\\/m1KVBKmTyxsFGEBOCVnaW3pncH06VRV\\\\\\\/wXvWbSaPEvqQr5v+XzjDPgNYd4efWS21uDLuJsSqMRQYOdid+4neFumcNZJwc1iAbdwcsOyMBVGEY2XrwmvEJJO7yN+b3el6fcb30I7EqwVgO9ENUFKs0JShZscA9NtrpSKzkFwR1JDPXfTv+8EGQ0srINGzGpcCd320hpf8c1FRbeCgVTpT2\\\\\\\/XM7+vLUGICDVHsHuxjurE\\\\\\\/dv\\\\\\\/jZ5OiK4A1F6aTG7LKwUHu1N9uEqYkE1cd5yAORDXjxLEwalVkM17DoS2WQO0I5yWJy9SPBavOKmv6E3N\\\\\\\/zNSbXDVmFAfOaWcmShAh93axmre7Y9x2iqez5DLR571WBbEzI\\\\\\\/FITACf9uhfaBjHQeD6mr3y0r8tCiW6KeQxlEqo2LjpCjcG\\\\\\\/HJWnT56rW9NQdhUgUD6jJSInf\\\\\\\/e2ckDa513D7bxlM1pcMS\\\\\\\/+rdM2FzYRV8Qh0miV82Uh4LKDg4HQAesU0IpOBp22p00zh5kQzxMTGFVxe+qyJVvUenaO+FoOhGfmYFZpnxlIt9BZ6HLw8vCivS6KyYxc07zJ4qa\\\\\\\/xlSzdT9zKGAZPU9u\\\\\\\/aJolp8EzCoX6YlFNA96IHEOJxTQcI1RtSjvUcYXeF1NaOE6L+UQEuG73LK832xTM6Sb1YS2iePlE7q1fJaNajUq\\\\\\\/yrkK35upvq49Zp9e4c\\\\\\\/mUXGYU0xj1mIajpvxr1dTuq3Xk7yQvu+gU8weOZcd3KBHj84Dv\\\\\\\/tniQxaqGZn9krw5Dn2PSbN\\\\\\\/qijNMxy\\\\\\\/IjlByi0hCoj+gwid9c1R3+vIFMDiCBlDdKXz9JiT7zuxIT2dD8SvuqZWF1nvHJ9MmnQPO4onj+k9UOBBsc4GB4uNhZ4P8KtiSzJj\\\\\\\/fe\\\\\\\/IDBFHDG9Zm8r6NEITgjzib\\\\\\\/Rfu0W35DYAbXCXs7GgBNNjJRupFnv\\\\\\\/oDJLjn9HDsaLHTC728YoiZyztwhfo1UVnuZk4pI5O\\\\\\\/0qpZsscVJID0Xed2p\\\\\\\/LQuy10UU2OY+HgDnpiyFJM\\\\\\\/GXtwf8lDiDncWHlV3Qo6KqFpt2fUEkZIoEdpFJCJ8kL3C418lg6TC\\\\\\\/pcMrRd4WEDAr1A9Urq5AV\\\\\\\/6XTZNkOvkdHPhgRyU+w4Fu7bkMejiBLsxq4vaXbOg87F\\\\\\\/AqI+mh7vjrWRfGedgW5LeybjZclLxhDwVne1v9idGluBfufuempKN95sXcPsoA7jSXqJOOrYvMOln\\\\\\\/Nm7ECO8xOIHc06a1kGQPZGq\\\\\\\/XzUA7K\\\\\\\/jWY6dKKr6NgzN7NoWyHwSnw850sqbbahvIHCGv5iam01bAIwWZbrN67KfGCGbcegfo5OI\\\\\\\/N97nHBbGKkSF2Rb+VnsF44WIbOUInOhPLknfcTqw1sgREN1pmExHA4pyP330et+iI90fOuZbQUbl6RIWWXaHZO4M5a3LVMplKmcDDIoiAqT7p8l6VCo8TbryyO+tCauj5HD5Z3Fubqk2jLwPOaiLU13EaDOSPhdWJLmwQOrYVX91AeD0KXxjvU5y3NCHeIqum+BF1N3nQj\\\\\\\/6LOLpMHtyP1zYUUlZwwrZCyBz3F35Gt+yk4Hhhb8S\\\\\\\/fdbjVj1meZ7lckLW4S2hPhm1+tmzt1MDm4+GIY\\\\\\\/sr\\\\\\\/z8pNtgp789Rm4WNxvdlsJDjLMXjAjidbuhgEBzst2Qc0ymCCHRObopiXf1ui41mRrKQcmAPrEfU6aII1sx20z6b9Fg2H9a224v6hUjwxu\\\\\\\/S50pQLO1qFFxC\\\\\\\/wV5hr42orYhS2T5DsRefxvYFBhtaUgsjiuEXejfAfOjS5UrXqtpC\\\\\\\/4+2E4UA65ldw3+Q35k5ztbtmFNv5evltjkyQJvSudI9qhz\\\\\\\/\\\\\\\/7LuOVxsgRiOEebA6\\\\\\\/xPizLsOHGJVCIlTUrDoHGkFH2E2O9+hzy0ni6URio7XqdJcZVRJLQcuz+Febk\\\\\\\/a5gG8qteTLq\\\\\\\/\\\\\\\/ykyZgo2i5+bEhlhx+cP29N8DM86+3j1AEdP8GRvDTk8rcUr8RKGCwrz1rucSJFgy4A+qmVm49Zznq+r+hknPEM3sceH3b4Oz8OMk7JLhglHlvdOJzlzhz4Py6MojkKNE2Tie9plomw2jmfl6bM0\\\\\\\/Fok0YwivA8BW4IIS0drNq1I90QO36OB9I3dcT6zGd55jwX9KCIazRGfILv\\\\\\\/tGvayYwRVoaQ+jWxPMb1Vq+kql+qW40+K\\\\\\\/1QFqjDMmQduw3i0AFm3GJ9zP3E8N+F2iwLt7UB+DAoYnBecJNU9sXEiJl3a2WS4uRUlgrwl03rid27SGf2xmOixYr\\\\\\\/2m0G86BiK2Am+K4xw7TsM+sI6dxMvlF1vNnwwDBjAJDNiAZLvPrVpP\\\\\\\/08gv5CmNxhJtW6ZP\\\\\\\/USztjKqh3UrQsjlIEQqjffUIB0pDf9AZ3q2ub3gSDoioREU8Ej4Rsdfe1vWPlrQkD9CbqSj7tssBhC+tlIMPRqpup+oO\\\\\\\/2n23bXUtEo4StGIzf3gpDMMWAguQIJKnMQW8saQoIzImmByEfIirBkswFQwm+8u2Eaix+do\\\\\\\/gGp1G\\\\\\\/snFpn9otuEWNxm\\\\\\\/gCcY8Q6\\\\\\\/rlo1ur4pziRVcZ8vsChhWyzYautxQQv9QQLrjnYLrhBKJvGLepqnj77Uejlus7ZOvOrbbXaHr6W6gXyc3NljXHgN2kAh5eS8csrmUNlxtcdHnPfTnTvxASMiCC9TUBjQqJg1uEvWg4Yvmp4U8r9bZL0v4BvxRr0Qu5MM3dkXaxo3kHeDoNh6O0485+aOKrYSWhLGdR\\\\\\\/zSNUK56ByrhH7q29xLZ0RKhM251\\\\\\\/ANiSn5W0U7l1LEnTwUlm7FSlxhHP\\\\\\\/ZfsSSK9OeTZzxAt\\\\\\\/lDk\\\\\\\/q7J1br9EtuxmMN3+nTtoXURJqMU2cHWZbwSU5BQ8K73n0nGiH5Vn6ez8GsfQrVdSM5+9ma+2P6CGHWV5XCE9bZxgGx1IuPHkexcQj4Sv\\\\\\\/Zy+s2OEjVgXCoqpttiG2f8iiP824C7efdjU17txWhzn7vBEG8YwNX1MzrQHIgOtYl5l8NwdR4KZOYdIazdaTanD0gn9BAP08AdZDXFB\\\\\\\/abgQxVACDMrPEwqV0QV5NnrkOV83kV86+VL5nXKmmtA\\\\\\\/v11PGHzNndy6qw0MJf8OZU+Hfbq5X0IZ6YBt3Y5BB8A+1tOBp2dzlUb99EJyFKq+IGqJ08L\\\\\\\/kP0qdaGzuLtedg91X7mcpMa78Bf434ixzdo+CXgcYLvX9GlEyQk+\\\\\\\/eeG0n2Dc2wuml4Vml9ROkdtYhy6QBiGM7choV31GuU2QnZnS+7zyJ4yxG0ntnyFqkbAnSn5y0fdx0V\\\\\\\/jKVq0uw2BzZd5b8\\\\\\\/6YtEwvX8pSdG1WqUDoPsRf+TJLuH5BfsP7pTdkBHECdm9oXhLVxut\\\\\\\/aCrxV3gqL\\\\\\\/kQUY8Vgs1+EF1j+pbCcOqgHZPEBi6PCFIt07LSUi3QCsH4szwfkfH+TY1XuxQ181bjOnigxpLyWNF1spCvoFxjDILzTuxJJwZJM9dHdqu47qarsgsOL8b0Xt2r331a0yQKrasBtvIVpsq1fezm72LPekGeQuWtRJptN9XdNe9HnInXqb7cBVBHoZD6C3ytaeXxVyZ8Iels4aMipifWYwe5hKCBVoyfU57TZR3y\\\\\\\/3yyM0v4fq5RrPhiMZ9cC8OqFgss+eOiFDuB8IB03MxB3rSoQpmTUUqKhyDYSQ6bW6CI64kpT9Wq+9IVgJFzprquL9xMqhCm2kR6KyasAmCcnkVrnDixet83ZakFzBh86FKxm\\\\\\\/3Euc37MGMxw7TOfz8QV4FJZknnq8diISnHmkJUzSIVLTtnOxR8itAGEsKRxqHfG5hmadD4pWw5rNDHaNd5PGRTZz8lkza4rxpefDbsYGikJF9J+RFqrzlExsgWeCg7STTQD2x1kI8f2lipDBVVteoyj1NeiW938JsRtQAoKRbWBeHEG+Mpc\\\\\\\/d0olepouzMTWrYIU8ZBmeCzqVy3Vu2x5wpXRtf\\\\\\\/tbKn9raKdIzo8x9OLUSseSTtZbbi+EtA+neBAbQ7Gd5x5QOZdy56cXtfANoej7Tu1ouGLYwiKppsgDXMs28xS9fk\\\\\\\/6Ti0wwR3atHl+1mfSf4Bd6JO8RxKw4US12nRAuzDrpqOgwdGSiWDvAvbg4CDJH2UJHVDjjD48nUXa4vfS69tLSVNKFOeL0+R9U5JRQXjwgSQxiaXS0lDHt\\\\\\\/6P5ZIOILS5QKzgXT7jPI3+nBu4R9UgTNqPTyExma+1PjokUlcR5Yqnz7AG3kIwXAOoE5Im1FUKiugMgp5ZXDAuQ643sY2Q5o9jYRd96JghVxQI26OVhtduep38LIEAV7WdvGdAS0nN\\\\\\\/HIKnF1l5zfa0S5X871YkT7F1YYuFfX1SiIXYdUIkjS23DBWgeQBxlgxfgSm3t\\\\\\\/dMJVKkV5q8iJ3WtFr\\\\\\\/G\\\\\\\/fn\\\\\\\/\\\\\\\/DmLctQcImz\\\\\\\/RyV+zqgdGxqrcwPF21g4elfgE8gIMC6vTMuJQsrVzTyqWHX4SEyk9IWWdXJc7\\\\\\\/V+e3F4ePabvn+HBbwIuZTg0mUMCvoLbybH3H2pn2kZR4PG0YaoE2PQ9wMcDPyLZP3gK5ztr3edxEDNjlmqJILWRZfAb3q\\\\\\\/aIWwmb8wJY2ztMEhPoSGf8kf2\\\\\\\/NAauMlzYEkyHWiAD9rgFPV6k62C5USNo110XgmbZXk7kQ5G0tzIuDQPVAeH\\\\\\\/ovt7n0smTPFCB7pbt58nb3M3JK33yNVnXp+GTn8kfnYxhJcXc8pxO6c6+Am7sQgug3ADFC2Y3e+Qpqpo1d+YvOX6KKD\\\\\\\/F0JFXAPGyxiHxqscJFWFz\\\\\\\/H89MlhHbHa4l10lBXJWjZGFIPYrnROeSNJrliNYjuiTEG5Cg9uzGEMfwIGITzjHdxzQ1tmvsHBM4CMdm5f9hB6SYLWT95dw66WBIBFQc31fY2N9hztiXkwFEaz5RHC7AApR7RfRaYrvejvBmIakjPaCCTmRqYwDlo\\\\\\\/r33RPliM3oV4iVHUF1TnPd+L6b2EgaC7CORByEcHMX+DMP6zV8MFLicOHmmYJigU32UYSXwSLI1fhUXUDnRQOBmHTTcbNYptt0nRNqVR1LPrja3Rs\\\\\\\/x3TWV7L0n9peBpQxl4SBiBn0AAwaiNy9RoceoUthxuaEYYkUVRRtkirvVmSEvO3Xw0ykFiCafUiJaO14QuWVV\\\\\\\/00tVR+GFEZ7XeRgcbHHIhOmbVjHkb2RwMJFY1Wx81n8\\\\\\\/BcBcGlKEqxp7T1LnlmcILcS\\\\\\\/fahHFQ8y9CYKsH8+L0GaL5ReWVscaMEa8XDh3nipBaY5ykrCCqEfBLquIatwgAff2xbm43ucxIe74arxPg+nuHQdQZrst5HKhKyyzFos3ZPdCbJH1NWqgULddwHfyAA3SMgW4yGsQbc98KVFLU1etSo95TzlOOgO9XQu+s7RJELmzz1EpdpCa3HH8pt5OlchZ4Hz6eWIjTvP4JmnFxSVUwFVzYAnA5Tj8MZAFSwzmmWB33cfMLYSwCwkqIy1dh6jx6lMxydqTTdXgipe2wymYxEhDkGDtTxoXj02w6hJiMlzZDsik2AWFK842FN8lSyJbRp+ET6YzZow4k3EapVpx73WO1CfZeqi83zog8fOwWc+Rs5emVtPV\\\\\\\/mpwrYseAJ9p39AMZl2Oz7HNaUGWR9ySu4j\\\\\\\/iRDgpw3MRyGqkkXB7JZMMRDWfPb0uD9vDhn6a99YXcz9ANFjzGCGrFL91Xey5IenD0t9yXpU2aVYycSEONBgqV4uBm2COzfmJPqRuMN3gIKV9HYpTeZXUgAgZmgGjB0Wfhubf3fzyiUjufOQW7fj3QKx9HE\\\\\\\/tN8VfbgWQirlYuqzJGgL8yfXLpIWOLfO8yhNsYxhTh9HE42VGC7JO8h+O3s6xK68cGs\\\\\\\/8ZtHhwLuysB0RF\\\\\\\/q\\\\\\\/++UZUiMv7obkzg+99Cz7uCEQsbZev2znK2hgeHrLgTzY+hqszc1b3g7HpVYiyriCHiKV2pkvfTgnsFwD9njJ2BcrO+MHhfYFwzrJPDCALv+7VRRPuDFRNGp5BhX9jX9VY1R+pu5CilFHlzeesSCufjBZ8F5KsYBveDMuRnohOFSJudqn1mrhZfItj6ieSQoEVUgLXfFkIonTVrTOjQ\\\\\\\/8BlP0Odz4R8m5ZYp0eCa37Tx1A0M5G93zZtkD2INtFdf6\\\\\\\/gamvrxflOa1t\\\\\\\/sJx4NdCc+tdF4UePLBAailChPKQPB3g+s6gVbzKJrr6LMouA689MUOD3WtJa8aDR3YQckkWo\\\\\\\/rNqg0Zje0cQIfBdMraPm0MWMZsy7De5Rh\\\\\\\/P5kKRAqLOArHqT0oDLPwCg7WSlZL6dzu7PqiE0vcRMdRKbdzMCUxJ+OOAfhX94ZrEGLxPR0\\\\\\\/m5lcZuS34RicxiVHh4DK+cBXl5gIsaAH0gWdKN5uU4FnWlhQ30WdBrKWPcYVuoaQKVYgM0WtF5HKL7GJQaCmS1OAg7zhLdC\\\\\\\/FRzjj9Zj+zMW6fBaEXnmRAGCxxhhQ53eR0yoE\\\\\\\/F5DayR4h+DSvURVaoETcARy53B\\\\\\\/h7CyxRX03NIpshc0aFNGAQxa6vx\\\\\\\/v+XXZCwXYFg7O6UaGJVhn0VSXWCTPzBbYJTUNes5A+WYkju+5EaqhFhbBlMSTImK4ajp+zLOX1gcnvp9RbBxgDzsmrToSr9CBZratYlsCJXUU7wm6Awnrj\\\\\\\/oajJtHlgQwesG2rzFTczqRm5pJ6ptYZ4gtLjDpHilIzrd5JdS46dVP5QMOPxyfpgAkZAVh0lzQjDAjNZjjgKgFlOZ1ANRehTWonc03WjJWll\\\\\\\/8L6v0socbGUj5fkjCk7WYSnvBXPHp7hEOvN5wkziVfH42Zc12ilP+SZDzahz27sP5S9vh6aYYH3+MX0LNFw07UYkr3+o0majgsVcz+v2J0PPMw1G2aSiLcNXM2k4fjKtkw6xxQHIKkWPQtfdQ7LVHBg6K35rMgh\\\\\\\/RVRA90jM9hrRykddyFD7lMwd6FDN++Yqp0ntpWtBY0cGvn6BSgO0GYKToeCjXm5CZCPDknZWaosuLGdkXpp\\\\\\\/mnZzWj\\\\\\\/PEA2xRH8Y9AYpBfxZXSHtK43Dr0PusoNp5c4zDajZW1+S1Gmj0Yqg+BqJu95mX5LKqijWO3MWqqZO7iX2Y7eQDQl4XzpkQUtWjJpjGELuwA5aSdUBHfhR49aIhWzNLh3B3BVyV+dPqxHptoqpMF0EdXWhfOFVeazNB1pX8uhISQNDe7QUJc+ZEiFuezwVCISErMMNP9xtoswNOlZX8xcq7zimVTyCtKzamfk5lyEesrfTLEbJjOFATF+4EDPbe2lhgUhEYgU\\\\\\\/ARx+B4mp6Rq5f+JQxEhOr2ZorI4YQjwSWbIT+3cHNNokKsJ8t2rUuSNhwr9UUCr0TVrvTXXVDmlNxZMeb0RtBOnt\\\\\\\/dI+\\\\\\\/mB9nskqXKCUbIWt0MTOBRor62bu+MXLfrXeQjqdIibk2nDyl7nhkudIJ1g6eSuJUROXx+7lhGy8A0Rghj+GYI7G0J1ewxIJkiLlJCmT2s\\\\\\\/fTDuIQqjVCmLweVXrIaQ+4iDDKsaaa0Y\\\\\\\/jFl5cBNXP64qabG24xjXIKudyahQWdtoRx3ZWKAdL8vnVb\\\\\\\/bu22v4QniRMT+MGDyVd236KrytLPmMvchZkqsH4MECS2m74BO4ggEtHP1fCSNd2kfHhdhtk\\\\\\\/MgJMpy4IMnXpn6rydxGMevlfyxjvEt7CkpT79nzqexFiDMfIrJoO+7wtOcdCqhwEvrDif6caN2zU39zoIQJ65HBIH6b9WiAg89Al23yvCn6QYIdmYUe9qtzG7Cva6Y\\\\\\\/JrGZ9w0q3blHcO22\\\\\\\/a4kheMLI5f3mx7tGJ+M5Uvf+\\\\\\\/ndaCIp6J1KyGJdM1BGvxmbwPQghnQ7eWEM9k+jd5swV8HsiVS58y+XBdxZbXMIQ8N1s2qw4hQT7cevuphru8n56yke57XhgxKp0O0sx54MzQy5uuHkNUYn6TdcuWn46xKlxsnL\\\\\\\/YHEPxAjIZH2Xy6trjLLWnNxFhBTgF2zGReVhsmcueF9\\\\\\\/Opzqp\\\\\\\/+kFRKOu3\\\\\\\/xriiNLcwUzhabLsebolslUtT6zCO0antGQZSa1csmTWdMBuxWctTvjtDupQ90tgx6pYfdCQLS7w9XQSEJlWsFOzIGMrf5X9qTLZNh5AijlBpXHKJ4NNX\\\\\\\/h3c7IGHuuYKf0jvoo2JX7dOC125NefxXwTsKELkk8sTwTbSAb7Wro6aXC4q6N+8ggrz5Y034xZEz4t4aMDIDFDEI+DUWMHKI4x3yEzoffb69SrsJl3E6Xu0lruQW5nil2y0XTKqw8Jc5c0Enbl6oJnSURckCtXswuVcFbMj4HQ5satwGCd0I61g7TTiMf8utOgMoK7EmvYPxxLIgoHhJFIkQBqXpzM1IJJkNCOG3bb1SGshXDTjlT4R4WoPM5ADpFOvB35lPKdxiLFM2saIWSKw726DsIFcDQwuYJerSTumigvpI6tVG2bS4qCAe0d289+xulNZtN14OZERCaTRnNqpsEOXUdjGhuKnL5ZNKfMlAeWgnybhCPsW4MsHDiZhVV2g5cUdDzQ3QxSh4NoLreD6xLS+AR4Cx7J1chygWdKPQRBwQ4LZ5lFnEU08YUvMamYo1Pd3g+P8gDpCnhjbF6tXBhhhgKCDW9i76Y7mFS9pNvC2esnZAR9JcfqI3MW1NkUaRorb810A6t6zRLKlt3RtNJXOkkjFEK9ijn3ufFLCNFFYTH0jB+27XWAZ4wewdNg+7chAMZOqZy\\\\\\\/R9DJoyP523IN7VKaS7oNupMEAaNChnQl0KRQ\\\\\\\/q8HGfzGqOyQ5jAPuwd9suyXF9\\\\\\\/t6GGB8\\\\\\\/gdWGm99JidRzi9bL2KUWpv6c1UXpongv5Nv0GLgujDTaQ5lQrVko7jeZBiFW6yMjPDlqrWmhx72WbUz8H6LJ3Gdp795k6YTk0UcdwwL3V0yH0qBpyguyy4q0hEL69RnIDo4SvEuZHupb6f7J7C+boslg8jCp40nxc4v3LhXlU5brrlf7KZSAU0XeYFafiq8eZ560VFHkgUNg2IcQHHxhYMYsVX9cZ9uLldNlOMQ3R\\\\\\\/gAWOEToRIqoF7giXxX9xJVdWC2XlvA5gntKZgi5pyEiTmmaKnMJD1RLOIIrMOsOYyeaAwx7PWtH4BfhMgLJ7PPfP\\\\\\\/42VxxQ0tlJlKTFqIX1H0ay6yYFx+vF2iA9FcfgPy6LD1zrsrdzKputcyhWERYChMky+QoN91c0\\\\\\\/b63Z4IjI2DRy8ECtC2VuhJc893omZUbxDvElNw\\\\\\\/EkihojDjFOYE6C6RxvEfZk+ZTV\\\\\\\/FsDC3\\\\\\\/\\\\\\\/a5VTL\\\\\\\/l9dXRYGZIeob\\\\\\\/PYAuswolRQDMQ+S197K7gmbJ9OphQ8tfo8TilmSDCgCC7yXwhXjuhIqlUkw3IP1YKCfbzd2\\\\\\\/ctbeZqzsd4fzzv+ODaYBnbE1whXuIIu1pr0+dCCneDcJxlyX4DqR9NmVqiZkazlucinRYBc0GBGtS\\\\\\\/RdmRoyZtFbF1lksg8DvFdUQ5zZsHe4GwrV+TaVNWDPoREj3XKWV\\\\\\\/82K4T8YOMtAtonhgztJ3dldj\\\\\\\/9aaK0Ny6TcObus4lHevlYWvwpC\\\\\\\/1d8UuYb6Fd6eGN38Tfnfi6uxLFq48UqErM5l\\\\\\\/D0UaNTHMdNM3TZyQ+r1r0r9mpS2N0GT95QQA9voFG9KYemcpIy51WbvDSIzxHY4hYJyjC6WNfFwqkci\\\\\\\/jzsKkjlKOlapnXWLDQC9Gz0F9Vr\\\\\\\/t7IVl2abVlNQH457vOUV80MjkgLQ9KCfQ2VKX5TemyNk\\\\\\\/PbpCgZ5okMJpTnJmWoeO\\\\\\\/NcBX2lYAJQ8Y+LjUQi7\\\\\\\/PmVf84RoGIlYkNU0cKcEPyQQ\\\\\\\/7+3vvkvKCQ7cvrt024AxU90AZebZJrITMoB6tS6FjGdLZ6zK9ax2DAfBkYSX3dNOeuG8p9DPl\\\\\\\/PjtMUpfcKrh8G+ryB07WdARyjS9HHjYVIRi8fsreBHtVymJDkOd9gO1j2Iw7VJ2BNPeOfBoMgqKgpDzXii1vGrWNuEyGDu0YLhHqhY6WFH9uHWV8H\\\\\\\/SoOBM+3EItbq3E+9hIGkaRJvZ2\\\\\\\/P1MTZUb5HzGzGq\\\\\\\/YBXx5sQ5DrDHI5Isfzp96cpXgO6ZKsEbsRdcqdsKtK6WzHbn8rEXI1WKGcRgxvmvLIQapIS44GwJfm1gDuUbdBi1DBllq9afMx6Bymzo9XpGG9zF1TIdxCMQ35\\\\\\\/IsE4erkHLDAF\\\\\\\/wBvmEYcl11vV9nKldnjKAnHb1IwK6tOyP557t3Q+OyL6HkHwVFcsC5N8ka5X3phcpBSGIcXLaO6e1Z1\\\\\\\/ZCiIJluvBrtLRVhNWEo5pGh1pCpY9ykoJgOS1JPbsQp8WU\\\\\\\/qppCktr6U\\\\\\\/JXsAeK0kPfmVDILn4UtGauxiRYCdTD0dlKqjtRQ\\\\\\\/Glkk\\\\\\\/YHubKrHO7k079sp97XHy9urB6dTuYni3xJyciGxUoWbTjo8rKMd4MRSg8fyIbbcRQ9B\\\\\\\/zmcZL8F9ahYYX7\\\\\\\/NQTkcaJH25aFTs2xGouRW+84npxfbu7Xwoeeth+BUBxmIl2ARe4Ihrk2VvwJMy7xz1l\\\\\\\/XyhQUSLLm3AkqGbi6d3THFjxYi2XeHF\\\\\\\/F\\\\\\\/9dFzT+ip3mcc9I0FUZnstMtx9LcDt9MAorFMPInDscTlw5rDnkl3KmPxrSNagkDxiBk6OLxFwko+nsrsCc2HdeVble8mDCd92HipO8eww6r3dL4kMt+MbjB6xeDH\\\\\\\/miwiQxPX6F7h5HsoRadrDo7BKbleWbzhcvWIWaJ\\\\\\\/ZZxDd9Tid2t1VQ53HQKhKm3zbzY0+RZ9zjLzsN8QePFpIEYPAqlfIdPsdVktlFVl0uevv+ms981y8624pgYY6Nus2X1P3F3LH22j1+HO5dgXYDOyZJgmiDQnWMRDMEI9bNDdy+0deR9QJ6vnDgIx7bI0jNIAvAlPrMzUOXCaEVseExnuPJbu0rBEGBUoq9bKFN6k423lRLhQqZIgDNanAI77jxvoxc06Zbcw4zGnn17Rc3LEfVvUBbskM8onyUi2sZlfzx2LPlq5tkLzi9BZlueTYXCX\\\\\\\/xyfJtEgDqgchi1XJXhEA2OKhPh2BSIDvFsKXYebp5wC6DyTSp7gfYcHC9S4orGwaSRNU9lkmkpBsF+67yi1XKOXIpTkaqv3RP\\\\\\\/VRTzdg3F+TDDJi9jPOf8RhY2MezKbGH5sdgs7rIcKh6b2T3PayTJKFF0q52ZWjJ3VnTmhhf5xySzmdAPuLUP5GmMMJzWZlF4Tye1KwdybSK9cgBk6NrKBZwfPleSBZSBB5dd7ms2Xrj\\\\\\\/vm7ooWUlVjFKYgKbRmAUmnlW7Bg8kJBJCH8Q5TF5tY\\\\\\\/5D5obb4vvMlMY6Q1uU5tH8OO2NYqliOXqZ7nY\\\\\\\/\\\\\\\/Ytrs9FuwpM6xdqZxBCIdj+2uENr6fdHkHrmN21OZ0QA4KEFLpUVuBTTIxLpLpgYIIXW6llQJsm1rl7\\\\\\\/YnNQ1VpuisMd8dhd8SA9X9UWPBat1ryGqEklD3WN3nsaSNjfA3yG\\\\\\\/TGtZ0FohebU56T7NnWKJ6sRHQpehWeIQ97q+G9aZCMYtEIoWeQ3sTmh\\\\\\\/AIE0kUL1pLrHl3oIqxOVd9QQb41et8yLaaL\\\\\\\/OVL0x7D60zXoADawLMNh8USlluS490JWHI3ub\\\\\\\/\\\\\\\/5NXYbupio0SzOSjDQGkR0Moo7qm4tN7gcNCbvGhATrr7hLNEFYlW3tiYr0Nn9\\\\\\\/o9ZhTqDdRWftJ7BhrkKzOxRZiHjN3PYNE4Ef31wVyvqWQ+ERt9J+d13ib0f6RsLJcBuO5UNdW8VflTlfC9U4gEyyHYKQ465l3Uct0RHVZEqWnFF0FUiOseqGPy9X4o62nyTm5sHc9ZcW98zycUj\\\\\\\/SSZHRej9SIYy36dSTmJxUEpAbJwx5JIGelyw+mwJYvyhci\\\\\\\/+Ruj3CCMo1w8UY7E9BoY3zATDtgxW4iCbKcfd\\\\\\\/4Ubv0IccXBP+1ObiOzVYvbRaHkAeAihHEDBqCA\\\\\\\/m39VC3qFRfJamYkJUvGJNP+xpFCcVQ+0YeyE\\\\\\\/Mmt1F4hCSay9yugqFyD5ItytucsmaRNh7H1pnXO5RfzlTsws0kbyJ1h8ph9Dy80DdUdtFpUHBanWeduXwquvdKl+ak9D\\\\\\\/ldVPS7uLJfxBfvh0JQvitWMJ8H2XzdytCeNVGsLXr86XxQDAnRPRKmPqNYV0arrv9c0cixABAb1Iy5DJ04wKygZEt9fVJIIWykwyLaRpsihjtlY8KBNGb+NiLLFOY3Sr+zguD82cRssk95SOxoZWWqckG+XjTR7gpmFdeKLW9uvJN41VPGNyRQKbnGWaFpijU4umPGGkdIsJkOrEXMPF3KZl4mfDOeVaRWOf3ToUVeVFLd\\\\\\\/LZnNUj0fB9wdM9QBZauysw7xAlY6jauJwWeeNU\\\\\\\/t4INJr\\\\\\\/5Dntn9vF9WbAbN3mypAS\\\\\\\/vUPlkh2MWVxw2CbMbGiyGoESIpa+fJeAExyrnZx0fSrX8E8fcFZD9Elg+e2PxIt\\\\\\\/Llg1oCdM5+iKlIHoSA0c47tprOPVQ+jFazWkITtmL4XO1sL5Iz6qaLavf3WVmwbp2nJGaw\\\\\\\/CDEA0zAc\\\\\\\/fj8cgrsgB5Eqtuqm0vZty5SlOS3ZhuoTuvSF5YOanNl7Kqo1CR2DKyrR7pFX9IKFzhT5AHmXWUMehmV981giG\\\\\\\/q1yYfWRSqHGZThtNeka1sJFmw+CWiBAQrbDF5\\\\\\\/PT8OFSiArSBW5LW3ApIGVv\\\\\\\/Pzfdq7zOjpr3IxS0cnxEFoB6dpcl4HrnLOH8+Bnromp8eU10GnheTNVprctZmJ+uCcp\\\\\\\/H3vlbLHZehg8wvb+CeXb\\\\\\\/rNLWpiqCDSN9g9+8\\\\\\\/xAOrRpN3OVNeV4xERLN395Rq+AevgrnDJocjtWpm2K5xISB+7y+bXF3VXifPpx2Nvm1LRE0dWwt70InQQygX5gQpazCzC99JN23CeunbqlIRxc9wsdFNKmIlfGLnrFFcdCZRH+kFaaTbRkwtyquTYj4Tg00XG1zwOJ8ARmVWPNE\\\\\\\/2digldGRXLHFiDKhLHYURZznviSan1XF3ipmG3R\\\\\\\/Es2umgpiwdeINM4GUZKXpGYiIu6RS\\\\\\\/RHX0CpeJkiUTyus1Jr0\\\\\\\/9W\\\\\\\/Jxc4whDDd6trynNIUeDPj++O5j4s+Mp8ymHLwYmH47qHxeWLnNXCQnCw6C7t5+9AaRs7IEhZaCLlAoBnAvzNgSn3TByS27cN\\\\\\\/r8jU5v8mg1hFf07uzNup2KdFfXjSugXq1J1rmY7LV2g0GYqTT3WDetmcFUeXozgyhp\\\\\\\/q\\\\\\\/sRzsRco3WQZj1V2vz1iHgxaJbyB9EWj+ZZcULFM6t3KqexPVSbiTfJzXLWtb8NqE0AyvioncuWUb5YBR3o4hl7a6zXpWBYPo1vaf6r\\\\\\\/GeGq6zJv3hE5IU4IhmkWszjzQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ysex5TaOYaBqg1x5t9voCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002202134e4d7edcc3beafd1cc4061f1cd5652840d22afc8b4be663ae16a678307b6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eVmdPjQRVbJwczLUiTh4Ls+w11UYlUwLuHSeOGCkhKLsj3OrxXo1Hma+8qSXJu+J1LUCbzwHtE4SQwKxMv7SCnVhlltEGHYKCu\\\\\\\/GSz1RKDJHCAdkP3xaivU8gpZHWZQWFY6Fm821VcJTgKKVm4BhRLFuWdPa86bLEgMgT9Y1yokIUUzi5+b+mejRZ0CQIMXXN1gZuWKWZnSXVT2X4MRQd8asF9gMqgWxlTUGjYTuYRps6XrsQunkg\\\\\\\/0f2HR56Zl3\\\\\\\/wseDL9XsBHox1GQ9zh5uaOlZEz0PQS4vo99bXzcsxSi4GMsn2wm\\\\\\\/6IGs+wMm9lagsp10HbtUb1nfqbJRQOleANdRs1f70+8rbrXJvp5\\\\\\\/SEHDOXY39LJ49bTvemvO7FVhn8I+KVOdu1e3zv4vnG2O7kTmCPVAReVgO7foCRF39gdyeKxCb\\\\\\\/JAD4brmXlemXg7+3a2lAHp6w0yoWA07L933H34oUXBqZkoz9kvAEDtlLPw+8kSfsixGOEU0A6JFTg\\\\\\\/o6Ek6SNvCM4AeRrk0VTyFWoLwuMKO3zITOk11hYp5RI0B5SrMZk72kxaNbFACgo0izCyyVy2WZ04UghDQZkWRHRT9Y9WSGB0+thgEGtrtq+A1aIe9r45yHs1yEvnf2HXMGpYnM+kNqdkK977FtzzwDgaFhzICsPVFSlu\\\\\\\/dsDgTUH0K7iwKYiAU8yLJdxBGYkwtRcFUhKkEtuLpI2i3lb5Th0\\\\\\\/xRljZWWyA5pFLG9n12ZGTN97+6H2o+fBjBVcaFb6SFsUqC29hZIGZXIungQWzWPBNq23eSYrLRI3R2\\\\\\\/BVTEyUX1hY2CvDpPYZGFnVurhDk\\\\\\\/BPqzgW9te7uct++Y6mcf7TK8oWV7ZbSJq4J2\\\\\\\/I27Ftvla510CkzEg2VzEykM9U+x7kILcHYV4Kjh+ibfCsmlCm8p9i81bJqwjphRxTsmvvnmjLuEy+eiTtCAsDz0WAXxNDkke\\\\\\\/3krOiW5j85wMfSFHuZk6YQrqpFr+3Z110LS7qxVSMQXN\\\\\\\/7bsHedQyPXN8sSoPQHD13vzNvtUi+comGxuZO7qSaot3wIWFmscZmrtrQutsFWohmGyGB9Aaf34vTcpANzvJSaTRhWpwuMEN\\\\\\\/BCPp\\\\\\\/YhyTgndju+pOyFLXMZILTOFDxC+bYHiD6fhJnAZAx8oiilcg7DjCS25sMLkguztF\\\\\\\/dIS\\\\\\\/1hmGdsGH4xQJWFDxkQ\\\\\\\/a1Zzmr+Y4CUHDraEJzLnFISfLsztr2oTDQQtPc5waKVxZISPdiUPNIC4X0z7RBNEnqgyFyqZD6cJiVwJauiBMdXSihBIjg5hBOpb9kwNTlwm27in19YcVEmn1XbgaeF4WETJiIvlgw3QnzNPPg5x8OryVPyA8fO+SS7gjrBZc8RfOGB6w0gQY\\\\\\\/UtXApg46+ShPnfDZoQoRKDm\\\\\\\/GUB+l5D6MMBQlVh3MZDa\\\\\\\/mhI2AZTL31S2mDp3oXMbJDquF4gX89flWneDYVH3WeTmysgGW3EYJ+idBtwrp0RIE2hNy0GR2gI60+KW+jR1p6RDiEXqE5aVnnloUgYXLalO6Al3us1h0ZBD9pPgYfctjy6FZepXGX7AfC9cbtFO38qdcDAdfPXeRiw13Qbyh9LR+71EDetU9X0ogMHkkZ\\\\\\\/Znl5g4lZnuQCK8ivTlA37+s62MaVj101hWHTCqdSl3i5PhZR4iKkkFqdF8cHak2nQqtm77Mv+BP\\\\\\\/1YbWzixmeV80qOBRn7oLuq7zsZteiYZdFgWybS2bwrhZbsaENKN1e2gYTFZ2bpS+LBjDMcTK01C4iqcSGOKFq++1tdUZJ3\\\\\\\/Boa+ROhYP7\\\\\\\/qRUN\\\\\\\/DUfqUo0q+3XjSFmrWJSfd3d0FEiSgBd74EQtlANYL2Q1wq2GsI58rxLwgGpcT2RTeS1Pj8Xfoy2iXKpuE+OZtvKTsf+jafrmpwMWNWNkR8twPjn84gOmKczlhRVIvLO5Jq1+\\\\\\\/xiwgjL95Rjd5qj01CcviVNBLzTBOKkLkvZkqYcymzBVHrvZkXhuYk02E1jBNuw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+1UVFiMDSJzjUmaxv5VBZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c959de99a2999efcddcd16c025f7235dc1b468cdf11b3e54c93afadcee13d013\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490444 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2757" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WT1vCEKpI1o2XhWF9u5CKbPhVNM6WZh635ZPmQt0cj8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2757" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-s0oEJ3W3NSk7-o5rV7O73p5oK4gxidLVyoLda8fRj-s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2749" title="ovoD Co-selection: A Method for Enriching CRISPR/Cas9-Edited Alleles in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2773" title="A Missense Mutation in the Vacuolar Protein Sorting 11 (VPS11) Gene Is Associated with Neuroaxonal Dystrophy in Rottweiler Dogs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490444" id="node490444" data-pisa="ggg;8/8/2757" data-pisa-master="ggg;g3.118.200481" data-apath="/ggg/8/8/2757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying-peng Hua</span>, <span class="highwire-citation-author" data-delta="1">Ting Zhou</span>, <span class="highwire-citation-author" data-delta="2">Qiong Liao</span>, <span class="highwire-citation-author" data-delta="3">Hai-xing Song</span>, <span class="highwire-citation-author" data-delta="4">Chun-yun Guan</span> and <span class="highwire-citation-author" data-delta="5">Zhen-hua Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2757-2771;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200481 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490444" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying-peng Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying-peng%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing-peng%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ting Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATing%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qiong Liao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qiong%2BLiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liao%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQiong%2BLiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hai-xing Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hai-xing%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHai-xing%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chun-yun Guan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>National Center of Oilseed Crops Improvement, Hunan Branch, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun-yun%2BGuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChun-yun%2BGuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhen-hua Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhen-hua%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhen-hua%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhzh1468@163.com" class="" data-icon-position="" data-hide-link-title="0">zhzh1468@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2757" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490444" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490444/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2757.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490444" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490444/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2757.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490444" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490444/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2757.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490444" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490444/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2757.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490444" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490444/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Nitrogen (N) is a non-mineral macronutrient essential for plant growth and development. Oilseed rape (A<sub>n</sub>A<sub>n</sub>C<sub>n</sub>C<sub>n</sub>, 2<em>n</em> = 4<em>x</em> = 38) has a high requirement for N nutrients whereas showing the lowest N use efficiency (NUE) among crops. The mechanisms underlying NUE regulation in <em>Brassica napus</em> remain unclear because of genome complexity. In this study, we performed high-depth and -coverage whole-genome re-sequencing (WGS) of an N-efficient (higher NUE) genotype “XY15” and an N-inefficient (lower NUE) genotype “814” of rapeseed. More than 687 million 150-bp paired-end reads were generated, which provided about 93% coverage and 50<strong>×</strong> depth of the rapeseed genome. Applying stringent parameters, we identified a total of 1,449,157 single-nucleotide polymorphisms (SNPs), 335,228 InDels, 175,602 structure variations (SVs) and 86,280 copy number variations (CNVs) between the N-efficient and -inefficient genotypes. The largest proportion of various DNA polymorphisms occurred in the inter-genic regions. Unlike CNVs, the SNP/InDel and SV polymorphisms showed variation bias of the A<sub>n</sub> and C<sub>n</sub> subgenomes, respectively. Gene ontology analysis showed the genetic variants were mapped onto the genes involving N compound transport and ATPase complex metabolism, but not including N assimilation-related genes. On basis of identification of N-starvation responsive genes through high-throughput expression profiling, we also mapped these variants onto some key NUE-regulating genes, and validated their significantly differential expression between the N-efficient and -inefficient genotypes through qRT-PCR assays. Our data provide genome-wide high resolution DNA variants underlying NUE divergence in allotetraploid rapeseed genotypes, which would expedite the effective identification and functional validation of key NUE-regulating genes through genomics-assisted improvement of crop nutrient efficiency.</p></div><span class="kwd-group-title">KEYWORDS:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-variants" class="hw-term hw-article-keyword hw-article-keyword-genetic-variants" rel="nofollow">Genetic variants</a></li><li class="kwd"><a href="/keyword/genotypes" class="hw-term hw-article-keyword hw-article-keyword-genotypes" rel="nofollow">genotypes</a></li><li class="kwd"><a href="/keyword/nitrogen-use-efficiency" class="hw-term hw-article-keyword hw-article-keyword-nitrogen-use-efficiency" rel="nofollow">nitrogen use efficiency</a></li><li class="kwd"><a href="/keyword/rapeseed" class="hw-term hw-article-keyword hw-article-keyword-rapeseed" rel="nofollow">rapeseed</a></li><li class="kwd"><a href="/keyword/whole-genome-re-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-re-sequencing" rel="nofollow">whole-genome re-sequencing</a></li></ul><p id="p-3">Oilseed rape (<em>Brassica napus</em> L.), a high-value crop species, is widely cultivated and harvested for the production of vegetable oil, livestock protein meal and biodiesel (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blackshaw <em>et al.</em> 2011</a>). The allotetraploid <em>B. napus</em> (A<sub>n</sub>A<sub>n</sub>C<sub>n</sub>C<sub>n</sub>, ∼1,345 Mb, 2<em>n</em> = 4<em>x</em> = 38) originated from spontaneous interspecific hybridization of the diploid progenitors <em>Brassica rapa</em> (A<sub>r</sub>A<sub>r</sub>, ∼485 Mb, 2<em>n</em> = 2<em>x</em> = 20) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2011</a>) and <em>Brassica oleracea</em> (C<sub>o</sub>C<sub>o</sub>, ∼630 Mb, 2<em>n</em> = 2<em>x</em> = 18) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2014</a>) ∼7,500 years ago (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chalhoub <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bayer <em>et al.</em> 2017</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sun <em>et al.</em> 2017</a>). Approximate 70% of angiosperms, including <em>B. napus</em>, have undergone one or more chromosome doubling events during their evolutionary processes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Masterson 1994</a>). The allopolyploidy events occurring in <em>B. napus</em> generated many duplicated segments and homeologous regions within the genome (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chalhoub <em>et al.</em> 2014</a>), which poses great difficulty in mapping and cloning of gene(s) responsible for target agronomic traits.</p><p id="p-4">Recently, with the availability of rapeseed genome sequences combined with the roaring development of high-throughput next generation sequencing (NGS) technologies, it has now become convenient to explore genetic variability at the genome-wide scale by re-sequencing diverse rapeseed genotypes. NGS has provoked a revolution in plant genomics research and offers a wide range of applications (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Edwards <em>et al.</em> 2012</a>). For example, whole-genome re-sequencing (WGS) studies have also been widely utilized in gene identification, quantitative trait locus (QTL) mapping, genome diversification, evolutionary and phylogenetic analysis (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hua <em>et al.</em> 2016a</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">b</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stein <em>et al.</em> 2017</a>). Using a large set of high-density genetic polymorphisms identified by WGS, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> (2017)</a> identified numerous genomic loci associated with three seed-quality traits. Apart from bridging the gap of genotype to phenotype, WGS has immense potential to unravel the functional complexity of rapeseed genome and can promote molecular breeding to improve the agronomic traits of interest.</p><p id="p-5">Nitrogen (N) is a non-mineral macronutrient essential for plant growth and development (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Konishi and Yanagisawa 2014</a>). In most plants, inorganic N is acquired by roots in the form of NO<sub>3</sub><sup>-</sup>. For many species, NO<sub>3</sub><sup>-</sup> is not assimilated in the roots, but is secreted into the xylem sap for long-distance translocation to the shoot, where it enters the cells to be metabolized and/or stored in the vacuoles. Therefore, to reveal the genetic basis underlying differential N use efficiencies (NUEs), we focused on the genes responsible for efficient NO<sub>3</sub><sup>-</sup> uptake and translocation.</p><p id="p-6">Several plasma membrane transporters involved in NO<sub>3</sub><sup>-</sup> influx into the cell have been identified in <em>Arabidopsis thaliana</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012</a>), especially in the roots where the NPF (NRT1/PTR Family, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Léran <em>et al.</em> 2014</a>) members and NRT2 transporters are predominantly implicated. Among them, NRT1.1 and NRT2.1 (together with its partner protein NAR2.1) are the major dual- and high-affinity contributors to efficient NO<sub>3</sub><sup>-</sup> influx into root cells. AtNRT2.4 is also involved in root NO<sub>3</sub><sup>-</sup> uptake at very low NO<sub>3</sub><sup>-</sup> concentrations and in shoot phloem NO<sub>3</sub><sup>-</sup> loading (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kiba <em>et al.</em> 2012</a>). AtNRT2.5 is essential for efficient NO<sub>3</sub><sup>-</sup> uptake and remobilization in adult plants by participating in phloem NO<sub>3</sub><sup>-</sup> loading under severe N starvation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lezhneva <em>et al.</em> 2014</a>). When NO<sub>3</sub><sup>-</sup> enters root cells, NPF7.3/NRT1.5 mediates efflux of NO<sub>3</sub><sup>-</sup> to the xylem vessels, and NPF7.2/NRT1.8 retrieves NO<sub>3</sub><sup>-</sup> from the xylem sap into xylem parenchyma cells (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lin <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2010</a>; Chen <em>et al.</em> 2012). Currently, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> (2017)</a> found that Nitrogen Limitation Adaptation (NLA) is involved in source-to-sink remobilization of NO<sub>3</sub><sup>-</sup> by mediating the degradation of NRT1.7 in <em>Arabidopsis</em>. Taken together, NUE is a complex trait controlled by multiple genes implicated in NO<sub>3</sub><sup>-</sup> uptake, translocation and remobilization. The recent OsNRT1.1A and OsNRT1.1B identified in rice highlight the crucial role of efficient NO<sub>3</sub><sup>-</sup> uptake in NUE regulation (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hu <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2018b</a>).</p><p id="p-7"><em>B. napus</em> has a higher nutrient requirement for optimum seed yield than cereals (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Grant and Bailey 1993</a>), and it is extremely susceptible to N deficiency (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rathke <em>et al.</em> 2005</a>). Conventional crop management practices require the use of relatively high amounts of N fertilizers (from 150 to 300 kg of N hm<sup>-2</sup>) to ensure an optimum yield (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rathke <em>et al.</em> 2006</a>). Significant differences in NUEs, including N uptake efficiencies, N remobilization efficiencies and N assimilation efficiencies, among others, have been observed in allotetraploid rapeseed genotypes (Schulte auf’m Erley <em>et al.</em> 2007; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sorin <em>et al.</em> 2016</a>). However, the molecular mechanisms underlying efficient N uptake, transport and utilization remain poorly understood because of the rapeseed genome complexity. However, for the model crop rice (genome size: ∼450 Mb), WGS has contributed to the genetic basis identification of NUE variations between the two main subspecies (<em>indica</em> and <em>japonica</em>) of Asian cultivated rice (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hu <em>et al.</em> 2015</a>). Genetic diversity for NUEs in rapeseed genotypes can be therefore, investigated using high-throughput NGS-based WGS of genotypes with contrasting NUEs.</p><p id="p-8">In the present study, we carried out WGS of a low-N tolerant (N-efficient, higher NUE) genotype “Xiang-you 15” and a low-N sensitive (N-inefficient, lower NUE) genotype “814” of rapeseed on an Illumina HiSeq 4000 NGS platform. Further, we have identified numerous genomic variants, including 1,449,157 single nucleotide polymorphisms (SNPs), 335,228 insertions/deletions (InDels), 175,602 structure variations (SVs) and 86,280 copy number variations (CNVs), in these genotypes at a genome-wide scale. Combining the WGS and transcriptome sequencing data, we also evaluated functional significance of these polymorphic sites by correlating their presence (structural and functional annotations) in the NUE-regulating genes responsive to N limitation. What is noteworthy, the genome-wide high resolution SNP and InDel sites, discovered from the rapeseed genotypes with differential NUEs, would accelerate the identification and functional validation of key NUE-regulating genes for genomics-assisted crop improvement.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and hydroponic culture</h3><p id="p-9">In previous studies, we have identified an N-efficient genotype “Xiang-you 15” (“XY15”) and an N-inefficient genotype “814” of rapeseed (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Han <em>et al.</em> 2015a</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">b</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Han <em>et al.</em> 2016</a>). For isolation of genomic DNA (gDNA) and total RNA, the <em>B. napus</em> seedlings were hydroponically cultivated according to the method described by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hua <em>et al.</em> (2016a</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">b</a>). Plump seeds of a similar size were surface sterilized and then germinated on a piece of moist gauze immobilized on a black plastic tray. Uniform <em>B. napus</em> seedlings with similar hypocotyl and root lengths and cotyledon sizes after germination were transplanted into black plastic containers holding 10 L Hoagland solution. The nutrient solution was constantly aerated throughout the experiments and refreshed every 3 d, with one-quarter-strength solution initially added and increasing to one-half strength and eventually full strength.</p><p id="p-10">For RNA sequencing used for the identification of genes responsive to N limitation, the rapeseed seedlings were cultivated under high NO<sub>3</sub><sup>-</sup> (6.0 mM) for 10 d, and then transferred to low NO<sub>3</sub><sup>-</sup> (0.30 mM). At 0 h, 3 h and 72 h, the shoots and roots of the seedlings were individually sampled. A total of 27 RNA samples were subjected to an Illumina Hiseq X Ten platform (Illumina Inc., San Diego, CA, USA), which generated approximate 6.0-Gb of sequencing data with 150-bp paired-end (PE) reads for each sample. Transcript abundances (FPKM values) were determined from the RNA-seq data with the method described by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hua <em>et al.</em> (2017)</a>. Three independent biological replicates for each treatment were prepared for the high-throughput transcriptomic profiling.</p><p id="p-11">For quantitative reverse-transcription PCR (qRT-PCR) assays, the solution concentration of N was adjusted to 1.0 mM and 5.0 mM by reducing KNO<sub>3</sub> and replacing Ca(NO<sub>3</sub>)<sub>2</sub> by CaCl<sub>2</sub>, while K<sup>+</sup> was complemented by adding K<sub>2</sub>SO<sub>4</sub> (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> 2017</a>). The seedlings were cultivated in an illuminated growth chamber (300-320 μmol m<sup>-2</sup> s<sup>-1</sup>; 24° daytime: 22° night; 16 h photoperiod) for 15 d.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation and genome sequencing</h3><p id="p-12">High quality gDNA was extracted from fresh young leaves of the N-efficient genotype “XY15” and the N-inefficient genotype “814” with the cetyltrimethylammonium bromide (CTAB) mini-prep method (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Murray and Thompson 1980</a>). Integrity of gDNA was assessed by Bioanalyzer 2100 (Agilent Technologies, Singapore). WGS of the two rapeseed genotypes was performed on an Illumina HiSeq4000 (paired-end, read length = 150 bp) (Illumina Inc., San Diego, CA, USA) by Novogene Technologies (Beijing, China). Sample preparation and sequencing were performed according to the standard Illumina protocol (<a href="https://www.illumina.com/?langsel=/us/">https://www.illumina.com/?langsel=/us/</a>) as follows: quantified gDNA was treated with the ultrasonic wave to produce DNA fragments, which was purified using the QIA quick PCR kit. End repair was performed with poly-A on the 3′ends, then the adaptors were ligated, agarose gel electrophoresis was used to select fragments for PCR amplification. Sequencing was performed through establishing a library with Illumina HiSeq4000. The reads were aligned using the Burrows-Wheeler transformation. Low-quality reads with adaptor sequences and duplicated reads were filtered, and the remaining high-quality data were used in the sequence mapping.</p></div><div id="sec-4" class="subsection"><h3>Mapping of high quality clean reads to the reference genome</h3><p id="p-13">Read quality check and alignment were performed according to the standard Illumina analysis pipeline (<a href="https://www.illumina.com/?langsel=/us/">https://www.illumina.com/?langsel=/us/</a>). The adapter and low quality sequence reads were discarded and high quality sequences with mapping quality (MAPQ) <strong>≥</strong> 20 were retained for read alignment. Prior to alignment, raw reads were first trimmed based on the quality, compositions, and adapter sequences of nucleotides. After modulation, reads <strong>&lt;</strong> 30 bp was removed for further analysis. The trimmed reads were aligned to the rapeseed genome reference (<a href="http://brassicadb.org/brad/datasets/pub/Genomes/Brassica_napus/">http://brassicadb.org/brad/datasets/pub/Genomes/Brassica_napus/</a>).</p></div><div id="sec-5" class="subsection"><h3>Genome-wide calling of SNP, InDel, SV and CNV variants</h3><p id="p-14">The SAM tools software (v 0.1.18) was utilized to detect SNPs and InDels and investigate their quantity, type and distribution on chromosomes and gene coding regions. SVs and CNVs were investigated using the Control-FREEC software (FREEC v6.2) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boeva <em>et al.</em> 2012</a>).</p></div><div id="sec-6" class="subsection"><h3>Functional characterization of genetic variants</h3><p id="p-15">A gene ontology (GO) analysis of the genetic variants was performed by the PANTHER Classification System (<a href="http://www.pantherdb.org/data/">http://www.pantherdb.org/data/</a>) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mi <em>et al.</em> 2005</a>). The enriched GO items were illustrated with the word cloud generator WordArt (<a href="https://wordart.com/">https://wordart.com/</a>). Using the gene sequences from <em>A. thaliana</em> as seed sequences, a BLAST analysis was conducted to search for homolog sequences of rapeseed in the <em>Brassica</em> Database v. 1.1 (<a href="http://brassicadb.org/brad/">http://brassicadb.org/brad/</a>) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> 2015</a>).</p><p id="p-16">Gene co-expression networks were constructed to identify gene interactions and locate core genes that connect most neighboring genes involved in N starvation response in <em>B. napus</em>. The threshold of Pearson correlation values was set as the default parameters (<a href="http://plantgrn.noble.org/DeGNServer/Analysis.jsp">http://plantgrn.noble.org/DeGNServer/Analysis.jsp</a>), and then the gene co-expression networks were constructed with CYTOSCAPE v. 3.2.1 (<a href="http://www.cytoscape.org/">http://www.cytoscape.org/</a>) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kohl <em>et al.</em> 2011</a>).</p></div><div id="sec-7" class="subsection"><h3>Quantitative reverse-transcription PCR (qRT-PCR) assays</h3><p id="p-17">The qRT-PCR assays were used to identify the differential gene expression between the N-efficient and N-inefficient rapeseed genotypes. After the treatment of RNase-free DNase I with RNA samples, total RNA was used as the templates for complementary DNA (cDNA) synthesis with the PrimeScript RT reagent Kit with gDNA Eraser (Perfect Real Time) (TaKaRa, Shiga, Japan). The qRT-PCR assays for the detection of relative gene expression were performed using SYBR <em>Premix Ex Taq</em> II (Tli RNaseH Plus) (TaKaRa, Shiga, Japan) under an Applied Biosystems StepOne Plus Real-time PCR System (Thermo Fisher Scientific, Waltham, MA, USA). The thermal cycles were as follows: 95° for 3 min, followed by 40 cycles of 95° for 10 s, then 60° for 30 s. Melt curve analysis to ensure the primer gene-specificity was conducted as follows: 95° for 15 s, 60° for 1 min, 60°-95° for 15 s (+0.3° per cycle). Expression data were normalized using the public reference genes <em>BnaEF1-α</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Maillard <em>et al.</em> 2016</a>) and <em>BnaGDI1</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2014</a>), and relative gene expression was calculated with the 2<sup>-ΔΔC</sup><em><sub>T</sub></em> method (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Livak and Schmittgen 2001</a>).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-18">The raw sequences of whole-genome re-sequencing and transcriptome sequencing are available at the short-read archive (SRA) of the National Biotechonology Centre of Information (NCBI) under the BioProject ID PRJNA340053. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6724913">https://doi.org/10.25387/g3.6724913</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Overview of the WGS data of the N-efficient and N-inefficient genotypes</h3><p id="p-19">Whole genome re-sequencing of the N-efficient genotype “XY15” and the N-inefficient genotype “814” revealed the occurrence of gDNA polymorphisms at a genome-wide scale and their probable effect on differential low-N tolerance in these genotypes. In this study, we obtained ∼379 million reads in “XY15” and ∼308 million reads in “814”, respectively, with ∼98.6% high quality sequences (Q<sub>30</sub> passed quality score) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The obtained average read depth and coverage were 59.1 <strong>×</strong> and 94.2% for “XY15” and 49.2 <strong>×</strong> and 93.2% for “814”, respectively, which can also be seen in the Circos figure (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Such a high coverage and read depth indicates the high quality assembly and sequencing data. To avoid contamination with chloroplastic and mitochondrial DNA reads, all the reads showing a depth higher than 128<strong>×</strong> were removed from subsequent analysis (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Subbaiyan <em>et al.</em> 2012</a>). As shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, thousands of genomic variants between the N-efficient “XY15” and the N-inefficient “814”, including 1,449,157 SNPs, 335,228 InDels, 175,602 CNVs and 86,280 SVs, were randomly distributed across the 19 chromosomes of <em>B. napus</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490652/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490652/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of the whole-genome re-sequencing data of the N-efficient genotype “XY15”and the N-inefficient genotype “814”</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the genetic diversity between the whole-genome sequenced rapeseed cultivars (N-efficient cv. &#x201C;XY15&#x201D; and N-inefficient cv. &#x201C;814&#x201D;) and the de novo sequenced reference genome of &#x201C;Darmor-bzh&#x201D;. Overview of the genome-wide genetic variants between &#x201C;XY15&#x201D; vs. &#x201C;Darmor-bzh&#x201D; (A) and &#x201C;814&#x201D; vs. &#x201C;Darmor-bzh&#x201D; (B), which are delineated by the Circos program. In the Circos figure, the variants are as follows outside-to-inside: chromosomes (i), single nucleotide polymorphisms (SNPs) (ii), insertions/deletions (InDels) (iii), copy number variation (CNV) duplications (iv), CNV deletions (v), structure variations (SV; red: insertions; green: deletions; blue: inversion) (vi), and intra-/inter-chromosomal translocation (vii)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of the genetic diversity between the whole-genome sequenced rapeseed cultivars (N-efficient cv. &#x201C;XY15&#x201D; and N-inefficient cv. &#x201C;814&#x201D;) and the &lt;em&gt;de novo&lt;/em&gt; sequenced reference genome of &#x201C;Darmor-&lt;em&gt;bzh&lt;/em&gt;&#x201D;. Overview of the genome-wide genetic variants between &#x201C;XY15&#x201D; &lt;em&gt;vs.&lt;/em&gt; &#x201C;Darmor-&lt;em&gt;bzh&lt;/em&gt;&#x201D; (A) and &#x201C;814&#x201D; &lt;em&gt;vs.&lt;/em&gt; &#x201C;Darmor-&lt;em&gt;bzh&lt;/em&gt;&#x201D; (B), which are delineated by the Circos program. In the Circos figure, the variants are as follows outside-to-inside: chromosomes (i), single nucleotide polymorphisms (SNPs) (ii), insertions/deletions (InDels) (iii), copy number variation (CNV) duplications (iv), CNV deletions (v), structure variations (SV; red: insertions; green: deletions; blue: inversion) (vi), and intra-/inter-chromosomal translocation (vii).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F1.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2757/F1.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490591" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Overview of the genetic diversity between the whole-genome sequenced rapeseed cultivars (N-efficient cv. “XY15” and N-inefficient cv. “814”) and the <em>de novo</em> sequenced reference genome of “Darmor-<em>bzh</em>”. Overview of the genome-wide genetic variants between “XY15” <em>vs.</em> “Darmor-<em>bzh</em>” (A) and “814” <em>vs.</em> “Darmor-<em>bzh</em>” (B), which are delineated by the Circos program. In the Circos figure, the variants are as follows outside-to-inside: chromosomes (i), single nucleotide polymorphisms (SNPs) (ii), insertions/deletions (InDels) (iii), copy number variation (CNV) duplications (iv), CNV deletions (v), structure variations (SV; red: insertions; green: deletions; blue: inversion) (vi), and intra-/inter-chromosomal translocation (vii).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genome-wide identification and characterization of SNPs</h3><p id="p-21">SNPs are the most common type of genetic variation. A total of 1,449,157 SNPs were identified across the 19 chromosomes of <em>B. napus</em>, with 763,455 and 673,135 SNPs distributed on the A<sub>n</sub> and C<sub>n</sub> subgenomes, respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A, B</a>), showing biased subgenome polymorphisms. The SNPs appeared non-randomly distributed among different chromosomes, but also within each chromosome. In general, most of the SNPs were frequent in the distal parts of chromosomes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A, B</a>), which may correspond to genomic regions with higher recombination frequency and gene density. The SNP numbers were not correlated with the length of chromosomes or with gene density. The number of SNPs ranged from 26,467 (chr.A4) to 107,462 (chr.A9) with an average of 76,271 SNPs on each chromosome (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). The nucleotide diversity π (average number of SNPs per nucleotide) varied from 1.04 <strong>×</strong> 10<sup>−3</sup> (chr.C5) to 3.97 <strong>×</strong> 10<sup>−3</sup> (chr.A5), with an average value of π <strong>=</strong> 2.37 <strong>×</strong> 10<sup>−3</sup> on the A<sub>n</sub> subgenome and π = 1.28 <strong>×</strong> 10<sup>−3</sup> on the C<sub>n</sub> subgenome, respectively. There were a total of eight SNP hot-spot regions identified on the A1 (one), C2 (four) and C4 (three) chromosomes, where <strong>&gt;</strong> 1,000 SNPs occurred within a 1-kb genomic region (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jain <em>et al.</em> 2014</a>). In contrast, a sum of 623 and 2378 SNP cold-spot regions, where <strong>&lt;</strong> 20 SNPs occurred within a 1-kb genomic region (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jain <em>et al.</em> 2014</a>), were detected on the A<sub>n</sub> and C<sub>n</sub> subgenomes, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic distribution and frequency of single nucleotide polymorphisms (SNPs) between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) Genomic distribution of SNPs. The x-axis represents the B. napus chromosome sizes (Mb), while the y-axis represents the number of SNPs present at that point on each chromosome. (B) Graph delineating chromosome sizes of B. napus. (C) Number of SNPs on each chromosome. (D) Nucleotide diversity (&#x3C0;) on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the An or Cn subgenome but not anchored to specific chromosomes; &#x2018;Un&#x2019; represents the genome scaffolds whose locations are unknown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic distribution and frequency of single nucleotide polymorphisms (SNPs) between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) Genomic distribution of SNPs. The x-axis represents the &lt;em&gt;B. napus&lt;/em&gt; chromosome sizes (Mb), while the y-axis represents the number of SNPs present at that point on each chromosome. (B) Graph delineating chromosome sizes of &lt;em&gt;B. napus&lt;/em&gt;. (C) Number of SNPs on each chromosome. (D) Nucleotide diversity (&#x3C0;) on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the A&lt;sub&gt;n&lt;/sub&gt; or C&lt;sub&gt;n&lt;/sub&gt; subgenome but not anchored to specific chromosomes; &#x2018;Un&#x2019; represents the genome scaffolds whose locations are unknown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F2.medium.gif" width="440" height="423"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2757/F2.medium.gif" width="440" height="423"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Genomic distribution and frequency of single nucleotide polymorphisms (SNPs) between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. (A) Genomic distribution of SNPs. The x-axis represents the <em>B. napus</em> chromosome sizes (Mb), while the y-axis represents the number of SNPs present at that point on each chromosome. (B) Graph delineating chromosome sizes of <em>B. napus</em>. (C) Number of SNPs on each chromosome. (D) Nucleotide diversity (π) on each chromosome. “Ann” or “Cnn” represents the genome scaffolds anchored to the A<sub>n</sub> or C<sub>n</sub> subgenome but not anchored to specific chromosomes; ‘Un’ represents the genome scaffolds whose locations are unknown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The detected SNPs were categorized into two groups: transitions (A/G and C/T; Ts) and <em>trans</em>-versions (A/C, A/T, C/G and G/T; Tv) based on the nucleotide variations between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. Among the 1,449,157 SNPs, 834,335 (57.57%) SNPs belonged to the transition type, which was more than that of the <em>trans</em>-versions (614,821, 42.43%) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Concerning the transitions, the frequency of A/G type was higher than that of C/T; however, among the <em>trans</em>-versions, the frequency of the A/C sub-type (191,117, 31.08%) was in the largest numbers and G/C was the lowest (17.01%) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). The ratio of Ts to Tv were approximately 1.36, which was larger than the expected value (0.5).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Annotation of single nucleotide polymorphisms (SNPs) identified between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) Transition (Ts) and transversion (Tv) frequency of the SNPs. (B) Genic distribution of the SNPs. (C) Number of the synonymous (S) and non-synonymous (nS) SNPs. The upstream and downstream regions are defined as the 2.0-kb upstream intervals of the start codon (ATG) and the 1.0-kb downstream of the stop codons. The upstream/ downstream regions refer to the overlapped genomic intervals of the upstream and downstream areas between two neighboring genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Annotation of single nucleotide polymorphisms (SNPs) identified between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) Transition (Ts) and transversion (Tv) frequency of the SNPs. (B) Genic distribution of the SNPs. (C) Number of the synonymous (S) and non-synonymous (nS) SNPs. The upstream and downstream regions are defined as the 2.0-kb upstream intervals of the start codon (ATG) and the 1.0-kb downstream of the stop codons. The upstream/ downstream regions refer to the overlapped genomic intervals of the upstream and downstream areas between two neighboring genes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F3.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2757/F3.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490512" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Annotation of single nucleotide polymorphisms (SNPs) identified between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. (A) Transition (Ts) and transversion (Tv) frequency of the SNPs. (B) Genic distribution of the SNPs. (C) Number of the synonymous (S) and non-synonymous (nS) SNPs. The upstream and downstream regions are defined as the 2.0-kb upstream intervals of the start codon (ATG) and the 1.0-kb downstream of the stop codons. The upstream/ downstream regions refer to the overlapped genomic intervals of the upstream and downstream areas between two neighboring genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We then used the annotations of the reference “Darmor-<em>bzh</em>” genome to examine the distribution of SNPs within various genomic features. There were a sum of 426,596 SNPs, accounting for 29.44% of the total SNPs, detected in the genic regions (including exonic, intronic and splicing areas), whereas a larger proportion of SNPs (70.56%) were identified in the 2.0-kb upstream (promoter), 1.0-kb downstream, and other inter-genic regions. Within the SNPs located in the genic areas, more than half (241,444; 56.60%) the variations occurred in the coding sequences (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Non-synonymous SNPs that lead to amino acid changes in the protein products are more likely to lead to functional alterations, which may further affect plant performance. Therefore, we analyzed the effect of SNPs on amino acid substitution. In general, only a very small fraction (0.61%) of the SNPs occurring in the exons were mapped onto the stop codons (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Among these, 1,145 non-synonymous SNPs transformed stop codons to amino acid codons, whereas 346 non-synonymous SNPs transformed amino acid codons to stop codons. Among the SNPs mapped onto the non-stop codon regions, 143,975 SNPs did not result in alterations in the amino acid sequences, which were more than the non-synonymous types (95,977) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). The ratio of synonymous SNPs to non-synonymous SNPs was close to 1.50, which was similar to the result of our previous study (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hua <em>et al.</em> 2016a</a>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">b</a>). The SNPs identified in this study causing non-synonymous amino acid substitutions can also be utilized to directly identify causal genes responsible for rapeseed NUE variations in association studies.</p></div><div id="sec-12" class="subsection"><h3>Genome-wide identification and characterization of InDels</h3><p id="p-26">The WGS data revealed 335,228 InDels unevenly distributed over the <em>B. napus</em> genome (A1-A10, C1-C9) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A-B</a>), ranging from 7,242 (chr. A4) to 17,641 (chr. A3) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). Similarly to the SNPs, the InDels on the A<sub>n</sub> subgenome were also more than those on the C<sub>n</sub> subgenome (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). The lengths of InDels ranged from mono-nucleotide to poly-nucleotide (&gt; 10), whereas the frequency of which was not closely correlated with the variation length of nucleotides (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). In terms of the insertions and deletions with same lengths, the numbers were similar (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), which suggested the random occurrence of InDels between the N-efficient and N-inefficient genotypes. Di-nucleotide InDel (124,154) was the most frequent type, which accounted for more than one-third (37.04%) of the total InDels (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). Different from previous studies (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Hua <em>et al.</em> 2016b</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Jain <em>et al.</em> 2014</a>), we found that the InDels with even-length variations were remarkably dominant over than the odd-length variants (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>). Similar to the genomic distribution of SNPs, the majority of total InDel variants also occurred in the 2.0-kb upstream (promoter), 1.0-kb downstream, and other inter-genic regions, whereas fewer than one-fourth InDels (79,420; 23.69%) were identified in the genic sequences (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). Among the InDels occurring in the exons, only about two percent (290) were identified in the stop-codon regions. For the other InDels, a total of 5425 (40.19%) caused frame-shift deletion or insertion (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic distribution and frequency of insertions/deletions (InDels) between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) Genomic distribution of InDels. The x-axis represents the B. napus chromosome sizes (Mb), while the y-axis represents the number of InDels present at that point on each chromosome. (B) Graph delineating chromosome sizes of B. napus. (C) Number of InDels on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the An or Cn subgenome but not anchored to specific chromosomes; &#x201C;Un&#x201D; represents the genome scaffolds whose physical locations are unknown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic distribution and frequency of insertions/deletions (InDels) between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) Genomic distribution of InDels. The x-axis represents the &lt;em&gt;B. napus&lt;/em&gt; chromosome sizes (Mb), while the y-axis represents the number of InDels present at that point on each chromosome. (B) Graph delineating chromosome sizes of &lt;em&gt;B. napus&lt;/em&gt;. (C) Number of InDels on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the A&lt;sub&gt;n&lt;/sub&gt; or C&lt;sub&gt;n&lt;/sub&gt; subgenome but not anchored to specific chromosomes; &#x201C;Un&#x201D; represents the genome scaffolds whose physical locations are unknown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F4.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2757/F4.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Genomic distribution and frequency of insertions/deletions (InDels) between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. (A) Genomic distribution of InDels. The x-axis represents the <em>B. napus</em> chromosome sizes (Mb), while the y-axis represents the number of InDels present at that point on each chromosome. (B) Graph delineating chromosome sizes of <em>B. napus</em>. (C) Number of InDels on each chromosome. “Ann” or “Cnn” represents the genome scaffolds anchored to the A<sub>n</sub> or C<sub>n</sub> subgenome but not anchored to specific chromosomes; “Un” represents the genome scaffolds whose physical locations are unknown.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Annotation of insertions/deletions (InDels) identified between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) The frequency distribution of different InDel sizes. The x-axis represents the InDel sizes (bp): the positive and negative values indicate nucleotide insertion and deletion in &#x201C;XY15&#x201D; relative to &#x201C;814&#x201D;, respectively. (B) Genic distribution of the InDels. (C) The frequency distribution of large-effect InDels in the coding regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Annotation of insertions/deletions (InDels) identified between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. (A) The frequency distribution of different InDel sizes. The x-axis represents the InDel sizes (bp): the positive and negative values indicate nucleotide insertion and deletion in &#x201C;XY15&#x201D; relative to &#x201C;814&#x201D;, respectively. (B) Genic distribution of the InDels. (C) The frequency distribution of large-effect InDels in the coding regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F5.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/8/2757/F5.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490584" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Annotation of insertions/deletions (InDels) identified between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. (A) The frequency distribution of different InDel sizes. The x-axis represents the InDel sizes (bp): the positive and negative values indicate nucleotide insertion and deletion in “XY15” relative to “814”, respectively. (B) Genic distribution of the InDels. (C) The frequency distribution of large-effect InDels in the coding regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genome-wide identification and characterization of SVs</h3><p id="p-29">Structural variations (SV) are generally defined as genomic alterations that affect &gt;50 bp of DNA, which are shown to have a significant effect on evolution and species phenotype. In recent years, the advent of WGS makes it feasible to study these variants in depth. Contrary to SNPs and InDels, the numbers of SVs spanning across the A<sub>n</sub>A<sub>n</sub>C<sub>n</sub>C<sub>n</sub> genome were closely correlated with the chromosome sizes. In general, the SVs on the C<sub>n</sub> subgenome were significantly more than those on the A<sub>n</sub> subgenome. Specifically, the A9 of A<sub>n</sub> and C3 of C<sub>n</sub> chromosomes had the most two enriched SVs (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). In terms of the genic distribution of SVs, the coding regions, which would likely to have a more influential effect on phenotype than other genic regions, possessed the most SVs (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). The types of SVs can mainly be divided into five terms: genomic fragment insertion, deletion, inversion, intra-chromosomal translocation and inter-chromosomal translocation (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). Obviously, most SVs led to inter-chromosomal rearrangement, followed by chromosomal fragment deletion, whereas chromosomal insertion was the least (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). Among the SVs &lt; 1.0 kb, those less than 100 bp was the least (1.2–1.4%) category, and the SV length (13.0–14.1%) ranging from 200-300 bp was the most (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic annotation of structure variations (SVs) of the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D; compared with the rapeseed genome reference &#x201C;Darmor-bzh&#x201D;. (A) Number of SVs on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the An or Cn subgenome but not anchored to specific chromosomes; &#x201C;Un&#x201D; represents the genome scaffolds whose physical locations are unknown. (B) Genic distribution of SVs. (C) Number of different SV types. INS: insertion; DEL: deletion; INV: inversion; ITX: intra-chromosomal translocation; CTX: inter-chromosomal translocation. (D) Frequency distribution of SV length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic annotation of structure variations (SVs) of the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D; compared with the rapeseed genome reference &#x201C;Darmor&lt;em&gt;-bzh&lt;/em&gt;&#x201D;. (A) Number of SVs on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the A&lt;sub&gt;n&lt;/sub&gt; or C&lt;sub&gt;n&lt;/sub&gt; subgenome but not anchored to specific chromosomes; &#x201C;Un&#x201D; represents the genome scaffolds whose physical locations are unknown. (B) Genic distribution of SVs. (C) Number of different SV types. INS: insertion; DEL: deletion; INV: inversion; ITX: intra-chromosomal translocation; CTX: inter-chromosomal translocation. (D) Frequency distribution of SV length.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F6.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/8/2757/F6.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490493" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-30" class="first-child">Genomic annotation of structure variations (SVs) of the N-efficient genotype “XY15” and the N-inefficient genotype “814” compared with the rapeseed genome reference “Darmor<em>-bzh</em>”. (A) Number of SVs on each chromosome. “Ann” or “Cnn” represents the genome scaffolds anchored to the A<sub>n</sub> or C<sub>n</sub> subgenome but not anchored to specific chromosomes; “Un” represents the genome scaffolds whose physical locations are unknown. (B) Genic distribution of SVs. (C) Number of different SV types. INS: insertion; DEL: deletion; INV: inversion; ITX: intra-chromosomal translocation; CTX: inter-chromosomal translocation. (D) Frequency distribution of SV length.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genome-wide identification and characterization of CNVs</h3><p id="p-31">Copy number variants (CNVs) are a class of SVs and are defined as fragments of DNA that are present at variable copy number in comparison with a reference genome. CNVs can create new genes, alter gene dosage and reshape gene structures. They are considered to be major sources of genetic variation, and may influence phenotypic variation, gene expression and fitness (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yu <em>et al.</em> 2011</a>).</p><p id="p-32">Different from the other genetic variants, the CNVs did not show biased chromosomal distribution between the A<sub>n</sub> and C<sub>n</sub> subgenomes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). The numbers of CNVs were approximately 200 on each chromosome, although the chromosome sized varied from 17.4 Mb (A10) to 60.4 Mb (C3) (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). Similarly to SVs, the two most CNVs occurred in the exonic and intergenic (other than 1.0 kb up- and down-stream) regions (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). The CNVs contain two types: duplication and deletion (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>). In this study, copy number deletions of genes were more dominant over gene duplications in both the N-efficient and N-inefficient rapeseed genotypes (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7C</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic annotation of copy number variations (CNVs) of the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D; compared with the rapeseed genome reference &#x201C;Darmor-bzh&#x201D;. (A) Number of CNVs on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the An or Cn subgenome but not anchored to specific chromosomes; &#x201C;Un&#x201D; represents the genome scaffolds whose physical locations are unknown. (B) Genic distribution of CNVs. (C) Number of different CNV types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic annotation of copy number variations (CNVs) of the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D; compared with the rapeseed genome reference &#x201C;Darmor&lt;em&gt;-bzh&lt;/em&gt;&#x201D;. (A) Number of CNVs on each chromosome. &#x201C;Ann&#x201D; or &#x201C;Cnn&#x201D; represents the genome scaffolds anchored to the A&lt;sub&gt;n&lt;/sub&gt; or C&lt;sub&gt;n&lt;/sub&gt; subgenome but not anchored to specific chromosomes; &#x201C;Un&#x201D; represents the genome scaffolds whose physical locations are unknown. (B) Genic distribution of CNVs. (C) Number of different CNV types.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F7.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/8/2757/F7.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-33" class="first-child">Genomic annotation of copy number variations (CNVs) of the N-efficient genotype “XY15” and the N-inefficient genotype “814” compared with the rapeseed genome reference “Darmor<em>-bzh</em>”. (A) Number of CNVs on each chromosome. “Ann” or “Cnn” represents the genome scaffolds anchored to the A<sub>n</sub> or C<sub>n</sub> subgenome but not anchored to specific chromosomes; “Un” represents the genome scaffolds whose physical locations are unknown. (B) Genic distribution of CNVs. (C) Number of different CNV types.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Gene ontology (GO) analysis of the genetic variants</h3><p id="p-34">The GO enrichment analysis of functional significance allowed us to distinguish major biological functions of the genes with variants between the N-efficient and N-inefficient genotypes, further contributing to our understanding of the genetic basis underlying their differential NUEs.</p><p id="p-35">The GO terms could be grouped into the following three categories: biological process (BP), cellular component (CC) and molecular function (MF) (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). In the BP annotations, the nitrogen compound complex process was the most enriched, which was followed by the term of response to stimulus (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>). In the GO term for CC, in addition to the usual macromolecular complex, such as cytoskeleton and cell wall, much more attention was paid to the proton-transporting V-type and two-sector ATPase complexes (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>), which provide energy and proton gradients for ion (including inorganic NO<sub>3</sub><sup>-</sup>) active transport across the membranes (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Han <em>et al.</em> 2016</a>). In the MF category, the activities of catalase, ion binding, transferase, hydrolase, oxidoreductase, transporter, and kinase were the seven strongest GO enrichments (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8C</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Enrichment analysis of gene ontology (GO) terms of the genes detected with polymorphisms between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. Over-presentation of the biological process (A), cellular component (B) and molecular function (C) terms in the genes, which are delineated by the WordArt program. The bigger the font size, the more the corresponding GO terms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Enrichment analysis of gene ontology (GO) terms of the genes detected with polymorphisms between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. Over-presentation of the biological process (A), cellular component (B) and molecular function (C) terms in the genes, which are delineated by the WordArt program. The bigger the font size, the more the corresponding GO terms.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F8.medium.gif" width="259" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/8/2757/F8.medium.gif" width="259" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490590" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-36" class="first-child">Enrichment analysis of gene ontology (GO) terms of the genes detected with polymorphisms between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. Over-presentation of the biological process (A), cellular component (B) and molecular function (C) terms in the genes, which are delineated by the WordArt program. The bigger the font size, the more the corresponding GO terms.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Transcriptomics-assisted identification of genetic variants occurring in the genes regulating NUEs</h3><p id="p-37">Molecular mechanisms for NUEs are orchestrated by a set of NSR genes, such as those responsible for NO<sub>3</sub><sup>-</sup> uptake, translocation (mainly the NRT1 and NRT2 family members) and assimilation (mainly NO<sub>3</sub><sup>-</sup> reductase and glutamine synthetase genes). So, we identified the genetic variants occurring in the genes involved in N metabolism. However, we found that almost all of the genomic variations were mapped onto the NRT family genes but bot the NR and GS family genes. The results indicated that the differential NUEs between the two rapeseed genotypes used in the present study were predominantly attributed to their distinct NO<sub>3</sub><sup>-</sup> uptake and transport abilities, which was consistent with our previous study (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Han <em>et al.</em> 2016</a>).</p><p id="p-38">To accelerate the identification of genetic variants occurring in the genes regulating NUEs, we made a high-throughput transcriptome sequencing (RNA-seq) to characterize the key genes responsive to limited N stresses. According to the normalized expression results between each two biological replicates, <em>Pearson</em> correlation coefficients (R) were calculated, most of which were more than 0.90 between each pair of biological replicates (Figure S1A). The above results indicated that the RNA-seq sequencing data were of good quality. Subsequently, we were aimed at the in-depth identification of the differential gene expression profiling of <em>B. napus</em> under short- and long-term N limitations. In the shoots, a total of 3,279 and 4,346 genes were identified to be differentially expressed at 3 h and 72 h, respectively; in the roots, more DEGs were characterized, particularly at 72 h (Figure S1B). An intersection analysis through Venn diagram indicated that there were 119 genes were simultaneously in both shoots and roots at 3 h and 72 h (Figure S1B). The gene ontology (GO) enrichment analysis of functional significance allowed us to distinguish major biological functions of the DEGs under short- and long-term N limitations. Regardless of the shoots or the roots under both short-term and long-term N limitations, the transport or transporter category was highly enriched (Figure S1C).</p><p id="p-39">To further deduce more promising SNP/InDel variants potentially associated with NUEs, we focused on some key <em>NRT</em> family genes (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>) through transcriptomics-assisted gene co-expression network analysis (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), and mapped the variants generated by WGS to them. In the genome of <em>B. napus</em>, we identified four <em>NRT1.1</em> homologs, named after <em>BnaA8.NRT1.1</em> (BnaA08g24900D), <em>BnaA9.NRT1.1</em> (BnaA09g47380D), <em>BnaC8.NRT1.1a</em> (BnaC08g41560D) and <em>BnaC8</em>.<em>NRT1.1b</em> (BnaC08g15370D), respectively. First, we identified <em>BnaA9.NRT1.1</em> as the core member in this gene family through transcriptomics-assisted gene co-expression network analysis (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>). Further, the WGS data identified five SNPs and an InDel site occurred in <em>BnaA9.NRT1.1</em> between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. The genetic variants contained, all of which were located in the promoter of <em>BnaA9.NRT1.1</em> (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9A</a>). In the same way, we found nine <em>NRT2.1</em> homologs in the rapeseed genome, and <em>BnaC8.NRT2.1a</em> (BnaC08g43380D) was thought to be the key gene of the <em>NRT2.1</em> family (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9B</a>). The WGS result showed there were an SNP and an InDel in the gene promoter region (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9B</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490542/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490542/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Overview of the high-throughput RNA-seq data of the short- and long-term nitrogen limitation experiments</span><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Samples for the transcriptomics-assisted genetic variant identification of some key genes involved in nitrogen use efficiency regulating in Brassica napus. Transctiptional expression profiling, gene co-expression network analysis and genetic variants of BnaNRT1.1s (A), BnaNRT2.1s (B), BnaNRT1.5s (C) and BnaNRT1.8s (D). For RNA sequencing, the rapeseed seedlings were cultivated under high NO3- (6.0 mM) for 10 d, and then transferred to low NO3- (0.30 mM). At 0 h, 3 h and 72 h, the shoots and roots of the seedlings were individually sampled with three independent biological replicates. RNA-seq was performed on an Illumina Hiseq X Ten platform (Illumina Inc., San Diego, CA, USA), which generated approximate 6.0-Gb of sequencing data with 150-bp paired-end (PE) reads for each sample. Transcript abundances (FPKM values) were determined from the RNA-seq data with the method described by Hua et al. (2017). For the gene co-expression network performed by CYTOSCAPE v. 3.2.1, cycle nodes represent genes, and the size of the nodes represents the power of the interrelation among the nodes by degree value. Edges between two nodes represent interactions between genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Samples for the transcriptomics-assisted genetic variant identification of some key genes involved in nitrogen use efficiency regulating in &lt;em&gt;Brassica napus&lt;/em&gt;. Transctiptional expression profiling, gene co-expression network analysis and genetic variants of &lt;em&gt;BnaNRT1.1s&lt;/em&gt; (A), &lt;em&gt;BnaNRT2.1s&lt;/em&gt; (B), &lt;em&gt;BnaNRT1.5s&lt;/em&gt; (C) and &lt;em&gt;BnaNRT1.8s&lt;/em&gt; (D). For RNA sequencing, the rapeseed seedlings were cultivated under high NO&lt;sub&gt;3&lt;/sub&gt;&lt;sup&gt;-&lt;/sup&gt; (6.0 mM) for 10 d, and then transferred to low NO&lt;sub&gt;3&lt;/sub&gt;&lt;sup&gt;-&lt;/sup&gt; (0.30 mM). At 0 h, 3 h and 72 h, the shoots and roots of the seedlings were individually sampled with three independent biological replicates. RNA-seq was performed on an Illumina Hiseq X Ten platform (Illumina Inc., San Diego, CA, USA), which generated approximate 6.0-Gb of sequencing data with 150-bp paired-end (PE) reads for each sample. Transcript abundances (FPKM values) were determined from the RNA-seq data with the method described by Hua &lt;em&gt;et al.&lt;/em&gt; (2017). For the gene co-expression network performed by CYTOSCAPE v. 3.2.1, cycle nodes represent genes, and the size of the nodes represents the power of the interrelation among the nodes by degree value. Edges between two nodes represent interactions between genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F9.medium.gif" width="241" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/8/2757/F9.medium.gif" width="241" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490555" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-40" class="first-child">Samples for the transcriptomics-assisted genetic variant identification of some key genes involved in nitrogen use efficiency regulating in <em>Brassica napus</em>. Transctiptional expression profiling, gene co-expression network analysis and genetic variants of <em>BnaNRT1.1s</em> (A), <em>BnaNRT2.1s</em> (B), <em>BnaNRT1.5s</em> (C) and <em>BnaNRT1.8s</em> (D). For RNA sequencing, the rapeseed seedlings were cultivated under high NO<sub>3</sub><sup>-</sup> (6.0 mM) for 10 d, and then transferred to low NO<sub>3</sub><sup>-</sup> (0.30 mM). At 0 h, 3 h and 72 h, the shoots and roots of the seedlings were individually sampled with three independent biological replicates. RNA-seq was performed on an Illumina Hiseq X Ten platform (Illumina Inc., San Diego, CA, USA), which generated approximate 6.0-Gb of sequencing data with 150-bp paired-end (PE) reads for each sample. Transcript abundances (FPKM values) were determined from the RNA-seq data with the method described by <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hua <em>et al.</em> (2017)</a>. For the gene co-expression network performed by CYTOSCAPE v. 3.2.1, cycle nodes represent genes, and the size of the nodes represents the power of the interrelation among the nodes by degree value. Edges between two nodes represent interactions between genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Four <em>BnaNRT1.5s</em> (<em>BnaA5.NRT1.5</em>: BnaA05g35790D; <em>BnaA9.NRT1.5</em>: BnaA09g24330D; <em>BnaC5.NRT1.5a</em>: BnaC05g24580D; <em>BnaC5.NRT1.5b</em>: BnaC05g28620D) were identified in the rapeseed genome, and <em>BnaA5.NRT1.5</em> was identified to be the central member (<a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9C</a>). Further, WGS revealed that an SNP and an InDel occurred in the promoter of <em>BnaA5.NRT1.5</em> between “XY15” and “814” (<a id="xref-fig-9-6" class="xref-fig" href="#F9">Figure 9C</a>). Among the four <em>NRT1.8</em> homologs, <em>BnaA3.NRT1.8</em> (BnaA03g44820D) was considered to be the core gene member, and we found six SNPs occurred in its promoter (<a id="xref-fig-9-7" class="xref-fig" href="#F9">Figure 9D</a>).</p></div><div id="sec-17" class="subsection"><h3>Transcriptional identification of the differential expression of the NUE-regulating genes With genetic variants</h3><p id="p-42">Given that some genomic variants occurred in the key genes related to N-metabolisms, further, we would like to identify whether these variants lead to changes in their mRNA abundances between the N-efficient and -inefficient genotypes. Some genes randomly selected, including N transporters and N limitation adaptation genes, were subjected to qRT-PCR assays (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The results showed that the expression levels of both <em>BnaA9.NRT1.1</em> and <em>BnaC8.NRT2.1a</em> were higher in the roots of “XY15” than those of “814” (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10A, B</a>). However, for the NRT2.1 partner NAR2.1, its expression levels did not show obvious differences between the N-efficient and-inefficient genotypes under both low and moderate NO<sub>3</sub><sup>-</sup> supply except under high N conditions (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10C</a>). For <em>BnaAn.NRT2.4</em> and <em>BnaC3.NRT2.5</em> responsible for efficient NO<sub>3</sub><sup>-</sup> uptake and phloem loading, their mRNA levels were much higher than those of in the roots of “XY15” than those of “814”, particularly under low NO<sub>3</sub><sup>-</sup> (<a id="xref-fig-10-4" class="xref-fig" href="#F10">Figure 10D, E</a>). All of these results above-mentioned revealed that “XY15” had a much stronger capability of efficient NO<sub>3</sub><sup>-</sup> uptake in the roots.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490487/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490487/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Gene-specific primers used for qRT-PCR assays in this study</span><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative reverse-transcription PCR (qRT-PCR) assays of the expression of genes related to N metabolism with genetic variants between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. The relative expression levels of BnaA9.NRT1.1 (A), BnaC8.NRT2.1a (B), BnaC2.NAR2.1 (C), BnaAn.NRT2.4 (D), BnaC3NRT2.5 (E), BnaA5.NRT1.5 (F), BnaA3.NRT1.8 (G) and BnaC5.NLA (H) in the roots of B. napus seedlings, which were grown under low (1 mM), moderated (5 mM) and high (15 mM) nitrate conditions for 15 d. Columns denote means (n = 3), and error bars indicate the standard error." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-429978403" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative reverse-transcription PCR (qRT-PCR) assays of the expression of genes related to N metabolism with genetic variants between the N-efficient genotype &#x201C;XY15&#x201D; and the N-inefficient genotype &#x201C;814&#x201D;. The relative expression levels of &lt;em&gt;BnaA9.NRT1.1&lt;/em&gt; (A), &lt;em&gt;BnaC8.NRT2.1a&lt;/em&gt; (B), &lt;em&gt;BnaC2.NAR2.1&lt;/em&gt; (C), &lt;em&gt;BnaAn.NRT2.4&lt;/em&gt; (D), &lt;em&gt;BnaC3NRT2.5&lt;/em&gt; (E), &lt;em&gt;BnaA5.NRT1.5&lt;/em&gt; (F), &lt;em&gt;BnaA3.NRT1.8&lt;/em&gt; (G) and &lt;em&gt;BnaC5.NLA&lt;/em&gt; (H) in the roots of &lt;em&gt;B. napus&lt;/em&gt; seedlings, which were grown under low (1 mM), moderated (5 mM) and high (15 mM) nitrate conditions for 15 d. Columns denote means (n = 3), and error bars indicate the standard error.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2757/F10.medium.gif" width="359" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/8/8/2757/F10.medium.gif" width="359" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2757/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490658" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-43" class="first-child">Quantitative reverse-transcription PCR (qRT-PCR) assays of the expression of genes related to N metabolism with genetic variants between the N-efficient genotype “XY15” and the N-inefficient genotype “814”. The relative expression levels of <em>BnaA9.NRT1.1</em> (A), <em>BnaC8.NRT2.1a</em> (B), <em>BnaC2.NAR2.1</em> (C), <em>BnaAn.NRT2.4</em> (D), <em>BnaC3NRT2.5</em> (E), <em>BnaA5.NRT1.5</em> (F), <em>BnaA3.NRT1.8</em> (G) and <em>BnaC5.NLA</em> (H) in the roots of <em>B. napus</em> seedlings, which were grown under low (1 mM), moderated (5 mM) and high (15 mM) nitrate conditions for 15 d. Columns denote means (n = 3), and error bars indicate the standard error.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">For <em>BnaA5.NRT1.5</em>, regardless of high or low NO<sub>3</sub><sup>-</sup> conditions, its expression levels in the roots of “XY15” were more 20-fold than those of “814” (<a id="xref-fig-10-5" class="xref-fig" href="#F10">Figure 10F</a>). However, in terms of <em>BnaA3.NRT1.8</em>, we did not identify its significantly differential expression in the roots between “XY15” and “814”; moreover, the N-inefficient genotype showed higher mRNA levels than the N-efficient genotype (<a id="xref-fig-10-6" class="xref-fig" href="#F10">Figure 10G</a>). For <em>BnaC5.NLA</em>, under low NO<sub>3</sub><sup>-</sup>, its mRNA abundances were strongly induced in the roots of both “XY15” and “814”; moreover, the expression level in “XY15” was about 35-fold than that of “814” (<a id="xref-fig-10-7" class="xref-fig" href="#F10">Figure 10H</a>), which indicated the stronger adaptation ability of “XY15” to N limitation than “814”.</p><p id="p-45">Taken together, it appeared that some genes implicated in efficient N uptake and transport were significantly differentially expressed between the N-efficient genotype XY15 and the N-inefficient genotype 814. To further pin down the main-effect gene(s) responsible for their differential NUEs, we mapped these differentially expressed genes to the regions of the quantitative trait loci (QTL) regulating NUE that were previously reported. In a previous study, <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> (2017)</a> dissected the root morphological traits related to NUE under both high N and low N conditions, identified a main-effect NUE-specific QTL cluster on the A9 chromosome. Based on the finding, we retrieved its physical position via mapping the flanking molecular markers of the NUE-A9 QTL to the rapeseed genomic sequences. Through comparison of the NUE-A9 QTL physical interval and the differentially expressed genes having genetic variations, we found only <em>BnaA9.NRT1.1</em> was mapped onto the NUE-A9 QTL region, and it may be a core gene regulating differential NUEs between the N-efficient genotype “XY15” and the N-inefficient genotype “814”.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Differential genomic distribution of the genetic variants</h3><p id="p-46">Large and complex allopolyploid genomes, such as <em>B. napus</em>, wheat and cotton, propose an enormous challenge for genomic variant discovery because of the presence of multiple homeologous sequences (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bancroft <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lai <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fang <em>et al.</em> 2017</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Stein <em>et al.</em> 2017</a>). In addition, the repetitive nature of the polyploid genomes has been one of the major barriers to variant identification.</p><p id="p-47">In this study, through high-throughput, -depth and -coverage WGS of an N-efficient genotype “XY15” and an N-inefficient genotype “814”, we identified 1,449,157 SNPs, 335,228 InDels together with thousands of SVs and CNVs. In terms of their genomic distribution, the four types of variants can be categorized into three groups: (i) SNPs and InDels, (ii) SVs, and (iii) CNVs. For SNPs and InDels, the variant number and density of the A<sub>n</sub> subgenome were higher than those of the C<sub>n</sub> subgenome (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">4</a>) although the assembled C<sub>n</sub> sub-genome (525.8 Mb) was obviously larger than the A<sub>n</sub> sub-genome (314.2 Mb) (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chalhoub <em>et al.</em> 2014</a>). The results agreed that the A<sub>n</sub> sub-genome was more variable than the C<sub>n</sub> sub-genome (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fletcher <em>et al.</em> 2016</a>). On the contrary, for SVs, the variants on the C<sub>n</sub> sub-genome was more than that of the A<sub>n</sub> sub-genome (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>). In the case of CNV variants, different from SNPs, InDels and SVs, its number did not show obvious sub-genome bias, evenly distributing on the A<sub>n</sub> and C<sub>n</sub> sub-genomes (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7</a>). In general, the SNPs, InDels and SVs but not CNVs were largely dependent on the subgenome variability.</p></div><div id="sec-20" class="subsection"><h3>Variation bias of SNPs in allotetraploid rapeseed genotypes</h3><p id="p-48">Transition bias refers to the ratio of transition SNPs to transversion SNPs that were more than the expected value of 0.5. In general, transitions provide easy tolerance from selection pressure as they result into synonymous substitutions, which do not alter the conformational structures of protein unlike transversions (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wakeley 1996</a>). Therefore, transitions are usually favored over transversions. The transition bias phenomenon has also been found in soybean (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lee <em>et al.</em> 2016</a>) and rice (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Subbaiyan <em>et al.</em> 2012</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Jain <em>et al.</em> 2014</a>) and other plant species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Batley <em>et al.</em> 2003</a>). A higher Ts/Tv ratio is also indicative of low level of genetic divergence. These ratios are expected to decline with increasing genetic distance between the comparative genotypes as in due course of time; transversions eliminate the record of frequent transitions (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Draisma <em>et al.</em> 2001</a>). In our previous study, through WGS of boron-efficient and -inefficient genotypes of rapeseed, we identified the Ts/Tv value was about 1.3 (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Hua <em>et al.</em> 2016b</a>), which was close to the Ts/Tv result (1.35) of Huang <em>et al.</em> (2013). Both of them were close to the Ts/Tv result of 1.36 in the present study. It may be assumed that the Ts/Tv value close to 1.3 may be a common rule in allotetraploid rapeseed.</p><p id="p-49">Non-synonymous SNPs that lead to amino acid changes in the protein product are of major interest. Compared with synonymous SNPs, non-synonymous variations are more likely to lead to functional mutations altering phenotype (Huang <em>et al.</em> 2013). Thus, the ratio of synonymous SNPs to non-synonymous SNPs can be used as a molecular clock to mirror the evolution rates of species: the smaller the ratios, the faster the species evolve. In this study, the ratio of synonymous SNPs to non-synonymous SNPs were close to 1.50, which was identical to our previous study (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Hua <em>et al.</em> 2016b</a>), and this ratio may also be universal in rapeseed genotypes. Non-synonymous variants can lead to severe phenotypic consequences that could be either detrimental or beneficial (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ng and Henikoff 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pham <em>et al.</em> 2011</a>). Harmful polymorphisms are mostly discarded during evolution through purifying selection whereas beneficial variations could be fixed leading to differential responses of genotypes to biotic and abiotic stresses (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Parida <em>et al.</em> 2012</a>).</p></div><div id="sec-21" class="subsection"><h3>Genetic variants underlying differential NUEs in allotetraploid rapeseed genotypes</h3><p id="p-50">Through high-throughput WGS of low-phosphate-tolerant and -sensitive rice genotypes, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mehra <em>et al.</em> (2015)</a> identified more than two million DNA polymorphisms and further mapped them to key phosphate starvation responsive and root architecture genes. In this research, we mapped some genetic variants between the N-efficient and -inefficient genotypes to N-metabolism compounds and some ATPase complexes involved in the regulation of NUE (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8</a>). Specifically, we employed the gene co-expression network analysis to characterize the core NUE-regulating gene(s), and mapped the genetic variants to them (<a id="xref-fig-9-8" class="xref-fig" href="#F9">Figure 9</a>). In agreement with the general trend of genome-wide polymorphisms, most of the variants mapped to the NUE-regulating genes were located in the gene promoter regions, and this result indicated that the variants may influence the expression of NUE-regulating genes mainly through modulating their mRNA levels at the transcriptional level. Finally, we confirmed the effect of variants on gene expression via qRT-PCR assays. The qRT-PCR revealed that most of the NUE-regulating genes, such as NO<sub>3</sub><sup>-</sup> transporters and N limitation adaptation-related genes, had much higher transcript levels in the roots of the N-efficient genotype than those of the N-inefficient genotype (<a id="xref-fig-10-8" class="xref-fig" href="#F10">Figure 10</a>). Previous studies have shown that the N-efficient (higher NUE) genotypes of rapeseed had much stronger N uptake, xylem loading but with weaker vacuolar N storage relative to the N-inefficient (lower NUE) genotypes (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Han <em>et al.</em> 2016</a>). In the present research, we found that these processes-related genes indeed had some genetic variants through high-throughput WGS. Therefore, the genome-wide high resolution SNP and InDel sites, discovered from the rapeseed genotypes with differential NUEs, can be used for the development of molecular markers for the identification and functional validation of key NUE-regulating genes. Further, genomics-assisted molecular marker-assisted selection can serve as an efficient tool for the identification of elite crops germplasms and genetic improvement of key agronomic traits in crops.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">This study was financially supported in part, by the National Key Research &amp; Development Program of China (2017YFD0200100, 2017YFD0200103); National Natural Science Foundation of China (Grant No.31101596, 31372130); Hunan Provincial Recruitment Program of Foreign Experts; and the National Oilseed Rape Production Technology System of China; “2011 Plan” supported by The Ministry of Education of China.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6724913">https://doi.org/10.25387/g3.6724913</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: J. Ma</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-07"><span class="received-label">Received </span>May 7, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-19"><span class="accepted-label">Accepted </span>June 19, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Hua <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.1" data-doi="10.1038/nbt.1926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bancroft</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1926</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBancroft%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D762%26rft.epage%253D766%26rft.atitle%253DDissecting%2Bthe%2Bgenome%2Bof%2Bthe%2Bpolyploid%2Bcrop%2Boilseed%2Brape%2Bby%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1926%26rft_id%253Dinfo%253Apmid%252F21804563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21804563&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.2" data-doi="10.1104/pp.102.019422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mining for single nucleotide polymorphisms and insertions/deletions in maize expressed sequence tag data.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.102.019422</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DBatley%26rft.auinit1%253DJ.%26rft.volume%253D132%26rft.issue%253D1%26rft.spage%253D84%26rft.epage%253D91%26rft.atitle%253DMining%2Bfor%2BSingle%2BNucleotide%2BPolymorphisms%2Band%2BInsertions%252FDeletions%2Bin%2BMaize%2BExpressed%2BSequence%2BTag%2BData%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.102.019422%26rft_id%253Dinfo%253Apmid%252F12746514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiIxMzIvMS84NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.3" data-doi="10.1111/pbi.12742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurgobin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golicz</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Assembly and comparison of two closely related <em>Brassica napus</em> genomes.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1602</span>–<span class="cit-lpage">1610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D1602%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.4" data-doi="10.4141/cjps2011-002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blackshaw</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAndrew</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Alternative oilseed crops for biodiesel feedstock on the Canadian prairies.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">896</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4141/cjps2011-002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BSci.%26rft.volume%253D91%26rft.spage%253D889%26rft_id%253Dinfo%253Adoi%252F10.4141%252Fcjps2011-002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4141/cjps2011-002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.5" data-doi="10.1093/bioinformatics/btr670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeva</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleakley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiche</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cappo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr670%26rft_id%253Dinfo%253Apmid%252F22155870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22155870&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300043200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.6" data-doi="10.1126/science.1253435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalhoub</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Early allopolyploid evolution in the post-Neolithic <em>Brassica napus</em> oilseed genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">950</span>–<span class="cit-lpage">953</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1253435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChalhoub%26rft.auinit1%253DB.%26rft.volume%253D345%26rft.issue%253D6199%26rft.spage%253D950%26rft.epage%253D953%26rft.atitle%253DEarly%2Ballopolyploid%2Bevolution%2Bin%2Bthe%2Bpost-Neolithic%2BBrassica%2Bnapus%2Boilseed%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1253435%26rft_id%253Dinfo%253Apmid%252F25146293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDUvNjE5OS85NTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.7" data-doi="10.1046/j.1529-8817.2001.037004586.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Draisma</span>  <span class="cit-name-given-names">S. G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prud’homme van Reine</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A reassessment of phylogenetic relationships within the Phaeophyceae based on RUBISCO large subunit and ribosomal DNA sequences.</span> <abbr class="cit-jnl-abbrev">J. Phycol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">603</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1529-8817.2001.037004586.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPhycology%26rft.stitle%253DJournal%2Bof%2BPhycology%26rft.aulast%253DDraisma%26rft.auinit1%253DS.%2BG.%2BA.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D586%26rft.epage%253D603%26rft.atitle%253DA%2Breassessment%2Bof%2Bphylogenetic%2Brelationships%2Bwithin%2Bthe%2BPhaeophyceae%2Bbased%2Bon%2BRUBISCO%2Blarge%2Bsubunit%2Band%2Bribosomal%2BDNA%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1529-8817.2001.037004586.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1529-8817.2001.037004586.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170690400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.8" data-doi="10.1111/j.1467-7652.2012.00724.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Preface: advances in DNA sequencing accelerating plant biotechnology.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">622</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-7652.2012.00724.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DEdwards%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D621%26rft.epage%253D622%26rft.atitle%253DPreface%253A%2Badvances%2Bin%2BDNA%2Bsequencing%2Baccelerating%2Bplant%2Bbiotechnology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00724.x%26rft_id%253Dinfo%253Apmid%252F22765873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00724.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22765873&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306131400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.9" data-doi="10.1186/s13059-017-1167-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic insights into divergence and dual domestication of cultivated allotetraploid cottons.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1167-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1167-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1167-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.10" data-doi="10.1534/g3.115.021279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fletcher</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of polymorphisms associated with drought adaptation QTL in <em>Brassica napus</em> by resequencing. Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D793%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.115.021279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.11" data-doi="10.4141/cjps93-087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Fertility management in canola production.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Sci.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">670</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4141/cjps93-087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BSci.%26rft.volume%253D73%26rft.spage%253D651%26rft_id%253Dinfo%253Adoi%252F10.4141%252Fcjps93-087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4141/cjps93-087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.12" data-doi="10.1007/s11738-014-1744-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Nitrate reutilization mechanisms in the tonoplast of two <em>Brassica napus</em> genotypes with different nitrogen use efficiency.</span> <abbr class="cit-jnl-abbrev">Acta Physiol. Plant.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11738-014-1744-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BPhysiol.%2BPlant.%26rft.volume%253D37%26rft.spage%253D42%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11738-014-1744-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11738-014-1744-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.13" data-doi="10.1007/s00344-014-9439-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">V-ATPase and V-PPase at the tonoplast affect NO<sub>3</sub><sup>-</sup> content in <em>Brassica napus</em> by controlling distribution of NO<sub>3</sub><sup>-</sup> between the cytoplasm and vacuole.</span> <abbr class="cit-jnl-abbrev">J. Plant Growth Regul.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00344-014-9439-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BGrowth%2BRegul.%26rft.volume%253D34%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00344-014-9439-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00344-014-9439-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.14" data-doi="10.1104/pp.15.01377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jian</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Nitrogen use efficiency is mediated by vacuolar nitrate sequestration capacity in roots of <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1684</span>–<span class="cit-lpage">1698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.01377</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.01377%26rft_id%253Dinfo%253Apmid%252F26757990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTcwLzMvMTY4NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.15" data-doi="10.1038/ng.3337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Variation in <em>NRT1.1B</em> contributes to nitrate-use divergence between rice subspecies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">834</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D834%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3337%26rft_id%253Dinfo%253Apmid%252F26053497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26053497&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.16" data-doi="10.1111/pce.12731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Transcriptomics-assisted quantitative trait locus fine mapping for the rapid identification of a nodulin 26-like intrinsic protein gene regulating boron efficiency in allotetraploid rapeseed.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1601</span>–<span class="cit-lpage">1618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pce.12731</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D39%26rft.spage%253D1601%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.17" data-doi="10.1093/jxb/erw342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Physiological, genomic and transcriptional diversity in responses to boron deficiency in rapeseed genotypes.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">5769</span>–<span class="cit-lpage">5784</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erw342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw342%26rft_id%253Dinfo%253Apmid%252F27639094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erw342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27639094&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.18" data-doi="10.1007/s11104-017-3204-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-scale mRNA transcriptomic insights into the responses of oilseed rape (<em>Brassica napus</em> L.) to varying boron availabilities.</span> <abbr class="cit-jnl-abbrev">Plant Soil</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11104-017-3204-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSoil%26rft.volume%253D416%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11104-017-3204-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11104-017-3204-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.19" data-doi="10.1111/pbi.12133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moharana</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shankar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garg</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide discovery of DNA polymorphisms in rice cultivars with contrasting drought and salinity stress response and their functional relevance.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D12%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12133%26rft_id%253Dinfo%253Apmid%252F24460890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24460890&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.20" data-doi="10.1105/tpc.111.092221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feria-Bourrellier</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafouge</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lezhneva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutet-Mercey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Arabidopsis</em> nitrate transporter NRT2.4 plays a double role in roots and shoots of nitrogen-starved plants.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.111.092221</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.111.092221%26rft_id%253Dinfo%253Apmid%252F22227893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI0LzEvMjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.21" data-doi="10.1007/978-1-60761-987-1_18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohl</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiese</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warscheid</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cytoscape: software for visualization and analysis of biological networks.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">696</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-60761-987-1_18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DKohl%26rft.auinit1%253DM.%26rft.volume%253D696%26rft.spage%253D291%26rft.epage%253D303%26rft.atitle%253DCytoscape%253A%2Bsoftware%2Bfor%2Bvisualization%2Band%2Banalysis%2Bof%2Bbiological%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-987-1_18%26rft_id%253Dinfo%253Apmid%252F21063955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60761-987-1_18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21063955&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284120800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.22" data-doi="10.1093/jxb/eru267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanagisawa</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Emergence of a new step towards understanding the molecular mechanisms underlying nitrate-regulated gene expression.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">5589</span>–<span class="cit-lpage">5600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/eru267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru267%26rft_id%253Dinfo%253Apmid%252F25005135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/eru267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25005135&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.23" data-doi="10.1111/j.1467-7652.2012.00718.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berkman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single nucleotide polymorphism discovery from wheat next-generation sequence data.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">749</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-7652.2012.00718.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DLai%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D743%26rft.epage%253D749%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bfrom%2Bwheat%2Bnext-generation%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00718.x%26rft_id%253Dinfo%253Apmid%252F22748104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00718.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748104&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306131400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.24" data-doi="10.1007/s00438-016-1183-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jo</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of candidate genes for an early-maturing soybean mutant by genome resequencing analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">1561</span>–<span class="cit-lpage">1571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-016-1183-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D291%26rft.spage%253D1561%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-016-1183-2%26rft_id%253Dinfo%253Apmid%252F27033554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-016-1183-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27033554&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.25" data-doi="10.1016/j.tplants.2013.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Léran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varala</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiurazzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A unified nomenclature of NITRATE TRANSPORTER 1/PEPTIDE TRANSPORTER family members in plants.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2013.08.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.08.008%26rft_id%253Dinfo%253Apmid%252F24055139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24055139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330013700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.26" data-doi="10.1111/tpj.12626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lezhneva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feria-Bourrellier</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafouge</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutet-Mercey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The <em>Arabidopsis</em> nitrate transporter NRT2.5 plays a role in nitrate acquisition and remobilization in nitrogen-starved plants.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D80%26rft.spage%253D230%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12626%26rft_id%253Dinfo%253Apmid%252F25065551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25065551&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342849800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.27" data-doi="10.1105/tpc.110.075242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pike</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Arabidopsis nitrate transporter NRT1.8 functions in nitrate removal from the xylem sap and mediates cadmium tolerance.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1633</span>–<span class="cit-lpage">1646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.075242%26rft_id%253Dinfo%253Apmid%252F20501909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzUvMTYzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.28" data-doi="10.1105/tpc.108.060244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canivenc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C.S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepetit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mutation of the Arabidopsis NRT1.5 nitrate transporter causes defective root-to-shoot nitrate transport.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2514</span>–<span class="cit-lpage">2528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14396</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.060244%26rft_id%253Dinfo%253Apmid%252F18780802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMjUxNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The <em>Brassica oleracea</em> genome reveals the asymmetrical evolution of polyploid genomes.</span> <abbr class="cit-jnl-abbrev">Nature Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommun.%26rft.volume%253D5%26rft.spage%253D3930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.30" data-doi="10.1111/nph.14396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Nitrogen Limitation Adaptation (NLA) is involved in source-to-sink remobilization of nitrate by mediating the degradation of NRT1.7 in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14396</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D214%26rft.spage%253D734%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.31" data-doi="10.1006/meth.2001.1262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livak</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmittgen</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2<sup>-ΔΔC</sup><em><sub>T</sub></em></span>. <abbr class="cit-jnl-abbrev">Method. Methods</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/meth.2001.1262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DLivak%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D402%26rft.epage%253D408%26rft.atitle%253DAnalysis%2Bof%2Brelative%2Bgene%2Bexpression%2Bdata%2Busing%2Breal-time%2Bquantitative%2BPCR%2Band%2Bthe%2B2%2528-Delta%2BDelta%2BC%2528T%2529%2529%2BMethod.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.2001.1262%26rft_id%253Dinfo%253Apmid%252F11846609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.2001.1262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11846609&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173949500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.32" data-doi="10.1093/jxb/erw322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maillard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etienne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diquélou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trouverie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billard</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Nutrient deficiencies modify the ionomic composition of plant tissues: a focus on cross-talk between molybdenum and other nutrients in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">5631</span>–<span class="cit-lpage">5641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erw322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw322%26rft_id%253Dinfo%253Apmid%252F27625417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erw322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27625417&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.33" data-doi="10.1126/science.264.5157.421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masterson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Stomatal size in fossil plants: evidence for polyploidy in majority of angiosperms.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">424</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.264.5157.421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMasterson%26rft.auinit1%253DJ.%26rft.volume%253D264%26rft.issue%253D5157%26rft.spage%253D421%26rft.epage%253D424%26rft.atitle%253DStomatal%2BSize%2Bin%2BFossil%2BPlants%253A%2BEvidence%2Bfor%2BPolyploidy%2Bin%2BMajority%2Bof%2BAngiosperms%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.264.5157.421%26rft_id%253Dinfo%253Apmid%252F17836906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjQvNTE1Ny80MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjc1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.34" data-doi="10.1038/srep13090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehra</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giri</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide DNA polymorphisms in low phosphate tolerant and sensitive rice genotypes.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">13090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep13090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D13090%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep13090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep13090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.35" data-doi="10.1093/nar/gki418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazareva-Ulitsky</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kejariwal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandergriff</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The PANTHER database of protein families, subfamilies, functions and pathways.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">D284</span>–<span class="cit-lpage">D288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMi%26rft.auinit1%253DH.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD284%26rft.epage%253DD288%26rft.atitle%253DThe%2BPANTHER%2Bdatabase%2Bof%2Bprotein%2Bfamilies%252C%2Bsubfamilies%252C%2Bfunctions%2Band%2Bpathways%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki418%26rft_id%253Dinfo%253Apmid%252F15608197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15608197&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226524300058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.36" data-doi="10.1093/nar/8.19.4321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Rapid isolation of high molecular weight plant DNA.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4321</span>–<span class="cit-lpage">4325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/8.19.4321</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMurray%26rft.auinit1%253DM.G.%26rft.volume%253D8%26rft.issue%253D19%26rft.spage%253D4321%26rft.epage%253D4326%26rft.atitle%253DRapid%2Bisolation%2Bof%2Bhigh%2Bmolecular%2Bweight%2Bplant%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.19.4321%26rft_id%253Dinfo%253Apmid%252F7433111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/8.19.4321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7433111&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KL92600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.37" data-doi="10.1146/annurev.genom.7.080505.115630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting the effects of amino acid substitutions on protein function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genom. Hum. G.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genom.7.080505.115630</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.spage%253D61%26rft.epage%253D80%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bamino%2Bacid%2Bsubstitutions%2Bon%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.7.080505.115630%26rft_id%253Dinfo%253Apmid%252F16824020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.7.080505.115630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16824020&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.38" data-doi="10.1186/1471-2164-13-426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukerji</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohapatra</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNPs in stress-responsive rice genes: validation, genotyping, functional relevance and population structure.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">426</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DParida%26rft.auinit1%253DS.%2BK.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D426%26rft.epage%253D426%26rft.atitle%253DSNPs%2Bin%2Bstress-responsive%2Brice%2Bgenes%253A%2Bvalidation%252C%2Bgenotyping%252C%2Bfunctional%2Brelevance%2Band%2Bpopulation%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-426%26rft_id%253Dinfo%253Apmid%252F22921105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22921105&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.39" data-doi="10.1007/s00122-011-1627-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilyeu</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel FAD2–1A allele in a soybean plant introduction offers an alternate means to produce soybean seed oil with 85% oleic acid content.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">802</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1627-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPham%26rft.auinit1%253DA.%2BT.%26rft.volume%253D123%26rft.issue%253D5%26rft.spage%253D793%26rft.epage%253D802%26rft.atitle%253DA%2Bnovel%2BFAD2-1%2BA%2Ballele%2Bin%2Ba%2Bsoybean%2Bplant%2Bintroduction%2Boffers%2Ban%2Balternate%2Bmeans%2Bto%2Bproduce%2Bsoybean%2Bseed%2Boil%2Bwith%2B85%2525%2Boleic%2Bacid%2Bcontent.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1627-3%26rft_id%253Dinfo%253Apmid%252F21681491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1627-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681491&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.40" data-doi="10.1016/j.agee.2006.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rathke</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behrens</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diepenbrock</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Integrated nitrogen management strategies to improve seed yield, oil content and nitrogen efficiency of winter oilseed rape (<em>Brassica napus</em> L.): a review.</span> <abbr class="cit-jnl-abbrev">Agric. Ecosyst. Environ.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.agee.2006.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgric.%2BEcosyst.%2BEnviron.%26rft.volume%253D117%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.agee.2006.04.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.agee.2006.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000241292800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.41" data-doi="10.1016/j.fcr.2004.11.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rathke</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diepenbrock</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Effects of nitrogen source and rate on productivity and quality of winter oilseed rape (<em>Brassica napus</em> L.) grown in different crop rotations.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2004.11.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D94%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2004.11.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2004.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2757.42"><div class="cit-metadata"><cite>Schulte auf’m Erley, G., K. A. Wijaya, A. Ulas, H. Becker, F. Wiesler <em>et al</em>., <span class="cit-pub-date">2007</span> Leaf senescence and N uptake parameters as selection traits for nitrogen efficiency of oilseed rape cultivars. Physiol. Plant 130: 519–531.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.43" data-doi="10.1186/s40529-016-0125-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leport</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cambert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchereau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariette</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Nitrogen deficiency impacts on leaf cell and tissue structure with consequences for senescence associated processes in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Bot. Stud. (Taipei, Taiwan)</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s40529-016-0125-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BStud.%2B%2528Taipei%252C%2BTaiwan%2529%26rft.volume%253D57%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40529-016-0125-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40529-016-0125-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.44" data-doi="10.1111/pbi.12732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coriton</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousseau-Gueutin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiessl</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mapping of homoeologous chromosome exchanges influencing quantitative trait variation in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1478</span>–<span class="cit-lpage">1489</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12732</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D1478%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.45" data-doi="10.1111/j.1467-7652.2011.00676.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subbaiyan</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">D. L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katiyar</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadananda</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaddadi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide DNA polymorphisms in elite <em>indica</em> rice inbreds discovered by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">634</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-7652.2011.00676.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DSubbaiyan%26rft.auinit1%253DG.%2BK.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D623%26rft.epage%253D634%26rft.atitle%253DGenome-wide%2BDNA%2Bpolymorphisms%2Bin%2Belite%2Bindica%2Brice%2Binbreds%2Bdiscovered%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00676.x%26rft_id%253Dinfo%253Apmid%252F22222031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00676.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22222031&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.46" data-doi="10.1111/tpj.13669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The high-quality genome of <em>Brassica napus</em> cultivar ‘ZS11’ reveals the introgression history in semi-winter morphotype.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">468</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D92%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.47" data-doi="10.1016/0169-5347(96)10009-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The excess of transitions among nucleotide substitutions: new methods of estimating transition bias underscore its significance.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0169-5347(96)10009-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DWakeley%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D158%26rft.epage%253D162%26rft.atitle%253DThe%2Bexcess%2Bof%2Btransitions%2Bamong%2Bnucleotide%2Bsubstitutions%253A%2Bnew%2Bmethods%2Bof%2Bestimating%2Btransition%2Bbias%2Bunderscore%2Bits%2Bsignificance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252896%252910009-4%26rft_id%253Dinfo%253Apmid%252F21237791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(96)10009-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21237791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UB35100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2757.48" data-doi="10.1111/pbi.12873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Dissection of the genetic architecture of three seed-quality traits and consequences for breeding in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1336</span>–<span class="cit-lpage">1348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12873</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D16%26rft.spage%253D1336%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.49" data-doi="10.1093/aob/mcu135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification and characterization of improved nitrogen efficiency in interspecific hybridized new-type <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcu135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcu135%26rft_id%253Dinfo%253Apmid%252F24989788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcu135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24989788&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.50" data-doi="10.3389/fpls.2017.01709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dun</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic dissection of root morphological traits related to nitrogen use efficiency in <em>Brassica napus</em> L. under two contrasting nitrogen conditions.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1709</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01709</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1709%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.51" data-doi="10.1105/tpc.17.00809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Expression of the nitrate transporter gene <em>OsNRT1.1A/OsNPF6.3</em> confers high yield and early maturation in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">638</span>–<span class="cit-lpage">651</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.17.00809</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.17.00809%26rft_id%253Dinfo%253Apmid%252F29475937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjMwLzMvNjM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.52" data-doi="10.1038/ng.919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D1035%26rft.epage%253D1039%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bmesopolyploid%2Bcrop%2Bspecies%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.919%26rft_id%253Dinfo%253Apmid%252F21873998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21873998&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.53" data-doi="10.1093/database/bav093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Brassica database (BRAD) version 2.0: integrating and mining Brassicaceae species genomic resources.</span> <abbr class="cit-jnl-abbrev">Database (Oxford)</abbr> <span class="cit-vol">2015</span>: <span class="cit-fpage">bav093</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/database/bav093</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%2B%2528Oxford%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbav093%26rft_id%253Dinfo%253Apmid%252F26589635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/bav093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26589635&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.54" data-doi="10.1016/j.tplants.2012.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsay</span>  <span class="cit-name-given-names">Y. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Uptake, allocation and signaling of nitrate.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2012.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DWang%26rft.auinit1%253DY.%2BY.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D458%26rft.epage%253D467%26rft.atitle%253DUptake%252C%2Ballocation%2Band%2Bsignaling%2Bof%2Bnitrate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2012.04.006%26rft_id%253Dinfo%253Apmid%252F22658680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2012.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22658680&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307912200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.55" data-doi="10.1016/j.gene.2013.12.057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selection and evaluation of novel reference genes for quantitative reverse transcription PCR (qRT-PCR) based on genome and transcriptome data in <em>Brassica napus</em> L.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2013.12.057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D538%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.12.057%26rft_id%253Dinfo%253Apmid%252F24406618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.12.057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24406618&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2757.56" data-doi="10.1186/1471-2164-12-372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detection of copy number variations in rice using array-based comparative genomic hybridization.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-372</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYu%26rft.auinit1%253DP.%26rft.volume%253D12%26rft.spage%253D372%26rft.epage%253D372%26rft.atitle%253DDetection%2Bof%2Bcopy%2Bnumber%2Bvariations%2Bin%2Brice%2Busing%2Barray-based%2Bcomparative%2Bgenomic%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-372%26rft_id%253Dinfo%253Apmid%252F21771342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21771342&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2757.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2757.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2749" title="ovoD Co-selection: A Method for Enriching CRISPR/Cas9-Edited Alleles in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2773" title="A Missense Mutation in the Vacuolar Protein Sorting 11 (VPS11) Gene Is Associated with Neuroaxonal Dystrophy in Rottweiler Dogs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2757" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-k-kKEU4StDkIeR3lUwBBFWsxgBKzsxdaBTgwxG7sxR8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2757" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2757" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2757" class="active" data-icon-position="" data-hide-link-title="0">Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490444" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YDQAw6p571ZZnd5Yh_yyODUjT5ZA8txdjFEtVToUbQM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2757.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2757" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-aCHGJbMGX4eJixlJpmGyPfVUTO32v5wrYgsoZVnrTmA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2757" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2757.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490444" id="node490444--2" data-pisa="ggg;8/8/2757" data-pisa-master="ggg;g3.118.200481" data-apath="/ggg/8/8/2757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying-peng Hua</span>, <span class="highwire-citation-author" data-delta="1">Ting Zhou</span>, <span class="highwire-citation-author" data-delta="2">Qiong Liao</span>, <span class="highwire-citation-author" data-delta="3">Hai-xing Song</span>, <span class="highwire-citation-author" data-delta="4">Chun-yun Guan</span> and <span class="highwire-citation-author" data-delta="5">Zhen-hua Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2757-2771;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200481 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490444--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying-peng Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying-peng%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing-peng%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ting Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATing%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qiong Liao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qiong%2BLiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liao%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQiong%2BLiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hai-xing Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hai-xing%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHai-xing%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chun-yun Guan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>National Center of Oilseed Crops Improvement, Hunan Branch, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun-yun%2BGuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChun-yun%2BGuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhen-hua Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhen-hua%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhen-hua%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhzh1468@163.com" class="" data-icon-position="" data-hide-link-title="0">zhzh1468@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2757" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2757&amp;title=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2757&amp;title=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2757&amp;title=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2757&amp;text=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2757&amp;title=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2757&amp;t=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2757&amp;title=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2757&amp;title=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490444/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490444" id="node490444--3" data-pisa="ggg;8/8/2757" data-pisa-master="ggg;g3.118.200481" data-apath="/ggg/8/8/2757.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomics-Assisted Identification and Characterization of the Genetic Variants Underlying Differential Nitrogen Use Efficiencies in Allotetraploid Rapeseed Genotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying-peng Hua</span>, <span class="highwire-citation-author" data-delta="1">Ting Zhou</span>, <span class="highwire-citation-author" data-delta="2">Qiong Liao</span>, <span class="highwire-citation-author" data-delta="3">Hai-xing Song</span>, <span class="highwire-citation-author" data-delta="4">Chun-yun Guan</span> and <span class="highwire-citation-author" data-delta="5">Zhen-hua Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2757-2771;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200481 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490444--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying-peng Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying-peng%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing-peng%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ting Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATing%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qiong Liao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qiong%2BLiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liao%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQiong%2BLiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hai-xing Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hai-xing%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHai-xing%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Chun-yun Guan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>National Center of Oilseed Crops Improvement, Hunan Branch, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chun-yun%2BGuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChun-yun%2BGuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhen-hua Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Southern Regional Collaborative Innovation Center for Grain and Oil Crops in China, College of Resources and Environmental Sciences, Hunan Agricultural University, Changsha, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhen-hua%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhen-hua%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhzh1468@163.com" class="" data-icon-position="" data-hide-link-title="0">zhzh1468@163.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2757&amp;rft.epage=2771&amp;rft.atitle=Genomics-Assisted%20Identification%20and%20Characterization%20of%20the%20Genetic%20Variants%20Underlying%20Differential%20Nitrogen%20Use%20Efficiencies%20in%20Allotetraploid%20Rapeseed%20Genotypes&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hua%2C+Ying-peng&amp;rft.au=Zhou%2C+Ting&amp;rft.au=Liao%2C+Qiong&amp;rft.au=Song%2C+Hai-xing&amp;rft.au=Guan%2C+Chun-yun&amp;rft.au=Zhang%2C+Zhen-hua"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490444/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490444/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490444/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490444/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490444/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490444/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490444/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490444/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490444/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490444/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490444/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490444/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2757" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2757.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2757.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2757.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2757.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
