<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2603" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2603.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2603.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2603.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2603.atom" />
<meta name="HW.pisa" content="ggg;8/8/2603" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Dual sgRNA Approach for Functional Genomics in Arabidopsis thaliana" />
<meta name="DC.Identifier" content="10.1534/g3.118.200046" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Pauwels et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.119.400709" />
<meta name="DC.Description" content="Reverse genetics uses loss-of-function alleles to interrogate gene function. The advent of CRISPR/Cas9-based gene editing now allows the generation of knock-out alleles for any gene and entire gene families. Even in the model plant Arabidopsis thaliana , gene editing is welcomed as T-DNA insertion lines do not always generate null alleles. Here, we show efficient generation of heritable mutations in Arabidopsis using CRISPR/Cas9 with a workload similar to generating overexpression lines. We obtain for several different genes Cas9 null-segregants with bi-allelic mutations in the T2 generation. While somatic mutations were predominantly generated by the canonical non-homologous end joining (cNHEJ) pathway, we observed inherited mutations that were the result of synthesis-dependent microhomology-mediated end joining (SD-MMEJ), a repair pathway linked to polymerase θ (PolQ). We also demonstrate that our workflow is compatible with a dual sgRNA approach in which a gene is targeted by two sgRNAs simultaneously. This paired nuclease method results in more reliable loss-of-function alleles that lack a large essential part of the gene. The ease of the CRISPR/Cas9 workflow should help in the eventual generation of true null alleles of every gene in the Arabidopsis genome, which will advance both basic and applied plant research." />
<meta name="DC.Contributor" content="Laurens Pauwels" />
<meta name="DC.Contributor" content="Rebecca De Clercq" />
<meta name="DC.Contributor" content="Jonas Goossens" />
<meta name="DC.Contributor" content="Sabrina Iñigo" />
<meta name="DC.Contributor" content="Clara Williams" />
<meta name="DC.Contributor" content="Mily Ron" />
<meta name="DC.Contributor" content="Anne Britt" />
<meta name="DC.Contributor" content="Alain Goossens" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Dual sgRNA Approach for Functional Genomics in Arabidopsis thaliana" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Reverse genetics uses loss-of-function alleles to interrogate gene function. The advent of CRISPR/Cas9-based gene editing now allows the generation of knock-out alleles for any gene and entire gene families. Even in the model plant &lt;i&gt;Arabidopsis thaliana&lt;/i&gt;, gene editing is welcomed as T-DNA insertion lines do not always generate null alleles. Here, we show efficient generation of heritable mutations in Arabidopsis using CRISPR/Cas9 with a workload similar to generating overexpression lines. We obtain for several different genes Cas9 null-segregants with bi-allelic mutations in the T2 generation. While somatic mutations were predominantly generated by the canonical non-homologous end joining (cNHEJ) pathway, we observed inherited mutations that were the result of synthesis-dependent microhomology-mediated end joining (SD-MMEJ), a repair pathway linked to polymerase θ (PolQ). We also demonstrate that our workflow is compatible with a dual sgRNA approach in which a gene is targeted by two sgRNAs simultaneously. This paired nuclease method results in more reliable loss-of-function alleles that lack a large essential part of the gene. The ease of the CRISPR/Cas9 workflow should help in the eventual generation of true null alleles of every gene in the Arabidopsis genome, which will advance both basic and applied plant research.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2603" />
<meta name="citation_id" content="8/8/2603" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2603" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2603.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2603.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2603.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200046" />
<meta name="citation_pmid" content="29884615" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2603" />
<meta name="citation_lastpage" content="2615" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Laurens Pauwels" />
<meta name="citation_author_institution" content="Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium" />
<meta name="citation_author_institution" content="VIB Center for Plant Systems Biology, 9052 Ghent, Belgium" />
<meta name="citation_author_email" content="lapau@psb.ugent.be" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0221-9052" />
<meta name="citation_author" content="Rebecca De Clercq" />
<meta name="citation_author_institution" content="Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium" />
<meta name="citation_author_institution" content="VIB Center for Plant Systems Biology, 9052 Ghent, Belgium" />
<meta name="citation_author" content="Jonas Goossens" />
<meta name="citation_author_institution" content="Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium" />
<meta name="citation_author_institution" content="VIB Center for Plant Systems Biology, 9052 Ghent, Belgium" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7940-2699" />
<meta name="citation_author" content="Sabrina Iñigo" />
<meta name="citation_author_institution" content="Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium" />
<meta name="citation_author_institution" content="VIB Center for Plant Systems Biology, 9052 Ghent, Belgium" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5820-7109" />
<meta name="citation_author" content="Clara Williams" />
<meta name="citation_author_institution" content="Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium" />
<meta name="citation_author_institution" content="VIB Center for Plant Systems Biology, 9052 Ghent, Belgium" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1292-6313" />
<meta name="citation_author" content="Mily Ron" />
<meta name="citation_author_institution" content="UC Davis, Department of Plant Biology, Davis, CA 95616, US" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1682-7275" />
<meta name="citation_author" content="Anne Britt" />
<meta name="citation_author_institution" content="UC Davis, Department of Plant Biology, Davis, CA 95616, US" />
<meta name="citation_author" content="Alain Goossens" />
<meta name="citation_author_institution" content="Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium" />
<meta name="citation_author_institution" content="VIB Center for Plant Systems Biology, 9052 Ghent, Belgium" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1599-551X" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=N. J. Baltes;citation_author=D. F. Voytas;citation_title=Enabling plant synthetic biology through genome engineering.;citation_pages=120-131;citation_volume=33;citation_year=2015;citation_pmid=25496918;citation_doi=10.1016/j.tibtech.2014.11.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Bechtold;citation_author=B. Jaudeau;citation_author=S. Jolivet;citation_author=B. Maba;citation_author=D. Vezon;citation_title=The Maternal Chromosome Set Is the Target of the T-DNA in the in Planta Transformation of Arabidopsis thaliana;citation_pages=1875-1887;citation_volume=155;citation_year=2000;citation_issue=4;citation_pmid=10924482" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=S. Black;citation_author=E. Kashkina;citation_author=T. Kent;citation_author=R. Pomerantz;citation_title=DNA Polymerase θ: A Unique Multifunctional End-Joining Machine.;citation_pages=67;citation_volume=7;citation_year=2016;citation_doi=10.3390/genes7090067" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. K. Brinkman;citation_author=T. Chen;citation_author=M. Amendola;citation_author=B. van Steensel;citation_title=Easy quantitative assessment of genome editing by sequence trace decomposition.;citation_volume=42;citation_year=2014;citation_pmid=25300484;citation_doi=10.1093/nar/gku936" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. C. Canver;citation_author=D. E. Bauer;citation_author=A. Dass;citation_author=Y. Y. Yien;citation_author=J. Chung;citation_title=Characterization of genomic deletion efficiency mediated by clustered regularly interspaced short palindromic repeats (CRISPR)/cas9 nuclease system in mammalian cells.;citation_volume=289;citation_year=2014;citation_pmid=24907273;citation_doi=10.1074/jbc.M114.564625" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=T. Čermák;citation_author=S. J. Curtin;citation_author=J. Gil-Humanes;citation_author=R. Čegan;citation_author=T. J. Y. Kono;citation_title=A Multipurpose Toolkit to Enable Advanced Genome Engineering in Plants.;citation_volume=29;citation_year=2017;citation_pmid=28522548;citation_doi=10.1105/tpc.16.00922" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=R. Chari;citation_author=P. Mali;citation_author=M. Moosburner;citation_author=G. M. Church;citation_title=Unraveling CRISPR-Cas9 genome engineering parameters via a library-on-library approach.;citation_volume=12;citation_year=2015;citation_pmid=26167643;citation_doi=10.1038/nmeth.3473" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=X. Chen;citation_author=F. Xu;citation_author=C. Zhu;citation_author=J. Ji;citation_author=X. Zhou;citation_title=Dual sgRNA-directed gene knockout using CRISPR/Cas9 technology in Caenorhabditis elegans.;citation_pages=7581;citation_volume=4;citation_year=2014;citation_pmid=25531445;citation_doi=10.1038/srep07581" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=N.-H. Cheng;citation_author=J.-Z. Liu;citation_author=X. Liu;citation_author=Q. Wu;citation_author=S. M. Thompson;citation_title=Arabidopsis Monothiol Glutaredoxin, AtGRXS17, Is Critical for Temperature-dependent Postembryonic Growth and Development via Modulating Auxin Response.;citation_volume=286;citation_year=2011;citation_pmid=21515673;citation_doi=10.1074/jbc.M110.201707" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. W. Cho;citation_author=S. Kim;citation_author=Y. Kim;citation_author=J. Kweon;citation_author=H. S. Kim;citation_title=Analysis of off-target effects of CRISPR/Cas-derived RNA-guided endonucleases and nickases.;citation_volume=24;citation_year=2014;citation_pmid=24253446;citation_doi=10.1101/gr.162339.113" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. J. Clough;citation_author=A. F. Bent;citation_title=Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana.;citation_pages=735-743;citation_volume=16;citation_year=1998;citation_issue=6;citation_pmid=10069079;citation_doi=10.1046/j.1365-313x.1998.00343.x" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=J. Couturier;citation_author=J. Przybyla-Toscano;citation_author=T. Roret;citation_author=C. Didierjean;citation_author=N. Rouhier;citation_title=The roles of glutaredoxins ligating Fe–S clusters: Sensing, transfer or repair functions?;citation_pages=1513-1527;citation_volume=1853;citation_year=2015;citation_pmid=25264274;citation_doi=10.1016/j.bbamcr.2014.09.018" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. J. Denbow;citation_author=S. Lapins;citation_author=N. Dietz;citation_author=R. Scherer;citation_author=Z. L. Nimchuk;citation_title=Gateway-Compatible CRISPR-Cas9 Vectors and a Rapid Detection by High-Resolution Melting Curve Analysis.;citation_pages=1171;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01171" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. G. Doench;citation_author=E. Hartenian;citation_author=D. B. Graham;citation_author=Z. Tothova;citation_author=M. Hegde;citation_title=Rational design of highly active sgRNAs for CRISPR-Cas9–mediated gene inactivation.;citation_pages=1262-1267;citation_volume=32;citation_year=2014;citation_pmid=25184501;citation_doi=10.1038/nbt.3026" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Durr;citation_author=R. Papareddy;citation_author=K. Nakajima;citation_author=J. Gutierrez-Marcos;citation_title=Highly efficient heritable targeted deletions of gene clusters and non-coding regulatory regions in Arabidopsis using CRISPR/Cas9.;citation_pages=4443;citation_volume=8;citation_year=2018;citation_doi=10.1038/s41598-018-22667-1" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Edwards;citation_author=C. Johnstone;citation_author=C. Thompson;citation_title=A simple and rapid method for the preparation of plant genomic DNA for PCR analysis;citation_pages=1349-1349;citation_volume=19;citation_year=1991;citation_issue=6;citation_pmid=2030957;citation_doi=10.1093/nar/19.6.1349" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=A. Eid;citation_author=Z. Ali;citation_author=M. M. Mahfouz;citation_title=High efficiency of targeted mutagenesis in arabidopsis via meiotic promoter-driven expression of Cas9 endonuclease.;citation_pages=1555-1558;citation_volume=35;citation_year=2016;citation_doi=10.1007/s00299-016-2000-4" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=T. Fatscher;citation_author=V. Boehm;citation_author=B. Weiche;citation_author=N. H. Gehring;citation_title=The interaction of cytoplasmic poly(A)-binding protein with eukaryotic initiation factor 4G suppresses nonsense-mediated mRNA decay.;citation_volume=20;citation_year=2014;citation_pmid=25147240;citation_doi=10.1261/rna.044933.114" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=F. Fauser;citation_author=S. Schiml;citation_author=H. Puchta;citation_title=Both CRISPR/Cas-based nucleases and nickases can be used efficiently for genome engineering in Arabidopsis thaliana.;citation_pages=348-359;citation_volume=79;citation_year=2014;citation_pmid=24836556;citation_doi=10.1111/tpj.12554" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Feng;citation_author=Y. Mao;citation_author=N. Xu;citation_author=B. Zhang;citation_author=P. Wei;citation_title=Multigeneration analysis reveals the inheritance, specificity, and patterns of CRISPR/Cas-induced gene modifications in Arabidopsis.;citation_volume=111;citation_year=2014;citation_pmid=24550464;citation_doi=10.1073/pnas.1400822111" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=Z. Feng;citation_author=B. Zhang;citation_author=W. Ding;citation_author=X. Liu;citation_author=D.-L. Yang;citation_title=Efficient genome editing in plants using a CRISPR/Cas system.;citation_pages=1229-1232;citation_volume=23;citation_year=2013;citation_pmid=23958582;citation_doi=10.1038/cr.2013.114" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=P. Fernández-Calvo;citation_author=A. Chini;citation_author=G. Fernández-Barbero;citation_author=J.-M. Chico;citation_author=S. Gimenez-Ibanez;citation_title=The Arabidopsis bHLH transcription factors MYC3 and MYC4 are targets of JAZ repressors and act additively with MYC2 in the activation of jasmonate responses.;citation_volume=23;citation_year=2011;citation_pmid=21335373;citation_doi=10.1105/tpc.110.080788" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Y. Fu;citation_author=J. D. Sander;citation_author=D. Reyon;citation_author=V. M. Cascio;citation_author=J. K. Joung;citation_title=Improving CRISPR-Cas nuclease specificity using truncated guide RNAs.;citation_pages=279-284;citation_volume=32;citation_year=2014;citation_pmid=24463574;citation_doi=10.1038/nbt.2808" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Gao;citation_author=Y. Zhang;citation_author=D. Zhang;citation_author=X. Dai;citation_author=M. Estelle;citation_title=Auxin binding protein 1 (ABP1) is not required for either auxin signaling or Arabidopsis development.;citation_volume=112;citation_year=2015;citation_pmid=25646447;citation_doi=10.1073/pnas.1500365112" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=V. Gorbunova;citation_author=A. A. Levy;citation_title=Non-homologous DNA end joining in plant cells is associated with deletions and filler DNA insertions.;citation_volume=25;citation_year=1997;citation_pmid=9358178;citation_doi=10.1093/nar/25.22.4650" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. M. Guseman;citation_author=K. Webb;citation_author=C. Srinivasan;citation_author=C. Dardick;citation_title=DRO1 influences root system architecture in Arabidopsis and Prunus species.;citation_pages=1093-1105;citation_volume=89;citation_year=2017;citation_doi=10.1111/tpj.13470" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Haeussler;citation_author=K. Schönig;citation_author=H. Eckert;citation_author=A. Eschstruth;citation_author=J. Mianné;citation_title=Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR.;citation_pages=148;citation_volume=17;citation_year=2016;citation_pmid=27380939;citation_doi=10.1186/s13059-016-1012-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Methods;citation_author=S. J. Harrison;citation_author=E. K. Mott;citation_author=K. Parsley;citation_author=S. Aspinall;citation_author=J. C. Gray;citation_title=A rapid and robust method of identifying transformed Arabidopsis thaliana seedlings following floral dip transformation.;citation_pages=19-19;citation_volume=2;citation_year=2006;citation_pmid=17087829;citation_doi=10.1186/1746-4811-2-19" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. Iñigo;citation_author=A. N. Durand;citation_author=A. Ritter;citation_author=S. Le Gall;citation_author=M. Termathe;citation_title=Glutaredoxin GRXS17 Associates with the Cytosolic Iron-Sulfur Cluster Assembly Pathway.;citation_volume=172;citation_year=2016;citation_pmid=27503603;citation_doi=10.1104/pp.16.00261" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Jinek;citation_author=K. Chylinski;citation_author=I. Fonfara;citation_author=M. Hauer;citation_author=J. A. Doudna;citation_title=A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.;citation_volume=337;citation_year=2012;citation_pmid=22745249;citation_doi=10.1126/science.1225829" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. Jing;citation_author=R. Lin;citation_title=The VQ Motif-Containing Protein Family of Plant-Specific Transcriptional Regulators.;citation_volume=169;citation_year=2015;citation_pmid=26220951;citation_doi=10.1104/pp.15.00788" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol.;citation_author=R. A. Johnson;citation_author=V. Gurevich;citation_author=S. Filler;citation_author=A. Samach;citation_author=A. A. Levy;citation_title=Comparative assessments of CRISPR-Cas nucleases’ cleavage efficiency in planta.;citation_pages=143-156;citation_volume=87;citation_year=2015;citation_pmid=25403732;citation_doi=10.1007/s11103-014-0266-x" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=P. Kawalleck;citation_author=I. E. Somssich;citation_author=M. Feldbrügge;citation_author=K. Hahlbrock;citation_author=B. Weisshaar;citation_title=Polyubiquitin gene expression and structural properties of the ubi4-2 gene in Petroselinum crispum.;citation_pages=673-684;citation_volume=21;citation_year=1993;citation_issue=4;citation_pmid=8383552;citation_doi=10.1007/BF00014550" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=S.-T. Kim;citation_author=J. Park;citation_author=D. Kim;citation_author=K. Kim;citation_author=S. Bae;citation_title=Questioning unexpected CRISPR off-target mutations in vivo.;citation_year=2017;citation_doi=10.1101/157925" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=V. Y. Khodaverdian;citation_author=T. Hanscom;citation_author=A. M. Yu;citation_author=T. L. Yu;citation_author=V. Mak;citation_title=Secondary structure forming sequences drive SD-MMEJ repair of DNA double-strand breaks.;citation_pages=12848-12861;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx1056" />
<meta name="citation_reference" content="citation_journal_title=Chromosome Res.;citation_author=A. Knoll;citation_author=F. Fauser;citation_author=H. Puchta;citation_title=DNA recombination in somatic plant cells: mechanisms and evolutionary consequences.;citation_pages=191-201;citation_volume=22;citation_year=2014;citation_pmid=24788060;citation_doi=10.1007/s10577-014-9415-y" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Knuesting;citation_author=C. Riondet;citation_author=C. Maria;citation_author=I. Kruse;citation_author=N. Bécuwe;citation_title=Arabidopsis Glutaredoxin S17 and Its Partner, the Nuclear Factor Y Subunit C11/Negative Cofactor 2α, Contribute to Maintenance of the Shoot Apical Meristem under Long-Day Photoperiod.;citation_volume=167;citation_year=2015;citation_pmid=25699589;citation_doi=10.1104/pp.15.00049" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=C. LeBlanc;citation_author=F. Zhang;citation_author=J. Mendez;citation_author=Y. Lozano;citation_author=K. Chatpar;citation_title=Increased efficiency of targeted mutagenesis by CRISPR/Cas9 in plants using heat stress.;citation_pages=377-386;citation_volume=93;citation_year=2018;citation_doi=10.1111/tpj.13782" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=Y. Lei;citation_author=L. Lu;citation_author=H. Y. Liu;citation_author=S. Li;citation_author=F. Xing;citation_title=CRISPR-P: A web tool for synthetic single-guide RNA design of CRISPR-system in plants.;citation_volume=7;citation_year=2014;citation_pmid=24719468;citation_doi=10.1093/mp/ssu044" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=J.-F. Li;citation_author=D. Zhang;citation_author=J. Sheen;citation_title=Cas9-based genome editing in Arabidopsis and tobacco.;citation_pages=459-472;citation_volume=546;citation_year=2014;citation_pmid=25398353;citation_doi=10.1016/B978-0-12-801185-0.00022-2" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Li;citation_author=Y.-J. Li;citation_author=F.-J. Zhang;citation_author=G.-Z. Zhang;citation_author=X.-Y. Jiang;citation_title=The Arabidopsis UDP-glycosyltransferases UGT79B2 and UGT79B3, contribute to cold, salt and drought stress tolerance via modulating anthocyanin accumulation.;citation_pages=85-103;citation_volume=89;citation_year=2017;citation_doi=10.1111/tpj.13324" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=K.-J. Lu;citation_author=B. De Rybel;citation_author=H. van Mourik;citation_author=D. Weijers;citation_title=Regulation of intercellular TARGET OF MONOPTEROS 7 protein transport in the Arabidopsis root.;citation_volume=145;citation_year=2018;citation_pmid=29358212;citation_doi=10.1242/dev.152892" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=X. Ma;citation_author=Q. Zhang;citation_author=Q. Zhu;citation_author=W. Liu;citation_author=Y. Chen;citation_title=A Robust CRISPR/Cas9 System for Convenient, High-Efficiency Multiplex Genome Editing in Monocot and Dicot Plants.;citation_pages=1274-1284;citation_volume=8;citation_year=2015;citation_pmid=25917172;citation_doi=10.1016/j.molp.2015.04.007" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=Y. Mao;citation_author=H. Zhang;citation_author=N. Xu;citation_author=B. Zhang;citation_author=F. Gou;citation_title=Application of the CRISPR-Cas system for efficient genome engineering in plants.;citation_volume=6;citation_year=2013;citation_pmid=23963532;citation_doi=10.1093/mp/sst121" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=Y. Mao;citation_author=Z. Zhang;citation_author=Z. Feng;citation_author=P. Wei;citation_author=H. Zhang;citation_title=Development of germ-line-specific CRISPR-Cas9 systems to improve the production of heritable gene modifications in Arabidopsis.;citation_pages=519-532;citation_volume=14;citation_year=2016;citation_pmid=26360626;citation_doi=10.1111/pbi.12468" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. A. Moreno-Mateos;citation_author=C. E. Vejnar;citation_author=J. Beaudoin;citation_author=J. P. Fernandez;citation_author=E. K. Mis;citation_title=CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo.;citation_pages=982-988;citation_volume=12;citation_year=2015;citation_pmid=26322839;citation_doi=10.1038/nmeth.3543" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=C. Morineau;citation_author=Y. Bellec;citation_author=F. Tellier;citation_author=L. Gissot;citation_author=Z. Kelemen;citation_title=Selective gene dosage by CRISPR-Cas9 genome editing in hexaploid Camelina sativa.;citation_pages=729-739;citation_volume=15;citation_year=2017;citation_doi=10.1111/pbi.12671" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=A. Nagels Durand;citation_author=S. Iñigo;citation_author=A. Ritter;citation_author=E. Iniesto;citation_author=R. De Clercq;citation_title=The Arabidopsis Iron–Sulfur Protein GRXS17 is a Target of the Ubiquitin E3 Ligases RGLG3 and RGLG4.;citation_volume=57;citation_year=2016;citation_pmid=27497447;citation_doi=10.1093/pcp/pcw122" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Q. Ning;citation_author=Z. Y. Ma;citation_author=H. W. Huang;citation_author=H. Mo;citation_author=T. T. Zhao;citation_title=Two novel NAC transcription factors regulate gene expression and flowering time by associating with the histone demethylase JMJ14.;citation_volume=43;citation_year=2015;citation_pmid=25578968;citation_doi=10.1093/nar/gku1382" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Ordon;citation_author=J. Gantner;citation_author=J. Kemna;citation_author=L. Schwalgun;citation_author=M. Reschke;citation_title=Generation of chromosomal deletions in dicotyledonous plants employing a user-friendly genome editing toolkit.;citation_pages=155-168;citation_volume=89;citation_year=2017;citation_doi=10.1111/tpj.13319" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y. Osakabe;citation_author=T. Watanabe;citation_author=S. S. Sugano;citation_author=R. Ueta;citation_author=R. Ishihara;citation_title=Optimization of CRISPR/Cas9 genome editing to modify abiotic stress responses in plants.;citation_pages=26685;citation_volume=6;citation_year=2016;citation_pmid=27226176;citation_doi=10.1038/srep26685" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. A. Peterson;citation_author=D. C. Haak;citation_author=M. T. Nishimura;citation_author=P. J. P. L. Teixeira;citation_author=S. R. James;citation_title=Genome-Wide Assessment of Efficiency and Specificity in CRISPR/Cas9 Mediated Multiple Site Targeting in Arabidopsis.;citation_pages=e0162169;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0162169" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=M. W. Popp;citation_author=L. E. Maquat;citation_title=Leveraging rules of nonsense-mediated mRNA decay for genome engineering and personalized medicine.;citation_pages=1319-1322;citation_volume=165;citation_year=2016;citation_pmid=27259145;citation_doi=10.1016/j.cell.2016.05.053" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=F. A. Ran;citation_author=P. P. D. Hsu;citation_author=J. Wright;citation_author=V. Agarwala;citation_author=D. A. Scott;citation_title=Genome engineering using the CRISPR-Cas9 system.;citation_pages=2281-2308;citation_volume=8;citation_year=2013;citation_pmid=24157548;citation_doi=10.1038/nprot.2013.143" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. Ritter;citation_author=S. Iñigo;citation_author=P. Fernández-Calvo;citation_author=K. S. Heyndrickx;citation_author=S. Dhondt;citation_title=The transcriptional repressor complex FRS7–FRS12 regulates flowering time and growth in Arabidopsis.;citation_pages=15235;citation_volume=8;citation_year=2017;citation_pmid=28492275;citation_doi=10.1038/ncomms15235" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Saito;citation_author=R. Maeda;citation_author=N. Adachi;citation_title=Dual loss of human POLQ and LIG4 abolishes random integration.;citation_pages=16112;citation_volume=8;citation_year=2017;citation_pmid=28695890;citation_doi=10.1038/ncomms16112" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=S. Salomon;citation_author=H. Puchta;citation_title=Capture of genomic and T-DNA sequences during double-strand break repair in somatic plant cells.;citation_pages=6086-6095;citation_volume=17;citation_year=1998;citation_issue=20;citation_pmid=9774352;citation_doi=10.1093/emboj/17.20.6086" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. Sasaki-Sekimoto;citation_author=Y. Jikumaru;citation_author=T. Obayashi;citation_author=H. Saito;citation_author=S. Masuda;citation_title=Basic helix-loop-helix transcription factors JASMONATE-ASSOCIATED MYC2-LIKE1 (JAM1), JAM2, and JAM3 are negative regulators of jasmonate responses in Arabidopsis.;citation_volume=163;citation_year=2013;citation_pmid=23852442;citation_doi=10.1104/pp.113.220129" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=H. Shen;citation_author=G. D. Strunks;citation_author=B. J. P. M. Klemann;citation_author=P. J. J. Hooykaas;citation_author=S. de Pater;citation_title=CRISPR/Cas9-Induced Double-Strand Break Repair in Arabidopsis Nonhomologous End-Joining Mutants.;citation_volume=7;citation_year=2017;citation_pmid=27866150;citation_doi=10.1534/g3.116.035204" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=H. Tsutsui;citation_author=T. Higashiyama;citation_title=pKAMA-ITACHI Vectors for Highly Efficient CRISPR/Cas9-Mediated Gene Knockout in Arabidopsis thaliana.;citation_volume=58;citation_year=2017;citation_pmid=27856772;citation_doi=10.1093/pcp/pcw191" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=M. Ueda;citation_author=Z. Zhang;citation_author=T. Laux;citation_title=Transcriptional activation of Arabidopsis axis patterning genes WOX8/9 links zygote polarity to embryo development.;citation_pages=264-270;citation_volume=20;citation_year=2011;citation_issue=2;citation_pmid=21316593;citation_doi=10.1016/j.devcel.2011.01.009" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=T. Ushijima;citation_author=K. Hanada;citation_author=E. Gotoh;citation_author=W. Yamori;citation_author=Y. Kodama;citation_title=Light Controls Protein Localization through Phytochrome-Mediated Alternative Promoter Selection.;citation_pages=1316-1325.e12;citation_volume=171;citation_year=2017;citation_doi=10.1016/j.cell.2017.10.018" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=M. van Kregten;citation_author=S. de Pater;citation_author=R. Romeijn;citation_author=R. van Schendel;citation_author=P. J. J. Hooykaas;citation_title=T-DNA integration in plants results from polymerase-θ-mediated DNA repair.;citation_pages=16164;citation_volume=2;citation_year=2016;citation_pmid=27797358;citation_doi=10.1038/nplants.2016.164" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=M. Vazquez-Vilar;citation_author=J. M. Bernabé-Orts;citation_author=A. Fernandez-Del-Carmen;citation_author=P. Ziarsolo;citation_author=J. Blanca;citation_title=A modular toolbox for gRNA-Cas9 genome engineering in plants based on the GoldenBraid standard.;citation_pages=10;citation_volume=12;citation_year=2016;citation_pmid=26839579;citation_doi=10.1186/s13007-016-0101-2" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=G. T. H. Vu;citation_author=H. X. Cao;citation_author=F. Fauser;citation_author=B. Reiss;citation_author=H. Puchta;citation_title=Endogenous sequence patterns predispose the repair modes of CRISPR/Cas9-induced DNA double-stranded breaks in Arabidopsis thaliana.;citation_pages=57-67;citation_volume=92;citation_year=2017a;citation_doi=10.1111/tpj.13634" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=G. T. H. Vu;citation_author=H. X. Cao;citation_author=B. Reiss;citation_author=I. Schubert;citation_title=Deletion-bias in DNA double-strand break repair differentially contributes to plant genome shrinkage.;citation_pages=1712-1721;citation_volume=214;citation_year=2017b;citation_doi=10.1111/nph.14490" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=Z.-P. Wang;citation_author=H.-L. Xing;citation_author=L. Dong;citation_author=H.-Y. Zhang;citation_author=C.-Y. Han;citation_title=Egg cell-specific promoter-controlled CRISPR/Cas9 efficiently generates homozygous mutants for multiple target genes in Arabidopsis in a single generation.;citation_pages=144;citation_volume=16;citation_year=2015;citation_pmid=26193878;citation_doi=10.1186/s13059-015-0715-0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=X.-F. Xin;citation_author=K. Nomura;citation_author=K. Aung;citation_author=A. C. Velásquez;citation_author=J. Yao;citation_title=Bacteria establish an aqueous living space in plants crucial for virulence.;citation_pages=524-529;citation_volume=539;citation_year=2016;citation_doi=10.1038/nature20166" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=L. Yan;citation_author=S. Wei;citation_author=Y. Wu;citation_author=R. Hu;citation_author=H. Li;citation_title=High-Efficiency Genome Editing in Arabidopsis Using YAO Promoter-Driven CRISPR/Cas9 System.;citation_pages=1820-1823;citation_volume=8;citation_year=2015;citation_pmid=26524930;citation_doi=10.1016/j.molp.2015.10.004" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Yu;citation_author=M. McVey;citation_title=Synthesis-dependent microhomology-mediated end joining accounts for multiple types of repair junctions.;citation_volume=38;citation_year=2010;citation_pmid=20460465;citation_doi=10.1093/nar/gkq379" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=Z. Zhang;citation_author=Y. Mao;citation_author=S. Ha;citation_author=W. Liu;citation_author=J. R. Botella;citation_title=A multiplex CRISPR/Cas9 platform for fast and efficient editing of multiple genes in Arabidopsis.;citation_pages=1519-1533;citation_volume=35;citation_year=2016;citation_pmid=26661595;citation_doi=10.1007/s00299-015-1900-z" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y. Zhao;citation_author=C. Zhang;citation_author=W. Liu;citation_author=W. Gao;citation_author=C. Liu;citation_title=An alternative strategy for targeted gene replacement in plants using a dual-sgRNA/Cas9 design.;citation_pages=23890;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep23890" />
<meta name="citation_reference" content="citation_journal_title=FEBS J.;citation_author=J. Zhou;citation_author=J. Wang;citation_author=B. Shen;citation_author=L. Chen;citation_author=Y. Su;citation_title=Dual sgRNAs facilitate CRISPR/Cas9-mediated mouse genome targeting.;citation_pages=1717-1725;citation_volume=281;citation_year=2014;citation_pmid=24494965;citation_doi=10.1111/febs.12735" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Dual sgRNA Approach for Functional Genomics in Arabidopsis thaliana" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="Reverse genetics uses loss-of-function alleles to interrogate gene function. The advent of CRISPR/Cas9-based gene editing now allows the generation of knock-out alleles for any gene and entire gene families. Even in the model plant Arabidopsis thaliana , gene editing is welcomed as T-DNA insertion lines do not always generate null alleles. Here, we show efficient generation of heritable mutations in Arabidopsis using CRISPR/Cas9 with a workload similar to generating overexpression lines. We obtain for several different genes Cas9 null-segregants with bi-allelic mutations in the T2 generation. While somatic mutations were predominantly generated by the canonical non-homologous end joining (cNHEJ) pathway, we observed inherited mutations that were the result of synthesis-dependent microhomology-mediated end joining (SD-MMEJ), a repair pathway linked to polymerase θ (PolQ). We also demonstrate that our workflow is compatible with a dual sgRNA approach in which a gene is targeted by two sgRNAs simultaneously. This paired nuclease method results in more reliable loss-of-function alleles that lack a large essential part of the gene. The ease of the CRISPR/Cas9 workflow should help in the eventual generation of true null alleles of every gene in the Arabidopsis genome, which will advance both basic and applied plant research." />
<meta name="og-title" property="og:title" content="A Dual sgRNA Approach for Functional Genomics in Arabidopsis thaliana" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2603" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Reverse genetics uses loss-of-function alleles to interrogate gene function. The advent of CRISPR/Cas9-based gene editing now allows the generation of knock-out alleles for any gene and entire gene families. Even in the model plant Arabidopsis thaliana , gene editing is welcomed as T-DNA insertion lines do not always generate null alleles. Here, we show efficient generation of heritable mutations in Arabidopsis using CRISPR/Cas9 with a workload similar to generating overexpression lines. We obtain for several different genes Cas9 null-segregants with bi-allelic mutations in the T2 generation. While somatic mutations were predominantly generated by the canonical non-homologous end joining (cNHEJ) pathway, we observed inherited mutations that were the result of synthesis-dependent microhomology-mediated end joining (SD-MMEJ), a repair pathway linked to polymerase θ (PolQ). We also demonstrate that our workflow is compatible with a dual sgRNA approach in which a gene is targeted by two sgRNAs simultaneously. This paired nuclease method results in more reliable loss-of-function alleles that lack a large essential part of the gene. The ease of the CRISPR/Cas9 workflow should help in the eventual generation of true null alleles of every gene in the Arabidopsis genome, which will advance both basic and applied plant research." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>A Dual sgRNA Approach for Functional Genomics in Arabidopsis thaliana | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EPG40uitWUowuQf781NnJ99sus4hCGqwddZpxzte8JQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490460","apath":"\/ggg\/8\/8\/2603.atom","pisa":"ggg;8\/8\/2603","ac":{"\/ggg\/8\/8\/2603.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2603.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2603"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2603"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2603":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VFq7+8S3\\\\\\\/ptqEnYgxH2waQpy+XOPNpSGYaUwzdYniHH\\\\\\\/YqcMISucVVhp\\\\\\\/kLO+HCEcqXWK6D1LcEQrqXTxx5FnB7CaxgjAeB+AJqp0zJBN7t1+qL6bZk1U+kVZkoPzwIWsgriNKPYyeJcO5BeXO00H\\\\\\\/r2HEM+\\\\\\\/jJv4mJBkFLbotUoxUHn36h9OYE2U7Y5IBel2eoXZjgAJ2gHYN+z3HQWc+mjH0COizQBTQYXwt\\\\\\\/wk6K5X0T5j2eFBjUtt6sap54cFCfOhvhaDBG2m0u0x+kaSs8e2Us7oayLjoALW79rRxC+0yoH+DNMOvbUYtCg+I0eJKdqHbYYGmvBMQCnW9ODG5omYuQD\\\\\\\/IssNRYe7IOaPAqGY8T52QqVCCABBjD7RaKGm4rtVLDHPZYqITM6yqZr6bVoD7xOCJoQ2UHQrpBe3Bt4ebnAhl0ZsqZSENPU8yrY38\\\\\\\/9IXyW49DTRebmOmWdCcQiS7DnwYXyFwRjthnNmCwwYbspWrbCT8uCcSV63Sx33lDfCDXSzL2RTEwbdOdp3hQyJ4\\\\\\\/cRpXcWfoIWpAU4LO5oljtJTvXmvhWEuazt+aeOtRjZZDqjDg3EQ4sEXwO8qDdv\\\\\\\/NxbyWYRisyhjLh+mH1PGxgX2fxV7VG3zdzgLum74FaOQo3ltj9sPYrrvzr\\\\\\\/CVhMXuTtCHpusBwrO6grWHEncBcw+zLXZGJ5S5crpwXQR7Ywk4VWEaTlC39y3u1XhMRVdeDjtJg3sLdQ\\\\\\\/aHkbnCCOIfYVCvlkh7S9FC6WPcfilctKG04muPupz+Ss09VrbNJEHsND9WGFI++8b\\\\\\\/x3kJZ8xjIyXXfohYndJFyiGDCct4jKYttQfM7Fi1TnXN8uLnCr+WwyvS7qJ920NW4ZBkGLyiVy8ESISFKvZuWDKhDzPsmcxJNaWmFpE1+\\\\\\\/hcvb7j2CgZfLF7Y7XgDplOdMwu95u9i9I1Mh2RWo65F2bDkClMlBXa73cwE2xBFVIdZ5tds61DA390JaCdYqqAyOBtcRfe7OZxmcdUMLknd04AhhdPFRT7SuPxpiv2VhdJJY1BvdQSvgSOnQvdkqSNi14dYu5mpqlWOy\\\\\\\/Xvu1nj1zcTwWtbQnLqvMF2oPo7bqJKcjfWai+Dy3rS\\\\\\\/U3Bp2JRAb60mwsHQAuwTzG\\\\\\\/l0iSdSnyuAnj9wvoxFFXj8mb9m587krUUqzm\\\\\\\/gahIqERRQqmW7LvKTn85mZ9qfVV\\\\\\\/QYtQZdwOzx\\\\\\\/aKynJK5G2PzS5iKxVQMG57TN3ljhHskIZjstI+yP9Fg7PHa1b4kEaVESf6B\\\u0022,\\\u0022iv\\\u0022:\\\u0022nAphkqnLvqQ3P116K6\\\\\\\/2Fg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223bd678e8174b0cd6cd88ebcbad34ec911a3739327d206247549053237a2f428b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NchJE7PXJEr8i6wmNqtovPKxTOdm85XKsecWKddPS40uRwB2sIsB\\\\\\\/mtNRrz99gwFjQuwjUzJOXqd0SGubUjr8dLGfItt8003g3GfwNPnb9KC50F4eFqXCN78ia\\\\\\\/vsrzGJMIiN1Oa2WjhK4wYZLjKmIMMGBLTXMdlzHP4QyyeXMkab6bi7B1hdiDJvDvvLbund3Dq6qSaSRkZVaar3H0IK\\\\\\\/sypGwVY95Lq79NmgE1SLvS73WDMwjCUcFw3w3nDuvUV9zroTkK9Cqoa6ow1B28dOKlhhdn0jgLN1FXFCF3Q+ZsGX8wyXivOigOXWVbwEOYe+AEenulvy3ZNgCXI\\\\\\\/xV3MThyaQPOzotO37\\\\\\\/ml1kSecYLP88OD2t1Z2VY3hbfAzagrAg686wEve73HJlMotJjILL5RW5vLESE7ds1e1kXytF7uATbeSbKCtmFlsZOjZcKn97iZ3hv10Dq4Qqy+JF2EFYOWEvQY4OFCZMQE2Xj16hmCB1en6ubVFSmBReK20Nt9uAaQjZeYK8J2zAWbJmcN49aV5smGbzTKKEjG\\\\\\\/Vxk55g\\\\\\\/+RHaSA6N1L5i+9xuXratBEL1w\\\\\\\/nr4KlfCtxCN6sj0LXULwtvZHqxhWLi7Ip3YFv0oz5WlP8C09vIP9UTe1KlzlQY0IhlHqqeiAY++L3gTbGtWzUCEvYJfkgdzRGo+Yzd\\\\\\\/gNFdu0CIHFbn+cmpR6bPQL6yy39\\\\\\\/sviwaNuYdDWqiOCAlkZr2FNOkk9J98iKbWz28J63VWx7Ro5fCWibgXOVGFhbh7g5v3QFWYowxOsP45dpkGPBROoddEemPDMS7i+h9dzwE0Tg3zt4WMDBFkMCLiVqxr9nVqj0AxnTgtBYaVyQ6+p11zN\\\\\\\/ikfPyl3TaImX2pPrYHIHop0n\\\\\\\/nMDbOYcOjpbj5\\\\\\\/Y9Nj2R+FQQufy\\\\\\\/Ni3sPgIdtKbfGTxo9Q+PTORNENXztpQXYHR28iPCTLRJyMS90dJXAX2Z6LKn6d48CjUQhYPqhirEfz79tAVRl7tdsp5RYTcMTwMIkuCENbhjO0aPGnm4\\\\\\\/KLLSksUzLqhXUPvdw2lRUWI1xaP6cf+n3gpi4vvqt5nHOuhMD9mifqbYPi8j+53\\\\\\\/mBYYFy4gHJkuHXff27OyW6Tszv2ObAEP\\\\\\\/OPOWS\\\\\\\/9LBX8ph8yeczvx5m9fNeeoAMb5GQTalVhs9ZFOrfQIYBcSEtrDxQcwKUPfz5pM\\\\\\\/Xu4673TYy+AuBhZnYGJ8stz9NY1k0Lm\\\\\\\/GXCgzoUkW3fMi0z6waJoybQq+nsX5XRjZ19bLQKiHv7SDtMzrzzcB4hYRE\\\\\\\/K2RhY\\\\\\\/Yz8BziHdq1sVSL2oln021zO6ATbAKsg5PfOMjYLdRWpdGkXrbBmXAp0p1U1U5vnvFe43n0FUFfAKIKV\\\\\\\/FRDdgPCP0UZIN7nmTKxf9AgsRkitXbpUGdsd6RyxvHBT5uk\\\\\\\/Ymdg2d\\\\\\\/C\\\\\\\/9\\\\\\\/Vvdu03xX7vXYDOVA2KgT7NQEOwDaved8nnTgoN6OETXYfVjvq\\\\\\\/JDG3k21ZED9Pdeeyza26xdtWywVljEm7HaXVW9JxUYYO71Db2DgQ5sae2k24C+Nk4cbvbJaCXxZVmX7dcJ5Dx8kaxTvSvjY4u75s8znx1BQ5pYRZZrCQz0xcIKoGaTX+gtSIpskxoQK98y\\\\\\\/wLrVFGdEmlPB9HmXwX609ntCctZSEzMWfGfBoZZbJqXJ0hIPFanEX7cn6lhe2QckU3QJA83\\\\\\\/W\\\\\\\/VWL4ZDygspTYutQ+K3fU+GtOcGP3qj1G7EI7k7Uy4usmNHi77RhCEQ6YbPJkt1ssGCPhGPv3OfDZQ2yqy8ciWaljQpaoZ\\\\\\\/+KM0QBpkR39poH5\\\\\\\/Dm3dDCSkRWHaZI4XyEAFVxOmPss5ZIY1h+mo3Yc+zEvaDff6FV05eBUrmKoRp3dnVF8Yc6VaxKYBvP+KMXNpOtfZKs1EWpEYwQwbrTsDslBJAMhl3ansaYYm91Nkff71+4Xb1S8ai6OzsWYFYwF987uEbiwFyGfpYfVAE4o4qeEFw\\\\\\\/jejfu0K\\\\\\\/JL0cvCqKHcPSfyHtkeWxVIPzNHDWSAryqmO4\\\\\\\/bnhxUhjeLx5A1+61xf+ONL9IQhjBcsp1qtTyUf1KMPXfLGDxAzFcmeCDCm7Nr1J3hHt3RnyyyRJgSaMUrC3NoJ4+OcyQA\\\\\\\/Om8AFyw7pj9z8+tAE7m+bQyD94nuxZBV1i\\\\\\\/7mFcwOfwVJMxFu9d+6S429jYPqi1B2EG+JzKVfFZA6J\\\\\\\/lwUln7J5ZFc5RMS0anYdMsRFTuF565oKzcAH1atg6nFqmOA73fEhJ3fXnDPpYZYMHbK982j2GLhoR7V0gtmTjj6hGdTFi5MYzvT6PWnTK56A+Fw00lfusKpMJdYug\\\\\\\/gv6jXa8dZMNeQWda\\\\\\\/T9iMViUi5sh2K9YTyLQWXP1RlFHIs4XGNlkVSXtOJ08tkYTApzL0mhQrioQNrPuVBSHZK4vrVipkcK9BiCt3U4qvgN6\\\\\\\/tUiyTil9dAXYMnPFG9ihZJWwZlhGFReTv06fEp6W3MIQP824oanwRTyV0HVvguA75952De7s2wur\\\\\\\/5eWJ9hWEVOhZiBBgJr7rSvIFdfvoeJrq5Qih9KlsnnnwGIx3MgBpj7K2o8f70\\\\\\\/KojOeeDc22bGl15xuFxnr3iWNmVWrXn\\\\\\\/ZVOY7hltlKuIiZC7uFjqcTjYoV4uF9vARkkS542zDoBmlzvWzTMgRGbQZZvq3a1Vkk97iZnGgLf0mT+PbGwvd\\\\\\\/lqtTAH58LavwXqkqurC8cjaiJCTKAQDHYRz1Qy08oDuRe0433pTK6fmzqWIU4fo7PqsgB89W3JOOA24SpX19X0hg7zwOTIA3Co9bRZjh2u4NBIYtLKp4+GwTAZQh0db3mlPM6tiHMEAv2Dpk8ueXsZ\\\\\\\/vKsU2ECOiA7TzlcmOHwZoSy1tr7anrAomZu+n40oHzoRNqbsa3O7FacDNnmami3aTNXSXcCZxU+g7zXQV99kL3kcQg74BU1UQuvp8cXoYNmX26sh4b48nj1e4Kz3ZKKuSjwaDKZy0ejXiA51P7m6SbNu94yT0MI7L4vVECglPpMjcX8i9IZ0HXp\\\\\\\/VrmllzS+tYZJ7qzIQVRrU3XMOc1aj5sJoBmPfOGs2xvJD6v8VoKHne+XoDyzm9LkYfBViTUzIRlqNRHR6FfL9qopOGlYi6ZFVriYewLRCcZmWtHCzV7yZbEwoAeQ9YwSzxZcWCdea0LfMVhTFgsdW4eLV4GiR66t5Bptzzryx8Bzi87yosJh6GAzbZgLvlYgwfGtCBpaBNgFNe6G\\\\\\\/17Vh1OSAwz+baiSR2NIK64IjfamSXFPTjGywHNXMxMJ8jjU48wE2DtbHCV+aITcybvWxOOlaM1PYUU6JvybRGNSQoUCWihI2lYLcuQTWjKHu3fAfWBClnGld2tcRjPS0PKFyg1UufuJZlndGwMMeAaajTgInBc2WsbEI7v9TZ1\\\\\\\/AZx4fsEUKyAtpgobdWq1ozaSEMIyT\\\\\\\/\\\\\\\/lIOZ3+qLi4bBPrlzJ5zuA5w6PwNCjObD2Nfg5hsJWbNfkbMwoBuOZUsD5Dq7EZ9w8HwGsWDBsmGJp2DuMlm\\\\\\\/RzJLyQxdtIKss0KFMr5siMrljadAkODTGc42YhgphTcMvqwlfOea9rPISE2yQzNtR8devxBKR8qTIjHAgFQab12iGIGWwTz8ZeragN\\\\\\\/LrgCvM2WEJBRtKNEu89P1o0IQcCfT9MtUlG0THugsM8JDNtVLcw85kbmQTCg72HBZ8O9SdctpPZIw909pN\\\\\\\/2uKQuozbgK3lABc\\\\\\\/jDtsWs+Jq4YyQ3ZSZU23Of+sAf54vWebW4sZeFF0rr4oxvyXBtILDR8jwKS6S0BERfF7SOkogduSVhKFAsNvN96e3zpyNz84WRpXqfSosGv9+j8pDDog4W8onNBIRkTBfVSzZPl4SjXQI0\\\\\\\/IXV29fOaZSIF5TVGpvxrnljvbo8AUpyBeuP9rNugHF5CoTOLe5p3loX5ObdrXxok1E4WmtLlaz8rep1rNyVlTwpZVf6xdkoNBjlJhZ9epikUDd3onp5qTcaUcLMr5jhYRurHzQ+1Qs\\\\\\\/hJFkkFEGGSwLRkAAFzfIkiPjPmwAN3gS5NOTHD5kQfH\\\\\\\/QUST\\\\\\\/RWrDTk\\\\\\\/Hk6f0ChYA0z4UzFKYeX7LedHyPOm2dPHIKRaZ6irGqGIZeG\\\\\\\/o4QAK4OgGSrvUaZwL7enjLAD9w2vT+aKXxqN319Cm6qN5fMxNX65YcD4GS9oiyt\\\\\\\/uRCUFKdtMlyLKPcOtFpyAa\\\\\\\/96AfS2b6p7QvqsUbJbRCYbS9v0PRXufrmRjOaZpN7XfT\\\\\\\/pWbCw\\\\\\\/E1mup2yZyxT6gZ\\\\\\\/V+teM60xRxB\\\\\\\/ZTb9Mp+f\\\\\\\/2f6abGSRZREtTLg\\\\\\\/kl5P8qeJYy1QN3eQeGRoBy1icQ\\\\\\\/9cmAM97aD3GfU0kXcmhWLsGBDA2822iKcQRovAXUqUBWcsgGDXVDLVaV0dfDmboXt\\\\\\\/\\\\\\\/JpYvnv7qqFbsPfgJtnwhjxqq3WpK9OL\\\\\\\/7PAlEpPohKy4DfY4px4VFj8R\\\\\\\/A9uGnauqCPx5\\\\\\\/U2o41PeqBUXOQj4q4lJnVgOJ1Ilmtx6LIFLSMVPXy26IlRZnR9kDfqqaYMr2FT6jhY3lmTBPJ6IcOK9blOVykfOowx74mn+PoN6YM5ELPWljAfXc4urcyDW7rogaA+t5px9lYAWxKgwS8GYPs3iEH5oVDDPjA0zsXr8EUsKzK8WeHJRfwYOJWrMpyf7dlTwCZV3Xkc1eGvwQtMEvwl50dZ02TS72ROKPKHELGAejLMMYx0nuGRs4DAqt+GcMPRwMmea+JRaO9+8\\\\\\\/KPYChhvbdo5YztCuO96SdAaDr7eQBxsrVcu3NvL5nEuygNBP3jQzlkvNt\\\\\\\/jadal5Rt2yKWQgN8E9g3f4sxqGu7gQF04ria3wGbaw0jBQZrlGPzvTdftsmu7zjEkJ2vUXWvF+NUFUs1J2W2UOEcrzE59KwhwbSAGrRzn0dt0pQL3mI+xObOMOnsmAd2uoz1YirZ1t+\\\\\\\/6p90ce+ndt4UZXz6hcRRmns7NL+cBwLVCH9daXD0rHhdbVOgy5j3qd3RAx7bKkOeWeyS05o9XZTj2Iue5yKdJLlol3IElaM9dkjMZRDSF9YM2Uaxn+lNxna1B2ZWnZRlmo1kuz4VCW0aLAcbY1hXGfiydPFN46j7GCuvxWpRYJ4jtGBySf4TmWyz94ZlMtBHOU5MMi3ayYHbYZ8RvM4CipGUdbjx4uPn6+ROnYCQUR5G77IT7v3\\\\\\\/4ivT08UdBgWaUXns1qzJoKP2VepIh2CtAjyYGptGkK3WA4AHUF7PiZ3mCmJRjhgq9SWeto51OSm1VItl7wv1InSVcDnvpjzQbm4j12tYvmvp2FUR9nsi0Z6tLpB8p1gOM8fg2didnKcGa2Tyhdoug7pGraNTGXzRJ8dBmqh\\\\\\\/r+e\\\\\\\/K9niR9VWE\\\\\\\/R7v5r534zJIuoYoNpLKNA8nTqeej4B07R1gckWy9Tyo0N3xKJ1gjBEPbhEW7gARVy0XyNi\\\\\\\/TbPvV8okEYIm+LUCpecxDrtd+6WNsw96qib7sp5ISPJ9aAxl5AsaznkjIVXRzb+uLAXK2QiscSaMWCpyAccyHtXdYbjf5pGFhQxBeNU7EQX85Pvw8qvQ6b2Amm6msUitiJeAiUo8hBwE8dh4JjgEvcaK2PrnMzE0R4CnM56CYOv1x0xMT4zDjV6zgnJLCG2c2msgJsGElPWeUHbMeOW71DpP9RQ3kscxGVUykPi3tD\\\\\\\/mfvcYnOQS3YmBlMyjmM7PZvBxItMQa3vadRIyWd1lw6hSpihUhql93l80aKGsX+CZH+YJgGne9GAcLEPg6NL2jsNPhX4QJUScQa3ub8Y8nqHJDAB4xAK9ucmp+MoklvzUZhGnN0vU4poM\\\\\\\/vQrhxzl+u2QleWD8hJ8uXTajkaPufGZDkR1kdVnbU41vx6s6G+XOhnTjK0b1S+UqGs4h\\\\\\\/s8kjajfseBFnZ\\\\\\\/2Dq\\\\\\\/3rdYJCiKdwycwN2S\\\\\\\/IPvwq0zPR4ry6H0ndJAU8cCm3NHO+9DB78rH0TUbZUp6AmrHcv0DInFDhYR8wbCygY9rX6x9IUWGGfgkWNh5BzL42IXa77q4BiXpg2FW4c2MXIb5ExT5yNKG+290NWFkwL5KHxanWWpSJPRAj6U2hlfMLf5yB1uCKk473GyNnXkMnzgT845J\\\\\\\/7G07nfn4Gh5qNSMh4l2P07mf1ze2Ie97eZaCSbmoq2rXts25nUE9r5zxqdsUtqQ9tnJHtVUJL85XnXYyQf1mkbWQiPE5IkpDQbjOyHlb0cizlJGDWITBhDgAO48DzYq5THYeJvFhU7mpyv1fML279uIAtwgXZxkEGSFokKtPkR\\\\\\\/86UrBGh3x0lBrdjr69mO90C3rUyqJ8G8cwSEPaMlhFEk4lvLephFpCr0Yv9JVkhxy0xVHa\\\\\\\/Jjcn4QoUTsP32Og2gv6g5x+9A9xETrhdl\\\\\\\/msoejjSGP7bcvSONmgZd1mui3BJowIttM7HrwciVK9Dva8Enwgc04qIOUkH41QpNh1vTcQH5tx44OhocGK+BM16KlGXePU5HmKSvN9x89tDhqBOxQDEp2Kt3zTuDflhCm8IACgmiwOgBrtSAPgK7HZ3X1xx20ALnQbcEKDjZo\\\\\\\/ZfD+5ZwHUSn+98d71KkQ9qhw8BEvxK9piof\\\\\\\/ZJ8o+HShknhsokqbVZOBJlE4X2bHystVMI\\\\\\\/I7dGeQQbq+VDtFmX9kBIHke7VvzR5YYWLWG\\\\\\\/spCbteUJK4Xf79jUz+JK1zj9VpAXc3jVn102CFWUt17W\\\\\\\/hybRWgaJuHSmMQYYI8Ve6rBVOxHscgFoCeB8tG4mPXCNJ2KVra3JjLxNlGT2j3NT2u7ZzJnJrATnx4mQXYo\\\\\\\/jkwIF\\\\\\\/enOI1p5Z71nlt8YJpqYP7qX7dZNA0tlUbVyd6\\\\\\\/k0CnJZyS4yxJBTvDhGAvqyUbvreoUcvBIa6LRD7yMzweRXZseGZ5aMidysDJ9nV7WRa6RAR0uXeB43vt6wjnJJ8p71E9d59D5rHBkrGGdyY0QZXeOhN5BF1h\\\\\\\/ogbFTCIpIhxeTWO34ed5TAyOQs8fmBWVwmwfwAEcha4Y5+ApEid3XyWDAIRCtqDcQ72AlHlfdYgRPF+1i+3yH4JfITJB6P7AZI\\\\\\\/mOHgwApxNnMhcZFqOqPMiADst7NMNhEnTkgtQ0Pmq7CjkLUQYpqG8feAzu4LDnWfbC2vu+gy0jVckANJhEdYkhjLU+YzBECbJwTiUZkK0HWEPZPuRkjJzCvfsSa5IhbLZRhq+jJfS6Jkpf8olVJhijtt2CHVuT9Gjps1HIF0e2KEBOgH72YL0BxMwuBXeHdznFTE2yTGa1rF6k4k+hFlIP8esw54IdpO\\\\\\\/1aDjEnwVtRxZlXbILCB5J49qASm8QHKsshBp\\\\\\\/S1bav8J\\\\\\\/oM4Le2sCDblcObMTp9KI1lSdWXr39lBnxiwewRAK7MLDEaUiStmjwuuvy4G6Ps\\\\\\\/n+Jn6Tx3G+PJ\\\\\\\/W99GqwBjPKtSdDZP3jbt\\\\\\\/TIZKj6aIKbAzMn67hlVGnwghUyb7d7JkZxGPEFRiiC9OH9Q7JIdW40i7s0f\\\\\\\/7fCtWL4WeiT6+nAHIYQTjL9B9BkneHWE81gDeeBkynRV44968TvCrDhCD69HniTynzrMZRm\\\\\\\/FaocMEG9QAWLXolN3Ls7fwGDFFi4+uJL4w5L1\\\\\\\/tRqt5KCTr2J8xq4wf1E7wIvaTP5unvGK2y9TNBEecCZwOsVD0PpaVz+o5q2uplrYVH0rh0afIPNbYUKcBNxu8E8T9fsQP3ET\\\\\\\/0qlBEDfmCB\\\\\\\/nyclrb2tS6kwe8Wblr2UIMvWTDYgylsj2zA3JRnU+4LmYMQXh\\\\\\\/yriAcbJlmZiYxFi1ktiVzwzbaQ+d4u1XmRthDmyDQh+n\\\\\\\/gH5Duy5IKkn7BgaxvrLzx56TJCn0qwz7PSooKiRHCqSmf8rPP8bo85k5q1DYrrsku6N+E0yQ3SnrIAFPDUsNWKg5sYg96L5L7QoHjC8t95qz8X5ezMXbH4woL3VxixNz7TPdwXAIJoXJPufP2RfLtDQEfoex5BAPeb3v\\\\\\\/n\\\\\\\/L3ohEdpeneF7YGIvgGjrjaIqoJ57y2aWYxk48kJw3xo64OGM8W41aZjCxdAXovoPAsQdewSPRLTHGl8lPW1gYhKBKPnkslOHZ+Lb1uu7dWVvI3b1pZA+c4KwPyKZfwyeaSBVwjae7XsAKxm0FM\\\\\\\/Jr0DO01sk9Os00GP\\\\\\\/dSUNscB8LQB9J\\\\\\\/b5yLwUX04PCneLMNryxJC5\\\\\\\/1CkcnGNVz90Y52BLNnZfONKsdk7chv4Zgt1BZQAB9h6W+Ewg8OxmhH1NPUA\\\\\\\/0Pv7bn8bNwWz7I+o0NmD+r6sWzAAH9yxpfQjP4KjkbuAEaSoOcTw6xUg+qgWOdHQJPvESGQx99FprVYGb8Xh7YNVIaAHXay\\\\\\\/gxPmpa+xLinZZE41Qj3G+drzK2inoEiJEL2Yv0pk2O8z\\\\\\\/a0Y933GGnTN76hKwgH9VHACxZs\\\\\\\/pemuD8g65BKJl6To2ZxL\\\\\\\/2cWGJmgPNEz+hK8HNNIJoB0P3MZIf6vxopbq9vamkjNO0i6XmJEl5zs9clNJq52SqMIWNkgpjVqcB\\\\\\\/gl\\\\\\\/qY8VbDzl3DsRWw3D+545isC4YaBMF+73UkLwFqsLlprtw+i6I9AuunozM6\\\\\\\/LMobbpHDdh5fFFgqAlSl2vUewe1jTUNDvorcgLcwtaiWSub5jIz9NEBcdncEL+8F91K1ND1tzY1pg85cWuYSXYdYRVpic4cBF7\\\\\\\/xK1YTsBeCiYn\\\\\\\/FXfftUXH7yDKfDu2mRmmZyqTPeUtNJfSQXxv+qYqTp7BwxkCXj7vqlip0l\\\\\\\/j1yF1ETXtk58H+aMeqGGWE5pO7dwVrLEOHoIgR9zxWHeKcSKgw02V7sKAnVWIOL6mnhsDkZUf55HQ8O548GT76O9oG55mVDGdyGXP2AUafYUxt2Tb9A5KYb87PfT6jEb9FgLE9H59Kar9cw9K83V9ncBUSD6r\\\\\\\/s3bXIv23WwOosi+Sp3No+3RTaT+ouQhdHBuzTb2hf2fZcUt+95KWIcCj0ICRbO7iiR8VFSXsWM98GltRe2rrRXqFWapZ6DUg85V49clQkMXeHgyfaTzyj5fgB4JkhOjt3bI0rlORLNRL33zPAu++Ia6QEotBDl3G\\\\\\\/iCYC5qZ+uf9UEWhJ+NmzBcm9YH2pw6tbRbwbrJZw\\\\\\\/jEsEeZk94x3B1XSojQCFlucHHm9ymWcpaFFLTcSN\\\\\\\/VvsB4ZN4PCGdJXeWdUISTJqa2whXDpPkaTVkxMTTAKHF6YZUZIVcYrMoWHUeXtwqEIe6VOgucFercrdVlWToi35ykOkfHkYEXgADvlLDSlXRokGZjHkgq6n++1bG\\\\\\\/z2FeRW4W\\\\\\\/bInzZG7GZQiqvHL1078quBhoJFJh\\\\\\\/q8\\\\\\\/V\\\\\\\/k8EpzY1K4Dk2BO1cAP3eoedlWBJrzTyGGM4D7+fxahbiJSgzOo6TXjPfIwx8VlUzroFfkzdLhhonjb\\\\\\\/XIVY5KLqP60aGAEK3OlH4EApyqtN5+yqNk3nbgBNMy8b8niyGmjBiEDxOSAWnFR\\\\\\\/VuynRRe+bYIGa4mB\\\\\\\/rD8f3QixVXKGuoMVcOWDnbWRdQkx9176vBWwhIpu0kXNzjXpUe3Q+CYqP0JxtSZxn+PFHMAd0UCIaisOVbfuFPlTFVi3mRDQYjwPGiqjvH8wiP91u4QHQRrBMwyWvTLaelhLeom08lBHGKeGhTdAb4J0lVCY8Bq57OdUdvOS+nlq25pdtzoZ\\\\\\\/lp31QX6tGjIo1jgMy\\\\\\\/CDRh61Hy7WV\\\\\\\/JjlZNheLpXYgphoIqgOgAPseqx5p6eYEGFaY\\\\\\\/EhDj9ypkdrOVdgGLjFTpLattMuNpx6b3F\\\\\\\/uGrUF1HwdV7eImFw3ExWOQ1Ce8vbx1j8ERoAuYo4ii7bhDdiO\\\\\\\/x78nfS6k6h00dO2nbg99FsVuMpeuJ0QZuUcSz3XKZbMS7XAsOWcEWzJF5RhACYur+AOEIc0Yxe\\\\\\\/5SQE85H9dNeZimfAyVq6vuiUYwsCXQUvEncgBQwAY0E+XpTH9O9438YQpRZT8J02w9YmLJJYmxiFbet78Grv2PQCQSZWqreRmNH5YTP8JySNAcvMxO7ikfE\\\\\\\/G\\\\\\\/BU4R9N360g7b4Me3l0r3QEdu6QLPkXWGjnshYLAL\\\\\\\/NAiHT8Ue0Np3zsBDmn2lJLvhF76kpASf4Prsi2Dc1M7vAVP6zwZoB54RQSmxVOlAO66KAWny+4v29uHt36OWkyGB\\\\\\\/RaF6I1SzTrtlI5ILQeyAvZhQCHkC0I2mwtPJZP6YBDDdbPMTW1P3LiBkNDxwrrw3jXrHYJoTP+KN6lYNJ7sXQrwgQRTef8ghAIB1DENvsdqnHo36hFroyFIpsY7qfCqyDzu1DRJVWKzOYKkq2Z8ZQNDfJiIr5KWhBswY4\\\\\\\/M9ch6U8EA4p5JFCoy38JAUm24zUYG5p1TWp3ba017U6k8TACnA9MBp7y9Z7ydSLos1TW2lWJXgVn4D1rXbQ28A9Qyj\\\\\\\/bXZGU4RcOzIXK3TyxJPWxpYKKMVd73x\\\\\\\/WU+72JMxxggESpGJZOqzplWPPIqm+TNOuwCLQxJsrikzk5Ro0hDH0Z6hLF8TjSz9G6z832P2M\\\\\\\/PCJTunQp2SIdhvVIuyfRc0Cnfshm2S5SllAfRcSt88IgU4s8rg\\\\\\\/VUQ6nWqICRAGmO9r+GcR2MnCwEeDsqcotPYMf3eouNM0+zpSjmaDXPhQvnq680laF4sHQ2H5ziBPvTJzOjr3W2lGJNA3aq5hhUlF6sNeapYYd5YdlDHfsTV+Z9FfxlIsqwqvbpECE6iFPJyE\\\\\\\/VbiK6UyLrX5Wnzd9om+mmMNMUnZpQVM8hV3j9zSzwZMvo92OCIYM34wJ\\\\\\\/43URoFPHNCtaAERoFEoUlJ2pJpBF6ob3c1mfI\\\\\\\/p74xFjEQUAPBinJlnje\\\\\\\/XxF54UYLtUntl5AWPr6qfwf4SIvhx+n\\\\\\\/lCWnnQnNwNC5kc\\\\\\\/WDThJ6Z0uZQ7ThhOyJ5iYK0gX4EsNTKdTriScqn0R\\\\\\\/hHjZ9whbR02LCmBt79ie36kR\\\\\\\/4EtzuhZfXcnjxr1kNnP9metck7TZiUMt7DpPaSMj6R3xLX6ywVa2LzHTnnS+KvGtS0kVKfSjmC\\\\\\\/Bb3DRVWC2B67hqRBTOob\\\\\\\/E8bpzRY6wEFmChPcP72eZoUZOVfG69DEgfFerVUv49dRmpFtb2ZpRci\\\\\\\/TT0JbWXzwLuGUNv45CAdXGrCU4ftvuYeAEXyxCpAJZZEosLd\\\\\\\/gDcDIE7VhhTDJ9V2NZuPvKIuPG6LL6IdRkz0ev1La5ZCJk28sQsPPY0XdPDohl8BGIs+QZW1XDJnwWx+lbKRsHnIfqxiCOGmUkbifgcPaDEb2DsL6DLEZICwLTMdGS+EgewyVn8ieIZDUFZ7iVoJt1yeH\\\\\\\/659aVkNj2c9ktyV28SaDZ3yPTppG6k7qZJf\\\\\\\/u0bZE6dtdH0+X38jHMM9Bo+qp2McojGXyKRMyoUElw0jokewQKipJJoJLhR1e0JZ7myq02Zr827T+OQOrjosURe\\\\\\\/U0D3NP4\\\\\\\/A8CNI4SumngD1Y03scgf4vwjTxOEKZ4NSEoVnqs0aBaU+2tknX2e\\\\\\\/y+4elSUOzxPpt3zXzxbjENb9NUp0EuV1SpKSzD88ZjjX+czfnrw5PJM40omOZEwwx5YsI6E0NF3+ym7\\\\\\\/mYJsoU8Ebr79Zl9jli0Hg4gUco0u5nwILX5oShC7TTq3zLSbBFdWLboMqV1ETmTQQ0XvN676V64\\\\\\\/lY82qRA0LUPaNkEQbJXDMoKdSPgwgfm3It66UJ1M0l7etiAw+kJZ926IOcFTQr2DWvItxajOzGGahvlXPw79maCm60cUZwzCtRWbafZxZGAFF1+YO9kk4yyy09j1+WoCcQex8GIN5jS2\\\\\\\/iKPjuSWyl1fV81l59qR0Y94dB4jpTGLxO8rdwmQQbXGlyhqpNa08crtvljoyn5qjoF0YT802CTw18lYkpNBLXqmxVCX\\\\\\\/N9D6abjxHG3lLqtp5Rx3tldlehBU3d+JtxkypSPSYlprSKd8J\\\\\\\/b2lzBuKhpqL1rGODj7NbVOnkRaUISRGbtNFAE17EuJE9ITQt5huxWsBwO7DYJAZJate\\\\\\\/RZbcxBW2oe9zJw2xTjQCmpUzqXlWQJTTIToXGuJgOVBIL2Aq7nrdqumj7s3MN05r2JYiEDfbBZG2qOs\\\\\\\/+Motp1DK699i\\\\\\\/ObaMOzVq+cxgQxfP25AMFIURGyWm2B0fKQ59uJNaT\\\\\\\/eQqTqwqbnh3AQcPl5OZlHhRSjYMoljfkU5RxXPcaCtR\\\\\\\/rGweiBmZuy+793MgDGGDcWtuoMamZ0RKClylZF6l8Kc3V0fbOTMQfMXCJoFoT9HDtsAB5kidtaRZfvT0wf60p3iCnTUxvOdo\\\\\\\/ydCBPbZ8IwGJN78QeqFZCEeppayUBfnEDooeqNtAqWhO1Tg6LnnCNynlu0gCiIriDOgklu88uNfyqltNzjo2cCmup7aQlmQXcKVUF5j1U8mX9bZ9bVG48jfu4AKohwOhwqZvJQHTk7\\\\\\\/TVu\\\\\\\/fg2\\\\\\\/Ev5eF1W9Ej5Fx5MJknJQ2yzKyNwh3qVR3zUCA\\\\\\\/HLXwajXo1SWxGanfxnRkUZ7l\\\\\\\/lBI+\\\\\\\/E7E29P0bB+Wj75FlJeJd2seT19uDmNNBMMm3cbVMSGXVYfPCHm7YzQoiij34RPweD7YM72DSx3BWPWmz70sAqI8T0C2Sk4lC9\\\\\\\/UaughWwKFDVpZkdOLFfXua4NVWfk2\\\\\\\/lmdLMIn9rkKARJvTZeDv9JnLHJI+bcqJCsBuuAxWVtCiM8X4A+4MRXGlfLMnq47Ev0ER6Jm64GWpSPVc7Khv7ecunoiJ0XUK8XGNKmw\\\\\\\/ZxC5pD\\\\\\\/17QQJ2IrlGIbBkOylJ5El3gBMD56VEdYrFqUjjXcdXTTostnLMRDGGA\\\\\\\/aGciWNPgl1qprdLOyHPYRM6XE79WiMObNoIqnPcXAe+6Cf+j73fzr6ZWN\\\\\\\/jZaTdpQukTVKpOBCyjk4bwObj1PqnGtOrFOsxn0VCeNEJY2x5bsaEGKgpe0ldSUo4cuYNCT7czIr8ENcQqbKCcIlMcfMTIDHSBGXK0\\\\\\\/MAfVgHMedCi\\\\\\\/SbEX9yTlRZirMJXkHo9DenLthsweGNkbDWEhhj+a8yER8ZCZpOAXxKH1t4j1BSjP2IbmHZur+HYCkVTUxZkQz28oPWVk3AWUKZ4PcTolbMLvyRT5IcZ7I\\\\\\\/1j1wcyxfWWYlryKDSb9zcD6xIIHo8A5zJ8UDXac+HlLZsq306DWbDVwBpDzmlUjlwJUrpV0+GDJ4jStka2DPYGHSVCVgY1rqz7OdBaaC3vQ6wwcQmOX6mF8VokfK4Y0G4BCf7JIxYCURLL4\\\\\\\/26Oz2mTfk6isRcvFQT6ojsB1nM8M3d1R4JDXhIOqiZA8S8kDo5MOQ8pATU34XjNMQdR3YpUT9G4Kdn5+IUfrohsyasmaYoS80gvl3BAvHsF0uJLNVZaRAR6cC7NTYCPDDiA5IFbk8aMyVBrs8Yj5K\\\\\\\/d\\\\\\\/jyHhRoCDHCoijCkfkC5ixGMLZ1i6wGPXqjzZXZ4plAq3srEDHc7A\\\\\\\/MvZ\\\\\\\/VCHOvzthZ0aaPagqAeK2cfHr9ocq7e1gtaVMMRlWa1mha439hCHRZUWjWQL87x8BnZNfkbduCdW84iseTfHozE0Gpihr67w9sDbxedhwa0Ng4dKqlJh16vbBrtLcxzNzIWrCmJO2tmcl7ocZkwxtRifO2lrLXIWsR\\\\\\\/3JsCGW5eCuy2cxpc0F\\\\\\\/RiM7QFmFdPCbhOZiqNSKqbZzkT51DAMhbtvI+d8Hu4RpL9jV5Q31Bm6tur96wcF3C6ZYjN1mRD82x16OWw5vvoHkMMGV84mtLE4atZPMtAXC\\\\\\\/6+nkffsQ4KRbm3HSTTCJxhd86nX1oPn5yKGn8DJFljDa+O6KcdyGJ4AHqYMIx95rsMecmxt3PYkFqpn2ShaDYo44Sb8cfNv9LtJrJ8uDsc74+8p\\\\\\\/SSjYhE7+C3KxPHJ8uMI1sSi6ddsBiGR0EWzffbbck3TAZdoOEAAWOR6\\\\\\\/AaZ1IPy01Fdo8PGnXObMl3fZiUACMK8W1T4Twf76JvOd94HNCA2YQB5EBtwRUzxP9f8Spqirt6Lks7jhFPWocUKeVyGWsojhS3mrEVb29ZWr4tYzIr2GEV\\\\\\\/2pcwmNtX+A5TNmXpF\\\\\\\/78J2XMhstKLG\\\\\\\/5dosCgDfw+SHJsjHRVIUDLqnp2TZfnpkRQ+VVCYzC1kjkVSaxXzt0IuuYdJ6izOZPcWIUPyZoboyOAD3pWxfGuu7+fYUs9d5c6blPFvv5Z39zLePK\\\\\\\/UCIVpOINdGeSDckkKbvtNpfny2qI4irMG6OCDGVRZsqoIJBeIDpOU3DKoOafwts6mlEEQKoyd85QMX70HnhRy9RRjcWF1VMidJjSYATRVTl5JaW\\\\\\\/Ndgn\\\\\\\/qgMulP8xaevd9JUi5DRZu6zf3P3bIkUIiybpJe\\\\\\\/JypvIqH\\\\\\\/uBUtj\\\\\\\/CNWexW\\\\\\\/Km28UJO932OCo1i48J3fpq8mxaNeGU8RumLww8EXx50PxpVC7frtVjvk4Th2IePAUjfkJBCg8HcBrMIrvDkh5XoHzByRV54JAY4KiOJN1GGG8MYc2BzyLNpMz1r+aTaDDGwkYsCNBtFTUe4+yRWnumD\\\\\\\/lNmYXb5JtPQtTD1CUvYOj8GDL9+p2VKQxONxLEJmO3FjYJ0AxGEXCVqnvX2Sps4BMqUBVOmGT1pCJztFl3+bTYfsIVUKdksyOaEqIHSBoL2oM15eOpDuHWYrtYm0CSy5ISUvkN61GvGcxEhb+MFLCqmggTZr5WEHtSSN7QXjN0yitf3xBFkcDanceun27SzueCHdCeeScdWPHKyO6oD6oxp570sZ9Q12JJlc8x\\\\\\\/Q7lrV5f1YRP3EwzT0LiuHP+E4E4EKnEMdpiXqjxTxjl9Ut+pp6JAz4EuEoudBsh0WzsMM0egpIhp7Mbta648Dp0Jwo46nqRLsv6XtmulFyVdaRKmmeifKt1Br53m0tUOhf8ezYr8P3kOYLxcXdsOfhCR0R2zrspUIIlejzdkQPf\\\\\\\/c7nsRUTDBgTUJXtYVOqU8ZKpQjFIIj2ThPc5GaNyBeBUWoZ427\\\\\\\/\\\\\\\/gOi8vuhpgsmHRZdiKaj\\\\\\\/eNM8PMulGz5X7wlcukFleY\\\\\\\/SMb4uAneVGyu0DtukGGKXstV1XGOnjuWJoThO\\\\\\\/tTiDnqX9CY22czSwolzIMTj7mYAlzqVkTrUGiTWnh\\\\\\\/z1cvf+\\\\\\\/rRVjoeO38C+HWbflR9iMzWpYIBgGk0wykIM17\\\\\\\/rV1xtRrCUIRQxE8ES4Cw7gXhFpomC+9HQUGKzFzOC+kDbTwYpT6asOkx0L3bEuzBQLjB2UkiH1UczztvFDCCrvePwkwijSq4F2xi45H2UP1gLav\\\\\\\/95F0YSbw3Fm4aRmQfBktGboMTv4c1awwLgs6dXJd7imyIbPck+x0nuciWuXOjNsAJNas+L+oXOKXnpzqG6lPSWIp61pk53\\\\\\\/ETRgI14LSEexsCFAVn7A4z2w1x56y8UseJdo\\\\\\\/YKZBrA7L+fc2hEIl+2R5xN+zh7TpPgh2q6azaPripHfKRC2K6aYxR2LjY1ifzD3Ih\\\\\\\/NhGIGGSZMWIXuXG3cY0uNJrZoU\\\\\\\/TpfXnVGzVxdVNf+Ky3V0X9geWjeS9z\\\\\\\/nCzw6ZYyinkWKYP2vm4dNllZe6g5OOT5H5nG\\\\\\\/Hg2+xm9733MDLpC4BbZpaKv1ljwtlpNOHc9AbbiyHMbAKq8AxMDdWE7TW7ymxRKfsnZvGopKIVJEVWQ0DtdCwmccmBFafAFQwpa1GGFmqHtYqfTuoMVY6F2PnLhYCbpkIMK\\\\\\\/3JyKT0e7i0lzZVUc2cH5ccSPWv6\\\\\\\/KG4U9DgN4uHknG6gMHpEZZLiq7NZV8wngE1bAGn1QLD9qmnckyyLFtKV0eYy77VkT1uLyHDuv5a8fMcUl29lJUYcGy5SxupLyxwRNmHI259g+If03\\\\\\\/8HpTDYhDUWHGL+JfVIphBRdw5cLX\\\\\\\/U3XGkqG1rNAkmtRNE1TX1ASj7hoJQDcQA4wRTBUPepqXGsv04tI5D0fLqTrdP5S+kHf\\\\\\\/B\\\\\\\/4R1v8tbHsEWakTjPTyyjju03H6VLP0ttQGNK4No0b2BUvihQ1YhO8gniAJwXLYR6+j6UMstBy3uGv3Uvva2Er5w7IAdbof9IA0yhZngKvo5V3F8y5S4G3pKX0hRVg1tNWyl5Xtoui3q14NRIJrYmi2S9xpHb\\\\\\\/BsrX1v75\\\\\\\/92WYto\\\\\\\/FSZnhzQctfdOY7jalYYYrY6Q3n0NTmEQGpkKN9KZmCA2fdA4Ak2MFsfZPapC+dFsojOGRyiDmZECt1I9X2kmrwUemUsUWivBnFzXmug931PIxPxQiLfqDB1lklILkJO24zYhOx0rtbTx4RXdaU7NcxFRowihAD1lZ6FLI6ukRTDnzsRnLzQ3n9ZW243Tok\\\\\\\/ZX0WUY+eNCalTzNwe5Y66pldz3iwXtY94kZWA85XbQ7SjB8u7pbugJ9LjYWIGY\\\\\\\/vh2DX\\\\\\\/XIopvvhgb2VxGcNRXT4gCnvQjfcA8r72X640ZuGh17BKwHq\\\\\\\/FOmul84LiZfxYMOcRfZqB1D+\\\\\\\/etVdmCxciY\\\\\\\/0pKFspppMX6LtPU3aYr85Nuaww\\\\\\\/h7qGVUC9GYnV84xa8exjbOTEuM5DL0mrk0yP4NNNLLxgZTroUBxvzteUosvcpKwYMwnFtLoY0+pM+Y\\\\\\\/SSTIxYTvqzyWqR\\\\\\\/X8Kwa0o4+lnIuTfXUqLvDvLO1G1HvjkqVx48MPq6ANJ8mv6oTmEYBoYtiIOSKmj7BQgspoaN2UHfjhRzN4LGnuSQKSOchX\\\\\\\/mZ4kMRcvSbShRaxbL4yIBV7vbIGo+GxVvpB33M4f9xOTARxRmgpiB1mcV7NntvMATLasy1VmC3N9pq\\\\\\\/RUwCBWHBesdSUq0oPRdiDhg+AExDxAG87iHfFjK5\\\\\\\/sTCuBL31zGHeYLZ9ZG6wKAWTTr2SvsPrD8lq7I\\\\\\\/c4PPQ5a\\\\\\\/GcQpszA8EB3Ro9PhaPeBiH0g5nOwcZ10Vs0vqPLgadsKphjxpH62vR8uupO+cq1kg3DpfpaPC0o7SeXgjbnZTL4b\\\\\\\/LbqrgfCFqS62SuE0LUv6plS6bsyOvv4Tg0VgiLMmh+sbAdBOlsj6g6n+KVbCgR1yGUrJ5BY9gPuNbEpkBx2omAHIkNHioolAH4Z\\\\\\\/iSQPn7ChoTmSlaJOvJcUKFa6zO1LNxkL7eUVrunJX+46iSx4l7x6jT0+2948TJRLshQ74G6bHARWIPliIXYFkjxER4zAEEKcyeUHIh59GSIUJuAkkFM1przwBxf\\\\\\\/a+BtoqOoeoeHLWRDYDNjQrI9k4gA2isrwXAMbspUfEr18IrLVjVALy4\\\\\\\/bgaiZ6t3GTZCzCJlIT++Ok9aCUKFknQyxZIi+Z9FjykjQD5ZdcdER8mye7P6NkXuLibY7iJY2cMZDp96Tf3rSy3FcEUiR9c5qbOMBOda0Z0jy5JMlHI6HM\\\\\\\/RxYvOIIswHPsb7uTRC1+GONJ8PTWl07yKHgvWSlA0eon84Vgm7eiwcCuQUsSufndI8AANAq72UnmG4ZjbGbTnPTsIAOeLJvBkGzHev50iQxzW\\\\\\\/mYNhWJIhSlhsYauBAPQwwbsKQfL3Gyex0jlSVluyQvDhbIH3QOzf7O1V0hNX1RsnLGFA2Am52M1sz0OoJ3KvjPafv\\\\\\\/QXjoDgEKtHMoYodggBzk4iqs0bLDFxy3MK\\\\\\\/hgf3nMkczEv2vcHVZGn7C9k4PCWmYlPtzWgOI1WeH2BWY2E8zokG4XToNdnoymo0iAZv9I7chdFd66b+ticnaoxiyl14\\\\\\\/btCDvoQk18UwHtw6hwntN3387QYAPU8flv7JMqsdcHamnBICja\\\\\\\/SCbS1\\\\\\\/dEtnofXtBYLmrExbxNKAeqr7qM1VLr1nHLXxzMbpNEHzEmEztYP95tgfg8XOVzVDlwzrk2m\\\\\\\/SdIzoyrt7jYXBYdW7ffbR9jB0R7MgeQnenzXdiQenQRp386r45utYDxw2B2aSYe02GOhpzSbdkQJwp4CJmBF15jeFyU447LZlYM\\\\\\\/GvSIIu+cxzmT9wM+f3wZMSG6UCHVgHejkh60CZE8mG2DzzwjtHXdVu3DNjDCmbKLhHCOmEGndIny0klBvk0huYPgO29mEO8Q0fWkAXbS\\\\\\\/Zz5harsHwTzS40\\\\\\\/CJ9wCUKE0ldSBM+wJ8Wh9IgnRI\\\\\\\/iijf+cpuFUtPHbIquiH8Zhby1hmf\\\\\\\/IC1bZq6fM8fl+rwkbEQs5ezOE5VhLEvyv6T2GIvAnJi0Lf75gsT2GgwXQM9RN6ekNHusZRcPiXmBncMa5qhNO0S2oXEp6xTSlqNw1anhVBMJWsz+wMYrBiwFk0F\\\\\\\/RSKGorZ\\\\\\\/gA6uhcC33yU28guxylkrDPrUzteIk\\\\\\\/f1vPYPXnwCfTMtznuQuTurVrQ9RDaXBk+MhfijDwvpIxzXJpVHYj1+qT7nb13WziU3kbs+PpQ9hnnpOueneS+Z1AQkWZ+QX6F\\\\\\\/YibtkUDzdjHstJVZoZlo+nydej1I0RHy4JNhnqBU\\\\\\\/bDNo+6q8xIBLMteI6fT4K73cZRG0WqPsoxTt96SsuJHFzf7h7R\\\\\\\/RnNND58PTi68H8zEk7HCikzJlZR5UHPqBX\\\\\\\/eVbrb3JOigPhZ\\\\\\\/gQRegtlDrUHRgaoqDAMD4MFGn1j54JoD34bvlEyvSXBJk1RpbkZ2q4r6C5w+b+9UDCNFdEKUtkqSxAvMKbRwHumatNgd25OUu0HobVJykikblVP75Gj3TW5CsQENNFtQey\\\\\\\/0cbsh7VprZuVS4dXAU25IjcctWQ+o9ejdeVRLZcqnlah9gYl0wsQdwUsHW3YJ0KKg2GCiS5DHR4nH9zR9+rXkjL5FrdOafwqyCRtOIy32X0uBBWK5VR3VxxS4bPtxN+\\\\\\\/BguCfUPYhSsH+JbNm8t9B6A1TmWG5JMjLMHPe8hRxDyGT+a2pAXbf0CAk5NDUUEU7cFw9Xxa13zBn6Qv31uvPVlwHVj663sbClpw2HH2rpEjsszykXqZR5L1ZIHqrH7qS2VLrOkBvowrN7jJ3BwAVXhIx2RqcdbxENHu6F2A7aRSYFCdkaRWvb6CwdAnPEpTzuj84DBjH9aSS9y3aKmyzv53F6w3j6ZUiQKEm\\\\\\\/yvptU4tlNeCsAcGjAuGtyHOcqclkloH2omLo4LfXVOeMf2sPnQsju+tpioeZPlAAbT8aFMULFNUJ0KSUTuyzDSSV8\\\\\\\/L8803QYjqyqp7sKlQk6k3dVF\\\\\\\/YQUeHOwIq4VhpEF319D5dR3D5CVLxtUTWAgKS7HSHzhXzf3vq3MkWz5Ye7Y\\\\\\\/AxeZtOBm\\\\\\\/9RExzfMwXvC7pokhGI63WIPe\\\\\\\/Fc7Jl+wGjhul1CX3VX1u6Ec\\\\\\\/8m+jDiD5X6\\\\\\\/uDJc2l6H4aYHmb0DhxsELF\\\\\\\/Ew04sa4PaMvzaTnjiWKo5r+kxGxVDi4wgdZoeh\\\\\\\/1dqftbkO8+d0D\\\\\\\/AhcYxuirSswW\\\\\\\/N4ZmxIme1hjmlk5RfpQhaQ4MB4tv8bjLLnXqOX1uiuChmK0ED+P4Mchvxk2i7tz6mT2CrLE3JNFK38GgjCHiSrdksqc2fPo7y5U7mQDPHHmGbQ3z2h++4gdqE6MYawRxbZvzV\\\\\\\/h3qfAc6RwJ6Z4ziof+rIy1o51heX0p7t8kNpQjdRUI9cWIxptEMWUiJp\\\\\\\/EhI+Xi7SNMzSnJpVCmFJd6PeaKOY26At82jiqGHjALZadt31hvFZT\\\\\\\/7xOAcJcHGOSP+jUZz43D6gb3AFlptvhc4OKYV8nFw+wxZUS71n+WlhiwsXTcoJbeNYg3VuxOZ1SVgck1CPPtYiA+mAAPAWbVO775nGUnWrRGpVZZPUUE+dEDzLYFvf2aiVUSfcPRld8WqjbFtEEkM\\\\\\\/7feNjyqr+T0DZ7xP1TRCDN5lqVIlfQ6dyFlZ5d032JuYF90V0DUc1+xz5mVvIOoDTiJL2E8T\\\\\\\/\\\\\\\/HZKKEW5dbFP2PEluDbXiVmCHZnognIabcKgjP78lsg0qNLngeyeRSmUMYL9xshhTajkO4ekdqsybC2wsRLG9BQui3r81FG4ecZwDt8xbeAOv6tdS0\\\\\\\/I273kRkzZC9uZaMyX2TjYEsb9DogInb4DI4WgkrcsHtZWg\\\\\\\/PkWM1a7BWgPivWG\\\\\\\/+IZFyU4fNLfZnWJy2+6klMEBZLX8VylxHnt3L4lExkggqV+7qhXrSLu76JH1mdh5il28ypottGp6wgrzjyh1U9jJ8eZ+rzMzhzqcVxmukLB1fzlE3K2l3yiOOiLDFPTUQ6C4asqoA+X7uq9hJnR84+ccWJs8oJMM1zu3vnrF1jkFTwtZSJKPr6sxXVFZzJ6jUTGNaYUAiOg38zZleRvKIXIwwuNDo\\\\\\\/qiZ8Y4TKrUqebgNt8w2Sg1FGnFXGsmDaa\\\\\\\/itFF9LUa4c8RrEuFYB22McxPrS1zV00Ez9+IAW\\\\\\\/smZZ456UujfxLxp053\\\\\\\/ju1pJN6fTi75vv5Rx9ZEmt3sD6eGGcRcTaH9C0JBxDBgEZ+8Jj4HRSYGSFC8dAOpAQgYUD57tsXTDh6TuiFykANstKu2IodwTedBfLvmnLMW4NNViMVmnyLrMbsSlYUNow5dIQ31PwOIzl5fyMaz\\\\\\\/Dexu5OhXAHhbapZ3gP0KmUV47Jl7iJxwauZaVp+Hp\\\\\\\/5LcOKN36ggaXzzOE\\\\\\\/o93mNclkZrKB2CgkJ0kIHBY9IdUDJWb\\\\\\\/n2372EqIMtAzvlw2DjKDfUqRsFl3Z3AGzsyIHpx4X6EYnFOz+v1yQhuXRlXejAAK81eiFT+Ltn97oVK\\\\\\\/WIv9\\\\\\\/uCPPiigRjOCcT\\\\\\\/caB3wddcuAgXkeZEaYwOVjd0mwq9REtpRzPFnQphttgsIS1DmOKjKQCF9ODUk5tGvpOdhz5g5IB+yU8Z2C4IQsrbgen7atIu45T3UX0mGSa3T5Vj+t6UqBh1NzS0Ii0uWEOXbCuJ1U0XLijWMExp6+yeuEFJQaK9Uf02\\\\\\\/LsKdFY+2pY6mhu9EQ9Ct9xeyKcTb4Dd6SxTevdgE\\\\\\\/dKKU5QS1HTctRLhukfKbh\\\\\\\/+kRac6RNnYoZype5Kf9lU0j1ip3SQJItlD0xxEiQ4Yb+zhGU8Cl68HaVTTeOpRWzvGP6fU6CMutdDR1zH+aLgkep4GT0UULcXFeiSQO2\\\\\\\/r\\\\\\\/BeeK5tiNFldTZlNmbe1dC3IuHB\\\\\\\/dCGga0Oi089OrUDkqHbhi5yKUJX5IvDIbvh3cwAhruk5tXGgVuiLapgHl3+kUqa+dacAtmFewilKKtFomy12EsqdqwjJpleJSjLiznYMgThEYFcwWd6CmGFNwFFhQ8J0HVk9V384BY4a8aqEcSD+WORBvYscjHMMkfaJ+rSvXMVvXdecacvqyPJ+RLbHAK0hni5pt0x4UnJBtMx\\\\\\\/cldRiOR65UybzptEdkQzBL1cfiKDEYDPPYz3Cf7ymoUQndn1X7sFIvP6nsX2YYkSbeM44A\\\\\\\/DpWzj3bRLbSyvvlQT1zZi7uQOtPw3sGz4bjC7PectFTPs+IYyzMdDIp\\\\\\\/VyQKyU1SllSWaiXfUVvbtmjRpdl5P2LB8vh5aq1fOvfWSFB0mYfGsLa0obPUAxG9N5lwVcdYEqXufauIUhuAzcGqTBwsFFzjQQqhwbG8Hm+R0pKfxNyHrc8b7HxPnsuGyWfnT3NniBHJ\\\\\\\/NzdS6WN+H94uXGyxGwBAxAKtOqGfCoDLBKOast+s5EDDw4s1NXTVw7SF3VU3w8qd85zWVOFD\\\\\\\/Vu\\\\\\\/HGQh8zk6Obma0zt5VwwSYjd1zuc1rm1cAZdub3pGNSA7041bYUHFBT0\\\\\\\/L\\\\\\\/MZpr9KqVXaAQvxJCQ0sM4gb+f3W3Q0VHXZ3FAqST\\\\\\\/Yz423EVvQRRxgsjHRTa+4k5Uu359DAH8Ax4XYzlE\\\\\\\/6dFxTuv5+k+eT4qHFyMbZYHOlNe9yKKTH\\\\\\\/Q0txFI31npe0g5iPOV8kpiujm6oENljwaEI3\\\\\\\/794Wsq7Z9w8w9W3pUrRb8fHvowGwBcDMWVgysalOm1DnNdOlBw1+oW6\\\\\\\/AAO5FU4pCKS0PRWXU2wCAK1uzt9JV8VvaD7PCLpcpYcsU15WlANgthsuhWdbuHstilRpj+OfKyjYYv12ZohNMO4kksD01sVGCoPSdF03\\\\\\\/aimQwRxIBpzFXDOo\\\\\\\/07Y2kSKbJEpmn\\\\\\\/m2xEAS0oY2bre10ISg4b2sBExxwFZw\\\\\\\/+4c65iBb7pJN1Ka8oDnOh6gXGFlA\\\\\\\/aly+pRdEKoY0bYTatQFiNRm3VlMdGesxjXd3FBSmysvM3d3WEz6f4ghXHMUYlpi63o843VPRRHsmGX\\\\\\\/+4hEIGX2VPmlfFSsqE\\\\\\\/HNKj\\\\\\\/aVQPrG94aXj41C7AfySv2WANuSr+Wx6Ee\\\\\\\/owOaoIs5mVUICU8Tm0IdkUe8EDz3eV+NvP4Zjvhxes7Sk4o2YdbPcT\\\\\\\/jynJZGc5dm8p0IsCmxdiBa86NqfGCapZpS0QbCMMSVEUfV\\\\\\\/AXpNSBdWUZEM0kQjkfxBwMPIZMqmi8NNjScfRRZaBbfIydVi1Lrlafl6C380tY59u8EVmwwAhxq9kKyCFnEGjoKVKFXZgfjVXOrBxE0L\\\\\\\/tacg1E\\\\\\\/APItSOr6TQOfdvOy5ZiV758\\\\\\\/gh6JY5PmLDkoTk5ZIdLdrCSHfhfHMtLg970kUIdNI38RfpNOaYjrF9YVIxxLj0Ic89iS\\\\\\\/zydCSEAuxtVwSaNk59TkyooXpltU4RSrc0CYZLm9ZaVvktpQZWSw6BEKB6TeGMtc4wxlM55GoLRaG9D4a6Gnoc0cZ4Qgzm6OkIZZOPlGtOekx2M1ehcbJB4D4up16O8cEUqo6ArMWRJPnZhQFtNQejRmCCb9ZPJTmv99yjsjrcigTmJksBhN6x44hLtwWTjqGlVI+pwz7NTEmYKRZqfvdPOwfzaOT5TyCDdSalxfYm5T+bLHub4zTmXFHnEt\\\\\\\/kT61erhZkwRTtpitt+9a02YWzXmU8tKNfw1Wiq65jCYHu7geJ8o17zpR4A6Ojqb0PrmtnqA7t2cJZ6LkFTgLs7A32MqUb9fNPqqvllr\\\\\\\/9w6HnWlEzfdMyE0cTwBN0lTxeXz9+V7ue1PtKbRx5lpYDb14p5a4mcmP2AUvdXDrfuz8xdVMjwJ7rr7tbUIbSRLodKeJ\\\\\\\/lFqRhUtk1HJ\\\\\\\/pxm\\\\\\\/sjYb3fx4rNE6EbUObL2R96b3XqhqCUkGUhz8Yfhd1T9DMd7TQwLbukbmHaaboNpgxr6KWoWD625zFSmCGLPWyDf+xoFzOEd5BPODO8Ugv1lnBQZsGpJyZ9OoIgJ337UaSQQD2xSXFAR8dvtWPo70FzZ7JSKMkK7b1mDrrczhabVDk0gVRhXIcKomIQDsjQ65ENOaDB0vUX3u4EdU\\\\\\\/qRcK5NJU2L05BOFF0swqhfD4Vnjyjo1DDhFjkronqbP2\\\\\\\/AaVOiBx9q2veDk7QZbkWaWmh4Bzm6sFLBASARYcY23MLJ1P+vyrdQzQVOLpl9SmgAjKj9Qs5X3Ue7SMyBk3lfmPBpWO1T9UEL2NFjJBBmlFQZGlQDQLELDgI1D4b2CG4wrFD9MF4Pl5ih6A82ebBp1\\\\\\\/uKFggvBDw9DPzSQYENYusYdq3vK9sphyaZ2pSrAv\\\\\\\/ccPmUiuSQn8OiCVzWXUTBZ4VcylBu28FPrV\\\\\\\/OXXcryHFRzC1O5A5alAcyYd699HlH6lvu77s3E2Fx\\\\\\\/JPXVRkobW1\\\\\\\/hEAQoM8bjiGmNVR+RRiPo1L33NmcD0V3HtB4AinAaF2nnDSTMBXsm\\\\\\\/GNBQVSugzb0D9euGNPZqovnMRKLAMJZCw6CMxKKe+xM8FaI1dNbwAKKXO1FShgYKf4eV7ItYolciBt2DKnnUWnEIGOewDw5ngIYSN\\\\\\\/zjn0BogfaXPIYjAzv\\\\\\\/QHVjHN0quqQWazHXqUb1S14KJWQ49IHdLk1EakATaSnYt+f\\\\\\\/1CbSxniulJpVXAwpHyuKJImkPOHLbCeVrhF+jSCxa\\\\\\\/X1QBo4Ds0GMegMOrg1+17vY85lHeTeKDS1TSw6OeN6tNJvWzdYy72rHULcuQq0Y97q\\\\\\\/W2ZHjqgKxgpRCmdywtACp81PrMrFIAiZRE6ItsBecxFvys3TdFZkB\\\\\\\/HPhlwDA8\\\\\\\/Rfd5utp\\\\\\\/S9k7EFsY3SBJ+FJ4bJkJV6eeXRx9Rm3HYr7kqIFNMia3rC72DEvi11HVdWkzH1iMtm3mBxoO2MY28CPJXNnzIJ3UrUzWJy4pluuV3O8j\\\\\\\/hntt7xRw9OuVbj+0rT5R1KD4Y4Lc3U8KZzQiCY3Th3zlvMPdg4ArysZNk9Md2lp8bF2PcxSpCBM1+TQJzXy+ZCh2R1tzFhgYiQ01kWjfDaINbKljOvy0j62tU2rlJjDOjMQhJomg7qq1DmOjpivsWJGiGk9qWPOWAiCmlUGjl2XFZTDwMY6lIDGfbNvZ2f85Jo2X0L5KDQHsanUcItG10tVjyVCYLYYcg5yjR04x3b5Lx3sgbPxzCyH1tIDxOz6nbckeMfvGKo\\\\\\\/01pYn\\\\\\\/vx8w5axg8DVhiKn+8YHn6sxKcYWPnN7kLsplBFMpzZCu3ICOV2mqvUX1\\\\\\\/MNQI51jZY5waUsLjO79GgIVbJ4tFugOlhhQCMXb7ZgsdGjTJ0aOJAmvXRYWs\\\\\\\/w2+5UswiYcZeSahnznyZGTIs3kfAMZKdQEGsWUThchfExkuTlyZQJ7WieUV97L0iY61xwjHyJ+NB2rXws7zfZtx07U9d454qbY4f8qxKwx\\\\\\\/+eVmydPD\\\\\\\/lX07BJ8NOiTM9Dd\\\\\\\/HfSeMLq9MpDv9kD6we40Z8jMwV8oC25\\\\\\\/aS+IXYu1oVzplO\\\\\\\/bmwEDJFPVsISJ0P+vInLmNvsJodeZ2pB7xvAnWmQaLiC73zk6Ia2XrZvJ25L9t\\\\\\\/rhpYpivRRxtR8w5lana2D6RYw2EdV8F1xvMCk2iep+VksP65UKsz7Pa1S7MJtsnzyqlZea\\\\\\\/AK7Xij4JgUujtVbxDoNBX2t4GdN\\\\\\\/Cc1IsiZzhoy8B6fXXhgMi0EPD+6AeZuMF7WtpJO9HdsE9wIKuM9G1zQriImw3ODi1hSS9T1ZfFvawrtW9yV6RisY2Ccz8OxpxzfLTMxtE+Kkesuert098dPfdftq+J6rfswYBOzw3QsvF5NJ6Kxjezr\\\\\\\/IUgRFm4JogBgnZnHri9tP\\\\\\\/m4HMmXOd3KtZMSq\\\\\\\/\\\\\\\/l8YrEDURsQNorgpKz9LoIR9r0DCGSCVMULmROaoyvE6D39CsLk0hokNoRToTKPQ8vZiZlvygEu\\\\\\\/nOIiyGeHikHET0arP5mVB4p30Vn+B13MruRp2fwGm4NGZy1CHzWEMt9Wro8s8MYh9vJl21XZhPm3L6BjdxOCyq0cuQEqZNcXP+q\\\\\\\/PxeWK5ciiJAX39Mz8AXauIY6OoxXasLCr3QXKJvP2tEXzGawm\\\\\\\/UOjyUBQRib35hshea3d5Gge7X8RzZefzyHVeHkavqPPe4huDGwsaUP1Wx+Slndny9IHjHi2Z2yg5bw+ysA7hI25N0wgly1mmmJyrUHrp50oJRyisqOiK9CoFqUhVly+0cQbZ7qqcwANWwgWgmwCC8urAlhCJ51N5qmXlC4IPlg7lpE3\\\\\\\/\\\\\\\/vKwpEV8rHF3mwCkEN4BgHDKjxVhyGuPSYWYUfSoisBMQt5BTHXVYTF4grQuQJKqMdNZ1FiajuQQQDL9gJfrzZtAgAkgDHfwcrm\\\\\\\/0PwoxxzMYGOEr\\\\\\\/7hivL9pLo799zHpJ8FH1U0sQxm3wJ3lDUjrHNEc+6lfq2sFHCkFPXVCgw84V6IeGinuAcA85w\\\\\\\/lPjpypjBxx4d\\\\\\\/TscqtPlH1dB2gFjUW6\\\\\\\/sUsG37rg+0xLxZDQSGCmyPU\\\\\\\/MEH3ED0bMPVJ97ToAml3gRrQhjKmhbp0UMwwWboE9lDF1apVSINkUTUaD4Vnj3bx9D8KbSrLmv6rfx2k05nsw3MVP1t8vBLFAG5L\\\\\\\/dXKg2JU4iGWht0WO5Ljxt2KvNu2O9rFcJrMdo3jzBvqhXG3lU\\\\\\\/HS5mj2Ra5dDV9NA4ZTi5r642nzmGdL0IwzwL\\\\\\\/Z1izVRekSXE1YVcEM97RH+iREc84Gbl45LoJ\\\\\\\/6OnWsUCjPT5NkPgh9FumOsUI4rG6m0CJkBks7RSl9nMz9iV2byvYowtTu6CPqB8VcXA2HjZN3Ympb5OJk3hRlkb\\\\\\\/GahFy\\\\\\\/rwy5PQiRlP9QiAeSlbXeMEgljNzltBQ63msO\\\\\\\/2Y8XzwxYuD2ADChRbjwwzSnpIMYjg47loiNxfDo4z\\\\\\\/OeoWSTlLucGZkOG1jtuXB+c35U2y0S4uiBzRoWvOueoBbkT64pN1vnWtcukrDrCLyPKfiwqGvIxXXEZDg2Dhf\\\\\\\/LO52qHZ+LGjr+t19shUYemYYEbrTxTTY9hlAqJBTPRgG6VgzUbKPPBbZPPZZG0q8ac+iX\\\\\\\/ZzmraaLFtNVzlStKBWey4cMLoSIS1w4Lie6nV3BbNypF4hOJOfp3XdNYb\\\\\\\/SMl6kmAAbjidmOsOtwjmx\\\\\\\/nKC7SqCRTq39O+mgX+AWl6x38agmRBbM42CZygJGOdHhbAbMw\\\\\\\/9oJlzpnfHKI4LnyhFxJzJm2qrcAuHmZULRGRrRvSOEIkcSPe5QwtYfBxICyi2hXqjMzW2yvOMJpsGK2D9Hnlh7mGjskM3rwWMhf+memxoIr4TYltOsTawKWwMoUE4eX4Q5ewQYcMlOmSFpX8yN\\\\\\\/noW1OUYJWYvWQR4tw02Gx42yhxXIE9NX8sGOAJHw+sszmTwI+9dF1wYsBhFwpCo+Z9pMX9QalhcZ\\\\\\\/FlFeKL6h2vVzVUhyIqNxv\\\\\\\/KWynH9eQavltD7bCpRbqM3BI23C3ZX63dGDMMuHME5q7yFhseFdo4rGTfSeacYF7JiMpe6NSolICWxx9cdEciRDpIcKLvnHLVxnDuItLCXYBOVYq3lTc8icZZJvMY5Hv698GXoeTDiYhPvlGUIaTIkI\\\\\\\/ubFlnt0pfLLZ51bWBj9bNM6s\\\\\\\/Y5Uh54s\\\\\\\/viXFbapxvdt3N\\\\\\\/Xnp1fjXl9cbG7bzoFbwVcL2W3YbcV2lcBEAI\\\\\\\/OMdKMz5WUrc3K2WJPpavfJmCRi5VP6vvGNVSuWOt74A+B8PPjXwzvR\\\\\\\/6OXliA2G1KnSAHFUvGed9VqLOa2pCrM4bBmXkk5QS8d3cZR+An9\\\\\\\/aKK6eku2I8TBaTg715wylUIlNjDWDwdCCmcHzikKl0eNK5+GVWFMTSK2vkH3KDHHeZz7vFGBUVW1MYvEZEypSlxMC0kAOsVSaoFuUWmNzzp\\\\\\\/y6ofJFlXjv8k9zu4343212MN1tzpushxUKFmAZpqhwDkYVXlXaWnV0sHGgR2t2fGYj6KhywEK0T1+mSfUg+OaqLVCJxt+xExWh9fWiYO6ftdtNGIA8wBvt89bc7qTOw0D6Q3ahuasXDNzNBYGvELKxP5ZkL7+bHY4b4bRqrD9HWnIfo00AJrMRz6AFnzSGA+1yh3tZBgV2dbcg8sm+31q0\\\\\\\/o88qUsjqPKhakkGcLT8w3HVtzdbqkfNQNZrE1xKmOlY\\\\\\\/vh6aa3I4fxBhXsHasuYjXXTrO\\\\\\\/juofZHS+rhJXYjR39x3oaVCFqfz5rPOApRBFqez2l8Y0hDXkRadoUhbvUzDxadC+C0ApXE8lIiELOaLBmaYYd0fmXUauM+oR8N+IOJ1DUjZ3T1IsVgqj91u2MSTGnkWlAkBxZgo22cLVSizZeKNR+S1j6PnyTNZHV0KY1aEWNgkjTu9vC79U4ekwhgILrfZ4OJm92C7bEu2qsodTsKWvUyWjzo5ooQSXhoSyOPiPpsKwL8TJ80oc4pwtZ\\\\\\\/GqiVFgRdJuWvOz878VbKaNx8xldRefY1+ee6cNOfPhPzMnYLfNTas\\\\\\\/Cx7jxdfcBSvzMB7CmbRHi9+ofN6cJmzrZm9IZsbTR6zmBfyRPrwVm3qMcoSNeRpm+E15E8FbIUrEhOUGnWjglZ1C1qbSzpMQL\\\\\\\/jl9uYuGSfHRyLjx\\\\\\\/kVELmOhcyVcRh02xUFnn+I4PPHwP1Rzlxfn+7Iq2cETIVFw4SZcWlRKIpof\\\\\\\/kGoaWvfgqGLf1XPYYIRpzmUqEkaaHQnkeNZvyy2xWpKUNoC70WRUk+thb1fLsDODegpCvrVsbXpu5RcRO0L\\\\\\\/dQBYb73yNOmrrxYPscHpBQ+JUkMXxa5oVMQwDbI2qIBm0MAZQdGvlViEXDkt2g+ZLzoiHy9AO8G4H01q366ojAp8EuOIKrvVJ\\\\\\\/0O+Re1FmreP9+YiY\\\\\\\/lnZmqjlQTF4Jsjtx71H+yTWAmOtX\\\\\\\/kmyVT28ChhPNVVUL0QUO6i5XlPvM4FODRczW9NnJXc5LntgRvvM0bWhzEDyNgv5p1qRcaPBbAPp8Fdn2ka094Ei9LULVcT+E4vFw+A6y79ojqSn+fP1rPDzBEo8zVvKkCd+q2A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YK2PbvaXJmHVf9\\\\\\\/CJ9Gz6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220b490db77f574d6c8f29b1006de98548f653110993a0a59525e0462f13e533fe\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MZIDbX5jJzYEvnkehM9nsT5+u0CbniO+TmiKr2+dsJvhZX7WkrR+q+zYHbYCpET6kdV88pqYkio+JW092BnStPZorB0LnzEdb8vQYzLrZJ7OJEKLgdsAArgsedP2uzsILRwwZOJYb+rHBc9NrcbgHAe1Pb9k1zGJ4v1fj1xrOdSKsSECuUbCYhw4TtPFYSYfhLaatrqzojDvhqxNnnRq1I\\\\\\\/El+iZUREmfogcHcPdUzGWOu6QnsLdk+gFWNCapSYfz1Hi90typAH\\\\\\\/k1SMSO7fhkfAM56Fjf98ZA6R+t\\\\\\\/R6DVa2w\\\\\\\/65gxwAbWs8dB3HiDmUVZ1buPR80q4TRWwVVbE60XPsQI6E86H+hmhxDZSzlaGeA66Zt7y9NYeIcNNMwzux9s7hCt2\\\\\\\/m5Tq\\\\\\\/HcjsNDoWV3CcAyP6wy6VN+DnN76Cr7k\\\\\\\/zYQLfugQQI9P+T2ZytPl\\\\\\\/p\\\\\\\/Z5wuAHPuYFJf2QXAwVBo6UMvxB1WwZVKZ\\\\\\\/W54Ag0S33wtqpsHvRS6DPy\\\\\\\/iQb6D2lx9sHNvG3mgjazfz3a5UYmJ968w1hPdnlpV0RR5i5AtVzpCdQbmzEtBcPa4RAtiqFJX4ZqzRBpPXSPt9Y3uS097n2AIo23AFm6VIwaED6CJssKCZZt0508zeq3ZasPkBncAIdHiBKaTDnX\\\\\\\/IouIDMKWIYByc6G1omrTeE+0cpFDPlPWp8u0RFovjAdJLprOaCVokMbUFBwVLp6YLHytQo5n\\\\\\\/MlvYZDH94Sku9umbdgHCfgvnsuKjXWTtvufy8QJcHEaH\\\\\\\/ug0+bvAiqJrsLoqJyBL\\\\\\\/aulE4AkgAW\\\\\\\/PzHayPVBg9A0lOUBAwllSLe7Q4yhsaL+fYRxcofGb1lWiPgNlnFojIFIXwJP+F7Ep3PInYAjP9TRBTgRk0SK5X\\\\\\\/VqfB\\\\\\\/qG2C69L16rZZggHK9Gjr1iDzFo9fOvk0Dcu8AnaugETIVVcyIpbUgJzyB6hsZZIsMGmaWBYvztmbnFCBU19KoEyS8NO+kFpsBM85xYhZHKskwn9wWUST1rErTSPx9Ps5MQ0dIlLw\\\\\\\/Fr8xbO5Viz+tD0lrrxFt4gpKoBRgYVXcfzp1DvgN0byqSgEXmSOQl7No\\\\\\\/IM14fhpoCCk3upmQqixV3VqNKWman2Iku6+xypWX5A5kHhbg0UFattVV0Wr8\\\\\\\/Rgm15K6aVR\\\\\\\/2I+EHZhwyTdzPTvU9df4M2cmv2XwHxICWG+9LxNFqtulqXY2Y9Old1LYdQP6h\\\\\\\/iTH7Incsjdb3DDepPBHlpRkqw1xAAxmEY5nvE6Bv7QHeUCruEosrCXguvkk43UiXGcuxIRFNCywh4\\\\\\\/1VGHlO0J2Y56Ofl6pLj7p1ix8JsD2ZZaDZ4ztvhewaf00hsF3bxALuuNyWu3QgwXwMN9BG9m6bCTtUa4Nz5OoSaa9NZP2GBV6XtW\\\\\\\/2jWM6bSP7WQ\\\\\\\/+ReWHWUSQzW203pV13MqrF9mH6vyCPLXmQQGkCuMz32tRLAAAejnjXjyNBDzpEsCejmOezXo3MrGjMGM1P6mxqVNzPx\\\\\\\/sCcmRG0vWdMQIN7UleoZzs3iJ8KazqgNIOoj1TJCRcsSTQKTKlBMNd3Fz5GfqZukjckdNr8gdIqTr4U4Tg9vBgf7CLQvPIsR9KxgI6+ntrMhPfRpglG8nLRd1xWRPDsAKNfsdB1VFgROD0fd8pKkaAXiEs+S\\\\\\\/pYfv+fLX6AJwqYCWq4rjGDrbGXzuCylMySAAuXiPe0YX6ipa0LNDQj8eA9aVa5v+0t9vxghgJNhm6EjzkS6cejj7ErbV+jt8xNfHfZIK8PwweGXBRPbGJWNMz41InIE2xiBh8OHEuwh9AKclPCAwy4NdLVetrRAm9opNAOACAAgU\\\\\\\/m31vYVg5O2CQmz1H0WPez0P0YWY1qTAmsyuRPb6hSkxC5KOo2a8jm7N7FTdUza0kAnkEGscYtPD7STGc64YiHVlKqm+BqOP+t9UN+29lo\\\\\\\/KjY6y+iwd5tvSu41WArksqUDIYZrA0nuv8wBEp5dAHZKxd7qj4D+mVaYVPA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UPxR+Ay7sX4EheCAjlfNLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002205892af77745938ed0a8889443561786bd90f42701ce93a7ba36ca5de30f2f63\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490460 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2603" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Btdzc8JaT1mZA7Ay16twvMikBL-BGC7ssni5tXcmjq4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2603" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YWt1BRDpnKNuSq7LOvOFDtxHoIaTjC3O4WkyajfYjIo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2595" title="Involvement of YAP-1, the Homolog of Yes-Associated Protein, in the Wnt-Mediated Neuronal Polarization in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2617" title="The Doubletime Homolog KIN-20 Mainly Regulates let-7 Independently of Its Effects on the Period Homolog LIN-42 in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490460" id="node490460" data-pisa="ggg;8/8/2603" data-pisa-master="ggg;g3.118.200046" data-apath="/ggg/8/8/2603.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dual sgRNA Approach for Functional Genomics in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0221-9052" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laurens Pauwels</span>, <span class="highwire-citation-author" data-delta="1">Rebecca De Clercq</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7940-2699" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonas Goossens</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5820-7109" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sabrina Iñigo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1292-6313" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clara Williams</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1682-7275" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mily Ron</span>, <span class="highwire-citation-author" data-delta="6">Anne Britt</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1599-551X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alain Goossens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2603-2615;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200046 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490460" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laurens Pauwels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurens%2BPauwels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pauwels%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurens%2BPauwels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0221-9052" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laurens Pauwels</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lapau@psb.ugent.be" class="" data-icon-position="" data-hide-link-title="0">lapau@psb.ugent.be</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rebecca De Clercq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BDe%2BClercq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Clercq%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BDe%2BClercq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonas Goossens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonas%2BGoossens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goossens%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonas%2BGoossens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7940-2699" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonas Goossens</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sabrina Iñigo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sabrina%2BI%C3%B1igo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=I%C3%B1igo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASabrina%2BI%25C3%25B1igo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5820-7109" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sabrina Iñigo</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clara Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clara%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClara%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1292-6313" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clara Williams</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mily Ron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UC Davis, Department of Plant Biology, Davis, CA 95616, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mily%2BRon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMily%2BRon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1682-7275" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mily Ron</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anne Britt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UC Davis, Department of Plant Biology, Davis, CA 95616, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BBritt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Britt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BBritt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Alain Goossens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BGoossens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goossens%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BGoossens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1599-551X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alain Goossens</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2603" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490460" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2603.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490460" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2603.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490460" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2603.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490460" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2603.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490460" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490460/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-75019 first last"><a href="/content/9/11/3907" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - November 01, 2019</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Reverse genetics uses loss-of-function alleles to interrogate gene function. The advent of CRISPR/Cas9-based gene editing now allows the generation of knock-out alleles for any gene and entire gene families. Even in the model plant <em>Arabidopsis thaliana</em>, gene editing is welcomed as T-DNA insertion lines do not always generate null alleles. Here, we show efficient generation of heritable mutations in Arabidopsis using CRISPR/Cas9 with a workload similar to generating overexpression lines. We obtain for several different genes Cas9 null-segregants with bi-allelic mutations in the T2 generation. While somatic mutations were predominantly generated by the canonical non-homologous end joining (cNHEJ) pathway, we observed inherited mutations that were the result of synthesis-dependent microhomology-mediated end joining (SD-MMEJ), a repair pathway linked to polymerase θ (PolQ). We also demonstrate that our workflow is compatible with a dual sgRNA approach in which a gene is targeted by two sgRNAs simultaneously. This paired nuclease method results in more reliable loss-of-function alleles that lack a large essential part of the gene. The ease of the CRISPR/Cas9 workflow should help in the eventual generation of true null alleles of every gene in the Arabidopsis genome, which will advance both basic and applied plant research.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arabidopsis-thaliana" class="hw-term hw-article-keyword hw-article-keyword-arabidopsis-thaliana" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arabidopsis thaliana</em></a></li><li class="kwd"><a href="/keyword/genome-engineering" class="hw-term hw-article-keyword hw-article-keyword-genome-engineering" rel="nofollow">genome engineering</a></li><li class="kwd"><a href="/keyword/genome-editing" class="hw-term hw-article-keyword hw-article-keyword-genome-editing" rel="nofollow">genome editing</a></li><li class="kwd"><a href="/keyword/rna-guided-nuclease" class="hw-term hw-article-keyword hw-article-keyword-rna-guided-nuclease" rel="nofollow">RNA-guided nuclease</a></li><li class="kwd"><a href="/keyword/alt-ej" class="hw-term hw-article-keyword hw-article-keyword-alt-ej" rel="nofollow">alt-EJ</a></li><li class="kwd"><a href="/keyword/polymerase-theta" class="hw-term hw-article-keyword hw-article-keyword-polymerase-theta" rel="nofollow">polymerase theta</a></li><li class="kwd"><a href="/keyword/crisprcas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR/Cas9</a></li></ul><p id="p-3">The precise introduction of a DNA double-strand break (DSB) in a plant genome can now be accomplished through a variety of techniques (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baltes and Voytas 2015</a>). However, the advent of Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)/CRISPR associated protein 9 (CRISPR/Cas9)-based technology has brought reliable gene editing (GE) within the reach of non-specialized molecular biology labs. The power of CRISPR/Cas9 compared to predecessor techniques lies in both a consistent high efficiency and a simple two-component design. A generic nuclease, Cas9, is guided to a target DNA sequence (protospacer) by associating with an artificial single guide RNA (sgRNA) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jinek <em>et al.</em> 2012</a>). Changing the typically 20 nucleotide long target-specific spacer sequence in the sgRNA is sufficient for redirecting the RNA-guided engineered nuclease to another genomic locus. In addition, several sgRNAs with different targets can be co-expressed allowing for multiplexing as exemplified in <em>Arabidopsis thaliana</em> by targeting of the PYRABACTIN RESISTANCE1-LIKE (PYL) family of abscisic acid receptor genes (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2016</a>) or the GOLVEN family (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Peterson <em>et al.</em> 2016</a>).</p><p id="p-4">DSBs are readily recognized by the plant cell and repaired. The non-homologous end-joining (NHEJ) pathway results in imprecise repair, producing small insertions and/or deletions (indels) at the cut site (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Knoll <em>et al.</em> 2014</a>). In Arabidopsis, one base pair (bp) insertions (+1) are usually observed in somatic cells (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fauser <em>et al.</em> 2014</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Feng <em>et al.</em> 2014</a>). Alternative EJ (alt-EJ) uses a molecularly distinct mechanism and microhomologies flanking the cut site to guide repair. Also known as microhomology-mediated end joining (MMEJ), alt-EJ often detected by relatively larger deletions that are generated (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Knoll <em>et al.</em> 2014</a>). A model of synthesis-dependent MMEJ (SD-MMEJ) was proposed that can explain different MMEJ repair outcomes, including the presence of templated insertions at the junction (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yu and McVey, 2010</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Khodaverdian <em>et al.</em> 2017</a>). NHEJ-mediated indel-formation is used to generate loss-of-function mutants. If the indel causes a frame-shift, a non-functional truncated protein can be translated, and/or a premature stop codon will trigger nonsense-mediated decay (NMD) causing organized mRNA degradation by the cell (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Popp and Maquat 2016</a>).</p><p id="p-5">CRISPR/Cas9 technology has been established for Arabidopsis and is continuously being developed further (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Feng <em>et al.</em> 2013</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mao <em>et al.</em> 2013</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Fauser <em>et al.</em> 2014</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Feng <em>et al.</em> 2014</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ma <em>et al.</em> 2015</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2015</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Osakabe <em>et al.</em> 2016</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tsutsui and Higashiyama 2017</a>, <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em>, 2016</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Denbow <em>et al.</em> 2017</a>, <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Peterson <em>et al.</em> 2016</a>). Reports using CRISPR/Cas9 in Arabidopsis are emerging that are not technology-focused, but rather limited in number taking into account the widespread use of this model organism, the short generation time and its ease of transformation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gao <em>et al.</em> 2015</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ning <em>et al.</em> 2015</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Xin <em>et al.</em> 2016</a>, <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2016</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Guseman <em>et al.</em> 2017</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2017</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lu <em>et al.</em> 2018</a>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ritter <em>et al.</em> 2017</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Durr <em>et al.</em> 2018</a>). The difficulties of using CRISPR/Cas9 to generate mutants in Arabidopsis have been attributed to the unique floral dip system of transformation in which inflorescences of T0 plants are infected with <em>Agrobacterium tumefaciens</em>. Primary transformants (T1) are derived via this process from a transformed egg cell (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bechtold <em>et al.</em> 2000</a>). Chimerism, <em>i.e.</em>, the presence of at least 3 different alleles, points to Cas9 activity at later stages during somatic growth. This indicates that the mutation did not occur within the egg cell or zygote, but rather after the first cell division. Furthermore, even when mutations are detected in T1 somatic cells, often WT alleles are retrieved once the CRISPR/Cas9 T-DNA has been segregated away (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2015</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Durr <em>et al.</em> 2018</a>). This can be attributed to gene editing efficiency, <em>i.e.</em>, the percentage of cells not WT, as the limited number of cells that make up the germ line have to be mutated for heritability. Recently, low activity of SpCas9 at 21° has been suggested to be causal as there was an increase in CRISPR/Cas9 mutagenesis in both somatic and germline mutations at 37° (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">LeBlanc <em>et al.</em> 2018</a>).</p><p id="p-6">Here, we report and quantify high editing efficiencies in T1 somatic cells and inheritance of NHEJ-repaired alleles in Arabidopsis. In T1 somatic cells we observe mostly single base pair insertions, which are likely the result of cNHEJ. In contrast, inherited mutations for three different sgRNAs showed alt-EJ repair outcomes of which two consistent with criteria for SD-MMEJ. Our workflow allows us to obtain Cas9 null-segregants with bi-allelic mutations in the T2 generation. Moreover, it is compatible with a dual sgRNA approach, leading to deletion of gene fragments and greater confidence in producing loss-of-function alleles.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Design of sgRNAs</h3><p id="p-7">In general, sgRNAs were selected for specificity using CRISPR-P (<a href="http://crispr.hzau.edu.cn/CRISPR/">http://crispr.hzau.edu.cn/CRISPR/</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lei <em>et al.</em> 2014</a>), taking into account predicted on-target efficiencies using sgRNAscorer (<a href="https://crispr.med.harvard.edu/sgRNAScorer/">https://crispr.med.harvard.edu/sgRNAScorer/</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chari <em>et al.</em> 2015</a>). An updated overview of estimated sgRNA parameters by CRISP-OR (<a href="http://crispor.tefor.net/">http://crispor.tefor.net/</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Haeussler <em>et al.</em> 2016</a>) can be found in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490505/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490505/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">sgRNA parameters used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Cloning of CRISPR/Cas9 constructs</h3><p id="p-9">CRISPR/Cas9 constructs were cloned as previously described (Figure S1, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Fauser <em>et al.</em> 2014</a>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Ritter <em>et al.</em> 2017</a>). Briefly, for each guide sequence, two complementary oligos with 4bp overhangs (Supplementary Table S1) were annealed and inserted via a cut-ligation reaction with BbsI (Thermo) and T4 DNA ligase (Thermo) in a Gateway ENTRY sgRNA shuttle vector. This is either pEN-C1.1 (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Fauser <em>et al.</em> 2014</a>) for single sgRNA constructs, or pMR217 (L1-R5) and pMR218 (L5-L2) (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Ritter <em>et al.</em> 2017</a>) for the dual sgRNA approach. The 5′ overhang already contains the G initiation nucleotide of the AtU6-26 polIII promoter. Next, using a Gateway LR reaction (ThermoFisher), one or two sgRNA modules were then combined with pDE-Cas9 (Basta, <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Fauser <em>et</em> al 2014</a>) or pDE-Cas9Km (pMR169, <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Ritter <em>et al.</em> 2017</a>) to yield the final expression clone.</p></div><div id="sec-4" class="subsection"><h3>Plant transformation</h3><p id="p-10">Expression clones were introduced in the Agrobacterium strain C58C1 (pMP90) using electroporation, which was used to transform Arabidopsis using the floral dip method (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clough and Bent 1998</a>).</p></div><div id="sec-5" class="subsection"><h3>Plant Material and Growth Conditions</h3><p id="p-11"><em>Arabidopsis thaliana</em> Col-0 were grown at 21° under long day (16-h light/8-h dark) conditions. Rapid selection of seeds with kanamycin and phosphinothricin (BASTA) selection was performed as described (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Harrison <em>et al.</em> 2006</a>).</p></div><div id="sec-6" class="subsection"><h3>Selection of CRISPR/Cas9 mutants</h3><p id="p-12">Typically, 16 kanamycin- or BASTA-resistant T1 plants are selected <em>in vitro</em> and transferred to a growth room. After 14 days, a single leaf is harvested, and genomic DNA prepared using Edwards buffer (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edwards <em>et al.</em> 1991</a>). Next, 5 µl template gDNA was used as a template in a standard 20 µl volume PCR reaction using GoTaq (Promega) with the supplied Green GoTaq Reaction Buffer. For single sgRNA constructs, the amplicon was treated with ExoSAP-IT (Thermo) and sequenced by standard capillary sequencing at the VIB Genomics Core Facility (<a href="https://corefacilities.vib.be/gsf">https://corefacilities.vib.be/gsf</a>). Quantitative sequence trace data were decomposed using TIDE (<a href="https://tide.nki.nl/">https://tide.nki.nl/</a>) using standard settings, except for the indel size range, which was set on the maximum (50). Primers for TIDE were designed using Primer3 (<a href="http://bioinfo.ut.ee/primer3-0.4.0/">http://bioinfo.ut.ee/primer3-0.4.0/</a>) using standard parameters. Approximately 700 bp asymmetrically surrounding the Cas9 cut site was amplified. The amplification primer at 200 bp from the site was used for sequencing.</p><p id="p-13">For each independent T1 line, approximately 64 T2 seeds were selected on either BASTA or kanamycin. Resistant <em>vs.</em> sensitive seedlings were analyzed using a chi-squared test and lines presumably having a single T-DNA locus continued. Typically, 15 seedlings of the most promising line (highest T1 efficiency, expected segregation) were grown on non-selective media and genotyped for the presence of the T-DNA locus using Cas9-specific primers (Table S1). Cas9 null-segregants are then analyzed for modifications at the locus of interest. The most promising plants are then propagated to T3, in which absence of Cas9 and presence of the mutation/deletion is confirmed by PCR and sequencing.</p></div><div id="sec-7" class="subsection"><h3>Amplicon subcloning</h3><p id="p-14">For confirmation of TIDE spectra, the PCR amplicon was cut from gel, purified using GeneJET PCR purification kit (Thermo Scientific) and cloned into pJET1.2 using the CloneJET PCR cloning kit (Thermo Scientific). Individual clones were sequenced using capillary electrophoresis.</p></div><div id="sec-8" class="subsection"><h3>RT-qPCR</h3><p id="p-15">Seedlings were grown in the same conditions as in <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Iñigo <em>et al.</em> (2016)</a>. Seedlings were frozen in liquid nitrogen and total RNA was extracted using RNeasy plant mini kit (Qiagen) and DNAse I (Promega) treatment. Next, 1 µg of RNA was used for cDNA synthesis using iScript kit (Bio-Rad). qRT-PCR was performed on a LightCycler 480 system (Roche) using the Fast Start SYBR Green I PCR mix (Roche) with three biological repeats and three technical repeats. Data were analyzed using the second derivative maximum method and relative expression levels were determined using the comparative cycle threshold method.</p></div><div id="sec-9" class="subsection"><h3>Data and reagent availability</h3><p id="p-16">Vectors and plant lines are available upon request. Primer sequences are provided in Supplemental Table S1. Accession numbers of the genes used in this study: GRXS17, AT4G04950; VQ19/MVQ4, AT3G15300; VQ33/MVQ3, AT5G53830; WRKY20, AT4G26640; WRKY2, AT5G56270; JAM2/bHLH13, AT1G01260; GLB3, AT4G32690. T-DNA lines used: grxs17-1, SALK_021301; wrky2-1, SALK_020399. All supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6455840">https://doi.org/10.25387/g3.6455840</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>High gene editing efficiency in T1 somatic tissue</h3><p id="p-17">The vector pDE-Cas9 has successfully been used for gene editing (GE) in Arabidopsis (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Fauser <em>et al.</em> 2014</a>). It contains an Arabidopsis codon-optimized SpCas9 sequence, driven by the <em>Petroselinum crispum</em> Ubiquitin4-2 promoter (pPcUBI). As kanamycin resistance is used more often in our lab, both in Arabidopsis and in tomato, we used pDE-Cas9Km (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Ritter <em>et al.</em> 2017</a>) in which the basta resistance cassette in pDE-Cas9 is replaced with <em>nptII</em> (Figure S1). In order to evaluate these vectors, we initially designed nine sgRNAs targeting five genes of interest: <em>JASMONATE ASSOCIATED MYC2 LIKE 2</em> (<em>JAM2</em>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sasaki-Sekimoto <em>et al.</em> 2013</a>), <em>VQ19</em> and <em>VQ33</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jing and Lin 2015</a>), <em>HEMOGLOBIN 3</em> (<em>GLB3</em>) and <em>GLUTAREDOXIN S17</em> (<em>GRXS17</em>) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nagels Durand <em>et al.</em> 2016</a>). sgRNAs were designed to minimize possible off-target activity (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lei <em>et al.</em> 2014</a>), and when possible predicted sgRNA efficiencies were taken into account (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chari <em>et al.</em> 2015</a>). An updated overview of estimated sgRNA parameters by CRISP-OR (<a href="http://crispor.tefor.net/">http://crispor.tefor.net/</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Haeussler <em>et al.</em> 2016</a>) can be found in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Although it is currently unknown if the models for sgRNA efficiency, based on empirical data from metazoan cells holds true in plants, we anticipate that at least some sgRNA sequence parameters will be similar as CRISPR/Cas9 is a fully heterologous system. To better ensure the generation of loss-of-function alleles, sgRNAs were preferably chosen in the 5′ end of the first exon (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In the case of <em>JAM2</em>, we specifically designed two sgRNAs that targeted the sequence encoding the JAZ interaction domain (JID) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fernández-Calvo <em>et al.</em> 2011</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CRISPR/Cas9-induced somatic mutations in T1 Arabidopsis plants. A, genomic structure of the targeted genes and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns; white arrows gene orientation. sgRNA numbers are arbitrary identifiers. B, example result of a TIDE analysis. A leaf of a T1 plant expressing a CRISPR/Cas9 construct targeting GLB3 was used to prepare genomic DNA. The targeted region was amplified by PCR and sequenced using standard Sanger sequencing. TIDE software was used to visualize the indel spectrum and estimate overall editing efficiency (top right corner). Bars indicate the number of sequences with a given indel size. Pink bar (indel size of zero) represents WT or base substitution alleles. C, Verification of TIDE using sequencing of individual amplicon subclones. The PAM is highlighted in green, the triangle points to the Cas9 cut site. i, insertion, d, deletion, m, mutation are followed with the number of bases involved." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CRISPR/Cas9-induced somatic mutations in T1 Arabidopsis plants. A, genomic structure of the targeted genes and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns; white arrows gene orientation. sgRNA numbers are arbitrary identifiers. B, example result of a TIDE analysis. A leaf of a T1 plant expressing a CRISPR/Cas9 construct targeting &lt;em&gt;GLB3&lt;/em&gt; was used to prepare genomic DNA. The targeted region was amplified by PCR and sequenced using standard Sanger sequencing. TIDE software was used to visualize the indel spectrum and estimate overall editing efficiency (top right corner). Bars indicate the number of sequences with a given indel size. Pink bar (indel size of zero) represents WT or base substitution alleles. C, Verification of TIDE using sequencing of individual amplicon subclones. The PAM is highlighted in green, the triangle points to the Cas9 cut site. i, insertion, d, deletion, m, mutation are followed with the number of bases involved.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F1.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2603/F1.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490535" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">CRISPR/Cas9-induced somatic mutations in T1 Arabidopsis plants. A, genomic structure of the targeted genes and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns; white arrows gene orientation. sgRNA numbers are arbitrary identifiers. B, example result of a TIDE analysis. A leaf of a T1 plant expressing a CRISPR/Cas9 construct targeting <em>GLB3</em> was used to prepare genomic DNA. The targeted region was amplified by PCR and sequenced using standard Sanger sequencing. TIDE software was used to visualize the indel spectrum and estimate overall editing efficiency (top right corner). Bars indicate the number of sequences with a given indel size. Pink bar (indel size of zero) represents WT or base substitution alleles. C, Verification of TIDE using sequencing of individual amplicon subclones. The PAM is highlighted in green, the triangle points to the Cas9 cut site. i, insertion, d, deletion, m, mutation are followed with the number of bases involved.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The sgRNA cloning procedure (Figure S1A) uses the type II restriction enzyme BbsI and utilizes a 5′ ATTG overhang of which the G serves as the first nucleotide of the sgRNA when transcribed by the polymerase III promoter AtU6-26. Most sgRNAs were of the GN19-type with the 5′ G being the first transcribed base of a 20-bp long guide sequence. One sgRNA, JAM2-140, was of the GN20-type. An extra 5′ G or GG attached to the sgRNA should not hinder efficiency (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cho <em>et al.</em> 2014</a>). Another sgRNA VQ33-42, was a GN18-type. Truncated sgRNAs (tru-gRNAs) down to a 17bp guide sequence have been shown to be as efficient as 20bp guides in human cells (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fu <em>et al.</em> 2014</a>).</p><p id="p-20">For each single sgRNA construct, approximately 15 T1 Arabidopsis plants were selected on basta or kanamycin respectively. One of the first true leaves was harvested for genomic DNA extraction. A region spanning the predicted cut site was amplified by PCR and the amplicon sequenced by traditional Sanger sequencing. Arabidopsis CRISPR/Cas9 T1 plants are typically chimeric, defined as having at least three different alleles for a locus (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Feng <em>et al.</em> 2014</a>). Different cell files showed different indels in both alleles after NHEJ-mediated repair, leading to a range of detectable indels in a single leaf and a complex chromatogram. The quantitative sequence trace data were therefore decomposed using the Tracking of Indels by DEcomposition (TIDE) software (<a href="https://tide.nki.nl/">https://tide.nki.nl/</a>) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brinkman <em>et al.</em> 2014</a>). This results in an estimation of overall editing efficiency (percentage of cells not WT) and the spectrum and frequency of the dominant indel types (See <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> for an example for <em>GLB3</em>). Subcloning of amplicons followed by sequencing yielded similar profiles (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). Furthermore, examination of genomic DNA of different leaves yielded comparable but not identical patterns (Figure S2).</p><p id="p-21">All but one sgRNA had high editing efficiencies with the median efficiency being higher than 80% (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Notably, VQ33-38, the sgRNA predicted by all three algorithms to have the worst efficiency (Table S2) had one of the highest efficiencies <em>in planta</em>. Next, we used the data generated, to investigate chimerism in the T1 plants. The most frequently observed mutation is a 1 bp insertion, followed by deletions of increasing size (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Large insertions were very uncommon. However, depending on the sgRNA larger deletions of a particular size were often observed. Potentially this is related to MMEJ, whereby regions of microhomology help initiate polymerase Q repair by annealing of single-stranded DNA overhangs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Black <em>et al.</em> 2016</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Shen <em>et al.</em> 2017</a>). In summary, we show high rates of CRISPR/Cas9 mutagenesis in Arabidopsis T1 somatic tissue for most tested sgRNAs and that TIDE is a robust method to evaluate sgRNA efficiency.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="High gene editing efficiency in Arabidopsis T1 generation. A, boxplots showing TIDE estimated editing efficiencies for up to 15 T1 plants for nine different sgRNAs. +, mean; horizontal line, median; open circles, individual data points. B, heat map showing the number of T1 plants with at least 1% estimated frequency of an indel of a given size. Boxed are larger deletions (&gt; 6 bp) observed in 5 or more T1 plants." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;High gene editing efficiency in Arabidopsis T1 generation. A, boxplots showing TIDE estimated editing efficiencies for up to 15 T1 plants for nine different sgRNAs. +, mean; horizontal line, median; open circles, individual data points. B, heat map showing the number of T1 plants with at least 1% estimated frequency of an indel of a given size. Boxed are larger deletions (&gt; 6 bp) observed in 5 or more T1 plants.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F2.medium.gif" width="440" height="390"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2603/F2.medium.gif" width="440" height="390"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490645" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">High gene editing efficiency in Arabidopsis T1 generation. A, boxplots showing TIDE estimated editing efficiencies for up to 15 T1 plants for nine different sgRNAs. +, mean; horizontal line, median; open circles, individual data points. B, heat map showing the number of T1 plants with at least 1% estimated frequency of an indel of a given size. Boxed are larger deletions (&gt; 6 bp) observed in 5 or more T1 plants.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Inheritance of mutations</h3><p id="p-23">Focusing on <em>GLB3</em>, we investigated the heritability of mutations after selfing and selected for T2 progeny that had lost the T-DNA (Cas9 null-segregants). First, we identified three T1 lines with a single T-DNA locus by segregation analysis of the kanamycin resistance marker in T2 seedlings. Of these three lines we germinated 14 to 17 seedlings on soil, prepared genomic DNA and genotyped using Cas9 specific primers to identify null-segregants (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The genomic DNA of these plants was re-used to amplify the target site and sequencing data were analyzed using TIDE to identify genotypes at the target locus. All 15 tested null-segregants were found to be non-chimeric: 8 were WT, 5 heterozygous and 2 were homozygous. Hence, inherited mutations were present in the T2 progeny of all three independent T1 lines. Although we only detected the desired homo-allelic Cas9 null-segregants in the progeny of one T1 line, heterozygous alleles will lead to the desired genotypes in the next generation. An outcome also overrepresented in T1 somatic mutations for <em>GLB3</em>, and frequently observed in the inherited mutations from independent events was a 10 bp deletion, indicative of alt-EJ (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Lastly, we identified a heritable T to A substitution which led to a single nucleotide variation (SNV) and here results in a premature stop codon (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3C</a>). This occurs when a single bp deletion is followed by a single bp insertion, an event very rarely observed for CRISPR/Cas9 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kim <em>et al.</em> 2017</a>). In conclusion, the pDE-Cas9 vectors allow for efficient and inheritable genome editing in Arabidopsis with the possibility of producing transgene free homo-allelic mutants in the T2 generation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inheritance of CRISPR/Cas9 mutations. A, PCR amplification of the Cas9 transgene in T2 seedlings from 3 independent GLB3 lines: -1, -2 and -8. Genotypes for all Cas9 null-segregants were estimated using TIDE. NA, not assayed; WT, wild-type; m1, 1bp substitution. Boxed plants were continued. B-C, Sequence alignment of the targeted locus for Col-0 and glb3-3 (B, Line 1, plant 3) or glb3-4 (C, Line 1, plant 7). PAM is highlighted, the Cas9 cut site indicated with a triangle, and microhomology boxed. Mutated bases are in red, deleted bases replaced by a dash. The reading frame is marked. The stop codon generated by the mutation is underlined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inheritance of CRISPR/Cas9 mutations. A, PCR amplification of the Cas9 transgene in T2 seedlings from 3 independent GLB3 lines: -1, -2 and -8. Genotypes for all Cas9 null-segregants were estimated using TIDE. NA, not assayed; WT, wild-type; m1, 1bp substitution. Boxed plants were continued. B-C, Sequence alignment of the targeted locus for Col-0 and glb3-3 (B, Line 1, plant 3) or glb3-4 (C, Line 1, plant 7). PAM is highlighted, the Cas9 cut site indicated with a triangle, and microhomology boxed. Mutated bases are in red, deleted bases replaced by a dash. The reading frame is marked. The stop codon generated by the mutation is underlined.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F3.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2603/F3.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Inheritance of CRISPR/Cas9 mutations. A, PCR amplification of the Cas9 transgene in T2 seedlings from 3 independent GLB3 lines: -1, -2 and -8. Genotypes for all Cas9 null-segregants were estimated using TIDE. NA, not assayed; WT, wild-type; m1, 1bp substitution. Boxed plants were continued. B-C, Sequence alignment of the targeted locus for Col-0 and glb3-3 (B, Line 1, plant 3) or glb3-4 (C, Line 1, plant 7). PAM is highlighted, the Cas9 cut site indicated with a triangle, and microhomology boxed. Mutated bases are in red, deleted bases replaced by a dash. The reading frame is marked. The stop codon generated by the mutation is underlined.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Isolation of a new grxs17 CRISPR allele</h3><p id="p-25">Previously we characterized in detail two independent knock-out alleles of <em>GRXS17</em>, a gene encoding a component of the FeS cluster assembly pathway (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Iñigo <em>et al.</em> 2016</a>). The allele <em>grxs17-1</em> (SALK_021301) contains a T-DNA in the second exon (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), whereas the <em>grxs17-2</em> allele expresses an antisense construct (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2011</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation and analysis of the grxs17-3 allele. A, Gene and protein structure of GRXS17. The location of the grxs17-1 T-DNA and the sgRNA used in this study are indicated. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. TRX, thioredoxin domain; GRX, glutaredoxin domain. B, TIDE analysis of T1 line 9. Genomic DNA was PCR amplified and sequenced. The indel spectrum is visualized with an estimated overall efficiency and the frequency of each indel. C, PCR amplification of the Cas9 transgene. Null-segregants are boxed and the continued plant marked with a triangle. TIDE estimated genotypes for GRXS17 are given for the null segregants. D, Sequence alignment of the targeted locus for Col-0 and grxs17-3 (Line 9, plant 15). PAM is highlighted, the Cas9 cut site indicated with a triangle. Mutated bases are in red, deleted bases replaced by an en dash. The reading frame is marked. E, GRXS17 gene expression analyzed by RT-qPCR. Expression relative to Col-0 is plotted using primers annealing both at the 5&#x2032; and the 3&#x2032; of the transcript and the mutation. F, rosette phenotypes of Col-0, the T-DNA insertion line grxs17-1, the antisense line grxs17-2 and the grxs17-3 CRISPR allele." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation and analysis of the &lt;em&gt;grxs17-3&lt;/em&gt; allele. A, Gene and protein structure of GRXS17. The location of the &lt;em&gt;grxs17-1&lt;/em&gt; T-DNA and the sgRNA used in this study are indicated. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. TRX, thioredoxin domain; GRX, glutaredoxin domain. B, TIDE analysis of T1 line 9. Genomic DNA was PCR amplified and sequenced. The indel spectrum is visualized with an estimated overall efficiency and the frequency of each indel. C, PCR amplification of the Cas9 transgene. Null-segregants are boxed and the continued plant marked with a triangle. TIDE estimated genotypes for &lt;em&gt;GRXS17&lt;/em&gt; are given for the null segregants. D, Sequence alignment of the targeted locus for Col-0 and grxs17-3 (Line 9, plant 15). PAM is highlighted, the Cas9 cut site indicated with a triangle. Mutated bases are in red, deleted bases replaced by an en dash. The reading frame is marked. E, &lt;em&gt;GRXS17&lt;/em&gt; gene expression analyzed by RT-qPCR. Expression relative to Col-0 is plotted using primers annealing both at the 5&#x2032; and the 3&#x2032; of the transcript and the mutation. F, rosette phenotypes of Col-0, the T-DNA insertion line &lt;em&gt;grxs17-1&lt;/em&gt;, the antisense line &lt;em&gt;grxs17-2&lt;/em&gt; and the &lt;em&gt;grxs17-3&lt;/em&gt; CRISPR allele.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F4.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2603/F4.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490514" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Generation and analysis of the <em>grxs17-3</em> allele. A, Gene and protein structure of GRXS17. The location of the <em>grxs17-1</em> T-DNA and the sgRNA used in this study are indicated. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. TRX, thioredoxin domain; GRX, glutaredoxin domain. B, TIDE analysis of T1 line 9. Genomic DNA was PCR amplified and sequenced. The indel spectrum is visualized with an estimated overall efficiency and the frequency of each indel. C, PCR amplification of the Cas9 transgene. Null-segregants are boxed and the continued plant marked with a triangle. TIDE estimated genotypes for <em>GRXS17</em> are given for the null segregants. D, Sequence alignment of the targeted locus for Col-0 and grxs17-3 (Line 9, plant 15). PAM is highlighted, the Cas9 cut site indicated with a triangle. Mutated bases are in red, deleted bases replaced by an en dash. The reading frame is marked. E, <em>GRXS17</em> gene expression analyzed by RT-qPCR. Expression relative to Col-0 is plotted using primers annealing both at the 5′ and the 3′ of the transcript and the mutation. F, rosette phenotypes of Col-0, the T-DNA insertion line <em>grxs17-1</em>, the antisense line <em>grxs17-2</em> and the <em>grxs17-3</em> CRISPR allele.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">A T1 parental line described above that showed high editing efficiency (97.3%) in somatic tissue and had a single T-DNA locus was identified (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Two Cas9 null-segregants of the T2 progeny were genotyped using TIDE (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). This yielded the <em>grxs17-3</em> allele that was predicted to have a (-8,-8) genotype. Inspection of the sequence in T3 plants revealed an additional 4 base pair insertion, nevertheless leading to loss of the reading frame (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). Using RT-qPCR, we could observe strong downregulation (∼80%) of the entire <em>GRXS17</em> transcript (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4E</a>). This is probably the result of nonsense-mediated decay (NMD), a process triggering mRNA degradation in case a premature stop codon is present (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Popp and Maquat 2016</a>). However, as there is no exon-exon boundary 3′ of the premature stop codon, this can be a case of exon-junction complex (EJC)-independent NMD, wherein NMD is triggered by a long 3′ UTR (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fatscher <em>et al.</em> 2014</a>). Remarkably, the elongated leaf developmental phenotype present in both <em>grxs17-1</em> and <em>grxs17-2</em> was not visible in <em>grxs17-3</em> (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4F</a>). GRXS7 is a multidomain protein with an N-terminal thioredoxin (TRX) domain followed by three glutaredoxin (GRX) domains (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>). The human GRX3 ortholog has only 2 GRX domains, whereas the yeast Grx3/Grx4 orthologs have only one GRX domain (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Couturier <em>et al.</em> 2015</a>). We hypothesize that the <em>grxs17-3</em> allele is not a null allele and possibly expresses a C-terminally truncated GRXS17 protein with a functional TRX and GRX domain.</p></div><div id="sec-14" class="subsection"><h3>A dual sgRNA approach for gene deletions</h3><p id="p-28">Choice of the sgRNA target site is pivotal in generating a reliable knock-out. Genes can contain alternative start codons, have alternative first exon usage, exon skipping and/or C-terminally truncated proteins and therefore might still be partially functional as exemplified above. In-depth knowledge on the gene structure, transcript and protein is therefore advisable. However, in many cases this information is not complete. Therefore, we examined in Arabidopsis a dual sgRNA approach in which two sgRNAs target the same gene to remove a large segment (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2014</a>, <a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2016</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ordon <em>et al.</em> 2017</a>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Durr <em>et al.</em> 2018</a>).</p><p id="p-29">Using a MultiSite Gateway based sgRNA multiplexing approach we previously described (<a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Ritter <em>et al.</em> 2017</a>) we co-expressed two sgRNAs in pDE-Cas9Km. We used this method to target the gene encoding the transcription factor WRKY20, which is closely related to WRKY2, with two sgRNAs. For the latter, a characterized T-DNA insertion mutant <em>wrky2-1</em> is available representing a strong loss-of-function or null allele (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ueda <em>et al.</em> 2011</a>). We transformed the <em>wrky2-1</em> background with a dual sgRNA construct for <em>WRKY20</em>, predicted to remove a 247 bp fragment encoding the first WRKY protein domain in addition to putting the remainder of the sequence out of frame (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Without any phenotypic selection, we applied the same workflow as before. We selected four independent T1 lines showing high levels of the expected deletion and containing a single T-DNA locus (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). For each line, one or more null-segregants were identified in T2 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>) and genotyped for the <em>WRKY20</em> locus. Of seven Cas9 null-segregants successfully genotyped, two plants were homozygous for the expected deletion, three heterozygous and two wild-type (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>). Sequence analysis of two homozygous deletion mutants showed that <em>wrky2-1 wrky20-1</em> (plant A15-8) had the predicted 247 bp deletion, whereas the other allele <em>wrky2-1 wrky20-2</em> (plant B2-5) only had a 246 bp segment deleted, possibly restoring the reading frame (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>). This shows that a dual sgRNA approach for deleting gene fragments is feasible with relatively few numbers of genotyped plants.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="WRKY20 dual sgRNA approach. A, genomic structure of WRKY20 and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. B, PCR analysis of T1 lines. Leaf genomic DNA of 2 batches (A and B) of 20 chimeric T1 plants was PCR amplified. The expected size of the WT WRKY20 amplicon is indicated as well as the expected size of the deletion of 247 bp between Cas9 cut sites. Four continued T1 lines having one T-DNA locus are highlighted with green boxes. C, Cas9 PCR for the four continued lines in T2 generation. Putative Cas9 null-segregants are indicated with green boxes. D, Cas9 null-segregants were genotypes for WRKY20. The selected lines A15-8 (wrky2-1 wrky20-1) and B2-5 (wrky2-1 wrky20-2) are boxed. E, Sequence alignment of the simultaneously targeted loci for Col-0 and alleles wrky20-1 and wrky20-2. PAMs are highlighted, the Cas9 cut sites indicated with triangles. Deleted bases are indicated with dashed lines. The reading frame is marked." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;WRKY20&lt;/em&gt; dual sgRNA approach. A, genomic structure of &lt;em&gt;WRKY20&lt;/em&gt; and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. B, PCR analysis of T1 lines. Leaf genomic DNA of 2 batches (A and B) of 20 chimeric T1 plants was PCR amplified. The expected size of the WT &lt;em&gt;WRKY20&lt;/em&gt; amplicon is indicated as well as the expected size of the deletion of 247 bp between Cas9 cut sites. Four continued T1 lines having one T-DNA locus are highlighted with green boxes. C, Cas9 PCR for the four continued lines in T2 generation. Putative Cas9 null-segregants are indicated with green boxes. D, Cas9 null-segregants were genotypes for WRKY20. The selected lines A15-8 (&lt;em&gt;wrky2-1 wrky20-1&lt;/em&gt;) and B2-5 (&lt;em&gt;wrky2-1 wrky20-2&lt;/em&gt;) are boxed. E, Sequence alignment of the simultaneously targeted loci for Col-0 and alleles &lt;em&gt;wrky20-1&lt;/em&gt; and &lt;em&gt;wrky20-2&lt;/em&gt;. PAMs are highlighted, the Cas9 cut sites indicated with triangles. Deleted bases are indicated with dashed lines. The reading frame is marked.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F5.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/8/2603/F5.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490525" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child"><em>WRKY20</em> dual sgRNA approach. A, genomic structure of <em>WRKY20</em> and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. B, PCR analysis of T1 lines. Leaf genomic DNA of 2 batches (A and B) of 20 chimeric T1 plants was PCR amplified. The expected size of the WT <em>WRKY20</em> amplicon is indicated as well as the expected size of the deletion of 247 bp between Cas9 cut sites. Four continued T1 lines having one T-DNA locus are highlighted with green boxes. C, Cas9 PCR for the four continued lines in T2 generation. Putative Cas9 null-segregants are indicated with green boxes. D, Cas9 null-segregants were genotypes for WRKY20. The selected lines A15-8 (<em>wrky2-1 wrky20-1</em>) and B2-5 (<em>wrky2-1 wrky20-2</em>) are boxed. E, Sequence alignment of the simultaneously targeted loci for Col-0 and alleles <em>wrky20-1</em> and <em>wrky20-2</em>. PAMs are highlighted, the Cas9 cut sites indicated with triangles. Deleted bases are indicated with dashed lines. The reading frame is marked.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Next, we combined two sgRNAs targeting <em>VQ33</em> (VQ33-42 and VQ33-1) that displayed high efficiency when tested individually (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Working together, they are predicted to remove a fragment of 459 bp, virtually removing the <em>VQ33</em> coding sequence (Figure S3A). We proceeded with the same workflow as for <em>WRKY20</em> (Figure S3B-D). Out of four Cas9 null-segregants, two were homozygous for the expected gene fragment deletion, one heterozyogous and one WT. The allele <em>vq33-1</em> (plant 11-8), albeit it had an extra 1 bp insertion, still led to a 458 bp out-of-frame deletion (Figure S3E).</p><p id="p-32">In summary, we established a straightforward dual sgRNA approach to obtain plants homozygous for relatively large deletions of gene fragments in the T2 generation in <em>Arabidopsis thaliana</em>.</p></div><div id="sec-15" class="subsection"><h3>grxs17-4 confirms the grxs17-1 developmental phenotype</h3><p id="p-33">Next, we tried the dual sgRNA approach for <em>GRXS17</em>. We targeted the first sgRNA (GRXS17-133) at the 5′ end and the second sgRNA (GRXS17-67) at the 3′ end of the gene to remove 1953 bp and <em>GRXS17</em> almost entirely (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). The <em>GRXS17</em> locus was amplified for sixteen independent T1 plants using primers spanning the expected deletion. In comparison with <em>VQ33</em> and <em>WRKY20</em>, only two plants clearly showed bands of the expected size for the predicted deletion (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). Two identified Cas9-null segregants (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>) did not show the expected large deletion, but instead an indel was found at the first sgRNA site in the first exon leading to a frameshift (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>). The indel remarkably was a 30bp deletion combined with a 2bp insertion. We named this allele <em>grxs17-4</em>. Confirming our hypothesis that <em>grxs17-3</em> is indeed not a null allele, <em>grxs17-4</em> showed the leaf phenotype of <em>grxs17-1</em> and <em>grxs17-2</em> (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Iñigo <em>et al.</em> 2016</a>, <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6E</a>). In conclusion, in the event the dual sgRNA approach does not yield the designed gene fragment deletion, each individual sgRNA may lead to useful alleles.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A dual sgRNA approach for GRXS17. A, genomic structure of GRXS17 and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. B, PCR analysis of T1 lines. Leaf genomic DNA of 16 chimeric T1 plants was PCR amplified. The expected size of the WT GRXS17 amplicon is indicated as well as the expected size of the deletion of 1953 bp between Cas9 cut sites. One T1 line having one T-DNA locus that was continued is highlighted with a green box. C, Cas9 PCR for 8 T2 CRISPR plants. Putative Cas9 null-segregants are indicated with green boxes. D, Sequence alignment of the sequence surrounding the 5&#x2032; sgRNA site for Col-0 and grxs17-4 (Line 2, plant 8). PAM is highlighted, the Cas9 cut site indicated with a triangle. Mutated bases are in red, deleted bases replaced by an en dash. The reading frame is marked. E, representative rosette phenotypes of WT Col-0 (top) and grxs17-4 (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A dual sgRNA approach for &lt;em&gt;GRXS17&lt;/em&gt;. A, genomic structure of &lt;em&gt;GRXS17&lt;/em&gt; and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. B, PCR analysis of T1 lines. Leaf genomic DNA of 16 chimeric T1 plants was PCR amplified. The expected size of the WT &lt;em&gt;GRXS17&lt;/em&gt; amplicon is indicated as well as the expected size of the deletion of 1953 bp between Cas9 cut sites. One T1 line having one T-DNA locus that was continued is highlighted with a green box. C, Cas9 PCR for 8 T2 CRISPR plants. Putative Cas9 null-segregants are indicated with green boxes. D, Sequence alignment of the sequence surrounding the 5&#x2032; sgRNA site for Col-0 and &lt;em&gt;grxs17-4&lt;/em&gt; (Line 2, plant 8). PAM is highlighted, the Cas9 cut site indicated with a triangle. Mutated bases are in red, deleted bases replaced by an en dash. The reading frame is marked. E, representative rosette phenotypes of WT Col-0 (top) and &lt;em&gt;grxs17-4&lt;/em&gt; (bottom).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F6.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/8/2603/F6.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490541" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">A dual sgRNA approach for <em>GRXS17</em>. A, genomic structure of <em>GRXS17</em> and location of the sgRNAs. Dark green boxes designate exons; light green boxes, UTRs; solid lines, introns. B, PCR analysis of T1 lines. Leaf genomic DNA of 16 chimeric T1 plants was PCR amplified. The expected size of the WT <em>GRXS17</em> amplicon is indicated as well as the expected size of the deletion of 1953 bp between Cas9 cut sites. One T1 line having one T-DNA locus that was continued is highlighted with a green box. C, Cas9 PCR for 8 T2 CRISPR plants. Putative Cas9 null-segregants are indicated with green boxes. D, Sequence alignment of the sequence surrounding the 5′ sgRNA site for Col-0 and <em>grxs17-4</em> (Line 2, plant 8). PAM is highlighted, the Cas9 cut site indicated with a triangle. Mutated bases are in red, deleted bases replaced by an en dash. The reading frame is marked. E, representative rosette phenotypes of WT Col-0 (top) and <em>grxs17-4</em> (bottom).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Inherited mutations generated by synthesis-dependent MMEJ</h3><p id="p-35">It struck us that three of the final mutant alleles (<em>glb3-3</em>, <em>grxs17-3</em>, <em>grxs17-4</em>) we generated with a single sgRNA showed hallmarks of repair by alt-EJ. However, atypical for MMEJ, additional insertions were observed for <em>grxs17-3</em> and <em>grxs17-4</em>. Therefore, we analyzed the repair outcomes with criteria for SD-MMEJ, a model proposed for polymerase theta(PolQ)-mediated repair which also explains templated insertions (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Khodaverdian <em>et al.</em> 2017</a>). In this model, ends are resected at the DSB, after which microhomology regions called “primer repeat” (P1+P2) anneal via loops or hairpins to a region within 30 bp of the break. PolQ presumably then elongates the strand until another microhomology (MH1) is synthesized that has a counterpart (MH2) at the other side of the DSB, which then anneal to repair the break (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Khodaverdian <em>et al.</em> 2017</a>). In case for <em>grxs17-3</em> and <em>grxs17-4</em> we discovered that insertions could have been templated by neighboring sequences (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and Figure S4). In both cases, a primer repeat was present upstream of the cut site, which after ‘loop out’ formation was elongated with 5 and 3 bp respectively. In both cases the microhomology repeat only consisted out of a single base, which nevertheless fits criteria for SD-MMEJ (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Khodaverdian <em>et al.</em> 2017</a>). A direct repeat can be observed (mh+insert+p) of 9 and 7 bp respectively. In summary, the model for SD-MMEJ is capable of explaining the observed inherited alt-EJ-associated mutations. This suggests a role for PolQ in DSB-repair in the Arabidopsis germline.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="grxS17-3 is explained by loop out SD-MMEJ. A, GRXS17 wild-type sequence surrounding the Cas9 cut site, which is indicated with a triangle. Primer repeat (P) regions are indicated in red with P2 break-proximal. Microhomology repeats (MH) are indicated in green. B, End resection by 5&#x2032;-3&#x2032; nuclease activity and unwinding by helicase activity. C, loop formation by P1-P2 basepairing. D, templated elongation by polymerase activity. E, unwinding. F, annealing of mh2 with mh1 templated complimentary overhang. G, observed repair product in grxs17-3. Inserted nucleotides in blue, resulting direct repeat underlined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1778309367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;g&lt;em&gt;rxS17-3&lt;/em&gt; is explained by loop out SD-MMEJ. A, &lt;em&gt;GRXS17&lt;/em&gt; wild-type sequence surrounding the Cas9 cut site, which is indicated with a triangle. Primer repeat (P) regions are indicated in red with P2 break-proximal. Microhomology repeats (MH) are indicated in green. B, End resection by 5&#x2032;-3&#x2032; nuclease activity and unwinding by helicase activity. C, loop formation by P1-P2 basepairing. D, templated elongation by polymerase activity. E, unwinding. F, annealing of mh2 with mh1 templated complimentary overhang. G, observed repair product in &lt;em&gt;grxs17-3&lt;/em&gt;. Inserted nucleotides in blue, resulting direct repeat underlined.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2603/F7.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/8/2603/F7.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2603/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490631" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-36" class="first-child">g<em>rxS17-3</em> is explained by loop out SD-MMEJ. A, <em>GRXS17</em> wild-type sequence surrounding the Cas9 cut site, which is indicated with a triangle. Primer repeat (P) regions are indicated in red with P2 break-proximal. Microhomology repeats (MH) are indicated in green. B, End resection by 5′-3′ nuclease activity and unwinding by helicase activity. C, loop formation by P1-P2 basepairing. D, templated elongation by polymerase activity. E, unwinding. F, annealing of mh2 with mh1 templated complimentary overhang. G, observed repair product in <em>grxs17-3</em>. Inserted nucleotides in blue, resulting direct repeat underlined.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Efficient CRISPR/Cas9 gene editing in Arabidopsis</h3><p id="p-37">The CRISPR/Cas9 technology shows promise to speed up reverse genetics experiments in Arabidopsis. Here we demonstrate efficient recovery of Cas9-free Arabidopsis mutants using single and double sgRNA constructs in the T2 generation without phenotypic selection. Previous negative experiences with CRISPR/Cas9 have been attributed to the weak activity of the 35S promoter in germ-line cells (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2015</a>) or low activity of SpCas9 at 21° (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">LeBlanc <em>et al.</em> 2018</a>). The promoter used here, PcUBI, is expressed widely, but detailed expression in germ-line cells has not yet been studied (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kawalleck <em>et al.</em> 1993</a>). Other vector elements have been reported to play a role such as the vector backbone (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mao <em>et al.</em> 2016</a>), Cas9 coding sequence (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Johnson <em>et al.</em> 2015</a>) and the terminator sequence (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2015</a>). We did not observe any obvious differences using either <em>nptII</em> or <em>bar</em> as selection markers. Systematic analysis of all vector parameters is now achievable using modular cloning systems, which might allow identification of the best combinations (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vazquez-Vilar <em>et al.</em> 2016</a>).</p><p id="p-38">We consider the workflow presented here as already an acceptable workload comparable to the routine generation of overexpression lines (Figure S5). Nonetheless, several improvements have recently been developed. For example, a fluorescent marker for identification of transgenic T1 seeds has been reported (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Tsutsui and Higashiyama 2017</a>) and also cloned into pDE-Cas9 for CRISPR/Cas9 in <em>Camelina sativa</em> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Morineau <em>et al.</em> 2017</a>, <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Durr <em>et al.</em> 2018</a>). When Cas9 is driven with a promoter active in the egg cell, non-chimeric homozygous or bi-allelic mutants can already be retrieved in the T1 generation, although Cas9 null-segregants also only appear in T2 (<a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2015</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yan <em>et al.</em> 2015</a>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Mao <em>et al.</em> 2016</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Eid <em>et al.</em> 2016</a>). Finally, applying 37° heat treatments in the vegetative phase may improve SpCas9 activity and would be compatible with our workflow (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">LeBlanc <em>et al.</em> 2018</a>).</p></div><div id="sec-19" class="subsection"><h3>TIDE as a useful tool to study mutations</h3><p id="p-39">Efficiency of CRISPR/Cas9 also clearly depends on the choice of sgRNA, although all sgRNAs tested in this study were active to some degree. Several models have been constructed to predict on-target editing efficiency based on the sgRNA primary sequence and on-target efficiency data from metazoans (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Doench <em>et al.</em> 2014</a>, <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Moreno-Mateos <em>et al.</em> 2015</a>). Due to the lack of sufficient data, no plant-specific design models are currently available. As previously reported (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ordon <em>et al.</em> 2017</a>), we did not observe any obvious correlation between these predictions and our observed efficiencies in Arabidopsis. It is unclear why this is the case for a heterologous system such as CRISPR/Cas9. Therefore - for the time being - we continue to take into account metazoan models when designing plant sgRNAs. It has been suggested to pre-screen sgRNAs in protoplasts (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2014</a>). Given the ease of Arabidopsis transformation via floral dip, we conclude from this study that designing several sgRNAs for the same target and testing somatic mutations in T1 might be an equally rapid method to identify efficient sgRNAs, while simultaneously obtaining the desired mutants.</p><p id="p-40">Several methods have been used to study CRISPR/Cas9-induced mutations, most importantly cleaved amplified polymorphic sequence (CAPS), T7 endonuclease, next-generation sequencing and high-resolution melting curve analysis (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Denbow <em>et al.</em> 2017</a>). The method used here, TIDE (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brinkman <em>et al.</em> 2014</a>), has several advantages. First, it does not require a restriction enzyme site overlapping the Cas9 cut site as in CAPS. Second, it allows the starting genomic DNA to be relatively impure allowing for more economic DNA extraction methods compared to T7-based assays. Third, it uses standard capillary Sanger sequencing that can be readily performed for even a single sample. Fourthly, it can provide an insight in the indel spectrum of mosaics similar to next-generation sequencing as well as providing an idea of overall efficiencies. These TIDE efficiencies are likely an underestimation. For example, TIDE is unable to detect rare SNVs as observed for <em>glb3-4</em>. The <em>grxs17-3</em> allele also revealed that mutations can be more complex than predicted by TIDE: a predicted 8 bp deletion was actually a 12 bp deletion combined with a 4 bp insertion.</p></div><div id="sec-20" class="subsection"><h3>Know your target gene</h3><p id="p-41">The absence of the typical <em>grxs17</em> phenotype in the CRISPR allele <em>grxs17-3</em> is an example of how it is important to study independent alleles made with either different sgRNAs or with other methods when interpreting phenotypes of CRISPR/Cas9-generated alleles as knock-out effects. A seemingly widespread, but only recently discovered phenomenon is conditional alternative promoter selection, resulting in alternative N-termini (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ushijima <em>et al.</em> 2017</a>). This makes it difficult to predict if indels at the beginning of the first annotated exon will affect all protein isoforms in all conditions. When sufficient information is available, especially on alternative transcripts and protein domain structures, sgRNA target sites can be chosen to maximize the chance of a complete knock-out as a result of an indel mutation at that site. Additionally, one may disrupt the gene more dramatically by removing a larger gene fragment using a dual sgRNA approach. The use of CRISPR/Cas9 for gene deletion has been pioneered in mammalian systems (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2014</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zhou <em>et al.</em> 2014</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ran <em>et al.</em> 2013</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Canver <em>et al.</em> 2014</a>). In Arabidopsis, a dual sgRNA approach for gene deletion was reported by <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zhao <em>et al.</em> (2016)</a> and <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Ordon <em>et al.</em> (2017)</a>. In Zhao <em>et al.</em> homozygous deletion mutants were obtained for the <em>AtMIR827a</em> and <em>AtMIR169a</em> loci in the T2 or T3 generation, respectively. The size of the deletion and efficiency seem to correlate inversely in mammalian cells (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Canver <em>et al.</em> 2014</a>) and plants (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Ordon <em>et al.</em> 2017</a>). Similarly, when attempting to cut out a 1953 kb fragment in <em>GRXS17</em>, it failed to be inherited, while clearly being present in T1 somatic cells. In contrast, 247 bp and 459 bp fragment deletions were easily obtainable for <em>WRKY20</em> and <em>VQ33</em> respectively. Therefore, while deleting whole genes might be tempting, it is more practical targeting genes with two sgRNAs in the 5′ coding sequence. This has the additional advantage, that when one sgRNA has a low efficiency, the construct will still yield potential knock-out mutations at the other sgRNA site. It has been proposed from work in tomato protoplasts that in most cases when a single sgRNA is used, NHEJ results in perfect repair and therefore using two sgRNAs could be more efficient to obtain mutants (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Čermák <em>et al.</em> 2017</a>). Finally, the double-sgRNA approach has an advantage of easy visual genotyping of mutants based on amplicon lengths.</p></div><div id="sec-21" class="subsection"><h3>New alleles for GRXS17</h3><p id="p-42"><em>GRXS17</em> encodes the Arabidopsis ortholog of human <em>GRX3/PICOT</em> and yeast <em>Grx3/Grx4</em>. Although a role for GRXS17 in iron-sulfur cluster assembly is conserved in all of these organisms, plant-specific functions for GRXS17 are apparent (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Iñigo <em>et al.</em> 2016</a>, Knuesting <em>et al.</em> 2017). Interestingly, AtGRXS17, HsGRX3 and ScGrx3/4 differ in the number of GRX domains that are C-terminal of the TRX domain with three, two and one domain present, respectively. The new <em>grxs17-3</em> allele presented here might have residual expression of a truncated GRXS17 with only one GRX domain—similar to ScGrx3/4—and could therefore be helpful in studying plant-specific GRXS17 roles. More detailed molecular and phenotypical analysis of this allele and the other alleles generated in this study are not within the scope of this publication.</p></div><div id="sec-22" class="subsection"><h3>A role for SD-MMEJ in the Arabidopsis germ line?</h3><p id="p-43">We observed that - independent of the sgRNA and of the genomic locus – DSB repair in somatic cells predominantly results in one bp insertions and is followed by deletions of increasing size. These are typical outcomes of the cNHEJ repair pathway and confirm earlier reports (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Fauser <em>et al.</em> 2014</a>, <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Feng <em>et al.</em> 2014</a>). We also observed that for certain sgRNAs a relatively larger deletion of a particular size (7-14 bp) was more frequently observed than for other sgRNAs in somatic cells. These deletions are likely the result of alt-EJ-mediated repair that takes place in somatic cells, albeit at lower low levels. Remarkably, inherited mutations often showed hallmarks of alt-EJ. More precisely, we could show that two alleles studied here fulfilled all criteria for the model of SD-MMEJ repair. This model explains the presence of templated insertions in <em>grxs17-3</em> and <em>grxs17-4</em>. The combination of deletions with insertions (“filler DNA”) for DSB repair in plants was first reported in tobacco (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gorbunova &amp; Levy 1997</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Salomon &amp; Puchta 1998</a>). More recently, a related observation in Arabidopsis called microhomology-mediated synthesis-dependent strand annealing (MM-SDSA) was reported when studying CRISPR/Cas9-mediated DSB repair in regions with homology in somatic cells (Vu <em>et al.</em> 2017a, Vu <em>et al.</em> 2017b). However, while in MM-SDSA longer deletions and insertions were studied and reported (averaging 240 bp for deletions, Vu <em>et al.</em> 2017a), 30 bp has been proposed for SD-MMEJ as a limit of the P1 sequence from the cut site. This was rather arbitrarily chosen as it corresponds to the binding capability of TRIMERIC REPLICATION PROTEIN A (RPA) that binds and protects single stranded DNA (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Khodaverdian <em>et al.</em> 2017</a>). Moreover, the TIDE method used in our study limits detection up to 50 bp deletions. Therefore, it is unclear at the moment if MM-SDSA and the SD-MMEJ are the same or molecularly related. PolQ has been suggested to mediate SD-MMEJ and was essential for templated insertions in human cells (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Yu and McVey, 2010</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Saito <em>et al.</em> 2017</a>). The recent identification of Arabidopsis PolQ (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">van Kregten <em>et al.</em>, 2016</a>) may allow studying the role for PolQ in SD-MMEJ in the Arabidopsis germline. However, as polQ mutants are resistant to T-DNA integration (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">van Kregten <em>et al.</em>, 2016</a>), they could not be readily used in the workflow we present here to test the involvement of polQ in SD-MMEJ in Arabidopsis.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-44">This work was supported by the Research Foundation Flanders through the projects 1507013N, G005212N, G005312N and G005915N and a postdoctoral fellowship to L.P.; the Belgian Science Policy Organization for a postdoctoral fellowship to S.I. and the Agency for Innovation by Science and Technology in Flanders for a predoctoral fellowship to J.G.; the National Science Foundation for M.R.’s contribution to this work by grant # 1636397.</p><p id="p-45">We thank Carina Braeckman for Arabidopsis floral dip transformations, Karel Spruyt for photography, Wilson Ardiles Diaz for help with sequencing, Annick Bleys for help in preparing the manuscript, Holger Puchta for providing the pDE-Cas9 vector and Thomas Laux for providing <em>wrky2-1</em> seeds. We thank Lieven De Veylder, Thomas Jacobs and Ward Decaestecker for their helpful comments on the manuscript. L.P., A.B. and A.G. designed the research; L.P., R.D.C, S.I., J.G., C.W. and M.R. performed research; L.P. wrote the paper with help from all authors.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-46">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6455840">https://doi.org/10.25387/g3.6455840</a>.</p></li><li class="fn" id="fn-3"><p id="p-47"><em>Communicating editor: M. Estelle</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-01-16"><span class="received-label">Received </span>January 16, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-05-30"><span class="accepted-label">Accepted </span>May 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Pauwels <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.1" data-doi="10.1016/j.tibtech.2014.11.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baltes</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Enabling plant synthetic biology through genome engineering.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibtech.2014.11.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2014.11.008%26rft_id%253Dinfo%253Apmid%252F25496918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2014.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25496918&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bechtold</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaudeau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jolivet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maba</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The maternal chromosome set is the target of the T-DNA in the in planta transformation of Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1875</span>–<span class="cit-lpage">1887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBechtold%26rft.auinit1%253DN.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1875%26rft.epage%253D1887%26rft.atitle%253DThe%2BMaternal%2BChromosome%2BSet%2BIs%2Bthe%2BTarget%2Bof%2Bthe%2BT-DNA%2Bin%2Bthe%2Bin%2BPlanta%2BTransformation%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F10924482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE4NzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.3" data-doi="10.3390/genes7090067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashkina</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomerantz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DNA Polymerase θ: A Unique Multifunctional End-Joining Machine.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes7090067</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D7%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes7090067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes7090067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.4" data-doi="10.1093/nar/gku936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brinkman</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amendola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Steensel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Easy quantitative assessment of genome editing by sequence trace decomposition.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku936</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku936%26rft_id%253Dinfo%253Apmid%252F25300484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku936&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25300484&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.5" data-doi="10.1074/jbc.M114.564625"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canver</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dass</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yien</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of genomic deletion efficiency mediated by clustered regularly interspaced short palindromic repeats (CRISPR)/cas9 nuclease system in mammalian cells.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">21312</span>–<span class="cit-lpage">21324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M114.564625</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.564625%26rft_id%253Dinfo%253Apmid%252F24907273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvMzEvMjEzMTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.6" data-doi="10.1105/tpc.16.00922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Čermák</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtin</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gil-Humanes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Čegan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. J. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Multipurpose Toolkit to Enable Advanced Genome Engineering in Plants.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1196</span>–<span class="cit-lpage">1217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.16.00922</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00922%26rft_id%253Dinfo%253Apmid%252F28522548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI5LzYvMTE5NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.7" data-doi="10.1038/nmeth.3473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moosburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unraveling CRISPR-Cas9 genome engineering parameters via a library-on-library approach.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">826</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3473%26rft_id%253Dinfo%253Apmid%252F26167643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3473&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26167643&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.8" data-doi="10.1038/srep07581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dual sgRNA-directed gene knockout using CRISPR/Cas9 technology in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7581</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep07581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D7581%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep07581%26rft_id%253Dinfo%253Apmid%252F25531445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep07581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25531445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.9" data-doi="10.1074/jbc.M110.201707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">N.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Arabidopsis Monothiol Glutaredoxin, AtGRXS17, Is Critical for Temperature-dependent Postembryonic Growth and Development via Modulating Auxin Response.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">20398</span>–<span class="cit-lpage">20406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M110.201707</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.201707%26rft_id%253Dinfo%253Apmid%252F21515673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMjMvMjAzOTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.10" data-doi="10.1101/gr.162339.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kweon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Analysis of off-target effects of CRISPR/Cas-derived RNA-guided endonucleases and nickases.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.162339.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.162339.113%26rft_id%253Dinfo%253Apmid%252F24253446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI0LzEvMTMyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.11" data-doi="10.1046/j.1365-313x.1998.00343.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clough</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bent</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">743</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-313x.1998.00343.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DClough%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D735%26rft.epage%253D743%26rft.atitle%253DFloral%2Bdip%253A%2Ba%2Bsimplified%2Bmethod%2Bfor%2BAgrobacterium-mediated%2Btransformation%2Bof%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313x.1998.00343.x%26rft_id%253Dinfo%253Apmid%252F10069079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313x.1998.00343.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10069079&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078001900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.12" data-doi="10.1016/j.bbamcr.2014.09.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Couturier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybyla-Toscano</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roret</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didierjean</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouhier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The roles of glutaredoxins ligating Fe–S clusters: Sensing, transfer or repair functions?</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1853</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bbamcr.2014.09.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1853%26rft.spage%253D1513%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbamcr.2014.09.018%26rft_id%253Dinfo%253Apmid%252F25264274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbamcr.2014.09.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25264274&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.13" data-doi="10.3389/fpls.2017.01171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denbow</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nimchuk</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gateway-Compatible CRISPR-Cas9 Vectors and a Rapid Detection by High-Resolution Melting Curve Analysis.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1171%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-2" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.14" data-doi="10.1038/nbt.3026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doench</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartenian</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tothova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rational design of highly active sgRNAs for CRISPR-Cas9–mediated gene inactivation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1262</span>–<span class="cit-lpage">1267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D1262%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3026%26rft_id%253Dinfo%253Apmid%252F25184501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25184501&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.15" data-doi="10.1038/s41598-018-22667-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papareddy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez-Marcos</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Highly efficient heritable targeted deletions of gene clusters and non-coding regulatory regions in Arabidopsis using CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-22667-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D8%26rft.spage%253D4443%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-22667-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-22667-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.16" data-doi="10.1093/nar/19.6.1349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnstone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A simple and rapid method for the preparation of plant genomic DNA for PCR analysis.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1349</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/19.6.1349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdwards%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D1349%26rft.epage%253D1349%26rft.atitle%253DA%2Bsimple%2Band%2Brapid%2Bmethod%2Bfor%2Bthe%2Bpreparation%2Bof%2Bplant%2Bgenomic%2BDNA%2Bfor%2BPCR%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.6.1349%26rft_id%253Dinfo%253Apmid%252F2030957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/19.6.1349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2030957&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FE01300035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.17" data-doi="10.1007/s00299-016-2000-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eid</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahfouz</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High efficiency of targeted mutagenesis in arabidopsis via meiotic promoter-driven expression of Cas9 endonuclease.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1555</span>–<span class="cit-lpage">1558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00299-016-2000-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D35%26rft.spage%253D1555%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-016-2000-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-016-2000-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.18" data-doi="10.1261/rna.044933.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fatscher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehm</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiche</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gehring</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The interaction of cytoplasmic poly(A)-binding protein with eukaryotic initiation factor 4G suppresses nonsense-mediated mRNA decay.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1579</span>–<span class="cit-lpage">1592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.044933.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.044933.114%26rft_id%253Dinfo%253Apmid%252F25147240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8xNTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.19" data-doi="10.1111/tpj.12554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fauser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiml</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Both CRISPR/Cas-based nucleases and nickases can be used efficiently for genome engineering in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12554</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D79%26rft.spage%253D348%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12554%26rft_id%253Dinfo%253Apmid%252F24836556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836556&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339439200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.20" data-doi="10.1073/pnas.1400822111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multigeneration analysis reveals the inheritance, specificity, and patterns of CRISPR/Cas-induced gene modifications in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">4632</span>–<span class="cit-lpage">4637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1400822111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1400822111%26rft_id%253Dinfo%253Apmid%252F24550464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzEyLzQ2MzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.21" data-doi="10.1038/cr.2013.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficient genome editing in plants using a CRISPR/Cas system.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/cr.2013.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D23%26rft.spage%253D1229%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.114%26rft_id%253Dinfo%253Apmid%252F23958582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23958582&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325278000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.22" data-doi="10.1105/tpc.110.080788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Calvo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Barbero</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chico</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gimenez-Ibanez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Arabidopsis bHLH transcription factors MYC3 and MYC4 are targets of JAZ repressors and act additively with MYC2 in the activation of jasmonate responses.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">715</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.110.080788</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.080788%26rft_id%253Dinfo%253Apmid%252F21335373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvNzAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.23" data-doi="10.1038/nbt.2808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sander</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cascio</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improving CRISPR-Cas nuclease specificity using truncated guide RNAs.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2808%26rft_id%253Dinfo%253Apmid%252F24463574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24463574&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.24" data-doi="10.1073/pnas.1500365112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estelle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Auxin binding protein 1 (ABP1) is not required for either auxin signaling or Arabidopsis development.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">2275</span>–<span class="cit-lpage">2280</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1500365112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1500365112%26rft_id%253Dinfo%253Apmid%252F25646447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzcvMjI3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.25" data-doi="10.1093/nar/25.22.4650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorbunova</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Non-homologous DNA end joining in plant cells is associated with deletions and filler DNA insertions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4650</span>–<span class="cit-lpage">4657</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.22.4650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.22.4650%26rft_id%253Dinfo%253Apmid%252F9358178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.22.4650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9358178&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YJ40500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.26" data-doi="10.1111/tpj.13470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guseman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dardick</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">DRO1 influences root system architecture in Arabidopsis and Prunus species.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13470</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D1093%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.27" data-doi="10.1186/s13059-016-1012-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haeussler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönig</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eschstruth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mianné</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1012-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1012-2%26rft_id%253Dinfo%253Apmid%252F27380939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1012-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27380939&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.28" data-doi="10.1186/1746-4811-2-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aspinall</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A rapid and robust method of identifying transformed Arabidopsis thaliana seedlings following floral dip transformation.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1746-4811-2-19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BMethods%26rft.aulast%253DHarrison%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D2%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DA%2Brapid%2Band%2Brobust%2Bmethod%2Bof%2Bidentifying%2Btransformed%2BArabidopsis%2Bthaliana%2Bseedlings%2Bfollowing%2Bfloral%2Bdip%2Btransformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-2-19%26rft_id%253Dinfo%253Apmid%252F17087829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-2-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17087829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.29" data-doi="10.1104/pp.16.00261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iñigo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gall</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Termathe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Glutaredoxin GRXS17 Associates with the Cytosolic Iron-Sulfur Cluster Assembly Pathway.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">858</span>–<span class="cit-lpage">873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.16.00261</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.00261%26rft_id%253Dinfo%253Apmid%252F27503603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzIvMi84NTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.30" data-doi="10.1126/science.1225829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chylinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1225829</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225829%26rft_id%253Dinfo%253Apmid%252F22745249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ni84MTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.31" data-doi="10.1104/pp.15.00788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The VQ Motif-Containing Protein Family of Plant-Specific Transcriptional Regulators.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">378</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.00788</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00788%26rft_id%253Dinfo%253Apmid%252F26220951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjkvMS8zNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.32" data-doi="10.1007/s11103-014-0266-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative assessments of CRISPR-Cas nucleases’ cleavage efficiency in planta.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11103-014-0266-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%26rft.volume%253D87%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-014-0266-x%26rft_id%253Dinfo%253Apmid%252F25403732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-014-0266-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25403732&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.33" data-doi="10.1007/BF00014550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawalleck</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somssich</span>  <span class="cit-name-given-names">I. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldbrügge</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahlbrock</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisshaar</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Polyubiquitin gene expression and structural properties of the ubi4–2 gene in Petroselinum crispum.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00014550</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DKawalleck%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D673%26rft.epage%253D684%26rft.atitle%253DPolyubiquitin%2Bgene%2Bexpression%2Band%2Bstructural%2Bproperties%2Bof%2Bthe%2Bubi4-2%2Bgene%2Bin%2BPetroselinum%2Bcrispum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00014550%26rft_id%253Dinfo%253Apmid%252F8383552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00014550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8383552&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KR42700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.34" data-doi="10.1101/157925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Questioning unexpected CRISPR off-target mutations in vivo.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/157925</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F157925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/157925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.35" data-doi="10.1093/nar/gkx1056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khodaverdian</span>  <span class="cit-name-given-names">V. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanscom</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Secondary structure forming sequences drive SD-MMEJ repair of DNA double-strand breaks.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">12848</span>–<span class="cit-lpage">12861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253D12848%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.36" data-doi="10.1007/s10577-014-9415-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">DNA recombination in somatic plant cells: mechanisms and evolutionary consequences.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-014-9415-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2BRes.%26rft.volume%253D22%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-014-9415-y%26rft_id%253Dinfo%253Apmid%252F24788060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-014-9415-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24788060&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338283900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2603.37" data-doi="10.1104/pp.15.00049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knuesting</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riondet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maria</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruse</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bécuwe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Arabidopsis Glutaredoxin S17 and Its Partner, the Nuclear Factor Y Subunit C11/Negative Cofactor 2α, Contribute to Maintenance of the Shoot Apical Meristem under Long-Day Photoperiod.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1643</span>–<span class="cit-lpage">1658</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.00049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00049%26rft_id%253Dinfo%253Apmid%252F25699589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY3LzQvMTY0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.38" data-doi="10.1111/tpj.13782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeBlanc</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatpar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Increased efficiency of targeted mutagenesis by CRISPR/Cas9 in plants using heat stress.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13782</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D93%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.39" data-doi="10.1093/mp/ssu044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR-P: A web tool for synthetic single-guide RNA design of CRISPR-system in plants.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/ssu044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssu044%26rft_id%253Dinfo%253Apmid%252F24719468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssu044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24719468&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342930000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.40" data-doi="10.1016/B978-0-12-801185-0.00022-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cas9-based genome editing in Arabidopsis and tobacco.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-801185-0.00022-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D546%26rft.spage%253D459%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-801185-0.00022-2%26rft_id%253Dinfo%253Apmid%252F25398353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-801185-0.00022-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398353&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.41" data-doi="10.1111/tpj.13324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Arabidopsis UDP-glycosyltransferases UGT79B2 and UGT79B3, contribute to cold, salt and drought stress tolerance via modulating anthocyanin accumulation.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.42" data-doi="10.1242/dev.152892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">K.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Rybel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Mourik</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weijers</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Regulation of intercellular TARGET OF MONOPTEROS 7 protein transport in the Arabidopsis root.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">dev152892</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.152892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.152892%26rft_id%253Dinfo%253Apmid%252F29358212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxNToiMTQ1LzIvZGV2MTUyODkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.43" data-doi="10.1016/j.molp.2015.04.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Robust CRISPR/Cas9 System for Convenient, High-Efficiency Multiplex Genome Editing in Monocot and Dicot Plants.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1274</span>–<span class="cit-lpage">1284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2015.04.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D8%26rft.spage%253D1274%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2015.04.007%26rft_id%253Dinfo%253Apmid%252F25917172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2015.04.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25917172&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.44" data-doi="10.1093/mp/sst121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Application of the CRISPR-Cas system for efficient genome engineering in plants.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2008</span>–<span class="cit-lpage">2011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/sst121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fsst121%26rft_id%253Dinfo%253Apmid%252F23963532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/sst121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23963532&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327541200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.45" data-doi="10.1111/pbi.12468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development of germ-line-specific CRISPR-Cas9 systems to improve the production of heritable gene modifications in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D519%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12468%26rft_id%253Dinfo%253Apmid%252F26360626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26360626&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-2" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.46" data-doi="10.1038/nmeth.3543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Mateos</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vejnar</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaudoin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mis</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D982%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3543%26rft_id%253Dinfo%253Apmid%252F26322839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26322839&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.47" data-doi="10.1111/pbi.12671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morineau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellec</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gissot</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelemen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Selective gene dosage by CRISPR-Cas9 genome editing in hexaploid Camelina sativa.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12671</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D729%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.48" data-doi="10.1093/pcp/pcw122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagels Durand</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iñigo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iniesto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Clercq</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Arabidopsis Iron–Sulfur Protein GRXS17 is a Target of the Ubiquitin E3 Ligases RGLG3 and RGLG4.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1813</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/pcp/pcw122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcw122%26rft_id%253Dinfo%253Apmid%252F27497447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcw122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27497447&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.49" data-doi="10.1093/nar/gku1382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Two novel NAC transcription factors regulate gene expression and flowering time by associating with the histone demethylase JMJ14.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1469</span>–<span class="cit-lpage">1484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1382%26rft_id%253Dinfo%253Apmid%252F25578968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25578968&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.50" data-doi="10.1111/tpj.13319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ordon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gantner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemna</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalgun</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reschke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Generation of chromosomal deletions in dicotyledonous plants employing a user-friendly genome editing toolkit.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.51" data-doi="10.1038/srep26685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osakabe</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugano</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ueta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishihara</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimization of CRISPR/Cas9 genome editing to modify abiotic stress responses in plants.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">26685</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep26685</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D26685%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep26685%26rft_id%253Dinfo%253Apmid%252F27226176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep26685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27226176&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.52" data-doi="10.1371/journal.pone.0162169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haak</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">P. J. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-Wide Assessment of Efficiency and Specificity in CRISPR/Cas9 Mediated Multiple Site Targeting in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0162169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0162169</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0162169%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0162169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0162169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.53" data-doi="10.1016/j.cell.2016.05.053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Popp</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maquat</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Leveraging rules of nonsense-mediated mRNA decay for genome engineering and personalized medicine.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1322</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.05.053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D165%26rft.spage%253D1319%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.053%26rft_id%253Dinfo%253Apmid%252F27259145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27259145&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.54" data-doi="10.1038/nprot.2013.143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering using the CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2013.143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D2281%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.143%26rft_id%253Dinfo%253Apmid%252F24157548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24157548&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.55" data-doi="10.1038/ncomms15235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iñigo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Calvo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heyndrickx</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhondt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The transcriptional repressor complex FRS7–FRS12 regulates flowering time and growth in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">15235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms15235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D15235%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15235%26rft_id%253Dinfo%253Apmid%252F28492275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28492275&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.56" data-doi="10.1038/ncomms16112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adachi</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dual loss of human POLQ and LIG4 abolishes random integration.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">16112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms16112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D16112%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms16112%26rft_id%253Dinfo%253Apmid%252F28695890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms16112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28695890&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.57" data-doi="10.1093/emboj/17.20.6086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salomon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Capture of genomic and T-DNA sequences during double-strand break repair in somatic plant cells.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">6086</span>–<span class="cit-lpage">6095</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/emboj/17.20.6086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DSalomon%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D20%26rft.spage%253D6086%26rft.epage%253D6095%26rft.atitle%253DCapture%2Bof%2Bgenomic%2Band%2BT-DNA%2Bsequences%2Bduring%2Bdouble-strand%2Bbreak%2Brepair%2Bin%2Bsomatic%2Bplant%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.20.6086%26rft_id%253Dinfo%253Apmid%252F9774352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTcvMjAvNjA4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.58" data-doi="10.1104/pp.113.220129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasaki-Sekimoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jikumaru</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Basic helix-loop-helix transcription factors JASMONATE-ASSOCIATED MYC2-LIKE1 (JAM1), JAM2, and JAM3 are negative regulators of jasmonate responses in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.113.220129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.220129%26rft_id%253Dinfo%253Apmid%252F23852442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjMvMS8yOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.59" data-doi="10.1534/g3.116.035204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strunks</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemann</span>  <span class="cit-name-given-names">B. J. P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooykaas</span>  <span class="cit-name-given-names">P. J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Pater</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR/Cas9-Induced Double-Strand Break Repair in Arabidopsis Nonhomologous End-Joining Mutants.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.035204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035204%26rft_id%253Dinfo%253Apmid%252F27866150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMS8xOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.60" data-doi="10.1093/pcp/pcw191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsutsui</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higashiyama</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">pKAMA-ITACHI Vectors for Highly Efficient CRISPR/Cas9-Mediated Gene Knockout in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/pcp/pcw191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcw191%26rft_id%253Dinfo%253Apmid%252F27856772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcw191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27856772&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.61" data-doi="10.1016/j.devcel.2011.01.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ueda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laux</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptional Activation of Arabidopsis Axis Patterning Genes WOX8/9 Links Zygote Polarity to Embryo Development.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2011.01.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DUeda%26rft.auinit1%253DM.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D264%26rft.epage%253D270%26rft.atitle%253DTranscriptional%2Bactivation%2Bof%2BArabidopsis%2Baxis%2Bpatterning%2Bgenes%2BWOX8%252F9%2Blinks%2Bzygote%2Bpolarity%2Bto%2Bembryo%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2011.01.009%26rft_id%253Dinfo%253Apmid%252F21316593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2011.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21316593&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287646800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.62" data-doi="10.1016/j.cell.2017.10.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ushijima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotoh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamori</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Light Controls Protein Localization through Phytochrome-Mediated Alternative Promoter Selection.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1316</span>–<span class="cit-lpage">1325.e12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2017.10.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D171%26rft.spage%253D1316%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.10.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.10.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.63" data-doi="10.1038/nplants.2016.164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Kregten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Pater</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romeijn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Schendel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooykaas</span>  <span class="cit-name-given-names">P. J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">T-DNA integration in plants results from polymerase-θ-mediated DNA repair.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16164%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.164%26rft_id%253Dinfo%253Apmid%252F27797358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27797358&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.64" data-doi="10.1186/s13007-016-0101-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez-Vilar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernabé-Orts</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Del-Carmen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziarsolo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanca</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A modular toolbox for gRNA-Cas9 genome engineering in plants based on the GoldenBraid standard.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-016-0101-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D12%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-016-0101-2%26rft_id%253Dinfo%253Apmid%252F26839579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-016-0101-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26839579&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2603.65" data-doi="10.1111/tpj.13634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vu</span>  <span class="cit-name-given-names">G. T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiss</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017a</span> <span class="cit-article-title">Endogenous sequence patterns predispose the repair modes of CRISPR/Cas9-induced DNA double-stranded breaks in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D92%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2603.66" data-doi="10.1111/nph.14490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vu</span>  <span class="cit-name-given-names">G. T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiss</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2017b</span> <span class="cit-article-title">Deletion-bias in DNA double-strand break repair differentially contributes to plant genome shrinkage.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">1712</span>–<span class="cit-lpage">1721</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14490</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D214%26rft.spage%253D1712%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.67" data-doi="10.1186/s13059-015-0715-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">H.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Egg cell-specific promoter-controlled CRISPR/Cas9 efficiently generates homozygous mutants for multiple target genes in Arabidopsis in a single generation.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0715-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D144%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0715-0%26rft_id%253Dinfo%253Apmid%252F26193878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0715-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26193878&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.68" data-doi="10.1038/nature20166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">X.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nomura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aung</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velásquez</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bacteria establish an aqueous living space in plants crucial for virulence.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">539</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature20166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D539%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature20166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.69" data-doi="10.1016/j.molp.2015.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Efficiency Genome Editing in Arabidopsis Using YAO Promoter-Driven CRISPR/Cas9 System.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1820</span>–<span class="cit-lpage">1823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2015.10.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D8%26rft.spage%253D1820%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2015.10.004%26rft_id%253Dinfo%253Apmid%252F26524930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2015.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26524930&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.70" data-doi="10.1093/nar/gkq379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synthesis-dependent microhomology-mediated end joining accounts for multiple types of repair junctions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">5706</span>–<span class="cit-lpage">5717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq379%26rft_id%253Dinfo%253Apmid%252F20460465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20460465&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282173300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.71" data-doi="10.1007/s00299-015-1900-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botella</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A multiplex CRISPR/Cas9 platform for fast and efficient editing of multiple genes in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00299-015-1900-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D35%26rft.spage%253D1519%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-015-1900-z%26rft_id%253Dinfo%253Apmid%252F26661595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-015-1900-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26661595&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.72" data-doi="10.1038/srep23890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An alternative strategy for targeted gene replacement in plants using a dual-sgRNA/Cas9 design.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep23890</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D23890%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep23890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep23890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2603.73" data-doi="10.1111/febs.12735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dual sgRNAs facilitate CRISPR/Cas9-mediated mouse genome targeting.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/febs.12735</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJ.%26rft.volume%253D281%26rft.spage%253D1717%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ffebs.12735%26rft_id%253Dinfo%253Apmid%252F24494965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/febs.12735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24494965&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2603.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2595" title="Involvement of YAP-1, the Homolog of Yes-Associated Protein, in the Wnt-Mediated Neuronal Polarization in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2617" title="The Doubletime Homolog KIN-20 Mainly Regulates let-7 Independently of Its Effects on the Period Homolog LIN-42 in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2603" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OwglLDmv9WGUES7xbbsDQuOAvuLVg8lLQQ3UjDnOMMk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2603" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2603" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2603" class="active" data-icon-position="" data-hide-link-title="0">A Dual sgRNA Approach for Functional Genomics in Arabidopsis thaliana</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490460" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mHNQAqrKO3uy1HQ_t5Z9qpyQXu6QaJNXerlW8u-eRqo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2603.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2603" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RAx0RsWsbqotXfM5ajQR9w4Hlkjm1n1yao7e9Diaad0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2603" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2603.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490460" id="node490460--2" data-pisa="ggg;8/8/2603" data-pisa-master="ggg;g3.118.200046" data-apath="/ggg/8/8/2603.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dual sgRNA Approach for Functional Genomics in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0221-9052" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laurens Pauwels</span>, <span class="highwire-citation-author" data-delta="1">Rebecca De Clercq</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7940-2699" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonas Goossens</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5820-7109" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sabrina Iñigo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1292-6313" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clara Williams</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1682-7275" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mily Ron</span>, <span class="highwire-citation-author" data-delta="6">Anne Britt</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1599-551X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alain Goossens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2603-2615;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200046 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490460--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laurens Pauwels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurens%2BPauwels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pauwels%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurens%2BPauwels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0221-9052" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laurens Pauwels</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lapau@psb.ugent.be" class="" data-icon-position="" data-hide-link-title="0">lapau@psb.ugent.be</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rebecca De Clercq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BDe%2BClercq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Clercq%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BDe%2BClercq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonas Goossens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonas%2BGoossens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goossens%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonas%2BGoossens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7940-2699" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonas Goossens</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sabrina Iñigo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sabrina%2BI%C3%B1igo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=I%C3%B1igo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASabrina%2BI%25C3%25B1igo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5820-7109" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sabrina Iñigo</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clara Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clara%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClara%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1292-6313" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clara Williams</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mily Ron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UC Davis, Department of Plant Biology, Davis, CA 95616, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mily%2BRon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMily%2BRon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1682-7275" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mily Ron</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anne Britt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UC Davis, Department of Plant Biology, Davis, CA 95616, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BBritt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Britt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BBritt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Alain Goossens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BGoossens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goossens%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BGoossens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1599-551X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alain Goossens</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2603" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2603&amp;title=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2603&amp;title=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2603&amp;title=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2603&amp;text=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2603&amp;title=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2603&amp;t=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2603&amp;title=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2603&amp;title=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490460/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490460" id="node490460--3" data-pisa="ggg;8/8/2603" data-pisa-master="ggg;g3.118.200046" data-apath="/ggg/8/8/2603.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dual sgRNA Approach for Functional Genomics in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0221-9052" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laurens Pauwels</span>, <span class="highwire-citation-author" data-delta="1">Rebecca De Clercq</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7940-2699" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonas Goossens</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5820-7109" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sabrina Iñigo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-1292-6313" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clara Williams</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1682-7275" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mily Ron</span>, <span class="highwire-citation-author" data-delta="6">Anne Britt</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1599-551X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alain Goossens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2603-2615;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200046 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490460--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Laurens Pauwels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurens%2BPauwels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pauwels%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaurens%2BPauwels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0221-9052" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laurens Pauwels</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lapau@psb.ugent.be" class="" data-icon-position="" data-hide-link-title="0">lapau@psb.ugent.be</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rebecca De Clercq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BDe%2BClercq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Clercq%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BDe%2BClercq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonas Goossens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonas%2BGoossens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goossens%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonas%2BGoossens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7940-2699" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonas Goossens</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sabrina Iñigo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sabrina%2BI%C3%B1igo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=I%C3%B1igo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASabrina%2BI%25C3%25B1igo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5820-7109" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sabrina Iñigo</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clara Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clara%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClara%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1292-6313" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clara Williams</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mily Ron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UC Davis, Department of Plant Biology, Davis, CA 95616, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mily%2BRon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMily%2BRon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1682-7275" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mily Ron</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anne Britt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UC Davis, Department of Plant Biology, Davis, CA 95616, US</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BBritt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Britt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BBritt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Alain Goossens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ghent University, Department of Plant Biotechnology and Bioinformatics, 9052 Ghent, Belgium</div><div class='author-affiliation'><span class='nlm-sup'>†</span>VIB Center for Plant Systems Biology, 9052 Ghent, Belgium</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BGoossens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goossens%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BGoossens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1599-551X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alain Goossens</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2603&amp;rft.epage=2615&amp;rft.atitle=A%20Dual%20sgRNA%20Approach%20for%20Functional%20Genomics%20in%20Arabidopsis%20thaliana&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Pauwels%2C+Laurens&amp;rft.au=De+Clercq%2C+Rebecca&amp;rft.au=Goossens%2C+Jonas&amp;rft.au=I%C3%B1igo%2C+Sabrina&amp;rft.au=Williams%2C+Clara&amp;rft.au=Ron%2C+Mily&amp;rft.au=Britt%2C+Anne&amp;rft.au=Goossens%2C+Alain"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490460/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490460/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490460/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490460/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490460/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490460/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490460/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490460/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490460/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490460/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490460/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490460/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2603" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2603.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2603.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2603.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2603.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
