<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2841" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2841.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2841.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2841.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/8/2841/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2841.atom" />
<meta name="HW.pisa" content="ggg;8/8/2841" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in Phaseolus vulgaris" />
<meta name="DC.Identifier" content="10.1534/g3.118.200493" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Resende et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6533933" />
<meta name="DC.Description" content="The availability of high-density molecular markers in common bean has allowed to explore the genetic basis of important complex agronomic traits with increased resolution. Genome-Wide Association Studies (GWAS) and Regional Heritability Mapping (RHM) are two analytical approaches for the detection of genetic variants. We carried out GWAS and RHM for plant architecture, lodging and productivity across two important growing environments in Brazil in a germplasm of 188 common bean varieties using DArTseq genotyping strategies. The coefficient of determination of G × E interaction (c2 int ) was equal to 17, 21 and 41%, respectively for the traits architecture, lodging, and productivity. Trait heritabilities were estimated at 0.81 (architecture), 0.79 (lodging) and 0.43 (productivity), and total genomic heritability accounted for large proportions (72% to ≈100%) of trait heritability. At the same probability threshold, three marker–trait associations were detected using GWAS, while RHM detected eight QTL encompassing 145 markers along five chromosomes. The proportion of genomic heritability explained by RHM was considerably higher (35.48 to 58.02) than that explained by GWAS (28.39 to 30.37). In general, RHM accounted for larger fractions of the additive genetic variance being captured by markers effects inside the defined regions. Nevertheless, a considerable proportion of the heritability is still missing (∼42% to ∼64%), probably due to LD between markers and genes and/or rare allele variants not sampled. RHM in autogamous species had the potential to identify larger-effect QTL combining allelic variants that could be effectively incorporated into whole-genome prediction models and tracked through breeding generations using marker-assisted selection." />
<meta name="DC.Contributor" content="Rafael T. Resende" />
<meta name="DC.Contributor" content="Marcos Deon V. de Resende" />
<meta name="DC.Contributor" content="Camila F. Azevedo" />
<meta name="DC.Contributor" content="Fabyano Fonseca e Silva" />
<meta name="DC.Contributor" content="Leonardo C. Melo" />
<meta name="DC.Contributor" content="Helton S. Pereira" />
<meta name="DC.Contributor" content="Thiago Lívio P. O. Souza" />
<meta name="DC.Contributor" content="Paula Arielle M. R. Valdisser" />
<meta name="DC.Contributor" content="Claudio Brondani" />
<meta name="DC.Contributor" content="Rosana Pereira Vianello" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in Phaseolus vulgaris" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The availability of high-density molecular markers in common bean has allowed to explore the genetic basis of important complex agronomic traits with increased resolution. Genome-Wide Association Studies (GWAS) and Regional Heritability Mapping (RHM) are two analytical approaches for the detection of genetic variants. We carried out GWAS and RHM for plant architecture, lodging and productivity across two important growing environments in Brazil in a germplasm of 188 common bean varieties using DArTseq genotyping strategies. The coefficient of determination of G × E interaction (c&lt;sup&gt;2&lt;/sup&gt;&lt;i&gt;&lt;sub&gt;int&lt;/sub&gt;&lt;/i&gt;) was equal to 17, 21 and 41%, respectively for the traits architecture, lodging, and productivity. Trait heritabilities were estimated at 0.81 (architecture), 0.79 (lodging) and 0.43 (productivity), and total genomic heritability accounted for large proportions (72% to ≈100%) of trait heritability. At the same probability threshold, three marker–trait associations were detected using GWAS, while RHM detected eight QTL encompassing 145 markers along five chromosomes. The proportion of genomic heritability explained by RHM was considerably higher (35.48 to 58.02) than that explained by GWAS (28.39 to 30.37). In general, RHM accounted for larger fractions of the additive genetic variance being captured by markers effects inside the defined regions. Nevertheless, a considerable proportion of the heritability is still missing (∼42% to ∼64%), probably due to LD between markers and genes and/or rare allele variants not sampled. RHM in autogamous species had the potential to identify larger-effect QTL combining allelic variants that could be effectively incorporated into whole-genome prediction models and tracked through breeding generations using marker-assisted selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2841" />
<meta name="citation_id" content="8/8/2841" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2841" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2841.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2841.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2841.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200493" />
<meta name="citation_pmid" content="29967054" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2841" />
<meta name="citation_lastpage" content="2854" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rafael T. Resende" />
<meta name="citation_author_institution" content="Department of Forestry" />
<meta name="citation_author" content="Marcos Deon V. de Resende" />
<meta name="citation_author_institution" content="Department of Forestry" />
<meta name="citation_author_institution" content="Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil" />
<meta name="citation_author_institution" content="EMBRAPA Florestas, Colombo, PR 83411-000, Brazil" />
<meta name="citation_author" content="Camila F. Azevedo" />
<meta name="citation_author_institution" content="Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil" />
<meta name="citation_author" content="Fabyano Fonseca e Silva" />
<meta name="citation_author_institution" content="Department of Animal Science, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil" />
<meta name="citation_author" content="Leonardo C. Melo" />
<meta name="citation_author_institution" content="Common Bean Breeding Program" />
<meta name="citation_author" content="Helton S. Pereira" />
<meta name="citation_author_institution" content="Common Bean Breeding Program" />
<meta name="citation_author" content="Thiago Lívio P. O. Souza" />
<meta name="citation_author_institution" content="Common Bean Breeding Program" />
<meta name="citation_author" content="Paula Arielle M. R. Valdisser" />
<meta name="citation_author_institution" content="Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil" />
<meta name="citation_author" content="Claudio Brondani" />
<meta name="citation_author_institution" content="Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil" />
<meta name="citation_author" content="Rosana Pereira Vianello" />
<meta name="citation_author_institution" content="Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil" />
<meta name="citation_author_email" content="rosana.vianello@embrapa.br" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0644-5652" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res.;citation_author=R. C. Alvares;citation_author=F. C. Silva;citation_author=L. C. Melo;citation_author=P. G. Melo;citation_author=H. S. Pereira;citation_title=Estimation of genetic parameters and selection of high-yielding, upright common bean lines with slow seed-coat darkening.;citation_pages=gmr15049081;citation_volume=15;citation_year=2016;citation_doi=10.4238/gmr15049081" />
<meta name="citation_reference" content="citation_journal_title=Asian J. Crop Sci.;citation_author=A. Balcha;citation_title=Genetic variation for grain yield of common bean (Phaseolus vulgaris L.) in sole and maize/bean intercropping systems.;citation_pages=158-164;citation_volume=6;citation_year=2014;citation_doi=10.3923/ajcs.2014.158.164" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=A. D. Beattie;citation_author=J. Larsen;citation_author=T. E. Michaels;citation_author=K. P. Pauls;citation_title=Mapping quantitative trait loci for a common bean (Phaseolus vulgaris L.) ideotype.;citation_pages=411-422;citation_volume=46;citation_year=2003;citation_issue=3;citation_pmid=12834057;citation_doi=10.1139/g03-015" />
<meta name="citation_reference" content="citation_title=Crop Adaptation to Climate Change;citation_author=S. E. Beebe;citation_author=J. Ramírez;citation_author=A. Jarvis;citation_author=I. M. Rao;citation_author=G. Mosquera;citation_title=Genetic improvement of common beans and the challenges of climate change;citation_year=2011;citation_doi=10.1002/9780470960929.ch25" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Biazzi;citation_author=N. Nazzicari;citation_author=L. Pecetti;citation_author=E. C. Brummer;citation_author=A. Palmonari;citation_title=Genome-wide association mapping and genomic selection for alfalfa (Medicago sativa) forage quality traits.;citation_pages=e0169234;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0169234" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. L. Bitocchi;citation_author=E. Nanni;citation_author=M. Bellucci;citation_author=A. Rossi;citation_author=P. S. Giardini;citation_title=Mesoamerican origin of the common bean (Phaseolus vulgaris L.) is revealed by sequence data.;citation_volume=109;citation_year=2012;citation_pmid=22393017;citation_doi=10.1073/pnas.1108973109" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. W. Blair;citation_author=G. Iriarte;citation_author=S. Beebe;citation_title=QTL analysis of yield traits in an advanced backcross population derived from a cultivated Andean x wild common bean (Phaseolus vulgaris L.) cross.;citation_pages=1149-1163;citation_volume=112;citation_year=2006;citation_issue=6;citation_pmid=16432734;citation_doi=10.1007/s00122-006-0217-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Soil;citation_author=W. J. Broughton;citation_author=G. Hernandez;citation_author=M. Blair;citation_author=S. Beebe;citation_author=P. Gepts;citation_title=Beans (Phaseolus spp.) – model food legumes.;citation_pages=55-128;citation_volume=252;citation_year=2003;citation_doi=10.1023/A:1024146710611" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Caballero;citation_author=A. Tenesa;citation_author=P. D. Keightley;citation_title=The nature of genetic variation for complex traits revealed by GWAS and regional heritability mapping analyses.;citation_volume=201;citation_year=2015;citation_pmid=26482794;citation_doi=10.1534/genetics.115.177220" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res.;citation_author=P. C. B. Cardoso;citation_author=C. Brondani;citation_author=I. P. P. Menezes;citation_author=P. A. M. R. Valdisser;citation_author=T. C. O. Borba;citation_title=Discrimination of common bean cultivars using multiplexed microsatellite markers.;citation_pages=1964-1978;citation_volume=13;citation_year=2014;citation_doi=10.4238/2014.March.24.1" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Chepyshko;citation_author=C.-P. Lai;citation_author=L.-M. Huang;citation_author=J.-H. Liu;citation_author=J.-F. Shaw;citation_title=Multifunctionality and diversity of GDSL esterase/lipase gene family in rice (Oryza sativa L. japonica) genome: new insights from bioinformatics analysis.;citation_pages=309-309;citation_volume=13;citation_year=2012;citation_pmid=22793791;citation_doi=10.1186/1471-2164-13-309" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=K. A. Cichy;citation_author=T. G. Porch;citation_author=J. S. Beaver;citation_author=P. Cregan;citation_author=D. Fourie;citation_title=A Phaseolus vulgaris diversity panel for Andean bean improvement.;citation_pages=2149-2160;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.09.0653" />
<meta name="citation_reference" content="citation_author=D. Clifford;citation_author=P. McCullagh;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Progress in biophysics and molecular biology;citation_journal_abbrev=Prog Biophys Mol Biol;citation_author=K. L. Craig;citation_author=M. Tyers;citation_title=The F-box: a new motif for ubiquitin dependent proteolysis in cell cycle regulation and signal transduction.;citation_pages=299-328;citation_volume=72;citation_year=1999;citation_issue=3;citation_pmid=10581972;citation_doi=10.1016/S0079-6107(99)00010-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=V. M. V. Cruz;citation_author=A. Kilian;citation_author=D. A. Dierig;citation_title=Development of DArT marker platforms and genetic diversity assessment of the U.S. collection of the new oilseed crop lesquerella and related species.;citation_pages=e64062;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0064062" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: what is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=N. Dharmasiri;citation_author=S. Dharmasiri;citation_author=D. Weijers;citation_author=E. Lechner;citation_author=M. Yamada;citation_title=Plant development is regulated by a family of auxin receptor F box proteins.;citation_pages=109-119;citation_volume=9;citation_year=2005;citation_issue=1;citation_pmid=15992545;citation_doi=10.1016/j.devcel.2005.05.014" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet. Resour.;citation_author=D. A. Earl;citation_title=STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method.;citation_volume=4;citation_year=2012;citation_doi=10.1007/s12686-011-9548-7" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. A. Eberhart;citation_author=W. A. Russell;citation_title=Stability Parameters for Comparing Varieties;citation_pages=36-40;citation_volume=6;citation_year=1966;citation_issue=1;citation_doi=10.2135/cropsci1966.0011183X000600010011x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="EMBRAPA, 2015 Technological Solutions: Copo Medidor de perdas na colheita de feijão, Available from. [https://www.embrapa.br/busca-de-produtos-processos-e-servicos/-/produto-servico/2696/copo-medidor-de-perdas-na-colheita-de-feijao]." />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=GM Crops;citation_author=J. C. Faria;citation_author=G. E. Carneiro;citation_author=F. J. Aragão;citation_title=Gene flow from transgenic common beans expressing the bar gene.;citation_pages=94-98;citation_volume=1;citation_year=2010;citation_doi=10.4161/gmcr.1.2.11609" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. L. Ferreira;citation_author=J. E. S. Carneiro;citation_author=A. L. Teixeira;citation_author=F. F. Lanes;citation_author=P. R. Cecon;citation_title=Gene flow in common bean (Phaseolus vulgaris L.).;citation_pages=165-170;citation_volume=153;citation_year=2007;citation_doi=10.1007/s10681-006-9251-z" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. C. Gonçalves-Vidigal;citation_author=F. Mora;citation_author=T. S. Bignotto;citation_author=R. E. F. Munhoz;citation_author=L. D. Souza;citation_title=Heritability of quantitative traits in segregating common bean families using a Bayesian approach.;citation_pages=551-560;citation_volume=164;citation_year=2008;citation_doi=10.1007/s10681-008-9758-6" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=V. Hoyos-Villegas;citation_author=Q. Song;citation_author=J. D. Kelly;citation_title=Genome-wide association analysis for drought tolerance and associated traits in common bean.;citation_pages=1;citation_volume=10;citation_year=2016;citation_doi=10.3835/plantgenome2015.12.0122" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. L. Hyten;citation_author=I.-Y. Choi;citation_author=Q. Song;citation_author=R. C. Shoemaker;citation_author=R. L. Nelson;citation_title=Highly Variable Patterns of Linkage Disequilibrium in Multiple Soybean Populations.;citation_volume=175;citation_year=2007;citation_pmid=17287533;citation_doi=10.1534/genetics.106.069740" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=A. Iwata;citation_author=A. L. Tek;citation_author=M. M. S. Richard;citation_author=B. Abernathy;citation_author=A. Fonseca;citation_title=Identification and characterization of functional centromeres of the common bean.;citation_pages=47-60;citation_volume=76;citation_year=2013;citation_pmid=23795942" />
<meta name="citation_reference" content="citation_journal_title=J Amer Soc Hort.;citation_author=G. Jung;citation_author=D. P. Coyne;citation_author=P. W. Skroch;citation_author=J. Nienhuis;citation_author=E. Arnaud-Santana;citation_title=Molecular markers associated with plant architecture and resistance to common blight, web blight, and rust in common beans.;citation_pages=794-803;citation_volume=121;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=K. Kamfwa;citation_author=K. A. Cichy;citation_author=J. D. Kelly;citation_title=Genome-wide association study of agronomic traits in common bean.;citation_pages=1-12;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Kashiwagi;citation_author=E. Togawa;citation_author=N. Hirotsu;citation_author=K. Ishimaru;citation_title=Improvement of lodging resistance with QTLs for stem diameter in rice (Oryza sativa L.).;citation_pages=749-757;citation_volume=117;citation_year=2008;citation_issue=5;citation_pmid=18575836;citation_doi=10.1007/s00122-008-0816-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. M. Kolkman;citation_author=J. D. Kelly;citation_title=QTL Conferring Resistance and Avoidance to White Mold in Common Bean;citation_pages=539-548;citation_volume=43;citation_year=2003;citation_issue=2;citation_doi=10.2135/cropsci2003.0539" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. J. Leamy;citation_author=H. Zhang;citation_author=C. Li;citation_author=C. Y. Chen;citation_author=B.-H. Song;citation_title=A genome-wide association study of seed composition traits in wild soybean (Glycine soja).;citation_pages=18;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-016-3397-4" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z.-K. Li;citation_author=S.-B. Yu;citation_author=H. R. Lafitte;citation_author=N. Huang;citation_author=B. Courtois;citation_title=QTL x environment interactions in rice. I. heading date and plant height.;citation_pages=141-153;citation_volume=108;citation_year=2003;citation_issue=1;citation_pmid=12961067;citation_doi=10.1007/s00122-003-1401-2" />
<meta name="citation_reference" content="citation_journal_title=Front. Physiol.;citation_author=M. Malosetti;citation_author=J. M. Ribaut;citation_author=F. A. van Eeuwijk;citation_title=The statistical analysis of multi-environment data: Modeling genotype-by-environment interaction and its genetic basis.;citation_pages=44;citation_volume=4;citation_year=2013;citation_pmid=23487515;citation_doi=10.3389/fphys.2013.00044" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. Mangin;citation_author=A. Siberchicot;citation_author=S. Nicolas;citation_author=A. Doligez;citation_author=P. This;citation_title=Novel measures of linkage disequilibrium that correct the bias due to population structure and relatedness.;citation_pages=285-291;citation_volume=108;citation_year=2012;citation_issue=3;citation_pmid=21878986;citation_doi=10.1038/hdy.2011.73" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=O. Matika;citation_author=V. Riggio;citation_author=M. Anselme-Moizan;citation_author=A. S. Law;citation_author=R. Pong-Wong;citation_title=Genome-wide association reveals QTL for growth, bone and in vivo carcass traits as assessed by computed tomography in Scottish Blackface lambs.;citation_pages=11;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0191-3" />
<meta name="citation_reference" content="citation_title=Procedimentos para condução de experimentos de Valor de Cultivo e Uso em feijoeiro comum;citation_title=Instalação, avaliação e procedimentos experimentais;citation_author=L. C. Melo;citation_author=L. C. de Faria;citation_author=M. J. Del Peloso;citation_author=A. Sartorato;citation_author=C. A. Rava;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=S. M. Moghaddam;citation_author=S. Mamidi;citation_author=J. M. Osorno;citation_author=R. Lee;citation_author=M. Brick;citation_title=Genome-wide association study identifies candidate loci underlying agronomic traits in a Middle American diversity panel of common bean.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0012" />
<meta name="citation_reference" content="citation_author=W. M. Muir;citation_author=D. A. Stick;citation_pages=403-410;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=G. Mukeshimana;citation_author=L. Butare;citation_author=P. B. Cregan;citation_author=M. W. Blair;citation_author=J. D. Kelly;citation_title=Quantitative trait loci associated with drought tolerance in common bean.;citation_pages=923-938;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.06.0427" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Nagamine;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=V. Vitart;citation_author=C. Hayward;citation_title=Localising Loci underlying Complex Trait Variation Using Regional Genomic Relationship Mapping.;citation_pages=e46501-e46501;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23077511;citation_doi=10.1371/journal.pone.0046501" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=T. Nakedde;citation_author=F. J. Ibarra-Perez;citation_author=C. Mukankusi;citation_author=J. G. Waines;citation_author=J. D. Kelly;citation_title=Mapping of QTL associated with Fusarium root rot resistance and root architecture traits in black beans.;citation_pages=51-63;citation_volume=212;citation_year=2016;citation_doi=10.1007/s10681-016-1755-6" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Nienhuis;citation_author=S. P. Singh;citation_title=Combining Ability Analyses and Relationships Among Yield, Yield Components, and Architectural Traits in Dry Bean;citation_pages=21-27;citation_volume=26;citation_year=1986;citation_issue=1;citation_doi=10.2135/cropsci1986.0011183X002600010005x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=U. G. Okeke;citation_author=D. Akdemir;citation_author=I. Rabbi;citation_author=P. Kulakow;citation_author=J. L. Jannink;citation_title=Regional heritability mapping provides insights into dry matter content in african white and yellow cassava populations.;citation_pages=1-18;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.06.0050" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. M. Pastina;citation_author=M. Malosetti;citation_author=R. Gazaffi;citation_author=M. Mollinari;citation_author=G. R. A. Margarido;citation_title=A mixed model QTL analysis for sugarcane multiple-harvest-location trial data.;citation_pages=835-849;citation_volume=124;citation_year=2012;citation_issue=5;citation_pmid=22159754;citation_doi=10.1007/s00122-011-1748-8" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. M. K. C. Perseguini;citation_author=P. R. Oblessuc;citation_author=J. R. B. F. Rosa;citation_author=K. A. Gomes;citation_author=A. F. Chiorato;citation_title=Genome-wide association studies of anthracnose and angular leaf spot resistance in common bean (Phaseolus vulgaris L.).;citation_pages=e0150506;citation_volume=11;citation_year=2016;citation_pmid=26930078;citation_doi=10.1371/journal.pone.0150506" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_title=SELEGEN-REML/BLUP: Sistema estatístico e seleção genética computadorizada via modelos lineares mistos;citation_author=M. D. V. Resende;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=R. T. Resende;citation_author=M. D. V. Resende;citation_author=F. F. Silva;citation_author=C. F. Azevedo;citation_author=E. K. Takahashi;citation_title=Regional heritability mapping and genome-wide association identify loci for complex growth, wood and disease resistance traits in Eucalyptus.;citation_pages=1287-1300;citation_volume=213;citation_year=2017;citation_doi=10.1111/nph.14266" />
<meta name="citation_reference" content="citation_journal_title=Plant biology (Stuttgart, Germany);citation_journal_abbrev=Plant Biol (Stuttg);citation_author=M. Riemann;citation_author=C. Gutjahr;citation_author=A. Korte;citation_author=B. Danger;citation_author=T. Muramatsu;citation_title=GER1, a GDSL motif-encoding gene from rice is a novel early light- and jasmonate-induced gene.;citation_pages=32-40;citation_volume=9;citation_year=2007;citation_issue=1;citation_pmid=17048141;citation_doi=10.1055/s-2006-924561" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=V. Riggio;citation_author=O. Matika;citation_author=R. Pong-Wong;citation_author=M. J. Stear;citation_author=S. C. Bishop;citation_title=Genome-wide association and regional heritability mapping to identify loci underlying variation in nematode resistance and body weight in Scottish Blackface lambs.;citation_pages=420-429;citation_volume=110;citation_year=2013;citation_doi=10.1038/hdy.2012.90" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=M. Rodriguez;citation_author=D. Rau;citation_author=E. Bitocchi;citation_author=E. Bellucci;citation_author=E. Biagetti;citation_title=Landscape genetics, adaptive diversity, and population structure in Phaseolus vulgaris.;citation_pages=1781-1794;citation_volume=209;citation_year=2015;citation_doi=10.1111/nph.13713" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. F. Sánchez-Sevilla;citation_author=A. Horvath;citation_author=M. A. Botella;citation_author=A. Gaston;citation_author=K. Folta;citation_title=Diversity Arrays Technology (DArT) Marker platforms for diversity analysis and linkage mapping in a complex crop, the octoploid cultivated strawberry (Fragaria x ananassa).;citation_pages=e0144960;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0144960" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Schmutz;citation_author=P. E. McClean;citation_author=S. Mamidi;citation_author=G. A. Wu;citation_author=S. B. Cannon;citation_title=A reference genome for common bean and genome-wide analysis of dual domestications.;citation_pages=707-713;citation_volume=46;citation_year=2014;citation_pmid=24908249;citation_doi=10.1038/ng.3008" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. Shirali;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=S. Knott;citation_author=C. Hayward;citation_title=Regional heritability mapping method helps explain missing heritability of blood lipid traits in isolated populations.;citation_pages=333-338;citation_volume=116;citation_year=2015;citation_doi=10.1038/hdy.2015.107" />
<meta name="citation_reference" content="citation_journal_title=Documentos;citation_author=S. C. Silva;citation_author=A. B. Heinemann;citation_author=R. L. F. Paz;citation_author=A. O. Amorim;citation_title=Informações meteorológicas para pesquisa e planejamento agrícola, referentes ao ano de 2009, do município de Santo Antônio de Goiás, GO. EMBRAPA (CNPAF): Santo Antônio de Goiás.;citation_pages=1-32;citation_volume=256;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res.;citation_author=V. M. P. Silva;citation_author=J. A. N. Menezes-Júnior;citation_author=P. C. S. Carneiro;citation_author=J. E. S. Carneiro;citation_author=C. D. Cruz;citation_title=Genetic improvement of plant architecture in the common bean.;citation_pages=3093-3102;citation_volume=12;citation_year=2013;citation_doi=10.4238/2013.January.30.8" />
<meta name="citation_reference" content="citation_author=Q. Song;citation_author=G. Jia;citation_author=D. L. Hyten;citation_author=J. Jenkins;citation_author=E-Y. Hwang;citation_author=S. G. Schroeder;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Rev. Polít. Agríc.;citation_author=R. Souza;citation_author=A. E. Wander;citation_title=Aspectos econômicos da produção de feijão no Brasil.;citation_pages=43-54;citation_volume=23;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rep. Bean Improv. Coop.;citation_author=T. L. P. O. Souza;citation_author=H. S. Pereira;citation_author=L. C. Faria;citation_author=A. Wendland;citation_author=J. G. C. Costa;citation_title=EMBRAPA common bean breeding program: a current overview.;citation_pages=255-256;citation_volume=55;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. E. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=B. Collard;citation_author=E. Redoña;citation_title=Genome-wide prediction models that incorporate de novo GWAS are a powerful new tool for tropical rice improvement.;citation_pages=395-408;citation_volume=116;citation_year=2016;citation_doi=10.1038/hdy.2015.113" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Biol.;citation_author=F. F. Teixeira;citation_author=M. A. P. Ramalho;citation_author=A. F. B. Abreu;citation_title=Genetic control of plant architecture in the common bean (Phaseolus vulgaris L.).;citation_pages=577-582;citation_volume=22;citation_year=1999;citation_doi=10.1590/S1415-47571999000400019" />
<meta name="citation_reference" content="citation_title=Direct harvest of dry edible beans: A Nebraska perspective;citation_author=J. A. Thomas;citation_author=J. A. Smith;citation_author=R. G. Wilson;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Thorwarth;citation_author=E. A. A. Yousef;citation_author=K. J. Schmid;citation_title=Genomic prediction and association mapping of curd-related traits in genebank accessions of cauliflower. G3: Genes, Genomes;citation_volume=8;citation_year=2017;citation_doi=10.1534/g3.117.300199" />
<meta name="citation_reference" content="citation_title=Common bean improvement in the twenty-first century;citation_author=M. Thung;citation_author=I. M. Rao;citation_title=Integrated management of abiotic stresses;citation_year=1999;citation_doi=10.1007/978-94-015-9211-6_13" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. J. Trapp;citation_author=C. A. Urrea;citation_author=P. B. Cregan;citation_author=P. N. Miklas;citation_title=Detection of major QTL for yield under multiple abiotic and terminal drought stress in a recombinant inbred dry bean population.;citation_pages=1596-1607;citation_volume=55;citation_year=2014;citation_doi=10.2135/cropsci2014.11.0792" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=Y. Uemoto;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=V. Vitart;citation_author=C. Hayward;citation_title=The power of regional heritability analysis for rare and common variant detection: simulations and application to eye biometrical traits.;citation_pages=232;citation_volume=4;citation_year=2013;citation_pmid=24312116;citation_doi=10.3389/fgene.2013.00232" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=Y. Uemoto;citation_author=S. Sasaki;citation_author=T. Kojima;citation_author=Y. Sugimoto;citation_author=T. Watanabe;citation_title=Impact of QTL minor allele frequency on genomic evaluation using real genotype data and simulated phenotypes in Japanese Black cattle.;citation_pages=134;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12863-015-0287-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=M. G. Usai;citation_author=G. Gaspa;citation_author=N. P. P. Macciotta;citation_author=A. Carta;citation_author=S. Casu;citation_title=XVIth QTLMAS: simulated dataset and comparative analysis of submitted results for QTL mapping and genomic evaluation.;citation_pages=S1;citation_volume=8;citation_year=2014;citation_doi=10.1186/1753-6561-8-S5-S1" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. A. M. R. Valdisser;citation_author=W. J. Pereira;citation_author=J. E. Almeida Filho;citation_author=B. S. F. Müller;citation_author=G. R. C. Coelho;citation_title=In-depth genome characterization of a Brazilian common bean core collection using DArTseq high-density SNP genotyping.;citation_pages=423;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3805-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=P. A. M. R. Valdisser;citation_author=G. P. Pappas;citation_author=I. P. P. de Menezes;citation_author=B. S. F. Müller;citation_author=W. J. Pereira;citation_title=SNP discovery in common bean by restriction-associated DNA (RAD) sequencing for genetic diversity and population structure analysis.;citation_pages=1277-1291;citation_volume=291;citation_year=2016;citation_doi=10.1007/s00438-016-1182-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=E. Villordo-Pineda;citation_author=M. M. González-Chavira;citation_author=P. Giraldo-Carbajo;citation_author=J. A. Acosta-Gallegos;citation_author=J. Caballero-Pérez;citation_title=Identification of novel drought-tolerant-associated SNPs in common bean (Phaseolus vulgaris).;citation_pages=546;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.00546" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Hill;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=Y. Wang;citation_author=L. Li;citation_title=Molecular basis of plant architecture.;citation_pages=253-279;citation_volume=59;citation_year=2008;citation_pmid=18444901;citation_doi=10.1146/annurev.arplant.59.032607.092902" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=B. L. Welch;citation_title=The generalisation of student's problems when several different population variances are involved.;citation_pages=28-35;citation_volume=34;citation_year=1947;citation_issue=1-2;citation_pmid=20287819;citation_doi=10.1093/biomet/34.1-2.28" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. M. Willing;citation_author=M. Hoffmann;citation_author=J. D. Klein;citation_author=D. Weigel;citation_author=C. Dreyer;citation_title=Paired-end RAD-seq for de novo assembly and marker design without available reference.;citation_volume=27;citation_year=2011;citation_pmid=21712251;citation_doi=10.1093/bioinformatics/btr346" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. M. Wright;citation_author=J. D. Kelly;citation_title=Mapping QTL for seed yield and canning quality following processing of black bean (Phaseolus vulgaris L.).;citation_pages=471-484;citation_volume=179;citation_year=2011;citation_doi=10.1007/s10681-011-0369-2" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=N. Yamaguchi;citation_author=T. Sayama;citation_author=H. Yamazaki;citation_author=T. Miyoshi;citation_author=M. Ishimoto;citation_title=H. Quantitative trait loci associated with lodging tolerance in soybean cultivar “Toyoharuka”.;citation_pages=300-308;citation_volume=64;citation_year=2014;citation_doi=10.1270/jsbbs.64.300" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=H. Zhang;citation_author=D. Hao;citation_author=H. M. Sitoe;citation_author=Z. Yin;citation_author=Z. Hu;citation_title=Genetic dissection of the relationship between plant architecture and yield component traits in soybean (Glycine max) by association analysis across multiple environments.;citation_pages=564-572;citation_volume=134;citation_year=2015;citation_doi=10.1111/pbr.12305" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. H. Zuiderveen;citation_author=B. A. Padder;citation_author=K. Kamfwa;citation_author=Q. Song;citation_author=J. D. Kelly;citation_title=Genome-wide association study of anthracnose resistance in Andean beans (Phaseolus vulgaris).;citation_pages=e0156391;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0156391" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in Phaseolus vulgaris" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="The availability of high-density molecular markers in common bean has allowed to explore the genetic basis of important complex agronomic traits with increased resolution. Genome-Wide Association Studies (GWAS) and Regional Heritability Mapping (RHM) are two analytical approaches for the detection of genetic variants. We carried out GWAS and RHM for plant architecture, lodging and productivity across two important growing environments in Brazil in a germplasm of 188 common bean varieties using DArTseq genotyping strategies. The coefficient of determination of G × E interaction (c2 int ) was equal to 17, 21 and 41%, respectively for the traits architecture, lodging, and productivity. Trait heritabilities were estimated at 0.81 (architecture), 0.79 (lodging) and 0.43 (productivity), and total genomic heritability accounted for large proportions (72% to ≈100%) of trait heritability. At the same probability threshold, three marker–trait associations were detected using GWAS, while RHM detected eight QTL encompassing 145 markers along five chromosomes. The proportion of genomic heritability explained by RHM was considerably higher (35.48 to 58.02) than that explained by GWAS (28.39 to 30.37). In general, RHM accounted for larger fractions of the additive genetic variance being captured by markers effects inside the defined regions. Nevertheless, a considerable proportion of the heritability is still missing (∼42% to ∼64%), probably due to LD between markers and genes and/or rare allele variants not sampled. RHM in autogamous species had the potential to identify larger-effect QTL combining allelic variants that could be effectively incorporated into whole-genome prediction models and tracked through breeding generations using marker-assisted selection." />
<meta name="og-title" property="og:title" content="Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in Phaseolus vulgaris" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2841" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The availability of high-density molecular markers in common bean has allowed to explore the genetic basis of important complex agronomic traits with increased resolution. Genome-Wide Association Studies (GWAS) and Regional Heritability Mapping (RHM) are two analytical approaches for the detection of genetic variants. We carried out GWAS and RHM for plant architecture, lodging and productivity across two important growing environments in Brazil in a germplasm of 188 common bean varieties using DArTseq genotyping strategies. The coefficient of determination of G × E interaction (c2 int ) was equal to 17, 21 and 41%, respectively for the traits architecture, lodging, and productivity. Trait heritabilities were estimated at 0.81 (architecture), 0.79 (lodging) and 0.43 (productivity), and total genomic heritability accounted for large proportions (72% to ≈100%) of trait heritability. At the same probability threshold, three marker–trait associations were detected using GWAS, while RHM detected eight QTL encompassing 145 markers along five chromosomes. The proportion of genomic heritability explained by RHM was considerably higher (35.48 to 58.02) than that explained by GWAS (28.39 to 30.37). In general, RHM accounted for larger fractions of the additive genetic variance being captured by markers effects inside the defined regions. Nevertheless, a considerable proportion of the heritability is still missing (∼42% to ∼64%), probably due to LD between markers and genes and/or rare allele variants not sampled. RHM in autogamous species had the potential to identify larger-effect QTL combining allelic variants that could be effectively incorporated into whole-genome prediction models and tracked through breeding generations using marker-assisted selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in Phaseolus vulgaris | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jlDGgKj_VIkhmuficmYIOSy4wzqJlHlIIrDBgRg5WJc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490464","apath":"\/ggg\/8\/8\/2841.atom","pisa":"ggg;8\/8\/2841","ac":{"\/ggg\/8\/8\/2841.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2841.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2841"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2841"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2841":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222g\\\\\\\/vv5Hn4Ld0XxBX2RuaQvVUgnnqqlFzg379x9VmBCXMMF+vcGXhdJ66RFuCVaJRRJVUFxx9gSKmJJSJ2Z5yJii86JqcC3yQL4E+ov7KVm7oF9aFuYwjtJJJlxWHl17\\\\\\\/3rZH\\\\\\\/p2Hi2ngIvKfHuRXYYLlxHCYj8dpyPZpMELoRogVOrWPJUtLzeBXw5MAxbKexx12NpQh57RbsdG2IBck98LfMVFArl3ZjM2EQLiVJEbx7IvelD0dmxk9pRVG7uKRn6VvBzAILoWoWnLzz4lzZgStQYASu\\\\\\\/Mjb+q1DhL2sjFCePBhOcmhYvXRiN7kkflUof\\\\\\\/nA8imQsxHuz1YD8+muWZakKfTKXmBc4dY9L\\\\\\\/k07O+Vxl+8VxZ5oWPmL74Wg4QcLcK\\\\\\\/MoZGXZbMVIBHGuqMHID+0p7kVkJY9m8W+pV58UzeOSXu1Ao8CDKnUIrHQSpiqbjb7klmQZEdQ3mLZxk95I3VQF7h6F2nI2aRlnGb3mOhqJPJjrzRAqO\\\\\\\/5TdacXuENXlO9trCuu9WU23eWciEmB981BLB0TcKmUXldJx5nMRyH14bOOuYsZ+F\\\\\\\/LqqQnWKEKXN2B22J9JraQKxZoEKPxBeaXA+vFU\\\\\\\/BGRJmcYg8RdRoi2ukh3BRxwdkEG0NbyfumktwOm3d3G4TZz9Fexb+flGPci419VYl5F7gUj2Re3dhFiyHLxmh5A+NGMD7vDdIq0Rj20MDj3x2gaqebaYkXg3tTSZpnx7MmAZSKvg9EKOc3wMFgzBht6YX3fPjg\\\\\\\/2oGTARt+48vBMVAPoOTNUFJrKfYVzyWOFhnBkRn4CFS7K3WjtXtb4e\\\\\\\/n3hqUrA48KU7xL5LfYTNjsfDPVJpGGWj++Wl6taZR50RmGlWbVGcV97Za7AnbzqDp3bvUMWfmp92aegw1k+NzrHFmaklH8TVwjIchR7XRX47ENDhmqUwlErCDnEQOozxIylfZMWUPjjo188mV5\\\\\\\/y9L7Y7xXLZzLpJu+09LEOxkEXYrvi4WtaHUFwjppTWKiX76GDP+mT1FkOj7xI+iQXk\\\\\\\/eInaFzHHL2kwimLj09AXrQ2Mu7GS371AhVpJaioEXDeqV6R49OI1aIBE7IdpBb3gsgjaRAC3hssVxvR\\\\\\\/j60SZx6FsXInI\\\\\\\/su+4QMZVTQire7vlhr72f9P+S2UJ16qn17UzniAbdAic4yh\\\\\\\/1exioU5dTyUIu9jwsuKYAQvWyRPOFu1ugKyusvB4zBRs7OjFZJBsyrXJgd2Pz\\\\\\\/Dz\\\\\\\/sZ+XoQ7IpBNpIdy5aDcdgEygCKzOtB+vcEDi\\\u0022,\\\u0022iv\\\u0022:\\\u00224DXgI0dhpzPNgc6aE8w2XA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002288f07c6299c24d5e3acf0805f08fd5c5a26ac55c50e331b9cfa4effc08114472\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EH7o9LLk5pScUu3g57l80o1e1UQX9cZNZOrM5G+V58w\\\\\\\/2T\\\\\\\/2DtyQFGQqD11tPkzobcVoOD6o+uOzK8hUgrXmZo6g9qJ2rHcvCFG1VjEAjwAkLyvyWrWVWc93LAUnea8XrBewFJt5CGHQucDnW0VJp8xkW5HQUQCM6rrcXAn1YGW2WT\\\\\\\/E4NN\\\\\\\/YyJkZGlcioJs\\\\\\\/Rp5m\\\\\\\/7rjKU9Jw\\\\\\\/qT6WgQT\\\\\\\/P72DZN18oelUWdgr7itrIIQ50feKnjp14EtHthq\\\\\\\/jTwqvcBkxBzG9o+fbUd78ZwXWPQG18+HKUlQz1UHvsFYdyjCbEb66TBs2\\\\\\\/DY68xDEE9MqeeV25E4r9smWwhrKeIhQGYA8BBX3PuAG5LV57N37GNlUNXKsOoe0vhC3BWpaZMwdZC9URHi6QSqnEpac3kVoZ5ZA1NnjKSAH3TgmYNlucgcbKhrl+u+k6Ujp75dlxrO+5YNpARjJF2FRtgpc8nzo6aKVjukuHvBWORhQfrFco38iPbJGlvAuwPgxLAeDiGiMjy0OiRdyM+OqhgxhM9m8StVefm593AJVss+JBHY\\\\\\\/cK29NJnA5eU\\\\\\\/Q1HtOv6xntMIyQp4BO0V256Fb47HJdb9kYnsqRdTUaSbuycFc\\\\\\\/s6AMhNLPnUdJ4ZRDz6QV5Fsw8TY+Aqow9d9crGfpIeVcR2MxwjA\\\\\\\/X\\\\\\\/yxz9OMmtZu7qw6sU1XB1\\\\\\\/0BzWHZfpH8qftUz2xbdnPWwlbgCup+EyXV0wYKmr+qQjkRgnH1ZMpYdQEBwS6Z7eNv5fvgtU7+OO2OR5KEgdS2RdIdvXywyjRlg9gO\\\\\\\/4azI9o0t8ajeXCxX4cqhqSi2tF\\\\\\\/bwLZHz4Lz7wnyfu2u+z83vUTdH75\\\\\\\/SGCHF7P8EGCwAp0jQYTuAKsy56Xu1\\\\\\\/phDIYDDxay8CZQVnJmxTFYqoVMaZ0np1Bhbhy6Rc0lkJKSH5jKoF\\\\\\\/2wLSpvq2BwHa+jilh3mOnH3vA6sWfdew5P5TYMaxsbpQlnku6u4ueLUmtbKJq8w6KNXbiSNIeuwxf5ETod2+dCIs44bq3jSWpf8P96tnjegtx1DzQDcoM9oJqXIonJ6qE8PmqG7ni0F0PVaMHN2X9omxOaWn6hBEE+lgerNV73FFbz3XtibAbiZ0FYvH44grS5T4rUgUJDkcGxtNg0UtdE0fAovz350CLa9eq12Fe+jafaFYgRgWYBfRjGdWKFRbv9ekCD9N8248Gq\\\\\\\/ZOHf0SlqhVdvEOLtdlmrGkN\\\\\\\/Y58cnPUx\\\\\\\/zgmrIzEEpBrg0gv8vBLdABOdX\\\\\\\/dSWUQfwbCPIu5d7uOJzyz502L4mY\\\\\\\/4yj\\\\\\\/gTV7WntjNeA+fmuI\\\\\\\/cCv946QuinSPi4b5\\\\\\\/y9hr4r0yqDvWUrguvqlaEtb51FqN0IVVorrfkUg8puVkEKkOLyFBAp\\\\\\\/GLU320mcezPt4\\\\\\\/9oZqRbIQWUCJtbRuUZOnf3TIZPE1wYnzFjKtCmlCFKYcj3IWg+XMX1LjCfMw4\\\\\\\/BrJKfLxpXaXfIoR8Essc6zymrPUex0JOMg\\\\\\\/V6G3EptwPkAt4mK4H6LzeOb8fB5y2NxFOqI5TcWI8KJTwhOnFMP4cTtmbHT4IGrmxSlvv1HGvJ+QRzTYQUyR2MDVZ0l1DFm62g62P55swk1tS\\\\\\\/IopEsx2dTua9yNLnvod0IGS6k9pU2pU0AJFI8HG5eifaKpHt7aK06mrbAxkI2Fla8Axy7LtGQyJVNaQSp7VwZuLrcHFrdyHJjrNzEAEgvhBPtCE0ZEcLOTFrwZNCbBkpQSeP2ZePR8EBC+9UFUSSsmZwnlMaKcHtreKPrGf7dMDtFA\\\\\\\/6WrM+owSQREumNLzt0vJJDoT9lwTtbk1dTqwUoU7IKQ0lySrYHWbiOJSdG28iLpPHm1CDDwagdmw4ZlOZFvp+xGK5kJeHeSzxfePJbDOQ+b3JeTWb9n9crxzSEXYZ+SSrM6dNJ8SrhXi4eVQDzJlrXtdRXj1RfcLbGDqUP1x+6zMAsNBdYUc6UC8v6OMFW9ri1PiXIrQ3rUztgIY9JZfDSaX1eOPVuRJcTaSV+LivaDUNmEaSn5UYKeGRaiwfyDB6fS\\\\\\\/3e3anqbBeSd3k1wSYiP\\\\\\\/LxQ5hJgRuKPMlGEglxdjzeCKDjxxrBZSMV92noxFd7foPSEqaKCQwrm1UVHQelxeZuYQLMpXdeDSg+GBgfX6iIZo4cJpjQ\\\\\\\/n9aYH3rfxpXNyo5Mz9Xf0qNSKUoqmpvTgAVTR1+IHeXEHFboZbNzNmT51KAMtYPJSqCw6Qkxikpun5eAwZJvLLQ2E8X6Xx0MGRpn\\\\\\\/owanDbiedChq2hflpPD7UOe5bv4hHEfLJ0pID\\\\\\\/2Hwss\\\\\\\/SeUmz4YWFTpbO5ChjWnaMT95QmEvrZK8VPeYR18\\\\\\\/w9ggZ4jbqgmKh6ggUDzfjtUbpRqMmwQ+WGvSTLXfdhYip\\\\\\\/UdhQqrCU5lrF4TtqfALTFTM\\\\\\\/4bT2tmDXiwX9IYaUdRXe+BwXJdW0g4\\\\\\\/nwFvCQFQm2j\\\\\\\/Ea1QfB2LtKEHRKWgPR9FwbHTdHy8uMlvUXrTTqna5eXRaYUCMNjNQuNoFQ9GPM5tnfPikFxDoPFdqPGqp0LNsU9vnlDMKVy9wes1VdKIRhsIEWQjAda1B21r7xyLGqEcG47zYmDNpGAhkTD3inda5465+EAWn1xM8OiqEXQNT0yXlVNThwj6cHFKwIGmn6aOyvaeaijj6DnvNzZ2B1QXCYYUj8wSglsjfHwtykPOpUayBwTgqB2zU59Re2f9nNIYmlYhGyZbcGRecm7rtIpTeNdmaghjvTuYbLoZhUor\\\\\\\/IQNpsUFqXeLUMFQE6QdkhjKJYwdxrD1r2+QlhtPBIDuQqs5nEYdSaSgAoGdkdlNj6ucfBCVI68DUMZ5nnG4xlmAoI+OxBfDdy\\\\\\\/U7Ps9DR7Iyuwctb0IcoWsNiSETH0cIncvd8shm9jv+wzii5TyiMAdnVETV\\\\\\\/TFhW\\\\\\\/2rHMPINzHXpoUNnNW6rwlYsiMkdjGhnehOtfcbnCKmmoTsXPtjaKP9yU3IB0wYQLxz5lQgPY5sKiTfTI2TyLFaGWukbsye9h1mbFdQLQL5wJk4S6ghW\\\\\\\/gzzACz2OIGL4gftTRqruPBbBsyd1yiH5oH5fWchmTwCbUOFFj7UL5hwSC2u1II\\\\\\\/9heS9Xq2a8U7cynKF8psTTIvlh1u4OhGyI515b4kHWajFcGv7jAddvhDxGo\\\\\\\/oU3lOoAjJqfwc3HIzmN1tRR8gI0Cx3lPIyaL2mSpPnDBm4kj6auUK11WBtQKvHmAHpzVBR7zDsI\\\\\\\/OHsqFdH1eesgpybPYW5zUGpeMWNEWBrXbpx3haugHfgipLfD\\\\\\\/zJy1g0KlpTK\\\\\\\/SpGpFcwrlMJuuas956WZ0OjQlKzBejFhuAZRszYR7J0ReZ9VieZ1FCO5B+djZN6O4cP4+UMJmyxR6vDjgMtjQgwu7yse7p2EIpuxnoCwkBwCcuhEvs2paVyiOsnbkzzHGcq5rjBknF9U6Nq\\\\\\\/AHGrlg6vcSMZiuDmX3csPrbyS+nRC\\\\\\\/BTHSi3hu5499BWc6LBPpBHDZ5LG+JjGnp\\\\\\\/qYNHsjlfeUIIZJoH0QOfZNAx2\\\\\\\/oiw0C1I3BGc79gAw9c\\\\\\\/6XBdd6AUOtfgfoVabXAyxHnpvMFwm0Yp+qkgQTmGKDLGN2anSW3SAJ72L15UcmqHtgoF5osWkg6ky6FbgJzSI22bagw5WbrT4yDzA0ze1inGITgDWX1vY7g8U9ORqa\\\\\\\/S8M1S8d260t3mGq940HMl6+7ApECfEl+SRmPEv\\\\\\\/CG\\\\\\\/M9WgYYPVBLab1aeOm5vHsb4NQp\\\\\\\/A5RB+Q2OHiBIBnEvVkImyLWKuokWEVT2qmxRs1Gr+m31jHoUvJS7KT5+vzLb\\\\\\\/YBIEKNBIbo1qO13yWJC7rmIYjxr4nrX8yvGWM9v5TZTgwgD9ozIofqi6H+4ezXrOu6fISzhToGKoenWZ3VKVQOoKf7l8EfjCPY1o982CqYdwu6SmIqHE1jgWxh5cIrkJ8uKPBtc\\\\\\\/slRoZzhqOEk\\\\\\\/vQFhlarZrgGiKZJVJN77Ez4MrzBQl1VfIejgG5Ssi93r7QPu\\\\\\\/nIaXvpfwgWQBzwP5BR+87KqGtLzcZIwG6WJhi4ZQ4vbGsaaMqUIvI51WF8yLDw0CBKITOE\\\\\\\/JpDXGLwZb+7BD9MWfAKb1l0xTT6WzFnqdqJkt8gBG2pNYK1klZ\\\\\\\/LVHzNeifSKL7Fq5EXGywaQwpOuooYVs0AEpDWoRYDfOta7r4WWhp\\\\\\\/7XsMlacCddfZ5H8mZP5RtUgb3tVWhUDHSfB5WXRhls40ULjUphPpUMYmWxQ+Avj+wK2LtdVRRedR9HM9YFqja5qY1xyvfnoqPjdlUNLbTJ+NAWhhyED\\\\\\\/+UAoWPjY8EQmuznrMa9EQYVMVkBxz3tCMVei+MAcZPVGNYhkgocvjSlcahcf9Tej8zGnocmd7yx9T1aqoEIvtlmfBzgXYplmR4WEo\\\\\\\/zDZsWBUnx\\\\\\\/Saro3HBQvLC5cuA4qClILSK\\\\\\\/BSSt6Ti5Vd1Dtw8lk1RZPKybclbiXUuuGxm2oNQCJJm2nT8d+misJDYpNIUuXoP2Dbm6bai85el5ogshOpCaGeMfseIcFyc5o5FNvbEgz2ZEvKaLgSP+FF\\\\\\\/YEk6dVTYEqL2ra5O6rD9ZTd+XyTO5Ny6y5qMte2Jb+CJrLaV7XR4SIBp6w5zxgjE6iVknL3WyxYOYYed+negWEUYgfbsjSILbt1nWVZ2dtlJTcOY1LMeEzAGFUjm0ExAJgDCGsKHDdv4fHKJACBteyJiTGsR6LFXcFCxM5GrVLomJpBc4p8e+L78c3yolBXqqw5l89pe\\\\\\\/oB6yfdYQvrJjFGIa\\\\\\\/QAHqXvnHKtzcHor6InlvpHzTro4SUOO0NQSkkVv2hExmGe\\\\\\\/8FCCyW9uTKXuQpaFqpbJFbUUxbHgIp69tE99Knu6LwGx+2VyZpAkGVnzn5kGNMdxFU1Eh48ZyCGv7ncaRuH6brMVy3losnGRfdC6ExT0+ZYaYYVLdr3z60T7xcelmOhPWB+Sc8dyCl7Fa2Uqvsa\\\\\\\/X49eslvjg4b76K5dmfk7mOoJ0XJKFVD3EGPpCT9TZ4Unzf7+S149ki0yA15HIUo+g\\\\\\\/63DYVcq5BfT6lxeIcVl4VivPMLaaCzJBcx1o+glwTVTXzj1AsGVqcwbEAL4SW5mVamG9nWaoc5FdSbmO6uUbv0EFMUEtUm5CIe4AED7vCtjc+B\\\\\\\/DlJXfBsAJPq0aMixy7zb5N8GKS0KBrKoKmsGtOUVBr\\\\\\\/anXgRUg09fAb9CGfWkJ+NYQ7alD2D\\\\\\\/h+LWk8IhIbknbU51fukk4OPJ6ELCx2YRxy1z3vT6o38Z8Y9TbzzWLOdxUjminBKM83oR0FPFIcdYCc5ivdgAND5QOW2KeZHKi513\\\\\\\/59e7rdlGVZvkzCLQnOdEsfymCs2ze1li6ayaqLicvOPfERpcK15tkm6WaHBthp4a9dYWuhlCq6yUV+\\\\\\\/UQynbsz0ACIyWoOwZS5vYR3O39mC+Z8kuQ7xaGEhjtbc\\\\\\\/u1qcQ2mCs2zrAv0W31uVSNXR40MxtWAvG7pHeHFPpNeT7HutHdPvUL1+9PQxYurNHNYBjv5Bku5SJ+OTiUjUMx+BwTfjIK9vOTsG2OUEvA8odkVNhgngbY+0KWb7t1lDyzM5gIWAvNkGE4wBN8S8+317eJ5QwK+Kk9feA18JBRqeHScvSO4aoVkeDFV2LXXcdYUDWnPfrVPx7wxqQwqCJ7KHDFEzhohKmBbxbjxnhjy4M2iY98Tkn0RO16yPVKZ\\\\\\\/YlUTL1UOOl7GFkOQBkghilL7lGrmNCrN5MUntpJQn2iHz+E6QauTHEWlLcEN2sDA2Vuw7cim0iXCY0v\\\\\\\/9EgaHu5k\\\\\\\/YlUYXqnnvw+AEjFepLqK17tIijvAIzCEQ8J4afHkTdyzBNUBsSlwMQFmW948wUvjfkTyELKwq3R+jvp6Ssyu9ta73hvokH6xpETxyRelF2tq4Y\\\\\\\/Bx+j\\\\\\\/p4o5FlIiJfxnFKhH0rA0eSi0YqCK5S7SW8JE7QnAXlpxgwEXvQbAsQKQ1z39eifl84p4gn\\\\\\\/aY1vdfYWrBNvq+4RBSSnugLuDiqELIwJwVlRMPAVmDPafQ2+yUFjTEORjquNL5fsc28zRhbZyHJWcJET6XR4vHK2FX2V9rvtxEKybmlHUWkbUjmxvC8f0Y79cFHCTgvo5lQLNrLvlmHiLY\\\\\\\/h1t7vSCKeOS8eVJ2\\\\\\\/ZSzh7lIgvWJB5gQD3k+lQJEBkbZ3la0Y9cYOjElchkmxTtls91uaxm1ulMDWchcEQbwbp7Xx+oyau0ZWvwDbWRzkK7j3UhVv6eiv9Lrpev9njPQfiqXwi+jGo0SA8o8MqWlN3Dj3gRF5XfQhCYDIXPjTfjwEC5JCCnADK7zFT4n5nrzOd5u4Lww+0ybbE5FulNC+fpAoKnLYDVl0X2TNb1iviWLmP3OOcgED8HSy5bQ\\\\\\\/TOKwNJWU1d8S1m44NvKoeX98EHK1xXfSg6xY37y3iLL3XZL4+z62raMu4yqqqLHgJiCQ4tgLRiHPt+OFw0x1dnsv9vPdooVL4htJgVINtZtamcDes\\\\\\\/dxGf7mpAndznJVVGeTPw1M69umgr+S9f7gzI14M7HxERmb3TDhk2mfCG7cQsQh8uDGLA2GwDHbsPVX30ITziPGPbqKY6dXz8gzTkqtWPLjkNeeYrnHrY+xKq0ym73PcLDCqSwLeKsHBeUoceLULY4xCZpP3uev5hC1bDtBAKcmCbRFO6DTfB50W8nFsrWKNpaTb9roSPKsfEF\\\\\\\/8dE7AwPIWRSNsr5AQSu5cK3WSOrwbCj3vP+l9CaB5Y0uIQx+BSVTd+jrqwgjLv1IDHZm3pzWzjW39iTQHnbXDqom3v1cGSAaROWPO0rDwEqLP5ncm6t5ECUBfhyvsLy0nBRQFXKST9rFoUj7dUxUg7\\\\\\\/u0qL6Yn3yVnnKjn0EAf2gKxQiIVleO+F4BWlZvxtYg0Wa5Z1qKJNyxc8x449fbz1n1TpOQHPoLDoeF48SxcG056Xzq9VpV89M3dA7IDQKhbB2m4a1r9jDWcnzF6CAdD1VJv2YTBiZnNDPRSZefan3rMOSunf72OkJMp6v2HDW0i9qple65Vqc8np8oYMQTgD+ajz6JMfAKc0fAsAiBv\\\\\\\/uabF\\\\\\\/3sFm3XzI1UleE+X0DZfstLS4AgIp+ZkH9LIWQ\\\\\\\/lBxCjvuHmdDH+TsfT7T1YBH7eleVNQ9R21eqXLdBqK14xP4ScI8MjXteF6g7TM2piFcbOKc1YTRiX4KEhTO60TsagYJfE9kSXgr8Hs87z3kLi\\\\\\\/nO9OQPWs4ICtUlxecEVt+QTabEF2CbtRbVyiQAAc38r+isNOlkXgGQIS\\\\\\\/FNPPq7fwoJVQ4L3AnZxZLxA2bl5sYsE8\\\\\\\/fUIwHcOkrwpyRbZMEN\\\\\\\/huUvVN42Lps\\\\\\\/qzNKhcaiICMSH49a\\\\\\\/kIGounzXZlI2Bk5qRQzTLklywWqTRqFjceJYRQDro+25ND7Q8qna61Nlc92cSXvCsIYRG0eZoQlqIBGKorcwl3B\\\\\\\/apSRm9pVQjSxuBZ8bd+UaCKmdkcvgtlxzPdo8zuVF3gJw2QbrvFLOstwOMBPZVhLXZOEopxyfbZTfgUmOJmO5MflYQp5XEtab7HuPkyAt3CuF9sTydLcFBhHC0HSKdAEyeDfQY4mwXXG9xZTt\\\\\\\/YMAmk6DT2oAlcmYFslvmUU1hDqRen8D257LIHAky9NdzfFTUqvVN9UHmSIdWa2XaorcZJMEiOosW0d8e9BveBXQa7ax6f0IGKvHSknYPL7exPm5cLYRpAdnDK\\\\\\\/ECMLnNHUYf0QxPLSJwn6+lE+xgQ4ovDayNZKUOKZstSMIKkNnbwHuN03uH7\\\\\\\/b5RErXv5p70Nwg5kLkeAixfKeuvnGuaREDixCQbg0ZDnw8iDOFoEOrRCL4XwGY0rE4YOS9UwNlmc9Z+XKh8Sl3y4vpFAeMokYpxFFMe1D4UukP4uTzy6GMRCi08qkH\\\\\\\/iD0E2Q+nBSsu28W3dhMIdf2Vhy3qhAaq593\\\\\\\/BnX8zOLVTCYTevXYwf1\\\\\\\/uxRJGrnYiZjqlOWwhFTbnHCIa0nYrULtSFsGIrJqAeiJZLAaKnZHzM4QOEeSSVKHm1UsW587g6LqODwF93TO4wOWLDOjl4wOkPrAaND7r47NgOR1k1rxGA23PAk6zel3vqLzDTpyJgFAi2BV9y8k3BWTiitmY8IO7MoU0X3HV\\\\\\\/yRht0+zy9jxzbr\\\\\\\/eCetO1YnFhuGgtENiMYkhany8OsfETaZQ5G14qxRhl9wGIQWU7cjom6IhCwB+q8R2juzs2N1W2Z9hZWptGFyOKbG5cGiiFBdwoC1NWdEORxOHk7owSnGfKmeK7+WN4Jf02ETIBdDHP+eObZOGWnQjB2ppNaXv6NJPYNc0rIxYburYVHyb35cbLUN\\\\\\\/pJVrXf87l45d76PxbMQRwTRvQuOVragYILklxmj3Scd6uoAJaqPGONG+goliKLv1LyNu\\\\\\\/60WQl24f4irFeGxXJk0n6Bs8wl4u\\\\\\\/nbXmMVG+aXyq1NDJF67xrZ0EFebgX78EQFAt+6QFq4dadlqLf+2RAVlQ\\\\\\\/O22XRGCzMbxPoSdONsf1uXixo+gUrVz0aqb1fDv6mUaOLo4ySdhfX5OLbBD2qTEoT3eFjGLzR3yWnYIp4vSebNSma0UKedwwe3PSSUJ9Uq0B3weqcRAfcV6c4lxE1amNvHsLVGbkyg+adOTFY8gHduLvYNSCHibOnXmLK6jzjGi8Kwt\\\\\\\/\\\\\\\/GqiMPAvHtTDdHsswuXzjUK04EyaVTVK8uCUUBags38FTqR0dVwIkqmCL0Av8Osj3mwUEryz65yIbeCppfXW\\\\\\\/ndqOMI7jS5lO74oNpEO8hTQ86Y1OU4BM9LIchBhxQuC6jEyWRMze5oeR\\\\\\\/sfw5BIA+35r7uffDMynGpGwHsAML2kzjglxku5mRYS+7EUbzZtdz0cr9ndGgM17pp1moZDj8QyPSl2GH++L8jBLnc7obRFkDx+dZ1Vt5l\\\\\\\/4yS0\\\\\\\/dAH52fnG5+3CWbDx4R2EGT2WFBc0sJk7N4gpQdqS1\\\\\\\/kDtwOxXlt1V+PASZrskSwMSNZ9aJgdKHOuGm8zzRIAbq86PO\\\\\\\/qyv9BU0hpWLbMEcvmy9YZclHvrzOkmileY6R9jeM4p+BFKDAP+zvhqMhHQZFQDNopok9J5qvIAho2sUszdq3xAc7H7K2ZrCjDsLE0ibaTe3yef21pJWEB0+X5K2ku1ZpXVhymjjxkzX2C5IGqkPsFhJuwr1WubU2TuAg5+jxA6snh7Cx2LZXuCPfJyRTrxXmjN4GsPJgTLnmbi67csQFIthD+7oeWQhf4gP\\\\\\\/JktSv3KzKnmEqtLe0gAYo+\\\\\\\/QjkGBVqpHyUsVx0Tw+1Yzh0zqQek5xsHb5OWfBfUZf02RZNkY68UuM90sjI3XVtHgyO0Av7uFCcPRvtiPTQlnSR3AKQfdKye5LISvnU4DXOI6i6C9dk3JzNveOKTh\\\\\\\/Q76DRylngtiDDt\\\\\\\/O7bmle7y32OrKGo+ak6zb1eygW9gbc509DKD54\\\\\\\/Rs\\\\\\\/3bIxMhjhpxC4YSN1b2WCcZcuVJ83w39CThotTtHhNX4oYdUQ41a04lfZLD6NUlgyaGZy5SVf+sdBfPK8UV6MCBJLxmZy0sknVoR5AcRrBU8cID0TiPtm3Xb\\\\\\\/uocqcD3yPOOklkw\\\\\\\/yT2b5qOpnyU11RkzJhpgJpNbVd\\\\\\\/cc4mBtP6PD5wx9I9LMbNoMcDXrLwWpfNqOZVXIyUYuaKMbgSdX7GsUO6GwSHb0VRVcoCsT8EzxSWmbBQ+kVvbh\\\\\\\/Xflx5oLXipBhUnMBYAtKkoji3HbPkGaoLp7QKVCbqi\\\\\\\/4qLJKUv3UAtkYVg3i3eCb5cfRmaNMCrrfLumitkAwqEDIzpBhKpP3aC52blpB\\\\\\\/YHL\\\\\\\/xGLV+PsBbxBouVXQCRNFSFfc5bwtnQ32R3odu8KGs2guNeZQMbXxjJGPfo7tw0LYFMAOKs\\\\\\\/9G0sXh3B19GmswboAuMjXTcn8dIEpDmCp5d4eTt5pNO49EMhcuvCGo+czQ\\\\\\\/CiHaXTLIMbseSKG53Zz9ciJp1uBUE1GlmR0G4upjRTdo7w3tfJebfYcplm1ZY4\\\\\\\/klzML9q1V1SWjhylD3V44\\\\\\\/WwfmK7xXz71aRFNy6b6imjdprjci0i3db\\\\\\\/scM9t0gjssq7gCbJv9BJWna6iAuyryTaE8IoC8XiUNY35QN6GKJ7IX7erxSLERqU0QTr90cpKcUA6xh\\\\\\\/4htGqCBkfCG\\\\\\\/LSSqYs3dw8tWzUUjqXaSTmh7LQJ\\\\\\\/Ng2hZ9MzeG2py+JSVVuRBgfnInh1LCMLKDBqDWaFp+GFrWCsECONyx5fpZRBSb4U3MP7jVn+EjxyrAYiBoAPBqN2sKy5k3mUB8xM5wQ+eaDMoHm0QjfVE8zuoHyTz20VaZke+NAKet90wzZdYxRAoryNnpEa1yt+PEpMn8A2jhwwG+E\\\\\\\/V7fXSq+0FQSOJrmNqozcoIPOAq4MzRnlCvIqgq2JMHUOGA1SQ0xm2+Fip7l\\\\\\\/yndMf7657riReh5o7T4rN7dy53BT86WSv+TIbslbywFD+RQaxCE30eX1r7X5S0SxzF9tGWW1cCNRPwSTFsMRdY12br4sK+FjUIb45XB1irm2RPbu6yzXg71ZuDMwFTIvJJrjQVuLukrqbfhExFtjx\\\\\\\/hcgQP8xO8VlaxOM2te+nA7C7DHYTHuCJm+LeseKbK9Pvo7v+ITFbnTSEbYy6G5wteek5\\\\\\\/mmS3O1lwb\\\\\\\/rTwlTqvyeY7YZNxdiPIBbpSx5SCWhyjZ2Q2AEjiTo2e6+5vrjTIngC\\\\\\\/GRAMbX5MZRT7I3fQbte58Rq1vyvS9HSgC5XvQ\\\\\\\/7GpKfVOWUEkVEhz1jmEHFWduznvlyLb7JLrs6tQKjnpjrVzO61EtRPoUgss\\\\\\\/7\\\\\\\/krEVfsnQit2te0zIkqgi\\\\\\\/NgfHaFuQGU7V5xrmld1mkJIZTdr2v2Fut3YTA2nJlCWzv5VFYfSu6irLQg\\\\\\\/nVoi641Cetu2gqVyVyTa3t24AJqEZcVZtdhPAh023Z3HZmgjMACYktaLCBA2HgUgTJcA7Ku+utfNXmGNVUaPjLca\\\\\\\/Pt0PScDRmajn6b6HjBLPvivR7bas5Fk9DgfxqJ1K\\\\\\\/R26jv1BwiaT0A4k21s2jG0PLt48EGQVYOOpdtfvOtlz+PP5sB\\\\\\\/\\\\\\\/t53yR8H6OceJISwCbFyPLEvLW\\\\\\\/jHvKfiis\\\\\\\/la7PgqsV0ksdLSF\\\\\\\/G5vCX+7rlz5rU7hkDYtvbrtUiqmkjgNzxAI56LiE+IXqaMVkEg4raQMgtAEp7BVHoMS1JtPLZkJqJ8QSNzUMfESdmZ3eHEIEGqRpoP4u8rb2724x8gbl5IsJKvLsZ7mg1XRySlAlf3DwTlaek+e4QRRlgf2NGo2J9SjfDXFNZ\\\\\\\/7ctXDNkZhaAaIKJNT2Uloab99Z1es93LiyuzRhoqiunuMRTDVlCFMdDepLpjbSayLuq6hR3nBK\\\\\\\/+uGnuQEyQBkCD0cvYGaYCWpRkK2GdMqIRxxtRrJqIqBl2GW3z\\\\\\\/iANLL\\\\\\\/\\\\\\\/rF9ZX6OOZUEwK66nrvjrHVHc0ABkOSLWsSiTfB3cZ4LlMcPcEktNhFlMJMRqUaam0aRkOr7ESN3VIDdQ+PX9Mw1YjWGDgxcJHeuoEJST2EVvKgIQ3hiAtiWUW1EP2Pfw2PS2Yvk91Rh+t8yv3Y1eF+aRf4Otq\\\\\\\/HrBbNdFN0UIv0suZsfZRZqdigIkmtyjrtYot\\\\\\\/m3P5PDUt687RrQKEq5l0Wwh6iWQht\\\\\\\/wa3en1IPVLY9\\\\\\\/KObBbKFp7qDBL3xCK9th41XYB3ockBgvRCzxz\\\\\\\/R6Z8ZjzMV3wd0p4NKRLSFTUYx0St63hFKwwbtN8HkAUpLx1stweeEDb5PkOUwFqSnQ4aPToy3wi0Jdmptf0rdBYj0GuHBs1opICfSdM1eACtEpgHoudVx7XQKMg1Qw\\\\\\\/umMEAWjRQqr+K7U1P6AYkctOGvxt3wWycOrl7PTjNaQHp36W7EcbWh1ny90llaick5RFjt4HbintMs4Mz2dXbQeKFo7BHT5v6rnZu6I+GX77B1KMPOytaa1VFVVyC8Q1QPha5vwQaoo5KejFSY28L10R6XPBzj78A2saCxlWITBmHlmoNLK40\\\\\\\/mkwosk75iqM1QMOmmKzAWnyQBiGFlstD570itLtjyivhi78irxC3YwpnipLcST43+QHqRMSWLHmgDClOKswFYdqYWSIFGaJoKdibLqH8kiWIz9JSMIEkknaeWGa7qSHt1A9TXrfSyX9kjNnBIgOvit1isrt8SSGc13ZqGRu+eedyBOdHv5Xaq4QaeSEUBVZZoCkGlZVJNDptDv7wAM2r+bsD+5gBy6QUYCREtdS66V9Pjevw9VWqDNDmK4mbZkU85Z+C2uiA\\\\\\\/DV2o\\\\\\\/Wmw+JabvJWmch4t1a84aM3pjYpOx3r\\\\\\\/d+Exp\\\\\\\/WtzBLbw+ZXBuuvtt7HHtWE\\\\\\\/c8PsYc7jLGul\\\\\\\/uHAH\\\\\\\/LS9pkStzN2NPtZcJh2uygrfVyhSYg366bWw5o4l7lfog7cZ9+TlTJVgHo3nWgkUc7eKuKd4tJ9s7x2laFTuaqO9Amlq8AGRaEaVQKnoJmWiAnfHiBbusEswHP5dKtOHNSEYtTsabzooSPIS0S7UBzb+7iCC3sZ9QtLiw1FuBWBBWQvpvyptm2swRRrntf7Ea\\\\\\\/lzWdAJVAMXd\\\\\\\/zGa7j+jG2jrVIc4zB6753+eDceIBJA6a+c974Zf4Opp5UdPaCooCd1539C8x4RYCxMnCUodtXSr+O7FIczzKMpbHbwS2vebEHAJ++JEqw4pBlG0e\\\\\\\/WSnVkJTLicTeBb2dPpxWlOO\\\\\\\/52mwGPSySBngF9nBcxT4iBXp1K3+k3dvqhgaRHPTOwuZBSD8ylarx4cOPQZ0enHeHBwA7yak6Wgg\\\\\\\/vJ84MeFZ1TDnx9DkX1jeTKOEhXE8iv1ztzFQqiCpw1E+xjD1+qWGMUauseN1Ekpq9qf7wTU6yWaPNrCAK5ukiR1b733i8sWkVFG\\\\\\\/QoT46f866iTlb5B\\\\\\\/UKbk2bQd59yiffNExHfZMB+9ZFsnh86HNDO63pIN108+J\\\\\\\/u6afZ4IwcNh4Ysf71xZ97bc6S0qa06KK8WgVGy0cA+VbUT1vDTjJgAZNu3ydA3U61pzbJVBfoREqoMIh1ElKptwI\\\\\\\/z\\\\\\\/vU4LyBH6Omhx87esMu6mhfdGvD8n0Sh41vmEPdNAqgAmiXl1k6F8xa5lbS\\\\\\\/XVHapIRMy2VuRv5M6w1pTq7PyzCEJ7gu\\\\\\\/ysL+cRvnML8aC7yfinkcTdtsWbA4wfGmbFDLuw80UykON65nfza9ZzNTa8F+7ojXVlhylqf2Psl9f5pLzcFUWhTM4r\\\\\\\/Fi32UKPfAouw5Yd6dHyd0BMfnXTrjjbBQ0Jngr4Rc2xajiYbRY6lUGCCaSSZVF9Y9tsUtvUILTCIzguw0aIG+mlLS5+pLb1xHeVLTGn0DpSKvVAsPE6OOWli2WolVUSCIqatV8NHIXKlccl4HFxqjvUUD0eaaDSfGLdC0YouMjz1Fi4vZTxrLfkJRYBkzL7jHFH2gb6\\\\\\\/PcyzLPHjjpAuEkfTD0tWkIjtnWIeeWH5EcqhA02TXc+F4uECp+w4GnLTl66wkZMMg18yTuKaIg34Bfb91sV7FeZGgrrbmKGZkNXSVikG9u91+Hlo\\\\\\\/VeHMR0pBNlJBavgd+10dpfkZqduabIEU\\\\\\\/iUPMUsgjK+IgZK49wrpcdWRKCdaMeBwT6JBMTerCkc5aAkPjaTWEtvdMix2iG\\\\\\\/XamI3ixxMvwaqKtMt6yq8tcg6oAWFaoKUUOZYpfqljnsjqQ3yePit3nQTSph5zeNgWv59VKjhdMdKP+jK5KlHTjEci7YQm4sEeIhyRgXbKJ0sf5zL4EHqyg4STUVR9rg7IcA2Lt6dVJR6ksEDQWetNi7IB0w24CII7F6aXj+Z2PNN9SaVvrKXXMlpvX9qjp5p7rjiGWRUB+8ZcnbpNpdfZ3qBpnlShQlh73kEt2049mRu0E0KIXZryPdq5FY7LqQlRYJZLRJMFfOdtHgmPFlJyZExcilSja6SteQXboVNLR82K8XQoyyz3AspHjvXgOMSLIEHfopA9yiIRdn\\\\\\\/E2mz5V9LDVuOvhI67HlZcGiVLOEQ4pcwBoM+Gm8E604PkuvgwujpqTH0SxeI3dOwNoFBtgUT\\\\\\\/O8YNN4I6b9ZcstG+IWgL0NUD958ULicJ\\\\\\\/oT6d3yGlDHdzDV+EqywYRC\\\\\\\/L+d\\\\\\\/aZsob5ojgoeWp4pKl3qGlGji8LK8WOTsz49YUVUZxPRLwctS2FfNiswmIi62RGYpl+JiXJMZ15z0qg4XSmWDKYpcmYw3mHU5j3XRf\\\\\\\/m\\\\\\\/ch\\\\\\\/YpSGUu6nbjdYKU9Jcn2SMfNFL7v\\\\\\\/ffnJeOovsUe2eCRpklLRATqeYBNaY0w51NG7f+PWJOLoO8m8O316vOHiq2JIU87MImDXScBum55TFGohKN92X+GAKuVl6uo16woRRrMBHGGpk7csl8LmyDxc4rOttw1sg1fhAxBLQLyzUo54V5JDsZg9se\\\\\\\/YsuFdS6HKf\\\\\\\/Jfvl5WbK+Om738TwF3PWa6vTbmAHTWeRIwOUepnI2Xd7yujDkTnAiE72G9fw+SqxROsR0k+l3Wt9k96\\\\\\\/GxpfaVhGbPQepAMGP\\\\\\\/eCPhv\\\\\\\/ilgknEjff75lcHNmZoXGsWd6wR0AE4oPp8joqL64vEuE8Zbz9n9Q09VqP+23IUfGpRB55HlPsQt4\\\\\\\/151Pj09usuNV4fTpSyVwGK+fbTerSYW+WTU3QlfJq8CIwlqAcfTSFj9zPvJSGbqM8kNt52DNEgJz3EXKtCUVnDKjWtDwkktuPfsA4LbnypdTI6GsTNZ7pO5dHyV5YaBGo7CHsSTC2RlRSG8N1GQGfIXyv504rePeFB3Qpvc5eUrYkRvrqCYFhfB7kn5OLxEpSqzyMglMz3kRYyuAMW8lN9HtieQFs2nDkwNC8iUNZ1s9DOHTEJ+dP9cyh76io1+EvYOMCSbjhvapaIK59FqGjg1dn1BjJqMEXjotlP6eT\\\\\\\/oZrSR1pLDNDOjjogmmDNFauln0NqxI66uL2wA35BVxH25u4tFUh\\\\\\\/aZUQcdEuF3Ho99ci7pgiohAZq2IHG3uT7s2bEQ3vHLbUHJaXdN0NazVV0jy31VLjOuBVsCdgxxu0+A6FuoexapZJn9dSG\\\\\\\/\\\\\\\/IWrmgSDvScZ+f\\\\\\\/H\\\\\\\/BBNY++bjHIH6GJOQiQPuzx5FFVHw94pdJhb3MxhzUjHNPHsm5n7AnbgnIkiWRXFmo91LdWBMdEOhXSwArHOJeGmbwNlRCzMLzyWLDRrhQryPc3tLT+Zowj7KLVthEeHIeZDLyIcARrm8antRgecH7zpm8xSifgrcrwgFl0dTRGgGzBxV8NxEzrPuzyVRLaoSXe7QHz9sMRlKuACilIhJSB83Jx7eNfy5VmGA9hsNErgUHNcKKnugP2sjkZaXd+dgB\\\\\\\/FcTaeaOctmsFhAC9KAltQE\\\\\\\/qaYeWpvBJJ\\\\\\\/FY\\\\\\\/d6wMjkKWGErnbWr5SP2ykNBrs6s1avZ8klBiCPo+HJ2zdH1plEZ24uCdyGItMbJ1BZSIwxKfI387YggHpRk9vTGOyu6oKUa4x0deiIyKrHMDUHr9334WUR+tQqCLguwTg0L+1oBdsUeMKtlX6SS4otBx+51MIeGIZWCQGS4I1rbii3Aocvrma1Dk6uWWKst1qL2YFcbNQBqTZ2S5eruKcgNIJMpQHXLNM26+IFGy4Oblb760fLRpZdB3WwbAJSWO9lBLUjZnkw+aRcwnEx7noTF0c2hsOVFMAzxYo4ametMBjWv4bNYNsOaILHSXbPxYvRUclEZiueIytxdIK5wPQcZ+SPdVmvlk7Wm8vYS7\\\\\\\/b\\\\\\\/SvoiubJps3kNhMtQf99JMGoErf2VSqTGG7PSnhy17WfoQxxmwvyVG66X\\\\\\\/Hf9AIsZIr2FX9xL8AkQeACquewObxbc8vJKbCQBL46eYPTzZFaLPoPQkKTzep4jmy63pmuZno6uEVQVJ\\\\\\\/r6fCD\\\\\\\/uoegf7\\\\\\\/mL\\\\\\\/q+aT7XtRPqF\\\\\\\/qFOM4lO3+kWRaiTpbQRJXxYXLez6y5e8Bj51DzkHoVCD27DjGc26fU1BTBFWqfAkU5hsZYEk7wwdUdrrrmXlT6CJvyW06jeY0EnjluRXA3SMcCxXw1lPBVFkPz5e6cyPwkn1ELq7vOrMEuJmmyzrovK2lAhicTjiA0WT70enmnRyOQumPA2iYL0SbIXcTong9PIWUQSpstBXoZ43Um2iz0io14Tf3vYsXIYjwhyDb4Igfg7lIZiLOAcJ6s6Cj154ERDqI1C09V25s6qc5JdDe1e\\\\\\\/+K8S1BwePhvn45Is9zkuqmhFgpYqupo8JAyQT\\\\\\\/fM\\\\\\\/d\\\\\\\/ks0S0TNEJk3AUtSPmfmRciib0FT1rnHZ1GmYRbVGRjtYXu769GJLpc\\\\\\\/ch4fu4Zi07Wsngd3MK\\\\\\\/2jkdOf+6OuuNl6o+WUU28U4BFLNKrVh8noBsP19YOEzmuPth2yoR\\\\\\\/11aNkKOZpOytLr09EXqXU79UKAPS1KU5VkBsGdjoviZKBv8jL+a5T77vwV\\\\\\\/oE4lEB3+x5aT1EMoJzhI5KMEKksQDXrsa3O8edm3rDlTfWL6DIU2TNdF2KiiwCke8vZjBjq2bkIbOjnuN0AoDhcZgdT5P0lRKpy7RTKfoI\\\\\\\/xI\\\\\\\/UEgHI6XziZxGB8EgCL\\\\\\\/cXjlPaDCF13M+R0CR6F0bPiOQNz7e8Z1kgqzlTBPab5J6TEvYTpnrf0VRqzpCj8pcl3j9+HFCg7LhiX5UQpAXCjyZHdtm9dBOfMBQIM1Av+X1tf+gwfCbIlHU\\\\\\\/1xAu8dGViKK3UA55dRuiqD8XzYJ8zuprFPsRXSXKOSh3ZY+BS8OCWQL6jRiOejrj15PBrSbVsOvVx4IkQQqWaBh2huO2wYH0GEzAzvmAHS9Vjwr6j5T0kBxEU178fvHaaXvlUGBKASutxWrpoIahr16bpJgUnUcBbhPl2FEip7xrtdwutRIpEQY\\\\\\\/YCUHr91nmm1avEblNn36O5coc7D\\\\\\\/P71tdn+RAi8UbPArVUmphswM6kaiqkpILT5iVsryP86aQAGQWyHYicoc2my+VqGRAWHl\\\\\\\/yXVOn+RgToMy87s9vXPl9jfeOQEMvJT9FjfUDp4RKRmGhiIKROsJ9LiGQtXm+VhnzxZ8x5\\\\\\\/ppBbtCUgL7y0M7uCd+AHmTS9lrdX8QTWHa2LsSUr61OdjcGNr3La6PS5rc7IbfiY4f0\\\\\\\/E3cDHFdrNoQN+hcLR2tU3cW8sli+5HcbPJ8nrATd8+dgwKYVUJfhS56KWTPzKiCXBGgLMbi508HPtor5eZxQ\\\\\\\/f1Gh1tE8EmkgvtRcpu5IK1q8Bc3Yr112E3IvUqe6M1CkeZF1kDQl+Az8QU2JOOsZ3AqyQf4Kh2bAIyEKNxrw5Fqkd3U8s5PASlh\\\\\\\/TlHtK6KG\\\\\\\/qa3npV2mUbXGIY4NW6JuyFff3NV76qpqG5zcQPmA3KJz\\\\\\\/l4WafUUcL8UBw\\\\\\\/P53Nejv60NdlLcUeq9sspvXMeBqpurHb5W93Oc5AXvW\\\\\\\/TzvPuy\\\\\\\/crQgIxIWj9xxIetHAyBzj0VKUeITG2a6vBhY0C8zIaox5SKiB\\\\\\\/YFVuqkqJNPGtwfTClVkBOFl\\\\\\\/JbBeDuns0r2mj+4BdyYYVgdgM4lccQkWm6gCnyvXRCrjyb7RCuNGSVCHIUv47KH4474+jpac1voCuU9wK2kOuLK0oyaZuChNm7aNjf8XyE7PGfx\\\\\\\/UZe0zDeD0Nctt+WydNm5S1Doo0Ki5a\\\\\\\/dWgwSPGUFz60C7F9jtqEuvZNxkbqyz0Xu4b50w8uaXKYj5SfP8Vy1toRUoR\\\\\\\/cLNsLr\\\\\\\/77HmikwNA088f5dIOzY7LsWG3\\\\\\\/wwyXCEzKq8sert7cQCDLCL2VgBtd4Hjya+88VxOyvW8xWafHK\\\\\\\/va\\\\\\\/XFrJ8qUfD2dmpE0Vj4FAPCOY6ASWbv49vxVq0moIXFLIhmaDMNJYuWsxzrHcQP8MIozIll64SNewhVBQkCc8lr47LvqU1nCbNOjTB2SWQke1MsIlWUqeLV4bXU7MOj8vgk\\\\\\\/6bZwf0oIrH0wHjmLbXT\\\\\\\/lxyJhMyArRp36NpdNG38lhnfzcQkZzCiwG26I4Wfs4L7sPzE1ZvQojVGgFSRnVbvZsSf\\\\\\\/uWeZGccywdbFmS+3TEKZTLveF\\\\\\\/xzMfJdVUHdvcIfcRoqXUKX5c1DGiXTQsUTGIVooSN\\\\\\\/JW5lYET1CAUnsKO\\\\\\\/aIBRPVnRGoOJKCoAWAfr2WSyu4ldcvwDitbq6Dfx9U9t9f4wa1aN0jsnwwwRFAVGkjCBce5QriYpD1Bmj9OA+ejTl\\\\\\\/uMc42jTqlESzLoTTWDAV+Gnc07sbdGXeAuW8SYoeJEHJjrSiD+QwV\\\\\\\/4YuF7gf3eeF1mOoVeOotnxN872dtqQwyPmTHFXKKtDFNbzx1ri42\\\\\\\/el+HJjP8rC\\\\\\\/DI0MDJhDE2tGb7K7j7gXM\\\\\\\/hh5\\\\\\\/KepWr22WhDugrpPEyJqyUFvXz\\\\\\\/1iZOeSjwLoJpcs+rdh7F1N\\\\\\\/cnoKAlDNmZgqLC5QImPBewkLOQySPgIFh7v134QyAo86vomF450zVpOx1vc1nJSTElj0o792DbbQWXH0g0Vk\\\\\\\/fDFSgZj88iLCKg+QFSNQjyCFwnMGZyH48ucUHxf+Zl4aEKMHIx8Ci1GDw0ZvhZUDNR4vITVApkOxSILinhlUK5Ig6QwMm6ITeobecEeQAY8EUYivmuzr3Z86BaKXqp0Uybu1cnHSF6WLoWJAfRzmD3cNhx3pXJu98zfP5pPn1F0htLUaxqxPmpiUT5r1e5ifdrA5zqH3Ly0XGX6sJFoNlbxN0MypHrk41SYmomY758dD2mxLh5a5LE9lTdLCNOSO35dmQ0cCTiHkAidiQqTLp\\\\\\\/GK7cJ+AOHNVyJCeBAIxtPca4diclxS6oRVqvC\\\\\\\/mLP6YLht9EHC6ot1V5mqpM2h2msWCK+Li\\\\\\\/mKNN5Vj8MPyYJPUAK4+p3iDtqteZrc6VthPUIMlt6E1rmwFAodCiToSOPCWlw9JBimS\\\\\\\/CF0PapGosQF+esXGIxCYEb1f4HcuzgcM6Gd+D4wvIal4W994qZmb\\\\\\\/cJkX3Alaih4VfLMkUQ4P\\\\\\\/vgj\\\\\\\/gPq2LFaJo2dY15hDcPDuFk7YeCd+J7QKUApxQ\\\\\\\/2GZffOYfvEUEHSlHCswrVKLRNVQU\\\\\\\/4nbbfOepjAN8x4PONHW+jGDCfCbSRxsKMvuOEgKfAweDN\\\\\\\/F8sYhZCeC1e9JGDQjkK+1OQQdKZru2x\\\\\\\/1iaPNQo3aVan25\\\\\\\/IClIoqbcLK9EH5\\\\\\\/KKlwEKVonNptgSZFfSxrKU+a0W1fU1Ui4y87KFHlKUwqZATmc9je+BwtTaivczC9zEAdI8UtRXTskt85Mi2SITPoRaJnaLq+sw\\\\\\\/8nPB54CbdrnMu2fIlI7meYwUC7edPqEi8i82QS0be16YfabhIkFpux4abmty439ZvSZgoCHEBCDilGANoYlZuUk+iFbzJA0cUsyFaV1yTYfk6IFWQCg8+vX1hXjaZlXhmMw3rgcKzvT8ok6prkKz\\\\\\\/n9Lwp0wF1ewMPZxgP4NQWD9uLqxrllZqK94LNAA5jpw9SPqeIqHoL8+cN2U1IT9zLrUDYY+U6mCdk1cLTwwrIbmLQ70WqmF1LoneT3XevdcE9RCs0cDwGbUIiskyo\\\\\\\/G2\\\\\\\/bhzyObBcbPGDq3HVGXV3rbQi1modQjv2yafrMs9V1Awsn08qGLZzOdgXm\\\\\\\/H4vOLjCm7l1722zS58WpH7ybTZE6gACi2gwNxHmfu+5igWk\\\\\\\/CWq+IJ3t0ja0UyikXlu1u8\\\\\\\/Ol3eZLH\\\\\\\/1S\\\\\\\/fyB6o34t2nAx3wP8lxLUYLu8MzXDFDGeReONF41hr9eRocVhiZ8iqffYccKBR0n1rUqPgzbVB\\\\\\\/QpHT6KG8LXyH\\\\\\\/Z71wZz3iKtZX4Vnun+iu4jeXwgX4sBnuyJmykgS2PacOG7vte9JSq\\\\\\\/Jsj6TfoU9aRXuN9VPg3ICukdygBpm\\\\\\\/3ekSscG+Tw5Bxdk2NXUKvey1U3pbZi2qYJjJuvYoB1imh+emlhrHrwdTtnP1rjzmheUmViHmMefQ6Q+0F1slRRU+SbL0mZtKB1cW+wXrKL\\\\\\\/2GGYDvJ1fQXVeCpLRhAhP8xTdQ64Gx20yC8QlwZyuSdxGOZ+PGIXrycUugAbpcXuf\\\\\\\/76IrL96CR4v9qqYXPYX513ceyS1h5PAdWy4mzWKMiFUqL9a++VnHNqHSlw0ypGVEsPGNRthO+CgISEiOuvV8kvHDgH2mxyJBYU639t19n0o8JrUlyLen+RZ1AoHovl\\\\\\\/T\\\\\\\/EWQ+B94xzFq1QjTxPE62FLyy\\\\\\\/O+lBuZPzo9IRN4eaMc\\\\\\\/XZDZZvAFWhDhjDnNCcylR\\\\\\\/z0EjqjpsoRWZ3ej+1E7S6luJz7gPkcwm1YyR\\\\\\\/mJx24bwGXWAUfgBeDocrK5oKwp5N+fKEvnVrCGm804ytre\\\\\\\/wnidrtChOraajGMURMXLZJtDEbT5YYq8Svess+xbVywzelw0RzQ\\\\\\\/2eA8JO121Rq64mjaHgiHcoX2oyh2hIngCNU20RhAdQqasCmY1FdEv7lDEmiFd68h2G9iy5VsIeX49r5WhciUjmjrIjnKwo6pKP\\\\\\\/Yr+uagRkcWlhY52ri4wwrHnFXnMjK0ChBH7Ld72hWK1xcEy6VN7+QU7G9ZdaOU1SZ5RFYOnxGXjkcp3Bz1+2KIVKS4YryTrFVaME30hVORt0EkGDao+uiI1dpPiK7LJ2OjvNzH5IaZyTo5gWbUPRoV\\\\\\\/80O86gA6zdO1fmJ+uJwGXQrYWpFDZHlBcJodCh6XYGsGHgV4wT0AHkidsNh46c6YCk4Fx\\\\\\\/Y9okm+REE3lwcSWzXjzNDEaF\\\\\\\/a4yAsI2n5t0AzpUww2zsegIx1CrkTYklKO8C694ccwJb8\\\\\\\/8bPuBslJ58ZsT7o6ZzqrnCgTs7WxPKspp5+6bTFP6SYowerkuenFB\\\\\\\/zCipxvk8dRMy4JFs1Il4W+xOMYya30zHrS3oo6EkwwKdkdgYTcPmoq6pqJBCSROLwGIMo6DMqdiY0cm464NpN7wPGxYAmHhUi4MGNf0ad1S9wJV2DoBeA0ZGSILE1vmkbZrPnFiUYeB9Ad\\\\\\\/nbFdi6\\\\\\\/Lr1vyU6GdVqsUREeFaNbcAv6GO\\\\\\\/wk0VUT7F\\\\\\\/3ZMd+7pAc5hpZbGF3Wt56JFsT+UvXyj0MFnck0gCK3tEnkmi007vYkE0LTreBpxv3Ez8d+9yZgpbREGBfllZLNxY0cZXrf26kIeBK3NviXvWmesPQEaB5W5RfSGb7bOa\\\\\\\/Rlo+dY4qyhQjuIRS+chQp3MOofBHGATTwQ8EB+3BxTaJwL3WbbcaPRjvJCDuAjd66TuyClO1aZmmRUe+DRF5W2RKLPjHTTi6aQqvlVvUAfmEGgwhcJpfL08bHnB9c9kBbV+dPpfkeCl1PQ3\\\\\\\/b6KLbht8U5rVV7Gkw9AA7kCIJOcTmtJAvf62a1JHpM5kFVZFbktSaDmt+Z7eGTTha2Ff+ZQ2qDK\\\\\\\/wwnHbfPft9z3Br8xzyWhNGvzCpu\\\\\\\/kpkHuYzX7krNEUyRos+GOglT+RIoh6dhzMj4iPrFgjOWsi8Uwgn+E1uCOTintY5V2o4FKuUs1dHL1bF\\\\\\\/hopmSHUS8SsmpxnAOze01q3oPYlTam0bJwBD1JBomZLQNMXFbWEKwDHmY1EOelgFtXEaFGQssoXJK2midkb3IbMjy4bwp9WVxM6pSG2F+5gx98r1LMtGcfJhFDfVTYr+3enhkAihm6CfQy3i3nR9t2EAGobM3VK3w3qicuWE8fDuRiKh9\\\\\\\/P2c4RNnowajcv77BgvCYQeezH8+3M8OQG\\\\\\\/dgNgLRSp7dClgn1bqYyesu8A\\\\\\\/m5u3H3zUiGbH7wM3SNu9wdsmra9e6uOWf6b5B8kKQ6n+jEjs3\\\\\\\/QOxGGbgkfkBosgAVZq9whGDb4tBa0lclsZXrdJZHT\\\\\\\/fIOi\\\\\\\/EXMubHFSLJDY\\\\\\\/+fO3UIbd2CdVXyPYoMPKJmpstngxg7+vhMT2UliuY+MaiJ8\\\\\\\/3eLc2J\\\\\\\/fFwHMoA1CfLdnpBgES19k+BQwABh7sUUHVuEvaVjFr\\\\\\\/2\\\\\\\/bOOSkN3jsd5rsCXAiweyTxjYGRh80RF\\\\\\\/oj2MIR4ZH1Ubbw5s\\\\\\\/mU+EPYPF6Rl9CIrZdvECtG1Er+VRhY1qOAyns8E3HbVAlrfB6BNgxBQBB2JTqUiuNK1Hr2+LZPLBUlK98MVJwJgQswFPcrorqDQIdEBy+00G8ljoMC4zZRI9mJw9zX99Vr+P5MuQdodUPI50sK182xWJhUJk1ZI5M1SUb2K5m6k4lTx5moqew3703uua8LMnG7Cwl52tDY6rmhdpx5tVRjawHg6OxfGNu9qFskHr84\\\\\\\/WZfeJlFuhj2VZM4U8mhjNFe0ciBF\\\\\\\/NZCCElHnUOJhOI6w3Fh9QtBxfKvUbS18sStQ\\\\\\\/wPd4ELMQko5j3TXdLU8C2lrkFXXPEJq\\\\\\\/TbnqfiGoXg+UYUSjbm3\\\\\\\/tV2xsRSTiTNHv\\\\\\\/1VTyCeYv5vxORR8S1A7DAmjpDNiKXzS6zETuRQGQTg9o4ZVq1C9AI7TFoKi4Wa9NU\\\\\\\/gba7XdiTpSCDpfMxsEnQbdwFseFkG1IHm9uRowOkrv+Y5mBT9DLZT+tyjXdiBR\\\\\\\/uto8vG1zuoB0yplMngyMk6K6a1X1bf2P\\\\\\\/eXxYU7\\\\\\\/J4y8JNrslqibN6XQgXUnWdKeaIdWh28PeTQMpSoZRgqoZHHbtry1fZ5mfD0o\\\\\\\/hWju9D6BNSOxHgUGgDSVi5EMm4jCbViOQRWiozkR7lXdpVF5ZS7EKsryICTBfkgSb4Q6HMtCWbbpjcew27HLm\\\\\\\/wwMljz1A6PWo\\\\\\\/Bks\\\\\\\/UWfU63DPN1tjA65Yrmfa20lOKvYlNRmwvTDo3+vF3Hxh9ZhAGxKKR7VTA+6MZiwrvjn6tmFJtyZnqAqdiO+k\\\\\\\/xuqnUovPT8FFS7EFY3czsA3OSn8kmp8YXI601AoKDqsjHSnGxJrWvdc4fZJqAAe3GWhUDZ4rS+pNjFkQDYKZS84AvAbnmOqiSghsoW3FsLewV+T6cq5uNaqdb8Q9sRD\\\\\\\/NEB6QI+6Q6lagln+sKAr3y8+ksKaZIypvyjgE33abDHyKPs7cwkcIEJgD4XSIGibXnMhyVZtQLj\\\\\\\/K4GL8S4+vifNo9CIefY6qKCyCOLx8xW3bb1wCSO6Ujy3Qn\\\\\\\/Dv3sVR9NQ7AnFGqg0QhSP75jq56Xk4faIySqNUPjzMik8SY44dYd\\\\\\\/M3w2Y+oBrpwHdI9VtyjEt8g8pj+hTDTmQDpqEPv8PrrKWmwKwr41qK6z4xBpHbQ4H67ORTSOCtr3mEZc8s2kzYWUXig2nOuqB6KI609sLoscjaXiax0AdWI7YnGyhojq2jK6vYoET9X5Pp3WDUNNBoTAPHdRABMvd+X5JqBq4sskqP5X+rOSXNUiJd6v7nfb8opxW1m4gIooPb1a3pegZTMPyffiDHiowEXU+Zz5fKYkUDuvgR1veqmjC8PD59HcU4UZWKDdD9tnF77IdpM9ZTuqtfKuZGScyWfdvhEHE+h5guPTPBB8u\\\\\\\/C3WWEOyyfjrZW2U8UavBXbKnsJ+B27uC0MIfOhlCYpFwzHSIQWJJmQWo31XiuX3oFRdorA2hAdfR0z55Yy+SrJnTqd\\\\\\\/ZpRNVAkpPgjR9mAhJiPFy0aLQgGwuhtbI3fvyOjz0\\\\\\\/iuRy38G24z99mD6vzsCwoR4M\\\\\\\/VnD+TDRvoujd2HcadXV\\\\\\\/A9YeHvFBw2kIJIqS5A7CRwLUAFPWD+ryBetDeEVjDjMFJl7g\\\\\\\/HT4MEnGQfuUQTu5eNndnFBCI+0UuYgR0C9z9ncCumkYu+WE+5oNlewxr8EJJhMxjjEzPtGxI66h0VK5N1U2RD6Ev2zKRnuSvi4x9ELXwRx5EfKpUVz7qmjm0A3XOP0yPinGYBie9KGLE8ndGelM9tcnFAnzMbaB9EwM2kZ4xU6E4XtnVlvzOXD+YzGJCegl3LJwNFjazdxeihA6jKMvfc28FePfGmsXQ2VPkGWF4y0k6DXsX3lHs9\\\\\\\/S1nCXqwWHLXKeZOLUgIwwMvfUMB6UJ9eo2\\\\\\\/CF4eIAhVKyYBbDnvGW+dJDWKrLyEI5ZB9gWXXG7XmvUspeGq4G5XhZ5KOOhSpVOI5Wt7jckdJDayjw7BVFuDR9egIrKRYxiiv7IcMBS1sHYXsYu5uorlmn0Mxy595Dv1jV3zIs6xkTg7ghnxblQh8jcZunCcqRntXHeTrUt75JbvKI16oz\\\\\\\/9FPZSsH1ucx3sgZEgC0CfLGfuJ8garPh\\\\\\\/fopZgvHdetNPLJ1AB+8Pv9x2u75ygd5F+gKWbsmgJXqU4ueT3cZrsLlsUXqK3P92NGlqxyo25XJF5ahGd6PkZXFOE5pum0rLEO+WdOH9eIRDs6A7yxYdqyw\\\\\\\/Nmn3RssD4rKpmH3BB4VCOGPVI9z+fllCoQ\\\\\\\/VDiXeUnH1ocGX8OGJ+GCUFdw8dTfahK0ZLtZpXxnQv4ihB+yGRWa5C5R2gFnxbmnmNLQJbmTjS5fLDMD0CH2UCbEOjTDMB0drctgDX3JXeL96g2t4DLs3Wh1tyonf2z1c6ZKtZEpxLnMTVOra0tLr1ZSHkQ4XcvvL2UVUoqBmR7JjMwDfODwZ9Yl6BEehxcEHgWZtTC7HGRWjsCFBjU7BVmmTCpVNoBt7HezyBsRSBBBijypBxyBLN5muRCbQWBG77wfqQddh2BaPkcu8s2PA9mwrVnsAzoWkbFwJiFm\\\\\\\/dew46kz7Pr+HxDJO\\\\\\\/HEX6PYV2eaNkGwE31qCNLKkNel9bomFk1lWuNR7\\\\\\\/+uVcm+ctWqd5o7g+CG3lN6HUY+Zni1d69\\\\\\\/Kz7kMz31rfpbn5eMC6cb+uKJnxsZ2j4Y8tZVwS3N3nv4fhJspzwSlY\\\\\\\/8CmqherP5vE4pw4z7jcimEhShXuiT9OE\\\\\\\/pzLbKcJwbfMLITh9\\\\\\\/xP4wtBocwKF+qp3PVrPjrEDCEIF8aFzox47PevxcG534nDIsOQh9i29V4Iw2AHqDVLjacvBBlbO5BF0JwLo0v4Iv6dpPAO0SNflkCFj0zC2EjizYajemwlYL07O5fomJUwIJjZdvdDFSV+nHN05oEIJlFWEAdhMw7p5OgY\\\\\\\/nnNayhcKZ61mL7pTfII783vUbTmi0qNDZsybmziGzc3hDsZ0fjCE5G7ErwyCUT7HVJvg8ye7m77c4Xc6VF7NtmigKyfzyyuQwvKYZyRXQFz5AKPs\\\\\\\/a\\\\\\\/5qCcUjvOBMOxUr38JPJX+i+DEgByZzRvRyEAysEPTb9DrWIEHSr9a5oJnrhkkOO4HAQ26xoSt+BGgwAhY5UIMh1E3UIrZU2W3GubcZsKDdLDRYLsSVsreEJgLm97APrpSsc8ZYwA1SPlVa+tGQ52inpCh2aEu5lt0lk6Eb8ifR8HhJlYf2V0WpcNXJG2klciA74OFxm2jZmdXypb\\\\\\\/iaxaVofMGi5LtCqToAhtZIiORJPswJ0S\\\\\\\/CEdPl6CzKryoYRERrVDaGB0fiHZN9OTgRJ5W5d\\\\\\\/4Vd\\\\\\\/OlPhcWaw6azu4v1RYgpOJtYOa9PB7prjttOe19TNnM9KW\\\\\\\/DfZfrs3kdzvhiNxcKQrZkAoKUvXjPDY1\\\\\\\/ZBgIJBZUSU6dxandkl0Ys1lVJoE4Fz+bNUI9q9HFqRWNdih88Za\\\\\\\/mmN9hcbu4RH9+tq05yFolKs9KFmJC7ZWGhzP1xnFpnzQta0mKRQGouJzaz0nwPb1jJekI9j76P4OADpnjPWzb+sTBhGKtWdn+4KRUadRt1Dhp7zd3GgJtfqG0hhgRfo6BavD67ZbNegqK1XFOIXwGDtisuqiKMQWLy\\\\\\\/g0omMOKscoHvBlrf+d2ogtNhPmyOh5rYyTIX4zVwP16riCTE1Mhm1AueW8NfSvwhwxDxyjCRf6O78zKNcwGXM3hZazybetrVELdr5Y729Xof3+SsWTgV4DWl00o9\\\\\\\/jCBTQVgPf4jq0x\\\\\\\/aJArli4qsewoygHVzGV02XO2hwzkbb+q\\\\\\\/zhkJMLkGhhcqEMenZmujMX4LmnrWAOkqAl3FjljIeI7DaPSKVjjMJSikEQRYF+9esKwLgaOt9W4KJv8xm9v5JfkTM38sbjywV96SPLFYXLNgX6p13SzssfuqKYnmPMTwaMVaa3i6HTonHzfKijT15Jne+7d23tftKJGY4MV6E35rKJiHNXf6Lph+91ILJ4Pbp+G14pnovaK5okYAj6PjD5eQyh0D7JaGjfNYQV0w3D8CtSmKigFwC\\\\\\\/Lq\\\\\\\/ugU38hjUvtWB4MPZTzwJ8kWc68nW4HY6V\\\\\\\/cpx1MZCy+N8TvG4P2\\\\\\\/cCel2u83frREll\\\\\\\/UaZ5tLwNhQ2JdzPjXzWw33pmf5nlKhmgViLg7oCwN9sejV36NP3PI74zwoUNnhiIo63hinw\\\\\\\/nsTwR0cBZNhAL3WMYBQcYBPVdnu6tlcHd2UViZMnAHSUe1theJM9E4d2awW2I0rjsQZ4x\\\\\\\/TXQnPMxaOseRSgLUOs2l5kqAxQAVXYD78nRVS0UpraGDUwRLgWnDzWXZd6Y5Y8vzkqkrF4hTEsT5wZOmbDWwx3nnAvWmLfLjvJ2R48MHeRB7BZBNN8bjY0VPZZT+mjyV6knQlcoAeJcH6PKC5vB9XF6TXLcPI9vPzkQt+7ndMLX+SmTwQ5oXlAFuIwTmjaeB2DOocwW4TF2vMkIFy8HDgH3fEMu2SgQHGlBJUFkfPmkodTzyLjI1wARGmf79MZysjOTD6BtJjN3O7BfagnMpFpAkHcqFrEIW3cYzxPfvf8E+g3H1rEOD5SEMAdzI04HZ\\\\\\\/UPalPf86cDhS\\\\\\\/8rL+X8o1RFDMUQfEjq0JEP15WUQTndHBXyFbgtGSwfWUptHJNamJbR\\\\\\\/3o5MsMwqwQTGBVy5wp2mUBZmwYPWbHzQ6kt1FgEgmT1FjCOqZc7+8UeNdx3zqUl26A5KBlMB13WUiM07SNS3aqJD5JTGo0uUdYiKRyNnFG6Kj8o9Q11TEUV2EpzEwIoIOgJ3esRnTgpmdYWCnjUF2BmfZjkapAPMATr0nl9fl9B4DwyCYFI4DtF6anuZQwGSe5Gknsacca+wTKYcWCBfW7fPCox4UILNSYdkJKR34084NTqsglnMfvYvwrRLyAfCmu64IHVOY0GZqwGLRJj1pi2G5FUWvxCxHPtldN7ABgUekRQ1o9W6o7T9Q53AJglj6X7WR60rtNLEA+fC0+vjNkeROSsdcKGebf3PYm4A+2IvGVp0HYNxrIUm9cJuUK693CFWwDynF7a\\\\\\\/vTjBrVtHw9BhZfWUcQt\\\\\\\/1agGyXoMpKmtiewO6LGDc+VBXQ1aR4F21kaWGEk0SWIISS6RZY4fm2rNX760Va0YhGjoLPFYXXR4Q\\\\\\\/u+kgx1D\\\\\\\/+HS0KRE4MBWGVbhiJbV8+tNBfmvgU9Tjqiu57SK12tnO5qgn4miYbOjFnigH2RdYLxUOyqxE6DVK144DuacA1A3nMIy5e3GYuz1ofngEuQQ8Uczn9IS4rfF1GBHtu0TJqPexWyUic35s4Dgw97b6X552a3TNKRPPeQSrC3vqmo5Q\\\\\\\/A72RXPp8kwIVFNY8kkps0ZuZ7ApN3WUEak9cjhTvW+A40zOrcV0FGl4sbiGo2G1Ovxi6DMRd59xOdWTrLc6QVLckI1EZfQJkzVkQootxR13QGtftYN4qeTd5zLVBbSLVybbkmZp8spSpEnbPl1HtGgM11WN\\\\\\\/dzzVmy2SgcEAKPGi5Sye9V4r1D8CzKAPr\\\\\\\/ndfZeBhQWR+oszLLR6g2nbCvBTEpm8MvvI8xekXWXaFIbVZvAden1SEiSo2kAfi+BkGFRBCpLDFrajrIQb3RUpsNjNWL9OqwPM6MNHfhzbMB8FTsg97adi0UE51pEw8QeXS\\\\\\\/V\\\\\\\/G7roIof4Y9kaPfVjDMqDRH+VM7AF4y7S3AbJhFv9Qre8wNXJlgnGtvcoyZeVt9xlwakiQ7qKbZToviXVr3+fWxNxI8\\\\\\\/L+rIcWzm2OVPxSSwbZllK1qn3Dd\\\\\\\/txFfTFMluatLQNXtaA2o7XM9VWqqU1bITpsSydniOu9VPGvpW8e\\\\\\\/ebY7QUQYxYmygkfoB6O6uy1fVfKMWT0BwSe1+Qe4Mn60SGa2pJoG5wAWQJOBipFXfcmHxNZ95CvJZvRLuLwEVnkn42NQ6yvt0Xvii7ckUdQXROhEjicO9+ouWjJF2yU\\\\\\\/YfKFXnSIA+MiFqCUMeD4AthJLeyXKsQVK9sEWuAvhAN79QeY5XHrxYCBUN3tmXDra7RES20NUhohrLdy6iXU7qSPvLgTjr91zNdhQWtx5bVo6NPUubiwL5+3IV6cag2KrXeJFlH9RVvh1tw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vZqDeAI0pBseQwZrnvfi0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002272cb193e847c7e5b467ea179fe66d594b119296976ec4dc5c8365a8950dbe1d8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZNf6Sj+OjKo4ZOPdOMToNATtt3U8TICpr9UUxe6+0rut31jjffb+yLgyAUlXYrr4fVjm0P8o\\\\\\\/hB\\\\\\\/BCtrkDy\\\\\\\/AcJu1TigaVfYy6+sDtnb0Uf1Yo6YTCEO8cbrBdMDQixaJDkXEWLGFZ\\\\\\\/izPns8zqDiR0VlSWwuR0Mr344MmkfDn+iMTxo1LudaQDe9uJm4EkZleYTjrnWS4Z0o5nUmp\\\\\\\/Lt\\\\\\\/yPYofEI38+VAO2uibxUoLm6MTGJ6GZwABnPyTXURXqOv2qtutaaENEBjc1BM8wR1gm8Sw4x8uO5iV5RKJGo7kcJTnmBIPIMEOjCZ\\\\\\\/aCfq2xnxfaJwnfSDiu\\\\\\\/PjP7I92gv+jAiFETmF5sbOQGqXoYDWf5RsoxQDuVNEi14+16gyeqH7GJ3xFYY\\\\\\\/VOKGGARUR1gs+kwQ\\\\\\\/m3seb9biKOz5duIkbch52i00Rlke5gIm8jPy3PV5gBXrMCnbDAYjGn5U7BOlS\\\\\\\/9nql6NRGEN7Nl29lOxXd9Q+bfYlb1u0eguIzxm4OJjQrSz654wo3BnnNUqrkamZ70fG+gtwXwrnfmCteOZVDFGgQNy2VBq8XO7UwnUubUk6Ug0L6s\\\\\\\/MHQYFAKRU6kyuiHD9qAuc58\\\\\\\/ko6401s2QSzQ3iw6IDL\\\\\\\/Q1NZfs1Gp9CHAWD\\\\\\\/LCuNBdllaOPR7jtDjJffX6s2iN+GXCLQQrqz89v4djwwj8h+rlmUZBbBR7e7Zy01JCjyGmRpURFVkaC6iVpkr71tieMVYe0lJ48wwR38QdKt\\\\\\\/fZW2sm91ETvO\\\\\\\/Pt2clJ+2AX5n3rBaZAZOVfB+5hf6thJpBsl95MOVWfxX8VcZYSp81DNbR1NmNXZVLPUIBv5QEoyVF5goIK0isLikO8PlSQVbhTZYglw97ytITsCO8SFIpdkKPG9oAjJukO\\\\\\\/tGom25cb3RhEciUjlE4yqqctFt2ivMAo+HQnzKCgYEFmcDo++a6+R7BQIcdvmwvCyK\\\\\\\/jveUHC1mH4dGLHxoOEpCFWSnXcCOZ66FvvYyLCi8omRifizz8FhlNoNSm39iA1C88abhjkzgExVvNLefadC6DhBFpvZ9I95MR4YXvLpYJ3xzDb2TpeKsFHt5g8HFsPjTaeUgTVEN54tXGFE+hO9toHRiDIYCSxYFckcTc30KkB8xidKmXlXzll5og+96UIIH2GvvRBzPjkmjddZrZl5JslY1s686eoLKawaEE85jJ79N0F4jkhKLlcGThGrmWf6NOB7VHYF0rQZcW+f4q3SrGHm6vqGnysRTZ0zNvgit09mzAm70AIl+WfyC4JtJ5EGpkF8b5CpjutItoy+zg\\\\\\\/EnNuIzjcH\\\\\\\/sYU9hdTW0AZxK8WEh3EtLKBZi9F2Q71k46fwVtv+\\\\\\\/JFsPEabCI4CUhC6KG10kj0HHvzCOMCWnB\\\\\\\/RjRma8eDk7\\\\\\\/uKE8igXvUWt3pb7PSmSGdDHR0HGPgjnI1M0jSK50qD+hQSuO7CfaYvMlGDLLvJx3qEn7p7jVSV8mox6KgCnJ0JBnaMZCYOdmJzzaJHrBVJZA6JIUVPgTdMmIOClQVYhW6MWNPCu2fcw+2Lq7Ga42S+apNSj8bySz4gu8aVjqcu\\\\\\\/J9\\\\\\\/0umk1EN1C\\\\\\\/6GO3B5MmpQWXcxHXXdiZ4s\\\\\\\/EGmqhYuKLc8VTpROxi2fvoxsH7d\\\\\\\/Fp4REvWDas3tmEwwGAzGWDF5Pifvr4mg1bWFvgumVFieV7LrAFmRe3Ala14FbPFdbOed+UYJxxiObKXOKDxqpDTtz58cw+d9IJuB2lnLM\\\\\\\/NiD5Njr5fE8C16hkAdVk56ssCByj2qHqVX4qbN8mJqxZBbsXftXsG04GKaygmrImSFreQT5jsH0HHM2M3NC\\\\\\\/Lt+Pg6SLz2cUOEZEBQCXGDhnD\\\\\\\/1G5us5bwQgUjIXpzHQ5MZfCkIGApsn2nn2WdEwGR\\\\\\\/5jyuxtkrVkaQ50QiNw8IO5RzB+3PsrguLChP8HOOMAuXLJmLUuLU6b4e6SXE2B2SUZ9bKVd+VZZdXBlSTIaNhw5hztbmgwK2uk7xPy5FXSNc9Gw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wO4H6vWVjsJS4Q8IZVhY8Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227efbe2d00fd3536033824c1fdacdd15b1cde599ad70f85d78335452a0b250cf4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490464 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2841" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fv0-RJrbRxWnE2zIF8LTApajX0yH1PtnCAPqbS37mvQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2841" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4N9bCWsn5Vf7QtGOqbf70iim7v4UJDx2QxDuRacUwWM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2833" title="Generation and Phenotype Identification of PAX4 Gene Knockout Rabbit by CRISPR/Cas9 System" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2855" title="CORRIGENDUM" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490464" id="node490464" data-pisa="ggg;8/8/2841" data-pisa-master="ggg;g3.118.200493" data-apath="/ggg/8/8/2841.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in <em>Phaseolus vulgaris</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rafael T. Resende</span>, <span class="highwire-citation-author" data-delta="1">Marcos Deon V. de Resende</span>, <span class="highwire-citation-author" data-delta="2">Camila F. Azevedo</span>, <span class="highwire-citation-author" data-delta="3">Fabyano Fonseca e Silva</span>, <span class="highwire-citation-author" data-delta="4">Leonardo C. Melo</span>, <span class="highwire-citation-author" data-delta="5">Helton S. Pereira</span>, <span class="highwire-citation-author" data-delta="6">Thiago Lívio P. O. Souza</span>, <span class="highwire-citation-author" data-delta="7">Paula Arielle M. R. Valdisser</span>, <span class="highwire-citation-author" data-delta="8">Claudio Brondani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-0644-5652" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rosana Pereira Vianello</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2841-2854;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490464" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rafael T. Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rafael%2BT.%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Resende%20RT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARafael%2BT.%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcos Deon V. de Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>EMBRAPA Florestas, Colombo, PR 83411-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BDeon%2BV.%2Bde%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Resende%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BDeon%2BV.%2Bde%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Camila F. Azevedo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camila%2BF.%2BAzevedo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azevedo%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamila%2BF.%2BAzevedo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fabyano Fonseca e Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabyano%2BFonseca%2Be%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonseca%20e%20Silva%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFabyano%2BFonseca%2Be%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leonardo C. Melo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonardo%2BC.%2BMelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melo%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonardo%2BC.%2BMelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Helton S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helton%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelton%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thiago Lívio P. O. Souza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thiago%2BL%C3%ADvio%2BP.%2BO.%2BSouza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Souza%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThiago%2BL%25C3%25ADvio%2BP.%2BO.%2BSouza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paula Arielle M. R. Valdisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paula%2BArielle%2BM.%2BR.%2BValdisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdisser%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaula%2BArielle%2BM.%2BR.%2BValdisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Claudio Brondani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio%2BBrondani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brondani%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudio%2BBrondani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rosana Pereira Vianello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosana%2BPereira%2BVianello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vianello%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARosana%2BPereira%2BVianello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0644-5652" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rosana Pereira Vianello</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rosana.vianello@embrapa.br" class="" data-icon-position="" data-hide-link-title="0">rosana.vianello@embrapa.br</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2841" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490464" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2841.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490464" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2841.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490464" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2841.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490464" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2841.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490464" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490464/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The availability of high-density molecular markers in common bean has allowed to explore the genetic basis of important complex agronomic traits with increased resolution. Genome-Wide Association Studies (GWAS) and Regional Heritability Mapping (RHM) are two analytical approaches for the detection of genetic variants. We carried out GWAS and RHM for plant architecture, lodging and productivity across two important growing environments in Brazil in a germplasm of 188 common bean varieties using DArTseq genotyping strategies. The coefficient of determination of G × E interaction (c<sup>2</sup><em><sub>int</sub></em>) was equal to 17, 21 and 41%, respectively for the traits architecture, lodging, and productivity. Trait heritabilities were estimated at 0.81 (architecture), 0.79 (lodging) and 0.43 (productivity), and total genomic heritability accounted for large proportions (72% to ≈100%) of trait heritability. At the same probability threshold, three marker–trait associations were detected using GWAS, while RHM detected eight QTL encompassing 145 markers along five chromosomes. The proportion of genomic heritability explained by RHM was considerably higher (35.48 to 58.02) than that explained by GWAS (28.39 to 30.37). In general, RHM accounted for larger fractions of the additive genetic variance being captured by markers effects inside the defined regions. Nevertheless, a considerable proportion of the heritability is still missing (∼42% to ∼64%), probably due to LD between markers and genes and/or rare allele variants not sampled. RHM in autogamous species had the potential to identify larger-effect QTL combining allelic variants that could be effectively incorporated into whole-genome prediction models and tracked through breeding generations using marker-assisted selection.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/common-beans" class="hw-term hw-article-keyword hw-article-keyword-common-beans" rel="nofollow">Common beans</a></li><li class="kwd"><a href="/keyword/rhm-qtl" class="hw-term hw-article-keyword hw-article-keyword-rhm-qtl" rel="nofollow">RHM QTL</a></li><li class="kwd"><a href="/keyword/gwas-qtl" class="hw-term hw-article-keyword hw-article-keyword-gwas-qtl" rel="nofollow">GWAS QTL</a></li><li class="kwd"><a href="/keyword/dartseq" class="hw-term hw-article-keyword hw-article-keyword-dartseq" rel="nofollow">DArTseq</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">Heritability</a></li></ul><p id="p-3">Common bean (<em>Phaseolus vulgaris</em>) is a leguminous species reported as one of the most ancient crops in America, with a proposed Mesoamerican origin split into two major centers of genetic differentiation, namely Mesoamerican and Andean gene pools (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bitocchi <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Schmutz <em>et al.</em> 2014</a>). According to CGIAR (Consortium of International Agricultural Research Centers) (CGIAR 2015), approximately 12 million metric tons of common beans are produced annually worldwide, and Latin America is the largest producer, particularly Brazil and Mexico. Common bean is of great social and economic importance, being mostly cultivated by small landholders as well as being an important source of protein, minerals, fiber and carbohydrates (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broughton <em>et al.</em> 2003</a>). In Brazil, the common bean is cultivated in three annual harvests, producing approximately 2,670,735 tones on 1,840,696 hectares and estimated productivity of 1,451 kg/ha in 2015. Among the five producer regions, the most prolific are Centre-West (2,036 kg/ha), Southeast (1,621 kg/ha) and South (1,590 kg/ha), accounting for 19%, 29% and 40% of the national production, respectively. The Paraná (1,599 kg/ha) and Goiás (2,438 kg/ha) states are large producers with diverse cropping systems that vary according to edaphoclimatic, socioeconomic and regional market conditions (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Souza and Wander 2014</a>).</p><p id="p-4">Agronomic traits such as productivity and plant architecture are determinants in the choice of high-yield bean cultivars. Improved plant architecture and greater tolerance to lodging affect harvest loss, disease occurrence, crop management and mechanized harvest (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Teixeira <em>et al.</em> 1999</a>). Based on a study conducted over a two-year period by a Nebraska dry bean grower, the total field harvest loss for conventional harvesting was estimated on average at 49,5 kg/ha (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Thomas <em>et al.</em> 2016</a>). In Brazil, grain loss at harvest time is generally greater than 120 kg/ha, corresponding to ∼8.3% of the total yield (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">EMBRAPA, 2015</a>). In this case, increased yield at harvest is an important selection criterion for the development of new cultivars (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Thung and Rao 1999</a>). Genetic control of plant architecture has been reported to be predominantly due to genes with additive effects (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Silva <em>et al.</em> 2013</a>), while for yield both additive and non-additive effects have been shown to occur (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Nienhuis and Singh 1986</a>). A more detailed knowledge of the molecular basis of these important agronomic traits is critical to improve productivity in common bean.</p><p id="p-5">The application of molecular markers to plant breeding using modern statistical methods (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Malosetti <em>et al.</em> 2013</a>) has allowed breeders to accurately estimate the positions and effects of genomic regions associated with variation in quantitative traits (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kamfwa <em>et al.</em> 2015</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zuiderveen <em>et al.</em> 2016</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Perseguini <em>et al.</em> 2016</a>). This modern approach is very important for bean breeding programs, that are extremely dynamic due to demands of producing regions and consumer markets (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Souza <em>et al.</em> 2012</a>), and also extremely dependent of genotype-environment interactions (G × E), mainly in terms of tolerance to abiotic and biotic stresses (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beebe <em>et al.</em> 2011</a>). Understanding G × E under a molecular level is particularly important for self-pollinating plants that have adapted to constantly changing environments (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2003</a>), allowing breeders to extrapolate how much of the genetic gain obtained in one environment will be maintained in another one (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Eberhart and Russell 1966</a>).</p><p id="p-6">To date, an increasing number of SNP markers has been developed and applied to a diverse set of species at low cost, combining genome complexity reduction with next-generation sequencing (NGS) through RADseq (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Willing <em>et al.</em> 2011</a>), GbS (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Elshire <em>et al.</em> 2011</a>) and DArTseq strategies (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cruz <em>et al.</em> 2013</a>). For common bean, a few thousand SNPs are currently available, allowing explorations of genetic diversity and population structure (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rodriguez <em>et al.</em> 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cichy <em>et al.</em> 2015</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Valdisser <em>et al.</em> 2016</a>). In addition, SNPs have been used in the construction of dense linkage maps (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Song <em>et al.</em>, 2015</a>), allowing the identification of QTL associated with drought (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mukeshimana <em>et al.</em> 2014</a>), agronomic traits (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hoyos-Villegas <em>et al.</em> 2016</a>), disease resistance and root architecture (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nakedde <em>et al.</em> 2016</a>). GWAS have been successfully used to identify QTL related with biotic (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Zuiderveen <em>et al.</em> 2016</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Perseguini <em>et al.</em> 2016</a>), abiotic stress response (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Villordo-Pineda <em>et al.</em> 2015</a>, agronomic (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kamfwa <em>et al.</em> 2015</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Moghaddam <em>et al.</em> 2016</a>) and technological quality of grains (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cichy <em>et al.</em> 2015</a>) have also been identified.</p><p id="p-7">GWAS is already the most popular method to understand the genetic basis of complex traits in plants, since provide new insights on explaining the total genetic variance mainly the presence of small heritability (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Thorwarth <em>et al.</em> 2017</a>). An analytical approach, called regional genomic relationship mapping or, more simply, RHM, was proposed (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Uemoto <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nagamine <em>et al.</em> 2012</a>) to capture more of the genetic variation potentially associated with traits not detected by GWAS (missing genetic variation). RHM is based on a genomic relationship matrix among individuals based on common and rare SNPs found on small segments of the chromosome, thus combining information from common and rare variants that could be useful to enhance the incorporation of prior knowledge on the underlying genetic architecture. This method has shown high QTL detection power, with low rates of false positives and large fractions of variance explained when compared with other methods by using simulating data (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Usai <em>et al.</em> 2014</a>). In general, it is expected that RHM regions contain effects large enough to be detected by GWAS that would always be captured by RHM, whereas the opposite would not necessarily be true because of the additional small effect variants accounted for by RHM (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Caballero <em>et al.</em> 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Okeke <em>et al.</em> 2018</a>).</p><p id="p-8">To date, studies based on RHM analysis have been conducted with humans (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Shirali <em>et al.</em> 2015</a>), animals (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Riggio <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matika <em>et al.</em> 2016</a>) and, more recently, with plants (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Resende <em>et al.</em> 2017</a>). Thus, the aim of this study was to identify and characterize genomic regions that control productivity, lodging and architecture in common bean through GWAS and RHM analyses using SNP markers. In addition, the results of these approaches were compared to determine their performance in capturing additive genetic variation and determining the proportion of trait heritability accounted for by the loci they identified in the common bean genome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and methods</h2><div id="sec-2" class="subsection"><h3>Plant material and phenotyping</h3><p id="p-9">A total of 188 common bean accessions, including 91 landraces and 97 Brazilian and international cultivars/elite lines belonging to the Andean and Mesoamerican gene pools, were used in the GWAS and RHM for plant architecture, lodging and productivity (S1 Table). However, 580 accessions and four control cultivars were phenotyped in the field. Those 188 accessions effectively used for genotyping and mapping were selected among the initial set of 580 genotypes based on their highest genetic diversity (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Valdisser <em>et al.</em> 2017</a>). Field experiments were conducted in two important growing environments in Brazil: 1) At Embrapa Arroz &amp; Feijão experimental station (Goiás state), located at 16° 28’ 00” S latitude, 49° 17’ 00” longitude and 823 m altitude; and 2) at Embrapa experimental area in Ponta Grossa (Paraná state) located at 25° 05’ 42” S latitude, 50° 09’ 43” longitude and 969 m altitude.</p><p id="p-10">The experiment was carried out in augmented Federer blocks, in which the plots consisted of three 3-meters rows and four control cultivars with 20 repetitions each. Thus, it was designed 20 blocks with 29 accessions and four controls each. The control cultivars represent a sample of the variability from the main common bean market classes in Brazil. BRS Estilo is a carioca seeded cultivar and BRS Esplendor has black seeds, both with plants of erect architecture (indeterminate growth and type II plants). Jalo Precoce is a jalo seeded cultivar with plants of semi-erect architecture (indeterminate growth and type II plant). BRS Embaixador is from the dark red kidney market class and has plants of erect architecture (determined growth and plant type I). At Embrapa Arroz &amp; Feijão, common bean was cultivated in the winter growing season from April to June (2010) with average rainfall of 51.8 mm, average relative humidity around 60–70% and average temperature of 21.8°, ranging from 16° to 29° (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Silva <em>et al.</em> 2010</a>). At the experimental area in Ponta Grossa, common bean was cultivated in the dry growing season from December to April (2011) with average rainfall of 143.6 mm, average relative humidity around 50–60% and average temperature of 19.96°, ranging from 17.2° to 21.4° [<a href="https://pt.climate-data.org/location/4493/">https://pt.climate-data.org/location/4493/</a>].</p><p id="p-11">The winter season at Embrapa Rice and Beans Goiás (GO) and the dry season in Ponta Grossa (PR) represent distinct conditions for common bean cultivation. Winter growing season is carried out using high technology, including irrigation by central pivot, chemical fertilization and chemical control of pests. Soil diseases are more likely during winter, thus there is no relevant abiotic stress on this growing season, consequently leading to higher productivity in this period. In the dry season at experimental area in Ponta Grossa, common bean is cultivated without supplementary irrigation. Although the main diseases during the dry season are anthracnose (<em>Colletotrichum lindemuthianum</em>), angular leaf spot (<em>Pseudocercospora griseola</em>) and some bacterioses, the drought is the major abiotic stress that affects the yield in this condition.</p><p id="p-12">Agronomic trait architecture was determined by measuring branch insertion angles, length of guide and stem height at which all pods are above the soil surface; lodging tolerance was determined by the degree of inclination in relation to vertical. Both traits were observed at harvest maturity and evaluated on a scale from 1 (short guides, high pods, closer branches and without lodging) to 9 (long guides, low pods, open branches and lodged plants) as previously described by others (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Melo <em>et al.</em> 2009</a>). Productivity was determined by the weight of the beans harvested from each plot, and grain yield was expressed in kg ha<sup>-1</sup> at 13% humidity. Phenotypic correlations between the pairs of traits were given by the <em>Pearson</em> correlation at a confidence interval of 95%</p></div><div id="sec-3" class="subsection"><h3>Phenotypic modeling and genetic parameters</h3><p id="p-13">The adjustment of the phenotypic data were performed using the free software SELEGEN-REML/BLUP (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Resende 2007</a>), employing the augmented block model described in equation 1 below:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>where y is the data vector, f is the vector of the assumed fixed effects (means of the reference genotypes and mean of the population of main treatments at each site), g is the vector of the genotypic effects of the varieties (assumed to be random), b is the vector of the environmental effects of the blocks (assumed to be random), k is the vector of the effects of the genotype × environment interaction (random), and ε is the vector of the residual effects (random). The capital letters <strong>X</strong>, Z, S and T represent the incidence matrices for these effects. The vector of the genetic effects g can be divided into two groups: varieties and checks. The first group is given as random, with a multivariate normal distribution, a mean of zero and a covariance matrix <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-2.gif"/></noscript></span></span>, where A is the additive genetic kinship matrix and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-3.gif"/></noscript></span></span> is the genetic variance among the varieties of <em>P. vulgaris</em>. The second group is not relevant to the detection of QTL because its individuals are included in the model as a fixed effect to help correct for non-genetic effects (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pastina <em>et al.</em> 2012</a>). The corrected phenotypes used in the QTL mapping methods (GWAS and RHM) are given by <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-5.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-6.gif"/></noscript></span></span>.</p><p id="p-14">In order to contemplate the relevance of the G × E interaction, the respective variance component was included in the numerator of the joint heritability (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-7.gif"/></noscript></span></span>) aiming to evaluate the genetic behavior of the varieties over the two experimental conditions. This mentioned heritability was calculated as follows: <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-8.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-9.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-11.gif"/></noscript></span></span> are the components of variance between the varieties, the components of the interaction variety × environment, and the between blocks and residual effects, respectively. The coefficient of determination of the G × E interaction is given by <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-12.gif"/></noscript></span></span>, and the genetic correlation coefficient between sites is defined as <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-13.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-14.gif"/></noscript></span></span> is the genetic covariance between the two sites, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-16.gif"/></noscript></span></span> are the genetic variances at each of the two sites.</p></div><div id="sec-4" class="subsection"><h3>Genotyping and analysis With DArTseq method</h3><p id="p-15">In addition to analysis of SNPs generated by DArTseq method (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sánchez-Sevilla <em>et al.</em> 2015</a>) we also utilized SilicoDArT (DArT) markers which were extracted from the DArTseq libraries by DArTsoft14 pipeline. While a proportion of SilicoDArT markers is based on SNP polymorphism in restriction enzyme (RE) recognition sequences and therefore simply add to SNPs extracted from the sequenced tags, there were also additional sources of SilicoDArT polymorphisms, namely InDels in recognition sites and within the tags (restriction fragments) and, also, methylation variation at the genome level. The genome complexity reduction was achieved by using the RE <em>PstI</em> and Mse<em>I</em>. The DArT were genotyped as Presence Absence Variations (PAVs). DArT and SNP marker qualities were determined by reproducibility and call rate scores.</p></div><div id="sec-5" class="subsection"><h3>Linkage disequilibrium (LD)</h3><p id="p-16">The squared correlation coefficient (<em>r</em><sup>2</sup>) was used as LD measure over all markers pairs. The LD concept was relevant to inform an appropriate window length for RHM. When omitting the LD analysis, the windows length to be used in RHM can be empirically assumed, which decrease the power to detect true QTL regions. Additionally to <em>r</em><sup>2</sup>, the LD estimates were corrected for bias due to population structure and relatedness (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-17.gif"/></noscript></span></span>) using the LDcorSV (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mangin <em>et al.</em> 2012</a>). Genetic structure was inferred by Structure v 2.3.4 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pritchard <em>et al.</em> 2000</a>) and the most likely K was determined (∆K) as previously proposed (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Evanno <em>et al.</em> 2005</a>) using the program Structure Harvester (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Earl 2012</a>). LD decay curves were adjusted using a standard exponential function.</p></div><div id="sec-6" class="subsection"><h3>GWAS Analysis</h3><p id="p-17">The following model was used in the GWAS:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-19.gif"/></noscript></span></span> is the vector of phenotypes adjusted using the model described by equation 1; β is the fixed effect of intercept or general mean; u is the additive random effect of genotype/variety; <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-20.gif"/></noscript></span></span> refers to the fixed effect of the <strong>i</strong><sup>th</sup> marker; and ε is the residual effects of the model. The distributions and covariance structures of<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-21.gif"/></noscript></span></span> and ε are given by <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-23.gif"/></noscript></span></span>, respectively, where I is an identity matrix and G is the complete genomic kinship matrix, which can be represented based on the following equation [3]:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>where W is the incidence matrix formed by the SNP and DArT markers (sorted based on their position in the genome), assuming <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-26.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-27.gif"/></noscript></span></span> is the allele frequency of the <strong>i</strong><sup>th</sup> marker in the matrix <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-28.gif"/></noscript></span></span>. The determination of the significance of the loci markers was performed as described in the RHM method.</p></div><div id="sec-7" class="subsection"><h3>RHM Analysis</h3><p id="p-18">QTL mapping using the RHM method was performed as previously described in the literature (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Nagamine <em>et al.</em> 2012</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Resende <em>et al.</em> 2017</a>). In summary, RHM provides heritability estimates for genomic segments containing both common and rare allelic effects. Additionally, this model assumes G × E interaction, that enhance the knowledge on regional genetic variation related to environment adaptation. This mentioned RHM model presented in equation (4) was fitted using the <em>regress</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Clifford and McCullagh 2012</a>) R package:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">[4]</span></span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-30.gif"/></noscript></span></span>, u, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-31.gif"/></noscript></span></span> β and X are the same as described for the GWAS method, and v is the random regional genomic additive effect. The distribution and covariance structure of<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-32.gif"/></noscript></span></span> is given by <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-33.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-34.gif"/></noscript></span></span> is a matrix similar to G (equation 3), but using a subset of the matrix W. These subsets were determined by genomic “windows” or “regions” of two Mb in length overlapping by one Mb (<em>e.g.</em>, the three first regions are 0-1, 1-2 and 2-3 Mb) matching the estimated LD (see results). To test for the presence of regional variance (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-35.gif"/></noscript></span></span>) against the null hypothesis of no regional variance, a likelihood ratio test (LRT) was used, where L0 and L1 represent the likelihood values for the hypothesis of absence (H0) or presence (Ha) of regional variance respectively (<em>i.e.</em>, the complete model of equation 4 <em>vs.</em> the reduced model without <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-36.gif"/></noscript></span></span>).</p><p id="p-19">To explore the statistical test distribution of the null hypothesis and find the optimal threshold for each trait, the same permutation test with Bonferroni correction for multiple tests with a global <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-37.gif"/></noscript></span></span> used for GWAS was used for RHM. Genomic segments displaying significant <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-38.gif"/></noscript></span></span> were declared as regional QTL. The same whole-genome relationship matrix G was used to analyze all regions, differently from <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-39.gif"/></noscript></span></span> that varies over the genome positions. The G matrix used here (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">VanRaden 2008</a>) is an efficient method for processing genomic data by increasing the reliability of estimated breeding values in the presence of thousands of marker effects simultaneously. Because any consequential correlation generated between whole-genome and regional relationships would be very small and would tend to reduce the likelihood of detecting regional effects (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Nagamine <em>et al.</em> 2012</a>), the likelihood of a detected regional effect should increase when the same matrix is used for all regions. Otherwise, fitting the whole-genome relationship matrix G with a random effect accounts for relatedness and family structure.</p><p id="p-20">Whole genome heritability is given by <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-40.gif"/></noscript></span></span>, using the variance components of the reduced model of equation (4), <em>i.e.</em>, the model fitted without the regional genomic window components v. The regional heritabilities to each genomic window segment are <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-41.gif"/></noscript></span></span>, in the full model of equation [4]. These estimates were separately calculated for the DArT, SNP and the combined set of markers (SNP+DArT).”</p></div><div id="sec-8" class="subsection"><h3>Effects of allelic substitution</h3><p id="p-21">The average effects of allelic substitution at the identified QTL with no environmental interaction were statistically analyzed for overall significance using Welch’s <em>t</em>-test as implemented by R software version 3.1.3 (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Welch 1947</a>).</p></div><div id="sec-9" class="subsection"><h3>Identification of transcripts in regions containing joint-trait QTL</h3><p id="p-22">The QTL identified through GWAS and RHM joint analyses were placed into haplotype blocks previously identified for common bean (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Valdisser <em>et al.</em> 2017</a>), and nearby genes were identified using the common bean genome annotation available at Phytozome.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-23">The genome sequence and annotations are available at [<a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>]. Figure S1 presents the G matrices representing genomic relationship coefficients of <em>P. vulgaris</em> based on SNP and DArT markers. Figure S2 shows detailed genomic information of the association of DArT marker with lodging trait. Table S1 contain the list of common bean genotypes used in the molecular and phenotypic characterization. Table S2 presents the description of the functional annotation of the significant QTL detected in the joint analysis by RHM. Table S3 describe the summary of total trait and genomic heritability as well as the fractions captured by the markers in the analysis. Files containing the phenotypic and genotypic data used to perform the association analysis are available at Figshare. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6533933">https://doi.org/10.25387/g3.6533933</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Phenotypic evaluation of the experiment</h3><p id="p-24">Information on the fit of model 1 as well as the genetic parameters (genetic variance components, joint and regional heritabilies) of the two experiments is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. At site 1 (Goiás) none of the plots were lost; and all varieties (192 in total, given by 188 + 4 reference genotypes) provided phenotypic values for the analysis. In site 2 (Paraná), phenotypic information for architecture and lodging was collected for 136 varieties (132 + 4 reference genotypes); whereas for architecture a total of 190 (186 + 4) varieties were evaluated. For the traits plant architecture and lodging tolerance the obtained overall average values were 4.61 with standard deviation (SD) of 1.097 and 4.44 with SD of 1.634, respectively. Variations were observed in the means due to the environments, being more favorable for the environment of Goiás in the Central-West region (4.36 architecture and 4.32 lodging), compared to Paraná in the South region (4.96 architecture and 4.60 lodging), which was anticipated considering their geographic differences and the different crop yields. For productivity, the average value obtained for the combined experiments was 1,342 ± 653. A set of 70 and 66 accessions had above average values in Goiás (1,544 kg/ha) and Paraná (1,585 kg/ha), respectively, including 27 accessions (8 elite lines/cultivar and 19 traditional varieties) common to both environments with averages of 1,956 kg/ha (Goiás) and 2,080 kg/ha (Paraná). Correlation coefficients between plant architecture traits and productivity were weak and negative (<em>P</em> ≤ 0.05), ranging from -0.14 between productivity and lodging, to -0.18 between productivity and architecture, whereas between architecture and lodging it was a positive significant correlation (0.75).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490561/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490561/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genetic parameters (with standard deviation - SD) and phenotypic adjustment for the traits plant architecture (arch.), lodging (lodg.) and grain productivity (prod.) for common bean, estimated separately for each environment and in joint analysis, considering the 188 genotypes used in the GWAS and RHM</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Between the two experiments, estimates of accuracy in genotype selection (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-50.gif"/></noscript></span></span> ≥ 88%) showed high experimental quality and thus confidence in the ability to select for superior genotypes for architecture and lodging. For the trait productivity, accuracy estimates were reduced for the environment in Goiás (55%) and for the joint analysis (17%). The heritabilities, both at the mean level (<em>h</em><sup>2</sup> plot means<strong>)</strong> and at the level of the individual plots, were more similar between the environments for plant architecture (0.82), whereas for the traits lodging (0.87) and productivity (0.57), the highest estimates were obtained for the cultivation environment in the south of the country (Paraná). The differences between the heritabilities at the individual and average plot levels were very similar for all traits.</p><p id="p-27">The parameter <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-51.gif"/></noscript></span></span> represents how much total heritability (<span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-52.gif"/></noscript></span></span>) is due to the genetic interaction between sites. In this sense, it is possible to observe that the traits plant architecture and lodging had low values for <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-53.gif"/></noscript></span></span> compared to the trait productivity, indicating that the environment had a much more significant impact on productivity than on the other two traits. In addition, the values for <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-54.gif"/></noscript></span></span> indicate that the genotypes behaved very similarly at both sites with respect to the traits plant architecture and lodging. In contrast, the most productive genotypes at Goiás were not the same as those at Paraná (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-55.gif"/></noscript></span></span>).</p></div><div id="sec-13" class="subsection"><h3>Molecular marker genotyping and LD</h3><p id="p-28">For DArT, a total of 11,564 polymorphic markers were genotyped. These markers presented an average call rate of 95.2%, indicating less than 5% missing data. The average reproducibility score was 99.9%, with a result of 100% for the vast majority of markers (95.6%) and the lowest value estimated at 96.8%. A total of 6,286 SNPs from DArTseq were identified with an average call rate of 92% (∼8% missing data) and an average reproducibility score of 99.43%. Of the 17,850 markers evaluated, 6677 (3443 DArT and 3234 SNP) were identified in linkage equilibrium (37.4%). Estimated heterozygosity was lower than 1% in almost all chromosomes, with an average value of 0.89% (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), indicating the similarity of informativeness between dominant and codominant markers in autogamous plants.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490580/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490580/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution, heterozygosity and linkage disequilibrium (LD, <em>r</em><sup>2</sup>) of molecular markers (SNP and DArT), genotyped and analyzed individually for all chromosomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">The molecular markers were aligned with reference to the 11 chromosomes of the reference genome of <em>P. vulgaris</em> as represented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Both classes of markers presented a genome-wide distribution. The average number of SNPs per chromosome was 607, ranging from 439 on chromosome 4 to 867 on chromosome 2 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). A high density of markers was observed at the ends of each chromosome except for chromosome six, which showed a lower density at the initial portion. Chromosomes 1, 3, 7, 10 and 11 showed reduced marker density in the centromere region.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of 3443 DArT and 3234 SNP markers (6677 total markers) along the 11 chromosomes of common bean (y-axis). The x-axis represents chromosome position in Mb." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196359802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of 3443 DArT and 3234 SNP markers (6677 total markers) along the 11 chromosomes of common bean (y-axis). The x-axis represents chromosome position in Mb.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2841/F1.medium.gif" width="440" height="109"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2841/F1.medium.gif" width="440" height="109"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Distribution of 3443 DArT and 3234 SNP markers (6677 total markers) along the 11 chromosomes of common bean (y-axis). The x-axis represents chromosome position in Mb.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The relatedness matrix including all markers revealed two distinct groups representative of the Andean and Mesoamerican gene pools (S1 Fig), represented by 69 and 112 accessions, respectively. The average <em>r</em><sup>2</sup> for the full sample set and all markers (6677) corrected for relatedness and structure showed a faster rate of decay and dropped to half at a pairwise <em>r</em><sup>2</sup> value of 0.0312, estimated at 0.033 for DArT and 0.036 for SNP (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). An abrupt drop in LD decay was observed in the intervals spanning 1 to 2 Mb for the markers analyzed separately or together, which therefore seemed to be an appropriate window length for RHM.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele pair linkage disequilibrium (r2) across the entire P. vulgaris genome for all genotypes, plotted according to genetic distance in Mb and including both DArT and SNP markers. Decay lines for DArT (dotted line), SNPs (dashed line) and both marker types (solid line) are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196359802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Allele pair linkage disequilibrium (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) across the entire &lt;em&gt;P. vulgaris&lt;/em&gt; genome for all genotypes, plotted according to genetic distance in Mb and including both DArT and SNP markers. Decay lines for DArT (dotted line), SNPs (dashed line) and both marker types (solid line) are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2841/F2.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2841/F2.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490573" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Allele pair linkage disequilibrium (<em>r</em><sup>2</sup>) across the entire <em>P. vulgaris</em> genome for all genotypes, plotted according to genetic distance in Mb and including both DArT and SNP markers. Decay lines for DArT (dotted line), SNPs (dashed line) and both marker types (solid line) are shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Detection of associations via GWAS</h3><p id="p-33">The significance of genome association subjected to multiple testing corrections are described in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and provide an overview of the positioning of the markers, the fractions of the genomic heritabilities and the threshold [-<em>log10</em> (<em>p</em>-value)] of the identified QTL. A total of 60,093 association tests were performed (6,677 markers × 3 traits × 3 sites, <em>i.e.</em>, site 1, 2 and the combination of both sites). After the multiple testing corrections, eight QTL were identified for the three traits evaluated that were associated with three markers. For the traits architecture and lodging, the two QTL identified in Goiás and Paraná, as well as in the joint analysis, were associated with the marker DArT X3366188 located on chromosome 1 (position at 15.72 Mb). The significance of the identified QTL [−<em>log10</em> (<em>p</em>-value)] ranged from 3.95 (environment 1) to 4.53 (joint analysis). Fractions of explained genomic heritabilities were high, capturing up to 28.4% of additive genetic variation for architecture and 30.4% for lodging. For the trait productivity, two QTL were identified in environment 1. The QTL were associated with two closely positioned SNP markers on chromosome 3 (X3382215 and X3382284), with significant associations [–log<sub>10</sub>(p)] above 4.06 and fractions of explained heritabilities above 20%. All fixed QTL effects (m<sub>i</sub>) were positive.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490615/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490615/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Molecular markers and associated trait loci in common bean germplasm according to genome-wide association study (GWAS) for three traits, analyzed by environment and jointly</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Detection of associations via RHM</h3><p id="p-35">A total of 478 genomic windows were subjected to RHM analysis, each one covering a variable number of polymorphic SNPs and DArTs providing a coverage density of approximately one marker every 76.9 kb. Across the genome, regions spanned a minimum of two and a maximum of 107 markers. The mean and median number of SNPs within a region were 27.2 and 22, respectively.</p><p id="p-36">A set of 13 regional QTL were mapped considering a total number of markers varying from 3 to 41 across 5 chromosomes (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Six QTL were identified in chromosomes 1, 8, 9 and 11 as controlling plant architecture. QTL at chromosomes 1 (spanning 8 SNPs) and 8 (spanning 41 SNPs) were found in the Goiás environment and in the joint analysis. Explained RHM heritability ranged from 22 to 33%. For the plant lodging, three RHM QTL were identified in both environments through separated and joint analysis. The explained heritability ranged from 34.5 to 49.4%, with a minimum [-log10 (p)] of 2.69 in Paraná. This RHM QTL overlapped with an architecture-related QTL identified in Goiás environment and in the joint analysis. For plant productivity, RHM QTL were detected in chromosomes 1, 4 and 8 in both environments and the joint analysis, spanning from three (chrom. 8) to 31 SNPs (chrom.1), explaining from 11.6 to 40% of the genetic variance with minimum and maximum significances of 2.89 and 3.79, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490545/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490545/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Identification of QTL intervals in common bean germplasm through regional heritability mapping (RHM) using 2 Mb genomic segments with 1-Mb sliding window and their contribution to the additive genetic variance for traits plant architecture (arch.), lodging (lodg.) and grain productivity (prod.) evaluated by environment and jointly</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, a heat map shows RHM QTL distribution along the genome in both environments, with their respective heritability estimates presented. In addition to the RHM QTL that exceeded the significance threshold (red arrow), suggested additional associations along the genome are also represented in the heat map, such as the region labeled in dark blue on chromosome 4 above the interval of 25.25 Mb for the trait productivity.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide distribution of the significant QTL detected in the joint analysis by RHM along the 11 common bean chromosomes (y-axis) subdivided in 1 Mb windows for the three traits (right). Bar legends on the right correspond to regional heritability estimates. Red arrows indicate significant regions according to LRT test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196359802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide distribution of the significant QTL detected in the joint analysis by RHM along the 11 common bean chromosomes (y-axis) subdivided in 1 Mb windows for the three traits (right). Bar legends on the right correspond to regional heritability estimates. Red arrows indicate significant regions according to LRT test.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2841/F3.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2841/F3.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490566" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Genome-wide distribution of the significant QTL detected in the joint analysis by RHM along the 11 common bean chromosomes (y-axis) subdivided in 1 Mb windows for the three traits (right). Bar legends on the right correspond to regional heritability estimates. Red arrows indicate significant regions according to LRT test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Comparative analysis Between the GWAS and RHM results</h3><p id="p-40">Whereas the GWAS detected eight significant markers, the RHM method detected 222 significant markers distributed among 13 genomic regions (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Although from the statistical point of view, all the QTL had a 90% confidence interval (α=0.05) for the threshold, in general, the values estimated for the GWAS QTL were considerably larger (ranging from 3.95 to 4.89) than those estimated for the RHM QTL (2.58 to 3.79). This is most evident when we examine the GWAS QTL identified at position 15.72 on chromosome 1 for architecture and lodging whose minimum threshold was 3.95 <em>vs.</em> 2.66 for the RHM QTL.</p><p id="p-41">For the estimates of genomic heritability for both environments and the joint analysis, the GWAS-derived QTL explained approximately 23.2–28.4% of the genetic variance for architecture, 20.7–30.4% for lodging and 20.0–23.3% for productivity. In comparison, for the estimates obtained for the RHM-derived QTL, the levels of explained variance were higher, ranging from 47.6–57.6% for architecture, 34.5–46.8% for lodging and 14–40% for productivity.</p><p id="p-42">Total genomic heritability using the full set of markers (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-56.gif"/></noscript></span></span> captured between 72.09% and ≈100% of the heritability estimated from germplasm data (<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2841/embed/mml-math-57.gif"/></noscript></span></span>). Larger fractions were captured for architecture and lodging when compared with productivity (72%). The fraction of genomic heritability explained by RHM analysis was higher when compared to GWAS for both traits, architecture (58.02% <em>vs.</em> 28.3%) and lodging (46.83% <em>vs.</em> 30.3%) (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Of the three different QTL identified by GWAS, one (position 15.72) was contained in an RHM region identified for the same traits (architecture and lodging). An integrated view of all QTL identified by GWAS and RHM in both environments is provided in a Manhattan plot (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Several regions presented peaks at the same positions for both analyses but did not reach the significance threshold.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of results from GWAS in gray solid circles and RHM in black unfilled circles for the tree traits (plant architecture, lodging and grain productivity). Lines indicate thresholds of genome-wide significance after permutation tests at 5% Bonferroni correction. Gray dotted, GWAS; black dashed, RHM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196359802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Manhattan plot of results from GWAS in gray solid circles and RHM in black unfilled circles for the tree traits (plant architecture, lodging and grain productivity). Lines indicate thresholds of genome-wide significance after permutation tests at 5% Bonferroni correction. Gray dotted, GWAS; black dashed, RHM.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2841/F4.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2841/F4.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2841/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490549" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Manhattan plot of results from GWAS in gray solid circles and RHM in black unfilled circles for the tree traits (plant architecture, lodging and grain productivity). Lines indicate thresholds of genome-wide significance after permutation tests at 5% Bonferroni correction. Gray dotted, GWAS; black dashed, RHM.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The average effect of allelic substitutions was evaluated considering the change in mean trait value when an allele was replaced at the QTL for lodging (DArT X3366188). We observed that allelic substitution in the QTL<sub>GWAS</sub> significantly affected the lodging tolerance in both environments (S2 Fig).</p></div><div id="sec-17" class="subsection"><h3>Gene identification in QTL regions</h3><p id="p-45">Annotation analysis for architecture- and lodging-associated markers (SNPs and DArTs) identified by joint analysis placed the QTL in chromosome 1 at position 15.53 to 16.84 with eight markers on a haplotype block spanning approximately 1.31 Mb. This block contained 40 transcripts, of which 32 were annotated in 24 genes and eight transcripts with no predicted function (S2 Table). For the QTL associated exclusively with architecture on chromosome 8 at position 5.54 to 7.48 and containing 41 markers, a total of 206 transcripts were identified, of which 185 were annotated in 167 putative genes. For the regions containing the QTL associated with productivity placed at positions 29.66 to 29.94 and 29.94 to 31.32, six and 24 transcripts were identified, annotated in 4 and 17 putative genes, respectively.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Combined dominant and co-dominant markers for common bean</h3><p id="p-46">A predominance of homozygous loci is expected for the common bean, a preferentially autogamous species of which low outcrossing rates have been reported (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ferreira <em>et al.</em> 2007</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Faria <em>et al.</em> 2010</a>). In the present study, most estimates of heterozygosity were below 1% throughout the 11 chromosomes of common bean (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), consistent with the self-fertilizing reproductive system of the species and in accordance with previously reported (1.3%) by others (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cardoso <em>et al.</em> 2014</a>). Regarding the biological basis of the analyzed markers, <em>i</em>n this study DArT was scored as dominant (presence or absence of a probe target in the genome) and the conversion of these markers into a co-dominant pattern was possible due to the predominantly homozygous genome (alleles, AA and Aa, associated with presence and scored as dominant). The use of both types of markers produced by DArTseq provided complementary input data for our GWAS and RHM analysis, thus increasing the genome coverage and consequently the power of QTL detection (17,850 total markers evaluated). The large gaps (regions without marker coverage) identified along chromosomes in the present study were coincident with centromeric regions previously estimated for common bean (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Schmutz <em>et al.</em> 2014</a>). These genomic regions are expected to be gene-poor regions containing multiple and specific satellite repeats, with reduced recombination (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Schmutz <em>et al.</em> 2014</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Iwata <em>et al.</em> 2013</a>), which makes them largely refractory based on the method of complexity reduction during DArT library preparation. Theoretically, the application of sequencing technology, which does not involve restriction enzymes, could considerably increase the representativeness of markers along the genome.</p><p id="p-47">The genetic structure in Mesoamerican and Andean gene pools was confirmed based upon SNP and DArT data. In addition, the patterns of LD (<em>r</em><sup>2</sup>) defined both by individual marker class and using the combined data set were very similar, as shown by the LD-decay curves (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Thus, the use of a less expensive and easier to obtain SNP marker wouldn´t have reduced these analyses’ power to detect true associations between markers and quantitative trait loci in beans. Of the 17,850 markers, 37.4% were in LE in this population, and a regional mapping window of 1 Mb was calibrated for the RHM analysis. For RHM the density of markers within a given window affects the structure of the regional-based genomic relationship matrix (GRM). Thus, would be expected that a window with more markers would also be more explanatory taking into account the possibility to observe high covariances in GRM. In this context, we opted to define the window length according to LD extent over SNP distances. Thus, it was possible to ensure that RHM was not directly affected by window length, since the “degree of information” provided by LD extent was equally distributed among the windows. In relation to GWAS, since the markers were individually tested over the genome, the marker effect significance was not directly affected by this density. However, under this last approach, care should be taken since the poor coverage of SNP panel implies in more noticeable lack of information because the absence of LD concept in the statistical analysis.</p><p id="p-48">Knowledge of the genetic control of bean architecture and related traits are of great value in the development of effective strategies for selection. Architecture refers to the degree of branching, internode elongation, and shoot determinacy, among others (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wang and Li 2008</a>). When the stem is insufficient to prevent displacement of the plant from the vertical axis, lodging occurs, causing major constraints to the production of several crops (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kashiwagi <em>et al.</em> 2008</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yamaguchi <em>et al.</em> 2014</a>). In this study, the heritability estimates for plant architecture (81%) and lodging (79%) were considered of high magnitude and were consistent with results previously obtained for common bean for plant architecture (∼72.9%) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alvares <em>et al.</em> 2016</a>), and for lodging (78%) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Moghaddam <em>et al.</em> 2016</a>). Therefore, in addition to the high potential of the population under study for selection of these traits with predominance of additive genetics effects, more progress will be made in selection due mostly to genes, which is adequate for the marker-assisted selection (MAS). For plant architecture and lodging, the combined set of markers explained most of the additive heritability for the traits (≈100%), indicating that, theoretically, most relevant markers were located in one, or few linked window, and the every segregant QTL was bracketed by markers.</p><p id="p-49">For grain yield, the most important trait evaluated for common bean and other grain crops, a wide range of heritability values has been reported, which may be due to population effects, environmental factors or experimental accuracy. It has been estimated at ranges of 23–71% (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alvares <em>et al.</em> 2016</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gonçalves-Vidigal <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balcha 2014</a>), in agreement with the result found in our study (43%). Moreover, the estimated genomic heritability <em>(h<sup>2</sup><sub>SNP+DArT</sub>)</em> also captured a relatively large proportion (72%) of total trait heritability (<em>h</em><sup>2</sup><sub>total</sub>), which will be of great value in identifying favorable alleles for grain yield. Although the SNP analysis alone explained little more of the total genetic heritability (<em>h<sup>2</sup><sub>SNP</sub>=</em>74%), the DArT and SNP combination (<em>h<sup>2</sup><sub>SNP+DArT</sub></em> = 72%) provided a double genome coverage, increasing the power to detect QTL by taking advantage of LD between causal variants and markers.</p></div><div id="sec-20" class="subsection"><h3>Associations for plant architecture, lodging and grain productivity</h3><p id="p-50">In general, the proportion of genomic heritability explained by RHM for architecture, lodging and productivity was considerably higher (35.48 to 58.02) than that explained by GWAS (28.39 to 30.37) (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). For plant architecture and lodging, which showed high trait heritability (81% and 79%, respectively), RHM identified more genome-wide significance accounting for larger fractions of the additive genetic variance regions than did GWAS (S3 Table). As proposed by others (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Resende <em>et al.</em> 2017</a>), this improvement resulted from the RHM analytical method, which combines interval mapping with association analysis to capture variance across the whole population, powered by the combined effect of several closely linked loci at the target locus. In RHM, while a significant single locus can be overshadowed by many non-significant loci within the same window, the combined effect of significant loci in high LD between them tend to highlight the window in question. In GWAS, a significant single locus tends to be highlighted independently of the LD pattern between it and the other loci. In addition, RHM avoids false positive associations identified by GWAS since a significant marker from GWAS does not implies in significant window under RHM framework (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Riggio <em>et al.</em> 2013</a>).</p><p id="p-51">Based on GWAS, only one association was identified for the traits architecture and lodging, in both environment and joint analysis (chrom. 1 at 15.72 Mb). This was maintained for the lodging trait, with the same locus association observed for GWAS (explained 30%) and RHM (explained 46.8%) and hits on the same chromosome position with high consistency. As a significant correlation (0.7454) between lodging and architecture was identified, it reinforces that these mechanisms are associated and may be exploited together in further studies. Additional QTL for lodging were reported using the common bean biparental mapping population, of which four identified loci accounted for 39% of the genetic variance (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beattie <em>et al.</em> 2003</a>) and two QTL accounted for 9% (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kolkman and Kelly 2003</a>). More recently, based on GWAS analysis, a strong association for lodging was identified at chrom. 7 (46 Mb) that explained 21% of the phenotypic variation (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Moghaddam <em>et al.</em> 2016</a>). Compared with the high estimated values of lodging heritability (∼79%), we observed that not all loci acting to control this trait were identified in isolated studies, including ours. High heritability and low environmental effect, rather than providing information about the genetic architecture of the trait, it implies a greater correlation between phenotype and genotype (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Visscher <em>et al.</em> 2008</a>). The multiple different genomic regions associated with lodging suggest a trait with more genes involved.</p><p id="p-52">Architecture is a complex trait related to several morphological traits with evidence of environmental influence and predominantly additive genetic control (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Silva <em>et al.</em> 2013</a>). In the 1990s, based on RAPD markers five QTL for plant uprightness and branch density were described (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jung <em>et al.</em> 1996</a>). Posteriorly, using an increased set of markers (RAPD, SSR, STS, SCAR) six QTL for architecture-related traits (angle, height and hypocotyl diameter) explaining from 10.6 to 29.9 of the variance were reported (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beattie <em>et al.</em> 2003</a>), followed by seven more QTL for general plant height and plant width, explaining from 8 to 19% (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blair <em>et al.</em> 2006</a>). In such cases, QTL are difficult to compare, as they often employ different types of architecture measurements, but a consensus regarding a plant architecture QTL placed on chromosome 1 has been reported. In the present study, QTL for plant architecture was detect by both methods although RHM has captured twice as much heritability (58%). In addition, a QTL for architecture overlapped with a lodging QTL on chromosome 1, for which high heritability was found in the Goiás environment (38%) and in joint analysis (37%), being suggestive of a potential pleiotropic effect in such traits with benefits for breeding.</p><p id="p-53">To date, several QTL for important productivity-related traits in common bean have been identified, including seed weight, pods per plant, seeds per plant, and days to flowering (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Mukeshimana <em>et al.</em> 2014</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wright and Kelly 2011</a>). More recently, QTL studies based on linkage and GWAS analysis using a larger set of SNPs have provided clearer insight into the genetic basis of agronomic traits related to productivity in common bean (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kamfwa <em>et al.</em> 2015</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hoyos-Villegas <em>et al.</em> 2016</a>). In the present study, QTL associated with grain yield were identified in both GWAS and RHM analysis on chromosomes 1, 3, 4 and 8. Through GWAS, two QTL on chromosome 3 were identified in Goiás (explaining 30.23% of the variance), in accordance with previous GWAS (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Mukeshimana <em>et al.</em> 2014</a>) and linkage map analysis (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Blair <em>et al.</em> 2006</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Wright and Kelly 2011</a>). Two RHM QTL on chromosome 8, identified in both environments in the present study, were in accordance with significant SNP associations previously identified by <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Kamfwa <em>et al.</em> (2015)</a> based on GWAS, supporting the evidence for the presence of an important QTL conditioning seed yield at this position, or the presence of a substantial linkage disequilibrium among closely linked marker loci. In addition, environment-specific QTL on chromosomes 1 and 4 were reported in this study, in accordance with a number of QTL accounting for larger proportions of the phenotypic variance in seed yield previously identified through linkage analysis on chromosome 1 (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Trapp <em>et al.</em> 2014</a>) and 4 (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Blair <em>et al.</em> 2006</a>).</p><p id="p-54">For all traits evaluated in the present study, high threshold values [–log<sub>10</sub>(p)] were reported for GWAS QTL, but these QTL explained lower amount of genetic variance than those identified by RHM. The differences of permutation-based significance threshold between RHM and GWAS are mainly due to LRT (Likelihood Ratio Test) values, while in GWAS the markers are fitted as fixed effects, in RHM are fitted as random effects with variance attributed to predefined windows containing a variable number of SNPs (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Valdisser <em>et al.</em> 2017</a>). Thus, the likelihood functions (constructed from these distinct models) are different; and consequently, the magnitude of LRT values is also naturally different; even under the randomization realized by the same permutation procedure <strong>(</strong>α=0.05<em>)</em>. However, the probabilistic significance level (reflected in the threshold value) is assured to be the same for both approaches. At present, both strategies of QTL identification lacked the power to detect all the QTL variance as estimated by trait heritability at the established threshold. Therefore, a considerable proportion of heritability, corresponding to the proportion of genetic variation that was not significantly detected and declared, is still missing. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">4</a> show several suggestive regions found with RHM that did not have large enough effects to be declared significant at the genome-wide level. The weak signals detected in the 100 kb SNP window form a basis for further analyses using smaller window sizes (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Resende <em>et al.</em> 2017</a>). The proportion of unexplained genetic variation could be attributed to imperfect LD between markers and QTL (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Leamy <em>et al.</em> 2017</a>), since the extend of LD is variable along the genome (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hyten <em>et al.</em> 2007</a>). This suggests that the mapping resolution for RHM analysis could be improved using smaller windows (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Nagamine <em>et al.</em> 2012</a>). In addition, even with correction for population structure in the GWAS and RHM models to avoid unbiased results, the present study’s sample size may have contributed to reducing the power of analysis. A small sample size would have an even greater impact if causal allele variants are rare and provide a small average contribution to the phenotypic variance. The Mesoamerican and Andean gene pools had divergent processes of domestication and different geographic distributions accounting for a strong population structure and variable levels of genetic diversity (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bitocchi <em>et al.</em> 2012</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Schmutz <em>et al.</em> 2014</a>). Preferably, these populations should be evaluated separately, using a larger sample size to represent most of the available genetic variation and more representative genome-wide SNPs to increase the representation of all causal variants and provide more accurate estimates of LD. This is particularity important for traits with polygenic inheritance, for which this scenario would help to capture further associations with small effects underlying these traits and to find more precise estimates of QTL heritability (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Uemoto <em>et al.</em> 2015</a>).</p></div><div id="sec-21" class="subsection"><h3>Genes in the QTL regions</h3><p id="p-55">Previous studies have reported a QTL for lodging on chromosome 1, as described through a linkage analysis combining three populations of common bean (R<sup>2</sup> = 12.93) (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Moghaddam <em>et al.</em> 2016</a>). Additionally, a lodging QTL identified in the present study was consistently identified for the same trait on soybean, chromosome 19 (qLS19-1; R<sup>2</sup> = 19.8%), which is syntenic with common bean chromosome 1 (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Yamaguchi <em>et al.</em> 2014</a>). The group of genes (24) within the lodging and architecture QTL included transcription factors, proline-rich proteins, lipolytic enzyme, and plant hormones, most of them implicated in diverse aspects of plant growth and development and mechanical resistance, providing support for the hypothesis that co-expression of these genes affects the lodging and architecture traits (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dharmasiri <em>et al.</em> 2005</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Riemann <em>et al.</em> 2007</a>). The SNP allelic variants associated with these traits are unlikely to be functional variants themselves because they were not placed within genes. Rather, based on LD, they could be used to determine the haplotype containing the functional variant, which would become the focus for follow-up studies. A molecular marker associated with lodging and architecture would be of great interest to select accessions from gene banks, to evaluate segregant populations and inbred line conversion in controlled (greenhouse) conditions where plants are inadequately characterized in lodging and architecture potential. In addition, the information of low correlation between plant architecture and productivity indicated that a marker will be useful on selections of plants carrying the favorable allele, and also to fix it in their homozygous state in early generations. Under this scenario, care should be taken, since the major gene is only one that affect the trait, and no marker selection has been applied to the other genes, which contributes to the overall phenotype (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Muir and Stick 1998</a>).</p><p id="p-56">The positive QTL identified in the present study were derived from a genetically diverse group of germplasm, including many landraces that represent potentially useful materials for breeding efforts due to carrying different and potentially favorable alleles in architecture, lodging and yield-related traits. In addition, high-resolution chromosome analysis allowed increased precision in QTL mapping, because the associated variants are in LD with causal variants, opening new perspectives for MAS. By examining genes surrounding the QTL for productivity on chromosome 8, several candidate genes that may modulate plant growth and development, plant structure, cell cycle and morphogenesis were identified (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Craig and Tyers 1999</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chepyshko <em>et al.</em> 2012</a>). Adjacent QTL regions with several SNPs clustered and placed in an interval of 1.6 Mbp were detected by the RHM analysis, which represent an interesting candidate region for further investigation toward MAS.</p></div><div id="sec-22" class="subsection"><h3>Conclusions</h3><p id="p-57">In this study, three traits were evaluated in two important common bean growing environments in Brazil, enabling the identification of QTL, which is the first step toward the development of a useful molecular resource for marker-aided selection (MAS) breeding. Particularly for lodging (total <em>h</em>2 ≥ 34.5 ≤ 49.4), the genome region identified (by GWAS and RHM) with environmental stability and a significant effect of allelic substitution represents the starting point for applying MAS aiming to increase the allele frequency at this locus in the breeding germplasm. On the other hand, the traits plant architecture and grain productivity, where a high number of QTL were identified (by RHM) and some were environment-specific, present a greater challenge to breeders. Genomic regions that did not reach the significance threshold to be declared QTL (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">4</a>) were also suggestive of potential associations across the genome. Thus, by taking advantage of high resolution SNP genotyping and RHM’s effectiveness for identifying a large number of trait-associated markers, approaches that consider whole-genome effects and incorporate these data into predictive models, such as genome selection (GS), will exert great impact on breeding efficiency (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Spindel <em>et al.</em> 2016</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Biazzi <em>et al.</em> 2017</a>). As more genomic information becomes available, more data will be applied into common bean improvement, creating new opportunities for development and selection of elite breeding genotypes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">We would like to thank the National Council for Scientific and Technological Development (CNPq) for the grants to RTR, MDVR, FFS, LCM, HSP, CB and RPV. The Brazilian Agricultural Research Corporation (EMBRAPA) for providing financial support for this research (project number 02.12.12.005.00.00). The authors are very grateful to the reviewers for their valuable contributions to the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-59">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6533933">https://doi.org/10.25387/g3.6533933</a>.</p></li><li class="fn" id="fn-5"><p id="p-60"><em>Communicating editor: P. Brown</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-01-22"><span class="received-label">Received </span>January 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-27"><span class="accepted-label">Accepted </span>June 27, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Resende <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.1" data-doi="10.4238/gmr15049081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvares</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">H. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Estimation of genetic parameters and selection of high-yielding, upright common bean lines with slow seed-coat darkening.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">gmr15049081</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4238/gmr15049081</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%26rft.volume%253D15%26rft.spage%253Dgmr15049081%26rft_id%253Dinfo%253Adoi%252F10.4238%252Fgmr15049081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4238/gmr15049081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.2" data-doi="10.3923/ajcs.2014.158.164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balcha</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic variation for grain yield of common bean (<em>Phaseolus vulgaris</em> L.) in sole and maize/bean intercropping systems.</span> <abbr class="cit-jnl-abbrev">Asian J. Crop Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3923/ajcs.2014.158.164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsian%2BJ.%2BCrop%2BSci.%26rft.volume%253D6%26rft.spage%253D158%26rft_id%253Dinfo%253Adoi%252F10.3923%252Fajcs.2014.158.164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3923/ajcs.2014.158.164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.3" data-doi="10.1139/g03-015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beattie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaels</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauls</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping quantitative trait loci for a common bean (<em>Phaseolus vulgaris</em> L.) ideotype.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">422</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g03-015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBeattie%26rft.auinit1%253DA.%2BD.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D411%26rft.epage%253D422%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bfor%2Ba%2Bcommon%2Bbean%2B%2528Phaseolus%2Bvulgaris%2BL.%2529%2Bideotype.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg03-015%26rft_id%253Dinfo%253Apmid%252F12834057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g03-015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12834057&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2841.4" data-doi="10.1002/9780470960929.ch25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beebe</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosquera</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic improvement of common beans and the challenges of climate change</span>, <span class="cit-source">Crop Adaptation to Climate Change</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Yadav</span>  <span class="cit-name-given-names">R. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lotze-Campen</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Oxford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470960929.ch25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BAdaptation%2Bto%2BClimate%2BChange%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470960929.ch25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470960929.ch25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.5" data-doi="10.1371/journal.pone.0169234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biazzi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nazzicari</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pecetti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmonari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association mapping and genomic selection for alfalfa (<em>Medicago sativa</em>) forage quality traits.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0169234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0169234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0169234%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.6" data-doi="10.1073/pnas.1108973109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bitocchi</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nanni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellucci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giardini</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mesoamerican origin of the common bean (<em>Phaseolus vulgaris</em> L.) is revealed by sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">E788</span>–<span class="cit-lpage">E796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1108973109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1108973109%26rft_id%253Dinfo%253Apmid%252F22393017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzE0L0U3ODgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.7" data-doi="10.1007/s00122-006-0217-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iriarte</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beebe</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTL analysis of yield traits in an advanced backcross population derived from a cultivated Andean × wild common bean (<em>Phaseolus vulgaris</em> L.) cross.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-006-0217-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBlair%26rft.auinit1%253DM.%2BW.%26rft.volume%253D112%26rft.issue%253D6%26rft.spage%253D1149%26rft.epage%253D1163%26rft.atitle%253DQTL%2Banalysis%2Bof%2Byield%2Btraits%2Bin%2Ban%2Badvanced%2Bbackcross%2Bpopulation%2Bderived%2Bfrom%2Ba%2Bcultivated%2BAndean%2Bx%2Bwild%2Bcommon%2Bbean%2B%2528Phaseolus%2Bvulgaris%2BL.%2529%2Bcross.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0217-2%26rft_id%253Dinfo%253Apmid%252F16432734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0217-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16432734&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236486600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.8" data-doi="10.1023/A:1024146710611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broughton</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beebe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gepts</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Beans (<em>Phaseolus</em> spp.) – model food legumes.</span> <abbr class="cit-jnl-abbrev">Plant Soil</abbr> <span class="cit-vol">252</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1024146710611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSoil%26rft.volume%253D252%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1024146710611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1024146710611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183439100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.9" data-doi="10.1534/genetics.115.177220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The nature of genetic variation for complex traits revealed by GWAS and regional heritability mapping analyses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1601</span>–<span class="cit-lpage">1613</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177220%26rft_id%253Dinfo%253Apmid%252F26482794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS80LzE2MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.10" data-doi="10.4238/2014.March.24.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso</span>  <span class="cit-name-given-names">P. C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brondani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menezes</span>  <span class="cit-name-given-names">I. P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdisser</span>  <span class="cit-name-given-names">P. A. M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borba</span>  <span class="cit-name-given-names">T. C. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Discrimination of common bean cultivars using multiplexed microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1964</span>–<span class="cit-lpage">1978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4238/2014.March.24.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%26rft.volume%253D13%26rft.spage%253D1964%26rft_id%253Dinfo%253Adoi%252F10.4238%252F2014.March.24.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4238/2014.March.24.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.8.2841.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">CGIAR (Consortium of International Agricultural Research Centers)</span></li></ol><cite>. Common bean. Available from: [<a href="https://ciat.cgiar.org/what-we-do/breeding-better-crops/beans/">https://ciat.cgiar.org/what-we-do/breeding-better-crops/beans/</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.12" data-doi="10.1186/1471-2164-13-309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chepyshko</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J.-F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multifunctionality and diversity of GDSL esterase/lipase gene family in rice (<em>Oryza sativa</em> L. japonica) genome: new insights from bioinformatics analysis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DChepyshko%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.spage%253D309%26rft.epage%253D309%26rft.atitle%253DMultifunctionality%2Band%2Bdiversity%2Bof%2BGDSL%2Besterase%252Flipase%2Bgene%2Bfamily%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2Bjaponica%2529%2Bgenome%253A%2Bnew%2Binsights%2Bfrom%2Bbioinformatics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-309%26rft_id%253Dinfo%253Apmid%252F22793791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22793791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.13" data-doi="10.2135/cropsci2014.09.0653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cichy</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porch</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaver</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fourie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A <em>Phaseolus vulgaris</em> diversity panel for Andean bean improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2149</span>–<span class="cit-lpage">2160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.09.0653</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2149%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.09.0653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.09.0653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-other" id="cit-8.8.2841.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> The regress package. R Package version 1–3.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.15" data-doi="10.1016/S0079-6107(99)00010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Craig</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyers</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The F-box: a new motif for ubiquitin dependent proteolysis in cell cycle regulation and signal transduction.</span> <abbr class="cit-jnl-abbrev">Prog. Biophys. Mol. Biol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0079-6107(99)00010-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bbiophysics%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DProg%2BBiophys%2BMol%2BBiol%26rft.aulast%253DCraig%26rft.auinit1%253DK.%2BL.%26rft.volume%253D72%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D328%26rft.atitle%253DThe%2BF-box%253A%2Ba%2Bnew%2Bmotif%2Bfor%2Bubiquitin%2Bdependent%2Bproteolysis%2Bin%2Bcell%2Bcycle%2Bregulation%2Band%2Bsignal%2Btransduction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0079-6107%252899%252900010-3%26rft_id%253Dinfo%253Apmid%252F10581972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0079-6107(99)00010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10581972&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083198700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.16" data-doi="10.1371/journal.pone.0064062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">V. M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dierig</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development of DArT marker platforms and genetic diversity assessment of the U.S. collection of the new oilseed crop lesquerella and related species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e64062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0064062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De64062%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0064062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0064062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.17" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: what is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.18" data-doi="10.1016/j.devcel.2005.05.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dharmasiri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dharmasiri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weijers</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lechner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Plant development is regulated by a family of auxin receptor F box proteins.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2005.05.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DDharmasiri%26rft.auinit1%253DN.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D119%26rft.atitle%253DPlant%2Bdevelopment%2Bis%2Bregulated%2Bby%2Ba%2Bfamily%2Bof%2Bauxin%2Breceptor%2BF%2Bbox%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2005.05.014%26rft_id%253Dinfo%253Apmid%252F15992545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2005.05.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15992545&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230659000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.19" data-doi="10.1007/s12686-011-9548-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet. Resour.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12686-011-9548-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12686-011-9548-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12686-011-9548-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.20" data-doi="10.2135/cropsci1966.0011183X000600010011x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eberhart</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Stability parameters for comparing varieties.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1966.0011183X000600010011x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DEberhart%26rft.auinit1%253DS.%2BA.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D40%26rft.atitle%253DStability%2BParameters%2Bfor%2BComparing%2BVarieties%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1966.0011183X000600010011x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1966.0011183X000600010011x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19667594800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.21" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2841.22"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">EMBRAPA</span>, <span class="cit-pub-date">2015</span> Technological Solutions: Copo Medidor de perdas na colheita de feijão, Available from. [<a href="https://www.embrapa.br/busca-de-produtos-processos-e-servicos/-/produto-servico/2696/copo-medidor-de-perdas-na-colheita-de-feijao">https://www.embrapa.br/busca-de-produtos-processos-e-servicos/-/produto-servico/2696/copo-medidor-de-perdas-na-colheita-de-feijao</a>].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.23" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: A simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02553.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.24" data-doi="10.4161/gmcr.1.2.11609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aragão</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene flow from transgenic common beans expressing the bar gene.</span> <abbr class="cit-jnl-abbrev">GM Crops</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/gmcr.1.2.11609</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGM%2BCrops%26rft.volume%253D1%26rft.spage%253D94%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fgmcr.1.2.11609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/gmcr.1.2.11609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.25" data-doi="10.1007/s10681-006-9251-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">J. E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanes</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cecon</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene flow in common bean (<em>Phaseolus vulgaris</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-006-9251-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D153%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-006-9251-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-006-9251-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242852500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.26" data-doi="10.1007/s10681-008-9758-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves-Vidigal</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mora</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bignotto</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munhoz</span>  <span class="cit-name-given-names">R. E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability of quantitative traits in segregating common bean families using a Bayesian approach.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">560</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-008-9758-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D164%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-008-9758-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-008-9758-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.27" data-doi="10.3835/plantgenome2015.12.0122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoyos-Villegas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association analysis for drought tolerance and associated traits in common bean.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.12.0122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.12.0122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.12.0122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.28" data-doi="10.1534/genetics.106.069740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Highly Variable Patterns of Linkage Disequilibrium in Multiple Soybean Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1937</span>–<span class="cit-lpage">1944</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.069740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.069740%26rft_id%253Dinfo%253Apmid%252F17287533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE5MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tek</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">M. M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abernathy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonseca</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification and characterization of functional centromeres of the common bean.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D76%26rft.spage%253D47%26rft_id%253Dinfo%253Apmid%252F23795942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23795942&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324923900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skroch</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nienhuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnaud-Santana</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Molecular markers associated with plant architecture and resistance to common blight, web blight, and rust in common beans.</span> <abbr class="cit-jnl-abbrev">J Amer Soc Hort.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">794</span>–<span class="cit-lpage">803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAmer%2BSoc%2BHort.%26rft.volume%253D121%26rft.spage%253D794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.31" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamfwa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cichy</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association study of agronomic traits in common bean.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.09.0059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.32" data-doi="10.1007/s00122-008-0816-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kashiwagi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Togawa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirotsu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishimaru</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Improvement of lodging resistance with QTLs for stem diameter in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">757</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-008-0816-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKashiwagi%26rft.auinit1%253DT.%26rft.volume%253D117%26rft.issue%253D5%26rft.spage%253D749%26rft.epage%253D757%26rft.atitle%253DImprovement%2Bof%2Blodging%2Bresistance%2Bwith%2BQTLs%2Bfor%2Bstem%2Bdiameter%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0816-1%26rft_id%253Dinfo%253Apmid%252F18575836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0816-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18575836&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258616300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.33" data-doi="10.2135/cropsci2003.0539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolkman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">QTL conferring resistance and avoidance to white mold in common bean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2003.0539</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DKolkman%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D539%26rft.epage%253D548%26rft.atitle%253DQTL%2BConferring%2BResistance%2Band%2BAvoidance%2Bto%2BWhite%2BMold%2Bin%2BCommon%2BBean%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.0539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.0539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000181479700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.34" data-doi="10.1186/s12864-016-3397-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">B.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A genome-wide association study of seed composition traits in wild soybean (Glycine soja).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-3397-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-3397-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-3397-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.35" data-doi="10.1007/s00122-003-1401-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafitte</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">QTL× environment interactions in rice. I. heading date and plant height.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-003-1401-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DZ.%2BK.%26rft.volume%253D108%26rft.issue%253D1%26rft.spage%253D141%26rft.epage%253D153%26rft.atitle%253DQTL%2Bx%2Benvironment%2Binteractions%2Bin%2Brice.%2BI.%2Bheading%2Bdate%2Band%2Bplant%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1401-2%26rft_id%253Dinfo%253Apmid%252F12961067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1401-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12961067&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186957900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.36" data-doi="10.3389/fphys.2013.00044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The statistical analysis of multi-environment data: Modeling genotype-by-environment interaction and its genetic basis.</span> <abbr class="cit-jnl-abbrev">Front. Physiol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fphys.2013.00044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPhysiol.%26rft.volume%253D4%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffphys.2013.00044%26rft_id%253Dinfo%253Apmid%252F23487515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fphys.2013.00044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23487515&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.37" data-doi="10.1038/hdy.2011.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siberchicot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doligez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">This</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Novel measures of linkage disequilibrium that correct the bias due to population structure and relatedness.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.73</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMangin%26rft.auinit1%253DB.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D291%26rft.atitle%253DNovel%2Bmeasures%2Bof%2Blinkage%2Bdisequilibrium%2Bthat%2Bcorrect%2Bthe%2Bbias%2Bdue%2Bto%2Bpopulation%2Bstructure%2Band%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.73%26rft_id%253Dinfo%253Apmid%252F21878986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.73&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878986&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.38" data-doi="10.1186/s12711-016-0191-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matika</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riggio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anselme-Moizan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association reveals QTL for growth, bone and in vivo carcass traits as assessed by computed tomography in Scottish Blackface lambs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0191-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0191-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0191-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2841.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Faria</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Peloso</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sartorato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rava</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Instalação, avaliação e procedimentos experimentais</span>, <span class="cit-source">Procedimentos para condução de experimentos de Valor de Cultivo e Uso em feijoeiro comum</span>, <span class="cit-publ-name">Embrapa Arroz e Feijão Press</span>: <span class="cit-publ-loc">Santo Antônio de Goiás</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">L. C.</span></span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.40" data-doi="10.3835/plantgenome2016.02.0012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghaddam</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osorno</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association study identifies candidate loci underlying agronomic traits in a Middle American diversity panel of common bean.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.8.2841.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stick</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> Relative advantage of combining genes with major effects in breeding programs: Simulation results. Proc. 6th World Congress on Genetics applied to Livestock Production. 26: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.42" data-doi="10.2135/cropsci2013.06.0427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukeshimana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butare</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait loci associated with drought tolerance in common bean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">938</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.06.0427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D923%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.06.0427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.06.0427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.43" data-doi="10.1371/journal.pone.0046501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagamine</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitart</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Localizing loci underlying complex trait variation using regional genomic relationship mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNagamine%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46501%26rft.epage%253De46501%26rft.atitle%253DLocalising%2BLoci%2Bunderlying%2BComplex%2BTrait%2BVariation%2BUsing%2BRegional%2BGenomic%2BRelationship%2BMapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046501%26rft_id%253Dinfo%253Apmid%252F23077511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23077511&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.44" data-doi="10.1007/s10681-016-1755-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakedde</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibarra-Perez</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukankusi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waines</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mapping of QTL associated with Fusarium root rot resistance and root architecture traits in black beans.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-016-1755-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D212%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-016-1755-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-016-1755-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.45" data-doi="10.2135/cropsci1986.0011183X002600010005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nienhuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Combining ability analyses and relationships among yield, yield components and architectural traits in dry bean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1986.0011183X002600010005x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DNienhuis%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D27%26rft.atitle%253DCombining%2BAbility%2BAnalyses%2Band%2BRelationships%2BAmong%2BYield%252C%2BYield%2BComponents%252C%2Band%2BArchitectural%2BTraits%2Bin%2BDry%2BBean%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1986.0011183X002600010005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1986.0011183X002600010005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.46" data-doi="10.3835/plantgenome2017.06.0050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okeke</span>  <span class="cit-name-given-names">U. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Regional heritability mapping provides insights into dry matter content in african white and yellow cassava populations.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.06.0050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.06.0050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.06.0050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.47" data-doi="10.1007/s00122-011-1748-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pastina</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gazaffi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed model QTL analysis for sugarcane multiple-harvest-location trial data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1748-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPastina%26rft.auinit1%253DM.%2BM.%26rft.volume%253D124%26rft.issue%253D5%26rft.spage%253D835%26rft.epage%253D849%26rft.atitle%253DA%2Bmixed%2Bmodel%2BQTL%2Banalysis%2Bfor%2Bsugarcane%2Bmultiple-harvest-location%2Btrial%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1748-8%26rft_id%253Dinfo%253Apmid%252F22159754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1748-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22159754&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.48" data-doi="10.1371/journal.pone.0150506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perseguini</span>  <span class="cit-name-given-names">J. M. K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oblessuc</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">J. R. B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiorato</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association studies of anthracnose and angular leaf spot resistance in common bean (<em>Phaseolus vulgaris</em> L.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0150506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0150506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0150506%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0150506%26rft_id%253Dinfo%253Apmid%252F26930078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0150506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26930078&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.49" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2841.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">SELEGEN-REML/BLUP: Sistema estatístico e seleção genética computadorizada via modelos lineares mistos</span>, <span class="cit-edition">1st</span> edn. <span class="cit-publ-name">Embrapa Florestas</span>: <span class="cit-publ-loc">Colombo</span>. Available from: [<a href="http://www.det.ufv.br/ppestbio/docs/Software_Selegen_RemlBlup.zip">http://www.det.ufv.br/ppestbio/docs/Software_Selegen_RemlBlup.zip</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.51" data-doi="10.1111/nph.14266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azevedo</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Regional heritability mapping and genome-wide association identify loci for complex growth, wood and disease resistance traits in <em>Eucalyptus</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">1287</span>–<span class="cit-lpage">1300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14266</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D213%26rft.spage%253D1287%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.52" data-doi="10.1055/s-2006-924561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riemann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutjahr</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muramatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GER1, a GDSL motif-encoding gene from rice is a novel early light- and jasmonate-induced gene.</span> <abbr class="cit-jnl-abbrev">Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1055/s-2006-924561</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bbiology%2B%2528Stuttgart%252C%2BGermany%2529%26rft.stitle%253DPlant%2BBiol%2B%2528Stuttg%2529%26rft.aulast%253DRiemann%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D40%26rft.atitle%253DGER1%252C%2Ba%2BGDSL%2Bmotif-encoding%2Bgene%2Bfrom%2Brice%2Bis%2Ba%2Bnovel%2Bearly%2Blight-%2Band%2Bjasmonate-induced%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1055%252Fs-2006-924561%26rft_id%253Dinfo%253Apmid%252F17048141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1055/s-2006-924561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17048141&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.53" data-doi="10.1038/hdy.2012.90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riggio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matika</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stear</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association and regional heritability mapping to identify loci underlying variation in nematode resistance and body weight in Scottish Blackface lambs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">420</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2012.90</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D420%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.54" data-doi="10.1111/nph.13713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitocchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellucci</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biagetti</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Landscape genetics, adaptive diversity, and population structure in <em>Phaseolus vulgaris</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">1781</span>–<span class="cit-lpage">1794</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13713</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D209%26rft.spage%253D1781%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13713&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.55" data-doi="10.1371/journal.pone.0144960"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez-Sevilla</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botella</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaston</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folta</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity Arrays Technology (DArT) Marker platforms for diversity analysis and linkage mapping in a complex crop, the octoploid cultivated strawberry (Fragaria x ananassa).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0144960</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0144960</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0144960%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0144960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0144960&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.56" data-doi="10.1038/ng.3008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reference genome for common bean and genome-wide analysis of dual domestications.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">713</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3008%26rft_id%253Dinfo%253Apmid%252F24908249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24908249&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.57" data-doi="10.1038/hdy.2015.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirali</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Regional heritability mapping method helps explain missing heritability of blood lipid traits in isolated populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinemann</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paz</span>  <span class="cit-name-given-names">R. L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amorim</span>  <span class="cit-name-given-names">A. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Informações meteorológicas para pesquisa e planejamento agrícola, referentes ao ano de 2009, do município de Santo Antônio de Goiás, GO. EMBRAPA (CNPAF): Santo Antônio de Goiás.</span> <abbr class="cit-jnl-abbrev">Documentos</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDocumentos%26rft.volume%253D256%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.59" data-doi="10.4238/2013.January.30.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">V. M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menezes-Júnior</span>  <span class="cit-name-given-names">J. A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">P. C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">J. E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic improvement of plant architecture in the common bean.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">3093</span>–<span class="cit-lpage">3102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4238/2013.January.30.8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%26rft.volume%253D12%26rft.spage%253D3093%26rft_id%253Dinfo%253Adoi%252F10.4238%252F2013.January.30.8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4238/2013.January.30.8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">E-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> SNP assay development for linkage map construction, anchoring whole genome sequence and other genetic and genomic applications in common bean. G3 (Bethesda)5: 2285–2290.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wander</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Aspectos econômicos da produção de feijão no Brasil.</span> <abbr class="cit-jnl-abbrev">Rev. Polít. Agríc.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BPol%25EDt.%2BAgr%25EDc.%26rft.volume%253D23%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">T. L. P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">J. G. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">EMBRAPA common bean breeding program: a current overview.</span> <abbr class="cit-jnl-abbrev">Annu. Rep. Bean Improv. Coop.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRep.%2BBean%2BImprov.%2BCoop.%26rft.volume%253D55%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.63" data-doi="10.1038/hdy.2015.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redoña</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide prediction models that incorporate de novo GWAS are a powerful new tool for tropical rice improvement.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D395%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.64" data-doi="10.1590/S1415-47571999000400019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramalho</span>  <span class="cit-name-given-names">M. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abreu</span>  <span class="cit-name-given-names">A. F. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic control of plant architecture in the common bean (<em>Phaseolus vulgaris</em> L.).</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S1415-47571999000400019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BBiol.%26rft.volume%253D22%26rft.spage%253D577%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS1415-47571999000400019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S1415-47571999000400019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2841.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">Direct harvest of dry edible beans: A Nebraska perspective</span>, <span class="cit-publ-name">University of Nebraska Panhandle Research and Extension Center. Available from</span>. [<a href="http://extensionpublications.unl.edu/assets/pdf/ec309.pdf">http://extensionpublications.unl.edu/assets/pdf/ec309.pdf</a>].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.66" data-doi="10.1534/g3.117.300199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorwarth</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yousef</span>  <span class="cit-name-given-names">E. A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction and association mapping of curd-related traits in genebank accessions of cauliflower. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.117.300199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2841.67" data-doi="10.1007/978-94-015-9211-6_13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Integrated management of abiotic stresses</span>, pp. <span class="cit-fpage">331</span>–<span class="cit-lpage">370</span> in <span class="cit-source">Common bean improvement in the twenty-first century</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S. P.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dordrecht</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-94-015-9211-6_13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommon%2Bbean%2Bimprovement%2Bin%2Bthe%2Btwenty-first%2Bcentury%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-015-9211-6_13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-015-9211-6_13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.68" data-doi="10.2135/cropsci2014.11.0792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapp</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urrea</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miklas</span>  <span class="cit-name-given-names">P. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detection of major QTL for yield under multiple abiotic and terminal drought stress in a recombinant inbred dry bean population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.11.0792</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1596%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.11.0792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.11.0792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.69" data-doi="10.3389/fgene.2013.00232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uemoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitart</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The power of regional heritability analysis for rare and common variant detection: simulations and application to eye biometrical traits.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2013.00232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2013.00232%26rft_id%253Dinfo%253Apmid%252F24312116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2013.00232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24312116&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.70" data-doi="10.1186/s12863-015-0287-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uemoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Impact of QTL minor allele frequency on genomic evaluation using real genotype data and simulated phenotypes in Japanese Black cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0287-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0287-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0287-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.71" data-doi="10.1186/1753-6561-8-S5-S1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usai</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaspa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macciotta</span>  <span class="cit-name-given-names">N. P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">XVIth QTLMAS: simulated dataset and comparative analysis of submitted results for QTL mapping and genomic evaluation.</span> <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">S1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1753-6561-8-S5-S1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D8%26rft.spage%253DS1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-8-S5-S1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-8-S5-S1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.72" data-doi="10.1186/s12864-017-3805-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdisser</span>  <span class="cit-name-given-names">P. A. M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida Filho</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">B. S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coelho</span>  <span class="cit-name-given-names">G. R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">In-depth genome characterization of a Brazilian common bean core collection using DArTseq high-density SNP genotyping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3805-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3805-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3805-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.73" data-doi="10.1007/s00438-016-1182-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdisser</span>  <span class="cit-name-given-names">P. A. M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pappas</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Menezes</span>  <span class="cit-name-given-names">I. P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">B. S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SNP discovery in common bean by restriction-associated DNA (RAD) sequencing for genetic diversity and population structure analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">1277</span>–<span class="cit-lpage">1291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-016-1182-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D291%26rft.spage%253D1277%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-016-1182-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-016-1182-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.74" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.75" data-doi="10.3389/fpls.2015.00546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villordo-Pineda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Chavira</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giraldo-Carbajo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta-Gallegos</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caballero-Pérez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of novel drought-tolerant-associated SNPs in common bean (<em>Phaseolus vulgaris</em>).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">546</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.00546</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D546%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.00546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.00546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.76" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability in the genomics era concepts and misconceptions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.77" data-doi="10.1146/annurev.arplant.59.032607.092902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular basis of plant architecture.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.arplant.59.032607.092902</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DY.%26rft.volume%253D59%26rft.spage%253D253%26rft.epage%253D279%26rft.atitle%253DMolecular%2Bbasis%2Bof%2Bplant%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.59.032607.092902%26rft_id%253Dinfo%253Apmid%252F18444901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.59.032607.092902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18444901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256593200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.78" data-doi="10.1093/biomet/34.1-2.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">The generalization of ‘student’s’ problem when several different population variances are involved.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DWELCH%26rft.auinit1%253DB.%2BL.%26rft.volume%253D34%26rft.issue%253D1-2%26rft.spage%253D28%26rft.epage%253D35%26rft.atitle%253DThe%2Bgeneralisation%2Bof%2Bstudent%2527s%2Bproblems%2Bwhen%2Bseveral%2Bdifferent%2Bpopulation%2Bvariances%2Bare%2Binvolved.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F34.1-2.28%26rft_id%253Dinfo%253Apmid%252F20287819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/34.1-2.28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20287819&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1947UZ95100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.79" data-doi="10.1093/bioinformatics/btr346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willing</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreyer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Paired-end RAD-seq for de novo assembly and marker design without available reference.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2187</span>–<span class="cit-lpage">2193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr346%26rft_id%253Dinfo%253Apmid%252F21712251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21712251&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293620800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.80" data-doi="10.1007/s10681-011-0369-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping QTL for seed yield and canning quality following processing of black bean (<em>Phaseolus vulgaris</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-011-0369-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D179%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-011-0369-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-011-0369-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.81" data-doi="10.1270/jsbbs.64.300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamazaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyoshi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishimoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">H. Quantitative trait loci associated with lodging tolerance in soybean cultivar “Toyoharuka”.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1270/jsbbs.64.300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D64%26rft.spage%253D300%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.64.300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.64.300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.82" data-doi="10.1111/pbr.12305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sitoe</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic dissection of the relationship between plant architecture and yield component traits in soybean (<em>Glycine max</em>) by association analysis across multiple environments.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D134%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2841.83" data-doi="10.1371/journal.pone.0156391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuiderveen</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padder</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamfwa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association study of anthracnose resistance in Andean beans (<em>Phaseolus vulgaris</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0156391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0156391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0156391%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2841.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2833" title="Generation and Phenotype Identification of PAX4 Gene Knockout Rabbit by CRISPR/Cas9 System" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2855" title="CORRIGENDUM" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2841" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eCAMx_9WbhF-As0V0c6CYMMYdku3HPeo2lG_hpk9uLA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2841" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2841" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2841" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in Phaseolus vulgaris</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490464" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XiHlm-125-kS0qBM8DyZ9OQzTyLkCT7JDG_-tu46PsQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2841.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2841" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pS3ODzrfng3c0ODzSVd4g--4IWa9Yq4Zv-8g_PWXG0U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2841" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2841.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490464" id="node490464--2" data-pisa="ggg;8/8/2841" data-pisa-master="ggg;g3.118.200493" data-apath="/ggg/8/8/2841.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in <em>Phaseolus vulgaris</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rafael T. Resende</span>, <span class="highwire-citation-author" data-delta="1">Marcos Deon V. de Resende</span>, <span class="highwire-citation-author" data-delta="2">Camila F. Azevedo</span>, <span class="highwire-citation-author" data-delta="3">Fabyano Fonseca e Silva</span>, <span class="highwire-citation-author" data-delta="4">Leonardo C. Melo</span>, <span class="highwire-citation-author" data-delta="5">Helton S. Pereira</span>, <span class="highwire-citation-author" data-delta="6">Thiago Lívio P. O. Souza</span>, <span class="highwire-citation-author" data-delta="7">Paula Arielle M. R. Valdisser</span>, <span class="highwire-citation-author" data-delta="8">Claudio Brondani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-0644-5652" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rosana Pereira Vianello</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2841-2854;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490464--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rafael T. Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rafael%2BT.%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Resende%20RT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARafael%2BT.%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcos Deon V. de Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>EMBRAPA Florestas, Colombo, PR 83411-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BDeon%2BV.%2Bde%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Resende%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BDeon%2BV.%2Bde%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Camila F. Azevedo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camila%2BF.%2BAzevedo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azevedo%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamila%2BF.%2BAzevedo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fabyano Fonseca e Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabyano%2BFonseca%2Be%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonseca%20e%20Silva%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFabyano%2BFonseca%2Be%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leonardo C. Melo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonardo%2BC.%2BMelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melo%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonardo%2BC.%2BMelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Helton S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helton%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelton%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thiago Lívio P. O. Souza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thiago%2BL%C3%ADvio%2BP.%2BO.%2BSouza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Souza%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThiago%2BL%25C3%25ADvio%2BP.%2BO.%2BSouza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paula Arielle M. R. Valdisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paula%2BArielle%2BM.%2BR.%2BValdisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdisser%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaula%2BArielle%2BM.%2BR.%2BValdisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Claudio Brondani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio%2BBrondani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brondani%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudio%2BBrondani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rosana Pereira Vianello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosana%2BPereira%2BVianello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vianello%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARosana%2BPereira%2BVianello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0644-5652" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rosana Pereira Vianello</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rosana.vianello@embrapa.br" class="" data-icon-position="" data-hide-link-title="0">rosana.vianello@embrapa.br</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2841" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2841&amp;title=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2841&amp;title=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2841&amp;title=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2841&amp;text=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2841&amp;title=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2841&amp;t=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2841&amp;title=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2841&amp;title=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490464/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490464" id="node490464--3" data-pisa="ggg;8/8/2841" data-pisa-master="ggg;g3.118.200493" data-apath="/ggg/8/8/2841.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association and Regional Heritability Mapping of Plant Architecture, Lodging and Productivity in <em>Phaseolus vulgaris</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rafael T. Resende</span>, <span class="highwire-citation-author" data-delta="1">Marcos Deon V. de Resende</span>, <span class="highwire-citation-author" data-delta="2">Camila F. Azevedo</span>, <span class="highwire-citation-author" data-delta="3">Fabyano Fonseca e Silva</span>, <span class="highwire-citation-author" data-delta="4">Leonardo C. Melo</span>, <span class="highwire-citation-author" data-delta="5">Helton S. Pereira</span>, <span class="highwire-citation-author" data-delta="6">Thiago Lívio P. O. Souza</span>, <span class="highwire-citation-author" data-delta="7">Paula Arielle M. R. Valdisser</span>, <span class="highwire-citation-author" data-delta="8">Claudio Brondani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-0644-5652" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rosana Pereira Vianello</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2841-2854;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490464--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rafael T. Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rafael%2BT.%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Resende%20RT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARafael%2BT.%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcos Deon V. de Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forestry</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>EMBRAPA Florestas, Colombo, PR 83411-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BDeon%2BV.%2Bde%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Resende%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BDeon%2BV.%2Bde%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Camila F. Azevedo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camila%2BF.%2BAzevedo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azevedo%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamila%2BF.%2BAzevedo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fabyano Fonseca e Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science, Universidade Federal de Viçosa, Viçosa, MG 36570-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabyano%2BFonseca%2Be%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonseca%20e%20Silva%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFabyano%2BFonseca%2Be%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leonardo C. Melo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonardo%2BC.%2BMelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melo%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonardo%2BC.%2BMelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Helton S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helton%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelton%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Thiago Lívio P. O. Souza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Common Bean Breeding Program</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thiago%2BL%C3%ADvio%2BP.%2BO.%2BSouza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Souza%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThiago%2BL%25C3%25ADvio%2BP.%2BO.%2BSouza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paula Arielle M. R. Valdisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paula%2BArielle%2BM.%2BR.%2BValdisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdisser%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaula%2BArielle%2BM.%2BR.%2BValdisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Claudio Brondani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio%2BBrondani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brondani%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudio%2BBrondani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rosana Pereira Vianello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Laboratory of Biotechnology, EMBRAPA Arroz e Feijão, Santo Antônio de Goiás, GO 75375-000, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosana%2BPereira%2BVianello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vianello%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARosana%2BPereira%2BVianello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0644-5652" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rosana Pereira Vianello</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rosana.vianello@embrapa.br" class="" data-icon-position="" data-hide-link-title="0">rosana.vianello@embrapa.br</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2841&amp;rft.epage=2854&amp;rft.atitle=Genome-Wide%20Association%20and%20Regional%20Heritability%20Mapping%20of%20Plant%20Architecture%2C%20Lodging%20and%20Productivity%20in%20Phaseolus%20vulgaris&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Resende%2C+Rafael+T.&amp;rft.au=de+Resende%2C+Marcos+Deon+V.&amp;rft.au=Azevedo%2C+Camila+F.&amp;rft.au=Fonseca+e+Silva%2C+Fabyano&amp;rft.au=Melo%2C+Leonardo+C.&amp;rft.au=Pereira%2C+Helton+S.&amp;rft.au=Souza%2C+Thiago+L%C3%ADvio+P.+O.&amp;rft.au=Valdisser%2C+Paula+Arielle+M.+R.&amp;rft.au=Brondani%2C+Claudio&amp;rft.au=Vianello%2C+Rosana+Pereira"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490464/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490464/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490464/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490464/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490464/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490464/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490464/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490464/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490464/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490464/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490464/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490464/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2841" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2841.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2841.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2841.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2841.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
