<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2673" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2673.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2673.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2673.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2673.atom" />
<meta name="HW.pisa" content="ggg;8/8/2673" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Detecting de Novo Homoeologous Recombination Events in Cultivated Brassica napus Using a Genome-Wide SNP Array" />
<meta name="DC.Identifier" content="10.1534/g3.118.200118" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Higgins et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The heavy selection pressure due to intensive breeding of Brassica napus has created a narrow gene pool, limiting the ability to produce improved varieties through crosses between B. napus cultivars. One mechanism that has contributed to the adaptation of important agronomic traits in the allotetraploid B. napus has been chromosomal rearrangements resulting from homoeologous recombination between the constituent A and C diploid genomes. Determining the rate and distribution of such events in natural B. napus will assist efforts to understand and potentially manipulate this phenomenon. The Brassica high-density 60K SNP array, which provides genome-wide coverage for assessment of recombination events, was used to assay 254 individuals derived from 11 diverse cultivated spring type B. napus . These analyses identified reciprocal allele gain and loss between the A and C genomes and allowed visualization of de novo homoeologous recombination events across the B. napus genome. The events ranged from loss/gain of 0.09 Mb to entire chromosomes, with almost 5% aneuploidy observed across all gametes. There was a bias toward sub-telomeric exchanges leading to genome homogenization at chromosome termini. The A genome replaced the C genome in 66% of events, and also featured more dominantly in gain of whole chromosomes. These analyses indicate de novo homoeologous recombination is a continuous source of variation in established Brassica napus and the rate of observed events appears to vary with genetic background. The Brassica 60K SNP array will be a useful tool in further study and manipulation of this phenomenon." />
<meta name="DC.Contributor" content="Erin E. Higgins" />
<meta name="DC.Contributor" content="Wayne E. Clarke" />
<meta name="DC.Contributor" content="Elaine C. Howell" />
<meta name="DC.Contributor" content="Susan J. Armstrong" />
<meta name="DC.Contributor" content="Isobel A. P. Parkin" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Detecting de Novo Homoeologous Recombination Events in Cultivated Brassica napus Using a Genome-Wide SNP Array" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The heavy selection pressure due to intensive breeding of &lt;i&gt;Brassica napus&lt;/i&gt; has created a narrow gene pool, limiting the ability to produce improved varieties through crosses between &lt;i&gt;B. napus&lt;/i&gt; cultivars. One mechanism that has contributed to the adaptation of important agronomic traits in the allotetraploid &lt;i&gt;B. napus&lt;/i&gt; has been chromosomal rearrangements resulting from homoeologous recombination between the constituent A and C diploid genomes. Determining the rate and distribution of such events in natural &lt;i&gt;B. napus&lt;/i&gt; will assist efforts to understand and potentially manipulate this phenomenon. The Brassica high-density 60K SNP array, which provides genome-wide coverage for assessment of recombination events, was used to assay 254 individuals derived from 11 diverse cultivated spring type &lt;i&gt;B. napus&lt;/i&gt;. These analyses identified reciprocal allele gain and loss between the A and C genomes and allowed visualization of &lt;i&gt;de novo&lt;/i&gt; homoeologous recombination events across the &lt;i&gt;B. napus&lt;/i&gt; genome. The events ranged from loss/gain of 0.09 Mb to entire chromosomes, with almost 5% aneuploidy observed across all gametes. There was a bias toward sub-telomeric exchanges leading to genome homogenization at chromosome termini. The A genome replaced the C genome in 66% of events, and also featured more dominantly in gain of whole chromosomes. These analyses indicate &lt;i&gt;de novo&lt;/i&gt; homoeologous recombination is a continuous source of variation in established &lt;i&gt;Brassica napus&lt;/i&gt; and the rate of observed events appears to vary with genetic background. The Brassica 60K SNP array will be a useful tool in further study and manipulation of this phenomenon.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2673" />
<meta name="citation_id" content="8/8/2673" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2673" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2673.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2673.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2673.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200118" />
<meta name="citation_pmid" content="29907649" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2673" />
<meta name="citation_lastpage" content="2683" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Erin E. Higgins" />
<meta name="citation_author_institution" content="Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada" />
<meta name="citation_author" content="Wayne E. Clarke" />
<meta name="citation_author_institution" content="Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada" />
<meta name="citation_author" content="Elaine C. Howell" />
<meta name="citation_author_institution" content="School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK" />
<meta name="citation_author" content="Susan J. Armstrong" />
<meta name="citation_author_institution" content="School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7284-5702" />
<meta name="citation_author" content="Isobel A. P. Parkin" />
<meta name="citation_author_institution" content="Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada" />
<meta name="citation_author_email" content="isobel.parkin@agr.gc.ca" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=C. J. Allender;citation_author=G. J. King;citation_title=Origins of the amphiploid species Brassica napus L. investigated by chloroplast and nuclear molecular markers.;citation_pages=54-54;citation_volume=10;citation_year=2010;citation_pmid=20350303;citation_doi=10.1186/1471-2229-10-54" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Genet. Cytol.;citation_author=T. Attia;citation_author=G. Röbbelen;citation_title=Cytogenetic relationship within cultivated Brassica analyzed in amphihaploids from the three diploid ancestors.;citation_pages=323-329;citation_volume=28;citation_year=1986;citation_doi=10.1139/g86-048" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Genet. Cytol.;citation_author=T. Attia;citation_author=G. Röbbelen;citation_title=Meiotic pairing in haploids and amphidiploids of spontaneous vs. synthetic origin in rape, Brassica napus L.;citation_pages=330-334;citation_volume=28;citation_year=1986;citation_doi=10.1139/g86-049" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Bhullar;citation_author=R. Nagarajan;citation_author=H. Bennypaul;citation_author=G. K. Sidhu;citation_author=G. Sidhu;citation_title=Silencing of a metaphase I-specific gene results in a phenotype similar to that of the Pairing homeologous 1 (Ph1) gene mutations.;citation_volume=111;citation_year=2014;citation_pmid=25232038;citation_doi=10.1073/pnas.1416241111" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Bowers;citation_author=B. A. Chapman;citation_author=J. Rong;citation_author=A. H. Paterson;citation_title=Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events.;citation_pages=433-438;citation_volume=422;citation_year=2003;citation_issue=6930;citation_pmid=12660784;citation_doi=10.1038/nature01521" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Bus;citation_author=N. Korber;citation_author=R. J. Snowdon;citation_author=B. Stich;citation_title=Patterns of molecular variation in a species-wide germplasm set of Brassica napus.;citation_pages=1413-1423;citation_volume=123;citation_year=2011;citation_issue=8;citation_pmid=21847624;citation_doi=10.1007/s00122-011-1676-7" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Chalhoub;citation_author=F. Denoeud;citation_author=S. Liu;citation_author=I. A. Parkin;citation_author=H. Tang;citation_title=Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome;citation_pages=950-953;citation_volume=345;citation_year=2014;citation_issue=6199;citation_pmid=25146293;citation_doi=10.1126/science.1253435" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. Cifuentes;citation_author=L. Grandont;citation_author=G. Moore;citation_author=A. M. Chèvre;citation_author=E. Jenczewski;citation_title=Genetic regulation of meiosis in polyploid species: new insights into an old question.;citation_pages=29-36;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=19912546;citation_doi=10.1111/j.1469-8137.2009.03084.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. E. Clarke;citation_author=E. E. Higgins;citation_author=J. Plieske;citation_author=R. Wieseke;citation_author=C. Sidebottom;citation_title=A high-density SNP genotyping array for Brassica napus and its ancestral diploid species based on optimised selection of single-locus markers in the allotetraploid genome.;citation_pages=1887-1899;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2746-7" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. J. Doyle;citation_author=L. E. Flagel;citation_author=A. H. Paterson;citation_author=R. A. Rapp;citation_author=D. E. Soltis;citation_title=Evolutionary genetics of genome merger and doubling in plants.;citation_pages=443-461;citation_volume=42;citation_year=2008;citation_pmid=18983261;citation_doi=10.1146/annurev.genet.42.110807.091524" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=B. Friebe;citation_author=P. Zhang;citation_author=G. Linc;citation_author=B. S. Gill;citation_title=Robertsonian translocations in wheat arise by centric misdivision of univalents at anaphase I and rejoining of broken centromeres during interkinesis of meiosis II.;citation_pages=293-297;citation_volume=109;citation_year=2005;citation_issue=1-3;citation_pmid=15753589;citation_doi=10.1159/000082412" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=L. Grandont;citation_author=N. Cuñado;citation_author=O. Coriton;citation_author=V. Huteau;citation_author=F. Eber;citation_title=Homoeologous Chromosome Sorting and Progression of Meiotic Recombination in Brassica napus: Ploidy Does Matter!;citation_volume=26;citation_year=2014;citation_pmid=24737673;citation_doi=10.1105/tpc.114.122788" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=E. Greer;citation_author=A. C. Martin;citation_author=A. Pendle;citation_author=I. Colas;citation_author=A. M. Jones;citation_title=The Ph1 locus suppresses Cdk2-type activity during premeiosis and meiosis in wheat.;citation_volume=24;citation_year=2012;citation_pmid=22274628;citation_doi=10.1105/tpc.111.094771" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=S. V. Hatzig;citation_author=M. Frisch;citation_author=F. Breuer;citation_author=N. Nesi;citation_author=S. Ducournau;citation_title=Genome-wide association mapping unravels the genetic control of seed germination and vigor in Brassica napus.;citation_pages=221;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.00221" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=Z. He;citation_author=L. Wang;citation_author=L. Harper Andrea;citation_author=L. Havlickova;citation_author=K. Pradhan Akshay;citation_title=Extensive homoeologous genome exchanges in allopolyploid crops revealed by mRNAseq‐based visualization.;citation_pages=594-604;citation_volume=15;citation_year=2016;citation_doi=10.1111/pbi.12657" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. C. Howell;citation_author=M. J. Kearsey;citation_author=G. H. Jones;citation_author=G. J. King;citation_author=S. J. Armstrong;citation_title=A and C genome distinction and chromosome identification in Brassica napus by sequential fluorescence in situ hybridization and genomic in situ hybridization.;citation_volume=180;citation_year=2008;citation_pmid=18845839;citation_doi=10.1534/genetics.108.095893" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=E. Jenczewski;citation_author=K. Alix;citation_title=From Diploids to Allopolyploids: The Emergence of Efficient Pairing Control Genes in Plants.;citation_pages=21-45;citation_volume=23;citation_year=2004;citation_doi=10.1080/07352680490273239" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Jenczewski;citation_author=F. Eber;citation_author=A. Grimaud;citation_author=S. Huet;citation_author=M. O. Lucas;citation_title=PrBn, a Major Gene Controlling Homeologous Pairing in Oilseed Rape (Brassica napus) Haploids;citation_pages=645-653;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807785" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_author=W. J. Kent;citation_title=BLAT—The BLAST-Like Alignment Tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=N. Körber;citation_author=A. Bus;citation_author=J. Li;citation_author=J. Higgins;citation_author=I. Bancroft;citation_title=Seedling development traits in Brassica napus examined by gene expression analysis and association mapping.;citation_pages=136;citation_volume=15;citation_year=2015;citation_doi=10.1186/s12870-015-0496-3" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. I. Krzywinski;citation_author=J. E. Schein;citation_author=I. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: An information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_author=P. Lashermes;citation_author=Y. Hueber;citation_author=M.-C. Combes;citation_author=D. Severac;citation_author=A. Dereeper;citation_year=2016;citation_pmid=27440920;citation_doi=10.1534/g3.116.030858" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. R. Leitch;citation_author=I. J. Leitch;citation_title=Genomic Plasticity and the Diversity of Polyploid Plants;citation_pages=481-483;citation_volume=320;citation_year=2008;citation_issue=5875;citation_pmid=18436776;citation_doi=10.1126/science.1153585" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=K. Y. Lim;citation_author=A. Kovarik;citation_author=R. Matyasek;citation_author=W. Chase Mark;citation_author=J. Clarkson James;citation_title=Sequence of events leading to near-complete genome turnover in allopolyploid Nicotiana within five million years.;citation_pages=756-763;citation_volume=175;citation_year=2007;citation_issue=4;citation_pmid=17688590;citation_doi=10.1111/j.1469-8137.2007.02121.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Liu;citation_author=C. Qu;citation_author=B. Wittkop;citation_author=B. Yi;citation_author=Y. Xiao;citation_title=A high-density SNP map for accurate mapping of seed fibre QTL in Brassica napus L.;citation_pages=e83052;citation_volume=8;citation_year=2013;citation_pmid=24386142;citation_doi=10.1371/journal.pone.0083052" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=A. Lloyd;citation_author=A. Blary;citation_author=D. Charif;citation_author=C. Charpentier;citation_author=J. Tran;citation_title=Homoeologous exchanges cause extensive dosage‐dependent gene expression changes in an allopolyploid crop.;citation_pages=367-377;citation_volume=217;citation_year=2017;citation_doi=10.1111/nph.14836" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. Lysak;citation_author=M. A. Koch;citation_author=A. Pecinka;citation_author=I. Schubert;citation_title=Chromosome triplication found across the tribe Brassiceae.;citation_volume=15;citation_year=2005;citation_pmid=15781573;citation_doi=10.1101/gr.3531105" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=A. C. Martín;citation_author=M.-D. Rey;citation_author=P. Shaw;citation_author=G. Moore;citation_title=Dual effect of the wheat Ph1 locus on chromosome synapsis and crossover.;citation_pages=669-680;citation_volume=126;citation_year=2017;citation_doi=10.1007/s00412-017-0630-0" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=A. S. Mason;citation_author=J. Batley;citation_author=P. E. Bayer;citation_author=A. Hayward;citation_author=W. A. Cowling;citation_title=High-resolution molecular karyotyping uncovers pairing between ancestrally related Brassica chromosomes.;citation_pages=964-974;citation_volume=202;citation_year=2014;citation_pmid=24471809;citation_doi=10.1111/nph.12706" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. S. Mason;citation_author=E. E. Higgins;citation_author=R. J. Snowdon;citation_author=J. Batley;citation_author=A. Stein;citation_title=A user guide to the Brassica 60K Illumina Infinium SNP genotyping array.;citation_pages=621-633;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2849-1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=A. S. Mason;citation_author=J. Zhang;citation_author=R. Tollenaere;citation_author=P. Vasquez Teuber;citation_author=J. Dalton-Morgan;citation_title=High-throughput genotyping for species identification and diversity assessment in germplasm collections.;citation_pages=1091-1101;citation_volume=15;citation_year=2015;citation_doi=10.1111/1755-0998.12379" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Masterson;citation_title=Stomatal Size in Fossil Plants: Evidence for Polyploidy in Majority of Angiosperms;citation_pages=421-424;citation_volume=264;citation_year=1994;citation_issue=5157;citation_pmid=17836906;citation_doi=10.1126/science.264.5157.421;citation_georef=1994028739" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. G. Murray;citation_author=W. F. Thompson;citation_title=Rapid isolation of high molecular weight plant DNA;citation_pages=4321-4326;citation_volume=8;citation_year=1980;citation_issue=19;citation_pmid=7433111;citation_doi=10.1093/nar/8.19.4321" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. R. Nguepjop;citation_author=H.-A. Tossim;citation_author=J. M. Bell;citation_author=J.-F. Rami;citation_author=S. Sharma;citation_title=Evidence of Genomic Exchanges between Homeologous Chromosomes in a Cross of Peanut with Newly Synthetized Allotetraploid Hybrids.;citation_pages=1635;citation_volume=7;citation_year=2016;citation_doi=10.3389/fpls.2016.01635" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. D. Nicolas;citation_author=M. Leflon;citation_author=H. Monod;citation_author=F. Eber;citation_author=O. Coriton;citation_title=Genetic Regulation of Meiotic Cross-Overs between Related Genomes in Brassica napus Haploids and Hybrids.;citation_volume=21;citation_year=2009;citation_pmid=19190241;citation_doi=10.1105/tpc.108.062273" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. D. Nicolas;citation_author=H. Monod;citation_author=F. Eber;citation_author=A. M. Chèvre;citation_author=E. Jenczewski;citation_title=Non-random distribution of extensive chromosome rearrangements in Brassica napus depends on genome organization.;citation_pages=691-703;citation_volume=70;citation_year=2012;citation_issue=4;citation_pmid=22268419;citation_doi=10.1111/j.1365-313X.2012.04914.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=I. A. Parkin;citation_author=C. Koh;citation_author=H. Tang;citation_author=S. J. Robinson;citation_author=S. Kagale;citation_title=Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid Brassica oleracea.;citation_pages=R77;citation_volume=15;citation_year=2014;citation_pmid=24916971;citation_doi=10.1186/gb-2014-15-6-r77" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=I. A. Parkin;citation_author=A. G. Sharpe;citation_author=D. J. Keith;citation_author=D. J. Lydiate;citation_title=Identification of the A and C genomes of amphidiploid Brassica napus (oilseed rape).;citation_pages=1122-1131;citation_volume=38;citation_year=1995;citation_issue=6;citation_pmid=18470236;citation_doi=10.1139/g95-149" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. A. P. Parkin;citation_author=S. M. Gulden;citation_author=A. G. Sharpe;citation_author=L. Lukens;citation_author=M. Trick;citation_title=Segmental structure of the Brassica napus genome based on comparative analysis with Arabidopsis thaliana.;citation_volume=171;citation_year=2005;citation_pmid=16020789;citation_doi=10.1534/genetics.105.042093" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M.-D. Rey;citation_author=A. C. Martín;citation_author=J. Higgins;citation_author=D. Swarbreck;citation_author=C. Uauy;citation_title=Exploiting the ZIP4 homologue within the wheat Ph1 locus has identified two lines exhibiting homoeologous crossover in wheat-wild relative hybrids.;citation_pages=95;citation_volume=37;citation_year=2017;citation_doi=10.1007/s11032-017-0700-2" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. Riley;citation_author=V. Chapman;citation_title=Genetic control of the cytologically diploid behaviour of hexaploid wheat.;citation_pages=713-715;citation_volume=182;citation_year=1958;citation_doi=10.1038/182713a0" />
<meta name="citation_reference" content="citation_author=M. Rousseau-Gueutin;citation_author=J. Morice;citation_author=O. Coriton;citation_author=V. Huteau;citation_author=G. Trotoux;citation_year=2017;citation_pmid=28007837;citation_doi=10.1534/g3.116.036517" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=B. Samans;citation_author=B. Chalhoub;citation_author=R. J. Snowdon;citation_title=Surviving a Genome Collision: Genomic Signatures of Allopolyploidization in the Recent Crop Species Brassica napus.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2017.02.0013" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=M. E. Schranz;citation_author=M. A. Lysak;citation_author=T. Mitchell-Olds;citation_title=The ABC's of comparative genomics in the Brassicaceae: building blocks of crucifer genomes.;citation_pages=535-542;citation_volume=11;citation_year=2006;citation_issue=11;citation_pmid=17029932;citation_doi=10.1016/j.tplants.2006.09.002" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=A. G. Sharpe;citation_author=I. A. Parkin;citation_author=D. J. Keith;citation_author=D. J. Lydiate;citation_title=Frequent nonreciprocal translocations in the amphidiploid genome of oilseed rape (Brassica napus).;citation_pages=1112-1121;citation_volume=38;citation_year=1995;citation_issue=6;citation_pmid=18470235;citation_doi=10.1139/g95-148" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=R. J. Snowdon;citation_title=Cytogenetics and genome analysis in Brassica crops.;citation_pages=85-95;citation_volume=15;citation_year=2007;citation_issue=1;citation_pmid=17295129;citation_doi=10.1007/s10577-006-1105-y" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. Stein;citation_author=O. Coriton;citation_author=M. Rousseau-Gueutin;citation_author=B. Samans;citation_author=S. V. Schiessl;citation_title=Mapping of homoeologous chromosome exchanges influencing quantitative trait variation in Brassica napus.;citation_pages=1478-1489;citation_volume=15;citation_year=2017;citation_doi=10.1111/pbi.12732" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=E. Szadkowski;citation_author=F. Eber;citation_author=V. Huteau;citation_author=M. Lode;citation_author=C. Huneau;citation_title=The first meiosis of resynthesized Brassica napus, a genome blender.;citation_pages=102-112;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20149113;citation_doi=10.1111/j.1469-8137.2010.03182.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Udall;citation_author=P. A. Quijada;citation_author=T. C. Osborn;citation_title=Detection of chromosomal rearrangements derived from homologous recombination in four mapping populations of Brassica napus L.;citation_volume=169;citation_year=2005;citation_pmid=15520255;citation_doi=10.1534/genetics.104.033209" />
<meta name="citation_reference" content="U, N., 1935 Genome analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization. Japanese J. of Botany 7: 389–452." />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Wang;citation_author=H. Wang;citation_author=J. Wang;citation_author=R. Sun;citation_author=J. Wu;citation_title=The genome of the mesopolyploid crop species Brassica rapa.;citation_pages=1035-1039;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21873998;citation_doi=10.1038/ng.919" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=X. Wang;citation_author=K. Yu;citation_author=H. Li;citation_author=Q. Peng;citation_author=F. Chen;citation_title=High-Density SNP Map Construction and QTL Identification for the Apetalous Character in Brassica napus L.;citation_pages=1164;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.01164" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=J. F. Wendel;citation_title=Genome evolution in polyploids.;citation_pages=225-249;citation_volume=42;citation_year=2000;citation_issue=1;citation_pmid=10688139;citation_doi=10.1023/A:1006392424384" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Xiong;citation_author=R. T. Gaeta;citation_author=J. C. Pires;citation_title=Homoeologous shuffling and chromosome compensation maintain genome balance in resynthesized allopolyploid Brassica napus.;citation_volume=108;citation_year=2011;citation_pmid=21512129;citation_doi=10.1073/pnas.1014138108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Xiong;citation_author=J. C. Pires;citation_title=Karyotype and identification of all homoeologous chromosomes of allopolyploid Brassica napus and its diploid progenitors.;citation_volume=187;citation_year=2011;citation_pmid=21041557;citation_doi=10.1534/genetics.110.122473" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=Y. Yang;citation_author=Y. Shen;citation_author=S. Li;citation_author=X. Ge;citation_author=Z. Li;citation_title=High Density Linkage Map Construction and QTL Detection for Three Silique-Related Traits in Orychophragmus violaceus Derived Brassica napus Population.;citation_pages=1512;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01512" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=L. Zamariola;citation_author=C. L. Tiang;citation_author=N. De Storme;citation_author=W. Pawlowski;citation_author=D. Geelen;citation_title=Chromosome segregation in plant meiosis.;citation_pages=279;citation_volume=5;citation_year=2014;citation_pmid=24987397;citation_doi=10.3389/fpls.2014.00279" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Detecting de Novo Homoeologous Recombination Events in Cultivated Brassica napus Using a Genome-Wide SNP Array" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="The heavy selection pressure due to intensive breeding of Brassica napus has created a narrow gene pool, limiting the ability to produce improved varieties through crosses between B. napus cultivars. One mechanism that has contributed to the adaptation of important agronomic traits in the allotetraploid B. napus has been chromosomal rearrangements resulting from homoeologous recombination between the constituent A and C diploid genomes. Determining the rate and distribution of such events in natural B. napus will assist efforts to understand and potentially manipulate this phenomenon. The Brassica high-density 60K SNP array, which provides genome-wide coverage for assessment of recombination events, was used to assay 254 individuals derived from 11 diverse cultivated spring type B. napus . These analyses identified reciprocal allele gain and loss between the A and C genomes and allowed visualization of de novo homoeologous recombination events across the B. napus genome. The events ranged from loss/gain of 0.09 Mb to entire chromosomes, with almost 5% aneuploidy observed across all gametes. There was a bias toward sub-telomeric exchanges leading to genome homogenization at chromosome termini. The A genome replaced the C genome in 66% of events, and also featured more dominantly in gain of whole chromosomes. These analyses indicate de novo homoeologous recombination is a continuous source of variation in established Brassica napus and the rate of observed events appears to vary with genetic background. The Brassica 60K SNP array will be a useful tool in further study and manipulation of this phenomenon." />
<meta name="og-title" property="og:title" content="Detecting de Novo Homoeologous Recombination Events in Cultivated Brassica napus Using a Genome-Wide SNP Array" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2673" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The heavy selection pressure due to intensive breeding of Brassica napus has created a narrow gene pool, limiting the ability to produce improved varieties through crosses between B. napus cultivars. One mechanism that has contributed to the adaptation of important agronomic traits in the allotetraploid B. napus has been chromosomal rearrangements resulting from homoeologous recombination between the constituent A and C diploid genomes. Determining the rate and distribution of such events in natural B. napus will assist efforts to understand and potentially manipulate this phenomenon. The Brassica high-density 60K SNP array, which provides genome-wide coverage for assessment of recombination events, was used to assay 254 individuals derived from 11 diverse cultivated spring type B. napus . These analyses identified reciprocal allele gain and loss between the A and C genomes and allowed visualization of de novo homoeologous recombination events across the B. napus genome. The events ranged from loss/gain of 0.09 Mb to entire chromosomes, with almost 5% aneuploidy observed across all gametes. There was a bias toward sub-telomeric exchanges leading to genome homogenization at chromosome termini. The A genome replaced the C genome in 66% of events, and also featured more dominantly in gain of whole chromosomes. These analyses indicate de novo homoeologous recombination is a continuous source of variation in established Brassica napus and the rate of observed events appears to vary with genetic background. The Brassica 60K SNP array will be a useful tool in further study and manipulation of this phenomenon." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>Detecting de Novo Homoeologous Recombination Events in Cultivated Brassica napus Using a Genome-Wide SNP Array | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"T0ZZt_nmHH9MUYCHFUSiep_8y9oWVbEbMRFGWv6x41E","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490452","apath":"\/ggg\/8\/8\/2673.atom","pisa":"ggg;8\/8\/2673","ac":{"\/ggg\/8\/8\/2673.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2673.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2673"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2673"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2673":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MEjKbP3G36d\\\\\\\/C8uIZzxji6Bb2mJy6SQp0QTd3sYbbDtP+uPyGCudkzO00G3tFRv9oRY+ddvyxxQpA6JL+aWDJAxIRiPGh1hb0cOS9UcLlSR9aEwKjokn9GDGz+PMV79g7n3544qSxY06VocDFF+ETLOCKkjcgtmHtMsZK6i6aUbSzRYUJHVv5v41vAMlpwhfiFDqIvb6IbJAAYoZ\\\\\\\/S2+Sp8fB+7LH8QOVz6Q9dcKqJ5\\\\\\\/1IJGPqWctIq6h24rWytKBLvUtoHtJZipB3ZqpQ4A\\\\\\\/6G2w7Iihl7DzeJAF1biJhNSRbmpE+3mF1vK16EZqczPRWWSoXIZTJKgKeKy+CDzqZFew3xj0mCd2uF1nBEcJmvt3\\\\\\\/oTdq8P3mHC6t4Pu7Kx+nShAiEH7bnaaQPytjjxqM\\\\\\\/TCQtgmzCHTPNlapXNL6Ozum9l1d1e\\\\\\\/6eHT+dCUmHeWTzJKasEqpprS\\\\\\\/EdFBlqhFutpfA6qxv8ZZECSnBsWJuaPDzl\\\\\\\/BbYm36qkSOPVIE6cLrULM5RKMeTlNpACjM19nn4iQ93ivA1A328n51UUGWWXSHF4\\\\\\\/FCWydT4gh4eW86PwA3TJoFZaC6QFhgz2tuI3u+Vwvpi\\\\\\\/AUfXX0ZYRuCpIdrzOvJ2FLyBpiIpuBQdry+VLeuSNHkHY2DfbWXGaxb7vYQ5wgiOJgRyOrqe7ACCBjmXOEvai10Kewfz1meGezhXpGtDFcq0gvyeVRD+90sg7n6sQJzO84W5M\\\\\\\/I0sDBA7nKwzMHEIrVIE\\\\\\\/h1NOtcOns2Xib+xxejprSgbHpIDHVBK0Dn0WjDRMZJmXVeA6V2Ooc3wyhFOMXzMme3YpU00ZiQ9EvcziPG74CIQ7tbVStzo1OAHSRb2yIEUANMP4kZcGb1l1f6F67EAG9vJTY7+vbAFhbVZ8DE+zWhWpy8zcONM\\\\\\\/jKxKOJwNYD0lSXXGHADvHuricEcaIn+umEVLmLJWeddOg8q2wsYQzbqRWxWhue1EXgcww3JvcQZHdPWQqCHUyYxP6vuG1++4mlFfa+6fND1aXagEFhPWLhwjLhX6CrHgAwfDG70\\\\\\\/Q+H42AeNJ0EaCLbSaOrM5oSgUl7dE9x0KReEaf0Sc\\\\\\\/A+rfsBeTaOBN3vq\\\\\\\/eYsQ7m4xwRk108W+vVVWBBxB6tEyRkd6thht+vASE+Cld85VOO06e\\\\\\\/F00s+l2rRUpODbo1AUedMuKc+s8FEsDoSBYzGZyUVv1oKXHWvJsNBN0mz3LUqh2oSFRBjRDj8eGPoCZ7OJXCIkNxSz4OGZN9qk74TvZj7e790X7e\\\u0022,\\\u0022iv\\\u0022:\\\u00227vBtVKn5Sqbh71lB+TrMtg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227b645fe35d58f6c971d1d4bac2963614a88b28e51c4750d01a71616fdfe2b8bd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Qgdak41Gc1jm\\\\\\\/cqlEc\\\\\\\/RSDe+NtMUHqgCQ75G6ejRbAPgj9bFw3MX5CnlHt2lHgPYuxrmZVGGln5Ikq6QHnvy2BqewHKx5ambALDGufImc6OHPF56KMSp5WHoacDParSsMMJSvVtQdq2dlN\\\\\\\/AB3BAK9038XrMRO99jmWrIE5ktDLZSGZoFm93TTkGOh9ULFvckIKij\\\\\\\/\\\\\\\/eG8us4YazytfRy4yMPM++wy2a7pRj\\\\\\\/ArRqZzeMb6q3qLxQg7nTiB4XZwmLPJindxsCE0HHM20gT4E9KBOcv9JlEnkt5JX9LmYZnUhS+lM1+GDLELIndtaqDcFGbm9WvbOm8q42UZpIhOBmUwgrr0WO5+BrtJiKrzJZEe7rCrYEoeCFmRfHzlMQ9EoVUDxXW9cGObsGLAmTA2oRC1EjVqzM4MRhadJT0BcHqe9du3H9qonFqSeMZcUM9ylzDGS1oJ49OJPDGP+x0iKEcFykOVOgxLFxzEsmMukY4ullcr3pRgRNqBPjC6scqPbFemha5msTZnIp\\\\\\\/oZhkjclTo8lLbNoRAtLZ779KOCZNbhAilcWeiPPu7APyL62nEM1BKYk+UctbLXOXrrcvlwp5j3FN8grCn4ymbDY\\\\\\\/lc9yCQZIU1NSn0zE1TIzSxblDmnzAVVZltIln\\\\\\\/9aY8zseegY9paZKRkNqMWUAPEIXY8XECYGjzZCPRT8SRtNlNiqIFU\\\\\\\/U7c1ZSTVjnOKDiVEnz0R2EcE28ksvT5nASEA8yyzTYkRad1n0UlFq0VU0HC1N5AwOuZxPKGyRoSs2h25XzUdL2m18w1k2+9XpmRg0zRHXmHvjX1Sne6y4ttGNSn4OcezIM+R7QU1FgodvqU5A1d8KjSl\\\\\\\/T7WxrWNYniKMJzGTxaUN9soHiYMHH9XmjS\\\\\\\/l49O3xoJ2iPG7aYo+uIEC39HRXxlF3xJjPHrXIldwOmNPHtYlHDUqMNhX0HzowSG8IqjzJh4FWZugTzJJydRIv0agvViQjbiv+Zb3mLb57ysT\\\\\\\/FT8GsCUzvR7EhFncMmRzKBkXfIoDPbJEkm05vcRQ04LuN3LdLILaDzG80S39jV66Y7t+m\\\\\\\/jG5AmFubN86LAAsHg8KkLiUXJO4xPhmnPPgCdlprGubFQPouZtu6KAihTm2zr8LidO+PnnAOi\\\\\\\/7NbXkKjS2krkcdxKgXcEVKU3Y7hY4K3CCvlCfe5B4AeCZhrJHBsGigiGNMGG6tTpYWoxfhsUDhxTo8H0UKZxsTHQ6yJvHsTF5nULXSopnPOzQg\\\\\\\/OIogP4SSFM\\\\\\\/CQ19RJm156OH5mfNuE\\\\\\\/DTy9C6Iwea\\\\\\\/gUTHaKxQnHwo+5bmXnIT3agmoOEX3kU9Tn7vCr2IQZ99cR1pSqz9mAXJDDQKo7m2ugGEkIvx+JGX7myJP\\\\\\\/9mG1xfZcw5mbgAZxOyNQY1gery6He5KrpHQ+BFzKnG\\\\\\\/5dhgjojSX+sk9vZ4yn39KapjZI+CT4kvOQ3WD22QQjRJX24iSnf8c8IstJZtu4C46sxa6NpElmQvMcLVv\\\\\\\/83lFD1KgVUN38p6YSbfBJmuTPb1TQHiG1g+eK+vwi04e+cFXXBz2G8D9eAiJTJCB+JfrZv\\\\\\\/YLAJKY0qSH+CGKYPiTUxygNKkeDLcIhkNUlog+dFb1I1ldPahp0Zx4nJHPPxQJL9SdK9YFUFIOpn\\\\\\\/VPH8e3Ioe6cDptGrpMA+FuDyW5ZPUHvJEvWMF+EwBLRcjjcfqJisNXB3GqeVbeY0A2DVpEwUnstDWc7DAfVcMFyRuX5RKjO3N7IwY54J70rtkUXmGoFKlO0dUetwU2iB0NFsVN3SXGOHpRTWvgDoWZ6cMCi2GRweYz\\\\\\\/44trdXJx4o6JA3RcJiucgI6MAhdul5HmY40FM2Nv2XoDCl48hAAvwidW8Lx4FsXioYTGeAm+xV+suWfa8pp9sYjHDO3L2i3lX4xsdrdyuri+xses58XWFDc4BR0teSwSCl9CnJ34DlWwl7KsCffCTB9og5RRXWNSU17ZAgwpCWMGZwzq1OGPvoIVCt1oqM7+5bYb4sL94Qly0HWIQFeS4mqYnhDDZRF+aPRcHY\\\\\\\/H+PSQ6AuaGEMEwUvuc\\\\\\\/MP4uspBxXr6uqQByr1CCgdE2IL4Z4AmPGWO3KZKUbdclnQd8d2gzgtallvmpzJA9uLYRW95QPK0VZvHmrfTjtdkEiRsbdaXdATIBMtykxcU9UeKEdv0eBzKA2RrHMb9AGiCF2gqMEhWnnWxzdZb6mWQuwqeKyemY5xkwWSksB3BBtJiWuYuLDA2gwNGF1nqAQws7Y3VEDBHq6L+E3STZqR42jK+B0MoNveHzEoO4DRyKrnXuu41KEof8ZFzBMC7kLirZCApUe4yQS2KTK4rFBCKSZdFpu8bf4L1IT6b6k3IC2ISaxp+Tkqwtf\\\\\\\/lXRLs4EodBorj+n3vhSYxKUyI3EIYUguRlWNuYrwNNqpnDVB\\\\\\\/8L6Vgilb69Ii381PiWAhG2NV7I9Pvb1UZdriaZgpbh598hKwHAugBKpI0vJDuUygt00sUuDFlQLkdnDWwdAx8FIlctIqTj7SyzOHbBZHOULuX2iTha8PNMIcd6fTtJTwq7zE\\\\\\\/lYVqiSq8CmCS1\\\\\\\/wJDgdKO4auiCEUCgBeS7XXzp+nCxuA\\\\\\\/3eihMCI3sIQApMPQih9wLMeT\\\\\\\/pnoOylk0prCvJewCMNrYud\\\\\\\/J5ccj4NxDpQxUoSZHz0IE4eJAJAH6x4jXdDDvV6SVQ6gm2WAZEdI+vJv7I26SRYFKKMsQQOz2ClKKeI8wHJvuycvSknhwTkfVcwYpDgpmLFe4sGtO7oR8u1amg6mH9mWTHmESpmRVCXRs9j3sWzYQCyqmkc8ANRNByIzjOEDHw55FsF6vXYSiExN4ZuZZfve3+g2UxUYxHRY+SBeqcJ47F9FHhz9ybMy4\\\\\\\/2\\\\\\\/poPYLglOuZyDhK8atEtgW6iRgwOB5MYSKootG8Xw4LQ1QEv9QjNOwMm3wi9+uy1ieD5pyflLfqPMMTiPnRnTkTnt6fWb0\\\\\\\/11U1G47PkjJ5P8Y3XotXOrXt71AaQlOVS3snq99WugK25\\\\\\\/+0\\\\\\\/NWal2E0zmI21xRx+p0K8mFOsSfd9PxMObmomnYEJuMsHBL6jbaLF9VQU7Ls0lqNGhx+vG+C79bLAmuA0sguYKe6MXiR97ATxIw+AcDvCsHxkfvPbzjm\\\\\\\/FHZ7+cG4xHvKgiZsr3vawAaBkX0\\\\\\\/jmRtQ7i2F33e5A6m2kxQwV26yijlXBEHWIhpYbFNRSsbzOrUJdqiRmc26+CIlsfEnnrzOyVdwzX5jAa7nsGvIMHVBc\\\\\\\/1haex9kcG8JxQJk1IUTmjThcFaw+gfxjG8FdH9ucazlXR1LD0tlVJv5ImdKT9ZBK2d2FV+MasgwwSC+sRSPeDT8vwosKVYOZlIyjSYBcJ69YXumYBd2xrCLc68JTQKFb5uTDUbcApVt1Ayi8ziY\\\\\\\/jdwUWZ043nkkWwGezOdueHxRaU4Ij\\\\\\\/YPstuImZXIoZtgwT\\\\\\\/hFzkO86QxctQJC44SL8XH3\\\\\\\/JK+v+t5KlK4pSTnKJ13a0ibggXcaYEw5Rks54hbFZQ+BLnAl1BsGHhFJiXqfDp\\\\\\\/\\\\\\\/28XeHqBCRVRX\\\\\\\/CukAjROuyvIXHHSkUotcRAsJf9V3wO3pDrOoTUxhPXRG+Z5ERLOu4KRWEknxa6Xx2F0A7KOdBEAlIWvjyyppNkYTNO56TVFRxpRE7IqasM2TcTZ4L2IwGVM0bI6AkW6T5VIh5MMAMcpnE7R7wMoEywUvIsUPKsTIuus9JP61LpIoaQrmECZhv4sY6W6jjAEeNmBbpIry4Mxu1F\\\\\\\/sMuUO2wUvlWmZ9gY5nL4SLZK5ohReFhbeQ6zEF7DyYfninOCJ7oyKuBa4nniLSs0YRv1ygScm2kje4nsGMFEnV3KTrnJ6virYlNZaqULFIKrnEc2fWQvoDVno6FhH+rg50uZNZB\\\\\\\/YMiB6B9LD4fs9Hbv\\\\\\\/LaOqns3ytsPVhpuvGEZOxhJOPWbDj6r9crsUHmGqc9kmU586eq6EGhiUYwpBTpeamBcEG3qF3sdgNOjvl13EDXHB82slUhlhd3n4N99uofqc6ecaTTKIAViPINCXYZJtRGPCMTFTVvfcW1CwLx8DVxmuTfIxGq2LlRmgUM0YAZe8mH9A11IVOZ9zmfC4MVaTAIu4\\\\\\\/U2ONRIp4WfTd3xU0HMzNFCw6hRyVv4WnI\\\\\\\/ntl3H2yag9WBQvVjxE+iEVKZBNkO6Be4cOrH1qzvXHvHH4UmQ+iyb5qNhSdrSuZ3ubNAq8JQ2e5jT6+o41dSRLqoDW6PzeL2TdjfvDkJxWNu81qaSINldwgHQM2v4kGG0B5ph4nlQJZyK7a+nht63SQ681VQqJOpWIdn5oIDv3pHElsMxWipgva8nbDIHkFUS+vOWhHu7uTG30hN+1JqncnCFe3ZDjPxgBWUmK7Lp5FI+gxPxcQ4zeVfpJjGy47CDh8haLDME3IPAYJOMhofWW\\\\\\\/qJKkrHNn5300YqfaCMJsqUeNn2K4u3qzy9x6z3mt66x1s5kErRkwTO5F3N2KjGzQYWWyb3ApP90Xe3QAgMb6BvVC859wLFCuwEVxTLuv9nqswTv1D5fD44Z9hguSIZpZjLcLNsqK2tjhOY8FflL9QRAbAPH3TEF5PiS0vzdMPbkkHwl87G2\\\\\\\/r0EpN5cYiGdXCp6fu+W87kI7c9TPmyoVwwC5Oge6Y5l+RtS2QpROmzLVxnjEvsdikKYtRp8Q2dcO1kowMOmy9KEeAvzJWyvgnAsMgqiHFcNiAdXcqlCSUVEeecNWZehVCxNZgpIP1Z3pp12DDK0ZBY1cipJeeB8xyHLFZgj+GRrf0bZ6YayirlWFBzWh91K5RLBylu9ORRGrFBYukkzGp8GA374Qiw3cEGCnaxYZAdZdb+H3pathf3pSktFFhD1fPGu5jIEb4XdACaHLgSQfn0zo7nDpRdMjoJPGeImNyP\\\\\\\/SrrTdjXPR\\\\\\\/k3wR7sFr+qTMmZDZgZDBXfamgS9yb8dd7l14yGgJGYQCmUoUXESOsg2gZCjUbghnJdN\\\\\\\/wJ3p0GT05yESOs2AcNekjoaYC5fp041m1zDNPFsl7lbyvOmuUv46F+wbbElkOitEqgaJBkUV\\\\\\\/OQe5xMp0av08y\\\\\\\/KXqHs\\\\\\\/g2t8KBFyM39DUWvnmtzZcvFSOK8AW6HyWTdGpWWVBMLsR3eAPhg3NjOMuypsu+zpJKoshemkMlhS+5DnfFgMmSrbqRkg9pzt1+ZpxGLKFLLOoA4bTXZ8nSLOqtnKOLeaCfm1zNm7K+W6tKNkU19loiFwGOEpxMn7xMbRbErh5N3u3vSYO+6+vkY\\\\\\\/ODliRjYKxsVINcFtaNgTB+sp3rYBtOa6ebpahm0Y1AYl4ZKCe70BHekMGHRNDcyzJYaZnFVwFkHWJUeGOmC+MGdcaN2\\\\\\\/7Ztu7t10aa8d6HRQdmQE3+H7mtl0Dkz7cxm\\\\\\\/EtwxOUMVCDZnRfHEIOlx9w7JYqcI28USiC2d3YrE91vSAMCw4rH5AB+CW+qbjUczckDlruQDLUm4ScsaJAn9wJdiyki8mmfH45HgoxUolp6Ol3ZftZ+mmIfjZts3aDhOYZ4zvl1Uq47S9HaR63IYlkB00GjkYkmrZlkmTYdblO+Na2N72wyk9mYI5oj57cF1mqguUbqhtsPjFbVAfKPHplBsOxgLoNHE8Q\\\\\\\/jtdcWzm8iIzVuh2JaGlizYWgiVQWVt1vzJZL\\\\\\\/VWAO7nZX7aGVeC3V5K9bLeW\\\\\\\/7ocI4ch05MbaYL6xmEU+dNFpVCocHYQJrUvtXdO5M4G3xC1ZL\\\\\\\/xCgbfEqgjr45Bk6KKfzM5TsbqS3jvwI93F0SJ6yJ6WKLf7My5OjSnlSdVsSdM4Ls41hGqApj61HkDuxWjpJo7sxi9JxJr4bICG4\\\\\\\/DETpF0E7L9c7UByc6T7zKxjrvtdcN4yi6Np4WoxaouxemeHn9RKDi+dP3re2RfyKZXb4+Cl2g1ZeTwEAuyVDY+KNz2FkWomvl3j0i6hnBlPJI+3HyZAtlY1NgyZ1BMP2l\\\\\\\/N8LE+p1MAQbcWD\\\\\\\/s1DAGUoulv0VoKVYncUjJjWBP8VGg290YIkXuR5LpU6QSIdmCBZGCIKPZCEL+HEERVMY3afpK\\\\\\\/N+1kkylQP1McTETXgY7LqYuRJJj2wehTFJxahJ4jO96g82ATV6lEcVC86pWcbgT+YXAze7hnwcEYssCVdC2+7KXDYQtN\\\\\\\/a7P64b3Vu4JKSdLPJ0UgnWd4ydTF2bCJAKHcbRlALWQmFwye9872xfdSgeHdAQW1sWoT6IFRsTPyBnfbbR6LOQhYP1G53ZCztxlpLpiZ9WIZgBJc6GI08HE0Sph1cXFd6UKn0GfbGzpNRH\\\\\\\/LwMeI1Ag8AnuTddez5q0sX3ilmBPlVf2497hEDQTgu9GX2fWTd\\\\\\\/XW3TBNizCDxW19v2vkYd1ocXQKC0qM9hvyhUw\\\\\\\/mGp5aIyOFOkWcE6u0L08FwkXEeZMraGVy9M5lWKwtIREOiDiw+veMAed+adDqV7hEb7E4w\\\\\\\/hgfZX4PNsgfcm6HbFK24dSm7mCWd\\\\\\\/IK0XDrfi0liWab7JYqAEW0HPWIzE4+L01niDbdw1rQEZ7lt92lXTAPXxNk9jUJf3u2tv8sr8LG2nsMEez+4eYtlXLbfGeaJh4twbDNgGQlA\\\\\\\/j52VBNheZOvkGycCGoA7pcfdxfyu3vVnZ5LbkoCCK8ZptZCWJjZne4TCZtLHlegbRLJZ9s0rBstk4wyisq8YaM2wdIoAZM684udjcW23hR+p3WSOj4ajvxVdAnG2oS1\\\\\\\/Q9OpSqoZt3aXH1SRupQVPAL4JsG+xtHgnW4OZ6TO05I51Xs5nFzywuBMmEL842Si+Qi\\\\\\\/BrYAS4L8L4FCnn2Nh7yM5bqATUdx+1djkvzDMdGfH1q1YfHUOioTLzFPQcmIC8rPYX2LDjs59G72SBQifA5\\\\\\\/PUyPSzbMys+ZCUXKktYdOzJiynihPFbGJvYxP23qg7PowjFk4nznXj88iL9jq0c0kVlLHjZyPELRM3iJ1Gu6qlcjOFGmp9IlAMSlpkEPqrn4+8pW+goJWR\\\\\\\/ciqFf9TIv8wuBJhcr97e5C3uzFE\\\\\\\/Ct6RDDv5KDB0JnLmbmns+ysLuo2nzDqd6B5kTpYHpAZmu8bQF65sYyMWhhazXNtIgQWqGEVmzW3FC+COBf+pREZd09ALckgZfE3Z8Ibiuh\\\\\\\/rtKpwgGPQ9DQbKEacMXNd9Zn8tq3l9F7d3hZvZcnXzck11HsKEZQeovP0rKArzE+qfQnxmZ2hzUG\\\\\\\/FJr9qdlqqJ6yQ9Z3OpannqnvJNGej5kF\\\\\\\/jTXlOon43PtiEQPk+VMtNFCnUczqXwQtrwj7DCrNHqdab3jorOx2c6VpLHUAk7DuX7gIa6BvmzJ1JsNp\\\\\\\/zZ9cPWDT3yvqZf0vfpcqyFIToAQ64up+DnoHOjtiAwkpMQ1O7ZEn\\\\\\\/fHiwqNpnwx\\\\\\\/ZqDIfTeke0UiqpoRj+dHgtfsMM9nhuPRKgV3LcqhWSXrc9HMwUTY9o5d6Lbyc4sbjLM0TmM1SIq6umDxg2OxB0WHrd3eWtHCfltdbxrynBpl2a\\\\\\\/RiPJes9UXBT8SK8EVctmNkdTvbYKtTlCkA8JIK\\\\\\\/qGADCVnLT4mDNHHwgxGmuLlum6uuoRH\\\\\\\/kjyV\\\\\\\/q8Sm+jrwvuec0PNKdXGHsJTgKf0PwDhqENYdH40RERUjKABybgGjxfgYhaT3Pbk2YGrQcpO9tso3B6kE73dvpB5r1NDtCkq3L\\\\\\\/8LXZYqG+fwtA+ww20iqqaOynV87zLaW5Nef5KcJ8pfC\\\\\\\/IKKDGKmGKFB7OvHIoToi4yQAhkrRxuKWjYLVIIoSUkUffvRxSRlQGjZNs49Q+GPMEqtWlWOX\\\\\\\/3vUll39jEmJlTq2qwOei0Rl5SBh26VKhgNWOpZHZWC1w1r4EyYsxH5kFP+kkmWdAayAF9mouM554V8aP3jaeYjrHN5kJvlLgrVKymCI9F5\\\\\\\/5I58bempS07qCXKv8q\\\\\\\/OknX8Am\\\\\\\/uQeSBOLYyy+wiY0s3ZR8XkSRiEPbRMIJf7wJJ0YynZzA9f3Yfe5XstANbSv8wVLbn4MuPMTNxBYBWT7TgA\\\\\\\/x6i99siZUiR0kmGkHABdhDtYiyqOS7qszBo2F9zqu44oYgDDT3mYtLSq9iVEpMNA5pEbZkdD+ym7+0sz2xk5yAc+hVpAPTTSHQnoY1yMXcgzXNU81adYLYN6RqpoGuSbGLFiUrBoR6gSwdDEvfMpgD49CIXdNos4Ue9lSv+APYv\\\\\\\/7IAC7eoD8hxsilOPbp6oqwEQvGTyiBXH6\\\\\\\/O3zQIbC6xo+KuvnJUhyQ6OuNk6DNCV1QFxs8p0cKFAiu2TD27lYgTIeo5rmm\\\\\\\/u158CiOAfuVO1q2pgA5d9uGTX9tb6HCoFIaPJxzAYmPK1Amb9A5DL7FAxUI+HsVH+SNG4lU2+LDYsuSbV1TnDxyXilytOhW2mQTMwjgFnQs9xcy1oFu7rDzs6ELgrzFn6fgGmqZ02xLqrmHSbyogNMxnwLXUS1SV6OLsIEz607Zb0dJl4GcrI3czkAZlihj5FWAYY0StD9dXv86qrHW6DupDJHU1dzbZWAxfRp8YRGv9pk65bwEdG3uMAn8mvvrlqLZKqm1mEChzXS6wtJe2Iz39lygQBZMmefKJEUr8NuszgBw2iZvEVKkwrURdQadZSbZXxyxJ7tnR6u+TrKYYzpzkDObUA1AsSBtu7q9fp4MnelJ4lUgJVRAhwnCBQAtrv8EvJzxRBNwGMSarNm9wn1K4c\\\\\\\/rgP1WvtMZPeyQ6QiRTwKm5\\\\\\\/hSd8ZkiXztyoYAYdxkJMcg97aBuv+rfXagQHJVFxrJ11zC0T1AZrm7y1cwgnX4TdUCxyEAKJyMetZUJjN2LMR3Rr60bIYyAPVKXG1NgqHzcwU5\\\\\\\/tqngZN5vqJodjJf7FUeySgspUA6JF\\\\\\\/aO4F4SitHUUI2Dl2G92aAG0fQZJE173ZNLTADIQArv1wQLOeKdnLktlBTvaPhxRTUaMs3Pl5CZNZv0\\\\\\\/qjQ447QusePQ+wvUhHhaH7oFeAz3rJ9x4vZGUPA8Pv1TRU6WcssAhTI5N\\\\\\\/sRkFH8ETbULaqicp4PrMb+FgMRf9yHxsNIanHkqogw7iVWK4z3EHZxoJK+6Zsj2uPn+jDbacvS9PWqKLuQtL4yEDT0P\\\\\\\/suYivUNtvgAhhxqkZzrCTXztPg7uZp3133HvHMv5QgARfb2Ndoq7U+7KAJhApHBdmhYvfdWjyWFhZdTyVA8sZfVyWt7mCdJdBvZ0oVGjkFq\\\\\\\/LHhqKcsmT8L9siY7vUBr7T4lMgn1I2\\\\\\\/rV6TdXaLtenOhS4+Irye1LK6+E8d4eYpOVBoemr\\\\\\\/7G4EdJGhPTOeyEIVWuym0g7UT9yd2Qsc2bj9ZguZw+n0uR5UQbakomKn6+P0jKhmXia0tFuGURp+WrRC4rbHyHXiTVeaf7Dt20LPAtAVDuWYGS2fhS\\\\\\\/8UAI10N2V6BfZCVzmd9RAyZWMj3NB3K8AV9mqbUJOjZj1h8dd4InK5d5bEfAAAB6bF5iRh1rit9ukCSwbFstCtBRNNTSM\\\\\\\/B1RnWE6eyyu2EaD6P0XZ2wcy3VvYnWr9TvW72+E6mzc1Rv7UY3MGc1p994VJCxjIIuc7fg9U1f9SkQZW1fzcPxzas9L7m88w07bUbOdEl5UFfyHLnh28ZZ69aRr26U16Bfb+1nlpemnZHEuif3zQEIw7EGhSLb4rLjIf1EGbmUTq28zBuzx5zwhrm0F3YSLB3rc1W+ssSheuYGuEja5uF9XVYqDdX25ZJC0v3eb1TNTukiBLg7sjDUc2UK4vtsstud8b2oOMNJ8OeY+bU9p6LBmXN\\\\\\\/Q6dMalo+dLPTQg7IjbEZPx7RXnTRBs761NU5RCWsXjJG+jPDvGgJjNSEFeMo8ZDNnWzDvBxJGskeSBMcknpPXToiLOFct4F5gxRJ72K\\\\\\\/PNKfWhBWQz6SB8Mgi6pUWPEnxbjLvON3BTRLv3b6fFGfMLERuIJ5xFuWLg6Dax3T72WeQ6r4svL0fmElbkmuT28BmPWrIFXeeDP+p72DFEsoVuyVxaaMo+I0Fgw9vrlBBtxGm6HDdM9XrV04gS+VE55e3QmnmZU2kcJe3uEbvbQYWzpjGxXXKoJqdo4ylDQQfTXWKrJ+GEejKWbYofgQKFYJYwNUhNkl60oRnDYF6LWgc+FB9fQG7dlrd9Reo9CV9oI3RRCYFonV6Au+dGASL8Wydp9Is0bvqQTRY+zSEW+oH9zhhDUvHK5ktGSBZj7uhkPeaZTT0sntcBu7kXe1i9jkryJR9ylnfglD\\\\\\\/cPCamI4cjcfqd8hGI8fPwBlZ+sN9bgzmHl4jUjKg7PwV9sHN3vFXikodANpIsdAb16A2pWV20WnsVD7+3x4A0k850vVw91stKaMLTwB59sZk5zMzNLaDH\\\\\\\/LaJXY6YcoEhUTB6y\\\\\\\/DF0GHuw6djvGyZexDSoYfBZsD1eEg1KFLzlOyxfTwr56+c\\\\\\\/38RTezitAqcaw65tVvvNkqxyuGhfheQoZck2oakrA8upjFn\\\\\\\/hzn8AUtFR+iJ5XQa4EFn4KxkLznpG67LU8zWiyxdaZSRIhNStyrEwBs\\\\\\\/foYh95dRIpd\\\\\\\/SkLc55PqoZgrApfbiJEOHO9L5VEKbyNZjwvBdfG9\\\\\\\/9Wa8a6p\\\\\\\/AkDQxQYZYrA3GKQ5q\\\\\\\/M49l1BTyI4YxVUry+hK17Esp+xnL+KFh625kVK4lGQ6ZMImPmu5sOz556WYdIHxgg0jhW12I7QUxhLjs\\\\\\\/d58kW8e9TxQwDdkRlnkFVdVo2Irz9ZfEGiVDpCMTREkH4ZnE14UR0oqxUNRC76tG62l1ysijRglmho2Go3hNcg3hWk5DmSDsea3qFK7zHxtiRlr7JUngDKrVnzvWj\\\\\\\/nuVzZFz3ufBSYdpvDUAQgbClE\\\\\\\/mjTH3g2nbv5D\\\\\\\/IANhGIhaWPXG6\\\\\\\/9FMJ5ZCsUsoDLlSBjbHsa8Xu+VhmBC3TnTxutCT6apdn1DQ2+A4sbmqaJrRVyngFmuQSUIBKmhXR5dJGKe\\\\\\\/h2O9VQqEWYQBAqurlPblb9lSfcOE0vbLRxklH2+\\\\\\\/HiHllE6uG9e8KcsuL\\\\\\\/9kxDQBkyxsIjU9CbRYp34ocBBrDP9CEGS8Se4162ohYCeeYw9d6D0gE7fgdYB\\\\\\\/IBbsNjtyvg8bBiJie+QLDNPVmv6YQXAyaaFLJuAVwhJ7xwwvxybMJAMUliCZ\\\\\\\/A\\\\\\\/KKOsy3QIH1OhMeDZkw4pYSAzf2NuCWUuGavu62x7WuS+y705DXNR9InI4cJIu49y2vM8F54VxRXqXZjC457gLvueUBsehWTBWAmXY8ZrOjGL7AHGps5Zwh+J+Z8kJnWhxjLM4r5ZVK5pPiZLATsHInufM8zwMC3Ky6LgiIzc+RqwenUYMF3khaxUaYIs23h6dUZCcN\\\\\\\/BVEoW7DNfHrluCzSHnVMKTHER5eTWP3\\\\\\\/xiGMkrzVw719AFlUgLmqsSen2K0YQLj0RpQ3BkK63CL\\\\\\\/oIkeiLrMT3mvOVtrmWX8fCCND6pe9xUX55uowCvduDYm89kImB3Ku9G34gWcQg7P+iWhiSUczKLtmW1rSpLDN5ppGLu1bc3E5He2QKYBxW782ZeLvJRNRjekasTF9OHEi0IO4l9XWMJeHlk8GUFwvhY3ylp\\\\\\\/rgxDhcnlCmxn28U3W1lfAUl+O8Na\\\\\\\/18\\\\\\\/Z8okhGWwb9HIup1VOvOOF3UqPhgU3QMvMrhIqR+Ff\\\\\\\/PgZ2wexUXAGpI4akrWu2\\\\\\\/k+1nkywrv8XzJf6\\\\\\\/\\\\\\\/\\\\\\\/MmCe8Sodsxzir8SPOvrXpgWojQuyFUB\\\\\\\/jVmySjATv7+LVdljHIOD1XEdhgIaSThma2ye0\\\\\\\/sfD9M3keE09yUM0cCcGMOs9Cfe2nMAWj6JsjRB7TqC4OFLoeyr\\\\\\\/MkdFqmx0xrEdI4SgzJ79jBeZlxvmx0WW2L2wTIo\\\\\\\/2EbCzIKJqL4gmX9V4wZcSdm\\\\\\\/8lo4cOUVVIb4xbSApDJhnxiqwiBVA+rPwal1RWWUCLnlv6eHhVM3RdnUGv7XVNgp\\\\\\\/LfkeB45+vjJfhZWY6I0Wr38ZIbr9zkyxS+\\\\\\\/iXuurs\\\\\\\/NJgT5He1P9dg2J3FJZwCinXpqfYFGmaJSrWcWd\\\\\\\/ls8TPSFPz6xXmg7FL8QuTtnFSiY2lrSiqR5IlcV3ofYYx2iTyyyPfdMVl8oQoDX4b19cRBFA8ZwVfvhGQ6t6oXszOk6VDXZlgk5\\\\\\\/jZGMhL4pIjgxw89a\\\\\\\/954OPWSs5CKX5OqEvXuqLO\\\\\\\/NlPki\\\\\\\/S\\\\\\\/YOwEwqmMOn5OhcCIU0DBzCDWhXEZs8yGcnaE+Hun3dTf8kKgb52+WJN8\\\\\\\/R+cFIdHCEqesTC9moqFORoFSJfzgIirFXb7yhEAIgWIYLrWtZaRxAhFw4VH3LV8A\\\\\\\/VrRwhKCSUNFimemTeqUgzA3IoeT\\\\\\\/uBL0WMMymAUHX1riBsf5x0wHt8q70HFDrlKcnYenGru9YBu47V2ofLaKYB7Xm8Pbx7jF9m7qs\\\\\\\/9AQzqvsb84rfOIH2OGLgKIX0NjfZ7j78BxaCTX67zFJhfq+4i5O5HdT7hLLyA2Zszy6KFf9U8nCDC0la03jQJmn6o\\\\\\\/QCNk2ZkyxPHeWsAIbb31wXHXJIM5HmnOXsCKS9GUwNJTlES8quwFJLjmzdBxXkoNtb+ulvCjQN9\\\\\\\/LSnTsedUb0sVWE6z8o+BMChuXhhD6jQ9lfWOwHKamcrLSVNUXS2v\\\\\\\/DYZTC+Eb\\\\\\\/G+EqlwIXBdbMWCIXkqEXE6pXhiIcuwkPxWZECfPvyuHFR0rrKDkvUEO9dURFmKlAYPqFNF0VJOlP6LqYauatQH5QFuh\\\\\\\/4TPdLfTX96gtK2dLD5ln3Cm9xQVCbfdIxs9U1SmTUBquSWptNb\\\\\\\/iF27YbT+zsTG5k40cDT3ZboP+p1NJhDiR0Yk74pawSgkonGzokekkGpTfv63wuxgNUH39zfqCQN906x\\\\\\\/RAxUIjHa69w6zeomq86tGVdeNjM8Sw92IflHeNreVUKz6QbAMDbQpXB4P52dXD2Ap8GY2yYCgTwqfjwmvdq2lXUaJpDGogYqYqT72I0YRyedQIXq+otFb3JCREhCurvKUlJ1W\\\\\\\/wzUjjwVrK\\\\\\\/gD\\\\\\\/jIDrUIz0in7bLVcvDkH2VsPPv\\\\\\\/08snW9uBdFgQERaWHsFMMI9BLhEre++E9MJkCy2ZAH8F0IA2\\\\\\\/TFgkP6xEudGA3fPr0K5Otn\\\\\\\/tn5u6hoXRshMLlw8X8Ktz8\\\\\\\/5+6cqD57u6FZG+vDfc+gKyJzL7FrE47AG59Ni7LEcHvaO45WuCLEZg+Da5aI5aWRfIjIqfUDqvdc+P4WGRT65w5u9uiPCu4kvWh+BkbcNXkMsqRDyMi21RpggaVBF9PXY8eO\\\\\\\/TxqebYwP7iL3hxT1jFnVIXWGTohTqlZfp\\\\\\\/NEi5sftuy9gjsTp4TmOgmUFBdDnO14\\\\\\\/Do3Gyt82uS3bzrul1U3rDjDOfx+3duvJyEk9UcrARIKjLAZwofyT4Bwqq+1Gc5vhg9hPU\\\\\\\/SkJonTZNXlZj5ypoa0CeKuZJgAgFcURWEy6YWjSGkUi4NgK\\\\\\\/KfXQGUvXqKdaZa2u9Qwh91LRBnqGZpIMKzBMNbk2KOETcUoE8QPXKtlkWopSCIkuiZRVilMoVVFb9psEozF0YYSZy+Rqd7qJ\\\\\\\/VPRx6vqqJMx\\\\\\\/UwY89Th7AqD6lW639M7gZ6ySFT+EB4YH+LaOA5djTh2\\\\\\\/LCnUfRS4BPaZq5jaBvXBkvDQ7sn9HKEpSqeijWBuGL2RFf3JzpM8XtA9W9hRibqBdAwq\\\\\\\/ojH\\\\\\\/+YCZMQnHJLuuR5KIpp1mCCSIMTlDcNB8qgg52ij1uxzB9nMnx3YJaOKdznDFFt+yLT1C+oNmRBtUC6qHoMyRr85owR51axOoL6MoLdGx7sK6mDGtoNZ5LmxqgYmRUSVH8cpDel2q3Ml2W1mzZLwZNwR8NTXp5DqtnzR0WUnvZCOmEi9fgIaqDEO7+t1Rmi5w\\\\\\\/Pc5nkzVaCES1TyVxeGJKxYpdza34AkCP2vrMcATV4qcJlZaIUBQy1EeVgTKriX8L0iIJaircmA5bQm846JcwxvWDhEDdaT98\\\\\\\/4k3rthjb0ou\\\\\\\/axaNoiU2L8oIXVfCOX0+6ADJuJjqcemQIjPafAWhqbX31uCY7l\\\\\\\/kcttsoOkfBRQ8LDn8vhuFyMxub5vBPypnjz8xI7vzUZQqXcWBcd9ij2afem\\\\\\\/Vl9w1Yg6wWhPa3sQ80Ba5iv99sO\\\\\\\/e+gJPLQtJgEIYOrdR2RQfIagf107lg+20J1Hpn9RcZobA1Tg8NZZ4oxhyGjexT9Z6rBdIs8GCM8hXD7YhsK+Q7\\\\\\\/R6s4SZjUTVUIfCag0yXK\\\\\\\/AkmlELwHmvapENJZASC7PBgyulvdCkC\\\\\\\/Zee6n35VkYdTNv2Zte\\\\\\\/SMkeZjwM44Uob7bN28yh9RnHfvDkWHBeq2LGuytsQxPVMTBIifh6PgQmn4nIOb8sHjN\\\\\\\/KdpYCO5Zu7PrsdNGhuajDgUs2XeUN8Z0hFnmI6EN\\\\\\\/1nS\\\\\\\/i0yLRIRvx1oxZD+prMgfKhk9bzWgMMjJeAYcBF7ucNwFaaWigdzNbDGw9L\\\\\\\/m7mwPXqv\\\\\\\/X8jBdtV2mnLiMKSJu3COw97O1BpVlbevh98NGG9tN4M+M\\\\\\\/XECccnGq+nh6iDw0n3JuKurYJe4bot6OF1kuf+6SZ9PUJlxTycwrMc8NbZvyt0unDke8\\\\\\\/03bfjs4MN+6Qc0b90M4Y6DBoNIW9JsjNmmZBBRoJ\\\\\\\/44eCjKjN5QYVahodrAPI43BzaeBa9gSJvXqeYaC5ile8Fm38dnyBntpllKQcqtzHaW55SGrm0HujPvpskrwDgJA9R\\\\\\\/R2FjfPt5v\\\\\\\/US9GKmcQiAlwjmj+CtNamQZO7cOAVqz5s92VNBQFCLqWnMZNLYnI1G0E8U++D\\\\\\\/wCrWzBA\\\\\\\/i\\\\\\\/CwpTOysIsN\\\\\\\/dH7tubMi0Wc5bx8\\\\\\\/dFtW4O9fuOeRA3Q6qx06WeF6zX\\\\\\\/AGSrKiHA2VRXql4ySnmoCfTOeTEgzEJ7e1ftYnsSca84i+\\\\\\\/bqWHSKEI2XVKR1+oix56WL7dyHHgLOG6rR6sbrnrdndcvHFnOSp9m2mU7g17rTOW5+OVeRvb8gnTnEPnU74z3Gzz0ItDlN12b+5aI3oKI2NvGwFfFf+YIpxG2styifkLlR6wk\\\\\\\/PYOz+ZE1i\\\\\\\/R8cOXaFIlagtiobFOKi97s\\\\\\\/VkVy+g5+r3yrYNn+RubTPdYtzh5wZO9fJ8IMmuoaA0kOp5Mv1mxrIJlK9skerAHqL2VuE45Jm9jGQG+k4telhTm3cG2W4r7g5OsrKD5jTIwCV584d1cd2X+yB2yCsgYptTeTORjxFIl1aNCHpkG8xNo5DWIa9e6y3XFtaPbhrxZ13ybYJWhm9CJNsGr9cEbZJKGnx5iytdDqYyoGLUpMXmnY0EP2MM00U\\\\\\\/v0DnmyEdkggaZXso4w00yknq3WrFL4bHJ3yK049FLXROfhtuvYAokHwMAdx1YJiZITDVwrV2PTK\\\\\\\/X1F93jdT\\\\\\\/zf4sjvtiwgc81v0QBy5VWfnAz3FZO7DrsDEJBVqb1SVYERPDxCxUSGXD19hjw8BoDKCWBOZxtBGOmfFxi49QcWzxFBOAEPE+emVt9LHzkS2tmTbV2BrZ26MTmIP16rJEzckYILHgCBjNkv3f\\\\\\\/rQ+WB0hdd7bPI5h3Mfs8rfreYUTCxGkZSLSCnjmj3TzlTUotXKUBLuvDMNiU1oywnINxJE4kxFLxH4BKzKH57Ai7eICd5QfOZ7wDJRIJPshb\\\\\\\/mRCEQIafAEIWsymxTuBkzKxwo1fU43UNTDmOvF\\\\\\\/F+fUhZOk4rUF3ee+yYtVT9Fx1KFoHdiywrEXazNV3+kNAcVWmU\\\\\\\/V97i1QWSpS+2VrRfqD6uTPMQLwRQmpLLY+\\\\\\\/hLblr1s2JYftmzRlZ\\\\\\\/4XEWsQc\\\\\\\/lL4AE+W0AdP5WdPwks0BtdpOigkcJNwhLFJ6gXkGM9Z1UTY5RSNB+wehjK7yDVTnHYq9AYTbWd3kHr9qMcrbC+ymn5ul\\\\\\\/vhNF6kdmW8TMNkujCaOj4WgkQ5fQ\\\\\\\/\\\\\\\/X\\\\\\\/1\\\\\\\/e3IjREP7jKohi5ng+fptlWoTPtYsDmFv+TaXDeUXGnnhr\\\\\\\/aE7ny3BL7rdZsZXB\\\\\\\/jq5vPC3ZlTgKxiZtDMFZOYtIY0HqYIYsUKckeNCFMk+IXBKOyi9AG8lYX2G+b4hv1JLaj7pnv6Lyiwp1mPMcZxyAzwgec+U9rYbSTBZLRCleES0bYXzoK\\\\\\\/YS\\\\\\\/FE9cYBBA7NVnRAPuBqzrZ0smWP9QHALcCSaJlHysKCeXMu9MFP\\\\\\\/Ng\\\\\\\/XoKWosSKduQCBvCNsbnJMfPTKEKl6Ow6puRXuzRK+vGUwl\\\\\\\/R7036ilC6wmuKqtSlx8ORYTKP9g\\\\\\\/QX2aGV0odadbAJsQrWkw137yAHRubUIAq60XA1wgNTg3\\\\\\\/ELNyJVSQUUmRDBK80Rq1PCFynqXYkCqNH8pINuHkR5m707cs6tLWcm+QnwrbYElLBCSHSWa1VvdaefgJUQbaFaOqjHRlQMzqMjvMr97lBg\\\\\\\/lFdM\\\\\\\/3ZFM\\\\\\\/0VEoxiFY7G7cAFtSFfQeawe674epykr26eUgAXGatSXkup+oQalasf1GhUZ+kSY9czCp04njgdmvEXmOhBlPRS\\\\\\\/GcF1wYqN5wSVyep21k5fhdX3+bMuvjzQE4lwJWZq9Pn1znRL+SP8s8g7PPLgo4pII9ho+N3DwCPgZ1CNSwV7OFjhOESNxzOMmYbuxPFUIK28VJ1ypDbe5J2DWMOXO4bFWiL2O2n8+H9qeOC1t6qrDDOBe4aBK39p4q1nn0IrX+KfkcKtuVSa13Tvi\\\\\\\/HdNPxsXU6L5BB8amQXDKrS3FOoOk+\\\\\\\/qy9vuBWqf5kk8Q8wEAtgg0NsRplaZicVwTD97dWIfSE9vhYuSCh63Hv+FhnoKQvakV3wYbeGerj39I+26P9wuLYu0qzy1zw8XlFfUHnQ0baaz9OKN27LJ7fwS7aJHVFkeW+tzYHEu\\\\\\\/jyTqt4FvtFZ5hS+kY5yAyHD0mTGbBJS9V8c3O48r9B\\\\\\\/C+JGx6pp9Xq6GwGtci7ZI700ZdmPodAmG+y9OuxyZ43pJhY+NZmlNeosf2NSJThJoVLqstGANjTSd1f4B5zBpTN8UttumBtvuAke\\\\\\\/WcB30d0uCBhCLrVTxmkJQ5nEGJb11S6t45Gc9TfOHpe6rCecv0oL9bFDRATSb0cLp2DLh9q\\\\\\\/z7Rj\\\\\\\/f4nCS1LQaX\\\\\\\/lzJawW3IoyWrjwyyMpYYeWCjYsiuzP8oEvGJiRzb5gGA6nimRpM1lS6zrR4KsHSPDC4INZ22UWKJ7Sm4iNFTXeAWs1Pg3WeLR2djRjrTJjpxUbRyO7IWr+1SiXReqDMxtZ5pXfbgWFn0kM0SvrBHzSNGsO0VdIVC\\\\\\\/9lVys4kCD2iVo4KoJkTahoUYUQpHwH8zOm4wx20WE+wtyb2uPeE1u\\\\\\\/bDNhV1IVFZX3cKV2DUOFGckjY92dwq\\\\\\\/3xHNZWm4+6EQZdiSshNIs39z8LSS2F1X+GU6Mb0EoVoVyxwboKEWNZ5MbUTrCuq6Wydcfs\\\\\\\/XOG2s2omoQ8PB70zrMZXxbtvSDthNoYXz7XFN+hXwDXcfTAbShYjjHGbe+WOAL3born0re9x+gKbYQGIJk4sqyatzAXciYG0G1kctEgxOxruVzV+E6Y1VhGa4eCITG93fpw5\\\\\\\/k3eZmiQNMytqPsCi9xYhXb9ZR+uGPSlGeXOlPrrmaUUiM221H8iLAz2b5tghCS7KnQ+1WUsjBmNFbS1SwSeoZuTD9ReBb1eIUBIBvMqXXy0VyHpLC+P+c01ODfsTzhIU\\\\\\\/h4KGyTeEe\\\\\\\/eiHrbEpUZpI0pDQZN+tTLOz36qGecL4rh3cRBWGI6TwcpD1eS9knbJa4eXj8pFVAX3aOR5gUdUz60MGaoywlQfhPKCGW21IPJFd+SKuozN6Hv4pZZ9spVS639PK48erHGaqYT+wFUBE2WEruaAiMRxhWiIDitTQOH0u2pQH3i1Y+6mXFCQUD6Kq+5nsvLf3UIMR2kleFgMjCBanbtsBsBuNNE1pVmMhDgcxlrbQOKLa7MxFA7GX6+rueIEhqQxzC4qwiLBb9F3YYdC\\\\\\\/OI5Hu6bj2iR+gciKRgvQFNS\\\\\\\/ceSbmcNMcDXF\\\\\\\/mfXraoHqtFpn6Y907HsWv0eFckOAWarZdwXY4cGTjeHdBKL45zTQIoN1TPNYpsnpiRer3qnA86tQNFlgHZLGNcML5gtqx1fmU2Fi1zXQWcT5X2SrwZ2RnQxScawQQuxYeKpHpabXB06jrXc1wlvjDnkjre++cD1M9e+5sakutUrOdreHC2qN5q++44w1AGpSzyANh8HcwjBtUrktaCLZW7ZVTotWAqz5FYivW112vwEDTQcnwqCyzKTk509NrwDaP2bLlQHOvHVGM6JdcNDQqGClgF43FxbNlNg45QNyMfiPWTw1Nh4XmqQYwchic5ugwBoA3hX+jAijuj9Wx\\\\\\\/ldZNXSh6loezAwTqhiW\\\\\\\/NSqm81\\\\\\\/q4O59ctWwwVB8tw6bsTN1yNaQfSWr436ao6z4XkEEFph71D3RDXCyVArByiotJsZU4qU+d4pTH\\\\\\\/sHCK99YPKgjNTlqub\\\\\\\/dH3wdXhwG6BsyHA\\\\\\\/t5YUB4nwJd5UrD4GWz+tZoGxMaFt0EW+cOuL8pjcvZm+R82Wizy1ECZ6jl9qKSkw43kUfPs5chEBoaZ6HsL5kcTNcRhRMPoZdC10pQuPSIuyxGv6ns23iQDlgmUT2C\\\\\\\/u0F10iahsocfjV5Timso04nVnFA1NNXhslZ4zyJv\\\\\\\/KqH6YOp6reHZPDj5ZLwtM3hrBTMY6XU0xpiAiAXjcGBqosNWKKvSx1bRhJrYFUjO72KtmDpWbP1w8JwmU\\\\\\\/PF7mleeZxx+wnRh3FWNwA\\\\\\\/qg6HaDd51ThW2PswbTWf6E8Otcj2PgCeT8iabwCkOcqciP2NYNhKK4jUUUcHul4ZHwgnU4sFIVK2idSM1amysqqFaPLWr+RfyOlxCout2v\\\\\\\/LSDJACobFhvWpEPaRI\\\\\\\/+phPAl7Ru0UGJex4HOKIiwkr85RpS53OIpDXk1Nn9tuk6KS2LJJyhTu3gAbg\\\\\\\/BiNI8QGRgtk9YqXz8dm8adx+Nu2w78AKLJqCsGDP2x7s6O1k79X9znRTBdAk2PTUSP5iMvOdyT16cX8Vk3QgCVFHgMvX0WVaDJG5v8p2eNV3CEnFNjj7GrcrtExn\\\\\\\/2RoRHL8hP9mLUQ7YOFx0COsvYkM1+idUi60x60Yr4G9fTR7RneK4i3IQO3d5zytWc5a1ZoHcjC3GUufTlGbfLbn+qeMVKLwXvVuXEQF\\\\\\\/iOQmxrvI4h\\\\\\\/jWoBk0aWEDZlwlh6iQlRdUcltKciLHlKxKw1nsSedviPFQK5UcqDMA3l6x\\\\\\\/3bNavRIARAuxAunc5svGT\\\\\\\/z4c61DQ3r4hFFby\\\\\\\/kOcKu6d+JuCv2JdTgFZ+kdJ\\\\\\\/1Hp1Er2FnAgYWFITbWC61g\\\\\\\/+2vBdQyDpT5XiBhJEOcXHi4CiSTMufe9KKZJEsgtyN4diPyCxZei9fv1d9Q5kL10Ii5dlyzWVshz+fviIfSITRZs5UgCjUV0M4inha\\\\\\\/IPvUtPbh6VcYDxEpiNT9e0Ovp5qjKysYXQarchGDY0EHqBmJtDSV2axUbJAiqRTRj+xDXW91tRarFBAx0aBiPyfHHI91u\\\\\\\/P9XPtGLdCHW++0MSb+ywokJBZ4VMSl1uVjQHeJGb4L39debD5OCCMFMB3eTqqYPPVjN3p8gDD2jFVDnbj3rPrf+s0N4QlIJyXCRsVi8zHWrVEMoA6LLg+VeXa0y6IjKba\\\\\\\/xTrcoRHaViXGuFMpLAGAoW3V06XJa1zorEyfjKiGnxbL7zZH+sj7BZA3iX6HlppYCHPpi10mkmpt2Jfwoc1XdOSt6XC0xGAjqOeo1XDDxkyJNJC+BJw0nIIaj+JRBdprgxbJwwgJw0AZTNpfDRM9tiCi52v4Rbk1ikBmtxAREvYAN6AIQ9MxZ2UMR8qRQXJ0lmINs380CnIpL57\\\\\\\/1JoHKawrD92GTz3l9r+t5kIaxO0rC1O7XGmZjXoNJ4Ma5BENszsX17oynPc6cKECystGgAXhtnMrcfRDgzLZcA8Ezv\\\\\\\/wb4Ye7tON2UABzUiq2rDzTHFk2TpEJWQZqKkxYQOIw8MmYvuBwmeA37amU0oZJKTd5RcNUXyrOODIj8gScfTEuMN9G8fU2M4z1SLF1wNCmTYLeaKyCt9Q95oZE\\\\\\\/CeZDNs4KEHIy0ynpeTn09TkCuSYjw9zcDhpIFvJLhVrTZpasKX9\\\\\\\/MDC2\\\\\\\/WW8WzwCTTWAvc4WAox6Hzct\\\\\\\/ep+eK58gC8Ec+BgMGS5ENoxEq1wcPKYELEdTHBVh3au5tqkG\\\\\\\/kMgVx3rUL7+dQ8N9OBADyizEAoAt2A7O6zirc+dawiDr3IiLxQnP2WfX7iDFoc64lSYh1145QH1s4eTnDu7FoamowVzzea6qQc3Ge43Ry3qBMlZDchi+Fm5YruY7VxedtB5rtB1eu6xguw3VrTZx\\\\\\\/DuY16uILobWHTEoMFUQZHKmhHabMZJwHLRJp8UQC\\\\\\\/+l1BfXox6DkF8br37eNioDfnynyWhWezEWopG8TBVie7wrpQgfxOs82J6KOT3a9PUkQV7suygsnb9eOn1Mjm\\\\\\\/okw8QaclG3I6Bu44xGuBauKE4atJ+Aib7cyuH6qqYq4CC1VhUkbeDRytIQftBxKkdSBjIR1jaHy\\\\\\\/aOpMm6F460qSwhlhpVrrr+WPEavyB+adWewDHt5dUW5JupecJBPKeusVK38k8BMtDES9W5WEo5LywKBD1aL4twVze4D5UoeQ5liiagpcQErVR1n61wcrvI\\\\\\\/+mFb8BSRbityzBnnTxV7r7rnPjdeb7wZn4RGT\\\\\\\/Be2l4I8kP9aPyN9EW5a+AXp4CrR8JxqAOLGANBDkaTkJwUpUNcRHaOJhiA85Vc1wyl7HF6TOlPPKXrmQI8DDrJJv+\\\\\\\/TqthjHDjqjEjVKhIsnZmxqn14mduNpm7UJT4oWzy3VafPnG6D8sx8uh2wsMd+8z8Qg8+12dfZX44DSCMbr4NBQk5HGU91NGHzawW1qcGrn91FNFsJXsoeo0QMF9BbLSm\\\\\\\/Terex\\\\\\\/R31cZBWGnFCqDMJl2T4BT4csSiPSjzVIYL3wKhrCNzz6HVB\\\\\\\/Ie9KA7m02EGtP78sOTlHxe3gyQ13MJ3wHAjHTNm+1tcq4kCruDdRzGwA8iwce+H7QgsGcSX6asV\\\\\\\/59kPkaBSEM\\\\\\\/3StcrsrG2VODYiS\\\\\\\/WJvvZUeyGSxcxj5Tl0xUvmhVnDYhec54stjQVGewR0C72cE7UNq7KIWSHbrmBBF1B+2hVxLNvS0NC1xz7KA+1p0IeZYYA49uZkK32fGqia6LJEP\\\\\\\/h81K7gLx2MCttixFgVDh0gFXsJkHYMjZXMaeclSpSDFqSFlY4SEkcoqmm2QQOMIfb6YLVuHFlYvooXqWtZ8p4rLxt9TspovJAygvLrkm8ZndJQ+dCEf8vIZyA8rQ2DzFKSLm1ea4MlygMDLohmIpbKtFnTR667k1ibxtg8olBJEC3faqCyODwzcqZizBalyn476UlkVIly\\\\\\\/nXerjC963TWsTrYhk0elV+dAClsCQxK3CGK1uV9E84EG94fy5oe5hyW898IsEi+aTmyOepGDm653Lq5IyqVe\\\\\\\/euegwlNMJNPmGp\\\\\\\/hN+bPOiioYaQaNKBoOPWA9idjcq21dOIvnFROL77jTzb5TMdUaC9WeV0NXeLnonipHf8vYBJsvKPAG3ACoBkHjCYRhQv3Zdzmto3GfVWMJ5SVMhzgGh2TBQYzseIpZPFrMz9Wf6b64Qa\\\\\\\/XsdL30yiYz3AaPMvuQGqi6E+Aixxi1Ss0WeXxNG+QskeIQ+Wqpa0sLU9p+fwynzv8WjYDPPUpG++l3mA4QBSrK31iNleJ9s18RFuD7jxNAXTz0NavuqXkCQkjf7rldDRnipKlDQ1OFKLp\\\\\\\/giT7NMvcdpk1ufKXEBmmcML\\\\\\\/s\\\\\\\/0x7UfTaIDfauJUiDqVebjGqARESndrDDgrRmU9DMtb3cLY3V7YLGPxlBpDbRsqjXj3MhY3MNxFReF3iRox1SIeb+U4jMBZsziPoY7QqOHzTDy0OJ5qT3L4nxgPUxFDNP0aVSefGfliAzQkmFK+\\\\\\\/l7+b9sRjxEGT7txRFXnfQr7mW5bw\\\\\\\/516oT0DsX23Pi5j+HrpFaCJRG1UsUhGzhwBNX2TTtSKyPEQlUIwCR1sn1Pm0fdpEk7Us1wZDD8jndMW5kB6n0MiaBIZlPwMFzutpx45vfC7J4f\\\\\\\/oG8ju+RVCOVkJy7JYrkeAbk0+oY9CzYMW4VOpQdbZNWFeZz4VATVAOSDQCvTXU5YyAokIiz4Kd7+vJJE5ngqRMip\\\\\\\/3qsq8OJz2DaYXB+AuRYpEdML2heZS\\\\\\\/WmLFgkO+iSARFihCm4kkDxmJQMHht8uEIwY5hTCZ6nTwt4vraLiB2wITupRKxQhJr2j7SxNEqhfMBaJh+Zxg4q6gY2FRKaRjWrJYXqRR14GsR3VkQldyoi6jR9S6xHSu7EoMjg3lgvCLghvhzXZmVbEGPRZ24Yclxd42rM7bjx3KllRwAclVHqCsE3PRqV0pBEeS8T3SXU\\\\\\\/\\\\\\\/bR3mOorJu36LLIN92VDEiIUdCrSKEpKDCdHmjuNNuxaE71P1fEeIrWgX57olAtpnt+F3FmtO8189mGgwQ0dJ0TJgGSIlidapNMQMsZO22v+K0W0hyR3OaRedp\\\\\\\/+pELwjbj9TNvR8QO+e7zu5CZzHWQ+\\\\\\\/Y\\\\\\\/tMKuwEq9FinHpjihSqvROFJCVRgduKRHg9CISe8\\\\\\\/kkdW6eaXpcUiZPYqgx+ht9dv33CQyjc\\\\\\\/474h5hNO0j6gH3flxTrMFC\\\\\\\/k6FcL4xU4iZsI1dHjri9k0Ab6MlJEeKHk0pa+lHECJWDKvDUTIRM9BHjyguYl8nQDi2eBdhDqqBojQbx1X1IeVga\\\\\\\/7bFiEhoRC+l9MOf8L5cZPVeG4o76xF4ekx37Dap0GYJXjAcMfEIX0nzzDX04OJrzcbSQF4nSZN2IksoM5toW8JIhISZurEvKXTQWEHC8qp60GnQUnzh7BF2cKUNaZe\\\\\\\/wiaWeyD53Iuy9gwBSmz41bCZR6PTPD9\\\\\\\/diL7AdXzYt3iBLE0znWjtVlop+YhD2HjCvE8Nn9CAANp8UqcefdAQN4t2DTj3ernHzHWgASh2FU+4h1d4z4M\\\\\\\/2da\\\\\\\/Rf4\\\\\\\/esPNf19OzbaAO2ILm\\\\\\\/XqC9\\\\\\\/8BsXWhYOUZ17B9hQfAmIWuroRzjX6lerqzZnEY02INvV9cfEfQY2Ed9+\\\\\\\/kb94dNsmNB2p1slgzgBY2G7iKrL5E74gS1Ixuyy1xfrI8mcaFVavgsGrylLa9X1C\\\\\\\/4Q6Imgb\\\\\\\/SLU+HGmXYAImZwbE9PVXI8pdZ5+vnGyu33SKfQRqON4fvFXt9\\\\\\\/VVKkt\\\\\\\/UWGy1CVZxIS4U0Coed7zT20J6XCyFjvHpEVmrEyNCs8ntLU9X1PvJgk4DSJNxHDc\\\\\\\/uqXBTQvQADSGDxVuykjkcwb2yzvbXSrKlm1qcTXaomKjGvLNHYu8hhNjWzPp1OUINIt1gI4nggT2qRB98WhqVBNIciJ523L8oZhoGP1O2Y9xqx8Gbu8BfdWdUeZX79FgtCdxTTUgoIdWj6ntfjD+kN5zx7oVEniu6Wx92hZDMKWsfRylzO4ptNsQHZ77sQ3bLeKu0KCRlNwMNY6lyRxR7En+jLLknSQrmp\\\\\\\/0I\\\\\\\/fZJ3UGm1BOeYTpD0B61auA1\\\\\\\/PF9nH00cWOICmnk2iULvjI72RnohcXLY3Jqj2BoBy455Pv6nwi+GRuVoRrFxnKrysDt3r7r4oYNeMWNSA0ZlS82GIyR45vGnaiZHBEL0hran+uavXQzhUqPsRDMJpo5ymrxAHdhnv0Gs6DGkux\\\\\\\/DKjG3EsYvRI6Nc8MogS8+1OzWrqxwb7mFMLuPGovMnwfTqVewWztKH3HI47jQb88zcKVPtCW0Gyz5JEBc5lFBTkJWImhy8dnKTJcVTNVUO9dpaCE7oxukpysbxXxEHlbIJaLMSz6kaAJuQrMcKtkFxN3bP1Iuev+DqZwdzfqSbpGJ2oUb2TZB88zZoPVIse8Xn9QDiZ0o74BDbKghb0fgizbcoAQgl0M2AZqR3Lb0wyib700jiROpHuQWLFNKwPMbO8LCxDQZ1fq+L238egbP8bXpqidfakirGLf97UCDfkMSO1nnjlbhm5xMujHVEs0FycPeYlOfGCFFhfPnrCdBPpccPQrsUZPieBm25W34NqqGnDRmjOzDQ8G8tIErCG94GEB1ZJCuBnxtG6SDa+7YuC\\\\\\\/Qpnc9R1Jdsi36Y1+a65HOCA+u+hmUluN87ul2jQQUKWIeYp1kIDrxgfDKFocRdyYRNBlmwoxUhHQD0Oihwc5hOudLVSjX7T7iS7a8FctoSa+wwYdYmGRLtfmO72qPoHfJiEvow0a5pL6yO\\\\\\\/d3tFXfJUT4tMNls0RUaIcmIfDy5yUY48fsWUZzsrMS9m7krpFLPrksZ98EM4ZiQ9S1CvA48QrW23vZpLfxvipLqWunqqXNhGTbq2E8S8\\\\\\\/EbwhAH7WlrgVLSHupIuBody3\\\\\\\/WDiLKFofwM3em4f54UgoZPRpscLQ5P1J1fFCkbF1PaI8XTky6esOBg\\\\\\\/fR5EaVgxfuuboxRyn\\\\\\\/R2Q0kTavh4UpJ56zI4r\\\\\\\/TVKR\\\\\\\/2gRZ7NvCyZtcQrD7lQ0qCePP\\\\\\\/DQbVZUoq8jHrQPiG49GFNU3DTuleZ5I4OigNeAXO8WyELhjOVslsPed\\\\\\\/\\\\\\\/2IAwRUNZ0m05ijx7QZF3XiS134r5M69MkL8HToHj3eCycBrVw8Ydqpn6hp8vMcKnhtPWhBaF6r7mNOY+niUbllP7oAlD+eVH0HqR4ymnEOCRF\\\\\\\/HrUQ+\\\\\\\/X15A5NoX9Xc8ipJirD+M66r4sdqc59oeUEZ5VMzbQp9r9v0pCesvNCvzfGe\\\\\\\/Ag25mYHaFCBmTHI8RwHcVHsOnrbyUVABnKqgc2x8XE5fW6iY0PM0RVb6Ef8blMM25vEg+IM3x0kS40BRa8mKKcinPGyIUAWuxhbH1L5t\\\\\\\/NkXEBylpr0vldYCFoMlQ0gOaKaeHbQiNkoAWMyJ0IufcPsft\\\\\\\/1OfPgAP4EqILabtWtdL9GZqgxmRcpKNfpYx+3e5bGghHsX+Atc+ZREBKGvY7OqhBjoVfRwnw\\\\\\\/mamttLH6438K3PQFZZtdNJaa6Dzuip1cUpuLHA8ScBSVR36UEwO4gl5A\\\\\\\/nWQZmOkq3wEnBx26RJXTuaBEBEIwLsizNS5bEupfcbLj0nURuCt0yEEWEkOiNdIhglcL6xOSBmFPVIBtyF1Hrj7VKFsW2FrUlJP59M5net9x+9ansRY0DBKWR+phh2yvRWQ5\\\\\\\/h1q1ceoeeUZPkI1KLK2ulwBtLpGcbPxWxWUh8hPwj7MCr0oE92Y8Dc+zud8TBgiM8r6QSVgy2Zht7KDnaiG+qjp8kcGcS7xV\\\\\\\/qXSYvtq5MTEReQ1\\\\\\\/XGINaHBGGwf7a+kM9lwuhWqg1PoArs87W73yF9tEK756eSeSHcg4kLqyal7\\\\\\\/+eW3u5hqm1so3Qh9OjaorzJ2ZnnTSaNudvkKRH3tTomAUruQrUfnVb8X+b7ahhzdXt1aq8iYSYulOvCE8yDEyW4DlU6FXxNvqWuDC2AV3r65VQi0I2bbKa93FG3sxtqMYX2VQ2S3mtzDw\\\\\\\/HILaqvEQlQqHYE5kOre4qpD5iVPdHA47HvPXiSbUb3lgv5W8NuTr0Fm64Fiw\\\\\\\/AK3rEiNaQrFpwxjW8fXvztWWI1yL6Uax549moSWXZyUBw4EdfBmTH9ZBQj1jFzG9XDU90TSHTUBbyTXohKZZQlLJKLvI8NPPp0BPFij0sZxHhVjXwjqiOROO6hhmFTMoyX9H0BFyotBHfDbYFlJuiCXrzct0E1pajc0usN2yr6zbuH0mfd\\\\\\\/9j\\\\\\\/d65GDZd5fF+hqY6zJie2Gj5uQiQbwHLAzWq6kijozZyOofZqRLl8bPSORHusMUfwK14csH8MgXWGxVX5eAnKtDZgyl9Qs8PAKu4FhfcFQPn3Elt+sdZFTsI8FloBRFuXRkt\\\\\\\/eqk9QvmpwehgaFvxJ\\\\\\\/pomBU6lNJaLVE4fwHcuJw2Y8EbQKtF+qcy20\\\\\\\/KJCZNwextFvhjKH8kV1KzK4I+xKKqppyP1Dvqc8iYbzQrCBuPvPgtreEjTRd5pArUGstbkyPt59xpxIlCLz\\\\\\\/N99aeX6iN5yJTxeshH0LRkA45o\\\\\\\/jwnTvosrhzP1A21ZcdNfbcJwSW3aZpXga8Wacq30A\\\\\\\/mxCybY4WqMFxM59qPS8QEbM3luDrg4v47Tm+iSx+jJ7GTHxOAT4bCDBBK3iMX9agJpLZjbgQ30GU8v09ASHiGZIPbW0z5zkI1IY1i19\\\\\\\/59EIniUBxIQJEHsJRDViZVXg8atChcUyywNYh68qPHHZlHX9PMroWvd0vSGc4\\\\\\\/tJ70qQlOURrNnOkSS7HwIskfgD4wJn7h08Wil04TqY5FMKscBhEVj6TSVAbL939Hd4Npa8GOIeOV8skrwX6cD4Vp\\\\\\\/t9LAVzGCg1WudW60JGkxkqAepE\\\\\\\/TghVjUTUiELJtUrR47bWpTBH9BiGJyukknNh9yQ6\\\\\\\/DmgkxVm9aCEsMl3V1\\\\\\\/6Q5NV2IWjlGKt8\\\\\\\/q6VMqhSX6Cvvn3CM0uRNvCLtQlpPsFBjUthCKAqoTkaPnIADTAPr5L4jIqy79stZzVfdGfIXVgtfUippVSrq9Fi815XA7zilq4xkWLobfwZiCKhuywsqBAFTu5+TdD\\\\\\\/DnmrUqlICjQIaQpAJv4f1Y\\\\\\\/fUtztxLz8mWopn6Sbqp7ipUcLekWCT3wcU4c\\\\\\\/psMlUMwfz9+wMOWbuRC2GrzFeu\\\\\\\/eqSUwGefFlz0WnYX4c9O1gwrh1sS\\\\\\\/CBWafgtSVtxjvDtyx69AB0yWFBdR6RDhnoeewxSg9fMqNqND5p8D\\\\\\\/67V6ycRYgw8c4fx5jbMjMwbHzWbRsFljNJpOqgaC40WMPjb3G6FdWMAyVQ5\\\\\\\/CkISBSwvjleNUflnfnlm7VBD3CJPLdRwy8pv7m\\\\\\\/3GseDWsiBJxeweqNaP2SbIMT1fiNscoN8dArwgXlLGhazWqQNtzRX+Ukl7PI7ZRXYaRg7wA22rkOcVmHqSQn8\\\\\\\/VkpdmfXzlW2Y6owmIzELF\\\\\\\/V5JW3j17ZY+ln03wkKF2uaWm3xABUDn\\\\\\\/WEOd8Nw0PqoEnehd7+Fqz30PgLOQjmlQhrrOq\\\\\\\/l+GGGZOrosHbG9syPOTCQyVJHcFH8KKXtDaA0Wz\\\\\\\/o+Ii47ceUcX1MkHYUvKClEOmC19UHyuKm+XbvG4OsCqWi5uumpBTmHpI1mqt4ZGtbj2mb6TptO2gJD0iPpWOSw3ZFEPTxHObAdV0PB5b4R20FR2+UKWfJCDefKEtjDZgCQMX+C+ku\\\\\\\/Br5RgNfuFdgjrTGgTvNMpP5bHZIS+Retw7E7UUkDacnI4Q\\\\\\\/SoGCkGzXTY\\\\\\\/GOqzxXao4DtrjMpexubYyXebVfNK5t0\\\\\\\/ykvisKYJl6+v1kQbzqzskDBsF2gv5\\\\\\\/palQm\\\\\\\/bxhQKp7lm0WigjMxEuH9OCfy5NhNfgd0VA572m1JOex4FhIgo7OcVpu0nhaLnvCoa4UUQYGBwTxPZQf0BD8VHJLxxl61jyJUFiI0cO1XuyluyvuYYFuIBJ\\\\\\\/ICm5nR2mzUB9PXPRZk1tKGm15N4rC\\\\\\\/sDu9l02PX48jvfE\\\\\\\/IsQdrZs8Qo1AIqYPuhL5sgLkFsM5\\\\\\\/UMgtiMCJ7SDnyDrIUZLxiOKF8gfKuwYUNlREHwUYeqV\\\\\\\/eXKfjBQM73oyRslETJxJbR3hLcpq\\\\\\\/W0wWhalfp2D76i4VmRstBb4fFnOQfmT+hei5S4mAEVHCHtInZgDnxwjF49xxdqD4OLpgHY2sSwAC7zp6jekU9YjbSpb\\\\\\\/xxUaIIDkqgakPrnvjeuXYOl3rB32MMTFUrimHeGrqOxDn4CRwWxe3OsCqFqCPmASsGQ81wzCycSSxhFrOQDzF\\\\\\\/vUpGUmxkRkkSc\\\\\\\/Dr6Q9Cao4ztQ\\\\\\\/9QcNKLQBh6jVnV\\\\\\\/Lq9r2FS63hjCL3T+fxKVX1pRksKmKtb3fw3TdtU6f6z4mklLEJWRadN+zWnJ2llNq30bTbdPZTjDhdIkN0Efbwptx92NSrUQtnPuvoBDxaVlB6ozOmpXQODad19aSAY8Ih3D39oIB1tPaNlZwrR+sVBbrkWe7KdyzZ9Rqen\\\\\\\/8jZw5yNRZOtobKc8CIG+xx01kwT9HLsTe4N1SumiJL65+aOVrH5fDkYOn9117oa14AmoA\\\\\\\/LRBIPmS7cVauIi9U0g\\\\\\\/w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MiUKoUEOHqGY6noJz2n7uQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e50323ce2b2a81cb589a7a9fd20d947a8405268c1d703f4bb958db7edbcef782\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ef4gyAb4d63Gpg5OMyzBx+Njn1OI6aEUdJTsCXrp+7Eersrx2DvxMY\\\\\\\/jpyBY9hyR9W5Ey6wv5NccEV\\\\\\\/UC+UfEYvc7RRWSQX+iIk7QFql3EmaBxDqM164uew+tf8Frge5RC8AMDBRkRTgs1L+FOQHhk5VHtJExahGJ6zPvNUzzfnvbr9Zm5PpolbqzICQA0ZOkfG2zsNgNxx5OF+Qq+HzZUmeY3pTjT+3jGUqLlh+FJUNBSPkS+KWKViE+VqKhpVONN3QAo2HOYDkf1HG35eJAjeF6MLQDhqlIAfyD7g3rZB1PRoa3hFZh+oYzVMpOopS3kNl\\\\\\\/7dN9nJwNZwdvzz74rdMzZ1g7vwDRv7YGHZmLGF24t3v9hA1KldHTNXx\\\\\\\/WsvWqSm+YrxeG97\\\\\\\/N11DNsQII7OVZzHJbVzS8JTv7+F8PCu4OoTOCcuxF18BeScG\\\\\\\/tTQS6ieBhwcHJ8hQZ7hv2rc4seGdH5xi4IXtNuyZfdeXOcaaF5xHHJCvmrVBMtBQ82ulNQJLia48a1AZDxaUzhSie\\\\\\\/Hris9tSTW\\\\\\\/l6TwSR7nY8+cozPpawOmfQvHTuB6\\\\\\\/y8FhxxkJCVavUbY+AdPsVNxDaHljOm4w6Jr+iQ3oyYDbilysD\\\\\\\/Jr0YTHHk7AgbcYyuzgZrdW\\\\\\\/YzazArpbmZjnZN1sv4WOFuytYb9bO3N8reXhzIu5Q9NFld4bT4UGG+k9uihLG\\\\\\\/1rItivWxj0+5xDEyAR6WnZZRGZyKZtHLfo7UrOjsGr7VpZo75wV4Ib62f4fZu11m2+pTMH\\\\\\\/eEBvL5hWTo+xyIm3yzc7+aXMiwvJwbP+afrd8pkRS0Dw5pqTm2TPuMNybxktb8vYGLHyQmjd+ophMjjGD\\\\\\\/3H5iErwfpK8IvWTT0CxvRs2XD7HKB8nTDH6K1aXC\\\\\\\/iVd1LCnLheEDFgRt+x1HF2x2t\\\\\\\/\\\\\\\/eIoZGpn8NuU0DN6HNSK6SZjbkbfvqzOKpLdeDDuK2LnL48JZmzFOM2ZQGEvV1PQiBkOalYmVLaUG0WmTFXDra2BAAOXOTsy6wyc3G6Y8UZhjQ8Z4Upy17Wt2F6R7+XkQRafWutElH8iL9tn1Zdm774ZSWQAFGNXA9uq4bEgjoJ3jddys+gcBpHP8eLRxlTiPI9H7sA1phwjooHu8dQBPPloUEVgamtvkYupmHJkOavwcYP9LZPcs0b+GRTMhYqM9stYbA2m6dSL5ULtIc3XBS06SfdA4yJkfj8UFj9yLJmjNO2uFZZWXyIEVuYZLTBItmMTuLXGx\\\\\\\/TrjX+IohjAuanJ9lS5igj2EVddFFnRJR2SCZ59sthO0PDNpKjZIn9+awLgMrHiY+hWKRV5hCbe5u4NHmDOaoBKMjsi3H\\\\\\\/s9joUkHEQEosFagLJoUrb32AJEEVuYIrkcyGR++GW7YsuSR3o4RgfjLBHZv1ffbVqFWkX7Tbc7H3gO6jCI6tud7L8DWq5w0+7sJ9yxqsLli540CQoD+FgtTP7xrX4TjZ+Zmmyt3ac0L7rJGAtkbBRgFaCIECxHqZZAHkH3SFyERjQa2hgB6m0SkmgAhvoD8mvKGsdVq\\\\\\\/whv6Sp8qF9s1oPP9BA3iOSWPAifUvHFlphMvbBoafSm\\\\\\\/K0iecv1rYGwBMFhDwB4MpnJiYElujBMY6k0NvpKXspyrevBSGvcoJSN\\\\\\\/Vf5YkYTjZddNBtjcOS6cSHCZIR0CeGQ8o6sfiZycroqn2NifVK9VlIlDyFifKHsewUNXEchi3DiZ7uf56ctI0YL8RPGBQw6Ljp1sJpji4X3KZAl\\\\\\\/+uvFvgo8MvyTjXsgewHblo63EnwknKrM6ITEp+qIsxqGgSspsUjThI\\\\\\\/nhyePQyhgcvSzVb+tc6E8VJjTwydE35zPWc0GCAWERyC9WT\\\\\\\/Zsf1VEcpV\\\\\\\/PqsKWrCEjPQcT2rGZpA+9DIB1h2EPqZfPTlYmwOy5OB1V+5yD+dAp4h47RJu2ls9G3WnqiP6vJvXOltGbi3Du7D0YC4LW76gr\\\\\\\/hYgqMCKUdt1DOcxniLOlijaMwakOsA==\\\u0022,\\\u0022iv\\\u0022:\\\u00223AoL9syjGMT0pUXygWceNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e72e0ebe7b644244f0f3db3c2994007799a8bd5bdbbd086b173dd3a29b6582f7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490452 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2673" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BXrdjTyLHeOO3asY3v3t1AF9ub3llBS9vTcV5IYYKYw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2673" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ccAEHhtE7zj_sc0bOGou7Fiy_WmsmmfQ3PRo32bLpis" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2663" title="Idiopathic Scoliosis Families Highlight Actin-Based and Microtubule-Based Cellular Projections and Extracellular Matrix in Disease Etiology" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2685" title="Genetic Analysis of Signal Generation by the Rgt2 Glucose Sensor of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490452" id="node490452" data-pisa="ggg;8/8/2673" data-pisa-master="ggg;g3.118.200118" data-apath="/ggg/8/8/2673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting <em>de Novo</em> Homoeologous Recombination Events in Cultivated <em>Brassica napus</em> Using a Genome-Wide SNP Array</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erin E. Higgins</span>, <span class="highwire-citation-author" data-delta="1">Wayne E. Clarke</span>, <span class="highwire-citation-author" data-delta="2">Elaine C. Howell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7284-5702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan J. Armstrong</span> and <span class="highwire-citation-author" data-delta="4">Isobel A. P. Parkin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2673-2683;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200118 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490452" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erin E. Higgins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BE.%2BHiggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Higgins%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BE.%2BHiggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wayne E. Clarke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wayne%2BE.%2BClarke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clarke%20WE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWayne%2BE.%2BClarke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elaine C. Howell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elaine%2BC.%2BHowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howell%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElaine%2BC.%2BHowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Susan J. Armstrong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BJ.%2BArmstrong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armstrong%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BJ.%2BArmstrong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7284-5702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan J. Armstrong</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Isobel A. P. Parkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isobel%2BA.%2BP.%2BParkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parkin%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsobel%2BA.%2BP.%2BParkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:isobel.parkin@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">isobel.parkin@agr.gc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2673" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490452" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2673.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490452" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2673.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490452" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2673.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490452" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2673.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490452" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The heavy selection pressure due to intensive breeding of <em>Brassica napus</em> has created a narrow gene pool, limiting the ability to produce improved varieties through crosses between <em>B. napus</em> cultivars. One mechanism that has contributed to the adaptation of important agronomic traits in the allotetraploid <em>B. napus</em> has been chromosomal rearrangements resulting from homoeologous recombination between the constituent A and C diploid genomes. Determining the rate and distribution of such events in natural <em>B. napus</em> will assist efforts to understand and potentially manipulate this phenomenon. The Brassica high-density 60K SNP array, which provides genome-wide coverage for assessment of recombination events, was used to assay 254 individuals derived from 11 diverse cultivated spring type <em>B. napus</em>. These analyses identified reciprocal allele gain and loss between the A and C genomes and allowed visualization of <em>de novo</em> homoeologous recombination events across the <em>B. napus</em> genome. The events ranged from loss/gain of 0.09 Mb to entire chromosomes, with almost 5% aneuploidy observed across all gametes. There was a bias toward sub-telomeric exchanges leading to genome homogenization at chromosome termini. The A genome replaced the C genome in 66% of events, and also featured more dominantly in gain of whole chromosomes. These analyses indicate <em>de novo</em> homoeologous recombination is a continuous source of variation in established <em>Brassica napus</em> and the rate of observed events appears to vary with genetic background. The Brassica 60K SNP array will be a useful tool in further study and manipulation of this phenomenon.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/chromosomal-rearrangement" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-rearrangement" rel="nofollow">chromosomal rearrangement</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/brassica-napus" class="hw-term hw-article-keyword hw-article-keyword-brassica-napus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Brassica napus</em></a></li><li class="kwd"><a href="/keyword/neopolyploid" class="hw-term hw-article-keyword hw-article-keyword-neopolyploid" rel="nofollow">neopolyploid</a></li><li class="kwd"><a href="/keyword/duplicationdeletion" class="hw-term hw-article-keyword hw-article-keyword-duplication-deletion" rel="nofollow">duplication/deletion</a></li></ul><p id="p-3">The genomic relationship between the major Brassica species was first described by <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">U. N. (1935)</a> and is defined by three diploid species: <em>B. rapa</em> (A genome), <em>B. nigra</em> (B genome) and <em>B. oleracea</em> (C genome); and three allotetraploids created from each pair-wise hybridization of these genomes: <em>B. juncea</em> (A and B genomes), <em>B. napus</em> (A and C genomes) and <em>B. carinata</em> (B and C genomes). Of these Brassica species, <em>B. napus</em> (canola or oilseed rape) is the most economically important and is believed to have been formed in the last 10,000 years centered around Mediterranean Europe (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chalhoub <em>et al.</em> 2014</a>) and is now grown on all continents, harvested predominantly for its oil. The Brassica species provide an excellent platform for the study of genome evolution in polyploids since they encompass multiple ancient genome duplication events (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chalhoub <em>et al.</em> 2014</a>). These events include the gamma triplication event common to most eudicots, the α and β whole genome duplication common to all Brassicaceae, a Brassica lineage specific whole genome triplication that led to the formation of the Brassica diploids (or mesopolyploids), and most recently whole genome hybridization resulting in the three allopolyploid (or neopolyploid) species (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Masterson 1994</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lysak <em>et al.</em> 2005</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schranz <em>et al.</em> 2006</a>).</p><p id="p-4">Polyploid formation leads to a phase of genomic shock in response to the duplication of all genes, with concomitant gene balance and regulatory issues (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wendel 2000</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doyle <em>et al.</em> 2008</a>). During meiosis, homologous chromosomes pair and the ensuing recombination facilitates the production of viable gametes, each with a complete set of chromosomes. However, in neopolyploids formed from closely related species, a chromosome may have more than one potential pairing partner, termed a homoeologue. Cytological analysis of meiotic cells in neopolyploids shows formation of bivalents, unpaired univalents and multivalents of homologous and homoeologous chromosomes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Attia and Röbbelen 1986b</a>). Recombination between homoeologous chromosomes where all or part of a chromosome from one genome is replaced with the homologous regions from the second genome can result in inheritance of either the recombined segments from both homoeologues, with no apparent loss of genetic material, or only one of the recombined segments, leading to gain and loss of genetic material. The latter events have often been termed homoeologous non-reciprocal translocations (HNRT), although by their nature they are derived from reciprocal exchange, to prevent confusion such events will be referred to as homoeologous recombination (HeR) events or exchanges. Multivalent and univalent formation can also lead to unbalanced gametes. During anaphase I unpaired chromosomes either move to one pole or are split between the poles by the spindle apparatus. The complicated dissolution of multivalents leads to unpredictable separation of chromosomes, most of which will result in unbalanced gametes (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zamariola <em>et al.</em> 2014</a>). Such gametes can be nonviable or the resultant embryos may produce plants that are sterile or unfit for their current environment resulting in an overall decrease in yield (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jenczewski and Alix 2004</a>). The mechanisms responsible for the genetic stabilization (or diploidisation) of neopolyploids are of interest for maintaining fitness in crops, limiting gene flow to native plants, and exploiting the diploid gene pools of polyploid progenitors for novel traits.</p><p id="p-5">Polyploidy is very common in plants, including several important crop species such as wheat, cotton, canola, coffee and peanut, and evidence exists for at least some level of genetic control of chromosome pairing in all of these species (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cifuentes <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lashermes <em>et al.</em> 2016</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nguepjop <em>et al.</em> 2016</a>). The most well characterized of these is wheat where chromosome pairing control has been studied since the 1950s after discovery of the <em>Pairing homoeologous1 (Ph1)</em> locus that had a major effect on the control of homoeologue pairing and recombination (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Riley and Chapman 1958</a>). Using cytology it was observed that in plants lacking the <em>Ph1</em> locus there were more univalents and multivalents at metaphase I of meiotic cells rather than the typical prevalence of homologous bivalents, but a precise mechanism for this phenotype continues to be investigated (see (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Greer <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhullar <em>et al.</em> 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Martín <em>et al.</em> 2017</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rey <em>et al.</em> 2017</a>) for recent work). Though fewer studies have focused on the genetic control of pairing in Brassica, it is an excellent system for studying pairing control and homoeologous recombination because the allotetraploid species of the triangle of U (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">U. N. 1935</a>) can be recreated through crossing and subsequent chromosome doubling of the two constituent diploid species (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Snowdon 2007</a>). Researchers have successfully used sequential fluorescence <em>in situ</em> hybridization (FISH) and genomic <em>in situ</em> hybridization (GISH) to distinguish the A and C genome chromosomes in <em>B. napus</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Howell <em>et al.</em> 2008</a>), and a novel chromosome painting technique was used to identify all of the chromosomes from <em>B. rapa</em>, <em>B. napus</em> and <em>B. oleracea</em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Xiong and Pires 2011</a>). This makes it possible to not only identify homoeologous bivalents and multivalents but to pinpoint the chromosomes preferentially pairing in meiotic cells. Cytological analysis previously identified a major quantitative trait locus (QTL) that contributed to variation in homoeologous chromosome pairing in allohaploid <em>B. napus</em> plants of two genotypes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jenczewski <em>et al.</em> 2003</a>). In addition, this locus appeared to impact homologous recombination; however, it did not seem to contribute to variable homoeologous pairing in allotetraploids (diploids) of the same <em>B. napus</em> lines (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nicolas <em>et al.</em> 2009</a>).</p><p id="p-6">Molecular markers have previously been used to identify homoeologous recombination events in <em>B. napus</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Parkin <em>et al.</em> 1995</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sharpe <em>et al.</em> 1995</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Udall <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rousseau-Gueutin <em>et al.</em> 2017</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Stein <em>et al.</em> 2017</a>). By visualizing both A and C genome loci with restriction fragment length polymorphism (RFLP) markers it was possible to resolve homoeologous recombination events by the gain of an allele at one locus coupled with loss of an allele at the homoeologous locus (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Parkin <em>et al.</em> 1995</a>). This simultaneous gain and loss of alleles at genetically linked loci on homoeologous chromosomes provided evidence of HeR events. Such analyses of a population derived from a cross between a newly resynthesized <em>B. napus</em> (created by crossing a <em>B. rapa</em> and <em>B. oleracea</em> line followed by chromosome doubling to produce an allotetraploid) and an established <em>B. napus</em> parent line showed a significant increase in homoeologous recombination between the A and C genomes relative to a population derived from a cross between two adapted <em>B. napus</em> parent lines (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Parkin <em>et al.</em> 1995</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Sharpe <em>et al.</em> 1995</a>). While highly reproducible the laborious nature of RFLP markers makes them difficult to assess for a large number of lines across the whole genome. Simple sequence repeat (SSR) markers have been used to show reciprocal gain and loss of A1 and C1 loci in progeny of a resynthesised <em>B. napus</em> (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Szadkowski <em>et al.</em> 2010</a>) but SSRs offer only a marginal advantage in assay time compared to RFLP markers. The development of the Brassica 60K Infinium single nucleotide polymorphism (SNP) array (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2016</a>) provides a high-density genome-wide platform to assess homoeologous recombination. SNP arrays allow genotyping of hundreds of lines at thousands of loci in a matter of days and have been successfully used for genetic mapping in bi-parental populations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yang <em>et al.</em> 2017</a>), differentiating between the different Brassica species of U’s triangle (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mason <em>et al.</em> 2015</a>), identification of parental alleles in interspecific Brassica species (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mason <em>et al.</em> 2014</a>), and genome wide association studies (GWAS) on diverse sets of <em>B. napus</em> germplasm (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hatzig <em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Körber <em>et al.</em> 2015</a>). Use of the Brassica 60K array to identify segmental deletions in resynthesized <em>B. napus</em> has been combined with cytological analysis to identify translocations caused by homoeologous recombination in resynthesized <em>B. napus</em> individuals (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Rousseau-Gueutin <em>et al.</em> 2017</a>). Similarly, the lack of amplification at physically linked SNP loci was used in conjunction with re-sequencing data to reveal homoeologous exchanges underlying QTL for <em>B. napus</em> seed quality traits (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Stein <em>et al.</em> 2017</a>).</p><p id="p-7">This paper describes use of the Brassica 60K SNP array to identify <em>de novo</em> homoeologous recombination events in allotetraploid <em>B. napus</em>. The high-density coverage provided by the array allows for genome-wide detection of recombination events at a greater depth and higher resolution than previous marker-based assays. The efficacy of this method was tested by assaying levels of <em>de novo</em> homoeologous recombination in 10 testcross populations derived from established <em>B. napus</em> lines. These data provide a range of expected levels for such events in <em>B. napus</em> and define genomic regions more prone to homoeologous recombination.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Testcross population development</h3><p id="p-8">Ten <em>B. napus</em> lines were chosen from a collection of spring-type cultivars (ACSRsyn1, Bronowski, Daichousen (fuku), Maris Haplona, PAK85912, Surpass 400, Svalof’s Gulle, Topas, Tribune, Zhongyou 821) based on diverse geographical distribution (Canada, Poland, Korea, United Kingdom, Pakistan, Australia, Sweden, Canada, Australia, China, respectively) and where available molecular information (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bus <em>et al.</em> 2011</a>). Formation of ACSRsyn1 was created by crossing a <em>B. napus</em>/<em>B. oleracea</em> triploid with a <em>B. napus/B. rapa</em> triploid (DTN-1/<em>B. alboglabra</em> 89-5402//DTN-1/<em>B. rapa</em> Parkland) and selecting for an individual with a complete AACC genome followed by selfing for several generations (provided by Sally Vail, Agriculture and Agri-Food Canada, Saskatoon). Plants were grown in a greenhouse at 18° with a 16/8 hr photoperiod (day/night). Hand pollinations were used to cross the <em>B. napus</em> individuals with the Australian <em>B. napus</em> cultivar “Rainbow” to produce a testcross population for each line. Young leaf tissue of sixteen individuals for each of the populations was harvested and freeze-dried for DNA extraction. Three of the testcross populations, PAK85912, Zhongyou821 and Maris Haplona were expanded to 48 individuals each, though two of the PAK85912 progeny were determined to be selfs and were disregarded from the data set.</p></div><div id="sec-3" class="subsection"><h3>Brassica SNP array</h3><p id="p-9">High quality DNA was extracted from freeze-dried leaf tissue using a cetyltrimethylammonium bromide (CTAB) based method (Murray and Thompson 1980). DNA was quantified with the Quant-it Picogreen dsDNA assay kit (Life Technologies Inc., Burlington ON, Canada) and 200 ng was hybridized to the Brassica 60K Infinium array (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2016</a>) as described in the manufacturer’s protocol (Illumina Inc., San Diego, CA). The arrays were scanned using an Illumina HiScan and SNP data were analyzed using the genotyping module of the GenomeStudio software package (Illumina Inc.) using default settings with the exception of the no-call threshold, which was set at 0.05 and a custom cluster file was applied (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2016</a>). The software creates a two-dimensional image for each SNP marker where the graphical position of each individual is determined by the fluorescent intensity (R value - y axis) and the ratio of the two allele-specific fluorophores (θ value – x axis). Individuals are assigned a genotype based on their position in the graph. The software is designed for diploid species with two alleles at each locus (AA/BB) so in a typical cross between two homozygous parents, a classic three cluster profile is produced, the AA and BB clusters would reflect the genotype of each parent and the AB cluster would represent heterozygous progeny. Single copy SNP markers were pre-selected by aligning the flanking sequence provided in the manifest file for the Brassica 60K array to the <em>B. rapa</em> and <em>B. oleracea</em> genome assemblies (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Parkin <em>et al.</em> 2014</a>) using BLAT (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kent 2002</a>) and selecting those markers with &gt;90% identity in one diploid genome and &lt;90% identity in the other diploid genome. This resulted in a set of 38,970 markers that were then filtered for polymorphism between the parents of each population. The level of SNP polymorphism for each testcross population is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> for each linkage group and the GenomeStudio exported SNP data for each population is provided in Supplementary Tables S1-S10. Three or more consecutive missing or duplicated SNPs were used to identify affected regions; however, the majority of the events (91.5%) were defined by 10 or more physically linked SNP loci (Supplementary Table S11). HeR events were identified by analysis of the homoeologous regions in the testcross individuals as described in the RESULTS.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490633/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490633/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of distribution of polymorphic SNP markers per chromosome for each testcross population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Identification of homoeologous regions from the SNP array</h3><p id="p-10">The results of the BLAT alignment from aligning the flanking sequences from the SNP loci to the <em>B. rapa</em> and <em>B. oleracea</em> genome assemblies was also used to identify the top hit in each of the A and C genomes for each SNP probe. Probes with at least 50% identity in both diploid genomes were selected, resulting in 28,334 SNP markers mapped to the A and C genome (Supplementary Table S12) that could be used to determine the homoeologous alignment of the A and C genomes.</p></div><div id="sec-5" class="subsection"><h3>Detection of inherited HeR events using whole genome shotgun (WGS) data</h3><p id="p-11">DNA from Zhongyou821 was extracted from nuclei according to <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Parkin <em>et al.</em> (2014)</a>. A short insert (350 bp) Illumina DNA sequencing library was constructed according to the manufacturer’s instructions (Illumina, Inc.) and 125 bp paired end (PE) data were generated on the HiSeq2000 platform, providing in total 111 million (M) PE reads (estimated 23x coverage of 1200 Mb genome). Trimmomatic v0.32 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>) with the following parameters, LEADING:15 TRAILING:15 SLIDINGWINDOW:4:15 MINLEN:55, was used to remove low quality reads, short inserts and adapter sequences, resulting in 103 M high quality PE reads. A combined pseudo-reference genome was generated from concatenating the <em>B. rapa</em> and <em>B. oleracea</em> genome assemblies (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Parkin <em>et al.</em> 2014</a>). Bowtie2 v2.3.3.1 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Langmead and Salzberg 2012</a>) was used to align PE data to the pseudo-reference using the following parameters --local --sensitive --phred33 --minins 0 --maxins 1000 --no-mixed --no-discordant --no-unal --k 20 --dovetail. A custom perl script was used to retain the best alignment for each read as long as the next hit was significantly lower in call stringency. The overall alignment rate was 90.05%, with 92.8 M mapped PE reads. The resultant alignment file was analyzed using the R scripts described in <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Samans <em>et al.</em> (2017)</a>, which normalize the read depth across the length of each chromosome, identify regions of the genome where the read depth significantly differs from the chromosomal mean (at 1.5 SD), and finally compare homoeologous regions to define potential fixed HeR events.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-12">Tables S1-S10 contain the SNP marker data for all testcross populations. Table S11 lists all HeR, duplication and deletion events found in the testcross populations. Table S12 provides details of the SNP markers used for homoeologous alignment of A and C genomes. Table S13 shows the compressed SNP data which summarizes <em>de novo</em> chromosome gain and loss in each testcross individual. Table S14 lists HeR events in Zhongyou821 identified through whole genome sequencing. The WGS data for Zhongyou821 has been uploaded to the NCBI short read data archive (<a href="https://trace.ncbi.nlm.nih.gov/">https://trace.ncbi.nlm.nih.gov/</a>) under BioProject ID PRJNA454160. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6207530">https://doi.org/10.25387/g3.6207530</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Dissecting SNP marker patterns to identify homoeologous recombination events</h3><p id="p-13">Recombination rates for an individual can be determined by studying the products of meiosis or the genotypes of resulting progeny in the subsequent generation. Ten test populations were derived by crossing ten spring-type <em>B. napus</em> lines with the Australian <em>B. napus</em> cultivar Rainbow. In total, 256 individuals were assayed with the Brassica 60K SNP array, two lines were identified as self progeny. Initially 16 individuals for each of the 10 <em>B. napus</em> testcross populations were assessed and three populations with differing levels of observed events, PAK85912, Maris Haplona and Zhongyou821, were expanded to 46-48 individuals each. For each individual, the meiosis of both the <em>B. napus</em> line and the testcross parent Rainbow could be assessed, meaning the products of 508 meioses were evaluated.</p><p id="p-14">Although the nature of polyploid genomes can impact the use of hybridization based tools such as Infinium arrays, the Brassica 60K SNP array was designed such that ∼58% of the loci on the array were estimated to amplify a single genome (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2016</a>). However, conversely as much as 32% of the SNPs on the array identify both an A and C genome locus. Although intuitively these loci might appear useful in the analyses of homoeologous recombination, as detailed in <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mason <em>et al.</em> (2017)</a> since both loci use the same two fluorophores for the A and B alleles, they cannot readily be distinguished, thus such loci were eliminated from the analyses.</p><p id="p-15">Since homoeologous recombination events are relatively rare, affecting only a small part of the genome, the most common pattern observed for single locus assays, approximately 90% of the time, in all populations was the typical three cluster pattern, AA/AB/BB, expected for an F<sub>1</sub> with no gain or loss of alleles in the testcross individuals. The two parents were found in the AA and BB clusters respectively and all the progeny were found in the heterozygote cluster, suggesting normal homologous recombination and segregation had occurred during meiosis in the testcross parents (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Range of observed SNP patterns in Brassica napus testcross individuals. A) GenomeStudio image of a typical three cluster SNP with Parent 1 in red (genotype AA), Parent 2 in blue (genotype BB) and all testcross individuals of polymorphic populations in purple (genotype AB); B) GenomeStudio image of four cluster SNP with two AAB individuals shown in green; C) GenomeStudio image of five cluster SNP with one AAB individual shown in green and two ABB individuals shown in orange; D) GenomeStudio image of a SNP with no amplification in Parent 1 therefore all testcross individuals in polymorphic crosses have genotype B0 and are in the BB parental cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887735115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Range of observed SNP patterns in &lt;em&gt;Brassica napus&lt;/em&gt; testcross individuals. A) GenomeStudio image of a typical three cluster SNP with Parent 1 in red (genotype AA), Parent 2 in blue (genotype BB) and all testcross individuals of polymorphic populations in purple (genotype AB); B) GenomeStudio image of four cluster SNP with two AAB individuals shown in green; C) GenomeStudio image of five cluster SNP with one AAB individual shown in green and two ABB individuals shown in orange; D) GenomeStudio image of a SNP with no amplification in Parent 1 therefore all testcross individuals in polymorphic crosses have genotype B0 and are in the BB parental cluster.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2673/F1.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2673/F1.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490606" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Range of observed SNP patterns in <em>Brassica napus</em> testcross individuals. A) GenomeStudio image of a typical three cluster SNP with Parent 1 in red (genotype AA), Parent 2 in blue (genotype BB) and all testcross individuals of polymorphic populations in purple (genotype AB); B) GenomeStudio image of four cluster SNP with two AAB individuals shown in green; C) GenomeStudio image of five cluster SNP with one AAB individual shown in green and two ABB individuals shown in orange; D) GenomeStudio image of a SNP with no amplification in Parent 1 therefore all testcross individuals in polymorphic crosses have genotype B0 and are in the BB parental cluster.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The gain and loss of alleles due to homoeologous recombination created more complicated cluster patterns, polymorphic SNPs expected to amplify a single genome (A or C), produced patterns with the expected three, but also one, four and five distinct clusters and were used to score the testcross populations. In each population, between 4–19% of the polymorphic markers showed these aberrant cluster patterns. These loci had the hallmarks of single copy SNPs, with parental alleles optimally separated (AA θ value &lt;0.15, BB θ value &gt;0.85) and heterozygote genotypes falling equi-distant between the two, yet additional clusters were observed across the horizontal plane. Those showing four distinct clusters had testcross individuals falling into three different groups: 1) with either one of the two parents; 2) in the expected AB cluster; or c) in a new cluster between the AB group and one of the parents (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). These patterns can be explained by the gain or loss of alleles due to homoeologous recombination in one of the parents. Segregation of testcross individuals with either parent indicates they are missing an allele that should have been inherited from the other parent. Since these lines only carry an allele from one parent they were designated as genotype A0 (or B0). Individuals in the expected heterozygote cluster are presumed to have inherited one allele from each parent, and are therefore genotype AB. Based on the position of the new cluster between one of the parents and the AB group, those individuals are presumed to have inherited one allele from the first parent and two copies of the allele from the second parent and are therefore genotype ABB (or AAB). In other cases individuals were observed on both sides of the heterozygote cluster resulting in five clusters on the SNP image, indicating there were homoeologous exchanges occurring in both Rainbow and the other <em>B. napus</em> parent, which were inherited in some of the individuals, so all five genotypes, A0, AAB, AB, ABB and B0 are represented (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). Lastly, some SNPs had a single cluster containing one parent and all of the testcross individuals while the other parent showed no amplification, indicating the absence of the SNP locus in one of the parents (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>).</p></div><div id="sec-9" class="subsection"><h3>Defining the genomic position of the homoeologous recombination (HeR) events</h3><p id="p-18">Due to the genome specificity of the SNP assays and filtering of the SNP loci, the A and C genome loci were assessed independently, thus the gain/loss of homoeologous loci was not captured simultaneously by any one SNP assay. Homoeologous recombination (HeR) events by definition result in the exchange of chromosomal material between syntenic regions of the A and C genome within <em>B. napus</em>. Because only a single A or C genome locus can be scored for each SNP, it was necessary to first identify the syntenic regions between the genomes that can be identified by the SNP loci on the array, and then look for reciprocal allele gain and loss at SNP loci in those regions, thus determining the level and distribution of HeR events. The flanking sequences of the SNPs on the Brassica 60K array were aligned against the genomic sequence of <em>B. rapa</em> (A genome) and <em>B. oleracea</em> (C genome) which produced a clear alignment of homoeologues with comprehensive genome coverage that was in concordance with previous analyses (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Parkin <em>et al.</em> 2014</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In some cases two chromosomes were entirely aligned along their length, for example A1 and C1, while others had two or more homoeologous partners, such as A9 and A10, which were aligned with the top and bottom of C9, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Circos plot (Krzywinski et al. 2009) depicting alignment of the B. rapa and B. oleracea genomes, and summary of de novo homoeologous recombination events in 11 B. napus genotypes. The B. rapa chromosomes A1-A10 and B. oleracea chromosomes C1-C9 are shown in the outer ring. Homoeologous regions between the chromosomes as identified by the sequence homology of SNP probes are shown as colored links drawn in the center of the image. The three rings internal to the chromosomes show from outer to inner: duplication events as black tiles; chromosomal gain and loss due to de novo HeR as red and blue tiles, respectively; and deletions as gray tiles. The vertical black line on each chromosome represents the approximate centromere position (Wang et al. 2011; Parkin et al. 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887735115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Circos plot (Krzywinski &lt;em&gt;et al.&lt;/em&gt; 2009) depicting alignment of the &lt;em&gt;B. rapa&lt;/em&gt; and &lt;em&gt;B. oleracea&lt;/em&gt; genomes, and summary of &lt;em&gt;de novo&lt;/em&gt; homoeologous recombination events in 11 &lt;em&gt;B. napus&lt;/em&gt; genotypes. The &lt;em&gt;B. rapa&lt;/em&gt; chromosomes A1-A10 and &lt;em&gt;B. oleracea&lt;/em&gt; chromosomes C1-C9 are shown in the outer ring. Homoeologous regions between the chromosomes as identified by the sequence homology of SNP probes are shown as colored links drawn in the center of the image. The three rings internal to the chromosomes show from outer to inner: duplication events as black tiles; chromosomal gain and loss due to &lt;em&gt;de novo&lt;/em&gt; HeR as red and blue tiles, respectively; and deletions as gray tiles. The vertical black line on each chromosome represents the approximate centromere position (Wang &lt;em&gt;et al.&lt;/em&gt; 2011; Parkin &lt;em&gt;et al.&lt;/em&gt; 2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2673/F2.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2673/F2.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490536" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Circos plot (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Krzywinski <em>et al.</em> 2009</a>) depicting alignment of the <em>B. rapa</em> and <em>B. oleracea</em> genomes, and summary of <em>de novo</em> homoeologous recombination events in 11 <em>B. napus</em> genotypes. The <em>B. rapa</em> chromosomes A1-A10 and <em>B. oleracea</em> chromosomes C1-C9 are shown in the outer ring. Homoeologous regions between the chromosomes as identified by the sequence homology of SNP probes are shown as colored links drawn in the center of the image. The three rings internal to the chromosomes show from outer to inner: duplication events as black tiles; chromosomal gain and loss due to <em>de novo</em> HeR as red and blue tiles, respectively; and deletions as gray tiles. The vertical black line on each chromosome represents the approximate centromere position (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Parkin <em>et al.</em> 2014</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Genome-wide coverage and density of SNP markers along the chromosomes is important to ensure there is sufficient polymorphism between Rainbow and the 10 <em>B. napus</em> testcross parents because allele gain and loss can only be visualized when the two parents have opposing alleles. The large number of usable markers on the Brassica 60K array maximized the ability to identify homoeologous regions and determine if an event involved the two homoeologues or if only one genome was duplicated or deleted. The number of informative polymorphic markers in each testcross population is summarized in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and complete genotype information for all SNP markers is shown in Supplementary Tables S1-S10.</p><p id="p-21">For each of the testcross individuals the genotypes for each A genome SNP locus were aligned with the genotypes for the SNPs from the homoeologous region of the C genome to allow ready visualization of the reciprocal allele gain and loss. For a given individual, when the SNP loci from say the C genome showed an extra copy of the C genome allele from one parent (scored as ABB or AAB), the allele from the corresponding region in the A genome was missing (scored as A0 or B0) (Supplementary Table S13). This gain/loss pattern was observed along both homoeologues for at least three physically linked SNP markers in order to confidently determine a HeR exchange had occurred. The number of physically linked loci identifying HeR affected regions ranged from 3-1879, with only four events defined by the minimum number of loci, and the majority of the rearrangements (91%; 151/165 affected regions) were identified by 10 or more physically linked loci (Supplementary Table S11).</p></div><div id="sec-10" class="subsection"><h3>Confirmation of fixed HeR events detected with the SNP array</h3><p id="p-22">Recent studies using whole genome (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chalhoub <em>et al.</em> 2014</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Samans <em>et al.</em> 2017</a>) and transcriptome sequencing (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">He <em>et al.</em> 2016</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lloyd <em>et al.</em> 2017</a>) show evidence of historical HeR events which have become fixed in modern <em>B. napus</em> varieties. These types of events could be seen in the current analyses, where all individuals of a testcross family are not in the expected central heterozygous cluster for a set of homoeologous SNP loci, but are biased toward one parent at SNP loci for one genome while at the homoeologous loci, all individuals are in the other parental homozygous cluster, indicating a duplication and deletion, respectively. Detection of such an event is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> for a fixed HeR exchange between the A9/C9 chromosomes in the Zhongyou821 population. Whole genome shotgun sequencing of Zhongyou821 verified the presence of this event, as evidenced by a significant increase and decrease in normalized read depth as determined through sequence alignment to the A9 and C9 chromosomes, respectively (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Of the eight such HeR events that could be resolved using the sequence analyses, five were detected in the SNP array data, ranging in size from one to nine Mb (Supplementary Table S14). These data provided confirmatory evidence that SNP array analysis of F<sub>1</sub> populations can be used to identify historical HeR fixed events in <em>B. napus</em> lines.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of a fixed HeR event in Zhongyou821 using SNP array data and confirmed through whole genome re-sequencing. A) GenomeStudio images from A9; the SNP loci sampled at 1.9 Mb and 4.5 Mb show all testcross individuals (purple) in the AAB cluster biased toward the Zhongyou821 parent (red) and away from Rainbow (blue). Individuals in gray in the expected AB cluster are from other test cross populations without the genome rearrangement. Below the rearrangement (example at 11.3 Mb) normal SNP patterns were observed. B) Plot of normalized sequence read depth along the A9/C9 chromosomes from whole genome shotgun sequencing data of Zhongyou821. On A9 the first Mb is deleted and 2-8 Mb is duplicated, as opposed to C9 where the first Mb is duplicated and 2-11 Mb is deleted. C) GenomeStudio images from C9 showing no amplification of loci in Zhongyou821 (black) at 2.9 Mb and 9.0 Mb, while all testcross individuals (red) cluster with the Rainbow parent. At 18.1 Mb the parents are in the AA and BB clusters and the testcross lines (purple) are AB. The lone individual to the left is Zhongyou821 testcross line #26, which has an extra C9 chromosome and therefore still has the genotype AAB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887735115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Identification of a fixed HeR event in Zhongyou821 using SNP array data and confirmed through whole genome re-sequencing. A) GenomeStudio images from A9; the SNP loci sampled at 1.9 Mb and 4.5 Mb show all testcross individuals (purple) in the AAB cluster biased toward the Zhongyou821 parent (red) and away from Rainbow (blue). Individuals in gray in the expected AB cluster are from other test cross populations without the genome rearrangement. Below the rearrangement (example at 11.3 Mb) normal SNP patterns were observed. B) Plot of normalized sequence read depth along the A9/C9 chromosomes from whole genome shotgun sequencing data of Zhongyou821. On A9 the first Mb is deleted and 2-8 Mb is duplicated, as opposed to C9 where the first Mb is duplicated and 2-11 Mb is deleted. C) GenomeStudio images from C9 showing no amplification of loci in Zhongyou821 (black) at 2.9 Mb and 9.0 Mb, while all testcross individuals (red) cluster with the Rainbow parent. At 18.1 Mb the parents are in the AA and BB clusters and the testcross lines (purple) are AB. The lone individual to the left is Zhongyou821 testcross line #26, which has an extra C9 chromosome and therefore still has the genotype AAB.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2673/F3.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2673/F3.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490502" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Identification of a fixed HeR event in Zhongyou821 using SNP array data and confirmed through whole genome re-sequencing. A) GenomeStudio images from A9; the SNP loci sampled at 1.9 Mb and 4.5 Mb show all testcross individuals (purple) in the AAB cluster biased toward the Zhongyou821 parent (red) and away from Rainbow (blue). Individuals in gray in the expected AB cluster are from other test cross populations without the genome rearrangement. Below the rearrangement (example at 11.3 Mb) normal SNP patterns were observed. B) Plot of normalized sequence read depth along the A9/C9 chromosomes from whole genome shotgun sequencing data of Zhongyou821. On A9 the first Mb is deleted and 2-8 Mb is duplicated, as opposed to C9 where the first Mb is duplicated and 2-11 Mb is deleted. C) GenomeStudio images from C9 showing no amplification of loci in Zhongyou821 (black) at 2.9 Mb and 9.0 Mb, while all testcross individuals (red) cluster with the Rainbow parent. At 18.1 Mb the parents are in the AA and BB clusters and the testcross lines (purple) are AB. The lone individual to the left is Zhongyou821 testcross line #26, which has an extra C9 chromosome and therefore still has the genotype AAB.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>De novo homoeologous recombination in B. napus</h3><p id="p-24">The focus of the study was to identify <em>de novo</em> rearrangements and all further discussion and calculations are based on new HeR exchanges. Such events are seen only in a single individual in a given testcross population and are presumed to have happened during the meiosis that produced the F<sub>1</sub> gamete. Similarly in cases where the same duplication or deletion event was seen in more than one individual within a population the event was assumed to have been present but not yet fixed (or functionally heterozygous) in the parental line and thus only inherited in a subset of the population and was not considered in subsequent analysis. Generally these events seemed to occur at similar levels to <em>de novo</em> events and testcross populations with high levels of <em>de novo</em> HeR also exhibited high levels of these segregating events (Tables S1-S10).</p><p id="p-25">The products of 508 meioses were examined and 36 <em>de novo</em> events were found where both the deletion and duplication of SNP loci in the corresponding homoeologous regions could be detected (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In 54 instances only deletion of linked loci was observed and similarly in 39 cases only duplicated linked loci were evident (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). These unpaired duplications and deletions were not unexpected since cytological observations of pairing control in newly resynthesized <em>B. napus</em> has frequently shown bivalents, multivalents and unpaired univalent chromosomes (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Attia and Röbbelen 1986b</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Szadkowski <em>et al.</em> 2010</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Rousseau-Gueutin <em>et al.</em> 2017</a>), so it was expected there could be allele gain and loss due to the complicated dissolution of multivalents, and the segregation of unpaired chromosomes. Although such events indicate aberrant pairing only the 28% of observed events with the reciprocal gain and loss of multiple physically linked loci could be reliably attributed to homoeologous recombination in the testcross parent(s). Of the 93 events for which only gain or loss was visible in the marker data, 27% could be attributed to aneuploid individuals with an additional (13) or missing (12) chromosome (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). As identified in previous studies (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Rousseau-Gueutin <em>et al.</em> 2017</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Stein <em>et al.</em> 2017</a>) deletion events are easily visible in the genotype data output from GenomeStudio, but duplication events less so. The current assay identifies individual testcross lines with a duplication based on their θ value separation from the rest of the population caused by a difference in relative fluorescence of the two SNP alleles (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B and C</a>). However, since the software can only call AA/AB/BB genotypes these individuals are either automatically called as AB or as a missing value. In the genotype output, multiple linked markers with mismatches and missing calls can indicate a potential duplication, but this must be validated by studying the individual SNP images making it more difficult to identify duplications, particularly very small ones. The smallest HeR event observed was 0.09 Mb, the smallest deletion was 0.04 Mb and the smallest duplication was 0.28 Mb (Table S11). The largest HeR was 40.6 Mb, the largest deletion was 28.6 Mb, and the largest duplication was 35.3 Mb. The size and position of all <em>de novo</em> HeR events, deletions and duplications observed in the testcross populations is summarized in Supplementary Table S11 and visualized in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490602/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490602/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of recombination events in <em>B. napus</em> testcross populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">As expected chromosome exchanges at the ends of chromosomes that require only one recombination (61 events) were more common than internal exchanges that require a second recombination (43 events). Many of the HeR events (36%) as seen for A3/C3, A4/C4, A5/C5, and A9/C8 were effectively terminal, involving gain/loss of the chromosome ends (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). It was also noted that 12 events spanned the centromere position (excluding aneuploids) and 25 events appeared to have breakpoints localized to the centromere position (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Though centromeric breakpoints are not likely due to meiotic recombination, they are presumably a consequence of pairing between homoeologues and subsequent breakage during anaphase I and are therefore still indicative of meiotic abnormalities. In total, including aneuploids, A genome chromosomes or regions were gained 50 times and lost 44 times, while the C genome was duplicated 25 times and lost 46 times. This discrepancy is largely due to the fact that in 66% of the HeR events the A genome chromosome was duplicated and the C genome homoeologue was lost (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490472/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490472/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of recombination events on each chromosome from the 10 <em>B. napus</em> testcross populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Seventeen of the HeR events with concomitant gain and loss of genetic material were between A1 and C1 (47%) though unpaired deletion and duplication events for A1 and C1 were not disproportional (5% and 4%, respectively) compared to other chromosomes (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The other 53% of HeR events were not distributed evenly across the chromosomes. A9 was involved in five events, three with C8 and two with C9, and though C9 did not recombine with its other potential pairing partner A10, this association was previously observed in allohaploid <em>B. napus</em> so pairing of these chromosomes is certainly possible (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grandont <em>et al.</em> 2014</a>). All chromosomes had at least one event, though C4 had no unpaired duplications or deletions but had three gain/loss HeR events and A6, A7, A8, A10, C6 and C7 had no gain/loss HeR. Curiously one Maris Haplona individual (#19) had inherited an extra copy of both A7 and C6 from Rainbow. Similar to the bias observed for the HeR events, missing or extra copies of A1/C1 and A2/C2 represented almost half (11/25) of the 25 aneuploid lines, for the reminder of the genome there were more, 9 compared to 5 aneuploid lines involving the A genome, of which 3 lines had an extra A8 chromosome.</p><p id="p-28">Though the <em>B. napus</em> lines used are established breeding lines they showed varying rates of <em>de novo</em> homoeologous recombinaton. Rainbow and PAK85912 had the highest rates with 22% of individuals having at least one deletion, duplication or gain/loss HeR and Surpass 400 had the lowest rate with no events in the 16 individuals tested. The ACSRsyn1 line was produced through interspecific crossing (see materials and methods for pedigree information) and was expected to have a high rate of homoeologous recombination, but only four events were observed in the 16 testcross lines, which is equivalent to the rates observed in Rainbow and PAK85912 and presumably results from stability selected over multiple generations.</p><p id="p-29">One of the advantages of using the SNP array to detect HeR events is the relatively dense marker coverage as compared to RFLP or SSR markers. Smaller exchanges can be identified and the marker density allows for more precise physical positioning of the breakpoints. Calculating the distance from the identified recombination event to the nearest segregating marker gives an indication of how the density of the markers is an improvement over older marker technologies. In this data set the smallest interval to which the recombination could be positioned was estimated to be 172 bp, while the largest was 8.7 Mb with an average size of 0.4 Mb (Table S11).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-30">Though <em>B. napus</em> is an excellent system for studying homoeologous recombination, relatively few studies have analyzed the prevalence of <em>de novo</em> events, in part because of the limits of molecular marker technology. However, the high-density array formats for SNP markers make them ideal for quantifying homologous and homoeologous recombination rates, and the depth of coverage of markers on the array helps to overcome the limitations of studying each genome independently.</p><p id="p-31">In the current study, <em>de novo</em> HeR was assayed in 508 meioses derived from 11 established <em>B. napus</em> lines, of these PAK85912 and Rainbow proved the most unstable. For each line, between 21–22% of the testcross individuals showed evidence of <em>de novo</em> segmental chromosome duplication or loss (excluding aneuploidy) presumed to be the result of homoeologous recombination or homoeologous associations. Previous studies of AC amphihaploids have suggested genotype contributes to the observed rates of HeR in <em>Brassica</em> species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Attia and Röbbelen 1986a</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Attia and Röbbelen 1986b</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jenczewski <em>et al.</em> 2003</a>). The study by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Udall <em>et al.</em> (2005)</a> examined three populations from natural <em>B. napus</em> crosses and found the rate of homoeologous recombination in one population was more than twice that of the other two (1.09%, 0.49% and 0.43% of all recombination events were homoeologous). In the 46 PAK85912 testcross lines there were five confirmed HeR events as well as five deletions and 4 duplications that are likely caused by chromosome mispairing, recombination and segregation. Rainbow had 30 HeR exchanges, 22 deletions and 12 duplications in the 254 individuals studied. In contrast, Zhongyou821 only had five duplication/deletion events with no HeR observed in 48 testcross lines. Considering all populations and the increased marker depth, the observed levels are comparable to those seen in a study using RFLP markers by <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Sharpe <em>et al.</em> (1995)</a>, where HeR were identified in four of 174 individuals from a <em>B. napus</em> DH population derived from a Canadian spring-type and a European winter line. Further studies will be required to assess the true range in absolute rates of <em>de novo</em> HeR in established <em>B. napus</em>; however, the cumulative evidence that genotype contributes to variation in HeR in established <em>B. napus</em> suggests the evolution of a genetic mechanism(s) to control levels of homoeologous recombination, possibly inherited from the progenitor diploids, and the variation between genotypes implies the quantitative nature of this trait.</p><p id="p-32">As observed in other studies of HeR, including fixed and <em>de novo</em> events, the genome of origin and chromosome position contribute significantly to prevalence and directionality of the exchange (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Xiong <em>et al.</em> 2011</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Chalhoub <em>et al.</em> 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">He <em>et al.</em> 2016</a>; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Rousseau-Gueutin <em>et al.</em> 2017</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Samans <em>et al.</em> 2017</a>). Although all chromosomes showed evidence of <em>de novo</em> events, 38.8% of all 129 <em>de novo</em> events were contributed by the two pairs of homoeologous chromosomes (A1/C1 and A2/C2), which are syntenic along their entire length, indicating the importance of homology in determining efficiency of chromosome exchange. The gain and loss of sub-telomeric regions of homoeologous chromosomes (A3/C3, A4/C4, A5/C5 and A9/C8) was also more widely prevalent, a phenomenon previously referred to as “homogenisation” (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Sharpe <em>et al.</em> 1995</a>) and corroborates previous work that showed an increase in events with distance from the centromere (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nicolas <em>et al.</em> 2012</a>). However, it was also noted that 25 events were co-positioned with centromere locations (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). Centromeres tend to form breaks in ancestral karyotype blocks (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Parkin <em>et al.</em> 2005</a>) suggesting they may act as evolutionary breakpoints and chromosome fusion and fission has almost certainly played a significant role in shaping Brassica genomes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Friebe <em>et al.</em> 2005</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Schranz <em>et al.</em> 2006</a>).</p><p id="p-33">The A genome dominated HeR events such that it replaced the C genome in 66% of events, this bias held with segmental duplications, with 17 of the 26 duplications involving A genome chromosomes. In contrast to <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Samans <em>et al.</em> (2017)</a>, <em>de novo</em> segmental deletions were found to be higher in the A genome, 27 of the 42 deletions were from the A genome, though the chromosomal material lost was 120 Mb as compared to the C genome, which lost 149 Mb through deletions. This could represent ascertainment bias from the SNP array since proportionally there are a lower number of informative markers per Mb derived from the C genome making it more difficult to detect small deletions (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2016</a>). It was proposed in <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Samans <em>et al.</em> (2017)</a> that since the A genome is ∼25% proportionally smaller than the C genome that the bias toward overall loss of the C genome supports observations of genome size reduction in neo-polyploid evolution, and this was reflected in cumulative amounts of genetic material lost and gained between the sub-genomes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">He <em>et al.</em> (2016)</a> also evoked the known prevalence of interspecific crossing with <em>B. rapa</em> in <em>B. napus</em> breeding strategies, which could have led to biased capture of A genome regions by lone C genome chromosomes forming aberrant pairing structures in AAC triploids. One intriguing possibility for the apparent instability of the C genome in <em>B. napus</em> is the suggestion that paternal genomes are disproportionally affected in neopolyploids (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lim <em>et al.</em> 2007</a>), which would align with suggestions that a relative of <em>B. rapa</em> was the possible maternal progenitor of <em>B. napus</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allender and King 2010</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of genomic loss and gain (in Mb) due to deletion, duplication and HeR events for each chromosome. Gain and loss of genetic material due to HeR is shown in red and blue, respectively. Segmental deletions and duplications are shown in green and purple, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887735115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of genomic loss and gain (in Mb) due to deletion, duplication and HeR events for each chromosome. Gain and loss of genetic material due to HeR is shown in red and blue, respectively. Segmental deletions and duplications are shown in green and purple, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2673/F4.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2673/F4.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2673/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Summary of genomic loss and gain (in Mb) due to deletion, duplication and HeR events for each chromosome. Gain and loss of genetic material due to HeR is shown in red and blue, respectively. Segmental deletions and duplications are shown in green and purple, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Twenty-five aneuploid events were found among the 508 meioses, interestingly the A genome once more featured more dominantly than the C genome, with 14 of the events involving seven of the ten A genome chromosomes, and nine of which resulted in chromosome additions. As with other events A1/C1 and A2/C2 accounted for almost 50% of the events; such aneuploids were prevalent in early generations of resynthesized <em>B. napus</em> and chromosome compensation between the highly syntenic chromosomes in the polyploid nucleus is proposed to provide genome balance in the monosomic lines (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Xiong <em>et al.</em> 2011</a>). Although aneuploids are commonly produced in neopolyploids (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Comai 2005</a>) and allopolyploids are known for their genome plasticity (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Leitch and Leitch 2008</a>) the level observed (∼5% across all gametes) in established <em>B. napus</em> might appear high. However, none of the observed events including the aneuploids are fixed in the testcross lines, and it would be expected that continual selection, for example in the field, for highly fertile euploids would invariably negate the presence of aneuploids and other rearrangements unless they led to a selective advantage. It would be interesting to study the effect of different parental genotypes in the development of hybrids, since it might be expected that high levels of meiotic instability would impact hybrid vigor.</p><p id="p-36">The importance of homoeologous recombination in the developmental history of modern <em>B. napus</em> and its establishment as a major oilseed crop became apparent through sequence analyses of the <em>B. napus</em> genome that showed a number of important traits for Brassica oil and meal quality were derived from homoeologous recombination events (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Chalhoub <em>et al.</em> 2014</a>). An indication of continuing genome evolution in <em>B. napus</em> was shown when a homoeologous recombination event was found to be segregating in replicate lines from the variety used to generate the <em>B. napus</em> genome assembly (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lloyd <em>et al.</em> 2017</a>). The current study has shown that these events continue to happen frequently in natural <em>B. napus</em>, offering the opportunity to generate novel variation that could be exploited for crop improvement. The use of high-density SNP arrays has become the standard method for genetic mapping in segregating populations, genotyping of elite lines, studies of genotypic diversity, and identification of chromosomal deletions (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hatzig <em>et al.</em> 2015</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Körber <em>et al.</em> 2015</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mason <em>et al.</em> 2015</a>; <a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Rousseau-Gueutin <em>et al.</em> 2017</a>; <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Stein <em>et al.</em> 2017</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Yang <em>et al.</em> 2017</a>). The method described in this paper extends the use of SNP array to comprehensively measure the inheritance of homoeologous recombination events. The coverage of the Brassica 60K SNP array makes selection across the whole of the <em>B. napus</em> genome possible; in combination with cytological analysis this will present a complete picture of chromosome pairing at meiosis and the resulting homoeologous recombination that affects gamete viability, phenotypic variation and plant fitness. The SNP coverage in concert with the available <em>B. napus</em> genome sequence (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Chalhoub <em>et al.</em> 2014</a>) also allows for a precise identification of recombination hotspots and regions of recombination repression.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">This work was funded through the Agriculture and Agri-Food Canada Canadian Crop Genomics Initiative. SJA and ECH also acknowledge funding by BBSRC (grant award no. BB/1007633/1; Control of chromosomal meiotic pairing in the allopolyploid species, oilseed rape). The authors would like to thank Chu Shin Koh (Global Institute for Food Security, Saskatoon) for providing a config file for developing the Circos plot.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-38">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6207530">https://doi.org/10.25387/g3.6207530</a>.</p></li><li class="fn" id="fn-2"><p id="p-39"><em>Communicating editor: J. Wendel</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-02"><span class="received-label">Received </span>February 2, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-12"><span class="accepted-label">Accepted </span>June 12, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Higgins <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.1" data-doi="10.1186/1471-2229-10-54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allender</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Origins of the amphiploid species <em>Brassica napus L</em>. investigated by chloroplast and nuclear molecular markers.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">54</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-10-54</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DAllender%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D10%26rft.spage%253D54%26rft.epage%253D54%26rft.atitle%253DOrigins%2Bof%2Bthe%2Bamphiploid%2Bspecies%2BBrassica%2Bnapus%2BL.%2Binvestigated%2Bby%2Bchloroplast%2Band%2Bnuclear%2Bmolecular%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-10-54%26rft_id%253Dinfo%253Apmid%252F20350303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-10-54&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20350303&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.2" data-doi="10.1139/g86-048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röbbelen</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span><span class="cit-comment">a</span> <span class="cit-article-title">Cytogenetic relationship within cultivated Brassica analyzed in amphihaploids from the three diploid ancestors.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g86-048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BGenet.%2BCytol.%26rft.volume%253D28%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg86-048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g86-048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.3" data-doi="10.1139/g86-049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röbbelen</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span><span class="cit-comment">b</span> <span class="cit-article-title">Meiotic pairing in haploids and amphidiploids of spontaneous <em>vs.</em> synthetic origin in rape, <em>Brassica napus</em> L.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g86-049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BGenet.%2BCytol.%26rft.volume%253D28%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg86-049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g86-049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986D068600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.4" data-doi="10.1073/pnas.1416241111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhullar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennypaul</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidhu</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Silencing of a metaphase I-specific gene results in a phenotype similar to that of the Pairing homeologous 1 (Ph1) gene mutations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">14187</span>–<span class="cit-lpage">14192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1416241111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1416241111%26rft_id%253Dinfo%253Apmid%252F25232038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM5LzE0MTg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.6" data-doi="10.1038/nature01521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBowers%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D422%26rft.issue%253D6930%26rft.spage%253D433%26rft.epage%253D438%26rft.atitle%253DUnravelling%2Bangiosperm%2Bgenome%2Bevolution%2Bby%2Bphylogenetic%2Banalysis%2Bof%2Bchromosomal%2Bduplication%2Bevents.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01521%26rft_id%253Dinfo%253Apmid%252F12660784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12660784&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181801200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.7" data-doi="10.1007/s00122-011-1676-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korber</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patterns of molecular variation in a species-wide germplasm set of <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1413</span>–<span class="cit-lpage">1423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1676-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBus%26rft.auinit1%253DA.%26rft.volume%253D123%26rft.issue%253D8%26rft.spage%253D1413%26rft.epage%253D1423%26rft.atitle%253DPatterns%2Bof%2Bmolecular%2Bvariation%2Bin%2Ba%2Bspecies-wide%2Bgermplasm%2Bset%2Bof%2BBrassica%2Bnapus.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1676-7%26rft_id%253Dinfo%253Apmid%252F21847624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1676-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21847624&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.8" data-doi="10.1126/science.1253435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalhoub</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Plant genetics. Early allopolyploid evolution in the post-Neolithic <em>Brassica napus</em> oilseed genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">950</span>–<span class="cit-lpage">953</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1253435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChalhoub%26rft.auinit1%253DB.%26rft.volume%253D345%26rft.issue%253D6199%26rft.spage%253D950%26rft.epage%253D953%26rft.atitle%253DEarly%2Ballopolyploid%2Bevolution%2Bin%2Bthe%2Bpost-Neolithic%2BBrassica%2Bnapus%2Boilseed%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1253435%26rft_id%253Dinfo%253Apmid%252F25146293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDUvNjE5OS85NTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.9" data-doi="10.1111/j.1469-8137.2009.03084.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cifuentes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandont</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chèvre</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenczewski</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic regulation of meiosis in polyploid species: New insights into an old question.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2009.03084.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DCifuentes%26rft.auinit1%253DM.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D36%26rft.atitle%253DGenetic%2Bregulation%2Bof%2Bmeiosis%2Bin%2Bpolyploid%2Bspecies%253A%2Bnew%2Binsights%2Binto%2Ban%2Bold%2Bquestion.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03084.x%26rft_id%253Dinfo%253Apmid%252F19912546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03084.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19912546&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.10" data-doi="10.1007/s00122-016-2746-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieseke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidebottom</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A high-density SNP genotyping array for <em>Brassica napus</em> and its ancestral diploid species based on optimised selection of single-locus markers in the allotetraploid genome.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1887</span>–<span class="cit-lpage">1899</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2746-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1887%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2746-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2746-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.11" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.12" data-doi="10.1146/annurev.genet.42.110807.091524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolutionary genetics of genome merger and doubling in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.42.110807.091524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DDoyle%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D42%26rft.spage%253D443%26rft.epage%253D461%26rft.atitle%253DEvolutionary%2Bgenetics%2Bof%2Bgenome%2Bmerger%2Band%2Bdoubling%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091524%26rft_id%253Dinfo%253Apmid%252F18983261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983261&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.13" data-doi="10.1159/000082412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Robertsonian translocations in wheat arise by centric misdivision of univalents at anaphase I and rejoining of broken centromeres during interkinesis of meiosis II.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">297</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000082412</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DFriebe%26rft.auinit1%253DB.%26rft.volume%253D109%26rft.issue%253D1-3%26rft.spage%253D293%26rft.epage%253D297%26rft.atitle%253DRobertsonian%2Btranslocations%2Bin%2Bwheat%2Barise%2Bby%2Bcentric%2Bmisdivision%2Bof%2Bunivalents%2Bat%2Banaphase%2BI%2Band%2Brejoining%2Bof%2Bbroken%2Bcentromeres%2Bduring%2Binterkinesis%2Bof%2Bmeiosis%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000082412%26rft_id%253Dinfo%253Apmid%252F15753589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000082412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15753589&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.14" data-doi="10.1105/tpc.114.122788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandont</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuñado</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coriton</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huteau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Homoeologous Chromosome Sorting and Progression of Meiotic Recombination in Brassica napus: Ploidy Does Matter!</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1448</span>–<span class="cit-lpage">1463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.122788</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.122788%26rft_id%253Dinfo%253Apmid%252F24737673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzQvMTQ0OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.15" data-doi="10.1105/tpc.111.094771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pendle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colas</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Ph1 locus suppresses Cdk2-type activity during premeiosis and meiosis in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">152</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.111.094771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.111.094771%26rft_id%253Dinfo%253Apmid%252F22274628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI0LzEvMTUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.16" data-doi="10.3389/fpls.2015.00221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hatzig</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breuer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducournau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association mapping unravels the genetic control of seed germination and vigor in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.00221</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.00221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.00221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.17" data-doi="10.1111/pbi.12657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper Andrea</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havlickova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pradhan Akshay</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extensive homoeologous genome exchanges in allopolyploid crops revealed by mRNAseq‐based visualization.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">594</span>–<span class="cit-lpage">604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D594%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.18" data-doi="10.1534/genetics.108.095893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearsey</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A and C genome distinction and chromosome identification in <em>Brassica napus</em> by sequential fluorescence in situ hybridization and genomic in situ hybridization.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1849</span>–<span class="cit-lpage">1857</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.095893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.095893%26rft_id%253Dinfo%253Apmid%252F18845839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzE4NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.19" data-doi="10.1080/07352680490273239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenczewski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alix</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">From Diploids to Allopolyploids: The Emergence of Efficient Pairing Control Genes in Plants.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07352680490273239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D23%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352680490273239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352680490273239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220026200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenczewski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimaud</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">PrBn, a major gene controlling homeologous pairing in oilseed rape (<em>Brassica napus</em>) haploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJenczewski%26rft.auinit1%253DE.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D645%26rft.epage%253D653%26rft.atitle%253DPrBn%252C%2Ba%2BMajor%2BGene%2BControlling%2BHomeologous%2BPairing%2Bin%2BOilseed%2BRape%2B%2528Brassica%2Bnapus%2529%2BHaploids%26rft_id%253Dinfo%253Apmid%252F12807785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.21" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT—The BLAST-Like Alignment Tool.</span> <abbr class="cit-jnl-abbrev">Genome Research</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-fpage">664</span>. </cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.22" data-doi="10.1186/s12870-015-0496-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Körber</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bancroft</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Seedling development traits in <em>Brassica napus</em> examined by gene expression analysis and association mapping.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-015-0496-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D15%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-015-0496-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-015-0496-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.23" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascoyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: An information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.092759.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.24" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.25" data-doi="10.1534/g3.116.030858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lashermes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hueber</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Combes</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severac</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dereeper</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-comment">Inter-genomic DNA Exchanges and Homeologous Gene Silencing Shaped the Nascent Allopolyploid Coffee Genome (<em>Coffea arabica</em> L.). G3: Genes|Genomes|Genetics 6: 2937</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.030858</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.030858%26rft_id%253Dinfo%253Apmid%252F27440920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yOTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.26" data-doi="10.1126/science.1153585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomic Plasticity and the Diversity of Polyploid Plants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1153585</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLeitch%26rft.auinit1%253DA.%2BR.%26rft.volume%253D320%26rft.issue%253D5875%26rft.spage%253D481%26rft.epage%253D483%26rft.atitle%253DGenomic%2BPlasticity%2Band%2Bthe%2BDiversity%2Bof%2BPolyploid%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153585%26rft_id%253Dinfo%253Apmid%252F18436776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NS80ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.27" data-doi="10.1111/j.1469-8137.2007.02121.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovarik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matyasek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chase Mark</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarkson James</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sequence of events leading to near‐complete genome turnover in allopolyploid <em>Nicotiana</em> within five million years.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2007.02121.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DLim%26rft.auinit1%253DK.%2BY.%26rft.volume%253D175%26rft.issue%253D4%26rft.spage%253D756%26rft.epage%253D763%26rft.atitle%253DSequence%2Bof%2Bevents%2Bleading%2Bto%2Bnear-complete%2Bgenome%2Bturnover%2Bin%2Ballopolyploid%2BNicotiana%2Bwithin%2Bfive%2Bmillion%2Byears.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2007.02121.x%26rft_id%253Dinfo%253Apmid%252F17688590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2007.02121.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17688590&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249098400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.28" data-doi="10.1371/journal.pone.0083052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkop</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A high-density SNP map for accurate mapping of seed fibre QTL in <em>Brassica napus</em> L.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e83052</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0083052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De83052%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0083052%26rft_id%253Dinfo%253Apmid%252F24386142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0083052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24386142&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.29" data-doi="10.1111/nph.14836"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charif</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Homoeologous exchanges cause extensive dosage‐dependent gene expression changes in an allopolyploid crop.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">217</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14836</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D217%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14836&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.30" data-doi="10.1101/gr.3531105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pecinka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Chromosome triplication found across the tribe Brassiceae.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3531105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3531105%26rft_id%253Dinfo%253Apmid%252F15781573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzQvNTE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.31" data-doi="10.1007/s00412-017-0630-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martín</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">M.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dual effect of the wheat <em>Ph1</em> locus on chromosome synapsis and crossover.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">680</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-017-0630-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D126%26rft.spage%253D669%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-017-0630-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-017-0630-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.32" data-doi="10.1111/nph.12706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowling</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution molecular karyotyping uncovers pairing between ancestrally related Brassica chromosomes.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">964</span>–<span class="cit-lpage">974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12706</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D202%26rft.spage%253D964%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12706%26rft_id%253Dinfo%253Apmid%252F24471809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24471809&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.33" data-doi="10.1007/s00122-016-2849-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A user guide to the Brassica 60K Illumina Infinium SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">633</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2849-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D621%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2849-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2849-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.34" data-doi="10.1111/1755-0998.12379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollenaere</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasquez Teuber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalton-Morgan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-throughput genotyping for species identification and diversity assessment in germplasm collections.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D1091%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.35" data-doi="10.1126/science.264.5157.421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masterson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Stomatal size in fossil plants: evidence for polyploidy in majority of angiosperms.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">424</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.264.5157.421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMasterson%26rft.auinit1%253DJ.%26rft.volume%253D264%26rft.issue%253D5157%26rft.spage%253D421%26rft.epage%253D424%26rft.atitle%253DStomatal%2BSize%2Bin%2BFossil%2BPlants%253A%2BEvidence%2Bfor%2BPolyploidy%2Bin%2BMajority%2Bof%2BAngiosperms%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.264.5157.421%26rft_id%253Dinfo%253Apmid%252F17836906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjQvNTE1Ny80MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2673.36" data-doi="10.1093/nar/8.19.4321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Rapid isolation of high molecular weight plant DNA.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4321</span>–<span class="cit-lpage">4326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMurray%26rft.auinit1%253DM.G.%26rft.volume%253D8%26rft.issue%253D19%26rft.spage%253D4321%26rft.epage%253D4326%26rft.atitle%253DRapid%2Bisolation%2Bof%2Bhigh%2Bmolecular%2Bweight%2Bplant%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.19.4321%26rft_id%253Dinfo%253Apmid%252F7433111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/8.19.4321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7433111&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KL92600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.37" data-doi="10.3389/fpls.2016.01635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguepjop</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tossim</span>  <span class="cit-name-given-names">H.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evidence of Genomic Exchanges between Homeologous Chromosomes in a Cross of Peanut with Newly Synthetized Allotetraploid Hybrids.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.01635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D1635%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.01635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.01635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.38" data-doi="10.1105/tpc.108.062273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leflon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coriton</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic Regulation of Meiotic Cross-Overs between Related Genomes in <em>Brassica napus</em> Haploids and Hybrids.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.108.062273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.062273%26rft_id%253Dinfo%253Apmid%252F19190241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMzczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.39" data-doi="10.1111/j.1365-313X.2012.04914.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chèvre</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenczewski</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Non‐random distribution of extensive chromosome rearrangements in <em>Brassica napus</em> depends on genome organization.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2012.04914.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DNicolas%26rft.auinit1%253DS.%2BD.%26rft.volume%253D70%26rft.issue%253D4%26rft.spage%253D691%26rft.epage%253D703%26rft.atitle%253DNon-random%2Bdistribution%2Bof%2Bextensive%2Bchromosome%2Brearrangements%2Bin%2BBrassica%2Bnapus%2Bdepends%2Bon%2Bgenome%2Borganization.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2012.04914.x%26rft_id%253Dinfo%253Apmid%252F22268419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2012.04914.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22268419&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303658200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.40" data-doi="10.1186/gb-2014-15-6-r77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kagale</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid <em>Brassica oleracea</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R77</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2014-15-6-r77</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR77%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-6-r77%26rft_id%253Dinfo%253Apmid%252F24916971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-6-r77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24916971&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.41" data-doi="10.1139/g95-149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydiate</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Identification of the A and C genomes of amphidiploid <em>Brassica napus</em> (oilseed rape).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1122</span>–<span class="cit-lpage">1131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g95-149</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DParkin%26rft.auinit1%253DI.%2BA.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D1122%26rft.epage%253D1131%26rft.atitle%253DIdentification%2Bof%2Bthe%2BA%2Band%2BC%2Bgenomes%2Bof%2Bamphidiploid%2BBrassica%2Bnapus%2B%2528oilseed%2Brape%2529.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg95-149%26rft_id%253Dinfo%253Apmid%252F18470236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g95-149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18470236&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.42" data-doi="10.1534/genetics.105.042093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gulden</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Segmental structure of the Brassica napus genome based on comparative analysis with Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">781</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.042093</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.042093%26rft_id%253Dinfo%253Apmid%252F16020789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvNzY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.43" data-doi="10.1007/s11032-017-0700-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">M.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martín</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarbreck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uauy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Exploiting the <em>ZIP4</em> homologue within the wheat <em>Ph1</em> locus has identified two lines exhibiting homoeologous crossover in wheat-wild relative hybrids.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-017-0700-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D37%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-017-0700-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-017-0700-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.44" data-doi="10.1038/182713a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Genetic control of the cytologically diploid behaviour of hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">715</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/182713a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D182%26rft.spage%253D713%26rft_id%253Dinfo%253Adoi%252F10.1038%252F182713a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/182713a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.45" data-doi="10.1534/g3.116.036517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousseau-Gueutin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morice</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coriton</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huteau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trotoux</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-comment">The Impact of Open Pollination on the Structural Evolutionary Dynamics, Meiotic Behavior, and Fertility of Resynthesized Allotetraploid <em>Brassica napus</em> L. G3: Genes|Genomes|Genetics 7: 705</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036517</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036517%26rft_id%253Dinfo%253Apmid%252F28007837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi83MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.46" data-doi="10.3835/plantgenome2017.02.0013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalhoub</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Surviving a Genome Collision: Genomic Signatures of Allopolyploidization in the Recent Crop Species <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.02.0013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.02.0013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.02.0013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.47" data-doi="10.1016/j.tplants.2006.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schranz</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The ABC’s of comparative genomics in the Brassicaceae: building blocks of crucifer genomes.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2006.09.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DSchranz%26rft.auinit1%253DM.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D535%26rft.epage%253D542%26rft.atitle%253DThe%2BABC%2527s%2Bof%2Bcomparative%2Bgenomics%2Bin%2Bthe%2BBrassicaceae%253A%2Bbuilding%2Bblocks%2Bof%2Bcrucifer%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2006.09.002%26rft_id%253Dinfo%253Apmid%252F17029932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2006.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17029932&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242317100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.48" data-doi="10.1139/g95-148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydiate</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Frequent nonreciprocal translocations in the amphidiploid genome of oilseed rape (<em>Brassica napus</em>).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g95-148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSharpe%26rft.auinit1%253DA.%2BG.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D1112%26rft.epage%253D1121%26rft.atitle%253DFrequent%2Bnonreciprocal%2Btranslocations%2Bin%2Bthe%2Bamphidiploid%2Bgenome%2Bof%2Boilseed%2Brape%2B%2528Brassica%2Bnapus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg95-148%26rft_id%253Dinfo%253Apmid%252F18470235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g95-148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18470235&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.49" data-doi="10.1007/s10577-006-1105-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cytogenetics and genome analysis in Brassica crops.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-006-1105-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DSnowdon%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D95%26rft.atitle%253DCytogenetics%2Band%2Bgenome%2Banalysis%2Bin%2BBrassica%2Bcrops.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-006-1105-y%26rft_id%253Dinfo%253Apmid%252F17295129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-006-1105-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17295129&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244191500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.50" data-doi="10.1111/pbi.12732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coriton</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousseau-Gueutin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiessl</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mapping of homoeologous chromosome exchanges influencing quantitative trait variation in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1478</span>–<span class="cit-lpage">1489</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12732</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D1478%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.51" data-doi="10.1111/j.1469-8137.2010.03182.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szadkowski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huteau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lode</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huneau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The first meiosis of resynthesized <em>Brassica napus</em>, a genome blender.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2010.03182.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSzadkowski%26rft.auinit1%253DE.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D102%26rft.epage%253D112%26rft.atitle%253DThe%2Bfirst%2Bmeiosis%2Bof%2Bresynthesized%2BBrassica%2Bnapus%252C%2Ba%2Bgenome%2Bblender.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03182.x%26rft_id%253Dinfo%253Apmid%252F20149113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03182.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20149113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.52" data-doi="10.1534/genetics.104.033209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quijada</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detection of chromosomal rearrangements derived from homologous recombination in four mapping populations of <em>Brassica napus L</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">979</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.033209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033209%26rft_id%253Dinfo%253Apmid%252F15520255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzIvOTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.53"><div class="cit-metadata"><cite><span class="cit-comment">U, N., 1935 Genome analysis in Brassica with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization. Japanese J. of Botany</span> <span class="cit-vol">7</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.54" data-doi="10.1038/ng.919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D1035%26rft.epage%253D1039%26rft.atitle%253DThe%2Bgenome%2Bof%2Bthe%2Bmesopolyploid%2Bcrop%2Bspecies%2BBrassica%2Brapa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.919%26rft_id%253Dinfo%253Apmid%252F21873998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21873998&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.55" data-doi="10.3389/fpls.2015.01164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Density SNP Map Construction and QTL Identification for the Apetalous Character in <em>Brassica napus L</em>.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.01164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D1164%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.01164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.01164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.56" data-doi="10.1023/A:1006392424384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome evolution in polyploids.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1006392424384</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DWendel%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D249%26rft.atitle%253DGenome%2Bevolution%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1006392424384%26rft_id%253Dinfo%253Apmid%252F10688139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1006392424384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10688139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084535800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.57" data-doi="10.1073/pnas.1014138108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaeta</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Homoeologous shuffling and chromosome compensation maintain genome balance in resynthesized allopolyploid <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7908</span>–<span class="cit-lpage">7913</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1014138108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1014138108%26rft_id%253Dinfo%253Apmid%252F21512129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE5Lzc5MDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.58" data-doi="10.1534/genetics.110.122473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Karyotype and identification of all homoeologous chromosomes of allopolyploid <em>Brassica napus</em> and its diploid progenitors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.122473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.122473%26rft_id%253Dinfo%253Apmid%252F21041557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg3LzEvMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.59" data-doi="10.3389/fpls.2017.01512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High Density Linkage Map Construction and QTL Detection for Three Silique-Related Traits in <em>Orychophragmus violaceus</em> Derived <em>Brassica napus</em> Population.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1512%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2673.60" data-doi="10.3389/fpls.2014.00279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zamariola</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiang</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Storme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawlowski</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geelen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Chromosome segregation in plant meiosis.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2014.00279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2014.00279%26rft_id%253Dinfo%253Apmid%252F24987397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2014.00279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24987397&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2673.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2663" title="Idiopathic Scoliosis Families Highlight Actin-Based and Microtubule-Based Cellular Projections and Extracellular Matrix in Disease Etiology" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2685" title="Genetic Analysis of Signal Generation by the Rgt2 Glucose Sensor of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2673" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dWr68wbzG8XyNhIoqME3dOY2Zskav_q6sara1hRn-Ww" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2673" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2673" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2673" class="active" data-icon-position="" data-hide-link-title="0">Detecting de Novo Homoeologous Recombination Events in Cultivated Brassica napus Using a Genome-Wide SNP Array</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490452" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UGb1tFv6JTjR6e8b-oi5XZfwPUejcNm-mmwJWwM43TU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2673.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2673" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EriXS-PCjyL9q46aVg2hi_J2NeI7isBzoVSh6B9lx0E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2673" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2673.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490452" id="node490452--2" data-pisa="ggg;8/8/2673" data-pisa-master="ggg;g3.118.200118" data-apath="/ggg/8/8/2673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting <em>de Novo</em> Homoeologous Recombination Events in Cultivated <em>Brassica napus</em> Using a Genome-Wide SNP Array</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erin E. Higgins</span>, <span class="highwire-citation-author" data-delta="1">Wayne E. Clarke</span>, <span class="highwire-citation-author" data-delta="2">Elaine C. Howell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7284-5702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan J. Armstrong</span> and <span class="highwire-citation-author" data-delta="4">Isobel A. P. Parkin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2673-2683;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200118 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490452--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erin E. Higgins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BE.%2BHiggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Higgins%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BE.%2BHiggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wayne E. Clarke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wayne%2BE.%2BClarke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clarke%20WE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWayne%2BE.%2BClarke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elaine C. Howell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elaine%2BC.%2BHowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howell%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElaine%2BC.%2BHowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Susan J. Armstrong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BJ.%2BArmstrong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armstrong%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BJ.%2BArmstrong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7284-5702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan J. Armstrong</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Isobel A. P. Parkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isobel%2BA.%2BP.%2BParkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parkin%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsobel%2BA.%2BP.%2BParkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:isobel.parkin@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">isobel.parkin@agr.gc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2673" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2673&amp;title=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2673&amp;title=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2673&amp;title=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2673&amp;text=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2673&amp;title=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2673&amp;t=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2673&amp;title=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2673&amp;title=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490452/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490452" id="node490452--3" data-pisa="ggg;8/8/2673" data-pisa-master="ggg;g3.118.200118" data-apath="/ggg/8/8/2673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting <em>de Novo</em> Homoeologous Recombination Events in Cultivated <em>Brassica napus</em> Using a Genome-Wide SNP Array</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erin E. Higgins</span>, <span class="highwire-citation-author" data-delta="1">Wayne E. Clarke</span>, <span class="highwire-citation-author" data-delta="2">Elaine C. Howell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7284-5702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan J. Armstrong</span> and <span class="highwire-citation-author" data-delta="4">Isobel A. P. Parkin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2673-2683;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200118 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490452--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erin E. Higgins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BE.%2BHiggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Higgins%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BE.%2BHiggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wayne E. Clarke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wayne%2BE.%2BClarke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clarke%20WE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWayne%2BE.%2BClarke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elaine C. Howell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elaine%2BC.%2BHowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howell%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElaine%2BC.%2BHowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Susan J. Armstrong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biosciences, University of Birmingham, Edgbaston, Birmingham B15 2TT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BJ.%2BArmstrong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armstrong%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BJ.%2BArmstrong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7284-5702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan J. Armstrong</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Isobel A. P. Parkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Agriculture and Agri-Food Canada, 107 Science Place, Saskatoon, SK, S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isobel%2BA.%2BP.%2BParkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parkin%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsobel%2BA.%2BP.%2BParkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:isobel.parkin@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">isobel.parkin@agr.gc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2673&amp;rft.epage=2683&amp;rft.atitle=Detecting%20de%20Novo%20Homoeologous%20Recombination%20Events%20in%20Cultivated%20Brassica%20napus%20Using%20a%20Genome-Wide%20SNP%20Array&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Higgins%2C+Erin+E.&amp;rft.au=Clarke%2C+Wayne+E.&amp;rft.au=Howell%2C+Elaine+C.&amp;rft.au=Armstrong%2C+Susan+J.&amp;rft.au=Parkin%2C+Isobel+A.+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490452/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490452/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490452/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490452/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490452/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490452/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490452/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490452/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490452/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490452/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490452/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490452/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2673" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2673.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2673.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2673.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2673.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
