<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2805" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2805.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2805.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2805.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/8/2805/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2805.atom" />
<meta name="HW.pisa" content="ggg;8/8/2805" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes" />
<meta name="DC.Identifier" content="10.1534/g3.118.200367" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Brandt et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Balancing selection is defined as a class of selective regimes that maintain polymorphism above what is expected under neutrality. Theory predicts that balancing selection reduces population differentiation, as measured by FST. However, balancing selection regimes in which different sets of alleles are maintained in different populations could increase population differentiation. To tackle the connection between balancing selection and population differentiation, we investigated population differentiation at the HLA genes, which constitute the most striking example of balancing selection in humans. We found that population differentiation of single nucleotide polymorphisms (SNPs) at the HLA genes is on average lower than that of SNPs in other genomic regions. We show that these results require using a computation that accounts for the dependence of FST on allele frequencies. However, in pairs of closely related populations, where genome-wide differentiation is low, differentiation at HLA is higher than in other genomic regions. Such increased population differentiation at HLA genes for recently diverged population pairs was reproduced in simulations of overdominant selection, as long as the fitness of the homozygotes differs between the diverging populations. The results give insight into a possible “divergent overdominance” mechanism for the nature of balancing selection on HLA genes across human populations." />
<meta name="DC.Contributor" content="Débora Y. C. Brandt" />
<meta name="DC.Contributor" content="Jônatas César" />
<meta name="DC.Contributor" content="Jérôme Goudet" />
<meta name="DC.Contributor" content="Diogo Meyer" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Balancing selection is defined as a class of selective regimes that maintain polymorphism above what is expected under neutrality. Theory predicts that balancing selection reduces population differentiation, as measured by FST. However, balancing selection regimes in which different sets of alleles are maintained in different populations could increase population differentiation. To tackle the connection between balancing selection and population differentiation, we investigated population differentiation at the HLA genes, which constitute the most striking example of balancing selection in humans. We found that population differentiation of single nucleotide polymorphisms (SNPs) at the HLA genes is on average lower than that of SNPs in other genomic regions. We show that these results require using a computation that accounts for the dependence of FST on allele frequencies. However, in pairs of closely related populations, where genome-wide differentiation is low, differentiation at HLA is higher than in other genomic regions. Such increased population differentiation at HLA genes for recently diverged population pairs was reproduced in simulations of overdominant selection, as long as the fitness of the homozygotes differs between the diverging populations. The results give insight into a possible “divergent overdominance” mechanism for the nature of balancing selection on HLA genes across human populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2805" />
<meta name="citation_id" content="8/8/2805" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2805" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2805.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2805.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2805.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200367" />
<meta name="citation_pmid" content="29950428" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2805" />
<meta name="citation_lastpage" content="2815" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Débora Y. C. Brandt" />
<meta name="citation_author_institution" content="Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil" />
<meta name="citation_author" content="Jônatas César" />
<meta name="citation_author_institution" content="Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil" />
<meta name="citation_author" content="Jérôme Goudet" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, University of Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_email" content="diogo@ib.usp.br" />
<meta name="citation_author_email" content="jerome.goudet@unil.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5318-7601" />
<meta name="citation_author" content="Diogo Meyer" />
<meta name="citation_author_institution" content="Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil" />
<meta name="citation_author_email" content="diogo@ib.usp.br" />
<meta name="citation_author_email" content="jerome.goudet@unil.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7155-5674" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Alcala;citation_author=N. A. Rosenberg;citation_title=Mathematical constraints on F ST: biallelic markers in arbitrarily many populations.;citation_volume=206;citation_year=2017;citation_pmid=28476869;citation_doi=10.1534/genetics.116.199141" />
<meta name="citation_reference" content="citation_title=Encyclopedia of Life Sciences;citation_author=A. M. Andrés;citation_title=Balancing Selection in the Human Genome;citation_year=2011;citation_doi=10.1002/9780470015902.a0022863" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. B. Barreiro;citation_author=G. Laval;citation_author=H. Quach;citation_author=E. Patin;citation_author=L. Quintana-Murci;citation_title=Natural selection has driven population differentiation in modern humans.;citation_pages=340-345;citation_volume=40;citation_year=2008;citation_issue=3;citation_pmid=18246066;citation_doi=10.1038/ng.78" />
<meta name="citation_reference" content="citation_journal_title=Tissue antigens;citation_journal_abbrev=Tissue Antigens;citation_author=A. B. Begovich;citation_author=P. V. Moonsamy;citation_author=S. J. Mack;citation_author=L. F. Barcellos;citation_author=L. L. Steiner;citation_title=Genetic variability and linkage disequilibrium within the HLA-DP region: analysis of 15 different populations.;citation_pages=424-439;citation_volume=57;citation_year=2001;citation_issue=5;citation_pmid=11556967;citation_doi=10.1034/j.1399-0039.2001.057005424.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Bersaglieri;citation_author=P. C. Sabeti;citation_author=N. J. Patterson;citation_author=T. Vanderploeg;citation_author=S. F. Schaffner;citation_title=Genetic signatures of strong recent positive selection at the lactase gene.;citation_pages=1111-1120;citation_volume=74;citation_year=2004;citation_issue=6;citation_pmid=15114531;citation_doi=10.1086/421051" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Bhatia;citation_author=N. Patterson;citation_author=S. Sankararaman;citation_author=A. L. Price;citation_title=Estimating and interpreting FST: The impact of rare variants.;citation_volume=23;citation_year=2013;citation_pmid=23861382;citation_doi=10.1101/gr.154831.113" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=G. Bhatia;citation_author=N. J. Patterson;citation_author=B. Pasaniuc;citation_author=N. Zaitlen;citation_author=G. Genovese;citation_title=Genome-wide comparison of African-ancestry populations from CARe and other cohorts reveals signals of natural selection.;citation_pages=368-381;citation_volume=89;citation_year=2011;citation_issue=3;citation_pmid=21907010;citation_doi=10.1016/j.ajhg.2011.07.025" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. D. Bitarello;citation_author=C. de Filippo;citation_author=J. C. Teixeira;citation_author=J. M. Schmidt;citation_author=P. Kleinert;citation_title=Signatures of Long-Term Balancing Selection in Human Genomes.;citation_pages=939-955;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evy054" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_journal_abbrev=Immunogenetics;citation_author=J. M. Borghans;citation_author=J. B. Beltman;citation_author=R. J. De Boer;citation_title=MHC polymorphism under host-pathogen coevolution.;citation_pages=732-739;citation_volume=55;citation_year=2004;citation_issue=11;citation_pmid=14722687;citation_doi=10.1007/s00251-003-0630-5" />
<meta name="citation_reference" content="citation_author=D. Y. C. Brandt;citation_author=V. R. C. Aguiar;citation_author=B. D. Bitarello;citation_author=K. Nunes;citation_author=J. Goudet;citation_volume=5;citation_year=2015;citation_pmid=25787242;citation_doi=10.1534/g3.114.015784" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. Cammen;citation_author=J. I. Hoffman;citation_author=L. Knapp;citation_author=J. Harwood;citation_author=W. Amos;citation_title=Geographic variation of the major histocompatibility complex in Eastern Atlantic grey seals (Halichoerus grypus).;citation_pages=740-752;citation_volume=20;citation_year=2011;citation_issue=4;citation_pmid=21199032;citation_doi=10.1111/j.1365-294X.2010.04975.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=M. Nordborg;citation_author=D. Charlesworth;citation_title=The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.;citation_pages=155-174;citation_volume=70;citation_year=1997;citation_issue=2;citation_pmid=9449192;citation_doi=10.1017/S0016672397002954" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=D. Čížková;citation_author=J. G. de Bellocq;citation_author=S. J. E. Baird;citation_author=J. Piálek;citation_author=J. Bryja;citation_title=Genetic structure and contrasting selection pattern at two major histocompatibility complex genes in wild house mouse populations.;citation_pages=727-740;citation_volume=106;citation_year=2011;citation_issue=5;citation_pmid=20823902;citation_doi=10.1038/hdy.2010.112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. R. Abecasis;citation_author=C. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_journal_abbrev=Immunogenetics;citation_author=R. J. De Boer;citation_author=J. M. Borghans;citation_author=M. van Boven;citation_author=C. Keşmir;citation_author=F. J. Weissing;citation_title=Heterozygote advantage fails to explain the high degree of polymorphism of the MHC.;citation_pages=725-731;citation_volume=55;citation_year=2004;citation_issue=11;citation_pmid=14722686;citation_doi=10.1007/s00251-003-0629-y" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. de Filippo;citation_author=F. M. Key;citation_author=S. Ghirotto;citation_author=A. Benazzo;citation_author=J. R. Meneu;citation_title=Recent selection changes in human genes under long-term balancing selection;citation_volume=33;citation_year=2016;citation_pmid=26831942;citation_doi=10.1093/molbev/msw023" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Doherty;citation_author=R. M. Zinkernagel;citation_title=Enhanced immunological surveillance in mice heterozygous at the H-2 gene complex.;citation_pages=50-52;citation_volume=256;citation_year=1975;citation_issue=5512;citation_pmid=1079575;citation_doi=10.1038/256050a0" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=M. D. Edge;citation_author=N. A. Rosenberg;citation_title=Upper bounds on FST in terms of the frequency of the most frequent allele and total homozygosity: The case of a specified number of alleles.;citation_pages=20-34;citation_volume=97;citation_year=2014;citation_doi=10.1016/j.tpb.2014.08.001" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=C. Eizaguirre;citation_author=T. L. Lenz;citation_author=M. Kalbe;citation_author=M. Milinski;citation_title=Rapid and adaptive evolution of MHC genes under parasite selection in experimental vertebrate populations.;citation_pages=621-621;citation_volume=3;citation_year=2012;citation_pmid=22233631;citation_doi=10.1038/ncomms1632" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. Elhaik;citation_title=Empirical Distributions of F(ST) from Large-Scale Human Polymorphism Data.;citation_pages=e49837-e49837;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185452;citation_doi=10.1371/journal.pone.0049837" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=Y. Field;citation_author=E. A. Boyle;citation_author=N. Telis;citation_author=Z. Gao;citation_author=K. J. Gaulton;citation_title=Detection of human adaptation during the past 2,000 years.;citation_pages=1-18;citation_volume=0776;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=J. Goudet;citation_title=Hierfstat, a package for R to compute and test hierarchical F-statistics.;citation_pages=184-186;citation_volume=5;citation_year=2005;citation_doi=10.1111/j.1471-8286.2004.00828.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. A. Gourraud;citation_author=P. Khankhanian;citation_author=N. Cereb;citation_author=S. Y. Yang;citation_author=M. Feolo;citation_title=HLA Diversity in the 1000 Genomes Dataset.;citation_pages=e97282;citation_volume=9;citation_year=2014;citation_pmid=24988075;citation_doi=10.1371/journal.pone.0097282" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=P. W. Hedrick;citation_title=Pathogen resistance and genetic variation at MHC loci.;citation_pages=1902-1908;citation_volume=56;citation_year=2002;citation_issue=10;citation_pmid=12449477;citation_doi=10.1554/0014-3820(2002)056[1902:PRAGVA]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. W. Hedrick;citation_author=G. Thomson;citation_title=EVIDENCE FOR BALANCING SELECTION AT HLA;citation_pages=449-456;citation_volume=104;citation_year=1983;citation_issue=3;citation_pmid=6884768" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_author=A. V. Hill;citation_author=C. E. Allsopp;citation_author=D. Kwiatkowski;citation_author=N. M. Anstey;citation_author=P. Twumasi;citation_title=Common west African HLA antigens are associated with protection from severe malaria.;citation_pages=595-600;citation_volume=352;citation_year=1991;citation_issue=6336;citation_pmid=1865923;citation_doi=10.1038/352595a0" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Hofer;citation_author=M. Foll;citation_author=L. Excoffier;citation_title=Evolutionary forces shaping genomic islands of population differentiation in humans.;citation_pages=107-107;citation_volume=13;citation_year=2012;citation_pmid=22439654;citation_doi=10.1186/1471-2164-13-107" />
<meta name="citation_reference" content="citation_journal_title=Human immunology;citation_journal_abbrev=Hum Immunol;citation_author=J. A. Hollenbach;citation_author=G. Thomson;citation_author=K. Cao;citation_author=M. Fernandez-Vina;citation_author=H. A. Erlich;citation_title=HLA diversity, differentiation, and haplotype evolution in Mesoamerican Natives.;citation_pages=378-390;citation_volume=62;citation_year=2001;citation_issue=4;citation_pmid=11295471;citation_doi=10.1016/S0198-8859(01)00212-9" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Z. Hu;citation_author=Y. Liu;citation_author=X. Zhai;citation_author=J. Dai;citation_author=G. Jin;citation_title=New loci associated with chronic hepatitis B virus infection in Han Chinese.;citation_pages=1499-1503;citation_volume=45;citation_year=2013;citation_pmid=24162738;citation_doi=10.1038/ng.2809" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. L. Hughes;citation_author=M. Nei;citation_title=Pattern of nucleotide substitution at major histocompatibility complex class I loci reveals overdominant selection.;citation_pages=167-170;citation_volume=335;citation_year=1988;citation_issue=6186;citation_pmid=3412472;citation_doi=10.1038/335167a0" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. C. Jackson;citation_author=J. L. Campos;citation_author=K. Zeng;citation_title=The effects of purifying selection on patterns of genetic differentiation between Drosophila melanogaster populations.;citation_pages=163-174;citation_volume=114;citation_year=2014;citation_pmid=25227256;citation_doi=10.1038/hdy.2014.80" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Jakobsson;citation_author=M. D. Edge;citation_author=N. Rosenberg;citation_title=The relationship between F(ST) and the frequency of the most frequent allele.;citation_volume=193;citation_year=2013;citation_pmid=23172852;citation_doi=10.1534/genetics.112.144758" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=J. Klein;citation_author=A. Sato;citation_title=The HLA System- First of Two Parts;citation_pages=702-709;citation_volume=343;citation_year=2000;citation_issue=10;citation_pmid=10974135;citation_doi=10.1056/NEJM200009073431006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. A. Lawlor;citation_author=F. E. Ward;citation_author=P. D. Ennis;citation_author=A. P. Jackson;citation_author=P. Parham;citation_title=HLA-A and B polymorphisms predate the divergence of humans and chimpanzees.;citation_pages=268-271;citation_volume=335;citation_year=1988;citation_issue=6187;citation_pmid=3412487;citation_doi=10.1038/335268a0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. M. Leffler;citation_author=Z. Gao;citation_author=S. Pfeifer;citation_author=L. Ségurel;citation_author=A. Auton;citation_title=Multiple instances of ancient balancing selection shared between humans and chimpanzees.;citation_volume=339;citation_year=2013;citation_pmid=23413192;citation_doi=10.1126/science.1234070;citation_georef=2013044317" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Z. Li;citation_author=D. M. Absher;citation_author=H. Tang;citation_author=A. M. Southwick;citation_author=A. M. Casto;citation_title=Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation;citation_pages=1100-1104;citation_volume=319;citation_year=2008;citation_issue=5866;citation_pmid=18292342;citation_doi=10.1126/science.1153717" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Lighten;citation_author=A. S. T. Papadopulos;citation_author=R. S. Mohammed;citation_author=B. J. Ward;citation_author=I. G. Paterson;citation_title=Evolutionary genetics of immunological supertypes reveals two faces of the Red Queen.;citation_pages=1294;citation_volume=8;citation_year=2017;citation_doi=10.1038/s41467-017-01183-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=C. Loiseau;citation_author=M. Richard;citation_author=S. Garnier;citation_author=O. Chastel;citation_author=R. Julliard;citation_title=Diversifying selection on MHC class I in the house sparrow (Passer domesticus).;citation_pages=1331-1340;citation_volume=18;citation_year=2009;citation_issue=7;citation_pmid=19368641;citation_doi=10.1111/j.1365-294X.2009.04105.x" />
<meta name="citation_reference" content="citation_journal_title=Tissue antigens;citation_journal_abbrev=Tissue Antigens;citation_author=S. G. E. Marsh;citation_author=E. D. Albert;citation_author=W. F. Bodmer;citation_author=R. E. Bontrop;citation_author=B. Dupont;citation_title=Nomenclature for factors of the HLA system, 2010.;citation_pages=291-455;citation_volume=75;citation_year=2010;citation_issue=4;citation_pmid=20356336;citation_doi=10.1111/j.1399-0039.2010.01466.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Maruki;citation_author=S. Kumar;citation_author=Y. Kim;citation_title=Purifying selection modulates the estimates of population differentiation and confounds genome-wide comparisons across single-nucleotide polymorphisms.;citation_volume=29;citation_year=2012;citation_pmid=22826460;citation_doi=10.1093/molbev/mss187" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=R. J. S. McCairns;citation_author=S. Bourget;citation_author=L. Bernatchez;citation_title=Putative causes and consequences of MHC variation within and between locally adapted stickleback demes.;citation_pages=486-502;citation_volume=20;citation_year=2011;citation_issue=3;citation_pmid=21134013;citation_doi=10.1111/j.1365-294X.2010.04950.x" />
<meta name="citation_reference" content="Mendes, F. H. K., 2013 Seleção natural em genes HLA e seu efeito sobre regiões adjacentes do genoma. Ph.D. thesis, Universidade de São Paulo." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Meyer;citation_author=R. M. Single;citation_author=S. J. Mack;citation_author=H. Erlich;citation_author=G. Thomson;citation_title=Signatures of demographic history and natural selection in the human major histocompatibility complex Loci.;citation_volume=173;citation_year=2006;citation_pmid=16702436;citation_doi=10.1534/genetics.105.052837" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=D. Meyer;citation_author=G. Thomson;citation_title=How selection shapes variation of the human major histocompatibility complex: a review.;citation_pages=1-26;citation_volume=65;citation_year=2001;citation_issue=Pt 1;citation_pmid=11415519;citation_doi=10.1046/j.1469-1809.2001.6510001.x" />
<meta name="citation_reference" content="citation_journal_title=Immunogenetics;citation_author=D. Meyer;citation_author=V. R. C. Vitor;citation_author=B. D. Bitarello;citation_author=D. Y. C. Brandt;citation_author=K. Nunes;citation_title=A genomic perspective on HLA evolution.;citation_pages=5-27;citation_volume=70;citation_year=2018;citation_doi=10.1007/s00251-017-1017-3" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=H. C. Miller;citation_author=F. Allendorf;citation_author=C. H. Daugherty;citation_title=Genetic diversity and differentiation at MHC genes in island populations of tuatara (Sphenodon spp.).;citation_pages=3894-3908;citation_volume=19;citation_year=2010;citation_issue=18;citation_pmid=20723045;citation_doi=10.1111/j.1365-294X.2010.04771.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=C. Muirhead;citation_title=Consequences of population structure on genes under balancing selection.;citation_pages=1532-1541;citation_volume=55;citation_year=2001;citation_issue=8;citation_pmid=11580013;citation_doi=10.1554/0014-3820(2001)055[1532:COPSOG]2.0.CO;2" />
<meta name="citation_reference" content="Nunes, K., 2011 Populações ameríndias da América do Sul: estudo multi-locus e inferência histórico demográfica e seletiva. Ph.D. thesis, Universidade de São Paulo." />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. K. Oliver;citation_author=X. Lambin;citation_author=T. Cornulier;citation_author=S. B. Piertney;citation_title=Spatio-temporal variation in the strength and mode of selection acting on major histocompatibility complex diversity in water vole (Arvicola terrestris) metapopulations.;citation_pages=80-92;citation_volume=18;citation_year=2009;citation_issue=1;citation_pmid=19140966;citation_doi=10.1111/j.1365-294X.2008.04015.x" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=F. Prugnolle;citation_author=A. Manica;citation_author=M. Charpentier;citation_author=V. Guernier;citation_author=M. Population;citation_title=Pathogen-driven selection and worldwide HLA class I diversity.;citation_pages=1022-1027;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=15936272;citation_doi=10.1016/j.cub.2005.04.050" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Reynolds;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=ESTIMATION OF THE COANCESTRY COEFFICIENT: BASIS FOR A SHORT-TERM GENETIC DISTANCE;citation_pages=767-779;citation_volume=105;citation_year=1983;citation_issue=3;citation_pmid=17246175;citation_doi=10.1021/ja00342a019" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=M. Roesti;citation_author=W. Salzburger;citation_author=D. Berner;citation_title=Uninformative polymorphisms bias genome scans for signatures of selection.;citation_pages=94-94;citation_volume=12;citation_year=2012;citation_pmid=22726891;citation_doi=10.1186/1471-2148-12-94" />
<meta name="citation_reference" content="citation_journal_title=Tissue Antigens;citation_author=A. Sanchez-Mazas;citation_title=An apportionment of human HLA diversity.;citation_pages=198-202;citation_volume=69;citation_year=2007;citation_pmid=17445200;citation_doi=10.1111/j.1399-0039.2006.00802.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. H. Schierup;citation_author=X. Vekemans;citation_author=D. Charlesworth;citation_title=The effect of subdivision on variation at multi-allelic loci under balancing selection.;citation_pages=51-62;citation_volume=76;citation_year=2000;citation_issue=1;citation_pmid=11006634;citation_doi=10.1017/S0016672300004535" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. W. Slade;citation_author=H. I. McCallum;citation_title=Overdominant vs. Frequency-Dependent Selection at MHC Loci.;citation_pages=323;citation_volume=132;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Human immunology;citation_journal_abbrev=Hum Immunol;citation_author=O. D. Solberg;citation_author=S. J. Mack;citation_author=A. K. Lancaster;citation_author=R. M. Single;citation_author=Y. Tsai;citation_title=Balancing selection and heterogeneity across the classical human leukocyte antigen loci: a meta-analytic review of 497 population studies.;citation_pages=443-464;citation_volume=69;citation_year=2008;citation_issue=7;citation_pmid=18638659;citation_doi=10.1016/j.humimm.2008.05.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Takahata;citation_author=M. Nei;citation_title=Allelic genealogy under overdominant and frequency-dependent selection and polymorphism of major histocompatibility complex loci.;citation_pages=967-978;citation_volume=124;citation_year=1990;citation_issue=4;citation_pmid=2323559" />
<meta name="citation_reference" content="The 1000 Genomes Project Consortium, 2015 A global reference for human genetic variation. Nature 526: 68–74. 10.1038/nature15393" />
<meta name="citation_reference" content="The International HIV Controllers Study, 2010 The Major Genetic Determinants of HIV-1 Control Affect HLA Class I Peptide Presentation. Science 330: 1551–1557. 10.1126/science.1195271" />
<meta name="citation_reference" content="citation_journal_title=Genes and immunity;citation_journal_abbrev=Genes Immun;citation_author=P. R. Vanderborght;citation_author=A. G. Pacheco;citation_author=M. E. Moraes;citation_author=G. Antoni;citation_author=M. Romero;citation_title=HLA-DRB1*04 and DRB1*10 are associated with resistance and susceptibility, respectively, in Brazilian and Vietnamese leprosy patients.;citation_pages=320-324;citation_volume=8;citation_year=2007;citation_issue=4;citation_pmid=17396103;citation_doi=10.1038/sj.gene.6364390" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Wang;citation_title=Does GST underestimate genetic differentiation from marker data?;citation_pages=3546-3558;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13204" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. Li;citation_author=H. Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.1111/j.1558-5646.1984.tb05657.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Xu;citation_author=S. Li;citation_author=Y. Yang;citation_author=J. Tan;citation_author=H. Lou;citation_title=A genome-wide search for signals of high-altitude adaptation in Tibetans.;citation_volume=28;citation_year=2011;citation_pmid=20961960;citation_doi=10.1093/molbev/msq277" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Q. Zhou;citation_author=L. Zhao;citation_author=Y. Guan;citation_title=Strong Selection at MHC in Mexicans since Admixture.;citation_pages=e1005847;citation_volume=12;citation_year=2016;citation_pmid=26863142;citation_doi=10.1371/journal.pgen.1005847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="Balancing selection is defined as a class of selective regimes that maintain polymorphism above what is expected under neutrality. Theory predicts that balancing selection reduces population differentiation, as measured by FST. However, balancing selection regimes in which different sets of alleles are maintained in different populations could increase population differentiation. To tackle the connection between balancing selection and population differentiation, we investigated population differentiation at the HLA genes, which constitute the most striking example of balancing selection in humans. We found that population differentiation of single nucleotide polymorphisms (SNPs) at the HLA genes is on average lower than that of SNPs in other genomic regions. We show that these results require using a computation that accounts for the dependence of FST on allele frequencies. However, in pairs of closely related populations, where genome-wide differentiation is low, differentiation at HLA is higher than in other genomic regions. Such increased population differentiation at HLA genes for recently diverged population pairs was reproduced in simulations of overdominant selection, as long as the fitness of the homozygotes differs between the diverging populations. The results give insight into a possible “divergent overdominance” mechanism for the nature of balancing selection on HLA genes across human populations." />
<meta name="og-title" property="og:title" content="The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2805" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Balancing selection is defined as a class of selective regimes that maintain polymorphism above what is expected under neutrality. Theory predicts that balancing selection reduces population differentiation, as measured by FST. However, balancing selection regimes in which different sets of alleles are maintained in different populations could increase population differentiation. To tackle the connection between balancing selection and population differentiation, we investigated population differentiation at the HLA genes, which constitute the most striking example of balancing selection in humans. We found that population differentiation of single nucleotide polymorphisms (SNPs) at the HLA genes is on average lower than that of SNPs in other genomic regions. We show that these results require using a computation that accounts for the dependence of FST on allele frequencies. However, in pairs of closely related populations, where genome-wide differentiation is low, differentiation at HLA is higher than in other genomic regions. Such increased population differentiation at HLA genes for recently diverged population pairs was reproduced in simulations of overdominant selection, as long as the fitness of the homozygotes differs between the diverging populations. The results give insight into a possible “divergent overdominance” mechanism for the nature of balancing selection on HLA genes across human populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"drkprTKUwGilxJQSggGEGqLq8lPHCs_PyTibfu81uvQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490447","apath":"\/ggg\/8\/8\/2805.atom","pisa":"ggg;8\/8\/2805","ac":{"\/ggg\/8\/8\/2805.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2805.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2805"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2805"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2805.abstract":true,"\/content\/8\/8\/2805":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PuRt48OaCUd\\\\\\\/oze0I7bcmPgX+hQ0tR1v8BJwCekkLUMTl+wa4RCzgKBX6QHdjni5DhmPYQlMM7GIDfR4PSP4+0U3phqPt2Of4LmUyEE6mQxJGFQeh+LyP\\\\\\\/ZmHb36peJ2bxubWo5acmTJg7tWDYkzYVeSv0XuUf5IY7QcxUzF5u5kRIBhzDl4rMk3RDPHfJh\\\\\\\/B7wAjh9yYiuQK+Ht3MwrRjzj7C1AL3YL4MXSyBk6K1mrHWBpfXGUTWXKpLj8Fd6+yj8SRRPyrBrxLoxPi0jDQHUc60ISuM+pAzqVb4REz9Oke5TetuElCi0AwQgysdeu8XBp5bGdR8XyOc54FIH30Fs\\\\\\\/BznVnZLo6u38dVp9SM41plyGgTXBjIWzDFK41UBmcImkrHtpiy7o+RdCPFUYmHYfp\\\\\\\/Aib3jQNAqLCJY1nOp8tqbNm2DII3MnzWIclwZrLBhXwzqaBaxetTQtAfapJcP9WcG5iEzi7sLAzvfF3obpFz+m6r5oDbMNWeeHbnpHLxtk29NN8XMOzF2I2YrOBss4cJl8vK1qS2eWkug7XAliW5vshcypHmdy3tn3D1mVeC6eBhfmppn6YLFBbGYigqV2rsOGzz27IgTS7307CgQLd\\\\\\\/OOKpP7OO8I2EJY2RpicwQtA6xG5gmBglI7p4y1rPk93x4\\\\\\\/zWj6q+LMCXkxE8qepWL4dvbdry02mZY6CsSBzF3Dflr9RgAj7VC92Q6OiY5FUdR3k7kfRzuVm1V9YOWvzyYtYOSlxKI\\\\\\\/4iRtaI3ZVvmdb7PFhEh0jrRPLQKVAkQ31d2QtQVtITlXS5\\\\\\\/\\\\\\\/E+kEd8+d9SHHR\\\\\\\/E155emsQJQK1HqZLcskZVJacFLEiKrli5jQ8WPZg8fSJf027TTU0Tsrd9FWdWj7cOiB8bnzDMwi9bOQ7N2MMQvapjqFWEiA5+lz\\\\\\\/qH0VzKjiOVZ4DNw4EoVjAc+ptXmBONex0jFrz0H39u96DjfnJBqPkclrZuXWPZnN9Z4UzRNRLjgUtjioYJq82P3lrqmGtkt0PRQrfdCUNX7l5ehBiWyBcIX8QMt0QvhsPhFf4vnF2d2QVafxRROIUatvFNZgNZnwVD3z4NIUIK2KtFyuzALCs\\\\\\\/9F\\\\\\\/X2jDNKaeIsjzDveuz7+jyu\\\\\\\/hOgMk9ymWHLSn+jrgh5R9DQtXwYk6r23JXTKkc1Y4u+uPSCJCtsbP8melIk10e5GhI\\\\\\\/1YT5SYJ2D41scK61uCuY0oIDslNbC3SfdVOR1FDK7hJds0gXCo7+O\\\\\\\/9DreQ+ZwODxpum6NMWZmWp7Jk+h3a\\\u0022,\\\u0022iv\\\u0022:\\\u00229a6j2tdptf8buKUKt0M34A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ebf7ae4f28c52ef362a0a3974b26d592ddeccfe31226616df0dbf376d5e3508\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BrFkm9KSJgXrzdJ1oSpTphxZTGe2rWheVWtS\\\\\\\/vsquSx8OU6XzOjmWQB6h1h1w+dLdSphcCoGrUWBcFuYgEOU47QTsQcYO6nYap6VwD2hf+VZqTKArQf7EqT1XtHe6U+T6SXU1nNy\\\\\\\/j5L4Y6QzrTONEzdeWi0kDnATxeNDcgbAcnJbREcGbW+vX8BBlXYvxzgcX+jcg3Ss7+RqdNRWOuK7SkyfD5XJ8WtHgWz\\\\\\\/7Y6YEj8j+GAPti7uA52\\\\\\\/mvy1D7by7+QVWX+K\\\\\\\/y4uOF0drGiE1s2rFBt5zbQs8ef343nHtUziN4J374kpCzpy1IL9T66PKl\\\\\\\/wXnxYiJo\\\\\\\/lOsCjvo0CYm55Sc0EzK5U3nC3GKWt\\\\\\\/4T2Zg890109UuWABtL9NmDayWjfP43I6OFJgzn2cHGArUTPgcLD0bNGDQqhTijUaouVXIFk+clS0WOqWqsFK0MJGlpAWQ7SadnuJfLW4H9dRBK1Y84JY8H\\\\\\\/zy+SoLG\\\\\\\/Q3Oajotv8O2eEaIQ4Zo3UdRs9dN10+zaR+V0aAu4\\\\\\\/doR13lcJWpHk8ZF6+bSJ9nJdEXUwVGpwWObWgTgibphCEXd2zm8emOQHSg4zDMyCWecTgJEBIzNICEGs8mUF\\\\\\\/2eDy6skOuOxXPpJOvoLfmQ7FZJZrk9i+YaKoKaUjXSkvPGeXRNkXIhqvAy4VtjBvYC9XmajmIAmF1AGfmfiqvY66Hzh\\\\\\\/mrDO3fqTdz+cxSelrizwMWJVX9LLIzejEALuWg9eUqe6QJifTrIe0VgohzLe0u2+q5x80gHQBDofK8wRDTE90t7ZWmMGR5HsIELmR2+j76VfLwkotzBWs25RnPs3TrUzkMxISf7GkSvEr251rR1BZUpIYNzbwCPgbrMz1OyQUIl6N9b9SbgZAZj4OT\\\\\\\/4cLYX\\\\\\\/VhxCTMiT7Pj1tgv5fhCW+fc1Yv0tK97RHvr7dBS+ICfZVBVO7MtPUHpAugfHdpw\\\\\\\/TuttKPNMC1UyI6U28\\\\\\\/ks6fdPByUBYdDtYUNlyfSnrSNBPIP\\\\\\\/dF4Nx2+92PD6EqaUH9TPffMb\\\\\\\/Hs0cZO0rV6WU9KPYk5OSZyvAyPyoPV+etKdna9y5\\\\\\\/MA4U2iIVP3wndtekIXe03kVbJ5kwNM4Zx3omCfox6272e5VWHPr5SCXtnCjTqus16pD5xQC9we9LD+CQE5MPVIT4X187Bg1Ad2Y16ss52iot\\\\\\\/SR8u0Ek8Ei5b4poZnu6aZGIWWrzdtivsXA6\\\\\\\/fyWr8MaFlAOrvUnPedX7mq5yuBy0lbJdUR3f1nE7MbY7RR3uaaTfJY1UptqqnVOw47UFpVbRReAUYCBcb2nPu330A1DBV1RyPZ2Av7q801FwPUI4ThwX5UNS7vYUp4HJ2mVUEvirdXgG4hdiGLbfFlQxXN9btlqpw2\\\\\\\/AHnxt4MxPR+fXi2KmxPuPabURru2dK6EIPX+f12uGn5AAhJ39GZhJpYwAGgjWZXN+wapv70B+nqNorqI8q9yvehdyj65Te695EVS\\\\\\\/EsCXfV5xIyq\\\\\\\/KyXFMsmKL1gmDe8tElFb8tBP9pl0yxr4Q1YqM09+Hx6GnZ+RXFc5C0tJ7iao8wJuIYGciTLCIAuGk\\\\\\\/SoSkoX+cAigKSA2nLb44wTzamoOEMFJMK9NO6hoCzreY9HENVqzblvaQThq68Ib8Fo7nWfJkIbqVLdfaCmw7x3s+VoUv2EXZHFJ73km\\\\\\\/DPeIo+DsEseYbvb7\\\\\\\/M9BZfIW\\\\\\\/Zw3CrTE5cSXpsj8Z1alI9Gbw6DaIngJOdY5zdCAC4FtYHZjHUKPoMC2HzLgzf8zUPspSQkeW25Mr7nsCVyehxuR1Nr3Onaz1gS9oLlD2TkrN71S1PkA2xSjxw3sUremNiUGKhP3boN8aUw1d2kaZZGWY2un9iROE9IxgcYe1vI5oUID72WMRiSqnRYz4K5j908fiwcTBZn1qWmH\\\\\\\/Qua\\\\\\\/pCqPt6Bgpsi5swyw2pVoeeMBt\\\\\\\/EAQ9WGRPyNEuaT9o7ToFqOfMSljaegrkSBGZMv56MakiqfRiF\\\\\\\/Qz11gTHU8t6KWTS7UsiGNh0jscN7o6aeES65bZ3q3xe+cAtD9RUUrjRJMnEN0sNdXZGVufxeIy7nr3zMxY9QLCzXGkRutIXjfxAdoG0lBdhiLrmfcmbdMs80Ghixw+DAWW1\\\\\\\/sAj\\\\\\\/1neRRJg0txGdL2DLEY0n8bZpvG\\\\\\\/Y7ty3XCJuzeNYCJCVzUj8GhodetahaycyhlSUGL33HFjmmF\\\\\\\/kFOow7xdF3fIzjnpRsIFuJHKONo3TFvYgp7Z3bKA73nZfN6NTkLxQZ0SzJBrbADjUSaAvoLFJ1QdUyiits\\\\\\\/ddx8u52\\\\\\\/6xk\\\\\\\/C\\\\\\\/rq8uW5qhmdIFCFsNH3NuvpopEx0LRUTfsoVcgalfXWP76KTMDzPk2a5iiw0kLwd82fmpbpMEIFnuziSIL3OrN1a2Fq7pS+bvI8gjInaZ9EucWCW0OIAw2ON0GwIQ\\\\\\\/Vzo3HR3nx4rLUfAAwMKLKeKEElBoe+oqhwt1iCdKbagOdD+QmPCAn1OPYBWPpJZFJLQ1gGCZEIFpN3Rnj8lv68IpHl5QT6Qzbqa10UV5aT0RKopXk7XdX4x25DgfAqJPc64aicKP92hTR7Tmkwp13YFtYnAC4SRqydmqENu6c1S58wuyc0gi\\\\\\\/jKbr+QfnEaAPqpkJRQl5mv\\\\\\\/\\\\\\\/9AUNJ2w\\\\\\\/Mr53sqpx0EkDsH9m8cn0oq\\\\\\\/I24pdsvjnk0UCr0ftdTy6JKoB0E1PfE44oWjO6a2khUPTRxYUwUiSm1ptKCKcTqAxfbK8Zr8Pstw98pU4PtAFZHQMmNRi\\\\\\\/FXLXyo1Edo+c7YbmlTquDyLujLGjn7Yd2Huz6XpD3tPPyiyH3Bl9wOz2gg68kejCWHhmhB+6pwN5Xma7BGppea98qn2\\\\\\\/NT26+TtFNn93MLv9u6UL3ynsNaCa1g+\\\\\\\/msRr7fo4mmhkAmihVjZeM3eMP+FFWZITORZRSM1fzcufHgWUB\\\\\\\/u+cYRhSZ1WbWeFLO7Om6tK3nBeITvRH69Gn+96ZM\\\\\\\/BL0pfJ3Ogp1kXWnSX7gbRzvqRpJEpF3AGXWbQaZL3M7pVTSvvYbvBHfmXjP9UIjjz2b\\\\\\\/\\\\\\\/VFhYG1LFS4ZoEnbXo\\\\\\\/Sz8hE+usqb4zY441eHKosL7YiSNjYU4hdLnf6N8wf8Qph8Yq5685YqFMqYwHnRExx+jpdq7oJITI8L3NQGaEonALgpvwEldR5e4+oX7s8ykIakJ3\\\\\\\/C45wO46NZPm3rSDB6z1N2AZtKZmIYYSs4cn6tHv29pdGM7yfkd95CUYddHWlBRRfos4SysbXlQQvBO+G\\\\\\\/b3C6Q\\\\\\\/c7PoX\\\\\\\/VEtWlKXGMdtiLQZwGBnkLtszJm3YZv+OMd+4epNf\\\\\\\/O+tMzBTYeqLrYYJRJEIGG+eap0Zca4ZVrK9ZhbQGAXZvfU1DPbCwRYO5WMxmAUb7wfat3FsbjBiqFVRpOj+l9PCOmC41vQTgV1RKRxFBnkoUTQNuO78r8cqxRdljoz8OH6NbC1\\\\\\\/3\\\\\\\/DT2oUvtGj9sj\\\\\\\/QarNV3Uu+1z3VHw2\\\\\\\/fq2rlhNBgy0PZ2g7u6ju8oeVr\\\\\\\/oXe0A9Grs6iAK7tG97WpcwJYsUcrn68KUjsVeBpIB1VvSvRF1BMUcTqyEtfuiQg8J5Jb8IRY1CpDDjA1vkgOh9lpeHaO96qaFWtdZ8kS8Czr8WZcfDNzoY2pajS9qrRpv7x46BA+F2AkPS2SD\\\\\\\/dCHR9AtfbMNTr5D\\\\\\\/vn03ooEqDX3hc1g0k2eJp10SbB7KlUPT32EmpMY7nM5Q+qsu+oNXXC\\\\\\\/\\\\\\\/spoKPZwGRYsU1\\\\\\\/gpb8\\\\\\\/NVclVs7pfZtN9V71ypepThiTUQM4lRlPinCmZ3lBzRMXHzFirtopERB+a+bEThxBnTgr+CsEFf6Jsb4yfK+Zxoa5QQ3X7PdDfXUR+144G3iyGBi2pOafJYXcWiCxDyAJRjxpjVu+VaxVWfyi8kn6WKRmh30JPA1pEXhY5jvBGqsT8lMuDbka6DBFGQPnraxJZ6cv3PJKTHZAfWc7gtK50PgqxQZH9cuiyTse66ChvBuQEPCluaeq8p9Ly1NCe4vvIc6uADY8ShjhHT460869IQ4VvVKmfnITlFEJARQeTVE8YO3jf4aHnlKWlokFHc7nLL8ZFPlltLSQ9IPpnIT0NndKlbEmPUXI2nWwG2o1th8kFbvgJouwc41FVqZWUC8c71XTdqsL0dVaaBf2rcivUA6rF6QkdF7jXHhoQiiBIO0ZEiGzMc1G\\\\\\\/QNwxR0EzpE+HNwRZAlp5PaCgFtYgw5DuwwwnDHHDdz4vT2lPtli7R5gVrbKgE15nWRjmUQhfYqrlotf0woXi6FBWd9hPKKe\\\\\\\/zuOj8cPUYPwJpglxqktV4Zbseyu1QKhYMWXU3HTvXYRFeEnF897gNcQqzZqjXn+MYb69F6cC5zDYuVzuyd3GgqvDV4izyU+DbUqY\\\\\\\/UN1FmEY\\\\\\\/7JKmPpmEunnG+Jd0xQLPTGHDYguimeT+EEp8FcEb6HWaqynQD\\\\\\\/g8qzP2ezlN7OtTezEHz0xJBidRKLvUK3LbKe2HtqJNSeVoXX9yE53L5CE6WBFFe0kibb1sVGxenxGOB9h\\\\\\\/5dpHaUhVqdDs7xSu3Tr4VLtIk0unEFbqkzx3AcrvVBQKrHG\\\\\\\/dj\\\\\\\/FnttW29pbDM8XXehzHXCp+\\\\\\\/V1yPNkUTbJIQHSCkLMwYus6zv5mg9shBkffO1aQ+hMSZ8OLxkTxZSbt8VNW8QER7FibtoiysxNeBUZyPZfQR0v7RqPGaPvK1yR2hbtIp34zBKH2UiXkIoXFSQjKPbXi6iDfZxhoArvrsRCGv33FCoK5ZVYkxQUOqEr72bI+jPpSnrvInOOQq9E\\\\\\\/rhGt8l3VvWb9IN8fKyJYCpoM+UEtx7kHwQhldThFVyn8AtWMs5mxz95afvZpQ9jo+cqJeUZn5RSrfcJeE62QKPcHpF8v1C+QY7nyjlqlkoZGyg3qR3Hc1bQO5HeYL1wlJxgl3IZyr9Ob+LP2iVR4v5W+6Jmzd9eWsrWhIRXub\\\\\\\/2gnN2YnHkPN75RTuIGEOxZ\\\\\\\/i7+sKSgRTZgCEyh370Ww44wpih2o9p\\\\\\\/zO5n8CYolcFJxMIMUa+\\\\\\\/MdaH99wGjAP+2gu\\\\\\\/IhXuOZCrYvuKQRvTHlHv9N8u6U04bx1XOyCuYwPiQ9ETu2sg7EiVvkOTCpHnnSd3kJm7FLkqYAcFcG+wa+yaspn7yW60ihwTPgZEFFe5kDfMZDEfQOMKPhGlQaDGruVk6igZDRZyfNLiWEjX3fKgbaMW0evO+IVKQ1nOlUXqKLSZy2wcFn4hgBF1vdjVG4GNTKty1jOJprw9+ZTdF88aaYbGkp7sWQYe5cEZfnrOQbuyQmEtedbfkjVnCi\\\\\\\/0SIUwsijdqIIUMmzQ6\\\\\\\/xu1tst7apQMh1qx8Q1SfWTfpugIF7Z4NxpjYZVySse1larNXYKD1MPaVbk0Oxiiw2NI5QoBI1YGSgipT5KNFIJR+X6aIp5eGPlWQjM4Y\\\\\\\/qnCoG6EqgG\\\\\\\/68uA3Vtm\\\\\\\/VphshhWYfDUv\\\\\\\/WuKtqWOk7K+yDPoSn8oNY3sDIApxTsC5FBLmgE\\\\\\\/CKLcizYFOp3yEAeHpC63ReKeG05D9r9a\\\\\\\/TkOVGG7oojnzFot+9Fd9vQZBhzPlelD+HehGPKnuFq7o2Qn9NDmlL6bImR+tMmgfF7q6ztc2rjmTsVUfz6V98QVbwUvowZLiV9Zc9LijAdXOZforsmRQNiMNiST8AqMpvELKbYwt3Cmy44SbO2wCdOl1wGhKt0NjukFRuinPmVMBMUllt5Lhomn+ROsN1g5ngovROe9d9xTa+\\\\\\\/q7YzrMI+3nfIQqFs6YMg8XATUxfKNjA0Idesl34d59lJJP1XFPPnTyKtrW8C767xpVwahNw6HMhkky3k6d5Z1+Fk1VdlivJdjKuo0lL5Zv2S36L8J6UAZcsst+97R\\\\\\\/j1dZcqg+4H4DJCXSFchrtTmmW9eUMREQjrjL2XhrODxJ0WLh8dAQjOXFjktZQ5RUDBNBMtWlD89s2zOpYYBqymLQHuJL3WdAcCTfI6sTK0ek0BxNobTTyi79aNnM\\\\\\\/KSfadbjOMe\\\\\\\/gzMfIdlwVuwpoKb5DiFjNOUk7KoJuH6KZGxnXbbSDkTLkADUsDtFMQ9HD2y59W+8b2cYLO9W\\\\\\\/oryi3DpnhfIse9+9qqSrkfJxy2kADE+hQKPpU9eHhi4j503\\\\\\\/q7If6jZ9ltyiqmwKrNndtxehVXq+vaGFCN3uPGU82\\\\\\\/\\\\\\\/0+LkmlFPKy2HXK8O69BV0e3U0naCGgFZIFymoB9Eu+sKljhVapppCFdgUcWTuABWcUDqLc7Tld64tqgpG+ghSOT2MCf8BG8BJz7Kix\\\\\\\/LWrrjHoCUtvdU6zXYWBVpoNQ9oMuZhfO+XltMSDUkfGWahC\\\\\\\/uUs603NFkS2Gr3ZdzU2QnOmOeMfUSJBE9ij8mwJsTINtLW0LVOCyqlV3U2f0yJAWXklu3+l8xTUPjb2Zwu0mtYyNNxNZ+fDzf5Pv1hvk+xrtgjzWSViUHR6sR1rAYoNBEqTMFOWSmJr6FoT6Su\\\\\\\/F9c\\\\\\\/Hz\\\\\\\/ngFzgEskTIHMMljbzrLA8n9yVD\\\\\\\/jn7YXcS1YuRfKA5lC+\\\\\\\/9sYjOdBUQyYeoS720HxsjDbuhNFHCxBnAL4QZKDdTrjBDNoy0WDWAblFxhe58wBY9zGdCt+abWiBPuqr48yaY8JraMeqNbXl2zxUfiOaWKdqG2P8qiUawLOckjK9bjr5KyKaX7vYwVslg8VEPdSHZBAhGPKowZxlZN7d2tu01PKohHKM1DxopP+kqHW0ytjg4evusc3GnHQ2sojURQrgvxUhVilui24h4zkfeWUfs+A68XTDHVxCAjIjEbe3jmz1lsdRwdv4kVeCYx5\\\\\\\/STdidFRoxwpbxGHggXWrkStvXXR6t2b5wEyLVAw1Rz9sMi3hUKdiPxfqGnAGsWViHB5Ft5EibdVvVSjOXj94u3ONXqArELdCQZp4wG0YLRjbw1T0wPudjT5qQyLqAJUwMYCVopHbPRZtLn7H9aXoZF2DDPqDM6MaUCwWYGuOekBC1l4J1MI+3abk5KEnW+6kJQWHOE8qeiD3T5nihg5YzBQy3KkJQVTJxwnirisWBEoXJc9t75MuoEyQUk6XB2wrbHm0N7Kbyg0HPdDcztC\\\\\\\/JKFvUFGpKq8o5Hg8hdLetqnKoq6CqUI0NEL0Y\\\\\\\/pCV35IKOxWI7zO8DxWODNRFARfJs4ksjLYYdLfiul50yTrQONAL\\\\\\\/UsEvMdymu04ewixqdGfErzL\\\\\\\/PzG\\\\\\\/BPKXgbMGhHJ4r+OVfMpTBe1v9D9Vthn1Ko8cFgfjaj6UF+0oGqojg61DIcUPb9UtzMywG59xkaawvI+MggB6qt7KD8+z+oU57Fwy7SSbiAgkL1333dLhNEw61xlCIv8dHehgahMUvNyE+jWRuchUazk5vXlL8lEjmzxH\\\\\\\/+AAYyVeESoJpcr6JJ\\\\\\\/EquTz6ntP8ekEwo6467cUcEubRw3PbC2ZMBJdtuZT8MPy1s8etoRE0jiF4zgVP6UtCqcmzvcAADIgF37YmDXp\\\\\\\/U\\\\\\\/Y37x2pXOipfDPRTF+d2EkfgRlAf4rLY+BRrpJxzXRnwVB9pC+Jb2Vg6tMJPQJC77zOuPbjDku4xi0CMQ\\\\\\\/MgaM3hDtgzunVtG3JNn3Pm0qxcFPcEG9TzHdvAZDPTc7ppkFAfQU16aRObP2SIvqziYvrXWeaq52GqA3yuZJtOCAw90k4WlxrnUNU6k4D680+Q\\\\\\\/jTY\\\\\\\/snfuIUgA3tQuzoMFOzXYOZKdMvG5IMoGTcdvxnxEmkVth4kwD9a8l1wJ7jywEpyMaHfLDE7pYM+mxmTgwEiFKPTjXq8foPMHhKGeIB0jPFXz58IlxBZ4dEKRt2wdRITR7FoRn7XQ0D6BP6pIZ5Jen+dQ6k1jiM+lG9MGUqCqAjTc8UwUK7c4NmrTOwMe7ds62JUohK97tIpWn3O0txD3fNcE2cBgjZx01LUXIHSr9WUrJwLd9QsqjnRE2y8gWCABRsWJvb8e\\\\\\\/aHC1RelbLi\\\\\\\/PMtr2jy8GhT25M+dZ8yVKzex4BmwjWv2NKJNxnaGlqMVU2E8+d+vqngpZFlD085+iF8OHlQfS8KA2FIKBvwAA7ugwVzWehJ6KAfenEIUDEcuyh7xTIlSEesr3FRWdGk7YXm4Xicxv\\\\\\\/3RbKNFw1mB9ilv7Pplk1DB\\\\\\\/LLc\\\\\\\/Le9wpma16FwJNcwqBRGngGeGmtvwmL++nFk+7DbTOgdcOaxSBMKrI\\\\\\\/ElGy22L4s8l1BLfRwMVMwr1BdTPi\\\\\\\/Q\\\\\\\/JV5iHUwI53r6UVl3VamlkS93JqpAzN6lY3wCIx+7zcD+QPVttxhjuD1DVS4aKipWb3JkJaooGBebR7aHFV0IYW++aLYZVAlBAZ9vUtDPMI+TyiVZ+OUZgAauDm5sXiI5GJDHC9DJILq2yek\\\\\\\/BeIgDHzPZ9rXCfQgdrH0YnYz90+eOLTCDB7IN0lMtnJ1TKwctjYu6WbTK16d0V\\\\\\\/hhY+pL3FPGd3tCpVGEEgzFWdA3HiFQX6a3JCyQR\\\\\\\/klAx1yL8Gf5OB40xDJPX7gtrlBg9y4sIbJaX6zD25c\\\\\\\/vOy4n853Hx1IRF0kMSC6Sy3DrBW1OJ6Vun\\\\\\\/N\\\\\\\/RPsD2bZmQQ+NhTEHi5pFPM5MEqKGJerDIzudiUADkuZG95LdUMG7BjewOvPP4xQiueQ8jGpvYF1cxiDeyhU7NeuacYfV\\\\\\\/wm66W\\\\\\\/yfqnrpwthla3Q\\\\\\\/g1I9VQCMAmWoCBshEG\\\\\\\/kNMCRDvQexCrVwvLoWiE5iGMpOHNYBb8jvdWU1viy8gW4lvkulqI+9rnwzfFtAMNWbWPDQSBR+xbZXyQ1pjEKHo555LDcIopDRw+WapTfCoLJytFbzxksL2a2NQY1suAnHd5s5CWPhXFlHwGG9PNojN9DtiNzIQZZjokgFFp+8v0l116eqDs0pmNxBFSin9s64QsdDqx9pVid+K4u2B2OZOTLtgXF5Y5ViyvJYaEdRMeRhfcFaGlzXLK9VBWmA0HJON9MQPLFph57tXIvm+6K713y5nFtJ0dLSbvBRXMVOfCJaeDL2dgwjY3HyryG4Igf7WDjrBxZfkOS0eRP9Yf5TqTnrNAMRY\\\\\\\/abPXRq\\\\\\\/AmHJ27TXJMuFip3GmqGVp7OAaDC23Boj66lQEiPhsbNePLOxIsg7uIWDdz\\\\\\\/47b56CwVfvkVS4yhCsOzY8syrbtXQfRnURE+eMRcPagmolscMz5MK4LEmdTlahMrhd1sve3bXxyiK4ago3Bwnbd2pLKLO7BpVTKApwAs+7neVBybISBED\\\\\\\/Lcjga1tK3SbVP\\\\\\\/TXOL1JIxIZn8\\\\\\\/aYGjpgeC151IP2dxcvVgWnaEESQRRGAQwewSZRVv0uRWXkO4P+cAtBunbZlXSY2CNrgw3kvl3WyoIA7+QSwmnxEjghgD1vGKVkWkhN4vLILp4bvshPd\\\\\\\/+UpRBN8fwfpgZuzL8OAMknCd45B2yk7zrfV5kpHciQamajUSe9da1nn8XpjAO+1aBqBMF8C5GBFVvobe3bOxu+F5vxKFwp+JoEVARGca2aGDbHdfeDw5j+ahhsu8i32B89I4g55LpVNNHFd0x7e00bo5IHamo1D1MyP+fNJGU20wVJey5ncPxkZBjkj1WjoG6aaYHvEKFPAhDRymZaieNl6LJ0p3pj+f5Gyvs2rPO\\\\\\\/LxjCbZQq\\\\\\\/6HMcbZSJAi5bxqG+hJFD\\\\\\\/qmq8C7EDEV1SR5PnJiVivjpNX1HkWWd5w8TGJrbT+cfaITE+JAZoCBAKqa5c5GcT5cHS\\\\\\\/bIdJq0jaRkrKT1VdgGWVuBDBPSyUeuubWyCV2VAzyIZtf2bjuZb68TcQHBybZ+bIupQzOypPyli69Qmj\\\\\\\/GXPqOpEJ9a0IZAIKxedovQQiF02hSU9SURHfZgSaIwC0G21h9UXmpCj26mZEBF0VXF2Kq\\\\\\\/mss+wNyFxM0DJATmK3AEFw5vZRy4mi1c3zxL5Qdi\\\\\\\/waQyJk1iVnGcRYYspj55+SHvbJzJKNleScuarcvEsiW+749iRif4O0ByNWMnDnns9YbP+ajILOW9bvHRuxUTqYGvU0FTbmxNDOX5V3qtbHWSI1qj7p3CgUKA6sxUVIeTUPMS6PUIRyqepF59Wj9eUJzROw7q2KjPu2yi+HGR\\\\\\\/lFxN1tmE4WEKltXsU4A389C276\\\\\\\/Zv3o4+Nh\\\\\\\/8jPMR2hWzhmSw7FbUmjhgjxYAS7KGHIGoLx+8EnQOxfU7Ezzpsu2TOIzPxFtoHS9Hep\\\\\\\/YmX8hRkpMYFbWFPQWFj8kUwUpAf91SkQ90WdNHETA5gsbensObEo6cMnTwGVYuTmKEzDVlaKR\\\\\\\/tA+oHzH96+6BHTrRSR7rxx1vivhu+wO344lrlITRUH0ULxx3bus2zDXkVq2NOEA3DLewifLLg1DN8KKjrPsnLecD7+G+ElqVJYlt0pjLf8lgBMgsQ0GMaDGCxbIhDK634SM+nHKuKGaSei\\\\\\\/VRPIhM57S2L14jl1c6EePmVk3tfSmdxcWcxthMbf1ZqnwmPEyx8opw7NDT9db+76aFaQbD3UH6bRo\\\\\\\/cVt\\\\\\\/QB3JhobxPEwKFAARJ7RLHDQZnscUmbBiYZKWn1cohWRW2CZ9Bfxjuw0h67H4pzVc+TfE2iEc0uMPUyqjuhSUWpgGwt3PhkFoVV7T+bpnkBDOHrRCLQsHnHGNd4RwpuOTjYecm+weJGgqRSJhzQCj2tslkaFoTzEowPuKjZAUGURDl\\\\\\\/gqgGkUpTWgOnFSQfB115u85uOpZ4mlAjcaMlkNgh8m+\\\\\\\/6ndDv3PEexaHykB01d7RqsOIgWDIPe6H4+HzOwCG8ATpyPuinL5\\\\\\\/Hog4f+gt88617mERxV4LIfEP15gAQN1qMdlVcsQKPksGzPB8\\\\\\\/Iwi9ZIDJytvKGVxuFvke79fMXUB4bduc55Y6hZy6w8gyLV7mDeGysTpauqL4ZmifNaDZx9pDHEqA3A1d8AfhYj1jeBCabGmWLosmS\\\\\\\/gYd7upBlVHDeTzai9Vgs8OA3zjH2MxAHybWxoeouKbTqdOy6fznui7+zJvJYC8KjPHC5M9m\\\\\\\/GYE3u5tG8EQdTuiWSIIG8A+q8jlxgzBSBEGS2WYxvNdwSh+kPpI7cXFFx5uVL+ksnyB\\\\\\\/U4qqRA9TmjkzTTnc3BrDfCI5IkgQC8JsoNvSTeYomR80\\\\\\\/4rXMHJN3FbmkDr\\\\\\\/r59t5S2LPlAJrsrXd2LUDFP\\\\\\\/1jwzR8UGCpGvtvp2nWRnBXKwTl+WlhFV7n6Lgy125mNB23qpMKh185dxxZXk99PthzrN6FBNx6\\\\\\\/+A\\\\\\\/LFGiJO\\\\\\\/XXPAXEOYibTQUTWVMRiP38SglXd3ucvYVlNOajMCTG2Df+cV6BA8YDbRbcubeaEksVjWfvZSAKS\\\\\\\/4iaxPXxKGzsrTc7O7Tl6HFEJIZtjJK+ixcjJlM0Pc30LcswRjYjE3sNfJcIOFSkV2fcA7HW\\\\\\\/bRULb46Tw1YvGF4WFmv4evIs8o2mkyxXk4dNoXAey0rwbCxYTgEU7HHvBDj5+e3MITTX3akYeYgU1lZ9QIjXBHo1Uc8n1TuYYMg2kZKOugjQs\\\\\\\/2RCduLk\\\\\\\/86a8VzF6rOr3vpoICIeGCYmsbCUWeuX4P3MgUOm48yPFl9sCJy1gqrj7gSuWty\\\\\\\/SbfiUW6WC618IHU\\\\\\\/5R8AXBoWK5wINJMPM+jJAaMcN+BTkhPWAGMBWcw+1BHlnRjFroKkYi\\\\\\\/UiNg7o3JQSO+C3hqhKZN2GL1SYzvUEmGq58JNLNQiZulSOQ\\\\\\\/uguwRxTaPkd4V9Hc8Y\\\\\\\/AEV2Okb4Ze7\\\\\\\/caQJoQLz+99qA2tP53BL3nbqeNyIdk9gLfwLLxAvYnTyL0ogUyYBH3qC0MgMbNJNwIQTW+emQlBqOJWnqCwHWhQUE\\\\\\\/gzq\\\\\\\/mQ+h3HpQYUEMQYbrn0Tkdw+iJUXI6IOCQPcntZ5Ms2M6D3Ecs95w3MYrDUvTjZM1i7Oq82NrNrGKZ2eswg02+9CUkqHRGw\\\\\\\/hfbbl8NTKDMeCpEJy+RgiH5A2hd9I1f8n64SiDmWAwhC+xQT5O6n51c0PiLqsp\\\\\\\/y\\\\\\\/eM0yTSdtxlyA+0584tTbU7a0neMckP+\\\\\\\/Z3CyuXXKK+lkMP+OC3H1Rl\\\\\\\/gtVne+ahmB2hlvOrConP0z3sSjacnna0xjzPWmP9BvHfDW6xjEePFSKwQpEq3LqzYTXsfDei5M0iOpLACCKxBYLtXX0LqPrwa\\\\\\\/1ah5siourCWXM2lxewsuqxJD2L1rfJjatD70Uj1y7yEQjG5zir86J9y\\\\\\\/eJBjAtyy01MINCT4\\\\\\\/MhKOCQQnvZDo9tyzfdpFHdAaokis1WxyUECZSZ1z0ZKB92xw15eF0NatyfNWjDrSJKD0qNjkiDaMj56iU\\\\\\\/NpCuXo2lxrnKd6+9jhu7CWNs5vJHRyd0pf4UBuDbQNNF1EVoSZ94czoc0ONTUtyytiFTxxiQjucvKljnmk0b\\\\\\\/h\\\\\\\/izMJCEe0PY8iV5iaPdllu2XMyQDVVIT80AghsPrRw9N\\\\\\\/hKv+YgLeBsBFKB1l4s0N6isUJ2wubw8OSBvZJmRSuNcy5Nf\\\\\\\/Uk30C5PdavGvIAKktnyU0nbL4GbVWQEATlqgZnS2sDH+\\\\\\\/tUIpC76sSh5VMg\\\\\\\/TSe8DaCXorMCOZWFuCKQ7iAR8VLccSOpbEkqtEdShx3Dxp72wTMPbiN2VZTEqk9vdWrTjg1PsNzsAYwiOBbQBoA6xRJj0CGAaWphOJlmde4OVQr6K10F1eEhxFCMsMLOWYvqKcfKz+zLHy2ty\\\\\\\/cqBBQjPBDMLegKY\\\\\\\/O0JL++hiDeNsClfmjhDap8B6i3IaKRHWtO+c3LNY4XDIbxJXA6G5wdoHYwVliVEcKInTGtliiXItRuh667hYOV\\\\\\\/ayDr9P5d4Z4YeJTg8B1\\\\\\\/VNDny1mOlGAEtVfv0Bg2FSXZRtf1JzZ2MXJo\\\\\\\/xoL26RJo32wjURrsi2rION9XIz+uDjJIlTG8Ba2poSoo6d8yaRBx3roNF9+gVWzU8NucO0BpIizM91GP5FALwAhRVfhPzF7tdGJcPvuMNvSl6iKOr2MsSEgHJbWK1amWzdH0nlLXNhowatIKManZCA5QSu8DGIzsX\\\\\\\/OSrewNQDuXrx7kwzyqd+DI4Mls4Ebwotl7d4h8OPj\\\\\\\/0VqAFI3WeSekE41\\\\\\\/5GkZa1fWW+EN1ru7Sz6CGU6dMu3N5BJZ819ZRdXg3mslDokARvZp5Y1sELWNnM1aVQG486Gr9\\\\\\\/mphWa7p97u8NQu6Ve4rp3tdJNDzNRi++UlhoqOw\\\\\\\/Gntib+gOfyayppW7N3spsyR7L\\\\\\\/OcYHdpsl16ppStx5BJC7qQ6BE9s948DRy0F17fFxg01LgW8nhPd3segUHjbU4fDC6SCW2Xf1E\\\\\\\/pi50+jcOctkNTouByYdJHPKwk8dd0rXZwJOTrykhM0O3bCaax6GVfJPseHoBg5Y0ACRBAp6jUPPFh2vOzXOfwiqOWGFMD1qLCFceQDnjIU9p\\\\\\\/esBBOweTwDPWfuqBeT4HiRNcHHvwlyfUREIeVccOSQt6ahu0n7cbOOWwD85PiaPNdQQs9A6Z6ikD4osD6aYzfMIBwckrqxp83yh5Ts53owTCNB7bzFxn\\\\\\\/b0zKfUes8qq2zGbLWFHhJ7eyXt6pULc6igk8jIz4pjHeYa0qgypWmQzCKPcUoWygoRlRSAvs\\\\\\\/Ixa9i4sHKWycsFZ\\\\\\\/y1GMLy2EqpV4yLx33D0W6yBCbc092xMqnc2hZ2KcoHtF8RcoqkP4tb4MBQMH\\\\\\\/SW2uB9AlcLUaXcXkGva+F7pmVUSu8Zua\\\\\\\/MNSDecdJFJ1VmnkNRptdmmlMo0GHTJ94Lu6PzvYGjTe+kyFCKqjXZW4FZPPHha11x07JkfwuPLqV0wbBNg\\\\\\\/MGSHjmjnqRV1141vU9wEjhdwnKYPyRZMgLRSVr8jlycFKGkzdWwqRaalGWb5F9vSM+ZATIhZuc+eZOfw8B6JNfMATjnWDsbOoamq\\\\\\\/6QNfOyqlS5tiA7hkssE5VhlWRCEAxV2dWxvj9XKdXbwXe0Amcy7sJn+Fw\\\\\\\/jimiax\\\\\\\/qvJxkX\\\\\\\/80Rh7ejvMJlNHmJsaae4iddMg\\\\\\\/0oE5mxaahE9XMj+7tDlWs5BD9BcJ9cytVgAnk1tVqhbiTMNYEj2q1endpKkccwQwt5LtwmXYcMsIvl536RXGrSDtnRsfj5XFks+CcA6o\\\\\\\/HaQVk2LTTZTXRG7Mc4XtgZ9G7R\\\\\\\/kY3hsIS4hcPu9SpkBAJvQNokjXY2OuJlgiNNotYEMJRnMF38iDKZCvncjg5QPxKwT83jxeKzKOKjXih7Wp5wqZkzxABFP0rl3dlkUbAd1sqSVEL0R\\\\\\\/WcbI44uHQDJkR0h3T8krCNV3xju59LttlMpl5H\\\\\\\/LvLkq7nEpn2G41fIouPeEnwvQv+C\\\\\\\/J\\\\\\\/NHb2qWqUFnb+bKW1OWyXQXc85vEqA9p6iHZ9k7OtdpuCtu2EDUXvBnfE\\\\\\\/hIWF42D3vYN28UcKcE08hy393WTeYayR2+4\\\\\\\/W4bIMkkNmn9Pcs93Wt8jafBBrtyoJKgPYmxIsLKD1LPaDbtuA9ifzBUa7Nv5vWgGAAeh5IuIH1SIRLpZro7qp6DqhCw\\\\\\\/wB4THjXUERU1XNTrEWrEkxZZDdiPdpzaKdTVbuxxZaJfXj+2T4qRLKIV5glSeLUZzxwxT4is5TndZdiOaRdUrmOc0XItncdi4VHHNl17lmP+uaa3mYfox0BcRy9hub7z\\\\\\\/1jqqEeGIW426+iHHWxf1OEyQPfjS0rCm\\\\\\\/CJl2jXgTBydMKJdb56s8keMTAtgve1lttXLXyQF8gRMn5krbkCJkpPm2rEY\\\\\\\/LN7SVakhJdE44mII4ZUhLMGZmCoIV6VWdjXeKMmynliAvS2\\\\\\\/Q4tJTWAQukScQMpPxAc2Rb\\\\\\\/ZB7eSBvwjmJ5qk1R0pGNMuZgskEfd32nWOgTx5NvIBM6fTkLI3+uH65tgqKLi2FJODjtUQl6ZHhxcXUlhN35RL9I9ttpzH+u5JJ4pD85C9oPiMny+aZuU8GieEZFtcIJcOHLXlZVtVsDucC\\\\\\\/8sKQkLrqhiry0q7PdJbTSZ41VlU8z8H3PcnA3faWM7me0VmSmAbf02lto1DaXvjK1alNxPigC5seu8p11ylyCEdDiP4DZLlC0mkZAa++ImuTztBmpSE1Hk0SXImD7mXIH5wMEKFeTqWj4cRmJ5Sf5yVZIYLxH6\\\\\\\/E6hJsRKCBEitdBqc5iFPia1dz1KSK\\\\\\\/uZPdntPaoEwbTnu1ojdx9cDAIWVDP1ta4BVkEMi68GIMHCcpdkgS2C+Subdwhplv\\\\\\\/2xdghYVk7xfmOR0C7Nnu3SQjSZUA3ukp7IWgxO12rQkKO49Sx8Ip+yoHKOYzIniF2L9S6TLPuIM2QaxOR1qT9nqD\\\\\\\/M+so7NQAeIpExJoNTD6lzsYHuzHoYLvaxSumGKpzLN\\\\\\\/nFREsGhjfG6YdrQEA2pQwlXJj7CoCHxjIwgzz9Hht7pSenzGERsvmU7+BzdyRJaoqm6w76a+rWaAG63upwEgZFjsWmy+x0FFoVkwj58NBy0Vp8PhUjNnwTO9hQXAjVFe574QMqOG8F8tkWJPu8Z2pkx53PegWOYp8nXnUQchPt2csmqmb7IaB0Z1ksef\\\\\\\/RkKsqQ0OkxnLOjiJHzV4Irl0JCKTdmyTHsgMWVPg76v2cfwKqVTaNK8Gi45QPs8HdXvrVQcB7bq6z5DPcFkAJLj0zG3Nw+\\\\\\\/2TDbLUXwlxmaQftNawtFN77qth4HfAMRNk244ZD2EsrvOpt8tBmHVKXSdf0VMTGzlLA68iqo\\\\\\\/tIHrJbAtvPq2mbSz2vX6GbsbBWfhzVM\\\\\\\/k6A1pz+QVB5C0k3EMCEvtP+JwATrmTBpjGGpQnYOPUfhJ5m8Difka\\\\\\\/iKJ3Zr\\\\\\\/pGXVRXuQPhnftek2LxVlwGJlE2\\\\\\\/8dFrO5sqv3ML2+GzZcEKATelglWyyeXdb3S2p+TY5GBib6Mu4w4+JXRPr90rJx5oyeu30R4vVKzX6b0GuectGRHrU7pSG0kRU0OoGXL78LU6QDuCSD0hI79VeRqxSkuaCpGQDMyg+s1xVMDyjiQXw6t6cyiQGXOlRaZwuRp4beyy1USXqlheEfGmPKdEhKRsBYRe6N9NRSqf78NlfwgelqPqCOlUeOe7ZTDwORZINMk\\\\\\\/PqsSOnafKtJPrkknsiSVsZqb9XGTZr1yqAKjOEs6oFoOsEMWYnt4JVwx051o5uoJLkrX7ha2ysuK6s1arUAU4Iq3ornU3rx5pSr1hDY8dZvGWrsMyXVgRXmF87AaQ2xvTnCbaUSxjEPWR4qjdzVy\\\\\\\/haXlfP9kw9n8dZR\\\\\\\/4eg3TgGsvi5F\\\\\\\/xUHSsg5na1D5r7Id6YXTnA5ZtNPl50gjSxaT8a5RsZa0Pkikcg2KEvligStW9\\\\\\\/zMoNtdZWc3zz4IV1GmrsTJC8edTIcAke6AkeFQNJCXmfyhvQrwjXuA5ADmAZidJl75DCYjRTWA04H\\\\\\\/oqRMutLM6xEaj2EubhX3K1Eu2to9Vs2CIMHrJhhma2M3M1Opu\\\\\\\/cHTPzRF3xWE3MXPzhZUwwTAR9gWOjCcp+8PW8tslphBSczMBfIAUVv7bXRLRXm5Iru4TvQ5soUQeEiVtm3IgBYRdJI5NE98sQsRsRWe7lt1w02206o3UvsI7Igro7MVUbo17nCDrhz90XyD\\\\\\\/sudqe42aeTPWkuhShkVvW527wXFaVko3t44BXXoqaIT7IC392GxOi6NV+wTWCYlcfwLC8uZp0flERhPpTphrx6uWUMwAvOgchmBbDQkdAbRYIlChr\\\\\\\/5yREI\\\\\\\/xJhCqtSnB10YzUwIZaeA4hCQu6nlyyuCdJY5qyIU+OVTeQlDSk1XpE6ZI9QBe7d1IolYM5vZ5KEwipJitwvVgkvNnFp0+kZ95ZMD8ANOagzETGNUf8FGIY+NatuxZgLOBmArX1TOSYHM4gMN7QIhsQO+2saAnN9Vq2DXTn3CMAqjxQp5HuJcPn\\\\\\\/0A8eHQggLibdfNeomy8\\\\\\\/0S5xXro7MpW2qc+pEJC\\\\\\\/\\\\\\\/C+f7QVqvVvGWyx3kI\\\\\\\/IkgtHbp9WlDT9OWk9S8Tn23hKqQEn7xQArcy1Z8kiStZSDbsIVwuLB\\\\\\\/+09HYakYObs0W1FcHn23icw0e8Z8NmcWw170I6tf8HxFKeIP\\\\\\\/zzpz147Vlby6YvAcBx8d4pcnuK4t5WUTfSDeVsmGtLI81HVhcZ8Y3J113Vj+R3UoFDOpEEKBIMC1XpJ+wgYAxeUL1my+IHpe6e5lraPoK0bv\\\\\\\/40ZoIb9+7hWdlmjdwQqHsra0yJEw8eXGGR+7se6ZUc4EQ2QxACXbNATUv7bo0mPuVMZq6gn+M+OabTCw412\\\\\\\/Kh7cX7QTQy5Dk2LeuY8SofZvd62zz8JDb2hJmGza7tFDuWE8mwHQfoUXlr9GZj0NmmYBJH8XoTL2yPLBw9ZTG9GeaOBuv1F6w34GytHrIZ7P\\\\\\\/S7hKS6B+mUY6rn5ssGr0n3EEl2uF6cJQ+CuZAUXLxdrcQfe6V4xF1vXlZWSWaY88NkzeygwaVs\\\\\\\/l75HLo9irQ5e4+53aacgboKfL+sBbp3DEdALTN1e8emIPQPo\\\\\\\/QFkpoXsQOalXCMKLHQKSpCPnp1HdhQmMtVIgEg06SUFxOTMgDpx1OruJ435PxPzjQ7F8CUVFOvw8Mg3n4j+jp94+KVVzijb3cQCWmPK5p52+OBjjxxPF6a3BdVCCu6iufVTZxT7KjZTE1Jj1w74rub2yCUV8JJjRcMrPtyBHT+GV05uv\\\\\\\/OlzgMboqWFdDXpf3vxMkCEcqVIkwbMj+UA3k6CdV6GlV\\\\\\\/+rW3sfnvkUkYmVh\\\\\\\/fhtWkwS+C7DEc4wty6Paq\\\\\\\/ZVTDsFuPU5s1fs2x64b8bjwOlqih8o3+7YNq3ZY1oHEHPbJpjCxyiG845A7jAOz4GVxrTP2Sb7ksk5jMWhYTLbaTg5H\\\\\\\/2paDXpd+YM06SG1sbafD+0Ve5TBr4BVxRppVqRrA1DoYKKIXff0fz\\\\\\\/7rLaQeWUHr8ajvN5HiZayL7UfwnGh6c\\\\\\\/VV0ak\\\\\\\/qitxE19qETCSTk6qMAxzH8hhbIAeu23yWsEVXXtIe0uv1xePrSZkpTzfhuezu9nBkjhXTuzZXIossR+rTMp5fhvon5+45\\\\\\\/4ZkMFaPf1bvcOpGRyeU9hBzHt\\\\\\\/m7v\\\\\\\/IR6DrkNcFHpI2MRzhWshrwXKNXQO4QB43bXO9IyniTn1O3SOHNomCQwoyK\\\\\\\/XGqwBXH2I+SCmkbhhZuwR0f47K0KBB7X2Qo43+7j5AJzYSWf0uapQPbN7JKj3T\\\\\\\/nJxfhD7p22A3NnW\\\\\\\/cD6DfHGpssDIn5oNmaRiLVmkwo7fXRYFoJtLP1lCBV5a3ElWbrzQX+rXhoCdiodN\\\\\\\/6M2VuzurkEkQHTK2NjaiXRP73ww4bEgv\\\\\\\/LFfKWMgc8mmMjb0XM2RMxebzb6oVQ54UHtk6O+Ryisp94Bs3w1KgijaVaozRMCx4w+I1vn\\\\\\\/+H4M3I+UBMAXU\\\\\\\/d6QHcDkG3rx3XOeP0zrgJPhlM\\\\\\\/bpMAe0Eb1+jFoVeatxBRsf8xaXLD7zXa863LKRMnwPSONCZ5MYzK9z6f0jsOH2bFcKSm\\\\\\\/qvQJYXldwMVL4hDVsSKxR7Gbl5UO9TQs4Usd+Ae3rlpObYGXMhei\\\\\\\/yJf03uxljvOYP0Ku1jUsv60nvj7W58HteGICg7kSq6MEkMW7tivhB8cuehVfNY9YAw0Fx\\\\\\\/c0FObq0vJPrSu7xfjYBz5unhgH5ivoq734np26liHeaXaQjIVZyvQTVJSaMnQUYTBVkH\\\\\\\/mh7ioMIBjXvwFEbYm++kHuxo6yL3EzVdz5vqPxugWQmPRkJ37EAYPjetcMx6DFeUCcMwXXJpkgMi1xBwSxAYpLFPVn4jTYYV+hjCrDUFUjwNRHOTUezcXY8p6ENVR0L8tIDr045uvKWGnzRYfOErFBj49l2fasZbW9QK+5dvs4R8VLccKEm9+8vCnHgs6owdZZ25LERPSOPitgOCeLcQTX+8hfUvcfsbvZXflwHSSq7yA6myVZ44CE4AXoy5zIPpk80KUA4hIKqBRaddDEPySVIKCKexwSKE+Y3YKMxe6x6DbaApj7+SZB9U14OioOq6lXWVtlzB3E4gy88uKXAYJP\\\\\\\/yoJK1TjuteDSx6veSE40VVbn8WusCspRzEHpG6fINQBZVL5U5SKAZwr7DxCK0MPglPtLm6wVGImbhGzqLz0uhBYIkdRlfwAyBrtUmh6GPiVU41wOJmlLwG+9NLr36W7YfCX\\\\\\\/qpLRE5MSSbzv9HoigVXVCJY8Xj686\\\\\\\/27IHhMIM+NeIjtfGIGfwVVc1LW1Dia2JWp6DclDli6+TigDdUzo53fWXKKsAykVdkJJ\\\\\\\/nq+QvLiKqQ5EDNuScxoXfRAP2dELFOzX749URGNwwIi33QLfx9a5RERbp8eQaO+eymRtpWA6JYi2kpDc7QYZb+\\\\\\\/cUWTivddACmfHKE+\\\\\\\/Fufy4ENrE\\\\\\\/naWyre1A9ZifA9elDbiMJQkgfTynLVyRH5hqJolmZXyDAN4aaAjArJdOjhbNCkdA4oSmUf4\\\\\\\/q\\\\\\\/LG6lb2RbgFrM9kc0PNMGLZBz5pVFYWU4P+zK8jwkDRtkpYdeyShvMXCLCH3dr4+o4cSKrOAype20qZGqq1fRacskps8yx5q2SNtxvPR5zeL6DZ6hMqybMlHQcgKxMiJqhtdg1jVLHIB5\\\\\\\/npL07vdUJ4o6XJzaJKlb9Ks\\\\\\\/zq2gOiwNnzAtz\\\\\\\/cmHZZseL\\\\\\\/hFigU3WkUHrsTKY7RpTbv3PfRfadmSgINjqLIxzKz76ynz4S21s4PRqxS6q\\\\\\\/45Dq8kIPdQDR9DBqsuObD1fZ8vdl99O9gPDJtZRlJpm+03eDkxwNvoG2r4uu481HjkOjCEW3rmYHJES4PVwuINyOGtAdAGMvXkAOKBiaUb59EiAJNSefbLIaTgtwkVunvryvb+O9NupCOZ5\\\\\\\/15c5hxYLpX9G1Rljn31JCanULeBq87hhXtylS0TEvkRAJg\\\\\\\/mlWxB1\\\\\\\/Thsudh\\\\\\\/R3HsmzItysh+QQFwYZBvjqlJYwutEnOIKvqoXg\\\\\\\/Jjtzo\\\\\\\/G1Jtp5iO7qQIpsIadkG8tct1OG0qGcjEbToYTUgdBZ+NcBuDjKXgbjRnUR7L3chnFUTI4btDwdoKYU40JCj5mvNY931YqfYtDrRvUYPO0+\\\\\\\/udbs9u67yy33arAocZpzJjXsDdp0kUpq3Hl52mNmmev8YjOae59YTUkjriT3qab+Ozb124HfAIPSRkxCjPQmJGv4zchzqBAmsgY2wJc6VJap\\\\\\\/76GM\\\\\\\/+zwdF6P56oA9jhP1vnoPmb1DuaqmQaRRB\\\\\\\/Bdh5H\\\\\\\/ZO9XclulPE7FXhL+yfVwuROqbXkBuj4Wy+WH44HNBeoP\\\\\\\/ihzvLniCbpkT5OLSlb9YCSuw+mXrXJmfRFB4FlazGWu+Oe+teJJT6CrI7JwY5UZvimhhK8gABtWJTDyOvxAiALxGHTfwfIiTip2hDoG1UuSN971c4uCln+RoRI9KJv03\\\\\\\/elMiw6S3RgLg7WZaNfobp7s8+hPAr4ArVwhHPwL+fWKi2jD00\\\\\\\/WT3WxfrjGkgu5qKSsc9yiFbPbXbIHDDutFZ1UCd\\\\\\\/+R4wmRE6ymX\\\\\\\/36\\\\\\\/q6JG8O356eEY0gwtvr4QDEBxGXhclZjQsXn8nJqrOkggr25FpOTokWOnTnSeRZFKn8bTVU7DQcTSr00BuFFsXBFTcQZPYaiMWypXvEUfpEvV6h9GJ7jKiC6AyUY2Uji59t3XRksPOZS95wEWZANH7cI5BYkSkH8s5OruhDOAuURu+9pjVdyVHpvaypenrvWuhjHozV6X78+5Oo7Np64F2v7RPe10ZRYMy1lttsgj+q7WAxwe58Ae9kuaH9+OAXxOMuUTPg2v1khl6Yf2IHmMzcsMe5eeqVtkt0yVznD24C0DKpuBUXT3xN\\\\\\\/M9cBW5kcOT1OnXNnv0kvWZQOtJkCSJJRjBI4SXjZEyleMM7q9AyV4Am1Us6UlHGwODHekOff7rUUgZnn4ozSjFD6nHLbrw4TU2kiBCI9YjTI3vBF\\\\\\\/tArcojUqlLy9oHvav69hufRFU0+erFc0EX3Azt5KtZXzAM4wUGyJmoKraJ+1hc+Og4TnBL88Zk6OsmGMwSU1h6EorZjKWTr5j6cmtS9AUg9vKa96jZoTdeiu4t8JKqtreuhT48AMR\\\\\\\/2r7ybGZlIa9tU3xzTccn6mOEmBXk4nV1eQK87CcCYxMJd4\\\\\\\/1QPH9f185B9qSfC7jIw7Rt9PH6ijrVSOUcKbzM49tspwt5G5ukc+iy34Ow5DdZfbvgUIxVQPoMMP6ei2BVCdhIGPbrkswYF\\\\\\\/BGAYODILQmCY0\\\\\\\/AgqCnN\\\\\\\/HabwWrtxwg\\\\\\\/ljfjNNRu1Bl1wXiUJesNeSoBkiAJkB03gJHadNH740hgvfK98ceO8hgh1e6cLjSP7tvTWB3tY66WtDFLNSlgOH+C4bLRCnbu6DYS8vtqHm5fSf\\\\\\\/EO\\\\\\\/w3OvhToGuGohekJ3seniTk4NJ2GWq9iqL9N1f+t+e8ddWG88tOSPHazfilXqxot54WBiSJIYIu0OefGBkuiu+BmNQcE2O6svKeOz4kcRmBSx1ZmRZb+6AVIU6k1NAOnWGdOraj7IyNd+HK5xxtROTiz2gis66IbLRKvHeLCg8KD3FXg+X+ta6wAp7OZ4IY0G8+cFj+i3LCa0Jp4DXMUTU6LqzhR\\\\\\\/phLl3rRDXVxoIXlMe2jY\\\\\\\/MVaBxlGq1hfrSNl2a6WGHDP+npu6IonHODAx7nwaDMODCgR5itmT47PrEancqQOe8UWwYh0sM2J2Dy\\\\\\\/yb8aI1el1Rq06Etf66prtJ94yq479ur+quNsNZ6HqMXps5KI8PP4XrBjmZkvxSRWnL3lFk4dT5blTgobdv6WT+cPpNVr3Xn6Qz+ICRNk2D\\\\\\\/s6G+p8MnZ6x3WhJYqkAxOyptcWoyG2\\\\\\\/9xSReNlJqeV3ETLnVulDRwVQ1jqlvGCnqJFKrvgfAZXRaLjf7jVvg5Xxxo\\\\\\\/3mxcxYhYakyo71f4fQhArbvTjKc282VYzmkJBTh1e0T25K1EtqCGzWfyJFliNypwiC21vuuEF1uCpbALSy+BB9n8xxrLFEgVCVD9BuX85flVveBz8lhuHqPie1\\\\\\\/wDAIewwhpnAQKuMFRDIhCPmY4U3J9M4r7G8MnmeQwHGAiZodDVIliNg27EPO1Gh5NdTE0Vxu41tIvETBC1dleV6qajlCDMvRHmpLgPUZBfv\\\\\\\/3+i8KOT4CFOsuQL6Dhe4\\\\\\\/kHqTwQuP8TwyBtAfs41wnLFCLlGcn3abWyGc1eqWnfv7lXi3pA7l2Q9fTnE8sozEgudB5emhzFPAEC4kwkUhZOOoB2nj1J8a9E2pq6hRIjv+51g4JkWzWiK55pXsmAbFHLzEjiRuaqu03q\\\\\\\/i\\\\\\\/GYhO1CL8RIOUlMVJ63LtWp1tyE5M9w8SN4pApi12QhcZ6MKZ0BT214o5c3kWQmPGZ2MmvpP5rmHoU7zOhX5x1NT8FLpiL8e7LS8P7mCHTAFFOKioGR0FRtrqy19dCPqC8qG7SS3\\\\\\\/MG\\\\\\\/2i6UjzDu2TZJyvB0RRGB\\\\\\\/8gWhDOb0CaFbgWHHo1qJXGWrBywa2+Jwmx3YV7GbxzoU5IvkeUrDv1SMlX\\\\\\\/PbVy3ZtcAtxp0nmN23vx2uucgmcEt0y\\\\\\\/AYQ27IxoR7p0Vk\\\\\\\/+aiI\\\\\\\/4GRaJCOaMZCLtcP4Q\\\\\\\/SATkocHVEigSLAgrSqycp+rghRJwF78S15MdleMs23l07zQ5SHpCUhGfB2LJnmrrXDXBdfo94MHz6RiESkUeL30U48r9Krax5yz77A1C3uFwg5mp3N2y1Vfx9yY888z4oQfm1AEuhS7T3KUIJypscjrewuk+9zpSaDfNDKvPtgj7ce8FIFZeZPcRV2zdD+XomqG80B4TLbbu6qa+aYeiS2NKSFpxHmoB+ISDh2UtQLLU7b9aFWLF2l7JGbIgh10p3j1i07Se4kSrVbTjNsddSpug0ohLZqEFGswQ1dD9rX6I1UMkRkHluorWEzFxiamMBvJ+FDgyTu8zWon894LVp4ZNqKNzQj8QQff88YqdJxeETeYOttYcFhFqOmiMKFBqDGSCJ+\\\\\\\/dHpg41Gw9H6J3DzoK6dcjNml2yaJmBkPOAnl3bO7Ie9+3zBYK85UwbX6TV\\\\\\\/dla3Que+uM\\\\\\\/0pKKMCiBcSdme5u28AslilNi5ty5V2mbiniQmUqPEwE2DnWUjcnpCYaqykniOnSKhfRTPCPkSsKW\\\\\\\/GLWZhJGWg1o7TpAshBBZaxsgaVQpWQD5LMa9szanuwhmVURrLL7146u+0VKNSOR0ADp15dekJ3hxIZ1ClcEp\\\\\\\/Q0RWdTjw7GukVrSuSuQdKxMTs8mnoo42VAcK2pX4eJxsVz\\\\\\\/JUc\\\\\\\/WJJkBAWJeDYesF5YsEdb\\\\\\\/JhymKpQSIFsCO0iUJKr3tuFmzEQWjfIrpFs2vgv7Soq3MkBR9R3QCGcTfA5jY1ykPQyI+3W2SJwnHfGUSWWOjlhxz8anDv6oTptOTEnpRx6A+NT0TsPythaSdKsT3BGVEn2y3nJs4GKbpZNhm\\\\\\\/XsnIa6G3mJLtfwWUXuxTl+OKgSii5KGPsxKdlThHG5OOCXS6VzauNP2\\\\\\\/Z6jq2QNOMgAVEjsTNVKofaFtzQdx7by1nJPmVpOD2OsBDxdclz++fKJ4z5h5FjwlPxa2PoiBFnYTJcdmW\\\\\\\/M7QjBaEePhVXQ9EEWIw6qEYm\\\\\\\/hpVEOBhbfMU2inFIpa8YAtUbD\\\\\\\/KidVIGb+KGUM0PIZ90DVR\\\\\\\/ynXrQ9AS02dj8Fr73A0mWdDgJY6XN+RGRfm1sN42SwU+zgqcgKXi9RtgbzI+qmqAYbzTr+24kd7NTs5Gb3q3OU9wkKAfgi6j5Sf8dLvdnWsJzFfQfnmxKq\\\\\\\/cApXZK1IIYCFQNXJrVKfqOkCHQYxstLESBNKQ4kw1GGnXI5XJip3DCHsWJfVT1behwV4eQam0xEr4u8YyJdLClqRcJWZ23M5LMgZ8dlIiqKsXMbstkWEFc1yrS93rCyT\\\\\\\/9aUfVqhMTh53Avl2T+ZyaIuyrnmMd8ktp7ATFRh9+NvhwqA3hRNos1pxqBGMq3CVtK3kUrSgtIh5ZSl+GWJnPgeKhp7K12KZTnr7pD9Dp7HcB24JTFFAD2DKXc2S1TgypOESS\\\\\\\/H+X\\\\\\\/5apeeODH7V\\\\\\\/sup2g1CMAOCc07TtA0jAYb1fMF9vEb\\\\\\\/YhP8Hm4zrbv7bAvzS9ysTDeeeGuI5JuQDNeS2TGN1PuOagKJcMQxDv4812f7W2X5WKUWpdv6hTRX1KKt+i8bixpyDLCfP8CV75XF0adqCMrIT1ZC0f85+BM1LilY0ovd+fRtsMTnPveynUqRSbsA3\\\\\\\/\\\\\\\/9Dw4Ee9bM57cCM8lgv9AFNdk\\\\\\\/oFU+xA8YnyOMpReXeKK7pEaZ536u5Fjy8xo5p500isb9C9VIKuNq8s7YwJZIelKHZ2je4iR\\\\\\\/oVPOzhh9jPe9urRf9ojLx39Cu9vFeDqiov3ZSbowRyt+XQvyaKvYVST3Otc9edodkTaTPRzmWPwSPOtFmgJV0Z5NUz4wbPshiLJD7KrKVLZbJUkpBCkTfFgUJJ40vUbQiziFprH69uHJfPK1TWHDN2ouJZX++nZPaRJ1sQrOeM+o\\\\\\\/6eTg72iL53ZVUzW\\\\\\\/2Jwpp5diIBT1kfriy9iLEJ6DKrPTNMbCFTpeFuX+FtwM9TD5DuxXlfcDYUbaYp4\\\\\\\/Mk2uRP9VKYKPJzrQbxgiBdKkc5sb6h2+NpPH\\\\\\\/dZowxdFsXA1r8OME1KxBxGyJj88bnt5RARLe0FYjfDBBN2\\\\\\\/1CRId2wBvFBMEa8ZLSUzIA8PvOv2mw8VlWfo9n9YyTq83XDc7TANg9I+Kd6ycaZXoC\\\\\\\/kBxb7lYahdu3pxxmi01MpvEx4CulCZ5sVXxSW7HGT3VIj9m\\\\\\\/y81KlNo49cgFVu2fNW0AVkJbCPgvFfojz\\\\\\\/hoKmET\\\\\\\/k8tMXXhcUZT+iUjoLahc2SBqHh5Ar4klWjHnq6\\\\\\\/13Mz6Fue07b1fv0kXErvSRmOAZi9zpvT1YYofULvxMwNp3cSO5KY\\\\\\\/lrMl8OeE+PwFxeltpLeclUECJFPofehYUHM5muO2b+A4okALQbrh92PjqoJ3caSFlst1PE\\\\\\\/PXSzOrZ3H300RfLmZJeX+eaBOeqYTPVMocqjhUA65XFh\\\\\\\/uXRn6eWWzBfMMDRwQAvR20E2oSCfr53ZWXc0qnSfw1Bgltgy0WEfBs1z3wl2ehzz5WEMN\\\\\\\/RYK8RlpDNW\\\\\\\/5MZs6OBHzAZLE6YSDMvr3dvxJH4nb0WeT5LdqzwlKeSjjU9ZRpUwLidf1mUZPMkmgp56NmB40XrN6by1hY7cBxsrAdcBzCsqwPc6jYnpmFo4OaP7z25xJouZL+rkA88FXNiEjlmHOH7ZQX8nncYrmToxyvBa2w8mfIs12NMqlI92Je70m8aDm\\\\\\\/bUYtxfHlSgoUWuuKZ3llgFafZAm\\\\\\\/nkPDEuDUCWPXscam\\\\\\\/iZw+kOqqbYrb2OzWIlNdOaPSIktylifAXoSIjxuIINuhwGu38YpXsBO7ZGlq54zmDj233bsyfWsWwnHyAyL+\\\\\\\/Cn0MdnaAztz1zEoWIqyFz8ki4zhNaOQ29s9GPzpJouqwfJP4xTPt7e9uy0AquYUsLD6\\\\\\\/gZiAXpvw5lZxgyWrKQd0AlfA3\\\\\\\/hblhKJDw85jhZcyX5LMPbKoW51o0ESHKLnrs10ytI0HC8u+7YkNJdIEd8NDn\\\\\\\/oS+gjX1vVkGU\\\\\\\/q5T4jrJ5utBu8rr3KL7jx41ywCPELYn9lYYRemjYAuuTrHY2e+jHfhn0j1IfWh1oARNwr+hmPC5lRqWoaCWa0HFClwe07NkFOtOb0PHyxPSQmPOyD0FYIlCGz3KxGx8\\\\\\\/QTtdrkT\\\\\\\/mKqXv6Kh\\\\\\\/aItSHrE7muXPG25nOlcy9ycMrKo7CpT9PObquq49NxulW01jWqSY5u7xhcwkt2P5lCrgvzGWsqbOylb8lZvHb08QeHyC0R13\\\\\\\/1NGMxf9d+EyfO+f52kbt10ROjxFWRL\\\\\\\/ymFL\\\\\\\/HjlBqaLCkauxebASAA2meCHnAc5E+hLABdBsKvoqd\\\\\\\/4\\\\\\\/Gzjy794rMe6uK1OdNpNIvTQGv274eexs3sN8Smw\\\\\\\/TJICJDhMfxswhdxJ5AvgHIo5qPhbrUIg2vFwANQWV6FhTTzH4RCkLQQrCurE7Eaw\\\\\\\/QrtVsYRe49\\\\\\\/o\\\\\\\/Bd3JbNx4rxH9rm9+qPftwNV9QaoWYMn5JF9LbRjcxOb8Cy60qOLCNrbhIfz\\\\\\\/SodDXcrLTtJlLw9obcwIx4LsDIr4H7uQHS0tCbjx+kcdX3f75EpNvAFr0gV+GRNwrCiNsI8ahelkBfy\\\\\\\/VE7WZOzP7Zc\\\\\\\/EhwVfwArkUPyRKDDhBtA6tOD6aT7J7lPhfM2wBU9JWqnxNN5X7gVHthh6IMqKWC54xdKk+Yc82fHMscmxscIZU5VZ3LU7Ysny+3ly5SQ6P7kiwja6Ehw99ThImGCjktsLJXdFIQYH4eM7yPSSPTUP66x\\\\\\\/RhMICGZfg6OMQqZhAMJaveYFXD7LXDZM67bREGfwHYi3dS7pzkQyWHek\\\\\\\/aub4ala+CnhUs5vH\\\\\\\/+2rM\\\\\\\/YZUso39E9m\\\\\\\/O+Su5TCIfsU0fBmgkIc9JgtmwQBdKr4ScPV4nvGy+2wZEQDIydhCVXb2dEDczCJC3FE2t2tGZKJANiJE\\\\\\\/m+FmnMGzKFUn13Pv84+pB+F4ZeVYSYp+5vMhq4mZ988ujoCg\\\\\\\/oQJhgqIdbh4XSi659qdrDoFUliHokKK2BzRzO+H0VOSsg7r7+5nLSG3A1xUiLAnPfjgoRzywmD2u4LjcZVlpptIslEG+JMAQpzCBStF\\\\\\\/T7KuNaDwCgOodlpRlROxOr4fcj3RSYxlvX17x2lQnteMX2SoiEAE7qzJhvGmdsXmy7eTqpz3uYKV9cqGgNtgTv1THIBHUH3wl1fNVhIBkTtAYjXFQJpu9q\\\\\\\/ZcXD3pv44hRC7w8RKqhNyvB8GFlq7XKiVP7p3SxwdYja5djrFhZBVoD5K74Z3b5x5iRfy3ABj7xXUYIbulaKlKX6HJM4hdmNEzSLix\\\\\\\/2+g36\\\\\\\/27j2\\\\\\\/a6w8j+pcJjbP4lu5PaLnNw3hMvxIDcAqtMK1pS4o6OYLyr9ShfM+AMSw6RkMR71DvdjRxbYn4w\\\\\\\/siCMBMj9pSFMOARC2ch0IPOkZ7+9SesOrNulg8c1Die4HANrXdF3XXPgfiNQ\\\\\\\/isqwkwSjG364fgeKTB0YGADRpcquwusiu90M4Ba\\\\\\\/725BQsEV0GgXlCx6+RMY01FRe33XZVs8Egs7ovdZc1yyv8evuYeGqcwqy3iX3HiD5hspPVpkJzJ\\\\\\\/TsAf+U7rR3K7kYpN7x3cREsUwMPRNBlj0XROVsuYQxiI\\\\\\\/z1OYCH0Cy7\\\\\\\/xioD\\\\\\\/p4cObpZJv\\\\\\\/TwBHt3McU1hYpQooHEM4MH9LcyUM+vzsXHLVeO5wFdZLtaSH7fIplXvcijzRh9NdyQBV0bJByBEcWb\\\\\\\/KTouqHcX7\\\\\\\/64wIGi+0RURxzHaKjY740A0JPflveYior0UmR0kGnzHAFUwfN1K2+wie+5kma+qsNHpoRAaz5BVW06EKeC6goGD2GUupf6GWUi+JoSvbCE1Tmb6AdJYswfNWc8bKxhWpHVvfC7ItCSFHBKSp3sB15slPip+Prtut\\\\\\\/ArrzNSJftSqk4sGS3LCzTNaHDTuh8n2ECaJl0+z0jPU7Selkqmr\\\\\\\/3AKJl7d6\\\\\\\/I3zR0mWTVpg3RzOQpkXGA47BYkuaz1\\\\\\\/vQ6AImTFYKTqWoFE3buqJd5UZDwE5NFxXKgwI5QN2RKjIoJB3enTN6OXMLsl5n1rwJbOgvraco8wo2QVkfGmL+gVL5uplBiocgoUwRXtkCdn5UjIH03UrnH0HfPzaqT8tGy9vriwJaj47IksmxEhR1UR+cKBmPdzvf6zCeC\\\\\\\/vafMEfiy\\\\\\\/mAXzwqM\\\\\\\/gtarEJkblFGyIioSLC9qvNO7D3ziqA6YoaYF2cxaxwE0S0Qaj3AlqDN8dRcit0vYyWqgFXNW6Nut7V3g\\\\\\\/1cS2OlJOtrh8g\\\\\\\/rFEOBZTLHeqfBjJYWw2G9X8nn9GPtq6JvC5JE79odvxIXU+NRTzF0ZO1MvTnTBYh0uVu8v5yjBwHwYJb89fAQ3Ea7GiJvcIaVECcbiQTRKunSLKkkaCLZKrQWn0Fxe6uhCb1lWSz4pHchsaBunVMv2ibg8z33hxSCWiYiipgjczPbm53kRCUHLY3GjKFEu9VoC1ss3CoKF84I0cB4uOK3Z1zSvjwVNYnxkBZx+7lpos\\\\\\\/XUuo3p3tw6YjUC2y\\\\\\\/12NM89HCHZqyTBzM8tTbofdeuJlspt\\\\\\\/zL8VXRT3G9\\\\\\\/CUjgnyw783JDDGSr3E2a4NI15K\\\\\\\/\\\\\\\/QXmERtk\\\\\\\/BFBvw5cpyGJJ6w+imOJA4bpAZ5PzRTAZAJYsW8fQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022gLTjVCpT0RkflDfR9fKK+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002261b5bb900b75edec3bc5720d5395ae5d7300238792959106b0e435ee65fb21e5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022giqd6DCH99m8FsNo4FvJUA\\\\\\\/TYCb2ZIpSrJkIHTshb8I2yb+n4eNn9x3zoXJkP8dqhQqYi0xkH2uDEYSf6rg\\\\\\\/a+xI79+pUjHop0LkLvlDHf19diauDilVuP6l5ox6pzOQBjrqnv\\\\\\\/q09\\\\\\\/90bdrXR1i6+64ThseQNVlM+Q5Yxi5EazCG1YK\\\\\\\/jbCLEOSUnhNy+ES+\\\\\\\/T69tOH+FHDZafRPD01c\\\\\\\/5ed6tObgm785uoSfpLSINX1XK3wSDk6fmNuXP1H\\\\\\\/CyX9w0P4JMEz07BudzrKEHTHjQl4Y35ZTs5\\\\\\\/yGuTeukzGUfqhjD\\\\\\\/k27Yjdrn2JSEQHWz4WtFuhVKfvORSNwmHLXfHBjwrfSlXLTH+SWiS7OLhFMoMGmxHlJAdDXvdE7xTCtUAV7dRqSMmLBNBTARlykBn37WRTJyeACQzmf3Lz2R0oJ2Xim0ffAl\\\\\\\/IWZIE8ncmfv9\\\\\\\/Z2N4Tz28vcD0lNK8vcIVBCdeksmCqHVHmGlHjRF+rc67sLxLy8jUBt5I4J0PohRobwGwBCtSBh8PiN3f5zhohczapOFzW+NI5A4AQ5KcDjbfJhuh3hd7LrAYyzluoG\\\\\\\/ZSR1OFCdM06rZhpKAI+6teWui0eEze\\\\\\\/3AC7L6S6JDOSksDNX4Hai4mnu4oyYzhxWIwaPC+66rdO9H9u4\\\\\\\/oNbk0xC\\\\\\\/LkGhSERjimoinghSIm2C6ilb4TZ+Vj5frQ\\\\\\\/AbHf19leTOdQbb3qrJBIpiWuFCYEfxf\\\\\\\/zYWKi+SmDKdeJQKYULlJJFupgGlWRQyA7z8u9n7OvojvzxheVqA6wEnVma3Fn6egerAnarGuETx0nUextD\\\\\\\/YOjvW5AwPVY+7Pnen3I\\\\\\\/7VIuPQbAjhxmnwwhI\\\\\\\/XI\\\\\\\/DJ+QCX4bOhNm3IJDRAzFJMZzFyPgmttnh+2irwPfkAza5\\\\\\\/1KgG1G3Udb3sqYugqOgr+CbxAk1pvfsE1fhmmMSDCedUo9PB5SrDo3I4KMGoV6aTdljjobR3AM7a2MwZqt3kKCn5MudGRXqX0Kc2TalLqvf2hJJK+2DDGVNzzGE3KbaL+DwUlR2dudJrOVByKQAKumbzGuCR+uq6aYNOjDp3hzdWY1ugUW5qY26jGs4Q+IJs5V7cQy9B6udu+h98cYG4k0RKtEAA\\\\\\\/OpgnjgneYwo+MqYB+MkGC5lZzPMmdRSJ290UeJA9BJ+1oXkMSnxTSHeNahy7rvI9ZGT09a50g9iY\\\\\\\/JILwz6HSbdhE7YaBRs3VcfFeJwnK4KrJYfZCNhDl5tzBPjv4VOS\\\\\\\/LxnCvERyNAaL\\\\\\\/GqiBcTR6TGop6oAMYqNkTamXm4dDw22ms2hDfnIoBUfEgztzwE2O8wp+cYAsm8svj5OzvYVYk7aTVPVyZU3KXKcKASctLMAhT+xF\\\\\\\/gOjJ0NcU9Qa3DjQSq5gbcqumf5togpMnoZwskte1XTmkmSMzfSGOP6+nt1lMlkqXtENBzPGblFNatolcL38AVsXo8foJRXV383JeGFj7pjMKC8e9q9uyMw7nh3dElQTP+0AI6sC+NKKXGAgm9kyjNqAKZBR3SC\\\\\\\/JsXFRiEorRMmfLxVoxqKrLuxZsin76fUeBB\\\\\\\/29c+J8NuiGfz316fSbehL7aajx5daWVue1K46ML+qq9OwI3SWfCn1ukADv0lCCRMs6tfZjcNCdQemHJACVUU5LTekHfXNIc9za2R+yxhoNcs22E2K5fHcPGbicpZOiylYyuLdPW7yF4ytkTCAnjPekeAvJnTQuo19h3f2Z\\\\\\\/NRqwYePJz11U7\\\\\\\/kRz\\\\\\\/wBwapnAJYrxWyruQ6gPHrTilYOxkj6a6yFK6Ir99qmonavywKRC6FKsumxagJsI59t0YjoR99dBQWVsZ0FYS2FydHUBvisbj7QnuRUdowUyB0W0icch+EMdctQH3evMI6OUYjPgaITbOhgI1iTNro5qttPRhAzNaj6wl5RWOJYIHlqN3i87j1EZ4esmmf0rYQHnCN\\\\\\\/kwcW6rEH\\\\\\\/IM1H2Aw5zhAKtH3WJ77qzBr7ELS+hAJPY4Io2qFzEH3Vl7ptebBdbirX0LLDZji+3xvZtIY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022aqh5wJr0BVmAH+jC7Ruv2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223e0dd8dacefc252896df6d52444feb31b4c868637086a5bd632719525c034ebb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490447 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2805" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JEMR3d6yPCgbLMYxYBqjeQKJzDvTLHFbPXlGuJrv6v8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2805" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5Hht1z4CqnQmzIP9G3D_UGLYhTJghMB0Q9ayciTQysc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2781" title="Cotton Late Embryogenesis Abundant (LEA2) Genes Promote Root Growth and Confer Drought Stress Tolerance in Transgenic Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2817" title="On the Relationship Between High-Order Linkage Disequilibrium and Epistasis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490447" id="node490447" data-pisa="ggg;8/8/2805" data-pisa-master="ggg;g3.118.200367" data-apath="/ggg/8/8/2805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Débora Y. C. Brandt</span>, <span class="highwire-citation-author" data-delta="1">Jônatas César</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5318-7601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Goudet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diogo Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2805-2815;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200367 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490447" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Débora Y. C. Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A9bora%2BY.%2BC.%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20DY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD%25C3%25A9bora%2BY.%2BC.%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jônatas César </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B4natas%2BC%C3%A9sar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=C%C3%A9sar%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B4natas%2BC%25C3%25A9sar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jérôme Goudet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolution, University of Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Swiss Institute of Bioinformatics, University of Lausanne, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGoudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGoudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5318-7601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Goudet</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a>
<a href="mailto:jerome.goudet@unil.ch" class="" data-icon-position="" data-hide-link-title="0">jerome.goudet@unil.ch</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diogo Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diogo%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiogo%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diogo Meyer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a>
<a href="mailto:jerome.goudet@unil.ch" class="" data-icon-position="" data-hide-link-title="0">jerome.goudet@unil.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2805" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490447" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2805.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490447" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2805.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490447" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2805.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490447" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2805.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490447" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Balancing selection is defined as a class of selective regimes that maintain polymorphism above what is expected under neutrality. Theory predicts that balancing selection reduces population differentiation, as measured by FST. However, balancing selection regimes in which different sets of alleles are maintained in different populations could increase population differentiation. To tackle the connection between balancing selection and population differentiation, we investigated population differentiation at the HLA genes, which constitute the most striking example of balancing selection in humans. We found that population differentiation of single nucleotide polymorphisms (SNPs) at the HLA genes is on average lower than that of SNPs in other genomic regions. We show that these results require using a computation that accounts for the dependence of FST on allele frequencies. However, in pairs of closely related populations, where genome-wide differentiation is low, differentiation at HLA is higher than in other genomic regions. Such increased population differentiation at HLA genes for recently diverged population pairs was reproduced in simulations of overdominant selection, as long as the fitness of the homozygotes differs between the diverging populations. The results give insight into a possible “divergent overdominance” mechanism for the nature of balancing selection on HLA genes across human populations.</p></div><span class="kwd-group-title">KEYWORDS</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/hla" class="hw-term hw-article-keyword hw-article-keyword-hla" rel="nofollow">HLA</a></li><li class="kwd"><a href="/keyword/population-differentiation" class="hw-term hw-article-keyword hw-article-keyword-population-differentiation" rel="nofollow">population differentiation</a></li><li class="kwd"><a href="/keyword/balancing-selection" class="hw-term hw-article-keyword hw-article-keyword-balancing-selection" rel="nofollow">balancing selection</a></li></ul><p id="p-4">Natural selection is one of the forces shaping the genetic variation within and the differentiation between populations. In the case of a locus where a variant is favored in one population but not in another (<em>i.e.</em>, in which selection drives local adaptation), we expect differentiation to exceed that under purely demographic processes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lewontin and Krakauer 1973</a>). This is the case for well known examples, such as the regulatory variant that generates lactase persistence in adulthood, which is more frequent in Europeans (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bersaglieri <em>et al.</em> 2004</a>) and variants of the <em>EPAS1</em> gene that provide adaptation to high altitude in Tibetans (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Xu <em>et al.</em> 2011</a>). Purifying selection, on the other hand, is more common and removes most variants which would contribute to differences among populations. Therefore, it is expected to reduce genetic differentiation at the focal locus with respect to a strictly neutral scenario (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barreiro <em>et al.</em> (2008)</a>), while differentiation in surrounding genomic regions may increase due to the lower effective population size (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> 1997</a>).</p><p id="p-5">A third regime, balancing selection, is related to diversity and differentiation in more complex ways. By definition balancing selection encompasses all selective regimes that result in increased genetic diversity relative to neutral expectations. The increased variability can result from a variety of processes, often with different underlying biological properties: frequency-dependent selection, heterozygote advantage, selection varying over temporal and geographic scales (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrés 2011</a>). As a consequence, the expectations regarding population differentiation under balancing selection represent a challenging theoretical and empirical question.</p><p id="p-6">Across human populations, the loci with the strongest evidence for balancing selection are the classical HLA class I and II loci (especially the <em>HLA-A</em>, <em>-B</em>, <em>-C</em>, <em>-DRB1</em>, <em>-DQB1</em> and <em>-DQA1</em> loci), which are the human Major Histocompatibility Complex (MHC) genes. These genes encode proteins that mediate a critical step of the adaptive immune response, which is the binding of peptides for presentation on the surface of the cellular membrane. The HLA-peptide complex is surveyed by T-cell receptors, which may trigger an immune response when a non-self peptide is identified (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Klein and Sato 2000</a>). Balancing selection at HLA loci has been strongly supported by a wide variety of methods, with evidence including an excess of alleles at intermediate frequency with respect to neutral expectations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hedrick and Thomson 1983</a>), higher non-synonymous to synonymous substitution rate (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hughes and Nei 1988</a>) and <em>trans</em>-specific polymorphism (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lawlor <em>et al.</em> 1988</a>) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meyer and Thomson 2001</a>, for a review).</p><p id="p-7">Although balancing selection at HLA genes is well documented, the evidence from most studies is compatible with different mechanisms that are difficult to disentangle: heterozygote advantage (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Doherty and Zinkernagel 1975</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Takahata and Nei 1990</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">De Boer <em>et al.</em> 2004</a>), frequency-dependent selection (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Slade and McCallum 1992</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Borghans <em>et al.</em> 2004</a>) and selection that varies over time and space (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eizaguirre <em>et al.</em> 2012</a>) have all been proposed to act on the HLA genes.</p><p id="p-8">Interestingly, these selective regimes are theoretically compatible with both increased or reduced population differentiation. For example, the coevolution between HLA and pathogens could create a mechanism of frequency-dependent selection, or rare allele advantage. Under this scenario, rare HLA alleles would be advantageous, since few pathogens would have evolved resistance to them (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Meyer and Thomson 2001</a>). Rare allele advantage is expected to increase the effective migration rate: migrants will often be rare in the population they arrive to, and thus will be advantageous and increase in frequency in the new population (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Schierup <em>et al.</em> 2000</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Muirhead 2001</a>). Therefore, this regime of balancing selection is expected to reduce population differentiation.</p><p id="p-9">However, there are also cases where balancing selection is associated to an increase in population differentiation. For example, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de Filippo <em>et al.</em> (2016)</a> identified four non-HLA loci which were under balancing selection in Africans, but positive selection in Eurasians, resulting in large frequency differences between these regions. In the case of HLA genes, selection may be population-specific, with the different sets of pathogens in each population selecting locally advantageous HLA variants. Under these scenarios we expect an increase in population differentiation.</p><p id="p-10">Evidence in support of population-specific pathogen selection for humans comes from the finding that HLA and pathogen diversities across populations are correlated (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Prugnolle <em>et al.</em> 2005</a>), and from theoretical studies showing that population-specific pathogen selection models of balancing selection provide a better explanation for observed HLA variation than heterozygote advantage (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hedrick 2002</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Borghans <em>et al.</em> 2004</a>).</p><p id="p-11">Pathogen-driven selection implies that specific HLA alleles are more effective in presenting antigens of certain pathogens, to which the population is exposed. Support for this assumption comes from associations between disease susceptibility, resistance or progression with genetic variation at HLA. For example, variants in <em>HLA-B</em> are associated to the progression to clinical disease after HIV infection (The International HIV Controllers Study 2010), variants in <em>HLA-C</em>, <em>-DP</em> and <em>-DQ</em> are associated to clearance of hepatitis B virus (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2013</a>), alleles in <em>HLA-DRB1</em> are associated to susceptibility or resistance to leprosy (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Vanderborght <em>et al.</em> 2007</a>) and alleles at <em>HLA-B</em>, <em>-DQ</em> and <em>-DR</em> are associated to resistance to severe malaria (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hill <em>et al.</em> 1991</a>).</p><p id="p-12">Previous studies which measured population differentiation at the HLA genes found evidence ranging from increased to reduced population differentiation. For example, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Meyer <em>et al.</em> (2006)</a> found no significant difference between differentiation at HLA genes and a set of neutral markers used as a control (microsatellites), while <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sanchez-Mazas (2007)</a> found lower differentiation at HLA loci than in their genome-wide control (microsatellites and RFLPs). A limitation of these studies is that differences between the neutral genetic markers and the sequence data used for HLA genes introduce confounding variables, making it difficult to determine the roles of selection or characteristics inherent to the marker (<em>e.g.</em>, mutation rate and diversity). Another study compared differentiation on markers of the same type (microsatellites) located at HLA genes or near them and those located in other genomic regions, which serve as controls (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nunes 2011</a>). This study found increased differentiation in regions near HLA genes. Nonetheless, some issues remain unresolved: <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nunes (2011)</a> was mainly interested in native American populations, and used a limited number of markers. Furthermore, the complexity of the mutational mechanism of microsatellites complicates the interpretation of results.</p><p id="p-13">For non-model organisms a similarly wide array of results have been found, with the MHC region (which contains genes homologous to HLA) showing either equal (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Miller <em>et al.</em> 2010</a>), higher (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Loiseau <em>et al.</em> 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Oliver <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cammen <em>et al.</em> 2011</a>) or lower (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McCairns <em>et al.</em> 2011</a>) differentiation than genome-wide averages. These contrasting results could be due to differences in selective regimes among species, or even to variation in selection among genes within a species (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Čížková <em>et al.</em> 2011</a>).</p><p id="p-14">In summary, it remains unclear whether balancing selection on HLA genes drives increased differentiation due to selection favoring adaptation to locally occurring pathogens, or whether it results in decreased genetic differentiation due to the maintenance of shared polymorphisms among populations.</p><p id="p-15">Here, we revisit the question of population differentiation at the HLA genes using sequence data from worldwide human populations. We investigate variation at SNPs, which have the advantage of allowing the use of genomic data as an empirical control for HLA SNPs, assuming similar mutation rates for SNPs in the MHC region and the reminder of the genome. Differently from scans that seek genome-wide significance for specific SNPs, we <em>a priori</em> define a set of putatively selected SNPs to be surveyed (those within or close to HLA genes). We relate differences in FST between HLA and non-HLA SNPs to the degree of polymorphism in each of these groups, drawing on recent findings concerning the constraints imposed by allele frequencies on measures of differentiation. Finally, we perform simulations and find a plausible selective regime that reproduces our results.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>SNP Data</h3><p id="p-16">SNP genotypes were acquired from the integrated Variant Call Format (VCF) files from phase 3 of the 1000 Genomes Project (1000G) (The 1000 Genomes Project Consortium 2015), which are available at <a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20130502/">ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20130502/</a>.</p><div id="sec-3" class="subsection"><h4>Sample filters:</h4><p id="p-17">The 1000G Phase 3 sequenced 2504 individuals from 26 populations. Six of those populations are admixed: Americans of African Ancestry in the Southest USA (ASW); individuals of Mexican Ancestry from Los Angeles in the USA (MXL); Puerto Ricans from Puerto Rico (PUR); Colombians from Mendellín, Colombia (CLM); African Caribbeans from Barbados (ACB); and Peruvians from Lima, Peru (PEL). We focus on the effect of balancing selection on differentiation of diverging populations, and therefore we excluded admixed populations from our analyses. After excluding admixed populations, a total of 2000 individuals in 20 populations were kept (see Table S1).</p></div><div id="sec-4" class="subsection"><h4>Site filters:</h4><p id="p-18">The 1000 Genomes phase 3 data includes variants discovered via high coverage exome targeted resequencing as well as low coverage whole genome resequencing. Because of the higher coverage in exonic regions, all comparisons between HLA SNPs and non-HLA SNPs (which we treated as a control set) were made within the same functional category (<em>e.g.</em>, intronic or exonic). To this end each SNP was annotated using ANNOVAR (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wang <em>et al.</em> 2010</a>). Our findings for differences in FST between HLA and non-HLA SNPs were qualitatively the same when using either exonic or intronic regions, so throughout the paper we focus on the results for exonic regions.</p><p id="p-19">Single nucleotide variants with more than 2 alleles in the 1000G data were excluded from our analyses, to keep consistency with our analytical results on the relationship between minor allele frequency and population differentiation, which are defined for biallelic variants. In total, 954,219 SNPs in 1000G are exonic and biallelic. Of these, 832,892 are polymorphic in the set of populations we analyzed, with 525 located within HLA genes, and 1193 located within peri-HLA genes (see “Definition of HLA, peri-HLA and control regions” section, below).</p><p id="p-20">The use of a single reference genome to map Next Generation Sequencing (NGS) reads creates mapping bias at some HLA SNPs in the 1000 Genomes Project phase 1 dataset (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brandt <em>et al.</em> 2015</a>). Another potential source of error in NGS data for the HLA region is due to copy number variation of some genes, which could cause alleles to be under or over-called, if there are more or less copies of the gene present in the reference genome, respectively. We therefore excluded the SNPs within the HLA genes which have unreliable frequency estimates in the 1000 Genomes phase 1 dataset, compared to gold-standard sequencing data generated by Sanger sequencing (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brandt <em>et al.</em> 2015</a>). After applying this filter, 38 out of 525 exonic biallelic SNPs in the HLA genes were excluded (Table S3). In total, 487 SNPs were kept in the HLA group, 1193 in the peri-HLA group, and 831,174 in the control group.</p><p id="p-21">In the present study we analyze this filtered version of the 1000 Genomes phase 3 data instead of phase 1 or Sanger sequencing data analyzed in <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Brandt <em>et al.</em> (2015)</a> because it has a larger sample size and includes more populations. Also, SNPs identified as unreliable in the 1000 Genomes phase 1 data were generally the same across different populations, which supports the application of this filter to phase 3 data used here.</p></div></div><div id="sec-5" class="subsection"><h3>Estimation of FST and minor allele frequency</h3><p id="p-22">Population differentiation was calculated as the proportion of variance in allele frequencies among populations (<em>a</em>), relative to the total genetic variance (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-1.gif"/></noscript></span></span>, with <em>b</em> and <em>c</em> referring to the variance components between individuals within populations and between gametes within individuals, respectively):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>To obtain FST values we used the <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Weir and Cockerham (1984)</a> estimator implemented in VCFtools v0.1.14 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danecek <em>et al.</em> 2011</a>). The <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Weir and Cockerham (1984)</a> estimator was chosen because it is unbiased when sample sizes are large and similar, as in the case of our dataset. FST was calculated per SNP i) over all populations and ii) for pairs of populations. In the first case, all sites that were variable in at least one of the 20 populations were used to estimate F<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-3.gif"/></noscript></span></span> In the second case, only sites that were variable in at least one population in the pair were used to estimate FST between the two populations.</p><p id="p-23">When summarizing FST over multiple SNPs, we compared two approaches: i) computing a simple average of FST at individual SNPs and ii) using the “ratio of averages” approach, suggested by <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Reynolds <em>et al.</em> (1983)</a>, in which we first estimate the numerator (<em>a</em>) and denominator (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-4.gif"/></noscript></span></span>) of FST for each SNP, and then compute the averages of <em>a</em> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-5.gif"/></noscript></span></span> for the desired set of SNPs, and finally compute the ratio of both averages. This second approach provides the least biased estimate of F<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-6.gif"/></noscript></span></span> whereas performing a simple average of the FSTs of each SNP can lead to an underestimation of differentiation, especially in datasets rich in rare variants (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Reynolds <em>et al.</em> 1983</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bhatia <em>et al.</em> 2013</a>). Unless otherwise stated, we used the “ratio of averages” approach to compute F<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-7.gif"/></noscript></span></span> Variance components (<em>a</em> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-8.gif"/></noscript></span></span>) were obtained using a minor modification of the VCFtools source code described in the Extended Materials and Methods.</p><p id="p-24">Allele frequency (AF) was calculated per SNP, per population using VCFtools v0.1.14b (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Danecek <em>et al.</em> 2011</a>). Minor allele frequency (MAF) was calculated as <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-9.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-10.gif"/></noscript></span></span> Global minor allele frequency (over all populations) was calculated as the frequency of the rarest allele in the pooled sample of 2000 individuals.</p></div><div id="sec-6" class="subsection"><h3>Definition of HLA, peri-HLA and control regions</h3><p id="p-25">We define “HLA SNPs” as those contained within the coding sequence of the classical HLA genes <em>HLA-A</em>, <em>-B</em>, <em>-C</em>, <em>-DRA</em>, <em>-DRB1</em>, <em>-DQA</em>, <em>-DQB1</em>. Previous studies of <em>HLA-DPA1</em> and <em>-DPB1</em> found weak or no evidence of balancing selection (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Solberg <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Begovich <em>et al.</em> 2001</a>), and even instances of directional selection (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hollenbach <em>et al.</em> 2001</a>), making them inappropriate for our question concerning the role of balancing selection on differentiation at HLA loci. Accordingly, our analyses also showed that population differentiation at <em>HLA-DPA1</em> and <em>HLA-DPB1</em> genes is different from that of other HLA genes (see Results and Figure S4). Therefore, those loci were excluded from our analysis unless otherwise mentioned.</p><p id="p-26">Peri-HLA genes were defined as those that flank the HLA genes and have higher diversity relative to the average of chromosome 6 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mendes 2013</a>), indicating that their increased polymorphism is driven by hitchhiking to the strongly selected HLA loci (Table S2). These genes are located 119kb to 256kb from the closest HLA locus. All SNPs outside both the HLA and peri-HLA genes comprised the control group.</p></div><div id="sec-7" class="subsection"><h3>Statistical test of FST differences</h3><p id="p-27">When comparing FST values between HLA and control SNPs, we control for the fact that HLA SNPs are located within a small set of genes, resulting in higher linkage disequilibrium (LD) and statistical non-independence than for SNPs in the control group, which comprise a genomewide set. To account for this effect, we designed a strategy to sample the control SNPs in a way that approximates the LD structure among the HLA SNPs.</p><p id="p-28">To do this, we first randomly sampled an exonic SNP and extracted the MAF of all the other exonic SNPs within that gene. Each sampled SNP was assigned to one of six MAF bins: one bin for rare SNPs (0-0.01), and five bins with approximate width of 0.1 (0.01-0.1, 0.1-0.2, 0.2-0.3, 0.3-0.4, 0.4-0.5). This combination of bins allowed us to sample SNPs from various frequency classes, without creating excessively sparse intervals. We then went on to draw another random exonic SNP (from another gene), repeating this process until for each MAF bin we had the same number of SNPs as for the HLA SNPs. If a specific MAF bin was complete (<em>i.e.</em>, contained the number of HLA SNPs in that MAF interval) before others, we only kept SNPs from the sampled gene for the MAF bins which were still incomplete (see Figure S5). As a result of this sampling scheme, the control sample contained SNPs collected from a smaller subset of genes than if SNPs were randomly drawn from the genome. In this way our control SNPs had similar properties to the HLA SNPs: high LD among the sampled SNPs and matching MAF distribution. This entire process was repeated 1000 times, providing 1000 sets of control SNPs.</p><p id="p-29">We compared the FST distribution of each of the 1000 sets of control SNPs to that of the HLA SNPs, applying a Mann-Whitney test. We recorded the number of comparisons where the difference between FST distributions was significant (p &lt; 0.05).</p></div><div id="sec-8" class="subsection"><h3>Haplotype level analyses</h3><p id="p-30">In addition to SNP based analyses, we also investigated population differentiation when alleles are defined by the coding sequence of each HLA gene (classically referred to as an “HLA allele”). We treat these analyses as “haplotype level”, where haplotypes are defined by a combination of SNPs along an HLA gene (<em>i.e.</em>, each allele in these analyses is an intragenic haplotype).</p><p id="p-31">Phasing of SNPs in extremely variable regions like the HLA is complicated by the high SNP density and polymorphism. Therefore, rather than estimating intragenic haplotypes directly from the SNP data, we used a publicly available dataset which provides HLA allele calls for samples in the 1000G data based on Sanger sequencing (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gourraud <em>et al.</em> 2014</a>), available at the dbMHC website (<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20140725_hla_genotypes/20140702_hla_diversity.txt">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20140725_hla_genotypes/20140702_hla_diversity.txt</a>).</p><p id="p-32">We restricted these haplotype level analyses to <em>HLA-A</em>, <em>-B</em>, <em>-C</em>, <em>HLA-DRB1</em> and <em>-DQB1</em>, which are reported in <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gourraud <em>et al.</em> (2014)</a>. HLA allele calls were coded so as to only distinguish alleles with nonsynonymous differences (<em>i.e.</em>, only the first two fields of the allele names were used, as described in the HLA nomenclature system) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Marsh <em>et al.</em> (2010)</a>). When the HLA typing method generated an ambiguous allele call, the most common allele was chosen.</p><p id="p-33">FST values of multiallelic HLA haplotypes can’t be directly compared to those of biallelic SNPs because multiallelic loci tend to have lower allele frequencies, which constrains the maximum value of FST (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jakobsson <em>et al.</em> 2013</a>). To allow FST values at the HLA haplotypes to be compared to this null distribution, we recoded each HLA gene as a series of biallelic loci. This recoding was done by treating each allele at each gene as “allele 1”, and all other alleles as “allele 2”. The <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Weir and Cockerham (1984)</a> FST estimator was then computed as described for SNPs, using the <em>wc</em> function of the hierfstat R package (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Goudet 2005</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-34">Supplementary material is available on Figshare. Extended Materials and Methods shows the modification of VCFtools source code to get variance components of FST. File S1 shows the constraints of minor allele frequency on F<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-11.gif"/></noscript></span></span> File S2 is a detailed description of the balancing selection simulations. Figure S4 shows FST as a function of MAF for each HLA gene. Figure S5 is a flowchart describing the steps of the resampling algorithm used to account for LD among HLA SNPs. Figure S6 shows distributions of FST in bins of MAF lower than 0.01. Figure S7 shows distributions of FST per MAF bin, including outliers. Figure S8 shows hierarchical population differentiation (FCT and FSC) along chromosome 6. Table S1 shows sample sizes per population. Table S2 is a list of peri-HLA genes and their genomic coordinates. Table S3 is a list of sites excluded due to excess of genotyping errors. Table S4 shows the results of the statistical test of differences in FST distributions in HLA and non-HLA SNPs, accounting for LD. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6671753">https://doi.org/10.25387/g3.6671753</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Higher FST in HLA genes</h3><p id="p-35">Initially we compared the distribution of FST among SNPs from the HLA, peri-HLA and control groups (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of FST values for exonic SNPs from the following groups: outside HLA and peri-HLA regions (control group), within HLA genes and within peri-HLA genes. The number of SNPs in each group is shown in parentheses. Density was estimated for each category of SNPs at 190 points equally spaced along the range of observed FST values, using a Gaussian kernel. Density at each point was normalized by dividing by the sum of densities at each category, such that the height represents the proportion of SNPs with that FST value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of FST values for exonic SNPs from the following groups: outside HLA and peri-HLA regions (control group), within HLA genes and within peri-HLA genes. The number of SNPs in each group is shown in parentheses. Density was estimated for each category of SNPs at 190 points equally spaced along the range of observed FST values, using a Gaussian kernel. Density at each point was normalized by dividing by the sum of densities at each category, such that the height represents the proportion of SNPs with that FST value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F1.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2805/F1.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490651" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-36" class="first-child">Distribution of FST values for exonic SNPs from the following groups: outside HLA and peri-HLA regions (control group), within HLA genes and within peri-HLA genes. The number of SNPs in each group is shown in parentheses. Density was estimated for each category of SNPs at 190 points equally spaced along the range of observed FST values, using a Gaussian kernel. Density at each point was normalized by dividing by the sum of densities at each category, such that the height represents the proportion of SNPs with that FST value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The distribution of FST of SNPs within HLA genes is shifted toward higher values (median = 0.027) compared to control SNPs (median=<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-12.gif"/></noscript></span></span>), and the distributions are significantly different (p-value <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-13.gif"/></noscript></span></span> two-tailed Mann-Whitney test).</p><p id="p-38">Theory predicts that balancing selection affects only a narrow genomic region, the size of which is defined by the intensity of selection and the recombination rate (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> 1997</a>). To evaluate if the increased differentiation at coding SNPs within HLA genes was also observed in loci that flank the HLA, we applied the same test to the peri-HLA genes. Although the shift is less striking than for HLA SNPs, peri-HLA SNPs also have an FST distribution with significantly higher values (median = 0.006, p-value <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-14.gif"/></noscript></span></span> two-tailed Mann-Whitney test; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-12" class="subsection"><h3>Lower FST in HLA when accounting for MAF</h3><p id="p-39">The effect of balancing selection is to shift the site frequency spectrum (SFS) of selected loci toward an excess of intermediate frequency variants. This shift is precisely what we see in the data, with the SFS for HLA SNPs showing a shift to intermediate frequencies compared to control SNPs. The peri-HLA SNPs occupy an intermediate position in the SFS (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of minor allele frequency (MAF) for exonic control, peri-HLA genes and HLA genes. MAF of SNPs at the HLA and peri-HLA genes is higher compared to other genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of minor allele frequency (MAF) for exonic control, peri-HLA genes and HLA genes. MAF of SNPs at the HLA and peri-HLA genes is higher compared to other genes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F2.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2805/F2.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490625" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Distribution of minor allele frequency (MAF) for exonic control, peri-HLA genes and HLA genes. MAF of SNPs at the HLA and peri-HLA genes is higher compared to other genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Constraints imposed by allele frequencies on FST have been a topic of recent investigation (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Roesti <em>et al.</em> 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Maruki <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Elhaik 2012</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Jakobsson <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Edge and Rosenberg 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alcala and Rosenberg 2017</a>), and it has been shown that SNPs with very low minor allele frequencies (MAF) are bounded to low FST values. This relationship between MAF and FST is empirically illustrated for the 1000 Genomes exome data in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, which shows that FST is constrained to low values mainly in the range of low MAF (up to <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-15.gif"/></noscript></span></span> in the 1000G dataset). For MAF values above 0.15 the constraint is no longer evident. In File S1, we analytically show the relationship between MAF and the maximum possible value of F<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-16.gif"/></noscript></span></span> which has also been shown by <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population differentiation, measured by F as a function of minor allele frequency at biallelic exonic SNPs from the 1000 Genomes Project phase 3 data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population differentiation, measured by F&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-17&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; as a function of minor allele frequency at biallelic exonic SNPs from the 1000 Genomes Project phase 3 data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2805/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490474" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Population differentiation, measured by F<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F3/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F3/embed/mml-math-17.gif"/></noscript></span></span> as a function of minor allele frequency at biallelic exonic SNPs from the 1000 Genomes Project phase 3 data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">This relationship between MAF and maximum FST suggests that the large number of rare variants in the 1000G dataset, and the relative paucity of low MAF variants in the HLA SNPs, can explain the differences in FST distributions between control and HLA SNPs seen in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>.</p><p id="p-44">To account for the effects of differences in SFS between HLA and control SNPs when contrasting population differentiation among those groups, we compared the FST of HLA and control SNPs within bins of MAF values (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Contrary to what we observed without controlling for MAF, we now find that HLA and peri-HLA SNPs have significantly lower FST than at the control SNPs (Mann-Whitney two-tailed test p-value <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-18.gif"/></noscript></span></span> for all bins of MAF &gt; 0.01). The bin with MAF &lt; 0.01 shows a similar pattern when further split into smaller bins of MAF (Figure S6).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FST distributions per minor allele frequency (MAF) bin. HLA and peri-HLA SNPs show lower FST than control SNPs in all bins with MAF  Outliers (points above the 3rd quartile by 1.5 times the interquartile range, or below the 1st quartile by the same amount) were removed from figure, but not from statistical test, for better visualization. Figure S7 shows FST distributions including outliers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;FST distributions per minor allele frequency (MAF) bin. HLA and peri-HLA SNPs show lower FST than control SNPs in all bins with MAF &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-19&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Outliers (points above the 3&lt;sup&gt;rd&lt;/sup&gt; quartile by 1.5 times the interquartile range, or below the 1&lt;sup&gt;st&lt;/sup&gt; quartile by the same amount) were removed from figure, but not from statistical test, for better visualization. Figure S7 shows FST distributions including outliers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F4.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2805/F4.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490534" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">FST distributions per minor allele frequency (MAF) bin. HLA and peri-HLA SNPs show lower FST than control SNPs in all bins with MAF <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F4/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F4/embed/mml-math-19.gif"/></noscript></span></span> Outliers (points above the 3<sup>rd</sup> quartile by 1.5 times the interquartile range, or below the 1<sup>st</sup> quartile by the same amount) were removed from figure, but not from statistical test, for better visualization. Figure S7 shows FST distributions including outliers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">This approach used SNPs outside the HLA and peri-HLA genes as controls for the HLA SNPs. However, because the HLA SNPs are located in 5 genes, they are not independent, due to both intra and inter-locus associations. As a consequence, our p-values could be inflated by treating a set of correlated SNPs as independent, and comparing them to a set of control SNPs which are in their majority independent. We controlled for this non-independence using a resampling approach, in which sets of linked SNPs were sampled from our control group, and their FST values were compared to those from the HLA SNPs, at each MAF bin (see Materials and Methods). After controlling for the non-independence of HLA SNPs, we confirmed that FST at HLA SNPs was significantly lower than at the resampled SNPs, at all bins of MAF higher than 0.01 (Table S4).</p><p id="p-47">If instead of comparing FST distributions, one is interested in comparing average FST between different genomic regions, it is again necessary to account for the constraint of MAF on FST. Computing a simple average FST of individual SNPs creates a bias leading to an underestimation of F<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-20.gif"/></noscript></span></span> the effect of the bias being more pronounced when there are more rare variants in the dataset (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bhatia <em>et al.</em> 2013</a>). To control for the effect of rare variants reducing average F<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-21.gif"/></noscript></span></span> we estimate average FST for multiple loci by separately computing averages of the numerator and denominator of the AMOVA-based FST estimator, and then taking their ratio (see Methods). <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Reynolds <em>et al.</em> (1983)</a> and <a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Weir and Cockerham (1984)</a> showed that this is the least biased estimate of average FST across multiple loci. This “ratio of averages” approach downweights the contribution of variants with low MAF to the average F<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-22.gif"/></noscript></span></span> resulting in higher average F<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-23.gif"/></noscript></span></span></p><p id="p-48">We explored how these different averaging methods impact the FST at HLA genes, and found that the “ratio of averages” approach (which controls for MAF) results in substantially lower average FST at the HLA SNPs (0.04) than in the control SNPs (0.09). In stark contrast, using the average of individual loci FST (<em>i.e.</em>, an “average of ratios”), we found higher FST values for the HLA SNPs (0.03) than genome-wide (0.01), as in our initial analysis that did not account for MAF (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The effect of different FST averaging approaches further emphasizes the importance of accounting for the differences between the site frequency spectrum of HLA and control SNPs when assessing population differentiation.</p><div id="sec-13" class="subsection"><h4>HLA-DP genes:</h4><p id="p-49">The classical <em>HLA-DPA1</em> and <em>-DPB1</em> genes were excluded from the previous analysis because they show weak or no evidence of balancing selection (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Solberg <em>et al.</em> 2008</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Begovich <em>et al.</em> 2001</a>), and some evidence of directional selection (supported by the observation that within individual populations a small number of alleles are present at a high frequency) (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hollenbach <em>et al.</em> 2001</a>).</p><p id="p-50">Consistently with being under a different selective regime, <em>HLA-DPA1</em> and <em>-DPB1</em> show a pattern of population differentiation which is different from the other classical HLA loci: FST at these genes is higher than in the control SNPs, even when minor allele frequency is controlled for (Figure S4).</p></div><div id="sec-14" class="subsection"><h4>Contrasting FST of HLA SNPs and haplotypes:</h4><p id="p-51">Next, we explored population differentiation at the haplotype level (<em>i.e.</em>, with alleles defined by the coding sequence of each HLA gene). Haplotype level analyses were motivated by the idea that the fitness of individuals is more likely to be determined by the combination of SNPs they carry in a gene, rather than by individual SNPs, since it is the combination of SNPs that determines the peptides which HLA molecules present.</p><p id="p-52">To compare FST for HLA haplotypes to a null distribution, we recoded each HLA gene as a biallelic locus (see Materials and Methods). Population differentiation at the recoded HLA haplotypes was then compared to differentiation at control SNPs and HLA SNPs, while controlling for minor allele frequency, as was done for SNPs.</p><p id="p-53">HLA haplotypes show population differentiation intermediate between HLA SNPs and control SNPs. Differentiation at HLA haplotypes was not significantly different from HLA SNPs in any MAF bin, and was not significantly different from control SNPs, except for the two MAF bins between 0 and 0.08 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Thus, despite the existence of haplotypes which are specific to certain world regions, when the average MAF is considered and global FST is quantified, the degree of differentiation of HLA haplotypes tends to be lower than that of control SNPs.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population differentiation (FST) as a function of minor allele frequency for the exonic control SNPs (gray), for exonic HLA SNPs (red) and for the haplotype-level HLA alleles (yellow) after recoding as biallelic (see Materials and Methods). No haplotypes had frequency higher than 0.2, therefore MAF bins were redefined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population differentiation (FST) as a function of minor allele frequency for the exonic control SNPs (gray), for exonic HLA SNPs (red) and for the haplotype-level HLA alleles (yellow) after recoding as biallelic (see Materials and Methods). No haplotypes had frequency higher than 0.2, therefore MAF bins were redefined.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F5.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/8/2805/F5.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490621" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-54" class="first-child">Population differentiation (FST) as a function of minor allele frequency for the exonic control SNPs (gray), for exonic HLA SNPs (red) and for the haplotype-level HLA alleles (yellow) after recoding as biallelic (see Materials and Methods). No haplotypes had frequency higher than 0.2, therefore MAF bins were redefined.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">It is worth noting that ambiguous HLA haplotype calls (<em>i.e.</em>, instances where the typing method provided a set of possible allele calls) were resolved by choosing the assignment that minimized population differentiation. Thus, a more reliable assessment of haplotype-level differentiation will require less ambiguous haplotype calls.</p></div></div><div id="sec-15" class="subsection"><h3>FST at HLA SNPs depends on divergence times</h3><p id="p-56">Our previous analyses examined global F<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-24.gif"/></noscript></span></span> which captures patterns of differentiation among all 20 populations retained from the 1000 genomes full dataset. Next, we asked how specific populations contributed to our findings. In order to investigate this question, and to evaluate how the geographical scale (within and among continent comparisons) influences differentiation at the HLA, we analyzed FST between all pairs of populations.</p><p id="p-57">We found that the lower differentiation at HLA SNPs as compared to control SNPs seen in our previous results (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) in seen for highly diverged populations (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) (<em>i.e.</em>, contrasts involving populations from different continents).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average FST at the HLA and at the control SNPs for each pair of populations. Points with a single color represent pairs of populations from the same continent. Average FST was computed using the &#x201D;ratio of averages&#x201D; approach, described in the main text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average FST at the HLA and at the control SNPs for each pair of populations. Points with a single color represent pairs of populations from the same continent. Average FST was computed using the &#x201D;ratio of averages&#x201D; approach, described in the main text.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F6.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/8/2805/F6.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490510" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-58" class="first-child">Average FST at the HLA and at the control SNPs for each pair of populations. Points with a single color represent pairs of populations from the same continent. Average FST was computed using the ”ratio of averages” approach, described in the main text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">However, population pairs within the same continent show higher differentiation at the HLA SNPs compared to control SNPs. We also observed this effect in an independent analysis of 51 populations from the Human Genome Diversity Project (HGDP) SNP dataset (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2008</a>), with FSC (differentiation among populations within continents) values showing a peak in the MHC region (Figure S8).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Population differentiation at HLA SNPs</h3><p id="p-60">In an overall analysis of FST among worldwide populations, we found significantly decreased genetic differentiation at HLA SNPs. We have shown that this result is critically dependent on the use of methods which appropriately account for the properties of the site frequency spectrum of HLA genes.</p><p id="p-61">The decreased differentiation at HLA genes counters the expectation of a model of adaptation to local pathogens driving differentiation. However, we found that the overall pattern of lower differentiation reveals a greater complexity when we compare populations with different divergence times.</p></div><div id="sec-18" class="subsection"><h3>Mind the MAF</h3><p id="p-62">Our results highlight the importance of accounting for minor allele frequency (MAF) when comparing FST values for different sets of SNPs, as previously shown by others (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Roesti <em>et al.</em> 2012</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Maruki <em>et al.</em> 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Elhaik 2012</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Jakobsson <em>et al.</em> 2013</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Edge and Rosenberg 2014</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wang 2015</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alcala and Rosenberg 2017</a>). Not accounting for MAF leads to an underestimation of population differentiation for sets of SNPs rich in rare variants. When comparing sets of SNPs from genomic regions with different MAF distributions, the constraint of MAF on FST may result in a misleading interpretation of the selective regime acting on each region.</p><p id="p-63">We empirically demonstrated the constraint of FST by MAF in the 1000 Genomes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), and <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) have demonstrated the same effect in the HGDP dataset (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2008</a>). In <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a>, the constraint is given in terms of the frequency of the most frequent allele, and it is strongest when the frequency of the most frequent allele is above 0.85, which is equivalent to MAF below 0.15.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relation between  under neutrality and balancing selection. The black line indicates FSTneutral = FSTselection. (A) daughter populations remain under the same regime of overdominance as the ancestral population (shared overdominance). (B) One of the daughter populations experiences a shift in the fitness values (divergent overdominance), remaining under overdominance but with a new equilibrium value (changed by a value of ). Under divergent overdominance, for recent divergence times we find balancing selection can transiently increase population differentiation, so long as selection is strong (s = 0.05 or greater, )." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relation between &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under neutrality and balancing selection. The black line indicates FSTneutral = FSTselection. (A) daughter populations remain under the same regime of overdominance as the ancestral population (shared overdominance). (B) One of the daughter populations experiences a shift in the fitness values (divergent overdominance), remaining under overdominance but with a new equilibrium value (changed by a value of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Under divergent overdominance, for recent divergence times we find balancing selection can transiently increase population differentiation, so long as selection is strong (s = 0.05 or greater, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F7.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/8/2805/F7.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490511" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-64" class="first-child">Relation between <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F7/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F7/embed/mml-math-25.gif"/></noscript></span></span> under neutrality and balancing selection. The black line indicates FSTneutral = FSTselection. (A) daughter populations remain under the same regime of overdominance as the ancestral population (shared overdominance). (B) One of the daughter populations experiences a shift in the fitness values (divergent overdominance), remaining under overdominance but with a new equilibrium value (changed by a value of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F7/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F7/embed/mml-math-26.gif"/></noscript></span></span>). Under divergent overdominance, for recent divergence times we find balancing selection can transiently increase population differentiation, so long as selection is strong (s = 0.05 or greater, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F7/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F7/embed/mml-math-27.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">Our results differ slightly from <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a> in the slope of the diagonal that shows the constraint of FST at low MAF, which is steeper in <a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a>, <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>. Also, our results presented in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> show more SNPs along that diagonal, and some seemingly above where the theoretical maximum would be, especially at low MAF. Although we have not directly tested the causes of these differences, we point out two potential explanations. First, different FST estimators were used: we estimated FST according to <a id="xref-ref-64-5" class="xref-bibr" href="#ref-64">Weir and Cockerham (1984)</a>, while <a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a> calculated parametric F<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-28.gif"/></noscript></span></span> Second, we have computed FST among 20 populations in four major geographic groups (listed in Table S1), while <a id="xref-ref-1-9" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a> compared seven geographic regions, therefore excluding within-region comparisons with the lowest FST values.</p><p id="p-66"><a id="xref-ref-1-10" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a> also arrived at the same results for the upper boundary of FST as a function of allele frequency in biallelic markers as we present on Supplemental File S1. Their work presents a more general investigation of the constraints of the frequency of the most frequent allele on F<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-29.gif"/></noscript></span></span> using simulations under different migration models. <a id="xref-ref-1-11" class="xref-bibr" href="#ref-1">Alcala and Rosenberg (2017)</a> discuss the effect of this constraint in reducing the power of outlier tests which use high FST as a signature of local positive selection. Here we emphasize the effect of this constraint on interpreting FST values at regions under balancing selection, where the depletion of rare variants leads to higher overall population differentiation than in other genomic regions, if no control for MAF is performed. However, when variants with similar MAF are compared, population differentiation in the region under balancing selection is actually lower, as we show for SNPs within HLA genes.</p><p id="p-67">While the constraint of MAF on FST strongly affects differentiation at regions under positive and balancing selection, it is less problematic in regions under purifying selection. Under this selective regime, both an enrichment of low frequency variants and low population differentiation are expected. Since population differentiation in low frequency variants is constrained to low values, the relationship between MAF and FST leads both signatures in the same direction. This effect has been demonstrated in <em>Drosophila melanogaster</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jackson <em>et al.</em> 2014</a>).</p></div><div id="sec-19" class="subsection"><h3>Contrast to previous studies</h3><p id="p-68">Two classical HLA genes, <em>HLA-DPA1</em> and <em>HLA-DPB1</em>, were exceptions to our finding of lower population differentiation at HLA SNPs. The SNPs in these genes showed higher population differentiation, even when MAF was accounted for (Figure S4). This finding supports previous evidence that HLA-DP genes are under a different selective regime from other classical HLA genes. For example, a genomic scan performed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bhatia <em>et al.</em> (2011)</a> also found a SNP near a HLA-DP gene with unusually high population differentiation (rs2179915, which is 30kb from <em>HLA-DPA2</em>). Similarly, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Barreiro <em>et al.</em> (2008)</a> found unusually high FST at the <em>HLA-DPB2</em> locus, applying a threshold value of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-30.gif"/></noscript></span></span> which is in the tail of FST distributions at any MAF bin. Together, these results indicate that the HLA-DP genes are not under balancing selection, but rather under directional selection that differs among populations. One possibility is that HLA-DP has suffered a recent change in selective regimes, in which variation maintained by balancing selection becomes positively selected in specific populations, as has been shown in other loci (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">de Filippo <em>et al.</em> 2016</a>).</p><p id="p-69">The studies of <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bhatia <em>et al.</em> (2013)</a> and <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Barreiro <em>et al.</em> (2008)</a> were designed with an emphasis on the detection of extremely high differentiation, and thus did not capture the pattern of low differentiation which is characteristic of most HLA loci. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hofer <em>et al.</em> (2012)</a>, on the other hand, used an approach where regions of adjacent SNPs with extreme FST were scanned for, and found unusually low FST for <em>HLA-C</em>. The method used by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hofer <em>et al.</em> (2012)</a> evaluates FST as a function of the heterozygosity between populations, which for biallelic markers is equivalent to the correction for MAF we applied here. These results show that when a test is designed to account for the possibility of unusually low FST values, and when the effect of minor allele frequency (or heterozygosity) on FST is accounted for, a signature of low differentiation which would otherwise not be detected can be found.</p></div><div id="sec-20" class="subsection"><h3>Divergence time effect</h3><p id="p-70">By taking advantage of multiple populations made available by the 1000 Genomes project, we also examined if the excess of low differentiation at HLA SNPs holds at all timescales of differentiation. Interestingly, we find that for population pairs from the same continent FST at HLA SNPs is equal to or higher than in the control SNPs (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). For highly diverged population pairs (those from different continents), we consistently find lower differentiation among HLA SNPs. This result shows that a specific set of SNPs may differ in how they deviate from the genomic background depending on the timescale of population divergence.</p><p id="p-71">To understand the process driving the increased differentiation for recently diverged populations (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>) we used a simulation approach. Although many different forms of balancing selection could be operating (<em>e.g.</em>, frequency-dependent, selection varying over time or space, etc.), here we explore a single and well understood regime, heterozygote advantage (or overdominance).</p><p id="p-72">First, we simulated a scenario where an ancestral population is under symmetric overdominant selection (equal fitness to both homozygotes), and splits into two daughter populations, both under the same selective regime as the ancestor (details in Supplemental File S2). We refer to this scenario as “shared overdominance”, and find that it results in differentiation between the daughter populations being reduced with respect to neutral expectations (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a> and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>), in accordance with previous results (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Schierup <em>et al.</em> 2000</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of HLA differentiation under two regimes of overdominance. Each tree represents a population pair experiencing selection according to fitness values presented in the boxes above the tips. The equilibrium frequencies expected under these fitness values are also shown ( and ). In &#x201D;shared overdominance&#x201D; both daughter populations share the same fitness values. In &#x201D;divergent overdominance&#x201D; the fitness values for one of the homozygotes changes in one of the daughter populations, even though the regime is still one of overdominance. In the divergent overdominance scenario, the equilibrium frequencies at the selected site differs among populations, and drives increased differentiation when divergence is recent (FST HLA &gt; FST neutral). For the shared overdominance scenario FST neutral &gt; FST HLA throughout the entire history of population divergence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of HLA differentiation under two regimes of overdominance. Each tree represents a population pair experiencing selection according to fitness values presented in the boxes above the tips. The equilibrium frequencies expected under these fitness values are also shown (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-32&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). In &#x201D;shared overdominance&#x201D; both daughter populations share the same fitness values. In &#x201D;divergent overdominance&#x201D; the fitness values for one of the homozygotes changes in one of the daughter populations, even though the regime is still one of overdominance. In the divergent overdominance scenario, the equilibrium frequencies at the selected site differs among populations, and drives increased differentiation when divergence is recent (FST HLA &gt; FST neutral). For the shared overdominance scenario FST neutral &gt; FST HLA throughout the entire history of population divergence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F8.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/8/2805/F8.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-73" class="first-child">Schematic representation of HLA differentiation under two regimes of overdominance. Each tree represents a population pair experiencing selection according to fitness values presented in the boxes above the tips. The equilibrium frequencies expected under these fitness values are also shown (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F8/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F8/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F8/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F8/embed/mml-math-32.gif"/></noscript></span></span>). In ”shared overdominance” both daughter populations share the same fitness values. In ”divergent overdominance” the fitness values for one of the homozygotes changes in one of the daughter populations, even though the regime is still one of overdominance. In the divergent overdominance scenario, the equilibrium frequencies at the selected site differs among populations, and drives increased differentiation when divergence is recent (FST HLA &gt; FST neutral). For the shared overdominance scenario FST neutral &gt; FST HLA throughout the entire history of population divergence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-74">However, if the two populations remain under overdominant selection but differ for at least one of the homozygote fitness values (a scenario we refer to as “divergent overdominance”), differentiation can be increased with respect to neutrality for small divergence times (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a> and <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). This result can be understood if we consider that the populations are extremely similar at the time of the split, so the effect of selection will be to favor changes in allele frequency between them (since homozygote fitness and therefore equilibrium frequencies, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-33.gif"/></noscript></span></span> of selected alleles will differ). As the two populations further diverge, the neutral sites will continue to diverge and will surpass the differentiation for the case of overdominance. Thus, by assuming that the fitness values of an overdominant model can change over time –which is highly plausible if we consider heterogeneity in pathogen populations affecting HLA fitness– increased FST at HLA for recently diverged populations can be explained.</p><p id="p-75">In <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>, we further explore parameters of population growth (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-34.gif"/></noscript></span></span>) and difference in equilibrium frequencies of alleles under selection after the population split (<em>δ</em>). We show that only when the equilibrium frequency difference after the split (<em>δ</em>) and the selection coefficient (<em>s</em>) are large we recover higher FST under balancing selection than under neutrality (negative <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-35.gif"/></noscript></span></span>) for recent divergence times, and lower FST under balancing selection than under neutrality (positive <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-36.gif"/></noscript></span></span>) for longer divergence times. Different equilibrium frequencies prior to the population split (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-37.gif"/></noscript></span></span>) are explored in Figure S3, and we show that high FST at low divergence times is attained irrespective of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/embed/mml-math-38.gif"/></noscript></span></span></p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Difference between  under neutrality and  under overdominance (), for different selective scenarios. For all scenarios shown here, we used an initial equilibrium frequency of 0.5 (), and varied population split times, size fold change (), selective coefficient (s) and difference of equilibrium frequencies after split (&#x3B4;). Notice that for the case of a small change in equilibrium frequency after the split (), the  under neutrality is always greater then the  under divergent overdominance ( for every time after the divergence). For higher values of frequency difference () and selection coefficient ( the  under neutrality is smaller than the  under divergent overdominance () for times closer to the split of the populations while for times far from the split the  under neutrality becomes greater then the  under divergent overdominance ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315295289" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Difference between &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-39&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under neutrality and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under overdominance (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-41&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), for different selective scenarios. For all scenarios shown here, we used an initial equilibrium frequency of 0.5 (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and varied population split times, size fold change (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), selective coefficient (&lt;em&gt;s&lt;/em&gt;) and difference of equilibrium frequencies after split (&lt;em&gt;&#x3B4;&lt;/em&gt;). Notice that for the case of a small change in equilibrium frequency after the split (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under neutrality is always greater then the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under divergent overdominance (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for every time after the divergence). For higher values of frequency difference (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and selection coefficient (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under neutrality is smaller than the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under divergent overdominance (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-52&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) for times closer to the split of the populations while for times far from the split the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under neutrality becomes greater then the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under divergent overdominance (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2805/F9.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/8/2805/F9.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2805/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490533" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-76" class="first-child">Difference between <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-39.gif"/></noscript></span></span> under neutrality and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-40.gif"/></noscript></span></span> under overdominance (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-41.gif"/></noscript></span></span>), for different selective scenarios. For all scenarios shown here, we used an initial equilibrium frequency of 0.5 (<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-42.gif"/></noscript></span></span>), and varied population split times, size fold change (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-43.gif"/></noscript></span></span>), selective coefficient (<em>s</em>) and difference of equilibrium frequencies after split (<em>δ</em>). Notice that for the case of a small change in equilibrium frequency after the split (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-44.gif"/></noscript></span></span>), the <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-45.gif"/></noscript></span></span> under neutrality is always greater then the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-46.gif"/></noscript></span></span> under divergent overdominance (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-47.gif"/></noscript></span></span> for every time after the divergence). For higher values of frequency difference (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-48.gif"/></noscript></span></span>) and selection coefficient (<span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-49.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-50.gif"/></noscript></span></span> under neutrality is smaller than the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-51.gif"/></noscript></span></span> under divergent overdominance (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-52.gif"/></noscript></span></span>) for times closer to the split of the populations while for times far from the split the <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-53.gif"/></noscript></span></span> under neutrality becomes greater then the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-54.gif"/></noscript></span></span> under divergent overdominance (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/F9/embed/mml-math-55.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-77">Consistent with our results, recent studies based on signatures of selection other than FST have identified very recent adaptive change at HLA loci (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Field <em>et al.</em> 2016</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhou <em>et al.</em> 2016</a>), a process which could contribute to differentiation at a local scale without erasing signatures of long-term balancing selection.</p></div><div id="sec-21" class="subsection"><h3>Identifying selective regimes on HLA loci</h3><p id="p-78">Selection on HLA genes is now widely understood as a complex process, for which selective regimes may shift over time in intensity, biological targets, or mode of selection. For example, while only long-term balancing selection can account for features of HLA variation such as <em>trans</em>-species polymorphism and SFS with a marked excess of intermediate frequency variants (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bitarello <em>et al.</em> 2018</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Leffler <em>et al.</em> 2013</a>), there are also clear signals of recent positive selection, such as local ancestry deviation in recently admixed populations (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Meyer <em>et al.</em> 2018</a>, Table 3) and a high Singleton Density Score (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Field <em>et al.</em> 2016</a>). Further, there are recent studies indicating that different selective regimes can act simultaneously on HLA genes, depending on the biological level of organization examined (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lighten <em>et al.</em> 2017</a>).</p><p id="p-79">In this study we explored population differentiation at the HLA genes, and show that a divergent overdominance selective regime can reproduce the relationship between population differentiation and divergence time we observed at HLA genes (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a>). In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we summarize our predictions of how other modes of selection could affect population differentiation, and we detail those predictions below.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490571/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490571/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Modes of balancing selection and their effects on population differentiation. Balancing selection regimes simulated in this study are shared and divergent overdominance. The last column shows the effects of those regimes on population differentiation measured by FST, as well as our predictions of the effects other balancing selection regimes (local and frequency-dependent selection) on FST. <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/T1/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/T1/embed/mml-math-56.gif"/></noscript></span></span> refers to FST under balancing selection and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/T1/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2805/T1/embed/mml-math-57.gif"/></noscript></span></span> refers to FST under neutrality. <em>A</em> and <em>a</em> represent two different alleles at a biallelic locus. <em>M</em> and <em>m</em> represent the major and minor allele at a biallelic locus, at any given time, respectively. <em>p</em> represents the frequency of the most frequent allele (<em>M</em>). <em>s</em> and <em>t</em> are selection coefficients, and <em>h</em> is the dominance coefficient. Subscripts 1 and 2 represent population specific parameters</span><div class="sb-div caption-clear"></div></div></div><p id="p-80">Using simulations, <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> (1997)</a> showed that local selection increases population differentiation relative to neutrality, while frequency-dependent selection decreases population differentiation. These results are somewhat intuitive given that in the first case genotype fitness differ among populations, but they are the same in the second case (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We do not expect to see selected sites switching from more to less differentiated than neutral ones (or vice versa) as a function of divergence time in neither of those scenarios. However, such a switch was observed in the case of divergent overdominance, with selected SNPs shifting from greater to lower divergence with respect to neutral sites.</p><p id="p-81">To sum up, other forms of balancing selection can generate lower FST relative to neutrally evolving sites, as we observed at the HLA genes over long periods of time, but only divergent overdominance can also account for the additional feature of increased differentiation among recently diverged populations. Another non-exclusive mechanism that could explain increased differentiation at recently diverged populations, as well as decreased differentiation at longer divergence times, are shifts from balancing to positive selection, which have been reported in other genes (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">de Filippo <em>et al.</em> 2016</a>).</p><p id="p-82">Another dimension to be explored is the biological level of organization on which the selection regimes described in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> can act on HLA genes (<em>e.g.</em>, SNPs, genes, groups of genes). <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lighten <em>et al.</em> (2017)</a> investigated differences in population differentiation on two levels of HLA variation: HLA alleles (which we refer to as “HLA haplotypes”), and HLA supertypes, which are broader categories that encompass groups of HLA haplotypes with similar peptide binding properties. <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Lighten <em>et al.</em> (2017)</a> show that selection acts differently in those two levels, with HLA supertypes being stable over time and showing little differentiation, while the specific HLA alleles belonging to each supertype show extensive turnover (often with marked differences between populations).</p><p id="p-83">We have touched on this issue by looking at one level of organization lower than <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Lighten <em>et al.</em> (2017)</a>, when we contrasted FST at HLA SNPs and haplotypes. We hypothesized that low population differentiation at HLA SNPs could be compatible with high differentiation at HLA haplotypes, relative to neutral sites. This would be expected if particular HLA proteins were locally selected by population-specific pathogens. However, our dataset did not have enough resolution to answer this question, and we found that population differentiation at HLA haplotypes is intermediate between HLA SNPs and neutral sites, and not significantly different from either. Although we did not find evidence for high differentiation at HLA haplotypes with the limited data we had access to, we believe this is an interesting open question that could be addressed when an appropriate dataset becomes available (one with phased HLA haplotypes in multiple populations, along with an extensive set of neutral markers to be used as control).</p><p id="p-84">It is possible that selection acts simultaneously at different levels of organization at the HLA genes, and involves a combination of the regimes described in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. It is important to acknowledge this possibility if we want to fully understand the processes maintaining genetic diversity at the HLA genes.</p></div></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusion</h2><p id="p-85">We have explored the degree of population differentiation at HLA loci and compared it to that of neutrally evolving sites. Our findings confirm that tests for selection based on FST are sensitive to the frequency at which variants are segregating, an effect that is particularly pronounced for the HLA genes, which show highly skewed site frequency spectra (with an excess of intermediate frequency alleles). When this skew is accounted for, we find that over long spans of time HLA SNPs show decreased population differentiation.</p><p id="p-86">We also found that for recent divergence (involving population pairs from the same continent) differentiation at SNPs in the HLA genes was often higher than for neutrally evolving sites. We used simulations to show that a regime of overdominant selection maintained constant for diverging populations cannot explain high differentiation at low divergence times. However, an alternative overdominance model where two divergent populations are under overdominance, but differ in the fitness values of the homozygotes (<em>i.e.</em>, divergent overdominance), reproduces the observed pattern of lower differentiation over long spans of time, and higher at shorter timescales.</p><p id="p-87">A complete and systematic exploration of the different regimes of balancing selection and different levels of biological organization they act upon, along with their interactions with demographic history, represents an important challenge. Our results document that population differentiation can be informative in the process of proposing models and designing tests to understand the selective pressures acting on HLA loci.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-88">We thank one anonymous reviewer and Noah Rosenberg for important suggestions which improved the manuscript. This research was financially supported by the São Paulo Research Foundation (FAPESP), The Brazilian National Council for Scientific and Technological Development (CNPq), The National Institutes of Health (NIH), and the Swiss National Science Foundation (SNSF). DYCB was funded by FAPESP scholarships #2012/22796-9 and #2013/12162-5, JC was funded by FAPESP scholarship #2015/19990-6 and DM has a FAPESP research grant #12/18010-0 a CNPq productivity grant #308167/2012-0 and NIH GM075091. JG was supported by grant 31003A-138180 of the SNSF.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-89">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6671753">https://doi.org/10.25387/g3.6671753</a>.</p></li><li class="fn" id="fn-3"><p id="p-90"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-04-30"><span class="received-label">Received </span>April 30, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-21"><span class="accepted-label">Accepted </span>June 21, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Brandt <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.1" data-doi="10.1534/genetics.116.199141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alcala</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mathematical constraints on F ST: biallelic markers in arbitrarily many populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199141</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199141%26rft_id%253Dinfo%253Apmid%252F28476869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzE1ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2805.2" data-doi="10.1002/9780470015902.a0022863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrés</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Balancing Selection in the Human Genome</span>. <span class="cit-source">Encyclopedia of Life Sciences</span>. John Wiley &amp; Sons, Ltd: Chichester. pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470015902.a0022863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEncyclopedia%2Bof%2BLife%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470015902.a0022863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470015902.a0022863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.3" data-doi="10.1038/ng.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barreiro</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintana-Murci</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural selection has driven population differentiation in modern humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.78</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBarreiro%26rft.auinit1%253DL.%2BB.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D340%26rft.epage%253D345%26rft.atitle%253DNatural%2Bselection%2Bhas%2Bdriven%2Bpopulation%2Bdifferentiation%2Bin%2Bmodern%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.78%26rft_id%253Dinfo%253Apmid%252F18246066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246066&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548400026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.4" data-doi="10.1034/j.1399-0039.2001.057005424.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begovich</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moonsamy</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mack</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barcellos</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic variability and linkage disequilibrium within the HLA-DP region: analysis of 15 different populations.</span> <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1034/j.1399-0039.2001.057005424.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DBegovich%26rft.auinit1%253DA.%2BB.%26rft.volume%253D57%26rft.issue%253D5%26rft.spage%253D424%26rft.epage%253D439%26rft.atitle%253DGenetic%2Bvariability%2Band%2Blinkage%2Bdisequilibrium%2Bwithin%2Bthe%2BHLA-DP%2Bregion%253A%2Banalysis%2Bof%2B15%2Bdifferent%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1399-0039.2001.057005424.x%26rft_id%253Dinfo%253Apmid%252F11556967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1399-0039.2001.057005424.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11556967&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.5" data-doi="10.1086/421051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bersaglieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanderploeg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic Signatures of Strong Recent Positive Selection at the Lactase Gene.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/421051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBersaglieri%26rft.auinit1%253DT.%26rft.volume%253D74%26rft.issue%253D6%26rft.spage%253D1111%26rft.epage%253D1120%26rft.atitle%253DGenetic%2Bsignatures%2Bof%2Bstrong%2Brecent%2Bpositive%2Bselection%2Bat%2Bthe%2Blactase%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F421051%26rft_id%253Dinfo%253Apmid%252F15114531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/421051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15114531&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221651900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.6" data-doi="10.1101/gr.154831.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating and interpreting FST: The impact of rare variants.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1514</span>–<span class="cit-lpage">1521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.154831.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.154831.113%26rft_id%253Dinfo%253Apmid%252F23861382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTUxNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.7" data-doi="10.1016/j.ajhg.2011.07.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genovese</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide Comparison of African-Ancestry Populations from CARe and Other Cohorts Reveals Signals of Natural Selection.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2011.07.025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBhatia%26rft.auinit1%253DG.%26rft.volume%253D89%26rft.issue%253D3%26rft.spage%253D368%26rft.epage%253D381%26rft.atitle%253DGenome-wide%2Bcomparison%2Bof%2BAfrican-ancestry%2Bpopulations%2Bfrom%2BCARe%2Band%2Bother%2Bcohorts%2Breveals%2Bsignals%2Bof%2Bnatural%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.07.025%26rft_id%253Dinfo%253Apmid%252F21907010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.07.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21907010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.8" data-doi="10.1093/gbe/evy054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bitarello</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Filippo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Signatures of Long-Term Balancing Selection in Human Genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">955</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D939%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.9" data-doi="10.1007/s00251-003-0630-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borghans</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beltman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Boer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MHC polymorphism under host-pathogen coevolution.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">732</span>–<span class="cit-lpage">739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00251-003-0630-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DBorghans%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D55%26rft.issue%253D11%26rft.spage%253D732%26rft.epage%253D739%26rft.atitle%253DMHC%2Bpolymorphism%2Bunder%2Bhost-pathogen%2Bcoevolution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-003-0630-5%26rft_id%253Dinfo%253Apmid%252F14722687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-003-0630-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14722687&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189202400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.10" data-doi="10.1534/g3.114.015784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">D. Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguiar</span>  <span class="cit-name-given-names">V. R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitarello</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Mapping bias overestimates reference allele frequencies at the HLA genes in the 1000 Genomes Project phase I data. G3 (Bethesda) <span class="cit-vol">5</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.015784</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.015784%26rft_id%253Dinfo%253Apmid%252F25787242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNS85MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.11" data-doi="10.1111/j.1365-294X.2010.04975.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cammen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Geographic variation of the major histocompatibility complex in Eastern Atlantic grey seals (Halichoerus grypus).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">740</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2010.04975.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCammen%26rft.auinit1%253DK.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D740%26rft.epage%253D752%26rft.atitle%253DGeographic%2Bvariation%2Bof%2Bthe%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bin%2BEastern%2BAtlantic%2Bgrey%2Bseals%2B%2528Halichoerus%2Bgrypus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04975.x%26rft_id%253Dinfo%253Apmid%252F21199032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04975.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21199032&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286837300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.12" data-doi="10.1017/S0016672397002954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672397002954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D70%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D174%26rft.atitle%253DThe%2Beffects%2Bof%2Blocal%2Bselection%252C%2Bbalanced%2Bpolymorphism%2Band%2Bbackground%2Bselection%2Bon%2Bequilibrium%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsubdivided%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002954%26rft_id%253Dinfo%253Apmid%252F9449192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9449192&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071240500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.13" data-doi="10.1038/hdy.2010.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Čížková</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bellocq</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S. J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piálek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryja</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic structure and contrasting selection pattern at two major histocompatibility complex genes in wild house mouse populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DCizkova%26rft.auinit1%253DD.%26rft.volume%253D106%26rft.issue%253D5%26rft.spage%253D727%26rft.epage%253D740%26rft.atitle%253DGenetic%2Bstructure%2Band%2Bcontrasting%2Bselection%2Bpattern%2Bat%2Btwo%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bgenes%2Bin%2Bwild%2Bhouse%2Bmouse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.112%26rft_id%253Dinfo%253Apmid%252F20823902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20823902&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289713700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.14" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.15" data-doi="10.1007/s00251-003-0629-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Boer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borghans</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Boven</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keşmir</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissing</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Heterozygote advantage fails to explain the high degree of polymorphism of the MHC.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00251-003-0629-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.stitle%253DImmunogenetics%26rft.aulast%253DDe%2BBoer%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D55%26rft.issue%253D11%26rft.spage%253D725%26rft.epage%253D731%26rft.atitle%253DHeterozygote%2Badvantage%2Bfails%2Bto%2Bexplain%2Bthe%2Bhigh%2Bdegree%2Bof%2Bpolymorphism%2Bof%2Bthe%2BMHC.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-003-0629-y%26rft_id%253Dinfo%253Apmid%252F14722686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-003-0629-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14722686&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189202400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.16" data-doi="10.1093/molbev/msw023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Filippo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Key</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghirotto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benazzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meneu</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recent selection changes in human genes under long-term balancing selection</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1435</span>–<span class="cit-lpage">1447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw023%26rft_id%253Dinfo%253Apmid%252F26831942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26831942&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.17" data-doi="10.1038/256050a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doherty</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinkernagel</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Enhanced immunological surveillance in mice heterozygous at the H-2 gene complex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">256</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/256050a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDoherty%26rft.auinit1%253DP.%2BC.%26rft.volume%253D256%26rft.issue%253D5512%26rft.spage%253D50%26rft.epage%253D52%26rft.atitle%253DEnhanced%2Bimmunological%2Bsurveillance%2Bin%2Bmice%2Bheterozygous%2Bat%2Bthe%2BH-2%2Bgene%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F256050a0%26rft_id%253Dinfo%253Apmid%252F1079575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/256050a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1079575&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AG55100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.18" data-doi="10.1016/j.tpb.2014.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edge</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Upper bounds on FST in terms of the frequency of the most frequent allele and total homozygosity: The case of a specified number of alleles.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2014.08.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D97%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2014.08.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2014.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.19" data-doi="10.1038/ncomms1632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eizaguirre</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenz</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milinski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rapid and adaptive evolution of MHC genes under parasite selection in experimental vertebrate populations.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DEizaguirre%26rft.auinit1%253DC.%26rft.volume%253D3%26rft.spage%253D621%26rft.epage%253D621%26rft.atitle%253DRapid%2Band%2Badaptive%2Bevolution%2Bof%2BMHC%2Bgenes%2Bunder%2Bparasite%2Bselection%2Bin%2Bexperimental%2Bvertebrate%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1632%26rft_id%253Dinfo%253Apmid%252F22233631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22233631&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.20" data-doi="10.1371/journal.pone.0049837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elhaik</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Empirical distributions of F(ST) from large-scale human polymorphism data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0049837</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElhaik%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De49837%26rft.epage%253De49837%26rft.atitle%253DEmpirical%2BDistributions%2Bof%2BF%2528ST%2529%2Bfrom%2BLarge-Scale%2BHuman%2BPolymorphism%2BData.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049837%26rft_id%253Dinfo%253Apmid%252F23185452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185452&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaulton</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detection of human adaptation during the past 2,000 years.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">0776</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D0776%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.22" data-doi="10.1111/j.1471-8286.2004.00828.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Hierfstat, a package for R to compute and test hierarchical F-statistics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1471-8286.2004.00828.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D5%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2004.00828.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2004.00828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227598500059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.23" data-doi="10.1371/journal.pone.0097282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gourraud</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khankhanian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cereb</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feolo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">HLA Diversity in the 1000 Genomes Dataset.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e97282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0097282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De97282%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0097282%26rft_id%253Dinfo%253Apmid%252F24988075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0097282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24988075&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.24" data-doi="10.1554/0014-3820(2002)056[1902:PRAGVA]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Pathogen resistance and genetic variation at MHC loci.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1902</span>–<span class="cit-lpage">1908</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2002.tb00116.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHedrick%26rft.auinit1%253DP.%2BW.%26rft.volume%253D56%26rft.issue%253D10%26rft.spage%253D1902%26rft.epage%253D1908%26rft.atitle%253DPathogen%2Bresistance%2Band%2Bgenetic%2Bvariation%2Bat%2BMHC%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282002%2529056%255B1902%253APRAGVA%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12449477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2002)056[1902:PRAGVA]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12449477&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179241700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evidence for balancing selection at HLA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHedrick%26rft.auinit1%253DP.%2BW.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D456%26rft.atitle%253DEVIDENCE%2BFOR%2BBALANCING%2BSELECTION%2BAT%2BHLA%26rft_id%253Dinfo%253Apmid%252F6884768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA0LzMvNDQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-8.8.2805.26" data-doi="10.1038/352595a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allsopp</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anstey</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twumasi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-comment">Common west African HLA antigens are associated with protection from severe malaria</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/352595a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHill%26rft.auinit1%253DA.%2BV.%26rft.volume%253D352%26rft.issue%253D6336%26rft.spage%253D595%26rft.epage%253D600%26rft.atitle%253DCommon%2Bwest%2BAfrican%2BHLA%2Bantigens%2Bare%2Bassociated%2Bwith%2Bprotection%2Bfrom%2Bsevere%2Bmalaria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F352595a0%26rft_id%253Dinfo%253Apmid%252F1865923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/352595a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1865923&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GB21100046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.27" data-doi="10.1186/1471-2164-13-107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary forces shaping genomic islands of population differentiation in humans.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHofer%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.spage%253D107%26rft.epage%253D107%26rft.atitle%253DEvolutionary%2Bforces%2Bshaping%2Bgenomic%2Bislands%2Bof%2Bpopulation%2Bdifferentiation%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-107%26rft_id%253Dinfo%253Apmid%252F22439654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22439654&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.28" data-doi="10.1016/S0198-8859(01)00212-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollenbach</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Vina</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erlich</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">HLA Diversity, Differentiation, and Haplotype Evolution in Mesoamerican Natives.</span> <abbr class="cit-jnl-abbrev">Hum. Immunol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">378</span>–<span class="cit-lpage">390</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0198-8859(01)00212-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bimmunology%26rft.stitle%253DHum%2BImmunol%26rft.aulast%253DHollenbach%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D62%26rft.issue%253D4%26rft.spage%253D378%26rft.epage%253D390%26rft.atitle%253DHLA%2Bdiversity%252C%2Bdifferentiation%252C%2Band%2Bhaplotype%2Bevolution%2Bin%2BMesoamerican%2BNatives.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0198-8859%252801%252900212-9%26rft_id%253Dinfo%253Apmid%252F11295471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0198-8859(01)00212-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11295471&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168062900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.29" data-doi="10.1038/ng.2809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">New loci associated with chronic hepatitis B virus infection in Han Chinese.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2809</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D1499%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2809%26rft_id%253Dinfo%253Apmid%252F24162738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24162738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.30" data-doi="10.1038/335167a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Pattern of nucleotide substitution at major histocompatibility complex class I loci reveals overdominant selection.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/335167a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHughes%26rft.auinit1%253DA.%2BL.%26rft.volume%253D335%26rft.issue%253D6186%26rft.spage%253D167%26rft.epage%253D170%26rft.atitle%253DPattern%2Bof%2Bnucleotide%2Bsubstitution%2Bat%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bclass%2BI%2Bloci%2Breveals%2Boverdominant%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F335167a0%26rft_id%253Dinfo%253Apmid%252F3412472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/335167a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3412472&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988P964700056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.31" data-doi="10.1038/hdy.2014.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of purifying selection on patterns of genetic differentiation between Drosophila melanogaster populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.80</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.80%26rft_id%253Dinfo%253Apmid%252F25227256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25227256&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.32" data-doi="10.1534/genetics.112.144758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edge</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The relationship between F(ST) and the frequency of the most frequent allele.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144758</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144758%26rft_id%253Dinfo%253Apmid%252F23172852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.33" data-doi="10.1056/NEJM200009073431006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The HLA System.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">702</span>–<span class="cit-lpage">709</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJM200009073431006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DKlein%26rft.auinit1%253DJ.%26rft.volume%253D343%26rft.issue%253D10%26rft.spage%253D702%26rft.epage%253D709%26rft.atitle%253DThe%2BHLA%2BSystem-%2BFirst%2Bof%2BTwo%2BParts%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJM200009073431006%26rft_id%253Dinfo%253Apmid%252F10974135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJM200009073431006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10974135&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089100500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.34" data-doi="10.1038/335268a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ennis</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parham</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">HLA-A and B polymorphisms predate the divergence of humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/335268a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLawlor%26rft.auinit1%253DD.%2BA.%26rft.volume%253D335%26rft.issue%253D6187%26rft.spage%253D268%26rft.epage%253D271%26rft.atitle%253DHLA-A%2Band%2BB%2Bpolymorphisms%2Bpredate%2Bthe%2Bdivergence%2Bof%2Bhumans%2Band%2Bchimpanzees.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F335268a0%26rft_id%253Dinfo%253Apmid%252F3412487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/335268a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3412487&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q047700065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.35" data-doi="10.1126/science.1234070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ségurel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple instances of ancient balancing selection shared between humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">1578</span>–<span class="cit-lpage">1582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1234070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1234070%26rft_id%253Dinfo%253Apmid%252F23413192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNy8xNTc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.37" data-doi="10.1126/science.1153717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southwick</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casto</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Worldwide human relationships inferred from genome-wide patterns of variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1153717</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D319%26rft.issue%253D5866%26rft.spage%253D1100%26rft.epage%253D1104%26rft.atitle%253DWorldwide%2BHuman%2BRelationships%2BInferred%2Bfrom%2BGenome-Wide%2BPatterns%2Bof%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153717%26rft_id%253Dinfo%253Apmid%252F18292342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2Ni8xMTAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.38" data-doi="10.1038/s41467-017-01183-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lighten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopulos</span>  <span class="cit-name-given-names">A. S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammed</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolutionary genetics of immunological supertypes reveals two faces of the Red Queen.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-017-01183-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D1294%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-01183-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-01183-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.39" data-doi="10.1111/j.1365-294X.2009.04105.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loiseau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garnier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chastel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julliard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Diversifying selection on MHC class i in the house sparrow (Passer domesticus).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2009.04105.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLoiseau%26rft.auinit1%253DC.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D1331%26rft.epage%253D1340%26rft.atitle%253DDiversifying%2Bselection%2Bon%2BMHC%2Bclass%2BI%2Bin%2Bthe%2Bhouse%2Bsparrow%2B%2528Passer%2Bdomesticus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04105.x%26rft_id%253Dinfo%253Apmid%252F19368641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04105.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19368641&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.40" data-doi="10.1111/j.1399-0039.2010.01466.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">S. G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodmer</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontrop</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nomenclature for factors of the HLA system, 2010.</span> <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1399-0039.2010.01466.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DMarsh%26rft.auinit1%253DS.%2BG.%26rft.volume%253D75%26rft.issue%253D4%26rft.spage%253D291%26rft.epage%253D455%26rft.atitle%253DNomenclature%2Bfor%2Bfactors%2Bof%2Bthe%2BHLA%2Bsystem%252C%2B2010.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-0039.2010.01466.x%26rft_id%253Dinfo%253Apmid%252F20356336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-0039.2010.01466.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20356336&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275732300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.41" data-doi="10.1093/molbev/mss187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Purifying selection modulates the estimates of population differentiation and confounds genome-wide comparisons across single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3617</span>–<span class="cit-lpage">3623</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mss187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss187%26rft_id%253Dinfo%253Apmid%252F22826460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22826460&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.42" data-doi="10.1111/j.1365-294X.2010.04950.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCairns</span>  <span class="cit-name-given-names">R. J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourget</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Putative causes and consequences of MHC variation within and between locally adapted stickleback demes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2010.04950.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMcCairns%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D486%26rft.epage%253D502%26rft.atitle%253DPutative%2Bcauses%2Band%2Bconsequences%2Bof%2BMHC%2Bvariation%2Bwithin%2Band%2Bbetween%2Blocally%2Badapted%2Bstickleback%2Bdemes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04950.x%26rft_id%253Dinfo%253Apmid%252F21134013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04950.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21134013&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-thesis" id="cit-8.8.2805.43"><div class="cit-metadata unstructured">Mendes, F. H. K., 2013 <em>Seleção natural em genes HLA e seu efeito sobre regiões adjacentes do genoma</em>. Ph.D. thesis, Universidade de São Paulo.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.44" data-doi="10.1534/genetics.105.052837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Single</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mack</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erlich</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Signatures of demographic history and natural selection in the human major histocompatibility complex Loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2121</span>–<span class="cit-lpage">2142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.052837</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.052837%26rft_id%253Dinfo%253Apmid%252F16702436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIxMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.45" data-doi="10.1046/j.1469-1809.2001.6510001.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">How selection shapes variation of the human major histocompatibility complex: a review.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1469-1809.2001.6510001.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DMeyer%26rft.auinit1%253DD.%26rft.volume%253D65%26rft.issue%253DPt%2B1%26rft.spage%253D1%26rft.epage%253D26%26rft.atitle%253DHow%2Bselection%2Bshapes%2Bvariation%2Bof%2Bthe%2Bhuman%2Bmajor%2Bhistocompatibility%2Bcomplex%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-1809.2001.6510001.x%26rft_id%253Dinfo%253Apmid%252F11415519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-1809.2001.6510001.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11415519&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168307900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.46" data-doi="10.1007/s00251-017-1017-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitor</span>  <span class="cit-name-given-names">V. R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitarello</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">D. Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A genomic perspective on HLA evolution.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00251-017-1017-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.volume%253D70%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-017-1017-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-017-1017-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.47" data-doi="10.1111/j.1365-294X.2010.04771.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic diversity and differentiation at MHC genes in island populations of tuatara (Sphenodon spp.).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3894</span>–<span class="cit-lpage">3908</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2010.04771.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMiller%26rft.auinit1%253DH.%2BC.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D3894%26rft.epage%253D3908%26rft.atitle%253DGenetic%2Bdiversity%2Band%2Bdifferentiation%2Bat%2BMHC%2Bgenes%2Bin%2Bisland%2Bpopulations%2Bof%2Btuatara%2B%2528Sphenodon%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04771.x%26rft_id%253Dinfo%253Apmid%252F20723045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04771.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723045&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281797400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.48" data-doi="10.1554/0014-3820(2001)055[1532:COPSOG]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muirhead</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Consequences of population structure on genes under balancing selection.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1532</span>–<span class="cit-lpage">1541</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2001.tb00673.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMuirhead%26rft.auinit1%253DC.%2BA.%26rft.volume%253D55%26rft.issue%253D8%26rft.spage%253D1532%26rft.epage%253D1541%26rft.atitle%253DConsequences%2Bof%2Bpopulation%2Bstructure%2Bon%2Bgenes%2Bunder%2Bbalancing%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1532%253ACOPSOG%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11580013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1532:COPSOG]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11580013&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171149800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-thesis" id="cit-8.8.2805.49"><div class="cit-metadata unstructured">Nunes, K., 2011 <em>Populações ameríndias da América do Sul: estudo multi-locus e inferência histórico demográfica e seletiva</em>. Ph.D. thesis, Universidade de São Paulo.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.50" data-doi="10.1111/j.1365-294X.2008.04015.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornulier</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piertney</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spatio-temporal variation in the strength and mode of selection acting on major histocompatibility complex diversity in water vole (Arvicola terrestris) metapopulations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2008.04015.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOliver%26rft.auinit1%253DM.%2BK.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D80%26rft.epage%253D92%26rft.atitle%253DSpatio-temporal%2Bvariation%2Bin%2Bthe%2Bstrength%2Band%2Bmode%2Bof%2Bselection%2Bacting%2Bon%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bdiversity%2Bin%2Bwater%2Bvole%2B%2528Arvicola%2Bterrestris%2529%2Bmetapopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.04015.x%26rft_id%253Dinfo%253Apmid%252F19140966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.04015.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19140966&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.51" data-doi="10.1016/j.cub.2005.04.050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prugnolle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guernier</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Population</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Pathogen-Driven Selection and Worldwide HLA Class I Diversity.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1022</span>–<span class="cit-lpage">1027</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2005.04.050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPrugnolle%26rft.auinit1%253DF.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1022%26rft.epage%253D1027%26rft.atitle%253DPathogen-driven%2Bselection%2Band%2Bworldwide%2BHLA%2Bclass%2BI%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.04.050%26rft_id%253Dinfo%253Apmid%252F15936272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.04.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15936272&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229984100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.52" data-doi="10.1021/ja00342a019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Estimation of the coancestry coefficient: basis for a short-term genetic distance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DReynolds%26rft.auinit1%253DJ.%26rft.volume%253D105%26rft.issue%253D3%26rft.spage%253D767%26rft.epage%253D779%26rft.atitle%253DESTIMATION%2BOF%2BTHE%2BCOANCESTRY%2BCOEFFICIENT%253A%2BBASIS%2BFOR%2BA%2BSHORT-TERM%2BGENETIC%2BDISTANCE%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja00342a019%26rft_id%253Dinfo%253Apmid%252F17246175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzMvNzY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.53" data-doi="10.1186/1471-2148-12-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzburger</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Uninformative polymorphisms bias genome scans for signatures of selection.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-12-94</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DRoesti%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DUninformative%2Bpolymorphisms%2Bbias%2Bgenome%2Bscans%2Bfor%2Bsignatures%2Bof%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-12-94%26rft_id%253Dinfo%253Apmid%252F22726891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-12-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726891&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.54" data-doi="10.1111/j.1399-0039.2006.00802.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Mazas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An apportionment of human HLA diversity.</span> <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1399-0039.2006.00802.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2BAntigens%26rft.volume%253D69%26rft.spage%253D198%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-0039.2006.00802.x%26rft_id%253Dinfo%253Apmid%252F17445200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-0039.2006.00802.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17445200&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245754300052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.55" data-doi="10.1017/S0016672300004535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vekemans</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The effect of subdivision on variation at multi-allelic loci under balancing selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300004535</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSchierup%26rft.auinit1%253DM.%2BH.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D62%26rft.atitle%253DThe%2Beffect%2Bof%2Bsubdivision%2Bon%2Bvariation%2Bat%2Bmulti-allelic%2Bloci%2Bunder%2Bbalancing%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004535%26rft_id%253Dinfo%253Apmid%252F11006634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11006634&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089181400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">H. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Overdominant <em>vs.</em> Frequency-Dependent Selection at MHC Loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D132%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.57" data-doi="10.1016/j.humimm.2008.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mack</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Single</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Balancing selection and heterogeneity across the classical human leukocyte antigen loci: a meta-analytic review of 497 population studies.</span> <abbr class="cit-jnl-abbrev">Hum. Immunol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.humimm.2008.05.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bimmunology%26rft.stitle%253DHum%2BImmunol%26rft.aulast%253DSolberg%26rft.auinit1%253DO.%2BD.%26rft.volume%253D69%26rft.issue%253D7%26rft.spage%253D443%26rft.epage%253D464%26rft.atitle%253DBalancing%2Bselection%2Band%2Bheterogeneity%2Bacross%2Bthe%2Bclassical%2Bhuman%2Bleukocyte%2Bantigen%2Bloci%253A%2Ba%2Bmeta-analytic%2Breview%2Bof%2B497%2Bpopulation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.humimm.2008.05.001%26rft_id%253Dinfo%253Apmid%252F18638659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.humimm.2008.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18638659&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258089000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Allelic genealogy under overdominant and frequency-dependent selection and polymorphism of major histocompatibility complex loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D967%26rft.epage%253D978%26rft.atitle%253DAllelic%2Bgenealogy%2Bunder%2Boverdominant%2Band%2Bfrequency-dependent%2Bselection%2Band%2Bpolymorphism%2Bof%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bloci.%26rft_id%253Dinfo%253Apmid%252F2323559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzQvOTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2805.59" data-doi="10.1038/nature15393"><div class="cit-metadata"><cite>The 1000 Genomes Project Consortium, <span class="cit-pub-date">2015</span> A global reference for human genetic variation. Nature <span class="cit-vol">526</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature15393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2805.60" data-doi="10.1126/science.1195271"><div class="cit-metadata"><cite>The International HIV Controllers Study, <span class="cit-pub-date">2010</span> The Major Genetic Determinants of HIV-1 Control Affect HLA Class I Peptide Presentation. Science <span class="cit-vol">330</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1557</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1195271</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1195271%26rft_id%253Dinfo%253Apmid%252F21051598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMC8xNTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.61" data-doi="10.1038/sj.gene.6364390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vanderborght</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pacheco</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moraes</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoni</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">HLA-DRB1*04 and DRB1*10 are associated with resistance and susceptibility, respectively, in Brazilian and Vietnamese leprosy patients.</span> <abbr class="cit-jnl-abbrev">Genes Immun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.gene.6364390</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Band%2Bimmunity%26rft.stitle%253DGenes%2BImmun%26rft.aulast%253DVanderborght%26rft.auinit1%253DP.%2BR.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D320%26rft.epage%253D324%26rft.atitle%253DHLA-DRB1%2A04%2Band%2BDRB1%2A10%2Bare%2Bassociated%2Bwith%2Bresistance%2Band%2Bsusceptibility%252C%2Brespectively%252C%2Bin%2BBrazilian%2Band%2BVietnamese%2Bleprosy%2Bpatients.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.gene.6364390%26rft_id%253Dinfo%253Apmid%252F17396103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.gene.6364390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17396103&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247115900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.62" data-doi="10.1111/mec.13204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Does GST underestimate genetic differentiation from marker data?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3546</span>–<span class="cit-lpage">3558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D3546%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.63" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20601685&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.64" data-doi="10.1111/j.1558-5646.1984.tb05657.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb05657.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1984.tb05657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1984.tb05657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.65" data-doi="10.1093/molbev/msq277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A genome-wide search for signals of high-altitude adaptation in Tibetans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msq277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq277%26rft_id%253Dinfo%253Apmid%252F20961960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msq277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20961960&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287112300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2805.66" data-doi="10.1371/journal.pgen.1005847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Strong Selection at MHC in Mexicans since Admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005847%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005847%26rft_id%253Dinfo%253Apmid%252F26863142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26863142&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2805.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2781" title="Cotton Late Embryogenesis Abundant (LEA2) Genes Promote Root Growth and Confer Drought Stress Tolerance in Transgenic Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2817" title="On the Relationship Between High-Order Linkage Disequilibrium and Epistasis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2805.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hB3O-77hv895B99mJOR4qszY7AMHh2ib2RhSIX5uA6s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2805" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2805" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2805" class="active" data-icon-position="" data-hide-link-title="0">The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490447" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gRrCICF9SRRTR5hYMH5r_ealLE0SZfMUAprz55pCKUY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2805.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2805" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_9EJeffWbjDWIHJyzjjLk5GGg_e9kqveZZ4JWw0BKkM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2805" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2805.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490447" id="node490447--2" data-pisa="ggg;8/8/2805" data-pisa-master="ggg;g3.118.200367" data-apath="/ggg/8/8/2805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Débora Y. C. Brandt</span>, <span class="highwire-citation-author" data-delta="1">Jônatas César</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5318-7601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Goudet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diogo Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2805-2815;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200367 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490447--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Débora Y. C. Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A9bora%2BY.%2BC.%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20DY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD%25C3%25A9bora%2BY.%2BC.%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jônatas César </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B4natas%2BC%C3%A9sar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=C%C3%A9sar%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B4natas%2BC%25C3%25A9sar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jérôme Goudet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolution, University of Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Swiss Institute of Bioinformatics, University of Lausanne, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGoudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGoudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5318-7601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Goudet</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a>
<a href="mailto:jerome.goudet@unil.ch" class="" data-icon-position="" data-hide-link-title="0">jerome.goudet@unil.ch</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diogo Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diogo%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiogo%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diogo Meyer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a>
<a href="mailto:jerome.goudet@unil.ch" class="" data-icon-position="" data-hide-link-title="0">jerome.goudet@unil.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2805" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2805&amp;title=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2805&amp;title=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2805&amp;title=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2805&amp;text=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2805&amp;title=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2805&amp;t=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2805&amp;title=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2805&amp;title=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490447/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490447" id="node490447--3" data-pisa="ggg;8/8/2805" data-pisa-master="ggg;g3.118.200367" data-apath="/ggg/8/8/2805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effect of Balancing Selection on Population Differentiation: A Study with HLA Genes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Débora Y. C. Brandt</span>, <span class="highwire-citation-author" data-delta="1">Jônatas César</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5318-7601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Goudet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diogo Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2805-2815;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200367 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490447--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Débora Y. C. Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A9bora%2BY.%2BC.%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20DY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD%25C3%25A9bora%2BY.%2BC.%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jônatas César </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B4natas%2BC%C3%A9sar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=C%C3%A9sar%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B4natas%2BC%25C3%25A9sar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jérôme Goudet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolution, University of Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Swiss Institute of Bioinformatics, University of Lausanne, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGoudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGoudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5318-7601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Goudet</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a>
<a href="mailto:jerome.goudet@unil.ch" class="" data-icon-position="" data-hide-link-title="0">jerome.goudet@unil.ch</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diogo Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departamento de Genética e Biologia Evolutiva, Universidade de São Paulo, São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diogo%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiogo%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diogo Meyer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a>
<a href="mailto:jerome.goudet@unil.ch" class="" data-icon-position="" data-hide-link-title="0">jerome.goudet@unil.ch</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2805&amp;rft.epage=2815&amp;rft.atitle=The%20Effect%20of%20Balancing%20Selection%20on%20Population%20Differentiation%3A%20A%20Study%20with%20HLA%20Genes&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Brandt%2C+D%C3%A9bora+Y.+C.&amp;rft.au=C%C3%A9sar%2C+J%C3%B4natas&amp;rft.au=Goudet%2C+J%C3%A9r%C3%B4me&amp;rft.au=Meyer%2C+Diogo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490447/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490447/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490447/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490447/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490447/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490447/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490447/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490447/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490447/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490447/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490447/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490447/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2805" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2805.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2805.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2805.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2805.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
