<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/8/2709" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/8/2709.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/8/2709.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/8/2709.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/8/2709/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/8/2709.atom" />
<meta name="HW.pisa" content="ggg;8/8/2709" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding" />
<meta name="DC.Identifier" content="10.1534/g3.118.200171" />
<meta name="DC.Date" content="2018-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Humble et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6713975" />
<meta name="DC.Description" content="Recent advances in high throughput sequencing have transformed the study of wild organisms by facilitating the generation of high quality genome assemblies and dense genetic marker datasets. These resources have the potential to significantly advance our understanding of diverse phenomena at the level of species, populations and individuals, ranging from patterns of synteny through rates of linkage disequilibrium (LD) decay and population structure to individual inbreeding. Consequently, we used PacBio sequencing to refine an existing Antarctic fur seal ( Arctocephalus gazella ) genome assembly and genotyped 83 individuals from six populations using restriction site associated DNA (RAD) sequencing. The resulting hybrid genome comprised 6,169 scaffolds with an N50 of 6.21 Mb and provided clear evidence for the conservation of large chromosomal segments between the fur seal and dog ( Canis lupus familiaris ). Focusing on the most extensively sampled population of South Georgia, we found that LD decayed rapidly, reaching the background level by around 400 kb, consistent with other vertebrates but at odds with the notion that fur seals experienced a strong historical bottleneck. We also found evidence for population structuring, with four main Antarctic island groups being resolved. Finally, appreciable variance in individual inbreeding could be detected, reflecting the strong polygyny and site fidelity of the species. Overall, our study contributes important resources for future genomic studies of fur seals and other pinnipeds while also providing a clear example of how high throughput sequencing can generate diverse biological insights at multiple levels of organization." />
<meta name="DC.Contributor" content="Emily Humble" />
<meta name="DC.Contributor" content="Kanchon K. Dasmahapatra" />
<meta name="DC.Contributor" content="Alvaro Martinez-Barrio" />
<meta name="DC.Contributor" content="Inês Gregório" />
<meta name="DC.Contributor" content="Jaume Forcada" />
<meta name="DC.Contributor" content="Ann-Christin Polikeit" />
<meta name="DC.Contributor" content="Simon D. Goldsworthy" />
<meta name="DC.Contributor" content="Michael E. Goebel" />
<meta name="DC.Contributor" content="Jörn Kalinowski" />
<meta name="DC.Contributor" content="Jochen B. W. Wolf" />
<meta name="DC.Contributor" content="Joseph I. Hoffman" />
<meta name="article:published_time" content="2018-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recent advances in high throughput sequencing have transformed the study of wild organisms by facilitating the generation of high quality genome assemblies and dense genetic marker datasets. These resources have the potential to significantly advance our understanding of diverse phenomena at the level of species, populations and individuals, ranging from patterns of synteny through rates of linkage disequilibrium (LD) decay and population structure to individual inbreeding. Consequently, we used PacBio sequencing to refine an existing Antarctic fur seal (&lt;i&gt;Arctocephalus gazella&lt;/i&gt;) genome assembly and genotyped 83 individuals from six populations using restriction site associated DNA (RAD) sequencing. The resulting hybrid genome comprised 6,169 scaffolds with an N50 of 6.21 Mb and provided clear evidence for the conservation of large chromosomal segments between the fur seal and dog (&lt;i&gt;Canis lupus familiaris&lt;/i&gt;). Focusing on the most extensively sampled population of South Georgia, we found that LD decayed rapidly, reaching the background level by around 400 kb, consistent with other vertebrates but at odds with the notion that fur seals experienced a strong historical bottleneck. We also found evidence for population structuring, with four main Antarctic island groups being resolved. Finally, appreciable variance in individual inbreeding could be detected, reflecting the strong polygyny and site fidelity of the species. Overall, our study contributes important resources for future genomic studies of fur seals and other pinnipeds while also providing a clear example of how high throughput sequencing can generate diverse biological insights at multiple levels of organization.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/08/01" />
<meta name="citation_mjid" content="ggg;8/8/2709" />
<meta name="citation_id" content="8/8/2709" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/8/2709" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/8/2709.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/8/2709.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/8/2709.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200171" />
<meta name="citation_pmid" content="29954843" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2709" />
<meta name="citation_lastpage" content="2722" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Emily Humble" />
<meta name="citation_author_institution" content="Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany" />
<meta name="citation_author_institution" content="British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom" />
<meta name="citation_author_email" content="emily.lhumble@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6841-1269" />
<meta name="citation_author" content="Kanchon K. Dasmahapatra" />
<meta name="citation_author_institution" content="Department of Biology, University of York, Heslington, York, YO10 5DD, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2840-7019" />
<meta name="citation_author" content="Alvaro Martinez-Barrio" />
<meta name="citation_author_institution" content="Science for Life Laboratories and Department of Cell and Molecular Biology, Uppsala University, Uppsala, Sweden" />
<meta name="citation_author" content="Inês Gregório" />
<meta name="citation_author_institution" content="Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany" />
<meta name="citation_author" content="Jaume Forcada" />
<meta name="citation_author_institution" content="British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2115-0150" />
<meta name="citation_author" content="Ann-Christin Polikeit" />
<meta name="citation_author_institution" content="Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany" />
<meta name="citation_author" content="Simon D. Goldsworthy" />
<meta name="citation_author_institution" content="South Australian Research and Development Institute, 2 Hamra Avenue, West Beach, South Australia, 5024, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4988-9085" />
<meta name="citation_author" content="Michael E. Goebel" />
<meta name="citation_author_institution" content="Antarctic Ecosystem Research Division, SWFSC, NMFS, NOAA, La Jolla, California 92037-1023 USA" />
<meta name="citation_author" content="Jörn Kalinowski" />
<meta name="citation_author_institution" content="Centrum für Biotechnologie, CeBiTec, University of Bielefeld, Universitätsstr. 27, 33615 Bielefeld, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9052-1998" />
<meta name="citation_author" content="Jochen B. W. Wolf" />
<meta name="citation_author_institution" content="Division of Evolutionary Biology, Faculty of Biology, Ludwig-Maximilians-Universität Munich, Planegg-Martinstried, Germany" />
<meta name="citation_author_institution" content="Science for Life Laboratories and Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2958-5183" />
<meta name="citation_author" content="Joseph I. Hoffman" />
<meta name="citation_author_institution" content="Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany" />
<meta name="citation_author_institution" content="British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5895-8949" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=U. Arnason;citation_title=Comparative chromosome studies in Pinnipedia.;citation_pages=179-226;citation_volume=76;citation_year=1974;citation_issue=2;citation_pmid=4135821;citation_doi=10.1111/j.1601-5223.1974.tb01340.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. Balloux;citation_author=W. Amos;citation_author=T. Coulson;citation_title=Does heterozygosity estimate inbreeding in real populations?;citation_pages=3021-3031;citation_volume=13;citation_year=2004;citation_issue=10;citation_pmid=15367117;citation_doi=10.1111/j.1365-294X.2004.02318.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=V. R. Beklemisheva;citation_author=P. L. Perelman;citation_author=N. A. Lemskaya;citation_author=A. I. Kulemzina;citation_author=A. A. Proskuryakova;citation_title=The ancestral carnivore karyotype as substantiated by comparative chromosome painting of three pinnipeds, the walrus, the steller sea lion and the baikal seal (Pinnipedia, Carnivora).;citation_pages=e0147647;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0147647" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. Benestan;citation_author=T. Gosselin;citation_author=C. Perrier;citation_author=B. Sainte-Marie;citation_author=R. Rochette;citation_title=RAD genotyping reveals fine-scale genetic structuring and provides powerful population assignment in a widely distributed marine species, the American lobster (Homarus americanus).;citation_pages=3299-3315;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13245" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Bérénos;citation_author=P. A. Ellis;citation_author=J. G. Pilkington;citation_author=J. M. Pemberton;citation_title=Genomic analysis reveals depression due to both individual and maternal inbreeding in a free-living mammal population.;citation_pages=3152-3168;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13681" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=C. A. Bonin;citation_author=M. E. Goebel;citation_author=J. Forcada;citation_author=R. S. Burton;citation_author=J. I. Hoffman;citation_title=Unexpected genetic differentiation between recently recolonized populations of a long-lived and highly vagile marine mammal.;citation_pages=3701-3712;citation_volume=3;citation_year=2013;citation_pmid=24198934;citation_doi=10.1002/ece3.732" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol. Sociobiol.;citation_author=C. A. Bonin;citation_author=M. E. Goebel;citation_author=J. I. Hoffman;citation_author=R. S. Burton;citation_title=High male reproductive success in a low-density Antarctic fur seal (Arctocephalus gazella) breeding colony.;citation_pages=597-604;citation_volume=68;citation_year=2014;citation_doi=10.1007/s00265-013-1674-7" />
<meta name="citation_reference" content="Bonner, W. N., 1964 The Fur Seal of South Georgia. British Antarctic Survey Report 1–95." />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=S. Bourgeois;citation_author=H. Senn;citation_author=J. Kaden;citation_author=J. B. Taggart;citation_author=R. Ogden;citation_title=Single-nucleotide polymorphism discovery and panel characterization in the African forest elephant.;citation_pages=113;citation_volume=48;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=B. W. Bowen;citation_author=A. L. Bass;citation_author=L. Soares;citation_author=R. J. Toonen;citation_title=Conservation implications of complex population structure: lessons from the loggerhead turtle (Caretta caretta).;citation_pages=2389-2402;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969722;citation_doi=10.1111/j.1365-294X.2005.02598.x" />
<meta name="citation_reference" content="citation_journal_title=Antarct. Sci.;citation_author=I. Boyd;citation_title=Pup production and distribution of breeding Antarctic fur seals (Arctocephalus gazella) at South Georgia.;citation_pages=17-24;citation_volume=5;citation_year=1993;citation_doi=10.1017/S0954102093000045" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. S. Carlson;citation_author=M. A. Eberle;citation_author=M. J. Rieder;citation_author=Q. Yi;citation_author=L. Kruglyak;citation_title=Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.;citation_pages=106-120;citation_volume=74;citation_year=2004;citation_issue=1;citation_pmid=14681826;citation_doi=10.1086/381000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. R. Chapman;citation_author=S. Nakagawa;citation_author=D. W. Coltman;citation_author=J. Slate;citation_author=B. C. Sheldon;citation_title=A quantitative review of heterozygosity-fitness correlations in animal populations.;citation_pages=2746-2765;citation_volume=18;citation_year=2009;citation_issue=13;citation_pmid=19500255;citation_doi=10.1111/j.1365-294X.2009.04247.x" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=N. Chen;citation_author=E. J. Cosgrove;citation_author=R. Bowman;citation_author=J. W. Fitzpatrick;citation_author=A. G. Clark;citation_title=Genomic consequences of population decline in the endangered Florida scrub-jay.;citation_pages=2974-2979;citation_volume=26;citation_year=2016;citation_doi=10.1016/j.cub.2016.08.062" />
<meta name="citation_reference" content="citation_title=Such is the Antarctic;citation_author=L. Christensen;citation_year=1935" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=D. W. Coltman;citation_author=J. Slate;citation_title=Microsatellite measures of inbreeding: a meta-analysis.;citation_pages=971-983;citation_volume=57;citation_year=2003;citation_issue=5;citation_pmid=12836816;citation_doi=10.1554/0014-3820(2003)057[0971:MMOIAM]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. W. Coltman;citation_author=J. G. Pilkington;citation_author=J. A. Smith;citation_author=J. M. Pemberton;citation_title=Parasite-mediated selection against inbred soay sheep in a free-living island populaton.;citation_pages=1259-1267;citation_volume=53;citation_year=1999;citation_doi=10.2307/2640828" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. A. Conte;citation_author=T. D. Kocher;citation_title=An improved genome reference for the African cichlid, Metriaclima zebra.;citation_pages=724;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1930-5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=V. P. Cruz;citation_author=M. Vera;citation_author=B. G. Pardo;citation_author=J. Taggart;citation_author=P. Martinez;citation_title=Identification and validation of single nucleotide polymorphisms as tools to detect hybridization and population structure in freshwater stingrays.;citation_pages=550-556;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12564" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Y. Dutheil;citation_author=S. Gaillard;citation_author=E. H. Stukenbrock;citation_title=MafFilter: a highly flexible and extensible multiple genome alignment files processor.;citation_pages=53;citation_volume=15;citation_year=2014;citation_pmid=24447531;citation_doi=10.1186/1471-2164-15-53" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. Ekblom;citation_author=J. Galindo;citation_title=Applications of next generation sequencing in molecular ecology of non-model organisms.;citation_pages=1-15;citation_volume=107;citation_year=2011;citation_issue=1;citation_pmid=21139633;citation_doi=10.1038/hdy.2010.152" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=R. Ekblom;citation_author=J. B. W. Wolf;citation_title=A field guide to whole-genome sequencing, assembly and annotation.;citation_pages=1026-1042;citation_volume=7;citation_year=2014;citation_pmid=25553065;citation_doi=10.1111/eva.12178" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=H. Ellegren;citation_title=GENOME SEQUENCING AND POPULATION GENOMICS IN NON-MODEL ORGANISMS;citation_pages=51-63;citation_volume=29;citation_year=2014;citation_pmid=24139972;citation_doi=10.1016/j.tree.2013.09.008" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. C. English;citation_author=S. Richards;citation_author=Y. Han;citation_author=M. Wang;citation_author=V. Vee;citation_title=Mind the Gap: Upgrading Genomes with Pacific Biosciences RS Long-Read Sequencing Technology.;citation_pages=e47768-e47768;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185243;citation_doi=10.1371/journal.pone.0047768" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. D. Etter;citation_author=S. Bassham;citation_author=P. A. Hohenlohe;citation_author=E. A. Johnson;citation_author=W. A. Cresko;citation_title=SNP discovery and genotyping for evolutionary genetics using RAD sequencing.;citation_pages=157-178;citation_volume=772;citation_year=2011;citation_pmid=22065437;citation_doi=10.1007/978-1-61779-228-1_9" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. A. Ferguson-Smith;citation_author=V. Trifonov;citation_title=Mammalian karyotype evolution.;citation_pages=950-962;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=18007651;citation_doi=10.1038/nrg2199" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. D. Foote;citation_author=Y. Liu;citation_author=G. W. C. Thomas;citation_author=T. Vinar;citation_author=J. Alföldi;citation_title=Convergent evolution of the genomes of marine mammals.;citation_pages=272-275;citation_volume=47;citation_year=2015;citation_pmid=25621460;citation_doi=10.1038/ng.3198" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Forcada;citation_author=J. I. Hoffman;citation_title=Climate change selects for heterozygosity in a declining fur seal population.;citation_pages=462-465;citation_volume=511;citation_year=2014;citation_pmid=25056064;citation_doi=10.1038/nature13542" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=E. D. Fountain;citation_author=J. N. Pauli;citation_author=B. N. Reid;citation_author=P. J. Palsbøll;citation_author=M. Z. Peery;citation_title=Finding the right coverage: the impact of coverage and sequence quality on single nucleotide polymorphism genotyping error rates.;citation_pages=966-978;citation_volume=16;citation_year=2016;citation_pmid=26946083;citation_doi=10.1111/1755-0998.12519" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. M. Francis;citation_title=pophelper: an R package and web app to analyse and visualize population structure.;citation_pages=27-32;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12509" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. MacCallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_title=Polar Biology;citation_journal_abbrev=Polar Biology;citation_author=S. D. Goldsworthy;citation_author=J. McKenzie;citation_author=B. Page;citation_author=M. L. Lancaster;citation_author=P. D. Shaughnessy;citation_title=Fur seals at Macquarie Island; post-sealing colonisation, trends in abundance and hybridisation of three species;citation_pages=1473-1486;citation_volume=32;citation_year=2009;citation_issue=10;citation_doi=10.1007/s00300-009-0645-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. M. Gray;citation_author=J. M. Granka;citation_author=C. D. Bustamante;citation_author=N. B. Sutter;citation_author=A. R. Boyko;citation_title=Linkage disequilibrium and demographic history of wild and domestic canids.;citation_volume=181;citation_year=2009;citation_pmid=19189949;citation_doi=10.1534/genetics.108.098830" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_author=I. Gustavsson;citation_title=The chromosomes of the dog.;citation_pages=187-189;citation_volume=51;citation_year=1964;citation_doi=10.1111/j.1601-5223.1964.tb01926.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. J. Haasl;citation_author=B. A. Payseur;citation_title=Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites.;citation_pages=158-171;citation_volume=106;citation_year=2011;citation_issue=1;citation_pmid=20332809;citation_doi=10.1038/hdy.2010.21" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. Hagenblad;citation_author=M. Olsson;citation_author=H. G. Parker;citation_author=E. A. Ostrander;citation_author=H. Ellegren;citation_title=Population genomics of the inbred Scandinavian wolf.;citation_pages=1341-1351;citation_volume=18;citation_year=2009;citation_issue=7;citation_pmid=19368642;citation_doi=10.1111/j.1365-294X.2009.04120.x" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=N. Harmegnies;citation_author=F. Farnir;citation_author=F. Davin;citation_author=N. Buys;citation_author=M. Georges;citation_title=Measuring the extent of linkage disequilibrium in commercial pig populations.;citation_pages=225-231;citation_volume=37;citation_year=2006;citation_issue=3;citation_pmid=16734681;citation_doi=10.1111/j.1365-2052.2006.01438.x" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=S. Hendricks;citation_author=B. Epstein;citation_author=B. Schönfeld;citation_author=C. Wiench;citation_author=R. Hamede;citation_title=Conservation implications of limited genetic diversity and population structure in Tasmanian devils (Sarcophilus harrisii).;citation_pages=977-982;citation_volume=18;citation_year=2017;citation_doi=10.1007/s10592-017-0939-5" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=W. G. Hill;citation_title=Estimation of effective population size from data on linkage disequilibrium.;citation_pages=209-216;citation_volume=38;citation_year=1981;citation_doi=10.1017/S0016672300020553" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variances and covariances of squared linkage disequilibria in finite populations.;citation_pages=54-78;citation_volume=33;citation_year=1988;citation_issue=1;citation_pmid=3376052;citation_doi=10.1016/0040-5809(88)90004-4" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Biol.;citation_author=J. I. Hoffman;citation_author=J. Forcada;citation_title=Extreme natal philopatry in female Antarctic fur seals (Arctocephalus gazella).;citation_pages=71-73;citation_volume=77;citation_year=2012;citation_doi=10.1016/j.mambio.2011.09.002" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. I. Hoffman;citation_author=I. L. Boyd;citation_author=W. Amos;citation_title=Exploring the relationship between parental relatedness and male reproductive success in the Antarctic fur seal Arctocephalus gazella.;citation_pages=2087-2099;citation_volume=58;citation_year=2004;citation_issue=9;citation_pmid=15521464;citation_doi=10.1554/04-099" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. I. Hoffman;citation_author=I. L. Boyd;citation_author=W. Amos;citation_title=Male reproductive strategy and the importance of maternal status in the Antarctic fur seal Arctocephalus gazella.;citation_pages=1917-1930;citation_volume=57;citation_year=2003;citation_issue=8;citation_pmid=14503632;citation_doi=10.1554/02-530" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. I. Hoffman;citation_author=J. Forcada;citation_author=W. Amos;citation_title=Exploring the mechanisms underlying a heterozygosity-fitness correlation for canine size in the Antarctic fur seal Arctocephalus gazella.;citation_volume=101;citation_year=2010;citation_pmid=20457623;citation_doi=10.1093/jhered/esq046" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. I. Hoffman;citation_author=S. M. Grant;citation_author=J. Forcada;citation_author=C. D. Phillips;citation_title=Bayesian inference of a historical bottleneck in a heavily exploited marine mammal.;citation_pages=3989-4008;citation_volume=20;citation_year=2011;citation_issue=19;citation_pmid=21895820;citation_doi=10.1111/j.1365-294X.2011.05248.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. I. Hoffman;citation_author=F. Simpson;citation_author=P. David;citation_author=J. M. Rijks;citation_author=T. Kuiken;citation_title=High-throughput sequencing reveals inbreeding depression in a natural population.;citation_volume=111;citation_year=2014;citation_pmid=24586051;citation_doi=10.1073/pnas.1318945111" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. I. Hoffman;citation_author=M. A. Thorne;citation_author=R. McEwan;citation_author=J. Forcada;citation_author=R. Ogden;citation_title=Cross-amplification and validation of SNPs conserved over 44 million years between seals and dogs.;citation_pages=e68365;citation_volume=8;citation_year=2013;citation_pmid=23874599;citation_doi=10.1371/journal.pone.0068365" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. I. Hoffman;citation_author=P. N. Trathan;citation_author=W. Amos;citation_title=Genetic tagging reveals extreme site fidelity in territorial male Antarctic fur seals Arctocephalus gazella.;citation_pages=3841-3847;citation_volume=15;citation_year=2006;citation_issue=12;citation_pmid=17032279;citation_doi=10.1111/j.1365-294X.2006.03053.x" />
<meta name="citation_reference" content="citation_journal_title=Polar Biol.;citation_author=R. Hucke-Gaete;citation_author=L. P. Osman;citation_author=C. A. Moreno;citation_author=D. Torres;citation_title=Examining natural population growth from near extinction: the case of the Antarctic fur seal at the South Shetlands, Antarctica.;citation_pages=304-311;citation_volume=27;citation_year=2004;citation_doi=10.1007/s00300-003-0587-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Huisman;citation_author=L. E. B. Kruuk;citation_author=P. A. Ellis;citation_author=T. Clutton-Brock;citation_author=J. M. Pemberton;citation_title=Inbreeding depression across the lifespan in a wild mammal population.;citation_volume=113;citation_year=2016;citation_pmid=26979959;citation_doi=10.1073/pnas.1518046113" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=E. Humble;citation_author=A. Martinez-Barrio;citation_author=J. Forcada;citation_author=P. N. Trathan;citation_author=M. A. S. Thorne;citation_title=A draft fur seal genome provides insights into factors affecting SNP validation and how to mitigate them.;citation_pages=909-921;citation_volume=16;citation_year=2016;citation_pmid=26683564;citation_doi=10.1111/1755-0998.12502" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. E. Johnston;citation_author=J. C. McEwan;citation_author=N. K. Pickering;citation_author=J. W. Kijas;citation_author=D. Beraldi;citation_title=Genome-wide association mapping identifies the genetic basis of discrete and quantitative variation in sexual weaponry in a wild sheep population.;citation_pages=2555-2566;citation_volume=20;citation_year=2011;citation_issue=12;citation_pmid=21651634;citation_doi=10.1111/j.1365-294X.2011.05076.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_title=adegenet: a R package for the multivariate analysis of genetic markers.;citation_volume=24;citation_year=2008;citation_pmid=18397895;citation_doi=10.1093/bioinformatics/btn129" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=M. Kardos;citation_author=M. Åkesson;citation_author=T. Fountain;citation_author=Ø. Flagstad;citation_author=O. Liberg;citation_title=Genomic consequences of intensive inbreeding in an isolated wolf population.;citation_pages=124-131;citation_volume=2;citation_year=2018;citation_doi=10.1038/s41559-017-0375-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Kardos;citation_author=A. Husby;citation_author=S. E. McFarlane;citation_author=A. Qvarnström;citation_author=H. Ellegren;citation_title=Whole-genome resequencing of extreme phenotypes in collared flycatchers highlights the difficulty of detecting quantitative trait loci in natural populations.;citation_pages=727-741;citation_volume=16;citation_year=2016;citation_pmid=26649993;citation_doi=10.1111/1755-0998.12498" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. Kardos;citation_author=G. Luikart;citation_author=F. W. Allendorf;citation_title=Measuring individual inbreeding in the age of genomics: marker-based measures are better than pedigrees.;citation_pages=63-72;citation_volume=115;citation_year=2015;citation_pmid=26059970;citation_doi=10.1038/hdy.2015.17" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=M. Kardos;citation_author=H. R. Taylor;citation_author=H. Ellegren;citation_author=G. Luikart;citation_author=F. W. Allendorf;citation_title=Genomics advances the study of inbreeding depression in the wild.;citation_pages=1205-1218;citation_volume=9;citation_year=2016;citation_doi=10.1111/eva.12414" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. Kawakami;citation_author=N. Backström;citation_author=R. Burri;citation_author=A. Husby;citation_author=P. Olason;citation_title=Estimation of linkage disequilibrium and interspecific gene flow in Ficedula flycatchers by a newly developed 50K single-nucleotide polymorphism array.;citation_pages=1248-1260;citation_volume=14;citation_year=2014;citation_pmid=24784959;citation_doi=10.1111/1755-0998.12270" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=L. F. Keller;citation_author=D. M. Waller;citation_title=Inbreeding effects in wild populations.;citation_pages=230-241;citation_volume=17;citation_year=2002;citation_doi=10.1016/S0169-5347(02)02489-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. M. Kiełbasa;citation_author=R. Wan;citation_author=K. Sato;citation_author=P. Horton;citation_author=M. C. Frith;citation_title=Adaptive seeds tame genomic sequence comparison.;citation_volume=21;citation_year=2011;citation_pmid=21209072;citation_doi=10.1101/gr.113985.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv." />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=S. Liu;citation_author=E. D. Lorenzen;citation_author=M. Fumagalli;citation_author=B. Li;citation_author=K. Harris;citation_title=Population genomics reveal recent speciation and rapid evolutionary adaptation in polar bears.;citation_pages=785-794;citation_volume=157;citation_year=2014;citation_pmid=24813606;citation_doi=10.1016/j.cell.2014.03.054" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. M. Malenfant;citation_author=D. W. Coltman;citation_author=C. S. Davis;citation_title=Design of a 9K illumina BeadChip for polar bears (Ursus maritimus) from RAD and transcriptome sequencing.;citation_pages=587-600;citation_volume=15;citation_year=2015;citation_pmid=25187336;citation_doi=10.1111/1755-0998.12327" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=T. C. Marshall;citation_author=D. W. Coltman;citation_author=J. M. Pemberton;citation_author=J. Slate;citation_author=J. A. Spalton;citation_title=Estimating the prevalence of inbreeding from incomplete pedigrees;citation_pages=1533-1539;citation_volume=269;citation_year=2002;citation_issue=1500;citation_pmid=12184822;citation_doi=10.1098/rspb.2002.2035" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=D. E. McCauley;citation_title=Genetic consequences of local population extinction and recolonization.;citation_pages=5-8;citation_volume=6;citation_year=1991;citation_issue=1;citation_pmid=21232411;citation_doi=10.1016/0169-5347(91)90139-O" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=S. D. McKay;citation_author=R. D. Schnabel;citation_author=B. M. Murdoch;citation_author=L. K. Matukumalli;citation_author=J. Aerts;citation_title=Whole genome linkage disequilibrium maps in cattle.;citation_pages=74-74;citation_volume=8;citation_year=2007;citation_pmid=17961247;citation_doi=10.1186/1471-2156-8-74" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=B. H. McRae;citation_author=P. Beier;citation_author=L. E. Dewald;citation_author=L. Y. Huynh;citation_author=P. Keim;citation_title=Habitat barriers limit gene flow and illuminate historical events in a wide-ranging carnivore, the American puma.;citation_pages=1965-1977;citation_volume=14;citation_year=2005;citation_issue=7;citation_pmid=15910319;citation_doi=10.1111/j.1365-294x.2005.02571.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. R. S. Meadows;citation_author=E. K. F. Chan;citation_author=J. W. Kijas;citation_title=Linkage disequilibrium compared between five populations of domestic sheep.;citation_pages=61-61;citation_volume=9;citation_year=2008;citation_pmid=18826649;citation_doi=10.1186/1471-2156-9-61" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. M. Miller;citation_author=R. M. Malenfant;citation_author=P. David;citation_author=C. S. Davis;citation_author=J. Poissant;citation_title=Estimating genome-wide heterozygosity: effects of demographic history and marker type.;citation_pages=240-247;citation_volume=112;citation_year=2014;citation_pmid=24149650;citation_doi=10.1038/hdy.2013.99" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. M. Miller;citation_author=J. Poissant;citation_author=J. W. Kijas;citation_author=D. W. Coltman;citation_title=A genome-wide set of SNPs detects population substructure and long range linkage disequilibrium in wild sheep.;citation_pages=314-322;citation_volume=11;citation_year=2011;citation_doi=10.1111/j.1755-0998.2010.02918.x" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=J. M. Miller;citation_author=J. Poissant;citation_author=R. M. Malenfant;citation_author=J. T. Hogg;citation_author=D. W. Coltman;citation_title=Temporal dynamics of linkage disequilibrium in two populations of bighorn sheep.;citation_pages=3401-3412;citation_volume=5;citation_year=2015;citation_doi=10.1002/ece3.1612" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=D. W. Mohr;citation_author=A. Naguib;citation_author=N. Weisenfeld;citation_author=V. Kumar;citation_author=P. Shah;citation_title=Improved de novo genome assembly: synthetic long read sequencing combined with optical mapping produce a high quality mammalian genome at relatively low cost.;citation_year=2017;citation_doi=10.1101/128348" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. A. Morin;citation_author=G. Luikart;citation_author=R. K. Wayne;citation_title=SNPs in ecology, evolution and conservation.;citation_pages=208-216;citation_volume=19;citation_year=2004;citation_doi=10.1016/j.tree.2004.01.009" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=W. Nie;citation_author=J. Wang;citation_author=W. Su;citation_author=D. Wang;citation_author=A. Tanomtong;citation_title=Chromosomal rearrangements and karyotype evolution in carnivores revealed by chromosome painting.;citation_pages=17-27;citation_volume=108;citation_year=2011;citation_doi=10.1038/hdy.2011.107" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=P. Nietlisbach;citation_author=L. F. Keller;citation_author=G. Camenisch;citation_author=F. Guillaume;citation_author=P. Arcese;citation_title=Pedigree-based inbreeding coefficient explains more variation in fitness than heterozygosity at 160 microsatellites in a wild bird population.;citation_volume=284;citation_year=2017;citation_pmid=28250184;citation_doi=10.1098/rspb.2016.2763" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. Ogden;citation_author=K. Gharbi;citation_author=N. Mugue;citation_author=J. Martinsohn;citation_author=H. Senn;citation_title=Sturgeon conservation genomics: SNP discovery and validation using RAD sequencing.;citation_pages=3112-3123;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12234" />
<meta name="citation_reference" content="citation_journal_title=Norsk Geografisk Tidsskrift -. Nor. Geogr. Tidsskr.;citation_author=O. Olstad;citation_title=Trekk av Sydishavets dyreliv.;citation_pages=511-534;citation_volume=2;citation_year=1928;citation_doi=10.1080/00291952808551483" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=A. J. Osborne;citation_author=R. Brauning;citation_author=J. K. Schultz;citation_author=M. A. Kennedy;citation_author=J. Slate;citation_title=Development of a predicted physical map of microsatellite locus positions for pinnipeds, with wider applicability to the Carnivora.;citation_pages=503-513;citation_volume=11;citation_year=2011;citation_pmid=21481208;citation_doi=10.1111/j.1755-0998.2010.02962.x" />
<meta name="citation_reference" content="citation_journal_title=Polar Biol.;citation_author=B. Page;citation_author=A. Welling;citation_author=M. Chambellant;citation_author=S. D. Goldsworthy;citation_author=T. Dorr;citation_title=Population status and breeding season chronology of Heard Island fur seals.;citation_pages=219-224;citation_volume=26;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=M. R. Payne;citation_title=Growth of a fur seal population.;citation_volume=279;citation_year=1977;citation_doi=10.1098/rstb.1977.0072" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. Pendleton;citation_author=R. Sebra;citation_author=A. W. C. Pang;citation_author=A. Ummat;citation_author=O. Franzen;citation_title=Assembly and diploid architecture of an individual human genome via single-molecule technologies.;citation_pages=780-786;citation_volume=12;citation_year=2015;citation_pmid=26121404;citation_doi=10.1038/nmeth.3454" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=L. Peters;citation_author=E. Humble;citation_author=N. Kröcker;citation_author=B. Fuchs;citation_author=J. Forcada;citation_title=Born blonde: a recessive loss-of-function mutation in the melanocortin 1 receptor is associated with cream coat coloration in Antarctic fur seals.;citation_pages=5705-5717;citation_volume=6;citation_year=2016;citation_doi=10.1002/ece3.2290" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. W. Poelstra;citation_author=H. Ellegren;citation_author=J. B. W. Wolf;citation_title=An extensive candidate gene approach to speciation: diversity, divergence and linkage disequilibrium in candidate pigmentation genes across the European crow hybrid zone.;citation_pages=467-473;citation_volume=111;citation_year=2013;citation_pmid=23881172;citation_doi=10.1038/hdy.2013.68" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=W. Pootakham;citation_author=C. Sonthirod;citation_author=C. Naktang;citation_author=P. Ruang-Areerate;citation_author=T. Yoocha;citation_title=De novo hybrid assembly of the rubber tree genome reveals evidence of paleotetraploidy in Hevea species.;citation_pages=41457;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep41457" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=R. Poplin;citation_author=V. Ruano-Rubio;citation_author=M. A. DePristo;citation_author=T. J. Fennell;citation_author=M. O. Carneiro;citation_title=Scaling accurate genetic variant discovery to tens of thousands of samples.;citation_year=2017;citation_doi=10.1101/201178" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. J. Puechmaille;citation_title=The program structure does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem.;citation_pages=608-627;citation_volume=16;citation_year=2016;citation_doi=10.1111/1755-0998.12512" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Rašić;citation_author=I. Filipović;citation_author=A. R. Weeks;citation_author=A. A. Hoffmann;citation_title=Genome-wide SNPs lead to strong signals of geographic structure and relatedness patterns in the major arbovirus vector, Aedes aegypti.;citation_pages=275;citation_volume=15;citation_year=2014;citation_pmid=24726019;citation_doi=10.1186/1471-2164-15-275" />
<meta name="citation_reference" content="citation_journal_title=Biol. Invasions;citation_author=A. M. Reitzel;citation_author=J. A. Darling;citation_author=J. C. Sullivan;citation_author=J. R. Finnerty;citation_title=Global population genetic structure of the starlet anemone Nematostella vectensis: multiple introductions and implications for conservation policy.;citation_pages=1197-1213;citation_volume=10;citation_year=2007;citation_doi=10.1007/s10530-007-9196-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. L. Remington;citation_author=J. M. Thornsberry;citation_author=Y. Matsuoka;citation_author=L. M. Wilson;citation_author=S. R. Whitt;citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome.;citation_volume=98;citation_year=2001;citation_pmid=11562485;citation_doi=10.1073/pnas.201394398" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. T. Rodríguez-Ramilo;citation_author=J. Wang;citation_title=The effect of close relatives on unsupervised Bayesian clustering algorithms in population genetic structure analysis.;citation_pages=873-884;citation_volume=12;citation_year=2012;citation_pmid=22639868;citation_doi=10.1111/j.1755-0998.2012.03156.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. G. Ross;citation_author=C. Russ;citation_author=M. Costello;citation_author=A. Hollinger;citation_author=N. J. Lennon;citation_title=Characterizing and measuring bias in sequence data.;citation_pages=R51;citation_volume=14;citation_year=2013;citation_pmid=23718773;citation_doi=10.1186/gb-2013-14-5-r51" />
<meta name="citation_reference" content="citation_title=Molecular cloning: a laboratory manual;citation_author=J. Sambrook;citation_author=E. F. Fritsch;citation_author=T. Maniatis;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. E. Seeb;citation_author=G. R. Carvalho;citation_author=L. Hauser;citation_author=K. Naish;citation_author=S. Roberts;citation_title=Single-nucleotide polymorphism (SNP) discovery and applications of SNP genotyping in nonmodel organisms.;citation_pages=1-8;citation_volume=11;citation_year=2011;citation_pmid=21429158;citation_doi=10.1111/j.1755-0998.2010.02979.x" />
<meta name="citation_reference" content="citation_journal_title=Methods. Ecol. Evol.;citation_author=A. B. A. Shafer;citation_author=C. R. Peart;citation_author=S. Tusso;citation_author=I. Maayan;citation_author=A. Brelsford;citation_title=Bioinformatic processing of RAD-seq data dramatically impacts downstream population genetic inference.;citation_pages=907-917;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. Slate;citation_author=P. David;citation_author=K. G. Dodds;citation_author=B. A. Veenvliet;citation_author=B. C. Glass;citation_title=Understanding the relationship between the inbreeding coefficient and multilocus heterozygosity: theoretical expectations and empirical data.;citation_pages=255-265;citation_volume=93;citation_year=2004;citation_issue=3;citation_pmid=15254488;citation_doi=10.1038/sj.hdy.6800485" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Slatkin;citation_title=Linkage disequilibrium--understanding the evolutionary past and mapping the medical future.;citation_pages=477-485;citation_volume=9;citation_year=2008;citation_issue=6;citation_pmid=18427557;citation_doi=10.1038/nrg2361" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. Stapley;citation_author=J. Reger;citation_author=P. G. D. Feulner;citation_author=C. Smadja;citation_author=J. Galindo;citation_title=Adaptation genomics: the next generation.;citation_pages=705-712;citation_volume=25;citation_year=2010;citation_issue=12;citation_pmid=20952088;citation_doi=10.1016/j.tree.2010.09.002" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Stoffel;citation_author=B. A. Caspers;citation_author=J. Forcada;citation_author=A. Giannakara;citation_author=M. Baier;citation_title=Chemical fingerprints encode mother-offspring similarity, colony membership, relatedness, and genetic quality in fur seals.;citation_volume=112;citation_year=2015;citation_pmid=26261311;citation_doi=10.1073/pnas.1506076112" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=M. A. Stoffel;citation_author=M. Esser;citation_author=M. Kardos;citation_author=E. Humble;citation_author=H. Nichols;citation_title=inbreedR: an R package for the analysis of inbreeding based on genetic markers.;citation_pages=1331-1339;citation_volume=7;citation_year=2016;citation_doi=10.1111/2041-210X.12588" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=M. A. Stoffel;citation_author=E. Humble;citation_author=K. Acevedo-Whitehouse;citation_author=B. L. Chilvers;citation_author=B. Dickerson;citation_title=Recent demographic histories and genetic diversity across pinnipeds are shaped by anthropogenic interactions and mediated by ecology and life-history.;citation_year=2018;citation_doi=10.1101/293894" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. Szulkin;citation_author=N. Bierne;citation_author=P. David;citation_title=Heterozygosity-fitness correlations: a time for reappraisal.;citation_pages=1202-1217;citation_volume=64;citation_year=2010;citation_issue=5;citation_pmid=20148954;citation_doi=10.1111/j.1558-5646.2010.00966.x" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=S. M. Townsend;citation_author=I. G. Jamieson;citation_title=Molecular and pedigree measures of relatedness provide similar estimates of inbreeding depression in a bottlenecked population.;citation_pages=889-899;citation_volume=26;citation_year=2013;citation_pmid=23496652;citation_doi=10.1111/jeb.12109" />
<meta name="citation_reference" content="Untergasser, A., I. Cutcutache, T. Koressaar, J. Ye, B. C. Faircloth et al., 2012 Primer3–new capabilities and interfaces. 40: e115." />
<meta name="citation_reference" content="citation_journal_title=R. Soc. Open Sci.;citation_author=D. L. J. Vendrami;citation_author=L. Telesca;citation_author=H. Weigand;citation_author=M. Weiss;citation_author=K. Fawcett;citation_title=RAD sequencing resolves fine-scale population structure in a benthic invertebrate: implications for understanding phenotypic plasticity.;citation_volume=4;citation_year=2017;citation_doi=10.1098/rsos.160548" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=N. Vijay;citation_author=C. M. Bossu;citation_author=J. W. Poelstra;citation_author=M. H. Weissensteiner;citation_author=A. Suh;citation_title=Evolution of heterogeneous genome differentiation across multiple contact zones in a crow species complex.;citation_pages=13195;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms13195" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Wang;citation_title=The computer program structure for assigning individuals to populations: easy to use but easier to misuse.;citation_pages=981-990;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12650" />
<meta name="citation_reference" content="citation_title=A voyage towards the south pole performed in the years 1822–1824. Longman;citation_author=J. Weddell;citation_year=1825" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Mixed self and random mating at two loci.;citation_pages=247-262;citation_volume=21;citation_year=1973;citation_issue=3;citation_pmid=4731639;citation_doi=10.1017/S0016672300013446" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E.-M. Willing;citation_author=C. Dreyer;citation_author=C. van Oosterhout;citation_title=Estimates of genetic differentiation measured by F(ST) do not necessarily require large sample sizes when using many SNP markers.;citation_pages=e42649-e42649;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22905157;citation_doi=10.1371/journal.pone.0042649" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. P. Wynen;citation_author=S. D. Goldsworthy;citation_author=C. Guinet;citation_author=M. N. Bester;citation_author=I. L. Boyd;citation_title=Postsealing genetic variation and population structure of two species of fur seal (Arctocephalus gazella and A. tropicalis).;citation_pages=299-314;citation_volume=9;citation_year=2000;citation_issue=3;citation_pmid=10736028;citation_doi=10.1046/j.1365-294x.2000.00856.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. L. Younger;citation_author=G. V. Clucas;citation_author=D. Kao;citation_author=A. D. Rogers;citation_author=K. Gharbi;citation_title=The challenges of detecting subtle population structure and its importance for the conservation of emperor penguins.;citation_pages=3883-3897;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14172" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=H. Zhang;citation_author=P. Meltzer;citation_author=S. Davis;citation_title=RCircos: an R package for Circos 2D track plots.;citation_pages=244;citation_volume=14;citation_year=2013;citation_pmid=23937229;citation_doi=10.1186/1471-2105-14-244" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
<meta name="twitter:description" content="Recent advances in high throughput sequencing have transformed the study of wild organisms by facilitating the generation of high quality genome assemblies and dense genetic marker datasets. These resources have the potential to significantly advance our understanding of diverse phenomena at the level of species, populations and individuals, ranging from patterns of synteny through rates of linkage disequilibrium (LD) decay and population structure to individual inbreeding. Consequently, we used PacBio sequencing to refine an existing Antarctic fur seal ( Arctocephalus gazella ) genome assembly and genotyped 83 individuals from six populations using restriction site associated DNA (RAD) sequencing. The resulting hybrid genome comprised 6,169 scaffolds with an N50 of 6.21 Mb and provided clear evidence for the conservation of large chromosomal segments between the fur seal and dog ( Canis lupus familiaris ). Focusing on the most extensively sampled population of South Georgia, we found that LD decayed rapidly, reaching the background level by around 400 kb, consistent with other vertebrates but at odds with the notion that fur seals experienced a strong historical bottleneck. We also found evidence for population structuring, with four main Antarctic island groups being resolved. Finally, appreciable variance in individual inbreeding could be detected, reflecting the strong polygyny and site fidelity of the species. Overall, our study contributes important resources for future genomic studies of fur seals and other pinnipeds while also providing a clear example of how high throughput sequencing can generate diverse biological insights at multiple levels of organization." />
<meta name="og-title" property="og:title" content="RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/8/2709" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recent advances in high throughput sequencing have transformed the study of wild organisms by facilitating the generation of high quality genome assemblies and dense genetic marker datasets. These resources have the potential to significantly advance our understanding of diverse phenomena at the level of species, populations and individuals, ranging from patterns of synteny through rates of linkage disequilibrium (LD) decay and population structure to individual inbreeding. Consequently, we used PacBio sequencing to refine an existing Antarctic fur seal ( Arctocephalus gazella ) genome assembly and genotyped 83 individuals from six populations using restriction site associated DNA (RAD) sequencing. The resulting hybrid genome comprised 6,169 scaffolds with an N50 of 6.21 Mb and provided clear evidence for the conservation of large chromosomal segments between the fur seal and dog ( Canis lupus familiaris ). Focusing on the most extensively sampled population of South Georgia, we found that LD decayed rapidly, reaching the background level by around 400 kb, consistent with other vertebrates but at odds with the notion that fur seals experienced a strong historical bottleneck. We also found evidence for population structuring, with four main Antarctic island groups being resolved. Finally, appreciable variance in individual inbreeding could be detected, reflecting the strong polygyny and site fidelity of the species. Overall, our study contributes important resources for future genomic studies of fur seals and other pinnipeds while also providing a clear example of how high throughput sequencing can generate diverse biological insights at multiple levels of organization." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8.cover-source.jpg" />
    <title>RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"p0fZabnAvelPnGay47wfoc_NFNomzY2HT6zcEHc-i3k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490463","apath":"\/ggg\/8\/8\/2709.atom","pisa":"ggg;8\/8\/2709","ac":{"\/ggg\/8\/8\/2709.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/8\/2709.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2709"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/8\/2709"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/8\/2709":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bSj+peoIpRsV1gKof282ahK43hVPaQ0ThCSEST0hCMajrifFQzzXukLCa12kt1qUV0ZtVHGEnsb263rGCOag9PNGe6WpcG1uQek8oGsvtux\\\\\\\/W5iG\\\\\\\/D5LWiM5zjgRCLif1IVRhFfr7XJFkLp7iQNIlsHe2tawq1dZofOwxvwyQg5hOk99KyVXFrcg4ENMDiGp2t5v0DHtyYcAkjZt9X7idrEYvDBdvQbwqGR77JGcaJ5EVekdYV6myJJ8iEX0NVW9ej\\\\\\\/IdR6KkWouuKQ6aixMKomN\\\\\\\/+Om65lBXqvr9+zqfFiW8VfBpE8u+TVOjMXdNjMpQXSno\\\\\\\/\\\\\\\/rjSBIEdH33RikcATdZ5o2LRUk2pgZzlgvmMuPhlN3kXngjHC2xzFJeyOz23X5\\\\\\\/FaTYkY3HWgfiOHWhI5fV+xl2pYOVmwnWSwbhmn\\\\\\\/z3I6cDyiu64TEkTdWXg3x6aYuz962vz9uh2c4d29KB3nGl0So1lNRT9i8HawY4Rp\\\\\\\/T9pORoVN4i6ijuGW\\\\\\\/GlgrhnxSzWoWmIjs23iagqd2E1TRrR1bvH+JE8X311aHeklEp4GjAcv\\\\\\\/BFjiCx7J+7+omzY1lTxHoLeB5ZseTSsuhjEo2sv4yOR\\\\\\\/41Hy4L1JKxpksrOZDogrV0g2tgZ5AIYLTpFVSuE3xrpUz5ZCS6056QjN7kuowVi9tvURd7tnIixSiehiSqw1a1JUSYCAZdHEUPDfoVtvnhCaPo79gZ06UCqlJQwrt+9tlWNjDr6n9uk976nex8cp162chvhuhAA5XYyIDeNg509Uu7tn8GUMDd54kV80br6wLTznxg7jfKfJqpUS+71CM3aiC737nKVSQMr6LFeKiJ9arpRdyG6K8AnadR5fm8oVrIubQkpeG+Q0\\\\\\\/CgDqOoDNxozPODGPGH8rlX\\\\\\\/QRNiGp4GQdI31knCWyOPhE5MbZ\\\\\\\/ripkhFc0whbPJT1T2xabXDvKmsJESKqr\\\\\\\/Bj5p\\\\\\\/NJPVYygQuFkUxwlJkHFHYWbinTL8trMI2jqn1ULX1n0Nuzd7IqREmiB4+yfH+PRXYUCTlB0rqAmfA5EGsKWrFdS0lBu\\\\\\\/OL5nZUKbll7piLUfE6UkRYmHP6G3k5bQnCrqjz5mzSJj04ZzzWD4qirET4bF4oVa9ZZigAIyQkkzw+Lpp9F+OvsNGrVdm01tkbucw\\\\\\\/+h+bIjsrpBuyU7oIsWxYjD7XTOdbDGx9Khm+iwNJe0UYrSmUiwQJZntD+94X59o33n2C2GXl9D5jTocpC3E2aEkR8VzIO2XsMbRSKv\\\\\\\/1hinT0OL+dkhN6iO\\\u0022,\\\u0022iv\\\u0022:\\\u0022w3ofa8aAzMtp+fAnbtb9\\\\\\\/w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e896bad85e4d2b94ea539c1a736dec3b4b6c1b64c2626c27249e71ed49231bf5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/ju+5OFO0Kgc4Ig64VBOQxIj31Sd6mcte\\\\\\\/8U8k8j+bY9zHItNnRXiSZG1Ho+0JTEimnj25s63DLsA+hzYA1UZh1Z\\\\\\\/d7Aa491BknNo\\\\\\\/qJDOKmSxsl+p2urzznUlPLhvLr\\\\\\\/PTdvoxWUkV7LVWUo7eUOKHjYnFaQmf4v0Dgi7g0j\\\\\\\/O7ePRyZBkApcIiwwMi2Zs8CpOuSX8F0rotBuIcTBKxcQLRewi83cpsZ\\\\\\\/w0BFR9bogneq17nOqIFhp\\\\\\\/MA2Vp6fASa5ukH9t1jegvkhV8gNlTyuEsJlKimUXyRAsgy0dSZnQKVDaLTgJaX5PHaBBlO9smphWmRb+Ugtm1IpDsSmBtJiGrfktgNcGWu9knEdNzxsPpvJDUZQnUzMjdiE0BDNvjotsCU292mMJbHU2F+jGbWGpuOH6+tH4vvg9dPzWsAILYlrThOI4g8B2C+tepHBHR+mE4fxpXWr81x3RgmlkoLgokd1AeVHiT49nNV5FqNYcZ1+ovoDI4uNRAPLX8mA5WetLZRBSzJWLhZySLUpal7ctwU3+MYZm1kPX8nUedNo335w81nMh8GdscCtDcnQBEMj1LtAHo9s+MT64IzC+5dn5EH6XXYnpncxhw1B2fXb8tLGaPruug6tj+6WkdwSWG7KcGcgR+M2bcn9Hi2P01muFhgyOmPV9M6w4ieqlKQl85UfG\\\\\\\/SGHex1mq+bi5U\\\\\\\/HN3e4zHozBp\\\\\\\/m0AQei48MPkpchguYVYKxVQxW5LCuJliKFZtSXfk9++oJ\\\\\\\/vvzae3T62Zmf\\\\\\\/gJ5njAQbb0ajXw3eexVsHTvPe3HftWh1S9hsvkumedpEN7Sc0Y8sCM7djsHvsIYWGc0nXGE77frYm13c+VeOhk5DkanVO9uJ9gZ8S4lEyeVHzERu7TS\\\\\\\/96m4uiOQIgIFJvYw4jsp94Mjh4DvlLVIGHKco+Q\\\\\\\/VnkYc54agKVvRdd2e2ioX1xIDDel6L022sgb6JTl+OHoZFNiONJneJM\\\\\\\/gQl+Dwj1WnPhEiZ7mjGzrD9oDsFqzggVz0POcFAO6nQ3dmEc7lXpR+elhPSRmMwBBotEDCZgUlogiMK+IqTwJlz8XZ5ubDK3BRyF8chg5HbQoLSybbj+\\\\\\\/ikW1vpkLprT7ACLuvAMoZ7nBry1fED+J41f9dkoojspxx+Dc\\\\\\\/1fM5z2TojjFBYOikdQsnRdWRxAM6VFPeIedyMrgN1nDpUC2xu5g5xzebQEplZx3q4Zc4TDaDGWe39\\\\\\\/hZ18gZ8mM7DRqBnEfxJHsMpc\\\\\\\/4vvSwE1+JqLSJNvrOTqIwo09JkB6QotrdluZsOHM04Brhaq+eXOJdxhAsq5kC+77cnIas1ndfpX+jSmRTPONgn1Homlr3kGH9iVxzmFl8JsEQUSOhpppW2QewwEknXDFsvbft1PCJ3ZGgysqim\\\\\\\/+dwKtyJFVy5ZIVlGQTzUaih8Qci4u4O8t4\\\\\\\/vnl4n3FtXA5vFlpqeHfAgdJMJobP+cq3mwK+gVmcHF8\\\\\\\/Now9b2JrsBaEE7nKEAFYaGSz58W4b+rlgRk2P81b63GhTGTXipPGjydpLNSdcrKGYKmGHCaQjHmYHuvaOI3Xe7KiAHNDEDC6g6+MH9oKAGJAgSHtbQKVN0xPZ9oYExBVZ8n\\\\\\\/SsVJNnJH9U+pCNbbVUf3uG5keWZeqEnVFDOTL\\\\\\\/uxJ8UOIQdWLSPMjOKEils\\\\\\\/GVUO3qLM+zKHvan2jcXu0XpvKE2Jy6NJonyLKxGD61894Haq0mdVQtL2H7Zu5UkyZb1HYt5nLXbekYHsEtpT99PIPGpDxfyQ0ngb1iPIg5GzJkP3\\\\\\\/7YtxcrKh9ibwRhybVBPkEkQK8jiWQ4S8zAWsW7AhL\\\\\\\/Ld9GXKPN\\\\\\\/5M\\\\\\\/3y9EHaPWwlCWnNzj+DsutvdD8kTTdvCftwMXRi\\\\\\\/En6peoX\\\\\\\/AdQoJoI8K93L\\\\\\\/sBnxBcVDB5xaeL85CupCtCuLRgibi\\\\\\\/DmvIHKYUaAtJol60IXBTI07y8Mtwb4JJ6geVARAu4JI5Qn3WjNIEY+FUdwSdf8doAVuZmMQ9l\\\\\\\/DNJ\\\\\\\/uEnafRRskx3VAHm3xH6KIZG8Z0BI79eQ6AlN4zAjS9fltOHfBAeNhZcNR5\\\\\\\/68uidcp+DJcMJ7QDvuf8GGLOOk5SMUPUzZB46jwJwjwEWDFqyWTaszqarqRc8hg5g384fKm6pnAHg3sf7WZ3VTeFkYIGztwC\\\\\\\/9bCnNeQwQ2mv\\\\\\\/7zXoauo3HI\\\\\\\/kDd\\\\\\\/ZtF7vvRCG4zOqTGD\\\\\\\/\\\\\\\/\\\\\\\/AtLSNFv+y9vpi5GjjYoIEw3k5JMSm3g7gAZETtSBLyB5NnFxSwiGA\\\\\\\/RJGB7R4J5uSwXHnj8Sw1yRk5KuzCXzP+V2ZU8DCFnXPLVS8onrfn+5nedylpUKkJgljuyqyw29OdFfxFfG\\\\\\\/BP9TKxzt6XaQ4IpUrITaOyzOgE5tEctD7Y3p\\\\\\\/OsHryH8lyj\\\\\\\/T5NOBveOTF8ogvclxNYFD\\\\\\\/wR4zdnRswmMmHxe87W23y0h9iipR2ZKEJJVxfJIUSYJNap2k+JsgqL15OwaN14CT1P5pkYOOelisTKuFlnCO15WdUo1HHa2jTQN\\\\\\\/QByYqjYxINwF9mc3oQBEPsS0yOIdk+t8J6kbIYXJI2yzp609OFjx0UqK+ONeb90PxCYUlbZ0L1RZHJxU+0v18DIG7\\\\\\\/4RpNGMcrfrBwOGEhnsJQthMBByccjKLXI2eSc+0ObdpgwuOsqX16A55RkgsWY8KrDB\\\\\\\/taNnfgrcv8ELUWRVeazoSZf2QGMCRbj\\\\\\\/lQxmz79WbG10S4eh+OZKp0e0FCHhKCyjiTtD0h+8ViwJW\\\\\\\/9X3cp6Ap25Wlefua5Cx\\\\\\\/935gJJlKsHFzNMAN6UowhblVPLc2llVFYRMFNTQMudGqV0XjTq13Qx6zjGMtGPAq8nylvv3xR23b5fvcPa0X7jtxdUDAszMke29ozRc9EqcIR7ZGc\\\\\\\/ZYBu4tXqQ2wYf7Xs7n395zDpOJJUV6b2fuWfZsGmBsyZDz4pJIvSZQ2nrSImG4xGDDIbB96YIPR4hcgC+l99SCOof1limqwHJoGQ4pyGDxWpmiwYpUdVRY8OflLQVuGmxr8tVDndVaht5CJtYiPoPdrOF+bPkrAikHKAnUtuT5fRvV+zVfjT8coDKMV1iOuJNncCAwRucUu4I1aGwIMaTjhEhtIZVpvUsW2\\\\\\\/h7xetjRdS1TS35wxqKiQb44FCzgdOIuTgkZA31XeTuSWjNxUM5MOsMx\\\\\\\/Nhk+vpsrFldut8FuTxP0oQxtykU22d2dlntpUqfN2aJ29hfue4o\\\\\\\/920Ez3jvPmp\\\\\\\/aWZeAV6Xf74lLkk8rot2LNgdxaZTtjifXkn9pVFrlllbzcYDScvD5cbA4S\\\\\\\/gQBtIp9m8uN26qw9vA3IaA3uzJcStpk+8knIQhrDgfVxtaTKsemIbBIcFYduiYri5HrkL6xR1gWHgFxfkZlYPZZkn91uXXfi2lO\\\\\\\/P\\\\\\\/7quIU5q+c0UwcFghV\\\\\\\/ThKlQOPcE7d3i1ktzDIelxinnrVqiszB6JpQJP7QAbs1L762XY6a43AsU\\\\\\\/a+UqA2fFWQ9TTp6HyymkAh6malNUWKbMeZIlp+AHO7b\\\\\\\/IeONZDvf5gStHogi3nrlJHbiLQ8yVmeL7xzDc7JVAW2RLT54V50ocllnUYgyGhZQ4vpPaWGd2Q1f6djnzwKMHKk3SnrBm1sHtIImfgRRjagzKCWCvjKzgMJt01qYVRLr9jmoD+kg8w5FKCmvq3wurbCIvDjMth1TQ2J1CDHF2yFU7D+Mz4M2gsDNoy5it6dcr1tgR4hbzyfyEu9rmKysJwhPbphaTDSUz6NMhC8s\\\\\\\/0USG6LFG+97vE224JXUKvBKpfqyZuW\\\\\\\/FBzPhkkfNE3f6+HZcGWRaFEOcFoEE1XQwemkS06CO\\\\\\\/9SSpHIx4fgob1K0hhdoZQu4vPAotA8I\\\\\\\/rjXLtt9DtLBm+1Z3rL3C4i1e30iZV3i1vCyFIaXitP+beJk2Dc39+2j1ZndegSp2mHN\\\\\\\/2l\\\\\\\/ggS9+5FdRsqmlVsFQKK4UQ9SakesSaiD7Ti99pxcRqwQ5kHU26Qt5lf\\\\\\\/qZCTJFU+yc4dBmQDRvvmdRjB2tjCXu2p0sHxqgkYnPSR6jhESk89t3swn1WBHaLwS99cC6BnoNgR3dV3ftRKoY\\\\\\\/iAoT284y3Dg1XUv4WblPMougb7i4xjMaMVbC7k9CaYs5ma3ktzg0hOcw6xMJvxc79BPIUiVFgi5EoNqKU\\\\\\\/wDI0vqdaDaYwaJrSiknfIexwNkywXDE+GZw4bMKMTnOlKMV7dvMG4E\\\\\\\/ZVOx9YmiN0fcTKjrVSb7S7vf0n+zHA\\\\\\\/Im6mt5yr8ogoI9wnxrIx631OQ6UWdfsIBL\\\\\\\/pqUgfbne7qp5vMziJ0XbUuRYCEiQ9gdPrn5rvu6dmfo6N9GaxuJDFpm7QJhJEs5tLUN2kNMsr7bmHbIXNrrORG+lvbuKkPEvTQ03Oblh1too184Pc8IeKqrQNVgjN2KQLoT5biOT\\\\\\\/Zk5lSI7MK6h4GYnMizeCiptVmGs+31598KOR7AZzWF3DnC0EXqkMEi70lDP6ujdiuBjlJ7xxjkeJcwijtSzH0eaROCW8ykz6TmGFtrDdBY\\\\\\\/67MjXT80XVpYznj+QBugw9EUSEiJJkVY+4KDDo2an6jJBrliWeXA+NpUtmbnpAli4KcLp9Ix8fCrqxL2lCqea2fqDJ36y92NR5mnmHFhMPNfFcXd0Nzmf0Zh0orDfLUiM0R1KO9mJx\\\\\\\/OR8rbmIfmAqYgqipyfMsrumQaLeruXQQyZz0cx07muoYIDCAgcdm56ibXIkgzHonsJ3QxNNzB8khypO1C6Up0hcXvo0dWbMhxLYkCXnqR9Og5ahME6sH0qV0\\\\\\\/6j3IH\\\\\\\/USyaX+MlX4TOBAdUOkT0vYbucsgbAydwW+CO+pz\\\\\\\/HWdq8iEMgTBURwTX0yZsvDNQRYOnl3O3vrtoTJtDSmutQmlU2uSe4Q4nAL\\\\\\\/PMMFkgWBSegSugh07oqNErg6f3777SulU\\\\\\\/daul22GxSPMcpzFZ+Qq8WLBIUSSpfhu\\\\\\\/n8GfMQre3pHBTtXzLmSKfLaaixoRdDklPt\\\\\\\/L\\\\\\\/CypmIysv22vvN033db31q+muC+VeUVcjOoAtq0r+3cmOJEbxJvGw7\\\\\\\/lQ3VeedKAgWDceLaThi09nwQCskR9g5TjpNFVqisZhkrXhsv0GIM9p0\\\\\\\/Ae3R3tFeXgGKyO04Cd13y\\\\\\\/07I\\\\\\\/VQkf433Y+MxeEu8eGCJUX8y5nme56NkCm43W1t6ruUtdckK2FOWvpTuO1IcfpUjOOqeH5+Qq088mHoWtApMwUDQWdRB\\\\\\\/DUkV15y1jTzdqif1P68DsxoZ\\\\\\\/UmbRJ\\\\\\\/Vz4flibF8Dfr9f1qMEE7vXMsKCZ702LOhS2QfWeOSgWeYU8MAUXHdn1yiCTvadyJzqfB\\\\\\\/5SZsOwsYn9OGCNdLCndW2P7272Umuy2p7IplF0955C9il61xBvBQGlDO9BXBjEzV87AwoVrYQVGUH3KfLIwgty9V4eP7Al7umG+17qv8PNTP5z9byDp7vFWF6FNTCHqeun0SbKg6DCULdfiEXRk\\\\\\\/Whm0ePGvGFEog5XmHkrHLA3JQVIyVvCUVMGJQr11pPBON277zPv3dcis+Zvc8jhneteos8vFx+TeUYszkxiOXhXZfiC5aAVmKe63Koy6NMXrLqlD0n8ecfMOuiI6N9NL15o6lErOxpDsoCGLqCf5T5mWu1kmAkMo5kFCJQ8TVM2ZLOuZsT\\\\\\\/rUyajq\\\\\\\/TJtHwL6IH8bRDn69jEgKuIuOm20VzyxyxO82b32NF7yXMYHIkPuRn6YZM+anWk4DGYNYaLMDShUoXam3V6yxyjb6OMHvgDWyuE0awLrIjDvMOl+\\\\\\\/cZBFKa1jN3iDgLatVmMjpoe7JoGDN7C9ni0xKSLcfFZSPMkp04KJQEcPLrOh3pg+GnbsR+ecfYcn6jm5rSpUvijKQ\\\\\\\/V3VA5c1tj3PMHrMMcn22S3Bh2yrmpy0UG6OmNZ34Gr8hIEJo4Fg0cZlccE\\\\\\\/Ad9+TORTTnlZs+KBKxFvI\\\\\\\/LYyfXaHdpuhf1QEzBNQJtWK\\\\\\\/SvRw4vY\\\\\\\/h2xDtB4VAwrLkn5Pkl8K1QPtDOZskHr9bAWBuGcpLcJgwo4Pu0m\\\\\\\/0qASqjOTITmaZWUobDXpsNHT3CSEwT2yBAMXsMmKDM3nuCR0lXyCnBOL0l8X6nfgy6PlXtS1PzQANKEFXtZJBuwibuPumc\\\\\\\/8+LX+6d2hcHZzZIlUR8\\\\\\\/7brMuLZVcmVrsuLS1zXcnqddCpngkfXInUG6wt4\\\\\\\/YXswDDKOQazVssij2ARwhiSjkV8BAF+C\\\\\\\/WDEGh90ioOebAzFd3hky+2FlE0ZDNQi6DRiVTNXdyv0dIr3g56QXXeA34z9SUIIwb7m7pmuba\\\\\\\/sOpVxnqT7MM21+zo7ioSdYhWdfJ5I19S4MGvLUwh5qQTHnAdjv9wksRqM447EamxQsuCW2xCi\\\\\\\/q8Tbsl2+RxzAU6ygo7jIqKYjCtnMWnFw8lSK9TQlMDEOSHznTuhnj8Hn49gVhv1ChTrPpN5WcNToLi+nB0RcMSQH\\\\\\\/Vu5X09k\\\\\\\/T2HDQNP8Qm4Bld+lja5FJfPf7JWFUGsI4ZOB5OAPWqbAD9gyJ6GtvmRQNc9rwObjhEh6qEwqNQrtNdNgcmI2fq93AF7Cpazp4fpdkGisMH9EUArpfEhWNUdspH8xhgOidFJ\\\\\\\/lWXjKYLEdFUWfOeUUA4SC8fcKz7u0F8qZN1tv9By407DAwmZf4WP1u+fvFuGvzYfzbMMogWhli0BH2AkVvKk5xjrt+62umk5PY6a\\\\\\\/Y0QHzfUlPol+x27VeEXuuwfSdTEL2VH00KYUWXtgSHShPLC2g+L2NSb+mjpwzbV1SNe2AlZ6xEJ6\\\\\\\/Fqf9Op3oxXaiivVMp2LhE4I7\\\\\\\/BUJ\\\\\\\/ID7\\\\\\\/5nH1qpnhn3Wqj+7Dl48BzPdLOq1wHq97VnQ7A1IJEZZzsCEJDazaSULrTTjAYa03U2Pdu2wPh0ujEfDByDH9JL++wodBTBFjZc+XweBIvbJdes3Qdy8H84p5eKyQhtEg2ZD+8I1gh7NSmcTWwX3tpdbXf81LZRedpRuhG2h146rxsbWqP\\\\\\\/v\\\\\\\/uWNrV0jmhFSo25lObKdwZB90o4nb42uLXHzvGjyUUnoDKWxF\\\\\\\/czUI9O8OG+YNaWbPxmXnaqIycv5w+Vl3H9i+WF4iGfk7HH7Ti\\\\\\\/Fuh4UX0NrLf3uoSmRohq9m4uH3ja3Zp1lSkMiqKT1cB+SaTuFvE6thstfXMBdFrtvKxwVwQwxOQvEfxS4Zxc2NLsFTxnDpSq4BIASLJAgnh0ybmWPDtu25a7mwh9Jy47lXheIK7+At9a4XVZYQkJRCw\\\\\\\/bsBt1xfq5EgU6q5+aahKC9GZ7RwCu6RkYVlt5XcQ8wskBb3fX8WS7Sm0CRKhW3voi8fhJ2ceY8U6l81YWWvT1QdDadq86Py1o4Tj7giy60vk5cLaIn1St\\\\\\\/TSNY1JK6Tm7rnPAcGHXtIqoq+eBnQXpqidi8\\\\\\\/7RJhZTNlhZiaCew7MZaAUDjC9sJk3HgobiJOpIpWkzWF9TfVMEeinYQVahdQQYRjbYr+lh7Ib6ui6czyd0nZBDtXT+qrX93D\\\\\\\/60brxxL1UgUoc2QJ\\\\\\\/lsbmJq4SAQOoaKj1DSL0eUV9H+brZ2NFBNDh2YOGM1U52jg1O80MWo5Ro10rxY3VFidYcC65EFe8AmbWJUtojVM7\\\\\\\/HrWij01ly8l3MVHGkj4VaPdhqERzQ\\\\\\\/7\\\\\\\/GugITwt4iTaE6gskrwsjdaUEiYfZG9yezb1EkpZsvcLUef4jcc62VZIVgmt7yPvDZ5JjrU7lrY2bC3XebZO1ozHDwj+Fwe8zRj+R1LeY4+jLzO6tLXCnyZEHiGKs45o+UqhxGjN96q40kCxwyfZ9ryBurnJwri7NpDXFyUhwEZ4a9tFenMkcVwL907e5+6XQZ6Sn6kmYlMaSMG59e51Ze+dReGWWBDmtmyqxer3V3QZopiOjzPJkqeWNUAayw\\\\\\\/ts123mQvnOwFCd2Hz8WlxL+H5IbKnIjEipY\\\\\\\/GFKlcEMUY8I8TJ\\\\\\\/iZ+AjZ1ju4JxO34FBC1NEEz+am0PabK3qxQxeoq9wxexeuFHPI+xxsLA5y3M+HS3kbVzNx5UmeDviOrmde2YLh4ETrolgCGGz1UyfO34aWyOhj1J90p7WGssglLLDiKJJtAD5t\\\\\\\/WeEVF4E2qKZANj4GNTdQeC5GE8HmwZmfSvmkJE1vnpw9Vji3+1Bc4THwKYcD6JkUdIJr2OEwvJ2+tF2YNccOiicapVftyI0gAE8YQP2v+w6YGbFuDFtesms+E9Mxbus+VZO996011AyN5VoTOh88HcjbiTWn9shCpt3s4LOybK25Smcj45xE9z96Dm7n49HtT2RnNvyHY1f8kUWAYR7ZVvxCmkKMN2EE74VxNh1wEMfOrApKs0l1r+qa5tUQFLo3JixOxjxr+sUh6U9KfW7SX6tcrksKhdKzN2C8CGuq4Da1CuxSp28vzKDQ8g3vUuMcCbGLlyRkAx2qGWEa2JohReEs0tQ3v8jRRVVFXe7rzazDY2BE\\\\\\\/PUGJory\\\\\\\/8TERk1EpJjCxbdrKzvdK2hpjrQ4b38aZ8SG05QSUF2Amvzw+bDhBK69t0Ku6Btd0azJjpwdojf1qgqaXcl8ZN+uKw8X7E48KySHvmxSZoBKJXYBetjv7VeYaqhjwrcjCbN5GEuh2Gf2Baw0H1n7Ikl0Wi9NtxRVXv8xmtPvuJUJ0SUott7QOea3yJsNlz7\\\\\\\/g14RUusC8+0sbAMv2NxdL2GS01hN5LS1MxohuS9eiokR9oSqsUm5KSRr1z6vzjpLvw+M9W7ZUq2zx9t6HSMTK0voFRivEFd9d37MdFHmJCeE3TuvVN0L85BPaV9G0cyIis19t7iFeakiLtKeP2kBGUaaPhh2Zvl7EJjpaadHa4nz9AjcYvX9ODaG1+UViMYghYyezCdQMEJ0hQ+0x\\\\\\\/wFaoxOpp1U9ivuAbVsXxebiOAklP3rNwjNH5bCt6Alcd050FPptCKqZHRWZWF17o6w3SCpyCriivWDfwE2SSe8JPH2IhjCCzTcyc\\\\\\\/FDO7wbjBTyhBv4FgGCDMcVvn4FffStGf5I3YAB1asOFADYKCznMU3BjSm5vjfDGAEldmhpmYM8gpnnHSFhXC+CNTTymHLrQ9xED2MlCGYlhWAcYCJgG5IG0KKHd14bZNNZV3QR78j68SA1vPiRKweIktqwXCL1k\\\\\\\/YZN9NE8\\\\\\\/4UPiOL\\\\\\\/HJwOPEf3oFZoQK05b4Hf3tg4ysLElxeYShXnMN51xnxyCLDfYTjWZ5kPUHJIGvdAR4StpY9BMrpVDXsEOCLMpyNJpsapSQNBdn+q0aoSbYvDFYlRvTnZpJvuaFl2UBWwWjhyEYgnSdziDvKTfYaiPaXq\\\\\\\/QtzRtp6onDWZRtjSDP7g5q5mBtV7GUDGUkyEjRzPGeHra6aTMhGuCxmRa1N+h1KxKmY9t2SdqgLIxkYxAMnccWnHNW+6gWMaRcPlyM0fVj5DLtcC7olOvD4lT44HOKFtpZUQ9grjC2IhveUocDGRUvpdvMMIEMrOt36EZptB7VLF1O4IDwJ73R6L2fA9Ww4PEvLDGhO4mTSg3ZUoE\\\\\\\/EKlKK5kqI9is+Cg9cHD1K1d5z5r9EhZn4hDKIY70REZ0M66DrPo\\\\\\\/sHM9mikra1LJfhURrz\\\\\\\/cQSug7VDtjt9lpm7M+pSpKsGhvmmQKHAQ3D5hJXAtRZJWAI7mjEP74x6DVCve3e4tYo+EzhWHbwcK+r1IOMNElBK89uq5kuNVuRHvGNglbAvfo0S6UWEONMxIr1pAsLWOSYCWvh7hSd5nE5WbRcTOeTlL6KdjB6g2fdq7vKjGsjatQLWf8JSbqmm6JbJMr+wtnV7Sm4ttdfFcoVcTs+6JmMR+JcNJcN7qrcwELzQCfEYLhTyjRXiSGyoU2E5\\\\\\\/F35afzaHtuk1oPusf5wvjJprzXTbQZbheVcD0QMVQHzsGpQSNpA2Ettpez5TUdjNY92WIHdBIDlZ9Icuzt9mLNzecZWuO4toQMrq2taZTUyIoFZM+OBHwmkauCFD1r5WSIZa\\\\\\\/M8vz3uwSxp83dH+iKYnJeU6FkfV9Wr4SYNxYc\\\\\\\/aSm+G7n3QmcgjquyVErNix6Z+vGg\\\\\\\/SpuB5HeaX9YjLKR65qIvq4U9yaOVAq88sbwd23ba\\\\\\\/mi8Uhhexa\\\\\\\/6ZS518rSHYt10hGdvnoutb1qWUiSgmyh0YnM86Zj5l8pGtYNPbUJ9aK+RaFbBdznlb2p9CIuvqXQGTLwrR29sFI2O9pNq875c\\\\\\\/TcBh1NrTeji855mEcO7UXrs1vbeGosnsE633mbElEZ7JRiSbldeUJvVIhVGQrr6\\\\\\\/bjzYEUI381sx4mh+vhCmTG0UxgKcPoq3cMwLq2lTrQCQz5YjaJQ07C1ADZZ9qi2428emShXkFSs15ltOa0rYcD3ubLOOsgcKwbWr07emmbVGYaSOyTM+llWcW\\\\\\\/kApWQwuLwQyH4zumDjkoNEG9hsAyxRb\\\\\\\/s2k0nY1nCH9t578XNmfv4IFJkra9\\\\\\\/49IcIpP\\\\\\\/QLXZM9JdN9aENaSbpr697TSR+FZmCRBvkrGkiMrvqS49VVAVHIe6ddIIqnhaJKYzcwcH0Wuo+xkq57J\\\\\\\/VaCKDDnXXpdv715zVkwbRCsTDQe1kMRFZH\\\\\\\/SzzDAjfRo1jNuEYJ5suRUaRNiQsVqJiagmN\\\\\\\/mWU64ih8tFBUbbOIxnxNzqRelEWUJDXIcR7A6huxO5sJeT5foJIn79AzLTuwY2kKmVJHR3HGgmqEM6qLNwCipaM+oDw5aGIfPd+ERnRs7csN5fcbTMYfk7pOZ9HBC9JQXRKnNVcsYgUy4\\\\\\\/SVKM0\\\\\\\/AmnJWGRI\\\\\\\/LBUARrqUMXmMMhNT1BnAuLcA8V3t2yvq\\\\\\\/vP5M1CAB7pWLQMF6Yx9dRbIqcUdUqf69awmRNuC4QGKA3O1\\\\\\\/CX9+U6tzuK4wN7PYkie3K9QkWkk0z7R5TmEOiywbA0AKBy9+83dZMsOJMdYwiXbtcCs5854ErUTQgvJnMLzz71km+Z2mk2gmlggGGsW7gTJX0d8SMcHmjGnc17W0txpQ6J7CBCzUhbO+z70zPnz\\\\\\\/NBT\\\\\\\/gKaIRl7C1t0Ey2262DuslqNnsOh9DFNHjL8fyLKMOqk9Qyfs5NyMAPFZgVsa4fq6YHPy13imheuFG20UbQwN7iMM8YNbhoaqbLdErxAJ1l\\\\\\\/wM2HKpL2Kvl6PTGKjii\\\\\\\/rTPlwNvBxlDf\\\\\\\/c9q+cRq1pxBFe1bD\\\\\\\/624K+50Gt7QY4hBbgfPYvlMQKSimtBLbQ6pTIwgW61S\\\\\\\/HlLPILe8X+jEicQ9v6amh9NgaSaAWUxSF8eN84KUkbhsrqTqn56R2GCKaCUS16WA0RmEYmtcBbS3asHt7sP\\\\\\\/x078gKMLVaiTArlx0GMh4Ay6NaZtBLfeQY+G5XDppBFCO7p4wTaknEGBxBjixEA14qnkUcoEa0P6GDdVeL04iWjUTFlstDGBdTOIwsP5zik\\\\\\\/QkD9CoQb1PZytmEErvG2V6y3aHyWxZ5JZPeMABwXrhAi0W0JikCT\\\\\\\/LW+qWiCZMpWEo15tAQyV\\\\\\\/JNqdW2R9t9\\\\\\\/qeSF5znf9khlEPAoM0yPuIBlH9FYiuTf8HDKf+HAZqXWlC1Gsq12nEIV0taPb2O4FnvFLzPXteMFOPq7syKCHs699aFb6GrHr5Vrknw3oUFQ6f2RJSOHiJ3Z1aO9GlcjE6DQ+RrNXqmbNk\\\\\\\/YlDD5tEmHCXFH5QbYBiTqVLApzmxtveipvms2khghjlFGq6EHHJiFRHkXK2cA09gNW+7ru38tjd5qecTyMaCU4YUb9Y62PQvGxl06YhjXa7+Bn\\\\\\\/MjG6zWutNAkb7kN7YbdhyW4u2Q1d0pweFQPznGKi4zin9tY7TtZUWfZeqTvVH2em+d4sQqXF4qXLvSbNKgHnIMYJOpkCOhwxvowbaDKp1ovho7xmpoPpilHnq\\\\\\\/7GEmnUwlRDNMnTAjAA2Go60m0Me4Oj4wKZRqrfEKor44g48g29c9ZST0zeEXmR3T9VPyTJgy0blzefIbewISC0VE0QBkItX185P8\\\\\\\/Tu3hAfS8WcUBE7fnWbzFoZsobtD6BwySs\\\\\\\/8YfgpR6dj\\\\\\\/2XrgB4mxJc+JW5obx2tWKH+\\\\\\\/fa\\\\\\\/HSgJGLampEzxk4tXxaY2FjTPNyjnmNlbEuN0yOql1mLkA3ht8Ewp2p9FfxHDOEcznZeUAPDXJE5QCmAmKBa0yh2WEHHji9qPKDnMiyFEZBuLx4FHn\\\\\\\/IzUGaxd03EOOvyfXY4RxFISgHC2TbQsrumyAwtK3bhzTHuSFcYFPMQFjyF\\\\\\\/DQN4B1iRvDeC6bHobuKZN8n+jMPQpKC9\\\\\\\/JGymWvYigp+YLb0Kl5pf1QnBev9PWqNndH4l9keukkQRlUD3tGJulWRqsTvG6dbUqTCtPiSG4y8vybY8NZ9FRw0sIbTcORJPpViH3gv+UwVNcFApPTlhrIK2unmB4k2KpjZfNr7zAu61H0fLjZSBYrbYfxOwO6ynsbXk4LRWKsqFi7PaiVvs5NVgiFBVNtN5LkG92GhA9yvrmhmHCKKGsGhY\\\\\\\/KAd45Zi2S5Vf0VxyTDGCdjeCmaTAbrGLHszImrll3h8WlvY\\\\\\\/GFt1dZKsQXM6L1EUciaj6fOM9eGG3zfgIJl8E50vVtOSfyQtJLurYNZHlyxPmxDILbjHef4YlyO7u4ZTrdxsM\\\\\\\/T3oncmtEeHAi3kkp4MU2JqQ4D0U0NAiqGeCOE3Sgtn3VnKT+nmBdTDqFESDci5On+fMBS8CSW+eQY5d7BPa8n6HNu+EoAmTQxKMmeF8enNJstLsnSoldynM\\\\\\\/BJtOLWYErHqzLqPXAXm0Wh1xO7tiENtj1H7Y2pr\\\\\\\/TfXJflbP5oM7sc7TFRPdsfnJdPCOqrC8196APY\\\\\\\/xxM4P26IA69j19fv7Bu3SpNuXwhL6Q04AyyRpPppvI0YxrUe9qOcnhnvbDZEbwJJqCOdWGkS3z9tN2vFyPO25mHXyJHWALgdVBBpSaZE2tg\\\\\\\/n8JYi03Ialr\\\\\\\/pIIV9TxoLoONvXKGM3YAzoRzK7C\\\\\\\/t9+WlIPkyzhVgxlY54UXb0RtwSyU8u6E8Yo7sxMTz66WnH9jGpX+xssmKAgwwGIyx1KekR4DDnw53iGfS9EExnaCLtE5n979DnoV9+5X4H0tNSXDLUC9407X6qJ5BGmUaUxoKmd+MHAtBL37h3CTTTqmStcwhgRdIwPPi5EhPrmAacQHS845Mr2lfcb+yja\\\\\\\/UvpV2CCrRk1jQrkrECPPqSnUQK5rb2Xgd\\\\\\\/hjUmSxsU1y\\\\\\\/3kKs6xpRvmgX7waZV+GiqyJfHufbfXXY2lQGRlOC3qB1fp8TgTMP0uc3EBE3E2WIxIytUdHfaXI\\\\\\\/2KSLqCebZijbWPyn+8Ma7Vr6NQWGj5WSWljRp72QLhwLqFwttlKN0Eo8sNj1v8tfzEJIlgAuAcfNQaerWlMUhjsSERYLYzteKDCL3+E5fJN1j93cr0kcMPnt\\\\\\\/MrzNRBI5uJgzg2QKbZnVC6sTdK27JAtlAmy77SEY\\\\\\\/tenmYtZb66pp0nd776a0U9uMnJyKOEi3uZP5Q9REg+DfzJqa\\\\\\\/qwqGUukLn\\\\\\\/i27BfJzMBVZwtt53oPiivVsQB2R\\\\\\\/H989uiQp+jWV08SdVJji9ogyTqPpnB+t4ngwVPP5+vUzpYXnvbGpBXxSovB4T+DfxZxaymiyDkRWG\\\\\\\/uG9OKIZg6J6gveEgNBHlhxwqKzxGqAwXNPlBJItxVOcM2MGl03\\\\\\\/a1cEz6Y\\\\\\\/gShf8Cy+G4uadNYPK37XInszn7xSEKRoWzpJLHmstfoom2uRa3sCZEZFou3HfmgW7rqXtzG2sSpBiyUOeAb938GdvdfSR9wldKvEctMvuPGRPwte+\\\\\\\/GxukoK+rQ9Rx7q+LOvchPsOljFGcRf6gRzT7RlZhsgcyVdildlYwgLAdbiv6Evvm1fhQzGo9uLb+nlwBnv5laDS29ON11Xq1I1LVAOkfGXHgTfJ0TGGbfJdCjMxuDQSV7vUBlEAL8H+Q8DGEqb890Z8RwPtaPfDuY0hlG8dH0yRJNr3PNGalO9Jo8uUxFSFy7qdB4ptG6sH+1uoHzBOtxYKuqAU5n2K\\\\\\\/x1pG6Ejx\\\\\\\/sE6h5v\\\\\\\/oHO9i3fjP23Xd94fzRBTY30QxG7QQ2rCbTgC8GfeHaiZYjIqX\\\\\\\/2K\\\\\\\/Y4tyLdGu143QvYO5qselaKuklBdJ+Px4pT4MEwxja2XmF+xe4pIgiIUqvZSDKrbiNf2OSzoV7yTYMrkYqQXZM2cIuCgn4RugAn3KwCjLvA3U3U8t2XJW5L7Q5ojNmEZIzx56Z28zAxL7tEqoFxNm3sh+c6RmguqA9julHovJeK2EYfrPnZLCWo+bcjQABtOxpnmgcrtL6rUr0K4hqY6ZPgvsMdupm1e9t1kxg9Gp1ZGf5GQGx1dREiIxkvWQ+e\\\\\\\/Lb8lPPxJ1Ms03OkwP0TLH4pBrNQ4XPcxpya4QiNoul+806xiOe+wM0jPaGNzkoXjEwFzrcTx7cRynj6eOm4KZOccxnZhbUFQWHVIjNQI\\\\\\\/Fh8861HF078PxCf8bCzvzdfgLW2kFjKrGrvfN7gJlRIMr\\\\\\\/z6CVIJL5v11D0wP5MvtpN3W7muQzaun7kOe5dkcdfw6Ezj6tUnLO7LXscd4cH+EFJC7lt5D60bjyzaxjvg06OtOstyQwo70cq1yZMmFMy3jH5Y1VomYjIsNiIu34IpyWRBBDOMdRXSzS5Cuzajr2jaJ72EhyUw57bhTC99PMIOPesmt5QSvd3izHTm1vfRNhamDYCzH1WB+XFr\\\\\\\/tkdpdF6w++RUztHdn4Cwm7WNK3bjEQ93jWn1g2kCukLoHDgAu\\\\\\\/NyzNpjBv3USMYv2elKCz17x2Sli5UAmIAY9OMAjbxUF54L3XL1HX5nKUGwPellyJAsQKnR8FKonwb0fFJkbEps0BEQelgjK4rPehIAJjKfR5sM\\\\\\\/BxdtKhRHVXlMdGsvGpdjd4taZ89SqAAjJEuPfxNLwOMCN4Vo3Xpm23nXCoK6qeKydEi+OL0TeMTU8fwFIqLzwlWLRS6SR74xJ\\\\\\\/iJnlbV29pLaugRg+98vY8qYL909DSytFs373RIHiSxUflUmPAC3rccXnShR0QoOpLi2wAXZPSi0DLmEhHc6CEahEQiYOwnd7rDidBp1etRpVUPg70kHOYIDSyU2B7jGealNZ2IoK1F6o6ZPpO2NEzzSpJDzEXRkze3Hca3uu\\\\\\\/O+ohW3DM0BPBwzmyinnZEO5BSTL87RZ4qbUTZ88xR\\\\\\\/bY3MJgfSjjOGzASud9ZMZ\\\\\\\/7bsGZY1owQJpVH7SyRw8IijqFjES2HuJ+l\\\\\\\/PdjmQjhedPTx41EnOBcm2aJCGbvfKy6zxMIhvHlB4LP5nVT7si5SB6r+zGwRD66fRpJFFbSEh9ZT1j81ppiCn7B5rfFgL46IqyWyO8MaEZ\\\\\\\/cN51ve5Imd65U4eSaZYNDZscw5\\\\\\\/QwHsp+jPjGZQcP9mgqRIxabhcs2MkH9SqhmPuGNoGEtCgh9Mh3UXM7Q\\\\\\\/VPD\\\\\\\/tk7cuCaBmbSH37cJ4zL4XzcMCgjN42kYjLHlkprCI\\\\\\\/KCNKYbAzLm\\\\\\\/vEjsfBSVsaT1lmNrAtVqho58FBnkQTpWlrMq+67aV7o0T\\\\\\\/dqmF80FXP5KRAZ8jVaJdvIwF9YfoXbCydVo+CatUhn3XmNAe9N5XJFsFF\\\\\\\/9RXjBP2H6igp47L+E2pTAUWjb8LTiGrq1W\\\\\\\/9oEzzw4TBqPmOly6U892+tJowxI03WFr+6CPkfGd9cL5UNtXY8NbyPPmE64Z63c2lSmmjPWjvBSzTG6iU3+SU3SVzGVTFrrfwWciJPdH9bRqf\\\\\\\/oehgpWXwkvpoQcQeppp+MPB34hv\\\\\\\/+ZpOVG7ePTVMwWR3wZNnQRE6ir1oQv7rwKFhfdv\\\\\\\/m\\\\\\\/nS5EpBxWbFfbY+cjpA1hEWEx0uLseH+s4Aq4M\\\\\\\/9RfPEUaNkzK23n1SDhReEc5nXilX41f08npVTniC41I0jWJxDf96UQEAG\\\\\\\/me4TUhpe+MlII09NBbWMjmwt9W0GRNoPv1SETzGNvSKNG+46IGodDVRt6IxTBCeDNXwr8Nzh0B9DaAR6NAokYMKJn9HpWTpjFHjycRsZxk5Zhh5EM3110BkpqMwVmP2+x2+syJbF1rH6Qg\\\\\\\/S0\\\\\\\/i+GgfyXFTwv\\\\\\\/fqdIrBT1489saDq1Ws6VlkDsRHFDbkL4xHQvkmgi31F1qBVdKEFODe+cXA1+NNscxJnyc\\\\\\\/FUb9yjt8pCSl6+dGb9K3wDi6eCJP\\\\\\\/FCTx\\\\\\\/8HxKmM2t6akBmh6ck2WyAF0OekWp4hl5J1GxJrVtkkfBh1U6WN5LSreLE0n4Wa41fmfq8ux6BjTUe36MImb0nSkB+FmQV+LFN18aFijozUM9C8ZNoAKONvjiMJSha+Cnx2Z\\\\\\\/qxuolReqDK1P0YIXpdpc0YMgvY15sVDUkbUbgmn31luw18DMsFV1XqgDos1HtpU5OCEYvSn\\\\\\\/EmiwDNy3lWq4PhWbVOLWKhQfq244l2DU5R8sSpuy\\\\\\\/FpoOvr8Iui3P4WHDUl+QAXMFzfd\\\\\\\/VumZUlS1w65kJzUfQ0uncPnObydhCgj4d9oHSGDQfnWhAgTrOdE\\\\\\\/7jtK4+i25s95wOxw7C30+iNoh6RQ9RRnlyU4wjX4d9qO37ETpd+1eXc\\\\\\\/TFeww4rlspIZegDi36q0DvQhU9i4D\\\\\\\/VExwYhbXSt8DSPjUemc3artrcZqDDHlbNDYt9+mEhRu0JUMz5X1zYhvw44XTFkhwG0RfjWZMmSM6UJMqzQ+xNxYfwHoichtx1JsZHTNMqEpdzIjW6kU\\\\\\\/oX0c97L\\\\\\\/EKjl3IEjSPFZLsvVBStZqwP1usuzMn3BckDVb1ORtgy0DJJtAWnJ3aw5HEUXBKI4o8P+dLLfXL3XkSvSiLA\\\\\\\/FbOpwEC0zB5u0zyL5mMwORMuaMYyerVK3KMqDaH9d8Y078kt+OBtuwskfSILSZq7ypNEU8YcudK+Rl1EjtHkI7CbwfICM2qNgaIwMfnU9ufj7YHA\\\\\\\/3CxsmAWQp1I4k9Ks4jScDfNou8d0jKwuto\\\\\\\/wCrHyI3QnFTJOoHsvUjQSKShYurWqHTIwbhgUw4gsC1A2Mwn\\\\\\\/ru\\\\\\\/IUTgIXWbdwpD\\\\\\\/Qc5mpXd9NKUrHEGhIHy5zvRZZQbxNIE1FsxeSdD0oxnPVRlEkM8FwB2PP9FKVEQocfRzbakVUzrePoAej0MLFzCoN4iaKWZkqgp83ftpYhCfNG\\\\\\\/ec\\\\\\\/BCwmojYcog7s5f63J\\\\\\\/\\\\\\\/NORK\\\\\\\/MbWHcfGaZ5s0vGlyvQeajC\\\\\\\/Uh31Y9kQdE1Vz\\\\\\\/zFZS6hPAvFhtp1cAA80G\\\\\\\/wU+riQh2v2D3toxVaj9d8AY03408rFnPjhdd1mI8\\\\\\\/XF4FbjyzQMK9olEyWHOhDgfmlNk4j2gz0f6+m7UyFAJ3Y\\\\\\\/kRjwiWehEpiXkYaoneRW+LfTqq\\\\\\\/zt2r++mSL0dPtTmc8zR+9At0B5F03lX1uHP9TCtbhU2VySTjoIbpDVmpFktvcS9fHQxRUOtWOCspZ404eRe2fkzV6Qi4LHUAf+V16EU6ij6ufhPKz8ULDi+2ArVwaCwGEPmQqVeoSs53jNpFFXAewTJLc+h0yMszirClFmtndSuO8VCbVmPQDqLvjSEyTHMsFxuYOJBYWl8fLLoirUDVuQil6LvxTYeWVxfnbIylq0WJakwUBpcZHcwRMB49RABzJobFIHv\\\\\\\/ksKn+F+CI82yn28dTF0q3mHiuH\\\\\\\/+I5CtXSc9l8ZuECCeTB1kL1UViRrqo7V1n5R8XqZOmIx1eVBHLqgDtQzaEpuqvN5ksPOC+Rdt2gMHmstmQNepJo1w1FSCj+BxjJJRHSlau5swv4M7nPFs0N0p6V9QPI8TaGk7TgPFirsdJu8POvcl4RcVQVv3fzMPsI6den0VMlSDltTEENzNbStx61sxzMsWCtwcFE2BzXB8U5IknkYO2rS+WTBfPYCS\\\\\\\/+Ucwb3eXZ5SlCktMhCX2yMyybhE+zukI1NZ92Mx2IyZmF97nGq59cb8pOLshWdfP6q0+Jetrp8qIrt4dhyxhqeXR+2gz9zQtqSWXkjF1BN3mkRBckyH\\\\\\\/iM7OGo6Ib9bS5PCkrTJiFont9nTNAP47fsMl4n3eENGriCuYBFooOlr+3lWunpjTcf1BJwsv\\\\\\\/enjGwI+BGg1+CHgaCE\\\\\\\/ScCqs2e+C81pdI05DPdIvGLJe29f\\\\\\\/fN1OfCGVjFwEFn456P9VHdJpXiCLYLvc7HlOHT1\\\\\\\/293ZyZ\\\\\\\/B4eElcVnDgikG3e+BO5IIj6CIF32fAZWhJxXzlQIpSEB4mWwP4ngVl4W2jCLrWKMHihWtZi15LJAs1DL5aoanKbxduvgxT6QbUkqvQDXHXBzKZ4ZlZXJMmRuQ89kuHZmG\\\\\\\/p\\\\\\\/KMkzlfRgb2GCijaKnfewePCvIpuF+kQGZu1HL6ulBpceY5nZlX83lszHpVeaJid4taOu8\\\\\\\/Q98TZLbRCJAgQVl0OdU47\\\\\\\/Q0Ai+NmWAZiBOAbZtNlG3rKzHtrLq6aU59VxRIeVD1noEkJNSF2bEVbs+jw6vCxyjY0ePTN6FggVgJWEweoXZl5Q5OrJOyHhlAqb8NbUHOs33r2AaYQXPVh1gMTxRphmwuLnTCsmrgzc2hMTsEb1rK49PmF0C8dnQH\\\\\\\/hfeparz\\\\\\\/FS69PId3XWKkL4ujcpUP3bUZV5p7o5\\\\\\\/QRXNGhhlHA8xNYXSwbDIGQbB0OPDhqcqzr\\\\\\\/JYwam3ETASOs4v8C8TNmKWlBrg7WlKAeTG\\\\\\\/3RM+MsDSHOUuQo2FgkseP5ubqzqbTZphEqQmkZ6KMk43tVUvg9KvRKxhYLt8Hs2sPKxprsDg+PDuD5i6DvxAc3Yrclr4ZsIFW9K\\\\\\\/22dqrcBa5wIXJDlzNAO8zmAgZAM9K37K+qjal5OFN5LofWTX0f3Cltz330LYAjgkF2zlqL9JbjDjfwOguZaPJAwm2TcD32rAsnmcahErjhbSmPY6VCs2fC8UrfPkJh5HDyOx+23g3oU2Uu2QehD\\\\\\\/IYKdfRbyDAvNDJgkFycJ58ilnhKcL1F1Ua\\\\\\\/uvpG234G2iYLuZOFTDAkZTZ+B531QSfDjWfU\\\\\\\/dVT\\\\\\\/MuPEu4ZmRqiD35uGzVpSqBnWiXEOVx9YiWL7Gjzn7ZisjSLzV0BvthGY0YeGnwNQtXU9T1gnhYWipEJArq7mIyD+RQxphe00IhheCwT85W2JcQbGT8GibEeosmMTxfAC5g+eD9f0dgfVLz4YJo4zX5vhdQAjIYpotYcy+BQtYe2gaHFzT0xtXAEPIhxZd2kT82ZNJNie1y92SS1\\\\\\\/Xn7rwSXK1hXtPLSpi44do8ZOcZWLj1yRDjhmukPPawV+44qZ\\\\\\\/C62+Mn6\\\\\\\/c8cUEjWbcOWFtvSpypjQpzQ9Xdoy6sLSsQaO8gEooqYWorvX1uhcpEylLcb1vZx9jddK38rc3q\\\\\\\/l7qLFzPNb0B7oTi+JQQmtQ6Q6LfdKeLJcbKw\\\\\\\/zxHasX4qgig2\\\\\\\/TD9alUiBsfBrChX3JzlofTW5l2KK63VIIrm8XJw4FU9mNEvKMfzOILn1uxREgvzr+oBKAG57QCOnd\\\\\\\/yz4eVIly8arQBjgHmr47kFaR1DhBSWd7sfQtr+iNhKeZIMn49oN1\\\\\\\/DtNXb7lwZXFOX8Tg8tPxfKhAFHG2Sovl2qG7diMVoOcPA1qsyQaWYnvQ7UCl5xEUN5qkPEM544v\\\\\\\/BxcAnDDLm5J2rna9DnfzWN0x5PEHD+E47vxYV9lu4jdoZsW6fz3rvWj01B0smxzCjsHJpLA\\\\\\\/DUmm+zxXOeWWM8XAU4smmaHqR\\\\\\\/QQK0494+AwVgLDtXVOzw3Dzpw\\\\\\\/4c87RKawKfg2GQ2muURkBmJusxrrfMn2tR4Awyxib4fTOfGyfKrN2rCVFZK1+VNweqHwUSPW\\\\\\\/\\\\\\\/aaLAFNAPOwuZPR6wmxEyXFTv31R0dz+UD75YGwRp8NHEIpron1lbf2xBmrFwvyKe\\\\\\\/wlJtnfHsoUqmZ63tQOisGVy\\\\\\\/scpeh1ALgnrt\\\\\\\/Bs8kH3FQrvmdMh8ePdq+qZ6kSzqCLFGa4RvbSxjghSeR6mSZ7t5W\\\\\\\/qlkW9LZY1RaUUobsEZJNkh+BBVzCOeuK3rSHoSaL8I9QXpbc49upN+im7pi0ssitn4fJ\\\\\\\/3khRqyNMFXqsV4g94pUWC3by080iFWfXe8dKnxVqWBUUQQd\\\\\\\/TloAyqJD7WL+n+O3xxBQQc1VwjDWNCENn6fGlduxUuNaB+6Ex0vRMDKizK0hlC5ESIj7a6aABlS44ERY7na6dypm8B8FIWgtg1oaOtuOOKTSTMP9z3Yud3T952vTgoeWdE3TSTU4OWt3uwQ6Fldi8aQqdvxbtjwIkDTvWO0zOqjKlZRc+uIvWPOcKF++DUlLpcovhHzPitzOKbpzqwkPpQLq06BvKYSZURGbhd58FMAc88qPREo0TDApzazutgA3zutSPFCDeNLK2rHMPhmL7AI3X7xNp6\\\\\\\/WGVvzf1q24DRPCOeqaVS6nL4O\\\\\\\/XxkJX0u9yzOlItcm4EbvJ4eoi1\\\\\\\/LjyrZQl5nlIxy\\\\\\\/AbykFdce8lKzquxz9Fllr1wT7tKx1d6oc2HqGL\\\\\\\/FWQvnh2Zdb0XCFicA5650btEZR\\\\\\\/ecnGoMdsAscwzvqCk2N3ELPBKBFfP\\\\\\\/5LRg7wtg\\\\\\\/RZFG5HnVgEXDk46OdYnHqGFaBoz73PqVIf3gYk6mEck2a\\\\\\\/md2AST5jq5BkPPfrDGa5KDKZdZYJMRHpsxRhXqqkNjD1c1dv53FT51fpMOL7jjB82AvIXAP9R4skL5h2Ta7FQE2Mo4h3z0oT\\\\\\\/pHZ3DRoKewUwzzUWDBA1jRwY6X0wvEBQZTyYVFGgEhLzGjOy1aqrApM921\\\\\\\/sLs9J4LNAMX\\\\\\\/QI7ZAEQrEehXz5FC1Dtcn5PEZF6sipPxIGhfkYfd+8aQKNq0iao1\\\\\\\/sQS7qfMh5keiQNn5U5hcIqeg\\\\\\\/E07wd0BFXOHknjExtqF4t6kpeYwbD0iyTyvehRvltLqRCqHZ8eAFqROWP1+9M\\\\\\\/L08OjttqdvTGUcBh8IHA3IyUKTgDB3lYiJewohLqsK1dRsrobV+VLfrnGZd0D06bmH7pIsUf0jF7JeudoTqTugew6TEsCfofz+kgHhXmwKtW3OeXykLInjetyIQSQOPfrb3Kg70pxzoiQvM6MRJu6ZoEPd1mJ+AVjqmijRkx0EXIOK\\\\\\\/TLrEnrv2JvD6bVWbO1uEWAtdfoyCXlDLXR94e1HwN9rzJvwuVfwf2GdSgbiY8ougqpMTkxcCp20tRyFngIADGFw+62jeWbeTJDZHM83XiXfzPXaRPxU6utf+PWxM1M7uyAa61xpFdgnMAQUoyq05kFI00woRDG+0ITaFIXYdfjTdGouwsNuonQoIe773zFu1uMEmHa6wyVe0UfR8J6oPdJE0SS44zImaA0JcLuh\\\\\\\/HwDaJYxU3JlXq0jcTIQGkyiniVjbgqJ8zLiNHVnuqUgAyk+sozMXKs8EbnXY4kNfrxp3hD+zI+reeTQRhcQD+naa68v3n5r+1DIZfB9scwEdFw0ogSw9KbiiUn9e3ioMK+dPsaO9XvRpkzyAbHcW0nS5P2q23TI+95DnDYkz67GddcpB\\\\\\\/ra5AlIbXW5M5Bjo4AGQuDfeOPI1tj\\\\\\\/OeuZYq0Eixdi1Oh9X7R1nK\\\\\\\/CWZsn8qAnlQ2Pi0BP711icSKSvfZ\\\\\\\/LITfT9j\\\\\\\/eM1Bx3aVgP7a5jnZSiEgoWuolveHCKlljizBTegpS0U+n3a5XjjfZMWuKuthTrTXzsoX4+hq6M7nlU5Ec8fyU9Vpk2Y7lo6+KTEC4ZN9Exrwb8sZyZ7UXJDkrJoT3oICmyWT4M+jPBfcOLW7kr8\\\\\\\/ODYPKb+pXh30IxMBvpdR\\\\\\\/Vi+dFP1bGrrEEWtq1jo8bqm2P2Np+s9pvqmkPCltypfuLdpEx683JxVqZ5A7GErrIN99JmEPEG5kYakNZWURZfRfpef3hJVWE6Uz5dt31dXWfIEcoX9\\\\\\\/u7yuY0xIbYo5TKXDYKCCQaO6OKzMwvowqDRIVe4cazGT5zbR6NMWqq\\\\\\\/Y5ylG4KfZBmkFcJMrsjRxU+ogwZLWldYGV1cGxuCLjupvvoZRFWCF4Gq4JMES2V5+rY3htYotu9Ld7LcIoQ3UsOfaNS8ER8RHod0W\\\\\\\/RrP57nvDkP3Xl2P9auOiNsezgEUpn0SltTJVOO6ubqTHsiwwu1w5KUhDmWkCfU5haM1lwkLsBejh+5p\\\\\\\/bK0J+7CdDExtER2F6mhUkijUC+SrWmyQEmQ9UeE9s\\\\\\\/iSnviouooecCWGEkiUOhfEA5mUM49qJGkWavpnUM5lwUCH085TqUUUzYjqcuXPaIGe4CTpB5dUlFjzheYrKUa\\\\\\\/5jI0okJCrz6P0PXAxMYVQNIDeA+8XAGrn3lO6qnySzb019MLrjoA4AH28TzxGnEfJZiRODXZDEKGb7f\\\\\\\/1tS\\\\\\\/MInkkhrtlVcxW7kcmY1sd4grzL58Q3n2VLsNz+WUHtZ1GdoPjAl9y4TCqfvUAHkf\\\\\\\/Dq\\\\\\\/TPIaXwCz+94+7TexxC9dBQERbXQNQ9ya\\\\\\\/0BGrr9KIYy6oJV28MDkC81ALQoLKJ8lrHBEKKLAU4WB4yUxlK4ZA4D0AHOzRT8tCVzFsIrWiZTAp81dBuX0mWhhOAWKW6i12RwbqGXei1StJThRTRDjEqkaoA1mRpmGfWV8GhHCz3xtNOVXFJeWIv5otsHFku7HIeVHzPYVEwNeORxXv3FCPnqUJzbiOiTcf\\\\\\\/FVsDpKlNeqHA\\\\\\\/6IM6pbD\\\\\\\/6Zs2Fbl40iEQiBjcYIaPgAmUPzVU3SF1tCRE5fevBzrNYRJe4amys81m1r6GIJ7EjeZHEmoVlhsrUqdid1N1Z+AxMGp6vK3sEXRA0RJ7cqvgwgiPrlfLsIg0FEVIdq9kTkVNmPw+1J1lW9Gdj+v8E+JIzbXF0V+lVwHIDmv\\\\\\\/JWTmJBcTAOFTxN6NTxhZrtrqV3NhOiyWO1DTxbzEFoU+GgSz9nVxgw6ALks9pwp4MwH56rIm9FivhMW6nYd178TIlePqzy\\\\\\\/xyn4QWVxrkw1\\\\\\\/wPw\\\\\\\/Hit6HzTISgMaLcEfH9GX5K1OMWZ+a9sEWRJpYWRb8HWIUs0Bit2rk7k9XAIz9nFol3G37PCxGBFkt1IwfZuDoIroMCW\\\\\\\/BJlH568Qsvu7fWh0nbLSL8tw0SCdYnoo03Qa25mybTU1bIjkkn+GM1dmxisNEBePt7apCv2YW7ht\\\\\\\/S0uo2aRGT1hCBU6iZGtyK4Eosvt1sVJkmGNURLt15VbL2NO7483ouoMdZdmSBDiqGVMA+5xeB9OIk7ubbiBcxg\\\\\\\/UPeabHJ\\\\\\\/bc3UA4Z60Et5H4Ckqc\\\\\\\/KmIsQJxZ0WdV45xX7WMG6PeXCLMASh4Zc7z9PNHuMfRKnLeYhWrT1Sir9XzLXwy8tgOVxtT8+E5YgiZjmUYj1GxR+pV4mp7mvs8gPcPyxgn6\\\\\\\/KYr5Pf0rLfOFJk6NTOblKFLkLWqP55W5Zqo2uPA8b99iibRsEHlExEyUixxIu82rfwrTEPeSEdGZ0qiiztrFBXa8rGZPjryBCLzCGHEnccbH\\\\\\\/oouTGT3kCkOmP27gh4K8w3ILDX2Mz\\\\\\\/LURle3oSlI5wFJOjmqQEitdwQAqeVjslVVT4e6JVhzYRla8JoiuyGlVYzHmPY2Fu3JtibDZUEShWXh9AiGWMPesau+nxwF5CQsS239o2q7djMGiYr0icTQfx6xE1hEZfSAS4KSW4wWEn5gCrV10EVXiz06+JC+2jE9gHvOuYOnuuFCc6\\\\\\\/nq4zBN9wx8JgtO\\\\\\\/6pX7O414kCzBIMR7h33TpExHM5D3GFQPaV\\\\\\\/u0jO8tIQtA\\\\\\\/HSCT14VO70kmduzWPE230FA1543HZaG8NMICiE9FYMm5ztdd0xLdvDujtlG\\\\\\\/XlNZsJGDtG+bInsgD\\\\\\\/gr6tLKIfUDlMliblnQ3SwgWmfvVOcR6OczEC06K7HZwzuY7f8uIlzP3SFsLShLwgYpjmamMVQmWnUPYHUxn\\\\\\\/6KhGaviUwZs570KVF3lijnpTADkRWIRBNfCxdmwNZoMXCUYa00kvYx4+HF5lc+g5o3eRT66C27rSmIDE9LjTnL+jhWiA\\\\\\\/blLAJRuC39eCbeyFfc4yzbzjHZ1eOSgO36jJZ+8X+oVYTMrIVc2+XJCrKLFXCTVEp+ewoaGR9SOiqomfIL5M3MUX2C6KjAA1DierYkF7MKT\\\\\\\/NWipAdTs+0LiX9FaCzgk4mQWVCgoouhKshqkl9cC2mv\\\\\\\/t8CpwAw9zofhOKlBiG+0s4\\\\\\\/sPY+MPFpZISIqpXkP2Du2jNXLyg8oYkeUhtDqTrTYXmCLV0fYYrv+D+guSjeAajoa\\\\\\\/JNZ+XIQuhBvIUPfBGuXavR8AYHTMHd2zfqzeS5yCdEs0Xw5tTz83dVcxAvNNk62GEHQrktPsMHW5Bycaek8QhhGWbL5ohBqZpxrrRMV+rdnzpu2FEW7DU21mde49ct8pbbxtAPXK8yDfzWzknGDQxNhhl4ZXsoV7eTx\\\\\\\/+zFuxvd4iuo5\\\\\\\/tY+JOmYdAYCxckPwZRPDWLRg7pMAMDY+nhRURdGj2LTYWr4bGUmY7gDZRwb2XV0goF5yPPgbtIxYR+QP2j6iNOuzqsSczUnMqaf+7oA+XJMYTzqUCFxxvFnmrBsyowUsFEuaIfuKRAV4l7TFhJxV2NpGnLmOwn47M+WzrQ1kpOhcOvZplkQt2O9JR6xdiJMv8xUGUhf6JJy9GTjvCtP4Wud2iuTTmH07eal9rcwY4aXUVqyVPVR6TzIfciXfwPCObudxjv4DWBj9g\\\\\\\/KXIE0lRWmLg7klYZbWKWNxJrKdk7oYIM5f6DTFtkGQAYt4epKLx4T7sm9YTWKClv83vTPmL7WPvJoefm8j69vwBA+rpBqF26CbCqLcZ\\\\\\\/eQGgbqFRuuaxueY5RDwwngyTGVK6WIaMDxdH4svTsAsfLzui50vSCXauYPKA9u1MgGF96CU91B6EBRMETU+GpIqkmwB+ITrzZNN6iRXLBltgiObqbdEuIw4UFmGauVEkB47XApQCuf10RlDbuY4ostgEnsPDoA9QcV3j1EAa8c98N9hUhXh5+l420fF0OiGGFELpvKgqcHpXCYhILHB4kUQ8rVMyDY1Y7Vll\\\\\\\/Ag1LuTQzXCyri5ENCxDOpZhoznB884EeuEnVFUJzC7c1YfdqdcZxYH4d+uyK5sno6TM4NcROzLKMoktIb7GfTDMMrCsAJ9nRoUi8CHKXc3q7xJ\\\\\\\/aNzNvxpCyPxcDCfda2pVk7fHjUXevdFagD0NvM0aZ73o4Q7A0Yyao1vLkltf9RKllGCgkvDfdM2LoOHMe+O2bZHaIJa4FTeLgWaGQdtehq+WtQcJ4+lAIheV4lNL8sUV4g9BYJqsqi9KcpU1v3iQlvFeB3yPF3QLfqJSI9T3ABKY0DVcnLlsT+N16w1Ersqq5kSW6zMRnvTyuPb3h0lVof3Fdtg9SYMJFu\\\\\\\/ZFbKCI3hfZ7k7jy3cjcYw9ZebZdaRnNvv\\\\\\\/IkvWM6jqJS4iyEIJbpzBnepoth705eMJZrXGrnage\\\\\\\/8w0R864WczfyjWGCz2snVS\\\\\\\/6aCasKYbSwlXd9DIF0d9uFbMmusHqPf8wZ7T4t4DuTychr5B4Au1FIBZpLM0+TTag3sfpbqQnau\\\\\\\/ZEzSYeidWyrsM4BPpguZl8X3ap+Xg83ilk4KBwedHim15wlyNUiLwk4kBfSgWIwb\\\\\\\/XJKEkctgU\\\\\\\/OKteOyo7BkEk2IAsyS6dYaN+bQ7e0gSzopsS3TO9gpc1WlZnX+mPrhtcaZ5CrNno\\\\\\\/3gybNMnMSpIB76KTNQSH5jKzHRqwOvcMnxE+qrqqhaqJ6UkqJtI+BMg9u8nj+b\\\\\\\/H7a+G35pIRFTexKHJuqN08QYYuiChZOgYeMDySwpAMVSMuan\\\\\\\/eVaqra5LEuDBMenvkgpCTa3d7itpdANFoz+cCpQca14+B02ozE8B\\\\\\\/RE4MsNGNwUov8cIDAW3TEVF1C9yLxn1Sh457ndsWuGSV1y7C0Ej\\\\\\\/MNjaf2IM0MgSKvrKBxPHPTMLwtsBsL9dSQ4WZOW7oklFhPQkPriI4lK96xwFJ8od8gIZ727IqWk5WSySekvllZ\\\\\\\/2EQgOEnrO4dWmrIrAOAxykvj79S8AkV6UXoc8L16M\\\\\\\/gIdEUZD9KRtrQM0t7YimWT3eqJlnjRjpQy9+0IyDOAiQHQb7yhizsz06QLvxOAKjN1zWHU2TfqGSF4FC4s3A7QKqEhTdfd0yCQvT4v1xWxsLUXVukMeo5NKcIUmikaE3DQuHnBScHYcBezk5wQAAzmQ8Zuhvp+HFTSOXnDZo9FHAUIHkayKsWGbUes4JDQgXTM2Q6EBCej9PuMeGWpzR4S8z3sNpQOTygyO4uQAJm5f+TEKG1o4c7wRb48fFa7iwMAxJq1r2LFCowUSM2MyJmhuV58QXF124D2L5teCLJRfULVvG+ihd8eRbR7DnDL0\\\\\\\/88j0V5qVOkWAYZRgVlrCE4AI+LeRM2mwdmZym2Du30\\\\\\\/poi+1Ofrw3fyw9eO+nVMWQ0eBLdqffSFpTgp4Mosyt0dBelAcU+7iUQuN6QNO3vt\\\\\\\/PLHe8eql3yBRKFfC9K+wdK7ncwSMI5SmHkO6+5969N9DEqTvwfnlIxCVHwSSqynbkGTJf1+\\\\\\\/H28pBjyzW\\\\\\\/TrtcyqBqP3dgaTmomqJI1sivmynNeYC2lkHeQ90Jg+mw8xtzmUoWp3fnnrSaLYfuwyakoK7IUkOWx0mYVBZeuLqW+PnEiZBS9an5jnnhOMMg6OQr6LhmDaiBHz6GuJlwh26blt1mFqDidQWw0zlKvXE8mfmjW4p+CL\\\\\\\/GrvcawpG77Fk93kX3e3dt6mIxtyclPHE14NudNqMld6d+DkpBks0etCV3bwN3okccviP53Z+0m885sKtOdXDNcTEXBAkVOf7WU3CHA7ebnozca5z50fatrhZzMsVTf0pwYv81Iwb22lxN9fUqVaZYe37o+1U\\\\\\\/sqWHemSSjDuDx\\\\\\\/L5a0fo7UIjb6B2WsnYojFJiC\\\\\\\/cR2o698QQUF1DRQmvHI5LM0k5nQNuA42bqEm84sgdj9Y5U6Acn+qJdg0fXc+6S0ehoEKpNvcUzF2ovPnAlJaVJPyGh1wjeJ+7\\\\\\\/ecNeIbNWLyb65WQxqvanM8v0EXvAbRTpyeMnysZIN4TggU\\\\\\\/zgs+KwTa0zqf0A8Tst\\\\\\\/pT+MqP0uJyPihRL7lvRe7rwuA3UdVU4QFFQ9rfpsz4K4C+ZhgS64Wbowibstmky8NZjEVMWuzW3S\\\\\\\/MXwzwg\\\\\\\/wp\\\\\\\/k41dmzBCWy02rNfQPhnpEPY8xdXRyGleQxuA+MnO+ooFUulAmpWt126IFbzcOWjbMdl9fYfewXxlz547Goc6R490olEnLLS92YAS3ewp08HcI0GBO6oZgTvxi0ZpqyuLM7mGl2M\\\\\\\/Li+kdqnM6\\\\\\\/9NzV9ROCX1r+q3hl5ZzhU3zSX5OVaQ2KaOPgmZcH7Ulm70LoAQAuKHIRKbyDEfcpdXVGX7IDUoZe5u5FePn\\\\\\\/6m+mOijvn2zKOjTtQLef+ZQOHz23Xb20brGdMGhkcZyUGq\\\\\\\/vjhBQW1Hon\\\\\\\/o7zxUxiKMVdVzrb\\\\\\\/7eYDny\\\\\\\/d56k8R00M+Z1rlKYJU3MCR3Y6UcP2dc2+6TD\\\\\\\/teXn5ZOoOMw87K3NUPAW3xG\\\\\\\/T987i2iB\\\\\\\/BuVTQsxJGJHD1iUPKFM7u7f5F1iDwsn5IzREOa4U\\\\\\\/RNUjk\\\\\\\/DBqsjydQW3C454Nzo3EWdAPa9Hh4CWp5Oxo7LY0ZSbn871T8gZPTJ8IVBsRAYmmu92o\\\\\\\/Oe+80n0VgCWWjyEaaPMAug4zyq6\\\\\\\/uSrg9sJDmTnFkM9Qq0GBLATcXHNinjUGcqGGWIQK28KQd+1e3mu2SFqoAg\\\\\\\/PZCyT+q28TMW+u5vp3S8uqdzGkq2VMx9zqhkYB9aP7y1fBY+nm0RQ1gJ+rBuBo7JKIhoLDyCxbP3tPHLgvFh\\\\\\\/dxujglHSmIsnzXD\\\\\\\/z8zvD0NqD3CJokjGSJHUSKOlZ4Y+DKG5iEY1+Ha8L2CxjGY9VQZTGNL9zMB3Hb\\\\\\\/jkS6zpPSLSDnDrMF8X+DvMqBjzY7ft\\\\\\\/SaHX4gnSRu4jRck2rA+5uhBro\\\\\\\/QHpG1adR3eqF+l6ws\\\\\\\/vBej0pyBFkrl2JuSx\\\\\\\/\\\\\\\/LqcgT+D7ATqeRkOmLF\\\\\\\/My4uDFcidnlpkXRiUY1N74N1Gs+60cQrhigH5ja4vklqAnGO+YXPbyFQNVM\\\\\\\/PLgYKMLw8EcdfwKQJ4fbEWJgbtHfdEhhJPZL3EQfgJBzTBVCwmHVQjJY6jW1aPfdursbs40wHzGn3F7YcomiNiIckGYB38kXw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ydDk3hfbOOlnsSAqUGxSBw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225134c55fedf0d14dde90137ece44b8c9c434a7b143f9b2656b4d4801e9a3c2c5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BYjb+MamqKbb+f2BwEE4srV\\\\\\\/egzrKdHdfI85pqQOSfDA79\\\\\\\/tA+dN87uHhdExUOYb\\\\\\\/q0v75oBaJFsJJxPf4eQnom6MICYXKN32QZyGt5InPu\\\\\\\/kRfvjRbnJfLjqRuC9X75n9VDFb5c9OVrvGg8QL0eHcJNuu7mEBmn6\\\\\\\/s2fxpb0Eu2Hnnpo5XkvJAhtMt1zK9y2qUnMXZC0hcZTTCirmvxA+aWvm8SNwkw2I\\\\\\\/PVjiep1hR2XFraD4iBe02WzpnF2idaq729pFDrzDzq6q1vylBTJzMtTdlWdZvja6ZxknVgrTVuzSUVh8ICtYd3GI+GSiwBSDgpiaKQdS+uOU\\\\\\\/jSf0pTxC6x5xiHUyOlVrfChY4kY8MWs3Da\\\\\\\/bptwy6quKyfuL+Qcr1Z5WGbnARFPMJS6fvni63GjmJBXpA\\\\\\\/FF6sc4rpkEa5vKoifiGpbAXsggIY3Hd+a8g4Qep5u4M9ahZybMrncUGHDIj0igZ9vTJ0cqdM4LjYW7TNaNA52Pa07\\\\\\\/pnP815NhJK+WsIxtJUQoJGKbZP9mYrzymcIA8xFvfEFUe2vAavNjoxXJrJ2a2z+1g1SS5MGeel+XBAkYQGUZaIgWCU2iqRYPbQT4AyTjB9NpdNnbLyTT\\\\\\\/TIuKECp6IIa5NjshC\\\\\\\/FIKujv+4PDlxJXUAZD\\\\\\\/PJbBgamVkNbzPRLU+TRJB8WTkpqDcu2QynQnDImZDr1PgTxNGrolmm\\\\\\\/klFKXxkcWuE0xSshGbDCavF+1bIFRW4zfbpJvsRFBQ4FVq3tlK5MUUfMlFLM9FLJfT5sSbd56Cg8MEeWINHHl5bKrZpJ5f1VfC6sUxVvb6xfiHkx3OWy7deY3G9EDkutsM3geoQ8H04wyHjDHzyx\\\\\\\/HnfZUa+wkJR4YzGRh+NLGaxV1bQja0S2kQ2X7R5UBrw+\\\\\\\/TPczBSC6\\\\\\\/JZN39eSbx4O+mxeXVqqqU+5V7zIzNDu4lVBrVDgLUtCC0f2r2aWdNzuzP7e8mYGQpMO+Zj3u075Sz0Tr9NVm46hl\\\\\\\/MIwOUdZcT+Qtw7j\\\\\\\/vrX2OZa9EpSPJMD6F+ZsI38YplB5evIqC\\\\\\\/7VGDR9uCRmndnECX3Q8WpWlg7CKVvr61QQ677cdaLUe48aVJBul34dCEzhGxMFBUFt5QdWFCbyMXbUc2b3SDgMJZbvEAXQRrZ75P2JTnE2eIAJg1Iia0lU1r2Lb0n6HwapC1LJ5FRIx3FPngAkt2A1h7DQjsbw0lskSZ6sdpeBEyjTR\\\\\\\/gnESZJTtFG8Z3Pd7q\\\\\\\/Cb+6gts7sZp8kosAMM5tMb3GZb3U5gKgz2bvDB33LDq1rgIjmavGXlBE0Zto+eF1urMieeI\\\\\\\/\\\\\\\/fHu34Xazgxf5+e66kI67U15elGGl1r4IlFTph10abp+\\\\\\\/x6TQlkftJi\\\\\\\/g6RPIezLEbvB7gG0PObqOCS7fUG7nU1JG2eLzEhW136B5Cdv\\\\\\\/GbqSXrAF9mT16hLroqyYbX16p73wit6cTHaNydSfB4da9drIS1pgo70QgPrG0JPW44WbpzZVauo2PUSL1uo6yYErfpudR\\\\\\\/\\\\\\\/ET6c6AMTj5vqfxRnpnZlVz420r9cwgftvBuxlAE4iMAJ6dlRFBo6kBASlkLlo1uSKAJMLCK6YHkDyb06Tm4IlZ4OqkTGDQfqPePWpsYO+RIE30uqna9SJl\\\\\\\/jS4INdkJ8KQouddLR0mPGbfGmFn+21zHPhageG0IKq1iXzKX1k31IAaPGTGWQ1cm\\\\\\\/5XmjVJoOieexwN\\\\\\\/4Jwu+lorVQmsaOhz6Yv2A9liLG\\\\\\\/TaJPjVp4kAtWG5IN0kmIx66TYkqJwmugY6mD2z6gDcSXNZTMcBqiRgJCO06oAKX07bCZYYjb+FLwhF49rq0FGqVc\\\\\\\/ryfvmC58cxacMBNWnEcTXv\\\\\\\/XSYekPGJO7MNNBz1XTJm7jx0MalTJCxlm8tLUSgJUuc3zsyfZVDi9iI0BGuI6xzh8+7jtsbUx+drwkqx4xVL4i+bcP8c5E2FmsTPqzwcJd2YYcJieGDLLWmA21lYBGYoOOfUSicbflYQMOA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iPrzT7QRIv33EE6JfmregQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002205687d0d6572774843017407b2b71ca3b58eba886f5176826c7d641317bca5a9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490463 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2709" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7_oO-LwMabzt-vFnpmnFrFXqHZyuDanhM4pjeuJoDuU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2709" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NpmkDaDIUITVYVg6C-eQqvX3MYoDxnO2umowlp200BQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2697" title="Targeted Chromosomal Rearrangements via Combinatorial Use of CRISPR/Cas9 and Cre/LoxP Technologies in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2723" title="Fission Yeast CENP-C (Cnp3) Plays a Role in Restricting the Site of CENP-A Accumulation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490463" id="node490463" data-pisa="ggg;8/8/2709" data-pisa-master="ggg;g3.118.200171" data-apath="/ggg/8/8/2709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6841-1269" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emily Humble</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2840-7019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kanchon K. Dasmahapatra</span>, <span class="highwire-citation-author" data-delta="2">Alvaro Martinez-Barrio</span>, <span class="highwire-citation-author" data-delta="3">Inês Gregório</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2115-0150" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaume Forcada</span>, <span class="highwire-citation-author" data-delta="5">Ann-Christin Polikeit</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4988-9085" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simon D. Goldsworthy</span>, <span class="highwire-citation-author" data-delta="7">Michael E. Goebel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-9052-1998" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jörn Kalinowski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-2958-5183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jochen B. W. Wolf</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-5895-8949" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph I. Hoffman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2709-2722;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200171 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490463" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emily Humble </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BHumble%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Humble%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmily%2BHumble%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6841-1269" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emily Humble</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emily.lhumble@gmail.com" class="" data-icon-position="" data-hide-link-title="0">emily.lhumble@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kanchon K. Dasmahapatra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of York, Heslington, York, YO10 5DD, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanchon%2BK.%2BDasmahapatra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dasmahapatra%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKanchon%2BK.%2BDasmahapatra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2840-7019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kanchon K. Dasmahapatra</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alvaro Martinez-Barrio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Science for Life Laboratories and Department of Cell and Molecular Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alvaro%2BMartinez-Barrio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez-Barrio%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlvaro%2BMartinez-Barrio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Inês Gregório </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=In%C3%AAs%2BGreg%C3%B3rio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greg%C3%B3rio%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIn%25C3%25AAs%2BGreg%25C3%25B3rio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaume Forcada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaume%2BForcada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forcada%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaume%2BForcada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2115-0150" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaume Forcada</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ann-Christin Polikeit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann-Christin%2BPolikeit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polikeit%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn-Christin%2BPolikeit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Simon D. Goldsworthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>South Australian Research and Development Institute, 2 Hamra Avenue, West Beach, South Australia, 5024, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BD.%2BGoldsworthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goldsworthy%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BD.%2BGoldsworthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4988-9085" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simon D. Goldsworthy</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michael E. Goebel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Antarctic Ecosystem Research Division, SWFSC, NMFS, NOAA, La Jolla, California 92037-1023 USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BGoebel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goebel%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BE.%2BGoebel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jörn Kalinowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centrum für Biotechnologie, CeBiTec, University of Bielefeld, Universitätsstr. 27, 33615 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BKalinowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalinowski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rn%2BKalinowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9052-1998" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jörn Kalinowski</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jochen B. W. Wolf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Division of Evolutionary Biology, Faculty of Biology, Ludwig-Maximilians-Universität Munich, Planegg-Martinstried, Germany</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Science for Life Laboratories and Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BB.%2BW.%2BWolf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolf%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BB.%2BW.%2BWolf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2958-5183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jochen B. W. Wolf</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joseph I. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BI.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20JI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BI.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5895-8949" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph I. Hoffman</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/8/2709" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490463" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490463/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2709.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490463" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490463/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2709.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490463" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490463/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/8/2709.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490463" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490463/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/8/2709.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490463" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490463/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Recent advances in high throughput sequencing have transformed the study of wild organisms by facilitating the generation of high quality genome assemblies and dense genetic marker datasets. These resources have the potential to significantly advance our understanding of diverse phenomena at the level of species, populations and individuals, ranging from patterns of synteny through rates of linkage disequilibrium (LD) decay and population structure to individual inbreeding. Consequently, we used PacBio sequencing to refine an existing Antarctic fur seal (<em>Arctocephalus gazella</em>) genome assembly and genotyped 83 individuals from six populations using restriction site associated DNA (RAD) sequencing. The resulting hybrid genome comprised 6,169 scaffolds with an N50 of 6.21 Mb and provided clear evidence for the conservation of large chromosomal segments between the fur seal and dog (<em>Canis lupus familiaris</em>). Focusing on the most extensively sampled population of South Georgia, we found that LD decayed rapidly, reaching the background level by around 400 kb, consistent with other vertebrates but at odds with the notion that fur seals experienced a strong historical bottleneck. We also found evidence for population structuring, with four main Antarctic island groups being resolved. Finally, appreciable variance in individual inbreeding could be detected, reflecting the strong polygyny and site fidelity of the species. Overall, our study contributes important resources for future genomic studies of fur seals and other pinnipeds while also providing a clear example of how high throughput sequencing can generate diverse biological insights at multiple levels of organization.</p></div><span class="kwd-group-title">Key words:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/pacbio-sequencing" class="hw-term hw-article-keyword hw-article-keyword-pacbio-sequencing" rel="nofollow">PacBio sequencing</a></li><li class="kwd"><a href="/keyword/rad-sequencing" class="hw-term hw-article-keyword hw-article-keyword-rad-sequencing" rel="nofollow">RAD sequencing</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism-snp" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism-snp" rel="nofollow">single nucleotide polymorphism (SNP)</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium-ld" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium-ld" rel="nofollow">linkage disequilibrium (LD)</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/genomic-inbreeding-coefficient" class="hw-term hw-article-keyword hw-article-keyword-genomic-inbreeding-coefficient" rel="nofollow">genomic inbreeding coefficient</a></li></ul><p id="p-3">Advances in short read sequencing technologies, in particular Illumina sequencing, have made it possible to generate genome assemblies as well as dense genetic marker datasets for practically any organism (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ekblom and Galindo 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ellegren 2014</a>). However, assemblies based solely on short read data tend to be highly fragmented, even with assembly strategies that incorporate medium length insert libraries (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gnerre <em>et al.</em> 2011</a>). Consequently, although such assemblies can be generated rapidly and cheaply, there has been growing interest in technologies that incorporate longer range information to improve scaffold length and contiguity. For example, Pacific Biosciences (PacBio) single molecule real-time (SMRT) sequencing generates read lengths in the order of several kilobases (kb) that have proven effective in gap filling, resolving complex repeats and increasing contig lengths across diverse taxa (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">English <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Conte and Kocher 2015</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Pootakham <em>et al.</em> 2017</a>).</p><p id="p-4">In parallel to these and related developments in genome sequencing technologies, reduced representation sequencing approaches such as restriction site associated DNA (RAD) sequencing (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Peterson <em>et al.</em> 2012</a>) are providing unprecedented levels of genetic resolution for population genetic and genomic studies (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Morin <em>et al.</em> 2004</a>; <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Stapley <em>et al.</em> 2010</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Seeb <em>et al.</em> 2011</a>). By sequencing and assembling short stretches of DNA adjacent to restriction cut sites and interrogating the resulting tags for sequence polymorphisms, RAD sequencing can facilitate the acquisition of large genome-wide distributed single nucleotide polymorphism (SNP) datasets incorporating multiple individuals.</p><p id="p-5">The above approaches show great promise for studying wild populations where genomic resources are typically absent. For example, information from model organisms with well-characterized genomes can facilitate studies of their wild relatives as long as patterns of synteny between the two can be established. Knowledge of synteny can facilitate gene annotation, assist in gene mapping and help to elucidate the genetic basis of fitness variation by identifying genes closely linked to loci responsible for inbreeding depression (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Johnston <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ekblom and Wolf 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Kardos <em>et al.</em> 2016b</a>).</p><p id="p-6">High density SNP markers mapped to a reference genome can furthermore provide insights into processes that shape levels of variation within genomes. For example, the positional information of genomic loci can be used to characterize patterns of linkage disequilibrium (LD). LD is a central concept in population genetics because it is closely associated with factors such as effective population size (<em>N<sub>e</sub></em>), genetic drift, historical fluctuations in population size, population structure, inbreeding and recombination (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Slatkin 2008</a>). Understanding the strength and extent of LD can aid in the inference of demographic history and has important implications for identifying genetic variants underlying key fitness traits through genome-wide association analyses or quantitative trait locus mapping (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Carlson <em>et al.</em> 2004</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Miller <em>et al.</em> 2015</a>; Kardos <em>et al.</em> 2016). Nevertheless, the genomic pattern of LD has only been described in a handful of wild organisms. Typically, LD decays within a few tens to hundreds of kilobases (kb) in large and unstructured populations (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Poelstra <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Kawakami <em>et al.</em> 2014</a>; <a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Vijay <em>et al.</em> 2016</a>), but can extend for several megabases (Mb) in smaller, isolated, heavily bottlenecked and/or inbred populations of species such as wolves and sheep (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hagenblad <em>et al.</em> 2009</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Miller <em>et al.</em> 2011</a>).</p><p id="p-7">In addition to facilitating the characterization of genome-wide patterns of variation, dense genomic markers can also be used to describe variation at the population and individual level, even without positional information. For example, studies are increasingly employing approaches such as RAD sequencing to obtain large datasets in order to reliably characterize population genetic structure (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Malenfant <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benestan <em>et al.</em> 2015</a>; <a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Younger <em>et al.</em> 2017</a>) and many are uncovering patterns that had previously gone undetected (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Reitzel <em>et al.</em> 2007</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Ogden <em>et al.</em> 2013</a>; <a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Vendrami <em>et al.</em> 2017</a>). A precise understanding of population structure is critical for the delineation of management units for conservation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bowen <em>et al.</em> 2005</a>) as well as for avoiding false positives in genome-wide association studies (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Johnston <em>et al.</em> 2011</a>). Knowledge of population structure can also help to elucidate contemporary and historical barriers to gene flow (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">McRae <em>et al.</em> 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hendricks <em>et al.</em> 2017</a>) as well as patterns of extinction and recolonization (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">McCauley 1991</a>).</p><p id="p-8">A major topic of interest at the level of the individual is the extent to which inbreeding occurs in natural populations (Kardos <em>et al.</em> 2016) and its consequences for fitness variation and population demography (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Keller and Waller 2002</a>). Pedigree-based studies, typically of isolated island populations and often involving polygynous species, have uncovered widespread evidence of inbreeding in the wild (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Marshall <em>et al.</em> 2002</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Townsend and Jamieson 2013</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Nietlisbach <em>et al.</em> 2017</a>). However, the extent of inbreeding in large, continuous and free-ranging populations remains open to question. On the one hand, simulations have suggested that inbreeding will be absent from the vast majority of wild populations with the possible exception of highly polygynous and/or structured populations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balloux <em>et al.</em> 2004</a>). On the other hand, associations between microsatellite heterozygosity and fitness (heterozygosity fitness correlations, HFCs) have been described in hundreds of species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chapman <em>et al.</em> 2009</a>) and it has been argued that these are highly unlikely to arise in the absence of inbreeding (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Szulkin <em>et al.</em> 2010</a>). Due to the high sampling variance of microsatellites, there has been growing interest in the use of high density SNP data to reliably quantify inbreeding, and recent empirical and simulation studies suggest that this can be achieved with as few as 10,000 SNPs (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Kardos <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">2018</a>). Consequently, with approaches like RAD sequencing, it should be possible to quantify the variation in inbreeding in arguably more representative wild populations.</p><p id="p-9">The Antarctic fur seal (<em>Arctocephalus gazella</em>) is an important marine top predator that has been extensively studied for several decades, yet many fundamental aspects of its biology remain poorly understood. This highly sexually dimorphic pinniped has a circumpolar distribution and breeds on islands across the sub-Antarctic, with 97% of the population concentrated on South Georgia in the South Atlantic (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The species was heavily exploited by 18<sup>th</sup> and 19<sup>th</sup> Century sealers and was thought to have gone extinct at virtually all of its contemporary breeding sites (<a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Weddell 1825</a>). However, in the 1930s a small breeding population was found at South Georgia (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonner 1964</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Payne 1977</a>), which in the following decades increased to number several million individuals (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Boyd 1993</a>). While it is generally believed that the species former range was recolonized by emigrants from this large and rapidly expanding population (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Boyd 1993</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Hucke-Gaete <em>et al.</em> 2004</a>), one would expect to find little or no population structure under such a scenario. However, a global study using mitochondrial DNA resolved two main island groups (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Wynen <em>et al.</em> 2000</a>) while microsatellites uncovered significant differences between South Georgia and the nearby South Shetland Islands (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bonin <em>et al.</em> 2013</a>), implying that at least two relict populations must have survived sealing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Individual assignment to genetic clusters based on STRUCTURE analysis for K = 4 using 27,592 SNPs. Each horizontal bar represents a different individual and the relative proportions of the different colors indicate the probabilities of belonging to each cluster. Individuals are separated by sampling locations as indicated on the map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025565988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Individual assignment to genetic clusters based on STRUCTURE analysis for &lt;em&gt;K&lt;/em&gt; = 4 using 27,592 SNPs. Each horizontal bar represents a different individual and the relative proportions of the different colors indicate the probabilities of belonging to each cluster. Individuals are separated by sampling locations as indicated on the map.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2709/F1.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/8/2709/F1.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490543" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Individual assignment to genetic clusters based on STRUCTURE analysis for <em>K</em> = 4 using 27,592 SNPs. Each horizontal bar represents a different individual and the relative proportions of the different colors indicate the probabilities of belonging to each cluster. Individuals are separated by sampling locations as indicated on the map.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Antarctic fur seals have been intensively studied for several decades at a small breeding colony on Bird Island, South Georgia, where a scaffold walkway provides access to the animals for the collection of detailed life history and genetic data. Genetic studies have confirmed behavioral observations of strong polygyny (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bonner 1964</a>) by showing that a handful of top males father the majority of offspring (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Hoffman <em>et al.</em> 2003</a>). Furthermore, females exhibit strong natal site fidelity, returning to within a body length of where they were born to breed (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hoffman and Forcada 2012</a>), while adults of both sexes are highly faithful to previously held breeding locations (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Hoffman <em>et al.</em> 2006</a>). Together these behavioral traits may increase the risk of incestuous matings. In line with this, heterozygosity measured at nine microsatellites has been found to correlate with multiple fitness traits including early survival, body size and reproductive success (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Hoffman <em>et al.</em> 2004</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Forcada and Hoffman 2014</a>). However, such a small panel of microsatellites cannot provide a very precise estimate of inbreeding (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Slate <em>et al.</em> 2004</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Balloux <em>et al.</em> 2004</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Miller <em>et al.</em> 2014</a>) and therefore high density SNP data are required to generate more detailed insights into the variance in inbreeding in the population.</p><p id="p-12">Here, we used PacBio sequencing to improve an existing Antarctic fur seal genome assembly comprising 8,126 scaffolds with an N50 of 3.1 Mb (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Humble <em>et al.</em> 2016</a>). We additionally RAD sequenced 83 individuals, mainly from South Georgia but also from an additional five populations, to generate a large dataset of mapped genetic markers. The resulting data were then used to investigate synteny with the dog (<em>Canis lupus familaris</em>) and, within the focal South Georgia population, to characterize the pattern of LD decay as well as the variance in inbreeding. Finally, using data from both RAD sequencing and 27 microsatellites, we investigated the strength and pattern of population structure across the species range and compared the ability of the two marker types to resolve genetic differences between island groups. Our hypotheses were as follows: (i) We expected to find strong synteny between the fur seal and dog, the closest relative with an annotated, chromosome-level genome assembly; (ii) Given that Antarctic fur seals are free-ranging with large population sizes, we predicted that LD would decay very rapidly, although it is also possible that the historical bottleneck could have resulted in elevated levels of LD; (iii) We hypothesized that nuclear markers would detect the same two island groups as previously found with mitochondrial DNA as well as possibly resolve finer scale structuring. We furthermore predicted that RAD sequencing would provide greater power to capture genetic differences than microsatellites; (iv) Finally, we anticipated finding variation in inbreeding consistent with knowledge of the species mating system as well as previous studies documenting HFCs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Hybrid genome assembly and PacBio DNA library preparation</h3><p id="p-13">We first used the program GapCloser v1.12 to fill gaps in the existing fur seal genome v1.02 (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Humble <em>et al.</em> 2016</a>) (NCBI SRA: BioProject PRJNA298406) based on the paired end information of the original Illumina reads. This approach closed 45,852 gaps and reduced the amount of N space in the assembly from 115,235,953 bp to 78,393,057 bp (v1.1, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Following this, we generated SMRT sequencing data from the DNA used for the original genome assembly (NCBI SRA: BioSample SAMN04159679) following the protocol described in <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Pendleton <em>et al.</em> (2015)</a>. First, 10 μg of pure genomic DNA was fragmented to 20 kb using the Hydroshear DNA shearing device (Digilab, Marlborough, MA) and size-selected to 9–50 kb using a Blue Pippin according to the standard Pacific Biosciences SMRT bell construction protocol. The library was then sequenced on 64 PacBio RSII SMRT cells using the P6–C4 chemistry. This yielded a total of 58 Gb (∼19x) of sequencing data contained within 8,101,335 subread bases with a mean read length of 7,177 bp (median = 6,705 bp; range = 50–54,622 bp).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490604/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490604/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome assembly statistics for successive improvements to the original Antarctic fur seal genome assembly</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">Next, we used PBJelly v15.8.24 and blasr (<a href="https://github.com/PacificBiosciences/blasr">https://github.com/PacificBiosciences/blasr</a>) with default parameters to align the PacBio sequencing reads to the gap-closed assembly to generate a hybrid genome assembly (v1.2). Lastly, we followed a two-step strategy to remove any indels introduced by single molecule real-time sequencing (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Ross <em>et al.</em> 2013</a>). We first used Quiver (contained in the SMRT/2.3.0 suite: GenomicConsensus v0.9.2) with the refineDinucleotideRepeats option to perform initial assembly error correction. Due to this step being computationally demanding, we ran it separately for each scaffold. Next, we mapped the original Illumina reads (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Humble <em>et al.</em> 2016</a>) to the quiver assembly (v1.3) using BWA MEM v0.7.15 (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Li 2013</a>) and used Picard tools to sort and mark duplicates. We then used PILON v1.22 (<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Walker <em>et al.</em> 2014</a>) to perform the final error correction step to generate assembly v1.4.</p></div><div id="sec-3" class="subsection"><h3>Genome alignment</h3><p id="p-17">We aligned the fur seal scaffolds from assembly v1.4 to the dog genome (<em>Canis lupus familiaris</em> assembly version CanFam3.1, GenBank accession number GCA_000002285.2) using LAST v746 (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Kiełbasa <em>et al.</em> 2011</a>). First, the dog genome was prepared for alignment using the command lastdb. We then used lastal and last-split in combination with parallel-fastq to align the fur seal scaffolds against the dog genome. Using the program MafFilter v1.0.0, we then processed the resulting multiple alignment format (maf) file and estimated pairwise sequence divergence between the two species (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dutheil <em>et al.</em> 2014</a>). Finally, we extracted alignment coordinates from the maf file using bash commands to allow subsequent visualization with the R package RCircos v1.2.0 (<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Zhang <em>et al.</em> 2013</a>).</p></div><div id="sec-4" class="subsection"><h3>Sampling and DNA extraction</h3><p id="p-18">Tissue samples were collected from 57 Antarctic fur seal individuals from Bird Island, South Georgia. These comprised 24 triads consisting of 24 pups, 16 mothers and 17 fathers of which five mothers and two fathers sired multiple offspring (Figure S1). Additional samples were obtained from the main breeding colonies across the species range (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>): Cape Shirreff in the South Shetlands (<em>n</em> = 6), Bouvetøya (<em>n</em> = 5), Kerguelen (<em>n</em> = 5), Heard Island (<em>n</em> = 5) and Macquarie Island (<em>n</em> = 5). Skin samples were collected from the inter-digital margin of the fore-flipper using piglet ear notching pliers and stored in 20% dimethyl sulphoxide saturated with NaCl at –20°. Skin samples from the South Shetlands were collected using a sterile 2mm biopsy punch and stored in 95% ethanol. Total genomic DNA was extracted using a standard phenol-chloroform protocol (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Sambrook <em>et al.</em> 1989</a>).</p></div><div id="sec-5" class="subsection"><h3>Microsatellite genotyping</h3><p id="p-19">All samples were genotyped at 27 polymorphic microsatellite loci (see Table S1), previously found to be in Hardy-Weinberg equilibrium (HWE) in the study population at South Georgia and unlinked (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Stoffel <em>et al.</em> 2015</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Peters <em>et al.</em> 2016</a>). These loci were PCR amplified in three separate multiplex reactions (see Table S1) using a Type It Kit (Qiagen). The following PCR profile was used for all multiplex reactions except for multiplex one: initial denaturation of 5 min at 94°; 28 cycles of 30 sec at 94°, 90 sec at 60°, and 30 sec at 72°, followed by a final extension of 30 min at 60°. The PCR profile of multiplex one only differed from this protocol in the annealing temperature used, which was 53°. Fluorescently labeled PCR products were then resolved by electrophoresis on an ABI 3730xl capillary sequencer and allele sizes were scored using GeneMarker v1.95. To ensure high genotype quality, all traces were manually inspected and any incorrect calls were adjusted accordingly.</p></div><div id="sec-6" class="subsection"><h3>RAD library preparation and sequencing</h3><p id="p-20">RAD libraries were prepared using a modified protocol from <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Etter <em>et al.</em> (2011)</a> with minor modifications as described in <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Hoffman <em>et al.</em> (2014)</a>. Briefly, 400 ng of genomic DNA from each individual was separately digested with <em>SbfI</em> followed by the ligation of P1 adaptors with a unique 6 bp barcode for each individual in a RAD library, allowing the pooling of 16 individuals per library. Libraries were sheared with a Covaris S220 and agarose gel size-selected to 300–700 bp. Following 15–17 cycles of PCR amplification, libraries were further pooled using eight different i5 indices prior to 250 bp paired-end sequencing on two Illumina HiSeq 1500 lanes.</p></div><div id="sec-7" class="subsection"><h3>SNP genotyping</h3><p id="p-21">Read quality was assessed using FastQC v0.112 and sequences were trimmed to 225 bp and demultiplexed using process_radtags in STACKS v1.41 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Catchen <em>et al.</em> 2013</a>). We then followed GATK’s best practices workflow for variant discovery (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Poplin <em>et al.</em> 2017</a>). Briefly, individual reads were mapped to the Antarctic fur seal reference genome v1.4 using BWA MEM v0.7.10 (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Li 2013</a>) with the default parameters. Any unmapped reads were removed from the SAM alignment files usingSAMtools v1.1 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Li 2011</a>). We then used Picard Tools to sort each SAM file, add read groups and remove PCR duplicates. Prior to SNP calling, we performed indel realignment to minimize the number of mismatching bases using the RealignerTargetCreator and IndelRealigner functions in GATK v3.6. Finally, HaplotypeCaller was used to call variants separately for each individual. Genomic VCF files were then used as input to GenotypeGVCFs for joint genotyping. The resulting SNP dataset was then filtered to include only biallelic SNPs using BCFtools v1.2 (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Li 2011</a>) to obtain a dataset of 677,607 SNPs genotyped in 83 individuals. Subsequently, we applied a variety of filtering steps according to the analysis being performed as shown in Figure S2 and described below.</p></div><div id="sec-8" class="subsection"><h3>SNP validation</h3><p id="p-22">To provide an indication of the quality of our SNP dataset, we attempted to validate a representative subset of loci using Sanger sequencing. First, we randomly selected 50 loci whose 70 bp flanking sequence contained no secondary SNPs and mapped uniquely to the fur seal reference genome and with initial depth of coverage and minor allele frequency (MAF) filters of 5 and 0.05 respectively. We then designed oligonucleotide primers using Primer 3 (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Untergasser <em>et al.</em> 2012</a>) to PCR amplify each putative SNP together with 100–200 bp of flanking sequence. Each locus was PCR amplified in one fur seal individual that had been genotyped as homozygous at that locus and one that had been genotyped as heterozygous. PCRs were carried out using 1.5 µL of template DNA, 20 mM Tris–HCl (pH 8.3), 100 mM KCl, 2 mM MgCl2, 10x Reaction Buffer Y (Peqlab), 0.25 mM dNTPs, 0.25 mol/L of each primer and 0.5U of Taq DNA polymerase (VWR). The following PCR profile was used: one cycle of 1.5 min at between 59° and 62° depending on the primers used (Table S2), 60 sec at 72°; and one final cycle of 7 min at 72°. 5 µL of the resulting PCR product was then purified using shrimp alkaline phosphatase and exonuclease I (NEB) following the manufacturer’s recommended protocol. All fragments were then sequenced in both directions using the Applied Biosystems BigDye Terminator v3.1 Cycle Sequencing Kit (Thermo Fisher Scientific) and analyzed on an ABI 3730xl capillary sequencer. Forward and reverse reads were aligned using Geneious v10.2.3 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Kearse <em>et al.</em> 2012</a>). Heterozygous sites were identified as those with two peaks of roughly equal intensity but with around half the intensity of a homozygote.</p></div><div id="sec-9" class="subsection"><h3>Linkage disequilibrium decay</h3><p id="p-23">Prior to quantifying linkage disequilibrium, we filtered the SNP dataset as shown in Figure S2A. First, to minimize the occurrence of unreliable genotypes, we removed individual genotypes with a depth of coverage below eight or above 30 using VCFtools (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Danecek <em>et al.</em> 2011</a>). Genotypes with very low depth of coverage have a greater likelihood of being called incorrectly as it can be difficult to distinguish between homozygotes and heterozygotes when very few reads are present. Similarly, genotypes with very high depth of coverage are more likely to be spurious as high coverage can result from misalignment due to the presence of paralogous loci or repeats (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Fountain <em>et al.</em> 2016</a>). Second, as including SNPs from short scaffolds can downwardly bias LD values, we retained only SNPs located on the longest 100 scaffolds of the assembly (min length = 6.6 Mb, max length = 33.1 Mb). Third, as an additional quality filtering step, we used information on known parental relationships to identify loci with Mendelian incompatibilities using the mendel function in PLINK v1.9 and removed these from the dataset. Fourth, to minimize any possible confounding effects of population structure and relatedness, we focused on adult individuals sampled from the single largest population of South Georgia. Fifth, to provide an informative dataset while further minimizing genotyping error, we discarded SNPs with a MAF of less than 0.1 and/or called in less than 50% of individuals using PLINK. As a final quality control step, we also removed SNPs that did not conform to Hardy-Weinberg equilibrium (HWE) with a <em>p</em>-value threshold &lt; 0.001 using the–hwe function in PLINK.</p><p id="p-24">Using the final dataset of 25,068 SNPs genotyped in 33 individuals (Figure S2A), we used the –r2 function in PLINK to quantify pairwise LD between all pairs of SNPs located within 500 kb of each other. We visualized LD decay with distance by fitting a nonlinear regression curve using the nls package in R, where the expected value of <em>r</em><sup>2</sup> under drift-recombination equilibrium (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-1.gif"/></noscript></span></span>) was expressed according to the Hill and Weir function (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hill and Weir 1988</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-2.gif"/></noscript></span></span>where <em>n</em> is the number of gametes scored, <em>P</em> = <em>4N<sub>e</sub>c</em>, <em>N<sub>e</sub></em> is the effective population size and <em>c</em> is the recombination fraction between sites (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Remington <em>et al.</em> 2001</a>).</p></div><div id="sec-10" class="subsection"><h3>Population structure</h3><p id="p-25">Prior to quantifying population structure, we filtered the full SNP dataset as shown in Figure S2B. We did not initially filter the dataset for SNPs with low depth of coverage, as for the analysis of population structure we wanted to retain as many SNPs as possible that were genotyped across all of the populations. We also did not remove individuals with large amounts of missing data in order to maximize the representation of all populations in the final dataset. Nevertheless, because closely related individuals can bias population genetic structure analysis by introducing both Hardy-Weinberg and linkage disequilibrium (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Rodríguez-Ramilo and Wang 2012</a>; <a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Wang 2017</a>), we used known parentage information to remove adults and related pups (full and half siblings) from the South Georgia dataset. This strategy also enabled us to attain a more even number of samples per population (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Puechmaille 2016</a>). Second, SNPs with a MAF of less than 0.05 and/or called in less than 99% of individuals were discarded using VCFtools. Third, we removed SNPs that were out of HWE at the population level with a <em>p</em>-value threshold &lt; 0.001 using the –hwe function in PLINK. Fourth, SNPs were pruned for LD using the –indep function in PLINK. We used a sliding window of 50 SNPs, a step size of 5 SNPs and removed all variants in a window above a variance inflation factor threshold of 2, corresponding to <em>r</em><sup>2</sup> = 0.5.</p><p id="p-26">Using the final dataset of 27,592 SNPs genotyped in 37 individuals (Figure S2B), we first visualized population structure by performing a principal components analysis (PCA) using the R package adegenet (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Jombart 2008</a>). We then used a Bayesian clustering algorithm implemented by the program STRUCTURE to identify the number of genetic clusters (<em>K</em>) present in the dataset. We performed STRUCTURE runs for values of <em>K</em> ranging from one to six, with five simulations for each <em>K</em> and a burn-in of 100,000 iterations followed by 1,000,000 Markov chain Monte Carlo iterations. We used the admixture and correlated allele frequency models without sampling location information. The R package pophelper (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Francis 2017</a>) was then used to analyze the STRUCTURE results, parse the output to CLUMPP for averaging across iterations and for visualizing individual assignment probabilities. The optimal <em>K</em> was selected based on the maximum value of the mean estimated <em>ln</em> probability of the data (Ln Pr(<em>X</em> | <em>K</em>)) as proposed by <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Pritchard <em>et al.</em> (2000)</a> and the Δ<em>K</em> method of <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Evanno <em>et al.</em> (2005)</a>. For comparison, we also implemented the above analyses using microsatellite data for the same individuals.</p></div><div id="sec-11" class="subsection"><h3>Inbreeding coefficients</h3><p id="p-27">Prior to quantifying inbreeding, we filtered the SNP dataset as shown in Figure S2C. First, we wanted a dataset with as few gaps as possible so we discarded one individual with more than 90% missing data. Second, we removed individual genotypes with a depth of coverage below eight or above 30 using vcftools. Third, we removed loci with Mendelian incompatibilities, and fourth, we again restricted the dataset to the focal population of South Georgia. Fifth, we discarded SNPs with a MAF of less than 0.05 and/or called in less than 70% of individuals using vcftools. Finally, we filtered the SNPs for HWE as described previously and pruned linked SNPs out of the dataset using the –indep function in PLINK with the parameters shown above.</p><p id="p-28">Using the final dataset of 9,853 SNPs genotyped in 56 individuals (Figure S2C), we calculated four genomic estimates of individual inbreeding: standardized multi-locus heterozygosity (sMLH; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Coltman <em>et al.</em> 1999</a>), an estimate based on the variance of additive genotype values (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-3.gif"/></noscript></span></span>), an estimate based on excess homozygosity (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-4.gif"/></noscript></span></span>) and an estimate based on the correlation of uniting gametes, which gives more weight to homozygotes of the rare allele at each locus (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-5.gif"/></noscript></span></span>) (<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Yang <em>et al.</em> 2011</a>). sMLH was calculated using the sMLH function in the R package inbreedR (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Stoffel <em>et al.</em> 2016</a>) and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-7.gif"/></noscript></span></span> were calculated in GCTA v1.24.3 (<a id="xref-ref-120-2" class="xref-bibr" href="#ref-120">Yang <em>et al.</em> 2011</a>). When inbreeding varies among individuals, heterozygosity measured at one part of the genome will be correlated with heterozygosity measured at another, a phenomenon termed identity disequilibrium (ID) (<a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">Weir and Cockerham 1973</a>; <a id="xref-ref-109-2" class="xref-bibr" href="#ref-109">Szulkin <em>et al.</em> 2010</a>). We quantified the extent of ID using the measure <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-8.gif"/></noscript></span></span>in the R package inbreedR (<a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Stoffel <em>et al.</em> 2016</a>) where significant <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-9.gif"/></noscript></span></span> values provide support for variance in inbreeding in the population. Finally, we compared the resulting <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-10.gif"/></noscript></span></span> value with the variance in our inbreeding coefficients to determine the expected correlation between estimated (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-11.gif"/></noscript></span></span> and realized <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-12.gif"/></noscript></span></span> levels of inbreeding (<a id="xref-ref-109-3" class="xref-bibr" href="#ref-109">Szulkin <em>et al.</em> 2010</a>) given as:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-13.gif"/></noscript></span></span></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-29">The PacBio sequencing reads and the final Antarctic fur seal genome assembly are available at the European Nucleotide Archive (ENA) under BioProject ID PRJEB26995. The raw RAD sequencing reads are available at the short-read archive (SRA) of the National Biotechonology Centre of Information (NCBI) under BioProject ID PRJNA473050, SRA accession SRP148937. Code for the analysis is available at <a href="https://github.com/elhumble/seal_rad_2018">https://github.com/elhumble/seal_rad_2018</a>. Supplemental material and SNP genotypes are available at Figshare: <a href="https://doi.org/10.25387/g3.6713975">https://doi.org/10.25387/g3.6713975</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Hybrid genome assembly</h3><p id="p-30">We used PacBio sequencing to improve an existing Antarctic fur seal genome assembly. Using PBJelly, we were able to close a total of 45,394 gaps, resulting in a 40% reduction in overall gap space (assembly v1.2, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Subsequent assembly correction with Quiver resulted in a total of 11,319,546 modifications to the PBJelly assembly consisting of 291,179 insertions, 1,117,226 substitutions and 9,911,141 deletions. Finally, PILON corrected 653,246 homozygous insertions (885,794 bp), 87,818 deletions (127,024 bp) and 34,438 single-base substitutions and closed an additional 2,172 gaps in the Quiver assembly. Overall, gap closing and error correction resulted in a hybrid Antarctic fur seal assembly with a total length of 2.3 Gb (v1.4, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The number of scaffolds in the genome was reduced from 8,126 to 6,169 such that 50% of the final assembly is now contained within the longest 108 scaffolds (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-15" class="subsection"><h3>Genome synteny</h3><p id="p-31">To investigate synteny between the Antarctic fur seal and the dog, we aligned the fur seal scaffolds to the dog genome (CanFam3.1). We estimated overall sequence divergence between the two species to be 13.8%. Visualization of the full alignment revealed that all of the dog chromosomes are represented in the fur seal assembly (Figure S3). Alignment of the 40 longest fur seal scaffolds (min length = 10.7 Mb, max length = 33.1 Mb) revealed strong chromosomal synteny between the two genomes, with the vast majority of the fur seal scaffolds mapping exclusively or mainly to a given dog chromosome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Specifically, for 37 of the scaffolds, over 90% of the total alignment length was to a single dog chromosome, with 26 of those aligning exclusively to a single dog chromosome. Only one scaffold (S4 in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) aligned in roughly equal portions to two different dog chromosomes (62% to D5 and 38% to D26).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny of the longest 40 Antarctic fur seal scaffolds (10.7&#x2013;33.1 Mb; right, prefixed S) with dog chromosomes (left, prefixed D). Mapping each fur seal scaffold to the dog genome resulted in multiple alignment blocks (mean = 2.1 kb, range = 0.1&#x2013;52.8 kb) and alignments over 5 kb are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025565988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Synteny of the longest 40 Antarctic fur seal scaffolds (10.7&#x2013;33.1 Mb; right, prefixed S) with dog chromosomes (left, prefixed D). Mapping each fur seal scaffold to the dog genome resulted in multiple alignment blocks (mean = 2.1 kb, range = 0.1&#x2013;52.8 kb) and alignments over 5 kb are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2709/F2.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/8/2709/F2.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490605" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Synteny of the longest 40 Antarctic fur seal scaffolds (10.7–33.1 Mb; right, prefixed S) with dog chromosomes (left, prefixed D). Mapping each fur seal scaffold to the dog genome resulted in multiple alignment blocks (mean = 2.1 kb, range = 0.1–52.8 kb) and alignments over 5 kb are shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>RAD sequencing and SNP discovery</h3><p id="p-33">RAD sequencing of 83 fur seal individuals generated an average of 5,689,065 250 bp paired-end reads per individual. After mapping these reads to the reference genome, a total of 677,607 biallelic SNPs were discovered using GATK’s best practices workflow for variant discovery (see Materials and methods for details), with an average SNP coverage of 727. We then filtered the dataset in three different ways (Figure S2) to generate datasets suitable for the analysis of LD decay, population structure and inbreeding respectively.</p></div><div id="sec-17" class="subsection"><h3>SNP validation</h3><p id="p-34">To provide an indication of the quality of our SNP dataset, we used Sanger sequencing to validate 50 randomly selected loci. For each locus, we sequenced a single heterozygote and a single homozygote individual based on the corresponding GATK genotypes. For 40 of these loci, we successfully obtained genotypes for both individuals (Table S2). Concordance between the GATK and Sanger genotypes was high, with 76/80 genotypes being called identically using both methods, equivalent to a validation rate of 95%. The four discordant genotypes were all initially called as homozygous with GATK but subsequently validated as heterozygous with Sanger sequencing.</p></div><div id="sec-18" class="subsection"><h3>LD decay</h3><p id="p-35">The pattern of LD decay within South Georgia was quantified based on 25,068 SNPs genotyped in 33 individuals and located on the 100 longest fur seal scaffolds. LD was found to decay rather rapidly, with <em>r</em><sup>2</sup> reaching the background level by around 400 kb (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Strong LD (<em>r</em><sup>2</sup> &gt;= 0.5) decayed by 5 kb and moderate LD (<em>r</em><sup>2</sup> &gt;= 0.2) by around 15 kb.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of linkage disequilibrium (r2) against physical distance between SNPs in the Antarctic fur seal. LD was calculated using 25,068 filtered SNPs from the 100 largest scaffolds of 33 South Georgia adults. Gray points indicate observed pairwise LD values. The dark gray curve shows the expected decay of LD in the data estimated by nonlinear regression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025565988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of linkage disequilibrium (r&lt;sup&gt;2&lt;/sup&gt;) against physical distance between SNPs in the Antarctic fur seal. LD was calculated using 25,068 filtered SNPs from the 100 largest scaffolds of 33 South Georgia adults. Gray points indicate observed pairwise LD values. The dark gray curve shows the expected decay of LD in the data estimated by nonlinear regression.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2709/F3.medium.gif" width="440" height="373"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/8/2709/F3.medium.gif" width="440" height="373"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490482" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Plot of linkage disequilibrium (r<sup>2</sup>) against physical distance between SNPs in the Antarctic fur seal. LD was calculated using 25,068 filtered SNPs from the 100 largest scaffolds of 33 South Georgia adults. Gray points indicate observed pairwise LD values. The dark gray curve shows the expected decay of LD in the data estimated by nonlinear regression.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Population structure</h3><p id="p-37">Next, we used a dataset of 37 pups genotyped at 27 microsatellites and 27,592 SNPs to quantify the pattern and strength of population structure across the species circumpolar range. PCA of the microsatellite dataset uncovered weak clustering with South Georgia, the South Shetlands and Bouvetøya tending to separate apart from Kerguelen, Heard and Macquarie Islands along the first PC axis (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). However, considerable scatter and no clear pattern of separation was found along either PC2 or PC3 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4A</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">4C</a>). By contrast, population structure was more clearly defined in the PCA of the SNP dataset. Specifically, the first PC axis clearly resolved two distinct island groups, the first comprising South Georgia, the South Shetlands and Bouvetøya and the second comprising Kerguelen, Heard Island and Macquarie Island (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). Within the first island group, Bouvetøya clustered apart from South Georgia and the South Shetlands along PC2 (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>) while all three locations clustered apart from one another along PC3 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatterplots showing individual variation in principal components (PCs) one and two (panels A and B), and one and three (panels C and D) derived from a principal component analysis conducted using 27 microsatellites (panels A and C) and 27,592 SNPs (panels B and D). The amount of variance explained by each PC is shown in parentheses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025565988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scatterplots showing individual variation in principal components (PCs) one and two (panels A and B), and one and three (panels C and D) derived from a principal component analysis conducted using 27 microsatellites (panels A and C) and 27,592 SNPs (panels B and D). The amount of variance explained by each PC is shown in parentheses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2709/F4.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/8/2709/F4.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490643" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Scatterplots showing individual variation in principal components (PCs) one and two (panels A and B), and one and three (panels C and D) derived from a principal component analysis conducted using 27 microsatellites (panels A and C) and 27,592 SNPs (panels B and D). The amount of variance explained by each PC is shown in parentheses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">To test whether population structure could be detected without prior knowledge of the sampling locations of individuals, we used a Bayesian approach implemented within STRUCTURE (<a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Pritchard <em>et al.</em> 2000</a>). This program works by partitioning the data set in such a way that departures from Hardy-Weinberg and linkage equilibrium within the resulting groups are minimized. Separately for the microsatellite and SNP datasets, five replicate runs were conducted for each possible number of clusters (<em>K</em>) ranging from one, implying no population differentiation, through to six, which would imply that all of the populations are genetically distinct. For the microsatellite dataset, Ln Pr(<em>X</em> | <em>K</em>) and Δ<em>K</em> both peaked at two, indicating support for the presence of two genetically distinct populations (Figure S4A and C). Membership coefficients for the inferred clusters are summarized in Figure S5A and indicate the presence of a Western population comprising individuals from South Georgia, the South Shetlands and Bouvetøya, and an Eastern population comprising individuals from Kerguelen, Heard Island and Macquarie Island.</p><p id="p-40">For the RAD dataset, Ln Pr(<em>X</em> | <em>K</em>) also peaked at two but remained high for <em>K</em> = 3 and 4, while Δ<em>K</em> reached its maximum at <em>K</em> = 4 (Figure S4B and D). To explore this further, we plotted membership coefficients for <em>K</em> = 2–6 for both the microsatellite and SNP datasets. For the former, no evidence of population structure was found beyond <em>K</em> = 2, with successive increases in <em>K</em> merely introducing additional admixture (Figure S5A). By contrast, for the latter, plots corresponding to <em>K</em> values greater than two clearly resolved further hierarchical structure (Figure S5B). Results for <em>K</em> = 4 are shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, in which Kerguelen, Heard and Macquarie Islands are resolved as a single population, while South Georgia, the South Shetlands and Bouvetøya can be readily distinguished based on their corresponding group membership coefficients.</p></div><div id="sec-20" class="subsection"><h3>Inbreeding</h3><p id="p-41">Finally, we investigated inbreeding in the focal population at South Georgia using data from 9,853 SNPs genotyped in 56 individuals (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Identity disequilibrium differed significantly from zero (0.0052; bootstrap 95% confidence interval = 0.0008–0.0091, <em>P</em> = 0.023, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>) providing evidence for variance in inbreeding within the sample of individuals. Each individual’s level of inbreeding was quantified from the SNP dataset using four different genomic inbreeding coefficients (sMLH, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-14.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-16.gif"/></noscript></span></span>). All four of these measures were inter-correlated, with correlation coefficients (<em>r</em>) ranging from 0.69 to 0.84. (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C–E</a>). Furthermore, the variances of all four measures fell within the 95% confidence interval of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-17.gif"/></noscript></span></span> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), suggesting that the expected correlation between the estimated and realized level of inbreeding does not differ significantly from one.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Distribution of genomic inbreeding coefficients () for 56 individual fur seals from South Georgia; (B) Distribution of identity disequilibrium () estimates from bootstrapping over individuals. The vertical dashed line represents the empirical  estimate and the horizontal black line shows the corresponding 95% confidence interval based on 1000 bootstrap replicates. The vertical colored lines represent the variance in four different inbreeding coefficients. Panels C, D and E show pairwise Pearson&#x2019;s correlation coefficients between  and ,  and sMLH." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025565988" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Distribution of genomic inbreeding coefficients (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-18&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) for 56 individual fur seals from South Georgia; (B) Distribution of identity disequilibrium (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-19&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) estimates from bootstrapping over individuals. The vertical dashed line represents the empirical &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-20&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; estimate and the horizontal black line shows the corresponding 95% confidence interval based on 1000 bootstrap replicates. The vertical colored lines represent the variance in four different inbreeding coefficients. Panels C, D and E show pairwise Pearson&#x2019;s correlation coefficients between &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-21&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-22&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-23&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and sMLH.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/8/2709/F5.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/8/2709/F5.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/8/2709/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490607" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">(A) Distribution of genomic inbreeding coefficients (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-18.gif"/></noscript></span></span>) for 56 individual fur seals from South Georgia; (B) Distribution of identity disequilibrium (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-19.gif"/></noscript></span></span>) estimates from bootstrapping over individuals. The vertical dashed line represents the empirical <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-20.gif"/></noscript></span></span> estimate and the horizontal black line shows the corresponding 95% confidence interval based on 1000 bootstrap replicates. The vertical colored lines represent the variance in four different inbreeding coefficients. Panels C, D and E show pairwise Pearson’s correlation coefficients between <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-22.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/F5/embed/mml-math-23.gif"/></noscript></span></span> and sMLH.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-43">Advances in high throughput sequencing technology have afforded researchers the opportunity to generate genome assemblies and genomic marker datasets for virtually any species from which high quality DNA can be collected. These resources allow a broad variety of questions in ecology and evolution to be addressed with greater power and precision than was possible with traditional methods. In this study, we utilized PacBio sequencing to refine an existing Antarctic fur seal genome assembly and combined this with RAD sequencing to characterize synteny with the dog genome, elucidate the rate of LD decay, resolve global population structure and quantify the variance in inbreeding. Our results provide new insights at multiple levels of organization that enrich our understanding of an important Antarctic marine top predator and indicate the general promise of these and related approaches for tackling broad-reaching questions in population and evolutionary genetics.</p><div id="sec-22" class="subsection"><h3>Genome alignment</h3><p id="p-44">An important outcome of this study is a significantly improved Antarctic fur seal genome assembly. This was achieved through three iterative steps involving gap filling, inclusion of long PacBio reads and assembly error correction respectively. Overall, the number of scaffolds was reduced by around one quarter, while N50 almost doubled to over 6 Mb and the proportion of gaps was reduced by around an order of magnitude to around half a percent. This represents an improvement over existing pinniped assemblies such as the walrus (<em>Odobenus rosmarus divergens</em>, GenBank accession number GCA_000321225.1) and Weddell seal (<em>Leptonychotes weddellii</em>, GenBank accession number GCA_000349705.1), which both have lower N50 values (2.6 and 0.9 Mb respectively). The improved Antarctic fur seal genome will therefore serve as an important resource for the wider pinniped community. However, there is still considerable room for improvement as a handful of other marine mammal genome assemblies incorporating longer range information show higher levels of contiguity, such as the polar bear (<em>Ursus maritimus</em>, N50 = 15.9 Mb), killer whale (<em>Orcinus orca</em>, N50 = 12.7 Mb) and Hawaiian monk seal (<em>Neomonachus schauinslandi</em>, N50 = 22.2 Mb) (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Liu <em>et al.</em> 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Foote <em>et al.</em> 2015</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Mohr <em>et al.</em> 2017</a>).</p><p id="p-45">To further quantify genome quality and to explore patterns of synteny, we mapped the scaffolds of our new assembly to the dog genome. The resulting alignment revealed almost complete coverage of the dog chromosomes. This is in line with the observation that the total length of the assembly has not changed appreciably between versions and suggests that the assembly is near-complete, with the exception of the Y-chromosome for which sequence data are currently lacking as the genome individual is a female. In general, carnivore genomes show high levels of synteny (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arnason 1974</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ferguson-Smith and Trifonov 2007</a>), with pinnipeds in particular exhibiting highly conserved karyotypes indicative of slow rates of chromosomal evolution (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beklemisheva <em>et al.</em> 2016</a>). By contrast, the domestic dog has an extensively re-arranged karyotype differentiated from the ancestral carnivore karyotype by over 40 separate fission events (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Nie <em>et al.</em> 2011</a>). To provide insights into the extent of conservation of chromosomal blocks between seals and dogs, we mapped the longest 40 fur seal scaffolds to the dog genome. We found a clear pattern whereby all but one of the scaffolds mapped exclusively or mainly to single chromosomes, indicating the conservation of large genomic tracts often several Mb in length. The remaining scaffold mapped to two dog chromosomes in roughly equal proportions, suggestive of either a fission event in the lineage leading to dogs or a fusion event in the lineage leading to seals. By focusing only on the largest scaffolds, we had little power to detect multiple chromosomal rearrangements, although these are to be expected given a substantial increase in the number of chromosomes in dogs (2n = 74) relative to the seal (2n = 36) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Gustavsson 1964</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arnason 1974</a>). Nevertheless, the observed high degree of synteny is consistent with previous studies revealing both strong sequence homology and the conservation of polymorphic loci between seals and dogs (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Osborne <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Hoffman <em>et al.</em> 2013</a>).</p></div><div id="sec-23" class="subsection"><h3>SNP discovery and validation</h3><p id="p-46">Our study identified a total of 677,607 SNPs in a discovery pool of 83 individuals. These markers will be useful for future studies including the planned development of a high-density SNP array. However, not all SNPs are suitable for every analysis due to differential sensitivity to missing data, low depth of sequencing coverage and the inclusion of low frequency alleles (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Shafer <em>et al.</em> 2017</a>). We therefore carefully considered how best to filter our SNP dataset for each of our main analyses. For LD decay, we applied relatively strict filters as we sought a high-quality dataset with consistently high coverage across individuals. For population structure, it was important to have as many SNPs as possible represented in all of the sampling locations, so we did not remove genotypes with low coverage but instead filtered the data to retain SNPs genotyped in at least 99% of individuals. Conversely, for the estimation of inbreeding, we honed in on a reduced subset of higher quality SNPs with greater average depth of coverage, that were in Hardy-Weinberg and linkage equilibrium, and which showed no evidence of Mendelian incompatibilities.</p><p id="p-47">Even with stringent filtering, it is possible to retain SNPs in a dataset that have been called incorrectly. We therefore attempted to validate 50 randomly selected loci by Sanger sequencing selected individuals with homozygous and heterozygous genotypes as determined from the RAD data. For the 40 loci that we were able to successfully sequence, around 95% of the Sanger genotypes were identical to the RAD genotypes. Although this validation step required additional experimental effort, our results compare favorably with other studies (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cruz <em>et al.</em> 2017</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bourgeois <em>et al.</em> 2018</a>) and thus give us confidence in the overall quality of our data.</p></div><div id="sec-24" class="subsection"><h3>Linkage disequilibrium decay</h3><p id="p-48">We used the genomic positions of SNPs mapping to the largest 100 scaffolds to quantify the pattern of LD decay in the focal population of South Georgia. We found that LD decays rapidly, with moderate LD extending less than 20 kb. This is despite the species having experienced a population bottleneck in the 19<sup>th</sup> century, which would be expected to increase LD. A direct comparison with other organisms is hindered both by a paucity of data for most species and by the use of different measures for quantifying LD. However, our results are broadly in line with other wild vertebrate populations such as Alaskan gray wolves, polar bears and flycatchers, where moderate LD also extends less than 10–30 kb (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gray <em>et al.</em> 2009</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Malenfant <em>et al.</em> 2015</a>; Kardos <em>et al.</em> 2016). Extended LD has been documented in a number of species but in most cases this is associated with extreme bottlenecks, such as those experienced during domestication (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Harmegnies <em>et al.</em> 2006</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">McKay <em>et al.</em> 2007</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Meadows <em>et al.</em> 2008</a>). Although Antarctic fur seals are generally believed to have also experienced a strong historical bottleneck, a recent Bayesian analysis suggested that this may have been less severe than thought, with the effective population size probably falling to several hundred (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Hoffman <em>et al.</em> 2011</a>). Furthermore, the population recovered from the bottleneck within a few generations, which could have mitigated the increased genetic drift and inbreeding effects that elevate and maintain strong LD. Additionally, the population is currently estimated to number around 2–3 million individuals (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Boyd 1993</a>) and is one of the most genetically diverse pinnipeds (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Stoffel <em>et al.</em> 2018</a>). Therefore, given that LD is a function of both recombination rate and population size (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hill 1981</a>), the rapid decay of LD in this species might also be a reflection of high long-term effective population sizes.</p></div><div id="sec-25" class="subsection"><h3>Population structure</h3><p id="p-49">To provide further insights into the recovery of Antarctic fur seals globally, we quantified population structure across the species geographic range. Microsatellite genotypes provided evidence for two major geographic clusters, the first corresponding to South Georgia, the South Shetlands and Bøuvetoya, and the second corresponding to Kerguelen, Heard and Macquarie Island. By contrast, the RAD data uncovered an additional level of hierarchical structure, resolving South Georgia, the South Shetlands and Bøuvetoya as distinct populations. This is consistent with simulation studies suggesting that thousands of SNPs should outperform small panels of microsatellites at resolving population structure (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Haasl and Payseur 2011</a>) as well as with more recent empirical studies that have directly compared microsatellites with SNPs (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Rašić <em>et al.</em> 2014</a>; <a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">Vendrami <em>et al.</em> 2017</a>). Furthermore, many of our populations had sample sizes of around five individuals yet could still be clearly distinguished from one another. This is in line with a recent simulation study suggesting that sample sizes as small as four individuals may be adequate for resolving population structure when the number of markers is large (<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">Willing <em>et al.</em> 2012</a>). Thus, our results have positive implications for studies of threatened species for which extensive sampling can be difficult but where understanding broad as well as fine-scale population structure is of critical importance.</p><p id="p-50">It is generally believed that Antarctic fur seals were historically extirpated from virtually all of their contemporary breeding sites across the sub-Antarctic, with the possible exception of Bøuvetoya, where sealing expeditions were more sporadic (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Christensen 1935</a>) and around a thousand breeding individuals were sighted just a few decades after the cessation of hunting (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Olstad 1928</a>). South Georgia was the first population to stage a major recovery, probably because a number of individuals survived at isolated locations inaccessible to sealers around the South Georgia mainland (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bonner 1964</a>). Consequently, several authors have speculated that emigrant individuals from the expanding South Georgia population may have recolonized the species former range (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Boyd 1993</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Hucke-Gaete <em>et al.</em> 2004</a>). However, <a id="xref-ref-119-2" class="xref-bibr" href="#ref-119">Wynen <em>et al.</em> (2000)</a> resolved two main island groups with mtDNA, while <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bonin <em>et al.</em> (2014)</a> found that significant differences between the South Shetland Islands and South Georgia with microsatellites. Our results build on these studies in two ways. First, the two major clusters we resolved using both microsatellites and RAD sequencing are identical to those identified by <a id="xref-ref-119-3" class="xref-bibr" href="#ref-119">Wynen <em>et al.</em> (2000)</a>, suggesting that broad-scale population structure is not simply driven by female philopatry but is also present in the nuclear genome. Second, within the Western part of the species range, we not only found support for the South Shetlands being different from South Georgia, but also Bøuvetoya, suggesting that relict populations probably survived at all three of these locations. By contrast, no sub-structure could be detected within the Eastern part of the species range, which taken at face value might suggest that a single population survived sealing in this region. Consistent with this, historical records suggest that fur seals went locally extinct at Heard and Macquarie islands (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Page <em>et al.</em> 2003</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Goldsworthy <em>et al.</em> 2009</a>) and these populations may therefore have been recolonized by surviving populations in the Kerguelen archipelago. Thus, our study highlights the importance of relict populations to species recovery while also providing some evidence in support of local extinctions having occurred.</p></div><div id="sec-26" class="subsection"><h3>Inbreeding</h3><p id="p-51">Delving a level deeper, we investigated individual variation in the form of inbreeding. A recent meta-analysis has shown that small panels of microsatellites are almost always underpowered to detect variation in inbreeding (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Miller <em>et al.</em> 2014</a>). By contrast, a handful of recent studies have shown that tens of thousands of SNPs are capable of accurately quantifying inbreeding (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Hoffman <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Huisman <em>et al.</em> 2016</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bérénos <em>et al.</em> 2016</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chen <em>et al.</em> 2016</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Kardos <em>et al.</em> 2018</a>). While empirical studies to date have largely focused on small, isolated populations where inbreeding may be common, it is less clear how prevalent inbreeding could be in larger, free-ranging populations. We found several lines of evidence in support of inbreeding in fur seals. First, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-24.gif"/></noscript></span></span> was significantly positive indicating identity disequilibrium within the sample of individuals. Second, the variance of all genomic inbreeding coefficients was found to lie within the 95% confidence intervals of <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-25.gif"/></noscript></span></span> and therefore we can expect our estimates to reflect the realized level of inbreeding in the population. Third, all inbreeding coefficients were strongly intercorrelated, suggesting that our markers are uncovering consistent information about variation in genome-wide homozygosity caused by inbreeding.</p><p id="p-52">Our results are in some respects surprising given that Antarctic fur seals number in the millions and are free-ranging and highly vagile. However, the species is also highly polygynous, with a handful of top males fathering the majority of offspring (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Hoffman <em>et al.</em> 2004</a>) and females exhibiting strong natal site fidelity (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Hoffman and Forcada 2012</a>), which could potentially lead to matings between individuals with shared ancestry. As demographic effects can also generate variance in inbreeding <em>sensu lato</em>, we also cannot discount the possibility that the historical bottleneck contributed toward the variation we see today. To test this, we would need to quantify the length distribution of runs of homozygosity, which would require denser SNP data.</p><p id="p-53">Our work builds upon another recent study that used RAD sequencing to quantify inbreeding in wild harbor seals (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Hoffman <em>et al.</em> 2014</a>) where a higher estimate of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-26.gif"/></noscript></span></span> was found, indicative of a greater variance in inbreeding within the sample. However, the study focused on stranded seals, many of which died of lungworm infection and may therefore have been enriched for unusually inbred individuals. In the current study, pups were sampled at random from within a single breeding colony, together with their parents. Consequently, our sample should be more representative of the underlying distribution of inbreeding within the population. In line with this, our estimate of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/8/2709/embed/mml-math-27.gif"/></noscript></span></span> is more similar to those obtained in wild populations of other polygynous mammals such as Soay sheep and red deer (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bérénos <em>et al.</em> 2016</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Huisman <em>et al.</em> 2016</a>).</p><p id="p-54">Our results are consistent with previous studies documenting HFCs for numerous traits in the South Georgia population (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Hoffman <em>et al.</em> 2004</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">2010</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Forcada and Hoffman 2014</a>) and suggest that these may well reflect inbreeding depression. More generally, literally hundreds of studies have documented HFCs across the animal kingdom (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Coltman and Slate 2003</a>) and it has been strongly argued that these HFCs are highly unlikely to occur when there is no variance in inbreeding (<a id="xref-ref-109-4" class="xref-bibr" href="#ref-109">Szulkin <em>et al.</em> 2010</a>). The fact that we found variation in inbreeding in a large, free-ranging population is consistent with this notion and therefore contributes toward a growing body of evidence suggesting collectively that inbreeding could be more common in wild populations than previously thought.</p></div><div id="sec-27" class="subsection"><h3>Conclusion</h3><p id="p-55">We have generated an improved genome assembly for an important Antarctic marine top predator and used RAD sequencing to provide diverse insights from the level of the species through the population to the individual. Focusing on the larger South Georgia population, we characterized rapid LD decay and uncovered significant variation in individual inbreeding, while population-level analyses resolved clear differences among island groups that emphasize the importance of relict populations to species recovery. RAD sequencing and related approaches might conceivably be applied to other wild species to characterize patterns of LD decay, elucidate fine scale population structure and uncover the broader prevalence of inbreeding and its importance to wild populations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank the British Antarctic Survey’s field assistants on Bird Island for the collection of tissue samples and David Vendrami and Martin Stoffel for useful discussions about SNP filtering. We would like to acknowledge Anika Winkler who carried out the Illumina sequencing. We would also like to thank Rene Malenfant and one anonymous reviewer for their helpful comments. Samples from Cape Shirreff, South Shetland Islands were collected under U.S. Marine Mammal Protection Act permit #16074. This work contributes to the Ecosystems project of the British Antarctic Survey, Natural Environmental Research Council, and is part of the Polar Science for Planet Earth Program. This project was funded by a Deutsche Forschungsgemeinschaft standard grant (HO 5122/3-1) and was part of the SFG TRR 212 (NC<sup>3</sup>). It was also supported by the Swedish Research Council FORMAS (231-2012-450) as well as by core funding from the Natural Environment Research Council to the British Antarctic Survey’s Ecosystems Program. KKD was supported by the Natural Environment Research Council (grant NE/K012886/1) and AMB was supported by the Knut and Alice Wallenberg Foundation as part of the National Bioinformatics Infrastructure Sweden at SciLifeLab.</p><p id="p-57">AUTHOR CONTRIBUTIONS</p><p id="p-58">EH and JIH conceived and designed the study. IG and JIH carried out the DNA extractions and microsatellite genotyping. KKD carried out the RAD library preparation. A-CP performed the SNP validation. JF, SG, MG, KKD, JK, JIH and JBWW contributed materials and funding. AMB assembled the new version of the genome assembly with input from JBWW. EH carried out the SNP calling and analyzed the data. EH and JIH wrote the manuscript. All of the authors commented on and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-59">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6713975">https://doi.org/10.25387/g3.6713975</a>.</p></li><li class="fn" id="fn-4"><p id="p-60"><em>Communicating editor: M. Jakobsson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-21"><span class="received-label">Received </span>February 21, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-18"><span class="accepted-label">Accepted </span>June 18, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Humble <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.1" data-doi="10.1111/j.1601-5223.1974.tb01340.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnason</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Comparative chromosome studies in Pinnipedia.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-5223.1974.tb01340.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DArnason%26rft.auinit1%253DU.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D179%26rft.epage%253D226%26rft.atitle%253DComparative%2Bchromosome%2Bstudies%2Bin%2BPinnipedia.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1974.tb01340.x%26rft_id%253Dinfo%253Apmid%252F4135821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1974.tb01340.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4135821&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.3" data-doi="10.1111/j.1365-294X.2004.02318.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Does heterozygosity estimate inbreeding in real populations?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3021</span>–<span class="cit-lpage">3031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2004.02318.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D3021%26rft.epage%253D3031%26rft.atitle%253DDoes%2Bheterozygosity%2Bestimate%2Binbreeding%2Bin%2Breal%2Bpopulations%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02318.x%26rft_id%253Dinfo%253Apmid%252F15367117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02318.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15367117&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223863000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.4" data-doi="10.1371/journal.pone.0147647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beklemisheva</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perelman</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemskaya</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulemzina</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proskuryakova</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The ancestral carnivore karyotype as substantiated by comparative chromosome painting of three pinnipeds, the walrus, the steller sea lion and the baikal seal (Pinnipedia, Carnivora).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0147647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0147647</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0147647%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0147647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0147647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.5" data-doi="10.1111/mec.13245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benestan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosselin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sainte-Marie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rochette</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RAD genotyping reveals fine-scale genetic structuring and provides powerful population assignment in a widely distributed marine species, the American lobster (<em>Homarus americanus</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3299</span>–<span class="cit-lpage">3315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D3299%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.6" data-doi="10.1111/mec.13681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bérénos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilkington</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic analysis reveals depression due to both individual and maternal inbreeding in a free-living mammal population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3152</span>–<span class="cit-lpage">3168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D3152%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.7" data-doi="10.1002/ece3.732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goebel</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Unexpected genetic differentiation between recently recolonized populations of a long-lived and highly vagile marine mammal.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">3701</span>–<span class="cit-lpage">3712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.732</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D3%26rft.spage%253D3701%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.732%26rft_id%253Dinfo%253Apmid%252F24198934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24198934&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.8" data-doi="10.1007/s00265-013-1674-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goebel</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High male reproductive success in a low-density Antarctic fur seal (<em>Arctocephalus gazella</em>) breeding colony.</span> <abbr class="cit-jnl-abbrev">Behav. Ecol. Sociobiol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00265-013-1674-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%2BSociobiol.%26rft.volume%253D68%26rft.spage%253D597%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00265-013-1674-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00265-013-1674-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-8.8.2709.9"><div class="cit-metadata unstructured">Bonner, W. N., 1964 The Fur Seal of South Georgia. British Antarctic Survey Report 1–95.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourgeois</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senn</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Single-nucleotide polymorphism discovery and panel characterization in the African forest elephant.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D48%26rft.spage%253D113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.11" data-doi="10.1111/j.1365-294X.2005.02598.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bass</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soares</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Conservation implications of complex population structure: lessons from the loggerhead turtle (<em>Caretta caretta</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02598.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBowen%26rft.auinit1%253DB.%2BW.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2389%26rft.epage%253D2402%26rft.atitle%253DConservation%2Bimplications%2Bof%2Bcomplex%2Bpopulation%2Bstructure%253A%2Blessons%2Bfrom%2Bthe%2Bloggerhead%2Bturtle%2B%2528Caretta%2Bcaretta%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02598.x%26rft_id%253Dinfo%253Apmid%252F15969722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02598.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969722&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.12" data-doi="10.1017/S0954102093000045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Pup production and distribution of breeding Antarctic fur seals (<em>Arctocephalus gazella</em>) at South Georgia.</span> <abbr class="cit-jnl-abbrev">Antarct. Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0954102093000045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntarct.%2BSci.%26rft.volume%253D5%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0954102093000045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0954102093000045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993KQ24500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.13" data-doi="10.1086/381000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/381000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DCarlson%26rft.auinit1%253DC.%2BS.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D120%26rft.atitle%253DSelecting%2Ba%2Bmaximally%2Binformative%2Bset%2Bof%2Bsingle-nucleotide%2Bpolymorphisms%2Bfor%2Bassociation%2Banalyses%2Busing%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F381000%26rft_id%253Dinfo%253Apmid%252F14681826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/381000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681826&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187723500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.14" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.15" data-doi="10.1111/j.1365-294X.2009.04247.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheldon</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A quantitative review of heterozygosity-fitness correlations in animal populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2746</span>–<span class="cit-lpage">2765</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2009.04247.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DChapman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D18%26rft.issue%253D13%26rft.spage%253D2746%26rft.epage%253D2765%26rft.atitle%253DA%2Bquantitative%2Breview%2Bof%2Bheterozygosity-fitness%2Bcorrelations%2Bin%2Banimal%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04247.x%26rft_id%253Dinfo%253Apmid%252F19500255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04247.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19500255&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267284100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.16" data-doi="10.1016/j.cub.2016.08.062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosgrove</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic consequences of population decline in the endangered Florida scrub-jay.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2974</span>–<span class="cit-lpage">2979</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2016.08.062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D2974%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2016.08.062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2016.08.062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2709.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-source">Such is the Antarctic</span>, <span class="cit-publ-name">Hodder and Stoughton</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.18" data-doi="10.1554/0014-3820(2003)057[0971:MMOIAM]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Microsatellite measures of inbreeding: a meta-analysis.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">971</span>–<span class="cit-lpage">983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2003.tb00309.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DColtman%26rft.auinit1%253DD.%2BW.%26rft.volume%253D57%26rft.issue%253D5%26rft.spage%253D971%26rft.epage%253D983%26rft.atitle%253DMicrosatellite%2Bmeasures%2Bof%2Binbreeding%253A%2Ba%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282003%2529057%255B0971%253AMMOIAM%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12836816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2003)057[0971:MMOIAM]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12836816&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183500900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.19" data-doi="10.2307/2640828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilkington</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Parasite-mediated selection against inbred soay sheep in a free-living island populaton.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1259</span>–<span class="cit-lpage">1267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D1259%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000082389200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.20" data-doi="10.1186/s12864-015-1930-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocher</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An improved genome reference for the African cichlid, <em>Metriaclima zebra</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1930-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D724%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1930-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1930-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.21" data-doi="10.1111/1755-0998.12564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Identification and validation of single nucleotide polymorphisms as tools to detect hybridization and population structure in freshwater stingrays.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12564</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.22" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.23" data-doi="10.1186/1471-2164-15-53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MafFilter: a highly flexible and extensible multiple genome alignment files processor.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-53</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-53%26rft_id%253Dinfo%253Apmid%252F24447531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24447531&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.24" data-doi="10.1038/hdy.2010.152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekblom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Applications of next generation sequencing in molecular ecology of non-model organisms.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DEkblom%26rft.auinit1%253DR.%26rft.volume%253D107%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D15%26rft.atitle%253DApplications%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bin%2Bmolecular%2Becology%2Bof%2Bnon-model%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.152%26rft_id%253Dinfo%253Apmid%252F21139633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21139633&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291635500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.25" data-doi="10.1111/eva.12178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekblom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A field guide to whole-genome sequencing, assembly and annotation.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1026</span>–<span class="cit-lpage">1042</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D7%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12178%26rft_id%253Dinfo%253Apmid%252F25553065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553065&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.26" data-doi="10.1016/j.tree.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing and population genomics in non-model organisms.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D29%26rft.spage%253D51%26rft.atitle%253DGENOME%2BSEQUENCING%2BAND%2BPOPULATION%2BGENOMICS%2BIN%2BNON-MODEL%2BORGANISMS%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139972&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330148000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.27" data-doi="10.1371/journal.pone.0047768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vee</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0047768</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEnglish%26rft.auinit1%253DA.%2BC.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47768%26rft.epage%253De47768%26rft.atitle%253DMind%2Bthe%2BGap%253A%2BUpgrading%2BGenomes%2Bwith%2BPacific%2BBiosciences%2BRS%2BLong-Read%2BSequencing%2BTechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047768%26rft_id%253Dinfo%253Apmid%252F23185243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185243&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.28" data-doi="10.1007/978-1-61779-228-1_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNP discovery and genotyping for evolutionary genetics using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">772</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-228-1_9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEtter%26rft.auinit1%253DP.%2BD.%26rft.volume%253D772%26rft.spage%253D157%26rft.epage%253D178%26rft.atitle%253DSNP%2Bdiscovery%2Band%2Bgenotyping%2Bfor%2Bevolutionary%2Bgenetics%2Busing%2BRAD%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-228-1_9%26rft_id%253Dinfo%253Apmid%252F22065437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-228-1_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22065437&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321876000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.29" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software structure: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02553.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.30" data-doi="10.1038/nrg2199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferguson-Smith</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trifonov</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mammalian karyotype evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">950</span>–<span class="cit-lpage">962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFerguson-Smith%26rft.auinit1%253DM.%2BA.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D950%26rft.epage%253D962%26rft.atitle%253DMammalian%2Bkaryotype%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2199%26rft_id%253Dinfo%253Apmid%252F18007651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18007651&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251551100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.31" data-doi="10.1038/ng.3198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foote</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">G. W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alföldi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Convergent evolution of the genomes of marine mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D272%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3198%26rft_id%253Dinfo%253Apmid%252F25621460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25621460&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.32" data-doi="10.1038/nature13542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Climate change selects for heterozygosity in a declining fur seal population.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">511</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">465</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13542</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D511%26rft.spage%253D462%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13542%26rft_id%253Dinfo%253Apmid%252F25056064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25056064&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.33" data-doi="10.1111/1755-0998.12519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fountain</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauli</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palsbøll</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peery</span>  <span class="cit-name-given-names">M. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Finding the right coverage: the impact of coverage and sequence quality on single nucleotide polymorphism genotyping error rates.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12519</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D966%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12519%26rft_id%253Dinfo%253Apmid%252F26946083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26946083&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.34" data-doi="10.1111/1755-0998.12509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">pophelper: an R package and web app to analyse and visualize population structure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.35" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacCallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1017351108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.36" data-doi="10.1007/s00300-009-0645-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldsworthy</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaughnessy</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fur seals at Macquarie Island: post-sealing colonisation, trends in abundance and hybridisation of three species.</span> <abbr class="cit-jnl-abbrev">Polar Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1473</span>–<span class="cit-lpage">1486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00300-009-0645-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPolar%2BBiology%26rft.stitle%253DPolar%2BBiology%26rft.volume%253D32%26rft.issue%253D10%26rft.spage%253D1473%26rft.epage%253D1486%26rft.atitle%253DFur%2Bseals%2Bat%2BMacquarie%2BIsland%253B%2Bpost-sealing%2Bcolonisation%252C%2Btrends%2Bin%2Babundance%2Band%2Bhybridisation%2Bof%2Bthree%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00300-009-0645-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00300-009-0645-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000270187100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.37" data-doi="10.1534/genetics.108.098830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granka</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutter</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Linkage disequilibrium and demographic history of wild and domestic canids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098830</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098830%26rft_id%253Dinfo%253Apmid%252F19189949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS80LzE0OTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.38" data-doi="10.1111/j.1601-5223.1964.tb01926.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustavsson</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The chromosomes of the dog.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-5223.1964.tb01926.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.volume%253D51%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1964.tb01926.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1964.tb01926.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19642221B00005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.39" data-doi="10.1038/hdy.2010.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haasl</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.21</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaasl%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D158%26rft.epage%253D171%26rft.atitle%253DMulti-locus%2Binference%2Bof%2Bpopulation%2Bstructure%253A%2Ba%2Bcomparison%2Bbetween%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Band%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.21%26rft_id%253Dinfo%253Apmid%252F20332809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20332809&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285336100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.40" data-doi="10.1111/j.1365-294X.2009.04120.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagenblad</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostrander</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of the inbred Scandinavian wolf.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1351</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2009.04120.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHagenblad%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D1341%26rft.epage%253D1351%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bthe%2Binbred%2BScandinavian%2Bwolf.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04120.x%26rft_id%253Dinfo%253Apmid%252F19368642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04120.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19368642&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.41" data-doi="10.1111/j.1365-2052.2006.01438.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harmegnies</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farnir</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buys</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georges</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Measuring the extent of linkage disequilibrium in commercial pig populations.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2052.2006.01438.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHarmegnies%26rft.auinit1%253DN.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D225%26rft.epage%253D231%26rft.atitle%253DMeasuring%2Bthe%2Bextent%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bcommercial%2Bpig%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2006.01438.x%26rft_id%253Dinfo%253Apmid%252F16734681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2006.01438.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16734681&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.42" data-doi="10.1007/s10592-017-0939-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hendricks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönfeld</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiench</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamede</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Conservation implications of limited genetic diversity and population structure in Tasmanian devils (<em>Sarcophilus harrisii</em>).</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10592-017-0939-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D18%26rft.spage%253D977%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10592-017-0939-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10592-017-0939-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.43" data-doi="10.1017/S0016672300020553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Estimation of effective population size from data on linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300020553</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D38%26rft.spage%253D209%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300020553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300020553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981MX44600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.44" data-doi="10.1016/0040-5809(88)90004-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Variances and covariances of squared linkage disequilibria in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(88)90004-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D78%26rft.atitle%253DVariances%2Band%2Bcovariances%2Bof%2Bsquared%2Blinkage%2Bdisequilibria%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252888%252990004-4%26rft_id%253Dinfo%253Apmid%252F3376052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(88)90004-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3376052&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988M750300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.45" data-doi="10.1016/j.mambio.2011.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extreme natal philopatry in female Antarctic fur seals (<em>Arctocephalus gazella</em>).</span> <abbr class="cit-jnl-abbrev">Mamm. Biol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mambio.2011.09.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BBiol.%26rft.volume%253D77%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mambio.2011.09.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mambio.2011.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.46" data-doi="10.1554/04-099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Exploring the relationship between parental relatedness and male reproductive success in the Antarctic fur seal <em>Arctocephalus gazella</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">2087</span>–<span class="cit-lpage">2099</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2004.tb00492.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHoffman%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D58%26rft.issue%253D9%26rft.spage%253D2087%26rft.epage%253D2099%26rft.atitle%253DExploring%2Bthe%2Brelationship%2Bbetween%2Bparental%2Brelatedness%2Band%2Bmale%2Breproductive%2Bsuccess%2Bin%2Bthe%2BAntarctic%2Bfur%2Bseal%2BArctocephalus%2Bgazella.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F04-099%26rft_id%253Dinfo%253Apmid%252F15521464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/04-099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15521464&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224339500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.47" data-doi="10.1554/02-530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Male reproductive strategy and the importance of maternal status in the Antarctic fur seal <em>Arctocephalus gazella</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1917</span>–<span class="cit-lpage">1930</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2003.tb00598.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHoffman%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D57%26rft.issue%253D8%26rft.spage%253D1917%26rft.epage%253D1930%26rft.atitle%253DMale%2Breproductive%2Bstrategy%2Band%2Bthe%2Bimportance%2Bof%2Bmaternal%2Bstatus%2Bin%2Bthe%2BAntarctic%2Bfur%2Bseal%2BArctocephalus%2Bgazella.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F02-530%26rft_id%253Dinfo%253Apmid%252F14503632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/02-530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14503632&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185073500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.48" data-doi="10.1093/jhered/esq046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Exploring the mechanisms underlying a heterozygosity-fitness correlation for canine size in the Antarctic fur seal Arctocephalus gazella.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esq046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesq046%26rft_id%253Dinfo%253Apmid%252F20457623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esq046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20457623&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280316600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.49" data-doi="10.1111/j.1365-294X.2011.05248.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bayesian inference of a historical bottleneck in a heavily exploited marine mammal.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3989</span>–<span class="cit-lpage">4008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2011.05248.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHoffman%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D20%26rft.issue%253D19%26rft.spage%253D3989%26rft.epage%253D4008%26rft.atitle%253DBayesian%2Binference%2Bof%2Ba%2Bhistorical%2Bbottleneck%2Bin%2Ba%2Bheavily%2Bexploited%2Bmarine%2Bmammal.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05248.x%26rft_id%253Dinfo%253Apmid%252F21895820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05248.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21895820&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295230000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.50" data-doi="10.1073/pnas.1318945111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rijks</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuiken</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-throughput sequencing reveals inbreeding depression in a natural population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">3775</span>–<span class="cit-lpage">3780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1318945111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1318945111%26rft_id%253Dinfo%253Apmid%252F24586051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzEwLzM3NzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.51" data-doi="10.1371/journal.pone.0068365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorne</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogden</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cross-amplification and validation of SNPs conserved over 44 million years between seals and dogs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0068365</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68365%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068365%26rft_id%253Dinfo%253Apmid%252F23874599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068365&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.52" data-doi="10.1111/j.1365-294X.2006.03053.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trathan</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic tagging reveals extreme site fidelity in territorial male Antarctic fur seals <em>Arctocephalus gazella</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3841</span>–<span class="cit-lpage">3847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2006.03053.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHoffman%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D3841%26rft.epage%253D3847%26rft.atitle%253DGenetic%2Btagging%2Breveals%2Bextreme%2Bsite%2Bfidelity%2Bin%2Bterritorial%2Bmale%2BAntarctic%2Bfur%2Bseals%2BArctocephalus%2Bgazella.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.03053.x%26rft_id%253Dinfo%253Apmid%252F17032279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.03053.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17032279&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241157400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.53" data-doi="10.1007/s00300-003-0587-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hucke-Gaete</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osman</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Examining natural population growth from near extinction: the case of the Antarctic fur seal at the South Shetlands, Antarctica.</span> <abbr class="cit-jnl-abbrev">Polar Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">304</span>–<span class="cit-lpage">311</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00300-003-0587-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPolar%2BBiol.%26rft.volume%253D27%26rft.spage%253D304%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00300-003-0587-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00300-003-0587-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220189000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.54" data-doi="10.1073/pnas.1518046113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inbreeding depression across the lifespan in a wild mammal population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">3585</span>–<span class="cit-lpage">3590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1518046113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518046113%26rft_id%253Dinfo%253Apmid%252F26979959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzEzLzM1ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.55" data-doi="10.1111/1755-0998.12502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Humble</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Barrio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trathan</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorne</span>  <span class="cit-name-given-names">M. A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A draft fur seal genome provides insights into factors affecting SNP validation and how to mitigate them.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">921</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D909%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12502%26rft_id%253Dinfo%253Apmid%252F26683564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26683564&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.56" data-doi="10.1111/j.1365-294X.2011.05076.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickering</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beraldi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping identifies the genetic basis of discrete and quantitative varation in sexual weaponry in a wild sheep population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2555</span>–<span class="cit-lpage">2566</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2011.05076.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DJohnston%26rft.auinit1%253DS.%2BE.%26rft.volume%253D20%26rft.issue%253D12%26rft.spage%253D2555%26rft.epage%253D2566%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bidentifies%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bdiscrete%2Band%2Bquantitative%2Bvariation%2Bin%2Bsexual%2Bweaponry%2Bin%2Ba%2Bwild%2Bsheep%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05076.x%26rft_id%253Dinfo%253Apmid%252F21651634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05076.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21651634&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291402600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.57" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">adegenet: a R package for the multivariate analysis of genetic markers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btn129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.58" data-doi="10.1038/s41559-017-0375-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Åkesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fountain</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagstad</span>  <span class="cit-name-given-names">Ø.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liberg</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic consequences of intensive inbreeding in an isolated wolf population.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-017-0375-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D2%26rft.spage%253D124%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-017-0375-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-017-0375-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.8.2709.59" data-doi="10.1111/1755-0998.12498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McFarlane</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Whole-genome resequencing of extreme phenotypes in collared flycatchers highlights the difficulty of detecting quantitative trait loci in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">741</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12498</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12498%26rft_id%253Dinfo%253Apmid%252F26649993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12498&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26649993&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.60" data-doi="10.1038/hdy.2015.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Measuring individual inbreeding in the age of genomics: marker-based measures are better than pedigrees.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.17%26rft_id%253Dinfo%253Apmid%252F26059970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059970&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.61" data-doi="10.1111/eva.12414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomics advances the study of inbreeding depression in the wild.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12414</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D9%26rft.spage%253D1205%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.62" data-doi="10.1111/1755-0998.12270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olason</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimation of linkage disequilibrium and interspecific gene flow in <em>Ficedula</em> flycatchers by a newly developed 50K single-nucleotide polymorphism array.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1248</span>–<span class="cit-lpage">1260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D1248%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12270%26rft_id%253Dinfo%253Apmid%252F24784959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24784959&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.63" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.64" data-doi="10.1016/S0169-5347(02)02489-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waller</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Inbreeding effects in wild populations.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0169-5347(02)02489-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D17%26rft.spage%253D230%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252802%252902489-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(02)02489-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000175024300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.65" data-doi="10.1101/gr.113985.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiełbasa</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Adaptive seeds tame genomic sequence comparison.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">493</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.113985.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113985.110%26rft_id%253Dinfo%253Apmid%252F21209072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvNDg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.66" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-other" id="cit-8.8.2709.67"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.68" data-doi="10.1016/j.cell.2014.03.054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenzen</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population genomics reveal recent speciation and rapid evolutionary adaptation in polar bears.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">794</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.03.054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D785%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.03.054%26rft_id%253Dinfo%253Apmid%252F24813606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.03.054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24813606&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335765500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.69" data-doi="10.1111/1755-0998.12327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malenfant</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Design of a 9K illumina BeadChip for polar bears (<em>Ursus maritimus</em>) from RAD and transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">600</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12327</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12327%26rft_id%253Dinfo%253Apmid%252F25187336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25187336&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.70" data-doi="10.1098/rspb.2002.2035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalton</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimating the prevalence of inbreeding from incomplete pedigrees.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1539</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2002.2035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DMarshall%26rft.auinit1%253DT.%2BC.%26rft.volume%253D269%26rft.issue%253D1500%26rft.spage%253D1533%26rft.epage%253D1539%26rft.atitle%253DEstimating%2Bthe%2Bprevalence%2Bof%2Binbreeding%2Bfrom%2Bincomplete%2Bpedigrees%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2002.2035%26rft_id%253Dinfo%253Apmid%252F12184822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjY5LzE1MDAvMTUzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.71" data-doi="10.1016/0169-5347(91)90139-O"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCauley</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic consequences of local population extinction and recolonization.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0169-5347(91)90139-O</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DMcCauley%26rft.auinit1%253DD.%2BE.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D8%26rft.atitle%253DGenetic%2Bconsequences%2Bof%2Blocal%2Bpopulation%2Bextinction%2Band%2Brecolonization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0169-5347%252891%252990139-O%26rft_id%253Dinfo%253Apmid%252F21232411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0169-5347(91)90139-O&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21232411&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991EX20900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.72" data-doi="10.1186/1471-2156-8-74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murdoch</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Whole genome linkage disequilibrium maps in cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-8-74</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DMcKay%26rft.auinit1%253DS.%2BD.%26rft.volume%253D8%26rft.spage%253D74%26rft.epage%253D74%26rft.atitle%253DWhole%2Bgenome%2Blinkage%2Bdisequilibrium%2Bmaps%2Bin%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-8-74%26rft_id%253Dinfo%253Apmid%252F17961247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-8-74&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17961247&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.73" data-doi="10.1111/j.1365-294x.2005.02571.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewald</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keim</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Habitat barriers limit gene flow and illuminate historical events in a wide-ranging carnivore, the American puma.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1977</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294x.2005.02571.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMcRae%26rft.auinit1%253DB.%2BH.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1965%26rft.epage%253D1977%26rft.atitle%253DHabitat%2Bbarriers%2Blimit%2Bgene%2Bflow%2Band%2Billuminate%2Bhistorical%2Bevents%2Bin%2Ba%2Bwide-ranging%2Bcarnivore%252C%2Bthe%2BAmerican%2Bpuma.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294x.2005.02571.x%26rft_id%253Dinfo%253Apmid%252F15910319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294x.2005.02571.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15910319&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229190300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.74" data-doi="10.1186/1471-2156-9-61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">E. K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linkage disequilibrium compared between five populations of domestic sheep.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-9-61</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DMeadows%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D9%26rft.spage%253D61%26rft.epage%253D61%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bcompared%2Bbetween%2Bfive%2Bpopulations%2Bof%2Bdomestic%2Bsheep.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-9-61%26rft_id%253Dinfo%253Apmid%252F18826649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-9-61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18826649&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.75" data-doi="10.1038/hdy.2013.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malenfant</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poissant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimating genome-wide heterozygosity: effects of demographic history and marker type.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.99</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D240%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.99%26rft_id%253Dinfo%253Apmid%252F24149650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24149650&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331626500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.76" data-doi="10.1111/j.1755-0998.2010.02918.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poissant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth cit-collab">International Sheep Genomics Consortium</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A genome-wide set of SNPs detects population substructure and long range linkage disequilibrium in wild sheep.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">322</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2010.02918.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D314%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02918.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02918.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.77" data-doi="10.1002/ece3.1612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poissant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malenfant</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogg</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Temporal dynamics of linkage disequilibrium in two populations of bighorn sheep.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3401</span>–<span class="cit-lpage">3412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.1612</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D5%26rft.spage%253D3401%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.1612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.1612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.78" data-doi="10.1101/128348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohr</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naguib</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisenfeld</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improved <em>de novo</em> genome assembly: synthetic long read sequencing combined with optical mapping produce a high quality mammalian genome at relatively low cost.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/128348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F128348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxMjgzNDh2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.79" data-doi="10.1016/j.tree.2004.01.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">SNPs in ecology, evolution and conservation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2004.01.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D19%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2004.01.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2004.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220842400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.80" data-doi="10.1038/hdy.2011.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanomtong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chromosomal rearrangements and karyotype evolution in carnivores revealed by chromosome painting.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D108%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.81" data-doi="10.1098/rspb.2016.2763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nietlisbach</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camenisch</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arcese</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pedigree-based inbreeding coefficient explains more variation in fitness than heterozygosity at 160 microsatellites in a wild bird population.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">20162763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2016.2763</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2016.2763%26rft_id%253Dinfo%253Apmid%252F28250184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjg0LzE4NTAvMjAxNjI3NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.82" data-doi="10.1111/mec.12234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugue</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinsohn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senn</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sturgeon conservation genomics: SNP discovery and validation using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3112</span>–<span class="cit-lpage">3123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.83" data-doi="10.1080/00291952808551483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olstad</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1928</span> <span class="cit-article-title">Trekk av Sydishavets dyreliv.</span> <abbr class="cit-jnl-abbrev">Norsk Geografisk Tidsskrift -. Nor. Geogr. Tidsskr.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00291952808551483</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNorsk%2BGeografisk%2BTidsskrift%2B-.%2BNor.%2BGeogr.%2BTidsskr.%26rft.volume%253D2%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00291952808551483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00291952808551483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.84" data-doi="10.1111/j.1755-0998.2010.02962.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osborne</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brauning</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development of a predicted physical map of microsatellite locus positions for pinnipeds, with wider applicability to the Carnivora.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">513</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2010.02962.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02962.x%26rft_id%253Dinfo%253Apmid%252F21481208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02962.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21481208&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welling</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambellant</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldsworthy</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Population status and breeding season chronology of Heard Island fur seals.</span> <abbr class="cit-jnl-abbrev">Polar Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPolar%2BBiol.%26rft.volume%253D26%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.86" data-doi="10.1098/rstb.1977.0072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Growth of a fur seal population.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.1977.0072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1977.0072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEwOiIyNzkvOTYzLzY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.87" data-doi="10.1038/nmeth.3454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pendleton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sebra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">A. W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ummat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franzen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembly and diploid architecture of an individual human genome via single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">780</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3454</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D780%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3454%26rft_id%253Dinfo%253Apmid%252F26121404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26121404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.88" data-doi="10.1002/ece3.2290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humble</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kröcker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Born blonde: a recessive loss-of-function mutation in the melanocortin 1 receptor is associated with cream coat coloration in Antarctic fur seals.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">5705</span>–<span class="cit-lpage">5717</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.2290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D5705%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.2290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.2290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.89" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double Digest RADseq: An inexpensive method for de novo snp discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0037135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.90" data-doi="10.1038/hdy.2013.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poelstra</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An extensive candidate gene approach to speciation: diversity, divergence and linkage disequilibrium in candidate pigmentation genes across the European crow hybrid zone.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.68</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D111%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.68%26rft_id%253Dinfo%253Apmid%252F23881172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23881172&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327178100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.91" data-doi="10.1038/srep41457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pootakham</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonthirod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naktang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruang-Areerate</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoocha</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title"><em>De novo</em> hybrid assembly of the rubber tree genome reveals evidence of paleotetraploidy in <em>Hevea</em> species.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">41457</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep41457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D41457%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep41457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep41457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.92" data-doi="10.1101/201178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruano-Rubio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Scaling accurate genetic variant discovery to tens of thousands of samples.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/201178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F201178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyMDExNzh2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.93" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.94" data-doi="10.1111/1755-0998.12512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puechmaille</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The program structure does not reliably recover the correct population structure when sampling is uneven: subsampling and new estimators alleviate the problem.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">627</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D608%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.95" data-doi="10.1186/1471-2164-15-275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rašić</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipović</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide SNPs lead to strong signals of geographic structure and relatedness patterns in the major arbovirus vector, <em>Aedes aegypti</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-275%26rft_id%253Dinfo%253Apmid%252F24726019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24726019&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.96" data-doi="10.1007/s10530-007-9196-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reitzel</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finnerty</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Global population genetic structure of the starlet anemone <em>Nematostella vectensis</em>: multiple introductions and implications for conservation policy.</span> <abbr class="cit-jnl-abbrev">Biol. Invasions</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1197</span>–<span class="cit-lpage">1213</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10530-007-9196-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BInvasions%26rft.volume%253D10%26rft.spage%253D1197%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10530-007-9196-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10530-007-9196-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.97" data-doi="10.1073/pnas.201394398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitt</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">11479</span>–<span class="cit-lpage">11484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.201394398</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.201394398%26rft_id%253Dinfo%253Apmid%252F11562485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjAvMTE0NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.98" data-doi="10.1111/j.1755-0998.2012.03156.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Ramilo</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The effect of close relatives on unsupervised Bayesian clustering algorithms in population genetic structure analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">884</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2012.03156.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D12%26rft.spage%253D873%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2012.03156.x%26rft_id%253Dinfo%253Apmid%252F22639868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2012.03156.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22639868&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.99" data-doi="10.1186/gb-2013-14-5-r51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russ</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterizing and measuring bias in sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-5-r51</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR51%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-5-r51%26rft_id%253Dinfo%253Apmid%252F23718773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-5-r51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23718773&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2709.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Molecular cloning: a laboratory manual</span>, <span class="cit-publ-name">Cold Spring Harbour Laboratory Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.101" data-doi="10.1111/j.1755-0998.2010.02979.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single-nucleotide polymorphism (SNP) discovery and applications of SNP genotyping in nonmodel organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2010.02979.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02979.x%26rft_id%253Dinfo%253Apmid%252F21429158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02979.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429158&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">A. B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peart</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tusso</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maayan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brelsford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bioinformatic processing of RAD-seq data dramatically impacts downstream population genetic inference.</span> <abbr class="cit-jnl-abbrev">Methods. Ecol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">907</span>–<span class="cit-lpage">917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods.%2BEcol.%2BEvol.%26rft.volume%253D8%26rft.spage%253D907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.103" data-doi="10.1038/sj.hdy.6800485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veenvliet</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Understanding the relationship between the inbreeding coefficient and multilocus heterozygosity: theoretical expectations and empirical data.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800485</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSlate%26rft.auinit1%253DJ.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D255%26rft.epage%253D265%26rft.atitle%253DUnderstanding%2Bthe%2Brelationship%2Bbetween%2Bthe%2Binbreeding%2Bcoefficient%2Band%2Bmultilocus%2Bheterozygosity%253A%2Btheoretical%2Bexpectations%2Band%2Bempirical%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800485%26rft_id%253Dinfo%253Apmid%252F15254488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15254488&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223526900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.104" data-doi="10.1038/nrg2361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linkage disequilibrium–understanding the evolutionary past and mapping the medical future.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2361</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D477%26rft.epage%253D485%26rft.atitle%253DLinkage%2Bdisequilibrium--understanding%2Bthe%2Bevolutionary%2Bpast%2Band%2Bmapping%2Bthe%2Bmedical%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2361%26rft_id%253Dinfo%253Apmid%252F18427557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18427557&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255953500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.105" data-doi="10.1016/j.tree.2010.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feulner</span>  <span class="cit-name-given-names">P. G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smadja</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation genomics: the next generation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2010.09.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DStapley%26rft.auinit1%253DJ.%26rft.volume%253D25%26rft.issue%253D12%26rft.spage%253D705%26rft.epage%253D712%26rft.atitle%253DAdaptation%2Bgenomics%253A%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2010.09.002%26rft_id%253Dinfo%253Apmid%252F20952088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2010.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20952088&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284795800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.106" data-doi="10.1073/pnas.1506076112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caspers</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giannakara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chemical fingerprints encode mother-offspring similarity, colony membership, relatedness, and genetic quality in fur seals.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E5005</span>–<span class="cit-lpage">E5012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1506076112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1506076112%26rft_id%253Dinfo%253Apmid%252F26261311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzM2L0U1MDA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC84LzI3MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.107" data-doi="10.1111/2041-210X.12588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humble</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">inbreedR: an R package for the analysis of inbreeding based on genetic markers.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.12588</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D7%26rft.spage%253D1331%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.108" data-doi="10.1101/293894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humble</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acevedo-Whitehouse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chilvers</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickerson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Recent demographic histories and genetic diversity across pinnipeds are shaped by anthropogenic interactions and mediated by ecology and life-history.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/293894</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F293894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIyOTM4OTR2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOC8yNzA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.109" data-doi="10.1111/j.1558-5646.2010.00966.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szulkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heterozygosity-fitness correlations: a time for reappraisal.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1202</span>–<span class="cit-lpage">1217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2010.00966.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DSzulkin%26rft.auinit1%253DM.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D1202%26rft.epage%253D1217%26rft.atitle%253DHeterozygosity-fitness%2Bcorrelations%253A%2Ba%2Btime%2Bfor%2Breappraisal.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.00966.x%26rft_id%253Dinfo%253Apmid%252F20148954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.00966.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20148954&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277096400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.110" data-doi="10.1111/jeb.12109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamieson</span>  <span class="cit-name-given-names">I. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular and pedigree measures of relatedness provide similar estimates of inbreeding depression in a bottlenecked population.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">899</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D889%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12109%26rft_id%253Dinfo%253Apmid%252F23496652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23496652&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-other" id="cit-8.8.2709.111"><div class="cit-metadata unstructured">Untergasser, A., I. Cutcutache, T. Koressaar, J. Ye, B. C. Faircloth <em>et al.</em>, 2012 Primer3–new capabilities and interfaces. 40: e115.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.112" data-doi="10.1098/rsos.160548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vendrami</span>  <span class="cit-name-given-names">D. L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telesca</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigand</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">RAD sequencing resolves fine-scale population structure in a benthic invertebrate: implications for understanding phenotypic plasticity.</span> <abbr class="cit-jnl-abbrev">R. Soc. Open Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">160548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsos.160548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR.%2BSoc.%2BOpen%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsos.160548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveW9wZW5zY2kiO3M6NToicmVzaWQiO3M6MTA6IjQvMi8xNjA1NDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzgvMjcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.113" data-doi="10.1038/ncomms13195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vijay</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bossu</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poelstra</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissensteiner</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolution of heterogeneous genome differentiation across multiple contact zones in a crow species complex.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms13195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D13195%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms13195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms13195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.114" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.115" data-doi="10.1111/1755-0998.12650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The computer program structure for assigning individuals to populations: easy to use but easier to misuse.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">981</span>–<span class="cit-lpage">990</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D981%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-book" id="cit-8.8.2709.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weddell</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1825</span> <span class="cit-source">A voyage towards the south pole performed in the years 1822–1824. Longman</span>, <span class="cit-publ-name">Rees, Orme, Brown and Green, London</span>, <span class="cit-publ-loc">Hurst</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference  in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.117" data-doi="10.1017/S0016672300013446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Mixed self and random mating at two loci.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300013446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D247%26rft.epage%253D262%26rft.atitle%253DMixed%2Bself%2Band%2Brandom%2Bmating%2Bat%2Btwo%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300013446%26rft_id%253Dinfo%253Apmid%252F4731639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300013446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4731639&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.118" data-doi="10.1371/journal.pone.0042649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willing</span>  <span class="cit-name-given-names">E.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreyer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oosterhout</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimates of genetic differentiation measured by F<sub>ST</sub> do not necessarily require large sample sizes when using many SNP markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e42649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0042649</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWilling%26rft.auinit1%253DE.%2BM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De42649%26rft.epage%253De42649%26rft.atitle%253DEstimates%2Bof%2Bgenetic%2Bdifferentiation%2Bmeasured%2Bby%2BF%2528ST%2529%2Bdo%2Bnot%2Bnecessarily%2Brequire%2Blarge%2Bsample%2Bsizes%2Bwhen%2Busing%2Bmany%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0042649%26rft_id%253Dinfo%253Apmid%252F22905157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0042649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22905157&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.119" data-doi="10.1046/j.1365-294x.2000.00856.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wynen</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldsworthy</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bester</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Postsealing genetic variation and population structure of two species of fur seal (<em>Arctocephalus gazella</em> and <em>A. tropicalis</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-294x.2000.00856.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWynen%26rft.auinit1%253DL.%2BP.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D314%26rft.atitle%253DPostsealing%2Bgenetic%2Bvariation%2Band%2Bpopulation%2Bstructure%2Bof%2Btwo%2Bspecies%2Bof%2Bfur%2Bseal%2B%2528Arctocephalus%2Bgazella%2Band%2BA.%2Btropicalis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.2000.00856.x%26rft_id%253Dinfo%253Apmid%252F10736028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.2000.00856.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10736028&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085800500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.120" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: A Tool for Genome-wide Complex Trait Analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.121" data-doi="10.1111/mec.14172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Younger</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clucas</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The challenges of detecting subtle population structure and its importance for the conservation of emperor penguins.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3883</span>–<span class="cit-lpage">3897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D3883%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-8.8.2709.122" data-doi="10.1186/1471-2105-14-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meltzer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RCircos: an R package for Circos 2D track plots.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-14-244</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-244%26rft_id%253Dinfo%253Apmid%252F23937229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23937229&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F8%2F2709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/8/2709.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/8/2697" title="Targeted Chromosomal Rearrangements via Combinatorial Use of CRISPR/Cas9 and Cre/LoxP Technologies in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/8/2723" title="Fission Yeast CENP-C (Cnp3) Plays a Role in Restricting the Site of CENP-A Accumulation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/8"><span>Volume 8 Issue 8, August 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/8.cover-source.jpg?itok=mN_lceRD" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/8/2709" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-D6_tVXwatyE65IlsE15KsDXtgeJuJxlwchkJs9Qq0mg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/8/2709" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/8/2709" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/8/2709" class="active" data-icon-position="" data-hide-link-title="0">RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490463" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-k6rvCBdycQ-cx2PwZkTVz1MOSGF59Stz1l4GhxqOOJM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2709.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/8/2709" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-P08O1lYLcB3sE1M_9FJJsqRzEo088hDkLi0F5HsJM44" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/8/2709" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/8/2709.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490463" id="node490463--2" data-pisa="ggg;8/8/2709" data-pisa-master="ggg;g3.118.200171" data-apath="/ggg/8/8/2709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6841-1269" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emily Humble</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2840-7019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kanchon K. Dasmahapatra</span>, <span class="highwire-citation-author" data-delta="2">Alvaro Martinez-Barrio</span>, <span class="highwire-citation-author" data-delta="3">Inês Gregório</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2115-0150" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaume Forcada</span>, <span class="highwire-citation-author" data-delta="5">Ann-Christin Polikeit</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4988-9085" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simon D. Goldsworthy</span>, <span class="highwire-citation-author" data-delta="7">Michael E. Goebel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-9052-1998" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jörn Kalinowski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-2958-5183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jochen B. W. Wolf</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-5895-8949" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph I. Hoffman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2709-2722;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200171 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490463--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emily Humble </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BHumble%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Humble%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmily%2BHumble%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6841-1269" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emily Humble</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emily.lhumble@gmail.com" class="" data-icon-position="" data-hide-link-title="0">emily.lhumble@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kanchon K. Dasmahapatra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of York, Heslington, York, YO10 5DD, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanchon%2BK.%2BDasmahapatra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dasmahapatra%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKanchon%2BK.%2BDasmahapatra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2840-7019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kanchon K. Dasmahapatra</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alvaro Martinez-Barrio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Science for Life Laboratories and Department of Cell and Molecular Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alvaro%2BMartinez-Barrio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez-Barrio%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlvaro%2BMartinez-Barrio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Inês Gregório </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=In%C3%AAs%2BGreg%C3%B3rio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greg%C3%B3rio%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIn%25C3%25AAs%2BGreg%25C3%25B3rio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaume Forcada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaume%2BForcada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forcada%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaume%2BForcada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2115-0150" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaume Forcada</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ann-Christin Polikeit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann-Christin%2BPolikeit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polikeit%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn-Christin%2BPolikeit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Simon D. Goldsworthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>South Australian Research and Development Institute, 2 Hamra Avenue, West Beach, South Australia, 5024, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BD.%2BGoldsworthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goldsworthy%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BD.%2BGoldsworthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4988-9085" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simon D. Goldsworthy</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michael E. Goebel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Antarctic Ecosystem Research Division, SWFSC, NMFS, NOAA, La Jolla, California 92037-1023 USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BGoebel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goebel%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BE.%2BGoebel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jörn Kalinowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centrum für Biotechnologie, CeBiTec, University of Bielefeld, Universitätsstr. 27, 33615 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BKalinowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalinowski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rn%2BKalinowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9052-1998" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jörn Kalinowski</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jochen B. W. Wolf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Division of Evolutionary Biology, Faculty of Biology, Ludwig-Maximilians-Universität Munich, Planegg-Martinstried, Germany</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Science for Life Laboratories and Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BB.%2BW.%2BWolf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolf%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BB.%2BW.%2BWolf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2958-5183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jochen B. W. Wolf</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joseph I. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BI.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20JI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BI.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5895-8949" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph I. Hoffman</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/8/2709" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/8/2709&amp;title=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/8/2709&amp;title=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/8/2709&amp;title=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/8/2709&amp;text=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/8/2709&amp;title=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/8/2709&amp;t=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/8/2709&amp;title=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/8/2709&amp;title=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490463/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490463" id="node490463--3" data-pisa="ggg;8/8/2709" data-pisa-master="ggg;g3.118.200171" data-apath="/ggg/8/8/2709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RAD Sequencing and a Hybrid Antarctic Fur Seal Genome Assembly Reveal Rapidly Decaying Linkage Disequilibrium, Global Population Structure and Evidence for Inbreeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6841-1269" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emily Humble</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2840-7019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kanchon K. Dasmahapatra</span>, <span class="highwire-citation-author" data-delta="2">Alvaro Martinez-Barrio</span>, <span class="highwire-citation-author" data-delta="3">Inês Gregório</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2115-0150" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaume Forcada</span>, <span class="highwire-citation-author" data-delta="5">Ann-Christin Polikeit</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4988-9085" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Simon D. Goldsworthy</span>, <span class="highwire-citation-author" data-delta="7">Michael E. Goebel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-9052-1998" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jörn Kalinowski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-2958-5183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jochen B. W. Wolf</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-5895-8949" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph I. Hoffman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2709-2722;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200171 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490463--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emily Humble </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BHumble%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Humble%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmily%2BHumble%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6841-1269" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emily Humble</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emily.lhumble@gmail.com" class="" data-icon-position="" data-hide-link-title="0">emily.lhumble@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kanchon K. Dasmahapatra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of York, Heslington, York, YO10 5DD, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanchon%2BK.%2BDasmahapatra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dasmahapatra%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKanchon%2BK.%2BDasmahapatra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2840-7019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kanchon K. Dasmahapatra</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alvaro Martinez-Barrio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Science for Life Laboratories and Department of Cell and Molecular Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alvaro%2BMartinez-Barrio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez-Barrio%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlvaro%2BMartinez-Barrio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Inês Gregório </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=In%C3%AAs%2BGreg%C3%B3rio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greg%C3%B3rio%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIn%25C3%25AAs%2BGreg%25C3%25B3rio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaume Forcada </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaume%2BForcada%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forcada%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaume%2BForcada%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2115-0150" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaume Forcada</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ann-Christin Polikeit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann-Christin%2BPolikeit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polikeit%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn-Christin%2BPolikeit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Simon D. Goldsworthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>South Australian Research and Development Institute, 2 Hamra Avenue, West Beach, South Australia, 5024, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BD.%2BGoldsworthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goldsworthy%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASimon%2BD.%2BGoldsworthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4988-9085" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Simon D. Goldsworthy</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michael E. Goebel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Antarctic Ecosystem Research Division, SWFSC, NMFS, NOAA, La Jolla, California 92037-1023 USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BGoebel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goebel%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BE.%2BGoebel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jörn Kalinowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Centrum für Biotechnologie, CeBiTec, University of Bielefeld, Universitätsstr. 27, 33615 Bielefeld, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BKalinowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalinowski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rn%2BKalinowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9052-1998" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jörn Kalinowski</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jochen B. W. Wolf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Division of Evolutionary Biology, Faculty of Biology, Ludwig-Maximilians-Universität Munich, Planegg-Martinstried, Germany</div><div class='author-affiliation'><span class='nlm-sup'>***</span>Science for Life Laboratories and Department of Evolutionary Biology, Uppsala University, Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BB.%2BW.%2BWolf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolf%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BB.%2BW.%2BWolf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2958-5183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jochen B. W. Wolf</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joseph I. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Behaviour, University of Bielefeld, Postfach 100131, 33501 Bielefeld, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>British Antarctic Survey, High Cross, Madingley Road, Cambridge, CB3 OET, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BI.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20JI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BI.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5895-8949" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph I. Hoffman</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2709&amp;rft.epage=2722&amp;rft.atitle=RAD%20Sequencing%20and%20a%20Hybrid%20Antarctic%20Fur%20Seal%20Genome%20Assembly%20Reveal%20Rapidly%20Decaying%20Linkage%20Disequilibrium%2C%20Global%20Population%20Structure%20and%20Evidence%20for%20Inbreeding&amp;rft.volume=8&amp;rft.issue=8&amp;rft.date=2018-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Humble%2C+Emily&amp;rft.au=Dasmahapatra%2C+Kanchon+K.&amp;rft.au=Martinez-Barrio%2C+Alvaro&amp;rft.au=Greg%C3%B3rio%2C+In%C3%AAs&amp;rft.au=Forcada%2C+Jaume&amp;rft.au=Polikeit%2C+Ann-Christin&amp;rft.au=Goldsworthy%2C+Simon+D.&amp;rft.au=Goebel%2C+Michael+E.&amp;rft.au=Kalinowski%2C+J%C3%B6rn&amp;rft.au=Wolf%2C+Jochen+B.+W.&amp;rft.au=Hoffman%2C+Joseph+I."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490463/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490463/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490463/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490463/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490463/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490463/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490463/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490463/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490463/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490463/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490463/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490463/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/8/2709" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/8/2709.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/8/2709.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/8/2709.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/8/2709.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
