<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/477" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/477.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/477.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/477.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/4/477/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/477.atom" />
<meta name="HW.pisa" content="ggg;5/4/477" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation" />
<meta name="DC.Identifier" content="10.1534/g3.115.016725" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Varona et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Epigenetics has become one of the major areas of biological research. However, the degree of phenotypic variability that is explained by epigenetic processes still remains unclear. From a quantitative genetics perspective, the estimation of variance components is achieved by means of the information provided by the resemblance between relatives. In a previous study, this resemblance was described as a function of the epigenetic variance component and a reset coefficient that indicates the rate of dissipation of epigenetic marks across generations. Given these assumptions, we propose a Bayesian mixed model methodology that allows the estimation of epigenetic variance from a genealogical and phenotypic database. The methodology is based on the development of a T matrix of epigenetic relationships that depends on the reset coefficient. In addition, we present a simple procedure for the calculation of the inverse of this matrix ( T−1 ) and a Gibbs sampler algorithm that obtains posterior estimates of all the unknowns in the model. The new procedure was used with two simulated data sets and with a beef cattle database. In the simulated populations, the results of the analysis provided marginal posterior distributions that included the population parameters in the regions of highest posterior density. In the case of the beef cattle dataset, the posterior estimate of transgenerational epigenetic variability was very low and a model comparison test indicated that a model that did not included it was the most plausible." />
<meta name="DC.Contributor" content="Luis Varona" />
<meta name="DC.Contributor" content="Sebastián Munilla" />
<meta name="DC.Contributor" content="Elena Flavia Mouresan" />
<meta name="DC.Contributor" content="Aldemar González-Rodríguez" />
<meta name="DC.Contributor" content="Carlos Moreno" />
<meta name="DC.Contributor" content="Juan Altarriba" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Epigenetics has become one of the major areas of biological research. However, the degree of phenotypic variability that is explained by epigenetic processes still remains unclear. From a quantitative genetics perspective, the estimation of variance components is achieved by means of the information provided by the resemblance between relatives. In a previous study, this resemblance was described as a function of the epigenetic variance component and a reset coefficient that indicates the rate of dissipation of epigenetic marks across generations. Given these assumptions, we propose a Bayesian mixed model methodology that allows the estimation of epigenetic variance from a genealogical and phenotypic database. The methodology is based on the development of a &lt;b&gt;T&lt;/b&gt; matrix of epigenetic relationships that depends on the reset coefficient. In addition, we present a simple procedure for the calculation of the inverse of this matrix (&lt;b&gt;T&lt;sup&gt;−1&lt;/sup&gt;&lt;/b&gt;) and a Gibbs sampler algorithm that obtains posterior estimates of all the unknowns in the model. The new procedure was used with two simulated data sets and with a beef cattle database. In the simulated populations, the results of the analysis provided marginal posterior distributions that included the population parameters in the regions of highest posterior density. In the case of the beef cattle dataset, the posterior estimate of transgenerational epigenetic variability was very low and a model comparison test indicated that a model that did not included it was the most plausible.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/477" />
<meta name="citation_id" content="5/4/477" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/477" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/477.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/477.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/477.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.016725" />
<meta name="citation_pmid" content="25617408" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="477" />
<meta name="citation_lastpage" content="485" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Luis Varona" />
<meta name="citation_author_institution" content="Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain" />
<meta name="citation_author_institution" content="Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain" />
<meta name="citation_author_email" content="lvarona@unizar.es" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6256-5478" />
<meta name="citation_author" content="Sebastián Munilla" />
<meta name="citation_author_institution" content="Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain" />
<meta name="citation_author_institution" content="Departamento de Producción Animal, Facultad de Agronomía, Universidad de Buenos Aires, 1417, Ciudad Autónoma de Buenos Aires, Argentina" />
<meta name="citation_author" content="Elena Flavia Mouresan" />
<meta name="citation_author_institution" content="Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain" />
<meta name="citation_author" content="Aldemar González-Rodríguez" />
<meta name="citation_author_institution" content="Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain" />
<meta name="citation_author" content="Carlos Moreno" />
<meta name="citation_author_institution" content="Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain" />
<meta name="citation_author_institution" content="Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain" />
<meta name="citation_author" content="Juan Altarriba" />
<meta name="citation_author_institution" content="Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain" />
<meta name="citation_author_institution" content="Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain" />
<meta name="citation_reference" content="citation_title=Animal Traditions: Behavioural Inheritance in Evolution;citation_author=E. Avital;citation_author=E. Jablonka;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=D. P. Barlow;citation_title=Competition--a common motif for the imprinting mechanism?;citation_pages=6899-6905;citation_volume=16;citation_year=1997;citation_issue=23;citation_pmid=9384569;citation_doi=10.1093/emboj/16.23.6899" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Eccleston;citation_author=N. DeWitt;citation_author=C. Gunter;citation_author=B. Marte;citation_author=D. Nath;citation_title=Introduction epigenetics.;citation_pages=395-400;citation_volume=447;citation_year=2007;citation_doi=10.1038/447395a" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=M. Esteller;citation_title=Epigenetics in Cancer;citation_pages=1148-1159;citation_volume=358;citation_year=2008;citation_issue=11;citation_pmid=18337604;citation_doi=10.1056/NEJMra072067" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mckay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. P. Feinberg;citation_title=Phenotypic plasticity and the epigenetics of human disease.;citation_pages=433-440;citation_volume=447;citation_year=2007;citation_issue=7143;citation_pmid=17522677;citation_doi=10.1038/nature05919" />
<meta name="citation_reference" content="citation_title=Markov Chain Monte Carlo in Practice;citation_title=Model determination using sampling-based methods;citation_author=A. E. Gelfand;citation_pages=145-161;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=A. E. Gelfand;citation_author=A. F. M. Smith;citation_title=Sampling-based approaches to calculating marginal densities.;citation_pages=398-409;citation_volume=85;citation_year=1990;citation_doi=10.2307/2289776" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_title=Bayesian Methods in Animal Breeding Theory;citation_pages=217-244;citation_volume=63;citation_year=1986;citation_issue=1;citation_doi=10.2527/jas1986.631217x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=U. Grossniklaus;citation_author=W. G. Kelly;citation_author=A. C. Ferguson-Smith;citation_author=M. Pembrey;citation_author=S. Lindquist;citation_title=Transgenerational epigenetic inheritance: how important is it?;citation_pages=228-235;citation_volume=14;citation_year=2013;citation_pmid=23416892;citation_doi=10.1038/nrg3435" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=E. Heard;citation_author=R. A. Martienssen;citation_title=Transgenerational epigenetic inheritance: myths and mechanisms.;citation_pages=95-109;citation_volume=157;citation_year=2014;citation_pmid=24679529;citation_doi=10.1016/j.cell.2014.02.045" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_title=A simple method for computing the inverse of a numerator relationship matrix used in the prediction of breeding values.;citation_pages=69-83;citation_volume=32;citation_year=1976;citation_doi=10.2307/2529339" />
<meta name="citation_reference" content="citation_title=Application of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=I. R. Henderson;citation_author=S. E. Jacobsen;citation_title=Epigenetic inheritance in plants.;citation_pages=418-424;citation_volume=447;citation_year=2007;citation_issue=7143;citation_pmid=17522675;citation_doi=10.1038/nature05917" />
<meta name="citation_reference" content="citation_journal_title=The Quarterly Review of Biology;citation_journal_abbrev=The Quarterly Review of Biology;citation_author=E. Jablonka;citation_author=G. Raz;citation_title=Transgenerational epigenetic inheritance: prevalence, mechanisms, and implications for the study of heredity and evolution.;citation_pages=131-176;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19606595;citation_doi=10.1086/598822" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=J. Jamrozik;citation_author=S. P. Miller;citation_title=Genetic evaluation of calving ease in Canadian Simmentals using birth weight and gestation length as correlated traits.;citation_pages=42-49;citation_volume=162;citation_year=2014;citation_doi=10.1016/j.livsci.2014.01.027" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Law;citation_author=S. E. Jacobsen;citation_title=Establishing, maintaining and modifying DNA methylation patterns in plants and animals.;citation_pages=204-220;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20142834;citation_doi=10.1038/nrg2719" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. Mathew;citation_author=A. M. Bauer;citation_author=P. Koistinen;citation_author=T. C. Reetz;citation_author=J. Léon;citation_title=Bayesian adaptive Markov chain Monte Carlo estimation of genetic parameters.;citation_pages=235-245;citation_volume=109;citation_year=2012;citation_issue=4;citation_pmid=22805656;citation_doi=10.1038/hdy.2012.35" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=K. Meyer;citation_title=Variance components due to direct and maternal effects for growth traits of Australian beef cattle.;citation_pages=143-177;citation_volume=11;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. D. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=R. L. Quass;citation_title=Computing the diagonal elements of the inverse of a large numerator relationship matrix.;citation_pages=949-953;citation_volume=32;citation_year=1976;citation_doi=10.2307/2529279" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=R. Quintanilla;citation_author=L. Varona;citation_author=M. R. Pujol;citation_author=J. Piedrafita;citation_title=Maternal animal model with correlation between maternal environmental effects of related dams;citation_pages=2904-2917;citation_volume=77;citation_year=1999;citation_issue=11;citation_pmid=10568458" />
<meta name="citation_reference" content="citation_title=Bayesian Statistics IV;citation_title=How many iterations in the Gibbs sampler;citation_author=A. E. Raftery;citation_author=S. M. Lewis;citation_pages=763-774;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. Reik;citation_title=Stability and flexibility of epigenetic gene regulation in mammalian development.;citation_pages=425-432;citation_volume=447;citation_year=2007;citation_issue=7143;citation_pmid=17522676;citation_doi=10.1038/nature05918" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=W. Reik;citation_author=J. Walker;citation_title=Imprinting mechanisms in mammals.;citation_pages=154-164;citation_volume=8;citation_year=1998;citation_issue=2;citation_pmid=9610405;citation_doi=10.1016/S0959-437X(98)80136-6" />
<meta name="citation_reference" content="citation_title=Not by Genes Alone: How Culture Transformed Human Evolution;citation_author=P. J. Richerson;citation_author=R. Boyd;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. T. Rodríguez-Ramilo;citation_author=L. A. García-Cortés;citation_author=O. González-Recio;citation_title=Combining genomic and genealogical information in a reproducing kernel Hilbert spaces regression model for genome-enabled predictions in dairy cattle.;citation_pages=e93424;citation_volume=9;citation_year=2014;citation_pmid=24671175;citation_doi=10.1371/journal.pone.0093424" />
<meta name="citation_reference" content="citation_title=Razas Ganaderas Españolas Bovinas. MAPA-FEAGAS;citation_author=A. Sánchez;citation_author=J. Ambrona;citation_author=L. Sánchez;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. J. Schmitz;citation_title=The Secret Garden--Epigenetic Alleles Underlie Complex Traits;citation_pages=1082-1083;citation_volume=343;citation_year=2014;citation_issue=6175;citation_pmid=24604184;citation_doi=10.1126/science.1251864" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. Tal;citation_author=E. Kisdi;citation_author=E. Jablonka;citation_title=Epigenetic contribution to covariance between relatives.;citation_volume=184;citation_year=2010;citation_pmid=20100941;citation_doi=10.1534/genetics.109.112466" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Varona;citation_author=D. Sorensen;citation_title=Joint analysis of binomial and continuous traits with recursive model: a case study using mortality and litter size in pigs.;citation_volume=196;citation_year=2014;citation_pmid=24414548;citation_doi=10.1534/genetics.113.159475" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=L. Varona;citation_author=I. Misztal;citation_author=J. K. Bertrand;citation_title=Threshold-linear versus linear-linear analysis of birth weight and calving ease using an animal model: I. Variance component estimation;citation_pages=1994-2002;citation_volume=77;citation_year=1999;citation_issue=8;citation_pmid=10461973" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=L. Varona;citation_author=N. Ibañez-Escriche;citation_author=R. Quintanilla;citation_author=J. L. Noguera;citation_author=J. Casellas;citation_title=Bayesian analysis of quantitative traits using skewed distributions.;citation_pages=179-190;citation_volume=90;citation_year=2008;citation_issue=2;citation_pmid=18426621" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. S. Wang;citation_author=J. J. Rutledge;citation_author=D. Gianola;citation_title=Bayesian analysis of mixed linear models via Gibbs sampling with an application to litter size in Iberian pigs.;citation_pages=91-115;citation_volume=26;citation_year=1994;citation_doi=10.1051/gse:19940202" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=N. A. Youngson;citation_author=E. Whitelaw;citation_title=Transgenerational epigenetic effects.;citation_pages=233-257;citation_volume=9;citation_year=2008;citation_pmid=18767965;citation_doi=10.1146/annurev.genom.9.081307.164445" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Epigenetics has become one of the major areas of biological research. However, the degree of phenotypic variability that is explained by epigenetic processes still remains unclear. From a quantitative genetics perspective, the estimation of variance components is achieved by means of the information provided by the resemblance between relatives. In a previous study, this resemblance was described as a function of the epigenetic variance component and a reset coefficient that indicates the rate of dissipation of epigenetic marks across generations. Given these assumptions, we propose a Bayesian mixed model methodology that allows the estimation of epigenetic variance from a genealogical and phenotypic database. The methodology is based on the development of a T matrix of epigenetic relationships that depends on the reset coefficient. In addition, we present a simple procedure for the calculation of the inverse of this matrix ( T−1 ) and a Gibbs sampler algorithm that obtains posterior estimates of all the unknowns in the model. The new procedure was used with two simulated data sets and with a beef cattle database. In the simulated populations, the results of the analysis provided marginal posterior distributions that included the population parameters in the regions of highest posterior density. In the case of the beef cattle dataset, the posterior estimate of transgenerational epigenetic variability was very low and a model comparison test indicated that a model that did not included it was the most plausible." />
<meta name="og-title" property="og:title" content="A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/477" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Epigenetics has become one of the major areas of biological research. However, the degree of phenotypic variability that is explained by epigenetic processes still remains unclear. From a quantitative genetics perspective, the estimation of variance components is achieved by means of the information provided by the resemblance between relatives. In a previous study, this resemblance was described as a function of the epigenetic variance component and a reset coefficient that indicates the rate of dissipation of epigenetic marks across generations. Given these assumptions, we propose a Bayesian mixed model methodology that allows the estimation of epigenetic variance from a genealogical and phenotypic database. The methodology is based on the development of a T matrix of epigenetic relationships that depends on the reset coefficient. In addition, we present a simple procedure for the calculation of the inverse of this matrix ( T−1 ) and a Gibbs sampler algorithm that obtains posterior estimates of all the unknowns in the model. The new procedure was used with two simulated data sets and with a beef cattle database. In the simulated populations, the results of the analysis provided marginal posterior distributions that included the population parameters in the regions of highest posterior density. In the case of the beef cattle dataset, the posterior estimate of transgenerational epigenetic variability was very low and a model comparison test indicated that a model that did not included it was the most plausible." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"c7-lcmNuz5tFzHLcuvot2G1HmUpmWf2e51p2dIAfRew","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459806","apath":"\/ggg\/5\/4\/477.atom","pisa":"ggg;5\/4\/477","ac":{"\/ggg\/5\/4\/477.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/477.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/477"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/477"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/477.abstract":true,"\/content\/5\/4\/477":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225AXn0\\\\\\\/1i2gXZzVowWKZHU6mUrqhs2a4S847if3RuKDR133nZ6dzKcJYJJzGTWbtFQksoD3XCcD1IfES5NIwURsfoXz5tpz88c\\\\\\\/FWIHxeNU3OlETxrHW\\\\\\\/0\\\\\\\/25s6Ax7a+RVXGeodidlToG+2vBJRVLh3LHxiidlVXmrlFEgNTPeBp5yGLZNNQKv5SY\\\\\\\/KX0Qn4xJsjj+FWJ5AmqV6puUjK\\\\\\\/Tas9PqmrzRg5tpH\\\\\\\/qNFn1KfO3a\\\\\\\/zLizMr4pVIlYtXPjpfmQznW4f09DAdd\\\\\\\/+JPfv5hgxgj5H4uBW1W+leyxUNfK7\\\\\\\/DC04FEkqIDy85VCSaAnxRgPFkyg4PRG5KI3Z+LFCUyeYvYqHgiMGpfqEYJB0jfa+wt\\\\\\\/WVYrPLnEMQ95lVIJzzeqvfmRn5XGjPcYzHw2P1zRBc+ACt1OeZd7b1aeze2kyCYaIKThKMW\\\\\\\/\\\\\\\/5ajeVqP6Qp6Y8SLPchSOuRx0afYNR8ULW\\\\\\\/lc0Z8XXqT7xmpS57ieFMQhOOoDSEWwZqDzfCADOj9dUrJTTjI8GTQv+WIXSZ\\\\\\\/OMITs9IIMGTrLbTxktCJpaPaRdYz4gPGS65hgt1Kv53oZxPTyDFLz\\\\\\\/3j+W88JouZIVRvZUbG6y0+oO8rJDktpuj4eTltsjatsk1vK16dwWBRedsCP2JZDwmsALJjIZxRbElRMaVowOI2I7v7u+mxV\\\\\\\/RJgB2rtFLMwvivYWiT10kFpfMcEVK\\\\\\\/FFT3WLS+\\\\\\\/KX9tOnTdZTJC+V\\\\\\\/0+mcLbtInbmgVj55CfGgcEE8+dot5\\\\\\\/bzecM3eb4cRIAw6VSvt\\\\\\\/U63LdjBx\\\\\\\/qgoI2fSgmwinb2SPL+H1rmvnIhLWJ\\\\\\\/FH4Uex7jN5UF5bk5xZh7TZi4uCW129xgzOTvcstkLHot3ZXFccGKJOmSY7rRxrflnkvTlqrzZxnOU0iQIelAC4abotnfcrshQSoFwsCyskz6kdoN\\\\\\\/p7zhJKDTxb3AxzD3HHiLebQkyQeNqsFKr0cErYaWELGA7ldCLuRpHpcubLgGfSHnajZRB7JRv\\\\\\\/R9674aDfvElaBF4P0qS7RDiXUfER139UxnwY1DnNndkfo1A4qXyb8ZLLr0pfMF\\\\\\\/kzRTRv7axAoc0zWRehpuQDM9IR0Fk+oGjDa7XGPp8pPxNLuoBn6Kc75qkm3LCotvMzgl365\\\\\\\/ozS+LB9KBh1muvwjD3i5a74jMkFKXYAz0HeR69gZS16foTcwE8BHDid9xB9CKiPzMXp9\\\\\\\/9AG2LNI0JgGl+xr38SN35gFcQ24nc71gT2ugNBGOL4qHIoxPjWgwIc+uK6F9\\\u0022,\\\u0022iv\\\u0022:\\\u0022w+HYxz+nK0XyUYZkHbDOGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a015587e3717654ff273e4744419e845b462688c1046276a7f7ba9af6b334afa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tRMVTAuOMyE83icKS\\\\\\\/0tIfDCuI8wmlLo8u48LnyNiwo+aN3yHdhf6srGnStKInjHdiVz0kgt57Usz4QuLqLYW\\\\\\\/aqjj1CGXkyrR2nsZpj8pQUKJS2ui8bWNi9Uw9niBCkDJ1kk5JtrnfMl3rlwadpi\\\\\\\/fVMNnm64EAOb2VjFU2BlL3b65icV\\\\\\\/wj5x5+\\\\\\\/WwgWLapOIfg7zJQWoEvbbRES0CtFoNZIga72o2HqL8dC1tkUvOX2eh6fcD+08h+irvWtlq+BbVoAoo\\\\\\\/WaZNh51FCoXy0SuAoxaKktGAu0tOCuYDFib\\\\\\\/dBcSmrzHcvCPu4K0AEDYB4M9gM6yQ8B5nnMH3tvcTFBJ9clfLNQb2m1rpZKtRB+jOq3hEWoRA8fhwayN8qrMOMj+IYc5FperHP3Pm6GyHtFfkLhBJOcAnU7wNk5VDT8BAkuA7fFSh8b\\\\\\\/RbgVBpn1Q8CGWAQcu1iIwFSW1MbxX4al22MYWWXFI0WEN\\\\\\\/5dcSTU63VUNKsDc+KwTvtWlzAf8Fhrp8A8Srh4pED2bDv1sGfBLa2Nq8Pw7p8hDbjB5AjFHHKMsYJ5x9FZ7roPbmebfxPuwSXHpzdnjjqVLjbUeKFrZnyeYS2zxFy+R6hRUFfIJPHw0Hv\\\\\\\/gJLouzl6pBj5OiDmmRyxwIlFVCG\\\\\\\/+IS09\\\\\\\/31wZLwtqmeTzdu5jExzrmo6e1Rh99yA6q163ChGO7u+9qx5MrQqokhvBE6uF82OVF\\\\\\\/rowsJa9WlL3u3tAXhKKnB+6EZ5ZdAy1uxuBIDOZvrBHQFEB5lhfLoH4geL4hpCVvHfe6x58gCQgZqeB5rZUQW19cRacXKP5TOLONeipfpr79b87+Wk7EJqdtVY4Xe09quu8Qd6iw1qBDFvrTufK3tYnE8R3WEg48ah+ZJyDHszY+atvkkgGUSq97LiXrFtw5Ea90bsKBYbGozHtLb\\\\\\\/+LuGuFZ5k\\\\\\\/hxLKON7QDa8NskkTIvRxOQkgehIVGoYyDoc5HxHhcPGVjYZuW2v6pfyF87NwOkvLQsfbOCl4zqSBLUw\\\\\\\/lbevnndLpBv4UxVLoJc1hJ6QW6VOZcFzMRdoalrnmWWyLl7x\\\\\\\/LMXgbKHbHd5f6nO9+GMtVjD4g2yFV7CMaeVA8C+Bqj0\\\\\\\/9eSp9OmTscNKQtFhvVnnhN7xiuCHfLw156lRoNfqlK1tQJuy2rmoKueKrdZDi36\\\\\\\/mol+w+9F8oxY1zsMVJ1pAZ4dBYtLHrUcSYveRp89\\\\\\\/ZGl+Sihxi\\\\\\\/6Is0+AlGhdq8fohb13CGA1yaCREwc0K\\\\\\\/o\\\\\\\/4T9d0bNsiOf7uGQii315qON9hLh9tDeRIWAz9HDU5bwXvcxty5Wi9b6W266VzAFx9kSkL2uiJiyf+pJM1Eh2fDBOrDuakzPdkP61bao4q2RNWtnETzXhloy2kS17wPcBZUiNGgKiUsPfvO12FwDtFFvaR8hkj4ANtORHxCVhq0\\\\\\\/d0ZngrD6iIWvwLyboJy37woWxAx4wfBpd2ZMqcCUP2tA3H68oNadF9UVHqfIr+cnK\\\\\\\/eGvk2Y1W1nkGmN27hEAL\\\\\\\/PCJjMO+D0apxOTLt8LS48pWOW5ntAhRFPpEnUzNTjmk0+r6jHRVjvzvwM8zY0vuxemoxEc1ncOhWJxlhYWxLTDrEdFoX2wbQWBZEoLXYpND4APPkLjSr+gPa5GQgA9E4KlM3CtEVDP\\\\\\\/z1iLTuXZXUch0BD16AJUvbadQJ2\\\\\\\/2oyMa\\\\\\\/PJh2udNziMnhng0z3q3bu0VI8XMPB1Z2+oUbjdBsl0Og7jkYKAB5vnosskbsLlN3Xusn4wxX2C3vxBti97f1EpAF7px06XBTHed5n6c+mhrNQQI2AaEx8bdz7vAetLYbVp0iEpHl4z+\\\\\\\/D1KnWOIL56JNpoiUD215uT4xykW4u7qa1RetWNU75m1vIoUncIuYn1Ot2qzfdibfc86PjiZiT4nxcIySmu57BSlaQRB9N1hoq+CWqxSS9\\\\\\\/WKdKpncxJjdivu7fmbA4CNjX+8BSpFpgo0yxEhcxPJL2NJSY+7C4BdZC8s31z57Mkak\\\\\\\/pOqMHhZAnxyT+W92kThaysVxu80Nld6VLaMGXULwDNb2+jlrq8n\\\\\\\/2X\\\\\\\/+QE7WN2ckuyoV\\\\\\\/RHM8VBUhu3H5igeAj5GGYz8qh+AjT0QbSd4UjTXekzrS\\\\\\\/msBoH\\\\\\\/kDKrVLPNuOnSUkOfmdqnbV1eXAgOqA7qGL0fWkMtqlNOFfYZl5Ebjbj7fKrx0NJT9vXeChEhBMstDCy8uA6n0QWqAdcwc\\\\\\\/gqi1p7ItgTMOdsoOiLYjyvsioZPBNtKTdxFKmYjxuujtxZHQChNHDMfC6dkvwNuVa0lePPnBrxJaSzu5hmxJnx1LQoerZPy5y19mgz8jCVYJ8BVz\\\\\\\/ybuA632+4XBdsOpBk5Epc2mBXLQ1erPsJogmgVNbE+3b3fc+h1h7FYk0lyCmE9QHl3QTfCYmbIL0Fh2FJIY+po9+SpSTbxo7EdCLzncSiFIo6zXdoGWbU14fF2JhgTuzefGDLPrdeo7bqhZoRt9y0hKKw98MhGwlRSiXH1YoSmYzW23U+ZIn\\\\\\\/BELZ3VYxZTIVUDZ9YK2cpGCDc6PyU61DnkmHG7uhHcKN\\\\\\\/KONtZsHX\\\\\\\/C5PjaIvITJOhsBPLjl4a8OS3bzGMlmlECPPR9dS8KEEMujbQBtnaWUZLjy5e+WeCUEwK+uEQPLuP1XpbB7\\\\\\\/\\\\\\\/LF+7Az408mYqg534TSPJqFknoV1YD3J7MZoe0Kacg811GvZCTZmBHjdZJ69raXnxmtSrBKxYzmbAks0rFqDFLhKAuCH6JbGhYbnt5Uu302eD6dySxP412uB6+yT6kebe2zLANbfmL8Z1RDO+W4yurwL1+r87\\\\\\\/TDvNb6MVoaKxrcDLjCjUlFY+UxSyo2Fh3L0zFMgiYSqQolc1kdVL7iAaF41exmDJShHR4I4JpgehizX0Fijikn4QdusaOknT\\\\\\\/Rc6+BmzQ9+ZAx7p\\\\\\\/SIjDlGpigvhzgaS41sOyoFrBnfNxtN6lKg05EkNP5JQHPWymgpBujCIlLEMwvI8ZU0wYUZtZZrBjJBBIYOLi8UVevvHshMQUFtPGg6q7D8grKbXTxsbwtxszY\\\\\\\/pPbq9D1hXgrRG2ESsdnrRiIkJvzT6Yst03b5Q\\\\\\\/4qaxYNnmM0w8sGJYd6msvm29sis8g+7YnEHqRIa0oh1IJaA\\\\\\\/2amFfCH0BBI84B8BthUBSXbJApPom9I\\\\\\\/\\\\\\\/DvAD2j5U9mWhZvB4Bde1SuRcMk7yi1\\\\\\\/WctjN3ineC6Q4kGurMgIAR6PRMbYU5pcRa5P0I4HsbVvi7dghR330BGv+AYi8AtwOylQj3NOVoStRD4wD0wFahumoEOzTIfw+UYSwNuMYKTyNJUIixSNVVajquOUxVpSvbOTcRkSIWh9XekzbFcifE6AviKBIxvP9PECvhiOYsZEAKEPweX0TsyTdKkRMhEKENTAjmT\\\\\\\/v1h+TftyRX6yHzqryJBjJH9j1NjAjKsItWwRpFYgFX+LUl7gN55LsYpp9bjqvW3MOLauYSMd31kMom2GUL7+LZYL5DGj5r1l+DGQr8BVD3bZgqWjjwGinM7b2acL2ALw5MACOykj1kQyNQBXhYcOWTlYGMqwe2ulsmbzaIb3sOrtcLXBI6P6Rpc6ym48VB75w+UCsKQ+Odlon3QywRk8ga9kYIIOQTigk8KsPwIV46wYYPsFVkaVrlMz9m4XkojSAy42uzdj0fyVFUc\\\\\\\/sNu8CPqUDwUHuvMSbj4ztOS3N88iBV7EkFv9naFouE1bthyKxVMcixrbR2yIEtKxadAT6V+YcyAcvi\\\\\\\/3NbdBYG9OPmp5Kma3VrsD3D8\\\\\\\/YreRt9EpSspOaOwxa2Hxymg9cPn10vj7fsVPs4OrUAKMciUPNTKP2tZr0nYaCGKKtksamI2uj5ipE35mxv73L4Q8s\\\\\\\/HOSkUHv12OFdnGJ1+Y8wPaGT6A\\\\\\\/qJEpSh0yw1tGqdQQPhcNReRejUbT9ZOmE3orrdz3s+D0Fobsf2nbY55zHkx9KNNaGFnQlUqxNtqFfaqACHaGcxE\\\\\\\/ZTr0BRvzk2F6lM+jZxp1MCyv4F8zDTWDCUhOU5j5pWmDvDSVnPhHn9+GGmOpOeFIHJYRujHYnBOSSfyypIvzBXOO0h9Z5y+Jqui1lwaLnNYJ61qny9pmVWutCy5lw3U0y8VJ1uja48gH6jvindE4erLlA8qJ2Dqf0S2bPpYpmGiz0yJwLYNWp6JfQnqNO8aMcJUjKgvXaShAxEMS8MaDOhS30EqQGHDWKiDArgLQEpia5d\\\\\\\/ogsUyA9H07K5hxj8ZdPA\\\\\\\/xWh9sVwIgOET\\\\\\\/L6+aGtf0hC7pySAwRzbmmiay1H4JItRYAPpxcY7Cn5xWAXe5F+IdUa3bR38MVAPULJdkwYQiCWvVl50VTReAeh7MAzmX5j5U3hYoHKOyXGQWFxj5nk3AuxbMv4SOw8MiYM4yJY2ardFFAFZ6+\\\\\\\/lk40tmD09nKJadrTQCUaCV+Mrzmw4jVbGrNamInT6NtrXN15pq1tv0sTVnrpPuA2Xyg6UgXloZJxTOAWMELm6gWIt5uMPYfxYHEfxp3Gtb1TI5sAnlQtOVw+9kCr9vNSyVptFBATOUWBXpMjXlR8D1O\\\\\\\/HEr+VMIPGReltRBlrOr6GRhFNYoE5oSjuEY55EbAU5lSA1UwhWR8EefrNGDVUpWIqNK0Iz6wW6jzTweorsn3rEHaEdSrQ6MLW+weNeivnNH1fCvt+H4dXhSaXkOlcE8jud\\\\\\\/R5w4h84Ru8zIvLy630THSF\\\\\\\/tEKSN4COn1p508noQ7UexhTwurnHegx\\\\\\\/PhQZFQIUJnGYOyqlE\\\\\\\/dRsWzii2GkhPCKG1Aik1ycfAiUh07vstKzaSYREIQ+c8c7WKwHS7F1E4izKXzu5Lku\\\\\\\/1WtR1sJchb2F\\\\\\\/QxWrAZFyEFLTEH15c\\\\\\\/wSrCs5jQzFFI7dkgLw6rnBpOtzHX8xnOw2FjLP+iKxdnBnRbIp7r+SUby\\\\\\\/BnwX5+3IEXWAMJ6gJUsilFtswH7E0c33OLNkfocTIU++O76jkoCbUDM+78ulluytsK9kFOETcbmy0NnV18+AthLtjWdR8CKjPBPXgsAsnbjUe8+4RCpu+9HXWgRBNbIvyH0\\\\\\\/YgBwquX7pQVz0PN0RQ\\\\\\\/E90jl9HHsZpk4zHCGdWO+6vpwCzptjaZP8TepIwcBZVK2iEjsKHYcUrH9gyPTsg5vAjMYp9eVud3uPJfQfBqkPBnqJTLgjN\\\\\\\/vLgfpvbGceu2W2WNLaVITzA\\\\\\\/idw44WK6m322iMTLG7O9rdRRwMEFp7BzfNyygEAVBwmdgd7PKmqJ8gL1Lqtdkh0qJX+xL\\\\\\\/UP+Gyb3FxUVPXk57KGumwWtyozPRtU4YGWUD+iZZ2+jybPWHRz5jeby7ZVYu8yjdwX\\\\\\\/51MbNFMmyQQeada9Agw98dLhLKO+DeTkZUkFOS8guhxW2UUX+y8KN1d4Wq1k4cXOEhl2SMjZ67+KiVzc8auTEvkDqNGAheTFUk1HEH3Tmer6tHav2RgunDO3mGO5pTqaoyWRpIsm\\\\\\\/rTR4rQ91WYoJRtoOHafbmLATjMK8eDO1sGE\\\\\\\/4JEPbm3ZH8t9GZur9HpUAYpu5rGcHinsajG0OroJvkUgkIufpE63Uiri11L0Y+3Ji9mQ08P0MK5G4lYli2TFwAKN5vkB4jXVz0sQn5mGhimPbPkCP41kPJo2+Oef3kn8AcFkcPgZCznT3oB\\\\\\\/p\\\\\\\/2qLv05XhIuJyyGKjxOEsuF7\\\\\\\/vbnGcRyC18urg3WH9Vn2flL8YTHt8saYdy6dRDfHcsisgxAeotZokG2c4vtbxXsD\\\\\\\/6VgdIjuk5cjSZSsuYNGFAncsbrEefBTfxOCw0e1\\\\\\\/70SIzVt1Tmn4vTPdF\\\\\\\/1EQk\\\\\\\/4c30mFbbRrdi2RCS0CCwVXYyWw9NV89BUCYkXxbKWd3rpUPlggRxz8ajA3HQ3HLn0MexVC32fOdYCOx9zLksmugSicqFSEB7MoW6C3ASHVY1oeBpG7Q5yP9VXg4niqkaoRg1w3hh\\\\\\\/irB+2HWXRIGpNqxK66qYo83eh9MXWFppDAQXPCtFQn1I5uf00IbqHdM8JnduW3O6nD\\\\\\\/18KmJJS6bbGGcf+49u9hj+0ZPRm4AjPOYJvpPZq7pUuFfeH7WWf1afjdUY9NLnXSEkEoHYnEISkevZ4REtGfux0NNMgwKbftYbZB1oJduvmSpQoia4R4t7eqppoUqu8UrdDtUJMTVPmXzWdDPMx49NGhYv06gQQrOHkT0wdgksoWN5D6OBiC3ZIpPsOPyWYJY\\\\\\\/CTfTdLGQOCT9870DNO33Hcsa7iG4M3STRqsqlmdKMBKKY4uk\\\\\\\/Ru3IbyHIujMF0eG\\\\\\\/Xbond0+1yoxhEYKukuwt00ZrxfDyWd8WrQxGl0MUUa6P9LIia85KOGV3ADXLoisLm13m9Uvs7vBGetrnSB\\\\\\\/F85Ol\\\\\\\/20AJUxH7CMYzkJziHp03aG6ezOCoXn+\\\\\\\/bN1qZ8\\\\\\\/cH3MD2dtiozwtTc4Zb+2ukKqiLJJ90Bpg9UTEV0J8khiGVvfJK0f\\\\\\\/UcJNMTE0l+nRKI6iZ3DH6qzzMErMfYwlJ5okdnTzZtfszMCAufZuJm9NvwsfmhlzMfDDZefbzhSo\\\\\\\/AZ70KdnTTXESJQ9vvxExDeQAcXu5+wpmVjTsh8xt3fMaMq8D4BFSaEuzwtlq8wI5Juym47FmWRn\\\\\\\/zR23BYFEnKQxd2lhpWW27dyyZcvBB0aneMgbuC5rsjwwe+ZXg\\\\\\\/Ld4TwGnYjjt\\\\\\\/YWUHJDf0sE+Fsm6DtaOBKHoPyfnB8NmpVN5yVpAo4uxkjA8uWLM9FSsjbotsjULD+UPwJUxB7QzUVm1xWlg7Q+8uMm0XAXR\\\\\\\/yZKWHwAh+tVSGTjDnh2rGr+vbGLPCDLYY4QqVRSAr6jmsWRf52IWU4mNtmxccjt\\\\\\\/puUD9fesJA0Mzgl8hbcy6ut0K5iHuhtx3lJqQszLWGe48voHgB0GWnwPdnCw7JRzW+5Uyi\\\\\\\/FbL50IRQeDfj+cTrR9pLJ4kpaNtijaA7KMEGUs7SnJTxccIGOUIQvGKOd84wGcI5Y9e2c1Ap+ok0bFUhQMB0Mqqx2zPqtJhat1QzwqE+E76Z5dcVkX8lidHS12LNMEhLGHr+6cmBvVzbDwKRKFDYFRs5yIMGiw65tnK8rTa2Ajmqvp1PZsB2+jVb23LTRkqvDmDgzYWENH1VyOula\\\\\\\/Y3cPsQ3kJr56y8a2pxA2HtXF0USdJyCJQQDqvj1lOvxbDyiriH5cdkdDARRDld6vV4Dn0gl4hPAKuKzpD9mi87XT9Tvjs1ML0yHBfizn\\\\\\\/AEQh67wbQwHczlq+LipVxNTOp347PVYPlfvxqZ+Hk2eE1eCkzFHGYh1LESi48SM9Lc9SwWG\\\\\\\/6Nn4624CUxaml0IqrtxWceIlE+p8MmBlV\\\\\\\/3ecgbZtcqWXhBiHg\\\\\\\/Hr7aEd2YNXY87Rf\\\\\\\/DmnfWiXBHizr6SJCv3CdASIKtoSVp6XBCeMj0EVSiygw4hLfWtpGsfP+gwVMAnDyjmMR51tU7Pj+WzZxhkC6BvK1vxpwXb7lI3Esf0Ykj5wRLLMnMHjzkgEcggRW+ac3NQIshawMtRdUOAgJH7G1FiD8jmBvNxaN8ju2DUlqbcuuv79Elabn2rwYUGoQEISdyxmhQnXPYxNO42zykJQfKQDs7byTeArr7yt0TSkgw+dXEFihiNE0VNz6LrZ84EpfrlHN5+56YI5PyQV133k+8ZUmmbiYsua+WWfiWfEyXs2M0\\\\\\\/weiG06Oq1mxmQqzw7fV\\\\\\\/IjmooUbQD3BnDuJ1v2oJkP9PbiTukQ6Dffds1fV8n+IHGyP5NtYaItEjRNMRbAVrhytF4eAyY2jhUPoRGn0aILoFZrv05HLOKQ09C1aOwxmHHGAMaRQcnqhOq7y7OsCKqwizAusKj\\\\\\\/N6dKAsSS7gPIdTPHjy1wAzLnHPyO2zolTchS6NhtwKOcJa6HYv8IOJruDQ+gdWB8fyccAquiehftMLP0o0mbaYWb9yDY5KNGcCKTBRlB7rBZpxjf7ZXOnHHjK8XEP5h9heo2F7V0AjKQGmMFTX4mdHtmSYYCQ4Agp0z\\\\\\\/b7TajrrnruxOyzOodcduARbHMZlgZRUtUAyHvKbQyV5XAewejvi7q3SFKwczP31uISA66eM6HVPSQFb4GG\\\\\\\/JIKa0jm9Ti4MDmUaPmJy+06+WVCvu45rHlFSbH5Tmjz2Y6C1p0tIrbAK+t0BA2qWN2t6qRH2fZFjm1ay00iTF1fLyHKHQGK9jpCmwYTfa6rXMvAZzJ7SQw+ZshjgV777CzSx8BrX9f2h4aB4NXaH0ZrUjJ8DheH5cIMjyGefR4Ngmxbx5hxM96Lh53d2hgaOYt3xx1Gyhr+82V8DAJPvgMP5SraCFdJVXobhMxYWVP6qFygiZgLLYTSRUu44wwYRGkM11y3xK9lQhldLM5RD59w3fF9kDQgm8BrCl\\\\\\\/dLvwOBPyTI\\\\\\\/ExNd3Yo0AmvRuBKWJSvjxZXFBIUJlZKJEDyUqrRQxqAfOq11MyKBn+\\\\\\\/xwHVJe3XUUcENlw6XapMYVuFvD3eHj\\\\\\\/TCx8fV6KS3EuLXHTieq1SfTePP+CzFHFBPW8KEL4C1f5ycMkEA8Dv9zCgTJ8U1hf3Ox85gNZxaDptuCioK9Xx0JVgvXO8UyRh1yw5aOZxXlln4CeXaJKlKr3ykC4ySQaMv5DY3ianx98wXRLqIW5PBwMTDv4dWSqkEEZhtgO8yue1DX7+p3ZpaKicFnzTSdSJMOrfyOakBua9kaZ0rUBrQ8fIcyvbHmoFIpD8rsPZrIvRX6lX\\\\\\\/4kmPxeJsrok3D85p7Pzux7bhTeunre6sgnepdn90aZj8imdzxWo5SZ\\\\\\\/ArJFPnQN4TEIeoFmU1aRsIrF3gSu9Qy3xYg45BV46EJMLPRZdvUUIBA0MbolfMnxjDt6k\\\\\\\/IXGkbfbpR7KfBQhiTzpo1ogEbLNRUloVMep2gswlMAvMVvkkpTMPlfYXiq5AxoAik5fbRovsw6d\\\\\\\/4+Eii\\\\\\\/otbTdUl3hFUQP4szepUd2XRkLqo7f3duLJFAsUWmmnXnJfUuEyj1N2xlck6YBK+A3vbcI9ifgrbto6lNP7bs8\\\\\\\/EDYnx7PL8bLIEosGNUBJPnGjVdt3H8Twux0jojpcwSbb6ppZ16hSO2CkIfgAGNwWHkdhR1RGLzY4Lt2wQ4+I1XncExqjK2YcVx4ZxoUCoEFjc2cgWPzLhtjYNYf\\\\\\\/vdLfc7O\\\\\\\/eAmCQBXeMGdWf\\\\\\\/EpzyRFjOQxK5KUN2cqecg1G3iW6xqEIHvAU9QB8RY078R5EAFNnKvcY8iKd68M5OpNTGhCYY0Ddz4EGSqfTXS4oahOIwzA6PE7XmoHbfFTqRqF1KE6HlA0EremPCpivd01GcTuAZL0RcWvsoVqgC+eIN1e37XZbDO2HMNg1tE7cDPCb++wVSKvCu6n2DQy6la3\\\\\\\/WHUX94czHuaqzb+nR9LjID2MfOoGH1gP+wP\\\\\\\/lcuB3zZLxh4Dja04h6GazejyrzI2lLHkb+JGDIgR0ANZHyFV0BL64TqR7RJu6Q8pO\\\\\\\/UbRfjqZuPA8MZbO9bPZnktyN0gq4S45Wc0NZZl7q6Ca1NwhJHJhUpeyX\\\\\\\/lL2L3KLUSjHrDSrNDv5MAlSdTylZI5YJgLMZbwi7qJg9coV9u0hzhYASjz19CmOAtOE\\\\\\\/JNuhUchf9WbIl2XYJG0bZk3\\\\\\\/vm9lwhTIhAqAgbHTodtQ1AmbAMVnJJOh+l2DBTCj1rBWsAtE88GDbDdJfQ251oekzrzAGrboVngO5apM2UHMo6J4beop1jWpnyBEW0rWxMMRYF4OuDCraOaqtmp7Yq4gZmXmFhdyNSj\\\\\\\/CAvdKuBmbID8aMVWh8ux0Qmub8pAxpq5dB7k\\\\\\\/fd+7gQplUF8ODHT3TLAzCCyokuZJoORqu2XgwX4u3Loj\\\\\\\/H2r3m\\\\\\\/qTSSTNJXHg9kVU8wvElzRLylbO2eMryBzI+SUgdfT7OEXhsg2FaHuiic4vNdv5XppsiGPCDESl6Zm8sm71mQv8KQY\\\\\\\/LcqFmmISmgYNKl5D\\\\\\\/rplUXF3VxUg1IlGoMtMS7HPQVqBoNQO8GUs\\\\\\\/+k5eK0myEuCRCNor3kkLqWFW+bfdx\\\\\\\/DuIlXtlQXehDtV1TL6ieoW+ySAnf\\\\\\\/dls8R+w0BY1+N3Z4SZnmkBRRmtEul\\\\\\\/u4fEFMGaiQULCiokfrehF2glmO71JAI0mJvJQJs10e69s2BrKbWh4S2noAMA2SQpKORCNr8o\\\\\\\/abYe0sIRzOL3Is4t\\\\\\\/BO0K2RzQatYNcBEPyLAwuCngaHp\\\\\\\/xpm6P5GYYFD0J9e47rimuG5ydj4DsPb82eN0G3uRLnKarnzKUc0R1KWlzhQU4tQd7LR2zQikcdTEDXpSxM6lINKCKMo+JmhOlgx9qB3gxmrorsoB+\\\\\\\/SSGOca6oTGza3tjP1tOU\\\\\\\/vSjnkMxq68A0ViIxHkzt9CrbazyuTcjGKpmyUEYu8q3CBr7V27LhpTntDGUm+dXG0U5gdZijIqIdLqEZdAE7+w4k\\\\\\\/9wNOILvMMBpb8\\\\\\\/XoFeh5\\\\\\\/KF1G5auKDktYzL+QS9soFBHawZH8L7G\\\\\\\/fLwlQICXPtGss9KeQKteGV23ZwKtob037hcJSJvZs2dVlXHBZjFtGFsb4eJ+h33\\\\\\\/vwFV+OXtLp+7K5YOGVmY+PhlMVGZRR5r4VVaBf5mPE5OHsLK1Pd4QBZHABEnG9CUlZTa\\\\\\\/+mdeUdKrbbHjxYZ5\\\\\\\/JMuELdjzZObVOQZzU50T4SRyESO\\\\\\\/eGqejI5ipH\\\\\\\/KnsPwkdvw6uqYQ2fNMuo2HQZtS4WqqeZIk628DPh00KYaRravVVA9JP+MfBFdH2OxKOIP\\\\\\\/W\\\\\\\/SzkviNQwSXJomh0oaeTyjAB8WTK5y8pyy1Ciu7WBvE0pYTZ4F1mBCpi8auF+E8+LSOq6I2HiAUNNDi5toWDhs5WNSYyTU++Su+wKuZ7hzeGLy+DSxAq7Ij4Ke1RCZDEf9W33JJMgga1vaR6+0OhE5kgyw\\\\\\\/yX1ioYO2Tauww0VybJ5KtLcgaPnGUHJduCFohmWxWaMlvej+zVpENAPJCiIgiNJtKmMtL9knZSDylXWNiKd1wPoi8owDq3\\\\\\\/jhpLqddDgB7mYvvdHhQB\\\\\\\/OA1hTKqZik1JY\\\\\\\/3xtxymb33XfUZd7h7zPpHIAlU37f1DXMHnf\\\\\\\/vQa1eG3e6jAyFhpE2bMlXqgkGOP3UummQVFCX+rVZCGszZEDaIQ01iTOteAhOyQTlGsYwaE2WNyn6t8JAfJXu5Q3cbKJ0b\\\\\\\/rPeHSyKg3BaxRz8YAFjkq4HNsZbaosFa2gtte8opZhIVqrTx0PJ55B2TE82trLu2CAY8LMUKNiVUL93MVDPHE+T6TcXkCwcIL6ngFo1Odbp4C\\\\\\\/Uq3\\\\\\\/p74U0\\\\\\\/Qa\\\\\\\/IXeSuSjVvdYPS2fomzQftmvse+65BNJrqEQ6VS3oPFjTIm6Vx4SwS\\\\\\\/aKMEQwtEfeZCSKPyLuuAMcS2RF0GyxKel4eA2lOm7+8bg3PlsMFsHzr4JYzGl0Hj\\\\\\\/53S6JjzmWfGBNjZ6GolO2ZnruVFknXtZ+ugpykQQUSnTzl1Ag4ygP+fPZfT20n++JOFuZgRrlNC7o+FwueZxgHdQ\\\\\\\/tgWNL4hLu+vpJVlcslhKGJ8E7\\\\\\\/nxMLqOHgQdrEQp+y4RAkTKYD8C612tvNtTT+RUAqgnneavai57BfscSQqaOE\\\\\\\/Rjg4n7Zr68RuvueFVkj95yCCzRg23gypVEXsLcXxND2uJI5kIw4eXAGmKtFm39OhZEvrNx937XCH2py35qrJrDuGZfjPXolXC0K3O6MdO6QOYxlxPLiLE5yjwK2SV1jUBbuAU18bgo5pnKk6aMVKepyR1dhc3J56oLRMDko7NmsZz+PchKoUoL4K7pk2+b9LjzJGcsfz6d5x\\\\\\\/vPBRn+lsMaTm5ZCzJZP7ebXUSxqRisd1WFLM5Ij+jw4DfG3HMlYtJza35rTmDRMFk\\\\\\\/AgwfDJkaTqL0XcQMplOGbckiRV0Xl9blDTpLDBr\\\\\\\/fCGBXHtx72f0xTNDaQ90wCJisK4rNfpf8DgfzBl5TflNsiSpenNKngtFn5I9WRQgQahejB\\\\\\\/em3eQIYi1w0aVtJd45qurTJKv2MqlYVSQwEizPaYNIuuxeKxPNAojxFhDtf63As3w9gyw5j\\\\\\\/0JoSUnPsmQP7mPPHqte5XGxCMV79z82zwxAjx\\\\\\\/p+FKjVkfRCxPmzCBH1prIcDr042ch4aC8jgHUK2zvPwk3NMFn05fdp2vMJ\\\\\\\/6ykT0Pvoft8lEGkicPpuTv+iQSpw+0a0+xFMw2kt+oe78XXlbCtZvoTKlPqQsmR0ynuXAGdyULHdbXqNJuplNkUTPJBrsEcMDGgwYSGWEW1jVcnr\\\\\\\/wC705XO28zVOBrKURXo5F8TnhsX6OlASU923QsrJ7tQ0NE3Rpb5pcqRCgcIVKrFhZXlpe0qwosIpC9ahP1TYMsWLO0Xd5fjmyC7MmkOw+PlBZfp\\\\\\\/LZLSADCPXYfNH\\\\\\\/Snt5pBV3EDNO4WXUom9cKGxE363YDtuqJmkckJhDB7r7pfgoznrOMBRckgrd6ZH2sgkwCR8PqazHVUVNWMIJRaKy2EulCHF7z6LAeaBwkKIPr6C7VIfUIaCmGTO8mpFincLEU7JgY0w9NXH4dwg81QQXOvvuYXAJ3OhNoE1KtPZPP3Q1M0MEyRBPGUrtTCJSn\\\\\\\/oQjMLvfCCw\\\\\\\/sGFbcWhJ\\\\\\\/n\\\\\\\/89+CgukSBcT8af8crdToXgsx3AGIsQUU+G6gTPUn6yA6+lByzMA4o53LClddn8a3NCtUZOeVn7ohn+4j7Yc+He+NtXTDRx8n5lI\\\\\\\/XdJU9PVQS8SxGiBlhBrQ5iL99LL90zntX0wQso7RExg\\\\\\\/HjQ7sl5Nslv+9+6J8O7Ib2Ozvg418RTGO0kGUpjErHsPGQskccYG6CyoeKka8LCqlO\\\\\\\/u6OYWM+lNERkl3bpfCoWaEKsNo9Gt1UG+EBWN0hGEeT1oKrEVSvfF3WTNXDccZxd6zIV5o33MqeZqflC+77R4n5SJemrDXL8v+ldFOtL980EmrD\\\\\\\/nfDSmYG3faFFxEdHGmKJ4YChvekLOjl6X5TEj2gK12YrnC\\\\\\\/ku65xIieryTuXrazlp6JvMau2IIJQzWCtfWROtDQdKB+\\\\\\\/Ee0Ba5kJh7m9Sh\\\\\\\/GFNme744l4V9BI7EhmUdCnS9OqK\\\\\\\/tHKhqIwz4qwwe8t37EfwRFpahyPA3ovzGyZBEi\\\\\\\/EteLGZeqAn\\\\\\\/MGZ444YQcEVCbK50V51PvbZvkY88a6zekH4mJZ2Xp9rI8e4sS0NUiCbEdSpROjJr72zXpeuddaOW5ptGhZd8C58ypuFvw5B3+RyJhFI7mNmr4Epv9pGeQCmDNqmIFwt2+sl5LOCQDL+vzSObDR5T0nLpMQVjKmFdYvMGbzx+IhFncu+awPpwJpfjX\\\\\\\/\\\\\\\/e5KQwF9oxpe0vh9yO\\\\\\\/n9WW1YRSvWjArY\\\\\\\/7k\\\\\\\/kIsW+839GpxsRGbS1+j9uwIyyWUT2bpZB0uKqwDKe7cgzcrMx7uldGAnuGK5o+dzz+rQ4ftoQFck3IvHNg4vacR8RHu5lxuyID6WslHy8rjcTO7ILos9KwmQTAawJ+4cx6+hH1u5URl6lhNqRUvpel+JNYxtn0Xq32YXbmG+M+M9oJcXIIVPKqf5sLW23o2H9SLi7gigIJb0ahF3B4khxL5Udc0NeXwQoO\\\\\\\/lHbczKSzpl58FLiR6T2xNq2jKECAyveTlG2gcgEnA\\\\\\\/uiEzTANGgFmllML3gG5\\\\\\\/CAeedj0IgpXxpyP\\\\\\\/CLHKfFcyqGgILDyJ0TgKtAf1OfUjydaX8s0p6OwFelSH8FqXCAeMoYvanYu8ODi6j7AAMFcIp4fFWrTywdPkeWO4WS8rM5V6gQP58t5OYQ4abovJ8cpRE16P2TvanNYEIo\\\\\\\/79O4h8usaliXKnulJzzEQGVd8HRWKU9Rlabobc9WmQzaC71MTZzvLKvoYQ1xwjLc\\\\\\\/4rtRF8+1OKzmjNgWEjN2Ow+BPC6K5HlWPe1wcAGWuebZU7sw1JkbYPuwClhGMsY2SLJofA5LEx+OKxfDSzYRpUOi1HnkxUaPHTCWQWULg1OOmVZSTt06887upEsqWUNruS0BMxvs4X3dDqvcLK18nxebmBMPcvL0gtBTYlie2XbPCNqtP1lAbA84hpeNLcVPW2NVwBKv41654vCTBBmkeOMwcJoSsZ9D6ef9kV40i6g+3O9phaTBRyPJFvQm8FayalRVxJNl3TvvVShDJN5JmxEG8Rwp\\\\\\\/jLNhihJIcnurMonMYdY3hHrnJIucu6U3L1KT6v\\\\\\\/FHmvNM5YY\\\\\\\/ICb+xdvxTtZCq33d52iPtaQ5L7Dp62dhRMajHzDXPXWiatYdATYMVuLv7a5k3mnUJiU5JkRhYVfRAanVk2xtz0UaIgOVvvLTANDMyce+oD0A2Z6hhI3IqvB4B4S4EW1wT5PhoNd\\\\\\\/oMTbCjEwyZOkZarqm9eY\\\\\\\/3KzyJjvcJw6fQySCz5PSvdCAGLFQzExH+oceUaLJXvyssBUmdo4tQzfEvBsXQ1ZmGJUJzgACSPKlxav2MuvZYaIipC6HzZPOf9\\\\\\\/gWozDnLlpSNzJJ+wbghwv3ZPNw338komLM5eD7hU4+baEDgdFfQiTSXghR\\\\\\\/YRSKdJtclwRUnXz1ZYV6nXDw4p\\\\\\\/fclg6xAm038T4\\\\\\\/+TI6jTvAzXj90vt+hbGmpMvQSjsPY9iFvFyERJK+K\\\\\\\/vri0Mdl884rQ2YDWFli3qa3HNKgCbtMrJqNl8haD+zwX7SE5O+Rzfm56Lyb4PpyA9fZMahiQvLl9hqfO2rp+EkSVXwL8DEoh9DZgJQKA4Md326XfCcFPqY7d7RGybLyXm019i4uS0dVYWNIswXGgGLWN+1cCprvV8u0TKI7gBZrG6l2S2aN2r6JhhHMYkjqTwblyr+41RJyUUuBDK7lqoFmKd2VAZsX9FXiZ9CjR2bBbI33up9zWSBdAShCsvOlPrKa96NLzeLAaUnG0FVW7VkR7dcI0mbbwdbZ64ScU0HO\\\\\\\/Z8YQ4eizMg71\\\\\\\/cRem+r2fSvZn1YyZGsueuKYmP9RzbBLDMJg45QXpkcUYiLlBt1MJy8pOHLB5P9m2YQRrl20c6PsLeEhfPoeGvYwiPelpwmyIqjzbrcsjs3MFa1U7\\\\\\\/mwB\\\\\\\/woApVBd4EvThnEIYaIgE3PUkwfG\\\\\\\/69Kuj4SqqLhTOaL2ALXq1a1+nSKm44TLYUmgQsxoPRIr+\\\\\\\/ebBUPeR7AilzQ3SHs2kN6ASmdVRtZGoPV+o78twXNHhSvzmQyslxP+jQ5CPPKo+9s6v1NUl5s9DjdeTYUI5ao3mEZwanEGNYhr8qPdZUgo\\\\\\\/opFnIu0NszuRnhZaGdS1BTcqwHJIn1L3PbE+OSF0dRDcIW9MgtwMSDeNhyL7A9AnLXhFJ5IgMNfJ1rirhrpofTr+Kl20gs9EnIrlmwoS+J8wh40sCG7dw+Sqpijk3WQcD7mzHcYzYKAlRmBvEZPatXPey21thUP7885tZ0QflcHGiTLaW\\\\\\\/GxHOV12p2um3ZE\\\\\\\/6aZwnhjHIX6FK2c3xEFF6Q3pbM9NPQ2qJiU1LiLm+urSeGxyYD1byFb+nIWF1dYze3ufha1F0Fw01SFERTDUcMg1uvwMnIVmrykA1t\\\\\\\/HzQkXlnYsOSonuqpVkTWjqh9PGaBBrFqaTNL+MafotHqekh32SPdjWQfm623+UzkmdDej9sXwdfee7uq4j+ud+trambDjAntP+K9\\\\\\\/I7ZeNPDu7mVAAH+uOoz40SPlgNWbe3elfH7zG+2uC5YPe3oCCg8FUkFojPQM88uhr2lPTu+SnEvDOwlaeDKzMizs76kiyoGwjNUvW49zv7XuRfN7HZ9k\\\\\\\/ghVlaqaCshQ8tzckrZqEGWkrmOFSw8mwJ3xFS9D2fZ8QsmEc1JgDKICsEVlxSfCzOff5hYjI\\\\\\\/Ke6T5aaprHFJbJv4Onr7NfS8JRb06LLbyva80WFbM8fDy6A7NnJsLD8yzXKYP5RyDc1RP96LsEdRK2N7YG641Fmxfj1P3\\\\\\\/2KmRBaTyb\\\\\\\/htk0tCCh7v2L+ppI75qVmz2FUJfjaqRuR+Kxf172PIfqmPk+U5UFnT5jLBK\\\\\\\/9y1OiCw3cjgThDmtMkHHhlD4tB7+gRimd0PQu1\\\\\\\/owsv957coVT6KUG78c9GRI69YMJctW750\\\\\\\/08CK0i4KQxrRY0PK\\\\\\\/5I8q1JCpjX7yu695oIt7GjSV1Jr46HdsQUNIGouncQoGK39pKOe5raFZxSSKA2VQc8OVAHvdyI6ZPCl32ufWWaE2Lx\\\\\\\/xnt0ODLozv5r3QGATKQnd775sMARDCPxD8FGED+HHXkjefITKr3ZNQZ5nPCfk8bb1f\\\\\\\/ocHEe\\\\\\\/rTTjNZ2hqnVOrDeEH7SDUj15aZBMzk0tvIiZCYesDU1faS8iULKNIXP2WbC9k2iD6XhCQS+Pe8n1usJxFMK2ofvW6F3q9qefgyT0ZzF\\\\\\\/nIXdauIMDgnsJHQltHFYIuUXQiSa4SwwBl2+nVBOmIXZK05oDPU1KFgO09JmW5sDK8GLzM5KQXnyRENPXtbBCjzZTVuDcmPgyCHoPBQAj3TvZxtVe5jkzB\\\\\\\/UhnworO\\\\\\\/\\\\\\\/y8MPfuI+258oFgggqXlW\\\\\\\/ebxYBCFoNACnAcritpov86fK6zej84RdKV7skt2feZSl\\\\\\\/75DKddsOje4ZAngAwsWWGcTnKXoNL+MTrkfxNwPbF4NI0cPSoverIzNEKsuacwvqlxv09xW3y0buJIvBETn81\\\\\\\/UQK\\\\\\\/ukJ2UJYnyb5t3A5QYj+NA9FQIFsoD9+sdd88pW0TQKUcheT93Zfe0BZG4lpjO5Sq7mQl9\\\\\\\/5QjC89Mm\\\\\\\/MtcEHuAfyRA9CBdaVNiWmOfXwqAsuH1xqxx37fTpPaAe1z+o+XYlmrkH4QzYNbEj+Skae3BtADfRUlwaF\\\\\\\/PtuAI+LPSbpn\\\\\\\/8LHwOdAyJ3NN3rPEeokxYb72uR0\\\\\\\/PgPatykoXMQGyf+L4VZ7R56Lq7fHbi2fwXIf9PiNXLzFti98gHyVl5Y7vGOX0MzrGF6wY59hhBE7uUf\\\\\\\/MfjPwPbnUca4IiVnXC9ZzWOKr8P9xTzS2Enst8NnJQvF+ls0dz+ZTVPrd0EF01QTpW5drHcMwGcAu34gPHJcyvb9lDlG4lVgNF1Kb9puU9d+ZMZ0vBRZDmKJTKU\\\\\\\/EREKDtYxIGqsdVfoLX2mtbsAOHi9u61t3RsictTLiHXxMx4g5kxa3yyZg6VdZ6FIHl6f+yerwjMjt8NUtQP7lOZ6B0JW3YCvt7eNQ5h7Ek9md5VwlRA\\\\\\\/8Dvt0YVaTdP+IKE441V\\\\\\\/yvzahM+rFqjW62oUzcld3aXiS2HGkzQPakS5Nmj57DytamnzaqsROS6\\\\\\\/3adPdYXr+\\\\\\\/fL6DLKQP6tgfYIpbv8zy39OvY4B5kbT5TOM3sqTSG7yeMGi3CJbRoV4706KjItI50RRUhDXl\\\\\\\/Gkk4zqZqT\\\\\\\/DRBcGRJEL3hIPobRDP77H64e2uu\\\\\\\/Tgt1pph44AgasgO0fTeBWlYp\\\\\\\/1yL6TttVMNdkBr2ugpoGza2I1uhjO7jLNG1mdWjt8QwMji4\\\\\\\/0YHnL2LgWnxSWo5+Tfx6s1ibCTNMPycG7c+nypo\\\\\\\/OnvIrFvf3z\\\\\\\/6ORUnDn6D9uhULProKJJRkbD1pblq8HeXPcModXq2F7PjVOS0CBACc4NqTXknz2PrJC7G64YfXyjh0ASXef4s4s9r9Sa5ZgU31jy7P3hdW6c7UBvaZHkElQWFnPbaHmRWeOXwhRH3c5wi84\\\\\\\/MS\\\\\\\/OR2FhIRvnF+bb3jQ2RIhhh3ErxHlYScUGSEB5X7JwZJG1YwX\\\\\\\/UJstmfVPRn+hfarradMv6gbVPSff2QiIo\\\\\\\/7X0Bt5FR6JkX75h7B5O6AMYLlvRmU0FK+\\\\\\\/zqGtKweA+uPF5FvS4p4sU50MytY3CsTpNzEDOjGNMMonliF0Tj5Fktupo6RPcgpAW2ppsI\\\\\\\/ZG88zfzrQ9GLQ44n4etdEPbIkzs9Rv+XhExNX6Ss1RJlMXq+y5hrgtzRXwBY15va2LV2IHLbxsfstp83sdDh+HL7PSek9s8oCFVsWlapniehNaBA3kambweqWE20BwzrTnUJ5MBh3FLonBdJD7wX+kQuHtxMBXGcgkXSbCCErVNMwKSr+jvrhLdwFtMULr2Z41UUoMZY4Nj7Tm2ZpUUVyUBtl6T8OMC7W3tyX9J+pyq3jnS6vb+awZ15e0CQaTN7OTrJ+U1H5JAFtXiYSZXUL6RHLzYCNvq5XX4i770+waTUhJ8WdRCBXqlJ1h3om6lQf8HfxiWo+S3fKRuq57Yeh8BVjH2CIddT+iLkyhnpZEmYEyGLt6zNUvu2F6pEmHew3tuzFU+mpLfMmaDuZu2JgYHZeBI6iXQNBK32PLtPA7IBnM4UwxUCDFZ9sMOsjcgp0AOv79CpNTU9oH68Q539gH1WGRIlbQUOcpfNpPe7zIUxnaPE5dGJ1HVzeU2qEcx2X8mEbwLR+B3UQ3NRGT3C9FGdtSOVeJCDn8VGtJ28lYjPI3MzO3wDvAQrVvhkDHGafGzIfZitIBJCPMqjRju1yS+EhSJ4pi5ziFpV89Lu4yUsh3y36Y5A5HFKYgBGSe9tI9LjZHU1xkqZsWAgGg8SCn7OeDU9ISFtaRjpcqOlklTktATNMr3SmxXkwzkTIEpDqH23\\\\\\\/tVL7ig8G1lQwd4mEmrn8hUu+bD+9FkrZ\\\\\\\/VeF0AUfXUR\\\\\\\/SO7OF+Ob9Rbb8pncpVZQY23IS0ryTvB+KFTFFBiu3TRdk6ozseSRxQ4iYEruoSUz8sXT\\\\\\\/dj00gWcEQUUEYtbzXAYVba9q545LL2wCtWoYRq2xA6iO4bCQ5R8u7+SOH5UaQJWq+yReFsQvD0W\\\\\\\/vk+bLdngT07uBjhjzpQ6uq6Zcva12KMqekpBpARePoS9TFSg5ZvNREx\\\\\\\/wxgEqeDl2VTsujeiTx3ExA25JvNLCycOE9ccs5eT2O6AFB\\\\\\\/4CebDW2kwIdaCAbQpw\\\\\\\/d0COgh8jwozUVDTahZ98BFYtCMPS4sCB2SVbzVDnB2wPyalhR+BXDOHvgprHPoYm3ybrJ4L8MYrBBPNYUmLdSGsWZMWBLM\\\\\\\/937w2IyT2+dvnAa1gZGCjN2zfyAXKI+kso09jBsnNkqLR3bF6FtmQq\\\\\\\/HxHxRKwOk8yJSux2KTvZQ72tsY1jA+nX+I5OSFurXq5nQrRknDPYoATMfZ\\\\\\\/XiKE84EtYRV6TqKjMbuJXlaDdsYOiuXcLMGu46ALReSyCpl9kaea6vzbYYcvtl97fCxLrnxdV2+oyGU8HVsloYUMYGoVMShMw+JtrH\\\\\\\/morD2zkUaCFQ8tR9pnRljxkXK79hF4hm\\\\\\\/kLPG9g4Q+Cl7Dci2wnYhTUnvwM0NVIU\\\\\\\/XXbq5JATgB008oSzTIcnnyAEcC85INe3Gz6e1TX+vTPkLxwVlGIAhEtyA+mpuqs7CBOTNSZVDGLoyKD3QwVL6Re\\\\\\\/0YLNdUbp032SBPHyYVkiZuCMFW1hFWORkY5vywwydBXB44AbD1nuSO6oHidS+Ik2BdbuaWBsTUYHyktk78UagLwnnGBmvFlFLYuT8H+8DiETFa2Fjw0ROCuF4kHON5Mu5K1D2hPru0NITOxT2eboCqjVfZLViPz52aj6j9jQ6KCXjDjVaOExNJMMeQBcCI79XX4AHMAc4V+wAzkW3IdO1D1dCFTCeKHytLVgYwMzEuOAqtrMo4jJA7B3MjWRa2X+n7gQ+E1GmnyVc3uNFoxYC2FoxP+7E1x4DPTQH6SCWlvP+3kbZoqKpv2kXyxviVkEju1e1D99FL5oWc5W8YFB4hxRdn6tX5VKZz03lCAkNLO7qFCFKkN3DIyiZaS2K4l\\\\\\\/ibJHXkMgKtUAbfjToREbkQgTZiaKjYf6A3vEWltABDvlXd7EzokLeY4JoLgSxtzSVkpD2DLh0O9ebvloifrx7RaKFRCCGjlE\\\\\\\/loZmQWhYRY7lCaIaC5j0BViduX9q9c1CZTa1O+BBSZRc8P3P+S\\\\\\\/U2tvLMbeafF1rEZjR4iF8WWQgVynawtFGw6wOSsIIpXb64yoVN8IAE5P\\\\\\\/BNzSJ6dFojHVKCyRYiLscBEZB\\\\\\\/R\\\\\\\/4WlM5pMC6tb\\\\\\\/+kiDUnTpZj1V8vwsGVU6r1grTd7qOuuDtupRG+eQQn+9M6d8Gq8Ea+k+jj7jwZyeR1oCZVBOKfATfDsGH5iu5C3jHUpExPpcUWEbtAAgIRcWRT7otCGTVSi6FaJSzb0ZIw80DBQtQ+68i58Wq\\\\\\\/bfoA3lB9GOiryeNbQdDrFQEEm+b8xUkYKhzuLi+QbjWyff0+2l67RP4OoC9i1CxQ9o4HDJZWYn5WkelDeX3FWFulX271Ohr88f5gsHPBlvv0tfyLcIhk04MGMPUo\\\\\\\/2dTK6KN24pNg4KWjAwUxDJ4pIDl7EHzKVTKxFlf41MpF2hXi9CgW1xfO\\\\\\\/TzdG\\\\\\\/CsqFNvW43B\\\\\\\/cyAeNOi+OPNBZ3M+nJHWth7BUsFX2M8lrkYkCiVjQn3GYmMZxOXo+u0Rr2GfUTwahfMOxs5GciuDrKUuYHvc++L68KtW7f6w\\\\\\\/WX5IasHqvmJDmJJldZ2GN2f\\\\\\\/dxgi8RpLLbBdls5LCOfgq5E7rMTDyfcbLwDkbWsVJDSFZ6OLDPLOid5cAK+475Yeh4dg9sy\\\\\\\/cl4Rd+KYvwr4+r\\\\\\\/Z9BsUlr\\\\\\\/UQVdvKzGnWS\\\\\\\/VdeJabG0GVkskGR8HmxXjF\\\\\\\/h64BE5tb5ku6se1Al9HnFDWeIOyJVBD9r\\\\\\\/UnyA0cd6sxPZT7eLRzZ0wUz0V1Pn72It1\\\\\\\/rEcsWEsZ91nDHfmruOLsjf0TbqHcl2\\\\\\\/YDphzd0W7GWXtA+pa91Iki3rM2xaszGvJKFj0nLZ9E\\\\\\\/cINmB0Hh\\\\\\\/gR1tF7tJ4kHdYQH6\\\\\\\/MztpsK5tFfckujTr4Q0oHero0\\\\\\\/jWzzglgeB9Z+i61M9VLOThtsZxi3FxWiuVDqY\\\\\\\/pZTEkBXS35kAe0tFfqaqsi\\\\\\\/HQBPFcfvHqHQr0ZCw1miVze+X\\\\\\\/TC9vKMgkjRcccoxmSwB\\\\\\\/TDF94g+Y6bevBKdmp5mTuAeYD6A0nXWGR5AP4hQMmgs8BN++vF181iCxtzKFgpdeBWTn4Q7NGUUmjQIu8gb0XARhFFfv6TS2kLlz0yMXT3WJLHCKYHw8G4\\\\\\\/SmjjzeV9C0Zj2+P7qW\\\\\\\/KDVuf+6PTyHZStyITTj8UBUorlO96ly+hLSTMFuA9v1+46zhlnQKDwLx+5ZWi9WatOlUNlgLSbjdM9eMijEVOBb6FhLoCJ+38kAAQpRTYPYBVZraYcShQKApnwhdzFrlv8v\\\\\\\/XRkiKCo\\\\\\\/SOI3kAFthHihI+aEUMAqeC4mMfkuKcO1jmuhloB1OYC0sM1iCZXBNPN7DvdBJpL3MjLCnhX7yWIw+Xng5Zk\\\\\\\/jmcaQefbdTU2X\\\\\\\/HK9qPgYsViaoiQ4VNNvxceWEHEM+gTa2Sr6lU12T337FKG2BA+gkeeHNKd3Mdesp68Dj9HhUP7QAIzBNhHwPYoPqClrHcg6zXOsHgVzXM7+R5c\\\\\\\/pV5ItfVxiZk\\\\\\\/uSlsWwBJBJFCfITjgsQ65dIjvTqc+18p\\\\\\\/ORZ4ONp9ddO5Ep\\\\\\\/7ruGspHY3xyvdeZwWKZ6nFUqoNYHOOuRX78KcU4mWDkPPJyNrPRQvvw9OCaNlSBIXc8l5LLPisnpn4oRI7LLIvwaPnlGaw9YbBycsL6C1fJXu7OjNvDt3Wh5L9v6HXgBJ3QW3XZYaE7GoSaP0DH2F8QEtd9vAnAHTGKnC+M1Co7vLX9kMrxEU\\\\\\\/wc77Lzb+UYNK1x5goblupWDP47p+r+5y6vIlIZIVjmqDr+lw72PI8ahkortudaRp3j4AXfZMnat8VAjSW7oWfimC+4GH+mPsWTy2KH8KoHeSgUuho+VQGoeU2\\\\\\\/FliJQx7j4qbFiSOjWABuP07XcXWd8kkBEsp7MJJC+NzeSzSQ+NBVqMCQ\\\\\\\/dg1r6BV8B1AlDemmrxcn5K1KxwKJsb4CCd4BmRiKw5CzymgJFvJa6kAWM3xGdhXsyCyN7U4ycg\\\\\\\/QX4J4sSHv+ZvNe7r\\\\\\\/zl7FHekf4QPXFKQlrz8w9cecDwDfv4AwMvb3nvz6WV88xicVJhnySuAI8uBY\\\\\\\/r5WpLySxWUGMfVnaxehlIaTaDMq3u\\\\\\\/3usar1QhjWxeGIlKCtFjJVbobLFRgpgBR3Q3XcS3KB+62o4km2N7IW2w4mWaTyDp9pJOwQSZq1oM3aZWzQrLkhoGkwPbdTYnzY\\\\\\\/y3onhZypMs6TbE\\\\\\\/0LShpP5ndjK2MUQVDjAaj6ltgVaU3QB9kpmYTR3hj6udpKFQBQEOR8a+UBxrscRvv+ukZGMw57JtZq7Kd6o7\\\\\\\/OB9MmKXiTGEHkuih8diF6ri9wdJv7Vw0+0LconIt7uP\\\\\\\/dmj6KTKbDFreCuUobCfPNP+GpGexAChlpXLXcUl0XGGF88A7g1GO+lCJNnL+d5bjUOf1lavHfFkBe9+TnKFZAQfLkBFfYrT\\\\\\\/odN2DdBa60rza2szKc+t9oBZllAyy+4vkduxqXARgD1tdLptAx6HdsbqqBXAdNeXjG2jV23WJjteWSam8klCJcHqi4QNy22B0nAiwXvfiMSyE9WHxudlmQj2U6zwECrCkANz6lJ42NUF+ViwOrtumOtvyjIS4Nekv9gRJb3to3vx6mNLnazAEDRwMQG053cPDJD7glil1EkqfC7+xxpZebsRVJXigP65FUbMOfUYtJLiaj53HFXK18bhu10bbBxmWClDUx1DMtaYKkX1lH3rDAlZZPRNt3L+E\\\\\\\/6WEoy686\\\\\\\/jvuNyRdC5LQl+azxFYu6\\\\\\\/28Z1ch5zwoDqe9m3cKfJY4F\\\\\\\/6MAV8BsuYo6eO+8eLLuz2tsW34Y4vTwOuFDpVk9\\\\\\\/o8VvAoKlh29mx17cnzRb41lB+yQg31MG9WaKk0LQQ3ISdR92racoO0GgljyTuDZJe+lJifgnCrw35RDuWwn8KwVTlapC6qB2o3WR\\\\\\\/tUN+saujnnC1rz8UNBX37bRbxrsml50AidvPKI44b\\\\\\\/baJJRzjFvf5fR4\\\\\\\/+7SN1VBm6bIXtjFQ+KZ\\\\\\\/ug2uC3wO\\\\\\\/d+6kpWd9LagqheOKl2yiSuEyBweCeSDrEQ0YQ9EZv9HamZeGwD43m0VjGYanE1aztJJYx6inQxL5xndqNATmaS3GUoRT0Qf23ulU2ov8xnAxqGTCXPdfsL4IqtgurmZZbxBIGTuY6HY9NxDzQ65dUeITDvT+Dyf6Ti8lHLN2u\\\\\\\/JJpPPBIjEbanccYNxZxh0FcWHxHoa7HfPBLGZlk+qq8\\\\\\\/2C2zCB0wRoJgeMb5f9Jb6E75VPA\\\\\\\/Vqh900NTSIN\\\\\\\/0AMJzzpgdCdtCjv8eXscreK9F2mJzrkJiVcEDcKhy4sKK0qNCfHAvlBzb7juGZc0ezhYNbfjRsqZo+93kDq1Mprobd5YE6fJpQP4UKdHybb\\\\\\\/ioO0Dc1f\\\\\\\/5MgINcsMye\\\\\\\/AFjedsBS2ExzsRNopqK4rKFi2I5tjp6eamMMlHl4sOUwriKMGn3+tqemL6tph7970yhhzAvmYZQMN2\\\\\\\/MpHTiYKdTWI1xpZIpMTRS+My2lGYCloA7eiu7ueX0nSzQInDawKHPZbJVP5YtfxfpgxJhtHbdngXWWZhZsuxC\\\\\\\/yHBY0LcRT2xO0NXZDWMLYGoX\\\\\\\/6tbMj56XJajfBOp\\\\\\\/gydlmvsPfhdTClb5VHICXkn10mVlWdDeLXw153plZOU8T2O\\\\\\\/cJcuq4QRjUezHrUXB66Xex\\\\\\\/EshQrvr973BoJdLawOJDBbr8Bl2cpAXxmkBIW8DuIrxANDc8FAOi5jjBHWquwINGVrr7MxTCwTKHwDPpuN5GVVw4E5EtIMsRA0g3pqNd0wpmULERKy0ryU4i1h6l\\\\\\\/2UOiJfP+g6Fq0Nc\\\\\\\/UYNMHgeemX+meaP7GCIwP1F9XeRhwf7kZ+7FMZ34YLYHk2WcF20sHNDMsclRpZlpQPsxAR75xi\\\\\\\/FRKfrE+zrEAEL\\\\\\\/XqYAcCitAZmBxhB1sfuOiUq2rJUEB\\\\\\\/TXKyP7Gt0\\\\\\\/QTanfnpe9+1ZodMw4zlTqN2\\\\\\\/XYg5dS8bacAqMB63MWn2ECla9R+te+ox80seUAQHk9eUYmCWXRL5rjW21rmTAHbq+22ZVkjuUyK2TOxv2nyUOgmeLp\\\\\\\/Fdu8TC+SmHbgzBrf\\\\\\\/\\\\\\\/xcQCfbOBfOakUHN44mjwsdI8fuBMxuL61TbbMHezxk0LZ5td2oQUy2IPk6iFfR8QZABBQ0DbS1IY\\\\\\\/dNbDcebsKIGDL+wZZgpM6DiIWcf70IoNx7FukfsZCyyVilNUy6LdF5J2Fp9kMx8OmuLoXLDxO6Ga0UVWyQJyI+OJ1rGaMcAZLVt8XjbY5oCwHU70sLj6J84po1wTIcJ1\\\\\\\/3IpdgqaBdFFko0nUeg+KlY1CbR37+2u0upE5vc7nCQEfrybV2fjdTYdE0liYQFyXXBg\\\\\\\/bzpvM+qhhB6gGOgDmaCMVUOVHNVAD8F0jqp60na4dpYm00nv\\\\\\\/zR9m2i71uLl02vfMqH0ijnj+DvjMNjFD6eCkFUv9qsIYWxZAyUyqZHZk\\\\\\\/51fr4oiT\\\\\\\/gPIwMoU5yWuXuT4zUWZEoc+64U0LBMNKYqtQl\\\\\\\/uibhkhMOVVVL3qAi7Jmvy+YAboXH3+s2ihAmLcTP6U3vi5sWQAIKbvENOt6bX0DfmPjLuAHfyWMuRz39BIyGW0TLYg8r9yKfkuK3NVKHHLs52mikm+enP4\\\\\\\/f8nAmC7aWr4d2QtF\\\\\\\/8zhbDBLP09L3zQU6YzKo6L+vw8PGf5flWRUadMpd\\\\\\\/zLEoG43uk\\\\\\\/uaoFN1V+XWlELedSTACTKLHWG+gRVvT\\\\\\\/iTwKqO9+BeospL4xACCaDlSWNmZlS+47o0dakk7OJ1iVzrbikimiRQ4E7vbBkfZQ19TsPP+LIRFOLX7hAyA+bffWSPRQab4aTyDUo37Slj87IscLFPbHGl+CHSKOM2oN5HOYmKCKclH7\\\\\\\/B5+rw1AYXLKWyJ\\\\\\\/9r6Qr4UtDf6LnS\\\\\\\/M42qPUtdu6aUiV1Dc2hFPhdQarpwkqyTrkGjBx2kW4Klh\\\\\\\/811EKjCu7IYGPeon1mcUyX0+qvgvwrtHmcZx56fP0mSPwzHje5z7rJV3qPTc5EzzwR8akYpCbIhVgt4SoYeVBBzyp\\\\\\\/SMu0fjmL1cNct\\\\\\\/JMhJdFsY6Z0e7vx5DLDL4jT\\\\\\\/Ceuhfv6segdTmdcQtguUuSyVfHMACnq2PAezFVNXqvk6WnsOyobaWMXwCYtJty4jEmh7WTID7EQ8mRabpJr0rGjxDEKiuP7h1AvKIF7s6xofFCJtEH1wVREzAVV7LQMHyqmX2whUv32hmXpgKoAPf1v96B0x8o+t1XAlVkFzXo2MRkc7FGlRQFIp1qm6ymIpHwwwzTmqz3k0Yv2ba472\\\\\\\/auvBraNhF+wcIqLqu7Vfo9uUQrXeao+\\\\\\\/XuAZSyIwBGubz6S\\\\\\\/pxujtLNyLMOJXX1f5zwPuuRBkCL8tqehodLT5GsPt0PIVy6isOWnCPsQ+nFIirVDvd7eeaBTotQpCy3R6eaEsSkO3VUquW7cR5hMndnLNZx6ER18pYFICDi2w\\\\\\\/nrxWz0FYMpad9hYQC5f1rwWzuiIoqdTAUx5KBPaB2GVsAMFmPDYgmsa\\\\\\\/wk5KT0F4jQurPwdBnhhSmTPzgvrXTIT00QvLPm\\\\\\\/2vLTjQgGH0EC3XETB5YipgmBgjHLOMo\\\\\\\/UuSvwkHeN26y245pVGCedsdZtYnJyjNELSjdgFg\\\\\\\/IWnGpuN0SWr3p1\\\\\\\/CkHe8SHYcBSj\\\\\\\/VwDmnYLmmeYbqeAQjKmnB1RZJ8XAdR\\\\\\\/wOll8Iiab0g+PSuNdpxPjG9M\\\\\\\/Zb4nyPcw9mAGuIGCg7bBDUxQmk5bDeN96i2fLP\\\\\\\/pM1P7mD5yvt76NdIDzn7i16NkSQD5Z\\\\\\\/KAa4n3JiIDEQ8Lq1LHdAaZM9oWiihYeSVygS9QwZL3sBldXRxD1LjinM3KVTN\\\\\\\/Rjbvah+PTCfMuHSdLeEu82P\\\\\\\/jy1MyP1YGOk7TC54jFUXz7Vd6veXv5jIBfn6FShy5rroNPN4DMk2LX4QBFTHge1HBV2Y4t1VYQEL5RhpH4OYaUAX6l3ID+lmlOySYkUsplFig0o4RifMaZIr7xkhqn2LX7oHyUlWyklD5r9QOUSKpdQmJsVYuZxf7n64gzdvtuOt1RkjtMcBnRW1\\\\\\\/ok+51iFDvbpJS4MAUyRHm02PB\\\\\\\/qx0fhy9Tk04fqep95B5O0EwR1D0qhBaB6aSOkr8oU3aMdzad9e3ZDD6pLDttC8oT1gOnc3BaRaU3iJw9bXfYqKBDju5SAyaM2yeyoEo6+MVSLJbcq3t\\\\\\\/KwMsI4NhPswgU3yQ\\\\\\\/se40tcZ8cbpVbeTAG7tcKLmGsQPQ\\\\\\\/sWZttkkROmXopC3Cp39haewS4FbWfhBPcOt2B43ZodxebXIVGMrw4GL14F+FZRPplSN8YtQzLrGysZLikuz7HdiZYMz5iDdeIkLtB2\\\\\\\/cavzwby+wlfqSHTWZTJ\\\\\\\/Tp6PkIUWrAfftkaHfj5tge\\\\\\\/uAv0QBRNwQDfIvN6YPElI7Wrbem1Om4QraqIsCdHvXZ7mlg82uhzFfRtaTasdt9Cm++fzqFLoQ3Azds3uSGaRMec16Su1jeOzXGqf8z4RTW0RR+2C4xIZtH8OSiulGLGcvU7jKqtLThDCHWnbGdvh67A++6mFG127iGRddoxazPVMKQCzdXvvJ4fPvm+btZXZDGDdQGZX9MD4Mp2sT0RsokO9Kbd3IDM66MoQY9ERrktjsM9sYgv6B0kiSotWioOyYMNPe91BFODTdRMTyMQQoPmbVt1UqT8K4QMerAfGU357wNQpLjPwjzCkk19DrSxHMKV5f99JEeoLqGlrxmJlOOZshsrtbWTadUqIMRuGZkGdTwRkf4DXT1b5OMNvNOetXKUxOMk32f1\\\\\\\/T7CEiv6iyImq24h0mcC1\\\\\\\/haAwutX+reSORU+D+shujJ46mu4fHZan8ijuuphfg4ncmPyUAOzn\\\\\\\/H3u4LZlgSUmhg02b3tKDUot0dq8WpMGITV\\\\\\\/yvu\\\\\\\/eNu25hfOAVPjdUE2oBw8whL6SFp3s7XCgnbXCLxHvkoF1tabiezOH1EWuc4OIZU6f07puVZaexpgTfre7zIDlNSmrsVefr7g9JRVK\\\\\\\/9AAOHfmNx00+7LgZ0RCM5oRk77AiKXivjngHVl9jVw0VfS\\\\\\\/pwjEoeIcnJg4gC4w5o0xH1naGVRuUZcAvrm\\\\\\\/AyXtQr0oIcxGQrSQJzVmAUYguWLAuDAX5c2vhzdhe6e6lbk7NcFg5KXm8arZqnh5T5CioFdHBzmcVvivLQwS9XjgBJyj9TOENzarU0uIlwe3dJVdikKkRUDMgaRbDef4F9bkvsKS5RAnFN1gwOJdW4HN2LgCSAWg2IGZjDHXLU8JfxID0vNr7PsIiWbNTePX7ArRlccK+C04knh3m4+Qmz9ebVoCuCiAej9PODtSbJar0edzmS24mhJ1rw1YGDJND8M+F2l0naXBwuUEg0STMkqMQj1MGcCNV\\\\\\\/giireX5eReKNu9dPfjP4J6ObCczMbcxzJzLs7NeM9SytnZ4Z491asbG9NNmRqLGkTSvLS3Rz1URn9heScKiB8S6G0S9FAf4MbBjBBajsf8vOKILw7MdZq9vgszrIRz8vAwyDc6WeaGu5t5j6Kjv1DSe\\\\\\\/w0xFXCVidEW2zWDOgzdnKIpJIjPyU7HR+7C53M\\\\\\\/+5kIjZ4rsvW44Mvo3hVfjQ62XGadzFZXNgqIiJWWSdt7Ilq6r+9dDmBwn2xrV6XJa3nmxQEe2v0k6keobd8WDfj+JZVQ6floqpuAi39HwbHQwVsa57irhbsYN6c3PRWuX5BJwusAyzWytFkMpD8zHSHVvG3QlcQF5xlLUHQuw4\\\\\\\/jp3cVQr\\\\\\\/+V2Nd19fozDwr5\\\\\\\/XvsEbsEwLHzAnkXU4UX2wCTftcB2sqv2XRk8esXEiGRBbPBXgPmeWw\\\\\\\/JP4wDVr1gsZrCgMdmp4Y2O1t7sC2x8hoW5XrAAg1wHCJggFOmejak0w8mb6B4ADl8oD2sC9tJEEZ8bRIvL8eFFIA00XIJJlGWldLF4qsZg4dVK7PxMtuxi\\\\\\\/lSqmL53b0hnJnyrU6Kuj\\\\\\\/iWX2liBiCcWkBLJfF5FCsVHWdnjj6\\\\\\\/rmoOW6hyLpO5a17RFW\\\\\\\/hJLYT7fSgK2xzfh9tWbOQtxhjl6QeUUMsjPouwXSc0yhCcs2N40\\\\\\\/oUOVdgEym2C8kjKEQM9005JEQ5KBek2Z1lxntVPzrXekaX8AphUsFPRvq1hhFwqjEEZrprvW023mLXg\\\\\\\/GJmP+fuUeZBKJTiFOTifMBmsvXHnJUJEC5P\\\\\\\/hwAwKfdEyCpQ\\\\\\\/F3UCsLTLraexYzB\\\\\\\/1n4CUeoSFW8E04VUqZM6x1aHSDWXV5\\\\\\\/+uA\\\\\\\/r7lRrfi1EyTr7TqeIaQiDiToCZrwYu1Ud65dj\\\\\\\/3Guw1Kf+PL3x5+UbsVtRlTGywvAT7lnH\\\\\\\/tL30GmTWtPbu7ZDXIbNiGSLhJQQjXG+itt0J\\\\\\\/r1pP8ssXjAjPR0lxcQw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022xXJgcaK4q51dZ8PyyIusjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225f3a2a1dc049268c2fd4eaea866de5ebe681df60ce2b9097c1b7d3bed8bbf351\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022a4jQ2PWahk4RklebSPc5m0YluWSOhnx35fdwTz3PcxhPXxZcKnGC7VtfoogvXUSYjXTz\\\\\\\/v+Gbr\\\\\\\/2SoCTbx1k4vgkoIgNk8klJ+QGlbicK9zxk7od8YsgchGts5iKW6wVffgdMP1J6ebjCHCxi7vzKbKYrKEakHksORRUUTdoSzpjm2PJwARroDbi64ktchYkBtJOSkM2fwzkZQgi3TIivIgLI7YPqXSFnvqgFD1MG7I9eES3BiDu3VWlfH5VdrxeZKzrT5G4D+ROrx52VRMy3\\\\\\\/f0Od4dRuZF9R9GQFXW+ZlfYq3up1XzzBpdYo05vTgXkKwfd4oNeToDqT7Q9Q4a5t+kvvdWEAR\\\\\\\/wIK14k9Std\\\\\\\/2Ocf4zgWui0+iDvF4HPR1o9GYS3IRv16j5EsXB4lj2eerxDDy0Tni\\\\\\\/MTPCHcrQk465tGKdKVKlaZMXNnf\\\\\\\/nihAHOfwG2FAyv0g+K4splqdR\\\\\\\/EmlAduSgZW9iIDo2gxB\\\\\\\/uAR6hrEEDDDHV9QIS7QZRdwcKpsn7tszDsSL3VUJm6\\\\\\\/C73Pvf70rCQ29Tt03SfGglkMIHt0lhfVjcKad7w4XLHxJhCWCU46EUKTfg9ZfWxlnMH6sYIEW+2h1dzj4DjfuToQIfgk0CnYQ2TTqkk39IqMBKrMngk5oQ2ofKGFuiDj\\\\\\\/Lk7aNad20Dfl57p\\\\\\\/OyCADohyB1+MPeHYx09BtQF7eogOBVNzi8kQPVGX99ZP65CDbSq9HNDAFTEiflAZNb64p0Z3sWesiuRyOJ65NfAO7kvsqzy9bhYZD3kUvqeimNrS+A957gO68uX4cT2Y1d8RTn5xH1ouaMInfAmNPgXC+WWbklksLcFJUZXWWJztbgTZI0E5BM3+yFFUa\\\\\\\/XgIM13QfcbJ4wH\\\\\\\/n0vuQegHm7XR2f5HhvIGIq0r7P\\\\\\\/4LF1KGQhqmlJXaf3OzQjpuzmWfUBsAgS0wbIy5i+JPW0uXBSTU5NzSs95LaDQUFUsXPs5pLy5voJya9Wf1h40SBZzeOpRewp60HC\\\\\\\/W6hH3NpoAGrbNmpBYjYaLBwIw\\\\\\\/wjQjfwLKbrM9KIE4sJybn\\\\\\\/J107nw5JzpUrUYAo2LCkdNCDdgNY4pomwKoaiGoKpDSNZ8fvKYRWiI\\\\\\\/18WNc3EheMEc2TyQuTJQ5kp8BuTacLEdE\\\\\\\/+mN3pnWGNoEd\\\\\\\/IM7Ue3hKdjl1RrBwHFupYxjUCLgYlSIfzzxZ0lEGgBfr\\\\\\\/mfx4bKz6XrAQXKk6DYwwilxs\\\\\\\/iAkkgUYtx24\\\\\\\/7hw1blwTNqW8MbT0CyEImWfB+MTK0tI2PD\\\\\\\/R13ILUdIbpVKXjlH6PvyAlzYyM0yog8tPn7cLa8ispe8fIiEpuNzWMdgFlJsRpWC18mNG8vZZg8Lj+SvyJWx6a0z\\\\\\\/HHqWZFb7R\\\\\\\/a1PV4zgICOfErW87EOtoAiyon+JWQi\\\\\\\/Xnw03RSe4wql0zWV2dM8XgEn7QlMhG64SRemwMInVZEOCuU1UksLBnpE1os0RH+2+PETblnBPt8DQHspJ3fkWeJsVPpxIZ+RsX8zlGudymVwmWNe2VbVnFA4ize2Hs6OeyrptANwVh0tFhghUtQk\\\\\\\/f2ULFOcJ2ne1zU9IjgdA\\\\\\\/sF1FkvU\\\\\\\/\\\\\\\/oFX51dV7Z+HYYmDfTIn6nzvUfe\\\\\\\/5g0EQd+\\\\\\\/Peb0ZR9+J0blRIEtqeJLO7bo29zGwBl1KSfLOr037ure5vIjS2m0lTDcgKL9RGlCFq1vjDptYRYZAnSWodsG2TNscEKZhtySYc4WuK7+XTWhPeWN6Clgywzpj9vkLAtjFJXKviLExB9QY0WaiYr6j5GaLE3k3\\\\\\\/4qLhtywCJ0w7xK+UPeQecoVLEoiDyiZgZ1PcFoK7vu+zLlFDFkwr0zHz1NbAcb5+msmifJ45m797z4bT6xPKH+J\\\\\\\/hCv9kY+rL00V4ZB2x6iLe5dtMMASaZarswt9TCuu9gs2reTAyyRGz1iIqjrWH1mr23hacGLPz43T4nlPCwP5gsATk6mnRHmquvUHI4ZiT0Fs0\\\\\\\/hWNMMmQMjd82Z0fCtBF0O\\\\\\\/bsGNNAzDFfk0\\\\\\\/SUU\\\\\\\/jKEa805E0\\\\\\\/\\\\\\\/mZpmtIyYwUeRxJ9vsrHDHU9KHpNEw+9onhEjyNLAZ9zBL1kVlvL8KQjWrTogN2cIp7V4ouPBOJuqlID5qGur+8x3DIdNYB76K\\\\\\\/dpLxDMg==\\\u0022,\\\u0022iv\\\u0022:\\\u00224eUYnOiiVTQubLuRtfZfcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022300a1772dab15e994f7d68251e3e219d06f5a06a597eb28818cbd62eaedec135\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459806 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/477" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YS9QHs5lRQ1zQZ0kHawyCKrchx4HvDxXLd8gs6-eJO0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/477" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PSlCxg5WyYl0vcDyPe6aOWnTxrPajegWqyf62Bj7OVY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/467" title="Identification of Nonviable Genes Affecting Touch Sensitivity in Caenorhabditis elegans Using Neuronally Enhanced Feeding RNA Interference" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/487" title="Identification of eQTLs for Hepatic Xbp1s and Socs3 Gene Expression in Mice Fed a High-Fat, High-Caloric Diet" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459806" id="node459806" data-pisa="ggg;5/4/477" data-pisa-master="ggg;g3.115.016725" data-apath="/ggg/5/4/477.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6256-5478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Varona</span>, <span class="highwire-citation-author" data-delta="1">Sebastián Munilla</span>, <span class="highwire-citation-author" data-delta="2">Elena Flavia Mouresan</span>, <span class="highwire-citation-author" data-delta="3">Aldemar González-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Carlos Moreno</span> and <span class="highwire-citation-author" data-delta="5">Juan Altarriba</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">477-485;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016725 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459806" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis Varona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVarona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varona%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BVarona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6256-5478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Varona</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lvarona@unizar.es" class="" data-icon-position="" data-hide-link-title="0">lvarona@unizar.es</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sebastián Munilla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Producción Animal, Facultad de Agronomía, Universidad de Buenos Aires, 1417, Ciudad Autónoma de Buenos Aires, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A1n%2BMunilla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munilla%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A1n%2BMunilla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elena Flavia Mouresan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BFlavia%2BMouresan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mouresan%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BFlavia%2BMouresan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aldemar González-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldemar%2BGonz%C3%A1lez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Rodr%C3%ADguez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAldemar%2BGonz%25C3%25A1lez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos Moreno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BMoreno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreno%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BMoreno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Altarriba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BAltarriba%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Altarriba%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BAltarriba%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/477" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459806" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459806/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/477.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459806" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459806/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/477.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459806" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459806/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/477.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459806" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459806/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/477.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459806" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459806/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Epigenetics has become one of the major areas of biological research. However, the degree of phenotypic variability that is explained by epigenetic processes still remains unclear. From a quantitative genetics perspective, the estimation of variance components is achieved by means of the information provided by the resemblance between relatives. In a previous study, this resemblance was described as a function of the epigenetic variance component and a reset coefficient that indicates the rate of dissipation of epigenetic marks across generations. Given these assumptions, we propose a Bayesian mixed model methodology that allows the estimation of epigenetic variance from a genealogical and phenotypic database. The methodology is based on the development of a <strong>T</strong> matrix of epigenetic relationships that depends on the reset coefficient. In addition, we present a simple procedure for the calculation of the inverse of this matrix (<strong>T<sup>−1</sup></strong>) and a Gibbs sampler algorithm that obtains posterior estimates of all the unknowns in the model. The new procedure was used with two simulated data sets and with a beef cattle database. In the simulated populations, the results of the analysis provided marginal posterior distributions that included the population parameters in the regions of highest posterior density. In the case of the beef cattle dataset, the posterior estimate of transgenerational epigenetic variability was very low and a model comparison test indicated that a model that did not included it was the most plausible.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/epigenetics" class="hw-term hw-article-keyword hw-article-keyword-epigenetics" rel="nofollow">epigenetics</a></li><li class="kwd"><a href="/keyword/bayesian-analysis" class="hw-term hw-article-keyword hw-article-keyword-bayesian-analysis" rel="nofollow">Bayesian analysis</a></li><li class="kwd"><a href="/keyword/genetic-variance" class="hw-term hw-article-keyword hw-article-keyword-genetic-variance" rel="nofollow">genetic variance</a></li><li class="kwd"><a href="/keyword/resemblance-between-relatives" class="hw-term hw-article-keyword hw-article-keyword-resemblance-between-relatives" rel="nofollow">resemblance between relatives</a></li></ul><p id="p-3">Epigenetics studies variations in gene expression that are not caused by modifications of the DNA sequence (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Eccleston <em>et al.</em> 2007</a>) and is now considered as one of the most important fields of biological research. Several biochemical mechanisms that alter gene activity (DNA methylation, histone modifications, etc.) underpin epigenetic processes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jablonka and Raz 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Law and Jacobsen 2010</a>). A number of studies have emphasized the importance of epigenetics in cancer (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Esteller 2008</a>) and other human illnesses (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Feinberg 2007</a>) as well as in other mammal (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Reik 2007</a>) and plant traits (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Henderson and Jacobsen 2007</a>). Gene activity modifications may occasionally occur in a sperm or an egg cell and can be transferred to the next generation, denoted as transgenerational epigenetic inheritance (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Youngson and Whitelaw 2008</a>), a phenomenon that has been reported in a wide range of organisms (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Jablonka and Raz 2009</a>). Despite this, the magnitude of the phenotypic variation that is explained by epigenetic processes still remains unclear (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Grossniklaus <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Heard and Martienssen 2014</a>).</p><p id="p-4">From the perspective of quantitative genetics, the presence of transgenerational epigenetic inheritance involves a redefinition of the covariance between relatives. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a> developed a model for the calculation of the covariance between relatives for asexual and sexual reproduction as a function of epigenetic heritability (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-1.gif"/></noscript></span></span>), the reset coefficient (<em>v</em>), and its complement, the epigenetic transmission coefficient (<em>1 − v</em>). According to these authors, the covariance between relatives is reduced as the number of opportunities to dissipate (or reset) the epigenetic marks increase. Therefore, for sexual diploid organisms, the covariance between parent and offspring is greater than the covariance between full sibs, and the covariance between half sibs is greater than the covariance between an uncle and its nephew, despite the fact that the additive numerator relationship between them is identical (0.5 and 0.25, respectively).</p><p id="p-5">In animal and plant breeding, the standard procedure for estimating variance components is by means of a linear mixed model (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Henderson 1984</a>) that includes systematic or random environmental effects, one or more random genetic effects, and a residual. The variance components are estimated by using likelihood-based procedures (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Patterson and Thompson 1971</a>) or Bayesian approaches (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gianola and Fernando 1986</a>). Under the Bayesian paradigm, the standard procedure for determining the posterior distribution of the variance components uses the Gibbs Sampler algorithm (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gelfand and Smith 1990</a>) that involves an updated iterative sampling scheme from the full conditional distributions of all the unknowns in the model.</p><p id="p-6">The aim of this study is to present a Bayesian linear mixed model that allows one to estimate the heritable epigenetic variability, the reset coefficient, and the epigenetic transmission coefficient based on genealogical and phenotypic information. The model is illustrated with two simulated datasets and with one example that considers a beef cattle database.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Statistical model</h3><p id="p-7">The standard mixed linear model is described by the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-2.gif"/></noscript></span></span>where <strong>y</strong> is the vector of phenotypic records, <strong>b</strong> is the vector of systematic effects, <strong>u</strong> is the vector of random additive genetic effects, and <strong>e</strong> is the vector of residuals. Then, <strong>X</strong> and <strong>Z</strong> are the matrices that link the systematic and additive genetic effects with the data. The usual assumption for the prior distribution of <strong>u</strong> and <strong>e</strong> are the following multivariate Gaussian distributions (MVN):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-3.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-5.gif"/></noscript></span></span> are the additive genetic and residual variances, respectively, and <strong>A</strong> is the numerator relationship matrix. Further, the prior distribution of <strong>b</strong> is commonly assumed to be a uniform distribution. Conjugate priors for the variance components are the following inverted chi-square distributions:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-6.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-8.gif"/></noscript></span></span> are the prior values of the variances and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-10.gif"/></noscript></span></span> are their corresponding prior “degrees of belief”.</p><p id="p-8">To estimate transgenerational epigenetic variability, this standard model can be expanded to:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-11.gif"/></noscript></span></span>where <strong>w</strong> is the vector of individual epigenetic effects. Note that the incidence matrix (<strong>Z</strong>) is the same for <strong>u</strong> and <strong>w</strong> and that both genetic and epigenetic effects are assumed to be independent. The prior distribution of <strong>w</strong> is defined as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-12.gif"/></noscript></span></span>where <strong>T</strong> is the matrix of epigenetic relationships between individuals and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-13.gif"/></noscript></span></span> is the transgenerational epigenetic variance. As before, the prior distribution of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-14.gif"/></noscript></span></span> is defined as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-15.gif"/></noscript></span></span>with hyperparameters <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-17.gif"/></noscript></span></span>.</p><p id="p-9">The structure of the <strong>T</strong> matrix is defined by the recursive relationship between the epigenetic effect of one individual (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-18.gif"/></noscript></span></span>) with respect to the epigenetic effects of its father (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-19.gif"/></noscript></span></span>) and mother (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-20.gif"/></noscript></span></span>):<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-21.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-22.gif"/></noscript></span></span>As defined by <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a>, <em>v</em> is the reset coefficient and (1 − <em>v</em>) is the epigenetic transmission coefficient. The reset coefficient represents the proportion of epigenetic marks across the parental genome that are expected to be erased, whereas its opposite, the epigenetic transmission coefficient, indicates the proportion that are transmitted. Furthermore, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-23.gif"/></noscript></span></span> is the residual epigenetic effect of the <em>ith</em> individual, independent from w<em><sub>i</sub></em><sub>,</sub>, and whose distribution is:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-24.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-25.gif"/></noscript></span></span>assuming that the variance of transgenerational epigenetics effects (V()) is constant across generations:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-26.gif"/></noscript></span></span>In matrix notation:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-27.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where the <strong>P</strong> matrix defines a recurrent relationship with the epigenetic effects of the father and mother. For nonbase individuals, the <em>ith</em> row of the <strong>P</strong> matrix contains a parameter λ in the column pertaining to the father and mother of the <em>ith</em> individual. The rest of elements are null.</p><p id="p-10">Furthermore if:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-28.gif"/></noscript></span></span>then<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-29.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-30.gif"/></noscript></span></span> is a diagonal matrix with entries equal to <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-31.gif"/></noscript></span></span> for base individuals, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-32.gif"/></noscript></span></span> for individuals with one known ancestor, and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-33.gif"/></noscript></span></span> for individuals whose father and mother are known. The prior distribution for λ will be assumed to be uniform, between 0 and 0.5.</p></div><div id="sec-3" class="subsection"><h3>Parameter estimation through a Gibbs sampler</h3><p id="p-11">The Gibbs sampler algorithm is an iterative, updating sampling scheme that obtains samples from the marginal posterior distributions of all the unknowns in a model. It requires samples from the full conditional distributions of all the parameters. In the proposed model, the set of parameters can be classified into three main groups: i) the location parameters (<strong>b</strong>, <strong>u</strong>, and <strong>w</strong>); ii) the parameter λ associated with matrix <strong>T</strong>; iii) the variance components (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-34.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-35.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-36.gif"/></noscript></span></span>).</p><p id="p-12">The full conditional distributions for these groups are:</p><div id="sec-4" class="subsection"><h4>Sampling of the location parameters (b, u, w):</h4><p id="p-13">The conditional distributions of the location parameters are univariate Gaussian (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 1994</a>), with parameters drawn from the following mixed model equations:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-37.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-38.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-39.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-40.gif"/></noscript></span></span>.</p><p id="p-14">Specifically, the full conditional distribution for the <em>ith</em> location parameter (s<sub>i</sub>) is:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-41.gif"/></noscript></span></span>given the multivariate Gaussian nature of the conditional distribution of the location parameters.</p><p id="p-15">A key limitation of the procedure is the calculation of the <strong>A<sup>−1</sup></strong> and <strong>T<sup>−1</sup></strong> matrices. Whereas <strong>A<sup>−1</sup></strong> is calculated by the standard Henderson’s rules (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Henderson 1976</a>) only once, the <strong>T<sup>−1</sup></strong> matrix needs to be calculated afresh in each cycle as the parameter λ is updated. An algorithm to set-up the <strong>T<sup>−1</sup></strong> matrix is presented in the appendix of this work (<em>see <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix</a></em>).</p></div><div id="sec-5" class="subsection"><h4>The full conditional distribution of λ:</h4><p id="p-16">The conditional distribution of λ is developed from the recursive definition of the epigenetic effects (equation 1). It is a truncated univariate Gaussian distribution (TN) between 0 and 0.5 due to the prior distribution of the parameter.<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-42.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-43.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-44.gif"/></noscript></span></span>where <em>n<sub>1</sub></em> is the number of individuals with known fathers and mothers, <em>n</em><sub>2</sub> is the number of individuals with only father known, and <em>n</em><sub>3</sub> is the number of individuals with only mother known.</p></div><div id="sec-6" class="subsection"><h4>The full conditional distributions of the variance components (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-45.gif"/></noscript></span></span>):</h4><p id="p-17">The conditional distributions for the variance components are the following inverted χ<sup>2</sup> distributions<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-46.gif"/></noscript></span></span>where <em>nan</em> is the number of individuals in the population, <em>ndat</em> is the number of phenotypic data, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-48.gif"/></noscript></span></span> are the hyperparameters for <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-49.gif"/></noscript></span></span>.</p></div></div><div id="sec-7" class="subsection"><h3>Simulated datasets</h3><p>To evaluate the procedure, we simulated two different datasets. The first one had a relative high percentage of variation caused by additive and transgenerational epigenetic effects (35% and 20%, respectively) and an intermediate reset coefficient (<em>v</em> = 0.40). In the second dataset, a lower percentage of variation was explained by additive genetic and transgenerational epigenetic effects (15% and 10%, respectively) and the reset coefficient was greater (<em>v</em> = 0.80). Each dataset was composed by a three-generation population. The base populations included 3000 individuals (1500 sires and 1500 dams) and each of the two subsequent generations were composed by 3000 full sib families of 10 individuals each one. The sire and dam for each family were sampled randomly from the individuals of the previous generation. The genetic and epigenetic effects for each individual (<em>u</em><sub>i</sub> and <em>w</em><sub>i</sub>) in the base population were generated from the following Gaussian distributions:<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-50.gif"/></noscript></span></span>Furthermore, the genetic and epigenetic effects for the individuals (<em>u<sub>j</sub></em> and <em>w<sub>j</sub></em>) of the second and third generation were obtained by sampling from:<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-51.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-52.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-54.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-55.gif"/></noscript></span></span> were the additive genetic and transgenerational epigenetic effects of the father and the mother of the <em>jth</em> individual, respectively. In addition, one phenotypic record (<em>y<sub>i</sub></em>) was generated for every individual by:<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-56.gif"/></noscript></span></span>where μ was a general mean set to 100 units. The values of the parameters for the two datasets were:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-19">Dataset 1.<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-57.gif"/></noscript></span></span></p></li></ul><p>being</p><span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-58.gif"/></noscript></span></span><ul class="list-simple " id="list-2"><li id="list-item-2"><p id="p-20">Dataset 2.<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-59.gif"/></noscript></span></span></p></li></ul><p id="p-21">The Gibbs sampler was implemented using own software written in Fortran 95. The analysis consisted of a single chain of 1,250,000 cycles, and the first 250,000 were discarded. Each analysis took 36 hr using a single thread of an Intel Xeon E5-2650 of 2.00 GHz. The source code is included as Supporting Information, <a href="http://www.g3journal.org/content/suppl/2015/01/23/g3.115.016725.DC1/FileS1.zip">File S1</a>. Convergence was checked by the visual inspection of the chains and with the test of <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Raftery and Lewis (1992)</a>. All samples were stored for calculating summary statistics.</p></div><div id="sec-8" class="subsection"><h3>Pirenaica beef cattle data</h3><p id="p-22">As an illustrative example, we used a database of phenotypic records from the yield recording system of the Pirenaica beef cattle breed. The Pirenaica breed is a meat-type beef population from northern Spain with an approximate census of 20,000 individuals that are typically reared under extensive conditions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sánchez <em>et al.</em> 2002</a>). The data set was made up of 78,209 records for birth weight with an average value of 41.52 kg and a raw standard deviation of 4.65 kg. In addition, a pedigree file including 125,974 individual-sire-dam records was used. This information was provided by the National Pirenaica Breeders Confederation (<em>Confederación Nacional de Asociaciones de Ganado Pirenaico</em>; <a href="http://www.conaspi.net">http://www.conaspi.net</a>). Animal Care and Use Committee approval was not required for this study as field data were obtained from the Yield Recording System of the Pirenaica breed; furthermore, data were recorded by the stockbreeders themselves, under standard farm management, with no additional requirements.</p><p id="p-23">The full model of analysis was:<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-60.gif"/></noscript></span></span>Where <strong>b</strong> is the vector of fixed systematic effects that included sex (2 levels) and age of the mother (16 levels), <strong>u</strong> and <strong>m</strong> are the vectors of direct and maternal additive genetic effects with 125,974 elements, <strong>p</strong> is the vector of permanent environmental maternal effects (21,143 levels), <strong>h</strong> is the vector of the random herd-year-season effects (12,925 levels), <strong>w</strong> is the vector of transgenerational epigenetic effects (125,974 levels), and <strong>e</strong> is the vector of the residuals. Furthermore, <strong>X</strong>, <strong>Z<sub>1</sub></strong>, <strong>Z<sub>2</sub></strong>, and <strong>Z<sub>3</sub></strong> are the incidence matrices that link the different effects with the phenotypic data. Appropriate uniform bounded distributions were assumed for the systematic effects and for each variance component (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-61.gif"/></noscript></span></span>), defined by hyperparameters (<em>n<sub>x</sub></em> = −2 and <em>s<sup>2</sup><sub>x</sub></em> = 0). Furthermore, the prior distribution for the (co) variance matrix between direct and maternal genetic effects (<strong>G</strong>) was the following inverted Wishart:<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-62.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-63.gif"/></noscript></span></span>being <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-64.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-65.gif"/></noscript></span></span> the direct and maternal genetic variances and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-66.gif"/></noscript></span></span> the covariance between them. In this case, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-67.gif"/></noscript></span></span> was set to −3 and <strong>G<sub>0</sub></strong> to a 2 × 2 matrix of zeroes to define a uniform distribution.</p><p id="p-24">Two statistical models (I and II) were fitted. Model I includes transgenerational epigenetic variance (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-68.gif"/></noscript></span></span>), whereas Model II does not. For each model, the Gibbs sampler was implemented with a single chain of 3,250,000 cycles, and the first 250,000 were discarded. Convergence was checked by the visual inspection of the chains and with the test of <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Raftery and Lewis (1992)</a>. The models were compared based on the pseudo-log-marginal probability of the data (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gelfand 1996</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Varona and Sorensen 2014</a>) by computing the logarithm of the Conditional Predictive Ordinate (LogCPO) calculated from the Markov chain Monte Carlo (MCMC) samples. It was calculated as:<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-69.gif"/></noscript></span></span>where <strong>y</strong><sub>-i</sub> is the vector of data with the <em>ith</em> datum (y<sub>i</sub>) deleted, <em>M<sub>k</sub></em> is the <em>kth</em> model and<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-70.gif"/></noscript></span></span>being <em>Ns</em> is the number of MCMC draws, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-71.gif"/></noscript></span></span> is the <em>jth</em> draw from the posterior distribution of the parameters of the <em>kth</em> model.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><p id="p-25">From the perspective of quantitative genetics, the estimation of variance components is obtained from the statistical information provided by the covariance between the phenotypes of relatives (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Falconer and Mckay 1996</a>). <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a> proposed a simple model for the resemblance between relatives under transgenerational epigenetic inheritance for asexual and sexual reproduction. This model assumes that transgenerational epigenetic effects are not correlated with the additive genetic ones, and that they are distributed under a Gaussian law. After the central limit theorem, this may be explained by a large number of epigenetic marks randomly distributed across the genome. In addition, the distribution pattern of these marks is assumed independent between individuals unless they are relatives. In that case, some of these marks should have not been erased during the meiosis drawing them apart from their common ancestor. The population rate of erasure of these marks is measured through the reset coefficient (<em>v</em>).</p><p id="p-26">In this study, we propose a procedure that makes use of the aforementioned authors’ definition for sexual diploid organisms to estimate the parameters under a Bayesian mixed model framework. Our approach makes it feasible to estimate transgenerational epigenetic variability from huge datasets of genealogical and phenotypic data. The key to the procedure is the definition of a <strong>T</strong> matrix of (co) variance between transgenerational epigenetic effects. This matrix exclusively depends on a single parameter (λ) which is directly related to the reset coefficient (<em>v</em>) put forward by <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a>. As an example, for a simple pedigree of seven individuals (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), the <strong>T</strong> matrix is:<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-72.gif"/></noscript></span></span>As <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a> stated, if we compare the covariance between full sibs (individuals 5 and 6) and the covariance between sire-offspring (individuals 1 and 4), the covariance is greater for the sire offspring pair, λ <em>vs.</em> <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-73.gif"/></noscript></span></span>, as λ is defined as being within the interval (0, 0.5). Note that if the reset coefficient is 0, then λ is 1/2 and the <strong>T</strong> matrix becomes equivalent to the numerator relationship matrix (<strong>A</strong>), where the covariances between full sibs and sire offspring are equivalent. Similarly, the uncle−nephew covariance (individuals 5 and 7) is <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-74.gif"/></noscript></span></span>, lower than the covariance between half sibs (individuals 4 and 5) which is only <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-75.gif"/></noscript></span></span>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/477/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of a pedigree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-374239096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of a pedigree.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/477/F1.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/477/F1.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/477/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/477/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465830" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Example of a pedigree.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The mixed model implementation of the covariance between relatives requires the inverse of the <strong>T</strong> matrix to construct the mixed model equations, in a similar manner as the numerator relationship matrix in the standard model (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Henderson 1984</a>). In this study, we describe a simple procedure for the calculation of this <strong>T<sup>-1</sup></strong> matrix (see the <em><a id="xref-app-1-2" class="xref-app" href="#app-1">Appendix</a></em> section). The procedure takes into account the recursive nature of the transgenerational epigenetic effects, using an argument equivalent to <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Quass (1976)</a>, for the inverse of numerator relationship matrix (<strong>A<sup>−1</sup></strong>), and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Quintanilla <em>et al.</em> (1999)</a>, for dam-related permanent maternal environmental effects. The main consequence is that the inverse of the <strong>T</strong> matrix can be sequentially constructed by reading the pedigree of the population, in the light of very simple rules. These rules are close to <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Henderson’s (1976)</a> rules for constructing the inverse of the <strong>A</strong> matrix.</p><p id="p-29">For the example pedigree, the <strong>T<sup>−1</sup></strong> matrix is:<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-76.gif"/></noscript></span></span>Given this algorithm to set-up the <strong>T<sup>−1</sup></strong> matrix, the implementation of a Gibbs sampling approach becomes straightforward; it merely requires sequential iterative sampling from Gaussian (<strong>b</strong>, <strong>u</strong>, and <strong>w</strong>), truncated Gaussian (λ) and inverted χ<sup>2</sup> distributions (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-77.gif"/></noscript></span></span>). The only minor complication is that the <strong>T<sup>−1</sup></strong> matrix depends on parameter λ and consequently must be calculated in each Gibbs sampler’s iteration. One possible alternative to avoid this matrix inversion, although not to avoid its updating in each cycle, is to implement an approach similar to the one proposed by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Rodríguez-Ramilo <em>et al.</em> (2014)</a> for the genomic relationship matrix. However, setting up the <strong>T</strong> matrix is computationally more demanding than the calculation of its inverse with the algorithm proposed in this study.</p><p id="p-30">The proposed procedure was first checked with simulated data. The summary of the marginal posterior distributions for the variance components, additive genetic (<em>h<sup>2</sup></em>), and epigenetic (<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-78.gif"/></noscript></span></span>) heritability, reset coefficient (<em>v</em>), and epigenetic transmission coefficient (1 − <em>v</em>) for both cases of simulation are presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The greatest posterior density at 95% for all parameters in the model included the simulated values. However, some details of the results should be highlighted. In the second case of simulation, with lower <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-79.gif"/></noscript></span></span> and λ, the posterior standard deviation (and the HPD95) for the <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-81.gif"/></noscript></span></span> were remarkable wider (66.19 and 67.42, respectively). The cause of this phenomenon is that there is a statistical confounding between the epigenetic and residual variance components when the reset coefficient (<em>v</em>) is very high, because the <strong>T</strong> and <strong>I</strong> matrices becomes very similar. To illustrate this fact, in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, we present the joint posterior densities of <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-82.gif"/></noscript></span></span> and <em>v</em> for both cases of simulation. The results of the first case of simulation showed posterior independence between both parameters, whereas in the second, the marginal posterior density presented a half-moon shape. This indicates that for large values of <em>v</em>, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-83.gif"/></noscript></span></span> (and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-84.gif"/></noscript></span></span>) may take any value on its support with a fairly equal probability. In other words, in the first simulated case the model showed very good ability to discriminate between both parameters, whereas in the second it did not. In addition, mixing of the MCMC procedure in the second case of simulation was clearly worst, and adaptive MCMC algorithms, such as the proposed by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Mathew <em>et al.</em> (2012)</a> may represent an interesting alternative for its implementation in large data sets.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465837/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465837/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">PM, PSD, and HPD95 for simulation cases I and II</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/477/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Joint posterior distribution of the transgenerational epigenetic heritability () and the reset coefficient (v) in the first case of simulation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-374239096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Joint posterior distribution of the transgenerational epigenetic heritability (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-96&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the reset coefficient (&lt;em&gt;v&lt;/em&gt;) in the first case of simulation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/477/F2.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/477/F2.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/477/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/477/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Joint posterior distribution of the transgenerational epigenetic heritability (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/F2/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/F2/embed/mml-math-96.gif"/></noscript></span></span>) and the reset coefficient (<em>v</em>) in the first case of simulation.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/477/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Joint posterior distribution of the transgenerational epigenetic heritability () and the reset coefficient (v) in the second case of simulation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-374239096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Joint posterior distribution of the transgenerational epigenetic heritability (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and the reset coefficient (&lt;em&gt;v&lt;/em&gt;) in the second case of simulation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/477/F3.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/477/F3.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/477/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/477/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465923" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Joint posterior distribution of the transgenerational epigenetic heritability (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/F3/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/F3/embed/mml-math-97.gif"/></noscript></span></span>) and the reset coefficient (<em>v</em>) in the second case of simulation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">It must be noted that the sources of information for the estimation of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-98.gif"/></noscript></span></span> and <em>v</em> (or λ) comes from the comparison between the covariance between different categories of relatives (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Following <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a>, one estimate of <em>v</em> can be achieved from the difference between the estimates of covariance between half-sib and uncle−nephew relationships, a difference that becomes almost null for low values of λ (or high <em>v</em>), as seen in the second case of simulation (23.10 <em>vs.</em> 22.62). On the contrary, with greater values of λ(or lower <em>v</em>), as in the first case of simulation, this difference is higher (60.60 <em>vs.</em> 57.36), and, thus, the amount of information available for estimation purposes increases. This uncertainty in the estimation of λ (or <em>v</em>) is also reflected in the estimation of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-99.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-100.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-101.gif"/></noscript></span></span> and implies wider posterior marginal densities. Nevertheless, it is important to emphasize that the joint posterior mode (<em>v</em> = 0.80, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-102.gif"/></noscript></span></span> = 0.13) in the second case of simulation was very close to the simulated parameters (0.80 and 0.10, respectively).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465786/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465786/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Expected covariance between relatives in cases of simulation I and II</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">We have also checked our model with a real dataset. In <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, summary statistics of the marginal posterior distributions for the parameters and LogCPO values are presented for models I and II in the analysis of birth weight from the Pirenaica beef cattle population. The posterior mean estimates for heritability ranged between 0.38 (Model I) to 0.41 (Model II), and the posterior mean estimates for transgenerational epigenetic heritability was only 0.04 (Model I), with a greatest posterior density at 95% that ranged between 0.00 and 0.11. These results are coherent with the output of the model comparison test (LogCPO), which pointed to Model II as the more plausible. If we focus in this latter model, the estimates of direct and maternal heritability were within the range of estimates in the literature for these traits (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Meyer 1992</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Varona <em>et al.</em> 1999</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jamrozik and Miller 2014</a>), and the absence of the epigenetic transgenerational heritability is coherent with the fact that most of the epigenetic marks are erased during the meiosis in mammals (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Jablonka and Raz 2009</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schmitz 2014</a>). However, it should be highlighted that the posterior mean estimate of the reset coefficient under Model I was surprisingly low (0.20), although its posterior standard deviation was very high (0.20), and that the HPD95 region covered almost all the parametric space (0.01−0.89). This wide range reflects the absence of information to properly estimate the parameter, given the low magnitude of <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-111.gif"/></noscript></span></span>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465812/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465812/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">PM, PSD, and HPD95 for models I and II</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">It is worth noting that the procedure also can provide epigenetic breeding values in the fields of animal and plant breeding. These are calculated by weighting the phenotypic information of relatives according to the magnitude of the elements of the <strong>T</strong> matrix. In fact, for epigenetic effects, the weight of distantly related individuals is lowered as the number of opportunities to reset the epigenetic marks increases. Both additive genetic and epigenetic effects can be used for prediction of the future performance of the individual: the expected genetic response (R) after one cycle of mass selection should be <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-131.gif"/></noscript></span></span>, where <em>i</em> is the intensity of selection and <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-132.gif"/></noscript></span></span> is the phenotypic variance. However, further research must be undertaken to develop adequate indices of selection that consider genetic and epigenetic effects. Although both of them affect the immediate future performance of the offspring, epigenetic effects are diluted in future generations as the epigenetic marks are cleaned. It is important to note that if this selection pressure is relaxed, the average epigenetic effect declines to zero with a rate of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-133.gif"/></noscript></span></span>, where <em>n</em> is the number of generations without selection.</p><p id="p-39">It should be further noted that the proposed model uses a very basic definition of transgenerational epigenetic inheritance, as it assumes equal epigenetic variance for all individuals in the population. However, when phenotypic records are recorded through the life of the individual, transgenerational epigenetic variance can be modeled as age dependent, based on the consideration that the number of epigenetic marks was accumulated in the genome of the individuals. Moreover, in the proposed model, it is assumed that the λ parameter (or the reset coefficient) is equal for fathers and mothers, and, in future research, it seems reasonable to assume a different transmission coefficient for males and females, allowing for the presence of sex differential genomic imprinting (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barlow 1997</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Reik and Walker 1998</a>). The model can be refined by the inclusion of a hierarchical Bayesian paradigm that includes systematic effects for any environmental factors that may influence the epigenetic transmission coefficient. In addition, a more profound approach could even allow for the consideration of a genetic determinism of the reset coefficient with a model similar to that proposed by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Varona <em>et al.</em> (2008)</a> for the degree of asymmetry of a skewed Gaussian distribution.</p><p id="p-40">Finally, and as mentioned by <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Jablonka and Raz (2009)</a> and <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Tal <em>et al.</em> (2010)</a>, epigenetics may be viewed as a more wide ranging concept that could include several types of cultural transmission (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Avital and Jablonka 2000</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Richerson and Boyd 2005</a>). The procedure suggested in this work also could be applied to the analysis of human or animal datasets that involve any kind of transgenerational transmission, even those not directly related to gene expression.</p><p id="p-41">By way of a conclusion, we can say that this paper presents an original procedure for the estimation of transgenerational epigenetic variability based on some generalized assumptions. It is hoped that the proposal will lead to future research on variations of epigenetic transmission abilities caused by environmental and/or genetic factors.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank the Pirenaica Breeders Association (Confederación Nacional de Asociaciones de Ganado Pirenaico) for the provision of data used in this work. We also thank the useful comments of the editor and the reviewers. This research was funded by grant AGL2010-15903 (Ministerio de Ciencia e Innovación, Spain).</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-10"><h3>Derivation of the <strong>T<sup>−1</sup></strong> matrix</h3><p id="p-42">The vector of epigenetic effects (<strong>w</strong>) can be represented as:<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-134.gif"/></noscript></span></span>where the <strong>P</strong> matrix defines a recurrent relationship with the epigenetic effects of the father and mother. For nonbase individuals, the ith row of the <strong>P</strong> matrix contains a parameter λ in the column pertaining to the father and mother of the ith individual. The rest of the elements are null.</p><p id="p-43">Furthermore, if:<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-135.gif"/></noscript></span></span>then<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-136.gif"/></noscript></span></span>given that <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-137.gif"/></noscript></span></span></p><div id="sec-11" class="subsection"><span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-138.gif"/></noscript></span> <span class="disp-formula-label">(A1)</span></span><p id="p-44">From <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-139.gif"/></noscript></span></span>, it is possible to define a <strong>Q<sub>T</sub></strong> matrix as:<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-140.gif"/></noscript></span> <span class="disp-formula-label">(A2)</span></span>If we consider the structure of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-141.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-142.gif"/></noscript></span></span>, the <strong>Q<sub>T</sub></strong> matrix has a diagonal structure with 1s for base individuals and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-143.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-144.gif"/></noscript></span></span> for those with one or two known parents, respectively.</p><p id="p-45">Finally, replacing <a id="xref-disp-formula-39-1" class="xref-disp-formula" href="#disp-formula-39">(A2)</a> into <a id="xref-disp-formula-38-1" class="xref-disp-formula" href="#disp-formula-38">(A1)</a>:<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-145.gif"/></noscript></span></span>Taking advantage of this structure, it is possible to define the following algorithm for constructing <strong>T</strong><sup>−1</sup> by reading the pedigree information and calculating for each individual (i) in the pedigree:</p><ol class="list-alphalower " id="list-3"><li id="list-item-3"><p id="p-46">If the father and mother are unknown;</p></li></ol><p id="p-47">add 1 to the diagonal (i,i)</p><ol class="list-alphalower " id="list-4"><li id="list-item-4"><p id="p-48">If only one parent (p) is known;</p></li></ol><p id="p-49">add <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-146.gif"/></noscript></span></span> to the diagonal (i,i)</p><p id="p-50">add <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-147.gif"/></noscript></span></span> to the elements (i,p), (p,i)</p><p id="p-51">add <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-148.gif"/></noscript></span></span> to the element (p,p)</p><ol class="list-alphalower " id="list-5"><li id="list-item-5"><p id="p-52">If both ancestors (p and m) are known;</p></li></ol><p id="p-53">add <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-149.gif"/></noscript></span></span> to the diagonal (i,i)</p><p id="p-54">add <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-150.gif"/></noscript></span></span> to the elements (i,p), (i,m), (m,i), (p,i)</p><p id="p-55">add <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/477/embed/mml-math-151.gif"/></noscript></span></span> to the elements (p,p), (p,m), (m,p), (m,m)</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-57">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.016725/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.016725/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-58"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-01"><span class="received-label">Received </span>December 1, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-01-19"><span class="accepted-label">Accepted </span>January 19, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Varona <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avital</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jablonka</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Animal Traditions: Behavioural Inheritance in Evolution</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.2" data-doi="10.1093/emboj/16.23.6899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Competition—a common motif for the imprinting mechanism?</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">6899</span>–<span class="cit-lpage">6905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DBarlow%26rft.auinit1%253DD.%2BP.%26rft.volume%253D16%26rft.issue%253D23%26rft.spage%253D6899%26rft.epage%253D6905%26rft.atitle%253DCompetition--a%2Bcommon%2Bmotif%2Bfor%2Bthe%2Bimprinting%2Bmechanism%253F%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.23.6899%26rft_id%253Dinfo%253Apmid%252F9384569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMjMvNjg5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC80NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.3" data-doi="10.1038/447395a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eccleston</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeWitt</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marte</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nath</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Introduction epigenetics.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D447%26rft.spage%253D395%26rft_id%253Dinfo%253Adoi%252F10.1038%252F447395a%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/447395a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000246693100032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.4" data-doi="10.1056/NEJMra072067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esteller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Epigenetics of cancer.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">1148</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DEsteller%26rft.auinit1%253DM.%26rft.volume%253D358%26rft.issue%253D11%26rft.spage%253D1148%26rft.epage%253D1159%26rft.atitle%253DEpigenetics%2Bin%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra072067%26rft_id%253Dinfo%253Apmid%252F18337604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra072067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18337604&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253877200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mckay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Longmans Green</span>, <span class="cit-publ-loc">Harlow, Essex</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.6" data-doi="10.1038/nature05919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feinberg</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phenotypic plasticity and the epigenetics of human disease.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFeinberg%26rft.auinit1%253DA.%2BP.%26rft.volume%253D447%26rft.issue%253D7143%26rft.spage%253D433%26rft.epage%253D440%26rft.atitle%253DPhenotypic%2Bplasticity%2Band%2Bthe%2Bepigenetics%2Bof%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05919%26rft_id%253Dinfo%253Apmid%252F17522677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17522677&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246693100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Model determination using sampling-based methods</span>, pp. <span class="cit-fpage">145</span>–<span class="cit-lpage">161</span> in <span class="cit-source">Markov Chain Monte Carlo in Practice</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Gilks</span>  <span class="cit-name-given-names">W. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D. J.</span></span>. <span class="cit-publ-name">Chapman &amp; Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.8" data-doi="10.2307/2289776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Sampling-based approaches to calculating marginal densities.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D85%26rft.spage%253D398%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2289776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2289776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990DJ33200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.9" data-doi="10.2527/jas1986.631217x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Bayesian methods in animal breeding theory.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D217%26rft.epage%253D244%26rft.atitle%253DBayesian%2BMethods%2Bin%2BAnimal%2BBreeding%2BTheory%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1986.631217x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1986.631217x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986D043100027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.10" data-doi="10.1038/nrg3435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grossniklaus</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson-Smith</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pembrey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transgenerational epigenetic inheritance: how important is it?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3435%26rft_id%253Dinfo%253Apmid%252F23416892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23416892&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.11" data-doi="10.1016/j.cell.2014.02.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martienssen</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transgenerational epigenetic inheritance: myths and mechanisms.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.02.045%26rft_id%253Dinfo%253Apmid%252F24679529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.02.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24679529&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333968900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.12" data-doi="10.2307/2529339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">A simple method for computing the inverse of a numerator relationship matrix used in the prediction of breeding values.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D32%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BM07600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Application of Linear Models in Animal Breeding</span>. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph, ON</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.14" data-doi="10.1038/nature05917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Epigenetic inheritance in plants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHenderson%26rft.auinit1%253DI.%2BR.%26rft.volume%253D447%26rft.issue%253D7143%26rft.spage%253D418%26rft.epage%253D424%26rft.atitle%253DEpigenetic%2Binheritance%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05917%26rft_id%253Dinfo%253Apmid%252F17522675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17522675&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246693100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.15" data-doi="10.1086/598822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jablonka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transgenerational epigenetic inheritance: prevalence, mechanisms and implications for the study of heredity and evolution.</span> <abbr class="cit-jnl-abbrev">Q. Rev. Biol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BQuarterly%2BReview%2Bof%2BBiology%26rft.stitle%253DThe%2BQuarterly%2BReview%2Bof%2BBiology%26rft.aulast%253DJablonka%26rft.auinit1%253DE.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D131%26rft.epage%253D176%26rft.atitle%253DTransgenerational%2Bepigenetic%2Binheritance%253A%2Bprevalence%252C%2Bmechanisms%252C%2Band%2Bimplications%2Bfor%2Bthe%2Bstudy%2Bof%2Bheredity%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F598822%26rft_id%253Dinfo%253Apmid%252F19606595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/598822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19606595&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266124200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.16" data-doi="10.1016/j.livsci.2014.01.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jamrozik</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic evaluation of calving ease in Canadian Simmentals using birth weight and gestation length as correlated traits.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D162%26rft.spage%253D42%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.01.027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.01.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.17" data-doi="10.1038/nrg2719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishing, maintaining and modifying DNA methylation patterns in plants and animals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">204</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLaw%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D204%26rft.epage%253D220%26rft.atitle%253DEstablishing%252C%2Bmaintaining%2Band%2Bmodifying%2BDNA%2Bmethylation%2Bpatterns%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2719%26rft_id%253Dinfo%253Apmid%252F20142834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20142834&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.18" data-doi="10.1038/hdy.2012.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathew</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koistinen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reetz</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bayesian adaptive Markov chain Monte Carlo estimation of genetic parameters.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMathew%26rft.auinit1%253DB.%26rft.volume%253D109%26rft.issue%253D4%26rft.spage%253D235%26rft.epage%253D245%26rft.atitle%253DBayesian%2Badaptive%2BMarkov%2Bchain%2BMonte%2BCarlo%2Bestimation%2Bof%2Bgenetic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.35%26rft_id%253Dinfo%253Apmid%252F22805656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22805656&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Variance components due to direct and maternal effects for growth traits of Australian beef cattle.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D11%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.20" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recovery of inter-block information when block sizes are unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjU4LzMvNTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzQ3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.21" data-doi="10.2307/2529279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quass</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Computing the diagonal elements of the inverse of a large numerator relationship matrix.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D32%26rft.spage%253D949%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CQ86300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quintanilla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piedrafita</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Maternal animal model with correlation between maternal environmental effects of related dams.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">2904</span>–<span class="cit-lpage">2917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DQuintanilla%26rft.auinit1%253DR.%26rft.volume%253D77%26rft.issue%253D11%26rft.spage%253D2904%26rft.epage%253D2917%26rft.atitle%253DMaternal%2Banimal%2Bmodel%2Bwith%2Bcorrelation%2Bbetween%2Bmaternal%2Benvironmental%2Beffects%2Bof%2Brelated%2Bdams%26rft_id%253Dinfo%253Apmid%252F10568458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10568458&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">How many iterations in the Gibbs sampler</span>, pp. <span class="cit-fpage">763</span>–<span class="cit-lpage">774</span> in <span class="cit-source">Bayesian Statistics IV</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">J. O.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dawid</span>  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. F. M.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.24" data-doi="10.1038/nature05918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reik</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Stability and flexibility of epigenetic gene regulation in mammalian development.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReik%26rft.auinit1%253DW.%26rft.volume%253D447%26rft.issue%253D7143%26rft.spage%253D425%26rft.epage%253D432%26rft.atitle%253DStability%2Band%2Bflexibility%2Bof%2Bepigenetic%2Bgene%2Bregulation%2Bin%2Bmammalian%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05918%26rft_id%253Dinfo%253Apmid%252F17522676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17522676&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246693100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.25" data-doi="10.1016/S0959-437X(98)80136-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reik</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Imprinting mechanisms in mammals.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DReik%26rft.auinit1%253DW.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D154%26rft.epage%253D164%26rft.atitle%253DImprinting%2Bmechanisms%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252898%252980136-6%26rft_id%253Dinfo%253Apmid%252F9610405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(98)80136-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9610405&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073519100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richerson</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Not by Genes Alone: How Culture Transformed Human Evolution</span>. <span class="cit-publ-name">University of Chicago Press</span>, <span class="cit-publ-loc">Chicago</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.27" data-doi="10.1371/journal.pone.0093424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Ramilo</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Cortés</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Combining genomic and genealogical information in a reproducing kernel Hilbert spaces regression model for genome-enabled predictions in dairy cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e93424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De93424%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093424%26rft_id%253Dinfo%253Apmid%252F24671175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24671175&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.477.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrona</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Razas Ganaderas Españolas Bovinas. MAPA-FEAGAS</span>. <span class="cit-publ-name">Spain</span>, <span class="cit-publ-loc">Madrid</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.29" data-doi="10.1126/science.1251864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The secret garden—epigenetic alleles underlies complex traits.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchmitz%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D343%26rft.issue%253D6175%26rft.spage%253D1082%26rft.epage%253D1083%26rft.atitle%253DThe%2BSecret%2BGarden--Epigenetic%2BAlleles%2BUnderlie%2BComplex%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251864%26rft_id%253Dinfo%253Apmid%252F24604184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDMvNjE3NS8xMDgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzQ3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.30" data-doi="10.1534/genetics.109.112466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tal</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kisdi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jablonka</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Epigenetic contribution to covariance between relatives.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.112466%26rft_id%253Dinfo%253Apmid%252F20100941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NC80LzEwMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNDc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.31" data-doi="10.1534/genetics.113.159475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Joint analysis of binomial and continuous traits with recursive model: a case study using mortality and litter size in pigs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159475%26rft_id%253Dinfo%253Apmid%252F24414548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvNjQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzQ3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Threshold-linear versus linear-linear analysis of birth weight and calving ease using an animal model: I. Variance component estimation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1994</span>–<span class="cit-lpage">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DVarona%26rft.auinit1%253DL.%26rft.volume%253D77%26rft.issue%253D8%26rft.spage%253D1994%26rft.epage%253D2002%26rft.atitle%253DThreshold-linear%2Bversus%2Blinear-linear%2Banalysis%2Bof%2Bbirth%2Bweight%2Band%2Bcalving%2Bease%2Busing%2Ban%2Banimal%2Bmodel%253A%2BI.%2BVariance%2Bcomponent%2Bestimation%26rft_id%253Dinfo%253Apmid%252F10461973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10461973&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibañez-Escriche</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintanilla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguera</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casellas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian analysis of quantitative traits using skewed distributions.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVarona%26rft.auinit1%253DL.%26rft.volume%253D90%26rft.issue%253D2%26rft.spage%253D179%26rft.epage%253D190%26rft.atitle%253DBayesian%2Banalysis%2Bof%2Bquantitative%2Btraits%2Busing%2Bskewed%2Bdistributions.%26rft_id%253Dinfo%253Apmid%252F18426621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18426621&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.34" data-doi="10.1051/gse:19940202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutledge</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Bayesian analysis of mixed linear models via Gibbs sampling with an application to litter size in Iberian pigs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19940202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19940202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NL04000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.477.35" data-doi="10.1146/annurev.genom.9.081307.164445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youngson</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitelaw</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Transgenerational epigenetic effects.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DYoungson%26rft.auinit1%253DN.%2BA.%26rft.volume%253D9%26rft.spage%253D233%26rft.epage%253D257%26rft.atitle%253DTransgenerational%2Bepigenetic%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164445%26rft_id%253Dinfo%253Apmid%252F18767965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18767965&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F477.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259629000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/477.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/467" title="Identification of Nonviable Genes Affecting Touch Sensitivity in Caenorhabditis elegans Using Neuronally Enhanced Feeding RNA Interference" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/487" title="Identification of eQTLs for Hepatic Xbp1s and Socs3 Gene Expression in Mice Fed a High-Fat, High-Caloric Diet" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/477.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1iE6PmGA_-vg-O21t5kro9rQR2n-54sd9WfLF8zyfYM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/477" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/477" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/477" class="active" data-icon-position="" data-hide-link-title="0">A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459806" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-k_g6eoiRDj0cnHGpLVdnIkS2SL5_NyjG3oQtuNQwxUc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/477.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/477" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-PAHDzqb0icqzygsGlAVu1eLFrkH9GkLXp6eCo7MbIJQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/477" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/477.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459806" id="node459806--2" data-pisa="ggg;5/4/477" data-pisa-master="ggg;g3.115.016725" data-apath="/ggg/5/4/477.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6256-5478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Varona</span>, <span class="highwire-citation-author" data-delta="1">Sebastián Munilla</span>, <span class="highwire-citation-author" data-delta="2">Elena Flavia Mouresan</span>, <span class="highwire-citation-author" data-delta="3">Aldemar González-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Carlos Moreno</span> and <span class="highwire-citation-author" data-delta="5">Juan Altarriba</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">477-485;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016725 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459806--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis Varona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVarona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varona%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BVarona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6256-5478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Varona</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lvarona@unizar.es" class="" data-icon-position="" data-hide-link-title="0">lvarona@unizar.es</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sebastián Munilla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Producción Animal, Facultad de Agronomía, Universidad de Buenos Aires, 1417, Ciudad Autónoma de Buenos Aires, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A1n%2BMunilla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munilla%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A1n%2BMunilla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elena Flavia Mouresan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BFlavia%2BMouresan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mouresan%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BFlavia%2BMouresan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aldemar González-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldemar%2BGonz%C3%A1lez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Rodr%C3%ADguez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAldemar%2BGonz%25C3%25A1lez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos Moreno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BMoreno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreno%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BMoreno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Altarriba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BAltarriba%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Altarriba%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BAltarriba%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/477" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/477&amp;title=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/477&amp;title=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/477&amp;title=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/477&amp;text=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/477&amp;title=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/477&amp;t=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/477&amp;title=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/477&amp;title=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459806/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459806" id="node459806--3" data-pisa="ggg;5/4/477" data-pisa-master="ggg;g3.115.016725" data-apath="/ggg/5/4/477.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Model for the Analysis of Transgenerational Epigenetic Variation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6256-5478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Varona</span>, <span class="highwire-citation-author" data-delta="1">Sebastián Munilla</span>, <span class="highwire-citation-author" data-delta="2">Elena Flavia Mouresan</span>, <span class="highwire-citation-author" data-delta="3">Aldemar González-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Carlos Moreno</span> and <span class="highwire-citation-author" data-delta="5">Juan Altarriba</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">477-485;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016725 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459806--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis Varona </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVarona%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varona%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BVarona%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6256-5478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Varona</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lvarona@unizar.es" class="" data-icon-position="" data-hide-link-title="0">lvarona@unizar.es</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sebastián Munilla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Producción Animal, Facultad de Agronomía, Universidad de Buenos Aires, 1417, Ciudad Autónoma de Buenos Aires, Argentina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A1n%2BMunilla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munilla%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A1n%2BMunilla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Elena Flavia Mouresan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BFlavia%2BMouresan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mouresan%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElena%2BFlavia%2BMouresan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aldemar González-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldemar%2BGonz%C3%A1lez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Rodr%C3%ADguez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAldemar%2BGonz%25C3%25A1lez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carlos Moreno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BMoreno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreno%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BMoreno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Altarriba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Unidad de Genética Cuantitativa y Mejora Animal, Universidad de Zaragoza, 50013, Zaragoza, Spain</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Instituto de Biocomputación y Física de los Sistemas Complejos (BIFI), Universidad de Zaragoza, 50018, Zaragoza, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BAltarriba%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Altarriba%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BAltarriba%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=477&amp;rft.epage=485&amp;rft.atitle=A%20Bayesian%20Model%20for%20the%20Analysis%20of%20Transgenerational%20Epigenetic%20Variation&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Varona%2C+Luis&amp;rft.au=Munilla%2C+Sebasti%C3%A1n&amp;rft.au=Mouresan%2C+Elena+Flavia&amp;rft.au=Gonz%C3%A1lez-Rodr%C3%ADguez%2C+Aldemar&amp;rft.au=Moreno%2C+Carlos&amp;rft.au=Altarriba%2C+Juan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459806/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459806/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459806/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459806/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459806/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459806/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459806/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459806/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459806/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459806/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459806/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459806/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/477" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/477.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/477.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/477.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/477.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
