<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/655" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/655.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/655.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/655.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/655.atom" />
<meta name="HW.pisa" content="ggg;5/4/655" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK" />
<meta name="DC.Identifier" content="10.1534/g3.115.017012" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 McCormick et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Recent advances in variant calling made available in the Genome Analysis Toolkit (GATK) enable the use of validated single-nucleotide polymorphisms and indels to improve variant calling. However, large collections of variants for this purpose often are unavailable to research communities. We introduce a workflow to generate reliable collections of single-nucleotide polymorphisms and indels by leveraging available genomic resources to inform variant calling using the GATK. The workflow is demonstrated for the crop plant Sorghum bicolor by (i) generating an initial set of variants using reduced representation sequence data from an experimental cross and association panels, (ii) using the initial variants to inform variant calling from whole-genome sequence data of resequenced individuals, and (iii) using variants identified from whole-genome sequence data for recalibration of the reduced representation sequence data. The reliability of variants called with the workflow is verified by comparison with genetically mappable variants from an independent sorghum experimental cross. Comparison with a recent sorghum resequencing study shows that the workflow identifies an additional 1.62 million high-confidence variants from the same sequence data. Finally, the workflow’s performance is validated using Arabidopsis sequence data, yielding variant call sets with 95% sensitivity and 99% positive predictive value. The Recalibration and Interrelation of genomic sequence data with the GATK (RIG) workflow enables the GATK to accurately identify genetic variation in organisms lacking validated variant resources." />
<meta name="DC.Contributor" content="Ryan F. McCormick" />
<meta name="DC.Contributor" content="Sandra K. Truong" />
<meta name="DC.Contributor" content="John E. Mullet" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recent advances in variant calling made available in the Genome Analysis Toolkit (GATK) enable the use of validated single-nucleotide polymorphisms and indels to improve variant calling. However, large collections of variants for this purpose often are unavailable to research communities. We introduce a workflow to generate reliable collections of single-nucleotide polymorphisms and indels by leveraging available genomic resources to inform variant calling using the GATK. The workflow is demonstrated for the crop plant &lt;i&gt;Sorghum bicolor&lt;/i&gt; by (i) generating an initial set of variants using reduced representation sequence data from an experimental cross and association panels, (ii) using the initial variants to inform variant calling from whole-genome sequence data of resequenced individuals, and (iii) using variants identified from whole-genome sequence data for recalibration of the reduced representation sequence data. The reliability of variants called with the workflow is verified by comparison with genetically mappable variants from an independent sorghum experimental cross. Comparison with a recent sorghum resequencing study shows that the workflow identifies an additional 1.62 million high-confidence variants from the same sequence data. Finally, the workflow’s performance is validated using &lt;i&gt;Arabidopsis&lt;/i&gt; sequence data, yielding variant call sets with 95% sensitivity and 99% positive predictive value. The Recalibration and Interrelation of genomic sequence data with the GATK (RIG) workflow enables the GATK to accurately identify genetic variation in organisms lacking validated variant resources.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/655" />
<meta name="citation_id" content="5/4/655" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/655" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/655.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/655.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/655.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017012" />
<meta name="citation_pmid" content="25681258" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="655" />
<meta name="citation_lastpage" content="665" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ryan F. McCormick" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Sandra K. Truong" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="John E. Mullet" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_email" content="jmullet@tamu.edu" />
<meta name="citation_reference" content="citation_title=Bioinformatics;citation_author=S. Anders;citation_author=P. T. Pyl;citation_author=W. Huber;citation_title=HTSeq-A Python framework to work with high-throughput sequencing data;citation_volume=31;citation_year=2015;citation_pmid=25260700;citation_doi=10.1093/bioinformatics/btu638" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=S. Bout;citation_author=W. Vermerris;citation_title=A candidate-gene approach to clone the sorghum Brown midrib gene encoding caffeic acid O-methyltransferase.;citation_pages=205-214;citation_volume=269;citation_year=2003;citation_issue=2;citation_pmid=12756532" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. J. Brown;citation_author=W. L. Rooney;citation_author=C. Franks;citation_author=S. Kresovich;citation_title=Efficient mapping of plant height quantitative trait loci in a sorghum association population with introgressed dwarfing genes.;citation_volume=180;citation_year=2008;citation_pmid=18757942;citation_doi=10.1534/genetics.108.092239" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Regis.;citation_author=G. Burow;citation_author=R. Klein;citation_author=C. Franks;citation_author=P. Klein;citation_author=K. Schertz;citation_title=Registration of the BTx623/IS3620C recombinant inbred mapping population of sorghum.;citation_pages=141-145;citation_volume=5;citation_year=2011;citation_doi=10.3198/jpr2010.04.0219crmp" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Cao;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=T. Günther;citation_author=S. Bender;citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations.;citation_pages=956-963;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21874002;citation_doi=10.1038/ng.911" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Evans;citation_author=R. F. McCormick;citation_author=D. Morishige;citation_author=S. N. Olson;citation_author=B. Weers;citation_title=Extensive variation in the density and distribution of DNA polymorphism in sorghum genomes.;citation_pages=e79192;citation_volume=8;citation_year=2013;citation_pmid=24265758;citation_doi=10.1371/journal.pone.0079192" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=R. H. Higgins;citation_author=C. S. Thurber;citation_author=I. Assaranurak;citation_author=P. J. Brown;citation_title=Multiparental Mapping of Plant Height and Flowering Time QTL in Partially Isogenic Sorghum Families;citation_pages=1593-1602;citation_volume=4;citation_year=2014;citation_issue=9;citation_pmid=25237111;citation_doi=10.1534/g3.114.013318" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=J. D. Hunter;citation_title=Matplotlib: A 2D graphics environment.;citation_pages=90-95;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.55" />
<meta name="citation_reference" content="citation_title=arXiv;citation_title=Towards better understanding of artifacts in variant calling from high-coverage samples.;citation_author=H. Li;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Li;citation_author=M. Nordborg;citation_author=L. M. Li;citation_title=Adjust quality scores from alignment and improve sequencing accuracy;citation_pages=5183-5191;citation_volume=32;citation_year=2004;citation_issue=17;citation_pmid=15459287;citation_doi=10.1093/nar/gkh850" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Liu;citation_author=S. Han;citation_author=Z. Wang;citation_author=J. Gelernter;citation_author=B.-Z. Yang;citation_title=Variant callers for next-generation sequencing data: a comparison study.;citation_pages=e75619;citation_volume=8;citation_year=2013;citation_pmid=24086590;citation_doi=10.1371/journal.pone.0075619" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=E. S. Mace;citation_author=S. Tai;citation_author=E. K. Gilding;citation_author=Y. Li;citation_author=P. J. Prentis;citation_title=Whole-genome sequencing reveals untapped genetic potential in Africa’s indigenous cereal crop sorghum.;citation_pages=2320;citation_volume=4;citation_year=2013;citation_pmid=23982223" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. K. Monaco;citation_author=J. Stein;citation_author=S. Naithani;citation_author=S. Wei;citation_author=P. Dharmawardhana;citation_title=Gramene 2013: comparative plant genomics resources.;citation_volume=42;citation_year=2014;citation_pmid=24217918;citation_doi=10.1093/nar/gkt1110" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. T. Morishige;citation_author=P. E. Klein;citation_author=J. L. Hilley;citation_author=S. M. E. Sahraeian;citation_author=A. Sharma;citation_title=Digital genotyping of sorghum–a diverse plant species with a large repeat-rich genome.;citation_pages=448;citation_volume=14;citation_year=2013;citation_pmid=23829350;citation_doi=10.1186/1471-2164-14-448" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. P. Morris;citation_author=P. Ramu;citation_author=S. P. Deshpande;citation_author=C. T. Hash;citation_author=T. Shah;citation_title=Population genomic and genome-wide association studies of agroclimatic traits in sorghum.;citation_volume=110;citation_year=2013;citation_pmid=23267105;citation_doi=10.1073/pnas.1215985110" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_title=Next-generation sequencing data interpretation: enhancing reproducibility and accessibility.;citation_pages=667-672;citation_volume=13;citation_year=2012;citation_issue=9;citation_pmid=22898652;citation_doi=10.1038/nrg3305" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. Nordborg;citation_author=T. T. Hu;citation_author=Y. Ishino;citation_author=J. Jhaveri;citation_author=C. Toomajian;citation_title=The pattern of polymorphism in Arabidopsis thaliana.;citation_pages=e196-e196;citation_volume=3;citation_year=2005;citation_issue=7;citation_pmid=15907155;citation_doi=10.1371/journal.pbio.0030196" />
<meta name="citation_reference" content="citation_journal_title=Genome Med;citation_author=J. O’Rawe;citation_author=T. Jiang;citation_author=G. Sun;citation_author=Y. Wu;citation_author=W. Wang;citation_title=Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.;citation_pages=28;citation_volume=5;citation_year=2013;citation_pmid=23537139;citation_doi=10.1186/gm432" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genomics;citation_author=M. Pirooznia;citation_author=M. Kramer;citation_author=J. Parla;citation_author=F. Goes;citation_author=J. Potash;citation_title=Validation and assessment of variant calling pipelines for next-generation sequencing.;citation_pages=14;citation_volume=8;citation_year=2014;citation_pmid=25078893;citation_doi=10.1186/1479-7364-8-14" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=J. Puritz;citation_author=C. M. Hollenbeck;citation_author=J. R. Gold;citation_title=dDocent: a RADseq, variant-calling pipeline designed for population genomics of non-model organisms.;citation_pages=e431;citation_volume=2;citation_year=2014;citation_pmid=24949246;citation_doi=10.7717/peerj.431" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. J. Schmitz;citation_author=M. D. Schultz;citation_author=M. A. Urich;citation_author=J. R. Nery;citation_author=M. Pelizzola;citation_title=Patterns of population epigenomic diversity.;citation_pages=193-198;citation_volume=495;citation_year=2013;citation_pmid=23467092;citation_doi=10.1038/nature11968" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Sims;citation_author=I. Sudbery;citation_author=N. E. Ilott;citation_author=A. Heger;citation_author=C. P. Ponting;citation_title=Sequencing depth and coverage: key considerations in genomic analyses.;citation_pages=121-132;citation_volume=15;citation_year=2014;citation_pmid=24434847;citation_doi=10.1038/nrg3642" />
<meta name="citation_reference" content="citation_journal_title=The USENIX Magazine;citation_author=O. Tange;citation_title=GNU parallel—the command-line power tool.;login;citation_pages=42-47;citation_volume=36;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdóttir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_pages=178-192;citation_volume=14;citation_year=2012;citation_pmid=22517427" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=S. K. Truong;citation_author=R. F. McCormick;citation_author=D. T. Morishige;citation_author=J. E. Mullet;citation_title=Resolution of genetic map expansion caused by excess heterozygosity in plant recombinant inbred populations.;citation_volume=4;citation_year=2014;citation_pmid=25128435;citation_doi=10.1534/g3.114.012468" />
<meta name="citation_reference" content="citation_title=From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline;citation_author=G. A. Van Der Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. del Angel;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=W. Xu;citation_author=P. K. Subudhi;citation_author=O. R. Crasta;citation_author=D. T. Rosenow;citation_author=J. E. Mullet;citation_title=Molecular mapping of QTLs conferring stay-green in grain sorghum (Sorghum bicolor L. Moench).;citation_pages=461-469;citation_volume=43;citation_year=2000;citation_issue=3;citation_pmid=10902709" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=L.-Y. Zheng;citation_author=X.-S. Guo;citation_author=B. He;citation_author=L.-J. Sun;citation_author=Y. Peng;citation_title=Genome-wide patterns of genetic variation in sweet and grain sorghum (Sorghum bicolor).;citation_pages=R114-R114;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=22104744;citation_doi=10.1186/gb-2011-12-11-r114" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Recent advances in variant calling made available in the Genome Analysis Toolkit (GATK) enable the use of validated single-nucleotide polymorphisms and indels to improve variant calling. However, large collections of variants for this purpose often are unavailable to research communities. We introduce a workflow to generate reliable collections of single-nucleotide polymorphisms and indels by leveraging available genomic resources to inform variant calling using the GATK. The workflow is demonstrated for the crop plant Sorghum bicolor by (i) generating an initial set of variants using reduced representation sequence data from an experimental cross and association panels, (ii) using the initial variants to inform variant calling from whole-genome sequence data of resequenced individuals, and (iii) using variants identified from whole-genome sequence data for recalibration of the reduced representation sequence data. The reliability of variants called with the workflow is verified by comparison with genetically mappable variants from an independent sorghum experimental cross. Comparison with a recent sorghum resequencing study shows that the workflow identifies an additional 1.62 million high-confidence variants from the same sequence data. Finally, the workflow’s performance is validated using Arabidopsis sequence data, yielding variant call sets with 95% sensitivity and 99% positive predictive value. The Recalibration and Interrelation of genomic sequence data with the GATK (RIG) workflow enables the GATK to accurately identify genetic variation in organisms lacking validated variant resources." />
<meta name="og-title" property="og:title" content="RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/655" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recent advances in variant calling made available in the Genome Analysis Toolkit (GATK) enable the use of validated single-nucleotide polymorphisms and indels to improve variant calling. However, large collections of variants for this purpose often are unavailable to research communities. We introduce a workflow to generate reliable collections of single-nucleotide polymorphisms and indels by leveraging available genomic resources to inform variant calling using the GATK. The workflow is demonstrated for the crop plant Sorghum bicolor by (i) generating an initial set of variants using reduced representation sequence data from an experimental cross and association panels, (ii) using the initial variants to inform variant calling from whole-genome sequence data of resequenced individuals, and (iii) using variants identified from whole-genome sequence data for recalibration of the reduced representation sequence data. The reliability of variants called with the workflow is verified by comparison with genetically mappable variants from an independent sorghum experimental cross. Comparison with a recent sorghum resequencing study shows that the workflow identifies an additional 1.62 million high-confidence variants from the same sequence data. Finally, the workflow’s performance is validated using Arabidopsis sequence data, yielding variant call sets with 95% sensitivity and 99% positive predictive value. The Recalibration and Interrelation of genomic sequence data with the GATK (RIG) workflow enables the GATK to accurately identify genetic variation in organisms lacking validated variant resources." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0Xq95HJ-3K2sbhQ-JAOoTd5X7j8_rVhP_U3LXxO19r8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459796","apath":"\/ggg\/5\/4\/655.atom","pisa":"ggg;5\/4\/655","ac":{"\/ggg\/5\/4\/655.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/655.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/655"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/655"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/655":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FNEBuz1N8EUZ4IJ9VUgOK73fuBk8IE6hOR61tUsZ5riKvGkvg77DPxQFS0KX2M7fdCkwtUz5Il+QKZfd4hoZl9sXsfjOEeLNYcuv+t5Bzv6XkeAmIZXJNcqdxDJR5yT+mT3nqb0rAZj+T6mKwukKA89AYKY6bFSWizNIkAjXOblUwWvMQAXnbOGtoEWt7so6WzXvP8HiNAOUO2AinLqCiRGcUj7ua4ZhL+wHsO1Fl+AZ3QExm4rSIWtupMAx6DwfjtskYXdlhHuf2Tpd5bD0jY2e6TKcgr54ekJ2yjYXmRcWmEP66G+myb09ylZC5C8ysFdl3ULamKWi2ga0tjnczCfFIDhJ9TOsXF88NlPEziPptAZmJqcXTdj2Hwk723+jbWpCsPCDICcR7oOy+wcrDiFPf4FnDSOg1yimvFtiiQJLoF3wKpfg7H9IYsPDSSQbxo9DmcGFMtQtQXWJOFxBMmNoY4BKIl8SzMO\\\\\\\/n3sFtB3S5g4ZGK44VmZ9g1URthgfRt3eQgjiE2X8sl5h3g7Ds1n5bRA2uGIechDPhtFF3Nk38l7H00znkdVFtQZNGjWYDo0vh6D2uMYpCVz4AAA3lkMWHIf4s0CegZwGqVn49\\\\\\\/ew4GfXljDlvSvh\\\\\\\/0jbjJrlqPxlPBBbe9H9r\\\\\\\/dEitpyI7l0AtxwrTBMbVs6ARR9utNtdE3dq3W5Et0YgKz9APbO3ATSKkfBvN7qxqRb3MThg1EuP2hNbr1eEBfxSjM76VI4GgxwD99z3e4MkMmF2uiYM4ybqnIkrEPCQ6ZuoV0HDVWtozn6yTaoCFnvMH2BvOU+3FwDe5g8+VK8awwuPmRvpA8oi3rm+QuBGITnK45+7CGT3gyhMD58yqby+drs2NA97UJakNTZ8V7vsAWQRsPQbNapUDzOPgfH7jRn37d3jKfUxjPBrqp0scc3YZ0HLhlCHqkaLwFMufENikyP1HO15myU\\\\\\\/VuLS3EhHmwkh7i+9vkbMd4wczEI9fTz1el4MXnfsrGBYw299d8FdyD\\\\\\\/p17cUB27vskSVHp4RkBWRtBYcmHjnAkwvS+ybPYmjvVjs\\\\\\\/GBn7ae\\\\\\\/b0chnoXAleCpyMzOT+zy5Kc6SdNSws+ADfWmCmKATGgwSEIc9bXaxqg8xwjL9l\\\\\\\/SMZZBKbypfqvNGsRGEuYMhioUMqN1cidUoOlmkAAyR2XUCgLIDS6c2ZtIBmKY0y5XrEmtZ6PzvFojLaUkc\\\\\\\/xSUbO\\\\\\\/UYXIw\\\\\\\/m1LEGpMblPoWBN6v4CtPWVfYAlPW3c2OAPSu06u1ha6EPbSs0\\\u0022,\\\u0022iv\\\u0022:\\\u00229hLkScT91qgWrR950sJldQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022168d55ce01b6326146d048ce235e13d82fa111ef8b036b831aa37451da840df3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tjv77C15hkTqJiWIYhq\\\\\\\/knyK+DUcp6dxOjEwxmB50nedrMlhDUi2vU6cyT6VKFPB43z6g5XhL\\\\\\\/1Y9v\\\\\\\/XT4URqqZpbtVizkh6O\\\\\\\/X2kXLDH3og4nDRKv4ZYYbYJ3ycAE6s7hMnd4oiN3t5PcrANVP64ajV\\\\\\\/lpzsySoRAROpU+Hzos03X44k+OUSgHlhbu97V69Rn45dpMugbo5dXSbTu\\\\\\\/KlYPN7ToiCfao6uju88Lgmak\\\\\\\/BCeE+emUdXrmvsuwqGqMa5gKrWge5MLKwLsuDHeC1545nvd3LVyXLV+zYAHdbIhnNvFDfmwRKlrTFk3YHJVpTX+fhDSfblR7wwF8Cv5pPH6ti\\\\\\\/mvc+NJAsIeVzRZ3kvB2gvz3l\\\\\\\/Uf8KY7B5mB\\\\\\\/jn1WkdiPbkKKUmpZr0r0lmTY97CfY5Vjcvl5yzc0KqEQpV4oNE8mMwEbGipsoyoAyacy29PPIai+4QNj1VdSHoOi8OUT0FtBNsj2GQgI7gn0pdpSiYYiHAn27E2kgfwQQIfmqGJJ38GgKDNb9bxSZ91LE6zuOGA+KsReRVds\\\\\\\/46CEz5y0oPTNaNf4PjvVK8DsweTERY2QvWV673bYh6LxAcVAphnKbPirzChNW3wFtJ1zjSdzW+V8mSq5lLupmBkQZOYSAWc0c5zziO5yK1vVpQRGbXjJECMnrx63FPe4QOlIKLyx3YoPKM1BbxXpsL8J1VytM8UR5K9apzNEgcHtCDsG2ZWa+I2GMVicqg0uMhjBDsCLXnB4OnwcLe7xbuss+1ieClZkkLfbmSSE2ImB5Ymnh4D8q4oxfgKTeumXgAt4mTZBC69NCC63C+xer2F+3LHZa4G+PCt5eRvYpbFP7q0R\\\\\\\/+m74ohJsZ0h4qiKWUFZ6CPjfECSDyLkVYxT+86uK78f75YHprjKLadv23RfxTKg+PKUU0df9ULHjPlomsYof8OfD\\\\\\\/iQfG+8xPivEAqqGyrgYORjn92yD0Aqsvaq0aO1buIMcgDKCSRkZ0Ljv1loWmFgYIPcd2sU95FCl+u0C9fMGC4ltdB76GXEs1xCCTGPmDPil3pjpb1yDYN5JrHudtFEpcUGLsga9Iv1T8M6lw9SE5RgXHUJA5dqSJ+sQ7isJaFLmAag7eGCFC0WThE6BL6iLMDz+0\\\\\\\/vsIm2SHMJzkLm+6V05u04MhdVl6TqILhW8VectDW4EIzbcRZXHpgIMg61jSVAqJGSMP\\\\\\\/CKDMwuIYcZtFbFtqc8PyhnC1jrW1JQ+evmm2LRUzP+pdyIRuIib7RbfS4lPfvg2yZPxGDLRBtZVTPcFDhCzNXMCheeYpQ90PTNmhZJXSNBA4mQgUJWIhIRV2QAxJ4A+I5sa5iJ6H27K+fOK\\\\\\\/7+9doer+tkttVXzMCwGRltu7BB9RZkH7FyvYyVy+4px+2e8YELOPxcIFLm4Gh2bpwPjmoEVPlunvfLuOgag7wMAPappgnWsf\\\\\\\/jg8\\\\\\\/cOpXukEmN2DbBGxV8t3OFDOwbY9DLcs+EsUNLwdhOX1YlwMcS2e35ftB1wJm9qVPmcTsmRdmIURCRtlOk7Y+kIs5n2lZM1RFUBRmVok15CHnJukdNxrksBrdcRLiAMap5tfw73LtUEhU47DK4benYCprN+e4zpvwIFcLctf\\\\\\\/nQ7KFJxFuoVvc+RD\\\\\\\/uKgmKhiFbDS\\\\\\\/7c2je+HSSWnzTvvPEeyThvTE3ULX9JAmHbkoW4hdNEvdZ1rod1PB3ttlX4JS6jBfeDfYl2MGkyJiEbU70UBa+RcjneWiihHCQLtLFqyuPYcCKXjdz3YZj95AaxgoNM7QTydszW4YQ1AUYVaWNttS668I\\\\\\\/BjPX7Ez\\\\\\\/sfYpmUvrweDIp+6AOJTQShaFk1rPaRcVNo141xI9fkr\\\\\\\/f9mYoTt\\\\\\\/\\\\\\\/mBU0uhe++6TX30OzbNffMTbJwk3qBdyayuiKzExtVH+sW5id\\\\\\\/P1QFMSP+o4hfkD5+aQCJjO1IW4RaqKUms9igWB9+yuDxnfy8kdnoTSxjKHcjscRM3QTFXMJNJTiepgc525uBlX8pqQ8yGCNp\\\\\\\/hoq7KiBQCZ5WfK3s3QHOZ\\\\\\\/Ro3DmaB3mmAFSU6MAnmMJ4kSm4j9dPR\\\\\\\/JG\\\\\\\/WX0F4ukqjisbX2OiPtaedQMdzN6BxwfzJAzcq8f9gZlX7z6TRcKHjUAeGOsZqhKYDPKJGy0R15Ekzo9NSFwNJ2LQmtARqGvToCNKkHKoIOuzsDEF3UmadkJklrbayv2ulc4lDahZdpOPV9dZfx+kw+Ypj0FzdLCEh0EK2hNqHMsZO213G2WprlHyWkDl38GnOUVDbHZ1UTi\\\\\\\/fUb1y9BMbOB2\\\\\\\/qg7gjVT0ZMADqdPmNHUiKXXnmFjlOnSi1TDzg\\\\\\\/XzTI8Dr9+HU6GkBRDXlXdmQ7wJN2HnTbBgiausFc4cjpolaYqqy5oyEWlveqWjF3IHLsldhyi+erc\\\\\\\/HkFDZH+2EbxAgUoOsLq0Lbcl3tWxsPMixiXul0A0dHKrKVd99XIRkW37R3hHN3Ovg6pFC\\\\\\\/OJySM6jPGV6he7lqxsMvLsdKmVBXGcYgL9OBclsK0PP\\\\\\\/uPDYMS4ZvkLPmND4XSONSoGsVz9TPQNqy4Un7sbZkbdY\\\\\\\/hM30zJS0AWBNSaDECQWW+nBUF5OyJbvObppx3ElCCY6B79k0ZaKH5YCFPFVVvQBOm5PpujeyAkjmJM2mM1m4cCGhg7x+liiL6HbzO3OWDThNrJcmvdvI1DY9lDisvHfawlMfhNw937N+gdMs8v5EgJCjX7u28rwqIbxrhUW45COYX8rhaKX7c3Hn1FXeM0lT\\\\\\\/1GZBl1KT8DiLYQwpAaUf3KB2dwcobB5MjQXsVPnfdaYhkV2GRePXhERIjm2qJFoVT0cJIsvt0mfLnBExOhhx6PYr80WE8IewqAPHKAvuL4abk4T+NOaX+u2rhP00\\\\\\\/JDhD4e0gL6loAsJ6MnNG43gJOJp\\\\\\\/tR8k5qQBwLRuo4DsK66mjY\\\\\\\/\\\\\\\/80ET\\\\\\\/4hTyL08XYImWb8gFF5mKdU4Fr26dJ0mGG\\\\\\\/FzdjihJXq7S9VMRnDEBvpdrVxF896zJp+BOwq11ZC4gfEz9q2Sow4NClUBH6rJDzazowNA4LXAGQNd8mw4\\\\\\\/sknBbgwW69z+MLP+vxZ0iOWMHjlZKhEsNOUTrQ68pqgMfDsggWxo3sBw10JqJ1bMDoDzeM4Ws32moaAQ+Hf2LonZEX2hoN5rvw8unL8j3OtMD\\\\\\\/tJ9RetqMwMEIiK9VYN0vB7Q9LcZDSSnN2kG3+Bkh35cyV6N2gmqLpkGWWFm5k1\\\\\\\/eWRd7WCD6O8d1JVkludOkqXrCsa8w4F6zatqoN4ZVRuwvTNovGUFp6SGxHcIpYZJ6R8rrC9qJYNtBBboioSP+zCv+m5XdGgUjUYJZ\\\\\\\/0TG4q8s07x7k\\\\\\\/wyNrQp2OyfdC7xASlTKIpOyyHzHGoXTWq0tP7cGXaGDIaSTab0X6pkPym25jAveBH4QpRuHkczZDUv00CPXfFWDuveiWkwx4iVgBiL47V2qTM7Q2vs\\\\\\\/bOP1F0SXXkwUi5jeefYVAqHwWiqv7o6zhIjcis\\\\\\\/gApgwuxXeiYzLpnihuCaqCtI6TGmKjDJObg0jt8tYd2YFqj6kLKVNvneNbn\\\\\\\/P9yBMJ9AOeyHspd1kZkisAByXvnKOnMkKRKtKjhrNLZ5u278AGGCEi3\\\\\\\/AyV6nDPBEE2EP3dl5r82sO\\\\\\\/TPOYnB1h1EpeLEMbUaKGPL8DT2jJhlwZXRntxlyB2MPlYwSLrxxL7+pxUtQzTjhGYzn7HvTI3f0RFxdZMb3xkILazNa00jKzq9LiaUlk3Lh8cFI4lqyDa5xqxhJfPpctIvGq3nJckNsflj8jWxz5JEH+k4QghrqYEyUsKGkrZE3TKFVJTjtV6fp8hm7dH19Bzrwrff0NmWIYIaE\\\\\\\/r4hUh+tP9PwHmV6UZZo0TrLDSKRU8gPqIBgFw4NYUI7ceyDMed7L1+H3gTe1fR5PZs8smMDBaankd45d94DrdNNX6qc0PVvBoaJ3wkO9j87EO2pFgozxAOBx8BKC\\\\\\\/XECifV0thAkXelWBRIyRObAnerGTwWrE2G5GpNJDLOCGcuXk3BdipSZKb\\\\\\\/pBsXoYk49Uz1oaoSNhj2Ex3matFexidha8N6DwpruzW7skomkEJojYxUzpir4bImIAAzlEvTxzBCTc8vQcXV3znhJQaIKeJUiTPAY2wyOMMdsaiqxWzIOQTO9JMjmjxdww3HLsSB6l\\\\\\\/UJOqBLeJvjVRKL4R4RrCtg9xOIlVFm8aX1YnpNeBtTMrmqcdu82q75o5q6pvFAcjQ5orLqJdB8abdmDMOe32A6KyxEZGP\\\\\\\/Mr6EljLomJKsEGZ7+VllXFUKmR0sfgVjo7YIY7dFdexzBR\\\\\\\/75Kp2hRLCyA9vfNws1kzg+Gxuy3PgTI319uOtWIuA8bIfT8Hc8b7xYmYcVOVJGgz8atlLchs9Cu+dJpyj+XgfKRW8V5gTrBxGgZ+8AuB0BKQSSanFSS6cVjGKxNCH18O8RB9WgTIBHJ1SJ3hyjysNpni1ih91uRlpbLfiGoN3UlOBe+DCHOOVRsN3fS0a6nv0Ag7h7XiMK7z8PEq1Sucwd28wjPOz6RhUXZYClIAb2IL7VrOaP+C6upUJdk9bU6q3o6lXWRHwufuob131phOV6fYhBkxLBLxhxuHij6FUEpqF\\\\\\\/rF98KqcLq5DmK7jWotvfbm6PMezEG7SLmupPfC6RV0XtZj7F0bLlbqG9klAhLum1kA2R6FKDduWWNqCL+Vkv7bI+dnhi5tTMo9HvkHCZhEXtM1RJ66+qQ2p8dhYtMSuS4r9vWejOZmad0rLty2gc5e6PgEy\\\\\\\/iMcuMdsNyEeF\\\\\\\/A8y\\\\\\\/Gx2HihXGqytToVzwwRtoFmgjn1jHnW+HDfQGvcFRapLpjh6bYIpp9BqRRH8PkmKHpcIxblcWj7\\\\\\\/15vBemfdV9eJcv73podWNDB8p+w3P8LHEhaqYHTecB+gbQKhHAD08OuA8Mp6o7HWevjinQbrkyOPIsAckBJAVkzfgYyITiQeU3jw8bYsDtMaoaJtLyrsMci7avNKB41A5CEw6abIrzLzBkBVLl4YUq9CvfgnRkzOtBxa3fsMYgAoExMwZg1upNR33hCqbFFLGvo30OXN9bRczCoPR0I7sxCK7WwIKSl3Prc\\\\\\\/j7sCPMBs3+13zf4PW7jwcmtByBjBWxg\\\\\\\/XD92J63VX6jtxoT4Tt6jBzcEz9gRJYoej0Twa3CUOIQbbBqoAadPGRG8NMi7NzTEdU4dCKyugpYmkOHsl6L3AFLUw44brZ+XSlTzBW7N3DIj2TMXJQtRVH4N8r944k\\\\\\\/k8SgoDBKZXc7WVIwuUJiKmkldHaLwzDOSvvHtVPKpGe57Kmvrx3piTESJtnisrAs7Rq5vuLwJ9J+D+t332ZkhOYA0DbC5OJPyq0U1VTDmCkp2IS1MEChXVzwq4YSyO0vJa8LYl4m6meo4dtiwACIsY+6Wc\\\\\\\/lO3+Xt7Y+KYUHX1D6svNrDLwsqMA+Pl+gBkiiGuM4dZyUcEplxo7Vofs5v0GtcuTLyz1dn7TX8uih5JH+z3C+fUMo\\\\\\\/8znfhMEqN9312OedWKR8OrnqGfntRgAv++YR4CoiPlO9lfKt5gNZdVOfgPYwSKBeatAUQEi4kM0K0cOWX4WAGcssNJjRtCO5u4c+L\\\\\\\/n52+xqChU48Hc0yAQia+KwkTM0NzfAIcxoNa47UFp5cW02hBfcbj+Uy0nvTNdF4RLdPREAzmB3A038Z63It3LU429\\\\\\\/HfYqG08qBVPCpQk3mf1IqJKNVMy2cjYyy8o5J9RdLkAgo39MzEeUXxH4yKyjOuMOy5pKn5dUF0GON3IZadSL9ye1tGddhYCqOnmD95IruYt3UiN\\\\\\\/1KZThsDFFVk1qIULthjcj4vnVGz3Jte\\\\\\\/jxh+i+LO\\\\\\\/K5JQFWXJIaPf3qE+bvMJFv\\\\\\\/wZ3eaiH\\\\\\\/a0jvj4tAxFLrqUoTMx+xta8oZCyIr+esFMwbXeblGQegsaUlf6lfnmlIAFRXv61dpcUXopaSA+FAj0yy06rslh1Ml9K7Ww5+fPV\\\\\\\/+Miyenj795LbEOiTm5Y3eZgl\\\\\\\/MN+0mxvB\\\\\\\/Az0nWVCXBcUHBJTLaRAiJUZ0ksph5SiCvFbXXnaqS3HR9vN1asqJuSkok6Ne2OtyEwbk5zv+psK8NF8PVpQHaV9uDnTNdpdxxkx\\\\\\\/4afKQ9yECG7hZlYWTEd6QjmM29vo79WLW1HQs6BrNHEoXlr\\\\\\\/ZpoBms\\\\\\\/xHhgDIXOZIxdO3dUDNyevBnPghs4WiD\\\\\\\/\\\\\\\/70969xYy82oUJny7bnVJ\\\\\\\/QQpv0TwzBESK7LoojGEgWH5IyDylXJPeZlzFo9Rq2F56aUGpXBpNe78lA\\\\\\\/4NfOLK491\\\\\\\/FgWg+7NaWq24ALa2skOTc98yXMFtiQ1pg7NpOQm4Nc0SyyZNf\\\\\\\/VW5+YASJ0hOe9D6sfCjYRvEjcTsAsxq+Q6HdY7Cu4YvcIsfH98oDkfLqvtwnZrewQY24ZTvh8Bd4YWHBPuuBJDYKdSfJXAD8rU3HOa4pjzb1jwCBZzK9NGbCUF7zWmoFpT8wTrrsEES1yXBda6ZLbS4BSoYVy3fZHoV1+B4mrpYTiJGj8rWUsJPk7OIeaY2lO7gEDpcZrVPIq3aQ0JcM7AS8edDz8nxZyLyT3pTAszrXcXm6zfAqPNRFR2HQitbG9KsbbDgvu0uJrjbmv778cR3CTTNlwOF2CN1fOs\\\\\\\/vDU+UwmP5Yzvt+auMMYd+82s\\\\\\\/enBDFotV4quzZbz53WX0DCnwj23XshRZ7v4sDJR+gHClFJWG84TOuZdc1hfeVOE+E8jcOXJeAHm1RjHq\\\\\\\/DxNLB59VJUnWa6X8vZNrEBv13dU2Xj9FjZM6XlBD3Xlej0wFXXlbqpoXUlYwd18sQAGUhlWcVxE5grKFrAgJcv2IYzrWxX9pG4xB9zlwzzhR50RZxZ4XfhR4GxtqqYJ42FQemQuWh87+qbUYZ4+cVymSGDGY4uPyYMqhPErXoEgO9\\\\\\\/2ExSG3sxCDISeAX3gy7IsmKC6JY6+P0yeHHkBAs03DpWGc5NnKrseiADA9MB4w4P3IzLE756I2vA22GbRuGGWSIfPNB7\\\\\\\/cIlcIiGWvb4g2EyWSdq28ga\\\\\\\/sOIgzl9IsggsScj7DuPmhIoXTbHgwsxdCOJ7DdkzmJW+CVtMa5BE7pNKJW\\\\\\\/vOmJw\\\\\\\/AlKN5se9UlsRXPp36aOMKVFehtOMpZZldSZgwrxUF0qdb0Cc0pv3txhFXFTug4wv2gL5HMbSwDC7JgCPhOLfHjKivdz8Eczk95y66XsEhQUooBhbvsuJwZ0tt77skdf294ms\\\\\\\/DwQl5n610xxqqJpB55\\\\\\\/RiBN39yKoG6PwIoxJMD91fF7Soi1hXpRsj9azQsLIR\\\\\\\/TJ\\\\\\\/K0nKNgrBXfFbfyMY2BdoCic7M36e2Ukcxx8OohQ8Gad1CCtvRSRNiqioU6QSlyiUmOiPHHAhBdhiFsEh1H68o9wVDaIofkW\\\\\\\/2Z+q10VUaHE\\\\\\\/qr4vNlecbwa1qPf5Z31C5pfI\\\\\\\/MrY2knYtzZ8rzZ\\\\\\\/a2IeRJ9zzqo6hV3gu\\\\\\\/4H8ESmtoTeAlKe0sx8uDu\\\\\\\/837mULgvSJZMja5yu6t3xFL4vP3BwzU4\\\\\\\/qFHc+Iu1A+pK0dJd9qPSg7c29L2Z03CqogQZ+ztvDvIIBcwYpyxkswKAb5NiqAXMQPgS6aAYtKZ6KdzWIPBjnCVy\\\\\\\/+V+QAzRA0bLQiyYxvPyHGQsOjWKn4kbkiLt9XyVDhqnV3r78VGBgank8Xh6LkfICdnIIa2OB2AuuMjDUUrtLDsytENYc6zm\\\\\\\/kSjtfvZRLeFXcimhZnKrQ8RaY1irFWWNZE+qxb1yY\\\\\\\/iLdrD2ZOztDJtIiRAz1jCVODscCw+1s+YBqgZEypfJgUe2Xzkdp2IZM\\\\\\\/J7epDd0FsvU3e7VeKT8Kuvy+vrm6BzWhtq\\\\\\\/DqHZnVVR2a4UTfhyDn35UeXJH0BvDif0m45VqGLA5WjSnXiBa8VvX6cKA752A1Tu6stOLRnJcwQtwTS+1Xf5jGy8VO1J6Rn+yGaMV7a9qJexJCIKLN4qBq6FRKzZJ+e9yHPQbsNSjoauVgZXatNRhWsinHPtQO4tg\\\\\\\/a9sncR8o5zSOaSew0eLGynynEfgRxIK5nO5ooXdeokXzHyvKFpy\\\\\\\/nAfb4MlosNLUFECCiYdTCwdG6jrbmQC4zhyJqcgir94jyQ00lryLLnfZZiS3DXIW1W6NQJIdU2pKdZLO0CDOKGa5Gk29qVy3\\\\\\\/I2mkiR\\\\\\\/+sYFg7zXE1dXRMotVWhOSdf4NQDVcM13itRh7EOfares6bYIBA4A+DwGAkzf5YfSu\\\\\\\/SA8huLrzilD+II1oj3xBJr5qUPCdDDap80lAVJK\\\\\\\/nxJ5y32CqB4b1qLX+xrRiaDU0uoGojLlyYvuVS42ge+4AhSDy1lhsseKTvVZl744+6JYsScr\\\\\\\/V+3yA8dBteFoetnEefVBklMoFu3YBEdgik5SDjboXxHcYiSCb57KhLzVKnm4g0sdcfI\\\\\\\/7QQX0gPQKtxhVupvTKdg2YjGam5wP5tcBslr7Ep7icxI7g9c2NzXLGAaM\\\\\\\/+XK+4qqkbTGSebxo+8K2uJ57\\\\\\\/S4cp7EfGq6Rsu+AzIndwyr6YII8ObRtOmwJyYfZWZZmZ8cwVgjUImGaEZPTWjdGwcsfk6fgqbYECFLLik5v+F+AOzgK5zCeO3k8A7Bzu1HzP55B3anUbTwqHInYEJgs8Zv2EQPZ76+a5BnKkog\\\\\\\/yQPMCOQNa7euurDCDWx1TLS1pE6IFH\\\\\\\/Wj901Y1taHkQb3SiRRzH5jRcf2of\\\\\\\/iyeOckI0rfo8aifYRSngFq33RBfepaDZMqmypB0oAX5wT5vqFmdi4SuNb4M\\\\\\\/jsBZNoelrF0DQXzWCccuhvLyypQRTz\\\\\\\/RdEbKaV2znabo0aa2meZw02V8TMf58wcUb3t0Q59yb4tAISfW\\\\\\\/AfDqpnB\\\\\\\/cZ5xd+BebUVrYf5YiD1VDaBJM323MnAM9sBCOIlZCfya8ijN2f4pu56HWg5xDnZoLb4\\\\\\\/XDRjt8EkZoIx2CGdzxXEF3XcMPZzdjix5djBMuz5p1ub\\\\\\\/ul\\\\\\\/Hq5Wzf1z9x5YzX3YEnF1MwKdVhut0IIbHen9G6LLa2AbqBLwtL7x3in\\\\\\\/QkZSta2ATSG+ouL+8xNbJO0mrUt\\\\\\\/yoWQWcKnK5U99pauaNTd5ZX+\\\\\\\/0Z8hv76XD\\\\\\\/A17FX\\\\\\\/gjshoQ0IS1zoYRE1++k098yDHebdd54\\\\\\\/BNOh+8MN9lXn22sz5etUC\\\\\\\/77GMFcKH1IqRlHHwTQrIbKbA1hYRfjkvacV\\\\\\\/hCHgq84aaIZMQkfwMPIjQE0ARCtycFXFjzn5o5QeMx8dRDUZQl\\\\\\\/CV8zMVUx5SReS+cPWQ1azKOVIrSIY3a6IqMs\\\\\\\/WNxMSi5b1tWd7bCDYdZxppopAvrGlC5LGNsqFlx28M2HbRO40db6IAU8Ogxts0eRYWotiaeIK+T1v8dpXvBSbyGVhc+d91Nc\\\\\\\/Ibqz9ht4iri8WzR6RaYKn4nkAa2etGUnMMP4mFnfNxl4RK3kmXV1AECozjSYXMW77ZIZwBJgWS+6KCXVFHsLf2AxqEEca1CTCzycGmbnIQAF+Qhx2DxRdiVaEwVmAdGlcyblBqWazegJwm+DhieGPiN\\\\\\\/QDkCt3eP3BV8WP3OPm5A66k9\\\\\\\/LQ\\\\\\\/mInnJCSbzfk4YsdD4u0RbEJ09bhCg2iOn3TWgrU1RXtbyFgNYcPalfZ88uVLRthofvvvhVzSps2oyskYQOWHu4kOq+ze14vQn5Kro+mlwZKGHbNfcUp+KIfIlLvf3Iee3ufpVRI39YSNPZvU\\\\\\\/kS8bQw0G1zBsFdNTm49Mzgq5QFTvrxDrj5WlVxxOgXkrowBkA23DznzHv9gtUD0JzoWbi3vPbI+W\\\\\\\/AvzaaRSbReUzQdKgbCAVNQBz4\\\\\\\/GeHApVP14CZOHtaBqnn5GheSi08Ip8dvMWCmRq9prpMuNk4y6AM5LBM\\\\\\\/NT1mmbLvyP7+g8162djQ1Q3ydAxGQrJxKsw9G\\\\\\\/fwSrqqLzn+Vk+l6b1gxqXs5wT2GVjG068yAR7GGfGsb\\\\\\\/s+MvNNoe8qgO01RfsvHth+TRnkXTo404xx2uLqElz39ruCkfuyosmd7ZFmLPhKcWT8i6LMVSlDl\\\\\\\/gXnx+5rSbmQ8ZWT6zaoGal6FZekhx2J0xv9Ws7omfqgs\\\\\\\/oPJgJDA2v+cXZ4qOiGXxbiynIknsLWd5E0hkeCOM3Q8pytWiqTPkxt6Cq1UOEQIDBe48vUmQIKi+fnez48IMEz5XLYN4BOHTUdQedDZFHUePXGRdKK6f11Zgu+OelEgVxS7jz0\\\\\\\/oyt735SgGmuHqi6Z\\\\\\\/wmG9TnuNDqvIyPVw73cauv2bsY2C88TYvDElJaSeV2ese90o4SaMGF6OoeiE70uTLcrkQAw3u2o5CRhi+9TRtrquIt3IvLkI3qRdenSDkiuADIQ7T7+4LI\\\\\\\/vwj+At8xOFa5h1hRxf\\\\\\\/wxSmuQpcn1nanBfHh26yQUfHnVfCtWQ38oXDEPBCD5SH1decZW3UG2eaET66kLPnG3GSFlz6x4OXYOt7jTBuO5qllAdbS03WlTgqLK87bcfk+B24FeH\\\\\\\/W5Lzw1AfpRMozip8enzu+4lZ1CmpfR0tOCG84ETvoFfpNBRtDiXpUTwVHCBi305FEke43hSMDHdfARB96EzwkVCVzfjxEHuhRn5+LwoI2d1E+ANym1Rl\\\\\\\/GIRupdTBn9paDNkJoodaALuAVVyeaLKdMTSYm296Z4PzxcidPM9z3i\\\\\\\/kqK0WbndRz\\\\\\\/PvuyBxAKn+PRc5+1YbgoayaRNvWfXQU0AERW8+TpH4EE1qTBIs\\\\\\\/P8vJWDhTQZlQ3sHAmXE9AwD5yinaWW\\\\\\\/M6bRVogeYRKyh2rDAbT9Cx+fExtfAKz\\\\\\\/EKfYl0GnLGImPpsGy4\\\\\\\/XXhuXLyo1X9bC73iCf0zqFulR61CQ8OAL\\\\\\\/wAUMyOpDuTO2orlROkXooimuldd+5l5S4ZfshcwtZgXzNGapy8J0nKRLnsDehW2fhMgKVnJWciYjhL6fuPm4wS2T7uoQwmjqg6Z2o2+P96opAeMa6De9qkQZ3nNBveLNZxW5LqCW7andlWckjdirJ73+nsfBJ49EGOeq5+vib4Vjr52RzIPXKuFWuEPIxBPdeT999jZqND+zcLjIvIAEsfXg9tvmU6GTbdtEYDENH\\\\\\\/HXxivX+SbBLDudgowAB1B0KCrAC4T0zuw9T7asZiTd1aN7aTuO3lBfdFmIHxGsvmxauZj5rnL0JXZ1P4F9zWUl7loqcWWjgapMFA2wNnQbNizBb7xquimY63Xg8rhrkKVtsGH12yommRdZIRAbFZacJ+NanlbGPc\\\\\\\/TYTu0rRiVOoIwirex0vv8WQrDdF\\\\\\\/lrucQHTCA2lInN4Q+DH\\\\\\\/mqHHaCt33Wv4qRRLEzOPKgfyA0wLQOomWdTjWrAhmSwQCIy\\\\\\\/KY\\\\\\\/B3\\\\\\\/rcW8XKvO+kk1hXpIB44w8o1pa6CCkAYWtWTSacaFGxXM22DOm4KbJ8ZGaKqTJuCMuF+L7CKGk8kPzQsAPkGYb5ZXesLoaA7c6+p3AIqKh22PZ7SR0Z82EihxOhI2SukZGXR148xDa2He4YeRKSBIyYqQyNYypPqjxInQ7xXjof0h4wlODLwrHUsMqXZKYqhz\\\\\\\/1Wb1XM4PBPVvl3U4TsQdZaAPq6fg\\\\\\\/RE4Czvvq5r54eS0uOBxaFspfKAV0Zf0cTvYfSIJ1u+iEnjkCGgxn7m+nHxSSih9qO5MNUMnOCJQMtqCgWsSsJloQYX2KQYAe3tR3PGxi2A9mI0OVe7ARJALijNpWVED9iuXNP7bq0Koe\\\\\\\/NTIgfGNSu9\\\\\\\/F7gnKdDDC26ubkK3Zrg38VpvmMU31hkp8Jcn7Q8To0Ou4ByhnMhnOwwVEkPHtXMV9IuQQb57xAlqJYsAETrRgRVkCyuYJ5bVyIJwICFcpH9l9Np6bRzAQEztBYr0LTWdP35xsqNfwo\\\\\\\/y3Pgx7PpGoZ0T8WE\\\\\\\/kSLx27ELtrNUxSl2KIHDQNS082D5dogATddQHHvJXxD5fyCmu9UO05TAMIHT6cJ1dpTxvkQ1vKdUAqLv7D1\\\\\\\/5r5UcAVx2aGzkw2ZeX9Wex2oLWwfMtQd59b9mxoYHTdhbTmqNfN5bjlRpd2r7pgjGr7eZyFADyf\\\\\\\/mu\\\\\\\/8ni+mfQ1UcSOKqRzhGwvR+NYW6TBQYvo4KRMobb+orP9Vp6SvWgI3i65R0d4651pNQQ4HMvbiGXeURMWrSdA6EOWApMSle5AfqbjNn1ZxYCU8AG5jUVS+5qDkZsnblYxWyHbsK\\\\\\\/RGG9Etz4yFXxaPuZ+bdfNy8YqtGr6nfHMr3gdqUbpiYf8z2B2icKrcr43owi8oaS0INf9C3Z9e4oqeEOZuZVIWItMW+pkDQyUwUhokilpDKsQ1UHvpkkuIIxAUbsxC65Mh6A55B63eNBNAPqMYueHPWXc4VYtB97kLPWgaxPL9Am1jU1agZaM+UIt96U4faFp2DlGWEC\\\\\\\/fzLKlcyx2QAqzSLQPmPVWsW7QgKj8CDFmkTRp2wbTsFIx61FYAsMyE9\\\\\\\/Cb\\\\\\\/CTbe\\\\\\\/UGULwQ0xoVu7TF3woOwH3JE7AQAtk5ehC8hgV1Q\\\\\\\/n\\\\\\\/Xpavfh\\\\\\\/pgCajp7e\\\\\\\/\\\\\\\/wVaxFseQm97qmfxgM6yunfrY5nepIVmLiyhF87efCJTe0NbJ1juKlNh5\\\\\\\/GqXWS0mR+y9RefivyaKGg7A4FIGJ0sU8ma15tM1yOYdJ1NTHd6Nzz2eQ6e3dpAPs7hhVT8pc5jGYadOi8Y4tTXJeKQBduloX\\\\\\\/t0IzvgFEkM4H5SVB63xsrhL2h5r65eiPu9dDyWZYkZPYFryUAakvmGbhfgs+aqu6RZt8SG\\\\\\\/jYMR5D23\\\\\\\/\\\\\\\/ShxaO5P3xyzGgMSHRSEv2Q3Ibx9W7JUzH3bK0apE4qEAELXDMPxNbY53C1PeaO4JGNttP4i5zVttJiAbQY11H0cGN7T9rmnGSbQE0ybnRWqk37Bvq70mX9\\\\\\\/RlKCzBmJmT99dBshsWDViqGUL5g\\\\\\\/dmgGC7dswAKkMztgUkfgom5fZzW+2nzm+bwAzSRWGXnd9CZh4cLr7Y8D1W+aQwAOdyqYc4HsBRz5muXN9OCGQg3zow0ZjVYJiwsC778m+xdMPFSqoaORRUMGJMV1yoFZ5aZqJ63aSIjgCUKYiUm4jDOMBXPIa36g3z39kY9N13py5Le3ArO8oNjZ4QTeN8R0irevlczu+VEu726S7ecpvVP1OfH2JjhUzN3TG\\\\\\\/x+c4nwHruVYS3Fj\\\\\\\/Ht7HxUNFb2Y7fEp64R7QHT3jIXI5xh3DlfcaavRhLKTZdEaouD5RnS39eS4ITIZDDbX3xKLRoMHARJbeU0byl+zJiu6uPBRjoXJipo39onbHeku+hzy4RmXa\\\\\\\/wX99KVPWloJ8wMPlnKiUldFl76Q6ggri2J2Rg+MCP21HFqXkt7xQz9wj9y7xgurY\\\\\\\/XnCMiDQZCQ\\\\\\\/uJ3BedAPNABdvpz9\\\\\\\/2IBB2wrxlOiomcYd82yHbVmMdGjWDk9OkDSiw46tEg6TD40SR0BFX09P+Z1+LpwTn5sCqEJUuIJ0j2NeBTe4XnyC5JdpSDW0xiWj4fe607rrle1mM8x3cngknv1ecXaIx8C9Ny0rlLKyN6HXkF2yYEjdHw0Wv3icIfxXINo\\\\\\\/UUlcp5adU0QXV+Zb7RIEH9vKkg3sUhnuhqED1M4sYnYoQ3JE+04xufWOnMp+3gPYqAfQJ8LMAJ06p8nCpEjmuUvdEm\\\\\\\/eElEy4AY2aZz27lYlbexkeGUOTC4wAKSMidAg1LxApGEo+bxlkyJ6GWzVDf4L9b4LK8OMzmPvtwmDQ+4rGVGkatypkAbhLYFVAh+0106OmAgOh9iW33Sl4cPsLRHFZY+00d+FtlorxSfYkMBfU8MFeC2j+DXnLcKZoAsyb6iekOMf472gN\\\\\\\/3+pgYdUcrzMd\\\\\\\/TR\\\\\\\/JTjtq8ZVKUnVsf3yPCXZ3Sms4uFJ3knm6zuJTkASF\\\\\\\/jTltXceuiI+9SPuI+LWl5k1ycXHvFUlHgTYPpJ+SCKRdTYMvoB27hSBckmyCocDLoZVKhVkh20pI0oVQKP7PoNijD8Kb\\\\\\\/toABpFggXdx8yRwFKXkJumdrPtbl0+092TgYUNg7LiWak1zn9AOkyrbXqwzUyIS\\\\\\\/Jg5HxgdTbZ\\\\\\\/k\\\\\\\/QmU9GOT8Q+FoOY2uEnZaYqAnIhJuyL9GWuUeHrm9lpsOrV1gSXNVOvnEq77Ya8vvhxt\\\\\\\/XfVphP12TQDh\\\\\\\/v7gRw5J4XavrlG0pW+7uOLy7EDpamsjNT+A7coqQmh\\\\\\\/mIpls8WgLWtgp0HEJNZQIVUFeMjVhP+9RzHfaMjfbXWFMwjPrulhspsmZnlkN7J\\\\\\\/x+wAbjSbPol+uHXC\\\\\\\/G6BOo6Qe5G30cALxrAXn06FNKuVbU47iL33ymhO09Oj4fa5tZ5rNkBq1E8ZVvRGpEBOSnjZVM9pYrX+A3KRo5N9NJFuu0ZLzz8MSLuYNZirmCP+7qscP3\\\\\\\/hyr9M3OAhCxofPaHerS8GMuNXnTV6HI\\\\\\\/hsD6oHsfBDjfL3E9VlHRwiPD4X8JkxYJquJfoi1qpoWyOZ3UMAH+55bORwruJjjUgWBBekN2gI8hxBPpn\\\\\\\/nm9eES85tt+bjBMRBknGdxmViH\\\\\\\/qTosjOqoxHWpb2SxLImWbkg0OzBXrGOcvob3mUfsx8GEqa85D5rdgTeW4jR2z7Lj7q\\\\\\\/K2ZYsg9n4J1B914mn5qsa5j5WG2bEj7LHO\\\\\\\/ANgoHGCfkIKrhy4AgLY732OjDINFJDnnDFNEI\\\\\\\/cc4A3OUrCfULsRf4P9ohw9vK9fWzeznHLnTseIEzIAFJsfeSOuVr+uGAQmwhgHn4XCseo1hA3WypRQ3FUSSZaPAPeR0NnjmXr1mnqgrwb7hDyHrZw1qr+FWRoZCZ+ntF020lwW8eyTJt3c5j1rdiG3gflUstSaLvEpz7c8K\\\\\\\/LJdJffajtaA5a9JT\\\\\\\/W9\\\\\\\/eEO35t7eYvnuLD4fKB2hx3RIxqYIVTyEOJ5HiGQ42QWqabT3P1DG0UY+2T1h4S7hFb\\\\\\\/D32\\\\\\\/fs\\\\\\\/lcdZZBtsAuBofXa3Cy78zLplpVu7KsXlICxbACvGhSJ2+HGHVg2RBMJ9+\\\\\\\/\\\\\\\/NbZNnLtM0O1QQf5ms\\\\\\\/Cpb4EGfqcv0wBjEHKwxMsiqILrswXl4gcMOPGvlb4BRv958X8dpagUTb4FA4IV6ONGIO8VWR1W+Ru99Yf+ZXHYAfUhPKkkeh7eI0Zp8wz87XiBgcJGySg5alYYi1Cg+ZWDDYHEdx10Jd6jNA2GKhFYXC09r2hEDWCgDlp6ieMz6GaNOtwmUrMWY1pPtjHql8O6Q8xY9dYsa89UtReZ+zMxw1PSN2aQoKxtRlwb9CpICXWB6LLZ\\\\\\\/albGsLU80SvovV7IxnVVfM0xuyNvdTUKiMPQMUqw8fUd7fLCRbtfPjnIJ34uq\\\\\\\/6JBPNzTCaTCWrdLiAQx1HuDS16p7CoBviCM8uMEw2fsVr5U\\\\\\\/k6cFKWTukbea0C4snv2C59NicLJOiG\\\\\\\/PAoh96qUqed+5ofFy9G1S38AL1A48Wz1FMIgiDyNnXEFjKehaRt\\\\\\\/OmmD25cB0JmmBk7L1sxOUXL2oTxTFHWofq4yfrZLjNE3KpACiD2iyRDIfyTBx0I0yGfy\\\\\\\/Hobyh\\\\\\\/vD4HXkk5W46uBbMtp75o8CW2uYNZyjKybwgRyBRdN9M6M4FyV06BOu3oWv73TWzJDV9lXpvDsxFSioPpwaBW9Fip5eWW6ksPVqPyMv8eOyl7SVadCeX1omczhsMOIQEZqXJV\\\\\\\/Sdad0ruuXXQewn5TxGqCAr+pyTPkmoinQ3UL+jlRkyJh25+sv9N4ASjMJkpcaM8+AqGWhqBtTV71NOHg1tsSYwjbUuFzEChvafMBnl5LZyPq9dh7pp+zaELgwCG09kyW9wxAmmKx5CVBbHtSkLCBHMyRBkg5+J9a\\\\\\\/B25QKEl6O68Tpz+NBNmVmmMs6SdUhs1kTQxOc9AXW+ZJ165nkXD42AfWfG60BDn2aXfuP+VSIrwW7XauK02DdwCfLUYiGCiEgGSaufcOPcNEn9jKgdJY2\\\\\\\/c\\\\\\\/qM6eDdnKf9habPvkBfud9Ne9gwf70k6DnT1uYdk8eeuXYEYmaxpl74U1\\\\\\\/fpu48A1sP6F14KVWL3w3ORgFFfTggqS4sAXwf7ITtuwp\\\\\\\/CT1ydOWvjL5ed5xKt3hOfif8UihXXyzlSqMCv4fo+IZSwcV5iaCjuMP+oAV2ECWpKu8a8HJd4X0Wp7JP\\\\\\\/k3idHW1YGwvzNAtL1arOELNn8uzkTndQ6zMVvOA1mpFEBI0Tx8BVgdGpnsBc6X87XkShLUUHbROIqVAfebmEmZVNRZbt0zAA5aTIQtcj+DmobIKGeM+zHv564udKFhgRyMnaz60kagpwMtXfrqSt7samAVOaDXp53rM6VCBWjg97VSbwmNu+wmE8UJLN8WXKeWj8LZFPlZKw5UzPzq5RDwrZniAU1siAxfrwbAH4lZpLok6ghgqaR0e37teJsciFyg+n0w7eTtZGxhS+eT1JeaFNBYxSpu33MjwWxN8i6EEfgVTeEykRFkGRU1IOKS3fvJWKzMCRFGSFu2M50WLDa\\\\\\\/L3Y\\\\\\\/w4xMxZVc7Y2dDmVEFtilH6TQNAhFwXg\\\\\\\/\\\\\\\/+nv5XpYTXw+0hYc1pdimsKnar4dyPzpPjRaDIC5E51xzxOpYNoZSG8qEy5l59Qmi7LWSLEoF63frYwK3Mj3ok6O6ZN6ngRLbFt4EnuJCzBOSr4LmrZ67BBbwiWfkeofwQcCPwyQ6a0yx5TkZBZshuUansQtV7kFcJOOEy9kHUMxCZ5ikY8DrSdKFUmtvNXYZ5omghkDkjdOVF9YumSm3CxugcUpiM0PZqkg4gRUgpnBVa6eCyMfKFqO9xs3P0\\\\\\\/pxetb5A2pPguFtt58pejls8\\\\\\\/TM8e3gJUKex9BGJIW5Bh8pqBrLf3o+btVupsAfwDVhYqWDY37IPu4V9vpi9l18+JhlY\\\\\\\/OrCNzgoBNjDFOxIw5EXidHiPd1VkuBJerc81G9trcW8KDt\\\\\\\/cMZGDZMChAqnTe0v0\\\\\\\/zbBbofm6KiceiM9gybyKAl3m9cRjhx9fFqyQwPYQkvtx\\\\\\\/ubgJ92iIn5vsVJMBpNQvIdukYo6YGVs4QOQFuZ2u4v41A5oTYAfxoNnl2NZERJfVd3Ch4RqBSbxgMyq8DRN\\\\\\\/uojidn58sINzLNTB76DP2uEKklm4I1+Wd2fDyJ2I4eNRGIkq0Az6yIQ+oGRHaSxjACd+52\\\\\\\/1vDuJHYxJUPWMqTrsbrPZMy+SpcEUJ1FxIwz8yZiUl+VTpCvAbGg\\\\\\\/AabZELaunTXotTqiztYVonUK25CwmZTR3PT0X7r9c8XC4DnsxtDuFAusbsePqZCqja9gdYGtBZVGF69GxKGgNJF6Vn+Y2CQOjme8ZCN48R5gLF7tTsZd\\\\\\\/hTHhAE1QgIzFsM0JWeD0+pDxaoYl4ecdbl3RBKKgCe3xz+RIbOQgRs4Uq+mMcCfwgZc805ZFW+R2KuorKAAPIb+NfFgl2bIN5r6C\\\\\\\/QcTqmSGo5iI2cPzTtesFLkg9j9iWGspItw98EzScv1jHbpGK\\\\\\\/rwVEA7pDe4JqWAC4NyvyTJOuz5sU0StfMAer1UW\\\\\\\/4j8QXdVc9\\\\\\\/9f2Rfd5kwCSTRuEV3BlgpQts38N5a0Aa8lSdkdMsRHAVypfVtsGXQD3ZLGcR9wd+kdc4C0XPaFXMz630D+HdA1REXIQS4WxUbkZeHdEWyeHoL5QBrNeO10fA8lijg4a6Reh5qim1DAhh31R9pTmrUtxllVjqkv4v9tB9Wk817XpBCJAfL\\\\\\\/tl2wMivMFEHFd9rBaggqpjNbhkU\\\\\\\/U6iboXeOmSKVZ7jMmLGmrq\\\\\\\/hZIMvmuP9KbRUy2\\\\\\\/ay7738U9uYV6hQLMKsVTSlnMEXL1ZPNEA92Jgq2BQ+HG2VPubpzaFecTZO9UYQM\\\\\\\/vIZgeSaUbeLOufeRsD+yTAc+R2ae4EneCD53LH8PwqGNMZpM1UowNUtxiDYWfwTMrvu91zg6bOfJm8N6NP3+76RLENEZHjGU1RO7SVKXwxHB1tIiPOjysqItqyoxIS9qHj5YKG7LP\\\\\\\/NQWc\\\\\\\/DUQUSRXsiIKfIvdg6E2F0WabMhXYbGCi94rnUmwyA9kB5bNKGYN3y8OhAQCTLJj\\\\\\\/sC026vwVCdX8Wt6oDGdxFmbyDERT4\\\\\\\/flrQhQdyNxF99OoR39Ei3\\\\\\\/lqWhIcozmV2qv6YITu6r0UyPMBpmGM2tJsnIx5UZD\\\\\\\/8eXVTwwM1cCexUHm4Uv0l3BGlBOu8B0h1O0IAbwHAS6aRapbdbaK+MC\\\\\\\/FGSXHiq28cbm5CkMcbrQ0Ppe4S5WQ9m04OBjk21rZocMA3Lj6aYWzU0iHiASUlwGc135uEn\\\\\\\/TCRry3ll1OIzrQmmkfLppbSPm8GjpUIvonarQrp8zw96uoRfUD8DFKgUFL4hck8AbC6M34fHa8\\\\\\\/TQtNAusdLT\\\\\\\/CaFJuVa\\\\\\\/sdnECYV+tfXoIkKK5L2zD4MDdLaLrfS8iNFROblK4ezGAzIcPfNg02jkr63B\\\\\\\/Cef6ddjLn\\\\\\\/a6adNE9yG4zbzoic1wkfl+stDbGZq3+0BZEbgj+ZW5KGBmgmrFOPyaZR4kE34lHc5YLMNEfDNPqV\\\\\\\/AWjdnzhSMj8r1bCldYksybp96iyhGREPH4fpcFd+vda4jlChhqcxONtfzJi48q0++w\\\\\\\/bhMS7OAXVYnmT\\\\\\\/s1Ug7PIWg62WLl9KVUE3y4qBkE45tWj9QTOQQ6ly5rxRPxJqSzQuoqId4v948eV4e43TFANZ60DGA3JHRI4ypIOqep1QwTaQ2toHGmV7cAI3ZAzfQouZUFbWm552PKFAzudUFxpX7HnOrPqX5jrkZJksf5k7dTn9z\\\\\\\/938N4OTZ1pBSctCBsaZIdn0GDGAmtve7sM1FyV7U11GZ7GaQEw6z\\\\\\\/4vampt5l0R2e1Z3Jjocpli5IRjooPp91QteikKW9gdsC6zThO\\\\\\\/\\\\\\\/AxkpGlLFUdMrGTme9MokWun30Vt7KoDvYLN3EGSQjhdxC78iB5KMHmxbqmcIL1cGXdm1GfsbtbWy1PRgkhA+\\\\\\\/6BweLEnnTwwEEjYeCKAfaBRXlrPiTTPSXKBqgWMXjqBVlLvtlFzhvkdvdg+kv7Ud4f89RLM7VUHklmCB24M015Gm9lh84NEgcYqAlGBYpUM1jZCuku1lXNFUq7j24ZBOW3SMnDuuq4\\\\\\\/UnEwbLDaFqLMTgNHOD9ynzwqGPfeRWiSnFR\\\\\\\/ucI458RUycpdkgAii7slkGS91\\\\\\\/inDe5+ksRqTBYyjApz7saF\\\\\\\/C5xYWH8xOU0QSpD0XDkHsXx5L3jS2o91Xqj91gptsmGatUBNCOcNJeUd4PlDBN3CtpxLxSiUhmzQ\\\\\\\/p3tduAAoIQLVHowGArCBbohepD1PBaBbdSgWcVUDdswbibFTxCxVt7vskG8SYPEUvv5S1fpuPb5\\\\\\\/0rPjB\\\\\\\/7pwFm+yRljD5RorjF0oagjRHlszkcsmVuqJT5F0H1lllS5s7ZOHHaIlgUFJncuKNTgRHfH2Yzr12PeQvY245oOCvsT1rBti3LeCWTBUij4Iae1YSibpyYiowULzY4XJaqyF2QVlN545GYR3lXHYf9T2+lRSigFLE7LAHd1G3jJr0Sgjy708IThnBqu6aTIEavhBZBsRmLo7ITERIQavDU8zDvbL+qtu7EaXEgMuAy6lDsbgq3BZS3DE+fokY\\\\\\\/ZkcOQOA7uucRQ70JUP9IGIDJ2DZhgiL5WcDDigMHP\\\\\\\/gS1D3lg4O6whgVBjdewehfCMH0z9Zku7jOZpYJvqQse0bILEEVJIhfKzsqCfOD+7ie+z1JowXfbcn0wHjFnDbrRXH+yrwvWrXKMm5JkGupiWStIrqAl8NX+ICcQo1ZNXQRwaHklXw+2SnEpG2eTKE5qVyqkUOM5SCu16zrM1dGBGmduDL97DZeqkcFU1hQIaX8ErLSv48girR7W\\\\\\\/B+0Ly3BM5keKeJkEi+h8A\\\\\\\/qda2W\\\\\\\/9H063AfEw9b9ulj76o\\\\\\\/2R27wO3HX4aZFWbnp43jczwAY9JRdjLwWTrhPtzqiMm0Tr1bzQOSrM88M2Cp72KJAWmJHvNwGfU9lSafqL+F8w6fFi5fJHMSErpO8Zy3Zms4yuJ+MslyASHHJSowVxAISHAOYopCDlA+bYrit5L0th2qo+uEqTQtvNhMLWETmH6w1IXtZh8CFX7JVyDH4kQuMXp5LY7ykcUW5s85arHiJjqh88j99muAhzrNLXvVAJICW8AQaXcb7qucbh1yrFL\\\\\\\/P+m2knRQTJkvGEwzbfXXDPKKReMcP3nsGjvFtIOh8NNHkyWjWlKiBcehbHV9uUrR\\\\\\\/B5J3Sp1tmQg9Ni2tnEsqTGqDybwAFdmpXXfeuVE6RBJkvO4Zm8jadfFYJw7F8xl5IbC0fFoYJWQYeL64qMTaJrmb6tWji83GuY2FStAaTveYOVYB03w6\\\\\\\/wXrsSHqCSEsyXgOw3C6datEzHh\\\\\\\/5dbut5J6VHwOB2shronwDcYTBaqI9rXYaHhu\\\\\\\/cJr8B71pnN6HCWOz2\\\\\\\/lpx6m23Caa\\\\\\\/jodvUa\\\\\\\/2Rez\\\\\\\/Ba26faQoNyh+jARBuSeLyH2YnF53LpKDQYxDKERytcg7z8qUF98CKOMOMCnTQ6\\\\\\\/xepFq64GVRpv6UbQ9BzyhQeBMfEIdQ9EEMDr2LNYqzEp00RY7dXDFtFyoYeqZgOOwe74WzhNNgGt1\\\\\\\/dyXF0yu8gFiAOjZeG94vEdQB4rHGh27gNqgik+8O1t\\\\\\\/yV\\\\\\\/9c9dHCyMhRB2Dg+0NGmiVAFUwh6ZCgzCKBZx8XNohgAkhIyGp5zhlImRbSixC5WPLct46FQr3QhtYLDzZa0zs3e2RKSmXL4ZG4wvJgjp6ZJX3EfzGj2VHcivbgws\\\\\\\/oeCPqLwFSh6E+BFapAclEzbIvyc+erwopezTCET9Soya8HgAIR+M4XrGvEMvLqI6AlCe9tnTiYDuhiC524ukYzQ4wjT4d5nBU22abqNKr4RU\\\\\\\/dV87bmN5bkK20FQXCsTLILxGQsUKo7pdfm+BzwzwuNNxDH3R4fD9XZblSNA53i5Tz2eI\\\\\\\/vDn1van3d9oetEVD0koJnYyY8Wlg1Qsb7meIRYWJjlaob6iaWuZ87LIuQF8tEQzxVGtnfVXIZqS0JReW5qz9gm+b62+77yJxPGUzLbF0UKYFwZ4DTGK9Kv0XIxkAC5npsrmSSEyoer100z5IUumQ9NDJRKnCc90e+Skbg0AVDGXE5RBUQ\\\\\\\/LpG75kJxLZDXMiakGLk2Rit7uGPa06Qt8+6EvKDNAfLs8hE8Re6x\\\\\\\/xMfKSKiq+rBQXM8ESHwoSKqu4rNJTzsPJmqeXRV23BwxlefrmHmoxwc+sh8Vuax38G35wRMvISnkNVMirdacOZs+JXktPAy891qhI28SecQWdVqCdEAJfpUORTW7ShnI+2TllmNw9mACNnO643mC1IWWHKdN0Kxw5Hzq61Jj2MaSjkMLtNrsu6It81R7XcNSAbR02IB2cFEmw8W84lbZvyTRKR14yRILv2LFYnp3v2tMQZQK2ZhGTBzpqmrglPmX56+OgKZtNZB+4S5V2JoNViF1b6RQD8g+3hHQd9osWMUGzjSE9JeHwIWjzP3L1xxmDO295RtsjNfi0vbPm43MjqW8UNgBF4rZuOWQDNjP+xQDOfrMDI2xg\\\\\\\/Y+yXywkoVKHcgchvhrxHbY3IzrzHDoBgfLoT\\\\\\\/7KA1pqVU+jXjtvPFzEqrqh7u8sU8J83uU6OLtYibl\\\\\\\/pCe4l0jLHEtfjo4X12HNp6\\\\\\\/zk3tsR00AHjFJkohCDsruhcO2BPW4E7JgYdT2Najw5\\\\\\\/dpjFfg\\\\\\\/RMr6HiDC8yujkxT\\\\\\\/48eVYJEYvAjGhU+OXqiFUGwTv\\\\\\\/Rj8X92GADTMm2IsvEydhvMGcX4JCBtzh3Bsm260oFYZ4dE7oX6nDYEle7ArqrEGNuDKkmLrlTz6M2KP69Pahrj391fIOzGV25cTUe+C9fs5sPWxQQ\\\\\\\/akEKfkOUY239YKifhlVsuKvwRI1WIm95yhU9zeXoXM9Gkjxi+aVUECqHN22l4KL1dGJQ+ZqVPhnD59wdOw1aaDxlhk+urU\\\\\\\/ddOHGuOjRs6ax\\\\\\\/vO0DFkamVxLX8UlMe55MQD0v+Ydo2TZWPOXHGJKeB5Xk2EN0P5IW1hM4Dc1y4OwYqiQU0rI4ZDBuxeRMB3eJUDq60wq6WeUrRYAKCT80VwNb95yaEeAQBpfVJYNB6yAYB0DFj\\\\\\\/Ooot24h3AFM\\\\\\\/32HBUz9OjfVMuSm79aLunoCzBOcRMYTiNBA5PLH463G1XY3eyJCE7mMKRUwWjzLBU5td9eRIHjv28u+HaQJpSWqZAojiTKZOHDKxZRGpyhVpjVn0Dfkc4yuJDT2EmAJ7kR9PV+ViWaL8Dsb3IuSgPeR972LY3bkVgDPdboPlUxUphwUXy9qmnkFol6TLiGzeXhAMj3L3Pgd4ou9mmEOV9xdlQQpqC+XDAo2w1WzCqtAp\\\\\\\/foTZkVSzH2Tw5GqZA6wIfrBEc3X0n4fp8MzixqSk7Gj1y+2EPIkppgKyRgHM3GgZ\\\\\\\/YbAHAacCgMv4u7aTewQmPePEtIJhFIOCY2bDZ8cGGY5VaQso2pMiCe+8\\\\\\\/b0mU+brg4PoGLq8Bg7fgCW6jif+JwW\\\\\\\/iv\\\\\\\/W5reIVtvXGyBiJv+TxOs1EzjIvayeQuFhRnBoCUL0JXDIhIWVz+c4TI65o8gutg5I+yiF0cm4QkL75n5oqFtp78DTmRPexrrOWVoJ+eQawDyWKmEs10GIVMKrcoRC85MkXMVnsvd9bctYwJjcnxH1vHV5E0k2OnQL8PIzg1t9FYdMNgvieOtF7Q6\\\\\\\/ffTMrjfekTN1efb9GZ9t9kY9XR9uJAuoZWg0OUC5mp5iPrZ78z4WcIM2uW2Dex5HCDvRFu\\\\\\\/xhSC2O8VnYkszGR+bnDcqjklgm6Lb1SYarpkB7KtWcTjwsmstahSye2Q2kThREI97TTkekzZziIt7yHZutICcCN96SIo6WuNCnXCWKMNyXApyB+r1SyoSzsXJGf5I5rKT1eOQ738eM8JljRA9VSbAv+YL+vB0\\\\\\\/Ba2W1wDx\\\\\\\/gHKntTG01NCPUf84x04pO1IIlDjamgRty6xB\\\\\\\/jhVHh2G+W1F07gI+ezVqsYShomuK7HSaJr\\\\\\\/nigtueCppz7ZwCUBqcPFNs\\\\\\\/hsUbV92FDDF+BN8dih7G\\\\\\\/qMKuOOrDyxwrfKMoOqN+iuXGs59UIsu4pHWxgOqd2KzeZYqPgt14sMl64XD4KI3tScok2KL6WcS9+oQjqYA1pl3dw554kr\\\\\\\/DyP+7t1yJUYY8kvug3ZWy8RI0r4TOKg9tBXCvt4tXRaM4egZAqJPTWeY3TCU3BSWuZa+i4yCNHtn9Vv0mFbnZyp4UQ8DSRqLTqM\\\\\\\/fmP3I3kIyzLYsSKAwLXcY7J8WGhXE2bctS12mFeizh0ouEGNYbkWeSNWY8yhTM6q7nbzLprqhhBYJdULAv1E7q61t874g3DyVFCSIYbV60qbC0WdHoT1JP8GXMSqBRCJD98LWgP\\\\\\\/1+0OFZqzvzoIeFN0sTcsQS5Gw\\\\\\\/wwadBR5IBaPPcqeLdrw44uzMi335DECF71twBVtRniNnwEAe9QDCFBnIFOJPV1sj\\\\\\\/hC86pwPF6UxLtJowMvLye3TPDS8G3gLCvqETunLXv0Kpcch93Dw2QHk1SceEXWnMzMrGJVkxa03AB1B0cUk7EZ\\\\\\\/lCJF3dmkmHETlheMQSzP5\\\\\\\/c1xQHcBqLbbXE3vGzQK0i3QNhAU\\\\\\\/iPswzsgkl8YVOjk8UA9LhVaVlTiLygm6gmNTe8fb+NfDJuq1sA0BqC2zk3gr0lSj9Vca5V2mEBdmxeUT9L1oIsYwsySwERzdN\\\\\\\/Wxr4eH4X7vSJoNMIxUoB4g0jHrRdxQcK7hNGj0qoSd9+TFXh8cRTN6rQFlCNgILl4O1+94SDW2cOTMsm4ByEN6MRXWBaFo4C4arzyUfdJgduC9lLy3j6RwBaZ7+Jdx+EUZNeVcy0zH2LOhY5ErFOTBgfeaU+ZPTlL3NQ1hAr8Ze4F9y3Prps74Ii+0XP7Wa0iPmVuXFeK6fzOJrdr2SSpm5A6AwAd33Z+YhyJ3\\\\\\\/q\\\\\\\/DUs7vT5LcxQJBm2kBoqTa3bblVNhX3Kw9iidX1vT3PsDI97weOFAfrHYwVUbqx\\\\\\\/C2xCS75ohj9X6qobGwFOZrNxVa7a8\\\\\\\/yKjixfbDHdKSTCZLXh1DAzLCMohWVjDTpeJiZpHnhU2NxahQb\\\\\\\/URez8NNsUz7Q8mUN7AOJJ7Ns2i5nRo8VuFd0GDNfxGRypQn1aczbgOBGgReXF3ah4\\\\\\\/j9on4dMJb8VeE4leKnqnabxtTi9sOq06NbOTwizEu+B+CE\\\\\\\/O3hRCmwu1\\\\\\\/Mg+goc7zo5dMLZamIMZAZlJBE75mfAaiqNBdjTizuHwal+ERYQ0rFj6\\\\\\\/fqMnnlQ3MkZ6Pbr6CnsTq++rQCieVmAGMnLOOFasfY7fz+e5f8zQ\\\\\\\/O+vwHryAtG5pAC3AYchBdtqtA2lK9M5zNSpAADioei9k694k0X4XeFPwHVyTqFxM4JxqIuSXBn87WRXczpGbf5XgW1MAkHix\\\\\\\/lHqpzkYj0OIkXNjHJlLA6TLtXEJOkVUW0F3lFlF5LUzToIaaMroy6e4h9I0T9wkahCI4My5PHr80s9v3MfkVKwg5MQ6+uSbl2mo7tXjgPmgXNOulSm3rfBmdcoKSCMi9NR0mrqdPq09NlFcqnx6nHh70XtqVpfxJmJNBSK8aB7N1VDlz97viVSWtXvAo5b8gb5eVICkgwVHNA2kbJTRV4j1\\\\\\\/zwpOAR1O2gVZLEDYqe8LPt5+LRtgCAVl1N+33MdD9NvN5NoNtCGPJHrscsCcqGXowPYjFZ8SI7btGsJzN4awzggVSosUsNedYuZ\\\\\\\/PTwU0n5GGBiAk+z\\\\\\\/1oUZyMdyId7VDUq77+Vw4++SMkOfh1noIMYxNK29XQBID\\\\\\\/4g3LD2\\\\\\\/1vqXqvGFIKnZbukFQjV5rxsi9rIPZyC6uCKYA09GDTB452J3B3v8q7c7zi6+mdNaPLhFiRIN4Z3NvUB3YE1PJsZkwBr145kQmIMRRWsRt524wfjlZ9rC9EOztCcNwsrAN4hsYIK7uQvXO2TyDHRLfwE4fSzU9+DP37DGCf41lA5+R1oLNBsUy00z1CFhP\\\\\\\/wRF\\\\\\\/s\\\\\\\/nJaR7owFo45hPQ\\\\\\\/pVVFVTTYA9znSIM1miUiYsLKbxc6JGz+JcRwx8AZwn4UsJYF70p3j2Vj\\\\\\\/aYpsJRkLCZK4+zxJJDf6u0Zn9xDnUSgTUyobZRgSoJvXMlyw53EkJD1kPSRh1GFzet0kwJ+GkHAHQERlHWo5GHhLDRU+b3PESCVQfMndTkTRMus6xxHjGY7syA8zN0oQQudUOieCpS\\\\\\\/V76fElPxCzyjAXt34B1ekSu7zc40viTPKylSQFoJoXvQckfhXLcwN4xxA5XnNhO31Sg89wwG4jJVe2vYOAjl4DHbAAhyA\\\\\\\/gg8CZXbnYSYjUEnoesHjt+yjxMVi2MgP3DBbf2QiPjdTU+VpFSAbavHL0hP3KGHQ3g6ovn\\\\\\\/gAV\\\\\\\/8VyJWEH4Csm9RPsS2NOzAHEIGLvYyudIA\\\\\\\/34\\\\\\\/XD81iVt1yq88R+1NnrvSLJiTS+lk8pmOFKRC2FbXE7oDH3Cv3G18vBjqMw56y+htl3GR6hlz0p4JFCFtMRpRsVhPmaTeg+RcPngGbTIwzlJKoq34HXVo\\\\\\\/QUKUWoKqJ4uO0qPTlttkisOtH0RbRwRgDuddjayraPuHLs3CM8ir8drR+5Wpe1HlWb3P9\\\\\\\/H2oh1js0gSrsvCkH5jEMVf\\\\\\\/WrC7stIZfuYk88kCiZdQQgZC1VBBkuH6vKh86+2uaCQVhGB7c\\\\\\\/J1DDLunsbuPXgt5Y3HAgzidfCFmar9PvsCJuHMp8cri21UBEDn+cXclnHQGauFyK+Z7nzAYVtVjlnrMG2a5s+g9tDOiO2Ze6WN3sYonsOo5cg3ZmaESnH0FC3gSiWr2A0KBQpD4H0oy2SpUn8h9OExdQdA2tsVdIgte2GGQM+TH77umyGoW5kHVWdkZ6gJpCBaRIZ4MJzceaX+bfEQ8+feqldW+MTS3pXbVMkkghUDey6KqTqjUxw+STNql+4IW5ZKtcB1Ica8PG7NnCqMRXgOLkBXJDjudqi5s\\\\\\\/Rpt8ekyDTem3VpG4z8e5SBWKR6d0TD\\\\\\\/nBmbXnY17t8gFzpNazBSq2dHdn8+4x++T6wTipOWnkKt8qgdFh0FdWPnkmg7vKlojzI58ebeEFw8gmaSQ3\\\\\\\/wzueb+qbS6B8mPZXOw\\\\\\\/lJFd91fZu\\\\\\\/3qh6Es5U2XspUs5YzlOz9xCsR3kmuAxwbSr+7m8IS+EfQdK\\\\\\\/hbQN7lCC++mUb0wviWhh3Q+MOutJclJSd6t8DyE5bVyxHec\\\\\\\/u7sChizrtRz4BbSNi2Z+hEnOcHmTldFwb4hsvm79yn00AODXtNDb5jDSvDbEoZrTg8IwSl1DPXyn8ecePNhUhFN7ArSQWtoPCwxU\\\\\\\/ifribOO8TJszZMbUWAoU9F\\\\\\\/t2bp\\\\\\\/wfwW8juXaSRDpGtaYjsOrbCRkTwpNKbfMSnAp+MQGo\\\\\\\/P188ZDMX3nrMy1tAPj5pVlOzH3pz6gRjIcWuWgeszMBA3TYTpw9dphrG3+GhZiuz6yyD39G4je79eoMrI9uFdt8z192hK3xlx+Mn6GXWyGy3dPh8I6giRhbxNuKruRFlSkDV97g5dA3CCR72eTdW1iFARzBMlyYvi1Uag96TQYe3EINyW6L9X8Sah0zSAoWZnHytu0VayC6CfU8S+zfp\\\\\\\/FDj0doLoqVfHWpfEyhxDzRH2h0NZ2+\\\\\\\/YxIg2J1NuyRkKxHtLk2vHoJVSUES44jTjb8eRpedhH36xl+1LroD+kXiDEBV8kJtD4yyuQD+rse0sM9pi\\\\\\\/uWLIzX81BP4uzq5MO05vVMzJqZgFCacxhZ3FoiJLlLKeDblVrgmdPs8K+Gf8a\\\\\\\/6DVzWqXHZtAyAPz4aFALyYquto4l3rh+BliDGb0ASImBFZxPA6Z9gFc9QFChG4rk8r3GHtRztDyt7cojJs4LPhHACh0CTxavUy04DbyPEMrSRKJtx5kWIMo9HOu9sGEOZl+5Iohty2KR2eUrRzw9PytBWADFda68iBAQrVpr\\\\\\\/SN0YYeFxO0fq3C6uMWtJyO\\\\\\\/v3RChw9gG7RJjaAsAScmcMovmcv1E2PNhaO6TIF7Vek9i+eY19kZVtjJxQWuEMbLEdO45m5Wt7ThpsfjIqrt9wDjm3ZyiJhNnq1ORqJwjn1bySZCGvwPM63s2ODqTc72TUaro1CoigmIPIM1b64e\\\\\\\/fI8pbdJ88\\\\\\\/e+K9NA8zy6JK+zycTJAbQPgGWQ\\\\\\\/DTp44iO+0fS+MuFJs0JaXRx+W\\\\\\\/P2fXxPbX+gW6L\\\\\\\/x2uegw+5GSkOk1MqF7r1NveJB0gr0zN0kfUCm5Vvt\\\\\\\/Nrhi1XpXPmVA+Ui0O72MHXCADDENdDRQb\\\\\\\/31m\\\\\\\/j4U7jU2TZFlF5V9OVsKEDXhSNgYhFxkrvua6ZjWyftpLkZtP+Zy2BQ4Mk49cucQizbunzpO25g3bxtsu+IOSxqqlkV1uAYZPRVDFUHBJqX6pEoqjeYtscRWsdVMyLyH2RoD75zHxJymQQvukcXdCLMnni10wY\\\\\\\/euyxugxAu6z0yl6y3XEDQmJJAnEkd53BMVULgtJRondCaqmCNE2RZeyGg\\\\\\\/BYvYt97Q6IjnxNCSo3vP4YV0jz+R4z\\\\\\\/YKyEfSJGDMVeWUm5G6ml\\\\\\\/9YTW6tAomBqHAcdp0+NvCZIEmZqRuesucskT9KlZM2Y6C71I3EhbAGjO1cUMJezMY3euhHp4DuExYgViLHgf+DPdMF+SeapOfiLGw4hHwFvxfDfiPfltwxXub+VJu9GVF0c+aNB364vS2eyxGwqegajIaeuNh2GuBRdBnaZ4Soqhm3Q6pRx9z+JMcEawE\\\\\\\/vw5CJuoB8fNrJeM8yMbLWCq9VFUFyORev2R7kGNu3+DFXyQU8WddRPJi0\\\\\\\/ziw\\\\\\\/1nnFzhLu2vYyXsYYvScvaHF\\\\\\\/MzfODUI\\\\\\\/cG8D0fkV4VDvCUnsqwQwKHIXrPvEk41Xii7+MA3LCjawqNCkf6PwHFjYqiZfjaax80Pdzk9Bjgf2tB5ch6jRtclXLxyI8zNVOhy9S+dofj1ITNzWbOHyQm66EHBEOinlFfRIsQ\\\\\\\/sV6dswFtOaHhv48rah8CPPSxl38krTqlEv+JTdlGw\\\\\\\/8sciEShyC5+gvUsT2IqRasl4\\\\\\\/N7wOcxFIWuAPSJEQIQmiMPrVBMjgMgS9z6xdmZ9u11fXWDNzYHkjXLZ5YGVcIl1NjhquEs7P8C4rsecz3BxSU5PCYuhfeEE7CFVOrfe2qKBfb0H3qC\\\u0022,\\\u0022iv\\\u0022:\\\u0022PTNjUAHmb6HNKzzP4ddKJA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e74b9920ffe77e9f2e6c84e09781ca4ded66d49e986db4b8f570deaab5a52673\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SAIWiGuDpNtzuF1xBkjHBkNWeNOMuxhIMaz6xXf1p9x3qWiqUopsAOTBvoCgY7Kl3bDhesgYesgU6tpK6FsK8cQMR2K34DB8VIXcjD1daoRr8gwfa73uc8aY6gcBpQmtgA\\\\\\\/ot3zyrJ8ikrBvypyLBHdxVCsqn7SKETu3r\\\\\\\/+uh2P2w3L3W1exZHVj1HJy4c+j4q7itVcaAzbmTmxJ+m+12obwRS1w4EQZ9PDBXlLxGoVcJFy1qNUeRIkIJha1sxMOlWms38\\\\\\\/8oz\\\\\\\/w7fSjcHqluR0xrGa+jiiKEkOyksh+FQ9ltv7dhPzrGBgUjqO+Wgkp3vgmOnO1szlPLFbtyUX0Eqe\\\\\\\/\\\\\\\/vsDNlHicpuAXXkBuw2hWmNimVOD4uz1w9HKhqWsBEnC4sE3bBKUSMXJ9U35Eq7HAj31CYQIqoqevjfgviriKFB0\\\\\\\/KLNj5N1vb7YN2kGdmqbTUX5uPC7AmPeBzQSHiSn6Zx3F9KABGiYpLngJDZHy8bDyQqoygKmfVQQ+TuzsXeAHTCRYjcQ5xUDZrCtTRcmJ4tyU8JiOVk4MRbRxhUqNB4eJqPcKRgEQIuMhMP48pnRrbaj4X+X8WzTcHE2IIajx7\\\\\\\/V+BoP39U8V9a\\\\\\\/Q6wAezKKXq9qKJmJnHTxLPfcIXjv+T+kWxE5OaoRzJammrBjb4ZSiLYHLjaVcvb3BJnOAc58lupRKVA7wRW5nJWHfMpLsQRVF30CLFPXnaxhIJBgvPyUV1kLQ2CfAQvOl6vmfKGsvZyhmlV4FZnI5yY0ANBW\\\\\\\/tvO61ZWTjKnVtUiizLQAv+C9etRXKLOK77ZfcYu5t\\\\\\\/YwHKxuIJAeErxWBRxnXYTbjZvgqfSMieNNRg5f+qgYCpAEgRypgOg7U2rJ\\\\\\\/Og88To0tJ+KvPeAV5zkCikwaeWjzrIS3pbXWWCjKuq2njqoJMppykFoHMiZ8lvfm7MvM77Z3PisVu8fItkjMuG8k5T88Gb3Pm73pIEdCGKo61D3OQMkpfn6ZNkxbKJRTU74E97djGSpDNbR\\\\\\\/C3hMzlaNFE1\\\\\\\/IN0CyzNH\\\\\\\/ax4fBZgFsy501XnmGn5I3Cg3bDCwiCyksyJSVH6gStz4QNaELe8zL1TduAzkd9kuzToy0I7Uel5L0RXloumxADcmg0nvhyhSbCLMBMcblHCHBET1vhbhil86pQMfB3dNOEz85YE+eEy0TbRaJHYgVbBvFS+gZ9xAXrIUcc4AR+UXjRDZ5DebrEtHI+JeCK\\\\\\\/YDLK\\\\\\\/Lefg8+De9Sj5NAjigr+d4CSvOOGq1UbSUZqrGAHsLlWhVi17kcAO6wgiJV1VbCN+4d1IwxPzL7mQ4aoP59c3vj55HQvHxWvg1tri7xbego6GUgmuob6GxHLwnno3VQEHZXE6ASaHpe9aaOIhbpLm79LXY5W5QsougUKBeZ02MN3Nk2FLZjzjpX072r0fDsDPpENYqPJNUgl27Rm2EEl0cXQATz0Hi\\\\\\\/g18Ew7sDGX4qdfysMFpHxYX3fYRNbViLbCpG7y1anM2540VKIURsns7zmOX4WdRUyFms3VsYVxeFW9gmxrRAWospkN\\\\\\\/lr8N39tFXBQLbdGXM\\\\\\\/wJfv+P4sa7VlrpqJPZuDwDZwi8VuXxCM2F1IxfookIrpDOBKAwD8OOPiHOomw+V76VoDxYNaui9VY8o\\\\\\\/xeAkz+D1G9tGWy+wKvGLXUQAXUAFsMzh1VfcC7m2X3AX4ZqrsIXIedpQMkwPLiO2RqRu6OjFwyD4djQ0mV1xbp5g8Zt38WuzJX+iocOkRiIWRT5s7M9+ALYt0XtNYnMFUXAzt1vFM7kiQPtHIvOF5zH5AoVWJ4e6hhBHSx7JiHt1Phn\\\\\\\/bjbfuXLMtyoe3OaCNYmjaQm8LxaFKmEwmQSUZyc2126sVWwwxfnbSQP2lI1fstis+skDlqnBCrO6+cESO4smcyBjPyr+3AIG1b628fQCbSSokXOs39p\\\\\\\/V0sbftxsF+V6ov3YegQs02w5nwzaO9QDsIPxQ4M56FKtvoudpNMyslYfdqyIkExJkvKPhND30\\\\\\\/tZZJfLzXfJh7IKE=\\\u0022,\\\u0022iv\\\u0022:\\\u00223JAPYIG8w5mJ10YrpjCR6A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002264c4ffe06d6799c935447359125cf52508ba37514888c39f2abdc7a61f50cbb5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459796 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/655" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Tkarx2pegvVVPCGZKuJqJyDCwmNtiQeFA1PyM3VVmVE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vjBkoMfvAoKlV1XL0xHrGr6OZixUfwcjSTCoK_2xPHo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/639" title="Conserved RNA-Binding Proteins Required for Dendrite Morphogenesis in Caenorhabditis elegans Sensory Neurons" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/667" title="Drosophila melanogaster Activating Transcription Factor 4 Regulates Glycolysis During Endoplasmic Reticulum Stress" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459796" id="node459796" data-pisa="ggg;5/4/655" data-pisa-master="ggg;g3.115.017012" data-apath="/ggg/5/4/655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan F. McCormick</span>, <span class="highwire-citation-author" data-delta="1">Sandra K. Truong</span> and <span class="highwire-citation-author" data-delta="2">John E. Mullet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">655-665;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017012 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459796" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan F. McCormick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BF.%2BMcCormick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCormick%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BF.%2BMcCormick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sandra K. Truong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BK.%2BTruong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truong%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BK.%2BTruong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John E. Mullet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BMullet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mullet%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BE.%2BMullet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmullet@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">jmullet@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/655" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459796" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459796/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/655.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459796" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459796/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/655.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459796" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459796/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/655.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459796" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459796/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/655.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459796" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459796/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Recent advances in variant calling made available in the Genome Analysis Toolkit (GATK) enable the use of validated single-nucleotide polymorphisms and indels to improve variant calling. However, large collections of variants for this purpose often are unavailable to research communities. We introduce a workflow to generate reliable collections of single-nucleotide polymorphisms and indels by leveraging available genomic resources to inform variant calling using the GATK. The workflow is demonstrated for the crop plant <em>Sorghum bicolor</em> by (i) generating an initial set of variants using reduced representation sequence data from an experimental cross and association panels, (ii) using the initial variants to inform variant calling from whole-genome sequence data of resequenced individuals, and (iii) using variants identified from whole-genome sequence data for recalibration of the reduced representation sequence data. The reliability of variants called with the workflow is verified by comparison with genetically mappable variants from an independent sorghum experimental cross. Comparison with a recent sorghum resequencing study shows that the workflow identifies an additional 1.62 million high-confidence variants from the same sequence data. Finally, the workflow’s performance is validated using <em>Arabidopsis</em> sequence data, yielding variant call sets with 95% sensitivity and 99% positive predictive value. The Recalibration and Interrelation of genomic sequence data with the GATK (RIG) workflow enables the GATK to accurately identify genetic variation in organisms lacking validated variant resources.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gatk" class="hw-term hw-article-keyword hw-article-keyword-gatk" rel="nofollow">GATK</a></li><li class="kwd"><a href="/keyword/variant-calling" class="hw-term hw-article-keyword hw-article-keyword-variant-calling" rel="nofollow">variant calling</a></li><li class="kwd"><a href="/keyword/genomic-sequence-data" class="hw-term hw-article-keyword hw-article-keyword-genomic-sequence-data" rel="nofollow">genomic sequence data</a></li></ul><p id="p-3">The decreasing cost of high-throughput sequencing has led to a proliferation of template preparation methods and sequence data (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sims <em>et al.</em> 2014</a>). The abundance of sequence data has motivated an interest in leveraging available data to identify genetic variation, and software development has kept pace with this demand as exemplified by the Broad Institute’s open-source Genome Analysis Toolkit (GATK). The GATK can integrate evidence for variants from multiple samples with joint genotyping, and it enables the use of validated single-nucleotide polymorphisms (SNPs) and indels to improve the accuracy of variant calling. Additionally, the GATK’s methods are implemented in a manner amenable to reads originating from a variety of template preparation methods and sequencing platforms (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Depristo <em>et al.</em> 2011</a>). However, many research communities lack the large, validated collections of SNPs and indels necessary for the GATK’s Best Practices procedures because of the investment necessary to produce and curate such collections (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Van Der Auwera <em>et al.</em> 2013</a>). As an alternative to large-scale variant validation studies, we developed the Recalibration and Interrelation of genomic sequence data with the GATK (RIG) workflow to integrate information from multiple genomic sources and identify reliable sets of variants.</p><p id="p-4">The GATK has gained extensive adoption in the human genomics research community due in part to the methods it uses to account for known error sources during variant calling; accounting for these error sources enables the GATK to consistently outperform other modern variant callers in benchmarking studies (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Depristo <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nekrutenko and Taylor 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pirooznia <em>et al.</em> 2014</a>). Multiple sources of error exist, including incomplete or incorrect reference assemblies, erroneous realignment of reads to the reference genome (particularly in low complexity regions and around indels), inaccurate base quality scores, and suboptimal variant filtration parameters (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Depristo <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li 2014</a>). Features of the GATK address a number of these error sources, and we briefly describe three of the features most relevant to the design of the RIG workflow. The first feature is Base Quality Score Recalibration (BQSR), where the base quality scores assigned by the sequencer are corrected with scores empirically determined from the read group data using validated variants; these recalibrated scores more accurately reflect the true reliability of the base calls, thus correcting biases introduced by sequencing platforms (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li <em>et al.</em> 2004</a>). The second feature is the GATK’s joint genotyping methodology that can integrate the evidence for a variant from many samples on reasonable time scales; this allows data from thousands of samples to be considered when evaluating the existence of a variant. The third feature is Variant Quality Score Recalibration (VQSR), where raw variant calls are assigned probabilities of being true variants based on the behavior of training variants in the raw variant calls using machine learning techniques (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Depristo <em>et al.</em> 2011</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Van Der Auwera <em>et al.</em> 2013</a>). These probabilities allow users to decide which variants to use in downstream analyses based on desired levels of specificity and sensitivity, where high specificity indicates a low false-positive rate, and high sensitivity indicates a low false-negative rate. Two of these three features, BQSR and VQSR, require a collection of reliable variants to function effectively, and their benefits are inaccessible without such a resource.</p><p id="p-5">Although many research communities lack large, validated resources of known SNPs and indels, some communities, namely agricultural research communities, often have access to a variety of genomic data sources that can be used to identify reliable genetic variants for use with the RIG workflow. Two characteristics influence the optimal use of these data sources with the RIG workflow: (i) the method used to produce the source’s raw data from which variants are called, and (ii) the experimental design behind the source. First, many methods are available to produce the raw data from which variants are called, including reduced representation sequencing, whole-genome sequencing (WGS), SNP chips, Sanger sequencing, and RNA sequencing. Variants identified from two different methods can be considered more reliable than those identified in only one, as they are less likely to be artifacts introduced by a specific method. The RIG workflow can take advantage of multiple data sources by using variants found from one data source to inform the analysis of a second, read-based data source; by providing variants obtained from orthogonal methods, the reliability of variant resources used in BQSR and VQSR can be improved. Second, the experimental design behind the source also influences the reliability of the variants obtained from the source. Two experimental design elements influencing the reliability of a genomic variant are (i) if the variant segregates according to Mendelian expectations, and (ii) how often the variant is observed in independent samples. Genotyping large experimental crosses provides variants where Mendelian violations can be identified and the variants are observed at high frequencies in independent samples; as such, experimental crosses represent one of the most reliable sources of genetic variants. Association panels or population samples can also provide a reliable source of variants given a sufficiently large sample size and minor allele frequency. When Mendelian violations cannot be identified or when sample sizes are small (as is common with resequencing designs), variants are considered less reliable. For our use case with sorghum, we (i) generated an initial set of variants using reduced representation sequence data from an experimental cross and association panels, (ii) used the initial variants to inform variant calling from WGS data of resequenced individuals, and (iii) used variants identified from WGS data for recalibration of the reduced representation sequence data. By considering the method used to produce the raw data from which variants are called and the experimental design behind the data source, available genomic sequence data can be optimally leveraged to improve variant calling and subsequent analyses.</p><p id="p-6">Here we present the RIG workflow to formalize the process of incorporating available genomic sequence resources when calling SNPs and indels with the GATK. The RIG workflow is designed to leverage available genomic data in a manner that maximizes the information available to joint genotyping and to produce collections of reliable variants sufficiently large to perform BQSR and VQSR; this provides the benefits of the GATK’s methods even in the absence of a large collection of validated variants, and it is readily applicable to organisms with a reference genome sequence and moderate sequence data resources. As an example, we describe the RIG workflow using <em>Sorghum bicolor</em> sequence data and show that it readily interrelates reduced representation and WGS data to generate variant calls. We evaluate the performance of the RIG workflow for sorghum sequence data using a collection of genetically validated variants, and we compare the output of the RIG workflow with variant calls from a recent sorghum study. Finally, we validate the workflow with <em>Arabidopsis</em> sequence data and show that high sensitivity and specificity is readily achieved.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sorghum analyses</h3><p id="p-7">The RIG workflow described in the <em>Results</em> section was designed as a generalization of our use cases in leveraging existing <em>Sorghum bicolor</em> genomic resources to take advantage of the GATK’s strengths. Here we describe the process of transitioning from exclusive use of the naive pipeline to use of the initial informed and informed pipelines as an example of executing the RIG workflow and constructing variant resources (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/655/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phase I of the RIG workflow. Phase I of the RIG workflow defines the five entities necessary for the execution of Phase II. Once the first three entities, the analysis target, database of likelihoods, and variant resource(s) are defined, the user considers a hypothetical case based on those first three entities to estimate the contents of the remaining two: the hypothetical database of likelihoods and the shared variants. If a user is unable to make a prediction regarding the latter two entities, the entities can either be treated as empty sets, or the user can use the GATK to carry out the necessary procedures to generate an estimate. Once all five entities are defined, the user can proceed to Phase II. RIG, Recalibration and Interrelation of genomic sequence data with the GATK; GATK, Genome Analysis Toolkit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582328943" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phase I of the RIG workflow. Phase I of the RIG workflow defines the five entities necessary for the execution of Phase II. Once the first three entities, the analysis target, database of likelihoods, and variant resource(s) are defined, the user considers a hypothetical case based on those first three entities to estimate the contents of the remaining two: the hypothetical database of likelihoods and the shared variants. If a user is unable to make a prediction regarding the latter two entities, the entities can either be treated as empty sets, or the user can use the GATK to carry out the necessary procedures to generate an estimate. Once all five entities are defined, the user can proceed to Phase II. RIG, Recalibration and Interrelation of genomic sequence data with the GATK; GATK, Genome Analysis Toolkit.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/655/F1.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/655/F1.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/655/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/655/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465864" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Phase I of the RIG workflow. Phase I of the RIG workflow defines the five entities necessary for the execution of Phase II. Once the first three entities, the analysis target, database of likelihoods, and variant resource(s) are defined, the user considers a hypothetical case based on those first three entities to estimate the contents of the remaining two: the hypothetical database of likelihoods and the shared variants. If a user is unable to make a prediction regarding the latter two entities, the entities can either be treated as empty sets, or the user can use the GATK to carry out the necessary procedures to generate an estimate. Once all five entities are defined, the user can proceed to Phase II. RIG, Recalibration and Interrelation of genomic sequence data with the GATK; GATK, Genome Analysis Toolkit.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/655/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phase II of the RIG workflow. Phase II of the RIG workflow determines whether VQSR, BQSR, or both are appropriate, given the entities defined in Phase I. The workflow always proceeds through an analysis pipeline, characterized as the naive, the initial informed, and the informed pipelines shown in Figure 3. The end result of the workflow is the production of a variant resource that can be used in future analyses. RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; BQSR, Base Quality Score Recalibration. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582328943" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phase II of the RIG workflow. Phase II of the RIG workflow determines whether VQSR, BQSR, or both are appropriate, given the entities defined in Phase I. The workflow always proceeds through an analysis pipeline, characterized as the naive, the initial informed, and the informed pipelines shown in Figure 3. The end result of the workflow is the production of a variant resource that can be used in future analyses. RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; BQSR, Base Quality Score Recalibration. &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/655/F2.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/655/F2.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/655/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/655/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-9" class="first-child">Phase II of the RIG workflow. Phase II of the RIG workflow determines whether VQSR, BQSR, or both are appropriate, given the entities defined in Phase I. The workflow always proceeds through an analysis pipeline, characterized as the naive, the initial informed, and the informed pipelines shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. The end result of the workflow is the production of a variant resource that can be used in future analyses. RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; BQSR, Base Quality Score Recalibration. </p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/655/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RIG pipelines. These are analysis pipelines that are traversed as part of Phase II of the RIG workflow. They correspond to cases where neither BQSR nor VQSR are appropriate (naive pipeline), where only VQSR is appropriate (initial informed pipeline), or where both BQSR and VQSR are appropriate (informed pipeline). When traversed, the informed pipeline emulates the GATK&#x2019;s Best Practices (Van Der Auwera et al. 2013). RIG, Recalibration and Interrelation of genomic sequence data with the GATK; BQSR, Base Quality Score Recalibration; VQSR, Variant Quality Score Recalibration; GATK, Genome Analysis Toolkit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582328943" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RIG pipelines. These are analysis pipelines that are traversed as part of Phase II of the RIG workflow. They correspond to cases where neither BQSR nor VQSR are appropriate (naive pipeline), where only VQSR is appropriate (initial informed pipeline), or where both BQSR and VQSR are appropriate (informed pipeline). When traversed, the informed pipeline emulates the GATK&#x2019;s Best Practices (Van Der Auwera &lt;em&gt;et al.&lt;/em&gt; 2013). RIG, Recalibration and Interrelation of genomic sequence data with the GATK; BQSR, Base Quality Score Recalibration; VQSR, Variant Quality Score Recalibration; GATK, Genome Analysis Toolkit.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/655/F3.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/655/F3.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/655/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/655/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465883" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-10" class="first-child">RIG pipelines. These are analysis pipelines that are traversed as part of Phase II of the RIG workflow. They correspond to cases where neither BQSR nor VQSR are appropriate (naive pipeline), where only VQSR is appropriate (initial informed pipeline), or where both BQSR and VQSR are appropriate (informed pipeline). When traversed, the informed pipeline emulates the GATK’s Best Practices (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Van Der Auwera <em>et al.</em> 2013</a>). RIG, Recalibration and Interrelation of genomic sequence data with the GATK; BQSR, Base Quality Score Recalibration; VQSR, Variant Quality Score Recalibration; GATK, Genome Analysis Toolkit.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/655/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Construction of variant resources. After VQSR, multiple tranches are evaluated to choose specific and sensitive sets of variants for use in downstream analyses and to designate as variant resources. Tranches correspond to VQSLOD cutoffs above which a specified percentage of the variants designated as truth during VQSR are retained in the tranche. For example, a 95% tranche indicates the VQSLOD cutoff at which 95% of the variants designated as truth during VQSR would be retained. Accordingly, lower tranche percentages have greater specificity, lesser sensitivity, and contain fewer variants, and lower percentage tranches are subsets of greater percentage tranches. Here we show a 90% tranche being chosen as the specific variant resource and the 95% tranche being chose as the sensitive variant resource; both are subsequently added to the collection of variant resources. Note that the specific variant resource generated here is a subset of the sensitive variant resource. VQSR, Variant Quality Score Recalibration; VQSLOD, logarithm of odds ratio that a variant is real vs. not under the trained Gaussian mixture model; " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582328943" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Construction of variant resources. After VQSR, multiple tranches are evaluated to choose specific and sensitive sets of variants for use in downstream analyses and to designate as variant resources. Tranches correspond to VQSLOD cutoffs above which a specified percentage of the variants designated as truth during VQSR are retained in the tranche. For example, a 95% tranche indicates the VQSLOD cutoff at which 95% of the variants designated as truth during VQSR would be retained. Accordingly, lower tranche percentages have greater specificity, lesser sensitivity, and contain fewer variants, and lower percentage tranches are subsets of greater percentage tranches. Here we show a 90% tranche being chosen as the specific variant resource and the 95% tranche being chose as the sensitive variant resource; both are subsequently added to the collection of variant resources. Note that the specific variant resource generated here is a subset of the sensitive variant resource. VQSR, Variant Quality Score Recalibration; VQSLOD, logarithm of odds ratio that a variant is real &lt;em&gt;vs.&lt;/em&gt; not under the trained Gaussian mixture model; &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/655/F4.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/4/655/F4.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/655/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/655/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465774" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-11" class="first-child">Construction of variant resources. After VQSR, multiple tranches are evaluated to choose specific and sensitive sets of variants for use in downstream analyses and to designate as variant resources. Tranches correspond to VQSLOD cutoffs above which a specified percentage of the variants designated as truth during VQSR are retained in the tranche. For example, a 95% tranche indicates the VQSLOD cutoff at which 95% of the variants designated as truth during VQSR would be retained. Accordingly, lower tranche percentages have greater specificity, lesser sensitivity, and contain fewer variants, and lower percentage tranches are subsets of greater percentage tranches. Here we show a 90% tranche being chosen as the specific variant resource and the 95% tranche being chose as the sensitive variant resource; both are subsequently added to the collection of variant resources. Note that the specific variant resource generated here is a subset of the sensitive variant resource. VQSR, Variant Quality Score Recalibration; VQSLOD, logarithm of odds ratio that a variant is real <em>vs.</em> not under the trained Gaussian mixture model; </p><div class="sb-div caption-clear"></div></div></div><p id="p-12">At the time of transitioning from the naive pipeline to the initial informed and informed pipelines with <em>Sorghum bicolor</em> sequence data, we had access to reduced representation sequence data generated internally by Digital Genotyping using the restriction enzyme NgoMIV and a collection of WGS data generated from multiple groups (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zheng <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Evans <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Mace <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Morishige <em>et al.</em> 2013</a>). Using reduced representation sequence data for a 423 member recombinant inbred line population, we used the naive pipeline to produce variant calls (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Truong <em>et al.</em> 2014</a>). Preprocessing of the reads prior to variant calling, including read-mapping to version 1 of the <em>Sorghum bicolor</em> reference assembly, was performed using BWA and Picard (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Paterson <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Li and Durbin 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Picard Team 2014</a>). Variants were genetically mapped with R/qtl, and variants segregating as expected in these calls were used to create a Family Reference Variant Resource that contained 6849 SNPs and 2164 indels (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>). The Family Reference Variant Resource was considered a highly specific variant resource. Of note, the genetic positions of markers found in this manner are also being used to anchor unplaced super contigs in the <em>Sorghum bicolor</em> reference genome assembly (J. Schmutz, personal communication). Similarly, reduced representation sequence data for 733 sorghum germplasm samples processed with the naive pipeline were used to produce a Population Reference Variant Resource containing 62,022 SNPs and 20,801 indels. This variant resource was considered specific because we enforced a genotyping rate of 60% and a minor allele frequency of 0.05. The hard filtering parameters that we use in the naive pipeline for reduced representation sequence data can be found within the implementation on GitHub at <a href="https://github.com/MulletLab/RIG">https://github.com/MulletLab/RIG</a>.</p><p id="p-13">We sought to use these variants found in reduced representation sequence data to improve the analysis of WGS data of the 49 publicly available WGS data sources (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Zheng <em>et al.</em> 2011</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Evans <em>et al.</em> 2013</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Mace <em>et al.</em> 2013</a>). To do this, we chose 10 individuals from the 49 that represented diverse sorghum germplasm accessions. We attempted to maximize diversity so that the sensitive variant resource constructed after the initial informed and informed pipelines had been executed would include many of the variants present in the next group of individuals for BQSR; this enabled use of the informed pipeline in the following iterations as the remaining 39 samples were processed (individuals were processed 10 at a time due to hard disk space limitations).</p><p id="p-14">With the Family and Population Reference Variant Resources and the 10 WGS samples as analysis targets, we met the requirements for VQSR but not BQSR (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). As such, we followed the initial informed pipeline. For VQSR, the Family and Population Reference Variant Resources were both designated as truth, training, and known variants, and priors set to 15.0 and 7.0, respectively. Although these settings worked for our use case, they may not always be applicable; however, we typically follow these general rules: only highly specific variant resources should be designated as truth; variant resources designated as training do not need to be highly specific, but their priors should be set accordingly; all resources designated as truth should also be designated as training; and resources designated as truth and training can also be designated as known. Other details along with the annotations used for training the SNP and indel Gaussian mixture models can be found with the implementation on GitHub at <a href="https://github.com/MulletLab/RIG">https://github.com/MulletLab/RIG</a>.</p><p id="p-15">Generating variant resources following VQSR is a highly user-driven process that depends largely on the user’s confidence in the variant resources designated as truth for VQSR, and it requires examining multiple tranches resulting from VQSR (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Tranches represent cutoffs based on variant resources designated as truth during VQSR, and they are generated by considering the VQSLOD scores (logarithm of odds ratio that a variant is real <em>vs.</em> not under the trained Gaussian mixture model) of truth variants that are present in the recalibrated raw variants. For example, if 90% of the truth variants found in the raw variants had a VQSLOD score over 1.5, then the 90% tranche would contain all variants in the raw variants that had a VQSLOD score over 1.5. We typically pick two tranches after VQSR, a specific tranche and a sensitive tranche, by examining the behavior of VQSLOD scores of multiple tranches (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Specific tranches typically come from tranches where the VQSLOD score changes by small amounts even as the tranche percentage is decreased, and sensitive tranches are typically a non-negative VQSLOD score tranche that is more inclusive than the specific tranche.</p><p id="p-16">Having generated a temporary sensitive variant resource from the initial 10 WGS samples using the initial informed pipeline, we proceeded down the informed pipeline with those 10 samples to generate a sensitive Whole-Genome Sequence Variant Resource. We then iteratively processed the remaining 39 samples in groups of 10 (9 on the final iteration) using the informed pipeline and updating the Whole-Genome Sequence Variant Resource each iteration; we continued to use only the Family and Population Reference Variant Resources for VQSR (to enforce that variants designated as truth for VQSR had been identified using a different sequencing template preparation method), and we used the newest sensitive Whole-Genome Sequence Variant Resource for BQSR. Upon completion of all 49 genomes, we used the newest Whole-Genome Sequence Variant Resources for BQSR and VQSR of the association panel data (sensitive for BQSR and specific for VQSR) to generate the Sensitive Population Reference Variant Resource (97.5% tranche) that was used for the genome-wide association study.</p><p id="p-17">The IF set used to examine the recalibration of WGS variants was constructed from a second biparental recombinant inbred line population (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Xu <em>et al.</em> 2000</a>). Variants from this population were generated in the same fashion as the Family Reference Variant Resource (<em>i.e.</em>, using NgoMIV Digital Genotyping, the naive pipeline, and checking for Mendelian segregation in R/qtl). The Raw, Sensitive, and Specific sets used in the comparison with the IF set were derived from the 100% tranche, the 95% tranche, and 75% tranche of the recalibrated WGS variants (<a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/017012SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS2.pdf">Table S2</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/FigureS1.pdf">Figure S1</a>). The Raw, Sensitive, and Specific sets used for comparison with the Gramene42-Mace2013 set originate in the same manner, but excluded indels, SNPs on super contigs, and variants not found in 1 of the 47 samples to be comparable with the Gramene42-Mace2013 set. Variants and genotypes for 171 individuals from the Sensitive Population Reference Variant Resource were used with downstream analysis tools to perform the association mapping described and depicted in <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS3.pdf">Table S3</a>.</p></div><div id="sec-3" class="subsection"><h3><em>Arabidopsis</em> analyses</h3><p id="p-18">Publicly available WGS for five accessions (ICE50 ICE134, ICE150, ICE213, and Leo-1) from <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cao <em>et al.</em> (2011)</a> were processed using the naive pipeline and stringently hard filtered (parameters available on GitHub). Publicly available Sanger sequence for 20 accessions (Ag-0, Bor-1, Br-0, Ei-2, Got-7, Gu-0, Hr-5, Kin-0, Kondara, Ms-0, Mz-0, NFA-8, Nok-3, PNA-17, Rmx-A02, Sorbo, Sq-8, Uod-1, Wa-1, Yo-0) were obtained from the Supporting Information of <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a>. Publicly available WGS for the same 20 accessions from <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schmitz <em>et al.</em> (2013)</a> were processed through the initial informed pipeline, and VQSR was performed using the stringently filtered variants from <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cao <em>et al.</em> (2011)</a> as a training set (prior of 7.0) and as a truth set. The resulting 95% tranche was used for BQSR as the WGS data for the 20 accessions were then processed through the informed pipeline. The <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cao <em>et al.</em> (2011)</a> variants were again used for VQSR. All alignments and variant calling were done against the version 10 <em>Arabidopsis</em> reference genome (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2"><em>Arabidopsis</em> Genome Initiative <em>et al.</em> 2000</a>).</p><p id="p-19">To estimate error rates of the RIG workflow for WGS data, the resulting variant calls for the 20 accessions were compared to Sanger data from <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a> and variants from the Gramene database build 43, accessed January 2015 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Monaco <em>et al.</em> 2014</a>). This requires the assumption that the Sanger data were 100% specific (<em>i.e.</em>, no false positives), and that the combination of the Sanger data and the Gramene build 43 variants were 100% sensitive (<em>i.e.</em>, no false negatives). Although the WGS data strongly suggest that these assumptions are false, this still provides a useful baseline for comparison; however, we expect that the true sensitivity and specificity achieved in this comparison are greater than the values obtained since false positives in the Sanger data translate to decreased sensitivity and false negatives in the Sanger data and Gramene build 43 translate to decreased positive predictive value. Genomic intervals used to evaluate performance were defined as a subset of the 861 intervals from <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a>. Because many of the Sanger reads had an abundance of Ns at the beginning and end of the read, 50 bp from the ends of each interval were removed. Excluding intervals that did not have &gt;90% of the bases covered at greater than 15 depth in all 20 WGS samples and &gt;90% coverage in all 20 Sanger samples yielded 419 intervals that covered 200,887 bp of the genome.</p><p id="p-20">Two of the accessions (Got-0 and Ms-0) were dropped from the comparison due to extensive disagreement between the Sanger variants and the WGS variants, potentially due to not truly being the same accession. We also found a number of sites that were heterozygous in the WGS accessions that that had been manually curated by <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a> to Ns in the Sanger data. Because this generates what appears to be a false positive in the WGS data, we used the Sanger data to identify false negatives, and variants from both the Sanger data and <em>Arabidopsis</em> variants contained in Gramene build 43 to identify false positives. Variants from the <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a> Sanger data contained in the designated genomic intervals but not contained in a tranche of the WGS data were considered false negatives for the purpose of calculating sensitivity. Variants contained in the WGS data but not in either the <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a> or the variants present in Gramene build 43 were considered false positives for the purpose of calculating positive predictive value. Variant site counts used for calculating sensitivity and positive predictive value are available in <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS4.pdf">Table S4</a>.</p><p id="p-21">For the comparison, we report positive predictive value instead of specificity as a metric for false positives since the number of true negatives is far larger than the number of false positives, always leading to specificity values greater than 99.9%. As such, the performance of a tranche with a sensitivity of 95% and a positive predictive value of 99% is interpreted as a tranche where 95% of the true variants that existed were called and that 99% of the variants called are true variants.</p></div><div id="sec-4" class="subsection"><h3>Code and hardware</h3><p id="p-22">Our implementation of the workflow and pipelines are available on GitHub at <a href="https://github.com/MulletLab/RIG">https://github.com/MulletLab/RIG</a> as a series of Bash scripts to serve as an example, to provide the annotations we used for hard filtering and VQSR, and to list all of the additional software version numbers used. GATK’s Scala-based job submission controller, Queue, is suggested for implementing pipelines for the GATK for distributed computing resources; our implementation is in Bash because we experienced slowdowns in job submissions over time when using Queue (v3.1-1) on the Whole System Genomics Initiative cluster present at Texas A&amp;M University.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>RIG: Recalibration and Interrelation of genomic sequence data with the GATK</h3><p id="p-23">The RIG workflow is a generalization of procedures to leverage existing genomic data when using the GATK v3.0+. Specifically, the workflow determines whether VQSR and/or BQSR are appropriate to perform, and the workflow iteratively constructs reliable variant resources for future use with the GATK. The procedures of the RIG workflow are divided into two phases: Phase I, where the entities necessary for workflow execution are defined (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), and Phase II, where those entities are used to execute the workflow (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="sec-7" class="subsection"><h4>RIG Phase I: define RIG entities:</h4><p id="p-24">Phase I of the RIG workflow defines the five entities necessary for execution of Phase II (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The first entity is an <strong><em>analysis target</em></strong>. The analysis target contains the sequence reads from which the user intends to call variants. Reads of the analysis target should be preprocessed by read-mapping, duplicate marking (if applicable), and indel realignment; this entity is depicted as a stack of BAM-format files in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>. The second entity is a <strong><em>database of likelihoods</em></strong>. The database of likelihoods contains the likelihood that a variant exists at a genomic position for all evaluated positions; this database consists of one or more gVCF-format flat files obtained from past GATK analyses of analysis targets produced by similar template preparation methods (<em>i.e.</em>, a database of likelihoods for WGS samples should not be used with an analysis target of reduced representation samples). This entity is depicted as a stack of circles in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>, and it can be defined as empty. The third entity is a set of <strong><em>variant resources</em></strong>. These are one or more files of VCF-format variant calls, and these calls should be conceptually (and physically, if necessary) partitioned into one or both of two categories: specific variant resources with low false positive rates, and sensitive variant resources with low false negative rates; a specific resource is necessary for VQSR and a sensitive resource is necessary for BQSR. As with the database of likelihoods, the variant resources can be empty and likely will be when first executing the workflow. The fourth and fifth entities can either be (i) constructed hypothetically based on a user’s expectations of the first three entities, or (ii) they can be empirically determined by performing the necessary analyses with the first three entities using the GATK. The fourth entity is a <strong><em>hypothetical database of likelihoods</em></strong> that is generated after adding the genotype likelihoods called from the analysis target to the existing database of likelihoods. The fifth entity is a set of <strong><em>shared variants</em></strong>. Shared variants are variants contained in both the hypothetical database of likelihoods and in the chosen variant resources; shared variants can be specific, sensitive, or both (or empty) depending on the classification of the variant resource they were found in. Once all five entities are defined, the analysis target, the database of likelihoods, the variant resources, the hypothetical database of likelihoods, and the shared variants, a user can proceed to Phase II.</p></div><div id="sec-8" class="subsection"><h4>RIG Phase II: execute analysis:</h4><p id="p-25">The initial question of Phase II of the RIG workflow determines whether VQSR is appropriate based on the number of variants contained within the specific shared variants (RIG recommends at least 10,000 SNPs and 10,000 indels; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). A specific variant resource is required since false positives negatively impact the training of the Gaussian mixture models during VQSR, whereas false negatives have lesser effect. If the specific shared variants do not satisfy these criteria, then the RIG workflow enters the naive pipeline in which called variants are hard filtered using user-designated filtration criteria such as depth (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Variants passing user-designated filtration criteria can then be added to the collection of variant resources. Once the naive pipeline has been used to analyze enough analysis targets, the collection of variant resources may be sufficiently large to answer yes to the initial question.</p><p id="p-26">If the specific shared variants contain at least 10,000 SNPs and 10,000 indels, the next question addresses if the samples and variants in the database of likelihoods (if it is not empty) had previously undergone BQSR. If not, and if the reads corresponding to the samples used to generate the database of likelihoods are available, then the analysis target is updated with those reads, the database of likelihoods is set to empty, and the user returns to RIG Phase I (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>) with the new analysis target and the empty database of likelihoods.</p><p id="p-27">If the reads used in the construction of the database of likelihoods had previously undergone BQSR, or if the database of likelihoods is empty, then the final assessment determines whether the analysis target and the sensitive shared variants are appropriate for BQSR. A sensitive variant resource is necessary since false negatives cause BQSR to treat true variants in the analysis target as errors and will skew quality scores down, whereas false positives have a lesser chance to skew quality scores up. If BQSR is appropriate, the user follows the informed pipeline which emulates the GATK’s Best Practices (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Van Der Auwera <em>et al.</em> 2013</a>). If BQSR is not appropriate, the user first uses the initial informed pipeline in which VQSR is performed on the raw variants to generate a <em>temporary sensitive variant resource</em> which is used during the execution of the informed pipeline that immediately follows the initial informed pipeline (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-28">Construction of variant resources and adding them to the collection of variant resources is the end step of any path through the RIG workflow (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Deciding the criteria for generating the variant resources at the end is a highly user-driven process that should consider the specific properties of the analysis target. For example, we generate highly specific variant resources from experimental crosses based on markers that segregate as expected (see the section <em>Materials and Methods</em>). Additionally, the variant annotations used for hard filtering and VQSR should differ based on how reads should behave in the analysis target; that is, reduced representation data should not use the same annotations as WGS data for hard filtering and VQSR because reads are not distributed around variants in a similar manner. To provide an example, we discuss the methods we use to select VQSR tranches and construct variant resources in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and in the section <em>Materials and Methods</em>. We also have made our code available on GitHub at <a href="https://github.com/MulletLab/RIG">https://github.com/MulletLab/RIG</a> as an example and to provide the parameters and variant annotations we use.</p></div></div><div id="sec-9" class="subsection"><h3>Interrelation of genomic data enables a specificity and sensitivity framework for variant calls</h3><p id="p-29">In accordance with the RIG workflow, we used reduced representation data of an experimental cross and association panels to enable both BQSR and VQSR of WGS data of 49 resequenced individuals for the crop plant <em>Sorghum bicolor</em>. By interrelating data sources produced by different template generation methods with the RIG workflow, we enforced that the variants used to train the VQSR Gaussian mixture models that determine a variant’s VQSLOD score (logarithm of odds ratio that a variant is real <em>vs.</em> not real under the trained Gaussian mixture model) were found orthogonally, providing additional confidence that the variants used for training were real variants. Additionally, the differences in reliability of the training variants due to the different experimental designs were also considered for training of the VQSR models; variants from the experimental cross were assigned a higher prior likelihood of being correct than those from the association panels. By following the RIG workflow, each SNP and indel in the raw WGS variants was assigned a VQSLOD score that reflects its reliability. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> depicts the process of interrelating data for VQSR and the resulting VQSLOD scores of variant calls. While interrelating data from different template generation methods may be optimal, we also obtained good performance by following similar processing logic using only <em>Arabidopsis</em> WGS data. In this way, the RIG workflow enables one of the greatest strengths of the GATK: the ability to put variant calls in a probabilistic framework that allows users to define where on the sensitivity and specificity spectrum the variants should sit for their target downstream application.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/655/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interrelation of different genomic sequence data sources using the RIG workflow. (A) Schematic of how variants from reduced representation sequence (RR) data present in whole-genome sequence (WGS) data can be used to VQSR the WGS raw variants and assign VQSLOD scores to those variants. (B&#x2212;D) Visualization of the genomic region of Sb07g003860, a gene involved in sorghum midrib coloration (Bout and Vermerris 2003). (B) the Sbi1.4 gene annotation; (C) shows the assigned VQSLOD scores for variants called in the region from WGS data; (D) shows the depth of coverage and mapped sequence reads for reduced representation and WGS data, respectively, for one sorghum line (BTx642). The RIG workflow enables variants called in the reduced representation sequence data to be used to inform and recalibrate the WGS analyses, and vice versa. This puts all of the variant calls into the GATK&#x2019;s probabilistic framework whereby variants can be filtered based on their reliability. Users interested in more sensitive or specific call sets can choose more inclusive or exclusive tranches, respectively, by changing the cutoff indicated by the blue dotted line in Panel C. The common and standardized file formats emitted by the GATK enable downstream interoperability between analysis and visualization tools, such as the Integrative Genomics Viewer that produced (B) and (D) (Thorvaldsd&#xF3;ttir et al. 2012). RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; VQSLOD, logarithm of odds ratio that a variant is real vs. not under the trained Gaussian mixture model; GATK, Genome Analysis Toolkit. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582328943" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interrelation of different genomic sequence data sources using the RIG workflow. (A) Schematic of how variants from reduced representation sequence (RR) data present in whole-genome sequence (WGS) data can be used to VQSR the WGS raw variants and assign VQSLOD scores to those variants. (B&#x2212;D) Visualization of the genomic region of Sb07g003860, a gene involved in sorghum midrib coloration (Bout and Vermerris 2003). (B) the Sbi1.4 gene annotation; (C) shows the assigned VQSLOD scores for variants called in the region from WGS data; (D) shows the depth of coverage and mapped sequence reads for reduced representation and WGS data, respectively, for one sorghum line (BTx642). The RIG workflow enables variants called in the reduced representation sequence data to be used to inform and recalibrate the WGS analyses, and &lt;em&gt;vice versa&lt;/em&gt;. This puts all of the variant calls into the GATK&#x2019;s probabilistic framework whereby variants can be filtered based on their reliability. Users interested in more sensitive or specific call sets can choose more inclusive or exclusive tranches, respectively, by changing the cutoff indicated by the blue dotted line in Panel C. The common and standardized file formats emitted by the GATK enable downstream interoperability between analysis and visualization tools, such as the Integrative Genomics Viewer that produced (B) and (D) (Thorvaldsd&#xF3;ttir &lt;em&gt;et al.&lt;/em&gt; 2012). RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; VQSLOD, logarithm of odds ratio that a variant is real &lt;em&gt;vs.&lt;/em&gt; not under the trained Gaussian mixture model; GATK, Genome Analysis Toolkit. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/655/F5.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/4/655/F5.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/655/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/655/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465893" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Interrelation of different genomic sequence data sources using the RIG workflow. (A) Schematic of how variants from reduced representation sequence (RR) data present in whole-genome sequence (WGS) data can be used to VQSR the WGS raw variants and assign VQSLOD scores to those variants. (B−D) Visualization of the genomic region of Sb07g003860, a gene involved in sorghum midrib coloration (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bout and Vermerris 2003</a>). (B) the Sbi1.4 gene annotation; (C) shows the assigned VQSLOD scores for variants called in the region from WGS data; (D) shows the depth of coverage and mapped sequence reads for reduced representation and WGS data, respectively, for one sorghum line (BTx642). The RIG workflow enables variants called in the reduced representation sequence data to be used to inform and recalibrate the WGS analyses, and <em>vice versa</em>. This puts all of the variant calls into the GATK’s probabilistic framework whereby variants can be filtered based on their reliability. Users interested in more sensitive or specific call sets can choose more inclusive or exclusive tranches, respectively, by changing the cutoff indicated by the blue dotted line in Panel C. The common and standardized file formats emitted by the GATK enable downstream interoperability between analysis and visualization tools, such as the Integrative Genomics Viewer that produced (B) and (D) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Thorvaldsdóttir <em>et al.</em> 2012</a>). RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; VQSLOD, logarithm of odds ratio that a variant is real <em>vs.</em> not under the trained Gaussian mixture model; GATK, Genome Analysis Toolkit. </p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Evaluation of recalibrated variants from the RIG workflow</h3><p id="p-31">Although a formal evaluation of the accuracy of variant calling pipelines remains unfeasible for nonsimulated sequence data (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Li 2014</a>), we estimated the performance of the workflow using both sorghum and <em>Arabidopsis</em> sequence data. For the sorghum data, we compared the variants called from sorghum WGS data via the RIG workflow to (i) a collection of reliable variants that were not used to train the VQSR models and (ii) a previously published sorghum variant calling analysis. We then used the sorghum WGS variants to recalibrate reduced representation data, and used the recalibrated variants for a genome-wide association study. Lastly, we further validated the performance of the RIG workflow using publicly available Sanger sequence and WGS data from <em>Arabidopsis</em>.</p><div id="sec-11" class="subsection"><h4>Evaluation of recalibrated sorghum variants:</h4><p id="p-32">First, we examined the overlap between the recalibrated sorghum WGS variants and a collection of reliable variants that were not used to train the VQSR models. This collection of reliable variants, hereafter referred to as the Independent-Family (IF) set, originated from a biparental cross genotyped using a reduced representation method; the IF set was obtained in a similar manner to the Family Reference Variant Resource that was used for training during VQSR, and the IF set represented a set of highly specific, genetically mappable variants (see the section <em>Materials and Methods</em>). Of the 10,737 SNPs and 3740 indels in the IF set, 10,557 SNPs and 3632 indels had also been called from the 49 WGS samples (of which 2 samples represented the parents of the biparental cross). The IF variants present in the recalibrated WGS variants had median VQSLOD scores of 8.22 and 5.29 for SNPs and indels, respectively, suggesting that the trained Gaussian mixture models correctly assigned true variants with highly positive VQSLOD scores (<a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS1.pdf">Table S1</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS2.pdf">Table S2</a>). Furthermore, the proportion of IF set variants that were also contained in the 95% and 75% tranches correspond to their respective tranche cutoffs, indicating that the tranche cutoffs were functioning as expected. Since tranche cutoffs represent the VQSLOD score over which a certain proportion of variants from the designated VQSR truth set will be retained, we expected the proportion of IF variants present in each tranche to approximate the tranche cutoff. As expected, proportions of the IF set retained in each tranche were similar to the tranche cutoff. For example, the 95% SNP tranche retained 97% of the SNPs in the IF set, and the 95% indel tranche retained 94% of the indels in the IF set (<a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS2.pdf">Table S2</a>). These results indicate that the Gaussian mixture models for the WGS data were adequately trained and that the tranche cutoffs were functioning as expected.</p><p id="p-33">Second, we compared the recalibrated sorghum WGS variants to a previously published sorghum variant calling analysis. The previous study from <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Mace <em>et al.</em> (2013)</a> called SNPs and indels from 47 sorghum WGS samples; the SNP calls were recently made available as part of Gramene build 42 (accessed September 2014), hereafter referred to as the Gramene42-Mace2013 set (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Monaco <em>et al.</em> 2014</a>). After excluding noncomparable variants from the calls produced by the RIG workflow (<em>i.e.</em>, indels, SNPs on super contigs, and variants not found in the 47 samples), we obtained a Raw set comprised of 18,160,612 SNPs. We constructed an additional two sets from this Raw set for comparison: the Sensitive set, derived from the 95% tranche and comprised of 8,071,250 SNPs, and the Specific set, derived from the 75% tranche and comprised of 3,353,064 SNPs. Of the 6,450,628 SNPs in Gramene42-Mace2013 set, 5,002,099 were present in the Raw set. It is difficult to conclusively attribute the 1,448,529 SNP difference to any specific factors, and high discordance between different variant callers is not uncommon (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">O’Rawe <em>et al.</em> 2013</a>); we note that <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Mace <em>et al.</em> (2013)</a> did not perform BQSR nor realignment around indels prior to calling SNPs, and they also used a different SNP calling algorithm. The overlapping 5,002,099 SNPs were used to compare the distribution of VQSLOD scores between the four sets (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Because the VQSLOD score of all of the SNPs in the comparison were assigned under the same Gaussian mixture model and because the model was adequately trained as shown by the IF validation, comparisons of the relative sensitivity and specificity between the sets can be made. Given two sets of variants with similar VQSLOD distributions, the larger of the two sets contains more variants that are as likely to be true positives than the smaller set and is thus more sensitive. Furthermore, given two sets of variants where the VQSLOD distribution of one set contains a greater proportion of high VQSLOD score variants, the set with the greater proportion of high VQSLOD score variants contains variants that are more likely to be true positives and is thus more specific. As such, we find that the Raw set is the most sensitive but least specific; correspondingly, the Specific set is the most specific but least sensitive (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). The Sensitive set produced by the RIG workflow shows a dramatic improvement over the Gramene42-Mace2013 set in that it contains 1,620,622 more SNPs than the Gramene42-Mace2013 set while the median VQSLOD score remains similar with fewer negative VQSLOD scores, suggesting that the RIG workflow enabled greatly increased sensitivity without a corresponding loss in specificity.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/655/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of VQSLOD score distributions for RIG-produced variant sets and a variant set from a previous study. VQSLOD (log of odds that a variant is real vs. not under the trained Gaussian mixture model) scores were calculated during VQSR of SNPs found in whole-genome sequence data using a Gaussian mixture model trained using SNPs originally found in reduced representation sequence data. For the 5,002,099 SNPs from Gramene42-Mace2013 that had been assigned VQSLOD scores in the Raw set produced by the RIG workflow, the median VQSLOD score is similar to the median of the 8,071,250 SNPs in the Sensitive set. The Sensitive set contains 1,620,622 more SNPs than the 6,450,628 SNPs in Gramene42-Mace2013, suggesting that the RIG-enabled VQSR allowed for a considerably more sensitive call set without a corresponding loss in specificity. VQSLOD, logarithm of odds ratio that a variant is real vs. not under the trained Gaussian mixture model; RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; SNP, single-nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582328943" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of VQSLOD score distributions for RIG-produced variant sets and a variant set from a previous study. VQSLOD (log of odds that a variant is real &lt;em&gt;vs.&lt;/em&gt; not under the trained Gaussian mixture model) scores were calculated during VQSR of SNPs found in whole-genome sequence data using a Gaussian mixture model trained using SNPs originally found in reduced representation sequence data. For the 5,002,099 SNPs from Gramene42-Mace2013 that had been assigned VQSLOD scores in the Raw set produced by the RIG workflow, the median VQSLOD score is similar to the median of the 8,071,250 SNPs in the Sensitive set. The Sensitive set contains 1,620,622 more SNPs than the 6,450,628 SNPs in Gramene42-Mace2013, suggesting that the RIG-enabled VQSR allowed for a considerably more sensitive call set without a corresponding loss in specificity. VQSLOD, logarithm of odds ratio that a variant is real &lt;em&gt;vs.&lt;/em&gt; not under the trained Gaussian mixture model; RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; SNP, single-nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/655/F6.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/4/655/F6.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/655/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/655/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465925" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Comparison of VQSLOD score distributions for RIG-produced variant sets and a variant set from a previous study. VQSLOD (log of odds that a variant is real <em>vs.</em> not under the trained Gaussian mixture model) scores were calculated during VQSR of SNPs found in whole-genome sequence data using a Gaussian mixture model trained using SNPs originally found in reduced representation sequence data. For the 5,002,099 SNPs from Gramene42-Mace2013 that had been assigned VQSLOD scores in the Raw set produced by the RIG workflow, the median VQSLOD score is similar to the median of the 8,071,250 SNPs in the Sensitive set. The Sensitive set contains 1,620,622 more SNPs than the 6,450,628 SNPs in Gramene42-Mace2013, suggesting that the RIG-enabled VQSR allowed for a considerably more sensitive call set without a corresponding loss in specificity. VQSLOD, logarithm of odds ratio that a variant is real <em>vs.</em> not under the trained Gaussian mixture model; RIG, Recalibration and Interrelation of genomic sequence data with the GATK; VQSR, Variant Quality Score Recalibration; SNP, single-nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">As a final validation of the workflow with sorghum variants, we used a set of variants from reduced representation sequence data that had been recalibrated with WGS data to reproduce genome wide association results from the sorghum literature. There were 171 individuals contained within our reduced representation samples that had also previously been phenotyped as part of a sorghum association panel (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2008</a>). After recalibrating the reduced representation data with the WGS data, we used the genotypes for these 171 individuals and phenotypes from <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> (2008)</a> to calculate genome wide associations (<a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS3.pdf">Table S3</a>) and reproduced known sorghum height QTL (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Morris <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Higgins <em>et al.</em> 2014</a>). As such, the recalibrated reduced representation variants produced by the RIG workflow are useful for common downstream analyses, and these analyses are readily executable due to the GATK’s use of standard file formats.</p></div><div id="sec-12" class="subsection"><h4>Evaluation of recalibrated Arabidopsis variants:</h4><p id="p-36">Some organisms may not have sufficient data available from different template preparation methods to execute the RIG workflow as we did for sorghum. As such, we validated the performance of the RIG workflow using only WGS data as both the source of reliable variants and the analysis target. Efficacy of RIG was determined by comparing the variant calls produced by RIG from publicly available <em>Arabidopsis</em> WGS data against a collection of known variants from Sanger sequence data and variants present in the Gramene database (build 43; accessed January 2015) (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> 2005</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Cao <em>et al.</em> 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Schmitz <em>et al.</em> 2013</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Monaco <em>et al.</em> 2014</a>).</p><p id="p-37">The comparison used variant calls from 419 genomic intervals spanning 200,887 bp (containing at least 2,850 SNP and 375 indels) for 18 <em>Arabidopsis</em> accessions. Variants from the Sanger sequence data not present in the RIG variants were considered false negatives, and RIG variants not present in either the Sanger data or the Gramene build 43 set were considered false positives; these values were used to estimate sensitivity and positive predictive value of multiple tranches produced with RIG from <em>Arabidopsis</em> WGS data (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/13/g3.115.017012.DC1/TableS4.pdf">Table S4</a>). This yielded a conservative estimate of RIG variant calls whereby 95% sensitivity and 99% positive predictive value are achieved in one tranche with the RIG workflow (the 99.0% tranche in this case). As shown in the sorghum data, larger percentage tranches are more sensitive but less specific; smaller percentage tranches are less sensitive but more specific. The optimal choice of tranche will, again, depend on the downstream application for which the variant set will be used. We note that the sensitivity does not correspond with the tranche cutoffs as well as they did in the sorghum validation; this may be a result of the low sensitivity of the Sanger variants due to manual removal of variant calls by <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Nordborg <em>et al.</em> (2005)</a> during data curation. Ultimately, this <em>Arabidopsis</em> validation in combination with the sorghum validation demonstrates that the RIG workflow can produce accurate call sets from a variety of genomic data sources.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465888/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465888/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Performance of tranches from <em>Arabidopsis</em> WGS sequence data</span><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-39">The GATK has been shown to outperform other variant calling methods in benchmarking studies, and the RIG workflow enables the analysis benefits afforded by the GATK to research communities lacking validated variant resources (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Pirooznia <em>et al.</em> 2014</a>). RIG also provides access to features absent in current reduced representation sequence data analysis platforms. Two popular reduced representation sequence data analysis solutions, TASSEL and Stacks, are highly specialized for their respective data sources (GBS and RAD-seq, respectively), and they perform well in their target domains; however, they lack features that readily allow the interrelation of WGS with reduced representation sequence data, as well as the ability to accurately call indels (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Catchen <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Glaubitz <em>et al.</em> 2014</a>). RIG provides a means to access both of these features, as well as benefit from accuracy gains from BQSR, joint genotyping, and VQSR. For organisms with a reference genome, the RIG workflow stands as a useful analysis alternative applicable to both reduced representation and WGS data, and RIG is also readily applicable to exome and RNA-seq data due to the GATK’s flexibility. However, because the GATK, and by extension, RIG, cannot operate without a reference genome, software like TASSEL and Stacks will continue to fill important analysis roles, although this may change if software like dDocent, which allows users to take advantage of some of the GATK’s benefits even in the absence of a complete reference genome, gain adoption (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Puritz <em>et al.</em> 2014</a>). Ultimately, RIG was developed in the context of a genetics lab seeking accurate variant calls from multiple sequence data sources for agriculturally important organisms with a reference genome, and we expect it will be beneficial to those with similar use cases.</p><p id="p-40">The RIG workflow requires that the shared variants are comprised of 10,000 SNPs and 10,000 indels for VQSR; however, the GATK developers have successfully used considerably fewer to good effect (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Depristo <em>et al.</em> 2011</a>). We chose 10,000 for both SNPs and indels as the requirement because we have obtained useful results using these values; the values are not a hard rule. As such, the user can construct their own values by evaluating the VQSR and BQSR reports produced by the GATK to determine whether (i) the Gaussian mixture models were adequately trained to distinguish between variants of differing reliability, and (ii) whether the empirically determined base quality score recalibrations appear reasonable for the sequencing platform.</p><p id="p-41">In cases in which sequence data from different template preparation methods are not available, it will not be possible to identify shared variants from orthogonal approaches as we did with sorghum sequence data. We ensured the variants designated as truth for VQSR originated from an analysis target produced by a different method (<em>e.g.</em>, variants found in reduced representation data were used for VQSR of a WGS analysis target). This enforced that variants used in VQSR were found in two independent template preparation methods to approximate variants found using orthogonal methods. Since such genomic resources may not always be available, we also evaluated performance of a use case where only WGS data were available, and we showed that high levels of sensitivity and positive predictive value can be achieved using only WGS data. In cases in which the analysis target is the only source of variants, we and other GATK users have had some success by taking the analysis target through the naive pipeline, hard filtering to generate a temporary sensitive variant resource, and using that temporary sensitive variant resource to BQSR the analysis target. This procedure is iteratively repeated until the BQSR results from the current and preceding iteration converge, and then a specific variant resource is generated by stringently hard filtering to use as a bootstrapped variant resource in VQSR. This ultimately skews VQSR based on the annotations used to hard filter the variants during bootstrapping, but communities lacking sufficient data sources may find this procedure to be an acceptable alternative.</p><p id="p-42">The RIG workflow enables research communities to use the GATK (i) to interrelate different sequencing template preparation methods such as reduced representation and WGS into common, standardized file formats; (ii) accurately call genetic variants from genomic sequence data; and (iii) to iteratively refine variant resources. The RIG workflow will contribute to progress in construction of more complete catalogs of genetic variation, and the ability to readily interrelate variants from different sequence data sources using the GATK will increase the rate at which variants associated with a phenotype lead to the identification of the genetic variation that causes the phenotype.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We’d like to thank the developers and communities surrounding the following software projects for helpful advice, forum discussions, and making their code available: GATK (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Van Der Auwera <em>et al.</em> 2013</a>), matplotlib (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hunter 2007</a>), Biopieces (<a href="http://www.biopieces.org">www.biopieces.org</a>), GNU Parallel (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tange 2011</a>), and HTSeq (Anders <em>et al.</em> 2014). We acknowledge Whole Systems Genomics Initiative (WSGI) for providing computational resources and systems administration support for the WSGI HPC Cluster. This work was supported in part by Great Lakes Bioenergy Research Center and the Perry Adkisson Chair (to J.E.M).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-44">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017012/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017012/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-11-17"><span class="received-label">Received </span>November 17, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-12"><span class="accepted-label">Accepted </span>February 12, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 McCormick <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-5.4.655.1" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyl</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HTSeq-A Python framework to work with high-throughput sequencing data</span>. <span class="cit-source">Bioinformatics</span> <span class="cit-vol">31</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMS8yLzE2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.2" data-doi="10.1038/35048692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Arabidopsis Genome Initiative</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bout</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vermerris</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A candidate-gene approach to clone the sorghum brown midrib gene encoding caffeic acid o-methyltransferase.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DBout%26rft.auinit1%253DS.%26rft.volume%253D269%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D214%26rft.atitle%253DA%2Bcandidate-gene%2Bapproach%2Bto%2Bclone%2Bthe%2Bsorghum%2BBrown%2Bmidrib%2Bgene%2Bencoding%2Bcaffeic%2Bacid%2BO-methyltransferase.%26rft_id%253Dinfo%253Apmid%252F12756532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12756532&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183870300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.4" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.5" data-doi="10.1534/genetics.108.092239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient mapping of plant height quantitative trait loci in a sorghum association population with introgressed dwarfing genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092239%26rft_id%253Dinfo%253Apmid%252F18757942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.6" data-doi="10.3198/jpr2010.04.0219crmp"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Registration of the BTx623/IS3620C recombinant inbred mapping population of sorghum.</span> <abbr class="cit-jnl-abbrev">J. Plant Regis.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegis.%26rft.volume%253D5%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2010.04.0219crmp%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2010.04.0219crmp&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.7" data-doi="10.1038/ng.911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCao%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D956%26rft.epage%253D963%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Bmultiple%2BArabidopsis%2Bthaliana%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.911%26rft_id%253Dinfo%253Apmid%252F21874002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874002&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.655.8" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci de novo from short-read sequences.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCatchen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DStacks%253A%2BBuilding%2Band%2BGenotyping%2BLoci%2BDe%2BNovo%2BFrom%2BShort-Read%2BSequences%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.9" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.10" data-doi="10.1371/journal.pone.0079192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Extensive variation in the density and distribution of DNA polymorphism in sorghum genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e79192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De79192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0079192%26rft_id%253Dinfo%253Apmid%252F24265758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0079192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24265758&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.11" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.12" data-doi="10.1534/g3.114.013318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurber</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assaranurak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiparental mapping of plant height and flowering time QTL in partially isogenic sorghum families.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1593</span>–<span class="cit-lpage">1602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DHiggins%26rft.auinit1%253DR.%2BH.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1593%26rft.epage%253D1602%26rft.atitle%253DMultiparental%2BMapping%2Bof%2BPlant%2BHeight%2Band%2BFlowering%2BTime%2BQTL%2Bin%2BPartially%2BIsogenic%2BSorghum%2BFamilies%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013318%26rft_id%253Dinfo%253Apmid%252F25237111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNTkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.13" data-doi="10.1109/MCSE.2007.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Matplotlib: A 2D graphics environment.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.655.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Towards better understanding of artifacts in variant calling from high-coverage samples.</span> <span class="cit-source">arXiv</span> 1404.0929.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.15" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.16" data-doi="10.1093/nar/gkh850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Adjust quality scores from alignment and improve sequencing accuracy.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">5183</span>–<span class="cit-lpage">5191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DM.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5183%26rft.epage%253D5191%26rft.atitle%253DAdjust%2Bquality%2Bscores%2Bfrom%2Balignment%2Band%2Bimprove%2Bsequencing%2Baccuracy%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh850%26rft_id%253Dinfo%253Apmid%252F15459287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNy81MTgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.17" data-doi="10.1371/journal.pone.0075619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelernter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B.-Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Variant callers for next-generation sequencing data: a comparison study.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e75619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De75619%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0075619%26rft_id%253Dinfo%253Apmid%252F24086590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0075619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24086590&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilding</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prentis</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome sequencing reveals untapped genetic potential in Africa’s indigenous cereal crop sorghum.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2320%26rft_id%253Dinfo%253Apmid%252F23982223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23982223&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.19" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.20" data-doi="10.1093/nar/gkt1110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monaco</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naithani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dharmawardhana</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gramene 2013: comparative plant genomics resources.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D1193</span>–<span class="cit-lpage">D1199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1110%26rft_id%253Dinfo%253Apmid%252F24217918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi9EMS9EMTE5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.21" data-doi="10.1186/1471-2164-14-448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahraeian</span>  <span class="cit-name-given-names">S. M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Digital genotyping of sorghum–a diverse plant species with a large repeat-rich genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D448%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-448%26rft_id%253Dinfo%253Apmid%252F23829350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23829350&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.22" data-doi="10.1073/pnas.1215985110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomic and genome-wide association studies of agroclimatic traits in sorghum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215985110%26rft_id%253Dinfo%253Apmid%252F23267105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMi80NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.23" data-doi="10.1038/nrg3305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Next-generation sequencing data interpretation: enhancing reproducibility and accessibility.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">667</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNekrutenko%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D667%26rft.epage%253D672%26rft.atitle%253DNext-generation%2Bsequencing%2Bdata%2Binterpretation%253A%2Benhancing%2Breproducibility%2Band%2Baccessibility.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3305%26rft_id%253Dinfo%253Apmid%252F22898652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22898652&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.24" data-doi="10.1371/journal.pbio.0030196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishino</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jhaveri</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The pattern of polymorphism in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De196%26rft.epage%253De196%26rft.atitle%253DThe%2Bpattern%2Bof%2Bpolymorphism%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030196%26rft_id%253Dinfo%253Apmid%252F15907155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15907155&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.25" data-doi="10.1186/gm432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Rawe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Med</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed%26rft.volume%253D5%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm432%26rft_id%253Dinfo%253Apmid%252F23537139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23537139&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.26" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.655.27"><div class="cit-metadata unstructured">Picard Team, 2014 Picard. Available at: <a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>. Accessed: February 26, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.28" data-doi="10.1186/1479-7364-8-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pirooznia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parla</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potash</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Validation and assessment of variant calling pipelines for next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Hum. Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenomics%26rft.volume%253D8%26rft.spage%253D14%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1479-7364-8-14%26rft_id%253Dinfo%253Apmid%252F25078893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1479-7364-8-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25078893&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.29" data-doi="10.7717/peerj.431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gold</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">dDocent: a RADseq, variant-calling pipeline designed for population genomics of non-model organisms.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253De431%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.431%26rft_id%253Dinfo%253Apmid%252F24949246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24949246&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.30" data-doi="10.1038/nature11968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urich</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nery</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelizzola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Patterns of population epigenomic diversity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11968%26rft_id%253Dinfo%253Apmid%252F23467092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23467092&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316652300042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.31" data-doi="10.1038/nrg3642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudbery</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilott</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing depth and coverage: key considerations in genomic analyses.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3642%26rft_id%253Dinfo%253Apmid%252F24434847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24434847&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tange</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GNU parallel—the command-line power tool.;login</span>. <abbr class="cit-jnl-abbrev">The USENIX Magazine</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BUSENIX%2BMagazine%26rft.volume%253D36%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D14%26rft.spage%253D178%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.655.34" data-doi="10.1534/g3.114.012468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Resolution of genetic map expansion caused by excess heterozygosity in plant recombinant inbred populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012468%26rft_id%253Dinfo%253Apmid%252F25128435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTAvMTk2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.655.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Der Auwera</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Angel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subudhi</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crasta</span>  <span class="cit-name-given-names">O. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenow</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Molecular mapping of QTLs conferring stay-green in grain sorghum (<em>Sorghum bicolor</em> l. moench).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DXu%26rft.auinit1%253DW.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D461%26rft.epage%253D469%26rft.atitle%253DMolecular%2Bmapping%2Bof%2BQTLs%2Bconferring%2Bstay-green%2Bin%2Bgrain%2Bsorghum%2B%2528Sorghum%2Bbicolor%2BL.%2BMoench%2529.%26rft_id%253Dinfo%253Apmid%252F10902709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10902709&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.655.37" data-doi="10.1186/gb-2011-12-11-r114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide patterns of genetic variation in sweet and grain sorghum (<em>Sorghum bicolor</em>).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZheng%26rft.auinit1%253DL.%2BY.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253DR114%26rft.epage%253DR114%26rft.atitle%253DGenome-wide%2Bpatterns%2Bof%2Bgenetic%2Bvariation%2Bin%2Bsweet%2Band%2Bgrain%2Bsorghum%2B%2528Sorghum%2Bbicolor%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-11-r114%26rft_id%253Dinfo%253Apmid%252F22104744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-11-r114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22104744&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/655.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/639" title="Conserved RNA-Binding Proteins Required for Dendrite Morphogenesis in Caenorhabditis elegans Sensory Neurons" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/667" title="Drosophila melanogaster Activating Transcription Factor 4 Regulates Glycolysis During Endoplasmic Reticulum Stress" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aO1yS9_5nJ-Vj_xtND9U8p5e-wzO5o_AHI8V0iC2sNE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/655" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/655" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/655" class="active" data-icon-position="" data-hide-link-title="0">RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459796" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ChqXDLWgcCCcdI9A6k8rubTwUkmCiWG0aEx3e_b7JxI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/655.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/655" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YkQaGUXyN3JU0afmrlIR1tNueheOrUfCVr0QFSmsVJ4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/655" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/655.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459796" id="node459796--2" data-pisa="ggg;5/4/655" data-pisa-master="ggg;g3.115.017012" data-apath="/ggg/5/4/655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan F. McCormick</span>, <span class="highwire-citation-author" data-delta="1">Sandra K. Truong</span> and <span class="highwire-citation-author" data-delta="2">John E. Mullet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">655-665;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017012 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459796--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan F. McCormick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BF.%2BMcCormick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCormick%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BF.%2BMcCormick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sandra K. Truong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BK.%2BTruong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truong%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BK.%2BTruong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John E. Mullet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BMullet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mullet%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BE.%2BMullet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmullet@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">jmullet@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/655" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/655&amp;title=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/655&amp;title=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/655&amp;title=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/655&amp;text=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/655&amp;title=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/655&amp;t=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/655&amp;title=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/655&amp;title=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459796/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459796" id="node459796--3" data-pisa="ggg;5/4/655" data-pisa-master="ggg;g3.115.017012" data-apath="/ggg/5/4/655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RIG: Recalibration and Interrelation of Genomic Sequence Data with the GATK</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan F. McCormick</span>, <span class="highwire-citation-author" data-delta="1">Sandra K. Truong</span> and <span class="highwire-citation-author" data-delta="2">John E. Mullet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">655-665;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017012 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459796--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan F. McCormick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BF.%2BMcCormick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCormick%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BF.%2BMcCormick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sandra K. Truong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BK.%2BTruong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truong%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BK.%2BTruong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John E. Mullet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BMullet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mullet%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BE.%2BMullet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmullet@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">jmullet@tamu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=655&amp;rft.epage=665&amp;rft.atitle=RIG%3A%20Recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=McCormick%2C+Ryan+F.&amp;rft.au=Truong%2C+Sandra+K.&amp;rft.au=Mullet%2C+John+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459796/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459796/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459796/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459796/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459796/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459796/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459796/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459796/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459796/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459796/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459796/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459796/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/655" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/655.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/655.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/655.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/655.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
