<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/593" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/593.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/593.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/593.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/593.atom" />
<meta name="HW.pisa" content="ggg;5/4/593" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Global Diversity Lines–A Five-Continent Reference Panel of Sequenced Drosophila melanogaster Strains" />
<meta name="DC.Identifier" content="10.1534/g3.114.015883" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Grenier et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Reference collections of multiple Drosophila lines with accumulating collections of “omics” data have proven especially valuable for the study of population genetics and complex trait genetics. Here we present a description of a resource collection of 84 strains of Drosophila melanogaster whose genome sequences were obtained after 12 generations of full-sib inbreeding. The initial rationale for this resource was to foster development of a systems biology platform for modeling metabolic regulation by the use of natural polymorphisms as perturbations. As reference lines, they are amenable to repeated phenotypic measurements, and already a large collection of metabolic traits have been assayed. Another key feature of these strains is their widespread geographic origin, coming from Beijing, Ithaca, Netherlands, Tasmania, and Zimbabwe. After obtaining 12.5× coverage of paired-end Illumina sequence reads, SNP and indel calls were made with the GATK platform. Thorough quality control was enabled by deep sequencing one line to &gt;100×, and single-nucleotide polymorphisms and indels were validated using ddRAD-sequencing as an orthogonal platform. In addition, a series of preliminary population genetic tests were performed with these single-nucleotide polymorphism data for assessment of data quality. We found 83 segregating inversions among the lines, and as expected these were especially abundant in the African sample. We anticipate that this will make a useful addition to the set of reference D. melanogaster strains, thanks to its geographic structuring and unusually high level of genetic diversity." />
<meta name="DC.Contributor" content="Jennifer K. Grenier" />
<meta name="DC.Contributor" content="J. Roman Arguello" />
<meta name="DC.Contributor" content="Margarida Cardoso Moreira" />
<meta name="DC.Contributor" content="Srikanth Gottipati" />
<meta name="DC.Contributor" content="Jaaved Mohammed" />
<meta name="DC.Contributor" content="Sean R. Hackett" />
<meta name="DC.Contributor" content="Rachel Boughton" />
<meta name="DC.Contributor" content="Anthony J. Greenberg" />
<meta name="DC.Contributor" content="Andrew G. Clark" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Global Diversity Lines–A Five-Continent Reference Panel of Sequenced Drosophila melanogaster Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Reference collections of multiple &lt;i&gt;Drosophila&lt;/i&gt; lines with accumulating collections of “omics” data have proven especially valuable for the study of population genetics and complex trait genetics. Here we present a description of a resource collection of 84 strains of &lt;i&gt;Drosophila melanogaster&lt;/i&gt; whose genome sequences were obtained after 12 generations of full-sib inbreeding. The initial rationale for this resource was to foster development of a systems biology platform for modeling metabolic regulation by the use of natural polymorphisms as perturbations. As reference lines, they are amenable to repeated phenotypic measurements, and already a large collection of metabolic traits have been assayed. Another key feature of these strains is their widespread geographic origin, coming from Beijing, Ithaca, Netherlands, Tasmania, and Zimbabwe. After obtaining 12.5× coverage of paired-end Illumina sequence reads, SNP and indel calls were made with the GATK platform. Thorough quality control was enabled by deep sequencing one line to &amp;gt;100×, and single-nucleotide polymorphisms and indels were validated using ddRAD-sequencing as an orthogonal platform. In addition, a series of preliminary population genetic tests were performed with these single-nucleotide polymorphism data for assessment of data quality. We found 83 segregating inversions among the lines, and as expected these were especially abundant in the African sample. We anticipate that this will make a useful addition to the set of reference &lt;i&gt;D. melanogaster&lt;/i&gt; strains, thanks to its geographic structuring and unusually high level of genetic diversity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/593" />
<meta name="citation_id" content="5/4/593" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/593" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/593.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/593.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/593.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015883" />
<meta name="citation_pmid" content="25673134" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="593" />
<meta name="citation_lastpage" content="603" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jennifer K. Grenier" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="J. Roman Arguello" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015" />
<meta name="citation_author" content="Margarida Cardoso Moreira" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015" />
<meta name="citation_author" content="Srikanth Gottipati" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Translational Medicine and Think Team, Otsuka Pharmaceutical Development and Commercialization, Inc., Princeton, New Jersey 08540" />
<meta name="citation_author" content="Jaaved Mohammed" />
<meta name="citation_author_institution" content="Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Sean R. Hackett" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Quantitative and Computational Biology, Princeton University, Princeton, New Jersey 08544" />
<meta name="citation_author" content="Rachel Boughton" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Anthony J. Greenberg" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Andrew G. Clark" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_email" content="ac347@cornell.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Andolfatto;citation_author=J. D. Wall;citation_author=M. Kreitman;citation_title=Unusual Haplotype Structure at the Proximal Breakpoint of In(2L)t in a Natural Population of Drosophila melanogaster;citation_pages=1297-1311;citation_volume=153;citation_year=1999;citation_issue=3;citation_pmid=10545460" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. Arnold;citation_author=R. B. Corbett-Detig;citation_author=D. Hartl;citation_author=K. Bomblies;citation_title=RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling.;citation_pages=3179-3190;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12276" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. F. Ayroles;citation_author=M. A. Carbone;citation_author=E. A. Stone;citation_author=K. W. Jordan;citation_author=R. F. Lyman;citation_title=Systems genetics of complex traits in Drosophila melanogaster.;citation_pages=299-307;citation_volume=41;citation_year=2009;citation_issue=3;citation_pmid=19234471;citation_doi=10.1038/ng.332" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Baudry;citation_author=B. Viginier;citation_author=M. Veuille;citation_title=Non-African populations of Drosophila melanogaster have a unique origin.;citation_volume=21;citation_year=2004;citation_pmid=15014160;citation_doi=10.1093/molbev/msh089" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Begun;citation_author=C. F. Aquadro;citation_title=African and North American populations of Drosophila melanogaster are very different at the DNA level.;citation_pages=548-550;citation_volume=365;citation_year=1993;citation_issue=6446;citation_pmid=8413609;citation_doi=10.1038/365548a0" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=R. H. Blair;citation_author=D. J. Kliebenstein;citation_author=G. A. Churchill;citation_title=What can causal networks tell us about metabolic pathways?;citation_pages=e1002458-e1002458;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22496633;citation_doi=10.1371/journal.pcbi.1002458" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. S. Breunig;citation_author=S. R. Hackett;citation_author=J. D. Rabinowitz;citation_author=L. Kruglyak;citation_title=Genetic basis of metabolome variation in yeast.;citation_pages=e1004142;citation_volume=10;citation_year=2014;citation_pmid=24603560;citation_doi=10.1371/journal.pgen.1004142" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Cardoso-Moreira;citation_author=J. R. Arguello;citation_author=A. G. Clark;citation_title=Mutation spectrum of Drosophila CNVs revealed by breakpoint sequencing.;citation_pages=R119;citation_volume=13;citation_year=2012;citation_pmid=23259534;citation_doi=10.1186/gb-2012-13-12-r119" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Corbett-Detig;citation_author=D. L. Hartl;citation_title=Population genomics of inversion polymorphisms in Drosophila melanogaster.;citation_pages=e1003056;citation_volume=8;citation_year=2012;citation_pmid=23284285;citation_doi=10.1371/journal.pgen.1003056" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. B. Corbett-Detig;citation_author=C. Cardeno;citation_author=C. H. Langley;citation_title=Sequence-based detection and breakpoint assembly of polymorphic inversions.;citation_volume=192;citation_year=2012;citation_pmid=22673805;citation_doi=10.1534/genetics.112.141622" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. W. Davey;citation_author=T. Cezard;citation_author=P. Fuentes-Utrilla;citation_author=C. Eland;citation_author=K. Gharbi;citation_title=Special features of RAD sequencing data: implications for genotyping.;citation_pages=3151-3164;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12084" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. David;citation_author=C. Bocquet;citation_author=E. Pla;citation_title=New results on the genetic characteristics of the Far East race of Drosophila melanogaster.;citation_pages=253-260;citation_volume=28;citation_year=1976;citation_issue=3;citation_pmid=829088;citation_doi=10.1017/S0016672300016943" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Biology;citation_journal_abbrev=J. Exp. Biol.;citation_author=W. F. Eanes;citation_title=Molecular population genetics and selection in the glycolytic pathway;citation_pages=165-171;citation_volume=214;citation_year=2011;citation_issue=2;citation_pmid=21177937;citation_doi=10.1242/jeb.046458" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. M. Early;citation_author=A. G. Clark;citation_title=Monophyly of Wolbachia pipientis genomes within Drosophila melanogaster: geographic structuring, titre variation and host effects across five populations.;citation_pages=5765-5778;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12530" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Gautier;citation_author=J. Foucaud;citation_author=K. Gharbi;citation_author=T. Cézard;citation_author=M. Galan;citation_title=Estimation of population allele frequencies from next-generation sequencing data: pool-vs. individual-based genotyping.;citation_pages=3766-3779;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12360" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=A. Ghazalpour;citation_author=B. J. Bennett;citation_author=D. Shih;citation_author=N. Che;citation_author=L. Orozco;citation_title=Genetic regulation of mouse liver metabolite levels;citation_pages=730-730;citation_volume=10;citation_year=2014;citation_issue=5;citation_pmid=24860088;citation_doi=10.15252/msb.20135004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Glinka;citation_author=L. Ometto;citation_author=S. Mousset;citation_author=W. Stephan;citation_author=D. De Lorenzo;citation_title=Demography and Natural Selection Have Shaped Genetic Variation in Drosophila melanogaster: A Multi-locus Approach;citation_pages=1269-1278;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668381" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Greenberg;citation_author=S. R. Hackett;citation_author=L. G. Harshman;citation_author=A. G. Clark;citation_title=A hierarchical Bayesian model for a novel sparse partial diallel crossing design.;citation_volume=185;citation_year=2010;citation_pmid=20157001;citation_doi=10.1534/genetics.110.115055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=A. J. Greenberg;citation_author=S. R. Hackett;citation_author=L. G. Harshman;citation_author=A. G. Clark;citation_title=Environmental and genetic perturbations reveal different networks of metabolic regulation.;citation_pages=563;citation_volume=7;citation_year=2011;citation_pmid=22186737" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. R. Haddrill;citation_author=K. R. Thornton;citation_author=B. Charlesworth;citation_author=P. Andolfatto;citation_title=Multilocus patterns of nucleotide variability and the demographic and selection history of Drosophila melanogaster populations;citation_pages=790-799;citation_volume=15;citation_year=2005;citation_issue=6;citation_pmid=15930491;citation_doi=10.1101/gr.3541005" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. T. Hu;citation_author=M. B. Eisen;citation_author=K. R. Thornton;citation_author=P. Andolfatto;citation_title=A second-generation assembly of the Drosophila simulans genome provides new insights into patterns of lineage-specific divergence.;citation_volume=23;citation_year=2013;citation_pmid=22936249;citation_doi=10.1101/gr.141689.112" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Ràmia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster Genetic Reference Panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Jumbo-Lucioni;citation_author=J. F. Ayroles;citation_author=M. M. Chambers;citation_author=K. W. Jordan;citation_author=J. Leips;citation_title=Systems genetics analysis of body weight and energy metabolism traits in Drosophila melanogaster.;citation_pages=297-297;citation_volume=11;citation_year=2010;citation_pmid=20459830;citation_doi=10.1186/1471-2164-11-297" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT—the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila Synthetic Population Resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=M. Crepeau;citation_author=C. Cardeno;citation_author=R. Corbett-Detig;citation_author=K. Stevens;citation_title=Circumventing heterozygosity: sequencing the amplified genome of a single haploid Drosophila melanogaster embryo.;citation_volume=188;citation_year=2011;citation_pmid=21441209;citation_doi=10.1534/genetics.111.127530" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=K. Stevens;citation_author=C. Cardeno;citation_author=Y. C. G. Lee;citation_author=D. R. Schrider;citation_title=Genomic variation in natural populations of Drosophila melanogaster.;citation_volume=192;citation_year=2012;citation_pmid=22673804;citation_doi=10.1534/genetics.112.142018" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. J. Y. Laurent;citation_author=A. Werzner;citation_author=L. Excoffier;citation_author=W. Stephan;citation_title=Approximate Bayesian analysis of Drosophila melanogaster polymorphism data reveals a recent colonization of Southeast Asia.;citation_volume=28;citation_year=2011;citation_pmid=21300986;citation_doi=10.1093/molbev/msr031" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Lavington;citation_author=R. Cogni;citation_author=C. Kuczynski;citation_author=S. Koury;citation_author=E. L. Behrman;citation_title=A small system–high-resolution study of metabolic adaptation in the central metabolic pathway to temperate climates in Drosophila melanogaster.;citation_volume=31;citation_year=2014;citation_pmid=24770333;citation_doi=10.1093/molbev/msu146" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W.-P. Lee;citation_author=M. P. Stromberg;citation_author=A. Ward;citation_author=C. Stewart;citation_author=E. P. Garrison;citation_title=MOSAIK: a hash-based algorithm for accurate next-generation sequencing short-read mapping.;citation_pages=e90581;citation_volume=9;citation_year=2014;citation_pmid=24599324;citation_doi=10.1371/journal.pone.0090581" />
<meta name="citation_reference" content="citation_title=Drosophila Inversion Polymorphism;citation_title=Inversion polymorphism in Drosophila melanogaster;citation_author=F. Lemeunier;citation_author=S. Aulard;citation_pages=339-406;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Li;citation_author=W. Stephan;citation_title=Inferring the demographic history and rate of adaptive substitution in Drosophila.;citation_pages=e166-e166;citation_volume=2;citation_year=2006;citation_issue=10;citation_pmid=17040129;citation_doi=10.1371/journal.pgen.0020166" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Massouras;citation_author=S. M. Waszak;citation_author=M. Albarca-Aguilera;citation_author=K. Hens;citation_author=W. Holcombe;citation_title=Genomic Variation and Its Impact on Gene Expression in Drosophila melanogaster.;citation_pages=e1003055-e1003055;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23189034;citation_doi=10.1371/journal.pgen.1003055" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. M. Matzkin;citation_author=T. J. S. Merritt;citation_author=C.-T. Zhu;citation_author=W. F. Eanes;citation_title=The structure and population genetics of the breakpoints associated with the cosmopolitan chromosomal inversion In(3R)Payne in Drosophila melanogaster.;citation_volume=170;citation_year=2005;citation_pmid=15781702;citation_doi=10.1534/genetics.104.038810" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. T. McVean;citation_title=A Genealogical Interpretation of Linkage Disequilibrium;citation_pages=987-991;citation_volume=162;citation_year=2002;citation_issue=2;citation_pmid=12399406" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. Ometto;citation_author=S. Glinka;citation_author=D. De Lorenzo;citation_author=W. Stephan;citation_title=Inferring the effects of demography and selection on Drosophila melanogaster populations from a chromosome-wide scan of DNA variation.;citation_volume=22;citation_year=2005;citation_pmid=15987874;citation_doi=10.1093/molbev/msi207" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population genomics of sub-Saharan Drosophila melanogaster: African diversity and non-African admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Rausch;citation_author=T. Zichner;citation_author=A. Schlattl;citation_author=A. M. Stütz;citation_author=V. Benes;citation_title=DELLY: structural variant discovery by integrated paired-end and split-read analysis.;citation_volume=28;citation_year=2012;citation_pmid=22962449;citation_doi=10.1093/bioinformatics/bts378" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=C. J. F. Scheitz;citation_author=Y. Guo;citation_author=A. M. Early;citation_author=L. G. Harshman;citation_author=A. G. Clark;citation_title=Heritability and inter-population differences in lipid profiles of Drosophila melanogaster.;citation_pages=e72726;citation_volume=8;citation_year=2013;citation_pmid=24013349;citation_doi=10.1371/journal.pone.0072726" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Schlötterer;citation_author=H. Neumeier;citation_author=C. Sousa;citation_author=V. Nolte;citation_title=Highly structured Asian Drosophila melanogaster populations: a new tool for hitchhiking mapping?;citation_volume=172;citation_year=2006;citation_pmid=16204221;citation_doi=10.1534/genetics.105.045831" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Thornton;citation_author=P. Andolfatto;citation_title=Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of Drosophila melanogaster.;citation_volume=172;citation_year=2006;citation_pmid=16299396;citation_doi=10.1534/genetics.105.048223" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. H.-C. Wei;citation_author=J. K. Grenier;citation_author=D. A. Barbash;citation_author=A. G. Clark;citation_title=Correlated variation and population differentiation in satellite DNA abundance among lines of Drosophila melanogaster.;citation_volume=111;citation_year=2014;citation_pmid=25512552;citation_doi=10.1073/pnas.1421951112" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. S. Wesley;citation_author=W. F. Eanes;citation_title=Isolation and analysis of the breakpoint sequences of chromosome inversion In(3L)Payne in Drosophila melanogaster.;citation_pages=3132-3136;citation_volume=91;citation_year=1994;citation_issue=8;citation_pmid=8159716;citation_doi=10.1073/pnas.91.8.3132" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Global Diversity Lines–A Five-Continent Reference Panel of Sequenced Drosophila melanogaster Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Reference collections of multiple Drosophila lines with accumulating collections of “omics” data have proven especially valuable for the study of population genetics and complex trait genetics. Here we present a description of a resource collection of 84 strains of Drosophila melanogaster whose genome sequences were obtained after 12 generations of full-sib inbreeding. The initial rationale for this resource was to foster development of a systems biology platform for modeling metabolic regulation by the use of natural polymorphisms as perturbations. As reference lines, they are amenable to repeated phenotypic measurements, and already a large collection of metabolic traits have been assayed. Another key feature of these strains is their widespread geographic origin, coming from Beijing, Ithaca, Netherlands, Tasmania, and Zimbabwe. After obtaining 12.5× coverage of paired-end Illumina sequence reads, SNP and indel calls were made with the GATK platform. Thorough quality control was enabled by deep sequencing one line to &gt;100×, and single-nucleotide polymorphisms and indels were validated using ddRAD-sequencing as an orthogonal platform. In addition, a series of preliminary population genetic tests were performed with these single-nucleotide polymorphism data for assessment of data quality. We found 83 segregating inversions among the lines, and as expected these were especially abundant in the African sample. We anticipate that this will make a useful addition to the set of reference D. melanogaster strains, thanks to its geographic structuring and unusually high level of genetic diversity." />
<meta name="og-title" property="og:title" content="Global Diversity Lines–A Five-Continent Reference Panel of Sequenced Drosophila melanogaster Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/593" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Reference collections of multiple Drosophila lines with accumulating collections of “omics” data have proven especially valuable for the study of population genetics and complex trait genetics. Here we present a description of a resource collection of 84 strains of Drosophila melanogaster whose genome sequences were obtained after 12 generations of full-sib inbreeding. The initial rationale for this resource was to foster development of a systems biology platform for modeling metabolic regulation by the use of natural polymorphisms as perturbations. As reference lines, they are amenable to repeated phenotypic measurements, and already a large collection of metabolic traits have been assayed. Another key feature of these strains is their widespread geographic origin, coming from Beijing, Ithaca, Netherlands, Tasmania, and Zimbabwe. After obtaining 12.5× coverage of paired-end Illumina sequence reads, SNP and indel calls were made with the GATK platform. Thorough quality control was enabled by deep sequencing one line to &gt;100×, and single-nucleotide polymorphisms and indels were validated using ddRAD-sequencing as an orthogonal platform. In addition, a series of preliminary population genetic tests were performed with these single-nucleotide polymorphism data for assessment of data quality. We found 83 segregating inversions among the lines, and as expected these were especially abundant in the African sample. We anticipate that this will make a useful addition to the set of reference D. melanogaster strains, thanks to its geographic structuring and unusually high level of genetic diversity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>Global Diversity Lines–A Five-Continent Reference Panel of Sequenced Drosophila melanogaster Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TgEfkcYb9zan88b7sxgsq5shRInTOYnihW5N65O6yrw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459810","apath":"\/ggg\/5\/4\/593.atom","pisa":"ggg;5\/4\/593","ac":{"\/ggg\/5\/4\/593.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/593.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/593"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/593"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/593":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oqXHXWCqd4zPiPbOUb4LeeBneKnbN5eYB6S0xBxjW\\\\\\\/3E1Nvk1mcgddn3mGM\\\\\\\/HtjSZcZKWW9hN+5hDKyTZwOAqoiWcGUDDrqjeKs\\\\\\\/AovaoEiBv6t\\\\\\\/M8a0tGYgklT3FwV0UI89Q0pqGAjLAeZOIbAHKgM9tuFeRVfFeaSa8idY9lfEjW8vo2g9cfLrsthAzE5sZUC3QvXfHF7K+tnmZ3RrM5STO\\\\\\\/5xzCWYB86ZHy1XbwX+QVi5Jko2LfKkq6bi16ejFCxa+cvg2I5Il+YPn\\\\\\\/OhIWnm9y03JqzIfYUOf0n0eLRJ3ltH6vgMBbbCu1SeKcs9I\\\\\\\/A4tPPNpbA\\\\\\\/+fvFXSc8R3\\\\\\\/CWUbY3ltrSPXGroHm1+uxgxnwp8bP27UqnRsVtC0ClXHfBTGoiQ7OZjZvNeKWYP1xz8tf1BgkNpoVGGDTjCryYFJWtPov7+e60d6ufzpsq2P5tqjyblVvGNgTFs6byKhT0qCUaMW6JGDxNEW9+fqo7paGAhFRtaD4g72psNH8es2iv\\\\\\\/+gXdsylLyfLLuDHGK5QeYGhYtoJQ8kIE3ddguaaZoTGyznhnBl\\\\\\\/9NXMlm7apxgYG147uWnYz\\\\\\\/UwyaTl9hXFRDknYp69tGWr0YQ1Re3mt7+MOZEtEhPH15mrMPsk8WAkJ\\\\\\\/unqWGOicGK1wvM\\\\\\\/1NlZwrRT9DKY4cAV2EeU1gO2SQ2kLugduU9tmTO3SNN8+QG2qMaYUR\\\\\\\/suY7aW6olUv6fwoyS0pHe4y3BOalVy0o5qBIjihGdiA6ToWryrFfeQkUZHuOq+Q6nPbVOBcVVSKtxIDWxEKfhxWHLTcwap+gU5iEqYv\\\\\\\/SBwGBkThn1KqXtoqAjMhjjlkHoDFzVprO3F0vkaDyF07JGzjYOHZgP29BUypfsvza\\\\\\\/K8pWCQ0aHe83HwSENd5bnOyRzDBkBbzpohuQHbf6Lg\\\\\\\/mh\\\\\\\/AwLAvZ49AeJF8zCHhUINmkuP0RmanWeJ7A5djCHBCbH1b9hooMssmiQXIOFSVZO3XXm8k2rCeEkvWIsa1tYFJ716AZE2eOxUasr4DaQ0U6GE7V2kNu0hEnezHAqlnVsnVzen6GJ7mtgrfUw1PPF25IOICq3JOhIXoveErjTnyeinrrqd\\\\\\\/7eveNllIxWKhrv5M\\\\\\\/K1SKOSogNXJbgLwdbM4GDn6jY9SmbHAbUxjMPbnhFBYegKruweQZdeI7CtG0iLZ6vhriciJPhFnfvMYZBtRnYyBu2\\\\\\\/fVynoQiCo2O7o+OlboQx3Y+P0A5UOIx8yM5JY5APcd5bwRpC55baVWlYUZw13M9\\\u0022,\\\u0022iv\\\u0022:\\\u0022knPTEcthELVDpIuyVO4a\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cee58fe2d5774463adac4264f49da6bcb9cadfd40a1199caf1c2f42e0318f80d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L+YEuGp+pQEK\\\\\\\/wH6Br1bjBKRZIMHIeK\\\\\\\/nsR1AsSF9FwLyMzYTUgHLnMEz+nbwgtONWMbrt0oJErOW99WNB7BjBBx\\\\\\\/xq4Uvqt4JG7MUjLkqvzQO3NgN1d6tAsHOF94m5MBw9l5fR+pJA8hzA2PSXsnDcippaSvDxGsBqhjvVyXJtugtHBLTe21iOvzvN7lSAQGzq6\\\\\\\/nn8q6C1I22\\\\\\\/BANPYgUAEmHad+8f21OXCQqdsY\\\\\\\/+73j2fYT3UnYsHFJ\\\\\\\/vfJSwFtwhLOpryohueZnTwuONA7kauMtyYI0FZjwmeOKwzs5hdK3+9O4UgeK0OnXvYKHvA9ifXa09xfhaKhjagIMUbbx63AtlGOr0hey\\\\\\\/8AJEZ4+RRVAmj5Qps5VCOTCdyoZ\\\\\\\/2P4ZCT0EgAo0i6alYW0y2XV7ch4jn4KWvCNTKv\\\\\\\/bZfWUDK0VwFc8IAU4b9l\\\\\\\/KsUgQtUduZmcZ15HvSQJs9aGl3tcKJl6gSme0yxJPTqqU3mwTYqcUIkbqQFIPm2ljylUkIiiOB+fh8cd66SrxI\\\\\\\/Ow2fguP4teRrH\\\\\\\/ZePtoWthm\\\\\\\/Z6KXHGvjCkXLMOowW11E1Cusa97uDWfz2MfHpF2nmEyPlfYfmRV6zA8PTZyNQ6o19PdmyjzpmnDzqGeEUw5A6yEGodO87xE3JqC\\\\\\\/7rs6MLgUTMP9rczirpAyFLPfeg+ftjq0\\\\\\\/ZdE9C\\\\\\\/o1CBNIgTZc46O1vgIkUYCYQ5Kb\\\\\\\/3Vh5qB4pYR5mzN6JVEd\\\\\\\/\\\\\\\/lOC6hhI4r17EaK762hCXoVKvcWMr2f4mJFXh5Z37RPp1KL9801hc\\\\\\\/ryoicTex258L6wVe6+FwAulAOXE5KWS6+ApG\\\\\\\/LbUtRRkzZjHsvFuqiofzINN1koubcqsvNuXfxmDz6J2yLYrWwAUkqMpCxstTD3LgKeE19CUvKTVi81io60wzz4dsA742JyH59OG0r7OORDjZYwGRmQgUJsdzcjrjdOUmKbzRjoN1OikDRYMz9lpR5vjHevHKyKk\\\\\\\/kFEoEZ1p6qnA07FGmAzJTfNGIRFFHkPU9HGUlfHifZ53fqp6PvPKt0tmt2REh22VPwVoervcomnKOPkvABfHvFkwbWjLygCwSeYIhsT4PtYNvEQGP6MhhadHf52QGUnf8Q8mda3ziY7f2lEEm2UjTvrO3pSt400j7jpwAIPSspcM6EWF3NAZm21e86MtjyEpnIsGKQ\\\\\\\/B4d3BQw6OEdsIxqfR\\\\\\\/uMQGw2dFrF\\\\\\\/vKoTX6NVp+aWUqSENlFOa5KNzXid444Ez6j1FBrLpss8249kShgP76H04adQduekvz6iY+ROahTgqhD1Bfi\\\\\\\/sL5HyQcoUc11e6cV\\\\\\\/Q9UAwV0PQX3nwPrqrXocnQ4LCePfDNX1dzIaIT2EZNutuXhpFrj6ukpdmIwM\\\\\\\/2z09DLVdH+87KS\\\\\\\/X9YICYczecI1D2jhinqjgu6b2U9ZKmecjZWr9Rw\\\\\\\/yBVZ37LBFDhD2G7qkb6mmprfdGfYNWWlqd6Gt\\\\\\\/yvVV5OGDhp543zvy8KIp4WKgxw6z7CsKPUJAdoZQiWOFTBVi6Qppp\\\\\\\/MNqDvmOtdMUNBGT2HxMUuWAaYPPxUtc3B7qZbQ+PPvHy+2Lf0qmbJexxBUl59Lc6J3GDEZzoRwyU3JI6lUXggyu7ZHnkwFof6+kO6QNyeF4EZAOzsAfd\\\\\\\/qhcHXXwjt5LlpAX3JPC9SNgVNaA1YeqvF2YOnuUJGe03rempRnNilTGK4SfpCHA6DjhcGyloXNkMkACJcdRGcGBpuzK+PPa5+MVPODBemLZ4KIBlMvoh94J1vmH1epWr5m+kv19H3Bou3sYztEl+xTNKhOlMQEQ3Pile+OkEdTeTOGAEllb4kv3hJ1dq7sTG6Ib19eJXlcnLrdD8P7zDBjgUD06HPM9\\\\\\\/frN0dgN91\\\\\\\/he82YVZV5wyccZM1uqJT8dDVMghl+j8qyRFuuEdL4MJmtaSQ2ubwmZkLRV0FwQxpy3ijyTmWzdGwlrhj3VbLSQSfChaQ6vzRwGtuRqIyWt2dfG7AiB3Car4JqE07pfsVMxT01arfq1wvSXVCuTynM+NJOl15ByEOmyLsiVWnwl5s93J2acgOC33zVgoV8YQZ4C8tErTAxrY6rFNlshlkDK3LwvbDjmouO4w9F+dWOtgZwqRBcJsVjYAQ8Nn2CqPzBOx136SEWxknorTb7bbRgGghX2CpnNTUZplesJhvC9JGNXbTOoVndMDc7wpVHc0kpmUgeDzd8exewWYC24f\\\\\\\/N9ijxNZiZdfLM9As4D2uxAsfTHNC1SR7pQwsouQ6q18IcrPUx\\\\\\\/9CnlAOBZHqhOUV\\\\\\\/jT5CEXiYwox0vdezJp6YWKCbW6XPZOzG7lmpob6N0ARjvK2O8kPnf5xyuylUIu9kZZKWMWT+0y06k43gHVkAb6dJdOI8IzE1BjY4ot\\\\\\\/t\\\\\\\/CrHqGwB2YkBk5LWP\\\\\\\/PJnI41mrlOYyyoV0ggfen3yAuDgVOZkBP88RVmKwbajg4vV8THQcRlkkyFGoH6z6I3Np3XnyVCjTY1IJfZpKZA1BTtchGWcdltX5qVDLAp6aX+YDpH9WsZqixt2W0Y4OoVJOlqusVlRgIHl5LWGT8Ld0jauSOD43XKqhN+ymxAHnkRLzHIyx2BLfMYT9Y3xyNIU9imnWoLnzPAiLUsdng\\\\\\\/7EYnsXcz27TTUMLW1JHQVL4aHTsfl0UQf3YLnCUQDS1ySRyYwHtYm0TpZJR3j6AQgBnm27BfaOuTrwbl16EKN14J80TLGBgQmowmtNMeEvjL8WZk+0rBSShrxbEVZCLd124i70Yx+w\\\\\\\/azLzPye4ZYh5+rptPdIAgpTWaw\\\\\\\/RBGo\\\\\\\/HoGe7O8wKnguQ82EG+M429F18uqnYISNAZ1A6j1IsOPLTUFcL1mjT2gthZP7oJNeJCt7Dy3PjGmPodrultKAPBVvY7bYrZOfD1uvrDtCVrLeipsbgimwLewa8h4XnOlLFvFvEH1aVJPJTMYNsN8iwE9pGYspljf0ueexaOytGSRMgJtw471PAWA9AwwASacQI8YfFkJZCBSMQwS70brjl2Hb8\\\\\\\/++R93mxxwmKkR\\\\\\\/a3IshBGJgbtsJa\\\\\\\/2RlWc7OulUGRTV+W8E6Bb6AyMaZPper9Izny818Vg+1Si6Zvit\\\\\\\/vR2zA+pbmT7tTpG90uTtIrh\\\\\\\/zqezyEpIyHr1eh6WXNfFW0SnsMVRBa48M16X2Q8MNkk9kz4VwR2N9PpeHSyoRTzVYlKty1KVqccR2bN2w1f4rgwU7J7XcJyKUfRVenf307Hb1uGb4alX3BheMiOdR4L38dUCHAw\\\\\\\/aEqvDKOHUzcdXjXozvHhTbY8TRkBAGMsU1s0M+hdo5xMlagZ5RiCJHlT6k\\\\\\\/CAHF0TJRZr4n+F7ePSHUQw+6cPE4HIRozo5oh6LsQO7G0psFicCNwP+k4ebeh8JWhvUWJ91phwvuYPu56a0bJuZ\\\\\\\/+zqDq72Xu31JOsRi0vsy58\\\\\\\/EsTdqW+QiRQv4c28eJRKfJZ+h+h5btvZMhO2vl710NbT3RiHFe+U+uaNg7t74KTSkHxKnBCEdZckcizC6xCAYl36PM4LL6f9TL7cV43hsEj81EzzarH\\\\\\\/CEqeZFpBO8Ey91S0eJFK5NgWTVBMFi2\\\\\\\/CXq3MrBboXbxpnxwbbLO\\\\\\\/mfVYdgiigrcBed\\\\\\\/1WsxWt3VfzhoSTI7NVv5pTLk52AU8ImA8USVpMnUG\\\\\\\/awTDZ12zv6G3KjuoWI42Hq1XWTtUw1m0HEHBa4gngEOTPLbkZEC6NziJ4LdycJ8mPQNugSCXIGlyyybqSO1cHYR3vievO9HrHXcTOJrEp8ayfhF20HW0XcsWqzanO5ChDICsdTydFsK8Q42pKS7kwIBj4gh8q0evCx\\\\\\\/tUkh2sPFLdeCaDRDXSvlr65bx2tq\\\\\\\/SQR4JBcieuRFZSspzSK66UoaOwJwDK8ckKvNH9vn9F4uhzwElrkI1EEhK1rsrCPARdkb\\\\\\\/pdPglzsfhx4RHftZbBDo1yDd7CVHIVKKlUR4j3baL8StWB22pnkYdhw+q\\\\\\\/gdC47pfP7QLKaqPOhfVrC41guxPrIkgKa4F82AgybxO9NGqxtqxfdZjxacvr5h3R7wSSO4hE3uXyhti94Tx5Z7xXTc8CK7anxZobSSFyYCyHI5QhEj97fieFj1+qnxmW6Cr3GcrVLYZTFoEFyOlXQxDxQWUGz\\\\\\\/Vw9dgnJ7SnIt2p9WwKrzD\\\\\\\/97tCiZb1j0a+xJvay2hZ43eIZiHCS3Y2bpRQc\\\\\\\/2xzbdTw+3I6aWnVejxK0HgY0nPxJ3n0ZOuWN79TZuICKsT35Dtg9CQ7IsTkS66wnUH8bStbDXt0P5WJy8tFu0HXDMNzoLHenrJ9IQRZjNcym4vEJVw0wmLcGbfIceOmhUlDe6VKIKv9MYF7EV3lADsoyjXDqCY5ceBbpPuD0FmNNZteNJe5WIUskknOV+5W\\\\\\\/yselyQxTmhKmEczUIMVamxdejk2KjFrMBJ93enjf4gp+xEVYDVVFRvLr6xPnWOOjT7fRv2JrWysSyuYEHQ1Sa+HXTdirP2svOMnQCcylOaKKDpKdW2kue4bdCYhOBwivWcO+KYYRXA9UmdGQQ8fwTN+o2Iui3sDm6DCjoyqT0Dt5\\\\\\\/1JMvn8sQQe3W5NxqNPt7MOcJP5+roXnaFIlKR2RZuNUSYzqv7G\\\\\\\/gsn0N5cZ4vMf\\\\\\\/EjFdu6g9W7reK6BqZRRW39UaA\\\\\\\/u7NG+UOG6Ebj88tUVUNBV1WD4jRb2wwu5pciXHAMX+Ip8LKbOiPRC1p\\\\\\\/5X70ezRjRqFuhjM9G4lulENAvf9yPX\\\\\\\/Ljo6brdV+Vl9DZYAJG8Vy90gU6aFOgk+2BJ4oxsMCc0G\\\\\\\/e2cZnVbtw1cIc\\\\\\\/nhnBOx0y\\\\\\\/voi3nwdvQYt7F+9FXdbBqrVkBwrFRBbVvTxqHA3taUxd08mbIbbJ12wdWGnHUsuE\\\\\\\/pjUeAB1HahO3vvVfmn5Ys3fZ\\\\\\\/u9WRsXvbuSrQsNp0CjcS8DNa5dWoBFBJKeaqcTx7W9TMH0STMhcC9COHtXbjjwo4vHYYOVWKakcpNANCRdS5v9Nizvtb9Z2GqP2M0cQ+VGWsd07tN1Uac2FcXQkTRnUUEMvZs58x\\\\\\\/y5h6K13x+685uwh6zwx59BNVIjVXh\\\\\\\/i1wRl7dT12NrC8sk4Gdl+0dlj3yDbsOi0SpIwdnYUrDHsidY7kOyXIX5rjg6eXuM0iJckIXDssIx9qNeawE5p+XbbwNv0904INgy5hpG6F7G9KduuFbeNwPlGX7REdaevSKM83VLTBFk4y2MFGFMY69d9krHmlq\\\\\\\/Mk50sQ0wRDoGraEWQZIjDg9WWebGsCYu1yvB9rJpr4hHP\\\\\\\/VKgqphamZYhz\\\\\\\/qOuYx6\\\\\\\/DbvqGAs+aARwwPAxgwtzqJVNFIwV2XRand5Wpb+umPCq6SUGObzPpeAf2CKcK0dqWl5AJLQ8RY2sjBtE6BvKA1PGS1iaNTHAZPOFgAGijAy992a\\\\\\\/97eXyeLIVoMcV9LAj\\\\\\\/BuGNfr56zfc6iTlLK\\\\\\\/9HfMwTpBXBeDsr4pNl3gsdvHiUe+cQRduPvOkoiGgq1HjMzNu9PZ6avBGtoPDo4yGPG0ykpUViRUUfKbbHVmBpklYV6Gw33+jfGh7UP3fHAGhmBCa3U72ddSNaW+4crpKEX2qsmUHs4kagKci1gj5MEYtH3slLChw1LoyPRTVL+xDRnMhJVh8cQOXMEYN25yq0KFlDF7+xf+bXtDRWt65mW9o4b6jXfoihhLHH\\\\\\\/F9s57NgxmYFzaAlhj4LfYc2q6TnLQXMek46uEIbsWHvynR4WNHUqeejXf+8Feiz33m7QTEOXSbWe+nBQL0cnO2w\\\\\\\/XA2vfg+4qgkUJ2goAHSYvylwM0D2rCduGGOCPWrDCmD7lAylJO1TPyo6kFJvaqS6X6Cgbqcde5LMijL8TAuuArqkVm4LAXoo0aXUks1RmUniqxZFWxTx7DP80ydjib2tv3TNOUhMIvjUBqaXopAxNOX26b0bitXdJR0cv7esVHmDf9qAQeZTGdFS938f365UTehij2bbNXMDHwhT\\\\\\\/1sVqKXcJEq9+fwwF6yPdTF2waUTlpxE1zctc8slBJxuxcxFJ5nEnKUfa570aPOvxTXmBsRVOzl\\\\\\\/d5UOAZK0CJ0FbGGLtjV551jjCM5vAQs11nEmgbYi75xCRpMToMuRLW7jOpzrg9FHvLmBQEm52jVZcCmy\\\\\\\/KImqafskJ2jblSHRSCRYqboJB6\\\\\\\/1j+KPUyUVSTAcW+CSjb4x\\\\\\\/JSQXxBC1926CEKRdCRSl4zV0C3MLw3P\\\\\\\/1lYU3Q9RgrGEFnV4BKbHaSsgzpDGLUM8LTvJkZgW+6SuYbsSuvTjFQ19ZgpSR\\\\\\\/BG+8g74Hvf1OIjdDj506YzadPOCIJ19Jhn++cmf3FtlRhsLCF5JYtvDdBxznSkQneF5nmqZQ+MfU3ZX3f59DX2DpWlurZQ7p\\\\\\\/sKacGgQHrg6fMEB5XXIHuCrzxtEMAzGjjOpMnlNYmia96wWIYSYzR2mJW4+18HFcYVmLrPRWYVMeMSyH6UgqJTt6WTw+I9JP9zmk2gMILvZyOnV32gaxqsLPck5dEpMLJZd1GccxAFGq0SFy2SvCKKE63JKHdu8SXEagvbB422wV96NrB4O7msYVKWJa7kwirlz8I4Cf8tp+E4YBxr9JysSkFybrFClLZCOrBVOgnxF6P25LERj\\\\\\\/NcHZMsRxQU1uiHkzHCDm1c\\\\\\\/JLSCh0dQAEOkUUPXIt8715zjKZ5z2SnK0s347PScJ4j9LWntTw0SAKB\\\\\\\/Oi5eBeWfoguPqigYB3cGxzCyZmeI\\\\\\\/kywBltxkdcoTnKdm04BMxMB3u7QiBT1y2eGysPKdqqx\\\\\\\/+xzk5FupUXdCwi+qy2B5yYNsqHbQnWHWuJe0\\\\\\\/x7DAYBsZtppITgmUJBtBErnjQ9nK7udAcav9ABxHhvxFlVkTf\\\\\\\/oz+eOSfsYXqnjYsQeDjnsJmMc\\\\\\\/KMg2L2KrPcMKTM7\\\\\\\/NvYpAxeBCnQ69KqACj+AKC+P6SYS\\\\\\\/yLn7kEAGfGihnwR0Yj4RniLmB3OxmHwqs3C8q8I2IfgciF2Sgf+g1Rg\\\\\\\/MaVLRAonq3DpYBkEb5IzQmrAF2296RUFWepwL5RvMo1Oqh6mU2GnvPwtSflzrKdCSBtXvG3BmmvSbTI6VO2xoj3No2v7t3+0uNvtk+1neJA5ZQ7D0jbdYmR+h1IbdqxgEJ7yNji7+F1\\\\\\\/qsxwu0aLl+zErPjbDYzkabSqA\\\\\\\/Dl0r2fbCJdPG3GQoS4Fb+MKnKq3b\\\\\\\/GNImqrEa5QW8YSdVO7nEKESRMQAQi2VotrGnl2M+C589jo9rglbcQMRsB3tzTQGnb8wYo2hd9Ifxvkph9ktmHPOf+MF82k1YyhtUYJj37w7M+xJaEprLtrS6xJDeZ7maAxaWsBWMpE+pJPXoD2MF5XIC2wp9LpD2GIfktvq+9gEcyIDYlw7OhLzlkHd73imVm3m7xnpQVQ9xtf35D3MupclM7GGHwFkBtyjf3RmR6DAjs08Z4U8dipEh8bvVhIjXwwvRydLWE2SJxy5mpegJGpPhNY\\\\\\\/Pz7hoEYp+zrGT5mVQIeHOZHsTdnwGVm\\\\\\\/loED2FWVhxVwuPy9Kb3u4HWeZfSM\\\\\\\/WDCFV534XxEZ2tNodxa\\\\\\\/8jDNEPp\\\\\\\/LJUCbWNXif2b0hkc58bC0R\\\\\\\/uvIfSDh7+Rr\\\\\\\/U5635+j6Zwz5mbNpacsn0L3iCECi+kklHUL0DPRMe88B32tsC+U\\\\\\\/OuPzggrg1+ksW20JVoDWDBejylZN84qgwSWvZD3zNWMCg+drtoOhYcQdELjdBYXvinpKT29C9ksNGvuETF9BwcwH30SLESvjHbQvb1UpiGINSJ4bLRlGb7LNhdmNvMa7AOLepmvhBA5ut11d\\\\\\\/zJEejBLDSgGX5JJBvzRqoPeDP7ZSgV\\\\\\\/ZrIT2nGH7ryCLt4Uba8UmfGlqG2C14qE2p1hQD5rRkLyNcBB7Yc+h2gl2pMZ+tPHQzWAlIFG5neVbG0IfZ7MKgFJzfLQjXcqxBDj1Dz+Jp1aDE3cPWL5ODfkXRQs9PoD2MfJbafY88npiMyCV\\\\\\\/CTyaHlq9UqV5m1Wx0xui\\\\\\\/WvzA7TS5c0tkbt8Rx8yErqGoJsfz2+G0qjuT2W7u62R1eywPwYN7x8EnyJuZlRJEUlGSWcQzLSaS4h8mj\\\\\\\/8V0+irzEwLMiCKSL9rp\\\\\\\/R6mBuqtgP0c6l80umSnowDpc+6tgvTx2Ly7b\\\\\\\/1f+YT3p9A5CZXD2by4b58SsD7xsVPhCVRhu5Id6Ui459PHydEp5fHXpmwX9xT\\\\\\\/MadEIxAOmL0EU6EyNU9NRL1I4jS81U+h7x\\\\\\\/OPqQx94i6F4LpWMZ6mWzdYgSTPwUP7u2CRwvSfMlOGGvmw\\\\\\\/AcEZNjbNl4qYXFiaLuhpeGDNWPVTG9FPEYzHKVvXjIxYrpy6eJQIfTd79JSiZe783XMIfHCi6YRe3Hrsp+rVk53aP\\\\\\\/pv\\\\\\\/I+Pn3AglI7e94JYT1t\\\\\\\/482I5QVElAsKHKjMiNRYD4pznR13Ys1I806zJ\\\\\\\/XDRLEJbSRw++0\\\\\\\/BD+AStQ6dgHoCKUQDJAx5Gz+fRQRBGgNaDeVuwvEzP25LF6btw3h3vPg0Xqs9QHxHXJTOHC7HxKiBVSYOKnRWb4y2eE5lKxAAqC63vinkuADi\\\\\\\/aL7USh56qh\\\\\\\/HYOlGhSmaY7DENcETcDOm3lBxNdRRyY9AEg5cz9Pwtgvju9Oz6IL+dpl7M6bPcjD9m55qztU1W94TSWmlbAkiMBAE7ClXZFFU0mBiYjwI89SCJu6eebd5d8JgpXG04jd67NJjp2n44xewi+QVDOnpYSwjbzAvQxXMzcuokJEIMTDn6BKAAqRQuVM5SewNvIKtODkCDk9qc2nlvJIVSvXMQ5BynvHXVUV65WOkszjZlXy2GBt1x\\\\\\\/gyBbiqpZfNDLnX4q7KFNb8IEs6ZJOyrdA8CrbgVMVXt\\\\\\\/BiAFHbafYVEE1HUc\\\\\\\/bv1yA5c4Y5reOmyiTd82tCXAJR8ey4zqfda8dAHeNQCjEgncdu4JwOyKENyfeRnKc2YJsFeZ8AMQw9yFp4GU7bQEY\\\\\\\/3g9fKu0Ru+zSVUO4liNUCwgoEbikknCsTUPxDyUnkUdxxfekqgm4T0cWJFWwUn5hEr9EOVCi8jJnjtBR0TuKSTDEY7gk\\\\\\\/X1sJaGKflPvC3UHNv6REDX7DDfPvY9EeYOpBVZKrHMM8rqJKPoYuw2QqLsds0v3J9Gd7hNZLAp\\\\\\\/VR5PXQ+yl9201SuX7OlBJ9pTiTo2\\\\\\\/8BBr247jMSPSBh5Q2zLbbO8gEPm0g3+em8RDCGaM14Mjmt\\\\\\\/Rq+190tsX01P5JTy1JV\\\\\\\/MXBdzMC2uCq02baaUuFRGH29pFZ+3bet+6cak1LvJnJ2q+Yqk3T\\\\\\\/U\\\\\\\/NDWVb0Dkx+0j2djehcZ\\\\\\\/z1oFmZPmVIJvhqK4iqNV8rRHtzq9zw6KgVVb9ME6QBI768S0BlEru4fPz9KJRGVfEFdNDlIpjxjmCx\\\\\\\/Fc7cZySGiOLg4fm9R0JaBvrfAY5upTXSDmF1rRoYb+1l1LrwQ21+dXqh8vLNllLNXMO071BOe1vS+pavmKuJFjZUwjtbZsCBqKynaT2jcRB28nYpv8Ym7S9gJjLA06B51frHmgSw7CvczXKxdNHxVQwDSZvJy8u3mb0OIqaqSX1\\\\\\\/V9CSX0C4TA6QHXvQCp6PwnSji62pSXFHNOWD0UvjRUoHG0XOe\\\\\\\/uuruVHavhvTsjr6INZqykCv1EQh63qzmu38k3sNX6SvnRp5ck5tIsBpdZDikuz1SphDNPvHDwcrt7LMUhLdC1lBXjCIk9SAuDbDHmCiIxY8Yj4taeo4PUgU\\\\\\\/ZgxIiwDer7hFwmacvFy5sSQsGW5SZXE0uuDIwAIyuCqckF2iDpKKISt5uWAgEGRuB8o+K+bL46gb1utut5\\\\\\\/hCZ+GqYZm0rCiE2Posqbf2GN78jVOq5je6Iup35lFaYHi3ikS+J+JMVcs6vCkUCRUJZQBZYEXyO5xAgktRS5elEXkaCv1khXC\\\\\\\/dbjrjEWfK3QRtNGj83uFWhnnCGf86dAi0izgP449vYUq4F+X+RAD7nPOUdYhFcwSGbPAnvRM89gBYyQPasESxT5l\\\\\\\/htahumZ1nwruiJN3CuJo9sA15JvVvCBiwAYCEc\\\\\\\/UPGqEtChtKZjPJajXtczpdH6KshNjK8eosHf3ccgIDHS3koAxKxYAHTjhsyaH3AQF6iq6zFR6RnidMvwqMzdpIj1fPAq3WZP2PCLpypw1fJdPaowyC5bItiYh+AVxWDyRkJIgaNLR7WN0CYtvlOe\\\\\\\/ZzlyQNuqu2Of8dUUfuUfCYUSJhnM2GsORDI08YnJyp641B04Xt0lf0auKnFWjJgmGRc+y0QiUtX6t\\\\\\\/LeuKkyDj8HkXapk9Tl\\\\\\\/oIjIxLl2BMnb8rowNNeBY6McsfM4tAsJTsj4FJCFRfYc34Vq96qIrgmMv1J3dEB21bF46fbjEi559vUDVoyudoecVe2vgOgsR2lbtnb9BDMP2J7R4VgSMLSFO4H6IjouLwWbl9ZFDMUy3PJs53eNDzUt0ayUHE8ndQtmqK6U4veOZkNqEADnDLq91ceh+B+Zb5BR25OX2nvE8xFUGjfTtCz1L5GpKj6U9R9wb0iJpIvi9oW+rXfg3vh5oqx6QO38clo8Qsmizb3toxFY2lZAINLBNsCrtq3llotIi2vTLGq\\\\\\\/8\\\\\\\/YXoUYFm+L0posG0lGDdP12IsjpCQoe8BwdCXSVg2IN8jDUi8MoP3uyYXLkHfIBpewpsoJZSaAPUQIrHi4uDgu7Pk5MF0iHXM2YQYWQzzttaNGTdDS7r0dXSqadFqGvICt0lAfhC0\\\\\\\/EDCkqh1xZl2qjmyU2TThCL5heHXbgkSqL0oa7wuLR7KmkT6qQgRcnsqKlBl7EyJz36H5SN\\\\\\\/dqj+fnxAQzi5HEtKej3BTUBu\\\\\\\/Bt3e\\\\\\\/bLXJdaJaEaY9e6HY0e51ZWFRYOCNaISG2ksit+lXNoLxRYSzlZRFw7aklsjvp2jU982YDmEK0RgNOMJsZXFTi8eXlgCDZEu88TAH7e4Tmbj0TcXQJQB0Wz4w0SF8I1lRX\\\\\\\/zuWZa+41V2\\\\\\\/7+9rfzpDWZTzLvfCWgYufAaDXUFPdiicPk87tcfxDCTEIafnrEG9qxmy8ftNIQdY2S+zwaJ43EsBYspfopu7B8aQzZ8M+BNGNCw0JSXDe9drnYbIUSt34mRTQknX78o1Zj0cKyEnEic+D9N0H33X+iSyfLWHPaKmnIAO8cpCQGuHWrP2GrDN+jloNfBYajukY6lqw\\\\\\\/oBqdkNujRJL1Gn6wWaE9kHIUyjx1yJiaXxFaSgt2uYBH9VauZN8oOftOGuh58ipvH10DpL2hI+SmsDTxeuEuJAk4dygQSXQAlWYZTY1b7y+sWf7qbCpNJubYKq9BmBhMG5HsQBffyATdGxUjylwNC8vwQ4U5eTrL34Zk34V+RvMhMum1Xz+aexzc1JaMq\\\\\\\/5BMKZE\\\\\\\/7OBgd\\\\\\\/Zr+uEvh\\\\\\\/bS7esZEhCgAGXG0RBHxB1WBPNz9EMbJjwtbtkjXqBkWNVSQrLpqrFpK075AX347wCYbgPGUv1Fh\\\\\\\/UF5dqGAJ3bjFoYkm64JKc6nDDHl1yqnCin4L+JRrAHn9cHm1cS2mdVWK+Ns5amvMci05MMO5in0HeFwAukoay29b5+MrqBVLPNkomw6p7ilwzjENMA1rWPYGH03zEZeqGiVynOhbBRM3Oadd\\\\\\\/8ka1dllNzXr+pP7nG02IoZFkVBwUJdXjV3VT8C3JHSaZ0bjG+O5PNFRfNPi\\\\\\\/WTb+IQaJalyy\\\\\\\/+VYZW7LlHb1x96jkeGLoNnQlcggnqSfyzg+CfCksTN4rwGajWeaiLBqM5ZV4iK7yC+T8SoiIuIRNZK\\\\\\\/CGWEx8wskz384Dj5d6GGHjjZ27wUIc9SVM94QZPgZIlfQbstVvr8dHT2i9j46+5fTxraohsihONbIWjWn0J82GRUwCqN+CkIWlFP6vAJF1c+RdII9VyjEk7EcyzGehofE+S9qfF+GzwIOjIjuhHFleMRuE2bZlvC2cSdhlQbU+ztrisyupWw07bpIq4NlfZrL98VLPeqU\\\\\\\/dgmp10Co\\\\\\\/FfSn45O87lqIbMF2o20Khd42\\\\\\\/KZ6rvXPROeXdALwqU\\\\\\\/wX5hi8WktdcEwGlkLtBrpX\\\\\\\/iNH7afCX5UiCK9bMLHlUCU0ujwKGgJnPSSlsmgpGQi\\\\\\\/7WOWWPSanibYwvvL1F\\\\\\\/gHfYdWIUtehf2ThJptRZjD6KaA3Lkwxo\\\\\\\/P+Mt2EvN1O5+bAyFyqAsqifU1rvyGZUOoCxhBcyxixapB9an1NwhJc3GupPxSWEcnEHfxpN8CiGH9Pp9vw7C3utDPPG+t244sWzqECofdDATeXB1sH4c1sPm+Ff5aLemO7ztiCGXheS6Nll1CM5Ktnl88lsoY5wqFDrcsmHIfLPUmLtIaxAZLzO\\\\\\\/trTGoGJqt+ROensbNBuT\\\\\\\/LM0LgF33zglkmAZtdtwHh7H6aXdKIn2kZRmPB6vpFozQQyS93dTniZn94jgA1vsW1+7WQkgd4Ht\\\\\\\/ZV9fQbt9F0LPt6IKHGW7YhpZL6XO2KGgUfM6+VXk+jukM6tPo6YJRAit6L7RXbGmMVJOsg2hgeSU+J3eTpWgILZG2nKxMUjuhJ4rorummZqHZsixPyBwFp3YZd6qHrpOVGfI5\\\\\\\/KJc89DUdj2NsLKL\\\\\\\/f3qG8KhhfjtCdqi\\\\\\\/iA3aKzVTuUPo0y9ivIUvt6BxkHnbptr2eTVmY2hocygUkJ+zaA0sO0QOQ9DM6AahpBMsrq7s4ISd1F6PTHCkm9nSk0pIL2eCgO1kb8KRE+vgz4QD+wYS+2jNKGk3hIEceqcXLmt54FOaWxd8biwI+wDEY2y8haTEHW\\\\\\\/LKHmWHd5obY8sh\\\\\\\/EMckSmvr\\\\\\\/zK\\\\\\\/Vmv8CFpxY\\\\\\\/zgA7fgJrbmid1YLBAwI1DmSUbvE2KpLFp47iLFYQKAR+o1sUoiGccFVvcFMWgEtZTRxgVmIgD5Xh+EpeEzzDC\\\\\\\/vhFas7CLsJzq4ieXBoOMjplnHrX0bwMSY5h1o7RrXcpvlB9i6qpOdVzWc\\\\\\\/9BDsbnKpAwnnvSyIRvjxEYW7s4S4k2u8EXNZGZ7x+Ji3bAYI373gByYh0IUQljlVETjsaSLUYY63zob8QCG+Tj8ipH63As3NVSaAOcOpLB1eyx0h4wLhZBVpZj8myVnkmiPVpbdNUx2Gxl3C4Y3yaopDcM6U4wvdGTN57tmLowV1rjFJfirqAKAi2I7EHU9TqLTeXtP+ZAGTvlet\\\\\\\/qcvRBIPLV0iE+dI5XKQ54eTqmUIidgyNni9YL8u9VfvzfSgB0nahmM8JGhv5Ymeljcr80qndMm8635h3tLVsWWkgbu1bm0SrTSZV3KDUnhEYpstpwLv\\\\\\\/ts1b7kjKJkDu5do7PvtmPRE82CHgJuUSWniZWEUHXyxapYQf0yV\\\\\\\/Qj\\\\\\\/QOzX+svwE1FgI46B26DHTI6dGEL10QHOBFoEbOJy4qOkVv58F5q6lOsqCzfXRRYC9n2XGf9YqhLg2v16j3CuAS7l5c311SYBqQuIU+8xO2f8HewN37VGpORV9UzE9Q84Xzq9PXuIbQzAVot0DU4QoCivVjR2+YAR7sM+CmBUKmA3WmxlfHsRdFBJd1hA2zpS7NPcu8X2EGPhD8+z4psW3HMyCARwnyUEEDYlCroNK2\\\\\\\/MFBiQm46l5XuX2v9CpTAAC5Xy8sdW5SmqLl4ndx\\\\\\\/uAnNwc11u\\\\\\\/oL2sKIdmlPwrn6gikcEwJ\\\\\\\/OsPDUlA3fzeh8AN8hP8VW83U49exrMvq6uU0mc2EtyJ5m1mAbAFYShzSVlXOLBa\\\\\\\/KhR3E252vawA4RG4Kcbu+IrGs38xNQWaTqLtJvQcEz+3U0xVuibx0x4O78ZDR7jKgaSe2kujZhg4ARrdQlC0WIRwy5\\\\\\\/WgwEYNVz3OMFJ1ENBF9h\\\\\\\/R7YFAd\\\\\\\/AngxCj3rtUbsD1eRijhsicl\\\\\\\/3U7P\\\\\\\/jdR6dJEXIk8u\\\\\\\/fxsn7WTpanhgFwts4+awmys8eNUA51qQkwrvFHkyTnBsNfjE+\\\\\\\/kqWjiWvGDOcWYr8TUrQUz+q5BL9KhZAA+hXQAUqtCakhtPX0NqYjnCM7g6YpzFG2GvTSX\\\\\\\/Yxu+47iG0PMT215YreuyhvMTx\\\\\\\/IiuAsg5w6MkBf7GhEmD78+2JmOR0aPVc7Pw6MGmjQzlRvP3XsoQ4f\\\\\\\/kMm3F0iEg7H0Ufj9ijpMxcJSgsgh2rk71Etg8dMrFNy19E\\\\\\\/A11isKq\\\\\\\/lO4JUOvcmnp7esBHjOWqenSmKDEb8MTAX\\\\\\\/r+J9OTHRA7p\\\\\\\/SY1EswP+kiXNdzsJohtyDUy\\\\\\\/7xlyIod2aSAQ3C1+yDIUCOD23bYf4kypNhgSswPgxskFP1bt70vV94\\\\\\\/RdJZQLLQNn7jfyi03V46BBH90WOEmjKkgnKwZiHRwDL\\\\\\\/p3svw4oYVhIKOCmSP+TJCdONECiqrm9px\\\\\\\/lvS+i98Lv441FtyO8u3sd9Oh21mlcXg2faXJlhxf3Ok2+xETi4xCRlySZOg42mbmR6jadGj6sDqsrMxfOPO3SGNHIjHuhl\\\\\\\/b92wdoMEWB7XBWHr8wwSYfnfqJ6sMUVWWyar6y+tBFUjw2nr1wpZEseQYIg3DODn8XW2nL9XnQfJuuXyYHU4we6k8KtATMUjxfFNfvEGMKASqXZRjTYj4m+euoDNheSKzrgXPf2nxQdDr\\\\\\\/gx2fsJ6efgKwWUDI+yDmE4EQWYQf3cD54dYBH7d1nLDG9UljpcVQHXrPvZR0jQZnJcFYGoXsIbnudsXWcTMcLKgDGZpvf7\\\\\\\/Bqc8lt9rAXoXqtZlFyhg6CP1j+uNRV03VUyWd5aE8kypiNrDHA8UpXMO0Ib7C18t7vz+tceyPnJ2XhpJN47pyNLMd5OhaEL2T\\\\\\\/CEUQGf\\\\\\\/SogXNFzhqAqviDqI+rZS1UmQi9sI9bqPXJe8YPDZGqbt1jI42xSCGZyEJ3n9lE38ESkO753kxJ7lhNdfULrn879FMwDOz5+KHR5edrNkQbY1Tprv59GSAmkYbfmBhPCIxwr1pV3Minx95ssi\\\\\\\/166paefD24yK0KPAQK+dsuceLU56ctfCwI82BB\\\\\\\/SvoH9RMeBEQ2ZVdH3t4ktEUXgfb8sFs6bYqAlqc7UMGVK+dyaDcUL0IcP0BcB9d5qYQE\\\\\\\/g9jEdYuFSKW9bj0dChHsM1PJkV26CAN1\\\\\\\/VBlZpo+GhiX1JWFk81ZJX4y0cOnV2U5PwVZ6Pwg9lRmLeAfnDiWoRQWLyywHyqph0Ojpy5zbrOKOj4loxaQklisCXm1M9XIGp3rUXHy4ETTxPCGzlcfBYd6daWOkzGIVC+cBc5Zl3NfERAedoHkl5vgeg0rwc12EWiPWRDs3ovyUB\\\\\\\/wUTVThp7IllbQ9D2oWIZQN\\\\\\\/z\\\\\\\/HS3XqcFsOESfD5Jisib62\\\\\\\/Ve\\\\\\\/IZb4msys2qPm6h0CoJ4asSXEGHw7KNUzbasxDm0rD2hRVH\\\\\\\/6+0hRnNB0EmZReJEb4mdVwMGOdKq84tWUY6BlQDr5Kr\\\\\\\/R9a8oihZsl15u0Uflfw+5F+QCEkZkQlbKsOnKFR6GBn48bt\\\\\\\/gAx6o0ZEznr1sNSoCA5jgpMcVQRHR9lUrRfD+df0+3e4D79ZkoZ05rmMU13w2NYpqy1H+h0vsTqivz8OOktEjQj7q6ZTVK1Q8hsKSkplOwFdOejFLYWxoc5yXfAtHFPQPWL1BmQHGx8NMi0swDDLUbLgX0sQNomGYDjw9sGn+B5tWhIIUw1mewpoeMEWpw1JUvnXaD8OQuHgZJfze9FJ0MdZ8CJ7L4eGtRcIITFwK2U3MHSdP6dUIrPJU6LrOgbJcdOSEEAK3heaTFAAAeGpy2svGVZl\\\\\\\/LJumHpoKeif2hVK3s9dMVf3wrueWMnenhzR9CLTxfAqbafqYVqmLop3Ppm\\\\\\\/jNeLBSBh3T6gH3WmE0pFo1qm0kzGCaSV1QdnNW0ZWUuEqu9yt8tto93Ee5LIv62vdP\\\\\\\/4+83ZijOPIQOHHWFF6pcGC9Z7M1EaMiR14c3ZPmCAHXyx7VUCAxpkKDPMuAwgYROb7QC5DAcxmt1thfetsPkWcMZ\\\\\\\/JbZEJhC8ohpJH8A8Wp7fG2zSEaQFcvrpY7X\\\\\\\/o5igdBj6685+pn+mQGqA52ojvdjc3vvZO4fZwYcOSqiH3Nv2otHzRP7iHnC9sEqbUPkGb1VnJBUOUPuhMgBTi\\\\\\\/bac1YqKOj3tyFIuw6gvsHlFX3JxaE9flhCTpPDQoynQDzYugeiK+2KQPe6rh95FD+ZIMLPSGk8RMV3biZvTkzf5Lckl7U\\\\\\\/P\\\\\\\/wNwM2XkFgR2O0khDHkCtZ5vOfauZIehazBhrA0F2FGOlrPA44YLR7Py49Nz8inBcOV969HZDfULZpHmJfAyEzwxNjlAenDWWtOR9As6p8F9vCvxuvRRL+rTkHeL\\\\\\\/S\\\\\\\/mBajek+ZmQ049tArLRAcaKHQYJNd1g3wDdideq58ba4E5bMZTLHe6PFCccmIghEpYmwfk1\\\\\\\/ocGr7ufipugHGNzQR+hUuHrE2MwnFbHhxesfx2X0OBS\\\\\\\/ePFtjsdppJUQdHlejp+4YJxczMk1l9UXRYbp7WhiXvOGHOPZ+lXzL0btZakfHKD5RzexYtBJEFS0ptE8Oc6sW6pwvLa3MojB6CCyk8k6eF\\\\\\\/mxj\\\\\\\/g9fSQK3GIn1Nuk\\\\\\\/5inderIQEqFxYctDlxMAWtNppBDT1xrR13MJJBIJ+MGC2aPHejh\\\\\\\/SQ9cYl\\\\\\\/wVSTXq+zXLGa5774M7g91xyD7nqR1yOXchZgjzH3GfqF4k19gCLBgzFfh7vr0uBI2tRbgZdJ4v7ou64x+DcKNf7QjJCK+BXQ2uv5wooByg6gjxoPzEF6ZiRXipoVA+levOoZhrpwF5Tz4sylISftgrgjND\\\\\\\/cfPXNPd68xal9eHLEzjjwwUVcin1aVZPz8xH7P\\\\\\\/7h8UVpRFIN6Lx1B4whdocZ10Rx\\\\\\\/PlzrGRdHQl+eTnxAL24ZQJgr9xIiBr+NF8XHRpuUxmxovjWcuReGNBK726VNoGw5IcEqsgMAmliRi2NGU3cXsBT0PT1bsqgYxt0bbXfCni27\\\\\\\/5klxEEsizrZv3ZzNT2Hu7e7LkzXCh4\\\\\\\/SLIIIeWi5CotYTdU9FI32ee26wAAcvRY8bF9q4hzDL+Uc7zEsPpX5iudNNNnHTvG6HXMGU368MO8uZIEmdh9ctR0\\\\\\\/mFVadrb5X5lRfYKvTXVapbazjVV8VWxpP7rFCKw0p1kIoiXaEC5+XCESjWHNTHwRlPEh3AvO3+8Cyf6tf9U0NedeevGQjLENsSfyIfwNMHo839RZN1xuwj0hX+8oR3KkOn9mmWwq4jSm07E2PsRtjTbNIwFSu2nOLoib1vj5E9E1uOSmDjCMA8rPqEi373GShEoyKeXWQUqA3L3wtTe35WJsN8+\\\\\\\/qtgRfGk6u6o0byQB6vfaDWeArHUkh7VWnn1oJMcKau0L1a0lisAFEPgvE0emw16iq3DMhy7MO18lrrWlSjUDt5lUclalX1OoRJHBMfT9K0plMIZOmz\\\\\\\/mujPjsCbNV4TGp+iYTMrwJu+Y0hGjttqrTiypK4WA7+yNCedUcr+mdU8+XVaxSFVKrzXJRM69DF3zpL4n+GXcEPuIpGkM+b8iWjyfcJu8Gl52NQzai65NCuyFx+V6lf23YYm5SZ7S0IvZ\\\\\\\/\\\\\\\/p7VCkRCo+EgbHysCHRbw+bZCnQyDB7WoZShmTv1rFlreVcK7rdtttNkmhLnHq6ENePwg+8+d609aGANLrCJdOTRRR1+5XIVlS1FSjXKzA423N2OsIjlFq2y8qyDILA03EUjgqmOFzQlWp7E+oN2EhvMpFhOEDTv1AiIaZ60cBO3Jatu9ZcbPW6xWV7nRGA9SbzjDviRvO5m0WM3BBS2r3gRCso4pN6la82zL\\\\\\\/unGh0BJnHFfClw6hsiDZ8QcSO7jlWtHFE3q8HEzUHV71ThDHPexZ\\\\\\\/rXZIzp4wLSehrN+pTl+VIv+tCSXvdebt8im6LS6su5EwmFP1vDPKeB8UUwEGpqSb3bvnQ6viV8ySeCmYPE\\\\\\\/5d803xQOskd4\\\\\\\/EwSGTfnrydSn3pSwfWuzIB7\\\\\\\/hivrQshfueLplq9J5H8HLCauXEKuTuzNgbEbKKBcM0r0On1x975oMhKV1SzRhcIO5OVoXV03qq9L64mF3xiDZPPmbYv3de0SFGwAUirf5Yi8hQNp0iTPBm+xRX6tmXtczhhZOAx6yVnE2xXYOlsqc5f06OX4mYEobBeFwRluY06k1rc+lHdcbucQ0Sv6rzX6VYElsj1U4ITRRONALaAbqkUihaX241bpM1bT0n+Ah996BNoitXyy6fJI3685VHwYx6IRHJg+4hc5IMciKfG27iRsLFF19Zt6IcNHnkuBvz3GBw7Dtz\\\\\\\/oxWbsoYakEBzCChOfKYfZAY0SBtBHYYb47PyQ+vgGXikk50y+ot63776mWxfLaPYLA5iP73XYIXdJvaNJEW3NH\\\\\\\/AfAtOqB75br\\\\\\\/m8HriRlsoJI52\\\\\\\/DGsKa1UEFHHC858DvjRHPNztt\\\\\\\/mBzerDI9Qt4GW5XjqcDLy4jjap54W9JNPp9mjU+EnvfVOISEdrndFLSp2lEZR9WubQB1tWlo+CC4kZSAbbDKl6G0lDrJ40iuR+pi+CKDhmmK+JU+VPgfnGBkTzfDygiIAlAzakyOYDZ9cpk7ynalCUpa6GNxd4KjR0MQqyLJOvL3NWtnzMQ8Wk7ArkO7ywy+f5d1ZNN6hbqSIzJkCawr4LbEJKn03\\\\\\\/\\\\\\\/rGokuttLshky6lJJWC6x7dDtLjT6HifTNomZHgEH3NpTVteSXH+dh4cSvDG8cvylpsuZJDo9zOiRMhH8IGOrok3TYuC9nLLf7O2ncm8P7ET9qcp28dA7V8Vfk8x15iRbnHErV56g8ir9MtSsBd7a6M\\\\\\\/In9bWmojtwagTXkBz1zYe7tainRnFsXDL6uG9oEAmxUsov\\\\\\\/dF8NwMgcbxpvSx5UDukRt+tHq6RDP0sIbDiOyAnEJYc7XOuCa2dSomU+6w2Bejjbhf\\\\\\\/V0TmfmIHQh0fomSpBlwRrnDFB8WUegFKZzK0ox\\\\\\\/qQp12GoKSv1siLiE2fy+pKDzrJzCgXHSssJ2esqzGID85hTUQPKwrqXfkdNs\\\\\\\/9x39swOfHY3bjEt36uJobh37grdRDfkSvJDxy63KJTNhWpv17LvtKtiqZsfkHq5ZCTHQxaw5sAepSST1GCQuSa+wx1Jt0q0dY7ZVU1r4bJDh99vNB\\\\\\\/PQYzvvcFLsK6GsP0Siwu9j9iQ5aF13faUL8bJK8WzvRn\\\\\\\/B2KqzG8O+Dwj2VmyNLy5+Gp0+vLI2ZdqQAydc7vFu+YuPIIM8g8rCKE57L1gzDlJr1OaYamnf1Fvrt0nQsMkm6EBRhLgDDMAYTw\\\\\\\/bg9lW0QWP\\\\\\\/Fkuy5yJfFbqY4KuVRzqocZpmGF+eRaGBpQJT4z9BgLpzNVQQHtcALjdu2pARKbOYLV0zXIqsRoujqDZ8gMel3w7iBvGtxKyHSr79lC5TMMwLWLC2eElv5jMXh6vcmGwvFTzfFHk910oDSbxiNTxsbRJu2EQsukVg67Y984mbIXBrXCsXrrUhPM2l2Lk\\\\\\\/l71BoQPXLk8Is16L41j9k80WoIAhV+3I1HbvQhIUOga1KGyTu\\\\\\\/9wJZUe66GVZ0eRLhkAdE+E\\\\\\\/livO9xCCEBaSXOU0pdExvF90r2fsA3iIOEu6opmlZRAWXKk\\\\\\\/O2+XJu5WkRUnZIGSnKPwZBnzTf9MKObaJc6BAh3Do+mubUbdZ2SY1tZoCdkw8anv7DWI71CPM15QmQM6lc58iXWppn1XtSjiZv9eniPikGdSBsAlglbhs\\\\\\\/0FuV5DI50ZhrxvsIOFbtyjaVWckO21Dp8\\\\\\\/z4Y6I7h3cN6JZbMaDHTcVUvSiz+G++VSG5GQWEEnVBja1mAYW6kWfpP8RWBPXXHUPumDU\\\\\\\/NBZkD0J8FlAzFe\\\\\\\/om2u5sQg04zAhrURXClMW2JP1jSwN6JJwyiydiYZTWFSRyvVldO5c9TM7UadPTlZi0g8p9pdZVjhXgLaBJpmTWtE14iLsXLO6bHREMDlxN+0bOr8YnrVcJ0h1x82MYWA76jQBv6ZaRpR5+pOZS4MdSpahiSK6oe1R61oDWlUUiQMD6vPg\\\\\\\/nx5LPAIo8kYibh34afCwfRXLCDfOlMTUm76a\\\\\\\/sK42O3ISpbNEwvNuR47YzB3u4MQmdRbRnecgU+NXI4rK4Rf5QC2YXYhpZcCAV2faHGu7HMCb4skuBnBb7qWmTOwOAnhtbunIrr\\\\\\\/9lwXZpteoixp2bq\\\\\\\/1iy5H6SndgVKKUgRDnME9lNTpJIpDGTDX\\\\\\\/I6E3w4Qc++juNPisC\\\\\\\/FyJ72SlUaRlmXlnXcvJIhjjcnCYdNsl5rq6r+pMJFDcI++VTEXQVm5Owijam2H9Wx\\\\\\\/dqo59Gk4XhaPDu4K1VzguqhhtnN+PSmpOksd8KOMDZJZLqLXWJd7aLfWx+3c0dE2Ak8HuvIW\\\\\\\/tEKwxLr90nAE3M+fmCKfXmPlmBrZnltfSyPv25l+FSsl5ZllIolpcYPPXZQrxetduYjAnD89C6A7j92IXDXPRZDU0IQbFE3AOAgyVGvKQBSUv5Fk\\\\\\\/dLCnl4f3ZwFBC5vHwC7OpOJ+Q2nLlPks3NmwvLCOgDHOZRE+tiETfhjzAt0QhfSY0RRkAsbIMb9+pvAqJ9KKD9L7CIJvb\\\\\\\/7qDVGUiVTE2Fo8YwXbgCZo3VdJocFK7ypzvUcWAiWu2bOmejqCwJPtvG2nYiXGO5IdlK1Qlc2xGBEcaXaBa0Cl61TzOkDdG\\\\\\\/pxHm70xjtn1ZG4Zexe8+wrMF\\\\\\\/4AtJ2bV01mL\\\\\\\/Y3a+rBEC\\\\\\\/76P83f99+Wl2igxpuCUJQXh2trQBIxc5F3hhLHzGXgEnlCyV+7mn5gPiEGR\\\\\\\/ubo+FGKG4dIgL24olYMCpIrFmVjXN4FLoadHjEhPPFEGWk8PeAX\\\\\\\/ISdyIsZiaAYU3OREYfPV8nhhuvP0fx8lsvWnaqacz\\\\\\\/GebFu4Mig\\\\\\\/jU9w\\\\\\\/lYCprEEK6dhd0vIfuU0sg9i\\\\\\\/8a9nGYSDQzxBtNi0zeje1q1DtQSqiejKkiGWNARbpiodEifkZrEJJSy4Fxhjjqokl+KcuDMDh8FbjZctJS\\\\\\\/fEMf7+fPUWjIzkOeYGC85pduFh+5G8eNM7Da1FXKukGBAw18zPzbJ6mKUdxxceVSMkGTZFGmAxAskpw9ZA4HyOlgsd\\\\\\\/iDFDxMeND7tIxtSfIg0O6BKIf9TFtE8mpLY83nBSEJ0JIsvcTLAcRWDBvt9d1qrhBhPHpmM4\\\\\\\/BpIy8r\\\\\\\/LsKo5UeyBVX4h9bKe9r5w1zi8fIOv\\\\\\\/T09X7jqI43SSN1BgOFPZTWS5Io6JsA2bi9Y9g86\\\\\\\/ZYjRc1GhadjuKPujkUgGHXCt+yqXsqFqg5VtvQNgn+Y3Ukijl2z88rpmdTb6FatNOHU9YrFPCvk43lwr6LLUVqjbtIq8FW9TLMEzoSkYzT5fH9ehlzIZSmaTopfP9xkXY+cDFHQHDD8EIoXM3r42qFRXXQnIzwr6RYQQKdrcH1nqzNmr+KZGJaKc4aiCwmtm1acCMhMwvJ1C5c2SHz228PqMLTh+kVzR3G0qMMQWNq8Qx9VQ55adBja4hKYa\\\\\\\/RNmrJ2gvudQdt5bn+9qIkajqrXNLsJbR0NiwyO6qTRxSjdcLaeYe7+iEjc9rOANZlHOS1O\\\\\\\/TCgkN5DBqadvo+VoKYfhzEJ1\\\\\\\/D80JgusN8z9YCdB6y5gTXNdANbg0CvHSzGEtlMVm5WQLB9rDvygIltxCgAU6o3s+jHW\\\\\\\/eIhDPzmty\\\\\\\/4RgTVNnZ6EKE1gK011BRh5U4ufDq2RlpcMvi+lmsbjtNGZF2gWv+ad\\\\\\\/MNAwgKNcvUfgWvoU3+5xlfTFWepxzsTSH9K4LvREZQkoJnIacIG2wFnFzUos0mQG0zasGW6T9lwYkpJVCu117tQQxfQild5IbSmmQsr1BPxl67FFlcgTP\\\\\\\/bOk3oOQMofoWNDIW8axHSNbKEWbbi5miEXlwxe7uUmSv6z0tq2Npq8t6KhNARTGPEA0R5zuwMh\\\\\\\/RvEnBgVtMTDAGxAxLNMWpVmvcGubfHKCA3xCl1HnFSwqI4T2d+CtPm5YDjJPOIKQmlcGunpmIxXetrw890tbFpYE0qvTBvwlFedVcmW2FuDH2Bh5+KclSto+3cKvem37jY3sxhqfpZ6qz42OTe+GuO7l85m3RoCKtCLhmSDv\\\\\\\/8+lRgT\\\\\\\/PaPrx1qwB2WDc5kDAyTWSkGtKZ\\\\\\\/Wi3Q+bKcRdYr7eVJ3dKsdlpERVOw\\\\\\\/n\\\\\\\/+S23E45f8XHzVP8NkRlLSCKNW6ANJ4Bbwg8RyV0WchSUH6xAbnQ2NyYjbIRK1SBFPQnuFdMkydOMtoH\\\\\\\/wPxGxJL+W1WFFeR9Zf6X1x\\\\\\\/BEkB0qHu0HK1QbZFCfJteomF9eC1F8I1O5l66TcTPVf1dWC1G89ynwyS+DmsfMMcOFz+9n6+xy1bzpnNiXcW8mdAH6LoLaAa0o87n5OSZtVQleGU4N\\\\\\\/ffoshMPErw+2qwbPUDSqEvRwzwJprkFKbDLKz2y9KF+BBrjEM+nowrmkW5mihAQu+I42\\\\\\\/Ud8PElDTwMnCYPpGUl\\\\\\\/HK7RwU\\\\\\\/YUU6ug5aeuqVTiXMxnRrltwclhBfwP+2HtPzaaui3ywH3rV+sziSRPc9RDVAf0zbzxycHlHAqcHPHm0WQzW+034835ouYsD9v9lI4uWnVFZM8+Dt8X1pM+bCJGcuEfUMl0j0Vm9R2NeIorId4g58cyls4sQeyN3g2CmfrTO1iC5+NZGH0XMRqsZ2d0aJlaR7PalgleC5elD9IqbLuf7sPanYYA6eBdt4YnHO9bVqvjPNIUzf+Xh1s5hPhVe33c3FStQ9YmQEFgXYfS\\\\\\\/wDJOw8ZzkFVQUH+mlqSWer597hUWO331+ubO2nXgkPRkXwlxngdNXHukgOl6ym0uW+st+s9BY7cVNz7JwQ4qh5WMDBx724t9olUxDXnGiWnDzY+z+hkHYx8qeoHu9KP78zpBEfyFTRE1\\\\\\\/tr7\\\\\\\/ludplTv3ngC\\\\\\\/5VjISBBDVBnVkunQAYWJFdtuJq1CL4LrCUGQS6btwETGaio8KFIlj0AJgJF+BP+3Bxt28ZY28YlD65sWBYQ0blwjaqAdXZPpEfyXZfBPEUcCNGoQU\\\\\\\/Yv1TGxecjoUucrcWtlUcHqGqY7aIpAvkoY2vMHdM\\\\\\\/RHVEJqYFjhu36FIH8XUVvKDFUJdkIuLTGlxSqHNUVnPHEE5uLAeynmQmK\\\\\\\/8BhrN2KHeJxBk5a0I2fvvHTlCkX4lYmeKyqL0voGduUkCsmlWAroQM42utfNAPNHjJPKlZX3FhYozSwLXtEo0GY5Miq+SAoi+BcFyEicSq+\\\\\\\/hgH\\\\\\\/gP2DK6VICyAk30\\\\\\\/VjPo\\\\\\\/sinFfMnOjMOHbKygqQTNO3L7uxWqwG902q24AXQ\\\\\\\/Z\\\\\\\/Az8c\\\\\\\/BORYysqi4SQM9yVwtL9ujuB3z6UcyLSJ6AHIln6wIMpO7VVeKYYrSlFGn5YrAbSBhjtB\\\\\\\/J57eL5xN6tLVak6kMBu7vq\\\\\\\/+D1p8AtkOxUzEHaVD2YHLjimcpCG9Mc6W1bjiuNpOa9NDSRA5\\\\\\\/BTNbW9mgA9ddh8EfdevqDjhw+jK\\\\\\\/BoYexlPbufgfAck315M7QcPGV9pzWDZCVVUrZHSdZuaoY3lgC58VZeDSu+Ke9P8E8WgGHiMpdUB1B1AOCwVH88fkorxER0oLQOtcw7xXssaLP6BdkgGTVAKjUZOy0KiKUIWmNPyBT1h\\\\\\\/PxlpFvoSjNQZ5cQ9cT7MWvUMCTvHGaHfjQeg+GAv5xf1GBO68+4f3jIMvpFB7NfwsJSL2Pd8GMx4QCfwFr8xweMoQfBJAvJHaIu7SLw0q4\\\\\\\/Bgk2Pf8tBrzlYlKI9bh5JbeFBDzy5VxbcJ7bAfIIifXy4j5AguQlsp4qiCaJAICh1m3i8B4qDN\\\\\\\/BONVLkWDg2JxceZ8UWJFAYh1NZWG2AXPkP4NOcWv9bDom6OdfaNmbKhfubT\\\\\\\/vcoQ50Sa3nlVXTQGj9chfRNtHLTDMfe3DfsNOTxndIYmqB3jDICZ9\\\\\\\/Sx4LhIq4cxZA+eOWDAUB1kncS9m6yFp0XYjftEF5bAd\\\\\\\/6Ae9SN5V1K+ib9+3a0ih44RQhVeGanDBnyHcxIHEpf58UzpAEU4KY9PbguoVKklslDYwBva\\\\\\\/j+MzqYtQ5iqu3UtFNvmKHzi87YpsVccJVAAjqatcGN1V9gIk87IJp9Ax8QyTo0bQ61Y\\\\\\\/3HsVvqa5xGdcYjVi8CgsCWZxU2j5thZmm9bq9QaLYt3Ylk5TAGGArusf3oAo0PWuch6YKGs+WuUiHxhU5q28l4WsCz9mhvrVYMh5O73GezMQe5IeYCE2ARAmJ1jOIsRGHyxwSDWIV2gb+2DTfJh9FWHLHIwy7H551qAUIyb\\\\\\\/cW2CUb1OEa5g24LTwXgh59aFDewBsA2tbqxEJ4CkoB6Yqf8Vhd9VVWoV0bvYsU2dSL6WBxKgZEskD8Mgd3p0V+lG13ZgL4LeSQJkPw8frLyYLeBgCLCs+87LFW\\\\\\\/YBf7p7jZ2w9Ehq\\\\\\\/MDQkYb34uTqwuSYPs+2dKjvl7PIaYBBU4UrzdxTlJfzwZVWrooSR\\\\\\\/u\\\\\\\/d\\\\\\\/tM84yMMDn8n4gqMHkCsxUoQxa25HVVn1K+Z7mKmhOoBAg0UBfG\\\\\\\/UhiBlDqAYBvWCOXECa+WBKt\\\\\\\/dgbbBFnNJaduR2iUfGK5DF7Sqdu36WxHW+KIb9RG\\\\\\\/yqUs9wbxq3zNx0W7sCnUiCuG5phvarcamMnnyJQu2k29tj\\\\\\\/oPqIj+eHlIzcGPmAd2Tuxk+11ZoEVJjRrxcmhlrTW1GfgRM8oAuUS1cTfijjU+eVmFCwRAWgVqEBf6ht94INTJsO3PMN17KMPWSZiMMAFAqoTBEj1BWB16J3\\\\\\\/Jo9xert\\\\\\\/+SRKqj9PGseFOu2lI0I\\\\\\\/ZHCUncGapQJLi1gwjItm8pn54r6nLYAJVYlcptruIgCURdI8tM0Kk1f9NOxCU\\\\\\\/3XnNnkg3rtwCYljcci7GnD3A3Kn\\\\\\\/8uFXXatNwuvPtP4W8qd19jdM\\\\\\\/ziUM4sltbNvntzSTJNlOeVznq2lC\\\\\\\/8rDV\\\\\\\/Wa6NSomXVxfyUET5eFRMRRSvtrbkU2feMvdybOVSjdVCdCkL1qIThv6vgJmWbeYl6UCw5tIXFfxFgBJjCjmujwELv2QV\\\\\\\/YhSStqXQRFTKTQ2Gq2m\\\\\\\/qNFnasYD\\\\\\\/6Ufib1Y4v4DekGcsjVwi9VjNdVIl5AUMC03Qpnzjnssgqi+1jYE31BuQYm\\\\\\\/M10pGl8SA\\\\\\\/P0KBnIYrJmv+QMV6ahq7HkZWPIh9fmMRgYLT7C+y63bu2O7UAV0mpmRX8GM3PluwFM3R00t2QZ6BvPTvz43sVaeZOcN44Pd+QLGSO5kE8idaSsJLc67VZr1ogkasdF6EcaLJvWIyiLAOQCsLH7QLzYCvyJuYBFGGAjFPtoCglQy+Xl4aQ3rkO0zAreQ3sndz0+WgnFFOM\\\\\\\/ll6YUJCK8lyaO5N0MSZLYPO9L4wYeFQjl8QQao74Q\\\\\\\/gbMCo\\\\\\\/0t1rPB4HAROW1vTHM6bxg\\\\\\\/3ikQsDFXgBaImd0C92EBZ6Hqssqqz72zsG3jeYCjMEqUlk7xHSCJpXqmBYXzHIiyYReS46hxI\\\\\\\/qenMTABQEJUNHmxejnYRtc7aG+0pgcYaOuGhQYcNo3Q\\\\\\\/rOz9+m8nu7ZuiXyynk2c8tIzCc1Kxdodjb60LaWZZvX1Lb7\\\\\\\/PVNpomWCdZoTfye94caspuGybL5Ku+C6bVmyJhGOIfDzP+RgN5MQDIUFyMOgBpt6move4Z0pCcg2vIQQXDMj\\\\\\\/\\\\\\\/BAlmYK2NceIm+0xo3EmEY3MMYjuRt8kuCm8iFH6DtYEu0ImvnkYYNyvcnhsYpUg7M0ipuowZzD5G0PcoPw1XBf37QNa1HPcP5Rn\\\\\\\/tur6qmgCXoKYV0iAtlcvGz5pq50sNTtUTex2idjUwIRnwlhV9eh\\\\\\\/0XqSfkcA+v26zQDvU16SpNiPfWVB+UgXVBl5XugaJbrQcr600TgXWJKfgYmqXSDPU81QH6RztNCXznKv\\\\\\\/k2qJlq3AZoqPF13DkHDmxxeK4BSE5ctDcYt0FBoNSf4vTEiawJ14TGQDuj1YCznJdAHeJbQinFWkbCNLu8TmzDf+Dujn0akQ0YvsAvDMMHJxdz0iKHl684\\\\\\\/boD4dzg2HgkLDh7Kn9Ue2IOyUTgpipVYyNNJLRPAVarnn0c\\\\\\\/5yZ9V1VmGnLE88KVf88SJX4r4qHTW6tm3wZe5CDEibNw52MRgaTgc8Pzi5wIj\\\\\\\/4IdU6KZ5TB5gRw+LxqVLhJMuXSIcmxin6+tvS+W5H3Z4ex7Fj6MBD8QwTjaYEwwmWxbr7E\\\\\\\/Q7tl8iAsoIJWvO8DKw7bz2USaiQZvBd69y5JEQt\\\\\\\/Roy2I\\\\\\\/YEFUvr1SPUFE8TnjjiPzZdUaZ15GCVO2I3xiaHHLteENxlwaHSCd74EZ4N7e3bqSvwLbMQ1HZl16F2Kj5AeVV\\\\\\\/\\\\\\\/EX85hcnFfhgp0lOkBgd9nH\\\\\\\/3joBzOaR9+odOLHjoAOhuRcB3qQ3\\\\\\\/ru9X01jTS3q6wxl0eyJXdaWbZvFcrz1IeDO6j\\\\\\\/+yOUMMYnRPHScPAQdy4Pm1MO9YX7Z52nLHs2JtSyn6GPnR\\\\\\\/PKjLeKEmrdPrfhdlg1M7idyxrdduFQXnCZphEwqIwzldLCMdbqmsEg+vMoPdeeIzcUE0cSj1uJK2zthrs6iCwDEYxyQuOBgFGHjnY86rVo\\\\\\\/Ic6rlfi+okAlCUnYHYLRnD3\\\\\\\/THbglrQOWWanyCnRQBtqlL3a2J2c9Ri54WSibpGJVnfowmvdIwB3\\\\\\\/FOBoObnmPiUnE+AgCwqtz15UHU\\\\\\\/akBWesOZ3oKuuvJHSumYmz7QFd5OvytuVCJwqoQwy2A6l5ltv5oBoX940NZhtK4L3+3SY3nnSst+sAwDKKY+2LUHxKzBnjXZWUpARx5siuzv5Q7A\\\\\\\/zRscJVrYFFvnJRw\\\\\\\/5hPZN4T0rn4AVkuyUYPSVAsMBkspfqm3vHWhwRcaCgloRdPncHYlpdt84SMDmMPrqH5ao3vKGSTJjNURTX9rHIf2G92fBhiJd3Lp34diDFc3anjKmeKNCxopwcz7nKStqO09\\\\\\\/Myujzn97UWcwFLf4ejbQFFWnadeqYS8i60mcwPa7AuPFhJjGSrhpBeB4eiQRYAK1qBH5E9Musf1S4L7EHS54XjdyX1B\\\\\\\/66Kl2bd2imS5hT0Nn+mKpxBEITX6zOVSVi0dnfFP5MTl5fu51kvReKcVDbh6gbCxvyXMT93mREoGaUFzMcunYShitkeDZV00Bkn0+BM+1XG03G3lBInhkKaYjuQqHZuFDqEeM4mzsPueRFzuCalBWppLDk0jAkGx\\\\\\\/6fagxzUVZehI9A\\\\\\\/+mgGmQjTCeYB9Gd4QSKkodE\\\\\\\/Zf7xRRcPUw8NJY7460GETBMoTUTkXO4g340bhNx\\\\\\\/E2jFAftmAicj36FDQ09WmikSQkwP6Ws0scor\\\\\\\/TgIUUmbQL0vo54Vlc7qk5WwMWjukFtU\\\\\\\/ZTG7u+CYqsiZW\\\\\\\/ApjZ6oIRt+1VKw15RS54Kmk90gBwxOwHfIsk80b0deP4zyhblWHBnPasTHnskWxjpFk0qmtZ3tSGCJdeWh8TviR8jBg7gHRipkBCmdtii+WAN+yoETDCVk1aOA0V1GoOF0dANCKZgYp807cxpC\\\\\\\/Knrw\\\\\\\/1tWISjJcJ\\\\\\\/Bol\\\\\\\/uBs65vrUXKkCDV8K8FQyRtb2+uCn\\\\\\\/beEVxaV2i8UZTQuIhP5RzFnE4fnUfT40jgspa2kaJ8\\\\\\\/SnJyQtq8rHrY1QpkHdSg0cSHOqvWapoAsK0fu6NawGE71mv+ZMNG9VGMke7OAiJzonRRZ20hP5Yi0tWIbH2YBCXoE1C98TL2VvnYHV0nR1VHgLUHgBF8BtJCEJ\\\\\\\/nDAtAHA2gj8N+pMhBtTTMAFM7HkN7fblOoj2s76GwM0C0XqNCtcakaoQWew03nPJS8nLui3bhfNmfVNsbw4OBNUn5oiXLgiJ2md0g3xNsUEkqYz\\\\\\\/8s0anfZquoAz8J+Xh31NsDoeAZ2oTlAoR1RNFK0aSufhAT4oU5ZBIlEPb7juKYXmnyxmEy9tWDjaLT+xc+XeNoyNUp8\\\\\\\/AoEsuwYqie2wq3MB1PDGHcDOb4gtS1tNrC6HuqX923A1YvkDUBZtySKv1QIO8UK7AviF0FDnpQZUmtNW5oLvA1ymp\\\\\\\/SfE5rBpegVtHvDdQp5zQ1uo6akaPZl82l66k5clLX3ncN\\\\\\\/gaWpwMWzHhisQZuMUJWF\\\\\\\/x2eoWu35Ls9HeMuQ9rF2SXRCcd4nmcdhCJNds9\\\\\\\/S6wg0fUabrFPavw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022JHspCa4Yf0lIDbKXoheJQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262005a6fb5d9f23014e36bc3bd970c93bcc0f04c1355bd9b70fb13feca5cd700\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FK1NLExxfxu6CVZ6p95hkSkVJ1nYxmiBWWaB3\\\\\\\/IZAChafY9DLQAgHFWumS59y0nr+Jsnq1xs55u131BYaarl3fI04Sij\\\\\\\/RSsYFTCqCBSzI48aOo9KzONCTK0TbpvRHSGrj9nd\\\\\\\/LDr6Xyo7XXuHz84aAT9rOj95UOb+CaD\\\\\\\/fI9Z3vbIeDyfOsJ8yYbSxQ3MALtL\\\\\\\/r+R6pFaN8VA0vyNLpdsxAdaOaRpmt0WMtFzf2KuxY1e86h0WvE8koGCsadILox6WfQ9Ddp\\\\\\\/hTuH1cvo7kmalpgrowqOZs4OLUO6+scDpV0anIQwa9lcrR29ic4Q+1fOhjyD\\\\\\\/Tose10mxhQTEy\\\\\\\/ZSqYLdVQQHooTzx936th8W5b2gf5xa8EeiFO6sPHKZDAXzqkvxm7PVQNYXVtK2grV2cPqFU1VhdK++0rCNQs0XegOlFIt0ppsBYXMJ4k7RxZ5VoxIIf2wVlfzGaYw6nUh036oO\\\\\\\/caqIZP5M3Tyo9wfeQ+D9vS2+QxxgicNFAJK\\\\\\\/DGkm7oTCzWD4ME3ZAgf5PUQrKTUVS5HeWETD3LjrYkgHFqcJ6OydHYEW24cY7Ois4Va6c3OEFLiQhu60R7c+fCezkUoc5DTNq3HiTYds9I7QTvAB0TTo3jIOZqqPFY1mBXwZQTFbHQxxYcXIpfoIRTYqRDqTfMxcg3nrmMOjKDl14+0Os+hMFkRyELkJ7MOzmJe99NcZKiN6QjXP0UvfsBn6oNNapWPLB0A9Zd2\\\\\\\/JdDKJlo2sfRDzkG1e8PInKZIpclCcC5Y8vYZNOU0PgiXf7hv7W6tIx9QiOWBPWBKgbzuyK\\\\\\\/htWYKGC2cZcumbQmSp93b8IITD6\\\\\\\/s3xmFQUuwRtnolV92wxwhulYoqFmDDC1p4dqouPx2gs3gcoa73yJnxPcz3xLVNPkb7tqr7ksRbAyAo5SoVm3Ef2gl8Pf5x6iHsUxWT7VfqCpzOj8Y9u+N303sRrM+t5qznZII3gjj5PV6usMJtuA2TBJMTcPgFmagPIjR0rnuVY3BcnrPsqkCg04kAMDTRcJjmqKXQTWh\\\\\\\/N9EaIvrUU9o+QNPtsAKP4qHbXVqSMrFrPqcTS6BOvdpmn6Mq9Z4tmAUTlVxPUDDEVKpSKRp2gl9Vl4ahOq6AFlZ1LBhaDY2KdyEi7GFET75kjKFIt9\\\\\\\/R6ZJUQ5mGq7F3rVVj6AdkWLfKRTpgn6fVQfOkq0XS88YS1CFFLIX7gdMV6R8H2efLTzH5Bl8JNDtPPYPNxGaq7t\\\\\\\/BikEcEhsnVPkMoibf0D\\\\\\\/92gBXnSwcuRIA98h2wCXMnevsHEhh2Y7XODNhc1JjmYZHuj84lTXqb0kSvNSSmLYBrv\\\\\\\/HxDLCZ9yZsorwiePbL2lU6SnyqUIiGc+K4+r8r88SYPfQBKYIgfMTVZ9NjLgWGd0sBWfVxb6+tbiQQdLN+AZVVMFEL8RGKtQALXcG+GcDIlbwj8bTbhavry0c4OHKsxyxbXh51RUVaKfPQm+Qc6HsHYJKQjBkIYUs10p7MiTgWCPkLK091+\\\\\\\/MC1LgekgdP\\\\\\\/ksbwuF4ZaFcNnggOCrgUst\\\\\\\/HMW0XM3XRRcz7rz26sfpRiyQXS3mnOpFi3qjMIeZR3W\\\\\\\/kpj9Qejx7dD1yRFGNeU0+bAbv37uniAAZ0TiZS7ZPW12ivB6AXZ9s+EVeECMBEMYAMTH5sbfOjYAM+RQk95Lou\\\\\\\/pFMBv0mn0tmzNfMPrbdz9q8efjA76eDu1PiMaIeY22P7EIwjC0cmlq\\\\\\\/Fc\\\\\\\/YWlHQ5s7KeFLlf6GVQlWbGrEfg2OrAayJ6DP4MGpBB9txG4wvv6SBj8G2R+Z3zCxBKGKfB8d10Xf\\\\\\\/7PcC5K9vPAIxLHjRZZUYhJad3K+ow24PGl55019ZNxoXFmHqH1fvxSa4NBjJ\\\\\\\/LtxHYS94J7YmF3zEfZDuvhRn84h2ljJqAoXJUuESPYm+ZgFUxhIYHgl+\\\\\\\/mTDTaR3fGI6TKCXcE4v2DEMTT+ye\\\\\\\/v76Wens2zDZZIRSbFmIfAl6krzxiQa55nhoXSqknhVw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Cm65XP7yfrocr4DUAGyeOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002244914453978d80649f59e42303f6e31bd31e60717329abf470d205fad5009cc1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459810 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/593" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e89Kz9LvglNu7KW7pnFwqzsBbr3cJm9y7O9aMAa5TOE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/593" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gDU1lokIqLBFwkBgA2C30tRUr8QW28eJjls0larR9aU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/583" title="Adaptive Evolution of Genes Involved in the Regulation of Germline Stem Cells in Drosophila melanogaster and D. simulans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/605" title="Gene Regulation by H-NS as a Function of Growth Conditions Depends on Chromosomal Position in Escherichia coli" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459810" id="node459810" data-pisa="ggg;5/4/593" data-pisa-master="ggg;g3.114.015883" data-apath="/ggg/5/4/593.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Global Diversity Lines–A Five-Continent Reference Panel of Sequenced <em>Drosophila melanogaster</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennifer K. Grenier</span>, <span class="highwire-citation-author" data-delta="1">J. Roman Arguello</span>, <span class="highwire-citation-author" data-delta="2">Margarida Cardoso Moreira</span>, <span class="highwire-citation-author" data-delta="3">Srikanth Gottipati</span>, <span class="highwire-citation-author" data-delta="4">Jaaved Mohammed</span>, <span class="highwire-citation-author" data-delta="5">Sean R. Hackett</span>, <span class="highwire-citation-author" data-delta="6">Rachel Boughton</span>, <span class="highwire-citation-author" data-delta="7">Anthony J. Greenberg</span> and <span class="highwire-citation-author" data-delta="8">Andrew G. Clark</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">593-603;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015883 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459810" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennifer K. Grenier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BK.%2BGrenier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grenier%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BK.%2BGrenier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">J. Roman Arguello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BRoman%2BArguello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arguello%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BRoman%2BArguello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Margarida Cardoso Moreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margarida%2BCardoso%2BMoreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreira%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargarida%2BCardoso%2BMoreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Srikanth Gottipati </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Translational Medicine and Think Team, Otsuka Pharmaceutical Development and Commercialization, Inc., Princeton, New Jersey 08540</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikanth%2BGottipati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gottipati%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikanth%2BGottipati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaaved Mohammed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaaved%2BMohammed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammed%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaaved%2BMohammed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean R. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Quantitative and Computational Biology, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BR.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BR.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rachel Boughton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BBoughton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boughton%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BBoughton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Anthony J. Greenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BJ.%2BGreenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenberg%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BJ.%2BGreenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andrew G. Clark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BG.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clark%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BG.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ac347@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ac347@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/593" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459810" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459810/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/593.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459810" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459810/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/593.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459810" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459810/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/593.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459810" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459810/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/593.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459810" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459810/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Reference collections of multiple <em>Drosophila</em> lines with accumulating collections of “omics” data have proven especially valuable for the study of population genetics and complex trait genetics. Here we present a description of a resource collection of 84 strains of <em>Drosophila melanogaster</em> whose genome sequences were obtained after 12 generations of full-sib inbreeding. The initial rationale for this resource was to foster development of a systems biology platform for modeling metabolic regulation by the use of natural polymorphisms as perturbations. As reference lines, they are amenable to repeated phenotypic measurements, and already a large collection of metabolic traits have been assayed. Another key feature of these strains is their widespread geographic origin, coming from Beijing, Ithaca, Netherlands, Tasmania, and Zimbabwe. After obtaining 12.5× coverage of paired-end Illumina sequence reads, SNP and indel calls were made with the GATK platform. Thorough quality control was enabled by deep sequencing one line to &gt;100×, and single-nucleotide polymorphisms and indels were validated using ddRAD-sequencing as an orthogonal platform. In addition, a series of preliminary population genetic tests were performed with these single-nucleotide polymorphism data for assessment of data quality. We found 83 segregating inversions among the lines, and as expected these were especially abundant in the African sample. We anticipate that this will make a useful addition to the set of reference <em>D. melanogaster</em> strains, thanks to its geographic structuring and unusually high level of genetic diversity.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/d-melanogaster" class="hw-term hw-article-keyword hw-article-keyword-d-melanogaster" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">D. melanogaster</em></a></li><li class="kwd"><a href="/keyword/global-diversity-lines" class="hw-term hw-article-keyword hw-article-keyword-global-diversity-lines" rel="nofollow">global diversity lines</a></li><li class="kwd"><a href="/keyword/whole-genome-sequences" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequences" rel="nofollow">whole-genome sequences</a></li><li class="kwd"><a href="/keyword/inversion-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-inversion-polymorphism" rel="nofollow">inversion polymorphism</a></li><li class="kwd"><a href="/keyword/residual-heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-residual-heterozygosity" rel="nofollow">residual heterozygosity</a></li></ul><p id="p-4">The human commensal dipteran <em>Drosophila melanogaster</em> has long played a central role as an experimental genetic model system, as well as a preeminent model for understanding evolutionary and population genetic processes. As one of the first fully sequenced eukaryotic genomes, <em>D. melanogaster</em> offers a wealth of genomic and genetic resources, including increasing characterization of natural genetic variation found in extant populations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pool <em>et al.</em> 2012</a>). Because <em>Drosophila</em> biology is being increasingly explored through the use of high-throughput and “omics” approaches, these maps of genetic variation can be placed within a systems biology framework, wherein they are used as natural perturbations for the dissection of complex traits (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ayroles <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blair <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Massouras <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breunig <em>et al.</em> 2014</a>). The initial motivation to generate these lines was to have wide geographic/climatic provenance to maximize metabolic variation and to use interline differences to explore the genotype−phenotype relation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jumbo-Lucioni <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ghazalpour <em>et al.</em> 2014</a>). We intend that these lines will serve as a foundation for advancing ongoing modeling efforts at prediction of metabolic and other complex traits from DNA sequence and other “omics” data sets (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jumbo-Lucioni <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eanes 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lavington <em>et al.</em> 2014</a>).</p><p id="p-5">Although several population genomic projects have been published recently, they have either been specifically designed for association mapping purposes and thus were generated from a single location and/or from a specific crossing scheme (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">King <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2014</a>), or they were constructed solely for characterizing population genetic variation, without stable lines for functional follow-up studies (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Pool <em>et al.</em> 2012</a>). A significant difference between our current effort and these previous efforts is that we have aimed to capture variation that exists between geographically diverse populations. As a result, these data will not only serve as an anchor for systems biology approach to complex traits such as metabolic regulation, but also will be generally informative from an evolutionary genetic standpoint. In particular, these data are expected to be well suited for demographic inferences and investigations of local adaptation because of the inclusion of lines from multiple globally distributed populations with distinct evolutionary histories.</p><p id="p-6">In this current article, our intent is to provide a thorough description of the genetic makeup of these lines, which we refer to as the Global Diversity Lines, and detail their creation and our procedures for ensuring data quality. We also aim to highlight salient features of initial analyses characterizing single-nucleotide polymorphisms (SNPs) and small indel variation, as well as evidence for population structure (specific analyses dealing with focused topics will appear in independent articles). We have generated genome sequences for 84 lines to a depth of ~12.5× coverage. We have generated variant calls for SNPs, indels, and large inversions. Independent validation of all classes of variant calls has demonstrated that they are of high quality. A striking feature of inbred <em>D. melanogaster</em> genomes, and one that has gained more recent attention, is residual heterozygosity, especially in lines derived from more equatorial populations (<em>e.g.</em>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langley <em>et al.</em> 2011</a>). We also observe such heterozygous blocks, and through extensive computational and experimental steps to characterize their nature, find that large regions of residual heterozygosity in inbred lines correlate highly with inversions.</p><p id="p-7">As expected, the African samples are the most diverse for all classes of mutation, but with significant heterogeneity remaining within the other four populations. By sampling from five highly dispersed regions, we anticipate that these lines will be of utility for studies that benefit from maximal genetic diversity, and in answering questions that entail aspects of interpopulation differences that may have been driven in part by local adaptation. The original intent in generating these lines was not for association mapping, but rather was to serve as a reference set of lines of high diversity for building and testing predictive models of metabolic regulation and other complex traits in <em>Drosophila</em>. Despite the fact that the lines differ at a large number of nucleotides and insertions/deletions, those differences impose a correlation structure across lines that is informative for a systems biology approach to quantitative models of complex phenotypes. Those modeling papers will appear elsewhere, and it is our intention here to simply provide an introduction to this resource set of lines and to describe the basic attributes of their genetic differences.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">We determined and vetted the whole genome sequences of 84 + 1 inbred <em>Drosophila</em> lines representing five global populations (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Greenberg <em>et al.</em> 2010</a>; the “+1” is for line ZW184, recovered in Zimbabwe but appearing to be a very recent migrant, and so is left out of population structure studies). An expanded version of our <em>Materials and Methods</em> section is provided as <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/015883SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>. To summarize, genomic DNA was extracted from pools of 50 adult females, and we generated paired-end 100 nt Illumina reads at average 12.5× depth for each line (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 2). Sequence reads were aligned with the Burrows-Wheeler Alignment Tool (BWA, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li and Durbin 2009</a>; <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 3) to the reference <em>D</em><em>. melanogaster</em> genome, and SNPs and small indel genetic variants were called using the Genome Analysis Tool Kit (GATK; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Depristo <em>et al.</em> 2011</a>; <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 4). Genetic variant calls were validated in two ways, by resequencing one line to 100× depth and by double-digest restriction-site associated (ddRAD; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Peterson <em>et al.</em> 2012</a>) resequencing of a consistent subset of the genome for 12 lines, which informed additional filtering to create the final variant call sets (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 6). We noted that some subregions of the genome in most lines had an unexpectedly high frequency of heterozygous SNP genotypes that validated at high frequency, which we define as “heterozygous blocks” (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 5). Finally, we investigated the whole-genome sequence dataset for large chromosomal inversions using a custom bioinformatics pipeline (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cardoso-Moreira <em>et al.</em> 2012</a>; M. Cardoso-Moreira, J. R. Arguello, D. Riccardi, S. Gottipati, J. K. Grenier, and A. G. Clark, unpublished data) that uses several available tools for genome mapping and structural variation detection [Novoalign (<a href="http://www.novocraft.com">www.novocraft.com</a>), Mosaik (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lee <em>et al.</em> 2014</a>), Delly (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rausch <em>et al.</em> 2012</a>), and BLAT (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kent 2002</a>); <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, parts 3 and 10]. Candidate inversions were validated by polymerase chain reaction (PCR) across at least one of the inversion breakpoints, and the genotype of each line was determined by looking for reads supporting the presence of the inversion breakpoint and/or the reference sequence bridging the breakpoint (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 11). Alignment files and final genetic variant genotypes (vcf files), as well as companion files including 1) het blocks per line, 2) genome callability (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 9), and 3) regions of genetic identity by descent (IBD; <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, part 8), are described in <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS1.pdf">Table S1</a>.</p><p id="p-9">Preliminary molecular evolution and population genetic analyses were carried out to provide additional data quality checks, as well as to provide initial broad characterizations of inter- and intrapopulation variation (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a>, parts 13-16). To summarize, our divergence analyses were based on an updated five-species whole-genome alignment that we generated [<em>D. melanogaster</em> (dm3), <em>D. simulans</em> (droSim2), <em>D. sechellia</em> (droSec1), <em>D. erecta</em> (droEre2), and <em>D. yakuba</em> (droYak2)], within which we include the recently improved <em>D. simulans</em> assembly (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hu <em>et al.</em> 2013</a>). Population genetic analyses were carried out with the final SNP calls, but were further masked based on variant callability and regions of genetic IBD. Several analyses use an additional SNP subset, referred to as “neutral” SNPs, that fall within small introns or in fourfold degenerate coding positions as determined by our SNPeff annotation (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cingolani <em>et al.</em> 2012</a>).</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Variant discovery</h3><div id="sec-4" class="subsection"><h4>Variant calling pipeline:</h4><p id="p-10">We resequenced the genomes of 84+1 inbred lines of <em>D. melanogaster</em> sampled from five globally diverse populations (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Greenberg <em>et al.</em> 2010</a>) to characterize the genetic variants represented in this collection. Genomes were sequenced at an average 12.5× depth per line (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS2.pdf">Table S2</a>), and an average of 92% of the reads from each line mapped to the reference <em>D. melanogaster</em> genome. Both SNPs and small indels were called using a pipeline (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) based on GATK v1 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Depristo <em>et al.</em> 2011</a>). We followed the best practices guidelines of GATK to improve variant calls. Notably, the Base Quality recalibration step reduced the number of euchromatic variant sites (mapped to X, 2L, 2R, 3L, 3R, and 4) by nearly 50%, primarily caused by a decrease in the number of heterozygous SNP calls.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variant call pipeline. Genetic variants were determined using a pipeline similar to the Genome Analysis Tool Kit (GATK) Best Practices recommendations (https://www.broadinstitute.org/gatk/guide/best-practices) and supplemented with additional filtering steps. First, reads were mapped to the D. melanogaster reference genome with the Burrows-Wheeler&#xA0;Alignment Tool (BWA). After merging files for all lines, GATK (version 1) was used to locally realign reads near indels and to create and filter a preliminary set of single-nucleotide polymorphism (SNP) calls. These preliminary SNP calls were used for GATK base quality and variant quality recalibration. After this GATK pipeline, variants were further filtered using variant recalibration site filter and genotype quality filters. SNPs also were filtered when located near an indel call in the same line. See File S1 for details. IBD, identity by descent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variant call pipeline. Genetic variants were determined using a pipeline similar to the Genome Analysis Tool Kit (GATK) Best Practices recommendations (https://www.broadinstitute.org/gatk/guide/best-practices) and supplemented with additional filtering steps. First, reads were mapped to the &lt;em&gt;D. melanogaster&lt;/em&gt; reference genome with the Burrows-Wheeler&#xA0;Alignment Tool (BWA). After merging files for all lines, GATK (version 1) was used to locally realign reads near indels and to create and filter a preliminary set of single-nucleotide polymorphism (SNP) calls. These preliminary SNP calls were used for GATK base quality and variant quality recalibration. After this GATK pipeline, variants were further filtered using variant recalibration site filter and genotype quality filters. SNPs also were filtered when located near an indel call in the same line. See File S1 for details. IBD, identity by descent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F1.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/593/F1.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Variant call pipeline. Genetic variants were determined using a pipeline similar to the Genome Analysis Tool Kit (GATK) Best Practices recommendations (<a href="https://www.broadinstitute.org/gatk/guide/best-practices">https://www.broadinstitute.org/gatk/guide/best-practices</a>) and supplemented with additional filtering steps. First, reads were mapped to the <em>D. melanogaster</em> reference genome with the Burrows-Wheeler Alignment Tool (BWA). After merging files for all lines, GATK (version 1) was used to locally realign reads near indels and to create and filter a preliminary set of single-nucleotide polymorphism (SNP) calls. These preliminary SNP calls were used for GATK base quality and variant quality recalibration. After this GATK pipeline, variants were further filtered using variant recalibration site filter and genotype quality filters. SNPs also were filtered when located near an indel call in the same line. See <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a> for details. IBD, identity by descent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">After calling variants with GATK, we immediately observed that the distribution of heterozygous genotype calls was highly nonrandom across the genome of each line. The number of segregating sites within a line was expected to be low because of the 12 generations of sib-pair inbreeding for these lines, but we found contiguous stretches in the genome with a high frequency of heterozygous calls in many lines. These blocks of heterozygosity have also been found in the <em>Drosophila</em> Genetic Reference Panel lines albeit to a smaller extent (Table S4 in <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Mackay <em>et al.</em> 2012</a>) and are a known feature of <em>D. melanogaster</em> populations, particularly those of African origin (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Langley <em>et al.</em> 2011</a>). We defined “heterozygous blocks” as genomic regions that harbored an excess of contiguous heterozygous SNP calls. These blocks are seen along all chromosomes, except the X chromosome, where they are relatively rare, and chromosome four, where they are absent. On average there are 6.9 blocks per line. Overall, the average size of a “het block” was 5.4 Mb, though their distribution is bimodal (first mode ~300 kb, second mode ~24 Mb). Although a minority (37%) of the “het blocks” exceeds 2 Mb in length, these large blocks encompass more than 2.7 Gb of the total 2.9 Gb cumulative span, and some “het blocks” extend the length of a full chromosome arm. Notably, the third chromosome and the Zimbabwe populations harbor an excess of these large blocks. The “het block” intervals are available in bed file format (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS1.pdf">Table S1</a>).</p><p id="p-13">To assess the false-positive rate for the variant calls, and to examine whether additional sites or calls should be filtered out, we validated the SNP and small indel calls in two ways. First, we resequenced a single line (ZW155) to 100× depth. SNPs and small indels were called from aligned reads using a simple read-count ratio to avoid systematic genotyping error. Variant sites were identified as sites with &gt;100 reads for which &gt;10% of reads supported an alternate allele from the reference genome; heterozygous sites were called when two different alleles each had support from &gt;10% of reads. Overall, there was very high agreement between the “10×” GATK and “100×” read-count ratio genotype calls for ZW155 SNPs, greater than 99% for homozygous sites (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS2.pdf">Figure S2</a>). The agreement for heterozygous SNPs was lower (81%) overall and clearly different within “het blocks” (93%) compared with outside of “het blocks” (30%).</p><p id="p-14">As a second method of validation we generated ddRAD libraries (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Peterson <em>et al.</em> 2012</a>) for 12 lines. SNPs were called in a similar manner to the “100×” ZW155 validation set using read-count ratios at variant sites with &gt;100× coverage in the ddRAD libraries. The number of validated sites per line was much lower because of the reduced representation of the ddRAD libraries, but the validation rate for SNP homozygous sites was again greater than 99% (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS2.pdf">Figure S2</a>). The validation of SNP heterozygous sites was initially much lower at 62%, and did not improve within “het blocks.” This discrepancy is most easily reconciled by the fact that reduced representation libraries under-represented one chromosome as a result of variants that alter a restriction site or fragment size (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arnold <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gautier <em>et al.</em> 2013</a>).</p><p id="p-15">We investigated whether the variant quality score (VQS) per site or the genotype quality score (GQ) for individual genotype calls correlated with validation rate. Using the “100×” ZW155 validation set, we found that GQ clearly correlated with validation (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS3.pdf">Figure S3</a>) but VQS did not (data not shown). Genotype calls with a greater GQ validated at a greater rate. We did find that, following variant quality score recalibration, the sites with the lowest VQS-recalibrated also had lower validation rates, especially heterozygous sites. We used a combination of GQ score, VQS recalibrated-flag, and genotype to filter out classes of genotype calls with validation rates below 90%, including all heterozygous calls outside of “het blocks.” This filtering reduced the number of nonreference SNP genotypes by 12% and the number of variant SNP sites by 5%. Finally, we masked SNP genotype calls if the SNP was within 5 nt of an indel call in the same line. There are more than 5.75 M euchromatic SNPs in the final set of variant sites (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS3.pdf">Table S3</a>), with 97% of genotypes called across 84+1 lines.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variant count summary. (A) A total of 5.78 M SNP sites and 971 k small indel sites were discovered in the final panel of 84+1 Global Diversity lines. About half the variant sites per chromosome are shared among more than one population, with the Zimbabwe population contributing the majority of populations-specific variant sites for both (B) single-nucleotide polymorphisms (SNPs) and (C) small indels. The ZW184 line has suspect provenance, and is excluded from B and C (this line is the &#x201C;+1&#x201D; in our designation of 84+1 lines). *Chromosome 4 counts are &#xD7;10,000 in panels B and C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variant count summary. (A) A total of 5.78 M SNP sites and 971 k small indel sites were discovered in the final panel of 84+1 Global Diversity lines. About half the variant sites per chromosome are shared among more than one population, with the Zimbabwe population contributing the majority of populations-specific variant sites for both (B) single-nucleotide polymorphisms (SNPs) and (C) small indels. The ZW184 line has suspect provenance, and is excluded from B and C (this line is the &#x201C;+1&#x201D; in our designation of 84+1 lines). *Chromosome 4 counts are &#xD7;10,000 in panels B and C.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F2.medium.gif" width="440" height="113"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/593/F2.medium.gif" width="440" height="113"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465891" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-16" class="first-child">Variant count summary. (A) A total of 5.78 M SNP sites and 971 k small indel sites were discovered in the final panel of 84+1 Global Diversity lines. About half the variant sites per chromosome are shared among more than one population, with the Zimbabwe population contributing the majority of populations-specific variant sites for both (B) single-nucleotide polymorphisms (SNPs) and (C) small indels. The ZW184 line has suspect provenance, and is excluded from B and C (this line is the “+1” in our designation of 84+1 lines). *Chromosome 4 counts are ×10,000 in panels B and C.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To validate small indel calls, we used the same “100×” ZW155 deep resequencing data (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS4.pdf">Figure S4</a>). Coverage of indels in the ddRAD dataset was too low and yielded uncertain representation of heterozygous sites to use for validation. Similar to the SNP calls, we used a combination of GQ score and genotype to filter out classes of genotype calls with validation rates below 75%, including all heterozygous calls outside of “het blocks.” A lower validation threshold was used for small indels because the only heterozygous calls retained (GQ = 99, inside “het block”) had a validation rate of ~75%, which set a maximum for homozygous calls as well to retain the same density of variant genotypes across lines. In total we identified nearly 1M euchromatic small indels (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS3.pdf">Table S3</a>), with 80% of genotypes called across 84+1 lines.</p><p id="p-18">The impact of the variant call pipeline was quite different between SNPs and small indels, as well as within and outside of “het blocks.” The base quality score recalibration reduced the number of heterozygous calls similarly across all chromosomes and lines such that “het blocks” retained about half the number of preliminary heterozygous calls whereas regions outside of “het blocks” retained very few heterozygous genotype calls with lower GQ scores. In contrast, base quality score recalibration had minimal effect on homozygous calls. The number of heterozygous calls for a given line and chromosome is variable, with some lines having a comparable number of heterozygous and homozygous calls and others having very few heterozygous calls (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS5.pdf">Figure S5</a>). These patterns indicate the presence or absence, respectively, of large “het blocks.” The pattern for small indels is similar, including the reduction in heterozygous calls across all lines in the base recalibration step and the prevalence of heterozygous calls for certain lines and chromosomes indicating the presence of ‘het blocks’, although the filtering step reduced the number of calls more significantly than for SNPs (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-5" class="subsection"><h4>Gene annotation:</h4><p id="p-19">We next investigated the distribution of variant calls with respect to gene annotation. Variants with high predicted impact and located in exons were least likely to be retained in the variant call pipeline (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), indicating that as call quality increased due to recalibration and filtering, the false-positive rate of high impact sites decreased. Additionally, we observe a clear mod-3 pattern in small indels located in coding regions, which is magnified by the recalibration and filtering steps (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS7.pdf">Figure S7</a>). These patterns of change in the distribution of variant annotations indicate the improved quality of the variant calls at each step of the pipeline.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene annotation. The number of variant sites falling within exons was disproportionately reduced during the variant call pipeline for both (A) single-nucleotide polymorphisms (SNPs) and (B) small indels compared with variant site locations, particularly due to the GATK base quality recalibration step. The percent of variant sites retained after filtering is shown below the gene annotation category. (C) Similarly, SNP and small indel sites annotated as high impact were disproportionately reduced during the variant call pipeline compared to lower impact sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Gene annotation. The number of variant sites falling within exons was disproportionately reduced during the variant call pipeline for both (A) single-nucleotide polymorphisms (SNPs) and (B) small indels compared with variant site locations, particularly due to the GATK base quality recalibration step. The percent of variant sites retained after filtering is shown below the gene annotation category. (C) Similarly, SNP and small indel sites annotated as high impact were disproportionately reduced during the variant call pipeline compared to lower impact sites.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F3.medium.gif" width="440" height="125"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/593/F3.medium.gif" width="440" height="125"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465803" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Gene annotation. The number of variant sites falling within exons was disproportionately reduced during the variant call pipeline for both (A) single-nucleotide polymorphisms (SNPs) and (B) small indels compared with variant site locations, particularly due to the GATK base quality recalibration step. The percent of variant sites retained after filtering is shown below the gene annotation category. (C) Similarly, SNP and small indel sites annotated as high impact were disproportionately reduced during the variant call pipeline compared to lower impact sites.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>Genomic regions inferred to be IBD:</h4><p id="p-21">Investigation of genetic IBD revealed relatively few genomic regions containing signatures of recent common ancestry. As observed previously in <em>D. melanogaster</em> population genomic data (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>), most regions possessing candidate IBD stretches were restricted to the lowly recombining regions of pericentromeric regions. After we excluded these regions, 30 chromosomal stretches that were shared between single pairs of individuals within the same population remained. The median size of these stretches was estimated to be 5.64 Mb, and they were observed on all chromosome arms other than the 4th. The 30 genomic regions found IBD in a pair of lines are available in bed file format (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS1.pdf">Table S1</a>).</p></div><div id="sec-7" class="subsection"><h4>Large inversions are common and are associated with heterozygous blocks:</h4><p id="p-22">The identification of genomic regions with segregating variation within lines (“het blocks”) despite 12 generations of sib-pair inbreeding suggests that this heterozygosity is being actively maintained. One explanation would be the presence of large chromosomal inversions, known to be common in <em>D. melanogaster</em>—especially in equatorial populations (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lemeunier and Aulard 1992</a>). Large inversions protect regions of the genome from recombination and, when recessive deleterious alleles are present between the inversion breakpoints on both segregating chromosomes, can prevent complete inbreeding of lab stocks. The hypothesis that blocks of heterozygosity are the result of the presence of recessive lethal alleles and the presence of large inversions leads to a clear and testable prediction: heterozygous blocks should be associated with the presence of <em>heterozygous</em> inversions.</p><p id="p-23">We developed a bioinformatic pipeline to detect candidate inversions from the mapped whole-genome sequencing reads. Candidate inversions were assayed by PCR across the predicted breakpoints and sequence-verified; this validation process indicated a high false-positive rate for inversion prediction, but also did confirm the presence of at least 10 large inversions across the Global Diversity Lines.</p><p id="p-24">We independently identified and validated all of the eight large <em>D. melanogaster</em> inversions with known molecular breakpoints (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wesley and Eanes 1994</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto <em>et al.</em> 1999</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matzkin <em>et al.</em> 2005</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Corbett-Detig <em>et al.</em> 2012</a>), including four common and cosmopolitan inversions (In(2L)t, In(2R)NS, In(3L)P, and In(3R)P); two rare cosmopolitan inversions (In(3R)K and In(3R)Mo); and two recurrent endemic inversions (In(X)A and In(X)Be); see <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS2.pdf">File S2</a> for breakpoint sequences. We identified at least one line carrying each of the known large inversions; most inversions were found in lines from more than one population, except the rarer X-linked inversions which were each found in a single African line (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS4.pdf">Table S4</a>).</p><p id="p-25">We also identified the molecular breakpoints of two additional inversions (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS2.pdf">File S2</a>). One of the inversions matches well the cytogenetic limits for In(3L)62D:68A ‘Ok,’ which has been described as a recurrent endemic inversion in Africa (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lemeunier and Aulard 1992</a>), a fact that agrees well with the exclusive presence of this inversion in the Zimbabwe population. The other inversion we identified on chromosome 3R does not match perfectly the cytogenetic limits of previously described inversions but is located in the proximity of several candidates (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lemeunier and Aulard 1992</a>).</p><p id="p-26">For most large inversions, a line often had reads indicating both the presence of the inversion chromosome and the reference chromosome, indicating both chromosomes are segregating in the line. In total we identified 83 instances of heterozygous inversions segregating in our lines and 10 instances of homozygous inversions (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/TableS4.pdf">Table S4</a>). Of the 83 heterozygous inversions, 81 (98%) are located within heterozygous blocks (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS8.pdf">Figure S8</a>); none of the 10 homozygous inversions are associated with heterozygous blocks. This strongly supports the hypothesis that the lingering blocks of heterozygosity result from the presence of recessive deleterious alleles in linkage with inversions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465842/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465842/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Large inversions explain many regions of heterozygosity in the Global Diversity Lines</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">The explanatory power of the seven inversions found as heterozygous varies between populations. In the Zimbabwe population these seven inversions are sufficient to explain 90% of large heterozygous blocks. Outside of Africa, these seven inversions account for a smaller percentage of heterozygous blocks: 58% in the Tasmanian population, 57% in the Beijing population, 43% in the Netherlands population, and 32% in the Ithaca population. That only seven inversions could account for such a large fraction of heterozygous blocks (57% across the whole dataset) is quite surprising. After all, there are hundreds of inversions segregating in <em>D. melanogaster</em> (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lemeunier and Aulard 1992</a>), including four rare cosmopolitan inversions and eight endemic recurrent inversions that do not have characterized molecular breakpoints. Our work suggests that if all inversions segregating in our set of lines were identified, they could explain all large blocks of heterozygosity found in our lines. However, these inversions may be difficult to identify using short reads, as careful investigation of the remaining candidate inversions called by our bioinformatics pipeline do not align with a large fraction of unexplained blocks of heterozygosity.</p><p id="p-32">The presence of large inversions and associated heterozygous blocks in the inbred Global Diversity Lines implies that most individual flies in the stock may be heterozygous, because homozygous flies may have low chance of survival or be infertile. We developed PCR-based genotyping assays for inversions and for linked SNPs and confirmed that individual fly genotypes are typically heterozygous within inversion-associated heterozygous blocks (data not shown). Furthermore, to directly test whether heterozygous blocks contain recessive deleterious alleles, we designed a genetic approach to force selected chromosomes to be homozygous (see <a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FileS1.pdf">File S1</a> part 12). For five of six chromosomes tested, we found that homozygotes for either of the chromosomes segregating in the original stock were significantly underrepresented or entirely absent relative to heterozygous sibs.</p><p id="p-33">There was one exception to the observation that inversions are heterozygous in individual flies and do not support a healthy homozygous stock. We found that one inversion, In(3L)62D:68A, is often homozygous in Zimbabwe line ZW155 (1 of 5 males, 4 of 10 females), and that homozygous flies are fertile. The noninverted chromosome appears to harbor a recessive lethal allele, as it was never found homozygous in individual flies from this line. Furthermore, the inversion chromosome from this line generated close to the expected ratio of homozygous viable progeny in the genetic test, although derived homozygous stocks appear weak after a few generations. Surprisingly, given that flies homozygous for the In(3L)62D:68A chromosome in the ZW155 line appear viable and fertile, the In(3L)62D:68A inversion was sampled 10 times in the Zimbabwe lines and is never found homozygous despite 12 generations of sib-pair inbreeding. Thus we suspect that the In(3L)62D:68A inversion chromosome has sufficiently low fitness to avoid homozygosity.</p></div></div><div id="sec-8" class="subsection"><h3>Population genetic-based tests of the data</h3><p id="p-34">The quality of the sequence data indicates that it is suitable for molecular population genetic analyses. Our initial work highlights strong population differences among the five populations, as well as differences between autosomes and the X chromosome within populations. Patterns of nucleotide diversity and LD are consistent with Zimbabwe having the largest effective population size, while non-African populations are significantly less diverse and more similar to one another than to Zimbabwe. Notable differences in allele frequency exist among combinations of populations, but also at putatively functional sites that are monomorphic in all but one population, where a high frequency derived allele is found. In a companion paper we will investigate questions related to neutral and selective forces leading to population differentiation, as well as autosomal and sex chromosome differences.</p><div id="sec-9" class="subsection"><h4>Population structure:</h4><p id="p-35">As a first look into the extent of population structure between samples, we calculated genome-wide <em>F<sub>ST</sub></em> using the SNP data (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Consistent with an out of African migration (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Begun and Aquadro 1993</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Glinka <em>et al.</em> 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Haddrill <em>et al.</em> 2005</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ometto <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Stephan 2006</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thornton and Andolfatto 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Laurent <em>et al.</em> 2011</a>), Zimbabwe is the most differentiated population, as can be seen by the narrow and more transparent edges in the network (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; average <em>F<sub>ST</sub></em> autosome: 0.08; X chromosome: 0.139), with the four non-African populations more closely related to each other. These levels of <em>F<sub>ST</sub></em> are in general agreement with reports from smaller datasets from the same populations (or proximal geographically). Interestingly, after Zimbabwe, Beijing is the next most differentiated. A previous survey also showed that other Asian strains were among the more differentiated within world-wide samples (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schlötterer <em>et al.</em> 2006</a>). Although the average <em>F<sub>ST</sub></em> for Beijing is not as large as some of these other Asian estimates (average <em>F<sub>ST</sub></em> autosome: 0.05; X chromosome: 0.08), they provide additional evidence that Asia may harbor some of the more divergent <em>D. melanogaster</em> populations globally.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population distance network for the five populations as measured by genome-wide FST . Nodes represent each of the five populations (B: Beijing; I: Ithaca; N: Netherlands; T: Tasmania; Z: Zimbabwe), with edges representing the estimated distances measured by FST between all pairs of populations. Increased edge width and transparency corresponds with increased migration/gene flow (and decreased FST /differentiation)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population distance network for the five populations as measured by genome-wide &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; . Nodes represent each of the five populations (B: Beijing; I: Ithaca; N: Netherlands; T: Tasmania; Z: Zimbabwe), with edges representing the estimated distances measured by &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; between all pairs of populations. Increased edge width and transparency corresponds with increased migration/gene flow (and decreased &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; /differentiation).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F4.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/4/593/F4.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465897" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Population distance network for the five populations as measured by genome-wide <em>F<sub>ST</sub></em> . Nodes represent each of the five populations (B: Beijing; I: Ithaca; N: Netherlands; T: Tasmania; Z: Zimbabwe), with edges representing the estimated distances measured by <em>F<sub>ST</sub></em> between all pairs of populations. Increased edge width and transparency corresponds with increased migration/gene flow (and decreased <em>F<sub>ST</sub></em> /differentiation).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Principal component analysis performed on the genome-wide “neutral” sites, which should best reflect demographic processes, reveals notable population structure among all populations (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Plotting the first two principal components cleanly separates not only Zimbabwe but also Beijing. However one line annotated as having an African origin (ZW184) does not group with the other Zimbabwe lines; we have excluded this line from subsequent population genetic analyses because its Zimbabwe origin is suspect, although it remains included in the global set of lines for modeling and mapping applications. Plotting the additional principal components (or removing the Zimbabwe lines, data not shown) reveals that even the Ithaca, Tasmania, and Netherlands population samples can be clearly separated.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure revealed by principal on autosomal &#x201C;neutral&#x201D; sites. Principal component clustering plots demonstrate the genetic structure present among the five populations. Left panel displays the strong separation of the Zimbabwe and Beijing populations from the remaining three populations based on the first two principal components. The right panel displays further separation of the Netherland, Tasmania, and Ithaca populations based on principal components two and three. Although less structured, clear separation is still observed for these latter three populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population structure revealed by principal on autosomal &#x201C;neutral&#x201D; sites. Principal component clustering plots demonstrate the genetic structure present among the five populations. Left panel displays the strong separation of the Zimbabwe and Beijing populations from the remaining three populations based on the first two principal components. The right panel displays further separation of the Netherland, Tasmania, and Ithaca populations based on principal components two and three. Although less structured, clear separation is still observed for these latter three populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F5.medium.gif" width="440" height="160"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/4/593/F5.medium.gif" width="440" height="160"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465810" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Population structure revealed by principal on autosomal “neutral” sites. Principal component clustering plots demonstrate the genetic structure present among the five populations. Left panel displays the strong separation of the Zimbabwe and Beijing populations from the remaining three populations based on the first two principal components. The right panel displays further separation of the Netherland, Tasmania, and Ithaca populations based on principal components two and three. Although less structured, clear separation is still observed for these latter three populations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Combined site frequency spectra (SFS):</h4><p id="p-39">Another data quality check examined the unfolded SFS for four SNP classes combined across the full dataset and partitioned into an X chromosome set and an autosomal set: (1) nonsynonymous, (2) synonymous, but not fourfold degenerate, (3) intergenic, and (4) a “neutral” class comprised of fourfold degenerate and small intronic SNPs (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The number of SNPs (~167,000) was balanced to match the limiting counts from the ‘neutral’ set.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Site frequency spectra for four single-nucleotide polymorphisms (SNP) classes. Unfolded (polarized for ancestral state) SFS are shown for the four SNP classes separated for the autosomes (left) and the X chromosome (right). Insets display the two extreme ends of the distributions where differences between the SNP classes are most pronounced." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Site frequency spectra for four single-nucleotide polymorphisms (SNP) classes. Unfolded (polarized for ancestral state) SFS are shown for the four SNP classes separated for the autosomes (left) and the X chromosome (right). Insets display the two extreme ends of the distributions where differences between the SNP classes are most pronounced.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F6.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/4/593/F6.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465806" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Site frequency spectra for four single-nucleotide polymorphisms (SNP) classes. Unfolded (polarized for ancestral state) SFS are shown for the four SNP classes separated for the autosomes (left) and the X chromosome (right). Insets display the two extreme ends of the distributions where differences between the SNP classes are most pronounced.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">The shapes of the SFS are consistent with the validation results (shown previously), showing the expected selection-driven enrichment of low frequency SNPs and characteristic decrease in abundance among more frequent variants. Among the rarest variants, there is a roughly 20% excess abundance of nonsynonymous SNPs (compared with synonymous SNPs), consistent with the role of purifying selection working to eliminate deleterious mutations from the population. The right tail of the SFS reflects the abundance of high frequency derived SNPs, and the excess (compared to a neutral expectation) seen for all classes of SNPs may reflect SNPs that have risen to high frequency either neutrally or due to selection (either because of direct selection or linkage to selected variants), motivating a more complete follow-up analysis.</p></div><div id="sec-11" class="subsection"><h4>Population-specific SFS and private variants:</h4><p id="p-42">Partitioning the SFS into population-specific SFS further highlights population differences (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS9.pdf">Figure S9</a>). For example, the larger effective population size of the Zimbabwe sample is apparent caused by the overall greater number of variant sites, both for the autosomes and the X chromosome. In particular, most of the contribution in the singleton class is provided by the Zimbabwe lines (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS9.pdf">Figure S9</a>). In addition, the abundance of high frequency−derived alleles varies depending on both the class of site and the population being examined. These differences are not unexpected given the complex interactions of demographic and selective forces, and are currently being investigated.</p><p id="p-43">A major interest of the field is lineage-specific differentiation and the extent to which such differences might be driven by local adaptation. As a first look into this question we extracted all SNPs that were private to a single population and that also had a relatively high allele frequency (≥20%). <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7, A−C</a> plots the counts for these alleles stratified by allele frequency bins. The finding of thousands of population-specific nonsynonymous, synonymous and UTR SNPs is highly unlikely, given a simple island model with the observed <em>F<sub>ST</sub></em> (these counts ranged from 1387 for Ithaca to 52,079 for Zimbabwe). These initial observations suggest a role for natural selection-driving patterns of differentiation and motivate a more thorough analysis of local adaptation detectable with these data.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the Number of Potentially Functional Population-specific single-nucleotide polymorphisms. High-frequency derived allele counts for genic (A and B) and UTR regions (C) at frequencies between 20% and 100% are plotted for five populations. All sites in the plot are based on the IBD and callability-masked SNP dataset, have " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of the Number of Potentially Functional Population-specific single-nucleotide polymorphisms. High-frequency derived allele counts for genic (A and B) and UTR regions (C) at frequencies between 20% and 100% are plotted for five populations. All sites in the plot are based on the IBD and callability-masked SNP dataset, have &lt;20% missing data, and carry &#x2265;70% posterior probability for their ancestral state. Panels (D) and (E) attempt to normalize the nonsynonymous and untranslated region (UTR) counts by showing the ratio to the putatively more neutral synonymous counts. IBD, identity by descent.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F7.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/4/593/F7.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465881" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-44" class="first-child">Summary of the Number of Potentially Functional Population-specific single-nucleotide polymorphisms. High-frequency derived allele counts for genic (A and B) and UTR regions (C) at frequencies between 20% and 100% are plotted for five populations. All sites in the plot are based on the IBD and callability-masked SNP dataset, have &lt;20% missing data, and carry ≥70% posterior probability for their ancestral state. Panels (D) and (E) attempt to normalize the nonsynonymous and untranslated region (UTR) counts by showing the ratio to the putatively more neutral synonymous counts. IBD, identity by descent.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Population-specific diversity and linkage disequilibrium (LD) estimates:</h4><p id="p-45">As indicated by the SFS previously, significant heterogeneity in genome-wide nucleotide diversity is observed across all populations (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Median genome-wide diversity levels (π) range from ~0.3% (Beijing) to ~0.6% (Zimbabwe); Watterson’s θ ranges from ~0.03% to ~0.07%. Consistent with previous studies, broad-scale patterns of within-genome diversity covary with local recombination intensity and display reduced diversity near the telomeres and centromeres (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>). In addition, the X chromosomes of all populations—except for Zimbabwe —display a marked reduction in diversity levels as compared with the normally recombining autosomes (excluding chromosome 4).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/593/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide diversity summaries. (A) Sliding window plots display summary statistics for the single-nucleotide polymorphism data along all four chromosome arms. The window size for chromosomes other than the 4th are 10 kb, with stride length equal to 5 kb; Windows for the small 4th chromosome are 500 bp with stride length equal 250 bp. Population abbreviations are: B = Beijing, I = Ithaca, N = Netherlands, T = Tasmania, and Z = Zimbabwe. &#x201C;Polymorphism&#x201D; refers to the average pairwise nucleotide difference (&#x3C0;), &#x201C;Poly/Div&#x201D; refers to the polymorphism divided by divergence, and &#x201C;TajD&#x201D; refers to Tajimas D. (B) Boxplots summarize the diversity data shown in (A) for all chromosomes. Heterogeneity in genome-wide nucleotide diversity (&#x3C0;) is observed across all populations (one-way analysis of variance, F4, 146050 = 2583.8, P " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1178467986" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide diversity summaries. (A) Sliding window plots display summary statistics for the single-nucleotide polymorphism data along all four chromosome arms. The window size for chromosomes other than the 4th are 10 kb, with stride length equal to 5 kb; Windows for the small 4th chromosome are 500 bp with stride length equal 250 bp. Population abbreviations are: B = Beijing, I = Ithaca, N = Netherlands, T = Tasmania, and Z = Zimbabwe. &#x201C;Polymorphism&#x201D; refers to the average pairwise nucleotide difference (&#x3C0;), &#x201C;Poly/Div&#x201D; refers to the polymorphism divided by divergence, and &#x201C;TajD&#x201D; refers to Tajimas &lt;em&gt;D&lt;/em&gt;. (B) Boxplots summarize the diversity data shown in (A) for all chromosomes. Heterogeneity in genome-wide nucleotide diversity (&#x3C0;) is observed across all populations (one-way analysis of variance, &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;4, 146050&lt;/sub&gt; = 2583.8, &lt;em&gt;P&lt;/em&gt; &lt;&lt; 0.01; all pairwise comparisons contribute significantly to this result {all Tukey post-hoc comparisons &lt;em&gt;P&lt;/em&gt; &lt; &lt; 0.01}) . The most notable difference is between the non-African populations&#x2019; and the Zimbabwe&#x2019;s X chromosomes. Non-African X chromosomes display 37&#x2013;46% less diversity within the X chromosome as compared with the normally recombining autosomes (Wilcoxon &lt;em&gt;P&lt;/em&gt; &lt;&lt; 0.01), whereas the Zimbabwe sample has a slight excess (2%). As a result, the autosomal-chromosome X comparison for Zimbabwe is nominally significant in the opposite direction (Wilcoxon &lt;em&gt;P&lt;/em&gt; = 0.011). Within population comparisons of Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt; values (&lt;em&gt;D&lt;/em&gt;) on the X &lt;em&gt;vs.&lt;/em&gt; autosomes are also significantly different (Wilcoxon &lt;em&gt;P&lt;/em&gt; &lt;&lt; 0.01 for all contrasts), as are most comparisons across populations (one-sample Wilcoxon &lt;em&gt;P&lt;/em&gt; &lt;&lt; 0.01). Tukey post-hoc comparisons indicate that only the Ithaca-Beijing X chromosome and the Tasmania-Ithaca autosome comparisons are nonsignificant (&lt;em&gt;P&lt;/em&gt; &gt; 0.05). Zimbabwe stands out as having a significantly negative mean autosomal &lt;em&gt;D&lt;/em&gt;, whereas for the X chromosome it is not different from zero (Wilcoxon &lt;em&gt;P&lt;/em&gt; &gt; 0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/593/F8.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/4/593/F8.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/593/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/593/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465799" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-46" class="first-child">Genome-wide diversity summaries. (A) Sliding window plots display summary statistics for the single-nucleotide polymorphism data along all four chromosome arms. The window size for chromosomes other than the 4th are 10 kb, with stride length equal to 5 kb; Windows for the small 4th chromosome are 500 bp with stride length equal 250 bp. Population abbreviations are: B = Beijing, I = Ithaca, N = Netherlands, T = Tasmania, and Z = Zimbabwe. “Polymorphism” refers to the average pairwise nucleotide difference (π), “Poly/Div” refers to the polymorphism divided by divergence, and “TajD” refers to Tajimas <em>D</em>. (B) Boxplots summarize the diversity data shown in (A) for all chromosomes. Heterogeneity in genome-wide nucleotide diversity (π) is observed across all populations (one-way analysis of variance, <em>F</em><sub>4, 146050</sub> = 2583.8, <em>P</em> &lt;&lt; 0.01; all pairwise comparisons contribute significantly to this result {all Tukey post-hoc comparisons <em>P</em> &lt; &lt; 0.01}) . The most notable difference is between the non-African populations’ and the Zimbabwe’s X chromosomes. Non-African X chromosomes display 37–46% less diversity within the X chromosome as compared with the normally recombining autosomes (Wilcoxon <em>P</em> &lt;&lt; 0.01), whereas the Zimbabwe sample has a slight excess (2%). As a result, the autosomal-chromosome X comparison for Zimbabwe is nominally significant in the opposite direction (Wilcoxon <em>P</em> = 0.011). Within population comparisons of Tajima’s <em>D</em> values (<em>D</em>) on the X <em>vs.</em> autosomes are also significantly different (Wilcoxon <em>P</em> &lt;&lt; 0.01 for all contrasts), as are most comparisons across populations (one-sample Wilcoxon <em>P</em> &lt;&lt; 0.01). Tukey post-hoc comparisons indicate that only the Ithaca-Beijing X chromosome and the Tasmania-Ithaca autosome comparisons are nonsignificant (<em>P</em> &gt; 0.05). Zimbabwe stands out as having a significantly negative mean autosomal <em>D</em>, whereas for the X chromosome it is not different from zero (Wilcoxon <em>P</em> &gt; 0.05).</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Within-population comparisons of Tajima’s <em>D</em> values (<em>D</em>) on the X <em>vs.</em> autosomes also consistently demonstrated significant differences between the chromosomes (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). Each of the four non-African populations has a significantly positive mean <em>D</em> value for both the autosomes and the X chromosome, consistent with past bottlenecks. Zimbabwe stands out as having significantly negative mean autosomal <em>D</em> values, and a mean <em>D</em> for chromosome X that is not different from zero, possibly indicating a recent population expansion (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Glinka <em>et al.</em> 2003</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ometto <em>et al.</em> 2005</a>). Historical population size differences can also be observed in differences in linkage disequilibrium (LD) over physical distance (<a href="http://www.g3journal.org/content/suppl/2015/02/11/g3.114.015883.DC1/FigureS10.pdf">Figure S10</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McVean 2002</a>): Zimbabwe has significantly less LD than the other populations (excluding chromosome 4), with Beijing and Netherlands possessing the highest levels of LD and Ithaca and Tasmania displaying intermediate levels. For all samples, the bulk of LD is lost within 100-150 bps, which is consistent with previous observations (<em>i.e.</em>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2014</a>).</p><p id="p-48">The aforementioned collection of preliminary population genetic analysis serve to show that the collection of Global Diversity Lines have genetic attributes consistent with known history of <em>Drosophila</em> phylogeography, and they serve as a reassuring parity check attesting to the accuracy of SNP genotype calling. More detailed analyses aimed at inferring demographic parameters for this data set will be submitted as an independent manuscript elsewhere.</p></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-49">An excellent starting point for developing a “systems biology”–based analysis of natural variation is to generate a reference set of inbred fully sequenced lines having maximal genetic diversity (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ayroles <em>et al.</em> 2009</a>). Toward this end, and to advance quantitative approaches relating genomic and phenotypic variation, we have characterized genomic variation among <em>D. melanogaster</em> sampled from five disparate populations. Most immediately, these data will provide a foundation for modeling complex traits such as metabolic regulation, a dynamic phenotype that varies between these populations (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Greenberg <em>et al.</em> 2010</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Scheitz <em>et al.</em> 2013</a>), and which is amendable to increasingly numerous and sensitive “omic” assays. Additionally, this collection of variants—and the available inbred lines—will broadly benefit a wide range of screens and assays seeking to understand geographically localized adaptations. For example, the whole-genome sequencing dataset has also been used to characterize <em>Wolbachia</em> and mitochondrial haplotypes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Early and Clark 2013</a>), repeat sequences (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wei <em>et al.</em> 2014</a>), and immune gene evolution (A. Early, J. R. Arguello, M. Cardoso-Moreira, S. Gottipati, J. K. Grenier, and A. G. Clark, unpublished data) in the Global Diversity Lines.</p><p id="p-50">Extensive effort has been invested into variant calling and empirical validation. The validation procedures that we have followed, involving the combination of orthogonally generated data, revealed notable context-dependent differences that do not allow for simple genome-wide variant call thresholding (<em>i.e.</em>, within and outside of heterozygous blocks, or for different genotype calls). In total this new data set is comprised of ~5.78 M SNPs and nearly 1 M small indels, providing a rich reservoir of naturally occurring variants with multiple mutations in every gene.</p><p id="p-51">The general agreement between our initial population genetic analysis and previous population genomic studies (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Begun and Aquadro 1993</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>) would seem to confirm that our data are of sufficient quality to allow for drawing population genetic inferences. The pronounced population structure among shared variants, particularly on the X chromosome, highlights the necessity of accounting for the underlying demographic processes that have led to genetic and phenotypic differences among these lines. As expected, our African sample carries the signatures of being ancestral to all of the non-African samples (high diversity and low LD; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Begun and Aquadro 1993</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Glinka <em>et al.</em> 2003</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Haddrill <em>et al.</em> 2005</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Ometto <em>et al.</em> 2005</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li and Stephan 2006</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Thornton and Andolfatto 2006</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Laurent <em>et al.</em> 2011</a>). In addition, Zimbabwe carries an excess of singletons (highlighted by the negative Tajima’s <em>D</em> values), which has been observed previously (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Glinka <em>et al.</em> 2003</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Ometto <em>et al.</em> 2005</a>). Regarding the non-African samples, the Asian sample was observed as the second most divergent population. Although there had previously been suspicion of an independent migration from Africa to Asia (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">David <em>et al.</em> 1976</a>), more recent data and demographic modeling argued against it (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baudry <em>et al.</em> 2004</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Schlötterer <em>et al.</em> 2006</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Laurent <em>et al.</em> 2011</a>). In line with a unique out-of-Africa origin, the Beijing sample is more closely related to the other non-African samples than to Zimbabwe. The range of diversity estimates for non-African genomes (0–1.5%), with a median below 0.5% is also consistent with previous reports (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Ometto <em>et al.</em> 2005</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Laurent <em>et al.</em> 2011</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>). However, we tend to see somewhat elevated intermediate allele frequencies (positive Tajima’s D) compared to some reports, possibly because this is the first global-scale genome-wide estimate, or possibly because of the laboratory inbreeding. But regardless of this issue, it is very clear that we find numerous population-specific variants—particularly derived alleles that are segregating at high frequency—highlighting the exciting opportunities to investigate cases of population-specific differences and local adaptation.</p><p id="p-52">Our experimental and bioinformatics investigation of the relationship between heterozygous blocks and inversions has provided an important focus on the high frequency and chromosome-scale impact of large inversions in <em>D. melanogaster</em>. The <em>de novo</em> generation and subsequent tolerance of inversions is increasingly recognized to be a major feature of this species’ genome (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Schlötterer <em>et al.</em> 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Langley <em>et al.</em> 2012</a>). The basic biology underlying this phenomenon is of fundamental interest but is also relevant for mapping and quantitative genetic modeling efforts within this species. For example, it is important to determine what portion of phenotypic variance in adaptive traits is mediated by genetic variation captured in these inversions. To fully understand the impact of the past demographic and selection histories of <em>D. melanogaster</em> in influencing the genetic basis of complex traits, it will be essential to characterize the population dynamics of inversions, including the age, geographic distribution, and impact of individual inversions and the modes of selection that maintain and modulate their frequency in natural populations. In any case, the wide geographic sampling of these lines makes them valuable for the investigation of global demography, inference of past action of diverse forms of natural selection, analysis of gene family evolution, and relating these patterns of variation and their inferred historical causes to measured phenotypic differences among the lines, and they serve as a complement to the already valuable sets of reference lines of <em>D. melanogaster</em> drawn from other natural populations.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank Julien Ayroles, Eric Stone, and Alex Coventry for advice about initial SNP calling approaches. This work was supported in part by National Institutes of Health grant R01 DK074136 to AGC and Lawrence G. Harshman. J.R.A. was supported in part by a Cornell Center for Comparative and Population Genomics Fellowship. M.C.-M. was supported by a postdoctoral fellowship from the Portuguese Foundation for Science and Technology (co-financed by POPH/FSE). J.M. was supported by National Institutes of Health grants R01-GM102192 (awarded to Adam C. Siepel) and R01-GM083300 (awarded to Eric C. Lai).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-54">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015883/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015883/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-55"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-11-25"><span class="received-label">Received </span>November 25, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-05"><span class="accepted-label">Accepted </span>February 5, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Grenier <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Unusual haplotype structure at the proximal breakpoint of In(2L)t in a natural population of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D153%26rft.issue%253D3%26rft.spage%253D1297%26rft.epage%253D1311%26rft.atitle%253DUnusual%2BHaplotype%2BStructure%2Bat%2Bthe%2BProximal%2BBreakpoint%2Bof%2BIn%25282L%2529t%2Bin%2Ba%2BNatural%2BPopulation%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F10545460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My8zLzEyOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.2" data-doi="10.1111/mec.12276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3179</span>–<span class="cit-lpage">3190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3179%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.3" data-doi="10.1038/ng.332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systems genetics of complex traits in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAyroles%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D307%26rft.atitle%253DSystems%2Bgenetics%2Bof%2Bcomplex%2Btraits%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.332%26rft_id%253Dinfo%253Apmid%252F19234471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19234471&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263640200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.4" data-doi="10.1093/molbev/msh089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudry</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viginier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veuille</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Non-African populations of <em>Drosophila melanogaster</em> have a unique origin.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1482</span>–<span class="cit-lpage">1491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh089%26rft_id%253Dinfo%253Apmid%252F15014160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTQ4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.5" data-doi="10.1038/365548a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">African and North American populations of <em>Drosophila melanogaster</em> are very different at the DNA level.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">548</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D365%26rft.issue%253D6446%26rft.spage%253D548%26rft.epage%253D550%26rft.atitle%253DAfrican%2Band%2BNorth%2BAmerican%2Bpopulations%2Bof%2BDrosophila%2Bmelanogaster%2Bare%2Bvery%2Bdifferent%2Bat%2Bthe%2BDNA%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F365548a0%26rft_id%253Dinfo%253Apmid%252F8413609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/365548a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8413609&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MA66100054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.6" data-doi="10.1371/journal.pcbi.1002458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">What can causal networks tell us about metabolic pathways?</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBlair%26rft.auinit1%253DR.%2BH.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1002458%26rft.epage%253De1002458%26rft.atitle%253DWhat%2Bcan%2Bcausal%2Bnetworks%2Btell%2Bus%2Babout%2Bmetabolic%2Bpathways%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002458%26rft_id%253Dinfo%253Apmid%252F22496633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22496633&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.7" data-doi="10.1371/journal.pgen.1004142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breunig</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic basis of metabolome variation in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004142%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004142%26rft_id%253Dinfo%253Apmid%252F24603560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24603560&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.8" data-doi="10.1186/gb-2012-13-12-r119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arguello</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutation spectrum of <em>Drosophila</em> CNVs revealed by breakpoint sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D13%26rft.spage%253DR119%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-12-r119%26rft_id%253Dinfo%253Apmid%252F23259534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-12-r119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23259534&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.9" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.10" data-doi="10.1371/journal.pgen.1003056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of inversion polymorphisms in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003056%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003056%26rft_id%253Dinfo%253Apmid%252F23284285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284285&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.11" data-doi="10.1534/genetics.112.141622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequence-based detection and breakpoint assembly of polymorphic inversions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141622%26rft_id%253Dinfo%253Apmid%252F22673805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.12" data-doi="10.1111/mec.12084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes-Utrilla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Special features of RAD sequencing data: implications for genotyping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3151</span>–<span class="cit-lpage">3164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3151%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.13" data-doi="10.1017/S0016672300016943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bocquet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pla</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">New results on the genetic characteristics of the Far East race of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDavid%26rft.auinit1%253DJ.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D253%26rft.epage%253D260%26rft.atitle%253DNew%2Bresults%2Bon%2Bthe%2Bgenetic%2Bcharacteristics%2Bof%2Bthe%2BFar%2BEast%2Brace%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300016943%26rft_id%253Dinfo%253Apmid%252F829088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300016943&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=829088&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976CW39900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.14" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.15" data-doi="10.1242/jeb.046458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eanes</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular population genetics and selection in the glycolytic pathway.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DEanes%26rft.auinit1%253DW.%2BF.%26rft.volume%253D214%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D171%26rft.atitle%253DMolecular%2Bpopulation%2Bgenetics%2Band%2Bselection%2Bin%2Bthe%2Bglycolytic%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.046458%26rft_id%253Dinfo%253Apmid%252F21177937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjk6IjIxNC8yLzE2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.16" data-doi="10.1111/mec.12530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Early</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Monophyly of Wolbachia pipientis genomes within <em>Drosophila melanogaster</em>: geographic structuring, titre variation and host effects across five populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">5765</span>–<span class="cit-lpage">5778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D5765%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000327278700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.17" data-doi="10.1111/mec.12360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foucaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimation of population allele frequencies from next-generation sequencing data: pool-<em>vs.</em> individual-based genotyping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3766</span>–<span class="cit-lpage">3779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3766%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000321552800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.18" data-doi="10.15252/msb.20135004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shih</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic regulation of mouse liver metabolite levels.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D730%26rft.epage%253D730%26rft.atitle%253DGenetic%2Bregulation%2Bof%2Bmouse%2Bliver%2Bmetabolite%2Blevels%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20135004%26rft_id%253Dinfo%253Apmid%252F24860088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEwLzUvNzMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glinka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ometto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mousset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Lorenzo</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Demography and natural selection have shaped genetic variation in <em>Drosophila melanogaster:</em> a multi-locus approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGlinka%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1269%26rft.epage%253D1278%26rft.atitle%253DDemography%2Band%2BNatural%2BSelection%2BHave%2BShaped%2BGenetic%2BVariation%2Bin%2BDrosophila%2Bmelanogaster%253A%2BA%2BMulti-locus%2BApproach%26rft_id%253Dinfo%253Apmid%252F14668381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzEyNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.20" data-doi="10.1534/genetics.110.115055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harshman</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A hierarchical Bayesian model for a novel sparse partial diallel crossing design.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115055%26rft_id%253Dinfo%253Apmid%252F20157001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzEvMzYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harshman</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Environmental and genetic perturbations reveal different networks of metabolic regulation.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D7%26rft.spage%253D563%26rft_id%253Dinfo%253Apmid%252F22186737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22186737&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.22" data-doi="10.1101/gr.3541005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Multilocus patterns of nucleotide variability and the demographic and selection history of <em>Drosophila melanogaster</em> populations.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">790</span>–<span class="cit-lpage">799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHaddrill%26rft.auinit1%253DP.%2BR.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D790%26rft.epage%253D799%26rft.atitle%253DMultilocus%2Bpatterns%2Bof%2Bnucleotide%2Bvariability%2Band%2Bthe%2Bdemographic%2Band%2Bselection%2Bhistory%2Bof%2BDrosophila%2Bmelanogaster%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3541005%26rft_id%253Dinfo%253Apmid%252F15930491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzYvNzkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.23" data-doi="10.1101/gr.141689.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A second-generation assembly of the <em>Drosophila simulans</em> genome provides new insights into patterns of lineage-specific divergence.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.141689.112%26rft_id%253Dinfo%253Apmid%252F22936249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjIzLzEvODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.24" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ràmia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 <em>Drosophila melanogaster</em> Genetic Reference Panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.25" data-doi="10.1186/1471-2164-11-297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jumbo-Lucioni</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leips</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Systems genetics analysis of body weight and energy metabolism traits in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DJumbo-Lucioni%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.spage%253D297%26rft.epage%253D297%26rft.atitle%253DSystems%2Bgenetics%2Banalysis%2Bof%2Bbody%2Bweight%2Band%2Benergy%2Bmetabolism%2Btraits%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-297%26rft_id%253Dinfo%253Apmid%252F20459830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20459830&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.26" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT—the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.27" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic dissection of a model complex trait using the <em>Drosophila</em> Synthetic Population Resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.28" data-doi="10.1534/genetics.111.127530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Circumventing heterozygosity: sequencing the amplified genome of a single haploid <em>Drosophila melanogaster</em> embryo.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127530%26rft_id%253Dinfo%253Apmid%252F21441209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvMjM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.29" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation in natural populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.30" data-doi="10.1093/molbev/msr031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">S. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werzner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Approximate Bayesian analysis of <em>Drosophila melanogaster</em> polymorphism data reveals a recent colonization of Southeast Asia.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2041</span>–<span class="cit-lpage">2051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr031%26rft_id%253Dinfo%253Apmid%252F21300986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzcvMjA0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.31" data-doi="10.1093/molbev/msu146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavington</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogni</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuczynski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koury</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behrman</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A small system–high-resolution study of metabolic adaptation in the central metabolic pathway to temperate climates in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2032</span>–<span class="cit-lpage">2041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu146%26rft_id%253Dinfo%253Apmid%252F24770333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzgvMjAzMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.32" data-doi="10.1371/journal.pone.0090581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stromberg</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MOSAIK: a hash-based algorithm for accurate next-generation sequencing short-read mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90581%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090581%26rft_id%253Dinfo%253Apmid%252F24599324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24599324&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.593.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemeunier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aulard</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Inversion polymorphism in <em>Drosophila melanogaster</em></span>, pp. <span class="cit-fpage">339</span>–<span class="cit-lpage">406</span> in <span class="cit-source">Drosophila Inversion Polymorphism</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Krimbas</span>  <span class="cit-name-given-names">C. B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.34" data-doi="10.1371/journal.pgen.0020166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inferring the demographic history and rate of adaptive substitution in Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De166%26rft.epage%253De166%26rft.atitle%253DInferring%2Bthe%2Bdemographic%2Bhistory%2Band%2Brate%2Bof%2Badaptive%2Bsubstitution%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020166%26rft_id%253Dinfo%253Apmid%252F17040129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17040129&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.35" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.36" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> Genetic Reference Panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.37" data-doi="10.1371/journal.pgen.1003055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waszak</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albarca-Aguilera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holcombe</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation and its impact on gene expression in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMassouras%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003055%26rft.epage%253De1003055%26rft.atitle%253DGenomic%2BVariation%2Band%2BIts%2BImpact%2Bon%2BGene%2BExpression%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003055%26rft_id%253Dinfo%253Apmid%252F23189034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23189034&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.38" data-doi="10.1534/genetics.104.038810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matzkin</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merritt</span>  <span class="cit-name-given-names">T. J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eanes</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The structure and population genetics of the breakpoints associated with the cosmopolitan chromosomal inversion In(3R)Payne in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038810%26rft_id%253Dinfo%253Apmid%252F15781702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzExNDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.39" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A genealogical interpretation of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D162%26rft.issue%253D2%26rft.spage%253D987%26rft.epage%253D991%26rft.atitle%253DA%2BGenealogical%2BInterpretation%2Bof%2BLinkage%2BDisequilibrium%26rft_id%253Dinfo%253Apmid%252F12399406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzIvOTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.41" data-doi="10.1093/molbev/msi207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ometto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glinka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Lorenzo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Inferring the effects of demography and selection on <em>Drosophila melanogaster</em> populations from a chromosome-wide scan of DNA variation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2119</span>–<span class="cit-lpage">2130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi207%26rft_id%253Dinfo%253Apmid%252F15987874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8yMTE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.42" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.43" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of sub-Saharan <em>Drosophila melanogaster</em>: African diversity and non-African admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.44" data-doi="10.1093/bioinformatics/bts378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zichner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stütz</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DELLY: structural variant discovery by integrated paired-end and split-read analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">i333</span>–<span class="cit-lpage">i339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts378%26rft_id%253Dinfo%253Apmid%252F22962449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvaTMzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.45" data-doi="10.1371/journal.pone.0072726"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheitz</span>  <span class="cit-name-given-names">C. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Early</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harshman</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heritability and inter-population differences in lipid profiles of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e72726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De72726%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0072726%26rft_id%253Dinfo%253Apmid%252F24013349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0072726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24013349&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F593.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.46" data-doi="10.1534/genetics.105.045831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumeier</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Highly structured Asian <em>Drosophila melanogaster</em> populations: a new tool for hitchhiking mapping?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.045831%26rft_id%253Dinfo%253Apmid%252F16204221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvMjg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.47" data-doi="10.1534/genetics.105.048223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1607</span>–<span class="cit-lpage">1619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048223%26rft_id%253Dinfo%253Apmid%252F16299396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE2MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.48" data-doi="10.1073/pnas.1421951112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">K. H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbash</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Correlated variation and population differentiation in satellite DNA abundance among lines of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">18793</span>–<span class="cit-lpage">18798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1421951112%26rft_id%253Dinfo%253Apmid%252F25512552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzUyLzE4NzkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.593.49" data-doi="10.1073/pnas.91.8.3132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wesley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eanes</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Isolation and analysis of the breakpoint sequences of chromosome inversion In(3L)Payne in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3132</span>–<span class="cit-lpage">3136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWesley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D91%26rft.issue%253D8%26rft.spage%253D3132%26rft.epage%253D3136%26rft.atitle%253DIsolation%2Band%2Banalysis%2Bof%2Bthe%2Bbreakpoint%2Bsequences%2Bof%2Bchromosome%2Binversion%2BIn%25283L%2529Payne%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.8.3132%26rft_id%253Dinfo%253Apmid%252F8159716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5MS84LzMxMzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/593.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/583" title="Adaptive Evolution of Genes Involved in the Regulation of Germline Stem Cells in Drosophila melanogaster and D. simulans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/605" title="Gene Regulation by H-NS as a Function of Growth Conditions Depends on Chromosomal Position in Escherichia coli" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/593" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-22ecYHFrin771CDoFGT-sS40bzvsWjxe4g6iLKMV1vQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/593" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/593" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/593" class="active" data-icon-position="" data-hide-link-title="0">Global Diversity Lines–A Five-Continent Reference Panel of Sequenced Drosophila melanogaster Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459810" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HoXDzqkAxVu3HiSNc1Qks2nfJc8qSdtZk22zbiAFxqA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/593.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/593" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-mix5IeVkW1er9Ujpj7T0eLmggQdchvh8-Uxu63_VVzs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/593" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/593.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459810" id="node459810--2" data-pisa="ggg;5/4/593" data-pisa-master="ggg;g3.114.015883" data-apath="/ggg/5/4/593.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Global Diversity Lines–A Five-Continent Reference Panel of Sequenced <em>Drosophila melanogaster</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennifer K. Grenier</span>, <span class="highwire-citation-author" data-delta="1">J. Roman Arguello</span>, <span class="highwire-citation-author" data-delta="2">Margarida Cardoso Moreira</span>, <span class="highwire-citation-author" data-delta="3">Srikanth Gottipati</span>, <span class="highwire-citation-author" data-delta="4">Jaaved Mohammed</span>, <span class="highwire-citation-author" data-delta="5">Sean R. Hackett</span>, <span class="highwire-citation-author" data-delta="6">Rachel Boughton</span>, <span class="highwire-citation-author" data-delta="7">Anthony J. Greenberg</span> and <span class="highwire-citation-author" data-delta="8">Andrew G. Clark</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">593-603;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015883 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459810--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennifer K. Grenier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BK.%2BGrenier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grenier%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BK.%2BGrenier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">J. Roman Arguello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BRoman%2BArguello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arguello%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BRoman%2BArguello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Margarida Cardoso Moreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margarida%2BCardoso%2BMoreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreira%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargarida%2BCardoso%2BMoreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Srikanth Gottipati </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Translational Medicine and Think Team, Otsuka Pharmaceutical Development and Commercialization, Inc., Princeton, New Jersey 08540</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikanth%2BGottipati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gottipati%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikanth%2BGottipati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaaved Mohammed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaaved%2BMohammed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammed%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaaved%2BMohammed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean R. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Quantitative and Computational Biology, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BR.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BR.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rachel Boughton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BBoughton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boughton%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BBoughton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Anthony J. Greenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BJ.%2BGreenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenberg%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BJ.%2BGreenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andrew G. Clark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BG.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clark%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BG.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ac347@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ac347@cornell.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/593" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/593&amp;title=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/593&amp;title=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/593&amp;title=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/593&amp;text=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/593&amp;title=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/593&amp;t=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/593&amp;title=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/593&amp;title=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459810/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459810" id="node459810--3" data-pisa="ggg;5/4/593" data-pisa-master="ggg;g3.114.015883" data-apath="/ggg/5/4/593.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Global Diversity Lines–A Five-Continent Reference Panel of Sequenced <em>Drosophila melanogaster</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennifer K. Grenier</span>, <span class="highwire-citation-author" data-delta="1">J. Roman Arguello</span>, <span class="highwire-citation-author" data-delta="2">Margarida Cardoso Moreira</span>, <span class="highwire-citation-author" data-delta="3">Srikanth Gottipati</span>, <span class="highwire-citation-author" data-delta="4">Jaaved Mohammed</span>, <span class="highwire-citation-author" data-delta="5">Sean R. Hackett</span>, <span class="highwire-citation-author" data-delta="6">Rachel Boughton</span>, <span class="highwire-citation-author" data-delta="7">Anthony J. Greenberg</span> and <span class="highwire-citation-author" data-delta="8">Andrew G. Clark</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">593-603;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015883 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459810--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennifer K. Grenier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BK.%2BGrenier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grenier%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BK.%2BGrenier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">J. Roman Arguello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BRoman%2BArguello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arguello%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BRoman%2BArguello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Margarida Cardoso Moreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Integrative Genomics, University of Lausanne, Lausanne, Switzerland CH-1015</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margarida%2BCardoso%2BMoreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreira%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargarida%2BCardoso%2BMoreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Srikanth Gottipati </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Translational Medicine and Think Team, Otsuka Pharmaceutical Development and Commercialization, Inc., Princeton, New Jersey 08540</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikanth%2BGottipati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gottipati%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikanth%2BGottipati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jaaved Mohammed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaaved%2BMohammed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammed%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaaved%2BMohammed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean R. Hackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Quantitative and Computational Biology, Princeton University, Princeton, New Jersey 08544</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BR.%2BHackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hackett%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BR.%2BHackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rachel Boughton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BBoughton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boughton%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachel%2BBoughton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Anthony J. Greenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BJ.%2BGreenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenberg%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BJ.%2BGreenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andrew G. Clark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biological Statistics and Computational Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BG.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clark%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BG.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ac347@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ac347@cornell.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=593&amp;rft.epage=603&amp;rft.atitle=Global%20Diversity%20Lines%E2%80%93A%20Five-Continent%20Reference%20Panel%20of%20Sequenced%20Drosophila%20melanogaster%20Strains&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Grenier%2C+Jennifer+K.&amp;rft.au=Arguello%2C+J.+Roman&amp;rft.au=Moreira%2C+Margarida+Cardoso&amp;rft.au=Gottipati%2C+Srikanth&amp;rft.au=Mohammed%2C+Jaaved&amp;rft.au=Hackett%2C+Sean+R.&amp;rft.au=Boughton%2C+Rachel&amp;rft.au=Greenberg%2C+Anthony+J.&amp;rft.au=Clark%2C+Andrew+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459810/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459810/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459810/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459810/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459810/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459810/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459810/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459810/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459810/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459810/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459810/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459810/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/593" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/593.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/593.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/593.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/593.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
