<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/615" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/615.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/615.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/615.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/4/615/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/615.atom" />
<meta name="HW.pisa" content="ggg;5/4/615" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix" />
<meta name="DC.Identifier" content="10.1534/g3.114.016261" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Zhang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Obtaining accurate predictions of unobserved genetic or phenotypic values for complex traits in animal, plant, and human populations is possible through whole-genome prediction (WGP), a combined analysis of genotypic and phenotypic data. Because the underlying genetic architecture of the trait of interest is an important factor affecting model selection, we propose a new strategy, termed BLUP|GA (BLUP-given genetic architecture), which can use genetic architecture information within the dataset at hand rather than from public sources. This is achieved by using a trait-specific covariance matrix ( T ), which is a weighted sum of a genetic architecture part ( S matrix) and the realized relationship matrix ( G ). The algorithm of BLUP|GA (BLUP-given genetic architecture) is provided and illustrated with real and simulated datasets. Predictive ability of BLUP|GA was validated with three model traits in a dairy cattle dataset and 11 traits in three public datasets with a variety of genetic architectures and compared with GBLUP and other approaches. Results show that BLUP|GA outperformed GBLUP in 20 of 21 scenarios in the dairy cattle dataset and outperformed GBLUP, BayesA, and BayesB in 12 of 13 traits in the analyzed public datasets. Further analyses showed that the difference of accuracies for BLUP|GA and GBLUP significantly correlate with the distance between the T and G matrices. The new strategy applied in BLUP|GA is a favorable and flexible alternative to the standard GBLUP model, allowing to account for the genetic architecture of the quantitative trait under consideration when necessary. This feature is mainly due to the increased similarity between the trait-specific relationship matrix ( T matrix) and the genetic relationship matrix at unobserved causal loci. Applying BLUP|GA in WGP would ease the burden of model selection." />
<meta name="DC.Contributor" content="Zhe Zhang" />
<meta name="DC.Contributor" content="Malena Erbe" />
<meta name="DC.Contributor" content="Jinlong He" />
<meta name="DC.Contributor" content="Ulrike Ober" />
<meta name="DC.Contributor" content="Ning Gao" />
<meta name="DC.Contributor" content="Hao Zhang" />
<meta name="DC.Contributor" content="Henner Simianer" />
<meta name="DC.Contributor" content="Jiaqi Li" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Obtaining accurate predictions of unobserved genetic or phenotypic values for complex traits in animal, plant, and human populations is possible through whole-genome prediction (WGP), a combined analysis of genotypic and phenotypic data. Because the underlying genetic architecture of the trait of interest is an important factor affecting model selection, we propose a new strategy, termed BLUP|GA (BLUP-given genetic architecture), which can use genetic architecture information within the dataset at hand rather than from public sources. This is achieved by using a trait-specific covariance matrix (&lt;b&gt;T&lt;/b&gt;), which is a weighted sum of a genetic architecture part (&lt;b&gt;S&lt;/b&gt; matrix) and the realized relationship matrix (&lt;b&gt;G&lt;/b&gt;). The algorithm of BLUP|GA (BLUP-given genetic architecture) is provided and illustrated with real and simulated datasets. Predictive ability of BLUP|GA was validated with three model traits in a dairy cattle dataset and 11 traits in three public datasets with a variety of genetic architectures and compared with GBLUP and other approaches. Results show that BLUP|GA outperformed GBLUP in 20 of 21 scenarios in the dairy cattle dataset and outperformed GBLUP, BayesA, and BayesB in 12 of 13 traits in the analyzed public datasets. Further analyses showed that the difference of accuracies for BLUP|GA and GBLUP significantly correlate with the distance between the &lt;b&gt;T&lt;/b&gt; and &lt;b&gt;G&lt;/b&gt; matrices. The new strategy applied in BLUP|GA is a favorable and flexible alternative to the standard GBLUP model, allowing to account for the genetic architecture of the quantitative trait under consideration when necessary. This feature is mainly due to the increased similarity between the trait-specific relationship matrix (&lt;b&gt;T&lt;/b&gt; matrix) and the genetic relationship matrix at unobserved causal loci. Applying BLUP|GA in WGP would ease the burden of model selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/615" />
<meta name="citation_id" content="5/4/615" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/615" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/615.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/615.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/615.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016261" />
<meta name="citation_pmid" content="25670771" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="615" />
<meta name="citation_lastpage" content="627" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhe Zhang" />
<meta name="citation_author_institution" content="National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany" />
<meta name="citation_author" content="Malena Erbe" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany" />
<meta name="citation_author" content="Jinlong He" />
<meta name="citation_author_institution" content="National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China" />
<meta name="citation_author" content="Ulrike Ober" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany" />
<meta name="citation_author" content="Ning Gao" />
<meta name="citation_author_institution" content="National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China" />
<meta name="citation_author" content="Hao Zhang" />
<meta name="citation_author_institution" content="National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China" />
<meta name="citation_author" content="Henner Simianer" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany" />
<meta name="citation_author_email" content="hsimian@gwdg.de" />
<meta name="citation_author_email" content="jqli@scau.edu.cn" />
<meta name="citation_author" content="Jiaqi Li" />
<meta name="citation_author_institution" content="National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China" />
<meta name="citation_author_email" content="hsimian@gwdg.de" />
<meta name="citation_author_email" content="jqli@scau.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=M. A. Cleveland;citation_author=J. M. Hickey;citation_author=S. Forni;citation_title=A common dataset for genomic analysis of livestock populations.;citation_pages=429-435;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. J. de Koning;citation_author=L. McIntyre;citation_title=Setting the Standard: A Special Focus on Genomic Selection in GENETICS and G3;citation_pages=1151-1152;citation_volume=190;citation_year=2012;citation_issue=4;citation_pmid=22491887;citation_doi=10.1534/genetics.112.139907" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=D. B. Allison;citation_title=Predicting genetic predisposition in humans: the promise of whole-genome markers.;citation_pages=880-886;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21045869;citation_doi=10.1038/nrg2898" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=X. Ding;citation_author=Z. Zhang;citation_author=X. Li;citation_author=S. Wang;citation_author=X. Wu;citation_title=Accuracy of genomic prediction for milk production traits in the Chinese Holstein population using a reference population consisting of cows.;citation_pages=5315-5323;citation_volume=96;citation_year=2013;citation_pmid=23746588;citation_doi=10.3168/jds.2012-6194" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Dudbridge;citation_title=Power and predictive accuracy of polygenic risk scores.;citation_pages=e1003348;citation_volume=9;citation_year=2013;citation_pmid=23555274;citation_doi=10.1371/journal.pgen.1003348" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=J. van Heerwaarden;citation_author=J. L. Wegrzyn;citation_author=C. D. Nelson;citation_author=J. Ross-Ibarra;citation_title=Patterns of population structure and environmental associations to aridity across the range of loblolly pine (Pinus taeda L., Pinaceae).;citation_volume=185;citation_year=2010;citation_pmid=20439779;citation_doi=10.1534/genetics.110.115543" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=H. Eding;citation_author=T. H. E. Meuwissen;citation_title=Marker-based estimates of between and within population kinships for the conservation of genetic diversity.;citation_pages=141-159;citation_volume=118;citation_year=2001;citation_doi=10.1046/j.1439-0388.2001.00290.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=H. Gao;citation_author=O. F. Christensen;citation_author=P. Madsen;citation_author=U. S. Nielsen;citation_author=Y. Zhang;citation_title=Comparison on genomic predictions using three GBLUP methods and two single-step blending methods in the Nordic Holstein population.;citation_pages=8-8;citation_volume=44;citation_year=2012;citation_pmid=22455934;citation_doi=10.1186/1297-9686-44-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the Bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=Z. Guo;citation_author=D. M. Tucker;citation_author=D. Wang;citation_author=C. J. Basten;citation_author=E. Ersoz;citation_title=Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations;citation_pages=263-272;citation_volume=3;citation_year=2013;citation_issue=2;citation_pmid=23390602;citation_doi=10.1534/g3.112.005066" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Hayes;citation_author=J. Pryce;citation_author=A. J. Chamberlain;citation_author=P. J. Bowman;citation_author=M. E. Goddard;citation_title=Genetic architecture of complex traits and accuracy of genomic prediction: coat colour, milk-fat percentage, and type in Holstein cattle as contrasting model traits.;citation_pages=e1001139-e1001139;citation_volume=6;citation_year=2010;citation_issue=9;citation_pmid=20927186;citation_doi=10.1371/journal.pgen.1001139" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=C. R. Henderson;citation_title=Rapid Method for Computing the Inverse of a Relationship Matrix;citation_pages=1727-1730;citation_volume=58;citation_year=1975;citation_issue=11;citation_doi=10.3168/jds.S0022-0302(75)84776-X" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=J. M. Hickey;citation_author=G. Gorjanc;citation_title=Simulated data for genomic selection and genome-wide association studies using a combination of coalescent and gene drop methods.;citation_pages=425-427;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. L. Hu;citation_author=C. A. Park;citation_author=X. L. Wu;citation_author=J. M. Reecy;citation_title=Animal QTLdb: an improved database tool for livestock animal QTL/association data dissemination in the post-genome era.;citation_volume=41;citation_year=2013;citation_pmid=23180796;citation_doi=10.1093/nar/gks1150" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=H. N. Kadarmideen;citation_title=Genomics to systems biology in animal and veterinary sciences: progress, lessons and opportunities.;citation_pages=232-248;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.04.028" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. H. Lee;citation_author=J. H. van der Werf;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Predicting unobserved phenotypes for complex traits from whole-genome SNP data.;citation_pages=e1000231-e1000231;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18949033;citation_doi=10.1371/journal.pgen.1000231" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=Z. Liu;citation_author=F. Seefried;citation_author=F. Reinhardt;citation_author=S. Rensing;citation_author=G. Thaller;citation_title=Impacts of both reference population size and inclusion of a residual polygenic effect on the accuracy of genomic prediction.;citation_pages=19-19;citation_volume=43;citation_year=2011;citation_pmid=21586131;citation_doi=10.1186/1297-9686-43-19" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. S. Lund;citation_author=A. P. de Roos;citation_author=A. G. de Vries;citation_author=T. Druet;citation_author=V. Ducrocq;citation_title=A common reference population from four European Holstein populations increases reliability of genomic predictions.;citation_pages=43-43;citation_volume=43;citation_year=2011;citation_pmid=22152008;citation_doi=10.1186/1297-9686-43-43" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. K. Matukumalli;citation_author=C. T. Lawley;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=M. F. Allan;citation_title=Development and characterization of a high density SNP genotyping assay for cattle.;citation_pages=e5350-e5350;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390634;citation_doi=10.1371/journal.pone.0005350" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Meuwissen;citation_author=M. Goddard;citation_title=Accurate prediction of genetic values for complex traits by whole-genome resequencing.;citation_volume=185;citation_year=2010;citation_pmid=20308278;citation_doi=10.1534/genetics.110.116590" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Misztal;citation_author=A. Legarra;citation_author=I. Aguilar;citation_title=Computing procedures for genetic evaluation including phenotypic, full pedigree, and genomic information;citation_pages=4648-4655;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700728;citation_doi=10.3168/jds.2009-2064" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=U. Ober;citation_author=J. F. Ayroles;citation_author=E. A. Stone;citation_author=S. Richards;citation_author=D. H. Zhu;citation_title=Using whole-genome sequence data to predict quantitative trait phenotypes in Drosophila melanogaster.;citation_pages=e1002685-e1002685;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22570636;citation_doi=10.1371/journal.pgen.1002685" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. J. Resende;citation_author=P. Muñoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard dataset of loblolly pine (Pinus taeda L.).;citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Riedelsheimer;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines.;citation_pages=452-452;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22947126;citation_doi=10.1186/1471-2164-13-452" />
<meta name="citation_reference" content="citation_title=J. Roy. Stat. Soc. Ser. B. (Stat. Method.);citation_author=M. Stone;citation_title=Cross-validatory choice and assessment of statistical predictions.;citation_pages=111-147;citation_volume=36;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Strandén;citation_author=D. J. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=I. Stranden;citation_author=O. F. Christensen;citation_title=Allele coding in genomic evaluation.;citation_pages=25-25;citation_volume=43;citation_year=2011;citation_pmid=21703021;citation_doi=10.1186/1297-9686-43-25" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. I. Vazquez;citation_author=G. de los Campos;citation_author=Y. C. Klimentidis;citation_author=G. J. Rosa;citation_author=D. Gianola;citation_title=A comprehensive genetic approach for improving prediction of skin cancer risk in humans.;citation_volume=192;citation_year=2012;citation_pmid=23051645;citation_doi=10.1534/genetics.112.141705" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=H. Wang;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=A. Legarra;citation_author=R. L. Fernando;citation_title=Genome-wide association mapping including phenotypes from relatives without genotypes in a single-step (ssGWAS) for 6-week body weight in broiler chickens.;citation_pages=134;citation_volume=5;citation_year=2014;citation_pmid=24904635" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. L. Wilcox;citation_author=H. V. Amerson;citation_author=E. G. Kuhlman;citation_author=B. H. Liu;citation_author=D. M. O’Malley;citation_title=Detection of a major gene for resistance to fusiform rust disease in loblolly pine by genomic mapping.;citation_pages=3859-3864;citation_volume=93;citation_year=1996;citation_issue=9;citation_pmid=8632980;citation_doi=10.1073/pnas.93.9.3859" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Wimmer;citation_author=C. Lehermeier;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=Y. Wang;citation_title=Genome-wide prediction of traits with different genetic architecture through efficient variable selection.;citation_volume=195;citation_year=2013;citation_pmid=23934883;citation_doi=10.1534/genetics.113.150078" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. Wolc;citation_author=C. Stricker;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. E. Fulton;citation_title=Breeding value prediction for production traits in layer chickens using pedigree or genomic relationships in a reduced animal model.;citation_pages=5-5;citation_volume=43;citation_year=2011;citation_pmid=21255418;citation_doi=10.1186/1297-9686-43-5" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=J. F. Liu;citation_author=X. D. Ding;citation_author=P. Bijma;citation_author=D. J. de Koning;citation_title=Best linear unbiased prediction of genomic breeding values using trait-specific marker-derived relationship matrix.;citation_pages=e12648;citation_volume=5;citation_year=2010;citation_pmid=20844593;citation_doi=10.1371/journal.pone.0012648" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=Z. Zhang;citation_author=X. D. Ding;citation_author=J. F. Liu;citation_author=Q. Zhang;citation_author=D.-J. de Koning;citation_title=Accuracy of genomic prediction using low-density marker panels.;citation_pages=3642-3650;citation_volume=94;citation_year=2011;citation_issue=7;citation_pmid=21700054;citation_doi=10.3168/jds.2010-3917" />
<meta name="citation_reference" content="citation_journal_title=Chin. Sci. Bull.;citation_author=Z. Zhang;citation_author=Q. Zhang;citation_author=X. D. Ding;citation_title=Advances in genomic selection in domestic animal.;citation_pages=2655-2663;citation_volume=56;citation_year=2011;citation_doi=10.1007/s11434-011-4632-7" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=U. Ober;citation_author=M. Erbe;citation_author=H. Zhang;citation_author=N. Gao;citation_title=Improving the accuracy of whole genome prediction for complex traits using the results of genome wide association studies.;citation_pages=e93017;citation_volume=9;citation_year=2014;citation_pmid=24663104;citation_doi=10.1371/journal.pone.0093017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Obtaining accurate predictions of unobserved genetic or phenotypic values for complex traits in animal, plant, and human populations is possible through whole-genome prediction (WGP), a combined analysis of genotypic and phenotypic data. Because the underlying genetic architecture of the trait of interest is an important factor affecting model selection, we propose a new strategy, termed BLUP|GA (BLUP-given genetic architecture), which can use genetic architecture information within the dataset at hand rather than from public sources. This is achieved by using a trait-specific covariance matrix ( T ), which is a weighted sum of a genetic architecture part ( S matrix) and the realized relationship matrix ( G ). The algorithm of BLUP|GA (BLUP-given genetic architecture) is provided and illustrated with real and simulated datasets. Predictive ability of BLUP|GA was validated with three model traits in a dairy cattle dataset and 11 traits in three public datasets with a variety of genetic architectures and compared with GBLUP and other approaches. Results show that BLUP|GA outperformed GBLUP in 20 of 21 scenarios in the dairy cattle dataset and outperformed GBLUP, BayesA, and BayesB in 12 of 13 traits in the analyzed public datasets. Further analyses showed that the difference of accuracies for BLUP|GA and GBLUP significantly correlate with the distance between the T and G matrices. The new strategy applied in BLUP|GA is a favorable and flexible alternative to the standard GBLUP model, allowing to account for the genetic architecture of the quantitative trait under consideration when necessary. This feature is mainly due to the increased similarity between the trait-specific relationship matrix ( T matrix) and the genetic relationship matrix at unobserved causal loci. Applying BLUP|GA in WGP would ease the burden of model selection." />
<meta name="og-title" property="og:title" content="Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/615" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Obtaining accurate predictions of unobserved genetic or phenotypic values for complex traits in animal, plant, and human populations is possible through whole-genome prediction (WGP), a combined analysis of genotypic and phenotypic data. Because the underlying genetic architecture of the trait of interest is an important factor affecting model selection, we propose a new strategy, termed BLUP|GA (BLUP-given genetic architecture), which can use genetic architecture information within the dataset at hand rather than from public sources. This is achieved by using a trait-specific covariance matrix ( T ), which is a weighted sum of a genetic architecture part ( S matrix) and the realized relationship matrix ( G ). The algorithm of BLUP|GA (BLUP-given genetic architecture) is provided and illustrated with real and simulated datasets. Predictive ability of BLUP|GA was validated with three model traits in a dairy cattle dataset and 11 traits in three public datasets with a variety of genetic architectures and compared with GBLUP and other approaches. Results show that BLUP|GA outperformed GBLUP in 20 of 21 scenarios in the dairy cattle dataset and outperformed GBLUP, BayesA, and BayesB in 12 of 13 traits in the analyzed public datasets. Further analyses showed that the difference of accuracies for BLUP|GA and GBLUP significantly correlate with the distance between the T and G matrices. The new strategy applied in BLUP|GA is a favorable and flexible alternative to the standard GBLUP model, allowing to account for the genetic architecture of the quantitative trait under consideration when necessary. This feature is mainly due to the increased similarity between the trait-specific relationship matrix ( T matrix) and the genetic relationship matrix at unobserved causal loci. Applying BLUP|GA in WGP would ease the burden of model selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pCHrjoIAzQ03a3h83QBwXoTdddmQdWrQVoVG87tAa4U","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459811","apath":"\/ggg\/5\/4\/615.atom","pisa":"ggg;5\/4\/615","ac":{"\/ggg\/5\/4\/615.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/615.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/615"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/615"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/615?related-urls=yes\u0026legid=ggg;5\/4\/615":true,"\/content\/5\/4\/615":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lXJ+wU0HFvGKJifHG59fXR4EOsxDwS9thbiI9gct7S3bRg9W1fTKZRPpYM9RQeajIx\\\\\\\/PoZu\\\\\\\/+I+jWCOuHZmtgGXhwxmLPXQFyZLqB\\\\\\\/RHC0sYB9uRjKRB9HERmJfHPjQm5Gk\\\\\\\/LkpaZqxD\\\\\\\/6ah7sOGB9jOES886ee4XhUwhOjhePEBP3B32rzKZhuzv22+7ynYT8h5PK0CW82ZiPuLy7us8RMfSBlQPtW124hsdXdUcx1t+uTXiWEcW\\\\\\\/PDl2r1CAeDODwLL4rq7Z0Jl+qI54aguOPwwG\\\\\\\/JKbH5HKx92kYvR+AB4QV9QixLl\\\\\\\/n\\\\\\\/FJoHyqUqEHAUFhk3np74va8IfdnPbfoHq9BJvLNTgG7rJQbDq6Lqx2BXSp9kmPd8b00uYywurO+jZy+N1nkgMEvJbh0Op2CTJBaOuV4wHgYdsddcu8gKv+oa4HxWxYuhN\\\\\\\/91fUCNZz2s\\\\\\\/N7hhUhDBfGDQ4Lez29GVnustxfGh6TS7kL3j4FjeVpHvxahyEqwUYGM+QRINH9yFu1DrW0WA8n0LnPnvcGpPOLfM2c+Ya3og9Z3Io804VvvyX28k2P+vIJ2a1JszUngL2S7895J7QaYeSd7SCx20F6kbCUJoLl3vSLecieVMjUb5YkW83yUUui1lT+bQ371yxAEX8Lm3M\\\\\\\/Qfz5zud5YjiiWSfd0OYeWBKV9Pa+MKnRcj4ZiDoQe9T\\\\\\\/8xuzR4qg0H7NULJ9G9bjLfQn6lQQ1rhKYWrunyGrWsVWLcUuRdkKHJe\\\\\\\/+AzaMkX5168b\\\\\\\/nBuzVycko5O5UOvI9zK49lZDLFc3Nyu+u3sWt+9i53+q\\\\\\\/Ru8wMo\\\\\\\/ie9susWQqhBVFZHL4b0OtspFEyBCQWuqnY6htKtN7vjHlF9\\\\\\\/LCYseX7oH8TLBO8tbZisYPFhZzkGBzoiKlZbpl8mzGGxKCPf2n08\\\\\\\/7GTdToUeqTSbZEcxrzL2u2DdPDdSLctq00gv67AGLXVwisSice\\\\\\\/y2a7V\\\\\\\/rIk1csh51g+w9Ssw\\\\\\\/XYE4TwPMe8AQx+0PY22BxHKBbH+ylXJOWA4b6MTvAIHgzc\\\\\\\/HF1EZvS3kH2qNWa+Neyfxnbmjp3N55Oq\\\\\\\/YYUOzhRCqFP2tlT5dpjhcGJCK9uKfp3l\\\\\\\/ZIr8COEZH83x4inYf3l9mnOFxCfLX4KHeYjQmvuUfuRZhEAPzpjOSzrYg583lRsUcxghWGdaNfIf0Eo5D0ddCm6eTPdoJ5N\\\\\\\/MSdP4a4dsMIUu6kgiu0OMKGdUt+JEmYvbP11MYc4tem4GTUorq7sGM+6OL7ixnGG2tkzi0Aa8iBp\\\u0022,\\\u0022iv\\\u0022:\\\u0022EXDifSYNFsK9KXgqxiZfLQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022778c1880aaeed2281e3e3a0a590086e9863f95c7175b0e20f38d227244a883a4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Wp60GaoigwZWcbBAB\\\\\\\/vzzJ\\\\\\\/HCRsCH7g9OcotCUXnMkEr95kMqNLlB\\\\\\\/q9bcK60d+gtrJ0QVShlpyFt5\\\\\\\/d4MeHZmlY9L0nSzAmrVCkJlrGAjfNn9WrYqnlF8QYWzEwasZVfr6euPcd8MXpuC1LH6VP81y+V+T8tBJb2L4s1uCN26bJHMog19wiMbiF+ThLZndUO+F0\\\\\\\/zLt0LFzSKGKVD8uvwaAgBjioRSAZ3SPUcDFXa\\\\\\\/hrxzhgmFWpwGVDhkDiaXtNd0m+5jirhr0e0GsYbXi7uvTWH+stqxHuGFB3nH1eFzfel+tZHP86cruROCzzp5b0AN4lDvr\\\\\\\/0al2N5Q23ph7SxZvd5xDZ2SGnixrRFbtGsY8LAIUsYmsNGdjrtxUkD2wI6DyMbnyu4\\\\\\\/lMnN6Sc7Pn+\\\\\\\/R5afK77if1gDqU1uNVccdKWjnHmiK3PLjiXvHw4E0oBFk2EkVHiVurmtx9Q2JajwDhD9kSFwuvRCqviIC28RNtO8rhWbXVx3TD5yXWWseWjkRKgynytxCw0h5cL6wGqBbmmyjf2z+k9dCo\\\\\\\/qKW2dJRegrA0KJbLw5Av24DzI2a9j+j2uzi94BIb6yb8Xz5EiRbNJuWu4AHH6RlKQgWNQi0ZiAaXWN\\\\\\\/IpP50MqWPyLQCAiO5NmjyqD+RxhiH8VwSGRFWjRV55jb\\\\\\\/QhLsqZti\\\\\\\/8qTgWIKNDTtVcRGnSFXp0y\\\\\\\/0o1+Fc5BaVrhtoAiLRVDiY\\\\\\\/b7+\\\\\\\/rF8L\\\\\\\/QUBijqaNWaXln9hmGOhHoyeu0xzDIclt5GboSxW1KODWtfF7DXjRQJ8rG6HrDnKD2h5bgHptox3YAkXCXXPoqFsmHJdYcaUP5uRQ7MLxxWb7q5VnGbNMV72v\\\\\\\/2LeZ84udvmMeANbA\\\\\\\/i7Sf+iemLJmE\\\\\\\/tVpaKGtDKMwtVkoK2bnMPWQdHGFMg\\\\\\\/dqX3VcBdCwhFQ3x5v6\\\\\\\/GytGNs6Lj+5lYl8k08wusEKAEamHy4iXl\\\\\\\/nAN6RJ5PzaBxF8xE9cpZ106ed0a8Mu\\\\\\\/IGaf+GPa8P4YtveUwa8ofNPw++YAX9vGbrtadpNzBiUMsi0CUe5nVbzVCA+PCYYq0z6glBHE\\\\\\\/5bvwi0pLZu7RQ07c7iHA0ytfCu2HfzDpvfA5v519CM4Zs7cZ5VGR+mGXxnnKydPZgEymR40aFMB8Ub2zmmmks4IBEprEN8Leau351wgLhn\\\\\\\/4irInKQ+mB1KFKnJKxytr+5DJ6MxRB1sVlLiIAcahCsQJpTbfqZjSlFmCeHlIrPpW2Q1T3lMfvq+63epAwczkfy5eKmSGcZwPiZ0t4ATaSGnQLhWYVqyy7Pirq+zdCQeupK0gdORKtKmVPxFHtJauVtkp1eubKyOLK5B47FXZmm2aUJdbna\\\\\\\/W5b6TNxqM5wbF0Shr+0ptBjIhq5xoDxvIatD21JB7hzaLrb500EcKWCzKYA4kMKy+bhPD2E8FG2A7yaWRN4TOS5fMaXmWwt6NsObwPfwXvCYkYaAWkQIBltY5oGe57hbj+4iyK+tfy8eCbRmWOPslLSAfxnt5QaQA+rSA6k1+3UMO4Sh5zMIU0RoSpR6jYge\\\\\\\/peOGSAWjSPkQnj3RTZm2XZW6XxfcupiT1hHPbwTxhBNm7E5egniOfdNv1fEViXQmFrO5DpNYsrtJGc2cbHJwCnly9e2gbjcRL0Lqvgn3uky\\\\\\\/gHriTOIOdwFTXrrxJ\\\\\\\/kJzmvNeV8Jq9fzezKnYwNVYDgz3dI8J5GKFsMF3NdGr9b0q+35BVP7PvC1GaGoGaP9H47vAXt1ufCUdoYGU7kkJSwzCZdNAUeJqe5LpRs94OAVI3\\\\\\\/OqnoX4sX9HI6q5vEBn3y5UN0xb3xfnODIigFKpNCDnptmeSyBCVmJUJbUc+P8LiLCinJwmqtZLeRpxV2TW\\\\\\\/5GIBcxzGRxpJy+2Orv+WrBmQQ6xgCd1gACnxYuuGp9Jup6GnQiGcD85xP\\\\\\\/REPBQVx5kIMONsRoPhUYh1HIjQntMCnOezEEeASzyhoAuLhtCXm\\\\\\\/PJ86McECZXipAGVQelicK0p9YMkHEzile\\\\\\\/t5U9GW7e3gcra6MwmGvrKHlpLxFudJ2RhgIgsvwrrZG8n\\\\\\\/8ZE8XzyByNaV84xCOHG0tSVR1dAqE3Onvr8SiwsXsZXXn49\\\\\\\/4f1orSKz4TQUwOOBUYVUhDQ56uXoKrBmJIJxbr0M5FomRpZOWuWS\\\\\\\/1hC6BwWYh69nAllL+f1ASY+JGzOLuSBV3vVJp\\\\\\\/fc6bivPkB8LFvmZ3EQnq77\\\\\\\/Yz3nIBMnBcvhSLtjG+IcPPnrRW21q7cBYCLyhU0ySVR02XoT\\\\\\\/TqFK2IeWWgjlkSHrf2ndaTAss8wS6a\\\\\\\/gtNS4JvR6sIgg1+QfymxqRzPbS7cRch+SDcphPQhaILPKrpVs1w6Vet7VH7NmbhWbOEiMH2ETKzi+0hT4Vpu23e2hYlf+UiqeDBeW2fAeZv\\\\\\\/fXIvPT0LawVppc\\\\\\\/QfZuVmCOGkFDu2EkVaUBhDXFcKMXN6GOvhFHYX1bu6UPM+JsPpfMw1ogHW+lOPc5GJ7+uoPBd0fQO5pgga9EbJLNMBYJDJaMqYc\\\\\\\/4GZZBkm2a9F+AARHZ+aKfQ1GnHlsed\\\\\\\/8MMXxbgTf4H3EOk8JBZmUBSNRVWwHWII6uC2riPKWD+uAMTbM7ZqPp3tleaqnYdKDmHf\\\\\\\/y+7iI5tOrKeFxV4OXicy4UTHUKh9UJvyeSigX1290B7LyaM21jr9eliNzTeTpIqU2sNRcq3cHZAVCPcNt6beEotPRQPq0sIkX8+Dnd7qXbdaSWzOVAlxM7o1aHSQtot+NeA3JoKS7LAHNk+35fH1gRSawN8yIArWI8NXH7egtBYVo74YOPs2Se\\\\\\\/H5FoN5Z8IwNAxIpslzPYwZDupMkdxXz4WePRHfhSuCwx7L6ZeaVj9tt1pTcKFuX40fIkrjD2WGW9zRy4p+w959sj8jt\\\\\\\/9VtpnyXhYSZ3WMj7JE4vubUF6okOSwZTG2vvAcNU8Goc3m1+lawYSOahbtbWQ5W7vG6NSHsiyAuknByB+DKDzCDWI4y0UtlE2fkCptJrFUI\\\\\\\/Dhw5O+7IaSVoHLE6QO2qmi0g5CQq8\\\\\\\/\\\\\\\/Q5JQSks8xQ6hU8O5fYD\\\\\\\/\\\\\\\/N+HuDfJWIeDU8rCO6QS4UobF8lVUyvR+85bqGn5nbqIvU7No+oTZxmqJ8h\\\\\\\/iOl3U\\\\\\\/ofj2dRozmou25Z5JWdQb\\\\\\\/oyHcjjsGKl5OMxODTYI+pZOWBXZnuu82f7Pn388qa78EZ9zM+DTb9Fr\\\\\\\/cFWOlMOb\\\\\\\/+2lFFMxIe4GgAdydgYhiOBpxlftR7sYqMl8uZ7QHzj5kDJJHEFiTnMA8pf8Yq34SwUZkNqcDyJCc7N\\\\\\\/a04e9W8F2h8A3Nl7j0fqYH\\\\\\\/+CTfbMMcZExuNZv0ofqv0UFgzSlkDgOC4Hnq2QEaU7V71zMvM+GXadz1v7kUFNgIqKck1R9faJWZiwihT\\\\\\\/rsSC2Pc\\\\\\\/L28aVLMFw96Ngac5W\\\\\\\/orhAnRSgP4W2qlhN6kZgaTTTvpDQTyha1rjiEyoZ9PpoJllPETDH80en16PuRG7K8ewb9768OdgNv2Sza801BVfM883HRIRRUQk7V34eqHhM5YTnhxm66\\\\\\\/ojeWu7RVM91nxdZMNHKX7lGe6iNQQltjMoWYgr1CIaL8VM91XgkQLcYvfGqt9LcJ31dBdhgxSVdhEDnssYDiNW1pqW0\\\\\\\/4k71zvzOkaomwjljgk8NYNa3fO5VJ3Ga4zVLZqrYwvGY73waRRowJYevWmbcummeig3lLDf+iIP9haBxwHW0774JuoDHYuAvJCRfnmGbyjpzfyNfqbJx8nAa8nBKM7lNIFeUVRXJXKTUxMoDIKO4G7TS3s\\\\\\\/jtaFq9uVxpaptW7s8vxWIg6PlpsYQzjpPpLBbZekuULhvZ76+P6nXL6hzj7pnZgc2fuy\\\\\\\/MHx64YwdmMJzHa\\\\\\\/GjMNWow2v9+FHbtTV3N29qETKq1vPk2ZEHullk5lThEyCXYgki\\\\\\\/LMinqT2L38ENu5SIs1ybEF+dNabgvazkZ8SRbA1y1rqKCkPvj1Lff4CLZZ\\\\\\\/5u0bH88kwzjXsPKfyklPlIsyRjc0phIKBIu\\\\\\\/WDL7yIHUfdaUaOljTwhK5b79+6cl\\\\\\\/wyLUu8ClaY1e9Mpt\\\\\\\/N\\\\\\\/xRu4cLQtTVqATRPNJSGTp6LcmzS9DVNM4fLpri25Z95lajqcQZesJyZAorsx02QkGdwW5Bz6x47ytTJNaDFWKTisDquiQuClG5mmuQOqTcnJZ2DvyerVjHQbPQInCHVJ66Yz4uGoAyKT9r5SPxQfZMeSkMOVIt5Y3Wq4E2n9jgCpyD\\\\\\\/HUPYtExkZSxdnznd5CDa5P8daSWhx9yoPt2EFcLet77fkS0DGqhMfns3H+gi2F9+YGpnxMV87TxpkrSV83ceixhiexUAkgcz21oTyaYd7bqgbQTUVd4IbuN3REe7TEFYyqQsD9Sg2Hp+ggSFsQVJrodUO4If7Vnp8a0fOCUbZvi6v4ruG2TEMlTIxhVSyD8TM9HufIT\\\\\\\/eXzuvMfanKVUFz63XHdGQ\\\\\\\/cvydCwHx0m8g2PQatEMxqSwzGaW9JcTf9EEvXnPIqddMOkirzoZHIOazrWD16f9OO6kZM4cX39L+YvNkCtrSi7C2kdVpUvhJgx32DH6TcpG5zbmA3LYWNtX4xGOzGUo1WGGtQ3LHXP1yHYTy7Ew9G9P7DYY5hw4Cj4Pqi8DZEOTVt6YcEHag4h93jcYga+ZhG8snA4vF4JUxffAsGt6w8vxIcvXZxVrACgWE+PgLkMsgBzxuMR1qvJibMDkmhGR8bBSO31lwJN6prCjC2Q1eBy13uCAvWii+odFbJjBFUFoyN\\\\\\\/mLZraOk0uEuxZloo95+p5UMI5dcBQI66Se1tmRgMoV\\\\\\\/RCrKro03SQzYfmPszgpHUzHc5nWbm5zPITIg23bmB+8ZQS+RjzdJKWjBI6B9FA\\\\\\\/xvVFbt5V57ToUOSMWWzSFIQ6kgcoxrg11o5BakEek4\\\\\\\/ilaT6w4nmbmjD4VllO8+q1FB2u\\\\\\\/Gv7sNaZdpnuOe+qy9iwIJk1MK7AdYg6ojH01xsPsDEZtyBiOmH2NLw82wmn0NorSnNJ8F8SM9Hle5WtXZmGrpLCqjYUdfv0CwGAVJZOARJeuV0D1DgfL6KEm58Pk3i02tzF3zzQTHkDdij+6C4PFcHxJ8aDvsEyUWAB6IOxfHa7TUs2TNdlTs8HINtsxYGrMcgDumg683hEIlsGOEMud4vmEWVpWIB5V7MQwRL3nX1F2PLXxkpCk38scVwA4Wy5IQMzwkMovmijzAWCCxOzVmKhRfWLDugyZrUyfVUyr1CjttPEVjMTnpJfDeJJfMifl5hC9UKcCqLusyLbOW7m6oFD5sVNTg4Ogb2vWwuEofNmraHzTKq0ZbwJrvlsHCNj\\\\\\\/m8IL0y52ufkuvPSOE58Y0CVGg7P8UVWx5LPEOMkqtLwMMJoQV\\\\\\\/PBAUng2QsQXltmdf9wCE6VdtG7zUXApcMYjPgRgzdlvZv99kQ8t1Mp4Se7\\\\\\\/Cn8qnUxSjGAGE\\\\\\\/ahw8Tes2yEQ+88m4iMZTJfuI5NFC3oYji5GJ37+Qd6+h9GdJKDVDAtsK3Fh9+oRwze4YQ8t41nGW6Zu9lSV7Umg970eEMYlVOEr9gEyQsLNc7w2WHqENARWtmYeJPq+PZU+jY8dPSuWHmmRVBPAJCc2OJcn8GcmuKESCeip8fnGcsIeqFYobKYdFFUStHQff74XYKPhfBno\\\\\\\/dxwv1HkJp7wB7md3imkDa7kf54\\\\\\\/7B4IHaVl8w7DR7sRETg1GnRCG4Mz8oJ5SnhHubBStvK1W5yJAB2iOA08uepdPDlbVTSnv23bArpCfzAnznxd9WlVx6x2jL2TGIPdZZ1onmAIvzwLji6fPeMSjArVW\\\\\\\/CrmZ\\\\\\\/wLeWyU2e88vQCpnEM2hlG31mmfOLXCOtkB1ffnwMH9yM9bVc7dZ+AIGiCNdlw5m9cxW6FC8Y3PD55o18xzWvQA88x6gCbjyO8oBB\\\\\\\/pCsXji1H8k6gWNRdEXooG+J\\\\\\\/RW0kNyZAuhG+H0Ok94bw8BKVVH5wnybq4g8vWmPm1tAeZm9vp7ucYyIrohrTZxGY0MPWR9Lgce5wojz6JkGFAowH5KAg6yorarBRNHNsDgf6R\\\\\\\/vPJMaujzbveFCJ+dDg075ve56s6OrI\\\\\\\/yOZ3auRLHt0Fk4z3scbCAFSeYqEtz0Vye7XO6FhJTrn4PNWTEQx\\\\\\\/Kyl3h1b6XRy5eSjWHqEHmkXkoJvYk2qT+FV3clEmiqINbMmSjh0iPntaIiONCe57vWhc7NdH0DmiQGxgUtDfzOIw\\\\\\\/rCetO892VkeXRg6En\\\\\\\/ls11UjYZDPZnkMG5V7g4N6bUbi74fC8Cv+rIDMwas+m9dIFiqKTFyTG2Rf\\\\\\\/WNQiLyacnTyXQL2nAVZOfgd93lXNX+lnttOCtOvV7HXHnRWHrZEvJcQjlO1E404rcQiodaE8iNtFfjcyI\\\\\\\/5GuhQ5n00AFvyPBRIqvXQ2a2rzLwPGH1x9mlW3HsUPNWtSa+v1xxTL8naTiYBfNSLvloe9OFEDCiEjtuiiKcGSQSq5zzl9x3+0G\\\\\\\/KIzWx3SFBle6DVEFQfyKetZGMmBiZKLtcCD3D8VMeI2H16K3szgMc9iCTpoq3JSjzgb6Ti2ZEJOX86iv2z8kZ7lbJuDGa8vg37QRR\\\\\\\/fRxhw+ga0pABZP3H6V5lpqZ5q0kR0pNN5IIyJAnQiNRntSFbXqu6q310ZcvzHBjUh\\\\\\\/IYC05SHXWUUlg4ZWt4gZfJci8P\\\\\\\/dyH\\\\\\\/GMh8kLXUuOSJsEWPJcuS3R9DDJqzTeUnH0M8kTccFnvkYnAl4g9kb\\\\\\\/QXvNPQMVvW29DzmEf+52BHDDTvDxp2GWWo4lSH1mScvEvNe08JeJR0w3e4nO7FgJm8ge9QhAmByNsvwr3rlgLMbGMhfDTSZgCD0pWY5b+FQHQqUKLU1wV7gsyGPlxpUaJDx+b2shK72CCV\\\\\\\/DZdU58eQ7ynjyXZlohuzxom4YAc1+U3e\\\\\\\/uxAou78U5VydZWMxVTAUghR2sh9D3UncftnaO6bs1a2r7nbJtnqvSdGw7sq1TWbAxn+lolRCohlH2\\\\\\\/I7Z7B\\\\\\\/3ttZ\\\\\\\/\\\\\\\/3ZviQ3G2XBQ9OgIPV4sBiR4oDqEfW7aqrwxJeKuqhnB0B+PhmqKgiSACWIhd\\\\\\\/TVWILHWmJl9LNrkdccXUWEuEaJW1vSZZA2hYm\\\\\\\/eDfZlOagwixRvD4PUMk2daUzK3lAChqGUvvEkFTZKzjhTg0cLKcYRGlwmalz7mFA1oL+vE61EOeJZhZCdmzkNtCYzXV5kQTKDgjyCFzuKwu6Pp9qnrMm34IFRUkzk1NQE\\\\\\\/aeTYVIT\\\\\\\/MWBv4x+f5ZVzgc\\\\\\\/bDFLhb2xdnGNrUFHHIqPOh+kLWWVuedXUZvhDTqfr3PzENyS83lYq9lb1+YaE2gLfEOzlKDeVyjsZPJh5yFKy0xjtGUuYnW+6RZPKz01oBJFRxqVGiV6MBVP2FWYhpJ9ANPynf7he\\\\\\\/kLKv415yvacemsOHThmY8OEWqKOF5wM9+HFvAcZ5VOPfby+HC42qkOe6bf+TagVycux1+gPc5uSjxovqjCiMRAzedkuhdI6qQMnHMbQCt1dtr5ihvuRAdAOw2FW1cyWkk7uccWxhJ+QaR5AcV2niz1TlxSsrRB6sCYh5yx9A\\\\\\\/HhP1dtfz1lIvPlBjHccpKHgNj1Y9fO+kHVYJi4rR2tPRSxm9ZN\\\\\\\/n6f5XNnXaOGhMQr\\\\\\\/j\\\\\\\/8vmzVuNYGQH8XUNA7lmElk0YXyNp2ejsvSnuUVTHXwN14lrP2Snoatx5GJz5rttC7lGVEwR2t+mOAQ5TyFVKhKQPbtOn6X6j\\\\\\\/lvKoPi8036wUY2Oag2kuAeGwZ\\\\\\\/6Leq4XutU1rmN+1U2zq4WAxdHNE8uFnj7XNHvA8YREdO2AxlXYUSVzGHyOyzj9DqKZf92nwkSTE4fcKp66ikZ1Vuli\\\\\\\/hmX9CEWmYoIkzftNCICGRUG+tWXmktGqVIV7ASteEHqeZq8cHswfmLz6uAJZjElUHB3CQuvAf3xv7RvQ5Nv0Eoqzk9WwLM1sWLBOtquJr41jOHoM3D73EkLOF9AXBp+lYxVteuLnLq8MaX7u73K0xPCeGSJOHYOmX\\\\\\\/60l+C43LLG56JpOrStH+D3YSrczolaGr8Qjx4ySEBsBOPce\\\\\\\/E+f6P1vQ\\\\\\\/snkOosJ81h89zjCxFGRYgJOtstrASMbzJPERtuXmjyrKpAUz66AdjgMjAky\\\\\\\/5Wgdvw2X4CTYqj+igkCKSrmUe\\\\\\\/k1BThfBzCgjlrcwge4S4O7+YnY2ucEjkZ0Szs9acNCTSPScFvUSyLHEEbcAvOTJWreo0AzsKJO2s8HGooOJHeo8ub08A1dphn7FrwMl8FGcG8JzIH0evZChuw1hEeHVsMC7VnidjpdIXINdeuDNBKt4R6crCXfg6dPAabHNJFUkks27tnBk5oCQYYQ0qZ7fpkKl3URJq6opv3lO0evIWxovc\\\\\\\/W5AzfR+5yP5jb37CSvAJn7Na0y1cxKau2rxNC+l8PSLxKWqIFlnUUfTQmI44vkfJdZtdkSr5AxJM9dOkdbzu65UKopAItAMnYJhirU29FKVSLjZ81absYt1WB2P90mtToodtP3vBGdM6rxb03HzSmawSpQM4hw784+QhHKzH3LouP6Iq4+D1qI2oYDy3tb+7c9tCzGs0cYNCcMuq85wt60Vyzu6OomhjqERIgOEp1n1vItgrr4Ncn+Wo+xMEXElkOHN7TKmWF4CxcJksSCxunxDnvYyaItRAjNUZTS+tRotMmCnSGZGjB+48nrf11VBPHd+F+ZmI9zyzRDlOrN+uxKPBgcBJQXuzbz32VKSn3rtRt6atJXdKfduon1svDwWxEGXgrEjOlE4+z26GlsY6qqqe8mwT6S3quqzRb2iAkcCUANaqvbGyGOJFZZKkTbPGqrOqvv313iUiBFhHCiq+5C4NcyxMmyLynDtV7TGJKNCrTwU+lrluSAxFURXy3s9rv05VWyzDvULN9hsQtZBGuZLOh6sC1bx9suNdNSlCxcVCGFDj4angct5v16jA6e29Tl8U723vpQfhG2Nxh9YJrQzvxKVe9CEUkrxNPjHflUnbVbIH4yDGdKfqwXilHlg0Ho5vDpIxwYyOsB4pOfIfAMyGD7ZmB8Pc1SvrM+LGUwCJNctmqxpqUY6Q\\\\\\\/UilQjBsKSLKbTVsVgECi+LDNOzmW\\\\\\\/3Bqf9F+hk4\\\\\\\/jyl3I3\\\\\\\/AbaSzSAEYdRbD1wBNeOmz+YPXtYUBEURsXk5Q2lJZMVxeP69oqqWFFNj4H\\\\\\\/3fcget\\\\\\\/4+mUrXA+OEigub+QhHWXuE4Jceye1Q\\\\\\\/To+1Dmf\\\\\\\/quOCmI45vIqE9U21Z\\\\\\\/jHYP5mp2XIfAVGBoVhjdUKztxVKJXDN32+f+qgnjJlWOerF7ZtnsRb2eljJQwXp5xuRB2k5RftcCU0JXHszMJTse3tUlAidK+3UDdxpS5R3stSNnHPHAgqD931b4XI9dk0vFP05gx5k4ii+Cgq\\\\\\\/JMZzjhup5DcbrEQRG\\\\\\\/abf9SH7IuV26i41c2rtWvxBLTfJTpy5R6bDHcbiNL9MyKA4h1eLWQeg5s7yI9dfjabFDQ7l5j43s\\\\\\\/Thqh8T+A5W4+gpGq\\\\\\\/EHq8tum\\\\\\\/UxodTok4gYev7NvQDiAqrD0zrWHrbFgoY4eMaMXCRTIyTmLLumWMiBSIiGiOHOCjLh6ZReaVLYlP3S\\\\\\\/m7u90vyaEIT9+1fyVDWVFcQ77oNx0IjhwmJo2LA3Jew4itrzJK2QI\\\\\\\/BoS\\\\\\\/5L+fwowx0FKAnhAFYC+lFDtun403BfaiNcjFlk4HYIoZ6QyLcqYOb5UkQxRivwtwgWcUJm0V2zNJ+OW7QwaRi8YvH9z67GsIYRl3uR7oM0aomlFN9o5Y2hQqEf6Opd\\\\\\\/3KuGZMxy2JwPlfJm9HwueuRR0Ew9lAjyaBPUl243uCWBfscjaicci+R6JY7Ekc\\\\\\\/oxurwo8eRn3EVc9+WcJlV28zOso22Nb0TEziXCj0bpE7kAffrHLtXHXB+5s+VIvUN1qeFMhad1nPNfKsFyJqB3Dbys8fYOsiEqcfolJypGP2T8lAQFdv86Rmi8+KULvxG3kdqmUU+hJoyKjzIJX37Qwf2laQCoHZk2fJcae76LS+wX3bG3um+CH17UI2dKToKlOrYSEBoVem9tPxYCwfJ3Dn+ha3BPnLg8mejlF0PgZu2lVEdp8UZL6u4TPnghsWlzX+CCi5bcmven2C+gWhcjtqhaNimC3QcIfBZ3V9k9tCvqyUAcBCvVuw7725h9hBz\\\\\\\/90lB6Ihf6duZ+lA\\\\\\\/51rjVpdNTnamq1C\\\\\\\/dXpdCDfAhz4WALHRCV4\\\\\\\/qIQt4cLEMKyY3geYBR9MAVkLdLeort4dMr3ezmAmV\\\\\\\/+8gatVZGx9aL9sfR1ThxaxgRVHPivWen5XQyY\\\\\\\/SCV7R51KNPlWm+naGlyuwcKApj8xtVvNzz3qmxO3LPT+DifHpWa9fvdpeLlKlhI1EPoPiEhcKao2QT4AJ1XfR8A30jiqCMjiCLJ0hqMziu+J5AzCIzttJl3oxqSkREy8c\\\\\\\/2NwhDD3PpgBFfbV24kkQiCHKatEpFj8kWISxYViebBcinezxFCMCjoX5\\\\\\\/JyWIt1mu77fRBCmk6Y1DQ750AkIP93TWd6anz27dbcCfIHBoHn1qIjhwk6\\\\\\\/M8rrJFe\\\\\\\/YC3DCqCM18ne4odbHKHcmhdlEqFPwLQLBJ8FRksnJjekx20CNWis\\\\\\\/vmt0RlXWRmYE9rFfrE9E77QrQbu3TOrC5WJYsw11vURc5eRUZjvX\\\\\\\/pyltL3hxzn\\\\\\\/fW2mT+sqc6aMVkMrIPdMF7eMH\\\\\\\/FjKYkq0lFI\\\\\\\/qqg0yi102GsP7Jp55TcphRtoHkk0h8\\\\\\\/B3gVzXpoSFAKO0nCALc7mIpTwrCyZiUrq61LYuSQwdO2wiW8g75AnH93Tp1W0PnHIqzTqLY\\\\\\\/2GwtoecRigYlPSKITjeD0InlfOhpQib8LTcqSqKEDQWENjVkrExCwf2EqynpadiATWeJm0t3d2gBiQEp7ZDCOTxjDBMWaXTSVHejrUBBAkFnAqtc3BpXDseRFq2hdA2iRnl4cG4iOJMJdSL2wQcE0HOeOJfrZKpcNfqTj2IQW1QIDzR9w3F1SPnGkzYrYBHuabI62MMqmfmzn4W\\\\\\\/0nIHNfrT6Ka3+Lb0ko2LjgksxO+fNKkuJFS2XIAYJwj3ldh1Lq1dAZZFTxhu3I5XUNu3EFXNaJkbVSXKgW+WlSPSWiOyERJWVYRsVCSnucB9fjpLHpw38+F8m2LOMAkkdGD+24cVlIW76oupHUEbfizDBqS8PaFMtgYNdaWVeZAnsg6vDncl4I3yB3R2DP6tmEbVCi2QFZAhajKk2GtXrPZAh++AUXbtp5Pofp0igH+V3NkEeXe\\\\\\\/GIAA6hkf2VY1UgK\\\\\\\/BUmdU+AE+vtJ3bvppG1lilvGbY7LtVoNcorJhySTnXrBAR3Q5kRRydm\\\\\\\/kSX\\\\\\\/ve3\\\\\\\/06daYmBYkuQk7HaKgtEL0zsB6zZYLT06XQ\\\\\\\/14jmotLr6MG56KtaRS7BTDZFKX+toSDo74fErzXiBKRqG+gXulpoaSL4HEfrrWBZOtLz67eT08GZoXNbUzJg3nWdf6EughwadUyhpeY6+mF64qDToDVxDGs18a3PDI\\\\\\\/yiy1\\\\\\\/+mJwNvQw+I0C\\\\\\\/usadIu3QZ0O85KN1p+Z0Hs3SZ6ftvRANrO+ccOfaaMa2OgsCnww6HLMCIeWzP9ITYFDLzBPvJjIQRBv1mdXaPrVP\\\\\\\/UC64j48wWki9xHzOIgQWkN3flbj5cyW1AqxYE50EFYvafbp65Li7DjCeVXplH2G9SyoPP0LDhPcjn0MeTLtehojIbbYU2v0pLNpW13WtOuBDmlD9oyeM4wB7Rz9g0cIyxGQBp664ntj8FQAsxcXAexpmIuBsxx\\\\\\\/hX5WEfp2kbQhymsgWL+7I\\\\\\\/9b7rRTMSG8+g8GnyN1zghKpAvJvza7w87u5QRvF+nUOfObKlo\\\\\\\/cmZp0n+2kAM5cWnKAgyxaPnSiJ6\\\\\\\/Nz9fx7siSyW\\\\\\\/SZUmoubkZ547uZLdR4+dqWkfAqJAZavO0skVtYoJlLW1YoJGnVCzLR4aa34yXY9Nb1Yw9xN2U3kMNyUlExO0bwVZ99CC5f7DQK5ReRt6KEJqO36Rx4CHD4ygz\\\\\\\/rGCp\\\\\\\/xS2J3z1TiwJ3IS0g3U+Iueaw2JjWTAITA4MkvQxXAe2PnURUDQ9SAaUpMYenf4vB1fEMZ1pooa919PbDgXIHQqN8BfDzJpyfQ9Nz0VO2IX8Q2Q4SndumX1KBbIvdwLi\\\\\\\/smzY+owRjL3mRa79RTSfX7Ws04krviZSue0coJBuPxjzEgI2SlxsGwfv4qEr3w1GCiI1nqQCDKSqvkE9sZ+S9cVtwxMkQON1AXmeKRvUvIuX\\\\\\\/y7oyPVdFH1E4MfK1VMHxjVULwhQbFd3X43WzL0HuIMCXsikHN8RGA1seXqnmjrwIuckGNWaLjQmRwr8F5lGu6XEe412vitcMTTbYeZWRpJ3RhvalvSyEqls8AULFFSLeyZRSohYh8K+5yVntyBtun4Gl87QoPtKTpcJPrMuTXv2cw8xRFbWdAYueWQ5\\\\\\\/7S2a+M3HDu0bKary6lq+hb5zgGVjPypURuqsb\\\\\\\/2Xso4+fSMpuURxXCz0SiUVIOLH9UU24z6G7w5JJACb0rK5vgPQXI4dB2gH+oJkzIZU3kF1xEnKyMX0K7ptLuRhL8Geho2DiN+mtHVwuCzMm2xSBfdFPmgcv+mp5IUe\\\\\\\/TcdceD8GgBS\\\\\\\/6j4YDah+1jDa2f\\\\\\\/RULjx3KCobvTUkGVwuVUY7eGGYrKrvh+86MMEqhMJ90hXjHilURkDv766m9WVaJH11\\\\\\\/I5AVh9jvd+lVhG67ikzs0zDbiu0eUrobJGRDZ6ATWbkGUFXnwZA612pMZsMOXLskNm+o\\\\\\\/z3JgLDQyKqUMdEkh1XLyB\\\\\\\/S0iDCA9PngnNgaO89ceSVuz0vmhRalmmgrBR3O1So4\\\\\\\/H8yhKebCb6Zd5qCSddrVPoRXRaFfAOoUrLQO+rLzSISXztF+BK1lzSKpA0rBR5ZzW1XjRczR0VYE94XaxnVe+fQPkkNGAnwtT5g4Btb+lyRGfgVraWAtXD5buokYON4JxLoiGsSeGpHI0qmk5QIrOA1y2ATFC+o4QoeeZWEDY1be93M+UkPO+nig0fcMGMptR2FOY3IES5EbY5h8Bf4+lsy0nwKgVJ1nujoFKUgB34jg\\\\\\\/BJOhFbr8IrDDnq6RwKmb2iKT3ipZ0SGEZ5Zrh7o3bII5AQbuIIk7+2zaxpNxOFv7uO5+xAHqqSnSwdba6e+6DAP4mY03aNYYWLhuzqDg1dW55zDuAxsVYZWGPuSSALthvXGQQWYk5YnjVaCX3ZGS0LX3myC\\\\\\\/hUPG0ZvQtT1sB+s9LNO00jjm5ks\\\\\\\/ivSToCEE2\\\\\\\/DsltaNAHQBkEql\\\\\\\/KQJqm3pLmM1eUKD4Q4YoSWRAm6kUIKBIXOmRC8XHQnI7xgnTz4SN49pQIRuSUWWOhgFQt9Oott7ggen+lUOBlTTiVD9dZ84vBCXMPuw09qEr8RGzwLrAHYa5sm7M8AF1uQA34eQH8I\\\\\\\/4SgIOGRZGNZZm7okQk\\\\\\\/KMPv4rkLwMjJH9Rc5Hbi2sJEPfHP54i2+4IjqrAogd6fN+0\\\\\\\/U2CNHzvJxxXRBgIXeop3sTbe6Pn4teg\\\\\\\/YQeIGIxqtHbYOElJ+Qs7GDwfiaT3cRhvWE4N0U4OiH\\\\\\\/ezfZK2yS\\\\\\\/RZXqbwa2dcGqw0vPpffnv\\\\\\\/bdmGDbPAfHyMAftYI04CgFirBIAyLUdPHhfXCZf2pMJS5qik7E\\\\\\\/8pkaMSKHlImVBl1IjwmqM+wsgKnMWhm3dsASDeNZfu4ieENo+OozJR213QpWsfey1sbcV9Ddj9zETheg6YfgzdQveiSyn8KSq7TjHRGIUlXnxlvCzG78J9HcjRnGQSd7VRHZkvFUgiuszUDBsRxeyBPB+9Y0ngsB3EOv6u3vHlqQkiLD9xbY+Ff0dyodzgcAPcw0RX\\\\\\\/n5AhNu17Kt0yN2pQ9AkuUR4aGBto1mXjh7QJtCLxp9TiKB4dVJvjCe1MJvNLO3+5IgxutL6iaNiNQynpLXe0hH49Tyv4rJk67eyMjPDoZyUQeJ5DMLmjFVhg9MmwrekBbnMIYXkXjiILkxnt\\\\\\\/+WFzTd\\\\\\\/oAZUHwpFu2EECs73IHJGu+hcS+AatxeqHdMtSl+Y9rxEpIscL4WIO1pPfGDN8f+L+0zqcXgTNSweqthky+lKuixPyYY7PDsEldUQZovnqN5oSbe\\\\\\\/RQHJLGjQVAeirwm3lLIUhyRo7\\\\\\\/L4MI8hDSaGw9kU3doau3FkpIhg+uluXGCFOdMhPH2hZuba8BCFLRRDxXsQzgDyO+Eybw\\\\\\\/rCK0UtufaC7K\\\\\\\/VLVMRbWL+wpIK1GdY95sutBKWI8\\\\\\\/UeIk1sE\\\\\\\/8zPSGLJcvX3qLTwy1Iz9MC6qDWdHyIZnACLP6oko3ZZi1Ph0ImitCxzxTzPBOtzWK7EwERkIuqF9TyTBBqchNn1aKT9Y0sQeYyka+hc6V6pmtiIhbTobQrNT82Wpn0Gf4E\\\\\\\/F0ZQS+s03d9Rv1NcH5nIr5BJhnMWk+tmaq4LslxzRzk8UrSsKsxMtUARDVz7BI28OrCGOWhMkKRByYj+d+wOEMxaTKaaETTDMq0rq+4HkwOajhnFneGCE4hGpk9S6btQ\\\\\\\/vKsv22OLMj5nlQ+1x9cez2sF372wQpEbf39O5QQaOmk1kG21mvi8GLRvJgvmUCVZVWcfdlYcoF8d12VKJhOMaYG6pALNLsyA7wVR6OmpMvNkG3jLc2ZR7hwwtD2WY8BQO9GmT3j5DDf2Mc66umw7ACLqorHP\\\\\\\/l4BqY4JWkdpK30Grr9z+7L1UiUPOnJJoL5GoypQSUElwFT8i21VlsSwqvxsPB\\\\\\\/upUhEhHn3vxZCq8PGijEQZFgSbPa26nvTHdv8Ba7onMNEel+I0CxEpX5hwSb5AIKTPepkO3vSNDP0c9fxnpJJSsc8hLgMwe9CPsZhECrGVlt4y2BACPOFxxvx7Kk8WOcEAj0FBoUiHr7+cJq7D9HYyQk8TlWYksAZ3qi2U+f2SalPLeyd6aYXz0Y2Lv4LabXhmtNmjnlL2MS0sdzxb44NLXmm\\\\\\\/CP+wNNYVAVGviH2b9M8AIc9tpBIkQ8UwGtV7lNLkyjwKuMTWAlFs6lJa4C2VJsOzcE6FxvqjMD8H7UKdmom6CPHUrkiZuK1mcqHQn167aGQzKpimQzhdSaRtiaodCgnHPsM7n2jfG+j1JMA\\\\\\\/47yelUgwwZRpHLLqgUd\\\\\\\/ipSz\\\\\\\/sUjU7j977pfXg97Tl4hwz3W0zQkVEBT3nosgDc4eCBYB3HImWRCFOY+r6o5nfpso8hpWJH9MbpwtQzCzxAXhRxOt+YqkUddj2finr7SgY4ZCUisbseF0CDsERHXg8Pozm00X32m8KmMkLXCwI7KfXF8PMHgzqko7ZKEP+2HxZ0\\\\\\\/+Ska+ojXEWQaESkQzVXIRThV1yRG+OHOXG2J0xSmc93cQPORGuNohlSKEb2J4YIqtq3MH5KcsJFqkUv2ZGKkIrFwh+IISVX+PDYUKkeVfeAHtrFqj+kIuUq9cexzrguu2jphIBMVDIBXOKYtuOEhSAS2tFOgdtIx7NiU9WELNMwPL1epjm2iAz84KAbMKfYHiCnQiUAN+H6U1xMng3GOpHi5lGWmLHVAvcCUk59OSutuO53\\\\\\\/kD8E4cwjVRvGS1\\\\\\\/3AgC432PhRNFIcDMg5wblzuacag0WI+bo35s4Qg4pzvwC04pxw8nryBSbqOwszMuLT+rMOpaR3bCewCWNYTHdmaLWp\\\\\\\/fmVD4tbbqR9Z1fknU9SJAtRrHuWrG38hAfPzB3NDqkQvmMrkM3I8HxihkWCI9ic3xP2242MFdc2pAE\\\\\\\/pmd3EUpIEgIgSO\\\\\\\/hOxDz5irTQ1XZ3+SScrszXaxHUwNRM1mhwMJsvdnuStrWnarzjMo1v0IhSe0HQLxuM4W+gyOWOkC+la034TFz\\\\\\\/sjO3+yRurXU+oqRX3C+XXpiqYSvw3AI\\\\\\\/t9A9gICftPBU30TbIz5FEtssyT17jJ0aRbPoJ6eYgPbax8H2jah1ZeAWohE0TCu8rQn07mNjWFpFJrxewCu9LPRxRlb5vnqPr\\\\\\\/b1xbe4C89GoXbvtaUK7dwk\\\\\\\/oi3KHXgEBEI6+WXkYEx17VmmYD8xdDoFeNKtZzN8ORpVtbiIj86r977tglPBXQfVa5cwFC53lSlxoFiEhfuMYZlTxB+C3c44zL+JuHsCrRfWGk4CkQb2bmOOgKW6dCh0f\\\\\\\/KpZzFzPXnEB\\\\\\\/6VjaDdcEW7O1FuorWkf+zWwg96JnjVyrBw4UecOBWCZTCS9OOzZToAYD1VMuAZlVI\\\\\\\/VLwlSR5HJ674vO9O9HBTBjhv2PSVcEKlXVbbAE2RcVRjeKC7YIxOnXllaJM2yRe\\\\\\\/hcjlb8Kby1FCl+4622NcHe8Kh06XM6YirIPJ2KwqTJJil9cW0O5cSXNDLs57D+Xv574Tv8pV1\\\\\\\/EE9bADPC\\\\\\\/noPNKWty9nFekmQRghPErT3yPB5SrbxVDr5KTqcbtSjsaOtcd+4VMH7POeZqMxUmMmlGbEVyj7ti2C1foxsEm70lMV6CVgVGv\\\\\\\/OOteiqQ8kbbig27TDEd3tZXgWSCSIeNe\\\\\\\/0SJ\\\\\\\/26rtqK3KiR4gYLUAegggIZNzSW4I9AOgk2CygvtuFCfizmQWxAPPIJ616+laf9ZKXQ6GMoPPnJ1TCQkbOgbtEijG+17AHyqktPfCxPrnevCVncKmIfAE2a1ml\\\\\\\/ok8puumDN0WzVAoRgG5Ds3k9C0jf7Cj+MguA5TW915U9E2\\\\\\\/cW+pOuhkd5hNOHtBbnU12UiRxxHr4mR1j52R+5YiM9jEwSWnZKA7Y\\\\\\\/DY2K5VO8SszbqirMYsbO7nQLr\\\\\\\/E9jZaugVfeZBhWh3\\\\\\\/86v4ISouF2oUiY6OrJa9SlofNuB+SZBC0Bxp5BsVSS18MVUVCgOc0b5gLn9RvTzZZa\\\\\\\/pw727xqDRYIXnxDGAR\\\\\\\/fo7uEmYLWrrsCc5kUehUeBRhD3KN\\\\\\\/GoDVrrLhGQrdEXDf9cz4hN4972EDrqntigfr3l6MWgFEq5S1V\\\\\\\/ISG4FLop9t0Tz4hO8Dfs6u6FZcBB+l1scaZieUrCm7YpDYPaugxp70bRNcqzD9Es9uMWUyNx\\\\\\\/J9m4UiDG5GFT5aE2hSw3mAPLz15MYIKT7lMnva1Nq8lSZM6AydiyvMRtEe3sd8xDvZKukxF8rOSoB6U\\\\\\\/2A5qxnkTAfILQWmQl9fyaEAcf+ibtJw8K1E5r\\\\\\\/8nnE6BSduWjoXcqR\\\\\\\/FBZdqFw8NJsIOgTLBDqtSiqytOLfbw7QX7mYsJ4I+fKSPYabiSB3ebxHv+Lz0OPp70rsKcePCZa6zRfgT4OqMix0DxuhDpZ1\\\\\\\/kDBcQEdArtAw7YhfD7Dr68N\\\\\\\/9Y8PI+o38vYtvZKFiAyT+ncaUZABZZCN8x4aghnRLI75WWxNp2joFyI3ebuJe1hQrLVfsbEV4LRhvgHwLpR4ZzW07Q1Jmxu+g20JD4cU9\\\\\\\/NSREoeCv2ysA62045cJ5e4iKX9tWD2\\\\\\\/VRS8OBesTe86SWGGK9kakdw6ZN5FSMPVg0gd1SpCWvJZ7SoSKTeNg\\\\\\\/PLzFwL4s5TawMLDVa0K\\\\\\\/u4vwY\\\\\\\/jN\\\\\\\/PVtbNJs6qwsCNoLpBmhdquHNbrcIxdMGMEu9fxS93PdpZAJIGqu39EpkjRyYJP+j4tbKXhwEMdSu4Gw62cOqcqPFrvJ\\\\\\\/02WwiIB4OwiL6TEq+nlBy5oW+6YTLuCpoSkI3mPdY\\\\\\\/Z+smgDBi97NsWMzOzoDZZP3+IW56pwKwqP7o7QQeaM72UjBOq7gJQqUROyDzJs4poLGxHbLMABN0sVsk8j6wHUvBUvzeb8vUAf1Cpj2BFgIHiAK+Yk\\\\\\\/r1olnKsz38RDgVt6H98E5iG5yPUH1njHWKXZ5laeZL47Hk4jKiLgUzSf7eLKDgyCH0uWn36Dgioh6qmfyu7ov6NlZYgJ5Sdut6uaw5dAyVOt1NmQXCtJ1jj4wKHNNUuSPOQl7ZUuCHBvJ5Eyc5VOI3i\\\\\\\/\\\\\\\/iSFb3en3NAqKQgWNUESBgOHLtjbgVPmlYb\\\\\\\/o3xx2sQ2pTV0AqUSHBX6Jr9c2fF4dZmjju17w3jABryzzsipC8b7ZmDrJesF6Shf\\\\\\\/rp06qFSkF92Z6BzBK2iBLLY\\\\\\\/V2zdiWxjKb0Om+lX7BYQAMhazV1HeVMmp4qlgTJSPj\\\\\\\/wPRGgKcFWdQJudfm4IWHxYNzfLVXZLvT3sLwID3Cdm5r5i5eJTxxQ46ZHFt67iVMBiRpgQcnoKJanwEpfymjKTgkqNlTXBXTkvOIiiDLSQAeWK86dpu9zIIymoIrFPMs5PqxiYJgL3EWUB5w3e8fJnzbuKdSs80RtBGkFCzZxY\\\\\\\/y+qbrdUs+uCez085i9dek2ExrdqsOMq9UwJya2dYczCfeaMjIZSN0ByGq32LGHaBkxP8xeR5HvDVXErzraAl6WC9xrnG8Q+26HcsNwCrpf+5X8zS9MZej2jzKZEhiXjiljr3IA7ThtbUzKLp6aa7\\\\\\\/StMVEf+R0zCWcgNYZ1\\\\\\\/q7xMWIayvl3oeHsP+7r38PLAg\\\\\\\/h+Lh3lq8qleiR28uu6tu+YAEMWpW40vrMD5EUp0UH5nD3heUXdyWj7z4pFcGb5Zxgf\\\\\\\/fZOF9yNMD8Fb00M8ySbCe1SvQhHFH4gVJ8czFsgUlJ\\\\\\\/n89tZrbSVjfHYawl5bJdY3Qh3bgfWkz1uz9z2SlY3Ku9a7+0CbvGfyv4D8NSNcZBAl\\\\\\\/43hOrfkme4V7GvRh0d1QhmP3nRvHqRQAw3A+GordVAaln8+AW\\\\\\\/n4QFIhnMci56G+d3BvUsdZ7PCm3aGRRgmxvBohr4\\\\\\\/CJ\\\\\\\/6mcjzpjAQvkqoZBeMbszd5RzIFgWR5t28x5qPzAYDoZnzSqATriELL1whBsd74XrupJh4ja0lJuBaFFqsxriVIQJHYSp3ZsWyGrdZdvgoPbGd5aYaj1ndkDjnYwmPcyCLp+Kot0FsJFmnBDdcpDKKc8XwI8iI3brV4LgfEI4C\\\\\\\/DMgPeuRZRvrU976tBuqOhx7GxXxtoVHzlv22nlfcXnVwd299fBrUGogT7fJOoY69HJnxjMoQOikNxsqgRgrkirdPaqPTSALrq\\\\\\\/5oo7lTFIQx3j6oiUJpWkT2m7v4bp5LT4WYdjG8ffFV52ny2g0uMHvrUpMGEjcs0KgizYWl1pds6u3xoVGwiw5i5x0ZcbuWLa44i601DXYTiivmgYl1vhcxGeWIJTC1vzspfTDHF5fOaRdY1zQOpMWYeT\\\\\\\/C6\\\\\\\/ZdABsFx+55pr6Xq+ewwE9Auo1iYtZ04\\\\\\\/8JUz1o4v7deFa78KIkq88XvE4u6RhNslVu9v1Prg3kG1F+e1UpAcMySqKykRSUFv2nfoSeCS\\\\\\\/v4dEHf6nHMTcyDJx7Fi0B9NWuneAOmnVIS98UQQZB+ObSM+5wCb52p3tvOP9NfX5HZjvX4f0Kf4ONxHsoaJoeDSIh1m5FObDTdb3L7sLwgUA4R9oqm8Lb5FmFr3S6aQ07DlbPLaS+i6ZPodIoBHUA8FR0tmzdoltYGRJ89yFyO0xC6QVVcwtJ20f+T1N8oWZMN\\\\\\\/ZM3xwuiXJHqLQQEEtkaZCNCf04uozmp07Z8KZC1jQStTf7CJBqR26DNwVSR3lGUj4woLs6\\\\\\\/LvdB9stN1xeLwC8aOwRR9FRZEQfo+XyLH0mOnwJqEKjUfsmFWJ8eJ6JAjWGTy2IAfjGH5qRaJi0OY3QBjnOT+iNk+PDC2t6z76QfUR4y6vTXg4iDT+ZMq9rGW1Un+p6uy5jQRRDQJZiRVsbk\\\\\\\/Oe6K9JSlcz8SDfu\\\\\\\/t0t9bZAF0uf85Nfu5XWMGAZ3PWuBG\\\\\\\/ugkaExujToJP0YgPE9mgUvxlvEcHzIVIy1qcoWEvDxSysKllqKnsvSR969jUYQ\\\\\\\/bVnYL84U91Ed84963mhjPzqrtkE4nBOCgBceMzJocvvccVmIZGUCkTGcwkmN2l\\\\\\\/2FHKI2daZ1EUWH+WaDnLANN4rUosLYVXxKhCDkyrYHC0z0Ggf21x96K+u9HEU2beK1dir4mrJgWaft2yQykuhaXJqpRMGu2np7DLWZveIhIRo6AmyGrZ9\\\\\\\/2Ts9ozupkzhHt3ft+9y2hAkyioLlmEu2snvLI8z4sn7rk9cz8Urvxhk6+WcFA6\\\\\\\/yIjEK2UjbihrI03SVNl6grRhg6LffB87EVYaeYW+LOYbCd7dPDFqWn9ydL6+gE44eJMPLesIW9L+k4ooqSGYWjS9yn+wEcYKxuP4Yim1t5WXPvRszl55rdbT17nWFVP\\\\\\\/8+6fxW75MtziiGYUZC0VF1Sn7E711iwWSNZMPg28FyJK+Ygl\\\\\\\/cUHGpky01wCjG4dzWgmgBdMnTdP4kWWG4ko2anEuj4pN1pS4OHvNj9a1ErkqoOrgevqAZQNLEk1JAKIzkka9Wey83sjiweZQ9xNRE6wDamwZ1B2U2wT3WtPg6qgDyb8BLrRAVh0j6d2UagMZRHJO1NKwGZqXi8u33jojx\\\\\\\/ftQk4gn1nfayEwFT+FV1BR14bjNzoF+81E9xfCNcnJHCEE+ZQoxINvnOKO0nzewzBgKi6JuretiQAJprPE5RVnhXaglYz0YSXR5OH9tAy3n2y1GL9W9obxrerGL1iL217VzRsxCseAKGwLl3W7wfQkVgrmFTrV7ZGTLmUBLCc53LWrMpJ6uNIgx6MdANbqO5wfzxtUWqPjNMEgbKki6Kc5Snr4EiPprDJF1j+2EC1OXSNJuNwfo0Qx7pkpaFuQ8W1zVEJ6aEIdscv\\\\\\\/4tmXf5o\\\\\\\/hisCvKaMKqQ3FAAJUupar7G9sEKqv8KmkfS8Xh8rrs0QCz5jKBNddXpykFF4LS5mF+P74Be\\\\\\\/zKn+unUpgF0ET6n8+UzFaXlVN0ax7\\\\\\\/INIWfky1Uy4I+Q+pTKBSKnyCGSf5oF8VtlSNKWZZ1YzVuLEZ4HCmF+hqrCUuE\\\\\\\/ZrGxnm1x1uMf9KtuW3DROO7I8LI28MU+Km\\\\\\\/uwXQDHpWogFU6FktYdBYSIvSw2xUXKkVxS5X6HBnSQMndthJTLt2wOKSa1uApgdoLbdCNUlFA9bOz2h9LGWJmWYjPzd01I50yOu6LtPcLCf39kzPNQFoUAhxbudl7QT2NClsJI7a7WWUADqN7zAZFVAYywDquUJvLS8bM+Ehi8qpVzkVTfbBSBd\\\\\\\/aBvsxahN5C3S4Ob03SA9RQwVFDWmPKnEOI69ENcqnvNJNzIKKviifpFSi3gJdfGQSHMdh3+DNcW3IleLdF0Bo6v9spoakcaiyGrdEn8Hbk2I9gbn7I7AbZXikFqWpoCy\\\\\\\/0yDnFgD3GXiIrtLyfqzKmqzneW0+IL83WAIgCTAwUBEu5s3OdJjDKuHjOTuIU9Oo0UBDtWBH4Ray3sMJwCbOfmHnYasMnGb3uvF4lipEYHujJjsQS6LnzUEqxa+z5UdUxCKdvojVs5\\\\\\\/usGAKCJrA6EAPIim+Mwdvz1VvIUAdWuhDH++tuF\\\\\\\/QeNMc9iwWoC1UE3z92erPHSyU\\\\\\\/brH6MdVye6hrkEAIYLK\\\\\\\/XF0I0Ml60ewT3RQGpJvTi70aZ2ZVqrYml2SH5Cn6UIXakXqfpiosiPiliS450QgmfuIMVW5AaLyf1XfS7gcQX6J2dFTdFlZWm2gEsMYDMKsTIBmL3g9Eut+gtG94csvMxBgzHGURzzfKEmLm2ak\\\\\\\/31r2V2UO8L+ifybPfXza0Q7IpKoeLrp\\\\\\\/2\\\\\\\/d5tALd+l7JFQXpOcsFlSsAUxEQNQ8pt5elmICFTekxymPy\\\\\\\/IQqiZm6RF799F\\\\\\\/rdFDazrQefnvTnssjdC7lwb0Az+ceW8YSr4Advcg1nHKhD2vUnrEy\\\\\\\/BwtPm4iIuI3kPTNmBF\\\\\\\/YT1f6d16oHP5EaFloZGI2FIz2Xyp\\\\\\\/GH8yMUw1RGM9i6iL5ZyfxAlUKhmgAVtSU6HeIWtunfX98ePcQVKqAst4KM8qFf9EF3LkQNC9PcRtmEEcDhPZfVyJbdzvZ8XvSwfurltVcq8Nsioct\\\\\\\/pnaMsw9kegGJcasfzjKL3CB032y0cLZ3GdpAnKeGioxlH5Jd44o8puCZW1D8RBr6eT+VC71KtcKv8uQwJGkrr0d\\\\\\\/a3mmtIIXWFzD0QctRQeQrwg7tLEDKkBIs3AawH\\\\\\\/pKLYAeuiF7Kcap8OOiZ9VX0PgTe3yGaz5sSlyhQmF\\\\\\\/41JklH8bQDpJ+wTjQjWtzmSOE13dkj11HSN39fy5XuzJFdfvbvPj+zDEhZsFFlAmneGZ+EfrXYxIy3r13yNRTquzWqwoZ6hV82nSb8\\\\\\\/PkDpOE+4Y37ksEpa7TpB\\\\\\\/GV0JCWta0sBcavaDKVs4Nv0jN9mcsgXi9dj5+U6e2GrGcmDJr5oyLP6hUu2wWpaKopAgcKhF88Eb\\\\\\\/hyU4FvamPal9O\\\\\\\/knRSADr0bj4GnLvOEKvH3nz3Faeto9G9Ce4ssaV1jKtwy6FL721lejuRBO2lIR58sg1kGEvx29oDh9EtUZHPJ3IETJqAFQKLAhiLH0RD\\\\\\\/tLmrajWu09Lu6wSzNokGlm3aN+2WM6NDqgW5bGUwZxB0kLFQyks7IFt7INOtlvJm3St8P6cY27SWnqvjq9aMrUStSD7l4HYZEQvf\\\\\\\/978e9QoipOqMwNe0e2NnY3t1\\\\\\\/xsEamN7jb1bbOUPM2D2yX8QinscvlKzXJxMpZCTTAJ5vNPoRRhqHaDqolSbxz4Y\\\\\\\/iPJ6egJBoj4BaHoM\\\\\\\/OjFh765gTEVbi3H3tDa2rM2Jo4dfDNEly6Bwls\\\\\\\/dP\\\\\\\/pU8iipjVqfMbF7LdsA9\\\\\\\/dVGYOOjAgP0u8zu\\\\\\\/Nk+jv4kMcVPGRvHUuNa4NpQRH+Za3PXCLq+RSANqNwD03AsR\\\\\\\/iVSiPCY9a7iNeKexbZLN8GTp3oC1J8kHI+g1ruOxxZ2+pebSaNHSp1roAZkiplcz\\\\\\\/gKUc8sIywtrygfmwEhqeClNbMO27PqV0gCw2M3SVyMalNNY8tgybMHuROv1UIckmCT34srUScEBetxdkMx3JR3iGIXWWMOe0q3VzMAnai07XPlqhyNSc+TkQaBLYkb4Cec53LxuaU\\\\\\\/SSfElzSfVdUP4TFMl+EUpEWKFl9gk2CRVihhIsTjuCClDsdf+9sdeiu+ZAOe+jDiA\\\\\\\/y97yZcv9re2p3ddSUJTAJ6kyYsq\\\\\\\/wUm6t1\\\\\\\/uUpNpZ9Otv5skRTobmQNA5Wb4g5GRZVqgVSZytmXtZbH4EGeJgJgDhhztzf43B52gdJ+Hh7VVi\\\\\\\/qYom+4fIe7R\\\\\\\/qTr9JuYfildw5z8iKa7TCfNrSD1ZJSjioHo+sLZfHQan3zQHReITVX58G6NWlnTrT\\\\\\\/OhgqEAW+I\\\\\\\/mUc41AhPG09OVJH0\\\\\\\/pBJg2NZc3OPpBXYP9+Bo2dAjoThIF1oC2Mx1c6ieXJ2p72sW3daMnbOzb5xRd+45VFxhwEAYGrS0\\\\\\\/q5Np+CHWPQ4KCieYSxJCw+4XsOQ0\\\\\\\/dvz63H15EimhA5L+r8+WZ+g1vwAomW+WbRsE9bdEeHm0dDBVaNVHbR\\\\\\\/zqMT8vsS1nnP\\\\\\\/vDVkesOOH+rG2f50s7cuhGxDspMs3pyAveP4Qhm7LxIIqu0G12W08QA7Is\\\\\\\/eadEDCu16Yrs\\\\\\\/8YA8YySr4lc3NSt5TyOCRpPPQ4KsndRlKFGPmv78UbPYey+UAy\\\\\\\/y5XnEfqDJGLrIChJyRJgZk225KAiS68Y3mQCtqc8fw63Q2AxOjbqqOcLsHnN2Lvoi99GkCHJFcfz047IANYCLrLMcUn9Kb7CvFI2KH3aonFJW1yaCbaGowuzq62s9JJQTtztwqzJ0j99etmr0yHwbQ5abA9r49QRHzGj4n+7WapzhyFrfDc8U3mkNmxr\\\\\\\/EgUGkFlBzxHsHvDjaKiM6GS8gjuVDW9+m2\\\\\\\/jhKxo86l5hH2B0bfEucOd8E2IuW6xDo5vDEgyKvtaHKMHGsK1H3CbfOO4pzvHbTs7vRIx431y9JGVTmZcxJei6\\\\\\\/DSIpAZP+m5p\\\\\\\/5h60g\\\\\\\/EpWYbdKxXCugnL6iuUz6EkxG3Feg\\\\\\\/B0HB8ciDIVaxy\\\\\\\/CpgP0qNQZ8wQGFi53nVEOWPg3x6syloAHjZOZ1c3VbGudnoHGbn\\\\\\\/Pl3ZEeNXcMnVB7ye36MHaV1ySS5qUPhNCFj0tTNqc3nil71RvWTOE9jy\\\\\\\/tSmeorTbwgMTdhVqTdW7Ti81ocLDXXSNRYBDIWhsp2a9Vpo4iDIWtdNXgLyEgCx3vb+dtTJUGSx\\\\\\\/0CmHyngrG4QkOq6Usb4PBJJBqFIaTytpGQdydcmCbROrktBDBzvwUqFFIUCokSrfy1ThX2To4lqkjfroK6jN4eCNqNf5cvPwBiwrg08RyOJvXxDGUOjWjUV2jxerYzyoCnxOKES6zEZs\\\\\\\/9z\\\\\\\/w9mGZ\\\\\\\/9lCoXmOrralN1UcSfziuk1jaNbwtLqm95dOyogYvAPGpj9aOjShA9W8dCSlxs4HD9\\\\\\\/CGQ\\\\\\\/NunDTxkqCxGJ7mu8NdzQrZHHWKqf1XMzdXmxhI68MFAEk8Leg3EpKLMRz\\\\\\\/2tyG0MFqiAn78ZGQZjHkJInzoI9uaqdEs1SbUwvoVWPLan\\\\\\\/Rja2sQZ\\\\\\\/3ZHfGIi5jK1ZaV9JQOiUe7JHnygatD+HkC8kDT7729h3K20i5f9dsoBqZ9yIa8l8ONUx8HilCKVqjrFcetoYbdNMrjAEibswkQfMleXdcG0OfDTemjIbTdJsE+VJJ5s7dnuKABYKykv0NwsLbfWixAFJ2oeH7cnH0FmlD0UqiLhA5Yhq+UsaVaCgJg0cWX+h7UNcaqzOMwQcx+4jYEL9d\\\\\\\/7ehl9E1vCC66H0F5ykjVfaU4rkSP02Nte5HTbf819cmg1Xf\\\\\\\/Im9P5iNFvOlmowZD\\\\\\\/88m9\\\\\\\/+1lf+8YZwBgwO8eU0b9DWm\\\\\\\/0zxsvvgUsZ7n53S3VotZk6Y39R3ozyDXc\\\\\\\/KZ9K2bWnZx6d2DwfYxhWkabmVePDRq5bGWhPKcei65lnH+ZOSMxmWO4JfGjRowtkPgm2IBPnqO6DmKOfjrcd1Jhh3Zo9AaBHFWIDMjp5+Q\\\\\\\/iPw43KUT8ovIPpeeXdcwbtGeLMlDr3T7HYmGdrS6pjl64tNczRUB7tYP0pGou+Uru9H9cs6zRyzA9uzjBod8jfli4PjRw6zmO7PrcdGZo\\\\\\\/fHaYxJNzYGzGkgbjbwyAfyrRypa4Ym4lQ0XP5Nsd\\\\\\\/gfQ6YbflvUODwoThUwHcVwOhNufJUXG9OYhta0j6Wy9M29y0IyzGnOOzHUA5vGQlKJGIhFfZBbJvwXPvZFR8o3rZ+fSPXcwLQGk5pQwkqfPxYb\\\\\\\/SjbTsTN9Zp78v5VlydLsC9VUFeoXiQ0bn4Ci\\\\\\\/sFbJrT5MvSj2HFKVbavfA\\\\\\\/9uKNzYndngxbh3qjHoI9\\\\\\\/9OoQtE98L9OZqKCWf2OOpmjYeFf\\\\\\\/sbYw0ejVvoTlWo4j4Y2ArMkxoJi5EtG1phZPwSc0NnnNBdT+mYJAVRFRTH9jdkbGcovFz1VvCkzGqbr35YRv8z4oXob6Qhisf1AoAI+FlvMaLvoaEzJShvyLiCUCipa1IeRf+tEKmiT3fVqs+ZOCpo\\\\\\\/bkq3pvQ\\\\\\\/HkLabuhPvU9cQN8Kl\\\\\\\/C5cL4wOfy93W8rGigwtqEkYlyYOoUXlgCR5fAotuAGo\\\\\\\/jKa+vegELpixJrJI3+AaA4WdSzFZcPhwX7JRL2r3\\\\\\\/IFUNoni4Xcl3k6UczJBKY6d1iiImMGU2l2IzcSE2lPQgBpDeF1U72hzd7DOrTp0+bXP\\\\\\\/BdnL9+QCjpULmquTI1gBomVUnCEt0I\\\\\\\/Ffeya6UEg08Sw5sg3BNdTclMdtS7tSNIPcYyHxhXKQAJeqUmNRJ5dEfZdPpVOGk64JuKemFRDBGYa7GLe4XQtX6raFEhZqE098WEo4olk4rsQbXElFFbVHkkOh1ryOPqP3\\\\\\\/W2hfanIefDZB0Gd\\\\\\\/P7OwqFVexmMf4+qUb6KJ7YmL23IbxZrfDw26gQebA9LmnXOIdu+VzqBYphvoz94qbjTNr\\\\\\\/KKxI2kjC2Ye79sVcTZtwSpDNS64DL5cXtD3whQDWPCXCrbS+O971rFeAC\\\\\\\/FmpZJ5nvHChTM1TgOU3RAHwnQWK3roNJcO6jiJe1PJb7j1wXsWIcT0cuDnwGfRFH1xuTexxrvHE0asvOEaZCSAdiMdxgdYss8MPeIuF1TFFoBsWpuLvzz8vaRVadCn4aHQQ3ErjFhI2ZTVJ3M6A5vHXFk4X1dGDmbXsc45Pacg1gm9EUe2EkcIYrt3Ph3U3CUubZjz\\\\\\\/Sj5yRKOhA6QjqkUECoppfNOuRpggtvJCqOGnDLednzyrEKeZNp258FOVA1eiJgBZVrwboGdvlUb7c\\\\\\\/mzrxNPdXEo4a8eBW1eAz9NvDpi\\\\\\\/HkE\\\\\\\/Ry2aljfBm60l1MRaaJelQT695UfH+jO+TU3mvdXDQBrLjo5xhRraqvPmbFBkbXFGd823xJqen+TZ+WOMKMbAuxXyxnWc05+GM4xSTdIfdse5xGOHwpd4nmhqlZ3xrMjJPz3OgaDuD2Bbk8iwWuh9O+vS7u0mnpgKb2w1o4d9uQOp7Km2nmWIvz4gb4azSTjvnuk5o4MnZJL5cp11dSxIGhBoGjElrNUtSYRmwddwSrTv+enpbwP9e79yEKgu+2WPMENWgh8UDqhdPrMgdY\\\\\\\/pnkCJd3GmVnW5Tpee+chDs\\\\\\\/Ca8wdlmY42xyElUIlRXQYTMqVQWKiqCVtbNwWBlUcEIRwawYl5KY2fGh33OjjIn5qcWgg06e3cpLOlA2\\\\\\\/LIgqkXBohA6UXs1qObxvTIyFlfpuCMBbQi7LZFj65s6o1FD1piu\\\\\\\/FmIvM1vNVu2pKeKinN9U16cG++KHy2aSEu8Bj0XEl9aVGUHF5RIHktALDmBgvwcDZpLZ08Bx15wXCfmcYKqV1wnkykapMkzOInbfrXj\\\\\\\/9VZCIbMe+DsxG5BVDbFaty1OxjZXzFtes4R\\\\\\\/RTooOdYq3LLaFnyR\\\\\\\/+u94tsHZQmMrIaH7BLi+ny4HivxOT90TETXxDzkvjC3fH50b7tzfUeHCF7kLN230z\\\\\\\/MyW1IK5h4ljtf+WYYBxelyZSpFeyCW49SP3m1y5OOnc9G9M6us\\\\\\\/XsRkQFHgXoC4O8lHU+7cZpYhfgZy9E4CqX9bxYiOAS2q1PXpxXqixQRhkFWxbFgvqjfuv\\\\\\\/EWnX5rl2gfLl1DcpqLypmGhzmUqMvhhsg54Zry3Xor0jW2YidjhLC9Td8vXZZl2BRVUrfIR42k\\\\\\\/N9u4lAhnpJbBx23whGuKQEeRisLJWxEu8\\\\\\\/Igorhp8FbMxIUVqEsTuNOnLf7QZudXYenPJFE\\\\\\\/UG3VBCGAd824wBGSGGfn1Hse7DOZre3s6L0Y7qSetm1\\\\\\\/nG7YjUZKQVmVUs15ORJZRt8HS7ERdDRlHoHPZfbEYLtL5YR0IaZpbHFpVvP58nvHo8cbB3ShRzzI4T6lFBqNlBeZ+zObvrcyKxymzD+mu0wKMkXzVCZQXlqEUp\\\\\\\/I3ZBNPfhFadNxWPsFZEihMIJu2XzFuUMHAQldVJ1JwGCchW\\\\\\\/VStXQe2wNYoQM\\\\\\\/L0svzT5KD\\\\\\\/eXyOvwdnCtxj54uMZ71Ifrydj2nlS2Id0FqzsF9Wu9+wHf8T2CRla3hfLhqUuhJ2o\\\\\\\/M2THRQqVDadjtnwAat8crQFObUV+AflYeXn7ffLlw1vSSYhDMDsG1PJAeowvuwY68Yk1IOFEr7YRSsBtdx9ZP7CxPYKIk1kBeZgE3bBjWXLaX\\\\\\\/vucArPvX0Jo6b90hDmSfOFGf4JTuMu2xrFfbEPB0Ld83\\\\\\\/gmJAlbr7asHA1KuOZHqeksSJKA19jEtO3mpACgkXBQdYU4HVXGGOWcG046oahW\\\\\\\/UmPT\\\\\\\/83xOFVb\\\\\\\/uJE5ePmBroYM17Yn0gDauC9sn4vKpIzBNcVIDsnoj9hpHyMzxKSpQwEa3QEShoNPTtrzd7qfh8OIQtI93bR9v6vHEyG5ImiS1FenHKPzOVBANCFPlTmdpbwmHh6XAQ3KpGBerrKaxjdkYsxcLrIMA8N+DsQJZpbdAu4nF9t62om5CYCoFO3umzBv6WZIJRSBnzuc8LTAZLntjneVqVgOvVKGZSjS8Efw4sTCoBOvybjiJo22ZjsGxiaJ5a9z2TWpErcHYy2tYm4JPITI6zD+bVwaWLOekAdS\\\\\\\/3arkSx02XBn2Iqqyt1w3To6gH16ko4G3AqZ28+dSHNweAO7Qx0fiMsSWV\\\\\\\/A9EBGJA30eR34wipps7uwISDxQv2VkvoT67NAKHyD8xyuoIefe\\\\\\\/E92jXug598EZKu3jVZ3FOvwNDt4A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022515pqusUxpj4btwQXdwwcA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d1bab348a6b592daf4eb9627224bbd56096395dbd7dd83504e9ce98360a8a806\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TwdPvJWEGuXpvj6rASWr3nJ1qDkhMn+Z4e4+rXvWpxLXQZE7+uQgS5dWgu8oiGwe06qGU3OYJVpgtkXhvAMlCHBbDtF6teCvBqs5ou5AlVO4iaV5ppBjJ21eCsxLlISBor13vNIIwnvvFebcay6dIE\\\\\\\/vaGvbLOaGEf9FtK9kz7z2m17lo+ypXnTTlVLv7hJzeSwRqB+ANe3BeFMDXENdH9HqcagMTHTEXdtT5aIpEzckQWdxrbBf8ZaVc8jNvtBO1pMB6yzUB\\\\\\\/eRYKUzWNeqKXy7Lr60wcgYrDepMhPS0kP0UtPTULOH741jZdFZg4l+ZDfBQgvgEMbQsH\\\\\\\/I1NgoGgopKyO3z8avhvgVL3RObTXc4g\\\\\\\/75h8ZvBQxb0VxfIF0isKmEqyq3c4Fr0U\\\\\\\/Sw9NV1TVHBufS4fql4P4nY34sigF\\\\\\\/VcxOQ7TPFN0hDjUehHxyIhvluibhfBAHH5eGQwtfETIla2O5qA9k2YrNT2DRla5+18z22revmR3U74MVW4j4K0HcRink9swXZ+iG\\\\\\\/FJ0hTW1UtBPLLrjcqw2DMysXNKpOJfyW5T1pO46Ejn7JefKHG8fDopwCVxqEC9Fl9Fb5dt8SCFG0\\\\\\\/pUoePk\\\\\\\/2LDwVhGVRk6Mk5L2oUFr4G\\\\\\\/sRqYpAU4EhJ6zPMKCTGQX0SNGj7RffeByvr8lCE791RFOiVWh5kxqE22CfCpaJo8qh9zbEkcMb1X7cLam84cWojlIn\\\\\\\/ygoP1ip6ofgewARAtjl5glRI6KHdMpK7MHpJ9uEuEp+tNTon3pOClZE2rhfwyzZcJ1+X5PLgU2c\\\\\\\/fZddzDRHOnOyfW8yjRNGkRR1PQ7m2wNDmBa6GDq+04wyzm9frEyWQWnVfimMeThVMbd+dQ41gIT6AHxDObW2r1YxKkcr8S3e5rRJxDn3YdZ2yN\\\\\\\/EpkyOPWTWYGZOBvui1\\\\\\\/EGEjy8OsOj9i6Po2Mh97RxIKG7lR1t0OxFBsR4d5gw8g+fj1IjOhTH6W2abPQB9L6lBMyHXGhFjuCOD2u92KuNTxMNV6Q5YB\\\\\\\/ixFEjRlzs1IKz2gvKc74dcobMxa4DWKIrn\\\\\\\/VNY5Lek+0K75wZoNMUFG77HRjn4wBTTJGXzR7R2Jsbf96z3Z9bub1WYwzoUGj9bTx\\\\\\\/X9dVGXAbg3FnEL8nTgd7rB3TAyRzkxgzZ3qtVpKQKxJ335xkAGwl+DbGHj6B8aUGkrhr+J6SOE6ytS\\\\\\\/l0zEKDx436NefyR0dfeItLfgYBrk4LWCTjlJGBxC4NqZgxs7WoIQi7U0SJg\\\\\\\/YmsP2jmADkKsgwGCBHR21OSHm2lTtbMhYO7EvaKunLxsxgYuXKhqHrr1CIw1YVEMbqqHH4r6+aZa2LTUJuLGK33yeXDNK58XlUgLkqKwQOSiauXzdKbipQx02RMVK4jkBwUQCXYvOxgFWQc+l4iHEB9zR3kPkzFl7pUJDn6TAnOtj9DZrw5mARUyQSYD5H3fYbyFoB7rUCEJp04EeSU5bcvZbtq+pdE1JlbytzKOo4AyuVmhDfPwcfEL8B3f2bxUzfI27UtNYC+nc6neumaq31uv7PIIhFFeb6ob6sUPdZxaH7eYkxpXcLoDxjNJKv4bpOInronnvBxRKA5GEgs9AQVNzcmRFOeJjCetmmNWJcrt46WHxj4Q3RicL4XzitP4tI0TqoLrJLRtZjqkFZpoZyGSzCQCtmZNUnDmylSO83kttBC5TYaqV5Bm6q45ZKtJyr7hmDXv5\\\\\\\/orjeOPhIu8CbQUQiJTsLUMQzL9vgeA34vDGbrA5XY2L\\\\\\\/KvyKJlLu09Ga7X4l+xfI9nF\\\\\\\/QGaDl0+hTbg48WU6\\\\\\\/lbRtF4RFEPp7MF5oI3T9AhMsxioCoRse89y0pyxkFc6hFNapURTur\\\\\\\/O6jzIM3PwE6GuPVWZmnihenk0pgA+W8UWPKl4eXXFBjIc8a1RDOJ8icbPBEdnNYCXlPFEsgu6UAP28FCiKwH8bdhT6y18H805la3u3UpPnEzbBQs8\\\\\\\/pfqL\\\\\\\/IZKcdIuOB+K8dzlX90g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Izp+g+yN6k3q54Mv4\\\\\\\/tUCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285206d17677203791ab00d96069bf3b2f5190417cd5902c87370b232ac4145ef\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459811 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/615" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5za9R2B_kxvn_VFLEpYqV3t_8tQv8Kid-sH1zBPICj8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/615" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2IsJ-YkECx1E_m3IgmzNuWhk5mAzmAlwwETM8rIwrKk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/605" title="Gene Regulation by H-NS as a Function of Growth Conditions Depends on Chromosomal Position in Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/629" title="Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459811" id="node459811" data-pisa="ggg;5/4/615" data-pisa-master="ggg;g3.114.016261" data-apath="/ggg/5/4/615.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhe Zhang</span>, <span class="highwire-citation-author" data-delta="1">Malena Erbe</span>, <span class="highwire-citation-author" data-delta="2">Jinlong He</span>, <span class="highwire-citation-author" data-delta="3">Ulrike Ober</span>, <span class="highwire-citation-author" data-delta="4">Ning Gao</span>, <span class="highwire-citation-author" data-delta="5">Hao Zhang</span>, <span class="highwire-citation-author" data-delta="6">Henner Simianer</span> and <span class="highwire-citation-author" data-delta="7">Jiaqi Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">615-627;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016261 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459811" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhe Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhe%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhe%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malena Erbe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malena%2BErbe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erbe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalena%2BErbe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jinlong He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinlong%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinlong%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulrike Ober </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrike%2BOber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ober%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlrike%2BOber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ning Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ning%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANing%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henner Simianer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henner%2BSimianer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simianer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenner%2BSimianer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hsimian@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hsimian@gwdg.de</a>
<a href="mailto:jqli@scau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jqli@scau.edu.cn</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jiaqi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaqi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiaqi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hsimian@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hsimian@gwdg.de</a>
<a href="mailto:jqli@scau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jqli@scau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/615" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459811" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459811/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/615.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459811" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459811/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/615.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459811" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459811/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/615.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459811" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459811/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/615.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459811" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459811/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Obtaining accurate predictions of unobserved genetic or phenotypic values for complex traits in animal, plant, and human populations is possible through whole-genome prediction (WGP), a combined analysis of genotypic and phenotypic data. Because the underlying genetic architecture of the trait of interest is an important factor affecting model selection, we propose a new strategy, termed BLUP|GA (BLUP-given genetic architecture), which can use genetic architecture information within the dataset at hand rather than from public sources. This is achieved by using a trait-specific covariance matrix (<strong>T</strong>), which is a weighted sum of a genetic architecture part (<strong>S</strong> matrix) and the realized relationship matrix (<strong>G</strong>). The algorithm of BLUP|GA (BLUP-given genetic architecture) is provided and illustrated with real and simulated datasets. Predictive ability of BLUP|GA was validated with three model traits in a dairy cattle dataset and 11 traits in three public datasets with a variety of genetic architectures and compared with GBLUP and other approaches. Results show that BLUP|GA outperformed GBLUP in 20 of 21 scenarios in the dairy cattle dataset and outperformed GBLUP, BayesA, and BayesB in 12 of 13 traits in the analyzed public datasets. Further analyses showed that the difference of accuracies for BLUP|GA and GBLUP significantly correlate with the distance between the <strong>T</strong> and <strong>G</strong> matrices. The new strategy applied in BLUP|GA is a favorable and flexible alternative to the standard GBLUP model, allowing to account for the genetic architecture of the quantitative trait under consideration when necessary. This feature is mainly due to the increased similarity between the trait-specific relationship matrix (<strong>T</strong> matrix) and the genetic relationship matrix at unobserved causal loci. Applying BLUP|GA in WGP would ease the burden of model selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/whole-genome-prediction" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-prediction" rel="nofollow">whole-genome prediction</a></li><li class="kwd"><a href="/keyword/genetic-architecture" class="hw-term hw-article-keyword hw-article-keyword-genetic-architecture" rel="nofollow">genetic architecture</a></li><li class="kwd"><a href="/keyword/trait-specific-relationship-matrix" class="hw-term hw-article-keyword hw-article-keyword-trait-specific-relationship-matrix" rel="nofollow">trait specific relationship matrix</a></li><li class="kwd"><a href="/keyword/blupga" class="hw-term hw-article-keyword hw-article-keyword-blupga" rel="nofollow">BLUP|GA</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">With the availability of high-density single-nucleotide polymorphisms (SNPs) covering the whole genome of many animal and plant species, the genetic merit of a genotyped individual can be more accurately estimated via a combined analysis of phenotypes and genotypes. This method usually is termed genomic prediction (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>) in animal and plant breeding and is the cornerstone of genomic selection programs (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goddard and Hayes 2009</a>). Equivalently, whole-genome prediction (WGP) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">de los Campos <em>et al.</em> 2010</a>) in human genetics aims at an improved prediction of complex disease phenotypes based on genome-wide markers. Recently, WGP was paid close attention by scientists from animal breeding (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Goddard and Hayes 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wolc <em>et al.</em> 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cleveland <em>et al.</em> 2012</a>), plant breeding (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Riedelsheimer <em>et al.</em> 2012a</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2013</a>), and human genetics (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vazquez <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013b</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dudbridge 2013</a>). In many countries, WGP has been implemented successfully in dairy cattle breeding programs as the standard genetic evaluation method because of its potentiality in accelerating genetic progress (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vanraden <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lund <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ding <em>et al.</em> 2013</a>). The genotypic data analyzed in WGP are not only from high-density SNP chips but also from whole-genome sequencing (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meuwissen and Goddard 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ober <em>et al.</em> 2012</a>).</p><p id="p-4">A critical concern in WGP is its predictive ability, which is usually measured as the correlation between the predicted genetic values and the true genetic/phenotypic values in a validation population (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2008</a>) and is termed accuracy. Many factors affect the accuracy of WGP, such as marker density, population size, underlying trait genetic architecture, relatedness between training and validation individuals, and prediction model (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2011b</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2013a</a>). Among all the aspects in WGP, genetic architecture of the complex trait under consideration is known as one influential factor affecting not only the accuracy of WGP but also the relative superiority of different prediction methods (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hayes <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Riedelsheimer <em>et al.</em> 2012b</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>). Both empirical (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eckert <em>et al.</em> 2010</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hayes <em>et al.</em> 2010</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>) and simulation (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2011a</a>) studies evidenced the robustness of the predictive ability of the genomic best linear unbiased prediction (GBLUP) method (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vanraden 2008</a>) in the prediction of complex traits in highly related populations, which is the typical situation in agricultural populations. When major genes or even genes with moderate effects exist, however, variable selection models, such as BayesB (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>), can predict genomic estimated breeding values (GEBVs) more accurately (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>). Therefore, GBLUP can generally be considered optimal with traits in compliance with the classical infinitesimal model, while models based on variable selection such as BayesB are considered preferable for traits with a genetic architecture including large or moderate effect quantitative trait loci (QTL) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wimmer <em>et al.</em> 2013</a>).</p><p id="p-5">If the genetic architecture of a trait under consideration is unknown, it is hard to choose a proper WGP approach without a serial of laborsome model comparisons. These situations are quite common, such as with traits of low heritability, populations of small size, and novel traits with unknown genetic architecture. This increases the computing burden and uncertainty of implementation for WGP. Hence, a sophisticated model which blends the robustness of GBLUP and the advantage of BayesB appears attractive from an implementation point of view.</p><p id="p-6">Previously, we proposed a genomic BLUP model in which the realized relationship matrix (<strong>G</strong> matrix) was replaced with a trait-specific variance-covariance matrix denoted as <strong>TA</strong> in (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2010</a>), and denoted as <strong>T</strong> hereinafter. Following this, several approaches were proposed to enhance the predictive ability of WGP or the performance of GWAS by building the matrix <strong>T</strong> with different strategies. A weighted GBLUP model was reported to have greater predictive ability compared with GBLUP on complex human traits (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013b</a>), in which the <strong>G</strong> matrix was weighted with rescaled <em>P</em>-values calculated in a GWAS. Within the framework of single step procedure (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Misztal <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>), an “iterated-GBLUP” algorithm was proposed for genome-wide association study (GWAS) purpose by building a trait-specific <strong>G</strong> matrix weighted by derived marker effects within the iteration procedure (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2014</a>). In <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> (2014)</a>, the publicly available GWAS / QTL mapping results were used as prior weights to build a <strong>T</strong> matrix in the BLUP-given genetic architecture (BLUP|GA) approach. Similarly, a systems genomic BLUP model, including two <strong>G</strong> matrices built with markers with known or unknown biological function, was proposed theoretically, which can account for and differentiate SNPs with known biological roles in the phenotypic or disease outcomes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kadarmideen 2014</a>). All previous results support the fact that using a trait-specific variance-covariance matrix can be a useful way to improve the performance of the relevant models, though the optimal way to build <strong>T</strong> is yet to be investigated.</p><p id="p-7">The approach suggested in this study is based on BLUP|GA proposed by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> (2014)</a> and uses a trait-specific variance-covariance matrix <strong>T</strong> (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2010</a>). Different from <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> (2010)</a>, the trait-specific variance-covariance matrix <strong>T</strong> used in this study is a weighted sum of the general realized relationship matrix <strong>G</strong> and a relationship matrix <strong>S</strong> built based on prior information on genetic architecture. While in <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> (2014)</a> the genetic architecture part in <strong>S</strong> was built based on information reported in public GWAS databases, we now present an approach how <strong>S</strong> can be constructed using only information extracted from the dataset at hand. Then, we will investigate the performance of the new strategy with three model traits observed in the German Holstein dairy cattle population. Additionally, to validate the feasibility of the new strategy of BLUP|GA in varying genetic architectures, we will apply the approach on three publicly available common datasets. Finally, the performance of BLUP|GA will be compared with benchmark genomic selection approaches, and features of a trait-specific matrix and the reason why it can improve WGP accuracy will be discussed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>WGP models</h3><p id="p-8">Animal breeders have been using the Henderson’s mixed model equations (MMEs) to predict the breeding values of selection candidates for decades (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henderson 1975a</a>). Conventionally, the variance-covariance structure of additive genetic effects among all individuals was described by the pedigree-based numerator relationship matrix <strong>A</strong> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Henderson 1975b</a>). Recently, with the availability of high-density genetic markers covering the whole genome, a marker-based genetic relationship matrix (generally termed as <strong>G</strong> matrix) is used to replace the <strong>A</strong> matrix in MME (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eding and Meuwissen 2001</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Vanraden 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yang <em>et al.</em> 2010</a>). The statistical model for best linear unbiased prediction (BLUP) using marker-based genetic relationship matrix can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>in which <strong>y</strong> is a vector of phenotypic values; μ is the overall mean; <strong>u</strong> is a vector of additive genetic values for all individuals in the model which is assumed to be multivariate normal <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-2.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-3.gif"/></noscript></span></span> being the additive genetic variance, and <strong>G</strong> the genomic relationship matrix for all individuals (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Vanraden 2008</a>); <strong>e</strong> is the residual term with <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-5.gif"/></noscript></span></span> is the residual variance; <strong>X</strong> and <strong>Z</strong> are incidence matrices relating the overall mean and additive genetic values to the phenotypic records. The MME corresponding to model (1) is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-6.gif"/></noscript></span>, <span class="disp-formula-label">(2)</span></span>in which <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-7.gif"/></noscript></span></span>. Following previous studies (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Vanraden 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> 2009</a>), the <strong>G</strong> matrix in our study is defined as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-8.gif"/></noscript></span>, <span class="disp-formula-label">(3)</span></span>where <strong>M</strong> is an adjusted marker genotype matrix including <em>m</em> SNPs in columns and <em>n</em> individuals in rows. Here, the genotypes are coded as 0, 1, and 2 representing the copy number of the second allele, and then adjusted by 2<em>p<sub>i</sub></em> in each column, where <em>p<sub>i</sub></em> is the allele frequency of the second allele at the <em>i</em>th locus in the base population. Since the use of different allele frequencies <em>p<sub>i</sub></em> does not affect the accuracy of prediction (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stranden and Christensen 2011</a>) we use uniformly <em>p<sub>i</sub></em> = 0.5 for all SNPs to build all genomic relationship matrices as in former studies (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>). The approach using only the <strong>G</strong> matrix defined as in <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">equation (3)</a> was named GBLUP.</p><p id="p-9">In a previous study (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>), we showed that incorporating GWAS results into WGP via an approach termed BLUP|GA outperforms GBLUP in many situations. Although we use the same BLUP|GA model as in <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> (2014)</a> in the present study, the GA information (<strong>S</strong> matrix) now is extracted from the dataset at hand, rather than from prior GWAS results in the public domain. To run BLUP|GA, in the present study the <strong>G</strong> matrix in the standard MMEs (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equation 2</a>) was replaced by a trait-specific <strong>T</strong> matrix, which was calculated as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-9.gif"/></noscript></span>, <span class="disp-formula-label">(4)</span></span>where ω is an overall weight for the genetic architecture part (<strong>S</strong> matrix), which is a key parameter in BLUP|GA, and denoted as ‘<em>weight</em>’ in the following. The <strong>S</strong> matrix used in the present study includes the following information sources: (1) a proportion of SNPs (‘top SNPs’, <em>top%</em>) selected according to their size of effect estimated in the training population; (2) possibly the <em>n</em> SNPs adjacent (right and left) to each of the top SNPs (<em>nflank</em>), which are included to account for linkage disequilibrium between the top SNP and its flanking SNPs; and (3) the corresponding sizes of marker effects for all selected SNPs. Correspondingly, the <strong>S</strong> matrix was defined as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-10.gif"/></noscript></span>, <span class="disp-formula-label">(5)</span></span>in which, <strong>M</strong><sub>1</sub> represents the genotype matrix of <em>m</em><sub>1</sub> selected SNPs (top SNPs and their adjacent SNPs), which is a subset of <strong>M</strong>. <strong>D</strong> is a diagonal matrix, and its diagonal elements diag(<strong>D</strong>) contain as weights the estimated marker effects for the selected SNPs in <strong>M</strong><sub>1</sub>. To ensure analogy of <strong>S</strong> and <strong>G</strong>, diag(<strong>D</strong>) was rescaled such that the mean of diag(<strong>D</strong>) was 1.</p><p id="p-10">By using an equivalent algorithm as proposed by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Strandén and Garrick (2009)</a>, the marker effect estimates from RRBLUP (ridge regression BLUP) and the genetic values <strong>u</strong> from GBLUP can be obtained in one procedure, which facilitates our top SNP selection, <strong>T</strong> matrix building and GEBV solving. A combined AI-EM restricted maximum likelihood algorithm (AI-average information, EM-expectation maximization) was used to estimate variance components (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-11.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-12.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-13.gif"/></noscript></span></span>) via the DMU software package (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Madsen <em>et al.</em> 2006</a>). To run BLUP|GA, three parameters <em>weight</em> (<em>ω</em> in <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">equation 4</a>), <em>top%</em> (a proportion of top SNPs), and <em>nflank</em> (<em>n</em> SNPs adjacent to each of the top SNPs) need to be assigned. Both <em>weight</em> and <em>top%</em> range from 0 to 1, and 0 was assigned to <em>nflank</em> as default value accompanied with a maximum value of 10 in our calculation. In this study, we tentatively used a grid search strategy to identify the optimal parameters.</p><p id="p-11">The full population usually includes a reference population and a candidate population. The entire procedure of using BLUP|GA includes two stages: a training stage by obtaining the optimal parameters from the reference population, and an application stage by applying the identified optimal parameters to the candidate population. Within the training stage, the reference population is divided as training and validation set to conduct cross validation (CV), hence ‘folds’ and ‘replicates’ were used in this stage. The algorithm used in this study is as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-12">Build <strong>G</strong> matrix with <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">equation (3)</a>, calculate <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-14.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-15.gif"/></noscript></span></span> is the generalized least-square solution for fixed effect <strong>b</strong> (the overall mean μ in <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">eq.(2)</a>), <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-16.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-17.gif"/></noscript></span></span> as defined in <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">equation (2)</a>;</p></li><li id="list-item-2"><p id="p-13">Calculate the additive genetic values of GBLUP <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-18.gif"/></noscript></span></span> (optional step, run this step only when standard GBLUP solutions are needed);</p></li><li id="list-item-3"><p id="p-14">Calculate the marker effects (RRBLUP solution)<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-19.gif"/></noscript></span>,</span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-20.gif"/></noscript></span></span>, as defined in <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Strandén and Garrick (2009)</a>;</p></li><li id="list-item-4"><p id="p-15">Given <em>top%</em>, <em>nflank</em>, and ω, select the SNPs with the largest size of marker effect <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-21.gif"/></noscript></span></span> (top SNPs), build <strong>M</strong><sub>1</sub> and <strong>D</strong>, build <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-22.gif"/></noscript></span></span>, build <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-23.gif"/></noscript></span></span>;</p></li><li id="list-item-5"><p id="p-16">Calculate <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-24.gif"/></noscript></span></span>, and get BLUP|GA solution <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-25.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-26.gif"/></noscript></span></span>;</p></li><li id="list-item-6"><p id="p-17">Given different <em>top%</em>, <em>nflank</em>, and ω, repeat steps 4 and 5 to predict the GEBVs of validation set, which is also a grid search of the optimal parameters for BLUP|GA within the current fold;</p></li><li id="list-item-7"><p id="p-18">Repeat steps 1−6 within a cross-validation procedure by providing different folds of datasets and obtain the global optimal parameters by maximizing the average accuracy across all folds/replicates within the reference population;</p></li><li id="list-item-8"><p id="p-19">Apply the globally optimal parameters obtained from reference population to the full population to predict GEBVs of candidate population (never used in steps 1−7) by running steps 1−5 once.</p></li></ol><p id="p-20">The aforementioned algorithm obtains the optimal parameters from the reference population by applying CV and grid search via steps 1–7 (denoted as “training stage” in the sections to follow), and applies these parameters to the candidate population via step 8 (denoted as “application stage” in the following). It should be noted that the steps 1–5 are the core algorithm of BLUP|GA. Given a dataset, and given all parameters, GEBV solutions for BLUP|GA can be obtained from these five steps. It will not matter whether this dataset is a training / reference or both training / reference and validation / candidate set, since the matrix <strong>Z</strong> can indicate it correctly.</p></div><div id="sec-3" class="subsection"><h3>The German Holstein genomic prediction population</h3><p id="p-21">A German Holstein genomic prediction population comprising 5024 bulls, provided by Vereinigte Informationssysteme Tierhaltung w.V., was used to validate our new strategy based on the BLUP|GA approach. All bulls were genotyped with the Illumina Bovine SNP50 Beadchip (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Matukumalli <em>et al.</em> 2009</a>). After quality control, 42,551 SNPs remained for our further analyses. Highly reliable conventional estimated breeding values (EBVs) of three traits, milk fat percentage (FP), milk yield (MY), and somatic cell score (SCS), were available for all bulls. Statistics of the phenotypes are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. These three traits were used in the present study because of the well-established knowledge from previous studies (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hu <em>et al.</em> 2013</a>; <a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>) and their representative distinct genetic architectures. They may represent three genetic architectures of complex traits that are composed of (1) one major gene and a large number of small effect loci (FP), (2) few moderate effect loci and many small effect loci (MY), and (3) many loci with small effects (SCS), respectively. EBVs of the three traits were used as “phenotypes” in our WGP model. To consider the performance of WGP methods in smaller population size, we randomly selected subsets of 4000, 2000, 1000, 500, 250, and 125 bulls from the full dataset, respectively. This datasets are available in <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FileS2.txt">File S2</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465826/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465826/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of datasets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Public datasets</h3><p id="p-24">Besides the dairy cattle dataset, we analyzed three additional publicly available datasets for model validation, (1) the loblolly pine dataset (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> 2012</a>), (2) the livestock dataset simulated by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hickey and Gorjanc (2012)</a>, and (3) the common dataset provided by the 2012 QTL-MAS Workshop (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Usai <em>et al.</em> 2012</a>). The first two datasets were chosen because they were recommended by the Genetics Society of America (GSA) for genomic selection model comparison (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">de Koning and McIntyre 2012</a>). Because these two datasets were analyzed by many researchers with a variety of WGP models before, they serve as a good reference for method comparisons. The third dataset was chosen because it was also analyzed in detail by all participants of the 16th QTL-MAS Workshop and results were compared by organizers (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Usai <em>et al.</em> 2012</a>).</p><div id="sec-5" class="subsection"><h4>Loblolly pine dataset:</h4><p id="p-25">The loblolly pine genomic selection dataset provided by <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> (2012)</a> comprises 951 individuals from 61 families with each individual systemically phenotyped for 17 traits. These individuals were genotyped with an Illumina Infinium assay (Illumina, San Diego, CA) originally designed with 7216 SNPs in a previous study (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Eckert <em>et al.</em> 2010</a>). A subset of 4853 polymorphic SNPs was used in <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> (2012)</a> as well as in the present study. Among the 17 traits, we selected 6 traits according to the results in <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> (2012)</a>: four traits (Rust_bin, Rust_gall_vol, Density, and Rootnum) for which advantages of using BayesA or BayesCpi were found and two additional traits (CWAC and Rootnum_bin) were selected due to their representative heritabilities (highest and lowest in the remaining 13 traits).</p></div><div id="sec-6" class="subsection"><h4>GSA-simulated dataset:</h4><p id="p-26">This simulated livestock dataset (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hickey and Gorjanc 2012</a>) comprised 2000 individuals in generations 4 and 5 (1000 individuals per generation) as reference population, and 1500 individuals in generation 6, 8, and 10 (500 individuals per generation) as candidate population. In this simulation, ~1.67 million segregating sites were generated, and within these sites 60,000 and 9000 sites were randomly selected as SNP markers and candidate QTL, respectively. The 9000 candidate QTL were selected with minor allele frequencies &lt;0.3 or without restriction to their minor allele frequencies. The additive genetic effects of all candidate QTL was sampled from either a standard normal distribution or a gamma distribution [shape parameter = 0.4, scale parameter = 1.66 (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>)]. Hence, phenotypes on four traits, denoted by PolyUnres, GammaUnres, PolyRes, and GammaRes, for each individual in the reference population were generated with a heritability of 0.25. We predicted the genetic merit of each candidate individual using phenotypic data from reference individuals within each of the 10 replicates of the simulated datasets (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Hickey and Gorjanc 2012</a>).</p></div><div id="sec-7" class="subsection"><h4>QTL-MAS 2012 common dataset:</h4><p id="p-27">The third public dataset we analyzed is the 16th QTL-MAS Workshop dataset simulated by <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Usai <em>et al.</em> (2012)</a>, which can be downloaded from <a href="http://qtl-mas-2012.kassiopeagroup.com/en/dataset.php">http://qtl-mas-2012.kassiopeagroup.com/en/dataset.php</a>. It includes 3000 training individuals in generation 1−3 and 1000 validation individuals in generation 4. The genome comprises five chromosomes and 4000 SNPs per chromosome. To generate the genetic effect, 50 QTL were selected randomly with their effects drawn from a gamma distribution (shape parameter = 0.42, scale parameter = 5.4). Three genetically correlated quantitative trait phenotypes were generated based on the 50 QTL for each of the 3000 training individuals. We predicted the genetic merit for 1000 validation individuals using the data from the training individuals.</p></div></div><div id="sec-8" class="subsection"><h3>Model validation and accuracy</h3><p id="p-28">For the dairy cattle dataset, the full dataset was divided into two subsets, a reference population with randomly selected N bulls (N = 125, 250, 500, 100, 2000, and 4000), and a candidate population with the rest (5024-N) bulls. In the training stage, a fivefold cross-validation procedure (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stone 1974</a>) was conducted to compare the predictive ability of different WGP models and to obtain the optimal parameters for BLUP|GA in the reference population. In a fivefold CV, we randomly divided the reference individuals into five groups (folds) with equal size and used four folds to train the model and to predict the GEBVs of the fifth fold. This was repeated five times, so that individuals in each of the five folds were predicted once. The fivefold CV was repeated 20 times, resulting in 20 averaged results for predictive abilities or unbiasedness, respectively. In the application stage, optimal parameters obtained from training stage were applied to the full dataset (both reference and candidate) to assess the predictive ability of BLUP|GA and other WGP methods in the candidate population. This was done to mimic the practical situation. The accuracy for this dataset was defined as the correlation of GEBV with conventional EBV in the validation set in the training stage or conventional EBV in the candidate population in the application stage, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-27.gif"/></noscript></span></span>. The regression of GEBV on EBV, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-28.gif"/></noscript></span></span>, was used to assess the unbiasedness.</p><p id="p-29">For the three public datasets, different validation procedures were used to make the results from the present study being comparable with those reported in the original studies, in which these public datasets were analyzed with several WGP methods (<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> 2012</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Usai <em>et al.</em> 2012</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>).</p><div id="sec-9" class="subsection"><h4>Loblolly pine dataset:</h4><p id="p-30">For the loblolly pine dataset, one replicate of 10-fold CV was conducted to validate the BLUP|GA method, and hence only results from the training stage were reported. Similar validation procedures were used to assess several methods, such as BayesA, BayesCpi, RRBLUP, and BLASSO, in <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> (2012)</a>. The accuracy for this dataset was defined as the correlation of GEBV with deregressed phenotypes (<em>DP</em>) <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-29.gif"/></noscript></span></span>.</p></div><div id="sec-10" class="subsection"><h4>GSA common dataset:</h4><p id="p-31">In each of the 10 GSA common datasets (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Hickey and Gorjanc 2012</a>), the genetic merit of the candidate individuals was predicted from the reference set using nine GS methods as presented in the review article of (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>). We assessed the BLUP|GA with this dataset in the same manner as (<a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>). The accuracy was determined as the correlation between GEBV and simulated true breeding values (<em>TBV</em>) <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-30.gif"/></noscript></span></span> in the candidate sets.</p></div><div id="sec-11" class="subsection"><h4>QTL-MAS 2012 dataset:</h4><p id="p-32">The 2012 QTL-MAS Workshop common dataset includes only one reference population and one candidate population (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Usai <em>et al.</em> 2012</a>). Hence, it is suitable to demonstrate the entire calculation strategy of BLUP|GA with this dataset. A fivefold cross validation procedure with 10 replicates was conducted within the reference population to obtain the optimal parameters for BLUP|GA, and then these parameters were applied to predict GEBV for the candidate population. The performance both in the training stage and the application stage are reported. Accuracy in this dataset was defined as the correlation between GEBV and simulated TBV in the validation set of the training stage or in the candidate set of the application stage, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-31.gif"/></noscript></span></span>.</p></div></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>GEBV accuracies and marker effects in the dairy cattle dataset</h3><p id="p-33">We first validated BLUP|GA in the dairy cattle population with 5024 bulls (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) via two validation procedures. For the three validated traits (FP, MY, and SCS), the accuracy and unbiasedness of BLUP|GA and GBLUP, obtained from the mean of 20 replicates of fivefold cross-validation (training stage), are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The accuracy of BLUP|GA and GBLUP in the candidate population calculated from the application stage and parameters for <em>weight</em>, <em>top%</em> and <em>nflank</em> used for the application stage and derived from the training stage are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. For scenario N = 5024, <em>i.e.</em>, all available individuals were used in the training stage, only the optimal parameters derived from the reference population are reported. In both validation procedures, the trend and size of the advantage of BLUP|GA over GBLUP is consistent. BLUP|GA dominated GBLUP for FP and MY, and the advantage decreased with an increased size of the reference population (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), which suggests that using “correct” prior information is particularly important for small datasets, as also noted by <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> (2013a)</a>. By using BLUP|GA with the optimal set of parameters in a small population (N = 125), the accuracies of genomic prediction were increased by 82.2% and 23.8% in the training stage, and 113.4% and 30.8% in the application stage, for FP and MY, respectively (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). An advantage of 5.6% and 1.9% in the training stage could still be observed for the two traits even if we used the whole population (N = 5024, <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). BLUP|GA did not consistently outperform GBLUP for SCS in any of the scenarios investigated (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), which suggests that a zero <em>weight</em> should be assigned to <strong>S</strong> for this trait in BLUP|GA. With respect to the unbiasedness, both BLUP|GA and GBLUP performed well for all scenarios with this dataset. The optimal parameters applied to BLUP|GA are provided in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>. Generally, the optimal <strong>S</strong> matrix in this dataset was built with a small proportion of top SNPs and five adjacent SNPs on right and left, respectively. While for all traits between 0.01 and 0.5% of the top SNPs were accounted for—with a tendency toward a smaller proportion in smaller reference sets—the average optimal <em>weight</em> assigned to those selected SNPs was 32, 4.0, and 2.0% for FP, MY, and SCS, respectively (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/615/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of genomic prediction using genomic best linear unbiased prediction (GBLUP) and BLUP-given genetic architecture (BLUP|GA). Points showed the average accuracies of each fivefold cross validation from scenarios using different population sizes. Results for fat percentage (FP), milk yield (MY), and somatic cell score (SCS) are presented with blue filled cycles, green filled squares, and red filled triangles, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1321135435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy of genomic prediction using genomic best linear unbiased prediction (GBLUP) and BLUP-given genetic architecture (BLUP|GA). Points showed the average accuracies of each fivefold cross validation from scenarios using different population sizes. Results for fat percentage (FP), milk yield (MY), and somatic cell score (SCS) are presented with blue filled cycles, green filled squares, and red filled triangles, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/615/F1.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/615/F1.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/615/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/615/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">Accuracy of genomic prediction using genomic best linear unbiased prediction (GBLUP) and BLUP-given genetic architecture (BLUP|GA). Points showed the average accuracies of each fivefold cross validation from scenarios using different population sizes. Results for fat percentage (FP), milk yield (MY), and somatic cell score (SCS) are presented with blue filled cycles, green filled squares, and red filled triangles, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465900/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465900/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Accuracy and unbiasedness of genomic prediction in the dairy cattle dataset from training stage</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465894/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465894/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Accuracy of BLUP|GA and GBLUP and the optimal parameters used in the application stage in the dairy cattle dataset</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">In addition, we compared the marker effects estimated from different population sizes for fat percentage (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), milk yield (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/016261SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS1.pdf">Figure S1</a>), and somatic cell score (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS2.pdf">Figure S2</a>), respectively. To ensure that the effects estimated from different traits and / or different population sizes were comparable, we rescaled the marker effects so that the average absolute value of marker effect was 1 for each scenario. It is clear that the marker with the highest estimated effect was found around DGAT1 (Diacylglycerol O-Acyltransferase 1, the peak in the left side of chromosome 14) for all scenarios in fat percentage (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and milk yield (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS1.pdf">Figure S1</a>). Although decreasing the population size (<em>N</em>) from 5024 to 125, the scaled peak value decreased from 60 to 9 for fat percentage (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), from 26 to 7 for milk yield (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS1.pdf">Figure S1</a>), but no apparent decrease was observed for SCS (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS2.pdf">Figure S2</a>). Similar effects on both the size and variance of SNP effects were reported in <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> (2011)</a> for a German Holstein dataset.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/615/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of the marker effects estimated for fat percentage. The marker effects (gi) were estimated using ridge regression best linear unbiased prediction and rescaled so that the average marker effect was 1, in order to make the sizes of marker effect from different population sizes (N) or different traits comparable." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1321135435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plot of the marker effects estimated for fat percentage. The marker effects (&lt;em&gt;g&lt;/em&gt;&lt;sub&gt;i&lt;/sub&gt;) were estimated using ridge regression best linear unbiased prediction and rescaled so that the average marker effect was 1, in order to make the sizes of marker effect from different population sizes (N) or different traits comparable.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/615/F2.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/615/F2.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/615/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/615/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465908" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-47" class="first-child">Manhattan plot of the marker effects estimated for fat percentage. The marker effects (<em>g</em><sub>i</sub>) were estimated using ridge regression best linear unbiased prediction and rescaled so that the average marker effect was 1, in order to make the sizes of marker effect from different population sizes (N) or different traits comparable.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>GEBV accuracies in public datasets</h3><p id="p-48">To validate BLUP|GA in a variety of applications, we additionally analyzed three common datasets: loblolly pine dataset (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> 2012</a>), GSA-simulated dataset (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Hickey and Gorjanc 2012</a>), and 2012 QTL-MAS Workshop dataset (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Usai <em>et al.</em> 2012</a>) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The accuracy for GBLUP and BLUP|GA calculated in this study and the optimal parameters for BLUP|GA are listed in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. The parameters include the proportion of selected top SNPs (<em>top%</em>), overall weights assigned to the <strong>S</strong> matrix (<em>weight</em>), and the number of flanking markers selected accompanied with each top SNP (<em>nflan</em>k). The reported accuracies achieved by the genomic selection methods BayesA, BayesB, BayesC, and RRBLUP in other studies are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465889/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465889/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Accuracy and optimal parameters of BLUP|GA for common datasets obtained from the training stage</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465850/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465850/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Accuracy for different genomic selection models in the three validation datasets</span><div class="sb-div caption-clear"></div></div></div><div id="sec-15" class="subsection"><h4>Loblolly pine:</h4><p id="p-58">For the six selected traits in the loblolly pine dataset, BLUP|GA showed a consistent advantage over other methods (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). We expected that BLUP|GA outperform GBLUP and its equivalent method RRBLUP for the first four traits, because advantages of variable selection methods over GBLUP/RRBLUP have already been observed by <a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Resende <em>et al.</em> (2012)</a>. Results confirmed our assumption, and BLUP|GA outperformed GBLUP/RRBLUP and even variable selection methods such as BayesA. It should be noted that, for Rustbin and Gall, BLUP|GA increased the accuracy by 29.2% and 46.0% compared with GBLUP, and by 12.6% and 23.6% compared with BayesA (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>), respectively. For the remaining two traits (CWAC and Rootnum_bin), BLUP|GA yielded a similar accuracy as GBLUP/RRBLUP (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>). It is obvious that the optimal overall weight (<em>ω</em>) decreased from trait Gall (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), Rustbin to Rootnum_bin, accompanied with the decreased advantage of BLUP|GA.</p></div><div id="sec-16" class="subsection"><h4>GSA-simulated dataset:</h4><p id="p-59">The GSA-simulated common dataset has been well investigated by <a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> (2013)</a>. The BLUP|GA accuracy is greater than GBLUP accuracy by 0.104 (23.3%) and 0.081 (19.8%) for the trait GammaUnres and GammaRes, respectively (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). BLUP|GA and GBLUP performed equally well on the two polygenic traits, PolyUnres and PolyRes. This result met our expectation since the traits GammaUnres and GammaRes were controlled by a small number of genes (N = 900) (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Hickey and Gorjanc 2012</a>), suggesting a characteristic underlying trait genetic architecture, which is not the case for PolyUnres and PolyRes.</p><p id="p-60"><a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> (2013)</a> provided the accuracy of 10 GS approaches in their <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/TableS1.pdf">Table S1</a>. To make the methods comparison easier, we extracted the comparable results from <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> (2013)</a> and showed them in <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>. When averaged across the four traits, BLUP|GA performed as well as BayesB, slightly underperformed BayesC by 0.002, and outperformed BayesA and RRBLUP by 0.002 and 0.047, respectively.</p></div><div id="sec-17" class="subsection"><h4>QTL-MAS 2012 dataset:</h4><p id="p-61">For the three traits of the 2012 QTL-MAS Workshop dataset, we derived the BLUP|GA model parameters with the 3000 individuals in the reference population by CV (training stage), and then predicted the genetic merit for 1,000 candidate individuals (application stage). In the two stages, accuracy of both GBLUP and BLUP|GA were calculated, respectively (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> and <a id="xref-table-wrap-5-6" class="xref-table" href="#T5">Table 5</a>). According to optimal parameters obtained from training stage (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>), only ~0.3% top SNPs and their flanking SNPs were used to build the <strong>S</strong> matrix and finally forming the <strong>T</strong> matrix for each trait. The accuracy of BLUP|GA was 0.779, 0.843, and 0.838 for trait 1, 2, and 3, respectively (<a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Table 5</a>). In addition, we collected the accuracy reported by the 2012 QTL-MAS Workshop organizers from <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gaspa <em>et al.</em> (2012)</a> and presented them in <a id="xref-table-wrap-5-8" class="xref-table" href="#T5">Table 5</a> for method comparison. The achieved accuracy from BLUP|GA is the highest among all the approaches including BayesA, BayesB, GBLUP, and RRBLUP. An exception is the accuracy of the approach ‘GLASSO_20’ (proposed by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gaspa <em>et al.</em> (2012)</a>) which was 0.853 for trait 2. Although this is slightly greater than the accuracy obtained by BLUP|GA, the accuracy of ‘GLASSO_20’ was 0.778 for trait 1, and was absent for trait 3. Hence we did not compare BLUP|GA with ‘GLASSO_20’ in <a id="xref-table-wrap-5-9" class="xref-table" href="#T5">Table 5</a>. On average across the three traits, BLUP|GA outperformed GBLUP and BayesB by 0.072 (9.6%) and 0.007 (0.9%) in accuracy, respectively (<a id="xref-table-wrap-5-10" class="xref-table" href="#T5">Table 5</a>).</p><p id="p-62">To illustrate the pattern of predictive ability of BLUP|GA with respect to the parameters assigned, the accuracy of BLUP|GA for trait T3 in the candidate population with weights ranging from 0.1 to 0.99, <em>top%</em> ranging from 0.05 to 10% and <em>nflank</em> = 6 is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Given the aforementioned parameters, the accuracy of BLUP|GA ranged from 0.597 (<em>weight</em> 0.99, <em>top%</em> 0.05%) to 0.848 (<em>weight</em> 0.55, <em>top%</em> 0.5%). In this calculation, inferiority of BLUP|GA performance was only observed when too large weight was given to a very small proportion of top SNPs, while for a wide range of parameter combinations (<em>weight</em> 0.2−0.9%, <em>top%</em> 0.2–1.0%) superior performance of BLUP|GA compared to BayesB was observed (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/615/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of the best linear unbiased prediction&#x2212;given genetic architecture (BLUP|GA) accuracy for trait T3 in the validation population from QTLMAS dataset. The accuracy of best linear unbiased prediction-given genetic architecture (BLUP|GA) (&#xD7;100) calculated with the assigned weight (vertical axes) and top% (horizontal axes) is shown in each cell of the heat map. Red area shows scenarios that BLUP|GA performs worse than genomic best linear unbiased prediction (0.758), green area shows scenarios that BLUP|GA performs better than BayesB (0.828). The optimal parameter combination obtained from reference population by cross validation is shown in black box." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1321135435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heat map of the best linear unbiased prediction&#x2212;given genetic architecture (BLUP|GA) accuracy for trait T3 in the validation population from QTLMAS dataset. The accuracy of best linear unbiased prediction-given genetic architecture (BLUP|GA) (&#xD7;100) calculated with the assigned weight (vertical axes) and &lt;em&gt;top%&lt;/em&gt; (horizontal axes) is shown in each cell of the heat map. Red area shows scenarios that BLUP|GA performs worse than genomic best linear unbiased prediction (0.758), green area shows scenarios that BLUP|GA performs better than BayesB (0.828). The optimal parameter combination obtained from reference population by cross validation is shown in black box.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/615/F3.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/615/F3.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/615/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/615/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-63" class="first-child">Heat map of the best linear unbiased prediction−given genetic architecture (BLUP|GA) accuracy for trait T3 in the validation population from QTLMAS dataset. The accuracy of best linear unbiased prediction-given genetic architecture (BLUP|GA) (×100) calculated with the assigned weight (vertical axes) and <em>top%</em> (horizontal axes) is shown in each cell of the heat map. Red area shows scenarios that BLUP|GA performs worse than genomic best linear unbiased prediction (0.758), green area shows scenarios that BLUP|GA performs better than BayesB (0.828). The optimal parameter combination obtained from reference population by cross validation is shown in black box.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-18" class="subsection"><h3>Genetic variance explained by top SNPs</h3><p id="p-64">Results in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a> clearly show that BLUP|GA improved the accuracy for FP and MY, but not for SCS. To determine the feature of a trait on which the accuracy of WGP can be improved, we calculated the genetic variance explained by each marker as <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-32.gif"/></noscript></span></span>, where <em>p</em> and <em>α</em> are the allele frequency and the estimated allele substitution effect for the marker under consideration. Then, we sorted all markers by their size of estimated effects (|<em>α|</em>) in decreasing order, and finally plotted the cumulative proportion of genetic variance explained by the ordered SNPs for each scenario. The proportion of genetic variance explained by the top 1%, 10%, and 100% SNPs are shown in panels A, B, and C in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. Interestingly, the differences among the three curves occur mainly at the top SNPs, especially for the top ~0.1% SNPs (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), and the curves are nearly parallel for the remaining part (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). For fat percentage, more than one third of the genetic variance is explained by the top 1% SNPs; moreover, only the top 0.1% SNPs (43 SNPs) explain ~28% of the genetic variance. By contrast, the top 1% SNPs explain only ~13% of the genetic variance for somatic cell score (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/615/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative proportion of genetic variance explained by single-nucleotide polymorphisms (SNPs). The top 1% (A), 10% (B) and 100% (C) SNPs were sorted by the size of estimated effects in decreasing order. Results for fat percentage, milk yield, and somatic cell score were plotted with blue solid lines, green dash lines and red dotted lines, respectively. The marker weights for genomic best linear unbiased prediction are shown by black solid lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1321135435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cumulative proportion of genetic variance explained by single-nucleotide polymorphisms (SNPs). The top 1% (A), 10% (B) and 100% (C) SNPs were sorted by the size of estimated effects in decreasing order. Results for fat percentage, milk yield, and somatic cell score were plotted with blue solid lines, green dash lines and red dotted lines, respectively. The marker weights for genomic best linear unbiased prediction are shown by black solid lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/615/F4.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/4/615/F4.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/615/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/615/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465800" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-65" class="first-child">Cumulative proportion of genetic variance explained by single-nucleotide polymorphisms (SNPs). The top 1% (A), 10% (B) and 100% (C) SNPs were sorted by the size of estimated effects in decreasing order. Results for fat percentage, milk yield, and somatic cell score were plotted with blue solid lines, green dash lines and red dotted lines, respectively. The marker weights for genomic best linear unbiased prediction are shown by black solid lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">Additionally, we analyzed the proportion of genetic variance explained by top SNPs in scenarios with different population sizes (<em>N</em>) for all the three model traits in the dairy cattle dataset. By decreasing the population size from 5024 to 125, not only the size of estimated marker effects (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), but also the proportion of genetic variance explained by top SNPs (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS5.pdf">Figure S5</a>) decreased for all three traits. The same trend was observed for milk yield in another study within the German Holstein population (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2011</a>). However, the impact of population size differed among traits. For example, while decreasing the population size from 5024 to 125, the genetic variance explained by top 1% SNPs decreased from 36 to 12% for FP (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS3.pdf">Figure S3</a>), but it changed from 13 to 11% only for SCS (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS5.pdf">Figure S5</a>).</p><p id="p-67">To validate the explanatory power of the pattern observed in the cattle dataset, we plotted the proportion of genetic variance explained by top SNPs for Rust_bin and Rootnum_bin in the loblolly pine dataset (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS7.pdf">Figure S7</a>). By decreasing the population size from 807 to 202 (~25%), the genetic variance explained by top 1% SNPs decreased from 18.5 to 16.4% for rust_bin (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS6.pdf">Figure S6</a>), but no decrease was observed for rootnum_bin (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS7.pdf">Figure S7</a>).</p></div><div id="sec-19" class="subsection"><h3>Trait-specific genetic variance-covariance matrix</h3><p id="p-68">To investigate the causes of the differential predictive ability of GBLUP and BLUP|GA, we plotted heat maps of the <strong>G</strong> matrix and the three <strong>S</strong> matrices (component of <strong>T</strong> matrix; see the section <em>Materials and Methods</em> for details) built for the three traits in the dairy cattle dataset (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Individuals in all <strong>S</strong> matrices were ordered by the genotypes of SNP with the largest estimated effect on fat%. The <strong>S</strong> matrix for FP and MY showed apparent blocks (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, B and C</a>), which is reflecting the three DGAT1 genotypes, and was distinct from the <strong>G</strong> matrix (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>), whereas the difference between <strong>S</strong> and <strong>G</strong> for somatic cell score (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>) was only marginal.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/615/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat maps of the realized relationship matrix (G) and three trait-specific relationship matrices (S) in dairy cattle dataset. The G matrix was built with all markers (A), and S matrices were built with top 1% SNPs for fat% (B), milk yield (C), and somatic cell score (D), respectively. These matrices were calculated with the genotypes of 1000 randomly selected bulls, and these bulls were sorted by their genotypes of the SNP with the largest marker effects for each trait." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1321135435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heat maps of the realized relationship matrix (G) and three trait-specific relationship matrices (S) in dairy cattle dataset. The &lt;strong&gt;G&lt;/strong&gt; matrix was built with all markers (A), and &lt;strong&gt;S&lt;/strong&gt; matrices were built with top 1% SNPs for fat% (B), milk yield (C), and somatic cell score (D), respectively. These matrices were calculated with the genotypes of 1000 randomly selected bulls, and these bulls were sorted by their genotypes of the SNP with the largest marker effects for each trait.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/615/F5.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/4/615/F5.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/615/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/615/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465815" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-69" class="first-child">Heat maps of the realized relationship matrix (G) and three trait-specific relationship matrices (S) in dairy cattle dataset. The <strong>G</strong> matrix was built with all markers (A), and <strong>S</strong> matrices were built with top 1% SNPs for fat% (B), milk yield (C), and somatic cell score (D), respectively. These matrices were calculated with the genotypes of 1000 randomly selected bulls, and these bulls were sorted by their genotypes of the SNP with the largest marker effects for each trait.</p><div class="sb-div caption-clear"></div></div></div><p id="p-70">The results suggest that the advantage of BLUP|GA over GBLUP is the more pronounced the more the <strong>T</strong> and <strong>G</strong> matrices differ from each other. Therefore, we quantified the distance between <strong>T</strong> and <strong>G</strong> by calculating the standard deviation of the element-wise difference between the two matrices, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/615/embed/mml-math-33.gif"/></noscript></span></span>. Next we calculated the linear regression of absolute increased accuracy of BLUP|GA over GBLUP (termed ∆ in the following) on σ. The scatter plots and regression lines for each dataset are shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The intercepts were set to zero for all regressions since zero is the expectation when <strong>T</strong> = <strong>G</strong>. All regression coefficients are significant (<em>P</em> &lt; 0.01). The largest regression coefficient was observed for the cattle dataset (2.17) while the smallest one was observed for the pine dataset (0.84).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/615/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regression of absolute increased accuracy of best linear unbiased prediction -given genetic architecture (BLUP|GA) over genomic best linear unbiased prediction (&#x2206;) on the distance between T and G matrices (&#x3C3;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1321135435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Regression of absolute increased accuracy of best linear unbiased prediction -given genetic architecture (BLUP|GA) over genomic best linear unbiased prediction (&#x2206;) on the distance between T and G matrices (&#x3C3;).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/615/F6.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/4/615/F6.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/615/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/615/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465817" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-71" class="first-child">Regression of absolute increased accuracy of best linear unbiased prediction -given genetic architecture (BLUP|GA) over genomic best linear unbiased prediction (∆) on the distance between T and G matrices (σ).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-72">In the present study, we propose a new strategy of WGP to implement the BLUP|GA approach and validate this strategy in a variety of genetic architectures, including livestock, crop, and simulated datasets. Although many methods exist in this discipline, the new strategy of BLUP|GA provides a plausible alternative to established methods and is shown to flexibly adapt to the given genetic architecture of the trait of interest.</p><p id="p-73">BLUP|GA could effectively improve the predictive performance of WGP compared to GBLUP. We validated the performance of BLUP|GA and GBLUP in four different datasets (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The validated traits included various genetic architectures, such as traits with major gene(s) [fat percentage in dairy cattle, rust resistant in loblolly pine (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wilcox <em>et al.</em> 1996</a>)], traits with loci of moderate effect (milk yield in dairy cattle, three traits in QTL-MAS, GammaUnres and GammaRes in GSA), and traits with only small effect loci (somatic cell score, PolyUnres and PolyRes in GSA). Results showed that BLUP|GA outperformed GBLUP in 20 of 21 scenarios (three traits by seven population sizes) in the dairy cattle dataset (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>) and in 12 of 13 traits in the three public datasets (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). In addition, BLUP|GA performed better than BayesA, BayesB, and BayesC in 12 of 13 traits in the three public datasets (<a id="xref-table-wrap-5-11" class="xref-table" href="#T5">Table 5</a>). In the cattle dataset, BLUP|GA showed advantage over BayesB with N = 125 and 500, but not for N = 2000 for the trait fat% (<a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/TableS1.pdf">Table S1</a>). Because GBLUP and BayesB were suggested as standard approaches for method validation, our results strongly indicate that BLUP|GA is a promising approach that has the potential to flexibly account for effects of genetic architecture when they are relevant.</p><p id="p-74">An interesting question raised by the favorable performance of BLUP|GA in these tested datasets is, why could BLUP|GA improve the prediction ability compared with GBLUP? The difference between GBLUP and BLUP|GA lies in the variance-covariance matrix. BLUP|GA replaces the <strong>G</strong> matrix with a <strong>T</strong> matrix. Since the <strong>T</strong> matrix was a combination of genetic architecture part (<strong>S</strong> matrix) and the <strong>G</strong> matrix according to <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">equation 4</a>, the favorable performance of BLUP|GA is mainly due to the genetic architecture part (<strong>S</strong> matrix) and the way we build the <strong>T</strong> matrix (weighted sum, <a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">equation 4</a>).</p><p id="p-75">To understand the reason of increase in accuracy, we first investigated heat maps of these relationship matrices. The contrast between the heat maps of <strong>S</strong> from three dairy cattle model traits and the <strong>G</strong> matrix (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>), together with the accuracy from GBLUP and BLUP|GA (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>), strongly indicate that the genetic architecture part (<strong>S</strong> matrix) could effectively improve the prediction ability of the <strong>G</strong> matrix. The block structure of <strong>S</strong> for FP mainly reflects the three DGAT1 genotypes, which is a validated major gene affecting milk traits in dairy cattle. This finding suggests that the increase in accuracy is possibly due to the increased similarity between <strong>T</strong> and the genetic relationship at unobserved causal loci (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013b</a>) compared with <strong>G</strong> matrix used in GBLUP. The improvement of accuracy for SCS is the least, while the similarity between the <strong>S</strong> matrix of SCS and <strong>G</strong> matrix is the greatest among the three traits (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). In addition, we calculated the regression of increased accuracy of BLUP|GA over GBLUP (∆) on the distance between <strong>T</strong> and <strong>G</strong> matrices (σ). Significant regression coefficients between them indicate that the <strong>T</strong> matrix used in BLUP|GA is the main explanation for the increased accuracy, and the amount of similarity between <strong>S</strong> and <strong>G</strong> matrices also affects the increment of accuracy. These observations together explain why we benefit from the <strong>S</strong> matrix for MY and FP but not for SCS. </p><p id="p-76">On the basis of these observations, we speculate that for a large number of quantitative traits, the <strong>S</strong> matrix built from their inferred genetic architecture is similar to the averaged <strong>G</strong> matrix. This similarity might come from a (quasi) infinitesimal genetic background or might also arise for a trait that is controlled by a limited but not extremely small number of genes distributed across the genome. If this is the case, it is not hard to image why the <strong>G</strong> matrix is so robust in its prediction ability and why we can successfully apply the same <strong>G</strong> matrix or numerator relationship matrix (<strong>A</strong> matrix (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Henderson 1975b</a>)) to genetic evaluation for different traits. Although these approached are robust, they are not optimal for all genetic architectures. With the availability of more precise genetic data (such as whole-genome sequences) and the ongoing research toward a more comprehensive understanding of the underlying genetic architectures, more and more quantitative traits could benefit from more informative <strong>S</strong> matrices built on such external information sources by using the approach suggested in an earlier study (<a id="xref-ref-52-8" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>).</p><p id="p-77">The way we build the <strong>T</strong> matrix ensures BLUP|GA being flexible to shift between the infinitesimal and major gene genetic architectures. The <strong>T</strong> matrix comes from a weighted sum of a realized / averaged information matrix (<strong>G</strong>) and a trait specific genetic architecture matrix (<strong>S</strong>) (defined in <a id="xref-disp-formula-4-4" class="xref-disp-formula" href="#disp-formula-4">equation 4</a>). Hence, the selection of the optimal overall weight in <a id="xref-disp-formula-4-5" class="xref-disp-formula" href="#disp-formula-4">equation 4</a> is important for the performance of BLUP|GA. Previous studies showed that improper weights can lead to suboptimal model predicting ability (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gao <em>et al.</em> 2012</a>; <a id="xref-ref-52-9" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>). The ideal situation is that the selected weight for <strong>S</strong> is (1) small for infinitesimal model traits to shift the genetic model to be more similar to GBLUP, and (2) large for traits affected by major gene(s) to shift the genetic model to be more similar to BayesB. Expectedly, the actual selected optimal weights are relatively large for the traits fat percentage in cattle and rust resistance in pine, and small for the traits somatic cell score in cattle and PolyUnres in the GSA dataset (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>). Hence, given a proper weight and an informative <strong>S</strong> matrix to form a <strong>T</strong> matrix, the predictive ability of BLUP|GA is promising by including this <strong>T</strong> matrix.</p><p id="p-78">To explore the potential of BLUP|GA, three optimal parameters must be provided or fitted: (1) the proportion of top SNPs (<em>top%</em>), (2) the number of flanking SNPs adjacent each top SNP (<em>nflank</em>), and (3) the overall weight for matrix <strong>S</strong> (<em>weight</em>). The three parameters are determined to maximize the predicting ability of BLUP|GA by maximizing the similarity of <strong>T</strong> and the genetic relationship at unobserved causal loci. The first two parameters are used to select the subset of ‘important markers’ while building an <strong>S</strong> matrix. The parameter <em>nflank</em> is relevant to the global and local level and range of linkage disequilibrium in the dataset. Although <em>nflank</em> is defined as a parameter to capture the linkage disequilibrium among the top SNPs and their flanking SNPs, the contribution of linkage disequilibrium to additive variance is not considered in this approach (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola <em>et al.</em> 2009</a>). We did not optimize <em>nflank</em> for the pine dataset due to the lack of marker map information (<em>nflank</em> = 0 for pine, <a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a>). In addition, the third parameter and a vector of marker weights corresponding to each selected marker are also used to build the <strong>T</strong> matrix via <a id="xref-disp-formula-4-6" class="xref-disp-formula" href="#disp-formula-4">equation 4</a> and <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">equation 5</a>.</p><p id="p-79">In this study, we determined the optimal parameters, especially the <em>top%</em> and <em>weight</em>, through a grid search within the parameter spaces in the training stage. This validation procedure can maximize the predicting ability of BLUP|GA within the reference set, though it may provide only suboptimal parameters for the application stage (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Hence, there is room to further improve the performance of WGP, and other means of determining the optimal parameters deserve further investigation. Compared with other variable selection models, such as BayesB, the computational burden of BLUP|GA is low (minutes <em>vs.</em> hours), and mainly dependent on the number of genotyped individuals and the training strategy used. The strategy proposed in this study to determine the optimal parameters for BLUP|GA might not be the most efficient one but is at least an effective way to obtain parameters that improve predictive ability, and similar strategies were used in other studies (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2011</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gao <em>et al.</em> 2012</a>; <a id="xref-ref-52-10" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>). Some alternative strategies to choose the BLUP|GA parameters, such as determining weights by variance component estimation, were tested but generally did not lead to improved performance, especially so with reference datasets of limited size (results not shown). More efficient ways to optimize the necessary parameters for BLUP|GA will be investigated in our further studies.</p><p id="p-80">Although results from training stage strongly demonstrate the advantage of BLUP|GA over GBLUP, results from both training stage and application stage are reported for the dairy cattle dataset and the QTLMAS dataset to show the entire computational strategy proposed for BLUP|GA. In practice, a training stage is necessary for BLUP|GA to determine the optimal parameters, especially for <em>weight</em> and <em>top%</em>. The way to determine the parameters can be very flexible, such as by CV or other means. These optimized parameters can then be used as basis for genomic prediction with BLUP|GA in a candidate population. Though the deviation in accuracies observed between the two stages is different in the two tested datasets (<a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Table 2</a>, <a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>, and <a id="xref-table-wrap-4-9" class="xref-table" href="#T4">Table 4</a>), and the optimal parameters obtained from training stage might not be the best in application stage (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, 0.838 <em>vs.</em> 0.848), the advantage of BLUP|GA over GBLUP exists (<a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a>, <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-10" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-12" class="xref-table" href="#T5">Table 5</a>) in both stages (training stage and application stage). This result indicated that BLUP|GA may be useful also under practical circumstances.</p><p id="p-81">In most implementations of WGP, the GBLUP approach is recommended as a benchmarking method because of its proven robustness. Are there suitable diagnostics that can be derived from the data at hand, which indicate whether the performance of GBLUP can be improved upon by alternative models that take the genomic architecture into account? We suggest as diagnostics the plot of cumulative genetic variance curves (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figures 2</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS5.pdf">Figure S5</a> for the full and decreased population sizes in the cattle dataset and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/09/g3.114.016261.DC1/FigureS7.pdf">Figure S7</a> for the traits rust_bin and rootnum_bin in the pine dataset). From these curves, it becomes evident that a high proportion of variance explained by a small proportion of top SNPs is indicative for the potential to outperform GBLUP by BLUP|GA or some other approach accounting for genetic architecture. A second diagnostic is the difference between the <strong>G</strong> and the <strong>S</strong> matrix, which can be expected visually (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> for the three cattle traits) or summarized in a suitable statistic such as the standard deviation of element wise differences σ. Our results suggest that the advantage of BLUP|GA will be the larger, the more differentiated these matrices are. The suggested diagnostics can be derived from a dataset at hand, so that decisions on the choice of an appropriate model can be made on the spot. However, it should be noted that BLUP|GA as implemented in this study is a general approach which encloses GBLUP as a special case (with either <em>weight</em> or <em>top%</em> being 0), and thus will “automatically” find the best model—which might be close to GBLUP in many cases—when calibrated. So, applying BLUP|GA would ease the burden of model selection in the practice of WGP.</p><p id="p-82">With the rapid reduction of sequencing cost, sequence data will soon be available for the use in breeding programs. In this case, if the number of individuals involved was less than the number of markers, WGP approaches based on some sort of genomic relationship matrix are computationally much more efficient than approaches based on the estimation of SNP effects (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ober <em>et al.</em> 2012</a>). On the basis of the aforementioned ideas, it might be an efficient strategy to construct the realized relationship matrix (<strong>G</strong>) with a sample of evenly spaced markers first. Then, one could improve the prediction model of BLUP|GA by building an <strong>S</strong> matrix with the top SNPs extracted from the dataset in hand according to their effect size as proposed in this study, the significance of association (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013b</a>), or from public sources as proposed in <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kadarmideen (2014)</a> and in our former study (<a id="xref-ref-52-11" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2014</a>).</p><p id="p-83">The new strategy applied for BLUP|GA is a favorable alternative to the standard GBLUP model, which better accounts for the genetic architecture of the quantitative trait under consideration. This feature is mainly due to the increased similarity between trait specific variance-covariance relationship matrix <strong>T</strong> and the genetic relationship matrix at unobserved causal loci. Given a subset of important markers that possibly locate in QTL regions and proper corresponding weights, genomic prediction could be successfully conducted with the BLUP|GA model in the four datasets analyzed in the present study and in most cases was found to be among the best methods for genomic prediction. The pattern of curves from cumulative proportion of genetic variance explained by top SNPs and the element wise standard deviation between <strong>G</strong> and <strong>T</strong> matrices might be a good diagnostic indicator for model selection.</p></div><div class="section" id="sec-21"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-84">We thank the vit (Vereinigte Informationssysteme Tierhaltung w.V.) for providing the German Holstein data. We also appreciate very much the suggestions and comments by two reviewers and the associate editor which all improved the manuscript considerably. This work was supported by the National Natural Science Foundation of China (31200925, 31371258), the earmarked fund for China Agriculture Research System (CARS-36), the Ph.D. Programs Foundation (the Doctoral Fund) of Ministry of Education of China (20124404120001). H.S., M.E., and U.O. acknowledge the funding by the German Federal Ministry of Education and Research within the AgroClustEr “Synbreed – Synergistic plant and animal breeding” (Funding ID: 0315528C). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-23"><p id="p-85">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016261/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016261/-/DC1</a></p></li><li class="fn" id="fn-24"><p id="p-86"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-01-08"><span class="received-label">Received </span>January 8, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-05"><span class="accepted-label">Accepted </span>February 5, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Zhang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.615.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A common dataset for genomic analysis of livestock populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.3" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.4" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.5" data-doi="10.1534/genetics.112.139907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Setting the standard: a special focus on genomic selection in GENETICS and G3.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253Dde%2BKoning%26rft.auinit1%253DD.-J.%26rft.volume%253D190%26rft.issue%253D4%26rft.spage%253D1151%26rft.epage%253D1152%26rft.atitle%253DSetting%2Bthe%2BStandard%253A%2BA%2BSpecial%2BFocus%2Bon%2BGenomic%2BSelection%2Bin%2BGENETICS%2Band%2BG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139907%26rft_id%253Dinfo%253Apmid%252F22491887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzExNTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.6" data-doi="10.1038/nrg2898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predicting genetic predisposition in humans: the promise of whole-genome markers.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">880</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D880%26rft.epage%253D886%26rft.atitle%253DPredicting%2Bgenetic%2Bpredisposition%2Bin%2Bhumans%253A%2Bthe%2Bpromise%2Bof%2Bwhole-genome%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2898%26rft_id%253Dinfo%253Apmid%252F21045869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2898&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21045869&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.7" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.8" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.9" data-doi="10.3168/jds.2012-6194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of genomic prediction for milk production traits in the Chinese Holstein population using a reference population consisting of cows.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">5315</span>–<span class="cit-lpage">5323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D5315%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2012-6194%26rft_id%253Dinfo%253Apmid%252F23746588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2012-6194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746588&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.10" data-doi="10.1371/journal.pgen.1003348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudbridge</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Power and predictive accuracy of polygenic risk scores.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003348%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003348%26rft_id%253Dinfo%253Apmid%252F23555274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555274&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.11" data-doi="10.1534/genetics.110.115543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda L</em>., Pinaceae).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115543%26rft_id%253Dinfo%253Apmid%252F20439779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.12" data-doi="10.1046/j.1439-0388.2001.00290.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Marker-based estimates of between and within population kinships for the conservation of genetic diversity.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D118%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0388.2001.00290.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0388.2001.00290.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169434900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.13" data-doi="10.1186/1297-9686-44-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">U. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison on genomic predictions using three GBLUP methods and two single-step blending methods in the Nordic Holstein population.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGao%26rft.auinit1%253DH.%26rft.volume%253D44%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DComparison%2Bon%2Bgenomic%2Bpredictions%2Busing%2Bthree%2BGBLUP%2Bmethods%2Band%2Btwo%2Bsingle-step%2Bblending%2Bmethods%2Bin%2Bthe%2BNordic%2BHolstein%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-8%26rft_id%253Dinfo%253Apmid%252F22455934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22455934&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.615.14"><div class="cit-metadata unstructured">Gaspa, G., M. G. Usai, S. Casu, A. Carta, and N. P. P. Macciotta, 2012 Comparative analysis of submitted results on genomic evaluation and applied methods. Available at: <a href="http://qtl-mas-2012.kassiopeagroup.com/presentazioni/23_Gaspa.pdf">http://qtl-mas-2012.kassiopeagroup.com/presentazioni/23_Gaspa.pdf</a>. Accessed February 12, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.15" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and the Bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.16" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.615.17" data-doi="10.1534/g3.112.005066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of across-environment genome-wide prediction in maize nested association mapping populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D263%26rft.epage%253D272%26rft.atitle%253DAccuracy%2Bof%2BAcross-Environment%2BGenome-Wide%2BPrediction%2Bin%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005066%26rft_id%253Dinfo%253Apmid%252F23390602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8yNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.18" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.19" data-doi="10.1371/journal.pgen.1001139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic architecture of complex traits and accuracy of genomic prediction: coat colour, milk-fat percentage, and type in Holstein cattle as contrasting model traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De1001139%26rft.epage%253De1001139%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Band%2Baccuracy%2Bof%2Bgenomic%2Bprediction%253A%2Bcoat%2Bcolour%252C%2Bmilk-fat%2Bpercentage%252C%2Band%2Btype%2Bin%2BHolstein%2Bcattle%2Bas%2Bcontrasting%2Bmodel%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001139%26rft_id%253Dinfo%253Apmid%252F20927186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927186&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.20" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span><span class="cit-comment">a</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.21" data-doi="10.3168/jds.S0022-0302(75)84776-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span><span class="cit-comment">b</span> <span class="cit-article-title">Rapid method for computing the inverse of a relationship matrix.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1727</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D58%26rft.issue%253D11%26rft.spage%253D1727%26rft.epage%253D1730%26rft.atitle%253DRapid%2BMethod%2Bfor%2BComputing%2Bthe%2BInverse%2Bof%2Ba%2BRelationship%2BMatrix%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252875%252984776-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(75)84776-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1975AX14500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.615.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Simulated data for genomic selection and genome-wide association studies using a combination of coalescent and gene drop methods.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.23" data-doi="10.1093/nar/gks1150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reecy</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Animal QTLdb: an improved database tool for livestock animal QTL/association data dissemination in the post-genome era.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D871</span>–<span class="cit-lpage">D879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1150%26rft_id%253Dinfo%253Apmid%252F23180796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9EODcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.24" data-doi="10.1016/j.livsci.2014.04.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadarmideen</span>  <span class="cit-name-given-names">H. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics to systems biology in animal and veterinary sciences: progress, lessons and opportunities.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.04.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.04.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.25" data-doi="10.1371/journal.pgen.1000231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Predicting unobserved phenotypes for complex traits from whole-genome SNP data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BH.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000231%26rft.epage%253De1000231%26rft.atitle%253DPredicting%2Bunobserved%2Bphenotypes%2Bfor%2Bcomplex%2Btraits%2Bfrom%2Bwhole-genome%2BSNP%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000231%26rft_id%253Dinfo%253Apmid%252F18949033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18949033&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.26" data-doi="10.1186/1297-9686-43-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rensing</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Impacts of both reference population size and inclusion of a residual polygenic effect on the accuracy of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D43%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DImpacts%2Bof%2Bboth%2Breference%2Bpopulation%2Bsize%2Band%2Binclusion%2Bof%2Ba%2Bresidual%2Bpolygenic%2Beffect%2Bon%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-19%26rft_id%253Dinfo%253Apmid%252F21586131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21586131&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.27" data-doi="10.1186/1297-9686-43-43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducrocq</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A common reference population from four European Holstein populations increases reliability of genomic predictions.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLund%26rft.auinit1%253DM.%2BS.%26rft.volume%253D43%26rft.spage%253D43%26rft.epage%253D43%26rft.atitle%253DA%2Bcommon%2Breference%2Bpopulation%2Bfrom%2Bfour%2BEuropean%2BHolstein%2Bpopulations%2Bincreases%2Breliability%2Bof%2Bgenomic%2Bpredictions.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-43%26rft_id%253Dinfo%253Apmid%252F22152008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22152008&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.4.615.28"><div class="cit-metadata unstructured">Madsen, P., P. Sørensen, G. Su, L. H. Damgaard, H. Thomsen <em>et al.</em>, 2006 DMU—a package for analyzing multivariate mixed models, in <em>8th World Congress on Genetics Applied to Livestock Production</em>, Brazil.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.29" data-doi="10.1371/journal.pone.0005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and characterization of a high density SNP genotyping assay for cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMatukumalli%26rft.auinit1%253DL.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5350%26rft.epage%253De5350%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bfor%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005350%26rft_id%253Dinfo%253Apmid%252F19390634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390634&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.30" data-doi="10.1534/genetics.110.116590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accurate prediction of genetic values for complex traits by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116590%26rft_id%253Dinfo%253Apmid%252F20308278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.32" data-doi="10.3168/jds.2009-2064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Computing procedures for genetic evaluation including phenotypic, full pedigree, and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4648</span>–<span class="cit-lpage">4655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMisztal%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4648%26rft.epage%253D4655%26rft.atitle%253DComputing%2Bprocedures%2Bfor%2Bgenetic%2Bevaluation%2Bincluding%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2064%26rft_id%253Dinfo%253Apmid%252F19700728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700728&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.33" data-doi="10.1371/journal.pgen.1002685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using whole-genome sequence data to predict quantitative trait phenotypes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DOber%26rft.auinit1%253DU.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002685%26rft.epage%253De1002685%26rft.atitle%253DUsing%2Bwhole-genome%2Bsequence%2Bdata%2Bto%2Bpredict%2Bquantitative%2Btrait%2Bphenotypes%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002685%26rft_id%253Dinfo%253Apmid%252F22570636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570636&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.34" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard dataset of loblolly pine (<em>Pinus taeda L</em>.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.35" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.36" data-doi="10.1186/1471-2164-13-452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D452%26rft.epage%253D452%26rft.atitle%253DComparison%2Bof%2Bwhole-genome%2Bprediction%2Bmodels%2Bfor%2Btraits%2Bwith%2Bcontrasting%2Bgenetic%2Barchitecture%2Bin%2Ba%2Bdiversity%2Bpanel%2Bof%2Bmaize%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-452%26rft_id%253Dinfo%253Apmid%252F22947126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22947126&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.615.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Cross-validatory choice and assessment of statistical predictions.</span> <span class="cit-source">J. Roy. Stat. Soc. Ser. B. (Stat. Method.)</span> <span class="cit-vol">36</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRoy.%2BStat.%2BSoc.%2BSer.%2BB.%2B%2528Stat.%2BMethod.%2529%26rft.volume%253D36%26rft.spage%253D111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.38" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical note: derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.39" data-doi="10.1186/1297-9686-43-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Allele coding in genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D43%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAllele%2Bcoding%2Bin%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-25%26rft_id%253Dinfo%253Apmid%252F21703021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21703021&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.615.40"><div class="cit-metadata unstructured">Usai, M. G., G. Gaspa, S. Casu, N. P. P. Macciotta, and A. Carta, 2012  QTLMAS 2012: Simulated data set. Available at: <a href="http://qtl-mas-2012.kassiopeagroup.com/">http://qtl-mas-2012.kassiopeagroup.com/</a>. Accessed February 15, 2105.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.41" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.42" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: reliability of genomic predictions for North American Holstein bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.43" data-doi="10.1534/genetics.112.141705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comprehensive genetic approach for improving prediction of skin cancer risk in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141705%26rft_id%253Dinfo%253Apmid%252F23051645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE0OTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association mapping including phenotypes from relatives without genotypes in a single-step (ssGWAS) for 6-week body weight in broiler chickens.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D134%26rft_id%253Dinfo%253Apmid%252F24904635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24904635&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.45" data-doi="10.1073/pnas.93.9.3859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amerson</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhlman</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Detection of a major gene for resistance to fusiform rust disease in loblolly pine by genomic mapping.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">3859</span>–<span class="cit-lpage">3864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWilcox%26rft.auinit1%253DP.%2BL.%26rft.volume%253D93%26rft.issue%253D9%26rft.spage%253D3859%26rft.epage%253D3864%26rft.atitle%253DDetection%2Bof%2Ba%2Bmajor%2Bgene%2Bfor%2Bresistance%2Bto%2Bfusiform%2Brust%2Bdisease%2Bin%2Bloblolly%2Bpine%2Bby%2Bgenomic%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.9.3859%26rft_id%253Dinfo%253Apmid%252F8632980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5My85LzM4NTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.46" data-doi="10.1534/genetics.113.150078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide prediction of traits with different genetic architecture through efficient variable selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150078%26rft_id%253Dinfo%253Apmid%252F23934883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.47" data-doi="10.1186/1297-9686-43-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stricker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arango</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Breeding value prediction for production traits in layer chickens using pedigree or genomic relationships in a reduced animal model.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DWolc%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DBreeding%2Bvalue%2Bprediction%2Bfor%2Bproduction%2Btraits%2Bin%2Blayer%2Bchickens%2Busing%2Bpedigree%2Bor%2Bgenomic%2Brelationships%2Bin%2Ba%2Breduced%2Banimal%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-5%26rft_id%253Dinfo%253Apmid%252F21255418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21255418&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.48" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.49" data-doi="10.1371/journal.pone.0012648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Best linear unbiased prediction of genomic breeding values using trait-specific marker-derived relationship matrix.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253De12648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012648%26rft_id%253Dinfo%253Apmid%252F20844593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844593&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.50" data-doi="10.3168/jds.2010-3917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of genomic prediction using low density marker panels.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3642</span>–<span class="cit-lpage">3650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D94%26rft.issue%253D7%26rft.spage%253D3642%26rft.epage%253D3650%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bprediction%2Busing%2Blow-density%2Bmarker%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3917%26rft_id%253Dinfo%253Apmid%252F21700054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21700054&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.51" data-doi="10.1007/s11434-011-4632-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Advances in genomic selection in domestic animal.</span> <abbr class="cit-jnl-abbrev">Chin. Sci. Bull.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2655</span>–<span class="cit-lpage">2663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin.%2BSci.%2BBull.%26rft.volume%253D56%26rft.spage%253D2655%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11434-011-4632-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11434-011-4632-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.615.52" data-doi="10.1371/journal.pone.0093017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improving the accuracy of whole genome prediction for complex traits using the results of genome wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e93017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De93017%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093017%26rft_id%253Dinfo%253Apmid%252F24663104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24663104&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F615.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/615.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/605" title="Gene Regulation by H-NS as a Function of Growth Conditions Depends on Chromosomal Position in Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/629" title="Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/615?related-urls=yes&amp;legid=ggg;5/4/615" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NAY-nlnKDfSvkzcwjq8Kli0nbp37DpRaTEaejKy3TuY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/615" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/615" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/615" class="active" data-icon-position="" data-hide-link-title="0">Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459811" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pob1qrEKYJXy8TB1qz_-k3ysmRATKttj9uRez4AIbuE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/615.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/615" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IKXK7Pr4JmcfkF6ky_VeLLxL_lylw_VgpwXLB3kULHc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/615" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/615.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459811" id="node459811--2" data-pisa="ggg;5/4/615" data-pisa-master="ggg;g3.114.016261" data-apath="/ggg/5/4/615.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhe Zhang</span>, <span class="highwire-citation-author" data-delta="1">Malena Erbe</span>, <span class="highwire-citation-author" data-delta="2">Jinlong He</span>, <span class="highwire-citation-author" data-delta="3">Ulrike Ober</span>, <span class="highwire-citation-author" data-delta="4">Ning Gao</span>, <span class="highwire-citation-author" data-delta="5">Hao Zhang</span>, <span class="highwire-citation-author" data-delta="6">Henner Simianer</span> and <span class="highwire-citation-author" data-delta="7">Jiaqi Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">615-627;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016261 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459811--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhe Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhe%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhe%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malena Erbe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malena%2BErbe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erbe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalena%2BErbe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jinlong He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinlong%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinlong%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulrike Ober </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrike%2BOber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ober%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlrike%2BOber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ning Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ning%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANing%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henner Simianer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henner%2BSimianer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simianer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenner%2BSimianer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hsimian@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hsimian@gwdg.de</a>
<a href="mailto:jqli@scau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jqli@scau.edu.cn</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jiaqi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaqi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiaqi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hsimian@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hsimian@gwdg.de</a>
<a href="mailto:jqli@scau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jqli@scau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/615" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/615&amp;title=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/615&amp;title=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/615&amp;title=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/615&amp;text=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/615&amp;title=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/615&amp;t=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/615&amp;title=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/615&amp;title=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459811/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459811" id="node459811--3" data-pisa="ggg;5/4/615" data-pisa-master="ggg;g3.114.016261" data-apath="/ggg/5/4/615.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhe Zhang</span>, <span class="highwire-citation-author" data-delta="1">Malena Erbe</span>, <span class="highwire-citation-author" data-delta="2">Jinlong He</span>, <span class="highwire-citation-author" data-delta="3">Ulrike Ober</span>, <span class="highwire-citation-author" data-delta="4">Ning Gao</span>, <span class="highwire-citation-author" data-delta="5">Hao Zhang</span>, <span class="highwire-citation-author" data-delta="6">Henner Simianer</span> and <span class="highwire-citation-author" data-delta="7">Jiaqi Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">615-627;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016261 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459811--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhe Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhe%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhe%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malena Erbe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malena%2BErbe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Erbe%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalena%2BErbe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jinlong He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinlong%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinlong%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulrike Ober </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrike%2BOber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ober%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlrike%2BOber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ning Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ning%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANing%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henner Simianer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Sciences, Animal Breeding and Genetics Group, Georg-August-Universität Göttingen, Göttingen 37075, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henner%2BSimianer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simianer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenner%2BSimianer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hsimian@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hsimian@gwdg.de</a>
<a href="mailto:jqli@scau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jqli@scau.edu.cn</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jiaqi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Research Center for Breeding Swine Industry, Guangdong Provincial Key Lab of Agro-animal Genomics and Molecular Breeding, College of Animal Science, South China Agricultural University, Guangzhou 510642, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaqi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiaqi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hsimian@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">hsimian@gwdg.de</a>
<a href="mailto:jqli@scau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jqli@scau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=615&amp;rft.epage=627&amp;rft.atitle=Accuracy%20of%20Whole-Genome%20Prediction%20Using%20a%20Genetic%20Architecture-Enhanced%20Variance-Covariance%20Matrix&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhang%2C+Zhe&amp;rft.au=Erbe%2C+Malena&amp;rft.au=He%2C+Jinlong&amp;rft.au=Ober%2C+Ulrike&amp;rft.au=Gao%2C+Ning&amp;rft.au=Zhang%2C+Hao&amp;rft.au=Simianer%2C+Henner&amp;rft.au=Li%2C+Jiaqi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459811/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459811/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459811/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459811/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459811/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459811/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459811/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459811/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459811/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459811/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459811/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459811/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/615" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/615.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/615.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/615.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/615.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
