<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/541" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/541.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/541.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/541.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/4/541/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/541.atom" />
<meta name="HW.pisa" content="ggg;5/4/541" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population" />
<meta name="DC.Identifier" content="10.1534/g3.114.015263" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Lorenz et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Maize silage is forage of high quality and yield, and represents the second most important use of maize in the United States. The Wisconsin Quality Synthetic (WQS) maize population has undergone five cycles of recurrent selection for silage yield and composition, resulting in a genetically improved population. The application of high-density molecular markers allows breeders and geneticists to identify important loci through association analysis and selection mapping, as well as to monitor changes in the distribution of genetic diversity across the genome. The objectives of this study were to identify loci controlling variation for maize silage traits through association analysis and the assessment of selection signatures and to describe changes in the genomic distribution of gene diversity through selection and genetic drift in the WQS recurrent selection program. We failed to find any significant marker-trait associations using the historical phenotypic data from WQS breeding trials combined with 17,719 high-quality, informative single nucleotide polymorphisms. Likewise, no strong genomic signatures were left by selection on silage yield and quality in the WQS despite genetic gain for these traits. These results could be due to the genetic complexity underlying these traits, or the role of selection on standing genetic variation. Variation in loss of diversity through drift was observed across the genome. Some large regions experienced much greater loss in diversity than what is expected, suggesting limited recombination combined with small populations in recurrent selection programs could easily lead to fixation of large swaths of the genome." />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="DC.Contributor" content="Timothy M. Beissinger" />
<meta name="DC.Contributor" content="Renato Rodrigues Silva" />
<meta name="DC.Contributor" content="Natalia de Leon" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Maize silage is forage of high quality and yield, and represents the second most important use of maize in the United States. The Wisconsin Quality Synthetic (WQS) maize population has undergone five cycles of recurrent selection for silage yield and composition, resulting in a genetically improved population. The application of high-density molecular markers allows breeders and geneticists to identify important loci through association analysis and selection mapping, as well as to monitor changes in the distribution of genetic diversity across the genome. The objectives of this study were to identify loci controlling variation for maize silage traits through association analysis and the assessment of selection signatures and to describe changes in the genomic distribution of gene diversity through selection and genetic drift in the WQS recurrent selection program. We failed to find any significant marker-trait associations using the historical phenotypic data from WQS breeding trials combined with 17,719 high-quality, informative single nucleotide polymorphisms. Likewise, no strong genomic signatures were left by selection on silage yield and quality in the WQS despite genetic gain for these traits. These results could be due to the genetic complexity underlying these traits, or the role of selection on standing genetic variation. Variation in loss of diversity through drift was observed across the genome. Some large regions experienced much greater loss in diversity than what is expected, suggesting limited recombination combined with small populations in recurrent selection programs could easily lead to fixation of large swaths of the genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/541" />
<meta name="citation_id" content="5/4/541" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/541" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/541.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/541.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/541.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015263" />
<meta name="citation_pmid" content="25645532" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="541" />
<meta name="citation_lastpage" content="549" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author_email" content="alorenz2@unl.edu" />
<meta name="citation_author" content="Timothy M. Beissinger" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Renato Rodrigues Silva" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Natalia de Leon" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. K. Anderson;citation_author=G. G. Doyle;citation_author=B. Brigham;citation_author=J. Carter;citation_author=K. D. Hooker;citation_title=High-Resolution Crossover Maps for Each Bivalent of Zea mays Using Recombination Nodules;citation_pages=849-865;citation_volume=165;citation_year=2003;citation_issue=2;citation_pmid=14573493" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=O. Argillier;citation_author=V. Méchin;citation_author=Y. Barriere;citation_title=Inbred Line Evaluation and Breeding for Digestibility-Related Traits in Forage Maize;citation_pages=1596-1600;citation_volume=40;citation_year=2000;citation_issue=6;citation_doi=10.2135/cropsci2000.4061596x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. D. Barrett;citation_author=H. E. Hoekstra;citation_title=Molecular spandrels: tests of adaptation at the genetic level.;citation_pages=767-780;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=22005986;citation_doi=10.1038/nrg3015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=B. Vaillancourt;citation_author=S. Deshpande;citation_author=K. Barry;citation_title=A genome-wide scan for evidence of selection in a maize population under long-term artificial selection for ear number.;citation_volume=196;citation_year=2014;citation_pmid=24381334;citation_doi=10.1534/genetics.113.160655" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. J. Berg;citation_author=G. Coop;citation_title=A population genetic signal of polygenic adaptation.;citation_pages=e1004412;citation_volume=10;citation_year=2014;citation_pmid=25102153;citation_doi=10.1371/journal.pgen.1004412" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_title=ASReml-R Reference Manual;citation_author=D. G. Butler;citation_author=B. R. Cullis;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. V. Butruille;citation_author=H. D. Silva;citation_author=S. M. Kaeppler;citation_author=J. G. Coors;citation_title=Response to Selection and Genetic Drift in Three Populations Derived from the Golden Glow Maize Population;citation_pages=1527-1534;citation_volume=44;citation_year=2004;citation_issue=5;citation_doi=10.2135/cropsci2004.1527" />
<meta name="citation_reference" content="citation_title=Specialty Corns;citation_title=Silage corn;citation_author=J. G. Coors;citation_author=J. G. Lauer;citation_pages=347-392;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Coors;citation_author=K. Albrecht;citation_author=E. Bures;citation_title=Ear-Fill Effects on Yield and Quality of Silage Corn;citation_pages=243-247;citation_volume=37;citation_year=1997;citation_issue=1;citation_doi=10.2135/cropsci1997.0011183X003700010043x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Coque;citation_author=A. Gallais;citation_title=Genomic regions involved in response to grain yield selection at high and low nitrogen fertilization in maize.;citation_pages=1205-1220;citation_volume=112;citation_year=2006;citation_issue=7;citation_pmid=16552555;citation_doi=10.1007/s00122-006-0222-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=N. de Leon;citation_author=J. G. Coors;citation_title=Twenty-Four Cycles of Mass Selection for Prolificacy in the Golden Glow Maize Population;citation_pages=325-333;citation_volume=42;citation_year=2002;citation_issue=2;citation_doi=10.2135/cropsci2002.0325" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=J. Dudley;citation_author=R. Lambert;citation_title=100 generations of selection for oil and protein in corn.;citation_pages=79-110;citation_volume=24;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Falke;citation_author=C. Flachenecker;citation_author=A. Melchinger;citation_author=H. Piepho;citation_author=H. Maurer;citation_title=Temporal changes in allele frequencies in two European F(2) flint maize populations under modified recurrent full-sib selection.;citation_pages=765-776;citation_volume=114;citation_year=2007;citation_issue=5;citation_pmid=17322961;citation_doi=10.1007/s00122-006-0443-7" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=J. B. Fan;citation_author=A. Oliphant;citation_author=R. Shen;citation_author=B. G. Kermani;citation_author=F. Garcia;citation_title=Highly Parallel SNP Genotyping;citation_pages=69-78;citation_volume=68;citation_year=2003;citation_issue=0;citation_pmid=15338605;citation_doi=10.1101/sqb.2003.68.69" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=T. Frey;citation_author=J. Coors;citation_author=R. Shaver;citation_author=J. Lauer;citation_author=D. Eilert;citation_title=Selection for Silage Quality in the Wisconsin Quality Synthetic and Related Maize Populations;citation_pages=1200-1208;citation_volume=44;citation_year=2004;citation_issue=4;citation_doi=10.2135/cropsci2004.1200" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Berard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=X. Gao;citation_author=J. Starmer;citation_author=E. R. Martin;citation_title=A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms.;citation_pages=361-369;citation_volume=32;citation_year=2008;citation_issue=4;citation_pmid=18271029;citation_doi=10.1002/gepi.20310" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genomic impacts of drift and selection for hybrid performance in maize.;citation_author=J. P. Gerke;citation_author=J. W. Edwards;citation_author=K. E. Guill;citation_author=J. Ross-Ibarra;citation_author=M. D. McMullen" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=T. Gustafson;citation_author=J. Coors;citation_author=N. de Leon;citation_title=Evaluation of S2-topcross selection for maize (Zea mays L.) silage yield and quality in the Wisconsin Quality Synthetic population.;citation_pages=1795-1804;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.12.0725" />
<meta name="citation_reference" content="citation_journal_title=Maydica;citation_author=A. Hallauer;citation_title=Methods used in developing maize inbreds.;citation_pages=1-16;citation_volume=35;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Research;citation_author=C. N. Hansey;citation_author=A. J. Lorenz;citation_author=N. de Leon;citation_title=Cell wall composition and ruminant digestibility of various maize tissues across development.;citation_pages=295-304;citation_volume=3;citation_year=2010;citation_doi=10.1007/s12155-010-9100-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps: Molecular population genetics of adaptation from standing genetic variation.;citation_volume=169;citation_year=2005;citation_pmid=15716498;citation_doi=10.1534/genetics.104.036947" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. L. Hinze;citation_author=S. Kresovich;citation_author=J. D. Nason;citation_author=K. R. Lamkey;citation_title=Population Genetic Diversity in a Maize Reciprocal Recurrent Selection Program;citation_pages=2435-2442;citation_volume=45;citation_year=2005;citation_issue=6;citation_doi=10.2135/cropsci2004.0662" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. N. Hirsch;citation_author=S. A. Flint-Garcia;citation_author=T. M. Beissinger;citation_author=S. R. Eichten;citation_author=S. Deshpande;citation_title=Insights into the effects of long-term artificial selection on seed size in maize.;citation_volume=198;citation_year=2014;citation_pmid=25037958;citation_doi=10.1534/genetics.114.167155" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. B. Hufford;citation_author=X. Xu;citation_author=J. van Heerwaarden;citation_author=T. Pyhäjärvi;citation_author=J. Chia;citation_title=Comparative population genomics of maize domestication and improvement.;citation_pages=808-811;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660546;citation_doi=10.1038/ng.2309" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. Jung;citation_author=M. Casler;citation_title=Maize Stem Tissues: Impact of Development on Cell Wall Degradability;citation_pages=1801-1809;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2006.02-0086" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=V. Keeratinijakal;citation_author=K. R. Lamkey;citation_title=Genetic Effects Associated with Reciprocal Recurrent Selection in BSSS and BSCB1 Maize Populations;citation_pages=78-82;citation_volume=33;citation_year=1993;citation_issue=1;citation_doi=10.2135/cropsci1993.0011183X003300010013x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. E. Kemper;citation_author=S. J. Saxton;citation_author=S. Bolormaa;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Selection for complex traits leaves little or no classic signatures of selection.;citation_pages=246;citation_volume=15;citation_year=2014;citation_pmid=24678841;citation_doi=10.1186/1471-2164-15-246" />
<meta name="citation_reference" content="citation_title=The Genetics and Exploitation of Heterosis in Crops;citation_title=Population genetics of increased hybrid performance between two maize populations under reciprocal recurrent selection;citation_author=J. A. Labate;citation_author=K. R. Lamkey;citation_author=M. Lee;citation_author=W. L. Woodman;citation_pages=127-137;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=C. Lamkey;citation_author=A. Lorenz;citation_title=Relative effect of drift and selection in diverging populations within a reciprocal recurrent selection program.;citation_pages=576-585;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.07.0484" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Lorenz;citation_author=T. Gustafson;citation_author=J. Coors;citation_author=N. d. Leon;citation_title=Breeding maize for a bioeconomy: A literature survey examining harvest index and stover yield and their relationship to grain yield.;citation_pages=1-12;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.02.0086" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=V. Méchin;citation_author=O. Argillier;citation_author=Y. Hébert;citation_author=E. Guingo;citation_author=L. Moreau;citation_title=Genetic Analysis and QTL Mapping of Cell Wall Digestibility and Lignification in Silage Maize;citation_pages=690-697;citation_volume=41;citation_year=2001;citation_issue=3;citation_doi=10.2135/cropsci2001.413690x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Myles;citation_author=J. Peiffer;citation_author=P. J. Brown;citation_author=E. S. Ersoz;citation_author=Z. Zhang;citation_title=Association mapping: Critical considerations shift from genotyping to experimental design.;citation_volume=21;citation_year=2009;citation_pmid=19654263;citation_doi=10.1105/tpc.109.068437" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=A. Oliphant;citation_author=D. Barker;citation_author=J. Stuelpnagel;citation_author=M. Chee;citation_title=Rescue of completed but unread HIV-1 antibody ELISA microplates by freezing.;citation_pages=56-56, 60;citation_volume=32;citation_year=2002;citation_issue=1;citation_pmid=11808699" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Parts;citation_author=F. A. Cubillos;citation_author=J. Warringer;citation_author=K. Jain;citation_author=F. Salinas;citation_title=Revealing the genetic structure of a trait by sequencing a population under selection.;citation_volume=21;citation_year=2011;citation_pmid=21422276;citation_doi=10.1101/gr.116731.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=L. Moreau;citation_author=H. Monod;citation_author=E. Kuhn;citation_author=A. E. Melchinger;citation_title=Recovering power in association mapping panels with variable levels of linkage disequilibrium.;citation_volume=197;citation_year=2014;citation_pmid=24532779;citation_doi=10.1534/genetics.113.159731" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. C. Romay;citation_author=A. Butrón;citation_author=A. Ordás;citation_author=P. Revilla;citation_author=B. Ordás;citation_title=Effect of recurrent selection on the genetic structure of two broad-based Spanish maize populations.;citation_pages=1493-1502;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.10.0552" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Saghaimaroof;citation_author=K. Soliman;citation_author=R. Jorgensen;citation_author=R. Allard;citation_title=Ribosomal DNA spacer-length polymorphisms in barley: mendelian inheritance, chromosomal location, and population dynamics.;citation_pages=8014-8018;citation_volume=81;citation_year=1984;citation_issue=24;citation_pmid=6096873;citation_doi=10.1073/pnas.81.24.8014" />
<meta name="citation_reference" content="citation_journal_title=Anim. Feed Sci. Technol.;citation_author=E. C. Schwab;citation_author=R. D. Shaver;citation_author=J. G. Lauer;citation_author=J. G. Coors;citation_title=Estimating silage energy value and milk yield to rank corn hybrids.;citation_pages=1-18;citation_volume=109;citation_year=2003;citation_doi=10.1016/S0377-8401(03)00210-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. L. Turner;citation_author=A. D. Stewart;citation_author=A. T. Fields;citation_author=W. R. Rice;citation_author=A. M. Tarone;citation_title=Population-based resequencing of experimentally evolved populations reveals the genetic basis of body size variation in Drosophila melanogaster.;citation_pages=e1001336-e1001336;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437274;citation_doi=10.1371/journal.pgen.1001336" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. van Heerwaarden;citation_author=M. B. Hufford;citation_author=J. Ross-Ibarra;citation_title=Historical genomics of North American maize.;citation_volume=109;citation_year=2012;citation_pmid=22802642;citation_doi=10.1073/pnas.1209275109" />
<meta name="citation_reference" content="citation_title=A genetic architecture of seed composition in soybean is refined by genome-wide association scans across multiple populations G3;citation_author=J. N. Vaughn;citation_author=R. L. Nelson;citation_author=Q. Song;citation_author=P. B. Cregan;citation_author=Z. Li;citation_pages=2283-2294;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Genetic Data Analysis II;citation_author=B. S. Weir;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Wimmer;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=C. C. Schon;citation_title=Synbreed: A framework for the analysis of genomic prediction data using R.;citation_volume=28;citation_year=2012;citation_pmid=22689388;citation_doi=10.1093/bioinformatics/bts335" />
<meta name="citation_reference" content="citation_journal_title=Plant Genetic Resources;citation_author=R. J. Wisser;citation_author=P. J. Balint-Kurti;citation_author=J. B. Holland;citation_title=A novel genetic framework for studying response to artificial selection.;citation_pages=281-283;citation_volume=9;citation_year=2011;citation_doi=10.1017/S1479262111000359" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. J. Wisser;citation_author=S. C. Murray;citation_author=J. M. Kolkman;citation_author=H. Ceballos;citation_author=R. J. Nelson;citation_title=Selection mapping of loci for quantitative disease resistance in a diverse maize population.;citation_volume=180;citation_year=2008;citation_pmid=18723892;citation_doi=10.1534/genetics.108.090118" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. I. Wright;citation_author=I. Vroh Bi;citation_author=S. G. Schroeder;citation_author=M. Yamasaki;citation_author=J. F. Doebley;citation_title=The Effects of Artificial Selection on the Maize Genome;citation_pages=1310-1314;citation_volume=308;citation_year=2005;citation_issue=5726;citation_pmid=15919994;citation_doi=10.1126/science.1107891" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Maize silage is forage of high quality and yield, and represents the second most important use of maize in the United States. The Wisconsin Quality Synthetic (WQS) maize population has undergone five cycles of recurrent selection for silage yield and composition, resulting in a genetically improved population. The application of high-density molecular markers allows breeders and geneticists to identify important loci through association analysis and selection mapping, as well as to monitor changes in the distribution of genetic diversity across the genome. The objectives of this study were to identify loci controlling variation for maize silage traits through association analysis and the assessment of selection signatures and to describe changes in the genomic distribution of gene diversity through selection and genetic drift in the WQS recurrent selection program. We failed to find any significant marker-trait associations using the historical phenotypic data from WQS breeding trials combined with 17,719 high-quality, informative single nucleotide polymorphisms. Likewise, no strong genomic signatures were left by selection on silage yield and quality in the WQS despite genetic gain for these traits. These results could be due to the genetic complexity underlying these traits, or the role of selection on standing genetic variation. Variation in loss of diversity through drift was observed across the genome. Some large regions experienced much greater loss in diversity than what is expected, suggesting limited recombination combined with small populations in recurrent selection programs could easily lead to fixation of large swaths of the genome." />
<meta name="og-title" property="og:title" content="Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/541" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Maize silage is forage of high quality and yield, and represents the second most important use of maize in the United States. The Wisconsin Quality Synthetic (WQS) maize population has undergone five cycles of recurrent selection for silage yield and composition, resulting in a genetically improved population. The application of high-density molecular markers allows breeders and geneticists to identify important loci through association analysis and selection mapping, as well as to monitor changes in the distribution of genetic diversity across the genome. The objectives of this study were to identify loci controlling variation for maize silage traits through association analysis and the assessment of selection signatures and to describe changes in the genomic distribution of gene diversity through selection and genetic drift in the WQS recurrent selection program. We failed to find any significant marker-trait associations using the historical phenotypic data from WQS breeding trials combined with 17,719 high-quality, informative single nucleotide polymorphisms. Likewise, no strong genomic signatures were left by selection on silage yield and quality in the WQS despite genetic gain for these traits. These results could be due to the genetic complexity underlying these traits, or the role of selection on standing genetic variation. Variation in loss of diversity through drift was observed across the genome. Some large regions experienced much greater loss in diversity than what is expected, suggesting limited recombination combined with small populations in recurrent selection programs could easily lead to fixation of large swaths of the genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dsiwqKNaMxbfdNYs4JjuYF7YGSxE15_2BNDiOCb3B68","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459802","apath":"\/ggg\/5\/4\/541.atom","pisa":"ggg;5\/4\/541","ac":{"\/ggg\/5\/4\/541.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/541.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/541"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/541"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/541":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229sfu0l+JYGXOew7h+I49KaXThoIyM8maBD\\\\\\\/Mtst+6NrJyfZ7Fsrn+lFWmmvEEiuLEPXf6+s8dnzjmv5CnpEqp0nD+R987Hdg\\\\\\\/dJGec8dWFhTliIh00EnyYjgpjeqpdujfLOzANtNJ8yPDDFU5G4dIOPfJfCTdEe11csI7n\\\\\\\/zrjHzoq5KT5JJl6y10jcMOcHxAaquuclt1wALabIQlaLIzmyZpbqY80tPNSgck8UtTPwimgAJPRafkrcOkqcmPlSDom4HRlkEcbqzvjL6FCh2ZG8N0Zrku6JxgdVFxBqok7MejJ1qd\\\\\\\/m1AMLDyv7A5d2F4+iYDvCw0TOOH8IsjBaP4AjnWOLnJefQMl7i4jUksDXd9th0FezCUo\\\\\\\/9sWh9nrdzwy7wpu9CQhpLPmqkC\\\\\\\/jPnuULIHurhJSHMVywvI6rBCc+y4X6XtPx4RckAPhCofh+zbMpAUw8CXeGHS4bTmzez40w8DL2VEDrLDNWzLAcdayNNE9BN\\\\\\\/jK+N0pWnAkswVecUabuG1i0ve2DPRl4ZvVqWGnMB8dpeE\\\\\\\/pexN4R5UB1qewx5F4hkE5FXuTBPHag+cDta\\\\\\\/75SKcJDToKxbS5+sQETEhFP3EpzjGKLRelT1r8ojQQdicBhLEgyuCkyLqBQpCcbfUIGmaqDNdyY4HELbZ21vPf0p24xtdr2UR+8Rn1HCf6FBSjgjh6Ue8TaY9P1NRo6TPXHo30cjiujJxijfGipIWVhufkQkjUZET7Kd9U76K0uRH29fKHLSvxvkr\\\\\\\/EEuhQ8Gta4QmZTKg3ET6u7R5eEQao0DurZhORXuX68rPtkJjvnzWB32RX6xUBSihQQGYW4zjE3uDnhieS826mZURHf8kZc9fOwKbeMDxj50zJSrj9GSdAo1lckavmETNE1278lj8A9iZprJvfu10f1IZDwjWIbU7+IQq5FGzc59HoAd7wkzYUdlSz37mrSRnjTbYDta9kJFbBjtA3yPYKipFQmwbeH4LGGbhm8bIP56DhyhHCMhpk4hy8H3Nb2lFE4GXdhPVxFSYZ\\\\\\\/RqxAveHf23wV8qD4BetlwzysFcjk3jRHW1qw8Ab3C0wGpsNiUV0PViKDEyp1rw+lFynUWjfVNVMZodZvAzVQ2Y+rcIPGOepjuMwdUFo9A7Wd7scnlhUEsXKBkZD8MD2mVAkvTPsRcFWtY\\\\\\\/uVJ2Ia0lIrHrivFUQvTW4gyCjV+AqGGcqRqz+3HoqwIenz1D8RR+c8VL2VcdQOmjLOgk3eBYJWUKaJnTLcE2xVm2xXggn6cfyDC124\\\u0022,\\\u0022iv\\\u0022:\\\u0022lsnh1tUPHO39k78i9qi5Tg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229afb1b0cc396d20e48cb33b287f89b53e23aa0c7e463a775fc6ec8149ac6694b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XIVa1F0iHb6nzSfQvXPZii8G+9VAyCsVh+FvoWpGIpDtliUQ07Vr7T\\\\\\\/Y4Gqbu0IU+YeYlmun8SIbjdaQtWHKdD9Gs5pMABPjEpiXPNK6aaSrXupCnUOFwRUazGhuMFclph31nabipukqkXLn0rh6teaT+QDAgKGZ2d\\\\\\\/J5fGqqaHj27jyr+dBQwShMSsxv4m5F\\\\\\\/X1IOXJybMonv3fLsXSZJSyt8RswHOBzwRVd3sdG+\\\\\\\/16qvKm3b9P38qTjLZXxaJA+ggpn07Hfw56tQzoX3NCazg4rqC5FqqsFgQHIHcI0AJ6BpHllcWK57xHP3I3jGuBp9YncfyspgYLqwKPG3uS0380LySYCOUhowOUlooOJ+3TNPRnghWrQzOTKkCtO\\\\\\\/UZiX+dDSsxcBiH8SWpPswWClTwfv3PmbQaDEzuRWHZuG5EN1b+2lOqbpX8IwEnUC2z3gOwpb2VEsMv6rLMXBlobR+969evGbZM5y6fZsQmcA6XEFE86RtY4+t9VQ1E\\\\\\\/ByzTZrBFXRsNqqweZoUpHwqZQ3mnEpKSuj187W3SQypC7Ag48Mz5IosBs2EX37eWulBYFGNUNMdKGFb3wJ94p2x5DxIktxg5BmVaBVCgNRm93h8rb67fQAVK8YGouqaZbvyDAjTmuNHzT6aljqQaTjOno7W004Pn1b5i6q3EY+IX\\\\\\\/LyReE32g0hRh78LUOd8vmNOoQ6v5wdR4sQ83uyPLj4F5ZJjITpi+V\\\\\\\/lb6xz9CUkXpeSv6VibnpkyUDIm2ACHrSpiY+sbBcVnFrGKqF7Shm4f\\\\\\\/eXK1bz2GBg4t6F5qo1d\\\\\\\/cHGKBcuz2g3OyK9Jbm7GA86J+CpwUXXDI+H+4noArzoKv9UEwl5SRuxLCnfk9RAG4ENWb6hezM0JAgtSJCdycEKP8wdWtzRVTSZd07djshk+Bjd+PSaU+X6pc1j2MkV5ezlZ0+SCOpP2Le6Nf8a0DucMuSicg+CwXLX\\\\\\\/fINvUcbHkhS+xcHt351rUYaRg9hQD3IuMmHiZGHXIUPtl7o3ET02axuoQ5rBBk5SYLRV42Rci3qw06Z02ptwTeX5BmYHEQmQt5GlzylFbbhws9a1SdDa3c6xY5fQryY9KA1fg9G3j7iyZQwP2ga0HAOtNMMOBHsbirXlHJ7biqiH4gcVFk6dLXTTQk0iTtofbAuoitErK9vuKNoXmhRr852lzrTGFPb60U3NX6z9WJSTMsCrujC4Kfchph5x9\\\\\\\/OInfbgrst2pyjkbxK2cXfebWE9fI2dMmw7rZEYDIybloPYe29mdC3FH6F0Rfy+1TH3ZNLVaW6h36r+gbvrQx5mAKMI1c41zIQCs9p3OYDjAURCqnfHiTfBek6UUtpL34RPhfKAGpNB9KcToRG6hoMIuEm85zgTJx35NIu1vmA6Z8yZf1+cFp+4H+DPydCBqshkJzxaZHnMaYNxJ8wWdj60k2ckLrrpBrBf6O6UHPzckaKhvI7TKTZ\\\\\\\/bT\\\\\\\/DWK9ZD1I0QfAKTdeIziBGopApxC6FttWA5d+zZhE\\\\\\\/DMgZHrnaUQv4\\\\\\\/wmkv5GDJj+PEr39HMJzzT4CBLgUOoo2ybXQYP3tVeAx9AE3bgSv90MD2vUxUnp7ORaEik6ODhA71ohB3EW4gevrVyCPug6uy07Vx7P3DO7TWPcK1ZRRchhl3CzxQqSrTiJcoVDYjuXWLruKhMfdp+j3Iw\\\\\\\/yH7MhO9v+n3MLFjVZ+TuszuFum6Etxr8ozbXTDFiPMVcBC5ATk07gBhXmDfxzFUhl2aOIpD\\\\\\\/pc056dTyYt9wJkkCgZgn2qFxzuJYnraGcZQx4WGuIqFAVkyJmXuxW+S+3MINYUOSAER4g2lQ1LPFTXL8sb4x1lQ70fGq6FRa9R7Gob8R3qD4fvfT7IcL\\\\\\\/qGbYp\\\\\\\/6Rj7SbsUvIX9Wbp4CVagk3xxJEPQcT6zyV92JbGPUiik9fgTKXBaGYeSa9F2lS9bpPT1rAPCeNe1CrnDTfrIq0\\\\\\\/ubuyVrayC2tGbg4em02WyD1bcr+tX3mtlmCUIHXsGBIGEihSmdzlCwUJNVBaL8lR4VZHxCtRL28Afmqs8Re92qBLb+e9ajwghUi42h0AJx+T9BIXNvhK4ynRxmwNsycojGmc8F8J\\\\\\\/CrzUqMJFsJf95L9Ky06LrSbZAFjpzSP7dqAnWCyFyP3gVmkC3j6ym2zumlT9+0P1H9+MAR6QZcxQ4WYQ+\\\\\\\/2KE76NswQ4\\\\\\\/jpkn2h8mC6fk2HNc5THLWAbnRlS3WssF\\\\\\\/NVECWxcCmZmfb6s6sSnHPwlxjY9V3kPSPNuTLnJpmXZ2I+A+LMIlARR6qEcuZmrXfLh1+K2C63cOQGEM7QbvoxAh595aOq+b5VHmXzIYYIDCtIN+uhX8G5oV989AODPicwSKP+f0oO5NA06cvHUCwqRGuZ3Rrdyleb4EM+t4WH7kBU0C24UmxHfdYYYqfcWgn0+IRcivB3hksATYet8hI8EneD79pK8gA94zJHgGvkACI2hHUA8OylQ20SJbY9LcRQssEwWXTSAJvcjtbwqQ73frRiiNC548eKl1EIVZ6pxqXYdCv0gA1eHYJ9QkL2SbN8jG971St7wpTgbhE676OUtRHTWIlTasPiDZ1xKQkOzkzM3\\\\\\\/IFNHowDMBQRpH6KQlI22+vNHjV0unhCWW0XZuL0SmuPs2EXViTzmeaDgprHIWcKq2Mix6ZTqBkVrWn1dzbNSijd1tBFvGoOXGT3S2qRk5GpYDZY+oW2yTcbUzA7D4\\\\\\\/nJJDrEZ5e7+Y+HGzqRN+6Qn\\\\\\\/\\\\\\\/SaQn8PYjBt8tUYoklt8ybw3xvcjPY3uT+Oq5s4+uIJwHfDHxl9FT7niZqi\\\\\\\/FhlaPBoyPbWG9rzccy55EDepPBeTdrcEJ8acKHRCob1d1FR059yuxCIF+OgX\\\\\\\/UP6htR3s7kv5j0+RjrO8MJfxvNN+bihuT9FuQGrZFjJK3MtaDBQdVqhs0osfVq9VETbHNmiWtn4e40ljsXodkT\\\\\\\/z0\\\\\\\/tRuXhBiIFDyqNElSfhQDRUO+ayPDWYSX9dkPFrERgFoKiV7V\\\\\\\/Pl3EjgXiN6qT7AxcHcVHwNXMOsr820IbK6a3hNP5A+fKiUmyxFvSmM6Zi66AcMg70iSA\\\\\\\/3VTK8Oi+qBm7IDrYi7JoZEowScVZ4027MXWDXPPOaEW2FyVftn26F0C8nJzry8LIrym9FRwsfJuukTr6zyYnxG2rjp+9SmUfqrczhVPmMz6rs3O87WYdgFzIIuKX6Xa4mnjJ\\\\\\\/67vraLO3XCgyWHtoQHvUcZffG5GqQJ5nN+mk4iFfEBQX\\\\\\\/Ogv93xdEhxV01ZFQzJXZWVY1AeHZWLTH40L5zjkDwzaI+\\\\\\\/yPZhX2ZQAUKZm9embD97BdYLgP0DR8C\\\\\\\/gM+Z+OLlA4m440MXIXN5EpimgVljsEH0BIaWJsc4A7UiIWTMIj18xUswXneCKLdLdPfmleMVgULNMl6\\\\\\\/y6xAYB6ZlqnRFiGQA8SMcJJsnNELOqbXtWHCmlxXD37L+nfG52PjpNckZnUV3NiAJxeFBf64HPbuEWrvaRn4ggnpWWz8AqsTdle9OVNxFbZrplSt8uV6zD7c+U7tj45GPHccTmdhhWgWJx6q1kxZDznWHyCsCRymJwmjfLnRHAI4e1c8ZXK0LTRRztm4RljsMnJtYNMxV45VUvLhC2DFU51g9xByIeGwgCl5FCDwMayCDtlhmg5uTKTnj1XRXrcXEYek9\\\\\\\/mEAe5DoBJaz01LZPFLD1BKUZIrEoMgr9fGAr1SnXZfuC172mxpqoBptJ\\\\\\\/Wc2OG\\\\\\\/C0h3mWPaWTod6QSOc+0xryzGzV0PQ3dThzXWkkpl2LJpMJpYSmv6yUqXC12RoIbeMqz9JLLC9ldaZbNWGzCuTlu6fJ6t1jrmaObPzZvj2SvUd9P091UVVp8C+AFF0O1uIEl1yNxwd3qlfkw4l5m0o2YOZEpeD9rnzWFH8C5J4T8aUukQXKda7A52ol5WD+FARaDDXt+kimPpKj5UJ+CX6cgxvcniEh3yO1kMlEdA8UsFg2V4+RPpm\\\\\\\/fgUH0LaiEGrxp1QCWKGLjRvRFEqANa0hq4KByn1LJeMS4PvWlMhlHEgHm1lQwO8CVSD0a5UpjK7lGBgN1N8yHRMebePOiweYwQtz4bG4p2t7Ut\\\\\\\/6CwqvF1n3pHNYVoHGFWY7dY6HOm\\\\\\\/W6WHXTfoV\\\\\\\/yspPEpY3dQXIIFeX+MsGHowBISKPgKTIxVjPDVszoQsHoyTuT46X6wpp0UFYr56gYdSplc5mJHvo8jnzDBhe2u1W3HYO2wE6lSQi9IJCht\\\\\\\/KrPKMSMxIBxctgqPk+WuaXyhVXKoO4WP4BdBVSWA\\\\\\\/iS+KKj31FyKc4b2VAjM12PJlICUflcu2znAvzywk99cXonMEt1tSLC\\\\\\\/eGSw\\\\\\\/OBLyV4TIEjJXKh5KqAnK\\\\\\\/ziAZZi9WDOa4d9L0pNp7eNEG4MzNN02JlDyPGIJOFiPkPr7zU2z2iRCAFqob7yN5Xz2he53wbPNaa1SADyQ5\\\\\\\/Uc6KBR8XonJp20SdXCyET5n9vYwNO39O7EDGNSn0fQ4sJET5fJZv0CQVOBcH5UQZb1bSKzE7Ogbvb2VZJEg067suZddo6Q3CT\\\\\\\/OYyU071PUyV1cBWferczbi2ZV+FWXAHVvY\\\\\\\/4sI4cLCtXQ0vXXjGVwsP25fGy5xlanlM7+AV0Mb75yTRTZAhaA+p8ZNHrtp8bzhZNJbSw5YACIdzZsZiT+XMgPhCMSoow46asy3ycxecpYLMvuZ66LSAAE8pOVW81VFVQvJKdZT8LPIuplOTJifdauJnl+4Qh7p3faYOy3nyrzxQOyDP4cnG6GBvfVXHuHpaV+VfRuo5nrau1mBtIz6XHvXxP+Lzp8IAHRSrn7vB3\\\\\\\/2EJH\\\\\\\/YDFJATWWx6BwWSKmYbRWt8EZ7Y92Bx41eS80EjAItNuJclHVk1aOf1UEcRYR4C1SsT41t9f0Xuwx46YKt10xGzNHuz0O\\\\\\\/+asUwas\\\\\\\/g8dUzs60Jd4rPhA+iOOhaYijbhcKX1\\\\\\\/mK0cHwmFD9b433vAkDACwWR0OyvJAq5H+gCQOljVWa4mjWLBduilqJABvx+Ewfq8KElqRAW6gZsITuWNFSbyQrV6xAl2HaXaxyUpIAOM\\\\\\\/8anm0YtEf+v5ljJLdknTINO84NlNTZiaxqaHzbBgnI37y6rBqnEs2Yyc9F4tWIurIDY6yA9IQHK7F3hJVmxr7R4f97mCk8XDxd+czp0QZpfkHWmnhz3ChM4zgnsewAb5OPVBFXSDgunFPzLDi\\\\\\\/7NSOloGjt2R9vym\\\\\\\/Zr1tbbL6usA9Wwm6\\\\\\\/d\\\\\\\/E7KPgF0ao7X3TRgDzMNWNHvAoWw0kQHuP69fgogo3LULsSGU0uljXzojpSiTNA70PFSbsreU7B1e0KufUOUMi6mLMx\\\\\\\/JP2VU5180PVHy3R\\\\\\\/LwIGMCL3\\\\\\\/SKP2BrCNi3c6TSBn5PyDkK\\\\\\\/XlXPNtaaH2LfZPzGqw60ABin7XWX1lalKjzByFJ+UyfDyQsih3Ue3O281pgJFXN15V4WVnyiV+CJ0QDjvM9OBjSyzJofWDhxhp+uyZ1g63f0RV2Sb0xG5UGMY9mxmEW\\\\\\\/GAJTl6BXIMV8FPq3zYMoqgD0qE1dJmeeSRiaTiGOc5\\\\\\\/2wkUF6+NyFUmG7jgu0opzbm5n4aGX9otHPuUjsASPHJ\\\\\\\/IJTPdXWWrHrB\\\\\\\/SNENMYVLQotFX5pjP9MaUmqRlUVVyMT\\\\\\\/8LOH0GgYMxlCTBKpIQaRx3i+L4bzfNm\\\\\\\/HPWRuw7bknvuHh6tXv7YK5hlG4IGG+uz8qxh0FRDK3zUop4ZF9JUqfTXZoVytgsXjJsyOnb6ThBrdXQXvP30CWv0hVYQAPylZRXmWPcAN1zbF8VYUc7iX4MW4e6ADFaDjUPyrEM0eQaNEbOSUAIINvVS0VOLHM7T57BLIjIElng6jehimVqeZY2vEG9umH\\\\\\\/pvOswbVAXcpPVXMr5SnLuw0dV5lA7sAgLi7yVMA9ICctH0GEqMdFs1TheqW9VKeVzd6hPxFXvOIzPfxBzOwcNGTjP0ouNHspl8I6B0ePwgn37p7ohGyhLJWXYwlVV8m0Gc4QHo2ngmlVKJTg5MtPwx+YTe\\\\\\\/yp5NUNSTwtv0pMW40zhgaEobSEIWTU5rUlxXegFqqMTqqu9k6lLfSzze8m54VsIGcKZOfPvtWaGiTJqgCEbEFz8nlm7xAGLXnSH6tRgYtCbms5i8I\\\\\\\/XbcKFiMg4tOcfGvRCLMSSRrLN3k0bln39ldknTPK6T+KoXS2\\\\\\\/AzdZpwvSDGup6FxMna+JZPqVzE4I1+CxM8eToebxC5zLmFDcpuNcjZq7kgdN1Tv9QNEgCZ6Zd9H1EYhX5Fkf6W0hzBEo4rPGNSPFRQYJBkkNLk7rfGCLAvwCv41H4LMQg1Fd\\\\\\\/v9OC5LV4dAr1Cy4sRkorNqxicE4bQW1hoPdYHexZ4ITKz3pRhhAdLOeEFyRvbppJlm6Ddx7bcewZnL3eoVVzEn7QnsyGM5oP2Ilg7vATL5EVoh5y11rtRfnZ2UkqBi5+98GJAFN\\\\\\\/JkjtnA6Pla7I\\\\\\\/uFyEl2Xsh3CBQywKQ9Y4NQGpqQDDR009EygGCpxMRXl6fTmPp5P6YLFJwhDh\\\\\\\/hjK\\\\\\\/gIqzqhJ8GjJfzMDAdqsalOOaFjqr5PvPC8VBkrHB9fCgextKAYt72ROt0333idR5cwESEpH4e5hFUlDAg+f1SZw+9L9kb2aqqiyAI3LNwUgIyfr9PU1\\\\\\\/jW2iIt9jnQkIzd1c6RVnI68XywP43mAnZ3Nz0DG8dCa5uoGuYFYnMe3fTymK2AIuay4XKRf9ObwsGeuk\\\\\\\/hSiNMabERx9DDKIjh2tzyFehKcYX7BEPgOiaVRIBAzpR3sEntmHbf4VTpTDTCiteU8mYszh0wDE6CxD0nYAJ\\\\\\\/bSuIFgWbH0L1p6zMSDO55im+0l5bQn\\\\\\\/RuhAaZrS0+rkzmJmrYFWqs47hC3O32Sf9W1pzUxQlz+ZrMMUiz+DB64QMin6EjaAbN6wkkbJj822w2gDlC\\\\\\\/vefx39ABLheVo7fgXnyDZbboJFgiY7ljP59ZOgLJKbCHiiFhyQsZsOYO7pF8eHVKTM2o8cgekn2EO1G6iNNmLcpHzjZ5JWcEvIMo48\\\\\\\/ufkGNE3P122mJ0OuTiXuQY7rHjxs2BEOI9zGYhbNj9\\\\\\\/4U6k8xJUWYigGgFVtwXW1pCzyhVLTN2nGnV+rSTH+1ZFlWhRDzsbV6y4pghgCD+g9+zuKiN2W89i4xvXTSQFc7AHf2e7Gp\\\\\\\/Rl2pfzAkpGhKc\\\\\\\/ri+UfcIwXlxtnPGpMz3bWxHTGIe1nwHfI6m2ibPP8Sl4H85NeP7SMa1pk3UIGWulNL6qIm5zChlPQBIthglPK9HsLkfhhTgOjTCNKe7j7VbJclbR24LQl6Tg4qSaJND4vA3PpfhJ08heCWJ1ntKWfPiK\\\\\\\/FS8Ds\\\\\\\/T0hDdLBqbMAdCo+G9xvuqm6gIjKghTKSeyIyw8280roAwaHCplaydKPHjjxfHO1hdHCsEdB4RluVywd+OVHTKEgsypvgjZ8ldFbl0s\\\\\\\/UPZgBw8raL8D6QOAX0VBQFoD9sade5PDog1nbvHmV3pHwLieE55yx2hXiWSkvo78BOdheTvcTVVOOb2aAy\\\\\\\/8+6xeDT0Hcd5wgYrFoHJJ4h21OM3i7CV6QpfgEteLl1zrwaL7FmXZHiZtMqUhOX7VbrXW2fUxvD7ID9CSGL0GEbrpAOzUASaIF4faSLul92XJ0DKJfGTaGDRcFDMo49FCUemob2l0kK7v\\\\\\\/R06a6I3KPfXHIPacKNBggOHiiO1pjUin9xISMn44cwHZqRgyjNaIDNbJhLExdp\\\\\\\/VoqEO1alVTZTR+QD1F4v1FgcOhWJVQgjvcC5Ito0PNSZmpinYzVAjiBsmYhcPupQdKwYU\\\\\\\/I7RF\\\\\\\/nUfbtcP5yA9kauMpCltizLNiJyNe+kN2oMZhpWdPgWHoNfyvjkEvCpaaHIt4Jn32O+4DGTMSVuFq6zfT1ftzugyEb0tIZXK8GwmrXrZs19Y6xWjtaercUdZ3oKHQeJsygGW8W\\\\\\\/NgP7Kl+gZneV\\\\\\\/en+Tg9+rlfP+nmz\\\\\\\/FwR56pyI1F618DujPPIUF7\\\\\\\/gROcCZw6Q5vwkEGQPFDGxoEEp1+cuZNzlt9jWWe8vPzJVVdNI7cl0xyeXJfNDlZV3nU\\\\\\\/c9MU1M00l2zItn+o3cuZwS5\\\\\\\/aDW0NQPH4w98kKVbE+mzQy6GwuHqWuUu6r\\\\\\\/B\\\\\\\/kbRIQ3zUSAVZLK3O+56hXZwHSIOaubZIVpnlwc9gzJbFLQYbW\\\\\\\/VlDti\\\\\\\/aJsQk2oJV\\\\\\\/s81xjAg4tOCju6131sa+zZ7yIV22AqTHMjYqMNtWD8dWezFd7mjqaUhgkeKWHt4LRUyFgtdIUxf3PagjEHgkl4hE9I5mBOhrDZtrvBjnA2GEK4kv8tm3WgclMIl9KJ9EbgDzE+p9ssJwYldh4YR9HS1S2AtiJBn\\\\\\\/2Pe6C+GwFW3Ab8CmEqND+AGuiDJ7YbWacgLgQPfmQqX17N6y2IDMRGy8wcHVopmQp2EqL\\\\\\\/FgZgDRdO1ubqPu7kOddei\\\\\\\/7IcyV64oVeK1wzhIdtb+o\\\\\\\/sm9ERF5WXqNuVbhZTdXU+vwN4a3Hmx7JixjyFCP\\\\\\\/eXQpRrc9JG6dW+selVp08ic0CSc2e5gOjzUwHwVtIHiCP14GsJydcUXQR7XI1dTvItAF6EfMLKriImL0IghVATHPbZwCSFJ+Z2LXLsDn6m747gcoUCOZSFC8v44YpBskBnyibncpfgB6JSFPely+vtnMoVhJSzATHi6OCFHvtE9OPXUBcgucPAZ3MCShtKrqOvb4BJtW4o147DQ0ah3MS\\\\\\\/G4zogssd7sP61p6O0AHn1PkIkcaWl8hgqLtsAjiFv\\\\\\\/tdFE5WjcQJwyGdJM\\\\\\\/5oRVfuJ7HYqAOgOnYmiPxIJvTa5DeYef4kKkoNVj8B0QiJpUKxPX0Fl3x8qLWdwdWkO5L2+c3CVDZkGtxEYvMuZsRuKEIXd4z3FLjhqaIHHIBDQ6cAeAn7IxqOEEW89c4DDSTGKAyDRFcE05baEYBmaeM+v+fPfNLRz2pDQYLhgfoAco9DwoYQ0cZSmBKcAz+mRzRzT\\\\\\\/Y3h8oqkQVeN1NXLTvC3LESNFBXovh9sJ4W4x6MyMwBot9Gw4RHdAayS5Isx0F\\\\\\\/ddGr1PRN03EDvgBFTz+I1yPuNtFHL1T+iI2bxICe0jO2fgHqqWQUkkx0cRNazhlbK+O6p2VtDd\\\\\\\/hceuvRXSt+0bT45EiIKQuDFRX+hjQGnjmTtI8jbcL\\\\\\\/EMupuLdunHZPlQDYR4Jmv9Fx2yHVY0PKEd1PyCmfR7A9kmjWeWQHlLiobmPTHfA8Q3OCNr5Jy0rWUlysExcGyTwH2tQESuIS5sAZ+9U\\\\\\\/XzWyl1a5oQv7mOAS9pbU1M9JZC4eiIVWvd3Tw\\\\\\\/4zS2QObQ8PzIKLig2MdO\\\\\\\/GnnO0BKDvTAVjWNhKHAq1jSDqZVwxScqxedAZAkGgoA7zY7XFDs7d0+WAbkhjR6VgIDWveLkPImHjcP2Cyu2GA0qbjfBTEHKDKRS+8kMA+WJqR6WhUTwzpFmvV0Tqme8cUnIc0tfjoDWLzMHYphDouStqT4CIUOZJ8z\\\\\\\/mHXffNmqljtZTctRIEAa1b+3UNGUedyO5RRftDDCQ\\\\\\\/qaLvI31jMkv+MFeJmnHUvXco8PYHqwZus+GnbJYWCxa9QTlSYrC9CuC099otDwARAh92o8AXuU3HYbn4Uvlw8IlApHk\\\\\\\/qVI3PhlKbo+rbgKVKCZuJaVlv\\\\\\\/fdy2Ph2IL1ws2nYv9g17Q+0Nah0+wZxuZtoXA1J15dl8gg+n2YSHHrLvZeWlNxEfhPV6\\\\\\\/5h4QxgPN9qXbxQHp09\\\\\\\/aHS0PPwP9bAD1EGWsiIqpbTXQ9yJaTiZbe748Cdwslwg5wqva11YSZFoFcGV7XY\\\\\\\/uaeUMa25ESdRhSguXkxLm52H\\\\\\\/nBLxtETtf+xfgMxlvGtYcfVCtbqJeoJNX4GKxd\\\\\\\/b7A\\\\\\\/q0LrNpL28Z+DBrCCG6Cg11jLezF5xipNVdVSXTTKjkicBMgIebskrcrwLR9BFx1lFYO3M6V10qRga7SiTgE4xLe0ey0h9piUUxpaB2ONt0lxcXXZ1yIZGAGVvoN+XLUr\\\\\\\/gSp\\\\\\\/jzLdbX44Hb5twCQr+d4SbnMsD2bj8Bh3WW+Uv+SmoDXVqly1Y77ZKyX3ujIQSCk8fvtMjcz\\\\\\\/BJTakNmTrrmzGGI5J2zFL9LP0oPjPpUz+GPnNFdNzrqdqT4j8RrY\\\\\\\/YquLD1YgmE5QYHazR7olYetegCPNKNP6W+N\\\\\\\/hhcQ1vUbf+pDtfMDGmC2MNYuMaZNteC8z7ukr0\\\\\\\/fG8fC8qYFhHgEUyVzhzvMZk07R9gku3GBjfYFlPdfO4+EcjLFxXpdgqw059DpNj\\\\\\\/NPqZO6vtaqPoiqzdtbD5yIzOGZYYU8eGqHEZkrmu4oIOmIf7caDV5kl2BRr0uuY4oVWPVOMJI61YYHuY2QgpIzbsIL134nxVFVx1EDAOeRQEUlhdix20oCLuIRH5Mm0UCG126EbpxO2rb+j40Tpbv6+5mCMOWmFys1Bax3lK84IwulfOlillER2E+fMMRX3EWjeMnAgPL4o7SvKXhTBayWbyCGi925FLxtc2CZ4dRYgXwC17EymWouB4gfsC\\\\\\\/7a3gR9o+7ChbVyy8EI13NTVxhN+hEBcV7eespKgql0Pplqs8Oll\\\\\\\/AlZWAhGImHmb5SP1lkis7VoXgNj95Q3RijF\\\\\\\/eiu\\\\\\\/dEtKMucvTrqHK+QAeJ3j86r6vxXuq0pRVIL0z8yguyYWASxhfcfLVG7Ogqy9PGqD+pMFPND+gGSKjeBNr7oRWjR5ZutXc0hkMsuuU\\\\\\\/qegPl46xQ2NCG6Kj6BhpN\\\\\\\/S19vkvioJk7wwP\\\\\\\/3yr+H0rGNia5yK5hhaVajkr8UInkGggkdOYJW4xBgZxvS5NxBVFh4fXEiKu6YvYWTSC5biXKXmLAAREce+Ap7IqQb0JrazDfCJxI4fslevyHn7TqrlzusuAVROuRI9MJw3psnoQ+Z1C0FEPefUPxjKwq1h2DgkhnskxtOBCNB7xDOB7ne42MxURvJPXNVN6aUb3\\\\\\\/v6CUpuPDzNrvY1RA87DrjchFJ3sNVPHUyl2fl+fn7dM8GdYrds7Rm1SO1uICDBbN7cXGUAXaNkIBWuaAKCdOgYXfByiuXERoYkwP\\\\\\\/JRdK1AS57lEpzGZp7oaCU3ss87keahiHfjkMwtDZquEgYXKKMmvUKPF1DaFuxn1lfOray9zE7sx8l9MzhM6T0ddjdos+zzXf9snNBdFyghd64dhM4fbtCsRkg3AgtUzkUaYYVhutdPGBbH20JN9t6EL2FZpT2ehgT4OIU7lC2CcUDIExs1u7cyClfYo23wtsFTR\\\\\\\/WBkwfPEVOtZpnZsgwXXo4bKZTkn2QihEFmTtSu7TWoICCXSdaUwq3OtszVzY4\\\\\\\/3SQ\\\\\\\/zxLJd1cW91urZlqKF0rXy85jbnIAorXFc45OMXFWPBwPBFN3gEYE0HwHIPIIyOvxYbzjhM4liCeP6obFo55BnT9NqOKAE6mCc2Vt8zfcPM7tnMNzhpPvC+RyDx3IzRBLgqKPQsx2Adc6QaX6o1uShc9mqT329GcRv7QvhzMDmLd0DxDuET69zrsNhqAx\\\\\\\/u9kfJ9lOQZ6n6EYZf6u7XlXWAVxxCzM8Z5ey59o6owKwYs0YLez6mg1SpAgkeeQOq4Vt8fqlBiflcG+ecmg+wmpmqgj4FJf3slltl5jp+vrPnhZq45B3qM6rFRf+PRJJiP143ovvlBSJm8Gzuqt6g\\\\\\\/OYi\\\\\\\/bVeQxIco\\\\\\\/WQ56i96\\\\\\\/Au0hiM0gJX4kAlJ+pWYD6oWaDZbBhpfBuJiVCuHcM3nhHTlruhoX05i9ScF1GNrj2d26qRFU5XPXy62CMz6R76LdcrKDjA2DxWSPUHw3\\\\\\\/5ahibdg+X2sgTdjaM\\\\\\\/KfgNMmlOH71xE60hhnRBVz30Ga6AAkB\\\\\\\/DvGCdwyTXKktGnWLD2Smg97wPrRQ2FQbXVZ7xUO3FgaXSP4ZqLnwU3nsiMrG90Le28hc4U4EqFq0HKPpXVUy5L2vdhKsLT1Z0\\\\\\\/jhPkd1hQmQ5JNQp37iy1AcGFDpXpMN+\\\\\\\/e25fR5W135bBe\\\\\\\/lMiGRmWEu+9A7vJXnttssCLJS3pm\\\\\\\/OUcfB30D9dVTx\\\\\\\/53sOpVxFU32HytrmJj3\\\\\\\/i3r9sBXODCmjAQZMg5khzj8PvLyk4UEEMxAhIHbNOEiYr3TpaMMCpBhGfhubUyy3s+gPUdoLY33eoB2fFdqE7OHMETi0SG2Un5NyJ71tTli7hSrkuRRIFfCa25rWix+b9ujkTrii2oREduqhW7WzVv6fLHoxz4xVPBhCK1DE25MA\\\\\\\/cdyINmECAXqVKF0sqkiqbJfzvsa9lxHd9lpysDFkfPjMW66Ve1a4JelRcgaFkzJHnvk4lTSoj9lATKqjADscink6Kg\\\\\\\/XlMJ1rsZB0Ub1AjNfpGRzPyu84Zkc9+bMQ2mkqjS2fCspZOYGczDg87UQRxsjERh3FM7lYq94\\\\\\\/dVFVoPzG5Tf9rpAAH4I6U20icPf1dd0LvWMh6+sbyh4GQ3ugK6xpuq\\\\\\\/j32RPcKk9QlOPkmcmkcFHrdv0Lo+zNiMDH4kRiGNFXjmqoVAM+re2ANOXDiwhvfHlz3jzy7Md0XF\\\\\\\/HN5Xe8W7NbM1sI\\\\\\\/vaaUb2NGvlr6Pwm3WhfjJxt8ei90IOghY7GgOqsQg8B9UFJJXrek07eRjA4bHBeg9KvCfmtMRoZnRBBOvhizWaDnwMSOT9ASi3lQO4REDeCxnd13tsIIMFiU+Fwe8xamZel67CVepc232uik+lDSsx77YAQerYblXBub2UX5aYnAYfsUYSJScopijzxT70GCb7V7aHbCZON+OmPTcecvPvGxuWmqxWM4E1tka\\\\\\\/l7P45KbGu8NXkyZ6E1MTtmGIGYzkUf\\\\\\\/rF6TbojPrhR4Fr9uDf8l8N\\\\\\\/6zcWoUrYsv60ffhbl3lMtnmyg4uT26XIp4pdi8wPAbQBNPY50wcdf+NRMrED3s4X+9knW+QjfeegKyHR4IpmbhED8fGKCoC4BItOnmxF4Iq+aRIQWywHj82QVy9ntm1r+z8Xc6E4PR0ojqaA+VzOtkSN2FjbGxzfZEfodLLySyoHh7AVBxSKjQEdkcmoG7CewgnB9wwzY8xC6fvK2luURmj7OH1DgFlRHY55CZyCi\\\\\\\/5tkIARN1llUIzFnKTta5MzOQt+BJckDXRXwiK6Jvfu6R+SULe90K+Cl0bXyeXMZb52ArnMiPmOzW\\\\\\\/uiFVQT5hYg0qfjtH6W67bPNTsZPNHSE1cegfkEO6WY9QariNeQONrOle1lLH6LYa+II18EbJeBzRJikjN83Ad7xGCW2F5uW\\\\\\\/Src5qcjPrIVUKpJ1JdZpzq\\\\\\\/myKHLnu7pdJrXHDCGoXtKa9URMWOJoEBewdc51xt8\\\\\\\/LXrqUIxxQAEY02Ci8MZwfAm1d0yumL2cUX\\\\\\\/6ZrsM5dAGtwDLr5zbhKrLKroeyiQQYrBqIos2vTvu3boOKNlZ227hFJdq1AAvbDDeHkhn7dPQ1FkZH6F\\\\\\\/I+BfNZ8KQ80fW1Ps0l2V+EXHD9Wbn+NyXjaAGNeRSs4I4IeLy6Bsn2TdcnlCucR2\\\\\\\/TJXie27GA53S\\\\\\\/dh9ieuo\\\\\\\/GHPquSfXD+xJjqqmG1X0FCNdr80HVBH\\\\\\\/LrLlluoKksHKMLlIm3ZaIJiosHkmatXUBgmb4uDoBsPVuhPH70MutezQQoTHeElv7zBJBsmpyc54p0VFVbgjhiIL92egeKGkqsPkoIcgL\\\\\\\/ds6LYQPxroji993z\\\\\\\/g8Tkjlm+8HIXdN3V859J1Ln+Cm9mMEBMSMU\\\\\\\/0RUuQTlmGSujaOuNPgczkcOUX7oc3mgGTRVX0ehTqDQSBMwaTVDceSMqbQVp1RT9xY+mbibWAiq\\\\\\\/f2no298jlzVflQCOHMaEPWe6tV+9mLVPKPnmbW3MMxI+KdWXK4FMxKMSrqibuXdgEMJvIJccBS6ieVmA7OKEIazoiuv0QS6Q1QtI3hJFZHsyTwhnnVzDVYClGrdTKoqsquR0Z28dU7MuPy7IXsRMc2ZoPb\\\\\\\/qtwdgGRPYuM8FAO8Hz+5m8mxx1fNp9Up7MkBdehcpZbQQOvFhPM3dVylY0Lwh+aqMZ5pPgxFYOBUOVWHngAz\\\\\\\/WTPXNL1mY5Et6aNEH\\\\\\\/gKdljORSrqEK2Fn0Zxzayf3bBeFr6Xt073DmwKL1DYHhufpAdx9nMN8o7Ru6H\\\\\\\/WH0PriVIcvG6\\\\\\\/39Ng3DBVJvzVXWa71ZNlah69D9OpNYBceoWbIr9VELt3vptIhx6a7thu8NoUmGrzKMB+xJ+YPY5o9KXlRhC\\\\\\\/F3AKrHKBopTyBgZJteDPBUm7f1CGKBzCjVw0qEJ13b2iJOEpYQ2jl+bwM6G1ySP1Chuk9cQmhv2fYqutucnjRWxDV+ypEZOYM97FsRB71NRcEJQ7e4+e8+C6brkQ2bPFgwECoNnQlQLdNu9tFddvMKQs5fwPzN9F5e4FO7P6sDDpnzYvbD5uY0QhFtaAjX+nIjYMYfX+Pi\\\\\\\/HqKdv+W6D508q4a9WW+c0xpzEtRNJCGD9MNzIprSOFA7740pE04e0SKp2AmZ1WXQfm1vhljT8YN3TrZPtep2ZTPpc8y\\\\\\\/bJrgw2+XCIEfCNESD3dz0tXGjXKEE8pcNw\\\\\\\/uBqK+DxpnjkWwUdPw2pITtM3oFhDd5lLcRCWxk15wB4xuA9J7FLbqy96tB5OqgLQ\\\\\\\/5FRFTtVaa+wOqM4x0JDBYzjomHBMyr7qYatSQhaB6Ph0qFsVJAeNbOnIiH+FxQniArDB6kIFFbJfGUKB0sneuG420HXyUI735qUHrQoVtY\\\\\\\/LnEn1Mm8NUygQaBzDslJUEkAKht\\\\\\\/2UQBW8\\\\\\\/c9lNnYyZBvRFFwOQa6\\\\\\\/NrbxMECmymAMfzNlPBLYImYNLjJgDXJ9N1qgry2VGMtZaBxTrMT3fhjCnIEM7hI9f4I1cyKsnKSWD69fFuKQ\\\\\\\/j3TY6FMfRfot4pXB3F6Xe\\\\\\\/ni5JJLEZ6EUP3Q10HZv9Cah+reukMSxzOYupq69Hvh2crmHElDsxAUjFG4HfegIAxKN2XoKOemF\\\\\\\/4v+cJ26jMyWpRA12wzw9mkxtOXVRa2xm1cn\\\\\\\/hVaGeoMZDnokrdNnO+1RN2y6G3NiOltJ3IXNwHB4bzO6CJYx82L44YQ6BO\\\\\\\/qUFVEm456HLvZmQRuGaZB7qUOIrhGGzFwX6V33xCNHreUGvGHsg2sNqLfawc8Z7vDX\\\\\\\/THmyQAeXRxvBsyRlv+ksObnMZgr\\\\\\\/9vh7qTZQslW8Rk8GY96a1RR6CvzhiVj66pU94ORg5IRX85h77lhGXYNM+PXntBDWjj9\\\\\\\/95LalA8dNXH7\\\\\\\/2rHyK4Y4G4TqZ5FlBmY2b9AboolH7ZhrKThE0Ek3+XpBQwVlXacdTrB9tpj\\\\\\\/BNng93WCnanXb6O\\\\\\\/0dSBtLMkYN9tfZQp2Iusp2\\\\\\\/shTqjdeR1sEkkt1E02O3ccNpM8u9Qm1UvrbvzJNgzTIyHzDKCvQ5Os7pViagpzV6tqET+O3yF1lfVEoTYOqh+GOeyj2xGi1fBoOXk5M39wpuzt7tRKnPj7ep\\\\\\\/86trSHO1WTpnQwXB8fOiNa2Vym9dGN5dDuYbaDRB41CZbd2vyOVXWNBMSXdeWeymFFfLLk+ywwCW\\\\\\\/dasTw8ePJ8asy\\\\\\\/dmSh2dWx9q27o1VMInbAAjxu7vNJ\\\\\\\/OlRceU7tF\\\\\\\/b0bAgej1geIjx\\\\\\\/ye52IJaLrkDgPnSnGfN2gDRHjhMj4xyHajs8xnrQHhNS0XMUOrQ2zkvJRiqs8vPFdZl4shjHnz5YYOjK7j\\\\\\\/uSDf0NFWhny98BvnIvSME+rK4c7sxwWj4u6T1beN5F8tXGgQccORzdo5Cs18DnAlKHi+4J2hEw+D\\\\\\\/u13Yzmryokt3lNHkR4\\\\\\\/xxBgmRAa+uTxre8RQcGnb\\\\\\\/BENk3M4vZ+GrbcO21VPFXqNUHYaxjEY4bAR2Dks2reAkD8RWxCCRMPCE8GeBYLZzyH992TD+hrnZ6buaX5RQDRJLs4Rl3xfZshIwJf+dMndp7Hm3b+b5UK3rR588twQFVnE8bBLVCLhRT76sfv3jUqNEooxn5\\\\\\\/k3fmuk3vsDozRVKXHAsGIEFPa9ePmAsDiFhiJHj2Mf4q\\\\\\\/+AEsy3vtfdAtx9ipNT0oZvn71FSTzhX6H1h6WZiC05O\\\\\\\/IFRHJIgIrVePX4qCDmN9JpyKVzX+1Wv8k6GQkoHHH17loj9JNfLWC22d8oZ2DO6X7GUwu0rI1TbN6gcAP+zaW8L81e7vlyKWT0CtXu+bpxzuVfoO\\\\\\\/X8kOxINpVYTygz3Gkg6mbrEuMNdMViBLyqK1c5O0GvxWQLfDMZCEh8lXZOHH2ydfTiQxzhEq48F+jR8OwK\\\\\\\/xlRb6YSTHTSeQJqTqtiCd7HjldENKJ1N2gdC8qBH8+ooQ5ip98NLQt0zbSGEUaoivXlH6Xfe3eJ+6jGnc\\\\\\\/hOAy13gfxVC2TBuaYjEJobcobW6Tv0s3xqm8BbedzOfN3fbyCzPYBQOdyJwNidkogVCkYNzr6\\\\\\\/LvZD0L0PDAIp6PH2CkSBxxpdeUjVc7M4NzMD7mr0BZenLdSnkCnkdW3dj2hIuXDi29QybSET+xUdGdfyaNf8Pd0D1x6L208cmSBekrBnhYJyDvHRA33fsbS4qVaPeB9RJm+2ghhjLeQ0eLVy+6CmIkULq\\\\\\\/N1VSPZRa4IN03rOMpKJj7pdGQphHanlKQxdEm84Jn0SgqHO1ya71b9kr1+bDq81AA0O8fZHNFR5MWUJZLrFttkGxOSzCk62p29W3yyJSC3AQowrS6gXxaAY9T2ctsGC9deQ5z5JfBPLP24zl+nMHoqu5QA0YTr0R4EpXjuGBrn2cwNdfEBpSkq8oTe9TpEcv4K9UHgWIBYYD2aJPICa2nJtvvyB19UshFj30BRArfsRCvCX+qRg4Gcinrn+ldLc7FQg08HgjbK5uvmqPieCSrHDzQ6Bn4TsR0iGja5FX\\\\\\\/u1erj13xSVblwN4JPcNMD7qLXkE8KmLW7voF1Zl9xBmdpbJFpk+MEBYDO+VD9iKfU+3vKUUgPIreCRS9\\\\\\\/kp01biYyVbkD+4HujYZ3MwM\\\\\\\/NSCcNcbDzQXXYIDUXiQICN5PbTv+iih\\\\\\\/Gu5X6lkyMKXZ0EbsbiEd5Bj9PA\\\\\\\/PyDhb3JmxJNUy+TYeEOIbH5QvtMyVdg9JItY1WxN1sEMhIXeTHbM2MRwB54WuLu3c7MHAPFuPogK5kAm0hvdfhmxTcnC8Wth1p5ZsDEr2rz0bXHbmMyx7V99gpIfEiGz8wpIlM+\\\\\\\/XpZvLTpyb7MiLgxwOvztsAwUDdpfJwIkIEqNokfPRuWJeJqnmTSTBqEQbE8mxN41\\\\\\\/C73BHWKjo8zgpSvjryE5PfJms19pSEhaAOSx7+XgaX15VgwhtYSKO+yGWyBafOgbGlh7+pYBQlRO6RW\\\\\\\/zzTTSnT\\\\\\\/K4ouSe1Wb3Tpfo5\\\\\\\/4FeWWuuelZBshBEH9oO3emYPPuvleAVMcGxpFy2OgUH6+EPBXarrzEsZL0XcAyNkjeX1uO+9CrbT7CBMGqICnMxgGIENmpn38ulzzxvJPsGFs2JyYEpN7BCx4xTySSiLSrvyTJhmnVXIHW6TWVrEcoFQHeQSG+8lNqUpRTtJd6\\\\\\\/yuwHvW1l\\\\\\\/w+qE9+EDy9osyX1Zew+JbHYA48G6IJ4N7amFfJxgSSgIgvw9cuUuw4sJ7g\\\\\\\/fNmcdFqP1oyKgReyQu3PQkhG50HpnsZFBqc9CltDSUomokNwSiLMQQWj1fA+t\\\\\\\/W9EujFaARZ7SgpcnYnmaRRlHDpLHZ0Q4grN+N4zIlanmL1wVpqVPNN5clVoXFnwPDl2ZlTHzkbZvHNYs+MBlnHG96HJoW0b8rDwGaiF3wgY+X4Mh9yQyRHQq91IeViqS9EsZcXtFhdqQVcetYDGyAiXRa0R8xGUKGtaWoSpyCtk\\\\\\\/FXI700VepLdV1nDo1jhjsR2MjujUFmn7S9ybtGIrN4jL470gjVMfYm1ZhueAguucu86fKBlpQJz9eh2\\\\\\\/3IE3yZIr4pD7RagHTiKcjVW3KSRwMMWOta8iqRL4RoBowtQzuoe+Vafawmny03Kprapr8e6bDyr8Jjc0k8YGk1M+5hmK4PG0F6H59V+XuvedSPnpS6Z9O8Hw3L+oZV0L3uWbUVgLyxC\\\\\\\/GYwxaaPBC4RIeJBhpzOnYQ0UOAY5cfYKhWsUQvumGKZ45ILeHjP\\\\\\\/PyyozCnOw9CvV2dTbjIZAOEfG4ZXvM\\\\\\\/YB8bpebyF1wLxWTbOuwydwTTtbzrGCxk3IP\\\\\\\/l9msV2WUVcYQ8gY5nLpvfdrgMh+s05rnksQOg4UzGLvi5fTtnFMk85oiGdRRdSyW47q2dpxi8nzvz1IzxvaSbrGkYKEXaCCfMnEe0dwNnJXmD1oFi8om9CrA9z\\\\\\\/m4AwlGuGf+EiMHuUSv\\\\\\\/TRUs56z2Ux+OlDBkbVRQrVVpkiEDzM2m6qEHeqHA3ougZ4z0az6oOqGh3qc39ea5e7QcDGEYGuUJIMql+EdHtKmhgZAvLDegCh9ox6Yk0c97r39gZmH0mnGiqjKQbUpOaKyHbglknB9i8GqZ3BNVDGSU5yQtx7F2XveeQeDoux7PMNtPCumAmhD6mOUiB9nheGUmGq22HM0uXx2DX9YGYkSi5haLzavKjEDLydz6sWYpg4HLZAjo71sNL0bnX8qxJFv3twEk4xIX9xaTeAyORLHFzcKD0nFJvKnUeTwUA7kiMxB0Y67Ah4v6d7D0DuJ395NoEEvRKy4PT7LSev1n7tuCe+2yC7tWf0X5gRdwp5RL4v8p+tKZ8MDikF1qzvAvWhSAjBH8idBqxi8QKhUzuoW5Wb9+24t7j930uuIZbfsGTAymTQkmr7sqD8BL6pAf9Av7ZkufoHO\\\\\\\/D1g3ZCYQmfB7B62VHEInRompzc8vipHgBKqJlfhokizPozV8kkomSaJ0wAoqnlnc1dNlbBkIVfd7JKkMdWsO8jMvr8pQ1zPx9BmWCBwWLKJpNsKzCCNlVSi4+3a8IzU\\\\\\\/CeX8jiOBB8+wSf8KUm+KbidJzF3qlGUSzoI3D\\\\\\\/HRq03SQ73WHjJjLBDAxwy1+BCCzC65Ano031b4ZhsdJfI0mCOlNIH+nL44AiUUX6BUoZeO6yVHi7frl9SHvY7+nJq5woli1vA+3\\\\\\\/di0kqTapkA5krUhKcbvIBcsjw\\\\\\\/lvdyB6n6HB5I+LS692JWh84A1s27oDxCsC6IQTXlUFVhnpL5y7X4tBBc7TGK0MEt1o65kZaEa7IVHk8TS2hRyfNJNpZH8ESdcyO6Fv0R\\\\\\\/vZ9Nw5MkJ3v\\\\\\\/gmsfjDmB66nyXJWawN4\\\\\\\/5ENtv0I1fHfrvwwgOoRTSVVnpyooQtXDr6uwiZzDFilXdJEyli+RYg5O2g47boKZYaPztdOD3U5PJ0sx9OJ9Clusq\\\\\\\/vPoNwNw9hinU7MhlEFrowI9PczyPedKTlw3bv4A\\\\\\\/RfcWVenXWycyWk5sewOvOxwaK4HcEYZBUjWgY7jprtlBVYc2W1UyGi8z09pVXnz28EyGXQAJwFyX35Xrt\\\\\\\/IvrWYmrecn7W8HoPb\\\\\\\/pfhMCoC9j6gAzQHnbjHU59bAxA5xwfjAYwglOvBEwU6XT\\\\\\\/BeefYoKslxjPYrSddafsVOgSyUwiwXoko\\\\\\\/pqflO3szK4g41knYJMOh9ZhaaU0BTAdVUrXRWet59yDw7y3XStTTgUigYMxInwLJQYGHL0aYct6hqTzgHE3QgpHnBVg+Hy1p640RTteebMiy8Nbxh7y6b9meQDQjh0Ui353r4m4bV4Q0U1pq41HF65bfuYIRRrWJ3NFoFidjrjSFugguAd0WBKSP9bg6T37pwmPgSXgan9n77DCF\\\\\\\/QlYwXB5pJjpTNHmDgESwQRqXEtchAEu9y208T5n1Wv5awa47WHvLq\\\\\\\/Mp6NUdnk+TKDKb4d3ofD7BXBrihL\\\\\\\/1++LHRVDzvdvP6ftKsU45VoZfMMXzOxltm1Gd2ezNqbp3nodzgIJPNpDYIATM1dsCB9z7792qdsrXH4ei60eJ0ZxxkHwdFO2Da5b0CBvWevfhZfIYZkAyrgpEHym7x6tgLXFPpBs+7vCbBOVSIvknpun9aOj4rBrhI9s1b+kfSjG2YqqwWIqlGVFrMIUBDozfy9nq3J+oudpGT0NHrY97No1tZzLfatWrxu2CbVUjQLGWOZ+u5hHGtWl3qbLlvkig3jHjN\\\\\\\/xC7mGkSeJDBASOrz5HyWIRMGAoX\\\\\\\/SX40iVKbzhjeJ4djB\\\\\\\/60IiSTm\\\\\\\/nRU6vowa29TSGBsfTev96fohbeEN1zZU7MldHxb+gSZvVaERs7OqxZ3wHU2gtc6rZ+42cbxZbJfpMF19edHpOhWeo0hn8Low3W0TT3SK3WhFsfmKYqC5nEM5o7ZTQoYRiyPj2OXfFaZTTBWipEFN7vLPWwNPx3bhLf7oMt8mn3wXUN34AcBgcZfz9d0\\\\\\\/Uf3nC8eNcRm7C4yt+HvTnuI6Xi69vVIhGcJd3fQUO4lfW6OU2W\\\\\\\/Oj1XadBq3cMfnKK0z8LI5sT+cOJfGeBlmLHB+K5fKQzJqNl4sSLGiYo5mpiaYbnlorg6Q64S8evQ1hZdG6y8E+CL+aW89obd+JpH9OUxWMhg6++\\\\\\\/00mw1usfR9g+zmvQiuxQM4VMaij7jGxbCF3uFXKjGwYxg\\\\\\\/4NFhHt65CxCyiB6+vY2AvfqOjzYR4qu27xzzRfhNBrORYoE7yyDfoIv5pp0\\\\\\\/yOh2Hl\\\\\\\/X3uSgc5sQ+8HLsAL33wBX1DhVSJI0ab4k+y3rsYHFY+mU2kJux\\\\\\\/Qdh8GCRgCfiyT0hDN7Z4rzzGbioS8EtYY0zi3bKV9e6tkl8+umjfb4MCV2TWDsvEIXv+Hiq2va1jFFo9SfR6aHuLPqi2JyIdqEXo7Lzz9kcq7X2jwRHbv4ElIWWyt5sO9\\\\\\\/Gr+7ZK1\\\\\\\/uR5Mn1jG6de\\\\\\\/oBVvDJ\\\\\\\/qAv9TMe5L6P85PFL0gMA21eL367msn+zS1F56mtG2MPAfZHNgBy3PItWkofSyRM3+Qsxp33zRXk7qJl5afX8ABbfaTGVyrgwjbROJMfxNb65kuDBzW9v+JSOfk8JNc3sLsnvAzmabvRfnA0++3H4NvtFiPWA9qjCwnYYAxJtP5yPilNpHmPZQQYio5BKncOXtE4zloDDB\\\\\\\/XtNPdZPdqgaOCG4UI5RnOIGro\\\\\\\/Z4CWMWwzdC2v+X47pEI\\\\\\\/NZBSA+zwXTOuP9LbKeovZpkZO\\\\\\\/yX2vzD\\\\\\\/z5Lg0Th+rVQIHqDVINilSmvOpQ7Qj0+3qez9II91IpoCBx0svwHXTOS8IP0UQMx8xyxM5kO4oU+fRLi7yOivGV2beECr+4ARA539pbpbhOWGpP9WDp\\\\\\\/zNj3r+UZVFUk5xmHmuKxGAjw1xcJNiGTbItgcmlUUpFRrkplNepHM6AF7xhAbgBsORk4Lo+UovK0wNHCJ6ai+DoBwzdRjDQQ9kFDtsCc2sqTq9T0TIi+yj5qc+F4yj6PkdO4qm1DFcwW\\\\\\\/QT0KOU+RPOJ6Yy3KSi1xRFPRFZ09zUmXWayXA69J2k5p+lBVPQNn38VZnd44lM7kPKGhJz9rbyVoqsXVHTOEvadFgLsuYMFBmuxTIPb+puDys4sG2XRMPAcWI5n9Q3jT2H1ATvEqY5\\\\\\\/vx3PiUWpo453rtzB53a2yze8Gnh6RXFVBOqvDvPy8tjYLj+8Wa1bS9OsIpzXuQMWS6OWZ5RCQ5BdJBJabql7sbzz1UZdpgCKP9+sLAPC1giNQbRJE206wNVG8K8eJkUEHExdYrOUHBWza\\\\\\\/s5VuNW5O\\\\\\\/ry\\\\\\\/p6tnJVXHQUjtI27ui6ggdABxZmhRGp8Dzj6kR0yMd14FMZaghBHF1IuINMgEBT9+47b92LQkJJSnYglBl\\\\\\\/8iorzewC6ilnGIzAsgU2uwqzi+fRSrsmP7cnS1GjJ9NGscRtA6VGcTyZDZJedMhs0kWE8SLq06m+zj57kF1ICe4dYxAQPBSuXcRuB2AT7yWUZhK86FlYyWQLWVR+nRo55N5bFbigRYqmrV1JfxJmJwS\\\\\\\/fIQj93Vl2+8MrEhhwsEPXjpNDWMdQOc1H7FKrsOwOSiaDByK1QAFEIbvXfsMQNdBE7Y27viW90JklkD3ZZTyKk5YDFApzvdxwUKC2t3mP\\\\\\\/jAWz42QodEYWoF6ZE+xI0QNFAkM1okp1IsQLqefjfEeOUPDk1sL+Gbz83m7TSZofeTwQtIkVjJfdqoHEA9Sn\\\\\\\/nd0GXoo780b+nR+4Cg5WZYjow5G90w+INOYzb+EUSanDO9lIW\\\\\\\/bpojVynoQGgkIEjy9c5Sgx\\\\\\\/VdhBpYNWnSDYvayhMOiUhkqubbxgU76FJKsJcKF3PrGiAjRGaIlrBunRzkK46Cb28hL4Ho0HE9hG4QjpvsUPckxn0Y7JTvThL6sacv7FlD4tSk3Gea2jcBh+6UFZ69HBIsWcfu+26ysuL3zMIoFhCzYyil2ddswmBLNxvjKMoy3wWC1BdzyJ68vuK8EluNx\\\\\\\/9PjGi7KzpFRnKW3HpMMUfv1rETHhIioBaPWc1jmZ6uTwszUJzOXWoXVBxqHgnqJ1ULIyDr6Y3Fr6gtLa0vn3YcIRADSz4rGCCVr1raqW0Zb03bFlz6VrozeVsCQjYVk0MEzy4nfXtySM+oZOMdlB7caMUaPYjinoYnLsX3Tlh25BkmUxeSnvkffR39aG2yB3pQHTiBxC24SRkw1TNr1GLwYVGH0xYf4XLQiTQ1SopKQNxRVGcPVgDXa7TN9GVuqrR0keDKyA5JDQz0UbOUYjoTc0u9nGp8DXr8CARFf8MklXtClma2FHMzDiog39r1Q1LQjJuZ0XMzVBab38s2I9\\\\\\\/1HB0q7p\\\\\\\/5oKmDvhRjWW2PKxbXYcBu1xYous2LR\\\\\\\/HFt8utCgkHpBLnGKlabMYridApwal7+0bzEQ3zfqrOPfKXk512VeUmZZelHhQFEQX8\\\\\\\/jCw1f\\\\\\\/jay9NAWIpQ8VSOAIZseBrNKri8fQZzmKrdcHZQX66htPu1Fh+AzdtE1fOl0PQ6r4pzXPbODFzSHO9qla\\\\\\\/W2\\\\\\\/AblCcKdkksrJDssqok\\\\\\\/cB1Zo3+Gf6jZvLpgP85Ol1FC7P2eweTuxu4PRxT6TPg3x9qfVwsL3ZFzG31PO6EpRQVkkTYvQc6W+iss7TJhf9Ob1hdDvgn73E+95pRVFgFR4pe9tkiji9x1gzvnUZWd9i45U1IVslrcncnw5I3sXWTketOU3nAMwdSbReRvye0gT1R6J7thQMidm9fu9n8aWHf14RHjhfuBIDLyO1uICmCyYGSDGhDIAzkAKsQO6ZV783GAtfzChKD9Zki9IA0jG4\\\\\\\/HbDu48xBrJXI1kSiSkE3Sem8WLDh63pSdhtM30+uWEi4UtUpaHotdJggDEwbxzv0rCBBzJAWcOgp8rP173MewQgF6pkkitB2TwM1\\\\\\\/LIAD0hBmm6zjimMjAR6XauBTN5XIKEIZ87KA\\\\\\\/cejiErvXObZ+gTBdxwoTdW7jK+yDz+ukhjoLrbtieKegZwxMqa+9Tb2p8ibzBksxMuy2HrOPDs+pkxoysHPhzugaBgLmTT+mGEUj\\\\\\\/N1EIPxciCkecyHUZoBZ2lTdo3hDXwO1kUPKY+qtZNtOA4N0v\\\\\\\/PUISuYftWSbBrsWvkCfZ3fI3LgE9QreY1aaoSAdd0Dz4DosAWgGgGHJriaiDtrzGDl9NT0XRV\\\\\\\/Q7ZMKyGzmjQeou2aAJK4GgaCDIm5RQWUAROltfru47XNABB8oPvlADn+SuALdovhKXWPqFGMUS8vjFCmfHd+qeeLvHiYwPJMmt7UuUs3ht68WyXl3xm2Zr52XbQsZlVw3hZnpb0s0i12LNQJsEtLIQIAu06SanvsyKhUwrt6SW0Wbi46uVIYKnXJDA9QMHf75tWr7MMqB+qzsFCUAlJvFy6nnEqqL2eaUWPSfpU\\\\\\\/IZjphB82SLKZ6pAFc9ft7N5hbBxyXUm3qZ3vAigTKMMXNX10n43jMucR9vYwF6sD4ZAEmfdSbdHS+a01pBX2Q2e2A+5pSsSA8LTbptCeLgNMgvfRz9c6q+wCNqLt9wureLFPp\\\\\\\/xfYEGTka9nSGImODOHrXuJHJla2grehTy+glXNnWrxGQcFWcFHPG9S3dXYoIyHvOQPm+j4fZN0gC6Uiy\\\\\\\/PRFcEYBrk+KPu\\\\\\\/XUdzigokRNkqnpM8euUzRexhyd\\\\\\\/eRMlTLnbY1Gxl0K3FPnUdxXzQLvNgj1xkYkHvar9sF7i818XUl1WHdbelxvHrVVrQLZgWykaWai04Jhm+J3RpKEtXbKL7\\\\\\\/gBbwtB2gUeCbSziGJP6\\\\\\\/DMdKWNJnO30VVzCo7NkcB8hPIzyOMGmC+2B3owpz0y+2gpW8p9YgbKHg42vFsPyFch8mIlo8mHKKuZ\\\\\\\/fd3M9GPSYqq9oXkIE5SVNG3w8wNMrvAqL\\\\\\\/jPrdI3pYHCQQFZqi2SQ\\\\\\\/A1m8T8Bze8riJadeBG4gCl2jrxSAPWF\\\\\\\/d2NlnF84sdHOU2s98vwHlyP5p1bOTEz77weM2lsuU2Ro2HycbKHnk176SUwlKCi2JC9addPQloE4wLWRpRHRwqGdR1CVoOZQDXc49CVz+COtfL1xDdRIdbyrXX4d2JmLwD0cPJq7y7pfu5cbrrS2byIBfyr3PsGZlTkSjAhHfZtTOq9HhwzK5IGDY2ZVozFgXh5UhV3qa+0A+J+Yx+nPczFCQ5bWh8idevqWoXtCFTyo2y\\\\\\\/RCR8Q6l3OOkEiv98tUBlfwu8iiRBeqgTLnd2eK6c0ZYrVbEK\\\\\\\/f2YaCjaRdptZeyCack+1Tkfoy4sm9o2jldCZjY2+eTs5wRedhgI3i+A5y2SvV+p6dSAS1SFpO1Fs3UbmGr2FBPVIC47jwxcA1ezAXoYnDt77Ec9yJA2bxUW+t5zviku9+fiS++l6EAJBUJaWd+0EZ5AOYlsjXvAZpL7LZheu52u+V2F00vKhUIx9NGIc3UqltWcfQoM0SkyLNBNi7Z5uOdfrF9kf1bQpY5\\\\\\\/vFa9fRntzBG8DMir9merJ1O2\\\\\\\/UEk+7S+gAkU6cdiTJnll\\\\\\\/XsxCEzOoqH+luOMrtt0\\\\\\\/84A2tRfok17IhmR4FwVc1eYQzOETaHpllhVVF7BNJgiIOqwczi1Zwzca6i0t+70hJcb8p0ZkVZ3tgqWzR+SAHdAbux59ccu\\\\\\\/SW1abS0BwhvgXIvFbcP3A3XBBT8lldx5LrXxd4zvIHe7nhqr\\\\\\\/EpXwV8YlKGa962dxYcw+304VKjdjlwCiVm38O6XnnTP+Smrw61ypS9tS9a8UTTV06A1VJsDFaA5K9u9kkxzQxqtYUhfh+Q1rxLQuZcjngCYbOMQ86EuIQNnkigyvYBPcqKh+SIdWQdbia1KSK4lUZqPY48r3sfdLMOZg4\\\\\\\/HgQLLDi4nP\\\\\\\/D33o7sgS3t3KlcuOQexdFZZZgduIty\\\\\\\/CaxflbvQrs0EE9v1HFi2Suz0pgIPieisLulqxKOTyezW1Hhrlvd9wOe9g6zbM\\\\\\\/Rf0+88lX6UZsDygWqrVN5LPzDx9DzeF5ORgRM7P35WHsNk21+dJ\\\\\\\/9r0D4vAmfz8Xf1jvyo7PxxNnVwNSUBJwHLkzJGxMW2p59zYdhamb4Lvoz2UtokVpoQH0LCMtXZrrLxzda9uXM9UsG4oD1mobtBxwpOur03O\\\\\\\/McHoUlpr5M8f5GUdvfTFD7PCELhPeKWr1FqAi7ZsdHDT3LadkGqtHLbpnwnM1+vj0eJJySoA8aSvARlItTMCeGPEayQ3oGPGqKZGh8KUNkUtm+3eqzWbO32qZAE0aPosBTCwxDPXzlUfQ2qGUVjTpBf+ZGV\\\\\\\/urtx5I49eg4UeC58YGUOWNdFuHwIofYTlCBnimeNRLtAGb6qiYn9c4KRAPIkgMrRZRvtenRLnB7K2wvGbWYIhP3\\\\\\\/9HNVpH7e97ogL\\\\\\\/sh2jel8AllN+NDVa+kl0yKaq4BjPZGxER5Se+NuVqcOQv5q9iV3Y1LbLglQwFd\\\\\\\/Ko8LHg2Uz7WwGcdNrbR\\\\\\\/0hfWtF2KQklNOR65CRee9mOZMpo7w8Jja8dAlRnImp7prkBFQ2yeQDcWT7X5fP9oe8zkxvfIvHV1A9ZzCOk0pU9txSgToEnvnaM63kmCJcW\\\\\\\/r3+755AWb5EJQC0iO6i4HZiIoCPjGuKze3dGQ0KI0oRycm9oBDwc69PcYLb2CodykdKZxW9P1dk81wL7PsZsY45QF3Wy2Dy\\\\\\\/o7It\\\\\\\/7pMl6MTmPCCb5kGL78JBOS4qzJ5MyY+dk6Sf+oVZaIltxevC1\\\\\\\/krMyfo9aOCUC3HA6msSlp\\\\\\\/DVIQ08mHuSf7mV+zToPmvfhPpdypSh6dyI1ofAmzFNDs3GRXeKMAA8\\\\\\\/XdztRgfHQEs+lUDMrpFKljQbdCM7lc+m3sv1AKK0WOJ6FzoyfflckcQPLyJ0\\\\\\\/b35aGvp2FT7a9GB5vaDJ1KMRAt7o3OFkUVA8yrk1KOzsvaRty6HkJLahlKCcPUsUmf5eKFilGq1mqBpXiKwKRzXKRmWeBgcPlz1q\\\\\\\/OEpzGqlW8+QcAgcZi\\\\\\\/r+g0N0fR4YvqC0hzgnmN2QIoRKB1p2wyWZApLxRpV3\\\\\\\/k+YI1Yid+jVlFX63GCUnte4uG2lth7+rgMX8Jv4bJNu9bFEyFMhk8gjauD3sQsFu1upprPtQut7HsXwgFDR9KKyFijaLK7bAwC52axj3Zb8Wr+TEK1kgVpFwDwM3JsLO5+GXIZSrMsB7GuLS9mzGuPTokxMnwBHmnVEYBZtDljwibhn\\\\\\\/d3t6ord1W+OgN00Qcz5iidJKxuhEMkVwr0jIr6NQHhbCpQPSe3qHtLXwCHJhzkaeELFu7VYYchZtbrc\\\\\\\/eVF9eytgonZQ20OSJPxlf6w5WVX1+iHqBKqycxRc4z79RfD6hlCjHrbmWv+wH5KREKAxs7VmX0I0Xr9X71yZ6be\\\\\\\/k91AF7WUkExyKRphLCLCJ51UsLgfIbh090\\\\\\\/23fDMS59u404qbz4jx8pL8x6eQSyv+CoHKLORzezVjaA0bB3hDqtuyHlL1wgh1IeCcCPP7KmuUvphW\\\\\\\/Q\\\\\\\/Pp4bUsvg4EImr5SNam0NlJxcoanGo6KhLgTGqxz0qdymwCh\\\\\\\/Ohmurx27lT7eKMvO4r8DkNCEtM+a+JbPY2i71pNmt8WUvN5t\\\\\\\/hgFpjuPdz6EkmN\\\\\\\/eh0Zi65EUVM3ZcSDRcXgrSQPU+TRqYXOO6gPFogLtw2ieUJX0jAaKxyh0enOm4DHjpOX8ALqgTRObXZy3k0N\\\\\\\/IgUvb3Zh12ZulQVfzHE+AoYHidc0JQrsOKC6vG2zhtjfCBN9kCKWmxrdLUahPXxDCD7OS6o5rBLBlUbgMTfyczOhYJbwYR+ljshpGbnUazWK6uTvOsKc5wIbqg5PoV5HBTEmHLC\\\\\\\/DwICcjeGMEUq18J5KG4\\\\\\\/TqLhznyoJGhvMGj5muchZXEsrOuvotB8PlYubDR\\\\\\\/\\\\\\\/09jIWwj17Eou3zKxUEUmgjB3OxQUWASRP4yoqMp2Y2+Nvb2j+IvX9qiHuQzhCfjirHBek3aqsHz59UNIz85xwIPGm+5yUQU5jNH7VtiklaCbx2ActTI+VysmuNFBIGV7qJZJA8yeQy4RCsAgUCdQGqWOhMuRGin8gnsPujCeJ1ougazEy+OlI+Y18Y4iOttm8+7arVk07dBJWCiRaomCu\\\\\\\/a3A8eMqk1hY6NvV3UYNDXyUcCQ0wkVH0MF7gD5tdNz5ovogkLJYAn+Qof9\\\\\\\/bTI5HsB9eMn+exKWwgeMujf2cWyNQiS090O84+C91SbDyu\\\\\\\/TvUgazL+2p\\\\\\\/L1DTsutHD5A9gjgpFC45m15pAA2j4FkRXdycr3JNmcHErlPnPIPyvLtjTQ1SGx1\\\\\\\/JSvo\\\\\\\/8uez+yYCuybQKVFJhQokMZYS71\\\\\\\/K82WHjLU81+brrGy45eV3FwtRNw3YUBSg5kTrOGNva2hnPoHnkwJ\\\\\\\/eZgOkXjsUVNzxP7HNhGP1azUBl+78BHsCbFN+U6eqvH9paG\\\\\\\/PLegAurzW2rhtNdMKZZBvbr0skWBMuxoxAvdqop8fzvsH0eosuetEFUjbYXJz2F2qr2juO+eNZldCk\\\\\\\/AIs25d\\\\\\\/VkY9J\\\\\\\/nWasG\\\\\\\/tEdDzs1VlFFQaZxVV+AaMPqzKKCbCAzLBzUJmkgOJIBrY8doN3X6LGLEjuAjzTrEBi3fpMOhsKD96XPpWwihB0asqS9K5dsJ3spIFkD9suGaL9MsQ1C7rPXOilQPf5O57PFDm\\\\\\\/OfQk7UPFIWthqPPDNa\\\\\\\/zfdXfIdXQZ4abmdWPPMWM\\\\\\\/aQB1L5JC\\\\\\\/By2rA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022f1kyRRZtQvhGomwBUd\\\\\\\/0kg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226d5bd05d4cd45d40e37211656bd6a8b1003172fad6c9c14279fde2c48f674708\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PACdsx6n\\\\\\\/IXAOqUOtmlCeIwohkEdKl3L89CrqKh05kl6kvNmvTpasArNGEoDHHquKDWkBRwy2BW6FJhUlpjHdxw6\\\\\\\/Jpk2V97UfZPNwXrEJytC1tbpDnAAEryB+WYPpGYVAYA5tTtsZxuxpyWehHKlxu9ymzGc5ycs0de2OuSoJVzJ04EKWq\\\\\\\/ZUTMLF3eNxV3MwwVETMao0ubiAO4PW9Zhv761xcL8xI30Zzgw0XvsIzQq2aopSG1FujFp9XisvoB\\\\\\\/\\\\\\\/yW85a2HX+95G8TwnqCx+iURUPga1FPI4OHfYS7w5q1t\\\\\\\/r7tMSZXAndkj+HqURNvDNMpaB\\\\\\\/FAB3ZvHs4H6XHBA6GMA+ahpGZt1+bflcbO7Dm0yKBoZllkeGebBCtyaWJofmKMzNODJh\\\\\\\/E5MEEsaKZqReY0yAwPkbIuRB8zbyqrOV2DZpLFzXg5DJgDc8PaB9QZtSQvQ4zMJnQgi5PiowjHYIqwlN1+IRMKjN2Il6dlUdi+pmm26iwUQWMxLxy\\\\\\\/OeNRjaJAJiA1Nj6tdqbHGC7V1n5xKQ8GLoKlF8BN2ZoycV79+LblhH\\\\\\\/eo6639laOD0MSiVneWO1fDU2uY02HC6CM8KtU7GNTnbd21Z5WOGcuqqbKARCoQ6MKPX8nzdDeDWe8+G1CCm4Ow3ZPZS5VycXTVBxqgzJ\\\\\\\/0DGy3m3Jy13y84tSk5rwQeP82TsvymvTORVrjpXkjAKISOYtk0\\\\\\\/i8mQPHQoQdfRDgEj3zwFPXzojo1kZNnxtsaV+4sl+xmSjOiUt6D0AD7j+Pjkw1Tb\\\\\\\/VIr06Cqhb9b4ac0TU+9hzdRaKL2NCR6IrNYMVjl6fSzvUWegpa6No+EDaCwn5GWPmUXLdA\\\\\\\/8lioALOM9L79TDDzPzWx+TfdFNoQM7wVo5r1bA6kzWwB86HmLNQGRYNq8qP07dff\\\\\\\/1U1hEw58sC8v2e2pWG4byCiq8jxS\\\\\\\/S86jJNH6yko5LyzV0wwQMaV2udDozaLPqwNM1lE3vD9m92ckV5gDFkbkJumyvMaCfjXxTubCsYRCBwL\\\\\\\/Nx7+LAtEOfEeiYOXkuypeAp7Js\\\\\\\/P\\\\\\\/pBaMVkamaluXFrc6Iz\\\\\\\/D1CjVGxbBw57FFEwC1ZVPnfTsZ2SNCdBcYZ\\\\\\\/p6YE43bYeUnjNRYEJcPDqduaYp4YDlurbuDNQ7yQET3ZVZM2OuZ9WMNnl\\\\\\\/YfT\\\\\\\/gV\\\\\\\/E2co0yLffbMMtkz2GIwD4pO8Tl7GDg64Tq\\\\\\\/tps1+RVymbG3Q4MYyTuKqiSq2FdLkWHnjWBXM5wHHWENVHmnTu9OSr9H63WZg\\\\\\\/he40ZKQU25nWmmXqZiqxLhbtFWkLxp1Pz1DZ5tngJvgXwD6xVBETnXhShntHL7FkWqYqCtBkAnVPbZYZ5kize4fThPGQfkLrrvOW++mbjGPa0CX7rOJJDHv92k8BjAXTaRbTaSDZw0BGCH2o5ErRYWDsJW7dJO1BY7uKFCMfr2IopgGCQjnBj6D\\\\\\\/6mtSXVX3qPlg3INCWkIloqDJWgntfIR+f4aN+\\\\\\\/pjCaXJqvUx7qWUb9ZQSDNQxIAMwoqRkx8UPYUardR2ScFZUjlLLsB6mS83J8T6NJfP7urF7dX0zBMM3mpBGUPtK2ugtttLMK9r6Lp4TPRxTvk3KiNF6LTovMuaDfFgc3rn8v4sPasXCFG11afSoirgAP7RIoiYBKbRSjP3iO1j3t63R8BbDwGXRerDUIzUEHr1cPzCqzbKPwMM29rlxYYRF\\\\\\\/X+x5MwzAUuSPRt+hVxJQV5z57m8isH5prlLXoR6o+bQpHDrWkMiz4Jm4Eb2pOO++VBpom44D5a1+0TBGnoiW7uPJtnRASHTNgD\\\\\\\/SSA3E6DkwIw0UkqKKAnEwOslRp8bqZAPpG7TWkKBLUnKYwsQkN69ijo4YIhjqlK1YDIDcIsD9Ro\\\\\\\/53+WuVhr5qzoikvTPA6qCR2FGgCIJmJqDDViuaJnzdJ35RVGHr0r656OxYkzwJvbMS1FywDxgV3cXE2YVZwf1UtS9mnRMACqfQ3QeDNjNf9hi+ZxkWg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vxEtEa2qgaPdwSr9eH+LlQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fa6429cefeaf938fd10025ca9b755215193176a0ab62a7f8a182c9a7347e6a98\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459802 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/541" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DYOzJ-XnAcnbd041il5z-0quLnSvlS0CmWMgDfE00vE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/541" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iddHeNRkIwgHBmYE8gQ-DgFe3xDpGf1f7LdKtjvqdQQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/531" title="Heritable Variation in Courtship Patterns in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/551" title="Comparison of Larval and Adult Drosophila Astrocytes Reveals Stage-Specific Gene Expression Profiles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459802" id="node459802" data-pisa="ggg;5/4/541" data-pisa-master="ggg;g3.114.015263" data-apath="/ggg/5/4/541.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron J. Lorenz</span>, <span class="highwire-citation-author" data-delta="1">Timothy M. Beissinger</span>, <span class="highwire-citation-author" data-delta="2">Renato Rodrigues Silva</span> and <span class="highwire-citation-author" data-delta="3">Natalia de Leon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">541-549;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015263 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459802" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alorenz2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">alorenz2@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy M. Beissinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BM.%2BBeissinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beissinger%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BM.%2BBeissinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Renato Rodrigues Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renato%2BRodrigues%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Silva%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARenato%2BRodrigues%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Natalia de Leon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2Bde%2BLeon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Leon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2Bde%2BLeon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/541" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459802" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459802/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/541.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459802" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459802/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/541.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459802" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459802/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/541.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459802" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459802/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/541.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459802" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459802/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Maize silage is forage of high quality and yield, and represents the second most important use of maize in the United States. The Wisconsin Quality Synthetic (WQS) maize population has undergone five cycles of recurrent selection for silage yield and composition, resulting in a genetically improved population. The application of high-density molecular markers allows breeders and geneticists to identify important loci through association analysis and selection mapping, as well as to monitor changes in the distribution of genetic diversity across the genome. The objectives of this study were to identify loci controlling variation for maize silage traits through association analysis and the assessment of selection signatures and to describe changes in the genomic distribution of gene diversity through selection and genetic drift in the WQS recurrent selection program. We failed to find any significant marker-trait associations using the historical phenotypic data from WQS breeding trials combined with 17,719 high-quality, informative single nucleotide polymorphisms. Likewise, no strong genomic signatures were left by selection on silage yield and quality in the WQS despite genetic gain for these traits. These results could be due to the genetic complexity underlying these traits, or the role of selection on standing genetic variation. Variation in loss of diversity through drift was observed across the genome. Some large regions experienced much greater loss in diversity than what is expected, suggesting limited recombination combined with small populations in recurrent selection programs could easily lead to fixation of large swaths of the genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genomic-diversity" rel="nofollow">genomic diversity</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/silage-composition" class="hw-term hw-article-keyword hw-article-keyword-silage-composition" rel="nofollow">silage composition</a></li><li class="kwd"><a href="/keyword/association-mapping" class="hw-term hw-article-keyword hw-article-keyword-association-mapping" rel="nofollow">association mapping</a></li></ul><p id="p-3">Silage production is the second most important use of maize in the United States following grain production (USDA <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">National Agricultural Statistics Service 2014</a>). With the expected increase in consumption of animal, especially dairy, products worldwide, as well as regulations currently in place in the United States and other countries in the world related to the need to increase the presence of biomass-derived biofuel production, improving forage yield and composition has become an area of substantial research and development.</p><p id="p-4">Maize silage is forage of high quality and yield (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Coors and Lauer 2001</a>). A major difference between maize silage and other types of forage relates to the contribution of the grain, which represents approximately 50% of the total biomass in average temperate maize hybrids in the United States. (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lorenz <em>et al.</em> 2010</a>). Cell wall–bound carbohydrates provide another important energy source for ruminant animals. Breeding for silage production in maize, therefore, involves the simultaneous improvement of forage yield and cell wall composition. A substantial amount of work has been dedicated to establishing associations between the relative contribution of compositional properties—digestibility, carbohydrate concentration, and protein—to animal productivity (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schwab <em>et al.</em> 2003</a>). Summative equations, such as MILK2006 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Shaver <em>et al.</em> 2006</a>), combine forage composition and yield to calculate expected milk per hectare, which can be directly used as a selection criterion.</p><p id="p-5">The Wisconsin Quality Synthetic (WQS) maize population was developed by the University of Wisconsin maize breeding program nearly three decades ago and is currently in its fifth cycle of recurrent selection for high-quality stover and high forage yield (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Frey <em>et al.</em> 2004</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> 2010</a>). <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> (2010)</a> evaluated forage yield and composition for each cycle of WQS <em>per se</em> as well as topcrosses to two commercial testers. Linear improvements were observed in whole plant yield, stover yield, and whole-plant composition both in the population <em>per se</em> as well as topcross evaluations. Although stover quality <em>per se</em> did not improve through selection, milk yield on the basis of Mg ha<sup>−1</sup> has increased 24%. Changes in silage yield have been greater than changes in silage composition, suggesting that the current selection protocol tends to emphasize improvements in forage yield compared with composition (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> 2010</a>). Eight inbred lines have been released from the different cycles of this population (W601S, W602S, W603S, W604S, W611S, W612S, W613S, and W614S) and made available to the public.</p><p id="p-6">Recurrent selection in plant breeding is a cyclical process of evaluation, selection, and recombination practiced within a closed population with the goal of improving the mean population performance while maintaining genetic variation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 2010</a>). Maintaining genetic variation in a population undergoing recurrent selection is critical for continued response to selection, but achieving an intensity of selection sufficient for making genetic gain can be antagonistic to this goal. Studies reporting changes in average diversity at the molecular marker level within maize recurrent selection programs have been frequently performed and have found decreases in diversity in proportion to that expected through genetic drift alone (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Butruille <em>et al.</em> 2004</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Romay <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Labate <em>et al.</em> 1999</a>). Low marker densities prevented these studies from examining the distribution of the genetic diversity across the genome. This is important to examine because diversity in some genomic regions may have been maintained by chance, whereas diversity in other regions may have been completely lost by chance through the fixation of large swaths of the genome due to infrequent recombination. Only two to three crossovers per chromosome are expected in maize (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson <em>et al.</em> 2003</a>)</p><p id="p-7">Dense genotyping of populations undergoing recurrent selection can also be used for identifying signatures of selection, as has been performed with model organisms (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Parts <em>et al.</em> 2011</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Turner <em>et al.</em> 2011</a>) and agricultural species such as maize (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wright <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hufford <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wisser <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Coque and Gallais 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falke <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hirsch <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>). When combined with genomic information, an array of statistical methods, both widely recognized and recently proposed, hold great promise for identifying genes underlying phenotypic response to selection and impacts of selection on genomic structure (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lewontin and Krakauer 1973</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett and Hoekstra 2011</a>). A disadvantage of selection mapping stems from the fact that selection is often not performed for a single trait, making it impossible to estimate effects of individual loci on specific traits.</p><p id="p-8">Association mapping is another option for identifying loci underlying variation for traits of interest within breeding populations. A major setback of this approach, however, is low power to detect rare alleles for populations of moderate size (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Myles <em>et al.</em> 2009</a>). Another issue highly relevant to the application of association mapping to populations undergoing recurrent selection is the fact that alleles conferring favorable values for traits are expected to change in frequency through selection and thus contribute to structure between the different cycles of selection. When population structure is corrected for using a mixed linear model (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yu <em>et al.</em> 2006</a>), power to detect these alleles contributing to genetic differences between cycles is reduced (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rincent <em>et al.</em> 2014</a>). <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wisser <em>et al.</em> (2011)</a> proposed combining selection mapping and association mapping to overcome deficiencies of both methods for dissecting the genetic architecture underlying response to selection.</p><p id="p-9">Assessing the impact of recurrent selection on the distribution of diversity across the genome would further the understanding of how drift and selection shape genomic architecture. Moreover, identifying genomic regions influencing forage composition and yield would be beneficial to silage breeding. With this in mind, the objectives of this study were to identify loci controlling variation for maize silage traits through association analysis and the assessment of selection signatures and to describe changes in the genomic distribution of gene diversity through selection and genetic drift in the WQS recurrent selection program. To accomplish this, individuals from multiple cycles of the WQS recurrent selection program were genotyped using a high-density SNP array. Phenotypic data were collated from historical records of the long-term WQS recurrent selection program.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm</h3><p id="p-10">Details on the formation of WQS can be found in <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Frey <em>et al.</em> (2004)</a> and <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> (2010)</a>. The breeding protocol utilized to advance WQS is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Briefly, for cycles zero through three, between 400 and 500 S<sub>1</sub> families of WQS were initially screened for general agronomic suitability in a high-plant-density replicated trial in South Central Wisconsin. The same S<sub>1</sub> families were simultaneously self-pollinated in the breeding nursery. Approximately 50% to 67% of the S<sub>1</sub> families were discarded based on the stress trial. During the following season, S<sub>1:2</sub> families descended from random plants within selected S<sub>1</sub> families are crossed to testers belonging to the Stiff Stalk heterotic group. Resulting topcross hybrids are evaluated at two locations the following summer. Evaluations used standard field plot techniques for silage hybrids to estimate forage yield and composition (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Frey <em>et al.</em> 2004</a>; <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> 2010</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/541/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of the selection protocol utilized to advance the Wisconsin Quality Synthetic (WQS) population. A second generation (S2) topcross selection method is utilized to improve this germplasm. Inbreds derived from succeeding cycles of improvement are developed and released. Population improvement and inbred development occur simultaneously. The red oval highlights the approximately the 20 S2 families that originate each subsequent cycle of selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1561384664" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic of the selection protocol utilized to advance the Wisconsin Quality Synthetic (WQS) population. A second generation (S&lt;sub&gt;2&lt;/sub&gt;) topcross selection method is utilized to improve this germplasm. Inbreds derived from succeeding cycles of improvement are developed and released. Population improvement and inbred development occur simultaneously. The red oval highlights the approximately the 20 S&lt;sub&gt;2&lt;/sub&gt; families that originate each subsequent cycle of selection.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/541/F1.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/541/F1.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/541/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/541/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465836" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Schematic of the selection protocol utilized to advance the Wisconsin Quality Synthetic (WQS) population. A second generation (S<sub>2</sub>) topcross selection method is utilized to improve this germplasm. Inbreds derived from succeeding cycles of improvement are developed and released. Population improvement and inbred development occur simultaneously. The red oval highlights the approximately the 20 S<sub>2</sub> families that originate each subsequent cycle of selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Details of the forage composition analysis are provided below. Advancing WQS from cycle four to cycle five involved the same procedures except 200 S<sub>1</sub> families were initially screened instead of 400–500.</p><p id="p-13">After the fall harvest, the top 20 S<sub>1:2</sub> families presenting the highest milk production index based on the MILK2006 prediction were selected. These S<sub>1:2</sub> families were recombined using the bulk entry method, whereby each selected progeny is crossed with each other selected progeny and each cross contributes equally to the next cycle of WQS. In this setup, population improvement and inbred development occurred simultaneously as superior finished (S<sub>6</sub>) lines were identified through the process of selfing, topcrossing, and evaluating.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data</h3><p id="p-14">Starting in 1997, WQS silage yield trials were conducted to select the best 20 S<sub>2</sub> families for advancing the WQS to the next cycle. WQS C0, C1, C2, C3, and C4 were trialed in 1997, 2000, 2003, 2006, and 2010–2011, respectively. All trials were planted at either West Madison Agricultural Research Station (WMARS; Madison, WI) or Arlington Agricultural Research Station (AARS; Arlington, WI) or both. In 1997, a trial of WQS C0 S<sub>2</sub> topcross families was planted at one location, WMARS, with four replications. In 2000, a trial of WQS C1 S<sub>2</sub> topcross families was planted at two locations, WMARS and AARS, with two replications per location. The WQS C2 and C3 S<sub>2</sub> topcross families were evaluated at WMARS and AARS using three replications at each location in 2003 and 2006, respectively. However, the AARS location in 2006 was abandoned because of a severe windstorm that caused extensive lodging. In 2010, a trial of WQS C4 S<sub>2</sub> topcross families was planted at WMARS and AARS with two replications, but data quality from AARS in 2010 was very poor and it was therefore discarded. The WQS C4 trial was replanted at WMARS in 2011 using two replications to provide an additional environment for evaluation. All trials consisted of two-row plots, 6.08 m long and with 0.76 m spacing between rows, arranged in a randomized complete block design. Planting densities were common for silage production in the region. Different testers were used across cycles. Tester LH119 was used in WQS C0, LH198 was used in WQS C1, HC33 was used in WQS C2, and LH244 was used in WQS C3 and C4. All testers used are highly related to B73.</p><p id="p-15">Most recently, nutritional quality is evaluated using MILK2006, a summative equation for calculating milk yield based on factors that affect whole-plant maize silage feed quality, including yield, dry matter (DM) content, neutral detergent fiber (NDF) content, NDF digestibility (NDFD), protein, and starch (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Schwab <em>et al.</em> 2003</a>). Previous versions of this summative equation were utilized in earlier cycles of the WQS selection program. In MILK2006, as well as previous versions, each component (NDF, protein, and starch) is weighted to take into account its respective digestibility. Starch and protein digestibility are traditionally treated as constant, whereas the digestibility of the cell wall, or NDFD, is measured separately for each resulting hybrid. <em>In vitro</em> true digestibility (IVTD), acid detergent fiber (ADF), NDF, crude protein, and starch are predicted using a global near-infrared reflectance calibration developed in-house at the University of Wisconsin (<a href="https://cornbreeding.wisc.edu/nirs">https://cornbreeding.wisc.edu/nirs</a>). Wet chemistry procedures to develop the calibration set are described elsewhere (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Frey <em>et al.</em> 2004</a>; <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> 2010</a>). The summative equation is then used to develop predictions of milk yield described as kg milk yield Mg<sup>−1</sup> DM and kg milk yield ha<sup>−1</sup>.</p></div><div id="sec-4" class="subsection"><h3>Genotyping</h3><p id="p-16">Remnant seed of available S<sub>2</sub> families from WQS C0 to WQS C4 and of S<sub>1</sub> families from WQS C5 were germinated. Immature leaf tissue was collected from 10 individual plants and pooled to represent each of the S<sub>2</sub> (for WQS C0 to WQS C4) and S<sub>1</sub> (in the case of WQS C5) families, respectively, selected at each cycle. Genomic DNA was extracted from each sample using a modified CTAB method (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Saghai-Maroof <em>et al.</em> 1984</a>). Samples were then genotyped using the Illumina MaizeSNP50 BeadChip, an Infinium HD assay (Illumina, Inc. San Diego, CA) with 56,110 SNP markers distributed across the maize genome (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ganal <em>et al.</em> 2011</a>). Alleles for each sample were called using the Genotyping Module within the Illumina Genome-Studio software. The built in GenCall data analysis software, which relies on the GenTrain clustering algorithm, was used for automatic clustering and calling of genotypes (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Oliphant <em>et al.</em> 2002</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fan <em>et al.</em> 2003</a>). To maintain only the highest-quality SNPs, a GenCall threshold of 0.6 was used. This filtering resulted in a dataset of 17,719 high-quality SNPs to be used for further analysis. The mean frequency of missing data was 0.07, with a range of 0 to 0.20. Of these markers, 15,646 were polymorphic, with polymorphic markers being defined as those with minor allele frequencies greater than 0.025. Missing marker scores were imputed using Beagle (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Browning 2009</a>) implemented in the R package Synbreed (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wimmer <em>et al.</em> 2012</a>). Imputation accuracy was defined as the mean posterior probability of the most likely genotypes and calculated using the <em>gprobsmetrics</em> utility in the Beagle package. The average imputation accuracy in this SNP dataset was greater than 99% for all chromosomes.</p></div><div id="sec-5" class="subsection"><h3>Analysis of phenotypic data</h3><p id="p-17">Data from different cycles were kept separate and the initial phenotypic data analysis was performed for each cycle separately. The following mixed linear model was fit to the phenotypic data<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-2.gif"/></noscript></span></span> is the observation of the <em>i</em><sup>th</sup> family evaluated in the <em>j</em><sup>th</sup> environment in the <em>k</em><sup>th</sup> replication; μ is the intercept; <em>g</em><sub>i</sub> is the effect of the <em>i</em><sup>th</sup> family; <em>l</em><sub>j</sub> is the effect of the <em>j</em><sup>th</sup> environment; <em>gl</em><sub>ij</sub> is the interaction between the <em>i</em><sup>th</sup> family and <em>j</em><sup>th</sup> environment; <em>b<sub>k</sub></em><sub>(</sub><em><sub>j</sub></em><sub>)</sub> is the effect of the <em>k</em><sup>th</sup> replicate nested within the <em>j</em><sup>th</sup> environment; and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-3.gif"/></noscript></span></span> is the residual. Environment and replicate effects were modeled as fixed effects. Family and family-by-environment interaction effects were modeled as random effects assumed to be independent and identically distributed. Variance components were estimated using restricted maximum likelihood and best linear unbiased predictions (BLUPs) for each trait were calculated for families. Each cycle was analyzed separately. All calculations were performed using the statistical analysis software ASReml-R (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Butler <em>et al.</em> 2009</a>).</p><p id="p-18">Variance components were used to calculate broad-sense heritability (<em>H</em>) on a family-mean basis as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-5.gif"/></noscript></span></span> is the variance among families, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-6.gif"/></noscript></span></span> is the variance due to family-by-environment interaction effects, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-7.gif"/></noscript></span></span> is the residual variance, <em>e</em> is the number of environments, and <em>r</em> is the number of replications in each environment.</p></div><div id="sec-6" class="subsection"><h3>Genomic heritability</h3><p id="p-19">The proportion of variation among S<sub>2</sub> family BLUPs across cycles explained by the genomic relationship matrix was calculated. The genomic relationship matrix among all families was calculated as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-8.gif"/></noscript></span></span>where <strong>W<sub>C</sub></strong> is the centered genotype matrix, and <em>p<sub>l</sub></em> and <em>q<sub>l</sub></em> are allele frequencies at the <em>l</em><sup>th</sup> locus (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Endelman and Jannink 2012</a>). The following G-BLUP model was fit to the data:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-9.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-10.gif"/></noscript></span></span> is the vector of family BLUPs; <strong>b</strong> is a vector of fixed year effects (corresponding to selection cycle); <strong>u</strong> is a vector of random additive genetic values where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-11.gif"/></noscript></span></span>; <strong>e</strong> is a vector of residuals; and <strong>X</strong> and <strong>Z</strong> are incidence matrices relating <strong>b</strong> and <strong>u</strong> to <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-12.gif"/></noscript></span></span>, respectively. All calculations were made using ASReml-R (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Butler <em>et al.</em> 2009</a>) and the variance components <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-14.gif"/></noscript></span></span> were estimated. Genomic heritability was calculated as <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-15.gif"/></noscript></span></span> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">De Los Campos <em>et al.</em> 2013</a>).</p></div><div id="sec-7" class="subsection"><h3>Association mapping</h3><p id="p-20">A genome-wide association analysis for each trait was performed using the model:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-16.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-17.gif"/></noscript></span></span>, <strong>X</strong>, <strong>b</strong>, <strong>Z</strong>, <strong>u</strong>, and <strong>e</strong> are as above; <strong>m</strong> is a vector of marker effects; and <strong>W</strong> is a matrix comprising marker scores. The association analysis was implemented using EMMA (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kang <em>et al.</em> 2008</a>). A statistical threshold of <em>P</em> = 10<sup>−4</sup> was used to declare significant marker-trait associations. Because a preliminary analysis indicated no markers surpassed this threshold, no further effort was made to better define the statistical threshold to correct for multiple testing.</p></div><div id="sec-8" class="subsection"><h3>Selection mapping and gene diversity</h3><p id="p-21">Allele frequencies in WQS C2 and WQS C5 were calculated based on their maximum likelihood estimate, <em>i.e.</em>, the observed number of copies of the minor allele divided by twice the number of individuals with an observed genotype. WQS C2 was utilized rather than WQS C0 or C1 because samples from the earlier cycles of selection did not include enough individuals for reliable estimates of allele frequencies. SNP-specific <em>F<sub>ST</sub></em> values based on a comparison of C2 and C5 were computed according to <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-18.gif"/></noscript></span></span> where <em>s</em><sup>2</sup> is the sample variance of allele frequency between populations, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-19.gif"/></noscript></span></span> is the mean allele frequency across populations, and <em>r</em> is the number of populations (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Weir and Cockerham 1984</a>).</p><p id="p-22">Significance thresholds were determined via drift simulations of the demographic history of the WQS population, assuming linkage equilibrium between markers. Simulations were conducted within R (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">R Core Team 2014</a>). For each SNP in C2, a simulated allele frequency in C5 was created according to the WQS selection protocol, incorporating generations of selfing, crossing, evaluating, and recombining based on the precise number of individuals utilized at each step in the WQS program. Allele frequencies in C2 and C5 were used to calculate simulated <em>F<sub>ST</sub></em> values for each SNP. The maximum <em>F<sub>ST</sub></em> value observed across SNPs was recorded. This process was repeated 1000 times. The 95% quantile of maximum <em>F<sub>ST</sub></em> values observed from simulations was taken as a simulated 95% significance threshold that accounts for multiple testing across all 17,590 SNPs. The R script used for simulations is provided (Supporting Information, <a href="http://www.g3journal.org/content/suppl/2015/02/02/g3.114.015263.DC1/FileS1.R">File S1</a>).</p><p id="p-23">The above simulations assumed linkage equilibrium. This is a conservative approach because it allows for more independent tests than may truly be appropriate; since SNPs are inherited in linked segments, the true number of independent loci may be lower than the number of SNPs. To explore this possibility, the effective number of markers (<em>M<sub>eff</sub></em>), were computed with the <em>simpleM</em> software (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gao <em>et al.</em> 2008</a>). The above simulation strategy was again used, but with the results of <em>simpleM</em> incorporated. To achieve this, the C2 starting population was simulated by sampling <em>M<sub>eff</sub></em> SNPs, where <em>M<sub>eff</sub></em> was obtained utilizing the parameters PCA_cutoff = 0.99 and 0.95. Ultimately, utilizing the <em>M<sub>eff</sub></em> SNPs instead of the total number of SNPs did not result in a substantive difference in the estimated significance threshold. Therefore, thresholds obtained via linkage equilibrium simulations were utilized throughout this experiment.</p><p id="p-24">We also performed an enrichment analysis to assess if there is an excess of loci displaying a large change in allele frequency. This was achieved by using the previously described simulations to identify the expected 95% and 99% quantiles of <em>F<sub>ST</sub></em> over the course of the experiment. Then, the observed proportion of loci exceeding these quantiles was computed. Theoretically, 5% and 1% of loci will exceed these quantiles, assuming no selection.</p><p id="p-25">Gene diversity (<em>D</em>; <em>i.e.</em>, expected heterozygosity) was estimated for each SNP and for each selection cycle from WQS C2 through C5 using <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-20.gif"/></noscript></span></span><sub>,</sub> where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-21.gif"/></noscript></span></span> is the gene diversity estimate for the <em>l</em><sup>th</sup> SNP in the <em>c</em><sup>th</sup> selection cycle, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-22.gif"/></noscript></span></span> is the allele frequency of that SNP, and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-23.gif"/></noscript></span></span> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Weir 1996</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Association mapping</h3><p id="p-26">A total of 648 individuals from the WQS population were genotyped for this study. Most individuals came from WQS C2 to C5, whereas only 16 individuals were genotyped from WQS C0 and C1 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) because of germination problems most likely a result of seed source age. Both genotype and phenotype data were available for between 240 and 300 families from WQS C1 to C4, depending on the trait.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465942/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465942/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Individuals with genotypic and phenotypic data included in the evaluation of the WQS population</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">Trait means and ranges are displayed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> to provide an overview of the phenotypic data analyzed for the association analysis. The range in silage yield within a given cycle was, on average, 45% of the mean. On the other extreme, the range in IVTD within a given cycle was, on average, 8.5% of the mean. Broad-sense heritabilities on a family-mean basis for each cycle were mostly moderate to high (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). An exception was the <em>H</em> for yield in WQS C2. Broad-sense heritability was generally higher for dry matter and starch, and lower for ADF and NDF. The <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-24.gif"/></noscript></span></span> within cycles was mostly low except for dry matter, CP in WQS C2, and starch WQS C4. The <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/embed/mml-math-25.gif"/></noscript></span></span> across cycles was also low. This indicates that while the genotype accounts for a fair proportion of the phenotypic variation within a given cycle or year of evaluation, little of the variation is captured by an additive relationship matrix.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465819/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465819/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean, minimum, and maximum of each trait in each year (cycle) of evaluation</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465798/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465798/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Broad-sense heritability on a family-mean basis (<em>H</em>) and genomic heritability (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/T3/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/T3/embed/mml-math-26.gif"/></noscript></span></span>) for each trait in each selection cycle of the WQS selection program and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/T3/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/541/T3/embed/mml-math-27.gif"/></noscript></span></span> across cycles</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">Using phenotypic data from historical field trials, an association analysis was performed to identify important genomic regions controlling variation for these yield and compositional traits. Unfortunately, no statistically significant associations (<em>P</em> &lt; 10<sup>−4</sup>) were made for any trait (<a href="http://www.g3journal.org/content/suppl/2015/02/02/g3.114.015263.DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-11" class="subsection"><h3>Selection mapping</h3><p id="p-36">A wide range of <em>F<sub>ST</sub></em> values was observed between SNPs. Since the selection protocols and number of selected individuals at each generation were recorded throughout this experiment, this enabled a simulation-based approach for identifying significance thresholds for the boundaries of <em>F<sub>ST</sub></em> expected to result from drift alone. The 20 S<sub>2</sub> families selected within each cycle led to a strong bottleneck that the population repeatedly experienced, resulting in a high significance threshold. To obtain 95% and 99% probabilities of no false positives, respectively, significance thresholds were set at <em>F<sub>ST</sub></em> = 0.743 and 0.707.</p><p id="p-37">There were no SNPs that exceeded these significance thresholds (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Often, much more lenient outlier thresholds are utilized for selection-mapping experiments. Outlier thresholds involve comparing the observed data with its own empirical distribution, thereby guaranteeing that a specified proportion of the data is identified as candidates for selection. Utilizing a 99% outlier threshold in this study would have meant setting the significance threshold at <em>F<sub>ST</sub></em> = 0.340 and identifying 175 “significant” SNPs. Simulations demonstrated that this significance value is substantially lower than the effects of drift may allow.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/541/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FST values between WQS cycle 2 and WQS cycle 5, computed for each SNP. The dashed green line depicts an empirical 99% outlier threshold. Blue and red lines show simulation-based multiple testing corrected significance thresholds, which control for the magnitude of drift that could reasonably be expected according to the selection protocol that was used." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1561384664" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; values between WQS cycle 2 and WQS cycle 5, computed for each SNP. The dashed green line depicts an empirical 99% outlier threshold. Blue and red lines show simulation-based multiple testing corrected significance thresholds, which control for the magnitude of drift that could reasonably be expected according to the selection protocol that was used.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/541/F2.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/541/F2.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/541/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/541/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465807" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child"><em>F<sub>ST</sub></em> values between WQS cycle 2 and WQS cycle 5, computed for each SNP. The dashed green line depicts an empirical 99% outlier threshold. Blue and red lines show simulation-based multiple testing corrected significance thresholds, which control for the magnitude of drift that could reasonably be expected according to the selection protocol that was used.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Additionally, by evaluating drift simulations without accounting for multiple testing, we determined that it is expected that 5% and 1% of SNPs will exceed <em>F<sub>ST</sub></em> values of 0.214 and 0.328, respectively, due to drift alone. We used these values to assess whether there is enrichment for high-<em>F<sub>ST</sub></em> SNPs in the data. We observed that 6.139% and 1.137% of SNPs exceed these uncorrected thresholds, respectively, indicating there is little evidence of enrichment for SNPs displaying high <em>F<sub>ST</sub></em>.</p></div><div id="sec-12" class="subsection"><h3>Reduction of gene diversity</h3><p id="p-40">Despite no strong signatures of selection and marker-trait associations, an examination of <em>D</em> for each locus shows that reductions were not uniform across the genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). A large reduction in <em>D</em> was observed in regions on chromosomes 2 (∼132 million bp), 3 (∼55 million bp), and 4 (∼78 million bp). These regions of relatively greater loss in diversity were defined visually by examining the <em>D</em> plots in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. Average <em>D</em> across all loci was reduced from 0.352 in C2 to 0.285 in C5. Although average genome-wide <em>D</em> was only reduced by 19% from C2 to C5, average <em>D</em> in these regions on chromosomes 2, 3, and 4 was reduced by 62%, 79%, and 67%, respectively. The large region on chromosome 2, for example, had an average <em>D</em> of 0.355 in C2, which is very close to the average genome-wide <em>D</em> in C2. By C5, however, the average <em>D</em> was only 0.135, which is well below 1 SD of <em>D</em> (genome-wide SD = 0.131).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/541/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene diversity for each SNP evaluated in the Wisconsin Quality Synthetic selection program from cycle 2 (WQS C2) to C5" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1561384664" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Gene diversity for each SNP evaluated in the Wisconsin Quality Synthetic selection program from cycle 2 (WQS C2) to C5&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/541/F3.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/541/F3.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/541/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/541/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465871" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Gene diversity for each SNP evaluated in the Wisconsin Quality Synthetic selection program from cycle 2 (WQS C2) to C5</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-42">The first objective of this study was to identify loci controlling variation for traits important to silage breeding using a combination of association and selection mapping. Despite moderate to high entry-mean heritabilities within cycles (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) and documented genetic gain for silage yield and composition in WQS (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Frey <em>et al.</em> 2004</a>; <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> 2010</a>), no significant results were obtained.</p><p id="p-43">The genetic complexity underlying silage quality and yield is antagonistic to identifying loci contributing to variation and therefore selection response. Although the genetics underlying mechanisms involved in cell wall digestibility could be complex, the dependence of silage quality on grain content, which is related to grain yield, surely makes silage quality increasingly complex. Grain is highly digestible and accounts for approximately 50% of total dry matter of silage (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Coors and Lauer 2001</a>). Also, variation in the effectiveness of the ear as a sink can influence stover composition through its effect on dry matter partitioning and transport of sugars to the ear (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Coors <em>et al.</em> 1997</a>). Stage of plant development at which plants are harvested contributes to variation in silage quality (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jung and Casler 2006</a>). If genetic variation for time to maturity exists within a population, then this variation will be confounded with variation for stover quality. Finally, plant components vary for digestibility and fiber concentrations, and genetic variation exists for digestibility of specific plant components (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hansey <em>et al.</em> 2010</a>). Therefore, the genetic complexity of silage quality on a whole-plant basis could easily equal that of grain yield given its dependence on grain yield and plant morphology.</p><p id="p-44">Because starch content in silage and sink-source dynamics are important contributors to quality, and genotype-by-environment (G×E) interactions are an important source of variation for grain yield, it is not surprising that silage compositional traits are highly influenced by G×E interactions, which has been observed in previous studies (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Argillier <em>et al.</em> 2000</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mechin <em>et al.</em> 2001</a>). This source of variation reduces the contribution of the genetic signal to the total variation, decreasing power to detect marker-trait associations and selection signatures across years. On top of possible strong G×E effects, epistatic interactions could reduce the contribution of main allelic effects, and thus result in a loss of power for making associations. Although comparing variance components and thus heritabilities is fraught with issues because of high standard errors, examination of <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> shows that the proportion of variation accounted for by the additive genomic relationship matrix is low relative to the entry-mean broad-sense heritability in most cases. This suggests the importance of interactions underlying the variability for these traits, both epistatic interactions within cycles as well as G×E interactions across cycles. Another confounded source of variation is allele-by-tester interactions. As noted in <em>Materials and Methods</em>, different testers were used in the different cycles, opening the possibility for tester interaction to dilute the main allelic effects. The testers used were all highly related, being B73 types, and therefore the importance of this source of variation is likely less than if unrelated testers were used.</p><p id="p-45">We demonstrate that although significant genetic gain has been realized for important silage traits within WQS, no strong selection signature was left on the genome. There are at least two reasons for this finding. First, the genetic signal underlying variation for silage yield and composition is highly complex, likely comprising many small main and interaction effects distributed across the entire genome. This hypothesis is supported by the lack of marker-trait associations found in this study. Second, it is possible that selection acted on standing genetic variation caused by old mutations, meaning that a casual polymorphism is not necessarily associated with any particular haplotype. Such soft selective sweeps (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hermisson and Pennings 2005</a>) do not leave a strong selection signature and are difficult to detect using molecular markers.</p><p id="p-46">The lack of a strong selection signature found by this study is in good company among other similar findings on complex traits in agricultural species. <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kemper <em>et al.</em> (2014)</a> found little to no signature on the genome of cattle left by selection for milk yield, despite enormous genetic gain for this trait, and large differences between cattle breeds. Likewise, selection for grain yield in maize has left only very subtle, if any, selection signatures (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gerke <em>et al.</em>, in press</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Van Heerwaarden <em>et al.</em> 2012</a>). Once again, this is despite substantial genetic gain for grain yield accomplished within both a recurrent selection program (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gerke <em>et al.</em>, in press</a>) and commercial breeding (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Van Heerwaarden <em>et al.</em> 2012</a>).</p><p id="p-47">Given that genetic gain has occurred (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Frey <em>et al.</em> 2004</a>; <a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Gustafson <em>et al.</em> 2010</a>), these observations indicate that the gain realized has been accomplished through subtle allele frequency shifts at many loci. It is encouraging to know that breeders are able to simultaneously increase the frequency of many small-effect alleles, therefore achieving genetic gain on highly complex traits. However, great difficulty in figuring out the causal mechanisms underlying genetic gain for complex traits limits our understanding of the genetics underlying selection response. It is clear that new and more powerful methods are required to identify signatures left by selection on highly polygenic traits. Researchers in population genomics have realized this and have begun developing such methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berg and Coop 2014</a>).</p><p id="p-48">Another implication of this study is that caution should be taken when using historical phenotypic data from recurrent selection programs for association mapping of complex traits. While we recognized our power was limited because of only modest population sizes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), we believed, based on the moderate to high <em>H</em>, the trait data from individual cycles was of high-enough quality to detect marker-trait associations. Clearly, that was a wrong assumption. Little of the phenotypic variance across cycles (and thus years) was additive genetic variance, with the majority likely being caused by genetic-by-year interactions given the complexity of the silage compositional traits and their interaction with grain yield. Our experience suggests that historical data are of limited value for association genetics on complex traits prone to genotype-by-year interactions. We recommend that all genotypes be re-evaluated across multiple years and locations to maximize power for detecting associations. It is recognized that the dataset size used herein is relatively small compared with some other historical datasets, and historical data could be useful if vast quantities are available (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Vaughn <em>et al.</em> 2014</a>).</p><p id="p-49">Recurrent selection is a systematic method to increase allele frequency of a base population, and therefore increases the probability a superior inbred line is derived from that base population through selection and inbreeding (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hallauer 1990</a>). Recurrent selection in plant breeding generally involves the selection of multiple individuals or families (typically 10–50) for recombination each cycle. A theoretical advantage of recurrent selection compared with simple inbreeding and selection is that genetic variation is maintained, leading to sustained genetic gain over time (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bernardo 2010</a>). Response to recurrent selection has reportedly continued after many cycles (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dudley and Lambert 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Keeratinijakal and Lamkey 1993</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">De Leon and Coors 2002</a>). Using molecular markers, several studies on maize populations undergoing recurrent selection have shown that observed average losses in gene diversity (<em>i.e.</em>, expected heterozygosity) are approximately equal to that expected by theory assuming genetic drift and a given effective population size (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lamkey and Lorenz 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Labate <em>et al.</em> 1999</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hinze <em>et al.</em> 2005</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Butruille <em>et al.</em> 2004</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Romay <em>et al.</em> 2012</a>). None of the aforementioned studies, however, used marker densities great enough to observe variation in diversity loss across the genome. By genotyping individuals from multiple cycles of selection of the WQS with more than 15,000 high-quality, informative SNPs, we were able to assess the degree to which gene diversity reductions vary across the genome. Very few studies in maize have examined the effects of recurrent selection using high-density SNPs (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Gerke <em>et al.</em>, in press</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2014</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hirsch <em>et al.</em> 2014</a>). Although we observed that most loci followed expectations, a few genomic regions experienced substantial loss of diversity presumably through the combination of chance and the low number of crossovers occurring on each maize chromosome. A similar observation was made by <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Gerke <em>et al.</em>, in press</a>. Using the same Illumina Infinium array, these authors observed that a number of large genomic regions within the BSSS/BSCB1 recurrent selection populations became completely fixed for one haplotype after 16 cycles of selection. Based on the selection procedures used, it was difficult to determine if this was caused by drift or selection. It appears that the regions on chromosomes 2, 3, and 4 are headed for the same fate in the WQS recurrent selection program.</p><p id="p-50">Given the erratic nature of drift in recurrent selection programs with relatively small effective population sizes, combined with the limited number of crossovers occurring on any given maize chromosome each generation, it is entirely possible for a population to become fixed for one haplotype across a large swath of genomic space. This means that while genome-wide diversity in a population may be seemingly satisfactory for continued progress, diversity within specific regions could be inadequate. If these regions harbor loci important for traits of interest, then genetic gain would be compromised and the population would be prevented from reaching its full potential. A major advantage to the routine use of high-density markers in a breeding program would be the ability to monitor genomic variability in allelic diversity and, ultimately, to identify any regions that would benefit from targeted injections of allelic diversity.</p></div><div class="section conclusions" id="sec-14"><h2 class="">Conclusions</h2><p id="p-51">This is the first report of an analysis on genetic gain for silage yield and composition at the genomic level. No strong genomic signatures were left by selection on silage yield and quality in the WQS, likely due to the complexity underlying these traits. The role of selection on standing genetic variation could also be contributing to the lack of strong signatures. Variation in loss of diversity through drift was observed across the genome. A few large regions experienced much greater loss in diversity than what is expected, indicating limited recombination and population sizes in recurrent selection programs could lead to fixation of large swaths of the genome.</p></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">This project is supported by the National Research Initiative or Agriculture and Food Research Initiative Competitive Grants Program grant no. 2012-67013-19460 from the USDA National Institute of Food and Agriculture. T.M.B. was partially supported by a gift to the University of Wisconsin, Madison, Plant Breeding and Plant Genetics program from Monsanto, as well as funding from the University of Wisconsin Graduate School. R.R. was supported by a fellowship from CNPq–Brazil.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-53">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015263/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015263/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-54"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-11-04"><span class="received-label">Received </span>November 4, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-01"><span class="accepted-label">Accepted </span>February 1, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Lorenz <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brigham</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooker</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High-resolution crossover maps for each bivalent of <em>Zea mays</em> using recombination nodules.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnderson%26rft.auinit1%253DL.%2BK.%26rft.volume%253D165%26rft.issue%253D2%26rft.spage%253D849%26rft.epage%253D865%26rft.atitle%253DHigh-Resolution%2BCrossover%2BMaps%2Bfor%2BEach%2BBivalent%2Bof%2BZea%2Bmays%2BUsing%2BRecombination%2BNodules%26rft_id%253Dinfo%253Apmid%252F14573493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzIvODQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.2" data-doi="10.2135/cropsci2000.4061596x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argillier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Méchin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barriere</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inbred line evaluation and breeding for digestibility-related traits in forage maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DArgillier%26rft.auinit1%253DO.%26rft.volume%253D40%26rft.issue%253D6%26rft.spage%253D1596%26rft.epage%253D1600%26rft.atitle%253DInbred%2BLine%2BEvaluation%2Band%2BBreeding%2Bfor%2BDigestibility-Related%2BTraits%2Bin%2BForage%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.4061596x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.4061596x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000165887700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.3" data-doi="10.1038/nrg3015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular spandrels: Tests of adaptation at the genetic level.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D767%26rft.epage%253D780%26rft.atitle%253DMolecular%2Bspandrels%253A%2Btests%2Bof%2Badaptation%2Bat%2Bthe%2Bgenetic%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3015%26rft_id%253Dinfo%253Apmid%252F22005986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22005986&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.4" data-doi="10.1534/genetics.113.160655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide scan for evidence of selection in a maize population under long-term artificial selection for ear number.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160655%26rft_id%253Dinfo%253Apmid%252F24381334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.5" data-doi="10.1371/journal.pgen.1004412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A population genetic signal of polygenic adaptation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004412%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004412%26rft_id%253Dinfo%253Apmid%252F25102153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25102153&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.541.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Breeding for Quantitative Traits in Plants</span>, <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.7" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.541.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml-R Reference Manual</span>, <span class="cit-publ-name">Queensland Dep. of Primary Industries</span>, <span class="cit-publ-loc">Brisbane, QLD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.9" data-doi="10.2135/cropsci2004.1527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butruille</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Response to selection and genetic drift in three populations derived from the Golden Glow maize population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1527</span>–<span class="cit-lpage">1534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DButruille%26rft.auinit1%253DD.%2BV.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D1527%26rft.epage%253D1534%26rft.atitle%253DResponse%2Bto%2BSelection%2Band%2BGenetic%2BDrift%2Bin%2BThree%2BPopulations%2BDerived%2Bfrom%2Bthe%2BGolden%2BGlow%2BMaize%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.1527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.1527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.541.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauer</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Silage corn</span>, pp. <span class="cit-fpage">347</span>–<span class="cit-lpage">392</span> in <span class="cit-source">Specialty Corns</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span> <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.11" data-doi="10.2135/cropsci1997.0011183X003700010043x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bures</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Ear-fill effects on yield and quality of silage corn.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCoors%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D243%26rft.epage%253D247%26rft.atitle%253DEar-Fill%2BEffects%2Bon%2BYield%2Band%2BQuality%2Bof%2BSilage%2BCorn%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1997.0011183X003700010043x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1997.0011183X003700010043x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WE97700043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.12" data-doi="10.1007/s00122-006-0222-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic regions involved in response to grain yield selection at high and low nitrogen fertilization in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCoque%26rft.auinit1%253DM.%26rft.volume%253D112%26rft.issue%253D7%26rft.spage%253D1205%26rft.epage%253D1220%26rft.atitle%253DGenomic%2Bregions%2Binvolved%2Bin%2Bresponse%2Bto%2Bgrain%2Byield%2Bselection%2Bat%2Bhigh%2Band%2Blow%2Bnitrogen%2Bfertilization%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0222-5%26rft_id%253Dinfo%253Apmid%252F16552555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0222-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16552555&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236585600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.13" data-doi="10.2135/cropsci2002.0325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Twenty-four cycles of mass selection for prolificacy in the Golden Glow maize population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253Dde%2BLeon%26rft.auinit1%253DN.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D325%26rft.epage%253D333%26rft.atitle%253DTwenty-Four%2BCycles%2Bof%2BMass%2BSelection%2Bfor%2BProlificacy%2Bin%2Bthe%2BGolden%2BGlow%2BMaize%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.0325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.0325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174093500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.14" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">100 generations of selection for oil and protein in corn.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D24%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.541.16"><div class="cit-metadata unstructured">Endelman, J. B., and J. Jannink, 2012 Shrinkage estimation of the realized relationship matrix. G3 (Bethesda) 2: 1405–1413.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.17" data-doi="10.1007/s00122-006-0443-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falke</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flachenecker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Temporal changes in allele frequencies in two European F2 flint maize populations under modified recurrent full-sib selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFalke%26rft.auinit1%253DK.%2BC.%26rft.volume%253D114%26rft.issue%253D5%26rft.spage%253D765%26rft.epage%253D776%26rft.atitle%253DTemporal%2Bchanges%2Bin%2Ballele%2Bfrequencies%2Bin%2Btwo%2BEuropean%2BF%25282%2529%2Bflint%2Bmaize%2Bpopulations%2Bunder%2Bmodified%2Brecurrent%2Bfull-sib%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0443-7%26rft_id%253Dinfo%253Apmid%252F17322961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0443-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17322961&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.18" data-doi="10.1101/sqb.2003.68.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliphant</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kermani</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Highly parallel SNP genotyping.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DFAN%26rft.auinit1%253DJ.-B.%26rft.volume%253D68%26rft.issue%253D0%26rft.spage%253D69%26rft.epage%253D78%26rft.atitle%253DHighly%2BParallel%2BSNP%2BGenotyping%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2003.68.69%26rft_id%253Dinfo%253Apmid%252F15338605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjc6IjY4LzAvNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.19" data-doi="10.2135/cropsci2004.1200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaver</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eilert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selection for silage quality in the Wisconsin quality synthetic and related maize populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1200</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFrey%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D1200%26rft.epage%253D1208%26rft.atitle%253DSelection%2Bfor%2BSilage%2BQuality%2Bin%2Bthe%2BWisconsin%2BQuality%2BSynthetic%2Band%2BRelated%2BMaize%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.1200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.1200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000222582400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.20" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (zea mays L.) SNP genotyping array: Development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.21" data-doi="10.1002/gepi.20310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starmer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DGao%26rft.auinit1%253DX.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D361%26rft.epage%253D369%26rft.atitle%253DA%2Bmultiple%2Btesting%2Bcorrection%2Bmethod%2Bfor%2Bgenetic%2Bassociation%2Bstudies%2Busing%2Bcorrelated%2Bsingle%2Bnucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20310%26rft_id%253Dinfo%253Apmid%252F18271029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18271029&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255471100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guill</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-article-title">The genomic impacts of drift and selection for hybrid performance in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.23" data-doi="10.2135/cropsci2009.12.0725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustafson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evaluation of S2-topcross selection for maize (<em>Zea mays</em> L.) silage yield and quality in the Wisconsin Quality Synthetic population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1795%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.12.0725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.12.0725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Methods used in developing maize inbreds.</span> <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D35%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1990CW67800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.25" data-doi="10.1007/s12155-010-9100-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cell wall composition and ruminant digestibility of various maize tissues across development.</span> <abbr class="cit-jnl-abbrev">BioEnergy Research</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BResearch%26rft.volume%253D3%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-010-9100-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-010-9100-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.26" data-doi="10.1534/genetics.104.036947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Soft sweeps: Molecular population genetics of adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036947%26rft_id%253Dinfo%253Apmid%252F15716498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIzMzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.27" data-doi="10.2135/cropsci2004.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinze</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Population genetic diversity in a maize reciprocal recurrent selection program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">2435</span>–<span class="cit-lpage">2442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHinze%26rft.auinit1%253DL.%2BL.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D2435%26rft.epage%253D2442%26rft.atitle%253DPopulation%2BGenetic%2BDiversity%2Bin%2Ba%2BMaize%2BReciprocal%2BRecurrent%2BSelection%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233147100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.28" data-doi="10.1534/genetics.114.167155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into the effects of long-term artificial selection on seed size in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167155%26rft_id%253Dinfo%253Apmid%252F25037958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvNDA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.29" data-doi="10.1038/ng.2309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative population genomics of maize domestication and improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHufford%26rft.auinit1%253DM.%2BB.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D808%26rft.epage%253D811%26rft.atitle%253DComparative%2Bpopulation%2Bgenomics%2Bof%2Bmaize%2Bdomestication%2Band%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2309%26rft_id%253Dinfo%253Apmid%252F22660546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660546&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.30" data-doi="10.2135/cropsci2006.02-0086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Maize stem tissues: Impact of development on cell wall degradability.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DJung%26rft.auinit1%253DH.%2BG.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1801%26rft.epage%253D1809%26rft.atitle%253DMaize%2BStem%2BTissues%253A%2BImpact%2Bof%2BDevelopment%2Bon%2BCell%2BWall%2BDegradability%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.02-0086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.02-0086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.31" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.32" data-doi="10.2135/cropsci1993.0011183X003300010013x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keeratinijakal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Genetic effects associated with reciprocal recurrent selection in BSSS and BSCB1 maize populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DKeeratinijakal%26rft.auinit1%253DV.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D82%26rft.atitle%253DGenetic%2BEffects%2BAssociated%2Bwith%2BReciprocal%2BRecurrent%2BSelection%2Bin%2BBSSS%2Band%2BBSCB1%2BMaize%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1993.0011183X003300010013x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1993.0011183X003300010013x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993KT47500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.33" data-doi="10.1186/1471-2164-15-246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxton</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolormaa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selection for complex traits leaves little or no classic signatures of selection.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-246%26rft_id%253Dinfo%253Apmid%252F24678841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24678841&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.541.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labate</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Population genetics of increased hybrid performance between two maize populations under reciprocal recurrent selection</span>, pp. <span class="cit-fpage">127</span>–<span class="cit-lpage">137</span> in <span class="cit-source">The Genetics and Exploitation of Heterosis in Crops</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J. G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">ASA-CSSA-SSSA</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.35" data-doi="10.2135/cropsci2013.07.0484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Relative effect of drift and selection in diverging populations within a reciprocal recurrent selection program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D576%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.07.0484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.07.0484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.37" data-doi="10.2135/cropsci2009.02.0086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustafson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">N. d.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Breeding maize for a bioeconomy: A literature survey examining harvest index and stover yield and their relationship to grain yield.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.02.0086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.02.0086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000273842000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.38" data-doi="10.2135/cropsci2001.413690x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Méchin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argillier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hébert</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guingo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic analysis and QTL mapping of cell wall digestibility and lignification in silage maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">690</span>–<span class="cit-lpage">697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMechin%26rft.auinit1%253DV.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D690%26rft.epage%253D697%26rft.atitle%253DGenetic%2BAnalysis%2Band%2BQTL%2BMapping%2Bof%2BCell%2BWall%2BDigestibility%2Band%2BLignification%2Bin%2BSilage%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2001.413690x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2001.413690x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170696200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.39" data-doi="10.1105/tpc.109.068437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association mapping: Critical considerations shift from genotyping to experimental design.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2194</span>–<span class="cit-lpage">2202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.068437%26rft_id%253Dinfo%253Apmid%252F19654263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMjE5NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.541.40"><div class="cit-metadata unstructured">National Agricultural Statistics Service, 2014 National Statistics for Corn. Available at: <a href="http://www.nass.usda.gov/Statistics_by_Subject/result.php?08638011%E2%80%93B478%E2%80%933942%E2%80%93B44F%E2%80%93FA130ADDE283%E2%80%93sector=CROPS&amp;group=FIELD%20CROPS&amp;comm=CORN">http://www.nass.usda.gov/Statistics_by_Subject/result.php?08638011–B478–3942–B44F–FA130ADDE283&amp;sector=CROPS&amp;group=FIELD%20CROPS&amp;comm=CORN</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliphant</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuelpnagel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BeadArray (TM) technology: Enabling an accurate, cost-effective approach to high throughput genotyping.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DWitt%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D56%26rft.epage%253D56%252C%2B60%26rft.atitle%253DRescue%2Bof%2Bcompleted%2Bbut%2Bunread%2BHIV-1%2Bantibody%2BELISA%2Bmicroplates%2Bby%2Bfreezing.%26rft_id%253Dinfo%253Apmid%252F11808699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11808699&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173286200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.42" data-doi="10.1101/gr.116731.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revealing the genetic structure of a trait by sequencing a population under selection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.116731.110%26rft_id%253Dinfo%253Apmid%252F21422276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.541.43"><div class="cit-metadata unstructured">R Core Team, 2014 R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.44" data-doi="10.1534/genetics.113.159731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Recovering power in association mapping panels with variable levels of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159731%26rft_id%253Dinfo%253Apmid%252F24532779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.45" data-doi="10.2135/cropsci2011.10.0552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butrón</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ordás</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Revilla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ordás</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effect of recurrent selection on the genetic structure of two broad-based Spanish maize populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1493%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.10.0552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.10.0552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.46" data-doi="10.1073/pnas.81.24.8014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saghaimaroof</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soliman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allard</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Ribosomal DNA spacer-length polymorphisms in barley–Mendelian inheritance, chromosomal location, and population-dynamics.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">8014</span>–<span class="cit-lpage">8018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSaghai-Maroof%26rft.auinit1%253DM.%2BA.%26rft.volume%253D81%26rft.issue%253D24%26rft.spage%253D8014%26rft.epage%253D8018%26rft.atitle%253DRibosomal%2BDNA%2Bspacer-length%2Bpolymorphisms%2Bin%2Bbarley%253A%2Bmendelian%2Binheritance%252C%2Bchromosomal%2Blocation%252C%2Band%2Bpopulation%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.24.8014%26rft_id%253Dinfo%253Apmid%252F6096873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODEvMjQvODAxNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.47" data-doi="10.1016/S0377-8401(03)00210-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaver</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauer</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coors</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating silage energy value and milk yield to rank corn hybrids.</span> <abbr class="cit-jnl-abbrev">Anim. Feed Sci. Technol.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BFeed%2BSci.%2BTechnol.%26rft.volume%253D109%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0377-8401%252803%252900210-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0377-8401(03)00210-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.541.48"><div class="cit-metadata unstructured">Shaver, R., J. Lauer, J. Coors, and P. Hoffman, 2006 Corn silage evaluation: MILK2000 challenges and opportunities with MILK2006. Available at: <a href="http://www.uwex.edu">http://www.uwex.edu</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.49" data-doi="10.1371/journal.pgen.1001336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarone</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population-based resequencing of experimentally evolved populations reveals the genetic basis of body size variation in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001336%26rft.epage%253De1001336%26rft.atitle%253DPopulation-based%2Bresequencing%2Bof%2Bexperimentally%2Bevolved%2Bpopulations%2Breveals%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bbody%2Bsize%2Bvariation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001336%26rft_id%253Dinfo%253Apmid%252F21437274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437274&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.50" data-doi="10.1073/pnas.1209275109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Historical genomics of North American maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">12420</span>–<span class="cit-lpage">12425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1209275109%26rft_id%253Dinfo%253Apmid%252F22802642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMxLzEyNDIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genetic architecture of seed composition in soybean is refined by genome-wide association scans across multiple populations G3</span>: <span class="cit-vol">4</span>:<span class="cit-fpage">2283</span>–<span class="cit-lpage">2294</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.541.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Genetic Data Analysis II</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.53" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.54" data-doi="10.1093/bioinformatics/bts335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Synbreed: A framework for the analysis of genomic prediction data using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts335%26rft_id%253Dinfo%253Apmid%252F22689388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTUvMjA4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.55" data-doi="10.1017/S1479262111000359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel genetic framework for studying response to artificial selection.</span> <abbr class="cit-jnl-abbrev">Plant Genetic Resources</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenetic%2BResources%26rft.volume%253D9%26rft.spage%253D281%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1479262111000359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1479262111000359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.56" data-doi="10.1534/genetics.108.090118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolkman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection mapping of loci for quantitative disease resistance in a diverse maize population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090118%26rft_id%253Dinfo%253Apmid%252F18723892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNTgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.57" data-doi="10.1126/science.1107891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The effects of artificial selection on the maize genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">1310</span>–<span class="cit-lpage">1314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWright%26rft.auinit1%253DS.%2BI.%26rft.volume%253D308%26rft.issue%253D5726%26rft.spage%253D1310%26rft.epage%253D1314%26rft.atitle%253DThe%2BEffects%2Bof%2BArtificial%2BSelection%2Bon%2Bthe%2BMaize%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1107891%26rft_id%253Dinfo%253Apmid%252F15919994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDgvNTcyNi8xMzEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.541.58" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F541.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/541.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/531" title="Heritable Variation in Courtship Patterns in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/551" title="Comparison of Larval and Adult Drosophila Astrocytes Reveals Stage-Specific Gene Expression Profiles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/541" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-E60n42kOvnScLJd6tzrHqaQpF4EIgzbkgL9pFEPB3ZY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/541" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/541" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/541" class="active" data-icon-position="" data-hide-link-title="0">Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459802" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-l5qR9d7-HNr7JoUGWaTqN_-gcMdDSlSCcm8YsYG5fEc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/541.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/541" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wI67PIuS0Mk9HLwp00qehCysV7wZesg60H7GSG6eFAM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/541" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/541.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459802" id="node459802--2" data-pisa="ggg;5/4/541" data-pisa-master="ggg;g3.114.015263" data-apath="/ggg/5/4/541.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron J. Lorenz</span>, <span class="highwire-citation-author" data-delta="1">Timothy M. Beissinger</span>, <span class="highwire-citation-author" data-delta="2">Renato Rodrigues Silva</span> and <span class="highwire-citation-author" data-delta="3">Natalia de Leon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">541-549;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015263 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459802--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alorenz2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">alorenz2@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy M. Beissinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BM.%2BBeissinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beissinger%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BM.%2BBeissinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Renato Rodrigues Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renato%2BRodrigues%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Silva%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARenato%2BRodrigues%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Natalia de Leon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2Bde%2BLeon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Leon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2Bde%2BLeon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/541" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/541&amp;title=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/541&amp;title=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/541&amp;title=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/541&amp;text=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/541&amp;title=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/541&amp;t=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/541&amp;title=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/541&amp;title=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459802/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459802" id="node459802--3" data-pisa="ggg;5/4/541" data-pisa-master="ggg;g3.114.015263" data-apath="/ggg/5/4/541.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection for Silage Yield and Composition Did Not Affect Genomic Diversity Within the Wisconsin Quality Synthetic Maize Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron J. Lorenz</span>, <span class="highwire-citation-author" data-delta="1">Timothy M. Beissinger</span>, <span class="highwire-citation-author" data-delta="2">Renato Rodrigues Silva</span> and <span class="highwire-citation-author" data-delta="3">Natalia de Leon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">541-549;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015263 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459802--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alorenz2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">alorenz2@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy M. Beissinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BM.%2BBeissinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beissinger%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BM.%2BBeissinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Renato Rodrigues Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renato%2BRodrigues%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Silva%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARenato%2BRodrigues%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Natalia de Leon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2Bde%2BLeon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Leon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2Bde%2BLeon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=541&amp;rft.epage=549&amp;rft.atitle=Selection%20for%20Silage%20Yield%20and%20Composition%20Did%20Not%20Affect%20Genomic%20Diversity%20Within%20the%20Wisconsin%20Quality%20Synthetic%20Maize%20Population&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lorenz%2C+Aaron+J.&amp;rft.au=Beissinger%2C+Timothy+M.&amp;rft.au=Silva%2C+Renato+Rodrigues&amp;rft.au=de+Leon%2C+Natalia"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459802/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459802/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459802/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459802/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459802/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459802/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459802/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459802/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459802/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459802/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459802/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459802/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/541" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/541.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/541.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/541.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/541.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
