<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/569" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/569.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/569.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/569.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/4/569/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/569.atom" />
<meta name="HW.pisa" content="ggg;5/4/569" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model" />
<meta name="DC.Identifier" content="10.1534/g3.114.016097" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Lopez-Cruz et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License(http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) models use genome-wide genetic information to predict genetic values of candidates of selection. Originally, these models were developed without considering genotype × environment interaction(G×E). Several authors have proposed extensions of the single-environment GS model that accommodate G×E using either covariance functions or environmental covariates. In this study, we model G×E using a marker × environment interaction (M×E) GS model; the approach is conceptually simple and can be implemented with existing GS software. We discuss how the model can be implemented by using an explicit regression of phenotypes on markers or using co-variance structures (a genomic best linear unbiased prediction-type model). We used the M×E model to analyze three CIMMYT wheat data sets (W1, W2, and W3), where more than 1000 lines were genotyped using genotyping-by-sequencing and evaluated at CIMMYT’s research station in Ciudad Obregon, Mexico, under simulated environmental conditions that covered different irrigation levels, sowing dates and planting systems. We compared the M×E model with a stratified ( i.e. , within-environment) analysis and with a standard (across-environment) GS model that assumes that effects are constant across environments ( i.e. , ignoring G×E). The prediction accuracy of the M×E model was substantially greater of that of an across-environment analysis that ignores G×E. Depending on the prediction problem, the M×E model had either similar or greater levels of prediction accuracy than the stratified analyses. The M×E model decomposes marker effects and genomic values into components that are stable across environments (main effects) and others that are environment-specific (interactions). Therefore, in principle, the interaction model could shed light over which variants have effects that are stable across environments and which ones are responsible for G×E. The data set and the scripts required to reproduce the analysis are publicly available as Supporting Information." />
<meta name="DC.Contributor" content="Marco Lopez-Cruz" />
<meta name="DC.Contributor" content="Jose Crossa" />
<meta name="DC.Contributor" content="David Bonnett" />
<meta name="DC.Contributor" content="Susanne Dreisigacker" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Ravi P. Singh" />
<meta name="DC.Contributor" content="Enrique Autrique" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) models use genome-wide genetic information to predict genetic values of candidates of selection. Originally, these models were developed without considering genotype × environment interaction(G×E). Several authors have proposed extensions of the single-environment GS model that accommodate G×E using either covariance functions or environmental covariates. In this study, we model G×E using a marker × environment interaction (M×E) GS model; the approach is conceptually simple and can be implemented with existing GS software. We discuss how the model can be implemented by using an explicit regression of phenotypes on markers or using co-variance structures (a genomic best linear unbiased prediction-type model). We used the M×E model to analyze three CIMMYT wheat data sets (W1, W2, and W3), where more than 1000 lines were genotyped using genotyping-by-sequencing and evaluated at CIMMYT’s research station in Ciudad Obregon, Mexico, under simulated environmental conditions that covered different irrigation levels, sowing dates and planting systems. We compared the M×E model with a stratified (&lt;i&gt;i.e.&lt;/i&gt;, within-environment) analysis and with a standard (across-environment) GS model that assumes that effects are constant across environments (&lt;i&gt;i.e.&lt;/i&gt;, ignoring G×E). The prediction accuracy of the M×E model was substantially greater of that of an across-environment analysis that ignores G×E. Depending on the prediction problem, the M×E model had either similar or greater levels of prediction accuracy than the stratified analyses. The M×E model decomposes marker effects and genomic values into components that are stable across environments (main effects) and others that are environment-specific (interactions). Therefore, in principle, the interaction model could shed light over which variants have effects that are stable across environments and which ones are responsible for G×E. The data set and the scripts required to reproduce the analysis are publicly available as Supporting Information.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/569" />
<meta name="citation_id" content="5/4/569" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/569" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/569.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/569.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/569.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016097" />
<meta name="citation_pmid" content="25660166" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="569" />
<meta name="citation_lastpage" content="582" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marco Lopez-Cruz" />
<meta name="citation_author_institution" content="Department of Plant, Soil and Microbial Sciences, Michigan State University (MSU), East Lansing, Michigan 4882," />
<meta name="citation_author" content="Jose Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico" />
<meta name="citation_author" content="David Bonnett" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico" />
<meta name="citation_author" content="Susanne Dreisigacker" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University (KSU), 4011 Throckmorton Hall, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="USDA-ARS and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853, and" />
<meta name="citation_author" content="Ravi P. Singh" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico" />
<meta name="citation_author" content="Enrique Autrique" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Epidemiology &amp; Biostatistics and Statistics departments, Michigan State University 909 Fee Road, East Lansing, Michigan 48824" />
<meta name="citation_author_email" content="gcampos@uab.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Boer;citation_author=D. Wright;citation_author=L. Feng;citation_author=D. W. Podlich;citation_author=L. Luo;citation_title=A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize.;citation_volume=177;citation_year=2007;citation_pmid=17947443;citation_doi=10.1534/genetics.107.071068" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=H. J. Braun;citation_author=S. Rajaram;citation_author=M. van Ginkel;citation_title=CIMMYT’s approach to breeding for wide adaptation.;citation_pages=175-183;citation_volume=92;citation_year=1996;citation_doi=10.1007/BF00022843" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=P. L. Cornelius;citation_author=R. Trethowan;citation_author=G. McLaren;citation_title=Modeling Additive x Environment and Additive x Additive x Environment Using Genetic Covariances of Relatives of Wheat Genotypes;citation_pages=311-320;citation_volume=47;citation_year=2007;citation_issue=1;citation_doi=10.2135/cropsci2006.09.0564" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Crossa;citation_author=J. Burgueño;citation_author=P. L. Cornelius;citation_author=R. Trethowan;citation_author=A. Krishnamachari;citation_title=Modeling Genotype x Environment Interaction Using Additive Genetic Covariances of Relatives for Predicting Breeding Values of Wheat Genotypes;citation_pages=1722-1733;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2005.11-0427" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. C. Dawson;citation_author=J. B. Endelman;citation_author=N. Heslot;citation_author=J. Crossa;citation_author=J. Poland;citation_title=The use of unbalanced historical data for genomic selection in an international wheat breeding program.;citation_pages=12-22;citation_volume=154;citation_year=2013;citation_doi=10.1016/j.fcr.2013.07.020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. A. Eberhart;citation_author=W. A. Russell;citation_title=Stability Parameters for Comparing Varieties;citation_pages=36-40;citation_volume=6;citation_year=1966;citation_issue=1;citation_doi=10.2135/cropsci1966.0011183X000600010011x" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=K. W. Finlay;citation_author=G. N. Wilkinson;citation_title=The analysis of adaptation in a plant breeding program.;citation_pages=742-754;citation_volume=14;citation_year=1963;citation_doi=10.1071/AR9630742" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquin;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Malosetti;citation_author=J. Voltas;citation_author=I. Romagosa;citation_author=S. E. Ullrich;citation_author=F. A. van Eeuwijk;citation_title=Mixed models including environmental covariables for studying QTL by environment interaction.;citation_pages=139-145;citation_volume=137;citation_year=2004;citation_doi=10.1023/B:EUPH.0000040511.46388.ef" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Malosetti;citation_author=J. M. Ribaut;citation_author=M. Vargas;citation_author=J. Crossa;citation_author=F. A. van Eeuwijk;citation_title=A multi-trait multi-environment QTL mixed model with an application to drought and nitrogen stress trials in maize (Zea mays L.).;citation_pages=241-257;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9594-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Meyer;citation_author=M. Kirkpatrick;citation_title=Perils of parsimony: properties of reduced-rank estimates of genetic covariance matrix.;citation_volume=180;citation_year=2008;citation_pmid=18757923;citation_doi=10.1534/genetics.108.090159" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. Moreau;citation_author=A. Charcosset;citation_author=A. Gallais;citation_title=Use of trial clustering to study QTL x environment effects for grain yield and related traits in maize.;citation_pages=92-105;citation_volume=110;citation_year=2004;citation_issue=1;citation_pmid=15551040;citation_doi=10.1007/s00122-004-1781-y" />
<meta name="citation_reference" content="citation_journal_title=G3(Bethesda);citation_author=P. Pérez-Rodriguez;citation_author=D. Gianola;citation_author=J. M. Gonzalez-Camacho;citation_author=J. Crossa;citation_author=Y. Manes;citation_title=A comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.;citation_pages=1595-1605;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=H. P. Piepho;citation_title=Analyzing genotype-environment data by mixed models with multiplicative effects.;citation_pages=761-766;citation_volume=53;citation_year=1997;citation_doi=10.2307/2533976" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. P. Piepho;citation_title=Empirical best linear unbiased prediction in cultivar trials using factor analytic variance covariance structure.;citation_pages=195-201;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050885" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Ruthkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. B. Smith;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_title=The analysis of crop cultivar breeding and evaluation trials: an overview of current mixed model approaches.;citation_pages=449-462;citation_volume=143;citation_year=2005;citation_doi=10.1017/S0021859605005587" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=F. A. van Eeuwijk;citation_author=M. Malosetti;citation_author=X. Yin;citation_author=P. C. Struik;citation_author=P. Stam;citation_title=Statistical models for genotype by environment data: from conventional ANOVA models to eco-physiological QTL models.;citation_pages=883-894;citation_volume=56;citation_year=2005;citation_doi=10.1071/AR05153" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=F. Yates;citation_author=W. G. Cochran;citation_title=The analysis of groups of experiments.;citation_pages=556-580;citation_volume=28;citation_year=1938;citation_doi=10.1017/S0021859600050978" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) models use genome-wide genetic information to predict genetic values of candidates of selection. Originally, these models were developed without considering genotype × environment interaction(G×E). Several authors have proposed extensions of the single-environment GS model that accommodate G×E using either covariance functions or environmental covariates. In this study, we model G×E using a marker × environment interaction (M×E) GS model; the approach is conceptually simple and can be implemented with existing GS software. We discuss how the model can be implemented by using an explicit regression of phenotypes on markers or using co-variance structures (a genomic best linear unbiased prediction-type model). We used the M×E model to analyze three CIMMYT wheat data sets (W1, W2, and W3), where more than 1000 lines were genotyped using genotyping-by-sequencing and evaluated at CIMMYT’s research station in Ciudad Obregon, Mexico, under simulated environmental conditions that covered different irrigation levels, sowing dates and planting systems. We compared the M×E model with a stratified ( i.e. , within-environment) analysis and with a standard (across-environment) GS model that assumes that effects are constant across environments ( i.e. , ignoring G×E). The prediction accuracy of the M×E model was substantially greater of that of an across-environment analysis that ignores G×E. Depending on the prediction problem, the M×E model had either similar or greater levels of prediction accuracy than the stratified analyses. The M×E model decomposes marker effects and genomic values into components that are stable across environments (main effects) and others that are environment-specific (interactions). Therefore, in principle, the interaction model could shed light over which variants have effects that are stable across environments and which ones are responsible for G×E. The data set and the scripts required to reproduce the analysis are publicly available as Supporting Information." />
<meta name="og-title" property="og:title" content="Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/569" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) models use genome-wide genetic information to predict genetic values of candidates of selection. Originally, these models were developed without considering genotype × environment interaction(G×E). Several authors have proposed extensions of the single-environment GS model that accommodate G×E using either covariance functions or environmental covariates. In this study, we model G×E using a marker × environment interaction (M×E) GS model; the approach is conceptually simple and can be implemented with existing GS software. We discuss how the model can be implemented by using an explicit regression of phenotypes on markers or using co-variance structures (a genomic best linear unbiased prediction-type model). We used the M×E model to analyze three CIMMYT wheat data sets (W1, W2, and W3), where more than 1000 lines were genotyped using genotyping-by-sequencing and evaluated at CIMMYT’s research station in Ciudad Obregon, Mexico, under simulated environmental conditions that covered different irrigation levels, sowing dates and planting systems. We compared the M×E model with a stratified ( i.e. , within-environment) analysis and with a standard (across-environment) GS model that assumes that effects are constant across environments ( i.e. , ignoring G×E). The prediction accuracy of the M×E model was substantially greater of that of an across-environment analysis that ignores G×E. Depending on the prediction problem, the M×E model had either similar or greater levels of prediction accuracy than the stratified analyses. The M×E model decomposes marker effects and genomic values into components that are stable across environments (main effects) and others that are environment-specific (interactions). Therefore, in principle, the interaction model could shed light over which variants have effects that are stable across environments and which ones are responsible for G×E. The data set and the scripts required to reproduce the analysis are publicly available as Supporting Information." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ORTodm9TXJsYtV5POlxI8TWk4fuZFNh0ZDMctrx1_yQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459804","apath":"\/ggg\/5\/4\/569.atom","pisa":"ggg;5\/4\/569","ac":{"\/ggg\/5\/4\/569.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/569.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/569"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/569"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/569.supplemental":true,"\/content\/5\/4\/569":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HnHj7V+BaSTXznSAmGPBFyMfx8rmvyyw4cAWgTmifaT+U6eIXeED8eC1lMnilX27pLLV31lGVDldt27VnN0dgC4UX1jKVn3oO9MMVU1OeY12nh2pWIFJL72ZkYe9YGCzKv7rq0ddctEZ4UpYsxGIGGJeUMaPpzTVYAi5Tjs24I9tkEiBKuRX5SiqoIPGv0GrikpvmRlPm\\\\\\\/CNMQT6kppVUBzi1PtSRbGjeNKP4civE38n4bMVA73q4SX0h6C0QQUTJ\\\\\\\/2Kkd3SwlCHWCX8qXHC5GkVmh1ZcMwuk\\\\\\\/XKNn4aKDWpN2wrYrX+FFi62bfmsMaOAbRItGTMoZaXdHs6NelYyX1xyfyc7zCkGmtY0WZcPjTAvoNTeWj92vfCFCY23ptFOYkrj+jwvQ4YprlxLIMBEDvtsNZDBTg8eglClSeBnMMyE+9bVl1ux7VDUVhL3wB2YB8qCrS0yJQJgXRfLzOeUqUt6RMCVo0IEKNFWmlgk2XsvqsMpLoAZs3w6ORiIfmir3Q0I7TGiOmdOkVh1cW56G7ECrtA8O4fzXJSmWn+ieUgtMmeTLpMn7v4SSlOH41R29hPJjC6qffRKbsd21fq24LiriO5qBxGpDGWuXu+950FhPAQ93AUXAgM4gkq6+ZrGGn2lVOsKdsCOi5N4yRsFiXtHAPQ0JuIvkQGGUh4QzkalwwN6GgjMNJmNlpTuDAMbGaMkd\\\\\\\/YxCxdi8Wxn9DwYKNGKoFf0iU3kuvrQvMBrSePpDBnjsH5P4A9ebWR1iAKxG6vwqpozBTvAzicmXDLwc0gyYLSAui8NBLUt+8\\\\\\\/5HPWfLwqvE+XMxmR0KVlnQkBvDrSVdD3ixceql+0p4JXJNxIqUq6AT51dmYSGTTu6iBR9J2WumNA9koHxsYJps1hYYsx3TLrxeZvgG8SKF8njHpzBvfzlXz3HjRhMJrcuBUbhd66w4vHotxxbI98JN3jJP\\\\\\\/lbDRaOyhkGdWy3YVDl0fWEMRYGNh89u53L8iuJIvdPAIvHJMkwFZywz\\\\\\\/l2wvXCP86tvAbp4+vGAmSBIIcm\\\\\\\/tWNh9OMWL7uOAuh5xr4KW36xa2\\\\\\\/YZzsjJFM7b0XKY9Fchvb528hyNWe3igS\\\\\\\/NL5EGEWbR6ucO8JwdVPYACEbIiOD07V\\\\\\\/efUx07UnkViE\\\\\\\/0mskGExq5wuZyCE\\\\\\\/oiiDGaaFMecD32TspmCt6ZOQypnY9WBDQB\\\\\\\/0omNj0kFqGtisOtLhEuR4spz7ps2p0siGZVW3gFK4ITQUQzUhT3TDKRmaZrP6wcFSPrrCn\\\\\\\/k8DfpfH\\\u0022,\\\u0022iv\\\u0022:\\\u0022w\\\\\\\/jhEi2x3Oq2AnK6nV7qHw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e87eb188300bd6070ed724910d5396794d6f859f0360fd7389d6e49f3ae49507\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022b6GbzR0rIvk0TRs8pEp4taiRyFgSFG+T7ThdwdCH9wgNXsZl\\\\\\\/BS64zg6hJNu3sDdLwJev1UozKXlG3ZL2FRD8Xlhr5XMDc0a6WtWNLWAK7eKyinhn0JKjJEHA8PirZsd8rMmiJNwSzC2v+J8It4DgbQfS2eAGrZYh4IMIHUuLakStLmiVhrul2cgBYPn+rv5mD7abSV47kKazBenc+d3brv4iJGudKj7fTdUTzn98Lm5ut2HyvzioBIc2iM2dQ4rwO3MMoaku4n9mxTOV37YJ7Ll0TOo+11LiexEjBmIgozjLG3ZmoT9NRZTGMq0k9VIRoMxqCYvq3xmL7puZ1q\\\\\\\/NN24l9pycKyptJlejN9jVP25l2ofijDemXPRDMizo8NIdc7DIKhN6ggONFgYxlqLa2lGaHJBAC+5ifB9sxEt\\\\\\\/ZI3efgFWybjWCz8YGMksaUN4yXfrVDNNw4O0jwhGKH7Ic8nHqlMJio1bppbcZHG5rN8AIg4dkWNFMNIuGIPHybQVkzc7y3LzSnHvvMGqh0MELiWTc6wro\\\\\\\/fBNy\\\\\\\/B1GeIs7rzfHLtvyqdQ\\\\\\\/ys4xoo\\\\\\\/qIR1O4TR7F8ea+yxHXHVJRHylhs8pf+0vu2p4CHqYQQtYiaXA+8y3vB\\\\\\\/Rgs7OUfbPYyD0NDgDxgHq5zIWvKj9YO6kioGfPAD3ABwHuCgdyeun\\\\\\\/veQ5xwYumXBvy\\\\\\\/eHu906LrZENYGcxZrguZWxE4V05W5xauSSfY+Py7Jt7v1aXsSVz6wwjObshXx0jxJea6AGzPDa6XJk8m0A0BgohxI92VoscI\\\\\\\/blQDwtIcOlbFe+cPp0cw1YWmAZRqbv6RK3u7bZsB5sYCiNRbUwwwuCvTHh1bt9lBBkv3NuJpmSQxFpKNyl1ZitSROvyZad65XeBdUcEJVjOkD+G+FVNiKAjatvCDETX+dj17lD0IiZIwD1IUcM0pN58xe01\\\\\\\/g2wblExF86pmMlC9+9KAKzjQObiwAkelkVQ2K+1lE43mOPIY982gHLwtTDmKp8j3qwwMqhz4l9ieMrEUA8JXtqGnHwWlilyCKoQ809XdnAJPUUCktlmto6JPzJ+FHmoPhXz7F2OqpUzn\\\\\\\/i8dsIV7SekBWnoMtXc7KzoyGOEJNorMddXJyGbwPxyKMX2m6GmLl9\\\\\\\/aFOrF8nOJR9j6+e0\\\\\\\/TaY9NxoCLrKicUlXFa2blX6+CzZNMY+uwnTkJBQsHiob5LlxCat\\\\\\\/Kjv2D3W8+hmtjbo3v3T+mpqfCc+igZ0\\\\\\\/7wTLppFtV9Rd1XlqwBCnuAOYI8Gq9+FMTxpV298YqbffmWH4ZA++EXNqjTH+yoqrydTDiQjsbZRo8vx9eiYvaZLXVCQ1x68nS1aG9n3vcpY+ZAdnMlzUwj\\\\\\\/NNbwPhvU7ZAY7O9wsDx21wdVEs\\\\\\\/lS8dE0QeqS3cNVxx8Z7RDrFvKN0f+uTyFcoCMZ0U7Fh5nlwyd\\\\\\\/0KMBbA1aFgiyb4tfEBRmX27GlvdVzwpZrah6YvS2\\\\\\\/iyNPB1EbjRRqrqedQiqQXo2EpvPVEQvPBNEzOOFAcwr40vubclIMoFjJsfEpDxP0zFmORYWIG9SBvRopSFTx\\\\\\\/66JEoL6RGZln3zDWx9rGJ8gE1VNcGQDwxc25qfA5onoYVdSADxpep0rp+EboDYfOf8LeNqQmJYotb1Aw1g1blRd2ICt3n6ax7YaRAaFu1PDdlVkBt6JzgMmgWGu3u4c4m90oAG2CxWYv71hwdib83jWrV5qM90razgNQRzWbjC11mdv+8xoNUa\\\\\\\/h3r0Ulwh9utZzN9w7m+yhkELOUNqP\\\\\\\/pM0sLT04BkmGLu+dAFRfwXFbd9jvVlQAWk3uBMeJranMcfE7Tumw+k83\\\\\\\/KJoN4rRf3vgaXSsOn7f+N+MMv5UL7EDW0RZyIaGzYWRUHSHO8dr\\\\\\\/0zq2D5uG7A+mBddcnJI8kv2IfxOWhssE50q6xVpzNjgqDaRcoI\\\\\\\/MhNl9LInGFFGZi3TT9G9e+yupFobUpwFzKTO1nSvJaePN31+x1u6W\\\\\\\/Icip+urlSqX7famBXHOwPyaRHwGIESKciHR3n0IjK7yxH0Hxj+u4yiq0DC7LGygcSrYDfMDxuN0gzCo9F7z3D7UA9QtMGOFbfVBJFP0\\\\\\\/4oHrAQmEI826SD7uVdD7uMOpD3w\\\\\\\/CrarLYn3chIYl\\\\\\\/\\\\\\\/IUQE5H2U43fp4s9hn\\\\\\\/I8IoR7H6YmUIcuU5FZ+4ke09Zq2\\\\\\\/7myggAyaIm4yxgU5FrVepulZQlJBZr\\\\\\\/r0qCPCFsKXCaEJDa5QTl5DdgWKT81Llc+FgPZ81Jg7eTBjUl4KN0gonZTw7guWz1fU5EiS6oZPG2638flRHjKvn6FMpLKU77Kqd9fc8xN86fLsX4kahMnyDvyCCQr6e5NmvSNB2VPLP3hF9Has+2PEMGqvp+kiHv2KvDMDBSK7cWeSUSxr4IGmgO2fyPpS0yOwsvWjglhb+Ru1wSb0KFtRPbjFIk6qNX7dpYvH3BEHjphRarUlwyeT8bHsevjih1AJtdWpPhHA4Gb8QjtTS+DMJDD0oOvSL6cA34vu143WBSR3IvuTSJhof0IF2ZAJF7akRa8wz71XUsQVcTFVnz8D8nrAv\\\\\\\/z2Bke9NH7a16z5K6XxwtYMIT3zPG8V0Zed8ugA99mnFgtXz6NB6Fi6GMRMffEgTzlcjuvKWNdw2\\\\\\\/Oya\\\\\\\/1C9pItgjPvpV1izxL41hQMdt+vzXyqhkqS3kpGCqqlAmOyjEEWb96fDRGD0XCrM0i9CjTZVmMiiy2zjHQhDcYUr\\\\\\\/IFTh3oXDjfESKxD\\\\\\\/4fQZspdRaNJAsG3j4F9RPSZvPQebAZG4WxsH6TiY+NK7vKS8sLv\\\\\\\/Y9Hlms8gQhFnrQoGDI++OuVyiTB8fZ46lMKwx5iIi4Q8lPjbwwq1CxP9xOkDWGK26hCHH6ZsJ1XEVzfyuNhS321c5\\\\\\\/+FFZt2XIiz+GWrPEbx\\\\\\\/5i2DZwn54+ZJ5JHBfeWM9WBYBzAx5SYm+oBsuUXSRcoISyaI+vpbMDNSjAOK5y3Okb0YDkOi2ERf59el+lsjj4QBjFNPtDlwSFxS468ejRox7MgVG0t0MVFg\\\\\\\/fRWZgPxdIWsXkdmQpemBk5b9lFzWiwElskz0+ePf1rS1bzfRulGP5fICzfFHDLiifYDb1AL0GsfRtMq+qadG2tnmtri9fdF6B30FLQrvOeuoEHS+vTwFCFvDPQ0uqYn7WHCPA0a5GR1Msfy3QsSIlC44g48MqfGcrFR5oMA8RHNlHxY2tdpIr4a6P94Fyj07muyrGvfUmP8Sx3h1p\\\\\\\/yT4qy2Aiw3rA2n5+D\\\\\\\/Cj\\\\\\\/aQin+FLDfaucqlx7jcE8xq5e+uEnRGh9rqNtB\\\\\\\/\\\\\\\/vF1et+55I2GcKlRnFgYu+TtDA0jsgvpwNLyUh2ga2i2w8h0Mw+xi9ksMTnsSW4OWfpJw2Z6U\\\\\\\/8EXYkG9chJD2J+PVGnoMPydKqIvSToTCP9pUjJQCKqEWgNEwoeL5GToykTPMBmtuzMVIkfaIOvXP2J\\\\\\\/0BxNuRty2s0JVphonOYa3SkfcRIXBd6FioRMRQDD1\\\\\\\/5iY7UehCC0rGfuTw\\\\\\\/tDwAQzyFvJ40u0uf7R7tqDJ4u+EBKRH9asdH3MXVHNL4WGrAGhTEuJPmK2vPo5OHReYeEeR1yPvyXn\\\\\\\/\\\\\\\/seff8tJL2N\\\\\\\/rHtSHivZT4F9PUXbNPhFXWVKgc09akNoO7x+FoB3u7TSuNez6tTDPk\\\\\\\/cNZKmavFOWP\\\\\\\/BVZJbU2b\\\\\\\/K1sz+9FRmXzK6Rh1Ur9XbqSYEbLs9F+gRsAenoJ74rV44Wp3jJjpeRvPYWyG8byv278HEH1B0IUthPHyifl5kaZR6L0Rlj17WovKOi9Ot925oFP7QTN0bDJ5zEsrmeSWPLTDWBQCGZYqF1koo5c8M0cAY3WRea8zdgtvAlWYRWjUKbAH7Cbtj49mmxN8Fih6VaEEXPTJmxg5WKQJwJ80KKxXczrK5dN+hffDj8aSr0ScuwBa8FRnm5Qd91zRubNvLz5HSaI0dgO+Z1lIxvsmoFNrHZTn7gNuK2+pdZORNp2jlCw04c3GatI3uEo9YfhElZ0HQzx1lTkGJ5eW5JTP0WfDEvRpFBWWeW40VOgaQdMD5EJHs5wAzKLAxjEBOeHsnvgDLraJIVdG0A2aK84uLq6fw\\\\\\\/PJ4fNJYWIkn0cVqGcBunrIvKdXtZCEpTpyXlbQOtb3eJTGt97xqQ\\\\\\\/iRdTcCC0KE1sG\\\\\\\/l5d4C9gBA47SagcpC4JZ7or\\\\\\\/2ehunDfLDR2\\\\\\\/eETz4OtAn40poo+Znqq4ZSHXvKTVKAePurhsVyLHCirm45IO06nQE0nD3fjIUXa080HT1lf9KfaVhZO\\\\\\\/bo73AyI79hqcBst26FjQ+45vVpzShar20YVb312D33FTIhHUFGVYVHAI0CE9uSid7A1UQtAWbnhq6joxGUTh\\\\\\\/\\\\\\\/raiHkuJkUqsM5ry65F8XM0oLmtNS6yVEqccvpPgnJXr8Ls7T8oI1Otff\\\\\\\/kPZNuDzctCOjjFg5sgreQPUMwlFE7BH5L7MbHUNd8977EPrFHv3KL21Fm4YEw+Zio\\\\\\\/1OBCJ37oBdfmHofVEiNyd0Z5O+liQNCBdk05SBxn43UvDp4VDP0Tr0q0ZRL1DN0X9B44t8pcT+\\\\\\\/kcFdRg+mNUq7iaVVfEnRnkx4tUeICQEU5AkA\\\\\\\/7qZumQMzdozjNJFVBzZ\\\\\\\/oG7A6Toeq6DZlMZMie1BVvPAnPxQYO4KC4nZYD4ArOdk0BjhlBkc1M8n+OypQoSewybPZhAi+HnkU5iTypzdJvA7bnKx0oIqSmRMp5GcHH\\\\\\\/ofE4N7ma6LqKv6v4tjJLIxMgllrSg\\\\\\\/6y8wazQz6OPQRiQUYy92gq+XSHRWSOs5+SJI7GieDg2S6sZja2cYzfDrfxbn6PmbThAL9NLTB0PF3lV04Rgp2B1VP\\\\\\\/LzAqLFcEfWBvOOhgxX5vQYKYmgu46lGAF1Q7eirrrEXzOM3Relj+BaDA4+Af0u6ZmrwE4Uk7KjPJlgKfElIBlMWhtEo\\\\\\\/y1YKQ5f2K\\\\\\\/M\\\\\\\/M+maB7tX5en6TPdfmi4HeUH6gNhqhd\\\\\\\/gxITG2NzczvxH8ko6Fra\\\\\\\/sU\\\\\\\/Ny9tWvfldmEHbvriBnsLKec6\\\\\\\/ThiuFcFtXGYHS0Y16JJWAH\\\\\\\/Z7gXwuB6gLY0nUwhnL9ACR6vctT5q9lXtDPvrzug4DBy4rwrOErVf0pWZQPbpTDakwoZ9TtDFnYlEAAV8eBdS6K0TcAw12F\\\\\\\/4oQjrKQeWakewyv+wAzvpnK7bcvgFtTBSGzXG0N+wWgkrRVBElAhnMQnIoyHOdyam55NIifgW66WkzM6arfui4WGjE4OuTfZxkfFyZaqBSYY8JNPeUZjQ8zBoCCFcBR6WWE05Uo5YB5PEQIMQ5X5HRM5HGqEviI8JzArs0MYIBwG9Aj+x8\\\\\\\/GG\\\\\\\/UyBi6mSA2S1dCKXN3x8QXWoK3Vxr\\\\\\\/p8iEj6mGMgbx8DxXNRnEuPGVnK8L3IwteW6\\\\\\\/nVs+3xJyp\\\\\\\/O1TH2DSK8o7mjlPIpL7fKsduVSb9i7ww4sxX0Hdqr0Ngrp\\\\\\\/nvoKP\\\\\\\/43dvHgxB4GBPb\\\\\\\/7vCICnzhuVuZUPOySVYsi009aBVkFMWBgOT8tNbpUrl0O4vlwijUDyNY4fZ9jzN9AxDa9f2TQVHHy5TEZWUwE5I48BWeVTfi2cWq2WWRdNJM0Yyxf6ETeYXwo2vD5NlEhW3jOXGMq8aubfcOybEtUGvMFTpczOhLIyHumOjS+cUGl+f8CuPaTo8UiIr5SBZiR\\\\\\\/N004PxR9wctYUxMBsHSNIu4ABYb7gV1Sr\\\\\\\/6GWuIMNo\\\\\\\/vQdD2JnFYugTBKyjZwyHWIKM6\\\\\\\/QWqj5Pd2wZ7NmNYYSByy9nNWzCIUVzSuyLzTBmqOQfJjZK8LANAZ\\\\\\\/w4bOtV2wYopbJVCHKnYQjCi+WqCsHDDjmBJOiXIpAZKmBvZCwhSxY62R\\\\\\\/SyfKPoyizDqEApEzL+NpG3FrEpxuGJsIo+KcFE4rMNtm+XbPRMZe8reC23bKF5vxQXsHuqWX+\\\\\\\/fD6E5whwoC\\\\\\\/PGjNT+yw4zJteb6F+KPy9afYQkd1IuV5jkzg8QkkTdKUtM945tMxrEa+nqIROm2ydxAGdcs2ODYPZwOTfbY0k+o5uP4QvifBb8l0FackdX6t612HKmAvQFpMgbGy3urr0mSxdbxri4U\\\\\\\/uG63Em\\\\\\\/l+ar\\\\\\\/kKYeynbvHIt1L8fERiScZE4Ut\\\\\\\/JtWShCT\\\\\\\/ZlNvDTXSA0ND47f79kLoa4Uywk3TzbzQXvAz2r7\\\\\\\/Jx6isoDXBlv2Xy+vf33AfUMlE1mWZ+PfnNzHE2KCT5hiF+Uein66UaXselBMx5GozFb5mj+Akt9jUM1Z6lIjg+ltUra0ih+G6Fi0ueWYlkxmN\\\\\\\/khnATd87Hz6qgBc4w1Xml7GbuSPJ2b\\\\\\\/2wvBn5iYDPohHYFMRx1MZRQk9zZakXfJzdWTHbau0bGL8ErEsa1LVDMsp0tyfIkB2cJjBqYN0qhvm6t30XxqJ5lL9SPiBzyO34LvJTRYtMzvAOrSz0YQKzu2uK4NSnFSms\\\\\\\/A6oICodtAqdSAb6j\\\\\\\/fPaK3veUMLuVmV8mRUoWLf81wEP6nIe1iXarQsv0rsOCgG3EUYNILGxNTWfYblkPwl1DjmqXcgmELNLT4\\\\\\\/aiqgcsKwe2PlUkNbgusURji56FvEZVEuzwW80tpbv8LDblTh0Zn6VnVjY4u1TVuJaCAhdezVVtZoA1jXRal3rtmKsw5HVDRHheT90WZUMzNwySacwRVwQ2Sq7RY8FageKttb6IQEG7oDuYZA8+5lgH3Hn8uhWJEzT5gKGfagA1i+Vl9YJl2JO3mVloxt1WDiSrmS0DXr3T1+u4PTn5wCYIYIKSepLQ4OWl8NwNeP8sED9BQEj8zctaKPKquzDmkEBGS7aoaUDhv\\\\\\\/wAq\\\\\\\/OCpxSTEJE2k5mlgM+tZohyXJuqkkRCOeQART56MTqDnSZzGWPmRPr5BlSMqnW3R\\\\\\\/uYkD6XJ6OEx38Xc4dnodj5CVEqBVg13+jFRzaWE8WyAMzLek2rPRzrZ9uWX9olwoUrOs71V1EZWCGvwbBi9oZrEOTZuSwj\\\\\\\/FM6FmtrLnN4rk5u7YxXfQ3Ms58Ymk2bZIz+02EJD1lJiAozmZQTJ5GssTgVEYGSsFRdE6UNtgv3vErMpjoLfb37CIhDkatpFD+so6QvWY88D8bNXHLEPqbqcHfX9w5RTg88viswXma7Cv+qzWAcX6DEY\\\\\\\/dlzwFEJ0t+9lEfKxCy5MVip+j9PVS1wigCa9knaz0dFbVRZ0XMf3nU63nDuFqqf3y817EO\\\\\\\/V80tZYDPy3kJKXxomp8X0ZGxjPCGSORyvB\\\\\\\/Obm+WTvFnjUsCqfT3KKc\\\\\\\/GQ3JcQSjEDNk2G6vgSFLIW8nX1BZmaSlqFuw\\\\\\\/ZmO9VikGpzlIp42F2h8j94FwF2ANVHMrznMw+BQQIWMBf4RNPuTtshFWPm59JYdLKakuJAQOOJ3nNQsMKPQRsyAjT4x8qdX6VbVpPYiMilQq31NuKJCd3dtyHFP3gucnEJnTYFBm4EYcLZuWkp\\\\\\\/ZQJNwRfvq6PcZPhVAkecnbVw+Bx6MG7V70xymx5AaBEvn9q3t4Pzb7YvkjnH6tz8myO\\\\\\\/hNqRXJYDX8wuJjubablNCvQ697XYK\\\\\\\/ZvsSKGCe4pmsWQfWpa3dJ7Lz6FlWhqhgmfVdv5ww9JGI74izYX6mmmn5fOqFmRECHCBXYTmy+dVo2G0WaTODfeMy+zbhKS9tV6fxR2g6UUtZWIplz2ix81RM0MKc97fNSPZ+FL5q5ktEPyoeQrffcoc3x1LcC8okQLVktW+6Xj\\\\\\\/ZHRB5iZtyTc9kqQE4zXOETzWPblmI350yhRTRl3ErrXqeM1Wofws5H6jaOP\\\\\\\/to53pYCokX9v+h1JWJW5MaJOv8nc\\\\\\\/4MAkfL22G8X0M+bwXwb4DMarwtyL8BcJYiIJlnUF9s3EprcygeIrifC9u6WtZJqim77lorSvJPKhgNgJGvdTV0MgILy4MCr\\\\\\\/3\\\\\\\/smr5SXL11SqxUM\\\\\\\/6xPAdk6Z6topmDxeZLHQ4Tu4yQVCFxd7D48bMmSdDzv41Wyced6i80nYGh125+h0I3A5nHaO9JQqx7zC3ZOHK9nG6gYlwhY1k+kT0U+KMkje2Wejc+vRV3Bv5eYSuvLjiVmbx41Vb2lDgwN\\\\\\\/N+6F9ktp9kTz9YPg6H4iOJfxnVz2tRYhe4o7Qfc7cayxy0xfFKzp2iS2BUP4NgjdqtM34BCBvH63EiY3RX8EJJ9MxJzl1STLjnB3cvL7oPgGv8ch3y\\\\\\\/W4tktxYXH8ouVZFIX4nUPyhErq0fwk2Dr3lldLU1rysc+CpaZGYYPzPq6kTDtxPWcUcY3J+6sUhMFVbh6Mb3MixMaMY3TQgE3ptJdFlZg30HReXB5441Cle+TrZHdgAr4d4dw2sQWc1LS8DvSaLaF+74Dnouh4J4Nqv0KLXJlI71ToyMcvnUwpvgl7fgf0CsrCVMq7S8BVhxYAGguQcMoL6\\\\\\\/3JhHb+4CuYqYHLB6EZKj9\\\\\\\/XOyd1sDZfnGd7QCRK6UPOl3gFwMCAsyCJLz4vsmvFBY+iFxWd5GZiWAUn4A6H6FeNvU2X7nXKsCdtwJru1s\\\\\\\/mBgdaxBOkatchvsTi9ktGmRP5kOgLJbCa7B7WTIeazP+vYqOh0dG\\\\\\\/vh6krXnRgouMO5vGih8fd9qxdDII\\\\\\\/gwGCqYnIs6xqQ8f2SMJ3nsL1OCckbOG5h9sE+Sf5a2Id4TiZx7Jh4Wime4ixOZEfTiGpGSR2LpGiDjEzGXBam6J7FiqvGUhz0Kq3xYJz4aLxX3Y0aZlF+VtDauVw4aWtQvMYce8JYqgms7Qiy\\\\\\\/TQM6orJb9XFmMtHbhxvye00CQIRfWV\\\\\\\/dMQbInpv4x0iFCRnrzBVqn5lnJ1A9smrzOomUN0+3\\\\\\\/wbciZaQy4I4m8TZ+2PB\\\\\\\/hM6pNTyZxxogrU77qJoYnqAnwrCOep0EalEEIYvgOerk2rUgwJlOpBKE0fWc8EpdKxihaN6Hpy2+Lnm945DfHf3WS46+cjJyyYG5itoZESAA\\\\\\\/2MKZ8wUgnNwxbkBd8MJPmqIfiY4rvTx1ThGhjrUIFUq+keObCUicjZXJ1VR5h+fYRWY9kC1AQtZiHe0hwJKrQBwIW8vk9CV77QH8tPJuT7PxGrl2zRY\\\\\\\/wzJvkKBsmVRiPPTZsNpxhr4epgWvLO55tTJDpWykuSxIlNgPCpui1Uebx+rp544Z+u\\\\\\\/OpBp4p2WG3v5F5SAbwbnuHbbVXN1T1FZsoHgYXZ1SOYubEZH+fsschlXv3sL5TLCqDXO8H0b\\\\\\\/xv0ST3h+KHHxNgRT9HMei4qwhp4RTGBVa89yCBX1S0gv8oYvoBekbWhnHF5SFCy2zgfLcxyq58n+F6jdSOtzGqcNU\\\\\\\/GPSsOVzJfvqwzeHOG6wCt8grCv1fXHlOCs2QmbQ7oVIwDrwEfMUE2BRs8JV2hBj8NjdgvF17GaVfyLTh0cHwrbRV5pQlAobWA0M+6hF+WhcE07zd8x40uPK4X3oSl0hk0TQDwl6l36r1h5WCojVfO8dMNMmmrJOL8+FBKgj\\\\\\\/okoRi8sdP\\\\\\\/gjxMjK82BxeQsc7d6UUmmtLgx\\\\\\\/ZUNm+u35tcro2PHrZT2IfQgayqitpEYxJmKdcR90arHJYL27UD9gkLFibtHlkRLgClGHp2KebtQAdt3ECWIOqjsihqRi7AZ9qZ9868KQ7TMC7u7C9NUDa9Imla9\\\\\\\/fXn+l6ufL3BuRFg5aqATgkJexqTrOA7ZTZi04FRoyuqVgjsk5U3eFubQDDtybUtIfGu9nju65R87dThZfeTsQu0XHxpVp3VdoVXl\\\\\\\/aX7tVSVU6ShCy0MNzDwXYtOMlC6J2oe+rE6Igtj23hMIAEsdevz0B\\\\\\\/2xn\\\\\\\/JaOdmgVv2Y11WuaWW\\\\\\\/Hns4BpdIZQNr1o3JGW\\\\\\\/NzDWI71Q3FOqXoHCU1Fjq\\\\\\\/ykdlaM7hNN3mXsMi3OqckAS57eOJMOEbdbO2fbI\\\\\\\/HQqj3LKtp22Js99wIr1EW5spHz4aDruqrezqlFtc70Dwm7ZWwkOdqqhWtRVhyU6WtWYWb2aR3BHVf9K7FTm7L+Tf6FmhaMEH87x8AIWfmCzMxkzrEPG6FuO0dx\\\\\\\/z2\\\\\\\/NqfmNpL3Pucd\\\\\\\/6BOfonjOMG+tRmn9TIlFtuC7bkLpy2kb26uy9Knf02\\\\\\\/bsDskIDoFOto6B9rdp+d5IW16hPz6vK2sp480WXHNYXVK\\\\\\\/jxbMVMhz\\\\\\\/ePqr7CHBjDN+mzR9arZSQzgaxdXrKGr46wVBsj4yBhdR3Y07nanqx5QMG+JcBPILG+PbMlP8Oph8LZMaJto29iejVEMgRWOQZ8a7FENMR0+rdHYH6ziv+Y+iWyC4XscI29ZLHMdXvcAI0q7V1dS8jxvS0bLDVWBjqSYEL645M7qFF6Mo7GdttrL7GScYymOz+s+TzhJ2IrSa0ZA5LeGzbE29NIVVGpmAG1qwfL6JKBcWg4xQ5PIBMJ7SRWujFY4fjfbCWooorSXbx1I6dtKPKkxrWG9pFTWWozY95MSTtfFYFK+CLDj5Ve89cZeN6MN1eZsXS4qgs4Z+reEtQCsE9\\\\\\\/Z6nCTIR\\\\\\\/JKmLyQzTQ0kpAXMLJWvhLMMESjL6iLdJUNUO5527m35EyHBuzgl0U4hwzDpuyaRZ8of0ixioSJ3wGCHy50y01Tf5YR+1O\\\\\\\/JpuvW+axKtRyxVyZe98YkFdqFjq4UgO0i+zWlaNNvU+\\\\\\\/b77eiyWyDLhp8izKrIYtud3RfHps5E\\\\\\\/\\\\\\\/wgxKXVV8MAtuN2PcPnwu4pFTCA5iaIRETs\\\\\\\/rWj91mhINZxvu\\\\\\\/M\\\\\\\/hGnu7TgnWjaY8TZ30E\\\\\\\/V3X9yAFfjiP8AS\\\\\\\/av9i+PsCVBsJOk5N3QfqrSEFwrNZjKu6mlb+DLndmOrAj1Um85+CfYhes9tsfRzkM4lOm3aqL\\\\\\\/DaNsQ9PhV6Bhg6YBxq6y+3Zy8inpnxzh8SEahJ+wD0104mgWknqMSxDTA5FohyKW0OxNeyN9SnEuAAqbYr3FEJXdBgO1ZwWBVAfnfR\\\\\\\/y2NNzpNcI2FUbY9fAaxm8IMPrOjRRLMALObgMYJYqaHtZHCZilL9cmKKrD+w3I\\\\\\\/Itg2HgMY\\\\\\\/9V0zdi8Kn4Fn7h7ZDBCrGp\\\\\\\/PxRnVlzvK0c80CK9olZ+9onG6xc+OpcDWTl1uKNvgs57WV6INEcT5R7vwJ+i31QBod5T1QfWgvZpzP5Ia4P8BMfQH5XejCIhsnyEzGKEV3\\\\\\\/rPAyZqLpme+z+OeWMGp\\\\\\\/hBT1\\\\\\\/AOAqBwh0fPBDyDcZ9ubMHPU6My+cE+bMfOhFyQjyCiMfteOWIRpwnr9IHbkEcxErk8oM4puY6tNJN9JSmNZ\\\\\\\/GDF8RlcMvTKM0yeuAjU8Y\\\\\\\/Dnk3Di3sq5eCaWyvTQaHDT51h6HIx8wEJ3WnzzM2Y\\\\\\\/BLkiem76mjF6pRtC46aQZj29Ru7ckjOkmrHJzuZbLIWceBuFY7yQNpUBmWd6STIKymj5LZlPS68bWxzQKfDrPwtEPR4OvjpiO9uAU+eELsQQU2PnNgs\\\\\\\/8+7Kqbsm\\\\\\\/y\\\\\\\/BNcqtAMB6E7XsXXoSmv5ikCVSPCV6FUt2DgTEPi0hCZNTU2zAguxmPGESxa2CjC8yZTZf\\\\\\\/VfHXzkgApK6KiqD4Odu5aqDYGq\\\\\\\/wI9O2ZuKoeZxEWRP0WvdbELNiA0dS7fDlANwXRnnQFzbPwNgV2px3cYiZSyJ2Qg1xIMokZ3TuI8TmGRhI9vxCj4M7pH++wYrXaq1fr2M\\\\\\\/0kZ9N9nKBv7KHtYy+aF4U3+eWnkH+3ABh78apn7ajNmKE83Wd1YR7mHc2iBhHhI30SEuNiTE+A3zA\\\\\\\/fhJfbGt6a8MiufRlKEGB3YAQULZo8nc6mKGShp18CH59YSVUCzCv10P+AyKrZmCM0u5rmNtZ3oLDUU8ZZA\\\\\\\/9KyGhBlHNEDvU0DwOuIpR43RxcAQpsqK\\\\\\\/NNoyLilzv1slLEblaFutzD1ror2Wn62zNWf9\\\\\\\/6xsKtsl5p9FQD4iyL48HLuAAdhGwwDivakeQ8ySmI3VqSrqkiZRs3zuLv0\\\\\\\/DYAjJVwVhzt6AOBzr9NTdZLXhJHA97Qdasg4FtlqTxdxoQ66BPpjuxd2qQhTc2SPL2l\\\\\\\/QzolrMCcA7w2qqeaU+1T8flVvT5CYQU+8uFS5cyYfWdKtHA8aPbGJViSzhn0XDnbnMrzXvxLUv9Lh\\\\\\\/KKY6\\\\\\\/B20OsBXG\\\\\\\/+gXLQJ3gMVZTtThwDFmbn\\\\\\\/cB1vTc0wdct2xC\\\\\\\/vFpnUWn4I0XWxJx0lj\\\\\\\/7sgX+IDF\\\\\\\/pGfV8WuhZpHlzraXUnWVPRxs+Ncl1XrRDO\\\\\\\/LepSDvGTdQFDmfwbq0Pu5WnwWN3SX35QFCWJvk6qHDl+ZmhJkMmcf1OuIINs\\\\\\\/UdwsD3rteZrHNTThdDK1UOBfDnwaMx7x7Tu8T6zNEeD53XdWz1PsuJdsgG+5wYrzwDD093Yd0eUCPb69HNVX4YrFK2rFkxAFZShLI8SHDOAlfnDvuFUrXHpttt72it\\\\\\\/oiid98hlCM0cNtaAAJdmIoEbmWvYocV\\\\\\\/4tuvn41vBow73IOaZPV2Zq0QlMNdlc17CxF0\\\\\\\/SFq1IbbdtfhBlZS+4Y1AwF1g5uywcFedYBGyh3mVXdFx0acaechAIZ6Xl6rstUGKaeUuDuq9D7HRzeJj6WRQCDayVlffKMAbCwFK9yDmzrMOK4JfR5v\\\\\\\/09PCMx\\\\\\\/379myxsNDqd6VoEL42ONRg7WyPi2p0jv\\\\\\\/Zy4LpotlFzVJZsBr6Ka7qiGIOHXHVRSb9enPaixUbIX07jiaWnmw2lnQ28dk5GnL56FepdOsNd572Tc3wG9TQ336ygEcyAId5lAXJ6ZnF85Xm4FnLAEyLRYrQkBSUzbfm2xtl\\\\\\\/Bc1PMpmO4MwPI7Gtlsa5YOTLAcvSoBFhZU9XC0MsxFOcJNQWWQhwMCQrvgrpiT5ZxirHmbLTSYrz0BnPszxzkDv1SQypJhPsKSIgrYEuMlhAHrC9xLkSQpNYlyUVsbngwnCSvfUoY1BvzHkDWbn7y7WyIRGH08xu5ESB0bh\\\\\\\/maljm0Lpcot2yLT\\\\\\\/OmKuHjVezTrnTLjeIwi2TRxvRkm\\\\\\\/FearSAXldJoqM2DGwi6VcrVSAY6RcvaIxYd\\\\\\\/bZb+Kd\\\\\\\/LF0GdRxC5Oe8AB7L1Nl9g\\\\\\\/h5TSqmUJEMh1hggnOEsOzMgZT3OC70iQ18EfdjNdBlq40gMB+oPmnLmRLn4RA\\\\\\\/OLabYtw7L4u0lCZtgBGfEaaP51uAMT3BdfNgDDe+mYXuFYB4X4ATwjBs+1kEZ2OYWL8r\\\\\\\/pSpIFN1CKoPWlajT7nk76yofxAcHwIZkjhfMh2IQf8oh6QRnyM323Ys7RkuW3DAWIk0evoaOh\\\\\\\/+gpUml9OjGqE4JU1r0Qn7cSIlioJf\\\\\\\/uhR+VDTrJwOBKz+0PYiZsh8p42Nb9632JVvaPT6ZlXfDmN9h3r\\\\\\\/v5yMahtCgtAIA9TVIGT8Fb7QsuLNdSAjXGEUmsOODDvdUoEuF7cPBzxAOt8m1a3l6UTTiUWGUVO7ODWkjK1d246DlJSrjM8ou87LTI0PO4WwV3AGI99MxgvSBhnlG6pzBmbGRcMohr04uoQ3p\\\\\\\/1KFCR2qKad+jHp2w8epEG6U8RhmvWSDEn2wUcTF0uefsz5DLeO\\\\\\\/l6h7pJ9tXLuV6+tKSzEBg\\\\\\\/Qw+oP\\\\\\\/4kTW8dLHY\\\\\\\/4xiwgetx1Ea9xpR\\\\\\\/S8vuW6MymJMQCo0LimuuxL7AxJx7VOYJyX+rjKuOqxouT3e6f04uWn+v+oYe2h6siraV\\\\\\\/Y7Rs3ogu0YdGBc10zAGBjVVsDGo9vshBwp7\\\\\\\/d5lD+brOQ4TNlpYkkEgMuDCkFLvTYX9eSH8mMAz0pqv5dgPN7qYiypqKGd1CHk41dPFiCE55k89VBTGFH\\\\\\\/q+5aIlExIswo0FqQEE0gZCDpzYrXFjOlKQOFvjspnjYvfI2puYgBAwIW9hIooFyHOxYsnr+dMdYMENZKqSBzTM9sqFDD9\\\\\\\/1mkB2u5BTUXGMAmtyj3K5Aq\\\\\\\/HfdCCSEW1+UusF6Z6faWHpB6LzaH6bVx1J3kNqNsX9U44j5DIEYZH720MzJyZYgrNfoOBe9jQprIfZ2IFr1ycvolilPfsew87AHdhaLH1F1dwGVlUPPtdQjvfozu165MWx7NFb0YO8q9fWYd2hi+JC1lOoKWWkFDNVxp4NHXzhvZVRERxkWpTddAoTgLcucYdPamh1NNvvvnUS4f8j1MBhFYlOE1FadjcVkO0yMtGAOmDkPhcVQ6A8EbiAX2OQXE3Fanl92VmA8IYBBIof4k5Q4AzstF1dsebYLJ3n8B3Yn79ieqJzPCkdJR5tSqkHEqyzEAiRHc9KBbSh2UfAze8OJBrbStXvdlXOCvDjVYjXhwCB6WMdjbTmM1zYkJ2\\\\\\\/kbeyLxEc5MPN5kx25WMDgqZsp\\\\\\\/jqEexq5uZe9IVbo3CD1Ij5lDDzh994e5MnxTTpA5b3ckQKgOwGppjH9PkyYQQK8ooHrMBW4\\\\\\\/O8hQ4QrSgkTa4WBZW5EmQJAwa+Ia\\\\\\\/ePHBzRA7yEyYEGtiiR08s3auAfVyEwVyK8v1Gjh6rCc0Tq7Y1RqaLdtjV3qefQLv\\\\\\\/OR7G5ErjDc5KbQAO+YGza3p6WFeB8vzhohGpmlY3uMkvRV5d6fWssDNMOHs9ZtQ9C0XLGFwgDn7TajyOW3qY4UwLqeQ2Jh0c4uE+iD4e7xlWrQoblAlEa0UIgekX6sBj4Zmsbj0lvXuHfG+21xdrbxDjqdY3Hj9ZWO3Z9OtgqK+i0viXumT707Aqc67Jz2lBkh9uIJXV0tM4ENpoIc4xvWmuvMt\\\\\\\/+Yge8V0v9neigbicWv5hAw6drBfLfVq0te7Q9Ogise6oQbxpYSe0\\\\\\\/st3bdFJb9YlzjKIkMgXXy\\\\\\\/63wI0Nl3M7LC1IcZ\\\\\\\/w5ssrV\\\\\\\/j4wOCnT2vVc\\\\\\\/vEAvniV+HAIvm4soA\\\\\\\/7x38\\\\\\\/mq65\\\\\\\/BBlVvLS7PzZd7bEI\\\\\\\/ArPs0uwNr2PANgukOCHupIgKqRaiNSS93Rih\\\\\\\/iKYVdrJ1R+ycgf912ssSjQMnKkvmJmNJVGGblcLOdFU44ww9caB2lmfPUEqieMdBFonPszfT9SPKIBVUCetOzRv6FuKGDWiL6EjIBMFVFfCuaV1PBsgayHXhDMGiWlQgZeyYLQNw9BMp68WZDhXI2vz28ceijLeg28yGv+3y7YSsiJ7c6S34IxX3lu5qtasqv5j5sP5G5GxWHj6KIgTn+jOQbneJWZ6QPsXPOqzetPHpQZH7JE1fmt4jw5cYqa+Iey05apY0R+8pdUe\\\\\\\/V4cZojQSRBdfZdjkBoR+J9ooPHg+lZpOip844S5LgXZTA200\\\\\\\/VvEXr+hRgfH+GeME+mQz62MYl9EFZSyRXqolSyrtJczo0PvFfFpld03eQKXNjUEe79E4lSI8otK89pZtGTKgFzi8u+y6NFDscdCEPpqaF1vzHofAye1mD9yFJmmoxC03XZCoVZ7FgTkIeug7rzVPBBkOsVGKvBprwBS67ijhIh\\\\\\\/DCsSHb97u6+TLYlZkmDDz2KsTfAv3gTd6KVGnkB9TdWnBiJViL\\\\\\\/51MfD8FSHGh8yRsMlKsTyPgalIt+bvY7JW3joQY0uay8+ZdCwGlcTLAE5fPlEaKR0dM2VqKKUsBr\\\\\\\/rNyrPEyKIq5WLITq09Jkv04Xo35JZ3E5QRuPLmpsbfcTMhvS1PWjZ80lVsBJrnKDveullnLvb4hQMuNnC8NV72nBLqg83a1uqZ+1LP+DIGZFNiFQlRxj5Gn98McxkeR1D5jdZnVuAsM+RtPsZOiLAmn89x3x85OY1FABjNM5td5vLpn9DpcjXMM+lqQ1mExMREK6bs\\\\\\\/42oZlPa1WwHhqZa1FdvLlbrw6gikv8AUQxeH3ON1PP\\\\\\\/wAAhylxrfkT7q2o5QnfBduRpi2Bc0\\\\\\\/DXR+76Wx\\\\\\\/yIdCn5HISJlzahY85mw5\\\\\\\/PXq8bbh35PgKK68gypKKToiHtl\\\\\\\/LpdVN\\\\\\\/s218Yrc9mDwDr7dOf9WxIUpsW4FLpkLS\\\\\\\/vKbe8bmdWMxIuoCCkYE7bkHr8bfGewCNLkD6YScrusqmynyDYXEFBmoS0y+hvt01vXo7VbFin+b3zRU\\\\\\\/KU2OyliXnUx8tNEwrImVfd42vtyJgbdaPyUB\\\\\\\/0uUDjkzHcF8zuKR3W5dJHE3OEkmBb09o8QU5cwmAzwhUFGN3LkpafDAJAk6ms4Lcgv9P\\\\\\\/m8vsg9nKrENICzoBA\\\\\\\/HCtEuQJcZkwZh9ask+ZpsT2HaoDnYMSrtJZ0njjGRwcimH9yDtU0no+kzAlOPk1rVnMQf\\\\\\\/M2m7+tThkepHIxrYFXnMHQDT3Zt7lh5JN3H8oby+GOfa0T1S6TT4yBLLTzBgtcHDhQB7J2CeeTSpYc2kEFmdnwgcsDN3+NONsfWMl8OUbzleuDNSQSRR3BDR4qeWqRQfd4QGZPUXR46PDY9q0G9Ssx4j46qRqtZIfbfd+FTzxEr7SyfTrH0ujHSvivWSo1vtiRF5IeP15K+2+045sCVq\\\\\\\/UZpLtiy3mBCTZi6QAwWK7DRuyVviNLV7pmBd128AMEmJ1SV+ZogIhFkluQQ4bpwF5ASR\\\\\\\/QH34c33lXEmYVPFQtF1\\\\\\\/HSXSWpaUwDwP1u1RtWXpMKs8YOMeDfIJgVyZOYUFssMXTWe+kYz2PsAbLGPBNs2TGIJWRHxoP9aAhTImZ3qS0gLe8uxXsIGc1hTEbsr2TlIvNifteUOPHmLRI+QxVrFR0PMBTT+drnoAIC0\\\\\\\/AkubOW+frUfmqTV2fpE+r74dCjEEExRcnCaC7vHfkEwqC20x2mA9BBBNVowaUtB3NmFjedUvoMpTHjHj2dPacNxO6tFFH3teonXdVPyJ3N1Wf\\\\\\\/mSAIOt7J9Ny\\\\\\\/Db7ERZNH+P4u2f0ijr27aS9LXUG1YGUl\\\\\\\/QJJrp2fZv1SlQ8eAXIvHpK8ld0jpP2DLKkH5DU8MlQL8FlKvRBMsHGfUaPWfBaokuyQoFd0vHmR1rcr5h0CerScKg3idyJQVy2oZY\\\\\\\/aMyIZo2mkHlAlij0mFuTfldILK3ZjEVuCJ\\\\\\\/YSBd4sA0xmUcJ5VgrbGJJFokUN0UazyDZjLyuDxn6L7QBzbxQtP\\\\\\\/k3SSpia9O37McqamVdaOW6Zv14ZPsx+KjZQ43YoFHdijoxc93jc7jBVm5\\\\\\\/5yymdJf4kw02jNUFGP8g5jUORZdVBSfq\\\\\\\/UcVOfzy8M0yIahl2IMpDPyw3guBT\\\\\\\/GnEFGM8SrWPE+yvXEp1CB5awrnQky1rVXr23uNeaqBYhclP4MoQ+5ZLg\\\\\\\/p2VucgrNCLEcIJFNCkq7f+bYxPQp3q9+k+UuR2r9bKgvYRZC7Y9R6futQcnH\\\\\\\/fe9+dwM4hiYUqRIKkcuk0\\\\\\\/mwXyXE4DdUk8K2zvYDijQ5deNnifmouYcy14jqvJEFVUoKL69FdfFYbgD3NTIylMUGRTEPdHzH5nqXJY1pP9CuTVJ5mRdsvsInU+mFqAuUs1iqVhdc5Ear3Ak8mCZHq7m3xLIzo6dEo\\\\\\\/up8Em8yRotR6aWUbKXAo7X58kk6ksqiasF\\\\\\\/zWLHrO7N+3wBGMH4\\\\\\\/d5JTaHFPH3MTs7poY+FaySw1in65zL1BJpEA36+nlLbm4bnZ9Ho43natXsvDqRs3p9vE9UXrHaxPqAzz3dFTlmjtuRRHyNmdy6ebUT2B8aNsFnyIlvbWGT3TudfrTCysISUeq7+NgNCv0xMAZ7ETgi9l43+QGKSwwCSTsOqWrb0HZWZ7ydg72FwUqe8ZsEC5120DbySxhffoRu3WHjJfbJIo\\\\\\\/n8NdtnKQyGyULfJRyXV1iznvlvuWc21yyl\\\\\\\/GHUf3L1SaRFfUKr5aT32C5MutKKUImyQoTg\\\\\\\/sYPYQF0JQuldyyxsl8K3vGG4lj6HwrX3MWIwAs7+Ril+gW0vkZY4CYZtGztTKpUQW27dzgS\\\\\\\/K\\\\\\\/b2VdDWTlj\\\\\\\/r\\\\\\\/C+t3oX5YCWsNSRvZsFGwxxpXgZtBm6p6x0XzkinJdHXuhNesOhLPD1mPjfOXrtWoZRKqgqnPKYsWZabOVjwnI9La2GusEJkvuaIKtqkfWEAjTGnahYAPLXcmIbiremQcX4CbJB6KaDFl5wcNeS8YaeIjoSo\\\\\\\/p7IhoanC1gYyMr2h12M7B1+9nSC1A4CcYc5ukJ+BZEjD5ZK08tm\\\\\\\/Gji14qcQlHodE0tm9mQcNl1jdfcYDWoyi42+jM1Hhvgvebj49L\\\\\\\/x47nPVMZx67njo63Hi03rnyBWZT4A022U8+PT6\\\\\\\/+nv3ZzJyEsc4GXJo3bToPsBWC6XoCpaeb6\\\\\\\/dqE4jtXPoUeI2EvRQ6XOK+zdbehPB7WqATHxVVEcIYii52wPINm84Fni80mMFoZeVkZS0JiXDn9NWsyy8AtAn5Zq88Pjcc6UtUoSgtencY6W2cj3IMRivBrfe8vKvLlaQ1zRdJiITcfkbQFM0gnnnowq4UQhbAa2PJpUcv8w5pXFFTlTVwLk9TbP6c3\\\\\\\/R\\\\\\\/wsW6poUwVT5EQ0rV6RF3P2ICUzXbQF2sW8Rtjpyx1Qw4Awy+QToBivhNxL6dSEBYUvH2pY5IYkDnKnH7PQB87Z81v2+4ndxpGhIf0YYrSY6L0ayVRowOPpoCB0LmfM5VS5TOCNcsX2Y+hD3d60YXE8i0F0bFGjZIkG+gBsbT5ftPAC+WXuVRTQABO3eB2vmXVvAVxgEz04\\\\\\\/RQUn2siZeDCwFns+Uh49D8eSEz2Aiqg\\\\\\\/5fkxSUc1q4Craj3vQCUIurWViJG4VtDMQuB5fjXnmA+o6Av9uCXm\\\\\\\/BHcXfpsFSaaaz9o8YoPp1klN9LJ7+SA3jr10pIddYKLEGzhDE4eF4MpPGZUIgzwVvBl1f9IpU1UEyaadTw5+\\\\\\\/v9IFuSJbAY\\\\\\\/XNFIGDVFewa8OfG4neQL+dtuZ2FR+6wSwhcdC+5mT+s9COdUpa+r1\\\\\\\/GQ2l2hpvqYQVEh0YvEFEDdUXv8Y+O2sZ45DHpXMNezJiMsoA8IUz8lj6cpOy5TOjqZbB595WhfYH+oplatgJgX+gXgjTPCfaWD1eHdNNmPJ6k3vDKB+yHu+ByVxX1IL+RE1+NEXdOKImJoAn8iNZXzbCOUrQjjCdo+EBTZMNvaQrT0Kk4X+ZTheod656n1Vro+AusUUekmmeu64OlYEnAc+KLJOUCEWze6BkHeKh7AB44Ghmjt9bjb0O5psuKJl5CT389nicDwXjaOuPyvK+kmRn7KoSDYFBxDbEA6q4A2n5ky6AENqYeZN31zbGMNFy1YH+WGtsGOawSwd3Uk1nZc8SAjI4jiWb1YTpi7ESuIG4GqBadq\\\\\\\/38Jj5Aw9vTfJ49C758HY5jR9ulOHxOts1lr9Bp5D1G4niSCMGb8pLGL7qvCEDDZ+pQxVnWtwJ\\\\\\\/8tUaU3x1qtDTg9+VtvQx59cCdxVs87YATUEMH6fBuv+39Mu5vFleSbzRvkzbJCCQV\\\\\\\/TVF8gt1P2K+VrHM0Ev5VO+EhuEmwgF53BPDHA5rnUNz1coh6RcvIYOEVVmu3bW4WWGascriZPDZhxkzWBKpxqzfhOU0hsmhDq+BNZWiaHslTNtYHfMUpbHTIJszL062WUxISIyNtH1YEM1BPC8MXK74xQ\\\\\\\/T+koq7DQcKb5sP9Ln9\\\\\\\/I7ZyflKkCnTK4oSSUESrLEPhvuARAwDJGBQREhbyOXbHEHoinkpEKAjUkJS1KbkR5iK36mX3nylxVQ8ndBW6N4W6hti4UJ2EctM4vxRX8rQwpFefDZrZ1hEIZ7OURi+td2doJCLR01JdLl+PS06kXRfwBDMpWNtI+iekojKnYTS7E3dqD2xB\\\\\\\/p1J27giwvzwBtHFalARub4h2n1UeMQ2jh5i\\\\\\\/1LqVmPyzi\\\\\\\/vwjJEETGa20L63bq02lyJDfnV8rLlkwSdmDVf3a6L91k+rZZgRIGraC5FGUk9NkXeWckfGbNK2MPPrYi9z5wZLfDAf8umphqOJAP74pg3idElKlK4nYW7hOUf\\\\\\\/s595pVkn7caQetmraGoRapBvHz5OTGaHsIN3nFQJLovkLtwF6MSSffL2SbTdEBII2ugwpHl209o5CB5SiqRHMkilUKJdEzXsvgHPGjTqO39EW0eMK6ysSDJsppYmcFWRD8SEAlXwdwY6iixNlbaqB+anGQ18qX2AWrBSi8131xTf1xpd9o+Egs140mECstuZ3BOlLMB\\\\\\\/z8gVN3bIe73oAqioPXkC2NCcEcIfyLQnlaqcOM5aVHeKTClaBUvWR1gGzLPXkiVMDJg54IcKnoKUsKWFesKFbYcYC8D2WYrHhKSJ+R6v+Q5fwvVyS9iWs\\\\\\\/AHTNaGf2CEl5NOrm6Jbk7paQ\\\\\\\/RfBAI40H+O3InA+nEU0N3rHPlUxTGBpGSGk\\\\\\\/aINB9KyvE5e11tjrdjEImF1gqFMSYkVL+kcuEzDc0NfAjkIZvPUGZHnZdb1lv2pAWu+5f3Pqhngt6iyH6nr\\\\\\\/qZJ6wJ9ut5wc+MqB5Gs2EZgp19q9hlMaAO1HILFnGDHjRQ5VKdKGLdR4Yqj3N9aaR0eGPuCuInO3pZy\\\\\\\/\\\\\\\/04k8ClelYjXczQ\\\\\\\/SZpDPkBOZTI37xYQmqiLo\\\\\\\/YYLoeDdc9tpk\\\\\\\/czjmMLgatXFcMRogqMIpNtZHAwJRsdjLRB+CLnNTSV7s0PgrT82QbMl8Z2eVgyBBb0gj80W\\\\\\\/yIi9yQEV\\\\\\\/lhqySfRPiF4hXi6UDsun3kt1T\\\\\\\/5RygyseB1p9NVa9+eY8b37lBbG5w7MyNN02Fq\\\\\\\/WrXgarr1ycWEvowRuzTns0o8OhvUKpMUPAkuBoPg\\\\\\\/SGwH0\\\\\\\/X42lI9dmw7v5aN1uczuEaHgEgiSXuKApoKllrbeRtxNX9+epLBLx8hXMa9ejslO9tDBm05t98JrhEW2qL2q7CVyCERCFXyJlqFZKHifWxIpToxIbzNV7T0uhv0a60SZXjMd9J67c6EvMoMYTmJAXoJB\\\\\\\/lqmZ5V2dRgqwlcnN2wsMZ7Ja2NeE3OHax8AHF93Lq+bvShICYUYpt0OHECLEnzzYv29G8HGMU12DIBV4AtX5T+Dn3f3y8j9CZN\\\\\\\/4uJQgRtoc9Ho6S73uYyO6KjfgxdDLW3ovYrS\\\\\\\/vsl0g85+tLFf7C6+HH3GOAIeNQYVLzTuqmIQA9NUl5nN5C\\\\\\\/J+ywBj\\\\\\\/ln2y6O+Lyu\\\\\\\/F5Q97wHnNSW\\\\\\\/CliNDvnYoLTqhcQ8kktFGT08obWwArtdq4YFsmcpRfTCs3N6ZF31x175zMfDZ2Hw2ZpxwkfDNlFnLRTvOzpN2exHtRiMI5S7VuBZnYVBWJOMhiZYikav6zjq5RSdUDR3e+O5Xtuf8l5iZJWCk0vuaJCPyH7yVvqj5e60EEGjPxMxdHx6n+0BTESa\\\\\\\/eL9UDL3EBF8oZuR\\\\\\\/t5rXmoZkmdLxnXJoGbFUb67qc4XNKZypBhbYRygXtgmZFGF7RHgxhU6WBITK5+hn097qLPtbHcXQPYjnXc4i7gBIoOnPog5aO52dmVfCnPT1R3exHoXaRbi6Llta01m5y6KBtnLnCvgBjECFjlYiB9RwocxPys4f+ejnl5rIYJud\\\\\\\/qoF+1PR0YV75553WYkmHUN62ieJ+0y8Tntx9j9b1EVZvezGJ8Zb2vFvAqTOTUN5iSCGkEwuAw5SmsT4qDT6cX4FyFeNWsgF82+GX7hL8oN7csoLoUBSwC1xLWBVLbp2iQVADBAAUVfkxmg3DseROg7\\\\\\\/Biacwhls4GQUHE419NdRm2GEDwF3aFK+FBd4JZW9\\\\\\\/Bv8r6i+kA\\\\\\\/321Y3SrNdEPRwZHW2gcxEMLSzFmazF78C7BB+qVgPwh\\\\\\\/Vu8HIyigW9Rc3xBpuOy\\\\\\\/LwF5MFimRvCpwn55+7D6b+8PrUZPPHu96+WdKnHHlz2ZXMi+1XPXOAJIgBKoc+vBycoAF8mBYdmtE1ZLj3jsUh2gQDMTVvPgXFcsBT1s6tXPicha24JjxzMIMZqmQk0mLNZWhjU65\\\\\\\/llPYwXU\\\\\\\/j0EnluwKuZHKCgR9rWb8f2ZLDl2le+0cBYQn+X8Hmei3UofDqbyV3\\\\\\\/DR5xf1h7P5eVtb7bb3fLt+9lpKCatKRZa6c296pRc5mQpFZ7IfEZ3+s0SLTXa2lBEfNH3Umvw9jPoEiOo58iUycUFKvwm2DCJPu+RSOErGogi47HJjXJw\\\\\\\/mn4sCpCdIDvTSYgB016c1XUzTu3A706KtFIW2agfyEci4iataMOOre0pNXfdE8ksBg\\\\\\\/x46Bd69sXhYN+BDA3R6iynlhWx+5S3hQ1aZlmbchyiaiEO\\\\\\\/21WjWr+RyWJwSqJ71H9fTRrvpT++tbF+LqAT5iq20mF7txJz6icrwpyrPCh6JkQCQl2Ffx89cIOPQXp4djrxZpM7uYqqefpqQKfRkkUz0IxHWj3vrFq3ymKINaUO6uz+GqrcaLgdrctnBH4e11lg+N4g3ElgNdMJjuEdWuznYzmUFN72rK4bXg\\\\\\\/OjW7cQkIbvfBmJx1pdEumXIn8Kwp9yh2PRd8fnu8d\\\\\\\/dXEMXCxyv52Bx+Cz7xp17z72Uy9ChRxROBTkPKyx0UqBxy+hY6YxnWhGnnh8JQ1PfrNk3zkjHf\\\\\\\/hy25yV2p7BVMJvSjUWWnvha3QkuovzzPh6WjRta9I3bnPyAU+5ky8BWLDku66jDUQJutRr5pYJu9WDcidFCk5iocATo09WOqoYDX5D9jH3nDeuKGSCLVrov43zW2IfDmM+lvFfuhwvf470CrBqhDlfeGvRdauUs+kCuoNLV18daaa55DY2bBhPuEVnfpDKvUaTAzF31j4CBa7ud+HzDa63wYI\\\\\\\/CYOk5T4sfV8KYJggwE53R7dZ4Th3SRWW2SSmBQnJ+7chZ\\\\\\\/M1CdRl2UjB+msbXMntDmyVTIDcAAe\\\\\\\/Ksd3cEbQ9Xci5T7tPwfe8\\\\\\\/imiRKQx3OJR0IYWQEuIbkf9xWlhJOTclrxgIady061SLgWbi93Y5J\\\\\\\/9PwBWXxApfLY9+wWwCVYAOMkBlH1uPFEXH2a1vNn5m+lBbBUfyDTTeQEG0RPNiqenkcln6pQyPrTqDg0PajTPOdZrUcBX8PLZxPafcHrLc0Cm1giOoIhHepLDd2v6xtg1HRzNeiyuMOUr8Q0+++\\\\\\\/26Awi7231fJoSRd1pT23ksLFRUz5IOJQ0ZDDaTfwMcY0B8psS42aAPKjeFQV9ov14CJailvGovovMA21\\\\\\\/IeCf2Z6L+s0Hoyur5sxY\\\\\\\/rUDzTe43gftL7haof1CAoso4q6n927haK1MDaWxt6bhGf\\\\\\\/YIWxT1SZPhZ06VY43RafGiRfEYSVPSfNAFsvxTYel93X1gnJ8YeCxbfHGcZzdlb\\\\\\\/k4iZ16LJ8XQZiNwMDGLcgRbkN69DeV1shLSAM9w1ZrI\\\\\\\/d\\\\\\\/IUkyT5MYm5l9TIDefbb5\\\\\\\/1iXJk2qJr5O3ldBRpNFtrFEln6PEGKvlZo0yXKookbDUrgBkX35G3TF4PQLbM4OzigjEgvwlFjWA+V0EO5Rcqpi+cCmRVUvckg+zFb1pK0Ls8GlKsgRbWTKeJ0A1QFs2HNWDQiRmMou7g2vnTp3Jo5I2yNF13FXaOuHEv8D7Vj6zhnIHQ4BzP7MN43jRfzj\\\\\\\/LoA\\\\\\\/vDT+8u94Uv4KBHVLTB+oATWYZwTmLHTb6vIr5xIsJilhSaMFYlPm0p+cSuliy5KtdTN6bbWNXAcKArFhqrwspd\\\\\\\/eH3oYewoZhUNaESkCaIJCPNSoCAfxcQ9wHc9KPWqznrbm6+nZVbIlsCgckGjDC0c79\\\\\\\/bIb6U\\\\\\\/AvF+V\\\\\\\/r0SPMybagFeeeNF8H8g87Rm5z3FkHbL4Oop9judrEkKROdMtdXJYsCssk5Ql7kH8VD8lg7beiWBNslPsjTEzGofdV+Whf8tbei418NaET4wjyIHhu1UaNQ+sjmGHCt9R+5NJJOkBc+pUOEgYuQglQI7wHSNXkeKdxCellh9q\\\\\\\/zp5S8MvCSPLWjHfujBigmhtnlHsn6o65vcPw3fooUYuPzsX+qHL3WX7VWw8XcYlJMkfYd\\\\\\\/xZogiZDRvwSwy90LydLWtmJM8oKjyWj\\\\\\\/Gxb7mUMDFF80wAFCK+dKY5yJEmXi5NuA\\\\\\\/CxQXXluotkY3q7sJ2z7ziNaU4\\\\\\\/rf4EIWtzH5CpSiADLygz9TbkpY4Ax\\\\\\\/YsuhfVAdll6lb1HkDLr2+jmH6lrXNWP6mAWSr13tsxil4XdXb6U5IngWN+b237HteU4g9iXlk\\\\\\\/9VNhpuajduWxEbF5X4YakfPE4T3wECBoJ+DPTmYmkrbloS9OpssK\\\\\\\/+UeBgZAg4mxAB9W18M\\\\\\\/yu6AqH02biUDew9W95vLG3AJE3MMQXhNQRD\\\\\\\/CSJCabwU+LTK+qY2whomM7r1ku5GwH5QyBJv2XK7MegLT+yKAXxDzKyYuSbc2mV+vcvwzKlmEYgsG0wR\\\\\\\/5rCv2+eziIzjOfWaTVPqFIK5feecjYBJa9BE0EgaqLa4gSNl5v5NWcZp3DUs5vvHs3bCwvxQK2BelsOc5ctnbxjkoG8rBO653vuDnuwOhcWdyiLWaak4I4G\\\\\\\/vyU+DCd\\\\\\\/6\\\\\\\/7feJ3o7zhRlfUYFUlzSg4LezJYpL022D2hC1O2XJLh\\\\\\\/4eIAQZR\\\\\\\/2N8lcdVO2y3aOpj8sTV0GoTn7yRMaW6qibkOJ\\\\\\\/ZfnZok2F4UU22Sx\\\\\\\/JvmFWbfZ5a2xZYCgjsdRY7iJoeDJeCSsD+j8J6X6o6akc\\\\\\\/Hn9KJN4xJY62iPczpI++H4bA\\\\\\\/Te1J+n4otFp3KlH1rKBkQDzeamAnA9d\\\\\\\/G4NAooMKmY4jzZ4nscOz0xlijNQkz+97GD34Q68nczP1BiS5E4N12lFjOK1Tx8Vtj2g1PcTFPFdfPHBFL69y8BKBH1WE\\\\\\\/sFRIKw9cIf\\\\\\\/2\\\\\\\/6wksduI4rwRXyJq17oGGg12eziDB1gcledluFnrsMcI6rIhLPXJoLuJmcnSIStLvF+14GJLUazsU4B7OQDIrHtenDnM6kyRIEFZZNDX2G9hNcmMBZMTjkhxKtoxV5yIdHnr2KUg9t9CsSB0HTN9ryWJrnN97CKWsYVtlg9TJIJQX3kspb8oXudwnO\\\\\\\/vK5GnZ7l8CmQOBNv7cx7mbL6cQMXOeJYRzQceZ8mxsBozdVsRnRw3RYK9pP6h\\\\\\\/WHsnZ\\\\\\\/zO5Iiwu3YdJiLm8ie08xpfYCTXOESoEb98Pd1mqf03ZA3uG+k7H5Pdo9lSbkUEkk+TjZjw7Y6eSUNta2zEk05YHdVRnKlQ1TUS9Y06R9ZY2nmBLsKtfKe27t6WhEE4XFksmr5S5vyXWHjrh2bWG4Q1y6LCKVSIsBgRa3GDbhWy8H0P6R2eIK1KVPdXwRUcFo9Nj\\\\\\\/4OQ\\\\\\\/xkmBV0X80DpzySmIV3EoS2HQvemEAOKTTyFtd5SKGC92rjH2hkH0KP+scNylyO+ApHPEtCpvqbKZjAyQIBkcWtgSxU3Du+5Au\\\\\\\/JOLly3LZHt2wU+xPwVh0nJI4F6M82u8POq7OS20M\\\\\\\/QtrlmG3N\\\\\\\/pzUFm0Xg2m2e6UKRFC6TflvUuPH1lhDhckhOkwblddqx9K+OGLLd+GbTqulYWdzbpncCHxZtHa2TebTR3FrVDy6N99CqLtT6SJxL76FkWsq6VOIiH+JXHlyYwQrZ1pB93gDfNs+NkvnH1SupKvJEKoO+J6TXja6p\\\\\\\/KT0AOXHWmMKyzZQg1jwjf3m2GCxqGPFKLNwK211TPYB\\\\\\\/XW3SaJNZ\\\\\\\/ZdvAb0mEsFQeEPr\\\\\\\/YgVQPIh0JZ9wJOg9yANtGhuRgDNjzpgkp2rppTfuBTHKGLwg7bFd7O7x2w8uKKYl7V9MGYKlvF2Umt6J4UuxT5mEZQNmN\\\\\\\/aM9+kZkdrcPp+vCJuPjgYbvICB2naEjHY2b2POchwruMBSZu9y7+93eWyfYTrJd4PM6k9vy8uxzy8+kKfbNZC2KLobhiPLlfe93sBH8OrNDJVU1QS3kwyzCmJB31g8AuifTwwS5na5fByNk6CSL1DUZ6WUGT7MsQXP9AjAMjDEgClfY6dQ8equO4kTMh\\\\\\\/h6Ioiz+OA7mEIfirCCci1sGBbpN\\\\\\\/+rWYRylpfIvJ4v8lBfl9IXbC\\\\\\\/m1J5QZ+PYgC92hh5\\\\\\\/41FEs9z1t+0EfvxlvWZUCCH4spYarudgDpBBXlQkIb7b0dleV6ZrUBfyoUSePfIhwaPpr6uZhWbam91tUE7CjPvmmet\\\\\\\/hNcRE1Y050SJodOzA3Xge8UcNoAYmMOe47I7jRXkYSVTu63lINMzyOkO1X0UALyKIn4NSUhaBDxKvYNX+d1tybCCIYv3NCHvSTbqhbjZuq\\\\\\\/wnK2WlpscVczNFhYW6WzYnMy0YROMmP51U9GpCjw+gPCdfQXn2RGQHvL\\\\\\\/T79hb4lhL2DllQppqyKqemr8J5tiKoltNMB1GUEQkhQF5cQrNswAbRUT87b6i2shhOpIKnIyAARDMia+kqtG1Ttq0vsZQOzsheuAg5ZtM\\\\\\\/zPG8nyRhT3PCPlX2bVluw+zITX1LATIpWlr+jc\\\\\\\/p\\\\\\\/X0hmAraZBYzgoLwzuGiC4wibv8nljlD1hbqXLl\\\\\\\/p551WjI7R3FoNYfBDKZ+TrCxxD0AaPXOiy9Z6A4c54nutq7WUtWpeLtyUgFSk1lfkWYnrDe7uV6H0AlMgyVHrbmPlgllN9lCP4LzvEtnrc16WbXOlbB4o0EJmEzJbSNW4e2MNMj8bTGHOZIf40U7SufA00ChU6HBdyfGyJeC87vI3foyiGD6Wlh8jG2lQxnXsPczTqwpsCQPkc6MLpbvcLt+V3wNz8G1f4zvJq9jZ0cwHCYxw1B9QkhP6idjGfHXlN5ls4QYufDfM\\\\\\\/zFSnR43fjkN7jJ4q80CPlQlFAvX6FVqJyeXJXULeNsEfYvB7+h\\\\\\\/gcLtD97YEBE2gyzm+zDzaU4Ia5Sg1nS8BvMZE1\\\\\\\/jwx1IBCWWYbowEe\\\\\\\/um8FupoeEKMc3v9leMdUnA2yd7j1NVjLR1Z1AhA3n1nJ7Df4ClBQTybdpgKKnDdo0uObhDReY+QGHRX2mSgDx14vd6\\\\\\\/kAOgnX2riSV0ZK8auHUgNwPt7RE0fYkc51dkVgSwserqVEEgxKTPWkvurDoSfBRCdM0GmYpLAkjjDLpKHBPAhRoUHism8MhEciVS0w4yvnLS9ryX5WLK7783qYXldnnTVA2TAmaOuh7X8madCwURjsklUDDgFwmVNmgROXyAlLE0MvsUPwnZc1cXoBdfasWSnJlkeIzE6+OYli6rIvdv5CvsqNu4M78y9d7HyHL5euMyXyQIDsDFUeN3U7CDpcfAYmQD8+jqf4VRK5usANQIf1MsHvlWyskPHGyJwq0v4UJ0UMWpN0MWcrY4FS5u3\\\\\\\/X7fQue11VZpV7kjBnqmD2GM4OP7Jqu5UcxWLjS4+LHpiC\\\\\\\/7d9f6J0tNzQSRqLva8U8Kia2boXg+tC15ypBHRPs7IakpfB7FnwowEwYYK3NhIiW6SQjkOCztvTFd0geCl3up9yXTg0FeM1OrD4Ug\\\\\\\/h3xLtBFOKMsQBAmtKzh0+1TAveNwwHNVnrX8EnTz5dmYSxvFbuRq3v7EqKQHgYfBCTnS+NiDissrXyf4vSMmy6m+2pk\\\\\\\/HFuCooYLAgo6kFDlS3CxFLhT1646qkkn8OGIVIzz3P8Xv5U7BbPhXSg+YZvEj+qtVBjuQ9Z52XmLFbbQSKa2YKwBo2eSlvbvgJooUBKF+bzm\\\\\\\/uPaQ4Zcjenb80u+weUckB41RpS8tPcMAONYpSy0Z0PhG7Vz\\\\\\\/AiGRcMVUhbaaXxyJshBcaQytM9GZnTB7OajyFpup9tZvsPmjmmPyeGsS0zIMEWTuqMlGbFT7I1nlqHVhC9PRvhME3LxisDB0QOAy\\\\\\\/qEVwEeKl0XOl3KaREAyktTce0lnzyU2AU6TOEppBGwKORHFGI3azJCOgy0\\\\\\\/hTqMnB1Mk2CTVz2azOS284WH7T0RKSHZRlrxC6pWxu02vm9DkyeQ6RcT2+R82PXZPDIZWBupYxBlpIbwXtWvUWKpq7nCk9yVPAJ2DLkdL6TBQBWgMGZUGu5H3jOlEHH1J0vzBKoaiJNM+2uRGta2TybGgkUrtNCY76oDkJqm4ZPmTDZnRkSIpzb7froALpxOm1CFQe5IMMoo2s+QjeMJczFhYHbtAlKAIYyqGuSOuAfmfV75xhaiX3UqgiDpVokefk8oWHZ2yMn32UqVScbDX+RguuiMZbsadszzgNnweGNF\\\\\\\/I6GbM7RdR6xtjbOW42DJQHbWnoVMhunAPnLwnRCu\\\\\\\/FTEQTjbXh+4NVraufNnjkZAHvJWImu5cJhAW9wPVNom\\\\\\\/\\\\\\\/hnMsa3Lnd9S0qnhBzv8hHaNuX026DrtyBPpR7Z2TDglvCXv9+2CUAVhniHGsA4kcZfCCDRbQmG0ImNTatv0f9wwvCkyhZ8x1K3KM+wy8hcujuPp91JcqF5RK1YVbKtm4B9GV1eOM1zGo5lwzPWFHEEYAXWhP95s1YIpqJQlWoe+S4xbrZsw2Q1S5vy1RMunXjeu9uOe85XytMam+JIgrM0R84T+YHhLiCE2Rs4ZD0edzfiAdOpifOBEjkiqIb9HW0vJgV7GCWCmk9O6tYM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022IYzdjUMO\\\\\\\/6m51r+Z+R2hiQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b21e1cf01af31d4072adb8a0d1d11f27c035139d0483e9fd9b2b88503cc834fd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wNOosDzTrimDTZMJyyj1ZPyHgPR1DHiNgPSS73CzUZtsotJ42RLRMsWgU+QnZI6NqBFQbB4fn8BlEeZdk1YAfG+avvAQejDblj2JEEnRRhWw7K8e8\\\\\\\/ncnzq43iPF9Vbkrdb0TKIdY\\\\\\\/eRJiHjBOs2t\\\\\\\/dS2cLRI1fsoOsWJHavgwjMpR5jTNnVg0+11ES9JFxwwpj8p9ei1zXU++Z4q295Sm+x7vVG3\\\\\\\/+2Mpq1JxAw6eCcZbFUBpNdkpWZwhUxKIEe0LnfVoMMzZgX+3g2LpuUeqTMb0JnuOik56TcCePAzm9eJpOiQU\\\\\\\/6cLQbJAGmjgSPi5hKnaLEker+3RdGfUNL0Tbqo2QK99Q+vnHRcy+UNabYXK83TbX44Bsk5xhennKzgQHyFHIl1MJ1IwzncckV3WHVYytf11GSjcii9yWhb0TsCra8XLNMjIw6kwaFFrieZhrZCHGSx73nHpkwhy2Lb7Ogyz54RtgZjq5kNcQ4tOQ6qdnUJeWKEDbiuzoTiWt95cFz7Wc3Cw2j10jLVQkLIBJ+DaYgs6G91SXeYQ2eUcmU3ObA1nA8P2zaKT4Au4CN3hm+ipzRpTipK6+SDVJKjMLa3MGBWMinrSuQYlbeqsr1NLDiOJumWUuCet2k7myMcNxwgKSlX1gqQkYAKbkURW9atxD9K2O4kXf1tXjoPQMFJO3RyZpXpti31I1\\\\\\\/+Rzi387LRtbVgj33cysiF8xS78H+Cbjngv2bFFqpVmsvHBQVqq66HATZO8VeIMr8ApxEE\\\\\\\/JGCNh\\\\\\\/E64d6ylDifc1XQLBciIKx7jJ02kIObzVW+XAAqUfN5SioZYp8lvkgUOSFvmYTiURwaRm0FQH9jXUsin3OjUAQpbWJutmQXQCC0TjuAQk\\\\\\\/gUNfYHC6E1RqXDXFHZGoUGecViiwK+ej4dDjqhjlJH8bu7ziyJm3PTj0gjbe2awGXKwHMZeCWxz9n+Z2m19FqyV6XhElpWrzrbAdQYLuhtg41V9KviLQSy2rB4OEraGN06RfSQYHo8mhSDvLsuNoASiOdPXFTldrF7AaeijbF9sPsbJb+1ZPUNY9OIEBu8xKLxoFPrRisJSYlWFFzDW776J\\\\\\\/\\\\\\\/ihrqQfHHWTkAWLTFF4EU9FBd5EHPJvwNr7v2qxYD0J+v4ep80VxJ71T0ABnes\\\\\\\/T0bM2PD\\\\\\\/i6vHe5zo4LKJZTzZQ4V4yHH08xaCdxNLQ4mqUpimOpSkIED92MH7VmmI79G4QMbvgtTbM\\\\\\\/bVgxJzfMb3fm14lRwSbJIw5KGhgY2Zfp\\\\\\\/pM960uWGDYP2aKidEdrD5\\\\\\\/sEnJ1tmeeqFiqNu5QmGV7IMy\\\\\\\/L3D+wBll4xIPAxm\\\\\\\/cJkBCwOoAdO+Qk6Z1tLNsqCYRUjEU9k2wSps\\\\\\\/Z2fvM8rJUA9gsyvXlcXP7x3ttmpxl1uZTHE8OoWjGyJKbVhDYW8omqjzRIHMlhUEvUzAIqoDKjC3QSly7t7ngkq1PE6H2MnyQc131Kmeqs4aoGCuc5AG4godogTw46wF9eilw5YIwT4tF4bialOefzpzN34lQJ3ZuSnsRA5hSPxwGPfAbgI7yPcFWLkezmn7YLFhudR6xR4JB9YGaU+s2HDu8F8REDEKy8XLX\\\\\\\/ClvPUUG1J6u179R5r5suukoSisAbYi\\\\\\\/N8LRbsTHHJwQYZtoho7cP2Tlgav0LKiaIBtLdxoI\\\\\\\/F4cYR3xqfgIqGIsMAomgTVkCVCCF6RK4VNg1M+yBDD8CbLTxWjSOZkDCCLS8BrMWMs38qhRVjmo9XQa9DMQ40qKb6EwDhG1tE\\\\\\\/XyHjK6H93phupe1Gs8jfmfrDT7q3scuVyKtYokO91KzjsPgzwlJjIX+Edzj2pGC8ot1KmYmxgB0efUsVNvmbQhBYwBOXsf+FJ4OTDkTGql59qQeY1tVCLDcF0iYu\\\\\\\/sxcEg7R6mCgOi6\\\\\\\/OdXtCa\\\\\\\/yntQevjBip3GHGka9sPovt9E5zoRz72iPhyoM9wzRL8n9jsyeZfdRFLGcUr+6V21Ve8yEjcMK3Gswq2TgtMoiRtQnujwgSLiadkTL\\\\\\\/RsHvuvjTHQOlfwu41\\\\\\\/cdFaUBSmH61HmsuPdCG+S4FBwARl49m5xwKjRIZesfM4jwaW0qhzm6P00gYquQKtt8fse2Gs2\\\\\\\/reAb+VHmyDJjW4vDkq6l7A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GVGqro8n3UkFrx8xAfIMzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022571a63c8e965815ed2fd5db8d789c49a60ec7d42c0bd889e89d31c907da26c54\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459804 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/569" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tTI3MdBhHx0RQFzS2hy3TgQtiwQ3JR80bVEzMCUA1bI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/569" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Wp0cg9FSQ0JaWTl2xe4zi_tyslJ0uNvypHvwNXdD9eY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/559" title="Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/583" title="Adaptive Evolution of Genes Involved in the Regulation of Germline Stem Cells in Drosophila melanogaster and D. simulans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459804" id="node459804" data-pisa="ggg;5/4/569" data-pisa-master="ggg;g3.114.016097" data-apath="/ggg/5/4/569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marco Lopez-Cruz</span>, <span class="highwire-citation-author" data-delta="1">Jose Crossa</span>, <span class="highwire-citation-author" data-delta="2">David Bonnett</span>, <span class="highwire-citation-author" data-delta="3">Susanne Dreisigacker</span>, <span class="highwire-citation-author" data-delta="4">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="5">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="6">Ravi P. Singh</span>, <span class="highwire-citation-author" data-delta="7">Enrique Autrique</span> and <span class="highwire-citation-author" data-delta="8">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569-582;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016097 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459804" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marco Lopez-Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant, Soil and Microbial Sciences, Michigan State University (MSU), East Lansing, Michigan 4882,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BLopez-Cruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez-Cruz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BLopez-Cruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Bonnett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BBonnett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonnett%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BBonnett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University (KSU), 4011 Throckmorton Hall, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ravi P. Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BP.%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BP.%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Enrique Autrique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Enrique%2BAutrique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Autrique%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEnrique%2BAutrique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Epidemiology & Biostatistics and Statistics departments, Michigan State University 909 Fee Road, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gcampos@uab.edu" class="" data-icon-position="" data-hide-link-title="0">gcampos@uab.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/569" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459804" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/569.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459804" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/569.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459804" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/569.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459804" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/569.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459804" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459804/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection (GS) models use genome-wide genetic information to predict genetic values of candidates of selection. Originally, these models were developed without considering genotype × environment interaction(G×E). Several authors have proposed extensions of the single-environment GS model that accommodate G×E using either covariance functions or environmental covariates. In this study, we model G×E using a marker × environment interaction (M×E) GS model; the approach is conceptually simple and can be implemented with existing GS software. We discuss how the model can be implemented by using an explicit regression of phenotypes on markers or using co-variance structures (a genomic best linear unbiased prediction-type model). We used the M×E model to analyze three CIMMYT wheat data sets (W1, W2, and W3), where more than 1000 lines were genotyped using genotyping-by-sequencing and evaluated at CIMMYT’s research station in Ciudad Obregon, Mexico, under simulated environmental conditions that covered different irrigation levels, sowing dates and planting systems. We compared the M×E model with a stratified (<em>i.e.</em>, within-environment) analysis and with a standard (across-environment) GS model that assumes that effects are constant across environments (<em>i.e.</em>, ignoring G×E). The prediction accuracy of the M×E model was substantially greater of that of an across-environment analysis that ignores G×E. Depending on the prediction problem, the M×E model had either similar or greater levels of prediction accuracy than the stratified analyses. The M×E model decomposes marker effects and genomic values into components that are stable across environments (main effects) and others that are environment-specific (interactions). Therefore, in principle, the interaction model could shed light over which variants have effects that are stable across environments and which ones are responsible for G×E. The data set and the scripts required to reproduce the analysis are publicly available as Supporting Information.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/multienvironment" class="hw-term hw-article-keyword hw-article-keyword-multienvironment" rel="nofollow">multienvironment</a></li><li class="kwd"><a href="/keyword/genomic-best-linear-unbiased-prediction-gblup" class="hw-term hw-article-keyword hw-article-keyword-genomic-best-linear-unbiased-prediction-gblup" rel="nofollow">genomic best linear unbiased prediction (GBLUP)</a></li><li class="kwd"><a href="/keyword/marker-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-marker-&#xD7;-environment-interaction" rel="nofollow">marker × environment interaction</a></li><li class="kwd"><a href="/keyword/international-bread-wheat-screening-nursery" class="hw-term hw-article-keyword hw-article-keyword-international-bread-wheat-screening-nursery" rel="nofollow">International Bread Wheat Screening Nursery</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">The presence of genotype × environment(G×E) interactions in agricultural experiments usually is expressed as changes in the relative performance of genetic materials across environments; this can manifest as modifications of the ranking of genotypes across environments or simply as changes in the absolute difference in performance between pairs of genotypes. Accounting for G×E has always been a concern in the analysis of multienvironment plant breeding trials, and several models have been proposed and used for describing the mean response of genotypes over environments and for studying and interpreting G×E in agricultural experiments (<em>e.g.</em>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Yates and Cochran 1938</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Finlay and Wilkinson, 1963</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eberhart and Russell 1966</a>).</p><p id="p-4">The statistical treatment of G×E has evolved over time due to the development of statistical methods and because of changes in the information available, including the increased availability of DNA markers and of precise environmental information. Some approaches deal with G×E implicitly, without explicitly modeling gene × environment interactions; these include some of the early treatment of G×E (<em>e.g.</em>, the joint-regression analysis of <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Yates and Cochran 1938</a>), as well as more modern methods such as the multivariate pedigree- or marker-based models where G×E is modeled using structured or unstructured covariance functions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Piepho, 1997</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">1998</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Smith <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2007</a>). These approaches have proved to be effective for exploiting G×E; however, they do not shed light on the underlying basis of G×E (<em>e.g.</em>, the relative contribution of different genetic regions to stability and to G×E). When genomic and environmental covariate data are available, G×E can be modeled explicitly by means of marker × environment interactions (M×E). This approach was first used with sparse marker data in quantitative trait loci (QTL) analysis (QTL×E <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Moreau <em>et al.</em> 2004</a>) and also in multilocus models with markers that exhibited “significant” association with the trait of interest (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Boer <em>et al.</em> 2007</a>). The QTL×E approach has been further extended to multitrait, multienvironment settings (<em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Malosetti <em>et al.</em> 2004</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2008</a>).</p><p id="p-5">Recent developments in genotyping and sequencing technologies have made it possible to use dense genotypic information for genomic selection (GS) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Meuwissen <em>et al.</em> 2001</a>). Empirical evidence obtained with plant and animal breeding data has demonstrated that GS can outperform the prediction accuracy of pedigree-based methods or that of models based on a reduced number of loci (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pérez-Rodríguez <em>et al.</em> 2012</a>). This has prompted the relatively fast adoption of GS in plant and animal breeding. GS models originally were developed for a single trait evaluated in a single environment, and most analyses published so far are based on within-environment analyses.</p><p id="p-6">Recently, several studies have proposed using GS models that accommodate G×E. For instance, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em>(2012)</a> extended the single-trait, single-environment genomic best linear unbiased prediction(GBLUP) model to a multienvironment context and reported important gains in prediction accuracy with the multienvironment model relative to single-environment analysis. More recently, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heslot <em>et al.</em> (2014)</a> and <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jarquin <em>et al.</em> (2014)</a> considered modeling G×E using both genetic markers and environmental covariates. These studies also showed that modeling G×E can give substantial gains in prediction accuracy.</p><p id="p-7">Following ideas originally used for QTL analysis in multienvironment trials(<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Van Eeuwijk <em>et al.</em> 2005</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Boer <em>et al.</em> 2007</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Malosetti <em>et al.</em> 2004</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">2008</a>), we present GS models that accommodate G×E by explicitly modeling interactions between all available markers and environments. Relative to multivariate approaches where G×E is modeled using covariance parameters(<em>e.g.</em>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Smith <em>et al.</em> 2005</a>), the M×E approach has advantages and disadvantages. First, the M×E models presented here can be easily implemented using existing software for GS. Second, the model can be implemented using both shrinkage methods as well as variable selection methods. Third, the M×E model decomposes effects into components that are common across environments (stability) and environment-specific deviations; this information, which is not provided by standard multienvironment mixed models, can be used to identify genomic regions whose effects are stable across environments and others that are responsible for G×E. On the other hand, the M×E model imposes restrictions on the patterns of G×E, and, for reasons that we discuss in this article, the model is best suited for the joint analysis of positively correlated environments.</p><p id="p-8">In this study, we applied the M×E model to extensive data sets where wheat lines were evaluated under contrasting environmental conditions in replicated field trials. This allowed us to identify under which conditions the M×E model is most effective. We show theoretically and demonstrate empirically that the magnitude of the main and interaction variance is directly related to the phenotypic correlation between environments and that the M×E model performs best when the set of environments analyzed showed positive and similar correlations. Indeed, when the set of environments analyzed had moderate or high positive correlations, the M×E model yielded substantial gains in prediction accuracy relative to an across-environment GS model that assumes homogeneity of effects across environments, and, depending on the prediction problem, it either performed similarly or outperformed the stratified (<em>i.e.</em>, within-environment) analyses. In the rest of this article, we describe the methods used and present empirical results obtained when the M×E model was applied to three wheat data sets. We also provide, as online materials, scripts that implement the interaction models using the BGLR R-package (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez 2014</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">The data used in this study are from CIMMYT’s Global Wheat Program and consist of a set of wheat lines evaluated under managed environmental conditions; these conditions were designed to simulate target mega-environments. The wheat lines included in this study were later part of the 45th, 46th, and 47th International Bread Wheat Screening Nurseries and distributed worldwide.</p><p id="p-10">Three files containing phenotypic and genotypic information on the three data sets used in this study (45th, 46th and 47th International Bread Wheat Screening Nurseries) are provided as Supporting Information, <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS1.rdata">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS2.rdata">File S2</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS3.rdata">File S3</a>, respectively.</p><div id="sec-2" class="subsection"><h3>Phenotypic data</h3><p id="p-11">The phenotypic data consisted of adjusted grain yield(ton/ha) records collected during three evaluation cycles (W1: cycle 2010−2011, N = 732; W2: cycle 2011−2012, N = 672; and W3: cycle 2012−2013, N = 811); each cycle included a different set of advanced breeding lines. All trials were established at CIMMYT’s main wheat breeding station at Cd. Obregon, Mexico. The experimental design was an alpha-lattice with three replicates per line and environment. Wheat lines were evaluated under three irrigation regimes(2i = two irrigations giving moderate drought stress, 5i = five irrigations representing an optimally irrigated crop, and 0i = no irrigation or drip irrigation, representing high drought stress), two planting systems (B = bed planting; F = planting on the flat) and two planting dates (N = normal and H = late, simulating heat at the grain-filling stage). In the 2i and 5i regimes, irrigation was applied without measuring soil moisture, and each irrigation added 100 mm of water. Some of the trials were managed using no-tillage(hereinafter denoted as Z). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> gives the number of phenotypic records per simulated environment and cycle. The phenotype used in the analysis was the best linear unbiased estimate of grain yield obtained from a linear model applied to the alpha-lattice design of each cycle-environment combination.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465814/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465814/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of phenotypic records per cycle (W1, W2, and W3) and environment</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotypic data</h3><p id="p-13">Genotypes were derived using genotyping-by-sequencing technology (GBS; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Poland <em>et al.</em> 2012</a>). GBS markers with a minor allele frequency lower than 0.05 were removed. As is typical of GBS genotypes, all markers had a high incidence of uncalled genotypes. In our quality control pipeline, we applied thresholds for incidence of missing values aimed at maintaining relatively large and similar numbers of markers per data set. To this end, we removed markers with more than 60%(W1 and W2) or 80% (W3) missing values; this left 15,744 (W1 and W2) and 14,217(W3) GBS markers available for analysis. Finally, only lines with more than 2000 called GBS markers were used in the data analysis; this left 693 (W1), 670 (W2), and 807(W3) lines.</p></div><div id="sec-4" class="subsection"><h3>Statistical models</h3><p id="p-14">For each evaluation cycle (W1, W2, and W3), we considered three approaches: (i) a stratified analysis obtained by regressing phenotypes on markers separately in each environment (we refer to this approach, indistinctively, as to single-environment, within-environment or stratified analysis); (ii) a combined analysis based on a GS model where marker effects are assumed to be constant across environments (<em>i.e.</em>, ignoring M×E) (hereinafter referred to as the “across-environment” model); and (iii) using a M×E model that allows analyzing data from multiple environments jointly and accounts for G×E. Each of these approaches is discussed in the sections to follow.</p><div id="sec-5" class="subsection"><h4>Stratified analysis:</h4><p id="p-15">This model is obtained by regressing the phenotype vector containing the records available in the <em>j<sup>th</sup></em> environment, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-1.gif"/></noscript></span></span>, where <em>i</em> indexes lines (individuals) and <em>j</em> indexes environments, on markers using a linear model in the form:<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-2.gif"/></noscript></span></span>, (<em>i</em> = 1,2,…,<em>n</em> individuals; <em>j</em> = 1,2,…<em>s</em> environments; <em>k</em> = 1,2,…,<em>p</em> markers) or, in matrix notation,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(1a)</span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-4.gif"/></noscript></span></span> is an intercept, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-5.gif"/></noscript></span></span> is a matrix of marker-centered and standardized genotypes(<em>i.e.</em>, each marker was centered by subtracting the mean and standardized by dividing by the sample standard deviation), <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-6.gif"/></noscript></span></span> is a vector of marker effects and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-7.gif"/></noscript></span></span> is a vector of model residuals. Note that, in a full-factorial design where all lines are evaluated in all environments, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-8.gif"/></noscript></span></span>. Following the standard assumptions of the GBLUP model (<em>e.g.</em>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Vanraden, 2007</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">2008</a>), marker effects and model residuals were assumed to be independent of each other and both normally distributed: <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-9.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-10.gif"/></noscript></span></span>. Setting <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-11.gif"/></noscript></span></span> we have that the aforementioned model also can be represented as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-12.gif"/></noscript></span> <span class="disp-formula-label">(1b)</span></span>with <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-13.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-14.gif"/></noscript></span></span>obtained using the cross-product of (centered and standarized) marker genotypes and scaled by dividing by the number of markers. Because all markers were standardized to a unit variance, this gives an average diagonal value of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-15.gif"/></noscript></span></span> equal to one.</p><p id="p-16">Box 1a in <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS4.pdf">File S4</a> provides an R-script that implements the single-environment model described previously using the BGLR R-package (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez 2014</a>).</p></div><div id="sec-6" class="subsection"><h4>Across-environment GBLUP model:</h4><p id="p-17">another approach consists of assuming that effects of markers are the same across environments, that is: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-16.gif"/></noscript></span></span>; therefore the regression model <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">(1b)</a> becomes (assuming s = 3, for ease of notation):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-17.gif"/></noscript></span> <span class="disp-formula-label">(2a)</span></span>In a GBLUP-context, one will assume <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-18.gif"/></noscript></span></span> and the aforementioned model can be represented as a random effect model as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-19.gif"/></noscript></span> <span class="disp-formula-label">(2b)</span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-20.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-21.gif"/></noscript></span></span>, where<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-22.gif"/></noscript></span></span>is a marker-derived genomic relationship matrix. It is worth noting that, for balanced data, the model of expression <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">(2a)</a> is equivalent to fitting a genomic regression model using the average performance of each line across environments as a phenotype.</p><p id="p-18">Box 2a in <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS4.pdf">File S4</a> provides an R-script that implements the across-environment model described above using the BGLR R-package (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez 2014</a>).</p></div><div id="sec-7" class="subsection"><h4>M×E GBLUP model:</h4><p id="p-19">In the model in expression <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">(1a)</a>, marker effects (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-23.gif"/></noscript></span></span>) are estimated separately for each environment; therefore, in this model there is no borrowing of information across environments. On the other hand, in the model in expression <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">(2a)</a>, all data are used to estimate marker effects; however, in that model borrowing of information is achieved by assuming that effects are constant across environments. We now consider an interaction model that aims at benefiting from borrowing information across environments while allowing marker effects to change across environments. In the M×E model, the effect of the <em>k<sup>th</sup></em> marker on the <em>j<sup>th</sup></em> environment (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-24.gif"/></noscript></span></span>) is described as the sum of an effect common to all environments (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-25.gif"/></noscript></span></span>), plus a random deviation (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-26.gif"/></noscript></span></span>) peculiar to the <em>j<sup>th</sup></em> environment, that is <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-27.gif"/></noscript></span></span>. Therefore, the equation for data from the <em>j<sup>th</sup></em> environment becomes <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-28.gif"/></noscript></span></span>, or, in matrix notation and assuming, for ease of notation, only three environments,<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-29.gif"/></noscript></span> <span class="disp-formula-label">(3a)</span></span>where the vectors of main and interaction effects and model residuals were all assumed to be normally distributed, specifically: <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-30.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-32.gif"/></noscript></span></span>. The aforementioned model can be represented as a two-variance component GBLUP model, specifically, letting <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-33.gif"/></noscript></span></span>, <span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-34.gif"/></noscript></span></span>the model can be represented as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-35.gif"/></noscript></span>, <span class="disp-formula-label">(3b)</span></span>where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-36.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-37.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-38.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-39.gif"/></noscript></span></span> is as described previously and<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-40.gif"/></noscript></span></span>In this model, the main effect (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-41.gif"/></noscript></span></span>) allows borrowing information between environments (through the off-diagonal blocks of <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-42.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-43.gif"/></noscript></span></span> captures environment-specific effects. The relative importance of these two terms is determined by the corresponding variance components that are inferred from the data.</p><p id="p-20">Box 3a in <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS4.pdf">File S4</a> provides an R-script that implements the M×E model described previously using the BGLR R-package (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez 2014</a>).</p></div></div><div id="sec-8" class="subsection"><h3>Software</h3><p id="p-21">The aforementioned models can be implemented using standard software for GS. For our implementation, we used the R (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">R Core Team 2013</a>) package Bayesian generalized linear regression (BGLR, <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez 2014</a>). This software does not allow fitting group-specific error variances; therefore, in our interaction model and in our across-environment analyses, we fit the models assuming homogeneous error variances across environments. The code used to implement these models is provided in the Supporting Information; technical details and several examples of the use of the package for genome-enabled prediction can be found in <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez (2014)</a>. We used a Bayesian model assuming Gaussian priors for the marker effects. The BGLR package assigns scaled-inverse χ<sup>2</sup> densities to the variance parameters whose hyperparameters were given values using the default rules implemented in BGLR, which assign 5 degrees of freedom and calculates the scale parameter based on the sample variance of the phenotypes. Further details are given in <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodriguez (2014)</a>.</p></div><div id="sec-9" class="subsection"><h3>Statistical analysis</h3><p id="p-22">The aforementioned models were fitted to data from each of the cycles (W1−W3), separately. For each cycle data we performed analysis: (i) within-environment (see expression <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">1a</a>), or (ii) by pairs of environments or, (iii) using data from all environments together. These last two approaches were implemented either using the model in expression <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">(2a)</a> or the interaction model expression <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">(3a)</a>.</p><p id="p-23">Models were fitted to each of the full data sets to derive estimates of variance components. Subsequently, we assessed prediction accuracy using training-testing (<em>i.e.</em>, TRN-TST) random partitions (see below). For this validation procedure, all the parameters of the models, including variance components, were re-estimated from TRN data in each of the TRN-TST partitions. In all cases, inferences and predictions were based on 55,000 samples collected from the posterior distribution after discarding 5000 samples for burn-in.</p><p id="p-24">Prediction accuracy was assessed using 50 TRN-TST random partitions; we used this approach because with a replicated TRN-TST design one can obtain as many partitions as one needs and this allows estimating SEs of estimates of prediction accuracy more precisely than with a cross-validation approach. Following <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> (2012)</a>, we considered two different prediction problems. First (CV1), we assessed prediction accuracy of the models when TRN and TST data consist of disjoint sets of lines; this approach mimics the prediction problem faced by breeders when lines have not been evaluated in any field trials. To generate TRN and TST sets in CV1 we simply assigned completely at random 70% of the lines to TRN and the remaining 30% to TST.</p><p id="p-25">Second (CV2), we considered the problem of predicting the performance of lines in environments in which lines have not been evaluated. This validation design mimics the prediction problem faced by breeders in incomplete field trials where lines are evaluated in some but not all target environments. TRN-TST partitions for CV2 were obtained as follows: in each data set (<em>w = 1,2,3</em>), each line (<em>i = 1,…</em>,<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-44.gif"/></noscript></span></span>) had records in <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-45.gif"/></noscript></span></span> environments (<em>j = 1,…</em>,<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-46.gif"/></noscript></span></span> in W1 and W2, and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-47.gif"/></noscript></span></span> in W3); therefore, the total number of records available per data set was <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-48.gif"/></noscript></span></span>. To select the entries in the TST data set, we first chose <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-49.gif"/></noscript></span></span> IDs (<em>i.e.</em>, lines) at random and subsequently randomly picked one environment per line from the index <em>j = 1,…</em>,<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-50.gif"/></noscript></span></span>. The resulting cells <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-51.gif"/></noscript></span></span> were assigned to the TST data set, and the ones not selected through this algorithm were used for model TRN. Lines were sampled without replacement if <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-52.gif"/></noscript></span></span> and with replacement otherwise. Boxes 4a and 4b in <a href="http://www.g3journal.org/content/suppl/2015/02/06/g3.114.016097.DC1/FileS4.pdf">File S4</a> provide the R-code used to generate TRN-TST partitions in CV1 and CV2, and Boxes 5 and 6 illustrate how to fit models and evaluate prediction accuracy for a TRN-TST partition.</p><p id="p-26">For each TRN-TST partition, models were fitted to the TRN data set and prediction accuracy was assessed by computing Pearson’s product-moment correlation between predictions and phenotypes in the TST data set, within environment. The same TRN-TST partitions were used to assess the prediction accuracy of each of the models; this yielded 50 correlation estimates for each model, data set, and we report the average correlation (across partitions) and SEs.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-27"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows box-plots of adjusted yield per data set and environmental condition. As expected, average yield increased with the number of irrigation events and, other factors being equal, late planting (H) produced lower yields than normal planting (N). In all cases, the empirical distribution of grain yield within data set and environment was reasonably symmetric.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/569/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box-plot of adjusted grain yield by environment and cycle. Environments are denoted by a sequence of codes: 0i, 2i, and 5i denote the number of irrigation cycles; B/F denotes whether the planting system was &#x201C;bed&#x201D; (B) or &#x201C;flat&#x201D; (F); N/H denotes whether planting date was normal (N) or late (H, simulating heat); Z indicates no tillage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408125764" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box-plot of adjusted grain yield by environment and cycle. Environments are denoted by a sequence of codes: 0i, 2i, and 5i denote the number of irrigation cycles; B/F denotes whether the planting system was &#x201C;bed&#x201D; (B) or &#x201C;flat&#x201D; (F); N/H denotes whether planting date was normal (N) or late (H, simulating heat); Z indicates no tillage.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/569/F1.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/569/F1.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/569/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/569/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Box-plot of adjusted grain yield by environment and cycle. Environments are denoted by a sequence of codes: 0i, 2i, and 5i denote the number of irrigation cycles; B/F denotes whether the planting system was “bed” (B) or “flat” (F); N/H denotes whether planting date was normal (N) or late (H, simulating heat); Z indicates no tillage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> gives the SDs of grain yield for each environment-cycle combination and the (empirical) phenotypic correlations of adjusted grain yield across environments, within cycle. The average SD of grain yield was rather stable across environments, ranging from 0.41 to 0.65. In the first cycle (W1), correlations across environments were moderately positive, ranging from 0.22 to 0.53 and, as one would expect, environments with similar irrigation levels (<em>e.g.</em>, 0i and 2i) exhibited greater correlations than environments with very different numbers of irrigations (<em>e.g.</em>, 0i <em>vs.</em> 5i). In the second evaluation cycle (W2), three environments received five irrigations (5i) and one received none (0i). The correlations among environments with five irrigations were also moderately positive, ranging from 0.33 to 0.41. However, the correlation between drought environments (0i) and environments having five irrigations was almost zero (−0.05). Finally, in the third evaluation cycle (W3), the two environments with five irrigations and normal planting dates (5iBN and 5iFN) were positively correlated (0.55); however, environments with different irrigation levels or different planting dates (N/H) showed very low, and even negative, correlations. Interestingly, environments 0iFN and 5iBH, which differ across all factors, showed a moderate sample correlation (0.30).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465882/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465882/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">SD (diagonal) and sample correlation (lower-triangular) between grain yields evaluated under different environmental conditions, by cycle</span><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h3>Estimates of variance components</h3><p id="p-32"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> provide estimates of variance components per model and environment for cycles W1, W2, and W3, respectively. The estimates reported in these tables correspond to the full-data analyses.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465818/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465818/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimates of variance components (estimated posterior SD) by model and environment, cycle 2010−2011 (W1)</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465859/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465859/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Estimates of variance components (estimated posterior SD) by model and environment, cycle 2011−2012 (W2)</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465892/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465892/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Estimates of variance components (estimated posterior standard deviation) by model and environment, cycle 2012-13 (W3)</span><div class="sb-div caption-clear"></div></div></div><div id="sec-12" class="subsection"><h4>Stratified analysis:</h4><p id="p-39">The proportion of variance explained by the regression on markers (R<sup>2</sup> computed based on estimates of variance components) estimated from the stratified (single-environment) analysis ranged from moderate (∼0.4) to high (∼0.7); in general, models fitted data better in W3 (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>) than in W1 or W2.</p></div><div id="sec-13" class="subsection"><h4>Across-environment model:</h4><p id="p-40">The estimated residual variance of the across-environment model was typically larger (and consequently the R<sup>2</sup> was lower) than that of the interaction model indicating that a sizable proportion of the G×E went to the residual of the across-environment model.</p></div><div id="sec-14" class="subsection"><h4>M×E model:</h4><p id="p-41">In the interaction models, the total genomic variance can be partitioned into a main effect and an interaction component. This partition showed that the relative importance of the main effect was greater when the environments analyzed jointly were positively correlated. On the other hand, as one would expect, when the environments analyzed jointly had low correlations, the estimated interaction variance was greater. For instance, in W1, the analysis of pairs of environments exhibiting sample phenotypic correlations smaller than 0.3 (0iBN + 5iBNZ, 0iBN + 5iFN, and 5iBNZ + 5iFN) yielded estimates of variance components in the M×E model where the main effect explained less than 50% of the total genomic variance, computed as the sum of the main effect plus interaction variance estimates (see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). On the other hand, the pairs of environments showing correlations larger than 0.3 (2iBN + 5iBNZ and 2iBN + 5iFN) gave estimates of variance components where the main effect explained between 50 and 70% of the genomic variance. Finally, in W1, the pair of environments with the largest sample phenotypic correlation (0iBN + 2iBN) had estimates of variance components such that the main effect explained about 80% of the total genomic variance.</p><p id="p-42">Similar patterns were observed in W2 and W3. Indeed, in W2, the main effects of markers explained more than 60% of the genomic variance for pairs of environments having sample phenotypic correlations greater than 0.33 (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>); on the other hand, in the two environments showing a low correlation (0iFN + 5iBN), the main effect explained only about 10% of the genomic variance. In W3 data set, pairs of environments with sample phenotypic correlations smaller than 0.1 had an estimated proportion of genomic variance explained by main effects that was smaller than 0.2. At the other extreme, for the pair of environments showing the greatest correlation (5iBN + 5iFN), the proportion of variance explained by main effects was close to 0.8 (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Finally, as expected, in the joint analysis of all environments, the variance of the main effect was largest in W1 and W2 (0.316 and 0.436, respectively), where several pairs of environments had sample phenotypic correlations that were moderately high, and considerably smaller in W3 (0.113), where many pairs of environments had phenotypic correlations that were negative or close to zero.</p></div></div><div id="sec-15" class="subsection"><h3>Assessment of prediction accuracy</h3><p id="p-43">The average correlation and the estimated SD (both computed using 50 TRN-TST partitions, each with 70% of records in the TRN data set and 30% in TST data set) obtained in CV1 are reported in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>, <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>, and <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> and those obtained in CV2 are reported in <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 9</a>, <a id="xref-table-wrap-10-1" class="xref-table" href="#T10">Table 10</a>, and <a id="xref-table-wrap-11-1" class="xref-table" href="#T11">Table 11</a>. A summary of these results is given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. As one would expect, the levels of prediction accuracy (correlation) were slightly greater in CV2 than in CV1. In CV1, the stratified analysis and the interaction model performed similarly (average correlation of 0.48 and 0.47 for the stratified and interaction models, respectively), and the across-environment analysis was the worst one (the average correlation in CV1 was 0.33, that is about 30% lower correlation than the stratified analysis or the interaction model). On the other hand, in CV2, the interaction model gave the greatest levels of prediction accuracy (average correlation 0.53), this method was followed by the stratified analysis (average correlation of 0.48, that is, about 10% less, in the scale of correlation, than the interaction model), and the worst performing method was the across-environment analysis (this method had an average correlation in CV2 of 0.38, that is about 27% less than the interaction model).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465831/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465831/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Estimated prediction accuracy: correlation between predicted and adjusted grain yield, averaged over 50 TRN-TST partitions), cycle 2010−2011 (W1), CV1</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465784/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465784/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Estimated prediction accuracy (correlation between predicted and adjusted grain yield, averaged over 50 TRN-TST partitions), cycle 2011-2012 (W2), CV1</span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465829/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465829/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Estimated prediction accuracy (correlation between predicted and adjusted grain yield, averaged over 50 TRN-TST partitions): cycle 2012−2013 (W3), CV1</span><div class="sb-div caption-clear"></div></div></div><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465787/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465787/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9</span> <span class="caption-title">Estimated prediction accuracy: correlation between predicted and adjusted grain yield, averaged over 50 TRN-TST partitions, cycle 2010−2011(W1), CV2</span><div class="sb-div caption-clear"></div></div></div><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465890/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465890/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 10</span> <span class="caption-title">Estimated prediction accuracy (correlation between predicted and adjusted grain yield, averaged over 50 TRN-TST partitions, cycle 2011-2012 (W2), CV2</span><div class="sb-div caption-clear"></div></div></div><div id="T11" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465857/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465857/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 11</span> <span class="caption-title">Estimated prediction accuracy (correlation between predicted and adjusted grain yield, averaged over 50 TRN-TST partitions, cycle 2012-2013, (W3), CV2</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/569/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between phenotypes and predictions (average over 50 TRN-TST partitions) by model (stratified, interaction, and across-environment analysis), validation design (CV1, left, CV2 right), data set and simulated environment (horizontal axis). TRN-TST, training-testing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408125764" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between phenotypes and predictions (average over 50 TRN-TST partitions) by model (stratified, interaction, and across-environment analysis), validation design (CV1, left, CV2 right), data set and simulated environment (horizontal axis). TRN-TST, training-testing.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/569/F2.medium.gif" width="440" height="167"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/569/F2.medium.gif" width="440" height="167"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/569/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/569/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465801" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-68" class="first-child">Correlation between phenotypes and predictions (average over 50 TRN-TST partitions) by model (stratified, interaction, and across-environment analysis), validation design (CV1, left, CV2 right), data set and simulated environment (horizontal axis). TRN-TST, training-testing.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-16" class="subsection"><h4>Stratified analysis:</h4><p id="p-69">The within-environment analysis yielded prediction correlations ranging from moderately low (0.307 for environment 5iFN in W3) to moderately high (0.630 for environment 5iBH in W3).</p></div><div id="sec-17" class="subsection"><h4>Across-environment analysis:</h4><p id="p-70">Overall, this method was the worst performing one. In CV1 the across-environment analysis performed worse than the stratified analysis and the interaction model; this in every environment and dataset. In CV2, the joint analysis of data from different environments ignoring G×E performed worse than the stratified analysis when the pairs of environments analyzed together had a correlation lower than 0.3; however, the across-environment model tended to outperform the stratified analysis whenever the correlation between environments was larger than 0.4. On the other hand, the across-environment model was systematically outperformed by the M×E model both in CV1 and in CV2. In CV2, the difference in prediction accuracy between these two methods was low when the pairs of environments analyzed were positively correlated (<em>e.g.</em>, in W1, environments 0iBN and 2iBN; see <a id="xref-table-wrap-9-2" class="xref-table" href="#T9">Table 9</a>) and large, and in favor of the M×E model, when the set of environments analyzed were negatively correlated (<em>e.g.</em>, in W3, in the joint analysis of 0iFN and 5iBN).</p></div><div id="sec-18" class="subsection"><h4>M×E model:</h4><p id="p-71">As previously stated, in CV1, the M×E model performed similarly to the stratified analysis. However, in CV2, the joint analysis of all environments using the M×E model gave, relative to the stratified analysis, average gains in prediction accuracy ranging from 4.7 to 12.1% in W1, 4.1 to 29.6% in W2 and −0.2 to 22.3% in W3. The patterns of gain/loss in prediction accuracy achieved in CV2 with the M×E model, relative to the stratified analysis, were directly related to the correlation between environments and to the proportion of genomic variance explained by the main effects. Environments exhibiting positive correlations with other environments benefited greatly from the use of multienvironment models (<em>e.g.</em>, 0iBN or 2iBN in W1, 5iBH and 5iFN in W2, and 5iFN and 5iBN in W3). For these environments in all 50 partitions, the multienvironment model fitted to all environments jointly had higher prediction accuracy than the within-environment model (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-72">In CV2, the joint analysis of all environments using a M×E model gave, in general, a greater prediction accuracy than the one achieved with analyses of pairs of environments; this was particularly clear in W1 and W2 (<a id="xref-table-wrap-9-3" class="xref-table" href="#T9">Table 9</a> and <a id="xref-table-wrap-10-2" class="xref-table" href="#T10">Table 10</a>, respectively), with the only exception of 5iBN in W2, where the average prediction accuracy was slightly higher in the bivariate analysis than in the multienvironment model applied to all environments jointly. The situation in W3 was slightly different; here, the predictive performance of analyses based on pairs of environments was similar to that of the joint multienvironment analysis of all conditions (<a id="xref-table-wrap-11-2" class="xref-table" href="#T11">Table 11</a>).</p></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-73">Several studies have documented the benefits of using multienvironment models, relative to single-environment analysis (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dawson <em>et al.</em> 2013</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Jarquin <em>et al.</em> 2014</a>). Multienvironment analysis can model G×E interactions using covariance functions (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>), markers and environmental covariates (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Jarquin <em>et al.</em> 2014</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Heslot <em>et al.</em> 2014</a>) ,or by modeling M×E interactions. In this article, we adapted this approach, previously used in QTL models by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Moreau <em>et al.</em>(2004)</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Van Eeuwijk <em>et al.</em> (2005)</a>, <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Boer <em>et al.</em> (2007)</a>, and <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Malosetti <em>et al.</em> (2004</a>, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">2008)</a>, to whole-genome regression models where phenotypes were regressed on large numbers of genome-wide markers.</p><p id="p-74">Relative to the standard multienvironment models with structured or unstructured covariances such as those used by <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> (2012)</a>, the M×E model has advantages and disadvantages. On one hand, the interaction model: (i) is easy to implement using existing software for GS; (ii) leads to a decomposition of marker effects into components that are stable across environments (main effects) and environment-specific deviations (interactions) that can shed light on which genomic regions are most responsible for G×E; and (iii) can be implemented with any of the priors commonly used in GS, including not only shrinkage methods such as the GBLUP, but also variable selection methods. On the other hand, the M×E model imposes restrictions on co-variance patterns: the covariance between environments is forced to be positive and constant across pairs of environments. Therefore, the M×E model is best suited for the joint analysis of sets of environments that are positively and similarly correlated. In those cases, the parsimony of the M×E model can be advantageous; however, the pattern may be too restrictive in cases where the genomic variance cannot be approximated with the structure imposed by the interaction model (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Meyer and Kirkpatrick 2008</a>).</p><p id="p-75">Among the three data sets considered here, two of them, (W1 and W2) had patterns of phenotypic (sample) correlations with moderately positive correlations between pairs of environments, while the third one (W3) exhibited a great deal of G×E, with some pairs of environments exhibiting either null or negative sample phenotypic correlations. The environments included in our data sets were managed field conditions aiming to approximate three basic target sets of environments (mega-environments) comprising three main agroclimatic regions previously defined and widely used by CIMMYT’s Global Wheat Breeding Program (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Braun <em>et al.</em> 1996</a>). These three agroclimatic regions are represented by megaenvironment 1 (low rainfall and irrigated), megaenvironment 4 (drought), and megaenvironment 5 (heat). Results indicated that environments clustered more along the lines of full irrigation, drought, and heat than based on whether they include beds or flat planting systems, and zero or conventional tillage.</p><div id="sec-20" class="subsection"><h3>Variance components</h3><p id="p-76">The M×E model fitted the data much better than the across-environment model that ignored G×E. Furthermore, the estimates of variance components from the M×E model indicated that the proportion of genomic variance explained by the main effect of markers is directly related to the (sample empirical) phenotypic correlation between environments. Analytically, under the assumptions of the M×E model described and used in this article, if <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-54.gif"/></noscript></span></span> are the equations for the phenotype of the <em>i<sup>th</sup></em> line in environments <em>1</em> and <em>2</em>, respectively, then the phenotypic correlation can be expressed as a function of variance components, indeed <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4/569/embed/mml-math-55.gif"/></noscript></span></span>. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> displays the estimated phenotypic correlations derived using estimates of variance components obtained from analyses based on pairs of environments (see <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>) <em>vs.</em> the observed sample phenotypic correlations. Overall, the estimated phenotypic correlation based on variance components was linearly related to the sample phenotypic correlation for pairs of environments having positive sample phenotypic correlations. However, as the sample phenotypic correlation approached zero or became negative, the relationship flattened out. This happens because, in the interaction model, the covariance is represented by the variance of the main effects and, therefore, it is bound to be non-negative.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/569/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated phenotypic correlation (based on estimates of variance components) vs. sample phenotypic correlation. The estimated phenotypic correlation was obtained from estimates of variance components (see Table 3, Table 4, and Table 5) from the bivariate analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-408125764" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimated phenotypic correlation (based on estimates of variance components) &lt;em&gt;vs.&lt;/em&gt; sample phenotypic correlation. The estimated phenotypic correlation was obtained from estimates of variance components (see Table 3, Table 4, and Table 5) from the bivariate analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/569/F3.medium.gif" width="440" height="410"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/569/F3.medium.gif" width="440" height="410"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/569/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/569/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-77" class="first-child">Estimated phenotypic correlation (based on estimates of variance components) <em>vs.</em> sample phenotypic correlation. The estimated phenotypic correlation was obtained from estimates of variance components (see <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>) from the bivariate analysis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Prediction accuracy</h3><p id="p-78">The prediction analysis conducted in this study yielded levels of accuracy (correlation between phenotypes and predicted genomic values) that are consistent with previous reports for grain yield prediction accuracy using single- and multi-environment models (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>). Overall, the interaction model was either the best performing method (CV2) or performed close to the best performing method (in CV1 the stratified analysis and the interaction model performed similarly). These results are consistent with those of previous studies (<em>e.g.</em>, <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Jarquin <em>et al.</em> 2014</a>) that have used similar validation designs (CV1 and CV2) and have reported similar predictive performance of the stratified and multivariate analysis in CV1, and clear superiority of the multivariate approach in CV2. Additionally, we also considered an across-environment analysis that ignores G×E. In our study this approach was clearly the worst-performing one; this finding highlights the importance of considering G×E when analyzing multi-environment data.</p><p id="p-79">The gains in prediction accuracy obtained in CV2 with the M×E model were directly related to the correlations between environments. Considering environments that had positive phenotypic correlations among them, the use of the M×E model yielded in CV2 gains, relative to the stratified analysis, that were either moderate (on the order of 5%) or very substantial (on the order of 29%). In CV2 the only cases where the stratified analysis was better than the interaction model are those based on the joint analysis of pairs of environments that had close to null or negative correlations (<em>e.g.</em>, 5iBH in W3). This happened because, as discussed previously, the interaction model forces the covariance between environments to be non-negative.</p><p id="p-80">In prediction problems such as that of CV2, the superiority of the M×E relative to the stratified analysis can be attributed to the fact that the M×E model allows borrowing of information within line across environments, that is: when deriving predictions for a given line, the M×E model benefits from records from the same line collected in correlated environments (this borrowing of information also happens in the across-environment GBLUP; however, in the across-environment GBLUP borrowing of information within line across environments is achieved at the expense of forcing the effects to be constant across environments). This feature of the M×E model can be exploited in prediction problems such as CV2; however, such borrowing of information within line is not possible in CV1 and, consequently, the M×E model performs similarly to the stratified analysis for prediction of performance of lines that have no phenotypic records.</p></div><div id="sec-22" class="subsection"><h3>How many environments?</h3><p id="p-81">The interaction model can be applied to all available environments or to other sets (<em>e.g.</em>, pairs) of environments. For data sets such as W1, where all environments showed positive correlations of similar magnitude, the joint analysis of all environments was clearly superior to analyses based on pairs of environments. However, in data sets exhibiting complex co-variance patterns (such as W3), joint analysis of all environments using an interaction model imposes inadequate restrictions on co-variance patterns and, consequently, bivariate analysis seems more appropriate.</p></div><div id="sec-23" class="subsection"><h3>Extensions</h3><p id="p-82">In our study, because of the limitations of the software used, we implemented the M×E model in which the error variance was assumed to be homogeneous across environments. In principle, the model can be easily extended to accommodate environment-specific variances.</p><p id="p-83">In this study, we presented and applied the interaction model using Gaussian priors. We did this because the GBS marker data contain a large proportion of missing values. However, with high-density panels of high-quality markers (<em>e.g.</em>, single-nucleotide polymorphisms) it would make perfect sense to use other priors. For instance, it could be used with priors that induce differential shrinkage of estimates or variable selection (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2013</a>); such treatment would potentially aid in identifying sets of markers with effects that are stable across environments and others that are responsible for G×E.</p><p id="p-84">The M×E model presented in this article is an easy-to-implement and easy-to-interpret approach for modeling G×E in genomic models. The model allows decomposing marker effects and genomic variance into components that are stable across environments (main effects) and components that are environment-specific (interaction terms). The model can be implemented easily using existing software for GS. Predictions from the interaction model had either similar (CV1) or greater (CV2) accuracy than the single-environment analysis and were always more accurate than those derived from an across-environment analysis that ignored G×E; therefore, the proposed model should be useful for selection based on either stability (main effects only) or for target environments (based on total genomic value). The interaction model is not free of limitations; in particular, it is important to note that the genetic covariance between any pair of environments is represented by the variance of the main effect; therefore, it is restricted to being positive and the same for all pairs of environments analyzed jointly. Therefore, the model should be more effective when applied to subsets of environments that have positive and similar correlations.</p></div></div><div class="section" id="sec-24"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">We thank CIMMYT's Global Wheat Program that performed the experiments and collected the data analyzed in this study. We acknowledge the financial support provided to CIMMYT and Cornell University by the Bill and Melinda Gates Foundation. GDLC has received financial support from NIH grants GM099992 and GM101219.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-34"><p id="p-86">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016097/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016097/-/DC1</a></p></li><li class="fn" id="fn-35"><p id="p-87"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-04"><span class="received-label">Received </span>December 4, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-03"><span class="accepted-label">Accepted </span>February 3, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Lopez-Cruz <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License(<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.1" data-doi="10.1534/genetics.107.071068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2007</span> <span class="cit-article-title">A mixed-model quantitative trait loci (QTL) analysis for multiple-environment trial data using environmental covariables for QTL-by-environment interactions, with an example in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071068%26rft_id%253Dinfo%253Apmid%252F17947443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE4MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.2" data-doi="10.1007/BF00022843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajaram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ginkel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">CIMMYT’s approach to breeding for wide adaptation.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D92%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00022843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00022843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VT61200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.3" data-doi="10.2135/cropsci2006.09.0564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling additive × environment and additive × additive × environment using genetic covariance of relatives of wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBurgueno%26rft.auinit1%253DJ.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D311%26rft.epage%253D320%26rft.atitle%253DModeling%2BAdditive%2Bx%2BEnvironment%2Band%2BAdditive%2Bx%2BAdditive%2Bx%2BEnvironment%2BUsing%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.09.0564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.09.0564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244430100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.4" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.5" data-doi="10.2135/cropsci2005.11-0427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnamachari</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modeling genotype × environment interaction using additive genetic covariances of relatives for predicting breeding values of wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1722</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCrossa%26rft.auinit1%253DJ.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1722%26rft.epage%253D1733%26rft.atitle%253DModeling%2BGenotype%2Bx%2BEnvironment%2BInteraction%2BUsing%2BAdditive%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bfor%2BPredicting%2BBreeding%2BValues%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.11-0427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.11-0427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.6" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.7" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.8" data-doi="10.1016/j.fcr.2013.07.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2013</span> <span class="cit-article-title">The use of unbalanced historical data for genomic selection in an international wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D154%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2013.07.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2013.07.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.569.9"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodriguez, 2014 Bayesian generalized linear regression. R package version 1.0.1. Available at: <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>. Accessed February 17, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.11" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.12" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.13" data-doi="10.2135/cropsci1966.0011183X000600010011x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eberhart</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Stability parameters for comparing varieties.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DEberhart%26rft.auinit1%253DS.%2BA.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D40%26rft.atitle%253DStability%2BParameters%2Bfor%2BComparing%2BVarieties%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1966.0011183X000600010011x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1966.0011183X000600010011x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19667594800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.14" data-doi="10.1071/AR9630742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finlay</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">G. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">The analysis of adaptation in a plant breeding program.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">742</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D14%26rft.spage%253D742%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9630742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9630742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19636069A00005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.15" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.16" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.17" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.18" data-doi="10.1023/B:EUPH.0000040511.46388.ef"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voltas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romagosa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ullrich</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mixed models including environmental covariables for studying QTL by environment interaction.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D137%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AEUPH.0000040511.46388.ef%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:EUPH.0000040511.46388.ef&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223738600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.19" data-doi="10.1007/s10681-007-9594-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multi-trait multi-environment QTL mixed model with an application to drought and nitrogen stress trials in maize (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9594-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9594-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.21" data-doi="10.1534/genetics.108.090159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Perils of parsimony: properties of reduced-rank estimates of genetic covariance matrix.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090159%26rft_id%253Dinfo%253Apmid%252F18757923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8yLzExNTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.22" data-doi="10.1007/s00122-004-1781-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Use of trial clustering to study QTL x environment effects for grain yield and related traits in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMoreau%26rft.auinit1%253DL.%26rft.volume%253D110%26rft.issue%253D1%26rft.spage%253D92%26rft.epage%253D105%26rft.atitle%253DUse%2Bof%2Btrial%2Bclustering%2Bto%2Bstudy%2BQTL%2Bx%2Benvironment%2Beffects%2Bfor%2Bgrain%2Byield%2Band%2Brelated%2Btraits%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1781-y%26rft_id%253Dinfo%253Apmid%252F15551040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1781-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15551040&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226629900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.</span> <abbr class="cit-jnl-abbrev">G3(Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.24" data-doi="10.2307/2533976"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Analyzing genotype-environment data by mixed models with multiplicative effects.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D53%26rft.spage%253D761%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533976&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XE96500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.25" data-doi="10.1007/s001220050885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Empirical best linear unbiased prediction in cultivar trials using factor analytic variance covariance structure.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075377300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.26" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruthkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.569.27"><div class="cit-metadata unstructured">R Core Team, 2013 R: A Language and Environment for Statistical Computing. Vienna, Austria: R Foundation for Statistical Computing. Available at: <a href="http://www.R-project.org">http://www.R-project.org</a>. Accessed February 17, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.28" data-doi="10.1017/S0021859605005587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The analysis of crop cultivar breeding and evaluation trials: an overview of current mixed model approaches.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D143%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859605005587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859605005587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.29" data-doi="10.1071/AR05153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struik</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Statistical models for genotype by environment data: from conventional ANOVA models to eco-physiological QTL models.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D56%26rft.spage%253D883%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR05153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR05153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232165700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.31" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.569.32" data-doi="10.1017/S0021859600050978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">The analysis of groups of experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D28%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859600050978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859600050978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/569.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/559" title="Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/583" title="Adaptive Evolution of Genes Involved in the Regulation of Germline Stem Cells in Drosophila melanogaster and D. simulans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/569.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ucb1BTaCOHJKb2Vw_pF0y2WMjIdLzCsbvRdoa0_85iw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/569" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/569" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/569" class="active" data-icon-position="" data-hide-link-title="0">Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459804" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Jdb7LW70Xr7qwf4k6PeGV2t0yr3naMyf7tmnEcCqt_g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/569.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/569" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-G_yER0pcknKf75abZwhQ0VVnPvoVOpbf01sApMGE-a8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/569" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/569.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459804" id="node459804--2" data-pisa="ggg;5/4/569" data-pisa-master="ggg;g3.114.016097" data-apath="/ggg/5/4/569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marco Lopez-Cruz</span>, <span class="highwire-citation-author" data-delta="1">Jose Crossa</span>, <span class="highwire-citation-author" data-delta="2">David Bonnett</span>, <span class="highwire-citation-author" data-delta="3">Susanne Dreisigacker</span>, <span class="highwire-citation-author" data-delta="4">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="5">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="6">Ravi P. Singh</span>, <span class="highwire-citation-author" data-delta="7">Enrique Autrique</span> and <span class="highwire-citation-author" data-delta="8">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569-582;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016097 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459804--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marco Lopez-Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant, Soil and Microbial Sciences, Michigan State University (MSU), East Lansing, Michigan 4882,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BLopez-Cruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez-Cruz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BLopez-Cruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Bonnett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BBonnett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonnett%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BBonnett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University (KSU), 4011 Throckmorton Hall, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ravi P. Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BP.%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BP.%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Enrique Autrique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Enrique%2BAutrique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Autrique%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEnrique%2BAutrique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Epidemiology & Biostatistics and Statistics departments, Michigan State University 909 Fee Road, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gcampos@uab.edu" class="" data-icon-position="" data-hide-link-title="0">gcampos@uab.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/569" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/569&amp;title=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/569&amp;title=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/569&amp;title=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/569&amp;text=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/569&amp;title=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/569&amp;t=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/569&amp;title=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/569&amp;title=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459804/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459804" id="node459804--3" data-pisa="ggg;5/4/569" data-pisa-master="ggg;g3.114.016097" data-apath="/ggg/5/4/569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marco Lopez-Cruz</span>, <span class="highwire-citation-author" data-delta="1">Jose Crossa</span>, <span class="highwire-citation-author" data-delta="2">David Bonnett</span>, <span class="highwire-citation-author" data-delta="3">Susanne Dreisigacker</span>, <span class="highwire-citation-author" data-delta="4">Jesse Poland</span>, <span class="highwire-citation-author" data-delta="5">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="6">Ravi P. Singh</span>, <span class="highwire-citation-author" data-delta="7">Enrique Autrique</span> and <span class="highwire-citation-author" data-delta="8">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">569-582;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016097 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459804--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marco Lopez-Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant, Soil and Microbial Sciences, Michigan State University (MSU), East Lansing, Michigan 4882,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BLopez-Cruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez-Cruz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BLopez-Cruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Bonnett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BBonnett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonnett%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BBonnett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wheat Genetics Resource Center, Department of Plant Pathology and Department of Agronomy, Kansas State University (KSU), 4011 Throckmorton Hall, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ravi P. Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BP.%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BP.%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Enrique Autrique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>International Maize and Wheat Improvement Center(CIMMYT), Mexico D.F., Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Enrique%2BAutrique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Autrique%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEnrique%2BAutrique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Epidemiology & Biostatistics and Statistics departments, Michigan State University 909 Fee Road, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gcampos@uab.edu" class="" data-icon-position="" data-hide-link-title="0">gcampos@uab.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=569&amp;rft.epage=582&amp;rft.atitle=Increased%20Prediction%20Accuracy%20in%20Wheat%20Breeding%20Trials%20Using%20a%20Marker%20%C3%97%20Environment%20Interaction%20Genomic%20Selection%20Model&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lopez-Cruz%2C+Marco&amp;rft.au=Crossa%2C+Jose&amp;rft.au=Bonnett%2C+David&amp;rft.au=Dreisigacker%2C+Susanne&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Singh%2C+Ravi+P.&amp;rft.au=Autrique%2C+Enrique&amp;rft.au=de+los+Campos%2C+Gustavo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459804/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459804/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459804/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459804/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459804/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459804/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459804/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459804/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459804/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459804/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459804/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459804/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/569" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/569.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/569.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/569.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/569.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
