<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/629" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/629.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/629.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/629.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/629.atom" />
<meta name="HW.pisa" content="ggg;5/4/629" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference" />
<meta name="DC.Identifier" content="10.1534/g3.115.017095" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Maher and Hernandez. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Ortholog detection (OD) is a lynchpin of most statistical methods in comparative genomics. This task involves accurately identifying genes across species that descend from a common ancestral sequence. OD methods comprise a wide variety of approaches, each with their own benefits and costs under a variety of evolutionary and practical scenarios. In this article, we examine the proteomes of ten mammals by using four methodologically distinct, rigorously filtered OD methods. In head-to-head comparisons, we find that these algorithms significantly outperform one another for 38–45% of the genes analyzed. We leverage this high complementarity through the development MOSAIC, or M ultiple O rthologous S equence A nalysis and I ntegration by C luster optimization, the first tool for integrating methodologically diverse OD methods. Relative to the four methods examined, MOSAIC more than quintuples the number of alignments for which all species are present while simultaneously maintaining or improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality. Further, this improvement in alignment quality yields more confidently aligned sites and higher levels of overall conservation, while simultaneously detecting of up to 180% more positively selected sites. We close by highlighting a MOSAIC-specific positively selected sites near the active site of TPSAB1, an enzyme linked to asthma, heart disease, and irritable bowel disease. MOSAIC alignments, source code, and full documentation are available at &lt;http://pythonhosted.org/bio-MOSAIC&gt;." />
<meta name="DC.Contributor" content="M. Cyrus Maher" />
<meta name="DC.Contributor" content="Ryan D. Hernandez" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Ortholog detection (OD) is a lynchpin of most statistical methods in comparative genomics. This task involves accurately identifying genes across species that descend from a common ancestral sequence. OD methods comprise a wide variety of approaches, each with their own benefits and costs under a variety of evolutionary and practical scenarios. In this article, we examine the proteomes of ten mammals by using four methodologically distinct, rigorously filtered OD methods. In head-to-head comparisons, we find that these algorithms significantly outperform one another for 38–45% of the genes analyzed. We leverage this high complementarity through the development MOSAIC, or &lt;b&gt;M&lt;/b&gt;ultiple &lt;b&gt;O&lt;/b&gt;rthologous &lt;b&gt;S&lt;/b&gt;equence &lt;b&gt;A&lt;/b&gt;nalysis and &lt;b&gt;I&lt;/b&gt;ntegration by &lt;b&gt;C&lt;/b&gt;luster optimization, the first tool for integrating methodologically diverse OD methods. Relative to the four methods examined, MOSAIC more than &lt;i&gt;quintuples&lt;/i&gt; the number of alignments for which all species are present while simultaneously maintaining or improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality. Further, this improvement in alignment quality yields more confidently aligned sites and higher levels of overall conservation, while simultaneously detecting of up to 180% more positively selected sites. We close by highlighting a MOSAIC-specific positively selected sites near the active site of TPSAB1, an enzyme linked to asthma, heart disease, and irritable bowel disease. MOSAIC alignments, source code, and full documentation are available at http://pythonhosted.org/bio-MOSAIC.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/629" />
<meta name="citation_id" content="5/4/629" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/629" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/629.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/629.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/629.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017095" />
<meta name="citation_pmid" content="25711833" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="629" />
<meta name="citation_lastpage" content="638" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="M. Cyrus Maher" />
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, University of California, San Francisco, University of California, San Francisco, San Francisco, California" />
<meta name="citation_author" content="Ryan D. Hernandez" />
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, San Francisco, California" />
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California, San Francisco, San Francisco, California" />
<meta name="citation_author_institution" content="Institute for Quantitative Biosciences (QB3), University of California, San Francisco, San Francisco, California 94158" />
<meta name="citation_author_email" content="ryan.hernandez@ucsf.edu" />
<meta name="citation_reference" content="citation_title=2nd International Symposium on Information Theory;citation_title=Information theory and an extension of the maximum likelihood principle;citation_author=H. Akaike;citation_pages=267-281;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=A. Alexeyenko;citation_author=I. Tamas;citation_author=G. Liu;citation_author=E. L. L. Sonnhammer;citation_title=Automatic clustering of orthologs and inparalogs shared by multiple proteomes;citation_pages=e9-e15;citation_volume=22;citation_year=2006;citation_issue=14;citation_pmid=16873526;citation_doi=10.1093/bioinformatics/btl213" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=A. M. Altenhoff;citation_author=C. Dessimoz;citation_title=Phylogenetic and functional assessment of orthologs inference projects and methods.;citation_pages=e1000262-e1000262;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19148271;citation_doi=10.1371/journal.pcbi.1000262" />
<meta name="citation_reference" content="citation_pages=259-279" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Altenhoff;citation_author=A. Schneider;citation_author=G. H. Gonnet;citation_author=C. Dessimoz;citation_title=OMA 2011: orthology inference among 1000 complete genomes.;citation_volume=39;citation_year=2011;citation_pmid=21113020;citation_doi=10.1093/nar/gkq1238" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. M. Berman;citation_title=The Protein Data Bank;citation_pages=235-242;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592235;citation_doi=10.1093/nar/28.1.235" />
<meta name="citation_reference" content="citation_title=Nonlinear Programming;citation_author=D. Bertsekas;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. Blanchette;citation_author=W. J. Kent;citation_author=C. Riemer;citation_author=L. Elnitski;citation_author=A. F. Smit;citation_title=Aligning Multiple Genomic Sequences With the Threaded Blockset Aligner;citation_pages=708-715;citation_volume=14;citation_year=2004;citation_issue=4;citation_pmid=15060014;citation_doi=10.1101/gr.1933104" />
<meta name="citation_reference" content="citation_journal_title=Arterioscler. Thromb. Vasc. Biol.;citation_author=I. Bot;citation_author=G.-P. Shi;citation_author=P. T. Kovanen;citation_title=Mast cells as effectors in atherosclerosis.;citation_volume=35;citation_year=2015;citation_pmid=25104798;citation_doi=10.1161/ATVBAHA.114.303570" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=J. A. Capra;citation_author=M. Stolzer;citation_author=D. Durand;citation_author=K. S. Pollard;citation_title=How old is my gene?;citation_pages=659-668;citation_volume=29;citation_year=2013;citation_pmid=23915718;citation_doi=10.1016/j.tig.2013.07.001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. Chandrasekaran;citation_author=M. I. Jordan;citation_title=Computational and statistical tradeoffs via convex relaxation.;citation_volume=110;citation_year=2013;citation_pmid=23479655;citation_doi=10.1073/pnas.1302293110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=F. Chen;citation_author=A. J. Mackey;citation_author=J. K. Vermunt;citation_author=D. S. Roos;citation_title=Assessing performance of orthology detection strategies applied to eukaryotic genomes.;citation_pages=e383-e383;citation_volume=2;citation_year=2007;citation_issue=4;citation_pmid=17440619;citation_doi=10.1371/journal.pone.0000383" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. D. Ciccarelli;citation_author=T. Doerks;citation_author=C. von Mering;citation_author=C. J. Creevey;citation_author=B. Snel;citation_title=Toward automatic reconstruction of a highly resolved tree of life.;citation_pages=1283-1287;citation_volume=311;citation_year=2006;citation_issue=5765;citation_pmid=16513982;citation_doi=10.1126/science.1123061" />
<meta name="citation_reference" content="citation_journal_title=Bioorganic &amp; Medicinal Chemistry Letters;citation_journal_abbrev=Bioorganic &amp; Medicinal Chemistry Letters;citation_author=M. J. Costanzo;citation_author=S. C. Yabut;citation_author=H.-C. Zhang;citation_title=Potent, nonpeptide inhibitors of human mast cell tryptase. Synthesis and biological evaluation of novel spirocyclic piperidine amide derivatives.;citation_pages=2114-2121;citation_volume=18;citation_year=2008;citation_issue=6;citation_pmid=18272363;citation_doi=10.1016/j.bmcl.2008.01.093" />
<meta name="citation_reference" content="citation_journal_title=J. Immunol.;citation_author=Y. Cui;citation_author=J. S. Dahlin;citation_author=R. Feinstein;citation_author=L. G. Bankova;citation_author=W. Xing;citation_title=Mouse mast cell protease-6 and MHC are involved in the development of experimental asthma.;citation_volume=193;citation_year=2014;citation_pmid=25320274;citation_doi=10.4049/jimmunol.1302947" />
<meta name="citation_reference" content="citation_title=Atlas of Protein Sequence and Structure;citation_title=A model of evolutionary change in proteins;citation_author=M. O. Dayhoff;citation_author=R. M. Schwartz;citation_author=B. C. Orcutt;citation_pages=345-358;citation_year=1978" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=I. Ebersberger;citation_author=S. Strauss;citation_author=A. von Haeseler;citation_title=HaMStR: profile hidden markov model based search for orthologs in ESTs.;citation_pages=157-157;citation_volume=9;citation_year=2009;citation_pmid=19586527;citation_doi=10.1186/1471-2148-9-157" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. R. Eddy;citation_title=Accelerated Profile HMM Searches.;citation_pages=e1002195-e1002195;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22039361;citation_doi=10.1371/journal.pcbi.1002195" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Hamilton;citation_author=M. J. Sinnamon;citation_author=G. D. Lyng;citation_title=Essential role for mast cell tryptase in acute experimental colitis.;citation_volume=108;citation_year=2011;citation_pmid=21173247;citation_doi=10.1073/pnas.1005758108" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Henikoff;citation_title=Amino acid substitution matrices from protein blocks.;citation_pages=10915-10919;citation_volume=89;citation_year=1992;citation_issue=22;citation_pmid=1438297;citation_doi=10.1073/pnas.89.22.10915" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. Hulsen;citation_author=M. A. Huynen;citation_author=J. de Vlieg;citation_author=P. M. A. Groenen;citation_title=Benchmarking ortholog identification methods using functional genomics data.;citation_pages=R31-R31;citation_volume=7;citation_year=2006;citation_issue=4;citation_pmid=16613613;citation_doi=10.1186/gb-2006-7-4-r31" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=G. Jordan;citation_author=N. Goldman;citation_title=The effects of alignment error and alignment filtering on the sitewise detection of positive selection.;citation_volume=29;citation_year=2012;citation_pmid=22049066;citation_doi=10.1093/molbev/msr272" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT—the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The Human Genome Browser at UCSC.;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. I. Kuncheva;citation_author=C. J. Whitaker;citation_title=Measures of diversity in classifier ensembles and their relationship with the ensemble accuracy.;citation_pages=181-207;citation_volume=51;citation_year=2003;citation_doi=10.1023/A:1022859003006" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=A. Kuzniar;citation_author=R. C. van Ham;citation_author=S. Pongor;citation_author=J. A. Leunissen;citation_title=The quest for orthologs: finding the corresponding gene across genomes.;citation_pages=539-551;citation_volume=24;citation_year=2008;citation_issue=11;citation_pmid=18819722;citation_doi=10.1016/j.tig.2008.08.009" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Biostat.;citation_author=M. J. Van der Laan;citation_author=S. Gruber;citation_title=Collaborative double robust targeted maximum likelihood estimation.;citation_pages=17;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=Stat Appl Genet Mol Biol;citation_author=M. J. Van der Laan;citation_author=E. C. Polley;citation_author=A. E. Hubbard;citation_title=Super learner.;citation_pages=Article25-Article25;citation_volume=6;citation_year=2007;citation_pmid=17910531" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Liu;citation_author=B. Schmidt;citation_author=D. L. Maskell;citation_title=MSAProbs: multiple sequence alignment based on pair hidden Markov models and partition function posterior probabilities.;citation_volume=26;citation_year=2010;citation_pmid=20576627;citation_doi=10.1093/bioinformatics/btq338" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=W. P. Maddison;citation_author=L. L. Knowles;citation_title=Inferring phylogeny despite incomplete lineage sorting.;citation_volume=55;citation_year=2006;citation_pmid=16507521;citation_doi=10.1080/10635150500354928" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=S. T. Mashiyama;citation_author=M. M. Malabanan;citation_author=E. Akiva;citation_title=Large-scale determination of sequence, structure, and function relationships in cytosolic glutathione transferases across the biosphere.;citation_pages=e1001843;citation_volume=12;citation_year=2014;citation_pmid=24756107;citation_doi=10.1371/journal.pbio.1001843" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Massingham;citation_author=N. Goldman;citation_title=Detecting amino acid sites under positive selection and purifying selection.;citation_volume=169;citation_year=2005;citation_pmid=15654091;citation_doi=10.1534/genetics.104.032144" />
<meta name="citation_reference" content="citation_title=The NCBI Handbook;citation_author=J. McEntyre;citation_author=J. Ostell;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. D. Pruitt;citation_author=J. Harrow;citation_author=R. A. Harte;citation_author=C. Wallin;citation_author=M. Diekhans;citation_title=The consensus coding sequence (CCDS) project: identifying a common protein-coding gene set for the human and mouse genomes.;citation_volume=19;citation_year=2009;citation_pmid=19498102;citation_doi=10.1101/gr.080531.108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. D. Pruitt;citation_author=G. R. Brown;citation_author=S. M. Hiatt;citation_author=F. Thibaud-Nissen;citation_author=A. Astashyn;citation_title=RefSeq: an update on mammalian reference sequences.;citation_volume=42;citation_year=2014;citation_pmid=24259432;citation_doi=10.1093/nar/gkt1114" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. P. Pryszcz;citation_author=J. Huerta-Cepas;citation_author=T. Gabaldón;citation_title=MetaPhOrs: orthology and paralogy predictions from multiple phylogenetic evidence using a consistency-based confidence score.;citation_volume=39;citation_year=2011;citation_pmid=21149260;citation_doi=10.1093/nar/gkq953" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Punta;citation_author=P. C. Coggill;citation_author=R. Y. Eberhardt;citation_author=J. Mistry;citation_author=J. B. Tate;citation_title=The Pfam protein families database.;citation_volume=40;citation_year=2012;citation_pmid=22127870;citation_doi=10.1093/nar/gkr1065" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Remm;citation_author=C. E. Storm;citation_author=E. L. Sonnhammer;citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.;citation_pages=1041-1052;citation_volume=314;citation_year=2001;citation_issue=5;citation_pmid=11743721;citation_doi=10.1006/jmbi.2000.5197" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=D. F. Robinson;citation_author=L. R. Foulds;citation_title=Comparison of phylogenetic trees.;citation_pages=131-147;citation_volume=53;citation_year=1981;citation_doi=10.1016/0025-5564(81)90043-2" />
<meta name="citation_reference" content="citation_journal_title=Artif. Intell. Rev.;citation_author=L. Rokach;citation_title=Ensemble-based classifiers.;citation_pages=1-39;citation_volume=33;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Salichos;citation_author=A. Rokas;citation_title=Evaluating ortholog prediction algorithms in a yeast model clade.;citation_pages=e18755-e18755;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21533202;citation_doi=10.1371/journal.pone.0018755" />
<meta name="citation_reference" content="citation_journal_title=Current pharmaceutical design;citation_journal_abbrev=Curr Pharm Des;citation_author=C. P. Sommerhoff;citation_author=N. Schaschke;citation_title=Mast cell tryptase beta as a target in allergic inflammation: an evolving story.;citation_pages=313-332;citation_volume=13;citation_year=2007;citation_issue=3;citation_pmid=17313363;citation_doi=10.2174/138161207779313579" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_author=N. Alachiotis;citation_title=Time and memory efficient likelihood-based tree searches on phylogenomic alignments with missing data.;citation_volume=26;citation_year=2010;citation_pmid=20529898;citation_doi=10.1093/bioinformatics/btq205" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Stolzer;citation_author=H. Lai;citation_author=M. Xu;citation_author=D. Sathaye;citation_author=B. Vernot;citation_title=Inferring duplications, losses, transfers and incomplete lineage sorting with nonbinary species trees.;citation_volume=28;citation_year=2012;citation_pmid=22962460;citation_doi=10.1093/bioinformatics/bts386" />
<meta name="citation_reference" content="citation_title=J. R. Stat. Soc.;citation_author=M. Stone;citation_title=An asymptotic equivalence of choice of model by cross-validation and Akaike’s criterion.;citation_pages=44-47;citation_volume=39;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Sukumaran;citation_author=M. T. Holder;citation_title=DendroPy: a Python library for phylogenetic computing.;citation_volume=26;citation_year=2010;citation_pmid=20421198;citation_doi=10.1093/bioinformatics/btq228" />
<meta name="citation_reference" content="citation_journal_title=The Journal of asthma :  official journal of the Association for the Care of Asthma;citation_journal_abbrev=J Asthma;citation_author=M. Taira;citation_author=J. Tamaoki;citation_author=M. Kondo;citation_author=K. Kawatani;citation_author=A. Nagai;citation_title=Serum B12 tryptase level as a marker of allergic airway inflammation in asthma.;citation_pages=315-322;citation_volume=39;citation_year=2002;citation_issue=4;citation_pmid=12095181;citation_doi=10.1081/JAS-120002288" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Immunology;citation_journal_abbrev=J. Immunol.;citation_author=N. N. Trivedi;citation_author=Q. Tong;citation_author=K. Raman;citation_author=V. J. Bhagwandin;citation_author=G. H. Caughey;citation_title=Mast Cell {alpha} and {beta} Tryptases Changed Rapidly during Primate Speciation and Evolved from {gamma}-Like Transmembrane Peptidases in Ancestral Vertebrates;citation_pages=6072-6079;citation_volume=179;citation_year=2007;citation_issue=9;citation_pmid=17947681;citation_doi=10.4049/jimmunol.179.9.6072" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Evol. Comput.;citation_author=D. H. Wolpert;citation_author=W. G. Macready;citation_title=No free lunch theorems for optimization.;citation_pages=67-82;citation_volume=1;citation_year=1997;citation_doi=10.1109/4235.585893" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Yandell;citation_author=D. Ence;citation_title=A beginner's guide to eukaryotic genome annotation.;citation_pages=329-342;citation_volume=13;citation_year=2012;citation_issue=5;citation_pmid=22510764;citation_doi=10.1038/nrg3174" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Yu;citation_author=N. Zavaljevski;citation_author=V. Desai;citation_author=J. Reifman;citation_title=QuartetS: a fast and accurate algorithm for large-scale orthology detection.;citation_volume=39;citation_year=2011;citation_pmid=21572104;citation_doi=10.1093/nar/gkr308" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Prediction and characterization of enzymatic activities guided by sequence similarity and genome neighborhood networks.;citation_author=S. Zhao;citation_author=A. Sakai;citation_author=X. Zhang;citation_volume=3;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Ortholog detection (OD) is a lynchpin of most statistical methods in comparative genomics. This task involves accurately identifying genes across species that descend from a common ancestral sequence. OD methods comprise a wide variety of approaches, each with their own benefits and costs under a variety of evolutionary and practical scenarios. In this article, we examine the proteomes of ten mammals by using four methodologically distinct, rigorously filtered OD methods. In head-to-head comparisons, we find that these algorithms significantly outperform one another for 38–45% of the genes analyzed. We leverage this high complementarity through the development MOSAIC, or M ultiple O rthologous S equence A nalysis and I ntegration by C luster optimization, the first tool for integrating methodologically diverse OD methods. Relative to the four methods examined, MOSAIC more than quintuples the number of alignments for which all species are present while simultaneously maintaining or improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality. Further, this improvement in alignment quality yields more confidently aligned sites and higher levels of overall conservation, while simultaneously detecting of up to 180% more positively selected sites. We close by highlighting a MOSAIC-specific positively selected sites near the active site of TPSAB1, an enzyme linked to asthma, heart disease, and irritable bowel disease. MOSAIC alignments, source code, and full documentation are available at &lt;http://pythonhosted.org/bio-MOSAIC&gt;." />
<meta name="og-title" property="og:title" content="Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/629" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Ortholog detection (OD) is a lynchpin of most statistical methods in comparative genomics. This task involves accurately identifying genes across species that descend from a common ancestral sequence. OD methods comprise a wide variety of approaches, each with their own benefits and costs under a variety of evolutionary and practical scenarios. In this article, we examine the proteomes of ten mammals by using four methodologically distinct, rigorously filtered OD methods. In head-to-head comparisons, we find that these algorithms significantly outperform one another for 38–45% of the genes analyzed. We leverage this high complementarity through the development MOSAIC, or M ultiple O rthologous S equence A nalysis and I ntegration by C luster optimization, the first tool for integrating methodologically diverse OD methods. Relative to the four methods examined, MOSAIC more than quintuples the number of alignments for which all species are present while simultaneously maintaining or improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality. Further, this improvement in alignment quality yields more confidently aligned sites and higher levels of overall conservation, while simultaneously detecting of up to 180% more positively selected sites. We close by highlighting a MOSAIC-specific positively selected sites near the active site of TPSAB1, an enzyme linked to asthma, heart disease, and irritable bowel disease. MOSAIC alignments, source code, and full documentation are available at &lt;http://pythonhosted.org/bio-MOSAIC&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZuQy3OLLyPYyXbRxxFtsXcDMsYDERZqmO1pnstjBL3k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459795","apath":"\/ggg\/5\/4\/629.atom","pisa":"ggg;5\/4\/629","ac":{"\/ggg\/5\/4\/629.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/629.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/629"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/629"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/629":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NjomX8SlAIKcTO7l6eNdO0YkyejM6+9mT\\\\\\\/98Xppss7nFIs9IZU58HXtQP1YdCgH7IpIHp76LP0Tz6jpbgQKQJgq9rpfeksy8k0iMpXQWts2ytyCUUTbvg66Pgq\\\\\\\/PtBR7A6bFonY6XvNDS9cugieoucuyoKoVpxKvaCMgRu+peXNt9knB0WGsclBwRZ3dP+m\\\\\\\/yY85u\\\\\\\/wvMprb7iu\\\\\\\/DHcxwz3oZ07SCfx5zuzi+duxuxd2lyuUSEIc0KZ+ITMAerUbwrn7brFDasfwg0ZEMPVf5DE1Iq9OOUpmBatj6O7snpPivxXfyggMSrT6B7I\\\\\\\/QBDbHT6GfiRyC4Lfum0ZVI+K5fdpilOE2S9uZLCMwRe2HT7VgjPMX9P8mQ4o9IXJNeUnJM2HlAeug7Bzax3rcX4J+pf7IZ\\\\\\\/Cbu8iC1cIqEsnMZdyCyOOQDdFXzj7A6hAdPOnMwf+c5CNX+FrsmeDrRYd07yqgyh5LvCrSXxhMBvv\\\\\\\/tjBhQewD7ZI2xaLxLcANkLJGajD6gLAbfyBYY+NOzmFUzjzgFkTjvvx35AkE7JUDRP7Sor462EUkzFMOD1CWMpaQdiFNwIdoaLt8\\\\\\\/jJJaCZhDfSr8f2X6XnEcy8zJzKDXaHcZrv38qkcAxezzaHTGAgJBicb8gYCqJRTe3afRJalvSyswLBvJVWTNhWqQ\\\\\\\/63i5f0VAHCwQ8aqDhgO8whAKJcnT\\\\\\\/SNIROQC8qUQ+Q6AlhtQlL7YOKes8eovsZ0Qgaz231ZygqxHKY1nD077F8vjz5oGCUBZT8qu9r3MXLVso26ZXmjZv2YbyFCQ5A\\\\\\\/eZo6BvB9ndxbvK+YdEoI8NbW5Ktp4qcsHGb2dw5ty2ZWRY79E3M1Nf7Aa\\\\\\\/qyH965UM58XkKURSUNJ2GESus52OUdqkze0kNTdnMk1AyYjssl9WmjnfTJ18Sxl9Yj1+sNKYI3XI7B7oUU3NSszsmzEAQ9KecuyRud3Kgf\\\\\\\/QMFc8VU4zsar1jOm93qcjcgzFSTb5Ks3cM8UK3l2eOl6KxuPt2ePy+z+VgITi193pc5M30UlEMH+CMy6lUoausWryGIlBJ\\\\\\\/\\\\\\\/ufynO06ottQol3A++AKMjjHBguvxx42kCuFzQL8fhUsDmRxkIytu\\\\\\\/zXQx+\\\\\\\/lyMdO2rdzdT8ooI2qZ06mnsTa2G0yt3VcoE3Tg0T+r2+5\\\\\\\/keo0VkqtC8KWblgDnd0KV9xX8O9BrvQjX+96T8LuEQNjwT3DdF0NFeU0WZaeSvcla6RBmGeRxgzxXJ7XHeEOqXB1ImcIDm2M7HNP+6xE6MBL\\\u0022,\\\u0022iv\\\u0022:\\\u0022ynruzAZHKOs2fxNwfkissg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228f082666caaeabcfd0f436cb5264385f67c2540dff16dbeb709eed5df5cf259a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223GkGwI8e6I5HxyDwNs0SmU\\\\\\\/ZWf6fDedOdD6vNr4bSXOWYhuNZEcMQdtiF1W6GWqdMmUZ43rceJfovFLzhbYZANgparvs0JnrRIScc3ryi2EDxF2G+Fe2XrJjF93kVHtK6emZmdEewZuEhzmUAP3NbDYAfzSKOGhaT2HsBX3YUOsMBpjf089Ys9lPM3dG6a0lD+O3vikefnv7XeGVEmyhYkPDXAoQ85xv6sQMw1bW\\\\\\\/I6QigKW5rpEsM4h8eG+wP6MX2DwgStA7NyMOqWRjrHIOO2k6ZGU2PyKGCtfLpQXdVcaI30Bul9sIU+cuumwAdx0MND\\\\\\\/ZVI3Lw4rAE+KDoQWX4nrcGMohYOsyCN083vEb\\\\\\\/a+QoMUySGnsi8GQG5x+V5T87OqI6Xh68FN16hOlL4CrfRcY478bFGiQbEp\\\\\\\/k4gEwLlJKZ2D2CeBisvEFNlc\\\\\\\/LUoWdKnQYU67lOmsc9TeH12UYLDE476JIU\\\\\\\/RfPStuN3DhqNeHWOqJxrjyudx3Fs5Go+uy3AP7tv9vRsVo21eyvXllvsBOFCot4XYmGq76Thxm9CvyUKFiAlgGaHkaEd97hKOo\\\\\\\/kNirzaDIH7lbYeyfVAe0qYOh5Di7zJFm93QM4HL5zQRcUWEIrLynea5sC+tcX0pWp7K1GjzhrZpRImFFwYII4aceSgyPAezkS9rnAo6r0t2tk8evtpnbuGfXnHEkDkW64fl5f\\\\\\\/wu6v5\\\\\\\/FKnBbLcCHpcTNX\\\\\\\/1dviYutQVLpHFaVuercVc18VQ5v7Kik42uqPw95B6WOzp4Zwu4Dd5zGfii2\\\\\\\/6asj65bfaPB4vqItlQKQcrq7xy8FcVkQuwB\\\\\\\/\\\\\\\/o\\\\\\\/+HDVzr57oijdmjynH7NxtBcPnQT6UgXcdNE+TVd5s\\\\\\\/RijTpciml7chEbsiXKai2ar4q\\\\\\\/zamFqpeliJmJ0jMoVuvuqCwNhnbrBRxYRWkhcscKuHMi0xVXw0fRf39iXyt59MSPf3NBbGRUWQoVukgxNz1S9KwRyJ1yi4GHt5ivkmTJEbIXzEuEk+wZ8osuJUUKTayzaFrBLeW+8RCqiDs4+AbQayk3Y9DC2oyCIdTW9+8DvP3KtQ1vBQtEALKAcL4lfGdpGzpr5kJ6BrpXgam8BMQWmCtpE1n6uS7cblMdijXDV9TOjfxWeAYbDUiAdwRpVqIi6XaQGpeh5FdB7YD3e2n7o6BWisuyWXPKIFgMgsidYUrquekYRJBE\\\\\\\/xqeHkdGpWITv60bdbTPM1Addv3FeRq4QcIT3KdkEwISz\\\\\\\/s6yyJn6aSh2fjSadC97xnug8G6flYV9Jo7S2mDdDGNxojjO+jszx47aUDCpE0\\\\\\\/rsTMFT7MzMthHsRW2hFXVveRM\\\\\\\/T53wajor+xuw51q3xuMiKGLU\\\\\\\/0cqihUaVQYTvkCLNO8QPGsftd4h8\\\\\\\/CErkMLDvrRDq8ZK0sAChgyX9wcBlLSUMHrKdVPivUxzuiLoGIZpMw2sq+qI+sXh00FBmshZUYSIgjp4DU1fo6LMyKV2XaxxvfuhYEmSX+46Po9WLD3sP+0OOzAZLDsHHlByK87Qej12018nrN7Z8Bx+zpWOP7GWgE5tLKFe3FefuU+rehkmpOrmJ4AQJFHIU8BXajAIMXONFXYanTtCGhpxiP8VLvkdC8I0o9ZbtKL7JdxUefSr8V3wJFct73bbJEulneLqWlPxQCXwct\\\\\\\/2isHvWD+FmNnJU66\\\\\\\/kHRDtHqOzEk1n6YQsjYULLyfXZRbb2\\\\\\\/73jme+zgqDayJrS3PmIlNH4D0HWcT3Qf7YxIaFyRfGe+jHKyiDbFBHOW82tqnY0sqt+2u6hbEaacGVewgwSEOu3OEbkXnOpkZUHY1iCErZ\\\\\\\/IvKb3QaoEybQ2dROlvGPIV+dlKofhqOlUzUWwMsW+EYwss4SIDhTPd\\\\\\\/cB0fEMBthI7xdH3kJKOJ2NP2pTwlJVze2ubznD8o3s0RJQtAAHuAkWHXMw9jRPMtTqqPz7X9hqT2gKtbwVBh\\\\\\\/gVUHn0bj95TVCCQTWkH1kvTwrTThhpbaRaNgE1u2oTenqmHajzkarpUE5zlUYmeFqQxMf0spCoRoDhLpK4XNip+5HFb0lf547loXwlNMGUtS3tZbk3ZrL3zctuhl1Bv1GUE6\\\\\\\/nHUguwMW+TmKZ4DHrYBQdxVYRbt0R9e8vMrVq+l5sQnPsS2qiIaJrnKno5RF3bC62m0ATVpAtCy\\\\\\\/CvrlJuhpel5L0rZXs0HGP\\\\\\\/tO1gfT3pU2K9HBN2tde4VQUTjsHVJBX253Serly5+\\\\\\\/g6M0A0pI06YcGe7H7aRFRFELfbl8yGMIa7KplSHjrF8EgXzsgzwDSk8uU5d7ArnynWiefRzJHdPT49eFtbVzmNGocJ2D83tQOWGCFJHDUKVMzRqU3eVpcgNUuWSn5LGrYX0oBm61pkaBQnUotE1WifKwWvyxqaDpjJ+7PP4dCleGdj24gNUauPxlpVwqkis4LvjYoF80Xm+uF4cyBHMJEyZaP\\\\\\\/0Dl6V5u9SUznwm8e0Ey9eZnMEnj1GDAqHnZPk5l4gcm+lPLVnFqqeQeO1UqcRcK7h5PJ++X4wzg1qfi7o5J4ji+Hyp4MPyaLhI88txjQHdMID1whjolwuzHUncTaKtQ08yanLvo6y8Rxd6bmRUC0kYemnjDKMlL05QTXYjmvCDFCfjofx4H3zmG3Y7aQ57dG4VbzCL66qrQLrr3x8ScTg2vWB35FpBP+BB9tQj6owprfIAoTWxgsyOLvt2OPLfB1e1G5oId7lLMfbB9JOZUMLxptsi7MxMgxg\\\\\\\/YwjZL0fAnO2pdz2M7AjcbuLooXN\\\\\\\/RME+a3tw4X5TfTlvoYtiiRwU9vBAsKKNWcpAiqoS7BtUmWtOwgj1CgbX2QrPnrS7zeHAxabAtB6mG6FAKHgOvKSjds8KvlWSlMuEkjbaERIn85Dsy9jUzHd2i\\\\\\\/WR8GxWg2+PFliz1I6\\\\\\\/3PbKMyZ1SaysOD0+od01IDuN8sjDd0zhhUumBvHXBogihE0ffw6dXbSO1TAOdcHcSquxF3rjbhUiINuqDK\\\\\\\/coHSjC6WJtRHMklU9BAZpn+SYkWusbooZqUwNPRnvfk6ueGogZfPrAH+cpaNbnfRd4RdNVShx65XMCazCkzwhXtGK2ExOtbQN3cuX1uGedRuHkEDTZQt2owQAsrvhoIXSknHHXjtsxjDgK7vE4li4qMbnVaq7T3IU0TfEVrkLb\\\\\\\/Dg4ZbJIIVLL\\\\\\\/1J5sGfcsgdwiHbtMt+q60pnXD8ceh1d0InzCyfSXljcGk5+oYCZwXfAIQG++8BYfxq9WvTPzgFNoe2ux221hqyhgUWAhtPJmuKac97ug5V4WovFuerTT\\\\\\\/ARym4GcW7CSL\\\\\\\/kyvpNQt9RVLznCqHfAnf7Qyga4BHM1kRBP\\\\\\\/Caq1HLczPARdYdAVdrVTyN67cCwcsTKukjhqgRaZ+xmGXohrIlZ6etro9QKv8TZZzO7XgBDO83KLU1Ovv5y\\\\\\\/l1wsAJLjTKTcvddLgTFDMuuEYsGY91bfR1yQWZg3WYaWjEkJTXs879Kr+VKPgkf8gnlT9osOaFj0td4ZmDBagbSZjLSCznnRIZHkEF8ftph+r1OAzXqnZmcRBU3o1Vqr6aBKz2SX6PJjVabqmxmc3s6kL0ud28H3tzgnArGV7PEM6JMWEYuMNV3G27nc\\\\\\\/Jz+w43kT4qgmb5t2i0j\\\\\\\/\\\\\\\/bRex9JKK\\\\\\\/lt\\\\\\\/UzyAPgMRittvXVw377M39Qr0yj9td45vBdadLUOD3gpfWyDVuiB6fgJuiuxt5VkZ7pgVoJWRXHPMLTh5WkTRj4vPZ6Kwypk\\\\\\\/FCMiUzoPBDHQAzEzlkYcGUW6IUoizcT\\\\\\\/D+o9i4eCPX+DDbEE1zpQU27WvR8cF4kzLoWeN+sW+krdge0PhuWlvrw8AjSafdg7VK+ys8QHcCxBrf8q2XP4oyj8tSp32EoVOtxaKuH0+RIf9zCzoqckYmOtSoFw8iI\\\\\\\/7wCzRYn5Hfw8zvgXDXVpPh3QwbnKtVVGWHNFhbgbL7rQmdQrAGi1l0BqPvcu6bqrnXmucNgBEFC1SUNcm\\\\\\\/Z7C+sdpx7hswjwOGUM3C6W8+JqFA3EQfjVFTFVvqR1ICgVnSDzVpAsgo4nmk77KkMcKTo1lpExhFTSsBxUSCQw2SK1OfqLQWAQhMvrb3GhBBcrXtcJD1q4g8vYt4JBY7cxZQZv6q1V4vl7MOYzURKFpYFnLF8CL45tkvTJHDkA9g53BfoKQ0BBAVxSrtdhQijAKTh0Bu5Z7RAlHXySCxBRBVj\\\\\\\/IaKsB67PPfrEWPFCMOLC1s8CHPWfW+g\\\\\\\/j7kouE7r5m9p2vmuQPFtYPvIyWuR7eSVoMHW\\\\\\\/z8oiIUhvWzfjXbLkb1V5\\\\\\\/Vo6o2W9GDdKgYopcT1FFgVevEezU408uGB2hmiCGjTgV2gie6ZPzwnTtEbiiuSl156WHj8PDo+zVxEjU4uyUA9UXGX4XW6OeTCJWIe6dMu\\\\\\\/Rj\\\\\\\/uiT0PoKWAyv4tXa0jTdSl5Y8fgJiswh6wzJX7mXcxv0tDZ91xA6y\\\\\\\/9crUO3vZserrBSnAproJCrmrn5wgcCnt6Q0jCLswmN5UL9bNjIc+YJjI6sWzbnf1Xx4de+EeLkyrMNvkxptWPg1ZaEqgToP35pHj45ayeSiwQIaTbR7bCfk8o2sxc+WWUuEZ9GjV0opJwnlQNGPWJMAODia+4vv9WWH9jR1kjWpkvLyBAM\\\\\\\/VYg49rlcHQijFi7eIsm0YaBVnNLipKLx5flQqFkmVZzDEAVag5Ghh0WJinuLgLHgTfnDMl7XKVk+JrGew32qpl\\\\\\\/ngw8DIuZ\\\\\\\/OYz6ekU1yoyogPfS2npwMh\\\\\\\/S2bcNQAe7YihovQrqEvC3oM7+vSc5Oww6TUGYNMTXywI3a3w5zsmt9rEvJzXLYSs5W+nR+zFOPJRQwZjNvyxSd7fN6Kn1iVguTi+DWFKZHapB3kiAt1CrwbhMxqF+HxPVEmRKK5hTdM3CkfYQlct5neF4DT1HFOFKbGTw\\\\\\\/wq5tUjhmYOkgcxS+eezMzlO8w8BuElhHGQUepgl\\\\\\\/20l4bdPrslATV0F6rP75IZMBuhNtwLpEuVBiKYSCoAsuTEdnZCBunUPuirfnoNU2aH3Of9OHYSDvu46whCjRbHrE3vPOpuCIZcs8B8rJiM6Vs+tU+3Cxra\\\\\\\/EirpjlD6+YbhZKGCxDdnQXsWgCiyFhRhw4KAxqPQGSxX1w5eWoNAZ1b+SKDxbnpnMo5n+cHUpPDAnmVW3rHzVumkPvFM5kT9AF9QX5VAJ9TzLACvMWSNxViJ6VUX1OPNjhCOrZ9Th+7vqoQpUoZCCdZTXooDXtyXjCNGfU9VRlkCBDIscKxZe3WFN2F20qNOPVSzhJanaw3bhHyEzpQIBln508MjSSmvAOGwSbPE30\\\\\\\/mEPH1pS6JaGY7YJRSm6jZd2ZOHfep49dyzYCDIumPZexoCFvftWaIcB9WUk0At\\\\\\\/I6wtLCo63VJNebOCT5HHUOomhWKeHdmDPnQJ\\\\\\\/pAI2F3iM\\\\\\\/\\\\\\\/7ujylYF3BDcDt1+tMW\\\\\\\/JgDvReKFDtUSrk0z6oW+S6onP4nj8TauAA1t07VF13navL9\\\\\\\/kBU45SqQ1Ex9faOCmgqe\\\\\\\/vA\\\\\\\/xptAQ\\\\\\\/bCV5\\\\\\\/BZMJCVSvCRqrgu95AK1sSbpOSN7cERiwTrUu7Kbsn8R\\\\\\\/QxfZ0vxJqi9wCtNDcWLCfuED1APLpD3UxDX4wCkpYkXZeuXNQcqGLrXqed34rOcEaMCZGOpJuEMjb92CqZe6V4VZUo\\\\\\\/bgiNo2mFcu2l04k6tu0jR4tAT\\\\\\\/+Fo6kEElHAiCpJfyRoQUG6g1bQleYplVm7wAoGIrZ0C1xE0yK4S99cLNAj6KXAUohTSl0vXWT8j1p1ce7u0eukA4GPTvBGGL1S4W9khGhlnnik5PInGrq+62LywFJXae5xhOHSuQeQS5vVg6MLVPInUZ9pen5+ZX4e8SNwcIk4WKkl0uN59xFkzGG5rVJTEPYkTVtrGsO8TQ4otHxLM6++sJ7\\\\\\\/Y6LIOT8zaceItJP\\\\\\\/XxGcUxLpDqtft+4k8NFG8lkgCWI4a1oZp5emC6TqfKiPG\\\\\\\/vLv1pZbqT1iMQZFw62xda2k5wJEB2uoO\\\\\\\/zfWQS9uxadKeptTmnfY+kBWgqiDtG5iwDuykJTDH0EUwA6lmbHP+p9ibFuLzvhA4\\\\\\\/A218AprCVs+Ju2SGpxSTgYz71fAFcN8\\\\\\\/ZFgz9EHkY0VWRKh7yXgRr+kVW9dc9VcVPjNZAEvD0BArbG+FDrhcmdRsEEfsHyUvcIoPpjGGmL9zzxKx2kc0BfD461HdVHqOZAhZo\\\\\\\/wUYWIqwEWo+V4V8ZmGCapUiw+ZbQNv\\\\\\\/vdZlBqxGTF++IoAkgSdiBm4B02ke0jzU70AlTKbAaTyRiubqgpU4Fz9+Xpm+Bd1YOMJcidg1SnvQJ4mpK+Nd+MAuwJ9VTg\\\\\\\/HqeaxrV7WGVulsPUDaUD4u9V1DXkdfpSwGgrZkget26mVo6a7AA8r3ey+iUslUwcnThT9kcPn2YBDbZHOjcmExQo8cCcFyK5eXIAzq2Rl64SLHkebkbzeIGM12u23zQrnPtipwXyF\\\\\\\/0yi8+eLItkuxIvlygZNhLdj5VNcLFcgLgd5BC6isIcRd7w62hauBvghqPBZVcAac51UXrFXnbFITEThliXok2GRC3ArTY48uUVL+dwBE8B03g3q4ZMl5YmcunzyM\\\\\\\/bV8rT+2HMESzb87rlTO36xbBTv1msGLeYKopdlpOCOFrL+uJsfJfYPxvCZpyfvXgZFpO\\\\\\\/+MvAvmnYdqrLQn8urz9ndoB+CgDoNFoqKGrIL3xbw8012\\\\\\\/pp\\\\\\\/VEaTqoJHfr7NHE6fLAHUTXNXkX0ur598Dq9IkM4+hXgk2pnybozpcx5AVetn69RT9wHfbh0Y6Cfmn42qlGuApTiJefPhFAWLBgp0IoYzxCMmrOZDgD16iJX3rHgU2RfYkdBmiXy\\\\\\\/1uUXXNL45jRpa5uajRUsn1AGtvmqOpSfz4\\\\\\\/1jUSqU3FKuN9rhGvmSYFbLvLS3HKDu6LcnBblB0zxlwb+Jr60WBysNl9wO1DNmbCrhgIHTbBLAHdGp7xj6qcuGcntwWqRUpPV5i2tumQu+R1SrYx6J5jtFx52sEQMxmdmU7EikA6ZviLooJmmjU2qvAN09azo+N+4i8q\\\\\\\/T185D18zMmQNN3dJeg1unwi8ZxGuCWHE9k7rDns5inCHf\\\\\\\/JZN9sNzODLbG4n8uERfmdeYhIztyL7KMs9xDBlVIuinzr8iK+paWUeyo+BkuwM27kj6M8\\\\\\\/RAoE5nN68+jN5r765OBiwBU65B+wgvzRNQRWJw\\\\\\\/91Y3CTnEzMP6QlY9q2I2erWo+q1hXCM\\\\\\\/\\\\\\\/Lh7DSJzRvt43PRJUl+u5ZvfxCDHsm7ykNli5o7j0pzGmHf7GLjztRHTxP\\\\\\\/I90Z2hP5jjeusuDHH1TWcyExl7+WkW\\\\\\\/Om5uiunPGoxD9GSIIONHChZiQ1nVuaAfoH9axwHLFMvScG3VneaUOwHDa0tUs4RjYWy3WwM66t+Fj4bzx4pGD\\\\\\\/J\\\\\\\/bqgkDNRR58cBEY2gE9a5gvyLbfOp65MtI3lDcpVHQrPazRLhx6r0dVuJLeQMpK7LQxw0bDHUdgz7H2Vz5bilCIkyevtc4fg8ygNCza0yPnobF708kKG5sWBY9ykMFmT1Ug+fO49nqZBDg\\\\\\\/pQoA9OsyruG8PNHllPvQxmVLfnFDm0AHHvlgAw6gMmExbUUGQ9TmtEVyhR5xizzSlIM5MlddzJVasZX9A7B\\\\\\\/bjRrPGcKEEkgtVzYP5yZL6Z5qUUEQiYL8b2tzNYT9B2UqUBH+2e05JIcmRuobuzzFiUUdL6SMpj8ibI5YJpBGU0V\\\\\\\/8un6tnaFUGkdBsB9UgdqC3zkQT4cZddYxgI7mzniMP\\\\\\\/Rxg\\\\\\\/N+2ATHU90j5lA4dpNBFHOD5ZBFKVCYR9deL0mz+0SIKP4Tbu2XWliNV+OLqiPwbLwdr1aKsNcZsTI9dgW+xE7wttjDHm\\\\\\\/JL1rm7RQEzfDg1ariVsWcUhpHBR9zyASIpUPGwHM0nWbhaBbKvZ1KrZKaxgFo7tV5kVCIAGx6GWqAWyB8yMqK2kT4idCGizzy9Fpy5zxkfFSoF+ay4raCaPEH3Scs8p4I1ikh+d+Gqaot22s6HzrCC2cQHMgXRWgEXYR+14r7Sc\\\\\\\/NLZ95gfcPmbDyKY5NSxAJVLg2+w4RIHs6QmYbtkJ4i0ZI2dh0Yu2ei3ZgpbmDAtaw4AZpVwcWQm2tfIDhKLGBKAEFi1vLFKdSa3q2lbYRtHStdWBzkZPsjdILpunMvAifGE1Ln0OvlYJrAewvyBcyrGCIjucSZq\\\\\\\/UqwkT79LubSl7PsjNsT8pTpoIMGq9u1BqOiQWnDJ1LbACjBoWuS6XaOiYFJmfjQZ3yUbv80HOOHuU3+1\\\\\\\/+EvdYEckUzDOviBl74wWNf0eTBrghmPE\\\\\\\/\\\\\\\/nqNO4di0gxVBP6ogKjkknMawSUOEAqYNR1ze8FRe+shG35kEOA9CMnN6Ww5ENuc60qpfY3heV6tT02GwSH8JTe2nvuScg4CmYiY\\\\\\\/UU47Dlm1yehFB4y5+f6k6oSwooOr7lnecNcKrznvunIgPedQnmC0DVhQRYz+mZOcShQrSHUi+vMfWfqOzyaeH7\\\\\\\/ncKJj1yHoOb9xpTKPvAjxp28Qoj\\\\\\\/RuBVquddlmf2dlXAcqef11iwLGktBQrYWmMwSERhu4bIozvImq0huNB1Uh\\\\\\\/\\\\\\\/esvFZ8oAJUEjDJkr+8SbIusAY81kSsmR6GmJpUnbFsVHzWG3nMv5zKYAdqQELfCQeoPbxyF7XcbnQp4n9jo0ENRdY61XGxHuVSxgMk8WRsrcvoThdeKtW75mQBvBNIFEo8CQizh27DOWAm2OXObSyLAySHxFuLQQoix+sr91+Va3qksiAgFRxZpTOZx1Wk\\\\\\\/IP5s7Z2\\\\\\\/EuSNsBQhpLB2syocYf33DYDfNWgjdLOZ9m4RXX7ROQewJC1Inxg6Xs2Mj6v88t00rfOUyrfPkSQugNb5gOS1ZrWVJd0wkAAYBt1RCBWCXSXS87FfTvA6ocAExb0aBCD7d5i6D+K6q4SFWXsGq\\\\\\\/829ZUHC5csZ1rxdN8R6BkHHsGFIqwKohDhy8tulX0zm5w972wqoLISSLO\\\\\\\/xNoFqbBNfbu1EFcz9aGDU\\\\\\\/dmGhXyAFhFqN8CIJuIQ8NrEhZ\\\\\\\/\\\\\\\/XqMvqP77yxK6zNzbBnhk\\\\\\\/\\\\\\\/hm5R4H5OV7NUC2WlGrUo1i\\\\\\\/tQoCxsh6KVL8sLnh4i4rMn5M5DmJAYzjZD\\\\\\\/7tgOfK6ogi6ArbTsVFRzgp3796g3XEf6UPaEzkRC4VQTk2t0y3ewjJhP2YKZqjFui5ob4dztyv\\\\\\\/ut4Teqe0\\\\\\\/f13LVP90WciywdsJznq+B+JBPYXl2wldvBBLwEVrGJQ1Is\\\\\\\/cNRHL+lf7NC4DjqIikuYyV7R9kjJiVz9cKVK6T+PvQOo+BTbVdnSfW7CeqHAJpI\\\\\\\/60iuhnxMAL9h27SOvQph2Ca\\\\\\\/PvXWCft2mP3VuivKVT300MlfkxeblBDu1BJsTYS1rf76KglOXJNK29Pa1LWawa00yBtSZxAodVA3LMLZSJbJcNg+PRsnDlQlo++8ZrLa79jrWJuLhC0ZXwu7lQT+KkZQpqm4\\\\\\\/AH1AEUYxUc4G0qRUz1voFbadOKk5baofIDzX0AgJafRkyniQy1KDNudGnrsWFMHPz\\\\\\\/LPJTtZF\\\\\\\/DvhWjtiIaqcUtc40Ub9he8O1Y+KaHqkcYbJIBqF+nPKLQapGucxwM6Y3r5Al2foVJ9izWZ1ROxjWJhrwNVnS\\\\\\\/+oewY1CRcTNv7jqGZ98Ik3+YJiUsdL2jfDcfaGJD+rP9QwNlcXkXDGqAaqSPuKkDXZiaWTMFYvmcmvqynxwoZWk7EuR2IbCobhP8tfNHrvvac2\\\\\\\/ZfyDBVekKyMdSFCyXGLt\\\\\\\/a7Vp\\\\\\\/FMm5Vc+ZuZ3GL3Dzqgx+WG74GhmxFwXnb9KrephKriUreI7LzbNFRHN+VMko1ZOz366imxp+EnBfJHlLakc7ao89LRyTH9LlVfqxWALnBltBTpZg89HJsOpTmcWU8Hty\\\\\\\/jm26OxaJ9Uh7WAKHs5HWt+s0n6pvp11I0uQy0Vi8kqlud48I8Vd64t6nkYoGyW0jgfiikb\\\\\\\/MFNnOsWfnEGsee2utfnq751hhkYiSMbHom4Owg4MT7tK3eHbZikw6KvElt\\\\\\\/oZo7cVhlTYb97ci+YDTpeEEaYoqg072YHFHRvrdhSYo3jTGfBlrgd0IkPhYZYCZcX+jvdKMIqydyYoJr2+AZdi+DFN9hqpCvmxWsZ1fSnHFtWqqjTmzYt14zhNvBkbgwo1MtwFPGCWNlzfXFg5t5xtGUyv3JNnh50Haq9HFMlU\\\\\\\/yxddQcuDJUSpfVuV1AO9Ns1xtvoon\\\\\\\/KXCm2sgdEWQqt6JGtDOrCjhXLNpPA1igGgspcMbK8IwcEqr2FAPIbxy6Ql8G380jZdnJ3cs+qJqJJuOUPuZn\\\\\\\/DbZhdKX4LMraokNgH31AO6K62+dokoQ5TV4hIEVU2FsoDCr4\\\\\\\/U9zdz00711Of4+GUatemxztsW7B9srwoiXMQhWPCynGssOoyNXQI0eVlz7Nm+89XNBhc5oq0A9AxhBducTKr2U+mDJ4\\\\\\\/HGKDRgpU\\\\\\\/ntX51aU1FJErmDHecD7cnH53mGgPRIoAbHmhg9gMbh0zEkupzdAWEA\\\\\\\/GEO36Sn9ge+NlpCVPW9dR3SKkFdt0GFr1s\\\\\\\/1fPxVpQ5Grn8ROWhtWlNcA09rKcI3BEEJLDpJLJzEY8y+7W+WA5FRMyBRSbH99LD94D1OBIv8H841QT74hJjH+5EaIxEu77EAINx\\\\\\\/pra8OzSH2\\\\\\\/XL+KMY7vjWcBLS9j4Quu9RYh89I1n\\\\\\\/aJ\\\\\\\/sj+6z2nEgawEHWp6ByJhlNtFpl7J8+s4Op6uyzDC1t67KRv9AG+hnvow8Nrh77xnI+gCMTCwsSFlgKYEHmR5ZoYE6prEpJUK8DPSdMT5X0Kya9hV3c6kUU8QUNIvAVtZYw9IGY2BgxcIEJvGRNvDvkEn7ydljPgJr8JwyVcGP\\\\\\\/szNoOPSUrBOGlux\\\\\\\/yaRtTJmwYajn+jKJyU6MXKfSOcDYxbwDVBhEnSnmDIX9mWQfufU5532IJYnUVbWDNGS7X0oRSwmKiknhXh6CiK569bGvk\\\\\\\/77DtgSyqYk7IcOcnC1qq7erO76aXRs+b\\\\\\\/vh\\\\\\\/3sI0e3I0xf\\\\\\\/54g4tZTl5u60UcmDoYMbL5moBZmrEhXQyENi5M3AoAYvxTtIHRIR10hLvv0kXo2ljOo2S0NFDhWSjHX303AepOHjpVkCbzse6ampcK6oFaVD4CUHS21tyd+eUln36wn0UzrsH0+q2oKgjf7H+CdEfmBqMPJMO1juxlRaUDIjXcSQdpfrs2AI\\\\\\\/ggRuWKncbsrJFG4t\\\\\\\/clPXC9IuoN8fRKI49qSzE7zDzfG880yYHH23ykNQy3hME44PcA\\\\\\\/PKp9eERKnwWDC6EPkezZreQykmeG6pDH2pEjp0aIyKoIitBtgqYx\\\\\\\/bc4qXT+rAZ+hdHHnNj35QNqEr4Ug+2ymbb5TsqCAp+Mmb0aaDxxGO6gS6WRLCP3PpRXz+xWrD\\\\\\\/twhGaawPusAI5PUhQ4a4tCNMxTWN9J24HqZ2UUkJ+mhwnTm9YW4xnI\\\\\\\/boffLJfPih5aSS+4hcgICZKCdAS1z\\\\\\\/j1ZSLTp6MLB4X0K3EjKCupjfMqxGzqT4g3V2QyUofqGKaztuJiW4x82YGIV4UWrCuR3+7NPz7gHWwWMhS\\\\\\\/\\\\\\\/MpVOIQzzJcqrJEsPpx2PC78zx7XaQBLaXljo4n0DGHdR5AKgpnJhWvhdNpuPmrlDs+Cp6jLXGN9CJtkVTS5nDxYqnIAjCZ5JXRTkaTD9vOk8egIfIJw5leAyb8cc8SUOV+zuwr\\\\\\\/JJPZ+RB8uDtFMVhBAn0q4Zpt\\\\\\\/U3dGfsx4gj2cTSu5srNZFyxoVR\\\\\\\/NVhawtUTTwtuGH0gAGbh3+MgPWh2xxPas7mvHxzmr2JtbVDDi8an\\\\\\\/9jWpZpMQY\\\\\\\/r+G2HEYyKWoXCudhERktvVk60l7PpKG1LmrYEbigHSdoGGzzSeXH2hkHet7VeYpXegSR3hYEsVJkonf57MJyIPutgu9NzZfSXo53tvCbTCKQrrPeDGqoacmTdetxuoJMRuEsovdn9ploylwjxf4zkJ6sVegcxfJ08tVtn4fw4U8IoSzlUL9AO7Cr1RLm\\\\\\\/3OgdpFVCVbCMRGVagsfMoxUhl7gM\\\\\\\/dHiCHFl6AhnBVF69Udc4Qia3e6cn+Vtya8jMPWSCODcxwlEvDE6k3mVPMh1GXTqjgbQgyiJo8mupnoN0m6aasWh+\\\\\\\/BhEIU3lsqdQXqrVVvdZ62E57Q5CRmNEfPuJu3rA0e18ffjBascgol5vRGFXpYMwCc6F\\\\\\\/BZIwgb3xGEI+88omOtsn\\\\\\\/1BQv6\\\\\\\/yPrdaMY45M5LucpIhvMND79bYjq339qr7F4qxE3IitBvuDck+GyhPm4Y5WIPplxkN0PDeSsDPHon3hUQl6AM635xUI6Ov6ppDGqNOjJq1KfHWfOR2wlLUxa8wq8+xMBv8PbzxxQsAuqX1ToYO\\\\\\\/6SzbhQfD\\\\\\\/NkulC+k5FuLz8AaKY74eIZMmDerencPizK35JmUDgSk6nl8GqPSBZpOJkdGhOqOv7cOCUZVThnrWzAGiZXTYpB3bvZbzYDC1WhsCcdJ6Z17BDVT3twgGW59ZHhHtXYXFRrmwAHpCiSGqVtfHIqu7p27smrONeE78b8zFhXzW8X09SUQR7cMCZXY9BdabjpyfwxJIUH3kOa8SsSZdCwAwKutXpmtBtLeSvVro6LO4d3i3UjOQpvEvliAsVLYUoDQYPRsk63RbSnxgZ8YcUOSsTMCy09WZmaSQE4EKRaEq1pYO5ROb08AjvzO46JU2V5oRTqpsZeUpe0suAvvVAPQkBbOlJ0mHKRtTskqv3g045zBnAkTPPTDzo2NRszxVnNT\\\\\\\/nuOGte1+anxbDUnmUmXsKzd6Kldo+3+znyq8g5J1TMtBN2T7VyIhjkQNCeaUQsyvt26dJ21yebOI9D8ki2vJ2kilj9d6njJHDI2i9hY4Ow9pBwYD1p+fejeoTxo9iKSVFYFVibgLJOC3QpY0IUIUefSi0hL7zzDwkbL+9Rhu21HfEl1qBm+PZgD\\\\\\\/4J6C\\\\\\\/\\\\\\\/IysGESvNadb3jHQED9Sb8FRtdWa2P1UnBVsjWOR2q0VLSOhQ\\\\\\\/QHQ+MRhXFPwhVtV\\\\\\\/IuFdqbmzX+MyR5t0NTzP5\\\\\\\/uoR925kSdNiDe46\\\\\\\/G+6m0vKLMaGKv9qSk45WDIVUraUMeY903xpzbZxEe5RbmML2bZlbUqWlm+Xg6KkP3f7HfpfSkhADTiQO+hM\\\\\\\/9qQjtfAKEVP+UDfyz\\\\\\\/D3yKtkI2YGMxfPMTgsEdyr6CbXW0z\\\\\\\/shFtRCEyOHKrJLuA+cEbVsr7VxgVcVkDnlwY2iimmoRtCxy7NN3lTHUgnT5L3+60M7lcFZ9mJkL8Q2oVMHHG7NoHUgErzlp98uZLmo2p7puE9OPutJ5yb3R0Nod4KQ5sdQ2e63xqArd7KyZUb4Xl+AUpju0xzzQUgTqQgQvRhcDPGfmuCXQcT3BTjZRPOCBm\\\\\\\/7ziyebi9Cpyw5Jfg72uoxra\\\\\\\/7oowK34lTQ\\\\\\\/y34xQyI6xXeF4lp4TAEAzEyQKSdeyrA9FFB2cBaQ0WXkTGZmnK\\\\\\\/6hfjXBUuV7NDSFl1AD6qACNUY3oGGJzMuUw3Ea\\\\\\\/GDI35CQgnRWwkJApOFMgIW27OwCI0l5NWGefWakLe78NNV5jvPGZA1QnZJwSiGSATQTsuO73RY2EbA9my28wPN1Itska1PyKJIsC8\\\\\\\/JKyXz8+gRQgATG68GFQzHP2OD4QoW1eaHo6QQeK1s1RE2wWYbfN3mbnR7z0dZpWowhEJSzgIYEq\\\\\\\/KQJoLmP2gkVVkLAnPI1tHuWe5\\\\\\\/EjugrHBkz7IKezr2BLrAtVflUjp7XQ4Pouzq0T0qJt4MV5Nsz7S0xn9kamQLgf\\\\\\\/YTVCf\\\\\\\/rRWLJdvz5Rz1Dk7qkLlL3YoBjPQfzoayUq1lIgGKos6zTpXpVI\\\\\\\/eKZhCZh8BfOYGIToU+xv1QLdfU\\\\\\\/JJ0w9VZtbpBZFZmb1gT9EurET7i6CDd4FtRSV+zwdqyuyc8Km\\\\\\\/Ze9USt+9xAmnQbVDJ01BbXY8oTfOzwQ2bbesE7G2Vac7dRg1Ti3ZOv7Jlqr6vlNeauHg9JjC+XZJcgoF+RQcvUaorFlw0zT+JYeeFm6HlAG3yKtm2BjM1kFanSEiHpj59q+cB4w59mBX+oKsZ\\\\\\\/5cgNvDi8tk\\\\\\\/u\\\\\\\/VDDb7Qavp79WKCIoLAe+brCRZEp9Y8+RejHJSNQrm3NU8GM1B8iihVpUDQbVc1sgPuB3GxXcOsmMzV5GSCKZQLtrYruwYbfcM0tCG9ychw5epNDojnL\\\\\\\/+VvFEQdQfq2g\\\\\\\/k0h8699mMclR3YtPvtaFPfk3rU5FGVQmuxr\\\\\\\/kRtgtOZly78eLHHh3obK9JwwfWCJd6rdxyuGye1qzCYMA+ewzil\\\\\\\/b4IyvpJ0Z42rKigih8f7A0PZqiSShp94wTkD3PGjlcNMUuPUkpR5ELjLS\\\\\\\/ANknqclHWCt0skzDaXSnVL3pafXe6Iu0qRf7dTJq26yalYdFXhP6jzR2BdASoJwbZdvThCN8ooESTPAoNWZNHoYf7+pJ3Zg+GlNueVgVjKbu3MnN1bpQ1bVl0tE6ibKmF44Z2\\\\\\\/3ZT6IaYgpaW7aDYeiOgjqnLEJAaYwYQZv4zYFo9T3KkHe2WC9CgWgm9ah5z8pWDXO3JkVb\\\\\\\/SWbt4b0CnWtUi7vuapuT6PdDBwWxSg5hr3ezLjZFIwGSLe97d2BJ58odULdqe8UXFII6N+uL5lSmD7l\\\\\\\/VPgavbgqCVS72fUkwbFNPXFZ9b6SmHfTQ52wZeebs3CIJklv7Qh9TyfQ+x0LTRqrKspMHQyztqjuX7\\\\\\\/jYEuOWgHAG2r7gXSTA16qStVTsfniGL8x3xgh+rRFyOLRImwITyzwbn0ZaHsXMmmR0GvWJAxGHtrPERdLQV+zDtR2JqRa67nIfRsWAMGQZZhc\\\\\\\/+IonvKmVez5SgpI5Ot0a6oIeoaJbp8Vmtmw6C0d9OPPhtqC4D1ueipoHOSUpQ4aOOrBHyAturpu3XZfWoNaCu2q5\\\\\\\/87ohIOnFG8QxyVw2zsvbK2RSam7bEZZuLq4eg4Zk1YnRRK6xjQ\\\\\\\/ipVWtCDX40wgmqGJFifKcf4BFdOPSqxTLYVM+v4H92pxmkrnLwzJ3KZbeskU+LqdxcRLYA7\\\\\\\/2aiQT1+l+5aPqAJAkYgBjXdH8dTwoGaFovqU+fONr9By9kwNrVkUcnDM348AazCDZ6Xg1IChCdyiT79S\\\\\\\/0YqsFJSeS+IhPMz+w4Ui7fDisQOt4rP8a59jQmnCpnp+LaO01o+x+aPjnLD+8w62OQWo4CHpol5i+yw9WnL2kFzi45mOv12pWWaB0t4POkv0OP4H6A13inmT9cKwFvkyNlFzCeWraS0NrFCu6UJOTcJ0fURkGDZXkXVpSxht0sctlmHo7UCB0KK1gtrceh4FiCV+DySDwGSkL9rUKfcaFj0fBatRuiO8mekAmPCjZUc0WJQcutt7eMURLKBSzn7ytmmT7gJ2NQQ5c73KwqyZEn32lOG0rr70XrbXFSIilWEYMBBpwUGvC+ZjK1oEtgH\\\\\\\/3wuBJK2EvlCxzJSAvUY9vwM2it+eWtMx5bh6wCyMnW+8bkIHkhoS1PZuDvSPz+rOnjS9IBOLlwT+o0QY2StAqU2VrmrFLqEWFtrcKiD0MVfn4pAMuGxcGsJQM4IQo8vKWJt5b31tg33IiGaMn0VEfFH3ZACjZsDDKlRpJ1vX8k4yg+DxJFbuzkXDu1MlDMnP1iF1Tft5CkZCjgFYPyZ87\\\\\\\/Hlscfg8aCTMbzqZ3kdCVpIEvBLIm8E+NjBJBIP9ncVzM2BiulGHkqH2y+soYhKAa3IVSpU1FQWnzEvh4Di\\\\\\\/Fio5aXnedcwOQ0fQH9mYUzJrpnja2X\\\\\\\/036aWwpSpprbVO9au3ZKwJtkEgQQJsOl1u0OdPCY0TLbBysJb\\\\\\\/LYGz\\\\\\\/y9mUYFRO6V9tdxTFgCgqedjq7ynfw477l5qEqymjapY\\\\\\\/R98QBeugTAsfSDrF0Ftmp1mqSkQtSylYW4wTdu0U7hHLAxvaNx0ysqoF9TgLmztoATgQu3mHoZKuPtZMFruXQJtB29BEZK8dqd4cLVsdvcmHWSw+98qDn8EXflcCfY2suAM7itiF5sGXrvwPFXHzv252hWOXp6Ofl7F8rbniyPIXBwgMczL0AqBLOshJgOkqovcMr7Ypc9LYjWIdRrUBwr9SL\\\\\\\/1a2YmVKbsMht7w5TT\\\\\\\/NGwEHx9Cm9MLv\\\\\\\/IhYoUfD5QfJG+fVaJMC4O5ohoplzzP0CudcXcSuEeNBexVUrjfWpGVheisonq4ZYPPCCz9fMmV4yQNiy2yMChF\\\\\\\/fbC2ZqohbtkOHZuIvT1T8snVhZtkIUWUE5aO5ST4nKFSrod44q\\\\\\\/3\\\\\\\/Rhe3ThwwdIldlXzE1r+57R92rOmIUJzikgTMmltKiT5B1D8M1cSQA4q2OTGH\\\\\\\/qdQ2NQ11tEuFF4w6hKHz0e3wXwakyVxXtfFtXsYIg1lojjIShelBWGj0DHjIwJvLDsM6m5qwI2OueCAr9dXJd2cRy802gbayg9iaroy4QbVVhI+hiNsgBqd78rg\\\\\\\/Hy6p3NwgpSsiQ3SkWTyDOeHRn9lGz3V5cIFPm+plI5uADOuplHY67K+kRkRU0iKF0Enuh0sF+vLpSvmNJhJps+l217gIA2iAlO+zEd\\\\\\\/\\\\\\\/N5kqYOUph6viqm3b1dJDSdfR\\\\\\\/l9OtXQbd3+8uhmZXtUmybL3m7hu2kA7qJobK1z3xdwnsn0IiwRfDE4lNbZFMR1gjshm9NODUVP7RowYrx9uWaArLE5pDsix0rMdi61eaSdcO7kTd9momiPgp37gaVxeuaQ1ZkR+EZcERN\\\\\\\/ijz7JLWqbJ8B8oLDNcLHsjcVKJcGWMGsM8N21lDREwBqS4z8Wb33CQewlBcJRkR0BEEil9L4\\\\\\\/FgzSgdiA40LTxV7mnJL0CV4vPJ4aBqIvCM9oqCGXZp9bgUbiOGyTNgDgsl5mjTjsQ4ldVm0\\\\\\\/Lf1Di+sciksDcykoVGcAfhrBs02x6myo5Re5IBbdx8g9+gpqVRiXigfKWp8GWvoKF3wXzMBvV6DB+BPfpRRwlbMEY3KC9iB8moUgLOd9xbAK\\\\\\\/CvC7LI2NT2HQG2H9F8P0gemJI1ZXtfgMfkgIcDG1K\\\\\\\/dF0DqVvBz9mp6ixWJMVg3ZSbT9eMlZKod+VyvVKLm+GuJpNIhQofbzvu1phX0yb+6ki3mCLUmL0gKQadCY61zdTr5q1XpXIdAxnzl\\\\\\\/GrOnkt0mpDri3dG2qrmjghb8PEA08qJ0pf645VxB\\\\\\\/cen33mC2RiynWT6vN+92PaKL0XCtVevYAcpTUoKFe9VDuTsM0x+vu9lv6iaCyHgRAgYG249nW9rC3TGM5RxCabBbxVGj\\\\\\\/QiMNvtZSpq\\\\\\\/pqorTzkqV2po95EHm2l8t5eHt79q3DWf7enOC1QVOFe9wdz0l73j9ny7shML1GuYDegYUh3\\\\\\\/41F5qjbHnuXiAZYHRGS+lwHSkPEO57j6ULVhviODx4QGA4M2GffAtvfRJ5fThSOxa5TfwUGK0Ee1DKUyVWY2tdtk53leqFRHZ\\\\\\\/iwT0l94F7\\\\\\\/nvzmWyxT132T9oVq9iAb1\\\\\\\/kKrw7d\\\\\\\/KdhAhW0C5W6IhdE\\\\\\\/+jMuhxax52FZu4vXX9B0JvP6uReHgRq3GgKZiraVMM647\\\\\\\/pDwb41zRPUVxNpv3Z0aAsVzOqkbaaJQSOfPoHMLuOx1vC07fPMIE5\\\\\\\/wF9L4y9u+oHwNBoZtrsoSKIAHWW7SK0yEql+Vo326c6Y7SHEBRwQ3Ak83JWCoDLi9rzS\\\\\\\/IKcvpFcb1xR1FyAlN9OVnxqewRD0twSzT1REZ\\\\\\\/\\\\\\\/gtNwiFM74qLwsUsI5GhmzZAtLtZGcT5jTAX2cOIcGn5yQJ3f+dYTKyFP5+A3xZyzdjJ22RNizunxBOMgbhcCWrOPAyyec9ZJ8WqlVvbyeLSc1uKLVM5rH7OgwWfMm\\\\\\\/pkZExQy6\\\\\\\/7t8ccegwP3bil\\\\\\\/bk9Az557hJYmFHoiqGkdBFBntr\\\\\\\/sCWNz8qbfDy7Qxh1g8tlGfRKoyLoMyUjJQtzCcFZw74DZusGCKE43vG80yjGjnfhWH32shl6hR8IkD+pLHMXLX5fTuJ\\\\\\\/Esy1CLriHZeO8Ik3gdh2QD4sWG4NyQw3M3n62Gj7G4e8vcLaKACC5L72pjjvE5FFL\\\\\\\/HVhCaHkODNkVuLoHpf+LLLtRwqzx8XhmHFcLHQVEOhVFqPDoAcMnp58bzjBVhEtnU6p7g5aeUepdEYYkHRxhItkIrNKnTPr3gdMHzPL4kP7HtYh8JXARMq2zO7DXIoD43VpEyAhhkD6jgfR23fTgBH\\\\\\\/wtr+bMvAegrQmsB9BPFhiryndd7HUm8ot0Ct5qBU8hybJEjE6KfR64Y9Da7KtzciURpYpKgLKM5g3Lg222cThjXaVTHaEIJABg5dbSVUWiKn1BiORxLB5SFKu7SKEDlRbJQtVh9XfuXTvPhrnvtbw7oMxPiK1AV6mmqjh2INJelQHHDlaBiOtAWmgykPNHl1vYiEMSNkSmEq5kLAqPzdLmCvKgk5seWz5xSfw0siqEgem6hAMgmNUZoAwMZtPxP71YzbEQRkMr3aUaLun\\\\\\\/NIRbqeHZqkf6xkzwKGRIADqKtAVFuhb71bmLqqe\\\\\\\/cSvC5hmCtc1+CxceLNNQ\\\\\\\/sCP73BCLPgQJVarqlZDjLLR2SRXgRZUxSWDKnnHvpraBQQFwd1AJtOYsUICzEBncF4jiHdoeMCTiicTPofuYZLF+W7d9umAU94sPsesVSpsx5mwnqer56QHG6zUqetAGGUOP2tYlckbwI+ycNACwCODV0mdiVM7QGR3G3EGwoM9rollMdnUG5IvewiVA1q6q6TzbgKEc78R4tjVvme8lw31lbpAhdxgqcYmBBpmTjVlVjymCLE7SP\\\\\\\/JqWFZemxDIC4wCbQ2qPT3XhveCdK0Nqa8SexY05yv16Tkd5nBi\\\\\\\/a8O8fiuylucLvBChyt1+20O5c4sYESBt9ddzuF+pAd7xdsgG\\\\\\\/qj8PsyGULD7TJiOQj\\\\\\\/CI0ysswAD1+Yrkx4LshOXjpJwut6ZJSL+PzmFJSClrWshwY5xu91FWa+uX3Q6N1UhpOgfviGUi84nJ9+TcQ0cIoGwXnyJcRWkUk4lGd55hkymDT9Y4NGkZNUfNKcx8JcV6ngLAutMGzGW0nWFovKrq6\\\\\\\/EeORxCi4dgyVFKmTew7SMBlwSQ8YzUsAJXtvMGLZPR\\\\\\\/mzkSzSefxurPATy4IqVwzHYuNkyZU36pw32TO2QDRPijeCMb3M8P0Hbz7O5oVd1122X5v2nquxJjbxe1cCLNqGDF6Z3TIASU6DNwJGP7E7lg8I\\\\\\\/fGnHQ\\\\\\\/LQm9Acd7aDb47Sle1qYov2uqZz1h1g20BFCeJ0S0oSzPnKutbUmFz2VCZ8kxkxF9wVS2NNFIJ+uFFkWKFajc94TZU7FOM0pzFdkrBpNobZOzePzr4mei6fUI5dW7lwPxJJZJR9sQr5vPLbrI311Ft\\\\\\\/sEv4c\\\\\\\/Twga140R+ChqxJnR2KGfrFv3BXTJaSZxBzw\\\\\\\/gWeICQy3mr4E\\\\\\\/9iKuqz00a616SUFJsHD9tegd\\\\\\\/RXEycCgbANdfbzmh3ylMbtbOFq+Yhd0AFfyeYz7AVDu82GePMAVrsPdrQKBwa6Nc\\\\\\\/6u5L6rjgnd8r7YPA+4Ho5ItyD1503cKSGI96DKB4mLzzFbPCqWHbj81guHt75kM1krGRH+iZnyQee23EmEezbU2WaUpLnMyi28iEiytpgC1bQ668HPdpHPLxTvt5HrFTSub3W3NIvFKaEdF2ULcgs04suys\\\\\\\/0raFeG7yYHiFz5jB8+13oMrQ3LQuX8wLNHHUI6IAozk2rsAOF4lTd+j\\\\\\\/vlOCsK7ZyGVU0sY0eG7\\\\\\\/zXJtBBmhSbH1dgIBHcgzPKlrc9ZQJPJS4RHdbQnrJNDSLT8Tm9iOqDRx3CDXAd5SISWZ6Ofim4SS1jcXmCzr09dWa2W\\\\\\\/bg3A25SBGJ81hQo6bEK0rNE5LIQybK4Kxi2gnCl019Qmg78YoBoC72DsaeLm0BUQHGknuQU4GMQIlnbddm9lMDZKJk5KCpRdgRdXkY321rT82iCU2DQ4h1AXunzsGWlaCEhXv+C2IG\\\\\\\/4QpdBXSeGcSVWkt9ARFLaXXWnWz5vm0N0CHHRpgjgh87Dkb5urLf2ckRegXIIlCGc2I7W0tWMsokmACgDTznIB6DDxxm5FTWHD9NOblhQJlkpId+GNM0DEhP0qydgO8hBkp8ByIeyTLZ+6k78\\\\\\\/ktPzGHICYqO5bqR0MqyfIxKgHAurgPPxEGJJZBmfd2jxzjvI0SWO0D0ykLmrIDzmvXZ+B9up5xXWYll09G0cjNLx3RcZHsEgYfW9GldFt6LI91I0kDloZKVV+Fdj9SXGrSk9O+dwjyJmzxuUtubrQfqCb9qJvJi9TkKU2c2j05Lv1EnEfAx4uQOKz66FQt6uozxuYaU9cI8O\\\\\\\/CtMhSnANVLF7ebLG3BoCcjv0wOW0PKfywNxjUcRiw9XUBhddlVgbeA8770iZyl1sJXvSjcw8AVHnJUvGTDmY4wLAyFEdTkIbrWbvt5Px9SXJvb5lB6pxYU0tom\\\\\\\/JrQ1GmQ4BtfkQwmjlha4b403W5cCq\\\\\\\/umubBEBZ+8FWeyvMXTjepXwCsCDBE1vq78BYoOBF8EFrCms+dLlaBWp+enCS+MwRplSLvITW1HToCNeVyFh0q5GCr55njbFkV8RQmcCYFNicqfxX8XVW2o84Oju7chtKJ1hkQIqlFXapxWuuq2dajz\\\\\\\/Buh47NTF1cxwXQjbyw3+l3Il3cXcKH\\\\\\\/3XDLDxn4DaiySruetsY8G6DzKddExDimhCg4jEFNSiKRNROWcZakC4ADv51gdf8dGZAXxXHF3WwR7CXAbd1RaX\\\\\\\/KfyQneraOy+zbQxypEAI2\\\\\\\/GuI5UqV5tKeARHz5JDjbM1bIG3BGiNqWMw\\\\\\\/3cEiEk7DA4JyMFFXGjr7AtmdyKXYYZdjsjIhUo0KDIRROfALydWaqkO9rS5+c7uNENcU05iGNCp3ZOFL\\\\\\\/ure8RQOuVuMKH4HaeB\\\\\\\/CxU9XrdjyvGQRoUkiH+d6IP6gd\\\\\\\/ib6mWubrhg\\\\\\\/f3hpMQ0ROU33Lwqvgj2M63s1oQC5DzIub3SkkImEWaU4UI6CXf5vcH2xbaxpEStCvgr3L9zkodpFcEEEw5dwipQDWWNTz7ATcywxgLz8JE+42+P2+3E4G+fxQfJuTwkubDqTRiwoKW540ggFAjXOl2DV1wiLYqnEkJcS0vm2nu5QcvBJPIPMLW9K5yyYiJuCOKKSYSq\\\\\\\/CTTVBe\\\\\\\/3vOYXwCA3P783skpyht5nEcKztpuknwIJYdAAh1VkZ1fnpK+8xNEO474w6TkZ5stYxuSRHaEDsTrazIclyJKvDbIQuJLVbrBIVZgOZ4MlzVoe\\\\\\\/L\\\\\\\/G+DlpM9wq9XvDRTf8dBi\\\\\\\/y6hRH2tRsi9hSiteqY\\\\\\\/cSD2DNAzyME0vsRWgtVREe7TMPHiZq8gAIWhj0hyjouU56O63tLmxbqyvAeAgi2hfyvA5FPI0DEhlHBjjKltccTe1fsDS0qqb2bT9sCHMFUEIUEcfY3JG\\\\\\\/jUchv1Po\\\\\\\/e41K9Ps5s3tV1+ToE\\\\\\\/KKqNIpdjfj8xYtoJJz0kz7vmWxhdMjA3WqDsj+OowbkEKVpZO\\\\\\\/2BhTglX2l27b1WoP4Ab\\\\\\\/QxtZcP6XCrRGKgWVBG6PKznFPOQF87i5hXTYsB\\\\\\\/TLM7cx4UpKxx6Ih6obX6zlG1Z1aXxK27xSbVfESH3MLhr9IVdEbLgZ+IL3kUszPSWR+1vX+ecytPztWp6CGm+WBpQrPnqdw7GECLt2aQ82Ch8hGyYAdJKDnit9JyTaNsnih3cloOzlewV1SJhYwSxZlDx1pBHnaInbp337cax1kJ02WFqoUtUOeYDe8aczSXUl6PuBHXcPsRsAL+qu4Rg2ggIuQDu4Hb\\\\\\\/jUUAPa+vHRWI3qJYu68VUAAK86zKlRh4bGeXi1l6Nhz52d7u2C62mRCCwfxG\\\\\\\/SHfSLSxPSzUIuYsyTZD0zg1BRw2\\\\\\\/jM9swf0dv9SNoX+o9izpn\\\\\\\/wyNMfTtzzNy45YOi2x9sO7WvGVxrEi5vxney1XWP9Vp3oDJi0+aekDqJhtoGX2Mnj1fL03O9WYUFs6CEVu9AtTtP7NB5mRMwMj+lSDzONp+h6lavB0y12aiENc0zbwzGMEAxzKizL\\\\\\\/vmXaOqec5HzjCPMDeI\\\\\\\/qc+L5bs0Gvwcy7J4bCFRqQY0w9hQP4ZzDVdgSWEs1T2fcq5DqzV1OrxzTY3+RiHMTOz8\\\\\\\/9YC+GZiu+maVkXwiZQphQDnjNzQ\\\\\\\/axj\\\\\\\/RYhSDQ4K+CQk236a7Z7UzSSvH9z49RIyccM\\\\\\\/h6Vn\\\\\\\/DMJO3ch1Jog7QF18iFhE\\\\\\\/ScuHhwflXL1pXbZ\\\\\\\/IXzJjExQwoo56rWBZoLbpnw9dZQ5KpL1yAq5ig\\\\\\\/zeo1ZS5ex0POpRsC7RHoLqOPecyD0Ijcb3mTXpftskG+UEyB2I8qaZxPfRg+zVfeAEDPMBG8IkPyG0ZRE9xboJdCHqQY9yDTaguq2uixR7I6N6mtez7x13QKg5cc0u3AI3E0jpjijlxrHTK7HDqzvFDUVT46nerGnRAwbglkx\\\\\\\/aMm8Yp9T2kEFVcZRnl5okj1J15aFcNxahWNOllR5pYqgSa4hfpTtvtvIprZM\\\\\\\/Sfjjh5DqIXLb+lnc2EgIrKKVqHqQLp\\\\\\\/OA+L28dBOV5p0x\\\\\\\/xO1qrrCYqIVx1qqC5RF8i8+qTwVh5aBpN1YZHjGf3o6wnSrd+XA5L9i5rtmPzjGvPYASCY39wv8ByWk9Yu3OQumq3mxEe9cw6Bd1N43jalTzBORjNwzlbvXXRp1xYSu8IXWcdBMQz+uTaVb6ydlx0XUJiTBcfKlgywcBDJY+Q4iZYevH+MG4GS\\\\\\\/3JLqzUbR9UG7EsYBlYS+d8Q33BT+wQxMQI\\\\\\\/v0BwJfwBIaXdJOBW29IkuzLsqBaiuoP5C7\\\\\\\/EaqyYO7HC34ITFFKobUxWSvYXsu2DN\\\\\\\/b02rclu5DrgInIw9JebQ5GOtvRcOfgakBZCtQbdo4AlZw8VtBdm6wAiWmlXqGNLF+cPL\\\\\\\/kwAfkBkL+h9cerIsX1CpmC7ap4j5nG1tak1yGeQy5dz6C7lE1536k2b3pgEBwCiajKOu0YMEmJrzx5EuVbUi1uJJyuWd1pCeDa2OM34bnvHLlZqeKjKvp5oK0119B2KADqV31LiPfRQGk+5GAVFUOErBLqNk\\\\\\\/VN0OQOeiKm4N9rkR5PCZS2EPt9Md2o6kla804a905feaNFG6vzQCgUWBgT5G2tUMoazafffNsTfV1w5JMrw2AybUdKwMDv5KUi3+dsX2kxm\\\\\\\/7HhH0uzihaTMr2FY3YtmWwidRErSuJ7MtyzhxF0i9Q2a45d76zK0jfDaXxjlhss5XhUW4JJqlLPciCEqnTBTnZ9OC38ib5nWj+DikE1HG77xPBBexON+Wb81\\\\\\\/t6UM+gS5f+quK6JmN5YbOyeEchaglZTPNXUtBx2+\\\\\\\/grVj1G0Xg\\\\\\\/HyqtmtJC5sw6KXJvMJ9NFKfSz7H5\\\\\\\/4f9S1rYNWz7VwBS+NBbXQousPK1ZJpj6yPF32x+KD1iWQQz36z4o1nRfKYK+fYVhk\\\\\\\/G35DAWPpJHPt1HbIKN1ZEfhNR4Mo6PuSJoxQ4i+kVY4QjAXtmHx5ngESHMuDfDKcQGVo\\\\\\\/cBVfM05y\\\\\\\/lsvlSVEO8beZJT23wMiOdMmmlBK3F1bC6rKmWtvH2zIjjtqQ73hj+TvJpxjSo\\\\\\\/RUYlwGGPTkyV7wA9\\\\\\\/MkVJVHbBG2Y+GCWmDsjaZF0DusYBf5w\\\\\\\/q17kKI\\\\\\\/EqtFzCgXzq31\\\\\\\/kMCcuaXkORoTGJBRjXrSDzRZV\\\\\\\/uabeayc05mZ9YwNrcbFi7r9bgjoWeAVn9NVkoNMBm+6maDkxNlLywQNTPIVREN4M2Qh+Hb62iNL2sBMs\\\\\\\/MwNqkrUZq5NDkuvy5GuUJXuopioQ2Hzr8DC87TF3CTEesNZv9lri8td\\\\\\\/FMcISs4JGQUVEDQ8aJdHtfoOvtlUIxi0jCCMCaZmCnmWdc2+5TdKcigY0d4KEI2C3so\\\\\\\/8ahYujQidprZ3EYoeolZDDiEh2wukaXJw0+OOGletjfFJRZTLlX0s8RkQ4Owb+FPUHjBbJDKZ4iR44tIScHf9GkkLKuA+mv2w5Zb4ZIVprLfddc99SGx2vyVVlAEw6BxMI3AnQWKlM9X14ML2EGvWdVp0eFjh4Vd+l+yJ5r3E2Y3Ac82CyjVbEpY37QYtsKzfCWI2p6bVP0r5bMiSKOEEp1v+7r4NanpxeABoS\\\\\\\/h31trfWYwXKvNOb\\\\\\\/XB5ml+dOJyS\\\\\\\/2w15Zz1zktGXhFIPE0Ana27DxD6Lx\\\\\\\/u75+WgIiL6w+dw9uv\\\\\\\/7DEYdvO2lTIxrARlpLS2QUT6q42qeWCHYvo4uDGoViEtac0QifQN5IR2Dy1+NljUismej3DAHCXMk3hQFnxzo2nNAYFcl3gB3FI5prXhua5HJduNzKUxc7RJG9ClZK7UQ\\\\\\\/77b5ENaUfHkfxmNIeMUcMIGJNz5\\\\\\\/wXpxvnVnsTQ\\\\\\\/TaKq\\\\\\\/z3Jk\\\\\\\/upbE0ZRyWwtadAO02i\\\\\\\/Om0Ur1ECG2qiZB6tqoxwGF3M3WcbwME0KyIIvbmzSfhnRi0cu5CJU5TQO1r0KM3sw1ENKLuWGnYZjSUKTvE7WQTq+xL+I4NR3PHKoaul7oyE4ln6HN\\\\\\\/U+f5YDya7WPEOZbcarAdgYqD5rXDT\\\\\\\/x\\\\\\\/fvc2tfUUsqTH+q1sEQJi4rqoDREyzxCg6wI\\\\\\\/ykKlitKtYAFiKUCa9fIqrgf9o5FZVA4KVaHeLfa2KqgyRiwuk\\\\\\\/NBQC6KXyI0sRfwGOaY2EYZhOfT1CvfD4gubgf+71d2XW8B+v\\\\\\\/27E+tw523EdjfYsIs+OIYyRr5qsXrGUePpMzsIfd0GrOLZf2jIiXsM0548v6ClOugO\\\\\\\/IdYV4k8xmuxQxoE+fsjmX2aYQ5UW1xW3fKLfGZtS3er6K\\\\\\\/vH+dDVWA8l1a8mV+GbCunMAAqEofURan5CMd0yPWN4Pz\\\\\\\/gTszKNmTMUY1\\\\\\\/w\\\\\\\/YmxIFB7zv7oiEPVd62nJ96VXYRBLqEK+xPE1yUsDekIUZcburZrDEpDoYOaTIVYdjLvNMrGzAKkZf4aDKuy64dZlLrDzEJYwXWw76xa0eZ03XuFl9dbPHF\\\\\\\/0UZKWQBaM7YEPVLsK7Tf+HceiG9kXfvlqxXdCtv5OJvnZ\\\\\\\/ZTTOJ8O1lMe1LoSEOTb0uN4vfQwPqo4JoNU8qYRxIPpvCphfMUohP\\\\\\\/qR+p5SsNBAbfGtR9bmDmtQTno+0J\\\\\\\/sxDzVrFF+CkYTK7LbiZrsKCckVUfew4ujae0uzRDJATAgSuqDfGPJTEg7IK3iXNs43tg2dPxpBd\\\\\\\/Q0lqhTiu4a3avSYnLG455TGSyMhUA\\\\\\\/1Zj+WYhb1UNXexpVSCu8lYaZk7JhuVC+6\\\\\\\/jo1\\\\\\\/JvjJJo3yipAvDkjPCwdGnLRnZPCs5WG7JDG9GUv\\\\\\\/bJxrWMEUIHRlwNes\\\\\\\/Q\\\\\\\/i6NsrE\\\\\\\/5cl0mZxKuInY5japQwEc62VKWERsLr9MZfeGhmNWtRP4ac1smRr4K2hGAAaGMQ5nGAVzFPzeTF+enIWihucrxALttSN1rf5zbxwAL183U3OvoRR1vTYSikrqBwQswrw2MqKVSvmoxivokgakLTZLmJy0jYx3uBpsuTiHfLaClxv+EuTvCuowtH+SZ2NMv+DSfZEKzwpHogbacwgzW3Ln5b9jOz70Ma4ULKy2mIxAgYnY6Mao2qAzSqOY05huCsj36RVpytJ6RA+DLhf81xxnOr\\\\\\\/HMZ8Wi3DnSLw+tZYw1zHGZnESDxfWftY5VNDGfaz9IJTz\\\\\\\/ayNb4QqcQZ23htgYzSRZ8vd2ImB8FzaYY7Jrs6fIuzVEv\\\\\\\/hhlLqdyuh4vV3CaAxNPmznyV8Ozq2LB4lqcs+gnnXO5Yh\\\\\\\/ObL8EjEavtHD1BghsU9MtkDSygfZ51XGPinDJqfLUPVziZWBe4XPzj6kclQ7kGdm7\\\\\\\/reRvGid+8H2baoDZidBfldjNpz0B5L95WoQIF\\\\\\\/s6DVITbtOiSUpDtA5wwo8KxVbIcMi1JgJOaUazvcg875IqDNMY3+RuYvsg+iqTlWM8Z9sYyr676vuvyJfcdnG9NgZ26goDshKHD1FZjVbBud\\\\\\\/BpGBjqi7x+iL7zEPxo1DueVXfoWdRZqwY+1fPikrajCIj72+qIaEjJ79MWuL\\\\\\\/DN7OvO8EVL+YT18nq3ti\\\\\\\/L4zL0LsOcJU03HjIKpyd3fZVmZPeUs6z8evMUGgNOWg1Hwu1FwTiku1UjAHurpwxmAh\\\\\\\/3xViA1X\\\\\\\/C+ZfCipH0VDXi9c05\\\\\\\/hacu\\\\\\\/JCjobuEWdcA02Y4I0fnXmP00y2IKhxAYi3yUbFnull2GNLR6YibG4hOwLa5lpDXWjCOzUYmX3PCMgR2gQdSWwSV03QEQp7beag+8Qg6Tm7mNr6OgaozGVVkmH7BFrX8prXVXTChJJXn\\\\\\\/AaZZ7NTcaNUstBoC6UEWjUsddNWeSDqxLFu5JLGA0IID0R6VWmEFsL70OaU+BiVPLstkiH39a9BlqN9AGMqk8xPj+flElgN\\\\\\\/il22jm\\\\\\\/1zYabneGAAa1JBActJx2\\\\\\\/I4IA7jOsdsd9uCRaxUbLMddHO5N7pj8Z3GXtcXe2g5LulBohaCdBgN7ny9UDP2JdLgQmewHwx23kw1ji\\\\\\\/Io9utaxwpVsjwQIkGRyMv0iUP2Vii7kz+YEvgmPa1X2XoB2vdqC+4cSUk6EYBqjE7w3MXJ06zzOv21q\\\\\\\/BMqJl9WQ9btQCfFIhqCuI6EBBmufYIqKfDncZla5DDpuB3AXtrKFxLSn4aYC91Hi2\\\\\\\/fNJNzd7eaxbPuuxvlsRk9ma\\\\\\\/2UMABVeWKYjvNhS5bj6+e2uBFAxk9z\\\\\\\/W6UvKL89sEcRbWaGzIZdyW0G0+P\\\\\\\/MivliGAOX02lEO\\\\\\\/EqqCiWKR65pIk4WN6QnXEoLvYU+\\\\\\\/f4ggxYLZMRR1XpuLRs54Y2izlGRQRlM+VZMY3GGd76iox\\\\\\\/h59RB9xV8PgMUnQmrrr7z7rBPF5r\\\\\\\/19tj0cmwW1d2e+x8qH4jKyf1grah7CuUvHyRj8EC\\\\\\\/Zrm8gMmEqVvxOUroUbqzgIRUspkjgF8Fzd7D\\\\\\\/Q+yatC3fvJsnIbwJZHTQ8J7BDOdzBdHwJ+Gk1jHmH8Hbn9qHP8diaFRuzpQMjbutPRn6BpmpPEM0zJOKRqojORm\\\\\\\/WfMdDuRmQZkMr+zBy9F90pDXqRsX42BPJME56uOVeQ0d9PEaXXHjqqtwPtwkw4AVW+hHhlZMZRgdhnX98TJL7N51Tc3J7dFRbbTAJ\\\\\\\/Dgjbp4zvBUPQD5HKIWmpHBJKKbZqqYEwKRpmo4saLs7T4qQ6Cq6\\\\\\\/gl5FHDXvH4wyEsnnVeAzRGOlwMFNn0TvgHhyI3JArFSULBRyCgAKHc+mkmiCmSQNZB60rY5CyUNxHLHqIN+nYzUKqIZQqKCgpXl2xWBuHkZd4QE1bEciQbEomkF0Xqbid2t7EJFuuhQ2jLZHF6Alu5LC+c04LBYE6PRCG0LzUMklVGNirciGv25VFog4hZrXKd5wOklBM2LLC8eEt+eyQ6wz22Xn2ip0PDHRmXUbHWimaPwvHqFheL5YqrBSvBe6\\\\\\\/ULbFiu4b9XY9RCFgVxFGKdLhC32lL9l7HZCd2qv2FUnv58wXOuh3ZZntxakYTDO6gp35eamMpj18vfQsnUKJw8QmJcmYgWt7\\\\\\\/F6zifAWpeG69nXwT60IBGPyF6430d4zLLXkeRFGXs0cDC\\\\\\\/azpQVduuiV\\\\\\\/EGLSLEV7qo\\\\\\\/NhKGuYxEUHvcNMd4YhHnWphDgQNHJszHy76NqygQtpCTzeHtJm4Ywi3j411A5LPttravjKQTp6Rs7WhFgWP0B6Qu8+E1iqAZZXyNQFDh0tesM9j5FNZL6yHc2D5Uu8UG4ZRgJlYW4zt51MniCut7Vu2x45QvSMbkRSDKdCcwGVglPfmTW0LSQOan7mt5\\\\\\\/bzmQtnNOVIvB+ai8zt8G3BNXefYwf1mTPbLzW5Dq5YM\\\\\\\/GBLrSkvOtuhhLepIQqTTXqgPlyp51ovWm+tjOn\\\\\\\/E1VOWYdSVLnVoOTI+nKdVxbFwMOjhXZNAXzSp24rgXCbckumGcJjj8aap+pqB6qJbTxqx3A8yIoIDVLWdA6FJXEhuMpNiePOSG6UglQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00229j+LrJAvGC4\\\\\\\/Gr99xVhyeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224b6f31959828459f30225a00a67901e4cfb2d66233366b8e6f7f14f30054716a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Cy0R9+QZkampxtcwgzWRsg8f5GdNVJ9jXlc4E\\\\\\\/xMyHv2Xnv61UbBDhqY2KUk\\\\\\\/\\\\\\\/tZF1pcJW1IVbKKe260HoZE8VYdMA569xOZtwBDv2dSI7+qRnLRg0jmUP\\\\\\\/VmtM061XQbl+MZPobDlVvk\\\\\\\/I2w0trJT3k55qgUZshcmwY\\\\\\\/3c3Fiu\\\\\\\/yNOoQRqVCsvtm42d8B7stZAz7O6zzXTrvcuheDl6ZrBUxJ3ThfuswvtJk9pmWtazTIOkoHKUdHrS2tynHJcOLbjgRA7xkQSFkTCj4X\\\\\\\/PzZ8w020qQZhvl8Nh8TcXAbmggdjILuo4wcMohf1ksxRdvOdNKvMGAgnzVxYNyjqfrf3bShXjqd8U2vPe\\\\\\\/HA+7WTdf6Raf3h1c\\\\\\\/EGuOKqKrBMaVKsMkwIi5vVorfeJ\\\\\\\/Cqt317VCoV29QGF4lyp9ffz6Eu9GWelb66k7zoXruVUCaC\\\\\\\/hnruFC0XF4A7DqiQFSbF9N4aKfEtaghFd+ISDvNc8kxo\\\\\\\/IsaZuUInZGF9kQtNu+MKbp9p29xYsYyTeGooMya7su7nz1gjSm8nrcD8OdiYNIj7dMt9zjrhOoq+PJlVr5TvXomM7Rgj8INYPp27IkfmZBDZuguuix745cRaOjJ1kJj3C5ixF1OIUxwSTsUr0IGasuv2Px+nnIn2w11FFRzAwNpFVgw0laDXagk1amISuofLC2g2m7WNVf4w1s6jl8DQUN8VVPF38zGauii1lE8oUTHWg7cL71GEScG2l5TlyebF30I8L3u99yRuL7ofTEQ0pGhqJ0Q2Eyk2XVlU3WPOyYsOEDqEsD\\\\\\\/cZZYzmL\\\\\\\/xwdtpTlQdDqRFoet6mx2Fn40yEy2SViPvdruEhv65kttraV3RZLralLSG1+nDbauY6vEQ7BOlraIkItMljCmPLpVvn7qru9sOqvLboE2Mhvj86Bqv5LRyDKi7p9nqrbNtBgejVe0vDilQhOgf1D8WCnThdvhhgE+Ry0WNxCMVz08IvPYRwRQf80sm87aUiK0vcs\\\\\\\/6nrAj0mfsO2H2Av0o7usOyNwQP\\\\\\\/2rSM\\\\\\\/Y7lhXMERnazarHwGb\\\\\\\/Fq9GiJBvMrcxqPcF7jJcWXd0A+hH+FqV9U5FexT+1XbwlFf3nkU+s9YaJCmkSkRoWs5TJAeKrs78RnvrJaLAgmetUZoWKNqxeiuah0Imsn0JIDp+vnOIuKprOJFt64yTYP32CLbUN2\\\\\\\/CN+bsqz3gjbPoavqESQpKDM7sUJgJNNH5K4h9sHO\\\\\\\/9fjfYMqYcnkpFo5HmkseYwKtQUOT8XoQ3UpYsU8N3lSMnluljIu8cbL+WPzswSLF7hk3DZuhXdhegS+5080qfFN4pFKdeHms\\\\\\\/C3F6JS7md7zu2xEijIs+qfzBRe09v9+9WbLB7O0WDs3jAoBOWvdL3PemhTyo0ibX6NkkjNm52E9uAloNWSpVkDTd+3V\\\\\\\/1XZGBy1dWPzLZC+JY85yXrQKSQBogSIEJdY9zPrsmt2qFQ9HV9dQ09lrGZsZIqXqWITThv6GOz5Tuxnh6Get2yRSusFS6Iw4QLeBn+1X8\\\\\\\/oqPp7Lungr5AgGnhC\\\\\\\/c6+PYWVE954sT\\\\\\\/wrvToQG6UT95erl8IEDYr4rzhvBCwB9kGzdCed07XwC\\\\\\\/cy7F7Zgl1AK2pAn4oF4B6muWmPWbgDGSq86PPAjeEk77dT5dHYZum\\\\\\\/6J4g4o8zjbMSLn\\\\\\\/IHAMFtDPOn0wlT+rUvwKLcNM\\\\\\\/ym\\\\\\\/8ONetSogS\\\\\\\/rOJDJrEZKrR8GmuoNUJaA95YCPQDWTacdkOpIs7R9\\\\\\\/CfQRTtPdFmXsBQ6jmskGLhYSFG6V5zu4CN3uFZGgQ9bsfNz51EfRa7uCP0tC4RgKrxqAGOHNYsh\\\\\\\/cwiUiyoOoM9jjFrobOdGgGTgOFebEHjobSrPMxtERHnSs3LxhsJTVgLEH2enkq76PRsBi94gBvO74pBRmex\\\\\\\/NaEiLrWujSIHMLvrL7yKfM+DEqfZj8L21k4GNYzNhq10rj+1hA6Oo5pEOyH5Fd\\\\\\\/ypqqEQ1V5nT8IFGHJUsaArZf7FOg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jx444YI4hR3HKMtHBiEMJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227790c3fadd44f758a2ba73f0799b0bca7f15b5e11f0e1b5de02ee3626a06d448\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459795 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/629" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tTFok4yGypyWonHP4aUUkAdhTtN8DddvyEJJSHqv_bM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/629" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-m7fwBbMO_gmaYVACuqYJPCtO_YPKpzhGXKC00eq-nW0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/615" title="Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/639" title="Conserved RNA-Binding Proteins Required for Dendrite Morphogenesis in Caenorhabditis elegans Sensory Neurons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459795" id="node459795" data-pisa="ggg;5/4/629" data-pisa-master="ggg;g3.115.017095" data-apath="/ggg/5/4/629.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">M. Cyrus Maher</span> and <span class="highwire-citation-author" data-delta="1">Ryan D. Hernandez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">629-638;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017095 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459795" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Cyrus Maher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, University of California, San Francisco, University of California, San Francisco, San Francisco, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BCyrus%2BMaher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maher%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BCyrus%2BMaher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan D. Hernandez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, San Francisco, California</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Human Genetics, University of California, San Francisco, San Francisco, California</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Quantitative Biosciences (QB3), University of California, San Francisco, San Francisco, California 94158</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BD.%2BHernandez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hernandez%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BD.%2BHernandez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ryan.hernandez@ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">ryan.hernandez@ucsf.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/629" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459795" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459795/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/629.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459795" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459795/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/629.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459795" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459795/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/629.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459795" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459795/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/629.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459795" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459795/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Ortholog detection (OD) is a lynchpin of most statistical methods in comparative genomics. This task involves accurately identifying genes across species that descend from a common ancestral sequence. OD methods comprise a wide variety of approaches, each with their own benefits and costs under a variety of evolutionary and practical scenarios. In this article, we examine the proteomes of ten mammals by using four methodologically distinct, rigorously filtered OD methods. In head-to-head comparisons, we find that these algorithms significantly outperform one another for 38–45% of the genes analyzed. We leverage this high complementarity through the development MOSAIC, or <strong>M</strong>ultiple <strong>O</strong>rthologous <strong>S</strong>equence <strong>A</strong>nalysis and <strong>I</strong>ntegration by <strong>C</strong>luster optimization, the first tool for integrating methodologically diverse OD methods. Relative to the four methods examined, MOSAIC more than <em>quintuples</em> the number of alignments for which all species are present while simultaneously maintaining or improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality. Further, this improvement in alignment quality yields more confidently aligned sites and higher levels of overall conservation, while simultaneously detecting of up to 180% more positively selected sites. We close by highlighting a MOSAIC-specific positively selected sites near the active site of TPSAB1, an enzyme linked to asthma, heart disease, and irritable bowel disease. MOSAIC alignments, source code, and full documentation are available at <a href="http://pythonhosted.org/bio-MOSAIC">http://pythonhosted.org/bio-MOSAIC</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multiple-sequence-alignment" class="hw-term hw-article-keyword hw-article-keyword-multiple-sequence-alignment" rel="nofollow">multiple sequence alignment</a></li><li class="kwd"><a href="/keyword/ortholog-detection" class="hw-term hw-article-keyword hw-article-keyword-ortholog-detection" rel="nofollow">ortholog detection</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/positive-selection" class="hw-term hw-article-keyword hw-article-keyword-positive-selection" rel="nofollow">positive selection</a></li><li class="kwd"><a href="/keyword/open-source-software" class="hw-term hw-article-keyword hw-article-keyword-open-source-software" rel="nofollow">open source software</a></li></ul><p id="p-4">Orthologs are genes that derive from a common ancestral gene but that have diverged from one another through speciation. This route is in contrast to paralogs, which arise through gene duplication within a given genome. It is common in comparative genomics and phylogenetics to extract evolutionary information about a particular gene from its alignment with orthologous sequences. To enable this analysis, orthologs must first be inferred, making ortholog detection (OD) an indispensible first step in a variety of phylogenetic inference tasks (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ciccarelli <em>et al.</em> 2006</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yandell and Ence 2012</a>).</p><p id="p-5">In general, existing OD methods can be classified as tree-based, graph-based, or a hybrid of the two (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Altenhoff and Dessimoz 2012</a>). Tree-based methods may use reconciliation techniques between gene and species trees or may rely on the gene tree alone. Graph-based methods can use a variety of metrics to quantify similarity between sequences, including Basic Local Alignment Search Tool (BLAST) scores or sequence identity. Information about the conserved gene neighborhood also may be included in this context. Techniques such as Markov clustering may then be applied to create orthologous groups, or one may simply define clusters based on a graph’s existing connections (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kuzniar <em>et al.</em> 2008</a>).</p><p id="p-6">Unfortunately, the few benchmarking studies that have sampled broadly from this methodologic diversity have provided equivocal results. Although there are general patterns in relative effectiveness between methods, performance is highly context-dependent and does not always favor more sophisticated approaches (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hulsen <em>et al.</em> 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altenhoff and Dessimoz 2009a</a>). This is discouraging from the point of view of identifying a single best OD method, but it also suggests a new and relatively facile avenue for methodologic improvement. By harnessing differences between OD methods, a wide variety of algorithms may play complementary roles within a cooperative inference framework.</p><p id="p-7">We begin our analysis with a comprehensive comparison of four popular and methodologically distinct OD methods: (1) MultiParanoid, a reciprocal-BLAST plus Markov clustering method (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexeyenko <em>et al.</em> 2006</a>); (2) TBA, a synteny-based aligner used to produce University of California Santa Cruz’s MultiZ alignments (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blanchette <em>et al.</em> 2004</a>); (3) six-frame translated BLAT, a fast, approximately-scored protein query approach that does not rely on predicted proteomes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kent 2002</a>); and (4) OMA, a well-established tree-graph hybrid method (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Altenhoff <em>et al.</em> 2011</a>). Applying these methods to OD in a range of primates and closely related mammals, we demonstrate that methodological performance varies widely by species and appears to depend critically on genome quality.</p><p id="p-8">Next, we characterize the striking performance gains yielded by combining these methods. This is demonstrated using sequence identity, phylogenetic tree concordance, and hidden Markov model-based functional agreement. This improvement in alignment quality translates to higher estimated levels of overall conservation. At the same time, we detect up to 180% more positively selected sites. We close by highlighting a novel positively selected site (PSS) near the active site of TPSAB1, an enzyme linked to asthma and irritable bowel disease.</p><p id="p-9">The implementation of this novel approach for the integration of diverse ortholog detection methods is presented as the software tool, MOSAIC, or <strong>M</strong>ultiple <strong>O</strong>rthologous <strong>S</strong>equence <strong>A</strong>nalysis and <strong>I</strong>ntegration by <strong>C</strong>luster optimization. MOSAIC is implemented as a well-documented python package that can be installed using easy_install bio-mosaic from the command-line. MOSAIC alignments, source code, and full documentation are available at <a href="http://pythonhosted.org/bio-MOSAIC">http://pythonhosted.org/bio-MOSAIC</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Retrieval of orthologs</h3><p id="p-10">For each human consensus coding sequence (version GRCh37.p9), we sought to retrieve orthologs for chimp, gorilla, orangutan, rhesus macaque, marmoset, bushbaby, cat, cow, and horse using four methodologically distinct methods: (1) MultiParanoid (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alexeyenko <em>et al.</em> 2006</a>); (2) TBA (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Blanchette <em>et al.</em> 2004</a>); (3) six-frame translated BLAT (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kent 2002</a>); and (4) OMA (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Altenhoff <em>et al.</em> 2011</a>; July 2013 release). We used MultiParanoid over OrthoMCL (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2003</a>) because the latter produced no errors or output after careful execution of the thirteen-stage analysis protocol. For all methods, genomic data were retrieved for the genome builds listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465880/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465880/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome builds</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">For MultiParanoid (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Alexeyenko <em>et al.</em> 2006</a>), an all-<em>vs.</em>-all blast search was run using the following command structure:<q id="disp-quote-1">blastp -db $blastdatabase -query [query file] -out [output file] -evalue .01 -num_threads [number of threads] -outfmt 6 -db_soft_mask 21 -word_size 3 -use_sw_tback</q></p><p id="p-13">From this output, ortholog predictions were produced using the standard MultiParanoid protocol.</p><p id="p-14">For BLAT (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kent 2002</a>), genomes for each species of interest were downloaded from the NCBI Entrez Genome database (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">McEntyre and Ostell 2002</a>). Queries were conducted using the following command structure:<q id="disp-quote-2">blat -q=prot -t=dnax -minIdentity=70 –extendThroughN [genome file] [query file] [output file]</q></p><p id="p-16">In the case of MultiZ (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Blanchette <em>et al.</em> 2004</a>), CCDS orthologs were downloaded directly from the UCSC genome browser (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kent <em>et al.</em> 2002</a>). For OMA (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Altenhoff <em>et al.</em> 2011</a>), ortholog predictions were downloaded from omabrowser.org (December 2012 release). For genes with more than one CCDS, orthologs were mapped to each analyzed transcript. Finally, ortholog predictions from metaPhOrs (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pryszcz <em>et al.</em> 2011</a>) were retrieved from release v201009 (June 2012).</p></div><div id="sec-3" class="subsection"><h3>Filtering putatively non-orthologous sequences</h3><p id="p-17">All ortholog detection methods produce false positives. For example, this can result when a gene deletion on one lineage means that no true ortholog exists in a given species. Typically, these issues are dealt with through rigorous filtering of input alignments. The intuition is that by applying a stringent sequence similarity filter, we can remove the vast majority of evolutionarily unrelated genes. We use this filtering approach to ensure that only credible, putatively orthologous sequences are included in the analysis. Because of heterogeneity in genome quality, similarity cutoffs were chosen heuristically, considering the known level of genome-wide divergence between human and the species of interest, as well as the overall distributions of percent identity between putative orthologs in the two species. Specifically, we first chose a cutoff based on the species-specific levels of percent identity to human. We then updated these numbers based on spot checks of borderline alignment cases. These cutoffs were as follows: chimp: 82%, gorilla: 77%, orangutan: 75%, rhesus macaque: 73%. A cutoff of 70% was employed for marmoset, bushbaby, cat, cow, and horse. For applications where consistency across methods is not important, these cutoffs could be chosen using downstream quality metrics such as those presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. Note that such an approach would still require the user to specify a tradeoff between the quality and number of orthologs.</p></div></div><div class="section other1" id="sec-4"><h2 class="">MOSAIC: OD integration as cluster optimization</h2><p id="p-18">MOSAIC provides a highly flexible, graph-based framework for integrating diverse OD methods. For a given reference sequence, proposal orthologs are conceptualized as nodes in a graph, connected with edges weighted according to the pairwise similarity between sequences (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The task of OD integration is then to choose proposal orthologs from each species such that a chosen measure of intra-cluster similarity is optimized.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic of the sequence selection algorithm. Steps: (1) Construct graph; (2) Choose the sequence from a random OD method for each species; (3) Iterate through species. For each species, pick the orthologs with highest similarity to the current best choices for all other species; (4) Return current best choices if no changes are made after iterating through all species; (5) To find global optimum, repeat steps 1-4 with random sampling paths." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A schematic of the sequence selection algorithm. Steps: (1) Construct graph; (2) Choose the sequence from a random OD method for each species; (3) Iterate through species. For each species, pick the orthologs with highest similarity to the current best choices for all other species; (4) Return current best choices if no changes are made after iterating through all species; (5) To find global optimum, repeat steps 1-4 with random sampling paths.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F1.medium.gif" width="349" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/629/F1.medium.gif" width="349" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465953" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">A schematic of the sequence selection algorithm. Steps: (1) Construct graph; (2) Choose the sequence from a random OD method for each species; (3) Iterate through species. For each species, pick the orthologs with highest similarity to the current best choices for all other species; (4) Return current best choices if no changes are made after iterating through all species; (5) To find global optimum, repeat steps 1-4 with random sampling paths.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-5" class="subsection"><h3>MOSAIC optimizes (weighted) pairwise similarities</h3><p id="p-20">To begin, MOSAIC calculates pairwise similarities between all orthologs from different species. Percent identity- and BLAST-based similarity metrics are provided by default, but user-defined similarity metrics are also accepted. These similarity scores define edge weights, which are used to construct a graph such as the one presented at the top of <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Once this full graph is constructed, it is highly recommended that it be quality filtered using user-specified similarity cutoffs. This step is necessary to minimize the effect of gene loss, duplication, etc. Once the graph is cleaned, MOSAIC then chooses at most one proposal ortholog from each species so that the overall pairwise similarity between accepted sequences is optimized.</p><p id="p-21">To accommodate user priorities, pairwise similarities can be weighted such that sequences from different species contribute unequally to the total similarity score. For uniform weights, this is equivalent to maximizing the average pairwise similarity. In the case where only similarity to a reference sequence is of interest, this reduces to simply accepting the ortholog for each species that is most similar to the reference.</p></div><div id="sec-6" class="subsection"><h3>Optimization is carried out using cyclic coordinate descent</h3><p id="p-22">For <em>m</em> OD methods and <em>s</em> species, there are up to <em>m<sup>s</sup></em> possible integrated alignments. In the case analyzed in this paper, <em>m = 4</em> and <em>s = 10</em>. This translates to over a million possible integrated alignments for each of the ~25,000 reference sequences considered. It is clear to see from this example that an exhaustive optimization becomes quickly infeasible. Therefore, MOSAIC choses optimal clusters using cyclic coordinate descent (CCD), an efficient non-derivative optimization algorithm (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bertsekas 1999</a>).</p><p id="p-23">In <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, we illustrate the way CCD functions in the context of MOSAIC. After the full graph that includes all orthologous sequences is built, random orthologs from each species are chosen as the current best. MOSAIC then loops through the species of interest in a random order. For each species, MOSAIC chooses the sequence that optimizes cluster tightness, given the current best sequences for all other species. This process is repeated until no further improvements can be made to cluster tightness. Finally, because CCD is prone to finding local rather than global optima, this entire process is repeated multiple times with random starting points and sampling paths.</p></div><div id="sec-7" class="subsection"><h3>Scoring and optimization procedures for this study</h3><p id="p-24">For the alignments presented here, we consider a protein set with relatively low levels of evolutionary divergence. We chose percent identity as our metric for sequence divergence. Note that several other popular scoring functions are implemented in MOSAIC. For more distantly related species, the application of scoring matrices (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dayhoff <em>et al.</em> 1978</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henikoff 1992</a>) or Hidden Markov Models (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ebersberger <em>et al.</em> 2009</a>) may be preferable. To reduce computational costs related to pairwise alignment, we considered only similarities between orthologs and the human target sequence. The optimization procedure was then equivalent to choosing, for each species, the ortholog among all methods that is most similar to the human sequence. This approach corresponds to the arguments edgefunc=’perID’, optrule=’pairwise’ when calling the Mosaic constructor in mosaic.py (see: <a href="http://pythonhosted.org/bio-MOSAIC/Module.html">http://pythonhosted.org/bio-MOSAIC/Module.html</a>).</p></div><div id="sec-8" class="subsection"><h3>Example: measuring similarity</h3><p id="p-25">Percent identity was calculated as the percent of sites in the human sequence that were identical in the orthologous sequence. For example, the hypothetical sequence below would be scored as 71% identical (5/7), because there are two mismatches between the seven sites present in the human sequence and the character to which those sites are aligned in the chimp sequence (sites where the human sequence has been deleted or the outgroup has an insertion are ignored):</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-26">Human A W V A - T F D</p></li><li id="list-item-2"><p id="p-27">Chimp - W V R Y T F D</p></li></ul></div><div id="sec-9" class="subsection"><h3>A note on gene loss, duplication, and divergent evolution</h3><p id="p-28">For any query protein, there is a risk that a related gene in another genome has been deleted, or has not maintained the same function and so provides inapplicable evolutionary information. In the case of deletions, it is unlikely that a non-homologous gene would be suitably similar to be classified as an ortholog. Divergence in function would be expected to sharply increase sequence divergence. In many cases such functionally distinct can be removed by suitably stringent sequence similarity filters.</p><p id="p-29">Another pitfall in ortholog detecion is gene duplication within a particular lineage. This results in so-called in-paralogs, which may inject additional bias if, compared to the query protein, the most functionally similar of the set is not the most similar at the sequence level. Although this is possible, it is the exception and not the rule under reasonable models of evolution. Indeed, experimental data from several model systems has demonstrated that there is an extremely high correlation between functional conservation and sequence conservation (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mashiyama <em>et al.</em> 2014</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhao <em>et al.</em> 2014</a>). Taking the single most similar sequence from a paralogous group is therefore a rational and effective (albeit imperfect) approach to this problem. For this reason, MOSAIC does not exclude putatively orthologous sequences that have paralogs in the source genomes. It rather picks the paralogous sequence that is most likely to share the same function as the other putatively orthologous sequences under examination. We will show that this decision allows us to capture more putative orthologs while simultaneously improving ortholog quality by all commonly used metrics.</p><p id="p-30">In summary, MOSAIC is adapted to producing multiple sequence alignments (MSAs) that are functionally informative at the site-level. For other applications, researchers may wish to infer genomic events such as gene loss, duplication, horizontal gene transfer, and/or incomplete lineage sorting (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Capra <em>et al.</em> 2013</a>). This involves jointly examining functionally diverged paralogous groups alongside their corresponding orthologs. This task generally requires a combination of tools such as MultiParanoid (to infer paralogs; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Remm <em>et al.</em> 2001</a>), RaxML (to build gene and species trees; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stamatakis 2014</a>), and Notung (to reconcile gene trees with species trees and infer evolutionary events; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stolzer <em>et al.</em> 2012</a>). For applications such as this, MOSAIC alignments can still be leveraged to help ensure the inclusion of relevant sequences. Likewise, reconstructed evolutionary histories can be used to flag, among tens of thousands of automatically generated MOSAIC alignments, those exceptional cases that could benefit most from manual inspection.</p></div><div id="sec-10" class="subsection"><h3>Multiple sequence alignment</h3><p id="p-31">Retrieved sequences were jointly aligned to query proteins using MSAprobs (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2010</a>), a multithreaded aligner with better performance benchmarks than many top aligners, including ClustalW, MAFFT, MUSCLE, ProbCons, and Probalign (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2010</a>). Importantly, MSAprobs has the further advantage of providing, for each column of an alignment, dependable estimates of the confidence of the alignment at the site.</p></div><div id="sec-11" class="subsection"><h3>Quality assessment</h3><p id="p-32">One approach to evaluating the performance of ortholog detection methods is to restrict analysis to validation sets, which usually consist of small, curated gene groups from unicellular model organisms (<em>e.g.</em>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Salichos and Rokas 2011</a>). Known true positive and true negative relationships allow researchers to calculation power and sensitivity, and even provide the possibility of applying supervised learning techniques such as support vector machines or random forest classifiers. Although such results are assuredly internally valid, it is unclear whether these results generalize beyond this small and highly biased subset of genes. As a simple example, “true” orthologous relationships often are restricted to cases where synteny is also maintained. Filtering out nonsyntenic orthologous sequences will thus significantly bias performance metrics toward OD methods that use syntenic information. Furthermore, this approach to quality assessment does not allow researchers to evaluate performance on their own arbitrary dataset. It is for these reasons that we decided to use the quality metrics described below. Strictly speaking, the details necessary to truly establish orthology are buried deep in evolutionary time. We therefore believe that ortholog detection is perhaps better viewed as an unsupervised learning problem that is amenable to graph-based cluster optimization.</p></div><div id="sec-12" class="subsection"><h3>Sequence identity</h3><p id="p-33">MOSAIC optimizes pairwise sequence similarity. In this example, sequence identity is used as the similarity measure, and pairwise similarities are weighted such that only concordance with the human reference sequence is considered. To achieve greater separation between metrics used for optimization and assessment, comparisons of sequence identity were performed in the context of the full MSAs. We believe this choice is sensible because it is the quality of the MSA that is of primary importance to many downstream phylogenetic inference tasks. In addition, this approach allows us to indirectly incorporate information about intra-cluster similarity. As an MSA incorporates increasingly divergent sequences, performance relative to pairwise alignments is expected to progressively degrade.</p></div><div id="sec-13" class="subsection"><h3>Tree concordance</h3><p id="p-34">For each MSA, gene trees were built using RAxML (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stamatakis and Alachiotis 2010</a>). An unweighted Robinson-Foulds (RF) distance (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Robinson and Foulds 1981</a>) was then calculated between each gene tree and the known species tree using the python module dendropy (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sukumaran and Holder 2010</a>). Briefly, the unweighted RF distance counts the number of operations required to transform one tree into the other. This quantity is equal to the total number of splits that are present in one tree but not the other. To normalize for variations in tree size, we then divided this distance by the sum of the total number of splits in the gene and species trees (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yu <em>et al.</em> 2011</a>). To summarize the genome-wide distribution of normalized RF distances, we took the area under the curve of the cumulative distribution function. This was limited to values less than 0.4, since beyond this value there is little difference between the observed curves (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/017095SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS5.pdf">Figure S5</a>). This metric is superior to, <em>e.g.</em>, calculating the proportion of genes below a given threshold because it up-weights smaller RF distances as opposed to, in effect, using non-zero uniform weights below the cutoff value.</p></div><div id="sec-14" class="subsection"><h3>Functional concordance</h3><p id="p-35">Profile HMMs were downloaded from the PfamA protein families database (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Punta <em>et al.</em> 2012</a>). Each sequence was then annotated using the top scoring functional class retrieved by querying that sequence against the database of all PfamA protein family HMMs. This search was conducted using HMMER3 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Eddy 2011</a>). Functional concordance was then measured as a binary quantity, corresponding to whether or not a putative orthologous sequence had the same inferred function as its cognate human sequence. It is important to note that not all PfamA HMMs are functionally validated. In cases where experimental validation is unavailable, these HMMs provide a family-specific scoring function that nevertheless yields information not contained in naïve sequence identity measures.</p></div><div id="sec-15" class="subsection"><h3>Evolutionary analysis</h3><div id="sec-16" class="subsection"><h4>Gene-level conservation:</h4><p id="p-36">Alignments were analyzed using phylogenetic analysis by maximum likelihood (PAML) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yang 2007</a>). For each alignment three models were fit: (1) a neutral model where the ration of the non-synonymous to synonymous substitution rates (dN/dS) is fixed at one, (2) a conservation model where dN/dS is less than or equal to one, and (3) a positive selection model where some fraction of the sequence is fit under the conservation model, while another dN/dS parameter is estimated freely for the remainder of the sequence. Because evolutionary models are not in general nested, we performed model selection via the popular Akaike information criterion, a method that penalizes a model’s fit by its number of included variables (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akaike 1973</a>) and is asymptotically equivalent to maximizing the model’s predictive performance on unseen data (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stone 1977</a>).</p><p id="p-37">Despite rigorous model selection procedures, in rare cases PAML may estimate very high levels of selection over a tiny proportion of a given sequence (even a single site), leading to greatly inflated average levels of dN/dS. To reduce the influence of outlying estimates of selection, all dN/dS values greater than 3 were excluded for the analysis. For all methods, this corresponded to less than 0.05% of all sequences.</p></div><div id="sec-17" class="subsection"><h4>Site-level positive selection:</h4><p id="p-38">The program sitewise likelihood ratio (SLR) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Massingham and Goldman 2005</a>) was used to estimate the number of positively selected sites in each sequence. To eliminate false positives caused by poorly aligned sites, we filtered out all sites estimated by MSAprobs to be aligned to less than 95% confidence. All included positively selected sites estimated at 95% confidence or greater by SLR were included in the subsequence comparison.</p><p id="p-39">To assess agreement in PSS, we calculated the degree of overlap between PSS from all pairs of methods. This was calculated as the size of the genome-wide intersection between sites divided by the union of said sites.</p></div></div><div id="sec-18" class="subsection"><h3>Mapping positively selected sites onto three-dimensional structures</h3><p id="p-40">We leveraged UniProt mapping files (<a href="http://www.uniprot.org/docs/pdbtosp">http://www.uniprot.org/docs/pdbtosp</a>; accessed 9/30/14) to determine which proteins had a relevant structure in the Protein Data Bank (PDB; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berman 2000</a>). We then aligned sequences between PDB structures and candidate genes to determine the degree of coverage and to obtain a mapping between residues. We found 2003 genes for which there was a structure with greater than 70% coverage. Of these, 787 had PSS results from all five ortholog detection methods from at least one species. Reasons for missing data comprise more than 4 missing species in source MSAs and lack of convergence in the PSS calculation. Within this set of 787 genes, 76 proteins had PSS from MOSAIC that were not found with any of the component methods. From this point, the example of TPSAB1 was quickly identified by manual inspection. We then downloaded PDB structure 2ZEC to visualize the location of positively selected sites. To validate sequences used in the analysis, we blasted each ortholog against the human SwissProt database. TPSAB1 was the most similar human protein in each case, confirming that we had retrieved best bidirectional hits. Annotations for each protein were also consistent with alpha/beta tryptases activity (<a href="http://www.g3journal.org/content/suppl/2015/02/23/g3.115.017095.DC1/TableS1.pdf">Table S1</a>).</p></div></div><div class="section results-discussion" id="sec-19"><h2 class="">Results and Discussion</h2><div id="sec-20" class="subsection"><h3>Ortholog detection methods frequently outperform one another</h3><p id="p-41">We begin with a comprehensive comparison of four popular, methodologically distinct OD methods. In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, we show the head-to-head performances of these different methods for a range of primates and closely related mammals. Performance is assessed using alignments between all human consensus coding sequences (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pruitt <em>et al.</em> 2009</a>) and their corresponding orthologs from each method. For each possible ortholog (defined by human target sequence and species of origin), we examine whether sequence identity to human is at least five percentage points greater for one method <em>vs.</em> another. We otherwise consider the two methods to be tied. By this metric, one method significantly outperforms another 38−45% of the time. Importantly, no method uniformly outperforms all others, underlining the complementarity of the chosen algorithms. For each method, distributions of percent identity and relative performance by species are presented in <a href="http://www.g3journal.org/content/suppl/2015/02/23/g3.115.017095.DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/23/g3.115.017095.DC1/FigureS2.pdf">Figure S2</a>,</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of sequence identity levels between methods. Heat map of the percent of orthologs for which MultiParanoid (MP), OMA (OMA), BLAT (BL), and MultiZ (MZ) outperform one another. Performance is based on percent identity of each method&#x2019;s orthologs to the human sequence. One method is considered to outperform another method if it improves percent identity by at least five percentage points. Text in diagonal cells shows the number of orthologs identified by each method, colored by the percent of orthologs for which a given method outperforms all the others." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of sequence identity levels between methods. Heat map of the percent of orthologs for which MultiParanoid (MP), OMA (OMA), BLAT (BL), and MultiZ (MZ) outperform one another. Performance is based on percent identity of each method&#x2019;s orthologs to the human sequence. One method is considered to outperform another method if it improves percent identity by at least five percentage points. Text in diagonal cells shows the number of orthologs identified by each method, colored by the percent of orthologs for which a given method outperforms all the others.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F2.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/629/F2.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-42" class="first-child">Comparison of sequence identity levels between methods. Heat map of the percent of orthologs for which MultiParanoid (MP), OMA (OMA), BLAT (BL), and MultiZ (MZ) outperform one another. Performance is based on percent identity of each method’s orthologs to the human sequence. One method is considered to outperform another method if it improves percent identity by at least five percentage points. Text in diagonal cells shows the number of orthologs identified by each method, colored by the percent of orthologs for which a given method outperforms all the others.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-21" class="subsection"><h4>Combining multiple sequence alignments with MOSAIC:</h4><p id="p-43">It is well-known in theory (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wolpert and Macready 1997</a>) and in practice (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Van Der Laan and Gruber 2010</a>) that the comparative performance of competing statistical inference algorithms often varies by context. Rather than search for a single best algorithm, researchers have sought to integrate a variety of methods in order to reap the benefits of methodological complementarity (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Van Der Laan <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rokach 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chandrasekaran and Jordan 2013</a>). As might be expected, the gains yielded by this approach generally scale with the quality of the individual methods integrated, the number of methods included, and, importantly, the diversity of the comprised algorithms (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kuncheva and Whitaker 2003</a>).</p><p id="p-44">Having observed the complementarity between OD methods, we sought to develop a structure for the automatic integration of methodologically distinct OD methods such as those described above. We term this framework MOSAIC, or <strong>M</strong>ultiple <strong>O</strong>rthologous <strong>S</strong>equence <strong>A</strong>nalysis and <strong>I</strong>ntegration by <strong>C</strong>luster optimization. MOSAIC allows for the flexible integration of diverse OD methods through the application of standard or user-defined metrics of sequence similarity and ortholog cluster quality. By the use of the specified similarity metrics, clusters of proposed orthologs are built. These orthologs are then adopted or rejected to optimize cluster completeness and quality (<em>e.g.</em>, similarity to a reference sequence or average pairwise similarity).</p><p id="p-45">Having presented a schematic of the algorithm itself in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, we provide in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> a view of example inputs and output MSAs. These are illustrations of real alignments for carbonic anhydrase 12, an enzyme critical to a number of biological functions, including the formation of bone, saliva, and gastric acid (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pruitt <em>et al.</em> 2014</a>). MSA columns that are aligned to below 95% confidence are displayed in red and masked from the analysis. Orthologs that were not returned for a given species are denoted with a horizontal black bar. Those that were filtered using pre-integration sequence identity cutoffs (see the section <em>Materials and Methods</em>) are indicated with gray bars. Sequence identity is measured based on pairwise realignment to the human sequence. Note that, just as when employing a single ortholog detection method, this filtering step is critical to guaranteeing alignment quality.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of integration process for carbonic anhydrase 12. MSA columns that are aligned to below 95% confidence are displayed in red. Orthologs that were not returned for a given species are denoted with a horizontal black bar. Those that were filtered using pre-integration sequence identity cutoffs are indicated with gray bars with the global percent identity from pairwise alignment to human included. Species name label colors denote the species of origin for orthologs in the MOSAIC alignment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustration of integration process for carbonic anhydrase 12. MSA columns that are aligned to below 95% confidence are displayed in red. Orthologs that were not returned for a given species are denoted with a horizontal black bar. Those that were filtered using pre-integration sequence identity cutoffs are indicated with gray bars with the global percent identity from pairwise alignment to human included. Species name label colors denote the species of origin for orthologs in the MOSAIC alignment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F3.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/629/F3.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Illustration of integration process for carbonic anhydrase 12. MSA columns that are aligned to below 95% confidence are displayed in red. Orthologs that were not returned for a given species are denoted with a horizontal black bar. Those that were filtered using pre-integration sequence identity cutoffs are indicated with gray bars with the global percent identity from pairwise alignment to human included. Species name label colors denote the species of origin for orthologs in the MOSAIC alignment.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-22" class="subsection"><h3>Method integration increases the number of included sequences</h3><p id="p-47">The gains afforded by MOSAIC vary by species and increase with the number of methods that are included (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). When all four component methods are included, MOSAIC more than quintuples the number of alignments for which all species are present (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). We observe in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a> that the largest improvements are seen for gorilla, bushbaby, and cat. Importantly, orthologs for each of these three species are rescued by different methods (OMA for gorilla, MultiParanoid for bushbaby, and Blat for cat. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS3.pdf">Figure S3</a> for further details). In the sections that follow, we will demonstrate that MOSAIC captures these additional sequences while simultaneously improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="OD power and the effect of pooling methods (A) The cumulative proportion of human transcripts for which an ortholog was detected, stratified by species. Envelopes illustrate results from pooling an increasing number of methods. (B) The cumulative number of human transcripts as a function of the maximum number of missing species allowed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;OD power and the effect of pooling methods (A) The cumulative proportion of human transcripts for which an ortholog was detected, stratified by species. Envelopes illustrate results from pooling an increasing number of methods. (B) The cumulative number of human transcripts as a function of the maximum number of missing species allowed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F4.medium.gif" width="440" height="170"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/4/629/F4.medium.gif" width="440" height="170"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465856" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">OD power and the effect of pooling methods (A) The cumulative proportion of human transcripts for which an ortholog was detected, stratified by species. Envelopes illustrate results from pooling an increasing number of methods. (B) The cumulative number of human transcripts as a function of the maximum number of missing species allowed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>MOSAIC improves sequence identity</h3><p id="p-49">MOSAIC achieves massive gains in the number of retrieved orthologs while slightly improving average levels of sequence identity. Although MOSAIC directly optimizes sequence identity, this result is noncircular for two reasons. First, average levels of sequence identity could be reduced by preferentially adding sequences from the lower end of the sequence identity distribution. This result would be consistent with a scenario in which most methods correctly inferred that a gene was deleted on a particular lineage. Second, MOSAIC optimizes sequence identity measured from pairwise global alignments. In the validation phase, we calculated this metric in the context of the full MSA. That is, we do not realign to the human sequence in a pairwise fashion as we do in the optimization phase. Rather, we measure sequence identity based on the alignment specified within the full MSA. We believe this choice is sensible because it is the quality of the MSA that is of primary importance to many downstream phylogenetic inference tasks. In addition, this approach allows us to indirectly incorporate information about intra-cluster similarity. As an MSA incorporates increasingly divergent sequences, performance relative to pairwise alignments is expected to progressively degrade.</p></div><div id="sec-24" class="subsection"><h3>MOSAIC improves functional concordance</h3><p id="p-50">We employed profile HMMs from the Protein Families Database A (PfamA) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Punta <em>et al.</em> 2012</a>) and HMMER3 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Eddy 2011</a>) to ascertain putative functional concordance between proposed orthologs and the human consensus coding sequences of interest. PfamA builds HMMs via curated alignments of small numbers of representative members from each protein family. It is important to note that not all PfamA HMMs are functionally validated. In cases where experimental validation is unavailable, these HMMs provide a family-specific scoring function that yields information not contained in naïve sequence identity measures.</p><p id="p-51">With HMMER3, we queried protein sequences against all PfamA protein family profiles, annotating each protein according to its top protein family hit. This allowed for an ascertainment of functional concordance that is more comprehensive than relying on gene-by-gene annotation across species, while retaining many of the advantages of manual curation where it exists. This assessment reveals that, for the set of orthologous sequences proposed by all methods, MOSAIC provides levels of functional concordance that are slightly better than the best performing component method (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Gains are particularly large for gorilla, bushbaby, and cat orthologs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS4.pdf">Figure S4</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MOSAIC improves alignment quality. We show the fold improvement of each method over the worst performing method in four categories: sequence identity, functional concordance, phylogenetic concordance, and number of orthologs detected." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;MOSAIC improves alignment quality. We show the fold improvement of each method over the worst performing method in four categories: sequence identity, functional concordance, phylogenetic concordance, and number of orthologs detected.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F5.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/4/629/F5.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">MOSAIC improves alignment quality. We show the fold improvement of each method over the worst performing method in four categories: sequence identity, functional concordance, phylogenetic concordance, and number of orthologs detected.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>MOSAIC improves phylogenetic concordance</h3><p id="p-53">Phylogenetic concordance was ascertained by calculating the normalized, unweighted Robinson-Foulds distance (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Robinson and Foulds 1981</a>) between gene trees and the established species tree (Altenhoff and Dessimoz 2009b). This metric is equal to the sum of the number of splits in one tree that are not present in the other, scaled by the total number of splits present across the two trees. Accordingly, larger Robinson-Foulds distances correspond to worse agreement between gene and species trees. On a gene-by-gene basis, this metric should be interpreted with caution, since post-speciation admixture and incomplete lineage sorting can lead to true discordance between the species tree and the phylogenetic history of a particular gene (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Maddison and Knowles 2006</a>). At greater levels of divergence, loss of signal and homoplasy may similarly confound the analysis in some cases. However, at the level of the genome, higher concordance between gene trees and the known speciation process strongly suggests a relative improvement in OD.</p><p id="p-54"><a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> presents a comparison of genome-wide phylogenetic concordance (see the section <em>Materials and Methods</em> for details on this metric). MultiZ performs the best of any individual method, likely due to its utilization of syntenic information. Surprisingly, OMA, the OD method that incorporates phylogenetic tree information, exhibits the worst performance according to this tree-based metric. MOSAIC, on the other hand, provides significant performance gain over all component methods, including a 59% increase in phylogenetic concordance compared to OMA.</p></div><div id="sec-26" class="subsection"><h3>Increased ortholog quality leads to more conservation <em>and</em> positively selected sites</h3><p id="p-55">Having demonstrated an increase in ortholog quality using tree-, function-, and similarity-based measures of quality, we next sought to assess the influence of increased alignment quality on estimated levels of selection. To assess gene-level conservation, we applied PAML (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yang 2007</a>) with automated likelihood-based model selection. To ascertain site-level positive selection, we used SLR, a method shown to have a greater power and a lower false positive rate than PAML’s popular Bayes Empirical Bayes method (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Massingham and Goldman 2005</a>).</p><p id="p-56">Because varying numbers of sequences can sway evolutionary estimates in unpredictable ways due to, <em>e.g.</em>, inhomogeneous levels of selection across organisms, we assessed the performance of MOSAIC relative to each method by matching the species present in each alignment. We refer to this approach as MOSAIC<sub>matched</sub>. In the case of both PAML and SLR, synonymous substitution rates in coding DNA are used as a background against which to test for changes in rates of non-synonymous substitution. We compared our performance to that of metaPhOrs, an OD integration method that works on tree-based methods only. Since the metaPhOrs database provides only protein sequences for its alignments, no evolutionary comparison with this method was possible given the available data. However, we demonstrate in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS6.pdf">Figure S6</a> that MOSAIC outperforms <a href="http://orthology.phylomedb.org/">metaPhOrs</a> according to the metrics presented in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, despite integrating nearly half the number of OD methods in this example. An analysis with a matching number of OD methods was not possible because metaPhOrs is available only as a pre-calculated database.</p><p id="p-57">In <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>, we see that MOSAIC leads to greater gene-level conservation (lower dN/dS) compared with every method except Blat, for which the difference was not statistically significant. Full distributions of dN/dS for each method are presented in <a href="http://www.g3journal.org/content/suppl/2015/02/23/g3.115.017095.DC1/FigureS7.pdf">Figure S7</a>. Despite greater levels of conservation, MOSAIC was able to detect ~30–180% more positively selected sites than any of its component methods. This was not due to an increase in the inferred rate of positive selection. Rather, most of this increase in power was attributable to the fact that more sites were aligned to high confidence and therefore included in the analysis. This step of filtering for alignment quality is important because site-wise estimates of positive selection are highly sensitive to short poorly aligned regions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jordan and Goldman 2012</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A comparison of evolutionary estimates. (A) The relative difference of MOSAICmatched vs. each component method for: (1) the number of positively selected sites, (2) the number of confidently aligned sites, and for reference, (3) the average level of conservation across all alignments. (B) The agreement between positively selected sites (1) between MOSAIC and component methods, and (2) among component methods. Fractional overlap values are plotted as Venn diagrams to illustrate the two methods being compared." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A comparison of evolutionary estimates. (A) The relative difference of MOSAIC&lt;sub&gt;matched&lt;/sub&gt; &lt;em&gt;vs.&lt;/em&gt; each component method for: (1) the number of positively selected sites, (2) the number of confidently aligned sites, and for reference, (3) the average level of conservation across all alignments. (B) The agreement between positively selected sites (1) between MOSAIC and component methods, and (2) among component methods. Fractional overlap values are plotted as Venn diagrams to illustrate the two methods being compared.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F6.medium.gif" width="440" height="132"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/4/629/F6.medium.gif" width="440" height="132"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465958" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-58" class="first-child">A comparison of evolutionary estimates. (A) The relative difference of MOSAIC<sub>matched</sub> <em>vs.</em> each component method for: (1) the number of positively selected sites, (2) the number of confidently aligned sites, and for reference, (3) the average level of conservation across all alignments. (B) The agreement between positively selected sites (1) between MOSAIC and component methods, and (2) among component methods. Fractional overlap values are plotted as Venn diagrams to illustrate the two methods being compared.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">To investigate the quality of the positively selected sites detected by MOSAIC, we assessed concordance with and between component methods. For a pair of method, we measure overlap by dividing the total size of the intersection between positively selected sites by the total size of the union. These results are shown in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>. We observe that the minimum overlap between MOSAIC and a component method (MOSAIC/Blat) is still better than the best overlap between component methods (MultiZ/OMA). Averaging over comparisons, we find the improvement in concordance with <em>vs.</em> between component methods is statistically significant beyond computational precision (<em>P</em> &lt; 1e-16).</p></div><div id="sec-27" class="subsection"><h3>Understanding MOSAIC’s improvements in performance</h3><p id="p-60">As we have mentioned, ensemble-based inference frameworks have been shown to operate effectively in a wide variety of statistical contexts. For the methods examined here, MOSAIC may provide increased MSA quality for a variety of reasons, not all of which are related to the performance of the component algorithms. For example, the ability to specify a custom scoring function for pairwise similarity gives MOSAIC an advantage over component methods. For example, BLAST-based scoring such as that used by MultiParanoid may in some cases reward substitutions over than sequence identity. This is the result of building substitution matrices on libraries of sequences that are much more highly diverged than those within mammalian proteomes.</p><p id="p-61">The quality of a genome assembly also plays an important role. For methods like MultiZ, improperly assembled genomic segments may provide misleading information about orthologous relationships. Similarly, naïve methods such as BLAT will miss proteins whose exons are spread across unassembled genomic segments. Other methods such as MultiParanoid and OMA may be robust to this effect since they draw from more sophisticated proteome predictions. However, the stringent and sometimes arbitrary filtering required for proteome prediction cause these methods to miss sequences that can be found using BLAT’s six-frame genomic translation.</p></div><div id="sec-28" class="subsection"><h3>Better alignments may yield new insights into human evolution</h3><p id="p-62">We next sought to examine the biological significance of some of the positively selected sites identified uniquely by MOSAIC. This led us to Tryptase Alpha/Beta 1 (TPSAB1), a tetrameric serine protease that has been implicated in the pathogenesis of asthma (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Taira <em>et al.</em> 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cui <em>et al.</em> 2014</a>), heart disease (Bot <em>et al.</em> 2014), inflammatory bowel disease (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hamilton <em>et al.</em> 2011</a>), and other disorders with allergic and/or inflammatory components (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sommerhoff and Schaschke 2007</a>). Shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> is the three-dimensional structure of a TPSAB1 tetramer with inhibitor (white) bound (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Costanzo <em>et al.</em> 2008</a>). In orange, distal to the active site, is the positively selected residue detected by component methods and by MOSAIC. Note that positive selection at this location is active only outside of the great apes, with a fixed lysine observed in human, chimp, gorilla, and orangutan (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS9.pdf">Figure S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS10.pdf">Figure S10</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/629/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example: a MOSAIC-specific PSS in Tryptase Alpha/Beta 1 (TPSAB1). The tetrameric TPSAB1 structure is shown with positively selected sites highlighted. The site detected by component methods and by MOSAIC is colored orange, whereas the MOSAIC-specific PSS is featured in red. A bound inhibitor (white) pinpoints the active site of the enzyme." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-560790314" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example: a MOSAIC-specific PSS in Tryptase Alpha/Beta 1 (TPSAB1). The tetrameric TPSAB1 structure is shown with positively selected sites highlighted. The site detected by component methods and by MOSAIC is colored orange, whereas the MOSAIC-specific PSS is featured in red. A bound inhibitor (white) pinpoints the active site of the enzyme.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/629/F7.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/4/629/F7.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/629/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/629/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-63" class="first-child">Example: a MOSAIC-specific PSS in Tryptase Alpha/Beta 1 (TPSAB1). The tetrameric TPSAB1 structure is shown with positively selected sites highlighted. The site detected by component methods and by MOSAIC is colored orange, whereas the MOSAIC-specific PSS is featured in red. A bound inhibitor (white) pinpoints the active site of the enzyme.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">In red, directly within the proteolytic pore, is the site identified by MOSAIC as positively selected. This residue is a positively charged arginine in humans. This would be expected to modify the electrostatics of ligand binding. In chimp, we instead observe a kink-inducing proline. We might anticipate this change to have a large steric effect, possibly allowing the inward-facing unstructured loop to act as a more rigid lid closing over top of the substrate, or as a modifier of subunit contacts. Importantly, these changes occurred repeatedly in mammals. Proline is observed at this position in rhesus macaque and marmoset. Arginine, on the other hand, is present in gorilla and horse (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS9.pdf">Figure S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS10.pdf">Figure S10</a>). In orangutan, we observe a histidine: another positively charged amino acid.</p><p id="p-65">Throughout this examination, we must be cognizant that tryptases evolved rapidly during primate evolution (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Trivedi <em>et al.</em> 2007</a>). The expansion of this gene family can itself be viewed as an example of positive selection. However, the presence of several paralogs creates the risk of inappropriately aligning pseudo-orthologous sequences that have evolved to serve divergent functions. Given the challenges, this case study provides an excellent opportunity to compare the high-throughput performance of MOSAIC to that of manually curated alignments.</p><p id="p-66">As a first step, we showed that each proposal ortholog was a best bidirectional hit to TPSAB1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/TableS1.pdf">Table S1</a>). Next, we compared our sequences to those retrieved manually by <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Trivedi <em>et al.</em> (2007)</a>. Although we notice a few minor discrepancies between the two sets of alignments (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS9.pdf">Figure S9</a> <em>vs.</em> <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1/FigureS11.pdf">Figure S11</a>, reproduced from <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Trivedi <em>et al.</em> 2007</a>), these differences do not alter our conclusion of human-relevant positive selection at the highlighted site in the proteolytic core of TPSAB1. Illustrations of component alignments from each method are shown in <a href="http://www.g3journal.org/content/suppl/2015/02/23/g3.115.017095.DC1/FigureS8.pdf">Figure S8</a>.</p><p id="p-67">In this paper we have introduced a novel algorithm, MOSAIC, which is capable of integrating an arbitrary number of methodologically diverse ortholog detection methods. We have demonstrated that MOSAIC provides large increases in power relative to its component methods, while simultaneously maintaining or improving functional-, phylogenetic-, and sequence identity-based measures of ortholog quality. Further, given the same number of species, MOSAIC alignments include more columns aligned with high confidence. This translates to higher levels of estimated conservation, and simultaneously, a greatly increased number of positively selected sites detected. Moreover, MOSAIC’s positively selected sites agree better with those from component methods than component results do with each other. This suggests that not only does MOSAIC detect more positively selected sites—these sites are more reproducible and are detected due to an increase in alignment quality. Finally, we illustrated the significance of this increase in power by highlighting a positively selected site near the active site of the tryptase TPSAB1. Given the role of this enzyme in asthma and other allergic and inflammatory disorders, we feel that this case study is worthy of experimental follow-up.</p><p id="p-68">In summary, MOSAIC provides the unique flexibility to incorporate any OD method that may be available now or in the future. It can therefore capture the entire swath of methodologic diversity, thereby improving OD performance, and allowing researchers to take advantage of methodological gains in a variety of areas of OD research. In addition, it provides the flexibility to adapt scoring and optimization procedures to the set of species under study. In future work, it will be interesting to ascertain how optimal procedures vary between species sets that have differing mean levels of divergence and markedly different patterns of evolution. For example, mammals and prokaryotes will likely have distinct optimal parameter values within MOSAIC. This tool is available a python package that can be installed using easy_install bio-mosaic from the command-line. MOSAIC alignments, source code, and full documentation are available at <a href="http://pythonhosted.org/bio-MOSAIC">http://pythonhosted.org/bio-MOSAIC</a>. </p></div></div><div class="section" id="sec-29"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-69">We thank Raul Torres, Lawrence Uricchio, Nicolas Strauli, and Zachary Szpiech for their feedback regarding the manuscript. We would also like to thank Nathan Pearson for inspiring the paper title. This work was partially supported by the National Institutes of Health (grant numbers HG007644, P60MD006902, UL1RR024131, 1R21HG007233, 1R21CA178706, and 1R01HL117004) and an Alfred P. Sloan Foundation Research Fellowship to R.D.H. M.C.M. was supported by the Epidemiology and Translational Science program at the University of California, San Francisco, a National Institutes of Health F31 Predoctoral Fellowship (grant number 1 F31 CA180609-01), and a University of California, San Francisco Lloyd M. Kozloff Fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-70">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017095/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-71"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-29"><span class="received-label">Received </span>October 29, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-06"><span class="accepted-label">Accepted </span>February 6, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Maher and Hernandez</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.4.629.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Information theory and an extension of the maximum likelihood principle</span>, pp. <span class="cit-fpage">267</span>–<span class="cit-lpage">281</span> in <span class="cit-source">2nd International Symposium on Information Theory</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Czaki</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-ed"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">B. N.</span></span>. <span class="cit-publ-name">Akademiai Kiado</span>, <span class="cit-publ-loc">Budapest</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.2" data-doi="10.1093/bioinformatics/btl213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexeyenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamas</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Automatic clustering of orthologs and inparalogs shared by multiple proteomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">e9</span>–<span class="cit-lpage">e15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DAlexeyenko%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D14%26rft.spage%253De9%26rft.epage%253De15%26rft.atitle%253DAutomatic%2Bclustering%2Bof%2Borthologs%2Band%2Binparalogs%2Bshared%2Bby%2Bmultiple%2Bproteomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl213%26rft_id%253Dinfo%253Apmid%252F16873526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMi8xNC9lOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.3" data-doi="10.1371/journal.pcbi.1000262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altenhoff</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dessimoz</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Phylogenetic and functional assessment of orthologs inference projects and methods.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DAltenhoff%26rft.auinit1%253DA.%2BM.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000262%26rft.epage%253De1000262%26rft.atitle%253DPhylogenetic%2Band%2Bfunctional%2Bassessment%2Bof%2Borthologs%2Binference%2Bprojects%2Band%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000262%26rft_id%253Dinfo%253Apmid%252F19148271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19148271&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.629.4"><div class="cit-metadata"><cite>Altenhoff, A. M., and C. Dessimoz, 2012 Inferring orthology, pp. <span class="cit-fpage">259</span>–<span class="cit-lpage">279</span> in <em>Evolutionary Genomics. Vol. 855. Methods in Molecular Biology</em>, edited by M. Anisimova. Humana Press, Totowa, NJ.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.5" data-doi="10.1093/nar/gkq1238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altenhoff</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonnet</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dessimoz</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">OMA 2011: orthology inference among 1000 complete genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D289</span>–<span class="cit-lpage">D294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1238%26rft_id%253Dinfo%253Apmid%252F21113020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0QyODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.6" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Protein Data Bank.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvMjM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.629.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertsekas</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-source">Nonlinear Programming</span>. <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Athena Scientific</span>, <span class="cit-publ-loc">Nashua, NH</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.8" data-doi="10.1101/gr.1933104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanchette</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riemer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elnitski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Aligning multiple genomic sequences with the threaded blockset aligner.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBlanchette%26rft.auinit1%253DM.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D708%26rft.epage%253D715%26rft.atitle%253DAligning%2BMultiple%2BGenomic%2BSequences%2BWith%2Bthe%2BThreaded%2BBlockset%2BAligner%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1933104%26rft_id%253Dinfo%253Apmid%252F15060014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzQvNzA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.4.629.9" data-doi="10.1161/ATVBAHA.114.303570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bot</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">G.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovanen</span>  <span class="cit-name-given-names">P. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mast cells as effectors in atherosclerosis.</span> <abbr class="cit-jnl-abbrev">Arterioscler. Thromb. Vasc. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArterioscler.%2BThromb.%2BVasc.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1161%252FATVBAHA.114.303570%26rft_id%253Dinfo%253Apmid%252F25104798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYXR2YmFoYSI7czo1OiJyZXNpZCI7czo4OiIzNS8yLzI2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.10" data-doi="10.1016/j.tig.2013.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stolzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">How old is my gene?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.07.001%26rft_id%253Dinfo%253Apmid%252F23915718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23915718&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.11" data-doi="10.1073/pnas.1302293110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandrasekaran</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Computational and statistical tradeoffs via convex relaxation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E1181</span>–<span class="cit-lpage">E1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1302293110%26rft_id%253Dinfo%253Apmid%252F23479655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzEzL0UxMTgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.12" data-doi="10.1371/journal.pone.0000383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vermunt</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Assessing performance of orthology detection strategies applied to eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChen%26rft.auinit1%253DF.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253De383%26rft.epage%253De383%26rft.atitle%253DAssessing%2Bperformance%2Bof%2Borthology%2Bdetection%2Bstrategies%2Bapplied%2Bto%2Beukaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000383%26rft_id%253Dinfo%253Apmid%252F17440619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17440619&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.13" data-doi="10.1126/science.1123061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ciccarelli</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerks</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Mering</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Creevey</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Toward automatic reconstruction of a highly resolved tree of life.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCiccarelli%26rft.auinit1%253DF.%2BD.%26rft.volume%253D311%26rft.issue%253D5765%26rft.spage%253D1283%26rft.epage%253D1287%26rft.atitle%253DToward%2Bautomatic%2Breconstruction%2Bof%2Ba%2Bhighly%2Bresolved%2Btree%2Bof%2Blife.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1123061%26rft_id%253Dinfo%253Apmid%252F16513982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTEvNTc2NS8xMjgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.14" data-doi="10.1016/j.bmcl.2008.01.093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yabut</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Potent, nonpeptide inhibitors of human mast cell tryptase. Synthesis and biological evaluation of novel spirocyclic piperidine amide derivatives.</span> <abbr class="cit-jnl-abbrev">Bioorg. Med. Chem. Lett.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2BMedicinal%2BChemistry%2BLetters%26rft.stitle%253DBioorganic%2B%2526%2BMedicinal%2BChemistry%2BLetters%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D2114%26rft.epage%253D2121%26rft.atitle%253DPotent%252C%2Bnonpeptide%2Binhibitors%2Bof%2Bhuman%2Bmast%2Bcell%2Btryptase.%2BSynthesis%2Band%2Bbiological%2Bevaluation%2Bof%2Bnovel%2Bspirocyclic%2Bpiperidine%2Bamide%2Bderivatives.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bmcl.2008.01.093%26rft_id%253Dinfo%253Apmid%252F18272363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bmcl.2008.01.093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18272363&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.15" data-doi="10.4049/jimmunol.1302947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahlin</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feinstein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bankova</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mouse mast cell protease-6 and MHC are involved in the development of experimental asthma.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">4783</span>–<span class="cit-lpage">4789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1302947%26rft_id%253Dinfo%253Apmid%252F25320274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE5My8xMC80NzgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.629.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dayhoff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orcutt</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">A model of evolutionary change in proteins</span>, pp. <span class="cit-fpage">345</span>–<span class="cit-lpage">358</span> in <span class="cit-source">Atlas of Protein Sequence and Structure</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Dayhoff</span>  <span class="cit-name-given-names">M. O.</span></span>. <span class="cit-publ-name">National Biomedical Research Foundation</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.17" data-doi="10.1186/1471-2148-9-157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ebersberger</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">HaMStR: profile hidden Markov model based search for orthologs in ESTs.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DEbersberger%26rft.auinit1%253DI.%26rft.volume%253D9%26rft.spage%253D157%26rft.epage%253D157%26rft.atitle%253DHaMStR%253A%2Bprofile%2Bhidden%2Bmarkov%2Bmodel%2Bbased%2Bsearch%2Bfor%2Borthologs%2Bin%2BESTs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-9-157%26rft_id%253Dinfo%253Apmid%252F19586527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-9-157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19586527&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.629.18" data-doi="10.1371/journal.pcbi.1002195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accelerated profile HMM searches.</span> <span class="cit-source">PLoS Comput. Biol.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002195%26rft.epage%253De1002195%26rft.atitle%253DAccelerated%2BProfile%2BHMM%2BSearches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002195%26rft_id%253Dinfo%253Apmid%252F22039361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039361&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.19" data-doi="10.1073/pnas.1005758108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinnamon</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyng</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Essential role for mast cell tryptase in acute experimental colitis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005758108%26rft_id%253Dinfo%253Apmid%252F21173247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDgvMS8yOTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.20" data-doi="10.1073/pnas.89.22.10915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Amino acid substitution matrices from protein blocks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">10915</span>–<span class="cit-lpage">10919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHenikoff%26rft.auinit1%253DS.%26rft.volume%253D89%26rft.issue%253D22%26rft.spage%253D10915%26rft.epage%253D10919%26rft.atitle%253DAmino%2Bacid%2Bsubstitution%2Bmatrices%2Bfrom%2Bprotein%2Bblocks.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.22.10915%26rft_id%253Dinfo%253Apmid%252F1438297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODkvMjIvMTA5MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.21" data-doi="10.1186/gb-2006-7-4-r31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulsen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vlieg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">P. M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Benchmarking ortholog identification methods using functional genomics data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHulsen%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253DR31%26rft.epage%253DR31%26rft.atitle%253DBenchmarking%2Bortholog%2Bidentification%2Bmethods%2Busing%2Bfunctional%2Bgenomics%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-4-r31%26rft_id%253Dinfo%253Apmid%252F16613613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-4-r31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16613613&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.22" data-doi="10.1093/molbev/msr272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The effects of alignment error and alignment filtering on the sitewise detection of positive selection.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1125</span>–<span class="cit-lpage">1139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr272%26rft_id%253Dinfo%253Apmid%252F22049066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzQvMTEyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.23" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT—the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.24" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Human Genome Browser at UCSC.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.25" data-doi="10.1023/A:1022859003006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuncheva</span>  <span class="cit-name-given-names">L. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Measures of diversity in classifier ensembles and their relationship with the ensemble accuracy.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D51%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1022859003006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1022859003006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.26" data-doi="10.1016/j.tig.2008.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuzniar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ham</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pongor</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leunissen</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The quest for orthologs: finding the corresponding gene across genomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKuzniar%26rft.auinit1%253DA.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D539%26rft.epage%253D551%26rft.atitle%253DThe%2Bquest%2Bfor%2Borthologs%253A%2Bfinding%2Bthe%2Bcorresponding%2Bgene%2Bacross%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.08.009%26rft_id%253Dinfo%253Apmid%252F18819722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18819722&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261533700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Laan</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Collaborative double robust targeted maximum likelihood estimation.</span> <abbr class="cit-jnl-abbrev">Int. J. Biostat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBiostat.%26rft.volume%253D6%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.629.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Laan</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Super learner.</span> <span class="cit-source">Stat. Appl. Genet. Mol. Biol.</span> <span class="cit-vol">6</span>: <span class="cit-fpage">Article 25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DStat%2BAppl%2BGenet%2BMol%2BBiol%26rft.aulast%253Dvan%2Bder%2BLaan%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D6%26rft.spage%253DArticle25%26rft.epage%253DArticle25%26rft.atitle%253DSuper%2Blearner.%26rft_id%253Dinfo%253Apmid%252F17910531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17910531&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.29" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.30" data-doi="10.1093/bioinformatics/btq338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maskell</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MSAProbs: multiple sequence alignment based on pair hidden Markov models and partition function posterior probabilities.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1958</span>–<span class="cit-lpage">1964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq338%26rft_id%253Dinfo%253Apmid%252F20576627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTYvMTk1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.31" data-doi="10.1080/10635150500354928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maddison</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inferring phylogeny despite incomplete lineage sorting.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150500354928%26rft_id%253Dinfo%253Apmid%252F16507521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjc6IjU1LzEvMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.32" data-doi="10.1371/journal.pbio.1001843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mashiyama</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malabanan</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akiva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Large-scale determination of sequence, structure, and function relationships in cytosolic glutathione transferases across the biosphere.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1001843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D12%26rft.spage%253De1001843%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001843%26rft_id%253Dinfo%253Apmid%252F24756107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24756107&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.33" data-doi="10.1534/genetics.104.032144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massingham</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting amino acid sites under positive selection and purifying selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032144%26rft_id%253Dinfo%253Apmid%252F15654091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE3NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.629.34"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">McEntyre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Ostell</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite> (Editors), <span class="cit-pub-date">2002</span> <span class="cit-source">The NCBI Handbook</span>. <span class="cit-publ-name">National Center for Biotechnology Information</span>, <span class="cit-publ-loc">Bethesda, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.35" data-doi="10.1101/gr.080531.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harte</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekhans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The consensus coding sequence (CCDS) project: identifying a common protein-coding gene set for the human and mouse genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1316</span>–<span class="cit-lpage">1323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.080531.108%26rft_id%253Dinfo%253Apmid%252F19498102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTMxNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.36" data-doi="10.1093/nar/gkt1114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiatt</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibaud-Nissen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astashyn</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RefSeq: an update on mammalian reference sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D756</span>–<span class="cit-lpage">D763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1114%26rft_id%253Dinfo%253Apmid%252F24259432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9ENzU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.37" data-doi="10.1093/nar/gkq953"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryszcz</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MetaPhOrs: orthology and paralogy predictions from multiple phylogenetic evidence using a consistency-based confidence score.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq953%26rft_id%253Dinfo%253Apmid%252F21149260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM5LzUvZTMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.38" data-doi="10.1093/nar/gkr1065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Punta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Pfam protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D290</span>–<span class="cit-lpage">D301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1065%26rft_id%253Dinfo%253Apmid%252F22127870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMjkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzYyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.39" data-doi="10.1006/jmbi.2000.5197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storm</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">1041</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRemm%26rft.auinit1%253DM.%26rft.volume%253D314%26rft.issue%253D5%26rft.spage%253D1041%26rft.epage%253D1052%26rft.atitle%253DAutomatic%2Bclustering%2Bof%2Borthologs%2Band%2Bin-paralogs%2Bfrom%2Bpairwise%2Bspecies%2Bcomparisons.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.5197%26rft_id%253Dinfo%253Apmid%252F11743721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.5197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11743721&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173365300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.40" data-doi="10.1016/0025-5564(81)90043-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Comparison of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D53%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252881%252990043-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(81)90043-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LB66300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rokach</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ensemble-based classifiers.</span> <abbr class="cit-jnl-abbrev">Artif. Intell. Rev.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArtif.%2BIntell.%2BRev.%26rft.volume%253D33%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000273807600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.629.42" data-doi="10.1371/journal.pone.0018755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evaluating ortholog prediction algorithms in a yeast model clade.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">6</span>: <span class="cit-fpage">e18755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSalichos%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De18755%26rft.epage%253De18755%26rft.atitle%253DEvaluating%2Bortholog%2Bprediction%2Balgorithms%2Bin%2Ba%2Byeast%2Bmodel%2Bclade.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018755%26rft_id%253Dinfo%253Apmid%252F21533202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21533202&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.43" data-doi="10.2174/138161207779313579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sommerhoff</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaschke</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mast cell tryptase beta as a target in allergic inflammation: an evolving story.</span> <abbr class="cit-jnl-abbrev">Curr. Pharm. Des.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bpharmaceutical%2Bdesign%26rft.stitle%253DCurr%2BPharm%2BDes%26rft.aulast%253DSommerhoff%26rft.auinit1%253DC.%2BP.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D313%26rft.epage%253D332%26rft.atitle%253DMast%2Bcell%2Btryptase%2Bbeta%2Bas%2Ba%2Btarget%2Bin%2Ballergic%2Binflammation%253A%2Ban%2Bevolving%2Bstory.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138161207779313579%26rft_id%253Dinfo%253Apmid%252F17313363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138161207779313579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17313363&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.44" data-doi="10.1093/bioinformatics/btq205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alachiotis</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Time and memory efficient likelihood-based tree searches on phylogenomic alignments with missing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i132</span>–<span class="cit-lpage">i139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq205%26rft_id%253Dinfo%253Apmid%252F20529898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTIvaTEzMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.45" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.46" data-doi="10.1093/bioinformatics/bts386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stolzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sathaye</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vernot</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inferring duplications, losses, transfers and incomplete lineage sorting with nonbinary species trees.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">i409</span>–<span class="cit-lpage">i415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts386%26rft_id%253Dinfo%253Apmid%252F22962460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvaTQwOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-5.4.629.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">An asymptotic equivalence of choice of model by cross-validation and Akaike’s criterion.</span> <span class="cit-source">J. R. Stat. Soc.</span> <span class="cit-vol">39</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%26rft.volume%253D39%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.48" data-doi="10.1093/bioinformatics/btq228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holder</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">DendroPy: a Python library for phylogenetic computing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq228%26rft_id%253Dinfo%253Apmid%252F20421198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTIvMTU2OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.49" data-doi="10.1081/JAS-120002288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamaoki</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawatani</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagai</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Serum B12 tryptase level as a marker of allergic airway inflammation in asthma.</span> <abbr class="cit-jnl-abbrev">J. Asthma</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Basthma%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BAssociation%2Bfor%2Bthe%2BCare%2Bof%2BAsthma%26rft.stitle%253DJ%2BAsthma%26rft.aulast%253DTaira%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DSerum%2BB12%2Btryptase%2Blevel%2Bas%2Ba%2Bmarker%2Bof%2Ballergic%2Bairway%2Binflammation%2Bin%2Basthma.%26rft_id%253Dinfo%253Adoi%252F10.1081%252FJAS-120002288%26rft_id%253Dinfo%253Apmid%252F12095181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1081/JAS-120002288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12095181&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176427300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.50" data-doi="10.4049/jimmunol.179.9.6072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trivedi</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwandin</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caughey</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mast cell alpha and beta tryptases changed rapidly during primate speciation and evolved from gamma-like transmembrane peptidases in ancestral vertebrates.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">6072</span>–<span class="cit-lpage">6079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DTrivedi%26rft.auinit1%253DN.%2BN.%26rft.volume%253D179%26rft.issue%253D9%26rft.spage%253D6072%26rft.epage%253D6079%26rft.atitle%253DMast%2BCell%2B%257Balpha%257D%2Band%2B%257Bbeta%257D%2BTryptases%2BChanged%2BRapidly%2Bduring%2BPrimate%2BSpeciation%2Band%2BEvolved%2Bfrom%2B%257Bgamma%257D-Like%2BTransmembrane%2BPeptidases%2Bin%2BAncestral%2BVertebrates%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.179.9.6072%26rft_id%253Dinfo%253Apmid%252F17947681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE3OS85LzYwNzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.51" data-doi="10.1109/4235.585893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolpert</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macready</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">No free lunch theorems for optimization.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Evol. Comput.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BEvol.%2BComput.%26rft.volume%253D1%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1109%252F4235.585893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/4235.585893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.52" data-doi="10.1038/nrg3174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ence</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A beginner’s guide to eukaryotic genome annotation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DYandell%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D329%26rft.epage%253D342%26rft.atitle%253DA%2Bbeginner%2527s%2Bguide%2Bto%2Beukaryotic%2Bgenome%2Bannotation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3174%26rft_id%253Dinfo%253Apmid%252F22510764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22510764&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F629.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.53" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.54" data-doi="10.1093/nar/gkr308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zavaljevski</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reifman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">QuartetS: a fast and accurate algorithm for large-scale orthology detection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr308%26rft_id%253Dinfo%253Apmid%252F21572104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzEzL2U4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC82MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.629.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prediction and characterization of enzymatic activities guided by sequence similarity and genome neighborhood networks.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/629.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/615" title="Accuracy of Whole-Genome Prediction Using a Genetic Architecture-Enhanced Variance-Covariance Matrix" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/639" title="Conserved RNA-Binding Proteins Required for Dendrite Morphogenesis in Caenorhabditis elegans Sensory Neurons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/629" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vZ5c24EPfgMDeZXkkXNrDyR8p6X_PIxfLmTbmPiAzM0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/629" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/629" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/629" class="active" data-icon-position="" data-hide-link-title="0">Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459795" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qJDg2xBKClPO-fqobTccOHGNm62hQ0YbG12USTpQlcA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/629.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/629" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-dfK7W5XkXjl831PT2If1Jzfr7tKB5oDx7AjS-S1lrfI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/629" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/629.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459795" id="node459795--2" data-pisa="ggg;5/4/629" data-pisa-master="ggg;g3.115.017095" data-apath="/ggg/5/4/629.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">M. Cyrus Maher</span> and <span class="highwire-citation-author" data-delta="1">Ryan D. Hernandez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">629-638;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017095 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459795--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Cyrus Maher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, University of California, San Francisco, University of California, San Francisco, San Francisco, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BCyrus%2BMaher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maher%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BCyrus%2BMaher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan D. Hernandez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, San Francisco, California</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Human Genetics, University of California, San Francisco, San Francisco, California</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Quantitative Biosciences (QB3), University of California, San Francisco, San Francisco, California 94158</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BD.%2BHernandez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hernandez%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BD.%2BHernandez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ryan.hernandez@ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">ryan.hernandez@ucsf.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/629" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/629&amp;title=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/629&amp;title=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/629&amp;title=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/629&amp;text=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/629&amp;title=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/629&amp;t=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/629&amp;title=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/629&amp;title=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459795/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459795" id="node459795--3" data-pisa="ggg;5/4/629" data-pisa-master="ggg;g3.115.017095" data-apath="/ggg/5/4/629.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rock, Paper, Scissors: Harnessing Complementarity in Ortholog Detection Methods Improves Comparative Genomic Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">M. Cyrus Maher</span> and <span class="highwire-citation-author" data-delta="1">Ryan D. Hernandez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">629-638;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017095 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459795--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Cyrus Maher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, University of California, San Francisco, University of California, San Francisco, San Francisco, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BCyrus%2BMaher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maher%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BCyrus%2BMaher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan D. Hernandez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, San Francisco, California</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Human Genetics, University of California, San Francisco, San Francisco, California</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Quantitative Biosciences (QB3), University of California, San Francisco, San Francisco, California 94158</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BD.%2BHernandez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hernandez%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BD.%2BHernandez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ryan.hernandez@ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">ryan.hernandez@ucsf.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=629&amp;rft.epage=638&amp;rft.atitle=Rock%2C%20Paper%2C%20Scissors%3A%20Harnessing%20Complementarity%20in%20Ortholog%20Detection%20Methods%20Improves%20Comparative%20Genomic%20Inference&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Maher%2C+M.+Cyrus&amp;rft.au=Hernandez%2C+Ryan+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459795/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459795/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459795/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459795/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459795/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459795/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459795/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459795/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459795/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459795/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459795/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459795/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/629" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MOSAIC: OD integration as cluster optimization</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/629.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/629.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/4/629.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/4/629.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
