<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/4/559" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/4/559.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/4/559.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/4/559.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/4/559.atom" />
<meta name="HW.pisa" content="ggg;5/4/559" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis" />
<meta name="DC.Identifier" content="10.1534/g3.114.014571" />
<meta name="DC.Date" content="2015-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Mohd-Yusoff et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic structure can be altered by chemical mutagenesis, which is a common method applied in molecular biology and genetics. Second-generation sequencing provides a platform to reveal base alterations occurring in the whole genome due to mutagenesis. A model legume, Lotus japonicus ecotype Miyakojima , was chemically mutated with alkylating ethyl methanesulfonate (EMS) for the scanning of DNA lesions throughout the genome. Using second-generation sequencing, two individually mutated third-generation progeny (M3, named AM and AS) were sequenced and analyzed to identify single nucleotide polymorphisms and reveal the effects of EMS on nucleotide sequences in these mutant genomes. Single-nucleotide polymorphisms were found in every 208 kb (AS) and 202 kb (AM) with a bias mutation of G/C-to-A/T changes at low percentage. Most mutations were intergenic. The mutation spectrum of the genomes was comparable in their individual chromosomes; however, each mutated genome has unique alterations, which are useful to identify causal mutations for their phenotypic changes. The data obtained demonstrate that whole genomic sequencing is applicable as a high-throughput tool to investigate genomic changes due to mutagenesis. The identification of these single-point mutations will facilitate the identification of phenotypically causative mutations in EMS-mutated germplasm." />
<meta name="DC.Contributor" content="Nur Fatihah Mohd-Yusoff" />
<meta name="DC.Contributor" content="Pradeep Ruperao" />
<meta name="DC.Contributor" content="Nurain Emylia Tomoyoshi" />
<meta name="DC.Contributor" content="David Edwards" />
<meta name="DC.Contributor" content="Peter M. Gresshoff" />
<meta name="DC.Contributor" content="Bandana Biswas" />
<meta name="DC.Contributor" content="Jacqueline Batley" />
<meta name="article:published_time" content="2015-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic structure can be altered by chemical mutagenesis, which is a common method applied in molecular biology and genetics. Second-generation sequencing provides a platform to reveal base alterations occurring in the whole genome due to mutagenesis. A model legume, &lt;i&gt;Lotus japonicus&lt;/i&gt; ecotype &lt;i&gt;Miyakojima&lt;/i&gt;, was chemically mutated with alkylating ethyl methanesulfonate (EMS) for the scanning of DNA lesions throughout the genome. Using second-generation sequencing, two individually mutated third-generation progeny (M3, named AM and AS) were sequenced and analyzed to identify single nucleotide polymorphisms and reveal the effects of EMS on nucleotide sequences in these mutant genomes. Single-nucleotide polymorphisms were found in every 208 kb (AS) and 202 kb (AM) with a bias mutation of G/C-to-A/T changes at low percentage. Most mutations were intergenic. The mutation spectrum of the genomes was comparable in their individual chromosomes; however, each mutated genome has unique alterations, which are useful to identify causal mutations for their phenotypic changes. The data obtained demonstrate that whole genomic sequencing is applicable as a high-throughput tool to investigate genomic changes due to mutagenesis. The identification of these single-point mutations will facilitate the identification of phenotypically causative mutations in EMS-mutated germplasm.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/04/01" />
<meta name="citation_mjid" content="ggg;5/4/559" />
<meta name="citation_id" content="5/4/559" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/4/559" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/4/559.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/4/559.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/4/559.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.014571" />
<meta name="citation_pmid" content="25660167" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="559" />
<meta name="citation_lastpage" content="567" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nur Fatihah Mohd-Yusoff" />
<meta name="citation_author_institution" content="Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author_institution" content="Department of Cell and Molecular Biology, Faculty of Biotechnology and Biomolecular Sciences, Universiti Putra Malaysia, 43400, Serdang, Selangor, Malaysia" />
<meta name="citation_author" content="Pradeep Ruperao" />
<meta name="citation_author_institution" content="Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author_institution" content="Centre of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India" />
<meta name="citation_author" content="Nurain Emylia Tomoyoshi" />
<meta name="citation_author_institution" content="Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author" content="David Edwards" />
<meta name="citation_author_institution" content="Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author_institution" content="School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia" />
<meta name="citation_author" content="Peter M. Gresshoff" />
<meta name="citation_author_institution" content="Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author_email" content="p.gresshoff@uq.edu.au" />
<meta name="citation_author" content="Bandana Biswas" />
<meta name="citation_author_institution" content="Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author" content="Jacqueline Batley" />
<meta name="citation_author_institution" content="Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia" />
<meta name="citation_author_institution" content="School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Ashelford;citation_author=M. E. Eriksson;citation_author=C. M. Allen;citation_author=R. D’Amore;citation_author=M. Johansson;citation_title=Full genome re-sequencing reveals a novel circadian clock mutation in Arabidopsis.;citation_pages=R28-R28;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21429190;citation_doi=10.1186/gb-2011-12-3-r28" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=R. S. Austin;citation_author=D. Vidaurre;citation_author=G. Stamatiou;citation_author=R. Breit;citation_author=N. J. Provart;citation_title=Next-generation mapping of Arabidopsis genes.;citation_pages=715-725;citation_volume=67;citation_year=2011;citation_issue=4;citation_pmid=21518053;citation_doi=10.1111/j.1365-313X.2011.04619.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=B. Biswas;citation_author=P. K. Chan;citation_author=P. M. Gresshoff;citation_title=A novel ABA insensitive mutant of Lotus japonicus with a wilty phenotype displays unaltered nodulation regulation.;citation_volume=2;citation_year=2009;citation_pmid=19825632;citation_doi=10.1093/mp/ssp009" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. G. Caldwell;citation_author=N. McCallum;citation_author=P. Shaw;citation_author=G. J. Muehlbauer;citation_author=D. F. Marshall;citation_title=A structured mutant population for forward and reverse genetics in Barley (Hordeum vulgare L.).;citation_pages=143-150;citation_volume=40;citation_year=2004;citation_issue=1;citation_pmid=15361148;citation_doi=10.1111/j.1365-313X.2004.02190.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=B. J. Carroll;citation_author=D. L. McNeil;citation_author=P. M. Gresshoff;citation_title=Mutagenesis of soybean (Glycine max (L.) Merr) and the isolation of non-nodulating mutants.;citation_pages=109-114;citation_volume=47;citation_year=1986;citation_doi=10.1016/0168-9452(86)90057-9" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=P. K. Chan;citation_author=B. Biswas;citation_author=P. M. Gresshoff;citation_title=Classical ethylene insensitive mutants of the Arabidopsis EIN2 orthologue lack the expected ‘hypernodulation’ response in Lotus japonicus.;citation_pages=395-408;citation_volume=55;citation_year=2013;citation_pmid=23452324;citation_doi=10.1111/jipb.12040" />
<meta name="citation_reference" content="citation_journal_title=Landes Bioscience;citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=1-13;citation_volume=6;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=S. R. Cutler;citation_author=P. L. Rodriguez;citation_author=R. R. Finkelstein;citation_author=S. R. Abrams;citation_title=Abscisic acid: emergence of a core signaling network.;citation_pages=651-679;citation_volume=61;citation_year=2010;citation_pmid=20192755;citation_doi=10.1146/annurev-arplant-042809-112122" />
<meta name="citation_reference" content="citation_journal_title=Funct. Integr. Genomics;citation_author=J. Dalton-Morgan;citation_author=A. Hayward;citation_author=S. Alamery;citation_author=R. Tollenaere;citation_author=A. S. Mason;citation_title=A high-throughput SNP array in the amphidiploid species Brassica napus shows diversity in resistance genes.;citation_pages=643-655;citation_volume=14;citation_year=2014;citation_pmid=25147024;citation_doi=10.1007/s10142-014-0391-2" />
<meta name="citation_reference" content="citation_journal_title=Asiatic J. Biotechnol. Resour.;citation_author=K. G. Dube;citation_author=A. S. Bajaj;citation_author=A. M. Gawande;citation_title=Mutagenic efficiency and effectiveness of gamma rays and EMS in Cyamopsis tetragonoloba (L.) var. Sharada.;citation_pages=436-440;citation_volume=2;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Edwards;citation_author=J. Batley;citation_author=R. J. Snowdon;citation_title=Accessing complex crop genomes with next-generation sequencing.;citation_pages=1-11;citation_volume=126;citation_year=2013;citation_pmid=22948437;citation_doi=10.1007/s00122-012-1964-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=I. Chaudhry;citation_author=J. Taylor;citation_author=S. E. Neil;citation_title=Whole-genome sequencing profiling of mutagenesis in Caenorhabditis elegans.;citation_volume=185;citation_year=2010;citation_pmid=20439774;citation_doi=10.1534/genetics.110.116616" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. Fujita;citation_author=Y. Fujita;citation_author=Y. Noutoshi;citation_author=F. Takahashi;citation_author=Y. Narusaka;citation_title=Crosstalk between abiotic and biotic stress responses: a current view from the points of convergence in the stress signaling networks.;citation_pages=436-442;citation_volume=9;citation_year=2006;citation_issue=4;citation_pmid=16759898;citation_doi=10.1016/j.pbi.2006.05.014" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. M. Goodstein;citation_author=S. Shu;citation_author=R. Howson;citation_author=R. Neupane;citation_author=R. D. Hayes;citation_title=Phytozome: a comparative platform for green plant genomics.;citation_volume=40;citation_year=2012;citation_pmid=22110026;citation_doi=10.1093/nar/gkr944" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Greene;citation_author=C. A. Codomo;citation_author=N. E. Taylor;citation_author=J. G. Henikoff;citation_author=B. J. Till;citation_title=Spectrum of Chemically Induced Mutations From a Large-Scale Reverse-Genetic Screen in Arabidopsis;citation_pages=731-740;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807792" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=B. Hartwig;citation_author=G. V. James;citation_author=K. Konrad;citation_author=K. Schneeberger;citation_author=F. Turck;citation_title=Fast isogenic mapping-by-sequencing of ethyl methanesulfonate-induced mutant bulks.;citation_volume=160;citation_year=2012;citation_pmid=22837357;citation_doi=10.1104/pp.112.200311" />
<meta name="citation_reference" content="citation_title=Biotechnology in Agricultural Chemistry, ACS Symposium Series;citation_title=Selection for herbicide resistance at the whole-plant level;citation_author=G. Haughn;citation_author=C. R. Somerville;citation_pages=98-107;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=DNA Research;citation_journal_abbrev=DNA Res;citation_author=T. Kato;citation_author=T. Kaneko;citation_author=S. Sato;citation_author=Y. Nakamura;citation_author=S. Tabata;citation_title=Complete Structure of the Chloroplast Genome of a Legume, Lotus japonicus;citation_pages=323-330;citation_volume=7;citation_year=2000;citation_issue=6;citation_pmid=11214967;citation_doi=10.1093/dnares/7.6.323" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=S. H. Kazakoff;citation_author=M. Imelfort;citation_author=D. Edwards;citation_author=J. Koehorst;citation_author=B. Biswas;citation_title=Capturing the biofuel wellhead and powerhouse: the chloroplast and mitochondrial genomes of the leguminous feedstock tree Pongamia pinnata.;citation_pages=e51687;citation_volume=7;citation_year=2012;citation_pmid=23272141;citation_doi=10.1371/journal.pone.0051687" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Growth Regul.;citation_author=A. R. Kermode;citation_title=Role of abscisic acid in seed dormancy.;citation_pages=319-344;citation_volume=24;citation_year=2005;citation_doi=10.1007/s00344-005-0110-2" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=T. H. Kim;citation_author=M. Böhmer;citation_author=H. Hu;citation_author=N. Nishimura;citation_author=J. I. Schroeder;citation_title=Guard cell signal transduction network: advances in understanding abscisic acid, CO2, and Ca2+ signaling.;citation_pages=561-591;citation_volume=61;citation_year=2010;citation_pmid=20192751;citation_doi=10.1146/annurev-arplant-042809-112226" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=K. Lai;citation_author=M. T. Lorenc;citation_author=C. L. Hong;citation_author=P. J. Berkman;citation_author=P. E. Bayer;citation_title=Identification and characterisation of more than 4 million intervarietal SNPs across the group 7 chromosomes of bread wheat.;citation_pages=97-104;citation_volume=13;citation_year=2015;citation_pmid=25147022;citation_doi=10.1111/pbi.12240" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Journal;citation_journal_abbrev=Biochem. J.;citation_author=P. D. Lawley;citation_author=C. N. Martin;citation_title=Molecular mechanisms in alkylation mutagenesis. Induced reversion of bacteriophage T4rII AP72 by ethyl methanesulphonate in relation to extent and mode of ethylation of purines in bacteriophage deoxyribonucleic acid;citation_pages=85-91;citation_volume=145;citation_year=1975;citation_issue=1;citation_pmid=172067;citation_doi=10.1042/bj1450085" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Li;citation_author=X. Dai;citation_author=T. Liu;citation_author=P. X. Zhao;citation_title=LegumeIP: an integrative database for comparative genomics and transcriptomics of model legumes.;citation_volume=40;citation_year=2012;citation_pmid=22110036;citation_doi=10.1093/nar/gkr939" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Li;citation_author=C. Yu;citation_author=Y. Li;citation_author=T. W. Lam;citation_author=S. M. Yiu;citation_title=SOAP2: an improved ultrafast tool for short read alignment.;citation_volume=25;citation_year=2009;citation_pmid=19497933;citation_doi=10.1093/bioinformatics/btp336" />
<meta name="citation_reference" content="citation_journal_title=Biology;citation_author=M. T. Lorenc;citation_author=S. Hayashi;citation_author=J. Stiller;citation_author=H. Lee;citation_author=S. Manoli;citation_title=Discovery of single nucleotide polymorphisms in complex genomes using SGSautoSNP.;citation_pages=370-382;citation_volume=1;citation_year=2012;citation_pmid=24832230;citation_doi=10.3390/biology1020370" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Loveless;citation_title=Increased rate of plaque-type and host-range mutation following treatment of bacteriophage in vitro with ethyl methane sulphonate.;citation_pages=1212-1213;citation_volume=181;citation_year=1958;citation_issue=4617;citation_pmid=13541432;citation_doi=10.1038/1811212a0" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=C. M. McCallum;citation_author=L. Comai;citation_author=E. A. Greene;citation_author=S. Henikoff;citation_title=Targeting Induced Local Lesions IN Genomes (TILLING) for Plant Functional Genomics;citation_pages=439-442;citation_volume=123;citation_year=2000;citation_issue=2;citation_pmid=10859174;citation_doi=10.1104/pp.123.2.439" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=S. Minoia;citation_author=A. Petrozza;citation_author=O. D’Onofrio;citation_author=F. Piron;citation_author=G. Mosca;citation_title=A new mutant genetic resource for tomato crop improvement by TILLING technology.;citation_pages=69-69;citation_volume=3;citation_year=2010;citation_pmid=20222995;citation_doi=10.1186/1756-0500-3-69" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Mokry;citation_author=I. J. Nijman;citation_author=A. van Dijken;citation_author=R. Benjamins;citation_author=R. Heidstra;citation_title=Identification of factors required for meristem function in Arabidopsis using a novel next generation sequencing fast forward genetics approach.;citation_pages=256-256;citation_volume=12;citation_year=2011;citation_pmid=21599977;citation_doi=10.1186/1471-2164-12-256" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=Predicting the effects of amino acid substitutions on protein function.;citation_pages=61-80;citation_volume=7;citation_year=2006;citation_pmid=16824020;citation_doi=10.1146/annurev.genom.7.080505.115630" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=O. Olsen;citation_author=X. Wang;citation_author=D. von Wettstein;citation_title=Sodium azide mutagenesis: preferential generation of A.T--&gt;G.C transitions in the barley Ant18 gene.;citation_pages=8043-8047;citation_volume=90;citation_year=1993;citation_issue=17;citation_pmid=8367460;citation_doi=10.1073/pnas.90.17.8043" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. A. Perry;citation_author=T. L. Wang;citation_author=T. J. Welham;citation_author=S. Gardner;citation_author=J. M. Pike;citation_title=A TILLING Reverse Genetics Tool and a Web-Accessible Collection of Mutants of the Legume Lotus japonicus;citation_pages=866-871;citation_volume=131;citation_year=2003;citation_issue=3;citation_pmid=12644638;citation_doi=10.1104/pp.102.017384" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Perry;citation_author=A. Brachmann;citation_author=T. Welham;citation_author=A. Binder;citation_author=M. Charpentier;citation_title=TILLING in Lotus japonicus identified large allelic series for symbiosis genes and revealed a bias in functionally defective ethyl methanesulfonate alleles toward glycine replacements.;citation_volume=151;citation_year=2009;citation_pmid=19641028;citation_doi=10.1104/pp.109.142190" />
<meta name="citation_reference" content="citation_title=Abiotic Stress Adaptation in Plants: Physiological, Molecular and Genomic Foundation;citation_title=Stress signaling I: The role of abscisic acid (ABA);citation_author=C. D. Rock;citation_author=Y. Sakata;citation_author=R. S. Quatrano;citation_pages=33-73;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual;citation_author=J. Sambrook;citation_author=D. W. Russel;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=S. Sato;citation_author=S. Tabata;citation_title=Lotus japonicus as a platform for legume research.;citation_pages=128-132;citation_volume=9;citation_year=2005;citation_doi=10.1016/j.pbi.2006.01.008" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=S. Sato;citation_author=Y. Nakamura;citation_author=T. Kaneko;citation_author=E. Asamizu;citation_author=T. Kato;citation_title=Genome structure of the legume, Lotus japonicus.;citation_volume=15;citation_year=2008;citation_pmid=18511435;citation_doi=10.1093/dnares/dsn008" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=G. A. Sega;citation_title=A review of the genetic effects of ethyl methanesulfonate.;citation_pages=113-142;citation_volume=134;citation_year=1984;citation_issue=2-3;citation_pmid=6390190;citation_doi=10.1016/0165-1218(84)90033-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=X. Serrat;citation_author=R. Esteban;citation_author=N. Guibourt;citation_author=L. Moysset;citation_author=S. Nogués;citation_title=EMS mutagenesis in mature seed-derived rice calli as a new method for rapidly obtaining TILLING mutant populations.;citation_pages=5;citation_volume=10;citation_year=2014;citation_pmid=24475756;citation_doi=10.1186/1746-4811-10-5" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Evol Biol;citation_author=Y. Shiwa;citation_author=S. Fukushima-Tanaka;citation_author=K. Kasahara;citation_author=T. Horiuchi;citation_author=H. Yoshikawa;citation_title=Whole-Genome Profiling of a Novel Mutagenesis Technique Using Proofreading-Deficient DNA Polymerase Î´.;citation_pages=860797-860797;citation_volume=2012;citation_year=2012;citation_pmid=22675654" />
<meta name="citation_reference" content="citation_title=Int. J. Plant Genom.;citation_author=P. Sikora;citation_author=A. Chawade;citation_author=M. Larsson;citation_author=J. Olsson;citation_author=O. Olsson;citation_title=Mutagenesis as a tool in plant genetics, functional genomics and breeding.;citation_pages=314829;citation_volume=2011;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Bot. Res. Intl.;citation_author=A. Sri Devi;citation_author=L. Mullainathan;citation_title=The use of ethyl methanesulfonate to study the flower development in Capsicum annuum L. mutants.;citation_pages=4-9;citation_volume=5;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=C. N. Stewart;citation_author=L. E. Via;citation_title=A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications.;citation_pages=748-750;citation_volume=14;citation_year=1993;citation_issue=5;citation_pmid=8512694" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=D. Tagu;citation_author=G.L. Trionnaire;citation_author=S. Tanguy;citation_author=J.P. Gauthier;citation_author=J.R. Huynh;citation_title=EMS mutagenesis in the pea aphid Acyrthosiphon pisum.;citation_volume=4;citation_year=2014;citation_pmid=24531730;citation_doi=10.1534/g3.113.009639" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=O. Thompson;citation_author=M. Edgley;citation_author=P. Strasbourger;citation_author=S. Flibotte;citation_author=B. Ewing;citation_title=The million mutation project: a new approach to genetics in Caenorhabditis elegans.;citation_volume=23;citation_year=2013;citation_pmid=23800452;citation_doi=10.1101/gr.157651.113" />
<meta name="citation_reference" content="citation_journal_title=Brief Funct. Genomics;citation_author=M. Thudi;citation_author=Y. Li;citation_author=S. A. Jackson;citation_author=G. D. May;citation_author=R. K. Varshney;citation_title=Current state-of-art of sequencing technologies for plant genomics research.;citation_volume=11;citation_year=2012;citation_pmid=22345601;citation_doi=10.1093/bfgp/elr045" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. J. Till;citation_author=J. Cooper;citation_author=T. H. Ti;citation_author=P. Colowit;citation_author=E. A. Greene;citation_title=Discovery of chemically induced mutations in rice by TILLING.;citation_pages=19-19;citation_volume=7;citation_year=2007;citation_pmid=17428339;citation_doi=10.1186/1471-2229-7-19" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Till;citation_author=S. H. Reynolds;citation_author=E. A. Greene;citation_author=C. A. Codomo;citation_author=L. C. Enns;citation_title=Large scale discovery of induced point mutations with high-throughput TILLING.;citation_pages=524-530;citation_volume=13;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Journal of biosciences;citation_journal_abbrev=J Biosci;citation_author=R. K. Varshney;citation_author=H. Kudapa;citation_author=M. Roorkiwal;citation_author=M. Thudi;citation_author=M. Pandey;citation_title=Advances in genetics and molecular breeding of three legume crops of semi-arid tropics using  next-generation sequencing and high-throughput genotyping technologies.;citation_pages=811-820;citation_volume=37;citation_year=2012;citation_issue=5;citation_pmid=23107917;citation_doi=10.1007/s12038-012-9228-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=A. Wasilewska;citation_author=F. Vlad;citation_author=C. Sirichandra;citation_author=Y. Redko;citation_author=F. Jammes;citation_title=An update on abscisic acid signalling in plants and more.;citation_volume=1;citation_year=2008;citation_pmid=19825533;citation_doi=10.1093/mp/ssm022" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=Z. Xin;citation_author=M. L. Wang;citation_author=N. A. Barkley;citation_author=G. Burow;citation_author=C. Franks;citation_title=Applying genotyping (TILLING) and phenotyping analyses to elucidate gene function in a chemically induced sorghum mutant population.;citation_pages=103-103;citation_volume=8;citation_year=2008;citation_pmid=18854043;citation_doi=10.1186/1471-2229-8-103" />
<meta name="citation_reference" content="citation_journal_title=Funct. Integr. Genomics;citation_author=M. Zander;citation_author=D. A. Patel;citation_author=A. Van de Wouw;citation_author=K. Lai;citation_author=M. T. Lorenc;citation_title=Identifying genetic diversity of avirulence genes in Leptosphaeria maculans using whole genome sequencing.;citation_pages=295-308;citation_volume=13;citation_year=2013;citation_pmid=23793572;citation_doi=10.1007/s10142-013-0324-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Zhu;citation_author=H. Mang;citation_author=Q. Sun;citation_author=A. Hipps;citation_author=J. Hua;citation_title=Gene discovery using mutagen-induced polymorphisms and deep sequencing: application to plant disease resistance.;citation_volume=192;citation_year=2012;citation_pmid=22714407;citation_doi=10.1534/genetics.112.141986" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zuryn;citation_author=S. L. Gras;citation_author=K. Jamet;citation_author=S. Jarriault;citation_title=A strategy for direct mapping and identification of mutations by whole-genome sequencing.;citation_volume=186;citation_year=2010;citation_pmid=20610404;citation_doi=10.1534/genetics.110.119230" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
<meta name="twitter:description" content="Genetic structure can be altered by chemical mutagenesis, which is a common method applied in molecular biology and genetics. Second-generation sequencing provides a platform to reveal base alterations occurring in the whole genome due to mutagenesis. A model legume, Lotus japonicus ecotype Miyakojima , was chemically mutated with alkylating ethyl methanesulfonate (EMS) for the scanning of DNA lesions throughout the genome. Using second-generation sequencing, two individually mutated third-generation progeny (M3, named AM and AS) were sequenced and analyzed to identify single nucleotide polymorphisms and reveal the effects of EMS on nucleotide sequences in these mutant genomes. Single-nucleotide polymorphisms were found in every 208 kb (AS) and 202 kb (AM) with a bias mutation of G/C-to-A/T changes at low percentage. Most mutations were intergenic. The mutation spectrum of the genomes was comparable in their individual chromosomes; however, each mutated genome has unique alterations, which are useful to identify causal mutations for their phenotypic changes. The data obtained demonstrate that whole genomic sequencing is applicable as a high-throughput tool to investigate genomic changes due to mutagenesis. The identification of these single-point mutations will facilitate the identification of phenotypically causative mutations in EMS-mutated germplasm." />
<meta name="og-title" property="og:title" content="Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/4/559" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic structure can be altered by chemical mutagenesis, which is a common method applied in molecular biology and genetics. Second-generation sequencing provides a platform to reveal base alterations occurring in the whole genome due to mutagenesis. A model legume, Lotus japonicus ecotype Miyakojima , was chemically mutated with alkylating ethyl methanesulfonate (EMS) for the scanning of DNA lesions throughout the genome. Using second-generation sequencing, two individually mutated third-generation progeny (M3, named AM and AS) were sequenced and analyzed to identify single nucleotide polymorphisms and reveal the effects of EMS on nucleotide sequences in these mutant genomes. Single-nucleotide polymorphisms were found in every 208 kb (AS) and 202 kb (AM) with a bias mutation of G/C-to-A/T changes at low percentage. Most mutations were intergenic. The mutation spectrum of the genomes was comparable in their individual chromosomes; however, each mutated genome has unique alterations, which are useful to identify causal mutations for their phenotypic changes. The data obtained demonstrate that whole genomic sequencing is applicable as a high-throughput tool to investigate genomic changes due to mutagenesis. The identification of these single-point mutations will facilitate the identification of phenotypically causative mutations in EMS-mutated germplasm." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/4.cover-source.jpg" />
    <title>Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1VgcY91s8MEyyYxRfRyO23o431kqrRBZ-elDOeeiQyc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459805","apath":"\/ggg\/5\/4\/559.atom","pisa":"ggg;5\/4\/559","ac":{"\/ggg\/5\/4\/559.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/4\/559.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/4\/559"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/4\/559"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/4\/559":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iGi\\\\\\\/Qc9xmXmD394WljxVxifXegmcsehnNX6t18KCtdhCRlmpzFfswHSjfsgzSORNu5wNhjTl2hHuSZdd\\\\\\\/gUKNK\\\\\\\/Iizq+hkVRwQ\\\\\\\/k+ydEzGfe5Rdko7+QZ6WrXhhe4OQZjLwj7ouKg0UFzzmBJubRHZRhzFW241GFt3z6aWp+ZjXCNVxgArVsG9+JNNGp4RdLvAd1skdQGi3j200Sf5EFYq9xCm0CYn5QbQC5ffZ2fq2keU7y2N\\\\\\\/+nfMTR41+L73ZvSdMlK6n4gCjTlCP5nPEyrsrwjMcJH\\\\\\\/okrSdwT0aIBX+\\\\\\\/w5jkWBxuIPic3ve14Zt5nthxpeGcJB+GdSBJc2m1U\\\\\\\/hqKkddLT9Yf1b3ZDvdDgHtHGlyQWeDHFrbyg5uBuScAkTAaKt2fZ1fNbOFTwGn\\\\\\\/w06uXmLT3J\\\\\\\/fc8aMlXwQ7WbcUKFNnwGKBKqGmIGe+keMkzg6mQM2V44rAEfY5rtQqaBCtUi6UFDwbWRx+uXRhqsHaJQd7zo7Il3F+8NFTCqMzwkRg3jtud3lDIJn\\\\\\\/sZlNfhHx9NaVd8SRfg+4qnTuDuxZzss1wK0UJUNY7jNJQ1hbCyxyO8i6kzgjZPQ8owawzUUx\\\\\\\/IemjabbYIQCxOAybzlzv8vv8A69YHuw3XgVsYQkHuKsv54M\\\\\\\/tfcPuUWK0ODHNbvGAOnwM+\\\\\\\/b4jzONQ9mOMLUgfoSHFq8yGUCtagyTxsDWG8ZNGf8vsxbbWwxJCXmbpaoBV9+DxPf7JnIJACgH4n4RoohelEX8ogf4iew\\\\\\\/duspZ0i0BK35sy5Dx9eqwt7oGMvkI9h28KNocWJ5f6KMmfPOXDUQ3DLuElEyvE7w5UtDCG743VoONjBqP7B6nsIDRfFo4dxi\\\\\\\/EurJOxK0brPvy+ziF9xSBHwF9f2rgqb3rt6e4qJcnPeRl8FhviMAuUWIHq6vkaCeB6DWsfijdkayO+hy8NN+TErZGGoN3FZpJNIvck4xcotBjlpW+uBh\\\\\\\/ml\\\\\\\/OCZEDL+BH3a9aVgvwsQCjOvBOipjjpBXj0gTCYwPOLets1q+x5cZ8WNdAd2F7b1JD65m2vKdy59l2WhGn4725rxEhNqca0+7OcGXnhn8DqVrWeiTjWKs\\\\\\\/VXDj+NZkiyIZ3zmbo5JBOFGzzHc\\\\\\\/L4OK\\\\\\\/31TY92rSJYSK8TOhoZE8vlbowxaoZ2bD0qCpL6iH5\\\\\\\/YtePbbXPknHgN13yZcyLZIz7IaZRAih0diZWfLBg4UrPxhJ9w6vhGh87xZhjP4IFBbaxYL0OoWRF+ivykWNTZVukqkv2fH6kNGPkCE\\\u0022,\\\u0022iv\\\u0022:\\\u0022MUj9Q4U6dHxpl5ecVBTjBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022678b2ce6818ae258f45f0ff6b6fa533b876ff98672cbda63b65c9eba9555b0ff\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228XQlCoDVlyrPdO4+iCxNWhA\\\\\\\/tNjcVg0tipLCZyUrFFg9Aazu+cJU5l2ece6SwPYIADTKIH3l4pfKUszYQVIn8bIW9WB3fmJpMvbFIjQY0gMOcjvr70ZM5ROrcIlcfwthLzCW6OXmzmQgYWeEKbGlSUbGy9Cc\\\\\\\/rvXHIrDttbtZQxHkK38oiXdtU2K+s0AG5LGsk6sJ5DuDC0jD4GWX4\\\\\\\/qNwkNQgKoZqUbzamTyxN3F48PBu5EkAl2lCu0M7VPwFwqPLXnt49T08xkR1FnBV4\\\\\\\/o3VzQfNFPSnST2v16wyVzR2fAkK1vLFikfEBU8LQE7glWJb0MtkeftX9p+F7t3fOt1C+MGygyug0otPpNIZSdG3jUupHQVN6CU6keTI4L2sJTFb9vXzfUPlRrwpyVaOpULwD\\\\\\\/p3zLCI3UD5G6AB8fFWpsLU4OlqPFO4smqvq\\\\\\\/1r2XDD\\\\\\\/Jfs0y8pzV4T5mohGEm+i0Os4\\\\\\\/vifMPEJZeCv6zpfAr+hM0XYGf0qnre4HKDOTGaohzi848xbztXxet\\\\\\\/NsiIAwA9\\\\\\\/0PSfK76OBsJnXJNhHCBVp\\\\\\\/md2H+L2vs7H4+rmYNOca+6W\\\\\\\/wnoZNk5wYbwDEq50eKvh0ZEdDVhWG16gGUlHdWB5o7t5izpeCnJQw\\\\\\\/ZeLT14JWxj9tzoVrgWiHi06zTAn5bHLgtxRT8UISNYdnBjONH1vbKabxUpOZkUCjWrv04QeW9\\\\\\\/WwkRb70tZX3am3hBjXlvXY2KpK\\\\\\\/PCpMjVHHXYdb\\\\\\\/SPiSOZ\\\\\\\/+BVwziZhv7LCauslcPvSuwHBaYmmp2FUf6PM99q5tSs\\\\\\\/W9JIdd75mIpW5AiK4L5qOKM5zpFtofdhjs468i643Q5xI4is93lqYAS4fS7a3\\\\\\\/b86f1Hvt8G9SDBIT4629xzBPsIROBI9KxNO\\\\\\\/VzIc9Sb04fqCe\\\\\\\/CFrG9XWTB\\\\\\\/4curaWNR65LjdeHhdb7KOJDe0hLJ5WG3RuY2CMbWbCzNEyk4pe0FKlfkFJ5VpG40MnuqfJ1YQz3FkPnrhapnvJ0Q74mKYgXvsxGduQpEwBirmrEZtynj1DRqWKEb2G+MdfFzmxCwYV7MRgHwny0Av5maeMSPlGTPLX5yzX9BH0riDOO\\\\\\\/XU6WxVC14uvL\\\\\\\/VHI2KEBqj8OOhkzVtwHxX+WVvxrHk\\\\\\\/DSVjUjLrovRaSl96XnbCTBlUOruFf83xMIaxS4\\\\\\\/0X8Ot5azPR8ENzgRKGB5V1Y10I+zjScLanSjcrJ+WMcu5n+\\\\\\\/higv5zTBOCfsTDdmWnF3InRkFBUYrteabYVLBD9qSI8T\\\\\\\/7LamD57bpqhNJcDe5krXpTKKnXh32nMtccZTB05ClgrJDZ+yd1VXI\\\\\\\/7G\\\\\\\/a97\\\\\\\/AJj1ycWWKuDC51EB+L0jHrSZC6DyuFQv0KcxNPALad1mvsLsZUZ606vIL+hcUi+\\\\\\\/ym\\\\\\\/yjU3V+hnVmlGEVNgLnUjQGBfG7QOdQSA1H\\\\\\\/TG6ZgghFssUrEddcCV1Mb9PE8+7ybzcy\\\\\\\/uiPx6HTN55ulDRlaO44b9nAT3RlIeXnrqC4gCkvVrCtZrNJJ78yh9EBpEvUmzkLIcX0U\\\\\\\/V2m1ugf2E9A7AIKQAed6yMDdFXCu+7N+52TbQuixbKZTtod9SIuFnHEuBnfVO5NWx8RozsQDsBaXt4cl\\\\\\\/49nouU6a8ozkrVpK+gTV15sMLiHWnI0U4MrDZPKA2pzboZP6i5w1ibAaUQiG34CIEzpZ0q4\\\\\\\/u6cOMUBB4woxsp4A3hAoh0egozww6Bzt3jLA5gFjFhy084OpvRNADzYzX9Rhh9eT3jyI8fuTXT+7lZpDDSKxW1aL4eprp2UcxFBJCMiLmQFqPO2R9vSQ0At2DStF94oix5GgL12hKGTNI11vrUiVhSE5t0MA2nY1a72hcJ7xbVc4oEnlQiUksiD5VnvHTWrHRQrm31PF+D95+L3eZ\\\\\\\/7vYzKsD1N+NNkKzFOLDKuI5yGaeSu7UGkV2SC1BsTuGhRe\\\\\\\/1ry0TpPRVBD7GIPLteLU5QI0Ouet4VMhT7SC71eWQS36M1AdJHgNDBWTiSL9uy3kg0epAHWwTSNQdr8uijOm4UIooR\\\\\\\/l\\\\\\\/tRn3bZaD3UU+vQzATStgLJk9Hut7vv1qYHQxSZdmjIP3QpeIpy2lqQL73tdLAfFvwsq7hdOLRg00MtitbjqtY8zUBNesxViRYAHl8oVWsS5kMbk4kmtJkV3zdmXwEbAgiTMvZIsYPV+LTgkeo9f1X5+NinkLyXSHQLv5U8bKj4SgBMTpRAJ\\\\\\\/B4LaTwcIhVQ\\\\\\\/LpM2O40XylEyN2NxTqQuUZrjqTwSs2jX8KKGs6Nu\\\\\\\/wePrMXIL2Km4KNQsNtQVb8S6twPiE8DnuYKVN5yNsKMU4z\\\\\\\/9Hbjvxwfgt0xUswmgie2kMksjBfDCiIZQPWVZ41bBJ2SLaKVP4KjSPFk\\\\\\\/7knjt+smNd\\\\\\\/NS6O3xMd1rgwuu5YNjG2Z4Z5pGoL0xUMwIC6fjIxC9iJQZYlyxBHa2IkvbFSp38gIARJcYNqMjmsF8CxdiFDnAdQATuIOkgdtNbHoHgxmnXv30Db0OkOspxTtLyyTvQ3dloqTMdaMVT4Oy94TeevT\\\\\\\/xDN\\\\\\\/V+wd+IAJ3G4CmStSi02WZlaLTZnkYbx6gQFlQUIis9591tiqTDvhEnK2STujQCPxsyyaOeQ8NqOzHz66lkDAmryGkruJN8sNyV6zcJkE5LtLjLWJwDaylcpIvUIbh90mkYTj3Woc4brO\\\\\\\/\\\\\\\/aU3xpf+dTVnuQGiLW35GLmmzMUN1+iWeJu8YdRd92Fm3Ucj7zISnG8hPVkU+dxbBS+gVx3dkb4oXsTco0Z4yjrsOZ3BRVnm3HIYkFOFxQyJo357euyhbVXbgfW2aNHfmE6XmjcvgAOeDIrH5pav6sR75nWNS2p2JjhlSL+0\\\\\\\/Q28LBajwk6RwoRuDhDX\\\\\\\/rXjY9SIszTEoR82zoWNTDKAnNDKphun\\\\\\\/O0MXd3FyUqX6H42HgqQ2i5SCuHi8ZH2beW488oWxBhxmQ0bS3IXhZSLDp9ToXAootbxuvb3Ux2G+RhwNAmpJODHwGRiOtRAF3wVIuy4BiycebRBiEXhDbDEET2VWKRIT4AILjVgg4oKL2m9D0PmddTB+4uiKBdFrk4LuROk5lBpyGyS6bt8XG7kt8yHpHI+UNKHLjSaojH56QFbF1ZdOLshMxVgaPjYg3yOdCiyBzHza5wwP4spGyHtucnLH\\\\\\\/gfOtbrDI6VOCUEDTrv2mKYXWdXDAwRfJcsxoPDYdUeSQr+D7RdbpKn3r1p8Eimm3RULd8h5jrsGwwbz5YPcBQ\\\\\\\/clydoHUBtt8dIIeflNZpPqwH5T5G5x0HZyKBr3wEqbRbyoTKmJBVfz4OdcpMDXpJRq4kguw7EDphs6uLiKFuMXmRJ+kCFsJV8Xx92NC6t6bHu2o5WgFDz\\\\\\\/tVBJndRTT8shwcY83Dp+gJrnT7ZeeszxVeWLAhbXME1e\\\\\\\/nbC3+oTVhshegFZ7fjDF34o8kXZR72qCKulq9wmqoHfGP7QpwSKOAjrVVU+G4C2PemF62iUeBQgAuqgGKt+2n5OPEvvNnOW2pjIinxwVzB1Vfpz3mvvkkzupT+3HYG+6z\\\\\\\/hhWf8zlD5hbHBFIHKNdt4cF4BT3c3CLe85B6BmWxbk4e8mfjtISGsagTN9nMaIsqTZfXaomCHKBYNLBfF9Xy\\\\\\\/qf+YOwLbRODLHuUHC\\\\\\\/ToCj9xl0va6SKwwTPdc6nXrASuFDUisl75AapsZ7BD\\\\\\\/RrzzMMNPxAvudEa0sD10ZED6sLfglqWjW3fRG\\\\\\\/qXuq\\\\\\\/D5KqKpc1XJNYa3pSilbkBfs7BxHp7Ggpuo01Pl+5bWZEJpptuXrvxFjOjuTYn497aacIX1doWyvrg+QHlFe4v2f2zJgOPFCgxWJ0qq6OycvB1se8tbObF+M\\\\\\\/BXA3Z+KkOyDLWan0\\\\\\\/cSeN+fgYVqv7HAa9eoGPn9v12rbKymuLYNUqOt\\\\\\\/YkFBZkp4rGSTuM1tTc2d83dDFouOVIZwobNARYqyYm0QQ1uoVh1P7e4B6wiumErPnCYVcrOk2QkPeSUf1m7ZYVaMQXjIeXyr9jyudcsDca5EEPTclBtejwWRHmZRnTWIDzfhrPqryOtqcDZ04r+lhoc99ee7hqZAa6Ckvda5idvpZT7ilaT6fk6\\\\\\\/fGx1uSLAKC1VUk2vcLZZ2OGUgi0\\\\\\\/qgUgAUmiYSFNa7\\\\\\\/W\\\\\\\/AUodIIMvOmrX8Sab6f44DclOsvHeG+4zEQmPOHJooX2f39\\\\\\\/R2aKl\\\\\\\/FwN4Mm7Hqn+RIBAuiHXoXtq\\\\\\\/GUY+acgKO9O9SXRrJE4Opwm15uVl5Hth3p5g+bFOOXiJ0kt+EFEKqJW6Q8gdWS6lYyZ8vRvRDP20lJqxOn6zTN50Aqtv8Ej4ZB7GcSx9G2iSCXWbLonJak15roUIJ581uNTdBzw4yrtInqLtfRtppeuT+Am2UQ8mpz+3oqXaap8XI4zFGbybO8IKtd1NLA+ebS0kn6lwUV7HElRh3WhoscZ5NBRf78\\\\\\\/HL4xPrXAiqfUWudeMYk5NsS3dZmxl82SarvwmGkbAiSCm\\\\\\\/nyvkAj5K2a\\\\\\\/xeVHuc7kqcpnzjj+X92Ow0C+L5ZwDvpcNbI8TLi3DJxQAFJMpQr1zV6f9hXdmpK8iWTPhvAITm6VY4WwlhV+YokdP0Pv5WCczQJKlWpnkV3YlBz26L9FRJv0sk2ba0C9u4EOZZKlIFlwoqI+OrKYtKkhEkV\\\\\\\/sU2aMcR7g3uLZ\\\\\\\/v+yrBWvJvI8E8c++i12rZ1DtCKj4OHA2UHIUK\\\\\\\/\\\\\\\/A8GL5kk7E3dI5Me4BjEWlEHILNimPbZpny\\\\\\\/eWPX\\\\\\\/n3nilNr0M0nSSQV6zXDGr\\\\\\\/loqpIiMKPJD6stfjeFMY\\\\\\\/VWY2rJY1wa1cQcOnBpPPItdzPjZKKYGSP22bSrrGz5GicTObDealpKqmdmq49\\\\\\\/XbaIAOohR3NJp+0\\\\\\\/uPGoYqvzt0CTqJTDQowwhbdYP1Py1wj\\\\\\\/NNjRKgujcV8+51NcqLSFv2IJMqLuZ\\\\\\\/GGDL13sBP1xg+zVeeEAL5DyRFxl+xjd3UIHf5pVNMyRWhkCy1vqwSOKLriLVKFknPNMcViwpG1nMc2Kj3DheS9Zv2z7Q5CWHk\\\\\\\/omFOy3mGPZz55XtdzFAEs\\\\\\\/R\\\\\\\/MNqYjpyI8IiH82kT6iq+4d5CghQXqMQJzyHxZKeMa9pVq7SCYJ44A6luE5565mOcy+tpc6oavMQK8tWxd3hN8UsqnyitC1U3hJxoTdppxRum2\\\\\\\/D7kowUFJKL0uNFrnG\\\\\\\/e2mbYz5pkZfAX2kCuyDmNpzp4BHHsFx2zmM4B+xV5tfYcsKyCYW0fZxeZ5j9GFe\\\\\\\/7EZOcsZPefK21U\\\\\\\/lga21QV3Pggmqanivlfb7DlmtBFFcxZfHU1oDUaf5sImlwajA1OqKkGm0Qsj3DU5GmrwJKVbfzYx\\\\\\\/NHVi2Q+oh7TBJBGYm84liyHkGgRcUDueuwllJY5EnI+bJOj27qKM55rou0RXjqF1wwyFruAhvjr1Uav\\\\\\\/PZB4soK5QYaWVDB\\\\\\\/X+5SsUpqlviL69i5Fg00\\\\\\\/UpFK256fzMYiMZbCNCsxrHv8CLnknvet6fymGHpnJviuVP9VIhz6eF6wqfwENOMyUQXXYxdHnIn4pskuzhpSBOkTFnBby8e0oyrGqmnkIN9I8ZqDSJ9rlzq5OeJ3mSMl9cR3Kyy5wrazg5SwWPmZ3oV+8M3bSxn8xjDkNKH\\\\\\\/qNnbV+8M8kja5wMev7S3i5weeuI1sYUV3ikHcOcZIyMN67kqb0gtbhlp34rNcCczvPMirCSS0hylasSa46hXUVGlugmU1TzY95lBuBP5MUTM6hzhk1yQCQbytHWZpW0mBStoCRD7LqSgLvpMmhM6W69mWtTx0BT5HRm07Rg\\\\\\\/3UESBlq5b\\\\\\\/ubaC\\\\\\\/JnzLYOjKJEhnFX8SuuTgkJcSQL1i2LNuol09Ui\\\\\\\/wb0N6nToRQH0OCEglQLPs7ZorgGGNlYhVQfCqaWZDp4s6VLVZCBIeMTrKKVRQTeXran27YjuqHwk1cscbpJ5FFa8PwoAXI1xZ2ldqvb7tV2X2bnov\\\\\\\/uv7Kq0rmPgQaf+jO9nhOq2u3Kb4tVRKxkkKa1lc\\\\\\\/MAF2QP0FCJBEjMtjqW\\\\\\\/V1YUFEsABXTSx4TFLU1iUb6ujJFQDE919baQu1xO5C8Wi3S6n0\\\\\\\/ZpjrapzVR\\\\\\\/oMj2TAbjTcmUb02Zdfq1WRjR4y8fyf13dawaNaUx21N+2bea4r\\\\\\\/R6dLw9kpAe\\\\\\\/vvYDtHp\\\\\\\/Fd4DqVPi9nHuq3nvJPd3+uZC\\\\\\\/omU8JkD65SQ6NCPVYHx\\\\\\\/NMBYD7vMD+OGdg\\\\\\\/VUVUctxyXxMk4S5TUPPNsGFtshkn64npxhUEzZfEyc\\\\\\\/j8JHuFCs6opjfFttjptkcvuY5G6NGvwqU5OpGxPZgU1+bM8NZ+eeo57AX4Iz+fNn+jqMubyhFgDUtFM42waLs4EItkl6nlWzwGrJ\\\\\\\/6+wWsIzB\\\\\\\/7aEfKaQff0DD+VkOUgJuP3Lc8KJX99nD7M7670bcNLWV1\\\\\\\/dgJRx8o5dmfdtpYxZzyefCDcj5UHFBxpLkwsyS4xgz4k52ftC7UYRyBfHi3tuFfyhaMLu5sH18ZZTXfATa0u9gJTb6\\\\\\\/yfIuzAth9lusqqpxf0B0XnlNHPAxLqcdRXEoKpW9E92hty8BzVg\\\\\\\/ddN6+iTPzFYm62wA\\\\\\\/eQTQcMpoW9qrkSeTZeEPmKFIUaCQMUdSppZaJYIb9VgIvm2\\\\\\\/2jrLog7gmBY9Prs2tkALAZ8bY5oWfIehAafwQtGdQzQWsgTxpfFRwUslhy\\\\\\\/StG8rlxyfXx83jqxZsXaetE+TmWJugI6zkxkAuAJzn6g8fMaoZw8DIQFpCYQrgtSRWRkMOxatHpNGcpauIr1FLdifMPHL\\\\\\\/XxcSTQ6qkMF9Ep7KRBSg2Bl8FuiW8LZ0SC1Vgc5n6eOQtu7kNdU4QglhOjjQE3k+MuFbJdALKe9F\\\\\\\/7+qO751pmJwCXOY6\\\\\\\/QLC21z1XhZuooonwKbrXjUq3MDWJGkLSPJX7SX4ygPYtK9sni8F6X6UZT4B8DnAdV+CI\\\\\\\/hORoFSb3LuqBtei7soDLI1ufNSD6Rfe8cDtechKV+IVX6jOI9Ahd3m3ZlElwZfNSm+otGLk0K\\\\\\\/6LIZGimevtEPs9rxQZBL4hCRmYLTFBdTBi7cI43IoNV69zNvmGgG2NKzBYbTcYfTJyiG904cHcYc3ezv398FXDHxd34R\\\\\\\/iLbyaMDq3qYlVPo7iTPMmkYJWzZy1PN0+qMW+IMifLEyKnJNMUvJ9RlmbZSaNyMqcwmXylhek9C9WXMJqgU7ajatFebWIIDPUiHaW1z8B1FgpUMojv5Rd6HkUMrkYtTQdv\\\\\\\/NtQrCQAa7WorxU4hvH\\\\\\\/A4zNHY+r4QoUfBTlMbve0xeBNmDibXjcLJ8MwjXa0lULmEPzhpg+tQuCvs\\\\\\\/SjnDUnggQ6SRpP+jmo\\\\\\\/qWVtUNyUgu5LSyEW2dIQLMiC\\\\\\\/XGz2kG5rGXXCsUGgVK2m2PeaWf0Gl285OYF0QJ65aekNCoi4tGtaPyA9wT2CTC3DIgepqu8o2AMyaqFVHYqo+OP5GaIl76bonnYzLnkJFpUmnKRbnB2mkjJxM6Pb1V5Uvun7RGeXIfkgJSO\\\\\\\/bfk8jTI\\\\\\\/qpWiAfuaQ99C3NfCDJcRMZDenQiMytnzed1rbR5tcAHOyKdEv3WirzC5V43bh1Cvps\\\\\\\/lHyg+vvz7XyrBHyAbsiHrmV6arblqKNF+Dhtgl3xiO+i\\\\\\\/S\\\\\\\/IX4IipHbfTbJdhptWGVqltv9c22ujYuOsPWDDeWdeXzq3jyvyKTMllunON8NU0WgT2ItCQQjxkasDcXAYqiHOfsOUxZnseUTVhrDVrhRDqffS8EcqOUEvbiRw6sd51H8JCgEbXBEisjAiTsGiD8J6hHiXdmZLwGO2GZ5aVbUw1WOuOo\\\\\\\/MMk3BPc1HnPCcME3JeX17pgJbK4RoIXk3yuKF4YwbZ3Jbq53sAlL2vHECRHESjIu5Tcr\\\\\\\/cE8Xg1xC22e2yGRFpufqe3KhFxKlnjEo4X7OftBNsZVtyXIHTv5Q9gEpwM0izrnUuQ5EWjwWUjjub8jDqDiEBfsrfTbn2Q2\\\\\\\/gm\\\\\\\/CEPhUKdVVbri\\\\\\\/bAal\\\\\\\/uNQ0K7icx+9MQktRB0mKZ2yNXLMILzZG5ZmZtBxwjkxc3f5GatE6DYFxG4+K9KE2aNdRpUEVJHaIuJxiAhVKBRJ3AV1qfbkuziuhnU4vdjSKTMFq9HDqaGNhVdOUmp\\\\\\\/hxpNbRzauyTm5eBjppA7j0hOWrloxcUWeRGmOR0YX6Mq26FozyKUEb3mmj\\\\\\\/BYh2I0gaX+jU0\\\\\\\/JPX6LEK7eB6wrHDfp942l7yDS0cOgY+gxH7kiqqwq9W0c33Q\\\\\\\/6Jyezgb+BeV5QK8W7dAHVzrZgtVFZadWutW28jJ2JPYisaE5J5C2zNpUV0CNEDaHiX+1cyy\\\\\\\/\\\\\\\/ej6cm0ajAJg7XiW8X\\\\\\\/DWyqw3grC5aHh\\\\\\\/KaZe7F5dGbUOA3FfZ1NvGpaLy1q0P1911YrPGTbtCwxswBpMJfIEfXwi54CuKuyacl1f0jLy85iZt07DlLssAQAkq2Umlb3duYRFk8rqhJDwW4+2otHbNSJC1jOT5Kxzq5LCiewXm8qzvDaejPOFWNNDxqN9vn2y\\\\\\\/sI5RbTfbawceYGicHh5uEqT\\\\\\\/e18HMPFQw+ixBPYU34vBbFonvsFxtG0IlyUKpWALp3+LRcX47PrB\\\\\\\/v071rDWLWahfI94C3aEmd6t6U+2vwubP6V457BDVnsWjq0z9fIbBNm9Gp0bMHQdNqoZVbsv6wGRl6ew5ZLDPcQLLU1qe7cV0KPjABmwebuAgrf+uOlC2gxS1W\\\\\\\/nSFVW2n4N6WV9cnPC8Jw4GUlTVCSfgSq1DUPqdoI4E3uNoVhlqKFYPrwmtED0If6Mqrd5RZFuSL\\\\\\\/muOSpjw3cCAokqg9fjx0ZaX8VD4P2rUqIQbnn0PM9auf4NJMIMIGZRtsywqoNTOrdPEwS\\\\\\\/OQOFr21RSpp23Eg\\\\\\\/cki\\\\\\\/VGNde7CxmNrDqyBJSERUPcPqUIq34M\\\\\\\/szslQwP\\\\\\\/46jJol4U\\\\\\\/kZF5kMIdTuS3jbDod4YxFRTTAWE\\\\\\\/5BpNH6ylYEnPMZqJ4JrhVOWLUqndqRMHD5Yn4PSIcDBaSXP35WQbX\\\\\\\/cmkOR2kekw9twOxWrqnYKFtTSmm\\\\\\\/3HPu\\\\\\\/34Asxj0L3w2ZvaYVrOA\\\\\\\/zSfJSdJObpofFZVoIApeV1pJwyw8A95+eE4f58Bjvo+x7mKXRa3de46LKTcwqeDbOdAbXUth0xb6caPw99sMKNtoMH8SbmpQCsIdxOKA5WrgCyQGJ8T5sfht0gZdTO5y2sxoIrhkgLE2PuqzYCkxcNwt9tI\\\\\\\/J417TtoC4Du8r2wDCvK5QnjyPnnfjACnf0jTb3boj9eY0tIyjUMKEH\\\\\\\/SCUj6IGlzToCzPUxRIDyF69ZmxkyWgLAm9f6MdFoGlybz2FDbzX0Ake2xXjQsI97Ckkej4y2At2vKz\\\\\\\/XEVfnoGrrre0mxSreKgPzQSPTT9pd\\\\\\\/krI3x4lMQsmm8s8+3jeQ5y8Xba4pSBwYNQDazz4TvZqAA8Lln5\\\\\\\/7Xs7dHp0fVFR0veKC9AQApxnK86T\\\\\\\/+QgvxC19wqqOKg3CasHY5HhAN5IjdajlfVAUlvFtJK7Og9GoZ9T0Xot25U9r9d3kF7Ecqo2DfRpjBDt8r2WPXwVxb+MdSeTns0HA6iidrifJ51bmIUKWk9XmJdVgLIE9jeArY62xkAAZnXggqFfIVWdB4MPvgEhza5l71ykxfu5vCiy0X+mzq4NqUHBJWkbNPomPCjJLIn1wUIu8Qd6jio+mxjJ\\\\\\\/EFv3f0OUqtUfzb0SQ98TzsD8FZTnY687s+TuqHw7P2SDfIe+P\\\\\\\/tu0AWxp9vbPlLutVt3ceQLdpjOHY1+zmPHl2VuD4yj9hCFWwNklXeXIcOazJD6Y5yg8e1eSmgQr6L7SAKjLhIFVus3rxrbN7hJED9VdrccjIGnGWwb7eju70lKrUO7HkRw0ttc9yvACA2hFZMq8AO2Netx4ZCgsRILZRTA0RiTfACK8FA5re0mUYNzw0Y4K17kcPy3tyaB6opVgIBGzW6A0L0PdbjlejlyUIQN0MzRmfr1Oh3dRYJOhSUOkgLQ\\\\\\\/d4Svo\\\\\\\/67l3E8Va+zta8aFjx4nSZ9sjRzpO\\\\\\\/dxowRZoJGHxSwrvMmt39\\\\\\\/qO2dnlWjXhcWs56HsNb6XfMmHRMDEhE79JVm8yiWER5zqwdyZZQ4bo68ba4i\\\\\\\/bBlMtsjkzPOLnVptTd8Gk34OMlOEBbTXBHj3lnA5D2w4GRUe5vWrSQu9gZsgNgloX7OGWw7uBgICHboVjq\\\\\\\/9CuZFaNNA08yY\\\\\\\/i16\\\\\\\/szYNhtMyCPn8AFjJ9Y8ggSb5ZKTEipjFa8P3OJnErtGxAgg9kJZdPeC\\\\\\\/nTCz\\\\\\\/uTUtSY8BDz+1oxo\\\\\\\/Jc8lwM3kb7s84iImsw28j2WMXOLnt7+JQoTMUczw1VvzkTY2An3FW5aFDjj59hC5yeIBuWElB\\\\\\\/nQfaGaPcLMhdtdr1JgU4oDgWNrX98xiuWOTySs74LA1tJflEaW8DyaxIjDBEsaGWevfK5t2cv76SrB0e1sXlRqVje3judT79VH3GRinQjJLokIMnXzqtuMzok5M7zGuINU9+6AfVWD\\\\\\\/YJnd+ao768rBTeBQzo0u+N0Iqxze+MLvTS1yl1yUj2dzvtfMLaE0\\\\\\\/65lZ9T5wSQ0cCJVs6fZfNaBcgFCAVQTlLPqMjXc6Z5XOwiE2gK3oYsVkAU1tSMOgnx77io1EJtaviHKZ2z9948mvjwwGeD8Q4VgbH5tYTp7x9gpc7aR\\\\\\\/7F5lqNLBdQVdY4vVkZMN6tHCr7q0pskdNVR2iot6fW8pL0AxXRToZGSfcjhwqknEsTJnN5gxC0OGTmyMQtKMWbKrcCbzPqRZsMzHDOUXJvmu13VagS06ZYhA7eCpy4A1Y9c\\\\\\\/m804WggR1N1jj4TvDEYKm0K7aGiKq\\\\\\\/X9SOXbLwhH622uCUarLvOkeF\\\\\\\/kpwmE5crp2Lzfhf8gGCgtfpMzo54RN2PKa\\\\\\\/FLjCSYlefq2NaCTeekvh7GwEoInBebZ2ugouOC\\\\\\\/zoHy3YHaR46ADPBqJ8vSTwLkPuclTqjXAI6C+zWFt+KqyQ2mgCscYZw8K8cvi4snKaBB7YdjDUZ4trTOMgUaDkGWeO1HHQPt3KEj9rGH55RLhVrtEAdZXE0rqKG18pC0iKQilGh7OXBvANriVg9HgMU8fsEJbUX7eNWCL10pVReB6uE5RpRnr6PsXI1m73Fl5vRS5vMmGp1HQCW5mNTFZxrGcreOl3P6TSrDyhbXn5rMBDdPYVYe8J9cN05lFCO83hbFCXeNSRop6A6CMoLHzVvuBlvoroUaAMhPqK\\\\\\\/twfPjK4cZepCFJVhLZJAE5IJP5Hf9BF7nUS0Z6laf4aycPGqz7J2ZOa3G6kMbxPljC0GFzg4D5tNIyUf4ggd3yYjmAdo\\\\\\\/aune0rakagY4eos1D8GcmLCH8BwXbcCGkPcb6eYJQdtdi1YZ7\\\\\\\/ORRdmmUvNfG4czqJD6BYYujBq1uBvzIJc2wRtum2CYWXDwk91\\\\\\\/Ygf8V25mrPIREUeVl4lLf7JLrAV50DIMAY4\\\\\\\/PHs2z2tTYgzjwrydYCKW311Nu4ATZVQGQWNfRyk\\\\\\\/ois40KkrqKBCSRxVM\\\\\\\/6ri6r77fLfbzpEhY3Pi32P9m+xnrcpD3LXN4ZuoRCkWvQvUHQ26Vnr+ceDDEUNyVCXr4+aMZP1WWfjeWszrFRErWiFyJlX4JE5bhVZ2MXI3ezoQNjlSW4VHoyfaR3SLRDxMMOONhFV3vpgWDBVSZK7T4cEKoCxC6nHUBSB1lwpoc4+ZJDiXoKBflNjR1hM5vq5kRK2aY2VqkXK\\\\\\\/L6TEsrhitISg6LzaTM978UUlo63j2TU7zrSzYXNKCmSwK\\\\\\\/X5+w3WhdOVICBIm06zzMCQ1tCghyiGffCz\\\\\\\/vBzn8lgdJ2KWem94AZfadt9MO7N+kZfZ8CFRsAncdm\\\\\\\/UEZnAXPjRHcy\\\\\\\/pQ9bdK9i2+X9Hq88FjonKCKEhIhKQV7RmMSZPZxRKqkHaqEpdk6S8NqG+YQGwxPHbJA9WQeAA1ZtxFFnPEjRmZ6GBnt31sqq8HgGeVRfSkOiKafp7kgVD6MWA55xs+E7U\\\\\\\/XtS+Fc5nnxcjhwe48WHN21VPmYo3Q3lBpb4CIeBECh+\\\\\\\/Q1ka9A+qVdkC0BRSA7yI5QVaIvahKF6\\\\\\\/SJjyupmEF1k6uhvBqCKABDZ+UHz0\\\\\\\/5bv7slEtR0gf4bMGNRjAzpCDv3mpAxcDF6Olo8xY3dUetCNYUV8ntsIiNemQQFF2Mua9NLK3YEGBlwKsMFoxBXr7OHtKmWZLPAwUw24FtdDyYVCnxB1k2MTrDcIBcBL7\\\\\\\/RpyB0qh+Mv8y\\\\\\\/mmyaJsFHeRA6WjAGFezo0OYn1gN5bPdIthKXSUXVhA66czb4sQdxBILZCr\\\\\\\/0cLGrRNIT1GI06jlptjktGHtA++IEuI2mM+0CBk9\\\\\\\/LxQnOz5gd2G4cwbZj7B2HYZJlT3UEoGanWbrJDNVpgpxwNLwLPTLIn4YTOr42sP2KMaU5yvVvRyBEbNBpecYYSl7xQwj\\\\\\\/YPpWrxBJzP+IGvQPqIr2WnWzYsWa+L4wL23KDvt8bTfGE4I9I\\\\\\\/syotQfaje\\\\\\\/iqHnKKBfMaSs\\\\\\\/y3oUkOBOyr9OsWXEa85Ef0D36ImbudKxzgYEOioc7BcWPnyBCcyjr+\\\\\\\/Drv6SVHwXMpliWzDAIQhOD6ysJmcGv7BPJEYDwozgMBGTL\\\\\\\/piubNB7qshF\\\\\\\/gNCLhB4pBXBbZkfOP3uMZpjS3hr2bM57Ffy34izf9BP1Yzbx7JLIJNrBgFYtyf6RFdFJpQicyiNEvDvf2ItzU1aoHaXdPUlJ+m0Uukrc23MdkBzIyrCc+Kl3raUpjm8P5QHlrCgC\\\\\\\/aiOQzjluz+2gp\\\\\\\/Am2OgyrjhurN5DBvPemtGwdvQWQYH6khRtLPdrO5N7MlXnmFXxFbHROWv2VUSuLVsv4cTb92tlKlUAscKatrf1S47kD76vMb1JFGxbfiJ5BDlJgiTUjNNhcrFwljrMdJrFRBX4nPsErWdbz8nlS\\\\\\\/ePADstH1Slcp2m5qXbLkt6bK6DcZzjXFNCkVJhxXauk\\\\\\\/k0KQ1F6+4allLPXCqPwm4Aei6fLzqt6KOInHpvqbRqn+12+U3zpeRwj4VPvm5fkI62Gy1sFLADL+sb3NB9J6EixcZ9XMbvD9GU9EAih+syn\\\\\\\/d\\\\\\\/esdJBV7fg7wraqt0v5Qb1t6AusfqDQa5J\\\\\\\/rfJwhi9Ehwhb+R5zC1pKGZeKoC8gyvxr6RdFLPqJAlW\\\\\\\/T5qpWbdrhxg7\\\\\\\/87w1hUXHeHb34ZZzMkf+d9G6io\\\\\\\/ZwEogS6paDKU5kotPbs6aGCUaf6c87v2KXIt2kDq5t1PYQ\\\\\\\/qnhqjaX+Pj5zLVJaTKRSvoTQWrLMQzJkbjxNn\\\\\\\/hUfd0bQkapu9+djKxHKdoLLf5fHLCdg6lTFjdu3TU9lZAE4f2+8Bx+DOOctVV3cHViQ8uikQ2EOM9LoL6syIEMFEGZQUS8ZKfHQtsWS\\\\\\\/vMg6V4+ATFLPLuX7i2uUMxCQQr5O66g45OY96I7uKVCjet633Sh6y8gb6ibMaNK3pO07y4WtZJzEvZjD51qkhuR9XTBQmP+6K0aUjEHQS8VhkM21\\\\\\\/Hi8\\\\\\\/H+DaZ\\\\\\\/zOb5HVHcUNil1iGMD6LBjMri\\\\\\\/pBN9RYiV+FZ7Wavn1s1eScj4AvVRAUjiJPklz\\\\\\\/Xo5SnGLha20Rs8ciM3eS5Vw7xs7olH1AarEdEEjiZplB\\\\\\\/PuoWab64JWc\\\\\\\/KudXN9ojDWvLNOmujir80+da4U55TtrPU2xfm3dD705X7p+I\\\\\\\/kEroEFCbHBnooonGPJbksPRmlU4qjrbsNKr560AlGXp9hOv0dS7yEv5MZYsrmGHVfqk8TCvcdr4WloGmLlT6CvH69GwJ8pvL2nY9j3NOHJMn1HEYgv03EklX8+T0dg0EYy293zl\\\\\\\/aSOzxD9Gl1hqPhSAZtH\\\\\\\/I4vNgLHF2M3UWpmLF4aB6A35fQbVTv9c7HBM4Yz305smveD\\\\\\\/CbVrCzOR0bSF\\\\\\\/K0KbpmEUjEk4qEOQO6SeUTxLvXVqV5iJxI4aWkCrpIpXN4NL0gOu7ZGtIlmOmP\\\\\\\/Q1+XEDmaOTIMKFPzBgR6lTNoCAwrsUGbFxQk32vh0tvlqUxYsn\\\\\\\/5D8bEahbMxU3KVtK6xURdqmIr6Eras1RwRHPVba5viH\\\\\\\/Ln4QrVO0EOVusD\\\\\\\/0HEUv2fjBW9UZHl4FexQxKlKjcxzFSdPfwAF2PqGiXrJhfpuEntymvbRmhYtk72ELLduQuyvSOhqZ\\\\\\\/j3sE4RsaUaR6rx6t74as2pVlA6jv7Witd+s4ggs0FW8WOKTvAXY6HpGI0Y8l00gmXkS+UB85mEWoKpXoOu\\\\\\\/YKxemPN1lTEKiO+nHc6NoRoQH9Kl1SmAZZog9UfEzoceIeC1CHuX0+IJu53sPXOEisQffTq8OO4e+U1gHhGsoEz095lH5j+QeL5uRdWIkbMDmKZDgcncyp8bm3l7NvvJc9bNwiLn6PXMxSCpmRkmt5qifZD2KvGFTUKn1fAWoTUgyCcYFTpowIwe6lFnUkLuMml5Qnn1mryP76mHL5Mb6ELZl4WeVCZ2\\\\\\\/8xSgdoBQj6B0Mh8li1CzeK\\\\\\\/tGB9K5aTYwxzLwf13Tyk+v+CPqTK0aYg7VOtMkgxa0hj153\\\\\\\/4MR8RNw16QFJHzn3ANW7hq4ySJmZc4pJ1Wi7H5tTtJMvv8FQnsSIJQWK\\\\\\\/mBt+AquSKc1oFTqdInt6KJziTnX\\\\\\\/KceRbg2FZBACPS7wOMaw68B9PVGMbG66a2fOfCgWOrlie+MURkXjM+P1SC3M2KEFZVMPS7BGRoSUhOChOJ0SpbHycmSx7OBRG1ZwUrOutmwFJAHsVNIWvUe7qTKYZU+HIJON83+nhSlmFM+QlAtMvJwLQQaotNPpmq47a41swxP5\\\\\\\/5e8TPohCvVCxW7Sqtv1n7HIQj6I8axrSpVi9wozstzS9j3ZlHpFR3BYE0FaMGwzWHKHmjQe3ga+WdR5IUiWU5aL9mxig1MIAleVeoNwdyn82E6THHj89ujQ2Efjp0Yi\\\\\\\/PSyRAOjGmUHGAsI6oOsDt+ho8KCrtd2he1K+ZXlWe8gU23mLvJjSeT9LFrahJuK+dFaZwdf6t0wjt9pmU1WEKsUf5zQ9L6BtCoBUIPEiYIVGM0929cI18hGXEV50ywySdectfUQwjVHv0XYRJsCtV6gy1hw4tnTLbUStcllOuyBPSGf3kk7ndA1jXLxkYDZYUPhZinv8hKjCLvIWZZmDBixadpvvtoYJOcUdBMdPxFYfk83S8c9mbm925\\\\\\\/CFeLaoq5R2fgb0bn5m81J9D+IJCVDzj840sq4Nu3CVnhKbHXoauB1J0WY7p\\\\\\\/kHUn7IvqUgatPhlKg9HNSmQzW+xCnrnkf6lRblu5ZPPmJLwaqRWWLUqs8\\\\\\\/qba4uMI6ZLPySl4x9oKR8tx90ks4vJTtfy6fDM1XXTFT8mKazJ5vrgWOZj2ydj4aHqMtouB8t3K4DaDhfFXTM4NOpNKXTAsUpp6bRtbC3cERdMliA9dtb+ZWTBK3qmp77MbbM6kN12OFRVGf20WSBEmS+7pX+UzT7yUNenBRwbWJTwzb5MuamaAxTxEwip0o9zse66AJH\\\\\\\/NWz\\\\\\\/SAmgmN\\\\\\\/kJFjni6esFhPrD1jgrXGNVgiUIL1lYIEqXIH3MP9LA9otSZzeaqkpqCWvYbeyS9c5YU4hlX8wRn9zxawc7stYvxFUBz5GCjikR9cekLWDW3AKyIu1dGxXM9NnWmuW5wo3EmQ6LedK8kH\\\\\\\/uVN1mIV79QnH9RrDeZZb08Toicd6zDlzJ7PoP772q4IV1zXvm015N+dCMFg6zfEskVhJO6MgveXatGC7+g9SVjzouB\\\\\\\/dY+ZycTODfdxgPLHjqbhJ7PH9Ttc74UJorkfKnkNkUGTMD9zpGHrb3jYBWnz0MKmQ71USaDJ1nax4IFMGMV88euxv3zqfYj3QU0NNB1\\\\\\\/Pt\\\\\\\/aeLH59XDjRYwHxFhzNmq7hgNjLLtMjoUtHtUHiItqyndvKyxhuP\\\\\\\/Y1J70sU9+u9xGqs351gV73p7UQ2+Cmw3bnGNqCkAkt6ORUyr+TZ8LC00W8DKuW4kNEvg4EHHNxdR3oG6ff7USeezX14mOt0ov68\\\\\\\/\\\\\\\/EdlZWRy4CEbJEakSvTdEEdlB6cmtz\\\\\\\/0GhIYSvIfx2wEVWxVliIsGCiPCVz57rsPRsEb6RiWMeufXatL0VwakP+lDhNNJFAKR1v67oZZN1JGIYqVari0rQKPWPn6\\\\\\\/xppSvs5lEJp8MNrhCPuD48u93t4QfgMyJri\\\\\\\/4qbPkQkC1Xevqy\\\\\\\/AoVmeDAaVuF0CJkPR\\\\\\\/MGZes2b5r5m\\\\\\\/v2+0eWcZW49PV41nPrVHUyvMYZYN5WXIffRKybJpcfukmJ3GEU6zslDIzPLidLYwICwp7Q3ygRxkGabFO5YOnjlK723iif86NUIY6UvYt0olBTVBrOFp+vUQq0n2\\\\\\\/s+S4LPUR7LdKcdImK\\\\\\\/Q3oZVmEmAAlc1wPBSOyWlZqqQn2rZq7rWKkJR1BR4rbqWwHbM3YUbi8+Vn8qXMW\\\\\\\/ueX0R12SVFDlojZpDztzQ8rCIVf3frHDu2f5rNpgVLTwRDeh\\\\\\\/bGbHA7dPYEWROvCC1AS4ORAZ79r1a\\\\\\\/MXum1ybZQTbHJ+2+VLjfx6AprSuRdUYHG5aLrPgWeB\\\\\\\/Z5RJ53J2vUc4YuY9hXrF\\\\\\\/L0GFWClWkW4gEq8N49GaxGUX66gU3m8k4KGISYoYpUnZTzCh1QAypi50e7YvkkgwzSofOIa9X4asFbGRDFpk1cQEkQK+qeTWBVyeIQ6MuIwciLGaNuqpymq\\\\\\\/su4\\\\\\\/+Lk8Es0QAtr8jm9kdmA3qxG6xWkB9KzlIbHoCQlc+q2teSjw20DRu6H1rzHz+fyNT7akroFEBFo3djS\\\\\\\/jzXMVmskUQvqhoHHwBy9XwqYW\\\\\\\/2Hqg\\\\\\\/V4ARJLKtbq7vWJopdSJv1P4qbc01zcWq0OV\\\\\\\/IKi4F\\\\\\\/DSyUTogwBJEmLcYDO8dRGZJxjtQm6tmSZsaxaPXfBv4t6Txs0Qx+3lQoB+CUefS8cW4zPWpNAYvKaJVqOjvwftZYV9KKuRJ+RS998zc28ILsc4C1QeZe7YVYhnNQCXEuD4mSoPvETLgwaaA61OZ11NE\\\\\\\/2vy1qr9b018Lnd2JMJfagEoQEqwsxWKx1FISQ\\\\\\\/fojTud9BXlv+6sIB96b973yjKmxTDVr0S6U12Uh6Oo4SN4hbM\\\\\\\/wsR\\\\\\\/v51OSInWQfp01tez4qTuG2DHMPRvPJCcUGQsdke30yfqaLCh0vy6OoWq6clvYxopBOlp5gKEGjNQsLun2mo9e06FppVXtBqGnNFvq39OYkdBnUNu3HjwdMn+lgJgfn1\\\\\\\/Ar8Q9XRbh3ZpD8ALAieLfeVX1TliB7fNcziDDvtRuHHvGriUhm7Ktrho6DXq8apHWfGnEp3JuDAfzOFF+HhPFE61sjuUtyDhCnnyeS\\\\\\\/tBxqFj6PyFIkFywgOUtSQw\\\\\\\/cMXHrtsLQnEJjNki4kDysrW1XEd66\\\\\\\/nD6WaHqW\\\\\\\/7N3+YkVW9BWfZeYKP\\\\\\\/PcMKF8XwtqZssnLz0ZHuTXAcrgFnWNCqevqRoFEKGC7gjTOXKSR1XWJFfeflwK107Ay\\\\\\\/yLgt3YK59hK1798dkaop6P6cNMRY+50IIS6JDSk5QPkTKdRrsC+Dif6OOQnTKT4mM+NbCwHQ19pvAybShVb7ad9e7wQjiQaRGp8WbeiGCD4nFFilZuBL3ibX\\\\\\\/Gq06RpO+hVs2qsfLUION5Dpj6I7iOPJLbg0+8yYrjp9btD\\\\\\\/UPj37xZNhVM+IecYncPHSBp7zAdjkle3OPqGJvmoes2j3P5E5KaLnL7HoiKP16EKcWHIEt93bJKNhn5LqpFpt7bDIytF1Pnie\\\\\\\/fbKPAEADDw6aKxJPg1EkYQc3TYVIj3wv+SjfTlHm50AG\\\\\\\/xC9e6pSCa8\\\\\\\/ow4WI5KpwD5qe54DbfBXm89ulZe0TySCy4o5IV5uHHdnnSZavmXvwJpJZSCTonIRRY6HPvN0r3N+bxB0wQThBwkrx72sKkMTxzsF5biMEG4ZObUr10elVRCT9ADZZ3leoRBFgtncViDzDXuIWSaDyDA6wJHIIJHE+iVWuXhP1m98p1RbAJx6mzqcq4I3JKAipMbmSR1Cl1En0qwA0B3YAM2N0QeFweaMek7BP62EjYddY94HccZqrGjGznjVRRyRHhGcPTE7aHmkX\\\\\\\/tihm0FzdPTG0hJMI59LB8vvr6yQOxJWTtfs47NiWhXzsaPGBYXrTHqZPl+kCPF53pcMwNqG72wm5U36IXAMjenUCXr9Ka46K64Vr38\\\\\\\/\\\\\\\/JSYHh8gw3fxbY5ZqLrdHn+DXmi5AB9VA6xc+9TXXSpyFtidtMjBVBjUx3Rn+yaZYpfQwVW8T6WBeBQqciZlViT\\\\\\\/cC5X+aqcgYb3XpR3vlQQEM\\\\\\\/vx3vQEjU1k11PRQ5hVUv69LPM+c7RUIYitKgiXqCrAYcV+uA2wqWP7KZZR0M9p9gVEn5gOtft6UmUK4Pfzml5xcvW+++caOYrK79hPm\\\\\\\/ggpfndRwKpDf+5jsJ1rPVWPA5jKOWxkK7x9sS0tkKzWg9fP9Ab6F0C6X3FWKfLse5Kau9y58T404lT4ggQ\\\\\\\/YzpCOlGrugY88A7Fk+45bSQUQXUtSnmU+8kFEkttu4CFLljVa0HXz4LwcxeUtlGD02dfNz1ajRqfk\\\\\\\/yzRYesarcOKNSUvGcIP5LXmNAK9pEeCkgXGF+0LDEo9b+PYPhpmEIyIXumnABQGWaoC6lEaJVKfTxwoXgI6F1tS12tDG4L7b5z4sosqCuKNabErg\\\\\\\/fLdJIEJ0MKWUW4JyxHNlTEh+GQg9XLNc5lGzhg\\\\\\\/AQrPbk29EEMxV0gAD1Aje\\\\\\\/H26RlzQKcDNMao+cPtA3EA1\\\\\\\/DqqmZb9cqhYEAuL5VZnitDFGi4UEhbT13rd\\\\\\\/o\\\\\\\/LkVbyEk2BloEHlZ1Fjx2Clh8kbTQ2TizifNMDty8QoqeGbE6Gc8oaesCe4py3c4Qtyv1Xj0wtnEsURAb2QVQRe5L7DyRHwloilIm\\\\\\\/2dWXv8C3+5AhBqMRmXijAqcH4itc5ZXFGE6IqVBU4PGs0nlmnq3CQmLB8pDyu8TbRfCzeu2fkiPWK+MlDJlf2qRUlgvVw918GggRE3uiq+slnUZbb5Ou5wo9CdeodmF01QYVr9baGWXESD+oiX1YOcmzTo94LBveHeQClNMgmlbN016qRAXJ7IRZ3C5wx2rFJVG2N19WTNEe1odVwDArIP4K1l97b0pEiRbxDe\\\\\\\/9+G\\\\\\\/\\\\\\\/L+AuSkfZdUbzJelA+1X+shf0MvCef5\\\\\\\/XI9+JtroXg1AZ3ZMffxkvPAyxKDolJR4KITIcabo6Vy6\\\\\\\/MQoGeZftKWU+8Gxh5LRLt3VVImtVaCJlguFj7xgRwhFJVuUnb+7qRRaYLG9vHsrd7SeHGYcq6cjHu\\\\\\\/tY0IQQHD1IPJLCZAaLCt5EFnu+FIvR9X78u93uGsyemq72IzYw5TVpI9VFTvpfCdVweYcqiYGnEG\\\\\\\/HKWsczOuF4M1pkN8f\\\\\\\/DKjZfhtB3Wg8a2UlEC6kYjDiu6dMFwJQdD\\\\\\\/xrBNTUPSdh+aOfAm\\\\\\\/t3znT7ov4yH3wyiLVk+j1NT2y5DGYKXfRFbMNC8eSocOGlmKu9kt9ElAM44+m8Pk6PIDal+TIiij05V02602U4I6Y7tjFK7pCu\\\\\\\/wzqpDEKz6sP58OCy6D7tMl5gSrgURco1CwqINiUBgts4BD2V9jNAAiFeR0VaBoS+nyqx1WedIT3c4vn5OD4mNHjFb1LvJgVxkGWoi2KJlt\\\\\\\/fSb2MBO9Kb5tWHpWU+ZwszgeFnoISV6qmcPf5dyiL8k0Tm\\\\\\\/USQUIvcbwFyzoLzwL54GBoE4y2TyfRn270YaCRAnxfACN6Q3o2mqFmCrNXqIbk0yNtsJkTK5MDJMjkzI3X9ezbe6ZmN3cvVC+EurvaY0w0sleJhEqm8AMXHbAplZdk8i7GyRplOlljLBptoZL9SKQNVm5Aw4wDDbgckqRUrCfHL3azML+PQjOy\\\\\\\/1HGCYx+FX6HCGE6ODbXWkZcDt5\\\\\\\/l7HGkr1O27wIl95Oup4\\\\\\\/nMuxnczRE5HIwKmJuNMDr92m0mjjEi6M3CqcPeAHNUlrtCMs3cn6p2C4cf\\\\\\\/DsL5Ea+F2gFpkGeMb0mFL\\\\\\\/NwJswxfSpTtw+UBFaG4s1t+FT4hBi+Ni\\\\\\\/HcQ0pDID0EMi3knqbVj6I0xCtkQjrv9GT1Fa4lo+9JnSTtiSa2z0z\\\\\\\/fS7I35Xx\\\\\\\/mAvB+wsRdCK7TUKf+wjQJiiKafrqZDaZArkfwdP7CRKOvH6DG8Q+TSNBz3+Q37mA0jezcyLh1yPOmixf01znDfz\\\\\\\/sZt\\\\\\\/zUZXC4P5xXLpTR1kOZMw4G+9Tr1RiO0EQD1y1yoHVwsHTfV7rnp6tnSUUkN5XSmxyHqiWNZWfI4Yu62+1NmGVxHH0iVRk9Muto1W70vLdUMHcFFMjtahutzugKXBOr\\\\\\\/vrCG89z8kXUL4BA7EKPpzdiMYWsG5JFhSk5GGXCl5q1ywUx1xd9BMqrtf43f9+1bx46snap7hoLKQhfjaZ\\\\\\\/j4r2CybF6iamx5GR6XFXo\\\\\\\/i9UJ0RSnwmx51yu+A2aT\\\\\\\/OA3o6kWg6ELO0x7Yhum31aSV9AH2mMjVm9ckoITptsImG4ZPh\\\\\\\/wu3l5UO+gmYFXuRs2JEo\\\\\\\/oyamF0ATXP5vBUOz99xr6mXqe8wAn0vxYL4\\\\\\\/WaqyV5Ca9tLLHJQEUklstmJW\\\\\\\/SeYZZtwc0utJbjkK9d4uA6vl\\\\\\\/grSGb5ygGwYc8HyA16JJQE8cHOmks5HnR8XEBygBtVpFLJeccW2ccwmaQawGlM8Pd5TVtLfHLYEx0SCiebp+8uhJ8wtRVu7xj2v4VVqNxi6IxY5voz+lz21muJ++u6rwV5u5TsJk9bNiREmQ1nLfecM3jtjT70Q\\\\\\\/fULPRhLyb9tP+FZwVONlPj\\\\\\\/6wwJB5oAwRx8uX+US5LDJJwTDFl3gltjOMaMUmmxcFBMcjmwX8t4jfSRfx8fCk1DHR6J0cDwqqDc3UEDIn\\\\\\\/vT8gD0Re7G0LC1ZIx5Dulmsyg+meBABqb5rXtEepXwznp3BSrsbSASlZoCoaJkn6bzxOjOUAQ4nz1d4sMJS44Q5rtX7PA8iylWX6IvNae0YOPD19a1l\\\\\\\/h7gZ5i\\\\\\\/42u2QJqQ0WP\\\\\\\/3YTgbQl+eLnbJluoC\\\\\\\/CNci\\\\\\\/QQYzIGqxYzrX2We1F\\\\\\\/AA5XIiEUoXRmpJQFMGh2PTwdd9cQhQTtOnJ0WrLWSZqm+5fsDGzDsCmSxAdCKo3AjgSpuJchf9U0NqOR7U96h8K9TuqnmcA6t144VrOW5rHTB5uxF7dIMvIFtJNztRdPflB9lKAa57lagiA6rVnVTWsw9kNZjGcTV0CrEnWU2pQUz5SBbMz7z5G\\\\\\\/oS8XyoCtQ68P46IofgPiZO26XdjeYKatiBXBXo2BBwNEYo2nPNsytdCAP4ZagAeKLqNtAoTvkrYX7lo3YxlA3zC52JiSu769QlYdvWincbVEHp+ru1tSnRQ4Mb5wZGQMPYOpZK5LSn2\\\\\\\/iD6kZ50h\\\\\\\/U7vsFeSBfoIjnaBv6VpkhDlYjcxLXu\\\\\\\/cucn3irfpUNrGGDW15FvMM4KPFUMwipryBGiLdulURg9+ai+\\\\\\\/2FR14iC2jEquP3wBLaIeNPiLFgx+Szj0b7pIM1rMZBzPL1BZzZzcfn6zooAinNwseccmeMcApBdbz6hR4coNjMwzMLLJnLDd1N85jTLxCBRZon1QXwP6W+H0PgvPkK3vnna0VVMTuxewYfl\\\\\\\/3giMr7dlc\\\\\\\/8S7uzs35S509caRbCKB0tS0izEVZnDAV+\\\\\\\/Nl89xJwgeXF1r+fa7Mu3Btf\\\\\\\/y6ze3MpjCrZNWQ2\\\\\\\/FYyMQCKcFxGGy5BmHOAsdoAcSIPMeX1Xm53WYylDEm4pwzXDsbpe40U\\\\\\\/3MiZAmsWLdAW+Em+ReEt\\\\\\\/6lyjcyJ+7NLKJgZKjsba2+3iYgN9D1TJKfE08odpOOOa+5X2PKSAvbI3f56XFI9hfs5verrHYjhNK0\\\\\\\/v8\\\\\\\/tXrHQ5P7ZcG7fJCG2rqMceIW+zgxP3QUkHRSvraAnSyzkeDPPjEmOVKGJSYeb\\\\\\\/ArIuD03jDoqJmPUoF1+YXL5q+ZSjkNNfWykqAEuMAj0kdiV9CxS47TmKfkttp1V8hR7QcmFGjnmLv6nAl47PrTBMEsZGcYrBEu8Bs8IIm2g03G3WaM1D3IZGORnXc31\\\\\\\/Nkk5fY0nrJ3dckiZbNarYh58BYvAsV\\\\\\\/aPefCNDelGT60xWnfdnHa0vus\\\\\\\/GkfyBFhpI3hZR1eynuYgQSpq1WOcZsHLk32ERibENYDC2rAkkWP6WQ3wOeWL5LPTXCn41ze6\\\\\\\/PSQ3I1ok+Of+0JAdVbUTCMb3\\\\\\\/5n+mw5Inxvrv\\\\\\\/hXel4nQ799\\\\\\\/lkBQd\\\\\\\/hvCM108MIqSoZ6dj8Yi8Q3y3qDyRPC5vcT5IiLkmyxcbUnTcIc4qS0P1QG\\\\\\\/PmAV49uJ0ZlkCrIxSv6pWb\\\\\\\/RaW4bGYm++e6CbIFAdt4Ig\\\\\\\/KizGWI9TLHixCDxfOilYHyzoZnPX4u7XbTtqLw+LO720cBUaifRQKzcaO+zZd7XSOykLwQTr3IB4FLS+jpfSHgWrZdpm1ZI4Y4GtrK1miPabk2a+owI2sARnemEDsBZ8mRRt2Y\\\\\\\/+29BOklBwwyu15+u0r+bQk2OQuOa6Dfq+gqfb1kXaLSDhJ50oDmEzvq7ivrGm8OvxJvkqLF\\\\\\\/S3PyBvi\\\\\\\/CRqctDcK+NbR8ICERgH6rz6WCjP2NncRMd9WetDMGj+LBwj0f8TmVnHXpKYJzJYfw1Ak4zi0Ioepuy9kk42YBcMhj6F2SuJlx48MR4c7jOPTATslLPdW+Bqkb3IeRj3ytarjQb06HMG+IyBYqNaocfgokpp0j77b+KqSlVBpw08oqrQFKBsw\\\\\\\/MKQ4nEAmRmw6eMcYKT6l6fT524SyTv36Ikbidl3RaK41fm6EGNhXXASSB81rDk0BSVzq4KnAT5UJzM6tiz1TaFBRKZbAEmdfXk\\\\\\\/ASwHmT2izPDSwlwE63NwV2QLGaml0KBHUdXlUF7nGnAyQMenVjnFoCo6rlJtjBMkpOA8ym2\\\\\\\/GH++XuE+yWgMtaLP8nzVYID8beMH7oqYssmQXEAK14VqTh65yP5SRNjHTqw26ogJQ4XP4k\\\\\\\/gRS5TBnXPW35mh6+I8wff5Z8g4LBdeBcWPHfiSpx8eUWP\\\\\\\/DKqH9OmgFpBbrCrciv0nnJ1xVO+K6pNMq\\\\\\\/W8ZuAEnarCVy8YHf4I97Q8fXFpJpIRc2eMRlgX2TWTbtzl3DWofB8xtSmbUVPK6Uv0fjBYl6yp5n\\\\\\\/nLA4TUoG0DnOrPs9rJZta1cial7j1otV7qiIZFJarNxfL3u51TN2xY+LLldzJhPnKW+5DoOwcavjiXaUJWyRlqstUW1Zo7f2I0WSVQOGFsrNHzChVc1LB4rwrTzcaF9BTnZ2YUC7XHUYvbB6dJ9ap0PeNDu5mWqhn7On31RotstGS61T+U66gJws0KGvOv48akiN+FLnLfx3iQOReutYqNG+WQIY8Wz1DpQORqBy7KRmIs47L7N3fX6uPlCDjkfGr7X93lLGd\\\\\\\/oTMWCGYA+rAqXEb\\\\\\\/6+i4daS+XhEcUwRSdvy4sorKlMf0lMVDKTVv3d5CsVnzoTQGvLIQmDtaVDlSMXUj32uo5+d4aQKnfPyEGCq+M6RcQdBlfGD7XuLSXh\\\\\\\/kCb1aa\\\\\\\/CGV1HhTFFfRidoOSAFOSzBo2ywZZDu+hjqYlHFOG3mrRwd7SqaXVNoyYrW6H0juIjrbM1iyaHKBD\\\\\\\/8QvJYzDWvYiilj5D6roY1DkTwYfCJ2hpyNlH\\\\\\\/r\\\\\\\/+o+VO4WkKImiMaqsWR+jLwL1MDOo40G56hiHES5GkDcE6dRiWcD42Xs0QIxMM6CgrZ+dIyIjnmGq4M3Ter4D+DgUs3P9IJqoMH1HLgUsIzUmQc\\\\\\\/RVSKKx3NHo\\\\\\\/hRhmrOdKuvOmQNToMvzBa\\\\\\\/9iZDsiggnvDScstHO+4S0sDqcJBdLjKJULexHqhWE6SyELjAJXPIZeBdijDtE6rVM8Gv2iKRACNkkbnctVoPNgxU\\\\\\\/JBd\\\\\\\/jwfhBCoQrLysltYRb4a+YbEBm7TJT5N6Vf\\\\\\\/QNgoy86guhc7PnNym+BmULZh0L98beWXAa3z0IMX2pinQSd5JXNkghcED1beUvEsXWlKUCXq\\\\\\\/crtgMmYGw3vxKLadUxY94of2vbyaU8slKOApIyNjoDn\\\\\\\/h1HGCP6K1VAfsRBtaNhTNKBLS\\\\\\\/BkYc7rHYTw9gVOLUoSelO7zWRdDdNtKO0t6+v4nQPKcVfZfn9S+fZxQ0Wykkkw3LFY0Ikxh7t2TtAmSGTsowZxXJfqPUVBc5GgOJ8bFxILPhteOMMMRwOZvnmstQY1wj1NwkqVYptE6myoV5GJtlXdutVSiVACSPpMieI+TeT9TfDuRkvnL4YMjvo7W7\\\\\\\/7Vhje2kBZvEkLXt3mXJYbPVSUl8gxcYR2dQU7SzP\\\\\\\/njbx8\\\\\\\/E\\\\\\\/VpvUdki9GNNT7xrRSVxphtrhK0JKWmLtSb4gHpoOqlzgRwAoIUpOMQSn9B+ssZ5Glm2x8xQyJHOo7vhkrxAO0jMe4RiGZW84meKTfhP0VRiVusNl7SFow8YFyduok8YXVaCKDacjXU0CHYhP323et6Kmx6izrumWCkzJcG6vBwhQW\\\\\\\/Men7VzcCWoTNu7tX+12HDo7KLcEF\\\\\\\/GftbNU3DS3Ubnl3rmZH0iHaOH7x2YbwyKnFHRxdKcbs6CCtRx38u6MGMhQO+f+xmj3DtKyUlvWb3dN+f6aNpLoImvXWy1wbCK7X+f39GHq+EHHbIflEn1T4QmqgrB6REfi5OlUMM50B\\\\\\\/KOpUpyERBqhpJcoH0OkUjo3GXhatjmSMjSgsRSypNTc5CR0BkpOd3EIAQ5WrPRZxrp1N9rVhG1aLd8H+5SxpwObhinjkFz8A\\\\\\\/FEXHN+ZIsVMsKYOTFD6LMCvJy7itPW29pvsNyCZO41MNxqzXGt6iF4ytPRZGOZzHeATMeI69CuFRO2feSfN65BriJdbzKXShc9gmm0KahKKmYq+5giKqelEZOU99um\\\\\\\/QAPAy9CXC4\\\\\\\/o6HqcH4A6JsrRtKK83OPSxDpkwaod4GQUMKNrfPppk07dok9G\\\\\\\/PU\\\\\\\/tB3uzcJbKy833G3X\\\\\\\/zJCasl5buSxCXA+KHxwKep+y5qnSjcpun9FdqMBDsxJkB9oA07YlMZkUhyAWLnf4uE\\\\\\\/lMJQyg0knzat8GrKVKUPexz1MciKypToeKHehOeAKwb6kkUuovIw6Tq1mfnRaP51CV\\\\\\\/bJn1t\\\\\\\/igOYV3p1Z9xC14ckK1lUsqOsdNh+6YRHRKmLCrdsQi9cIA0Ng+n2kRNhDU2QSZ2f1e2RB0X2u+JnNscu1E3tCJQf6k\\\\\\\/bbIGqOtkcLZk6LetS9dVjCrfQ+rDvXXcvyHHKyX5xpyvAsvtWCylxkQIfnGLPaKHvFH1cAiajBykeZffmQroXuSZjWqb9ZOP2GaUwW119ylnV5yEjvW+8ywsb8FBnOmM8\\\\\\\/gJOKZ3J9QdaliRxnbswZB5Tvx8z0CRbMeZrvUK7sbh6dl5eYY+jeqqdM98MCSwAeUIu5yviRIZJpFb3vlOxdkWyjuD+0JO5eCywUckNyyMM5E3SisC9PE62XNrLL6LdephNIKyufKJ4GGQ2o9zcB3me+pgc26lm7Mtsj9qrdxkegonPqmlCKP9NiFMosSB8hzgVSqA7vQWA4quanvsHitvkk\\\\\\\/dbr7T2mIzH35eAM0Ppjgc9a9oZK+85thEuhij3rPIIekrMUTiKmdpRf7XCciT+mFwixsWt3h3q3UBvrqm1l1HUwos2dSWvZMX9UTHW2Cu0uOdP+cbh7SLSj3L7M5OCZbx9ybN39EsxZ59uPVdVrALGH6bNhGP61crxnU0LF\\\\\\\/cdkZxwLATLwvd52LT9zIgHLkRbTDYZh+zKsO6VWNQ47nPt0k6qjnHOW5w2bbXYT8ifMr7cBA64JT6ZOeNFJ\\\\\\\/evLBjyCyGDN5nmsemwC9XJEZmC6F\\\\\\\/0uIQRoso+Rdy30ujSM6+BzUjjz8HgKydUkvSkGds5jxxdpjLfZ9PRcU4+2za8GNWzjMB6l1+4tFnwBXSftINjmA+AKQydgJJpEqUeUj0pVWnkbJN99BMIzkQQFHoNVhSoztpuXBHzk9Fv8VxpIj\\\\\\\/42PMWRoUMml\\\\\\\/4iQv1Tlv1cTUQ5Xw1cj1o8gj45JT+nPGTyos6xYALALyTohB0A1I5ZqSPa5R2nWM9uLjgJi85vcBJukCzgrD09K5KKfGh2UOztKeYQzegCWsk\\\\\\\/9YMgAY2M9m5Rjjwc\\\\\\\/uKMG4g5QRoOL18x7tcwCft\\\\\\\/VGjpMMe4GGf+tyjPjtfFID+0M83xtipOMWmpbKup4Y+HMNlLCphRogk7nJNGdNKwt7O0PZ3MWgATcX7JtjZfEj39RL7mY3m2BT3f8atrUHI0LtbE7pqNn71k742st9wAKtDCq\\\\\\\/Hfttlt\\\\\\\/UlFxXncTNxxYo8c3tuxXXD2kNqxy9BIYOxP9CODgFFqbH5wAfG+hRv\\\\\\\/1niKzNkR+U0pkVjSWBmMmYP8VbF9ybsW2txZ7qhzQsHtJ9WYUMTej2u5sYZiZpfkc6plovTFjswDXhQTgz\\\\\\\/NhhQrl4kTc2v0rY592oPknoCPrVYsJ+5Kxtc+XaaOmm830KQs3fCxK+AxxZ5GLsSz2mWkbnwRfX9WOaiInUdlvGRubC+i140dT52T6a\\\\\\\/gHPVUvS7QVxmeRQpsoGZ0JxaZ2vfMNuXj4gEb6HhF1jFY7mgd4oAEJ8vtaUgzOZyd3fNtLtAmmFs5C\\\\\\\/M2pUHkOlNIak51p0ElryjeN12N6r0CwKet7ogrTkbHVUadhyYTSYVV445tKqbjnw1yK\\\\\\\/W11bXGrY0GgEKqC+d3gQjWOqjW0jtd3IXGuYprgJoG1LYTTLETL4aMfpcYiUnQlCasHp86Y76mlX4Q8JnKt4Kb3WjYtryLiDTG5pelkUJiPBrJiIFD6mmd3CKM2UhBJZ3URpCWVXQ57YiUn9fXb8cPJNGK7X9M1oobvCcTc+jc97bAgEDTXEtGIBlQPDyPt78FrLK\\\\\\\/pPkjrmia8t1d9GqvkHwTZ9I+LNFPjOlbe\\\\\\\/nza0EUctZBKXmvtMXBi20ulNRH2RtznhpOrypc+vmHJ+Na9TTi+nmlgXdYrp4PcT5aK4LbPQCr+vQ020AnIk+w09eaibMEsPIdNuzIOhZ7oXfR8YWGomY5+0O0umIcCTi7Bj84IaZXsEx9MKtTSWlgFl9YRWTuSTIpJIh5UZl\\\\\\\/GXQrzRQNiqDatB53fojZLf+XsyJJ3wAbnFhq5K5Q6lcFRY8v7sCPg\\\\\\\/aYa9ClpLwufra0XXJqfe1YZvf+Zz9Mh\\\\\\\/yNL4C9E9\\\\\\\/1W\\\\\\\/F6KFhHE5i5PbPx3QdOQU6AQ9RKbsPp0LLy1+6wwLT+53djmf9K1toqyrO6EQS\\\\\\\/bVpqfZqAlTOMk8THJ0nNeRCd6fFob1awtSwYD5bNA6RwNAZCDLxuiESUmsTVRjpR3fSTCdlDeJ4yohCYuftsIVLnd5sg\\\\\\\/9ZTtsmlMLF7EyA8yW7KnDI31LAsXZHS+H9JM9jTpEzE09lDW79hpKTh84CedGJ7JbnVvX1jxp692FGc4oKS51jECYckcev5\\\\\\\/Zq2zlvGSx6dLH3V4C051\\\\\\\/b7qD\\\\\\\/uXQ3o2pKhMStFyywDaIpbTbTDV5shItzAOlmzcO5Ylm65syKfVCFPSni3+x5yQYRYkHvHm6jzsfv3R1kBrYlm9tKFjvRlhQXb7SIzvRpVqr9j5LMN09D3eRxpvKF2eINy9hKmleFZ12sPmNM0wrxEvju3UWamsfNlfBGRoxCb6zDPWAl85qG3KU+c37v2klf+ry2fBlKKMkrVvWCYUf1wzPUkKgqQyEt1KpE1+S+E\\\\\\\/+1VaNJCS3TEmRnVFsLn7FrzgNBGXOtrHelrWzKk6\\\\\\\/3s6kBXDlbrPgy5Pe+cSFC2VZEESK51oSexIK8q09tXbbB\\\\\\\/xJWH+KNp\\\\\\\/RKfKvbVzYsvDseUJJlEkT\\\\\\\/N1mypRIDKTQkxqRWfdoR+gKcUm2t69MmX732JxA\\\\\\\/FlCEnPzvu71k1Iag+ub5VqgkNWDPsOvNGoaVV0hKb5UoJmZxIvWEc3yaTBGtIcgD1NApRegMetIm2QE29wT4bq1uHA1kGdJfNIrV\\\\\\\/8BamkQuWlTbJDfpuu7otfXfK41cLcyTvupc+W9GbpuCU4eK8LBj0kMDM6P03Ow43g==\\\u0022,\\\u0022iv\\\u0022:\\\u00229fN39PZ6ejOoEl3j0cSKOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002283dfb7921e5dce9b86358fc1219525f28b40e6430d6a03b97289fc0c5cb6fff5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J0O2FbGoLrAQZ3S7UQkFrQb+ojftl8bPZHdsrEGMWCRQurBP5pijO6bvIBCGFbsbxTGSrvuNq0rXDKQZoNWBrTwV8qcraRgrIEka1RnfKF9o2WnWJpu6d\\\\\\\/JKhseUG4DuUoUhVt+dFZFBs1Q5thCkdlCRKMBIY0\\\\\\\/02VskUE\\\\\\\/MjvwemQFdBNFZfx3Fc\\\\\\\/WP1U\\\\\\\/pUskxWaJlRGbTUYe0kGYrb4ywPQIBofjCIsJcryPa\\\\\\\/e+9r1E1t5psZtLGrXmzlfUbj5ehjsacmgt+DOTOK4QEqUgsYy3HAlQiVQf7z6y0z\\\\\\\/9AkZmkVdUzugvKbWFSIRSoPuuagjBPUFwO9oUqZdBNTszA77czgrvsLHNMr4ZqAC5J2pR8eccIrvV0t5KZrUS25wFRMsPuhNR09EG++PQu0wFQL8Eip+b+ofJtK9X3Np4cd3sF3bQ1A7qR1Zqc7r1XHwFwOpuUBKkLMs\\\\\\\/P6\\\\\\\/oNBHjXc1U667iDa6nFQx9YfPUMR9p75okslq1NpyOr1ZdwKbYVJsgnSQVy5oMBgHPhTKhRycW4RNMJ\\\\\\\/XE9JVavzX717Ggk94vfV3VxnR6Cbe\\\\\\\/dW4Znc1hY2XIPg7+nnazmyO293zj0KKQffy1gnwVaoB3j+gkB5CF2cvYRQxzetTGDzOhp2jMvhvrRt5Zn7pu69vTv96nTgojyl5c6JqinzDBnuYCuHuncIxY9G+JCp5C62\\\\\\\/PeDalSdaW7A\\\\\\\/wz\\\\\\\/0S5McqymOP2DRd0JijGBz+SXap9CVYv9g9mXm+avrWqr5X5EOxqNypZO6Gs2C3ZcFp6ngZJpPJ\\\\\\\/AE4ySldgFLWAS2z21F01RjTdsMAiNDXUMxaOTCHG3QnKLayBPMcsczzw+hJNznahWGYYMu2lUqXOwYlNZ5TGfQhOvaaFRypdbOvRs25x0pfoVVnoZpRWu0ItaFUU6o5+UFf\\\\\\\/a+X\\\\\\\/AyCzbuYQeCJxR\\\\\\\/Gtm\\\\\\\/8GJtRG\\\\\\\/UkTE1CfXlHLC1Vtp+UM6WpajRPHkGUZ78C8x+1LrWwhpw0bGDWDoQdjsRBPbu7epT3l8UUp2RRTbLm6FAD5uEHrxoCl5beXeB9aSQTH42hqVR992GG7pZg7QYWz1uC3UKF3MmbFd5uLdxrPIA6Mg1oyu9we18X78ADv+QJnX0ICn+Ml+s91oPx9kW55WXH5OuJnQvJUfb2wnTpKpf2+VSdkMWH6lfgYzmmc9ctWHVRxnXtwGZAefOFwDfvcKie62dg5E+tk9fYkafSUB1EIgWqL8TJ2Tcg\\\\\\\/S9RTFA8DEq5jjP+dewP9J7ULygo2q8JqyFKN1IPiNFoKqfNIbm7CZEqnaJ9vcBb8M6j7\\\\\\\/ovJUkJB6ixEdfUQF4ojRvRiZM0MVyeB2EGFlZzNm946aMW6eqK3KSufUgFcx52qjTBEBEVw173ruuWb9mm0IzFFIXapg7aqvBy+cT9KU5Ga41jiSISFCVA0n6GBtk54KgbJ\\\\\\\/rKsc3Slbi0+YfxklLAdkV1kBkmCYmq87DpCAekdXuP+oKPt+L6WNNhsgcg5XepuGWJztt2j6jymh\\\\\\\/OVQCFZfg62u1BhGN\\\\\\\/hMUamGRftL2oaYAuBhB72Do\\\\\\\/AMKLmNm4p+euoPcBhdLBRw3uCmVWyvZSNbiSBWQWanoPv7DqKn\\\\\\\/J5hV7b1DkpuaJTkY3MXSVkwpyETPWoiiBzDwATl\\\\\\\/O43vDNgsclBUBmXUCVInwODjJo67RIxxi\\\\\\\/5JApwpGIc6NQosaX0+\\\\\\\/R7O9MdMggCXEKHEyyuGrdfK8q1ZA2lnJ8f+SHCRKqwR\\\\\\\/m88TxEW0kWoj2QcBkZniA8MjnueWOJEJPR60\\\\\\\/u0I2bn77aZHPMAkJWD6niF848n1n4YT+4UC3OHTAM\\\\\\\/S7m2wi5APUwpf\\\\\\\/IIV\\\\\\\/1lazz5qQ8wnm9UEkJsStYiuvD4wgVud1sIdUlKyRuXNnocVLCNAX\\\\\\\/rG\\\\\\\/1thETFP5jQ9s\\\\\\\/UvaaDp5ZV9\\\\\\\/+5+jZzkuvA+lKg9tAQGRDQr97pPDvTqOarE1hKo1qLgXbMt9E+wZ3CBBrrskcLFihVVH3\\\\\\\/lnRYp18A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hf2G72SnAm96lrZfA1Aohg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225e503149755c3d3543e88fd327837cdf2b39b13459204dd768e66066eeea02db\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459805 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/559" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tbkrb791Xe2uRWjQYVcQfUPCiJVFjUKBBRXJyRfOy3A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/559" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aecf9UgnVV6lb6fd6-Q7aGNhafVLTTApeHnHtaVXBPM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/551" title="Comparison of Larval and Adult Drosophila Astrocytes Reveals Stage-Specific Gene Expression Profiles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/569" title="Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459805" id="node459805" data-pisa="ggg;5/4/559" data-pisa-master="ggg;g3.114.014571" data-apath="/ggg/5/4/559.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of <em>Lotus japonicus</em> Using Second-Generation Sequencing Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nur Fatihah Mohd-Yusoff</span>, <span class="highwire-citation-author" data-delta="1">Pradeep Ruperao</span>, <span class="highwire-citation-author" data-delta="2">Nurain Emylia Tomoyoshi</span>, <span class="highwire-citation-author" data-delta="3">David Edwards</span>, <span class="highwire-citation-author" data-delta="4">Peter M. Gresshoff</span>, <span class="highwire-citation-author" data-delta="5">Bandana Biswas</span> and <span class="highwire-citation-author" data-delta="6">Jacqueline Batley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">559-567;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014571 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459805" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nur Fatihah Mohd-Yusoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cell and Molecular Biology, Faculty of Biotechnology and Biomolecular Sciences, Universiti Putra Malaysia, 43400, Serdang, Selangor, Malaysia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nur%2BFatihah%2BMohd-Yusoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohd-Yusoff%20NF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANur%2BFatihah%2BMohd-Yusoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pradeep Ruperao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pradeep%2BRuperao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruperao%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APradeep%2BRuperao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nurain Emylia Tomoyoshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nurain%2BEmylia%2BTomoyoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tomoyoshi%20NE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANurain%2BEmylia%2BTomoyoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter M. Gresshoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BGresshoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresshoff%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BM.%2BGresshoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.gresshoff@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">p.gresshoff@uq.edu.au</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bandana Biswas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bandana%2BBiswas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Biswas%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABandana%2BBiswas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jacqueline Batley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BBatley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batley%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BBatley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/4/559" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459805" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459805/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/559.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459805" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459805/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/4/559.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459805" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459805/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/4/559.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459805" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459805/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic structure can be altered by chemical mutagenesis, which is a common method applied in molecular biology and genetics. Second-generation sequencing provides a platform to reveal base alterations occurring in the whole genome due to mutagenesis. A model legume, <em>Lotus japonicus</em> ecotype <em>Miyakojima</em>, was chemically mutated with alkylating ethyl methanesulfonate (EMS) for the scanning of DNA lesions throughout the genome. Using second-generation sequencing, two individually mutated third-generation progeny (M3, named AM and AS) were sequenced and analyzed to identify single nucleotide polymorphisms and reveal the effects of EMS on nucleotide sequences in these mutant genomes. Single-nucleotide polymorphisms were found in every 208 kb (AS) and 202 kb (AM) with a bias mutation of G/C-to-A/T changes at low percentage. Most mutations were intergenic. The mutation spectrum of the genomes was comparable in their individual chromosomes; however, each mutated genome has unique alterations, which are useful to identify causal mutations for their phenotypic changes. The data obtained demonstrate that whole genomic sequencing is applicable as a high-throughput tool to investigate genomic changes due to mutagenesis. The identification of these single-point mutations will facilitate the identification of phenotypically causative mutations in EMS-mutated germplasm.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/abscisic-acid" class="hw-term hw-article-keyword hw-article-keyword-abscisic-acid" rel="nofollow">abscisic acid</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/lotus-japonicus" class="hw-term hw-article-keyword hw-article-keyword-lotus-japonicus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Lotus japonicus</em></a></li><li class="kwd"><a href="/keyword/mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-mutagenesis" rel="nofollow">mutagenesis</a></li><li class="kwd"><a href="/keyword/second-generation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-second-generation-sequencing" rel="nofollow">second-generation sequencing</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li></ul><p id="p-3">Mutagenesis provides a powerful technique to improve plant breeding and assist functional and genomic analyses of crop plants. This technique was first introduced with the use of x-ray and radium radiations followed by fast neutron and gamma radiation (as reviewed in <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sikora <em>et al.</em> 2011</a>). Because such application of physical mutagens required specialized equipment, chemical mutagens were introduced later. Chemical mutagens are used widely because they are easier to handle and increase mutation frequency (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Loveless 1958</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Sikora <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Serrat <em>et al.</em> 2014</a>). Various chemical mutagens have been prepared, such as sodium azide, ethyl methanesulfonate (EMS), and <em>N</em>-ethyl-<em>N</em>-nitrosourea, which produce different side effects on the genetic structure of treated populations. These chemicals can cause point mutations, insertions, and/or deletions in the genomic strands, leading to phenotypic changes, which could be desirable traits for important crops (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Olsen <em>et al.</em> 1993</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Greene <em>et al.</em> 2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flibotte <em>et al.</em> 2010</a>).</p><p id="p-4">EMS, an alkylating agent, commonly is used as a chemical mutagen for DNA lesions. Unlike <em>N</em>-ethyl-<em>N</em>-nitrosourea, EMS induces a biased spectrum of G/C-to-A/T transitions. These transitions most likely occur due to the alkylation at the O<sup>6</sup> or N<sup>7</sup> position of guanine, which leads to the replacement of cytosine with thymine base pairing (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lawley and Martin 1975</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sega 1984</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Haughn and Somerville 1987</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Sikora <em>et al.</em> 2011</a>). Known as EMS canonical base substitutions, the high frequency of G/C-to-A/T changes has been observed upon EMS exposure in different organisms, including <em>Arabidopsis thaliana</em> (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Greene <em>et al.</em> 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Till <em>et al.</em> 2011</a>), <em>Oryza sativa</em> (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Till <em>et al.</em> 2011</a>), <em>L. japonicus</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Perry <em>et al.</em> 2009</a>), <em>Caenorhabditis elegans</em> (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Flibotte <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Thompson <em>et al.</em> 2013</a>), <em>Solanum lycopersicum</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Minoia <em>et al.</em> 2010</a>), and <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shiwa <em>et al.</em> 2012</a>) at different rates. EMS also tends to produce random point mutations and induces a low level of chromosomal breaks and lethal effects (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Greene <em>et al.</em> 2003</a>). These effects provide a competent survival rate and allow subsequent analyses to be performed for both forward and reverse genetics.</p><p id="p-5">The whole-genome sequence of <em>Lotus japonicus</em> ecotype <em>Miyakojima</em> (MG-20) is available, covering a total length of 315,073,275 bp (67% of the 472-Mb genome). A total of 91.3% of gene space is located in the genome sequence (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sato <em>et al.</em> 2008</a>). The genome sequences of the chloroplast (150,519 bp) and mitochondrion (380,861 bp) also have been assembled, by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kato <em>et al.</em> (2000)</a> and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kazakoff <em>et al.</em> (2012)</a>, respectively. Numerous bioinformatics materials on legume and nonlegume plants are also publicly available from various resources (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sato and Tabata 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goodstein <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2012</a>). With the current high-throughput tools, genome sequencing can be performed at an affordable cost (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thudi <em>et al.</em> 2012</a>). Many programs also have been developed for analyzing sequencing data <em>in silico</em>, offering good strategies to employ our study.</p><p id="p-6">In this study, we applied Illumina second-generation sequencing (2GS) to discover EMS effects and the mutation spectrum in the genome of model legume, <em>L. japonicus</em> ecotype MG-20. We randomly selected two mutagenized plant genotypes from an M3 population as our subjects that were deep-sequenced. Wild-type (WT) MG-20 also was resequenced and mapped to the reference genome from <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sato <em>et al.</em> (2008)</a> as a comparison to subtract natural variations and false positives. We aimed to scan the effects of EMS throughout the whole genome, regardless of the phenotypic characteristic, which resulted from the mutations in specific regions. We identified single-nucleotide polymorphisms (SNPs) and compared the base alterations that occurred between both the genomes. The data demonstrate how 2GS works as a high-throughput platform for rapidly identifying DNA changes in each EMS-induced genome. As an advantage over sequencing pooled mutants, scanning individual mutagenized genomes allows rapid analysis of the mutation spectrum and gives the actual picture of “corrupted” genetic structure. The output of this study also will provide information in the identification of genes mutated due to EMS mutagenesis and demonstrate the distribution of mutation is comparable in different germplasm of MG-20.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-7">A total of 4920 seeds of MG-20 were treated with 0.5% (v/v) EMS and grown as described in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Biswas <em>et al.</em> (2009)</a>. One hundred MG-20 seeds were soaked in sterile water as a germination control. After 3 wk, the phenotypes of the plants that survived (67.3%) were observed to examine physical effects on the plant growth. Physiological tests also were performed as reported by <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Biswas <em>et al.</em> (2009)</a> on abscisic acid (ABA) insensitivity and by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chan <em>et al.</em> (2013)</a> on ethylene insensitivity. In this study, two mutated germplasms (M3) were isolated from the ABA assay (called AM and AS) for sequencing to screen EMS effects on their genome sequences and compare them with the resequenced WT MG-20 genome. In short, AM is a homozygous dominant, ABA-insensitive mutant (confirmed by stability in segregating families), whereas AS is the WT ABA phenotype segregant that was generated from a self-regeneration of a heterozygous ABA-insensitive mutant. Thus, AM and AS should carry the same spectrum of SNPs due to EMS mutagenesis.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction</h3><p id="p-8">Genomic DNA was extracted from each individual plant after 1 mo of growth. The cetyl trimethylammonium bromide (CTAB) extraction method was adapted from <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stewart and Via (1993)</a> with minor modification. Plant tissues (about 1 g) were ground to powder in liquid nitrogen before adding 1 mL of CTAB extraction buffer. The mixture was incubated at 65° for 30−60 min. Five-hundred microliters of the mixture was transferred into a new 2-mL tube and 500 µL of CTAB buffer was added to each tube. Both tubes were incubated as before. Chloroform purification was performed followed by isopropanol precipitation and washing with 70% ethanol. The nucleic acid containing pellet was air dried and dissolved in 100 µL of sterile water and subjected to RNase treatment (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sambrook and Russel 2001</a>). Extracted genomic DNA was assessed using a spectrophotometer and agarose gel electrophoresis.</p></div><div id="sec-4" class="subsection"><h3>Sequencing and SNP identification</h3><p id="p-9">Whole-genome paired-end, 100-bp, short-sequence reads (&gt;10× coverage) for AM, AS, and WT MG-20 were generated using the Illumina Genome Analyzer IIx according to the manufacturer’s instructions. These three datasets were then mapped to the MG-20 reference genome (<a href="http://www.kazusa.or.jp/lotus/">www.kazusa.or.jp/lotus/</a>) using SOAP2 v2.21 with the option –r0 to retain only uniquely mapping read pairs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2009</a>). SNPs were called using SGSautoSNP 2.001 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lorenc <em>et al.</em> 2012</a>) wherein AM, AS, and WT were referred as different cultivars. To avoid false-positive output, only homozygous SNPs were selected for further analysis. The three datasets of paired reads have been deposited into the National Center for Biotechnology Information Short Read Archive database under accession: SRX719550 (AM), SRX729747 (AS), and SRX131060 (WT). All custom scripts are available on request by E-mail at dave.edwards@uq.edu.au.</p></div><div id="sec-5" class="subsection"><h3>SNP analysis</h3><p id="p-10">The resequenced WT genome was used for comparison to identify variants or base changes in the AS and AM genome sequences. The frequency of transitions and transversions generated in both mutated germplasms was also calculated. SNPs were categorized using SnpEff 3.0j (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cingolani <em>et al.</em> 2012</a>) according to their effect on <em>L. japonicus</em> MG-20−annotated genes (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Sato <em>et al.</em> 2008</a>).</p></div><div id="sec-6" class="subsection"><h3>Selection of genes involved in ABA perception and signaling pathways</h3><p id="p-11">A number of ABA candidate genes also were selected to test whether a SNP is located in their sequences. As a preliminary study, a total of 32 genes reported to be involved in ABA signaling were selected as candidate genes. These genes commonly are reported in the ABA-gene interaction in ABA perception and signaling pathways (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wasilewska <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cutler <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 2010</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Phenotypic effects</h3><p id="p-12">All control seeds successfully germinated and were well-developed. EMS-treated seeds had a germination rate of 67.3% (3313/4920). Among well-developed EMS-treated plants, 76 M1 plants showed abnormal phenotypes after 3 wk of growth. Almost 1% of these plants (30/3313) showed an albino phenotype, in which yellow sectors were observed. Some plants also had pale green patches (0.6%, 23/3313), early branching (0.2%, 7/3313) or a looped base (0.39%, 13/3313). Two other plants showed either unusual leaf shape or early flowering. Vivipary also was detected on a pod among this population. In this study, the impaired phenotypes of <em>L. japonicus</em> indicated the successful treatment of mutagenesis using 0.5% EMS, and therefore, the population could be utilized for subsequent analyses. We chose mutants at the third generation (M3) as subjects to ensure the mutation is stable and fixed (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Serrat <em>et al.</em> 2014</a>).</p></div><div id="sec-9" class="subsection"><h3>Sequencing and read mapping output</h3><p id="p-13">A total of 32,965,291, 34,020,296, and 25,737,274 paired reads were generated from the WT, AS, and AM genomes, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). At least 23% of reads from the sequenced genomes mapped to the reference. The percentage of mapped read pairs was 28.17%, 30.28%, and 23.59% from WT, AS, and AM, respectively, which resulted in more than 19× genome coverage. As a result, the genome coverage was sufficient to be applied for SNP calling between mutants and WT reads.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465903/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465903/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Outputs generated from Illumina sequencing to read mapping</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Frequency of mutation</h3><p id="p-16">After read mapping, homozygous SNPs were predicted, to identify SNPs that are unique in the mutant genome as opposed to WT. As a result, the frequency of mutation could be observed in assembled sequences of all chromosomes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Chromosomes 1 and 6 have the longest and shortest length of assembled sequences, respectively. Meanwhile, the unmapped contigs cover approximately 32.9 Mb of total assembled length. Our SNP calling showed that Chromosome 1 has the greatest number of SNPs with one mutation per 170 kb and 165 kb in the AS and AM genomes, respectively. Mutation frequency was the lowest in Chromosome 6, which had less than 6% of total SNP in both genomes. The change rates were one mutation per 222 kb for AS and 217 kb for AM. In total, mutation frequency in AM and AS was nearly identical 1490 SNPs and 1447 respectively. This frequency is reflected in the change rate of mutation in the AM genome (one homozygous mutation in every 202 kb) and AS genome (one homozygous mutation in every 208 kb).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465874/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465874/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Frequency of SNPs in individual chromosomes and unmapped regions of AS and AM mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">The frequency of transitions and transversions generated was analyzed (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In our mutants, EMS has generated 64.7% (AS) and 62.3% (AM) transitions as a percent of total mutation. Both mutants showed a bias to G/C-to-A/T transitions, which were the most frequent mutations that occurred (45.0% in AS and 34.9% in AM). Transversion mutation was also detected, but at lower percentages as listed in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>. The lowest percentage of mutation was C/G-to-G/C changes in AS and A/T-to- T/A changes in AM.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465937/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465937/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Spectrum of base mutation found in the AS and AM genomes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Distribution of transition and transversion mutations</h3><p id="p-20">Based on the mutation frequency, how each mutation type was located in individual chromosomes can be observed by calculating the percentage of mutation type in relation to the SNP total in each chromosome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Both genomes comprised of C/T or G/A transitions as the most frequent mutation type in their chromosomes or unmapped regions. In the AS genome, G/A transitions were the highest in Chromosomes 1, 2, 5, and 6. Chromosomes 3 and 4 and unmapped regions had C/T transitions as the highest percentage of mutation type. The lowest percentage of mutation type was C/G transversions which were present the least in all individual chromosomes and unmapped regions of the AS mutant. The distribution of transitions and transversions in the AM genome was relatively similar to the AS genome. The greatest percentage of mutation type in each chromosome of the AM genome was C/T transitions except Chromosome 3. Unmapped regions and Chromosome 3 of AM have G/A transitions as the greatest percentage of mutation type. Meanwhile, the lowest percentage of mutation type was detected to be either A/T or C/G transversions in the AM genome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/559/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative percentage of different mutation types. Our mutated genomes had varied percentages of base changes in each chromosome and unmapped regions. A high percentage of G/A and C/T changes was observed in both genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-95626878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative percentage of different mutation types. Our mutated genomes had varied percentages of base changes in each chromosome and unmapped regions. A high percentage of G/A and C/T changes was observed in both genomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/559/F1.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/4/559/F1.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/559/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/559/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465796" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Relative percentage of different mutation types. Our mutated genomes had varied percentages of base changes in each chromosome and unmapped regions. A high percentage of G/A and C/T changes was observed in both genomes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Distribution of SNPs across mutagenized genomes</h3><p id="p-22">We determined how many SNPs occurred every 1000 kb to show the distribution of mutations across our mutagenized genomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Regardless of mutation types, the distribution of SNPs is unique between AS and AM when comparing the same chromosome. SNPs were randomly located along the genomes with no specific chromosome position being particularly abundant or lacking in SNPs for both genomes. However, Chromosomes 1 and 2 were highly “corrupted” in their arms. Meanwhile, some chromosomes (Chromosomes 3, 4, and 5 for AS; Chromosome 3 for AM) have a high peak of SNPs toward their center. The density of mutations in every 1000 kb was relatively apparent in Chromosomes 1 and 2 of AS and AM. Meanwhile, Chromosome 6 had less dense mutations for both genomes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/559/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of mutation across individual chromosomes in both AS (left) and AM (right) genomes. Mutations were plotted in every 1000 kb of genomic sequences. Chromosome (Chr) number is shown above its designated graph. Chromosome position was plotted based on the assembled length of L. japonicus genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-95626878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of mutation across individual chromosomes in both AS (left) and AM (right) genomes. Mutations were plotted in every 1000 kb of genomic sequences. Chromosome (Chr) number is shown above its designated graph. Chromosome position was plotted based on the assembled length of &lt;em&gt;L. japonicus&lt;/em&gt; genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/559/F2.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/4/559/F2.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/559/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/559/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465860" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Distribution of mutation across individual chromosomes in both AS (left) and AM (right) genomes. Mutations were plotted in every 1000 kb of genomic sequences. Chromosome (Chr) number is shown above its designated graph. Chromosome position was plotted based on the assembled length of <em>L. japonicus</em> genome.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Coding and noncoding effects</h3><p id="p-24">Here, we used SnpEff to predict the effect of mutations on coding regions of annotated genes of <em>L. japonicus</em> for both mutants, as shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Our results showed that the greatest number of SNPs (34%) was predicted to be located in intergenic regions, followed by downstream and upstream regions of predicted genes (27% each). Only 7% and 3% of SNPs were located in exon and intron sequences, respectively. Intragenic regions also were predicted to have low EMS effect. The mutation percentage was very low at splice site donor and acceptors. In our mutagenized genomes, EMS effects did not contribute to a high fraction of nonsynonymous and synonymous changes (5% and 2%, respectively).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/4/559/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutation effects on codon sequences by type and region in our mutant genomes. Single-nucleotide polymorphisms were observed highly located in intergenic regions, upstream, and downstream parts of the annotated genes. Only a small percentage of nonsynonymous changes was predicted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-95626878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mutation effects on codon sequences by type and region in our mutant genomes. Single-nucleotide polymorphisms were observed highly located in intergenic regions, upstream, and downstream parts of the annotated genes. Only a small percentage of nonsynonymous changes was predicted.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/4/559/F3.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/4/559/F3.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/4/559/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/4/559/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465820" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Mutation effects on codon sequences by type and region in our mutant genomes. Single-nucleotide polymorphisms were observed highly located in intergenic regions, upstream, and downstream parts of the annotated genes. Only a small percentage of nonsynonymous changes was predicted.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Candidate genes</h3><p id="p-26">As preliminary research, we selected 32 notorious ABA genes (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) to determine whether any SNPs were located in their sequences. Orthologous genes of <em>L. japonicus</em> were identified based on <em>A. thaliana</em> and <em>Glycine max</em> sequences. Six were found to have orthologous sequences, which were positioned in unmapped regions of the MG-20 genome. The rest of the orthologs were found in the assembled chromosomes. SNPs were identified in three candidate loci, which were ABI2, ABI3, and ABI4. ABI2 and ABI4 were mutated in both mutants, whereas SNP was only found in ABI3 of the AS mutant.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465905/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465905/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">List of ABA candidate genes and their loci in the genome of <em>Arabidopsis</em>, soybean, and <em>Lotus</em></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-28">The effect of EMS mutagenesis initially can be observed through phenotypic changes of mutagenized plants, and numerous reports have been published on the effects. The abnormal phenotypic variance due to EMS exposure also has been reported previously in <em>L. japonicus</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Perry <em>et al.</em> 2003</a>) and other plants such as <em>Glycine max</em> (soybean; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carroll <em>et al.</em> 1986</a>), <em>Hordeum vulgare</em> L. (barley; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Caldwell <em>et al.</em> 2004</a>), <em>Sorghum bicolor</em> (L.) Moench <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xin <em>et al.</em> 2008</a>), <em>Solanum lycopersicum</em> (tomato; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Minoia <em>et al.</em> 2010</a>), and <em>Capsicum annuum</em> L. (chilli pepper; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sri Devi and Mullainathan 2012</a>). The percentage of the abnormal phenotypes varied in different plants depending on the concentration of EMS used (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Xin <em>et al.</em> 2008</a>). Phenotypic changes are not restricted to plant growth but also affect their physiological characteristics. For examples, reduced sensitivity to ABA was reported by <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Biswas <em>et al.</em> (2009)</a> in <em>L. japonicus</em> mutagenized by EMS. Response to high temperature also was affected in EMS-induced <em>Arabidopsis</em> mutants impaired in ABA and salicylic acid syntheses (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhu <em>et al.</em> 2012</a>).</p><p id="p-29">Sequencing data often are used to mine SNPs or polymorphisms among different cultivars for identification of traits and allelic variations in crops (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Varshney <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edwards <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zander <em>et al.</em> 2013</a>). The high-throughput technology also is applied in discovering causative mutations by pooling backcrossed segregant populations to increase SNP frequency and reveal mutated regions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashelford <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mokry <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hartwig <em>et al.</em> 2012</a>). Here, we individually sequenced two selected mutant genomes and used 2GS data to compare mutation distribution between these mutants.</p><p id="p-30">To our knowledge, this is the first report on the application of SGSautoSNP for discovery of SNPs induced by EMS mutagenesis. This tool uses assembled reads to identify homozygous SNPs without using the reference genome, which is only used for mapping reads. SGSAutoSNP was used to identify only homozygous SNPs because the identification of heterozygote SNPs leads to a high number of false positives, which would lead to an overrepresentation of the mutation frequency in the genome (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lorenc <em>et al.</em> 2012</a>). We also managed to rapidly identify the mutation spectrum occurring in the mutant genomes using the output data.</p><p id="p-31">Our read mapping has been successfully performed before SNP identification in each genome. However, a low percent of reads was mapped to our reference. A high percentage of paired reads was reported to map to the reference in other species such as <em>Caenorhabditis elegans</em> (average 92%; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zuryn <em>et al.</em> 2010</a>) and <em>A. thaliana</em> (average 73%; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Austin <em>et al.</em> 2011</a>). In our case, only an average of 27% paired reads mapped. Similar results were observed in other species using the same SNP prediction software [for example in canola (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dalton-Morgan <em>et al.</em> 2014</a>) and wheat (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lai <em>et al.</em> 2015</a>)]. The number of reads mapped relies on the parameters set up during the mapping procedure. We only selected paired reads that mapped and ignored reads that aligned as single reads and only used reads that mapped to a single unique location to generate more accurate read mapping (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2009</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Lorenc <em>et al.</em> 2012</a>). The assembled length of pseudomolecules in our MG-20 reference covers 67% of estimated genome size (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Sato <em>et al.</em> 2008</a>), which also influences read mapping output. In addition, reads that aligned to multiple positions also were removed to increase SNP calling accuracy and avoid false positives (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Lorenc <em>et al.</em> 2012</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Shiwa <em>et al.</em> 2012</a>). These factors have reduced the number of reads mapped in this study.</p><p id="p-32">Chromosome 3 is the largest among the six chromosomes of the MG-20 genome. However, the assembled length of Chromosome 1 is the longest, and presumably more complete, based on the genome assembly (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Sato <em>et al.</em> 2008</a>), allowing for better read mapping and a greater rate of SNP identification. This reflects the greatest number of SNPs predicted in Chromosome 1. Chromosome 6 has the lowest number of identified SNPs, because it is the shortest chromosome in assembled length (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Sato <em>et al.</em> 2008</a>). In addition, Chromosome 6 might contain more repetitive sequences and therefore less unique reads mapped, leading to a lower rate of SNPs being identified. Furthermore, the different rates of base changes between chromosomes may demonstrate the different capacity levels of each chromosome in tolerating mutation impact. There is insufficient evidence to reach clear conclusions; however, the presence of selection pressure, such as clustering of housekeeping genes on chromosomes, could contribute to inability to tolerate mutations. Additionally, not all SNPs will be identified across the genome due to repetitive regions and the stringency of read mapping, reflecting SNP/mutation density across chromosomes.</p><p id="p-33">A different number of SNPs between AS and AM was expected, because it is impossible to obtain an exact total of SNPs in different mutated genomes. The mutation rate of our mutants was 1/208 kb and 1/202 kb, which were greater than previously reported in <em>L. japonicus</em> (1/502 kb; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Perry <em>et al.</em> 2009</a>). Different rates of mutation also were reported in various plants mutagenized by EMS, as summarized by <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Sikora <em>et al.</em> (2011)</a>. Although only homozygous SNPs were taken into account in this study, the frequency of mutation was quite high. If heterozygous mutations were called, the frequency will be greater. Here, the frequency of mutation demonstrates the effectiveness of EMS to produce high mutagenesis as reported by many researchers (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dube <em>et al.</em> 2011</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Shiwa <em>et al.</em> 2012</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Serrat <em>et al.</em> 2014</a>).</p><p id="p-34">The mutation spectrums in our germplasms were not consistent with a previous report on TILLING work of mutagenized <em>L. japonicus Gifu</em> (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Perry <em>et al.</em> 2009</a>) wherein 97.6% of base changes were G/C-to-A/T mutation upon EMS mutagenesis. Similar to TILLed <em>Arabidopsis</em>, a high frequency of G/C-to-A/T transition (99%) also was identified (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Greene <em>et al.</em> 2003</a>). A lower rate of G/C-to-A/T base changes was observed in other plants, including barley (70%; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Caldwell <em>et al.</em> 2004</a>), rice (70%; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Till <em>et al.</em> 2007</a>), and tomato (60%; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Minoia <em>et al.</em> 2010</a>), showing a possibility of the presence of various base changes upon EMS exposure. However, transition mutations could not be denied as the most common base mutation in the EMS sphere as reported previously (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lawley and Martin 1975</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Sikora <em>et al.</em> 2011</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Shiwa <em>et al.</em> 2012</a>). In the TLLING analysis, a number of interesting genes with known sequences were selected for the identification of base changes in pooled mutated genomes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McCallum <em>et al.</em> 2000</a>). In this study, we adopted a high-throughput technology (2GS) to scan homozygous SNPs that are present throughout the individual mutated genomes compared with our resequenced WT MG-20. This provided a wider range of EMS effects in a genome. Our approach also neglected natural variation between the reference and the resequenced MG-20 genomes. Our reference was used merely for read mapping and has not been counted during SNP calling (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Lorenc <em>et al.</em> 2012</a>).</p><p id="p-35">Although the percentage of transversion mutations was relatively low in both genomes, they should not be omitted in a mutation study because they potentially are causal mutations for phenotypes of interest. Taking into account only homozygous mutations, our data also show that the presence of specific base changes is comparatively distributed among individual chromosomes and not abundantly located in a specific chromosome for both alleles. Comparing these two genomes, we found that transition and transversion mutations were present nearly consistently in the same chromosomes. These results reveal that the frequency of EMS-induced transition and transversion mutations is comparable between different individual mutants that were derived from the same mutagenized population.</p><p id="p-36">As mentioned previously, the total assembled length affects the SNP total identified in each chromosome and consequently, their distributions. The longer the assembled length, the denser the mutation distribution was found to be along the chromosome. The mutation distribution also was affected by the occurrence of assembled sequences at specific regions (<a href="http://www.kazusa.or.jp/lotus/">www.kazusa.or.jp/lotus/</a>). Chromosome 6 is the best example, in which low SNP peaks were detected and the mutations distributed broadly. Read mapping is very difficult in centromeric regions, and therefore the density in these regions has not been investigated in this study. Distribution of mutations was scattered due to the effect of EMS as a mutagen that causes a random mutation (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lawley and Martin 1975</a>; <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Greene <em>et al.</em> 2003</a>; <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Sikora <em>et al.</em> 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tagu <em>et al.</em> 2014</a>). A random distribution also was reported by <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Thompson <em>et al.</em> (2013)</a> in the genome of single <em>C. elegans</em> from different strains, which were mutagenized by EMS. They propagated mutagenized worms through 10 generations to obtain stable mutants. However, this is time-consuming for legumes, which have a longer generation period. Nevertheless, our 2GS data of third-generation mutants also could provide the literal effect of EMS mutagenesis throughout the individual genomes.</p><p id="p-37">EMS induces base changes or nucleotide substitution, which consequently alter codon sequences, leading to either nonsynonymous or synonymous effects. In genetics studies, nonsynonymous change is a favorable mutation effect because it gives a clue on which gene may be associated with a specific phenotype (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ng and Henikoff 2006</a>). Our results showed that EMS has arbitrary and broad effects of mutation on codon sequences. Mutated coding regions could represent nonsynonymous changes, which are useful to identify gene of interest for desirable phenotypes.</p><p id="p-38">To extend the implication of our data, preliminary work has been employed to specifically detect the presence of mutations in our genes of interest. A total of 32 genes reported to be involved in ABA signaling were selected as candidate genes. These genes are commonly reported in the ABA−gene interaction in ABA perception and signaling pathways (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wasilewska <em>et al.</em> 2008</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cutler <em>et al.</em> 2010</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 2010</a>). We chose to use ABA candidate genes because our mutants were isolated from a mutagenized population effected in response to ABA. We did identify base changes in several candidate genes, indicating a low rate of EMS effects on these sequences. In addition, identified SNPs were not only in the ABA-insensitive AM mutant, indicating they were background mutations. On the other hand, comparing mutations between AM and AS mutants would provide clues on the causal mutation of mutant phenotypes. However, other candidate genes also can be considered because ABA is involved in a wide range of plant growth and responses to environmental stresses. Its effects are varied in regulating the events of plant physiology, which require gene interaction and/or cross talks with other hormones (as reviewed in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kermode 2005</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fujita <em>et al.</em> 2006</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rock <em>et al.</em> 2010</a>). Because this research worked on an individual genome of mutants, another strategy has been developed to sequence pooled DNA from different mutants to remove the effect of background mutation and increase the likelihood of identifying the causative gene.</p><p id="p-39">In conclusion, this scanning revealed a detailed effect of EMS mutation on the whole genome of an individual mutant. Our results presented an overview of point mutations that occurred in the genome of mutants, which were usually pooled to identify SNPs. Here, EMS has produced a number of abnormal plants in our mutant population of <em>L. japonicus</em>. Our 2GS data also revealed how EMS efficiently mutates genomic sequences in an individual mutagenized plant. As expected, EMS created a random spectrum of mutation across the whole genome of our mutants and biased to G/C-to-A/T changes. However, other transition and transversion mutations also were identified with quite apparent fractions. Calling only homozygous SNPs has put a high confidence in identified base changes occurred. Mutation distribution apparently is distinct between mutated genomes, which derived from the same mutagenised population. Effects of SNPs on coding and noncoding regions could be manipulated to identify a causal mutation of a phenotype of interest. Our next question will be which gene is responsible for our abnormal phenotype of interest. This 2GS data will be further analyzed to discover the causative mutated gene.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank the Centre for Integrative Legume Research, the University of Queensland for giving the opportunity to run this work. Special thanks to Dr Stephen Kazakoff, Queensland Centre for Medical Genomics, the University of Queensland for the technical support. A scholarship from the Ministry of Higher Education, Malaysia, in financially supporting our student also is acknowledged.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-41"><em>Communicating editor: N. D. Young</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-30"><span class="received-label">Received </span>October 30, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-02"><span class="accepted-label">Accepted </span>February 2, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Mohd-Yusoff <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.1" data-doi="10.1186/gb-2011-12-3-r28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashelford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Amore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full genome re-sequencing reveals a novel circadian clock mutation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAshelford%26rft.auinit1%253DK.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253DR28%26rft.epage%253DR28%26rft.atitle%253DFull%2Bgenome%2Bre-sequencing%2Breveals%2Ba%2Bnovel%2Bcircadian%2Bclock%2Bmutation%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-3-r28%26rft_id%253Dinfo%253Apmid%252F21429190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-3-r28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429190&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.2" data-doi="10.1111/j.1365-313X.2011.04619.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidaurre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatiou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breit</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Provart</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation mapping of Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DAustin%26rft.auinit1%253DR.%2BS.%26rft.volume%253D67%26rft.issue%253D4%26rft.spage%253D715%26rft.epage%253D725%26rft.atitle%253DNext-generation%2Bmapping%2Bof%2BArabidopsis%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04619.x%26rft_id%253Dinfo%253Apmid%252F21518053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04619.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21518053&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293753800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.3" data-doi="10.1093/mp/ssp009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresshoff</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A novel ABA insensitive mutant of <em>Lotus japonicus</em> with a wilty phenotype displays unaltered nodulation regulation.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssp009%26rft_id%253Dinfo%253Apmid%252F19825632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssp009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19825632&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266353600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.4" data-doi="10.1111/j.1365-313X.2004.02190.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A structured mutant population for forward and reverse genetics in barley (<em>Hordeum vulgare</em> L.).</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DCaldwell%26rft.auinit1%253DD.%2BG.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D150%26rft.atitle%253DA%2Bstructured%2Bmutant%2Bpopulation%2Bfor%2Bforward%2Band%2Breverse%2Bgenetics%2Bin%2BBarley%2B%2528Hordeum%2Bvulgare%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2004.02190.x%26rft_id%253Dinfo%253Apmid%252F15361148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2004.02190.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15361148&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223816300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.5" data-doi="10.1016/0168-9452(86)90057-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresshoff</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Mutagenesis of soybean (<em>Glycine max</em> (L.) Merr) and the isolation of non-nodulating mutants.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D47%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9452%252886%252990057-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9452(86)90057-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986F882600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.6" data-doi="10.1111/jipb.12040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresshoff</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Classical ethylene insensitive mutants of the <em>Arabidopsis EIN2</em> orthologue lack the expected ‘hypernodulation’ response in <em>Lotus japonicus</em>.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D55%26rft.spage%253D395%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjipb.12040%26rft_id%253Dinfo%253Apmid%252F23452324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jipb.12040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23452324&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain <em>w<sup>1118</sup></em>; <em>iso</em>-2; <em>iso</em>-3.</span> <abbr class="cit-jnl-abbrev">Landes Bioscience</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLandes%2BBioscience%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.8" data-doi="10.1146/annurev-arplant-042809-112122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkelstein</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrams</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Abscisic acid: emergence of a core signalling network.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DCutler%26rft.auinit1%253DS.%2BR.%26rft.volume%253D61%26rft.spage%253D651%26rft.epage%253D679%26rft.atitle%253DAbscisic%2Bacid%253A%2Bemergence%2Bof%2Ba%2Bcore%2Bsignaling%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-042809-112122%26rft_id%253Dinfo%253Apmid%252F20192755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-042809-112122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20192755&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278924400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.9" data-doi="10.1007/s10142-014-0391-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalton-Morgan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alamery</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollenaere</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-throughput SNP array in the amphidiploid species <em>Brassica napus</em> shows diversity in resistance genes.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BIntegr.%2BGenomics%26rft.volume%253D14%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-014-0391-2%26rft_id%253Dinfo%253Apmid%252F25147024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-014-0391-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25147024&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dube</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bajaj</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gawande</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutagenic efficiency and effectiveness of gamma rays and EMS in <em>Cyamopsis tetragonoloba</em> (L.) var. <em>Sharada</em>.</span> <abbr class="cit-jnl-abbrev">Asiatic J. Biotechnol. Resour.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsiatic%2BJ.%2BBiotechnol.%2BResour.%26rft.volume%253D2%26rft.spage%253D436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.11" data-doi="10.1007/s00122-012-1964-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accessing complex crop genomes with next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1964-x%26rft_id%253Dinfo%253Apmid%252F22948437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1964-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22948437&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.12" data-doi="10.1534/genetics.110.116616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing profiling of mutagenesis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116616%26rft_id%253Dinfo%253Apmid%252F20439774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.13" data-doi="10.1016/j.pbi.2006.05.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noutoshi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narusaka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Crosstalk between abiotic and biotic stress responses: a current view from the points of convergence in the stress signaling networks.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DChittka%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D436%26rft.epage%253D442%26rft.atitle%253DCrosstalk%2Bbetween%2Babiotic%2Band%2Bbiotic%2Bstress%2Bresponses%253A%2Ba%2Bcurrent%2Bview%2Bfrom%2Bthe%2Bpoints%2Bof%2Bconvergence%2Bin%2Bthe%2Bstress%2Bsignaling%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2006.05.014%26rft_id%253Dinfo%253Apmid%252F16759898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2006.05.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16759898&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239374600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.14" data-doi="10.1093/nar/gkr944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Phytozome: a comparative platform for green plant genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1178</span>–<span class="cit-lpage">D1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr944%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC9EMS9EMTE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Spectrum of chemically induced mutations from a large-scale reverse genetic screen in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGreene%26rft.auinit1%253DE.%2BA.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D731%26rft.epage%253D740%26rft.atitle%253DSpectrum%2Bof%2BChemically%2BInduced%2BMutations%2BFrom%2Ba%2BLarge-Scale%2BReverse-Genetic%2BScreen%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Apmid%252F12807792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.16" data-doi="10.1104/pp.112.200311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartwig</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konrad</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turck</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast isogenic mapping-by-sequencing of ethyl methanesulfonate-induced mutant bulks.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.200311%26rft_id%253Dinfo%253Apmid%252F22837357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjAvMi81OTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.4.559.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haughn</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Selection for herbicide resistance at the whole-plant level</span>, pp. <span class="cit-fpage">98</span>–<span class="cit-lpage">107</span> in <span class="cit-source"><em>Biotechnology in Agricultural Chemistry</em>, <em>ACS Symposium Series</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">LeBron</span>  <span class="cit-name-given-names">H.</span></span> <em>et al</em>. <span class="cit-publ-name">American Chemical Society</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.18" data-doi="10.1093/dnares/7.6.323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Complete structure of the chloroplast genome of a legume, <em>Lotus japonicus</em>.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BResearch%26rft.stitle%253DDNA%2BRes%26rft.aulast%253DKato%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DComplete%2BStructure%2Bof%2Bthe%2BChloroplast%2BGenome%2Bof%2Ba%2BLegume%252C%2BLotus%2Bjaponicus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252F7.6.323%26rft_id%253Dinfo%253Apmid%252F11214967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjc6IjcvNi8zMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.19" data-doi="10.1371/journal.pone.0051687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kazakoff</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imelfort</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koehorst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Capturing the biofuel wellhead and powerhouse: the chloroplast and mitochondrial genomes of the leguminous feedstock tree <em>Pongamia pinnata</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e51687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De51687%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051687%26rft_id%253Dinfo%253Apmid%252F23272141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0051687&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23272141&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.20" data-doi="10.1007/s00344-005-0110-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kermode</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Role of abscisic acid in seed dormancy.</span> <abbr class="cit-jnl-abbrev">J. Plant Growth Regul.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BGrowth%2BRegul.%26rft.volume%253D24%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00344-005-0110-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00344-005-0110-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234299100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.21" data-doi="10.1146/annurev-arplant-042809-112226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Böhmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Guard cell signal transduction network: advances in understanding abscisic acid, CO<sub>2</sub> and Ca<sup>2+</sup> signalling.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DKim%26rft.auinit1%253DT.%2BH.%26rft.volume%253D61%26rft.spage%253D561%26rft.epage%253D591%26rft.atitle%253DGuard%2Bcell%2Bsignal%2Btransduction%2Bnetwork%253A%2Badvances%2Bin%2Bunderstanding%2Babscisic%2Bacid%252C%2BCO2%252C%2Band%2BCa2%252B%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-042809-112226%26rft_id%253Dinfo%253Apmid%252F20192751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-042809-112226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20192751&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278924400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.22" data-doi="10.1111/pbi.12240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berkman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification and characterisation of more than 4 million intervarietal SNPs across the group 7 chromosomes of bread wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12240%26rft_id%253Dinfo%253Apmid%252F25147022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25147022&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.23" data-doi="10.1042/bj1450085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Molecular mechanisms in alkylation mutagenesis. Induced reversion of bacteriophage T4rII AP72 by ethyl methanesulphonate in relation to extent and mode of ethylation of purines in bacteriophage deoxyribonucleic acid.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DLawley%26rft.auinit1%253DP.%2BD.%26rft.volume%253D145%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D91%26rft.atitle%253DMolecular%2Bmechanisms%2Bin%2Balkylation%2Bmutagenesis.%2BInduced%2Breversion%2Bof%2Bbacteriophage%2BT4rII%2BAP72%2Bby%2Bethyl%2Bmethanesulphonate%2Bin%2Brelation%2Bto%2Bextent%2Band%2Bmode%2Bof%2Bethylation%2Bof%2Bpurines%2Bin%2Bbacteriophage%2Bdeoxyribonucleic%2Bacid%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj1450085%26rft_id%253Dinfo%253Apmid%252F172067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6ODoiMTQ1LzEvODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.24" data-doi="10.1093/nar/gkr939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">P. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">LegumeIP: an integrative database for comparative genomics and transcriptomics of model legumes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1221</span>–<span class="cit-lpage">D1229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr939%26rft_id%253Dinfo%253Apmid%252F22110036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC9EMS9EMTIyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.25" data-doi="10.1093/bioinformatics/btp336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yiu</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SOAP2: an improved ultrafast tool for short read alignment.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1966</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp336%26rft_id%253Dinfo%253Apmid%252F19497933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.26" data-doi="10.3390/biology1020370"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of single nucleotide polymorphisms in complex genomes using SGSautoSNP.</span> <abbr class="cit-jnl-abbrev">Biology</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">370</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%26rft.volume%253D1%26rft.spage%253D370%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fbiology1020370%26rft_id%253Dinfo%253Apmid%252F24832230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/biology1020370&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24832230&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.27" data-doi="10.1038/1811212a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loveless</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Increased rate of plaque-type and host-range mutation following treatment of bacteriophage <em>in vitro</em> with ethyl methane sulphonate.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLoveless%26rft.auinit1%253DA.%26rft.volume%253D181%26rft.issue%253D4617%26rft.spage%253D1212%26rft.epage%253D1213%26rft.atitle%253DIncreased%2Brate%2Bof%2Bplaque-type%2Band%2Bhost-range%2Bmutation%2Bfollowing%2Btreatment%2Bof%2Bbacteriophage%2Bin%2Bvitro%2Bwith%2Bethyl%2Bmethane%2Bsulphonate.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F1811212a0%26rft_id%253Dinfo%253Apmid%252F13541432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/1811212a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13541432&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.28" data-doi="10.1104/pp.123.2.439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Targeting induced local lesions in genomes (TILLING) for plant functional genomics.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DMcCallum%26rft.auinit1%253DC.%2BM.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D439%26rft.epage%253D442%26rft.atitle%253DTargeting%2BInduced%2BLocal%2BLesions%2BIN%2BGenomes%2B%2528TILLING%2529%2Bfor%2BPlant%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.123.2.439%26rft_id%253Dinfo%253Apmid%252F10859174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjMvMi80MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.29" data-doi="10.1186/1756-0500-3-69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minoia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrozza</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Onofrio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piron</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosca</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A new mutant genetic resource for tomato crop improvement by TILLING technology.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DMinoia%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.spage%253D69%26rft.epage%253D69%26rft.atitle%253DA%2Bnew%2Bmutant%2Bgenetic%2Bresource%2Bfor%2Btomato%2Bcrop%2Bimprovement%2Bby%2BTILLING%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-3-69%26rft_id%253Dinfo%253Apmid%252F20222995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-3-69&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20222995&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.30" data-doi="10.1186/1471-2164-12-256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mokry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijman</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dijken</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heidstra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of factors required for meristem function in <em>Arabidopsis</em> using a novel next generation sequencing fast forward genetics approach.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMokry%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.spage%253D256%26rft.epage%253D256%26rft.atitle%253DIdentification%2Bof%2Bfactors%2Brequired%2Bfor%2Bmeristem%2Bfunction%2Bin%2BArabidopsis%2Busing%2Ba%2Bnovel%2Bnext%2Bgeneration%2Bsequencing%2Bfast%2Bforward%2Bgenetics%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-256%26rft_id%253Dinfo%253Apmid%252F21599977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21599977&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.31" data-doi="10.1146/annurev.genom.7.080505.115630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting the effects of amino acid substitutions on protein function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.spage%253D61%26rft.epage%253D80%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bamino%2Bacid%2Bsubstitutions%2Bon%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.7.080505.115630%26rft_id%253Dinfo%253Apmid%252F16824020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.7.080505.115630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16824020&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.32" data-doi="10.1073/pnas.90.17.8043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Wettstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Sodium azide mutagenesis: preferential generation of A·T → G·C transitions in the barley <em>An18</em> gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">8043</span>–<span class="cit-lpage">8047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOlsen%26rft.auinit1%253DO.%26rft.volume%253D90%26rft.issue%253D17%26rft.spage%253D8043%26rft.epage%253D8047%26rft.atitle%253DSodium%2Bazide%2Bmutagenesis%253A%2Bpreferential%2Bgeneration%2Bof%2BA.T--%253EG.C%2Btransitions%2Bin%2Bthe%2Bbarley%2BAnt18%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.17.8043%26rft_id%253Dinfo%253Apmid%252F8367460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTAvMTcvODA0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.33" data-doi="10.1104/pp.102.017384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welham</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pike</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A TILLING reverse genetics tool and a web-accessible collection of mutants of the legume <em>Lotus japonicus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">866</span>–<span class="cit-lpage">871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DPerry%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D3%26rft.spage%253D866%26rft.epage%253D871%26rft.atitle%253DA%2BTILLING%2BReverse%2BGenetics%2BTool%2Band%2Ba%2BWeb-Accessible%2BCollection%2Bof%2BMutants%2Bof%2Bthe%2BLegume%2BLotus%2Bjaponicus%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.102.017384%26rft_id%253Dinfo%253Apmid%252F12644638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzEvMy84NjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.34" data-doi="10.1104/pp.109.142190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welham</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TILLING in <em>Lotus japonicus</em> identified large allelic series for symbiosis genes and revealed a bias in functionally defective ethyl methanesulfonate alleles toward glycine replacements.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.109.142190%26rft_id%253Dinfo%253Apmid%252F19641028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTUxLzMvMTI4MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNC81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.559.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rock</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quatrano</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Stress signaling I: The role of abscisic acid (ABA)</span>, pp. <span class="cit-fpage">33</span>–<span class="cit-lpage">73</span> in <span class="cit-source"><em>Abiotic Stress Adaptation in Plants: Physiological</em>, <em>Molecular and Genomic Foundation</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Pareek</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sopory</span>  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bohnert</span>  <span class="cit-name-given-names">H. J.</span></span>and <span class="cit-ed"><span class="cit-name-surname">Govindjee</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dortrecht</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-5.4.559.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russel</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source">Molecular Cloning: A Laboratory Manual</span>, <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.37" data-doi="10.1016/j.pbi.2006.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>Lotus japonicus</em> as a platform for legume research.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D9%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2006.01.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2006.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000236167100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.38" data-doi="10.1093/dnares/dsn008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome structure of the legume, <em>Lotus japonicus</em>.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsn008%26rft_id%253Dinfo%253Apmid%252F18511435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE1LzQvMjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.39" data-doi="10.1016/0165-1218(84)90033-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sega</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A review of the genetic effects of ethyl methanesulfonate.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DSega%26rft.auinit1%253DG.%2BA.%26rft.volume%253D134%26rft.issue%253D2-3%26rft.spage%253D113%26rft.epage%253D142%26rft.atitle%253DA%2Breview%2Bof%2Bthe%2Bgenetic%2Beffects%2Bof%2Bethyl%2Bmethanesulfonate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0165-1218%252884%252990033-8%26rft_id%253Dinfo%253Apmid%252F6390190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0165-1218(84)90033-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6390190&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TX05100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.40" data-doi="10.1186/1746-4811-10-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serrat</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esteban</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guibourt</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moysset</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogués</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">EMS mutagenesis in mature seed-derived rice calli as a new method for rapidly obtaining TILLING mutant populations.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D10%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-10-5%26rft_id%253Dinfo%253Apmid%252F24475756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-10-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24475756&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiwa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukushima-Tanaka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasahara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horiuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole-genome profiling of a novel mutagenesis technique using proofreading-deficient DNA polymerase δ.</span> <abbr class="cit-jnl-abbrev">Int. J. Evol. Biol.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">860797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BEvol%2BBiol%26rft.aulast%253DShiwa%26rft.auinit1%253DY.%26rft.volume%253D2012%26rft.spage%253D860797%26rft.epage%253D860797%26rft.atitle%253DWhole-Genome%2BProfiling%2Bof%2Ba%2BNovel%2BMutagenesis%2BTechnique%2BUsing%2BProofreading-Deficient%2BDNA%2BPolymerase%2B%25CE%25B4.%26rft_id%253Dinfo%253Apmid%252F22675654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22675654&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.559.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikora</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chawade</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutagenesis as a tool in plant genetics, functional genomics and breeding.</span> <span class="cit-source">Int. J. Plant Genom.</span> <span class="cit-vol">2011</span>: <span class="cit-fpage">314829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BGenom.%26rft.volume%253D2011%26rft.spage%253D314829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sri Devi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullainathan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The use of ethyl methanesulfonate to study the flower development in <em>Capsicum annuum</em> L. mutants.</span> <abbr class="cit-jnl-abbrev">Bot. Res. Intl.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BRes.%2BIntl.%26rft.volume%253D5%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DStewart%26rft.auinit1%253DC.%2BN.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D748%26rft.epage%253D750%26rft.atitle%253DA%2Brapid%2BCTAB%2BDNA%2Bisolation%2Btechnique%2Buseful%2Bfor%2BRAPD%2Bfingerprinting%2Band%2Bother%2BPCR%2Bapplications.%26rft_id%253Dinfo%253Apmid%252F8512694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8512694&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LA81200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-other" id="cit-5.4.559.45" data-doi="10.1534/g3.113.009639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tagu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trionnaire</span>  <span class="cit-name-given-names">G.L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanguy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">J.P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">J.R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">EMS mutagenesis in the pea aphid <em>Acyrthosiphon pisum</em>.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009639%26rft_id%253Dinfo%253Apmid%252F24531730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNC82NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.46" data-doi="10.1101/gr.157651.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strasbourger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The million mutation project: a new approach to genetics in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.157651.113%26rft_id%253Dinfo%253Apmid%252F23800452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.47" data-doi="10.1093/bfgp/elr045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Current state-of-art of sequencing technologies for plant genomics research.</span> <abbr class="cit-jnl-abbrev">Brief Funct. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felr045%26rft_id%253Dinfo%253Apmid%252F22345601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo2OiIxMS8xLzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzQvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.48" data-doi="10.1186/1471-2229-7-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ti</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colowit</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Discovery of chemically induced mutations in rice by TILLING.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D7%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DDiscovery%2Bof%2Bchemically%2Binduced%2Bmutations%2Bin%2Brice%2Bby%2BTILLING.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-7-19%26rft_id%253Dinfo%253Apmid%252F17428339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-7-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17428339&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enns</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Large scale discovery of induced point mutations with high-throughput TILLING.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft.volume%253D13%26rft.spage%253D524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.50" data-doi="10.1007/s12038-012-9228-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudapa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thudi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Advances in genetics and molecular breeding of three legume crops of semi-arid tropics using next-generation sequencing and high-throughput genotyping technologies.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DVarshney%26rft.auinit1%253DR.%2BK.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D811%26rft.epage%253D820%26rft.atitle%253DAdvances%2Bin%2Bgenetics%2Band%2Bmolecular%2Bbreeding%2Bof%2Bthree%2Blegume%2Bcrops%2Bof%2Bsemi-arid%2Btropics%2Busing%2B%2Bnext-generation%2Bsequencing%2Band%2Bhigh-throughput%2Bgenotyping%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-012-9228-0%26rft_id%253Dinfo%253Apmid%252F23107917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-012-9228-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23107917&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.51" data-doi="10.1093/mp/ssm022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasilewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlad</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirichandra</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redko</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jammes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An update on abscisic acid signalling in plants and more...</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssm022%26rft_id%253Dinfo%253Apmid%252F19825533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssm022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19825533&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259103700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.52" data-doi="10.1186/1471-2229-8-103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkley</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Applying genotyping (TILLING) and phenotyping analyses to elucidate gene function in a chemically induced sorghum mutant population.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DXin%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.spage%253D103%26rft.epage%253D103%26rft.atitle%253DApplying%2Bgenotyping%2B%2528TILLING%2529%2Band%2Bphenotyping%2Banalyses%2Bto%2Belucidate%2Bgene%2Bfunction%2Bin%2Ba%2Bchemically%2Binduced%2Bsorghum%2Bmutant%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-8-103%26rft_id%253Dinfo%253Apmid%252F18854043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-8-103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854043&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.53" data-doi="10.1007/s10142-013-0324-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zander</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Wouw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenc</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identifying genetic diversity of avirulence genes in <em>Leptosphaeria maculans</em> using whole genome sequencing.</span> <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BIntegr.%2BGenomics%26rft.volume%253D13%26rft.spage%253D295%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-013-0324-5%26rft_id%253Dinfo%253Apmid%252F23793572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-013-0324-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793572&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F4%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.54" data-doi="10.1534/genetics.112.141986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hipps</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene discovery using mutagen-induced polymorphisms and deep sequencing: application to plant disease resistance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141986%26rft_id%253Dinfo%253Apmid%252F22714407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.4.559.55" data-doi="10.1534/genetics.110.119230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuryn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gras</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamet</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A strategy for direct mapping and identification of mutations by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119230%26rft_id%253Dinfo%253Apmid%252F20610404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvNDI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS80LzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/4/559.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/4/551" title="Comparison of Larval and Adult Drosophila Astrocytes Reveals Stage-Specific Gene Expression Profiles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/4/569" title="Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/4"><span>Volume 5 Issue 4, April 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/4.cover-source.jpg?itok=BjlNQ3MJ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/4/559" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DlYvEZLaj-chgYBBMQTFvl2ckN7DGyGVJm0W4KXsK8Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/4/559" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/4/559" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/4/559" class="active" data-icon-position="" data-hide-link-title="0">Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of Lotus japonicus Using Second-Generation Sequencing Analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459805" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tChbvhJ6LZMa5pygzs1abuPQNC3B3OuP9aph0zbchJI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/559.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/4/559" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-mDkY4mju4od_fC3MXqKQnyx2bqfD5LUFe_XAGoQkUg8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/4/559" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/4/559.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459805" id="node459805--2" data-pisa="ggg;5/4/559" data-pisa-master="ggg;g3.114.014571" data-apath="/ggg/5/4/559.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of <em>Lotus japonicus</em> Using Second-Generation Sequencing Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nur Fatihah Mohd-Yusoff</span>, <span class="highwire-citation-author" data-delta="1">Pradeep Ruperao</span>, <span class="highwire-citation-author" data-delta="2">Nurain Emylia Tomoyoshi</span>, <span class="highwire-citation-author" data-delta="3">David Edwards</span>, <span class="highwire-citation-author" data-delta="4">Peter M. Gresshoff</span>, <span class="highwire-citation-author" data-delta="5">Bandana Biswas</span> and <span class="highwire-citation-author" data-delta="6">Jacqueline Batley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">559-567;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014571 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459805--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nur Fatihah Mohd-Yusoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cell and Molecular Biology, Faculty of Biotechnology and Biomolecular Sciences, Universiti Putra Malaysia, 43400, Serdang, Selangor, Malaysia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nur%2BFatihah%2BMohd-Yusoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohd-Yusoff%20NF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANur%2BFatihah%2BMohd-Yusoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pradeep Ruperao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pradeep%2BRuperao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruperao%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APradeep%2BRuperao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nurain Emylia Tomoyoshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nurain%2BEmylia%2BTomoyoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tomoyoshi%20NE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANurain%2BEmylia%2BTomoyoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter M. Gresshoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BGresshoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresshoff%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BM.%2BGresshoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.gresshoff@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">p.gresshoff@uq.edu.au</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bandana Biswas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bandana%2BBiswas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Biswas%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABandana%2BBiswas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jacqueline Batley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BBatley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batley%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BBatley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/4/559" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/4/559&amp;title=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/4/559&amp;title=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/4/559&amp;title=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/4/559&amp;text=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/4/559&amp;title=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/4/559&amp;t=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/4/559&amp;title=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/4/559&amp;title=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459805/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459805" id="node459805--3" data-pisa="ggg;5/4/559" data-pisa-master="ggg;g3.114.014571" data-apath="/ggg/5/4/559.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Scanning the Effects of Ethyl Methanesulfonate on the Whole Genome of <em>Lotus japonicus</em> Using Second-Generation Sequencing Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nur Fatihah Mohd-Yusoff</span>, <span class="highwire-citation-author" data-delta="1">Pradeep Ruperao</span>, <span class="highwire-citation-author" data-delta="2">Nurain Emylia Tomoyoshi</span>, <span class="highwire-citation-author" data-delta="3">David Edwards</span>, <span class="highwire-citation-author" data-delta="4">Peter M. Gresshoff</span>, <span class="highwire-citation-author" data-delta="5">Bandana Biswas</span> and <span class="highwire-citation-author" data-delta="6">Jacqueline Batley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">559-567;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014571 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459805--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nur Fatihah Mohd-Yusoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cell and Molecular Biology, Faculty of Biotechnology and Biomolecular Sciences, Universiti Putra Malaysia, 43400, Serdang, Selangor, Malaysia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nur%2BFatihah%2BMohd-Yusoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohd-Yusoff%20NF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANur%2BFatihah%2BMohd-Yusoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pradeep Ruperao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pradeep%2BRuperao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruperao%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APradeep%2BRuperao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nurain Emylia Tomoyoshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nurain%2BEmylia%2BTomoyoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tomoyoshi%20NE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANurain%2BEmylia%2BTomoyoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Australian Centre for Plant Functional Genomics, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter M. Gresshoff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BGresshoff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresshoff%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BM.%2BGresshoff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.gresshoff@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">p.gresshoff@uq.edu.au</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bandana Biswas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bandana%2BBiswas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Biswas%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABandana%2BBiswas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jacqueline Batley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre of Integrative Legume Research, School of Agriculture and Food Science, The University of Queensland, St Lucia, Brisbane QLD 4072, Australia</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Plant Biology, University of Western Australia, Crawley, WA 6009, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BBatley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batley%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BBatley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=559&amp;rft.epage=567&amp;rft.atitle=Scanning%20the%20Effects%20of%20Ethyl%20Methanesulfonate%20on%20the%20Whole%20Genome%20of%20Lotus%20japonicus%20Using%20Second-Generation%20Sequencing%20Analysis&amp;rft.volume=5&amp;rft.issue=4&amp;rft.date=2015-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mohd-Yusoff%2C+Nur+Fatihah&amp;rft.au=Ruperao%2C+Pradeep&amp;rft.au=Tomoyoshi%2C+Nurain+Emylia&amp;rft.au=Edwards%2C+David&amp;rft.au=Gresshoff%2C+Peter+M.&amp;rft.au=Biswas%2C+Bandana&amp;rft.au=Batley%2C+Jacqueline"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459805/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459805/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459805/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459805/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459805/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459805/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459805/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459805/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459805/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459805/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459805/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459805/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/4/559" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/4/559.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/4/559.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/4/559.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
