<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/6/1027" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/6/1027.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/6/1027.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/6/1027.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/6/1027/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/6/1027.atom" />
<meta name="HW.pisa" content="ggg;4/6/1027" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures" />
<meta name="DC.Identifier" content="10.1534/g3.114.010298" />
<meta name="DC.Date" content="2014-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Howard et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Parametric and nonparametric methods have been developed for purposes of predicting phenotypes. These methods are based on retrospective analyses of empirical data consisting of genotypic and phenotypic scores. Recent reports have indicated that parametric methods are unable to predict phenotypes of traits with known epistatic genetic architectures. Herein, we review parametric methods including least squares regression, ridge regression, Bayesian ridge regression, least absolute shrinkage and selection operator (LASSO), Bayesian LASSO, best linear unbiased prediction (BLUP), Bayes A, Bayes B, Bayes C, and Bayes C π . We also review nonparametric methods including Nadaraya-Watson estimator, reproducing kernel Hilbert space, support vector machine regression, and neural networks. We assess the relative merits of these 14 methods in terms of accuracy and mean squared error (MSE) using simulated genetic architectures consisting of completely additive or two-way epistatic interactions in an F 2 population derived from crosses of inbred lines. Each simulated genetic architecture explained either 30% or 70% of the phenotypic variability. The greatest impact on estimates of accuracy and MSE was due to genetic architecture. Parametric methods were unable to predict phenotypic values when the underlying genetic architecture was based entirely on epistasis. Parametric methods were slightly better than nonparametric methods for additive genetic architectures. Distinctions among parametric methods for additive genetic architectures were incremental. Heritability, i.e. , proportion of phenotypic variability, had the second greatest impact on estimates of accuracy and MSE." />
<meta name="DC.Contributor" content="Réka Howard" />
<meta name="DC.Contributor" content="Alicia L. Carriquiry" />
<meta name="DC.Contributor" content="William D. Beavis" />
<meta name="article:published_time" content="2014-06-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Parametric and nonparametric methods have been developed for purposes of predicting phenotypes. These methods are based on retrospective analyses of empirical data consisting of genotypic and phenotypic scores. Recent reports have indicated that parametric methods are unable to predict phenotypes of traits with known epistatic genetic architectures. Herein, we review parametric methods including least squares regression, ridge regression, Bayesian ridge regression, least absolute shrinkage and selection operator (LASSO), Bayesian LASSO, best linear unbiased prediction (BLUP), Bayes A, Bayes B, Bayes C, and Bayes C&lt;i&gt;π&lt;/i&gt;. We also review nonparametric methods including Nadaraya-Watson estimator, reproducing kernel Hilbert space, support vector machine regression, and neural networks. We assess the relative merits of these 14 methods in terms of accuracy and mean squared error (MSE) using simulated genetic architectures consisting of completely additive or two-way epistatic interactions in an &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;2&lt;/sub&gt; population derived from crosses of inbred lines. Each simulated genetic architecture explained either 30% or 70% of the phenotypic variability. The greatest impact on estimates of accuracy and MSE was due to genetic architecture. Parametric methods were unable to predict phenotypic values when the underlying genetic architecture was based entirely on epistasis. Parametric methods were slightly better than nonparametric methods for additive genetic architectures. Distinctions among parametric methods for additive genetic architectures were incremental. Heritability, &lt;i&gt;i.e.&lt;/i&gt;, proportion of phenotypic variability, had the second greatest impact on estimates of accuracy and MSE.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/06/01" />
<meta name="citation_mjid" content="ggg;4/6/1027" />
<meta name="citation_id" content="4/6/1027" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/6/1027" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/6/1027.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/6/1027.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/6/1027.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010298" />
<meta name="citation_pmid" content="24727289" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1027" />
<meta name="citation_lastpage" content="1046" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Réka Howard" />
<meta name="citation_author_institution" content="Department of Statistics, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author_institution" content="Department of Agronomy, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author_email" content="howardr@iastate.edu" />
<meta name="citation_author" content="Alicia L. Carriquiry" />
<meta name="citation_author_institution" content="Department of Statistics, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="William D. Beavis" />
<meta name="citation_author_institution" content="Department of Agronomy, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_reference" content="citation_title=Mind and Body: The Theories of Their Relation;citation_author=A. Bain;citation_year=1873" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Prediction of Maize Single-Cross Performance Using RFLPs and Information from Related Hybrids;citation_pages=20-25;citation_volume=34;citation_year=1994;citation_issue=1;citation_doi=10.2135/cropsci1994.0011183X003400010003x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Molecular Markers and Selection for Complex Traits in Plants: Learning from the Last 20 Years;citation_pages=1649-1664;citation_volume=48;citation_year=2008;citation_issue=5;citation_doi=10.2135/cropsci2008.03.0131" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Neural networks :  the official journal of the International Neural Network Society;citation_journal_abbrev=Neural Netw;citation_author=V. Cherkassky;citation_author=Y. Ma;citation_title=Practical selection of SVM parameters and noise estimation for SVM regression.;citation_pages=113-126;citation_volume=17;citation_year=2004;citation_issue=1;citation_pmid=14690712;citation_doi=10.1016/S0893-6080(03)00169-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Cheverud;citation_author=E. Routman;citation_title=Epistasis and its contribution to genetic variance components.;citation_pages=1455-1461;citation_volume=139;citation_year=1995;citation_issue=3;citation_pmid=7768453" />
<meta name="citation_reference" content="citation_journal_title=Antiquity;citation_author=R. M. Clark;citation_title=A calibration curve for radiocarbon dates.;citation_pages=251-266;citation_volume=49;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Cockerham;citation_title=AN EXTENSION OF THE CONCEPT OF PARTITIONING HEREDITARY VARIANCE FOR ANALYSIS OF COVARIANCES AMONG RELATIVES WHEN EPISTASIS IS PRESENT;citation_pages=859-882;citation_volume=39;citation_year=1954;citation_issue=6;citation_pmid=17247525" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=R. R. Corbeil;citation_author=S. R. Searle;citation_title=Restricted maximum likelihood (REML) estimation of variance components in the mixed model.;citation_pages=31-38;citation_volume=18;citation_year=1976;citation_doi=10.2307/1267913" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=H. J. Cordell;citation_title=Epistasis: what it means, what it doesn't mean, and statistical methods to detect it in humans;citation_pages=2463-2468;citation_volume=11;citation_year=2002;citation_issue=20;citation_pmid=12351582;citation_doi=10.1093/hmg/11.20.2463" />
<meta name="citation_reference" content="citation_journal_title=Machine Leaming;citation_author=C. Cortes;citation_author=V. Vapnik;citation_title=Support-vector networks.;citation_pages=273-297;citation_volume=20;citation_year=1995" />
<meta name="citation_reference" content="citation_title=An Introduction to Support Vector Machines and Other Kernel-based Learning Methods;citation_author=N. Cristianini;citation_author=J. Shawe-Taylor;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=J. F. Crow;citation_title=On epistasis: why it is unimportant in polygenic directional selection;citation_pages=1241-1244;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308099;citation_doi=10.1098/rstb.2009.0275" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Stat. Neerl.;citation_author=P. M. C. de Boer;citation_author=C. M. Hafner;citation_title=Ridge regression revisited.;citation_pages=498-505;citation_volume=59;citation_year=2005;citation_issue=4;citation_doi=10.1111/j.1467-9574.2005.00304.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigrees.;citation_volume=182;citation_year=2009;citation_issue=1;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=D. B. Allison;citation_title=Predicting genetic predisposition in humans: the promise of whole-genome markers.;citation_pages=880-886;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21045869;citation_doi=10.1038/nrg2898" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_title=Genetics of the Evolutionary Process;citation_author=T. Dobzhansky;citation_year=1937" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=B. Efron;citation_author=T. Hastie;citation_author=I. Johnstone;citation_author=R. Tibshirani;citation_title=Least angle regression.;citation_pages=407-499;citation_volume=32;citation_year=2004;citation_doi=10.1214/009053604000000067" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_title=GenSel—User manual for a portfolio of genomic selection related analyses. Animal Breeding and Genetics, Iowa State University, Ames.;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. L. Fernando;citation_title=Marker assisted selection using best linear unbiased prediction.;citation_pages=467-477;citation_volume=21;citation_year=1989;citation_doi=10.1051/gse:19890407" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The correlation between relatives on the supposition of Mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Flint;citation_author=T. F. C. Mackay;citation_title=Genetic architecture of quantitative traits in mice, flies, and humans;citation_pages=723-733;citation_volume=19;citation_year=2009;citation_issue=5;citation_pmid=19411597;citation_doi=10.1101/gr.086660.108" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_title=Bayesian Data Analysis;citation_author=A. Gelman;citation_author=J. B. Carlin;citation_author=H. S. Stern;citation_author=D. B. Rubin;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=E. I. George;citation_title=The variable selection problem.;citation_pages=1304-1308;citation_volume=95;citation_year=2000;citation_doi=10.2307/2669776" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=D. Gianola;citation_author=H. Okut;citation_author=K. A. Weigel;citation_author=G. J. M. Rosa;citation_title=Predicting complex quantitative traits with Bayesian neural networks: a case study with Jersey cows and wheat.;citation_pages=87-87;citation_volume=12;citation_year=2011;citation_pmid=21981731;citation_doi=10.1186/1471-2156-12-87" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. J. Goodnight;citation_title=Quantitative trait loci and gene interaction: the quantitative genetics of metapopulation.;citation_pages=587-598;citation_volume=84;citation_year=2000;citation_pmid=10849084;citation_doi=10.1046/j.1365-2540.2000.00698.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Breed;citation_author=B. Guo;citation_author=W. D. Beavis;citation_title=In silico genotyping of the maize nested association mapping population.;citation_pages=107-113;citation_volume=27;citation_year=2011;citation_issue=1;citation_pmid=21289856;citation_doi=10.1007/s11032-010-9503-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Guo;citation_author=D. Wang;citation_author=Z. Guo;citation_author=W. D. Beavis;citation_title=Family-based association mapping in crop species.;citation_pages=1419-1430;citation_volume=126;citation_year=2013;citation_pmid=23620001;citation_doi=10.1007/s00122-013-2100-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection using low-density marker panels.;citation_volume=182;citation_year=2009;citation_pmid=19299339;citation_doi=10.1534/genetics.108.100289" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F. R. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=The impact of genetic relationship information on genomic breeding values in German Holstein cattle.;citation_pages=5-5;citation_volume=42;citation_year=2010;citation_pmid=20170500;citation_doi=10.1186/1297-9686-42-5" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_title=Applied Nonparametric Regression;citation_author=W. Hardle;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=D. A. Harville;citation_title=Extension of the Gauss-Markov theorem to include estimation of random effects.;citation_pages=384;citation_volume=4;citation_year=1976;citation_doi=10.1214/aos/1176343414" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning: Data Mining, Inference, and Prediction;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=C. R. Henderson;citation_title=Estimates of changes in herd environment.;citation_pages=706;citation_volume=32;citation_year=1949" />
<meta name="citation_reference" content="citation_title=Estimation of Variance and Covariance Components. Biometrics;citation_author=C. R. Henderson;citation_pages=226-252;citation_volume=9;citation_year=1953;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_author=O. Kempthorne;citation_author=S. R. Searle;citation_author=C. M. von Krosigk;citation_title=The estimation of environmental and genetic trends from records subject to culling.;citation_pages=192;citation_volume=15;citation_year=1959;citation_doi=10.2307/2527669" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_pages=3135-3142;citation_volume=71;citation_issue=11" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: A comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Communications in Statistics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_author=K. F. Baldwin;citation_title=Ridge regression: some simulations.;citation_pages=105-123;citation_volume=4;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_title=Ridge regression: biased estimation for nonorthogonal problems.;citation_pages=55-67;citation_volume=12;citation_year=1970;citation_doi=10.2307/1267351" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_title=Ridge regression: applications to nonorthogonal problems.;citation_pages=69-82;citation_volume=12;citation_year=1970;citation_doi=10.2307/1267352" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Huang;citation_author=E. R. Angeles;citation_author=J. Domingo;citation_author=G. Magpantay;citation_author=S. Singh;citation_title=Pyramiding of bacterial blight resistance genes in rice: marker-assisted selection using RFLP and PCR.;citation_pages=313-320;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050565" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Huang;citation_author=S. Richards;citation_author=M. A. Carbone;citation_author=D. Zhu;citation_author=R. R. H. Anholt;citation_title=Epistasis dominates the genetic architecture of Drosophila quantitative traits.;citation_volume=109;citation_year=2012;citation_pmid=22949659;citation_doi=10.1073/pnas.1213423109" />
<meta name="citation_reference" content="citation_title=The Principles of Psychology;citation_author=W. James;citation_year=1890" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=A. Karatzoglou;citation_author=A. Smola;citation_author=K. Hornik;citation_author=A. Zeileis;citation_title=kernlab - An S4 Package for Kernel Methods in R.;citation_pages=1-20;citation_volume=11;citation_year=2004" />
<meta name="citation_reference" content="citation_title=Doing Bayesian Data Analysis: A Tutorial with R and BUGS;citation_author=J. K. Kruschke;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Neural networks :  the official journal of the International Neural Network Society;citation_journal_abbrev=Neural Netw;citation_author=J. Lampinen;citation_title=Bayesian approach for neural networks--review and case studies.;citation_pages=257-274;citation_volume=14;citation_year=2001;citation_issue=3;citation_pmid=11341565;citation_doi=10.1016/S0893-6080(00)00098-8" />
<meta name="citation_reference" content="citation_title=Solving Least Squares Problems;citation_author=C. Lawson;citation_author=R. Hansen;citation_year=1974" />
<meta name="citation_reference" content="citation_title=The Genetic Basis of Evolutionary Change;citation_author=R. Lewontin;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Long;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_title=Application of support vector regression to genome-assisted prediction of quantitative traits.;citation_pages=1065-1074;citation_volume=123;citation_year=2011;citation_issue=7;citation_pmid=21739137;citation_doi=10.1007/s00122-011-1648-y" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Maenhout;citation_author=B. De Baets;citation_author=G. Haesaert;citation_author=E. Van Bockstaele;citation_title=Support vector machine regression for the prediction of maize hybrid performance.;citation_pages=1003-1013;citation_volume=115;citation_year=2007;citation_issue=7;citation_pmid=17849095;citation_doi=10.1007/s00122-007-0627-9" />
<meta name="citation_reference" content="citation_title=Systematics and the Origin of Species;citation_author=E. Mayr;citation_year=1942" />
<meta name="citation_reference" content="citation_journal_title=Bull. Math. Biophys.;citation_author=W. McCulloch;citation_author=W. Pitts;citation_title=A logical calculus of ideas immanent in nervous activity.;citation_pages=115-133;citation_volume=5;citation_year=1943;citation_doi=10.1007/BF02478259" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. A. McGilchrist;citation_title=REML estimation for survival models with frailty.;citation_pages=221-225;citation_volume=49;citation_year=1993;citation_issue=1;citation_pmid=8513103;citation_doi=10.2307/2532615" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. E. Melchinger;citation_author=H. F. Utz;citation_author=C. C. Schon;citation_title=Quantitative Trait Locus (QTL) Mapping Using Different Testers and Independent Population Samples in Maize Reveals Low Power of QTL Detection and Large Bias in Estimates of QTL Effects;citation_pages=383-403;citation_volume=149;citation_year=1998;citation_issue=1;citation_pmid=9584111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. Mohan;citation_author=S. Nair;citation_author=A. Bhagwat;citation_author=T. G. Krishna;citation_author=M. Yano;citation_title=Genome mapping, molecular markers and marker-assisted selection in crop plants.;citation_pages=87-103;citation_volume=3;citation_year=1997;citation_doi=10.1023/A:1009651919792" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. H. Moore;citation_author=S. M. Williams;citation_title=Epistasis and its implications for personal genetics.;citation_pages=309-320;citation_volume=85;citation_year=2009;citation_issue=3;citation_pmid=19733727;citation_doi=10.1016/j.ajhg.2009.08.006" />
<meta name="citation_reference" content="citation_journal_title=Theory Probab. Appl.;citation_author=E. A. Nadaraya;citation_title=On estimating regression.;citation_pages=141-142;citation_volume=9;citation_year=1964;citation_doi=10.1137/1109020" />
<meta name="citation_reference" content="citation_title=Numerical Optimization;citation_author=J. Nocedal;citation_author=S. J. Wright;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The Bayesian Lasso.;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the bayesian linear regression package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_author=P. Pérez-Rodiguez;citation_author=D. Gianola;citation_author=G. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Afr. J. Biotechnol.;citation_author=Z. Piao;citation_author=M. Li;citation_author=P. Li;citation_author=J. Zhang;citation_author=C. Zhu;citation_title=Bayesian dissection for genetic architecture of traits associated with nitrogen utilization efficiency in rice.;citation_pages=6834-6839;citation_volume=8;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. P. Piepho;citation_title=Ridge Regression and Extensions for Genomewide Selection in Maize;citation_pages=1165-1176;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.10.0595" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. H. Rice;citation_title=The evolution of canalization and the breaking of Von Baer’s laws: modeling the evolution of development with epistasis.;citation_pages=647-656;citation_volume=52;citation_year=1998;citation_doi=10.2307/2411260" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=G. K. Robinson;citation_title=That BLUP is a good thing: The estimation of random effects.;citation_pages=15-51;citation_volume=6;citation_year=1991;citation_doi=10.1214/ss/1177011926" />
<meta name="citation_reference" content="citation_journal_title=Neural networks :  the official journal of the International Neural Network Society;citation_journal_abbrev=Neural Netw;citation_author=E. Romero;citation_author=R. Alquézar;citation_title=Comparing error minimized extreme learning machines and support vector sequential feed-forward neural networks.;citation_pages=122-129;citation_volume=25;citation_year=2012;citation_issue=1;citation_pmid=21959130;citation_doi=10.1016/j.neunet.2011.08.005" />
<meta name="citation_reference" content="citation_title=Semiparametric Regression;citation_author=D. Ruppert;citation_author=M. P. Wand;citation_author=R. J. Carroll;citation_year=2003" />
<meta name="citation_reference" content="citation_title=Linear Models in Statistics;citation_author=G. B. Schaalje;citation_author=A. C. Rencher;citation_year=2000" />
<meta name="citation_reference" content="citation_title=Density Estimation for Statistics and Data Analysis. Monographs on Statistics and Applied Probability;citation_author=B. W. Silverman;citation_year=1986" />
<meta name="citation_reference" content="citation_title=Neural Networks for Statistical Modeling;citation_author=M. Smith;citation_year=1996" />
<meta name="citation_reference" content="citation_title=Support Vector Machines;citation_author=I. Steinwart;citation_author=A. Christmann;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=D. M. Titterington;citation_title=Bayesian methods for neural networks and related models.;citation_pages=128-139;citation_volume=19;citation_year=2004;citation_doi=10.1214/088342304000000099" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. G. Usai;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=LASSO with cross-validation for genomic selection.;citation_pages=427-436;citation_volume=91;citation_year=2009;citation_issue=6;citation_pmid=20122298;citation_doi=10.1017/S0016672309990334" />
<meta name="citation_reference" content="citation_title=The Nature of Statistical Learning Theory;citation_author=V. Vapnik;citation_year=1995" />
<meta name="citation_reference" content="citation_author=V. Vapnik;citation_author=A. Vashist;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Visscher;citation_author=C. S. Haley;citation_author=R. Thompson;citation_title=Marker-Assisted Introgression in Backcross Breeding Programs;citation_pages=1923-1932;citation_volume=144;citation_year=1996;citation_issue=4;citation_pmid=8978075" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. H. Waddington;citation_title=Canalization of development and inheritance of acquired characters.;citation_pages=563-565;citation_volume=150;citation_year=1949" />
<meta name="citation_reference" content="citation_title=Epistasis: Genetic Constraint Within. Populations and Accelerant of Divergence Among Them;citation_author=M. J. Wade;citation_year=2000" />
<meta name="citation_reference" content="citation_title=Spline Models for Observational Data;citation_author=G. Wahba;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Sankhya A.;citation_author=G. S. Watson;citation_title=Smooth regression analysis.;citation_pages=359-372;citation_volume=26;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Oper. Res.;citation_author=P. Xu;citation_author=L. Wang;citation_author=W. D. Beavis;citation_title=An optimization approach to gene stacking.;citation_pages=168-178;citation_volume=214;citation_year=2011;citation_doi=10.1016/j.ejor.2011.04.012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. S. Yandell;citation_author=T. Mehta;citation_author=S. Banerjee;citation_author=D. Shriner;citation_author=R. Venkataraman;citation_title=R/qtlbim: QTL with Bayesian Interval Mapping in experimental crosses.;citation_volume=23;citation_year=2007;citation_pmid=17237038;citation_doi=10.1093/bioinformatics/btm011" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=N. Yi;citation_author=D. Shriner;citation_title=Advances in Bayesian multiple quantitative trait loci mapping in experimental crosses.;citation_pages=240-252;citation_volume=100;citation_year=2008;citation_issue=3;citation_pmid=17987056;citation_doi=10.1038/sj.hdy.6801074" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Yi;citation_author=D. Shriner;citation_author=S. Banerjee;citation_author=T. Mehta;citation_author=D. Pomp;citation_title=An efficient Bayesian model selection approach for interacting quantitative trait loci models with many effects.;citation_volume=176;citation_year=2007;citation_pmid=17483424;citation_doi=10.1534/genetics.107.071365" />
<meta name="citation_reference" content="citation_journal_title=ILAR J.;citation_author=A. Yoshiki;citation_author=K. Moriwaki;citation_title=Mouse phenome research: implications of genetic background.;citation_volume=47;citation_year=2006;citation_pmid=16547366;citation_doi=10.1093/ilar.47.2.94" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=N. D. Young;citation_title=QTL MAPPING AND QUANTITATIVE DISEASE RESISTANCE IN PLANTS;citation_pages=479-501;citation_volume=34;citation_year=1996;citation_issue=1;citation_pmid=15012553;citation_doi=10.1146/annurev.phyto.34.1.479" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6.cover-source.jpg" />
<meta name="twitter:description" content="Parametric and nonparametric methods have been developed for purposes of predicting phenotypes. These methods are based on retrospective analyses of empirical data consisting of genotypic and phenotypic scores. Recent reports have indicated that parametric methods are unable to predict phenotypes of traits with known epistatic genetic architectures. Herein, we review parametric methods including least squares regression, ridge regression, Bayesian ridge regression, least absolute shrinkage and selection operator (LASSO), Bayesian LASSO, best linear unbiased prediction (BLUP), Bayes A, Bayes B, Bayes C, and Bayes C π . We also review nonparametric methods including Nadaraya-Watson estimator, reproducing kernel Hilbert space, support vector machine regression, and neural networks. We assess the relative merits of these 14 methods in terms of accuracy and mean squared error (MSE) using simulated genetic architectures consisting of completely additive or two-way epistatic interactions in an F 2 population derived from crosses of inbred lines. Each simulated genetic architecture explained either 30% or 70% of the phenotypic variability. The greatest impact on estimates of accuracy and MSE was due to genetic architecture. Parametric methods were unable to predict phenotypic values when the underlying genetic architecture was based entirely on epistasis. Parametric methods were slightly better than nonparametric methods for additive genetic architectures. Distinctions among parametric methods for additive genetic architectures were incremental. Heritability, i.e. , proportion of phenotypic variability, had the second greatest impact on estimates of accuracy and MSE." />
<meta name="og-title" property="og:title" content="Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/6/1027" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Parametric and nonparametric methods have been developed for purposes of predicting phenotypes. These methods are based on retrospective analyses of empirical data consisting of genotypic and phenotypic scores. Recent reports have indicated that parametric methods are unable to predict phenotypes of traits with known epistatic genetic architectures. Herein, we review parametric methods including least squares regression, ridge regression, Bayesian ridge regression, least absolute shrinkage and selection operator (LASSO), Bayesian LASSO, best linear unbiased prediction (BLUP), Bayes A, Bayes B, Bayes C, and Bayes C π . We also review nonparametric methods including Nadaraya-Watson estimator, reproducing kernel Hilbert space, support vector machine regression, and neural networks. We assess the relative merits of these 14 methods in terms of accuracy and mean squared error (MSE) using simulated genetic architectures consisting of completely additive or two-way epistatic interactions in an F 2 population derived from crosses of inbred lines. Each simulated genetic architecture explained either 30% or 70% of the phenotypic variability. The greatest impact on estimates of accuracy and MSE was due to genetic architecture. Parametric methods were unable to predict phenotypic values when the underlying genetic architecture was based entirely on epistasis. Parametric methods were slightly better than nonparametric methods for additive genetic architectures. Distinctions among parametric methods for additive genetic architectures were incremental. Heritability, i.e. , proportion of phenotypic variability, had the second greatest impact on estimates of accuracy and MSE." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6.cover-source.jpg" />
    <title>Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yDm41Ca9ad37dqbtyo_c4m2xeyBT4bNU3uLwi8HUqM4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459902","apath":"\/ggg\/4\/6\/1027.atom","pisa":"ggg;4\/6\/1027","ac":{"\/ggg\/4\/6\/1027.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/6\/1027.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/6\/1027"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/6\/1027"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/6\/1027.full":true,"\/content\/4\/6\/1027":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220FmBT6NrY5TN\\\\\\\/ApUQ6vm5o3t0VpThXw5oLdvb2r2h2EUw2msBrZNrKK6GwMdPrpZNS03Lc03IlJUn2tJe5q0fFlXek\\\\\\\/1BiG1AJuhhoKlsw\\\\\\\/hscB1UuJ2oomCAghIkot28hn6vYBMiHkNj6ZXR1uzSudpGWZR37GOea35tu2z8HP9+Kid8uO6U8+5CUGzheZGMO8T5698oiQ3zGWLYdf9Ja5ohhZL2mRSHKnjcbiMm5DQ0qGQNH\\\\\\\/20U2cZCVDWVlPT6PZYMuLtW+5G+ppIvnmOOricVmw7g0o8TTJHnK\\\\\\\/Id7CwtYTDIcL\\\\\\\/xvJ9bVe8tDHNrxipjUZ8nKm4fprkElcbnRt0xJl0ibrvLAkjso7Knr8PWNUTyVz8ky3uk\\\\\\\/\\\\\\\/meOlBvhZuZpArPuLNm3+UJnBbSRfRcpXCGKbf1czGsE33B6oDJakLAfiw3RFK0300tOhtMqXiHM5AFYw0m1l6GoBiBAspmhu2V1mEuD\\\\\\\/bMOLnREWoh7x\\\\\\\/QMlNJRQ7NuBYIOE4LXnBpHuzwpAPoF61tSB6ZyzxPw3RVdfrH3Lr\\\\\\\/I1p+\\\\\\\/bawURnemgngdT2zPjNS+v1N+wEcQ89KRsvrnM198JXqBzbFFJHfNbYMbqWDkWvxwE5af48mAruNEq86Mx5a3sg9ZkvpVisgikxlhwe0u09W+m5lDocPnl+yzDPOF\\\\\\\/oYOu7\\\\\\\/6vIkZxfbIpWJ6l4UXP+12+mEBMl6saJz1tpsWhkj7HEgJO1eCj2yUo7ousWOKwv6noPYk\\\\\\\/6B7GN2HRST+OMv19RMKvya7VAQiArw7scAE61CRvUha\\\\\\\/7o3Pi3T\\\\\\\/OHJjpNY\\\\\\\/WoWLpBdvWby4+m0RIil8xmd8pmv1ka8Tvj2w0BPhsoeY0+pLUj2gkJRp0eLWmpvMR6I9hhC1qzdHHtQROexVOAtx2t9RR9XX6Qep7TTskZzlw4\\\\\\\/ryI1u+IUppgHQfpFw2Da34dxo3B0AHDG\\\\\\\/FPe8LBsBKqqE\\\\\\\/BRR1nPsZ2fMXj8BFn6ZPGCkUsP09ImOYfera7igXSoACYLocLSmLidACuT2+KZ1nrDLu+CzQeM7BGsjXEs3EZjkiLU\\\\\\\/gB00yTceggTDxzru6ybiOQQthWFkB9R4wAuHYYfMyry6tJHLmF+MI9NjzNM9tXYfZR\\\\\\\/GCYtUsEtz33nxBYIZ\\\\\\\/XUYxc40DKffp3J4ITbQNsJsrcBdHYNOBZ1ZYRQzZ27BlQq5cvNofpJLESjbsx98SaCyP\\\\\\\/kHa0YgYXK+7z\\\\\\\/ccP3cnEwFTxH5rR4v0e5FL1bn726H7xSKrTuSmlbXCfUagyvE\\\u0022,\\\u0022iv\\\u0022:\\\u0022PPTqw1hK90bU+d6FRW9j7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c7b46226c17a66e50b3ff726eeb663cf781502959e7043c355803d100982c3ef\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KrJhqF5s\\\\\\\/EPYAv1YA55qHxRtiQ2HNR6K9ZpOn1HdROmF6ttk9t9J9kZ7GoyvVc5khJeP61FVh\\\\\\\/yZdP93HqoPJOA2JjyI7pGhb3vPgNLOrn5k\\\\\\\/\\\\\\\/RK3HjOUlqzvDyTdCftr2ffTNRavG9hixKYpTUbptCMrlACrMsB+Gwmw58CSpcRUE3Ar0a1T1lEALtRE85XceKe+xQaVeVtWD3rWsTpGJaebopXcJ+iJi30j9lYQ2w02B1tnNNRj46SKN1SBhTpJ\\\\\\\/z6K30yMocFrNnxOhTnl5z\\\\\\\/NgyosrPnm6UO0G2rvnRLB22gfo76dvfX1Ab2jv3seZozGCO4onlVZt0uEwPzU1ow9gPvZxrDbbLgMiJPLd\\\\\\\/+LBIppyWXitH0pi6tnsD3XxZxnOfUy9uKxfMMsHe6KT72B9p+pxSyi2G+8cU\\\\\\\/bi9G98NATOKkdx11IrqwbYIFKC25258yn5F5cXNyQllW4mrw4msShqi2j7ZHnk6lJh6Y4luN09Gzsop+9QZJm53KU+J8czl8jnR3SKal18pK6ETw7xPelKJBi+GZiH7c+U0p5bE\\\\\\\/KogzqUvmTy2NcwsXRFe8k3ccyjrANy+MpQztXZU3JUiw5mO56ZbeJcs+psH4FLxrcJ3c0si2uXisJMRQ5kQsgIZToeFpiHua0p+wfrPw+XJLAj8+v3QTCOgMXMADkuRWF+bPJPxErNTDUuxG056f30C6S\\\\\\\/EbMlcyfI7LkTT3b\\\\\\\/8D6I60JY\\\\\\\/hoUJ4J\\\\\\\/od\\\\\\\/38iVhw7+PwaajM\\\\\\\/pYw629vC9Jp80JN7y92C7WMkQ0h7FdRmT0N6W1jglRw\\\\\\\/3HnNdYoA4nC2BlZqa4o78O3Sdm3zQfsfC49ufVxp2UgFFqW1UMFf89XUGVY+a4co7G8tEO0xc00meZmdToEsYg6hnAHSbES+ZqCGAIhqyqajU4wjgBE+96XJSrDPZghEd+apBCchn8oj\\\\\\\/toiJbFvDDRWszY2y7boi5hGQI6lE37\\\\\\\/4MYqZgJEyx5z9f1X9+2kZaqrap0nlyjpk7RXZoJK3jiXDhSbYC88XxE7bbou2cVF6fjjiSsKIav7g9J\\\\\\\/O80QyyK5zazgrzLWgMro+oNa0N2rpNMGNREr27Mt+sPJoKD5D1VrZ8q3JbOBJCqvF2QBAt4uUHQ1PJ4ddnv6I1OCVOiw\\\\\\\/yFtb1JPL0atUqgMK4wnLHr553OFMoSpAsqgVgPGEzTS\\\\\\\/iSbAXzqPNn2f\\\\\\\/cGQHr7tqcBSyH03SbvY\\\\\\\/vKYf6MYora94TSUDjgkKZ12djYQi\\\\\\\/wwvPwVxW0o6+QaAHcz9NfkPKYJMX0bSX\\\\\\\/kfcluMOWasc\\\\\\\/oiqlR\\\\\\\/OCEkUx1vYsTJTS4JuG06EqS5dqIdI4SS4zp9ViL9FM45m3ygScSWvunEf+HdZS8ZhSQoA7pARmFZL9icH48TFZRTziJXEZcpYHWDn4ZbYJxO+wht\\\\\\\/\\\\\\\/fYQBeP2\\\\\\\/Y6b+utmCv6KwlF8zb+gRvtiyMk6fBNE5GJKYNWt5NtzDUrjOQ7ux2\\\\\\\/DTCArZSBacZDniUd1uCJZlBrP6ySaUfYPHzqU8BZ99bvRohtb4w\\\\\\\/8OO1aTH8f1eMwN6NjBsNGcZepzmxKmj7dCBvIfwco6WXwZSOqasrqiLflcGCPM6ehzBdaK6783hp0t7gnSLnizdC1ugVoXtTJ\\\\\\\/WYmOpXcyKV9DH5qvWBsX0Jewrx4jwqS0D1qU8owZf8QfNeo9tLvyrfYAbTmuRCgf2ZpCFbGm5+3hM7INLw0nwEMjhTj\\\\\\\/lyfEfcoHJYepKiMx2TRfugw2+o0jFz2hA5cUSJTSX2nHCPK2NFDIZDKFp1wX86UGY6NIXbm6fCTdefL0W3YOwqT+Qf6ytq6j\\\\\\\/3UaAIfMTKr4QHK7TWCEzkIiJCchyetFcz51+UQTVQLEUoVkQ03OsOi8nPyTso+KTtvNoRZlZYDTPDPCV1EN1W2yZvuJBVCS8YShlxcsuUkS0WHVh+i5Kk8HJFxTEmUvZz+mxXQGK7ru3HnlCGSvCU1uYbKiCO0eNYGaoU6D7QEK0xr5zKZJpow6mnUvIG+m2BL+asBYRVa0cg6GUtBMS32iuB1NsZ1kjdmWghcK7anwPefHmvJ0flIcYfEiwjjvP+72Ti+Bv3H\\\\\\\/XZjcnOsV2bgzqVF4atQhNWP8Xjm56s9R7J0Xn8oRJpjHCJGUx+KO2klV2OF8J61V+o38d40jiZHwiuwzfdUBtXTkWduWlpDHGr+Gq9NjQeGbc3rqrp4IPcp75HQug6aCd4I3Nj0mXiSmO+BAFRjGGHaWiJ0y5i\\\\\\\/3dXVXx\\\\\\\/E5CiA93wiMCfiO1ZCecE+K7glMF6+O6p4nhmMgVilEghhQf4KUO7G8eUx2FaY0nI\\\\\\\/fZFfmIz8yiJY6UkwZKJUxr0xQBbbLQl9hdEUCaR0qvh6UZwq7YKYoXWTwwrqxJ9R+vYjSCsCuerfwm1dZIG+iC8kB+2AQr383qOfHg9Wi7uaREASQmELWWcwK4H6M61H32hY65LDY6\\\\\\\/T5qDTjK4NIOwJq\\\\\\\/iwByhDMawiHffm\\\\\\\/ZrUbMKKaH3JLn8fm0m0e8XFJMKyYl8TItbIMue6kVOkJWOt92wQi7J6hZMIjOLcoD3dQ7+FzHLS5TQwo9PHn+QQk7Sj+ELh0OBs2dqACu4tW8xtyzdFsoq3m+5PuApBy+p5RKrCliy8JPFoL2+rZ0XJkyIP\\\\\\\/\\\\\\\/cIZP79hlUKZpAOA7IqLk0XUMf3+QPzk6y3xXqFYux4gsnaSd0Rs3KWEqaLw2Ap+o580ufm5R9f6M\\\\\\\/DSJgf+hYSj\\\\\\\/iUPZFfM4z236EQWFC7n\\\\\\\/DBIAlnbuNfi\\\\\\\/WgmV3zV3m86MxMxJKjUrXryYHPCNPaR7aJ6FIUJFvMKG9t8JoQykTNMoPMafb5yYPcjpsHOmI4VOenR6dsG1nBgcejwr68ZsI1NhFU42fAO7\\\\\\\/ibq+0sF9UChj\\\\\\\/VENGQO1XIlffU4FHMJY5n39mPGw241KIlr0rkZgJTH\\\\\\\/R\\\\\\\/x9cR+HHB+TEBsdhxRtkPWYzf3GKkuYV2f9HWczrk88Vo0TLn3A9yV9Y9tDuFc4gAzK2Bnd9NbXaYQTSPGx6QOSvkzPVqEFaJ0xSYfrIsERONtfUAu1O5L5v+zIXe+n2aswIYtH5PTk7xZxDSDJ04MoOgp99ZfHOBWKeInmDVSP9T7UAAVTAZ5eS5069jB\\\\\\\/YMdaVn1fIQryTWSs6lqzw78VKfHEXrLZcan3zYdUoGtI2YgVxV\\\\\\\/b8D7nS9c2IIz4rG\\\\\\\/zHrk3J9dbLWIMHmc8BjdZnV7S4y5ZKnQOLMCrkKfygLzVQ3aT62YPL4gHTrhz3bhNDbQyZ3PDPKjleGncjeiiFhat8hkypwgjTcmnLxPB8syE3LuRNNfekDk\\\\\\\/S+5nuGM6Tew8bxvznjPYGTK7rMCVXvK4lwoUHWBfSRUh7HRt1SV21wYolNfM9h9HO7\\\\\\\/HoYcai1+tadIXHQvTEIHD201nMKPELgxAr01mPK7Ey+I+w6eiOgaRDySHJFU\\\\\\\/eDGTq0zWTKw3u7f0vNlEwBdHYK1xt6B+N\\\\\\\/+hmSg0Id9QyMN4AZ9chaN700rOffHprIvRcaPrnB0vsX2imj\\\\\\\/r\\\\\\\/UTRZC+tflfjpE63juQG3jtURTj\\\\\\\/ibrnC5WYiJMpJgdSm+vvFFIn06qLsEJvBaUvFcdkXlo1yOxcwSxod9YDPLImPnvRqL+aT+omKbriK1DWmP8m+YtAtqermIyhgRgH\\\\\\\/nKjt8PPuX1txmwl6dOmTLCKjgfDFs24eFRoFa7J\\\\\\\/a3a\\\\\\\/tN\\\\\\\/FPnhNy8gwuozOt2ZTfutRIwvNKTCevGoS\\\\\\\/gnU\\\\\\\/NLk9O90eyAkTov54WV\\\\\\\/OnBH0c7LsF6zmmbyxBugB3rkRViDlELBucsnff4Vz1+fdrmNm+pEKOK\\\\\\\/if5MZdDPhfWuI\\\\\\\/DTNOWVXRdpwfJfPi6HC1bOEDONYSaLFizyulkMgfK0Ts7\\\\\\\/mv+J3sQ3aarBOl8XbMz1STlCFClQchauTJuNyYAzCwDHzkz9um1jJsbGzij36OhGTajTO8lRFjogHoeRQjT4gC2WQOEOBVuYaLVUKiS4pDxkTJk9FVGgXA2at8IK\\\\\\\/mD7LQcXrVv\\\\\\\/eeCz+thQ66uYYTBLmf4760L1l58J1rrYcRjIdG6tj6BQiU2ccFzmdSFJRdzUhek+Qls2ADvPmSQga4I1KAwhFT39zyGwGQe8wauWSFoT6cGwMTgqR\\\\\\\/rarLNPVRNWfHOy1t6fJIyTyuXyY6lv15H1YmzqJO38V+WdkXQEnIfR8WEqS4LLKelQocNwn0nZlZhoWoCIsdfOsSCaHkVcMJm3Kle4ckUX6iV\\\\\\\/WHwDQ09svpqoQw0L2ORG1j0ZNa6ymgOwqXA9jgovM8TYKIsnuEwuc5YJzySl7wt1j7HK9Duqoo4mjC1zUSvOz4nydwk384v35GTAOkDkKNYTdeaEm0fjosh7MjJLn9yP3bO1\\\\\\\/W0rVISsnP2AW+tVDkvxpqoJJZhtmJiEAu\\\\\\\/ZfO9129mbX+w2s\\\\\\\/vsctZae6YssPFKYF6P3NktbBkMJCNak1YUhKo3sJTcVfDvoXUh+q72cHH2ezXBn+yx5QrNzblT1lOoxAAG11es2eO0j1LrYXCNJT\\\\\\\/VBsyRHsIux0X9PNm4\\\\\\\/X0yeZOUXmqvvrV\\\\\\\/eRKkBnPwPixIVSfYeRysp0M4hcQgvbdGkFfUmUn67fa1wnSnYO6UyyUk4O91\\\\\\\/gniCsaBThIyjGupPx6pAy6nC74vMBg2ENwZXYcQpejD4L0Uq0z\\\\\\\/SZYcahT6WF1qXFCpBKMyL8hSkzKq5Dlj1HzSBZbMFFThRnuXt2RDMg\\\\\\\/Fp9X8QF31ev65t2SC+iU99E3A6U1j\\\\\\\/OJNZATKOpO6r3j6BCnpixLNkW3TJ8uoF6pTm0S6qvt3VN4TjXDcpFhUHZIdGbD1w4JhynTDMHIyFNehBVqOwafwZrCzJVep5+FlOAiSu8QxKJqr0QweXwxFxYczm47VL\\\\\\\/PBIyiq9zzMeIVeRq9EqtjosCfoeqTRlzs3wzj5HKW62XzFaoXoX\\\\\\\/5y+c3rm0HSEgTnYAzOcbx5AcEPKkf+mby+Xq6mzc8Q8zzmfk41MOmvH8Qa85d6z9uo8kl0xmMxuDmlQx5KvG2r\\\\\\\/NJLfq8FiCMs73z\\\\\\\/8wuEw58tQgbtWOUwGve0XngSpxPp4PM9ZQYeLhBNDyPHSmZw1aK8iVd69r4nxvU73IzeBkiZAWw+y7Q1jwk\\\\\\\/VLfD91f4ZbFjqZm1BCGEgpSjTLp11IiGOXVZze3+IjVW\\\\\\\/NYsIquYLhSAGFSfCIeAnfRMz7nX9hjm3n3ynosunTzHpxnTwYjhTrDtdq2JOZwI33WgvfzKGkh\\\\\\\/iLxSkdD6dmeBcG8HcnWanbsINSyo3sqqto8wAFHTPJ5fO\\\\\\\/zyApamuk1taqnSJwAtB2L\\\\\\\/MS\\\\\\\/3hN0nFJvjbW12erhkT\\\\\\\/koDEm2ZOyETpgv7zd6i0x8hqguXWy8qXtyWs7e91xzY987QRY\\\\\\\/LbLgVFJkao++9wsjIxzTbJNDfjHq3sng3PbAvRVn57DWPdVUby3S6gdGN+NpFOy5hGXpxlmhoYCjiGrN5kGIXD86wj2s\\\\\\\/8CpYRZ8c0Y4tlguNE15AL5eowaWQ7qLNAhXqCyI+aXCx5CiijTNXgDaWpR2i+23wT13\\\\\\\/aSZtkoVwaQno9wVxP1vo0NumabnZO14kxFDg8HnO2nqpdYb9058BeGZ7ad0bBDyyGrR0wsW+JWrdusTXHXHPnz2Agxhqf5RdjRlmiJmIUX\\\\\\\/ECQxKSwTn7XhgK2LXu8GQtA8MGJCkxLqr89x07LWywwv9kUdbGt9zLiNd3CveahbnlMGDNVJb84H2dRJ9Mm8NOk8B+lytvEuu9Rm7B3nNP0KzJxPE1IBTAH0ytxWdvgvISRKla1KfSk7wIYyf9MTNnoG+9Uy6TXtnwNrYH+FDGh8aF3EYT0bBaIqnTbyjIcu+xgaEOvxPk6olCbyZvq2lL0Uxl+uRD1NCyQSxlkcMl5nfhsswvl+liHCcIXz8BKe7RLx9CrQaZur3On6T2E0oN4bdH41fIsyzmJTsrkFHmXuNHwxDr9+rqldE152DxOrVpXDKAwrEpZGje3dpOBazECccBvm\\\\\\\/BbtuMSQAPF9QrfuSple6oXLvh99WtqOi1BJOD0NygE4\\\\\\\/dW\\\\\\\/iCI5Pirruq0pbRrli7VytLuFgRhzvf4dr3ZhnfKC5vBqBc20RuglNIw7hhzr1aLskamXJoodM6\\\\\\\/Y5Yq8pkCP05asqrb6wZkAmT03JdEQ1U\\\\\\\/I\\\\\\\/O0xr7REtmA9B0uVhwDg1qznQr+CNtp+lUMfdH46txVtthymSuInFT7aBUJ69o07XvUjk56iFzzl8oW6P3zgIhsykFK2KNFOFRsAyI\\\\\\\/EZsONE5hEyKlRTR\\\\\\\/fT9GTCmZQLEzUoiLzNIWGqsuRqt0ESwdLVsjAPGCsFBqKAjPe5m3Tt4ePhL5iZQgIR3OtlE7QPr0i7CL+yReEePLHFOG2RvgtiNzC4uMMdsqomdfkIINTHKijfVrH+cshQbGDuReGlDDD5l3GghiHZkuKYH1xjMi6fOm0qPd+eAbLtGwO3UZYJghxyxnFksrv6oHwZtzjXVEhUECEJIW02IUXOP6A90Legnlld3RKhCbGz3Tv6\\\\\\\/mBvQawyaGGP5ta4QCN2Qvc86sWY9yXYIadfrDqfYZK1MqLqVGM1P2iNaqwh9YqIHNORfhUKoZFi6gmqDBUiLDynWK6a+ca7YH3+HNyR2q35vGdBUeiuGeTnZuoSoxJrTGeBRjvoU4TT2MvSdrxC27lCD5ELX4iMGlwEMttc9iwqBub\\\\\\\/saYkWLRc1KjvQgx0f\\\\\\\/3IqFBatsPLJzVHcnzVbSV9dQDAIXUw5ye79teYs66K67X5LFSFe95RHykMDu4Ql82qFFAQ+1jjGiOHd7patTYFJb067sbYb0QZx+MLS3K1sk4VH+RsrJCzpwGxO0siuEBh8ex1Lyu2k3\\\\\\\/gg7XYqjrwpYf7JLvRK9KFT3a9HvW56jekBY2sT6w2SmuBArA+SiPEyYyKsn8uuiQAw2b5RNNByZzVxF3BQ4e2wXZkXw9beiJpKHAHP3SOQGxwLlOGfjmbKoak8zcD3F27tulkHe3Czcag8\\\\\\\/S8wqhlZYQGhw6s6eaVjKO+rzYD8M51URvE3Vu4Vr3jUw1ukUZR5SL5mQzJ1g8T5uyJ+sPihnjK7SSCFzDzntpru95SufiyUa1d0QIi09S87oUPPaXRVRopepWUhA4ARQEvgORzSxARJCZb7sOGjoEYVjB73aGAgKRVlZaXG4BCvj0mKNQNrpMN4yvAdZ1P5i\\\\\\\/rkl2wUDp6QNfhkUy4s8gNFDmJ1gUOYQQgyQhrmpzjP9tUY60O0R5Y6Ky982M1e6i0BzJnQYhPE3ZOCPFD9sekFrLbcwmTHXysRCtI0QXp182uBiF9Xe2wfkePqIytpvJhPq23xbnh5HYYVeOT8\\\\\\\/2hZAXi2k5f\\\\\\\/PGRuxkLTDs3E9doGZjcegtuC4MW2uxt7UMkwr6CDKIQFjPyrjq+GRZp3MURRM6gOB7a2+9Zb++4OEhrQVhhJzk1rwF5KacqUGQiIe6s9JeMnjZhPGIrlk7DwYtmFTUepxoBhKh1mC6K9WPdyT3FW5yC3RzZpDpMVtm4cWcuKNdtSarM5mNMtKb+pC0lIcQbRTz6TY\\\\\\\/51t2siWpgx1OvKNOqY4yubQFVZ8It3E8Z7pxySmhJ+KZTWf+vyPNOkZeFQPeT+P5xS9e4gxHaEAr5OJIcsAyJriEZRB\\\\\\\/l7L0sDjS+NcgFzF2KkNCIEl5xoMGRDdglQ6JrQe9uJ+n3hfgrN9cNLe8aJji8LQesLkFjfjFU78kR7sww5Fg2jZBvFG1qQHNJoayd7aJ1L+zWpALSN9BEzBux329LBH7pVW\\\\\\\/BtNJFfU9pwoWDp44zibYxN2uqQo\\\\\\\/121sfqUBYtsgOd\\\\\\\/67Y+VjswjBekcfKt4Vzw8LAO+dKteENvdMilWdOfOR48x7OzEnxVC5noHMMQ8jnybrfjdf1Plx6J9iaPUJ+n0CoGpoS99ly+ku79vHlX78F0V0ARKhKs6zSPKyVz24r+BsPx0uQAb\\\\\\\/qaL4c1dswSN+FZViN4mOX95aLYhtnzwclqjgyw1X\\\\\\\/Lle5joO50\\\\\\\/KHsS5hZfqfNgMpTKHA+LkZL6Qms3CVhwHsUKC\\\\\\\/RhWik2FtJB5wLIMXh5i\\\\\\\/eMVFF0QuAGCjemLz5baw0xKJzWAYt7a6vlDGBRSfjD4N2Lfdtq1DjRYHNBdZbZqKyCLM6mwrpUStPeNzzRPH19uU8uOYK78rSIL362Ku9Sm\\\\\\\/OmFriTwbusU2CbKa4Vo9JQJ3jekkyBQdsMH4G+xRoBWLd8yDO+tAfL57HdxWAz9gpNr2lPM\\\\\\\/0b+jLGtWf+vo8MLDb8ihTmmHxCm+Ybe8tOlgzwZbNA1Bd\\\\\\\/Y4YVwmtMHG0jtWUiT7x2QX396BHFABYmGsWI4ykRE8PMHSdX0GEibjowzSwsgcibbmPzMyGOuGjSEpWTAVWhR5Fb+FsEgnjAlcN3dWzPVmcD+P2A5PGIjFH6\\\\\\\/Qvc1sVU7I\\\\\\\/Xy60KZukjWk2ReXSsKSs\\\\\\\/f+4v5A9xs3zd82MYvqBG2LAi0zvpHddcboKxO+fCbkSFgdJ4YqaEdRG4wWktx\\\\\\\/ZiC30J1swEvJKFlsAK\\\\\\\/NnDbf6DQ0FK9f5YiNbOG\\\\\\\/yTkr8j4TrK50FR0fEFVINMZLmcmORzK4edc1Lu8uCZ2OxGg72POCMNyHyDB2qdQ0pvAgz\\\\\\\/i910V+T1kMh3QkqySzgbjr\\\\\\\/c4FFnCaZGlWMyWg1u\\\\\\\/rHCW7bK6jDk\\\\\\\/N3D6WdOv9YkRhewEzkQhkRQE0nwbWkWEOxcryQhjLQbZdsNOA94yd3+hriK1uf8BD+DI02stqhWOqy161Gc6QeZTV5FAKgHfeVkqHkkVRdAnmBOZiIgJczm\\\\\\\/4GYrzjh3fmlY5LzmlPsLvJBVDvCPtAgAqINDdlcI9zYnREfTvVFR6yNckhdblOdNdxgMzLkk3vcUoCaZu9tzdmIDMUd\\\\\\\/VeTpX+R1f\\\\\\\/iTuO0CQ5dtx7Z\\\\\\\/n2boEVnGPgVO+SeeZR3LcEivLl1JBxX2SyG3OMrSNTm333oE+e8\\\\\\\/JPnYFv0JcRRc48EdcrDncAuWfw3h5vagLxEsAauUQMfxws1KVUnGeNE4H\\\\\\\/RfC1X+NaPf6x68iv7nU6OQDbVdJNxmwco5ed7\\\\\\\/qGk58viL7lATSJuW6n5cYkrSIoEKsNZzgpdw+CJXfILhYLhxIOvCKudcYNfemZT1ONECVWhA0Q32zVJNaO7GUtxYGMYRA0Cahvug654ODF2DbwUl7w91dcBkADBobZFVYzT4gLB81Atj2dqojar\\\\\\\/eAQ7\\\\\\\/VHsYS7+uS+u5qh6EcNF3VKluf3s1LxENZWB9qhqRogbDHG3sse\\\\\\\/sCkg4zU5rZLN6YJZyt3dMhmSwu7HqDaYrxUxb6KW9bTExJJNHYS3cKePxTHiNbmcjCPrekJVnDw2AhhM1N4DgpPzNboFMmmhKGf4lbLdpKWiIC6KHz9rS9GJ0hphklTnYW2Ak3ja7gSnjvDksjdZwva+Bg02RFL3BXJkQouuDf4KMnO7mJWjx1ILN6kfSlUi4laD3PfuAOgUSVKcXgVphw4PI1Y85hbGIz4lRPNUrB+PaR6DW+bi0aV2IM4yXaQTxNmJUDurr6ag8Am421AASOi20MUPo\\\\\\\/i1afL++MOkzG7iRHIdHeNqSi\\\\\\\/AGC1fibW+SAE1sa5F0VQYBuHQpJLgP5\\\\\\\/l5lYgybBaIcn\\\\\\\/umYJaWREtrWv3y+RARVe4wcmzHtKywx70SO9+pTwdgFV2UdODobMswq82ir1kM3BLTusAc63Waxk\\\\\\\/hib1FicdaEa63Kfs8kOO9LVUra1syw1oVcu864mvSbFore4GsGlhyumiJ4ToYXShC9ZrjxC7HtfIjq6KvvXdtQH7g5TWiutESl8WUYY86rwsFlsomEcBjhjiseOin2XU64DzoZLqiJ6mA1h4CAS1Dqe\\\\\\\/OZJnNDRBtiK40AhKvoc24eAGvImaKX8KRkdi5IFG1kOABQjE8uz25wn7kKsckwI0NuE\\\\\\\/xqj+n9YqEX726ZD+\\\\\\\/LY6aoQoG5GcQVQqR1CUc7JZPZbZWA4JddNSK2WxIQHBWEE2OV0cN8QNT2XKKs1VSeFSBns0Y1xF9\\\\\\\/RYeYd+xT+08DoMzXPhcIc2MJJf+CgMAcNIoEmxWi+Sjczf6iv5i5ZNdkvGeSvHOdj64wHdWQR9rvg\\\\\\\/SJXgeSmKLG7XL2nOo8QKwGK34fYAz7Oybtp9kF3lsBn0sCb0Bc6KzbSkrzdWyKZEEZySeOOWATdeJGBbrVzJJKmN44lBRRLq7ZDzsEX56eT+38Jo4hIsHrrBoxzBSs7trXA3l1vvsRW9j3ggQrEMBMnZHwkWNBk0ZNq2ZdgZUottBxcsWBlsKhChK\\\\\\\/YBZDvkQ\\\\\\\/QeYjqI7z9IVcHFvmQAMJHbnRI5EQ+0c7zv8HkR9ihhQIkSb9FgB99e7WElmgUxbydyHdaFusXmRH9xFCNPZ2VmtryuW56SZJk7+Uof2Y9wVUlT5mW5LrVbf9sv19AN6+PWGOCnsPHvNoLwLWEFECVrlX2iQ\\\\\\\/nnmtBtgvA5AFRJRtP+NRX6WoZXleLNuWnB9dxmLGKrBNgnSwz\\\\\\\/KaJdFQvjw4IMPCtmLEiqamQPome+V7C7AP2qX+9n7fFPRr9ra\\\\\\\/ZWn2KlcrYmDXOVN9s9Ph1b5NJd9csL3yC2yssThr+aNv9+dade2C\\\\\\\/aeXoD2fueSwOHMtXL9HyIBzkPXWZmFPOFcIlQ1LD8XueLDy8FmbhlOnpmadMzRBH8aMqvvCXjOdcqnioYOte0Nu2Wj4xolnp5UlRXAHFec1IZ5p34H+91WOeGMe6xS+OikGQlQOgFzFIu+6\\\\\\\/b5BTAR+5xDN8VYOPu5hhvLbr7sWzANeQjW1quTBTfvD8gDcKSUxK\\\\\\\/Wh96SQjaEz62JLLXUj2\\\\\\\/bOpxoyuEXEdnAYTojEwFKk05TjJ1MBx7WEQccZArb9eazlhbK4JLPWsIjAqCkrVLQwKjz3wAPLi3F7YgL9UqmwRe4nDIrfbvTz2SL\\\\\\\/Z7X8QDoFXrLf58sd\\\\\\\/cKdONzxqJRc2lQ+9rFTFVYonk3by7vseZS84VytUtuXuIJMWDswkZtlSim3WJuumf68WW5oW929T7sJxMbLHdEtG5uvb3Bwy1V6xhTPOLnyEt1a+G4LB1pFNukwsfbr8v46JijogeximA57gyrdHTpxM5kLoLA\\\\\\\/e6I6\\\\\\\/LM8U6GigUhKMVGpzalC+I3i0norRl1xR6mfyYZEBKVbQdx2DjprUN4Jn0kfDT7XTrBhXcm1hVAqJfvYvcfHo8LgmVR0lfYeGl8PRoH1\\\\\\\/hufXDPJkEjYVWdS0pI1A\\\\\\\/xQIZ\\\\\\\/rr+DgTJlJL9CDq2uS02fDfL8GKh\\\\\\\/0kLCyKtR9F9VYE0booOyeYj\\\\\\\/w4twfG\\\\\\\/+f8j6qJvb7VVpXybfXy+kVnid0qfa7K3MCBIkiIGw3j16VoEFS1bYkG4tZ0vdhJrLg5b7kLWKAZI05tDqZVVSakCUtqi22r+ZEZmxgRUMY3\\\\\\\/NoieJt0iifjDuvCwG1C0KrzcXokGwuc9GgSzYzUfcBlA6tjKGlMyGBNNYrkI7VTs8ZiLd0a03snjHrqOUG3KmvClbNvw6XA3DfyzkiN2lqtnfhQvnh\\\\\\\/6CNaJj5vNS9NpnZ+xHfdIRjyrViMMz8B6EMlnipSo5NvfwkL5INIPSsq4ThEduBdJLjJPf8Ls\\\\\\\/99o3cwYSdJbTpYTCJD+tZx3Q4CCppS0HpXmK9o71BxsQ7r8IXhcRt3qMyr7ezw1s\\\\\\\/HCKWH9QZQ7agTyhCKuDX3Jaxqvmw\\\\\\\/bsKgjAU+vY7dbPyDgCr\\\\\\\/gd2IlUahM\\\\\\\/P8OrS6Jx08hTYELvWrzS+FCc+Cl\\\\\\\/Z7MhQ\\\\\\\/YiCITxxvJd92IALYYh0oxkh4bfnF\\\\\\\/L+DP4YQQ3Ri8sGGF6SNvBXWLubklQW9Pd92LzI9HJI\\\\\\\/CgK+h\\\\\\\/SBFvZ3Ba8lyTajZDM9AhjPcwj8UQRZBMd63fJdtVsJQXSDorr2pelew9UjRnjEno1nANqi5L9YYkghTac4+yCrrNXAoVgaPQFM3mb0C4IMBUkASfpk9XWZZOuUsOlhuIkNjanhLJmRxXXNJ42PfposAvEfU3y64Huc4lDBllK7E914FBAWF29Ec\\\\\\\/8aARkjXTAGs0v+AtNtiqi+vjgAsiv7Wt+o2U6jFrxZHhYIB9bihrS2b\\\\\\\/WjU+UqosuolyvFsmwV67QdOsGhPwuQJSiaOyI2Q0DM96dYHrOt1n1DCrm+PcEWQYjk10GJ6pp3xVgKEvEW5KkBhTt4MkwL1DqPTfe22F\\\\\\\/BdhPphjzkpbgiOki1tvfUMyYFDZq\\\\\\\/zIrstRjPGP2bDhyg63Mf83hVIW9sS4rg2UPyHUxNfbBHGi8oUYxhEwKwo\\\\\\\/VSsaX1CrAXmTG3QGW2r6emc8j58mzPa8goF4NAlxLhWQttTSEPJacA8q9cEWrcXBzVMLhLh7\\\\\\\/PDlFF\\\\\\\/aaKcnzRRGA5qnLE\\\\\\\/G6wf\\\\\\\/7eKsncaKJznf2f+yg2+TAUdlI5PP3Bc8zxC+EO2UoYBlsadKY36qOLt7v6k6dxdu\\\\\\\/DIA3E\\\\\\\/HeN5e+IxJZAcIXiwNmrLb7vGVpIccO7wD\\\\\\\/l1ibSER1c2NjEEuqH0WG7n2Pfp+QxC77iEkcDJS7ptMqggEol+pvuVd8zJrdlQDUUhXrkK6YG1WgA8GDdX4c6s1zQTN\\\\\\\/o97pCHy5HR5Q\\\\\\\/PH3wMxpPJdIVVpj2d5ifF9lHsW16HWm0G3DNyIepbuHyOKu5UPvhd6NYYHvqnaFtL3GO\\\\\\\/m1P0LRNZ0cM9l5POiDn3L2g+JyKPBaINAZMGtUa2FoMMZ0cINB55qmp87FHpwFViT4DiewH\\\\\\\/WCkqUpnzqiCzfrwLyE8pllNskRkKHsIGJPGqkMyd0pZdT6cg\\\\\\\/xpyg15rdVGacwxTmuRY3PJ5H\\\\\\\/bh81qLIqbuKh7gMy+ENYx5\\\\\\\/UKSq2MRYx17DNQyAnDmxMm+VIhotSd1GvDXdqSfyPezMtl8VR11Bo3STy3n1P8RsUO3UwwlF+ubSOJnbPJrqp8LmUSbyPGMx7Eb9oqCg6sfWAUh87lTXd0SY3ihulDIgVp6rcTKsYMSolbN\\\\\\\/pvYZnskf4EK\\\\\\\/gZxLEvWdLXMKAegOFdIdUMzpxdDtRwGdbGFySyxfZ\\\\\\\/tGszX5G8RoT6SVcNO3BH8Iux8eYgJBH5nLwbzXVmN518yLLN23rHwc4PiSGmi8cteSf+c4BaeXMnzpvluj6vrI5gzboCSYDn5lv05HodKALUQYfB3kcM7YMifkedRBzkuEbidyUOlpRhoUbCd6VMhmzBay3zvlz0nMYX7ywJgO7EDUc72hY2G4ZzaX0gpoK1urpFN5nGOcWUUJ0C01voxJ9sHTWqgwofN9IAtPZ4BsxMmDXLeqnhqP5bcBXCtnx4xCeoQdNHSD40y2+RAeiiWrB4HkCgKQ8DKTxXoiHfz\\\\\\\/PmCQNkHq3Fkmy6e8otrHj9Hi+Ub7430CUGhI0DQYDEku0EnKkfTP5gir3rgytFoSMnl005T5GroDyK\\\\\\\/sVIHk3s17JjrF8ozqaXcZaYA3VeVDR1K1NHhAOyIRmCncB6g6TBWQZEE+3itnkcZ7URotFU+Lu\\\\\\\/V+zBWxIsskoYAoqRPIRDgcK+6b4NJCz8bi0gn8FXQ9ATK8u5KZegSsP2sN+Iu7+Z8EwRY1HZ51LmDxNMWMvmzHWazAcLQs7DidO5wFMNbT9DShY4NAM0Fv6j9W+o3sjnudaxbgn+Wwt579H\\\\\\\/pOvsILcavSDvML6LP7ZsTJnmsQ59dELdeu2WX4Y8Aaz4bYu5bw3Eba6gZDwVJYCic4aKkJhV0PQs8hR345Aj8dq6AtflGgQgyu+dhB7iAICbA6Il5mBR0pkVTY5zpYQ2CZI\\\\\\\/RnhYWdrRais0EphpOYk7rxjiWv9vejSaJq1c8cXfUui7BZ+Go17IT5OFcPUgg+WVLapKgoT\\\\\\\/h+JWCFjf63xEFaZ1pfHe6+BEgWXXH5N\\\\\\\/iZ74EL7Bg1och+envboZzruh35Tsz\\\\\\\/MRH1AImlJYe4+vOYegjwOen1ctdrXVRDetGPpg2iof+IQBbo1ePMjizm44dtzmfTaq0BuYNEY+R1MeYboB9G0TwMx5lNE5gOmifFUciIQWAyiKzWaCBOMnSCkV217QYEyflvvnFef1oAI89LFDtokgy14+qSBLTWIeAZ4VhCjxefe50saRu4MixPcJOZRLIN3\\\\\\\/SIEdbPqLZGLqXCJGkUEoEhP6l8ofmXkyjR2gZNxdXmPKEPtNI5OKNi6KaGOptRgdNC+9WYy+qhiguKEb194eXp7jvatJ1gu0M1lWWOXbwjK4Fq3NCgx8lLapD6pOBxIJQqV79MfmOzJyLn+Fsib+9ir3patnz5byud3znLLlaRX+HXPLgUhTsa8o7mt6Kycb5mOlNQ2vzm0OTHYWy00Q1Ve5ztekl4iL\\\\\\\/Ml058jAgsQxr1qrq30NRYYAKHKNdVxzZSfC31N4uSpjJvOMqkRoEHZsXu7d3hCBoByvEezY\\\\\\\/tZQ2op+FssTM+Q7AyhOTWWUNB7EYNfyd6eqXtVvrLvRIWdUYslbx7fS\\\\\\\/dQIeO2kiMJm\\\\\\\/nw\\\\\\\/XiSTNlBV7QD4CSrJ2K3haUmEZwu7E0JJV\\\\\\\/yt465XWBpJEYLSLfgdjngThSflHdngH5SnZ0BomcspXbEngQwyqQsLxMR1JWHHFNCNMQWIDmTOLTCG6GI4gpPx0p7M7bwv4WnncvbJE56tqksyX19N9Ff4od+fl3VCU6tpHcKkpuTZvSae2DbJFLi8W1YVz9JQVJ5jTFYpa\\\\\\\/bYaD4ePCOWFA9b61ysPan3+mMx3YGc6hD77VJYSTcYhnyPvWlsBCG\\\\\\\/0tP12jsp0uZxtQHAm4n1O5S1C1UH358CM\\\\\\\/Yoi37hSd4ZIDxpXn+dvCuSiMEZlQlteibzP2fbRDn1nx+m\\\\\\\/Zyb+nkPRfpSldhhffWh5LmPlvKvIhdIV2FxM53C2toFyx1\\\\\\\/h6VCLDEFHujYFeT5FUq+ex+NDWJmNluNEOkPasmkjvMmKkA7gVGz8vKtPWen8iQqyU+0LkZPf0+RKJPdhZJfQ2hFJYsdLSTbgQvkYtkjrYjRVU\\\\\\\/wYJhlEuW5zY8wh2KRYBLMzW131Xo+q\\\\\\\/Q429qFf+ukF1XGqYKa8iAAhKxeyQX8FGzCvImrrev1Salp3yC\\\\\\\/I6H5eaoDuxlRX7kt0QWCMPhKuaaB4Wxm02IqTdS6GpyNMcdwILxi6b7t+Q39C4Wo7SRU\\\\\\\/dd24T5LZHYmhjanSukqw4WLAqw8\\\\\\\/ocOu21G7A4JqRyHJ1G5z8EI2BD3fsvby6wxPiUKbzMR0dP4IVZ86RLkDPDwFe5+5\\\\\\\/juqye2ubNGitc+K8KawaD+s1ak2o8uRCbq5gRv1dHe+nxOF\\\\\\\/vQ6s4Mp4W1JmD6enRU+6FZLp1\\\\\\\/imYNboo6KaQMUmsKN6QZ2FqxEm4jIRd0BpCcaes1RB5zL2thFV0xWD083CeG2fGRtnXRAXzIX9CMp6siWJud7v1o6RuEq3hgJMNNqKXvJRnKpa4h\\\\\\\/2VdBsDiFwpXAd9AN6z4wvrfSry4AbwkFvWZTiE++ibkogbqrtq18Y6iq72QVJIXfB9xwG2LyKDzbd8jMqVbmDNIkygu9sHqHJleqfKNmuOjCDztHtAx35feWFIJXdA9tgDRw2NA40F4Ss\\\\\\\/h0LnqHHmhgKsfK0MQjBt5Aa8uyNSsIhewANB3xOkB6mvDtVw1i3lupd8Etb6dzVFMXBYvFjRoVhlqsOynOpHguoq8lD5Tkrn1ogPrQ8t+fC4Asqs\\\\\\\/TAJ4oZ+V4t\\\\\\\/AgjCcBGFrXrURI5Rqk5Jiz9y9KEobwTizp9xsAKfaO\\\\\\\/XanS6oJZftIJIHY+KQs7AVGjCXuuB3Nfqt3LlMmk8F4zIqbK1W7+tSqyGt92FI2v5ifJD\\\\\\\/Uctrn9uOWb47WA3hP8MieRB0UTZOoDmZCRTriwxAvj7qMtwNO8iXOTbXMpeZ5SeHSyU689Si+gO5kJ0NaNHt6BuCYCW7XMCoR365jt9I7RqkCCkFiAViAmnbAcLPWdjSFIWCE61mv+gP4tABvADXAFhm6s0\\\\\\\/g9gj4cfjK12\\\\\\\/ox9MtQigwIZEj6G61OUXqElUFBZUQlLJu4qDUK\\\\\\\/HVuGhykuNnY\\\\\\\/VD9ZY4sgJr+BIIEk7r8KN1anfWUh2dgE9yr\\\\\\\/IWnpPW8eNe9B+\\\\\\\/r6ClOwsvKc0uj5VkXSLTp9qPT3mHJ4aCliNT3unKM4oWQ3DZ6EOdTEcE\\\\\\\/By3iL4ErU+EmJ2GjaQ9Lx8KY4heDLM26KpTA5SHFG+5Y3+GksM6Gj2jLE9guhkDkr\\\\\\\/seRJI52iDQyJ2fZ77\\\\\\\/oW4wGkTz+foJ09W9fk1\\\\\\\/b0m3LWJY5TQezcp6CWbrAQFS0pVQPbCxJjgxpfrrj5xfsZEhN\\\\\\\/o4aaetP8ebFxcpcTiIeBt8LWMNreVcDl0y5Ud3bIuPUXgLDo44XXIXiIHSAXnhnwIUvVzFd+upLfny5jNftkqt\\\\\\\/1NC8iltidY1W3Dw1MXIU1oKJqngpzIqEIOpbC7TMb86IoAbPdoSiLpbFbQAGm1VPW08i8oVLWf+e38gBCsGEKFkW8YoeophuOhYNV7GbKDq7ll3jLHBk99hgQeIt2MRRCwkl2j2VGwqeHToqOWaRkVkUQNSBuYs92bSBCe2npcvwS9AYeFS80XJD0qWq6eYR7bar2bQBDJVsK77E43nqZVbjtJmg3vvboylq2oe8E8AiG+PR+LiyzHqUIjFpj2C49tCMytaqNF9mfJLBLJTg2tbcBWdrK\\\\\\\/7C4RWzIusTxdCGCXE1FD2SKV0YbBjAEsABz7d3eyZuLKcaz1vUJSmDUJTP\\\\\\\/MhqMPjA07cavxZQHhmtq59K0JMF6tXE2qhLBPy2Z9RdblBaWwZvL8cbaxh2dE\\\\\\\/591T990tXVNAQ6RsySYkzc8suXJW5fgANHDj+vpa4jrnbcZv4DyEEwCoxBfmM7RmgCyzcWNVrPumq6HeFFc4cUfB0D1zjzB+XAhyMWKk5jvvOkjEDdAAgvENovtOtjyW7SK6Vgdr9lyLnh1hraM12GE3kustCm89UR237kI+g26Y65dDlWGTawB2vXHFJat6Be8pOySbaaj9\\\\\\\/PHSmOE\\\\\\\/j25I1vlwRMli4Q23EWzApVIaAA2EL1GQDBpjjhKh\\\\\\\/wxqoGAAnxg4JwRj9XUZMQbDeuAf1NvCQo5mRbPoo5EX1o1\\\\\\\/ymiBElyVHE+DIwyVEcT6UsKFN171bY36JB7217oMsLrcszphP94qCCRUR5iNVqBKR\\\\\\\/Jdz01mN7txc75C+gKmvzT4T+Y8whrThYc62xnyFSiRWBjVLqUn8j4f\\\\\\\/SGpMXfpQ0hLW08uHiZYnv8\\\\\\\/drbISqTm2s4QkPiSkaJZ8HlvHF\\\\\\\/WzE7u0RcVoEdmsUtgEb0lOhT54Er7yqVl0+6iXLgf3hGnf+DYUOn1H4wNdok6jvPZ4JWOswy\\\\\\\/gZkuAKvSG\\\\\\\/Ci8VDQkQ2T08UdLn77wWWHruWu5AzUIGRUdBpvk\\\\\\\/Te3jQstZIFLc08\\\\\\\/PlrGWh7FmIY+mA\\\\\\\/p6lS3uZ3KhG1nu0Rs2X1Ugzl4WRcF9Olwb3y+YiBVjjUdGld4A4rNwxbIg7lGNxWURsIBZyd7PILkCuFUMyfoq3a6NHCF\\\\\\\/11htwhTZN27nZih1Or3VqYW6KidJIvYi6SLDb5wgDzL2pPItedmnsWnUatGKvbcj2wj5CUzYSgNm6fV0EsVBewW6MmD6bF3CxEMZYNc3tMMnnZEkh8zWZ7kcgDd6p9xUwVKpahnjg2paxIjdyVmDhU28vs9HM6mBVa3BVR\\\\\\\/jqAf0qDbWhrVlHpwvGL3+kcdx40pVezXx0coQb21ryJAk8oINaM\\\\\\\/AtWclRYSxaTf5YsDQy4wiwsf5DQ7+b1eq6W5UIE1qt7Dac7SgfrGAOF2OS4xr6JS+1VsaHQLfp0i+yH4dfdT+RJmtdkBcDaG1vG7\\\\\\\/yWLXRsfIDw\\\\\\\/SB8S6XVkNOOf4S6N5s9LG+7M18bG+NFFDJlVDuzKXlvd7xHs3jK3lgU9gbDNgen82NRbr\\\\\\\/\\\\\\\/toMm8LWbKXzun6B7PlSgWcKJoJLm6Jk+j4EV5y6Pf2UG2cxfiT5b9UAqId9xOv8RiwRpy7SN1K\\\\\\\/H9SErPdagN6zn2r5hsEW8dgkBYIwX96QFq5m+m7m5sUYrSVIFWCBOVgIVxIlqqVHCMI7QmSJ20jciKvtSR0hxgoWSQV2ZGzUIr8b5y0O6igq4dAVI0lL9AykujYlQldj7OqRjnfQvGKLenr4AXZAie5dwePlP1JJ\\\\\\\/O+MnHd8CsvAD5vWj1SYRy0+Av3Lf8\\\\\\\/AA14Iz23t3rHoPZTS9aPLK\\\\\\\/OUUlzf880qYxp\\\\\\\/oLxfO+s4fRhYR83CIPbdQz1CU+nqPX5hXwCUdLGwR5rNcORY+7ldx3gWDTh6a+o5kq4lnoGoJ00PD0eupyM\\\\\\\/jih6OACOP0nMu43p66C+mqSOWfW\\\\\\\/C84sZ6O\\\\\\\/iVIoX0B4SN2lF9rTOM\\\\\\\/QnBqvlRmZt6yoNe8J0qMX1WwFWJJUxvXQvrzNsowfdwNcmB0Dh4F+WffjRoOlUxTxJlp9xneEVV9KmiEMP0\\\\\\\/sdfpP+YPP56fVlgD\\\\\\\/2IhTWFYVAQrRVrQ1\\\\\\\/rmxPoUIW17dy0i2bWmKfI2\\\\\\\/uHhUd4P\\\\\\\/maZXPyMf7t+N7G\\\\\\\/h6R2sb7zaqfLHb2PxNubWoP4wt09JeKm9EWUQzsdHV3g6vOpPq2UTAEFuUhh0NJHdc1XUCxgGxg8tIK\\\\\\\/OO7YpFs98IHyurPOZXoqrWD8yzB7vL1atSahu085Fxj6v5JVMn8GO4kp\\\\\\\/kWk8yaP8hJQ01BK02k9hNGHUfFblnHpcXqIoBQJqNbq2Rk\\\\\\\/fYjUqWPqpqWyZWjxMAOdXRfaLqCVqzwbogK7M4aOhfADugkxeLs2M8tKUD6RLnx36ofXyDKh6J8rs5+nwO1W9xYtWkCs4zqyVEGe0\\\\\\\/KoVMBbAWIHkqcBAKh9ZZC6w8GE9I\\\\\\\/KLhUgPDqBtWzrzpfze6c25fZzd6FJ5UVM9df9nNw35Y4qmgIBFocwcRsQs80RjUPiTw28BdKnh59XOtTej+HPLRlsCt11NrQs0PtE+hlcba\\\\\\\/GPHs78+filzMknzj92BNRLDWLsyPFOOq3HvUIBgWj1ZJNu9EM8\\\\\\\/NkWh1zx25S3YwlbO\\\\\\\/mBeou42IeJRLXjavfFf6N+8\\\\\\\/qYcswBHfTaAtUmssf5ub96Grr49pRGp6GWKWMRviXa6TKykJ0HrjxS0YZIDBA5Io\\\\\\\/H6WBuv6C7ED2CdV5SqiIzTlQQ0IT1nA5Hzype+vMPx2+BRie8+jz1BqSOoV+YF1KU13Ncvkv2hL1Tky40TSLu4JMd8FtHUsglJFjVpYy5UhZko7yU98pi5s06ZzxRS01Xzb+YsVejth1nQ727O+ToZRkE6SSJCW0z15hB8KuOfcED5DWJ0ZI\\\\\\\/1Em2lVBtLVjfjMX95zqOBWP7f5JoiL05xNEg6wBqacYWGqfAdssRaFfwZqdj2rzCAhHVXUMpuu+1tPqi0F7biPcFZhPSZSLxHLmqwJoudHQ0BWLkzb4jAXeYTFJZXlR7SOxg6G2lBgVPzbHNgTLZ3nDoviY\\\\\\\/AI5+pzvfhEbEuSko4VGt\\\\\\\/l98a0p3Yn0UCDfi6AxODEGXr8XQmwidcpN5ewcfUByX9kFlucwKKtOecfZSG46PqeMIUEizcNuQl8NcClftq2oPuUzEE4nVCacPQBt7B4WqJlQEX89EK\\\\\\\/deoBdY\\\\\\\/MjxmFD7oFut06aje5xaSU\\\\\\\/pO\\\\\\\/c\\\\\\\/zLdz394NBHEOmvSEyZTs+qTFiYctc47YZParuqZPHpRTq7O6trY09h+jsecodG8+KcEVYZsOz7zxrbmRFjGiF0nsaLFhNMjSwNEMW\\\\\\\/ZobXZqOTaUVCWDkrquf9PeyNRMS9Con7al7wFU5HDr3l1NttNiTzfDLVjYQmcPGBZ45P6v5uqP9BgqHlnc217oyrhCtkyx9\\\\\\\/kXRXsidfMvUN+WDy\\\\\\\/6d58RNb79NtvfkoBHrZ+aWf7a4w9yuWAhs7gEtd9YEKQvrc7MTS95k6HRR2wo5mQiF1xaEt0X3xOhuVt6zdVUvoMYeCWsODej5IgEiIhLLAPn58\\\\\\\/Y4tNzKe+iaXO3E2h4NnbfSg00NjNBLXF4ZZusKcBV16DfrySlIwriYBNvlxsGgK3ZUhE1l+7aCVDliwTLnxVq7OspZoIWU5KXpj2PSIOhrp6mlKW2XE5wgTdqGdBNGvryyWsa5q2KMz\\\\\\\/jgQf1dfHXDEPEdkoAOkrWcf5R7R3FG6yt93bgaNzRUR3yTL8NQjQI4n9psG2AO9TDwHobD2sylgs\\\\\\\/AvgmNzQsOAQ5lR3BtCoOsBSjQVaGyUmDycz2q3qqw7tTHJZ\\\\\\\/oBRfZMqPRpRn\\\\\\\/YE6AKiaaLss6p97XePOV2ce+5IYu0frUYTRcYb3\\\\\\\/TbAfXnjNyM84ddoTU6coLOQ7tpreX6JHNEsLVQnC\\\\\\\/hpOzQP30dY1h2FPPvRPkSCy\\\\\\\/dM1pLC6D3uGAFCtQgRU+HWbo9iBEoyCuAT\\\\\\\/ykBNMYmesiv0GQ1JBDL5vp7XPlbsEWqXmhETaculldXiKMIA9WGhYcRlPjVw8Edb3TnEkB1DGvb9AmqMA8fZPYBnYvVUMPhi9oHuEb9YD0+jVptVYuvKwOXgBxv7paPei9qdnxvWrvKjaRqrbc\\\\\\\/fy6OXPFLEkfGgpy6gBGzjj65OxYbr1buJill\\\\\\\/uVl4H2OYI5viSjz2rEVZdLn2QLB6YeoCWv7Eske\\\\\\\/fZYUx+HFB37JhmC7VakELIKmLtGCDzFYcG6Gu803XLfgLasVNpJOp6cAkeDghHbTEivhAo5Hk\\\\\\\/2AXb7reTenkvDqDbI51l8axMQj1vPOs5mdDZFFWOVGZncnBIRVPucmQl\\\\\\\/YaFBPgRnvpSjaoomnYj5y2CEij8SMwaXPYyr4uv81vigxnpVGdI5VR\\\\\\\/7K6BhWNNFaELPCAN03ZNHSgcFOsUuTBW+qytwDlN9HL+WtWevf0u7FT1ysBjljRWo3W3\\\\\\\/9\\\\\\\/tBXBMO89s9UnlhhT\\\\\\\/FqtufmR7bMOPfNNTvqZXFIot4FQCJ+5Je6+Jy7SHmSzRPfdGF7HBZ6fGNTwypWqr24HvIqikz+b1cnLLezl0wQV6\\\\\\\/X1TjmU\\\\\\\/xm8AuLYobnvhAkR8Ry2dUmul7MUovEX1tY2WVj\\\\\\\/S9PAOXhArJ7IGa7Incu6un9V+tppqyOV6qvawIDchpFxcO5xo1rgWvEvFj\\\\\\\/5BDf53CfHlwwCAzVkWxoarBIC1a4jgUCYuPMf7XCdZzG+7T3MJZvXzqoKNQ8\\\\\\\/FhsneFvZaPq1oxYRaI+KME3XD8DZ6ue0UFUDXaCF\\\\\\\/py9Nm6ZVJjhuCWZyxX51IZzbCnn7eDKKm52s0WpP623zYCbpSGSs4Kh5kPWk1Vf9+VkCZZk9oMFIrLg+IVmkpediLp1V2IheQ8pkcwhp6I+gaOfBryUW+v2PmsRMGBXlQ\\\\\\\/5zonsqF5V48MdgIhxMP9YQWkHWFXgcxJR5YqERtaUJHE00STcdNXQbinAs0Pd\\\\\\\/MyTQQYU5Zny+2aZCOiHoA7sBIyXn5pZ1cmNhTQg\\\\\\\/xcrHxz6dVogsCEDrg07Hxz0NLKJE5e7uhopN\\\\\\\/ZgNaE9UrNiXIxEnAjRXe7pa1bXQiHdTRo0sa3dyTteqSsZCmIt03krbH0p4SdMPWFfCfNNXpF6hirLdCvP2y6T51Yt0cDVfOh5\\\\\\\/5QJKVdIDYrIkqZw6LY+\\\\\\\/poZszoPnjRNHPCdbQS0gJtqDZQvcH9GpWs1eLxZSKZrShrAdHJJ8KaW8R3a9aEIFFoyaNOhaIZPQHC3JY2SCNUAgmAgTGFHEf05TFfro1WSDWtSHAMcdbCrlEC6qrp8aoKfsl50EqNbkdcJ\\\\\\\/m0LnMo18gmZNCRIjHLOeLP8zvjYL0ftAc3e9EGvz4kU9NnXsycW7lyV\\\\\\\/EuI4gl3F7cK68TmY1tr9\\\\\\\/xU6HatvwJLxFv01fxt2J7G+T1SSrUHQt7nTX\\\\\\\/6eegrJO4qmd7eo7PG00m0HBoN+FbPAHWVnXjEYr8cB3SmeePIS0Y8ak5LItvhRwRer4a1iiBeCo\\\\\\\/TGWTVhFamfn3AVAmhNi5y9tOfeuWTdqItNqKAiOkaiTTzCs9IIC3SsVOliZ4oi6IfSZZ3PiGULWWdSF0Xr\\\\\\\/\\\\\\\/dRjrfkWaSbzkB0XXP5RStSHx\\\\\\\/LHbPH0gHQUEALXotkz\\\\\\\/CvJLKb3cfEx7\\\\\\\/klxlkkF\\\\\\\/W1AQNbiLRXUKhfB+93hRfuSRcpgXMfDczoNMfUCQVwihvBvx65d3I62qIDzdo7kWqBBFmHteDw4TaJdcXcaO7TJIjo+jS0shoGmklwBqoqpnHEXhpF2SAh1C3NZ4kXJMCt97DYf4CSqh1sRtXMRyIH7MCuqk8qQvayqw+iqX0ykr3t4Rqx1GKs4Exctpe22i3niUYY9XL92OwknKo4OFYoKC7laUZnnGVaaE4TrPphiJ0PR++U94kiP5Jipq04ViIav62rb7yT\\\\\\\/aYIbJ0RWZj2AtMz3KnNaRNu7v6msimS61opbiUrp1KsDCDno54WDnAiLpRgxAblpeVeWwLF21NLbQYQuJPOBdJ6C67JlPqt+ZiHalxhgPwxLg4O4BF8ySCqq\\\\\\\/ZtlRi+tMdAguSA+yix+Oaf0mH\\\\\\\/bcKb5nTXLuEilEq\\\\\\\/k+HIzaS3Df8qdW0Um9bSvr+RV61dPMWAw99QhJs4m9+8vc7zgnwGXuILsGDIyhSZxGTimaDOH3L751m7laAScN0R6VqS\\\\\\\/cvOxyY31BuHyxeL6P6QvUG\\\\\\\/jKVGZw4kR9rjggtRAwtQMeJRd3CNXk3Eh3fSG\\\\\\\/DJxvmrU0EpuKIsv1Hz5wFNT1fFsdJkbwg7Xk3dQYzOOKFcLEOFT2QuEaRp1puQrDLg6kBLkeqxt0+yCsPwTjVUtCTWrLBoPSbPQ92qa45DierWKM0Opiff1grC59SoLF184jegmIt1GiVWEHxVNZUG6mkiTis4wmsKzFR9H8v8SMoe0OMjJKaf17\\\\\\\/TlK6e91otmMdsZhqwvwoaI9Oi48NPHW+uvgAoLND57NPWADH+rjCjl9JT+CZl8sWObhePhWAQUE8wjN4OFlL2LKRS91La7Fk8Yu1BpZB8\\\\\\\/jN3rHsPBXmpZfdT8YrjdADbFfYr1D6DQuZ4wo8pWl9Y7Jqjt4OnuyQ2J9esXP8VKyMjGScRI3cFkpaVwywzSHU8eCEzn2ZtVxLBOi0csRe8McVglBiq2NFNAmNWQ35Pme3h07kgU8ul9ekW0zLaWtFssdjlhNXhrg5oFPqgtpthKV8Ovnxk+Ica\\\\\\\/qAu6tbwD3bpuTqKsId\\\\\\\/HCviXVoJ5VqnveIbgqmaJXLWNs\\\\\\\/nbIEpDbpwKl0S9AFxaADho2Vrh2lPYKKvJw9TWLXXkduWv8iXaRWLXhwsZeznqimiL6dfeCODm1+p7KfR0Gp7Zc8h\\\\\\\/D0mBW5a66E0QefnRcYtBOoROPapBO4HTC+uz6eP+TFX7qoiNn3c09MfCyxn5Z+zuFivv\\\\\\\/GTtuIJDn77B\\\\\\\/SzH4lutuLJtzqJ6rJ4UAbZS8Hg2jCBXTNUUBSt9IWitdchb\\\\\\\/frqhVYmVNM\\\\\\\/OKBN2YhnZqGIXWf5MPvAOkVNJEOgJMxMOKiDiWdB8OkU9DtgDFshplusL0XPHJgrbo1SOkGb21d\\\\\\\/aMLAO8chXEw8oS5T9MNXWEnRjq+b6adhVSxWeMoCsULUDPeQV5IHmMqz8BuxYnz4\\\\\\\/xOynoPKN8mdzvTirvy79jLFgA+9gx27LazDUXE9xZIIJHd7JFwShcqohbqHOmqd3oJ28VZe7nQ861UfWXgeem5W1IrJvpCtuM+xVC\\\\\\\/JJjx5gemU\\\\\\\/nidIdnz0loyaHR0XT6vcpZL5A1TQnwDsM8pJ+1Pky+VoTAiZsFnC+fnKTjtl0jp+046fJH\\\\\\\/HLwv0YxEN1NbENcOz0MEdbRhxyzNF+LtEfl\\\\\\\/ezf4np6Tdigln4sna3letmY7GnAwhLVNxT7G7pQbsNs2IwjdkHtkTlcSmYnGwHz4nnKKY22RsgdfSds67ywyOxtQGBneRxR7voQ61jd6GR8pYpP4rCWZwx4wbjuADIGpdswu3u2xcwEzBRd7aPC0w4lakqhEU\\\\\\\/0\\\\\\\/pBifNoTB9Z1r9C1jfx2J\\\\\\\/NtodgUoYLS3BTC8aSW9YDnKCsXZqZ6VO9y85l3L4Sui8WRU5Zip0Sd1k1bK7D5XmExPIkjqt9Kkownsu7rGj7IgClRc63HUQuiuTcui7xOgRfT3p1t9wPoLbkvWj4JVXBh41hcbwibS+e8BrVIrYsf7nlIjw49WczvNfCv60\\\\\\\/JPUGjJvaxh5xyT37GZGG51VkutKNScKI+VL9ui2A+qXdqsE1wxnfxcJ\\\\\\\/j+sGKDvufWqvsXExqK0Iq5gCucIVe2k0XrRElirag5jlQENfkABUSojgbeB6oIk+00wCE5d4yA1nPqLt\\\\\\\/YCzaBDPkgtUliwUjw9IhOj1bTX\\\\\\\/1wFV+itFrB7IRIOrij4EFPkIcRCVbtbJnsZQJ7p2NvopVYJOpo8iGNGVraIjtFR5isKFDyEat99eiipiQf6E3JanLl1CdCpxGcPrC9LoX+RWtQrjyJfEBysPHHK+Xag9IPigqGKFXwVOh4FcmEdCdP87iFCMlrK0ZiNi0AfdgQLnJpRGloQUcoWj3RRDJrsG9RTdNKA+rb5s7M+GGpKxhbfoDC8r1RHsxy0xYUlZCIFb4s9F3Vr\\\\\\\/5QVOuZSUIIGRpq4RJpXw\\\\\\\/R3CJCyQ\\\\\\\/ay2JJh\\\\\\\/MDCv9HeCkyQbML6fM1fBoYRqAddoufpi\\\\\\\/IjMuF36h\\\\\\\/ENAH\\\\\\\/7IqllirAi9KibQ+bJm8O+bLnRC3Vkp4MHczljEOCGzo12VDeboaFa\\\\\\\/YqpwdtXtK2NJIqb1avhKypBtyD1xupND6\\\\\\\/cwXuVo+KdAdfMQzWYzAuMlQwihGPl1yju1Pr6AqXPmrp9F4LN6bhmaIUH5qm0c2a0\\\\\\\/fV9TuZZie9UXeUEJJxyz6RGrUXCTUgWaSpmHCb3TzJicHKsrNMXxq5pZS6jP7URq82KTQrHTSVdA5j62OVLiHsgZ9Tt9gIpdl+nRxYuSRpjiVozZrmTMFsEEOy+pNqtyJEHdkHua3VPX72u5Mu6r0pRVGP55dASn6mZz0K+MHLS5fZR0W\\\\\\\/nuJVneurD97yIi2ssMWjqKD5Q4nDzdhJxQv44ASci7lyT\\\\\\\/DJj56WBWPXole5w+NLoscnfFT2KiC79r3\\\\\\\/Ufx6+6Oa3ekooJXtCbH2k369TvIDDkCFbGcUXT5YsHR6khvzvgpxmHCkVDYbqQ19PXsPLM5pbLkifwne+IkYkG0Rfv+XjhzRmHkilFNVOPjX\\\\\\\/vHW\\\\\\\/A+9wJHxz517AASs8JpHFqOG3gmCiqQvN4xyc8zT8i0AQyb65Sti033IzcwdxTusxnKNgJnGoA6D92smzz8InYmo0cXviF\\\\\\\/a9Vh\\\\\\\/1efRbu+pE4XvLWciUOk0C0l9dpVwaMxnv+Ncw4qkbTiXxrrFObf17GFtEHcmGlyjbOx3KBorMobPrXdsPYZJDQb0m\\\\\\\/0qMFRspuaA6sDlYmR0KyoriCFny32937h+iika3eSjI351Y0sDjn0m0DGjeYiQEqXl1VPKi260\\\\\\\/A5umbThxvtPWtOrMwcd\\\\\\\/MyYYhMvm4Ihter7eOGnlYt4S2dbTcTAC4BbokmH+eHRurRhH37Z4h9VpYxJU\\\\\\\/+KjjjDNLv6UdGtPq2WwHN88VMo1as+WMR4xfwo+fX5o66U4SoS+bqgMt0gk6dSwQK8\\\\\\\/EjDjX\\\\\\\/KQPmK\\\\\\\/mIUxz0G4DipQXzAfWA2hHLqvP+b7lRAtBRHO7GfzhjNhuX7tRcBOkImnVDThQKv2lBciZV18w509Qo9MRlNXzvQqcNzGWg05wM6V7xAx0haz5ZtjBpDaq96oMYZ46NCV\\\\\\\/aRzLGmX3GMUjY7ftqBLqY20ToBzz8NiMY+54vzdI31VORI0GgfRO2p35YIoS+5ace7GuREMA8auSf1+6f2Oa+C\\\\\\\/BoUSbdlR6I5rKrIizQSvfDr0EZxxvzhYJ\\\\\\\/bXTdqruPVVA4+abWJuhGbNbe2jpu6DdxDvbiAk4gGI8jrRuACTaqpYm7iVmUBCRnYhEHu+vooXAcdSuPtszwIsUYqYgSgwhf9xValz6Ebr9FQMEdlFOpNZ0NIo2eyooaIKt\\\\\\\/LgtWP2Fby\\\\\\\/5LIUGMB9XHK9Pmc0HYnTU\\\\\\\/JAlHox6P0UCuN4dHeETpW2660ZTKFhjl0936SkUb5ldSpy9WtX7Bj9Ld7RNgUJ6eHAdxPL6Qrg63SfV2HQocJjdE3lb+TH7gm7a\\\\\\\/jisx1FpZ5fw6GdQW1EwG0n5bMleUWK\\\\\\\/RS0cxym7gNVC5Oc4D5ySnneRn10OVae5tHQqid\\\\\\\/X9DVsR\\\\\\\/q6u1brNthrKhyQvIxdyY3fuWf0bdkpUwNYgcVRN7gn6SmoEDwNt+E+WSkCzvBPg5DjA0HoviUhbdRtrg\\\\\\\/gv4LDkzAopiT5UHTIFdnTw3LK0jFWe9wZYMBaABj0b4suuVNvYO24bKO9pwpnaoCGZoINpq6cpKTHELsdPqotvZjYIkR2Jh+IfFX5QuwOpDnCceq+SQxw3nUir6OyIhH1SNv7BATkyQbaKILdV4pPYMUoYGuz5sDQnfnD4EKTH06uBC\\\\\\\/HuDD7bHiIr9EEpf1dbCmTo0puM0cwNxOWToDvz+jF7Z5JWdeUnsluhgpICrvJfnlKHe6BSXaxwD46UqvsLp+jFDuwZptvunDmyFe2afd5Jaz0H4Ti4ZIBpfVqSLpFhktRzxhk3PMXc\\\\\\\/MCPl38pJyyTrZhMzSfkBjJ6HfT2Y7IwiQ8+\\\\\\\/Wb\\\\\\\/rEJ8stel+WtmTXE6+LgeGfXhmFvtRSDNjwvWvxNOkWnPzZTpzby20CvHgHEl48GFvnjJ2a1yECXbLELkNO126NSDiR6jucXNo5PV6pio4azHPxKaYolVSHvRuoj\\\\\\\/e7CG3mPH7dAKPvnixwG5KduyQhl\\\\\\\/8R8TzFq18WWnGysxlai3XSCaUAEQ0I3rLjDdxN6o\\\\\\\/n4dUzz0WPBlW1ARRuOk\\\\\\\/yDiTq5NbmkXLhpWU2Z9hb+rB08n3enHpjzSeyuW3jhDs3Eo791zuOntWrO69ANBHiybB4v0IaFy3uU7WHvNaKymxwLa4TQRc+9D0Dujo9BXrbTfD\\\\\\\/WnamJg7mW6o9fMI4gOfVkfNmLP+HcEpJAW2BR+b+DAbjnBDvXlDS94Imqwq+hH3lVYnsTQCN0K7Uf7s4I6eKQQZNmVBEhzmrXeXVHHsrogb3\\\\\\\/1aua056q5lRlnGLHH85zpPURqLvrYFjowRWokQ9ISC0PgbVl93utZhdWFJEngeqIqbmk14\\\\\\\/TZEBeFHR8KVdddbI5i\\\\\\\/SBBS9IGJPovi4kKL+ZRA+nkSmS2xaUVle16ZWnQFae93Hssa0WACe9sDB8PaeLdgrX4woJGGs2zYFoz2NE10m4qkycacY3Z8IfVjkdN35u\\\\\\\/ViAquRNPufbmns\\\\\\\/FpmIqQfImZalUadlRflYhAzrCgMwDG7GhuUlmj87XecUtGLxEMrHur\\\\\\\/u5yyHRGjmUqWyBR\\\\\\\/SsqnCWJLeqKwabLNqyvI2y3iNvpDdm4M4OyrI9NWcDYx7U3dZhadLXMvg8ZWM4ARvRy4oXNSmWPpqA\\\\\\\/meady+lAVn1+jSisr0lfj+aOl9o2ir57PHXx0kMchE5K2j8olna1Cy8IHZyFyPLVtKar01tQrMdn3Xx561AIveCLIFmCrjrIc82T1oWCAUchlxOG853PKQvvuIdV2fPng7RPGs3rzGFfy7eJeA0zpe4K9Lck15MaZQK4Iu+0gsLsNTDEzTTF2AlZ8Bk02oybYNWNKelmlBqm0nQHeqTakyv0+ge2hgxAhRdyQqYAs2stf54PO3vRRJMWinjuUuchiEO6xaM0C1COtjzGWLJA1oZC1j4cksQw2fp8KRgZ3erIYZUx5gK43CLpbh8q1eCdrOWHTk3Uk0l1Eard6wUbe1OwSjoBFllKpgBgDFNnKtXF+\\\\\\\/0xDb\\\\\\\/LcK6Or6VvuKDNXsjSkB9ExYCgyMbwsL7fNC5XBxgZ6aluHow+MZ+v6LsOrUZSp\\\\\\\/lI86gB+adcaF3OXzYttmvMP8rYmINedKlqyDy7DP34dEet01CekUfJbF5OiABxfF+3XN\\\\\\\/5RUrjtR5oyvMoC2ufMe5vZPqYVUGohyI97HeJ0mm2IfzhJqsOQfWhYKyGoaHHgQorI+SuST39mBxJFaUyhOOj8zKfIZQdgYI7c2tgEGyCGM\\\\\\\/iBl0lppAz9suL3i3o3B0T5xrKsBgSAJeulnTnwxJpkHNs18h\\\\\\\/fmHhsMILf3OhNCtmWOy4zLZbRx\\\\\\\/2sbZCsW9ZHid2Z\\\u0022,\\\u0022iv\\\u0022:\\\u0022YBggSrtIdpimI\\\\\\\/CHuzAPKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aa533a9c65a7b9d26221b83b386ea63b12a88b88443f527fec46c4b7e12f3a1e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OtILhhWi4puX858RuqOot95jbSlcyLXGaEnrckwJqBp0MTUIdId8foTKoKlcvnFqjhR7gntLA2eupghyk+o+epz0+UKtprZwYZb4+JhU6pCtn6R4VqkYWis\\\\\\\/kQLnRRqkLrZBcbhOnbqb9V6X9EkW2g84mWvx0m9Pwq+J2VcT8wlmd8QPfvdHSbZeq4KjJvuNbiGs1z9SRKfNnh0lFNwkmexpOTS67FDpH80iliNs7tGCH5657iakc+TYN4e0gx6hbHe6hc6p6w8kr\\\\\\\/03j3f1zqL235rKWqXnB6NiRn0NAry\\\\\\\/VZhQNtyUxNnw\\\\\\\/G8\\\\\\\/dziAUCjRzy9b4Bk7uJ8554uxIaDecfAcZ96VQKwxHgZ64x\\\\\\\/XVZvgmRKY0CLGz+xXxJJEZOzrpmZbXU\\\\\\\/b3gtPc1QcmoI\\\\\\\/EolRRzLkFZNNcc\\\\\\\/aPGMrpElnV7qarSEDriimXY85pPk1XUF9xUcEfqLNoPontgsiGW3L7AW28Pb3ERe9IoKy4u9llxI4Y14tZz+UwzBgUFBDUo7s4fSwh26gYdhNMlm3RlCw9BtgusNx0\\\\\\\/md5yCm4EdPBbHkHGIfLJAdFGybay+BeSOgx+8+Pzzm0dd4kchj3rHPQ4GsadkfDSTe+FyH0eu2ViSxIymL6eC21v0tI7PaKmLuqiis2i00G7rjDoistjtPW7TEAYbhHbOtxkhTeG7Kg5404L5FgRJtDFtV6aZnZVe0QkrFLKLgRcPr9VGAduVKNDkYFXURY16l0sSIRA4OjRlskjiLPm5cviUkt5juxPLgDpaaC8ZP+JshPZKZNz8ce7FeLqax2g16HfTRjrjuc23Jf8hL+xDXFihUTiuMwc1ebIeFBlQu2ZdeZLZrSQ9C2sxJmJmdE9vqfguEOMGT7U3AvraPFzPbKznhcpnzm3EwXT1BWfioS\\\\\\\/8bEDw6gknQWkKy\\\\\\\/8Eap3QwbceKEMHIKCK8Yu7nX9boIcu0wij\\\\\\\/y0TzrbNNXkO69iq5C782oWKx111xX63RKzI6IpefjuJcFsEpoyambkMVZRqaGXOTfa47CTWVB97aBZhXPS15RmNXyWZPAIXK9qfwPH0tuvh89LLVGK1SPvnmRksbVmPEs4klWLHEspSVTci8Q935vNB\\\\\\\/EfQhHqoT27pVAmGOA1AxDvaNKlzEv+x+fUN00SF20jNzWx0SpFpUCrQDy6Z2nbQtNs7wlT3djokzUpMXpclc5o8xsh3qSk9OXKTHB15bJ41rUYSnN7cg2ix0kVlF+Ht9N\\\\\\\/Bx9Hcw\\\\\\\/vbag8vULJaQAWYO7qQyFJ716PNJyjJ\\\\\\\/2bdYAa9YqBo6kB7Hwk2KhM1cN4AHisVTJRoUPBYTqZYZw1VoUsGm3WoaSMvOmkY6QLkmyWZdx\\\\\\\/O2LdtH8iDtXgx7rN6aNZQ2RHVKPxD7FFMt36pf+ZMdRb7edj5EDI\\\\\\\/qVpCWOAtbslw5YG9eLUj3\\\\\\\/O2tlX4puLHpsmGgyeykepPRVlp0+klNFhO1LTvbExfhVT6ZGx0bRvyby\\\\\\\/Ugu04GvpPf7sE5oCMzXAEE6Fm7i1yLSYzSQW5wOEKLRb4gq4KUUNDdgJmAElJSlAZusCMiXJbr+StV8fjtRqkek4CqvhvNsS7RZitIEnmoYs4vPVL\\\\\\\/bZXfy2m388zsKJGgZYDbSgyTlZEEPVOTCKNHKVi7iqtgEA2GEYos6jnexOP4YrtARov1MfuAaPDEn5hPmPZLIM7JnF8u5JnNBZg5a69OBVzwr71\\\\\\\/y1C\\\\\\\/tgDdTLpPoiM\\\\\\\/rUYl6+jVhYgAW1Q1Cqc1VWPBAJsVlNsYqRMzJUNTl16kONBISvbflk4rX2K1aorhAGd7G1Xsnao0Le9erT8ZdAwl1SuX3Zetw+EJAqvGSjB7UzaOChrLq+ued03I7WrGKmErWITRDiurd3Sb8fES7gCk\\\\\\\/\\\\\\\/qN\\\\\\\/jSPueFhaG+qjNoAIguJAqkHGV7ThJXd4rmsutf9PUziVss46zcO+4A7M3EuOjLK+xwhvv4bKtggNgK\\\\\\\/zbS1vgFzOy056Q5IOXy\\\\\\\/IGsaW7yyTUwA5cb+dHWfZDx7SvR5aIIcIMtOYI8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nuSTyYMNy4LwPbguVN0qvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d30a4b3385182e524434bd59cc5b52084e787db88bb81daa747177eed1667970\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459902 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/6/1027" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dgupstuXd9X9g8iFf895jmOZX2useIx50zngtlkk-d4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/6/1027" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DPcqAVfrcfPyIEnsMI65oV6CUCbiJwHZCU1w43RZIoU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/6/1011" title="Sex-Determination System in the Diploid Yeast Zygosaccharomyces sapae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/6/1047" title="In Aspergillus nidulans the Suppressors suaA and suaC Code for Release Factors eRF1 and eRF3 and suaD Codes for a Glutamine tRNA" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459902" id="node459902" data-pisa="ggg;4/6/1027" data-pisa-master="ggg;g3.114.010298" data-apath="/ggg/4/6/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Réka Howard</span>, <span class="highwire-citation-author" data-delta="1">Alicia L. Carriquiry</span> and <span class="highwire-citation-author" data-delta="2">William D. Beavis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1046;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010298 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459902" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:howardr@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">howardr@iastate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alicia L. Carriquiry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BL.%2BCarriquiry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carriquiry%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlicia%2BL.%2BCarriquiry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William D. Beavis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBeavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beavis%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBeavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/6/1027" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459902" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459902/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/6/1027.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459902" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459902/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/6/1027.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459902" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459902/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/6/1027.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459902" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459902/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/6/1027.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459902" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459902/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Parametric and nonparametric methods have been developed for purposes of predicting phenotypes. These methods are based on retrospective analyses of empirical data consisting of genotypic and phenotypic scores. Recent reports have indicated that parametric methods are unable to predict phenotypes of traits with known epistatic genetic architectures. Herein, we review parametric methods including least squares regression, ridge regression, Bayesian ridge regression, least absolute shrinkage and selection operator (LASSO), Bayesian LASSO, best linear unbiased prediction (BLUP), Bayes A, Bayes B, Bayes C, and Bayes C<em>π</em>. We also review nonparametric methods including Nadaraya-Watson estimator, reproducing kernel Hilbert space, support vector machine regression, and neural networks. We assess the relative merits of these 14 methods in terms of accuracy and mean squared error (MSE) using simulated genetic architectures consisting of completely additive or two-way epistatic interactions in an <em>F</em><sub>2</sub> population derived from crosses of inbred lines. Each simulated genetic architecture explained either 30% or 70% of the phenotypic variability. The greatest impact on estimates of accuracy and MSE was due to genetic architecture. Parametric methods were unable to predict phenotypic values when the underlying genetic architecture was based entirely on epistasis. Parametric methods were slightly better than nonparametric methods for additive genetic architectures. Distinctions among parametric methods for additive genetic architectures were incremental. Heritability, <em>i.e.</em>, proportion of phenotypic variability, had the second greatest impact on estimates of accuracy and MSE.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/parametric" class="hw-term hw-article-keyword hw-article-keyword-parametric" rel="nofollow">parametric</a></li><li class="kwd"><a href="/keyword/nonparametric" class="hw-term hw-article-keyword hw-article-keyword-nonparametric" rel="nofollow">nonparametric</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/prediction" class="hw-term hw-article-keyword hw-article-keyword-prediction" rel="nofollow">prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-4">Complex quantitative traits are measured on a continuous scale and are controlled by a network of many genes, by the environment, and by genetic by environment interactions. Most traits of economical interest in agriculture (<em>e.g.</em>, grain yield) are measured on continuous scales, <em>i.e.</em>, they are quantitative. Understanding the complexity of these traits and accounting for the effects that are contributed by these genes and their interactions is not trivial.</p><p id="p-5">The gene-by-gene interaction or epistasis is an important research topic in quantitative genetics. Epistasis can be modeled in different ways (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cordell 2002</a>). Physiological epistasis is the difference in the phenotype when the genotype at a locus is influenced by the genotype at another locus or loci. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Fisher (1918)</a> defined epistasis as the deviation of the genotypic value from the contribution of the sum of additive effects at all functional loci in the genome. Fisher’s definition of epistasis is also known as statistical epistasis and has been used to quantify deviations from independence (<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Wilson 2004</a>). Epistasis has an important role in accounting for the genetic variation for quantitative traits, and excluding it from the prediction equations for simplicity can result in poor predictions of genetic gain (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cooper <em>et al.</em> 2002</a>).</p><p id="p-6">Most simulation studies of genomic selection (GS) methods (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> 2001</a>) have considered genetic architectures in which the number and relative magnitudes of quantitative trait loci (QTL) have varied. To our knowledge, no studies of GS methods have considered epistatic genetic architectures, although <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> predicted nonparametric methods would be better-suited for epistatic genetic architectures. Although theoretic models predict a significant role for epistasis in speciation (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dobzhansky 1937</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Mayr 1942</a>), adaptation (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Lewontin 1974</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Wade 2000</a>), and canalization (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Waddington 1949</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Rice 1998</a>), there is little empirical evidence from biometric studies of significant epistatic contributions to genetic variability. Biometric approaches, however, average across epistatic genotypic values at individual loci and contribute primarily to additive genetic variance (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cockerham 1954</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheverud and Routman 1995</a>). With development of low-cost high-throughput marker technologies, it has become possible to estimate epistatic interactions based on genotypic values for all possible pairwise genotypes in genome-wide association studies, although searches for higher-order interactions are still limited by experimental and computational resources (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Moore and Williams 2009</a>). These studies are beginning to reveal that epistasis is not the exception, but rather the most prevalent form of genetic architecture for quantitative traits (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Flint and Mackay 2009</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Huang <em>et al.</em> 2012</a>). Nonetheless, it was hypothesized that GS should provide accurate predictions because epistatic gene action will be translated primarily into additive genetic variance (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crow 2010</a>). Thus, for purposes of this study, we decided to evaluate GS methods for an extreme case of epistasis with 10 pairs of loci, each consisting of two alleles at equal frequencies and modeled using the principle of orthogonality (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Goodnight 2000</a>).</p><p id="p-7">The development of DNA markers in the 1980s was an important step in the process of identifying DNA segments that are statistically associated with quantitative traits, <em>i.e.</em>, QTL mapping and for marker-assisted selection (MAS). In MAS, markers and phenotypic information are used to guide indirect selection of a trait of interest (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Fernando</a> and <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Grossman 1989</a>). This approach is considered an improved and more efficient method for selection in plant breeding relative to phenotype pedigree–based approaches (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Mohan <em>et al.</em> 1997</a>). Extensive resources have been devoted to develop QTL mapping methodology as a component of MAS (<a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">Young 1996</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Melchinger <em>et al.</em> 1998</a>). Marker-assisted backcrossing (MABC) is one of the simplest examples of MAS. In MABC, genomic regions defined by markers closely linked to QTL are identified. These genomic regions are then introgressed into the elite lines through backcrossing (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo 2010</a>). In MABC, a plant with a desired gene, called a donor parent, is crossed with an elite or breeding line, called a recurrent parent. The goal is to introgress the desired gene into the genome of the recurrent parent (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Visscher <em>et al.</em> 1996</a>). Developing varieties can also involve accumulating multiple desired genes into a recurrent parent. The marker-assisted process for alleles at multiple loci is called gene pyramiding. MAS is widely used in gene pyramiding because the use of molecular markers gives the advantage of selecting the desired plants without extensive phenotyping. With traditional phenotyping, it is often impossible to distinguish among plants with all desirable alleles and the plants with some of the desirable alleles (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Huang <em>et al.</em> 1997</a>).</p><p id="p-8">MAS has been shown to be efficient and effective for traits that are associated with one or a few major genes with large effect but does not perform as well when it is used for selection of polygenic traits (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo 2008</a>). QTL detection also results in some false-negative and false-positive rates, and further QTL mapping does not guarantee that estimates of genetic effects are correct (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beavis 1994</a>). Also, for MAS to be useful, the interaction between the QTL and the genetic background has to be minimal, so the QTL has the same effect in different genetic backgrounds (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bernardo 2010</a> p. 223). The genetic background of an organism refers to all of its alleles at all loci that can interact with the locus where the QTL is located (<a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Yoshiki and Moriwaki 2006</a>).</p><p id="p-9">The parametric models and statistical methods introduced for QTL mapping and MAS do not address genetic improvement for quantitative traits that are influenced by a large number of genes with small effects. Some of the statistical challenges arising in MAS include the specification of threshold for multiple testing, the “large p, small n” problem [which refers to the situation when the number of predictors, p (marker data points) greatly exceeds the number of individuals, n, that have been evaluated in the study], difficulty of interpretation of effects due to collinearity among the explanatory/predictor variables, model assumptions that cannot be satisfied, and nonadditivity among genetic effects.</p><p id="p-10">With advanced molecular techniques that provide dense marker maps, it is possible to overcome some shortcomings of MAS. <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> (2001)</a> proposed predicting the genotypic value for individuals using all marker information simultaneously. Their proposed method and the subsequent derivative methods have been referred to as GS. They modeled the associations between the markers and a phenotype focusing on calculating a breeding value for an individual (which can be calculated as the sum of the average effect of the alleles for the individual’s genotype) instead of identifying significant marker–trait associations. In their approach they estimated the effect of each QTL and then used the sum of all estimates to calculate a genotypic value for the individual.</p><p id="p-11">In GS, individuals with both phenotypic and marker information (called the training set) are used to model the association between the phenotype and the genotype. The model is used to predict the phenotypic value of individuals for which only the marker information is available (called the validation set or testing set). In GS, all available markers are included in the model, not just those above a significant threshold, thus eliminating the problem of multiple testing.</p><p id="p-12">Effort is underway to find ways to model epistasis, the gene-by-gene interaction. In the presence of epistasis, the effect of one locus changes the effect of another locus on the phenotype. Usually several loci are involved, which means that multiway interactions may need to be modeled. Because the volume of marker data points available is huge, the number of epistatic interactions can be overwhelming and computationally intractable to estimate with parametric methods (<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Moore and Williams 2009</a>).</p><p id="p-13">More recently, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> stated that parametric approaches to GS have several drawbacks. The parametric model assumptions do not always hold (<em>e.g.</em>, normality, linearity, independent explanatory variables), which suggests the use of nonparametric methods. Also, the convenient partitioning of genetic variance into additive, dominance, additive × additive, additive × dominance, etc., only holds under conditions of linkage equilibrium, random mating of male and female parents, no inbreeding, no assortative mating, no (natural or artificial) selection, and no genotyping errors. In breeding programs, these conditions are all violated. <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> proposed nonparametric and semi-parametric methods to model the relationship between the phenotype and the markers that are available within the GS framework. <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> proposed nonparametric methods capable of accounting for complex epistatic models without explicitly modeling them.</p><p id="p-14">Herein, we review some existing statistical methods used in GS. First, we discuss the parametric methods in more detail. Then, we focus on the nonparametric and semi-parametric methods. Among parametric methods, we review linear least squares regression, penalized ridge regression, Bayes ridge regression, least absolute shrinkage and selection operator (LASSO), and Bayes LASSO methods, best linear unbiased prediction (BLUP), and some Bayesian alternatives used in GS (Bayes A, Bayes B, Bayes C, and Bayes C<em>π</em>). We also explain the nonparametric kernel regression using the Nadaraya-Watson estimator (NWE) and the semi-parametric reproducing kernel Hilbert space (RKHS) regression. Finally, we describe support vector machine (SVM) regression and neural networks (NN) applications to GS. <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">de los Campos <em>et al.</em> (2013)</a> give an overview of some of the parametric methods used in GS, and <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gianola <em>et al.</em> (2010)</a> provide information about some of the nonparametric models used in GS. <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Heslot <em>et al.</em> (2012)</a> compared some parametric and nonparametric GS methods. However, they did not consider epistatic genetic architectures in their simulated data. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Daetwyler <em>et al.</em> (2010)</a> discussed the impact of the genetic architecture in GS, but they defined genetic architecture by the effective population size and the number of QTL.</p><p id="p-15">Here, we use simulated data to compare the performance of the parametric models with the nonparametric procedures for predicting the genetic value for individuals in a <em>F</em><sub>2</sub> and a backcross (BC) populations. We simulate the <em>F</em><sub>2</sub> and the BC populations with low and high heritabilities and compare the two extreme genetic architectures. One architecture had only additive genetic effects from alleles at 30 loci, and the other had only two-way epistatic genetic effects among 30 loci. The performance of the methods is illustrated by comparing the accuracy of prediction, which we defined by the correlation between the true phenotypic value and the predicted phenotypic value and the mean squared error (MSE). We demonstrate the advantage of some nonparametric methods for the epistatic genetic architecture. Because the results for the <em>F</em><sub>2</sub> and BC populations were similar, we only illustrate the <em>F</em><sub>2</sub> population in this article. In the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010298/-/DC1/010298SI.pdf">Supporting Information</a>, we provide accuracy and MSE values for a simulated BC population with low and high heritabilities and with two extreme genetic architectures.</p><div class="section" id="sec-1"><div id="sec-2" class="subsection"><h3>Parametric methods in genome-wide selection</h3><div id="sec-3" class="subsection"><h4>Linear least-squares regression model:</h4><p id="p-16">In GS, the main goal is to predict the individual’s breeding value by modeling the relationship between the individual’s genotype and phenotype. One of the simplest models is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>i</em> = 1…<em>n</em> individual, <em>j</em> = 1…<em>p</em> marker position/segment, <em>y<sub>i</sub></em> is the phenotypic value for individual <em>i</em>, <em>μ</em> is the overall mean, <em>X<sub>ij</sub></em> is an element of the incidence matrix corresponding to marker <em>j</em>, individual <em>i</em>, <em>m<sub>j</sub></em> is a random effect associated with marker <em>j</em>, and <em>e<sub>i</sub></em> is a random residual. Typically, the residual term, <em>e</em>, is chosen to have a normal distribution with mean of 0 and variance of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-2.gif"/></noscript></span></span>. The model for the data vector <strong>y</strong> can be written as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>To estimate (<em>μ</em>, <em>m</em>), we can use least squares to minimize the sum of squared vertical distance between the observed response and the estimated response, which can be represented as |<strong>y</strong> − <strong>Xm</strong>|<sup>2</sup> (where | denotes the norm of a vector). The estimate of <strong>m</strong> obtained by solving the linear equations <em>X</em>′<em>X</em><strong>m</strong> = <em>X</em>′<strong>y</strong>. Then, it is estimated as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-4.gif"/></noscript></span></span>. For more details about linear models, the reader can refer to <em>Linear Models in Statistics</em> (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Schaalje and Rencher 2000</a>) or <em>Linear Models with R</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Faraway 2006</a>). The elements of the design matrix <strong>X</strong> depend on the number of different alleles present. For example, individuals having marker genotypes <em>AA</em>, <em>Aa</em>, <em>aa</em> have elements coded as −1, 0, and 1 in <em>X<sub>ij</sub></em>, respectively.</p><p id="p-17">One obvious problem with linear regression is that usually the number of markers (explanatory variables) available is much greater than the number of individuals with phenotypic information (response variables), which means that <em>p</em> is much greater than <em>n</em>, and it is impossible to perform the estimation. Using a subset of the markers can be an alternative (using a variable selection method like the forward, backward, or stepwise selection procedure) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">George 2000</a>), but it can still perform poorly if the relative ratio of the number of markers and the number of individuals is large or has multicollinearity, <em>e.g.</em>, linkage disequilibrium (LD) exists among the markers.</p><p id="p-18"><a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> (2001)</a> used a modification of least squares regression for GS. First, they performed least squares regression analysis on each segment separately using the model <strong>y</strong> = <em>μ</em> + <strong>X<sub>j</sub></strong><em>m<sub>j</sub></em> + <strong>e</strong>, where <strong>y</strong> is the vector of the phenotypic information, <em>μ</em> is the overall mean vector, <em>X<sub>j</sub></em> is the <em>j<sup>th</sup></em> column of the design matrix corresponding to the <em>j<sup>th</sup></em> segment, <em>m<sub>j</sub></em> is the genetic effect associated with the <em>j<sup>th</sup></em> segment, and <strong>e</strong> is the vector of the error terms. By plotting the log likelihood of this model, segments with significant effects were found. The segments with significant effect (QTL) were used for simultaneous estimation by the model: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-5.gif"/></noscript></span></span>, where <em>q</em> is the number of QTL. With this approach, they eliminated the problem of having more predictor (explanatory/independent) variables than regressands (response/dependent variables), but it does not fully take advantage of the whole marker information because only markers with a significant effect are included in the final model. To overcome some of the drawbacks of the linear regression approach, other methods for GS have been introduced.</p></div><div id="sec-4" class="subsection"><h4>Ridge regression:</h4><p id="p-19">In marker data, it is very likely that multicollinearity exists. As discussed in the previous section, multicollinearity can negatively affect the performance of variable selection methods. Further, least squares equations are inefficient when the determinant of the matrix <strong>X</strong>′<strong>X</strong> is close to zero due to column dependencies. Using a penalized regression model (ridge regression of <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Hoerl and Kennard 1970a</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">b</a>) can be a solution to this problem. The goal is to derive an estimator of <strong>m</strong> with smaller variance than the least squares estimator. There is a “price to pay” in that the ridge regression estimator of <strong>m</strong> is biased; the increase in bias is more than compensated by the decrease in variance, which results in an estimator <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-6.gif"/></noscript></span></span> with smallest MSE. Another advantage of ridge regression is that it can be used when a large amount of marker information is available, so it can overcome the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-7.gif"/></noscript></span></span> problem.</p><p id="p-20">Ridge regression adds an extra term to the likelihood function to shrink the regression coefficients by an amount depending on the variance of the covariates. It removes the problem of the columns of the design matrix being dependent on each other and, hence, the <strong>X</strong>′<strong>X</strong> matrix will be nonsingular. Instead of minimizing the sum of squared residuals, ridge regression minimizes the penalized sum of squares |<strong>y</strong> − <strong>Xm</strong>|<sup>2</sup> + <em>λ</em><sup>2</sup><strong>m</strong>′<strong>m</strong>, where <em>λ</em> is the penalty parameter, and the estimate of the regression coefficient is given by: <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-8.gif"/></noscript></span></span>, where <strong>I</strong> is a <em>p</em> × <em>p</em> identity matrix. The penalty parameter <em>λ</em> can be calculated by several different methods, for example, by plotting <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-9.gif"/></noscript></span></span> as a function of <em>λ</em> and choosing the smallest <em>λ</em> that results in a stable estimate of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-10.gif"/></noscript></span></span>. Another way to choose <em>λ</em> is by an automated procedure proposed by <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Hoerl <em>et al.</em> (1975)</a>. They claimed that a reasonable choice of <em>λ</em> is given by: <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-11.gif"/></noscript></span></span>, where <em>r</em> is the number of parameters in the model not counting the intercept, <em>s</em><sup>2</sup> is the residual mean square obtained by linear least squares estimation, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-12.gif"/></noscript></span></span> is the vector of least squares estimates of regression coefficients.</p><p id="p-21"><a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> (2001)</a> implemented ridge regression in GS by assuming that the marker effects (<em>m<sub>j</sub></em>′s <em>j</em> = 1…<em>p</em>) were random, and they were drawn from a normal distribution with <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-13.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-14.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-15.gif"/></noscript></span></span> represents additive genetic variance expressed among individuals and <em>n<sub>k</sub></em> is the number of marker loci (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Habier <em>et al.</em> 2007</a>).</p><p id="p-22">It can be shown that ridge regression is a special case of the BLUP (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Ruppert <em>et al.</em> 2003</a>), which we demonstrate after introducing BLUP. Thus, the mixed linear model can be implemented. Within the mixed model context, the restricted maximum likelihood (REML) estimation is a good choice for finding a reasonable value for the penalty parameter and estimating the variance components (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Henderson 1988</a>). <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Piepho (2009)</a> discusses some models that feature the ridge regression in terms of mixed models and uses REML for variance and penalty parameter estimation in GS.</p><p id="p-23">Ridge regression can also be viewed from a Bayesian perspective. In this case, we assume that the parameter vector <strong>m</strong> is random. We can account for the belief that the estimator of <strong>m</strong> has a small variance by a choice of a prior distribution. In particular, we can suppose that <strong>m</strong> ∼ <em>N</em>(0, ∑<em><sub>β</sub></em>), where ∑<em><sub>β</sub></em> is a known covariance matrix (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Boer <em>et al.</em> 2005</a>). Given that the likelihood of <em>y<sub>i</sub></em> (<em>i</em> = 1, 2, …<em>n</em>, where <em>n</em> is the number of individuals) has a normal distribution with mean <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-16.gif"/></noscript></span></span> and variance <em>σ</em><sup>2</sup>, the Bayesian estimator of <strong>m</strong> is the mean of the posterior distribution, and it is given by <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-17.gif"/></noscript></span></span> (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Judge <em>et al.</em> 1985</a> p. 286). Comparing <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-18.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-19.gif"/></noscript></span></span>, we can see that they are identical if <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-20.gif"/></noscript></span></span>.</p><p id="p-24"><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Pérez <em>et al.</em> (2010)</a> discussed the application of Bayesian ridge regression in GS. They assumed that the marker effects are independent and identically distributed (iid) and have a normal prior distribution with mean 0 and variance <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-21.gif"/></noscript></span></span>, where: <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-22.gif"/></noscript></span></span>. Then the mean of the posterior distribution <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-23.gif"/></noscript></span></span> is equivalent to <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-24.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-25.gif"/></noscript></span></span>.</p></div><div id="sec-5" class="subsection"><h4>Best linear unbiased prediction:</h4><p id="p-25">The BLUP theory and the mixed model formulation were first discussed by <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Henderson (1949)</a>, and they were influential for selection purposes in animal breeding (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Henderson 1959</a>). BLUP is a statistical procedure, and it is useful in situations when the data available are unbalanced (for example, in different locations the number of individuals is not the same), and it can accommodate family information (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bernardo 2010</a>). Since Henderson’s first work in BLUP, the theory has been widely expanded (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Henderson 1959</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">1963</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">1975a</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">1975b</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Harville 1976</a>). Since the 1990s, BLUP has been used not only in animal breeding applications (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Henderson 1984</a>) but also in plant breeding (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 1994</a>).</p><p id="p-26">BLUP was proposed as a tool in GS by <a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> (2001)</a>. The random effects model can be written in the form:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-26.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <strong>y</strong> is the (<em>n</em> × 1) phenotypic data vector, <em>μ</em> is the (<em>n</em> × 1) overall mean vector, <strong>Z<sub>j</sub></strong> is the <em>j<sup>th</sup></em> column of the design matrix, <em>m<sub>j</sub></em> is the genetic effect associated with the <em>j<sup>th</sup></em> marker, and <em>p</em> is the number of markers. The intercept, <em>μ</em>, is fixed, and <em>m<sub>j</sub></em> is the random effects with <em>E</em>(<em>m<sub>j</sub></em>) = 0, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-27.gif"/></noscript></span></span>, <em>Var</em>(<strong>e</strong>) = <em>σ</em><sup>2</sup><strong>I</strong>, and <em>Cov</em>(<strong>m</strong>, <strong>e</strong>) = 0. In the statistical literature, the vector of random effects is usually denoted by <strong>u</strong> instead of <strong>m</strong>. If other covariates are available, then we replace the intercept <em>μ</em> by <strong>X</strong><em>β</em> to include all the fixed effects. Then, we can write:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-28.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where <em>β</em> is a <em>p</em><sub>1</sub> × 1 vector of unknown fixed effects where usually the first element is the population mean, and <strong>X</strong> is the incidence matrix that relates <strong>y</strong> to <em>β</em>. The above equation is generally called a mixed model (or mixed effects model). The vector <em>β</em> is estimated by the best linear unbiased estimator (BLUE). In the biological literature, the term BLUP is occasionally used loosely and refers to both BLUE and BLUP. BLUP is the predictor of the random effects. It is a linear function of the data vector <strong>y</strong>. Within the linear functions of the data it is unbiased, which means that the expected value of the prediction is the same as the population parameter, and it can be formulated as <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-29.gif"/></noscript></span></span>. In addition within the unbiased linear predictors, it is the best in the sense of minimizing the MSE. BLUE is similar to BLUP in that it is a linear function of the data <strong>y</strong>, it is unbiased among the linear estimators, and it is best in the sense that it minimizes the MSE.</p><p id="p-27"><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Henderson (1953)</a> proposed that the BLUE and BLUP of (<em>β</em>, <strong>m</strong>) be obtained by maximizing the joint likelihood of (<strong>y</strong>, <strong>m</strong>) given by:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-30.gif"/></noscript></span></span>By maximizing the likelihood <em>L</em>(<strong>y</strong>, <strong>m</strong>) with respect to <em>β</em>, <strong>m</strong> and equating it to zero, we obtain a set of linear equations [known as Henderson’s mixed model equations (MME)]:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-31.gif"/></noscript></span></span>where <em>R</em> = <em>Var</em>(<strong>e</strong>) and <em>G</em> = <em>Var</em>(<strong>m</strong>). The solution to the MME is the BLUE of <em>β</em> and the BLUP of <strong>m</strong>. Henderson’s derivation assumes that <strong>m</strong> and <strong>e</strong> are normally distributed and maximizes the joint likelihood of (<strong>y</strong>, <strong>m</strong>) over the unknowns <em>β</em> and <strong>m</strong>. Maximizing the likelihood implies an optimization criterion of (<strong>y</strong> − <strong>X</strong><em>β</em> − <strong>Zm</strong>)′<strong>R</strong><sup>−1</sup>(<strong>y</strong> − <strong>X</strong><em>β</em> − <strong>Zm</strong>) + <strong>m</strong>′<strong>G</strong><sup>−1</sup><strong>m</strong>, and it can be viewed as the “ridge regression formulation” of the BLUP (<a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Ruppert <em>et al.</em> 2003</a>).</p><p id="p-28">We have assumed that <em>R</em> and <em>G</em> are known covariance matrices. In general, they are unknown and need to be estimated together with <em>β</em>, <strong>m</strong>. The REML approach to estimate the variance components maximizes the “restricted” likelihood associated with a specific set of linear combinations of the data. The restricted likelihood depends only on the variance components. REML produces unbiased estimates of the variance parameters <em>R</em> and <em>G</em>. More information about variance estimation using REML can be found in works by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Corbeil and Searle (1976)</a>, <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Harville (1976)</a>, and <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">McGilchrist (1993)</a>. There are other ways to derive the BLUP solution for <strong>m</strong>. <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Robinson (1991)</a> showed that that the BLUE solution to <em>β</em> can be written as: <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-32.gif"/></noscript></span></span>, and the BLUP solution to <strong>m</strong> can be written as: <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-33.gif"/></noscript></span></span>.</p></div><div id="sec-6" class="subsection"><h4>LASSO method:</h4><p id="p-29">To overcome the limitations of linear least squares, we can use the LASSO for GS. LASSO was first introduced by <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Tibshirani (1996)</a>, and <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Usai <em>et al.</em> (2009)</a> first implemented it in GS using cross-validation. We can write the model for individual <em>i</em> as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-34.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where <em>i</em> = 1…<em>n</em> individual, <em>j</em> = 1…<em>p</em> marker position, <em>y<sub>i</sub></em> is the phenotypic value for individual <em>i</em>, <em>X<sub>ij</sub></em> is an element of the incidence matrix corresponding to individual <em>i</em> and marker <em>j</em>, <em>m<sub>j</sub></em> is the marker effect for marker <em>j</em>, and <em>e<sub>i</sub></em> is the random residual. The LASSO estimate of the marker effect is obtained by minimizing the residual sum of squares <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-35.gif"/></noscript></span></span> subject to the constraint of the sum of the absolute value of the marker effects being less than a constant <em>s</em>, <em>s</em> ≥ 0, and we can write it as: <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-36.gif"/></noscript></span></span>. This constraint shrinks some of the marker effects and sets some of them to zero. One of the major differences between LASSO and ridge regression is that in LASSO as we increase the penalty, more marker effects will shrink to zero and in ridge regression all parameters will be reduced but still remain nonzero.</p><p id="p-30">The LASSO estimator of the regression coefficients <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-37.gif"/></noscript></span></span> can be found by an algorithm that was first described by <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Tibshirani (1996)</a> and used computational ideas from <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Lawson and Hansen (1974)</a>. First, we assume that the elements of the incidence matrix are standardized such that <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-39.gif"/></noscript></span></span>. Then, the algorithm describes a quadratic programming problem with 2<em><sup>p</sup></em> linear constrains, corresponding to the different signs for the regression coefficients <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-40.gif"/></noscript></span></span>. For example, if <em>P</em> = 3 then we have:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-41.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-42.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-43.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-44.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-45.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-46.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-47.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-48.gif"/></noscript></span></span>Let <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-49.gif"/></noscript></span></span> and for <em>k</em> = 1…2<em><sup>p</sup></em> let <em>γ<sub>k</sub></em> be a vector of indicator variables 1, 0, −1 depending on the signs of the regression coefficients corresponding to the <em>k<sup>th</sup></em> inequality. Also, let <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-50.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-51.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-52.gif"/></noscript></span></span>.</p><p id="p-31">The steps of the algorithm finding the LASSO estimator can be written as:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-32"><span class="list-label">1. </span>Let <em>E</em> = {<em>i</em><sub>0</sub>}, where <em>i</em><sub>0</sub> corresponds to the least squares estimate of <strong>m</strong> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-53.gif"/></noscript></span></span> sign<span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-54.gif"/></noscript></span></span>.</p></li><li id="list-item-2"><p id="p-33"><span class="list-label">2. </span>Find <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-55.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-56.gif"/></noscript></span></span> is minimized subject to <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-57.gif"/></noscript></span></span>.</p></li><li id="list-item-3"><p id="p-34"><span class="list-label">3. </span>If <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-58.gif"/></noscript></span></span>, then done.</p></li></ul><p id="p-35">If <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-59.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-60.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-61.gif"/></noscript></span></span>. Repeat steps 2 and 3.</p><p id="p-36">The algorithm described above is computationally intensive. <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Efron <em>et al.</em> (2004)</a> proposed a new model selection algorithm called least angle regression (LARS) that can be used in combination with LASSO estimation. LARS is similar to the traditional forward selection method. It starts with all the coefficients (marker effects) at zero. First, the marker that has the highest correlation with the phenotypic values is added into the model. The next marker added has to have a correlation with the residual that is at least as large. The third marker entered into the model is equiangular with the first two markers already in the model. At each iteration, a new marker is added, and the algorithm is accomplished in <em>p</em> iterations where <em>p</em> is the number of the available markers. However, for LASSO, the LARS procedure is modified. Because the LASSO has a constraint, the LARS procedure has to apply a restriction, so this model selection method is more closely related to the stepwise selection method. For a detailed description of LARS and the LARS–LASSO relationship, the reader can refer to <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Efron <em>et al.</em> (2004)</a>.</p><p id="p-37">One other important question is how to find the upper bound of the sum of the absolute value of the marker effects, <em>s</em>. Finding the best value for <em>s</em> can be viewed as the selection of the size of the best subset of markers. <a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">Usai <em>et al.</em> (2009)</a> used the cross-validation approach of <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Kohavi (1995)</a> with random subsampling replication. In every replication, the data are randomly divided into a training set and a validation set. The training set is used to estimate the marker effects using the LARS algorithm for the LASSO method. The estimated marker effects were used to calculate the genomic breeding values (GEBV) for the individuals in the validation set, and then the correlation coefficients between the GEBV and the true phenotypic value were reported. The LARS iterations were carried forward until the maximum correlation was reached.</p></div><div id="sec-7" class="subsection"><h4>Bayesian alphabet:</h4><p id="p-38"><a id="xref-ref-78-6" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> (2001)</a> proposed two hierarchical Bayesian models for GS denoted by Bayes A and Bayes B. In both methods the data and the variances of the marker positions need to be modeled. For individual <em>i</em> we can write:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-62.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>where <em>i</em> = 1…<em>n</em> individual, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-63.gif"/></noscript></span></span> marker position/segment, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-64.gif"/></noscript></span></span> is the phenotypic value for individual <em>i</em>, <em>μ</em> is the <em>n</em> × 1 dimensional overall mean vector, <em>X<sub>ij</sub></em> is an element of an incidence matrix for marker <em>j</em> and individual <em>i</em>, <em>m</em><strong><sub>j</sub></strong> is a random effect for marker <em>j</em>, and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-65.gif"/></noscript></span></span> is a random residual. In general the model can be written as: <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-66.gif"/></noscript></span></span>.</p><p id="p-39">Inferences about model parameters are based on the posterior distribution. By Bayes’ Theorem, the posterior is obtained by combining the prior distribution and the likelihood function. For detailed information about Bayesian methods, the reader can refer to <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Kruschke (2010)</a> or <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gelman <em>et al.</em> (2003)</a>.</p><p id="p-40">The difference between Bayes A and Bayes B lies in the way in which we model the variances of parameters. In both methods each marker position has its own variance. The Bayes A approach applies the same prior distribution for all of the variances of the marker positions. The scaled inverted (or inverse) chi-squared probability distribution χ<sup>−2</sup>(<em>ν</em>, <em>S</em><sup>2</sup>) can be used with degrees of freedom <em>ν</em> and scale parameter <em>S</em><sup>2</sup> as the prior distribution. This is a convenient choice because it is a conjugate prior so the posterior distribution is in the same family of distributions as the prior distribution. The posterior distribution is also a scaled inverse chi-square distribution <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-67.gif"/></noscript></span></span> where <em>n<sub>j</sub></em> is the number of haplotype effects at marker position <em>j</em>.</p><p id="p-41">The Bayes B approach seems more realistic for GS than Bayes A. The only difference between the two methods is the prior for the variance components. Bayes B assumes that not all markers contribute to the genetic variation. It has a prior density on the variance that is a mixture. It has a high probability mass at <em>σ<sub>mj</sub></em> = 0 and an inverted chi-square distribution when <em>σ<sub>mj</sub></em> &gt; 0. It can be summarized as <em>σ<sub>mj</sub></em> = 0 with prob= <em>π</em> and <em>σ<sub>mj</sub></em> ∼ <em>χ</em><sup>−2</sup>(<em>ν</em>, <em>S</em>) with prob=(1 − <em>π</em>).</p><p id="p-42">For the Bayes B method if <em>m</em>′<em>m</em> &gt; 0, then one cannot sample <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-68.gif"/></noscript></span></span>. So, we can sample <em>m<sub>j</sub></em> and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-69.gif"/></noscript></span></span> simultaneously by <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-70.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-71.gif"/></noscript></span></span> is the data that are corrected for the mean and for all genetic effects except <em>m<sub>j</sub></em>.</p><p id="p-43">To sample from the distribution<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-72.gif"/></noscript></span></span>, we can use the Metropolis-Hastings algorithm in the following way:</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-44"><span class="list-label">1. </span>Sample <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-73.gif"/></noscript></span></span> from the prior distribution of <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-74.gif"/></noscript></span></span>.</p></li><li id="list-item-5"><p id="p-45"><span class="list-label">2. </span><span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-75.gif"/></noscript></span></span> with probability of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-76.gif"/></noscript></span></span>.</p></li></ul><p id="p-46">Using simulated data, it was shown that the Bayesian methods perform better in terms of prediction accuracy than the linear least squares regression, the ridge regression, and the BLUP method (<a id="xref-ref-78-7" class="xref-bibr" href="#ref-78">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Habier <em>et al.</em> 2009</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2010</a>). However, as <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gianola <em>et al.</em> (2009)</a> pointed out, the choice of the degrees of freedom and the scale parameters of the scaled inverse chi-square distribution can influence the outcome. Improved Bayesian methods were developed by <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Habier <em>et al.</em> (2011)</a> to deal with the weakness of Bayes A and Bayes B. Bayes C uses a common variance for all SNPs, and for Bayes D the scale parameter of the scaled inverse chi-square distribution is estimated instead of specified by the user. Bayes C<em>π</em> and Bayes D<em>π</em> (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Habier <em>et al.</em> 2011</a>) are the modification of Bayes C and Bayes D where the probability of having a zero effect SNP <em>π</em> is estimated.</p></div><div id="sec-8" class="subsection"><h4>Bayesian LASSO:</h4><p id="p-47"><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Park and Casella (2008)</a> introduced the Bayesian LASSO method for estimating the regression coefficients. They used an idea from <a id="xref-ref-100-3" class="xref-bibr" href="#ref-100">Tibshirani (1996)</a> to connect the LASSO method with the Bayesian analysis. <a id="xref-ref-100-4" class="xref-bibr" href="#ref-100">Tibshirani (1996)</a> noticed that the LASSO estimates of the regression coefficients can be viewed as posterior mode estimates assuming that the regression coefficients have double exponential prior distributions. The Bayesian LASSO is also used in GS (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2010a</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Long <em>et al.</em> 2011</a>) using the hierarchical model with the likelihood function:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-77.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>where <strong>y</strong> is the <em>n</em> × 1 data vector, <em>μ</em> is the overall mean vector, <strong>m</strong> is a vector of the marker effects, and <strong>X</strong> is the design matrix that connects <strong>m</strong> to <strong>y</strong>. <em>N</em>(<em>μ</em> + <strong>Xm</strong>, <em>σ</em><sup>2</sup><strong>I</strong>) denotes the normal density with mean <em>μ</em> + <strong>Xm</strong> and variance <em>σ</em><sup>2</sup><strong>I</strong> where <strong>I</strong> is an <em>n</em> × <em>n</em> identity matrix. The prior distribution on the marker effects <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-78.gif"/></noscript></span></span> <em>j</em> = 1…<em>p</em> can be written as <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-79.gif"/></noscript></span></span>, and the prior distribution on <em>τ<sub>j</sub></em> is <em>p</em>(<em>τ<sub>j</sub></em>|<em>λ</em>) ∼ <em>Exp</em>(<em>λ</em>) where <em>Exp</em>(<em>λ</em>) denotes the exponential distribution with rate parameter <em>λ</em>. <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Park and Casella (2008)</a> and <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">de los Campos <em>et al.</em> (2009)</a> presented the full conditional distributions that were used to sample via the Gibbs sampler. <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">de los Campos <em>et al.</em> (2009)</a> expanded the model and assigned a prior distribution to <em>λ</em><sup>2</sup>. The prior has a Gamma distribution with shape parameter <em>α</em><sub>1</sub> and scale parameter <em>α</em><sub>2</sub>, and it can be written as <em>p</em>(<em>λ</em><sup>2</sup>) ∼ Γ (<em>α</em><sub>1</sub>, <em>α</em><sub>2</sub>); <em>λ</em> has two interpretations. In the Bayesian formulation, it is the rate parameter that controls the shape of the prior distribution of the <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-80.gif"/></noscript></span></span>. In the LASSO setting, <em>λ</em> controls the penalty for minimizing the MSE.</p></div></div><div id="sec-9" class="subsection"><h3>Nonparametric methods in genome-wide selection</h3><p id="p-48">In this section, we review some of the nonparametric estimation methods that have been proposed for the case where the form of the relationship between a response variable and a set of predictors is unknown. A popular approach, at least in terms of usage, is based on the kernel method proposed by <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Silverman (1986)</a> in the context of density estimation. In that context, the goal is to estimate the unknown density using a smooth curve (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Schucany 2004</a>). The kernel method is the most commonly used nonparametric estimation procedure.</p><p id="p-49">The kernel density estimator <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-81.gif"/></noscript></span></span> can be written in the form:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-82.gif"/></noscript></span></span>where <em>n</em> is the number of observations, <em>K</em> is the kernel function that satisfies the condition <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-83.gif"/></noscript></span></span>, <em>h</em> is positive real-valued smoothing parameter (also called window width or bandwidth), <em>x</em> is the focal point, and <em>X<sub>i</sub></em> is the <em>p</em> × 1 dimensional vector of dummy covariates for observation <em>i</em>. We can calculate <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-84.gif"/></noscript></span></span> at several focal points <em>x</em>, and the observations that are closer to the focal point will get a higher weight in the calculation, so the kernel function <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-85.gif"/></noscript></span></span> gives bigger weight to observations closer to the focal point. The kernel function <em>K</em> is usually chosen to be a symmetric unimodal density, so the kernel density estimator <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-86.gif"/></noscript></span></span> is also a density. A commonly used kernel function is the Gaussian kernel given by:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-87.gif"/></noscript></span></span>In this expression, observations with <strong>x</strong><em><sub>i</sub></em> coordinates closer to the focal point <strong>x</strong> are weighted more strongly in the computation of the fitted value <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-88.gif"/></noscript></span></span>. The window width provides information about the range of observations that are included (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Sheather 2004</a>). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows how the kernel density estimation changes with different bandwidth values. Using simulated data from a mixture of two normal distributions, the second, third, and fourth panels show how the estimation changes with the change of the bandwidth value.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The influence of the bandwidth in kernel density estimation. From left to right, the first plot shows simulated data from a mixture of two normal distributions. The second, third, and fourth plots show the Gaussian kernel density estimates using bandwidth values h = 0.1, h = 2, and h = 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The influence of the bandwidth in kernel density estimation. From left to right, the first plot shows simulated data from a mixture of two normal distributions. The second, third, and fourth plots show the Gaussian kernel density estimates using bandwidth values &lt;em&gt;h&lt;/em&gt; = 0.1, &lt;em&gt;h&lt;/em&gt; = 2, and &lt;em&gt;h&lt;/em&gt; = 10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F1.medium.gif" width="440" height="178"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/6/1027/F1.medium.gif" width="440" height="178"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464467" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-50" class="first-child">The influence of the bandwidth in kernel density estimation. From left to right, the first plot shows simulated data from a mixture of two normal distributions. The second, third, and fourth plots show the Gaussian kernel density estimates using bandwidth values <em>h</em> = 0.1, <em>h</em> = 2, and <em>h</em> = 10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">When <em>h</em> = 0.1, the data have strong influence on the density estimate, resulting in little bias and large variability among estimates. It is called an undersmoothed estimate. As we increase the bandwidth value, the estimates become smoother. When <em>h</em> = 10, the spread is too big and even the bimodal feature of the data disappears, which implies that the estimate is oversmoothed. Setting the bandwidth too large results in a large bias with little variance.</p><div id="sec-10" class="subsection"><h4>Nadaraya-Watson estimator:</h4><p id="p-52">In the context of GS, <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> considered the regression function:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-89.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span><em>i</em> = 1, 2, …, <em>n</em> where <em>y<sub>i</sub></em> phenotypic measurement on individual <em>i</em>, <strong>x</strong><em><sub>i</sub></em> is a <em>p</em> × 1 vector of dummy SNP covariates observed on individual <em>i</em>, <em>g</em>(.) is some unknown function relating genotypes to phenotypes, <em>g</em>(<strong>x</strong><em><sub>i</sub></em>) = <em>E</em>(<em>y<sub>i</sub></em>|<strong>x</strong><em><sub>i</sub></em>), and <em>e<sub>i</sub></em> is a random residual effect for individual <em>i</em> where <em>e<sub>i</sub></em> ∼ (0, <em>σ</em><sup>2</sup>) and is independent of <strong>x</strong><em><sub>i</sub></em>.</p><p id="p-53">The conditional expectation function can be written in the form:<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-90.gif"/></noscript></span></span>A nonparametric kernel estimator (<a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Silverman 1986</a>) can be used to obtain an estimate of <em>p</em>(<em>x</em>). The estimator has the form:<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-91.gif"/></noscript></span></span>where <strong>x</strong><em><sub>i</sub></em> is the observed p-dimensional SNP genotype of individual <em>i</em>, <em>i</em> = 1, 2, …, <em>n</em>. Similarly,<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-92.gif"/></noscript></span></span>Using these expressions, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Nadaraya (1964)</a> and <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Watson (1964)</a> showed that the conditional expectation function can be written as:<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-93.gif"/></noscript></span></span>The estimator is just a weighted sum of the observations <em>y<sub>i</sub></em>, <em>i</em> = 1…<em>n</em> and is called the NWE.</p><p id="p-54">The selection of the bandwidth, h, value is challenging. <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Hardle (1990)</a> discussed several approaches to select <em>h</em>, including the leave-one-out cross-validation (CV), penalizing functions, and plug-in methods. <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> used the leave-one-out CV approach to select the bandwidth. In this approach, first exclude the <em>i<sup>th</sup></em> observation (<em>y<sub>i</sub></em>, <strong>x</strong>) and fit the model to the other <em>n</em> − 1 observations. Using the marker information, predict <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-94.gif"/></noscript></span></span>. This is repeated for all <em>n</em> observations. The CV criterion is (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark 1975</a>):<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-95.gif"/></noscript></span></span>The CV estimate of <em>h</em> is the value of <em>h</em> that minimizes CV(h).</p></div><div id="sec-11" class="subsection"><h4>Reproducing kernel Hilbert space:</h4><p id="p-55"><a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> proposed a semi-parametric kernel mixed model approach in which they combined the nice features of a nonparametric model (described above) with a mixed model framework. The model can be written as:<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-96.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span>where <em>i</em> = 1, 2, …, <em>n</em>, <em>β</em> is a vector of fixed unknown effects (<em>e.g.</em>, physical location of an individual), <em>u</em> is a <em>q</em> × 1 vector of additive genetic effects, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-97.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-98.gif"/></noscript></span></span> are known incidence vectors, <em>g</em>(<strong>x</strong><em><sub>i</sub></em>) is an unknown function of the SNP data and the vector of residuals, and <em>e</em> is assumed to have a <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-99.gif"/></noscript></span></span> distribution. The vector containing additive genetic effects, <em>u</em> is distributed as <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-100.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-101.gif"/></noscript></span></span> is the additive genetic variance and <strong>A</strong> is the additive relationship matrix.</p><p id="p-56">The authors suggested two different methods for estimation in this model. The first strategy, denoted “Mixed Model Analysis,” consists of a two-step approach with a “corrected” data vector <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-102.gif"/></noscript></span></span> in the second step of the analysis. A Bayesian approach can also be used where one can draw samples from the pseudo posterior distribution <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-103.gif"/></noscript></span></span>, and then form semi-parametric draws of the total genetic value.</p><p id="p-57">The other method they suggested is the “Random <em>g</em>(.) function” approach, where it is assumed that <em>β</em>, <em>u</em> are known. In this case:<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-104.gif"/></noscript></span></span>and draws of <em>β</em><sup>(</sup><em><sup>j</sup></em><sup>)</sup>, <em>u</em><sup>(</sup><em><sup>j</sup></em><sup>)</sup> can then be obtained from the distribution <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-105.gif"/></noscript></span></span>.</p><p id="p-58">Finally, <a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">Gianola <em>et al.</em> (2006)</a> discuss estimation in the RKHS mixed model. The set-up is similar to the mixed model approach, but estimation of model parameters is performed using a penalized sum of squares approach. As before, the model can be written as:<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-106.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>where <em>i</em> = 1, 2, …, <em>n</em>. The penalized sum of squares is given by:<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-107.gif"/></noscript></span></span>where the penalty ‖<em>g</em>(<em>x</em>)‖ is a function of the second derivatives of <em>g</em>(<em>x</em>). The goal is to find <em>g</em>(<em>x</em>) that minimizes the penalized SS. <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Wahba (1990)</a> showed that the minimizer can be written as:<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-108.gif"/></noscript></span></span>where <em>K</em>(.,.) is the reproducing kernel.</p></div><div id="sec-12" class="subsection"><h4>Support vector machine regression:</h4><p id="p-59">SVM was proposed by Vapnik and discussed by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cortes and Vapnik (1995)</a>. SVM is a supervised learning technique that was originally developed as a classifier. A training data set is used to develop a maximum margin classifier that produces the largest possible separation between two classes of observations. In the linearly separable case, if observations (<em>x<sub>i</sub></em>) ∈ <em>R<sup>p</sup></em>, then the separator is a hyper-plane in <em>R<sup>p</sup></em><sup>−1</sup>.</p><p id="p-60">Because fitting a regression model essentially consists of finding an optimal projection of the observations on a lower-dimensional hyper-plane, the idea can be used to estimate the unknown regression function subject to restrictions. The reader can refer to <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Hastie <em>et al.</em> (2009)</a>, <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Steinwart and Christmann (2008)</a>, and <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Christianini and Shawe-Taylor (2000)</a> for a review of SVM. SVM regression was adopted by <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Maenhout <em>et al.</em> (2007)</a> and <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Long <em>et al.</em> (2011)</a> for GS in plant breeding. A nice feature of SVM regression in plant breeding applications is that the relationship between the marker genotypes and the phenotypes can be modeled with a linear or nonlinear mapping function that takes samples from a predictor space to an abstract, multidimensional feature space (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Hastie <em>et al.</em> 2009</a>).</p><p id="p-61">Suppose that we have a training sample <em>S</em> = {(<strong>x</strong><em><sub>i</sub></em>, <em>y<sub>i</sub></em>), <strong>x</strong><em><sub>i</sub> ε R<sup>n</sup></em><sub>,</sub> <em>y<sub>i</sub> ε R</em>, <em>i</em> = 1…<em>n</em>}, where <strong>x</strong><em><sub>i</sub></em> is a <em>p</em> dimensional vector containing the genotypic values for the <em>p</em> markers for individual <em>i</em>, and <em>y<sub>i</sub></em> is the phenotypic value for individual <em>i</em>. A model that describes the relationship between the phenotype and the genotype of an individual can be written as:<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-109.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>where <em>b</em> is a constant and <em>w</em> is a vector of unknown weights. The constant <em>b</em> reflects the maximum error we are willing to commit when estimating the weights <em>w</em>. We learn about the function <em>f</em>(<em>x</em>) by minimizing the expression <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-110.gif"/></noscript></span></span>. <em>L</em>(.) denotes the loss function that measures the quality of the estimation. The regularization parameter <em>λ</em> quantifies the trade-off between the sparsity and the complexity of the model. Increasing <em>λ</em> implies a higher penalty on the error. The norm ‖<strong>w</strong>‖ of vector <strong>w</strong> is inversely associated with model complexity; by choosing <em>w</em> to minimize ‖<strong>w</strong>‖, we reduce model complexity.</p><p>There are many loss functions used for SVM regression. Some of the popular loss function choices include the squared loss, absolute loss, and the <em>ε</em>-insensitive loss. Here, we present these loss function formulations.</p><ul class="list-simple " id="list-3"><li id="list-item-6"><p id="p-63"><span class="list-label">1. </span>The squared loss function has the form <em>L</em>(<em>y</em> − <em>f</em>(<em>x</em>)) = (<em>y</em> − <em>f</em>(<em>x</em>))<sup>2</sup>. It scales the loss quadratically by the size of the error. Using this loss function indicates that outliers are also weighted quadratically, which requires the user to deal with the outliers before the regression analysis.</p></li><li id="list-item-7"><p id="p-64"><span class="list-label">2. </span>The absolute loss function has the form <em>L</em>(<em>y</em> − <em>f</em>(<em>x</em>)) = |<em>y</em> − <em>f</em>(<em>x</em>)|. The absolute loss function scales the loss linearly by the size of the error eliminating the difficulty of using data sets with outliers.</p></li><li id="list-item-8"><p id="p-65"><span class="list-label">3. </span>The <em>ε</em>-insensitive loss function has a form:<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-111.gif"/></noscript></span></span></p></li></ul><p>where <em>ε</em> determines the number of support vectors used in the regression function. By definition (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Vapnik 1995</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Vapnik and Vashist 2009</a>), a support vector is a vector <strong>x</strong><em><sub>i</sub></em> that satisfies the equation <em>y<sub>i</sub></em>(<em>w</em><strong>x</strong><em><sub>i</sub></em> + <em>b</em>) = 1. Increasing <em>ε</em> implies that fewer support vectors are used in the fitting. The <em>ε</em>-insensitive loss function ignores the errors in the regression that have size less than <em>ε</em>. When the error is greater than <em>ε</em>, the loss is |<em>y</em> − <em>f</em>(<em>x</em>)| − <em>ε</em>.</p><p id="p-66"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> illustrates the absolute loss, squared loss, and <em>ε</em>-insensitive loss functions as a function of the error <em>y</em> − <em>f</em>(<em>x</em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss functions used for SVM regression. The first panel shows the absolute loss function. The second panel is the square loss function, and the last panel is the &#x3B5;-insensitive loss function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss functions used for SVM regression. The first panel shows the absolute loss function. The second panel is the square loss function, and the last panel is the &lt;em&gt;&#x3B5;&lt;/em&gt;-insensitive loss function.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F2.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/6/1027/F2.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464522" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-67" class="first-child">Loss functions used for SVM regression. The first panel shows the absolute loss function. The second panel is the square loss function, and the last panel is the <em>ε</em>-insensitive loss function.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">In the remainder, we focus on the <em>ε</em>-insensitive loss function, which needs a more robust representation to account for the noise in the data.</p><p id="p-69">We can add extra “cost” (or allow for additional uncertainty) by introducing non-negative “slack variables” <em>ξ</em> constrained as follows (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Long 2011</a>):</p><p id="p-70"><em>ξ</em><sub>1</sub><em><sub>i</sub></em> ≥ <em>y<sub>i</sub></em> − <em>f</em>(<strong>x</strong><em><sub>i</sub></em>) − <em>ε</em>, where <em>i</em> = 1, …, <em>n n</em> is the number of training observations,</p><p id="p-71"><em>ξ</em><sub>2</sub><em><sub>i</sub></em> ≥ <em>f</em>(<strong>x</strong><em><sub>i</sub></em>) − <em>y<sub>i</sub></em> − <em>ε</em>, where <em>i</em> = 1, …, <em>n</em>.</p><p id="p-72">We can now re-write the objective function to be minimized as:<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-112.gif"/></noscript></span></span>The solution to this constrained minimization problem has the form:<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-113.gif"/></noscript></span></span>(<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Nocedal and Wright 1999</a>). The solution depends on the training data through the inner product 〈<strong>x</strong><em><sub>i</sub></em>, <strong>x</strong><em><sub>j</sub></em>〉, which is a linear function of the observations.</p><p id="p-73">To take advantage of higher dimensional feature spaces, we can introduce the data via nonlinear functions. For example, we can replace the inner product of the data by a kernel function:<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-114.gif"/></noscript></span></span>Some commonly used kernel functions include:</p><ul class="list-simple " id="list-4"><li id="list-item-9"><p id="p-74"><span class="list-label">1. </span>The linear kernel <em>k</em>(<strong>x</strong>, <strong>z</strong>) = 〈<strong>x</strong>, <strong>z</strong>〉</p></li><li id="list-item-10"><p id="p-75"><span class="list-label">2. </span>The Gaussian radial basis function <em>k</em>(<strong>x</strong>, <strong>z</strong>) = exp(−<em>σ</em>‖<strong>x</strong> − <strong>z</strong>‖<sup>2</sup>), where <em>σ</em> is the bandwidth parameter</p></li><li id="list-item-11"><p id="p-76"><span class="list-label">3. </span>The Laplace radial basis function <em>k</em>(<strong>x</strong>, <strong>z</strong>) = exp(−<em>σ</em>‖<strong>x</strong> − <strong>z</strong>‖).</p></li></ul><p id="p-77">The solution to the minimization problem can also be written as a function of the kernel function. The resulting expression is <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-115.gif"/></noscript></span></span>. The choice of the kernel function and of the tuning parameters <em>λ</em>, <em>ε</em>, and <em>σ</em> are not straightforward. Because optimizing SVMs is not the focus of this article, we refer the reader to <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cherkassky and Ma (2004)</a>.</p></div><div id="sec-13" class="subsection"><h4>Neural networks:</h4><p id="p-78">NNs represent a nonparametric prediction procedure that captures additivity and epistasis by being able to model linear and complex nonlinear functions. The original idea of NN came from the theory of how neurons in the human brain work and interact, and how the brain conducts computations. In the NN, every unit is analogous to a brain neuron and the connections between them are analogous to synapses (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Hastie <em>et al.</em> 2009</a>). The first introduction of NNs in the context of brain architecture was presented by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bain (1873)</a> and <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">James (1890)</a>. <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">McCulloch and Pitts (1943)</a> developed a mathematical model for NNs.</p><p id="p-79">The basic layout of the NN is a two-stage network with three types of layers: an input layer; a hidden layer; and an output layer. This model is called the feed-forward NN and is illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A three-layer feed-forward neural network with K input layer units, L hidden layer units, and M output layer units." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A three-layer feed-forward neural network with &lt;em&gt;K&lt;/em&gt; input layer units, &lt;em&gt;L&lt;/em&gt; hidden layer units, and &lt;em&gt;M&lt;/em&gt; output layer units.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F3.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/6/1027/F3.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464478" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-80" class="first-child">A three-layer feed-forward neural network with <em>K</em> input layer units, <em>L</em> hidden layer units, and <em>M</em> output layer units.</p><div class="sb-div caption-clear"></div></div></div><p id="p-81"><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> shows a diagram of a three layer feed-forward NN with <em>K</em> input layer units, <em>L</em> hidden layer units, and <em>M</em> output layer units. <em>H</em><sub>1</sub>, <em>H</em><sub>2</sub>, …, <em>H<sub>L</sub></em> are called hidden layer units because they are not directly observed. When the NN is used to estimate a regression function, there typically is only one output layer unit. The hidden layer units are functions of linear combinations of the inputs, and the output layer units are functions of the hidden layer units. The output function of a feed-forward NN can be expressed in the following form:<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-116.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>where <em>K</em> is the number of units in the input layer, <em>I<sub>k</sub></em> is the <em>k<sup>th</sup></em> input, <em>β</em><sub>0</sub> <em>ε R<sup>M</sup></em> is the intercept (bias terms), <em>M</em> is the number of output layer units, <em>L</em> is the number of hidden layer units, <em>β<sub>l</sub></em> (<em>l</em> = 1, 2, …, <em>L</em>) are the output layer weights connecting the <em>l<sup>th</sup></em> hidden layer unit to the output layer units, <em>σ</em> is the activation function modeling the connection between the hidden layer and the output layer, and <em>w<sub>l</sub> ε R<sup>K</sup></em> and <em>b<sub>l</sub> ε R</em> are the unknown learning parameters of the hidden layer unit <em>l</em> (<em>l</em> = 1, 2, …, <em>L</em>) connecting the <em>k<sup>th</sup></em> neuron in the input layer to them (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Romero and Alquézar 2012</a>).</p><p id="p-82">In GS, typically <em>I<sub>k</sub></em> represents a vector of predictors (marker genotypes or other information) collected on individual <em>k</em> (<em>k</em> = 1, 2, …, <em>K</em>), where K is the number of individuals in the analysis. The activation function <em>σ</em> is typically chosen to be the sigmoid (logistic) or the Gaussian radial basis function.</p><p id="p-83"><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Gianola <em>et al.</em> (2011)</a> implemented NNs for GS using two real data sets. Other examples are shown in work by <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Lampinen and Vehtari (2001)</a> and <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Titterington (2004)</a>.</p></div></div></div><div class="section materials-methods" id="sec-14"><h2 class="">Materials and Methods</h2><p id="p-84">For the purpose of illustrating the parametric and nonparametric prediction approaches, simulated data were created by the R (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">R Development Core Team 2008</a>) package QTL Bayesian interval mapping (“qtlbim”) (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Yandell <em>et al.</em> 2012</a>). R can be downloaded from <a href="http://www.r-project.org">http://www.r-project.org</a>, the qtlbim package can be accessed by library(qtlbim) in R, and the description of the package can be found at <a href="http://cran.r-project.org/web/packages/qtlbim/qtlbim.pdf">http://cran.r-project.org/web/packages/qtlbim/qtlbim.pdf</a>. The reader can refer to <a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Yandell <em>et al.</em> (2007)</a> for detailed information about the qtlbim package. There are other publications as well where the qtlbim package is used to implement statistical methods. Some examples include <a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Yi <em>et al.</em> (2007)</a>, <a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Yi and Shriner (2008)</a>, and <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Piao <em>et al.</em> (2009)</a>. For comparing methods, we used a simulated <em>F</em><sub>2</sub> population with specifications listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464601/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464601/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Specification of the simulated <em>F</em><sub>2</sub> population: genetic architecture and heritability</span><div class="sb-div caption-clear"></div></div></div><p id="p-85">We simulated four sets of phenotypic and genotypic information for a <em>F</em><sub>2</sub> and a BC population. The results for the BC population can be found in the supporting information section. For each set we created 20 replicates, which yielded to a total of 80 phenotypic and 80 genotypic data sets. Within each replicate we created 25 different training–testing data sets. Half of the data sets assume only additive effects and half assume only epistatic effects without any additive effects. We only evaluated the two extreme genetic architectures. Finally, for each genetic architecture, we generated data with two different narrow sense heritabilities. The low heritability was determined to be 0.30, and the high heritability was 0.70. For each of the simulated combinations of population, genetic architecture, and heritability, the data contain phenotypic information for 1000 individuals and genotypic information for 2000 biallelic markers (the possible values coded as “A” and “H”) for each individual. Out of the 1000 individuals, 800 were chosen randomly to be in the training set to fit the model, and 200 individuals were in the testing set. We predicted the phenotype for the individuals in the testing set. The qtlbim package uses Cockerham’s model as the underlying genetic model. The simulated genome has 10 chromosomes, each having a specified length. The 2000 markers were distributed throughout the genome in such a way that each chromosome had 200 markers and the markers were equally spaced over the chromosomes. We assumed no missing genotypic values and no missing phenotypic values. The phenotypic values are normally distributed.</p><p id="p-86">For the additive model, we placed two QTL on each of the 10 chromosomes with either positive or negative additive effect. For the additive model we assumed no epistatic interaction.</p><p id="p-87">For the epistatic model, we only considered two-way interactions between the QTL. The interacting QTL were at the same genomic location as the QTL for the additive model, and only neighboring QTL were associated with each other, resulting in 10 two-way epistatic interactions, with each having either positive or negative epistatic effect on the phenotype. For the epistatic model, we assumed that the QTL contributed no additive effect. The phenotypic values were drawn from a normal distribution and are based on the <em>P</em> = <em>G</em> + <em>E</em> model. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the histograms of the simulated phenotypic values for the four population–genetic architecture–heritability combinations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The histograms of the simulated phenotypic values. The histograms represent the distribution of the phenotypic values for the F2 population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The histograms of the simulated phenotypic values. The histograms represent the distribution of the phenotypic values for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F4.medium.gif" width="440" height="205"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/6/1027/F4.medium.gif" width="440" height="205"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464432" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-88" class="first-child">The histograms of the simulated phenotypic values. The histograms represent the distribution of the phenotypic values for the <em>F</em><sub>2</sub> population.</p><div class="sb-div caption-clear"></div></div></div><p id="p-89">To compare the performance of the methods, we used cross-validation, where we divided the data into training sets and testing sets. The training sets were used to fit the models, and the testing sets were used to determine the performance of the particular method. The performance of the methods was calculated by the accuracy of prediction and the MSE. We define accuracy of prediction as the correlation between the true phenotypic values and the predicted phenotypic values. We evaluated parametric methods including parametric least squares regression, ridge regression, Bayesian ridge regression, BLUP, LASSO, Bayesian LASSO, Bayes A, Bayes B, Bayes C, and Bayes C<em>π</em>. We also evaluated nonparametric methods, including NWE, RKHS method, SVM, and NN. To implement the parametric and nonparametric methods, the statistical software R and software written in C++ provided by the Animal Science Department at Iowa State University were used. Specifications of the parameters and inputs for each method are described below.</p><div id="sec-15" class="subsection"><h3>Least squares regression</h3><p id="p-90">Because the number of markers exceed the number of the individuals in all simulated data sets, the idea of <a id="xref-ref-78-8" class="xref-bibr" href="#ref-78">Meuwissen (2001)</a> was adopted, and we first performed simple regression by coding each marker genotype as −1, 1. After fitting the 2000 simple linear models, one for each marker, we chose 300 of the markers with the most significant p-values. Then, these 300 markers were included into a final model, and we simultaneously used them to fit a linear model. To perform the linear regression, the <em>lm</em> function was used that can be found in the <em>stats</em> package (<a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">R Development Core Team 2008</a>) in R. Finally, the prediction for the testing set was performed by using the marker data for the testing set and the output for the estimated marker effects provided by the <em>lm</em> function.</p></div><div id="sec-16" class="subsection"><h3>Ridge regression</h3><p id="p-91">For this method, we used Gustavo de los Campos’ software written in R. The code implements the calculation of the ridge regression estimation of the marker effects discussed in the ridge regression section and uses these estimates to perform the prediction for the individuals in the testing set. For the procedure, all of the available phenotypic and marker information is used. In the estimation of the marker effect, the penalty parameter <em>λ</em> is chosen to have a value of: <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-117.gif"/></noscript></span></span>, where <em>h</em><sup>2</sup> is the narrow sense heritability and <em>Var</em>(<em>X</em>) is the sum of the 2000 marker variances. For all of the scenarios, we used <em>h</em><sup>2</sup> = 0.4.</p></div><div id="sec-17" class="subsection"><h3>Bayesian ridge regression</h3><p id="p-92">To fit the Bayesian ridge regression model, the function Bayesian linear regression (BLR) was used, which can be found in the BLR package (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de los Campos and Rodriguez 2010</a>) in R. For the specifications of the BLR function, we used a Gaussian prior for the marker effects with mean 0 and a common variance <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-118.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-119.gif"/></noscript></span></span> is unknown. <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6/1027/embed/mml-math-120.gif"/></noscript></span></span> is assigned to have a scaled inverse <em>χ</em><sup>2</sup> distribution with degrees of freedom <em>df<sub>BR</sub></em> = 5 and scale parameter <em>S<sub>BR</sub></em> = 0.01. The residual variance <em>σ<sub>E</sub></em> has a scaled inverse <em>χ</em><sup>2</sup> distribution with degrees of freedom <em>df<sub>E</sub></em> = 4 and scale parameter <em>S<sub>E</sub></em> = 1. The BLR function implements the Gibbs sampler, and the number of iterations is specified to be 20,000. We used 2000 iterations for the burn-in period without any thinning. To fit the Bayesian ridge regression, we used all available phenotypic and genotypic data.</p></div><div id="sec-18" class="subsection"><h3>BLUP</h3><p id="p-93">To implement BLUP, we used the <em>mixed.solve</em> function in R that can be found in the rrBLUP package (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Endelman 2011</a>). The available marker data were used as the design matrix for the random marker effects, and there was no fixed effect specified. The prediction was performed using the marker data for the testing set and the output for the predicted marker effects provided by the <em>mixed.solve</em> function.</p></div><div id="sec-19" class="subsection"><h3>LASSO</h3><p id="p-94">To predict phenotypic values in the testing set using the LASSO method, we used the <em>glmnet</em> function of the <em>glmnet</em> package (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Friedman <em>et al.</em> 2010</a>) in R. For the initial parameter values, the default setting was applied. The prediction was performed with the tuning parameter, <em>λ</em>, that minimized the average cross-validation error (<em>cvm</em>).</p></div><div id="sec-20" class="subsection"><h3>Bayesian LASSO</h3><p id="p-95">To fit the Bayesian LASSO method, the function BLR of the BLR package (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">de los Campos and Rodriguez 2010</a>) in R was used. The regularization parameter, <em>λ</em>, is specified to be random and has a Gamma prior distribution with shape parameter <em>α</em><sub>1</sub> = 0.53 and rate parameter <em>α</em><sub>2</sub> = 0.00005. The residual variance <em>σ<sub>E</sub></em> has a scaled inverse <em>χ</em><sup>2</sup> distribution with degrees of freedom <em>df<sub>E</sub></em> = 4 and scale parameter <em>S<sub>E</sub></em> = (<em>d<sub>f</sub></em> − 2)(1 − <em>h</em><sup>2</sup>)<em>Var</em>(<em>y</em>), where we specify that <em>d<sub>f</sub></em> = 4, <em>h</em><sup>2</sup> = 0.5, and <em>Var</em>(<em>y</em>) is the phenotypic variance. The Gibbs sampler was applied with 20,000 iterations, and 2000 iterations were in the burn-in period. The chain was not thinned.</p></div><div id="sec-21" class="subsection"><h3>Bayesian alphabet</h3><p id="p-96">To implement the Bayes A, Bayes B, Bayes C, and the Bayes C<em>π</em> models, software called <em>Gensel</em> (version 2.12) was used. GenSel was written by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fernando and Garrick (2008)</a> in C++ and is used for GS in animal breeding populations. The software is not available for the public. However, it is available to Iowa State University research collaborators working on GS.</p><p id="p-97">In GenSel Bayes A, Bayes B, Bayes C, and Bayes C<em>π</em> have been implemented. We used the settings for the four methods that are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464518/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464518/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Parameter specifications for Bayes A, Bayes B, Bayes C, and Bayes C<em>π</em> used in GenSel</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Nadaraya-Watson estimator</h3><p id="p-99">To use the NWE for predicting the phenotypic values in the testing set, first we formed the cross-validation criteria and we evaluated it on a grid of values. We examined the cross-validation criteria between 1 and 1000 and chose the value to be the bandwidth, <em>h</em>, that minimized the criteria. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows the bandwidth values that minimized each of the four data combinations for the NWE prediction. The code for calculating the optimal bandwidth value and for the prediction was written in R.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464453/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464453/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Bandwidth values used for each of the four combinations of genetic architectures and heritabilities for the Nadaraya-Watson prediction</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Reproducing kernel Hilbert space</h3><p id="p-100">The RKHS regression was based on methods and algorithms described by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">de los Campos <em>et al.</em> (2010b)</a> and the R implementation was developed by Gustavo <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">de los Campos <em>et al.</em> (2010b)</a>. To specify the RKHS regression, we chose the Gaussian reproducing kernel with the Euclidean distance for all of the eight combinations of genetic architectures, heritabilities, and population types. We fitted the model using three arbitrarily chosen bandwidth values. We performed the prediction for the testing set with each of the bandwidth values, and we averaged the three values of accuracy of selection and the three MSE values.</p></div><div id="sec-24" class="subsection"><h3>Support vector machine</h3><p id="p-101">To implement the SVM regression, we used the <em>ksvm</em> function of the <em>kernlab</em> package (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Karatzoglou <em>et al.</em> 2004</a>) in R. For the <em>ksvm</em> function we used epsilon-regression as the type and the radial basis (Gaussian) kernel as the kernel function. After fitting the model, the <em>predict</em> function was used to perform the prediction of the phenotypic values for the testing set. For the other input parameters, the default values were used.</p></div><div id="sec-25" class="subsection"><h3>Neural network</h3><p id="p-102">We implemented the NN model using the <em>brnn</em> function of the <em>brnn</em> package (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Rodriguez and Gianola 2013</a>) in R. This function fits a two-layer NN. We first map the input information into some basis function. Then, the inputs of the NN model are the marker-derived principal components. We specified the number of neurons to be three and the number of epochs to train to be 30 in the model. The other parameters were left at the default setting. For a detailed description of the application of the NN using the R package <em>brnn</em>, the reader can refer to <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Pérez-Rodiguez and Gianola (2013)</a> and <a id="xref-ref-86-3" class="xref-bibr" href="#ref-86">Pérez <em>et al.</em> (2013)</a>.</p></div></div><div class="section results-discussion" id="sec-26"><h2 class="">Results and Discussion</h2><p id="p-103">We compared 10 parametric and four nonparametric statistical GS methods. Comparisons were based on predicted accuracies of a simulated F2 progeny derived from crosses of inbred lines where genotypic variability was responsible for either 30% or 70% of the phenotypic variability. The underlying genetic architectures responsible for the genotypic variability consisted of 20 independently segregating biallelic loci that contributed equally either in an additive manner to a quantitative phenotype or through additive by additive epistatic interactions among 10 pairs of loci. Each GS method was applied to 20 sets of simulated progeny with 25 replicates for each of the four combinations of genetic architecture and heritability, which yielded 500 total replicates for each combination. Training sets were used to develop a model, and the model was used to predict phenotypes in the testing sets. Training sets consisted of simulated phenotypes and 2000 marker genotypes for 800 random progeny while the testing sets associated with the training sets consisted of the same information for 200 progeny derived from the same cross. The accuracy of prediction was determined by calculating the correlation between the predicted phenotypic values for the 200 individuals in the testing set with the simulated phenotypic values for the same 200 individuals. The MSE values were determined by calculating the sum of the squared differences between the 200 predicted phenotypic values in the testing set and the 200 simulated phenotypic values, and then dividing the sum by 200.</p><p id="p-104"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> report the average prediction accuracies and SE (sampling variabilities) of the 10 parametric and four nonparametric methods applied to the 500 replicates of the four combinations of genetic architecture and heritability. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> and <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> report the average MSE values and SE of the MSE values of the 14 methods applied to the 500 replicates of the four combinations of genetic architecture and heritability. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> each contain 14 boxplots of accuracy of prediction values for the 14 different methods. The boxplots show the distribution of the accuracy of prediction values for the 500 runs. <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>, <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>, <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11</a>, and <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 12</a> each contain 14 boxplots of MSE values for the 14 different methods. In each figure, the first 10 boxplots are for the parametric methods, and the last four (shaded) are for the nonparametric methods. These boxplots show the distribution of the MSE values for the 500 runs. The first plot of <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure 13</a> shows the ratio of the accuracy averaged over the parametric methods (excluding the least squares method because it is an outlier) and the accuracy averaged over the nonparametric methods, and the second plot of <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure 13</a> shows the ratio of the MSE averaged over the parametric methods (excluding the least squares method) and the MSE averaged over the nonparametric methods. The left sides of the plots show the ratios for the additive genetic architecture, and the right sides of the plots show the ratios for the epistatic genetic architecture. These summary plots clearly show the advantage of using nonparametric methods when epistasis is present. In both heritability scenarios, the parametric-to-nonparametric accuracy ratio is lower for the epistatic genetic architecture than for the additive genetic architecture. The parametric-to-nonparametric MSE ratio is higher for the epistatic genetic architecture than for the additive genetic architecture.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464535/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464535/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Mean and SE of the prediction accuracy values for the parametric and the nonparametric methods for the <em>F</em><sub>2</sub> population with heritability <em>h</em><sup>2</sup> = 0.70</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464410/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464410/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Mean and SE of the prediction accuracy values for the parametric and the nonparametric methods for the <em>F</em><sub>2</sub> population with heritability <em>h</em><sup>2</sup> = 0.30</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464407/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Mean and standard error of the mean squared error values for the parametric and the nonparametric methods for the <em>F</em><sub>2</sub> population with heritability <em>h</em><sup>2</sup> = 0.70</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464525/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464525/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Mean and standard error of the mean squared error values for the parametric and the nonparametric methods for the <em>F</em><sub>2</sub> population with heritability <em>h</em><sup>2</sup> = 0.30</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of accuracy of prediction for the F2 population with additive genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of accuracy of prediction for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with additive genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F5.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/6/1027/F5.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464502" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-109" class="first-child">The boxplots of accuracy of prediction for the <em>F</em><sub>2</sub> population with additive genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of accuracy of prediction for the F2 population with epistatic genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of accuracy of prediction for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with epistatic genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F6.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/6/1027/F6.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464460" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-110" class="first-child">The boxplots of accuracy of prediction for the <em>F</em><sub>2</sub> population with epistatic genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of accuracy of prediction for the F2 population with additive genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of accuracy of prediction for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with additive genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F7.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/6/1027/F7.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464575" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-111" class="first-child">The boxplots of accuracy of prediction for the <em>F</em><sub>2</sub> population with additive genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of accuracy of prediction for the F2 population with epistatic genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of accuracy of prediction for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with epistatic genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F8.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/6/1027/F8.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464565" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-112" class="first-child">The boxplots of accuracy of prediction for the <em>F</em><sub>2</sub> population with epistatic genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of mean squared error for the F2 population with additive genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of mean squared error for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with additive genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F9.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/4/6/1027/F9.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464403" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-113" class="first-child">The boxplots of mean squared error for the <em>F</em><sub>2</sub> population with additive genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of mean squared error for the F2 population with epistatic genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of mean squared error for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with epistatic genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F10.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/4/6/1027/F10.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464424" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-114" class="first-child">The boxplots of mean squared error for the <em>F</em><sub>2</sub> population with epistatic genetic architecture and heritability of 0.70. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of mean squared error for the F2 population with additive genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of mean squared error for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with additive genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F11.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/4/6/1027/F11.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464529" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-115" class="first-child">The boxplots of mean squared error for the <em>F</em><sub>2</sub> population with additive genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The boxplots of mean squared error for the F2 population with epistatic genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The boxplots of mean squared error for the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; population with epistatic genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F12.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12" src="https://www.g3journal.org/content/ggg/4/6/1027/F12.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464494" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12</span> <p id="p-116" class="first-child">The boxplots of mean squared error for the <em>F</em><sub>2</sub> population with epistatic genetic architecture and heritability of 0.30. The first 10 boxplots correspond to the parametric methods, and the last four (gray) boxplots correspond to the nonparametric methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of the parametric to nonparametric accuracy and MSE ratios. The left sides of the plots show the additive cases, and the right sides of the plots show the epistatic cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1994978902" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plots of the parametric to nonparametric accuracy and MSE ratios. The left sides of the plots show the additive cases, and the right sides of the plots show the epistatic cases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/1027/F13.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13" src="https://www.g3journal.org/content/ggg/4/6/1027/F13.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/1027/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13</span> <p id="p-117" class="first-child">Plots of the parametric to nonparametric accuracy and MSE ratios. The left sides of the plots show the additive cases, and the right sides of the plots show the epistatic cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-118">Genetic architecture responsible for the genetic contribution to the phenotypes had the greatest impact on differences of accurate predictions among the GS methods. If the genetic architecture for the trait is due to additive-by-additive epistasis among 10 pairs of independently segregating loci, then parametric GS methods are unable to predict the phenotypes in the testing sets (shown in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>, <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a>, and <a id="xref-fig-12-2" class="xref-fig" href="#F12">Figure 12</a>). In contrast, nonparametric methods, particularly the NWE, the RKHS, and SVM, provided predictions that are reasonably accurate, especially for traits with higher heritabilities (shown in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). Our results are consistent with the statement by <a id="xref-ref-35-9" class="xref-bibr" href="#ref-35">Gianola (2006)</a> that nonparametric methods should be able to better predict phenotypes that are based on genetic architectures consisting of epistatic interactions. If the underlying genetic architecture is additive, then parametric GS methods are slightly better than the nonparametric methods for both levels of heritability and types of segregating progeny. Both the accuracy of prediction and the MSE results suggest the same about the models in terms of predictive performance. When additive effects are present, the least squares regression performs the worst among the parametric methods, and the NWE performs the worst among the nonparametric methods (shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>, and <a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 11</a>). When epistasis is present, the nonparametric NWE, the RKHS, and the SVM perform significantly better than the parametric methods (shown in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>, <a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10</a>, and <a id="xref-fig-12-3" class="xref-fig" href="#F12">Figure 12</a>). Among the parametric methods, the least squares regression has the highest accuracy of prediction values when epistasis is present. However, least squares has the highest MSE values among the parametric methods as well when epistasis is present. It suggests that the least squares method estimates the QTL effects from both loci involved in the epistasis more accurately than the other parametric methods in the <em>F</em><sub>2</sub> population. The parametric methods other than the least squares are shrinking the QTL effects too much. Among the nonparametric methods, the NN showed poor predictive ability when epistasis is present. We know that NN is prone to over-fitting (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Lawrence <em>et al.</em> 1997</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Smith 1996</a>), which would affect prediction ability. Most of the results are consistent with the fact that parametric approaches assume that the explanatory variables fitted in the model are independent. When we only simulate additive effects, but not epistasis, the markers are assumed to be independent. In this case, we satisfy the parametric model assumption of having independent explanatory variables, so the parametric models have a larger predictive power than the nonparametric models. However, when we simulate epistasis, the markers are dependent, which violates the parametric model assumption. Nonparametric models can handle epistatic models without explicitly modeling the interactions.</p><p id="p-119">Recently, the inability of parametric GS methods to predict has been observed in experimental data. Parametric GS methods were unable to predict chill coma recovery, a quantitatively measured adaptive trait in Drosophila (Trudy F. C. MacKay, personal communication). Two-dimensional scans of the whole genome had previously revealed that the genetic architecture of this trait is composed primarily of interactions involving many loci. Thus, the simulated architectures used in our study are reasonable for many quantitative traits.</p><p id="p-120">The clear distinctions of estimated accuracies and MSE values between parametric and nonparametric methods when underlying genetic architecture is epistatic suggest that data analyses consisting of a combination of parametric and nonparametric GS methods could be used as a diagnostic to reveal the prevalent genetic architecture of the trait. It is likely that the true underlying genetic architecture consists of mixtures of additive and epistatic genetic effects, so the inferential limits of applying pairs of GS methods to data from samples of breeding populations as a diagnostic needs further investigation. However, the first step is to look at the extremes in terms of genetic architecture.</p><p id="p-121">Our results also suggest that if the goal of the research is to accurately predict the genotypic value of an individual, particularly for purposes of selection, and if the underlying genetic architecture of the traits are not known, then it is best to use the nonparametric NWE, the RKHS, or the SVM. Unfortunately, these methods do not provide interpretable inferences about relative weighting that is being applied to various regions of the genome, <em>i.e.</em>, inferences about specific allelic contributions to the trait are limited. If the goal is genetic improvement and the underlying genetic architecture is known to be additive, then parametric GS methods will provide better predictions for selection. It has previously been hypothesized (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Xu <em>et al.</em> 2011</a>) that if all specific desirable alleles are known, then gene stacking (genome construction) based on optimization approaches will be more effective and efficient than GS approaches. Thus, in the interest of both immediate and long-term genetic improvement goals, a combination of data analyses consisting of parametric, nonparametric GS methods as well as genetic mapping (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Guo <em>et al.</em> 2013</a>) should be applied to data derived from plant breeding populations.</p><p id="p-122">Although heritability did not affect the ability to distinguish among GS methods, it did affect estimated accuracies. When heritability is high and genetic architecture is additive, predictions are more accurate than for low heritability. When genetic architecture is based on epistasis and the trait exhibits low heritability, predictions are not very accurate for almost all GS methods. Even when the heritability is 0.70, the highest mean for prediction accuracy is 0.35, which indicates that further improvement of the models is necessary. Also, further research is needed to determine the affects of more complex plant breeding population structures. Typically, plant breeding population structures consist of inbred progeny derived from multiple crosses involving related and unrelated inbreds (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Guo and Beavis 2011</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Guo <em>et al.</em> 2013</a>). Thus, GS needs to accurately predict phenotypes not only among subsets of progeny from related families within generations but also among generations of related and unrelated families.</p><p id="p-123">In practice, plant breeders do not know the genetic architecture responsible for quantitative traits and the dynamics of selection for genetic improvement will tend to favor alleles that contribute to additive components. Genetic improvement is affected not only by the underlying genetic architecture but also by additional types of unpredictable genetic contributions including intra-locus dominance and genotype by environment interactions. Herein, we have demonstrated the superior ability of the nonparametric NWE, the RKHS, and the SVM methods to accurately predict phenotypes for additive by additive inter-locus interactions. We hypothesize that nonparametric GS methods also will enable more accurate predictions of individual genotypic value for traits that are affected by dominance and genotype by environment interactions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-124">We thank Rohan Fernando (Animal Science Department, Iowa State University) and Gustavo de los Campos (Biostatistics Department, University of Alabama) for their valuable input and help with the software development for this project. Also, we gratefully acknowledge the help of Dorian Garrick (Animal Science Department, Iowa State University) for providing access to the <em>GenSel</em> software.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010298/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010298/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-125"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-01-08"><span class="received-label">Received </span>January 8, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-03-18"><span class="accepted-label">Accepted </span>March 18, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Howard <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bain</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1873</span> <span class="cit-source">Mind and Body: The Theories of Their Relation</span>, <span class="cit-publ-name">D. Appleton and Company</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.6.1027.2"><div class="cit-metadata unstructured">Beavis, W. D., 1994 The power and deceit of QTL experiments: lessons from comparative QTL studies, pp. 250–265 in <em>Proceedings of the 49th Annual Corn and Sorghum Research Conference</em>, edited by D. B. Wilkinson. Washington, DC: American Seed Trade Association 250– 265.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.3" data-doi="10.2135/cropsci1994.0011183X003400010003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Prediction of maize single-cross performance using RFLPs and information from related hybrids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D25%26rft.atitle%253DPrediction%2Bof%2BMaize%2BSingle-Cross%2BPerformance%2BUsing%2BRFLPs%2Band%2BInformation%2Bfrom%2BRelated%2BHybrids%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400010003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400010003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994MV45600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.4" data-doi="10.2135/cropsci2008.03.0131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular markers and selection for complex traits in plants: learning from the last 20 years.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1649%26rft.epage%253D1664%26rft.atitle%253DMolecular%2BMarkers%2Band%2BSelection%2Bfor%2BComplex%2BTraits%2Bin%2BPlants%253A%2BLearning%2Bfrom%2Bthe%2BLast%2B20%2BYears%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000259792100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Breeding for Quantitative Traits in Plants</span>, <span class="cit-publ-name">Stemma Press, Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.6" data-doi="10.1016/S0893-6080(03)00169-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherkassky</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Practical selection of SVM parameters and noise estimation for SVM regression.</span> <abbr class="cit-jnl-abbrev">Neural Netw.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2Bnetworks%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BNeural%2BNetwork%2BSociety%26rft.stitle%253DNeural%2BNetw%26rft.aulast%253DCherkassky%26rft.auinit1%253DV.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D126%26rft.atitle%253DPractical%2Bselection%2Bof%2BSVM%2Bparameters%2Band%2Bnoise%2Bestimation%2Bfor%2BSVM%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0893-6080%252803%252900169-2%26rft_id%253Dinfo%253Apmid%252F14690712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0893-6080(03)00169-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14690712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188041400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Routman</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Epistasis and its contribution to genetic variance components.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCheverud%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D139%26rft.issue%253D3%26rft.spage%253D1455%26rft.epage%253D1461%26rft.atitle%253DEpistasis%2Band%2Bits%2Bcontribution%2Bto%2Bgenetic%2Bvariance%2Bcomponents.%26rft_id%253Dinfo%253Apmid%252F7768453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8zLzE0NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">A calibration curve for radiocarbon dates.</span> <abbr class="cit-jnl-abbrev">Antiquity</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiquity%26rft.volume%253D49%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1975AX43000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">An extension of the concept of partitioning hereditary variance for analysis of covariances among relatives when epistasis is present.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D859%26rft.epage%253D882%26rft.atitle%253DAN%2BEXTENSION%2BOF%2BTHE%2BCONCEPT%2BOF%2BPARTITIONING%2BHEREDITARY%2BVARIANCE%2BFOR%2BANALYSIS%2BOF%2BCOVARIANCES%2BAMONG%2BRELATIVES%2BWHEN%2BEPISTASIS%2BIS%2BPRESENT%26rft_id%253Dinfo%253Apmid%252F17247525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzOS82Lzg1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNi8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-4.6.1027.10"><div class="cit-metadata unstructured">Cooper, M., D. W. Podlich, K. P. Micallef, O. S. Smith, N. M. Jensen, <em>et al.</em>, 2002 Complexity, quantitative traits and plant breeding: a role for simulation modelling in the genetic improvement of crops. Quantitative Genetics, Genomics and Plant Breeding 143–166.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.11" data-doi="10.2307/1267913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbeil</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Restricted maximum likelihood (REML) estimation of variance components in the mixed model.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D18%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267913&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BG16700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.12" data-doi="10.1093/hmg/11.20.2463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cordell</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Epistasis: what it means, what it doesn’t mean, and statistical methods to detect it in humans.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2463</span>–<span class="cit-lpage">2468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DCordell%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D11%26rft.issue%253D20%26rft.spage%253D2463%26rft.epage%253D2468%26rft.atitle%253DEpistasis%253A%2Bwhat%2Bit%2Bmeans%252C%2Bwhat%2Bit%2Bdoesn%2527t%2Bmean%252C%2Band%2Bstatistical%2Bmethods%2Bto%2Bdetect%2Bit%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F11.20.2463%26rft_id%253Dinfo%253Apmid%252F12351582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8yMC8yNDYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Support-vector networks.</span> <abbr class="cit-jnl-abbrev">Machine Leaming</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2BLeaming%26rft.volume%253D20%26rft.spage%253D273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cristianini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shawe-Taylor</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">An Introduction to Support Vector Machines and Other Kernel-based Learning Methods</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.15" data-doi="10.1098/rstb.2009.0275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On epistasis: why it is unimportant in polygenic directional selection.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>(<span class="cit-issue">1544</span>): <span class="cit-fpage">1241</span>–<span class="cit-lpage">1244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCrow%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1241%26rft.epage%253D1244%26rft.atitle%253DOn%2Bepistasis%253A%2Bwhy%2Bit%2Bis%2Bunimportant%2Bin%2Bpolygenic%2Bdirectional%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0275%26rft_id%253Dinfo%253Apmid%252F20308099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.16" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.17" data-doi="10.1111/j.1467-9574.2005.00304.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">P. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafner</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ridge regression revisited.</span> <abbr class="cit-jnl-abbrev">Stat. Neerl.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">498</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BNeerl.%26rft.volume%253D59%26rft.spage%253D498%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9574.2005.00304.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9574.2005.00304.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-4.6.1027.18"><div class="cit-metadata unstructured">de los Campos, G., and P. P. Rodriguez, 2010 BLR: Bayesian Linear Regression. R package version 1.2. <a href="http://CRAN.R-project.org/package=BLR">http://CRAN.R-project.org/package=BLR</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.19" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.20" data-doi="10.1038/nrg2898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Predicting genetic predisposition in humans: the promise of whole-genome markers.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">880</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D880%26rft.epage%253D886%26rft.atitle%253DPredicting%2Bgenetic%2Bpredisposition%2Bin%2Bhumans%253A%2Bthe%2Bpromise%2Bof%2Bwhole-genome%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2898%26rft_id%253Dinfo%253Apmid%252F21045869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2898&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21045869&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.21" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.22" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1937</span> <span class="cit-source">Genetics of the Evolutionary Process</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.24" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.25" data-doi="10.1214/009053604000000067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnstone</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Least angle regression.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D32%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1214%252F009053604000000067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/009053604000000067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000221411000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-4.6.1027.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-publ-name">Pearson Education Limited, Essex, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.27"><div class="cit-metadata unstructured">Faraway, J. J., 2006 Linear Models with R. CRC Press, Boca Raton, FL.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>  <span class="cit-article-title">GenSel—User manual for a portfolio of genomic selection related analyses. Animal Breeding and Genetics, Iowa State University, Ames.</span> Available at: <a href="http://taurus.ansci.iastate.edu/gensel">http://taurus.ansci.iastate.edu/gensel</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.6.1027.29" data-doi="10.1051/gse:19890407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, and M. Grossman, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Marker assisted selection using best linear unbiased prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D21%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19890407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19890407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989CK31500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.30" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">The correlation between relatives on the supposition of Mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.31" data-doi="10.1101/gr.086660.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic architecture of quantitative traits in mice, flies, and humans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D723%26rft.epage%253D733%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bquantitative%2Btraits%2Bin%2Bmice%252C%2Bflies%252C%2Band%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086660.108%26rft_id%253Dinfo%253Apmid%252F19411597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvNzIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.32" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regularization paths for generalized linear models via coordinate descent.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span> URL <a href="http://www.jstatsoft.org/v33/i01/">http://www.jstatsoft.org/v33/i01/</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Bayesian Data Analysis</span>. <span class="cit-publ-name">Chapman and Hall/CRC, Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.34" data-doi="10.2307/2669776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">E. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The variable selection problem.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1304</span>–<span class="cit-lpage">1308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D95%26rft.spage%253D1304%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2669776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2669776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000165470300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.35" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.36" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and the bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.6.1027.37"><div class="cit-metadata unstructured">Gianola, D., G. de los Campos, O. Gonzlez-Recio, N. Long, H. Okut <em>et al.</em>, 2010 Statistical learning methods for genome-based analysis of quantitative traits. World Genetic Congress Applied to Livestock Production. Leipzig, Germany, CD-ROM Communication 0014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.38" data-doi="10.1186/1471-2156-12-87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okut</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Predicting complex quantitative traits with Bayesian neural networks: a case study with Jersey cows and wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DPredicting%2Bcomplex%2Bquantitative%2Btraits%2Bwith%2BBayesian%2Bneural%2Bnetworks%253A%2Ba%2Bcase%2Bstudy%2Bwith%2BJersey%2Bcows%2Band%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-87%26rft_id%253Dinfo%253Apmid%252F21981731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21981731&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.39" data-doi="10.1046/j.1365-2540.2000.00698.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Quantitative trait loci and gene interaction: the quantitative genetics of metapopulation.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D84%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00698.x%26rft_id%253Dinfo%253Apmid%252F10849084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00698.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10849084&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.40" data-doi="10.1007/s11032-010-9503-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">In silico genotyping of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBreed%26rft.aulast%253DGuo%26rft.auinit1%253DB.%26rft.volume%253D27%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D113%26rft.atitle%253DIn%2Bsilico%2Bgenotyping%2Bof%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-010-9503-4%26rft_id%253Dinfo%253Apmid%252F21289856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-010-9503-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21289856&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.41" data-doi="10.1007/s00122-013-2100-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Family-based association mapping in crop species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1419</span>–<span class="cit-lpage">1430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D1419%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2100-2%26rft_id%253Dinfo%253Apmid%252F23620001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2100-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23620001&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.42" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.43" data-doi="10.1534/genetics.108.100289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection using low-density marker panels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100289%26rft_id%253Dinfo%253Apmid%252F19299339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.44" data-doi="10.1186/1297-9686-42-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in German Holstein cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D42%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenetic%2Brelationship%2Binformation%2Bon%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BGerman%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-5%26rft_id%253Dinfo%253Apmid%252F20170500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20170500&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.45" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Extension of the bayesian alphabet for genomic selection. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardle</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Applied Nonparametric Regression</span>. <span class="cit-publ-name">Cambridge University Press, Cambridge, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.47" data-doi="10.1214/aos/1176343414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harville</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Extension of the Gauss-Markov theorem to include estimation of random effects.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D4%26rft.spage%253D384%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176343414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176343414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>The Elements of Statistical Learning: Data Mining</em>, <em>Inference</em>, <em>and Prediction</em></span>. <span class="cit-publ-name">Springer, New York, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">Estimates of changes in herd environment.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D32%26rft.spage%253D706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Estimation of Variance and Covariance Components. Biometrics</span> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">226</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.51" data-doi="10.2307/2527669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Krosigk</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">The estimation of environmental and genetic trends from records subject to culling.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D15%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1959WX52200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-other" id="cit-4.6.1027.52"><div class="cit-metadata unstructured">Henderson, C. R., 1963 Selection index and expected genetic advance. NAS-NRC Publ. 982.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.53"><div class="cit-metadata unstructured">Henderson, C. R., 1975a Use of all relatives in intraherd prediction of breeding values and producing abilities. J. Dairy Sci. 58: 1910.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.54" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span><span class="cit-comment">b</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph, Ontario</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-other" id="cit-4.6.1027.56"><div class="cit-metadata"><cite>Henderson, C. R., 1988 Simple Method to Compute Biases and Mean Squared Errors of Linear Estimators and Predictors in a Selection Model Assuming Multivariate Normality. Journal of Dairy Science <span class="cit-vol">71</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">3135</span>–<span class="cit-lpage">3142</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.57" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: A comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Ridge regression: some simulations.</span> <abbr class="cit-jnl-abbrev">Communications in Statistics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommunications%2Bin%2BStatistics%26rft.volume%253D4%26rft.spage%253D105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1975BD50700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.59" data-doi="10.2307/1267351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span><span class="cit-comment">a</span> <span class="cit-article-title">Ridge regression: biased estimation for nonorthogonal problems.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D12%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970F898700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.60" data-doi="10.2307/1267352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span><span class="cit-comment">b</span> <span class="cit-article-title">Ridge regression: applications to nonorthogonal problems.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D12%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970F898700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.61" data-doi="10.1007/s001220050565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angeles</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magpantay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Pyramiding of bacterial blight resistance genes in rice: marker-assisted selection using RFLP and PCR.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XV26800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.62" data-doi="10.1073/pnas.1213423109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Epistasis dominates the genetic architecture of Drosophila quantitative traits.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">15553</span>–<span class="cit-lpage">15559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213423109%26rft_id%253Dinfo%253Apmid%252F22949659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM5LzE1NTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1890</span> <span class="cit-source">The Principles of Psychology</span>. <span class="cit-publ-name">H. Holt and Company</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-other" id="cit-4.6.1027.64"><div class="cit-metadata unstructured">Judge, G. G., W. E. Griffiths, R. C. Hill, H. Lutkepohl, and T.-C. Lee, 1985 The Theory and Practice of Econometrics. New York: Wiley.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karatzoglou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smola</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornik</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeileis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">kernlab - An S4 Package for Kernel Methods in R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span> URL <a href="http://www.jstatsoft.org/v11/i09/">http://www.jstatsoft.org/v11/i09/</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.6.1027.66"><div class="cit-metadata unstructured">Kohavi, R. Mellish, C. S. ed., 1995 A study of cross-validation and bootstrap for estimation and model selection, pp. 1137–1143 in <em>Proceedings of the 14th International Joint Conference on Artificial Intelligence</em>, edited by C. S. Mellish. San Francisco, CA: Morgan Kaufmann Publishers.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruschke</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Doing Bayesian Data Analysis: A Tutorial with R and BUGS</span>. <span class="cit-publ-name">Academic Press, Burlington, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.68" data-doi="10.1016/S0893-6080(00)00098-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lampinen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, A. and Vehtari, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bayesian approach for neural networks review and case studies.</span> <abbr class="cit-jnl-abbrev">Neural Netw.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2Bnetworks%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BNeural%2BNetwork%2BSociety%26rft.stitle%253DNeural%2BNetw%26rft.aulast%253DLampinen%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D257%26rft.epage%253D274%26rft.atitle%253DBayesian%2Bapproach%2Bfor%2Bneural%2Bnetworks--review%2Band%2Bcase%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0893-6080%252800%252900098-8%26rft_id%253Dinfo%253Apmid%252F11341565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0893-6080(00)00098-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11341565&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168334700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.6.1027.69"><div class="cit-metadata unstructured">Lawrence, S., C. L. Giles, and A. C. Tsoi, 1997 Lessons in neural network training: Overfitting may be harder than expected, pp. 540–545 in <em>Proceedings of the Fourteenth National Conference on Artificial Intelligence</em>, <em>AAAI-97</em>. AAAI Press, Menlo Park, California.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-source">Solving Least Squares Problems</span>. <span class="cit-publ-name">Prentice Hall</span>, <span class="cit-publ-loc">Englewood Cliffs</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-source">The Genetic Basis of Evolutionary Change</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.72" data-doi="10.1007/s00122-011-1648-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Application of support vector regression to genome-assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1074</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLong%26rft.auinit1%253DN.%26rft.volume%253D123%26rft.issue%253D7%26rft.spage%253D1065%26rft.epage%253D1074%26rft.atitle%253DApplication%2Bof%2Bsupport%2Bvector%2Bregression%2Bto%2Bgenome-assisted%2Bprediction%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1648-y%26rft_id%253Dinfo%253Apmid%252F21739137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1648-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21739137&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.73" data-doi="10.1007/s00122-007-0627-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maenhout</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Baets</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haesaert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Bockstaele</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Support vector machine regression for the prediction of maize hybrid performance.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMaenhout%26rft.auinit1%253DS.%26rft.volume%253D115%26rft.issue%253D7%26rft.spage%253D1003%26rft.epage%253D1013%26rft.atitle%253DSupport%2Bvector%2Bmachine%2Bregression%2Bfor%2Bthe%2Bprediction%2Bof%2Bmaize%2Bhybrid%2Bperformance.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0627-9%26rft_id%253Dinfo%253Apmid%252F17849095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0627-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17849095&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250071900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayr</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1942</span> <span class="cit-source">Systematics and the Origin of Species</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.75" data-doi="10.1007/BF02478259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitts</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">A logical calculus of ideas immanent in nervous activity.</span> <abbr class="cit-jnl-abbrev">Bull. Math. Biophys.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull.%2BMath.%2BBiophys.%26rft.volume%253D5%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02478259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02478259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.76" data-doi="10.2307/2532615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGilchrist</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">REML estimation for survival models with frailty.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DMcGilchrist%26rft.auinit1%253DC.%2BA.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D221%26rft.epage%253D225%26rft.atitle%253DREML%2Bestimation%2Bfor%2Bsurvival%2Bmodels%2Bwith%2Bfrailty.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2532615%26rft_id%253Dinfo%253Apmid%252F8513103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2532615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8513103&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LB23000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Quantitative trait locus (QTL) mapping using different testers and independent population samples in maize reveals low power of QTL detection and large bias in estimates of QTL effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMelchinger%26rft.auinit1%253DA.%2BE.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D383%26rft.epage%253D403%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2B%2528QTL%2529%2BMapping%2BUsing%2BDifferent%2BTesters%2Band%2BIndependent%2BPopulation%2BSamples%2Bin%2BMaize%2BReveals%2BLow%2BPower%2Bof%2BQTL%2BDetection%2Band%2BLarge%2BBias%2Bin%2BEstimates%2Bof%2BQTL%2BEffects%26rft_id%253Dinfo%253Apmid%252F9584111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ5LzEvMzgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC82LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.79" data-doi="10.1023/A:1009651919792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nair</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishna</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genome mapping, molecular markers and marker-assisted selection in crop plants.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D3%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1009651919792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1009651919792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.80" data-doi="10.1016/j.ajhg.2009.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Epistasis and its implications for personal genetics.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMoore%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D85%26rft.issue%253D3%26rft.spage%253D309%26rft.epage%253D320%26rft.atitle%253DEpistasis%2Band%2Bits%2Bimplications%2Bfor%2Bpersonal%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.08.006%26rft_id%253Dinfo%253Apmid%252F19733727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19733727&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270104500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.81" data-doi="10.1137/1109020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadaraya</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">On estimating regression.</span> <abbr class="cit-jnl-abbrev">Theory Probab. Appl.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheory%2BProbab.%2BAppl.%26rft.volume%253D9%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1137%252F1109020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1137/1109020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nocedal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-source">Numerical Optimization</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.83" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Bayesian Lasso.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.84" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the bayesian linear regression package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-4.6.1027.85"><div class="cit-metadata unstructured">Pérez-Rodriguez, P., and D. Gianola, 2013 brnn: brnn (Bayesian regularization for feed-forward neural networks). R package version 0.3. <a href="http://CRAN.R-project.org/package=brnn">http://CRAN.R-project.org/package=brnn</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodiguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Technical Note: An R package for fitting Bayesian regularized neural networks with applications in animal breeding. J. Animal Sci. In press.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bayesian dissection for genetic architecture of traits associated with nitrogen utilization efficiency in rice.</span> <abbr class="cit-jnl-abbrev">Afr. J. Biotechnol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">6834</span>–<span class="cit-lpage">6839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAfr.%2BJ.%2BBiotechnol.%26rft.volume%253D8%26rft.spage%253D6834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.88" data-doi="10.2135/cropsci2008.10.0595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ridge regression and extensions for genomewide selection in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1165%26rft.epage%253D1176%26rft.atitle%253DRidge%2BRegression%2Band%2BExtensions%2Bfor%2BGenomewide%2BSelection%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.10.0595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.10.0595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000268109800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-web" id="cit-4.6.1027.89"><div class="cit-metadata unstructured">R Development Core Team, 2008 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.90" data-doi="10.2307/2411260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The evolution of canalization and the breaking of Von Baer’s laws: modeling the evolution of development with epistasis.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">647</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D52%26rft.spage%253D647%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000074621000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.91" data-doi="10.1214/ss/1177011926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">That BLUP is a good thing: The estimation of random effects.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D6%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177011926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177011926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.92" data-doi="10.1016/j.neunet.2011.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alquézar</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparing error minimized extreme learning machines and support vector sequential feed-forward neural networks.</span> <abbr class="cit-jnl-abbrev">Neural Netw.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2Bnetworks%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BNeural%2BNetwork%2BSociety%26rft.stitle%253DNeural%2BNetw%26rft.aulast%253DRomero%26rft.auinit1%253DE.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D122%26rft.epage%253D129%26rft.atitle%253DComparing%2Berror%2Bminimized%2Bextreme%2Blearning%2Bmachines%2Band%2Bsupport%2Bvector%2Bsequential%2Bfeed-forward%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neunet.2011.08.005%26rft_id%253Dinfo%253Apmid%252F21959130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neunet.2011.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959130&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruppert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wand</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Semiparametric Regression</span>. <span class="cit-publ-name">Cambridge Univ. Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaalje</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rencher</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Linear Models in Statistics</span>. <span class="cit-publ-name">Wiley Series in Probability and Statistics, Hoboken, New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.95"><div class="cit-metadata unstructured">Schucany, W. R., 2004 Kernel smoothers: an overview of curve estimators for the first graduate course in nonparametric statistics. Stat. Sci. 4: 663–675.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.96"><div class="cit-metadata unstructured">Sheather, S. J., 2004 Density estimation. Stat. Sci. 19: 588–597.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-source">Density Estimation for Statistics and Data Analysis. Monographs on Statistics and Applied Probability</span>. <span class="cit-publ-name">Chapman and Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Neural Networks for Statistical Modeling</span>. <span class="cit-publ-name">International Thomson Computer Press</span>, <span class="cit-publ-loc">Boston</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinwart</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christmann</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Support Vector Machines</span>. <span class="cit-publ-name">Springer, New York, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.101" data-doi="10.1214/088342304000000099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Titterington</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bayesian methods for neural networks and related models.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D19%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1214%252F088342304000000099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/088342304000000099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.102" data-doi="10.1017/S0016672309990334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usai</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">LASSO with cross-validation for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DUsai%26rft.auinit1%253DM.%2BG.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DLASSO%2Bwith%2Bcross-validation%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990334%26rft_id%253Dinfo%253Apmid%252F20122298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122298&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">The Nature of Statistical Learning Theory</span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vashist</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> A new learning paradigm: Learning using privileged information. Neural Networks 22: 544–557.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Marker-assisted introgression in backcross breeding programs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1923</span>–<span class="cit-lpage">1932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1923%26rft.epage%253D1932%26rft.atitle%253DMarker-Assisted%2BIntrogression%2Bin%2BBackcross%2BBreeding%2BPrograms%26rft_id%253Dinfo%253Apmid%252F8978075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzE5MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waddington</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">Canalization of development and inheritance of acquired characters.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D150%26rft.spage%253D563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Epistasis: Genetic Constraint Within. Populations and Accelerant of Divergence Among Them</span>. <span class="cit-publ-name">Oxford Univ. Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Spline Models for Observational Data</span>. <span class="cit-publ-name">Society for Industrial and Applied Mathematics</span>, <span class="cit-publ-loc">Philadelphia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Smooth regression analysis.</span> <abbr class="cit-jnl-abbrev">Sankhya A.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSankhya%2BA.%26rft.volume%253D26%26rft.spage%253D359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.1027.110"><div class="cit-metadata unstructured">Wilson, S. R., 2004 Epistasis, pp. 317–320 in Nature Encyclopedia of the Human Genome, Vol. 2. London: Nature Publishing Group.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.111" data-doi="10.1016/j.ejor.2011.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An optimization approach to gene stacking.</span> <abbr class="cit-jnl-abbrev">Eur. J. Oper. Res.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BOper.%2BRes.%26rft.volume%253D214%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ejor.2011.04.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ejor.2011.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.112" data-doi="10.1093/bioinformatics/btm011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkataraman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">R/qtlbim: QTL with Bayesian Interval Mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm011%26rft_id%253Dinfo%253Apmid%252F17237038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMy81LzY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvNi8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-web" id="cit-4.6.1027.113"><div class="cit-metadata unstructured">Yandell, B. S., and Y. Nengjun, with contributions from T. Mehta, S. Banerjee, D. Shriner, <em>et al</em>., 2012 qtlbim: QTL Bayesian Interval Mapping. R package version 2.0.5. <a href="http://CRAN.R-project.org/package=qtlbim">http://CRAN.R-project.org/package=qtlbim</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.114" data-doi="10.1038/sj.hdy.6801074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Advances in Bayesian multiple quantitative trait loci mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DYi%26rft.auinit1%253DN.%26rft.volume%253D100%26rft.issue%253D3%26rft.spage%253D240%26rft.epage%253D252%26rft.atitle%253DAdvances%2Bin%2BBayesian%2Bmultiple%2Bquantitative%2Btrait%2Bloci%2Bmapping%2Bin%2Bexperimental%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6801074%26rft_id%253Dinfo%253Apmid%252F17987056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6801074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17987056&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.115" data-doi="10.1534/genetics.107.071365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomp</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An efficient Bayesian model selection approach for interacting quantitative trait loci models with many effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1865</span>–<span class="cit-lpage">1877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071365%26rft_id%253Dinfo%253Apmid%252F17483424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE4NjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.116" data-doi="10.1093/ilar.47.2.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshiki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriwaki</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mouse phenome research: implications of genetic background.</span> <abbr class="cit-jnl-abbrev">ILAR J.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DILAR%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Filar.47.2.94%26rft_id%253Dinfo%253Apmid%252F16547366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImlsYXJqb3VybmFsIjtzOjU6InJlc2lkIjtzOjc6IjQ3LzIvOTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzYvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference  in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.1027.117" data-doi="10.1146/annurev.phyto.34.1.479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">N. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">QTL mapping and quantitative disease resistance in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DYoung%26rft.auinit1%253DN.%2BD.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D479%26rft.epage%253D501%26rft.atitle%253DQTL%2BMAPPING%2BAND%2BQUANTITATIVE%2BDISEASE%2BRESISTANCE%2BIN%2BPLANTS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.34.1.479%26rft_id%253Dinfo%253Apmid%252F15012553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.34.1.479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012553&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VG88500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/6/1027.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/6/1011" title="Sex-Determination System in the Diploid Yeast Zygosaccharomyces sapae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/6/1047" title="In Aspergillus nidulans the Suppressors suaA and suaC Code for Release Factors eRF1 and eRF3 and suaD Codes for a Glutamine tRNA" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/6"><span>Volume 4 Issue 6, June 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/6.cover-source.jpg?itok=I-KHzzO2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/6/1027.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NFDZPSuVWwZtY8GDU5JcaRLF6eUpO2Pe7oaNmm7SHcY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/6/1027" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/6/1027" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/6/1027" class="active" data-icon-position="" data-hide-link-title="0">Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459902" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cVJE1jSjRl9Pd6cZIkN-4UBSd996OYYvtLXR57-jKuY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/6/1027.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/6/1027" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZsrsLF8uZ_Wchb8LM4UMBZHvlQTEu0P65AoDZSBygAY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/6/1027" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/6/1027.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459902" id="node459902--2" data-pisa="ggg;4/6/1027" data-pisa-master="ggg;g3.114.010298" data-apath="/ggg/4/6/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Réka Howard</span>, <span class="highwire-citation-author" data-delta="1">Alicia L. Carriquiry</span> and <span class="highwire-citation-author" data-delta="2">William D. Beavis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1046;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010298 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459902--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:howardr@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">howardr@iastate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alicia L. Carriquiry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BL.%2BCarriquiry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carriquiry%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlicia%2BL.%2BCarriquiry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William D. Beavis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBeavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beavis%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBeavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/6/1027" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/6/1027&amp;title=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/6/1027&amp;title=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/6/1027&amp;title=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/6/1027&amp;text=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/6/1027&amp;title=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/6/1027&amp;t=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/6/1027&amp;title=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/6/1027&amp;title=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459902/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459902" id="node459902--3" data-pisa="ggg;4/6/1027" data-pisa-master="ggg;g3.114.010298" data-apath="/ggg/4/6/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Parametric and Nonparametric Statistical Methods for Genomic Selection of Traits with Additive and Epistatic Genetic Architectures</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Réka Howard</span>, <span class="highwire-citation-author" data-delta="1">Alicia L. Carriquiry</span> and <span class="highwire-citation-author" data-delta="2">William D. Beavis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1046;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010298 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459902--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:howardr@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">howardr@iastate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alicia L. Carriquiry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BL.%2BCarriquiry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carriquiry%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlicia%2BL.%2BCarriquiry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William D. Beavis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBeavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beavis%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBeavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1027&amp;rft.epage=1046&amp;rft.atitle=Parametric%20and%20Nonparametric%20Statistical%20Methods%20for%20Genomic%20Selection%20of%20Traits%20with%20Additive%20and%20Epistatic%20Genetic%20Architectures&amp;rft.volume=4&amp;rft.issue=6&amp;rft.date=2014-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Howard%2C+R%C3%A9ka&amp;rft.au=Carriquiry%2C+Alicia+L.&amp;rft.au=Beavis%2C+William+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459902/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459902/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459902/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459902/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459902/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459902/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459902/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459902/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459902/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459902/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459902/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459902/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/6/1027" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/6/1027.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/6/1027.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/6/1027.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/6/1027.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
