<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/6/993" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/6/993.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/6/993.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/6/993.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/6/993.atom" />
<meta name="HW.pisa" content="ggg;4/6/993" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers" />
<meta name="DC.Identifier" content="10.1534/g3.113.009738" />
<meta name="DC.Date" content="2014-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Zhang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.114.011692" />
<meta name="DC.Relation" content="10.1534/g3.114.011692" />
<meta name="DC.Description" content="Plants lack a germ line; consequently, during reproduction adult somatic cells within flowers must switch from mitotic proliferation to meiosis. In maize ( Zea mays L.) anthers, hypoxic conditions in the developing tassel trigger pre-meiotic competence in the column of pluripotent progenitor cells in the center of anther lobes, and within 24 hr these newly specified germinal cells have patterned their surrounding neighbors to differentiate as the first somatic niche cells. Transcriptomes were analyzed by microarray hybridization in carefully staged whole anthers during initial specification events, after the separation of germinal and somatic lineages, during the subsequent rapid mitotic proliferation phase, and during final pre-meiotic germinal and somatic cell differentiation. Maize anthers exhibit a highly complex transcriptome constituting nearly three-quarters of annotated maize genes, and expression patterns are dynamic. Laser microdissection was applied to begin assigning transcripts to tissue and cell types and for comparison to transcriptomes of mutants defective in cell fate specification. Whole anther proteomes were analyzed at three developmental stages by mass spectrometric peptide sequencing using size-fractionated proteins to evaluate the timing of protein accumulation relative to transcript abundance. New insights include early and sustained expression of meiosis-associated genes (77.5% of well-annotated meiosis genes are constitutively active in 0.15 mm anthers), an extremely large change in transcript abundances and types a few days before meiosis (including a class of 1340 transcripts absent specifically at 0.4 mm), and the relative disparity between transcript abundance and protein abundance at any one developmental stage (based on 1303 protein-to-transcript comparisons)." />
<meta name="DC.Contributor" content="Han Zhang" />
<meta name="DC.Contributor" content="Rachel L. Egger" />
<meta name="DC.Contributor" content="Timothy Kelliher" />
<meta name="DC.Contributor" content="Darren Morrow" />
<meta name="DC.Contributor" content="John Fernandes" />
<meta name="DC.Contributor" content="Guo-Ling Nan" />
<meta name="DC.Contributor" content="Virginia Walbot" />
<meta name="article:published_time" content="2014-06-01" />
<meta name="article:section" content="Genetics of Sex" />
<meta name="citation_title" content="Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Plants lack a germ line; consequently, during reproduction adult somatic cells within flowers must switch from mitotic proliferation to meiosis. In maize (&lt;i&gt;Zea mays&lt;/i&gt; L.) anthers, hypoxic conditions in the developing tassel trigger pre-meiotic competence in the column of pluripotent progenitor cells in the center of anther lobes, and within 24 hr these newly specified germinal cells have patterned their surrounding neighbors to differentiate as the first somatic niche cells. Transcriptomes were analyzed by microarray hybridization in carefully staged whole anthers during initial specification events, after the separation of germinal and somatic lineages, during the subsequent rapid mitotic proliferation phase, and during final pre-meiotic germinal and somatic cell differentiation. Maize anthers exhibit a highly complex transcriptome constituting nearly three-quarters of annotated maize genes, and expression patterns are dynamic. Laser microdissection was applied to begin assigning transcripts to tissue and cell types and for comparison to transcriptomes of mutants defective in cell fate specification. Whole anther proteomes were analyzed at three developmental stages by mass spectrometric peptide sequencing using size-fractionated proteins to evaluate the timing of protein accumulation relative to transcript abundance. New insights include early and sustained expression of meiosis-associated genes (77.5% of well-annotated meiosis genes are constitutively active in 0.15 mm anthers), an extremely large change in transcript abundances and types a few days before meiosis (including a class of 1340 transcripts absent specifically at 0.4 mm), and the relative disparity between transcript abundance and protein abundance at any one developmental stage (based on 1303 protein-to-transcript comparisons).&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/06/01" />
<meta name="citation_mjid" content="ggg;4/6/993" />
<meta name="citation_id" content="4/6/993" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/6/993" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/6/993.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/6/993.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/6/993.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009738" />
<meta name="citation_pmid" content="24939185" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genetics of Sex" />
<meta name="citation_firstpage" content="993" />
<meta name="citation_lastpage" content="1010" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Han Zhang" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author" content="Rachel L. Egger" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author_email" content="egger@stanford.edu" />
<meta name="citation_author" content="Timothy Kelliher" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author" content="Darren Morrow" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author" content="John Fernandes" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author" content="Guo-Ling Nan" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author" content="Virginia Walbot" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. F. Acosta;citation_author=H. Laparra;citation_author=S. P. Romero;citation_author=E. Schmelz;citation_author=M. Hamberg;citation_title=tasselseed1 Is a Lipoxygenase Affecting Jasmonic Acid Signaling in Sex Determination of Maize;citation_pages=262-265;citation_volume=323;citation_year=2009;citation_issue=5911;citation_pmid=19131630;citation_doi=10.1126/science.1164645" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. Aya;citation_author=G. Suzuki;citation_author=K. Suwabe;citation_author=T. Hobo;citation_author=H. Takahashi;citation_title=Comprehensive network analysis of anther-expressed genes in rice by the combination of 33 laser microdissection and 143 spatiotemporal microarrays.;citation_pages=e26162-e26162;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=22046259;citation_doi=10.1371/journal.pone.0026162" />
<meta name="citation_reference" content="citation_title=Embryology of Flowering Plants: Terminology and Concepts;citation_author=T. B. Batygina;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=The International journal of developmental biology;citation_journal_abbrev=Int J Dev Biol;citation_author=L. C. Boavida;citation_author=J. D. Becker;citation_author=J. A. Feijo;citation_title=The making of gametes in higher plants.;citation_pages=595-614;citation_volume=49;citation_year=2005;citation_issue=5-6;citation_pmid=16096968;citation_doi=10.1387/ijdb.052019lb" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. Bolduc;citation_author=A. Yilmaz;citation_author=M. K. Mejia-Guerra;citation_author=K. Morohashi;citation_author=D. O’Connor;citation_title=Unraveling the KNOTTED1 regulatory network in maize meristems;citation_pages=1685-1690;citation_volume=26;citation_year=2012;citation_issue=15;citation_pmid=22855831;citation_doi=10.1101/gad.193433.112" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. P. Calarco;citation_author=F. Borges;citation_author=M. T. A. Donoghue;citation_author=F. Van Ex;citation_author=P. E. Jullien;citation_title=Reprogramming of DNA methylation in pollen guides epigenetic inheritance via small RNA.;citation_pages=194-205;citation_volume=151;citation_year=2012;citation_issue=1;citation_pmid=23000270;citation_doi=10.1016/j.cell.2012.09.001" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=P. Casati;citation_author=X. Zhang;citation_author=A. L. Burlingame;citation_author=V. Walbot;citation_title=Analysis of leaf proteome after UV-B irradiation in maize lines differing in sensitivity.;citation_volume=4;citation_year=2005;citation_pmid=16043824;citation_doi=10.1074/mcp.M500173-MCP200" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=F. Couteau;citation_author=F. Belzile;citation_author=C. Horlow;citation_author=O. Grandjean;citation_author=D. Vezon;citation_title=Random Chromosome Segregation without Meiotic Arrest in Both Male and Female Meiocytes of a dmc1 Mutant of Arabidopsis;citation_pages=1623-1634;citation_volume=11;citation_year=1999;citation_issue=9;citation_pmid=10488231;citation_doi=10.1105/tpc.11.9.1623" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. Depuydt;citation_author=C. S. Hardtke;citation_title=Hormone signalling crosstalk in plant growth regulation.;citation_pages=R365-R373;citation_volume=21;citation_year=2011;citation_issue=9;citation_pmid=21549959;citation_doi=10.1016/j.cub.2011.03.013" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=P. Deveshwar;citation_author=W. D. Bovill;citation_author=R. Sharma;citation_author=J. A. Able;citation_author=S. Kapoor;citation_title=Analysis of anther transcriptomes to identify genes contributing to meiosis and male gametophyte development in rice.;citation_pages=78-78;citation_volume=11;citation_year=2011;citation_pmid=21554676;citation_doi=10.1186/1471-2229-11-78" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=C. G. Extavour;citation_author=M. Akam;citation_title=Mechanisms of germ cell specification across the metazoans: epigenesis and preformation;citation_pages=5869-5884;citation_volume=130;citation_year=2003;citation_issue=24;citation_pmid=14597570;citation_doi=10.1242/dev.00804" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=X. Feng;citation_author=H. G. Dickinson;citation_title=Packaging the male germline in plants.;citation_pages=503-510;citation_volume=23;citation_year=2007;citation_issue=10;citation_pmid=17825943;citation_doi=10.1016/j.tig.2007.08.005" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=X. Feng;citation_author=D. Zilberman;citation_author=H. Dickinson;citation_title=A conversation across generations: soma-germ cell crosstalk in plants.;citation_pages=215-225;citation_volume=24;citation_year=2013;citation_pmid=23410937;citation_doi=10.1016/j.devcel.2013.01.014" />
<meta name="citation_reference" content="citation_title=The Development of Drosophila melanogaster;citation_title=Spermatogenesis;citation_author=M. T. Fuller;citation_pages=71-147;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=R. B. Goldberg;citation_author=T. P. Beals;citation_author=P. M. Sanders;citation_title=Anther Development: Basic Principles and Practical Applications;citation_pages=1217-1229;citation_volume=5;citation_year=1993;citation_issue=10;citation_pmid=8281038;citation_doi=10.1105/tpc.5.10.1217" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=I. N. Golubovskaya;citation_author=O. Hamant;citation_author=L. Timofejeva;citation_author=R. C. Wang;citation_author=D. Braun;citation_title=Alleles of afd1 dissect REC8 functions during meiotic prophase I.;citation_volume=119;citation_year=2006;citation_pmid=16868028;citation_doi=10.1242/jcs.03054" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=T. Hobo;citation_author=K. Suwabe;citation_author=K. Aya;citation_author=G. Suzuki;citation_author=K. Yano;citation_title=Various spatiotemporal expression profiles of anther-expressed genes in rice.;citation_volume=49;citation_year=2008;citation_pmid=18776202;citation_doi=10.1093/pcp/pcn128" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Honys;citation_author=D. Twell;citation_title=Transcriptome analysis of haploid male gametophyte development in Arabidopsis.;citation_pages=R85-R85;citation_volume=5;citation_year=2004;citation_issue=11;citation_pmid=15535861;citation_doi=10.1186/gb-2004-5-11-r85" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. D. Huang;citation_author=F. J. Wei;citation_author=C. C. Wu;citation_author=Y. I. Hsing;citation_author=A. H. Huang;citation_title=Analysis of advanced rice anther transcriptomes reveal global tapetum secretory functions and potential proteins for lipid exine formation.;citation_volume=149;citation_year=2009;citation_pmid=19091874;citation_doi=10.1104/pp.108.131128" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=D. Jackson;citation_author=B. Veit;citation_author=S. Hake;citation_title=Expression of maize KNOTTED1 related homeobox genes in the shoot apical meristem predicts patterns of morphogenesis in the vegetative shoot;citation_pages=405-413;citation_volume=120;citation_year=1994;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=Y. Jiang;citation_author=B. Zeng;citation_author=H. Zhao;citation_author=M. Zhang;citation_author=S. Xie;citation_title=Genome-wide transcription factor gene prediction and their expressional tissue-specificities in maize(F).;citation_pages=616-630;citation_volume=54;citation_year=2012;citation_issue=9;citation_pmid=22862992;citation_doi=10.1111/j.1744-7909.2012.01149.x" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=T. Kelliher;citation_author=V. Walbot;citation_title=Emergence and patterning of the five cell types of the Zea mays anther locule.;citation_pages=32-49;citation_volume=350;citation_year=2011;citation_issue=1;citation_pmid=21070762;citation_doi=10.1016/j.ydbio.2010.11.005" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Kelliher;citation_author=V. Walbot;citation_title=Hypoxia Triggers Meiotic Fate Acquisition in Maize;citation_pages=345-348;citation_volume=337;citation_year=2012;citation_issue=6092;citation_pmid=22822150;citation_doi=10.1126/science.1220080" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=T. Kelliher;citation_author=V. Walbot;citation_title=Maize germinal cell initials accommodate hypoxia and precociously express meiotic genes.;citation_pages=639-652;citation_volume=77;citation_year=2014;citation_pmid=24387628;citation_doi=10.1111/tpj.12414" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=M. Levin;citation_author=T. Hashimshony;citation_author=F. Wagner;citation_author=I. Yanai;citation_title=Developmental milestones punctuate gene expression in the Caenorhabditis embryo.;citation_pages=1101-1108;citation_volume=22;citation_year=2012;citation_issue=5;citation_pmid=22560298;citation_doi=10.1016/j.devcel.2012.04.004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Li;citation_author=L. C. Harper;citation_author=I. Golubovskaya;citation_author=C. R. Wang;citation_author=D. Weber;citation_title=Functional analysis of maize RAD51 in meiosis and double-strand break repair.;citation_pages=1469-1482;citation_volume=175;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Lin;citation_author=H. Kong;citation_author=M. Nei;citation_author=H. Ma;citation_title=Origins and evolution of the recA/RAD51 gene family: Evidence for ancient gene duplication and endosymbiotic gene transfer.;citation_volume=103;citation_year=2006;citation_pmid=16798872;citation_doi=10.1073/pnas.0604232103" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=X.-C. Lu;citation_author=H.-Q. Gong;citation_author=M.-L. Huang;citation_author=S.-L. Bai;citation_author=Y.-B. He;citation_title=Molecular analysis of early rice stamen development using organ-specific gene expression profiling.;citation_pages=845-861;citation_volume=61;citation_year=2006;citation_issue=6;citation_pmid=16927200;citation_doi=10.1007/s11103-006-0054-3" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=H. Ma;citation_title=Molecular genetic analyses of microsporogenesis and microgametogenesis in flowering plants.;citation_pages=393-434;citation_volume=56;citation_year=2005;citation_pmid=15862102;citation_doi=10.1146/annurev.arplant.55.031903.141717" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Ma;citation_author=D. J. Morrow;citation_author=J. Fernandes;citation_author=V. Walbot;citation_title=Comparative profiling of the sense and antisense transcriptome of maize lines.;citation_pages=R22-R22;citation_volume=7;citation_year=2006;citation_issue=3;citation_pmid=16542496;citation_doi=10.1186/gb-2006-7-3-r22" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=J. Ma;citation_author=D. Duncan;citation_author=D. J. Morrow;citation_author=J. Fernandes;citation_author=V. Walbot;citation_title=Transcriptome profiling of maize anthers using genetic ablation to analyze pre-meiotic and tapetal cell types.;citation_pages=637-648;citation_volume=50;citation_year=2007;citation_issue=4;citation_pmid=17419846;citation_doi=10.1111/j.1365-313X.2007.03074.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Ma;citation_author=D. S. Skibbe;citation_author=J. Fernandes;citation_author=V. Walbot;citation_title=Male reproductive development: gene expression profiling of maize anther and pollen ontogeny.;citation_pages=R181-R181;citation_volume=9;citation_year=2008;citation_issue=12;citation_pmid=19099579;citation_doi=10.1186/gb-2008-9-12-r181" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=E. Mutasa-Göttgens;citation_author=P. Hedden;citation_title=Gibberellin as a factor in floral regulatory networks.;citation_volume=60;citation_year=2009;citation_pmid=19264752;citation_doi=10.1093/jxb/erp040" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=G. L. Nan;citation_author=A. Ronceret;citation_author=R. C. Wang;citation_author=J. F. Fernandes;citation_author=W. Z. Cande;citation_title=Global transcriptome analysis of two ameiotic1 alleles in maize anthers: defining steps in meiotic entry and progression through prophase I.;citation_pages=120-120;citation_volume=11;citation_year=2011;citation_pmid=21867558;citation_doi=10.1186/1471-2229-11-120" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. P. Pawlowski;citation_author=I. N. Golubovskaya;citation_author=L. Timofejeva;citation_author=R. B. Meeley;citation_author=W. F. Sheridan;citation_title=Coordination of Meiotic Recombination, Pairing, and Synapsis by PHS1;citation_pages=89-92;citation_volume=303;citation_year=2004;citation_issue=5654;citation_pmid=14704428;citation_doi=10.1126/science.1091110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. P. Pawlowski;citation_author=R. W. Chung-Ju;citation_author=I. N. Golubovskaya;citation_author=J. M. Szymaniaka;citation_author=L. Shid;citation_title=Maize AMEIOTIC1 is essential for multiple early meiotic processes and likely required for the initiation of meiosis.;citation_volume=106;citation_year=2009;citation_pmid=19204280;citation_doi=10.1073/pnas.0810115106" />
<meta name="citation_reference" content="citation_journal_title=Sex. Plant Reprod.;citation_author=P. M. Sanders;citation_author=A. Q. Bui;citation_author=K. Weterings;citation_author=K. N. McIntire;citation_author=Y.-C. Hsu;citation_title=Anther developmental defects in Arabidopsis thaliana male-sterile mutants.;citation_pages=297-322;citation_volume=11;citation_year=1999;citation_doi=10.1007/s004970050158" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=U. Schiefthaler;citation_author=S. Balasubramanian;citation_author=P. Sieber;citation_author=D. Chevalier;citation_author=E. Wisman;citation_title=Molecular analysis of NOZZLE, a gene involved in pattern formation and early sporogenesis during sex organ development in Arabidopsis thaliana;citation_pages=11664-11669;citation_volume=96;citation_year=1999;citation_issue=20;citation_pmid=10500234;citation_doi=10.1073/pnas.96.20.11664" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. Schmidt;citation_author=S. E. Wuest;citation_author=K. Vijverberg;citation_author=C. Baroux;citation_author=D. Kleen;citation_title=Transcriptome analysis of the Arabidopsis megaspore mother cell uncovers the importance of RNA helicases for plant germline development.;citation_pages=e1001155-e1001155;citation_volume=9;citation_year=2011;citation_issue=9;citation_pmid=21949639;citation_doi=10.1371/journal.pbio.1001155" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. J. Scott;citation_author=M. Spielman;citation_author=H. G. Dickinson;citation_title=Stamen structure and function.;citation_volume=16;citation_year=2004;citation_pmid=15131249;citation_doi=10.1105/tpc.017012" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. S. Skibbe;citation_author=J. F. Fernandes;citation_author=K. F. Medzihradszky;citation_author=A. L. Burlingame;citation_author=V. Walbot;citation_title=Mutator transposon activity reprograms the transcriptomes and proteomes of developing maize anthers.;citation_pages=622-633;citation_volume=59;citation_year=2009;citation_issue=4;citation_pmid=19453454;citation_doi=10.1111/j.1365-313X.2009.03901.x" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=D. R. Smyth;citation_author=J. L. Bowman;citation_author=E. M. Meyerowitz;citation_title=Early Flower Development in Arabidopsis;citation_pages=755-767;citation_volume=2;citation_year=1990;citation_issue=8;citation_pmid=2152125;citation_doi=10.1105/tpc.2.8.755" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=S. Song;citation_author=T. Qi;citation_author=H. Huang;citation_author=D. Xie;citation_title=Regulation of stamen development by coordinated actions of jasmonate, auxin and gibberellin in Arabidopsis.;citation_volume=6;citation_year=2013;citation_pmid=23543439;citation_doi=10.1093/mp/sst054" />
<meta name="citation_reference" content="citation_title=The Development of Drosophila melanogaster;citation_title=Developmental genetics of oogenesis;citation_author=A. Spradling;citation_pages=1-70;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=X. Tang;citation_author=Z.-Y. Zhang;citation_author=W.-J. Zhang;citation_author=X.-M. Zhao;citation_author=X. Li;citation_title=Global gene profiling of laser-captured Pollen Mother Cells indicates molecular pathways and gene subfamilies involved in rice meiosis.;citation_volume=154;citation_year=2010;citation_pmid=20959420;citation_doi=10.1104/pp.110.161661" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=L. Timofejeva;citation_author=D. S. Skibbe;citation_author=S. Lee;citation_author=I. Golubovskaya;citation_author=R. Wang;citation_title=Cytological Characterization and Allelism Testing of Anther Developmental Mutants Identified in a Screen of Maize Male Sterile Lines;citation_pages=231-249;citation_volume=3;citation_year=2013;citation_issue=2;citation_pmid=23390600;citation_doi=10.1534/g3.112.004465" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3-new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=V. Vernoud;citation_author=G. Laigle;citation_author=F. Rozier;citation_author=R. B. Meeley;citation_author=P. Perez;citation_title=The HD-ZIP IV transcription factor OCL4 is necessary for trichome patterning and anther development in maize.;citation_pages=883-894;citation_volume=59;citation_year=2009;citation_issue=6;citation_pmid=19453441;citation_doi=10.1111/j.1365-313X.2009.03916.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Sex Plant Reprod;citation_author=V. Walbot;citation_author=D. S. Skibbe;citation_title=Maize host requirements for Ustilago maydis tumor induction.;citation_pages=1-13;citation_volume=23;citation_year=2010;citation_issue=1;citation_pmid=20165959;citation_doi=10.1007/s00497-009-0109-0" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. Wang;citation_author=J. A. Oses-Prieto;citation_author=K. H. Li;citation_author=J. F. Fernandes;citation_author=A. L. Burlingame;citation_title=The male sterile 8 mutation of maize disrupts the temporal progression of the transcriptome and results in the mis-regulation of metabolic functions.;citation_pages=939-951;citation_volume=63;citation_year=2010;citation_issue=6;citation_pmid=20626649;citation_doi=10.1111/j.1365-313X.2010.04294.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Wang;citation_author=K. Wang;citation_author=D. Tang;citation_author=C. Wei;citation_author=M. Li;citation_title=The central element protein ZEP1 of the synaptonemal complex regulates the number of crossovers during meiosis in rice.;citation_volume=22;citation_year=2010;citation_pmid=20154151;citation_doi=10.1105/tpc.109.070789" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=D. Wang;citation_author=C. M. Adams;citation_author=J. F. Fernandes;citation_author=R. L. Egger;citation_author=V. Walbot;citation_title=A low molecular weight proteome comparison of fertile and male sterile 8 anthers of Zea mays.;citation_pages=925-935;citation_volume=10;citation_year=2012;citation_issue=8;citation_pmid=22748129;citation_doi=10.1111/j.1467-7652.2012.00721.x" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=R. C.-J. Wang;citation_author=G. L. Nan;citation_author=T. Kelliher;citation_author=L. Timofejeva;citation_author=V. Vernoud;citation_title=Planar cell polarity controls directional Notch signaling in the Drosophila leg;citation_pages=2584-2593;citation_volume=139;citation_year=2012;citation_issue=14;citation_pmid=22736244;citation_doi=10.1242/dev.077446" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Wellmer;citation_author=J. L. Riechmann;citation_author=M. Alves-Ferreira;citation_author=E. M. Meyerowitz;citation_title=Genome-wide analysis of spatial gene expression in Arabidopsis flowers.;citation_volume=16;citation_year=2004;citation_pmid=15100403;citation_doi=10.1105/tpc.021741" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. J. Wijeratne;citation_author=W. Zhang;citation_author=Y. Sun;citation_author=W. Liu;citation_author=R. Albert;citation_title=Differential gene expression in Arabidopsis wild-type and mutant anthers: insights into anther cell differentiation and regulatory networks.;citation_pages=14-29;citation_volume=52;citation_year=2007;citation_issue=1;citation_pmid=17666023;citation_doi=10.1111/j.1365-313X.2007.03217.x" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=H. Yang;citation_author=P. Lu;citation_author=Y. Wang;citation_author=H. Ma;citation_title=The transcriptome landscape of Arabidopsis male meiocytes from high-throughput sequencing: the complexity and evolution of the meiotic process.;citation_pages=503-516;citation_volume=65;citation_year=2011;citation_issue=4;citation_pmid=21208307;citation_doi=10.1111/j.1365-313X.2010.04439.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Q. Ye;citation_author=W. Zhu;citation_author=L. Li;citation_author=S. Zhang;citation_author=Y. Yin;citation_title=Brassinosteroids control male fertility by regulating the expression of key genes involved in Arabidopsis anther and pollen development;citation_pages=6101-6105;citation_volume=107;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=H. Yu;citation_author=M. Wang;citation_author=D. Tang;citation_author=K. Wang;citation_author=F. Chen;citation_title=OsSPO11–1 is essential for both homologous chromosome pairing and crossover formation in rice.;citation_pages=625-636;citation_volume=119;citation_year=2010;citation_pmid=20625906;citation_doi=10.1007/s00412-010-0284-7" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=S. Zhao;citation_author=R. Fernald;citation_title=Comprehensive algorithm for quantitative real-time polymerase chain reaction.;citation_pages=1047-1064;citation_volume=12;citation_year=2005;citation_issue=8;citation_pmid=16241897;citation_doi=10.1089/cmb.2005.12.1047" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6.cover-source.jpg" />
<meta name="twitter:description" content="Plants lack a germ line; consequently, during reproduction adult somatic cells within flowers must switch from mitotic proliferation to meiosis. In maize ( Zea mays L.) anthers, hypoxic conditions in the developing tassel trigger pre-meiotic competence in the column of pluripotent progenitor cells in the center of anther lobes, and within 24 hr these newly specified germinal cells have patterned their surrounding neighbors to differentiate as the first somatic niche cells. Transcriptomes were analyzed by microarray hybridization in carefully staged whole anthers during initial specification events, after the separation of germinal and somatic lineages, during the subsequent rapid mitotic proliferation phase, and during final pre-meiotic germinal and somatic cell differentiation. Maize anthers exhibit a highly complex transcriptome constituting nearly three-quarters of annotated maize genes, and expression patterns are dynamic. Laser microdissection was applied to begin assigning transcripts to tissue and cell types and for comparison to transcriptomes of mutants defective in cell fate specification. Whole anther proteomes were analyzed at three developmental stages by mass spectrometric peptide sequencing using size-fractionated proteins to evaluate the timing of protein accumulation relative to transcript abundance. New insights include early and sustained expression of meiosis-associated genes (77.5% of well-annotated meiosis genes are constitutively active in 0.15 mm anthers), an extremely large change in transcript abundances and types a few days before meiosis (including a class of 1340 transcripts absent specifically at 0.4 mm), and the relative disparity between transcript abundance and protein abundance at any one developmental stage (based on 1303 protein-to-transcript comparisons)." />
<meta name="og-title" property="og:title" content="Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/6/993" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Plants lack a germ line; consequently, during reproduction adult somatic cells within flowers must switch from mitotic proliferation to meiosis. In maize ( Zea mays L.) anthers, hypoxic conditions in the developing tassel trigger pre-meiotic competence in the column of pluripotent progenitor cells in the center of anther lobes, and within 24 hr these newly specified germinal cells have patterned their surrounding neighbors to differentiate as the first somatic niche cells. Transcriptomes were analyzed by microarray hybridization in carefully staged whole anthers during initial specification events, after the separation of germinal and somatic lineages, during the subsequent rapid mitotic proliferation phase, and during final pre-meiotic germinal and somatic cell differentiation. Maize anthers exhibit a highly complex transcriptome constituting nearly three-quarters of annotated maize genes, and expression patterns are dynamic. Laser microdissection was applied to begin assigning transcripts to tissue and cell types and for comparison to transcriptomes of mutants defective in cell fate specification. Whole anther proteomes were analyzed at three developmental stages by mass spectrometric peptide sequencing using size-fractionated proteins to evaluate the timing of protein accumulation relative to transcript abundance. New insights include early and sustained expression of meiosis-associated genes (77.5% of well-annotated meiosis genes are constitutively active in 0.15 mm anthers), an extremely large change in transcript abundances and types a few days before meiosis (including a class of 1340 transcripts absent specifically at 0.4 mm), and the relative disparity between transcript abundance and protein abundance at any one developmental stage (based on 1303 protein-to-transcript comparisons)." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/6.cover-source.jpg" />
    <title>Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"sZb-6tTMO9ZUc-ekLsLf-vI8uKu-dcus8UXV-6s4gpo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459892","apath":"\/ggg\/4\/6\/993.atom","pisa":"ggg;4\/6\/993","ac":{"\/ggg\/4\/6\/993.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/6\/993.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/6\/993"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/6\/993"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/6\/993":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224xWiq2HMvHD\\\\\\\/3cdkdckRzKfURBtwskuKiQnafdMbQLwmgMs6+gKEBnsgHHX6VfcEx8owgyMWVrF4aLMal8xOBBw9TRJwH7olJ6tkPTLXapTR25lo6zmzLKMvkgl+S3BakBKdjtwF8bMGC56wpMG5LNrHHm49bvWOfH5VZM1AC0Z4B\\\\\\\/fiEi7nTZy\\\\\\\/X1VaFs2UnAWCaHduKXoL2VrjtKJTjnXj6cELzhS2AQk1VbDMRzNF7D1SRou0Or5tvcPZcLroefgkrILTPO05UhMOS2VSqLZdspSMHbVN7gnVr5mUZ\\\\\\\/GauJvtlf72TIMG4NSmUlLAfPQUHN+LmANw163zuC\\\\\\\/yNajYmVxMax6aiT+U7BuaU4cfdWR4OiGi5bJZKvVVnEyOhbqSjbiKjNcPmzSGHr12\\\\\\\/yAoz7WWZRHGAbmFE4QiFRLd+bQCBjHxRj1\\\\\\\/fqgk70953Pk9dBz1+1aSf+6O0V5sgTu1x6SM3s\\\\\\\/oH\\\\\\\/nCXJL4vxN5REurEWFg+H7kwiDLQg2FLxqXO5b9TjwNCYrRBIi0awdRbgsZdriglBepaNrKQtHKlrV1lTV346NC\\\\\\\/O3zUEZD4HWoJX6CUlxDR2c8Qlk6hsNCJo9Z3\\\\\\\/5oVeFsQlu0RvC92ZWt3m11s\\\\\\\/2jJpIgoqCsFHM0uoQTFccM9GLRG7wccq38pQLhTnbE2tAJ+OhsbBD7eCcQ6LAcFv4L8\\\\\\\/7wx7sVXdsdbYM+sd4w4eIGWLn5cw\\\\\\\/RJ+FpUHOdFNx56jNxz40Ixj2bUxZB2aKctHtXPWvyU\\\\\\\/iZTyX522TexlRn7mg+0p8HLg9JyIWMPygGdOdYcZZkn2je4WMYTfTW7PkuL35WroskIp5X\\\\\\\/NVPNzsmeuHkhyVxhM8VGQx8m5QHPHyCoeoBW5ogxS7edCm7zoa0v+r+7m+28ucd2oB0J2DAqcOXGoH35E218jc6MGTr4MPqqjCtoCVdLKwBJOnWkfqtH5R6G72dicBkD\\\\\\\/bYDb7XISPac3x+JYOYI7t+WsaQFKlV1mje3nh6xDJnm\\\\\\\/vfm5vZmQ8quMjMesUEmUziHcWrublBKnYVaaqgOd2oAk1HBc1MWBzKfkK0AnR3sXFEKsDjiccjLlaqVyHXVpReh9O6i6\\\\\\\/Qd233u3Q1pcM9xUvr\\\\\\\/5xCAit\\\\\\\/Ib1YAcCl03U7pL83s7+wAO7zlAeSl75w1qgN2v0WiTJQlyDkH515rxppt7qDc4vunxndGalxDidloIBlhitgA7udGA29XR190aL060\\\\\\\/ooboc3oLq9cO7qJuL18ojdZEGr1oKuhFj4xwBR+OBrzhW\\\u0022,\\\u0022iv\\\u0022:\\\u0022spjoBLvtI6YnI6kxOpRalw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224f01a451a23ec04b1207ec92dda16a0fba348923c0142a91cad2b84b25019121\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qaGNl4rduxhTk+Nt5iub2qsQXgGtRaUrOvT0K1+FzHQKTPrIBjK+YPocrXGOsLxwZ6fqAiTDJWviyxyhKiV0cpqEgPruM5r4hhcR8fOoZn62wbb\\\\\\\/ZFu5WsJ1SHaxXp1nIY3Wsc56XK\\\\\\\/RS758aa0pVQZrQE4\\\\\\\/y7f7JplbAJlb+PjZOxhIXu26VL7iHfgnM5Zc5AiBuAsf1OKvCnjGzbcJ95M1USYON4V1GBsKvfAYoLJ2nlgduY3cVuwDT9mkN3YRU41zcbDvaztp9HPPqNLNo1nnWtkhzUmhndpYCGIj3rLNWoZo5sdbNnv04BdGAqYIpak\\\\\\\/qkDHx2W561Mh7VNyjvMXtkLgWV+ieXwUDVOM6XjMU9KK56\\\\\\\/U6gMoeXAs5aCJugrMk9Cqr\\\\\\\/kkav6ix9RGQHccqT8MTGbUdluQp1KaQ3ElkrHMEnqZRxw75mzom+vBDm14B0AnuRfGO7syb7FSX4IdL33FrDQ9fTYpfnD46PLh+zyDi5H3nkEIblhP8ovyDukonPd5CphCU5RqieiRzqO6GciQ3K8D\\\\\\\/SgM4XPkBViZ\\\\\\\/TIzWsavlzHm453Ca0a2T+++HXhMCYHMhWBVyiKPONeW2b4kDW30ZQH4PxyhukoXlU6d8r8iQhgTQreSrvEZKysrBBc2DcsD0ZEOwn4mPzfNci1j5yFml\\\\\\\/UQdN9xaMjeb8PL\\\\\\\/lKd1AD+8pMv3yta8OlzXG12xbefRZNAx2hQPrzw9z1WB993okRy93bIAAn2yYXBJtQrR4j7bpRgJKKWnyafw+rbgfh3O3C9QWz4rIip5lP1+q+1MozVXgPdLSW3\\\\\\\/Jan5tczc9BcgxqUTwVkziQKi8lqMSk4oVhzA6m3YMh6VKkvWLVdcvYIIQ8zN\\\\\\\/eFrMF\\\\\\\/Tm1XW0OuRejtsWEKvA2J1wuZZtj+VyuAQv0WSVhrXNgScw2qvskNj65gSBaUzDh1Xj8HFW4YZLqeJeT3rrItt2UEt69JUzLATt3F+gtNDw627tzVqjBFvzGHryXHi9PJAgoe6I1VbSUDURKN3iclzMdS2dMSzahP\\\\\\\/cLD+MCjm62Ydd8oBa+d9S4ok2Sp85SnTwYcv3QyN+avswCcZdmq3+2uSM\\\\\\\/K\\\\\\\/nvboYUMQM7m1penmZa0h+SdGu78Q1Yi8Wj\\\\\\\/RXxcnVPTfqrWgdh16E8FVNBUPqwBeDl4st2aV6PszDgngodsoNmBGiFGtkifFeGUlN1g+SDYtgwYadTZqqEUNpXxycv8rjUktabQlwZYBrK3Y84TX2hFcQn+zQb\\\\\\\/UA6btGQLAeF9uqX6wiJovbLwjGrSDrrmHQIscBbUphQwtTcDcNch9Y2nkvsSmrvR2auWWTBxlHkOrR7Gdi5aheoohO2pREdrRs6RaGaAfw8OyAQueqX\\\\\\\/xM63i2hXmiguYuuGwMoONjaUaAlrV1XULBo72zT3eTZh9uCyHzJoatg1BxItnNLh6aVVu8wl\\\\\\\/ZPOGjGLArD+eImPj4GrADQiwzPlrSWLxqJMsIGE1sE+y5iHoz7FnlRzB+10s7SZawAcM\\\\\\\/YxolPT1zIHndP6rfzZyEpgqr9rdRhheiHl4GS8RBd4A6V319gHOKxxPaMGPdD1W6CYUROdrZ05v00kk7kTlyViRjjlUzk2RGCFHncNXl9vKgmr2j1owV6KuR5jNCw0OsbF8NqrTmDsxzQv0\\\\\\\/hE1kjzeBImC0XHsoifrNFO63PqQLJUoWJia2bZhNqkFb3zZTm0HeCqCVLmm5LQlYHN2moG2pDLSL0UmYpsuBDztxoOCOxRkk3HTpKw\\\\\\\/pQJxuL\\\\\\\/c6+gsg0VUG7HWCBMEInYU1g8TXQcfsBRxvNLrVm9qyfpW+3pe09BJVRuLZAo56xkIjEe1JP4gJPMn7jF8ZpYNVD7lzAhfJWo6ANZ7BZi2dkmtBZqIB1MiRFeYBKuhfMMA+GjQbT8OcPUmQgtjS7DWg6ENcjiXDKCVjQu\\\\\\\/FWI2oC14AOYXXKCDijzl2ayA45\\\\\\\/6M5RLcaeWUjUcY+8go5UKXLSlTCV3H5rd3LRqeYEEkVCxkSQ7rpppsNyhm2nPauiTJ8Ihb3N6jTKLz3Dm23zHNi+32JUGqCi\\\\\\\/TgBauFtiCgXRptKU47gI3BjQtp91TBN\\\\\\\/uJhO6UBXlOqI7t4E0Hrxs4EFjirCS5Y8lsld+XnopyKFxoRNeTLSqcjf3Zw60ZF+C1d4aV8tahLpBVeHWtM3oHlMCYfN8ptHfCKa\\\\\\\/kFiJtQonK7lWwmEOng2dF5y9sVJWKmfS\\\\\\\/LflDg2FjOMhPudkrLUp+vJyvl4MdUUJDRu3CRz9ebQrV5RKuN6ozDt6TTOxtQXcuSG8XomhtYFXfNoA9KKVCP9IpSrb8NQ94nLaXYwRjStPX95TH35dtKbtQKRvhOVxJNO\\\\\\\/RagWZi7BhDadCDRtYFfFBbGMo6X5FV7OX87kIHRUc0utm1jJ1BMowe9q3zGjkDRmDdIxeTi1e4gsdpZsTZG3UmSoglqduueGcRNh9+mSzb1GX9w+5hpRXOlXht5QB86faFjKflGdz2396tPFN22mOxXz4ohLgATjQ8QpYkFOPgPeegzr2l5ljZH1qMShw0WkFmg0PWDqG1t6Rb0uKOqMtf2TGPj33+v0qsd5LwddJcKdUaQL7wWRPwIq4\\\\\\\/OmoRWZA5PFe6k0B4kokJ0KFH4qzn9USuMPqOPwTJ1UyRZ7H3pQPujerEMvaOX2yuqO8rCnLhYtGWLefS0BqHlUc1knXM5LIT4Mo5iccbgg8ZoNXM8mVdNTH95Y12fUVKMcbuwdIn\\\\\\\/DWonKHi58zAvH6MDrOomAu3H886YyFMlx5OCqt0ejhQBXCmKP0AKPxeHkX97+fTot9QWybgYSkvabo6FypuR0myBKK+F+vz5inQ5VTqLDKdp3QX4Gs06XWdY1307GmLb0L7TLsAKy4rij0iUyGHP5h9Uoyvt1WTgnlyIttrswKehDjFNCAaNCGcxkzYGXAK1Rk\\\\\\\/OuZjTkTBR2XXgoiu93ipgULTTPJu7Pp4BL1jPRxO+1lhnTnOjN\\\\\\\/k6ZSiqQeLL7EEphEzqfCEgRTC5iWtVzpZkiVrBRp1hq69yG7Re1SudCVw6rLf11dyd4fy8srPs6Ml2S+8wx5QZuvnNCiT5fJ\\\\\\\/xq2pZUD5CGf\\\\\\\/0ptTVxva2Dbs66fOPGepV1UwpI\\\\\\\/0L66Gir\\\\\\\/yoW+3nY+dNDMiKFCvPj9VYI6P6efutVqr7nIEKnMsEC+VHuTYIvlQVChA2BOoRCFjBlji6e6EqP\\\\\\\/5mlKws5DO2O+AXAlpnq3N9P27YOXUwvlQFEsKlydXsAFulF11NN6bfwxYM69z8G06bBDnF\\\\\\\/ch\\\\\\\/nr89m\\\\\\\/eFnO0UpS9b1RYkjzraKgrg\\\\\\\/yWtSGLz3vlkca4NQ6jl1jCrWRXPWwwwzogjb2hSvPo2fsMID50IyjZTUF9G0SKMLd9UPx75uMRK3YYseJCfwpc5A5wtH0W+SfOr\\\\\\\/KrYdjitzhdf8zH3QyZ67ApqLvpufGeYJsVewKTsoKgpl\\\\\\\/KtSRwxUVFrjv6od38urDSCNFCc2zh0Cy0YVliRCgcIFWMMn4lQrZGBGbjPEN+u0hqEjiXULclOjZaci80wtIetUK2JNh5CMqnZ2mXNOZa3SPW\\\\\\\/2ZmQnywo73ADFzaHuihQ5L1MZHmeZzZdi+dhUOzvu2F7637Y1DFEvZTffEfdCDjoK3vDWWJrNJT8rV1QiHHEzVR5MmX2ni8n1vlDpNvNJi\\\\\\\/V1uZZrIp7fCAK7VVP3nnnaoYoY9yidi6Pe9yzkbxOTZNdeUJ1HekxmqSvUIHcvRWINJOPMS0tbFCQczzKBmbP7WPuBIGHDJRpZeaOd4dPGg+PnG4z7YwZ2rEMWapzF25r20o\\\\\\\/pgaBe8mDn6EKfEwu0NaYDOY3DAEd2J0Nx5TWxlBfrnmfWvRPm2JHqkWPiC+QLSiaTEwzHMBxBnlRFfhrNcnq2haoQ3PlOcNHyuyfPVRkPpNfMAuh0a+GFESNPUA+f+hx9iHUTgKgt1STd3G3OnstEoYXChplRE6zdHfovBzNFQ5UYEtwB\\\\\\\/CfnNZUzGd4J02eXbe8AnSM0YfA4GLDorgMZD6DvyIHrpQrWBNnPuDKSKyuGAoIjBi87dRfKNBKG4V2xQO38\\\\\\\/dHc3fSkZveipPEbiQ8ptAr1AKJiLpq\\\\\\\/0M4mX8tikm1mJf5fSAXFBu+7yEsIRs0Vo\\\\\\\/+7zmIFTBd\\\\\\\/6sO191H0T3z9tAbqGH4gYDekWegyFoelHDhJUasKb7ay4B3j4lyYICB1odkEK+1bKlfgVxCcdsRlxHCbAPEeZBxCkvnAXQxde81fh+PSar1JRnBpkH3\\\\\\\/RWwxAQiS+3J3p1TXYhawOPBomlHZobCl6Gw7XRJGaLN2hUA5BQYigP6L9UNrjBAVp3HxGbTDm2QELqY09DWBZf5lwzu6h9gNowEDRohVDYryzAABvvkAt6j\\\\\\\/r07o\\\\\\\/6LlDCXQ20nYCEYOYYkjjlGp4Df\\\\\\\/LvCAMGqNvFrA+MgMRHsOd3I4noH3o3QfW992zFF4lE7uRnHTtlF5hHRqJ5VDjvn2C7NSQeXxuUUgmhrG26fEde1XMQjZ28fSMZRGQsOlYp3M8uvrWT8ff0z1PEYpbMvD4qImXu4JU0YFp78qOfrMxXBvPwlGGNjevBfHsXtCMYSj9iOIrRJhdxClsNuTMCy0TC7glUbQvWrccQOTT6Kh+pn3XAj6SG5NlpKIm80jvXBWeSszXEPEN42Wg8OqVJtGZ7tQb1pP8c1hS52Y5O5O2tbtqufAcNHFQM5H0KXXa7gNrG3p9LQxsr5lkYuWAIC8vqH+nv1n0c0x7JydqM23eLrRc6YIwA5vYkwXa2fOx0XCLijoqi\\\\\\\/N\\\\\\\/otpWieOHj2zdfl2RHeCekQwAYIYyvpkI65PqeYSUefiTXjVb+aIZxpk0nn2L\\\\\\\/v5GiCu3gfur8AsvyEdUfJM38X5jMp+eN4RYPsDDOvjxxgY5wotmxVDdWntp8\\\\\\\/ZITa2MHebWexl0CZDqR+IM\\\\\\\/DJdZbPD28QrzYqIEIdKVaPI7E8RecrlH13cpM8b8hirDbMCWD6L7KYQEIOkWDfdd1+8S46O9nDFXWF70Dm0EjPP6qTxZ8Fzhn+ElUgbYVkUWhSsTB2fy762ZMLFzGcR2zbJI8shswVoiwRD7m5gUUWO21xMFETcQpP+6sm8K7HXmGGEomERhhkC8Ex9dYH9XeSc7ZLoNqLYhOKoIrWx7pa\\\\\\\/6y9eZITLrI+PdgBOXbS8hKb1bL2xNvSwoReQDSPzzY8pmI9rbRhxHgUCW+eCPIC451qUUx16p+j+6hodx8g95L1JPeHf7Gp6JePMdQp+5eo9cPRFgARcm5ECALzGv8UmMjeevCVHor3xns7Oi8PW7QejCZUlqwt6Xntc6P+hKfkfIYyxUGe37YsMZnU8le4mGWjxbimMsCH6QngKHLgljiJ6\\\\\\\/vd3yQuXtWMoqXp9+T9lOJM8kEx0EpFIRLbhECl92KCUJxOC3D1beqDkTSmA\\\\\\\/xzvrLcz2LzBD6hPvZZVYRrQfHog+aOj+OZ5myYv471deAbEHzLoyAcly0gv\\\\\\\/kL1qVVR2hYI\\\\\\\/B6iJyHklDfyXBaS+t2ars4PHmrpv2IR3EUInRSM76reP39nSZ7kkYdF7WhNaehPrRgl1OBwBGTZJuZDy72I\\\\\\\/VnYBXWF72U6C4ll4YEMhjZRb5X8kmXr1FcVm2OG4vuFh4Lk+t0GeC4COgn53E7VdNork3pBatcawx3uefBqAvS0Iaw09\\\\\\\/aKA6mSPd1RD7uXjUlat5xEtX6vr57wJHxNprTnpkl8x2O\\\\\\\/asB5Sda0oWbfmGGIB8MkNgXyFghz\\\\\\\/lapisgU9xnVAHiC5gczjNlEQSnEI44iZjv\\\\\\\/ztDXf7BeciRkOSi96FfPxjYlkVjHlG8lom296XiFL5xh81NNCpXnZPfhZTTg\\\\\\\/CNbLQCwwmen37egLkMn4ECIG3b5KE1JBxKh3xjla0My7q6Ew72rJu5oXm\\\\\\\/8CS3BmzAA2JY21s2DmV3K3sT7JEOPp8isW7tkSYsJmt7AgvzYyqvljtU5lXUbBf1MDobUY8B4VC\\\\\\\/b2Hy8T\\\\\\\/N7h67gUzrx1RQJsSm3OcvaeSH7Q5DsfdfU1ZvZagW06\\\\\\\/1OpgCXBfzlMhbsJlRq4HVIqbcreAx2TEUhPIhX\\\\\\\/LjV2H+ML+STXdLu2+e\\\\\\\/l0bhZtgSMldTXfLhyAf5zFdvSeLd9sGDQeKJF\\\\\\\/sfa2CpXPGp50UHjc75TNQHn6rr3Jtd4zQTF6NYBj2F06aIeIYy0vn0MoUc4lW0Opu8t6N72sicOjJXch+YZ15x2gmgu0p83HBBN7GJ9pQddIK\\\\\\\/hdvT9Fayzdit8ginVZqvzOMe7Y3Z8nRhvqqMmMBpAiVO8h92zWkDhxYyoalrxJEjI0toAgK37vVeOwwZ8oNznww0YbfsJ4Dot4rchbm\\\\\\\/j3tlmym+B+DPROQHS9vBZWWm6z\\\\\\\/3EdCSwfK\\\\\\\/VO9OgxkeDiGIcUzqECvq9OaObxJ0BUF661S5HzDf8\\\\\\\/h8+usKNRE6gOhygUKNxcYEUtDY9UK3eYFcLGqMq4FOGoueokmYl8lxKLMGwpQjsyrhv1yxdAo7O\\\\\\\/xnyKa1Vor7Cz8leaynDFklKRDCaDyYa49wt7UABM7XGELCMVwi8pB3\\\\\\\/FbF0mFhTxl0HGdymg3HjC23tOkdSfy5Vyu7ipg7uhSAEG0tRLAe+UZ2Ec4uKj+8dABDtIlaaUgAX4rTQBQTQmz7hOyox30BWLgvSlTj689zKtpRkRBqlBvqMFv2kVz0Bja8z3j2rcJgWdPiQa\\\\\\\/Z0lbrJufZ6ZLywSRHg1qxZpKxhT8yObN1imOc6J3xiuDGQ10SAwg0ZFzyk+KmPkKE67gYPeQdderD\\\\\\\/GcQtj5nLhWncgknAk8bMYEBvlnq26jOoJMvwBPELApP8KBJl8y1F\\\\\\\/Zp3rKxB2TwshqAbfF8aoEOfKYIQTHZKYdWBcV9DgT+BUkJgj43xvLMLO808onMvFnprg4eKhcV9ObgnT7OOpUzfzd6kCFYKPT9jtbkki83gksPcUCqJUunxDT7uNXbVYDiAmlJcAhtmMStXZJbArsnI58cv+S6\\\\\\\/bXCnrwefO68X70sY8ZWjcERd8fbmwnzBVCvFLxPtKQfDZ5sLfxHerbQh7r0thNVPBr9eiQLAgFugNLXCg3wHeMd03NjyWtGgy\\\\\\\/5R5UVKoQedBZkc0po+407mHVb\\\\\\\/boWvgYR1uhLouqu7Y8m7ovDXBiYc8RGVgYeGfsVVVyf+EYobeFBAh9ZWmm1JQ5wwhXSNK+QN62wJ5QclKdQR08X7wHyBnVAuAmatbI3wNMC6xEgra0eJJGKWRlEQYKlz\\\\\\\/QtqEy1DYU3nfmvBQo\\\\\\\/wjhW0facLUNeo48f\\\\\\\/UArGzGm\\\\\\\/eaJomtU0kTvY5EBw0YP1AH6Smn4jMyBN8HBpatn0n87D0GFxUe56kKAFH3WlbZEt0wAxm1MUAAbsMz5P+9g3LDKYlmLjeRZNgQ3dFt2MqcKoRalPrOrZuWBo0lB5EomfY2LDGx78rAk\\\\\\\/7VptT3musGiLoVuQIqC7U28e7AoVUjOcAwd50a77f5dRC+\\\\\\\/nwCf+9NRc0f7Pnj\\\\\\\/Y5nmAcgfI6dAk6UaORcg3dou\\\\\\\/aYalUH42I+\\\\\\\/+bEOiY2U2kHu96nhU34+Xj\\\\\\\/JNXk7L0sLzDiHMyjtVAjgGli8l9GJUOTEqDJ5Q5PwEyK7nIRCLzHUlTNejfsyHHOSigAduuPwqDZ+XkwFK5l9v+vQIdVEJUR119NDcGNSMeo5PtTkwDWn2uvu+Spi9wg6yjgEGtd6oDuwg5LqN+sUUFIB1rV6GNr7E2oINVwA\\\\\\\/lbEQVqhA1+0yJRvnVpQoJNp+HMPGRmmv\\\\\\\/ZXtoILQch1e6KN5IfpM7YvNPiEW7FmrXDCUPred16WcR2uJNKd2dbGkFCkeufFACrigQ7ci0yKsAqG8bdKlRGr5K0zZjGEvKd99uQOiK25ZlWamh9h+N0aoK8YWyQXzkr1Ip2awc9R6rFVAp72Cn2iCvMj\\\\\\\/8LnNPbZvqUWY3P4EEGwJCMBpIlMKOuiRbmNwA1ohm1EYSkSbnei+\\\\\\\/59Z2Q8crNUHHckN7XKazeEHI4U6L8stZCwO3epBxjf8c37O8Or0EQJ4m8zzVtOfxQVLmH\\\\\\\/49poPeLRDAXWaBKzIEYSwmGOKcOXkrSD6Y390f4+6At\\\\\\\/0dr60upreaBNnap4VPtWAoaPRNpAOabuy+fjl1LPHRLyqR4jcWVw74sxURV2iaJl\\\\\\\/lZ5MdshYxCBUw2flEoI\\\\\\\/BojI2BkxFlnyVl1eEbYM2LgwV3hJd6se+L4dzE\\\\\\\/CYV17XnZEYtCwP3yLhHFFIdZBgQPtKYSvq\\\\\\\/f8OZGxwsRKAmz9ccBAgvokEl+DOO1n18wiDL\\\\\\\/3gJqc24Vhc0C9GgqUeG4qAp4YzhdDmaUIqhHaGk2JiT89Np77C8jtcgejpgsKfkKnIKf0aDLsn691m54oCIEWwxIB65cVGr0JA1lB1YDih5Q\\\\\\\/QFpKp3pighlrU22O6Gkgd8kVnLcUoP9GTQ1BHF9I8+MeSz0yUPI7S6cxgGSHWwF\\\\\\\/DJrP0ArO3NoiRW2j1qXuPPyNaj8IOtjgRQiU8J9myX7mFDHVyJ4Z5\\\\\\\/uXx5gYVA38yFj01bC4RaWHAL9kfg4SlE53cFCb4ofnsnLHaWv+FTic0F9URPyeNrWvyE4J9gwBuzssCQYWherz7kAEh01iaClhJkScLvd\\\\\\\/itW3gD3JfYNZ+YTqdDtEkyCuslJZlr5a96pWCcMubhLgScJHSnIz8+yD48oDL8+SnDGJ8yvjbwY52anIYax2Gl3swdTZOerLBnj7vyILIv1rCkiJH5qwxy3J04vAnHaveLee15XE1RSC16hVuWQ1iN15vxPnS1j197yLwzN+oDi2hszz3i09\\\\\\\/je041iVr3cQGgD69KPGFpk4IGx5VM34Mlc6GCf\\\\\\\/v3D\\\\\\\/Lw7F\\\\\\\/0ZMsw3rD3linMScxv12lQD0i\\\\\\\/sBAvI6y9UDeWRdggHddK7+WqurqvXcv8go0AgTwGsIXeWIwz76O1SzfS+mU5b9af6lcLcqF+LZsyPRiTIX3cXj1HMXoF0BUheXxiv5NXBS9D1PLd2u4orp+CXrPBnsgocyH8h7HtSlzK1SQUMVY75tDhMcOcwiiwLdtjeoVsVKCPxe2LIicfez4J6KwITTCgH3Flx9GKOUDctXZwcFbofTBbSfTmEyGFOa3J0eHwArt1pqtwDhjm2kB6gVWN5\\\\\\\/T8Z\\\\\\\/ltn4kaYVwyV92tNhcSX9VTsAcrCNqni75eqO9fYgt2kObuR2B1jqWY4DdnZXi0i8UBwY+TOSsrO0h3\\\\\\\/FlRaURiHSIhN1M7qKUmTpWIzTahKHHFW3ket2qvt86XExfDyhCKCinjIVAufhmGdpchpGSAdn5Iy63fSw\\\\\\\/Grmu4YqQGa++E8sAK7JKRxlWoZKikdm0DhNM4KypDZFyGbWHNrY4rW\\\\\\\/E9UJaVMPb3mpl51hTxuG6Vh01hkC7m0Rlg\\\\\\\/gAxR4aBRytY6qjuLPZU6pBlWtIqKlK4h3aaKkhG8cT\\\\\\\/94l1kMcZczPG550C6bTE5ayl+ON4LG3VJ\\\\\\\/+UIjo+Z+ZoJ1fjt94IOpTcQQDRtZ9\\\\\\\/XlgZBf2HgbxTEh24XSMkQPQdeZPCNvx9xAuquJR2pz0U7Y08Bvl1UyWBfnfQo32QCZ0pjZYLKnDcoD9jeQWuQ7mR168uR7A24Eixt4MS6F49bhBwJHUyKZ8j2D3AD7FI7SBYbcTVT51RRiABS8CmvpiMVZwRJTULnhxpd5m+wKEaHm9C3YPbh168+gUTspmPYixpTFdBovEfVcDa\\\\\\\/TkCu1Fty3MJNAHpJg7t+YOAIewRT2jKeya1oAtOqA6RWg9eva3f25lxSNdSjJNHfM2t4I25+10+GvSlvfi97Wj6h+qNC9odpJ2amXBdS5aelB6vwldGsMV7bvpSAWGifOetN0Q2b0ELklLNjhW3Vur6lMnAzYLJDyrXwSEvY4s7Lr7jaT10TZy8+RfM\\\\\\\/X5Wx+oBhPJlN5ZfiHVewTfyoDlco2BiCq3n1eHKHcISoe4h7tzDi6ok7bfic5v\\\\\\\/IBiD4GmtytvJtMI2URO33AHPohA3zmzTnJVaFX0cEBSGmGz1KchA61v6TCfeYxrxk7P6BFn6BRDxeP0SxRUPEIQe1\\\\\\\/UEawirbSke0g3gabPtSRdRpXjhKkSqXowsilt1EEob5ouyIZtwKkEDfRx9JbVJtGNhOAC9NHWdY4RymcQba6Om0gQWgYxeA3UaRq8j8\\\\\\\/5k9t39LPZi+dUHctf1\\\\\\\/GeWtz2eZqoLs432Gx3d2Oxbnl\\\\\\\/WEVgH84pjnzCUecPYAkleBV1wG80KBQJTaRHLTcE2q6lmX1oS+QduO0HVnMMzqJdkf+pPtoNQLKx168Ek2409wca5ZCBNKhcnTkvCOdeCwsjoD68Jn0X5+2KyX5xngxvd8ctUMSpjVsCfBicB+g05FJnqvKDJ2Ja5n5WGk1OifNx4ZB2FlE3lV5kCHbtedPc\\\\\\\/GS7qltsHm5jpowXKBgPwWPtAsOxyhJnNhP7msnNPu\\\\\\\/cUQRbM8VhQ0aQyIdH0aoDG7bGnYBWILtXyObG4mcrSfdPOn964rFAObSp+77wqdEpGAxThy5\\\\\\\/SpcYWbfe5WarvpL51iCb1Zl8rRgHXfnjJuTg+v630AIh\\\\\\\/jVh2rQZh6gxpw\\\\\\\/CAWFZXAEoHdZR238JWazVNXiaQToLwqf3xoa8QWLTLa72HrkqA\\\\\\\/Hh1Mk29sYbTo+71c+EG6pdqsD4+ds8SVIc8oaq6ZXde93uYQ1Xp4Qmp3eikW516oWt\\\\\\\/\\\\\\\/pZx14s0ZMpUSV1FmAxiTpEBzRY9Yzllp0GIWZHnu8291qfOdDWI+B+834qNCOge4Wwk7ldEfQbpbY+6ETXARwNWdDA6EmdI3\\\\\\\/Y7g6CBdc9EeV4dtowdFuwvj3WBRkVosrbqTmrchNDV+HT8sdM+Jbpf0Rzm7ylpHaXiYDdGAj\\\\\\\/bsDW7FqtJ5NV7WWQxVk93KZEsGK2Xa6Gf6OhVaeGlkB4Nri3POHCBJFC9tdyUiZkFT2\\\\\\\/VaVc7ThPP4AtQBwqULlf2bhHzzTJQKKG4Wiblf1UqWAvNs5Z6KZGU\\\\\\\/gcrNzFAbyJbJ25J15EDKh5nGkSLxuYrFIMeet8iAtHstpMtX1pDbRiZ7U+rxYSLk6d5PpD7OQLNlqIBdq09xIQx8mhtGuZ1z7WkiINfHAavdvijIyveuIBlHoHDtAHwmXeOfEZq0em3trMYaqQKTD3FtK1B2sb2TYaFP06x4xwYjkGrPMjOS\\\\\\\/oMDHR+6V1q6PZEKbsR7\\\\\\\/juaxD3VzV\\\\\\\/JvXkot9+E18ykds+pgEFB\\\\\\\/HyFPQnfvaebbHy7dRRfoU5HpG\\\\\\\/LyTyQoRi9IQRzGXQTZV0RTo\\\\\\\/\\\\\\\/R9ONcsEs40ui7Ni0tJA0vuzVCJ8MHqXjB5R3meTanHWVR4FmhEQuHqaDL4wuC5ZbeCogOcemnz+nT4axzeaONJ6xCF4NMTr6xlAIrEJTmfOhfBEmvKM+lm5aS2hh8PKDeHYeGgCOKVXKmLmTgQcP7NfHPYRYvvsXgy87MDy2nwQeb+AoiU0Q53Q\\\\\\\/bW+Iz3frP8Kw2tAZRa8x+wCQpboOPV9jKLkC2uS5TYqXS+w8CoPkCd96Kn0atQrILVk+nEnYFQ\\\\\\\/CjljgTR8aEtm5K1Rw+tDV\\\\\\\/F+njDhDwJp+bBH8Ww4rY4cZdX\\\\\\\/d8pfvPV4OGDX3GIzHXsW9djJWJZjfz\\\\\\\/OdETEJz6f76O9c7q0r6DE5KkJpY\\\\\\\/ZIMo4+Ds5GcnWEvUcPOOyS4fQfgfypvwFwD\\\\\\\/HH4XLg2jRgBiApttGN34xgiEAqKiQi5q2fwN7865yuZGlo5u3l8FFsdx4hcTdpafCShRSLQtNlWvXgXiFEq21THGTPmHIZJKL9\\\\\\\/lKyP545k470phNgjFmK+RKUk03OejT19vZbNw\\\\\\\/jLUkAp1iDUD+EaavE6f+fVyI0i3aFMnINel3M8ZIHm9bpQObVrFF6QSYfi0yt3LbDxgQcX38iykTunpz13bmHU96Lm0rZYb7+BNTG+DEhe6PPZiaSruqdaadU9pJZlmgL8krxFJ4vDb2R\\\\\\\/mSrUbIpJWt1Hkj3ppoN6eRCTgI96Vffexm2G9E+Xfokxethbgls+43qIdk8crlg4dVovz3vg1f9I6yuj\\\\\\\/FyjpTnxDyCQXOoehpJ++l802JZUAer0tfqde45Urt9JPvHmh6PT2J1Z9ID57DEGGDF1KhVboZ\\\\\\\/9F7jCGLCVCgdgFy0gIUhu5ePCo3CJi1R\\\\\\\/xn\\\\\\\/uh9mt66JHeoJxxreTM\\\\\\\/txsxzDqMgUdeJBt5ZGHgfwNki5rySjDP4fd7utAe\\\\\\\/d5057Xw9KzfZG8HWLeeAxGY1FH0dDd4c1C84WF+thrjwqIxWdB4Bj0iZskoGxcrPvpbByYb0u7IeszgbbI3Q6fPNhmtE9Nz6FbeNalpkEybtopBnGU4geA7nuGoFxsO8c4IRkCvnVgFzHuoXz0GK6hlYsaunBtktnWnlCnzIhOC+vKZNs6sgIVYw0wSfXOGCP7nmjlWgQvzAapav8n4dQNec5Lu9sTqu85G5ZYGG7wPlkqAtflrShJt\\\\\\\/8Lx1e0PKzA9CXw2j5q8kwIPeTZgAVNLonIq7QxCljfFrQCLBdVmN+WOqaZzJ4V0qDfi+WGsJnsx7nzdWJPoNtyKq4CXvEOrVIl8jh3OLeW1Uy6+ts+qKC289+slLQbycDPwezuBqWf0\\\\\\\/wn3hVZlcBJdb47fKqcB+lYk5zHTVFV852775n3d2GHOeusWEqtGG+TlAaRvVqtbMRCSvbisY1EN2obHIaYR6JWHELe5NB0aqb1CJR7U0aq7c+iHaFWROvsLCgXCBtDWIR8pgPROgnn9J8eOoq6X5offl6uAJqaFsjkMTj\\\\\\\/beYXlQbZ7N0P4NXzoF9NQyp4xsxJPvakz4p6bmSmZYJ4H\\\\\\\/R+A1cSJdwQ2TOXHLwbSqf+wIZFjNGK0\\\\\\\/V7qMfCdhFpSBkyooRzWjOPxkGyL\\\\\\\/\\\\\\\/hYhnBT+sbdM3xKcyWLnh\\\\\\\/pb\\\\\\\/KUt5ei3Dhnco7oH0Coqt7Z1u45qCnlJ8tfsULg2Mf5q\\\\\\\/hyuD25E3rerAuwQFvWFZ8j3a6KqZCD9gxEAluP\\\\\\\/DMqa8n23ik2H6wqqycaNKoUuUC7oyyiDwgA2RKNmJFJ8jXhho5wnRGpwUWJpPBB9x9e6bXXYir37Rg1L2e\\\\\\\/3d6vKWHRNvnBqlqRXvo8gt9CdkVNtU9dJH+rdgDZ0sm5jOU97Fl+dxjoKBO1n0+3U6WtCscJ6B\\\\\\\/e+93E+9KNXjRjbgZTnahgvOvqz6qBvusn29Z\\\\\\\/aGhtLVLY8iQKziwa46sR0G58anp0Ji\\\\\\\/hgq4e2J3\\\\\\\/RHCJ5tV6efWWNxoHSutefu4Jmgm5iTQi\\\\\\\/kyvqK4QNH8Wa1V41EuOgAUI0HmkQ8lXWTDSTOo6mHWf\\\\\\\/UwmU6oKOEtVOrR\\\\\\\/5hoadqV+t8QQ69g2eCFfBWKG0XgWDPHJl9hItTt+BAD7xuQt7GWoMIe7VTjJcpmY74GYMpckcxJ\\\\\\\/sdCrhPH+rcJ66YE4uSQDua\\\\\\\/7Rpc3mvQadCGCwR2GjfQikaS9HuLpH6DUV7trwoQ5CxiWGlPOFrz4HOMbxU+rHZ4eEaQm7\\\\\\\/K5rfXxp70Y87JYsNp9dlkm7\\\\\\\/BGXpzixh6BC8RPxumUkm5rKBp1raEu4ryGcKia4HyB5uHmqFl3k6nbiogrJWLQtn\\\\\\\/8dmiwVSCE6Oz6k+dWRq28pObbp\\\\\\\/CrvUauFNC\\\\\\\/pzVrb1\\\\\\\/UPca6SmZH4wXy95cOE14pb36klO3JOGhulQT5bCzf\\\\\\\/TaXaOY6MQdVRtMiXW+lJtGZhZ6gLLUgEzPuQcZJlrhGa+HkZxmYH6SnVRayoE+Jl+CTeqGjWPQwLmbWhOIy6w3Q8uFMtS\\\\\\\/x4cX71+l5gEiGyCmUIT6X9phKtcm\\\\\\\/kjWu+zZ1V5kRae\\\\\\\/0bEq9gjbOuJki0kUKVuczm2JRtYUDxcrGQE78aZvOgDOfYvZlWc4e9YZsSIXnxxyu2T7BuCNvPXYjbllv7XfHsMxurcsYJ+JN3LfNsVMtoM8XER5PWH3jnCq8+Ay\\\\\\\/GAgIY8RTt3BH4cTPg8ZF4iwEM83OYcRP7QDpFNbHYJJD0OhveJwFtrpetrMVA8BMb8+NfFllnHF1O\\\\\\\/Vgm5D85\\\\\\\/bqinHcuFy4wJZSrAergnyslKhrwTk5IUBOco2OTQd\\\\\\\/iA8X5C8GdDoHQ9lo5k3V7swHf2U5GYQLroQPvimLHIOCkyABm0bJH3KgoEt5dJPpFxVuCHPp64vCTyHYkhqP1jp+WEQ9BMZoUf2lflSzxhZC8wkmKeLeAc5svjWf2e7EgTSJ\\\\\\\/BDJVoFDvf3DDc1qLZrx7KZEwpsNsQrS30elAPJqtThV\\\\\\\/0MnXkITB7cpPcfRTKyYpxwmO52sDiLg8Ks8VbFXfvC6zR8p9AyPsTURBNaPfnBuwH3ETl4sv4iClJY2OwTJSF7iIwQhOmKI54IKccFAi\\\\\\\/6g8bX79\\\\\\\/c0DJlddVP\\\\\\\/72weH2ptLhxZio2RPFwF1GWk+cOfJIxqC0yRIdfzuPQOsheN5bDHvyRmkT7TojdKEyaLctDyX2BnXsgElILvi8hL2LULgNYVdBuVnx+R+dNXvfVd2y5QnxTt7zViXec9lX8YuegBviYESkbMavary\\\\\\\/cjcrt61oHJOuAGpNpc9tMuG0CTRFcbzY+Wf6cIUUG44WIBiF17Od8UFWt6xxc4FEbkvoVT7Fbu05WHWlrcCZyFODX0WLa7ZFGrjUS0rptN8DaHRLUq3MHv4upeEQQqCLg0RSRFhhw7kHfdABqAQ30GHfFO9NDWFo98SG\\\\\\\/A3c+NX4IU5Wp\\\\\\\/1mYivjj\\\\\\\/XKDO5QC0AquE4gqHXZzRhLIozAkFRJjVn49hotk2\\\\\\\/Ui2\\\\\\\/TUhXrdBli7lsCWzn0EQDJvoei\\\\\\\/X0GWJPoKqmct7NpPhgieV7pv6MN8JBG3K2sQQYwC8sdVEEd7LyK1qdSfVL3S3xuLs3DLN6+s+1LbA7rNHnxQw1gnZq6wb9lS3cUG5Y8hTkxKbRNEYnw47XG6s3dG9vltgQ+yfHZc3zsbxQS2WWwjGheYtShUhZFTzILSdLSh\\\\\\\/YMdWhuzh\\\\\\\/4hF0ptwJY2BWYbI31r7vabFAR4aKsiiD52bkMa6JYg3JQ\\\\\\\/r4DMh6r9SMMK5GYCqik7S4cEdKPYtX0JKoR2DCRkPd1xWLWemxhVvtISJ27HZe4MDeBUCY7C6ZpAYwN9Kq0ZjxVFgvDyazrQTZO9bXnGMhCB2FQvCN2ZqI7RK\\\\\\\/JxSmOLkBq5D3iiCbHypZRn\\\\\\\/l8ivQEH9Qm+edUjI\\\\\\\/Tb1WCpAbco9mh6oD\\\\\\\/OeDTG7rRRFSGFcQZz+Phtab0ZAEHk3CdlogtqruiRJgIL73mPO6BjQDcM\\\\\\\/m8cXApiidwvOCLymM1H\\\\\\\/W3ouc1mRtFPh\\\\\\\/mQQYapufGBuitpf77\\\\\\\/aOl1AT0A04E3Bz7\\\\\\\/BPLdPlRwiwvjW959pWJFzkxcD6dj07B\\\\\\\/kTwGrfH0h+50j8R2HBrGCx0XNVItz6zejnqeSfFh2ekcfmh3VtBrHTNmkMgPA6a0IJ7dWGN\\\\\\\/ObKyyIEOBXHeT4QUjAugRt5WVeLXCn3h2aEO4EfKbJEPJRt85WtXQgwDw6Yd0\\\\\\\/IpIcvR7xMvfPeEUprJEMO95oWCR2Z5yQh6J+PQzP6GNpUy0X5HI9s4YzAIM\\\\\\\/5\\\\\\\/i3ozbFD+gEhSU26A0E4EKgtfnXxWOgOW1LYAFgkbFBwa6kw242Jwh3wT7IpON2Hk117at0W0evyO6DUqF5DZOHi0XcQ4421gBgP9NGWWSvPZY\\\\\\\/j1dX4DU4WoRKU00iN+rOsgNYAOpiI+CB7YIzRGWUhg5anfAXGbYfL6tp47EdgoSeb7XcCEUwFCzs+fC0eOM7UXMuJpiWVBT19S8dMbEEitkBOzSLI9XhtJgax2Bpv0i0PyYtReltzpHFG5jteyHjAuOsx+tolJCFWheX8OD2W1FMVLM3OPGTJ5t6PRqIbtQoDOf5Hn\\\\\\\/kARc29mQp2PoOi3JJIGpKK8\\\\\\\/BgZF57wj9Rejw3NFLmEU0L3MBB1+9QGqjRGpkrliW\\\\\\\/CnbP7fd0em\\\\\\\/xukHiMJK8OPRs\\\\\\\/FO9JUaZnvKzC9alE2AMay0JH7iPWldQlzdCa4LkKndaLrBlpENQamKqg9gecmuy4nZq8pjSUBcoWjneNCN2pH41146j3X8nysf0Uif\\\\\\\/0r7v10rgKXO1vMtee6ByC\\\\\\\/Vm3CskUl9l0BHzXmxUGJZBXne2MQneaEta34jr6EtaScBXEnqbhiYZ1H3RI8t0vnYJamcvoJZ5i9TK6a0ykSuaZa2NhErRuXv2Wi1VYIZI5qbcSsucS5nHVMjjY9jWOlFCoW5sq68\\\\\\\/jbRtMlYLli\\\\\\\/2oflIAtp5ZzMlAcsZlUmZBfWOi\\\\\\\/C3aRMC1PqdDwD7iOdwVNqxb5JLeIqo5B4t03BuE8xW90pdYM2B4wVgxKn8Tacb4rCYykYMP7u43btPzANH5EhylqjgUerV1Bc4IgYMCXtn5qDeR762SZEU9lpmBiYxMqYXTMIUOzD9d0OnbuIVXKVxG0ztUhR9hvgVOJLxbuzOrJEgZjvdZCGXNfZ+hq1ej3CnPP0nIGfryf20dVHZ\\\\\\\/pr40FcGMNJMnLESb1o02xRoGmhTogQZcGFwcq7ml3yvnY1Bk61DmaGzxA1gfbteQPInncseRFAo0N1EMVP2KH8pj60YRd6OR5VoKWOEKknGov\\\\\\\/xu32RkS+sl9EFteInJq55wYsJzZPDkWaSYBrBpmSUtQLfXS5B\\\\\\\/FarPgGN4bBqOZPhg60w6CShRqDzxUNNqVFDP7ch\\\\\\\/xK8FC8pwqluCSYqHv2QEA\\\\\\\/Tihz77o96BFkfWF2PIrHQjVbDE\\\\\\\/aonrvs7qZcPlGQSCzqfNMgi9zzcFapbwKdXI74IJr1hNxac+kSM72vJTHOprLHJGeZc5T8g5KxSSoBaCwtJ1UzPuzRHlJ5u1WbXPlQFp1LQkHAIUSpVxM9I3boetlwMEFPiVADhYNOQsce4AoSNO950r7\\\\\\\/oZjUON\\\\\\\/egC+UlLSB+GBT0pAbizzzPtVgDOXobol5f5fJYdk\\\\\\\/gZpWr\\\\\\\/NB4h8bhRtiiF6OktynzTZ02EtB8LN0swxuPlB7p7+it2cbQY8HCMpvl0+cVFe8l8gpbJUApppJQBTG+zcW6P9qUlkxZgnL5fqBROCyuxjdB3tj+Z7CAS1e2kFL9Ty86BFtIMuKdLy+gAtwl6za9lGoVW8rLIxDrCW+\\\\\\\/HT3zkktNIZ\\\\\\\/3o0akB+Vh10Ts0lfWGrieTKA9LI\\\\\\\/7SnWP1mZFV3dNMOCTW7fmz3KYPPa6kMsF6KMP9bxZZ1A7jbfOBWISxFcXOX5Y4ztT9l6xB0MmxsCl3w88wgNSKc9sl9gc+KaLnNA5Ti2qFpOqW4s6BoIPh3VZlREM\\\\\\\/Gv8R3JaK89lnaTCRi6cqVU605hkZPL6GuWi1Kgbr+9lk5uSdALE9EN9bIJJ9oCFuKL\\\\\\\/kJe9mgjWifF\\\\\\\/eNa7O7qNFfcJ1PsSUGF7M\\\\\\\/ge2\\\\\\\/3fEXCpBPS3RFryaNT1Xn\\\\\\\/ztqHWOdzpr87vCXCguwJAD+qtY8p6i4EDXizKT7LvN5JZK1BUd+UZszVnRamKDmbhVmOV\\\\\\\/+jSplhAcwhUIvdEzwzWDFsECSzBwB0BlGuGqVDmIx8CPbkqWufU8PZvWrAGutQv7POia0\\\\\\\/zPktp\\\\\\\/mKIO6eLKRwlEU3kOYe7oYIDawrbTouWjo2YoCTU2LqkGUsKdwhANybRg+UpM0nDq7xdms1pppSh2otwS0\\\\\\\/DV+LtTp4eNQBOzyBLqTQoLuU7OZjW+RU1D\\\\\\\/XA1ca1rdAuteMFE+QeXV4SBY5wrO71yp5Lu\\\\\\\/D0LsNA+\\\\\\\/S62IjSzazZNVq1LU+AQ9NmSbKjNmDFZlL9nNBlKzoWycUAJ+UaEFawlAjIL6jOb1L\\\\\\\/LzSrcFN4uCqvuyJwil6w0lPUMxA1cirqeNneKCFKNjrZkoSIaZCGYg1xVmSFaHzPTraw+5rpOT9R\\\\\\\/oZyQKezy3SVo1ZP05AnZw\\\\\\\/tzCJuFB3FULugUREcmG5UtNqys6lLbyqW\\\\\\\/8ZrdF+Su4tR2dQvmWh5lXE8J9\\\\\\\/tddWFV6AlwkQ0KxKPbHia8ZZXk6l6CPbhwAqm2u\\\\\\\/eoyb45NnDE5BoplQ4eSnZzASf95FFcqzQvv\\\\\\\/\\\\\\\/pGI7nHJrsYv\\\\\\\/wv99oH0UOHaxbR4sjzu2ukoFhTlaFvH3\\\\\\\/Ng3mRlOgvkZ7BHeDgOdSu7xetVsYQUEa9p5zshTUl5zSCxVoB3GuV6Ju27w\\\\\\\/KCsxdm+dCZWZZckJG5WW5VLUy47V5w60r\\\\\\\/YReFKegmYWPYM3I3\\\\\\\/U535EgI9XeMAXVeSgwxyRnRwKx+xdb3wSGfq7Kvle+rheFMwdiIeDVqRDLqm7nTd3dHaqGBsq3Q4t2DNXlehs3rAkCi3c65FiU3F9sdCI9KkO8R2GXjSDy2EwlDNplPRhv9+oLRQnHNuzBdrWsgUsagV7xczVEhONMfNhbc9fWei8Ouk7s2UD5lucITrLQnLTlKj8KaVswDqBnjwUyCdib3I4NbdvrYE0hyDMovOdVnfLnmPyU0FbNaaZZwg186pdfwzc96LJrP4ABHZs0e47tNBqBO3TU+Q+oKpPZaBmxXb+Tl8bUg4AvnCu\\\\\\\/YXXcpd+MwcXdApGyKhzk1dh0NGm4E28lnerWiGmWx0C8HAIHpznZA+0\\\\\\\/Az5RVxNauD4bXBT5dMJhg7NQnX1I7a1m\\\\\\\/IQru47kWlVtkKZSLpPcygwI\\\\\\\/DZGj18pTCWs4TK9yIB8swA+IJC7wm757MiV6tbejfC1+bWLpsDZOkF79obXivEqE\\\\\\\/MvGWlzeC2LktKmJSAcYD6Al8XR5wauwfrR+HVsoB\\\\\\\/es8bhsX3mpR\\\\\\\/S72YdSfIAF6qcKhHxwRhhRgemmJLR0QtoLXIMDMq+aCUti3P1rpzmR54XiihePScipYLuIIJVHsuoWwoDKG2puBjll\\\\\\\/FqLv60IjuyxqmbR5T8TCRNH1kf9X+8dWs9Wq2WSqs+t2pMZnlIZDEY289UR\\\\\\\/TdP2g7Hl3yLCMi6HIarN4IMqiVD3YlpKoMlnYig0GOFuWGtfVew0rLpmeBdNt3+uOO\\\\\\\/uh5sZRSk79oApOubjKu4V34RBuBnOHL+w0kCpOPX12uXU8JCDyFb1uDDoEaiNAnZmlB91c0Y6+DbXxS5xrQRqEJDXqqQI8GprMc8T9a4BtuG5l2djHYez+efDC5bCNNSLnT\\\\\\\/po8aw+Hg\\\\\\\/THxUIGnGrnpaQgw0+\\\\\\\/tbDvsGkBPezWMsnNzshDz\\\\\\\/EynzihjATBHamNBnLfHe7OfZ5t7PxFP0YyFDbte1io0pSa4lXWOiFCajdLoQWmA7Upx6iaP7KlqdlzGH26mOj4pGZKaRZH5JSyLr93KvxN1g8y9S9TH3Si7wNC85WkWBanqljPZ0VnyZOaXPmjuGKIVTODMeF8Kq+sV\\\\\\\/7SHhUeiK2GZ44QWZ7qienPIMYN2eRVwqaxqfJ40pdq5fEHqRJrZnCKxn9GwHnx2OmZaxR9xVl0DyStrfovyrFkhSTiWUIYQXuu5EQz+GLG04+BlTQnhdH\\\\\\\/fVC3xdgHl1+ydUn1t48mx4Y6RD4tqtDJZi0EziNNd30YMc5j2yuNsHLlslM5e+eV\\\\\\\/tM3i8C1zak+y0nTKgF2acsHUjHvQDLj2UJrnM4bYNBnljXRzXKk6K4M4laUkyvXHdCRh986ZyLMxuDZibpkKWZhY95TZp9cACLOWc5cK1wTseFcDsq4L+v86dVDTK9yb2eR+HIyueFMJ2ha6G4LkbDXUgPgjwVYQ4mAmI\\\\\\\/c+8R1TWwQZvtwwtJKCW7Kgl2Ve9OAZGMhFS44fuuOvLOBHj4qrl4WxhqGU8AGk\\\\\\\/pTJFtVmeHxTwZQSOWpt98sdh5cR16iKGPg\\\\\\\/21Uz1v\\\\\\\/yQYsJRGec\\\\\\\/IAu4N8DASNL8ZV6M1uRLqxjZN2umYOw3Z26nt7V0nsgGfhus7z2MRFaPbDDpvyR1X91+FYP4sUvgOCdsrD+t0zmfCbAoHs1RiphHO+d8+v6vInfMiageZvFuqhVQWYjwnoEmvTa7YDuCbey7WzSZNydJRgVbSBbdpbBMzYblK6XcqT6rUhe4k4dHiBeXZpUsab110d7sw9O31RAuhb8vyj6GoyCm8RUz\\\\\\\/IH2OV5lQdJqAMMSUPYpKUgZEVletx8OjErNIOGTAxILw3k6+1oUjcUUbKUlpBxf6o8qrJaXYmp38J8t6jMeXoUYCgDEaBsQOHCmSB2kn6sAl\\\\\\\/kGSjg9hOU2KxAqbABU3DtJ5dDHpRKz\\\\\\\/1o26yVxbkpDidvEDLxxsdimAuvqRFkvbmlQWpWkQEw80UdJhKGw0V3HS9QJv6frojidfjx81UK6BASUhZwk+Z\\\\\\\/T8GHwyvZLNg+wi9iaK11Xzk7+L43U918UC+IEgtnMzuIKGHkKB6tnFQzKQwSoZ4QLaP468ZADg\\\\\\\/0WbZUxsV8ddM\\\\\\\/lYo8HrbgV+carbGu+sWbiQXVJGmpfriT5WeU2WT71aHrcXrWwve2fFdRuf\\\\\\\/izzJEj\\\\\\\/JvpagRhwRBD3XIo3EBmiwWPti5Xka9Oamv9TN5KDd5aIbMbMppJ\\\\\\\/DdOCJynK1Y05wesbQmZEfeuiSW2S4x4aWoQAD+0aJnk7qTs155MGou++DK3rm5inlEtW2drdQd55ouK0i2psbLrty+4SXkdA5o\\\\\\\/VZZ7mxnMYI7Z6t1Y5PnSI2hGRLDjD9crPqRUwnO5OvsqQBCmTj9qJ+2NT5aGuyPJogXJTdMvOBrsXXyZoKGAavObRBLTp\\\\\\\/tSJN+yRI9q+jS5tiMvAHafx2KmamP05EC3OoUqmaBSyQFAgUVnCw7GJWWbpuMhkqIusx\\\\\\\/TVYvvbqvR5wbd106Q9VhH5e\\\\\\\/\\\\\\\/kBf64G7qV5WR\\\\\\\/jJlWDidTixkHYL5bd45jtGz5gq+RhzKB5CA4CMA7\\\\\\\/hk1gI1iIwT2\\\\\\\/iWVcjLOpvwPQaiLwu0JpIJmRaDEg+2vejFibSoWSi1tWcf3Py\\\\\\\/myM9RL8kUEorO5X48wVctJOCDhrut4aZApLZpzWt+qrQRmdtyuJV2wrs5UPxyDhOZzCg06aqVYsLT76x1W86UTqdMITXyUzZbQau8OYct3ydQr2ewWCK+7XfqoJLrHyH6rsunX0QLklem8jnuoa7hWAdQxIHqPbPrYu+G3XYNJvMbviZ8NXOE+g3lLsYDzzjPekT7UAc5JWxofQTLPZJ0B5B8LDotmLyrdToiSZc0WoburHynsQLm0ic3MXFS9W8k9vXf5hdms7w0V+bbhFxrmvTexN1eHq+2hgZ6kF4mnLP1P3r1sEW63L3NiWg8+\\\\\\\/yVMx1BRkq6fij4vKkFTHuOHV9sxhkoT8P19T+P2QrY4AhhrRNOtCbxdjbb\\\\\\\/x2Sh3ZJ32vcSl2mE5MerHcRGaCZmH18XsTWqqTHY1+pX0X4pmM0B8TuVcjsgubPuIwr7IgJ1p0H4gexOZPLz7j+Yg6AIH5WPyl5I4rVpeJQPAPjyMI967miDeMdB8AQlTwmxn69hb65fgbiRWdrULPm0z3oK8oZk4WgESww\\\\\\\/m\\\\\\\/YyU5N+rSO2gnEtIiN02l\\\\\\\/LUcrpz1Q8u3aPkyQ7aT1zHc\\\\\\\/3Hnl2SVDsUppl6WUncBviikaR5hwCvgsNZcZavtE\\\\\\\/fUeLunE6pJ2aIwa4qgbxu7RQEJWsiTVgmrm5lP5mJXYfGVVO79kh0f44P9\\\\\\\/QPyhbKKEg02gkSGRB\\\\\\\/RuELBDOOPwofeadkV1\\\\\\\/jOnE5Zow6NPLDu2XzC9egO8Kh8ycKLsl5CdveVOvpn0AZl4y\\\\\\\/ejV55E2TmBypMirf96J589BInI19fZHoBsHCLba+aSGPgtUrEaLAuSD+GURhoA+a8twHxv2aNA8uAUX1JLX14cuKzKsPs4Ip\\\\\\\/Otdk3mt25gNdWk1wh2vK+qbPuISlneJwjCVzigZiKNvIyzpbCTPIMeeGNCkW8mXYdRMJyyDEdVV4JmKA+Tlw4rzrATj\\\\\\\/iTLT3t0HmLXjFd319O0cwoKfC7e+rzYXT1WhIPVREZcZhTTqQV3ovPmyYNysUNmahs07GKdiNYNo\\\\\\\/URexWhIRX0bVuyQkJI7EG3ZbfxPvNQjEinTtBVBqjiGkjgwnQyLerLYIf1C7NvGtJJfGUvb88U5HeblSlHqLBjgrZIkvway6cevZQmxU84gzsbesB6ElzN+mOf6YiqXQF0GDbdCWRlAgWBZx3ggtu66kAJVkqAd6ZaglKApCkrGhp9uKdqR7kflAtsXCYo1GEZYDVv8LxRqtnbs\\\\\\\/waqKyEyKBk1fNPdro61GNgPtzH5SZxa8oSrDbUcX5EMsq5mZRs5\\\\\\\/z8q0lyoiR4wWV8YGA37Ws\\\\\\\/V4rW3eRiT71zMIDKczM48Vgs66rByHDL9lKQtdM1pH3N3b\\\\\\\/0EhZJff592LdhKA7jIiMlbgyAs\\\\\\\/Fvjvgu3bqMk3FjFlAsln0A6XRGnYHleafkdNGNG0wH6vdx84jvOr7X7ngbLpgDynXB93YMjQTdtfAFWkRla3MCRYmQr8fMzpQjnO5Dc+cVcjECxoF5qzqU444\\\\\\\/uzH7D5elcY1WzwOdeLVdKdhsu1\\\\\\\/oALkARCLNu8MRhP1zLapRDKYvcfNbChL1W34uToSM2SmWeaKQL0CNmww7oytoQZwW8FMH0DVM839dg598JrKm3RwWNSZVB7k1\\\\\\\/T8ErzPi9g6cZwZnlEmXN9mu2dQqjhlrfCmk9Tx\\\\\\\/RJeBijj0DKKTSmDVPS6bCrRWfZbbkh9w3vB9MSTFagMiQ8EJkg7Pk2Y1xZ\\\\\\\/mCRbNOdagqK\\\\\\\/0jXNdDAAQBN9IEqv2WUbxE1M+6ZTa3uTriUL41NzkYnV3iQTi+tiR3ZDAnpLOVMjsFXGJ006BWM10+JXo7FJajs1PowvYIXAHRsOksPlPCMG4tLXgXzefK7gycQLXS5mssYu3Ghh\\\\\\\/q+vnZc05l4ofAUVpct0a8BUEDUAF+QNBUT3UElKlFwhcHiNaJVHhL5S2KG7O2uviux9f5mgfsCI7hmdGyqnJ+Ku6gL5o0+nFCt0T0yfY9sKgeKUrDMU+UEOD03jxTY4W6K8OsWfV3yDIsqyQuc\\\\\\\/x6fjQUnC0VbKEnXN0O1Ga87liU2MtSkfsg\\\\\\\/KSPt276s43pK3dQY6bSlm0MF9bITWBivrU3cMbZVXFHNV2T3j4gO65ms05\\\\\\\/fTCJ2lSuOV5QH4Y6usnh188Z2IHiMrVZQ65wLjr7ePp3lt0JAvEJdz+iYe80RdJsmeWwxl8mnNgrKALyj+NjaG3xjYXyruD6L5c7di\\\\\\\/d00Ckb0CURg7Fv6CsUpXvdErviiHqbO5dVR9iZTSX\\\\\\\/yJVfqS5JV8f5gJGgNBkw2Igl4nDb0tUBcCZ5Oerg6XnoZL6CYF33IGEkf9b0zhSNC2C2K8r9hLDlOBlySEp\\\\\\\/DyCYSHqeTM+VEELvkOXk7+sYLGavSUq7XtZGjngA0tLSHzItAvCB7cpaM1uJwQ17QN\\\\\\\/bIAcfWEDD8NlfO9M0cIyDlmmkOW7gRSvGa64gIXit8vdQqblxR2TGOQXxaLpxVbQ5XyQui2KuI0wA0d6miRYPPf3i0GNhEthIUG5nf+hDeAl\\\\\\\/4ilysRplNI+taEgbsNMqUxaF7UB7yUbLY3fLuIkMdDRPXGIj7BA53LQul1f+a3hOtKuZIJpaOXdIycTy1TMhT\\\\\\\/FRXpvxAW+owYMA8vJQXhPAb4nd5ZrmVvmElEkoT1AvUI3BdNQc0FOPF5hqnpOhDfTV6bmnrbAGUcl30lZUHzfOmuN9ilSQHFj8XFE4XVZn+vRJvOPh+732WV+woXO7OLR\\\\\\\/OXtjeFvvy6+K8m4e7Ryo3WwKX4UEu9bQnYalXTsXmJ8Xj1lPrGVxco2zpCSl57D8uHBY8k+ahefFATW9D1WAMgIW\\\\\\\/lNTzLijQFK4U2Cvt0FXKouc9K1gEVqgbnipsfqD34TPKscwnLxr4Xx6BHepNAxaAUNBDI1Qcm8sp6NrxRsvImNkTUhg3Cjj7muzyzq\\\\\\\/mIS74trIR+5a3wYGMxgBukwDxfWpRGzO3K85dIHxqx63a5JzdqhTbVOEjEk9NS3X40axG0kPoHg5aSU2U3vHFZ0QRbImlhV9LZYdLuqCJhKZj7M\\\\\\\/N8wPJG0bFFSkaYACTvZypJf5w+81m94BPHDBHj6h8UIC3ppxNcQhG6oT2Essuya15ZnPxs1Ct\\\\\\\/tLCTqepY4IAyYQW\\\\\\\/9BErWfISFDE0VBI5XuuV98G1HFz4ps\\\\\\\/EWAmw0itrLDmSS64SBF7hjBYjeB9lUIqE4dfSJkA3MBFkpM8hlZr+9wWL7NnGusxuQ13TXe786CM48XJxzqQ23opN55h\\\\\\\/kVle25MQsDkVdFpp4K2d25lBqOULvvxWORtLx5lEgvP07o9D8wzzeXcYs4Fu1oFFX8dGO866qTEkSQQwSQo+xlW4bplh28alMYmdeiHGVF7QWel1HvWT5b+u\\\\\\\/djtEH7\\\\\\\/BATA8EonaiEg5+neAf3pWvTf6cae+t8LJyLknJpFZRog5u0rpr7ZIp4nmlLCWhJnRGZI9nCC\\\\\\\/fKwRyD0orPAZv47z+yHpMipr8ME\\\\\\\/YSHynuFwNpv8xMsS6QxAkYkn2NAdkr3IEXs6CwaHulwbyfUAi3ptMT9088JvIvYvQyiFu9\\\\\\\/PzB4MlZHdU0kmIz+pcYDVlwiQpTN7s7o507ouvP5o48sNzKyT6QPeCLSCuvcyLIyASSfKS+wGiCDYismFsStIzwIRW50T\\\\\\\/PNga9hKXlJaT1Poz1AysRrGEFmDAvdQp1fDbeWlSM1fM4CwiU6Wtvn1OZEq8XtQ5AjFbQHM9k\\\\\\\/oCBTnO3qsQ\\\\\\\/lGeyagxmJneM7ipmDxVvGYPlBXfPIDuWVbA8TbiMGu3qMQslWIQxioDBe7PXr4gRYv8xiY1qF27fXKs1K3EsLv2amqVMLRDxGwxByZYbtfibjQMS3X7DswvOWEcR1pAlzQWIlZJYBeGgbjXW0OJaB+ohvbGZUC1LGtxX9TXikvK9kNJ8HuE0RIsBbu3JaI9dxaxWNE7AdtuqkRD6CGRSv7p6wZOntlMbXuUUSStxDe5i+zARhQQFrQClUChj10rVpqgpv46LFZnWkldJkIayhNcVVmroZW5qYLz6PnOdZhKV9XqBXDOFBHOv+bheb8Mqyru6yO9VAEe2qmfDV\\\\\\\/T66PPNbA2HpPFUJPn3XmrDA\\\\\\\/Z4jH60Q\\\\\\\/Qfx1YCNBQ+\\\\\\\/kjfktACmZSbut2dnpLbvUEMrAD33yZeFOopvkS1irrfYQI47KgoJdpXA4b0Db49sinliswlqFeDk\\\\\\\/Yi7e29yyMpu4LTutBHO6Dcq+GOoqx6448Fizv1e65AjD7phLrV7AHLr6vN4u7bqZT1lfZuSWLqUGSPDJ9wBnJX8p5HbZHMATmu8Ye4VnxEeVTZ+Pg9SNJhKZSZRdvIuUkEn51BcBNYWeu3vD57eOslyUu6c7gn0etMx4IPC22Ul0P9sdiTcnLULdr0\\\\\\\/kXRbpGBDVlb88r9lBitU0PpIPzUvBZgBCRaevUKtCNldkqLrvn2wH40tcqbvwVSZvQhkdU+pSF7TiZorUJH9iwHd2W2IxV1sbWkg24eGDX2\\\\\\\/AYXYXEu5QEFx5WHxGEmf6FptpmPMRM4P4nBlnHWWpX1k9uWr0++hMF0s8G3Z2my6Kc9J4lxwgVcD9EP8OQTzehQrzAC30UqiewGFR9f4uul0rHqNzLGdtBfz2QMTHAkN\\\\\\\/rSF\\\\\\\/GJ+s1Rw5HRQY2igdNSBsqKBU0vBcjj8tOH2QOa3b2z\\\\\\\/nWZVJcg58Dz0BtJjyDs6tc7mgiFVv1hP8BUxcWKV+D5JSChHaoeeuVvJRD\\\\\\\/NVf8uPkjTgSvhI9P44LkCckBhJpkRia+iwiR7yUPQuoD2kzSu+mMB2MkyhuyPpoY\\\\\\\/C\\\\\\\/yEA6JNKPJLvJc6FzDntotnmQ5jU8iIucf0OBVYlSLsdxEOLW2zZZxd1uPg7bc53FCL4gXabwZirHZ\\\\\\\/nLaJM0GuRZoH9qCfcRmizzU1K1YvrskFBrMO4\\\\\\\/tyBC9SSqI8ROkmkyc9U3sxVw64AsG+Qc4RUYc1PsqRRKZPS\\\\\\\/ErTXlL8G68vec5CCkRY1aTpP1W3DlNc8qTeS0Cx0XEz\\\\\\\/zahazW6vRDdetE7Dq9v+ObjJjxomIrxSKm0T24naxlR6MrSsIh5gFURalP\\\\\\\/IZWPGwSNhKa7Hf6EOOwLXhOtIDZwUwDYWXgJSF1mjSKClqvjlANNUKnSMFivit+Tc0cy5p42RZ06Bgr2ymWv28HktNcpyIvFNpG4X92w\\\\\\\/h5\\\\\\\/aAyG3h34twZipnpZqC7P6+BvWB1NdyNbeBGfB6RpFptc9g73MYfB91vpzLHdhaVX26zdnjhlqRjb+mnQyXq6Wn2wtLTQlzYREmhtHrewUCk\\\\\\\/pF9\\\\\\\/ZJ0dApbDlp6a2FivHMTij9sJGt9vA\\\\\\\/GZEsYjI6saH8SYkbPreiconJ48amAs9plmpXL9cUPdaz1cjjWzCV9NmaI0\\\\\\\/bYf4UJQsTMfR1Qc9T1Fl9ZkVUJhgerL7y4opM2C1zW1fdTw43dwVacOylck3KIbZR0deY7TLeQQ62AxCKwNzMem5SBY2yBaahg+YcBqEjjoS2ngOd8d\\\\\\\/q+J\\\\\\\/5hSIZRaCP1ILJYGy\\\\\\\/dgmMDSxeX\\\\\\\/xABGOJpJkjPTSx36TIHqSVnKxmp\\\\\\\/ft5N1ctR2gd6jJgnHiZbJBXYAzbUh5xhY02mG8GPg7925IdBLsRPmav4IgCYfMQMEm1xOSa5rQIZ269LMAv4+aTPtQcQjQD75kD1IkOXn6FK+6jkQSwBp63GqlxNf+pQ5KrJHRugtwcsP4JLdthuPioPcpO8irI86x2qz74UYUxPknCURCte6c5iXpSh3g0qNYH8s6kryk5bPVelKmf5jYd012opm3C8nmcS6jOF5KiZsJfmr1gkGStW8+qw4aO1DdUNFmvM9fGJy2BZt8J+eIxpGUfdCX6lUIiJnV22LrVWQeJaiOh9LpKt1PzRL0Q98KBEfuSNMQ\\\\\\\/elgCszQVWm0p0YBcGVW239NH86l9Ok7Pz9LviORtCqnBkL36BAGiBrkFL2HQwZ9vjR+hozpVZdZyBStqbs6Uc297c96uh5aOHhEZZOXa32qYoC3ZkKNDc8q5j9GfAGc+uxV56Ql3e5UvWdJ\\\\\\\/sJT8IcKpbRUpO\\\\\\\/H1BezW+YRyL5cVmbMIITL9s5Qi5kZDncSim7eUsICP9xCufcw8sFTK3k1ohb0wkTKWDEMvL3mnhQXV9Dc6j29O0T99ELGV1Z+LjfRGhSHCyJwyMZeESz1NolHqdzrig6HolzQ\\\\\\\/vObUnaO2X3hes5sfxw8mys\\\\\\\/trR3tktUEe1E2aECMyJGMgbd9etz3pP17hop07DlABL1L2+UCxIuS8wlfLo3pCkzvyrVR8Jabw4w\\\\\\\/GTILRh37b9gfl3SoAWa5+yGgM\\\\\\\/i\\\\\\\/WEhu\\\\\\\/9P1BJgptaXmzCddJJ5ceC11YHWYAi\\\\\\\/+UyO9eeBR4RQ8RDwi\\\\\\\/VNyzdPkmaYjv2sDLc3Xv69162a5cdtF01g4vyZTw\\\\\\\/S7ZBaYERc8sbYL8ma0Hq8FApYaVmKe0Uo9pZy7Kq14FHGciUl9rJdFY\\\\\\\/681iWFjXGcB3lYMFPd4ct+uED5jOU+zrV4lOmpOt8avhl\\\\\\\/OlbwApj68JnnKnA6sKDvpAfMKtTTmZK+J3pfEErTv40AOfGQ+H8nzpQnVQmjuJhKYYm6myX0gwhHPKNvCXaKuBX+7vTg0Esn7+yisHInOgXspB2Gpa4bhiTW4wu\\\\\\\/seANKnuK2OIURxQzFjrBFeID8RN3XJKTCywfH88rv2ntp0rAAyOj21wwQXschIIkyZX4jPyIeJBxATYz\\\\\\\/R9WG\\\\\\\/26qGcXvb2jc\\\\\\\/sH3fjaN7V7Ri86ncbItiH4i9H3bLmPrfSkQ93LZxzZGhMdv1nbicTlfuq8Bmsl4KZ\\\\\\\/I+bjfTyVsF7eKuZd+DtwnyIqPjt\\\\\\\/a49BAjWFH3yRN1sXJvUrNbJI0HJXXMQXPtbQmCrqJ5DGarx2nZUDAQ\\\\\\\/qRhosTyr\\\\\\\/IfYuMJvLRT\\\\\\\/8B\\\\\\\/d2BUcOELzlP5CngKnnHe+98GxgldVXicnJQR8GtRSeZmQdtD9PJEza8ZCkk0i94z7xD258itQUD3EvXhaJ\\\\\\\/id+DVR53+8I4f7KId8gNSPZ80TVjWs\\\\\\\/cudyiJBKKFhrTa1puLsyeYrrkIz0ncHuKUUkXopCUj3ezvzCl0GIC+fqa6Ca02G83NG\\\\\\\/y4jsN6XhNmeFDQc2Q\\\\\\\/hgIuHzB3o5knVqCByDzwLlYE8yqvlWY7i6cBY0hkstpymN9EZ4PCmp9lRV4DXDVdqs\\\\\\\/qWYpfTkYngaD2qY8AH53A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q1472qpPcljqVLEhfEq6uA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002236641b2280e2eb0753faefcaf4020ceb39f84abeaef1776949dbc61e22fd8834\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229xWe4btoWfmKaDXv8bYSCUcSfcQntX9fphOpZt7iIdTKtakzNnsZmaihi2vANwK6\\\\\\\/z+HRqtKGaR+dVNvCOc8x1vNNHh0vYeksJVQIqDefrsPXduV0m4\\\\\\\/7kK8HoRbrDMfzrqIKmgltPT7aP8i1cf3P6f2N49EiAqw9H7x0J+OqvImUCciZYPxM5rL6ptJUKazg0LSIkDPTAF\\\\\\\/luLnUr3n2PocH1BYsgCnXTB9B95r96klVSNnyFoT3Q3sLxGZ\\\\\\\/GYvEtut3Ndbdwbuy64Eu4OIbcT02KXgBkw4VSuyKDuM27fj2v6gDeaj6DdS0+uVpzqUNYbFchNvxrWN09\\\\\\\/3uJK5gtWwzRZ8NmLAwbFGtw16bX1wfWSpPI7bHbt1J3Fhq7P3mDj7lEBgXH+FJsgfn9vooXltJ1ruzEbxH+as7zFAjV3UfJdgR20HJJL7+XelUljoF9ql393+oGn+eNH2FntNHLG4muqdGnbZRcojj6d1p0dU+soRrql9P6SyBN\\\\\\\/LCbb7RMerLJ391tlU6dtQSjjyylDQ6VFCXUPG3nxJK7FAG6yWPv9\\\\\\\/fng3y7slbKDiPy+RK8A1N6QGxIklSMEZPHzmv2IHaqCXR57XzueIkhxtarDxVMcIy8Fj+yrn9HSCB5D9paggcrmbPGtXyzEEznELBsWMJ9YsYIg+xm6\\\\\\\/eyx3WLex8lpt5x0GLEro9VlVMT8DKgVcbb5FagsIjLYQD2Op8AavCTVLSxhbTh\\\\\\\/P3k2E6sHyKXUp1hZ1fuFSrt5LZMd88YPrNaA3z1rHu5c7fUrGbVYTMfb4yWBd2Q0tURxMm1LREajQmNETbiqE0F2KQbz4PzRZjb6iN+wxT97E8vHrfKQfhlJCmpRmYTav7l6HBqJlpIe\\\\\\\/h8qE0yTWkU7LW2KhaiUNESyEugXn47Ab7Jol364JWzPSaWBIuN6BXGXJs0Dqyi+ACLgzmOknWKpjXk3IszzHF4uizWEKQ+KLRB\\\\\\\/eFcq8ym1ZZJ2kkQrw6I5T9MRE2ObzFuQGdIiyCe\\\\\\\/weUqoCU4xDHBJe3+OC9s294723xwBIdz6hpJgDKsTd+DVAvz7JHedtNyAgu2XOp5IbC1KZLwjeVzyweDLhU4rxLqEin4FVMM8wHBdw7LUK\\\\\\\/TU4wAUF4YZenK6LQdoBXip570+iR8JhY8fAqU6HhqTtNbqnva5UWu\\\\\\\/DBZ4rLaeu\\\\\\\/GLLmsBmkajBwWXT3CWBUnCsJUp6vHXFW1Iy4PwTf+\\\\\\\/ggpPHUCaaO78mGsUuZsLNsZRNh4m0NL533nPhJ196zKpUCCz4rniycYVZ967ofNn717dxClHG1C+uzGXGrEIRRW+B9oLui8dKogvZ3TEe\\\\\\\/ESweNlwZOwWuBVsZvRA6d9QnL+BUKz+5xS+KPTb4FTZDW6\\\\\\\/awxKMnZsUC\\\\\\\/x2h9qOlpLTUbVTdXm4da9xjab1cF83pq4XxmT23ZiAyqar8X4rnDewlQsHaJWHgtfJWVlRuu48YsfMRWawKbPL5k6zFThXVqICN43jiwgJaOjPoWMRF7TjufJk5WQoFgasc2DUcVrNaUkkSCli+CtlNvHDMlZ+k+PNDGQIV6tJTNbAIZW1IU12CPrB0DECHZXuxgaOE7mkW3TAxZpG0f18RSrHAtlj\\\\\\\/Hhg1t+1M8hvyBJWddkdwO+iKdU4c4K6pQV1eQp7FiFV8fnqCsWaNVYiB\\\\\\\/cSCf4hrS7rPR0yVNH7KmeRb4KMniZe25qSU1NPhitInfWbYVn6NQpapqTG78TDM8jv1uEy8G9NSWY\\\\\\\/+QblbXEnQfSv8qpJECQioTU2D3+9OsBUTdZv2B6cwifz\\\\\\\/dn63RsTJpQP42mL\\\\\\\/1VixR3mh6fuk\\\\\\\/0d9uw7ra8vKvVeOvJmSS5xqMr9XGIbnbeSNHUOT47xL30e6s018qb10nAENIXSngnvXFiBL99ngI06+lfp3nh31geRT+XXgsa4h8x8oWNCze19VNRtbSbj7uv\\\\\\\/G\\\\\\\/yVleKzVE1VxmnkjdGHegD4u6Vw++Jb\\\\\\\/DQVQ1AAXHYnzywwQd0Hj07Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZzlPM8n3jVjYfCGynrS7jg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e1258c5b7ffbd1dacff5e03ce39ec4ef679911fc0b149d6362b0a656fcd0913\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459892 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genetics-of-sex">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/6/993" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-n_YoFUadeXkXm-ojbqiz9-qOI4cl5xta73XyA4P7VI8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/6/993" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hE90bBJvxoiRi8viqJJpk1FPRCPuJXu_2wrgNGELbbI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/6/983" title="Self-Incompatibility in Brassicaceae: Identification and Characterization of SRK-Like Sequences Linked to the S-Locus in the Tribe Biscutelleae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/6/1011" title="Sex-Determination System in the Diploid Yeast Zygosaccharomyces sapae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459892" id="node459892" data-pisa="ggg;4/6/993" data-pisa-master="ggg;4/6/993" data-apath="/ggg/4/6/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Han Zhang</span>, <span class="highwire-citation-author" data-delta="1">Rachel L. Egger</span>, <span class="highwire-citation-author" data-delta="2">Timothy Kelliher</span>, <span class="highwire-citation-author" data-delta="3">Darren Morrow</span>, <span class="highwire-citation-author" data-delta="4">John Fernandes</span>, <span class="highwire-citation-author" data-delta="5">Guo-Ling Nan</span> and <span class="highwire-citation-author" data-delta="6">Virginia Walbot</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1010;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009738 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459892" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Han Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Han%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rachel L. Egger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BL.%2BEgger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Egger%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARachel%2BL.%2BEgger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:egger@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">egger@stanford.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy Kelliher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BKelliher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelliher%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BKelliher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Darren Morrow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BMorrow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrow%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BMorrow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Fernandes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BFernandes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandes%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BFernandes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Guo-Ling Nan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Ling%2BNan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuo-Ling%2BNan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Virginia Walbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BWalbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walbot%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BWalbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/6/993" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459892" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459892/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/6/993.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459892" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459892/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/6/993.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459892" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459892/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/6/993.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459892" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459892/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/6/993.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459892" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459892/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67997 first last"><a href="/content/4/6/979" class="" data-icon-position="" data-hide-link-title="0">The Genetics of Sex: Exploring Differences - June 01, 2014</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Plants lack a germ line; consequently, during reproduction adult somatic cells within flowers must switch from mitotic proliferation to meiosis. In maize (<em>Zea mays</em> L.) anthers, hypoxic conditions in the developing tassel trigger pre-meiotic competence in the column of pluripotent progenitor cells in the center of anther lobes, and within 24 hr these newly specified germinal cells have patterned their surrounding neighbors to differentiate as the first somatic niche cells. Transcriptomes were analyzed by microarray hybridization in carefully staged whole anthers during initial specification events, after the separation of germinal and somatic lineages, during the subsequent rapid mitotic proliferation phase, and during final pre-meiotic germinal and somatic cell differentiation. Maize anthers exhibit a highly complex transcriptome constituting nearly three-quarters of annotated maize genes, and expression patterns are dynamic. Laser microdissection was applied to begin assigning transcripts to tissue and cell types and for comparison to transcriptomes of mutants defective in cell fate specification. Whole anther proteomes were analyzed at three developmental stages by mass spectrometric peptide sequencing using size-fractionated proteins to evaluate the timing of protein accumulation relative to transcript abundance. New insights include early and sustained expression of meiosis-associated genes (77.5% of well-annotated meiosis genes are constitutively active in 0.15 mm anthers), an extremely large change in transcript abundances and types a few days before meiosis (including a class of 1340 transcripts absent specifically at 0.4 mm), and the relative disparity between transcript abundance and protein abundance at any one developmental stage (based on 1303 protein-to-transcript comparisons).</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/archesporial-cell" class="hw-term hw-article-keyword hw-article-keyword-archesporial-cell" rel="nofollow">archesporial cell</a></li><li class="kwd"><a href="/keyword/cell-fate-specification" class="hw-term hw-article-keyword hw-article-keyword-cell-fate-specification" rel="nofollow">cell fate specification</a></li><li class="kwd"><a href="/keyword/multiple-archesporial-cells-1" class="hw-term hw-article-keyword hw-article-keyword-multiple-archesporial-cells-1" rel="nofollow">Multiple archesporial cells 1</a></li><li class="kwd"><a href="/keyword/mac1" class="hw-term hw-article-keyword hw-article-keyword-mac1" rel="nofollow">mac1</a></li><li class="kwd"><a href="/keyword/pre-meiotic-development" class="hw-term hw-article-keyword hw-article-keyword-pre-meiotic-development" rel="nofollow">pre-meiotic development</a></li><li class="kwd"><a href="/keyword/genetics-sex" class="hw-term hw-article-keyword hw-article-keyword-genetics-of-sex" rel="nofollow">genetics of sex</a></li></ul><p id="p-6">The plant life cycle exhibits an alternation of generations: haploid gametophytes contain some cells that differentiate as gametes and the diploid sporophyte contains the megasporocytes and microsporocytes that undergo meiosis to generate the haploid phase founder cells. Unlike animal embryogeny, during which a germ line of cells competent for meiosis is established (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Extavour and Akam 2003</a>), plant sporophytes are entirely somatic until reproduction (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Goldberg <em>et al.</em> 1993</a>). In angiosperms, age and environmental cues trigger the switch of a somatic shoot apical meristem to an inflorescence meristem that can generate flowers rather than stems and leaves. Flowers are almost entirely composed of somatic cells; within the central-most organ (carpel) and adjacent organ whorl (stamens), a few archesporial (AR) cells differentiate into meiocytes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hord and Ma 2007</a>). Meiosis generates haploid founder cells and then mitotic cell divisions in gametophytes ultimately produce an egg cell in each embryo sac and two sperm nuclei in each pollen grain (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Batygina 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boavida <em>et al.</em> 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Feng and Dickinson 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feng <em>et al.</em> 2013</a>). Fusion of an egg and one sperm cell during fertilization restores the diploid state in the embryo, completing the life cycle (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Goldberg <em>et al.</em> 1993</a>). Most of the world’s food supply depends on these processes as gamete fertilization initiates seed development encased within the maturing fruit; together, fruits and harvested mature seeds are our most important food sources. Modern agriculture also increasingly relies on hybrid seeds, combining favorable traits from two distinct parents for higher yield and plant vigor. Suppressing male fertility in one parent to permit facile crossing by pollen from the second partner is the key technology in hybrid seed production (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Scott <em>et al.</em> 2004</a>). Fundamental insight into male reproduction can thus result in agricultural innovations in the multi-billion dollar hybrid seed industry.</p><p id="p-7">Multiple transcriptomes and several proteomes have been reported for whole anthers, utilizing organs that have completed cell specification and are proceeding through meiosis (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lu <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ma <em>et al.</em> 2007</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wijeratne <em>et al.</em> 2007</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Skibbe <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2010a</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aya <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Deveshwar <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nan <em>et al.</em> 2011</a>) and for anther developmental stages after meiosis (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ma <em>et al.</em> 2008</a>). Using manual or laser capture microdissection, transcriptomes are also available for isolated meiotic plant cells (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tang <em>et al.</em> 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schmidt <em>et al.</em> 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yang <em>et al.</em> 2011</a>) and for isolated sporophytic anther tissues such as the tapetum (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hobo <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2009</a>). Mature pollen, containing the vegetative cell and sperm, has also been characterized from multiple species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Honys and Twell 2004</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ma <em>et al.</em> 2006</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calarco <em>et al.</em> 2012</a>). Collectively, these data provide rich resources for bioinformatic analysis of anther transcriptome progression during meiosis and gametophytic development.</p><p id="p-8">Sample staging and anther dissection during the earlier cell fate specification period are difficult in most flowering plants because typical inflorescences such as those of the model species <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em> (rice) contain a series of immature perfect flowers at successive stages. Consequently, each flower contains anthers at a different stage and contains both carpels and stamens. Additionally, anthers of most species are very small (&lt;0.05 mm in length) during cell type specification, and the patterning events required to build the layered somatic tissues occur in rapid succession. Given these limitations, initial anther developmental stages in rice and <em>A. thaliana</em> have been approached by analysis of whole flowers, including genetic ablation of the carpel to allow anther enrichment (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wellmer <em>et al.</em> 2004</a>). These whole flower studies, followed by <em>in situ</em> hybridization to localize specific transcripts (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schiefthaler <em>et al.</em> 1999</a>) and evaluation of mutants with defects in cell fate specification, provide a glimpse into the complexity of gene regulation during early anther development (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ma 2005</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Feng <em>et al.</em> 2013</a>).</p><p id="p-9">Maize (<em>Zea mays</em>) offers five distinct advantages for analysis of isolated anthers: because maize is monoecious, in the tassel carpels abort early, resulting in male-only tassel flowers; large cohorts of flowers are at the same stage in each tassel; development is highly regular with landmark events and chronology corresponding to anther length (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>); maize anthers are relatively large (0.2–0.7 mm) during anther patterning, which facilitates manual dissection; and there are temporal delays between specification events. The longer duration of early stages and intercalation of several-day cell proliferation periods between the completion of successive somatic cell fate specification events account for the increased size of maize anthers compared to <em>A. thaliana</em> or rice and makes maize anther stages easier to recognize and dissect (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>). For a size comparison between maize and <em>A. thaliana</em> over the course of pre-meiotic anther development, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/009738SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS1.pdf">Table S1</a> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Smyth <em>et al</em>. 1990</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sanders <em>et al</em>. 1999</a>).</p><p id="p-10">Maize anther development requires approximately 30 days from the formation of the stamen primordium through pollen shed. We previously profiled gene expression in six anther stages starting at day 5 (1.0 mm anther length), day 8 (1.5 mm, start of prophase I of meiosis), day 9 (2.0 mm, pachytene), day 15 (2.5 mm, tetrad), day 20 (3.0 mm, uninucleate gametophyte), day 30 (4.0 mm, binucleate gametophyte), and day 33 (mature, tricellular pollen) (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Ma <em>et al.</em> 2008</a>). Anther gene expression is very complex, encompassing ∼75% of the maize protein–coding genes, and highly dynamic with many stage-specific transcripts. The intricacy of the transcriptomes and utilization of anther-specific programs support striking conclusions from genetic analysis: defects in hundreds of genes confer male sterility in maize and other flowering plants (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Ma 2005</a>), and male sterility is at least 10-fold more common than female sterility (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Timofejeva <em>et al.</em> 2013</a>).</p><p id="p-11">To rectify the absence of comprehensive data on early anther development, we isolated maize anthers after initial cell fate specification events, during the mitotic proliferative periods, and during preparation for meiosis. Transcriptome profiling of these whole anther stages combined with cell types isolated by laser capture microdissection permitted elucidation of patterns of gene expression and likely cell type marker transcripts. Stage markers and patterns were tested by quantitative real-time PCR (qRT-PCR), and RNA <em>in situ</em> hybridization was used to evaluate a few presumptive cell type specific transcripts. Additionally, <em>multiple archesporial cells 1</em> (<em>mac1</em>) mutant anthers, which contain excess meiotically competent AR cells but lack normal subepidermal somatic cell types, were evaluated to gain further insight into the hierarchy of genetic programs generating initial cell types. To evaluate the temporal link between RNA expression and protein production, we established proteomes for three stages after 1-D gel fractionation into 11 size classes, isobaric protein sample labeling, and MS/MS detection. The transcriptomes and proteomes allow us to address: (1) the distinctiveness of each stage of anther progression; (2) the reuse of developmental modules at different stages; (3) the possibility that anthers store mRNA without translation; and (4) the likely cell type specific processes facilitating cell type specification and initial cell differentiation in maize anthers.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Transcriptome analysis</h3><div id="sec-3" class="subsection"><h4>Plant samples:</h4><p id="p-12">Except as noted, all anthers were collected from the W23 <em>bz2</em> (defect in vacuolar anthocyanin sequestration) inbred maize line. Anthers were dissected and staged by length using a micrometer. Anthers of the target length (±0.03 mm) were pooled from one tassel to generate a biological replicate for each size class. Cell counts of anthers at successive stages were obtained from confocal images of entire anther lobes. Images were collected using a Leica SP5 confocal microscope after propidium iodide staining as outlined by <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kelliher and Walbot (2011)</a>. A representative view was chosen near the center of each scanned anther, and the numbers and dimensions of each cell type were measured using Volocity software (PerkinElmer, Waltham, MA). The cell counts were extrapolated to estimate the total numbers for that organ by scaling to whole anther length. A minimum of three anthers of each length was evaluated. Volumes were measured with the line tool in Volocity using an average of the three dimensions (X, Y, Z) for each of three different cells for each cell type. Anthers of similar length were then classified into size bins and average values were calculated for both cell number and cell volume within those bins; each bin contained a minimum of four individually analyzed anthers. The results obtained with these methods are very similar to those reported by <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Kelliher and Walbot (2011)</a>, with a slight change to counting procedure and the addition of several younger stages.</p></div><div id="sec-4" class="subsection"><h4>RNA extraction, cDNA preparation, and qRT-PCR:</h4><p id="p-13">Total RNA from pools of staged, whole anthers was extracted using the Trizol-Plus procedure (Life Technologies, Grand Island, NY). DNase-treated RNA samples were evaluated on the 2010 Agilent Bioanalyzer (Santa Clara, CA) before being reverse-transcribed by the SuperScript III First-Strand Synthesis System (Life Technologies). The qRT-PCR data were obtained on an Opticon 2 (BioRad, Hercules, CA) using primers designed by primer3 software (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Untergasser <em>et al.</em> 2012</a>); primers are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS4.pdf">Table S4</a>. Data were analyzed using PCR Miner software (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhao and Fernald 2005</a>). For laser microdissected samples, 0.7 mm anthers from four plants were fixed, sectioned, and laser-microdissected to recover cell types as described previously (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>).</p></div><div id="sec-5" class="subsection"><h4>RNA in situ hybridization:</h4><p id="p-14">Tassels were vacuum-infiltrated on ice in 50% ethanol, 3.7% formaldehyde, 5% glacial acetic acid, 0.5% Triton X-100, and 1% DMSO, and then fixed overnight. Tassels were dehydrated through an ethanol series, cleared with Histoclear (National Diagnostics, Atlanta, GA), and then embedded in Paraplast Plus (Sigma-Aldrich, St, Louis, MO) following the protocols of <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jackson <em>et al.</em> (1994)</a>. Hybridizations were performed with probes transcribed using the DIG RNA Labeling Kit (T7/SP6) (Roche, <a href="http://www.roche-applied-science.com">http://www.roche-applied-science.com</a>). Sense and antisense probes were synthesized from PCR fragments amplified from cDNA clones obtained from the University of Arizona maize cDNA collection (<a href="http://maizecdna.org/">http://maizecdna.org/</a>). Probes were produced using T7 and SP6 polymerases directly from the cDNA clone; no gene-specific primers were used. RNA <em>in situ</em> hybridizations were performed on tassel sections containing anthers from 0.15 to 1 mm, as described previously (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Schiefthaler <em>et al.</em> 1999</a>). Estimation of anther length from <em>in situ</em> images is based on tissue and cellular morphology.</p></div><div id="sec-6" class="subsection"><h4>Microarray hybridization:</h4><p id="p-15">cDNA samples were amplified and labeled; the resulting cRNA was hybridized to a 4x44K format two-color microarray (Agilent Part number: G2519F; Design ID: 016047) in a balanced dye swap design as described previously (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>). The samples were arranged into three experimental loops. The first loop compared wild-type 0.15 mm, 0.25 mm, 0.4 mm, and 0.4 mm <em>mac1</em> (introgressed into the W23 inbred for five generations) anthers. In the second loop, wild-type 0.7 mm whole anther, AR cells, and inner somatic layers (middle layer and tapetum) were compared, with the outer somatic layers (epidermis and endothecium) to whole anther comparison as an out group. The third loop compared 0.2 mm wild-type with 0.2 mm <em>mac1</em>. The 1.0 mm wild-type data were from a separate experiment but used the same array and sample preparation procedures. Typically, a minimum of two replicates and up to four biological replicates were used. In cases in which two biological replicates were utilized, each was hybridized in duplicate for a total of four hybridization assessments.</p></div><div id="sec-7" class="subsection"><h4>Data analysis:</h4><p id="p-16">Slides were scanned and data were processed as described previously (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nan <em>et al.</em> 2011</a>). The resulting median foreground values for the red and green channels were normalized in two steps using the LIMMA package in R as follows: “within arrays” using the LOWESS method and “between arrays” using the quartile method. Probes with expression values more than 3.0 SD above the average foreground of the array’s negative controls were considered ON, resulting in an estimated false detection rate of 0.13%. The small number of probes with &lt;75% of the biological replicate measurements scored as ON were excluded from further analysis. Significance for differential expression was set at 1.5-fold (log2 ± 0.58) with a p-value ≤ 0.05 for evaluation of data within a single experimental loop.</p></div><div id="sec-8" class="subsection"><h4>Metabolic pathways analysis:</h4><p id="p-17">Transcripts constitutively expressed in whole anthers from the 0.15 to 1.0 mm stages were selected and log2 ratios of average intensities between each pair of consecutive stages were mapped in the Pathway Tools Omic Viewer (SRI, Menlo Park, CA) using the MaizeCyc database version 2.1 (<a href="http://pathway.gramene.org/gramene/maizecyc.shtml">http://pathway.gramene.org/gramene/maizecyc.shtml</a>). The list of pathways with differentially expressed genes was extracted using a two-fold cutoff for significance. The average intensities were extracted for all the constitutively expressed transcripts belonging to the hormone-associated pathways identified with at least one transcript meeting the two-fold cutoff. For the laser-microdissected cell type data, we identified pathways <em>ad hoc</em> that had high transcript enrichments in at least one tassel lobe zone and then calculated the percentage of differentially regulated genes for each sample.</p></div></div><div id="sec-9" class="subsection"><h3>Proteome analysis</h3><div id="sec-10" class="subsection"><h4>Anther samples and isobaric labeling:</h4><p id="p-18">Duplicate biological replicate samples of 0.4, 0.7, and 1.0 mm wild-type anthers were collected. Total protein was extracted as described by <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> (2012a)</a> and <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casati <em>et al.</em> (2005)</a> and quantified using a Bradford assay. Samples of 25 μg were loaded onto NuPAGE Novex Bis-Tris Precast gel cassette acrylamide gels (Life Technologies) along with 10 μl Novex Sharp Standard size standard (Life Technologies) in a separate lane and electrophoresed for 35 min at constant 200 V. Using a methanol-cleaned scalpel, 11 size-fractionated samples were collected from each lane (3.5–10 kD, 10–15 kD, 15–20 kD, 20–30 kD, 30–40 kD, 40–50 kD, 50–60 kD, 60–80 kD, 80–110 kD, 110–160 kD, 160–260 kD). The three lowest molecular weight fractions were digested in-gel using LysC (Promega, Madison, WI) as described by <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> (2012a)</a> and the other fractions were digested with Trypsin/LysC (Promega). After peptide extraction, each pool was dried to completion, after which peptides were labeled with amine-reactive tags (TMT) (Thermo Scientific, Rockford, IL) in a sixplex fashion with the following assignments: 0.4 mm replicate one, isobaric tag 130; 0.4 mm replicate two, isobaric tag 131; 0.7 mm replicate one, isobaric tag 126; 0.7 mm replicate two, isobaric tag 127; 1.0 mm replicate one, isobaric tag 128; and 1.0 mm replicate two, isobaric tag 129. After isobaric tagging, each fraction was pooled 1:1 and tested in a pilot analysis; the pilot assays were used to devise appropriate multiplex pooling mixtures.</p></div><div id="sec-11" class="subsection"><h4>Peptide sequencing:</h4><p id="p-19">The peptide pool of each representative fraction was then loaded on a nano-Acquity UPLC (Waters, Milford, MA) using a self-packed nano-UPLC column 25 cm in length with an internal diameter of 100 µM and a C18 matrix. Gradient lengths were 120 min for each fraction and the source was a Proxeon source directly in front of a LTQ Orbitrap Velos mass spectrometer (Thermo Fisher, San Jose, CA). The data acquisition was such that the survey scan was performed in the orbitrap mass analyzer at a resolution of 60,000 with 750,000 ions, and the MS/MS was performed using HCD for the top eight most intense precursor ions where the MSn target ion value was 75,000. RAW data were converted to mzXML format and searched on a Sorcerer platform using a project-generated maize database incorporating all six potential reading frames for each gene. Using a reverse decoy database approach, the dataset was then filtered to a 1% false discovery rate (FDR); approximately 2000 proteins were identified with high confidence. Using a virtual translation of all maize genes from the project-generated protein database, peptides were matched to the genome using Scaffold4 software (Proteome Software Inc., Portland, OR).</p></div><div id="sec-12" class="subsection"><h4>Data analysis:</h4><p id="p-20">To assess protein abundances quantitatively, only the constitutively present proteins were analyzed; this procedure is required from our experimental design in which each sample type (containing a particular isobaric tag) can be set as the reference for a comparison, but the software can only analyze quantitative values using a ratiometric algorithm. Therefore, proteins absent in any sample are excluded from the analysis and no analysis of stage-specific proteins is available. In total “change” proteins were identified at FDR of 1% and normalized for quantitative analysis using the Scaffold4 software package using log2 values to compare between sample types. Approximately 80% of the proteins identified by mass spectrometry matched an annotated maize gene; for ∼67% of these, there was a probe on the 4x44K Agilent microarrays, resulting in a total of 1303 protein-to-RNA comparisons.</p></div></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Anther ontogeny</h3><p id="p-21">Anthers are contained within the tassel, the terminal inflorescence at the top of the maize plant (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>); each tassel produces hundreds of spikelets, each containing two flowers (florets). Because large cohorts of florets are at the same developmental stage within a tassel, recovery of carefully staged anthers for microscopic or molecular analysis is straightforward. Similar to most flowering plants, maize initially produces perfect flowers with both reproductive organs; however, the stamens abort in the ear and the carpels abort in the tassel. Consequently, each tassel floret has three stamens but no carpel. Within a spikelet, the upper floret stamens develop approximately 1 day ahead of the lower floret. Each stamen consists of a slender filament connecting the terminal anther to the vegetative plant. At the initiation of meiosis, each anther consists of four equivalent lobes containing four rings of somatic cells encircling the meiocytes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Morphology, anatomical terms, and staging of maize anthers. (A) Major structures of a maize plant relevant to understanding anther ontogeny. The final panel also indicates the X-, Y-, and Z-axis designations used in confocal microscopic analysis of anther cells. The Y-axis is the length dimension used as a proxy for anther developmental stage. Abaxial and adaxial lobes of the anther are indicated in the transverse view, as are the intermicrosporangial stripes (IMS) 1 and 2 that separate the lobes. A color code for the anther lobe cell types is provided at the bottom of the figure. (B) Complete timeline of maize anther development from organ identity specification through pollen shed. The dotted box outlines the period corresponding to (C). Descriptors are provided for the key events in anther development across the entire timeline. The microspore stages are abbreviated: archesporial cells (AR); meiocytes (ME); quartet stage of microspores (Q); uninucleate microspore (UM); binucleate microspore (BM); and mature pollen (MP). (C) Tracings of anther lobes in transverse section after anther staging by length with the typical number of days of development since anther primordium inception are also indicated. Given the variable environment of plant growth in the field and greenhouse, days are approximate, but anther lengths are tightly correlated with developmental stage. The gray bar provides a guide to the key developmental events and the span of each event. The graph depicts the increase in cell numbers from 0.2 mm to 1.6 mm (germinal cell specification through entry into meiosis). Not included in the graph are the vasculature and connective tissue cells in the center of each anther, which constitute &#x223C;10% of all cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Morphology, anatomical terms, and staging of maize anthers. (A) Major structures of a maize plant relevant to understanding anther ontogeny. The final panel also indicates the X-, Y-, and Z-axis designations used in confocal microscopic analysis of anther cells. The Y-axis is the length dimension used as a proxy for anther developmental stage. Abaxial and adaxial lobes of the anther are indicated in the transverse view, as are the intermicrosporangial stripes (IMS) 1 and 2 that separate the lobes. A color code for the anther lobe cell types is provided at the bottom of the figure. (B) Complete timeline of maize anther development from organ identity specification through pollen shed. The dotted box outlines the period corresponding to (C). Descriptors are provided for the key events in anther development across the entire timeline. The microspore stages are abbreviated: archesporial cells (AR); meiocytes (ME); quartet stage of microspores (Q); uninucleate microspore (UM); binucleate microspore (BM); and mature pollen (MP). (C) Tracings of anther lobes in transverse section after anther staging by length with the typical number of days of development since anther primordium inception are also indicated. Given the variable environment of plant growth in the field and greenhouse, days are approximate, but anther lengths are tightly correlated with developmental stage. The gray bar provides a guide to the key developmental events and the span of each event. The graph depicts the increase in cell numbers from 0.2 mm to 1.6 mm (germinal cell specification through entry into meiosis). Not included in the graph are the vasculature and connective tissue cells in the center of each anther, which constitute &#x223C;10% of all cells.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F1.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/6/993/F1.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464469" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Morphology, anatomical terms, and staging of maize anthers. (A) Major structures of a maize plant relevant to understanding anther ontogeny. The final panel also indicates the X-, Y-, and Z-axis designations used in confocal microscopic analysis of anther cells. The Y-axis is the length dimension used as a proxy for anther developmental stage. Abaxial and adaxial lobes of the anther are indicated in the transverse view, as are the intermicrosporangial stripes (IMS) 1 and 2 that separate the lobes. A color code for the anther lobe cell types is provided at the bottom of the figure. (B) Complete timeline of maize anther development from organ identity specification through pollen shed. The dotted box outlines the period corresponding to (C). Descriptors are provided for the key events in anther development across the entire timeline. The microspore stages are abbreviated: archesporial cells (AR); meiocytes (ME); quartet stage of microspores (Q); uninucleate microspore (UM); binucleate microspore (BM); and mature pollen (MP). (C) Tracings of anther lobes in transverse section after anther staging by length with the typical number of days of development since anther primordium inception are also indicated. Given the variable environment of plant growth in the field and greenhouse, days are approximate, but anther lengths are tightly correlated with developmental stage. The gray bar provides a guide to the key developmental events and the span of each event. The graph depicts the increase in cell numbers from 0.2 mm to 1.6 mm (germinal cell specification through entry into meiosis). Not included in the graph are the vasculature and connective tissue cells in the center of each anther, which constitute ∼10% of all cells.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">As shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, an anther lobe primordium consists of only Layer1-derived (L1-d; presumptive epidermis) and Layer2-derived (L2-d; internal tissues) cells. Subsequent anatomical development is charted against anther length because this metric shows excellent correspondence to internal stages in maize anthers (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>). The first step in anther patterning is AR specification; in maize, this event requires a redox switch mediated by the MSCA1 glutaredoxin (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>). This process starts just after the 0.15 mm stage and a full column of AR cells is present in each anther lobe by 0.25 mm. The newly specified AR secrete MAC1, a dual-function ligand that suppresses AR proliferation but activates the single layer sheath of neighboring pluripotent L2-d cells to divide periclinally, a process that initiates while the AR column is still forming and is completed by 0.3 mm. Sister cells in the newly formed somatic bilayer acquire distinctive fates and cell shapes by the 0.35 mm stage: the subepidermal cells become endothecium (EN) and those adjacent to the AR become bipotent secondary parietal layer (SPL) cells (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012b</a>). After 3 days of within-layer proliferation, the final cell specification event is completed by 0.7 mm after the periclinal division of each SPL cell generates two daughters, a middle layer (ML) and tapetal (TAP) cell; the resulting lobes contain five tissues each composed of one cell type and arranged in a dartboard architecture (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>).</p><p id="p-24">Over the subsequent several days, rapid anticlinal cell division in all somatic cell types results in a longer and wider anther by the 1.0 mm stage (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). Each cell type has a characteristic cell shape and volume and the numbers of cells in each tissue are highly reproducible (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). After the 1.2 mm stage, division slows in all somatic cell types. From their inception the AR are the largest cells in the anther lobe; although specified first, their mitotic pace is much slower than the somatic cell types. From the initial column of two to five AR in each 0.2 mm anther lobe, a final number of ∼160 AR per lobe, haphazardly arranged in four columns each with ∼40 AR, is reached by 1.0 mm. AR mitosis finishes at 1.0 mm; over the subsequent several days these cells expand further and differentiate as pollen mother cells (PMC) in preparation for meiosis. In W23, the PMC are in prophase I by the 1.5 mm stage, by 2.0 mm in other inbreds, and meiotic stages are synchronous among all PMC in an anther over the following 6 days (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). All four meiotic products of a normal meiosis are viable, and their post-meiotic development involves two haploid mitotic divisions to generate the trinucleate pollen grain, containing two sperm nuclei encompassed within a large vegetative cell.</p></div><div id="sec-15" class="subsection"><h3>Transcriptome profiling of staged anthers</h3><p id="p-25">Three key stages were selected for transcriptome profiling: 0.4 mm during a period of rapid clonal proliferation after the generation of EN and SPL completes the second cell fate specification step; 0.7 mm after the SPL divisions complete the third cell fate specification event forming the ML and TAP; and 1.0 mm when AR mitotic divisions finish and the somatic and germinal cells have differentiated into stereotypic shapes and volumes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS2.zip">Table S2</a>). For comparison to the earliest steps in primordium differentiation, two earlier anther stages were included in the 0.4 mm experimental loop: 0.15 mm anthers prior to differentiation of the L2-d cells into AR and somatic tissues and 0.25 mm anthers in which AR occupy a column from lobe tip to base and the EN and SPL cells are being generated by periclinal division of the L2-d cells adjacent to the AR (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kelliher and Walbot 2014</a>). Analysis of numerous maize male-sterile mutants indicates that defects in somatic cell fate specification prior to 0.4 mm as well as at the 0.4 and 0.7 mm stages result in meiotic arrest (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Timofejeva <em>et al.</em> 2013</a>).</p><p id="p-26"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS2.zip">Table S2</a> reports two transcriptome data analyses. The Venn diagrams in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a> illustrate that at the early stages of anther development from 0.15 to 1.0 mm the transcriptome is very complex, with 21,959 transcripts commonly expressed among the 0.15 to 0.4 mm stages and 21,125 common transcripts among the 0.4 to 1.0 mm stages, plus numerous stage-specific or stage-restricted transcripts. With the exception of the 0.4 mm stage, there is considerable overlap in transcript accumulation between contiguous stages. The distinctiveness of the 0.4 mm stage is highlighted in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>, which quantifies the major classes of transcript gain or loss over the developmental progression. The number of stage-specific transcripts generally increases with developmental stage: 0.15 mm (N = 208, pluripotent cells present), 0.25 mm (N = 259, AR present and EN and SPL forming), 0.7 mm (N = 327, all lobe cell types are present), and 1.0 mm (N = 2614, when cell differentiation is advanced). The 0.4 mm stage after the three-layered somatic niche is established contains surprisingly few (11) stage-specific transcripts; this may reflect the persistence of transcripts from the previous stage and the absence of cell type specific differentiation transcripts, many of which are likely expressed after this stage.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Major patterns of transcript expression in immature maize anthers. (A) Overlap of ON expression between stages 0.15 mm, 0.25 mm, and 0.4 mm. (B) Overlap of ON expression between stages 0.4 mm, 0.7 mm, and 1.0 mm. (C) Graphical analysis of transcript gain and loss from the 0.15 mm to 1 mm stage. Counts above zero represent transcripts present in that stage classified by their relation to the previous stage (newly expressed or shared with previous stage), but stage-specific transcripts are separately categorized. Counts below zero indicate transcripts that were present in the previous stage but were not detected in the current stage. The previous category of these transcripts is also indicated by the similar but lighter color pattern." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Major patterns of transcript expression in immature maize anthers. (A) Overlap of ON expression between stages 0.15 mm, 0.25 mm, and 0.4 mm. (B) Overlap of ON expression between stages 0.4 mm, 0.7 mm, and 1.0 mm. (C) Graphical analysis of transcript gain and loss from the 0.15 mm to 1 mm stage. Counts above zero represent transcripts present in that stage classified by their relation to the previous stage (newly expressed or shared with previous stage), but stage-specific transcripts are separately categorized. Counts below zero indicate transcripts that were present in the previous stage but were not detected in the current stage. The previous category of these transcripts is also indicated by the similar but lighter color pattern.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F2.medium.gif" width="182" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/6/993/F2.medium.gif" width="182" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Major patterns of transcript expression in immature maize anthers. (A) Overlap of ON expression between stages 0.15 mm, 0.25 mm, and 0.4 mm. (B) Overlap of ON expression between stages 0.4 mm, 0.7 mm, and 1.0 mm. (C) Graphical analysis of transcript gain and loss from the 0.15 mm to 1 mm stage. Counts above zero represent transcripts present in that stage classified by their relation to the previous stage (newly expressed or shared with previous stage), but stage-specific transcripts are separately categorized. Counts below zero indicate transcripts that were present in the previous stage but were not detected in the current stage. The previous category of these transcripts is also indicated by the similar but lighter color pattern.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Each developmental stage shares many transcripts with the previous stage, with a low proportion of transcript disappearance (negative numbers below zero) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>), except at 0.4 mm. At this developmental stage when one L2-d cell layer (the SPL) is still bipotent, there are only 11 stage-specific transcripts and thousands of transcripts expressed at 0.25 mm are no longer detectable by microarray hybridization (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). This “trough” pattern at 0.4 mm is just one example of gene cohorts turned ON or OFF at key stages of development, but it is of particular interest biologically because of the lack of cell specification events at this stage and the magnitude of the OFF class. In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, the microarray intensity levels are shown for transcripts exhibiting the trough pattern at four of the five stages studied (all but 0.25 mm). For each pattern, a large number of the intensities before or after the trough are above the median level for the experiment (indicated with a dotted line): 39 of 155 at 0.15 mm; 551 of 1340 at 0.4 mm; 263 of 288 at 0.7 mm; and 422 of 524 at 1.0 mm (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS3.zip">Table S3</a>). Gene Ontology (GO) terms were used to classify these transcripts into categories. These GO assignments further demonstrate the dynamic nature of the anther transcriptome and the involvement of diverse cellular processes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Microarray intensities for four clusters of genes with detectable transcript in all but one stage. Clusters of transcripts shown are those not detected only at 0.15 mm (A), 0.4 mm (B), 0.7 mm (C), or 1.0 mm (D). To the right of each expression pattern is a histogram of the GO term counts associated with the transcripts after removal of any transcripts lacking GO terms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Microarray intensities for four clusters of genes with detectable transcript in all but one stage. Clusters of transcripts shown are those not detected only at 0.15 mm (A), 0.4 mm (B), 0.7 mm (C), or 1.0 mm (D). To the right of each expression pattern is a histogram of the GO term counts associated with the transcripts after removal of any transcripts lacking GO terms.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F3.medium.gif" width="242" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/6/993/F3.medium.gif" width="242" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464561" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Microarray intensities for four clusters of genes with detectable transcript in all but one stage. Clusters of transcripts shown are those not detected only at 0.15 mm (A), 0.4 mm (B), 0.7 mm (C), or 1.0 mm (D). To the right of each expression pattern is a histogram of the GO term counts associated with the transcripts after removal of any transcripts lacking GO terms.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">To assess transcript classification by microarray hybridization, qRT-PCR was used. Constitutive ON and constitutive OFF patterns were validated for 7/7 and 2/3 tests, respectively. Seven gene transcripts exhibiting the trough pattern at 0.4 mm were examined; only one showed the matching expression pattern and two transcripts were scored as exceedingly rare at 0.4 mm (C<sub>t</sub> value greater than 31) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS4.pdf">Table S4</a>); the other four were scored as present by qRT-PCR. Our array analysis sets a relatively high background threshold (three-times the SD of the non-hybridizing control probes) to maintain an FDR of 0.13%. These criteria mean that probes with very low expression values can be scored as OFF. Furthermore, some array probes may exhibit poor hybridization behavior, resulting in a false negative classification. To reexamine the nature of the trough at 0.4 mm, we evaluated expression of this gene set at the preceding stage. Of the 5450 transcripts missing specifically from the 0.4 mm stage, 5444 (99%) were below the median level of expression at 0.25 mm and 3321 (61%) were in the lowest quartile. Therefore, most transcripts “missing” at 0.4 mm were low or very low abundance transcripts at the previous stage and then decreased below the quantitative cutoff value in microarray hybridization.</p><p id="p-31">Are the transcripts absent at 0.4 mm markers for anther primordia that are permanently downregulated after the first round of differentiation? This does not appear to be the case because most (3523, 65%) are re-expressed in the subsequent 0.7 mm anther stage (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Some of the missing transcripts may represent genes responsible for executing layer-adding periclinal divisions and/or cell fate establishment. These developmental events are prevalent in 0.15 mm, 0.25 mm, and 0.7 mm anthers; however, at 0.4 mm, only within-layer anticlinal divisions occur, contributing to anther elongation (L-anticlinal divisions) and increases in girth (G-anticlinal divisions). Future research can address whether the decrease in transcript abundances at 0.4 mm reflects transient transcriptional repression or stage-specific mRNA degradation.</p></div><div id="sec-16" class="subsection"><h3>Proteomics to microarray comparison</h3><p id="p-32">To investigate if high-abundance transcripts observed in the microarray analysis would be translated at the same stage, anthers of 0.4 mm, 0.7 mm, and 1.0 mm were collected for protein identification by mass spectrometry. In total, 1967 proteins were identified in all three of these stages with an FDR of 1%. The use of TMT-sixplex isobaric tags allowed for direct, quantitative comparison between sample types. Two technical issues constrained the depth of proteome analysis: duration of the gradients during data acquisition and the use of TMT isobaric tags, which can result in some protein loss during the tagging procedure. Given that the total number of proteins identified is approximately 10% of the constitutively expressed transcripts, we conclude that this proteome analysis will only address abundant proteins.</p><p id="p-33">To evaluate the protein data in relation to the microarray results, log2 values of the 1303 proteins identified for which there was a corresponding gene probe on the microarray were compared in all pairwise combinations of developmental stages (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). If the hypothesis that large fold-changes in RNA between stages are accompanied by large changes in protein were true, then we would expect a linear relationship when protein values were plotted against the microarray values. Instead, we observe in all three comparisons that the relationship between transcript changes and proteins is much less predictable. This is also true when the RNA and protein data are binned into deciles and then compared (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS1.pdf">Figure S1</a>). In <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS1.pdf">Figure S1</a>, both protein and microarray abundances are represented as circles scaled to relative abundances of data within each decile of the respective data sets. The relatively uniform distribution of the data agrees with the representation in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> that transcript abundance is not a good indicator of protein abundance at a given developmental stage.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein abundance ratios compared to microarray intensity ratios. For the two stages indicated, log2 ratios of the protein abundance are plotted against the ratios of the microarray intensities for the 1303 gene types on both platforms at all three stages. (A) Comparing 0.7 mm to 0.4 mm. (B) Comparing 1.0 mm to 0.7 mm. (C) Comparing 1.0 mm to 0.4 mm. The non-linearity of the relationship between the two datasets emphasizes that a change in transcript abundance between two developmental stages (often days apart) is not necessarily associated with a similar change in the corresponding protein over that same interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Protein abundance ratios compared to microarray intensity ratios. For the two stages indicated, log2 ratios of the protein abundance are plotted against the ratios of the microarray intensities for the 1303 gene types on both platforms at all three stages. (A) Comparing 0.7 mm to 0.4 mm. (B) Comparing 1.0 mm to 0.7 mm. (C) Comparing 1.0 mm to 0.4 mm. The non-linearity of the relationship between the two datasets emphasizes that a change in transcript abundance between two developmental stages (often days apart) is not necessarily associated with a similar change in the corresponding protein over that same interval.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F4.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/6/993/F4.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464520" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Protein abundance ratios compared to microarray intensity ratios. For the two stages indicated, log2 ratios of the protein abundance are plotted against the ratios of the microarray intensities for the 1303 gene types on both platforms at all three stages. (A) Comparing 0.7 mm to 0.4 mm. (B) Comparing 1.0 mm to 0.7 mm. (C) Comparing 1.0 mm to 0.4 mm. The non-linearity of the relationship between the two datasets emphasizes that a change in transcript abundance between two developmental stages (often days apart) is not necessarily associated with a similar change in the corresponding protein over that same interval.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">A previous proteomics data set generated using similar methods analyzed proteins below 20 kD in meiotic anthers at stages 1.5 mm and 2.0 mm (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> 2012a</a>). To determine how dynamic small protein types are, we pooled these size classes (N = 226 proteins) from our 0.4 mm, 0.7 mm, and 1.0 mm datasets to compare to the 424 small proteins identified for the combined 1.5 mm and 2.0 mm stages identified by <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> (2012a)</a> (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS2.pdf">Figure S2</a>). These 424 proteins were identified at the same 1% FDR and peptide confidence levels as the TMT-multiplexed data presented in this study, although more stringent criteria reported in the original <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> (2012a)</a> study yielded a list of 359 proteins. These early and later stages are strikingly different: only 19 proteins were found in common between the two data sets for proteins in the size classes between 3.5 and 20 kD (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS2.pdf">Figure S2</a>). When all the proteins identified in the TMT-tagged proteome were included in the comparison, there were 120 proteins in common with the list of <a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> (2012a)</a>. The low-molecular-weight proteins identified may vary between these data sets because of a reduction in the number of proteins post-translationally cleaved after the onset of meiosis. Overall, the results indicate that the proteome for early anther development is very different from that in meiotic anthers.</p></div><div id="sec-17" class="subsection"><h3>Comparison between stages</h3><p id="p-36">In addition to the ON and OFF categories, we also investigated transcripts that are constitutively expressed at all stages (N = 22,257) but are differentially expressed between at least two developmental stages using a two-fold cutoff for significance. Few transcripts met these criteria from 0.15 mm to 0.4 mm followed by a large increase in such differentially regulated transcripts at subsequent stages. There were 144 transcripts (74 UP, 70 DOWN) between 0.15 mm and 0.25 mm and 109 transcripts between 0.25 mm and 0.4 mm (58 UP, 51 DOWN). In transitioning from the 0.4 mm to 0.7 mm stages, 810 transcripts were differentially expressed (427 UP, 383 DOWN); between 0.7 mm and 1.0 mm, 5339 transcripts (2071 UP, 3268 DOWN) were identified (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). We hypothesize that the large increase in significantly differentially regulated transcripts at 0.7 mm and 1.0 mm reflects cell differentiation of the EPI, EN, and AR by 0.7 mm and by all five lobe cell types at 1.0 mm.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pathway analysis of constitutive transcripts that are differentially regulated with the significance set at two-fold of normalized average intensities. (A) Number of differentially regulated transcripts compared between two consecutive anther stages: UP, increased by two-fold from the previous stage, and DOWN, decreased by two-fold from previous stage. Newly indicates not found in the previous stage. (B&#x2013;G) Log2 ratio of differentially expressed transcripts in GA12 biosynthesis (B), gibberellin inactivation (C), brassinosteroid biosynthesis II (D), jasmonic acid biosynthesis (E), cytokinin-glucoside biosynthesis (F), and cytokinin degradation (G) pathways." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pathway analysis of constitutive transcripts that are differentially regulated with the significance set at two-fold of normalized average intensities. (A) Number of differentially regulated transcripts compared between two consecutive anther stages: UP, increased by two-fold from the previous stage, and DOWN, decreased by two-fold from previous stage. Newly indicates not found in the previous stage. (B&#x2013;G) Log2 ratio of differentially expressed transcripts in GA12 biosynthesis (B), gibberellin inactivation (C), brassinosteroid biosynthesis II (D), jasmonic acid biosynthesis (E), cytokinin-glucoside biosynthesis (F), and cytokinin degradation (G) pathways.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F5.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/6/993/F5.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464492" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Pathway analysis of constitutive transcripts that are differentially regulated with the significance set at two-fold of normalized average intensities. (A) Number of differentially regulated transcripts compared between two consecutive anther stages: UP, increased by two-fold from the previous stage, and DOWN, decreased by two-fold from previous stage. Newly indicates not found in the previous stage. (B–G) Log2 ratio of differentially expressed transcripts in GA12 biosynthesis (B), gibberellin inactivation (C), brassinosteroid biosynthesis II (D), jasmonic acid biosynthesis (E), cytokinin-glucoside biosynthesis (F), and cytokinin degradation (G) pathways.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">To gain insight into the types of processes contained within the differentially regulated gene sets, transcripts were mapped to metabolic pathways. The current MaizeCyc database permits evaluation of 4049 genes organized into 479 pathways. All pathways with a two-fold upregulated or downregulated gene were chosen for closer inspection. Pathways well-represented in all four comparisons (0.25/0.15, 0.4/0.25, 0.7/0.4, and 1.0/0.7) include carbohydrate, amino acid, and lipid/fatty acid metabolism (both biosynthesis and degradation). This result indicates that there is ongoing turnover of basic metabolites necessary to support the fast pace of growth and development in immature anthers and that there is unexpectedly dynamic gene regulation of these core biochemical pathways.</p><p id="p-39">Anthers are unique organs because there is no meristem organizing cell production. Organ growth and cellular differentiation are either organized cell-autonomously or through perception of signals delivered through the vascular tissue or generated by one lobe cell type and perceived by another. Hormone production within each anther, with specific hormones playing larger roles at specific stages, is a logical hypothesis to explain growth coordination within anthers (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mutasa-Göttgens and Hedden 2009</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ye <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Depuydt and Hardtke 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Song <em>et al.</em> 2013</a>). Although constitutively expressed, a number of hormone-related genes are differentially regulated during pre-meiotic anther development (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS5.pdf">Table S5</a>). If the gene expression patterns are predictive of hormone levels, then the data indicate that there are substantial changes in hormone biology. Supporting this idea, among the 180 steps analyzed in the hormone-associated pathways, 18 correspond to a protein detected in the proteomics data set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS5.pdf">Table S5</a>) indicating that at least some of the steps evaluated are present as proteins. Comparing the 0.15 mm to 0.25 mm stages, two gibberellin A<sub>12</sub> (GA<sub>12</sub>) biosynthesis gene transcripts (GRMZM2G059308 and GRMZM2G093195) are upregulated (UP) and three gibberellin inactivation transcripts (GRMZM2G022679, GRMZM2G051619, and GRMZM2G155686) are downregulated (DOWN). The opposite pattern is present at the 0.7 mm and 1.0 mm stages (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, B and C</a>). These observations suggest the hypothesis that GA is involved in cell fate specification events—AR, EN, and SPL patterning—but is less important during anther anticlinal cell proliferation.</p><p id="p-40">In contrast, transcript levels of genes in the brassinosteroid biosynthesis II pathway were UP at both the 0.4 mm stage (GRMZM2G052336 and GRMZM2G107076) and the 1.0 mm stage (GRMZM2G040397, GRMZM2G108570, and GRMZM2G468439) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>), whereas the majority in the jasmonic acid biosynthesis pathway (GRMZM2G072653, GRMZM2G077316, GRMZM2G087192, GRMZM2G109130, GRMZM2G132903, GRMZM2G151087, GRMZM2G156712, and GRMZM2G156861) were UP only at the 1.0 mm stage (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>). These results also suggest that specific hormones may play instructive roles during landmark developmental events.</p><p id="p-41">Genes involved in cytokinin inactivation through formation of cytokinin-O-glucoside (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5F</a>) and cytokinin degradation (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5G</a>) showed complex yet interesting patterns. Two were DOWN at both the 0.7 mm stage and the 1.0 mm stage (cytokinin-O-glucoside biosynthesis: GRMZM2G363554; cytokinin degradation: GRMZM2G146644). Two were DOWN at the 0.7 mm stage and then were re-expressed at the 1.0 mm stage (cytokinin degradation: GRMZM2G348452 and GRMZM5G17173), whereas another one was UP at the 0.7 mm stage (cytokinin-O-glucoside biosynthesis: GRMZM2G004858) and then went DOWN at the 1.0 mm stage. Finally, a total of five genes in the cytokinin-O-glucoside biosynthesis pathway (GRMZM2G041699, GRMZM2G082249, GRMZM2G113794, GRMZM2G160523, and GRMZM135390) were UP only at the 1.0 mm stage. Collectively, these data suggest that cytokinin degradation and sequestration may be important in the modulation of cell division pace and hint at cell type specificity: AR divisions stop by 1.0 mm and the pace of somatic cell mitosis slows after 1.2 mm (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-42">Because the tassel develops within a whorl of immature leaves and maize anthers are encased in several outer organs, anthers grow in darkness during the developmental stages considered here. Therefore, it was surprising to find that anthers express genes involved in photosynthesis starting at the 0.7 mm stage. Examples include photosynthesis light reaction components (GRMZM2G080107 and GRMZM2G168143), Calvin-Benson-Bassham cycle enzymes (<em>Ssu1</em>-GRMZM2G098520 and <em>Gpa1</em>-GRMZM2G337113), CO<sub>2</sub> fixation (<em>Pep1</em>-GRMZM2G083841), and chlorophyllide, a biosynthesis factor (GRMZM2G084958 and GRMZM2G073351).</p></div><div id="sec-18" class="subsection"><h3>Transcript profiling of transcription factors and meiotic genes</h3><p id="p-43">To study whether master regulators control the transcriptional changes during anther development, we examined the expression profiles of known transcription factors (TFs). There are 2298 TF loci in maize and, as a result of alternative splicing, these encode 3307 predicted TFs classified into 56 families (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2012</a>). Among these, 1208 TF loci (52.6%) are represented by 1901 probes on the microarray. The distribution is a fair representation of each family (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>); 813 TFs represented by 1041 probes are constitutively expressed from 0.15 to 1.0 mm. During the initial stages of anther development, we found few changes in specific TF abundance (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS6.pdf">Table S6</a>). In contrast, 178 TF transcripts were increased (including 151 transcripts that are at least two-fold more abundant and 26 transcripts that are newly detectable) at 1.0 mm compared to 0.7 mm; 103 transcripts were decreased by more than two-fold at 1.0 mm (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS6.pdf">Table S6</a>) as well. These findings are consistent with the observation that there are both more transcript types and higher expression levels when anthers are approaching meiosis at 1.0 mm (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>), suggesting that large-scale transcriptome reprogramming is important for anther differentiation at this stage.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evaluation of transcription factor microarray data. (A) 3307 predicted TFs classified into 56 families (left pie chart); 1208 of them were represented on the microarray (right pie chart) in approximately equivalent proportions. (B) Transcription factor abundance changes at each stage. ON and OFF indicate detectability at that stage. UP and DOWN classifications for a transcript were determined relative to the previous stage using a two-fold minimum fold change." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evaluation of transcription factor microarray data. (A) 3307 predicted TFs classified into 56 families (left pie chart); 1208 of them were represented on the microarray (right pie chart) in approximately equivalent proportions. (B) Transcription factor abundance changes at each stage. ON and OFF indicate detectability at that stage. UP and DOWN classifications for a transcript were determined relative to the previous stage using a two-fold minimum fold change.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F6.medium.gif" width="440" height="367"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/6/993/F6.medium.gif" width="440" height="367"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Evaluation of transcription factor microarray data. (A) 3307 predicted TFs classified into 56 families (left pie chart); 1208 of them were represented on the microarray (right pie chart) in approximately equivalent proportions. (B) Transcription factor abundance changes at each stage. ON and OFF indicate detectability at that stage. UP and DOWN classifications for a transcript were determined relative to the previous stage using a two-fold minimum fold change.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Analysis of <em>mac1</em> mutants (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012b</a>) demonstrated that successful PMC entry into meiosis does not require normal anther somatic tissues. A properly differentiated PMC is required for successful entry into meiosis, however, because differentiation failure in <em>ameiotic1</em> (<em>am1</em>) mutants (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Nan <em>et al.</em> 2011</a>) results in cytologically normal PMC conducting mitosis instead of meiosis. Based on differential expression in <em>ameiotic1</em>, 297 genes were associated with meiotic preparation. Surprisingly, 104 of these are expressed by the 0.35 mm stage in laser-microdissected AR (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kelliher and Walbot 2014</a>). To focus on factors with more strictly defined roles in meiosis, we classified genes either as having established roles in maize or as maize genes that are homologs of proven meiotic factors in other plants (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Five meiotic genes have been characterized in maize: <em>Ameiotic1</em>, a plant-specific gene controlling the switch from the mitotic to meiotic cycle (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pawlowski <em>et al.</em> 2009</a>); <em>Absence of first division1</em>, a meiosis-specific gene that controls meiotic chromosome formation and sister chromatid cohesion (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Golubovskaya <em>et al.</em> 2006</a>); <em>Poor homologous synapsis1</em>, a novel gene required for loading recombination machinery onto chromosomes and for homologous pairing (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pawlowski <em>et al.</em> 2004</a>); and two paralogs of <em>Rad51</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lin <em>et al.</em> 2006</a>) — <em>Rad51a1</em> and <em>Rad51a2</em> — that play roles in homologous recombination and the repair of double-strand breaks (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2007</a>). Except for <em>Ameiotic1</em>, which is not detectable in 0.15 mm and 0.4 mm anthers, these genes are expressed constitutively at all five stages (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464457/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464457/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Known maize meiotic genes with a specific function and maize homologs of well-established meiotic genes in other flowering plants</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">Based on well-studied genes in rice and <em>A. thaliana</em>, additional meiotic factors were analyzed, including the following: <em>SPO11</em>, which generates double-stranded breaks along meiotic chromosomes (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Yu <em>et al.</em> 2010</a>); <em>DMC1</em>, which repairs double-stranded breaks using the sister chromatid as template initiating a crossover event (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Couteau <em>et al.</em>1999</a>); and <em>ZYP1</em>, a central component of the synaptonemal complex (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2010b</a>). These genes are also constitutively expressed throughout the duration of early anther development (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Because meiosis-associated genes are expressed in 0.15 mm anther cells or the very first AR, if present, it is clear that very immature anthers activate expression, albeit at a low level, of some meiosis-specific genes. To test this hypothesis further, we utilized the list of 297 genes mis-regulated in <em>am1</em> mutants (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Nan <em>et al.</em> 2011</a>) to generate a list of 222 genes whose functions have been well-annotated; 77.5% of them (172/222) were expressed constitutively while only five gene transcripts were undetectable until 1.0 mm when meiotic preparation has been considered to start as AR differentiate into PMC (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS7.pdf">Table S7</a>). Our data strongly suggest that young anthers are preparing for meiosis as early as anther organ identity has been set.</p></div><div id="sec-19" class="subsection"><h3>Transcriptome profiling of outer somatic, inner somatic, and AR cell types from 0.7 mm anthers</h3><p id="p-48">By the 0.7 mm stage, all cell fate specification events are completed, and the resulting cells are dividing anticlinally to fuel anther growth in length and girth. By this stage the EPI, EN, and AR have distinctive cellular properties and dimensions. The ML and TAP cell types will soon acquire distinctive cellular dimensions and staining properties (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>), visual indicators of cellular differentiation. Three samples were collected via laser capture microdissection (LCM) to compare directly to whole anthers of the same stage: the outer somatic sample contains EPI plus EN cells (EPI/EN), the inner somatic sample contains two layers (ML/TAP), and the AR cells were dissected as the third sample (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Compared directly to equivalently staged whole anthers, there were 19,743 common transcript types, 6728 present in whole anthers but absent from the LCM samples, but only a few were found in LCM samples but not whole anthers. The majority of those absent from the three LCM samples had low expression in whole anthers; only 235 (3.5%) are above the first quartile and 14 (0.2%) are above the median, including two TFs. This suggests that many of these are actually present in lobes, but below detection using our methods. Adding the 14 transcripts above the median to the 811 downregulated in all three LCM samples, we can assign 825 transcripts to non-lobe tissues (vasculature and connective cells) (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>); these are the first presumptive markers assigned to these non-lobe cell tissues. Each LCM sample also exhibited a large set of significantly enriched transcripts (1075 in EPI/EN, 955 in ML/TAP, and 2045 in AR) (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>). Particularly for the differentiated cell types (EPI/EN and AR), the enriched transcript types point to processes likely to be characteristic of specific cell types.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of enriched transcripts and pathway analysis in pre-meiotic anther cell types. (A) RNA samples from three laser-microdissected cell types were compared to equivalently staged whole anthers, and the distribution of significantly enriched transcripts is shown. The combined EPI/EN collection (green) had 1075 enriched transcripts (top left lobe). The combined ML/TAP collection (blue) had 955 enriched transcripts, and the germinal AR cell collection (red) had 2045 enriched transcripts. The other lobes in the diagram show the number of enriched transcripts shared between two (top right, bottom left) or all three cell type collections (bottom right). In the middle of the diagram, 811 enriched plus 14 specific transcripts were assigned to the connective and vasculature tissues by virtue of being DOWN or OFF in all three cell collections compared to whole anthers. (B) The data were analyzed using the MaizeCyc database to identify molecular pathways that were enriched in at least one of the cell types; the criterion was that &gt;45% of the composite enzymes had to be enriched in a particular cell type. Sixteen pathways fit this criterion, including 11 enriched only in AR cells, five enriched only in EPI/EN cells, and one, cellulose synthesis (required for construction of the cell wall), enriched in both AR and EPI/EN collections. Zero pathways were identified in the ML/TAP sample, likely because at this stage these cell types are just starting to differentiate after birth from a periclinal division of the bipotent SPL. (C) The number of transcripts annotated as encoding RNA binding proteins, RNA helicases, ribonucleoproteins (RNPs), ribosomal proteins, or translation initiation and elongation factors was much higher in the germinal cells than either of the two somatic cell collections. (D) Distribution of specific classes of transcription factors in the anther lobe cell types is indicative of massive transcriptional reprogramming during differentiation. Overall, the three cell type collections had similar numbers of enriched TFs and similar numbers of homeodomain (HD), C2H2, and bHLH TFs. Myb TFs were enriched in AR and EPI/EN cells but not in the ML/TAP, whereas only AR cells were enriched in bZIPs and MADS box TFs. Seven WRKY transcription factors were enriched in somatic cell types but none in the AR. (E) Expression intensities of pluripotency markers during the early anther stages normalized to the mean. These 21 markers are gradually shut-off during early anther development as greater numbers of cells differentiate away from pluripotency. All but three of these markers show elevation at the 0.4 mm stage in mac1 as compared to wild-type, supporting morphological interpretations of that mutant as having an extended period of pluripotency concomitant with failure to properly differentiate somatic (EN/SPL) cell types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of enriched transcripts and pathway analysis in pre-meiotic anther cell types. (A) RNA samples from three laser-microdissected cell types were compared to equivalently staged whole anthers, and the distribution of significantly enriched transcripts is shown. The combined EPI/EN collection (green) had 1075 enriched transcripts (top left lobe). The combined ML/TAP collection (blue) had 955 enriched transcripts, and the germinal AR cell collection (red) had 2045 enriched transcripts. The other lobes in the diagram show the number of enriched transcripts shared between two (top right, bottom left) or all three cell type collections (bottom right). In the middle of the diagram, 811 enriched plus 14 specific transcripts were assigned to the connective and vasculature tissues by virtue of being DOWN or OFF in all three cell collections compared to whole anthers. (B) The data were analyzed using the MaizeCyc database to identify molecular pathways that were enriched in at least one of the cell types; the criterion was that &gt;45% of the composite enzymes had to be enriched in a particular cell type. Sixteen pathways fit this criterion, including 11 enriched only in AR cells, five enriched only in EPI/EN cells, and one, cellulose synthesis (required for construction of the cell wall), enriched in both AR and EPI/EN collections. Zero pathways were identified in the ML/TAP sample, likely because at this stage these cell types are just starting to differentiate after birth from a periclinal division of the bipotent SPL. (C) The number of transcripts annotated as encoding RNA binding proteins, RNA helicases, ribonucleoproteins (RNPs), ribosomal proteins, or translation initiation and elongation factors was much higher in the germinal cells than either of the two somatic cell collections. (D) Distribution of specific classes of transcription factors in the anther lobe cell types is indicative of massive transcriptional reprogramming during differentiation. Overall, the three cell type collections had similar numbers of enriched TFs and similar numbers of homeodomain (HD), C2H2, and bHLH TFs. Myb TFs were enriched in AR and EPI/EN cells but not in the ML/TAP, whereas only AR cells were enriched in bZIPs and MADS box TFs. Seven WRKY transcription factors were enriched in somatic cell types but none in the AR. (E) Expression intensities of pluripotency markers during the early anther stages normalized to the mean. These 21 markers are gradually shut-off during early anther development as greater numbers of cells differentiate away from pluripotency. All but three of these markers show elevation at the 0.4 mm stage in &lt;em&gt;mac1&lt;/em&gt; as compared to wild-type, supporting morphological interpretations of that mutant as having an extended period of pluripotency concomitant with failure to properly differentiate somatic (EN/SPL) cell types.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F7.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/6/993/F7.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464555" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Distribution of enriched transcripts and pathway analysis in pre-meiotic anther cell types. (A) RNA samples from three laser-microdissected cell types were compared to equivalently staged whole anthers, and the distribution of significantly enriched transcripts is shown. The combined EPI/EN collection (green) had 1075 enriched transcripts (top left lobe). The combined ML/TAP collection (blue) had 955 enriched transcripts, and the germinal AR cell collection (red) had 2045 enriched transcripts. The other lobes in the diagram show the number of enriched transcripts shared between two (top right, bottom left) or all three cell type collections (bottom right). In the middle of the diagram, 811 enriched plus 14 specific transcripts were assigned to the connective and vasculature tissues by virtue of being DOWN or OFF in all three cell collections compared to whole anthers. (B) The data were analyzed using the MaizeCyc database to identify molecular pathways that were enriched in at least one of the cell types; the criterion was that &gt;45% of the composite enzymes had to be enriched in a particular cell type. Sixteen pathways fit this criterion, including 11 enriched only in AR cells, five enriched only in EPI/EN cells, and one, cellulose synthesis (required for construction of the cell wall), enriched in both AR and EPI/EN collections. Zero pathways were identified in the ML/TAP sample, likely because at this stage these cell types are just starting to differentiate after birth from a periclinal division of the bipotent SPL. (C) The number of transcripts annotated as encoding RNA binding proteins, RNA helicases, ribonucleoproteins (RNPs), ribosomal proteins, or translation initiation and elongation factors was much higher in the germinal cells than either of the two somatic cell collections. (D) Distribution of specific classes of transcription factors in the anther lobe cell types is indicative of massive transcriptional reprogramming during differentiation. Overall, the three cell type collections had similar numbers of enriched TFs and similar numbers of homeodomain (HD), C2H2, and bHLH TFs. Myb TFs were enriched in AR and EPI/EN cells but not in the ML/TAP, whereas only AR cells were enriched in bZIPs and MADS box TFs. Seven WRKY transcription factors were enriched in somatic cell types but none in the AR. (E) Expression intensities of pluripotency markers during the early anther stages normalized to the mean. These 21 markers are gradually shut-off during early anther development as greater numbers of cells differentiate away from pluripotency. All but three of these markers show elevation at the 0.4 mm stage in <em>mac1</em> as compared to wild-type, supporting morphological interpretations of that mutant as having an extended period of pluripotency concomitant with failure to properly differentiate somatic (EN/SPL) cell types.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">To determine if transcripts exhibiting zone-specific hybridization could be useful markers for zones or cell types, RNA <em>in situ</em> hybridization was performed on four transcripts (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>) that each showed increased expression (fold-change &gt;1.5-fold; p-value &lt; 0.05) in the somatic cell types (ML/TAP, EPI/EN) and whole anthers compared to the AR. By <em>in situ</em> localization with ∼0.7 mm anthers, both GRMZM2G034638 and GRMZM2G148074 show expression in the ML/TAP and at the peripheral edge of the lobe arch, specifically in a restricted zone of the EPI (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figures 8, A and C</a>). Interestingly, the male sterile mutant <em>outer cell layer 4</em> (<em>ocl4</em>) produces an extra subepidermal layer in peripheral anther lobes, and <em>Ocl4</em> is specifically expressed in normal EPI in the zone adjacent to the defective double endothecial layer observed in mutants (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vernoud <em>et al.</em> 2009</a>). The authors proposed that OCL4 protein could have a direct effect on anther cell layer differentiation by signaling between the epidermis and endothecium. Because GRMZM2G148074 encodes an HD-ZIP protein likely to be a TF and GRMZM2G034638 encodes an EDG-6 sphingosine 1-phosphate receptor and is likely to be involved in signaling, the pattern of expression in both the peripheral EPI arch and the ML/TAP hints at an unexpected relationship in developmental control between the external and internal somatic layers. Confirmation of two additional markers for the EPI peripheral arch is very intriguing, because it suggests that these cells may have a particularly important role in coordinating anther development.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/6/993/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In situ candidates for cell-enriched markers. Genes were chosen for in situ analysis to verify patterns selected as somatic-specific in the 0.7 mm laser-microdissected data set. Antisense probes (A, C, E, G) for each gene and the corresponding sense probes (B, D, F, H) are shown. As predicted, all genes showed significant enrichment in the somatic cells in lobes of anthers &#x2265;0.7 mm. (A, B) GRMZM2G034638, a predicted EDG-6 sphingosine 1-phosphate receptor. (C, D) GRMZM2G148074, an HD-ZIP protein with predicted DNA binding and transcription regulator activity. (E, F) GRMZM2G096171, a CheY-like protein with predicted signal transducer and transcription regulator activity. (G, H) GRMZM2G036092, a predicted DNA binding protein with transcription regulator activity. White arrows point to cell types where the signal is detected. The white section of the scale bars in the lower left of each panel corresponds to 150 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1780570175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;In situ&lt;/em&gt; candidates for cell-enriched markers. Genes were chosen for &lt;em&gt;in situ&lt;/em&gt; analysis to verify patterns selected as somatic-specific in the 0.7 mm laser-microdissected data set. Antisense probes (A, C, E, G) for each gene and the corresponding sense probes (B, D, F, H) are shown. As predicted, all genes showed significant enrichment in the somatic cells in lobes of anthers &#x2265;0.7 mm. (A, B) GRMZM2G034638, a predicted EDG-6 sphingosine 1-phosphate receptor. (C, D) GRMZM2G148074, an HD-ZIP protein with predicted DNA binding and transcription regulator activity. (E, F) GRMZM2G096171, a CheY-like protein with predicted signal transducer and transcription regulator activity. (G, H) GRMZM2G036092, a predicted DNA binding protein with transcription regulator activity. White arrows point to cell types where the signal is detected. The white section of the scale bars in the lower left of each panel corresponds to 150 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/6/993/F8.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/6/993/F8.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/6/993/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/6/993/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-51" class="first-child"><em>In situ</em> candidates for cell-enriched markers. Genes were chosen for <em>in situ</em> analysis to verify patterns selected as somatic-specific in the 0.7 mm laser-microdissected data set. Antisense probes (A, C, E, G) for each gene and the corresponding sense probes (B, D, F, H) are shown. As predicted, all genes showed significant enrichment in the somatic cells in lobes of anthers ≥0.7 mm. (A, B) GRMZM2G034638, a predicted EDG-6 sphingosine 1-phosphate receptor. (C, D) GRMZM2G148074, an HD-ZIP protein with predicted DNA binding and transcription regulator activity. (E, F) GRMZM2G096171, a CheY-like protein with predicted signal transducer and transcription regulator activity. (G, H) GRMZM2G036092, a predicted DNA binding protein with transcription regulator activity. White arrows point to cell types where the signal is detected. The white section of the scale bars in the lower left of each panel corresponds to 150 µm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Two other transcripts, GRMZM2G096171 and GRMZM2G036092, encoding a CheY-like protein and a DNA-binding protein, respectively, show specific expression in the ML/TAP in ∼0.7 mm anthers and are thus presumptive markers for those anther cell types (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8, E and G</a>). Although there is similar expression in these cell types with the sense control probes (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8, F and H</a>), this is not unusual with maize anther genes (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kelliher and Walbot 2014</a>). Globally approximately 20% of maize genes tested in a microarray experiment express an antisense transcript (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Ma <em>et al.</em> 2006</a>). The prevalence of antisense transcription during cell differentiation may indicate a mechanistic role for antisense transcripts to control anther development.</p></div><div id="sec-20" class="subsection"><h3>Analysis of tissue-specific pathways</h3><p id="p-53">Mapping the transcriptome data from the LCM samples onto the MaizeCyc database highlighted the cellular processes and biosynthesis pathways enriched in each sample type (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a>). There were five pathways highly enriched in the EPI/EN sample, including JA biosynthesis and phospholipid and cellulose biosynthesis. The identification of JA biosynthesis is intriguing, given its key role earlier as a signal in carpel abortion in tassel spikelets (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acosta <em>et al.</em> 2009</a>). JA is a gas that could readily diffuse from the EPI to cause carpel elimination from each floret; therefore, we hypothesize that EPI is the site of JA synthesis in anthers. Application of 1 µM JA around developing tassels during cell fate specification (anthers 0.3–0.7 mm) can result in cessation of anther development (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Walbot and Skibbe 2010</a>), suggesting that JA may also regulate anther developmental processes as suggested by the pathway analysis of whole anther hormone-related genes. Membrane and cellulose production could support the massive proliferation of EPI and EN to build the axial orthogonal array (<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>), a cellular bilayer that provides structural support to the anther. There were no pathways enriched in the ML/TAP sample, likely because these cell types are just starting to differentiate (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>).</p><p id="p-54">The AR is the most distinctive of the three tissue samples. Hypoxia is a critical physiological factor in triggering AR specification in anther primordia (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>). The AR at 0.7 mm are enriched in transcripts for alternative energy pathways that perform non-mitochondrial energy generation and permit cellular survival in hypoxic conditions, as elaborated in a recent article focused on earlier stages of AR cell development (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Kelliher and Walbot 2014</a>). For example, there is enrichment for glycolysis I and II and NAD/NADH activity (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7B</a>). Hypoxic conditions are likely to persist in the center of the tightly packed anther lobes, given the high metabolic demand of actively dividing and growing somatic cells and the low oxygen availability in the air space around tassels (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>). The AR are also enriched in pathways required for translation: genes encoding ribosomal proteins, translation factors, and tRNA charging enzymes are all enriched (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7C</a>). Predicted RNA-binding proteins are also highly represented in the AR, and are a common transcript type in both plant and animal reproductive cells (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Extavour and Akam 2003</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Schmidt <em>et al.</em> 2011</a>) to manage translation and to sequester mRNAs for later use. Two additional pathways enriched in AR are related to myo-inositol metabolism and gibberellin biosynthesis; however, not much is known about these two pathways in anthers. Genes required for the latter are activated both in AR cells and in whole anthers at the 0.15 mm to 0.25 mm stage during AR specification.</p><p id="p-55">To investigate whether the larger class of trough transcripts at 0.4 mm in normal anthers was quenched as a prerequisite for appropriate cell differentiation, we reexamined the restoration of gene expression for this set at 0.7 mm in the laser-microdissected cell type dataset. Although 2.3% of genes remained OFF at 0.7 mm and 62% showed no significant enrichment by cell type, 20.8% of the genes were scored as ON in a highly restricted pattern (169 in AR, 204 in ML/TAP, and 235 in EPI/EN), whereas 14.8% showed refined expression in two of three zones (143 in ML/TAP + EPI/EN, 188 in ML/TAP + AR, and 102 in AR + EPI/EN) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS8.zip">Table S8</a>). These could be examples of spatial refinement of transcripts expressed in undifferentiated cells that later become zone or cell type specific. The existence of 102 genes common between AR + EPI/EN, nearly as many as the somatic-only shared set of ML/TAP + EPI/EN, supports the case that AR and EPI are the abaxial and adaxial epidermal sides of an ancestral “flat fertile frond” that rolled up, fused along the edge, and generated a circular organ, the modern anther.</p><p id="p-56">Refinement of cell functions can require both increased and decreased gene expression. To analyze this hypothesis, we examined regional specialization in TF representation at 0.7 mm. Ninety TFs are significantly enriched in only one of the three sample types: 31 in EPI/EN, 26 in ML/TAP, and 33 in AR (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7D</a>). Of particular interest, only the AR cells were enriched in MADS box and bZIP TFs (two of each class). The EN/EPI and AR samples each had six Myb TFs but the ML/TAP sample had none. In contrast, WRKY family TFs were only enriched in the ML/TAP and EPI/EN cell types (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7D</a>). These data indicate that in addition to stage specificity of TF representation, there is also spatially restricted expression of TF as would be required for many aspects of cell differentiation.</p></div><div id="sec-21" class="subsection"><h3>Transcriptome profiling of <em>mac1</em> and delays in fate acquisition</h3><p id="p-57">To further understand the 0.4 mm expression trough, we examined gene expression in 0.4 mm <em>mac1</em> mutant anthers compared to wild-type anthers of the same size. During early anther growth, the MAC1 protein is secreted by AR cells after germinal fate acquisition, stimulating a periclinal division in the single-layered encircling L2-d tissue to produce somatic EN and SPL daughters. In the <em>mac1</em> mutant, the L2-d cells are unable to produce EN and SPL daughters but continue to generate supernumerary AR cells, extending the period of L2-d pluripotency (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012b</a>). Of the 3871 transcripts scored OFF at the 0.4 mm stage after being ON in both 0.15 mm and 0.25 mm anthers, more than 84% (3252) are ON in 0.4 mm <em>mac1</em>, supporting the idea that these transcripts are normally shut-off concomitant with the completion of the first round of cell fate acquisition. Persistence of expression in the <em>mac1</em> mutant of transcripts that are normally downregulated to very low levels or are completely missing substantiates the idea that transient suppression of many transcript types is an important developmental feature of the 0.4 mm stage during which newly established cell types begin to differentiate and proliferate anticlinally to build a larger anther.</p><p id="p-58">To define a set of transcripts strongly associated with pluripotency, <em>mac1</em> transcriptome data were filtered sequentially by three criteria. First, significant downregulation between 0.15 mm anther primordia and 0.25 mm partially differentiated anthers (log2 &lt; −0.58; p-value &lt; 0.05), then significantly downregulated again between 0.25 mm and 0.4 mm, and finally the requirement that transcripts remain low or OFF at later stages. Filtering for these parameters, we recovered 21 transcripts, many of which are already known to be highly abundant in tassel primordia (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolduc <em>et al.</em> 2012</a>), including a bHLH and two Yabby-like TFs (<a id="xref-fig-7-10" class="xref-fig" href="#F7">Figure 7E</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS9.zip">Table S9</a>). Eighteen of these have a higher intensity value in 0.4 mm <em>mac1</em> than in fertile anthers, and 10 can be classified as significantly enriched in this mutant, which is defective in specifying somatic cell fates. Thus, the classification of these 21 transcripts as anther pluripotency markers is supported by both wild-type and male-sterile mutant expression data.</p><p id="p-59">To further investigate the molecular processes involved in the transition from pluripotency to bipotency, we examined the transcriptome changes in the <em>mac1</em> mutant more thoroughly at both the 0.2 mm and 0.4 mm stages in direct comparison to fertile anthers of the equivalent size classes. At 0.2 mm, the difference between mutant and fertile is not yet apparent microscopically, and we would expect changes only in the set of genes whose transcription is directly affected by the lack of MAC1 protein. At 0.2 mm there are 301 genes differentially regulated (193 up, 108 down) in <em>mac1</em> compared to wild-type, along with 16 transcripts specific to wild-type (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS11.pdf">Figure S11</a>). By 0.4 mm, <em>mac1</em> anthers contain three-times to five-times more AR and lack both the EN and SPL layers characteristic of fertile anthers. At this stage there are 1082 differentially regulated transcripts (328 up, 754 down) in <em>mac1</em> compared to wild-type, along with 258 present in wild-type but missing from <em>mac1</em> (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS12.pdf">Figure S12</a>, <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS13.pdf">Figure S13</a>). To identify genes regulated directly by the MAC1 signaling pathway, we looked at the overlap of the two stages of transcriptome comparisons to wild-type (0.4 mm and 0.2 mm) and identified 23 downregulated and 24 upregulated transcripts (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS14.pdf">Figure S14</a>, <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009738/-/DC1/FigureS15.pdf">Figure S15</a>). Of the 754 genes downregulated in <em>mac1</em> only at the 0.4 mm anther length stage, 287 are upregulated at 0.7 mm in the EPI/EN, suggesting these are already EPI/EN transcripts at 0.4 mm. These are likely candidate differentiation markers for the EN (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS12.zip">Table S12</a>). Another 137 are also upregulated in the ML/TAP, suggesting these may be SPL transcripts at 0.4 mm (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1/TableS12.zip">Table S12</a>).</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-60">Prior studies reported W23 maize anther transcriptomes during pre-meiotic preparation (1.0 mm), several stages of meiosis (1.5 mm, 2.0 mm, 2.5 mm), and key stages of post-meiotic (3.0 mm, 4.0 mm, mature pollen) ontogeny (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Ma <em>et al.</em> 2008</a>). The current study analyzed the transcriptomes of much younger anthers, including the 0.15 mm primordium prior to L2-d cell fate setting, at 0.25 mm when the archesporial cells are present and are regulating periclinal division in the L2-d to establish the endothecial and secondary parietal founding cells, at 0.4 mm when all cell types are proliferating to increase anther size, at 0.7 mm after the final cell fate specification event accomplished by the SPL cells to establish the middle layer and tapetum, and at 1.0 mm when the phase of AR mitosis is finished and the somatic cells have all acquired distinctive shapes, volumes, and staining features indicative of cell differentiation. The transcriptome analyses for whole maize anthers are now available for all the landmark developmental stages, and several common themes are clear.</p><div id="sec-23" class="subsection"><h3>Transcriptome complexity</h3><p id="p-61">First, the anther transcriptome is very complex and dynamic. Such complexity was noted in the earliest studies of dicot anther and pollen RNA composition (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Goldberg <em>et al.</em> 1993</a>), and this has remained a consistent observation for all species evaluated. Why would anthers express so many genes given the relative simplicity of tissues and cell types? Gametophytic transcriptome complexity could be a selected feature to eliminate deleterious mutations — unlike animal gametes, the haploid cells of plants are transcriptionally active and synthesize cellular structural components, factors required for mitosis, metabolic enzymes, and proteins required for their specialized functions as gametes or helper cells. A haplosufficiency test in the microgametophytes could purge the genome of deleterious mutations in genes that are also important in diploid sporophytes. Prior to meiosis, all anther cells are diploid, and we speculate that the diverse gene expression program could be a pre-meiotic diploid robustness test—a process that allows for selection for alleles that are sufficient to program normal development in plants heterozygous for a non-functional allele.</p><p id="p-62">The relatively high number of genes expressed may also reflect a more general characteristic of plant stem cells, which are abundant in maize anthers as pluripotent L2-d cells up to the 0.25 mm stage and then as bipotent SPL up to the 0.7 mm stage. Do plant stem cells exhibit basal transcription of thousands of untranslated mRNAs as a consequence of a largely non-repressive global chromatin status? Low levels of transcript production in stem cells would later be refined during cell fate acquisition for thousands of genes that are either selectively upregulated to provide mRNA for protein production or downregulated to eliminate transcription. To test this idea, it would be interesting to determine whether the thousands of low-abundance transcripts in stem cells are translated into proteins and, if not, whether the mRNAs are stored for later use or degraded at a later stage. It is not uncommon to identify plant genes with genetically defined developmental roles that are also expressed outside their predicted cell-specific and temporal patterns. For example, the AR-somatic patterning gene <em>Mac1</em> is expressed in roots, which lack germinal cells and exhibit no mutant phenotype (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012b</a>). This could be a case of unregulated expression, or <em>Mac1</em> transcription may hint at a more general role of MAC1 in forming the nested rings of cells by periclinal division, a function that may be masked in <em>mac1</em> roots by a functionally redundant factor that is not expressed in anthers. Another example of ectopic transcription is the meiotic gene <em>DNA meiotic recombinase 1</em> (<em>Dmc1</em>). The protein is responsible for crossover formation in meiotic cells and, despite having no role in vegetative tissues, the transcript is found in both silks and embryos (<a href="http://maizegdb.org/cgi-bin/displaygenemodelrecord.cgi?id=GRMZM2G109618">http://maizegdb.org/cgi-bin/displaygenemodelrecord.cgi?id=GRMZM2G109618</a>). Deep proteome data would be necessary to address this issue of the meaning and timing behind transcription of such an enormously complex and dynamic anther transcript set.</p><p id="p-63">A third contributor to complexity may be the multiple roles played by anther cell types over the 30 d from inception to pollen shed. Organs such as leaves, stems, and roots are built with specific, generally terminally differentiated cell types to perform a more fixed range of functions. In contrast, anther cells continue to exhibit new properties as the organ matures. The germinal cells are an obvious example: initially they are mitotic AR, then PMC, then haploid meiotic products, then developing gametophytes, and, finally, they are mature pollen. Accompanying these major changes in germinal cell functions, somatic cells are also changing roles. For example, one of the first differentiated roles of tapetal cells is to secrete callase to remodel the callose coat of PMC cells (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2010a</a>) prior to meiosis; the tapetum is also the likely source of nutrients for the AR, PMC, and developing pollen. After meiosis, tapetal cells differentiate to synthesize and secrete a wide range of molecules that are deposited as exine components on developing pollen grains. In maize, the developing pollen stick to modified tapetal cell walls; the grains are arrayed helically within the lobe surrounding a large locular airspace (<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>). Eventually the tapetal cells die, and the remnant cell walls become part of the structural stiffening of the anther. Similarly, other somatic cell types undergo sequential changes in function to facilitate the ultimate role of pollen dispersal. Although pollen dispersal mechanisms differ among flowering plants, anther dehiscence inevitably involves changes in the epidermis and endothecium to allow pollen out of the encasing somatic tissues.</p><p id="p-64">A fourth possible explanation for transcriptome complexity is that germinal and somatic cells may synthesize then store mRNAs required at later stages of development. This is a long-recognized, major mechanism in animal pre-meiotic cells (for example, in <em>Drosophila melanogaster</em>) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Spradling 1993</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fuller 1993</a>). The abundance of transcripts corresponding to RNA binding GO annotation, particularly in AR cells, indicates that the role of stored mRNA should be explored in anther cells as well. The stored mRNAs of animal eggs and sperm are required to program early embryo development. In the case of plants, such stored mRNAs may be required during the cascade of differentiation events in both the germinal and somatic anther cells. In some animal embryos such as <em>Caenorhabditis elegans</em>, there are periods of massive transcript loss (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Levin <em>et al.</em> 2012</a>) ascribed to the degradation of maternally deposited mRNA to facilitate a differentiation step in moving from one larval stage to the next. The cohorts of anther genes exhibiting a trough of expression could represent similar cases in which elimination and then re-expression of a transcript type are required for normal development.</p><p id="p-65">One approach to documenting stored mRNA is to demonstrate a temporal separation between mRNA maturation and translation into protein products. Our initial anther proteomes are insufficient to answer this question on a broad scale because we sampled only the most abundant proteins in whole anthers. Much deeper proteome sampling or antibody reagents to individual protein types will be required to adequately address the possibility of stored mRNA during anther ontogeny. Of particular interest in this regard is the striking early synthesis of transcripts for proteins with documented roles in meiosis. Transcripts for a number of well-studied meiotic factors and a longer list of meiosis-associated transcripts appear 5 days before meiotic preparation is considered to start when AR differentiate as PMC. These transcripts are present during AR specification, albeit often at low levels, and persist through the period of AR mitotic proliferation. These observations are further evidence of the distinctiveness of AR once specified (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>).</p><p id="p-66">Reproduction is the purpose in constructing an anther, and the AR cells are the most distinctive cell type in the organ. They undergo successive differentiation events, from AR to PMC to haploid meiotic products, before dividing to produce binucleate microspores and, finally, mature, trinucleate pollen. With each stage, these reproductive cells exhibit distinct morphology, cytology, gene expression patterns, and functions. Much has been learned about the post-meiotic transcriptome of developing microspores and mature pollen, but comparatively little is known about the pre-meiotic stage. In size they dwarf the inner and outer somatic cell types, growing to a maximum diameter of 50 µm at the PMC stage (<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>). From this study and a recent one focused on germinal and somatic transcriptomes just following AR differentiation (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Kelliher and Walbot 2014</a>), the following principles are established in maize. First, based on gene expression profiles, AR cells differentiate rapidly from their L2-d precursors and are distinct from the specified somatic cell types. Second, mirroring the distinctive roles of these reproductive cells, isolated AR, PMC, and pollen exhibit distinct transcriptomes. Third, despite the distinctive features of each stage and contrary to prevailing models, the AR from the earliest stage measured express meiosis-associated genes. These transcripts, constituting approximately one-third of the meiosis-associated list (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Kelliher and Walbot 2014</a>) including several strictly meiotic factors (this study), are expressed at low levels prior to PMC differentiation but are nonetheless enriched in AR cells within 1 to 2 days of their specification. This observation merits additional study to clarify whether these precociously expressed transcripts are translated into protein, and whether the proteins are involved in early differentiation for meiosis, before the AR differentiate into PMC when meiotic preparation has been assumed to occur. Fourth, AR cells exhibit mainly autonomous development, because in the absence of normal somatic neighbors, as in the <em>mac1</em> mutant (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012b</a>; <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Kelliher and Walbot 2012</a>), AR mature to PMC and start meiosis.</p><p id="p-67">As established here, transcriptome profiling indicates that the AR initially differentiate by expressing transcripts involved in energy and RNA metabolism. Just 1 to 2 days after their birth, AR are enriched in the following: alternative metabolism transcripts, which are responsible for the AR thriving in a hypoxic (&lt;2% O<sub>2</sub>) environment; RNA binding transcripts, which may be involved in long-term RNA storage; and ribosomal protein transcripts, suggesting that AR boost translational capacity or build ribosomes with distinctive properties. Future studies could address carbon physiology of AR cells to define the energy budget and determine the roles of RNA binding and ribosomal protein gene expression patterns in supporting AR proliferation and development. One of several additional unanswered questions is whether the PMC are transcriptionally silent during meiosis, an assumption based mainly on analyses of animal meiotic cells. Post-meiotic animal gametes, particularly sperm, are typically transcriptionally silent, whereas post-meiotic haploid gametophytes including mature pollen competent for fertilization are transcriptionally active. Consequently, silencing during meiosis may be part of the developmental program establishing silencing in animal gametes, a process that does not occur in plants. The maize mutant <em>ameiotic1</em> is a reminder that achieving normal PMC cytology is not a guarantee that meiosis will occur, because in this mutant PMC conduct mitosis instead. Those studying meiosis <em>per se</em> have focused on chromosome-level events. This focus has ignored the differentiation of the cytoplasmic and non-chromosomal nuclear compartments of germinal cells. Undoubtedly, many aspects of meiotic preparation involve new features of these as well, as previewed in the AR transcriptome assessment in which energy generation, RNA binding proteins, and ribosomal proteins are so prominent.</p></div><div id="sec-24" class="subsection"><h3>Distinctiveness of early anther stages</h3><p id="p-68">We posed the question of whether each of the five stages of pre-meiotic anther development profiled would be distinctive. It is clear that four of the stages exhibit numerous stage-specific transcripts, indicative of distinctive processes and regulation of gene expression. This stage distinctiveness increases over developmental time, peaking at 1.0 mm when all cell types are differentiated. At 0.7 mm, the LCM samples of outer somatic cells (differentiated EN plus EPI) and differentiated AR exhibit distinctive transcriptomes and express genes correspondent with their functions. In contrast, the just-specified inner somatic layers (ML and TAP sister cells from a recent SPL periclinal division) show few zone-specific transcripts. This result indicates that specification is a required first-step in differentiation but, as is commonly assumed, that acquisition of differentiated cell features requires time.</p><p id="p-69">Surprisingly, the 0.4 mm stage when EPI, EN, SPL, and AR are present has only 11 stage-specific transcripts. There is also a large decrease in transcript diversity at 0.4 mm —many transcripts that were expressed at low levels at the preceding 0.25 mm stage are not detectable by microarray hybridization—yet many of these transcripts are detectable a few days later at the 0.7 mm stage. We found this observation very puzzling and investigated it through bioinformatics analyses and additional experiments. From these experiments it is clear that this transient trough in transcript diversity reflects an active process that is part of normal differentiation. Our working model invokes two explanations for the trough. First, genes associated with pluripotency should disappear at 0.4 mm because normally all lobe cells have acquired a specific fate (including the bipotent SPL) by this stage. Such pluripotency genes are represented in the set mis-expressed by <em>mac1</em> at 0.4 mm, a mutant in which the pluripotent L2-d persist for far longer than normal and continue to generate extra AR until ∼0.4 mm. Second, we invoke a cell biological feature. The stages flanking 0.4 mm are periods of periclinal division and, hence, cell patterning in new tissue layers. We propose that many transcripts contributing to these processes are transiently downregulated at 0.4 mm when all cell division is anticlinal and, hence, contributing to anther growth in length and girth. Re-expression of the periclinal division programs in the SPL from 0.55 mm to 0.7 mm results in ML and TAP specification (<a id="xref-ref-23-12" class="xref-bibr" href="#ref-23">Kelliher and Walbot 2011</a>). We consider this a presumptive case of reuse of a developmental module within the context of anther development, a question we sought to address in our study. Future work could focus on identifying the regulators of periclinal division to test whether a few transcription factors are master regulators controlling periclinal rather than anticlinal division. Because ectopic periclinal divisions in the somatic cells cause dramatic developmental errors in anthers and result in male sterility (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Timofejeva <em>et al.</em> 2013</a>), tight regulation of this process appears to be essential to allow meiosis to proceed even if the AR are apparently normal, as in <em>mac1</em> (<a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012b</a>). Finally, it is of interest to understand how the trough transcripts are modulated in abundance: is it primarily through transcriptional control over short-lived mRNAs or are there post-transcriptional mechanisms that ensure degradation of this large class of transcripts at 0.4 mm and smaller transcript cohorts at other stages? Whether transcriptional or post-transcriptional, what is the refined temporal pattern of both the loss and re-gain of these trough transcripts and are they co-regulated in a highly synchronous manner?</p><p id="p-70">In summary, the transcriptome and proteome profiling data for anther primordia through the stage of completion of AR mitosis provide a rich description of gene expression events at five stages of maize anther development. These data suggest many more questions than they answer, and the candidate genes and processes can guide future research identified in this study. In particular, we predict that further analysis of the regulation of periclinal division, analysis of transcriptome and proteome changes in developmental mutants, and <em>in situ</em> localization of mRNA and protein products will illuminate a more detailed understanding of anther development.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-71">We thank Dr. K. van der Linde and K. Murphy for commenting on drafts of the manuscript, and C. Adams of the Stanford Mass Spectrometry Facility for his assistance with the proteomics samples and methods. Research was supported by National Science Foundation (NSF) grant PGRP07-01880. R.L.E. was supported in part by an National Institutes of Health (NIH) Cell and Molecular Biology Training Grant (5T32GM007276-36), by the 2013 American Society of Plant Biologists—Pioneer Hi-Bred Graduate Student Fellowship, and by an Agriculture and Food Research Initiative Competitive Grant from the United States Department of Agriculture (USDA) National Institute of Food and Agriculture (2013-67011-21096). T.K. was supported in part by an NIH Biotechnology Training Grant (5-T32-GM008412-17).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009738/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-72">GEO records GSE43982 and GSE13118 contain all of the microarray data.</p></li><li class="fn" id="fn-6"><p id="p-73"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-29"><span class="received-label">Received </span>November 29, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-03-16"><span class="accepted-label">Accepted </span>March 16, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Zhang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.1" data-doi="10.1126/science.1164645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laparra</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmelz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>tasselseed1</em> is a lipoxygenase affecting jasmonic acid signaling in sex determination of maize.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAcosta%26rft.auinit1%253DI.%2BF.%26rft.volume%253D323%26rft.issue%253D5911%26rft.spage%253D262%26rft.epage%253D265%26rft.atitle%253Dtasselseed1%2BIs%2Ba%2BLipoxygenase%2BAffecting%2BJasmonic%2BAcid%2BSignaling%2Bin%2BSex%2BDetermination%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1164645%26rft_id%253Dinfo%253Apmid%252F19131630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMS8yNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.2" data-doi="10.1371/journal.pone.0026162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suwabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comprehensive network analysis of anther-expressed genes in rice by the combination of 33 laser microdissection and 143 spatiotemporal microarrays.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e26162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAya%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De26162%26rft.epage%253De26162%26rft.atitle%253DComprehensive%2Bnetwork%2Banalysis%2Bof%2Banther-expressed%2Bgenes%2Bin%2Brice%2Bby%2Bthe%2Bcombination%2Bof%2B33%2Blaser%2Bmicrodissection%2Band%2B143%2Bspatiotemporal%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0026162%26rft_id%253Dinfo%253Apmid%252F22046259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0026162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22046259&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.993.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Batygina</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Embryology of Flowering Plants: Terminology and Concepts</span>, <span class="cit-publ-name">Science Publishers</span>, <span class="cit-publ-loc">Ann Arbor</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.4" data-doi="10.1387/ijdb.052019lb"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boavida</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feijo</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The making of gametes in higher plants.</span> <abbr class="cit-jnl-abbrev">Int. J. Dev. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BInternational%2Bjournal%2Bof%2Bdevelopmental%2Bbiology%26rft.stitle%253DInt%2BJ%2BDev%2BBiol%26rft.aulast%253DBoavida%26rft.auinit1%253DL.%2BC.%26rft.volume%253D49%26rft.issue%253D5-6%26rft.spage%253D595%26rft.epage%253D614%26rft.atitle%253DThe%2Bmaking%2Bof%2Bgametes%2Bin%2Bhigher%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1387%252Fijdb.052019lb%26rft_id%253Dinfo%253Apmid%252F16096968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1387/ijdb.052019lb&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16096968&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233266200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.5" data-doi="10.1101/gad.193433.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolduc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yilmaz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mejia-Guerra</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morohashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Unraveling the KNOTTED1 regulatory network in maize meristems.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1685</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBolduc%26rft.auinit1%253DN.%26rft.volume%253D26%26rft.issue%253D15%26rft.spage%253D1685%26rft.epage%253D1690%26rft.atitle%253DUnraveling%2Bthe%2BKNOTTED1%2Bregulatory%2Bnetwork%2Bin%2Bmaize%2Bmeristems%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.193433.112%26rft_id%253Dinfo%253Apmid%252F22855831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI2LzE1LzE2ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.6" data-doi="10.1016/j.cell.2012.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calarco</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borges</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">M. T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Ex</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jullien</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reprogramming of DNA methylation in pollen guides epigenetic inheritance via small RNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCalarco%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D194%26rft.epage%253D205%26rft.atitle%253DReprogramming%2Bof%2BDNA%2Bmethylation%2Bin%2Bpollen%2Bguides%2Bepigenetic%2Binheritance%2Bvia%2Bsmall%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.09.001%26rft_id%253Dinfo%253Apmid%252F23000270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23000270&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309544200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.7" data-doi="10.1074/mcp.M500173-MCP200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casati</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlingame</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Analysis of leaf proteome after UV-B irradiation in maize lines differing in sensitivity.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1673</span>–<span class="cit-lpage">1685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M500173-MCP200%26rft_id%253Dinfo%253Apmid%252F16043824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjk6IjQvMTEvMTY3MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.8" data-doi="10.1105/tpc.11.9.1623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Couteau</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horlow</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandjean</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Random chromosome segregation without meiotic arrest in both male and female meiocytes of a <em>dmc1</em> mutant of <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DCouteau%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D1623%26rft.epage%253D1634%26rft.atitle%253DRandom%2BChromosome%2BSegregation%2Bwithout%2BMeiotic%2BArrest%2Bin%2BBoth%2BMale%2Band%2BFemale%2BMeiocytes%2Bof%2Ba%2Bdmc1%2BMutant%2Bof%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.11.9.1623%26rft_id%253Dinfo%253Apmid%252F10488231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjExLzkvMTYyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.9" data-doi="10.1016/j.cub.2011.03.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Depuydt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardtke</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hormone signalling crosstalk in plant growth regulation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">R365</span>–<span class="cit-lpage">R373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DDepuydt%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253DR365%26rft.epage%253DR373%26rft.atitle%253DHormone%2Bsignalling%2Bcrosstalk%2Bin%2Bplant%2Bgrowth%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.03.013%26rft_id%253Dinfo%253Apmid%252F21549959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.03.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21549959&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.10" data-doi="10.1186/1471-2229-11-78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deveshwar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovill</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Able</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapoor</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of anther transcriptomes to identify genes contributing to meiosis and male gametophyte development in rice.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DDeveshwar%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.spage%253D78%26rft.epage%253D78%26rft.atitle%253DAnalysis%2Bof%2Banther%2Btranscriptomes%2Bto%2Bidentify%2Bgenes%2Bcontributing%2Bto%2Bmeiosis%2Band%2Bmale%2Bgametophyte%2Bdevelopment%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-11-78%26rft_id%253Dinfo%253Apmid%252F21554676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-11-78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21554676&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.11" data-doi="10.1242/dev.00804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Extavour</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akam</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mechanisms of germ cell specification across the metazoans: epigenesis and preformation.</span> <abbr class="cit-jnl-abbrev">Dev.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">5869</span>–<span class="cit-lpage">5884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DExtavour%26rft.auinit1%253DC.%2BG.%26rft.volume%253D130%26rft.issue%253D24%26rft.spage%253D5869%26rft.epage%253D5884%26rft.atitle%253DMechanisms%2Bof%2Bgerm%2Bcell%2Bspecification%2Bacross%2Bthe%2Bmetazoans%253A%2Bepigenesis%2Band%2Bpreformation%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.00804%26rft_id%253Dinfo%253Apmid%252F14597570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMwLzI0LzU4NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.12" data-doi="10.1016/j.tig.2007.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Packaging the male germline in plants.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DFeng%26rft.auinit1%253DX.%26rft.volume%253D23%26rft.issue%253D10%26rft.spage%253D503%26rft.epage%253D510%26rft.atitle%253DPackaging%2Bthe%2Bmale%2Bgermline%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.08.005%26rft_id%253Dinfo%253Apmid%252F17825943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17825943&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250636600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.13" data-doi="10.1016/j.devcel.2013.01.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zilberman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A conversation across generations: soma-germ cell crosstalk in plants.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D24%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2013.01.014%26rft_id%253Dinfo%253Apmid%252F23410937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2013.01.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23410937&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315372700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.993.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Spermatogenesis</span>, pp. <span class="cit-fpage">71</span>–<span class="cit-lpage">147</span> in <span class="cit-source">The Development of Drosophila melanogaster</span>, edited by M. <span class="cit-ed"><span class="cit-name-surname">Bate</span></span>, <span class="cit-ed"><span class="cit-name-surname">Martinez-Arias</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Cold Spring Harbor Lab Press</span>, <span class="cit-publ-loc">Cold Spring Harbor</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.15" data-doi="10.1105/tpc.5.10.1217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beals</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Anther development: basic principles and practical applications.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DGoldberg%26rft.auinit1%253DR.%2BB.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D1217%26rft.epage%253D1229%26rft.atitle%253DAnther%2BDevelopment%253A%2BBasic%2BPrinciples%2Band%2BPractical%2BApplications%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.5.10.1217%26rft_id%253Dinfo%253Apmid%252F8281038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo5OiJwbGFudGNlbGwiO3M6NToicmVzaWQiO3M6OToiNS8xMC8xMjE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.16" data-doi="10.1242/jcs.03054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golubovskaya</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamant</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timofejeva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Alleles of <em>afd1</em> dissect REC8 functions during meiotic prophase I.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">3306</span>–<span class="cit-lpage">3315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.03054%26rft_id%253Dinfo%253Apmid%252F16868028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjExOS8xNi8zMzA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.17" data-doi="10.1093/pcp/pcn128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hobo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suwabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Various spatiotemporal expression profiles of anther-expressed genes in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1417</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcn128%26rft_id%253Dinfo%253Apmid%252F18776202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjEwOiI0OS8xMC8xNDE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.18" data-doi="10.1186/gb-2004-5-11-r85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honys</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twell</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transcriptome analysis of haploid male gametophyte development in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHonys%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253DR85%26rft.epage%253DR85%26rft.atitle%253DTranscriptome%2Banalysis%2Bof%2Bhaploid%2Bmale%2Bgametophyte%2Bdevelopment%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-11-r85%26rft_id%253Dinfo%253Apmid%252F15535861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-11-r85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15535861&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.993.19"><div class="cit-metadata unstructured">Hord, C. L., and H. Ma, 2007 Genetic control of anther cell division and differentiation, pp. 361–376 in <em>Plant Cell Monographs 9</em>, edited by D. P. S. Verma and Z. Hong. Springer-Verlag, Berlin.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.20" data-doi="10.1104/pp.108.131128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsing</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Analysis of advanced rice anther transcriptomes reveal global tapetum secretory functions and potential proteins for lipid exine formation.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">694</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.131128%26rft_id%253Dinfo%253Apmid%252F19091874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDkvMi82OTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veit</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hake</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Expression of maize KNOTTED1 related homeobox genes in the shoot apical meristem predicts patterns of morphogenesis in the vegetative shoot.</span> <abbr class="cit-jnl-abbrev">Dev.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DJackson%26rft.auinit1%253DD.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D405%26rft.epage%253D413%26rft.atitle%253DExpression%2Bof%2Bmaize%2BKNOTTED1%2Brelated%2Bhomeobox%2Bgenes%2Bin%2Bthe%2Bshoot%2Bapical%2Bmeristem%2Bpredicts%2Bpatterns%2Bof%2Bmorphogenesis%2Bin%2Bthe%2Bvegetative%2Bshoot%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMjAvMi80MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.22" data-doi="10.1111/j.1744-7909.2012.01149.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide transcription factor gene prediction and their expressional tissue-specificities in maize.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DJiang%26rft.auinit1%253DY.%26rft.volume%253D54%26rft.issue%253D9%26rft.spage%253D616%26rft.epage%253D630%26rft.atitle%253DGenome-wide%2Btranscription%2Bfactor%2Bgene%2Bprediction%2Band%2Btheir%2Bexpressional%2Btissue-specificities%2Bin%2Bmaize%2528F%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2012.01149.x%26rft_id%253Dinfo%253Apmid%252F22862992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2012.01149.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22862992&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.23" data-doi="10.1016/j.ydbio.2010.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelliher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Emergence and patterning of the five cell types of the <em>Zea mays</em> anther locule.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DKelliher%26rft.auinit1%253DT.%26rft.volume%253D350%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D49%26rft.atitle%253DEmergence%2Band%2Bpatterning%2Bof%2Bthe%2Bfive%2Bcell%2Btypes%2Bof%2Bthe%2BZea%2Bmays%2Banther%2Blocule.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2010.11.005%26rft_id%253Dinfo%253Apmid%252F21070762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2010.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21070762&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286547600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.24" data-doi="10.1126/science.1220080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelliher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hypoxia triggers meiotic fate acquisition in maize.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKelliher%26rft.auinit1%253DT.%26rft.volume%253D337%26rft.issue%253D6092%26rft.spage%253D345%26rft.epage%253D348%26rft.atitle%253DHypoxia%2BTriggers%2BMeiotic%2BFate%2BAcquisition%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1220080%26rft_id%253Dinfo%253Apmid%252F22822150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Mi8zNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.25" data-doi="10.1111/tpj.12414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelliher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Maize germinal cell initials accommodate hypoxia and precociously express meiotic genes.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D77%26rft.spage%253D639%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12414%26rft_id%253Dinfo%253Apmid%252F24387628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24387628&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330907900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.26" data-doi="10.1016/j.devcel.2012.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashimshony</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanai</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developmental milestones punctuate gene expression in the <em>Caenorhabditis</em> embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DLevin%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D1101%26rft.epage%253D1108%26rft.atitle%253DDevelopmental%2Bmilestones%2Bpunctuate%2Bgene%2Bexpression%2Bin%2Bthe%2BCaenorhabditis%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2012.04.004%26rft_id%253Dinfo%253Apmid%252F22560298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2012.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22560298&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304291700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golubovskaya</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional analysis of maize RAD51 in meiosis and double-strand break repair.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1469</span>–<span class="cit-lpage">1482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D1469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.28" data-doi="10.1073/pnas.0604232103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Origins and evolution of the recA/RAD51 gene family: Evidence for ancient gene duplication and endosymbiotic gene transfer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">10328</span>–<span class="cit-lpage">10333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0604232103%26rft_id%253Dinfo%253Apmid%252F16798872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzI3LzEwMzI4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.29" data-doi="10.1007/s11103-006-0054-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">H.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">S.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.-B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular analysis of early rice stamen development using organ-specific gene expression profiling.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DLu%26rft.auinit1%253DX.%2BC.%26rft.volume%253D61%26rft.issue%253D6%26rft.spage%253D845%26rft.epage%253D861%26rft.atitle%253DMolecular%2Banalysis%2Bof%2Bearly%2Brice%2Bstamen%2Bdevelopment%2Busing%2Borgan-specific%2Bgene%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-006-0054-3%26rft_id%253Dinfo%253Apmid%252F16927200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-006-0054-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16927200&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240019400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.30" data-doi="10.1146/annurev.arplant.55.031903.141717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular genetic analyses of microsporogenesis and microgametogenesis in flowering plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DMa%26rft.auinit1%253DH.%26rft.volume%253D56%26rft.spage%253D393%26rft.epage%253D434%26rft.atitle%253DMolecular%2Bgenetic%2Banalyses%2Bof%2Bmicrosporogenesis%2Band%2Bmicrogametogenesis%2Bin%2Bflowering%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.55.031903.141717%26rft_id%253Dinfo%253Apmid%252F15862102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.55.031903.141717&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15862102&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230282800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.31" data-doi="10.1186/gb-2006-7-3-r22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrow</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative profiling of the sense and antisense transcriptome of maize lines.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253DR22%26rft.epage%253DR22%26rft.atitle%253DComparative%2Bprofiling%2Bof%2Bthe%2Bsense%2Band%2Bantisense%2Btranscriptome%2Bof%2Bmaize%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-3-r22%26rft_id%253Dinfo%253Apmid%252F16542496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-3-r22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16542496&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.32" data-doi="10.1111/j.1365-313X.2007.03074.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrow</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcriptome profiling of maize anthers using genetic ablation to analyze pre-meiotic and tapetal cell types.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D637%26rft.epage%253D648%26rft.atitle%253DTranscriptome%2Bprofiling%2Bof%2Bmaize%2Banthers%2Busing%2Bgenetic%2Bablation%2Bto%2Banalyze%2Bpre-meiotic%2Band%2Btapetal%2Bcell%2Btypes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03074.x%26rft_id%253Dinfo%253Apmid%252F17419846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03074.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17419846&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246398400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.33" data-doi="10.1186/gb-2008-9-12-r181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skibbe</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Male reproductive development: gene expression profiling of maize anther and pollen ontogeny.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253DR181%26rft.epage%253DR181%26rft.atitle%253DMale%2Breproductive%2Bdevelopment%253A%2Bgene%2Bexpression%2Bprofiling%2Bof%2Bmaize%2Banther%2Band%2Bpollen%2Bontogeny.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-12-r181%26rft_id%253Dinfo%253Apmid%252F19099579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-12-r181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19099579&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.34" data-doi="10.1093/jxb/erp040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mutasa-Göttgens</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gibberellin as a factor in floral regulatory networks.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1979</span>–<span class="cit-lpage">1989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferp040%26rft_id%253Dinfo%253Apmid%252F19264752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjYwLzcvMTk3OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.35" data-doi="10.1186/1471-2229-11-120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nan</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronceret</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cande</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Global transcriptome analysis of two <em>ameiotic1</em> alleles in maize anthers: defining steps in meiotic entry and progression through prophase I.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DNan%26rft.auinit1%253DG.%2BL.%26rft.volume%253D11%26rft.spage%253D120%26rft.epage%253D120%26rft.atitle%253DGlobal%2Btranscriptome%2Banalysis%2Bof%2Btwo%2Bameiotic1%2Balleles%2Bin%2Bmaize%2Banthers%253A%2Bdefining%2Bsteps%2Bin%2Bmeiotic%2Bentry%2Band%2Bprogression%2Bthrough%2Bprophase%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-11-120%26rft_id%253Dinfo%253Apmid%252F21867558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-11-120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21867558&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.36" data-doi="10.1126/science.1091110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pawlowski</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golubovskaya</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timofejeva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meeley</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheridan</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Coordination of meiotic recombination, pairing, and synapsis by PHS1.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPawlowski%26rft.auinit1%253DW.%2BP.%26rft.volume%253D303%26rft.issue%253D5654%26rft.spage%253D89%26rft.epage%253D92%26rft.atitle%253DCoordination%2Bof%2BMeiotic%2BRecombination%252C%2BPairing%252C%2Band%2BSynapsis%2Bby%2BPHS1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1091110%26rft_id%253Dinfo%253Apmid%252F14704428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMDMvNTY1NC84OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.37" data-doi="10.1073/pnas.0810115106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pawlowski</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung-Ju</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golubovskaya</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szymaniaka</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shid</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maize AMEIOTIC1 is essential for multiple early meiotic processes and likely required for the initiation of meiosis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">3603</span>–<span class="cit-lpage">3608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810115106%26rft_id%253Dinfo%253Apmid%252F19204280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzkvMzYwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.38" data-doi="10.1007/s004970050158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bui</span>  <span class="cit-name-given-names">A. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weterings</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntire</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Anther developmental defects in <em>Arabidopsis thaliana</em> male-sterile mutants.</span> <abbr class="cit-jnl-abbrev">Sex. Plant Reprod.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSex.%2BPlant%2BReprod.%26rft.volume%253D11%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004970050158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004970050158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000078726500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.39" data-doi="10.1073/pnas.96.20.11664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiefthaler</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sieber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chevalier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Molecular analysis of NOZZLE, a gene involved in pattern formation and early sporogenesis during sex organ development in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">11664</span>–<span class="cit-lpage">11669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSchiefthaler%26rft.auinit1%253DU.%26rft.volume%253D96%26rft.issue%253D20%26rft.spage%253D11664%26rft.epage%253D11669%26rft.atitle%253DMolecular%2Banalysis%2Bof%2BNOZZLE%252C%2Ba%2Bgene%2Binvolved%2Bin%2Bpattern%2Bformation%2Band%2Bearly%2Bsporogenesis%2Bduring%2Bsex%2Borgan%2Bdevelopment%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.20.11664%26rft_id%253Dinfo%253Apmid%252F10500234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjAvMTE2NjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.40" data-doi="10.1371/journal.pbio.1001155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wuest</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vijverberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baroux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptome analysis of the <em>Arabidopsis</em> megaspore mother cell uncovers the importance of RNA helicases for plant germline development.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1001155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSchmidt%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253De1001155%26rft.epage%253De1001155%26rft.atitle%253DTranscriptome%2Banalysis%2Bof%2Bthe%2BArabidopsis%2Bmegaspore%2Bmother%2Bcell%2Buncovers%2Bthe%2Bimportance%2Bof%2BRNA%2Bhelicases%2Bfor%2Bplant%2Bgermline%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001155%26rft_id%253Dinfo%253Apmid%252F21949639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949639&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.41" data-doi="10.1105/tpc.017012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spielman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Stamen structure and function.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>(<span class="cit-supplement">Suppl</span>): <span class="cit-fpage">S46</span>–<span class="cit-lpage">S60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.017012%26rft_id%253Dinfo%253Apmid%252F15131249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjE0OiIxNi9zdXBwbF8xL1M0NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.42" data-doi="10.1111/j.1365-313X.2009.03901.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skibbe</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medzihradszky</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlingame</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutator transposon activity reprograms the transcriptomes and proteomes of developing maize anthers.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSkibbe%26rft.auinit1%253DD.%2BS.%26rft.volume%253D59%26rft.issue%253D4%26rft.spage%253D622%26rft.epage%253D633%26rft.atitle%253DMutator%2Btransposon%2Bactivity%2Breprograms%2Bthe%2Btranscriptomes%2Band%2Bproteomes%2Bof%2Bdeveloping%2Bmaize%2Banthers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03901.x%26rft_id%253Dinfo%253Apmid%252F19453454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03901.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19453454&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268961600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.43" data-doi="10.1105/tpc.2.8.755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyerowitz</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Early flower development in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DSmyth%26rft.auinit1%253DD.%2BR.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D755%26rft.epage%253D767%26rft.atitle%253DEarly%2BFlower%2BDevelopment%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.2.8.755%26rft_id%253Dinfo%253Apmid%252F2152125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjIvOC83NTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.44" data-doi="10.1093/mp/sst054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Regulation of stamen development by coordinated actions of jasmonate, auxin and gibberellin in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fsst054%26rft_id%253Dinfo%253Apmid%252F23543439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/sst054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23543439&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-4.6.993.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spradling</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Developmental genetics of oogenesis</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">70</span> in <span class="cit-source">The Development of Drosophila melanogaster</span>, edited by M. <span class="cit-ed"><span class="cit-name-surname">Bate</span></span>, <span class="cit-ed"><span class="cit-name-surname">Martinez-Arias</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Cold Spring Harbor Lab Press</span>, <span class="cit-publ-loc">Cold Spring Harbor</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.46" data-doi="10.1104/pp.110.161661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Global gene profiling of laser-captured Pollen Mother Cells indicates molecular pathways and gene subfamilies involved in rice meiosis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1855</span>–<span class="cit-lpage">1870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.161661%26rft_id%253Dinfo%253Apmid%252F20959420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU0LzQvMTg1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.47" data-doi="10.1534/g3.112.004465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timofejeva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skibbe</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golubovskaya</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cytological characterization and allelism testing of anther developmental mutants identified in a screen of maize male sterile lines.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DTimofejeva%26rft.auinit1%253DL.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D231%26rft.epage%253D249%26rft.atitle%253DCytological%2BCharacterization%2Band%2BAllelism%2BTesting%2Bof%2BAnther%2BDevelopmental%2BMutants%2BIdentified%2Bin%2Ba%2BScreen%2Bof%2BMaize%2BMale%2BSterile%2BLines%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004465%26rft_id%253Dinfo%253Apmid%252F23390600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8yMzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.48" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3-new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC8xNS9lMTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.49" data-doi="10.1111/j.1365-313X.2009.03916.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vernoud</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laigle</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meeley</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The HD-ZIP IV transcription factor OCL4 is necessary for trichome patterning and anther development in maize.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DVernoud%26rft.auinit1%253DV.%26rft.volume%253D59%26rft.issue%253D6%26rft.spage%253D883%26rft.epage%253D894%26rft.atitle%253DThe%2BHD-ZIP%2BIV%2Btranscription%2Bfactor%2BOCL4%2Bis%2Bnecessary%2Bfor%2Btrichome%2Bpatterning%2Band%2Banther%2Bdevelopment%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03916.x%26rft_id%253Dinfo%253Apmid%252F19453441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03916.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19453441&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269708400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.50" data-doi="10.1007/s00497-009-0109-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skibbe</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Maize host requirements for <em>Ustilago maydis</em> tumor induction.</span> <abbr class="cit-jnl-abbrev">Sex. Plant Reprod.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DSex%2BPlant%2BReprod%26rft.aulast%253DWalbot%26rft.auinit1%253DV.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D13%26rft.atitle%253DMaize%2Bhost%2Brequirements%2Bfor%2BUstilago%2Bmaydis%2Btumor%2Binduction.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00497-009-0109-0%26rft_id%253Dinfo%253Apmid%252F20165959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00497-009-0109-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20165959&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274657800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.51" data-doi="10.1111/j.1365-313X.2010.04294.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oses-Prieto</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlingame</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">The <em>male sterile 8</em> mutation of maize disrupts the temporal progression of the transcriptome and results in the mis-regulation of metabolic functions.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWang%26rft.auinit1%253DD.%26rft.volume%253D63%26rft.issue%253D6%26rft.spage%253D939%26rft.epage%253D951%26rft.atitle%253DThe%2Bmale%2Bsterile%2B8%2Bmutation%2Bof%2Bmaize%2Bdisrupts%2Bthe%2Btemporal%2Bprogression%2Bof%2Bthe%2Btranscriptome%2Band%2Bresults%2Bin%2Bthe%2Bmis-regulation%2Bof%2Bmetabolic%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04294.x%26rft_id%253Dinfo%253Apmid%252F20626649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04294.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20626649&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-2" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.52" data-doi="10.1105/tpc.109.070789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">The central element protein ZEP1 of the synaptonemal complex regulates the number of crossovers during meiosis in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.070789%26rft_id%253Dinfo%253Apmid%252F20154151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIyLzIvNDE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.53" data-doi="10.1111/j.1467-7652.2012.00721.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egger</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">A low molecular weight proteome comparison of fertile and <em>male sterile 8</em> anthers of <em>Zea mays</em>.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DWang%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D925%26rft.epage%253D935%26rft.atitle%253DA%2Blow%2Bmolecular%2Bweight%2Bproteome%2Bcomparison%2Bof%2Bfertile%2Band%2Bmale%2Bsterile%2B8%2Banthers%2Bof%2BZea%2Bmays.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00721.x%26rft_id%253Dinfo%253Apmid%252F22748129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00721.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748129&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.54" data-doi="10.1242/dev.077446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R. C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nan</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelliher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timofejeva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vernoud</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Maize <em>Multiple archesporial cell 1</em> (<em>mac1</em>), an ortholog of rice TDL1A, modulates cell proliferation and identity in early anther development.</span> <abbr class="cit-jnl-abbrev">Dev.</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">2584</span>–<span class="cit-lpage">2603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DCapilla%26rft.auinit1%253DA.%26rft.volume%253D139%26rft.issue%253D14%26rft.spage%253D2584%26rft.epage%253D2593%26rft.atitle%253DPlanar%2Bcell%2Bpolarity%2Bcontrols%2Bdirectional%2BNotch%2Bsignaling%2Bin%2Bthe%2BDrosophila%2Bleg%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.077446%26rft_id%253Dinfo%253Apmid%252F22736244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM5LzE0LzI1ODQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.55" data-doi="10.1105/tpc.021741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellmer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riechmann</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves-Ferreira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyerowitz</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-wide analysis of spatial gene expression in <em>Arabidopsis</em> flowers.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1314</span>–<span class="cit-lpage">1326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.021741%26rft_id%253Dinfo%253Apmid%252F15100403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzUvMTMxNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.56" data-doi="10.1111/j.1365-313X.2007.03217.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wijeratne</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Differential gene expression in Arabidopsis wild-type and mutant anthers: insights into anther cell differentiation and regulatory networks.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWijeratne%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D52%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D29%26rft.atitle%253DDifferential%2Bgene%2Bexpression%2Bin%2BArabidopsis%2Bwild-type%2Band%2Bmutant%2Banthers%253A%2Binsights%2Binto%2Banther%2Bcell%2Bdifferentiation%2Band%2Bregulatory%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03217.x%26rft_id%253Dinfo%253Apmid%252F17666023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03217.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17666023&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249828700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.57" data-doi="10.1111/j.1365-313X.2010.04439.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The transcriptome landscape of <em>Arabidopsis</em> male meiocytes from high-throughput sequencing: the complexity and evolution of the meiotic process.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D65%26rft.issue%253D4%26rft.spage%253D503%26rft.epage%253D516%26rft.atitle%253DThe%2Btranscriptome%2Blandscape%2Bof%2BArabidopsis%2Bmale%2Bmeiocytes%2Bfrom%2Bhigh-throughput%2Bsequencing%253A%2Bthe%2Bcomplexity%2Band%2Bevolution%2Bof%2Bthe%2Bmeiotic%2Bprocess.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04439.x%26rft_id%253Dinfo%253Apmid%252F21208307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04439.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208307&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286886100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Brassinosteroids control male fertility by regulating the expression of key genes involved in <em>Arabidopsis</em> anther and pollen development</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">6101</span>–<span class="cit-lpage">6105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D107%26rft.spage%253D6101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-4" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.59" data-doi="10.1007/s00412-010-0284-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">OsSPO11–1 is essential for both homologous chromosome pairing and crossover formation in rice.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D119%26rft.spage%253D625%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-010-0284-7%26rft_id%253Dinfo%253Apmid%252F20625906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-010-0284-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20625906&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.6.993.60" data-doi="10.1089/cmb.2005.12.1047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernald</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comprehensive algorithm for quantitative real-time polymerase chain reaction.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1047</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DZhao%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D1047%26rft.epage%253D1064%26rft.atitle%253DComprehensive%2Balgorithm%2Bfor%2Bquantitative%2Breal-time%2Bpolymerase%2Bchain%2Breaction.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2005.12.1047%26rft_id%253Dinfo%253Apmid%252F16241897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2005.12.1047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16241897&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233288200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/6/993.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/6/983" title="Self-Incompatibility in Brassicaceae: Identification and Characterization of SRK-Like Sequences Linked to the S-Locus in the Tribe Biscutelleae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/6/1011" title="Sex-Determination System in the Diploid Yeast Zygosaccharomyces sapae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/6"><span>Volume 4 Issue 6, June 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/6.cover-source.jpg?itok=I-KHzzO2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/6/993" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_BXyA5qJw_sFHk_7idjBZqc37EA1LGAwDR801eH2NSU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genetics-sex" class="" data-icon-position="" data-hide-link-title="0">Genetics of sex</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/6/993" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/6/993" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/6/993" class="active" data-icon-position="" data-hide-link-title="0">Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459892" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zFHNTANYyJpFJB__Z04Jel7XVS-lIRqjFX8t4wuXz9I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/6/993.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/6/993" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-I3yQxiRo0ABLcEu5wS1ECEB4fDrOI-MZa7sr0k6LbBg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/6/993" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/6/993.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459892" id="node459892--2" data-pisa="ggg;4/6/993" data-pisa-master="ggg;4/6/993" data-apath="/ggg/4/6/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Han Zhang</span>, <span class="highwire-citation-author" data-delta="1">Rachel L. Egger</span>, <span class="highwire-citation-author" data-delta="2">Timothy Kelliher</span>, <span class="highwire-citation-author" data-delta="3">Darren Morrow</span>, <span class="highwire-citation-author" data-delta="4">John Fernandes</span>, <span class="highwire-citation-author" data-delta="5">Guo-Ling Nan</span> and <span class="highwire-citation-author" data-delta="6">Virginia Walbot</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1010;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009738 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459892--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Han Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Han%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rachel L. Egger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BL.%2BEgger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Egger%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARachel%2BL.%2BEgger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:egger@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">egger@stanford.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy Kelliher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BKelliher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelliher%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BKelliher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Darren Morrow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BMorrow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrow%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BMorrow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Fernandes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BFernandes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandes%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BFernandes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Guo-Ling Nan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Ling%2BNan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuo-Ling%2BNan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Virginia Walbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BWalbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walbot%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BWalbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/6/993" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/6/993&amp;title=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/6/993&amp;title=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/6/993&amp;title=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/6/993&amp;text=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/6/993&amp;title=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/6/993&amp;t=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/6/993&amp;title=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/6/993&amp;title=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459892/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459892" id="node459892--3" data-pisa="ggg;4/6/993" data-pisa-master="ggg;4/6/993" data-apath="/ggg/4/6/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptomes and Proteomes Define Gene Expression Progression in Pre-meiotic Maize Anthers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Han Zhang</span>, <span class="highwire-citation-author" data-delta="1">Rachel L. Egger</span>, <span class="highwire-citation-author" data-delta="2">Timothy Kelliher</span>, <span class="highwire-citation-author" data-delta="3">Darren Morrow</span>, <span class="highwire-citation-author" data-delta="4">John Fernandes</span>, <span class="highwire-citation-author" data-delta="5">Guo-Ling Nan</span> and <span class="highwire-citation-author" data-delta="6">Virginia Walbot</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1010;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009738 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459892--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Han Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Han%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rachel L. Egger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachel%2BL.%2BEgger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Egger%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARachel%2BL.%2BEgger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:egger@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">egger@stanford.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy Kelliher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BKelliher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelliher%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BKelliher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Darren Morrow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BMorrow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrow%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BMorrow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Fernandes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BFernandes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernandes%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BFernandes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Guo-Ling Nan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guo-Ling%2BNan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuo-Ling%2BNan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Virginia Walbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BWalbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walbot%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BWalbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=993&amp;rft.epage=1010&amp;rft.atitle=Transcriptomes%20and%20Proteomes%20Define%20Gene%20Expression%20Progression%20in%20Pre-meiotic%20Maize%20Anthers&amp;rft.volume=4&amp;rft.issue=6&amp;rft.date=2014-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhang%2C+Han&amp;rft.au=Egger%2C+Rachel+L.&amp;rft.au=Kelliher%2C+Timothy&amp;rft.au=Morrow%2C+Darren&amp;rft.au=Fernandes%2C+John&amp;rft.au=Nan%2C+Guo-Ling&amp;rft.au=Walbot%2C+Virginia"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459892/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459892/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459892/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459892/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459892/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459892/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459892/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459892/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459892/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459892/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459892/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459892/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495849" id="node495849" data-pisa="ggg;10/3/967" data-pisa-master="ggg;g3.119.400963" data-apath="/ggg/10/3/967.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/967" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The <em>Drosophila</em> Post-mating Response: Gene Expression and Behavioral Changes Reveal Perdurance and Variation in Cross-Tissue Interactions</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495800" id="node495800" data-pisa="ggg;g3.119.400923v1" data-pisa-master="ggg;g3.119.400923" data-apath="/ggg/early/2020/02/12/g3.119.400923.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/12/g3.119.400923" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Sex Chromosome Evolution in Muscid Flies</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494619" id="node494619" data-pisa="ggg;9/11/3867" data-pisa-master="ggg;g3.119.400656" data-apath="/ggg/9/11/3867.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3867" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mapping of the Sex Determining Region on Linkage Group 12 of Guppy (<em>Poecilia reticulata</em>)</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genetics%20of%20Sex" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genetics of Sex</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/6/993" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/6/993.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/6/993.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/6/993.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/6/993.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
